

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/3/91/577454","@type":"ScholarlyArticle","name":"Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon  Pyrococcus furiosus ","datePublished":"2007-06-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm011","keywords":["DNA/RNA-binding protein","amino acid periodicity","support vector machine","archaea"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/3/91/577454","author":[{"name":"Fujishima, Kosuke","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka 997-0017 ,  Japan    2  Systems Biology Program, Graduate School of Media and Governance ,  Keio University ,  Fujisawa 252-8520 ,  Japan","@type":"Person"},{"name":"Komasa, Mizuki","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka 997-0017 ,  Japan    2  Systems Biology Program, Graduate School of Media and Governance ,  Keio University ,  Fujisawa 252-8520 ,  Japan","@type":"Person"},{"name":"Kitamura, Sayaka","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka 997-0017 ,  Japan    2  Systems Biology Program, Graduate School of Media and Governance ,  Keio University ,  Fujisawa 252-8520 ,  Japan","@type":"Person"},{"name":"Suzuki, Haruo","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka 997-0017 ,  Japan    2  Systems Biology Program, Graduate School of Media and Governance ,  Keio University ,  Fujisawa 252-8520 ,  Japan","@type":"Person"},{"name":"Tomita, Masaru","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka 997-0017 ,  Japan    3  Faculty of Environment and Information Studies ,  Keio University ,  Fujisawa 252-8520 ,  Japan","@type":"Person"},{"name":"Kanai, Akio","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka 997-0017 ,  Japan    3  Faculty of Environment and Information Studies ,  Keio University ,  Fujisawa 252-8520 ,  Japan","@type":"Person"}],"description":"Abstract.  Proteins play a critical role in complex biological systems, yet about half of the proteins in publicly available databases are annotated as function","pageStart":"91","pageEnd":"102","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon  Pyrococcus furiosus ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon  Pyrococcus furiosus " />
    <meta property="og:description" content="Abstract.  Proteins play a critical role in complex biological systems, yet about half of the proteins in publicly available databases are annotated as function" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/3/91/577454" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-06-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Fujishima, Kosuke" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka 997-0017, Japan" /><meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa 252-8520, Japan" /><meta name="citation_author" content="Komasa, Mizuki" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka 997-0017, Japan" /><meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa 252-8520, Japan" /><meta name="citation_author" content="Kitamura, Sayaka" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka 997-0017, Japan" /><meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa 252-8520, Japan" /><meta name="citation_author" content="Suzuki, Haruo" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka 997-0017, Japan" /><meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa 252-8520, Japan" /><meta name="citation_author" content="Tomita, Masaru" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka 997-0017, Japan" /><meta name="citation_author_institution" content="Faculty of Environment and Information Studies, Keio University, Fujisawa 252-8520, Japan" /><meta name="citation_author" content="Kanai, Akio" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka 997-0017, Japan" /><meta name="citation_author_institution" content="Faculty of Environment and Information Studies, Keio University, Fujisawa 252-8520, Japan" /><meta name="citation_title" content="Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon Pyrococcus furiosus" /><meta name="citation_firstpage" content="91" /><meta name="citation_lastpage" content="102" /><meta name="citation_doi" content="10.1093/dnares/dsm011" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=NCBI Reference Sequence RefSeq: a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D501-504; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Automated prediction of protein function and detection of functional sites from structure; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=14754-14759; " /><meta name="citation_reference" content="citation_title=A structure-based method for identifying DNA-binding proteins and their sites of DNA-interaction; J. Struct. Funct. Genomics.;  citation_year=2004;  citation_volume=5;  citation_pages=255-265; " /><meta name="citation_reference" content="citation_title=Discovery of uncharacterized cellular systems by genome-wide analysis of functional linkages; Nat. Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=1055-1062; " /><meta name="citation_reference" content="citation_title=Birth and death of orphan genes in Rickettsia; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=1575-1587; " /><meta name="citation_reference" content="citation_title=Structural biology sheds light on the puzzle of genomic ORFans; J. Mol. Biol.;  citation_year=2004;  citation_volume=342;  citation_pages=369-373; " /><meta name="citation_reference" content="citation_title=Expression cloning and characterization of a novel gene that encodes the RNA-binding protein FAU-1 from Pyrococcus furiosus; Biochem. J.;  citation_year=2003;  citation_volume=372;  citation_pages=253-261; " /><meta name="citation_reference" content="citation_title=Archaeal Pyrococcus furiosus thymidylate synthase 1 is an RNA-binding protein; Biochem. J.;  citation_year=2006;  citation_volume=393;  citation_pages=373-379; " /><meta name="citation_reference" content="citation_title=Cooperative regulation for Okazaki fragment processing by RNase HII and FEN-1 purified from a hyperthermophilic archaeon, Pyrococcus furiosus; Biochem. Biophys. Res. Commun.;  citation_year=2003;  citation_volume=309;  citation_pages=247-252; " /><meta name="citation_reference" content="citation_title=Cloning of a crustacean myosin heavy chain isoform: exclusive expression in fast muscle; J. Exp. Zool.;  citation_year=1993;  citation_volume=267;  citation_pages=578-586; " /><meta name="citation_reference" content="citation_title=Latent periodicity of serine–threonine and tyrosine protein kinases and other protein families; Comput. Biol. Chem.;  citation_year=2005;  citation_volume=29;  citation_pages=229-243; " /><meta name="citation_reference" content="citation_title=Kernel-based machine learning protocol for predicting DNA-binding proteins; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=6486-6493; " /><meta name="citation_reference" content="citation_title=Prediction of RNA-binding proteins from primary sequence by a support vector machine approach; RNA;  citation_year=2004;  citation_volume=10;  citation_pages=355-368; " /><meta name="citation_reference" content="citation_title=Support vector machines for predicting rRNA-, RNA-, and DNA-binding proteins from amino acid sequence; Biochim. Biophys. Acta.;  citation_year=2003;  citation_volume=1648;  citation_pages=127-133; " /><meta name="citation_reference" content="citation_title=Predicting rRNA-, RNA-, and DNA-binding proteins from primary structure with support vector machines; J. Theor. Biol.;  citation_year=2006;  citation_volume=240;  citation_pages=175-184; " /><meta name="citation_reference" content="citation_title=Proteins of the same fold and unrelated sequences have similar amino acid composition; Proteins;  citation_year=2006;  citation_volume=64;  citation_pages=275-279; " /><meta name="citation_reference" content="citation_title=LOCSVMPSI: a web server for subcellular localization of eukaryotic proteins using SVM and profile of PSI-BLAST; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W105-W110; " /><meta name="citation_reference" content="citation_title=pSLIP: SVM based protein subcellular localization prediction using multiple physicochemical properties; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=152" /><meta name="citation_reference" content="citation_title=InterPro—an integrated documentation resource for protein families, domains and functional sites; Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=1145-1150; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=263-266; " /><meta name="citation_reference" content="citation_title=Analysis of sequence periodicity in E. coli proteins: empirical investigation of the ‘duplication and divergence’ theory of protein evolution; J. Mol. Evol.;  citation_year=2003;  citation_volume=57;  citation_pages=149-158; " /><meta name="citation_reference" content="citation_title=Support vector machine classification on the web; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=586-587; " /><meta name="citation_reference" content="citation_title=miTarget: microRNA target gene prediction using a support vector machine; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=411" /><meta name="citation_reference" content="citation_title=Classifying noisy protein sequence data: a case study of immunoglobulin light chains; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=495-501; " /><meta name="citation_reference" content="citation_title=Comparing machine learning classifiers for diagnosing glaucoma from standard automated perimetry; Invest. Ophthalmol. Vis. Sci;  citation_year=2002;  citation_volume=43;  citation_pages=162-169; " /><meta name="citation_reference" content="citation_title=SVM-Prot: web-based support vector machine software for functional classification of a protein from its primary sequence; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3692-3697; " /><meta name="citation_reference" content="citation_title=Predicting functional family of novel enzymes irrespective of sequence similarity: a statistical learning approach; Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=6437-6444; " /><meta name="citation_reference" content="citation_title=Learning gene functional classifications from multiple data types; J. Comput. Biol;  citation_year=2002;  citation_volume=2;  citation_pages=401-411; " /><meta name="citation_reference" content="citation_title=Comparison of the predicted and observed secondary structure of T4 phage lysozyme; Biochim. Biophys. Acta.;  citation_year=1975;  citation_volume=405;  citation_pages=442-451; " /><meta name="citation_reference" content="citation_title=Genomic sequence of hyperthermophile, Pyrococcus furiosus: implications for physiology and enzymology; Meth. Enzymol.;  citation_year=2001;  citation_volume=330;  citation_pages=134-157; " /><meta name="citation_reference" content="citation_title=The SWISS-PROT protein sequence data bank and its new supplement TREMBL; Nucleic Acids Res.;  citation_year=1996;  citation_volume=24;  citation_pages=21-25; " /><meta name="citation_reference" content="citation_title=The Gene Ontology Annotation, GOA project: implementation of GO in SWISS-PROT, TrEMBL, and InterPro; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=662-672; " /><meta name="citation_reference" content="citation_title=A universal trend of amino acid gain and loss in protein evolution; Nature;  citation_year=2005;  citation_volume=433;  citation_pages=633-638; " /><meta name="citation_reference" content="citation_title=Arginine-rich regions mediate the RNA binding and regulatory activities of the protein encoded by the Drosophila melanogaster suppressor of sable gene; Mol. Cell. Biol.;  citation_year=2000;  citation_volume=20;  citation_pages=8198-8208; " /><meta name="citation_reference" content="citation_title=Identifying DNA-binding proteins using structural motifs and the electrostatic potential; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=4732-4741; " /><meta name="citation_reference" content="citation_title=Electrostatic interactions in a peptide–RNA complex; J. Mol. Biol.;  citation_year=2003;  citation_volume=331;  citation_pages=75-88; " /><meta name="citation_reference" content="citation_title=PSSM-based prediction of DNA binding sites in proteins; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=33" /><meta name="citation_reference" content="citation_title=Comparative genomics and evolution of proteins involved in RNA metabolism; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=1427-1464; " /><meta name="citation_reference" content="citation_title=RNA-binding strategies common to cold-shock domain- and RNA recognition motif-containing proteins; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=2223-2233; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/3/91/17462523/dsm011.pdf" /><meta name="description" content="Abstract.  Proteins play a critical role in complex biological systems, yet about half of the proteins in publicly available databases are annotated as function" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/3/91/577454" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/3/91/577454" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/3/91/577454");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm011';
    </script>
<title>
	Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon Pyrococcus furiosus | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./577454" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BetiPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppc2hpbWEsIEtvc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBmb3IgQWR2YW5jZWQgQmlvc2NpZW5jZXMsIEtlaW8gVW5pdmVyc2l0eSwgVHN1cnVva2EgOTk3LTAwMTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3lzdGVtcyBCaW9sb2d5IFByb2dyYW0sIEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpYSBhbmQgR292ZXJuYW5jZSwgS2VpbyBVbml2ZXJzaXR5LCBGdWppc2F3YSAyNTItODUyMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb21hc2EsIE1penVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBmb3IgQWR2YW5jZWQgQmlvc2NpZW5jZXMsIEtlaW8gVW5pdmVyc2l0eSwgVHN1cnVva2EgOTk3LTAwMTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3lzdGVtcyBCaW9sb2d5IFByb2dyYW0sIEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpYSBhbmQgR292ZXJuYW5jZSwgS2VpbyBVbml2ZXJzaXR5LCBGdWppc2F3YSAyNTItODUyMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXRhbXVyYSwgU2F5YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIGZvciBBZHZhbmNlZCBCaW9zY2llbmNlcywgS2VpbyBVbml2ZXJzaXR5LCBUc3VydW9rYSA5OTctMDAxNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTeXN0ZW1zIEJpb2xvZ3kgUHJvZ3JhbSwgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGlhIGFuZCBHb3Zlcm5hbmNlLCBLZWlvIFVuaXZlcnNpdHksIEZ1amlzYXdhIDI1Mi04NTIwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1enVraSwgSGFydW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIEFkdmFuY2VkIEJpb3NjaWVuY2VzLCBLZWlvIFVuaXZlcnNpdHksIFRzdXJ1b2thIDk5Ny0wMDE3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN5c3RlbXMgQmlvbG9neSBQcm9ncmFtLCBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWEgYW5kIEdvdmVybmFuY2UsIEtlaW8gVW5pdmVyc2l0eSwgRnVqaXNhd2EgMjUyLTg1MjAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9taXRhLCBNYXNhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIEFkdmFuY2VkIEJpb3NjaWVuY2VzLCBLZWlvIFVuaXZlcnNpdHksIFRzdXJ1b2thIDk5Ny0wMDE3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgRW52aXJvbm1lbnQgYW5kIEluZm9ybWF0aW9uIFN0dWRpZXMsIEtlaW8gVW5pdmVyc2l0eSwgRnVqaXNhd2EgMjUyLTg1MjAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FuYWksIEFraW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIEFkdmFuY2VkIEJpb3NjaWVuY2VzLCBLZWlvIFVuaXZlcnNpdHksIFRzdXJ1b2thIDk5Ny0wMDE3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgRW52aXJvbm1lbnQgYW5kIEluZm9ybWF0aW9uIFN0dWRpZXMsIEtlaW8gVW5pdmVyc2l0eSwgRnVqaXNhd2EgMjUyLTg1MjAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJQcm90ZW9tZS1XaWRlIFByZWRpY3Rpb24gb2YgTm92ZWwgRE5BL1JOQS1CaW5kaW5nIFByb3RlaW5zIFVzaW5nIEFtaW5vIEFjaWQgQ29tcG9zaXRpb24gYW5kIFBlcmlvZGljaXR5IGluIHRoZSBIeXBlcnRoZXJtb3BoaWxpYyBBcmNoYWVvbiBQeXJvY29jY3VzIGZ1cmlvc3VzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA3LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkNCSSBSZWZlcmVuY2UgU2VxdWVuY2UgUmVmU2VxOiBhIGN1cmF0ZWQgbm9uLXJlZHVuZGFudCBzZXF1ZW5jZSBkYXRhYmFzZSBvZiBnZW5vbWVzLCB0cmFuc2NyaXB0cyBhbmQgcHJvdGVpbnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPUQ1MDEtNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBwcmVkaWN0aW9uIG9mIHByb3RlaW4gZnVuY3Rpb24gYW5kIGRldGVjdGlvbiBvZiBmdW5jdGlvbmFsIHNpdGVzIGZyb20gc3RydWN0dXJlOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE0NzU0LTE0NzU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0cnVjdHVyZS1iYXNlZCBtZXRob2QgZm9yIGlkZW50aWZ5aW5nIEROQS1iaW5kaW5nIHByb3RlaW5zIGFuZCB0aGVpciBzaXRlcyBvZiBETkEtaW50ZXJhY3Rpb247IEouIFN0cnVjdC4gRnVuY3QuIEdlbm9taWNzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjU1LTI2NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIHVuY2hhcmFjdGVyaXplZCBjZWxsdWxhciBzeXN0ZW1zIGJ5IGdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIGZ1bmN0aW9uYWwgbGlua2FnZXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMDU1LTEwNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpcnRoIGFuZCBkZWF0aCBvZiBvcnBoYW4gZ2VuZXMgaW4gUmlja2V0dHNpYTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE1NzUtMTU4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBiaW9sb2d5IHNoZWRzIGxpZ2h0IG9uIHRoZSBwdXp6bGUgb2YgZ2Vub21pYyBPUkZhbnM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzQyOyAgY2l0YXRpb25fcGFnZXM9MzY5LTM3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgbm92ZWwgZ2VuZSB0aGF0IGVuY29kZXMgdGhlIFJOQS1iaW5kaW5nIHByb3RlaW4gRkFVLTEgZnJvbSBQeXJvY29jY3VzIGZ1cmlvc3VzOyBCaW9jaGVtLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNzI7ICBjaXRhdGlvbl9wYWdlcz0yNTMtMjYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmNoYWVhbCBQeXJvY29jY3VzIGZ1cmlvc3VzIHRoeW1pZHlsYXRlIHN5bnRoYXNlIDEgaXMgYW4gUk5BLWJpbmRpbmcgcHJvdGVpbjsgQmlvY2hlbS4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzkzOyAgY2l0YXRpb25fcGFnZXM9MzczLTM3OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29vcGVyYXRpdmUgcmVndWxhdGlvbiBmb3IgT2themFraSBmcmFnbWVudCBwcm9jZXNzaW5nIGJ5IFJOYXNlIEhJSSBhbmQgRkVOLTEgcHVyaWZpZWQgZnJvbSBhIGh5cGVydGhlcm1vcGhpbGljIGFyY2hhZW9uLCBQeXJvY29jY3VzIGZ1cmlvc3VzOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzA5OyAgY2l0YXRpb25fcGFnZXM9MjQ3LTI1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBvZiBhIGNydXN0YWNlYW4gbXlvc2luIGhlYXZ5IGNoYWluIGlzb2Zvcm06IGV4Y2x1c2l2ZSBleHByZXNzaW9uIGluIGZhc3QgbXVzY2xlOyBKLiBFeHAuIFpvb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI2NzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC01ODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhdGVudCBwZXJpb2RpY2l0eSBvZiBzZXJpbmXigJN0aHJlb25pbmUgYW5kIHR5cm9zaW5lIHByb3RlaW4ga2luYXNlcyBhbmQgb3RoZXIgcHJvdGVpbiBmYW1pbGllczsgQ29tcHV0LiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTIyOS0yNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtlcm5lbC1iYXNlZCBtYWNoaW5lIGxlYXJuaW5nIHByb3RvY29sIGZvciBwcmVkaWN0aW5nIEROQS1iaW5kaW5nIHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz02NDg2LTY0OTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgUk5BLWJpbmRpbmcgcHJvdGVpbnMgZnJvbSBwcmltYXJ5IHNlcXVlbmNlIGJ5IGEgc3VwcG9ydCB2ZWN0b3IgbWFjaGluZSBhcHByb2FjaDsgUk5BOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MzU1LTM2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VwcG9ydCB2ZWN0b3IgbWFjaGluZXMgZm9yIHByZWRpY3RpbmcgclJOQS0sIFJOQS0sIGFuZCBETkEtYmluZGluZyBwcm90ZWlucyBmcm9tIGFtaW5vIGFjaWQgc2VxdWVuY2U7IEJpb2NoaW0uIEJpb3BoeXMuIEFjdGEuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDg7ICBjaXRhdGlvbl9wYWdlcz0xMjctMTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHJSTkEtLCBSTkEtLCBhbmQgRE5BLWJpbmRpbmcgcHJvdGVpbnMgZnJvbSBwcmltYXJ5IHN0cnVjdHVyZSB3aXRoIHN1cHBvcnQgdmVjdG9yIG1hY2hpbmVzOyBKLiBUaGVvci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjQwOyAgY2l0YXRpb25fcGFnZXM9MTc1LTE4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbnMgb2YgdGhlIHNhbWUgZm9sZCBhbmQgdW5yZWxhdGVkIHNlcXVlbmNlcyBoYXZlIHNpbWlsYXIgYW1pbm8gYWNpZCBjb21wb3NpdGlvbjsgUHJvdGVpbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz0yNzUtMjc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MT0NTVk1QU0k6IGEgd2ViIHNlcnZlciBmb3Igc3ViY2VsbHVsYXIgbG9jYWxpemF0aW9uIG9mIGV1a2FyeW90aWMgcHJvdGVpbnMgdXNpbmcgU1ZNIGFuZCBwcm9maWxlIG9mIFBTSS1CTEFTVDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzEwNS1XMTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1wU0xJUDogU1ZNIGJhc2VkIHByb3RlaW4gc3ViY2VsbHVsYXIgbG9jYWxpemF0aW9uIHByZWRpY3Rpb24gdXNpbmcgbXVsdGlwbGUgcGh5c2ljb2NoZW1pY2FsIHByb3BlcnRpZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm/igJRhbiBpbnRlZ3JhdGVkIGRvY3VtZW50YXRpb24gcmVzb3VyY2UgZm9yIHByb3RlaW4gZmFtaWxpZXMsIGRvbWFpbnMgYW5kIGZ1bmN0aW9uYWwgc2l0ZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE0NS0xMTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNjMtMjY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBzZXF1ZW5jZSBwZXJpb2RpY2l0eSBpbiBFLiBjb2xpIHByb3RlaW5zOiBlbXBpcmljYWwgaW52ZXN0aWdhdGlvbiBvZiB0aGUg4oCYZHVwbGljYXRpb24gYW5kIGRpdmVyZ2VuY2XigJkgdGhlb3J5IG9mIHByb3RlaW4gZXZvbHV0aW9uOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MTQ5LTE1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VwcG9ydCB2ZWN0b3IgbWFjaGluZSBjbGFzc2lmaWNhdGlvbiBvbiB0aGUgd2ViOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTU4Ni01ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1pVGFyZ2V0OiBtaWNyb1JOQSB0YXJnZXQgZ2VuZSBwcmVkaWN0aW9uIHVzaW5nIGEgc3VwcG9ydCB2ZWN0b3IgbWFjaGluZTsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00MTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbGFzc2lmeWluZyBub2lzeSBwcm90ZWluIHNlcXVlbmNlIGRhdGE6IGEgY2FzZSBzdHVkeSBvZiBpbW11bm9nbG9idWxpbiBsaWdodCBjaGFpbnM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDk1LTUwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaW5nIG1hY2hpbmUgbGVhcm5pbmcgY2xhc3NpZmllcnMgZm9yIGRpYWdub3NpbmcgZ2xhdWNvbWEgZnJvbSBzdGFuZGFyZCBhdXRvbWF0ZWQgcGVyaW1ldHJ5OyBJbnZlc3QuIE9waHRoYWxtb2wuIFZpcy4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTYyLTE2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1ZNLVByb3Q6IHdlYi1iYXNlZCBzdXBwb3J0IHZlY3RvciBtYWNoaW5lIHNvZnR3YXJlIGZvciBmdW5jdGlvbmFsIGNsYXNzaWZpY2F0aW9uIG9mIGEgcHJvdGVpbiBmcm9tIGl0cyBwcmltYXJ5IHNlcXVlbmNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNjkyLTM2OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3RpbmcgZnVuY3Rpb25hbCBmYW1pbHkgb2Ygbm92ZWwgZW56eW1lcyBpcnJlc3BlY3RpdmUgb2Ygc2VxdWVuY2Ugc2ltaWxhcml0eTogYSBzdGF0aXN0aWNhbCBsZWFybmluZyBhcHByb2FjaDsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz02NDM3LTY0NDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlYXJuaW5nIGdlbmUgZnVuY3Rpb25hbCBjbGFzc2lmaWNhdGlvbnMgZnJvbSBtdWx0aXBsZSBkYXRhIHR5cGVzOyBKLiBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTQwMS00MTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdGhlIHByZWRpY3RlZCBhbmQgb2JzZXJ2ZWQgc2Vjb25kYXJ5IHN0cnVjdHVyZSBvZiBUNCBwaGFnZSBseXNvenltZTsgQmlvY2hpbS4gQmlvcGh5cy4gQWN0YS47ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9NDA1OyAgY2l0YXRpb25fcGFnZXM9NDQyLTQ1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzZXF1ZW5jZSBvZiBoeXBlcnRoZXJtb3BoaWxlLCBQeXJvY29jY3VzIGZ1cmlvc3VzOiBpbXBsaWNhdGlvbnMgZm9yIHBoeXNpb2xvZ3kgYW5kIGVuenltb2xvZ3k7IE1ldGguIEVuenltb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTMzMDsgIGNpdGF0aW9uX3BhZ2VzPTEzNC0xNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTV0lTUy1QUk9UIHByb3RlaW4gc2VxdWVuY2UgZGF0YSBiYW5rIGFuZCBpdHMgbmV3IHN1cHBsZW1lbnQgVFJFTUJMOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yMS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbmUgT250b2xvZ3kgQW5ub3RhdGlvbiwgR09BIHByb2plY3Q6IGltcGxlbWVudGF0aW9uIG9mIEdPIGluIFNXSVNTLVBST1QsIFRyRU1CTCwgYW5kIEludGVyUHJvOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTY2Mi02NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdW5pdmVyc2FsIHRyZW5kIG9mIGFtaW5vIGFjaWQgZ2FpbiBhbmQgbG9zcyBpbiBwcm90ZWluIGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzMzsgIGNpdGF0aW9uX3BhZ2VzPTYzMy02Mzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyZ2luaW5lLXJpY2ggcmVnaW9ucyBtZWRpYXRlIHRoZSBSTkEgYmluZGluZyBhbmQgcmVndWxhdG9yeSBhY3Rpdml0aWVzIG9mIHRoZSBwcm90ZWluIGVuY29kZWQgYnkgdGhlIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIHN1cHByZXNzb3Igb2Ygc2FibGUgZ2VuZTsgTW9sLiBDZWxsLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTgxOTgtODIwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgRE5BLWJpbmRpbmcgcHJvdGVpbnMgdXNpbmcgc3RydWN0dXJhbCBtb3RpZnMgYW5kIHRoZSBlbGVjdHJvc3RhdGljIHBvdGVudGlhbDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NDczMi00NzQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbGVjdHJvc3RhdGljIGludGVyYWN0aW9ucyBpbiBhIHBlcHRpZGXigJNSTkEgY29tcGxleDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzE7ICBjaXRhdGlvbl9wYWdlcz03NS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFNTTS1iYXNlZCBwcmVkaWN0aW9uIG9mIEROQSBiaW5kaW5nIHNpdGVzIGluIHByb3RlaW5zOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pY3MgYW5kIGV2b2x1dGlvbiBvZiBwcm90ZWlucyBpbnZvbHZlZCBpbiBSTkEgbWV0YWJvbGlzbTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTQyNy0xNDY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtYmluZGluZyBzdHJhdGVnaWVzIGNvbW1vbiB0byBjb2xkLXNob2NrIGRvbWFpbi0gYW5kIFJOQSByZWNvZ25pdGlvbiBtb3RpZi1jb250YWluaW5nIHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0yMjIzLTIyMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNC8zLzkxLzE3NDYyNTIzL2RzbTAxMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUHJvdGVpbnMgcGxheSBhIGNyaXRpY2FsIHJvbGUgaW4gY29tcGxleCBiaW9sb2dpY2FsIHN5c3RlbXMsIHlldCBhYm91dCBoYWxmIG9mIHRoZSBwcm90ZWlucyBpbiBwdWJsaWNseSBhdmFpbGFibGUgZGF0YWJhc2VzIGFyZSBhbm5vdGF0ZWQgYXMgZnVuY3Rpb24iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTU3NzQ1NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRC4P0xXTJyZLVK+knFYT896n+2TA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f91%2f577454" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f91%2f577454" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23256" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&amp;Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/3">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87194499">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194499">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194501">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194501">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194505">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194505">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194531">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194531">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194559">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194561">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194561">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/14/3/103/577859">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon <em>Pyrococcus furiosus</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kosuke Fujishima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka 997-0017</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Systems Biology Program, Graduate School of Media and Governance</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Fujisawa 252-8520</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kosuke+Fujishima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujishima K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kosuke Fujishima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mizuki Komasa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka 997-0017</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Systems Biology Program, Graduate School of Media and Governance</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Fujisawa 252-8520</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mizuki+Komasa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Komasa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Komasa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mizuki Komasa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sayaka Kitamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka 997-0017</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Systems Biology Program, Graduate School of Media and Governance</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Fujisawa 252-8520</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sayaka+Kitamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kitamura S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kitamura S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sayaka Kitamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruo Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka 997-0017</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Systems Biology Program, Graduate School of Media and Governance</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Fujisawa 252-8520</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruo+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haruo Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaru Tomita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka 997-0017</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Faculty of Environment and Information Studies</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Fujisawa 252-8520</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaru+Tomita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tomita M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tomita M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masaru Tomita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akio Kanai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka 997-0017</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Faculty of Environment and Information Studies</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Fujisawa 252-8520</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 235-29-0524. Fax. <div class="fax">+81 235-29-0525</div>. E-mail: <a href="mailto:akio@sfc.keio.ac.jp" target="_blank">akio@sfc.keio.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akio+Kanai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanai A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanai A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akio Kanai</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Takashi Ito.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 3, 1 January 2007, Pages 91–102, <a href='https://doi.org/10.1093/dnares/dsm011'>https://doi.org/10.1093/dnares/dsm011</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 June 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 March 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 April 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="577454" href="/dnaresearch/article-pdf/14/3/91/17462523/dsm011.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kosuke Fujishima, Mizuki Komasa, Sayaka Kitamura, Haruo Suzuki, Masaru Tomita, Akio Kanai;  Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon <em>Pyrococcus furiosus</em>, <em>DNA Research</em>, Volume 14, Issue 3, 1 January 2007, Pages 91–102, <a href='https://doi.org/10.1093/dnares/dsm011'>https://doi.org/10.1093/dnares/dsm011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/577454?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/577454?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/577454?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/577454?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/577454?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/577454?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/577454?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Proteome-Wide%20Prediction%20of%20Novel%20DNA%2FRNA-Binding%20Proteins%20Using%20Amino%20Acid%20Composition%20and%20Periodicity%20in%20the%20Hyperthermophilic%20Archaeon%20%26lt%3Bem%26gt%3BPyrococcus%20furiosus%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2007-06-15&amp;volumeNum=14&amp;issueNum=3&amp;author=Fujishima%2C%20Kosuke%3B%20Komasa%2C%20Mizuki&amp;startPage=91&amp;endPage=102&amp;contentId=10.1093%2Fdnares%2Fdsm011&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f91%2f577454" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87194499 id="87194499" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Proteins play a critical role in complex biological systems, yet about half of the proteins in publicly available databases are annotated as functionally unknown. Proteome-wide functional classification using bioinformatics approaches thus is becoming an important method for revealing unknown protein functions. Using the hyperthermophilic archaeon <em>Pyrococcus furiosus</em> as a model species, we used the support vector machine (SVM) method to discriminate DNA/RNA-binding proteins from proteins with other functions, using amino acid composition and periodicities as feature vectors. We defined this value as the composition score (CO) and periodicity score (PD). The <em>P. furiosus</em> proteins were classified into three classes (I–III) on the basis of the two-dimensional correlation analysis of CO score and PD score. As a result, approximately 87% of the functionally known proteins categorized as class I proteins (CO score + PD score &gt; 0.6) were found to be DNA/RNA-binding proteins. Applying the two-dimensional correlation analysis to the 994 hypothetical proteins in <em>P. furiosus</em>, a total of 151 proteins were predicted to be novel DNA/RNA-binding protein candidates. DNA/RNA-binding activities of randomly chosen hypothetical proteins were experimentally verified. Six out of seven candidate proteins in class I possessed DNA/RNA-binding activities, supporting the efficacy of our method.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA/RNA-binding protein&quot;">DNA/RNA-binding protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;amino acid periodicity&quot;">amino acid periodicity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;support vector machine&quot;">support vector machine</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="archaea">archaea</a></div>                            <h2 scrollto-destination=87194501 id="87194501" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The last decade has been a remarkable time in the field of genome science. DNA sequences from over 2400 species have been determined,<sup><a reveal-id="DSM011C1" data-open="DSM011C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and more are on the way. Correspondingly, the need for reliable functional annotation has become prominent. Most functional annotation is based on a sequence similarity approach,<sup><a reveal-id="DSM011C2" data-open="DSM011C2" class="link link-ref link-reveal xref-bibr">2</a></sup> but about half of the proteins registered in protein databases are classified as hypothetical because they lack similarity to functionally known proteins. Proteome-wide functional classification using bioinformatics approaches is becoming an important method for revealing unknown protein functions. For example, the recent exponential growth in Protein Data Bank entries has enabled highly accurate functional predictions to be made on the basis of structural similarities to three-dimensional profiles of proteins.<sup><a reveal-id="DSM011C3" data-open="DSM011C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM011C4" data-open="DSM011C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Comparative genome analysis using phylogenetic profiling has revealed a diversity of functional linkages among genes, and thus it can be a useful strategy for elucidating the functions of uncharacterized proteins.<sup><a reveal-id="DSM011C5" data-open="DSM011C5" class="link link-ref link-reveal xref-bibr">5</a></sup> However, although species-specific genes (ORFans) are known to encode many uncharacterized short peptides,<sup><a reveal-id="DSM011C6" data-open="DSM011C6" class="link link-ref link-reveal xref-bibr">6</a></sup> the functions of these peptides are difficult to predict with certainty using comparative genomics because they lack homology to those sequences currently in databases. More than 23 000 ORFans have been found in 60 microbial genomes, and, on the basis of structural studies, many are likely to encode expressed, functional, or even essential proteins.<sup><a reveal-id="DSM011C7" data-open="DSM011C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Therefore, alternative bioinformatics methods that can predict these uncharacterized protein functions at the proteome level are very useful.</p><p>For the past few years, we have been working on RNA metabolism in the hyperthermophilic archaeon <em>Pyrococcus furiosus</em><sup><a reveal-id="DSM011C8  DSM011C10" data-open="DSM011C8  DSM011C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> and reported on our experimental system in which an expression cloning method is used for extracting DNA/RNA-binding proteins at the proteome level. During this work, we observed that charged amino acids—such as aspartic acid, glutamic acid, arginine, and lysine—appeared both in the sequence of the novel RNA-binding protein FAU-1 and Ribonuclease E in a periodic manner.<sup><a reveal-id="DSM011C8" data-open="DSM011C8" class="link link-ref link-reveal xref-bibr">8</a></sup> It is possible that certain acidic and basic amino acid periodicities might affect the secondary or tertiary structure of a protein so that it gains DNA/RNA-binding activities. Amino acid periodicities are commonly observed features in the sequences of various proteins such as myosin and amyloids,<sup><a reveal-id="DSM011C11" data-open="DSM011C11" class="link link-ref link-reveal xref-bibr">11</a></sup> serine–threonine, and tyrosine protein kinases<sup><a reveal-id="DSM011C12" data-open="DSM011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and are known to be strongly correlated with their secondary structures.</p><p>The purpose of the current study was to demonstrate that a bioinformatics approach focusing on the periodicity in a protein's primary structure could be a suitable method for elucidating DNA/RNA-binding proteins. Previously, several support vector machine (SVM)-based methods were developed towards predicting DNA-binding and RNA-binding proteins on the basis of various amino acid profiles (i.e. overall composition, pseudo-amino acid composition, surface composition, electrostatic potential, and hydrophobicity).<sup><a reveal-id="DSM011C13  DSM011C16" data-open="DSM011C13  DSM011C16" class="link link-ref link-reveal xref-bibr">13–16</a></sup> SVM is one of the most powerful supervised learning algorithm that has recently widely been used in the field of bioinformatics. We describe here an SVM-based method for classifying known DNA/RNA-binding proteins from <em>P. furiosus</em> using amino acid composition and periodicity as feature vectors. The discriminant values (SVM output) derived from these profiles were defined as two new indices: composition (CO) score and periodicity (PD) score. Amino acid composition are known to be strongly correlated with protein secondary structure class<sup><a reveal-id="DSM011C17" data-open="DSM011C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and subcellular localization<sup><a reveal-id="DSM011C18" data-open="DSM011C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM011C19" data-open="DSM011C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and are assumed to support the protein function classification. Therefore, on the basis of the two-dimensional correlation analysis, we combined amino acid composition (CO score) with the PD score to further improve the performance of DNA/RNA-binding protein prediction. The two-dimensional correlation analysis was then applied to hypothetical proteins of <em>P. furiosus</em>, and promising candidates for being novel DNA/RNA-binding proteins were selected. DNA/RNA-binding activities of these candidate proteins were examined experimentally and many of them were confirmed to possess DNA/RNA-binding activities.</p>                            <h2 scrollto-destination=87194505 id="87194505" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87194506 id="87194506" class="section-title" data-legacy-id=s2a>2.1. Protein data set and functional annotations</h3>
<p>Automated annotations and amino acid sequences of proteins from the two archaeal species, <em>P. furiosus</em> (2057 proteins) and <em>Sulfolobus solfataricus</em> (2934 proteins), were taken from the EMBL database (<a class="link link-uri" href="http://www.ebi.ac.uk/embl/">http://www.ebi.ac.uk/embl/</a>; Release 83, June 2005). Each protein entry has a UniProt Knowledgebase (UniProtKB) accession code corresponding to its entry in either UniProtKB/Swiss-Prot (<a class="link link-uri" href="http://www.ebi.ac.uk/Swiss-Prot/">http://www.ebi.ac.uk/Swiss-Prot/</a>; Release 47, May 2005) or UniProtKB/TrEMBL (<a class="link link-uri" href="http://www.ebi.ac.uk/trembl/">http://www.ebi.ac.uk/trembl/</a>; Release 31, September 2005). Both databases contain information on the gene ontology annotation (GOA: a combination of electronic assignment and manual annotation), and protein data are from the domain databases InterPro<sup><a reveal-id="DSM011C20" data-open="DSM011C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and Pfam.<sup><a reveal-id="DSM011C21" data-open="DSM011C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Swiss-Prot data were used for the four prokaryotic and eukaryotic species—<em>Bacillus subtilis</em> (2799 proteins), <em>Escherichia coli</em> K12 MG1655 (4465 proteins), <em>Arabidopsis thaliana</em> (3454 proteins), and <em>Caenorhabditis elegans</em> (2655 proteins)—	as a reliable independent test set.</p><p>We defined ‘functionally known proteins’ as functionally annotated proteins in the Swiss-Prot or TrEMBL databases with additional GOA. TrEMBL protein entries with no additional annotation were categorized as ‘putative functional proteins’. Proteins annotated as ‘hypothetical’ in the database were defined as ‘hypothetical proteins’. DNA/RNA-binding proteins were defined as those proteins whose annotations included the following keywords in Swiss-Prot, TrEMBL, and GOA annotations: DNA, RNA, ribosome(al), RNP, ribonucleo-, helicase, nuclease, or nucleic acid binding. To reduce the bias of functional variety in the protein data set, the functionally known proteins of the six model species were filtered to remove homologous proteins at sequence identity level with E-value &lt; 1 × 10<sup>−4</sup> and short peptides &lt; 20 amino acids from future analyses. In total, we prepared 477 proteins of <em>P. furiosus</em>, 582 proteins of <em>S. solfataricus</em>, 914 of <em>B. subtilis</em>, 1436 of <em>E. coli</em>, 865 of <em>A. thaliana</em>, and 566 of <em>C. elegans</em> as a ‘representative set’ for the analysis (Table <a reveal-id="DSM011TB1" data-open="DSM011TB1" class="link link-reveal link-table xref-fig">1</a>).</p>                        <a id="87194509" scrollto-destination="87194509"></a>
<div content-id="DSM011TB1" class="table-modal"><div class="table-wrap"><div id="DSM011TB1" class="table-wrap-title" data-id="DSM011TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Functional classification table of the proteome data set of six model species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Species </th><th rowspan="3">Database </th><th colspan="5">Functionally known </th><th rowspan="3">Putative </th><th rowspan="3">Hypothetical </th><th rowspan="3">Total protein </th></tr><tr><th colspan="3">Representative set<sup>a</sup> </th><th colspan="2" rowspan="2">Redundant Total </th></tr><tr><th>DNA/RNA </th><th>Others </th><th>Total </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>TrEMBL + Swiss-Prot </td><td>157 </td><td>320 </td><td>477 </td><td>465 </td><td>942 </td><td>121 </td><td>994 </td><td>2057 </td></tr><tr><td><em>S. Solfataricus</em> </td><td>TrEMBL + Swiss-Prot </td><td>184 </td><td>398 </td><td>582 </td><td>730 </td><td>1312 </td><td>302 </td><td>1320 </td><td>2934 </td></tr><tr><td><em>B subtilis</em> </td><td>Swiss-Prot </td><td>204 </td><td>710 </td><td>914 </td><td>908 </td><td>1822 </td><td>1 </td><td>976 </td><td>2799 </td></tr><tr><td><em>E. coli</em> </td><td>Swiss-Prot </td><td>346 </td><td>1090 </td><td>1436 </td><td>1889 </td><td>3325 </td><td>1 </td><td>1139 </td><td>4465 </td></tr><tr><td><em>A. thaliana</em> </td><td>Swiss-Prot </td><td>223 </td><td>642 </td><td>865 </td><td>2389 </td><td>3254 </td><td>56 </td><td>144 </td><td>3454 </td></tr><tr><td><em>C. elegans</em> </td><td>Swiss-Prot </td><td>165 </td><td>401 </td><td>566 </td><td>1215 </td><td>1781 </td><td>0 </td><td>874 </td><td>2655 </td></tr></tbody></table></div><div content-id="DSM011TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Species </th><th rowspan="3">Database </th><th colspan="5">Functionally known </th><th rowspan="3">Putative </th><th rowspan="3">Hypothetical </th><th rowspan="3">Total protein </th></tr><tr><th colspan="3">Representative set<sup>a</sup> </th><th colspan="2" rowspan="2">Redundant Total </th></tr><tr><th>DNA/RNA </th><th>Others </th><th>Total </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>TrEMBL + Swiss-Prot </td><td>157 </td><td>320 </td><td>477 </td><td>465 </td><td>942 </td><td>121 </td><td>994 </td><td>2057 </td></tr><tr><td><em>S. Solfataricus</em> </td><td>TrEMBL + Swiss-Prot </td><td>184 </td><td>398 </td><td>582 </td><td>730 </td><td>1312 </td><td>302 </td><td>1320 </td><td>2934 </td></tr><tr><td><em>B subtilis</em> </td><td>Swiss-Prot </td><td>204 </td><td>710 </td><td>914 </td><td>908 </td><td>1822 </td><td>1 </td><td>976 </td><td>2799 </td></tr><tr><td><em>E. coli</em> </td><td>Swiss-Prot </td><td>346 </td><td>1090 </td><td>1436 </td><td>1889 </td><td>3325 </td><td>1 </td><td>1139 </td><td>4465 </td></tr><tr><td><em>A. thaliana</em> </td><td>Swiss-Prot </td><td>223 </td><td>642 </td><td>865 </td><td>2389 </td><td>3254 </td><td>56 </td><td>144 </td><td>3454 </td></tr><tr><td><em>C. elegans</em> </td><td>Swiss-Prot </td><td>165 </td><td>401 </td><td>566 </td><td>1215 </td><td>1781 </td><td>0 </td><td>874 </td><td>2655 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Representative set consists of proteins with amino acid length &gt; 20 and homology redacted using BLASTP (E-value &lt; 1 × 10<sup>−4</sup>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194509" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM011TB1" class="table-wrap-title" data-id="DSM011TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Functional classification table of the proteome data set of six model species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Species </th><th rowspan="3">Database </th><th colspan="5">Functionally known </th><th rowspan="3">Putative </th><th rowspan="3">Hypothetical </th><th rowspan="3">Total protein </th></tr><tr><th colspan="3">Representative set<sup>a</sup> </th><th colspan="2" rowspan="2">Redundant Total </th></tr><tr><th>DNA/RNA </th><th>Others </th><th>Total </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>TrEMBL + Swiss-Prot </td><td>157 </td><td>320 </td><td>477 </td><td>465 </td><td>942 </td><td>121 </td><td>994 </td><td>2057 </td></tr><tr><td><em>S. Solfataricus</em> </td><td>TrEMBL + Swiss-Prot </td><td>184 </td><td>398 </td><td>582 </td><td>730 </td><td>1312 </td><td>302 </td><td>1320 </td><td>2934 </td></tr><tr><td><em>B subtilis</em> </td><td>Swiss-Prot </td><td>204 </td><td>710 </td><td>914 </td><td>908 </td><td>1822 </td><td>1 </td><td>976 </td><td>2799 </td></tr><tr><td><em>E. coli</em> </td><td>Swiss-Prot </td><td>346 </td><td>1090 </td><td>1436 </td><td>1889 </td><td>3325 </td><td>1 </td><td>1139 </td><td>4465 </td></tr><tr><td><em>A. thaliana</em> </td><td>Swiss-Prot </td><td>223 </td><td>642 </td><td>865 </td><td>2389 </td><td>3254 </td><td>56 </td><td>144 </td><td>3454 </td></tr><tr><td><em>C. elegans</em> </td><td>Swiss-Prot </td><td>165 </td><td>401 </td><td>566 </td><td>1215 </td><td>1781 </td><td>0 </td><td>874 </td><td>2655 </td></tr></tbody></table></div><div content-id="DSM011TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Species </th><th rowspan="3">Database </th><th colspan="5">Functionally known </th><th rowspan="3">Putative </th><th rowspan="3">Hypothetical </th><th rowspan="3">Total protein </th></tr><tr><th colspan="3">Representative set<sup>a</sup> </th><th colspan="2" rowspan="2">Redundant Total </th></tr><tr><th>DNA/RNA </th><th>Others </th><th>Total </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>TrEMBL + Swiss-Prot </td><td>157 </td><td>320 </td><td>477 </td><td>465 </td><td>942 </td><td>121 </td><td>994 </td><td>2057 </td></tr><tr><td><em>S. Solfataricus</em> </td><td>TrEMBL + Swiss-Prot </td><td>184 </td><td>398 </td><td>582 </td><td>730 </td><td>1312 </td><td>302 </td><td>1320 </td><td>2934 </td></tr><tr><td><em>B subtilis</em> </td><td>Swiss-Prot </td><td>204 </td><td>710 </td><td>914 </td><td>908 </td><td>1822 </td><td>1 </td><td>976 </td><td>2799 </td></tr><tr><td><em>E. coli</em> </td><td>Swiss-Prot </td><td>346 </td><td>1090 </td><td>1436 </td><td>1889 </td><td>3325 </td><td>1 </td><td>1139 </td><td>4465 </td></tr><tr><td><em>A. thaliana</em> </td><td>Swiss-Prot </td><td>223 </td><td>642 </td><td>865 </td><td>2389 </td><td>3254 </td><td>56 </td><td>144 </td><td>3454 </td></tr><tr><td><em>C. elegans</em> </td><td>Swiss-Prot </td><td>165 </td><td>401 </td><td>566 </td><td>1215 </td><td>1781 </td><td>0 </td><td>874 </td><td>2655 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Representative set consists of proteins with amino acid length &gt; 20 and homology redacted using BLASTP (E-value &lt; 1 × 10<sup>−4</sup>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194509" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87194510 id="87194510" class="section-title" data-legacy-id=s2b>2.2. Amino acid periodicity</h3>
<p>To analyze amino acid periodicities, we used eight physico-chemical profiles (chemical, Sneath, Dayhoff, Stanfel, functional, charge, structural, and hydrophobicity)<sup><a reveal-id="DSM011C22" data-open="DSM011C22" class="link link-ref link-reveal xref-bibr">22</a></sup> to subdivide the 20 common amino acids into groups. For example, the ‘charge’ profile divided the 20 amino acids into three groups: DE, RKH, and others (ACFGILMNPQSTVW). In total, 23 amino acid groups were identified: DE, RK, NQ, CM, ST, ILV, RKH, FYW, AGP, MNQ, CST, DEQN, FHWY, AGPST, GAVLIP, DERKH, CGNQSTY, ACGPSTWY, RNDQEHK, ILMFV, AFILMPVW, ACGILMPSTV, and CDEGHKNQRSTY.</p><p>Amino acid periodicity was defined as the regular appearance of a certain amino acid group (<em>X</em>), <em>Y</em> (<em>Y</em> ≥ 3) times in a protein sequence with a period (the number of amino acids from one appearance to the next) of Z. Although a previous analysis in <em>E. coli</em> defined the range of periodicity as 2 to 50, to eliminate binal periodicities (ex: period 5 includes period 10), we used prime numbers and their multiples [2, 3, 5, 7, 8 (2 × 4), 9 (3 × 3), 11, 13, 15 (5 × 3), 17, 19]. To take into account the fluctuation of periodicities, we set the error range as ± 1. For example, in seq1 (XXXXAXXAXXXX), ‘A’ appears only twice, so no periodicity can be defined. Seq2 (XXBXXXXBXXXXBX) contains three ‘Bs’ with a period of five (‘B-5’ periodicity). Seq3 (XCXXXCXXCXXXCXXCX) contains five ‘Cs’ with multiple periodicities (two of length 3, two of length 4, and two of length 7). On the basis of the error range ± 1, length 4 is included in length 3; therefore, Seq3 is defined to have ‘C’ periods of only 3 and 7.</p>                            <h3 scrollto-destination=87194513 id="87194513" class="section-title" data-legacy-id=s2c>2.3. SVM classification of DNA/RNA-binding proteins based on amino acid periodicity and composition</h3>
<div class="block-child-p">SVM is a non-linear classifier creating a maximum-margin hyperplane by applying a kernel trick to the feature vectors. We performed two different SVM analysis on the basis of the individual data set of amino acid periodicity and amino acid composition. For amino acid periodicity, we calculated the relative coverage of the periodic region (<em>R</em>) of each training set (<em>i</em>) with 253 patterns of amino acid periodicities (<em>j</em>): 23 amino acid groups × 11 kinds (2, 3, 5, 7, 8, 9, 11, 13, 15, 17, 19 periods): <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01106.gif?Expires=2147483647&Signature=q~PdaOd6oidYXPDeUFSwWmzU~rusv~vNviBlHnltequt81bj5k0w3xXhw~3Yxbq0KSt7qFPHMads8YQC8qGVKCqLNB2B9YY9~huL1s~jDWntNrPyoeT7ypfcEUgE087bG~3YRa1pKEp0hirumyHwTJKmjVwtyS0U6lijgJ2nadugY~S69czsPm0bWUkym-ITLyE9q5UE6AvNR1M~mN7Qrwv2IlwTPmNl8h1leU3BoTp85JBRDzU36EQLq~PA5ExtinWrZ~84o1u6mUSUrU-aMwzot9gSiIB~iUg0gGnc~SM9TVvFVFRmgvr8AtVIdI5503sSR3oLhwgPplRRoI4GOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01106.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01106.gif?Expires=2147483647&Signature=ssGvyyAnycBZOGlOeqtqcNkz95B372lqAIEmtIY2Vb0TmTX1tWOgCf8V3lKsdlTbnOf2VgP3wzA3lGZZ3lKOP-ROw5ok9o78w8q6hBfJZd8ufR4eCbZEEn7Pa4StR60ajhIhvYeJbMowqJQEw0YOgBqRoSrgGAURgzRHSmgiHIC3KLXVp1mCs4wONSyauQhWkCK~fgWuyvR6q1F6QnUd0rZYSJ7TYiqzc-FRN7aUQBrMVbNDyJiUdmiHMgiNkrlnsq3C3ZOZKHRUDC6XTFURIz3J4HswDvUGjOd-HUsD5wQZHM89L21w2HvStw86tv1JrGyaQhVA0KxHk4uf-6ScDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm01106.gif" /></a></div></div></div> where <em>P</em> is the length of periodic region of periodicity <em>j</em> in a single protein <em>i</em>, and <em>N</em> is the full amino acid length of a single protein <em>i</em>. Thus, a transformed feature space is created from 253-dimensional feature vectors of periodic region <em>R</em>.</div><div class="block-child-p">For amino acid composition, we calculated the relative composition of amino acid (C) of each training set (<em>i</em>) with 20 types of amino acids (<em>k</em>) <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01107.gif?Expires=2147483647&Signature=qcrJKtgqPf~XeZ11A5bMflKhEvxjw9LZfl0hgI18J-eYorme6BB1JKW6lAdUoZer2dYO4oHKuq9oOHknIHLyoB88XBq0GGa6OkTcbLnHyG4cgxJ1hVTdZdfOZUjJ2Hq46sao-hHKsIB19HLXbQwfriPkebGfQg-Q~i465avYGv6BM~JhnAvJuNxrEGNHiJ13fnime3wMtIuuIxtQZKoxIzsvFH4pZad~JXPX1bPCGXpol0SeSQ-23Txci~PtbfHSCaP-enGT5TRGt3uoj~p~-CW1d8h6S2Oo59UAfL-gTO1vhDm8KaJ6OpX5QAKR00Wm4t26zz-oaIHe1exLLyFQRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01107.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01107.gif?Expires=2147483647&Signature=OkiX6zflT7L1deLR513wr7FNl0NPXMVe6K2rAFoPUAb67WmptbC0eFS~NcxSSzcKeaeaAYsH-2Z8tUEiKCShUnm37X17Sd-Wk9DEYaS7I0tTkIbbvBQTkLfN2an-qGJu5VktSK6Eq3UHjaQA8Y1SnIKc9xR7Em7FIfXaDu-kEckExXQMzvDb6u2fdzPpPJ4wvs7w5etGaoZI34evzmqghfyVANHl5w7scigxSWCDukuZnG~r9DlKfXgvCHxquX06n08UN3~O46bEbpFzeeFcp-~MrHBfhYimHT9tW9ebL1axszJAFHygBWLvrdVAYGuZUW4JSbRkNSM5DE64ZiA2BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm01107.gif" /></a></div></div></div> where <em>A</em> is the number of amino acid <em>k</em> in a single protein <em>i</em>, and <em>N</em> is the full amino acid length of a single protein <em>i</em>.</div><p>These factors were applied as feature vectors and classified into two distinct members: DNA/RNA-binding proteins and proteins with other functions. For SVM training, the data label for DNA/RNA-binding proteins was denoted as 1 and proteins with other functions was denoted as −1. SVM analysis in this study was performed using the default parameters in Gist package version 2.3, which contains software tools for SVM classification.<sup><a reveal-id="DSM011C23" data-open="DSM011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We have tested two types of kernel function (linear and radial basis) and selected the kernel function with higher prediction performance. As a consequence, maximum-margin hyperplane was applied to the protein test set on the basis of a radial basis function kernel (<em>r</em> = 1), and the discriminant value of each protein were defined as the PD score. Likewise, linear kernel-based maximum-margin hyperplane was applied for the protein set on the basis of amino acid composition, and discriminant values were defined as the CO score.</p>                            <h3 scrollto-destination=87194517 id="87194517" class="section-title" data-legacy-id=s2d>2.4. Validation of PD score performance</h3>
<p>The performance of the PD score at predicting novel proteins was validated on the basis of 10-fold cross-validation test. The 10-fold cross-validation test is one of the most reliable methods for estimating the performance of the predictor. For example, the 477 representative data set of <em>P. furiosus</em> was randomly split into 10 mutually exclusive subsets <em>D</em><sub>1</sub>, <em>D</em><sub>2</sub>, … , <em>D</em><sub>10</sub> of approximately equal size. Each subset was tested on the basis of the training using the rest of the nine subsets. Estimated accuracies were derived as average values.</p><div class="block-child-p">First, the classification accuracy of the PD score was compared with that of the randomly chosen single amino acid periodicities using receiver operating characteristic (ROC) studies. The ROC curve is a simple and effective method to compare the overall prediction performance of different methods including SVMs.<sup><a reveal-id="DSM011C24  DSM011C26" data-open="DSM011C24  DSM011C26" class="link link-ref link-reveal xref-bibr">24–26</a></sup> The ROC curve is represented by two indices: sensitivity and specificity. The sensitivity and specificity of the PD score were calculated using a 10-fold cross-validation test with a PD score cut-off of 0. Equations are represented as follows: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01108.gif?Expires=2147483647&Signature=gOKXr1TW5xShe8RDLuzutE29bowpG5SQb5EoVns28SIKu5085xlSjsSc9cttiTUTQ17h9twRhi43E84POHWZt9ytijyWflwMtpgGp3aIXVZOmkxb3hW6lcVzXNfgGetI42Pzb6e8LxAsd3EQisER2UNSbgaJsq5FFuP27RIzKRnslFc3l2LeenGNdQAriLqcjsnRN5Sakt3zHjOddflvkkNvktySRmz3r3owtkvB66q~lG7gp~uHtdDnWwF20jC3mavNH1SzIWjLma3LCZKyASUMQ53ZO2MWsI2YJecDb0oW0PNIqzEJdl7bds7-5xVpuqttnvP8hpD3y6VJnzF90g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01108.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01108.gif?Expires=2147483647&Signature=OyGl6WzZ2m0lxa~WJ3zCpRY2IsjiCjtls5Zxn4~ud76i8I5e8cYJfvtVIoryktrxTjqQumQIUmCMaKAQW-ZogPArDNRgZ3tdzNrdgWj9USkuRfil1NAoEU4FzZDT6N8K~-PunimVDOkjW2mFYdeHZGkiFA8eY2FNCm4DXbskTCsbd~qNYnQc~Cj9w9mhqPYsl1yeiV10Y~PykexfvckSFpOE6NvnzNT01EUGiUwZVQBHBJecdTF4nqzcMO0jaHfTKRk7~jt5wWGHT2idROFHXUya1-6iPgnWQAk2fIN8QF~VEa-SAg2F7dGP3jjEcwnhc5fUpgOcXdlMEOVdb8VK~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm01108.gif" /></a></div></div></div> where TP refers to true positive (number of DNA/RNA-binding proteins with PD score &gt; cut-off), FP refers to false positive (number of other proteins with PD score &gt; cut-off), FN refers to false negative (number of DNA/RNA-binding proteins with PD score &lt; cut-off), and TN refers to true negative (number of other proteins with PD score &lt; cut-off).</div><div class="block-child-p">Error bars were added for each data set representing the standard deviation values derived from the 10-fold cross-validation test. Secondly, PD score was compared against CO score (amino acid composition-based SVM) and other SVM-based protein function predictor, SVM-Prot.<sup><a reveal-id="DSM011C27" data-open="DSM011C27" class="link link-ref link-reveal xref-bibr">27</a></sup> To assess the PD score performance, we calculated the overall accuracy (ACC) for PD score using 10-fold cross-validation test. Training data set of SVM-Prot is fixed as a combination of 54 functional protein families and predicts several functional classes owing to the probability of correct prediction. SVM-Prot uses 1943 positive set and 1353 negative set for training DNA-binding proteins and 871 positive set and 1120 negative set for training RNA-binding proteins. In total, 104 feature vectors [composition (C), transition (T), and distribution (D)] were calculated for each amino acid group classified by four physicochemical properties, hydrophobicity, Van der Waals volume, polarity, and polarizability, and were introduced to generate the hyperplane for each protein family.<sup><a reveal-id="DSM011C27" data-open="DSM011C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSM011C28" data-open="DSM011C28" class="link link-ref link-reveal xref-bibr">28</a></sup> To equally validate the prediction performance of SVM-Prot with our method, the proteins that were predicted as DNA/RNA-related with the highest probability were regarded as DNA/RNA-binding proteins. SVM-Prot was applied to the representative data set and ACC is calculated as follows: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01109.gif?Expires=2147483647&Signature=hutZ90PomGtTK3tq-QCADcsuUwwhL5QwkkwhE7ZtJPBn44-oDVhfmb~JnKGLaJq1N1pF~2AqL0UzOl4IWA5chpt9Xe0M5vWUaw3Aa3RyyLRx4dhP9eOuQdLUz4~Bz9eYblhseCCjvNYItYWhg1ind8DilPJBa2o35V8FXsoEJHLQ9C3w0zDvN71BsENRClSxDSQdTNo5c4auiuJ5DgE7qpsa0KEYnsP6DVmS3XPjUAnIGwPNejnrDK~XOYKEkyiAwnMPQ6CTMYq7R~oNS4D1lj30Pl9FDh9YCl6NjRL1AuxqYbo2qKnk0-BiI3npSBTUJ6PjE2Y5p5hJ~~8Sd4Fqgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01109.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01109.gif?Expires=2147483647&Signature=NXuPHjmpL3TqAAAlxMbXoM1foiklY1C1hAVyFo680aJdSHHN0cCPjEtDsAmZ1gpdZyPp7nFFHEOaoSz6oGez3CJmevvTICqlJ1PXV~By5PCoZ~75GwQRP1f6MOxOmmWAbJsRuIqdbgPjIekvqA8okM4ynrflO01maFjLYLe7PkKaSRNbdF69CyLSDwO71GBB19ds2gVbFTLXh0ZyB~jCLNcjK4yaivKMAiTLQoVE7tAHr1Z9IKTpuUCcrfYO2b22JYRdDSKw0n0nOxv9zE7~P8bezgSTl7lwayFg8Rg04F8yD0-5obMYN5WiHgxsTcUUyqVBex~KcRcvuKEYn25pJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm01109.gif" /></a></div></div></div></div>                            <h3 scrollto-destination=87194521 id="87194521" class="section-title" data-legacy-id=s2e>2.5. Data fusion and threshold determination</h3>
<p>Two heterogeneous data set (amino acid periodicity and amino acid composition) were integrated by three different approaches (early, intermediate, and late integration).<sup><a reveal-id="DSM011C29" data-open="DSM011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Early integration performs single SVM with a feature vector of 253 (periodicity) and 20 (composition) dimensions. For intermediate integration, kernel values (kernel matrix) are separately computed on each data type and the summed kernel values are used for the training of SVM. Late integration performs SVM separately and later, the discriminant values were summed (i.e. CO + PD score). Three evaluation indices, SE, SP, and ACC, were calculated for the evaluation of three different integration methods.</p><div class="block-child-p">Thresholds for extracting DNA/RNA-binding proteins were determined by considering several indices. An index, positive predictive value (PPV), was adapted to measure the percentage of DNA/RNA-binding proteins among proteins above threshold (blue line in Supplementary Fig. S1). PPV is calculated as follows: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01110.gif?Expires=2147483647&Signature=tf22p1D-W2bHLSr1e5V1D7H2CXKibbVyEdYowNISdsR4ooIN~P0ngNl3hYhj-nBSpCoAuGle45c5EZgnU0U88Fhvyj8ag9YD3NbPImFk2jCXEN59RxNlZmFSUyDT3ONIuFVy3cT6dGqCfQWbbUjPfJnqDIPHS89fSq--4LiyEqLFaqGbHwPszJGHET~iXvRnhJBsp~YQ7FLv-XynjkP725nRIoEWSdA0rwVtLCDq3QMaFTx0SphaC19tCYDwH3dYtbuUHnm4yKHPEoaVpqksONsGJHoji5JKVpjdeV2CviGk24GyKCergEJN0ziQHR0Gk7zaZsOsOHj3M9hUfvOJGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01110.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01110.gif?Expires=2147483647&Signature=oY4i181HWcHlpf7pwTBP2SiD65St8~MlfcS0nhxxcuTO~HLSzcCTvPGLsFXqsXe6z9OZMJZinbMB-~x4m6FJuYISdo7wEF-S8RJYabS6wNaU4vk1QsOIx~jr6GPDq0vZpMr0BReCls~7uzEhxnPYMzVElDBRhhpsze7pJnHexdAlHA08WE7nJDnJrRqTZ8kodrKpRP2p7oMt~FRnqFhV2TPw49vlTx~FCl5KQPIKfejB1fOLpUJWW4d3EmhnSYBQefl40LQHHXk9TQ~K27zsf7X2qkUmtvI8w~IrryBWPEb-rhpZUEwcQZwKr7512fWx53TXgJYOe2P0eltlOTRP7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm01110.gif" /></a></div></div></div></div><div class="block-child-p">The final prediction decision is given by the calculated value of the Matthews correlation coefficient (MCC)<sup><a reveal-id="DSM011C30" data-open="DSM011C30" class="link link-ref link-reveal xref-bibr">30</a></sup> to determine the threshold value of the CO + PD score. MCC is a popular index for measuring the performance of prediction; maximum MCC provides efficient sensitivity and specificity. <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01111.gif?Expires=2147483647&Signature=QCGpcxPETPRvxOorVdHVROptYdlKUuKoN5eX46Lp6SYTfc4kWNBrLrhaILKpYwX0m1OPNOc2jE6x4jxFEPUZPmno172JflIKLQCpP2KmXVPf7W6LeWZy6VvGTHnoNfzaMsqH7Q~a2JV0NY5AtFxNVvTDFlmDqBoKGumTOGxFLkUBaMmUkRyBLUY7Z9MGV5-hkdq7YR2o4rbv7qRATKDfxvJsCOsgIzu-8yf9R1X0~H~bTe8F9YEJpCPowLqDVUZY3Eo-z5kUsJRdH43uzI8SNyHN6qZsHS0wRkr-fIxcK8UWss8uu0cmbWTJuvWo9ZqELYugWbb~A8OIO1JOM-Opog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01111.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01111.gif?Expires=2147483647&Signature=WlDaqLqRWyZfEEh~27jDNRWw7Z-Rv9YJYCTo-gimOR2HhP7nAWNNs0iS1ZMGx2xynYpBaG2EGWpKyhzoA5w6MTKP5ydHhzATYiFYUWvhIkzJPx9OB2h1j-K5LiRYJYHVP1Z0rP1XKRJ4e2vw9k4Z9j8dE8pleDUIY9QRaG2BqeljVhRwP1X9Jbr7~p6Wy9VP~VdZG2hCKPCZY67YoUCG~-jJekP8XUR4wcMa3v0QKKiz7sBxWX79MImI1dhv4oxocs5It5CjAC3GviM6Yx~dmDsYULQkKs-U9GEvCTb9HVDuMqlCDDHAzEZaPYqkH2dDYzrP9I6GeC9jdeJfEoAiuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm01111.gif" /></a></div></div></div> where TP refers to true positive (the number of DNA/RNA-binding proteins with CO + PD score &gt; cut-off), FP refers to false positive (the number of other proteins with CO + PD score &gt; cut-off), FN refers to false negative (the number of DNA/RNA-binding proteins with CO + PD score &lt; cut-off), and TN refers to true negative (the number of other proteins with CO + PD score &lt; cut-off).</div>                            <h3 scrollto-destination=87194525 id="87194525" class="section-title" data-legacy-id=s2f>2.6. Construction of expression vectors and purification of His-tagged recombinant proteins</h3>
<p>Genomic DNA of <em>P. furiosus</em> DSM3638 was isolated using a GNOME kit (BIO101, La Jolla, CA, USA) and partially digested with the restriction enzyme <em>Sau</em>3AI. The resulting DNA fragments were fractionated by electrophoresis on a 0.7% agarose gel. Fragments of 15kb were extracted from the gel and used as templates for PCR cloning. After PCR amplification using site-specific primers (Supplementary Table S4) with <em>Nde</em>I and <em>Xho</em>I sites at the 5′ and 3′ termini, respectively, each of the candidate genes was cloned into the pET-23b expression vector (Novagen, Madison, WI, USA). Insert DNA was sequenced and shown to be identical to database sequences.</p><p>Recombinant proteins were prepared as described previously.<sup><a reveal-id="DSM011C8" data-open="DSM011C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Briefly, <em>E. coli</em> strain BL21(DE3) was transformed with each expression plasmid; however, optimal protein production required <em>E. coli</em> strain BL21(DE3)pLysS for the expression of PF0565 and PF1473 proteins and strain HMS174(DE3)pLysS for the expression of PF1498. Transformants were grown at 37°C in Luria–Bertani medium containing 50µg/mL ampicillin and supplemented with 0.4mM isopropylthio-b-galactoside. After 14–16h of further growth at 30°C, cells were harvested by centrifugation (5000<em>g</em> for 10min at 4°C), and the recombinant proteins were released by sonication (2min) in buffer A (20mM Tris–HCl, pH 8.0, 5mM imidazole, 500mM NaCl, 0.1% NP40). The extracts were heat-treated at 85°C for 15min to destroy <em>E. coli</em> endogenous proteins and then centrifuged at 12000<em>g</em> for 10 min at 4°C to remove cellular debris. The recombinant proteins were purified in an Ni<sup>2+</sup>–Sepharose column according to the manufacturer's instructions (Amersham Pharmacia, Piscataway, NJ, USA). The peaks of the eluted proteins were pooled and dialyzed against buffer B (50mM Tris–HCl, pH 8.0, 1mM EDTA, 0.02% Tween 20, 7mM 2-mercaptoethanol, 10% glycerol).</p>                            <h3 scrollto-destination=87194528 id="87194528" class="section-title" data-legacy-id=s2g>2.7. Gel-shift assay</h3>
<p>Hokkaido System Science Co. (Hokkaido, Japan) chemically synthesized 5′ end FAM-labeled oligonucleotides. Binding reactions containing the oligonucleotide (125 or 500nM) and 0.1–0.5µg of purified recombinant protein were incubated for 15min at either room temperature (24°C) or 75°C in 20µL of DNA/RNA-binding buffer (10mM Tris–HCl, pH 7.5, 50mM NaCl, 0.5mM EDTA, 2.5mM MgCl<sub>2</sub>, 5% glycerol, 1mM dithiothreitol). The DNA/RNA-protein complexes were analyzed by 6% non-denaturing PAGE. The quantity of DNA/RNA-protein complexes was evaluated by scanning the fluorescent image with a computerized image analyzer, FX Pro (Bio-Rad Laboratories, Hercules, CA, USA). To sequence the oligonucleotides, we used the following two probes (Xiaojing et al., to be published separately): </p><ul class="roman-lower"><li><p>MPOR-27, 5′-r(GAAACAAGGAGAAAUGGUUCGUGUCCU)-3′,</p></li><li><p>MPOD-27, 5′-d(GAAACAAGGAGAAATGGTTCGTGTCCT)-3′.</p></li></ul>                            <h2 scrollto-destination=87194531 id="87194531" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87194532 id="87194532" class="section-title" data-legacy-id=s3a>3.1. Functional annotation of <em>P. furiosus</em> proteome and those of other model species</h3>
<p><em>P. furiosus</em>, <em>S. solfataricus</em>, <em>B. subtilis</em>, <em>E. coli</em>, <em>C. elegans</em>, and <em>A. thaliana</em> were used as model species. The hyperthermophilic archaeon <em>P. furiosus</em> was chosen for its topical importance in the evolution of the ancient architecture of DNA/RNA regulation<sup><a reveal-id="DSM011C31" data-open="DSM011C31" class="link link-ref link-reveal xref-bibr">31</a></sup> as well as for the thermal stability of its proteins, which enables easy generic purification. In addition, many <em>P. furiosus</em> protein functions remain unknown, which further justifies their study.</p><p>From the EMBL database (Release 83, June 2005), we extracted reliable protein function data for <em>P. furiosus</em> (EMBL accession number AE009950) by unifying information from the three annotated databases Swiss-Prot, TrEMBL, and GOA.<sup><a reveal-id="DSM011C32" data-open="DSM011C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSM011C33" data-open="DSM011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We defined the three categories of proteins on the basis of the number and quality of annotations (see Methods and materials section). For example, 2057 <em>P. furiosus</em> proteins were categorized into 942 functionally known proteins, 121 proteins with putative function, and 994 hypothetical proteins. To eliminate proteins with similar amino acid sequences, we performed a homology search among the 942 functionally known proteins using BLASTP (E-value &lt; 1 × 10<sup>−4</sup>) and reduced the protein data set to 477 non-redundant proteins for the periodicity analysis. To facilitate their use as a training data set for SVM learning, these functionally known proteins were further divided into 157 DNA/RNA-binding proteins and 320 proteins with other functions. The same procedure was applied to the EMBL data of the archaeon <em>S. solfataricus</em> (EMBL accession number AE006641) and the Swiss-Prot entries of the <em>B. subtilis</em>, <em>E. coli</em>, <em>A. thaliana</em>, and <em>C. elegans</em> proteomes (Table <a reveal-id="DSM011TB1" data-open="DSM011TB1" class="link link-reveal link-table xref-fig">1</a>).</p>                            <h3 scrollto-destination=87194535 id="87194535" class="section-title" data-legacy-id=s3b>3.2. Amino acid periodicity score (PD score) and prediction of the DNA/RNA-binding proteins</h3>
<p>To ascertain common features of amino acid periodicity throughout the DNA/RNA-binding protein sequences, we defined 23 amino acid groups using eight physico-chemical profiles (chemical, Sneath, Dayhoff, Stanfel, functional, charge, structural, and hydrophobicity). We prepared a total of 253 patterns of amino acid periodicities (23 groups × 11 non-redundant periodicities). For each training data set in the six model species, the relative coverage of periodic region <em>R</em> was calculated for 253 individual amino acid periodicities as feature vectors for SVM input. Radial basis function SVM classification was performed with default parameters using the software Gist, which allows users to apply a sophisticated machine-learning algorithm to the data.<sup><a reveal-id="DSM011C23" data-open="DSM011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> To quantitatively evaluate a DNA/RNA-binding protein at the proteome level, the discriminant value derived by SVM was defined as a novel index, the periodicity score (PD score), and was assigned to the representative protein data set of each of the six model species.</p><p>The performance of the PD score as a DNA/RNA-binding protein classifier was evaluated by applying the ROC curve to the representative set of <em>P. furiosus</em> proteins (Fig. <a reveal-id="DSM011F1" data-open="DSM011F1" class="link link-reveal link-table xref-fig">1</a>). Sensitivity and specificity of the PD score overwhelmed that of various individual amino acids periodicities such as RK7, CDEGHKNQRSTY8, MNQ19, and AFILMPVW11. This demonstrated that a combination of amino acid periodicities as a feature vector optimizes the system for the classification of DNA/RNA-binding proteins.</p>                        <a id="87194538" scrollto-destination="87194538"></a>
<div id="DSM011F1" data-id="DSM011F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01101.jpeg?Expires=2147483647&Signature=4NPYcR3OYLKP5Vbf8g6DpdF490NuecgAwoKY5IG8sWnB53WIS1yDPgwe6OemnyA7lYpCimMX42aYbtKXm5ZVVwRljI85kHtUQwWXN8e6trWPJIcDPYW4r61nlCM2flEJFQSPOWvZN-TsnNe6ENYS-zFVwVqbosxBtzK8TPwf4yUpQYmMp4fZDZ5wI2covvifnsrUVDQUo2pe6D7Hqe6iwLbbBrowy7cdsl-5x8NU5SnSpwtboZuu~iVrUOdNn10fc6CCGgPJFA8q~j3TtMb2Xwwo6WMtnLSUscxubyY6PeF9YmNDkL1hh5j9H03MuOx5p6t2RehsfYhzY~Lhx0sDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance of PD score. ROC curves of PD score (black) and single amino acid periodicities [RK7 (red), CDEGHKNQRSTY8 (blue), MNQ19 (green), and AFILMPVW11 (orange)]. For example, RK7 denotes the frequency of the periodic region of Arginine (R) and Lysine (K) appearing in the protein sequence with a periodicity of 7." path-from-xml="dsm01101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194538/dsm01101.jpeg" path-from-xml="dsm01101.jpeg" target="_blank">View large</a><a section="87194538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01101.jpeg?Expires=2147483647&Signature=idmT2tkkyGv4yknzt~IwsKyJmvl2EcIN5sxteapVV1V66ChAa2L5Wxix2~4xtOtyyuK6td3jlahI8nIx8-1t0z5VrQHIacbOu3yvFptXs58oed65C7SVGVzsmLZU6okaGjTqVBBvwQPCI4w9fn2H2fGJ14kkOxxmXffSrHupcQJY99uknwZn8zF6~oxn~hc4pNMvOijiGQmJr3355x6KyNYbZ4FxMIIrZTpQH4Vj0LQDOVLuAOv1pyJL2x3cZl4yde2ObqMn1NiFo0DeY~aD~7UeL6jtOii0ZE4WPYDAUYkwASPoltjD55VSLTJVHKDcms2U5Ziw2z0UzBZ6jR0n0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194538&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of PD score. ROC curves of PD score (black) and single amino acid periodicities [RK7 (red), CDEGHKNQRSTY8 (blue), MNQ19 (green), and AFILMPVW11 (orange)]. For example, RK7 denotes the frequency of the periodic region of Arginine (R) and Lysine (K) appearing in the protein sequence with a periodicity of 7.</p></div></div><div content-id="DSM011F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01101.jpeg?Expires=2147483647&Signature=4NPYcR3OYLKP5Vbf8g6DpdF490NuecgAwoKY5IG8sWnB53WIS1yDPgwe6OemnyA7lYpCimMX42aYbtKXm5ZVVwRljI85kHtUQwWXN8e6trWPJIcDPYW4r61nlCM2flEJFQSPOWvZN-TsnNe6ENYS-zFVwVqbosxBtzK8TPwf4yUpQYmMp4fZDZ5wI2covvifnsrUVDQUo2pe6D7Hqe6iwLbbBrowy7cdsl-5x8NU5SnSpwtboZuu~iVrUOdNn10fc6CCGgPJFA8q~j3TtMb2Xwwo6WMtnLSUscxubyY6PeF9YmNDkL1hh5j9H03MuOx5p6t2RehsfYhzY~Lhx0sDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance of PD score. ROC curves of PD score (black) and single amino acid periodicities [RK7 (red), CDEGHKNQRSTY8 (blue), MNQ19 (green), and AFILMPVW11 (orange)]. For example, RK7 denotes the frequency of the periodic region of Arginine (R) and Lysine (K) appearing in the protein sequence with a periodicity of 7." path-from-xml="dsm01101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194538/dsm01101.jpeg" path-from-xml="dsm01101.jpeg" target="_blank">View large</a><a section="87194538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01101.jpeg?Expires=2147483647&Signature=idmT2tkkyGv4yknzt~IwsKyJmvl2EcIN5sxteapVV1V66ChAa2L5Wxix2~4xtOtyyuK6td3jlahI8nIx8-1t0z5VrQHIacbOu3yvFptXs58oed65C7SVGVzsmLZU6okaGjTqVBBvwQPCI4w9fn2H2fGJ14kkOxxmXffSrHupcQJY99uknwZn8zF6~oxn~hc4pNMvOijiGQmJr3355x6KyNYbZ4FxMIIrZTpQH4Vj0LQDOVLuAOv1pyJL2x3cZl4yde2ObqMn1NiFo0DeY~aD~7UeL6jtOii0ZE4WPYDAUYkwASPoltjD55VSLTJVHKDcms2U5Ziw2z0UzBZ6jR0n0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194538&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of PD score. ROC curves of PD score (black) and single amino acid periodicities [RK7 (red), CDEGHKNQRSTY8 (blue), MNQ19 (green), and AFILMPVW11 (orange)]. For example, RK7 denotes the frequency of the periodic region of Arginine (R) and Lysine (K) appearing in the protein sequence with a periodicity of 7.</p></div></div><p>To further validate the performance of PD score, we conducted a comparative analysis upon amino acid composition and SVM-Prot.<sup><a reveal-id="DSM011C27" data-open="DSM011C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Amino acid composition is a widely used profile for predicting protein function, subcellular localization, and protein folding. We calculated 20 individual amino acid compositions as feature vectors for SVM input and defined a new indicator named as composition (CO) score. SVM-Prot is a general proteome-wide function-prediction software based on various features of primary sequences. Three indices, sensitivity (SE), specificity (SP), and overall accuracy (ACC), were calculated for the six model species, respectively, using 10-fold cross-validation to calculate the precise prediction efficiency (Table <a reveal-id="DSM011TB2" data-open="DSM011TB2" class="link link-reveal link-table xref-fig">2</a>). The three predictors have shown different characteristics in predicting DNA/RNA-binding proteins owing to the three indices. The PD score possessed highest overall accuracy; CO score had the highest overall sensitivity and SVM-Prot had the highest overall specificity. As a result, the prediction performance of the PD score was comparable with other methods but statistical significances cannot be observed on the basis of the comparative analysis. Thus, we combined the two indicators CO score and PD score to improve our DNA/RNA-binding prediction method.</p>                        <a id="87194540" scrollto-destination="87194540"></a>
<div content-id="DSM011TB2" class="table-modal"><div class="table-wrap"><div id="DSM011TB2" class="table-wrap-title" data-id="DSM011TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Prediction performance of PD score compared with CO score and SVM-Prot in the six model species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Sample size </th><th colspan="3">PD score </th><th colspan="3">CO score </th><th colspan="3">SVM-Prot </th></tr><tr><th> </th><th> </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>477 </td><td>72.7 (10.5) </td><td>81.1 (6.6) </td><td><strong>78.1 (5.8)</strong> </td><td>72.8 (9.8) </td><td>80.8 (9.9) </td><td>77.9 (6.9) </td><td><strong>73.1</strong> </td><td><strong>84.9</strong> </td><td>72.3 </td></tr><tr><td><em>S. solfataricus</em> </td><td>582 </td><td><strong>68.0 (11.9)</strong> </td><td>79.8 (6.8) </td><td>76.0 (5.4) </td><td>67.7 (18.4) </td><td><strong>84.1 (8.3)</strong> </td><td><strong>78.8 (6.7)</strong> </td><td>66.3 </td><td><strong>84.1</strong> </td><td>77.6 </td></tr><tr><td><em>B. subtilis</em> </td><td>914 </td><td>58.5 (11.1) </td><td>81.2 (5.5) </td><td><strong>75.9 (4.6)</strong> </td><td><strong>75.5 (13.8)</strong> </td><td>72.3 (7.4) </td><td>73.1 (6.1) </td><td>63.7 </td><td><strong>87.9</strong> </td><td>75.3 </td></tr><tr><td><em>E. coli</em> </td><td>1436 </td><td>58.2 (6.0) </td><td>83.4 (2.9) </td><td>77.0 (2.3) </td><td><strong>77.7 (8.9)</strong> </td><td>69.6 (6.7) </td><td>71.7 (4.0) </td><td>57.1 </td><td><strong>86.5</strong> </td><td><strong>81.3</strong> </td></tr><tr><td><em>A. thaliana</em> </td><td>865 </td><td>63.3 (8.7) </td><td><strong>87.5 (5.8)</strong> </td><td><strong>81.5 (3.7)</strong> </td><td><strong>69.5 (8.7)</strong> </td><td>84.9 (3.7) </td><td>80.9 (3.0) </td><td>66.8 </td><td><strong>87.5</strong> </td><td>79.1 </td></tr><tr><td><em>C. elegans</em> </td><td>566 </td><td>59.3 (8.1) </td><td><strong>84.0 (3.6)</strong> </td><td><strong>77.1 (2.2)</strong> </td><td>63.2 (12.1) </td><td>81.5 (5.8) </td><td>75.6 (4.9) </td><td><strong>65.1</strong> </td><td>83.6 </td><td>72.0 </td></tr><tr><td>Overall </td><td>— </td><td>63.3 </td><td>82.9 </td><td><strong>77.6</strong> </td><td><strong>71.1</strong> </td><td>78.9 </td><td>76.3 </td><td>65.3 </td><td><strong>85.8</strong> </td><td>76.3 </td></tr></tbody></table></div><div content-id="DSM011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Sample size </th><th colspan="3">PD score </th><th colspan="3">CO score </th><th colspan="3">SVM-Prot </th></tr><tr><th> </th><th> </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>477 </td><td>72.7 (10.5) </td><td>81.1 (6.6) </td><td><strong>78.1 (5.8)</strong> </td><td>72.8 (9.8) </td><td>80.8 (9.9) </td><td>77.9 (6.9) </td><td><strong>73.1</strong> </td><td><strong>84.9</strong> </td><td>72.3 </td></tr><tr><td><em>S. solfataricus</em> </td><td>582 </td><td><strong>68.0 (11.9)</strong> </td><td>79.8 (6.8) </td><td>76.0 (5.4) </td><td>67.7 (18.4) </td><td><strong>84.1 (8.3)</strong> </td><td><strong>78.8 (6.7)</strong> </td><td>66.3 </td><td><strong>84.1</strong> </td><td>77.6 </td></tr><tr><td><em>B. subtilis</em> </td><td>914 </td><td>58.5 (11.1) </td><td>81.2 (5.5) </td><td><strong>75.9 (4.6)</strong> </td><td><strong>75.5 (13.8)</strong> </td><td>72.3 (7.4) </td><td>73.1 (6.1) </td><td>63.7 </td><td><strong>87.9</strong> </td><td>75.3 </td></tr><tr><td><em>E. coli</em> </td><td>1436 </td><td>58.2 (6.0) </td><td>83.4 (2.9) </td><td>77.0 (2.3) </td><td><strong>77.7 (8.9)</strong> </td><td>69.6 (6.7) </td><td>71.7 (4.0) </td><td>57.1 </td><td><strong>86.5</strong> </td><td><strong>81.3</strong> </td></tr><tr><td><em>A. thaliana</em> </td><td>865 </td><td>63.3 (8.7) </td><td><strong>87.5 (5.8)</strong> </td><td><strong>81.5 (3.7)</strong> </td><td><strong>69.5 (8.7)</strong> </td><td>84.9 (3.7) </td><td>80.9 (3.0) </td><td>66.8 </td><td><strong>87.5</strong> </td><td>79.1 </td></tr><tr><td><em>C. elegans</em> </td><td>566 </td><td>59.3 (8.1) </td><td><strong>84.0 (3.6)</strong> </td><td><strong>77.1 (2.2)</strong> </td><td>63.2 (12.1) </td><td>81.5 (5.8) </td><td>75.6 (4.9) </td><td><strong>65.1</strong> </td><td>83.6 </td><td>72.0 </td></tr><tr><td>Overall </td><td>— </td><td>63.3 </td><td>82.9 </td><td><strong>77.6</strong> </td><td><strong>71.1</strong> </td><td>78.9 </td><td>76.3 </td><td>65.3 </td><td><strong>85.8</strong> </td><td>76.3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Predicted results are shown as SE (sensitivity) = TP/(TP + FN), SP (specificity) = TN/(TN + FP), and ACC (accuracy) = (TP + TN)/(TP + FN + TN + FP). Numbers in bold font indicate the highest index among the three classifiers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194540" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM011TB2" class="table-wrap-title" data-id="DSM011TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Prediction performance of PD score compared with CO score and SVM-Prot in the six model species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Sample size </th><th colspan="3">PD score </th><th colspan="3">CO score </th><th colspan="3">SVM-Prot </th></tr><tr><th> </th><th> </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>477 </td><td>72.7 (10.5) </td><td>81.1 (6.6) </td><td><strong>78.1 (5.8)</strong> </td><td>72.8 (9.8) </td><td>80.8 (9.9) </td><td>77.9 (6.9) </td><td><strong>73.1</strong> </td><td><strong>84.9</strong> </td><td>72.3 </td></tr><tr><td><em>S. solfataricus</em> </td><td>582 </td><td><strong>68.0 (11.9)</strong> </td><td>79.8 (6.8) </td><td>76.0 (5.4) </td><td>67.7 (18.4) </td><td><strong>84.1 (8.3)</strong> </td><td><strong>78.8 (6.7)</strong> </td><td>66.3 </td><td><strong>84.1</strong> </td><td>77.6 </td></tr><tr><td><em>B. subtilis</em> </td><td>914 </td><td>58.5 (11.1) </td><td>81.2 (5.5) </td><td><strong>75.9 (4.6)</strong> </td><td><strong>75.5 (13.8)</strong> </td><td>72.3 (7.4) </td><td>73.1 (6.1) </td><td>63.7 </td><td><strong>87.9</strong> </td><td>75.3 </td></tr><tr><td><em>E. coli</em> </td><td>1436 </td><td>58.2 (6.0) </td><td>83.4 (2.9) </td><td>77.0 (2.3) </td><td><strong>77.7 (8.9)</strong> </td><td>69.6 (6.7) </td><td>71.7 (4.0) </td><td>57.1 </td><td><strong>86.5</strong> </td><td><strong>81.3</strong> </td></tr><tr><td><em>A. thaliana</em> </td><td>865 </td><td>63.3 (8.7) </td><td><strong>87.5 (5.8)</strong> </td><td><strong>81.5 (3.7)</strong> </td><td><strong>69.5 (8.7)</strong> </td><td>84.9 (3.7) </td><td>80.9 (3.0) </td><td>66.8 </td><td><strong>87.5</strong> </td><td>79.1 </td></tr><tr><td><em>C. elegans</em> </td><td>566 </td><td>59.3 (8.1) </td><td><strong>84.0 (3.6)</strong> </td><td><strong>77.1 (2.2)</strong> </td><td>63.2 (12.1) </td><td>81.5 (5.8) </td><td>75.6 (4.9) </td><td><strong>65.1</strong> </td><td>83.6 </td><td>72.0 </td></tr><tr><td>Overall </td><td>— </td><td>63.3 </td><td>82.9 </td><td><strong>77.6</strong> </td><td><strong>71.1</strong> </td><td>78.9 </td><td>76.3 </td><td>65.3 </td><td><strong>85.8</strong> </td><td>76.3 </td></tr></tbody></table></div><div content-id="DSM011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Sample size </th><th colspan="3">PD score </th><th colspan="3">CO score </th><th colspan="3">SVM-Prot </th></tr><tr><th> </th><th> </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th><th>SE (%) </th><th>SP (%) </th><th>ACC (%) </th></tr></thead><tbody><tr><td><em>P. furiosus</em> </td><td>477 </td><td>72.7 (10.5) </td><td>81.1 (6.6) </td><td><strong>78.1 (5.8)</strong> </td><td>72.8 (9.8) </td><td>80.8 (9.9) </td><td>77.9 (6.9) </td><td><strong>73.1</strong> </td><td><strong>84.9</strong> </td><td>72.3 </td></tr><tr><td><em>S. solfataricus</em> </td><td>582 </td><td><strong>68.0 (11.9)</strong> </td><td>79.8 (6.8) </td><td>76.0 (5.4) </td><td>67.7 (18.4) </td><td><strong>84.1 (8.3)</strong> </td><td><strong>78.8 (6.7)</strong> </td><td>66.3 </td><td><strong>84.1</strong> </td><td>77.6 </td></tr><tr><td><em>B. subtilis</em> </td><td>914 </td><td>58.5 (11.1) </td><td>81.2 (5.5) </td><td><strong>75.9 (4.6)</strong> </td><td><strong>75.5 (13.8)</strong> </td><td>72.3 (7.4) </td><td>73.1 (6.1) </td><td>63.7 </td><td><strong>87.9</strong> </td><td>75.3 </td></tr><tr><td><em>E. coli</em> </td><td>1436 </td><td>58.2 (6.0) </td><td>83.4 (2.9) </td><td>77.0 (2.3) </td><td><strong>77.7 (8.9)</strong> </td><td>69.6 (6.7) </td><td>71.7 (4.0) </td><td>57.1 </td><td><strong>86.5</strong> </td><td><strong>81.3</strong> </td></tr><tr><td><em>A. thaliana</em> </td><td>865 </td><td>63.3 (8.7) </td><td><strong>87.5 (5.8)</strong> </td><td><strong>81.5 (3.7)</strong> </td><td><strong>69.5 (8.7)</strong> </td><td>84.9 (3.7) </td><td>80.9 (3.0) </td><td>66.8 </td><td><strong>87.5</strong> </td><td>79.1 </td></tr><tr><td><em>C. elegans</em> </td><td>566 </td><td>59.3 (8.1) </td><td><strong>84.0 (3.6)</strong> </td><td><strong>77.1 (2.2)</strong> </td><td>63.2 (12.1) </td><td>81.5 (5.8) </td><td>75.6 (4.9) </td><td><strong>65.1</strong> </td><td>83.6 </td><td>72.0 </td></tr><tr><td>Overall </td><td>— </td><td>63.3 </td><td>82.9 </td><td><strong>77.6</strong> </td><td><strong>71.1</strong> </td><td>78.9 </td><td>76.3 </td><td>65.3 </td><td><strong>85.8</strong> </td><td>76.3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Predicted results are shown as SE (sensitivity) = TP/(TP + FN), SP (specificity) = TN/(TN + FP), and ACC (accuracy) = (TP + TN)/(TP + FN + TN + FP). Numbers in bold font indicate the highest index among the three classifiers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194540" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87194541 id="87194541" class="section-title" data-legacy-id=s3c>3.3. Both CO score and PD score are required for efficient classification of DNA/RNA-binding protein predictions</h3>
<p>For the efficient classification of DNA/RNA-binding proteins, we performed three different methods (early integration, intermediate integration, and late integration) for integrating heterogeneous data sets on the basis of the context of SVM learning.<sup><a reveal-id="DSM011C29" data-open="DSM011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Late integration has shown the best performance for sensitivity, whereas intermediate integration has shown the best performance for specificity and accuracy (data not shown). Our observation was not consistent with the previous work by Pavlidis,<sup><a reveal-id="DSM011C29" data-open="DSM011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> in which early and intermediate integrations have shown high performance compared with the late integration. It is possible that the prediction performance can be altered by many factors such as evaluation indices, data types, kernel function parameters, and feature-selection algorithms.</p><p>To provide further insights into the relative performance of CO and PD scores and to extract efficient DNA/RNA-binding protein candidates, we have chosen the late integration method and carried out two-dimensional correlation analysis on the basis of CO score and PD score upon 477 functionally known proteins in <em>P. furiosus</em>. The correlation coefficient was <em>r</em> = 0.75 (all functionally known proteins) and <em>r</em> = 0.55 (DNA/RNA-binding proteins only), respectively. The 157 DNA/RNA-binding proteins located at the right-upper region of the two-dimensional plot, suggesting that both CO and PD scores are required for classifying proteins with DNA or RNA-binding activity (red and blue circles in Fig. <a reveal-id="DSM011F2" data-open="DSM011F2" class="link link-reveal link-table xref-fig">2</a>A). Then two different thresholds were determined on the basis of this newly defined CO score + PD score. First threshold is based on the highest overall accuracy (ACC) with a CO + PD score of 0.6 and the second threshold is based on the highest MCC with CO + PD score = –0.13. According to the Supplementary Fig. S1, the first threshold optimizes the extraction of reliable candidates for novel DNA/RNA-binding proteins (SE = 52.2%, SP = 96.3%, ACC = 81.8%, and PPV = 87.2%) and the second threshold optimizes the classification performance of CO + PD score (SE = 82.2%, SP = 80%, ACC = 80.7%, and PPV = 66.8%). On the basis of these thresholds, we classified proteins into three classes (class I–III) (Fig. <a reveal-id="DSM011F2" data-open="DSM011F2" class="link link-reveal link-table xref-fig">2</a>A). As a result, a total of 94 proteins including 82 DNA/RNA-binding proteins (Fig. <a reveal-id="DSM011F2" data-open="DSM011F2" class="link link-reveal link-table xref-fig">2</a>B) were categorized as class I proteins (CO + PD score &gt; 0.6).</p>                        <a id="87194544" scrollto-destination="87194544"></a>
<div id="DSM011F2" data-id="DSM011F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01102.jpeg?Expires=2147483647&Signature=mhjojSbn0OcBXkia-CY4hPlr1mnH3Zkjh4Juaj~yL7dk5hY4yf6sMgn54r1Q0NB5cSIcocpWKkH0rEpwTPzWU1XyWcelnfgUzSFrOfW6iEMOoBaL4olAh7GVrWfg9A4ugQeK812wvz792yYaX1iluDR5ExhMIYeuviQwcdRqREhyjj1gl4zxruith9BSfYqoXCvaDZfUwkPLDnmcheM69dJL5A6S4zrigN6-E4ns5N0a0sCvvqRZzWr40Qe~sk2K49ZQa~rK2~9tZM8xb7gocCSUKnr4yJve1pKzjRmPbH33hyxm8MFmNbbL3J3TXQVXRHtSXdi~LCfmablXuemO3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two-dimensional correlation analysis of DNA/RNA-binding proteins in P. furiosus on the basis of amino acid composition and periodicity. (A) A total of 477 functionally known proteins in P. furiosus were plotted on the two-dimensional correlation plot of CO score and PD score. The ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are shown. The two dotted lines represent a threshold of maximum MCC value of 0.59 for optimizing sensitivity and specificity (black arrow) and a maximum ACC value of 81.8% for optimizing the prediction of DNA/RNA-binding protein candidates (white arrow). The ranges of three classes are Class I, CO + PD score &gt; 0.6; class II, 0.6 &gt; CO + PD score &gt; –0.13; and class III –0.13 &gt; CO + PD score. (B) The numbers of ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are counted in classes I–III." path-from-xml="dsm01102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194544/dsm01102.jpeg" path-from-xml="dsm01102.jpeg" target="_blank">View large</a><a section="87194544" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01102.jpeg?Expires=2147483647&Signature=gNHqkTLns5LVYypRE55NGj1uRdtlhoMQuVhdMOwOsWc~vx63fsJ7h~19QDYW~64SBsFAd5-LENS3XSMlavdGHYiVmwX0AjsSUXqrYvh31SCgFEVtPP9GFPGRtZf2Zk6Ju2B-0lb4wO8MKudjMcUboAPm-qtHNyluwYfr8nIXOy6SGt1EPgs31ppTL2esaf5vKA6PGnabDGIvcH7Z94QnuzW5tuIWBPOfCiCrMJJw-b6GB8yTGnGxBmUsKXI7WOIyG~dejbOcvO1hRumgxCZrmdHELp19nRb~O~IeLRoOY6HBvGw8Y0xa71yifhvERtAvTR0FX95c~6gvoZ41Ir2HqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194544&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two-dimensional correlation analysis of DNA/RNA-binding proteins in <em>P. furiosus</em> on the basis of amino acid composition and periodicity. (<strong>A</strong>) A total of 477 functionally known proteins in <em>P. furiosus</em> were plotted on the two-dimensional correlation plot of CO score and PD score. The ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are shown. The two dotted lines represent a threshold of maximum MCC value of 0.59 for optimizing sensitivity and specificity (black arrow) and a maximum ACC value of 81.8% for optimizing the prediction of DNA/RNA-binding protein candidates (white arrow). The ranges of three classes are Class I, CO + PD score &gt; 0.6; class II, 0.6 &gt; CO + PD score &gt; –0.13; and class III –0.13 &gt; CO + PD score. <strong>(B)</strong> The numbers of ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are counted in classes I–III.</p></div></div><div content-id="DSM011F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01102.jpeg?Expires=2147483647&Signature=mhjojSbn0OcBXkia-CY4hPlr1mnH3Zkjh4Juaj~yL7dk5hY4yf6sMgn54r1Q0NB5cSIcocpWKkH0rEpwTPzWU1XyWcelnfgUzSFrOfW6iEMOoBaL4olAh7GVrWfg9A4ugQeK812wvz792yYaX1iluDR5ExhMIYeuviQwcdRqREhyjj1gl4zxruith9BSfYqoXCvaDZfUwkPLDnmcheM69dJL5A6S4zrigN6-E4ns5N0a0sCvvqRZzWr40Qe~sk2K49ZQa~rK2~9tZM8xb7gocCSUKnr4yJve1pKzjRmPbH33hyxm8MFmNbbL3J3TXQVXRHtSXdi~LCfmablXuemO3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two-dimensional correlation analysis of DNA/RNA-binding proteins in P. furiosus on the basis of amino acid composition and periodicity. (A) A total of 477 functionally known proteins in P. furiosus were plotted on the two-dimensional correlation plot of CO score and PD score. The ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are shown. The two dotted lines represent a threshold of maximum MCC value of 0.59 for optimizing sensitivity and specificity (black arrow) and a maximum ACC value of 81.8% for optimizing the prediction of DNA/RNA-binding protein candidates (white arrow). The ranges of three classes are Class I, CO + PD score &gt; 0.6; class II, 0.6 &gt; CO + PD score &gt; –0.13; and class III –0.13 &gt; CO + PD score. (B) The numbers of ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are counted in classes I–III." path-from-xml="dsm01102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194544/dsm01102.jpeg" path-from-xml="dsm01102.jpeg" target="_blank">View large</a><a section="87194544" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01102.jpeg?Expires=2147483647&Signature=gNHqkTLns5LVYypRE55NGj1uRdtlhoMQuVhdMOwOsWc~vx63fsJ7h~19QDYW~64SBsFAd5-LENS3XSMlavdGHYiVmwX0AjsSUXqrYvh31SCgFEVtPP9GFPGRtZf2Zk6Ju2B-0lb4wO8MKudjMcUboAPm-qtHNyluwYfr8nIXOy6SGt1EPgs31ppTL2esaf5vKA6PGnabDGIvcH7Z94QnuzW5tuIWBPOfCiCrMJJw-b6GB8yTGnGxBmUsKXI7WOIyG~dejbOcvO1hRumgxCZrmdHELp19nRb~O~IeLRoOY6HBvGw8Y0xa71yifhvERtAvTR0FX95c~6gvoZ41Ir2HqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194544&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two-dimensional correlation analysis of DNA/RNA-binding proteins in <em>P. furiosus</em> on the basis of amino acid composition and periodicity. (<strong>A</strong>) A total of 477 functionally known proteins in <em>P. furiosus</em> were plotted on the two-dimensional correlation plot of CO score and PD score. The ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are shown. The two dotted lines represent a threshold of maximum MCC value of 0.59 for optimizing sensitivity and specificity (black arrow) and a maximum ACC value of 81.8% for optimizing the prediction of DNA/RNA-binding protein candidates (white arrow). The ranges of three classes are Class I, CO + PD score &gt; 0.6; class II, 0.6 &gt; CO + PD score &gt; –0.13; and class III –0.13 &gt; CO + PD score. <strong>(B)</strong> The numbers of ribosomal proteins (red), rest of the DNA/RNA-binding proteins (blue), and other functionally known proteins (black) are counted in classes I–III.</p></div></div><p>Further observation of DNA/RNA-binding proteins has revealed a region-specific distribution of ribosomal proteins and other DNA/RNA-binding proteins. Ribosomal proteins are strongly affected by CO score and are dominant at the high range of CO score (CO &gt; 0.5). The CO score of other DNA/RNA-binding proteins ranged between 0 and 0.5 but some of them were dominant at high PD score region (0.25–1.5). As shown in Supplementary Table S1, this region includes 13 tRNA-processing enzymes (i.e. tRNA-synthetases, CCA-adding enzymes, and RNase P subunits), 11 DNA-binding proteins (i.e. DNA polymerase, DNA helicase, DNA primase, and reverse gyrase), three ribosomal proteins (i.e. ribosomal protein S3P and ribosomal protein L14e), and various transcription/translation-related proteins (i.e. SRP54, HTH-type transcriptional regulator, and transcription termination–anti-termination factor). We assume that PD score is an effective means of classifying DNA/RNA-binding proteins from a set of proteins, which cannot be distinguished by using amino acid compositions.</p>                            <h3 scrollto-destination=87194546 id="87194546" class="section-title" data-legacy-id=s3d>3.4. Selection and experimental verification of novel DNA/RNA-binding protein candidates</h3>
<p>The same procedure was applied to the 994 hypothetical proteins in <em>P. furiosus</em>. The two-dimensional plot of hypothetical proteins was similar to that of functionally known proteins as well as the protein ratio in classes I–III (Fig. <a reveal-id="DSM011F2" data-open="DSM011F2" class="link link-reveal link-table xref-fig">2</a> versus Fig. <a reveal-id="DSM011F3" data-open="DSM011F3" class="link link-reveal link-table xref-fig">3</a>). However, the number of proteins has decreased from the high CO score (CO score &gt; 0.5) region, which is known to be dominated by ribosomal proteins. As a result, 994 hypothetical proteins were classified into three classes (I–III) owing to the CO + PD score thresholds (Supplementary Table S2), and a total of 151 proteins were classified as strong candidates for novel DNA/RNA-binding proteins.</p>                        <a id="87194548" scrollto-destination="87194548"></a>
<div id="DSM011F3" data-id="DSM011F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01103.jpeg?Expires=2147483647&Signature=cKjU9ZFegEA9TX8OztaeVHkKKzx4qt-Aj~wAkEiptaoDR4Vn70hbVpIIP60RESNKkWDvD~CrSttiIhC~oN~Ss1BK2LobtowJzRmT~7kvf0ChabgImW2~NHMvM0q5kI5pBue3P8PTlJY1J4qKro7CyrxRQUj7PJ3YvuMrVSNO7tv8lw2sRaem-qDVdbkLhIP9l4Ilusqt70o~GXSwIsccKZAoKW-DWZcqTVwYlJGk~LNs8p3qGheFNLLCR1HBFdGHWzhEdYzMc4uTWBBIFpx~Nsb~gdrJqIi9Y761cn~dexI91Ks7p6x0cx1NBjAc~7A5jVS6StFLmr5ipQaQ7WCx9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two-dimensional correlation analysis of hypothetical proteins in P. furiosus on the basis of amino acid composition and periodicity. (A) The vertical and horizontal axes represent CO and periodicity PD scores, respectively. Distribution of the 994 hypothetical proteins is shown. Experimentally validated 14 candidate proteins are denoted as red circles (possessed DNA/RNA-binding activity) and black circles (no detectable DNA/RNA-binding activity) and the remaining proteins are shown as green circles. (B) The numbers of experimentally verified proteins with DNA/RNA-binding activities (red), protein with no DNA/RNA-binding activities (black), and the remaining hypothetical proteins (green) are counted for classes I–III." path-from-xml="dsm01103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194548/dsm01103.jpeg" path-from-xml="dsm01103.jpeg" target="_blank">View large</a><a section="87194548" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01103.jpeg?Expires=2147483647&Signature=yN~pbas2Dbhfgv4oDGA8SE7nqn1QfooXsk4QAS9R2c5FbZNx5lR1If~jMqkvoMfvJ4tX6kCKQWbZXnZT3DFg-SbEflW016fN0OOm9glkkAFfKluMeO1tlhSfhk14Kn27gxZXUvNSvJHOeP9ZVwQdALDuU6ciQiUfnbLYQaaHH8yJXrgDzQXV2Z2yMtm~~DHzUHy0Am8blqJSA1ZUdNGRSZcdQRXe50tvovRIt3MNPA~lhDYLQkxk9TXUqN36MvoPl4v64IMCbwAkZqT0PcfcbcJTL7O6JE9a~mLm5ItUUsRAFC9EysxouepH0Z0ye6YOJrZXkITTU9wNs~wQDlfQiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194548&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two-dimensional correlation analysis of hypothetical proteins in <em>P. furiosus</em> on the basis of amino acid composition and periodicity. <strong>(A)</strong> The vertical and horizontal axes represent CO and periodicity PD scores, respectively. Distribution of the 994 hypothetical proteins is shown. Experimentally validated 14 candidate proteins are denoted as red circles (possessed DNA/RNA-binding activity) and black circles (no detectable DNA/RNA-binding activity) and the remaining proteins are shown as green circles. <strong>(B)</strong> The numbers of experimentally verified proteins with DNA/RNA-binding activities (red), protein with no DNA/RNA-binding activities (black), and the remaining hypothetical proteins (green) are counted for classes I–III.</p></div></div><div content-id="DSM011F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01103.jpeg?Expires=2147483647&Signature=cKjU9ZFegEA9TX8OztaeVHkKKzx4qt-Aj~wAkEiptaoDR4Vn70hbVpIIP60RESNKkWDvD~CrSttiIhC~oN~Ss1BK2LobtowJzRmT~7kvf0ChabgImW2~NHMvM0q5kI5pBue3P8PTlJY1J4qKro7CyrxRQUj7PJ3YvuMrVSNO7tv8lw2sRaem-qDVdbkLhIP9l4Ilusqt70o~GXSwIsccKZAoKW-DWZcqTVwYlJGk~LNs8p3qGheFNLLCR1HBFdGHWzhEdYzMc4uTWBBIFpx~Nsb~gdrJqIi9Y761cn~dexI91Ks7p6x0cx1NBjAc~7A5jVS6StFLmr5ipQaQ7WCx9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two-dimensional correlation analysis of hypothetical proteins in P. furiosus on the basis of amino acid composition and periodicity. (A) The vertical and horizontal axes represent CO and periodicity PD scores, respectively. Distribution of the 994 hypothetical proteins is shown. Experimentally validated 14 candidate proteins are denoted as red circles (possessed DNA/RNA-binding activity) and black circles (no detectable DNA/RNA-binding activity) and the remaining proteins are shown as green circles. (B) The numbers of experimentally verified proteins with DNA/RNA-binding activities (red), protein with no DNA/RNA-binding activities (black), and the remaining hypothetical proteins (green) are counted for classes I–III." path-from-xml="dsm01103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194548/dsm01103.jpeg" path-from-xml="dsm01103.jpeg" target="_blank">View large</a><a section="87194548" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01103.jpeg?Expires=2147483647&Signature=yN~pbas2Dbhfgv4oDGA8SE7nqn1QfooXsk4QAS9R2c5FbZNx5lR1If~jMqkvoMfvJ4tX6kCKQWbZXnZT3DFg-SbEflW016fN0OOm9glkkAFfKluMeO1tlhSfhk14Kn27gxZXUvNSvJHOeP9ZVwQdALDuU6ciQiUfnbLYQaaHH8yJXrgDzQXV2Z2yMtm~~DHzUHy0Am8blqJSA1ZUdNGRSZcdQRXe50tvovRIt3MNPA~lhDYLQkxk9TXUqN36MvoPl4v64IMCbwAkZqT0PcfcbcJTL7O6JE9a~mLm5ItUUsRAFC9EysxouepH0Z0ye6YOJrZXkITTU9wNs~wQDlfQiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194548&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two-dimensional correlation analysis of hypothetical proteins in <em>P. furiosus</em> on the basis of amino acid composition and periodicity. <strong>(A)</strong> The vertical and horizontal axes represent CO and periodicity PD scores, respectively. Distribution of the 994 hypothetical proteins is shown. Experimentally validated 14 candidate proteins are denoted as red circles (possessed DNA/RNA-binding activity) and black circles (no detectable DNA/RNA-binding activity) and the remaining proteins are shown as green circles. <strong>(B)</strong> The numbers of experimentally verified proteins with DNA/RNA-binding activities (red), protein with no DNA/RNA-binding activities (black), and the remaining hypothetical proteins (green) are counted for classes I–III.</p></div></div><p>In order to verify that hypothetical proteins in class I actually possess DNA/RNA-binding protein activities, we randomly chose 17 hypothetical proteins from classes I–III (nine from class I, five from class II, and three from class III, Table <a reveal-id="DSM011TB3" data-open="DSM011TB3" class="link link-reveal link-table xref-fig">3</a>). All 17 recombinant proteins were overexpressed in <em>E. coli</em> and purified to near homogeneity (Fig. <a reveal-id="DSM011F4" data-open="DSM011F4" class="link link-reveal link-table xref-fig">4</a>). To study the DNA/RNA-binding properties of the candidate proteins, we first carried out gel-shift assays using 5′ FAM-labeled, 27 bp, multipotential oligoprobe RNA (MPOR-27) (Fig. <a reveal-id="DSM011F5" data-open="DSM011F5" class="link link-reveal link-table xref-fig">5</a>A). MPORs potentially possess four different secondary RNA structures (stem, bulge, loop, and single strand), which encompass the currently known structures corresponding to the activities of various RNA-binding proteins. The three proteins, PF0871, PF0678, and PF0840, aggregated in the loading well, so we removed them from the final results. A prominent shift of the RNA probe up the gel was observed in candidate proteins PF0029, PF0030, PF0565, PF1139, PF1473, PF1580, PF1912, and PF2062 (Fig. <a reveal-id="DSM011F5" data-open="DSM011F5" class="link link-reveal link-table xref-fig">5</a>B). Interestingly, the formation of certain nucleic acid–protein complexes appears to be temperature dependent. For example, PF1981 showed a significant shift at 75°C but not at 24°C (Fig. <a reveal-id="DSM011F5" data-open="DSM011F5" class="link link-reveal link-table xref-fig">5</a>C versus 5B). PF0029, PF0030, PF1139, and PF1580 also showed binding affinity with the multipotential oligoprobe DNA, MPOD-27 (data not shown). No significant shifts were observed in PF0547, PF1142, PF1488, PF1498, or PF1913, though agarose gel analysis of purified PF1498 revealed it to be a potential protein–nucleic acid complex (Fig. <a reveal-id="DSM011F5" data-open="DSM011F5" class="link link-reveal link-table xref-fig">5</a>D).</p>                        <a id="87194550" scrollto-destination="87194550"></a>
<div id="DSM011F4" data-id="DSM011F4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01104.jpeg?Expires=2147483647&Signature=geKF5cTRqNDr0Ea8x5XKFy2FsyH5lfxbGneRorLGLlrDcllgpo88Di5wJ1OVwvxBp38n~lWzSCSnYKfeV0FwNzzS4lUSxbz1tkMjXDD61aRH0QDZQHjs5F7nTIXlJcBvZeNyYe7NqmC8JGe-n~xfFP~J6SO3qtRgM9~g14BjKVWT2ogfRNjah3j729Nd80s0ytB2jtw2PK5R0k5Ys-bibmN364AddOnp6oZWIR4OkGJa3ftXbsXRERK66j9g1hVZiGM11AV2o96OqB4SaUZLiHHIefOuDt1YolEVmFjVuQKH7cUYgJc5piUXMrXs0HqgAyCibNDHGw6aDME2Jp7AsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Purification of the hypothetical proteins in P. furiosus. SDS–PAGE analysis of 17 purified candidate proteins. The positions of the purified candidate proteins are marked with black dots. For proteins of large and small molecular sizes, 10–20% (left column) and 15–25% (right column) gradient gel was used (M, protein size marker; Bio-Rad)." path-from-xml="dsm01104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194550/dsm01104.jpeg" path-from-xml="dsm01104.jpeg" target="_blank">View large</a><a section="87194550" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01104.jpeg?Expires=2147483647&Signature=tA~yvqo0htC-Uv7mr-L2BKs-x108PGBzmNpyi3P9h0HPbm6r96ggevOqInTCCf7yGaP1TVnw6Jvhqc0rQDYabgKh9udGzSDHEkMEPhpKu8mzmx~41CKUMFuZkKh8Tn1zQjgwHniW18R~3uhLROYQC7103kJ7krOKhfRPF7EyDy2QdWXmxyIG~-VXWcWYt0y~NBWu3ytS-F8FS7P500a3dgHXJLSJmb71x-H9A8YEbeEu28ewQ8PbMH0AcH-tdQaS55xIRyP18xFs87oJFlF6jxYo5lqllQkm~cFZqaiVA6TMnVzSLEdEv0XjZygp4A2s3VK8YE0vmLarcv5G2Tl16Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194550&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Purification of the hypothetical proteins in <em>P. furiosus</em>. SDS–PAGE analysis of 17 purified candidate proteins. The positions of the purified candidate proteins are marked with black dots. For proteins of large and small molecular sizes, 10–20% (left column) and 15–25% (right column) gradient gel was used (M, protein size marker; Bio-Rad).</p></div></div><div content-id="DSM011F4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01104.jpeg?Expires=2147483647&Signature=geKF5cTRqNDr0Ea8x5XKFy2FsyH5lfxbGneRorLGLlrDcllgpo88Di5wJ1OVwvxBp38n~lWzSCSnYKfeV0FwNzzS4lUSxbz1tkMjXDD61aRH0QDZQHjs5F7nTIXlJcBvZeNyYe7NqmC8JGe-n~xfFP~J6SO3qtRgM9~g14BjKVWT2ogfRNjah3j729Nd80s0ytB2jtw2PK5R0k5Ys-bibmN364AddOnp6oZWIR4OkGJa3ftXbsXRERK66j9g1hVZiGM11AV2o96OqB4SaUZLiHHIefOuDt1YolEVmFjVuQKH7cUYgJc5piUXMrXs0HqgAyCibNDHGw6aDME2Jp7AsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Purification of the hypothetical proteins in P. furiosus. SDS–PAGE analysis of 17 purified candidate proteins. The positions of the purified candidate proteins are marked with black dots. For proteins of large and small molecular sizes, 10–20% (left column) and 15–25% (right column) gradient gel was used (M, protein size marker; Bio-Rad)." path-from-xml="dsm01104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194550/dsm01104.jpeg" path-from-xml="dsm01104.jpeg" target="_blank">View large</a><a section="87194550" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01104.jpeg?Expires=2147483647&Signature=tA~yvqo0htC-Uv7mr-L2BKs-x108PGBzmNpyi3P9h0HPbm6r96ggevOqInTCCf7yGaP1TVnw6Jvhqc0rQDYabgKh9udGzSDHEkMEPhpKu8mzmx~41CKUMFuZkKh8Tn1zQjgwHniW18R~3uhLROYQC7103kJ7krOKhfRPF7EyDy2QdWXmxyIG~-VXWcWYt0y~NBWu3ytS-F8FS7P500a3dgHXJLSJmb71x-H9A8YEbeEu28ewQ8PbMH0AcH-tdQaS55xIRyP18xFs87oJFlF6jxYo5lqllQkm~cFZqaiVA6TMnVzSLEdEv0XjZygp4A2s3VK8YE0vmLarcv5G2Tl16Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194550&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Purification of the hypothetical proteins in <em>P. furiosus</em>. SDS–PAGE analysis of 17 purified candidate proteins. The positions of the purified candidate proteins are marked with black dots. For proteins of large and small molecular sizes, 10–20% (left column) and 15–25% (right column) gradient gel was used (M, protein size marker; Bio-Rad).</p></div></div>                        <a id="87194551" scrollto-destination="87194551"></a>
<div id="DSM011F5" data-id="DSM011F5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01105.jpeg?Expires=2147483647&Signature=dkRARYJFg1-kLiTTuy9TnoGY7xOCg0qbHhMq0fP6SFgrwWeDrppIirVuDjE9nvcHzMzTD9NiNW2NiI0yWg8Ocz70Vp55KJjElaG7d4cxtVAuJArDgQ8gVsghRDUOoDImZVHU3I2VBF3CxR5zmCov8DflKiKX8aKgyXr~1qrH~kunUzwiRqNtzWNqaHunPQFmLYdRnobOdInmwJu7r3vPS2kxB-CwJXPctBBVnZkHohu~Vm2tvJLXCjiUOaoSTpTNJG1Vr5sL5SwYvN12Mn9UYk86UqOFDTnS38z7GkmmvSixehSz8sGq8xsMrSYXfxikbR7Ux5blpW9kPWR0Qi12-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RNA-binding analysis of 14 hypothetical proteins in P. furiosus. (A) Nucleotide sequence and possible RNA secondary structure of multipotential oligoprobe RNA. (B) Detection of RNA-binding activity of 14 candidate proteins by gel-shift assay at room temperature (24°C). (C) Gel-shift assay of four candidate proteins with prominent RNA-binding activity at 75°C. (D) Analysis of purified protein peak fractions by 1.2% agarose gel. White bar indicates the existence of nucleic acids. Lanes 1 and 7 are DNA markers." path-from-xml="dsm01105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194551/dsm01105.jpeg" path-from-xml="dsm01105.jpeg" target="_blank">View large</a><a section="87194551" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01105.jpeg?Expires=2147483647&Signature=fPjmxR22QrquTa~2SjgU9xQqCf3wXQPTLUrRJQPTiJGZt9LrqqIYzEoORVrw~86k0lPTLF2rgJ-nHKdCTfy1yOW2n5GX1VMHqW2RtX2KjaN9LoEMYCQK37UC2kY3rPm70noEy65WAjtYlxcQqVsFqgaeAxdPmitPySVwAiM7IfD8U5qHXZy3FnZB6vgcvuZ96Yo6ZOpzc3d4vXFOp6wvFzxVst-lCbb4hdFpB7TCvyV-UbMM5AL0yVtii7gkCtzNZby-TOaOPzaYCc0qn1vGrAC1KGmVX55sYrFAl4TU5Z-QuIuvr6Nexbp3fQws-AEbziB7JjNlEDhD6ty9DQWaSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194551&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RNA-binding analysis of 14 hypothetical proteins in <em>P. furiosus</em>. <strong>(A)</strong> Nucleotide sequence and possible RNA secondary structure of multipotential oligoprobe RNA. (<strong>B</strong>) Detection of RNA-binding activity of 14 candidate proteins by gel-shift assay at room temperature (24°C). (<strong>C</strong>) Gel-shift assay of four candidate proteins with prominent RNA-binding activity at 75°C. (<strong>D</strong>) Analysis of purified protein peak fractions by 1.2% agarose gel. White bar indicates the existence of nucleic acids. Lanes 1 and 7 are DNA markers.</p></div></div><div content-id="DSM011F5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/m_dsm01105.jpeg?Expires=2147483647&Signature=dkRARYJFg1-kLiTTuy9TnoGY7xOCg0qbHhMq0fP6SFgrwWeDrppIirVuDjE9nvcHzMzTD9NiNW2NiI0yWg8Ocz70Vp55KJjElaG7d4cxtVAuJArDgQ8gVsghRDUOoDImZVHU3I2VBF3CxR5zmCov8DflKiKX8aKgyXr~1qrH~kunUzwiRqNtzWNqaHunPQFmLYdRnobOdInmwJu7r3vPS2kxB-CwJXPctBBVnZkHohu~Vm2tvJLXCjiUOaoSTpTNJG1Vr5sL5SwYvN12Mn9UYk86UqOFDTnS38z7GkmmvSixehSz8sGq8xsMrSYXfxikbR7Ux5blpW9kPWR0Qi12-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RNA-binding analysis of 14 hypothetical proteins in P. furiosus. (A) Nucleotide sequence and possible RNA secondary structure of multipotential oligoprobe RNA. (B) Detection of RNA-binding activity of 14 candidate proteins by gel-shift assay at room temperature (24°C). (C) Gel-shift assay of four candidate proteins with prominent RNA-binding activity at 75°C. (D) Analysis of purified protein peak fractions by 1.2% agarose gel. White bar indicates the existence of nucleic acids. Lanes 1 and 7 are DNA markers." path-from-xml="dsm01105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194551/dsm01105.jpeg" path-from-xml="dsm01105.jpeg" target="_blank">View large</a><a section="87194551" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm01105.jpeg?Expires=2147483647&Signature=fPjmxR22QrquTa~2SjgU9xQqCf3wXQPTLUrRJQPTiJGZt9LrqqIYzEoORVrw~86k0lPTLF2rgJ-nHKdCTfy1yOW2n5GX1VMHqW2RtX2KjaN9LoEMYCQK37UC2kY3rPm70noEy65WAjtYlxcQqVsFqgaeAxdPmitPySVwAiM7IfD8U5qHXZy3FnZB6vgcvuZ96Yo6ZOpzc3d4vXFOp6wvFzxVst-lCbb4hdFpB7TCvyV-UbMM5AL0yVtii7gkCtzNZby-TOaOPzaYCc0qn1vGrAC1KGmVX55sYrFAl4TU5Z-QuIuvr6Nexbp3fQws-AEbziB7JjNlEDhD6ty9DQWaSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194551&ar=577454&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RNA-binding analysis of 14 hypothetical proteins in <em>P. furiosus</em>. <strong>(A)</strong> Nucleotide sequence and possible RNA secondary structure of multipotential oligoprobe RNA. (<strong>B</strong>) Detection of RNA-binding activity of 14 candidate proteins by gel-shift assay at room temperature (24°C). (<strong>C</strong>) Gel-shift assay of four candidate proteins with prominent RNA-binding activity at 75°C. (<strong>D</strong>) Analysis of purified protein peak fractions by 1.2% agarose gel. White bar indicates the existence of nucleic acids. Lanes 1 and 7 are DNA markers.</p></div></div>                        <a id="87194552" scrollto-destination="87194552"></a>
<div content-id="DSM011TB3" class="table-modal"><div class="table-wrap"><div id="DSM011TB3" class="table-wrap-title" data-id="DSM011TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of experimentally validated hypothetical proteins in <em>P. furiosus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Class </th><th rowspan="2">Gene ID </th><th rowspan="2">Molecular mass (kDa) </th><th colspan="3">SVM analysis </th><th rowspan="2">Experimenal verification of DNA/RNA-binding activity </th></tr><tr><th>PD score </th><th>CO score </th><th>CO + PD </th></tr></thead><tbody><tr><td>I </td><td>PF1498 </td><td>16.5 </td><td>1.11 </td><td>1.55 </td><td>2.66 </td><td>+<sup>a</sup> </td></tr><tr><td>I </td><td>PF1139 </td><td>44.7 </td><td>1.44 </td><td>0.74 </td><td>2.18 </td><td>+ </td></tr><tr><td>I </td><td>PF2062 </td><td>11.0 </td><td>0.45 </td><td>1.36 </td><td>1.81 </td><td>+ </td></tr><tr><td>I </td><td>PF0565 </td><td>17.2 </td><td>0.43 </td><td>0.56 </td><td>0.99 </td><td>+ </td></tr><tr><td>I </td><td>PF1473 </td><td>26.7 </td><td>0.53 </td><td>0.44 </td><td>0.97 </td><td>+ </td></tr><tr><td>I </td><td>PF1580 </td><td>25.5 </td><td>0.40 </td><td>0.35 </td><td>0.75 </td><td>+ </td></tr><tr><td>I </td><td>PF1913 </td><td>18.5 </td><td>0.23 </td><td>0.42 </td><td>0.65 </td><td>− </td></tr><tr><td>II </td><td>PF1981 </td><td>27.5 </td><td>0.18 </td><td>0.37 </td><td>0.55 </td><td>+ </td></tr><tr><td>II </td><td>PF1912 </td><td>48.2 </td><td>0.29 </td><td>0.04 </td><td>0.33 </td><td>+ </td></tr><tr><td>II </td><td>PF0029 </td><td>56.4 </td><td>0.22 </td><td>0.03 </td><td>0.25 </td><td>+ </td></tr><tr><td>II </td><td>PF1488 </td><td>26.5 </td><td>−0.07 </td><td>0.31 </td><td>0.24 </td><td>− </td></tr><tr><td>III </td><td>PF0547 </td><td>50.7 </td><td>0.00 </td><td>−0.18 </td><td>−0.18 </td><td>− </td></tr><tr><td>III </td><td>PF1142 </td><td>31.5 </td><td>−0.48 </td><td>−0.41 </td><td>−0.89 </td><td>− </td></tr><tr><td>III </td><td>PF0030 </td><td>40.8 </td><td>−0.67 </td><td>−0.25 </td><td>−0.92 </td><td>+ </td></tr></tbody></table></div><div content-id="DSM011TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Class </th><th rowspan="2">Gene ID </th><th rowspan="2">Molecular mass (kDa) </th><th colspan="3">SVM analysis </th><th rowspan="2">Experimenal verification of DNA/RNA-binding activity </th></tr><tr><th>PD score </th><th>CO score </th><th>CO + PD </th></tr></thead><tbody><tr><td>I </td><td>PF1498 </td><td>16.5 </td><td>1.11 </td><td>1.55 </td><td>2.66 </td><td>+<sup>a</sup> </td></tr><tr><td>I </td><td>PF1139 </td><td>44.7 </td><td>1.44 </td><td>0.74 </td><td>2.18 </td><td>+ </td></tr><tr><td>I </td><td>PF2062 </td><td>11.0 </td><td>0.45 </td><td>1.36 </td><td>1.81 </td><td>+ </td></tr><tr><td>I </td><td>PF0565 </td><td>17.2 </td><td>0.43 </td><td>0.56 </td><td>0.99 </td><td>+ </td></tr><tr><td>I </td><td>PF1473 </td><td>26.7 </td><td>0.53 </td><td>0.44 </td><td>0.97 </td><td>+ </td></tr><tr><td>I </td><td>PF1580 </td><td>25.5 </td><td>0.40 </td><td>0.35 </td><td>0.75 </td><td>+ </td></tr><tr><td>I </td><td>PF1913 </td><td>18.5 </td><td>0.23 </td><td>0.42 </td><td>0.65 </td><td>− </td></tr><tr><td>II </td><td>PF1981 </td><td>27.5 </td><td>0.18 </td><td>0.37 </td><td>0.55 </td><td>+ </td></tr><tr><td>II </td><td>PF1912 </td><td>48.2 </td><td>0.29 </td><td>0.04 </td><td>0.33 </td><td>+ </td></tr><tr><td>II </td><td>PF0029 </td><td>56.4 </td><td>0.22 </td><td>0.03 </td><td>0.25 </td><td>+ </td></tr><tr><td>II </td><td>PF1488 </td><td>26.5 </td><td>−0.07 </td><td>0.31 </td><td>0.24 </td><td>− </td></tr><tr><td>III </td><td>PF0547 </td><td>50.7 </td><td>0.00 </td><td>−0.18 </td><td>−0.18 </td><td>− </td></tr><tr><td>III </td><td>PF1142 </td><td>31.5 </td><td>−0.48 </td><td>−0.41 </td><td>−0.89 </td><td>− </td></tr><tr><td>III </td><td>PF0030 </td><td>40.8 </td><td>−0.67 </td><td>−0.25 </td><td>−0.92 </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The DNA/RNA-binding activities were examined by gel-shift assay at room temperature (22°C) and 75°C.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The RNA-binding activities of PF1498 was examined by 1.2% agarose gel electrophoresis and ethidium bromide staining owing to the co-purification with endogenous nucleic acid in <em>E. coli</em> (Fig. <a reveal-id="DSM011F5" data-open="DSM011F5" class="link link-reveal link-table xref-fig">5</a>D).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194552" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM011TB3" class="table-wrap-title" data-id="DSM011TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of experimentally validated hypothetical proteins in <em>P. furiosus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Class </th><th rowspan="2">Gene ID </th><th rowspan="2">Molecular mass (kDa) </th><th colspan="3">SVM analysis </th><th rowspan="2">Experimenal verification of DNA/RNA-binding activity </th></tr><tr><th>PD score </th><th>CO score </th><th>CO + PD </th></tr></thead><tbody><tr><td>I </td><td>PF1498 </td><td>16.5 </td><td>1.11 </td><td>1.55 </td><td>2.66 </td><td>+<sup>a</sup> </td></tr><tr><td>I </td><td>PF1139 </td><td>44.7 </td><td>1.44 </td><td>0.74 </td><td>2.18 </td><td>+ </td></tr><tr><td>I </td><td>PF2062 </td><td>11.0 </td><td>0.45 </td><td>1.36 </td><td>1.81 </td><td>+ </td></tr><tr><td>I </td><td>PF0565 </td><td>17.2 </td><td>0.43 </td><td>0.56 </td><td>0.99 </td><td>+ </td></tr><tr><td>I </td><td>PF1473 </td><td>26.7 </td><td>0.53 </td><td>0.44 </td><td>0.97 </td><td>+ </td></tr><tr><td>I </td><td>PF1580 </td><td>25.5 </td><td>0.40 </td><td>0.35 </td><td>0.75 </td><td>+ </td></tr><tr><td>I </td><td>PF1913 </td><td>18.5 </td><td>0.23 </td><td>0.42 </td><td>0.65 </td><td>− </td></tr><tr><td>II </td><td>PF1981 </td><td>27.5 </td><td>0.18 </td><td>0.37 </td><td>0.55 </td><td>+ </td></tr><tr><td>II </td><td>PF1912 </td><td>48.2 </td><td>0.29 </td><td>0.04 </td><td>0.33 </td><td>+ </td></tr><tr><td>II </td><td>PF0029 </td><td>56.4 </td><td>0.22 </td><td>0.03 </td><td>0.25 </td><td>+ </td></tr><tr><td>II </td><td>PF1488 </td><td>26.5 </td><td>−0.07 </td><td>0.31 </td><td>0.24 </td><td>− </td></tr><tr><td>III </td><td>PF0547 </td><td>50.7 </td><td>0.00 </td><td>−0.18 </td><td>−0.18 </td><td>− </td></tr><tr><td>III </td><td>PF1142 </td><td>31.5 </td><td>−0.48 </td><td>−0.41 </td><td>−0.89 </td><td>− </td></tr><tr><td>III </td><td>PF0030 </td><td>40.8 </td><td>−0.67 </td><td>−0.25 </td><td>−0.92 </td><td>+ </td></tr></tbody></table></div><div content-id="DSM011TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Class </th><th rowspan="2">Gene ID </th><th rowspan="2">Molecular mass (kDa) </th><th colspan="3">SVM analysis </th><th rowspan="2">Experimenal verification of DNA/RNA-binding activity </th></tr><tr><th>PD score </th><th>CO score </th><th>CO + PD </th></tr></thead><tbody><tr><td>I </td><td>PF1498 </td><td>16.5 </td><td>1.11 </td><td>1.55 </td><td>2.66 </td><td>+<sup>a</sup> </td></tr><tr><td>I </td><td>PF1139 </td><td>44.7 </td><td>1.44 </td><td>0.74 </td><td>2.18 </td><td>+ </td></tr><tr><td>I </td><td>PF2062 </td><td>11.0 </td><td>0.45 </td><td>1.36 </td><td>1.81 </td><td>+ </td></tr><tr><td>I </td><td>PF0565 </td><td>17.2 </td><td>0.43 </td><td>0.56 </td><td>0.99 </td><td>+ </td></tr><tr><td>I </td><td>PF1473 </td><td>26.7 </td><td>0.53 </td><td>0.44 </td><td>0.97 </td><td>+ </td></tr><tr><td>I </td><td>PF1580 </td><td>25.5 </td><td>0.40 </td><td>0.35 </td><td>0.75 </td><td>+ </td></tr><tr><td>I </td><td>PF1913 </td><td>18.5 </td><td>0.23 </td><td>0.42 </td><td>0.65 </td><td>− </td></tr><tr><td>II </td><td>PF1981 </td><td>27.5 </td><td>0.18 </td><td>0.37 </td><td>0.55 </td><td>+ </td></tr><tr><td>II </td><td>PF1912 </td><td>48.2 </td><td>0.29 </td><td>0.04 </td><td>0.33 </td><td>+ </td></tr><tr><td>II </td><td>PF0029 </td><td>56.4 </td><td>0.22 </td><td>0.03 </td><td>0.25 </td><td>+ </td></tr><tr><td>II </td><td>PF1488 </td><td>26.5 </td><td>−0.07 </td><td>0.31 </td><td>0.24 </td><td>− </td></tr><tr><td>III </td><td>PF0547 </td><td>50.7 </td><td>0.00 </td><td>−0.18 </td><td>−0.18 </td><td>− </td></tr><tr><td>III </td><td>PF1142 </td><td>31.5 </td><td>−0.48 </td><td>−0.41 </td><td>−0.89 </td><td>− </td></tr><tr><td>III </td><td>PF0030 </td><td>40.8 </td><td>−0.67 </td><td>−0.25 </td><td>−0.92 </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The DNA/RNA-binding activities were examined by gel-shift assay at room temperature (22°C) and 75°C.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The RNA-binding activities of PF1498 was examined by 1.2% agarose gel electrophoresis and ethidium bromide staining owing to the co-purification with endogenous nucleic acid in <em>E. coli</em> (Fig. <a reveal-id="DSM011F5" data-open="DSM011F5" class="link link-reveal link-table xref-fig">5</a>D).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194552" target="_blank">View Large</a></div></div><p>During our investigation, six out of seven class I proteins, three out of four class II proteins, and one class III protein have shown potential DNA/RNA-binding activities (Table <a reveal-id="DSM011TB3" data-open="DSM011TB3" class="link link-reveal link-table xref-fig">3</a>). According to our previous works,<sup><a reveal-id="DSM011C8" data-open="DSM011C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSM011C9" data-open="DSM011C9" class="link link-ref link-reveal xref-bibr">9</a></sup> the systematic screening of <em>P. furiosus</em> genome using the expression cloning method has determined several DNA/RNA-binding proteins such as Thy-1 and FAU-1. Our system also demonstrated that approximately 10–20% of the <em>P. furiosus</em> gene products have shown to possess nucleic acid-binding activity. However, 80–90% of the <em>P. furiosus</em> gene products did not show any affinity to the nucleic acid in our gel-shift system. These results suggest that the 10 newly discovered proteins through the experimental procedure are good candidates as novel DNA/RNA-binding proteins, although their binding specificity remains unknown. The in vivo targets and precise biological functions of these proteins are to be further investigated.</p><p>According to the domain assignment of the InterPro/Pfam domain database<sup><a reveal-id="DSM011C20" data-open="DSM011C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSM011C21" data-open="DSM011C21" class="link link-ref link-reveal xref-bibr">21</a></sup> against <em>P. furiosus</em> proteome, 95–98% of the 942 functionally known proteins possessed domains related to those with known function (functional domains). On the other hand, for 994 hypothetical proteins, only 31–38% of the proteins possessed functional domains, 20% possessed domains of unknown function (DUF/UPF), and the remaining 43–50% lacked domain annotation (Supplementary Fig. S2). According to Supplementary Table 3, among the newly discovered 10 DNA/RNA-binding proteins, at least four ORFans are detected (PF0029, PF0030, PF0565, and PF1981), which completely lacked sequence similarity (E-value &gt; 0.1) compared with any of the Swiss-Prot protein entries. The remaining six proteins have shown sequential similarity to the uncharacterized proteins of nearest BLASTP hit (8.00e–07 &gt; E-value &gt; 0.0), which were conserved among <em>Pyrococcus</em> and <em>Methanococcus</em>, including two proteins (PF1473 and PF2062) with no Pfam domain annotation. Hence, we believe that the combination of CO score and PD score is a powerful indicator for predicting proteins with potential DNA/RNA-binding activities from the sequence-specific ORFans and a set of proteins having no obvious functional domains, although allowing that the sample size of validated proteins is still small.</p>                            <h3 scrollto-destination=87194555 id="87194555" class="section-title" data-legacy-id=s3e>3.5. Possible explanation of charged amino acid periodicity with DNA/RNA-binding activities</h3>
<p>As the amino acid composition within proteins varies among taxa,<sup><a reveal-id="DSM011C34" data-open="DSM011C34" class="link link-ref link-reveal xref-bibr">34</a></sup> our method removes the need to allow for the evolutionary gain and loss of amino acids and increases the generalization capability of SVM training. Charged amino acids, especially basic amino acids, have previously been suggested as a key component of nucleic acid-binding activity; for example, arginine-rich regions of the <em>Drosophila melanogaster</em> suppressor of sable gene<sup><a reveal-id="DSM011C35" data-open="DSM011C35" class="link link-ref link-reveal xref-bibr">35</a></sup> are thought to mediate specific RNA-binding activity. Similar features have been observed in the structural motifs of DNA/RNA-binding proteins that possess positive electrostatic potentials in the binding region.<sup><a reveal-id="DSM011C36" data-open="DSM011C36" class="link link-ref link-reveal xref-bibr">36</a></sup> On the basis of electrostatic potential, negatively charged amino acids (DE) conflict with DNA/RNA-binding. However, recent work has revealed that negative peptide charges contribute significantly to the electrostatic free energy of positively charged peptides and affect RNA binding,<sup><a reveal-id="DSM011C37" data-open="DSM011C37" class="link link-ref link-reveal xref-bibr">37</a></sup> suggesting the importance of not only basic regions but also, in some cases, acidic regions at the protein surface, for establishing DNA/RNA-binding functions. These results suggest that relative compositions of charged amino acids, especially basic amino acids, are very important for nucleic acid binding. In addition, our study has shown that certain class of DNA/RNA-binding proteins were efficiently classified by integrating amino acid periodicity with amino acid composition. Especially, charged amino acid periodicities have been observed throughout the protein sequence of various DNA/RNA-binding proteins, suggesting that not only amino acid compositions in the DNA/RNA-binding domain region but also the overall sequence feature of amino acid periodicity is useful for classifying DNA/RNA-binding proteins.</p><p>To gain insight into the relationship between charged amino acid periodicities and DNA/RNA-binding activity, a schematic representation of charged amino acid groups that appear periodically in the amino acid sequence of various DNA/RNA-binding proteins is given in Supplementary Fig. S3. The three proteins, signal recognition particle of 54kDa subunit (SRP54), DNA primase, and HTH-type transcriptional regulator lrpA, were chosen as an example for their characteristic features of possessing low CO score (–0.03 &lt; CO score &lt; 0.22) but relatively high PD score (0.56 &lt; PD score &lt; 0.88). An amino acid periodicity of both positively and negatively charged amino acids with various periodicities were widely found through protein primary sequence. The amino acid residues creating the periodicity (oblong boxes in Supplementary Fig. S3) are often conserved in the three-dimensional structures of orthologous proteins. Periodic region also covers DNA/RNA-recognition motif known as M domains and helix-turn-helix and active site of DNA primase. Our current study has shown that overall periodic features of charged amino acids throughout the protein primary structure may affect the organization of the secondary structures or the net charge of the protein surface in the tertiary structure in certain class of DNA/RNA-binding proteins. Further detailed analysis of the relationship between DNA/RNA-binding capacity and specific amino acid periodicity will be an important task with the help of other bioinformatics approaches such as the use of DNA/RNA-binding site prediction software,<sup><a reveal-id="DSM011C38" data-open="DSM011C38" class="link link-ref link-reveal xref-bibr">38</a></sup> a comparative genomics approach that predicts function on the basis of the comparison of various domains,<sup><a reveal-id="DSM011C39" data-open="DSM011C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and three-dimensional protein models.<sup><a reveal-id="DSM011C40" data-open="DSM011C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>In conclusion, we have presented a new method for predicting novel DNA/RNA-binding proteins at the proteome level by focusing on compositions and periodicities of amino acids with similar physico-chemical profiles (quantified as a novel index denoted as CO score and PD score). The two-dimensional correlation analysis of CO score and PD score effectively separated DNA/RNA-binding proteins from other functionally known proteins in <em>P. furiosus</em> as class I proteins. By applying the same method to the 994 hypothetical proteins, we extracted a list of 151 hypothetical proteins as novel DNA/RNA-binding protein candidates. Ten proteins with potential DNA/RNA-binding activities were determined experimentally, including four ORFans and two proteins with no domains. The two-dimensional correlation analysis of CO score and PD score is applicable to any organisms with complete genomic data. To conclude, our method is highly efficient for evaluating hypothetical proteins on the basis of DNA/RNA-binding function. The CO + PD scores can be further integrated with prediction results from various protein function predictors and annotation methods to validate uncharacterized proteins comprehensively. Further, the investigation of these newly discovered DNA/RNA-binding proteins might elucidate the role of undiscovered protein–DNA/RNA networks and the recognition of many non-conserved proteins throughout entire species.</p>                            <h2 scrollto-destination=87194559 id="87194559" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Asako Sato (Keio University, Japan) for technical assistance with the gel-shift assay. We also thank Jun Imoto, Nozomu Yachie, Shinichi Kikuchi, and Rintaro Saito (Keio University, Japan) for their helpful discussions. This research was supported in part by the Project for Development of a Technological Infrastructure for Industrial Bioprocesses in Research and Development of New Industrial Science and Technology Frontiers, the Ministry of Economy, Trade and Industry (METI), the New Energy and Industrial Technology Development Organization (NEDO) of Japan; a Grant-in-Aid for Scientific Research on Priority Areas; a Grant-in-Aid for the 21st Century Center of Excellence (COE) Program entitled ‘Understanding and Control of Life's Function via Systems Biology’ (Keio University); the Computer Simulation Project, Ministry of Education, Culture, Sport, Science and Technology, Japan; and Keio University.</p>                            <h2 scrollto-destination=87194561 id="87194561" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM011C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K. D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D. R</div></div>. </div><div class="article-title">NCBI Reference Sequence RefSeq: a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D501</div>-<div class="lpage">504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20Reference%20Sequence%20RefSeq%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.%20D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.%20R%20Maglott&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D501-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pazos</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Sternberg</div> <div class="given-names">M. J</div></div>. </div><div class="article-title">Automated prediction of protein function and detection of functional sites from structure</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">14754</div>-<div class="lpage">14759</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20prediction%20of%20protein%20function%20and%20detection%20of%20functional%20sites%20from%20structure&amp;author=F.%20Pazos&amp;author=M.%20J%20Sternberg&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=14754-14759" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0404569101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404569101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0404569101"> </span></div> </div></div></div></div></div><div content-id="DSM011C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLaughlin</div> <div class="given-names">W. A.</div></div>,  <div class="name"><div class="surname">Kulp</div> <div class="given-names">D. W.</div></div>,  <div class="name"><div class="surname">de la Cruz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">X. J.</div></div>,  <div class="name"><div class="surname">Lawson</div> <div class="given-names">C. L.</div></div>,  <div class="name"><div class="surname">Berman</div> <div class="given-names">H. M</div></div>. </div><div class="article-title">A structure-based method for identifying DNA-binding proteins and their sites of DNA-interaction</div>, <div class="source">J. Struct. Funct. Genomics.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">255</div>-<div class="lpage">265</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20structure-based%20method%20for%20identifying%20DNA-binding%20proteins%20and%20their%20sites%20of%20DNA-interaction&amp;author=W.%20A.%20McLaughlin&amp;author=D.%20W.%20Kulp&amp;author=J.%20de%20la%20Cruz&amp;author=X.%20J.%20Lu&amp;author=C.%20L.%20Lawson&amp;author=H.%20M%20Berman&amp;publication_year=2004&amp;journal=J.%20Struct.%20Funct.%20Genomics.&amp;volume=5&amp;pages=255-265" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10969-004-4902-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10969-004-4902-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10969-004-4902-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15704013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Date</div> <div class="given-names">S. V.</div></div>,  <div class="name"><div class="surname">Marcotte</div> <div class="given-names">E. M</div></div>. </div><div class="article-title">Discovery of uncharacterized cellular systems by genome-wide analysis of functional linkages</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1055</div>-<div class="lpage">1062</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20uncharacterized%20cellular%20systems%20by%20genome-wide%20analysis%20of%20functional%20linkages&amp;author=S.%20V.%20Date&amp;author=E.%20M%20Marcotte&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol.&amp;volume=21&amp;pages=1055-1062" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12923548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amiri</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Davids</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">S. G</div></div>. </div><div class="article-title">Birth and death of orphan genes in Rickettsia</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1575</div>-<div class="lpage">1587</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Birth%20and%20death%20of%20orphan%20genes%20in%20Rickettsia&amp;author=H.%20Amiri&amp;author=W.%20Davids&amp;author=S.%20G%20Andersson&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=1575-1587" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12832625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Siew</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Fischer</div> <div class="given-names">D</div></div>. </div><div class="article-title">Structural biology sheds light on the puzzle of genomic ORFans</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">342</div> (pg. <div class="fpage">369</div>-<div class="lpage">373</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20biology%20sheds%20light%20on%20the%20puzzle%20of%20genomic%20ORFans&amp;author=N.%20Siew&amp;author=D%20Fischer&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol.&amp;volume=342&amp;pages=369-373" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2004.06.073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2004.06.073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2004.06.073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15327940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Oida</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Matsuura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Doi</div> <div class="given-names">H</div></div>. </div><div class="article-title">Expression cloning and characterization of a novel gene that encodes the RNA-binding protein FAU-1 from <em>Pyrococcus furiosus</em></div>, <div class="source">Biochem. J.</div> , <div class="year">2003</div>, vol. <div class="volume">372</div> (pg. <div class="fpage">253</div>-<div class="lpage">261</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20cloning%20and%20characterization%20of%20a%20novel%20gene%20that%20encodes%20the%20RNA-binding%20protein%20FAU-1%20from%20Pyrococcus%20furiosus&amp;author=A.%20Kanai&amp;author=H.%20Oida&amp;author=N.%20Matsuura&amp;author=H%20Doi&amp;publication_year=2003&amp;journal=Biochem.%20J.&amp;volume=372&amp;pages=253-261" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj20021968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj20021968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj20021968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12614195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Imoto</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M</div></div>. </div><div class="article-title">Archaeal <em>Pyrococcus furiosus</em> thymidylate synthase 1 is an RNA-binding protein</div>, <div class="source">Biochem. J.</div> , <div class="year">2006</div>, vol. <div class="volume">393</div> (pg. <div class="fpage">373</div>-<div class="lpage">379</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Archaeal%20Pyrococcus%20furiosus%20thymidylate%20synthase%201%20is%20an%20RNA-binding%20protein&amp;author=A.%20Kanai&amp;author=A.%20Sato&amp;author=J.%20Imoto&amp;author=M%20Tomita&amp;publication_year=2006&amp;journal=Biochem.%20J.&amp;volume=393&amp;pages=373-379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BJ20050608" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20050608" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBJ20050608"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16176183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Itaya</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M</div></div>. </div><div class="article-title">Cooperative regulation for Okazaki fragment processing by RNase HII and FEN-1 purified from a hyperthermophilic archaeon, <em>Pyrococcus furiosus</em></div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2003</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">247</div>-<div class="lpage">252</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cooperative%20regulation%20for%20Okazaki%20fragment%20processing%20by%20RNase%20HII%20and%20FEN-1%20purified%20from%20a%20hyperthermophilic%20archaeon%2C%20Pyrococcus%20furiosus&amp;author=A.%20Sato&amp;author=A.%20Kanai&amp;author=M.%20Itaya&amp;author=M%20Tomita&amp;publication_year=2003&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=309&amp;pages=247-252" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2003.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2003.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2003.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12943689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cotton</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Mykles</div> <div class="given-names">D. L</div></div>. </div><div class="article-title">Cloning of a crustacean myosin heavy chain isoform: exclusive expression in fast muscle</div>, <div class="source">J. Exp. Zool.</div> , <div class="year">1993</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">578</div>-<div class="lpage">586</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20of%20a%20crustacean%20myosin%20heavy%20chain%20isoform%3A%20exclusive%20expression%20in%20fast%20muscle&amp;author=J.%20L.%20Cotton&amp;author=D.%20L%20Mykles&amp;publication_year=1993&amp;journal=J.%20Exp.%20Zool.&amp;volume=267&amp;pages=578-586" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-010X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-010X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-010X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8277231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laskin</div> <div class="given-names">A. A.</div></div>,  <div class="name"><div class="surname">Kudryashov</div> <div class="given-names">N. A.</div></div>,  <div class="name"><div class="surname">Skryabin</div> <div class="given-names">K. G.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V</div></div>. </div><div class="article-title">Latent periodicity of serine–threonine and tyrosine protein kinases and other protein families</div>, <div class="source">Comput. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">229</div>-<div class="lpage">243</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Latent%20periodicity%20of%20serine%E2%80%93threonine%20and%20tyrosine%20protein%20kinases%20and%20other%20protein%20families&amp;author=A.%20A.%20Laskin&amp;author=N.%20A.%20Kudryashov&amp;author=K.%20G.%20Skryabin&amp;author=E.%20V%20Korotkov&amp;publication_year=2005&amp;journal=Comput.%20Biol.%20Chem.&amp;volume=29&amp;pages=229-243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.compbiolchem.2005.04.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.compbiolchem.2005.04.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.compbiolchem.2005.04.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15979043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhardwaj</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Langlois</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">H</div></div>. </div><div class="article-title">Kernel-based machine learning protocol for predicting DNA-binding proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">6486</div>-<div class="lpage">6493</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kernel-based%20machine%20learning%20protocol%20for%20predicting%20DNA-binding%20proteins&amp;author=N.%20Bhardwaj&amp;author=R.%20E.%20Langlois&amp;author=G.%20Zhao&amp;author=H%20Lu&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=6486-6493" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki949" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki949" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki949"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16284202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">L. Y.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">C. Z.</div></div>,  <div class="name"><div class="surname">Lo</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Chung</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y. Z</div></div>. </div><div class="article-title">Prediction of RNA-binding proteins from primary sequence by a support vector machine approach</div>, <div class="source">RNA</div> , <div class="year">2004</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">355</div>-<div class="lpage">368</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20RNA-binding%20proteins%20from%20primary%20sequence%20by%20a%20support%20vector%20machine%20approach&amp;author=L.%20Y.%20Han&amp;author=C.%20Z.%20Cai&amp;author=S.%20L.%20Lo&amp;author=M.%20C.%20Chung&amp;author=Y.%20Z%20Chen&amp;publication_year=2004&amp;journal=RNA&amp;volume=10&amp;pages=355-368" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.5890304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.5890304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.5890304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14970381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cai</div> <div class="given-names">Y. D.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S. L</div></div>. </div><div class="article-title">Support vector machines for predicting rRNA-, RNA-, and DNA-binding proteins from amino acid sequence</div>, <div class="source">Biochim. Biophys. Acta.</div> , <div class="year">2003</div>, vol. <div class="volume">1648</div> (pg. <div class="fpage">127</div>-<div class="lpage">133</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Support%20vector%20machines%20for%20predicting%20rRNA-%2C%20RNA-%2C%20and%20DNA-binding%20proteins%20from%20amino%20acid%20sequence&amp;author=Y.%20D.%20Cai&amp;author=S.%20L%20Lin&amp;publication_year=2003&amp;journal=Biochim.%20Biophys.%20Acta.&amp;volume=1648&amp;pages=127-133" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1570-9639(03)00112-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1570-9639(03)00112-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1570-9639(03)00112-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12758155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y</div></div>. </div><div class="article-title">Predicting rRNA-, RNA-, and DNA-binding proteins from primary structure with support vector machines</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">240</div> (pg. <div class="fpage">175</div>-<div class="lpage">184</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20rRNA-%2C%20RNA-%2C%20and%20DNA-binding%20proteins%20from%20primary%20structure%20with%20support%20vector%20machines&amp;author=X.%20Yu&amp;author=J.%20Cao&amp;author=Y.%20Cai&amp;author=T.%20Shi&amp;author=Y%20Li&amp;publication_year=2006&amp;journal=J.%20Theor.%20Biol.&amp;volume=240&amp;pages=175-184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2005.09.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2005.09.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2005.09.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16274699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ofran</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Margalit</div> <div class="given-names">H</div></div>. </div><div class="article-title">Proteins of the same fold and unrelated sequences have similar amino acid composition</div>, <div class="source">Proteins</div> , <div class="year">2006</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">275</div>-<div class="lpage">279</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteins%20of%20the%20same%20fold%20and%20unrelated%20sequences%20have%20similar%20amino%20acid%20composition&amp;author=Y.%20Ofran&amp;author=H%20Margalit&amp;publication_year=2006&amp;journal=Proteins&amp;volume=64&amp;pages=275-279" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/prot.20964" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fprot.20964" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fprot.20964"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16565950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">H</div></div>. </div><div class="article-title">LOCSVMPSI: a web server for subcellular localization of eukaryotic proteins using SVM and profile of PSI-BLAST</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W105</div>-<div class="lpage">W110</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LOCSVMPSI%3A%20a%20web%20server%20for%20subcellular%20localization%20of%20eukaryotic%20proteins%20using%20SVM%20and%20profile%20of%20PSI-BLAST&amp;author=D.%20Xie&amp;author=A.%20Li&amp;author=M.%20Wang&amp;author=Z.%20Fan&amp;author=H%20Feng&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W105-W110" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sarda</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chua</div> <div class="given-names">G. H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">K. B.</div></div>,  <div class="name"><div class="surname">Krishnan</div> <div class="given-names">A</div></div>. </div><div class="article-title">pSLIP: SVM based protein subcellular localization prediction using multiple physicochemical properties</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">152</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=pSLIP%3A%20SVM%20based%20protein%20subcellular%20localization%20prediction%20using%20multiple%20physicochemical%20properties&amp;author=D.%20Sarda&amp;author=G.%20H.%20Chua&amp;author=K.%20B.%20Li&amp;author=A%20Krishnan&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15963230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T. K.</div></div>,  <div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">InterPro—an integrated documentation resource for protein families, domains and functional sites</div>, <div class="source">Bioinformatics</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1145</div>-<div class="lpage">1150</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterPro%E2%80%94an%20integrated%20documentation%20resource%20for%20protein%20families%2C%20domains%20and%20functional%20sites&amp;author=R.%20Apweiler&amp;author=T.%20K.%20Attwood&amp;author=A.%20Bairoch&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=1145-1150" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.12.1145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.12.1145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.12.1145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11159333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Howe</div> <div class="given-names">K. L.</div></div>,  <div class="name"><div class="surname">Sonnhammer</div> <div class="given-names">E. L</div></div>. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">263</div>-<div class="lpage">266</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=A.%20Bateman&amp;author=E.%20Birney&amp;author=R.%20Durbin&amp;author=S.%20R.%20Eddy&amp;author=K.%20L.%20Howe&amp;author=E.%20L%20Sonnhammer&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=263-266" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gatherer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mc Ewan</div> <div class="given-names">N. R</div></div>. </div><div class="article-title">Analysis of sequence periodicity in <em>E. coli</em> proteins: empirical investigation of the ‘duplication and divergence’ theory of protein evolution</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">149</div>-<div class="lpage">158</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20sequence%20periodicity%20in%20E.%20coli%20proteins%3A%20empirical%20investigation%20of%20the%20%E2%80%98duplication%20and%20divergence%E2%80%99%20theory%20of%20protein%20evolution&amp;author=D.%20Gatherer&amp;author=N.%20R%20Mc%20Ewan&amp;publication_year=2003&amp;journal=J.%20Mol.%20Evol.&amp;volume=57&amp;pages=149-158" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-002-2462-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-002-2462-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-002-2462-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562959" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pavlidis</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wapinski</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Noble</div> <div class="given-names">W. S</div></div>. </div><div class="article-title">Support vector machine classification on the web</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">586</div>-<div class="lpage">587</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Support%20vector%20machine%20classification%20on%20the%20web&amp;author=P.%20Pavlidis&amp;author=I.%20Wapinski&amp;author=W.%20S%20Noble&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=586-587" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14990457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Nam</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">Rhee</div> <div class="given-names">J. K.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W. J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">B. T</div></div>. </div><div class="article-title">miTarget: microRNA target gene prediction using a support vector machine</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">411</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miTarget%3A%20microRNA%20target%20gene%20prediction%20using%20a%20support%20vector%20machine&amp;author=S.%20K.%20Kim&amp;author=J.%20W.%20Nam&amp;author=J.%20K.%20Rhee&amp;author=W.%20J.%20Lee&amp;author=B.%20T%20Zhang&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-411" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-411" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-411"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16978421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zavaljevski</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stevens</div> <div class="given-names">F. J.</div></div>,  <div class="name"><div class="surname">Yackovich</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Reifman</div> <div class="given-names">J</div></div>. </div><div class="article-title">Classifying noisy protein sequence data: a case study of immunoglobulin light chains</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">495</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classifying%20noisy%20protein%20sequence%20data%3A%20a%20case%20study%20of%20immunoglobulin%20light%20chains&amp;author=C.%20Yu&amp;author=N.%20Zavaljevski&amp;author=F.%20J.%20Stevens&amp;author=K.%20Yackovich&amp;author=J%20Reifman&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=495-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1024"> </span></div> </div></div></div></div></div><div content-id="DSM011C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldbaum</div> <div class="given-names">M. H.</div></div>,  <div class="name"><div class="surname">Sample</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Comparing machine learning classifiers for diagnosing glaucoma from standard automated perimetry</div>, <div class="source">Invest. Ophthalmol. Vis. Sci</div> , <div class="year">2002</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">162</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparing%20machine%20learning%20classifiers%20for%20diagnosing%20glaucoma%20from%20standard%20automated%20perimetry&amp;author=M.%20H.%20Goldbaum&amp;author=P.%20A.%20Sample&amp;author=K.%20Chan&amp;publication_year=2002&amp;journal=Invest.%20Ophthalmol.%20Vis.%20Sci&amp;volume=43&amp;pages=162-169" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11773027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cai</div> <div class="given-names">C. Z.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">L. Y.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">Z. L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y. Z</div></div>. </div><div class="article-title">SVM-Prot: web-based support vector machine software for functional classification of a protein from its primary sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3692</div>-<div class="lpage">3697</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SVM-Prot%3A%20web-based%20support%20vector%20machine%20software%20for%20functional%20classification%20of%20a%20protein%20from%20its%20primary%20sequence&amp;author=C.%20Z.%20Cai&amp;author=L.%20Y.%20Han&amp;author=Z.%20L.%20Ji&amp;author=X.%20Chen&amp;author=Y.%20Z%20Chen&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3692-3697" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">L. Y.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">C. Z.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">Z. L.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">Z. W.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y. Z</div></div>. </div><div class="article-title">Predicting functional family of novel enzymes irrespective of sequence similarity: a statistical learning approach</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">6437</div>-<div class="lpage">6444</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20functional%20family%20of%20novel%20enzymes%20irrespective%20of%20sequence%20similarity%3A%20a%20statistical%20learning%20approach&amp;author=L.%20Y.%20Han&amp;author=C.%20Z.%20Cai&amp;author=Z.%20L.%20Ji&amp;author=Z.%20W.%20Cao&amp;author=J.%20Cui&amp;author=Y.%20Z%20Chen&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=6437-6444" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh984" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh984" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh984"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15585667" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pavlidis</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Weston</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Noble</div> <div class="given-names">W. S</div></div>. </div><div class="article-title">Learning gene functional classifications from multiple data types</div>, <div class="source">J. Comput. Biol</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">401</div>-<div class="lpage">411</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Learning%20gene%20functional%20classifications%20from%20multiple%20data%20types&amp;author=P.%20Pavlidis&amp;author=J.%20Weston&amp;author=J.%20Cai&amp;author=W.%20S%20Noble&amp;publication_year=2002&amp;journal=J.%20Comput.%20Biol&amp;volume=2&amp;pages=401-411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/10665270252935539" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F10665270252935539" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F10665270252935539"> </span></div> </div></div></div></div></div><div content-id="DSM011C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matthews</div> <div class="given-names">B. W</div></div>. </div><div class="article-title">Comparison of the predicted and observed secondary structure of T4 phage lysozyme</div>, <div class="source">Biochim. Biophys. Acta.</div> , <div class="year">1975</div>, vol. <div class="volume">405</div> (pg. <div class="fpage">442</div>-<div class="lpage">451</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20the%20predicted%20and%20observed%20secondary%20structure%20of%20T4%20phage%20lysozyme&amp;author=B.%20W%20Matthews&amp;publication_year=1975&amp;journal=Biochim.%20Biophys.%20Acta.&amp;volume=405&amp;pages=442-451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0005-2795(75)90109-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0005-2795(75)90109-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0005-2795(75)90109-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1180967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robb</div> <div class="given-names">F. T.</div></div>,  <div class="name"><div class="surname">Maeder</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">DiRuggiero</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Stump</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Yeh</div> <div class="given-names">R. K.</div></div>,  <div class="name"><div class="surname">Weiss</div> <div class="given-names">R. B.</div></div>,  <div class="name"><div class="surname">Dunn</div> <div class="given-names">D. M</div></div>. </div><div class="article-title">Genomic sequence of hyperthermophile, <em>Pyrococcus furiosus</em>: implications for physiology and enzymology</div>, <div class="source">Meth. Enzymol.</div> , <div class="year">2001</div>, vol. <div class="volume">330</div> (pg. <div class="fpage">134</div>-<div class="lpage">157</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20sequence%20of%20hyperthermophile%2C%20Pyrococcus%20furiosus%3A%20implications%20for%20physiology%20and%20enzymology&amp;author=F.%20T.%20Robb&amp;author=D.%20L.%20Maeder&amp;author=J.%20R.%20Brown&amp;author=J.%20DiRuggiero&amp;author=M.%20D.%20Stump&amp;author=R.%20K.%20Yeh&amp;author=R.%20B.%20Weiss&amp;author=D.%20M%20Dunn&amp;publication_year=2001&amp;journal=Meth.%20Enzymol.&amp;volume=330&amp;pages=134-157" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11210495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R</div></div>. </div><div class="article-title">The SWISS-PROT protein sequence data bank and its new supplement TREMBL</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1996</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">21</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20SWISS-PROT%20protein%20sequence%20data%20bank%20and%20its%20new%20supplement%20TREMBL&amp;author=A.%20Bairoch&amp;author=R%20Apweiler&amp;publication_year=1996&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=21-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/24.1.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.1.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F24.1.21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8594581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Camon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Magrane</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Barrell</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">The Gene Ontology Annotation, GOA project: implementation of GO in SWISS-PROT, TrEMBL, and InterPro</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">662</div>-<div class="lpage">672</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gene%20Ontology%20Annotation%2C%20GOA%20project%3A%20implementation%20of%20GO%20in%20SWISS-PROT%2C%20TrEMBL%2C%20and%20InterPro&amp;author=E.%20Camon&amp;author=M.%20Magrane&amp;author=D.%20Barrell&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=662-672" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.461403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.461403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.461403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jordan</div> <div class="given-names">I. K.</div></div>,  <div class="name"><div class="surname">Kondrashov</div> <div class="given-names">F. A.</div></div>,  <div class="name"><div class="surname">Adzhubei</div> <div class="given-names">I. A.</div></div>, et al. </div><div class="article-title">A universal trend of amino acid gain and loss in protein evolution</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">433</div> (pg. <div class="fpage">633</div>-<div class="lpage">638</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20universal%20trend%20of%20amino%20acid%20gain%20and%20loss%20in%20protein%20evolution&amp;author=I.%20K.%20Jordan&amp;author=F.%20A.%20Kondrashov&amp;author=I.%20A.%20Adzhubei&amp;publication_year=2005&amp;journal=Nature&amp;volume=433&amp;pages=633-638" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15660107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnage</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Brewer-Jensen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bai</div> <div class="given-names">W. L.</div></div>,  <div class="name"><div class="surname">Searles</div> <div class="given-names">L. L</div></div>. </div><div class="article-title">Arginine-rich regions mediate the RNA binding and regulatory activities of the protein encoded by the <em>Drosophila melanogaster</em> suppressor of sable gene</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">8198</div>-<div class="lpage">8208</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arginine-rich%20regions%20mediate%20the%20RNA%20binding%20and%20regulatory%20activities%20of%20the%20protein%20encoded%20by%20the%20Drosophila%20melanogaster%20suppressor%20of%20sable%20gene&amp;author=M.%20A.%20Turnage&amp;author=P.%20Brewer-Jensen&amp;author=W.%20L.%20Bai&amp;author=L.%20L%20Searles&amp;publication_year=2000&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=20&amp;pages=8198-8208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.20.21.8198-8208.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.20.21.8198-8208.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.20.21.8198-8208.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11027289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shanahan</div> <div class="given-names">H. P.</div></div>,  <div class="name"><div class="surname">Garcia</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J. M</div></div>. </div><div class="article-title">Identifying DNA-binding proteins using structural motifs and the electrostatic potential</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">4732</div>-<div class="lpage">4741</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20DNA-binding%20proteins%20using%20structural%20motifs%20and%20the%20electrostatic%20potential&amp;author=H.%20P.%20Shanahan&amp;author=M.%20A.%20Garcia&amp;author=S.%20Jones&amp;author=J.%20M%20Thornton&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=4732-4741" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15356290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Garcia</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Draper</div> <div class="given-names">D. E</div></div>. </div><div class="article-title">Electrostatic interactions in a peptide–RNA complex</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">331</div> (pg. <div class="fpage">75</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Electrostatic%20interactions%20in%20a%20peptide%E2%80%93RNA%20complex&amp;author=C.%20Garcia-Garcia&amp;author=D.%20E%20Draper&amp;publication_year=2003&amp;journal=J.%20Mol.%20Biol.&amp;volume=331&amp;pages=75-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(03)00615-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(03)00615-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(03)00615-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12875837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ahmad</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sarai</div> <div class="given-names">A</div></div>. </div><div class="article-title">PSSM-based prediction of DNA binding sites in proteins</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">33</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PSSM-based%20prediction%20of%20DNA%20binding%20sites%20in%20proteins&amp;author=S.%20Ahmad&amp;author=A%20Sarai&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15720719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anantharaman</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Aravind</div> <div class="given-names">L</div></div>. </div><div class="article-title">Comparative genomics and evolution of proteins involved in RNA metabolism</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1427</div>-<div class="lpage">1464</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20and%20evolution%20of%20proteins%20involved%20in%20RNA%20metabolism&amp;author=V.%20Anantharaman&amp;author=E.%20V.%20Koonin&amp;author=L%20Aravind&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=1427-1464" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.7.1427" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.7.1427" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.7.1427"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11917006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM011C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM011C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manival</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ghisolfi-Nieto</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Joseph</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Bouvet</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Erard</div> <div class="given-names">M</div></div>. </div><div class="article-title">RNA-binding strategies common to cold-shock domain- and RNA recognition motif-containing proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">2223</div>-<div class="lpage">2233</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-binding%20strategies%20common%20to%20cold-shock%20domain-%20and%20RNA%20recognition%20motif-containing%20proteins&amp;author=X.%20Manival&amp;author=L.%20Ghisolfi-Nieto&amp;author=G.%20Joseph&amp;author=P.%20Bouvet&amp;author=M%20Erard&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=2223-2233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.11.2223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.11.2223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.11.2223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11376140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=577454&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm011/1/dsm011_Supplementary_Data.zip?Expires=2147483647&amp;Signature=bi3gw9Hs42dnUFjH7K-GWOXRylyyLF168Hv92HS5JxBSeZ0tpQ~R7k7r-6YmrwoXsIlATiPkDwwCrFVxK-TCIVQPShd1eg2rXRjvxoaxYVYh~j1C0c0VLJrIz3gYjPMh1~CVXv1UPDQGVrLrTVfQZcBr-m~pVReG-YEHXtJfIZwrVSGR0yMdWwPecQCXjCbieGNx7HYvuillKe1jikLh0rka5rm7F8szWdmMEK3JlSOTeA2GVnCSlkE8dQYHXzNeX8FWkxtilLUbQznx8YhCpkAYdspBMAt-slz9-T2b-oXQOOq85qIxvtGdni3SGq8-DwaYXSBaXxK~0gT6kq~GxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">132</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248634100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm011"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="577454"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248634100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f3%2f91%2f577454">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000248634100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F3%2F91%2F577454" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/577454'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Proteome-Wide Prediction of Novel DNA/RNA-Binding Proteins Using Amino Acid Composition and Periodicity in the Hyperthermophilic Archaeon Pyrococcus furiosus" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm011");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f91%2f577454">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

