

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/6/581/2647451","@type":"ScholarlyArticle","name":"Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide","datePublished":"2016-08-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2016-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw041","keywords":["xenobiotics","evolution","sphingomonads","genome","mobile genetic elements"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/6/581/2647451","author":[{"name":"Tabata, Michiro","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Ohhata, Satoshi","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Nikawadori, Yuki","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Kishida, Kouhei","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Sato, Takuya","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Kawasumi, Toru","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Kato, Hiromi","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Ohtsubo, Yoshiyuki","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Tsuda, Masataka","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Nagata, Yuji","affiliation":"Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"}],"description":"γ-Hexachlorocyclohexane (γ-HCH) is a recalcitrant man-made chlorinated pesticide. Here, the complete genome sequences of four γ-HCH-degrading sphingomonad strai","pageStart":"581","pageEnd":"599","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide" />
    <meta property="og:description" content="γ-Hexachlorocyclohexane (γ-HCH) is a recalcitrant man-made chlorinated pesticide. Here, the complete genome sequences of four γ-HCH-degrading sphingomonad strai" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/6/581/2647451" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-08-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tabata, Michiro" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Ohhata, Satoshi" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Nikawadori, Yuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Kishida, Kouhei" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Sato, Takuya" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Kawasumi, Toru" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Kato, Hiromi" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Ohtsubo, Yoshiyuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Tsuda, Masataka" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Nagata, Yuji" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_title" content="Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide" /><meta name="citation_firstpage" content="581" /><meta name="citation_lastpage" content="599" /><meta name="citation_doi" content="10.1093/dnares/dsw041" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2016/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=International Pellet Watch: global monitoring of persistent organic pollutants (POPs) in coastal waters. 1. Initial phase data on PCBs, DDTs, and HCHs,;  citation_journal_title=Mar. Pollut. Bull;  citation_year=2009;  citation_volume=58;  citation_pages=1437-46; " /><meta name="citation_reference" content="citation_title=An overview on the accumulation, distribution, transformations, toxicity and analytical methods for the monitoring of persistent organic pollutants,;  citation_journal_title=Talanta;  citation_year=2010;  citation_volume=80;  citation_pages=1587-97; " /><meta name="citation_reference" content="citation_title=Organochlorine pesticides in soil, moss and tree-bark from North-Eastern Romania,;  citation_journal_title=Sci. Total Environ;  citation_year=2013;  citation_volume=456–457;  citation_pages=317-24; " /><meta name="citation_reference" content="citation_title=Bacterial degradation of xenobiotic compounds: evolution and distribution of novel enzyme activities,;  citation_journal_title=Environ. Microbiol;  citation_year=2005;  citation_volume=7;  citation_pages=1868-82; " /><meta name="citation_reference" content="citation_title=Evolution of efficient pathways for degradation of anthropogenic chemicals,;  citation_journal_title=Nature Chem. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=559-66; " /><meta name="citation_reference" content="citation_title=Molecular characteristics of xenobiotic-degrading sphingomonads,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2009;  citation_volume=81;  citation_pages=793-811; " /><meta name="citation_reference" content="citation_title=Biochemistry of microbial degradation of hexachlorocyclohexane and prospects for bioremediation,;  citation_journal_title=Microbiol. Mol. Biol. Rev;  citation_year=2010;  citation_volume=74;  citation_pages=58-80; " /><meta name="citation_reference" content="citation_title=Degradative plasmids from sphingomonads,;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2014;  citation_volume=350;  citation_pages=9-19; " /><meta name="citation_reference" content="citation_publisher=ASM Press, Washington, DC;  citation_title=Manual of Environmental Microbiology;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=The role of mobile genetic elements in bacterial adaptation to xenobiotic organic compounds,;  citation_journal_title=Curr. Opin. Biotechnol;  citation_year=2003;  citation_volume=14;  citation_pages=262-9; " /><meta name="citation_reference" content="citation_title=The whole genome sequence of Sphingobium chlorophenolicum L-1: insights into the evolution of the pentachlorophenol degradation pathway,;  citation_journal_title=Genome Biol. Evol;  citation_year=2011;  citation_volume=4;  citation_pages=184-98; " /><meta name="citation_reference" content="citation_title=Evolution of atrazine-degrading capabilities in the environment,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2012;  citation_volume=96;  citation_pages=1175-89; " /><meta name="citation_reference" content="citation_title=Metabolic characteristics of the species Variovorax paradoxus,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2013;  citation_volume=97;  citation_pages=541-60; " /><meta name="citation_reference" content="citation_title=Biodegradation of hexachlorocyclohexane (HCH) by microorganisms,;  citation_journal_title=Biodegradation;  citation_year=2005;  citation_volume=16;  citation_pages=363-92; " /><meta name="citation_reference" content="citation_title=Hexachlorocyclohexane (HCH) as new Stockholm Convention POPs-a global perspective on the management of Lindane and its waste isomers;  citation_journal_title=Environ. Sci. Pollut. Res. Int;  citation_year=2011;  citation_volume=18;  citation_pages=152-62; " /><meta name="citation_reference" content="citation_title=Aerobic degradation of lindane (γ-hexachlorocyclohexane) in bacteria and its biochemical and molecular basis,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2007;  citation_volume=76;  citation_pages=741-52; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the representative γ-hexachlorocyclohexane-degrading bacterium Sphingobium japonicum UT26,;  citation_journal_title=J. Bacteriol;  citation_year=2010;  citation_volume=192;  citation_pages=5852-3; " /><meta name="citation_reference" content="citation_title=Genomic organization and genomic structural rearrangements of Sphingobium japonicum UT26, an archetypal γ-hexachlorocyclohexane-degrading bacterium,;  citation_journal_title=Enzyme Microb. Technol;  citation_year=2011;  citation_volume=49;  citation_pages=499-508; " /><meta name="citation_reference" content="citation_publisher=Springer Verlag, Tokyo;  citation_title=Biodegradative Bacteria;  citation_year=2014;  citation_pages=19-41; " /><meta name="citation_reference" content="citation_title=Comparative genomic analysis of nine Sphingobium strains: insights into their evolution and hexachlorocyclohexane (HCH) degradation pathways,;  citation_journal_title=BMC genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1014" /><meta name="citation_reference" content="citation_title=Insights into ongoing evolution of the hexachlorocyclohexane catabolic pathway from comparative genomics of ten Sphingomonadaceae strains,;  citation_journal_title=G3;  citation_year=2015;  citation_volume=5;  citation_pages=1081-94; " /><meta name="citation_reference" content="citation_title=The lin genes for γ-hexachlorocyclohexane degradation in Sphingomonas sp. MM-1 proved to be dispersed across multiple plasmids,;  citation_journal_title=Biosci. Biotechnol. Biochem;  citation_year=2011;  citation_volume=75;  citation_pages=466-72; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the γ-hexachlorocyclohexane-degrading bacterium Sphingomonas sp. strain MM-1,;  citation_journal_title=Genome Announc;  citation_year=2013;  citation_volume=1;  citation_pages=e00247–13" /><meta name="citation_reference" content="citation_title=Degradation of β-hexachlorocyclohexane by haloalkane dehalogenase LinB from γ-hexachlorocyclohexane-utilizing bacterium Sphingobium sp. MI1205,;  citation_journal_title=Arch. Microbiol;  citation_year=2007;  citation_volume=188;  citation_pages=313-25; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of a γ-hexachlorocyclohexane-degrading bacterium, Sphingobium sp. strain MI1205,;  citation_journal_title=Genome Announc;  citation_year=2016;  citation_volume=4;  citation_pages=e00246-16; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of a γ-hexachlorocyclohexane degrader, Sphingobium sp. strain TKS, isolated from a γ-hexachlorocyclohexane-degrading microbial community,;  citation_journal_title=Genome Announc;  citation_year=2016;  citation_volume=4;  citation_pages=e00247–16" /><meta name="citation_reference" content="citation_title=Emendation of the genus Sphingomonas Yabuuchi et al. 1990 and junior objective synonymy of the species of three genera, Sphingobium, Novosphingobium and Sphingopyxis, in conjunction with Blastomonas ursincola,;  citation_journal_title=Int. J. Syst. Evol. Microbiol;  citation_year=2002;  citation_volume=52;  citation_pages=1485-96; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory, NY;  citation_title=Molecular cloning: a Laboratory Manual;  citation_year=1982; " /><meta name="citation_reference" content="citation_title=Identification and characterization of genes encoding a putative ABC-type transporter essential for utilization of γ-hexachlorocyclohexane in Sphingobium japonicum UT26,;  citation_journal_title=J. Bacteriol;  citation_year=2007;  citation_volume=189;  citation_pages=3712-20; " /><meta name="citation_reference" content="citation_title=Two different types of dehalogenases, LinA and LinB, involved in γ-hexachlorocyclohexane degradation in Sphingomonas paucimobilis UT26 are localized in the periplasmic space without molecular processing;  citation_journal_title=J. Bacteriol;  citation_year=1999;  citation_volume=181;  citation_pages=5409-13; " /><meta name="citation_reference" content="citation_title=Four new derivatives of the broad-host-range cloning vector pBBR1MCS, carrying different antibiotic-resistance cassettes,;  citation_journal_title=Gene;  citation_year=1995;  citation_volume=166;  citation_pages=175-6; " /><meta name="citation_reference" content="citation_title=A broad-host-range Flp-FRT recombination system for site-specific excision of chromosomally-located DNA sequences: application for isolation of unmarked Pseudomonas aeruginosa mutants,;  citation_journal_title=Gene;  citation_year=1998;  citation_volume=212;  citation_pages=77-86; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor, NY;  citation_title=Molecular Cloning: A Laboratory Manual;  citation_year=1989; " /><meta name="citation_reference" content="citation_title=Identification and characterization of genes involved in the downstream degradation pathway of γ-hexachlorocyclohexane in Sphingomonas paucimobilis UT26,;  citation_journal_title=J. Bacteriol;  citation_year=2005;  citation_volume=187;  citation_pages=847-53; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Acidovorax sp. strain KKS102, a polychlorinated-biphenyl degrader,;  citation_journal_title=J. Bacteriol;  citation_year=2012;  citation_volume=194;  citation_pages=6970-1; " /><meta name="citation_reference" content="citation_title=Distribution of γ-hexachlorocyclohexane-degrading genes on three replicons in Sphingobium japonicum UT26,;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2006;  citation_volume=256;  citation_pages=112-8; " /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison,;  citation_journal_title=BMC bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=376" /><meta name="citation_reference" content="citation_title=Basic local alignment search tool,;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=772-80; " /><meta name="citation_reference" content="citation_title=WWW-query: an on-line retrieval system for biological sequence banks,;  citation_journal_title=Biochimie;  citation_year=1996;  citation_volume=78;  citation_pages=364-9; " /><meta name="citation_reference" content="citation_title=Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance,;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=413-24; " /><meta name="citation_reference" content="citation_title=High-temperature-induced transposition of insertion elements in Burkholderia multivorans ATCC 17616,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2005;  citation_volume=71;  citation_pages=1822-8; " /><meta name="citation_reference" content="citation_title=Metabolic reconstruction of aromatic compounds degradation from the genome of the amazing pollutant-degrading bacterium Cupriavidus necator JMP134;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2008;  citation_volume=32;  citation_pages=736-94; " /><meta name="citation_reference" content="citation_title=The complete multipartite genome sequence of Cupriavidus necator JMP134, a versatile pollutant degrader,;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e9729" /><meta name="citation_reference" content="citation_title=Burkholderia xenovorans LB400 harbors a multi-replicon, 9.73-Mbp genome shaped for versatility,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=15280-7; " /><meta name="citation_reference" content="citation_title=Genomic and functional analyses of the gentisate and protocatechuate ring-cleavage pathways and related 3-hydroxybenzoate and 4-hydroxybenzoate peripheral pathways in Burkholderia xenovorans LB400,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e56038" /><meta name="citation_reference" content="citation_title=The aerobic pseudomonads: a taxonomic study,;  citation_journal_title=J. Gen. Microbiol;  citation_year=1966;  citation_volume=43;  citation_pages=159-271; " /><meta name="citation_reference" content="citation_title=Pleiotropic roles of iron-responsive transcriptional regulator Fur in Burkholderia multivorans,;  citation_journal_title=Microbiology;  citation_year=2008;  citation_volume=154;  citation_pages=1763-74; " /><meta name="citation_reference" content="citation_title=Identification of Burkholderia multivorans ATCC 17616 genetic determinants for fitness in soil by using signature-tagged mutagenesis,;  citation_journal_title=Microbiology;  citation_year=2014;  citation_volume=160;  citation_pages=883-91; " /><meta name="citation_reference" content="citation_title=Genomic analysis of the potential for aromatic compounds biodegradation in Burkholderiales,;  citation_journal_title=Environ. Microbiol;  citation_year=2012;  citation_volume=14;  citation_pages=1091-117; " /><meta name="citation_reference" content="citation_title=Complete genome sequence and comparative analysis of the metabolically versatile Pseudomonas putida KT2440,;  citation_journal_title=Environ. Microbiol;  citation_year=2002;  citation_volume=4;  citation_pages=799-808; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of linR, involved in regulation of the downstream pathway for γ-hexachlorocyclohexane degradation in Sphingomonas paucimobilis UT26,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2002;  citation_volume=68;  citation_pages=1803-7; " /><meta name="citation_reference" content="citation_title=The β-ketoadipate pathway and the biology of self-identity,;  citation_journal_title=Annu. Rev. Microbiol;  citation_year=1996;  citation_volume=50;  citation_pages=553-90; " /><meta name="citation_reference" content="citation_title=The missing link: Bordetella petrii is endowed with both the metabolic versatility of environmental bacteria and virulence traits of pathogenic Bordetellae,;  citation_journal_title=BMC genomics;  citation_year=2008;  citation_volume=9;  citation_pages=449" /><meta name="citation_reference" content="citation_title=The completely sequenced plasmid pEST4011 contains a novel IncP1 backbone and a catabolic transposon harboring tfd genes for 2,4-dichlorophenoxyacetic acid degradation,;  citation_journal_title=J. Bacteriol;  citation_year=2004;  citation_volume=186;  citation_pages=7161-74; " /><meta name="citation_reference" content="citation_title=Role of IncP-1beta plasmids pWDL7::rfp and pNB8c in chloroaniline catabolism as determined by genomic and functional analyses,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2012;  citation_volume=78;  citation_pages=828-38; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Sphingobium sp. strain HDIPO4, an avid degrader of hexachlorocyclohexane,;  citation_journal_title=Genome Announc;  citation_year=2013;  citation_volume=1;  citation_pages=e00749-13; " /><meta name="citation_reference" content="citation_title=Genetic exchange between bacteria in the environment,;  citation_journal_title=Plasmid;  citation_year=1999;  citation_volume=42;  citation_pages=73-91; " /><meta name="citation_reference" content="citation_publisher=Springer, the Netherlands;  citation_title=Management of Microbial Resources in the Environment;  citation_year=2013;  citation_pages=167-214; " /><meta name="citation_reference" content="citation_title=Conjugal transfer of polychlorinated biphenyl/biphenyl degradation genes in Acidovorax sp. strain KKS102, which are located on an integrative and conjugative element,;  citation_journal_title=J. Bacteriol;  citation_year=2012;  citation_volume=194;  citation_pages=4237-48; " /><meta name="citation_reference" content="citation_title=Novel LinA-type 3 δ-hexachlorocyclohexane dehydrochlorinase,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2015;  citation_volume=81;  citation_pages=7553-9; " /><meta name="citation_reference" content="citation_title=Stepwise enhancement of catalytic performance of haloalkane dehalogenase LinB towards β-hexachlorocyclohexane,;  citation_journal_title=AMB Express;  citation_year=2014;  citation_volume=4;  citation_pages=72" /><meta name="citation_reference" content="citation_title=Kinetic and sequence-structure-function analysis of LinB enzyme variants with β- and δ-hexachlorocyclohexane,;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e103632" /><meta name="citation_reference" content="citation_title=Properties and biotechnological applications of natural and engineered haloalkane dehalogenases,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2015;  citation_volume=99;  citation_pages=9865-81; " /><meta name="citation_reference" content="citation_title=Replication intermediate analysis confirms that chromosomal replication origin initiates from an unusual intergenic region in Caulobacter crescentus,;  citation_journal_title=Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=4441-51; " /><meta name="citation_reference" content="citation_title=The Sinorhizobium meliloti chromosomal origin of replication,;  citation_journal_title=Microbiology;  citation_year=2006;  citation_volume=152;  citation_pages=443-55; " /><meta name="citation_reference" content="citation_title=Interaction of the initiator protein DnaA of Escherichia coli with its DNA target,;  citation_journal_title=J. Biol. Chem;  citation_year=1995;  citation_volume=270;  citation_pages=17622-6; " /><meta name="citation_reference" content="citation_title=The ABCs of plasmid replication and segregation,;  citation_journal_title=Nat. Rev. Microbiol;  citation_year=2012;  citation_volume=10;  citation_pages=755-65; " /><meta name="citation_reference" content="citation_title=The repABC plasmid family,;  citation_journal_title=Plasmid;  citation_year=2008;  citation_volume=60;  citation_pages=19-37; " /><meta name="citation_reference" content="citation_title=Replication and control of circular bacterial plasmids,;  citation_journal_title=Microbiol. Mol. Biol. Rev;  citation_year=1998;  citation_volume=62;  citation_pages=434-64; " /><meta name="citation_reference" content="citation_title=Control of plasmid DNA replication by iterons: no longer paradoxical,;  citation_journal_title=Mol. Microbiol;  citation_year=2000;  citation_volume=37;  citation_pages=467-76; " /><meta name="citation_reference" content="citation_title=Mosaic plasmids and mosaic replicons: evolutionary lessons from the analysis of genetic diversity in IncFII-related replicons,;  citation_journal_title=Microbiology;  citation_year=2000;  citation_volume=146;  citation_pages=2267-75; " /><meta name="citation_reference" content="citation_title=Towards an integrated model of bacterial conjugation,;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2015;  citation_volume=39;  citation_pages=81-95; " /><meta name="citation_reference" content="citation_title=Complete nucleotide sequence of an exogenously isolated plasmid, pLB1, involved in γ-hexachlorocyclohexane degradation,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2006;  citation_volume=72;  citation_pages=6923-33; " /><meta name="citation_reference" content="citation_title=Analysis of the sequence and gene products of the transfer region of the F sex factor,;  citation_journal_title=Microbiol. Rev;  citation_year=1994;  citation_volume=58;  citation_pages=162-210; " /><meta name="citation_reference" content="citation_title=16S rDNA phylogeny and distribution of lin genes in novel hexachlorocyclohexane-degrading Sphingomonas strains,;  citation_journal_title=Environ. Microbiol;  citation_year=2005;  citation_volume=7;  citation_pages=1329-38; " /><meta name="citation_reference" content="citation_title=Insertion sequence-based cassette PCR: cultivation-independent isolation of γ-hexachlorocyclohexane-degrading genes from soil DNA,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2008;  citation_volume=79;  citation_pages=627-32; " /><meta name="citation_reference" content="citation_title=Insertion sequences;  citation_journal_title=Microbiol. Mol. Biol. Rev;  citation_year=1998;  citation_volume=62;  citation_pages=725-74; " /><meta name="citation_reference" content="citation_title=Distribution of centromere-like parS sites in bacteria: insights from comparative genomics,;  citation_journal_title=J. Bacteriol;  citation_year=2007;  citation_volume=189;  citation_pages=8693-703; " /><meta name="citation_reference" content="citation_title=De novo sequencing and assembly of the whole genome of Novosphingobium sp. strain PP1Y;  citation_journal_title=J. Bacteriol;  citation_year=2011;  citation_volume=193;  citation_pages=4296" /><meta name="citation_reference" content="citation_title=Complete genome sequence of Sphingobium sp. strain SYK-6, a degrader of lignin-derived biaryls and monoaryls,;  citation_journal_title=J. Bacteriol;  citation_year=2012;  citation_volume=194;  citation_pages=534-5; " /><meta name="citation_reference" content="citation_title=The Sphingomonas plasmid pCAR3 is involved in complete mineralization of carbazole,;  citation_journal_title=J. Bacteriol;  citation_year=2007;  citation_volume=189;  citation_pages=2007-20; " /><meta name="citation_reference" content="citation_title=Genome sequence of the dioxin-mineralizing bacterium Sphingomonas wittichii RW1,;  citation_journal_title=J. Bacteriol;  citation_year=2010;  citation_volume=192;  citation_pages=6101-2; " /><meta name="citation_reference" content="citation_title=Genome sequence of benzo(a)pyrene-degrading bacterium Novosphingobium pentaromativorans US6-1,;  citation_journal_title=J. Bacteriol;  citation_year=2012;  citation_volume=194;  citation_pages=907" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/6/581/8325722/dsw041.pdf" /><meta name="description" content="γ-Hexachlorocyclohexane (γ-HCH) is a recalcitrant man-made chlorinated pesticide. Here, the complete genome sequences of four γ-HCH-degrading sphingomonad strai" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/6/581/2647451" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/6/581/2647451" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/6/581/2647451");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw041';
    </script>
<title>
	Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2647451" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BajKATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBNaWNoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9oaGF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaWthd2Fkb3JpLCBZdWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpc2hpZGEsIEtvdWhlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBUYWt1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YXN1bWksIFRvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgSGlyb21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9odHN1Ym8sIFlvc2hpeXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VkYSwgTWFzYXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYXRhLCBZdWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcGFyaXNvbiBvZiB0aGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlcyBvZiBmb3VyIM6zLWhleGFjaGxvcm9jeWNsb2hleGFuZS1kZWdyYWRpbmcgYmFjdGVyaWFsIHN0cmFpbnM6IGluc2lnaHRzIGludG8gdGhlIGV2b2x1dGlvbiBvZiBiYWN0ZXJpYSBhYmxlIHRvIGRlZ3JhZGUgYSByZWNhbGNpdHJhbnQgbWFuLW1hZGUgcGVzdGljaWRlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1OTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVybmF0aW9uYWwgUGVsbGV0IFdhdGNoOiBnbG9iYWwgbW9uaXRvcmluZyBvZiBwZXJzaXN0ZW50IG9yZ2FuaWMgcG9sbHV0YW50cyAoUE9QcykgaW4gY29hc3RhbCB3YXRlcnMuIDEuIEluaXRpYWwgcGhhc2UgZGF0YSBvbiBQQ0JzLCBERFRzLCBhbmQgSENIcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1hci4gUG9sbHV0LiBCdWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTQzNy00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gb3ZlcnZpZXcgb24gdGhlIGFjY3VtdWxhdGlvbiwgZGlzdHJpYnV0aW9uLCB0cmFuc2Zvcm1hdGlvbnMsIHRveGljaXR5IGFuZCBhbmFseXRpY2FsIG1ldGhvZHMgZm9yIHRoZSBtb25pdG9yaW5nIG9mIHBlcnNpc3RlbnQgb3JnYW5pYyBwb2xsdXRhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGFsYW50YTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTE1ODctOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yZ2Fub2NobG9yaW5lIHBlc3RpY2lkZXMgaW4gc29pbCwgbW9zcyBhbmQgdHJlZS1iYXJrIGZyb20gTm9ydGgtRWFzdGVybiBSb21hbmlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBUb3RhbCBFbnZpcm9uOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1NuKAkzQ1NzsgIGNpdGF0aW9uX3BhZ2VzPTMxNy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGRlZ3JhZGF0aW9uIG9mIHhlbm9iaW90aWMgY29tcG91bmRzOiBldm9sdXRpb24gYW5kIGRpc3RyaWJ1dGlvbiBvZiBub3ZlbCBlbnp5bWUgYWN0aXZpdGllcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTg2OC04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGVmZmljaWVudCBwYXRod2F5cyBmb3IgZGVncmFkYXRpb24gb2YgYW50aHJvcG9nZW5pYyBjaGVtaWNhbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmUgQ2hlbS4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NTU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2hhcmFjdGVyaXN0aWNzIG9mIHhlbm9iaW90aWMtZGVncmFkaW5nIHNwaGluZ29tb25hZHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz03OTMtODExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9jaGVtaXN0cnkgb2YgbWljcm9iaWFsIGRlZ3JhZGF0aW9uIG9mIGhleGFjaGxvcm9jeWNsb2hleGFuZSBhbmQgcHJvc3BlY3RzIGZvciBiaW9yZW1lZGlhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gTW9sLiBCaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz01OC04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVncmFkYXRpdmUgcGxhc21pZHMgZnJvbSBzcGhpbmdvbW9uYWRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzUwOyAgY2l0YXRpb25fcGFnZXM9OS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFTTSBQcmVzcywgV2FzaGluZ3RvbiwgREM7ICBjaXRhdGlvbl90aXRsZT1NYW51YWwgb2YgRW52aXJvbm1lbnRhbCBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIG1vYmlsZSBnZW5ldGljIGVsZW1lbnRzIGluIGJhY3RlcmlhbCBhZGFwdGF0aW9uIHRvIHhlbm9iaW90aWMgb3JnYW5pYyBjb21wb3VuZHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjYyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB3aG9sZSBnZW5vbWUgc2VxdWVuY2Ugb2YgU3BoaW5nb2JpdW0gY2hsb3JvcGhlbm9saWN1bSBMLTE6IGluc2lnaHRzIGludG8gdGhlIGV2b2x1dGlvbiBvZiB0aGUgcGVudGFjaGxvcm9waGVub2wgZGVncmFkYXRpb24gcGF0aHdheSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xODQtOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBhdHJhemluZS1kZWdyYWRpbmcgY2FwYWJpbGl0aWVzIGluIHRoZSBlbnZpcm9ubWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTExNzUtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFib2xpYyBjaGFyYWN0ZXJpc3RpY3Mgb2YgdGhlIHNwZWNpZXMgVmFyaW92b3JheCBwYXJhZG94dXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz01NDEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2RlZ3JhZGF0aW9uIG9mIGhleGFjaGxvcm9jeWNsb2hleGFuZSAoSENIKSBieSBtaWNyb29yZ2FuaXNtcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2RlZ3JhZGF0aW9uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MzYzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXhhY2hsb3JvY3ljbG9oZXhhbmUgKEhDSCkgYXMgbmV3IFN0b2NraG9sbSBDb252ZW50aW9uIFBPUHMtYSBnbG9iYWwgcGVyc3BlY3RpdmUgb24gdGhlIG1hbmFnZW1lbnQgb2YgTGluZGFuZSBhbmQgaXRzIHdhc3RlIGlzb21lcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIFNjaS4gUG9sbHV0LiBSZXMuIEludDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE1Mi02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWVyb2JpYyBkZWdyYWRhdGlvbiBvZiBsaW5kYW5lICjOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUpIGluIGJhY3RlcmlhIGFuZCBpdHMgYmlvY2hlbWljYWwgYW5kIG1vbGVjdWxhciBiYXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTc0MS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSByZXByZXNlbnRhdGl2ZSDOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUtZGVncmFkaW5nIGJhY3Rlcml1bSBTcGhpbmdvYml1bSBqYXBvbmljdW0gVVQyNiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz01ODUyLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgb3JnYW5pemF0aW9uIGFuZCBnZW5vbWljIHN0cnVjdHVyYWwgcmVhcnJhbmdlbWVudHMgb2YgU3BoaW5nb2JpdW0gamFwb25pY3VtIFVUMjYsIGFuIGFyY2hldHlwYWwgzrMtaGV4YWNobG9yb2N5Y2xvaGV4YW5lLWRlZ3JhZGluZyBiYWN0ZXJpdW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fbnp5bWUgTWljcm9iLiBUZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9NDk5LTUwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyIFZlcmxhZywgVG9reW87ICBjaXRhdGlvbl90aXRsZT1CaW9kZWdyYWRhdGl2ZSBCYWN0ZXJpYTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTE5LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljIGFuYWx5c2lzIG9mIG5pbmUgU3BoaW5nb2JpdW0gc3RyYWluczogaW5zaWdodHMgaW50byB0aGVpciBldm9sdXRpb24gYW5kIGhleGFjaGxvcm9jeWNsb2hleGFuZSAoSENIKSBkZWdyYWRhdGlvbiBwYXRod2F5cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBnZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNpZ2h0cyBpbnRvIG9uZ29pbmcgZXZvbHV0aW9uIG9mIHRoZSBoZXhhY2hsb3JvY3ljbG9oZXhhbmUgY2F0YWJvbGljIHBhdGh3YXkgZnJvbSBjb21wYXJhdGl2ZSBnZW5vbWljcyBvZiB0ZW4gU3BoaW5nb21vbmFkYWNlYWUgc3RyYWlucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMDgxLTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbGluIGdlbmVzIGZvciDOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUgZGVncmFkYXRpb24gaW4gU3BoaW5nb21vbmFzIHNwLiBNTS0xIHByb3ZlZCB0byBiZSBkaXNwZXJzZWQgYWNyb3NzIG11bHRpcGxlIHBsYXNtaWRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9NDY2LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIM6zLWhleGFjaGxvcm9jeWNsb2hleGFuZS1kZWdyYWRpbmcgYmFjdGVyaXVtIFNwaGluZ29tb25hcyBzcC4gc3RyYWluIE1NLTEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQW5ub3VuYzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9ZTAwMjQ34oCTMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWdyYWRhdGlvbiBvZiDOsi1oZXhhY2hsb3JvY3ljbG9oZXhhbmUgYnkgaGFsb2Fsa2FuZSBkZWhhbG9nZW5hc2UgTGluQiBmcm9tIM6zLWhleGFjaGxvcm9jeWNsb2hleGFuZS11dGlsaXppbmcgYmFjdGVyaXVtIFNwaGluZ29iaXVtIHNwLiBNSTEyMDUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcmNoLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9MzEzLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgYSDOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUtZGVncmFkaW5nIGJhY3Rlcml1bSwgU3BoaW5nb2JpdW0gc3AuIHN0cmFpbiBNSTEyMDUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQW5ub3VuYzsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTAwMjQ2LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgYSDOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUgZGVncmFkZXIsIFNwaGluZ29iaXVtIHNwLiBzdHJhaW4gVEtTLCBpc29sYXRlZCBmcm9tIGEgzrMtaGV4YWNobG9yb2N5Y2xvaGV4YW5lLWRlZ3JhZGluZyBtaWNyb2JpYWwgY29tbXVuaXR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEFubm91bmM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWUwMDI0N+KAkzE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1lbmRhdGlvbiBvZiB0aGUgZ2VudXMgU3BoaW5nb21vbmFzIFlhYnV1Y2hpIGV0IGFsLiAxOTkwIGFuZCBqdW5pb3Igb2JqZWN0aXZlIHN5bm9ueW15IG9mIHRoZSBzcGVjaWVzIG9mIHRocmVlIGdlbmVyYSwgU3BoaW5nb2JpdW0sIE5vdm9zcGhpbmdvYml1bSBhbmQgU3BoaW5nb3B5eGlzLCBpbiBjb25qdW5jdGlvbiB3aXRoIEJsYXN0b21vbmFzIHVyc2luY29sYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gU3lzdC4gRXZvbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MTQ4NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciBMYWJvcmF0b3J5LCBOWTsgIGNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjbG9uaW5nOiBhIExhYm9yYXRvcnkgTWFudWFsOyAgY2l0YXRpb25feWVhcj0xOTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBnZW5lcyBlbmNvZGluZyBhIHB1dGF0aXZlIEFCQy10eXBlIHRyYW5zcG9ydGVyIGVzc2VudGlhbCBmb3IgdXRpbGl6YXRpb24gb2YgzrMtaGV4YWNobG9yb2N5Y2xvaGV4YW5lIGluIFNwaGluZ29iaXVtIGphcG9uaWN1bSBVVDI2LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE4OTsgIGNpdGF0aW9uX3BhZ2VzPTM3MTItMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBkaWZmZXJlbnQgdHlwZXMgb2YgZGVoYWxvZ2VuYXNlcywgTGluQSBhbmQgTGluQiwgaW52b2x2ZWQgaW4gzrMtaGV4YWNobG9yb2N5Y2xvaGV4YW5lIGRlZ3JhZGF0aW9uIGluIFNwaGluZ29tb25hcyBwYXVjaW1vYmlsaXMgVVQyNiBhcmUgbG9jYWxpemVkIGluIHRoZSBwZXJpcGxhc21pYyBzcGFjZSB3aXRob3V0IG1vbGVjdWxhciBwcm9jZXNzaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTgxOyAgY2l0YXRpb25fcGFnZXM9NTQwOS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm91ciBuZXcgZGVyaXZhdGl2ZXMgb2YgdGhlIGJyb2FkLWhvc3QtcmFuZ2UgY2xvbmluZyB2ZWN0b3IgcEJCUjFNQ1MsIGNhcnJ5aW5nIGRpZmZlcmVudCBhbnRpYmlvdGljLXJlc2lzdGFuY2UgY2Fzc2V0dGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0xNzUtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBicm9hZC1ob3N0LXJhbmdlIEZscC1GUlQgcmVjb21iaW5hdGlvbiBzeXN0ZW0gZm9yIHNpdGUtc3BlY2lmaWMgZXhjaXNpb24gb2YgY2hyb21vc29tYWxseS1sb2NhdGVkIEROQSBzZXF1ZW5jZXM6IGFwcGxpY2F0aW9uIGZvciBpc29sYXRpb24gb2YgdW5tYXJrZWQgUHNldWRvbW9uYXMgYWVydWdpbm9zYSBtdXRhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMTI7ICBjaXRhdGlvbl9wYWdlcz03Ny04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciwgTlk7ICBjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgQ2xvbmluZzogQSBMYWJvcmF0b3J5IE1hbnVhbDsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgZ2VuZXMgaW52b2x2ZWQgaW4gdGhlIGRvd25zdHJlYW0gZGVncmFkYXRpb24gcGF0aHdheSBvZiDOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUgaW4gU3BoaW5nb21vbmFzIHBhdWNpbW9iaWxpcyBVVDI2LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4NzsgIGNpdGF0aW9uX3BhZ2VzPTg0Ny01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIEFjaWRvdm9yYXggc3AuIHN0cmFpbiBLS1MxMDIsIGEgcG9seWNobG9yaW5hdGVkLWJpcGhlbnlsIGRlZ3JhZGVyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5NDsgIGNpdGF0aW9uX3BhZ2VzPTY5NzAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdHJpYnV0aW9uIG9mIM6zLWhleGFjaGxvcm9jeWNsb2hleGFuZS1kZWdyYWRpbmcgZ2VuZXMgb24gdGhyZWUgcmVwbGljb25zIGluIFNwaGluZ29iaXVtIGphcG9uaWN1bSBVVDI2LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjU2OyAgY2l0YXRpb25fcGFnZXM9MTEyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZU1hdGNoZXI6IGEgZ3JhcGhpY2FsIHVzZXIgaW50ZXJmYWNlIGZvciBETkEgc2VxdWVuY2UgY29tcGFyaXNvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBiaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9Mzc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFGRlQgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHNvZnR3YXJlIHZlcnNpb24gNzogaW1wcm92ZW1lbnRzIGluIHBlcmZvcm1hbmNlIGFuZCB1c2FiaWxpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz03NzItODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdXVy1xdWVyeTogYW4gb24tbGluZSByZXRyaWV2YWwgc3lzdGVtIGZvciBiaW9sb2dpY2FsIHNlcXVlbmNlIGJhbmtzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hpbWllOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9MzY0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRpbWUtc2VyaWVzIG1ldGFnZW5vbWljIGFuYWx5c2lzIHJldmVhbHMgcm9idXN0bmVzcyBvZiBzb2lsIG1pY3JvYmlvbWUgYWdhaW5zdCBjaGVtaWNhbCBkaXN0dXJiYW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00MTMtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGVtcGVyYXR1cmUtaW5kdWNlZCB0cmFuc3Bvc2l0aW9uIG9mIGluc2VydGlvbiBlbGVtZW50cyBpbiBCdXJraG9sZGVyaWEgbXVsdGl2b3JhbnMgQVRDQyAxNzYxNiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTE4MjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWJvbGljIHJlY29uc3RydWN0aW9uIG9mIGFyb21hdGljIGNvbXBvdW5kcyBkZWdyYWRhdGlvbiBmcm9tIHRoZSBnZW5vbWUgb2YgdGhlIGFtYXppbmcgcG9sbHV0YW50LWRlZ3JhZGluZyBiYWN0ZXJpdW0gQ3VwcmlhdmlkdXMgbmVjYXRvciBKTVAxMzQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz03MzYtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBtdWx0aXBhcnRpdGUgZ2Vub21lIHNlcXVlbmNlIG9mIEN1cHJpYXZpZHVzIG5lY2F0b3IgSk1QMTM0LCBhIHZlcnNhdGlsZSBwb2xsdXRhbnQgZGVncmFkZXIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTk3MjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CdXJraG9sZGVyaWEgeGVub3ZvcmFucyBMQjQwMCBoYXJib3JzIGEgbXVsdGktcmVwbGljb24sIDkuNzMtTWJwIGdlbm9tZSBzaGFwZWQgZm9yIHZlcnNhdGlsaXR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTUyODAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmQgZnVuY3Rpb25hbCBhbmFseXNlcyBvZiB0aGUgZ2VudGlzYXRlIGFuZCBwcm90b2NhdGVjaHVhdGUgcmluZy1jbGVhdmFnZSBwYXRod2F5cyBhbmQgcmVsYXRlZCAzLWh5ZHJveHliZW56b2F0ZSBhbmQgNC1oeWRyb3h5YmVuem9hdGUgcGVyaXBoZXJhbCBwYXRod2F5cyBpbiBCdXJraG9sZGVyaWEgeGVub3ZvcmFucyBMQjQwMCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTYwMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYWVyb2JpYyBwc2V1ZG9tb25hZHM6IGEgdGF4b25vbWljIHN0dWR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gR2VuLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5NjY7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xNTktMjcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGVpb3Ryb3BpYyByb2xlcyBvZiBpcm9uLXJlc3BvbnNpdmUgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvciBGdXIgaW4gQnVya2hvbGRlcmlhIG11bHRpdm9yYW5zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE1NDsgIGNpdGF0aW9uX3BhZ2VzPTE3NjMtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIEJ1cmtob2xkZXJpYSBtdWx0aXZvcmFucyBBVENDIDE3NjE2IGdlbmV0aWMgZGV0ZXJtaW5hbnRzIGZvciBmaXRuZXNzIGluIHNvaWwgYnkgdXNpbmcgc2lnbmF0dXJlLXRhZ2dlZCBtdXRhZ2VuZXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjA7ICBjaXRhdGlvbl9wYWdlcz04ODMtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgYW5hbHlzaXMgb2YgdGhlIHBvdGVudGlhbCBmb3IgYXJvbWF0aWMgY29tcG91bmRzIGJpb2RlZ3JhZGF0aW9uIGluIEJ1cmtob2xkZXJpYWxlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEwOTEtMTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2UgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBtZXRhYm9saWNhbGx5IHZlcnNhdGlsZSBQc2V1ZG9tb25hcyBwdXRpZGEgS1QyNDQwLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RW52aXJvbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz03OTktODA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGxpblIsIGludm9sdmVkIGluIHJlZ3VsYXRpb24gb2YgdGhlIGRvd25zdHJlYW0gcGF0aHdheSBmb3IgzrMtaGV4YWNobG9yb2N5Y2xvaGV4YW5lIGRlZ3JhZGF0aW9uIGluIFNwaGluZ29tb25hcyBwYXVjaW1vYmlsaXMgVVQyNiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTE4MDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIM6yLWtldG9hZGlwYXRlIHBhdGh3YXkgYW5kIHRoZSBiaW9sb2d5IG9mIHNlbGYtaWRlbnRpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTU1My05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1pc3NpbmcgbGluazogQm9yZGV0ZWxsYSBwZXRyaWkgaXMgZW5kb3dlZCB3aXRoIGJvdGggdGhlIG1ldGFib2xpYyB2ZXJzYXRpbGl0eSBvZiBlbnZpcm9ubWVudGFsIGJhY3RlcmlhIGFuZCB2aXJ1bGVuY2UgdHJhaXRzIG9mIHBhdGhvZ2VuaWMgQm9yZGV0ZWxsYWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgZ2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQ0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZWx5IHNlcXVlbmNlZCBwbGFzbWlkIHBFU1Q0MDExIGNvbnRhaW5zIGEgbm92ZWwgSW5jUDEgYmFja2JvbmUgYW5kIGEgY2F0YWJvbGljIHRyYW5zcG9zb24gaGFyYm9yaW5nIHRmZCBnZW5lcyBmb3IgMiw0LWRpY2hsb3JvcGhlbm94eWFjZXRpYyBhY2lkIGRlZ3JhZGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTcxNjEtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgSW5jUC0xYmV0YSBwbGFzbWlkcyBwV0RMNzo6cmZwIGFuZCBwTkI4YyBpbiBjaGxvcm9hbmlsaW5lIGNhdGFib2xpc20gYXMgZGV0ZXJtaW5lZCBieSBnZW5vbWljIGFuZCBmdW5jdGlvbmFsIGFuYWx5c2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9ODI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgU3BoaW5nb2JpdW0gc3AuIHN0cmFpbiBIRElQTzQsIGFuIGF2aWQgZGVncmFkZXIgb2YgaGV4YWNobG9yb2N5Y2xvaGV4YW5lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEFubm91bmM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPWUwMDc0OS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBleGNoYW5nZSBiZXR3ZWVuIGJhY3RlcmlhIGluIHRoZSBlbnZpcm9ubWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYXNtaWQ7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz03My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCB0aGUgTmV0aGVybGFuZHM7ICBjaXRhdGlvbl90aXRsZT1NYW5hZ2VtZW50IG9mIE1pY3JvYmlhbCBSZXNvdXJjZXMgaW4gdGhlIEVudmlyb25tZW50OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fcGFnZXM9MTY3LTIxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uanVnYWwgdHJhbnNmZXIgb2YgcG9seWNobG9yaW5hdGVkIGJpcGhlbnlsL2JpcGhlbnlsIGRlZ3JhZGF0aW9uIGdlbmVzIGluIEFjaWRvdm9yYXggc3AuIHN0cmFpbiBLS1MxMDIsIHdoaWNoIGFyZSBsb2NhdGVkIG9uIGFuIGludGVncmF0aXZlIGFuZCBjb25qdWdhdGl2ZSBlbGVtZW50LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5NDsgIGNpdGF0aW9uX3BhZ2VzPTQyMzctNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIExpbkEtdHlwZSAzIM60LWhleGFjaGxvcm9jeWNsb2hleGFuZSBkZWh5ZHJvY2hsb3JpbmFzZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTc1NTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RlcHdpc2UgZW5oYW5jZW1lbnQgb2YgY2F0YWx5dGljIHBlcmZvcm1hbmNlIG9mIGhhbG9hbGthbmUgZGVoYWxvZ2VuYXNlIExpbkIgdG93YXJkcyDOsi1oZXhhY2hsb3JvY3ljbG9oZXhhbmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BTUIgRXhwcmVzczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LaW5ldGljIGFuZCBzZXF1ZW5jZS1zdHJ1Y3R1cmUtZnVuY3Rpb24gYW5hbHlzaXMgb2YgTGluQiBlbnp5bWUgdmFyaWFudHMgd2l0aCDOsi0gYW5kIM60LWhleGFjaGxvcm9jeWNsb2hleGFuZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAzNjMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvcGVydGllcyBhbmQgYmlvdGVjaG5vbG9naWNhbCBhcHBsaWNhdGlvbnMgb2YgbmF0dXJhbCBhbmQgZW5naW5lZXJlZCBoYWxvYWxrYW5lIGRlaGFsb2dlbmFzZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz05ODY1LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBsaWNhdGlvbiBpbnRlcm1lZGlhdGUgYW5hbHlzaXMgY29uZmlybXMgdGhhdCBjaHJvbW9zb21hbCByZXBsaWNhdGlvbiBvcmlnaW4gaW5pdGlhdGVzIGZyb20gYW4gdW51c3VhbCBpbnRlcmdlbmljIHJlZ2lvbiBpbiBDYXVsb2JhY3RlciBjcmVzY2VudHVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz00NDQxLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU2lub3JoaXpvYml1bSBtZWxpbG90aSBjaHJvbW9zb21hbCBvcmlnaW4gb2YgcmVwbGljYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9NDQzLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGlvbiBvZiB0aGUgaW5pdGlhdG9yIHByb3RlaW4gRG5hQSBvZiBFc2NoZXJpY2hpYSBjb2xpIHdpdGggaXRzIEROQSB0YXJnZXQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTE3NjIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBQkNzIG9mIHBsYXNtaWQgcmVwbGljYXRpb24gYW5kIHNlZ3JlZ2F0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlcEFCQyBwbGFzbWlkIGZhbWlseSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYXNtaWQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0xOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwbGljYXRpb24gYW5kIGNvbnRyb2wgb2YgY2lyY3VsYXIgYmFjdGVyaWFsIHBsYXNtaWRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTQzNC02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbCBvZiBwbGFzbWlkIEROQSByZXBsaWNhdGlvbiBieSBpdGVyb25zOiBubyBsb25nZXIgcGFyYWRveGljYWwsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTQ2Ny03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9zYWljIHBsYXNtaWRzIGFuZCBtb3NhaWMgcmVwbGljb25zOiBldm9sdXRpb25hcnkgbGVzc29ucyBmcm9tIHRoZSBhbmFseXNpcyBvZiBnZW5ldGljIGRpdmVyc2l0eSBpbiBJbmNGSUktcmVsYXRlZCByZXBsaWNvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTQ2OyAgY2l0YXRpb25fcGFnZXM9MjI2Ny03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkcyBhbiBpbnRlZ3JhdGVkIG1vZGVsIG9mIGJhY3RlcmlhbCBjb25qdWdhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz04MS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgbnVjbGVvdGlkZSBzZXF1ZW5jZSBvZiBhbiBleG9nZW5vdXNseSBpc29sYXRlZCBwbGFzbWlkLCBwTEIxLCBpbnZvbHZlZCBpbiDOsy1oZXhhY2hsb3JvY3ljbG9oZXhhbmUgZGVncmFkYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz02OTIzLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgc2VxdWVuY2UgYW5kIGdlbmUgcHJvZHVjdHMgb2YgdGhlIHRyYW5zZmVyIHJlZ2lvbiBvZiB0aGUgRiBzZXggZmFjdG9yLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0xNjItMjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT0xNlMgckROQSBwaHlsb2dlbnkgYW5kIGRpc3RyaWJ1dGlvbiBvZiBsaW4gZ2VuZXMgaW4gbm92ZWwgaGV4YWNobG9yb2N5Y2xvaGV4YW5lLWRlZ3JhZGluZyBTcGhpbmdvbW9uYXMgc3RyYWlucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTMyOS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zZXJ0aW9uIHNlcXVlbmNlLWJhc2VkIGNhc3NldHRlIFBDUjogY3VsdGl2YXRpb24taW5kZXBlbmRlbnQgaXNvbGF0aW9uIG9mIM6zLWhleGFjaGxvcm9jeWNsb2hleGFuZS1kZWdyYWRpbmcgZ2VuZXMgZnJvbSBzb2lsIEROQSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT03OTsgIGNpdGF0aW9uX3BhZ2VzPTYyNy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zZXJ0aW9uIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTcyNS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdHJpYnV0aW9uIG9mIGNlbnRyb21lcmUtbGlrZSBwYXJTIHNpdGVzIGluIGJhY3RlcmlhOiBpbnNpZ2h0cyBmcm9tIGNvbXBhcmF0aXZlIGdlbm9taWNzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE4OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OTMtNzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIHNlcXVlbmNpbmcgYW5kIGFzc2VtYmx5IG9mIHRoZSB3aG9sZSBnZW5vbWUgb2YgTm92b3NwaGluZ29iaXVtIHNwLiBzdHJhaW4gUFAxWTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTQyOTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgU3BoaW5nb2JpdW0gc3AuIHN0cmFpbiBTWUstNiwgYSBkZWdyYWRlciBvZiBsaWduaW4tZGVyaXZlZCBiaWFyeWxzIGFuZCBtb25vYXJ5bHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9NTM0LTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTcGhpbmdvbW9uYXMgcGxhc21pZCBwQ0FSMyBpcyBpbnZvbHZlZCBpbiBjb21wbGV0ZSBtaW5lcmFsaXphdGlvbiBvZiBjYXJiYXpvbGUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTg5OyAgY2l0YXRpb25fcGFnZXM9MjAwNy0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBkaW94aW4tbWluZXJhbGl6aW5nIGJhY3Rlcml1bSBTcGhpbmdvbW9uYXMgd2l0dGljaGlpIFJXMSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz02MTAxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBiZW56byhhKXB5cmVuZS1kZWdyYWRpbmcgYmFjdGVyaXVtIE5vdm9zcGhpbmdvYml1bSBwZW50YXJvbWF0aXZvcmFucyBVUzYtMSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTQ7ICBjaXRhdGlvbl9wYWdlcz05MDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvNi81ODEvODMyNTcyMi9kc3cwNDEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSLOsy1IZXhhY2hsb3JvY3ljbG9oZXhhbmUgKM6zLUhDSCkgaXMgYSByZWNhbGNpdHJhbnQgbWFuLW1hZGUgY2hsb3JpbmF0ZWQgcGVzdGljaWRlLiBIZXJlLCB0aGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlcyBvZiBmb3VyIM6zLUhDSC1kZWdyYWRpbmcgc3BoaW5nb21vbmFkIHN0cmFpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNjQ3NDUxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZEG7Jkry+lpsYyhALXjVt5avYQY9" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f581%2f2647451" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f581%2f2647451" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=89339" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&amp;Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/6">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="48005553">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005553">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005556">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005556">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005569">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005569">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005594">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005594">4. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005596">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005596">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005598">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005598">Accession number</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005600">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005600">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005602">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005602">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005604">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005604">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/6/571/2647450">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michiro Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michiro+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michiro Tabata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Ohhata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Ohhata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohhata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohhata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Ohhata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Nikawadori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Nikawadori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nikawadori Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nikawadori Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Nikawadori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kouhei Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kouhei+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kouhei Kishida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuya Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuya+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takuya Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toru Kawasumi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toru+Kawasumi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawasumi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawasumi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toru Kawasumi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiromi Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiromi+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiromi Kato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Ohtsubo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Ohtsubo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtsubo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtsubo Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiyuki Ohtsubo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masataka Tsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masataka+Tsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuda M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masataka Tsuda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Nagata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw041-cor1" data-open="dsw041-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +81 22-217-5682. Fax: +81 22-217-5682. E-mail: <a href="mailto:aynaga@ige.tohoku.ac.jp" target="_blank">aynaga@ige.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Nagata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagata Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuji Nagata</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw041-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 581–599, <a href='https://doi.org/10.1093/dnares/dsw041'>https://doi.org/10.1093/dnares/dsw041</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 August 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 April 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 July 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2647451" href="/dnaresearch/article-pdf/23/6/581/8325722/dsw041.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Michiro Tabata, Satoshi Ohhata, Yuki Nikawadori, Kouhei Kishida, Takuya Sato, Toru Kawasumi, Hiromi Kato, Yoshiyuki Ohtsubo, Masataka Tsuda, Yuji Nagata;  Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide, <em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 581–599, <a href='https://doi.org/10.1093/dnares/dsw041'>https://doi.org/10.1093/dnares/dsw041</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647451?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparison%20of%20the%20complete%20genome%20sequences%20of%20four%20%CE%B3-hexachlorocyclohexane-degrading%20bacterial%20strains%3A%20insights%20into%20the%20evolution%20of%20bacteria%20able%20to%20degrade%20a%20recalcitrant%20man-made%20pesticide&amp;publicationDate=2016-08-30&amp;volumeNum=23&amp;issueNum=6&amp;author=Tabata%2C%20Michiro%3B%20Ohhata%2C%20Satoshi&amp;startPage=581&amp;endPage=599&amp;contentId=10.1093%2Fdnares%2Fdsw041&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f581%2f2647451" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>γ-Hexachlorocyclohexane (γ-HCH) is a recalcitrant man-made chlorinated pesticide. Here, the complete genome sequences of four γ-HCH-degrading sphingomonad strains, which are most unlikely to have been derived from one ancestral γ-HCH degrader, were compared. Together with several experimental data, we showed that (i) all the four strains carry almost identical <em>linA</em> to <em>linE</em> genes for the conversion of γ-HCH to maleylacetate (designated “specific” <em>lin</em> genes), (ii) considerably different genes are used for the metabolism of maleylacetate in one of the four strains, and (iii) the <em>linKLMN</em> genes for the putative ABC transporter necessary for γ-HCH utilization exhibit structural divergence, which reflects the phylogenetic relationship of their hosts. Replicon organization and location of the <em>lin</em> genes in the four genomes are significantly different with one another, and that most of the specific <em>lin</em> genes are located on multiple sphingomonad-unique plasmids. Copies of IS<em>6100</em>, the most abundant insertion sequence in the four strains, are often located in close proximity to the specific <em>lin</em> genes. Analysis of the footprints of target duplication upon IS<em>6100</em> transposition and the experimental detection of IS<em>6100</em> transposition strongly suggested that the IS<em>6100</em> transposition has caused dynamic genome rearrangements and the diversification of <em>lin</em>-flanking regions in the four strains.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="xenobiotics">xenobiotics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="sphingomonads">sphingomonads</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mobile genetic elements&quot;">mobile genetic elements</a></div>                            <h2 scrollto-destination=48005553 id="48005553" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<div class="block-child-p">In the early 20th century, rapid development in the chemical industry led to the production and wide use of numerous anthropogenic chemicals. Because they are often recalcitrant in the environment and toxic to humans and ecosystems, they have caused serious environmental problems.<a reveal-id="dsw041-B1 dsw041-B2 dsw041-B3" data-open="dsw041-B1 dsw041-B2 dsw041-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> Many bacterial strains capable of degrading man-made xenobiotic compounds have been isolated and characterized.<a reveal-id="dsw041-B4 dsw041-B5 dsw041-B6 dsw041-B7 dsw041-B8 dsw041-B9" data-open="dsw041-B4 dsw041-B5 dsw041-B6 dsw041-B7 dsw041-B8 dsw041-B9" class="link link-ref link-reveal xref-bibr"><sup>4–9</sup></a> Such strains are thought to have evolved to degrade xenobiotics within relatively short periods. However, with the exception of a few speculative examples, the evolutionary processes of these bacterial strains remain largely unknown.<a reveal-id="dsw041-B5" data-open="dsw041-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw041-B10 dsw041-B11 dsw041-B12 dsw041-B13" data-open="dsw041-B10 dsw041-B11 dsw041-B12 dsw041-B13" class="link link-ref link-reveal xref-bibr"><sup>10–13</sup></a> γ-Hexachlorocyclohexane (γ-HCH; also known as γ-BHC or lindane) is a completely man-made chlorinated pesticide that has caused serious environmental problems due to its toxicity and long persistence in upland soils.<a reveal-id="dsw041-B7" data-open="dsw041-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw041-B14" data-open="dsw041-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw041-B15" data-open="dsw041-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Only 60 years after the first release of γ-HCH into the environment, a number of bacterial strains that aerobically degrade γ-HCH have been isolated from geographically distant locations around the world.<a reveal-id="dsw041-B7" data-open="dsw041-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> An archetypal γ-HCH-degrading strain, <em>Sphingobium japonicum</em> UT26, was isolated from an upland experimental field to which γ-HCH had been applied once a year for 12 years, and its aerobic γ-HCH degradation pathway (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) and genome organization have been intensively studied.<a reveal-id="dsw041-B16 dsw041-B17 dsw041-B18 dsw041-B19" data-open="dsw041-B16 dsw041-B17 dsw041-B18 dsw041-B19" class="link link-ref link-reveal xref-bibr"><sup>16–19</sup></a> Recently, many draft genome sequences of other HCH (including not only γ-HCH but also other HCH isomers) degraders and their related but non-HCH-degrading strains were determined, and their comparative analyses have been published.<a reveal-id="dsw041-B20" data-open="dsw041-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsw041-B21" data-open="dsw041-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> These studies provided us some important primary information on the evolution of HCH-degraders with the involvement of plasmids and insertion sequences (ISs). However, the more detailed information (e.g. which and how plasmids/ISs are involved in the evolution of HCH-degraders) remains unclear. <div data-id="dsw041-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f1p.png?Expires=2147483647&Signature=MHQQ9mAv-ZB9F6LveWJ2B6~uTLTYIuekh0Nn3II-MBgS89tZiqjXbxiUbcSjPEyw8nHVTWReQrXkf50nD1Gax5vRxluJ3Rl4q7uFsrOYIMlH~xhjPXnD5cd-Wem10ojcAaG490bN1d-eemb8pwRG7LuDlZvKz8OZyAcsRcl2T301g59Fe1gu-SIU63u9e8nPedovwWIW~IIHbc3JzjltfIZy4SpwWkPWtMAZ82KiRQ-R6-v9790Sg7Ikq7E-VogzzCJAOxLzQ2Hr7HOh-px2HZz0-UnvA-cWEFNg6us75uNY~5KIHJUs6zfFlhz6K31G0jFhmdE~IgDu8dJ-htJgKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Degradation pathway of γ-HCH in UT26. Compounds: 1, γ-HCH; 2, γ-pentachlorocyclohexene; 3, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 4, 1,2,4-trichlorobenzene; 5, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; 6, 2,5-dichlorophenol; 7, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 8, 2,5-dichlorohydroquinone; 9, chlorohydroquinone; 10, acylchloride; 11, hydroquinone; 12, γ-hydroxymuconic semialdehyde; 13, maleylacetate; 14, β-ketoadipate; 15, 3-oxoadipyl-CoA; 16, succinyl-CoA; 17, acetyl-CoA. TCA, citrate/tricarboxylic acid cycle. Note that the compounds 4 and 6 are dead-end products. Spontaneous and non-enzymatic reaction is indicated by grey arrows. The reaction marked with ‘?’ has not been identified. Compounds in parentheses have not been directly detected." path-from-xml="dsw041f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005554/dsw041f1p.tif" path-from-xml="dsw041f1p.tif" target="_blank">View large</a><a section="48005554" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f1p.png?Expires=2147483647&Signature=vUJ85XRkZydxaDynQuzANiMAYgTuTol7~JOcarPrBt3G3pKGqoaiCX6VtQvtN8PMS2bN6XGY4SpoyhuUygYVu12FWaAk6FNESS0zRFvM4NTX2EDsg~uiLtOQwTaDyMHJM2HI46-qm66rTE5Tgjbf6PDtHh0MMaooOFbEQ71EQVsGCHXTFUtYOOvh952JHAmX4CGGXNcrJYsAkh86qDu~xgZh7GPbAd7wDRwsy1i0SF3tDPrCjM~cE-b7XeKqSqEJulAZ~8m3OCqpzyK~2QFBzop~LpaJf4HnNwZpgsEbkrIpH6nRd4h10ViUd5NmYkmROdqruTGxZCrhwqmlrfkdqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005554&ar=2647451&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw041f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Degradation pathway of γ-HCH in UT26. Compounds: 1, γ-HCH; 2, γ-pentachlorocyclohexene; 3, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 4, 1,2,4-trichlorobenzene; 5, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; 6, 2,5-dichlorophenol; 7, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 8, 2,5-dichlorohydroquinone; 9, chlorohydroquinone; 10, acylchloride; 11, hydroquinone; 12, γ-hydroxymuconic semialdehyde; 13, maleylacetate; 14, β-ketoadipate; 15, 3-oxoadipyl-CoA; 16, succinyl-CoA; 17, acetyl-CoA. TCA, citrate/tricarboxylic acid cycle. Note that the compounds 4 and 6 are dead-end products. Spontaneous and non-enzymatic reaction is indicated by grey arrows. The reaction marked with ‘?’ has not been identified. Compounds in parentheses have not been directly detected.</p></div></div><div content-id="dsw041-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f1p.png?Expires=2147483647&Signature=MHQQ9mAv-ZB9F6LveWJ2B6~uTLTYIuekh0Nn3II-MBgS89tZiqjXbxiUbcSjPEyw8nHVTWReQrXkf50nD1Gax5vRxluJ3Rl4q7uFsrOYIMlH~xhjPXnD5cd-Wem10ojcAaG490bN1d-eemb8pwRG7LuDlZvKz8OZyAcsRcl2T301g59Fe1gu-SIU63u9e8nPedovwWIW~IIHbc3JzjltfIZy4SpwWkPWtMAZ82KiRQ-R6-v9790Sg7Ikq7E-VogzzCJAOxLzQ2Hr7HOh-px2HZz0-UnvA-cWEFNg6us75uNY~5KIHJUs6zfFlhz6K31G0jFhmdE~IgDu8dJ-htJgKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Degradation pathway of γ-HCH in UT26. Compounds: 1, γ-HCH; 2, γ-pentachlorocyclohexene; 3, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 4, 1,2,4-trichlorobenzene; 5, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; 6, 2,5-dichlorophenol; 7, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 8, 2,5-dichlorohydroquinone; 9, chlorohydroquinone; 10, acylchloride; 11, hydroquinone; 12, γ-hydroxymuconic semialdehyde; 13, maleylacetate; 14, β-ketoadipate; 15, 3-oxoadipyl-CoA; 16, succinyl-CoA; 17, acetyl-CoA. TCA, citrate/tricarboxylic acid cycle. Note that the compounds 4 and 6 are dead-end products. Spontaneous and non-enzymatic reaction is indicated by grey arrows. The reaction marked with ‘?’ has not been identified. Compounds in parentheses have not been directly detected." path-from-xml="dsw041f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005554/dsw041f1p.tif" path-from-xml="dsw041f1p.tif" target="_blank">View large</a><a section="48005554" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f1p.png?Expires=2147483647&Signature=vUJ85XRkZydxaDynQuzANiMAYgTuTol7~JOcarPrBt3G3pKGqoaiCX6VtQvtN8PMS2bN6XGY4SpoyhuUygYVu12FWaAk6FNESS0zRFvM4NTX2EDsg~uiLtOQwTaDyMHJM2HI46-qm66rTE5Tgjbf6PDtHh0MMaooOFbEQ71EQVsGCHXTFUtYOOvh952JHAmX4CGGXNcrJYsAkh86qDu~xgZh7GPbAd7wDRwsy1i0SF3tDPrCjM~cE-b7XeKqSqEJulAZ~8m3OCqpzyK~2QFBzop~LpaJf4HnNwZpgsEbkrIpH6nRd4h10ViUd5NmYkmROdqruTGxZCrhwqmlrfkdqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005554&ar=2647451&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw041f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Degradation pathway of γ-HCH in UT26. Compounds: 1, γ-HCH; 2, γ-pentachlorocyclohexene; 3, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 4, 1,2,4-trichlorobenzene; 5, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; 6, 2,5-dichlorophenol; 7, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 8, 2,5-dichlorohydroquinone; 9, chlorohydroquinone; 10, acylchloride; 11, hydroquinone; 12, γ-hydroxymuconic semialdehyde; 13, maleylacetate; 14, β-ketoadipate; 15, 3-oxoadipyl-CoA; 16, succinyl-CoA; 17, acetyl-CoA. TCA, citrate/tricarboxylic acid cycle. Note that the compounds 4 and 6 are dead-end products. Spontaneous and non-enzymatic reaction is indicated by grey arrows. The reaction marked with ‘?’ has not been identified. Compounds in parentheses have not been directly detected.</p></div></div></div><p>In this study, to gain further insight into the functional evolution of bacterial genomes, the complete genome sequences of three other γ-HCH-degraders, <em>Sphingomonas</em> sp. MM-1,<a reveal-id="dsw041-B22" data-open="dsw041-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw041-B23" data-open="dsw041-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><em>Sphingobium</em> sp. MI1205,<a reveal-id="dsw041-B24" data-open="dsw041-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsw041-B25" data-open="dsw041-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and <em>Sphingobium</em> sp. TKS,<a reveal-id="dsw041-B26" data-open="dsw041-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> were determined. On the basis of our comparison of the complete genome sequences of these three strains and UT26, in association with several supporting experimental data, the evolution of γ-HCH-degrading bacterial strains is discussed.</p>                            <h2 scrollto-destination=48005556 id="48005556" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=48005557 id="48005557" class="section-title" data-legacy-id=SEC2.1>2.1. Bacterial strains, plasmids, and culture conditions</h3>
<p>Strains and plasmids used in this study are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. All sphingomonad (the collective name of <em>Sphingomonas</em>, <em>Sphingobium</em>, <em>Novosphingobium</em>, <em>Sphingopyxis</em>, and their related genera) <a reveal-id="dsw041-B27" data-open="dsw041-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> strains were cultured at 30°C in 1/3LB medium or 1/10W minimal medium.<a reveal-id="dsw041-B24" data-open="dsw041-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> If needed, antibiotics were added at the following concentrations: tetracycline (Tc) 20 μg/ml, nalidixic acid (Nal) 100 μg/ml, and kanamycin (Km) 50 μg/ml for UT26; Tc 5 μg/ml, Nal 100 μg/ml, gentamycin (Gm) 2 μg/ml, and Km 50 μg/ml for MM-1; Tc 2 μg/ml, Nal 100 μg/ml, and Km 50 μg/ml for TKS; and Tc 2 μg/ml, Nal 100 μg/ml, and Km 50 μg/ml for MI1205. <em>Escherichia coli</em> DH5α for genetic manipulation was grown at 37°C in LB.<a reveal-id="dsw041-B28" data-open="dsw041-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> If needed, antibiotics were added at the following concentrations: Tc 20 μg/ml, Gm 20 μg/ml, and Km 50 μg/ml. The solid media were prepared by the addition of 1.5% agar.</p>                            <h3 scrollto-destination=48005559 id="48005559" class="section-title" data-legacy-id=SEC2.2>2.2. Construction of plasmids and strains</h3>
<p>The 3.6-kb region containing the MM-1 <em>linKbLbMbNb</em> genes was amplified by PCR using the primer set of MM_linKLMN_F and MM_linKLMN_R (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), and the amplified fragment was cloned into a broad-host-range vector, pKS13P,<a reveal-id="dsw041-B29" data-open="dsw041-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> under the <em>linA</em> constitutive promoter<a reveal-id="dsw041-B30" data-open="dsw041-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> to generate pKSR1020. The whole region of the <em>linFb</em> gene of strain TKS was amplified by PCR using the primer set of TKS_MAR_F_Hind and TKS_MAR_R_Bam (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), digested with <em>Bam</em>HI and <em>Hin</em>dIII, and cloned into the corresponding sites of pBBR1-MCS2<a reveal-id="dsw041-B31" data-open="dsw041-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> to generate pBLFb. To disrupt the genomic <em>linFb</em> gene in TKS, an internal part of <em>linFb</em> was amplified by PCR using the primer set of TKS_MAR_single_F_Eco and TKS_MAR_single_R_Hind (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), digested with <em>Eco</em>RI and <em>Hin</em>dIII, and cloned into the corresponding sites of pEX18Gm.<a reveal-id="dsw041-B32" data-open="dsw041-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> The resultant plasmid pEDLFb was introduced into TKS by electroporation to select the Gm-resistant transformants. Subsequent PCR analysis of one such transformant (TKSdLFb) confirmed that its genomic <em>linFb</em> gene was disrupted by reciprocal homologous recombination through the single crossover-mediated integration of pEDLFb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).</p>                            <h3 scrollto-destination=48005561 id="48005561" class="section-title" data-legacy-id=SEC2.3>2.3. DNA manipulations and Sanger sequencing</h3>
<p>Established methods were employed for the preparation of plasmids and genomic DNAs, their digestion with restriction endonucleases, ligation, and agarose gel electrophoresis, and the transformation of <em>E. coli</em> cells.<a reveal-id="dsw041-B28" data-open="dsw041-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsw041-B33" data-open="dsw041-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Electroporation of sphingomonad strains was performed as described previously.<a reveal-id="dsw041-B34" data-open="dsw041-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> PCR for cloning was performed with KOD-Plus DNA polymerase (TOYOBO, Osaka, Japan). The primers used are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The Sanger sequencing was performed using an ABI PRISM 3130<em>xl</em> sequencer and ABI Prism Big Dye Terminator v3.1 Kit (Applied Biosystems).</p>                            <h3 scrollto-destination=48005563 id="48005563" class="section-title" data-legacy-id=SEC2.4>2.4. Genome sequencing and annotation analyses</h3>
<p>Fragment reads of genomic DNA of MM-1, MI1205, and TKS were obtained by the Roche 454 and Illumina HiSeq 2000 sequencing systems, and were assembled using the Newbler programme (Roche). The GenoFinisher and AceFileViewer programmes were used to finish the sequencing completion.<a reveal-id="dsw041-B35" data-open="dsw041-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> More detailed information for these procedures have been published elsewhere.<a reveal-id="dsw041-B23" data-open="dsw041-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw041-B25" data-open="dsw041-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw041-B26" data-open="dsw041-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Sequencing gap regions were amplified by PCR using KOD FX (TOYOBO) or Ex Taq (TaKaRa) by using the total DNA of the respective strains as templates, and the resultant DNA fragments were sequenced using primers for PCR amplification. Pulsed-field-gel electrophoresis was also performed as described previously<a reveal-id="dsw041-B36" data-open="dsw041-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> to support the whole genome sequencing data of MI1205 and TKS (data not shown). The annotation data of the complete genome sequences were obtained by PGAAP (Prokaryotic Genome Annotation Pipeline, <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genome/annotation_prok/" target="_blank">http://www.ncbi.nlm.nih.gov/genome/annotation_prok/</a>; 22 August 2016, date last accessed), and was curated with the dedicated software bundled in the GenomeMatcher programme (<a class="link link-uri" href="http://www.ige.tohoku.ac.jp/joho/gmProject/gmhome.html" target="_blank">http://www.ige.tohoku.ac.jp/joho/gmProject/gmhome.html</a>; 22 August 2016, date last accessed)<a reveal-id="dsw041-B37" data-open="dsw041-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> as well as by consulting the MiGAP auto-annotation system (Microbial Genome Annotation Pipeline, <a class="link link-uri" href="http://www.migap.org/" target="_blank">http://www.migap.org/</a>; 22 August 2016, date last accessed).</p>                            <h3 scrollto-destination=48005565 id="48005565" class="section-title" data-legacy-id=SEC2.5>2.5. Computational analyses of sequence data</h3>
<p>The nucleotide and protein sequences were analysed using the Genetyx programme version 13–18 (Genetyx Corp., Tokyo, Japan). Homology searches were performed using the BLAST programmes available at the National Center for Biotechnology Information website (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/BLAST/" target="_blank">http://www.ncbi.nlm.nih.gov/BLAST/</a>; 22 August 2016, date last accessed) with the default parameters. Venn diagram was depicted using the result obtained by BLASTClust analysis (<a class="link link-uri" href="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/" target="_blank">ftp://ftp.ncbi.nlm.nih.gov/blast/executables/</a>; 22 August 2016, date last accessed)<a reveal-id="dsw041-B38" data-open="dsw041-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> for all predicted open reading frames (ORFs) of the four γ-HCH degraders with a parameter set ‘-p T -L .6 -b T -S 60 -a 24’. Comparative analyses of DNA sequences were performed by GenomeMatcher using BLASTN with a parameter set ‘-F F -W 21 -e 0.01’. Conserved motifs and repeat sequences were searched by GenomeMatcher. All neighbour-joining phylogenetic trees shown in this study were constructed using MAFFT programme (<a class="link link-uri" href="http://mafft.cbrc.jp/alignment/software/" target="_blank">http://mafft.cbrc.jp/alignment/software/</a>; 22 August 2016, date last accessed)<a reveal-id="dsw041-B39" data-open="dsw041-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and visualized by NJplot software (<a class="link link-uri" href="http://pbil.univ-lyon1.fr/software/njplot.html" target="_blank">http://pbil.univ-lyon1.fr/software/njplot.html</a>; 22 August 2016, date last accessed).<a reveal-id="dsw041-B40" data-open="dsw041-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Transposable elements were identified by analyses of regions containing putative transposase genes, i.e. mutual BLASTN analysis with a parameter set ‘-F F -W 21 -e 0.01’and searches for inverted and direct repeats using Dot Match mode of GenomeMatcher. For detection of candidate ORFs relevant to metabolisms of aromatic compounds, all ORFs obtained from each genome were BLASTP-searched with parameters ‘-e 1e-5 -b 5 -F F’ and threshold identity ≥50%, query and reference sequence coverage ≥50% against an in-house database for the enzymes for degradation of aromatic compounds.<a reveal-id="dsw041-B41" data-open="dsw041-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a></p>                            <h3 scrollto-destination=48005567 id="48005567" class="section-title" data-legacy-id=SEC2.6>2.6. Entrapment of transposable elements</h3>
<p>pGEN500, an entrapment plasmid vector of transposable elements,<a reveal-id="dsw041-B42" data-open="dsw041-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> was introduced into UT26, MM-1, TKS, and MI1205 by electroporation or mating using <em>E. coli</em> S17-1, and the cells carrying pGEN500 were selected on the 1/3LB agar plate containing Tc. The pGEN500-containing strains were thereafter plated on 1/3LB agar containing Tc and 10% sucrose (w/v) at 30°C. The colonies formed on the plates were analysed for their resident pGEN500 derivatives. Such derivatives with enlarged sizes were postulated to be formed by the insertion of endogenous transposable elements in the <em>sacB</em> gene on pGEN500. The insertion event in each derivative was investigated by PCR using the multiple pairs of primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>, and the insert was sequenced by the Sanger method.</p>                            <h2 scrollto-destination=48005569 id="48005569" class="section-title" data-legacy-id=SEC3>3. Results and discussion</h2>
                            <h3 scrollto-destination=48005570 id="48005570" class="section-title" data-legacy-id=SEC3.1>3.1. Complete genome sequences of three γ-HCH-degrading sphingomonad strains</h3>
<p>Most of the aerobic γ-HCH-degrading bacterial strains that have been critically analysed at the genetic level are sphingomonads that belong to <em>Alphaproteobacteria.</em><a reveal-id="dsw041-B7" data-open="dsw041-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> We have previously published an article describing our detailed analysis of the UT26 genome.<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> In this study, the complete genome sequences of three other γ-HCH-degrading strains, <em>Sphingomonas</em> sp. MM-1,<a reveal-id="dsw041-B22" data-open="dsw041-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw041-B23" data-open="dsw041-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><em>Sphingobium</em> sp. MI1205,<a reveal-id="dsw041-B24" data-open="dsw041-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsw041-B25" data-open="dsw041-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and <em>Sphingobium</em> sp. TKS,<a reveal-id="dsw041-B26" data-open="dsw041-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> were determined.</p><div class="block-child-p">The basic genome organizations of the four γ-HCH-degrading strains isolated from different geographical areas are summarized in <a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Table 1</a>. Comparative analysis of the 16S rRNA genes indicated that these four γ-HCH-degrading strains are phylogenetically diverse among related sphingomonad strains (<a reveal-id="dsw041-F2" data-open="dsw041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). All predicted ORFs of these four strains (19,312) were clustered into 10,325 ORF clusters. <a reveal-id="dsw041-F3" data-open="dsw041-F3" class="link link-reveal link-table xref-fig">Figure 3</a> summarizes the result as a Venn diagram that shows the numbers of shared and unique ORF clusters among the four strains; these four strains each have 1,190–2,346 unique ORF clusters, but share only 1,288 ones. This observation supports the phylogenetic divergence of the four γ-HCH degraders. These results also strongly suggest that the four degraders independently acquired γ-HCH-degradation ability, and thus it is unlikely that the four strains have been derived from one ancestral γ-HCH-degrader. <div data-id="dsw041-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f2p.png?Expires=2147483647&amp;Signature=sRnazR87ZCx7IQZmeMbaQFdaKjlaZmd16LvZ2uSBZIxpYqd1Ta6PzE~fqJa4s6v38yWtY3MYlZmipVfsZ6GCp9MTz8fL1PQ2TqvKvSUjcO4248Vx-VtBbIAqlE8nhKzpf3HpShxbfynZEfJHQZbCMJir8QBWtlpKfvTrPcF-yBug8l6Nn3LgilFj7YGlrP2S8vluErPNj--L5xBnPeAQVEfGvAe0tkg6Wah3bO47sNzIZRtUKNANP1Pc8cxMJ~j8-eowdj7DVL6H0EORN2ilDr5EvR21obYtmto7B~tGRn6-fHDHp0yjjzCTy0I~oRUiOCAboBNFP-BnjshyR~dvow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of 16S rRNA genes of sphingomonad strains. Neighbor-joining phylogenetic tree of the conserved sites in 16S rRNA genes of 13 sphingomonad strains, S. japonicum UT26S (UT26_1, SJA_C1-r0010; UT26_2, SJA_C2-r0010; UT26_3, SJA_C2-r0040), Sphingobium indicum B90A (B90A, NR_042943), Sphingobium francense Sp+ (Sp+, NR_042944), Sphingobium sp. TKS (TKS_1, Chr1_62351_63846; TKS_2, Chr2_117006_118503; and TKS_3, Chr2_376042_377539_c), Sphingobium chlorophenolicum L-1 (L-1_1, Sphch_R0043; L-2_2, Sphch_R0058; L-1_3, Sphch_R0067), Sphingomonas sp. SKA58 (SKA58_1, SKA58_r00366; SKA58_2, SKA58_r18278), Sphingobium sp. MI1205 (MI1205_1, Chr1_64638_66133; MI1205_2, Chr2_561355_562850_c), Sphingobiums sp. SYK-6 (SYK6_1, SLG_r0030; SYK6_2, SLG_r0060), Sphingomonas wittichii RW1 (RW1_1, Swit_R0031; RW1_2, Swit_R0040), Sphingomonas sp. MM-1 (MM-1_1, Chr_1791835_1793331_c; MM-1_2, Chr_2084177_2085673_c), Sphingopyxis alaskensis RB2256 (RB2256, Sala_R0048), Novosphingobium sp. PP1Y (PPY_1, PP1Y_AR03; PPY_2, PP1Y_AR23; PPY_3, PP1Y_AR65), and N. aromaticivorans DSM 12444 (DSM_1, Saro_R0065; DSM_2, Saro_R0059; DSM_3, Saro_R0053) was constructed. 16S rRNA gene (rrsE: gene ID 7437018) of E. coli str. K-12 substr. W3110 (E. coli) was used as an out-of-group sequence. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. Sphingomonas sp. SKA58 should be Sphingobium sp. SKA58 on the basis of comprehensive 16S rRNA gene analysis. However, we used ‘Sphingomonas’ for the strain according to the database in order to avoid confusion. γ-HCH degraders are bolded." path-from-xml="dsw041f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005572/dsw041f2p.tif" path-from-xml="dsw041f2p.tif" target="_blank">View large</a><a section="48005572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f2p.png?Expires=2147483647&amp;Signature=FU6dmSn1sxcCrf4O5vAunZA5Qy0zx3AZoFfPvqrxx0mkHkRVC3mysgMqUQrv6rIbueIA3nmwMZjyzg7DOxGBvZ2wnUmG4E56vx9~yNfEqm-dbcQUmT9SO5kS4NDE~A7jLt8eiOXbGMcSOkRkuYW4fWlAvfmgT6dk~b-CY5roc~Z~T5iEtXAoPwGD~T-BY~pNtifJDUKoF9o-b52uMI~YhGA~3Gb7gy8tu7N5hdpRvTlESgnX6B6zTqvwZ2fp1hYSPGH9JM50aXyK~I70WW~qOzOiLotg2n4tI5vMym85gQaUWlSHst7xHNEnQQJ9Qm98BoCrAniK5J0m49P8i6DlUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005572&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of 16S rRNA genes of sphingomonad strains. Neighbor-joining phylogenetic tree of the conserved sites in 16S rRNA genes of 13 sphingomonad strains, <em>S. japonicum</em> UT26S (UT26_1, SJA_C1-r0010; UT26_2, SJA_C2-r0010; UT26_3, SJA_C2-r0040), <em>Sphingobium indicum</em> B90A (B90A, NR_042943), <em>Sphingobium francense</em> Sp+ (Sp+, NR_042944), <em>Sphingobium</em> sp. TKS (TKS_1, Chr1_62351_63846; TKS_2, Chr2_117006_118503; and TKS_3, Chr2_376042_377539_c), <em>Sphingobium chlorophenolicum</em> L-1 (L-1_1, Sphch_R0043; L-2_2, Sphch_R0058; L-1_3, Sphch_R0067), <em>Sphingomonas</em> sp. SKA58 (SKA58_1, SKA58_r00366; SKA58_2, SKA58_r18278), <em>Sphingobium</em> sp. MI1205 (MI1205_1, Chr1_64638_66133; MI1205_2, Chr2_561355_562850_c), <em>Sphingobiums</em> sp. SYK-6 (SYK6_1, SLG_r0030; SYK6_2, SLG_r0060), <em>Sphingomonas wittichii</em> RW1 (RW1_1, Swit_R0031; RW1_2, Swit_R0040), <em>Sphingomonas</em> sp. MM-1 (MM-1_1, Chr_1791835_1793331_c; MM-1_2, Chr_2084177_2085673_c), <em>Sphingopyxis alaskensis</em> RB2256 (RB2256, Sala_R0048), <em>Novosphingobium</em> sp. PP1Y (PPY_1, PP1Y_AR03; PPY_2, PP1Y_AR23; PPY_3, PP1Y_AR65), and <em>N. aromaticivorans</em> DSM 12444 (DSM_1, Saro_R0065; DSM_2, Saro_R0059; DSM_3, Saro_R0053) was constructed. 16S rRNA gene (rrsE: gene ID 7437018) of <em>E. coli</em> str. K-12 substr. W3110 (<em>E. coli</em>) was used as an out-of-group sequence. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. <em>Sphingomonas</em> sp. SKA58 should be <em>Sphingobium</em> sp. SKA58 on the basis of comprehensive 16S rRNA gene analysis. However, we used ‘<em>Sphingomonas</em>’ for the strain according to the database in order to avoid confusion. γ-HCH degraders are bolded.</p></div></div><div content-id="dsw041-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f2p.png?Expires=2147483647&amp;Signature=sRnazR87ZCx7IQZmeMbaQFdaKjlaZmd16LvZ2uSBZIxpYqd1Ta6PzE~fqJa4s6v38yWtY3MYlZmipVfsZ6GCp9MTz8fL1PQ2TqvKvSUjcO4248Vx-VtBbIAqlE8nhKzpf3HpShxbfynZEfJHQZbCMJir8QBWtlpKfvTrPcF-yBug8l6Nn3LgilFj7YGlrP2S8vluErPNj--L5xBnPeAQVEfGvAe0tkg6Wah3bO47sNzIZRtUKNANP1Pc8cxMJ~j8-eowdj7DVL6H0EORN2ilDr5EvR21obYtmto7B~tGRn6-fHDHp0yjjzCTy0I~oRUiOCAboBNFP-BnjshyR~dvow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of 16S rRNA genes of sphingomonad strains. Neighbor-joining phylogenetic tree of the conserved sites in 16S rRNA genes of 13 sphingomonad strains, S. japonicum UT26S (UT26_1, SJA_C1-r0010; UT26_2, SJA_C2-r0010; UT26_3, SJA_C2-r0040), Sphingobium indicum B90A (B90A, NR_042943), Sphingobium francense Sp+ (Sp+, NR_042944), Sphingobium sp. TKS (TKS_1, Chr1_62351_63846; TKS_2, Chr2_117006_118503; and TKS_3, Chr2_376042_377539_c), Sphingobium chlorophenolicum L-1 (L-1_1, Sphch_R0043; L-2_2, Sphch_R0058; L-1_3, Sphch_R0067), Sphingomonas sp. SKA58 (SKA58_1, SKA58_r00366; SKA58_2, SKA58_r18278), Sphingobium sp. MI1205 (MI1205_1, Chr1_64638_66133; MI1205_2, Chr2_561355_562850_c), Sphingobiums sp. SYK-6 (SYK6_1, SLG_r0030; SYK6_2, SLG_r0060), Sphingomonas wittichii RW1 (RW1_1, Swit_R0031; RW1_2, Swit_R0040), Sphingomonas sp. MM-1 (MM-1_1, Chr_1791835_1793331_c; MM-1_2, Chr_2084177_2085673_c), Sphingopyxis alaskensis RB2256 (RB2256, Sala_R0048), Novosphingobium sp. PP1Y (PPY_1, PP1Y_AR03; PPY_2, PP1Y_AR23; PPY_3, PP1Y_AR65), and N. aromaticivorans DSM 12444 (DSM_1, Saro_R0065; DSM_2, Saro_R0059; DSM_3, Saro_R0053) was constructed. 16S rRNA gene (rrsE: gene ID 7437018) of E. coli str. K-12 substr. W3110 (E. coli) was used as an out-of-group sequence. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. Sphingomonas sp. SKA58 should be Sphingobium sp. SKA58 on the basis of comprehensive 16S rRNA gene analysis. However, we used ‘Sphingomonas’ for the strain according to the database in order to avoid confusion. γ-HCH degraders are bolded." path-from-xml="dsw041f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005572/dsw041f2p.tif" path-from-xml="dsw041f2p.tif" target="_blank">View large</a><a section="48005572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f2p.png?Expires=2147483647&amp;Signature=FU6dmSn1sxcCrf4O5vAunZA5Qy0zx3AZoFfPvqrxx0mkHkRVC3mysgMqUQrv6rIbueIA3nmwMZjyzg7DOxGBvZ2wnUmG4E56vx9~yNfEqm-dbcQUmT9SO5kS4NDE~A7jLt8eiOXbGMcSOkRkuYW4fWlAvfmgT6dk~b-CY5roc~Z~T5iEtXAoPwGD~T-BY~pNtifJDUKoF9o-b52uMI~YhGA~3Gb7gy8tu7N5hdpRvTlESgnX6B6zTqvwZ2fp1hYSPGH9JM50aXyK~I70WW~qOzOiLotg2n4tI5vMym85gQaUWlSHst7xHNEnQQJ9Qm98BoCrAniK5J0m49P8i6DlUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005572&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of 16S rRNA genes of sphingomonad strains. Neighbor-joining phylogenetic tree of the conserved sites in 16S rRNA genes of 13 sphingomonad strains, <em>S. japonicum</em> UT26S (UT26_1, SJA_C1-r0010; UT26_2, SJA_C2-r0010; UT26_3, SJA_C2-r0040), <em>Sphingobium indicum</em> B90A (B90A, NR_042943), <em>Sphingobium francense</em> Sp+ (Sp+, NR_042944), <em>Sphingobium</em> sp. TKS (TKS_1, Chr1_62351_63846; TKS_2, Chr2_117006_118503; and TKS_3, Chr2_376042_377539_c), <em>Sphingobium chlorophenolicum</em> L-1 (L-1_1, Sphch_R0043; L-2_2, Sphch_R0058; L-1_3, Sphch_R0067), <em>Sphingomonas</em> sp. SKA58 (SKA58_1, SKA58_r00366; SKA58_2, SKA58_r18278), <em>Sphingobium</em> sp. MI1205 (MI1205_1, Chr1_64638_66133; MI1205_2, Chr2_561355_562850_c), <em>Sphingobiums</em> sp. SYK-6 (SYK6_1, SLG_r0030; SYK6_2, SLG_r0060), <em>Sphingomonas wittichii</em> RW1 (RW1_1, Swit_R0031; RW1_2, Swit_R0040), <em>Sphingomonas</em> sp. MM-1 (MM-1_1, Chr_1791835_1793331_c; MM-1_2, Chr_2084177_2085673_c), <em>Sphingopyxis alaskensis</em> RB2256 (RB2256, Sala_R0048), <em>Novosphingobium</em> sp. PP1Y (PPY_1, PP1Y_AR03; PPY_2, PP1Y_AR23; PPY_3, PP1Y_AR65), and <em>N. aromaticivorans</em> DSM 12444 (DSM_1, Saro_R0065; DSM_2, Saro_R0059; DSM_3, Saro_R0053) was constructed. 16S rRNA gene (rrsE: gene ID 7437018) of <em>E. coli</em> str. K-12 substr. W3110 (<em>E. coli</em>) was used as an out-of-group sequence. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. <em>Sphingomonas</em> sp. SKA58 should be <em>Sphingobium</em> sp. SKA58 on the basis of comprehensive 16S rRNA gene analysis. However, we used ‘<em>Sphingomonas</em>’ for the strain according to the database in order to avoid confusion. γ-HCH degraders are bolded.</p></div></div><div data-id="dsw041-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f3p.png?Expires=2147483647&amp;Signature=c~FjmPJ9r6YJzHBwh~7wKNz7CE-zrmmyOkmg86-n~sD4HPfEsp573k-AKUqHjjnuFhyueSzYUtBJmve8EdkbKTo8~hQm9St5~7dCT~9G3nbzkeqrhz5TNHhl9tlQLnh3TVf4~X7rmjNwk-VFetiEN9ye7cBQTB72BMmUaU7dcqsSG~gcj0gKzUC1IiLOZm3k08avDzBFppv6kL3485vZG4cBd5pB36zQ0WpatJEhNUHcPAElwrJ4M7P75RTvcIJIz~W9scBrxj9gRypdiNaedUwID3co1F5DZJbmFawCiPasb1la~d3wAvuqf5Qy-gtP8WbewdfRVxLfKO~3yZPAXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the number of shared and unique ORF clusters of four γ-HCH-degrading sphingomonad strains. Total ORFs (19,312) of the four strains were clustered into 10,325 ORF clusters by BLASTClust analysis. Numbers of total ORF clusters of these four strains are shown in parentheses under the strain names." path-from-xml="dsw041f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005572/dsw041f3p.tif" path-from-xml="dsw041f3p.tif" target="_blank">View large</a><a section="48005572" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f3p.png?Expires=2147483647&amp;Signature=KBdUEaw6dYRBTbnrjvjtTNJBAvTshDWhLuDL4EocBmYsxFGI0vbuRo9H7Wg4KzCizAhkgoaPTy-fEj10tH7nhnHNGaOl4YzP1pjPx3D12AvX5KjpQE7oCr4-gCY4laSVTEMtjjP33Hf69QLBuaYfYG5usSz6aaxAxmdCXWNbtsy5lomLbFsoLZ~n-5ZZycyiqvNtMLjN6KuU1UyClEThTCHZ4FZIaiM5HplcNYBIujhIViglYcXuBGa91cLXqu8DZWZGmI8ynXCFVHeWj7FRrmEhVosgX6PoTJYHqShLsh5AB7bkjClIZxl9tQyPXJcqt85sB7j3ioLiHTukQxEbLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005572&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the number of shared and unique ORF clusters of four γ-HCH-degrading sphingomonad strains. Total ORFs (19,312) of the four strains were clustered into 10,325 ORF clusters by BLASTClust analysis. Numbers of total ORF clusters of these four strains are shown in parentheses under the strain names.</p></div></div><div content-id="dsw041-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f3p.png?Expires=2147483647&amp;Signature=c~FjmPJ9r6YJzHBwh~7wKNz7CE-zrmmyOkmg86-n~sD4HPfEsp573k-AKUqHjjnuFhyueSzYUtBJmve8EdkbKTo8~hQm9St5~7dCT~9G3nbzkeqrhz5TNHhl9tlQLnh3TVf4~X7rmjNwk-VFetiEN9ye7cBQTB72BMmUaU7dcqsSG~gcj0gKzUC1IiLOZm3k08avDzBFppv6kL3485vZG4cBd5pB36zQ0WpatJEhNUHcPAElwrJ4M7P75RTvcIJIz~W9scBrxj9gRypdiNaedUwID3co1F5DZJbmFawCiPasb1la~d3wAvuqf5Qy-gtP8WbewdfRVxLfKO~3yZPAXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the number of shared and unique ORF clusters of four γ-HCH-degrading sphingomonad strains. Total ORFs (19,312) of the four strains were clustered into 10,325 ORF clusters by BLASTClust analysis. Numbers of total ORF clusters of these four strains are shown in parentheses under the strain names." path-from-xml="dsw041f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005572/dsw041f3p.tif" path-from-xml="dsw041f3p.tif" target="_blank">View large</a><a section="48005572" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f3p.png?Expires=2147483647&amp;Signature=KBdUEaw6dYRBTbnrjvjtTNJBAvTshDWhLuDL4EocBmYsxFGI0vbuRo9H7Wg4KzCizAhkgoaPTy-fEj10tH7nhnHNGaOl4YzP1pjPx3D12AvX5KjpQE7oCr4-gCY4laSVTEMtjjP33Hf69QLBuaYfYG5usSz6aaxAxmdCXWNbtsy5lomLbFsoLZ~n-5ZZycyiqvNtMLjN6KuU1UyClEThTCHZ4FZIaiM5HplcNYBIujhIViglYcXuBGa91cLXqu8DZWZGmI8ynXCFVHeWj7FRrmEhVosgX6PoTJYHqShLsh5AB7bkjClIZxl9tQyPXJcqt85sB7j3ioLiHTukQxEbLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005572&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the number of shared and unique ORF clusters of four γ-HCH-degrading sphingomonad strains. Total ORFs (19,312) of the four strains were clustered into 10,325 ORF clusters by BLASTClust analysis. Numbers of total ORF clusters of these four strains are shown in parentheses under the strain names.</p></div></div><div content-id="dsw041-T1" class="table-modal"><div class="table-wrap"><div id="dsw041-T1" data-id="dsw041-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Genome organization of four γ-HCH-degrading sphingomonad strains</p></div></div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th rowspan="2">Strain name </th><th rowspan="2">Isolated site </th><th rowspan="2">Source type </th><th rowspan="2">Replicon </th><th rowspan="2">Length (bp) </th><th colspan="3">Number of </th><th>Acc no. </th><th rowspan="2">Replicon type<a reveal-id="dsw041-TF1" data-open="dsw041-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2"><em>lin</em> genes<a reveal-id="dsw041-TF2" data-open="dsw041-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="2">Reference for strain </th></tr><tr><th><em>rrn</em> operon </th><th>ORF </th><th>IS<em>6100</em> </th><th> </th></tr></thead><tbody><tr><td><em>S. japonicum</em> UT26 </td><td>Tokyo, Japan </td><td>Soil artificially polluted </td><td>Chromosome 1 </td><td char=".">3,514,822 </td><td char=".">1 </td><td char=".">3,529 </td><td char=".">5 </td><td>AP010803 </td><td>Chr </td><td><em>linA, linB, linC, linKLMN</em> </td><td char=".">18 </td></tr><tr><td> </td><td> </td><td>with γ−HCH </td><td>Chromosome 2 </td><td char=".">681,892 </td><td char=".">2 </td><td char=".">589 </td><td char=".">2 </td><td>AP010804 </td><td>UT26_Chr 2 </td><td><em>linF, linGHIJ, linEb</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pCHQ1 </td><td char=".">190,974 </td><td char=".">0 </td><td char=".">224 </td><td char=".">4 </td><td>AP010805 </td><td>pCHQ1 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT1 </td><td char=".">31,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">2 </td><td>AP010806 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT2 </td><td char=".">5,398 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>AP010807 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,424,862 </td><td char=".">3 </td><td char=".">4,394 </td><td char=".">13 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingomonas</em> sp. MM-1 </td><td>Lucknow, India </td><td>Soil polluted </td><td>Chromosome </td><td char=".">4,054,833 </td><td char=".">2 </td><td char=".">3,801 </td><td char=".">0 </td><td>CP004036 </td><td>Chr </td><td><em>linKbLbMbNb</em> </td><td char=".">22 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>pISP0 </td><td char=".">275,840 </td><td char=".">0 </td><td char=".">251 </td><td char=".">1 </td><td>CP004037 </td><td>pCHQ1 </td><td><em>linF, linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP1<sup>c</sup> </td><td char=".">172,140 </td><td char=".">0 </td><td char=".">174 </td><td char=".">7 </td><td>CP004038 </td><td>UT26_Chr 2/pISP4 </td><td><em>linA, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP2 </td><td char=".">53,841 </td><td char=".">0 </td><td char=".">52 </td><td char=".">2 </td><td>CP004039 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP3 </td><td char=".">43,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">1 </td><td>CP004040 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP4 </td><td char=".">33,183 </td><td char=".">0 </td><td char=".">39 </td><td char=".">4 </td><td>CP004041 </td><td>pISP4 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td char=".">4,633,613 </td><td char=".">2 </td><td char=".">4,361 </td><td char=".">15 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. TKS </td><td>Kyushu, Japan </td><td>Sediment polluted </td><td>Chromosome 1<sup>c</sup> </td><td char=".">4,249,857 </td><td char=".">1 </td><td char=".">4,172 </td><td char=".">7 </td><td>CP005083 </td><td>Chr/UT26_Chr 2 </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>This study </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">989,120 </td><td char=".">2 </td><td char=".">843 </td><td char=".">0 </td><td>CP005084 </td><td>UT26_Chr 2 </td><td><em>linGHIJ</em>_homologue </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK1<sup>c</sup> </td><td char=".">520,614 </td><td char=".">0 </td><td char=".">470 </td><td char=".">6 </td><td>CP005085 </td><td>UT26_Chr 2/pCHQ1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK2 </td><td char=".">195,308 </td><td char=".">0 </td><td char=".">182 </td><td char=".">1 </td><td>CP005086 </td><td>pTK2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK3<sup>c</sup> </td><td char=".">87,635 </td><td char=".">0 </td><td char=".">92 </td><td char=".">8 </td><td>CP005087 </td><td>pISP4/pTK3_type 1/pTK3_type 2 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK4<sup>c</sup> </td><td char=".">75,938 </td><td char=".">0 </td><td char=".">86 </td><td char=".">6 </td><td>CP005088 </td><td>pUT1/pISP4 </td><td><em>linA, linC</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK5 </td><td char=".">53,908 </td><td char=".">0 </td><td char=".">76 </td><td char=".">0 </td><td>CP005089 </td><td>pLB1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK6 </td><td char=".">34,300 </td><td char=".">0 </td><td char=".">35 </td><td char=".">1 </td><td>CP005090 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK7 </td><td char=".">9,585 </td><td char=".">0 </td><td char=".">12 </td><td char=".">0 </td><td>CP005091 </td><td>pTK7 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK8 </td><td char=".">7,223 </td><td char=".">0 </td><td char=".">11 </td><td char=".">0 </td><td>CP005092 </td><td>pTK8 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK9 </td><td char=".">5,391 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>CP005093 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">6,228,879 </td><td char=".">3 </td><td char=".">5,987 </td><td char=".">29 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. MI1205 </td><td>Miyagi, Japan </td><td>Soil polluted </td><td>Chromosome 1 </td><td char=".">3,351,250 </td><td char=".">1 </td><td char=".">3,285 </td><td char=".">0 </td><td>CP005188 </td><td>Chr </td><td><em>linKLMN</em> </td><td char=".">24 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">567,154 </td><td char=".">1 </td><td char=".">516 </td><td char=".">0 </td><td>CP005189 </td><td>UT26_Chr 2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI1 </td><td char=".">292,135 </td><td char=".">0 </td><td char=".">299 </td><td char=".">7 </td><td>CP005190 </td><td>UT26_Chr 2 </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI2<sup>c</sup> </td><td char=".">287,488 </td><td char=".">0 </td><td char=".">323 </td><td char=".">7 </td><td>CP005191 </td><td>pUT1/UT26_Chr 2 </td><td><em>linA, linRD</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI3<sup>c</sup> </td><td char=".">88,374 </td><td char=".">0 </td><td char=".">102 </td><td char=".">7 </td><td>CP005192 </td><td>pLB1/pISP4 </td><td><em>linB, linC, linF''</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI4<sup>c</sup> </td><td char=".">32,974 </td><td char=".">0 </td><td char=".">45 </td><td char=".">3 </td><td>CP005193 </td><td>pISP3/pISP4 </td><td><em>linRED</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,619,375 </td><td char=".">2 </td><td char=".">4,570 </td><td char=".">24 </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="dsw041-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th rowspan="2">Strain name </th><th rowspan="2">Isolated site </th><th rowspan="2">Source type </th><th rowspan="2">Replicon </th><th rowspan="2">Length (bp) </th><th colspan="3">Number of </th><th>Acc no. </th><th rowspan="2">Replicon type<a reveal-id="dsw041-TF1" data-open="dsw041-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2"><em>lin</em> genes<a reveal-id="dsw041-TF2" data-open="dsw041-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="2">Reference for strain </th></tr><tr><th><em>rrn</em> operon </th><th>ORF </th><th>IS<em>6100</em> </th><th> </th></tr></thead><tbody><tr><td><em>S. japonicum</em> UT26 </td><td>Tokyo, Japan </td><td>Soil artificially polluted </td><td>Chromosome 1 </td><td char=".">3,514,822 </td><td char=".">1 </td><td char=".">3,529 </td><td char=".">5 </td><td>AP010803 </td><td>Chr </td><td><em>linA, linB, linC, linKLMN</em> </td><td char=".">18 </td></tr><tr><td> </td><td> </td><td>with γ−HCH </td><td>Chromosome 2 </td><td char=".">681,892 </td><td char=".">2 </td><td char=".">589 </td><td char=".">2 </td><td>AP010804 </td><td>UT26_Chr 2 </td><td><em>linF, linGHIJ, linEb</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pCHQ1 </td><td char=".">190,974 </td><td char=".">0 </td><td char=".">224 </td><td char=".">4 </td><td>AP010805 </td><td>pCHQ1 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT1 </td><td char=".">31,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">2 </td><td>AP010806 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT2 </td><td char=".">5,398 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>AP010807 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,424,862 </td><td char=".">3 </td><td char=".">4,394 </td><td char=".">13 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingomonas</em> sp. MM-1 </td><td>Lucknow, India </td><td>Soil polluted </td><td>Chromosome </td><td char=".">4,054,833 </td><td char=".">2 </td><td char=".">3,801 </td><td char=".">0 </td><td>CP004036 </td><td>Chr </td><td><em>linKbLbMbNb</em> </td><td char=".">22 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>pISP0 </td><td char=".">275,840 </td><td char=".">0 </td><td char=".">251 </td><td char=".">1 </td><td>CP004037 </td><td>pCHQ1 </td><td><em>linF, linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP1<sup>c</sup> </td><td char=".">172,140 </td><td char=".">0 </td><td char=".">174 </td><td char=".">7 </td><td>CP004038 </td><td>UT26_Chr 2/pISP4 </td><td><em>linA, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP2 </td><td char=".">53,841 </td><td char=".">0 </td><td char=".">52 </td><td char=".">2 </td><td>CP004039 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP3 </td><td char=".">43,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">1 </td><td>CP004040 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP4 </td><td char=".">33,183 </td><td char=".">0 </td><td char=".">39 </td><td char=".">4 </td><td>CP004041 </td><td>pISP4 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td char=".">4,633,613 </td><td char=".">2 </td><td char=".">4,361 </td><td char=".">15 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. TKS </td><td>Kyushu, Japan </td><td>Sediment polluted </td><td>Chromosome 1<sup>c</sup> </td><td char=".">4,249,857 </td><td char=".">1 </td><td char=".">4,172 </td><td char=".">7 </td><td>CP005083 </td><td>Chr/UT26_Chr 2 </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>This study </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">989,120 </td><td char=".">2 </td><td char=".">843 </td><td char=".">0 </td><td>CP005084 </td><td>UT26_Chr 2 </td><td><em>linGHIJ</em>_homologue </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK1<sup>c</sup> </td><td char=".">520,614 </td><td char=".">0 </td><td char=".">470 </td><td char=".">6 </td><td>CP005085 </td><td>UT26_Chr 2/pCHQ1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK2 </td><td char=".">195,308 </td><td char=".">0 </td><td char=".">182 </td><td char=".">1 </td><td>CP005086 </td><td>pTK2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK3<sup>c</sup> </td><td char=".">87,635 </td><td char=".">0 </td><td char=".">92 </td><td char=".">8 </td><td>CP005087 </td><td>pISP4/pTK3_type 1/pTK3_type 2 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK4<sup>c</sup> </td><td char=".">75,938 </td><td char=".">0 </td><td char=".">86 </td><td char=".">6 </td><td>CP005088 </td><td>pUT1/pISP4 </td><td><em>linA, linC</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK5 </td><td char=".">53,908 </td><td char=".">0 </td><td char=".">76 </td><td char=".">0 </td><td>CP005089 </td><td>pLB1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK6 </td><td char=".">34,300 </td><td char=".">0 </td><td char=".">35 </td><td char=".">1 </td><td>CP005090 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK7 </td><td char=".">9,585 </td><td char=".">0 </td><td char=".">12 </td><td char=".">0 </td><td>CP005091 </td><td>pTK7 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK8 </td><td char=".">7,223 </td><td char=".">0 </td><td char=".">11 </td><td char=".">0 </td><td>CP005092 </td><td>pTK8 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK9 </td><td char=".">5,391 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>CP005093 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">6,228,879 </td><td char=".">3 </td><td char=".">5,987 </td><td char=".">29 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. MI1205 </td><td>Miyagi, Japan </td><td>Soil polluted </td><td>Chromosome 1 </td><td char=".">3,351,250 </td><td char=".">1 </td><td char=".">3,285 </td><td char=".">0 </td><td>CP005188 </td><td>Chr </td><td><em>linKLMN</em> </td><td char=".">24 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">567,154 </td><td char=".">1 </td><td char=".">516 </td><td char=".">0 </td><td>CP005189 </td><td>UT26_Chr 2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI1 </td><td char=".">292,135 </td><td char=".">0 </td><td char=".">299 </td><td char=".">7 </td><td>CP005190 </td><td>UT26_Chr 2 </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI2<sup>c</sup> </td><td char=".">287,488 </td><td char=".">0 </td><td char=".">323 </td><td char=".">7 </td><td>CP005191 </td><td>pUT1/UT26_Chr 2 </td><td><em>linA, linRD</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI3<sup>c</sup> </td><td char=".">88,374 </td><td char=".">0 </td><td char=".">102 </td><td char=".">7 </td><td>CP005192 </td><td>pLB1/pISP4 </td><td><em>linB, linC, linF''</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI4<sup>c</sup> </td><td char=".">32,974 </td><td char=".">0 </td><td char=".">45 </td><td char=".">3 </td><td>CP005193 </td><td>pISP3/pISP4 </td><td><em>linRED</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,619,375 </td><td char=".">2 </td><td char=".">4,570 </td><td char=".">24 </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw041-TF1"><div class="fn"><p><sup>a</sup>See <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a></p></div></div><div content-id="dsw041-TF2"><div class="fn"><p><sup>b</sup><em>linF</em>′ and <em>linF</em>″ are probably pseudogenes (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p></div></div><div content-id="dsw041-TF3"><div class="fn"><p><sup>c</sup>Replicons having more than one <em>rep</em> genes.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw041-T1" data-id="dsw041-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Genome organization of four γ-HCH-degrading sphingomonad strains</p></div></div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th rowspan="2">Strain name </th><th rowspan="2">Isolated site </th><th rowspan="2">Source type </th><th rowspan="2">Replicon </th><th rowspan="2">Length (bp) </th><th colspan="3">Number of </th><th>Acc no. </th><th rowspan="2">Replicon type<a reveal-id="dsw041-TF1" data-open="dsw041-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2"><em>lin</em> genes<a reveal-id="dsw041-TF2" data-open="dsw041-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="2">Reference for strain </th></tr><tr><th><em>rrn</em> operon </th><th>ORF </th><th>IS<em>6100</em> </th><th> </th></tr></thead><tbody><tr><td><em>S. japonicum</em> UT26 </td><td>Tokyo, Japan </td><td>Soil artificially polluted </td><td>Chromosome 1 </td><td char=".">3,514,822 </td><td char=".">1 </td><td char=".">3,529 </td><td char=".">5 </td><td>AP010803 </td><td>Chr </td><td><em>linA, linB, linC, linKLMN</em> </td><td char=".">18 </td></tr><tr><td> </td><td> </td><td>with γ−HCH </td><td>Chromosome 2 </td><td char=".">681,892 </td><td char=".">2 </td><td char=".">589 </td><td char=".">2 </td><td>AP010804 </td><td>UT26_Chr 2 </td><td><em>linF, linGHIJ, linEb</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pCHQ1 </td><td char=".">190,974 </td><td char=".">0 </td><td char=".">224 </td><td char=".">4 </td><td>AP010805 </td><td>pCHQ1 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT1 </td><td char=".">31,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">2 </td><td>AP010806 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT2 </td><td char=".">5,398 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>AP010807 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,424,862 </td><td char=".">3 </td><td char=".">4,394 </td><td char=".">13 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingomonas</em> sp. MM-1 </td><td>Lucknow, India </td><td>Soil polluted </td><td>Chromosome </td><td char=".">4,054,833 </td><td char=".">2 </td><td char=".">3,801 </td><td char=".">0 </td><td>CP004036 </td><td>Chr </td><td><em>linKbLbMbNb</em> </td><td char=".">22 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>pISP0 </td><td char=".">275,840 </td><td char=".">0 </td><td char=".">251 </td><td char=".">1 </td><td>CP004037 </td><td>pCHQ1 </td><td><em>linF, linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP1<sup>c</sup> </td><td char=".">172,140 </td><td char=".">0 </td><td char=".">174 </td><td char=".">7 </td><td>CP004038 </td><td>UT26_Chr 2/pISP4 </td><td><em>linA, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP2 </td><td char=".">53,841 </td><td char=".">0 </td><td char=".">52 </td><td char=".">2 </td><td>CP004039 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP3 </td><td char=".">43,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">1 </td><td>CP004040 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP4 </td><td char=".">33,183 </td><td char=".">0 </td><td char=".">39 </td><td char=".">4 </td><td>CP004041 </td><td>pISP4 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td char=".">4,633,613 </td><td char=".">2 </td><td char=".">4,361 </td><td char=".">15 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. TKS </td><td>Kyushu, Japan </td><td>Sediment polluted </td><td>Chromosome 1<sup>c</sup> </td><td char=".">4,249,857 </td><td char=".">1 </td><td char=".">4,172 </td><td char=".">7 </td><td>CP005083 </td><td>Chr/UT26_Chr 2 </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>This study </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">989,120 </td><td char=".">2 </td><td char=".">843 </td><td char=".">0 </td><td>CP005084 </td><td>UT26_Chr 2 </td><td><em>linGHIJ</em>_homologue </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK1<sup>c</sup> </td><td char=".">520,614 </td><td char=".">0 </td><td char=".">470 </td><td char=".">6 </td><td>CP005085 </td><td>UT26_Chr 2/pCHQ1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK2 </td><td char=".">195,308 </td><td char=".">0 </td><td char=".">182 </td><td char=".">1 </td><td>CP005086 </td><td>pTK2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK3<sup>c</sup> </td><td char=".">87,635 </td><td char=".">0 </td><td char=".">92 </td><td char=".">8 </td><td>CP005087 </td><td>pISP4/pTK3_type 1/pTK3_type 2 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK4<sup>c</sup> </td><td char=".">75,938 </td><td char=".">0 </td><td char=".">86 </td><td char=".">6 </td><td>CP005088 </td><td>pUT1/pISP4 </td><td><em>linA, linC</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK5 </td><td char=".">53,908 </td><td char=".">0 </td><td char=".">76 </td><td char=".">0 </td><td>CP005089 </td><td>pLB1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK6 </td><td char=".">34,300 </td><td char=".">0 </td><td char=".">35 </td><td char=".">1 </td><td>CP005090 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK7 </td><td char=".">9,585 </td><td char=".">0 </td><td char=".">12 </td><td char=".">0 </td><td>CP005091 </td><td>pTK7 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK8 </td><td char=".">7,223 </td><td char=".">0 </td><td char=".">11 </td><td char=".">0 </td><td>CP005092 </td><td>pTK8 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK9 </td><td char=".">5,391 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>CP005093 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">6,228,879 </td><td char=".">3 </td><td char=".">5,987 </td><td char=".">29 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. MI1205 </td><td>Miyagi, Japan </td><td>Soil polluted </td><td>Chromosome 1 </td><td char=".">3,351,250 </td><td char=".">1 </td><td char=".">3,285 </td><td char=".">0 </td><td>CP005188 </td><td>Chr </td><td><em>linKLMN</em> </td><td char=".">24 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">567,154 </td><td char=".">1 </td><td char=".">516 </td><td char=".">0 </td><td>CP005189 </td><td>UT26_Chr 2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI1 </td><td char=".">292,135 </td><td char=".">0 </td><td char=".">299 </td><td char=".">7 </td><td>CP005190 </td><td>UT26_Chr 2 </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI2<sup>c</sup> </td><td char=".">287,488 </td><td char=".">0 </td><td char=".">323 </td><td char=".">7 </td><td>CP005191 </td><td>pUT1/UT26_Chr 2 </td><td><em>linA, linRD</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI3<sup>c</sup> </td><td char=".">88,374 </td><td char=".">0 </td><td char=".">102 </td><td char=".">7 </td><td>CP005192 </td><td>pLB1/pISP4 </td><td><em>linB, linC, linF''</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI4<sup>c</sup> </td><td char=".">32,974 </td><td char=".">0 </td><td char=".">45 </td><td char=".">3 </td><td>CP005193 </td><td>pISP3/pISP4 </td><td><em>linRED</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,619,375 </td><td char=".">2 </td><td char=".">4,570 </td><td char=".">24 </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="dsw041-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th rowspan="2">Strain name </th><th rowspan="2">Isolated site </th><th rowspan="2">Source type </th><th rowspan="2">Replicon </th><th rowspan="2">Length (bp) </th><th colspan="3">Number of </th><th>Acc no. </th><th rowspan="2">Replicon type<a reveal-id="dsw041-TF1" data-open="dsw041-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2"><em>lin</em> genes<a reveal-id="dsw041-TF2" data-open="dsw041-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="2">Reference for strain </th></tr><tr><th><em>rrn</em> operon </th><th>ORF </th><th>IS<em>6100</em> </th><th> </th></tr></thead><tbody><tr><td><em>S. japonicum</em> UT26 </td><td>Tokyo, Japan </td><td>Soil artificially polluted </td><td>Chromosome 1 </td><td char=".">3,514,822 </td><td char=".">1 </td><td char=".">3,529 </td><td char=".">5 </td><td>AP010803 </td><td>Chr </td><td><em>linA, linB, linC, linKLMN</em> </td><td char=".">18 </td></tr><tr><td> </td><td> </td><td>with γ−HCH </td><td>Chromosome 2 </td><td char=".">681,892 </td><td char=".">2 </td><td char=".">589 </td><td char=".">2 </td><td>AP010804 </td><td>UT26_Chr 2 </td><td><em>linF, linGHIJ, linEb</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pCHQ1 </td><td char=".">190,974 </td><td char=".">0 </td><td char=".">224 </td><td char=".">4 </td><td>AP010805 </td><td>pCHQ1 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT1 </td><td char=".">31,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">2 </td><td>AP010806 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pUT2 </td><td char=".">5,398 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>AP010807 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,424,862 </td><td char=".">3 </td><td char=".">4,394 </td><td char=".">13 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingomonas</em> sp. MM-1 </td><td>Lucknow, India </td><td>Soil polluted </td><td>Chromosome </td><td char=".">4,054,833 </td><td char=".">2 </td><td char=".">3,801 </td><td char=".">0 </td><td>CP004036 </td><td>Chr </td><td><em>linKbLbMbNb</em> </td><td char=".">22 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>pISP0 </td><td char=".">275,840 </td><td char=".">0 </td><td char=".">251 </td><td char=".">1 </td><td>CP004037 </td><td>pCHQ1 </td><td><em>linF, linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP1<sup>c</sup> </td><td char=".">172,140 </td><td char=".">0 </td><td char=".">174 </td><td char=".">7 </td><td>CP004038 </td><td>UT26_Chr 2/pISP4 </td><td><em>linA, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP2 </td><td char=".">53,841 </td><td char=".">0 </td><td char=".">52 </td><td char=".">2 </td><td>CP004039 </td><td>pUT1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP3 </td><td char=".">43,776 </td><td char=".">0 </td><td char=".">44 </td><td char=".">1 </td><td>CP004040 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pISP4 </td><td char=".">33,183 </td><td char=".">0 </td><td char=".">39 </td><td char=".">4 </td><td>CP004041 </td><td>pISP4 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td char=".">4,633,613 </td><td char=".">2 </td><td char=".">4,361 </td><td char=".">15 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. TKS </td><td>Kyushu, Japan </td><td>Sediment polluted </td><td>Chromosome 1<sup>c</sup> </td><td char=".">4,249,857 </td><td char=".">1 </td><td char=".">4,172 </td><td char=".">7 </td><td>CP005083 </td><td>Chr/UT26_Chr 2 </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>This study </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">989,120 </td><td char=".">2 </td><td char=".">843 </td><td char=".">0 </td><td>CP005084 </td><td>UT26_Chr 2 </td><td><em>linGHIJ</em>_homologue </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK1<sup>c</sup> </td><td char=".">520,614 </td><td char=".">0 </td><td char=".">470 </td><td char=".">6 </td><td>CP005085 </td><td>UT26_Chr 2/pCHQ1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK2 </td><td char=".">195,308 </td><td char=".">0 </td><td char=".">182 </td><td char=".">1 </td><td>CP005086 </td><td>pTK2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK3<sup>c</sup> </td><td char=".">87,635 </td><td char=".">0 </td><td char=".">92 </td><td char=".">8 </td><td>CP005087 </td><td>pISP4/pTK3_type 1/pTK3_type 2 </td><td><em>linB, linC, linF'</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK4<sup>c</sup> </td><td char=".">75,938 </td><td char=".">0 </td><td char=".">86 </td><td char=".">6 </td><td>CP005088 </td><td>pUT1/pISP4 </td><td><em>linA, linC</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK5 </td><td char=".">53,908 </td><td char=".">0 </td><td char=".">76 </td><td char=".">0 </td><td>CP005089 </td><td>pLB1 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK6 </td><td char=".">34,300 </td><td char=".">0 </td><td char=".">35 </td><td char=".">1 </td><td>CP005090 </td><td>pISP3 </td><td><em>linRED</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK7 </td><td char=".">9,585 </td><td char=".">0 </td><td char=".">12 </td><td char=".">0 </td><td>CP005091 </td><td>pTK7 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK8 </td><td char=".">7,223 </td><td char=".">0 </td><td char=".">11 </td><td char=".">0 </td><td>CP005092 </td><td>pTK8 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pTK9 </td><td char=".">5,391 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td>CP005093 </td><td>pUT2 </td><td><em>—</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">6,228,879 </td><td char=".">3 </td><td char=".">5,987 </td><td char=".">29 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="12"> </td></tr><tr><td><em>Sphingobium</em> sp. MI1205 </td><td>Miyagi, Japan </td><td>Soil polluted </td><td>Chromosome 1 </td><td char=".">3,351,250 </td><td char=".">1 </td><td char=".">3,285 </td><td char=".">0 </td><td>CP005188 </td><td>Chr </td><td><em>linKLMN</em> </td><td char=".">24 </td></tr><tr><td> </td><td> </td><td>with HCH isomers </td><td>Chromosome 2 </td><td char=".">567,154 </td><td char=".">1 </td><td char=".">516 </td><td char=".">0 </td><td>CP005189 </td><td>UT26_Chr 2 </td><td><em>—</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI1 </td><td char=".">292,135 </td><td char=".">0 </td><td char=".">299 </td><td char=".">7 </td><td>CP005190 </td><td>UT26_Chr 2 </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI2<sup>c</sup> </td><td char=".">287,488 </td><td char=".">0 </td><td char=".">323 </td><td char=".">7 </td><td>CP005191 </td><td>pUT1/UT26_Chr 2 </td><td><em>linA, linRD</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI3<sup>c</sup> </td><td char=".">88,374 </td><td char=".">0 </td><td char=".">102 </td><td char=".">7 </td><td>CP005192 </td><td>pLB1/pISP4 </td><td><em>linB, linC, linF''</em> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>pMI4<sup>c</sup> </td><td char=".">32,974 </td><td char=".">0 </td><td char=".">45 </td><td char=".">3 </td><td>CP005193 </td><td>pISP3/pISP4 </td><td><em>linRED</em> </td><td> </td></tr><tr><td colspan="3"> </td><td colspan="5"> </td><td colspan="4"> </td></tr><tr><td> </td><td> </td><td> </td><td>total </td><td char=".">4,619,375 </td><td char=".">2 </td><td char=".">4,570 </td><td char=".">24 </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw041-TF1"><div class="fn"><p><sup>a</sup>See <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a></p></div></div><div content-id="dsw041-TF2"><div class="fn"><p><sup>b</sup><em>linF</em>′ and <em>linF</em>″ are probably pseudogenes (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p></div></div><div content-id="dsw041-TF3"><div class="fn"><p><sup>c</sup>Replicons having more than one <em>rep</em> genes.</p></div></div></div></div></div><p>In addition to the genes for γ-HCH degradation (see below), putative genes for the degradation of various aromatic compounds, toluene/phenol, chlorophenol, anthranilate, and homogentisate, reside in the UT26 genome.<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> These genes constitute four clusters for the degradation of the respective compounds, and each cluster contains all the genes necessary for the conversion of each compound to the metabolites in the central metabolic pathway, strongly suggesting that UT26 is able to utilize these compounds. Similarly, several putative genes for the degradation of aromatic compounds were found in the MM-1, MI1205, and TKS genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The potential of the four γ-HCH degraders for the degradation of aromatic compounds was estimated more comprehensively by BLASTP search of all their ORFs against our previously constructed in-house database which consists of enzymes for the degradation of aromatic compounds,<a reveal-id="dsw041-B41" data-open="dsw041-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and the result was summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. The numbers of ORFs potentially involved in the degradation of aromatic compounds in the four strains (62, 46, 27, and 25 for TKS, UT26, MI1205, and MM-1, respectively) are much smaller than those in versatile recalcitrant pollutant degraders, <em>Cupriavidus necator</em> JMP134,<a reveal-id="dsw041-B43" data-open="dsw041-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsw041-B44" data-open="dsw041-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> and <em>Burkholderia xenovorans</em> LB400<a reveal-id="dsw041-B45" data-open="dsw041-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsw041-B46" data-open="dsw041-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> (149 and 135 for JMP134 and LB400, respectively). Especially, those in UT26, MI1205, and MM-1 are even smaller than those in typical metabolically versatile soil bacterial strains <em>Burkholderia multivorans</em> ATCC 17616<a reveal-id="dsw041-B47 dsw041-B48 dsw041-B49 dsw041-B50" data-open="dsw041-B47 dsw041-B48 dsw041-B49 dsw041-B50" class="link link-ref link-reveal xref-bibr"><sup>47–50</sup></a> and <em>Pseudomonas putida</em> KT2440<a reveal-id="dsw041-B51" data-open="dsw041-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> (73 and 62 for KT2440 and ATCC 17616, respectively). These results indicate that our sphingomonad strains are ‘specialists’ for γ-HCH degradation, but not ‘generalists’ for the degradation of many recalcitrant compounds.</p>                            <h3 scrollto-destination=48005574 id="48005574" class="section-title" data-legacy-id=SEC3.2>3.2. The <em>lin</em> genes for γ-HCH utilization</h3>
<p>UT26 converts γ-HCH to β-ketoadipate via reactions catalysed by dehydrochlorinase (LinA), haloalkane dehalogenase (LinB), dehydrogenase (LinC), reductive dechlorinase (LinD), ring-cleavage dioxygenase (LinE), and maleylacetate reductase (MAR) (LinF); β-ketoadipate is thereafter converted to succinyl-CoA and acetyl-CoA by succinyl-CoA:3-oxoadipate CoA transferase (LinGH) and β-ketoadipyl CoA thiolase (LinJ), respectively (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>).<a reveal-id="dsw041-B16" data-open="dsw041-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> In addition to genes for these catabolic enzymes and their regulatory genes (<em>linR</em> for <em>linDE</em> and <em>linI</em> for <em>linGHJ</em>),<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw041-B52" data-open="dsw041-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> the <em>linKLMN</em> genes encoding a putative ABC-transporter system are necessary for the γ-HCH utilization in UT26.<a reveal-id="dsw041-B29" data-open="dsw041-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The <em>linA</em>, <em>linB</em>, <em>linC</em>, and <em>linF</em> genes, and the <em>linRED, linGHIJ</em>, and <em>linKLMN</em> clusters are dispersed on the UT26 genome.<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Since the β-ketoadipate pathway is often used by environmental bacterial strains,<a reveal-id="dsw041-B53" data-open="dsw041-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> the <em>lin</em> genes for the conversion of γ-HCH to β-ketoadipate (<em>linA</em> to <em>linF</em>) are peculiar to the γ-HCH-degrading pathway. In particular, the <em>linA</em> gene is unique because it does not show significant similarity to any sequences in the databases except for the almost identical (&gt;90% identity) <em>linA</em> genes from other bacterial strains and metagenomes.<a reveal-id="dsw041-B7" data-open="dsw041-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw041-B16" data-open="dsw041-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p><p>The MM-1, MI1205, and TKS genomes carry <em>linA</em>, <em>linB</em>, and <em>linC</em> genes and a <em>linRED</em> cluster that are almost identical (&gt;98% identity at the DNA level) to those of UT26 (<a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The former two strains additionally carry a <em>linF</em> gene and <em>linGHIJ</em> cluster that are almost identical (&gt;98% identity at the DNA level) to those in UT26, strongly suggesting that γ-HCH is degraded in these two strains by the same pathway as in UT26 (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). The last strain lacks the <em>linF</em><sub>UT26</sub> and <em>linGHIJ</em><sub>UT26</sub> cluster for maleylacetate metabolism (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Although two copies of the truncated version of <em>linF</em> (named <em>linF</em><em>′</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) are present in the TKS genome, <em>linF'</em> was assumed not to encode functional MAR, since <em>linF'</em> misses more than one half of the intact <em>linF</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). TKS instead carries another putative gene, designated <em>linFb</em>, on Chr1. Although LinFb showed only 49% identity to LinF<sub>UT26</sub> (<a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>), it showed much higher similarity with other known MAR proteins, such as TfdF from <em>Bordetella petrii</em>,<a reveal-id="dsw041-B54" data-open="dsw041-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><em>Achromobacter denifrificans</em>,<a reveal-id="dsw041-B55" data-open="dsw041-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><em>Burkholderia</em> sp. M701 (Acc no. YP_008864525), and <em>Comamonas testosteroni</em><a reveal-id="dsw041-B56" data-open="dsw041-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> (83,78, 78, and 71%, respectively). Interestingly, <em>Sphingobium</em> sp. HDIPO4, a recently isolated HCH degrader, has the identical <em>linFb</em> gene, although its start codon is annotated at a position different from that in TKS. <a reveal-id="dsw041-B57" data-open="dsw041-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> To clarify the LinFb function, its gene in TKS was disrupted (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The resultant strain did not grow on a minimal agar plate supplemented with γ-HCH as a sole source of carbon and energy, and this growth defect was reversed by the supply of the intact <em>linFb</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). In addition, the γ-HCH utilization defect of UT1023d, a <em>linF</em> mutant of UT26,<a reveal-id="dsw041-B34" data-open="dsw041-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was reversed by the supply of the <em>linFb</em> gene (data not shown). These results clearly demonstrated that the <em>linFb</em> encodes a MAR that is functional for the γ-HCH utilization. Although the <em>linGHIJ</em><sub>UT26</sub> cluster was not found in the TKS genome, many homologues of <em>linG</em>, <em>linH</em>, and <em>linJ</em> were found (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Among these homologues, one set of <em>linGH</em> homologues is located just downstream of <em>linFb</em> on Chr1<sub>TKS</sub> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>A) and only one set of <em>linGHIJ</em> homologues exists as a cluster on Chr2<sub>TKS</sub> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>B). Some of these homologues may be functional for the β-ketoadipate metabolism, and this experimental confirmation is necessary. These results strongly suggested that γ-HCH is also degraded in TKS by the same pathway as in UT26 (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). <div content-id="dsw041-T2" class="table-modal"><div class="table-wrap"><div id="dsw041-T2" data-id="dsw041-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p><em>lin</em> genes of four γ−HCH-degrading sphingomonad strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th rowspan="3">Gene<a reveal-id="dsw041-TF5" data-open="dsw041-TF5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="3">Function </th><th colspan="2">UT26 </th><th colspan="4">MM-1 </th><th colspan="4">TKS </th><th colspan="4">MI1205 </th></tr><tr><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th></tr><tr><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th></tr></thead><tbody><tr><td><em>linA</em> </td><td>Dehydrochlorinase </td><td>156 </td><td>Chr1_1860686-1861156_c </td><td>156 </td><td>pISP1_13547_14017 </td><td>98 (153/156) </td><td>98 (466/471) </td><td>156 </td><td>pTK4_18583_19053 </td><td colspan="2">identical </td><td>156 </td><td>pMI2_260065_260535_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linB</em> </td><td>Halidohydrolase </td><td>296 </td><td>Chr1_1966541-1967431 </td><td>296 </td><td>pISP4_18957_19847_c </td><td>98 (293/296) </td><td>99 (883/891) </td><td>296 </td><td>Chr1_230419_231309 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI1_179020_179910_c </td><td>98 (292/296) </td><td>99 (886/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pTK3_34952_35842 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI3_39099_39989 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pMI3_41282_42172 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linC</em> </td><td>Dehydrogenase </td><td>250 </td><td>Chr1_566609-567361_c </td><td>250 </td><td>pISP1_147367_148119 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>Chr1_469607_470359_c </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI1_169764_170516_c </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pISP4_11370_12122 </td><td colspan="2">identical </td><td>250 </td><td>pTK3_74827_75579 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI3_15466_16218 </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pTK4_21888_22640 </td><td>99 (249/250) </td><td>99 (752/753) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linD</em> </td><td>Reductive dechlorinase </td><td>346 </td><td>pCHQ1_110947-111987_c </td><td>346 </td><td>pISP3_15908_16948 </td><td colspan="2">identical </td><td>346 </td><td>pTK6_21982_23022 </td><td colspan="2">identical </td><td>346 </td><td>pMI1_148490_149530_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI2_214796_215836 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI4_28637_29677 </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linE</em> </td><td rowspan="3">Ring-cleavage dioxygenase </td><td>321 </td><td>pCHQ1_114235-115200_c </td><td>321 </td><td>pISP3_12695_13660 </td><td colspan="2">identical </td><td>321 </td><td>pTK6_18769_19734 </td><td colspan="2">identical </td><td>321 </td><td>pMI1_151777_152742_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>321 </td><td>pMI4_25425_26390 </td><td>identical </td><td>99 (965/966) </td></tr><tr><td><em>linEb</em><a reveal-id="dsw041-TF6" data-open="dsw041-TF6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>320 </td><td>Chr2_564928_565890 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>pMI1_141726_142688_c </td><td>99 (319/320) </td><td>99 (961/963) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linR</em> </td><td>LysR-family transcriptional regulator </td><td>303 </td><td>pCHQ1_115332-116243 </td><td>303 </td><td>pISP3_11652_12563_c </td><td>99 (302/303) </td><td>99 (911/912) </td><td>303 </td><td>pTK6_17726_18637_c </td><td colspan="2">identical </td><td>303 </td><td>pMI1_152874_153785 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI2_210541_211452_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI4_24382_25293_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linF</em> </td><td rowspan="6">Maleylactate reductase </td><td>352 </td><td>Chr2_562332-563390_c </td><td>352 </td><td>pISP0_110260_111318_c </td><td>identical </td><td>98 (1046/1059) </td><td> </td><td> </td><td> </td><td> </td><td>352 </td><td>pMI1_144226_145284 </td><td>identical </td><td>99 (1049/1059) </td></tr><tr><td><em>linF'</em> </td><td> </td><td> </td><td>180 </td><td>pISP1_150100_150642 </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>Chr1_466905_467447_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>180 </td><td>pISP4_8847_9389_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>pTK3_77560_78102 </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em><em>linF</em>''</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI1_167400_167783_c </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI3_18199_18582 </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td><em>linFb</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>357 </td><td>Chr1_438688_439761_c </td><td>49 (174/350) </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linG</em> </td><td>Acyl-CoA transferase, alpha subunit </td><td>215 </td><td>Chr2_603108-603755 </td><td>215 </td><td>pISP0_152221_152868 </td><td>identical </td><td>99 (647/648) </td><td> </td><td> </td><td> </td><td> </td><td>239 </td><td>pMI1_102652_103371_c </td><td>100 (215/215) </td><td>99 (647/648) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linH</em> </td><td>Acyl-CoA transferase, beta subunit </td><td>212 </td><td>Chr2_603755-604393 </td><td>212 </td><td>pISP0_152868_153506 </td><td>identical </td><td>99 (637/639) </td><td> </td><td> </td><td> </td><td> </td><td>212 </td><td>pMI1_102014_102652_c </td><td>identical </td><td>99 (637/639) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linI</em> </td><td>IclR-family transcriptional regulator </td><td>267 </td><td>Chr2_602168-602971_c </td><td>267 </td><td>pISP0_151281_152084_c </td><td>99 (265/267) </td><td>99 (798/804) </td><td> </td><td> </td><td> </td><td> </td><td>265 </td><td>pMI1_103442_104239 </td><td>99 (263/265) </td><td>99 (798/804) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linJ</em> </td><td>Thiolase </td><td>403 </td><td>Chr2_600921-602132_c </td><td>403 </td><td>pISP0_150034_151245_c </td><td>identical </td><td>99 (1202/1212) </td><td> </td><td> </td><td> </td><td> </td><td>401 </td><td>pMI1_104281_105486 </td><td>99 (400/401) </td><td>99 (1202/1212) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linK</em> </td><td rowspan="2">Putative ABC transporter system, inner membrane protein </td><td rowspan="2">376 </td><td rowspan="2">Chr1_19347-20477 </td><td rowspan="2">366 </td><td rowspan="2">Chr_2358426_2359526 </td><td rowspan="2">65 (239/364) </td><td rowspan="2">83 (526/627) </td><td rowspan="2">316 </td><td rowspan="2">Chr1_40913_41863_c </td><td rowspan="2">95 (301/316) </td><td rowspan="2">88 (1004/11131) </td><td rowspan="2">369 </td><td rowspan="2">Chr1_40248_41357_c </td><td rowspan="2">86 (320/369) </td><td colspan="2">87 (739/840) </td></tr><tr><td><em>linKb</em> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linL</em> </td><td rowspan="2">Putative ABC transporter system, ATPase </td><td>282 </td><td>Chr1_20477-21325 </td><td> </td><td> </td><td> </td><td> </td><td>282 </td><td>Chr1_40065_40913_c </td><td>95 (268/282) </td><td>91 (721/786) </td><td>290 </td><td>Chr1_39376_40248_c </td><td>90 (252/280) </td><td>85 (627/734) </td></tr><tr><td><em>linLb</em> </td><td> </td><td> </td><td>268 </td><td>Chr_2359526_2360332 </td><td>75 (194/258) </td><td>83 (141/168) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linM</em> </td><td rowspan="2">Putative ABC transporter system, periplasmic protein </td><td>320 </td><td>Chr1_21329-22291 </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>Chr1_39099_40061_c </td><td>98 (314/320) </td><td>93 (899/963) </td><td>320 </td><td>Chr1_38410_39372_c </td><td>93 (297/319) </td><td>86 (827/958) </td></tr><tr><td><em>linMb</em> </td><td> </td><td> </td><td>317 </td><td>Chr_2360339_2361292 </td><td>63 (201/319) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linN</em> </td><td rowspan="2">Putative ABC transporter system, lipoprotein </td><td>202 </td><td>Chr1_22299-22907 </td><td> </td><td> </td><td> </td><td> </td><td>204 </td><td>Chr1_38477_39091_c </td><td>93 (190/204) </td><td>88 (538/611) </td><td>203 </td><td>Chr1_37784_38395_c </td><td>81 (168/205) </td><td>82 (384/464) </td></tr><tr><td><em>linNb</em> </td><td> </td><td> </td><td>193 </td><td>Chr_2361298_2361879 </td><td>42 (80/190) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="dsw041-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th rowspan="3">Gene<a reveal-id="dsw041-TF5" data-open="dsw041-TF5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="3">Function </th><th colspan="2">UT26 </th><th colspan="4">MM-1 </th><th colspan="4">TKS </th><th colspan="4">MI1205 </th></tr><tr><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th></tr><tr><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th></tr></thead><tbody><tr><td><em>linA</em> </td><td>Dehydrochlorinase </td><td>156 </td><td>Chr1_1860686-1861156_c </td><td>156 </td><td>pISP1_13547_14017 </td><td>98 (153/156) </td><td>98 (466/471) </td><td>156 </td><td>pTK4_18583_19053 </td><td colspan="2">identical </td><td>156 </td><td>pMI2_260065_260535_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linB</em> </td><td>Halidohydrolase </td><td>296 </td><td>Chr1_1966541-1967431 </td><td>296 </td><td>pISP4_18957_19847_c </td><td>98 (293/296) </td><td>99 (883/891) </td><td>296 </td><td>Chr1_230419_231309 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI1_179020_179910_c </td><td>98 (292/296) </td><td>99 (886/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pTK3_34952_35842 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI3_39099_39989 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pMI3_41282_42172 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linC</em> </td><td>Dehydrogenase </td><td>250 </td><td>Chr1_566609-567361_c </td><td>250 </td><td>pISP1_147367_148119 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>Chr1_469607_470359_c </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI1_169764_170516_c </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pISP4_11370_12122 </td><td colspan="2">identical </td><td>250 </td><td>pTK3_74827_75579 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI3_15466_16218 </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pTK4_21888_22640 </td><td>99 (249/250) </td><td>99 (752/753) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linD</em> </td><td>Reductive dechlorinase </td><td>346 </td><td>pCHQ1_110947-111987_c </td><td>346 </td><td>pISP3_15908_16948 </td><td colspan="2">identical </td><td>346 </td><td>pTK6_21982_23022 </td><td colspan="2">identical </td><td>346 </td><td>pMI1_148490_149530_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI2_214796_215836 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI4_28637_29677 </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linE</em> </td><td rowspan="3">Ring-cleavage dioxygenase </td><td>321 </td><td>pCHQ1_114235-115200_c </td><td>321 </td><td>pISP3_12695_13660 </td><td colspan="2">identical </td><td>321 </td><td>pTK6_18769_19734 </td><td colspan="2">identical </td><td>321 </td><td>pMI1_151777_152742_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>321 </td><td>pMI4_25425_26390 </td><td>identical </td><td>99 (965/966) </td></tr><tr><td><em>linEb</em><a reveal-id="dsw041-TF6" data-open="dsw041-TF6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>320 </td><td>Chr2_564928_565890 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>pMI1_141726_142688_c </td><td>99 (319/320) </td><td>99 (961/963) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linR</em> </td><td>LysR-family transcriptional regulator </td><td>303 </td><td>pCHQ1_115332-116243 </td><td>303 </td><td>pISP3_11652_12563_c </td><td>99 (302/303) </td><td>99 (911/912) </td><td>303 </td><td>pTK6_17726_18637_c </td><td colspan="2">identical </td><td>303 </td><td>pMI1_152874_153785 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI2_210541_211452_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI4_24382_25293_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linF</em> </td><td rowspan="6">Maleylactate reductase </td><td>352 </td><td>Chr2_562332-563390_c </td><td>352 </td><td>pISP0_110260_111318_c </td><td>identical </td><td>98 (1046/1059) </td><td> </td><td> </td><td> </td><td> </td><td>352 </td><td>pMI1_144226_145284 </td><td>identical </td><td>99 (1049/1059) </td></tr><tr><td><em>linF'</em> </td><td> </td><td> </td><td>180 </td><td>pISP1_150100_150642 </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>Chr1_466905_467447_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>180 </td><td>pISP4_8847_9389_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>pTK3_77560_78102 </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em><em>linF</em>''</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI1_167400_167783_c </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI3_18199_18582 </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td><em>linFb</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>357 </td><td>Chr1_438688_439761_c </td><td>49 (174/350) </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linG</em> </td><td>Acyl-CoA transferase, alpha subunit </td><td>215 </td><td>Chr2_603108-603755 </td><td>215 </td><td>pISP0_152221_152868 </td><td>identical </td><td>99 (647/648) </td><td> </td><td> </td><td> </td><td> </td><td>239 </td><td>pMI1_102652_103371_c </td><td>100 (215/215) </td><td>99 (647/648) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linH</em> </td><td>Acyl-CoA transferase, beta subunit </td><td>212 </td><td>Chr2_603755-604393 </td><td>212 </td><td>pISP0_152868_153506 </td><td>identical </td><td>99 (637/639) </td><td> </td><td> </td><td> </td><td> </td><td>212 </td><td>pMI1_102014_102652_c </td><td>identical </td><td>99 (637/639) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linI</em> </td><td>IclR-family transcriptional regulator </td><td>267 </td><td>Chr2_602168-602971_c </td><td>267 </td><td>pISP0_151281_152084_c </td><td>99 (265/267) </td><td>99 (798/804) </td><td> </td><td> </td><td> </td><td> </td><td>265 </td><td>pMI1_103442_104239 </td><td>99 (263/265) </td><td>99 (798/804) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linJ</em> </td><td>Thiolase </td><td>403 </td><td>Chr2_600921-602132_c </td><td>403 </td><td>pISP0_150034_151245_c </td><td>identical </td><td>99 (1202/1212) </td><td> </td><td> </td><td> </td><td> </td><td>401 </td><td>pMI1_104281_105486 </td><td>99 (400/401) </td><td>99 (1202/1212) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linK</em> </td><td rowspan="2">Putative ABC transporter system, inner membrane protein </td><td rowspan="2">376 </td><td rowspan="2">Chr1_19347-20477 </td><td rowspan="2">366 </td><td rowspan="2">Chr_2358426_2359526 </td><td rowspan="2">65 (239/364) </td><td rowspan="2">83 (526/627) </td><td rowspan="2">316 </td><td rowspan="2">Chr1_40913_41863_c </td><td rowspan="2">95 (301/316) </td><td rowspan="2">88 (1004/11131) </td><td rowspan="2">369 </td><td rowspan="2">Chr1_40248_41357_c </td><td rowspan="2">86 (320/369) </td><td colspan="2">87 (739/840) </td></tr><tr><td><em>linKb</em> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linL</em> </td><td rowspan="2">Putative ABC transporter system, ATPase </td><td>282 </td><td>Chr1_20477-21325 </td><td> </td><td> </td><td> </td><td> </td><td>282 </td><td>Chr1_40065_40913_c </td><td>95 (268/282) </td><td>91 (721/786) </td><td>290 </td><td>Chr1_39376_40248_c </td><td>90 (252/280) </td><td>85 (627/734) </td></tr><tr><td><em>linLb</em> </td><td> </td><td> </td><td>268 </td><td>Chr_2359526_2360332 </td><td>75 (194/258) </td><td>83 (141/168) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linM</em> </td><td rowspan="2">Putative ABC transporter system, periplasmic protein </td><td>320 </td><td>Chr1_21329-22291 </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>Chr1_39099_40061_c </td><td>98 (314/320) </td><td>93 (899/963) </td><td>320 </td><td>Chr1_38410_39372_c </td><td>93 (297/319) </td><td>86 (827/958) </td></tr><tr><td><em>linMb</em> </td><td> </td><td> </td><td>317 </td><td>Chr_2360339_2361292 </td><td>63 (201/319) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linN</em> </td><td rowspan="2">Putative ABC transporter system, lipoprotein </td><td>202 </td><td>Chr1_22299-22907 </td><td> </td><td> </td><td> </td><td> </td><td>204 </td><td>Chr1_38477_39091_c </td><td>93 (190/204) </td><td>88 (538/611) </td><td>203 </td><td>Chr1_37784_38395_c </td><td>81 (168/205) </td><td>82 (384/464) </td></tr><tr><td><em>linNb</em> </td><td> </td><td> </td><td>193 </td><td>Chr_2361298_2361879 </td><td>42 (80/190) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw041-TF4"><div class="fn"><p><sup>a</sup>c, encoding on complementary strand.</p></div></div><div content-id="dsw041-TF5"><div class="fn"><p><sup>b</sup><em>linF′</em> and <em>linF″</em>" are probably pseudogenes.</p></div></div><div content-id="dsw041-TF6"><div class="fn"><p><sup>c</sup><em>linEb</em> is homologue of PcpA (96% identity) and partially involved in γ-HCH degradation in UT26.<a reveal-id="dsw041-B34" data-open="dsw041-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p></div></div></div></div></div><div class="table-wrap"><div id="dsw041-T2" data-id="dsw041-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p><em>lin</em> genes of four γ−HCH-degrading sphingomonad strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th rowspan="3">Gene<a reveal-id="dsw041-TF5" data-open="dsw041-TF5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="3">Function </th><th colspan="2">UT26 </th><th colspan="4">MM-1 </th><th colspan="4">TKS </th><th colspan="4">MI1205 </th></tr><tr><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th></tr><tr><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th></tr></thead><tbody><tr><td><em>linA</em> </td><td>Dehydrochlorinase </td><td>156 </td><td>Chr1_1860686-1861156_c </td><td>156 </td><td>pISP1_13547_14017 </td><td>98 (153/156) </td><td>98 (466/471) </td><td>156 </td><td>pTK4_18583_19053 </td><td colspan="2">identical </td><td>156 </td><td>pMI2_260065_260535_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linB</em> </td><td>Halidohydrolase </td><td>296 </td><td>Chr1_1966541-1967431 </td><td>296 </td><td>pISP4_18957_19847_c </td><td>98 (293/296) </td><td>99 (883/891) </td><td>296 </td><td>Chr1_230419_231309 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI1_179020_179910_c </td><td>98 (292/296) </td><td>99 (886/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pTK3_34952_35842 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI3_39099_39989 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pMI3_41282_42172 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linC</em> </td><td>Dehydrogenase </td><td>250 </td><td>Chr1_566609-567361_c </td><td>250 </td><td>pISP1_147367_148119 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>Chr1_469607_470359_c </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI1_169764_170516_c </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pISP4_11370_12122 </td><td colspan="2">identical </td><td>250 </td><td>pTK3_74827_75579 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI3_15466_16218 </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pTK4_21888_22640 </td><td>99 (249/250) </td><td>99 (752/753) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linD</em> </td><td>Reductive dechlorinase </td><td>346 </td><td>pCHQ1_110947-111987_c </td><td>346 </td><td>pISP3_15908_16948 </td><td colspan="2">identical </td><td>346 </td><td>pTK6_21982_23022 </td><td colspan="2">identical </td><td>346 </td><td>pMI1_148490_149530_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI2_214796_215836 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI4_28637_29677 </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linE</em> </td><td rowspan="3">Ring-cleavage dioxygenase </td><td>321 </td><td>pCHQ1_114235-115200_c </td><td>321 </td><td>pISP3_12695_13660 </td><td colspan="2">identical </td><td>321 </td><td>pTK6_18769_19734 </td><td colspan="2">identical </td><td>321 </td><td>pMI1_151777_152742_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>321 </td><td>pMI4_25425_26390 </td><td>identical </td><td>99 (965/966) </td></tr><tr><td><em>linEb</em><a reveal-id="dsw041-TF6" data-open="dsw041-TF6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>320 </td><td>Chr2_564928_565890 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>pMI1_141726_142688_c </td><td>99 (319/320) </td><td>99 (961/963) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linR</em> </td><td>LysR-family transcriptional regulator </td><td>303 </td><td>pCHQ1_115332-116243 </td><td>303 </td><td>pISP3_11652_12563_c </td><td>99 (302/303) </td><td>99 (911/912) </td><td>303 </td><td>pTK6_17726_18637_c </td><td colspan="2">identical </td><td>303 </td><td>pMI1_152874_153785 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI2_210541_211452_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI4_24382_25293_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linF</em> </td><td rowspan="6">Maleylactate reductase </td><td>352 </td><td>Chr2_562332-563390_c </td><td>352 </td><td>pISP0_110260_111318_c </td><td>identical </td><td>98 (1046/1059) </td><td> </td><td> </td><td> </td><td> </td><td>352 </td><td>pMI1_144226_145284 </td><td>identical </td><td>99 (1049/1059) </td></tr><tr><td><em>linF'</em> </td><td> </td><td> </td><td>180 </td><td>pISP1_150100_150642 </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>Chr1_466905_467447_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>180 </td><td>pISP4_8847_9389_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>pTK3_77560_78102 </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em><em>linF</em>''</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI1_167400_167783_c </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI3_18199_18582 </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td><em>linFb</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>357 </td><td>Chr1_438688_439761_c </td><td>49 (174/350) </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linG</em> </td><td>Acyl-CoA transferase, alpha subunit </td><td>215 </td><td>Chr2_603108-603755 </td><td>215 </td><td>pISP0_152221_152868 </td><td>identical </td><td>99 (647/648) </td><td> </td><td> </td><td> </td><td> </td><td>239 </td><td>pMI1_102652_103371_c </td><td>100 (215/215) </td><td>99 (647/648) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linH</em> </td><td>Acyl-CoA transferase, beta subunit </td><td>212 </td><td>Chr2_603755-604393 </td><td>212 </td><td>pISP0_152868_153506 </td><td>identical </td><td>99 (637/639) </td><td> </td><td> </td><td> </td><td> </td><td>212 </td><td>pMI1_102014_102652_c </td><td>identical </td><td>99 (637/639) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linI</em> </td><td>IclR-family transcriptional regulator </td><td>267 </td><td>Chr2_602168-602971_c </td><td>267 </td><td>pISP0_151281_152084_c </td><td>99 (265/267) </td><td>99 (798/804) </td><td> </td><td> </td><td> </td><td> </td><td>265 </td><td>pMI1_103442_104239 </td><td>99 (263/265) </td><td>99 (798/804) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linJ</em> </td><td>Thiolase </td><td>403 </td><td>Chr2_600921-602132_c </td><td>403 </td><td>pISP0_150034_151245_c </td><td>identical </td><td>99 (1202/1212) </td><td> </td><td> </td><td> </td><td> </td><td>401 </td><td>pMI1_104281_105486 </td><td>99 (400/401) </td><td>99 (1202/1212) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linK</em> </td><td rowspan="2">Putative ABC transporter system, inner membrane protein </td><td rowspan="2">376 </td><td rowspan="2">Chr1_19347-20477 </td><td rowspan="2">366 </td><td rowspan="2">Chr_2358426_2359526 </td><td rowspan="2">65 (239/364) </td><td rowspan="2">83 (526/627) </td><td rowspan="2">316 </td><td rowspan="2">Chr1_40913_41863_c </td><td rowspan="2">95 (301/316) </td><td rowspan="2">88 (1004/11131) </td><td rowspan="2">369 </td><td rowspan="2">Chr1_40248_41357_c </td><td rowspan="2">86 (320/369) </td><td colspan="2">87 (739/840) </td></tr><tr><td><em>linKb</em> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linL</em> </td><td rowspan="2">Putative ABC transporter system, ATPase </td><td>282 </td><td>Chr1_20477-21325 </td><td> </td><td> </td><td> </td><td> </td><td>282 </td><td>Chr1_40065_40913_c </td><td>95 (268/282) </td><td>91 (721/786) </td><td>290 </td><td>Chr1_39376_40248_c </td><td>90 (252/280) </td><td>85 (627/734) </td></tr><tr><td><em>linLb</em> </td><td> </td><td> </td><td>268 </td><td>Chr_2359526_2360332 </td><td>75 (194/258) </td><td>83 (141/168) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linM</em> </td><td rowspan="2">Putative ABC transporter system, periplasmic protein </td><td>320 </td><td>Chr1_21329-22291 </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>Chr1_39099_40061_c </td><td>98 (314/320) </td><td>93 (899/963) </td><td>320 </td><td>Chr1_38410_39372_c </td><td>93 (297/319) </td><td>86 (827/958) </td></tr><tr><td><em>linMb</em> </td><td> </td><td> </td><td>317 </td><td>Chr_2360339_2361292 </td><td>63 (201/319) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linN</em> </td><td rowspan="2">Putative ABC transporter system, lipoprotein </td><td>202 </td><td>Chr1_22299-22907 </td><td> </td><td> </td><td> </td><td> </td><td>204 </td><td>Chr1_38477_39091_c </td><td>93 (190/204) </td><td>88 (538/611) </td><td>203 </td><td>Chr1_37784_38395_c </td><td>81 (168/205) </td><td>82 (384/464) </td></tr><tr><td><em>linNb</em> </td><td> </td><td> </td><td>193 </td><td>Chr_2361298_2361879 </td><td>42 (80/190) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="dsw041-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th rowspan="3">Gene<a reveal-id="dsw041-TF5" data-open="dsw041-TF5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th rowspan="3">Function </th><th colspan="2">UT26 </th><th colspan="4">MM-1 </th><th colspan="4">TKS </th><th colspan="4">MI1205 </th></tr><tr><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th><th rowspan="2">A.A. residues </th><th rowspan="2">Location<a reveal-id="dsw041-TF4" data-open="dsw041-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="2">Identity (%) to that of UT26 </th></tr><tr><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th><th>A.A. </th><th>Nucleotide </th></tr></thead><tbody><tr><td><em>linA</em> </td><td>Dehydrochlorinase </td><td>156 </td><td>Chr1_1860686-1861156_c </td><td>156 </td><td>pISP1_13547_14017 </td><td>98 (153/156) </td><td>98 (466/471) </td><td>156 </td><td>pTK4_18583_19053 </td><td colspan="2">identical </td><td>156 </td><td>pMI2_260065_260535_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linB</em> </td><td>Halidohydrolase </td><td>296 </td><td>Chr1_1966541-1967431 </td><td>296 </td><td>pISP4_18957_19847_c </td><td>98 (293/296) </td><td>99 (883/891) </td><td>296 </td><td>Chr1_230419_231309 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI1_179020_179910_c </td><td>98 (292/296) </td><td>99 (886/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pTK3_34952_35842 </td><td>97 (290/296) </td><td>99 (883/891) </td><td>296 </td><td>pMI3_39099_39989 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>296 </td><td>pMI3_41282_42172 </td><td>97 (289/296) </td><td>99 (883/891) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linC</em> </td><td>Dehydrogenase </td><td>250 </td><td>Chr1_566609-567361_c </td><td>250 </td><td>pISP1_147367_148119 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>Chr1_469607_470359_c </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI1_169764_170516_c </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pISP4_11370_12122 </td><td colspan="2">identical </td><td>250 </td><td>pTK3_74827_75579 </td><td>99 (249/250) </td><td>99 (752/753) </td><td>250 </td><td>pMI3_15466_16218 </td><td>99 (249/250) </td><td>99 (752/753) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>250 </td><td>pTK4_21888_22640 </td><td>99 (249/250) </td><td>99 (752/753) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linD</em> </td><td>Reductive dechlorinase </td><td>346 </td><td>pCHQ1_110947-111987_c </td><td>346 </td><td>pISP3_15908_16948 </td><td colspan="2">identical </td><td>346 </td><td>pTK6_21982_23022 </td><td colspan="2">identical </td><td>346 </td><td>pMI1_148490_149530_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI2_214796_215836 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>346 </td><td>pMI4_28637_29677 </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linE</em> </td><td rowspan="3">Ring-cleavage dioxygenase </td><td>321 </td><td>pCHQ1_114235-115200_c </td><td>321 </td><td>pISP3_12695_13660 </td><td colspan="2">identical </td><td>321 </td><td>pTK6_18769_19734 </td><td colspan="2">identical </td><td>321 </td><td>pMI1_151777_152742_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>321 </td><td>pMI4_25425_26390 </td><td>identical </td><td>99 (965/966) </td></tr><tr><td><em>linEb</em><a reveal-id="dsw041-TF6" data-open="dsw041-TF6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>320 </td><td>Chr2_564928_565890 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>pMI1_141726_142688_c </td><td>99 (319/320) </td><td>99 (961/963) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linR</em> </td><td>LysR-family transcriptional regulator </td><td>303 </td><td>pCHQ1_115332-116243 </td><td>303 </td><td>pISP3_11652_12563_c </td><td>99 (302/303) </td><td>99 (911/912) </td><td>303 </td><td>pTK6_17726_18637_c </td><td colspan="2">identical </td><td>303 </td><td>pMI1_152874_153785 </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI2_210541_211452_c </td><td colspan="2">identical </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>303 </td><td>pMI4_24382_25293_c </td><td colspan="2">identical </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linF</em> </td><td rowspan="6">Maleylactate reductase </td><td>352 </td><td>Chr2_562332-563390_c </td><td>352 </td><td>pISP0_110260_111318_c </td><td>identical </td><td>98 (1046/1059) </td><td> </td><td> </td><td> </td><td> </td><td>352 </td><td>pMI1_144226_145284 </td><td>identical </td><td>99 (1049/1059) </td></tr><tr><td><em>linF'</em> </td><td> </td><td> </td><td>180 </td><td>pISP1_150100_150642 </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>Chr1_466905_467447_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>180 </td><td>pISP4_8847_9389_c </td><td>98 (176/178) </td><td>99 (526/531) </td><td>180 </td><td>pTK3_77560_78102 </td><td>98 (176/178) </td><td>99 (526/531) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em><em>linF</em>''</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI1_167400_167783_c </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>127 </td><td>pMI3_18199_18582 </td><td>100 (122/122) </td><td>98 (366/371) </td></tr><tr><td><em>linFb</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>357 </td><td>Chr1_438688_439761_c </td><td>49 (174/350) </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linG</em> </td><td>Acyl-CoA transferase, alpha subunit </td><td>215 </td><td>Chr2_603108-603755 </td><td>215 </td><td>pISP0_152221_152868 </td><td>identical </td><td>99 (647/648) </td><td> </td><td> </td><td> </td><td> </td><td>239 </td><td>pMI1_102652_103371_c </td><td>100 (215/215) </td><td>99 (647/648) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linH</em> </td><td>Acyl-CoA transferase, beta subunit </td><td>212 </td><td>Chr2_603755-604393 </td><td>212 </td><td>pISP0_152868_153506 </td><td>identical </td><td>99 (637/639) </td><td> </td><td> </td><td> </td><td> </td><td>212 </td><td>pMI1_102014_102652_c </td><td>identical </td><td>99 (637/639) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linI</em> </td><td>IclR-family transcriptional regulator </td><td>267 </td><td>Chr2_602168-602971_c </td><td>267 </td><td>pISP0_151281_152084_c </td><td>99 (265/267) </td><td>99 (798/804) </td><td> </td><td> </td><td> </td><td> </td><td>265 </td><td>pMI1_103442_104239 </td><td>99 (263/265) </td><td>99 (798/804) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linJ</em> </td><td>Thiolase </td><td>403 </td><td>Chr2_600921-602132_c </td><td>403 </td><td>pISP0_150034_151245_c </td><td>identical </td><td>99 (1202/1212) </td><td> </td><td> </td><td> </td><td> </td><td>401 </td><td>pMI1_104281_105486 </td><td>99 (400/401) </td><td>99 (1202/1212) </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linK</em> </td><td rowspan="2">Putative ABC transporter system, inner membrane protein </td><td rowspan="2">376 </td><td rowspan="2">Chr1_19347-20477 </td><td rowspan="2">366 </td><td rowspan="2">Chr_2358426_2359526 </td><td rowspan="2">65 (239/364) </td><td rowspan="2">83 (526/627) </td><td rowspan="2">316 </td><td rowspan="2">Chr1_40913_41863_c </td><td rowspan="2">95 (301/316) </td><td rowspan="2">88 (1004/11131) </td><td rowspan="2">369 </td><td rowspan="2">Chr1_40248_41357_c </td><td rowspan="2">86 (320/369) </td><td colspan="2">87 (739/840) </td></tr><tr><td><em>linKb</em> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linL</em> </td><td rowspan="2">Putative ABC transporter system, ATPase </td><td>282 </td><td>Chr1_20477-21325 </td><td> </td><td> </td><td> </td><td> </td><td>282 </td><td>Chr1_40065_40913_c </td><td>95 (268/282) </td><td>91 (721/786) </td><td>290 </td><td>Chr1_39376_40248_c </td><td>90 (252/280) </td><td>85 (627/734) </td></tr><tr><td><em>linLb</em> </td><td> </td><td> </td><td>268 </td><td>Chr_2359526_2360332 </td><td>75 (194/258) </td><td>83 (141/168) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linM</em> </td><td rowspan="2">Putative ABC transporter system, periplasmic protein </td><td>320 </td><td>Chr1_21329-22291 </td><td> </td><td> </td><td> </td><td> </td><td>320 </td><td>Chr1_39099_40061_c </td><td>98 (314/320) </td><td>93 (899/963) </td><td>320 </td><td>Chr1_38410_39372_c </td><td>93 (297/319) </td><td>86 (827/958) </td></tr><tr><td><em>linMb</em> </td><td> </td><td> </td><td>317 </td><td>Chr_2360339_2361292 </td><td>63 (201/319) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="16"> </td></tr><tr><td><em>linN</em> </td><td rowspan="2">Putative ABC transporter system, lipoprotein </td><td>202 </td><td>Chr1_22299-22907 </td><td> </td><td> </td><td> </td><td> </td><td>204 </td><td>Chr1_38477_39091_c </td><td>93 (190/204) </td><td>88 (538/611) </td><td>203 </td><td>Chr1_37784_38395_c </td><td>81 (168/205) </td><td>82 (384/464) </td></tr><tr><td><em>linNb</em> </td><td> </td><td> </td><td>193 </td><td>Chr_2361298_2361879 </td><td>42 (80/190) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw041-TF4"><div class="fn"><p><sup>a</sup>c, encoding on complementary strand.</p></div></div><div content-id="dsw041-TF5"><div class="fn"><p><sup>b</sup><em>linF′</em> and <em>linF″</em>" are probably pseudogenes.</p></div></div><div content-id="dsw041-TF6"><div class="fn"><p><sup>c</sup><em>linEb</em> is homologue of PcpA (96% identity) and partially involved in γ-HCH degradation in UT26.<a reveal-id="dsw041-B34" data-open="dsw041-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p></div></div></div></div></p><p>The <em>linKLMN</em><sub>UT26</sub> homologues have been found in various bacterial strains<a reveal-id="dsw041-B16" data-open="dsw041-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw041-B29" data-open="dsw041-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and were also found in the MM-1, MI1205, and TKS genomes (<a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>). However, their similarities to <em>linKLMN</em><sub>UT26</sub> are lower (&lt;93% identity at DNA level; <a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>) than in the case of the other <em>lin</em> genes mentioned earlier, and this divergence roughly reflects the phylogenetic relationship of their hosts (<a reveal-id="dsw041-F2" data-open="dsw041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), suggesting that the <em>linKLMN</em> system is one of the inherent functions necessary for γ-HCH utilization in sphingomonads. In particular, the <em>linKLMN</em> homologues of MM-1, which is phylogenetically the most distant strain from UT26 (<a reveal-id="dsw041-F2" data-open="dsw041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), show a relatively low similarity with <em>linKLMN</em><sub>UT26</sub> (42–75% identities at the amino acid level; <a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), and they were designated <em>linKbLbMbNb</em>. To confirm their function for γ-HCH utilization, we attempted to disrupt the <em>linKbLbMbNb</em> gene cluster in MM-1. However, such an expected disruptant could not be constructed because unknown DNA rearrangements often occurred in MM-1. As an alternative confirmation, a plasmid containing the <em>linKbLbMbNb</em><sub>MM-1</sub> gene cluster was introduced into RE1, a <em>linKLMN</em><sub>UT26</sub> disruptant of UT26.<a reveal-id="dsw041-B29" data-open="dsw041-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The growth of RE1 in 1/3LB medium was inhibited by the addition of γ-HCH (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6A</a></span>),<a reveal-id="dsw041-B29" data-open="dsw041-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and this inhibition was suppressed by the supply of the <em>linKbLbMbNb</em> cluster (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6B</a></span>) as well as the <em>linKLMN</em><sub>UT26</sub> one (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6C</a></span>),<a reveal-id="dsw041-B29" data-open="dsw041-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> strongly suggesting that the both clusters function in the same way for the γ-HCH utilization. These results supported our hypothesis that the <em>linKLMN</em> system is one of the inherent functions necessary for γ-HCH utilization in sphingomonads. However, the possibility cannot be excluded that other functional homologue(s) of <em>linKLMN</em> system exist in MM-1.</p><p>All of the four strains carry almost identical <em>linA</em> to <em>linE</em> genes (designated ‘specific’ <em>lin</em> genes) (<a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>), suggesting they acquired such genes by lateral gene transfer. However, the specific <em>lin</em> genes are dispersed on multiple replicons in the four strains (<a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>). In UT26, <em>linA</em> to <em>linC</em> are located on Chr1, and only the <em>linRED</em> cluster is located on a plasmid. On the other hands, all the specific <em>lin</em> genes are dispersed on multiple plasmids with various combinations in other three strains, although additional copies of <em>linB</em> and <em>linC</em> are also located on Chr1 in TKS (<a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Furthermore, replicon types of such plasmids carrying the specific <em>lin</em> genes are various (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>). These observations indicate that these four strains did not simply acquire all the specific <em>lin</em> genes at once as a cluster. This contrasts with other aromatic compound-degrading strains, which can acquire a whole set of responsible genes by the conjugative transfer of plasmids and/or integrative and conjugative elements.<a reveal-id="dsw041-B58 dsw041-B59 dsw041-B60" data-open="dsw041-B58 dsw041-B59 dsw041-B60" class="link link-ref link-reveal xref-bibr"><sup>58–60</sup></a><div content-id="dsw041-T3" class="table-modal"><div class="table-wrap"><div id="dsw041-T3" data-id="dsw041-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Classification of sphingomonad plasmids and plasmid-type replicons</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th colspan="2" rowspan="2">Replicon<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Size (bp) </th><th rowspan="2">Host </th><th rowspan="2">Host feature </th><th colspan="3">RepA protein </th><th>Type </th><th>Gene cluster for conjugation </th><th>Genes for degradation </th><th>Accession number </th><th>Reference </th></tr><tr><th>Location </th><th>A.A. </th><th>Identity (%) to representative<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td colspan="2"><strong>UT26_Chr2</strong> </td><td char=".">681,892 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1203 </td><td char=".">400 </td><td>— </td><td><em>repABC</em> </td><td> </td><td><em>linF, linGHIJ</em> </td><td>AP010804 </td><td char=".">18 </td></tr><tr><td colspan="2"> L-1_Chr2 </td><td char=".">1,368,670 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>321340_322506 </td><td char=".">388 </td><td char=".">89 (359/400) </td><td><em>repABC</em> </td><td> </td><td><em>pcpBDR, pcpEMAC</em> </td><td>CP002799 </td><td char=".">11 </td></tr><tr><td colspan="2"> TKS_Chr2 </td><td char=".">989,120 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1206 </td><td char=".">401 </td><td char=".">89 (360/401) </td><td><em>repABC</em> </td><td> </td><td><em>linGHIJ_</em>homologue </td><td>CP005084 </td><td char=".">26 </td></tr><tr><td colspan="2"> MI_Chr2 </td><td char=".">567,154 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1209 </td><td char=".">402 </td><td char=".">83 (338/405) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP005189 </td><td char=".">25 </td></tr><tr><td colspan="2"> pNL2 </td><td char=".">487,268 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>209439_210644_c </td><td char=".">401 </td><td char=".">70 (284/401) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP000677 </td><td>unpublished </td></tr><tr><td colspan="2"> Lpl </td><td char=".">192,103 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>88922_90199 </td><td char=".">425 </td><td char=".">67 (288/425) </td><td><em>repABC</em> </td><td> </td><td> </td><td>FR856860 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>114750_115880_c </td><td char=".">376 </td><td char=".">55 (201/363) </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> TKS_Chr1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">4,249,857 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>252492_253622_c </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>CP005083 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>183355_184485 </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI1 </td><td char=".">292,135 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1224 </td><td char=".">407 </td><td char=".">42 (174/409) </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td>CP005190 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>312340_313539 </td><td char=".">399 </td><td char=".">42 (171/401) </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pCHQ1</strong> </td><td char=".">190,974 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>— </td><td><em>repABC</em> </td><td>Ti, F </td><td><em>linRED</em> </td><td>AP010805 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="2"> pSLGP </td><td char=".">148,801 </td><td><em>Sphingobium</em> sp. SYK-6 </td><td>lignin degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>AP012223 </td><td><a reveal-id="dsw041-B81" data-open="dsw041-B81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td colspan="2"> pSPHCH01 </td><td char=".">123,733 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>47083_48171_c </td><td char=".">362<a reveal-id="dsw041-TF9" data-open="dsw041-TF9" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>CP002800 </td><td char=".">11 </td></tr><tr><td colspan="2"> pISP0 </td><td char=".">275,840 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>4081_5244 </td><td char=".">387 </td><td char=".">98 (381/387) </td><td><em>repABC</em> </td><td>Ti </td><td><em>linF, linGHIJ</em> </td><td>CP004037 </td><td char=".">23 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT1</strong> </td><td char=".">31,776 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010806 </td><td char=".">18 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pISP2 </td><td char=".">53,841 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP004039 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP3</strong> </td><td char=".">43,776 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>37217_38329 </td><td char=".">370 </td><td>— </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP004040 </td><td char=".">23 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK6 </td><td char=".">34,300 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005090 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_1</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_960 </td><td char=".">319 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_2</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>52874_53821_c </td><td char=".">315 </td><td>— </td><td>iteron </td><td> </td><td><em>linB</em>, <em>linC</em>, <em>linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP4</strong> </td><td char=".">33,183 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>101_1012 </td><td char=".">303 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP004041 </td><td char=".">23 </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>158477_159388_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>19263_20174_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>56463_57374 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>30095_31006_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>Ti </td><td><em>linB, linC, linF''</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>18420_19331 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK2</strong> </td><td char=".">195,308 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1122 </td><td char=".">373 </td><td>— </td><td>iteron </td><td>Ti </td><td> </td><td>CP005086 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK7</strong> </td><td char=".">9,585 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1107 </td><td char=".">368 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005091 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK8</strong> </td><td char=".">7,223 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1032 </td><td char=".">343 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005092 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pNL1</strong> </td><td char=".">184,462 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>86362_87666 </td><td char=".">434 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP000676 </td><td>unpublished </td></tr><tr><td colspan="2"> pCAR3 </td><td char=".">254,797 </td><td><em>Novosphigobium</em> sp. KA1 </td><td>carbazole degradation </td><td>200594_201898_c </td><td char=".">434 </td><td char=".">91 (398/433) </td><td>iteron </td><td> </td><td> </td><td>AB270530 </td><td><a reveal-id="dsw041-B82" data-open="dsw041-B82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td colspan="2"> Mpl </td><td char=".">1,161,602 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>513635_514939_c </td><td char=".">434 </td><td char=".">85 (372/434) </td><td>iteron </td><td> </td><td> </td><td>FR856861 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pSWIT02 </td><td char=".">222,757 </td><td><em>Sphingomonas wittichii</em> RW1 </td><td>dioxin degradation </td><td>63589_64893 </td><td char=".">434 </td><td char=".">82 (356/429) </td><td>iteron </td><td> </td><td> </td><td>CP000701 </td><td><a reveal-id="dsw041-B83" data-open="dsw041-B83" class="link link-ref link-reveal xref-bibr">83</a> </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pLB1</strong> </td><td char=".">65,998 </td><td>unidentifed soil bacterium (<em>S. japonicum</em> UT26) </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>— </td><td> </td><td>Ti </td><td><em>linB</em> </td><td>AB244976 </td><td><a reveal-id="dsw041-B74" data-open="dsw041-B74" class="link link-ref link-reveal xref-bibr">74</a> </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>identical </td><td> </td><td>Ti </td><td><em>linB, linC, linF'</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pLA2 </td><td char=".">62,341 </td><td><em>Novosphingobium pentaromativorans</em> US6-1 </td><td>benzo(a)pyrene degradation </td><td>40543_41325 </td><td char=".">260 </td><td char=".">98 (256/260) </td><td> </td><td>Ti </td><td> </td><td>AGFM01000123 </td><td><a reveal-id="dsw041-B84" data-open="dsw041-B84" class="link link-ref link-reveal xref-bibr">84</a> </td></tr><tr><td colspan="2"> pTK5 </td><td char=".">53,908 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td char=".">97 (253/260) </td><td> </td><td>Ti </td><td> </td><td>CP005089 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT2</strong> </td><td char=".">5,398 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010807 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK9 </td><td char=".">5,391 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP005093 </td><td char=".">26 </td></tr></tbody></table></div><div content-id="dsw041-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th colspan="2" rowspan="2">Replicon<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Size (bp) </th><th rowspan="2">Host </th><th rowspan="2">Host feature </th><th colspan="3">RepA protein </th><th>Type </th><th>Gene cluster for conjugation </th><th>Genes for degradation </th><th>Accession number </th><th>Reference </th></tr><tr><th>Location </th><th>A.A. </th><th>Identity (%) to representative<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td colspan="2"><strong>UT26_Chr2</strong> </td><td char=".">681,892 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1203 </td><td char=".">400 </td><td>— </td><td><em>repABC</em> </td><td> </td><td><em>linF, linGHIJ</em> </td><td>AP010804 </td><td char=".">18 </td></tr><tr><td colspan="2"> L-1_Chr2 </td><td char=".">1,368,670 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>321340_322506 </td><td char=".">388 </td><td char=".">89 (359/400) </td><td><em>repABC</em> </td><td> </td><td><em>pcpBDR, pcpEMAC</em> </td><td>CP002799 </td><td char=".">11 </td></tr><tr><td colspan="2"> TKS_Chr2 </td><td char=".">989,120 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1206 </td><td char=".">401 </td><td char=".">89 (360/401) </td><td><em>repABC</em> </td><td> </td><td><em>linGHIJ_</em>homologue </td><td>CP005084 </td><td char=".">26 </td></tr><tr><td colspan="2"> MI_Chr2 </td><td char=".">567,154 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1209 </td><td char=".">402 </td><td char=".">83 (338/405) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP005189 </td><td char=".">25 </td></tr><tr><td colspan="2"> pNL2 </td><td char=".">487,268 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>209439_210644_c </td><td char=".">401 </td><td char=".">70 (284/401) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP000677 </td><td>unpublished </td></tr><tr><td colspan="2"> Lpl </td><td char=".">192,103 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>88922_90199 </td><td char=".">425 </td><td char=".">67 (288/425) </td><td><em>repABC</em> </td><td> </td><td> </td><td>FR856860 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>114750_115880_c </td><td char=".">376 </td><td char=".">55 (201/363) </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> TKS_Chr1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">4,249,857 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>252492_253622_c </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>CP005083 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>183355_184485 </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI1 </td><td char=".">292,135 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1224 </td><td char=".">407 </td><td char=".">42 (174/409) </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td>CP005190 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>312340_313539 </td><td char=".">399 </td><td char=".">42 (171/401) </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pCHQ1</strong> </td><td char=".">190,974 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>— </td><td><em>repABC</em> </td><td>Ti, F </td><td><em>linRED</em> </td><td>AP010805 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="2"> pSLGP </td><td char=".">148,801 </td><td><em>Sphingobium</em> sp. SYK-6 </td><td>lignin degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>AP012223 </td><td><a reveal-id="dsw041-B81" data-open="dsw041-B81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td colspan="2"> pSPHCH01 </td><td char=".">123,733 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>47083_48171_c </td><td char=".">362<a reveal-id="dsw041-TF9" data-open="dsw041-TF9" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>CP002800 </td><td char=".">11 </td></tr><tr><td colspan="2"> pISP0 </td><td char=".">275,840 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>4081_5244 </td><td char=".">387 </td><td char=".">98 (381/387) </td><td><em>repABC</em> </td><td>Ti </td><td><em>linF, linGHIJ</em> </td><td>CP004037 </td><td char=".">23 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT1</strong> </td><td char=".">31,776 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010806 </td><td char=".">18 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pISP2 </td><td char=".">53,841 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP004039 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP3</strong> </td><td char=".">43,776 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>37217_38329 </td><td char=".">370 </td><td>— </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP004040 </td><td char=".">23 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK6 </td><td char=".">34,300 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005090 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_1</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_960 </td><td char=".">319 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_2</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>52874_53821_c </td><td char=".">315 </td><td>— </td><td>iteron </td><td> </td><td><em>linB</em>, <em>linC</em>, <em>linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP4</strong> </td><td char=".">33,183 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>101_1012 </td><td char=".">303 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP004041 </td><td char=".">23 </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>158477_159388_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>19263_20174_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>56463_57374 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>30095_31006_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>Ti </td><td><em>linB, linC, linF''</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>18420_19331 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK2</strong> </td><td char=".">195,308 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1122 </td><td char=".">373 </td><td>— </td><td>iteron </td><td>Ti </td><td> </td><td>CP005086 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK7</strong> </td><td char=".">9,585 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1107 </td><td char=".">368 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005091 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK8</strong> </td><td char=".">7,223 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1032 </td><td char=".">343 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005092 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pNL1</strong> </td><td char=".">184,462 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>86362_87666 </td><td char=".">434 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP000676 </td><td>unpublished </td></tr><tr><td colspan="2"> pCAR3 </td><td char=".">254,797 </td><td><em>Novosphigobium</em> sp. KA1 </td><td>carbazole degradation </td><td>200594_201898_c </td><td char=".">434 </td><td char=".">91 (398/433) </td><td>iteron </td><td> </td><td> </td><td>AB270530 </td><td><a reveal-id="dsw041-B82" data-open="dsw041-B82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td colspan="2"> Mpl </td><td char=".">1,161,602 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>513635_514939_c </td><td char=".">434 </td><td char=".">85 (372/434) </td><td>iteron </td><td> </td><td> </td><td>FR856861 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pSWIT02 </td><td char=".">222,757 </td><td><em>Sphingomonas wittichii</em> RW1 </td><td>dioxin degradation </td><td>63589_64893 </td><td char=".">434 </td><td char=".">82 (356/429) </td><td>iteron </td><td> </td><td> </td><td>CP000701 </td><td><a reveal-id="dsw041-B83" data-open="dsw041-B83" class="link link-ref link-reveal xref-bibr">83</a> </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pLB1</strong> </td><td char=".">65,998 </td><td>unidentifed soil bacterium (<em>S. japonicum</em> UT26) </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>— </td><td> </td><td>Ti </td><td><em>linB</em> </td><td>AB244976 </td><td><a reveal-id="dsw041-B74" data-open="dsw041-B74" class="link link-ref link-reveal xref-bibr">74</a> </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>identical </td><td> </td><td>Ti </td><td><em>linB, linC, linF'</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pLA2 </td><td char=".">62,341 </td><td><em>Novosphingobium pentaromativorans</em> US6-1 </td><td>benzo(a)pyrene degradation </td><td>40543_41325 </td><td char=".">260 </td><td char=".">98 (256/260) </td><td> </td><td>Ti </td><td> </td><td>AGFM01000123 </td><td><a reveal-id="dsw041-B84" data-open="dsw041-B84" class="link link-ref link-reveal xref-bibr">84</a> </td></tr><tr><td colspan="2"> pTK5 </td><td char=".">53,908 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td char=".">97 (253/260) </td><td> </td><td>Ti </td><td> </td><td>CP005089 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT2</strong> </td><td char=".">5,398 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010807 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK9 </td><td char=".">5,391 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP005093 </td><td char=".">26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw041-TF7"><div class="fn"><p><sup>a</sup>The representative replicons of each type ones are indicated in bold in the first column.</p></div></div><div content-id="dsw041-TF8"><div class="fn"><p><sup>b</sup>Replicons having more than one <em>rep</em> genes.</p></div></div><div content-id="dsw041-TF9"><div class="fn"><p><sup>c</sup>Start codon is differently annotated for the same DNA region.</p></div></div><div content-id="dsw041-TF10"><div class="fn"><p><sup>d</sup>Identical with each other.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw041-T3" data-id="dsw041-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Classification of sphingomonad plasmids and plasmid-type replicons</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th colspan="2" rowspan="2">Replicon<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Size (bp) </th><th rowspan="2">Host </th><th rowspan="2">Host feature </th><th colspan="3">RepA protein </th><th>Type </th><th>Gene cluster for conjugation </th><th>Genes for degradation </th><th>Accession number </th><th>Reference </th></tr><tr><th>Location </th><th>A.A. </th><th>Identity (%) to representative<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td colspan="2"><strong>UT26_Chr2</strong> </td><td char=".">681,892 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1203 </td><td char=".">400 </td><td>— </td><td><em>repABC</em> </td><td> </td><td><em>linF, linGHIJ</em> </td><td>AP010804 </td><td char=".">18 </td></tr><tr><td colspan="2"> L-1_Chr2 </td><td char=".">1,368,670 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>321340_322506 </td><td char=".">388 </td><td char=".">89 (359/400) </td><td><em>repABC</em> </td><td> </td><td><em>pcpBDR, pcpEMAC</em> </td><td>CP002799 </td><td char=".">11 </td></tr><tr><td colspan="2"> TKS_Chr2 </td><td char=".">989,120 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1206 </td><td char=".">401 </td><td char=".">89 (360/401) </td><td><em>repABC</em> </td><td> </td><td><em>linGHIJ_</em>homologue </td><td>CP005084 </td><td char=".">26 </td></tr><tr><td colspan="2"> MI_Chr2 </td><td char=".">567,154 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1209 </td><td char=".">402 </td><td char=".">83 (338/405) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP005189 </td><td char=".">25 </td></tr><tr><td colspan="2"> pNL2 </td><td char=".">487,268 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>209439_210644_c </td><td char=".">401 </td><td char=".">70 (284/401) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP000677 </td><td>unpublished </td></tr><tr><td colspan="2"> Lpl </td><td char=".">192,103 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>88922_90199 </td><td char=".">425 </td><td char=".">67 (288/425) </td><td><em>repABC</em> </td><td> </td><td> </td><td>FR856860 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>114750_115880_c </td><td char=".">376 </td><td char=".">55 (201/363) </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> TKS_Chr1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">4,249,857 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>252492_253622_c </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>CP005083 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>183355_184485 </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI1 </td><td char=".">292,135 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1224 </td><td char=".">407 </td><td char=".">42 (174/409) </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td>CP005190 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>312340_313539 </td><td char=".">399 </td><td char=".">42 (171/401) </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pCHQ1</strong> </td><td char=".">190,974 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>— </td><td><em>repABC</em> </td><td>Ti, F </td><td><em>linRED</em> </td><td>AP010805 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="2"> pSLGP </td><td char=".">148,801 </td><td><em>Sphingobium</em> sp. SYK-6 </td><td>lignin degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>AP012223 </td><td><a reveal-id="dsw041-B81" data-open="dsw041-B81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td colspan="2"> pSPHCH01 </td><td char=".">123,733 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>47083_48171_c </td><td char=".">362<a reveal-id="dsw041-TF9" data-open="dsw041-TF9" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>CP002800 </td><td char=".">11 </td></tr><tr><td colspan="2"> pISP0 </td><td char=".">275,840 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>4081_5244 </td><td char=".">387 </td><td char=".">98 (381/387) </td><td><em>repABC</em> </td><td>Ti </td><td><em>linF, linGHIJ</em> </td><td>CP004037 </td><td char=".">23 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT1</strong> </td><td char=".">31,776 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010806 </td><td char=".">18 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pISP2 </td><td char=".">53,841 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP004039 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP3</strong> </td><td char=".">43,776 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>37217_38329 </td><td char=".">370 </td><td>— </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP004040 </td><td char=".">23 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK6 </td><td char=".">34,300 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005090 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_1</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_960 </td><td char=".">319 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_2</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>52874_53821_c </td><td char=".">315 </td><td>— </td><td>iteron </td><td> </td><td><em>linB</em>, <em>linC</em>, <em>linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP4</strong> </td><td char=".">33,183 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>101_1012 </td><td char=".">303 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP004041 </td><td char=".">23 </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>158477_159388_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>19263_20174_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>56463_57374 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>30095_31006_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>Ti </td><td><em>linB, linC, linF''</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>18420_19331 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK2</strong> </td><td char=".">195,308 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1122 </td><td char=".">373 </td><td>— </td><td>iteron </td><td>Ti </td><td> </td><td>CP005086 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK7</strong> </td><td char=".">9,585 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1107 </td><td char=".">368 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005091 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK8</strong> </td><td char=".">7,223 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1032 </td><td char=".">343 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005092 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pNL1</strong> </td><td char=".">184,462 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>86362_87666 </td><td char=".">434 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP000676 </td><td>unpublished </td></tr><tr><td colspan="2"> pCAR3 </td><td char=".">254,797 </td><td><em>Novosphigobium</em> sp. KA1 </td><td>carbazole degradation </td><td>200594_201898_c </td><td char=".">434 </td><td char=".">91 (398/433) </td><td>iteron </td><td> </td><td> </td><td>AB270530 </td><td><a reveal-id="dsw041-B82" data-open="dsw041-B82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td colspan="2"> Mpl </td><td char=".">1,161,602 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>513635_514939_c </td><td char=".">434 </td><td char=".">85 (372/434) </td><td>iteron </td><td> </td><td> </td><td>FR856861 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pSWIT02 </td><td char=".">222,757 </td><td><em>Sphingomonas wittichii</em> RW1 </td><td>dioxin degradation </td><td>63589_64893 </td><td char=".">434 </td><td char=".">82 (356/429) </td><td>iteron </td><td> </td><td> </td><td>CP000701 </td><td><a reveal-id="dsw041-B83" data-open="dsw041-B83" class="link link-ref link-reveal xref-bibr">83</a> </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pLB1</strong> </td><td char=".">65,998 </td><td>unidentifed soil bacterium (<em>S. japonicum</em> UT26) </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>— </td><td> </td><td>Ti </td><td><em>linB</em> </td><td>AB244976 </td><td><a reveal-id="dsw041-B74" data-open="dsw041-B74" class="link link-ref link-reveal xref-bibr">74</a> </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>identical </td><td> </td><td>Ti </td><td><em>linB, linC, linF'</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pLA2 </td><td char=".">62,341 </td><td><em>Novosphingobium pentaromativorans</em> US6-1 </td><td>benzo(a)pyrene degradation </td><td>40543_41325 </td><td char=".">260 </td><td char=".">98 (256/260) </td><td> </td><td>Ti </td><td> </td><td>AGFM01000123 </td><td><a reveal-id="dsw041-B84" data-open="dsw041-B84" class="link link-ref link-reveal xref-bibr">84</a> </td></tr><tr><td colspan="2"> pTK5 </td><td char=".">53,908 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td char=".">97 (253/260) </td><td> </td><td>Ti </td><td> </td><td>CP005089 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT2</strong> </td><td char=".">5,398 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010807 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK9 </td><td char=".">5,391 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP005093 </td><td char=".">26 </td></tr></tbody></table></div><div content-id="dsw041-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th colspan="2" rowspan="2">Replicon<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Size (bp) </th><th rowspan="2">Host </th><th rowspan="2">Host feature </th><th colspan="3">RepA protein </th><th>Type </th><th>Gene cluster for conjugation </th><th>Genes for degradation </th><th>Accession number </th><th>Reference </th></tr><tr><th>Location </th><th>A.A. </th><th>Identity (%) to representative<a reveal-id="dsw041-TF7" data-open="dsw041-TF7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td colspan="2"><strong>UT26_Chr2</strong> </td><td char=".">681,892 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1203 </td><td char=".">400 </td><td>— </td><td><em>repABC</em> </td><td> </td><td><em>linF, linGHIJ</em> </td><td>AP010804 </td><td char=".">18 </td></tr><tr><td colspan="2"> L-1_Chr2 </td><td char=".">1,368,670 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>321340_322506 </td><td char=".">388 </td><td char=".">89 (359/400) </td><td><em>repABC</em> </td><td> </td><td><em>pcpBDR, pcpEMAC</em> </td><td>CP002799 </td><td char=".">11 </td></tr><tr><td colspan="2"> TKS_Chr2 </td><td char=".">989,120 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1206 </td><td char=".">401 </td><td char=".">89 (360/401) </td><td><em>repABC</em> </td><td> </td><td><em>linGHIJ_</em>homologue </td><td>CP005084 </td><td char=".">26 </td></tr><tr><td colspan="2"> MI_Chr2 </td><td char=".">567,154 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1209 </td><td char=".">402 </td><td char=".">83 (338/405) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP005189 </td><td char=".">25 </td></tr><tr><td colspan="2"> pNL2 </td><td char=".">487,268 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>209439_210644_c </td><td char=".">401 </td><td char=".">70 (284/401) </td><td><em>repABC</em> </td><td> </td><td> </td><td>CP000677 </td><td>unpublished </td></tr><tr><td colspan="2"> Lpl </td><td char=".">192,103 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>88922_90199 </td><td char=".">425 </td><td char=".">67 (288/425) </td><td><em>repABC</em> </td><td> </td><td> </td><td>FR856860 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>114750_115880_c </td><td char=".">376 </td><td char=".">55 (201/363) </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> TKS_Chr1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">4,249,857 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>252492_253622_c </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linF', linFb, linKLMN</em> </td><td>CP005083 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>183355_184485 </td><td char=".">376 </td><td char=".">55 (200/363)<a reveal-id="dsw041-TF10" data-open="dsw041-TF10" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><em>repABC</em> </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI1 </td><td char=".">292,135 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1224 </td><td char=".">407 </td><td char=".">42 (174/409) </td><td><em>repABC</em> </td><td>F </td><td><em>linB, linC, linRED, linEb, linF, linF'', linGHIJ</em> </td><td>CP005190 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>312340_313539 </td><td char=".">399 </td><td char=".">42 (171/401) </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pCHQ1</strong> </td><td char=".">190,974 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>— </td><td><em>repABC</em> </td><td>Ti, F </td><td><em>linRED</em> </td><td>AP010805 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">520,614 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti, F </td><td> </td><td>CP005085 </td><td char=".">26 </td></tr><tr><td colspan="2"> pSLGP </td><td char=".">148,801 </td><td><em>Sphingobium</em> sp. SYK-6 </td><td>lignin degradation </td><td>1_1164 </td><td char=".">387 </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>AP012223 </td><td><a reveal-id="dsw041-B81" data-open="dsw041-B81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td colspan="2"> pSPHCH01 </td><td char=".">123,733 </td><td><em>Sphingobium chlorophenolicum</em> L-1 </td><td>PCP degradation </td><td>47083_48171_c </td><td char=".">362<a reveal-id="dsw041-TF9" data-open="dsw041-TF9" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>identical </td><td><em>repABC</em> </td><td>Ti </td><td> </td><td>CP002800 </td><td char=".">11 </td></tr><tr><td colspan="2"> pISP0 </td><td char=".">275,840 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>4081_5244 </td><td char=".">387 </td><td char=".">98 (381/387) </td><td><em>repABC</em> </td><td>Ti </td><td><em>linF, linGHIJ</em> </td><td>CP004037 </td><td char=".">23 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT1</strong> </td><td char=".">31,776 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010806 </td><td char=".">18 </td></tr><tr><td colspan="2"> pMI2<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">287,488 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linRD</em> </td><td>CP005191 </td><td char=".">25 </td></tr><tr><td colspan="2"> pISP2 </td><td char=".">53,841 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP004039 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1104 </td><td char=".">367 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP3</strong> </td><td char=".">43,776 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>37217_38329 </td><td char=".">370 </td><td>— </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP004040 </td><td char=".">23 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="2"> pTK6 </td><td char=".">34,300 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1113 </td><td char=".">370 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005090 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_1</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_960 </td><td char=".">319 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK3_2</strong><a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>52874_53821_c </td><td char=".">315 </td><td>— </td><td>iteron </td><td> </td><td><em>linB</em>, <em>linC</em>, <em>linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pISP4</strong> </td><td char=".">33,183 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>101_1012 </td><td char=".">303 </td><td>— </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP004041 </td><td char=".">23 </td></tr><tr><td colspan="2"> pISP1<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">172,140 </td><td><em>Sphingomonas</em> sp. MM-1 </td><td>γ-HCH degradation </td><td>158477_159388_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>F </td><td><em>linA, linC, linF'</em> </td><td>CP004038 </td><td char=".">23 </td></tr><tr><td colspan="2"> pTK3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">87,635 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>19263_20174_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linB, linC, linF'</em> </td><td>CP005087 </td><td char=".">26 </td></tr><tr><td colspan="2"> pTK4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">75,938 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>56463_57374 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linA, linC</em> </td><td>CP005088 </td><td char=".">26 </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>30095_31006_c </td><td char=".">303 </td><td>identical </td><td>iteron </td><td>Ti </td><td><em>linB, linC, linF''</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pMI4<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">32,974 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>18420_19331 </td><td char=".">303 </td><td>identical </td><td>iteron </td><td> </td><td><em>linRED</em> </td><td>CP005193 </td><td char=".">25 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK2</strong> </td><td char=".">195,308 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1122 </td><td char=".">373 </td><td>— </td><td>iteron </td><td>Ti </td><td> </td><td>CP005086 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK7</strong> </td><td char=".">9,585 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1107 </td><td char=".">368 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005091 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pTK8</strong> </td><td char=".">7,223 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_1032 </td><td char=".">343 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP005092 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pNL1</strong> </td><td char=".">184,462 </td><td><em>Novosphingobium aromaticivorans</em> DSM 12444 </td><td>aromatic compounds degradation </td><td>86362_87666 </td><td char=".">434 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>CP000676 </td><td>unpublished </td></tr><tr><td colspan="2"> pCAR3 </td><td char=".">254,797 </td><td><em>Novosphigobium</em> sp. KA1 </td><td>carbazole degradation </td><td>200594_201898_c </td><td char=".">434 </td><td char=".">91 (398/433) </td><td>iteron </td><td> </td><td> </td><td>AB270530 </td><td><a reveal-id="dsw041-B82" data-open="dsw041-B82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td colspan="2"> Mpl </td><td char=".">1,161,602 </td><td><em>Novosphingobium</em> sp. PP1Y </td><td>aromatic compounds degradation </td><td>513635_514939_c </td><td char=".">434 </td><td char=".">85 (372/434) </td><td>iteron </td><td> </td><td> </td><td>FR856861 </td><td><a reveal-id="dsw041-B80" data-open="dsw041-B80" class="link link-ref link-reveal xref-bibr">80</a> </td></tr><tr><td colspan="2"> pSWIT02 </td><td char=".">222,757 </td><td><em>Sphingomonas wittichii</em> RW1 </td><td>dioxin degradation </td><td>63589_64893 </td><td char=".">434 </td><td char=".">82 (356/429) </td><td>iteron </td><td> </td><td> </td><td>CP000701 </td><td><a reveal-id="dsw041-B83" data-open="dsw041-B83" class="link link-ref link-reveal xref-bibr">83</a> </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pLB1</strong> </td><td char=".">65,998 </td><td>unidentifed soil bacterium (<em>S. japonicum</em> UT26) </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>— </td><td> </td><td>Ti </td><td><em>linB</em> </td><td>AB244976 </td><td><a reveal-id="dsw041-B74" data-open="dsw041-B74" class="link link-ref link-reveal xref-bibr">74</a> </td></tr><tr><td colspan="2"> pMI3<a reveal-id="dsw041-TF8" data-open="dsw041-TF8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">88,374 </td><td><em>Sphingobium</em> sp. MI1205 </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td>identical </td><td> </td><td>Ti </td><td><em>linB, linC, linF'</em> </td><td>CP005192 </td><td char=".">25 </td></tr><tr><td colspan="2"> pLA2 </td><td char=".">62,341 </td><td><em>Novosphingobium pentaromativorans</em> US6-1 </td><td>benzo(a)pyrene degradation </td><td>40543_41325 </td><td char=".">260 </td><td char=".">98 (256/260) </td><td> </td><td>Ti </td><td> </td><td>AGFM01000123 </td><td><a reveal-id="dsw041-B84" data-open="dsw041-B84" class="link link-ref link-reveal xref-bibr">84</a> </td></tr><tr><td colspan="2"> pTK5 </td><td char=".">53,908 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_783 </td><td char=".">260 </td><td char=".">97 (253/260) </td><td> </td><td>Ti </td><td> </td><td>CP005089 </td><td char=".">26 </td></tr><tr><td colspan="13"> </td></tr><tr><td colspan="2"><strong>pUT2</strong> </td><td char=".">5,398 </td><td><em>S. japonicum</em> UT26 </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>— </td><td>iteron </td><td> </td><td> </td><td>AP010807 </td><td char=".">18 </td></tr><tr><td colspan="2"> pTK9 </td><td char=".">5,391 </td><td><em>Sphingobium</em> sp. TKS </td><td>γ-HCH degradation </td><td>1_654 </td><td char=".">217 </td><td>identical </td><td>iteron </td><td> </td><td> </td><td>CP005093 </td><td char=".">26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw041-TF7"><div class="fn"><p><sup>a</sup>The representative replicons of each type ones are indicated in bold in the first column.</p></div></div><div content-id="dsw041-TF8"><div class="fn"><p><sup>b</sup>Replicons having more than one <em>rep</em> genes.</p></div></div><div content-id="dsw041-TF9"><div class="fn"><p><sup>c</sup>Start codon is differently annotated for the same DNA region.</p></div></div><div content-id="dsw041-TF10"><div class="fn"><p><sup>d</sup>Identical with each other.</p></div></div></div></div></p><p>Although, in the present study, we only described the overall genetic repertoire of the <em>lin</em> genes for γ-HCH utilization, the composition of genes for LinA and LinB variants and their copy numbers and expression levels are important for the degradation performance of host strains toward HCH isomers, since the LinA and LinB variants show different levels of enzymatic activity toward different HCH isomers and their metabolites.<a reveal-id="dsw041-B7" data-open="dsw041-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw041-B16" data-open="dsw041-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw041-B61 dsw041-B62 dsw041-B63 dsw041-B64" data-open="dsw041-B61 dsw041-B62 dsw041-B63 dsw041-B64" class="link link-ref link-reveal xref-bibr"><sup>61–64</sup></a> However, in order to properly discuss this point from a genomic viewpoint, additional fundamental biochemical and experimental data will be needed.</p>                            <h3 scrollto-destination=48005580 id="48005580" class="section-title" data-legacy-id=SEC3.3>3.3. Replication/partition-encoding regions of plasmids and plasmid-type replicons in sphingomonad strains</h3>
<div class="block-child-p">The putative replication origins (<em>oriC</em>s) of the main chromosomes (Chr1s) of TKS, MI1205, and MM-1 were, as in the case with Chr1<sub>UT26</sub>,<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> found to be of alphaproteobacterial-chromosome type;<a reveal-id="dsw041-B65" data-open="dsw041-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsw041-B66" data-open="dsw041-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> these <em>oriC</em>s were located upstream of the uroprophyrinogen decarboxylase gene (<em>hemE</em>) with multiple DnaA boxes [TT(A/T)TNCACA] (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>).<a reveal-id="dsw041-B67" data-open="dsw041-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> On the other hand, as in the case of Chr2<sub>UT26</sub>, both Chr2<sub>TKS</sub> and Chr2<sub>MI1205</sub> have the plasmid-type replication and active partition systems.<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> These three plasmid-type chromosomes and the 21 plasmids in our four strains and the plasmids from other sphingomonads were, on the basis of the similarities of their RepA (DNA replication initiator) proteins, classified into 13 types (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Although the importance of plasmids in sphingomonads has been recognized,<a reveal-id="dsw041-B8" data-open="dsw041-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> no detailed analysis of their fundamental machineries was reported. In addition, RepA proteins of plasmids in sphingomonads show a very low level of similarity to those of well-studied plasmids (e.g. IncP-1, F, IincP-7, and IncP-9 plasmids), and thus we compared only plasmids in sphingomonads in this study. Since the RepA proteins of the 13 types are very divergent, they were further categorized into three major groups, in each of which the RepA proteins exhibit 22–60% identity: (i) the Chr2<sub>UT26</sub>- and pCHQ1-types (<a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>A), (ii) the pUT1-, pISP3-, pTK3_1-, and pTK3_2-types (<a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>B), and (iii) the pISP4-, pTK2-, pTK7-, and pTK8-types (<a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>C). Based on our BLASTP analysis, the RepA proteins of pNL1, pLB1, and pUT2 did not show similarity to those of any of the other types of plasmids listed in <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>, although the RepA of pUT2 was similar to those of the IncP-9 family of plasmids.<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Figure 4</a>D schematically shows the organizations of the <em>repA</em>-flanking regions in the 12 representative plasmids (note that pTK3 has three types of <em>repA</em> genes), and many of these regions also carry the putative replication origin (<em>oriV</em>) sequences as well as the putative genes for the active partition systems, each with the putative <em>parS</em> (<em>cis</em>-acting centromeric) sequences, the <em>parB</em> gene encoding the <em>parS</em>-binding protein, and the <em>parA</em> gene encoding the NTPase that is capable of binding the <em>parS</em>-ParB complex.<a reveal-id="dsw041-B68" data-open="dsw041-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> The Chr2<sub>UT26</sub>- and pCHQ1-type plasmids belong to the <em>repABC</em>-type plasmids,<a reveal-id="dsw041-B69" data-open="dsw041-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> and putative palindromic <em>parS</em> sequences were found (<a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>D and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The Chr2<sub>UT26</sub>-type plasmids have the <em>parA</em>-<em>parB</em>-<em>repA</em> cluster, and the order of these three genes is conserved in other typical <em>repABC</em>-type plasmids,<a reveal-id="dsw041-B69" data-open="dsw041-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> although the RepA proteins from the Chr2<sub>UT26</sub>-type plasmids are divergent in their sizes and similarities (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>A). In contrast, the pCHQ1-type plasmids have a <em>repA</em>-<em>parA</em>-<em>parB</em> cluster (<a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>D) with nearly identical RepA proteins (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Other types of plasmids except the pLB1-type were categorized as iteron-type plasmids (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>),<a reveal-id="dsw041-B70" data-open="dsw041-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a><sup>,</sup><a reveal-id="dsw041-B71" data-open="dsw041-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> and direct repeats (iteron), DnaA box, and <em>parS</em> sequences were found in their <em>repA</em>-flanking regions (<a reveal-id="dsw041-F4" data-open="dsw041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>D and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Each of the pISP1, Chr1<sub>TKS</sub>, pTK1, pTK3, pTK4, pMI2, pMI3, and pMI4 replicons appears to carry at least two <em>repA</em> genes, which are of different types (<a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">3</a>). This observation suggested the frequent occurrence of fusions of ancestral plasmids (see below). Similar mosaic replicons carrying more than one <em>repA</em> gene have been reported in various bacterial strains.<a reveal-id="dsw041-B72" data-open="dsw041-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Interestingly, six pISP4-type plasmids carry identical <em>repA</em> and <em>parA</em> genes, and five of them also have other types of <em>repA</em> genes (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>), suggesting a prevalent fusion event of replicons in the pISP4-type plasmids (see below). It is noteworthy that all six pISP4-type plasmids contain the <em>lin</em> genes (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>), indicating that this type of plasmid plays an important role in dissemination of the <em>lin</em> genes. <div data-id="dsw041-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f4ap.png?Expires=2147483647&amp;Signature=ns7xBSJN0zpj3-pf3ujMSxlplsKuAny3gV4N0YwAvRnUCUUCut0aYHYmEnQrJ5X~2eRVibY61amWuMIJl8US~qEghZ8fPBp6YOjBBUfx3z4IaZ2auO4TPuLq5OSh45KUPx1tI6dZDljyMEcn~Ukqjoo80HduMRtGWQJTaeA4KXCwCC7oyAnJ48Q6-4~NtMf50mli1EYMb1Ju6Z3URUfq0PAM~94OWBQPyk~FT3URfrcYci2HkM63PURsRFhkaxsaQOsPOR6GaTOm1WMf3nLyqV8qBVQTCLs6M8~7g-wwjKhq4Ejfp5GCTQMT-dgNOqQSUIIMZzHsJbnKLqSr5Tnqjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of putative RepA proteins of Chr2UT26- and pCHQ1- (A), pUT1- (B), and pISP4- (C) types plasmids and organizations of repA-flanking regions of 12 representative sphingomonad plasmids (D). Classification of 13 types of plasmids and information on RepA protein sequences are summarized in Table 3. Neighbour-joining phylogenetic trees of the conserved sites, 260 aa (A), 262 aa (B), and 257 aa (C), respectively, were constructed. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. RepA proteins of the representatives of the plasmid types (Table 3) are bolded. In panel D, the repA-flanking regions of plasmids whose putative RepA proteins show significant similarity are boxed. Pentagons indicate size and direction of ORF. Putative ORFs involved in replication and partition are filled with dark and light gray, respectively. Putative parS (palindromic TTN4CG N4AA) 79 and DnaA box [TT(A or T)TNCACA] 67 sequences are shown in red bars and red diamonds, respectively. Inverted repeats and repeat sequences are shown in blue and green bars, respectively. See Supplementary Table S6 for their sequences." path-from-xml="dsw041f4ap.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005581/dsw041f4ap.tif" path-from-xml="dsw041f4ap.tif" target="_blank">View large</a><a section="48005581" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f4ap.png?Expires=2147483647&amp;Signature=1qPcwwuvPWc6v43lfyTHn7Bri64raZax2LtGVIxhGOUiRIFK5yXGNy80kOVPw9HCTVvzddLugOavsBc-UtD3wFfvM-oDWWfsm5BZHYPDe0CKploI2ZRP4YB7UTV9n4zDf6o3aV6nfIgmbTyPxiFKM4Ic~7u9dYiDc3Y7amVaheFouksoUQbImkQgq-qguzH6-K~U6DsJ-9f~yf2KNQAvCvH6dbi9MJPuMWuZmHl~GJzEbOFmJfCOweYvIEc5vM3OxA6EI9UEJ~jQItfW2DlytSWgzLgMDdMA88NhelwrieheFQH-6yymIvo~bU6OBNsRiMYnjGEGGAXQhpsUkoBQ2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005581&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f4ap.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f4bp.png?Expires=2147483647&amp;Signature=gSuF-b1wu8KIUwiZ2rrI~lM8TTgH5080O2cPu7KPeBMIyJsLLV-hD~gcRtDynrHhg2tP5JrdEXMqZ49TOLPPC0mBXI1nRfRbmpXwN75EwBntgzHawuYjqWAQE8PFbid6EYMxI0a6IVtS~XaheQt5Nb1ld4yxbJgjgmRnHJC8F3lMsbo95ZiTAxN67gsEn2a9T7OFII4zbRIQ9nUSlBwfYlpKLCt1145PC~6YGSfDmOx93IikIHVs0GNOwp-pgJzvz-1psqyL5ZkVcyNmAS7k-CFT0~O1lP3JdqhpKJ6P0uzM6aPmayFHXzCjr3gC2hIK~veShvXHO0vIXtKAApxOOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of putative RepA proteins of Chr2UT26- and pCHQ1- (A), pUT1- (B), and pISP4- (C) types plasmids and organizations of repA-flanking regions of 12 representative sphingomonad plasmids (D). Classification of 13 types of plasmids and information on RepA protein sequences are summarized in Table 3. Neighbour-joining phylogenetic trees of the conserved sites, 260 aa (A), 262 aa (B), and 257 aa (C), respectively, were constructed. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. RepA proteins of the representatives of the plasmid types (Table 3) are bolded. In panel D, the repA-flanking regions of plasmids whose putative RepA proteins show significant similarity are boxed. Pentagons indicate size and direction of ORF. Putative ORFs involved in replication and partition are filled with dark and light gray, respectively. Putative parS (palindromic TTN4CG N4AA) 79 and DnaA box [TT(A or T)TNCACA] 67 sequences are shown in red bars and red diamonds, respectively. Inverted repeats and repeat sequences are shown in blue and green bars, respectively. See Supplementary Table S6 for their sequences." path-from-xml="dsw041f4bp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005581/dsw041f4bp.tif" path-from-xml="dsw041f4bp.tif" target="_blank">View large</a><a section="48005581" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f4bp.png?Expires=2147483647&amp;Signature=JcX~SG9Zam7wVeAGvZJ5P9dqNvijD3SnL4UBRiU4oGRaRlJFhkAvTBO9AnJjUEjEpfgWXAELtYr3-ipaZv62dBDBr1KbQSD2JyU6norIB9VQSPpTUHK6FufuPi00Z~~EDLrZVk0RLcFuhkGOVuz11Rn7JycIrEai4LSymEJxStnQ-V8XdqxZdJ6sUoAeH4C1YXMbQ~i-E0dyPXMD6kG3tK5q~294t2yJFaHGW-PRYn~PqZxqJAt-qmFHTAg7kq8L8GG8FtWecgAzxJtyCLuGoB5ycu87y3wD4TNcOeCt19nRGWOiKLNqKI~erhMaUHTJ00Nl0zSQaormPzQuGzodQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005581&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f4bp.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees of putative RepA proteins of Chr2<sub>UT26</sub>- and pCHQ1- <strong>(A)</strong>, pUT1- <strong>(B)</strong>, and pISP4- <strong>(C)</strong> types plasmids and organizations of <em>repA</em>-flanking regions of 12 representative sphingomonad plasmids <strong>(D)</strong>. Classification of 13 types of plasmids and information on RepA protein sequences are summarized in <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>. Neighbour-joining phylogenetic trees of the conserved sites, 260 aa (A), 262 aa (B), and 257 aa (C), respectively, were constructed. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. RepA proteins of the representatives of the plasmid types (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>) are bolded. In panel D, the <em>repA</em>-flanking regions of plasmids whose putative RepA proteins show significant similarity are boxed. Pentagons indicate size and direction of ORF. Putative ORFs involved in replication and partition are filled with dark and light gray, respectively. Putative <em>parS</em> (palindromic TTN<sub>4</sub>CG N<sub>4</sub>AA) <a reveal-id="dsw041-B79" data-open="dsw041-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> and DnaA box [TT(A or T)TNCACA] <a reveal-id="dsw041-B67" data-open="dsw041-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> sequences are shown in red bars and red diamonds, respectively. Inverted repeats and repeat sequences are shown in blue and green bars, respectively. See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> for their sequences.</p></div></div><div content-id="dsw041-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f4ap.png?Expires=2147483647&amp;Signature=ns7xBSJN0zpj3-pf3ujMSxlplsKuAny3gV4N0YwAvRnUCUUCut0aYHYmEnQrJ5X~2eRVibY61amWuMIJl8US~qEghZ8fPBp6YOjBBUfx3z4IaZ2auO4TPuLq5OSh45KUPx1tI6dZDljyMEcn~Ukqjoo80HduMRtGWQJTaeA4KXCwCC7oyAnJ48Q6-4~NtMf50mli1EYMb1Ju6Z3URUfq0PAM~94OWBQPyk~FT3URfrcYci2HkM63PURsRFhkaxsaQOsPOR6GaTOm1WMf3nLyqV8qBVQTCLs6M8~7g-wwjKhq4Ejfp5GCTQMT-dgNOqQSUIIMZzHsJbnKLqSr5Tnqjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of putative RepA proteins of Chr2UT26- and pCHQ1- (A), pUT1- (B), and pISP4- (C) types plasmids and organizations of repA-flanking regions of 12 representative sphingomonad plasmids (D). Classification of 13 types of plasmids and information on RepA protein sequences are summarized in Table 3. Neighbour-joining phylogenetic trees of the conserved sites, 260 aa (A), 262 aa (B), and 257 aa (C), respectively, were constructed. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. RepA proteins of the representatives of the plasmid types (Table 3) are bolded. In panel D, the repA-flanking regions of plasmids whose putative RepA proteins show significant similarity are boxed. Pentagons indicate size and direction of ORF. Putative ORFs involved in replication and partition are filled with dark and light gray, respectively. Putative parS (palindromic TTN4CG N4AA) 79 and DnaA box [TT(A or T)TNCACA] 67 sequences are shown in red bars and red diamonds, respectively. Inverted repeats and repeat sequences are shown in blue and green bars, respectively. See Supplementary Table S6 for their sequences." path-from-xml="dsw041f4ap.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005581/dsw041f4ap.tif" path-from-xml="dsw041f4ap.tif" target="_blank">View large</a><a section="48005581" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f4ap.png?Expires=2147483647&amp;Signature=1qPcwwuvPWc6v43lfyTHn7Bri64raZax2LtGVIxhGOUiRIFK5yXGNy80kOVPw9HCTVvzddLugOavsBc-UtD3wFfvM-oDWWfsm5BZHYPDe0CKploI2ZRP4YB7UTV9n4zDf6o3aV6nfIgmbTyPxiFKM4Ic~7u9dYiDc3Y7amVaheFouksoUQbImkQgq-qguzH6-K~U6DsJ-9f~yf2KNQAvCvH6dbi9MJPuMWuZmHl~GJzEbOFmJfCOweYvIEc5vM3OxA6EI9UEJ~jQItfW2DlytSWgzLgMDdMA88NhelwrieheFQH-6yymIvo~bU6OBNsRiMYnjGEGGAXQhpsUkoBQ2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005581&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f4ap.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f4bp.png?Expires=2147483647&amp;Signature=gSuF-b1wu8KIUwiZ2rrI~lM8TTgH5080O2cPu7KPeBMIyJsLLV-hD~gcRtDynrHhg2tP5JrdEXMqZ49TOLPPC0mBXI1nRfRbmpXwN75EwBntgzHawuYjqWAQE8PFbid6EYMxI0a6IVtS~XaheQt5Nb1ld4yxbJgjgmRnHJC8F3lMsbo95ZiTAxN67gsEn2a9T7OFII4zbRIQ9nUSlBwfYlpKLCt1145PC~6YGSfDmOx93IikIHVs0GNOwp-pgJzvz-1psqyL5ZkVcyNmAS7k-CFT0~O1lP3JdqhpKJ6P0uzM6aPmayFHXzCjr3gC2hIK~veShvXHO0vIXtKAApxOOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of putative RepA proteins of Chr2UT26- and pCHQ1- (A), pUT1- (B), and pISP4- (C) types plasmids and organizations of repA-flanking regions of 12 representative sphingomonad plasmids (D). Classification of 13 types of plasmids and information on RepA protein sequences are summarized in Table 3. Neighbour-joining phylogenetic trees of the conserved sites, 260 aa (A), 262 aa (B), and 257 aa (C), respectively, were constructed. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. RepA proteins of the representatives of the plasmid types (Table 3) are bolded. In panel D, the repA-flanking regions of plasmids whose putative RepA proteins show significant similarity are boxed. Pentagons indicate size and direction of ORF. Putative ORFs involved in replication and partition are filled with dark and light gray, respectively. Putative parS (palindromic TTN4CG N4AA) 79 and DnaA box [TT(A or T)TNCACA] 67 sequences are shown in red bars and red diamonds, respectively. Inverted repeats and repeat sequences are shown in blue and green bars, respectively. See Supplementary Table S6 for their sequences." path-from-xml="dsw041f4bp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005581/dsw041f4bp.tif" path-from-xml="dsw041f4bp.tif" target="_blank">View large</a><a section="48005581" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f4bp.png?Expires=2147483647&amp;Signature=JcX~SG9Zam7wVeAGvZJ5P9dqNvijD3SnL4UBRiU4oGRaRlJFhkAvTBO9AnJjUEjEpfgWXAELtYr3-ipaZv62dBDBr1KbQSD2JyU6norIB9VQSPpTUHK6FufuPi00Z~~EDLrZVk0RLcFuhkGOVuz11Rn7JycIrEai4LSymEJxStnQ-V8XdqxZdJ6sUoAeH4C1YXMbQ~i-E0dyPXMD6kG3tK5q~294t2yJFaHGW-PRYn~PqZxqJAt-qmFHTAg7kq8L8GG8FtWecgAzxJtyCLuGoB5ycu87y3wD4TNcOeCt19nRGWOiKLNqKI~erhMaUHTJ00Nl0zSQaormPzQuGzodQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005581&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f4bp.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees of putative RepA proteins of Chr2<sub>UT26</sub>- and pCHQ1- <strong>(A)</strong>, pUT1- <strong>(B)</strong>, and pISP4- <strong>(C)</strong> types plasmids and organizations of <em>repA</em>-flanking regions of 12 representative sphingomonad plasmids <strong>(D)</strong>. Classification of 13 types of plasmids and information on RepA protein sequences are summarized in <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>. Neighbour-joining phylogenetic trees of the conserved sites, 260 aa (A), 262 aa (B), and 257 aa (C), respectively, were constructed. Bootstrap values calculated from 1,000 resampling using neighbour-joining are shown at the respective nodes. Length of lines reflects relative evolutionary distances among the sequences. RepA proteins of the representatives of the plasmid types (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>) are bolded. In panel D, the <em>repA</em>-flanking regions of plasmids whose putative RepA proteins show significant similarity are boxed. Pentagons indicate size and direction of ORF. Putative ORFs involved in replication and partition are filled with dark and light gray, respectively. Putative <em>parS</em> (palindromic TTN<sub>4</sub>CG N<sub>4</sub>AA) <a reveal-id="dsw041-B79" data-open="dsw041-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> and DnaA box [TT(A or T)TNCACA] <a reveal-id="dsw041-B67" data-open="dsw041-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> sequences are shown in red bars and red diamonds, respectively. Inverted repeats and repeat sequences are shown in blue and green bars, respectively. See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> for their sequences.</p></div></div></div>                            <h3 scrollto-destination=48005582 id="48005582" class="section-title" data-legacy-id=SEC3.4>3.4. Highly conserved regions of replicons in sphingomonad strains</h3>
<div class="block-child-p">Although multiple members in each of the pCHQ1-, pUT1-, pLB1-, pISP3-, pUT2-, and pISP4-type plasmids have almost identical <em>repA</em>-containing regions, the sizes and gene contents of the plasmid members in each type are diverse (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Therefore, we compared the overall structures in the six types of plasmids (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The 6.4-kb <em>repA</em>-<em>parA</em>-<em>parB</em>-containing region and the 10-kb <em>repA</em>-<em>parA</em>-containing region are conserved in all members of the pCHQ1- and pUT1-type plasmids, respectively (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>AB). The 25.5-kb region containing the pLB1-type <em>repA</em> and <em>parB</em> genes is conserved in pMI3 and pTK5 (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>C). Most of this region is also conserved in pLA2, although the region is divided into two parts and the <em>parB</em> gene is lacking (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>C). pMI3 is a fusion plasmid of the pLB1- and pISP4-type plasmids because the 9.1-kb <em>repA</em>-<em>parA</em>-containing region commonly conserved in the pISP4-type plasmids (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>F) is, together with the <em>linC</em>- and <em>linF</em><em>″</em>-containing region, inserted into the continuous region on pLB1 (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>C). A part of the 9.1-kb conserved region is also present in pTK4 (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>B). The 4.5-kb region containing the pISP3-type <em>repA</em> and <em>parA</em> genes is conserved in pTK6 and pMI4 (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>D). Two 1,645-bp plasmids, pUT2 and pTK9, differ by only 9 bp (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>E). As mentioned above, all the pISP4-type plasmids except for archetypal pISP4 have, in addition to the common <em>repA</em> gene, other distinct <em>repA</em> genes (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>), and probably are fusion plasmids. IS<em>6100</em> or the Tn<em>3</em>-type transposon is located at the junctions of highly conserved regions of pISP4-type plasmids (<a reveal-id="dsw041-F5" data-open="dsw041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>F), suggesting that the pISP4-type plasmids are easily fused with other replicons via the transposition of IS<em>6100</em> and/or the Tn<em>3</em>-type transposon. The <em>repA</em>, <em>parA</em>, and <em>parB</em> genes of the Chr2<sub>UT26</sub>-type replication machineries were also located on Chr1<sub>TKS</sub>, pISP1, pMI2, Chr2<sub>TKS</sub>, pTK1, Chr2<sub>MI1205</sub>, and pMI1 (<a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>), and the former three replicons carry an almost identical 20.3-kb region that covers the Chr2<sub>UT26</sub>-type <em>repA</em>, <em>parA</em>, and <em>parB</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Our findings in this section clearly indicate that the replicons having highly conserved replication/partition genes are distributed among sphingomonad strains with frequent recombination events including replicon fusion. <div data-id="dsw041-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f5ap.png?Expires=2147483647&amp;Signature=ldCWU2PAo2b4xFwhn-p2~ohNmXi8AxxXH4zNZ6KvyaKABqk4rdZdbqLJx9SYN7OHt8rQQ74ZFjAEiFXMzvTuF-K-iq-vuUHKiou70c7Tm3nVnBLMKIDWlpbf11eQI2nVqH1f6PKyhds41X9avtVvOMmoDlEFeeD03LnObMmpwnRvDzY3ZnYbShOdIL4MoN0Rb2Pbg5SN8lmS9K~2PoS9n~RP1hwNdGh3Zu6hxKmchNOu-FVQGc1Tpz6lcW9Iboi0iN5EnUiuzg5Lr6GrNPMTJZ6MQJxhIbRKdrVzXAYjVLPZTm6GSqbdYuHXe1f4g2AdXWCWfu67TDfrzXhgIo5gRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structures of plasmids which have the highly conserved regions in the four γ-HCH-degrading sphingomonad strains. The highly conserved regions in pCHQ1- (A), pUT1- (B), pLB1- (C), pISP3- (D), pUT2- (E), and pISP4- (F) types plasmids are schematically shown. See Table 3 for the classification of plasmids. ORFs shown by pentagons are coloured as follows: red, rep and par genes; green, transposase gene of IS6100; cyan, putative genes for conjugal transfer; yellow, lin genes; and gray with gradient, transposition-related genes in other putative transposons. Almost identical regions are shown by the same background colours." path-from-xml="dsw041f5ap.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005583/dsw041f5ap.tif" path-from-xml="dsw041f5ap.tif" target="_blank">View large</a><a section="48005583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f5ap.png?Expires=2147483647&amp;Signature=VS5N5gg6OWr-mX1zENdsqYNW-qbLg2sxnDQzAK6QarhTwPIeLshYkUCWaFEYsALvP25ifM8JBk7T-mHTgj~TUq5Oo81R-A7pqzDV1FfJnMZgQfx9Fr8gfr6l1xz8r9fdzt8yjAGrYsnqixYaILn1WNsjvLIwS8aXeRK3anXwblFcrhOGrjz3dFuvGhn~XYPBX3h41mgGy8eBL8eO6gIvPLxqjIYlS4GPQrc0Xj-Bz4PeRBJ4I~lxmyJQX8tqz7Nkmk-zrFIukLwJ0CPyhfRL6G~GblxBHfGHQp6~9ZJP7A91uFIq-hAvMrXOmzshM6xu4R7L97OyqPVzREEx9wZakA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005583&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f5ap.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f5bp.png?Expires=2147483647&amp;Signature=B7njgCdqqxMHZK-0gJr9IJDxIJIqQEWB5haYNd-ZoxzITa2crZqVE3oKRf-D5fgjyCoLXwYLkdcyydkYo3XScLWd3H81h3PPXyRnfiAV0FK-9ye3Qg9MHgIg-tgtEjp0Px~qPhlrbTu712sA-9fzjhe0Lrr7P~4MT7DlWdR0bFxEvC1jXtn1Erxx5eZaD6zk4EH86EfrhbNzQ4nY8LONMPCesFvyq0wl49XLeSWseeN~uELBJyFjHP53zaB8lW5pgge-SxVROq2NitJPtDnxtUgrA0M6-dUzVGqXzyfrib6ErVoYcTyZ14~qfGG0GP4yFJfQIk6TKvu1rIeXIj~pHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structures of plasmids which have the highly conserved regions in the four γ-HCH-degrading sphingomonad strains. The highly conserved regions in pCHQ1- (A), pUT1- (B), pLB1- (C), pISP3- (D), pUT2- (E), and pISP4- (F) types plasmids are schematically shown. See Table 3 for the classification of plasmids. ORFs shown by pentagons are coloured as follows: red, rep and par genes; green, transposase gene of IS6100; cyan, putative genes for conjugal transfer; yellow, lin genes; and gray with gradient, transposition-related genes in other putative transposons. Almost identical regions are shown by the same background colours." path-from-xml="dsw041f5bp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005583/dsw041f5bp.tif" path-from-xml="dsw041f5bp.tif" target="_blank">View large</a><a section="48005583" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f5bp.png?Expires=2147483647&amp;Signature=h4vcv7V61k0GlaWmQzdO6J6wh6lAA93hy72SQ8w-fz6p~-Q0~4qIt5c1DQW0V4Z557PgpP1XVrJmLcW~6iQlcCYSmGy3r234y5~oDbkRkQj4MM8opzWN0SKyysK9jExpOB3lMiI8s23~Ja5VLHdTxDcI4S8fgKTFUxllANAIOFVvEneRrXGXdyXNORtV9aYCSebOQ1kMQItHcL66Qi4qpFTLbbqMaLTEsYQbqjXD71k7mPOC5mZzjo4Q56vSocthNQNCW7aTTByxfIqXpxl1B3mKdN1NyQVptWYjJLwv866EktymeHhe41deZk9TFnNkUkfwoXAOOiROyRD9ZwFdew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005583&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f5bp.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structures of plasmids which have the highly conserved regions in the four γ-HCH-degrading sphingomonad strains. The highly conserved regions in pCHQ1- <strong>(A)</strong>, pUT1- <strong>(B)</strong>, pLB1- <strong>(C)</strong>, pISP3- <strong>(D)</strong>, pUT2- <strong>(E)</strong>, and pISP4- <strong>(F)</strong> types plasmids are schematically shown. See <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a> for the classification of plasmids. ORFs shown by pentagons are coloured as follows: red, <em>rep</em> and <em>par</em> genes; green, transposase gene of IS<em>6100</em>; cyan, putative genes for conjugal transfer; yellow, <em>lin</em> genes; and gray with gradient, transposition-related genes in other putative transposons. Almost identical regions are shown by the same background colours.</p></div></div><div content-id="dsw041-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f5ap.png?Expires=2147483647&amp;Signature=ldCWU2PAo2b4xFwhn-p2~ohNmXi8AxxXH4zNZ6KvyaKABqk4rdZdbqLJx9SYN7OHt8rQQ74ZFjAEiFXMzvTuF-K-iq-vuUHKiou70c7Tm3nVnBLMKIDWlpbf11eQI2nVqH1f6PKyhds41X9avtVvOMmoDlEFeeD03LnObMmpwnRvDzY3ZnYbShOdIL4MoN0Rb2Pbg5SN8lmS9K~2PoS9n~RP1hwNdGh3Zu6hxKmchNOu-FVQGc1Tpz6lcW9Iboi0iN5EnUiuzg5Lr6GrNPMTJZ6MQJxhIbRKdrVzXAYjVLPZTm6GSqbdYuHXe1f4g2AdXWCWfu67TDfrzXhgIo5gRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structures of plasmids which have the highly conserved regions in the four γ-HCH-degrading sphingomonad strains. The highly conserved regions in pCHQ1- (A), pUT1- (B), pLB1- (C), pISP3- (D), pUT2- (E), and pISP4- (F) types plasmids are schematically shown. See Table 3 for the classification of plasmids. ORFs shown by pentagons are coloured as follows: red, rep and par genes; green, transposase gene of IS6100; cyan, putative genes for conjugal transfer; yellow, lin genes; and gray with gradient, transposition-related genes in other putative transposons. Almost identical regions are shown by the same background colours." path-from-xml="dsw041f5ap.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005583/dsw041f5ap.tif" path-from-xml="dsw041f5ap.tif" target="_blank">View large</a><a section="48005583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f5ap.png?Expires=2147483647&amp;Signature=VS5N5gg6OWr-mX1zENdsqYNW-qbLg2sxnDQzAK6QarhTwPIeLshYkUCWaFEYsALvP25ifM8JBk7T-mHTgj~TUq5Oo81R-A7pqzDV1FfJnMZgQfx9Fr8gfr6l1xz8r9fdzt8yjAGrYsnqixYaILn1WNsjvLIwS8aXeRK3anXwblFcrhOGrjz3dFuvGhn~XYPBX3h41mgGy8eBL8eO6gIvPLxqjIYlS4GPQrc0Xj-Bz4PeRBJ4I~lxmyJQX8tqz7Nkmk-zrFIukLwJ0CPyhfRL6G~GblxBHfGHQp6~9ZJP7A91uFIq-hAvMrXOmzshM6xu4R7L97OyqPVzREEx9wZakA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005583&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f5ap.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f5bp.png?Expires=2147483647&amp;Signature=B7njgCdqqxMHZK-0gJr9IJDxIJIqQEWB5haYNd-ZoxzITa2crZqVE3oKRf-D5fgjyCoLXwYLkdcyydkYo3XScLWd3H81h3PPXyRnfiAV0FK-9ye3Qg9MHgIg-tgtEjp0Px~qPhlrbTu712sA-9fzjhe0Lrr7P~4MT7DlWdR0bFxEvC1jXtn1Erxx5eZaD6zk4EH86EfrhbNzQ4nY8LONMPCesFvyq0wl49XLeSWseeN~uELBJyFjHP53zaB8lW5pgge-SxVROq2NitJPtDnxtUgrA0M6-dUzVGqXzyfrib6ErVoYcTyZ14~qfGG0GP4yFJfQIk6TKvu1rIeXIj~pHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structures of plasmids which have the highly conserved regions in the four γ-HCH-degrading sphingomonad strains. The highly conserved regions in pCHQ1- (A), pUT1- (B), pLB1- (C), pISP3- (D), pUT2- (E), and pISP4- (F) types plasmids are schematically shown. See Table 3 for the classification of plasmids. ORFs shown by pentagons are coloured as follows: red, rep and par genes; green, transposase gene of IS6100; cyan, putative genes for conjugal transfer; yellow, lin genes; and gray with gradient, transposition-related genes in other putative transposons. Almost identical regions are shown by the same background colours." path-from-xml="dsw041f5bp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005583/dsw041f5bp.tif" path-from-xml="dsw041f5bp.tif" target="_blank">View large</a><a section="48005583" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f5bp.png?Expires=2147483647&amp;Signature=h4vcv7V61k0GlaWmQzdO6J6wh6lAA93hy72SQ8w-fz6p~-Q0~4qIt5c1DQW0V4Z557PgpP1XVrJmLcW~6iQlcCYSmGy3r234y5~oDbkRkQj4MM8opzWN0SKyysK9jExpOB3lMiI8s23~Ja5VLHdTxDcI4S8fgKTFUxllANAIOFVvEneRrXGXdyXNORtV9aYCSebOQ1kMQItHcL66Qi4qpFTLbbqMaLTEsYQbqjXD71k7mPOC5mZzjo4Q56vSocthNQNCW7aTTByxfIqXpxl1B3mKdN1NyQVptWYjJLwv866EktymeHhe41deZk9TFnNkUkfwoXAOOiROyRD9ZwFdew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005583&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f5bp.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structures of plasmids which have the highly conserved regions in the four γ-HCH-degrading sphingomonad strains. The highly conserved regions in pCHQ1- <strong>(A)</strong>, pUT1- <strong>(B)</strong>, pLB1- <strong>(C)</strong>, pISP3- <strong>(D)</strong>, pUT2- <strong>(E)</strong>, and pISP4- <strong>(F)</strong> types plasmids are schematically shown. See <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a> for the classification of plasmids. ORFs shown by pentagons are coloured as follows: red, <em>rep</em> and <em>par</em> genes; green, transposase gene of IS<em>6100</em>; cyan, putative genes for conjugal transfer; yellow, <em>lin</em> genes; and gray with gradient, transposition-related genes in other putative transposons. Almost identical regions are shown by the same background colours.</p></div></div></div>                            <h3 scrollto-destination=48005584 id="48005584" class="section-title" data-legacy-id=SEC3.5>3.5. Genes for conjugal transfer of plasmids in HCH-degrading sphingomonads</h3>
<p>The genes for conjugal transfer consist of those encoding proteins involved in mating pair formation (Mpf) and DNA transfer and replication (Dtr).<a reveal-id="dsw041-B73" data-open="dsw041-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> The <em>mpf</em> genes encode proteins that assemble in a large macromolecular structure called the Type IV secretion system (T4SS), whereas the <em>dtr</em> genes encode proteins that bind to the DNA at the origin of transfer region, <em>oriT</em>, forming a structure called a relaxosome. This modular gene organization is shared by most conjugative systems, showing a high degree of gene synteny conservation. Among the sphingomonad plasmids listed in <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">Table 3</a>, conjugal transferability of pCHQ1 and pLB1 has been experimentally confirmed,<a reveal-id="dsw041-B36" data-open="dsw041-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsw041-B74" data-open="dsw041-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> and these two plasmids have putative gene clusters for conjugal transfer similar to the <em>vir</em> gene cluster of <em>Agrobacterium tumefaciens</em> Ti plasmid, consisting of genes for Mpf (VirB1 to VirB11) and Dtr (relaxase VirD2 and coupling protein VirD4)( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9A</a></span>). <a reveal-id="dsw041-B17" data-open="dsw041-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw041-B74" data-open="dsw041-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> Putative <em>vir</em> gene clusters were also found on pISP0, pTK1, pTK2, pTK5, and pMI3 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9A</a></span>), indicating the potential self-transferability of these plasmids. However, the level of similarity of each component to the counterpart of Ti plasmid is relatively low, and only the phylogenetic relationship of the putative cytoplasmic ATPase component (VirB4), which is the essential and most conserved component of T4SS,<a reveal-id="dsw041-B73" data-open="dsw041-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> encoded by these clusters is shown (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9B</a></span>). pCHQ1 has another putative gene cluster for conjugal transfer similar to the <em>tra</em> gene cluster of F plasmid,<a reveal-id="dsw041-B75" data-open="dsw041-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> and gene clusters similar to the <em>tra</em> gene cluster were also found on Chr1<sub>TKS</sub>, pMI1, pMI2, pTK1, and pISP1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9C</a></span>). As in the case with the gene clusters homologous to the <em>vir</em> gene cluster, the level of similarity of each component to its counterpart in F plasmid is relatively low, and only phylogenetic relationship of cytoplasmic ATPase component (TraC: VirB4 homologue in function) encoded by these clusters is shown (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9D</a></span>). However, <em>traI</em> and <em>traD</em>, which encode the relaxase and coupling protein, respectively, were not found on pCHQ1, and <em>traD</em> and <em>traG</em>, which encode the coupling protein and inner membrane platform component, respectively, are missing on pISP1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9C</a></span>). Further experimental confirmation is necessary to demonstrate the self-transferability of these plasmids having putative gene clusters for conjugal transfer.</p>                            <h3 scrollto-destination=48005586 id="48005586" class="section-title" data-legacy-id=SEC3.6>3.6. Transposable elements in four γ-HCH degraders</h3>
<p>Many putative transposable elements including IS elements and Tn<em>3</em>-type transposons were found in the genomes of the four γ-HCH degraders (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).<a reveal-id="dsw041-B18" data-open="dsw041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Although most of the IS elements are present as a single-copy form in the four strains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>), IS<em>6100</em> is, as in the case of UT26, the most abundant element in the MM-1, TKS, and MI1205 genomes (15, 29, and 24 copies, respectively) (<a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). This suggests that IS<em>6100</em> can transpose and increase its copy number in these γ-HCH degraders. To investigate the transposition activity of IS<em>6100</em> and other transposable elements, the IS entrapment methodology using pGEN500<sup>42</sup> was applied for the four γ-HCH degraders. We conducted several independent analyses for each strain, and detected the successful transposition of IS<em>6100</em>, IS<em>sp1</em>, IS<em>Sj02</em>, IS<em>Sj12</em>, and Tn<em>6134</em> in UT26, IS<em>6100</em>, IS<em>Sj02</em>, IS<em>Tks12</em>, Tn<em>6268</em>, and Tn<em>6269</em> in TKS, and IS<em>6100</em>, IS<em>sp1</em>, IS<em>Mi02</em>, IS<em>Mi08</em>, Tn<em>6137</em>, and Tn<em>6274</em> in MI1205 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). On the other hand, this IS-entrapment system did not work well in MM-1 because of the high-frequency generation of the spontaneous sucrose-resistant mutants without the insertion of transposable elements into the <em>sacB</em> gene on pGEN500 (data not shown).</p>                            <h3 scrollto-destination=48005588 id="48005588" class="section-title" data-legacy-id=SEC3.7>3.7. Inference of the past genome rearrangements via IS<em>6100</em></h3>
<div class="block-child-p">IS<em>6100</em> is often located in close proximity to the <em>lin</em> genes in the HCH-degrading strains and the metagenomic sequences from HCH-contaminated sites.<a reveal-id="dsw041-B76" data-open="dsw041-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a><sup>,</sup><a reveal-id="dsw041-B77" data-open="dsw041-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> IS<em>6100</em> with a size of 880 bp is a ‘replicative’ IS element (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>),<a reveal-id="dsw041-B78" data-open="dsw041-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> and its transposition without apparent preference of target specificity causes the duplication of IS<em>6100</em> with an 8-bp duplication of the target sequence. Therefore, the IS<em>6100</em> transposition can generate three types of DNA rearrangements (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>): intra-molecular transposition with a deletion/resolution (intra-replicon 1) or inversion (intra-replicon 2) event, and inter-molecular transposition with a fusion (inter-replicon) event. Our comparison of the regions just upstream and downstream of the 13 copies of IS<em>6100</em> present on Chr1, Chr2, pCHQ1, and pUT1 of UT26 revealed five pairs of 8-bp sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). On the basis of the IS<em>6100</em> transposition mechanism (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>), the most plausible past events caused by transposition of IS<em>6100</em> can be inferred (<a reveal-id="dsw041-F6" data-open="dsw041-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>A); it is indicated that not only simple transposition with inversion but also transposition accompanied with the fusion and resolution of replicons must have occurred. In a similar manner, seven, one, and four pairs of 8-bp sequences were found just upstream or downstream of IS<em>6100</em> in TKS, MM-1, and MI1205, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>), and the plausible past events mediated by transposition of IS<em>6100</em> in these strains are depicted in <a reveal-id="dsw041-F6" data-open="dsw041-F6" class="link link-reveal link-table xref-fig">Figure 6</a>B–D. <div data-id="dsw041-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f6ap.png?Expires=2147483647&amp;Signature=ZlIEQeUY3Gacu4neXb88O2-264NuiudPzXUoyFFunmxoqWF6irIp8-JLZLMiEa2VIkvm8QtWnbdzChTxLOtfgt0tt2Ssds2R-I9ZcwKe3qPpkrC0Uemx2XO0FKtoR1RIfFjsKGZhTu7-Srf9KfR6Y5N9P8Du8sTVoy4H7Q~eC-Y22i9yJDQ0VaqA50OkhWnVb58R8f473kVZiqZNbK0qigVFa4iNQPlWQMLdlxcCkA~GP6YdJRQ7qFCb2jRVLSFHfAtxn6Ocqo--ZUsYHg9QebMO-1GaE~z-OUe7BymU-Qu~XxNIPiQvQED-wVye0mh8VkjrjJ0EYPilvQyfNH5RbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inference of the past genome rearrangements via IS6100 in UT26 (A), TKS (B), MM-1 (C), and MI1205 (D). Blue pentagons, triangles with alphabet, and red pentagons indicate IS6100, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see Supplementary Table S8 and Supplementary Fig. S11 for detail: note that sequences shown in Supplementary Table S8 are cyan strands of 8-bp targets in Supplementary Fig. S11). Blue pentagons marked with internal white circle and triangle indicate the IS6100 element which transposed and mediated homologous recombination, respectively. IS6100 is a ‘replicative’ IS element, and it increases its copy number with the transposition (Supplementary Fig. S11). Only replicons carrying IS6100 are illustrated, and relative positions and directions of IS6100 and lin genes in each replicon are schematically shown. The IS6100 elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS6100s and lin genes in each replicon." path-from-xml="dsw041f6ap.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005589/dsw041f6ap.tif" path-from-xml="dsw041f6ap.tif" target="_blank">View large</a><a section="48005589" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f6ap.png?Expires=2147483647&amp;Signature=C~trAdH4srgIqQiumQX1so6v6YGeGHCJb2iKpxDyfFgQqEzw8203seOJrQpCTKEE-L6SQaqsEbLI~LvQjuFX0rBv7OTUXjUxvJdIdt0GRZVM0ZGtCbzJJDvbT3ASyLcFBAbh2lKQqCKLsg~LL86Ocl2aK6E5eoaX7HeazsvdPXaDvUINnxtpfENUHfC6FREQeGPOsEpS4pz-S3UN08ZlRJ9NHaj0DWTG12RNuRFOzabfEvNLSnrZfpfg3D-NKKtXotFSoEJu-L9TObRCn5kiWMt48wxwptR2Mw38NehacJ6FvcYsW2kpDVnADyVsYzaZaoekQ8uDLI3LgMUSKlgy~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005589&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f6ap.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f6bp.png?Expires=2147483647&amp;Signature=3UMqWKvdD7t3RaUFf~jdz7UGk5ycIx~zklpki7Gn9YEuc1WunQF0SvbzloheYMrYWsJyCPTpeZvgVWu4cJAUQIRGDOV7J6V1tig0BJi-nSPt3zc1ZCzh5FQ2sN--bAHxxTFtOQzeucr-hn3BsO7v3DmGEr0gXbbx53klrX3oB6h8DU71Pyw9SHq9jg43Aie5ZS8iJD4D8QApGD1YbFyCq9dGHV8WmaECT2mgacxwgBrbfRwy1p-~jXohlgxag9Q0LFtxZmcPOQ0QS2R1CJOst1VXDDxDRRVWNQrG8Pi0f~WktZNd~k~fr-lu~ma-ZEYi52Zw6P8EVUc3IbgC8f3B2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inference of the past genome rearrangements via IS6100 in UT26 (A), TKS (B), MM-1 (C), and MI1205 (D). Blue pentagons, triangles with alphabet, and red pentagons indicate IS6100, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see Supplementary Table S8 and Supplementary Fig. S11 for detail: note that sequences shown in Supplementary Table S8 are cyan strands of 8-bp targets in Supplementary Fig. S11). Blue pentagons marked with internal white circle and triangle indicate the IS6100 element which transposed and mediated homologous recombination, respectively. IS6100 is a ‘replicative’ IS element, and it increases its copy number with the transposition (Supplementary Fig. S11). Only replicons carrying IS6100 are illustrated, and relative positions and directions of IS6100 and lin genes in each replicon are schematically shown. The IS6100 elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS6100s and lin genes in each replicon." path-from-xml="dsw041f6bp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005589/dsw041f6bp.tif" path-from-xml="dsw041f6bp.tif" target="_blank">View large</a><a section="48005589" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f6bp.png?Expires=2147483647&amp;Signature=22~yEbIqAYvr78sGdDMpxZJdegSKkj1blgN2bob8COS44eIF6ZLQ~suhEfWM8ZeQVAtVUY83uKxhhJXNgPJMfXg1kJANp3IZ0xcee-B0YLRZjWnnu5g8na8f6gh3dLVGYDNX4QiEh3FUPzVhe6-VrqtvqmjTHUWnLfjB-L7Sd5YeZG3Z88fYANhfKjhFHmWIIvhm35tbWjkw2GBs~LnISINhxAem7FITTCTxsCP7jnGUfwsFCzvAIxzORld54n4nGMUF4o3~hnw~f1jud47Z9qmlPxWBXTHEzNtKny-e2s~Ji-Hb1JC9NJRzdXthQxJENDOdR9GJzqjdHS6btcfblA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005589&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f6bp.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f6cp.png?Expires=2147483647&amp;Signature=PX~NzMNQ~wLsuMuNsT8IkTKsQDHo6SiRzF8Z3F4aPZbzBL2YAzX3rWZN7NJ7VHdvr939ZLT5-Nm4zwn6KebZexc9PmNH3gX8fo3ghG83SqKdCuFVpElNWw~sOIhb3Zwblqh4evP7lQcM3xX7li6ZQLoAcLSejHwNp1SLlNMvGL2vFg2SCwiyV5Fl5N45LX9ybO6h1mgfY3fnkD1nSuKHu0Ep8~oZf1BoH5qerCSyv5woDmRHkvgZ-iPOIKrROa4wbVHN9MqtcH0UqkMS~nrAE8AYLiIm3DfAfVMYAJCD27CM5FQQB00~OVdt9v3FVrhBqCAeGG6Uu32Ug48ABytfkw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inference of the past genome rearrangements via IS6100 in UT26 (A), TKS (B), MM-1 (C), and MI1205 (D). Blue pentagons, triangles with alphabet, and red pentagons indicate IS6100, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see Supplementary Table S8 and Supplementary Fig. S11 for detail: note that sequences shown in Supplementary Table S8 are cyan strands of 8-bp targets in Supplementary Fig. S11). Blue pentagons marked with internal white circle and triangle indicate the IS6100 element which transposed and mediated homologous recombination, respectively. IS6100 is a ‘replicative’ IS element, and it increases its copy number with the transposition (Supplementary Fig. S11). Only replicons carrying IS6100 are illustrated, and relative positions and directions of IS6100 and lin genes in each replicon are schematically shown. The IS6100 elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS6100s and lin genes in each replicon." path-from-xml="dsw041f6cp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005589/dsw041f6cp.tif" path-from-xml="dsw041f6cp.tif" target="_blank">View large</a><a section="48005589" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f6cp.png?Expires=2147483647&amp;Signature=gadi12KrPFpICl2lMZVWpjQdoPlAOgMVJdpw6mQYOm3Hr8M3LXHYLGyYO9jMfdZVaJZdN7X6X0uF2erUbWEeQjP781t6x4jqPeYAIu2r050QNtXiVRyHYBOEAP2PkpGixqQ8kfmh0UYHmdcvmWusARVxxqBPbUBSKY8Dbv-sB4SzBlIWelpMgDLRnq0eub~9pv9zWDhv0gzz5-ta~KRCjwcBgB~kN2OO9Pooed4F09yd9Qzfup9TG7FXa-h3qJSOpVCIQIzGDVC12kC~5Bhe0YTB9DwwReo2Z9c0fJodE~kPc1IyUPu4CeSBbe-xMlsKri5vSvbJjnFrxz8MM9ufoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005589&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f6cp.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inference of the past genome rearrangements via IS<em>6100</em> in UT26 <strong>(A)</strong>, TKS <strong>(B)</strong>, MM-1 <strong>(C)</strong>, and MI1205 <strong>(D)</strong>. Blue pentagons, triangles with alphabet, and red pentagons indicate IS<em>6100</em>, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span> for detail: note that sequences shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> are cyan strands of 8-bp targets in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Blue pentagons marked with internal white circle and triangle indicate the IS<em>6100</em> element which transposed and mediated homologous recombination, respectively. IS<em>6100</em> is a ‘replicative’ IS element, and it increases its copy number with the transposition (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Only replicons carrying IS<em>6100</em> are illustrated, and relative positions and directions of IS<em>6100</em> and <em>lin</em> genes in each replicon are schematically shown. The IS<em>6100</em> elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS<em>6100</em>s and <em>lin</em> genes in each replicon.</p></div></div><div content-id="dsw041-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f6ap.png?Expires=2147483647&amp;Signature=ZlIEQeUY3Gacu4neXb88O2-264NuiudPzXUoyFFunmxoqWF6irIp8-JLZLMiEa2VIkvm8QtWnbdzChTxLOtfgt0tt2Ssds2R-I9ZcwKe3qPpkrC0Uemx2XO0FKtoR1RIfFjsKGZhTu7-Srf9KfR6Y5N9P8Du8sTVoy4H7Q~eC-Y22i9yJDQ0VaqA50OkhWnVb58R8f473kVZiqZNbK0qigVFa4iNQPlWQMLdlxcCkA~GP6YdJRQ7qFCb2jRVLSFHfAtxn6Ocqo--ZUsYHg9QebMO-1GaE~z-OUe7BymU-Qu~XxNIPiQvQED-wVye0mh8VkjrjJ0EYPilvQyfNH5RbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inference of the past genome rearrangements via IS6100 in UT26 (A), TKS (B), MM-1 (C), and MI1205 (D). Blue pentagons, triangles with alphabet, and red pentagons indicate IS6100, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see Supplementary Table S8 and Supplementary Fig. S11 for detail: note that sequences shown in Supplementary Table S8 are cyan strands of 8-bp targets in Supplementary Fig. S11). Blue pentagons marked with internal white circle and triangle indicate the IS6100 element which transposed and mediated homologous recombination, respectively. IS6100 is a ‘replicative’ IS element, and it increases its copy number with the transposition (Supplementary Fig. S11). Only replicons carrying IS6100 are illustrated, and relative positions and directions of IS6100 and lin genes in each replicon are schematically shown. The IS6100 elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS6100s and lin genes in each replicon." path-from-xml="dsw041f6ap.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005589/dsw041f6ap.tif" path-from-xml="dsw041f6ap.tif" target="_blank">View large</a><a section="48005589" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f6ap.png?Expires=2147483647&amp;Signature=C~trAdH4srgIqQiumQX1so6v6YGeGHCJb2iKpxDyfFgQqEzw8203seOJrQpCTKEE-L6SQaqsEbLI~LvQjuFX0rBv7OTUXjUxvJdIdt0GRZVM0ZGtCbzJJDvbT3ASyLcFBAbh2lKQqCKLsg~LL86Ocl2aK6E5eoaX7HeazsvdPXaDvUINnxtpfENUHfC6FREQeGPOsEpS4pz-S3UN08ZlRJ9NHaj0DWTG12RNuRFOzabfEvNLSnrZfpfg3D-NKKtXotFSoEJu-L9TObRCn5kiWMt48wxwptR2Mw38NehacJ6FvcYsW2kpDVnADyVsYzaZaoekQ8uDLI3LgMUSKlgy~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005589&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f6ap.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f6bp.png?Expires=2147483647&amp;Signature=3UMqWKvdD7t3RaUFf~jdz7UGk5ycIx~zklpki7Gn9YEuc1WunQF0SvbzloheYMrYWsJyCPTpeZvgVWu4cJAUQIRGDOV7J6V1tig0BJi-nSPt3zc1ZCzh5FQ2sN--bAHxxTFtOQzeucr-hn3BsO7v3DmGEr0gXbbx53klrX3oB6h8DU71Pyw9SHq9jg43Aie5ZS8iJD4D8QApGD1YbFyCq9dGHV8WmaECT2mgacxwgBrbfRwy1p-~jXohlgxag9Q0LFtxZmcPOQ0QS2R1CJOst1VXDDxDRRVWNQrG8Pi0f~WktZNd~k~fr-lu~ma-ZEYi52Zw6P8EVUc3IbgC8f3B2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inference of the past genome rearrangements via IS6100 in UT26 (A), TKS (B), MM-1 (C), and MI1205 (D). Blue pentagons, triangles with alphabet, and red pentagons indicate IS6100, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see Supplementary Table S8 and Supplementary Fig. S11 for detail: note that sequences shown in Supplementary Table S8 are cyan strands of 8-bp targets in Supplementary Fig. S11). Blue pentagons marked with internal white circle and triangle indicate the IS6100 element which transposed and mediated homologous recombination, respectively. IS6100 is a ‘replicative’ IS element, and it increases its copy number with the transposition (Supplementary Fig. S11). Only replicons carrying IS6100 are illustrated, and relative positions and directions of IS6100 and lin genes in each replicon are schematically shown. The IS6100 elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS6100s and lin genes in each replicon." path-from-xml="dsw041f6bp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005589/dsw041f6bp.tif" path-from-xml="dsw041f6bp.tif" target="_blank">View large</a><a section="48005589" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f6bp.png?Expires=2147483647&amp;Signature=22~yEbIqAYvr78sGdDMpxZJdegSKkj1blgN2bob8COS44eIF6ZLQ~suhEfWM8ZeQVAtVUY83uKxhhJXNgPJMfXg1kJANp3IZ0xcee-B0YLRZjWnnu5g8na8f6gh3dLVGYDNX4QiEh3FUPzVhe6-VrqtvqmjTHUWnLfjB-L7Sd5YeZG3Z88fYANhfKjhFHmWIIvhm35tbWjkw2GBs~LnISINhxAem7FITTCTxsCP7jnGUfwsFCzvAIxzORld54n4nGMUF4o3~hnw~f1jud47Z9qmlPxWBXTHEzNtKny-e2s~Ji-Hb1JC9NJRzdXthQxJENDOdR9GJzqjdHS6btcfblA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005589&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f6bp.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f6cp.png?Expires=2147483647&amp;Signature=PX~NzMNQ~wLsuMuNsT8IkTKsQDHo6SiRzF8Z3F4aPZbzBL2YAzX3rWZN7NJ7VHdvr939ZLT5-Nm4zwn6KebZexc9PmNH3gX8fo3ghG83SqKdCuFVpElNWw~sOIhb3Zwblqh4evP7lQcM3xX7li6ZQLoAcLSejHwNp1SLlNMvGL2vFg2SCwiyV5Fl5N45LX9ybO6h1mgfY3fnkD1nSuKHu0Ep8~oZf1BoH5qerCSyv5woDmRHkvgZ-iPOIKrROa4wbVHN9MqtcH0UqkMS~nrAE8AYLiIm3DfAfVMYAJCD27CM5FQQB00~OVdt9v3FVrhBqCAeGG6Uu32Ug48ABytfkw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inference of the past genome rearrangements via IS6100 in UT26 (A), TKS (B), MM-1 (C), and MI1205 (D). Blue pentagons, triangles with alphabet, and red pentagons indicate IS6100, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see Supplementary Table S8 and Supplementary Fig. S11 for detail: note that sequences shown in Supplementary Table S8 are cyan strands of 8-bp targets in Supplementary Fig. S11). Blue pentagons marked with internal white circle and triangle indicate the IS6100 element which transposed and mediated homologous recombination, respectively. IS6100 is a ‘replicative’ IS element, and it increases its copy number with the transposition (Supplementary Fig. S11). Only replicons carrying IS6100 are illustrated, and relative positions and directions of IS6100 and lin genes in each replicon are schematically shown. The IS6100 elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS6100s and lin genes in each replicon." path-from-xml="dsw041f6cp.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005589/dsw041f6cp.tif" path-from-xml="dsw041f6cp.tif" target="_blank">View large</a><a section="48005589" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f6cp.png?Expires=2147483647&amp;Signature=gadi12KrPFpICl2lMZVWpjQdoPlAOgMVJdpw6mQYOm3Hr8M3LXHYLGyYO9jMfdZVaJZdN7X6X0uF2erUbWEeQjP781t6x4jqPeYAIu2r050QNtXiVRyHYBOEAP2PkpGixqQ8kfmh0UYHmdcvmWusARVxxqBPbUBSKY8Dbv-sB4SzBlIWelpMgDLRnq0eub~9pv9zWDhv0gzz5-ta~KRCjwcBgB~kN2OO9Pooed4F09yd9Qzfup9TG7FXa-h3qJSOpVCIQIzGDVC12kC~5Bhe0YTB9DwwReo2Z9c0fJodE~kPc1IyUPu4CeSBbe-xMlsKri5vSvbJjnFrxz8MM9ufoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005589&amp;ar=2647451&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw041f6cp.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inference of the past genome rearrangements via IS<em>6100</em> in UT26 <strong>(A)</strong>, TKS <strong>(B)</strong>, MM-1 <strong>(C)</strong>, and MI1205 <strong>(D)</strong>. Blue pentagons, triangles with alphabet, and red pentagons indicate IS<em>6100</em>, 8-bp target sites, and lin genes, respectively. Triangles with the same alphabet mean identical sequence and direction (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span> for detail: note that sequences shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> are cyan strands of 8-bp targets in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Blue pentagons marked with internal white circle and triangle indicate the IS<em>6100</em> element which transposed and mediated homologous recombination, respectively. IS<em>6100</em> is a ‘replicative’ IS element, and it increases its copy number with the transposition (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Only replicons carrying IS<em>6100</em> are illustrated, and relative positions and directions of IS<em>6100</em> and <em>lin</em> genes in each replicon are schematically shown. The IS<em>6100</em> elements involved in the proposed past genome rearrangements are shown in larger size. Numbers in current forms of the four strains indicate locations of IS<em>6100</em>s and <em>lin</em> genes in each replicon.</p></div></div></div><div class="block-child-p">The specific <em>lin</em>-flanking regions in the four strains were compared (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). Not only the <em>lin</em> genes themselves (<a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>) but also their flanking regions are highly conserved (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). Interestingly, such conserved regions are located very close to IS<em>6100</em> and the distances between the IS<em>6100</em> copies and the <em>lin</em> genes are varied (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). This means that IS<em>6100</em> is likely to play a crucial ‘editing’ role in the ‘trimming’ of ‘unnecessary regions’ for HCH utilization and the ‘gathering’ of the specific <em>lin</em> genes. At least, it is the most plausible that the transposition of IS<em>6100</em> led to the diversification of the distribution and organization of the <em>lin</em> genes in the genomes. The distance between IS<em>6100</em> and <em>linA</em> is the longest in UT26 (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>A), and the <em>linB</em> gene in UT26 has no IS<em>6100</em> element in its flanking regions (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>B). Moreover, IS<em>6100</em> is located at only one side of <em>linC</em> (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>C) and the <em>linRED</em> cluster (<a reveal-id="dsw041-F7" data-open="dsw041-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>D) in UT26. These results suggested that UT26 is the closest to the prototype of the γ-HCH degrader, at least among the four strains examined in this study. <div data-id="dsw041-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f7p.png?Expires=2147483647&Signature=zo-jeP4CwNlwC56MzvYd6yecRFNO1lXUP-GlRwZP1rLWbBqkzIC3Jxj9L0nXHBeRXomiryVWAnM5SwKE6A9VemK5IhKKmWKb4RWTY7YvXSZr7J059Y8b3zEVgHRXGg6gTO1A~FYU5tj-TDTSe7XmcBMfT1Cj7BiRtZGEGRis8-hMeA4rXKkUMYpG3aC60w4vNLAZeeEWioMJlCj81ZOanruOXTXp8SPbcGTuL-iIZByfBz879OpVxRB10xJ4OjOmg1sdk0ublC169OtrKr8gDU2LBAaYQ6qWxrjYgQ-wMITzLGAVVVJhLFXu19M~l19Qb1mazpMkhLSWRRnaSsuHXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of regions containing the specific lin genes in the four γ-HCH-degrading sphingomonad strains. The regions containing linA(A), linB(B), linC(C), and linRED cluster (D) were compared. The regions homologous to each other were coloured in the gradient depending on the level of similarity as shown in explanatory note. The lin genes, transposase gene of IS6100, and other ORFs were shown by pentagons in red, blue, and orange, respectively. The pseudo ‘linE’ gene exists in pIM2 of MI1205 at the region corresponding to the linE gene in other plasmids." path-from-xml="dsw041f7p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005590/dsw041f7p.tif" path-from-xml="dsw041f7p.tif" target="_blank">View large</a><a section="48005590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f7p.png?Expires=2147483647&Signature=MbDxGcD3UNNxhg-xxKI4~tC6ZymTYQZpJZ5QZWbk8x5NtRsRBa8Nd~qvYSsh0aq8Z1wl2Pv~i9gDN1ihid98WeOiiCk4D8eqCgj8hygEDC~4YgW5HOB~k1v~k57DX7f2P9KaTxW6PUPUtufDuBfErK7qAf6-Uo-NU61K7jhl8Ktw8XkQqKQ3gffOcVr8iUtUe4dQzbLI0WtKjOAe5nAKJq5Te9atSmy~OtXAwwlWEpk3tOVZagKco8~es1j-vr6VYTSpQOzxZvxUDn9~dg~VuvKyAM2wcXrHCvubr8fFZuBHSZfARiwZX5nUMlTcoYbb2H0Pc2g1Ji7VJj-~e0fSeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005590&ar=2647451&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw041f7p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of regions containing the specific <em>lin</em> genes in the four γ-HCH-degrading sphingomonad strains. The regions containing <em>linA</em><strong>(A)</strong>, <em>linB</em><strong>(B)</strong>, <em>linC</em><strong>(C)</strong>, and <em>linRED</em> cluster <strong>(D)</strong> were compared. The regions homologous to each other were coloured in the gradient depending on the level of similarity as shown in explanatory note. The <em>lin</em> genes, transposase gene of IS<em>6100</em>, and other ORFs were shown by pentagons in red, blue, and orange, respectively. The pseudo ‘<em>linE</em>’ gene exists in pIM2 of MI1205 at the region corresponding to the <em>linE</em> gene in other plasmids.</p></div></div><div content-id="dsw041-F7" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/m_dsw041f7p.png?Expires=2147483647&Signature=zo-jeP4CwNlwC56MzvYd6yecRFNO1lXUP-GlRwZP1rLWbBqkzIC3Jxj9L0nXHBeRXomiryVWAnM5SwKE6A9VemK5IhKKmWKb4RWTY7YvXSZr7J059Y8b3zEVgHRXGg6gTO1A~FYU5tj-TDTSe7XmcBMfT1Cj7BiRtZGEGRis8-hMeA4rXKkUMYpG3aC60w4vNLAZeeEWioMJlCj81ZOanruOXTXp8SPbcGTuL-iIZByfBz879OpVxRB10xJ4OjOmg1sdk0ublC169OtrKr8gDU2LBAaYQ6qWxrjYgQ-wMITzLGAVVVJhLFXu19M~l19Qb1mazpMkhLSWRRnaSsuHXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of regions containing the specific lin genes in the four γ-HCH-degrading sphingomonad strains. The regions containing linA(A), linB(B), linC(C), and linRED cluster (D) were compared. The regions homologous to each other were coloured in the gradient depending on the level of similarity as shown in explanatory note. The lin genes, transposase gene of IS6100, and other ORFs were shown by pentagons in red, blue, and orange, respectively. The pseudo ‘linE’ gene exists in pIM2 of MI1205 at the region corresponding to the linE gene in other plasmids." path-from-xml="dsw041f7p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005590/dsw041f7p.tif" path-from-xml="dsw041f7p.tif" target="_blank">View large</a><a section="48005590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041f7p.png?Expires=2147483647&Signature=MbDxGcD3UNNxhg-xxKI4~tC6ZymTYQZpJZ5QZWbk8x5NtRsRBa8Nd~qvYSsh0aq8Z1wl2Pv~i9gDN1ihid98WeOiiCk4D8eqCgj8hygEDC~4YgW5HOB~k1v~k57DX7f2P9KaTxW6PUPUtufDuBfErK7qAf6-Uo-NU61K7jhl8Ktw8XkQqKQ3gffOcVr8iUtUe4dQzbLI0WtKjOAe5nAKJq5Te9atSmy~OtXAwwlWEpk3tOVZagKco8~es1j-vr6VYTSpQOzxZvxUDn9~dg~VuvKyAM2wcXrHCvubr8fFZuBHSZfARiwZX5nUMlTcoYbb2H0Pc2g1Ji7VJj-~e0fSeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005590&ar=2647451&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw041f7p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of regions containing the specific <em>lin</em> genes in the four γ-HCH-degrading sphingomonad strains. The regions containing <em>linA</em><strong>(A)</strong>, <em>linB</em><strong>(B)</strong>, <em>linC</em><strong>(C)</strong>, and <em>linRED</em> cluster <strong>(D)</strong> were compared. The regions homologous to each other were coloured in the gradient depending on the level of similarity as shown in explanatory note. The <em>lin</em> genes, transposase gene of IS<em>6100</em>, and other ORFs were shown by pentagons in red, blue, and orange, respectively. The pseudo ‘<em>linE</em>’ gene exists in pIM2 of MI1205 at the region corresponding to the <em>linE</em> gene in other plasmids.</p></div></div></div>                            <h3 scrollto-destination=48005591 id="48005591" class="section-title" data-legacy-id=SEC3.8>3.8. Conclusions and perspectives</h3>
<p>Our comparison of the complete genome sequences of four γ-HCH-degrading sphingomonad strains and gathering of experimental data in this study demonstrate or strongly suggest the following points: (i) the gene repertoires and genomic organizations of the four γ-HCH-degrading strains, which are phylogenetically dispersed among related sphingomonad strains (<a reveal-id="dsw041-F2" data-open="dsw041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), are relatively different from one another (<a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <a reveal-id="dsw041-F3" data-open="dsw041-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>); (ii) all four strains carry almost identical <em>linA</em> to <em>linE</em> genes for the conversion of γ-HCH to maleylacetate (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> and <a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>); (iii) considerably different genes are used for the metabolism of maleylacetate in TKS (<a reveal-id="dsw041-F1" data-open="dsw041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, <a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>); (iv) the <em>linKLMN</em> genes for the putative ABC transporter necessary for γ-HCH utilization are structurally divergent, and such divergence reflects the phylogenetic relationship of their hosts (<a reveal-id="dsw041-F2" data-open="dsw041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span> and <a reveal-id="dsw041-T2" data-open="dsw041-T2" class="link link-reveal link-table xref-fig">Table 2</a>); (v) most of the <em>linA</em> to <em>linJ</em> genes for the catabolic enzymes are located on several replicons whose replication/partition systems are highly conserved among sphingomonad plasmids (<a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsw041-T3" data-open="dsw041-T3" class="link link-reveal link-table xref-fig">3</a>); and (vi) the transposition of IS<em>6100</em> has caused dynamic genome rearrangements including the fusion and resolution of replicons and the diversification of <em>lin</em>-flanking regions in the four strains (<a reveal-id="dsw041-F6 dsw041-F7" data-open="dsw041-F6 dsw041-F7" class="link link-reveal link-table xref-fig">Figs. 6 and 7</a>).</p><p>Based on our results in this study, we propose that these γ-HCH-degraders were formed independently in different geographic regions through the recruitment of specific <em>lin</em> genes and genes for the metabolism of maleylacetate into ancestral strains that had the core functions (including the <em>linKLMN</em>-encoded one) of sphingomonads. Multiple plasmids whose replication/partition machineries are highly conserved in sphingomonads might have played important roles in the recruitment of the specific <em>lin</em> genes by their horizontal transfer. In addition, IS<em>6100</em> likely plays a crucial ‘editing’ role in the distribution and organization of the <em>lin</em> genes in genomes. In the future, our hypothesis may be confirmed in experiments using the four HCH degraders and their related but non-HCH-degrading and/or IS<em>6100</em>-free sphingomonad strains.</p>                            <h2 scrollto-destination=48005594 id="48005594" class="section-title" data-legacy-id=SEC4>4. Data availability</h2>
<p>The sequences with the annotation of replicons in MM-1, MI1205, and TKS have been deposited in DDBJ/EMBL/GenBank databases under the accession numbers shown in <a reveal-id="dsw041-T1" data-open="dsw041-T1" class="link link-reveal link-table xref-fig">Table 1</a>. Nucleotide sequences of the Tn<em>3</em>-type transposons, Tn<em>6268</em> to Tn<em>6278</em>, were deposited in the DDBJ/EMBL/GenBank databases under accession numbers LC102249 to LC102259, respectively.</p>                            <h2 scrollto-destination=48005596 id="48005596" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=48005598 id="48005598" class="section-title" >Accession number</h2>
<p>CP004036 to CP004041, CP005083 to CP005093, CP005188 to CP005193, and LC102249 to LC102259.</p>                            <h2 scrollto-destination=48005600 id="48005600" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=48005602 id="48005602" class="section-title" >Funding</h2>
<p>This work was supported by Japan Society for the Promotion of Science (JSPS) KAKENHI grant numbers (22380047 and 25292043).</p>                            <h2 scrollto-destination=48005604 id="48005604" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw041-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ogata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Takada</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Mizukawa</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">International Pellet Watch: global monitoring of persistent organic pollutants (POPs) in coastal waters. 1. Initial phase data on PCBs, DDTs, and HCHs,</div> <div class="source">Mar. Pollut. Bull</div> ., <div class="volume">58</div>, <div class="fpage">1437</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=International%20Pellet%20Watch%3A%20global%20monitoring%20of%20persistent%20organic%20pollutants%20(POPs)%20in%20coastal%20waters.%201.%20Initial%20phase%20data%20on%20PCBs%2C%20DDTs%2C%20and%20HCHs%2C&amp;author=Y.%20Ogata&amp;author=H.%20Takada&amp;author=K.%20Mizukawa&amp;publication_year=2009&amp;journal=Mar.%20Pollut.%20Bull&amp;volume=58&amp;pages=1437-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.marpolbul.2009.06.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.marpolbul.2009.06.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.marpolbul.2009.06.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19635625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">El-Shahawi</div> <div class="given-names">M. S.</div></div><div class="name string-name"><div class="surname">Hamza</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bashammakh</div> <div class="given-names">A. S.</div></div><div class="name string-name"><div class="surname">Al-Saggaf</div> <div class="given-names">W. T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">An overview on the accumulation, distribution, transformations, toxicity and analytical methods for the monitoring of persistent organic pollutants,</div> <div class="source">Talanta</div> , <div class="volume">80</div>, <div class="fpage">1587</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20overview%20on%20the%20accumulation%2C%20distribution%2C%20transformations%2C%20toxicity%20and%20analytical%20methods%20for%20the%20monitoring%20of%20persistent%20organic%20pollutants%2C&amp;author=M.%20S.%20El-Shahawi&amp;author=A.%20Hamza&amp;author=A.%20S.%20Bashammakh&amp;author=W.%20T.%20Al-Saggaf&amp;publication_year=2010&amp;journal=Talanta&amp;volume=80&amp;pages=1587-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.talanta.2009.09.055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.talanta.2009.09.055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.talanta.2009.09.055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20152382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tarcau</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Cucu-Man</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Boruvkova</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Klanova</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Covaci</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Organochlorine pesticides in soil, moss and tree-bark from North-Eastern Romania,</div> <div class="source">Sci. Total Environ</div> ., <div class="volume">456–457</div>, <div class="fpage">317</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organochlorine%20pesticides%20in%20soil%2C%20moss%20and%20tree-bark%20from%20North-Eastern%20Romania%2C&amp;author=D.%20Tarcau&amp;author=S.%20Cucu-Man&amp;author=J.%20Boruvkova&amp;author=J.%20Klanova&amp;author=A.%20Covaci&amp;publication_year=2013&amp;journal=Sci.%20Total%20Environ&amp;volume=456–457&amp;pages=317-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scitotenv.2013.03.103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scitotenv.2013.03.103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scitotenv.2013.03.103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Janssen</div> <div class="given-names">D. B.</div></div><div class="name string-name"><div class="surname">Dinkla</div> <div class="given-names">I. J.</div></div><div class="name string-name"><div class="surname">Poelarends</div> <div class="given-names">G. J.</div></div><div class="name string-name"><div class="surname">Terpstra</div> <div class="given-names">P.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Bacterial degradation of xenobiotic compounds: evolution and distribution of novel enzyme activities,</div> <div class="source">Environ. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">1868</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20degradation%20of%20xenobiotic%20compounds%3A%20evolution%20and%20distribution%20of%20novel%20enzyme%20activities%2C&amp;author=D.%20B.%20Janssen&amp;author=I.%20J.%20Dinkla&amp;author=G.%20J.%20Poelarends&amp;author=P.%20Terpstra&amp;publication_year=2005&amp;journal=Environ.%20Microbiol&amp;volume=7&amp;pages=1868-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-2920.2005.00966.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2005.00966.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-2920.2005.00966.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16309386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Copley</div> <div class="given-names">S. D.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Evolution of efficient pathways for degradation of anthropogenic chemicals,</div> <div class="source">Nature Chem. Biol</div> ., <div class="volume">5</div>, <div class="fpage">559</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20efficient%20pathways%20for%20degradation%20of%20anthropogenic%20chemicals%2C&amp;author=S.%20D.%20Copley&amp;publication_year=2009&amp;journal=Nature%20Chem.%20Biol&amp;volume=5&amp;pages=559-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nchembio.197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnchembio.197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnchembio.197"> </span></div> </div></div></div></div></div><div content-id="dsw041-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stolz</div> <div class="given-names">A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Molecular characteristics of xenobiotic-degrading sphingomonads,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">81</div>, <div class="fpage">793</div>–<div class="lpage">811</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characteristics%20of%20xenobiotic-degrading%20sphingomonads%2C&amp;author=A.%20Stolz&amp;publication_year=2009&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=81&amp;pages=793-811" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-008-1752-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-008-1752-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-008-1752-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19002456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lal</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Pandey</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Biochemistry of microbial degradation of hexachlorocyclohexane and prospects for bioremediation,</div> <div class="source">Microbiol. Mol. Biol. Rev</div> ., <div class="volume">74</div>, <div class="fpage">58</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemistry%20of%20microbial%20degradation%20of%20hexachlorocyclohexane%20and%20prospects%20for%20bioremediation%2C&amp;author=R.%20Lal&amp;author=G.%20Pandey&amp;author=P.%20Sharma&amp;publication_year=2010&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=74&amp;pages=58-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00029-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00029-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00029-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20197499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stolz</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Degradative plasmids from sphingomonads,</div> <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">350</div>, <div class="fpage">9</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degradative%20plasmids%20from%20sphingomonads%2C&amp;author=A.%20Stolz&amp;publication_year=2014&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=350&amp;pages=9-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2014.350.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2014.350.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2014.350.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24111699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, Biodegradation of organochlorine pesticides. In: <div class="person-group"><div class="name string-name"><div class="surname">Yates</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nakatsu</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div></div> (eds), <div class="source">Manual of Environmental Microbiology</div> , <div class="edition">4th edn.,</div> 
<div class="publisher-name">ASM Press</div>, 
<div class="publisher-loc">Washington, DC</div>. pp. 5.1.2-1–30.</div></div></div></div><div content-id="dsw041-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Top</div> <div class="given-names">E. M.</div></div><div class="name string-name"><div class="surname">Springael</div> <div class="given-names">D.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">The role of mobile genetic elements in bacterial adaptation to xenobiotic organic compounds,</div> <div class="source">Curr. Opin. Biotechnol</div> ., <div class="volume">14</div>, <div class="fpage">262</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20mobile%20genetic%20elements%20in%20bacterial%20adaptation%20to%20xenobiotic%20organic%20compounds%2C&amp;author=E.%20M.%20Top&amp;author=D.%20Springael&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Biotechnol&amp;volume=14&amp;pages=262-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(03)00066-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(03)00066-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(03)00066-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12849778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Copley</div> <div class="given-names">S. D.</div></div><div class="name string-name"><div class="surname">Rokicki</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Turner</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Daligault</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Nolan</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Land</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The whole genome sequence of <em>Sphingobium chlorophenolicum</em> L-1: insights into the evolution of the pentachlorophenol degradation pathway,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">4</div>, <div class="fpage">184</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20whole%20genome%20sequence%20of%20Sphingobium%20chlorophenolicum%20L-1%3A%20insights%20into%20the%20evolution%20of%20the%20pentachlorophenol%20degradation%20pathway%2C&amp;author=S.%20D.%20Copley&amp;author=J.%20Rokicki&amp;author=P.%20Turner&amp;author=H.%20Daligault&amp;author=M.%20Nolan&amp;author=M.%20Land&amp;publication_year=2011&amp;journal=Genome%20Biol.%20Evol&amp;volume=4&amp;pages=184-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evr137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevr137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevr137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22179583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Udikovic-Kolic</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Scott</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Martin-Laurent</div> <div class="given-names">F.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evolution of atrazine-degrading capabilities in the environment,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">96</div>, <div class="fpage">1175</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20atrazine-degrading%20capabilities%20in%20the%20environment%2C&amp;author=N.%20Udikovic-Kolic&amp;author=C.%20Scott&amp;author=F.%20Martin-Laurent&amp;publication_year=2012&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=96&amp;pages=1175-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-012-4495-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-012-4495-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-012-4495-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23076592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Satola</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Wubbeler</div> <div class="given-names">J. H.</div></div><div class="name string-name"><div class="surname">Steinbuchel</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Metabolic characteristics of the species <em>Variovorax paradoxus</em>,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">97</div>, <div class="fpage">541</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20characteristics%20of%20the%20species%20Variovorax%20paradoxus%2C&amp;author=B.%20Satola&amp;author=J.%20H.%20Wubbeler&amp;author=A.%20Steinbuchel&amp;publication_year=2013&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=97&amp;pages=541-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-012-4585-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-012-4585-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-012-4585-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Phillips</div> <div class="given-names">T. M.</div></div><div class="name string-name"><div class="surname">Seech</div> <div class="given-names">A. G.</div></div><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Trevors</div> <div class="given-names">J. T.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Biodegradation of hexachlorocyclohexane (HCH) by microorganisms,</div> <div class="source">Biodegradation</div> , <div class="volume">16</div>, <div class="fpage">363</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biodegradation%20of%20hexachlorocyclohexane%20(HCH)%20by%20microorganisms%2C&amp;author=T.%20M.%20Phillips&amp;author=A.%20G.%20Seech&amp;author=H.%20Lee&amp;author=J.%20T.%20Trevors&amp;publication_year=2005&amp;journal=Biodegradation&amp;volume=16&amp;pages=363-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10532-004-2413-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10532-004-2413-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10532-004-2413-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15865341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vijgen</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Abhilash</div> <div class="given-names">P. C.</div></div><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y. F.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Hexachlorocyclohexane (HCH) as new Stockholm Convention POPs-a global perspective on the management of Lindane and its waste isomers</div>, <div class="source">Environ. Sci. Pollut. Res. Int</div> ., <div class="volume">18</div>, <div class="fpage">152</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hexachlorocyclohexane%20(HCH)%20as%20new%20Stockholm%20Convention%20POPs-a%20global%20perspective%20on%20the%20management%20of%20Lindane%20and%20its%20waste%20isomers&amp;author=J.%20Vijgen&amp;author=P.%20C.%20Abhilash&amp;author=Y.%20F.%20Li&amp;publication_year=2011&amp;journal=Environ.%20Sci.%20Pollut.%20Res.%20Int&amp;volume=18&amp;pages=152-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11356-010-0417-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11356-010-0417-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11356-010-0417-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21104204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Ito</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Aerobic degradation of lindane (γ-hexachlorocyclohexane) in bacteria and its biochemical and molecular basis,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">76</div>, <div class="fpage">741</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aerobic%20degradation%20of%20lindane%20(%CE%B3-hexachlorocyclohexane)%20in%20bacteria%20and%20its%20biochemical%20and%20molecular%20basis%2C&amp;author=Y.%20Nagata&amp;author=R.%20Endo&amp;author=M.%20Ito&amp;author=Y.%20Ohtsubo&amp;author=M.%20Tsuda&amp;publication_year=2007&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=76&amp;pages=741-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-007-1066-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-007-1066-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-007-1066-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17634937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Complete genome sequence of the representative γ-hexachlorocyclohexane-degrading bacterium <em>Sphingobium japonicum</em> UT26,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">192</div>, <div class="fpage">5852</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20representative%20%CE%B3-hexachlorocyclohexane-degrading%20bacterium%20Sphingobium%20japonicum%20UT26%2C&amp;author=Y.%20Nagata&amp;author=Y.%20Ohtsubo&amp;author=R.%20Endo&amp;publication_year=2010&amp;journal=J.%20Bacteriol&amp;volume=192&amp;pages=5852-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00961-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00961-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00961-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20817768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Natsui</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Genomic organization and genomic structural rearrangements of <em>Sphingobium japonicum</em> UT26, an archetypal γ-hexachlorocyclohexane-degrading bacterium,</div> <div class="source">Enzyme Microb. Technol</div> ., <div class="volume">49</div>, <div class="fpage">499</div>–<div class="lpage">508</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20organization%20and%20genomic%20structural%20rearrangements%20of%20Sphingobium%20japonicum%20UT26%2C%20an%20archetypal%20%CE%B3-hexachlorocyclohexane-degrading%20bacterium%2C&amp;author=Y.%20Nagata&amp;author=S.%20Natsui&amp;author=R.%20Endo&amp;publication_year=2011&amp;journal=Enzyme%20Microb.%20Technol&amp;volume=49&amp;pages=499-508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.enzmictec.2011.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.enzmictec.2011.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.enzmictec.2011.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22142724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohhata</div> <div class="given-names">S.,</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, Appearance and evolution of γ-hexachlorocyclohexane-degrading bacteria, In: <div class="person-group"><div class="name string-name"><div class="surname">Nojiri</div>  <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Fukuda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kamagata</div> <div class="given-names">Y.</div></div></div> (eds) <div class="source">Biodegradative Bacteria</div> , 
<div class="publisher-name">Springer Verlag</div>: 
<div class="publisher-loc">Tokyo</div>, pp. <div class="fpage">19</div>–<div class="lpage">41</div>.</div></div></div></div><div content-id="dsw041-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Verma</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Oldach</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Comparative genomic analysis of nine <em>Sphingobium</em> strains: insights into their evolution and hexachlorocyclohexane (HCH) degradation pathways,</div> <div class="source">BMC genomics</div> , <div class="volume">15</div>, <div class="fpage">1014</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20analysis%20of%20nine%20Sphingobium%20strains%3A%20insights%20into%20their%20evolution%20and%20hexachlorocyclohexane%20(HCH)%20degradation%20pathways%2C&amp;author=H.%20Verma&amp;author=R.%20Kumar&amp;author=P.%20Oldach&amp;publication_year=2014&amp;journal=BMC%20genomics&amp;volume=15&amp;pages=1014" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-1014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-1014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25418849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pearce</div> <div class="given-names">S. L.</div></div><div class="name string-name"><div class="surname">Oakeshott</div> <div class="given-names">J. G.</div></div><div class="name string-name"><div class="surname">Pandey</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Insights into ongoing evolution of the hexachlorocyclohexane catabolic pathway from comparative genomics of ten Sphingomonadaceae strains,</div> <div class="source">G3</div> , <div class="volume">5</div>, <div class="fpage">1081</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20ongoing%20evolution%20of%20the%20hexachlorocyclohexane%20catabolic%20pathway%20from%20comparative%20genomics%20of%20ten%20Sphingomonadaceae%20strains%2C&amp;author=S.%20L.%20Pearce&amp;author=J.%20G.%20Oakeshott&amp;author=G.%20Pandey&amp;publication_year=2015&amp;journal=G3&amp;volume=5&amp;pages=1081-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.015933" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.015933" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.015933"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25850427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Ito</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The <em>lin</em> genes for γ-hexachlorocyclohexane degradation in <em>Sphingomonas</em> sp. MM-1 proved to be dispersed across multiple plasmids,</div> <div class="source">Biosci. Biotechnol. Biochem</div> ., <div class="volume">75</div>, <div class="fpage">466</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20lin%20genes%20for%20%CE%B3-hexachlorocyclohexane%20degradation%20in%20Sphingomonas%20sp.%20MM-1%20proved%20to%20be%20dispersed%20across%20multiple%20plasmids%2C&amp;author=M.%20Tabata&amp;author=R.%20Endo&amp;author=M.%20Ito&amp;publication_year=2011&amp;journal=Biosci.%20Biotechnol.%20Biochem&amp;volume=75&amp;pages=466-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.100652" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.100652" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.100652"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21389627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ohhata</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Complete genome sequence of the γ-hexachlorocyclohexane-degrading bacterium <em>Sphingomonas</em> sp. strain MM-1,</div> <div class="source">Genome Announc</div> ., <div class="volume">1</div>, <div class="fpage">e00247–13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20%CE%B3-hexachlorocyclohexane-degrading%20bacterium%20Sphingomonas%20sp.%20strain%20MM-1%2C&amp;author=M.%20Tabata&amp;author=Y.%20Ohtsubo&amp;author=S.%20Ohhata&amp;author=M.%20Tsuda&amp;author=Y.%20Nagata&amp;publication_year=2013&amp;journal=Genome%20Announc&amp;volume=1&amp;pages=e00247–13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/genomeA.00247-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FgenomeA.00247-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FgenomeA.00247-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23682148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ito</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Prokop</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Klvana</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Degradation of β-hexachlorocyclohexane by haloalkane dehalogenase LinB from γ-hexachlorocyclohexane-utilizing bacterium <em>Sphingobium</em> sp. MI1205,</div> <div class="source">Arch. Microbiol</div> ., <div class="volume">188</div>, <div class="fpage">313</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degradation%20of%20%CE%B2-hexachlorocyclohexane%20by%20haloalkane%20dehalogenase%20LinB%20from%20%CE%B3-hexachlorocyclohexane-utilizing%20bacterium%20Sphingobium%20sp.%20MI1205%2C&amp;author=M.%20Ito&amp;author=Z.%20Prokop&amp;author=M.%20Klvana&amp;publication_year=2007&amp;journal=Arch.%20Microbiol&amp;volume=188&amp;pages=313-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00203-007-0251-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00203-007-0251-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00203-007-0251-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17516046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohhata</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Nikawadori</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Complete genome sequence of a γ-hexachlorocyclohexane-degrading bacterium, <em>Sphingobium</em> sp. strain MI1205,</div> <div class="source">Genome Announc</div> ., <div class="volume">4</div>, <div class="fpage">e00246</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20a%20%CE%B3-hexachlorocyclohexane-degrading%20bacterium%2C%20Sphingobium%20sp.%20strain%20MI1205%2C&amp;author=M.%20Tabata&amp;author=S.%20Ohhata&amp;author=Y.%20Nikawadori&amp;publication_year=2016&amp;journal=Genome%20Announc&amp;volume=4&amp;pages=e00246-16" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27056230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohhata</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Kawasumi</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Complete genome sequence of a γ-hexachlorocyclohexane degrader, <em>Sphingobium</em> sp. strain TKS, isolated from a γ-hexachlorocyclohexane-degrading microbial community,</div> <div class="source">Genome Announc</div> ., <div class="volume">4</div>, <div class="fpage">e00247–16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20a%20%CE%B3-hexachlorocyclohexane%20degrader%2C%20Sphingobium%20sp.%20strain%20TKS%2C%20isolated%20from%20a%20%CE%B3-hexachlorocyclohexane-degrading%20microbial%20community%2C&amp;author=M.%20Tabata&amp;author=S.%20Ohhata&amp;author=T.%20Kawasumi&amp;publication_year=2016&amp;journal=Genome%20Announc&amp;volume=4&amp;pages=e00247–16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/genomeA.00247-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FgenomeA.00247-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FgenomeA.00247-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27056231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yabuuchi</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Kosako</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Fujiwara</div> <div class="given-names">N.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Emendation of the genus <em>Sphingomonas</em> Yabuuchi et al. 1990 and junior objective synonymy of the species of three genera, <em>Sphingobium</em>, <em>Novosphingobium</em> and <em>Sphingopyxis</em>, in conjunction with <em>Blastomonas ursincola</em>,</div> <div class="source">Int. J. Syst. Evol. Microbiol</div> ., <div class="volume">52</div>, <div class="fpage">1485</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emendation%20of%20the%20genus%20Sphingomonas%20Yabuuchi%20et%20al.%201990%20and%20junior%20objective%20synonymy%20of%20the%20species%20of%20three%20genera%2C%20Sphingobium%2C%20Novosphingobium%20and%20Sphingopyxis%2C%20in%20conjunction%20with%20Blastomonas%20ursincola%2C&amp;author=E.%20Yabuuchi&amp;author=Y.%20Kosako&amp;author=N.%20Fujiwara&amp;publication_year=2002&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol&amp;volume=52&amp;pages=1485-96" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12361250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Fritsch</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div></div> <div class="year">1982</div>, <div class="source">Molecular cloning: a Laboratory Manual</div> . 
<div class="publisher-name">Cold Spring Harbor Laboratory</div>: 
<div class="publisher-loc">NY</div>.</div></div></div></div><div content-id="dsw041-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Identification and characterization of genes encoding a putative ABC-type transporter essential for utilization of γ-hexachlorocyclohexane in <em>Sphingobium japonicum</em> UT26,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">189</div>, <div class="fpage">3712</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20genes%20encoding%20a%20putative%20ABC-type%20transporter%20essential%20for%20utilization%20of%20%CE%B3-hexachlorocyclohexane%20in%20Sphingobium%20japonicum%20UT26%2C&amp;author=R.%20Endo&amp;author=Y.%20Ohtsubo&amp;author=M.%20Tsuda&amp;author=Y.%20Nagata&amp;publication_year=2007&amp;journal=J.%20Bacteriol&amp;volume=189&amp;pages=3712-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01883-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01883-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01883-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17369300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Futamura</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Miyauchi</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Takagi</div> <div class="given-names">M.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Two different types of dehalogenases, LinA and LinB, involved in γ-hexachlorocyclohexane degradation in <em>Sphingomonas paucimobilis</em> UT26 are localized in the periplasmic space without molecular processing</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">181</div>, <div class="fpage">5409</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20different%20types%20of%20dehalogenases%2C%20LinA%20and%20LinB%2C%20involved%20in%20%CE%B3-hexachlorocyclohexane%20degradation%20in%20Sphingomonas%20paucimobilis%20UT26%20are%20localized%20in%20the%20periplasmic%20space%20without%20molecular%20processing&amp;author=Y.%20Nagata&amp;author=A.%20Futamura&amp;author=K.%20Miyauchi&amp;author=M.%20Takagi&amp;publication_year=1999&amp;journal=J.%20Bacteriol&amp;volume=181&amp;pages=5409-13" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10464214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kovach</div> <div class="given-names">M. E.</div></div><div class="name string-name"><div class="surname">Elzer</div> <div class="given-names">P. H.</div></div><div class="name string-name"><div class="surname">Hill</div> <div class="given-names">D. S.</div></div></div>, et al.  . <div class="year">1995</div>, 
<div class="article-title">Four new derivatives of the broad-host-range cloning vector pBBR1MCS, carrying different antibiotic-resistance cassettes,</div> <div class="source">Gene</div> , <div class="volume">166</div>, <div class="fpage">175</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Four%20new%20derivatives%20of%20the%20broad-host-range%20cloning%20vector%20pBBR1MCS%2C%20carrying%20different%20antibiotic-resistance%20cassettes%2C&amp;author=M.%20E.%20Kovach&amp;author=P.%20H.%20Elzer&amp;author=D.%20S.%20Hill&amp;publication_year=1995&amp;journal=Gene&amp;volume=166&amp;pages=175-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(95)00584-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(95)00584-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(95)00584-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8529885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoang</div> <div class="given-names">T. T.</div></div><div class="name string-name"><div class="surname">Karkhoff-Schweizer</div> <div class="given-names">R. R.</div></div><div class="name string-name"><div class="surname">Kutchma</div> <div class="given-names">A. J.</div></div></div>, et al.  . <div class="year">1998</div>, 
<div class="article-title">A broad-host-range Flp-FRT recombination system for site-specific excision of chromosomally-located DNA sequences: application for isolation of unmarked <em>Pseudomonas aeruginosa</em> mutants,</div> <div class="source">Gene</div> , <div class="volume">212</div>, <div class="fpage">77</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20broad-host-range%20Flp-FRT%20recombination%20system%20for%20site-specific%20excision%20of%20chromosomally-located%20DNA%20sequences%3A%20application%20for%20isolation%20of%20unmarked%20Pseudomonas%20aeruginosa%20mutants%2C&amp;author=T.%20T.%20Hoang&amp;author=R.%20R.%20Karkhoff-Schweizer&amp;author=A.%20J.%20Kutchma&amp;publication_year=1998&amp;journal=Gene&amp;volume=212&amp;pages=77-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(98)00130-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(98)00130-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(98)00130-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9661666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Fritsch</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div></div> <div class="year">1989</div>, <div class="source">Molecular Cloning: A Laboratory Manual</div> , <div class="edition">2nd, edn.</div> 
<div class="publisher-name">Cold Spring Harbor</div>: 
<div class="publisher-loc">NY</div>.</div></div></div></div><div content-id="dsw041-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Kamakura</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Miyauchi</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">Identification and characterization of genes involved in the downstream degradation pathway of γ-hexachlorocyclohexane in <em>Sphingomonas paucimobilis</em> UT26,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">187</div>, <div class="fpage">847</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20genes%20involved%20in%20the%20downstream%20degradation%20pathway%20of%20%CE%B3-hexachlorocyclohexane%20in%20Sphingomonas%20paucimobilis%20UT26%2C&amp;author=R.%20Endo&amp;author=M.%20Kamakura&amp;author=K.%20Miyauchi&amp;publication_year=2005&amp;journal=J.%20Bacteriol&amp;volume=187&amp;pages=847-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.3.847-853.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.3.847-853.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.3.847-853.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Maruyama</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Mitsui</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Complete genome sequence of <em>Acidovorax</em> sp. strain KKS102, a polychlorinated-biphenyl degrader,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">194</div>, <div class="fpage">6970</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Acidovorax%20sp.%20strain%20KKS102%2C%20a%20polychlorinated-biphenyl%20degrader%2C&amp;author=Y.%20Ohtsubo&amp;author=F.%20Maruyama&amp;author=H.%20Mitsui&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2012&amp;journal=J.%20Bacteriol&amp;volume=194&amp;pages=6970-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01848-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01848-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01848-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23209225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Kamakura</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Miyazaki</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Distribution of γ-hexachlorocyclohexane-degrading genes on three replicons in <em>Sphingobium japonicum</em> UT26,</div> <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">256</div>, <div class="fpage">112</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20%CE%B3-hexachlorocyclohexane-degrading%20genes%20on%20three%20replicons%20in%20Sphingobium%20japonicum%20UT26%2C&amp;author=Y.%20Nagata&amp;author=M.%20Kamakura&amp;author=R.%20Endo&amp;author=R.%20Miyazaki&amp;author=Y.%20Ohtsubo&amp;author=M.%20Tsuda&amp;publication_year=2006&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=256&amp;pages=112-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2006.256.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2006.256.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2006.256.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16487327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison,</div> <div class="source">BMC bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison%2C&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20bioinformatics&amp;volume=9&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div><div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div><div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool,</div> <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool%2C&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Standley</div> <div class="given-names">D. M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">772</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability%2C&amp;author=K.%20Katoh&amp;author=D.%20M.%20Standley&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=772-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perriere</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">WWW-query: an on-line retrieval system for biological sequence banks,</div> <div class="source">Biochimie</div> , <div class="volume">78</div>, <div class="fpage">364</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WWW-query%3A%20an%20on-line%20retrieval%20system%20for%20biological%20sequence%20banks%2C&amp;author=G.%20Perriere&amp;author=M.%20Gouy&amp;publication_year=1996&amp;journal=Biochimie&amp;volume=78&amp;pages=364-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0300-9084(96)84768-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0300-9084(96)84768-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0300-9084(96)84768-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Mori</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Maruyama</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance,</div> <div class="source">DNA Res</div> ., <div class="volume">22</div>, <div class="fpage">413</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Time-series%20metagenomic%20analysis%20reveals%20robustness%20of%20soil%20microbiome%20against%20chemical%20disturbance%2C&amp;author=H.%20Kato&amp;author=H.%20Mori&amp;author=F.%20Maruyama&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=413-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26428854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Genka</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Komatsu</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">High-temperature-induced transposition of insertion elements in <em>Burkholderia multivorans</em> ATCC 17616,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">71</div>, <div class="fpage">1822</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-temperature-induced%20transposition%20of%20insertion%20elements%20in%20Burkholderia%20multivorans%20ATCC%2017616%2C&amp;author=Y.%20Ohtsubo&amp;author=H.%20Genka&amp;author=H.%20Komatsu&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=71&amp;pages=1822-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.4.1822-1828.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.4.1822-1828.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.4.1822-1828.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15812007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perez-Pantoja</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">De la Iglesia</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Pieper</div> <div class="given-names">D. H.</div></div><div class="name string-name"><div class="surname">Gonzalez</div> <div class="given-names">B.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Metabolic reconstruction of aromatic compounds degradation from the genome of the amazing pollutant-degrading bacterium <em>Cupriavidus necator</em> JMP134</div>, <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">32</div>, <div class="fpage">736</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20reconstruction%20of%20aromatic%20compounds%20degradation%20from%20the%20genome%20of%20the%20amazing%20pollutant-degrading%20bacterium%20Cupriavidus%20necator%20JMP134&amp;author=D.%20Perez-Pantoja&amp;author=R.%20De%20la%20Iglesia&amp;author=D.%20H.%20Pieper&amp;author=B.%20Gonzalez&amp;publication_year=2008&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=32&amp;pages=736-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2008.00122.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2008.00122.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2008.00122.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18691224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lykidis</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Perez-Pantoja</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Ledger</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">The complete multipartite genome sequence of <em>Cupriavidus necator</em> JMP134, a versatile pollutant degrader,</div> <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e9729</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20multipartite%20genome%20sequence%20of%20Cupriavidus%20necator%20JMP134%2C%20a%20versatile%20pollutant%20degrader%2C&amp;author=A.%20Lykidis&amp;author=D.%20Perez-Pantoja&amp;author=T.%20Ledger&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e9729" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009729" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009729" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0009729"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20339589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chain</div> <div class="given-names">P. S.</div></div><div class="name string-name"><div class="surname">Denef</div> <div class="given-names">V. J.</div></div><div class="name string-name"><div class="surname">Konstantinidis</div> <div class="given-names">K. T.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title"><em>Burkholderia xenovorans</em> LB400 harbors a multi-replicon, 9.73-Mbp genome shaped for versatility,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">15280</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Burkholderia%20xenovorans%20LB400%20harbors%20a%20multi-replicon%2C%209.73-Mbp%20genome%20shaped%20for%20versatility%2C&amp;author=P.%20S.%20Chain&amp;author=V.%20J.%20Denef&amp;author=K.%20T.%20Konstantinidis&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=15280-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0606924103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0606924103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0606924103"> </span></div> </div></div></div></div></div><div content-id="dsw041-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Romero-Silva</div> <div class="given-names">M. J.</div></div><div class="name string-name"><div class="surname">Mendez</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Agullo</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Seeger</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genomic and functional analyses of the gentisate and protocatechuate ring-cleavage pathways and related 3-hydroxybenzoate and 4-hydroxybenzoate peripheral pathways in <em>Burkholderia xenovorans</em> LB400,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e56038</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20and%20functional%20analyses%20of%20the%20gentisate%20and%20protocatechuate%20ring-cleavage%20pathways%20and%20related%203-hydroxybenzoate%20and%204-hydroxybenzoate%20peripheral%20pathways%20in%20Burkholderia%20xenovorans%20LB400%2C&amp;author=M.%20J.%20Romero-Silva&amp;author=V.%20Mendez&amp;author=L.%20Agullo&amp;author=M.%20Seeger&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e56038" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0056038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0056038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0056038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23418504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanier</div> <div class="given-names">R. Y.</div></div><div class="name string-name"><div class="surname">Palleroni</div> <div class="given-names">N. J.</div></div><div class="name string-name"><div class="surname">Doudoroff</div> <div class="given-names">M.</div></div></div> <div class="year">1966</div>, 
<div class="article-title">The aerobic pseudomonads: a taxonomic study,</div> <div class="source">J. Gen. Microbiol</div> ., <div class="volume">43</div>, <div class="fpage">159</div>–<div class="lpage">271</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20aerobic%20pseudomonads%3A%20a%20taxonomic%20study%2C&amp;author=R.%20Y.%20Stanier&amp;author=N.%20J.%20Palleroni&amp;author=M.%20Doudoroff&amp;publication_year=1966&amp;journal=J.%20Gen.%20Microbiol&amp;volume=43&amp;pages=159-271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-43-2-159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-43-2-159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-43-2-159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5963505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yuhara</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Komatsu</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Goto</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Pleiotropic roles of iron-responsive transcriptional regulator Fur in <em>Burkholderia multivorans</em>,</div> <div class="source">Microbiology</div> , <div class="volume">154</div>, <div class="fpage">1763</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pleiotropic%20roles%20of%20iron-responsive%20transcriptional%20regulator%20Fur%20in%20Burkholderia%20multivorans%2C&amp;author=S.%20Yuhara&amp;author=H.%20Komatsu&amp;author=H.%20Goto&amp;author=Y.%20Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=Microbiology&amp;volume=154&amp;pages=1763-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2007/015537-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2007%2F015537-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2007%2F015537-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18524931" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Senbongi</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Ishibashi</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Identification of <em>Burkholderia multivorans</em> ATCC 17616 genetic determinants for fitness in soil by using signature-tagged mutagenesis,</div> <div class="source">Microbiology</div> , <div class="volume">160</div>, <div class="fpage">883</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Burkholderia%20multivorans%20ATCC%2017616%20genetic%20determinants%20for%20fitness%20in%20soil%20by%20using%20signature-tagged%20mutagenesis%2C&amp;author=Y.%20Nagata&amp;author=J.%20Senbongi&amp;author=Y.%20Ishibashi&amp;publication_year=2014&amp;journal=Microbiology&amp;volume=160&amp;pages=883-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.077057-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.077057-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.077057-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24530988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perez-Pantoja</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Donoso</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Agullo</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Genomic analysis of the potential for aromatic compounds biodegradation in Burkholderiales,</div> <div class="source">Environ. Microbiol</div> ., <div class="volume">14</div>, <div class="fpage">1091</div>–<div class="lpage">117</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20the%20potential%20for%20aromatic%20compounds%20biodegradation%20in%20Burkholderiales%2C&amp;author=D.%20Perez-Pantoja&amp;author=R.%20Donoso&amp;author=L.%20Agullo&amp;publication_year=2012&amp;journal=Environ.%20Microbiol&amp;volume=14&amp;pages=1091-117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2012.14.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2012.14.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2012.14.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22026719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">K. E.</div></div><div class="name string-name"><div class="surname">Weinel</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Paulsen</div> <div class="given-names">I. T.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Complete genome sequence and comparative analysis of the metabolically versatile <em>Pseudomonas putida</em> KT2440,</div> <div class="source">Environ. Microbiol</div> ., <div class="volume">4</div>, <div class="fpage">799</div>–<div class="lpage">808</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20and%20comparative%20analysis%20of%20the%20metabolically%20versatile%20Pseudomonas%20putida%20KT2440%2C&amp;author=K.%20E.%20Nelson&amp;author=C.%20Weinel&amp;author=I.%20T.%20Paulsen&amp;publication_year=2002&amp;journal=Environ.%20Microbiol&amp;volume=4&amp;pages=799-808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1462-2920.2002.00366.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1462-2920.2002.00366.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1462-2920.2002.00366.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12534463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miyauchi</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">H. S.</div></div><div class="name string-name"><div class="surname">Fukuda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Takagi</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Cloning and characterization of <em>linR</em>, involved in regulation of the downstream pathway for γ-hexachlorocyclohexane degradation in <em>Sphingomonas paucimobilis</em> UT26,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">68</div>, <div class="fpage">1803</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20linR%2C%20involved%20in%20regulation%20of%20the%20downstream%20pathway%20for%20%CE%B3-hexachlorocyclohexane%20degradation%20in%20Sphingomonas%20paucimobilis%20UT26%2C&amp;author=K.%20Miyauchi&amp;author=H.%20S.%20Lee&amp;author=M.%20Fukuda&amp;author=M.%20Takagi&amp;author=Y.%20Nagata&amp;publication_year=2002&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=68&amp;pages=1803-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.68.4.1803-1807.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.68.4.1803-1807.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.68.4.1803-1807.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11916699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harwood</div> <div class="given-names">C. S.</div></div><div class="name string-name"><div class="surname">Parales</div> <div class="given-names">R. E.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">The β-ketoadipate pathway and the biology of self-identity,</div> <div class="source">Annu. Rev. Microbiol</div> ., <div class="volume">50</div>, <div class="fpage">553</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%CE%B2-ketoadipate%20pathway%20and%20the%20biology%20of%20self-identity%2C&amp;author=C.%20S.%20Harwood&amp;author=R.%20E.%20Parales&amp;publication_year=1996&amp;journal=Annu.%20Rev.%20Microbiol&amp;volume=50&amp;pages=553-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.50.1.553" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.50.1.553" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.50.1.553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gross</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Guzman</div> <div class="given-names">C. A.</div></div><div class="name string-name"><div class="surname">Sebaihia</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">The missing link: <em>Bordetella petrii</em> is endowed with both the metabolic versatility of environmental bacteria and virulence traits of pathogenic Bordetellae,</div> <div class="source">BMC genomics</div> , <div class="volume">9</div>, <div class="fpage">449</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20missing%20link%3A%20Bordetella%20petrii%20is%20endowed%20with%20both%20the%20metabolic%20versatility%20of%20environmental%20bacteria%20and%20virulence%20traits%20of%20pathogenic%20Bordetellae%2C&amp;author=R.%20Gross&amp;author=C.%20A.%20Guzman&amp;author=M.%20Sebaihia&amp;publication_year=2008&amp;journal=BMC%20genomics&amp;volume=9&amp;pages=449" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18826580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vedler</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Vahter</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Heinaru</div> <div class="given-names">A.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The completely sequenced plasmid pEST4011 contains a novel IncP1 backbone and a catabolic transposon harboring <em>tfd</em> genes for 2,4-dichlorophenoxyacetic acid degradation,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">186</div>, <div class="fpage">7161</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20completely%20sequenced%20plasmid%20pEST4011%20contains%20a%20novel%20IncP1%20backbone%20and%20a%20catabolic%20transposon%20harboring%20tfd%20genes%20for%202%2C4-dichlorophenoxyacetic%20acid%20degradation%2C&amp;author=E.%20Vedler&amp;author=M.%20Vahter&amp;author=A.%20Heinaru&amp;publication_year=2004&amp;journal=J.%20Bacteriol&amp;volume=186&amp;pages=7161-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.21.7161-7174.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.21.7161-7174.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.21.7161-7174.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krol</div> <div class="given-names">J. E.</div></div><div class="name string-name"><div class="surname">Penrod</div> <div class="given-names">J. T.</div></div><div class="name string-name"><div class="surname">McCaslin</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Role of IncP-1beta plasmids pWDL7::<em>rfp</em> and pNB8c in chloroaniline catabolism as determined by genomic and functional analyses,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">78</div>, <div class="fpage">828</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20IncP-1beta%20plasmids%20pWDL7%3A%3Arfp%20and%20pNB8c%20in%20chloroaniline%20catabolism%20as%20determined%20by%20genomic%20and%20functional%20analyses%2C&amp;author=J.%20E.%20Krol&amp;author=J.%20T.%20Penrod&amp;author=H.%20McCaslin&amp;publication_year=2012&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=78&amp;pages=828-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.07480-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.07480-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.07480-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22101050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mukherjee</div> <div class="given-names">U.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Mahato</div> <div class="given-names">N. K.</div></div><div class="name string-name"><div class="surname">Khurana</div> <div class="given-names">J. P.</div></div><div class="name string-name"><div class="surname">Lal</div> <div class="given-names">R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence of <em>Sphingobium</em> sp. strain HDIPO4, an avid degrader of hexachlorocyclohexane,</div> <div class="source">Genome Announc</div> ., <div class="volume">1</div>, <div class="fpage">e00749</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Sphingobium%20sp.%20strain%20HDIPO4%2C%20an%20avid%20degrader%20of%20hexachlorocyclohexane%2C&amp;author=U.%20Mukherjee&amp;author=R.%20Kumar&amp;author=N.%20K.%20Mahato&amp;author=J.%20P.%20Khurana&amp;author=R.%20Lal&amp;publication_year=2013&amp;journal=Genome%20Announc&amp;volume=1&amp;pages=e00749-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/genomeA.00749-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FgenomeA.00749-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FgenomeA.00749-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24051321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davison</div> <div class="given-names">J.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Genetic exchange between bacteria in the environment,</div> <div class="source">Plasmid</div> , <div class="volume">42</div>, <div class="fpage">73</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20exchange%20between%20bacteria%20in%20the%20environment%2C&amp;author=J.%20Davison&amp;publication_year=1999&amp;journal=Plasmid&amp;volume=42&amp;pages=73-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/plas.1999.1421" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fplas.1999.1421" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fplas.1999.1421"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10489325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shintani</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nojiri</div> <div class="given-names">H.</div></div></div> <div class="year">2013</div>, Mobile genetic elements (MGEs) carrying catabolic genes, In: <div class="person-group"><div class="name string-name"><div class="surname">Malik</div>  <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Grohmann</div> <div class="given-names">E.,</div></div><div class="name string-name"><div class="surname">Alves</div>  <div class="given-names">M.</div></div></div> (eds), <div class="source">Management of Microbial Resources in the Environment</div> , 
<div class="publisher-name">Springer</div>: 
<div class="publisher-loc">the Netherlands</div>, pp. <div class="fpage">167</div>–<div class="lpage">214</div>.</div></div></div></div><div content-id="dsw041-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ishibashi</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Naganawa</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Conjugal transfer of polychlorinated biphenyl/biphenyl degradation genes in <em>Acidovorax</em> sp. strain KKS102, which are located on an integrative and conjugative element,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">194</div>, <div class="fpage">4237</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conjugal%20transfer%20of%20polychlorinated%20biphenyl%2Fbiphenyl%20degradation%20genes%20in%20Acidovorax%20sp.%20strain%20KKS102%2C%20which%20are%20located%20on%20an%20integrative%20and%20conjugative%20element%2C&amp;author=Y.%20Ohtsubo&amp;author=Y.%20Ishibashi&amp;author=H.%20Naganawa&amp;publication_year=2012&amp;journal=J.%20Bacteriol&amp;volume=194&amp;pages=4237-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00352-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00352-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00352-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22685277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shrivastava</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Prokop</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Novel LinA-type 3 δ-hexachlorocyclohexane dehydrochlorinase,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">81</div>, <div class="fpage">7553</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20LinA-type%203%20%CE%B4-hexachlorocyclohexane%20dehydrochlorinase%2C&amp;author=N.%20Shrivastava&amp;author=Z.%20Prokop&amp;author=A.%20Kumar&amp;publication_year=2015&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=81&amp;pages=7553-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01683-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01683-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01683-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26296732" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moriuchi</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Nikawadori</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Stepwise enhancement of catalytic performance of haloalkane dehalogenase LinB towards β-hexachlorocyclohexane,</div> <div class="source">AMB Express</div> , <div class="volume">4</div>, <div class="fpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stepwise%20enhancement%20of%20catalytic%20performance%20of%20haloalkane%20dehalogenase%20LinB%20towards%20%CE%B2-hexachlorocyclohexane%2C&amp;author=R.%20Moriuchi&amp;author=H.%20Tanaka&amp;author=Y.%20Nikawadori&amp;publication_year=2014&amp;journal=AMB%20Express&amp;volume=4&amp;pages=72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13568-014-0072-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13568-014-0072-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13568-014-0072-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25401073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pandey</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Lucent</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Kinetic and sequence-structure-function analysis of LinB enzyme variants with β- and δ-hexachlorocyclohexane,</div> <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e103632</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kinetic%20and%20sequence-structure-function%20analysis%20of%20LinB%20enzyme%20variants%20with%20%CE%B2-%20and%20%CE%B4-hexachlorocyclohexane%2C&amp;author=R.%20Pandey&amp;author=D.%20Lucent&amp;author=K.%20Kumari&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e103632" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0103632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0103632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0103632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25076214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Properties and biotechnological applications of natural and engineered haloalkane dehalogenases,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">99</div>, <div class="fpage">9865</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20and%20biotechnological%20applications%20of%20natural%20and%20engineered%20haloalkane%20dehalogenases%2C&amp;author=Y.%20Nagata&amp;author=Y.%20Ohtsubo&amp;author=M.%20Tsuda&amp;publication_year=2015&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=99&amp;pages=9865-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-015-6954-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-015-6954-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-015-6954-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26373728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brassinga</div> <div class="given-names">A. K.</div></div><div class="name string-name"><div class="surname">Marczynski</div> <div class="given-names">G. T.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Replication intermediate analysis confirms that chromosomal replication origin initiates from an unusual intergenic region in <em>Caulobacter crescentus</em>,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">29</div>, <div class="fpage">4441</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication%20intermediate%20analysis%20confirms%20that%20chromosomal%20replication%20origin%20initiates%20from%20an%20unusual%20intergenic%20region%20in%20Caulobacter%20crescentus%2C&amp;author=A.%20K.%20Brassinga&amp;author=G.%20T.%20Marczynski&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=4441-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.21.4441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.21.4441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.21.4441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11691932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sibley</div> <div class="given-names">C. D.</div></div><div class="name string-name"><div class="surname">MacLellan</div> <div class="given-names">S. R.</div></div><div class="name string-name"><div class="surname">Finan</div> <div class="given-names">T.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The <em>Sinorhizobium meliloti</em> chromosomal origin of replication,</div> <div class="source">Microbiology</div> , <div class="volume">152</div>, <div class="fpage">443</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sinorhizobium%20meliloti%20chromosomal%20origin%20of%20replication%2C&amp;author=C.%20D.%20Sibley&amp;author=S.%20R.%20MacLellan&amp;author=T.%20Finan&amp;publication_year=2006&amp;journal=Microbiology&amp;volume=152&amp;pages=443-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.28455-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.28455-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.28455-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16436432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schaper</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Messer</div> <div class="given-names">W.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Interaction of the initiator protein DnaA of <em>Escherichia coli</em> with its DNA target,</div> <div class="source">J. Biol. Chem</div> ., <div class="volume">270</div>, <div class="fpage">17622</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interaction%20of%20the%20initiator%20protein%20DnaA%20of%20Escherichia%20coli%20with%20its%20DNA%20target%2C&amp;author=S.%20Schaper&amp;author=W.%20Messer&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem&amp;volume=270&amp;pages=17622-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.29.17622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.29.17622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.29.17622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7615570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">U. M.</div></div><div class="name string-name"><div class="surname">Pappas</div> <div class="given-names">K. M.</div></div><div class="name string-name"><div class="surname">Winans</div> <div class="given-names">S. C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The ABCs of plasmid replication and segregation,</div> <div class="source">Nat. Rev. Microbiol</div> ., <div class="volume">10</div>, <div class="fpage">755</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ABCs%20of%20plasmid%20replication%20and%20segregation%2C&amp;author=U.%20M.%20Pinto&amp;author=K.%20M.%20Pappas&amp;author=S.%20C.%20Winans&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Microbiol&amp;volume=10&amp;pages=755-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2882" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2882" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2882"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23070556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cevallos</div> <div class="given-names">M. A.</div></div><div class="name string-name"><div class="surname">Cervantes-Rivera</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Gutierrez-Rios</div> <div class="given-names">R. M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The <em>repABC</em> plasmid family,</div> <div class="source">Plasmid</div> , <div class="volume">60</div>, <div class="fpage">19</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20repABC%20plasmid%20family%2C&amp;author=M.%20A.%20Cevallos&amp;author=R.%20Cervantes-Rivera&amp;author=R.%20M.%20Gutierrez-Rios&amp;publication_year=2008&amp;journal=Plasmid&amp;volume=60&amp;pages=19-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plasmid.2008.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plasmid.2008.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plasmid.2008.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18433868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">del Solar</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Giraldo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Ruiz-Echevarria</div> <div class="given-names">M. J.</div></div><div class="name string-name"><div class="surname">Espinosa</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Diaz-Orejas</div> <div class="given-names">R.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Replication and control of circular bacterial plasmids,</div> <div class="source">Microbiol. Mol. Biol. Rev</div> ., <div class="volume">62</div>, <div class="fpage">434</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication%20and%20control%20of%20circular%20bacterial%20plasmids%2C&amp;author=G.%20del%20Solar&amp;author=R.%20Giraldo&amp;author=M.%20J.%20Ruiz-Echevarria&amp;author=M.%20Espinosa&amp;author=R.%20Diaz-Orejas&amp;publication_year=1998&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=62&amp;pages=434-64" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9618448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chattoraj</div> <div class="given-names">D. K.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Control of plasmid DNA replication by iterons: no longer paradoxical,</div> <div class="source">Mol. Microbiol</div> ., <div class="volume">37</div>, <div class="fpage">467</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20plasmid%20DNA%20replication%20by%20iterons%3A%20no%20longer%20paradoxical%2C&amp;author=D.%20K.%20Chattoraj&amp;publication_year=2000&amp;journal=Mol.%20Microbiol&amp;volume=37&amp;pages=467-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01986.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.01986.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.01986.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10931340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Osborn</div> <div class="given-names">A. M.</div></div><div class="name string-name"><div class="surname">da Silva Tatley</div> <div class="given-names">F. M.</div></div><div class="name string-name"><div class="surname">Steyn</div> <div class="given-names">L. M.</div></div><div class="name string-name"><div class="surname">Pickup</div> <div class="given-names">R. W.</div></div><div class="name string-name"><div class="surname">Saunders</div> <div class="given-names">J. R.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Mosaic plasmids and mosaic replicons: evolutionary lessons from the analysis of genetic diversity in IncFII-related replicons,</div> <div class="source">Microbiology</div> , <div class="volume">146</div>, <div class="fpage">2267</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mosaic%20plasmids%20and%20mosaic%20replicons%3A%20evolutionary%20lessons%20from%20the%20analysis%20of%20genetic%20diversity%20in%20IncFII-related%20replicons%2C&amp;author=A.%20M.%20Osborn&amp;author=F.%20M.%20da%20Silva%20Tatley&amp;author=L.%20M.%20Steyn&amp;author=R.%20W.%20Pickup&amp;author=J.%20R.%20Saunders&amp;publication_year=2000&amp;journal=Microbiology&amp;volume=146&amp;pages=2267-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-146-9-2267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-146-9-2267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-146-9-2267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10974114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cabezon</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Ripoll-Rozada</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Pena</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">de la Cruz</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Arechaga</div> <div class="given-names">I.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Towards an integrated model of bacterial conjugation,</div> <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">39</div>, <div class="fpage">81</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20an%20integrated%20model%20of%20bacterial%20conjugation%2C&amp;author=E.%20Cabezon&amp;author=J.%20Ripoll-Rozada&amp;author=A.%20Pena&amp;author=F.%20de%20la%20Cruz&amp;author=I.%20Arechaga&amp;publication_year=2015&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=39&amp;pages=81-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25154632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miyazaki</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ito</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Complete nucleotide sequence of an exogenously isolated plasmid, pLB1, involved in γ-hexachlorocyclohexane degradation,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">72</div>, <div class="fpage">6923</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20nucleotide%20sequence%20of%20an%20exogenously%20isolated%20plasmid%2C%20pLB1%2C%20involved%20in%20%CE%B3-hexachlorocyclohexane%20degradation%2C&amp;author=R.%20Miyazaki&amp;author=Y.%20Sato&amp;author=M.%20Ito&amp;author=Y.%20Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=72&amp;pages=6923-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01531-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01531-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01531-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16963556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Frost</div> <div class="given-names">L. S.</div></div><div class="name string-name"><div class="surname">Ippen-Ihler</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Skurray</div> <div class="given-names">R. A.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Analysis of the sequence and gene products of the transfer region of the F sex factor,</div> <div class="source">Microbiol. Rev</div> ., <div class="volume">58</div>, <div class="fpage">162</div>–<div class="lpage">210</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20sequence%20and%20gene%20products%20of%20the%20transfer%20region%20of%20the%20F%20sex%20factor%2C&amp;author=L.%20S.%20Frost&amp;author=K.%20Ippen-Ihler&amp;author=R.%20A.%20Skurray&amp;publication_year=1994&amp;journal=Microbiol.%20Rev&amp;volume=58&amp;pages=162-210" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7915817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boltner</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Moreno-Morillas</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ramos</div> <div class="given-names">J. L.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">16S rDNA phylogeny and distribution of <em>lin</em> genes in novel hexachlorocyclohexane-degrading <em>Sphingomonas</em> strains,</div> <div class="source">Environ. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">1329</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=16S%20rDNA%20phylogeny%20and%20distribution%20of%20lin%20genes%20in%20novel%20hexachlorocyclohexane-degrading%20Sphingomonas%20strains%2C&amp;author=D.%20Boltner&amp;author=S.%20Moreno-Morillas&amp;author=J.%20L.%20Ramos&amp;publication_year=2005&amp;journal=Environ.%20Microbiol&amp;volume=7&amp;pages=1329-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2005.7.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2005.7.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2005.7.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16104856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fuchu</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ito</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Insertion sequence-based cassette PCR: cultivation-independent isolation of γ-hexachlorocyclohexane-degrading genes from soil DNA,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">79</div>, <div class="fpage">627</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion%20sequence-based%20cassette%20PCR%3A%20cultivation-independent%20isolation%20of%20%CE%B3-hexachlorocyclohexane-degrading%20genes%20from%20soil%20DNA%2C&amp;author=G.%20Fuchu&amp;author=Y.%20Ohtsubo&amp;author=M.%20Ito&amp;publication_year=2008&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=79&amp;pages=627-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-008-1463-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-008-1463-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-008-1463-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18425509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mahillon</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Insertion sequences</div>, <div class="source">Microbiol. Mol. Biol. Rev</div> ., <div class="volume">62</div>, <div class="fpage">725</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion%20sequences&amp;author=J.%20Mahillon&amp;author=M.%20Chandler&amp;publication_year=1998&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=62&amp;pages=725-74" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9729608" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Livny</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Yamaichi</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Waldor</div> <div class="given-names">M. K.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Distribution of centromere-like <em>parS</em> sites in bacteria: insights from comparative genomics,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">189</div>, <div class="fpage">8693</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20centromere-like%20parS%20sites%20in%20bacteria%3A%20insights%20from%20comparative%20genomics%2C&amp;author=J.%20Livny&amp;author=Y.%20Yamaichi&amp;author=M.%20K.%20Waldor&amp;publication_year=2007&amp;journal=J.%20Bacteriol&amp;volume=189&amp;pages=8693-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01239-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01239-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01239-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17905987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">D’Argenio</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Petrillo</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Cantiello</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">De novo sequencing and assembly of the whole genome of <em>Novosphingobium</em> sp. strain PP1Y</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">193</div>, <div class="fpage">4296</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20sequencing%20and%20assembly%20of%20the%20whole%20genome%20of%20Novosphingobium%20sp.%20strain%20PP1Y&amp;author=V.%20D%E2%80%99Argenio&amp;author=M.%20Petrillo&amp;author=P.%20Cantiello&amp;publication_year=2011&amp;journal=J.%20Bacteriol&amp;volume=193&amp;pages=4296" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.05349-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.05349-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.05349-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21685292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Masai</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Kamimura</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Kasai</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Complete genome sequence of <em>Sphingobium</em> sp. strain SYK-6, a degrader of lignin-derived biaryls and monoaryls,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">194</div>, <div class="fpage">534</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Sphingobium%20sp.%20strain%20SYK-6%2C%20a%20degrader%20of%20lignin-derived%20biaryls%20and%20monoaryls%2C&amp;author=E.%20Masai&amp;author=N.%20Kamimura&amp;author=D.%20Kasai&amp;publication_year=2012&amp;journal=J.%20Bacteriol&amp;volume=194&amp;pages=534-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.06254-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.06254-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.06254-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22207743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shintani</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Urata</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Inoue</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">The <em>Sphingomonas</em> plasmid pCAR3 is involved in complete mineralization of carbazole,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">189</div>, <div class="fpage">2007</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sphingomonas%20plasmid%20pCAR3%20is%20involved%20in%20complete%20mineralization%20of%20carbazole%2C&amp;author=M.%20Shintani&amp;author=M.%20Urata&amp;author=K.%20Inoue&amp;publication_year=2007&amp;journal=J.%20Bacteriol&amp;volume=189&amp;pages=2007-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01486-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01486-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01486-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17172338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">T. R.</div></div><div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div><div class="name string-name"><div class="surname">Saunders</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Detter</div> <div class="given-names">J. C.</div></div><div class="name string-name"><div class="surname">Halden</div> <div class="given-names">R. U.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Genome sequence of the dioxin-mineralizing bacterium <em>Sphingomonas wittichii</em> RW1,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">192</div>, <div class="fpage">6101</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20dioxin-mineralizing%20bacterium%20Sphingomonas%20wittichii%20RW1%2C&amp;author=T.%20R.%20Miller&amp;author=A.%20L.%20Delcher&amp;author=S.%20L.%20Salzberg&amp;author=E.%20Saunders&amp;author=J.%20C.%20Detter&amp;author=R.%20U.%20Halden&amp;publication_year=2010&amp;journal=J.%20Bacteriol&amp;volume=192&amp;pages=6101-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01030-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01030-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01030-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20833805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw041-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw041-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">Y. R.</div></div><div class="name string-name"><div class="surname">Kang</div> <div class="given-names">S. G.</div></div><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S. J.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Genome sequence of benzo(a)pyrene-degrading bacterium <em>Novosphingobium pentaromativorans</em> US6-1,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">194</div>, <div class="fpage">907</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20benzo(a)pyrene-degrading%20bacterium%20Novosphingobium%20pentaromativorans%20US6-1%2C&amp;author=Y.%20R.%20Luo&amp;author=S.%20G.%20Kang&amp;author=S.%20J.%20Kim&amp;publication_year=2012&amp;journal=J.%20Bacteriol&amp;volume=194&amp;pages=907" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.06476-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.06476-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.06476-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22275104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2647451&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw041/2/dsw041_Supp.zip?Expires=2147483647&amp;Signature=OLBVK9flEP5iAJ3ZKPIZEp1fW-khRf4bNYXMiRSD5~LbF8gRlkYxwc~5XCtwoZFgKMYRoJEgvJV2meUQVXpnVugUpivam-6jIysAglPt-CQIIck69-B-bwhoPai-AdEq92WZXqGQnCoBkNFJ~4tIWIcwnnMhMJmU9owYz1NPxA~n9Q~r8cjBB7oPepagbERPceiQEtllmcEiOBFEaheqcPF9ZrKWdZOn4Sp3V-kPrOv73inkq3pKZ6RIPMF-hSWPRbSoagIwCX3ySdXfYS8~VCcHORGf6fWsxoL1UsdYco97ksU7PcK-3S0DkVmNZQ30qCOKKGC1L9VaQJ6YPZ7dSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">625</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw041"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2647451"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f6%2f581%2f2647451">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000392723300008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F6%2F581%2F2647451" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2647451'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparison of the complete genome sequences of four γ-hexachlorocyclohexane-degrading bacterial strains: insights into the evolution of bacteria able to degrade a recalcitrant man-made pesticide" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw041");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f581%2f2647451">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

