

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/5/319/347881","@type":"ScholarlyArticle","name":"Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing","datePublished":"2015-09-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2015-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv015","keywords":["RNA-sequencing","transcript assembly","<italic>Oryza sativa</italic>","cufflinks","genome annotation","novel genes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/5/319/347881","author":[{"name":"Watanabe, Kenneth A.","affiliation":"1  School of Life Sciences ,  University of Nevada Las Vegas ,  4505 South Maryland Parkway, Las Vegas, NV 89154 ,  USA","@type":"Person"},{"name":"Homayouni, Arielle","affiliation":"1  School of Life Sciences ,  University of Nevada Las Vegas ,  4505 South Maryland Parkway, Las Vegas, NV 89154 ,  USA","@type":"Person"},{"name":"Tufano, Tara","affiliation":"1  School of Life Sciences ,  University of Nevada Las Vegas ,  4505 South Maryland Parkway, Las Vegas, NV 89154 ,  USA","@type":"Person"},{"name":"Lopez, Jennifer","affiliation":"1  School of Life Sciences ,  University of Nevada Las Vegas ,  4505 South Maryland Parkway, Las Vegas, NV 89154 ,  USA","@type":"Person"},{"name":"Ringler, Patricia","affiliation":"1  School of Life Sciences ,  University of Nevada Las Vegas ,  4505 South Maryland Parkway, Las Vegas, NV 89154 ,  USA","@type":"Person"},{"name":"Rushton, Paul","affiliation":"2  Texas A&amp;M AgriLife Research ,  17360 Coit Road, Dallas, TX 75252 ,  USA","@type":"Person"},{"name":"Shen, Qingxi J.","affiliation":"1  School of Life Sciences ,  University of Nevada Las Vegas ,  4505 South Maryland Parkway, Las Vegas, NV 89154 ,  USA","@type":"Person"}],"description":"Abstract.  Annotation of the rice (Oryza sativa) genome has evolved significantly since release of its draft sequence, but it is far from complete. Several publ","pageStart":"319","pageEnd":"329","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing" />
    <meta property="og:description" content="Abstract.  Annotation of the rice (Oryza sativa) genome has evolved significantly since release of its draft sequence, but it is far from complete. Several publ" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/5/319/347881" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-09-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Watanabe, Kenneth A." /><meta name="citation_author_institution" content="School of Life Sciences, University of Nevada Las Vegas, 4505 South Maryland Parkway, Las Vegas, NV 89154, USA" /><meta name="citation_author" content="Homayouni, Arielle" /><meta name="citation_author_institution" content="School of Life Sciences, University of Nevada Las Vegas, 4505 South Maryland Parkway, Las Vegas, NV 89154, USA" /><meta name="citation_author" content="Tufano, Tara" /><meta name="citation_author_institution" content="School of Life Sciences, University of Nevada Las Vegas, 4505 South Maryland Parkway, Las Vegas, NV 89154, USA" /><meta name="citation_author" content="Lopez, Jennifer" /><meta name="citation_author_institution" content="School of Life Sciences, University of Nevada Las Vegas, 4505 South Maryland Parkway, Las Vegas, NV 89154, USA" /><meta name="citation_author" content="Ringler, Patricia" /><meta name="citation_author_institution" content="School of Life Sciences, University of Nevada Las Vegas, 4505 South Maryland Parkway, Las Vegas, NV 89154, USA" /><meta name="citation_author" content="Rushton, Paul" /><meta name="citation_author_institution" content="Texas A&amp;M AgriLife Research, 17360 Coit Road, Dallas, TX 75252, USA" /><meta name="citation_author" content="Shen, Qingxi J." /><meta name="citation_author_institution" content="School of Life Sciences, University of Nevada Las Vegas, 4505 South Maryland Parkway, Las Vegas, NV 89154, USA" /><meta name="citation_title" content="Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing" /><meta name="citation_firstpage" content="319" /><meta name="citation_lastpage" content="329" /><meta name="citation_doi" content="10.1093/dnares/dsv015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2015/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=RNA-Seq: a revolutionary tool for transcriptomics;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=57-63; " /><meta name="citation_reference" content="citation_title=RNA-Seq: a method for comprehensive transcriptome analysis;  citation_journal_title=Curr. Protoc. Mol. Biol.;  citation_year=2010;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=From RNA-seq reads to differential expression results;  citation_journal_title=Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=220" /><meta name="citation_reference" content="citation_title=Global identification of human transcribed sequences with genome tiling arrays;  citation_journal_title=Science;  citation_year=2004;  citation_volume=306;  citation_pages=2242-6; " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=Assessment of transcript reconstruction methods for RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1177-84; " /><meta name="citation_reference" content="citation_title=A beginner's guide to eukaryotic genome annotation;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2012;  citation_volume=13;  citation_pages=329-42; " /><meta name="citation_reference" content="citation_title=Comparative study of de novo assembly and genome-guided assembly strategies for transcriptome reconstruction based on RNA-Seq;  citation_journal_title=Sci. China Life Sci.;  citation_year=2013;  citation_volume=56;  citation_pages=143-55; " /><meta name="citation_reference" content="citation_title=RNA-sequencing reveals previously unannotated protein- and microRNA-coding genes expressed in aleurone cells of rice seeds;  citation_journal_title=Genomics;  citation_year=2014;  citation_volume=103;  citation_pages=122-34; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=The sequence read archive;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D19-21; " /><meta name="citation_reference" content="citation_title=Improvement of the Oryza sativa Nipponbare reference genome using next generation sequence and optical map data;  citation_journal_title=Rice;  citation_year=2013;  citation_volume=6;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Phytozome: a comparative platform for green plant genomics;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=40;  citation_pages=D1178-86; " /><meta name="citation_reference" content="citation_title=WormBase 2014: new views of curated biology;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=42;  citation_pages=D789-93; " /><meta name="citation_reference" content="citation_title=The reference genome sequence of Saccharomyces cerevisiae: then and now;  citation_journal_title=G3;  citation_year=2014;  citation_volume=4;  citation_pages=389-98; " /><meta name="citation_reference" content="citation_title=FlyBase 102—advanced approaches to interrogating FlyBase;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D780-8; " /><meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice;  citation_journal_title=Science;  citation_year=2003;  citation_volume=301;  citation_pages=376-9; " /><meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison;  citation_journal_title=BMC Bioinform.;  citation_year=2005;  citation_volume=6;  citation_pages=31" /><meta name="citation_reference" content="citation_title=The human genome browser at UCSC;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=996-1006; " /><meta name="citation_reference" content="citation_title=OLego: fast and sensitive mapping of spliced mRNA-Seq reads using small seeds;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=5149-63; " /><meta name="citation_reference" content="citation_title=Prevalence of Trypanosoma cruzi infection among people aged 15 to 89 years inhabiting the Department of Casanare (Colombia);  citation_journal_title=PLoS Negl. Trop. Dis.;  citation_year=2013;  citation_volume=7;  citation_pages=e2113" /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=RNA-Seq: revelation of the messengers;  citation_journal_title=Trends Plant Sci.;  citation_year=2013;  citation_volume=18;  citation_pages=175-9; " /><meta name="citation_reference" content="citation_title=Transcriptionally active gene fragments derived from potentially fast-evolving donor genes in the rice genome;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1215-8; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Understanding a genome sequence; citation_publisher=Wiley-Liss, Oxford;  citation_title=Genomes;  citation_year=2002; " /><meta name="citation_reference" content="citation_title=Discovery and annotation of small proteins using genomics, proteomics, and computational approaches;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=634-41; " /><meta name="citation_reference" content="citation_title=lincRNAs: genomics, evolution, and mechanisms;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=154;  citation_pages=26-46; " /><meta name="citation_reference" content="citation_title=Alternative splicing in plants—coming of age;  citation_journal_title=Trends Plant Sci.;  citation_year=2012;  citation_volume=17;  citation_pages=616-23; " /><meta name="citation_reference" content="citation_title=Alternative splicing and evolution: diversification, exon definition and function;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=345-55; " /><meta name="citation_reference" content="citation_title=Transcription termination by nuclear RNA polymerases;  citation_journal_title=Genes Develop.;  citation_year=2009;  citation_volume=23;  citation_pages=1247-69; " /><meta name="citation_reference" content="citation_title=RNA-Seq of Arabidopsis pollen uncovers novel transcription and alternative splicing;  citation_journal_title=Plant Physiol.;  citation_year=2013;  citation_volume=162;  citation_pages=1092-109; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/5/319/996739/dsv015.pdf" /><meta name="description" content="Abstract.  Annotation of the rice (Oryza sativa) genome has evolved significantly since release of its draft sequence, but it is far from complete. Several publ" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/5/319/347881" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/5/319/347881" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/5/319/347881");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv015';
    </script>
<title>
	Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347881" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbxRPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgS2VubmV0aCBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE5ldmFkYSBMYXMgVmVnYXMsIDQ1MDUgU291dGggTWFyeWxhbmQgUGFya3dheSwgTGFzIFZlZ2FzLCBOViA4OTE1NCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG9tYXlvdW5pLCBBcmllbGxlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTmV2YWRhIExhcyBWZWdhcywgNDUwNSBTb3V0aCBNYXJ5bGFuZCBQYXJrd2F5LCBMYXMgVmVnYXMsIE5WIDg5MTU0LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUdWZhbm8sIFRhcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBOZXZhZGEgTGFzIFZlZ2FzLCA0NTA1IFNvdXRoIE1hcnlsYW5kIFBhcmt3YXksIExhcyBWZWdhcywgTlYgODkxNTQsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxvcGV6LCBKZW5uaWZlciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE5ldmFkYSBMYXMgVmVnYXMsIDQ1MDUgU291dGggTWFyeWxhbmQgUGFya3dheSwgTGFzIFZlZ2FzLCBOViA4OTE1NCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmluZ2xlciwgUGF0cmljaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBOZXZhZGEgTGFzIFZlZ2FzLCA0NTA1IFNvdXRoIE1hcnlsYW5kIFBhcmt3YXksIExhcyBWZWdhcywgTlYgODkxNTQsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJ1c2h0b24sIFBhdWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUZXhhcyBBJmFtcDtNIEFncmlMaWZlIFJlc2VhcmNoLCAxNzM2MCBDb2l0IFJvYWQsIERhbGxhcywgVFggNzUyNTIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoZW4sIFFpbmd4aSBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE5ldmFkYSBMYXMgVmVnYXMsIDQ1MDUgU291dGggTWFyeWxhbmQgUGFya3dheSwgTGFzIFZlZ2FzLCBOViA4OTE1NCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUaWxpbmcgQXNzZW1ibHk6IGEgbmV3IHRvb2wgZm9yIHJlZmVyZW5jZSBhbm5vdGF0aW9uLWluZGVwZW5kZW50IHRyYW5zY3JpcHQgYXNzZW1ibHkgYW5kIG5vdmVsIGdlbmUgaWRlbnRpZmljYXRpb24gYnkgUk5BLXNlcXVlbmNpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjMyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdjAxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLVNlcTogYSByZXZvbHV0aW9uYXJ5IHRvb2wgZm9yIHRyYW5zY3JpcHRvbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU3LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtU2VxOiBhIG1ldGhvZCBmb3IgY29tcHJlaGVuc2l2ZSB0cmFuc2NyaXB0b21lIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBQcm90b2MuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gcm9tIFJOQS1zZXEgcmVhZHMgdG8gZGlmZmVyZW50aWFsIGV4cHJlc3Npb24gcmVzdWx0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGlkZW50aWZpY2F0aW9uIG9mIGh1bWFuIHRyYW5zY3JpYmVkIHNlcXVlbmNlcyB3aXRoIGdlbm9tZSB0aWxpbmcgYXJyYXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMwNjsgIGNpdGF0aW9uX3BhZ2VzPTIyNDItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVsbC1sZW5ndGggdHJhbnNjcmlwdG9tZSBhc3NlbWJseSBmcm9tIFJOQS1TZXEgZGF0YSB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTY0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiB0cmFuc2NyaXB0IHJlY29uc3RydWN0aW9uIG1ldGhvZHMgZm9yIFJOQS1zZXE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTExNzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgYmVnaW5uZXIncyBndWlkZSB0byBldWthcnlvdGljIGdlbm9tZSBhbm5vdGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MzI5LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBzdHVkeSBvZiBkZSBub3ZvIGFzc2VtYmx5IGFuZCBnZW5vbWUtZ3VpZGVkIGFzc2VtYmx5IHN0cmF0ZWdpZXMgZm9yIHRyYW5zY3JpcHRvbWUgcmVjb25zdHJ1Y3Rpb24gYmFzZWQgb24gUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBDaGluYSBMaWZlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0xNDMtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQS1zZXF1ZW5jaW5nIHJldmVhbHMgcHJldmlvdXNseSB1bmFubm90YXRlZCBwcm90ZWluLSBhbmQgbWljcm9STkEtY29kaW5nIGdlbmVzIGV4cHJlc3NlZCBpbiBhbGV1cm9uZSBjZWxscyBvZiByaWNlIHNlZWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0xMjItMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhZmFzdCBhbmQgbWVtb3J5LWVmZmljaWVudCBhbGlnbm1lbnQgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB0byB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3BIYXQ6IGRpc2NvdmVyaW5nIHNwbGljZSBqdW5jdGlvbnMgd2l0aCBSTkEtU2VxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTExMDUtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZXF1ZW5jZSByZWFkIGFyY2hpdmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPUQxOS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZW1lbnQgb2YgdGhlIE9yeXphIHNhdGl2YSBOaXBwb25iYXJlIHJlZmVyZW5jZSBnZW5vbWUgdXNpbmcgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNlIGFuZCBvcHRpY2FsIG1hcCBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SaWNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz00IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5dG96b21lOiBhIGNvbXBhcmF0aXZlIHBsYXRmb3JtIGZvciBncmVlbiBwbGFudCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDExNzgtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdvcm1CYXNlIDIwMTQ6IG5ldyB2aWV3cyBvZiBjdXJhdGVkIGJpb2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQ3ODktOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWZlcmVuY2UgZ2Vub21lIHNlcXVlbmNlIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTogdGhlbiBhbmQgbm93OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMzsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9Mzg5LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbHlCYXNlIDEwMuKAlGFkdmFuY2VkIGFwcHJvYWNoZXMgdG8gaW50ZXJyb2dhdGluZyBGbHlCYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1ENzgwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGxlY3Rpb24sIG1hcHBpbmcsIGFuZCBhbm5vdGF0aW9uIG9mIG92ZXIgMjgsMDAwIGNETkEgY2xvbmVzIGZyb20gamFwb25pY2EgcmljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDE7ICBjaXRhdGlvbl9wYWdlcz0zNzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGdlbmVyYXRpb24gb2YgaGV1cmlzdGljcyBmb3IgYmlvbG9naWNhbCBzZXF1ZW5jZSBjb21wYXJpc29uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaHVtYW4gZ2Vub21lIGJyb3dzZXIgYXQgVUNTQzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz05OTYtMTAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T0xlZ286IGZhc3QgYW5kIHNlbnNpdGl2ZSBtYXBwaW5nIG9mIHNwbGljZWQgbVJOQS1TZXEgcmVhZHMgdXNpbmcgc21hbGwgc2VlZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTUxNDktNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZXZhbGVuY2Ugb2YgVHJ5cGFub3NvbWEgY3J1emkgaW5mZWN0aW9uIGFtb25nIHBlb3BsZSBhZ2VkIDE1IHRvIDg5IHllYXJzIGluaGFiaXRpbmcgdGhlIERlcGFydG1lbnQgb2YgQ2FzYW5hcmUgKENvbG9tYmlhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBOZWdsLiBUcm9wLiBEaXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMjExMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQgYXNzZW1ibHkgYW5kIHF1YW50aWZpY2F0aW9uIGJ5IFJOQS1TZXEgcmV2ZWFscyB1bmFubm90YXRlZCB0cmFuc2NyaXB0cyBhbmQgaXNvZm9ybSBzd2l0Y2hpbmcgZHVyaW5nIGNlbGwgZGlmZmVyZW50aWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NTExLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQS1TZXE6IHJldmVsYXRpb24gb2YgdGhlIG1lc3NlbmdlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTc1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbGx5IGFjdGl2ZSBnZW5lIGZyYWdtZW50cyBkZXJpdmVkIGZyb20gcG90ZW50aWFsbHkgZmFzdC1ldm9sdmluZyBkb25vciBnZW5lcyBpbiB0aGUgcmljZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTIxNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuZGVyc3RhbmRpbmcgYSBnZW5vbWUgc2VxdWVuY2U7IGNpdGF0aW9uX3B1Ymxpc2hlcj1XaWxleS1MaXNzLCBPeGZvcmQ7ICBjaXRhdGlvbl90aXRsZT1HZW5vbWVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3ZlcnkgYW5kIGFubm90YXRpb24gb2Ygc21hbGwgcHJvdGVpbnMgdXNpbmcgZ2Vub21pY3MsIHByb3Rlb21pY3MsIGFuZCBjb21wdXRhdGlvbmFsIGFwcHJvYWNoZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjM0LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1saW5jUk5BczogZ2Vub21pY3MsIGV2b2x1dGlvbiwgYW5kIG1lY2hhbmlzbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MjYtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGluIHBsYW50c+KAlGNvbWluZyBvZiBhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NjE2LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBhbmQgZXZvbHV0aW9uOiBkaXZlcnNpZmljYXRpb24sIGV4b24gZGVmaW5pdGlvbiBhbmQgZnVuY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zNDUtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gdGVybWluYXRpb24gYnkgbnVjbGVhciBSTkEgcG9seW1lcmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldmVsb3AuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTI0Ny02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLVNlcSBvZiBBcmFiaWRvcHNpcyBwb2xsZW4gdW5jb3ZlcnMgbm92ZWwgdHJhbnNjcmlwdGlvbiBhbmQgYWx0ZXJuYXRpdmUgc3BsaWNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTEwOTItMTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvNS8zMTkvOTk2NzM5L2RzdjAxNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQW5ub3RhdGlvbiBvZiB0aGUgcmljZSAoT3J5emEgc2F0aXZhKSBnZW5vbWUgaGFzIGV2b2x2ZWQgc2lnbmlmaWNhbnRseSBzaW5jZSByZWxlYXNlIG9mIGl0cyBkcmFmdCBzZXF1ZW5jZSwgYnV0IGl0IGlzIGZhciBmcm9tIGNvbXBsZXRlLiBTZXZlcmFsIHB1YmwiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0Nzg4MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGT4LXTrTf3mB9+AVvKaOf9CAzjGAg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f319%2f347881" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f319%2f347881" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&amp;Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/5">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4934499">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934499">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934501">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934501">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934506">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934506">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934528">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934528">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934665">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934665">4. Availability of software</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934676">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934676">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934678">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934678">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934680">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934680">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934682">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934682">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4934684">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4934684">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/5/307/347806">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/5/331/347913">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenneth A. Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">University of Nevada Las Vegas</div>, <div class="addr-line">4505 South Maryland Parkway, Las Vegas, NV 89154</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenneth+A.+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenneth A. Watanabe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arielle Homayouni</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">University of Nevada Las Vegas</div>, <div class="addr-line">4505 South Maryland Parkway, Las Vegas, NV 89154</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arielle+Homayouni">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Homayouni A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Homayouni A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Arielle Homayouni</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tara Tufano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">University of Nevada Las Vegas</div>, <div class="addr-line">4505 South Maryland Parkway, Las Vegas, NV 89154</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tara+Tufano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tufano T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tufano T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tara Tufano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jennifer Lopez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">University of Nevada Las Vegas</div>, <div class="addr-line">4505 South Maryland Parkway, Las Vegas, NV 89154</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jennifer+Lopez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lopez J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lopez J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jennifer Lopez</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Patricia Ringler</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">University of Nevada Las Vegas</div>, <div class="addr-line">4505 South Maryland Parkway, Las Vegas, NV 89154</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Patricia+Ringler">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ringler P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ringler P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Patricia Ringler</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paul Rushton</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Texas A&amp;M AgriLife Research</div>, <div class="addr-line">17360 Coit Road, Dallas, TX 75252</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paul+Rushton">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rushton P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rushton P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Paul Rushton</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qingxi J. Shen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">University of Nevada Las Vegas</div>, <div class="addr-line">4505 South Maryland Parkway, Las Vegas, NV 89154</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 702-895-4704. Fax. +1 702-895-3956. E-mail: <a href="mailto:jeffery.shen@unlv.edu" target="_blank">jeffery.shen@unlv.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qingxi+J.+Shen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shen Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shen Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qingxi J. Shen</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 319–329, <a href='https://doi.org/10.1093/dnares/dsv015'>https://doi.org/10.1093/dnares/dsv015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 September 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 April 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 July 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347881" href="/dnaresearch/article-pdf/22/5/319/996739/dsv015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenneth A. Watanabe, Arielle Homayouni, Tara Tufano, Jennifer Lopez, Patricia Ringler, Paul Rushton, Qingxi J. Shen;  Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing, <em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 319–329, <a href='https://doi.org/10.1093/dnares/dsv015'>https://doi.org/10.1093/dnares/dsv015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347881?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347881?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347881?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347881?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347881?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347881?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347881?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Tiling%20Assembly%3A%20a%20new%20tool%20for%20reference%20annotation-independent%20transcript%20assembly%20and%20novel%20gene%20identification%20by%20RNA-sequencing&amp;publicationDate=2015-09-03&amp;volumeNum=22&amp;issueNum=5&amp;author=Watanabe%2C%20Kenneth%20A.%3B%20Homayouni%2C%20Arielle&amp;startPage=319&amp;endPage=329&amp;contentId=10.1093%2Fdnares%2Fdsv015&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f319%2f347881" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4934499 id="4934499" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Annotation of the rice (<em>Oryza sativa</em>) genome has evolved significantly since release of its draft sequence, but it is far from complete. Several published transcript assembly programmes were tested on RNA-sequencing (RNA-seq) data to determine their effectiveness in identifying novel genes to improve the rice genome annotation. Cufflinks, a popular assembly software, did not identify all transcripts suggested by the RNA-seq data. Other assembly software was CPU intensive, lacked documentation, or lacked software updates. To overcome these shortcomings, a heuristic <em>ab initio</em> transcript assembly algorithm, Tiling Assembly, was developed to identify genes based on short read and junction alignment. Tiling Assembly was compared with Cufflinks to evaluate its gene-finding capabilities. Additionally, a pipeline was developed to eliminate false-positive gene identification due to noise or repetitive regions in the genome. By combining Tiling Assembly and Cufflinks, 767 unannotated genes were identified in the rice genome, demonstrating that combining both programmes proved highly efficient for novel gene identification. We also demonstrated that Tiling Assembly can accurately determine transcription start sites by comparing the Tiling Assembly genes with their corresponding full-length cDNA. We applied our pipeline to additional organisms and identified numerous unannotated genes, demonstrating that Tiling Assembly is an organism-independent tool for genome annotation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-sequencing">RNA-sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcript assembly&quot;">transcript assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Oryza sativa&quot;"><em>Oryza sativa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="cufflinks">cufflinks</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome annotation&quot;">genome annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;novel genes&quot;">novel genes</a></div>                            <h2 scrollto-destination=4934501 id="4934501" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>RNA-sequencing (RNA-seq) technology enables whole-transcriptome profiling via the collection and mapping of short cDNA fragments (reads) to a reference genome.<sup><a reveal-id="DSV015C1" data-open="DSV015C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV015C2" data-open="DSV015C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Regions of the genome where many reads align indicate such regions are highly expressed.<sup><a reveal-id="DSV015C3" data-open="DSV015C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Regions where no known gene has been annotated and a large number of reads align are indicative of an undiscovered gene.<sup><a reveal-id="DSV015C4" data-open="DSV015C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Reconstruction of transcripts can be obtained through a variety of computational strategies, each of which has its own benefits and drawbacks. These assembly algorithms fall into two general classes, <em>ab initio</em> assembly and <em>de novo</em> assembly. <em>Ab initio</em>, mapping-first approaches rely on the availability of a reference genome to which the short reads can be aligned.<sup><a reveal-id="DSV015C5" data-open="DSV015C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The major drawback of this method stems from the dependency of accurate transcript identification on the presence of a high-quality reference genome.<sup><a reveal-id="DSV015C6" data-open="DSV015C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The main benefit of <em>ab initio</em> assembly is the maximum sensitivity exhibited for gene identification,<sup><a reveal-id="DSV015C5" data-open="DSV015C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV015C7" data-open="DSV015C7" class="link link-ref link-reveal xref-bibr">7</a></sup> though higher sensitivity tends to result in a lower accuracy due to a higher number of false-positive genes being reported.<sup><a reveal-id="DSV015C7" data-open="DSV015C7" class="link link-ref link-reveal xref-bibr">7</a></sup> On the other hand, <em>de novo</em> transcript assembly, or assembly-first approaches, is independent of a reference genome and directly determines the transcripts of a genome through the short reads.<sup><a reveal-id="DSV015C5" data-open="DSV015C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Since this avenue is dependent solely on short-read data, genes with low read coverage due to low expression levels can result in inaccurate determination of full-length transcripts.<sup><a reveal-id="DSV015C8" data-open="DSV015C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The main benefit provided by <em>de novo</em> assembly methods is the lack of reliance on a reference genome, which makes it a vital tool for gene identification in organisms that lack a reference genome. Thus, both genome biology as well as the availability of an accurate and complete genome play major factors in the decision to use <em>ab initio</em> assembly versus <em>de novo</em> assembly.<sup><a reveal-id="DSV015C7" data-open="DSV015C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Recently, we published a clustering algorithm to identify novel protein- and microRNA-coding genes by searching only the unannotated regions of the rice genome.<sup><a reveal-id="DSV015C9" data-open="DSV015C9" class="link link-ref link-reveal xref-bibr">9</a></sup> However, this method relied on the genome being partially annotated. Analysis of the RNA-seq data with other transcript assembly software revealed that they failed to identify genes, were CPU intensive, or lacked documentation or recent software updates. Here, we report an improved algorithm for <em>ab initio</em> transcript assembly and novel gene identification, Tiling Assembly, to compensate for such shortfalls.</p><p>By combining Tiling Assembly with Cufflinks,<sup><a reveal-id="DSV015C10" data-open="DSV015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> thousands of potential novel genes were identified in the rice genome. To reduce the possibility of false-positive novel gene identification, stringent filters on minimum gene length, minimum gene expression level, and percent similarity of the potential novel genes to another region in the genome were included. Utilizing this pipeline, 767 high-confidence, unannotated genes were identified in the rice genome. By applying Tiling Assembly to other model organisms, we identified 200 potential novel genes in <em>Arabidopsis thaliana</em>, 126 in <em>Caenorhabditis elegans</em>, 361 in <em>Drosophila melanogaster</em>, and 460 in <em>Saccharomyces cerevisiae</em>. This study demonstrated that, by utilizing Tiling Assembly, many potential novel genes can be identified in even the most well-annotated genomes.</p>                            <h2 scrollto-destination=4934506 id="4934506" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4934507 id="4934507" class="section-title" data-legacy-id=s2a>2.1. The Tiling Assembly pipeline</h3>
<p>The Tiling Assembly pipeline, depicted in Fig. <a reveal-id="DSV015F1" data-open="DSV015F1" class="link link-reveal link-table xref-fig">1</a>, begins with alignment of RNA-seq short-read data to the appropriate genome using the alignment software, Bowtie<sup><a reveal-id="DSV015C10" data-open="DSV015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and Tophat.<sup><a reveal-id="DSV015C11" data-open="DSV015C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Exons of potential genes are identified based on the presence of regions containing overlapping reads. Exons too short to be identified by read alignment are identified by Tophat junction alignments. Gaps in the read coverage can cause single exons to be identified as two or more exons; thus, exons that are very closely spaced are linked together. To prevent accidental merging of exons, exons containing one or more junction alignments are separated into multiple exons. Finally, all of the exons identified are joined together by the junction alignments to form the assembled transcripts. </p>                        <a id="4934509" scrollto-destination="4934509"></a>
<div id="DSV015F1" data-id="DSV015F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01501.png?Expires=2147483647&Signature=29LUqHAS12QULPdK775EVLIBYBtlrSzrCE4WIk3zxt01-NVcKjj8JI7UD5JYLtpLBHOuj47WRgWvgdHe8A30s13hQRd4tWrpmL4-XOPI6Fn2GUWW-VD45LZ5~bk~9PKS29O5Nn1Zpg9Fzp74K7q3tE8bMv3aaAY1kNYmcac7PxsSNubLuRdulDDNW3r40Sfc3Mrt2y-FQUSp9~t3A6DHuAT~C6eihiynjjJeYGVaB~NSfSPuLOSyipkBI67nVC9UzkFmq8ZqZtQ0UQB9KmjAaovAEec1aSario3v8glIK80QvoS9~In2zuYsrcFOVnW9iY5nNz-t7E2miFfqE~ckbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart of the processes for gene identification by Tiling Assembly." path-from-xml="dsv01501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934509/dsv01501.tiff" path-from-xml="dsv01501.tiff" target="_blank">View large</a><a section="4934509" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01501.png?Expires=2147483647&Signature=3KkBk~EvJ9EQU8nTCWW-HiNFvZoJAIovbmAp-308ttPbd8KkjZw4l3tkaVECFBtcC3C3i~xyfZnw2GtUMXgujVT6MgXdwgoDkM~qmyjzoFV1BAU3F9QcfYF53lUjaR4EAf2mCwnfiROFFWzl1PlJTJSxwwwKgV6B99jOLtrqWkdo01fFXgvxSAwnKlTC2X36Gpzil2XvNDoRoaefZzuxWlnV2ZNhabKXuSPBbWGkYWCt7bRhn1KgqmOIC8cKpw4iNQ8e5hx6874akCrxDaRAFDii5kg0Xwe5nyiDcT~Byb4A5xKbJbs77clSYBKvEHUYgh95PRPHZvAK9dyi6bATyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934509&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart of the processes for gene identification by Tiling Assembly.</p></div></div><div content-id="DSV015F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01501.png?Expires=2147483647&Signature=29LUqHAS12QULPdK775EVLIBYBtlrSzrCE4WIk3zxt01-NVcKjj8JI7UD5JYLtpLBHOuj47WRgWvgdHe8A30s13hQRd4tWrpmL4-XOPI6Fn2GUWW-VD45LZ5~bk~9PKS29O5Nn1Zpg9Fzp74K7q3tE8bMv3aaAY1kNYmcac7PxsSNubLuRdulDDNW3r40Sfc3Mrt2y-FQUSp9~t3A6DHuAT~C6eihiynjjJeYGVaB~NSfSPuLOSyipkBI67nVC9UzkFmq8ZqZtQ0UQB9KmjAaovAEec1aSario3v8glIK80QvoS9~In2zuYsrcFOVnW9iY5nNz-t7E2miFfqE~ckbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart of the processes for gene identification by Tiling Assembly." path-from-xml="dsv01501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934509/dsv01501.tiff" path-from-xml="dsv01501.tiff" target="_blank">View large</a><a section="4934509" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01501.png?Expires=2147483647&Signature=3KkBk~EvJ9EQU8nTCWW-HiNFvZoJAIovbmAp-308ttPbd8KkjZw4l3tkaVECFBtcC3C3i~xyfZnw2GtUMXgujVT6MgXdwgoDkM~qmyjzoFV1BAU3F9QcfYF53lUjaR4EAf2mCwnfiROFFWzl1PlJTJSxwwwKgV6B99jOLtrqWkdo01fFXgvxSAwnKlTC2X36Gpzil2XvNDoRoaefZzuxWlnV2ZNhabKXuSPBbWGkYWCt7bRhn1KgqmOIC8cKpw4iNQ8e5hx6874akCrxDaRAFDii5kg0Xwe5nyiDcT~Byb4A5xKbJbs77clSYBKvEHUYgh95PRPHZvAK9dyi6bATyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934509&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart of the processes for gene identification by Tiling Assembly.</p></div></div><p>Gene identification begins with the detection of exons based on the short-read data. The aligned reads are loaded into a MySQL database, which is queried by Tiling Assembly to identify exons based on the presence of overlapping reads. To prevent misidentification of exons due to noise, a threshold may be set by the user to determine the minimum reads per kilobase of exon (RPKE) required to identifying an exon.</p><p>Exons shorter than the length of a read (50 nt) tend to have few to no reads aligned, regardless of gene expression, preventing them from being detected by considering only read alignment. To identify these short exons, Tiling Assembly relies on the partial read alignments from junction mapping. Tophat takes the reads that cannot be directly mapped to the genome and breaks them into two parts for independent alignment. The junctions that these reads map across are indicative of an intron. Tiling Assembly relies on these partial read alignments to detect short exons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>Low read coverage from genes with low expression often leads to gaps in the read coverage, causing alignment algorithms to mistakenly identify multiple exons where only a single exon is present. To avoid such false gaps, Tiling Assembly merges exons that are within a user-specified distance of each other. Linking closely spaced exons together, however, may result in an incorrect merging of exons.</p><p>Other factors, such as intron retention, pre-spliced mRNA, and noise, may also contribute to incorrectly merged exons because they result in reads mapping to intronic regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). To prevent mistakenly merged exons, Tiling Assembly searches for junction alignments within the identified exons. Exons containing both sides of a junction alignment are separated and trimmed based on the boundaries of the junction to ensure accurate exon–intron boundaries (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). The beginning of the first exon and the end of the last exon of a transcript cannot be determined by junction alignment.</p><p>Once these high-confidence exons are produced, Tiling Assembly assembles the exons into specific genes using junction alignments. To avoid false junctions between similar genomic regions, the user can specify a maximum length of a junction that skips over one or more exons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). In addition, the user can specify the size of very large junctions to be disregarded to avoid invalid junctions due to mapping errors.</p>                            <h3 scrollto-destination=4934515 id="4934515" class="section-title" data-legacy-id=s2b>2.2. RNA-seq and genome data</h3>
<p>The RNA-seq data used for rice were obtained from RNA extraction of rice aleurone performed in our laboratory, followed by library preparation and sequencing on the Illumina Hi-seq 2000 platform by the Huntsman Cancer Institute, University of Utah. The data were submitted to the Sequence Read Archive (SRA)<sup><a reveal-id="DSV015C12" data-open="DSV015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and are publicly accessible under the accession number SRP028376. The SRA accession numbers that were used for the analysis of other species were SRP022162 (<em>A. thaliana</em>), SRR590802-4 (<em>D. melanogaster</em>), SRR650494-5 (<em>C. elegans</em>), and SRR1019759 (<em>S. cerevisiae</em>). </p><ul class="list-simple"><li><p>The rice genome and annotation were downloaded from the MSU Rice Genome Annotation Project Release 7.0 (MSU R7)<sup><a reveal-id="DSV015C13" data-open="DSV015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> for <em>Oryza sativa</em> (<a class="link link-uri" href="http://ftp://ftp.plantbiology.msu.edu/pub/data/Eukaryotic_Projects/o_sativa/annotation_dbs/pseudomolecules/version_7.0/">ftp://ftp.plantbiology.msu.edu/pub/data/Eukaryotic_Projects/o_sativa/annotation_dbs/pseudomolecules/version_7.0/</a>).</p></li><li><p>The <em>A. thaliana</em> data were downloaded from the PhytozomeV10<sup><a reveal-id="DSV015C14" data-open="DSV015C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (<a class="link link-uri" href="http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=PhytozomeV10">http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=PhytozomeV10</a>).</p></li><li><p>The <em>C. elegans</em> data were downloaded from the WormBase<sup><a reveal-id="DSV015C15" data-open="DSV015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> (<a class="link link-uri" href="http://ftp://ftp.ensembl.org/pub/release-75/fasta/caenorhabditis_elegans/dna/">ftp://ftp.ensembl.org/pub/release-75/fasta/caenorhabditis_elegans/dna/</a>).</p></li><li><p>The yeast data were downloaded from the <em>Saccharomyces</em> Genome Database<sup><a reveal-id="DSV015C16" data-open="DSV015C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (<a class="link link-uri" href="http://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/">http://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/</a>).</p></li><li><p>The <em>D. melanogaster</em> data were downloaded from the FlyBase<sup><a reveal-id="DSV015C17" data-open="DSV015C17" class="link link-ref link-reveal xref-bibr">17</a></sup> (<a class="link link-uri" href="http://ftp://ftp.flybase.net/genomes/Drosophila_melanogaster/dmel_r5.56_FB2014_02/fasta/">ftp://ftp.flybase.net/genomes/Drosophila_melanogaster/dmel_r5.56_FB2014_02/fasta/</a>).</p></li></ul>                            <h3 scrollto-destination=4934518 id="4934518" class="section-title" data-legacy-id=s2c>2.3. Alignment of full-length cDNA to the rice genome</h3>
<p>The full-length cDNAs (FL-cDNAs)<sup><a reveal-id="DSV015C18" data-open="DSV015C18" class="link link-ref link-reveal xref-bibr">18</a></sup> were aligned to the rice genome using the Exonerate alignment software with the following parameters: model est2genome, geneseed 250, and bestn 1.<sup><a reveal-id="DSV015C19" data-open="DSV015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The FL-cDNAs and short-read data were loaded into the University of California, Santa Cruz (UCSC) Genome Browser,<sup><a reveal-id="DSV015C20" data-open="DSV015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> for visualization at the following address: <a class="link link-uri" href="http://shenlab.sols.unlv.edu/cgi-bin/hgGateway">http://shenlab.sols.unlv.edu/cgi-bin/hgGateway</a>.</p>                            <h3 scrollto-destination=4934520 id="4934520" class="section-title" data-legacy-id=s2d>2.4. Short-read alignment</h3>
<p>Short reads were aligned to MSU R7 via Bowtie version 2.1.0 and Tophat version 2.0.9 software and OLego.<sup><a reveal-id="DSV015C21" data-open="DSV015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The maximum junction length was set to 50,000 nt. Default values were used for all other parameters. The short-read data for all of the samples were merged. Transcript assembly was then performed using Cufflinks version 2.0.2 on the composite data to generate GTF files. The mapped short reads and junctions were loaded into a MySQL database. Tiling Assembly queried the database to identify exons and genes.</p>                            <h3 scrollto-destination=4934523 id="4934523" class="section-title" data-legacy-id=s2e>2.5. Sample size calculation for comparison of Tiling Assembly and Cufflinks genes with FL-cDNA</h3>
<p>The minimum sample size (<em>n</em>), needed to ensure the genes identified by Tiling Assembly and Cufflinks coincided with FL-cDNAs, was calculated using the following equation: <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">n</mi><mo xmlns="">=</mo><mstyle displaystyle="true" xmlns=""><mrow><mfrac><mrow><mi>N</mi><mi>x</mi></mrow><mrow><mo stretchy="false">[</mo><mo stretchy="false">(</mo><mi>N</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mrow><msup><mi>E</mi><mn>2</mn></msup></mrow><mo>+</mo><mi>x</mi><mo stretchy="false">]</mo></mrow></mfrac></mrow></mstyle></math>⁠</span>, where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">x</mi><mo xmlns="">=</mo><msup xmlns=""><mrow><msub><mi>Z</mi><mrow><mfenced close=")" open="("><mrow><mstyle displaystyle="true"><mrow><mfrac><mi>c</mi><mrow><mn>100</mn></mrow></mfrac></mrow></mstyle></mrow></mfenced></mrow></msub></mrow><mn>2</mn></msup><mi xmlns="">r</mi><mo stretchy="false" xmlns="">(</mo><mn xmlns="">100</mn><mo xmlns="">−</mo><mi xmlns="">r</mi><mo stretchy="false" xmlns="">)</mo></math>⁠</span>;<sup><a reveal-id="DSV015C22" data-open="DSV015C22" class="link link-ref link-reveal xref-bibr">22</a></sup><em>N</em> represents the total number of discrepant genes; <em>E</em> is the margin of error, which was set to 5%; the critical value <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>Z</mi><mrow><mfenced close=")" open="("><mrow><mstyle displaystyle="true"><mrow><mfrac><mi>c</mi><mrow><mn>100</mn></mrow></mfrac></mrow></mstyle></mrow></mfenced></mrow></msub></mrow></math></span> was set to 1.96 based on a 95% confidence level (<em>c</em>); and <em>r</em> was set to 60% since this value was the expected discrepancy rate contributed to alternative splicing.</p>                            <h2 scrollto-destination=4934528 id="4934528" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4934531 id="4934531" class="section-title" data-legacy-id=s3a>3.1. Short-read alignment</h3>
<p>The short-read data from our previous publication were used in this study since we obtained the data and can vouch for its quality.<sup><a reveal-id="DSV015C9" data-open="DSV015C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In addition, using the same data allowed for comparison between our previously published results and the results in this study. The short reads were aligned to MSU R7 using the latest available version of Bowtie and Tophat. Of the 157,773,782 reads, 151,492,182 reads (96.0%) were aligned to the rice genome. Of the reads that aligned, over 10% aligned to currently unannotated regions, indicating that there are potentially many unidentified novel genes in the rice genome, similar to what was found for the human genome.<sup><a reveal-id="DSV015C4" data-open="DSV015C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Though the focus on this study is on transcript assembly rather than mapping, we compared Tophat with the splice-sensitive mapping tool OLego. Both programmes identified junctions that the other did not, indicating that neither programme was 100% accurate (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Of the 158,314 junctions identified by OLego, 124,594 junctions (78.7%) matched identically with a junction identified by Tophat. Although OLego identified more junctions, most of the junctions uniquely identified by OLego (71.3%) were determined from a single read. Since nearly 80% of the OLego junctions matched identically with Tophat and most of the remaining were derived from a single read, we chose to keep within the Tuxedo suite software since Cufflinks was designed to work with Bowtie and Tophat. OLego may be used as an alternative mapping tool if so desired.</p>                            <h3 scrollto-destination=4934541 id="4934541" class="section-title" data-legacy-id=s3b>3.2. Benchmark analysis</h3>
<p>To determine the minimum level of read alignment required for reliable detection of transcripts, 80 highly expressed single-isoform genes with evenly distributed reads across the exons were inserted into a randomly generated test genome. A control sample containing about 40 million reads was aligned to the test genome and not a single read mapped. This demonstrated that the test genome was an ideal method for excluding noise. Single-isoform genes were used to simplify the problem of correctly identifying all exons, while also allowing us to accurately determine the baseline for exon detection. Four categories were used, containing 20 genes each: one exon, two exons, three exons, and four exons. Tophat was used to align RNA-seq data from the control sample to the genes. Prior to analysing its ability to identify exons at a specific expression level, Tiling Assembly was run on the data to ensure the genes were properly identified as single-isoform genes with the appropriate number of exons. The minimum expression level used by Tiling Assembly to detect exons for these experiments was set to 50 RPKE, based on our analysis in Section 3.3., which indicated that expression levels below this threshold led to the identification of noise reads as exons. To test the ability of Tiling Assembly to correctly identify the genes at specific expression levels, 1,000 reads were randomly selected from the pool of reads that aligned to each gene and run through Tiling Assembly. The number of randomly selected reads was incrementally reduced, and Tiling Assembly was run again. This process was repeated several times until 50 reads per gene were selected. To compare the performance of Tiling Assembly with a well-established assembly algorithm, the same procedure was performed using Cufflinks. The accuracy with which Tiling Assembly and Cufflinks identified the genes was determined using a point of first failure method, which assumed that the highest RPKE, where a gene was first misidentified, was the point where identification becomes unreliable. As can be seen in Fig. <a reveal-id="DSV015F2" data-open="DSV015F2" class="link link-reveal link-table xref-fig">2</a>, at an expression level of 100 RPKE, both Tiling Assembly and Cufflinks were able to identify single-exon and multi-exon genes with an 85% or higher accuracy rate. At a read depth lower than 100 RPKE, the accuracy rate dropped dramatically for both Cufflinks and Tiling Assembly, though Cufflinks' accuracy declined at a slower rate. These data indicate that Cufflinks and Tiling Assembly can accurately identify gene transcripts at a read depth of 100 RPKE or greater. </p>                        <a id="4934549" scrollto-destination="4934549"></a>
<div id="DSV015F2" data-id="DSV015F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01502.png?Expires=2147483647&Signature=FNfclT4TxfPW2AKX6J-D7vH7j8Px1gXHDtSOkPtT~HKgi96eqeWjz6TB-nmt0ifMo0PW20iHK55dehJauKQf6IefI5Br7fUWOFgr9aGkh9sSESeQnjnQJF0TJlQpPqseBJMVnezasqnKEHJlt1D9JQmcj3nopH-Dp86K-~1996dKrUBIjnyljVdHB1gev5MkyCmTMCI~kATccfafStkz4WjCnoRiR2aE2~N8rqKNOfATxoFZb~utHraPIN0WtG~0u74MhyEuXJegXsH8JbVyZPEtXwFgeAqdtX2xH8keO9lEeWq1~MjR-iC2fve2g0h5O5aFXfbxEOMrtFbfFTC-Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tiling Assembly and Cufflinks show similar accuracy predicting known genes inserted in a random test genome. A random test genome containing 80 highly expressed, single-isoform genes was created. Four categories were used, containing 20 genes each: (A) one exon, (B) two exons, (C) three exons, and (D) four exons. The number of reads aligned to each gene was varied to determine the read depth at which each programme failed to accurately predict the genes." path-from-xml="dsv01502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934549/dsv01502.tiff" path-from-xml="dsv01502.tiff" target="_blank">View large</a><a section="4934549" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01502.png?Expires=2147483647&Signature=fgpFE~FNM0YDn3wHYU3-Px7jCmIPxE-~V7~-iVMb1onNgJv7WM82tQdPZmUR5GaVJR3LHpjtD0GBCQTYKb3PNXnMpIUhwYlmRIRKqXsYwqTQ30Q6tuspuYQgn2qIv9M63yIHXaW-IXU56rQe14r13y0vYHRBSvC1HjHLsEr2wrGRLHKFgbNZ0Qxl-kwn8o20t4fgv9PlX~0onxncWCKmRR~mAFOqhx20ICBUFdzDVuRvVbjtp9rvazzmWq-tBjf5Qe9RvhpuGIYSamcPL8dghvKvFz9f8Q5tPfYVB14dICm4mDxGJBiguULRS-UZtftqyzRMcuHBtVrEfwo4YWZkwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934549&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tiling Assembly and Cufflinks show similar accuracy predicting known genes inserted in a random test genome. A random test genome containing 80 highly expressed, single-isoform genes was created. Four categories were used, containing 20 genes each: (A) one exon, (B) two exons, (C) three exons, and (D) four exons. The number of reads aligned to each gene was varied to determine the read depth at which each programme failed to accurately predict the genes.</p></div></div><div content-id="DSV015F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01502.png?Expires=2147483647&Signature=FNfclT4TxfPW2AKX6J-D7vH7j8Px1gXHDtSOkPtT~HKgi96eqeWjz6TB-nmt0ifMo0PW20iHK55dehJauKQf6IefI5Br7fUWOFgr9aGkh9sSESeQnjnQJF0TJlQpPqseBJMVnezasqnKEHJlt1D9JQmcj3nopH-Dp86K-~1996dKrUBIjnyljVdHB1gev5MkyCmTMCI~kATccfafStkz4WjCnoRiR2aE2~N8rqKNOfATxoFZb~utHraPIN0WtG~0u74MhyEuXJegXsH8JbVyZPEtXwFgeAqdtX2xH8keO9lEeWq1~MjR-iC2fve2g0h5O5aFXfbxEOMrtFbfFTC-Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tiling Assembly and Cufflinks show similar accuracy predicting known genes inserted in a random test genome. A random test genome containing 80 highly expressed, single-isoform genes was created. Four categories were used, containing 20 genes each: (A) one exon, (B) two exons, (C) three exons, and (D) four exons. The number of reads aligned to each gene was varied to determine the read depth at which each programme failed to accurately predict the genes." path-from-xml="dsv01502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934549/dsv01502.tiff" path-from-xml="dsv01502.tiff" target="_blank">View large</a><a section="4934549" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01502.png?Expires=2147483647&Signature=fgpFE~FNM0YDn3wHYU3-Px7jCmIPxE-~V7~-iVMb1onNgJv7WM82tQdPZmUR5GaVJR3LHpjtD0GBCQTYKb3PNXnMpIUhwYlmRIRKqXsYwqTQ30Q6tuspuYQgn2qIv9M63yIHXaW-IXU56rQe14r13y0vYHRBSvC1HjHLsEr2wrGRLHKFgbNZ0Qxl-kwn8o20t4fgv9PlX~0onxncWCKmRR~mAFOqhx20ICBUFdzDVuRvVbjtp9rvazzmWq-tBjf5Qe9RvhpuGIYSamcPL8dghvKvFz9f8Q5tPfYVB14dICm4mDxGJBiguULRS-UZtftqyzRMcuHBtVrEfwo4YWZkwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934549&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tiling Assembly and Cufflinks show similar accuracy predicting known genes inserted in a random test genome. A random test genome containing 80 highly expressed, single-isoform genes was created. Four categories were used, containing 20 genes each: (A) one exon, (B) two exons, (C) three exons, and (D) four exons. The number of reads aligned to each gene was varied to determine the read depth at which each programme failed to accurately predict the genes.</p></div></div><p>Cufflinks uses a parsimony algorithm to identify transcripts,<sup><a reveal-id="DSV015C23" data-open="DSV015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> using the smallest number of transcripts to explain all the reads. Thus, despite possible gaps in the reads, if the reads can be explained by a single transcript, Cufflinks assumes a single transcript. This assumption allows Cufflinks to correctly identify single-exon genes down to as low as 50 RPKE. The accuracy rates reported from the test genome favour Cufflinks; however, RNA-seq data in an actual genome would contain noise reads. These stray reads can cause Cufflinks to mistakenly identify them as individual genes or exons. In contrast, the set 50 RPKE cutoff utilized by Tiling Assembly for these experiments prevents such noise reads from being classified as genes.</p><p>To determine the rate of false-positive gene identification, 100 genes were indiscriminately selected and inserted into another random test genome. Both Tiling Assembly and Cufflinks found at least one gene at each of the locations, however, both also erroneously identified extra genes. Cufflinks reported 125 genes, whereas Tiling Assembly identified 117 genes. Investigation of the 25 extra genes identified by Cufflinks revealed numerous causes for misidentification, including missing junctions, gaps, and false identification of genes in intronic regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S6–S8</a>). While Tiling Assembly also overestimated the number of genes for similar reasons, it did so at a lower rate than Cufflinks. The main cause of this difference was due to identification of genes in intronic regions by Cufflinks, whereas Tiling Assembly disregarded these as noise.</p><p>Investigation on the ability of Tiling Assembly and Cufflinks to correctly identify exons in random test genomes revealed a trade-off between the two methods. While Cufflinks was better able to accurately identify genes in the test genome at expression levels lower than 100 RPKE, both programmes showed similar accuracy above 100 RPKE. Cufflinks was more likely to mistakenly identify intronic regions containing noise reads as genes, whereas Tiling Assembly's more conservative approach to exon identification decreased the identification of these false genes. Actual genomes contain many noise reads aligning on intronic and intergenic regions, in which case Tiling Assembly would identify fewer false-positive genes.</p>                            <h3 scrollto-destination=4934558 id="4934558" class="section-title" data-legacy-id=s3c>3.3. Application of Tiling Assembly identified 40,491 genes expressed in rice aleurone cells</h3>
<p>To determine the gene identification capabilities of Tiling Assembly in an actual genome, Tiling Assembly was applied to rice aleurone short-read data composited from four samples and aligned to MSU R7. Prior to application of Tiling Assembly to the rice genome, it was necessary to determine the minimum gene expression required for accurate detection of exons. This was determined through analysis of several genes, with <em>LOC_Os01g01010</em> used as a representative example. Tiling Assembly was run multiple times on these genes, with varied expression thresholds, to determine at what point the exons were accurately recognized. The threshold was incrementally reduced from 100 RPKE. At 50 RPKE, Tiling Assembly identified exons e3 and e4 of <em>LOC_Os01g01010</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>); however, lower expression thresholds resulted in identification of false exons due to noise read alignment. A threshold of 50 RPKE was thus used to ensure accurate exon identification.</p><p>Exons were identified from MSU R7 using several steps. First, identification of exons was achieved through analysis of overlapping reads aligning to the same region of the genome, with minimum expression of 50 RPKE required for these regions to be considered an exon. This step led to an initial identification of 207,908 potential exons.</p><p>Identification of exons via tiling of read alignments limits the size of the exons found to the length of a single read. Tiling Assembly gets around this limitation via analysis of junction alignments produced by Tophat, as described in Section 2.1. In this second step of exon identification, an additional 1,397 potential exons were identified, bringing the total number to 209,305 potential exons.</p><p>When a gene has low read coverage, there may be exons which contain gaps in read alignment. Since Tiling Assembly depends on overlapping reads to identify exons, such gaps lead to artificial fragmenting of exons. To avoid this issue, exons that were within 50 nt of each other, a space which could be closed by the length of a single read in our data set, were merged together. To determine whether this merging of exons was likely to result in erroneous merging of genes, the annotated genes in MSU R7 were investigated. It was found that only 0.36% of the non-overlapping annotated genes reside within 50 nt of one another, so linking exons 50 nt apart should not cause a significant misrepresentation of the number of genes obtained from the overall RNA-seq analysis. However, linking closely spaced exons together may result in false merging of exons of the same gene. Of the 209,305 potential exons found in our RNA-seq data, 50,895 fragments were within 50 nt of each other. After linking these closely spaced fragments together, there were 158,410 potential exons.</p><p>To fix any exons that may have been mistakenly merged, either by noise or linking of exons, Tiling Assembly utilized the junction alignments produced by Tophat. Tiling Assembly searched for exons that contained a Tophat junction to identify any exons that may have been mistakenly merged. Our goal in this analysis with Tiling Assembly was to find the most common isoform of a gene where intron retention is a possibility. Thus, if the density of reads aligning on the Tophat junction was &lt;50%, compared with those aligned to the adjacent regions, then the junction was considered to be an intron (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). This splitting of exons increased the total number of potential exons to 185,445 exons.</p><p>Once the exons were identified, the Tophat junction alignments were used to join exons together to form transcripts. Occasionally, Tophat maps false junctions across large distances due to sequence similarities to the actual junction elsewhere in the genome. To avoid considering these false junctions, Tiling Assembly was set to disregard junctions that skipped exons and spanned distances greater than 50k nt. In addition, in our previous study,<sup><a reveal-id="DSV015C9" data-open="DSV015C9" class="link link-ref link-reveal xref-bibr">9</a></sup> it was found that there were numerous areas of the genome where large numbers of reads mapped to small regions, often &lt;140 nt in length, due to high sequence similarity to other highly expressed regions of the genome. Tiling Assembly was thus set to disregard potential genes that had &lt;140 nt. After linking all of the exons together and removing these very small genes, 40,491 genes were identified, containing 136,164 exons. This number does not represent the entire rice genome because Tiling Assembly relies on transcriptome data for gene identification and not all genes are expected to be expressed in all tissues, such as aleurone.</p><p>The genes found by Tiling Assembly were compared with the annotated genes in MSU R7. Of the 40,491 genes identified, 28,019 overlapped with an annotated gene by at least 75%, and 10,129 genes by &lt;5% (Fig. <a reveal-id="DSV015F3" data-open="DSV015F3" class="link link-reveal link-table xref-fig">3</a>). Thus, 94% of the genes identified by Tiling Assembly either corresponded well to an annotated gene or by a minimal amount. The 10,129 minimally overlapping genes were considered as potential novel genes. </p>                        <a id="4934575" scrollto-destination="4934575"></a>
<div id="DSV015F3" data-id="DSV015F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01503.png?Expires=2147483647&Signature=Xdi8hpKl0ZhVYkiLhwPhvYEYvzaYjED4SQS45rD1ZgfYlRJU2cwTOb59XbeWq8nw~nE2eAS~hZb7L9hUPkEZxbz4qk2b7JW9a9FiW4sUWXh-6WiB5g5iEHjPnsh3FOp-DL2hyDpkRbp4lOyCanlgx6yKfUvFrllW~5YQuBBj03gCNiiLyYYTaypzn7OE8eZxrEzqGLdRPXqX~5DWPyHARyM5-uvy6cM1hWDL7-M8yd6YVyxnfERtgOChdoZUi2wVvGUHcwb4e6pnl6f48BCCrrAm1qmJKh4zZv5jE3sOX2P01otMVQec-auQESIvljrpgZ18cGdS4Y4BHTyWKZlS~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes identified by Tiling Assembly overlapped either well or minimally with annotated genes. The start and termination positions of genes identified by Tiling Assembly were compared with those of the MSU R7 genes to determine the amount of each Tiling Assembly gene corresponded with an MSU R7 gene. Of the 40,491 genes, 28,019 corresponded to an annotated gene by &gt;75%, and 10,129 by &lt;5%. This 5% category represents potential novel genes." path-from-xml="dsv01503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934575/dsv01503.tiff" path-from-xml="dsv01503.tiff" target="_blank">View large</a><a section="4934575" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01503.png?Expires=2147483647&Signature=X9BG7vl~4BR8eKDnHXb-OBZli4kZQlTsidNW38oAnp-lRtltxxIlzK9oCmLHnQMHqFK~CDEjCSe-708pVti0iolqNcVyh401Q59Oqu49QopUKrYG19KVaqzIcRMzVooDgoSJ75MAY2INShl96JtD6reTJGoSiVNMT-8UKCLjp1i6UtBCfPN2d9HEx1f5~blIzd40yu6YaTaxQ0cXE7dFsy7AgQlYA54SMy3XmjR31ZtilwIVZ-8nO7WoS8z65Jh-HXeNBCem83c8PPKIeNZqb00g5S1oiQ88akeJmQXYTmKnWgiWVK0bNVBYpNUCnXMTMkD~b9~PP0JwGomvyc0lFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934575&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genes identified by Tiling Assembly overlapped either well or minimally with annotated genes. The start and termination positions of genes identified by Tiling Assembly were compared with those of the MSU R7 genes to determine the amount of each Tiling Assembly gene corresponded with an MSU R7 gene. Of the 40,491 genes, 28,019 corresponded to an annotated gene by &gt;75%, and 10,129 by &lt;5%. This 5% category represents potential novel genes.</p></div></div><div content-id="DSV015F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01503.png?Expires=2147483647&Signature=Xdi8hpKl0ZhVYkiLhwPhvYEYvzaYjED4SQS45rD1ZgfYlRJU2cwTOb59XbeWq8nw~nE2eAS~hZb7L9hUPkEZxbz4qk2b7JW9a9FiW4sUWXh-6WiB5g5iEHjPnsh3FOp-DL2hyDpkRbp4lOyCanlgx6yKfUvFrllW~5YQuBBj03gCNiiLyYYTaypzn7OE8eZxrEzqGLdRPXqX~5DWPyHARyM5-uvy6cM1hWDL7-M8yd6YVyxnfERtgOChdoZUi2wVvGUHcwb4e6pnl6f48BCCrrAm1qmJKh4zZv5jE3sOX2P01otMVQec-auQESIvljrpgZ18cGdS4Y4BHTyWKZlS~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes identified by Tiling Assembly overlapped either well or minimally with annotated genes. The start and termination positions of genes identified by Tiling Assembly were compared with those of the MSU R7 genes to determine the amount of each Tiling Assembly gene corresponded with an MSU R7 gene. Of the 40,491 genes, 28,019 corresponded to an annotated gene by &gt;75%, and 10,129 by &lt;5%. This 5% category represents potential novel genes." path-from-xml="dsv01503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934575/dsv01503.tiff" path-from-xml="dsv01503.tiff" target="_blank">View large</a><a section="4934575" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01503.png?Expires=2147483647&Signature=X9BG7vl~4BR8eKDnHXb-OBZli4kZQlTsidNW38oAnp-lRtltxxIlzK9oCmLHnQMHqFK~CDEjCSe-708pVti0iolqNcVyh401Q59Oqu49QopUKrYG19KVaqzIcRMzVooDgoSJ75MAY2INShl96JtD6reTJGoSiVNMT-8UKCLjp1i6UtBCfPN2d9HEx1f5~blIzd40yu6YaTaxQ0cXE7dFsy7AgQlYA54SMy3XmjR31ZtilwIVZ-8nO7WoS8z65Jh-HXeNBCem83c8PPKIeNZqb00g5S1oiQ88akeJmQXYTmKnWgiWVK0bNVBYpNUCnXMTMkD~b9~PP0JwGomvyc0lFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934575&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genes identified by Tiling Assembly overlapped either well or minimally with annotated genes. The start and termination positions of genes identified by Tiling Assembly were compared with those of the MSU R7 genes to determine the amount of each Tiling Assembly gene corresponded with an MSU R7 gene. Of the 40,491 genes, 28,019 corresponded to an annotated gene by &gt;75%, and 10,129 by &lt;5%. This 5% category represents potential novel genes.</p></div></div><p>The 2,343 Tiling Assembly genes that overlapped with an annotated gene between 5 and 75% may be the result of undiscovered alternatively spliced forms of known genes. Since these transcripts were identified using RNA-seq data from rice aleurone cells, a tissue that has not previously been used for gene identification, the presence of potential unannotated alternative splice variants of genes is not surprising.</p>                            <h3 scrollto-destination=4934580 id="4934580" class="section-title" data-legacy-id=s3d>3.4. Application of Cufflinks identified 38,175 genes expressed in rice aleurone cells</h3>
<p>To compare the gene-finding capabilities of Tiling Assembly with a well-established assembly programme, Cufflinks was run on the same RNA-seq data. Of the 38,175 transcripts identified by Cufflinks, 32,969 overlapped with an MSU R7 annotation by at least 5%. The remaining 5,206 transcripts included multiple isoforms of the same genes. To reduce overrepresentation of the same gene, transcripts that were completely contained within another transcript were eliminated. This left 4,051 potential novel genes identified by Cufflinks. Of these, 48 genes were below the minimum 140 nt requirement used in this analysis and 18 genes were on unknown chromosomes. Therefore, 3,985 potential novel genes were identified by Cufflinks.</p>                            <h3 scrollto-destination=4934585 id="4934585" class="section-title" data-legacy-id=s3e>3.5. Comparison of the novel genes identified by Tiling Assembly and Cufflinks</h3>
<p>Among the potential novel genes identified by Tiling Assembly and Cufflinks, 1,316 genes were identified exclusively by Cufflinks, 7,460 by Tiling Assembly, and 2,669 by both (Fig. <a reveal-id="DSV015F4" data-open="DSV015F4" class="link link-reveal link-table xref-fig">4</a>A). After eliminating potential novel genes with low expression, using 100 RPKE as a threshold for gene identification based on our benchmark analysis (Fig. <a reveal-id="DSV015F2" data-open="DSV015F2" class="link link-reveal link-table xref-fig">2</a>), 4,690 genes were identified as potential novel genes. Of these, 3,473 genes were identified by Tiling Assembly, 52 by Cufflinks, and 1,166 by both (Fig. <a reveal-id="DSV015F4" data-open="DSV015F4" class="link link-reveal link-table xref-fig">4</a>B). </p>                        <a id="4934590" scrollto-destination="4934590"></a>
<div id="DSV015F4" data-id="DSV015F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01504.png?Expires=2147483647&Signature=ljuG3~I~O9aMuHkXJFzDSblN2y7PDppd0Jdu6re5JkCGbbW~Y6P7j91A27zpzREeUS2CrrmM2g6PsWUaYWmjbZ8INBEaYiaJrx-7z7ySbmJ720LP4lbT04mmkYM3MT3sbv-gq~0tusULdzeKFRSmxNYsxCwtjftbK-gR8k9DA4I7zJnZdMBugQ4oi9n1vuWRtsln3ZaiDDnL3k8mR7n8vY8gH6hbN-D-RUdVzQwJLmunmdYFLpjW-i5OxB4EfgdfDcjtsej15CU~vYa931TDDLhr2GEMQE3DE7bsfms5bMMvi8h1-WhvzQfzRzO9JlVaxi9LCmwR7fqp3k3Cmysc5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tiling Assembly identified over 2.5 times more novel genes than Cufflinks. Novel genes identified by Tiling Assembly and Cufflinks were compared to determine how much they overlapped. (A) When all novel genes were considered, 7,460 genes were exclusively identified by Tiling Assembly, 1,316 by Cufflinks, and 2,669 by both. (B) When only novel genes with high expression levels (≥100 RPKE) were considered, 3,473 genes were exclusively identified by Tiling Assembly, 52 by Cufflinks, and 1,166 by both." path-from-xml="dsv01504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934590/dsv01504.tiff" path-from-xml="dsv01504.tiff" target="_blank">View large</a><a section="4934590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01504.png?Expires=2147483647&Signature=Rq~tEMPy6N7LcOMgvo3EoMPEjNQun2HZ1MeYmirqM8fU22Ea2FTzmWN021uYp9KzEBkXFBcDE2xh1UPL263ldStSBRvkIb8U9FM9lfyCoZ~Yn48HUthfh-IHjzi4YeLBPIoC8mWr-MfSqBoEIhebuA2PmwK561Yp~aFZ2i6Ts~Xe4d-RYmlJVqiEGqKzJnsIBQWrBrpdElXKhzcGbhkXWv6EaXVRPaIENlrlm7zOVFMR6jXZUi~0dZ190w4mQAQdZgCFY4gMObL~rh7RjgpJdXkZ9aOJM-EQ25rYyDOCo2BSUu~VnUOJaWYAahPY6AMDqCByWcmptP9WKEyqjWsdYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934590&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tiling Assembly identified over 2.5 times more novel genes than Cufflinks. Novel genes identified by Tiling Assembly and Cufflinks were compared to determine how much they overlapped. (A) When all novel genes were considered, 7,460 genes were exclusively identified by Tiling Assembly, 1,316 by Cufflinks, and 2,669 by both. (B) When only novel genes with high expression levels (≥100 RPKE) were considered, 3,473 genes were exclusively identified by Tiling Assembly, 52 by Cufflinks, and 1,166 by both.</p></div></div><div content-id="DSV015F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01504.png?Expires=2147483647&Signature=ljuG3~I~O9aMuHkXJFzDSblN2y7PDppd0Jdu6re5JkCGbbW~Y6P7j91A27zpzREeUS2CrrmM2g6PsWUaYWmjbZ8INBEaYiaJrx-7z7ySbmJ720LP4lbT04mmkYM3MT3sbv-gq~0tusULdzeKFRSmxNYsxCwtjftbK-gR8k9DA4I7zJnZdMBugQ4oi9n1vuWRtsln3ZaiDDnL3k8mR7n8vY8gH6hbN-D-RUdVzQwJLmunmdYFLpjW-i5OxB4EfgdfDcjtsej15CU~vYa931TDDLhr2GEMQE3DE7bsfms5bMMvi8h1-WhvzQfzRzO9JlVaxi9LCmwR7fqp3k3Cmysc5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tiling Assembly identified over 2.5 times more novel genes than Cufflinks. Novel genes identified by Tiling Assembly and Cufflinks were compared to determine how much they overlapped. (A) When all novel genes were considered, 7,460 genes were exclusively identified by Tiling Assembly, 1,316 by Cufflinks, and 2,669 by both. (B) When only novel genes with high expression levels (≥100 RPKE) were considered, 3,473 genes were exclusively identified by Tiling Assembly, 52 by Cufflinks, and 1,166 by both." path-from-xml="dsv01504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934590/dsv01504.tiff" path-from-xml="dsv01504.tiff" target="_blank">View large</a><a section="4934590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01504.png?Expires=2147483647&Signature=Rq~tEMPy6N7LcOMgvo3EoMPEjNQun2HZ1MeYmirqM8fU22Ea2FTzmWN021uYp9KzEBkXFBcDE2xh1UPL263ldStSBRvkIb8U9FM9lfyCoZ~Yn48HUthfh-IHjzi4YeLBPIoC8mWr-MfSqBoEIhebuA2PmwK561Yp~aFZ2i6Ts~Xe4d-RYmlJVqiEGqKzJnsIBQWrBrpdElXKhzcGbhkXWv6EaXVRPaIENlrlm7zOVFMR6jXZUi~0dZ190w4mQAQdZgCFY4gMObL~rh7RjgpJdXkZ9aOJM-EQ25rYyDOCo2BSUu~VnUOJaWYAahPY6AMDqCByWcmptP9WKEyqjWsdYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934590&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tiling Assembly identified over 2.5 times more novel genes than Cufflinks. Novel genes identified by Tiling Assembly and Cufflinks were compared to determine how much they overlapped. (A) When all novel genes were considered, 7,460 genes were exclusively identified by Tiling Assembly, 1,316 by Cufflinks, and 2,669 by both. (B) When only novel genes with high expression levels (≥100 RPKE) were considered, 3,473 genes were exclusively identified by Tiling Assembly, 52 by Cufflinks, and 1,166 by both.</p></div></div>                            <h3 scrollto-destination=4934594 id="4934594" class="section-title" data-legacy-id=s3f>3.6. BLAST searches were used to eliminate potential novel genes with high sequence similarity to other genomic regions, resulting in 767 high-confidence, unannotated novel genes</h3>
<p>During read alignment, if a read can map to multiple locations within a genome, the read is randomly assigned to one of the locations.<sup><a reveal-id="DSV015C1" data-open="DSV015C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV015C24" data-open="DSV015C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Because of this, potential novel genes that have a high similarity to another region in the rice genome may be false-positive genes. In addition, the rice genome contains a number of transcriptionally active gene fragments with high levels of sequence identity to annotated protein-coding genes and genes which may be involved in regulation of those genes rather than functioning as protein-coding genes themselves.<sup><a reveal-id="DSV015C25" data-open="DSV015C25" class="link link-ref link-reveal xref-bibr">25</a></sup> BLAST<sup><a reveal-id="DSV015C26" data-open="DSV015C26" class="link link-ref link-reveal xref-bibr">26</a></sup> searches revealed that 774 genes showed &lt;25% sequence similarity to another region within the genome. Of these genes, seven genes had a footprint of &lt;140 nt and were filtered out, bringing the total number of potential novel genes with &lt;25% similarity to 767 genes (Fig. <a reveal-id="DSV015F5" data-open="DSV015F5" class="link link-reveal link-table xref-fig">5</a>). These 767 genes were considered to be high-confidence novel genes based on the following criteria: they were unannotated, highly expressed, and contained relatively unique sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Of these high-confidence novel genes, 151 genes were uniquely identified by Tiling Assembly and 26 by Cufflinks. The remaining 590 genes were identified by both. Tiling Assembly was not only capable of finding 97% of the high-confidence novel genes found by Cufflinks, but it also found an additional 151 genes. </p>                        <a id="4934599" scrollto-destination="4934599"></a>
<div id="DSV015F5" data-id="DSV015F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01505.png?Expires=2147483647&Signature=iYc3IITkFea~q1BgL3ztgkjuJgaQ9iBmIxOwJuqa1igP97fyznONZaLvUADGBiwnD6fEeEdT~YLFK~NIHHkh109yWFkEnhsPNRitAQoXRExOHPO70HAe7gYNJo~2WLuC9Nb97A8pWqr0aTwMnOxzaMrb4QrkJcrGUhblskvjCKcW3eLq3TpD10-TWBdHOcFK1NBkkU5cXTyPqYclVrSarWpqYqeGTmQBXe9XtDTeGTY7NkvvSuuYPS-6z5Gd0J~0LvW38rr1CmZIFmurfAZ1zs2FeV0Akb0GViKBWnW40c5idRb9pd33mRe~fEUmAdZi1t8P2GwYIsLf4Q3CTkNSqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tiling Assembly identified more novel genes than Cufflinks regardless of similarity to another genomic region. BLAST searches determined the level of similarity of the potential novel genes to other regions of the rice genome." path-from-xml="dsv01505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934599/dsv01505.tiff" path-from-xml="dsv01505.tiff" target="_blank">View large</a><a section="4934599" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01505.png?Expires=2147483647&Signature=3FGKfZQZPOhdA4BI1dgKkjb4Qh4NrjpH10Atby3p62U6UpCHL8gZ18C57NRSHEJeXFLCqxLT62lzpyzyVyzVAmXQUrxXU5aJ8UJL7KtKHKEPoj0TjVUpADwH61FV~sQy2EvfDHO4chdDZXBQLO6xxZeTR09bAo6Ps3w~vL7iLfE0eJbXWsXXcxqbbvD~~3Q5x3zROrA72bs-Ft8S4aklldiSCNT7B5lMCZ3CFRM3aC1mMwEzE33Mr8h8vqspb7Q8mBng9k-YOktbPtjPjzvXFLAJOEquMiyYgcoPLlrERzf0RkxZNd4jNjumoJW4uYQYVE5Yr9Ji8s2nELFC0UXoNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934599&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tiling Assembly identified more novel genes than Cufflinks regardless of similarity to another genomic region. BLAST searches determined the level of similarity of the potential novel genes to other regions of the rice genome.</p></div></div><div content-id="DSV015F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01505.png?Expires=2147483647&Signature=iYc3IITkFea~q1BgL3ztgkjuJgaQ9iBmIxOwJuqa1igP97fyznONZaLvUADGBiwnD6fEeEdT~YLFK~NIHHkh109yWFkEnhsPNRitAQoXRExOHPO70HAe7gYNJo~2WLuC9Nb97A8pWqr0aTwMnOxzaMrb4QrkJcrGUhblskvjCKcW3eLq3TpD10-TWBdHOcFK1NBkkU5cXTyPqYclVrSarWpqYqeGTmQBXe9XtDTeGTY7NkvvSuuYPS-6z5Gd0J~0LvW38rr1CmZIFmurfAZ1zs2FeV0Akb0GViKBWnW40c5idRb9pd33mRe~fEUmAdZi1t8P2GwYIsLf4Q3CTkNSqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tiling Assembly identified more novel genes than Cufflinks regardless of similarity to another genomic region. BLAST searches determined the level of similarity of the potential novel genes to other regions of the rice genome." path-from-xml="dsv01505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934599/dsv01505.tiff" path-from-xml="dsv01505.tiff" target="_blank">View large</a><a section="4934599" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01505.png?Expires=2147483647&Signature=3FGKfZQZPOhdA4BI1dgKkjb4Qh4NrjpH10Atby3p62U6UpCHL8gZ18C57NRSHEJeXFLCqxLT62lzpyzyVyzVAmXQUrxXU5aJ8UJL7KtKHKEPoj0TjVUpADwH61FV~sQy2EvfDHO4chdDZXBQLO6xxZeTR09bAo6Ps3w~vL7iLfE0eJbXWsXXcxqbbvD~~3Q5x3zROrA72bs-Ft8S4aklldiSCNT7B5lMCZ3CFRM3aC1mMwEzE33Mr8h8vqspb7Q8mBng9k-YOktbPtjPjzvXFLAJOEquMiyYgcoPLlrERzf0RkxZNd4jNjumoJW4uYQYVE5Yr9Ji8s2nELFC0UXoNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934599&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tiling Assembly identified more novel genes than Cufflinks regardless of similarity to another genomic region. BLAST searches determined the level of similarity of the potential novel genes to other regions of the rice genome.</p></div></div>                            <h3 scrollto-destination=4934602 id="4934602" class="section-title" data-legacy-id=s3g>3.7. Comparison to previously published results</h3>
<p>In our previous publication, we identified 553 novel genes using a combination of Cufflinks and a custom Clustering Algorithm.<sup><a reveal-id="DSV015C9" data-open="DSV015C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Clustering Algorithm was developed to identify novel genes based on the presence of reads aligning to unannotated regions of the rice genome. Comparing the 767 potential novel genes identified by Tiling Assembly and Cufflinks with the 553 novel genes identified in our previous publication, there were 461 genes that coincided (Fig. <a reveal-id="DSV015F6" data-open="DSV015F6" class="link link-reveal link-table xref-fig">6</a>A). There were 306 genes that were not identified in our previous publication. These additional 306 genes demonstrate that our new pipeline is superior to that previously reported. </p>                        <a id="4934607" scrollto-destination="4934607"></a>
<div id="DSV015F6" data-id="DSV015F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01506.png?Expires=2147483647&Signature=jQzuYfvumyzap39jv4TcII8pelVqkhTqnhz2OcBYZUbqJHERVidAbxju4O2rcBJBAKenVP46Ptrq~BQxTPTFuMOY5LuQGJEgKFUKZkIF4oBsgOErM8OTxuWX-7o-a0k26KfOKzG-HRULZRIYa2NEokfQ2X5IlJZTQXGMBUvbEOfbo4JbFWUXqqioroiRH2jO~lPcM0puD44-V0kU5jDiQRU4zqmHuB0EuF0ucleQuGF1UdrNxBkOivyw2GVFn17tZiYepcdwuLe9nFn4aMSuWzr3Ir7O06NZ9jdl1ll-u-OOtXtk~0ItxbEjiNlnva7JWhxEsV~aeEJSc-9-tcWIpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of Tiling Assembly and Cufflinks identified potential novel genes with those published in our previous study. (A) There were 92 genes identified in our previous study that were not classified as novel genes by Tiling Assembly or Cufflinks. While all of them were identified by Tiling Assembly or Cufflinks, slight changes in their length disqualified them from fitting into the category of potential novel genes. Of the 767 unannotated genes identified in this study, 306 genes were not identified in our previous study, demonstrating that our new pipeline is superior to that previously reported. (B) While all of the Clustering Algorithm genes were also found by Tiling Assembly, slight changes in their identification disqualified five from fitting into the category of potential novel genes." path-from-xml="dsv01506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934607/dsv01506.tiff" path-from-xml="dsv01506.tiff" target="_blank">View large</a><a section="4934607" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01506.png?Expires=2147483647&Signature=KgGfY3OpdqyuOwOHLLJCebEoOfYF7nt4NWdXM57fOmRLNMXi5S42S20mVmk6X9ElVqVPFBGaJAuD31i~vYT9NuPMz85yZRMf6cKz9JaDn7RLGDLcrpL8SV9tQ~SEjFcZP15ZIn~H1L83E9SxlrxPM4oDyGpWCMHargRbFtdRQASjGCLxP2tAv9xigSQI1WBLGrJ2FLgkfLu7d4XWOBuA2tJG6~6OY9BAffUulmVzWKiGIZ7ATrvpB15hEteqEOUdedjx3TFbzfH4e-EQrEKXrzmrCb-Qnhlk42JQ7xuhalvaytC3aeBf~Ff0gvl2FyZuLvblQ66yMd6szv0585wT5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934607&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of Tiling Assembly and Cufflinks identified potential novel genes with those published in our previous study. (A) There were 92 genes identified in our previous study that were not classified as novel genes by Tiling Assembly or Cufflinks. While all of them were identified by Tiling Assembly or Cufflinks, slight changes in their length disqualified them from fitting into the category of potential novel genes. Of the 767 unannotated genes identified in this study, 306 genes were not identified in our previous study, demonstrating that our new pipeline is superior to that previously reported. (B) While all of the Clustering Algorithm genes were also found by Tiling Assembly, slight changes in their identification disqualified five from fitting into the category of potential novel genes.</p></div></div><div content-id="DSV015F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01506.png?Expires=2147483647&Signature=jQzuYfvumyzap39jv4TcII8pelVqkhTqnhz2OcBYZUbqJHERVidAbxju4O2rcBJBAKenVP46Ptrq~BQxTPTFuMOY5LuQGJEgKFUKZkIF4oBsgOErM8OTxuWX-7o-a0k26KfOKzG-HRULZRIYa2NEokfQ2X5IlJZTQXGMBUvbEOfbo4JbFWUXqqioroiRH2jO~lPcM0puD44-V0kU5jDiQRU4zqmHuB0EuF0ucleQuGF1UdrNxBkOivyw2GVFn17tZiYepcdwuLe9nFn4aMSuWzr3Ir7O06NZ9jdl1ll-u-OOtXtk~0ItxbEjiNlnva7JWhxEsV~aeEJSc-9-tcWIpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of Tiling Assembly and Cufflinks identified potential novel genes with those published in our previous study. (A) There were 92 genes identified in our previous study that were not classified as novel genes by Tiling Assembly or Cufflinks. While all of them were identified by Tiling Assembly or Cufflinks, slight changes in their length disqualified them from fitting into the category of potential novel genes. Of the 767 unannotated genes identified in this study, 306 genes were not identified in our previous study, demonstrating that our new pipeline is superior to that previously reported. (B) While all of the Clustering Algorithm genes were also found by Tiling Assembly, slight changes in their identification disqualified five from fitting into the category of potential novel genes." path-from-xml="dsv01506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934607/dsv01506.tiff" path-from-xml="dsv01506.tiff" target="_blank">View large</a><a section="4934607" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01506.png?Expires=2147483647&Signature=KgGfY3OpdqyuOwOHLLJCebEoOfYF7nt4NWdXM57fOmRLNMXi5S42S20mVmk6X9ElVqVPFBGaJAuD31i~vYT9NuPMz85yZRMf6cKz9JaDn7RLGDLcrpL8SV9tQ~SEjFcZP15ZIn~H1L83E9SxlrxPM4oDyGpWCMHargRbFtdRQASjGCLxP2tAv9xigSQI1WBLGrJ2FLgkfLu7d4XWOBuA2tJG6~6OY9BAffUulmVzWKiGIZ7ATrvpB15hEteqEOUdedjx3TFbzfH4e-EQrEKXrzmrCb-Qnhlk42JQ7xuhalvaytC3aeBf~Ff0gvl2FyZuLvblQ66yMd6szv0585wT5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934607&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of Tiling Assembly and Cufflinks identified potential novel genes with those published in our previous study. (A) There were 92 genes identified in our previous study that were not classified as novel genes by Tiling Assembly or Cufflinks. While all of them were identified by Tiling Assembly or Cufflinks, slight changes in their length disqualified them from fitting into the category of potential novel genes. Of the 767 unannotated genes identified in this study, 306 genes were not identified in our previous study, demonstrating that our new pipeline is superior to that previously reported. (B) While all of the Clustering Algorithm genes were also found by Tiling Assembly, slight changes in their identification disqualified five from fitting into the category of potential novel genes.</p></div></div><p>There were 92 genes identified in our previous publication that were not considered as novel genes in this study. Most of these 92 genes were identified by Tiling Assembly in the initial steps but, due to slight differences in gene length, were filtered out as a result of low RPKE, high similarity to another region of the genome or overlap with an annotated gene. In addition, an older version of the rice genome and older versions of Cufflinks, Tophat, and Bowtie was used to identify genes in our previous publication. These factors resulted in a difference in the number of genes identified by Cufflinks in our previous publication, when compared with those reported in this study.</p><p>Of the 553 genes identified in our previous study, 124 were identified by Clustering Algorithm. These Clustering Algorithm genes were compared with Tiling Assembly and Cufflinks to determine the efficiencies of the algorithms (Fig. <a reveal-id="DSV015F6" data-open="DSV015F6" class="link link-reveal link-table xref-fig">6</a>B). Of the Clustering Algorithm genes, 112 genes were found by both Tiling Assembly and Cufflinks. There were five Clustering Algorithm genes that were not identified by Tiling Assembly or Cufflinks.</p><p>Upon closer inspection of these five genes, it appeared Tiling Assembly correctly identified genes within the same location; however, there were differences in how the genes were identified. Two genes identified by Clustering Algorithm were associated with previously annotated genes by Tiling Assembly and were not considered novel genes. One Clustering Algorithm gene was identified as two genes by Tiling Assembly, each of which was eliminated as a potential novel gene based on similarity to another genomic region. Slight differences in the boundaries of the remaining two genes, as identified by Tiling Assembly, resulted in shorter genes. This resulted in an increased percent similarity to another genomic region, thus eliminating them as potential novel genes. Though these five genes may be potential novel genes, they do not satisfy our requirements for consideration as high-confidence potential novel genes.</p>                            <h3 scrollto-destination=4934616 id="4934616" class="section-title" data-legacy-id=s3h>3.8. Open reading frame identification</h3>
<p>Many regions of the genome are actively transcribed, but do not produce protein products. To determine whether Tiling Assembly and Cufflinks identified novel protein-coding genes, the introns were removed and the longest possible open reading frame (ORF) associated with each of the 767 potential novel genes was determined. The predicted peptide lengths ranged from 23 to 4,737 codons. The average ORF length was 155 codons and more than half of the genes were 80–160 codons in length (Fig. <a reveal-id="DSV015F7" data-open="DSV015F7" class="link link-reveal link-table xref-fig">7</a>). Since random DNA sequences are statistically unlikely to be more than 50 codons long without containing a stop codon,<sup><a reveal-id="DSV015C27" data-open="DSV015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> the fact that most of the ORFs found code for longer sequences indicates that they are likely protein-coding genes. Though proteins as small as 20 amino acids have been discovered in other organisms, they are uncommon,<sup><a reveal-id="DSV015C28" data-open="DSV015C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and no ORFs with fewer than 23 codons were found in our data set. Only 14 novel genes (1.8%) had predicted ORFs &lt;40 codons and are probably not protein-coding genes. These genes may encode micro-RNAs or other non-coding RNAs.<sup><a reveal-id="DSV015C28" data-open="DSV015C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV015C29" data-open="DSV015C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In addition, during the development of MSU R7, a 50 codon threshold was used. These data indicate that the majority of high-confidence novel genes identified by Tiling Assembly and Cufflinks are likely protein-coding genes, and it is not likely that they are genes that merely failed to meet the MSU R7 50 codon threshold. </p>                        <a id="4934621" scrollto-destination="4934621"></a>
<div id="DSV015F7" data-id="DSV015F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01507.png?Expires=2147483647&Signature=25R9bbUMKqklf7Rc~2ZtanlwJlrVgoO27ECBeZybCxIQWK8FG3kbKbIwwqfL7fVRb-qDGoPFHvgWddj2dVqRkBhx6IRDGV5Q29jACsF8ruKl1TDjCwpUtRVFyegdfaD5M6H-ad1vFwZ3wc6sxvPum6T8PdGeg1fiiZ2zYIHiq-9FUjjlXkMsSksGFHL8V6JVelx3xnAOfxNldn4PkqEqtTM32TglvlEM7YANaPS8rPCJoViOcNlQGqPc8UuK4mCEdQp~gdvUs~nwnIn~CQAnWosni9SQGubkrE61NiYtXl6E7NKBi4zCBhdeGM1R0Vsr7nGuZ5w3wlYwgVZ3o6ySsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The peptide length distribution of potential novel genes is similar to that of annotated genes. The longest ORF was determined for each of the genes using an internally developed programme. The ORFs ranged from 23 to 4,737 codons in length, with an average length of 155 codons." path-from-xml="dsv01507.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934621/dsv01507.tiff" path-from-xml="dsv01507.tiff" target="_blank">View large</a><a section="4934621" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01507.png?Expires=2147483647&Signature=Z6e7Ld0NTjy4YTiVkVjjwo9dO68FvVc1e7p1O6qwlhJXBqoVlq9mPxp-T6WMP~zHa9hoFyv307F0d2YzKqwzi-diGB9bItc-N6rECyYttlDiCgy3JNhwEpAJc0xyyis8RpPmO7RVCBSj7HGoGlxyNRKLkcU29cIaN0f--EvdsuY~NN5BnAOiBnSbx63mzTqNpjDk~luJ~2TT4EqWx0EBoMBMBwH8GWyBzTeWhcimw-UO-6~Z5PH9NL8AIn2GGJO-UB6ri8eukeR1r7Gq1KlWtCL7FcQV8rSJ7pYk2uHyDi78Wcy5x0X8QPLdajDVLtABrEN10JHzreAQS22WDX4LKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934621&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01507.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The peptide length distribution of potential novel genes is similar to that of annotated genes. The longest ORF was determined for each of the genes using an internally developed programme. The ORFs ranged from 23 to 4,737 codons in length, with an average length of 155 codons.</p></div></div><div content-id="DSV015F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01507.png?Expires=2147483647&Signature=25R9bbUMKqklf7Rc~2ZtanlwJlrVgoO27ECBeZybCxIQWK8FG3kbKbIwwqfL7fVRb-qDGoPFHvgWddj2dVqRkBhx6IRDGV5Q29jACsF8ruKl1TDjCwpUtRVFyegdfaD5M6H-ad1vFwZ3wc6sxvPum6T8PdGeg1fiiZ2zYIHiq-9FUjjlXkMsSksGFHL8V6JVelx3xnAOfxNldn4PkqEqtTM32TglvlEM7YANaPS8rPCJoViOcNlQGqPc8UuK4mCEdQp~gdvUs~nwnIn~CQAnWosni9SQGubkrE61NiYtXl6E7NKBi4zCBhdeGM1R0Vsr7nGuZ5w3wlYwgVZ3o6ySsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The peptide length distribution of potential novel genes is similar to that of annotated genes. The longest ORF was determined for each of the genes using an internally developed programme. The ORFs ranged from 23 to 4,737 codons in length, with an average length of 155 codons." path-from-xml="dsv01507.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934621/dsv01507.tiff" path-from-xml="dsv01507.tiff" target="_blank">View large</a><a section="4934621" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01507.png?Expires=2147483647&Signature=Z6e7Ld0NTjy4YTiVkVjjwo9dO68FvVc1e7p1O6qwlhJXBqoVlq9mPxp-T6WMP~zHa9hoFyv307F0d2YzKqwzi-diGB9bItc-N6rECyYttlDiCgy3JNhwEpAJc0xyyis8RpPmO7RVCBSj7HGoGlxyNRKLkcU29cIaN0f--EvdsuY~NN5BnAOiBnSbx63mzTqNpjDk~luJ~2TT4EqWx0EBoMBMBwH8GWyBzTeWhcimw-UO-6~Z5PH9NL8AIn2GGJO-UB6ri8eukeR1r7Gq1KlWtCL7FcQV8rSJ7pYk2uHyDi78Wcy5x0X8QPLdajDVLtABrEN10JHzreAQS22WDX4LKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934621&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01507.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The peptide length distribution of potential novel genes is similar to that of annotated genes. The longest ORF was determined for each of the genes using an internally developed programme. The ORFs ranged from 23 to 4,737 codons in length, with an average length of 155 codons.</p></div></div><p>The 767 potential novel genes identified by Tiling Assembly were further analysed by performing a protein BLAST on each gene to determine whether they exhibited any sequence homology to known proteins. There were 641 genes that showed some level of sequence homology, with 99 genes having an <em>E</em>-value ≤0.0001. Of these, 97 genes were homologous to predicted proteins, one to a hypothetical protein, and one to a bacterial heat-shock protein. The remaining 126 genes did not exhibit any sequence homology to known proteins. These genes may encode lincRNAs, or other long non-coding RNAs.<sup><a reveal-id="DSV015C29" data-open="DSV015C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h3 scrollto-destination=4934626 id="4934626" class="section-title" data-legacy-id=s3i>3.9. Comparing Tiling Assembly and Cufflinks genes to FL-cDNAs</h3>
<p>MSU R7 compiles annotation data from multiple sources, including FL-cDNA sequences, ESTs, and gene prediction software.<sup><a reveal-id="DSV015C13" data-open="DSV015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> As such, many of the annotated genes are hypothetical and not known to be expressed. To further evaluate the accuracy of Tiling Assembly in the identification of expressed genes, the genes identified by Tiling Assembly were compared with over 28,000 published FL-cDNAs, collected and sequenced by Kikuchi et al.<sup><a reveal-id="DSV015C18" data-open="DSV015C18" class="link link-ref link-reveal xref-bibr">18</a></sup> These FL-cDNAs represent mRNA transcripts obtained from the rice plant and are thus more reliable than the computationally predicted transcripts in MSU R7. Of the 26,302 genes identified by Tiling Assembly with an expression level of at least 100 RPKE, 7,104 overlapped with a published FL-cDNA by &gt;90% of their sequences. If multiple FL-cDNA variants overlapped with the same Tiling Assembly gene, the FL-cDNA variant with the same number of exons as the Tiling Assembly gene was selected for comparison. There were 5,767 genes that matched in exon number with their corresponding FL-cDNAs. The remaining 1,337 genes (18.8%) are here referred to as discrepant genes. To determine the source of these discrepancies between Tiling Assembly and the FL-cDNA, seven different categories of classification were used: extra exon, missing exon, extra intron, missing intron, missing junction, gap, or multiple discrepancies. In the instance Tiling Assembly recognized an exon where the corresponding FL-cDNA did not, the discrepancy was categorized as an extra exon (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S11A</a>). In the instance Tiling Assembly did not identify an exon where the corresponding FL-cDNA did, it was categorized as a missing exon (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S11B</a>). In the instance Tiling Assembly recognized an intron within the corresponding exon of the FL-cDNA, it was categorized as an extra intron (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S11C</a>). In the instance Tiling Assembly recognized a single exon where the corresponding FL-cDNA recognized two exons, it was categorized to be a missing intron (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S11D</a>).</p><p>The analysis was also performed on the Cufflinks genes. Of the 26,876 genes identified by Cufflinks that had an expression level of at least 100 RPKE, 7,690 overlapped with a published FL-cDNA by &gt;90% of their sequences. Of these, 5,970 genes matched in exon number with their corresponding FL-cDNA and the remaining 1,720 genes (22.4%) were considered discrepant genes. Though Cufflinks identified more matching genes, the percentage and the number of discrepant genes were greater than Tiling Assembly.</p><p>Because it is time-consuming to analyse the causes of discrepancies for the 1,337 Tiling Assembly genes and the 1,720 Cufflinks genes, a portion of the gene pool was sampled for detailed analysis. The appropriate sample size for this comparison was calculated to be 290 genes, based on a 95% confidence level and a 5% margin of error (see Section 2.5.), and was rounded up to 300 genes. FL-cDNAs that had both a corresponding Tiling Assembly and Cufflinks gene that were discrepant were chosen for manual analysis. If a gene exhibited numerous reads, but showed a difference in the number of exons between the two data sets, alternative splicing was considered a possible cause. More than 60% of multi-exonic genes in plants are alternatively spliced,<sup><a reveal-id="DSV015C30" data-open="DSV015C30" class="link link-ref link-reveal xref-bibr">30</a></sup> with intron retention being the most common form of alternative splicing.<sup><a reveal-id="DSV015C31" data-open="DSV015C31" class="link link-ref link-reveal xref-bibr">31</a></sup> It is unlikely that the FL-cDNA dataset<sup><a reveal-id="DSV015C18" data-open="DSV015C18" class="link link-ref link-reveal xref-bibr">18</a></sup> contains all alternative splice variants of transcripts. In addition, Tiling Assembly was designed to identify a single splice variant. Therefore, it was expected that the majority of the discrepancies may be due to alternative splicing. Indeed, of the 300 discrepant Tiling Assembly genes analysed, 96.7% could be attributed to alternative splicing. Extra or missing introns were the most abundant cause of the discrepancy as would be expected for plants. The remaining 3.3% were attributed to missing junctions or gaps (Fig. <a reveal-id="DSV015F8" data-open="DSV015F8" class="link link-reveal link-table xref-fig">8</a>). Similar results were obtained for the Cufflinks dataset, with 96.3% due to possible alternative splicing and the remaining 3.7% attributed to missing junctions or gaps. </p>                        <a id="4934635" scrollto-destination="4934635"></a>
<div id="DSV015F8" data-id="DSV015F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01508.png?Expires=2147483647&Signature=xdbbpHq3GN2j-p~ThADl7ZrwFyG25t7uMiW8ZBfLceKegeYlLtk-0SbwnX4bccnaARMUPv6gm7sce6O5cbtvRxQfTgKtv0QzPAuHb2ut628MmFZfEH0ZwskXOs9t9m~W6bJzSWoeZ3Uy8FfLLBhpme1Hx6Ij3W6-kM~S3uR1jC-gNqUD4pJipQSOMSpQoVhdXBI3kkWvJfC4h4KGDmvF2PZASPt-x-IyMLM2M0Nn~yrwbJ6cXuUyEKxq8u2P3KUZtRZdO0oMMaYIIG-hxLHZnXWXx8Rn3sG~pNQU~DFGamf8YfjE8hWmXGospY42hPrKXF1bwWgt2Vj4XyPLOyq2mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The major cause of discrepancies between Tiling Assembly and Cufflinks and their corresponding FL-cDNAs were due to extra or missing introns. A sample of 300 FL-cDNAs that had both corresponding Tiling Assembly and Cufflinks transcripts were analysed to determine the major causes of discrepancies. Extra intron and missing intron were the major causes of discrepancies for both Cufflinks and Tiling Assembly. Of the Tiling Assembly transcripts, 96.7% of the discrepancies may be due to alternative splicing that resulted in extra introns, missing introns, extra exons, missing exons, or multiple discrepancies (shown in brackets in the above image). Of the Cufflinks transcripts, 96.3% of the discrepancies may be due to alternative splicing." path-from-xml="dsv01508.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934635/dsv01508.tiff" path-from-xml="dsv01508.tiff" target="_blank">View large</a><a section="4934635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01508.png?Expires=2147483647&Signature=aDLAvCvWSi29Xk5ahGTBewHHjomcVflfyQmbhD1Prfd5DCOwZhL0IRtd-mwA4UX-8OqocEN~lcmeWwhDYdxZ9qYhuxbkQonqmcbOylZTgUtxlwu0SwJe8r8-BjavL8y35rG1c4IiCIc2SjSqK6FXjobZ0xBbIqk8RzKoZiF-qOWRlbA37w~FpXROpl-hdddWbHGR-mHigsBlrNzdjcyPCw6WbrMlA4428Qv3bkYVatTHlKVmYS1MBhfCvM5qxT6NQs5a7h-NNRYRgkjDLZ79FoNyfMCMk8ZWqGfxV4sdplkCc60HeGFicTzACiWatw4VICASi9E02f4CFHFqZMUtYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934635&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01508.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The major cause of discrepancies between Tiling Assembly and Cufflinks and their corresponding FL-cDNAs were due to extra or missing introns. A sample of 300 FL-cDNAs that had both corresponding Tiling Assembly and Cufflinks transcripts were analysed to determine the major causes of discrepancies. Extra intron and missing intron were the major causes of discrepancies for both Cufflinks and Tiling Assembly. Of the Tiling Assembly transcripts, 96.7% of the discrepancies may be due to alternative splicing that resulted in extra introns, missing introns, extra exons, missing exons, or multiple discrepancies (shown in brackets in the above image). Of the Cufflinks transcripts, 96.3% of the discrepancies may be due to alternative splicing.</p></div></div><div content-id="DSV015F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01508.png?Expires=2147483647&Signature=xdbbpHq3GN2j-p~ThADl7ZrwFyG25t7uMiW8ZBfLceKegeYlLtk-0SbwnX4bccnaARMUPv6gm7sce6O5cbtvRxQfTgKtv0QzPAuHb2ut628MmFZfEH0ZwskXOs9t9m~W6bJzSWoeZ3Uy8FfLLBhpme1Hx6Ij3W6-kM~S3uR1jC-gNqUD4pJipQSOMSpQoVhdXBI3kkWvJfC4h4KGDmvF2PZASPt-x-IyMLM2M0Nn~yrwbJ6cXuUyEKxq8u2P3KUZtRZdO0oMMaYIIG-hxLHZnXWXx8Rn3sG~pNQU~DFGamf8YfjE8hWmXGospY42hPrKXF1bwWgt2Vj4XyPLOyq2mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The major cause of discrepancies between Tiling Assembly and Cufflinks and their corresponding FL-cDNAs were due to extra or missing introns. A sample of 300 FL-cDNAs that had both corresponding Tiling Assembly and Cufflinks transcripts were analysed to determine the major causes of discrepancies. Extra intron and missing intron were the major causes of discrepancies for both Cufflinks and Tiling Assembly. Of the Tiling Assembly transcripts, 96.7% of the discrepancies may be due to alternative splicing that resulted in extra introns, missing introns, extra exons, missing exons, or multiple discrepancies (shown in brackets in the above image). Of the Cufflinks transcripts, 96.3% of the discrepancies may be due to alternative splicing." path-from-xml="dsv01508.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934635/dsv01508.tiff" path-from-xml="dsv01508.tiff" target="_blank">View large</a><a section="4934635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01508.png?Expires=2147483647&Signature=aDLAvCvWSi29Xk5ahGTBewHHjomcVflfyQmbhD1Prfd5DCOwZhL0IRtd-mwA4UX-8OqocEN~lcmeWwhDYdxZ9qYhuxbkQonqmcbOylZTgUtxlwu0SwJe8r8-BjavL8y35rG1c4IiCIc2SjSqK6FXjobZ0xBbIqk8RzKoZiF-qOWRlbA37w~FpXROpl-hdddWbHGR-mHigsBlrNzdjcyPCw6WbrMlA4428Qv3bkYVatTHlKVmYS1MBhfCvM5qxT6NQs5a7h-NNRYRgkjDLZ79FoNyfMCMk8ZWqGfxV4sdplkCc60HeGFicTzACiWatw4VICASi9E02f4CFHFqZMUtYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934635&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01508.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The major cause of discrepancies between Tiling Assembly and Cufflinks and their corresponding FL-cDNAs were due to extra or missing introns. A sample of 300 FL-cDNAs that had both corresponding Tiling Assembly and Cufflinks transcripts were analysed to determine the major causes of discrepancies. Extra intron and missing intron were the major causes of discrepancies for both Cufflinks and Tiling Assembly. Of the Tiling Assembly transcripts, 96.7% of the discrepancies may be due to alternative splicing that resulted in extra introns, missing introns, extra exons, missing exons, or multiple discrepancies (shown in brackets in the above image). Of the Cufflinks transcripts, 96.3% of the discrepancies may be due to alternative splicing.</p></div></div><p>Applying the results from the sample of 300 discrepant Tiling Assembly genes, out of the 1,337 discrepant genes, it was expected that about 1,293 discrepancies (96.7%) may be due to alternative splicing events. The remaining 44 genes (3.3%) were expected to be the result of missing junctions, gaps, or other discrepancies. These 44 genes represented 0.6% of the 7,104 genes that overlapped with the FL-cDNAs. Therefore, it was expected that Tiling Assembly may be as high as 99.4% accurate in the identification of genes with at least 90% overlap.</p><p>Appling the results from the sample of 300 discrepant Cufflinks genes, out of the 1,720 discrepant genes, it was expected that 1,656 discrepancies were possible alternative splicing events. The remaining 64 genes were expected to be the result of missing junctions, gaps, or other discrepancies. These 64 genes represented 0.8% of the 7,690 genes that overlapped with the FL-cDNAs. Therefore, it was expected that Cufflinks may be as high as 99.2% accurate in the identification of genes with at least 90% overlap.</p>                            <h3 scrollto-destination=4934643 id="4934643" class="section-title" data-legacy-id=s3j>3.10. Identification of transcription start and termination sites by Tiling Assembly</h3>
<p>The transcription start and termination sites were compared between genes identified by Tiling Assembly with those identified by the FL-cDNAs (Fig. <a reveal-id="DSV015F9" data-open="DSV015F9" class="link link-reveal link-table xref-fig">9</a>). Only Tiling Assembly genes that overlapped with an FL-cDNA by at least 90% were considered. Of the 7,174 transcription start sites that satisfied the specified overlap threshold, ∼83% differed by ≤100 nt (Fig. <a reveal-id="DSV015F9" data-open="DSV015F9" class="link link-reveal link-table xref-fig">9</a>A). The transcription start sites predicted by Tiling Assembly were on average 30 nt upstream of the FL-cDNAs. These data demonstrated that Tiling Assembly is a reasonably reliable tool for the identification of transcription start sites. </p>                        <a id="4934646" scrollto-destination="4934646"></a>
<div id="DSV015F9" data-id="DSV015F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01509.png?Expires=2147483647&Signature=1gDgwcicK0suOBMGQzp0v5uXx7i1HwGdVE8y8UkSJ4ut0ynVeB3waMilDp3RbDaUnkriE97J-GJZdZjTTArNOHFV2xw8oyFXNyKQLL3Pm92EvTL-DPw1ztGo3J~i2iQ0sxB5m1S0Xz9bw-8-Dune0srsl1GGMKoKOjxfqfpuKRhNGHxLpU9gXaEisvGeArr7NuEQgK7md671eA35XP~6eEeEcaPO339R4eT5gFs62Cz3BQ7ANZ2X7h2-Rw7W4nkovj0G9qNsioAcTaFiqjHPH8aVXlUi2TBTX9YmHA1yZg02tg5NgVZY-mOE8y~z5SeqtSzHViY2RfgKY161GxPbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nucleotide differences between transcriptional start (A) and termination (B) sites of FL-cDNAs and Tiling Assembly genes. The transcriptional start and termination sites were compared with those of FL-cDNAs previously published. The distribution of the difference between the Tiling Assembly and FL-cDNA start and stop sites is presented here. A negative value indicates that the Tiling Assembly gene is shorter than the FL-cDNA and a positive value indicates that the Tiling Assembly gene is longer." path-from-xml="dsv01509.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934646/dsv01509.tiff" path-from-xml="dsv01509.tiff" target="_blank">View large</a><a section="4934646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01509.png?Expires=2147483647&Signature=W9oX6rzA3PwDvHz64kQdJOL3PPDa4Uhmzki8bIuwhyswUFpitBO1mTqqy4baRaZTGEKbRn9FPeQ6ASmOgT2UgTxKrqQsVslOt2Uc~-WwXdXzdpOubTM12OTh2x42Tnmsenr2I6Ihkz37775s1TT6d1ghHk3MeeO4MCx1DTBBFNFcFQOUhuUQA6hGB~x1bge9-UdXOD5C~rND3mCNnly24pFgxBZA2UCYSAtAByPByyjFKFtXLW33B7W8RhWXKuMqpmgcKIr3CcDjRmtLMVk3M0XR-j8HN~QZCgtnIIsXhb36T3nmpjrZRkZUaaPeOclTwKWWCMw~tfUH7b5XNPV7OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934646&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01509.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nucleotide differences between transcriptional start (A) and termination (B) sites of FL-cDNAs and Tiling Assembly genes. The transcriptional start and termination sites were compared with those of FL-cDNAs previously published. The distribution of the difference between the Tiling Assembly and FL-cDNA start and stop sites is presented here. A negative value indicates that the Tiling Assembly gene is shorter than the FL-cDNA and a positive value indicates that the Tiling Assembly gene is longer.</p></div></div><div content-id="DSV015F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/m_dsv01509.png?Expires=2147483647&Signature=1gDgwcicK0suOBMGQzp0v5uXx7i1HwGdVE8y8UkSJ4ut0ynVeB3waMilDp3RbDaUnkriE97J-GJZdZjTTArNOHFV2xw8oyFXNyKQLL3Pm92EvTL-DPw1ztGo3J~i2iQ0sxB5m1S0Xz9bw-8-Dune0srsl1GGMKoKOjxfqfpuKRhNGHxLpU9gXaEisvGeArr7NuEQgK7md671eA35XP~6eEeEcaPO339R4eT5gFs62Cz3BQ7ANZ2X7h2-Rw7W4nkovj0G9qNsioAcTaFiqjHPH8aVXlUi2TBTX9YmHA1yZg02tg5NgVZY-mOE8y~z5SeqtSzHViY2RfgKY161GxPbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nucleotide differences between transcriptional start (A) and termination (B) sites of FL-cDNAs and Tiling Assembly genes. The transcriptional start and termination sites were compared with those of FL-cDNAs previously published. The distribution of the difference between the Tiling Assembly and FL-cDNA start and stop sites is presented here. A negative value indicates that the Tiling Assembly gene is shorter than the FL-cDNA and a positive value indicates that the Tiling Assembly gene is longer." path-from-xml="dsv01509.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4934646/dsv01509.tiff" path-from-xml="dsv01509.tiff" target="_blank">View large</a><a section="4934646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv01509.png?Expires=2147483647&Signature=W9oX6rzA3PwDvHz64kQdJOL3PPDa4Uhmzki8bIuwhyswUFpitBO1mTqqy4baRaZTGEKbRn9FPeQ6ASmOgT2UgTxKrqQsVslOt2Uc~-WwXdXzdpOubTM12OTh2x42Tnmsenr2I6Ihkz37775s1TT6d1ghHk3MeeO4MCx1DTBBFNFcFQOUhuUQA6hGB~x1bge9-UdXOD5C~rND3mCNnly24pFgxBZA2UCYSAtAByPByyjFKFtXLW33B7W8RhWXKuMqpmgcKIr3CcDjRmtLMVk3M0XR-j8HN~QZCgtnIIsXhb36T3nmpjrZRkZUaaPeOclTwKWWCMw~tfUH7b5XNPV7OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4934646&ar=347881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01509.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nucleotide differences between transcriptional start (A) and termination (B) sites of FL-cDNAs and Tiling Assembly genes. The transcriptional start and termination sites were compared with those of FL-cDNAs previously published. The distribution of the difference between the Tiling Assembly and FL-cDNA start and stop sites is presented here. A negative value indicates that the Tiling Assembly gene is shorter than the FL-cDNA and a positive value indicates that the Tiling Assembly gene is longer.</p></div></div><p>Of the 7,174 transcription termination sites that satisfied the specified overlap threshold, ∼69% differed by ≤100 nt (Fig. <a reveal-id="DSV015F9" data-open="DSV015F9" class="link link-reveal link-table xref-fig">9</a>B). The transcription termination sites predicted by Tiling Assembly were on average 71 nt downstream of the FL-cDNAs. These data demonstrate that Tiling Assembly is less reliable at predicting the transcription termination sites. Overall, Tiling Assembly overestimated the length of the transcripts. This overestimation may be due to noise reads near the start and termination sites. Alternatively, past research has indicated that termination sites are variable.<sup><a reveal-id="DSV015C32" data-open="DSV015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Hence, the accuracy rates of Tiling Assembly in predicting transcription termination sites may be underestimated.</p>                            <h3 scrollto-destination=4934649 id="4934649" class="section-title" data-legacy-id=s3k>3.11. Application of Tiling Assembly to the genomes of model organisms</h3>
<p>Having demonstrated the effect of Tiling Assembly on detecting novel genes in rice, its performance was evaluated on other model organisms. Loraine et al.<sup><a reveal-id="DSV015C33" data-open="DSV015C33" class="link link-ref link-reveal xref-bibr">33</a></sup> discovered 5,312 transcriptionally active regions (TARs) in the unannotated regions of the <em>A. thaliana</em> genome; however, few filters were used to remove false-positive TARs. For instance, of the 5,312 TARs reported, 3,490 were the length of a single read (75 nt). However, this large number of TARs indicated that there may still be undiscovered genes in Arabidopsis. Using the same Tiling Assembly parameters used for identifying potential novel genes in <em>O. sativa</em>, 218 potential novel genes were identified in Arabidopsis, representing nearly 1% of all annotated genes. Of these 218 potential novel genes, 99 genes (45%) contained at least part of one or more TARs, and 35 genes (16%) had at least one TAR completely contained within the gene. It is likely that most of the TARs reported by Lorraine et al. did not correlate with a Tiling Assembly novel gene because the majority of them were identified based on individual reads. These individual reads may have resulted from genomic DNA contamination or noise caused by statistical mapping error.</p><p>To determine whether the ability of Tiling Assembly to find large numbers of unannotated genes was applicable to non-plant species, the algorithm was applied to several additional model organisms (Table <a reveal-id="DSV015TB1" data-open="DSV015TB1" class="link link-reveal link-table xref-fig">1</a>). Surprisingly, 458 novel genes were identified in <em>S. cerevisiae</em>, representing almost 7% of the known genes. Even though this model organism has been more intensively studied than Arabidopsis, this large number of potential novel genes may be due to the fact that only a single RNA-seq replicate was used in this study. Similar analysis was performed on <em>D. melanogaster</em> and <em>C. elegans</em>. The number of potential novel genes identified in each of these additional organisms may be improved by using parameters specific to the organism. </p>                        <a id="4934653" scrollto-destination="4934653"></a>
<div content-id="DSV015TB1" class="table-modal"><div class="table-wrap"><div id="DSV015TB1" class="table-wrap-title" data-id="DSV015TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Potential novel genes identified in other organisms by Tiling Assembly</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Ch </th><th>GS (Mb) </th><th>No. of genes </th><th>No. of novel genes </th><th>% Annot. genes </th></tr></thead><tbody><tr><td><em>O. sativa</em> </td><td>12 </td><td>381 </td><td>55,986 </td><td>767 </td><td>1.37 </td></tr><tr><td><em>A. thaliana</em> </td><td>5 </td><td>125 </td><td>25,498 </td><td>218 </td><td>0.85 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>16 </td><td>12 </td><td>6,603 </td><td>458 </td><td>6.94 </td></tr><tr><td><em>C. elegans</em> </td><td>6 </td><td>97 </td><td>47,060 </td><td>126 </td><td>0.27 </td></tr><tr><td><em>D. melanogaster</em> </td><td>4 </td><td>120 </td><td>17,294 </td><td>361 </td><td>2.09 </td></tr></tbody></table></div><div content-id="DSV015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Ch </th><th>GS (Mb) </th><th>No. of genes </th><th>No. of novel genes </th><th>% Annot. genes </th></tr></thead><tbody><tr><td><em>O. sativa</em> </td><td>12 </td><td>381 </td><td>55,986 </td><td>767 </td><td>1.37 </td></tr><tr><td><em>A. thaliana</em> </td><td>5 </td><td>125 </td><td>25,498 </td><td>218 </td><td>0.85 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>16 </td><td>12 </td><td>6,603 </td><td>458 </td><td>6.94 </td></tr><tr><td><em>C. elegans</em> </td><td>6 </td><td>97 </td><td>47,060 </td><td>126 </td><td>0.27 </td></tr><tr><td><em>D. melanogaster</em> </td><td>4 </td><td>120 </td><td>17,294 </td><td>361 </td><td>2.09 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>RNA-seq data downloaded from the SRA were used to investigate the ability of Tiling Assembly to find unannotated genes in additional model organisms. For each of the organisms, the same criteria were used for identifying potential novel genes as those used for rice.</p></div></div><div content-id="tblfn2"><div class="fn"><p>Ch: number of chromosomes; GS: genome size; Annot. genes: annotated genes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4934653" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV015TB1" class="table-wrap-title" data-id="DSV015TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Potential novel genes identified in other organisms by Tiling Assembly</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Ch </th><th>GS (Mb) </th><th>No. of genes </th><th>No. of novel genes </th><th>% Annot. genes </th></tr></thead><tbody><tr><td><em>O. sativa</em> </td><td>12 </td><td>381 </td><td>55,986 </td><td>767 </td><td>1.37 </td></tr><tr><td><em>A. thaliana</em> </td><td>5 </td><td>125 </td><td>25,498 </td><td>218 </td><td>0.85 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>16 </td><td>12 </td><td>6,603 </td><td>458 </td><td>6.94 </td></tr><tr><td><em>C. elegans</em> </td><td>6 </td><td>97 </td><td>47,060 </td><td>126 </td><td>0.27 </td></tr><tr><td><em>D. melanogaster</em> </td><td>4 </td><td>120 </td><td>17,294 </td><td>361 </td><td>2.09 </td></tr></tbody></table></div><div content-id="DSV015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Ch </th><th>GS (Mb) </th><th>No. of genes </th><th>No. of novel genes </th><th>% Annot. genes </th></tr></thead><tbody><tr><td><em>O. sativa</em> </td><td>12 </td><td>381 </td><td>55,986 </td><td>767 </td><td>1.37 </td></tr><tr><td><em>A. thaliana</em> </td><td>5 </td><td>125 </td><td>25,498 </td><td>218 </td><td>0.85 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>16 </td><td>12 </td><td>6,603 </td><td>458 </td><td>6.94 </td></tr><tr><td><em>C. elegans</em> </td><td>6 </td><td>97 </td><td>47,060 </td><td>126 </td><td>0.27 </td></tr><tr><td><em>D. melanogaster</em> </td><td>4 </td><td>120 </td><td>17,294 </td><td>361 </td><td>2.09 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>RNA-seq data downloaded from the SRA were used to investigate the ability of Tiling Assembly to find unannotated genes in additional model organisms. For each of the organisms, the same criteria were used for identifying potential novel genes as those used for rice.</p></div></div><div content-id="tblfn2"><div class="fn"><p>Ch: number of chromosomes; GS: genome size; Annot. genes: annotated genes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4934653" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4934654 id="4934654" class="section-title" data-legacy-id=s3l>3.12. Discussion</h3>
<p>There are relatively few publicly available transcript assembly programmes despite the vast increase in the use of RNA-seq. In this study, a novel assembly algorithm, Tiling Assembly, was developed to address the lack of established algorithms to identify transcribed regions as genes. This algorithm was compared with the Cufflinks transcript assembly software to evaluate its gene-finding capabilities in relation to established assembly software (Table <a reveal-id="DSV015TB2" data-open="DSV015TB2" class="link link-reveal link-table xref-fig">2</a>). It was concluded that Tiling Assembly found substantially more genes and found a lower number of false-positive genes, though Cufflinks ran somewhat faster and was able to identify multiple transcripts for a given gene. It was also determined that Tiling Assembly and Cufflinks had similar accuracy at predicting single and multi-exonic genes down to 100 RPKE (Fig. <a reveal-id="DSV015F2" data-open="DSV015F2" class="link link-reveal link-table xref-fig">2</a>). When the Tiling Assembly genes were compared to FL-cDNAs, the vast majority of discrepancies could be attributed to alternative splicing. Excluding those genes, Tiling Assembly appeared to be as high as 99.4% accurate in identification of genes. </p>                        <a id="4934657" scrollto-destination="4934657"></a>
<div content-id="DSV015TB2" class="table-modal"><div class="table-wrap"><div id="DSV015TB2" class="table-wrap-title" data-id="DSV015TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Comparison of Tiling Assembly to Cufflinks</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Tiling Assembly </th><th>Cufflinks </th></tr></thead><tbody><tr><td>Novel gene finding<sup>a</sup> </td><td>3,473 genes </td><td>52 genes </td></tr><tr><td>False-positive rate<sup>b</sup> </td><td>17 out of 100 genes </td><td>25 out of 100 genes </td></tr><tr><td>Ease of use </td><td>User interface </td><td>Command line </td></tr><tr><td>Algorithm </td><td>Read tiling </td><td>Bipartite graph </td></tr><tr><td>Minimum expression </td><td>100 RPKE </td><td>100 RPKE </td></tr><tr><td>Run time<sup>c</sup> </td><td>3–4 h </td><td>1.5 h </td></tr><tr><td>Transcripts identified </td><td>Single transcript </td><td>Multiple transcripts </td></tr></tbody></table></div><div content-id="DSV015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Tiling Assembly </th><th>Cufflinks </th></tr></thead><tbody><tr><td>Novel gene finding<sup>a</sup> </td><td>3,473 genes </td><td>52 genes </td></tr><tr><td>False-positive rate<sup>b</sup> </td><td>17 out of 100 genes </td><td>25 out of 100 genes </td></tr><tr><td>Ease of use </td><td>User interface </td><td>Command line </td></tr><tr><td>Algorithm </td><td>Read tiling </td><td>Bipartite graph </td></tr><tr><td>Minimum expression </td><td>100 RPKE </td><td>100 RPKE </td></tr><tr><td>Run time<sup>c</sup> </td><td>3–4 h </td><td>1.5 h </td></tr><tr><td>Transcripts identified </td><td>Single transcript </td><td>Multiple transcripts </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>These figures represent genes found exclusively by either TA or Cufflinks that have an expression level greater than 100 RPKE, prior to percent similarity filter.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>b</sup>Data from random genome with 100 known genes inserted. No filtering performed.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>c</sup>Excludes set-up time. Set-up for Tiling Assembly and Cufflinks is about the same time.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4934657" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV015TB2" class="table-wrap-title" data-id="DSV015TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Comparison of Tiling Assembly to Cufflinks</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Tiling Assembly </th><th>Cufflinks </th></tr></thead><tbody><tr><td>Novel gene finding<sup>a</sup> </td><td>3,473 genes </td><td>52 genes </td></tr><tr><td>False-positive rate<sup>b</sup> </td><td>17 out of 100 genes </td><td>25 out of 100 genes </td></tr><tr><td>Ease of use </td><td>User interface </td><td>Command line </td></tr><tr><td>Algorithm </td><td>Read tiling </td><td>Bipartite graph </td></tr><tr><td>Minimum expression </td><td>100 RPKE </td><td>100 RPKE </td></tr><tr><td>Run time<sup>c</sup> </td><td>3–4 h </td><td>1.5 h </td></tr><tr><td>Transcripts identified </td><td>Single transcript </td><td>Multiple transcripts </td></tr></tbody></table></div><div content-id="DSV015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Tiling Assembly </th><th>Cufflinks </th></tr></thead><tbody><tr><td>Novel gene finding<sup>a</sup> </td><td>3,473 genes </td><td>52 genes </td></tr><tr><td>False-positive rate<sup>b</sup> </td><td>17 out of 100 genes </td><td>25 out of 100 genes </td></tr><tr><td>Ease of use </td><td>User interface </td><td>Command line </td></tr><tr><td>Algorithm </td><td>Read tiling </td><td>Bipartite graph </td></tr><tr><td>Minimum expression </td><td>100 RPKE </td><td>100 RPKE </td></tr><tr><td>Run time<sup>c</sup> </td><td>3–4 h </td><td>1.5 h </td></tr><tr><td>Transcripts identified </td><td>Single transcript </td><td>Multiple transcripts </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>These figures represent genes found exclusively by either TA or Cufflinks that have an expression level greater than 100 RPKE, prior to percent similarity filter.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>b</sup>Data from random genome with 100 known genes inserted. No filtering performed.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>c</sup>Excludes set-up time. Set-up for Tiling Assembly and Cufflinks is about the same time.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4934657" target="_blank">View Large</a></div></div><p>Before applying filters for potential novel genes, 28,019 genes that overlapped at least 75% with an annotated gene (Fig. <a reveal-id="DSV015F3" data-open="DSV015F3" class="link link-reveal link-table xref-fig">3</a>) were found from the RNA-seq data using Tiling Assembly. MSU R7 annotation contains 55,986 genes, which would seem to imply that half the annotated genes in the rice genome were expressed in the aleurone cells. In our previous publication, we reported that 18,152 annotated genes were expressed in the aleurone cells.<sup><a reveal-id="DSV015C9" data-open="DSV015C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The criteria used for expression in our previous publication required an expression level of at least 1.0 read per kilobase of exon model per million mapped reads (RPKM), which is equivalent to ∼150 RPKE for this data. At 150 RPKE, Tiling Assembly identified 17,971 annotated genes, which were in line with our previous publication. Since 100 RPKE was determined as the minimum expression for gene detection, Tiling Assembly identified 20,230 expressed annotated genes in rice aleurone.</p><p>Comparison of the novel genes identified by Tiling Assembly with Cufflinks shows that Tiling Assembly identifies up to 74% more genes than Cufflinks (Fig. <a reveal-id="DSV015F4" data-open="DSV015F4" class="link link-reveal link-table xref-fig">4</a>). After eliminating genes that showed high similarity to another genomic region, the Tiling Assembly identified 151 more novel genes than Cufflinks (Fig. <a reveal-id="DSV015F5" data-open="DSV015F5" class="link link-reveal link-table xref-fig">5</a>). We analysed the data to determine possible causes for the large discrepancy in genes identified between the two programmes. One possible reason for this discrepancy was that Tiling Assembly was calling regions that were highly similar to another region, whereas Cufflinks disregarded them. BLAST search filters revealed that this was not the case. Another possible reason was that Cufflinks and Tiling Assembly found a gene within the same region, but the start and stop locations of those genes were different. Investigation of the positional overlap of the high-confidence novel genes found by each of the programmes verified that those genes unique to each programme did not overlap. A final possibility considered was that some novel genes detected by Cufflinks were longer than the corresponding Tiling Assembly gene and thus overlapped with an adjacent annotated gene. This overlap eliminated the Cufflinks gene as a potential novel gene. Analysis showed that there were 101 Cufflinks genes that overlapped with an annotated gene. These occurrences were attributed to two causes; the Cufflinks gene included regions with low read depth, which were considered noise reads by Tiling Assembly, or Cufflinks exons were merged based on junction alignments that were disregarded by Tiling Assembly because they were very long and skipped exons. In five cases, one of these long junctions skipped over an expressed region, which was called a novel gene by Tiling Assembly but an intron by Cufflinks. In 19 additional cases, these long junction alignments led to extremely long genes identified by Cufflinks which spanned multiple MSU R7 annotated and Tiling Assembly genes. There were nine expressed regions detected as a gene by Tiling Assembly where there was no corresponding Cufflinks gene, for unknown reasons.</p><p>Tiling Assembly is a heuristic, <em>ab initio</em> transcript assembly algorithm, which uses a read tiling approach to identify transcripts. Unlike <em>de novo</em> assembly algorithms such as Trinity,<sup><a reveal-id="DSV015C5" data-open="DSV015C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Tiling Assembly takes advantage of a sequenced genome to improve the accuracy of transcript assembly while decreasing CPU requirements. Tiling Assembly does not require an annotated genome, so it may be used for organisms where the genome is sequenced but the annotation is naive. Many of the current transcript assembly algorithms attempt to reproduce each of the isoforms available to a gene using a bipartite graph approach, which can lead to reporting of statistically probable, but non-real isoforms and dilution of expression levels of real isoforms as reads are assigned to the non-real isoforms. Tiling Assembly instead produces the longest possible isoform of a gene. Comparison of Tiling Assembly with a well-established transcript assembly programme, Cufflinks, revealed that Tiling Assembly's strengths lie in the accurate prediction of exons in the presence of noise and improved discovery of high-confidence novel genes.</p><p>In conclusion, we describe a heuristic approach to novel gene identification. Using this approach in combination with Cufflinks, 767 high-confidence unannotated genes were identified in rice. These genes contained predicted ORFs ranging from 40 to over 4,000 codons, with the majority showing sequence homology to known and predicted proteins. The accuracy of the genes identified by Tiling Assembly was validated through comparison with their corresponding FL-cDNAs, which implied Tiling Assembly may be as accurate as 99.4%. Tiling Assembly accurately predicted the transcription start sites to within 100 nt of the corresponding FL-cDNA, but was less accurate at predicting the transcription termination sites. Application of Tiling Assembly on <em>A. thaliana</em>, <em>D. melanogaster</em>, <em>C. elegans</em>, and <em>S. cerevisiae</em> identified hundreds of high-confidence novel genes, demonstrating that even in the most well-studied model organisms there are still undiscovered genes. This pipeline proves to be an effective way to identify novel genes in a diverse array of organisms. The novel genes identified here should be further studied to determine their functions and roles in their organisms.</p>                            <h2 scrollto-destination=4934665 id="4934665" class="section-title" data-legacy-id=s4>4. Availability of software</h2>
<p>Project name: Tiling Assembly</p><p>Project home page: <a class="link link-uri" href="http://shenlab.sols.unlv.edu/shenlab/">http://shenlab.sols.unlv.edu/shenlab/</a></p><p>Operating systems: Platform independent</p><p>Programming language: PERL</p><p>Other requirements: MySQL or SQLite</p><p>License: Open Source license GNU General Public License version 2.0</p><p>Restrictions to use by non-academics: license needed</p>                            <h2 scrollto-destination=4934676 id="4934676" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4934678 id="4934678" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work is supported by a USDA grant (2008-35100-04519). Funding to pay the Open Access publication charges for this article was provided by University of Nevada, Las Vegas, USA.</p>                            <h2 scrollto-destination=4934680 id="4934680" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Special thanks to Liyuan Zhang and Ling Kun Gu for their technical support. Thanks to Hiram Clawson at UCSC for help with the genome browser. Thanks to Dr Amei Amei for assistance with statistical analysis.</p>                            <h2 scrollto-destination=4934682 id="4934682" class="backsection-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=4934684 id="4934684" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Gerstein</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Snyder</div>  <div class="given-names">M.</div></div></div> <div class="year">2009</div>, <div class="article-title">RNA-Seq: a revolutionary tool for transcriptomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">10</div>, <div class="fpage">57</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-Seq%3A%20a%20revolutionary%20tool%20for%20transcriptomics&amp;author=Z.%20Wang&amp;author=M.%20Gerstein&amp;author=M.%20Snyder&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=57-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagalakshmi</div>  <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Waern</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Snyder</div>  <div class="given-names">M.</div></div></div> <div class="year">2010</div>, <div class="article-title">RNA-Seq: a method for comprehensive transcriptome analysis</div>, <div class="source">Curr. Protoc. Mol. Biol.</div> , <div class="fpage">1</div>–<div class="lpage">13</div>.</div></div></div></div><div content-id="DSV015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oshlack</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Robinson</div>  <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Young</div>  <div class="given-names">M.D.</div></div></div> <div class="year">2010</div>, <div class="article-title">From RNA-seq reads to differential expression results</div>, <div class="source">Genome Biol.</div> , <div class="volume">11</div>, <div class="fpage">220</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20RNA-seq%20reads%20to%20differential%20expression%20results&amp;author=A.%20Oshlack&amp;author=M.D.%20Robinson&amp;author=M.D.%20Young&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-12-220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-12-220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-12-220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21176179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bertone</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Stolc</div>  <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Royce</div>  <div class="given-names">T.E.</div></div> et al.  </div>. <div class="year">2004</div>, <div class="article-title">Global identification of human transcribed sequences with genome tiling arrays</div>, <div class="source">Science</div> , <div class="volume">306</div>, <div class="fpage">2242</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20of%20human%20transcribed%20sequences%20with%20genome%20tiling%20arrays&amp;author=P.%20Bertone&amp;author=V.%20Stolc&amp;author=T.E.%20Royce&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=2242-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1103388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1103388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1103388"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15539566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div>  <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Haas</div>  <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Steijger</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Abril</div>  <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Engstrom</div>  <div class="given-names">P.G.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Assessment of transcript reconstruction methods for RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1177</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=J.F.%20Abril&amp;author=P.G.%20Engstrom&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1177-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yandell</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ence</div>  <div class="given-names">D.</div></div></div> <div class="year">2012</div>, <div class="article-title">A beginner's guide to eukaryotic genome annotation</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">13</div>, <div class="fpage">329</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20beginner's%20guide%20to%20eukaryotic%20genome%20annotation&amp;author=M.%20Yandell&amp;author=D.%20Ence&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=13&amp;pages=329-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22510764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Zeng</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Shi</div>  <div class="given-names">T.</div></div></div> <div class="year">2013</div>, <div class="article-title">Comparative study of de novo assembly and genome-guided assembly strategies for transcriptome reconstruction based on RNA-Seq</div>, <div class="source">Sci. China Life Sci.</div> , <div class="volume">56</div>, <div class="fpage">143</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20study%20of%20de%20novo%20assembly%20and%20genome-guided%20assembly%20strategies%20for%20transcriptome%20reconstruction%20based%20on%20RNA-Seq&amp;author=B.%20Lu&amp;author=Z.%20Zeng&amp;author=T.%20Shi&amp;publication_year=2013&amp;journal=Sci.%20China%20Life%20Sci.&amp;volume=56&amp;pages=143-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11427-013-4442-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11427-013-4442-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11427-013-4442-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23393030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watanabe</div>  <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">Ringler</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gu</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Shen</div>  <div class="given-names">Q.J.</div></div></div> <div class="year">2014</div>, <div class="article-title">RNA-sequencing reveals previously unannotated protein- and microRNA-coding genes expressed in aleurone cells of rice seeds</div>, <div class="source">Genomics</div> , <div class="volume">103</div>, <div class="fpage">122</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-sequencing%20reveals%20previously%20unannotated%20protein-%20and%20microRNA-coding%20genes%20expressed%20in%20aleurone%20cells%20of%20rice%20seeds&amp;author=K.A.%20Watanabe&amp;author=P.%20Ringler&amp;author=L.%20Gu&amp;author=Q.J.%20Shen&amp;publication_year=2014&amp;journal=Genomics&amp;volume=103&amp;pages=122-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2013.10.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2013.10.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2013.10.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24200500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div>  <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div>  <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, <div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Leinonen</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sugawara</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shumway</div>  <div class="given-names">M.</div></div></div> <div class="year">2011</div>, <div class="article-title">The sequence read archive</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">39</div>, <div class="fpage">D19</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20read%20archive&amp;author=R.%20Leinonen&amp;author=H.%20Sugawara&amp;author=M.%20Shumway&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D19-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21062823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawahara</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">de la Bastide</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hamilton</div>  <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Kanamori</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">McCombie</div>  <div class="given-names">W.R.</div></div></div> <div class="year">2013</div>, <div class="article-title">Improvement of the <em>Oryza sativa</em> Nipponbare reference genome using next generation sequence and optical map data</div>, <div class="source">Rice</div> , <div class="volume">6</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20the%20Oryza%20sativa%20Nipponbare%20reference%20genome%20using%20next%20generation%20sequence%20and%20optical%20map%20data&amp;author=Y.%20Kawahara&amp;author=M.%20de%20la%20Bastide&amp;author=J.P.%20Hamilton&amp;author=H.%20Kanamori&amp;author=W.R.%20McCombie&amp;publication_year=2013&amp;journal=Rice&amp;volume=6&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1939-8433-6-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1939-8433-6-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1939-8433-6-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24280374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goodstein</div>  <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Shu</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Howson</div>  <div class="given-names">R.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Phytozome: a comparative platform for green plant genomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">D1178</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytozome%3A%20a%20comparative%20platform%20for%20green%20plant%20genomics&amp;author=D.M.%20Goodstein&amp;author=S.%20Shu&amp;author=R.%20Howson&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D1178-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harris</div>  <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Baran</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bieri</div>  <div class="given-names">T.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">WormBase 2014: new views of curated biology</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">D789</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WormBase%202014%3A%20new%20views%20of%20curated%20biology&amp;author=T.W.%20Harris&amp;author=J.%20Baran&amp;author=T.%20Bieri&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D789-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24194605" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Engel</div>  <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Dietrich</div>  <div class="given-names">F.S.</div></div>, <div class="name string-name"><div class="surname">Fisk</div>  <div class="given-names">D.G.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">The reference genome sequence of <em>Saccharomyces cerevisiae</em>: then and now</div>, <div class="source">G3</div> , <div class="volume">4</div>, <div class="fpage">389</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genome%20sequence%20of%20Saccharomyces%20cerevisiae%3A%20then%20and%20now&amp;author=S.R.%20Engel&amp;author=F.S.%20Dietrich&amp;author=D.G.%20Fisk&amp;publication_year=2014&amp;journal=G3&amp;volume=4&amp;pages=389-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.113.008995" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.113.008995" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.113.008995"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24374639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">St. Pierre</div>  <div class="given-names">S.E.</div></div>, <div class="name string-name"><div class="surname">Ponting</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Stefancsik</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">McQuilton</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Consortium</div>  <div class="given-names">F.</div></div></div> <div class="year">2014</div>, <div class="article-title">FlyBase 102—advanced approaches to interrogating FlyBase</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">D780</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FlyBase%20102%E2%80%94advanced%20approaches%20to%20interrogating%20FlyBase&amp;author=S.E.%20St.%20Pierre&amp;author=L.%20Ponting&amp;author=R.%20Stefancsik&amp;author=P.%20McQuilton&amp;author=F.%20Consortium&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D780-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24234449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kikuchi</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Satoh</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nagata</div>  <div class="given-names">T.</div></div> et al.  </div>. <div class="year">2003</div>, <div class="article-title">Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice</div>, <div class="source">Science</div> , <div class="volume">301</div>, <div class="fpage">376</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%2C%20mapping%2C%20and%20annotation%20of%20over%2028%2C000%20cDNA%20clones%20from%20japonica%20rice&amp;author=S.%20Kikuchi&amp;author=K.%20Satoh&amp;author=T.%20Nagata&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=376-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1081288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1081288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1081288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Slater</div>  <div class="given-names">G.S.C.</div></div>, <div class="name string-name"><div class="surname">Birney</div>  <div class="given-names">E.</div></div></div> <div class="year">2005</div>, <div class="article-title">Automated generation of heuristics for biological sequence comparison</div>, <div class="source">BMC Bioinform.</div> , <div class="volume">6</div>, <div class="fpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.S.C.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinform.&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-31"> </span></div> </div></div></div></div></div><div content-id="DSV015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div>  <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Sugnet</div>  <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Furey</div>  <div class="given-names">T.S.</div></div> et al.  </div>. <div class="year">2002</div>, <div class="article-title">The human genome browser at UCSC</div>, <div class="source">Genome Res.</div> , <div class="volume">12</div>, <div class="fpage">996</div>–<div class="lpage">1006</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20genome%20browser%20at%20UCSC&amp;author=W.J.%20Kent&amp;author=C.W.%20Sugnet&amp;author=T.S.%20Furey&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=996-1006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229102. Article published online before print in May 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229102. Article published online before print in May 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229102. Article published online before print in May 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12045153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Anczukow</div>  <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Krainer</div>  <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">M.Q.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">C.</div></div></div> <div class="year">2013</div>, <div class="article-title">OLego: fast and sensitive mapping of spliced mRNA-Seq reads using small seeds</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">5149</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OLego%3A%20fast%20and%20sensitive%20mapping%20of%20spliced%20mRNA-Seq%20reads%20using%20small%20seeds&amp;author=J.%20Wu&amp;author=O.%20Anczukow&amp;author=A.R.%20Krainer&amp;author=M.Q.%20Zhang&amp;author=C.%20Zhang&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=5149-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23571760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gutierrez</div>  <div class="given-names">F.R.S.</div></div>, <div class="name string-name"><div class="surname">Güiza</div>  <div class="given-names">M.L.T.</div></div>, <div class="name string-name"><div class="surname">Magally del Carmen</div>  <div class="given-names">M.E.</div></div></div> <div class="year">2013</div>, <div class="article-title">Prevalence of <em>Trypanosoma cruzi</em> infection among people aged 15 to 89 years inhabiting the Department of Casanare (Colombia)</div>, <div class="source">PLoS Negl. Trop. Dis.</div> , <div class="volume">7</div>, <div class="fpage">e2113</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prevalence%20of%20Trypanosoma%20cruzi%20infection%20among%20people%20aged%2015%20to%2089%20years%20inhabiting%20the%20Department%20of%20Casanare%20(Colombia)&amp;author=F.R.S.%20Gutierrez&amp;author=M.L.T.%20G%C3%BCiza&amp;author=M.E.%20Magally%20del%20Carmen&amp;publication_year=2013&amp;journal=PLoS%20Negl.%20Trop.%20Dis.&amp;volume=7&amp;pages=e2113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pntd.0002113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pntd.0002113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pntd.0002113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23505592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Williams</div>  <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div>  <div class="given-names">G.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Verk</div>  <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Hickman</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Piterse</div>  <div class="given-names">C.M.J.</div></div>, <div class="name string-name"><div class="surname">Van Wees</div>  <div class="given-names">S.C.M.</div></div></div> <div class="year">2013</div>, <div class="article-title">RNA-Seq: revelation of the messengers</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">18</div>, <div class="fpage">175</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-Seq%3A%20revelation%20of%20the%20messengers&amp;author=M.C.%20Van%20Verk&amp;author=R.%20Hickman&amp;author=C.M.J.%20Piterse&amp;author=S.C.M.%20Van%20Wees&amp;publication_year=2013&amp;journal=Trends%20Plant%20Sci.&amp;volume=18&amp;pages=175-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2013.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2013.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2013.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23481128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yang</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Deng</div>  <div class="given-names">X.W.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">L.</div></div></div> <div class="year">2009</div>, <div class="article-title">Transcriptionally active gene fragments derived from potentially fast-evolving donor genes in the rice genome</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1215</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptionally%20active%20gene%20fragments%20derived%20from%20potentially%20fast-evolving%20donor%20genes%20in%20the%20rice%20genome&amp;author=X.%20Wang&amp;author=Z.%20Yu&amp;author=X.%20Yang&amp;author=X.W.%20Deng&amp;author=L.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1215-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div>  <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Madden</div>  <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div>  <div class="given-names">A.A.</div></div> et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brown</div>  <div class="given-names">T.A.</div></div></div>, ed. <div class="year">2002</div>, <div class="article-title">Understanding a genome sequence</div>. <div class="source">Genomes</div> , <div class="publisher-name">Wiley-Liss</div>, <div class="publisher-loc">Oxford</div>.</div></div></div></div><div content-id="DSV015C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Tschaplinski</div>  <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Hurst</div>  <div class="given-names">G.B.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Discovery and annotation of small proteins using genomics, proteomics, and computational approaches</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">634</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20and%20annotation%20of%20small%20proteins%20using%20genomics%2C%20proteomics%2C%20and%20computational%20approaches&amp;author=X.%20Yang&amp;author=T.J.%20Tschaplinski&amp;author=G.B.%20Hurst&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=634-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.109280.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.109280.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.109280.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21367939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ulitsky</div>  <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Bartel</div>  <div class="given-names">D.P.</div></div></div> <div class="year">2013</div>, <div class="article-title">lincRNAs: genomics, evolution, and mechanisms</div>, <div class="source">Cell</div> , <div class="volume">154</div>, <div class="fpage">26</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=lincRNAs%3A%20genomics%2C%20evolution%2C%20and%20mechanisms&amp;author=I.%20Ulitsky&amp;author=D.P.%20Bartel&amp;publication_year=2013&amp;journal=Cell&amp;volume=154&amp;pages=26-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.06.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.06.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.06.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23827673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Syed</div>  <div class="given-names">N.H.</div></div>, <div class="name string-name"><div class="surname">Kalyna</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Marquez</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Barta</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Brown</div>  <div class="given-names">J.W.S.</div></div></div> <div class="year">2012</div>, <div class="article-title">Alternative splicing in plants—coming of age</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">17</div>, <div class="fpage">616</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20in%20plants%E2%80%94coming%20of%20age&amp;author=N.H.%20Syed&amp;author=M.%20Kalyna&amp;author=Y.%20Marquez&amp;author=A.%20Barta&amp;author=J.W.S.%20Brown&amp;publication_year=2012&amp;journal=Trends%20Plant%20Sci.&amp;volume=17&amp;pages=616-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2012.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2012.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2012.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22743067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keren</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lev-Maor</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Ast</div>  <div class="given-names">G.</div></div></div> <div class="year">2010</div>, <div class="article-title">Alternative splicing and evolution: diversification, exon definition and function</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">11</div>, <div class="fpage">345</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20evolution%3A%20diversification%2C%20exon%20definition%20and%20function&amp;author=H.%20Keren&amp;author=G.%20Lev-Maor&amp;author=G.%20Ast&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=345-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2776"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20376054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV015C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Richard</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Manley</div>  <div class="given-names">J.L.</div></div></div> <div class="year">2009</div>, <div class="article-title">Transcription termination by nuclear RNA polymerases</div>, <div class="source">Genes Develop.</div> , <div class="volume">23</div>, <div class="fpage">1247</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20termination%20by%20nuclear%20RNA%20polymerases&amp;author=P.%20Richard&amp;author=J.L.%20Manley&amp;publication_year=2009&amp;journal=Genes%20Develop.&amp;volume=23&amp;pages=1247-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1792809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1792809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1792809"> </span></div> </div></div></div></div></div><div content-id="DSV015C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV015C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loraine</div>  <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">McCormick</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Estrada</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Patel</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Qin</div>  <div class="given-names">P.</div></div></div> <div class="year">2013</div>, <div class="article-title">RNA-Seq of Arabidopsis pollen uncovers novel transcription and alternative splicing</div>, <div class="source">Plant Physiol.</div> , <div class="volume">162</div>, <div class="fpage">1092</div>–<div class="lpage">109</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-Seq%20of%20Arabidopsis%20pollen%20uncovers%20novel%20transcription%20and%20alternative%20splicing&amp;author=A.E.%20Loraine&amp;author=S.%20McCormick&amp;author=A.%20Estrada&amp;author=K.%20Patel&amp;author=P.%20Qin&amp;publication_year=2013&amp;journal=Plant%20Physiol.&amp;volume=162&amp;pages=1092-109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.211441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.211441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.211441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23590974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347881&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv015-dsv015supp_table1.xlsx?Expires=2147483647&amp;Signature=ACluXA1tJJuc6eGYBr6Crj8WYLwiUGwIRRSeWGUHYvmAur6OCwz~kWVM0PhDy1UBJATpMkGa5t133yrDZ8yzcmoNR1yRdnGvrvgupDCmpuPjvA1B3~YU4yFPtJUjePoEGzPUNoTRfLQJk7k0m3lFgnX65kzsvSK0t~EAc0gl66H7UcOzuJbKT1Ne27QeBdqu5jdfJIg~R4kFgzGhavwEphvov2~vREOu-qngy2HTufmhXYZVBpHL7LKcIJ0h~kso-yOFQDGiupeZjRHWMOQXgch7VaINOTZoxoHjGcTODffUQjjydSYw6qYBxa4B7I1Im5IBuzg3~0Fqa0dhxScC1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv015/2/dsv015-dsv015supp_figures.pptx?Expires=2147483647&amp;Signature=LgWzTk9xK1qsIofuE0JlQJLaTYsWQhPJzrIW26NTIypoRPdq-iT01rwP4CvjA3Zj~e5l4An4dxeOj9-iM7zqtQf1te-4nI-Z-qZYvHj2DKR3AOM~s7movwgc14YQSYgO1NKj1Y9TFPGmXdMtvTPwFNSeAsmm-xMVLzLlXwPGwVDiKY14BZLb6mMx6EbCCWB0seTw8OqueVgxnek7YUP8c30V8AQ5pOsA~bMgxv0ZEjJ90aqwT77UNZNqLGLJwHJiS5UyORDfCR~H3vwtsDx1yNdvmc-2g2THBdZWUlg2Zd~xwcpn3POORwgIpDMKS8VcJwjtBAZgtJu2fWub4MpruQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pptx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">372</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347881"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f5%2f319%2f347881">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362820100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F5%2F319%2F347881" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347881'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Tiling Assembly: a new tool for reference annotation-independent transcript assembly and novel gene identification by RNA-sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f319%2f347881">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

