

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1528,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/4/375/354164","@type":"ScholarlyArticle","name":"Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via  In situ  Error-Prone PCR","datePublished":"2013-04-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2013-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst016","keywords":["thermostable DNA ligase","directed evolution","amplification of circular plasmids","random mutagenesis libraries","error-prone PCR"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/4/375/354164","author":[{"name":"Le, Yilin","affiliation":"1  Biofuels Institute, School of Environment ,  Jiangsu University ,  Zhenjiang, Jiangsu 212013 ,  PR China","@type":"Person"},{"name":"Chen, Huayou","affiliation":"1  Biofuels Institute, School of Environment ,  Jiangsu University ,  Zhenjiang, Jiangsu 212013 ,  PR China","@type":"Person"},{"name":"Zagursky, Robert","affiliation":"2  Department of Chemical Engineering ,  University of Rochester ,  Rochester, NY 14627-0166 ,  USA","@type":"Person"},{"name":"Wu, J.H. David","affiliation":"2  Department of Chemical Engineering ,  University of Rochester ,  Rochester, NY 14627-0166 ,  USA","@type":"Person"},{"name":"Shao, Weilan","affiliation":"1  Biofuels Institute, School of Environment ,  Jiangsu University ,  Zhenjiang, Jiangsu 212013 ,  PR China","@type":"Person"}],"description":"Abstract.  Polymerase chain reaction (PCR) is a powerful method to produce linear DNA fragments. Here we describe the Tma thermostable DNA ligase-mediated PCR p","pageStart":"375","pageEnd":"382","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via  In situ  Error-Prone PCR","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via  In situ  Error-Prone PCR" />
    <meta property="og:description" content="Abstract.  Polymerase chain reaction (PCR) is a powerful method to produce linear DNA fragments. Here we describe the Tma thermostable DNA ligase-mediated PCR p" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/4/375/354164" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-04-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Le, Yilin" /><meta name="citation_author_institution" content="Biofuels Institute, School of Environment, Jiangsu University, Zhenjiang, Jiangsu 212013, PR China" /><meta name="citation_author" content="Chen, Huayou" /><meta name="citation_author_institution" content="Biofuels Institute, School of Environment, Jiangsu University, Zhenjiang, Jiangsu 212013, PR China" /><meta name="citation_author" content="Zagursky, Robert" /><meta name="citation_author_institution" content="Department of Chemical Engineering, University of Rochester, Rochester, NY 14627-0166, USA" /><meta name="citation_author" content="Wu, J.H. David" /><meta name="citation_author_institution" content="Department of Chemical Engineering, University of Rochester, Rochester, NY 14627-0166, USA" /><meta name="citation_author" content="Shao, Weilan" /><meta name="citation_author_institution" content="Biofuels Institute, School of Environment, Jiangsu University, Zhenjiang, Jiangsu 212013, PR China" /><meta name="citation_title" content="Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via In situ Error-Prone PCR" /><meta name="citation_firstpage" content="375" /><meta name="citation_lastpage" content="382" /><meta name="citation_doi" content="10.1093/dnares/dst016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2013/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Improving the quality of industrially important enzymes by directed evolution; Mol. Cell Biochem.;  citation_year=2001;  citation_volume=224;  citation_pages=159-68; " /><meta name="citation_reference" content="citation_title=Directed evolution of industrial enzymes: an update; Curr. Opin. Biotechnol.;  citation_year=2003;  citation_volume=14;  citation_pages=438-43; " /><meta name="citation_reference" content="citation_title=Directed evolution of enzyme stability; Biomol. Eng.;  citation_year=2005;  citation_volume=22;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=Laboratory-directed protein evolution; Microbiol. Mol. Biol. Rev.;  citation_year=2005;  citation_volume=69;  citation_pages=373-92; " /><meta name="citation_reference" content="citation_title=Directed evolution: selecting today's biocatalysts; Biomol. Eng.;  citation_year=2005;  citation_volume=22;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=High-throughput screening of enzyme libraries; Curr. Opin. Biotechnol.;  citation_year=2000;  citation_volume=11;  citation_pages=331-7; " /><meta name="citation_reference" content="citation_title=Novel methods for directed evolution of enzymes: quality, not quantity; Curr. Opin. Biotechnol.;  citation_year=2004;  citation_volume=15;  citation_pages=291-7; " /><meta name="citation_reference" content="citation_title=Approaches to DNA mutagenesis: an overview; Anal. Biochem.;  citation_year=1997;  citation_volume=254;  citation_pages=157-78; " /><meta name="citation_reference" content="citation_title=Randomization of genes by PCR mutagenesis; PCR Methods Appl.;  citation_year=1992;  citation_volume=2;  citation_pages=28-33; " /><meta name="citation_reference" content="citation_title=A simple method for site-directed mutagenesis using the polymerase chain reaction; Nucleic Acids Res.;  citation_year=1989;  citation_volume=17;  citation_pages=6545-51; " /><meta name="citation_reference" content="citation_title=Whole plasmid mutagenic PCR for directed protein evolution; Biomol. Eng.;  citation_year=2005;  citation_volume=22;  citation_pages=73-9; " /><meta name="citation_reference" content="citation_title=MEGAWHOP cloning: a method of creating random mutagenesis libraries via megaprimer PCR of whole plasmids; Meth. Enzymol.;  citation_year=2011;  citation_volume=498;  citation_pages=399-406; " /><meta name="citation_reference" content="citation_title=Creating random mutagenesis libraries using megaprimer PCR of whole plasmid; Biotechniques;  citation_year=2002;  citation_volume=33;  citation_pages=1033-8; " /><meta name="citation_reference" content="citation_title=Properties of an NAD+-dependent DNA ligase from the hyperthermophile Thermotoga maritima and its application in PCR amplification of long DNA fragments; Enzyme Microb. Technol.;  citation_year=2010;  citation_volume=46;  citation_pages=113-7; " /><meta name="citation_reference" content="citation_title=A highly efficient method for liquid and solid cultivation of the anaerobic hyperthermophilic eubacterium Thermotoga maritime; FEMS Microbiol. Lett.;  citation_year=2006;  citation_volume=259;  citation_pages=254-9; " /><meta name="citation_reference" content="citation_title=pHsh vectors, a novel expression system of Escherichia coli for the large-scale production of recombinant enzymes; Biotechnol. Lett.;  citation_year=2010;  citation_volume=32;  citation_pages=795-801; " /><meta name="citation_reference" content="citation_title=High-level expression of the xylanase from Thermomyces lanuginosus in Escherichia coli; World J. Microbiol. Biotechnol.;  citation_year=2008;  citation_volume=24;  citation_pages=275-80; " /><meta name="citation_reference" content="citation_title=A new reaction for colorimetric determination of carbohydrates; Anal. Biochem.;  citation_year=1972;  citation_volume=47;  citation_pages=273-9; " /><meta name="citation_reference" content="citation_title=DNA polymerase-catalyzed addition of nontemplated extra nucleotides to the 3′ end of a DNA fragment; DNA Cell Biol.;  citation_year=1993;  citation_volume=12;  citation_pages=763-70; " /><meta name="citation_reference" content="citation_title=An in vitro evaluation of a thermostable pectate lyase by using error-prone PCR; J. Mol. Catal. B Enzym.;  citation_year=2004;  citation_volume=27;  citation_pages=127-31; " /><meta name="citation_reference" content="citation_title=Directed evolution of a fucosidase from a galactosidase by DNA shuffling and screening; Proc. Natl Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=4504-9; " /><meta name="citation_reference" content="citation_title=Enhancing thermostability of Escherichia coli phytase AppA2 by error-prone PCR; Appl. Microbiol. Biotechnol.;  citation_year=2008;  citation_volume=79;  citation_pages=69-75; " /><meta name="citation_reference" content="citation_title=PCR approaches to DNA mutagenesis and recombination. An overview; Methods Mol. Biol.;  citation_year=2002;  citation_volume=192;  citation_pages=167-74; " /><meta name="citation_reference" content="citation_title=Preparing libraries in Escherichia coli; Methods Mol. Biol.;  citation_year=2003;  citation_volume=231;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=An improvement of the site-directed mutagenesis method by combination of megaprimer, one-side PCR and DpnI treatment; Anal. Biochem.;  citation_year=2004;  citation_volume=331;  citation_pages=401-3; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/4/375/1013053/dst016.pdf" /><meta name="description" content="Abstract.  Polymerase chain reaction (PCR) is a powerful method to produce linear DNA fragments. Here we describe the Tma thermostable DNA ligase-mediated PCR p" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/4/375/354164" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/4/375/354164" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/4/375/354164");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst016';
    </script>
<title>
	Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via In situ Error-Prone PCR | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./354164" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZM8PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZSwgWWlsaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9mdWVscyBJbnN0aXR1dGUsIFNjaG9vbCBvZiBFbnZpcm9ubWVudCwgSmlhbmdzdSBVbml2ZXJzaXR5LCBaaGVuamlhbmcsIEppYW5nc3UgMjEyMDEzLCBQUiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIEh1YXlvdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2Z1ZWxzIEluc3RpdHV0ZSwgU2Nob29sIG9mIEVudmlyb25tZW50LCBKaWFuZ3N1IFVuaXZlcnNpdHksIFpoZW5qaWFuZywgSmlhbmdzdSAyMTIwMTMsIFBSIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmFndXJza3ksIFJvYmVydCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ2hlbWljYWwgRW5naW5lZXJpbmcsIFVuaXZlcnNpdHkgb2YgUm9jaGVzdGVyLCBSb2NoZXN0ZXIsIE5ZIDE0NjI3LTAxNjYsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ild1LCBKLkguIERhdmlkIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDaGVtaWNhbCBFbmdpbmVlcmluZywgVW5pdmVyc2l0eSBvZiBSb2NoZXN0ZXIsIFJvY2hlc3RlciwgTlkgMTQ2MjctMDE2NiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hhbywgV2VpbGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZnVlbHMgSW5zdGl0dXRlLCBTY2hvb2wgb2YgRW52aXJvbm1lbnQsIEppYW5nc3UgVW5pdmVyc2l0eSwgWmhlbmppYW5nLCBKaWFuZ3N1IDIxMjAxMywgUFIgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlRoZXJtb3N0YWJsZSBETkEgTGlnYXNlLU1lZGlhdGVkIFBDUiBQcm9kdWN0aW9uIG9mIENpcmN1bGFyIFBsYXNtaWQgKFBQQ1ApIGFuZCBJdHMgQXBwbGljYXRpb24gaW4gRGlyZWN0ZWQgRXZvbHV0aW9uIHZpYSBJbiBzaXR1IEVycm9yLVByb25lIFBDUiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzgyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN0MDE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgdGhlIHF1YWxpdHkgb2YgaW5kdXN0cmlhbGx5IGltcG9ydGFudCBlbnp5bWVzIGJ5IGRpcmVjdGVkIGV2b2x1dGlvbjsgTW9sLiBDZWxsIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTIyNDsgIGNpdGF0aW9uX3BhZ2VzPTE1OS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0ZWQgZXZvbHV0aW9uIG9mIGluZHVzdHJpYWwgZW56eW1lczogYW4gdXBkYXRlOyBDdXJyLiBPcGluLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQzOC00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0ZWQgZXZvbHV0aW9uIG9mIGVuenltZSBzdGFiaWxpdHk7IEJpb21vbC4gRW5nLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYWJvcmF0b3J5LWRpcmVjdGVkIHByb3RlaW4gZXZvbHV0aW9uOyBNaWNyb2Jpb2wuIE1vbC4gQmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTM3My05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0ZWQgZXZvbHV0aW9uOiBzZWxlY3RpbmcgdG9kYXkncyBiaW9jYXRhbHlzdHM7IEJpb21vbC4gRW5nLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IHNjcmVlbmluZyBvZiBlbnp5bWUgbGlicmFyaWVzOyBDdXJyLiBPcGluLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTMzMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBtZXRob2RzIGZvciBkaXJlY3RlZCBldm9sdXRpb24gb2YgZW56eW1lczogcXVhbGl0eSwgbm90IHF1YW50aXR5OyBDdXJyLiBPcGluLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI5MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHByb2FjaGVzIHRvIEROQSBtdXRhZ2VuZXNpczogYW4gb3ZlcnZpZXc7IEFuYWwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1NDsgIGNpdGF0aW9uX3BhZ2VzPTE1Ny03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFuZG9taXphdGlvbiBvZiBnZW5lcyBieSBQQ1IgbXV0YWdlbmVzaXM7IFBDUiBNZXRob2RzIEFwcGwuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0yOC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUgbWV0aG9kIGZvciBzaXRlLWRpcmVjdGVkIG11dGFnZW5lc2lzIHVzaW5nIHRoZSBwb2x5bWVyYXNlIGNoYWluIHJlYWN0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz02NTQ1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBwbGFzbWlkIG11dGFnZW5pYyBQQ1IgZm9yIGRpcmVjdGVkIHByb3RlaW4gZXZvbHV0aW9uOyBCaW9tb2wuIEVuZy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz03My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBV0hPUCBjbG9uaW5nOiBhIG1ldGhvZCBvZiBjcmVhdGluZyByYW5kb20gbXV0YWdlbmVzaXMgbGlicmFyaWVzIHZpYSBtZWdhcHJpbWVyIFBDUiBvZiB3aG9sZSBwbGFzbWlkczsgTWV0aC4gRW56eW1vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDk4OyAgY2l0YXRpb25fcGFnZXM9Mzk5LTQwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3JlYXRpbmcgcmFuZG9tIG11dGFnZW5lc2lzIGxpYnJhcmllcyB1c2luZyBtZWdhcHJpbWVyIFBDUiBvZiB3aG9sZSBwbGFzbWlkOyBCaW90ZWNobmlxdWVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTAzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9wZXJ0aWVzIG9mIGFuIE5BRCstZGVwZW5kZW50IEROQSBsaWdhc2UgZnJvbSB0aGUgaHlwZXJ0aGVybW9waGlsZSBUaGVybW90b2dhIG1hcml0aW1hIGFuZCBpdHMgYXBwbGljYXRpb24gaW4gUENSIGFtcGxpZmljYXRpb24gb2YgbG9uZyBETkEgZnJhZ21lbnRzOyBFbnp5bWUgTWljcm9iLiBUZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTExMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2hseSBlZmZpY2llbnQgbWV0aG9kIGZvciBsaXF1aWQgYW5kIHNvbGlkIGN1bHRpdmF0aW9uIG9mIHRoZSBhbmFlcm9iaWMgaHlwZXJ0aGVybW9waGlsaWMgZXViYWN0ZXJpdW0gVGhlcm1vdG9nYSBtYXJpdGltZTsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI1OTsgIGNpdGF0aW9uX3BhZ2VzPTI1NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1wSHNoIHZlY3RvcnMsIGEgbm92ZWwgZXhwcmVzc2lvbiBzeXN0ZW0gb2YgRXNjaGVyaWNoaWEgY29saSBmb3IgdGhlIGxhcmdlLXNjYWxlIHByb2R1Y3Rpb24gb2YgcmVjb21iaW5hbnQgZW56eW1lczsgQmlvdGVjaG5vbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz03OTUtODAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWxldmVsIGV4cHJlc3Npb24gb2YgdGhlIHh5bGFuYXNlIGZyb20gVGhlcm1vbXljZXMgbGFudWdpbm9zdXMgaW4gRXNjaGVyaWNoaWEgY29saTsgV29ybGQgSi4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI3NS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgcmVhY3Rpb24gZm9yIGNvbG9yaW1ldHJpYyBkZXRlcm1pbmF0aW9uIG9mIGNhcmJvaHlkcmF0ZXM7IEFuYWwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTcyOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MjczLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBwb2x5bWVyYXNlLWNhdGFseXplZCBhZGRpdGlvbiBvZiBub250ZW1wbGF0ZWQgZXh0cmEgbnVjbGVvdGlkZXMgdG8gdGhlIDPigLIgZW5kIG9mIGEgRE5BIGZyYWdtZW50OyBETkEgQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTc2My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW4gdml0cm8gZXZhbHVhdGlvbiBvZiBhIHRoZXJtb3N0YWJsZSBwZWN0YXRlIGx5YXNlIGJ5IHVzaW5nIGVycm9yLXByb25lIFBDUjsgSi4gTW9sLiBDYXRhbC4gQiBFbnp5bS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpcmVjdGVkIGV2b2x1dGlvbiBvZiBhIGZ1Y29zaWRhc2UgZnJvbSBhIGdhbGFjdG9zaWRhc2UgYnkgRE5BIHNodWZmbGluZyBhbmQgc2NyZWVuaW5nOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9NDUwNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmhhbmNpbmcgdGhlcm1vc3RhYmlsaXR5IG9mIEVzY2hlcmljaGlhIGNvbGkgcGh5dGFzZSBBcHBBMiBieSBlcnJvci1wcm9uZSBQQ1I7IEFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Nzk7ICBjaXRhdGlvbl9wYWdlcz02OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UENSIGFwcHJvYWNoZXMgdG8gRE5BIG11dGFnZW5lc2lzIGFuZCByZWNvbWJpbmF0aW9uLiBBbiBvdmVydmlldzsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE5MjsgIGNpdGF0aW9uX3BhZ2VzPTE2Ny03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlcGFyaW5nIGxpYnJhcmllcyBpbiBFc2NoZXJpY2hpYSBjb2xpOyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjMxOyAgY2l0YXRpb25fcGFnZXM9MTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW1wcm92ZW1lbnQgb2YgdGhlIHNpdGUtZGlyZWN0ZWQgbXV0YWdlbmVzaXMgbWV0aG9kIGJ5IGNvbWJpbmF0aW9uIG9mIG1lZ2FwcmltZXIsIG9uZS1zaWRlIFBDUiBhbmQgRHBuSSB0cmVhdG1lbnQ7IEFuYWwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMzMTsgIGNpdGF0aW9uX3BhZ2VzPTQwMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvNC8zNzUvMTAxMzA1My9kc3QwMTYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb24gKFBDUikgaXMgYSBwb3dlcmZ1bCBtZXRob2QgdG8gcHJvZHVjZSBsaW5lYXIgRE5BIGZyYWdtZW50cy4gSGVyZSB3ZSBkZXNjcmliZSB0aGUgVG1hIHRoZXJtb3N0YWJsZSBETkEgbGlnYXNlLW1lZGlhdGVkIFBDUiBwIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTQxNjQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk697TCu6MEAoPRDJWwk/7m7ZOOYc=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f375%2f354164" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f375%2f354164" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13681" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&amp;Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/4">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5048882">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048882">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5048886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048886">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5048903">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048903">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5048946">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048946">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5048977">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048977">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5048989">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048989">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5048991">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5048991">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/4/355/353793">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/4/383/354335">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via <em>In situ</em> Error-Prone PCR
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yilin Le</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biofuels Institute, School of Environment</div>, <div class="insititution">Jiangsu University</div>, <div class="addr-line">Zhenjiang, Jiangsu 212013</div>, <div class="country">PR China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yilin+Le">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Le Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Le Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yilin Le</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huayou Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biofuels Institute, School of Environment</div>, <div class="insititution">Jiangsu University</div>, <div class="addr-line">Zhenjiang, Jiangsu 212013</div>, <div class="country">PR China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huayou+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huayou Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert Zagursky</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Chemical Engineering</div>, <div class="insititution">University of Rochester</div>, <div class="addr-line">Rochester, NY 14627-0166</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robert+Zagursky">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zagursky R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zagursky R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robert Zagursky</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J.H. David Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Chemical Engineering</div>, <div class="insititution">University of Rochester</div>, <div class="addr-line">Rochester, NY 14627-0166</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 25-85891836, +585-275-8499. E-mail: <a href="mailto:weilanshao@gmail.com" target="_blank">weilanshao@gmail.com</a> (W.S.), <a href="mailto:davidwu@che.rochester.edu" target="_blank">davidwu@che.rochester.edu</a> (J.H.D.W.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.H.+David+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">J.H. David Wu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Weilan Shao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biofuels Institute, School of Environment</div>, <div class="insititution">Jiangsu University</div>, <div class="addr-line">Zhenjiang, Jiangsu 212013</div>, <div class="country">PR China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 25-85891836, +585-275-8499. E-mail: <a href="mailto:weilanshao@gmail.com" target="_blank">weilanshao@gmail.com</a> (W.S.), <a href="mailto:davidwu@che.rochester.edu" target="_blank">davidwu@che.rochester.edu</a> (J.H.D.W.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Weilan+Shao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shao W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shao W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Weilan Shao</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 375–382, <a href='https://doi.org/10.1093/dnares/dst016'>https://doi.org/10.1093/dnares/dst016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 April 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 January 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 April 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="354164" href="/dnaresearch/article-pdf/20/4/375/1013053/dst016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yilin Le, Huayou Chen, Robert Zagursky, J.H. David Wu, Weilan Shao;  Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via <em>In situ</em> Error-Prone PCR, <em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 375–382, <a href='https://doi.org/10.1093/dnares/dst016'>https://doi.org/10.1093/dnares/dst016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/354164?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/354164?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/354164?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/354164?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/354164?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/354164?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/354164?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Thermostable%20DNA%20Ligase-Mediated%20PCR%20Production%20of%20Circular%20Plasmid%20(PPCP)%20and%20Its%20Application%20in%20Directed%20Evolution%20via%20%26lt%3Bem%26gt%3BIn%20situ%26lt%3B%2Fem%26gt%3B%20Error-Prone%20PCR&amp;publicationDate=2013-04-30&amp;volumeNum=20&amp;issueNum=4&amp;author=Le%2C%20Yilin%3B%20Chen%2C%20Huayou&amp;startPage=375&amp;endPage=382&amp;contentId=10.1093%2Fdnares%2Fdst016&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f375%2f354164" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5048882 id="5048882" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Polymerase chain reaction (PCR) is a powerful method to produce linear DNA fragments. Here we describe the <em>Tma</em> thermostable DNA ligase-mediated PCR production of circular plasmid (PPCP) and its application in directed evolution via <em>in situ</em> error-prone PCR. In this thermostable DNA ligase-mediated whole-plasmid amplification method, the resultant DNA nick between the 5′ end of the PCR primer and the extended newly synthesized DNA 3′ end of each PCR cycle is ligated by <em>Tma</em> DNA ligase, resulting in circular plasmid DNA product that can be directly transformed. The template plasmid DNA is eliminated by ‘selection marker swapping’ upon transformation. When performed under an error-prone condition with <em>Taq</em> DNA polymerase, PPCP allows one-step construction of mutagenesis libraries based on <em>in situ</em> error-prone PCR so that random mutations are introduced into the target gene without altering the expression vector plasmid. A significant difference between PPCP and previously published methods is that PPCP allows exponential amplification of circular DNA. We used this method to create random mutagenesis libraries of a xylanase gene and two cellulase genes. Screening of these libraries resulted in mutant proteins with desired properties, demonstrating the usefulness of <em>in situ</em> error-prone PPCP for creating random mutagenesis libraries for directed evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;thermostable DNA ligase&quot;">thermostable DNA ligase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;directed evolution&quot;">directed evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;amplification of circular plasmids&quot;">amplification of circular plasmids</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;random mutagenesis libraries&quot;">random mutagenesis libraries</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;error-prone PCR&quot;">error-prone PCR</a></div>                            <h2 scrollto-destination=5048886 id="5048886" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Directed evolution methods are increasingly employed to improve properties of an enzyme, including substrate specificity, activity, thermostability, high-temperature activity.<sup><a reveal-id="DST016C1  DST016C4" data-open="DST016C1  DST016C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> Success of directed protein evolution hinges on the efficiency in creating mutagenesis libraries and screening the libraries for the desired protein properties.<sup><a reveal-id="DST016C5" data-open="DST016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The library diversity is a key parameter and various protocols have been developed to achieve the goal of making high-quality mutagenesis libraries.<sup><a reveal-id="DST016C5" data-open="DST016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Random mutagenesis coupled to high-throughput screening<sup><a reveal-id="DST016C6" data-open="DST016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> has become a powerful tool for both basic study on the structure–function relationship of proteins and development of novel proteins with desired properties.</p><p>Many methods used for directed evolution of enzymes were reviewed by Lutz and Patrick.<sup><a reveal-id="DST016C7" data-open="DST016C7" class="link link-ref link-reveal xref-bibr">7</a></sup><em>In vitro</em> mutagenesis approaches have been described in both polymerase chain reaction (PCR)-based and non-PCR-based categories.<sup><a reveal-id="DST016C8" data-open="DST016C8" class="link link-ref link-reveal xref-bibr">8</a></sup> One of the most commonly used methods is error-prone mutagenic PCR,<sup><a reveal-id="DST016C9" data-open="DST016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> which introduces random mutations by reducing the fidelity of the DNA polymerase. Early techniques of error-prone PCR generally involved the following steps: (i) amplifying the target gene under error-prone conditions to generate randomly mutated sequences; (ii) digesting the ends of the amplified DNA fragment with restriction endonucleases; (iii) ligating the digested DNA fragment into a suitable expression vector with DNA ligase and (iv) transforming the ligated vector into a suitable host cell to obtain a mutant library. This process is similar to standard PCR cloning of a target sequence except that, for constructing a diverse mutagenesis library, a much larger pool of transformants are needed to increase the diversity of the library, making the process tedious and inefficient. As a result, various methods have been developed to streamline the process.</p><p>Previously reported procedures for whole-plasmid, mutagenic PCR using the entire plasmid as template and back-to-back primer set resulted in linear DNA products, necessitating a separate ligation step to generate circular plasmids before transforming the host cell.<sup><a reveal-id="DST016C10" data-open="DST016C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST016C11" data-open="DST016C11" class="link link-ref link-reveal xref-bibr">11</a></sup> MegaWHOP uses mutated target sequence as the PCR primer (Megaprimers), expression vector harbouring the wild-type gene as the template, and a high-fidelity DNA polymerase to produce whole length of plasmid.<sup><a reveal-id="DST016C12" data-open="DST016C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, the process mainly yields linear single-strand DNA (ssDNA), which is released from its circular template at the denaturation step during PCR. Only some of the ssDNA strands can anneal to form nicked circular double-strand DNA (dsDNA) to be directly transformed without subsequent ligation step (Step iii mentioned above). In addition, the newly extended, linear ssDNA cannot serve as a template to achieve exponential amplification. Finally, the procedure depends on methylation of template DNA and later on digestion of methylated DNA by <em>Dpn</em>I, which degrades the methylated plasmid template, before the mutated plasmid products are transformed.</p><p>To obtain nick-free, circular plasmid production by PCR and improve the efficiency of random mutagenesis or directed evolution, we report a novel method designated as ‘thermostable DNA ligase-mediated PCR production of circular plasmid’ (PPCP) for applications in directed evolution via <em>in situ</em> error-prone PCR. The PPCP method employs a long dsDNA primer pair to amplify the entire template plasmid. The dsDNA primer pair is designed to carry a selection marker different from the original selection marker of the template plasmid. Transformants are plated under the pressure of the new selection marker, resulting in the elimination of the original template plasmid without the need to pretreat the DNA with <em>Dpn</em>I. Furthermore, thermostable DNA ligase from <em>Thermotoga maritima</em> (<em>Tma</em> DNA ligase), which we previously reported,<sup><a reveal-id="DST016C14" data-open="DST016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> is employed to ligate the nick of the newly extended DNA strand, resulting in the formation of circular plasmid DNA as the amplification product that could directly be transformed. When <em>Taq</em> DNA polymerase is used under error-prone conditions, the method generates random mutagenesis libraries in a fast and efficient manner. By using this method, we performed directed evolution on XynA (<em>xylanase A</em>) from <em>Thermomyces lanuginosus</em> and CelA (<em>cellulase A</em>), CelB (<em>cellulase B</em>) from <em>T. maritima</em>. The results demonstrated that <em>in situ</em> error-prone PPCP is an efficient random mutagenesis method for directed evolution of genes and proteins.</p>                            <h2 scrollto-destination=5048903 id="5048903" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5048905 id="5048905" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains, plasmids and growth media</h3>
<p><em>Thermotoga maritima</em> (ATCC43589) was grown anaerobically at 80°C in a medium described previously.<sup><a reveal-id="DST016C15" data-open="DST016C15" class="link link-ref link-reveal xref-bibr">15</a></sup><em>Escherichia coli</em> BL21(DE3) was used as the host for gene cloning<em>. Escherichia coli</em> was cultured in Luria-Bertani (LB) medium which contained (per litre): 10 g Tryptone, 5 g yeast extract, 5 g NaCl, pH 7, and supplemented with 100 μg/ml ampicillin or 50 μg/ml kanamycin for selection. The Hsh (heat shock) system of expression vectors was employed for PPCP tests, which contains an Hsh promoter recognizable by an alternative sigma factor (σ<sup>32</sup>) of <em>E. coli</em>, and the expression of foreign genes was induced by a temperature up-shift.<sup><a reveal-id="DST016C16" data-open="DST016C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=5048909 id="5048909" class="section-title" data-legacy-id=s2b>2.2. Generating long complementary primer pair for swapping plasmid selection maker by PPCP</h3>
<p>The complementary PPCP primer pair containing the kanamycin resistance gene (884 bp) used in the experiment shown in Fig. <a reveal-id="DST016F2" data-open="DST016F2" class="link link-reveal link-table xref-fig">2</a> was generated by regular PCR using the plasmid pET-28a(+) as template. The regular PCR primers were KanFwd, 5′-<em>CCCTGATAAATGCTTCAATAATATTGAAAAAGGAAGAGT</em>ATGAGCCATATTCAACGGG-3′, and KanRev, 5′-<em>ATCAATCTAAAGTATATATGAGTAAACTTGGTCTGAC AG</em>TTAGAAAAACTCATCGAGC-3′ (italicized DNA sequences complementary to the flanking sequences of the ampicillin gene (863 bp) in the PPCP template pHsh described below; underlined DNA sequences complementary to the kanamycin resistance gene from pET-28a(+)). PCR was performed in 50 μl reaction solution containing 1.25 U <em>Pyrobest</em> DNA polymerase (TaKaRa) for 30 cycles, each cycle consisting of incubation at 94°C for 40 s, 52°C for 40 s and 72°C for 1 min 10 s. The 894 bp dsDNA fragment generated was subsequently used as PPCP primer pair for amplifying whole pHsh plasmid DNA as described below.</p>                            <h3 scrollto-destination=5048913 id="5048913" class="section-title" data-legacy-id=s2c>2.3. Amplification of whole pHsh plasmid DNA by PPCP with simultaneous selection maker swapping</h3>
<p>The PPCP experiment shown in Fig. <a reveal-id="DST016F2" data-open="DST016F2" class="link link-reveal link-table xref-fig">2</a> was performed in 20 μl reaction solution containing 50 ng pHsh-<em>Amp</em> (2.44 kb) as template, 0.5 U <em>Taq</em> DNA polymerase (TaKaRa), 0.1 μg/μl <em>Tma</em> DNA ligase,<sup><a reveal-id="DST016C14" data-open="DST016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> 80 ng PPCP primer pair prepared by PCR as described above, 0.2 mM each deoxynucleotide triphosphate, 2.0 mM MgCl<sub>2</sub> and 0.5 mM NAD<sup>+</sup>. PCR was performed for 25 cycles with each cycle consisting of incubation at 94°C for 30 s (denaturation), 62°C for 30 s (annealing), 72°C for 1.5 min (extension) and 60°C for 2 min (ligation).</p>                            <h3 scrollto-destination=5048919 id="5048919" class="section-title" data-legacy-id=s2d>2.4. Preparation of long primer pair from pHsh for <em>in situ</em> error-prone PPCP mutagenesis</h3>
<p>To amplify the entire pHsh expression vector by regular PCR for preparing the long primer pair used for the PPCP experiment shown in Fig. <a reveal-id="DST016F4" data-open="DST016F4" class="link link-reveal link-table xref-fig">4</a>, the following regular PCR primers were used: pHshRev 5′- pCCTCCATGGGTATATCTCCTT-3′ and pHshFwd 5′-pAAGCTTGAAGGCCGCTTCCGA-3′. The plasmid pHsh-<em>kan</em> (GenBank accession no. FJ571621) or pHsh-<em>amp</em> (GenBank accession no. FJ571619) was used as template. PCR of 30 cycles was performed in 50 μl of reaction solution containing 1.25 U <em>Pyrobest</em> DNA polymerase (TaKaRa), each cycle consisting of incubation at 94°C for 30 s, 54°C for 30 s and 72°C for 2 min.</p>                            <h3 scrollto-destination=5048925 id="5048925" class="section-title" data-legacy-id=s2e>2.5. <em>In situ</em> error-prone PPCP mutagenesis</h3>
<p>Error-prone PPCP was performed in 20 μl reaction solution containing 30 ng plasmid pHsh-<em>xynA</em>1, pHsh-<em>celA</em> or pHsh-<em>celB</em> (see below for their preparations) as template as described in the above, 0.5 U <em>Taq</em> DNA polymerase (TaKaRa), 0.1 μg/μl <em>Tma</em> DNA ligase,<sup><a reveal-id="DST016C14" data-open="DST016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> 200 ng PPCP primer pair from pHsh described above, 0.2 mM each deoxynucleotide triphosphate, 2.0 mM MgCl<sub>2</sub>, 0.5 mM MnCl<sub>2</sub> and 0.5 mM NAD<sup>+</sup>. Each cycle consisted of heating at 94°C for 1 min, annealing and extension at 72°C for 1.5 min and ligation at 60°C for 2 min.</p>                            <h3 scrollto-destination=5048931 id="5048931" class="section-title" data-legacy-id=s2f>2.6. Construction of plasmids pHsh-celA, pHsh-celB and pHsh-xynA1</h3>
<p>Endoglucanase gene <em>celA</em> was amplified by PCR using the genomic DNA of <em>T. maritima</em> as template and the following primers: 5′-CTGTGGTACTGATGACAAAACCGGGAACATC-3′ and 5′-GGGAAGCTTTCATTCTCTCACCTCCAGATC-3′. The amplicon was inserted into expression vector pHsh-<em>amp</em> at <em>Stu</em>І/<em>Hin</em>dШ sites to generate plasmid pHsh-<em>celA</em>. Endoglucanase gene <em>celB</em> of <em>T. maritima</em> was similarly amplified using primers 5′-CTAGCGTTGGTGCAACGGAC-3′ and 5′-GGGCTCGAGTTATTTTACAACTTCGACAG-3′, and cloned into pHsh-<em>amp</em> at <em>Stu</em>І/<em>Xho</em>І sites, resulting in pHsh-<em>celB</em>. pHsh-<em>xynA</em>1 was generated by cloning the <em>xynA</em>1 from <em>Thermomyces lanuginosus</em> as previously described.<sup><a reveal-id="DST016C17" data-open="DST016C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=5048937 id="5048937" class="section-title" data-legacy-id=s2g>2.7. Screening xylanase and cellulase random mutant libraries</h3>
<p>Mutant libraries of <em>celA</em>, <em>celB</em> and <em>xynA</em>1, respectively, were constructed by <em>in situ</em> error-prone mutagenic PPCP as described above. The resulting mutant libraries were transformed into <em>E. coli</em> by electroporation and screened for xylanase or cellulase activity by overlay assays. For detecting xylanase activity, <em>E. coli</em> colonies were overlaid with LB top agar containing 2% xylan and kanamycin (50 μg/ml), incubated at 30°C overnight and then the temperature shifted to 42°C for 5–6 h to induce gene expression. Positive clones were identified by a clear zone surrounding xylanase-expressing colonies. Cellulase activity was similarly detected as follows. <em>Escherichia coli</em> colonies were overlaid with LB top agar containing 2% carboxymethylcellulose (CMC, Sigma) and kanamycin (50 μg/ml). After incubation at 30°C overnight, the temperature was shifted to 42°C for 5–6 h to induce gene expression. Positive clones were identified by a depressed zone surrounding cellulase-expressing colonies.</p>                            <h3 scrollto-destination=5048941 id="5048941" class="section-title" data-legacy-id=s2h>2.8. Cellulase activity assay</h3>
<p>Cell lysates from cells harbouring a cellulase gene were prepared by sonication and used as the source of cellulase. The mutant and wild-type enzyme preparations were prepared from the same amount of cells. For cellulase assay, reducing sugars released from CMC by the cellulase activity in the cell lysate were quantified by using the 4-hydroxybenzoic acid hydrazide method.<sup><a reveal-id="DST016C18" data-open="DST016C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The reaction mixture comprises 100 μl of 0.5% (w/v) CMC in water, 90 μl of phosphate buffer (50 mM, pH 6.0) and 10 μl of properly diluted enzyme. After incubation at 90°C for 10 min, the reaction was stopped by adding 600 μl of 4-hydroxybenzoic acid hydrazide solution to the reaction mixture. The test tubes were incubated for 10 min in boiling water and chilled on ice. Reducing sugars were determined by measuring optical density at 410 nm using a spectrophotometer. One unit of cellulase activity is defined as the amount of enzyme releasing 1 μmol/ml reducing sugars using glucose as the standard.</p>                            <h2 scrollto-destination=5048946 id="5048946" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5048949 id="5048949" class="section-title" data-legacy-id=s3a>3.1. Swapping plasmid selection marker by thermostable DNA ligase-mediated PPCP</h3>
<p>PCR is a powerful method to amplify linear DNA fragments, which consists of repeated cycles of denaturation–annealing–elongation. Recently, we reported that adding a ligation step mediated by thermostable <em>Thermotoga maritima</em> (<em>Tma</em>) DNA ligase to form the repeated cycles of ‘denaturation–annealing–elongation–ligation’ enhances the amplification of long DNA fragment.<sup><a reveal-id="DST016C14" data-open="DST016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In both cases, the resulting PCR products are linear DNA. Here we describe that <em>Tma</em> thermostable DNA ligase can be employed in a new design for PCR amplification of circular plasmids. The procedure for thermostable DNA ligase-mediated PPCP is schematically illustrated in Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>. In this method, circular PCR products are generated by using a long complementary primer pair, which can be generated by regular PCR or synthesized, to extend over a circular template, followed by DNA ligation of the resultant nick between the 3′ end of the newly extended strand and the 5′ end of the primer. Furthermore, if the complementary primer set is designed to contain a second selection marker, the resulting plasmid product will harbour this new marker (Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>). The template plasmid carrying the original marker is eliminated when transformed host cells are cultured under the second selection pressure, eliminating the need of digesting PCR products with a methylated DNA-specific restriction enzyme mentioned above. The ability to swap the selection marker facilitates multiple rounds of mutagenesis and selection as will be discussed below. </p>                        <a id="5048955" scrollto-destination="5048955"></a>
<div id="DST016F1" data-id="DST016F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01601.jpeg?Expires=2147483647&Signature=hr25EqJ1RHfmCISF4Cshwor9KFYomp5NuW8NC2pR42DZ9lEWRA1M941w0kovQ~ehR3bpKQLdXOUasXG5w8ZRbO1USbdw6F0QTGUuZVdcz3VrvdnsSfTsY-rWNZalml3Ohs-r3O8np5jK521PagiqyDNYi6Vc~J8Wa9aJ2Uc6wEP~JvYoNIkUloaMpE5d4d3q5d25ugzOGTjfRG0GKph3S4CvXkz3ShXZGpqpBf9X7GxcRYtb51wu0CKN~jhF9niDj01RbAhAEgNxoztBpOm8sM8Ur8hDj927zHFF3FDGneohj~3yMzCSNTR399HYljVkl5cgi2uo7enGgLOHrifRLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of swapping selection marker of a plasmid by thermostable DNA ligase-mediated PPCP. The white boxed-region of the template plasmid (Vector 1) represents the original selection marker, which after the whole-plasmid PCR is replaced by a second selection marker built in the double-stranded PPCP primers (the black boxed-region in the primers and Vector 2). The procedure features the incorporation of a thermostable ligation step into the standard PCR procedure to ligate the nick remaining in the extended DNA strand, resulting in nick-free, circular plasmid DNA products. At the end of the PCR cycles, the template plasmid is eliminated by plating the plasmid-transformed host cells under the second selection pressure." path-from-xml="dst01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048955/dst01601.jpeg" path-from-xml="dst01601.jpeg" target="_blank">View large</a><a section="5048955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01601.jpeg?Expires=2147483647&Signature=NTWBGMvfknezdcpZQrY~0Rx53ImuFk7uSCYLdBNTiuB-nwRnqNAxsHjxYIJV-za76f8b0vBmdQCt7mKQaDi4EtQDZYK2yCQ3-UDLqwc2cRa4-8auGDrrQkSMm6-0ylxkdwVPLE3reoBCv9vHQUaxsUramYjXS89uwLxqPUdbK6K-Z0Gg-wxZobjUunvhu33vlz0gOCHxsMxBF8G2o~qkSANDyp1cO76U0oqATRIJFYVvmPSSEcs4e5rGbjSHN0-sCIIsRKQThljziCz1SRRK5achFU7v-xZIOj8V7OwotTCpVLDN2uk6ogzLjHMPQPrtE2AI8S4UaClGnox4Wbsbhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048955&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of swapping selection marker of a plasmid by thermostable DNA ligase-mediated PPCP. The white boxed-region of the template plasmid (Vector 1) represents the original selection marker, which after the whole-plasmid PCR is replaced by a second selection marker built in the double-stranded PPCP primers (the black boxed-region in the primers and Vector 2). The procedure features the incorporation of a thermostable ligation step into the standard PCR procedure to ligate the nick remaining in the extended DNA strand, resulting in nick-free, circular plasmid DNA products. At the end of the PCR cycles, the template plasmid is eliminated by plating the plasmid-transformed host cells under the second selection pressure.</p></div></div><div content-id="DST016F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01601.jpeg?Expires=2147483647&Signature=hr25EqJ1RHfmCISF4Cshwor9KFYomp5NuW8NC2pR42DZ9lEWRA1M941w0kovQ~ehR3bpKQLdXOUasXG5w8ZRbO1USbdw6F0QTGUuZVdcz3VrvdnsSfTsY-rWNZalml3Ohs-r3O8np5jK521PagiqyDNYi6Vc~J8Wa9aJ2Uc6wEP~JvYoNIkUloaMpE5d4d3q5d25ugzOGTjfRG0GKph3S4CvXkz3ShXZGpqpBf9X7GxcRYtb51wu0CKN~jhF9niDj01RbAhAEgNxoztBpOm8sM8Ur8hDj927zHFF3FDGneohj~3yMzCSNTR399HYljVkl5cgi2uo7enGgLOHrifRLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of swapping selection marker of a plasmid by thermostable DNA ligase-mediated PPCP. The white boxed-region of the template plasmid (Vector 1) represents the original selection marker, which after the whole-plasmid PCR is replaced by a second selection marker built in the double-stranded PPCP primers (the black boxed-region in the primers and Vector 2). The procedure features the incorporation of a thermostable ligation step into the standard PCR procedure to ligate the nick remaining in the extended DNA strand, resulting in nick-free, circular plasmid DNA products. At the end of the PCR cycles, the template plasmid is eliminated by plating the plasmid-transformed host cells under the second selection pressure." path-from-xml="dst01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048955/dst01601.jpeg" path-from-xml="dst01601.jpeg" target="_blank">View large</a><a section="5048955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01601.jpeg?Expires=2147483647&Signature=NTWBGMvfknezdcpZQrY~0Rx53ImuFk7uSCYLdBNTiuB-nwRnqNAxsHjxYIJV-za76f8b0vBmdQCt7mKQaDi4EtQDZYK2yCQ3-UDLqwc2cRa4-8auGDrrQkSMm6-0ylxkdwVPLE3reoBCv9vHQUaxsUramYjXS89uwLxqPUdbK6K-Z0Gg-wxZobjUunvhu33vlz0gOCHxsMxBF8G2o~qkSANDyp1cO76U0oqATRIJFYVvmPSSEcs4e5rGbjSHN0-sCIIsRKQThljziCz1SRRK5achFU7v-xZIOj8V7OwotTCpVLDN2uk6ogzLjHMPQPrtE2AI8S4UaClGnox4Wbsbhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048955&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of swapping selection marker of a plasmid by thermostable DNA ligase-mediated PPCP. The white boxed-region of the template plasmid (Vector 1) represents the original selection marker, which after the whole-plasmid PCR is replaced by a second selection marker built in the double-stranded PPCP primers (the black boxed-region in the primers and Vector 2). The procedure features the incorporation of a thermostable ligation step into the standard PCR procedure to ligate the nick remaining in the extended DNA strand, resulting in nick-free, circular plasmid DNA products. At the end of the PCR cycles, the template plasmid is eliminated by plating the plasmid-transformed host cells under the second selection pressure.</p></div></div><p>To demonstrate the PPCP and marker swapping concepts, we used a 2.44 kb plasmid, pHsh-<em>amp</em>, harbouring the ampicillin-resistant gene (<em>amp</em><sup>r</sup>) as the PPCP template. The PPCP primer pair was a linear 894 bp dsDNA fragment generated by standard PCR using pET-28a(+) as template as described in Materials and methods. The PPCP primer pair thus prepared harboured the kanamycin-resistant gene (<em>kan</em><sup>r</sup>) flanked by regions (39 bp each) complementary to the flanking sequences of <em>amp</em><sup>r</sup> in pHsh-<em>amp</em>. The PPCP primer pair was thus able to amplify the entire pHsh while simultaneously swapping the selection marker from Amp<sup>r</sup> to Kan<sup>r</sup> as illustrated in Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>.</p><p>PPCP was carried out with 25 cycles each consisting of denaturation of double-stranded template and DNA primer pair, annealing of primers to the respective complementary regions of the template extension of each primer over the template sequence, and ligation of the nick between the 3′ end of the newly extended strand to the 5′ end of the primer. The ligation step took advantage of the thermostable nature of <em>Tma</em> DNA ligase,<sup><a reveal-id="DST016C14" data-open="DST016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> allowing the enzyme to maintain its activity during multiple heating steps of PCR at 94°C and resulting in a nick-free, circular plasmid product ready to serve as a template in the subsequent PPCP cycles.</p><p>To verify that nick-free, circular DNA plasmids were obtained, the PPCP products were analysed by electrophoresis on 1.0% agarose gel. As shown in Fig. <a reveal-id="DST016F2" data-open="DST016F2" class="link link-reveal link-table xref-fig">2</a>, the double-stranded PPCP primer pair ran faster than the circular template pHsh DNA (lane 1). When <em>Tma</em> DNA ligase was not included in the amplification reaction, PCR DNA product ran slower than the circular plasmid DNA, which is presumed to be linear DNA product (lane 2). In contrast, with the inclusion of <em>Tma</em> DNA ligase in the amplification mixture (PPCP), the DNA product appears to be circular as it migrated to the same position as the circular template pHsh-<em>amp</em> DNA (lane 3), demonstrating that PPCP is capable of producing nick-free, circular DNA products. Furthermore, <em>E. coli</em> cells transformed by the PPCP amplification product were able to grow on LB plates containing kanamycin (data not shown), demonstrating successful swapping of the antibiotic selection marker. </p>                        <a id="5048960" scrollto-destination="5048960"></a>
<div id="DST016F2" data-id="DST016F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01602.jpeg?Expires=2147483647&Signature=mIOvKQh6-kWpFpKmwgwlonm4AvjgCAouq94dtD3f~~GZcBUpZqyvMrCgleAwuSEBE11UTHPLrRjN8zQ1IfBenVtsSF5Ue-wCB9VvrWZvOFoiEdn2SoU9uvJHwnBDva8ZdMG3leQL6ya6Y1XzolQyTS3wYQNmTRLtq-fRk4fv-S-0x8Zl3H1Mzqa4Qnvo~BK~4jKUYLtjnxng2qVYYZgilzoiTbvoTs6bOsSqbNXna39F8pXDKtEiJg7ma0YAxYjstnCeoN6RunxD~Ps-sZFKLLIVMj-z~suSogzCvBA9OCfaZH4SnvlDeeWNmc08c8OtML1bZVujyATd8n4zO7YfSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Agarose gel analysis of the thermostable DNA ligase-mediated PPCP using pHsh-amp as template. Lane M, DL5000 markers; Lane 1, control reaction containing template plasmid and primer but no Taq DNA polymerase or ligase; Lane 2, control reaction containing Taq DNA polymerase but no ligase; Lane 3, PPCP reaction containing both Taq DNA polymerase and Tma DNA ligase. The upper band in lane 2 is presumed to be linear PCR product. Incorporating the nick ligation step resulted in a nick-free, circular PPCP product (lane 3). The size of the PPCP primer pair was 962 bp." path-from-xml="dst01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048960/dst01602.jpeg" path-from-xml="dst01602.jpeg" target="_blank">View large</a><a section="5048960" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01602.jpeg?Expires=2147483647&Signature=TgTOy0S2Wh49UjgTYGKjGVIEaL3nU5hFlLU~JrXeKv3VoefVwRTvLRjceEHt2Zg6FCmtUjiiry4moPnCzbD2dKuRcmlbLTxovo1w2yr2voK0GJKqqlu22H5-Wz3EitUchw00Ve9QIRFfmQaF0LHY7yeaHH7TNBqilA88R5QIywXb9SvXNWt08Y4keSg3o5bebg9HlEND1J4ayhizz2pZ8VrxOTgj79DKd7UIoPLr74s6iT1gU0-j~zI6IhjJ7NGY4EtNBt1frm1iAE7b67BnpifWeDWltqgGlBlvw6QcapnibIG0lgr1yNQcqmXoBmb6s6wJui04bDoY~NImmR09-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048960&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Agarose gel analysis of the thermostable DNA ligase-mediated PPCP using pHsh-<em>amp</em> as template. Lane M, DL5000 markers; Lane 1, control reaction containing template plasmid and primer but no <em>Taq</em> DNA polymerase or ligase; Lane 2, control reaction containing <em>Taq</em> DNA polymerase but no ligase; Lane 3, PPCP reaction containing both <em>Taq</em> DNA polymerase and <em>Tma</em> DNA ligase. The upper band in lane 2 is presumed to be linear PCR product. Incorporating the nick ligation step resulted in a nick-free, circular PPCP product (lane 3). The size of the PPCP primer pair was 962 bp.</p></div></div><div content-id="DST016F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01602.jpeg?Expires=2147483647&Signature=mIOvKQh6-kWpFpKmwgwlonm4AvjgCAouq94dtD3f~~GZcBUpZqyvMrCgleAwuSEBE11UTHPLrRjN8zQ1IfBenVtsSF5Ue-wCB9VvrWZvOFoiEdn2SoU9uvJHwnBDva8ZdMG3leQL6ya6Y1XzolQyTS3wYQNmTRLtq-fRk4fv-S-0x8Zl3H1Mzqa4Qnvo~BK~4jKUYLtjnxng2qVYYZgilzoiTbvoTs6bOsSqbNXna39F8pXDKtEiJg7ma0YAxYjstnCeoN6RunxD~Ps-sZFKLLIVMj-z~suSogzCvBA9OCfaZH4SnvlDeeWNmc08c8OtML1bZVujyATd8n4zO7YfSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Agarose gel analysis of the thermostable DNA ligase-mediated PPCP using pHsh-amp as template. Lane M, DL5000 markers; Lane 1, control reaction containing template plasmid and primer but no Taq DNA polymerase or ligase; Lane 2, control reaction containing Taq DNA polymerase but no ligase; Lane 3, PPCP reaction containing both Taq DNA polymerase and Tma DNA ligase. The upper band in lane 2 is presumed to be linear PCR product. Incorporating the nick ligation step resulted in a nick-free, circular PPCP product (lane 3). The size of the PPCP primer pair was 962 bp." path-from-xml="dst01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048960/dst01602.jpeg" path-from-xml="dst01602.jpeg" target="_blank">View large</a><a section="5048960" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01602.jpeg?Expires=2147483647&Signature=TgTOy0S2Wh49UjgTYGKjGVIEaL3nU5hFlLU~JrXeKv3VoefVwRTvLRjceEHt2Zg6FCmtUjiiry4moPnCzbD2dKuRcmlbLTxovo1w2yr2voK0GJKqqlu22H5-Wz3EitUchw00Ve9QIRFfmQaF0LHY7yeaHH7TNBqilA88R5QIywXb9SvXNWt08Y4keSg3o5bebg9HlEND1J4ayhizz2pZ8VrxOTgj79DKd7UIoPLr74s6iT1gU0-j~zI6IhjJ7NGY4EtNBt1frm1iAE7b67BnpifWeDWltqgGlBlvw6QcapnibIG0lgr1yNQcqmXoBmb6s6wJui04bDoY~NImmR09-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048960&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Agarose gel analysis of the thermostable DNA ligase-mediated PPCP using pHsh-<em>amp</em> as template. Lane M, DL5000 markers; Lane 1, control reaction containing template plasmid and primer but no <em>Taq</em> DNA polymerase or ligase; Lane 2, control reaction containing <em>Taq</em> DNA polymerase but no ligase; Lane 3, PPCP reaction containing both <em>Taq</em> DNA polymerase and <em>Tma</em> DNA ligase. The upper band in lane 2 is presumed to be linear PCR product. Incorporating the nick ligation step resulted in a nick-free, circular PPCP product (lane 3). The size of the PPCP primer pair was 962 bp.</p></div></div><p>To demonstrate the important role of <em>Tma</em> DNA ligase in PPCP, we studied the effect of varying the amount of <em>Tma</em> DNA ligase on generating circular amplification product. We again observed that, without <em>Tma</em> DNA ligase, no circular DNA product was obtained (lane 1, Fig. <a reveal-id="DST016F3" data-open="DST016F3" class="link link-reveal link-table xref-fig">3</a>). In contrast, an increasing amount of circular DNA product was obtained with increasing amount of <em>Tma</em> DNA ligase added to the reaction mixture (lanes 2–5, Fig. <a reveal-id="DST016F3" data-open="DST016F3" class="link link-reveal link-table xref-fig">3</a>), indicating the need to determine the sufficient amount of DNA ligase required for optimal ligation to compensate for enzyme inactivation at high temperature during the denaturation step of the cycle. </p>                        <a id="5048962" scrollto-destination="5048962"></a>
<div id="DST016F3" data-id="DST016F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01603.jpeg?Expires=2147483647&Signature=ThTexIlyy6Hk9RneMSLkCqrL32EB8X1oXeR7ZWmbkh0ka8BUqcWWFHRvH1Px9Mj5i87c3q4n6fSCUs8JN9aYoIVNkXHsmqq095nB0AruFlLRMBPLMxwvf4fKdxIChc1Wn3LEFOnthLk1ArqhjssqPl77KHC00lYk31NzHlQhgx~GW-~Gw7gEnO8SBu7q30Nzymj1o7wrM0Tx8Bfn2hFtMPnxAcJh2lKAIfo2p6VzHD5ynQOyo3edeT~-jtEqeizhAFaJGowWP-xhYFzyxn7PdtmC2~cqYJ9nUunwGuFZIZJ~APyDLTcQ8lAW4ySUVaEqh~8RJzKxCdQ04HaxjCJeog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimization of thermostable DNA ligase-mediated PPCP by varying the amounts of Tma DNA ligase in 20 μl reaction. Lane M: λ EcoT14\xE2\x85markers; Lanes 1–5: 0, 0.06, 0.1, 0.2 and 0.3 μg of Tma DNA ligase, respectively. The PPCP long primer pair (2361 bp) was generated using pHshRev and pHshFwd as PCR primers, pHsh-kan as template, and Pyrobest DNA polymerase as described in Materials and methods. PPCP was carried out using pHsh-xynA1 as template and Taq DNA polymerase as described in Materials and methods. The unlabelled, upper band is presumed to be un-ligated, open circle plasmid as its amount diminished with the increasing amount of Tma ligase." path-from-xml="dst01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048962/dst01603.jpeg" path-from-xml="dst01603.jpeg" target="_blank">View large</a><a section="5048962" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01603.jpeg?Expires=2147483647&Signature=YW1kU-MITE4FuLYGYRKk8RRNK7V56M81MxmuVqmcMEdjcQMYEolfGmtom~eHLQ6S-ffWzqW9YL~ZISxGTIt4K-C9Mgg5Yk3syZlIi~8tHcmCK7vn8jVUosUhMboyV3elytgr8vfN0TVkx5PZ2uSICbeXYKUWiCAWVyPquFlBZqOQx~PpDpGova2TzeDI9NyigGZ5j3~Vdh-f5w1F1eCS115ob~5HouS8uLEk5lEtM0qoubcs92GCdN35UsETj0QN8mjxxo2zimvdJrLbvM3XnNEIWbsCfXKybhU4We9r7PGnfssp2rfKeKuKFqePMOjGyUzkg9riJ578yFblaOB0hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048962&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimization of thermostable DNA ligase-mediated PPCP by varying the amounts of <em>Tma</em> DNA ligase in 20 μl reaction. Lane M: λ EcoT14\xE2\x85markers; Lanes 1–5: 0, 0.06, 0.1, 0.2 and 0.3 μg of <em>Tma</em> DNA ligase, respectively. The PPCP long primer pair (2361 bp) was generated using pHshRev and pHshFwd as PCR primers, pHsh-<em>kan</em> as template, and Pyrobest DNA polymerase as described in Materials and methods. PPCP was carried out using pHsh-<em>xynA</em>1 as template and <em>Taq</em> DNA polymerase as described in Materials and methods. The unlabelled, upper band is presumed to be un-ligated, open circle plasmid as its amount diminished with the increasing amount of <em>Tma</em> ligase.</p></div></div><div content-id="DST016F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01603.jpeg?Expires=2147483647&Signature=ThTexIlyy6Hk9RneMSLkCqrL32EB8X1oXeR7ZWmbkh0ka8BUqcWWFHRvH1Px9Mj5i87c3q4n6fSCUs8JN9aYoIVNkXHsmqq095nB0AruFlLRMBPLMxwvf4fKdxIChc1Wn3LEFOnthLk1ArqhjssqPl77KHC00lYk31NzHlQhgx~GW-~Gw7gEnO8SBu7q30Nzymj1o7wrM0Tx8Bfn2hFtMPnxAcJh2lKAIfo2p6VzHD5ynQOyo3edeT~-jtEqeizhAFaJGowWP-xhYFzyxn7PdtmC2~cqYJ9nUunwGuFZIZJ~APyDLTcQ8lAW4ySUVaEqh~8RJzKxCdQ04HaxjCJeog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimization of thermostable DNA ligase-mediated PPCP by varying the amounts of Tma DNA ligase in 20 μl reaction. Lane M: λ EcoT14\xE2\x85markers; Lanes 1–5: 0, 0.06, 0.1, 0.2 and 0.3 μg of Tma DNA ligase, respectively. The PPCP long primer pair (2361 bp) was generated using pHshRev and pHshFwd as PCR primers, pHsh-kan as template, and Pyrobest DNA polymerase as described in Materials and methods. PPCP was carried out using pHsh-xynA1 as template and Taq DNA polymerase as described in Materials and methods. The unlabelled, upper band is presumed to be un-ligated, open circle plasmid as its amount diminished with the increasing amount of Tma ligase." path-from-xml="dst01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048962/dst01603.jpeg" path-from-xml="dst01603.jpeg" target="_blank">View large</a><a section="5048962" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01603.jpeg?Expires=2147483647&Signature=YW1kU-MITE4FuLYGYRKk8RRNK7V56M81MxmuVqmcMEdjcQMYEolfGmtom~eHLQ6S-ffWzqW9YL~ZISxGTIt4K-C9Mgg5Yk3syZlIi~8tHcmCK7vn8jVUosUhMboyV3elytgr8vfN0TVkx5PZ2uSICbeXYKUWiCAWVyPquFlBZqOQx~PpDpGova2TzeDI9NyigGZ5j3~Vdh-f5w1F1eCS115ob~5HouS8uLEk5lEtM0qoubcs92GCdN35UsETj0QN8mjxxo2zimvdJrLbvM3XnNEIWbsCfXKybhU4We9r7PGnfssp2rfKeKuKFqePMOjGyUzkg9riJ578yFblaOB0hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048962&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimization of thermostable DNA ligase-mediated PPCP by varying the amounts of <em>Tma</em> DNA ligase in 20 μl reaction. Lane M: λ EcoT14\xE2\x85markers; Lanes 1–5: 0, 0.06, 0.1, 0.2 and 0.3 μg of <em>Tma</em> DNA ligase, respectively. The PPCP long primer pair (2361 bp) was generated using pHshRev and pHshFwd as PCR primers, pHsh-<em>kan</em> as template, and Pyrobest DNA polymerase as described in Materials and methods. PPCP was carried out using pHsh-<em>xynA</em>1 as template and <em>Taq</em> DNA polymerase as described in Materials and methods. The unlabelled, upper band is presumed to be un-ligated, open circle plasmid as its amount diminished with the increasing amount of <em>Tma</em> ligase.</p></div></div>                            <h3 scrollto-destination=5048965 id="5048965" class="section-title" data-legacy-id=s3b>3.2. <em>in situ</em> error-prone PPCP for creating random mutagenesis libraries</h3>
<p>When <em>Taq</em> DNA polymerase is employed under error-prone conditions, e.g. in the presence of Mn<sup>2+</sup> (mutagenic PCR), PPCP can be used for generating random mutagenesis libraries. Since mutagenesis occurs <em>in situ</em> during the PCR reaction, eliminating the need of a separate process for generating or sub-cloning mutant DNA, we designated the procedure as <em>in situ</em> error-prone PPCP. In addition, since the extended DNA strand is ligated and serves as the template for the next round of amplification, mutations can accumulate over the subsequent PCR cycles. Furthermore, the target gene or gene segment can precisely be chosen by proper design of the long PPCP primer pair. Finally, the ability to swap a selection marker in PPCP facilitates the removal of the template plasmid during multiple rounds of mutagenesis. The <em>in situ</em> error-prone mutagenic PPCP can be coupled to a screening or selection procedure, facilitating directed evolution of the target gene or protein.</p><p>Figure <a reveal-id="DST016F4" data-open="DST016F4" class="link link-reveal link-table xref-fig">4</a> depicts the procedure of <em>in situ</em> error-prone PPCP. The target gene is cloned into the first vector carrying a selection marker, which serves as template. A second vector, which is the same as the first vector except for the selection marker and missing the DNA segment to be mutagenized, is linearized by regular PCR and serves as PPCP primer pair. </p>                        <a id="5048968" scrollto-destination="5048968"></a>
<div id="DST016F4" data-id="DST016F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01604.jpeg?Expires=2147483647&Signature=eimKXwQoxy~ogeVlfYeJZ6iVOLJnMYjCZQa2x2-CdqcTsRPFz3MwikfAyvjjsjyKeHS~IxYe5RGnuG1nVSbk7RS1qFy2reK6CZKl-T6RBU6uSxCYC19tKXzOe-3NOZh2YxBjA1ZAkUyv3YXD-LFcQ47e0XY28OM7kdaBBJ-I4XH1AnXFBUcyvKnZ~mFW1v3lzW02XckMHe33IBAVIUAeJ4h75~v2S3tW~nHoQNz9~Jk72En4pP6eZzJ5zYbi~j3mizIDzVbqF-A0sDGMSzG3XTxpyjfXLtjTYDMG4uMIlIv5hhWgLaLOMPiW9pJ8h9AT0jmyR6MEv4aNiwXSkqnWUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of directed evolution by in situ error-prone PCR using the thermostable DNA ligase-mediated PPCP method to create random mutagenesis libraries. (a) In situ error-prone PCR using PPCP. Primers (amplified from the vector with a swapped selection marker, e.g. from ampr to kanr by PPCP as described in the legend to Fig. 1) anneal to and cover the entire template plasmid sequence minus that of the target sequence (slashed boxed region); PPCP is performed as described in the legend to Fig. 1, except that the target DNA sequence is synthesized under error-prone conditions by using mutagenic DNA polymerase. (b) Transforming the circular PCR products into host cells and selecting for the second marker while screening mutants for desired mutation. (c and d) The process of (a) and (b) can be repeated for multiple rounds of PPCP mutagenesis, marker selection and functional screening." path-from-xml="dst01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048968/dst01604.jpeg" path-from-xml="dst01604.jpeg" target="_blank">View large</a><a section="5048968" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01604.jpeg?Expires=2147483647&Signature=z80ch5hgxA8t6uAgSF1YV66S5INN9ou9s~3Qsimd1GNlZTqvoL9uxjRgIpFSjLtgNli5QDUL0OHcyNl-vzKOKgvAF7WOxpqAwb1jsdSHmBFfTWqNkUMzR5lDfiuUI5J28WNNXSwgJ-oymiO2hNuMZNwup6Fmf6Kwr~5uTI2EUkqNv1vcL6ogZgZRRIS6mre3K-u8d0kd0LtjCOsyiYHocZg68fAbt7fqkJg3Yjkzbc1fJ8xO5tecP8OGT2HabAA1vPnvhHuVTlNMYzFpZSJBUf1v5GwpMG7~1VmH91GOFig0SvIUjoTx9j-NpCDu6VcUZG2Fr54ysVX2ostuPAy4Ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048968&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of directed evolution by <em>in situ</em> error-prone PCR using the thermostable DNA ligase-mediated PPCP method to create random mutagenesis libraries. (a) <em>In situ</em> error-prone PCR using PPCP. Primers (amplified from the vector with a swapped selection marker, e.g. from <em>amp<sup>r</sup></em> to <em>kan<sup>r</sup></em> by PPCP as described in the legend to Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>) anneal to and cover the entire template plasmid sequence minus that of the target sequence (slashed boxed region); PPCP is performed as described in the legend to Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>, except that the target DNA sequence is synthesized under error-prone conditions by using mutagenic DNA polymerase. (b) Transforming the circular PCR products into host cells and selecting for the second marker while screening mutants for desired mutation. (c and d) The process of (a) and (b) can be repeated for multiple rounds of PPCP mutagenesis, marker selection and functional screening.</p></div></div><div content-id="DST016F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01604.jpeg?Expires=2147483647&Signature=eimKXwQoxy~ogeVlfYeJZ6iVOLJnMYjCZQa2x2-CdqcTsRPFz3MwikfAyvjjsjyKeHS~IxYe5RGnuG1nVSbk7RS1qFy2reK6CZKl-T6RBU6uSxCYC19tKXzOe-3NOZh2YxBjA1ZAkUyv3YXD-LFcQ47e0XY28OM7kdaBBJ-I4XH1AnXFBUcyvKnZ~mFW1v3lzW02XckMHe33IBAVIUAeJ4h75~v2S3tW~nHoQNz9~Jk72En4pP6eZzJ5zYbi~j3mizIDzVbqF-A0sDGMSzG3XTxpyjfXLtjTYDMG4uMIlIv5hhWgLaLOMPiW9pJ8h9AT0jmyR6MEv4aNiwXSkqnWUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of directed evolution by in situ error-prone PCR using the thermostable DNA ligase-mediated PPCP method to create random mutagenesis libraries. (a) In situ error-prone PCR using PPCP. Primers (amplified from the vector with a swapped selection marker, e.g. from ampr to kanr by PPCP as described in the legend to Fig. 1) anneal to and cover the entire template plasmid sequence minus that of the target sequence (slashed boxed region); PPCP is performed as described in the legend to Fig. 1, except that the target DNA sequence is synthesized under error-prone conditions by using mutagenic DNA polymerase. (b) Transforming the circular PCR products into host cells and selecting for the second marker while screening mutants for desired mutation. (c and d) The process of (a) and (b) can be repeated for multiple rounds of PPCP mutagenesis, marker selection and functional screening." path-from-xml="dst01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048968/dst01604.jpeg" path-from-xml="dst01604.jpeg" target="_blank">View large</a><a section="5048968" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01604.jpeg?Expires=2147483647&Signature=z80ch5hgxA8t6uAgSF1YV66S5INN9ou9s~3Qsimd1GNlZTqvoL9uxjRgIpFSjLtgNli5QDUL0OHcyNl-vzKOKgvAF7WOxpqAwb1jsdSHmBFfTWqNkUMzR5lDfiuUI5J28WNNXSwgJ-oymiO2hNuMZNwup6Fmf6Kwr~5uTI2EUkqNv1vcL6ogZgZRRIS6mre3K-u8d0kd0LtjCOsyiYHocZg68fAbt7fqkJg3Yjkzbc1fJ8xO5tecP8OGT2HabAA1vPnvhHuVTlNMYzFpZSJBUf1v5GwpMG7~1VmH91GOFig0SvIUjoTx9j-NpCDu6VcUZG2Fr54ysVX2ostuPAy4Ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048968&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of directed evolution by <em>in situ</em> error-prone PCR using the thermostable DNA ligase-mediated PPCP method to create random mutagenesis libraries. (a) <em>In situ</em> error-prone PCR using PPCP. Primers (amplified from the vector with a swapped selection marker, e.g. from <em>amp<sup>r</sup></em> to <em>kan<sup>r</sup></em> by PPCP as described in the legend to Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>) anneal to and cover the entire template plasmid sequence minus that of the target sequence (slashed boxed region); PPCP is performed as described in the legend to Fig. <a reveal-id="DST016F1" data-open="DST016F1" class="link link-reveal link-table xref-fig">1</a>, except that the target DNA sequence is synthesized under error-prone conditions by using mutagenic DNA polymerase. (b) Transforming the circular PCR products into host cells and selecting for the second marker while screening mutants for desired mutation. (c and d) The process of (a) and (b) can be repeated for multiple rounds of PPCP mutagenesis, marker selection and functional screening.</p></div></div><p>To experimentally demonstrate the <em>in situ</em> error-prone PPCP concept, we chose a pair of identical expression vectors comprising different selection markers. Expression plasmid pHsh-<em>xynA</em>1 was constructed by cloning the xylanase gene (<em>xynA</em>1) from <em>T. lanuginosus</em> into vector pHsh-<em>amp</em> and used as the PPCP template. To generate the PPCP primer pair, a linear segment of pHsh-<em>kan</em> was amplified by regular PCR using pHshFwd and pHshRev PCR primers designed to amplify the entire pHsh-<em>kan</em> except for the multiple cloning sites. Error-prone PPCP using <em>Taq</em> DNA polymerase in the presence of Mn<sup>2+</sup>, <em>Tma</em> DNA ligase and the PPCP primer pair from pHsh-<em>kan</em> was performed over the <em>xynA</em>1 region of pHsh-<em>xynA</em>1 template (Fig. <a reveal-id="DST016F4" data-open="DST016F4" class="link link-reveal link-table xref-fig">4</a>).</p><p>The resulting PPCP products were analysed by agarose gel electrophoresis on 1.0% agarose gel (Fig. <a reveal-id="DST016F5" data-open="DST016F5" class="link link-reveal link-table xref-fig">5</a>A). Again, circular DNA molecules were observed only when <em>Tma</em> DNA ligase was added (lane 3). Transformants of circular PPCP products were obtained by selection on LB plates containing kanamycin. The xylanase activity of the transformants was determined by the xylan-overlay assay. The clear zones around colonies indicate active clones (Fig. <a reveal-id="DST016F5" data-open="DST016F5" class="link link-reveal link-table xref-fig">5</a>B). The size of the clear zone reflects the relative activity of the mutant while the lack of it indicates null mutation. </p>                        <a id="5048971" scrollto-destination="5048971"></a>
<div id="DST016F5" data-id="DST016F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01605.jpeg?Expires=2147483647&Signature=O2nRN28iJPjN7go83Swrt9Mv8o~vLnPqmDRavHLOy068WW0wwx2jfN771M88UuphRp9V3kyDur5GWVsxABk2JmKNSRATo7Ks4o5r8pIwNmuucP-4RyuWhkoxgzI5UNe68rcnwdpm2PCByaye~qTdCa81OmkJJBBxc7CB89HV9QfznVw6xeLuu751RXZBclv7apSLkakJgC~ZEY0GeDPLbInfWFMckg~ThmYhUGJwx0WE~qbzOT9B9PaWOCQUn7msbVePlD8GA3medRn70WzDw~tc35~pV~6ScmH4unXWHgV2FxFCNw~fMHVFSJ-10fRaeKWHveBs6NkB0fScIESOpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Directed evolution of xylanse and cellulase as examples of in situ error-prone mutagenic PCR mutagenesis using thermostable DNA ligase-mediated PPCP. (A) Agarose gel analysis of the PPCP mutagenesis products using pHsh-xynA1 as template and the PPCP primer pair from pHsh-kan. Lanes: M, λ EcoT14\xE2\x85 markers; 1, control reaction containing no Taq DNA polymerase or Tma ligase; 2, control reaction containing Taq DNA polymerase but no Tma ligase; 3, PCR containing both Taq DNA polymerase and Tma ligase. (B) Functional screening of E. coli transformants for xylanase activity using xylan-overlay assay as described in Materials and methods. Positive clones were identified by clear zones surrounding xylanase-expressing colonies (arrows). The mutants were derived from pHsh-xynA1. The size of xynA1 was 591 bp. (C) Functional screening of E. coli transformants for cellulase activity by CMC-overlay assay as described in Materials and methods. Positive clones were identified by depressed area surrounding cellulase-expressing colonies (arrows). The mutants shown were derived from pHsh-celB." path-from-xml="dst01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048971/dst01605.jpeg" path-from-xml="dst01605.jpeg" target="_blank">View large</a><a section="5048971" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01605.jpeg?Expires=2147483647&Signature=DUyQbch-JySJWxiV1edvXq4LM8dhnMglTaXAZ3WSkCy3dB0qvN2vXg1TRgwuhSgAQEWCsoIcMt7aRrSOwrw68pYbxkCZAb0sjQWopGEMzqh7pmHhW7nyRMFAVbUqsbMmqUny9wgAB4pFnfFwRle--LwiPHDNE2LLO-sc46yJRdJvMgj1uAHutCqcl7b-eXYBGqSNRtIgs2oE7EnXef97e9oHxudZGnXUlwOHnqq1e-GMIHsCuUa1NJBf3jTVrnAHgZWM7sFQ8KzjLwYkMJupJbZiwe3B6iqFFtpFKZO-wKeXJFbJ8jtYl6OljqWEE2OKzgTBMLtXQ4HZblUTRN~bnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048971&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Directed evolution of xylanse and cellulase as examples of <em>in situ</em> error-prone mutagenic PCR mutagenesis using thermostable DNA ligase-mediated PPCP. (A) Agarose gel analysis of the PPCP mutagenesis products using pHsh-<em>xynA</em>1 as template and the PPCP primer pair from pHsh-<em>kan</em>. Lanes: M, λ EcoT14\xE2\x85 markers; 1, control reaction containing no <em>Taq</em> DNA polymerase or <em>Tma</em> ligase; 2, control reaction containing <em>Taq</em> DNA polymerase but no <em>Tma</em> ligase; 3, PCR containing both <em>Taq</em> DNA polymerase and <em>Tma</em> ligase. (B) Functional screening of <em>E. coli</em> transformants for xylanase activity using xylan-overlay assay as described in Materials and methods. Positive clones were identified by clear zones surrounding xylanase-expressing colonies (arrows). The mutants were derived from pHsh-<em>xynA</em>1. The size of <em>xynA</em>1 was 591 bp. (C) Functional screening of <em>E. coli</em> transformants for cellulase activity by CMC-overlay assay as described in Materials and methods. Positive clones were identified by depressed area surrounding cellulase-expressing colonies (arrows). The mutants shown were derived from pHsh-<em>celB</em>.</p></div></div><div content-id="DST016F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/m_dst01605.jpeg?Expires=2147483647&Signature=O2nRN28iJPjN7go83Swrt9Mv8o~vLnPqmDRavHLOy068WW0wwx2jfN771M88UuphRp9V3kyDur5GWVsxABk2JmKNSRATo7Ks4o5r8pIwNmuucP-4RyuWhkoxgzI5UNe68rcnwdpm2PCByaye~qTdCa81OmkJJBBxc7CB89HV9QfznVw6xeLuu751RXZBclv7apSLkakJgC~ZEY0GeDPLbInfWFMckg~ThmYhUGJwx0WE~qbzOT9B9PaWOCQUn7msbVePlD8GA3medRn70WzDw~tc35~pV~6ScmH4unXWHgV2FxFCNw~fMHVFSJ-10fRaeKWHveBs6NkB0fScIESOpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Directed evolution of xylanse and cellulase as examples of in situ error-prone mutagenic PCR mutagenesis using thermostable DNA ligase-mediated PPCP. (A) Agarose gel analysis of the PPCP mutagenesis products using pHsh-xynA1 as template and the PPCP primer pair from pHsh-kan. Lanes: M, λ EcoT14\xE2\x85 markers; 1, control reaction containing no Taq DNA polymerase or Tma ligase; 2, control reaction containing Taq DNA polymerase but no Tma ligase; 3, PCR containing both Taq DNA polymerase and Tma ligase. (B) Functional screening of E. coli transformants for xylanase activity using xylan-overlay assay as described in Materials and methods. Positive clones were identified by clear zones surrounding xylanase-expressing colonies (arrows). The mutants were derived from pHsh-xynA1. The size of xynA1 was 591 bp. (C) Functional screening of E. coli transformants for cellulase activity by CMC-overlay assay as described in Materials and methods. Positive clones were identified by depressed area surrounding cellulase-expressing colonies (arrows). The mutants shown were derived from pHsh-celB." path-from-xml="dst01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5048971/dst01605.jpeg" path-from-xml="dst01605.jpeg" target="_blank">View large</a><a section="5048971" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst016/2/dst01605.jpeg?Expires=2147483647&Signature=DUyQbch-JySJWxiV1edvXq4LM8dhnMglTaXAZ3WSkCy3dB0qvN2vXg1TRgwuhSgAQEWCsoIcMt7aRrSOwrw68pYbxkCZAb0sjQWopGEMzqh7pmHhW7nyRMFAVbUqsbMmqUny9wgAB4pFnfFwRle--LwiPHDNE2LLO-sc46yJRdJvMgj1uAHutCqcl7b-eXYBGqSNRtIgs2oE7EnXef97e9oHxudZGnXUlwOHnqq1e-GMIHsCuUa1NJBf3jTVrnAHgZWM7sFQ8KzjLwYkMJupJbZiwe3B6iqFFtpFKZO-wKeXJFbJ8jtYl6OljqWEE2OKzgTBMLtXQ4HZblUTRN~bnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5048971&ar=354164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Directed evolution of xylanse and cellulase as examples of <em>in situ</em> error-prone mutagenic PCR mutagenesis using thermostable DNA ligase-mediated PPCP. (A) Agarose gel analysis of the PPCP mutagenesis products using pHsh-<em>xynA</em>1 as template and the PPCP primer pair from pHsh-<em>kan</em>. Lanes: M, λ EcoT14\xE2\x85 markers; 1, control reaction containing no <em>Taq</em> DNA polymerase or <em>Tma</em> ligase; 2, control reaction containing <em>Taq</em> DNA polymerase but no <em>Tma</em> ligase; 3, PCR containing both <em>Taq</em> DNA polymerase and <em>Tma</em> ligase. (B) Functional screening of <em>E. coli</em> transformants for xylanase activity using xylan-overlay assay as described in Materials and methods. Positive clones were identified by clear zones surrounding xylanase-expressing colonies (arrows). The mutants were derived from pHsh-<em>xynA</em>1. The size of <em>xynA</em>1 was 591 bp. (C) Functional screening of <em>E. coli</em> transformants for cellulase activity by CMC-overlay assay as described in Materials and methods. Positive clones were identified by depressed area surrounding cellulase-expressing colonies (arrows). The mutants shown were derived from pHsh-<em>celB</em>.</p></div></div><p>Using the xylan-overlay assay, we were able to determine the frequency of the null mutation as a function of Mn<sup>2+</sup> concentration and the number of thermal cycles. The results showed that the frequency of null mutations increased with the concentration of Mn<sup>2+</sup> and the cycle number of error-prone PPCP within the ranges tested (Table <a reveal-id="DST016TB1" data-open="DST016TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="5048973" scrollto-destination="5048973"></a>
<div content-id="DST016TB1" class="table-modal"><div class="table-wrap"><div id="DST016TB1" class="table-wrap-title" data-id="DST016TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Effects of Mn<sup>2+</sup> concentration and PPCP cycle number on null mutation frequency of xylanase</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>In situ</em> error-prone PPCP </th><th>Total number of transformants </th><th>Frequency of the null mutant clones (%) </th></tr></thead><tbody><tr><td colspan="3">Mn<sup>2+</sup> concentration (mM)<sup>a</sup> </td></tr><tr><td> 0 </td><td>446 </td><td>15 </td></tr><tr><td> 0.1 </td><td>509 </td><td>58 </td></tr><tr><td> 0.3 </td><td>461 </td><td>76 </td></tr><tr><td> 0.5 </td><td>561 </td><td>89 </td></tr><tr><td colspan="3">Number of cycles<sup>b</sup> </td></tr><tr><td> 15 </td><td>471 </td><td>59 </td></tr><tr><td> 25 </td><td>1028 </td><td>85 </td></tr></tbody></table></div><div content-id="DST016TB1" class="table-modal"><table><thead name="thead" align=""><tr><th><em>In situ</em> error-prone PPCP </th><th>Total number of transformants </th><th>Frequency of the null mutant clones (%) </th></tr></thead><tbody><tr><td colspan="3">Mn<sup>2+</sup> concentration (mM)<sup>a</sup> </td></tr><tr><td> 0 </td><td>446 </td><td>15 </td></tr><tr><td> 0.1 </td><td>509 </td><td>58 </td></tr><tr><td> 0.3 </td><td>461 </td><td>76 </td></tr><tr><td> 0.5 </td><td>561 </td><td>89 </td></tr><tr><td colspan="3">Number of cycles<sup>b</sup> </td></tr><tr><td> 15 </td><td>471 </td><td>59 </td></tr><tr><td> 25 </td><td>1028 </td><td>85 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>15 PCR cycles at various Mn<sup>2+</sup> concentration.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>0.1 mM Mn<sup>2+</sup> concentration with various number of PCR cycles.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5048973" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST016TB1" class="table-wrap-title" data-id="DST016TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Effects of Mn<sup>2+</sup> concentration and PPCP cycle number on null mutation frequency of xylanase</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>In situ</em> error-prone PPCP </th><th>Total number of transformants </th><th>Frequency of the null mutant clones (%) </th></tr></thead><tbody><tr><td colspan="3">Mn<sup>2+</sup> concentration (mM)<sup>a</sup> </td></tr><tr><td> 0 </td><td>446 </td><td>15 </td></tr><tr><td> 0.1 </td><td>509 </td><td>58 </td></tr><tr><td> 0.3 </td><td>461 </td><td>76 </td></tr><tr><td> 0.5 </td><td>561 </td><td>89 </td></tr><tr><td colspan="3">Number of cycles<sup>b</sup> </td></tr><tr><td> 15 </td><td>471 </td><td>59 </td></tr><tr><td> 25 </td><td>1028 </td><td>85 </td></tr></tbody></table></div><div content-id="DST016TB1" class="table-modal"><table><thead name="thead" align=""><tr><th><em>In situ</em> error-prone PPCP </th><th>Total number of transformants </th><th>Frequency of the null mutant clones (%) </th></tr></thead><tbody><tr><td colspan="3">Mn<sup>2+</sup> concentration (mM)<sup>a</sup> </td></tr><tr><td> 0 </td><td>446 </td><td>15 </td></tr><tr><td> 0.1 </td><td>509 </td><td>58 </td></tr><tr><td> 0.3 </td><td>461 </td><td>76 </td></tr><tr><td> 0.5 </td><td>561 </td><td>89 </td></tr><tr><td colspan="3">Number of cycles<sup>b</sup> </td></tr><tr><td> 15 </td><td>471 </td><td>59 </td></tr><tr><td> 25 </td><td>1028 </td><td>85 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>15 PCR cycles at various Mn<sup>2+</sup> concentration.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>0.1 mM Mn<sup>2+</sup> concentration with various number of PCR cycles.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5048973" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5048974 id="5048974" class="section-title" data-legacy-id=s3c>3.3. Directed evolution of cellulase genes by <em>in situ</em> error-prone PPCP mutagenesis</h3>
<p>We used <em>in situ</em> error-prone PPCP to improve the activities of two cellulases from <em>T. maritima</em>. The <em>T. maritima celA</em> and <em>celB</em> genes were individually cloned into expression vector pHsh-<em>amp</em> to generate pHsh-<em>celA</em> and pHsh-<em>celB</em> as described in Materials and methods. With each of these two plasmids serving as template, the same primer pair from pHsh-<em>kan</em> described above was used to construct random mutagenesis libraries by <em>in situ</em> error-prone PPCP. The amplification was performed over <em>celA</em> and <em>celB</em> using <em>Taq</em> DNA polymerase in the presence of 0.1 mM Mn<sup>2+</sup> and <em>Tma</em> DNA ligase. The amplification products were transformed into competent <em>E. coli</em> cells, and the mutant libraries screened on the LB plates containing kanamycin and CMC. Colonies producing larger depressed zones than wild-type transformants were identified as positive mutant clones displaying higher cellulase activities (see Fig. <a reveal-id="DST016F5" data-open="DST016F5" class="link link-reveal link-table xref-fig">5</a>C showing plate of <em>celB</em> mutants).</p><p>Approximately 2000 <em>celA</em> and <em>celB</em> clones were screened. The positive mutants were further cultured at 30°C and then 42°C. A positive mutant of <em>celA</em>, designated as <em>celA</em>-m1, exhibited 4.4 U/ml cellulase activity while the wild-type strain exhibited 2.9 U/ml activity when cells were grown under the same culture condition to the same cell density (OD<sub>600</sub> = 3.2). DNA sequence analysis revealed that <em>celA</em>-m1 contained five-point mutations, but only one non-synonymous codon change that resulted in the substitution of serine (TCC) for phenylalanine (TTC) at residue 102 (F102S). On the other hand, the synonymous (silent) mutations occurred at amino acid positions L203 (CTT → CTA), L222 (CTT → CTC), G234 (GGA → GGT) and A241 (GCA → GCT). A positive mutant of <em>celB</em>, designated as <em>celB-</em>m1, produced cellulase activity of 3.88 U/ml while the wild-type strain produced 0.89 U/ml under the same culture conditions. DNA sequence analysis revealed a single-point alteration at codon 61, H61L (CAT → CTT). These results confirm the <em>in situ</em> error-prone PPCP procedure in being able to rapidly mutate a target gene.</p>                            <h2 scrollto-destination=5048977 id="5048977" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p><em>Tma</em> DNA ligase from the hyperthermophilic bacterium <em>T. maritima</em> has a half-life of over 30 min at 95°C and thus is an ideal DNA ligase for PCRs.<sup><a reveal-id="DST016C14" data-open="DST016C14" class="link link-ref link-reveal xref-bibr">14</a></sup><em>Tma</em> DNA ligase requires NAD<sup>+</sup>, and a divalent cation such as Mg<sup>2+</sup>, Mn<sup>2+</sup> or Ca<sup>2+</sup>. Therefore, the ligase activity is enhanced when Mn<sup>2+</sup> is added to the reaction mixture, which has an added benefit of increasing the mutation rate of error-prone PCR. Our finding that Mn<sup>2+</sup> increased the rate of null mutation (Table <a reveal-id="DST016TB1" data-open="DST016TB1" class="link link-reveal link-table xref-fig">1</a>) is consistent with the error rate of <em>Taq</em> polymerase being dependent on the concentration of the cation. It is of note that <em>Tma</em> DNA ligase exhibits activity on DNA fragments with cohesive termini, but not on blunt-ended DNA molecules. Thus the ligase is unable to ligate the double-stranded PPCP primer pairs used in the reaction.</p><p>In conventional mutagenic PCR employing <em>Taq</em> DNA polymerase under error-prone conditions, the PCR products have single-base overhangs at the 3′ end of each strand,<sup><a reveal-id="DST016C19" data-open="DST016C19" class="link link-ref link-reveal xref-bibr">19</a></sup> which often needs to be removed or filled-in before cloning. In addition, mutant DNA fragments must be digested and ligated to a vector to obtain a mutant library.<sup><a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST016C20  DST016C22" data-open="DST016C20  DST016C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup> Low efficiency of the digestion and ligation steps is a common challenge, and the library is often plagued with plasmids with no or multiple inserts.<sup><a reveal-id="DST016C23" data-open="DST016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In the <em>in situ</em> error-prone PPCP method described here, thermostable DNA ligase repairs the nick, forming a nick-free, circular plasmid and circumventing many of the issues associated with conventional mutagenic PCR.</p><p>The double-stranded PPCP primer pair can be synthesized by conventional PCR. Specifically, a pair of conventional PCR primers can be designed to amply the corresponding region of an expression vector, which comprises the central region harbouring the selection marker, and the flanking regions of the cloning site. This dsDNA PCR product can be used as the primer pair in subsequent <em>in situ</em> error-prone mutagenic PPCP. For the experiments reported in this work, the primer pairs were amplified by a high-fidelity polymerase such as <em>Pyrobest</em> or <em>Pfu</em> DNA. The primer pair should be phosphorylated at 5′ ends for the ligase reaction.</p><p>PPCP primer strands are complementary to each other, thus self-annealing of the primer strands may compete with the annealing between the primer strands and the DNA template, decreasing the efficiency of the process. This problem can be alleviated by optimizing the ratio of primer pair to template DNA. Another important parameter is the temperature setting for the ligation step. Although DNA polymerase and DNA ligase have the maximal activities at 72 and 60°C, respectively, both enzymes exhibit sufficient activity within the range of 60–72°C. Even though we found that setting ligation temperature at 60°C works well for the purpose, it is likely that annealing, elongation and ligation all occur at all of the three temperature settings albeit to a different degree of activity.</p><p>In contrast to the conventional mutagenic PCR protocols, the PPCP method eliminates background transformants including those without the insert or target sequence that reduce the efficiency in library screening.<sup><a reveal-id="DST016C24" data-open="DST016C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Thus the need to run vector-only control to gauge the level of background resulting from vector self-ligation is eliminated.</p><p>In contrast to the whole-plasmid PCR of the MegaWHOP procedure, the PPCP method reported in this work facilitates exponential amplification. In the PPCP procedure, a newly extended DNA strand is ligated to form a circular DNA, which serves as a new template in the next PCR cycle. Furthermore, the product mixture of MegaWHOP must be digested by a <em>dam</em>-methylated DNA-specific restriction enzyme, such as <em>Dpn</em>I, to reduce the template plasmid.<sup><a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST016C25" data-open="DST016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In the <em>in situ</em> error-prone mutagenic PPCP method reported in this work, the use of the primer pair harbouring a selection marker, e.g. a drug resistance gene that is different from that of the template plasmid, allows marker swapping that completely eliminates the growth of the transformants of template plasmids without the need of <em>Dpn</em>I digestion. Thus no further treatment is needed for the PPCP products, which can be directly transformed into host cells. The differences between MegaWHOP and PPCP are summarized in Table <a reveal-id="DST016TB2" data-open="DST016TB2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="5048986" scrollto-destination="5048986"></a>
<div content-id="DST016TB2" class="table-modal"><div class="table-wrap"><div id="DST016TB2" class="table-wrap-title" data-id="DST016TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Comparison of thermostable DNA ligase-mediated PPCP with MegaWHOP</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Primer pair </th><th>Extension region </th><th>PCR condition </th><th>Products </th><th>Template removing </th><th>Exponential amplification </th><th>Multiple round mutagenesis </th><th>Ref. </th></tr></thead><tbody><tr><td>MegaWHOP </td><td>Mutated target gene (0.2–1 kb) </td><td>Vector sequence </td><td>High fidelity </td><td>Linear ssDNA and their hybrids (nicked dsDNA) </td><td><em>Dpn</em>I digestion </td><td>No </td><td>No </td><td><a reveal-id="DST016C12" data-open="DST016C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a> </td></tr><tr><td>PPCP </td><td>Vector sequence </td><td>Target gene </td><td>High fidelity or error-prone </td><td>Nick-free, circular dsDNA </td><td>Plate selection </td><td>Yes </td><td>Yes </td><td>This work </td></tr></tbody></table></div><div content-id="DST016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Primer pair </th><th>Extension region </th><th>PCR condition </th><th>Products </th><th>Template removing </th><th>Exponential amplification </th><th>Multiple round mutagenesis </th><th>Ref. </th></tr></thead><tbody><tr><td>MegaWHOP </td><td>Mutated target gene (0.2–1 kb) </td><td>Vector sequence </td><td>High fidelity </td><td>Linear ssDNA and their hybrids (nicked dsDNA) </td><td><em>Dpn</em>I digestion </td><td>No </td><td>No </td><td><a reveal-id="DST016C12" data-open="DST016C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a> </td></tr><tr><td>PPCP </td><td>Vector sequence </td><td>Target gene </td><td>High fidelity or error-prone </td><td>Nick-free, circular dsDNA </td><td>Plate selection </td><td>Yes </td><td>Yes </td><td>This work </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5048986" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST016TB2" class="table-wrap-title" data-id="DST016TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Comparison of thermostable DNA ligase-mediated PPCP with MegaWHOP</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Primer pair </th><th>Extension region </th><th>PCR condition </th><th>Products </th><th>Template removing </th><th>Exponential amplification </th><th>Multiple round mutagenesis </th><th>Ref. </th></tr></thead><tbody><tr><td>MegaWHOP </td><td>Mutated target gene (0.2–1 kb) </td><td>Vector sequence </td><td>High fidelity </td><td>Linear ssDNA and their hybrids (nicked dsDNA) </td><td><em>Dpn</em>I digestion </td><td>No </td><td>No </td><td><a reveal-id="DST016C12" data-open="DST016C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a> </td></tr><tr><td>PPCP </td><td>Vector sequence </td><td>Target gene </td><td>High fidelity or error-prone </td><td>Nick-free, circular dsDNA </td><td>Plate selection </td><td>Yes </td><td>Yes </td><td>This work </td></tr></tbody></table></div><div content-id="DST016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Primer pair </th><th>Extension region </th><th>PCR condition </th><th>Products </th><th>Template removing </th><th>Exponential amplification </th><th>Multiple round mutagenesis </th><th>Ref. </th></tr></thead><tbody><tr><td>MegaWHOP </td><td>Mutated target gene (0.2–1 kb) </td><td>Vector sequence </td><td>High fidelity </td><td>Linear ssDNA and their hybrids (nicked dsDNA) </td><td><em>Dpn</em>I digestion </td><td>No </td><td>No </td><td><a reveal-id="DST016C12" data-open="DST016C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST016C13" data-open="DST016C13" class="link link-ref link-reveal xref-bibr">13</a> </td></tr><tr><td>PPCP </td><td>Vector sequence </td><td>Target gene </td><td>High fidelity or error-prone </td><td>Nick-free, circular dsDNA </td><td>Plate selection </td><td>Yes </td><td>Yes </td><td>This work </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5048986" target="_blank">View Large</a></div></div><p>In PPCP, a generic primer pair can be employed for different target genes cloned into the multi-cloning sites of an expression plasmid vector. Such generic primer pair contains the entire vector sequence but missing the target gene in the cloning site (Fig. <a reveal-id="DST016F4" data-open="DST016F4" class="link link-reveal link-table xref-fig">4</a>). A second pair of primers containing a different selection marker can be similarly prepared so that multiple rounds of ‘error-prone PPCP – transformation – selection’ process can be conveniently performed over a target gene for directed evolution (Fig. <a reveal-id="DST016F4" data-open="DST016F4" class="link link-reveal link-table xref-fig">4</a>).</p><p>In conclusion, we demonstrated that a thermostable DNA ligase can be employed to mediate the amplification of circular DNA by adding a ligation step to conventional PCR cycles resulting in nick-free, circular plasmid DNA PCR products. We further showed that PPCP can be used to swap the selection marker of the template plasmid negating the need to pretreat the PCR DNA with <em>Dpn</em>I. Finally, when an error-prone DNA polymerase is used, PPCP can be used for <em>in situ</em> mutagenesis to create random mutagenesis libraries (<em>in situ</em> error-prone mutagenic PPCP). The method offers the following advantages: (i) nick-free, circular PCR products can be directly transformed into competent <em>E. coli</em> cells and easily selected by using an alternative antibiotic; (ii) a mutant library can be created by one-step error-prone amplification of variable DNA region in a plasmid and (iii) accumulation of positive mutations in one sequence can be obtained by multiple rounds of <em>in situ</em> error-prone mutagenic PPCP. This method offers a novel, convenient and efficient approach for improving genes and proteins through directed evolution. In this work, we showed how biomass-degrading enzymes can be improved by mutagenic PPCP for the development of green industries.</p>                            <h2 scrollto-destination=5048989 id="5048989" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by grants from the National Natural Science Foundation of China (31170027), PAPD of Jiangsu Higher Education Institutions, Nanjing Program of Science and Technology (201101094), the Natural Science Foundation of the Jiangsu Higher Education Institutions of China (12KJB180002) and Zhenjiang Agricultural Science and Technology support program (NY2012032). J.H.D.W. acknowledges the supports of a grant (DEFG02-94ER20155) from the Office of Science (BES) and a grant (DE-FG02-08ER64692) from the Office of Science (BER), the US Department of Energy.</p>                            <h2 scrollto-destination=5048991 id="5048991" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST016C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chirumamilla</div> <div class="given-names">R.R.</div></div>,  <div class="name"><div class="surname">Muralidhar</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Marchant</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nigam</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Improving the quality of industrially important enzymes by directed evolution</div>, <div class="source">Mol. Cell Biochem.</div> , <div class="year">2001</div>, vol. <div class="volume">224</div> (pg. <div class="fpage">159</div>-<div class="lpage">68</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1011904405002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1011904405002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20quality%20of%20industrially%20important%20enzymes%20by%20directed%20evolution&amp;author=R.R.%20Chirumamilla&amp;author=R.%20Muralidhar&amp;author=R.%20Marchant&amp;author=P.%20Nigam&amp;publication_year=2001&amp;journal=Mol.%20Cell%20Biochem.&amp;volume=224&amp;pages=159-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1011904405002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1011904405002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1011904405002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11693193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cherry</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Fidantsef</div> <div class="given-names">A.L.</div></div>. </div><div class="article-title">Directed evolution of industrial enzymes: an update</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">438</div>-<div class="lpage">43</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0958-1669(03)00099-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0958-1669(03)00099-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Directed%20evolution%20of%20industrial%20enzymes%3A%20an%20update&amp;author=J.R.%20Cherry&amp;author=A.L.%20Fidantsef&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=14&amp;pages=438-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(03)00099-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(03)00099-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(03)00099-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12943855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eijsink</div> <div class="given-names">V.G.H.</div></div>,  <div class="name"><div class="surname">Gaseidnes</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Borchert</div> <div class="given-names">T.V.</div></div>,  <div class="name"><div class="surname">van den Burg</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Directed evolution of enzyme stability</div>, <div class="source">Biomol. Eng.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">21</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bioeng.2004.12.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bioeng.2004.12.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Directed%20evolution%20of%20enzyme%20stability&amp;author=V.G.H.%20Eijsink&amp;author=S.%20Gaseidnes&amp;author=T.V.%20Borchert&amp;author=B.%20van%20den%20Burg&amp;publication_year=2005&amp;journal=Biomol.%20Eng.&amp;volume=22&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bioeng.2004.12.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bioeng.2004.12.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bioeng.2004.12.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15857780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yuan</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kurek</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">English</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Keenan</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Laboratory-directed protein evolution</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">2005</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">373</div>-<div class="lpage">92</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/MMBR.69.3.373-392.2005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/MMBR.69.3.373-392.2005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Laboratory-directed%20protein%20evolution&amp;author=L.%20Yuan&amp;author=I.%20Kurek&amp;author=J.%20English&amp;author=R.%20Keenan&amp;publication_year=2005&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=69&amp;pages=373-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.69.3.373-392.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.69.3.373-392.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.69.3.373-392.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16148303" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Otten</div> <div class="given-names">L.G.</div></div>,  <div class="name"><div class="surname">Quax</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">Directed evolution: selecting today's biocatalysts</div>, <div class="source">Biomol. Eng.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bioeng.2005.02.002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bioeng.2005.02.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Directed%20evolution%3A%20selecting%20today's%20biocatalysts&amp;author=L.G.%20Otten&amp;author=W.J.%20Quax&amp;publication_year=2005&amp;journal=Biomol.%20Eng.&amp;volume=22&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bioeng.2005.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bioeng.2005.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bioeng.2005.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15857778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olsen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Iverson</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Georgiou</div> <div class="given-names">G.</div></div>. </div><div class="article-title">High-throughput screening of enzyme libraries</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2000</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">331</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0958-1669(00)00108-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0958-1669(00)00108-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20screening%20of%20enzyme%20libraries&amp;author=M.%20Olsen&amp;author=B.%20Iverson&amp;author=G.%20Georgiou&amp;publication_year=2000&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=11&amp;pages=331-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(00)00108-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(00)00108-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(00)00108-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10975452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lutz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Patrick</div> <div class="given-names">W.M.</div></div>. </div><div class="article-title">Novel methods for directed evolution of enzymes: quality, not quantity</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">291</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.copbio.2004.05.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.copbio.2004.05.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20methods%20for%20directed%20evolution%20of%20enzymes%3A%20quality%2C%20not%20quantity&amp;author=S.%20Lutz&amp;author=W.M.%20Patrick&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=15&amp;pages=291-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.copbio.2004.05.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.copbio.2004.05.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.copbio.2004.05.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15296927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ling</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Robinson</div> <div class="given-names">B.H.</div></div>. </div><div class="article-title">Approaches to DNA mutagenesis: an overview</div>, <div class="source">Anal. Biochem.</div> , <div class="year">1997</div>, vol. <div class="volume">254</div> (pg. <div class="fpage">157</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1006/abio.1997.2428" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1006/abio.1997.2428</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Approaches%20to%20DNA%20mutagenesis%3A%20an%20overview&amp;author=M.M.%20Ling&amp;author=B.H.%20Robinson&amp;publication_year=1997&amp;journal=Anal.%20Biochem.&amp;volume=254&amp;pages=157-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/abio.1997.2428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fabio.1997.2428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fabio.1997.2428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9417773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cadwell</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Joyce</div> <div class="given-names">G.F.</div></div>. </div><div class="article-title">Randomization of genes by PCR mutagenesis</div>, <div class="source">PCR Methods Appl.</div> , <div class="year">1992</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">28</div>-<div class="lpage">33</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2.1.28" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2.1.28</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Randomization%20of%20genes%20by%20PCR%20mutagenesis&amp;author=R.C.%20Cadwell&amp;author=G.F.%20Joyce&amp;publication_year=1992&amp;journal=PCR%20Methods%20Appl.&amp;volume=2&amp;pages=28-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2.1.28" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2.1.28" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2.1.28"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1490172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hemsley</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Arnheim</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Toney</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Cortopassi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Galas</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">A simple method for site-directed mutagenesis using the polymerase chain reaction</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1989</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">6545</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/17.16.6545" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/17.16.6545</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20site-directed%20mutagenesis%20using%20the%20polymerase%20chain%20reaction&amp;author=A.%20Hemsley&amp;author=N.%20Arnheim&amp;author=M.D.%20Toney&amp;author=G.%20Cortopassi&amp;author=D.J.%20Galas&amp;publication_year=1989&amp;journal=Nucleic%20Acids%20Res.&amp;volume=17&amp;pages=6545-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/17.16.6545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F17.16.6545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F17.16.6545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2674899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsumura</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Rowe</div> <div class="given-names">L.A.</div></div>. </div><div class="article-title">Whole plasmid mutagenic PCR for directed protein evolution</div>, <div class="source">Biomol. Eng.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">73</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bioeng.2004.10.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bioeng.2004.10.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20plasmid%20mutagenic%20PCR%20for%20directed%20protein%20evolution&amp;author=I.%20Matsumura&amp;author=L.A.%20Rowe&amp;publication_year=2005&amp;journal=Biomol.%20Eng.&amp;volume=22&amp;pages=73-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bioeng.2004.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bioeng.2004.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bioeng.2004.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15857786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyazaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">MEGAWHOP cloning: a method of creating random mutagenesis libraries via megaprimer PCR of whole plasmids</div>, <div class="source">Meth. Enzymol.</div> , <div class="year">2011</div>, vol. <div class="volume">498</div> (pg. <div class="fpage">399</div>-<div class="lpage">406</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/B978-0-12-385120-8.00017-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/B978-0-12-385120-8.00017-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGAWHOP%20cloning%3A%20a%20method%20of%20creating%20random%20mutagenesis%20libraries%20via%20megaprimer%20PCR%20of%20whole%20plasmids&amp;author=K.%20Miyazaki&amp;publication_year=2011&amp;journal=Meth.%20Enzymol.&amp;volume=498&amp;pages=399-406" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21601687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyazaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takenouchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Creating random mutagenesis libraries using megaprimer PCR of whole plasmid</div>, <div class="source">Biotechniques</div> , <div class="year">2002</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1033</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Creating%20random%20mutagenesis%20libraries%20using%20megaprimer%20PCR%20of%20whole%20plasmid&amp;author=K.%20Miyazaki&amp;author=M.%20Takenouchi&amp;publication_year=2002&amp;journal=Biotechniques&amp;volume=33&amp;pages=1033-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12449380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pei</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Duan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Shao</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Properties of an NAD<sup>+</sup>-dependent DNA ligase from the hyperthermophile <em>Thermotoga maritima</em> and its application in PCR amplification of long DNA fragments</div>, <div class="source">Enzyme Microb. Technol.</div> , <div class="year">2010</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">113</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.enzmictec.2009.08.015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.enzmictec.2009.08.015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20of%20an%20NAD%2B-dependent%20DNA%20ligase%20from%20the%20hyperthermophile%20Thermotoga%20maritima%20and%20its%20application%20in%20PCR%20amplification%20of%20long%20DNA%20fragments&amp;author=Y.%20Le&amp;author=J.%20Peng&amp;author=J.%20Pei&amp;author=H.%20Li&amp;author=Z.%20Duan&amp;author=W.%20Shao&amp;publication_year=2010&amp;journal=Enzyme%20Microb.%20Technol.&amp;volume=46&amp;pages=113-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.enzmictec.2009.08.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.enzmictec.2009.08.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.enzmictec.2009.08.015"> </span></div> </div></div></div></div></div><div content-id="DST016C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">X.Q.</div></div>,  <div class="name"><div class="surname">Shao</div> <div class="given-names">W.L.</div></div>. </div><div class="article-title">A highly efficient method for liquid and solid cultivation of the anaerobic hyperthermophilic eubacterium <em>Thermotoga maritime</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2006</div>, vol. <div class="volume">259</div> (pg. <div class="fpage">254</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1574-6968.2006.00273.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1574-6968.2006.00273.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20highly%20efficient%20method%20for%20liquid%20and%20solid%20cultivation%20of%20the%20anaerobic%20hyperthermophilic%20eubacterium%20Thermotoga%20maritime&amp;author=Y.%20Jiang&amp;author=Q.%20Zhou&amp;author=K.%20Wu&amp;author=X.Q.%20Li&amp;author=W.L.%20Shao&amp;publication_year=2006&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=259&amp;pages=254-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2006.259.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2006.259.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2006.259.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16734788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Pei</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Shao</div> <div class="given-names">W.</div></div>. </div><div class="article-title">pHsh vectors, a novel expression system of <em>Escherichia coli</em> for the large-scale production of recombinant enzymes</div>, <div class="source">Biotechnol. Lett.</div> , <div class="year">2010</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">795</div>-<div class="lpage">801</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10529-010-0223-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10529-010-0223-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=pHsh%20vectors%2C%20a%20novel%20expression%20system%20of%20Escherichia%20coli%20for%20the%20large-scale%20production%20of%20recombinant%20enzymes&amp;author=H.%20Wu&amp;author=J.%20Pei&amp;author=Y.%20Jiang&amp;author=X.%20Song&amp;author=W.%20Shao&amp;publication_year=2010&amp;journal=Biotechnol.%20Lett.&amp;volume=32&amp;pages=795-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10529-010-0223-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10529-010-0223-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10529-010-0223-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20155484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yin</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Le</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Pei</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Shao</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">High-level expression of the xylanase from <em>Thermomyces lanuginosus</em> in <em>Escherichia coli</em></div>, <div class="source">World J. Microbiol. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">275</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11274-007-9469-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11274-007-9469-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-level%20expression%20of%20the%20xylanase%20from%20Thermomyces%20lanuginosus%20in%20Escherichia%20coli&amp;author=E.%20Yin&amp;author=Y.%20Le&amp;author=J.%20Pei&amp;author=W.%20Shao&amp;author=Q.%20Yang&amp;publication_year=2008&amp;journal=World%20J.%20Microbiol.%20Biotechnol.&amp;volume=24&amp;pages=275-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11274-007-9469-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11274-007-9469-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11274-007-9469-5"> </span></div> </div></div></div></div></div><div content-id="DST016C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lever</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A new reaction for colorimetric determination of carbohydrates</div>, <div class="source">Anal. Biochem.</div> , <div class="year">1972</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">273</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0003-2697(72)90301-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0003-2697(72)90301-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20reaction%20for%20colorimetric%20determination%20of%20carbohydrates&amp;author=M.%20Lever&amp;publication_year=1972&amp;journal=Anal.%20Biochem.&amp;volume=47&amp;pages=273-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0003-2697(72)90301-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0003-2697(72)90301-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0003-2697(72)90301-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5031119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">G.</div></div>. </div><div class="article-title">DNA polymerase-catalyzed addition of nontemplated extra nucleotides to the 3′ end of a DNA fragment</div>, <div class="source">DNA Cell Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">763</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1089/dna.1993.12.763" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1089/dna.1993.12.763</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20polymerase-catalyzed%20addition%20of%20nontemplated%20extra%20nucleotides%20to%20the%203%E2%80%B2%20end%20of%20a%20DNA%20fragment&amp;author=G.%20Hu&amp;publication_year=1993&amp;journal=DNA%20Cell%20Biol.&amp;volume=12&amp;pages=763-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/dna.1993.12.763" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fdna.1993.12.763" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fdna.1993.12.763"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8397833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakaniwa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nishimur</div> <div class="given-names">K.</div></div>. </div><div class="article-title">An in vitro evaluation of a thermostable pectate lyase by using error-prone PCR</div>, <div class="source">J. Mol. Catal. B Enzym.</div> , <div class="year">2004</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">127</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.molcatb.2003.10.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.molcatb.2003.10.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20in%20vitro%20evaluation%20of%20a%20thermostable%20pectate%20lyase%20by%20using%20error-prone%20PCR&amp;author=T.%20Nakaniwa&amp;author=T.%20Tada&amp;author=M.%20Takao&amp;author=T.%20Sakai&amp;author=K.%20Nishimur&amp;publication_year=2004&amp;journal=J.%20Mol.%20Catal.%20B%20Enzym.&amp;volume=27&amp;pages=127-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcatb.2003.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcatb.2003.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcatb.2003.10.005"> </span></div> </div></div></div></div></div><div content-id="DST016C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Dawes</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Stemmer</div> <div class="given-names">W.P.C.</div></div>. </div><div class="article-title">Directed evolution of a fucosidase from a galactosidase by DNA shuffling and screening</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">4504</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.94.9.4504" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.94.9.4504</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Directed%20evolution%20of%20a%20fucosidase%20from%20a%20galactosidase%20by%20DNA%20shuffling%20and%20screening&amp;author=J.H.%20Zhang&amp;author=G.%20Dawes&amp;author=W.P.C.%20Stemmer&amp;publication_year=1997&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=4504-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.9.4504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.9.4504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.9.4504"> </span></div> </div></div></div></div></div><div content-id="DST016C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">M.-S.</div></div>,  <div class="name"><div class="surname">Lei</div> <div class="given-names">X.G.</div></div>. </div><div class="article-title">Enhancing thermostability of <em>Escherichia coli</em> phytase AppA2 by error-prone PCR</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">69</div>-<div class="lpage">75</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00253-008-1412-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00253-008-1412-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancing%20thermostability%20of%20Escherichia%20coli%20phytase%20AppA2%20by%20error-prone%20PCR&amp;author=M.-S.%20Kim&amp;author=X.G.%20Lei&amp;publication_year=2008&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=79&amp;pages=69-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-008-1412-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-008-1412-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-008-1412-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18340444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">B.</div></div>. </div><div class="article-title">PCR approaches to DNA mutagenesis and recombination. An overview</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">167</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR%20approaches%20to%20DNA%20mutagenesis%20and%20recombination.%20An%20overview&amp;author=B.%20Shen&amp;publication_year=2002&amp;journal=Methods%20Mol.%20Biol.&amp;volume=192&amp;pages=167-74" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12494649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tobias</div> <div class="given-names">A.V.</div></div>. </div><div class="article-title">Preparing libraries in <em>Escherichia coli</em></div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">231</div> (pg. <div class="fpage">11</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preparing%20libraries%20in%20Escherichia%20coli&amp;author=A.V.%20Tobias&amp;publication_year=2003&amp;journal=Methods%20Mol.%20Biol.&amp;volume=231&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST016C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST016C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">L.</div></div>. </div><div class="article-title">An improvement of the site-directed mutagenesis method by combination of megaprimer, one-side PCR and <em>Dpn</em>I treatment</div>, <div class="source">Anal. Biochem.</div> , <div class="year">2004</div>, vol. <div class="volume">331</div> (pg. <div class="fpage">401</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ab.2004.04.019" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ab.2004.04.019</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improvement%20of%20the%20site-directed%20mutagenesis%20method%20by%20combination%20of%20megaprimer%2C%20one-side%20PCR%20and%20DpnI%20treatment&amp;author=D.%20Wei&amp;author=M.%20Li&amp;author=X.%20Zhang&amp;author=L.%20Xing&amp;publication_year=2004&amp;journal=Anal.%20Biochem.&amp;volume=331&amp;pages=401-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ab.2004.04.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ab.2004.04.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ab.2004.04.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15265749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Minoru Yoshida</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=354164&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">329</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="354164"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f4%2f375%2f354164">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000324223800005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F4%2F375%2F354164" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/354164'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Thermostable DNA Ligase-Mediated PCR Production of Circular Plasmid (PPCP) and Its Application in Directed Evolution via In situ Error-Prone PCR" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f375%2f354164">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

