

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2004,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/6/623/421469","@type":"ScholarlyArticle","name":"A Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy","datePublished":"2013-08-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2013-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst036","keywords":["cedar pollen allergy","GM crop","transgenic rice","transcriptome","whole-genome sequence"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/6/623/421469","author":[{"name":"Kawakatsu, Taiji","affiliation":"1  Genetically Modified Organism Research Center ,  National Institute of Agrobiological Sciences ,  2-1-2 Kan-nondai, Tsukuba, Ibaraki  305-8602 ,  Japan","@type":"Person"},{"name":"Kawahara, Yoshihiro","affiliation":"2  Agrogenomics Research Center ,  National Institute of Agrobiological Sciences ,  2-1-2 Kan-nondai, Tsukuba, Ibaraki  305-8602 ,  Japan","@type":"Person"},{"name":"Itoh, Takeshi","affiliation":"2  Agrogenomics Research Center ,  National Institute of Agrobiological Sciences ,  2-1-2 Kan-nondai, Tsukuba, Ibaraki  305-8602 ,  Japan","@type":"Person"},{"name":"Takaiwa, Fumio","affiliation":"1  Genetically Modified Organism Research Center ,  National Institute of Agrobiological Sciences ,  2-1-2 Kan-nondai, Tsukuba, Ibaraki  305-8602 ,  Japan","@type":"Person"}],"description":"Abstract.  Genetic modification (GM) by Agrobacterium-mediated transformation is a robust and widely employed method to confer new traits to crops. In this proc","pageStart":"623","pageEnd":"631","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy" />
    <meta property="og:description" content="Abstract.  Genetic modification (GM) by Agrobacterium-mediated transformation is a robust and widely employed method to confer new traits to crops. In this proc" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/6/623/421469" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-08-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kawakatsu, Taiji" /><meta name="citation_author_institution" content="Genetically Modified Organism Research Center, National Institute of Agrobiological Sciences, 2-1-2 Kan-nondai, Tsukuba, Ibaraki305-8602, Japan" /><meta name="citation_author" content="Kawahara, Yoshihiro" /><meta name="citation_author_institution" content="Agrogenomics Research Center, National Institute of Agrobiological Sciences, 2-1-2 Kan-nondai, Tsukuba, Ibaraki305-8602, Japan" /><meta name="citation_author" content="Itoh, Takeshi" /><meta name="citation_author_institution" content="Agrogenomics Research Center, National Institute of Agrobiological Sciences, 2-1-2 Kan-nondai, Tsukuba, Ibaraki305-8602, Japan" /><meta name="citation_author" content="Takaiwa, Fumio" /><meta name="citation_author_institution" content="Genetically Modified Organism Research Center, National Institute of Agrobiological Sciences, 2-1-2 Kan-nondai, Tsukuba, Ibaraki305-8602, Japan" /><meta name="citation_title" content="A Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy" /><meta name="citation_firstpage" content="623" /><meta name="citation_lastpage" content="631" /><meta name="citation_doi" content="10.1093/dnares/dst036" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2013/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=It is a long way to GM agriculture; Ann. Rev. Plant Biol.;  citation_year=2011;  citation_volume=62;  citation_pages=1-23; " /><meta name="citation_reference" content="Clive J. Global Status of Commercialized Biotech/GM Crops: 2011 2011 Ithaca, NY ISAAA ISAAA Brief " /><meta name="citation_reference" content="citation_title=Oral immunotherapy with transgenic rice seed containing destructed Japanese cedar pollen allergens, Cry j 1 and Cry j 2, against Japanese cedar pollinosis; Plant Biotechnol. J.;  citation_year=2013;  citation_volume=11;  citation_pages=66-76; " /><meta name="citation_reference" content="citation_title=Genomic changes in transgenic rice (Oryza sativa L.) plants produced by infecting calli with Agrobacterium tumefaciens; Plant Cell Rep.;  citation_year=2001;  citation_volume=20;  citation_pages=325-30; " /><meta name="citation_reference" content="citation_title=Genomic stability in Arabidopsis thaliana transgenic plants obtained by floral dip; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=1512-8; " /><meta name="citation_reference" content="citation_title=Plant proteins involved in Agrobacterium-mediated genetic transformation; Ann. Rev. Phytopathol.;  citation_year=2010;  citation_volume=48;  citation_pages=45-68; " /><meta name="citation_reference" content="citation_title=The draft genome of the transgenic tropical fruit tree papaya (Carica papaya Linnaeus); Nature;  citation_year=2008;  citation_volume=452;  citation_pages=991-6; " /><meta name="citation_reference" content="citation_title=Microarray analyses reveal that plant mutagenesis may induce more transcriptomic changes than transgene insertion; Proc. Natl Acad. Sci. USA.;  citation_year=2008;  citation_volume=105;  citation_pages=3640-45; " /><meta name="citation_reference" content="citation_title=Foreign gene products can be enhanced by introduction into low storage protein mutants; Plant Biotechnol. J.;  citation_year=2003;  citation_volume=1;  citation_pages=411-22; " /><meta name="citation_reference" content="citation_title=Mutants lacking glutelin subunits in rice: mapping and combination of mutated glutelin genes; Theor. Appl. Genet.;  citation_year=1997;  citation_volume=94;  citation_pages=177-83; " /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; EMBnet J.;  citation_year=2011;  citation_volume=17;  citation_pages=10-2; " /><meta name="citation_reference" content="citation_title=A rice glutelin gene family—a major type of glutelin messenger-RNAs can be divided into 2 classes; Mol. Genet. Genomics.;  citation_year=1987;  citation_volume=208;  citation_pages=15-22; " /><meta name="citation_reference" content="citation_title=Rice Annotation Project Database (RAP-DB): an integrative and interactive database for rice genomics; Plant Cell Physiol.;  citation_year=2013;  citation_volume=54;  citation_pages=e6" /><meta name="citation_reference" content="citation_title=Differential expression analysis for sequence count data; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R106" /><meta name="citation_reference" content="citation_title=Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=267" /><meta name="citation_reference" content="citation_title=T-DNA-mediated transfer of Agrobacterium tumefaciens chromosomal DNA into plants; Nature Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1015-7; " /><meta name="citation_reference" content="citation_title=Bacterial transposons are co-transferred with T-DNA to rice chromosomes during Agrobacterium-mediated transformation; Mol. Cells;  citation_year=2012;  citation_volume=33;  citation_pages=583-9; " /><meta name="citation_reference" content="citation_title=The genome of the natural genetic engineer Agrobacterium tumefaciens C58; Science;  citation_year=2001;  citation_volume=294;  citation_pages=2317-23; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Transpositional landscape of the rice genome revealed by paired-end mapping of high-throughput re-sequencing data; Plant J Cell Mol. Biol.;  citation_year=2011;  citation_volume=66;  citation_pages=241-6; " /><meta name="citation_reference" content="citation_title=Molecular spectrum of somaclonal variation in regenerated rice revealed by whole-genome sequencing; Plant Cell Physiol.;  citation_year=2012;  citation_volume=53;  citation_pages=256-64; " /><meta name="citation_reference" content="citation_title=Regenerant Arabidopsis lineages display a distinct genome-wide spectrum of mutations conferring variant phenotypes; Curr. Biol.;  citation_year=2011;  citation_volume=21;  citation_pages=1385-90; " /><meta name="citation_reference" content="citation_title=Rice structural variation: a comparative analysis of structural variation between rice and three of its closest relatives in the genus Oryza; Plant J Cell Mol Biol;  citation_year=2010;  citation_volume=63;  citation_pages=990-1003; " /><meta name="citation_reference" content="citation_title=The rate and molecular spectrum of spontaneous mutations in Arabidopsis thaliana; Science;  citation_year=2010;  citation_volume=327;  citation_pages=92-4; " /><meta name="citation_reference" content="citation_title=Spectrum of chemically induced mutations from a large-scale reverse-genetic screen in Arabidopsis; Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=731-40; " /><meta name="citation_reference" content="citation_title=Molecular characterization of mutations induced by gamma irradiation in rice; Genes Genet. Syst.;  citation_year=2009;  citation_volume=84;  citation_pages=361-70; " /><meta name="citation_reference" content="citation_title=Discovery of genome-wide DNA polymorphisms in a landrace cultivar of Japonica rice by whole-genome sequencing; Plant Cell Physiol.;  citation_year=2011;  citation_volume=52;  citation_pages=274-82; " /><meta name="citation_reference" content="citation_title=Signal transduction by IRE1-mediated splicing of bZIP50 and other stress sensors in the endoplasmic reticulum stress response of rice; Plant J.;  citation_year=2012;  citation_volume=69;  citation_pages=946-56; " /><meta name="citation_reference" content="citation_title=Analysis of ER stress in developing rice endosperm accumulating beta-amyloid peptide; Plant Biotechnol. J.;  citation_year=2010;  citation_volume=8;  citation_pages=691-718; " /><meta name="citation_reference" content="citation_title=High-efficiency TALEN-based gene editing produces disease-resistant rice; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=390-92; " /><meta name="citation_reference" content="citation_title=Efficient gene targeting by homologous recombination in rice; Nat. Biotechnol.;  citation_year=2002;  citation_volume=20;  citation_pages=1030-34; " /><meta name="citation_reference" content="citation_title=Development of an efficient Agrobacterium-mediated gene targeting system for rice and analysis of rice knockouts lacking granule-bound starch synthase (Waxy) and beta1,2-xylosyltransferase; Plant Cell Physiol.;  citation_year=2012;  citation_volume=53;  citation_pages=755-61; " /><meta name="citation_reference" content="citation_title=Integrative genomics viewer; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=24-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/6/623/1166944/dst036.pdf" /><meta name="description" content="Abstract.  Genetic modification (GM) by Agrobacterium-mediated transformation is a robust and widely employed method to confer new traits to crops. In this proc" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/6/623/421469" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/6/623/421469" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/6/623/421469");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst036';
    </script>
<title>
	Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./421469" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZFNPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdha2F0c3UsIFRhaWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2VuZXRpY2FsbHkgTW9kaWZpZWQgT3JnYW5pc20gUmVzZWFyY2ggQ2VudGVyLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDItMS0yIEthbi1ub25kYWksIFRzdWt1YmEsIEliYXJha2kzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhaGFyYSwgWW9zaGloaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQWdyb2dlbm9taWNzIFJlc2VhcmNoIENlbnRlciwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCAyLTEtMiBLYW4tbm9uZGFpLCBUc3VrdWJhLCBJYmFyYWtpMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgVGFrZXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFncm9nZW5vbWljcyBSZXNlYXJjaCBDZW50ZXIsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMi0xLTIgS2FuLW5vbmRhaSwgVHN1a3ViYSwgSWJhcmFraTMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRha2Fpd2EsIEZ1bWlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2VuZXRpY2FsbHkgTW9kaWZpZWQgT3JnYW5pc20gUmVzZWFyY2ggQ2VudGVyLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDItMS0yIEthbi1ub25kYWksIFRzdWt1YmEsIEliYXJha2kzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgV2hvbGUtR2Vub21lIEFuYWx5c2lzIG9mIGEgVHJhbnNnZW5pYyBSaWNlIFNlZWQtQmFzZWQgRWRpYmxlIFZhY2NpbmUgQWdhaW5zdCBDZWRhciBQb2xsZW4gQWxsZXJneSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjYyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN0MDM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JdCBpcyBhIGxvbmcgd2F5IHRvIEdNIGFncmljdWx0dXJlOyBBbm4uIFJldi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJDbGl2ZSBKLiBHbG9iYWwgU3RhdHVzIG9mIENvbW1lcmNpYWxpemVkIEJpb3RlY2gvR00gQ3JvcHM6IDIwMTEgMjAxMSBJdGhhY2EsIE5ZIElTQUFBIElTQUFBIEJyaWVmICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yYWwgaW1tdW5vdGhlcmFweSB3aXRoIHRyYW5zZ2VuaWMgcmljZSBzZWVkIGNvbnRhaW5pbmcgZGVzdHJ1Y3RlZCBKYXBhbmVzZSBjZWRhciBwb2xsZW4gYWxsZXJnZW5zLCBDcnkgaiAxIGFuZCBDcnkgaiAyLCBhZ2FpbnN0IEphcGFuZXNlIGNlZGFyIHBvbGxpbm9zaXM7IFBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgY2hhbmdlcyBpbiB0cmFuc2dlbmljIHJpY2UgKE9yeXphIHNhdGl2YSBMLikgcGxhbnRzIHByb2R1Y2VkIGJ5IGluZmVjdGluZyBjYWxsaSB3aXRoIEFncm9iYWN0ZXJpdW0gdHVtZWZhY2llbnM7IFBsYW50IENlbGwgUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTMyNS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzdGFiaWxpdHkgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgdHJhbnNnZW5pYyBwbGFudHMgb2J0YWluZWQgYnkgZmxvcmFsIGRpcDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xNTEyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHByb3RlaW5zIGludm9sdmVkIGluIEFncm9iYWN0ZXJpdW0tbWVkaWF0ZWQgZ2VuZXRpYyB0cmFuc2Zvcm1hdGlvbjsgQW5uLiBSZXYuIFBoeXRvcGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTQ1LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIHRoZSB0cmFuc2dlbmljIHRyb3BpY2FsIGZydWl0IHRyZWUgcGFwYXlhIChDYXJpY2EgcGFwYXlhIExpbm5hZXVzKTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MjsgIGNpdGF0aW9uX3BhZ2VzPTk5MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2FycmF5IGFuYWx5c2VzIHJldmVhbCB0aGF0IHBsYW50IG11dGFnZW5lc2lzIG1heSBpbmR1Y2UgbW9yZSB0cmFuc2NyaXB0b21pYyBjaGFuZ2VzIHRoYW4gdHJhbnNnZW5lIGluc2VydGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9MzY0MC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9yZWlnbiBnZW5lIHByb2R1Y3RzIGNhbiBiZSBlbmhhbmNlZCBieSBpbnRyb2R1Y3Rpb24gaW50byBsb3cgc3RvcmFnZSBwcm90ZWluIG11dGFudHM7IFBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz00MTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGFudHMgbGFja2luZyBnbHV0ZWxpbiBzdWJ1bml0cyBpbiByaWNlOiBtYXBwaW5nIGFuZCBjb21iaW5hdGlvbiBvZiBtdXRhdGVkIGdsdXRlbGluIGdlbmVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9MTc3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXRhZGFwdCByZW1vdmVzIGFkYXB0ZXIgc2VxdWVuY2VzIGZyb20gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgcmVhZHM7IEVNQm5ldCBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTEwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmljZSBnbHV0ZWxpbiBnZW5lIGZhbWlseeKAlGEgbWFqb3IgdHlwZSBvZiBnbHV0ZWxpbiBtZXNzZW5nZXItUk5BcyBjYW4gYmUgZGl2aWRlZCBpbnRvIDIgY2xhc3NlczsgTW9sLiBHZW5ldC4gR2Vub21pY3MuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTIwODsgIGNpdGF0aW9uX3BhZ2VzPTE1LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaWNlIEFubm90YXRpb24gUHJvamVjdCBEYXRhYmFzZSAoUkFQLURCKTogYW4gaW50ZWdyYXRpdmUgYW5kIGludGVyYWN0aXZlIGRhdGFiYXNlIGZvciByaWNlIGdlbm9taWNzOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9ZTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBhbmFseXNpcyBmb3Igc2VxdWVuY2UgY291bnQgZGF0YTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9UjEwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUgZGVmaW5pdGlvbiBvZiB0aGUgcGVkaWdyZWUgaGFwbG90eXBlcyBvZiBjbG9zZWx5IHJlbGF0ZWQgcmljZSBjdWx0aXZhcnMgYnkgbWVhbnMgb2YgZ2Vub21lLXdpZGUgZGlzY292ZXJ5IG9mIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTI2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVQtRE5BLW1lZGlhdGVkIHRyYW5zZmVyIG9mIEFncm9iYWN0ZXJpdW0gdHVtZWZhY2llbnMgY2hyb21vc29tYWwgRE5BIGludG8gcGxhbnRzOyBOYXR1cmUgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMDE1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCB0cmFuc3Bvc29ucyBhcmUgY28tdHJhbnNmZXJyZWQgd2l0aCBULUROQSB0byByaWNlIGNocm9tb3NvbWVzIGR1cmluZyBBZ3JvYmFjdGVyaXVtLW1lZGlhdGVkIHRyYW5zZm9ybWF0aW9uOyBNb2wuIENlbGxzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTgzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIG5hdHVyYWwgZ2VuZXRpYyBlbmdpbmVlciBBZ3JvYmFjdGVyaXVtIHR1bWVmYWNpZW5zIEM1ODsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTQ7ICBjaXRhdGlvbl9wYWdlcz0yMzE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NpdGlvbmFsIGxhbmRzY2FwZSBvZiB0aGUgcmljZSBnZW5vbWUgcmV2ZWFsZWQgYnkgcGFpcmVkLWVuZCBtYXBwaW5nIG9mIGhpZ2gtdGhyb3VnaHB1dCByZS1zZXF1ZW5jaW5nIGRhdGE7IFBsYW50IEogQ2VsbCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MjQxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBzcGVjdHJ1bSBvZiBzb21hY2xvbmFsIHZhcmlhdGlvbiBpbiByZWdlbmVyYXRlZCByaWNlIHJldmVhbGVkIGJ5IHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MjU2LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWdlbmVyYW50IEFyYWJpZG9wc2lzIGxpbmVhZ2VzIGRpc3BsYXkgYSBkaXN0aW5jdCBnZW5vbWUtd2lkZSBzcGVjdHJ1bSBvZiBtdXRhdGlvbnMgY29uZmVycmluZyB2YXJpYW50IHBoZW5vdHlwZXM7IEN1cnIuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTM4NS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmljZSBzdHJ1Y3R1cmFsIHZhcmlhdGlvbjogYSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBzdHJ1Y3R1cmFsIHZhcmlhdGlvbiBiZXR3ZWVuIHJpY2UgYW5kIHRocmVlIG9mIGl0cyBjbG9zZXN0IHJlbGF0aXZlcyBpbiB0aGUgZ2VudXMgT3J5emE7IFBsYW50IEogQ2VsbCBNb2wgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTk5MC0xMDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmF0ZSBhbmQgbW9sZWN1bGFyIHNwZWN0cnVtIG9mIHNwb250YW5lb3VzIG11dGF0aW9ucyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjc7ICBjaXRhdGlvbl9wYWdlcz05Mi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjdHJ1bSBvZiBjaGVtaWNhbGx5IGluZHVjZWQgbXV0YXRpb25zIGZyb20gYSBsYXJnZS1zY2FsZSByZXZlcnNlLWdlbmV0aWMgc2NyZWVuIGluIEFyYWJpZG9wc2lzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz03MzEtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIG11dGF0aW9ucyBpbmR1Y2VkIGJ5IGdhbW1hIGlycmFkaWF0aW9uIGluIHJpY2U7IEdlbmVzIEdlbmV0LiBTeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTM2MS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIGdlbm9tZS13aWRlIEROQSBwb2x5bW9ycGhpc21zIGluIGEgbGFuZHJhY2UgY3VsdGl2YXIgb2YgSmFwb25pY2EgcmljZSBieSB3aG9sZS1nZW5vbWUgc2VxdWVuY2luZzsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTI3NC04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lnbmFsIHRyYW5zZHVjdGlvbiBieSBJUkUxLW1lZGlhdGVkIHNwbGljaW5nIG9mIGJaSVA1MCBhbmQgb3RoZXIgc3RyZXNzIHNlbnNvcnMgaW4gdGhlIGVuZG9wbGFzbWljIHJldGljdWx1bSBzdHJlc3MgcmVzcG9uc2Ugb2YgcmljZTsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz05NDYtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIEVSIHN0cmVzcyBpbiBkZXZlbG9waW5nIHJpY2UgZW5kb3NwZXJtIGFjY3VtdWxhdGluZyBiZXRhLWFteWxvaWQgcGVwdGlkZTsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTY5MS03MTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtZWZmaWNpZW5jeSBUQUxFTi1iYXNlZCBnZW5lIGVkaXRpbmcgcHJvZHVjZXMgZGlzZWFzZS1yZXNpc3RhbnQgcmljZTsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTM5MC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGdlbmUgdGFyZ2V0aW5nIGJ5IGhvbW9sb2dvdXMgcmVjb21iaW5hdGlvbiBpbiByaWNlOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTAzMC0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYW4gZWZmaWNpZW50IEFncm9iYWN0ZXJpdW0tbWVkaWF0ZWQgZ2VuZSB0YXJnZXRpbmcgc3lzdGVtIGZvciByaWNlIGFuZCBhbmFseXNpcyBvZiByaWNlIGtub2Nrb3V0cyBsYWNraW5nIGdyYW51bGUtYm91bmQgc3RhcmNoIHN5bnRoYXNlIChXYXh5KSBhbmQgYmV0YTEsMi14eWxvc3lsdHJhbnNmZXJhc2U7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz03NTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aXZlIGdlbm9taWNzIHZpZXdlcjsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC82LzYyMy8xMTY2OTQ0L2RzdDAzNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2VuZXRpYyBtb2RpZmljYXRpb24gKEdNKSBieSBBZ3JvYmFjdGVyaXVtLW1lZGlhdGVkIHRyYW5zZm9ybWF0aW9uIGlzIGEgcm9idXN0IGFuZCB3aWRlbHkgZW1wbG95ZWQgbWV0aG9kIHRvIGNvbmZlciBuZXcgdHJhaXRzIHRvIGNyb3BzLiBJbiB0aGlzIHByb2MiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQyMTQ2OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSlmDRFR8BpHFmlCSUGAAeCu1+TpA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f623%2f421469" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f623%2f421469" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16367" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&amp;Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/6">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6245021">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245021">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245025">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245025">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245050">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245050">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245071">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245071">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245154">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245154">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245166">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245166">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245170">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245170">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6245175">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6245175">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/6/605/420907">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taiji Kawakatsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genetically Modified Organism Research Center</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kan-nondai, Tsukuba, Ibaraki</div><div class="addr-line">305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taiji+Kawakatsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawakatsu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawakatsu T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taiji Kawakatsu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Kawahara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Agrogenomics Research Center</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kan-nondai, Tsukuba, Ibaraki</div><div class="addr-line">305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Kawahara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawahara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawahara Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshihiro Kawahara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Agrogenomics Research Center</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kan-nondai, Tsukuba, Ibaraki</div><div class="addr-line">305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeshi Itoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumio Takaiwa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genetically Modified Organism Research Center</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kan-nondai, Tsukuba, Ibaraki</div><div class="addr-line">305-8602</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 29-838-8373. E-mail: <a href="mailto:takaiwa@nias.affrc.go.jp" target="_blank">takaiwa@nias.affrc.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumio+Takaiwa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takaiwa F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takaiwa F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumio Takaiwa</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 623–631, <a href='https://doi.org/10.1093/dnares/dst036'>https://doi.org/10.1093/dnares/dst036</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 August 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 May 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 July 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="421469" href="/dnaresearch/article-pdf/20/6/623/1166944/dst036.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Taiji Kawakatsu, Yoshihiro Kawahara, Takeshi Itoh, Fumio Takaiwa;  A Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy, <em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 623–631, <a href='https://doi.org/10.1093/dnares/dst036'>https://doi.org/10.1093/dnares/dst036</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/421469?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/421469?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/421469?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/421469?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/421469?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/421469?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/421469?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Whole-Genome%20Analysis%20of%20a%20Transgenic%20Rice%20Seed-Based%20Edible%20Vaccine%20Against%20Cedar%20Pollen%20Allergy&amp;publicationDate=2013-08-15&amp;volumeNum=20&amp;issueNum=6&amp;author=Kawakatsu%2C%20Taiji%3B%20Kawahara%2C%20Yoshihiro&amp;startPage=623&amp;endPage=631&amp;contentId=10.1093%2Fdnares%2Fdst036&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f623%2f421469" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6245021 id="6245021" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genetic modification (GM) by <em>Agrobacterium</em>-mediated transformation is a robust and widely employed method to confer new traits to crops. In this process, a transfer DNA is delivered into the host genome, but it is still unclear how the host genome is altered by this event at single-base resolution. To decipher genomic discrepancy between GM crops and their host, we conducted whole-genome sequencing of a transgenic rice line OSCR11. This rice line expresses a seed-based edible vaccine containing two major pollen allergens, Cry j 1 and Cry j 2, against Japanese cedar pollinosis. We revealed that genetic differences between OSCR11 and its host a123 were significantly less than those between a123 and its precedent cultivar Koshihikari. The pattern of nucleotide base substitution in OSCR11, relative to a123, was consistent with somaclonal variation. Mutations in OSCR11 probably occurred during the cell culture steps. In addition, strand-specific mRNA-Seq revealed similar transcriptomes of a123 and OSCR11, supporting genomic integrity between them.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cedar pollen allergy&quot;">cedar pollen allergy</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;GM crop&quot;">GM crop</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transgenic rice&quot;">transgenic rice</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome sequence&quot;">whole-genome sequence</a></div>                            <h2 scrollto-destination=6245025 id="6245025" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genetic modification (GM) is widely used to engineer organism traits for the purposes of basic research and applied science. In plants, <em>Agrobacterium tumefaciens</em>-mediated transformation is the most widely used system of genetic engineering. <em>Agrobacterium</em> is an endemic soil bacterium and induces crown gall formation in host plants.<sup><a reveal-id="DST036C1" data-open="DST036C1" class="link link-ref link-reveal xref-bibr">1</a></sup> There are numerous challenges in applying GM technologies to agriculture. To date, various types of GM crops, such as plants resistant to biotic and abiotic stresses, bioenergy crops, and biopharmaceutical crops, have been developed. Since the first commercialization of herbicide- or insect-resistant GM crop in 1996, 170 million hectares in 28 countries had been allotted to GM crops in 2012.<sup><a reveal-id="DST036C2" data-open="DST036C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>In a previous study, we generated transgenic rice (<em>Oryza sativa</em>) ectopically expressing the major Japanese cedar (<em>Cryptomeria japonica</em>) pollen allergens Cry j 1 and Cry j 2. The transgene products specifically accumulated to high levels in the endosperm of rice grain.<sup><a reveal-id="DST036C3" data-open="DST036C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In mice fed the transgenic rice seed, we observed suppression of specific IgE antibodies and a reduction in the clinical symptoms of pollinosis, such as sneezing frequency and infiltration of inflammatory cells.<sup><a reveal-id="DST036C3" data-open="DST036C3" class="link link-ref link-reveal xref-bibr">3</a></sup> We choose a single plant with a single transfer DNA (T-DNA) insertion site from this transgenic line and named as OSCR11.</p><p>Differences between GM crops and non-GM crops are one of the most cited public concerns regarding food safety and security. Genomic changes caused by <em>Agrobacterium</em>-mediated transformation have been reported.<sup><a reveal-id="DST036C4" data-open="DST036C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST036C5" data-open="DST036C5" class="link link-ref link-reveal xref-bibr">5</a></sup><em>Agrobacterium</em>-mediated transformation of rice requires a cell culture step, in which mutations called somaclonal variations arise. These mutations can be detected by Amplified Fragment Length Polymorphism (AFLP), Random Amplified Microsatellite Polymorphism (RAMP), and Random Amplified Polymorphic DNA (RAPD): polymerase chain reaction (PCR)-based analyses that depend on restriction enzyme sites, microsatellites, and specific primers, as has been reported in <em>Arabidopsis</em>.<sup><a reveal-id="DST036C5" data-open="DST036C5" class="link link-ref link-reveal xref-bibr">5</a></sup> T-DNA integration occurs at regions of microhomology between the T-DNA and host genome sequences and/or during the double-strand break (DSB) repair process. Chromatin factors and histone proteins of host plants are also involved in this process, revealing potential sources of damage to the plant host genome by <em>Agrobacterium</em>-mediated transformation.<sup><a reveal-id="DST036C6" data-open="DST036C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Next-generation sequencing (NGS) approaches such as whole-genome resequencing and RNA sequencing (RNA-Seq) offer the means to study the genomic differences between GM and non-GM crops. The genome sequence of virus-resistant transgenic papaya has been reported, but the differences between GM papaya and non-GM papaya were not addressed.<sup><a reveal-id="DST036C7" data-open="DST036C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Another study using microarray analyses found that gamma-ray mutagenesis altered the transcriptome of rice more profoundly than a transgenic modification.<sup><a reveal-id="DST036C8" data-open="DST036C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>As oral administration of OSCR11 seed is expected to have a therapeutic effect via the induction of oral immune tolerance against cedar pollen allergens, homozygous OSCR11 seed will be subjected to clinical trials as a treatment for cedar pollen allergy patients after authorization by the Pharmaceuticals and Medical Devices Agency of Japan. In this study, we conducted whole-genome sequencing and transcriptome analysis of genetically modified OSCR11 rice using NGS. OSCR11 has the background genotype of the Koshihikari low-glutelin mutant a123, which accumulates high levels of recombinant proteins in the endosperm by rebalancing seed storage proteins.<sup><a reveal-id="DST036C9" data-open="DST036C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The a123 genotype is a triple mutant with mutations in three glutelin genes (<em>a1</em>: <em>GluB-4</em>, <em>a2</em>: <em>GluA-2</em>, and <em>a3</em>: <em>GluA-1</em>), and was obtained by crossing a1, a2, and a3 mutants (Fig. <a reveal-id="DST036F1" data-open="DST036F1" class="link link-reveal link-table xref-fig">1</a>).<sup><a reveal-id="DST036C10" data-open="DST036C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The <em>a1</em> and <em>a2</em> mutations were obtained by gamma-ray treatment, and <em>a3</em> was obtained by ethyl methanesulfonate (EMS).<sup><a reveal-id="DST036C10" data-open="DST036C10" class="link link-ref link-reveal xref-bibr">10</a></sup> To accurately detect mutations induced by <em>Agrobacterium</em>-mediated transformation, we determined the whole-genome sequences and transcriptomes of a123 and compared them with those of OSCR11. </p>                        <a id="6245042" scrollto-destination="6245042"></a>
<div id="DST036F1" data-id="DST036F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/m_dst03601.png?Expires=2147483647&Signature=3vJxigSjJ5y4rdMBuxJRg1u61WMi2XU8Qy6mPWPOJr5zsSpMdv4gEwvl9Sl6e7JUvtGM1NGixyNqC9PBCC76ZWmFOpXmH6W8DDOkdeH9jHIFQABXkEMPhXsGsn77uMUjbJj37F6-9-si5k8EXwGMRBvKXjMHm1uTCU5TsEdDLVqLvFu5RwYep92kc4ZUXvLDsP13cocjtsVPKkD-Aw5z2QE4DHR8dR0W1TGOeqP-~08YFBA6oOpu0bTKaK3Lh-0XrpOp-jB7WmCct~HBT2JQJPP5XSafmbIQtwfgieH~69oo4tZjUxIG9C9TALHTe0dIZUmDsKCceJ89c5P4SAdwEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Origin of a123 and OSCR11. a1/gluB-4 and a2/gluA-2 mutations occurred after gamma-ray treatment, and the a3/gluA-1 mutation occurred after EMS treatment of the rice cultivar Koshihikari. a12 with both a1 and a2 mutations was obtained by crossing. a123 with all three mutations was obtained by crossing a12 and a3. OSCR11 was obtained by performing Agrobacterium-mediated transformation on a123." path-from-xml="dst03601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6245042/dst03601.tiff" path-from-xml="dst03601.tiff" target="_blank">View large</a><a section="6245042" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst03601.png?Expires=2147483647&Signature=KIsOO0hzZzRfljcRsEKxBUdiYsmD81CpuHaA-41bGunGOIaNDilBWk1wSYkTbWFGze-9s3WMuCPp-ODnb2zbljW0vrEnFJVcZ4jo6dfRdP4Kt8KET1ZnWtIy3KD9TU9zI1AN3QsVRfFTIDtIRcZtWbD5~RTO7ugYBVTxuAENjA3yCdeLORohF1pwMNMoPQMvUWiiO7gUTLt-fscGlCfqOf7C8A1R0OrmjldekGljEhWd~7AboeI9z-AURbPJx6P66Toa8u5hUGDo~HzIjR31GZ5lGHUDLoRTbsZ-cps7gLnTOkz8zWUpmIslCyUcvZBg7Pe~~3HsU5iROp-gZSspYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6245042&ar=421469&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Origin of a123 and OSCR11. <em>a1/gluB-4</em> and <em>a2/gluA-2</em> mutations occurred after gamma-ray treatment, and the <em>a3/gluA-1</em> mutation occurred after EMS treatment of the rice cultivar Koshihikari. a12 with both <em>a1</em> and <em>a2</em> mutations was obtained by crossing. a123 with all three mutations was obtained by crossing a12 and a3. OSCR11 was obtained by performing <em>Agrobacterium</em>-mediated transformation on a123.</p></div></div><div content-id="DST036F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/m_dst03601.png?Expires=2147483647&Signature=3vJxigSjJ5y4rdMBuxJRg1u61WMi2XU8Qy6mPWPOJr5zsSpMdv4gEwvl9Sl6e7JUvtGM1NGixyNqC9PBCC76ZWmFOpXmH6W8DDOkdeH9jHIFQABXkEMPhXsGsn77uMUjbJj37F6-9-si5k8EXwGMRBvKXjMHm1uTCU5TsEdDLVqLvFu5RwYep92kc4ZUXvLDsP13cocjtsVPKkD-Aw5z2QE4DHR8dR0W1TGOeqP-~08YFBA6oOpu0bTKaK3Lh-0XrpOp-jB7WmCct~HBT2JQJPP5XSafmbIQtwfgieH~69oo4tZjUxIG9C9TALHTe0dIZUmDsKCceJ89c5P4SAdwEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Origin of a123 and OSCR11. a1/gluB-4 and a2/gluA-2 mutations occurred after gamma-ray treatment, and the a3/gluA-1 mutation occurred after EMS treatment of the rice cultivar Koshihikari. a12 with both a1 and a2 mutations was obtained by crossing. a123 with all three mutations was obtained by crossing a12 and a3. OSCR11 was obtained by performing Agrobacterium-mediated transformation on a123." path-from-xml="dst03601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6245042/dst03601.tiff" path-from-xml="dst03601.tiff" target="_blank">View large</a><a section="6245042" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst03601.png?Expires=2147483647&Signature=KIsOO0hzZzRfljcRsEKxBUdiYsmD81CpuHaA-41bGunGOIaNDilBWk1wSYkTbWFGze-9s3WMuCPp-ODnb2zbljW0vrEnFJVcZ4jo6dfRdP4Kt8KET1ZnWtIy3KD9TU9zI1AN3QsVRfFTIDtIRcZtWbD5~RTO7ugYBVTxuAENjA3yCdeLORohF1pwMNMoPQMvUWiiO7gUTLt-fscGlCfqOf7C8A1R0OrmjldekGljEhWd~7AboeI9z-AURbPJx6P66Toa8u5hUGDo~HzIjR31GZ5lGHUDLoRTbsZ-cps7gLnTOkz8zWUpmIslCyUcvZBg7Pe~~3HsU5iROp-gZSspYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6245042&ar=421469&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Origin of a123 and OSCR11. <em>a1/gluB-4</em> and <em>a2/gluA-2</em> mutations occurred after gamma-ray treatment, and the <em>a3/gluA-1</em> mutation occurred after EMS treatment of the rice cultivar Koshihikari. a12 with both <em>a1</em> and <em>a2</em> mutations was obtained by crossing. a123 with all three mutations was obtained by crossing a12 and a3. OSCR11 was obtained by performing <em>Agrobacterium</em>-mediated transformation on a123.</p></div></div>                            <h2 scrollto-destination=6245050 id="6245050" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=6245055 id="6245055" class="section-title" data-legacy-id=s2a>2.1. Whole-genome sequencing, sequence alignment, and screening of variant candidates</h3>
<p>The generation of transgenic rice expressing Japanese cedar pollen allergens was described previously.<sup><a reveal-id="DST036C3" data-open="DST036C3" class="link link-ref link-reveal xref-bibr">3</a></sup> OSCR11, a rice line containing only a single T-DNA insertion site, was obtained by segregation. The host rice lines such as a123 and OSCR11 were cultivated in a greenhouse under natural light conditions for 2 weeks. Genomic DNA was extracted from leaves using a DNeasy plant mini kit (Qiagen). Paired-end libraries with 450–500 bp insert sizes were constructed and sequenced on a HiSeq2000 (Illumina) instrument using a single lane for each sample and a 100-bp paired-end sequencing protocol. Sequence data are deposited into the DNA Data Bank of Japan Sequence Read Archive (accession no. DRA000918). Adaptor sequences were trimmed using cutadapt 1.0<sup><a reveal-id="DST036C11" data-open="DST036C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with parameters: −e 0.1 –O 5 –m 20. Low-quality bases with a Phred Score of &lt;Q20 at either side of each read were also trimmed. Reads shorter than 20 bp in length were discarded. Trimmed paired-end reads were aligned either to the <em>Agrobacterium</em> C58 genome sequence using BWA, or to the reference rice Nipponbare genome sequence (Os-Nipponbare-Reference-IRGSP-1.0) using the CLC Genomics Workbench 5.1 (CLC bio) with the following parameters: similarity, 0.9; length fraction, 0.9; insertion cost, 3; deletion cost, 3; mismatch cost, 3; non-specific matching, ignored. Structural variations (SVs) in OSCR11 were screened using the CLC Genomics Workbench 5.1 with the following parameters: <em>P</em>-value threshold, 1.0E−4; paired-read orientation, yes. Reads properly mapped in pair were extracted, and realigned to the reference with the same parameters. Whole-genome sequencing reads derived from Koshihikari rice (DRR000022–DRR000030) were downloaded from the Sequence Read Archive, and single-end 32-bp reads were aligned to the reference using the same parameters. After removal of duplicated reads, coverage was analysed using SAMtools 0.1.18. Single-nucleotide polymorphism (SNP) candidates were screened using the CLC Genomics Workbench 5.1 with the following parameters: window length, 11; maximum gap and mismatch count, 3; minimum coverage, 1; minimum variant frequency, 75%. Deletion/insertion polymorphism (DIP) candidates were screened by using the CLC Genomics Workbench 5.1 with the following parameters: minimum coverage, 1; minimum frequency, 50%.</p>                            <h3 scrollto-destination=6245063 id="6245063" class="section-title" data-legacy-id=s2b>2.2. mRNA-Seq analysis</h3>
<p>Total RNA was isolated from developing rice seeds at 10–20 days after flowering from each of three independent a123 and OSCR11 plants using a standard phenol/SDS method.<sup><a reveal-id="DST036C12" data-open="DST036C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Following DNase I treatment, mRNAs were enriched using two iterations of a Dynabeads mRNA purification kit (Life Technologies). Libraries were generated using an Ion total RNA-Seq v2 kit (Life Technologies) and sequenced by Ion Torrent PGM (Life Technologies) with an Ion 318 Chip (Life Technologies). Low-quality nucleotides (Phred &lt;Q20) were trimmed, and reads were then mapped onto the RAP-DB annotation<sup><a reveal-id="DST036C13" data-open="DST036C13" class="link link-ref link-reveal xref-bibr">13</a></sup> of the IRGSP-1.0 genome using the CLC Genomics Workbench 5.1, allowing two mismatches for short reads, and requiring a minimum length and similarity of 0.9 for long reads. Non-uniquely aligning reads were randomly mapped. Three biological replicates were analysed for each line, and differentially expressed genes (DEGs) were identified with the DESeq R package using a 5% false discovery rate (FDR) cut-off.<sup><a reveal-id="DST036C14" data-open="DST036C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p>                            <h2 scrollto-destination=6245071 id="6245071" class="section-title" data-legacy-id=s3>3. Results and Discussion</h2>
                            <h3 scrollto-destination=6245076 id="6245076" class="section-title" data-legacy-id=s3a>3.1. Rice genome re-sequencing</h3>
<p>Whole-genome sequencing of a homozygous T5 transgenic OSCR11 plant and its parental line a123 was performed on the Illumina HiSeq2000 platform. We also obtained the genome sequence data of Koshihikari, the background cultivar of a123, from the previous report.<sup><a reveal-id="DST036C15" data-open="DST036C15" class="link link-ref link-reveal xref-bibr">15</a></sup> We obtained 259, 223, and 268 million high-quality reads for Koshihikari, a123, and OSCR11, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Several reports have shown that <em>Agrobacterium</em> genome sequences are occasionally integrated into the host genome together with T-DNA.<sup><a reveal-id="DST036C16" data-open="DST036C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST036C17" data-open="DST036C17" class="link link-ref link-reveal xref-bibr">17</a></sup> To address this issue, we aligned OSCR11 reads to the <em>Agrobacterium</em> C58 strain reference genome<sup><a reveal-id="DST036C18" data-open="DST036C18" class="link link-ref link-reveal xref-bibr">18</a></sup> using the BWA software.<sup><a reveal-id="DST036C19" data-open="DST036C19" class="link link-ref link-reveal xref-bibr">19</a></sup> We did not detect any OSCR11 whole-genome sequence reads that aligned with C58, indicating that no portions of the <em>Agrobacterium</em> genome had been integrated into the OSCR11 rice genome.</p>                            <h3 scrollto-destination=6245084 id="6245084" class="section-title" data-legacy-id=s3b>3.2. Detection of structural variations</h3>
<p>To detect SVs, we mapped all of our rice sequence reads against the Nipponbare reference genome (Os-Nipponbare-Reference-IRGSP-1.0) using the CLC Genomics Workbench 5.1. We screened SV candidates in OSCR11 by comparing the computed distances of the paired-end reads with their locations within the Nipponbare reference. A total of 149 insertions, 280 deletions, 12 inversions, and 39 instances of interchromosome swapping were called. We validated these SVs by manual inspection of read alignments and found that all of the detected insertions, inversions, and interchromosome swappings were false positives. Although in each case several sequence reads supported the detected polymorphism, these regions were also covered by paired-end reads that supported the Nipponbare reference genome arrangement. Following manual inspection of the sequence alignments, a total of 187 higher-confidence deletions remained. Deletion length varied from 67 bp to 54 kb, and no deletions were detected on chr06 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). We compared the mapping results obtained with OSCR11, a123, and Koshihikari for deletions, and found an OSCR11 specific deletion of 810 bp on chr02 of OSCR11 (Fig. <a reveal-id="DST036F2" data-open="DST036F2" class="link link-reveal link-table xref-fig">2</a>A). The deletion was confirmed by genomic PCR and DNA sequencing (Fig. <a reveal-id="DST036F2" data-open="DST036F2" class="link link-reveal link-table xref-fig">2</a>B). Active transposons have been reported in rice callus.<sup><a reveal-id="DST036C20" data-open="DST036C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The deleted region on chr02 of OSCR11 contains several repeat units that are similar to transposons, but the junction sequences revealed that transposon translocation was not involved in this deletion (Fig. <a reveal-id="DST036F2" data-open="DST036F2" class="link link-reveal link-table xref-fig">2</a>A). This result contrasts to previous reports analysing somaclonal variations in <em>Arabidopsis</em> and rice, in which SVs were not detected.<sup><a reveal-id="DST036C21" data-open="DST036C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST036C22" data-open="DST036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> However, since a deletion was only detected at a single locus and deletions in repeat regions are common,<sup><a reveal-id="DST036C23" data-open="DST036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> we could not conclude the deletion was caused by <em>Agrobacterium</em> infection. There are no annotated rice genes within the 10-kb region around the deletion, suggesting that the deletion is unlikely to alter the expression of nearby genes. </p>                        <a id="6245091" scrollto-destination="6245091"></a>
<div id="DST036F2" data-id="DST036F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/m_dst03602.png?Expires=2147483647&Signature=ZyJeEiX7BwGjrCek3k0uu8DTNazo0lWyxnCFJvDO1GfZujq5Te8EO5dsBV3UMFXKKwUdHTsKV77ys~UlrAU~m-7d4hVnfWpYIq8SsZZVx5dpP2hrfX6bNQvs-7LhbsRGMQISx5rOMGL2BgcsZ1cZ0gcwMoNdI~YcOoyXrPSosVpgSVIvBkjhtUU9RiUs3xc0tm7i2h3gSepHL7pJcYLhUbDbvhN2zeWpyUjmdCRq63RqYOR7jYlEzxhiLNnu01WfwND9cvQSZHoRbXtdlf3ZR3Ml~~3jqcssSp5fMWnLgWhrVrv6buf01Fs7X4zwEiqlgXH7vhJzvYG4EILCkEn2pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OSCR11-specific 810-bp deletion. (A) Comparison of mapped reads between Koshihikari, a123, and OSCR11. Data were visualized using an Integrative Genomics Viewer.33 A schematic representation of the deleted region is shown below. (B) Confirmation of the OSCR11-specific deletion by PCR using primers indicated in A." path-from-xml="dst03602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6245091/dst03602.tiff" path-from-xml="dst03602.tiff" target="_blank">View large</a><a section="6245091" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst03602.png?Expires=2147483647&Signature=GPAoX67uEMIbWkX34BydSCIpDTTKi-m9kF81qCwf68lsUMzBm4m6SouuVDU5hahXxQ8D-C9Cj5LgIz~sIwLXSQsas7Mb7V107mUw6Nq4cJlV3qoz3qwzSV5no8-3dW~Ths9qcSV75kj04BcPHfGL19QYKjvpt5CrO3j~n6hTfj9c562qPJXEF0Vhutqyi36G8bWGZ9Wf5XQyjIEb8XeOZy5-ZPeehW43BKWLbAIPpy7soVreFCLOF1a7vZH3PWwewl3Tp9v6XJ-w0wIXOdva9rVoD12s84dFli-8cjEBoBnenCbxhj18zrTl-I7I5GZRPHRw9Gkce7CwyC0yPraXKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6245091&ar=421469&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OSCR11-specific 810-bp deletion. (A) Comparison of mapped reads between Koshihikari, a123, and OSCR11. Data were visualized using an Integrative Genomics Viewer.<sup><a reveal-id="DST036C33" data-open="DST036C33" class="link link-ref link-reveal xref-bibr">33</a></sup> A schematic representation of the deleted region is shown below. (B) Confirmation of the OSCR11-specific deletion by PCR using primers indicated in A.</p></div></div><div content-id="DST036F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/m_dst03602.png?Expires=2147483647&Signature=ZyJeEiX7BwGjrCek3k0uu8DTNazo0lWyxnCFJvDO1GfZujq5Te8EO5dsBV3UMFXKKwUdHTsKV77ys~UlrAU~m-7d4hVnfWpYIq8SsZZVx5dpP2hrfX6bNQvs-7LhbsRGMQISx5rOMGL2BgcsZ1cZ0gcwMoNdI~YcOoyXrPSosVpgSVIvBkjhtUU9RiUs3xc0tm7i2h3gSepHL7pJcYLhUbDbvhN2zeWpyUjmdCRq63RqYOR7jYlEzxhiLNnu01WfwND9cvQSZHoRbXtdlf3ZR3Ml~~3jqcssSp5fMWnLgWhrVrv6buf01Fs7X4zwEiqlgXH7vhJzvYG4EILCkEn2pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OSCR11-specific 810-bp deletion. (A) Comparison of mapped reads between Koshihikari, a123, and OSCR11. Data were visualized using an Integrative Genomics Viewer.33 A schematic representation of the deleted region is shown below. (B) Confirmation of the OSCR11-specific deletion by PCR using primers indicated in A." path-from-xml="dst03602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6245091/dst03602.tiff" path-from-xml="dst03602.tiff" target="_blank">View large</a><a section="6245091" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst03602.png?Expires=2147483647&Signature=GPAoX67uEMIbWkX34BydSCIpDTTKi-m9kF81qCwf68lsUMzBm4m6SouuVDU5hahXxQ8D-C9Cj5LgIz~sIwLXSQsas7Mb7V107mUw6Nq4cJlV3qoz3qwzSV5no8-3dW~Ths9qcSV75kj04BcPHfGL19QYKjvpt5CrO3j~n6hTfj9c562qPJXEF0Vhutqyi36G8bWGZ9Wf5XQyjIEb8XeOZy5-ZPeehW43BKWLbAIPpy7soVreFCLOF1a7vZH3PWwewl3Tp9v6XJ-w0wIXOdva9rVoD12s84dFli-8cjEBoBnenCbxhj18zrTl-I7I5GZRPHRw9Gkce7CwyC0yPraXKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6245091&ar=421469&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OSCR11-specific 810-bp deletion. (A) Comparison of mapped reads between Koshihikari, a123, and OSCR11. Data were visualized using an Integrative Genomics Viewer.<sup><a reveal-id="DST036C33" data-open="DST036C33" class="link link-ref link-reveal xref-bibr">33</a></sup> A schematic representation of the deleted region is shown below. (B) Confirmation of the OSCR11-specific deletion by PCR using primers indicated in A.</p></div></div>                            <h3 scrollto-destination=6245096 id="6245096" class="section-title" data-legacy-id=s3c>3.3. Detection of transformation- and mutagenesis-induced polymorphisms</h3>
<p>To identify specific SNPs and DIPs caused by <em>Agrobacterium</em>-mediated transformation procedures and mutagenesis by gamma ray and EMS, we used the Nipponbare genome as a reference to map our high-quality and clonal reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>). For OSCR11 and a123, paired reads with a mapping distance of ∼500 bp were used. Reads that mapped non-uniquely to multiple genomic positions were discarded in this analysis. Uniquely mapped reads covered 81–95% of the reference genome with 11.3–33.2× coverage. Both coverage and depth of coverage for each chromosome were similar (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Rice SNP candidates supported by four or more reads using Illumina HiSeq sequencing and the IRGSP-1.0 genome as a reference are considered to be high-confidence SNPs.<sup><a reveal-id="DST036C21" data-open="DST036C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Therefore, we screened putative SNP using this criterion. Because the depth of coverage across the genome was different between lines, it was difficult to specifically determine whether the SNPs are unique to any particular genotype. To discriminate transformation-induced polymorphisms from false positives, we excluded polymorphisms (a) whose location was not covered by a123 reads, or (b) that were called in Koshihikari or a123 and supported by at least one SNP-containing read in OSCR11. Similarly, to discriminate mutagenesis-induced polymorphisms from false positives, we excluded polymorphisms (a) whose location was not covered in Koshihikari and OSCR11 or (b) that were called in Koshihikari.</p><p>In agreement with the results of a previous study,<sup><a reveal-id="DST036C15" data-open="DST036C15" class="link link-ref link-reveal xref-bibr">15</a></sup> 63 480 SNPs relative to the Nipponbare reference were detected by four or more reads in Koshihikari, and 67 721 and 93 110 SNPs relative to Nipponbare were detected in a123 and OSCR11, respectively (Table <a reveal-id="DST036TB1" data-open="DST036TB1" class="link link-reveal link-table xref-fig">1</a>). We identified 167 transformation-induced SNPs and 939 mutagenesis-induced SNPs, covering 95.3% and 93.1% of the genome, respectively (Tables <a reveal-id="DST036TB1" data-open="DST036TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DST036TB2" data-open="DST036TB2" class="link link-reveal link-table xref-fig">2</a>). The ratio of transitions-to-transversions in transformation-induced SNPs was 1.0, which is similar to the ratio detected during somaclonal variation (Fig. <a reveal-id="DST036F3" data-open="DST036F3" class="link link-reveal link-table xref-fig">3</a>A).<sup><a reveal-id="DST036C21" data-open="DST036C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST036C22" data-open="DST036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In contrast, the transition : transversion ratio in mutagenesis-induced SNPs was 2.1, which is similar to the 2.4∼2.7 ratio reported for spontaneous mutations in <em>Arabidopsis</em> (Fig. <a reveal-id="DST036F3" data-open="DST036F3" class="link link-reveal link-table xref-fig">3</a>A).<sup><a reveal-id="DST036C24" data-open="DST036C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Most mutations caused by EMS are transitions, which reflect the chemical properties of this mutagen.<sup><a reveal-id="DST036C25" data-open="DST036C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Morita <em>et al</em>.<sup><a reveal-id="DST036C26" data-open="DST036C26" class="link link-ref link-reveal xref-bibr">26</a></sup> reported that base substitutions accounted for 12.5% of mutations caused by gamma ray-inducing DSB in rice, and all of them were transversions. The relatively high transition-to-transversion rate in this study may reflect these tendencies. </p>                        <a id="6245105" scrollto-destination="6245105"></a>
<div content-id="DST036TB1" class="table-modal"><div class="table-wrap"><div id="DST036TB1" class="table-wrap-title" data-id="DST036TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of polymorphisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Effectiveness </th><th colspan="2">Transformation-induced<hr /> </th><th colspan="2">Mutagenesis-induced<hr /> </th></tr><tr><th>SNP </th><th>DIP </th><th>SNP </th><th>DIP </th></tr></thead><tbody><tr><td>No polymorphism sites </td><td>Yes </td><td>355 561 133 </td><td>355 657 126 </td><td>347 347 999 </td><td>347 436 870 </td></tr><tr><td>Polymorphism sites </td><td> </td><td>93 110 </td><td>6953 </td><td>67 721 </td><td>6903 </td></tr><tr><td> Non-unique </td><td>Yes </td><td>91 035 </td><td>6290 </td><td>58 316 </td><td>3042 </td></tr><tr><td> Not covered in other lines </td><td>No </td><td>1908 </td><td>635 </td><td>8466 </td><td>714 </td></tr><tr><td> Unique </td><td>Yes </td><td>167 </td><td>28 </td><td>939 </td><td>3147 </td></tr><tr><td>Covered sites </td><td> </td><td>355 652 335 </td><td>355 677 204 </td><td>347 407 254 </td><td>347 446 813 </td></tr><tr><td>Covered sites (%) </td><td> </td><td>95.3 </td><td>95.3 </td><td>93.1 </td><td>93.1 </td></tr></tbody></table></div><div content-id="DST036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Effectiveness </th><th colspan="2">Transformation-induced<hr /> </th><th colspan="2">Mutagenesis-induced<hr /> </th></tr><tr><th>SNP </th><th>DIP </th><th>SNP </th><th>DIP </th></tr></thead><tbody><tr><td>No polymorphism sites </td><td>Yes </td><td>355 561 133 </td><td>355 657 126 </td><td>347 347 999 </td><td>347 436 870 </td></tr><tr><td>Polymorphism sites </td><td> </td><td>93 110 </td><td>6953 </td><td>67 721 </td><td>6903 </td></tr><tr><td> Non-unique </td><td>Yes </td><td>91 035 </td><td>6290 </td><td>58 316 </td><td>3042 </td></tr><tr><td> Not covered in other lines </td><td>No </td><td>1908 </td><td>635 </td><td>8466 </td><td>714 </td></tr><tr><td> Unique </td><td>Yes </td><td>167 </td><td>28 </td><td>939 </td><td>3147 </td></tr><tr><td>Covered sites </td><td> </td><td>355 652 335 </td><td>355 677 204 </td><td>347 407 254 </td><td>347 446 813 </td></tr><tr><td>Covered sites (%) </td><td> </td><td>95.3 </td><td>95.3 </td><td>93.1 </td><td>93.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Effectiveness indicates whether the sites were used for polymorphism calling and comparison.</p></div></div><div content-id=""><div class="fn"><p>Covered sites were the sum of effective sites.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6245105" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST036TB1" class="table-wrap-title" data-id="DST036TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of polymorphisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Effectiveness </th><th colspan="2">Transformation-induced<hr /> </th><th colspan="2">Mutagenesis-induced<hr /> </th></tr><tr><th>SNP </th><th>DIP </th><th>SNP </th><th>DIP </th></tr></thead><tbody><tr><td>No polymorphism sites </td><td>Yes </td><td>355 561 133 </td><td>355 657 126 </td><td>347 347 999 </td><td>347 436 870 </td></tr><tr><td>Polymorphism sites </td><td> </td><td>93 110 </td><td>6953 </td><td>67 721 </td><td>6903 </td></tr><tr><td> Non-unique </td><td>Yes </td><td>91 035 </td><td>6290 </td><td>58 316 </td><td>3042 </td></tr><tr><td> Not covered in other lines </td><td>No </td><td>1908 </td><td>635 </td><td>8466 </td><td>714 </td></tr><tr><td> Unique </td><td>Yes </td><td>167 </td><td>28 </td><td>939 </td><td>3147 </td></tr><tr><td>Covered sites </td><td> </td><td>355 652 335 </td><td>355 677 204 </td><td>347 407 254 </td><td>347 446 813 </td></tr><tr><td>Covered sites (%) </td><td> </td><td>95.3 </td><td>95.3 </td><td>93.1 </td><td>93.1 </td></tr></tbody></table></div><div content-id="DST036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Effectiveness </th><th colspan="2">Transformation-induced<hr /> </th><th colspan="2">Mutagenesis-induced<hr /> </th></tr><tr><th>SNP </th><th>DIP </th><th>SNP </th><th>DIP </th></tr></thead><tbody><tr><td>No polymorphism sites </td><td>Yes </td><td>355 561 133 </td><td>355 657 126 </td><td>347 347 999 </td><td>347 436 870 </td></tr><tr><td>Polymorphism sites </td><td> </td><td>93 110 </td><td>6953 </td><td>67 721 </td><td>6903 </td></tr><tr><td> Non-unique </td><td>Yes </td><td>91 035 </td><td>6290 </td><td>58 316 </td><td>3042 </td></tr><tr><td> Not covered in other lines </td><td>No </td><td>1908 </td><td>635 </td><td>8466 </td><td>714 </td></tr><tr><td> Unique </td><td>Yes </td><td>167 </td><td>28 </td><td>939 </td><td>3147 </td></tr><tr><td>Covered sites </td><td> </td><td>355 652 335 </td><td>355 677 204 </td><td>347 407 254 </td><td>347 446 813 </td></tr><tr><td>Covered sites (%) </td><td> </td><td>95.3 </td><td>95.3 </td><td>93.1 </td><td>93.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Effectiveness indicates whether the sites were used for polymorphism calling and comparison.</p></div></div><div content-id=""><div class="fn"><p>Covered sites were the sum of effective sites.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6245105" target="_blank">View Large</a></div></div>                        <a id="6245108" scrollto-destination="6245108"></a>
<div content-id="DST036TB2" class="table-modal"><div class="table-wrap"><div id="DST036TB2" class="table-wrap-title" data-id="DST036TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Annotation of SNPs and DIPs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Mutagenesis </th><th>Transformation </th></tr></thead><tbody><tr><td colspan="3"> SNPs </td></tr><tr><td>  Intergenic </td><td>776 </td><td>137 </td></tr><tr><td>  Genic </td><td>163 </td><td>30 </td></tr><tr><td>  Exon </td><td>116 </td><td>24 </td></tr><tr><td>  Synonymous </td><td>76 </td><td>13 </td></tr><tr><td>  Non-synonymous </td><td>40 </td><td>11 </td></tr><tr><td>  Intron </td><td>16 </td><td>3 </td></tr><tr><td>  UTR </td><td>31 </td><td>3 </td></tr><tr><td>  Total </td><td>939 </td><td>167 </td></tr><tr><td colspan="3"> DIPs </td></tr><tr><td>  Deletion </td><td>1473 </td><td>17 </td></tr><tr><td>  Insertion </td><td>1675 </td><td>11 </td></tr><tr><td>  Intergenic </td><td>2426 </td><td>23 </td></tr><tr><td>  Genic </td><td>722 </td><td>5 </td></tr><tr><td>  Exon </td><td>118 </td><td>0 </td></tr><tr><td>  Synonymous </td><td>0 </td><td>0 </td></tr><tr><td>  Non-synonymous </td><td>118 </td><td>0 </td></tr><tr><td>  Intron </td><td>400 </td><td>2 </td></tr><tr><td>  UTR </td><td>204 </td><td>3 </td></tr><tr><td>  Total </td><td>3147 </td><td>28 </td></tr></tbody></table></div><div content-id="DST036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Mutagenesis </th><th>Transformation </th></tr></thead><tbody><tr><td colspan="3"> SNPs </td></tr><tr><td>  Intergenic </td><td>776 </td><td>137 </td></tr><tr><td>  Genic </td><td>163 </td><td>30 </td></tr><tr><td>  Exon </td><td>116 </td><td>24 </td></tr><tr><td>  Synonymous </td><td>76 </td><td>13 </td></tr><tr><td>  Non-synonymous </td><td>40 </td><td>11 </td></tr><tr><td>  Intron </td><td>16 </td><td>3 </td></tr><tr><td>  UTR </td><td>31 </td><td>3 </td></tr><tr><td>  Total </td><td>939 </td><td>167 </td></tr><tr><td colspan="3"> DIPs </td></tr><tr><td>  Deletion </td><td>1473 </td><td>17 </td></tr><tr><td>  Insertion </td><td>1675 </td><td>11 </td></tr><tr><td>  Intergenic </td><td>2426 </td><td>23 </td></tr><tr><td>  Genic </td><td>722 </td><td>5 </td></tr><tr><td>  Exon </td><td>118 </td><td>0 </td></tr><tr><td>  Synonymous </td><td>0 </td><td>0 </td></tr><tr><td>  Non-synonymous </td><td>118 </td><td>0 </td></tr><tr><td>  Intron </td><td>400 </td><td>2 </td></tr><tr><td>  UTR </td><td>204 </td><td>3 </td></tr><tr><td>  Total </td><td>3147 </td><td>28 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6245108" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST036TB2" class="table-wrap-title" data-id="DST036TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Annotation of SNPs and DIPs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Mutagenesis </th><th>Transformation </th></tr></thead><tbody><tr><td colspan="3"> SNPs </td></tr><tr><td>  Intergenic </td><td>776 </td><td>137 </td></tr><tr><td>  Genic </td><td>163 </td><td>30 </td></tr><tr><td>  Exon </td><td>116 </td><td>24 </td></tr><tr><td>  Synonymous </td><td>76 </td><td>13 </td></tr><tr><td>  Non-synonymous </td><td>40 </td><td>11 </td></tr><tr><td>  Intron </td><td>16 </td><td>3 </td></tr><tr><td>  UTR </td><td>31 </td><td>3 </td></tr><tr><td>  Total </td><td>939 </td><td>167 </td></tr><tr><td colspan="3"> DIPs </td></tr><tr><td>  Deletion </td><td>1473 </td><td>17 </td></tr><tr><td>  Insertion </td><td>1675 </td><td>11 </td></tr><tr><td>  Intergenic </td><td>2426 </td><td>23 </td></tr><tr><td>  Genic </td><td>722 </td><td>5 </td></tr><tr><td>  Exon </td><td>118 </td><td>0 </td></tr><tr><td>  Synonymous </td><td>0 </td><td>0 </td></tr><tr><td>  Non-synonymous </td><td>118 </td><td>0 </td></tr><tr><td>  Intron </td><td>400 </td><td>2 </td></tr><tr><td>  UTR </td><td>204 </td><td>3 </td></tr><tr><td>  Total </td><td>3147 </td><td>28 </td></tr></tbody></table></div><div content-id="DST036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Mutagenesis </th><th>Transformation </th></tr></thead><tbody><tr><td colspan="3"> SNPs </td></tr><tr><td>  Intergenic </td><td>776 </td><td>137 </td></tr><tr><td>  Genic </td><td>163 </td><td>30 </td></tr><tr><td>  Exon </td><td>116 </td><td>24 </td></tr><tr><td>  Synonymous </td><td>76 </td><td>13 </td></tr><tr><td>  Non-synonymous </td><td>40 </td><td>11 </td></tr><tr><td>  Intron </td><td>16 </td><td>3 </td></tr><tr><td>  UTR </td><td>31 </td><td>3 </td></tr><tr><td>  Total </td><td>939 </td><td>167 </td></tr><tr><td colspan="3"> DIPs </td></tr><tr><td>  Deletion </td><td>1473 </td><td>17 </td></tr><tr><td>  Insertion </td><td>1675 </td><td>11 </td></tr><tr><td>  Intergenic </td><td>2426 </td><td>23 </td></tr><tr><td>  Genic </td><td>722 </td><td>5 </td></tr><tr><td>  Exon </td><td>118 </td><td>0 </td></tr><tr><td>  Synonymous </td><td>0 </td><td>0 </td></tr><tr><td>  Non-synonymous </td><td>118 </td><td>0 </td></tr><tr><td>  Intron </td><td>400 </td><td>2 </td></tr><tr><td>  UTR </td><td>204 </td><td>3 </td></tr><tr><td>  Total </td><td>3147 </td><td>28 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6245108" target="_blank">View Large</a></div></div>                        <a id="6245109" scrollto-destination="6245109"></a>
<div id="DST036F3" data-id="DST036F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/m_dst03603.png?Expires=2147483647&Signature=v3msoOtMuWI2zmV99rEsSa~QCQh-5ipN6Ld9Wi2C8tV~YXgSh8lClWDuIjsX7YO8NzKiBL3qvFvzPN6eRO1Oearn6Yql7ayUARVTJ2B32zCVgXPHc13jvXibqWVNp9nhOReX14ObtZD6UHE~uyzBgq795uZ2SK7z5k1D4s52rtjKdFg3XHA7QxoZc56pKqdW60HoBqySufWyNdppN5mXKI0vUsgPstYA99BuP58Ul5xUU2kX1p~t~bLYmpZLEj8i9scC2ejr4C6aFEOtYstVjB6j6qnqBUkCujbrEN4Dck-FwJYffcuKh9FBcg~JQudCSoozq6Vn-S68rMIVSykitQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of specific classes of mutation. (A) Distribution of specific classes of mutagenesis-specific (black) and transformation-specific (white) base substitution mutation. Y-axis indicates the ratio of the number of each type of base change to that of total base changes. (B) Distribution of sizes of transformation-specific deletions (black), transformation-specific insertions (dark gray), mutagenesis-specific deletions (light grey), and mutagenesis-specific insertions (white). Y-axis indicates the ratio of the number of each length of deletion or insertion to that of total deletion or insertion. (C) Transgenic-specific deletion and insertion mutations. Locations and flanking sequences are shown. Mono- or di-nucleotide context are highlighted by black letters. Lower case and bold letters indicate deletion and insertion, respectively." path-from-xml="dst03603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6245109/dst03603.tiff" path-from-xml="dst03603.tiff" target="_blank">View large</a><a section="6245109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst03603.png?Expires=2147483647&Signature=csDHk-zfJQrk~1vIwyyKDnQXhwYplTuZmZlB~iM4X2HgBuq6g1k81uu7~ZEVXIFsCkPR3nqKsDoH1j3XWCZ1XUFULh6La4KmHfNWT2vzP9fVn0xXAmouo9kE0kBiA9vuA2ixNWJxgwTe1yIomwMcl6nfSFIPv7mLG-JgY48rgL7nz0b3zyISFXL9EBeEhOdojEODR-rT0bXNuVNQGVsjPcw8BoEhkJQ1f1QeoFj2TxO2BIKebapEHVPRsEMH~YNx45IzwfJiI0~2s3WQwdEQXk5mhs7O9jVPlVDF3IfbyCpq4UeGfUMc1j2LIPPh0w-8uGZjnpZpjoIke~18Pl1~Fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6245109&ar=421469&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of specific classes of mutation. (A) Distribution of specific classes of mutagenesis-specific (black) and transformation-specific (white) base substitution mutation. <em>Y</em>-axis indicates the ratio of the number of each type of base change to that of total base changes. (B) Distribution of sizes of transformation-specific deletions (black), transformation-specific insertions (dark gray), mutagenesis-specific deletions (light grey), and mutagenesis-specific insertions (white). <em>Y</em>-axis indicates the ratio of the number of each length of deletion or insertion to that of total deletion or insertion. (C) Transgenic-specific deletion and insertion mutations. Locations and flanking sequences are shown. Mono- or di-nucleotide context are highlighted by black letters. Lower case and bold letters indicate deletion and insertion, respectively.</p></div></div><div content-id="DST036F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/m_dst03603.png?Expires=2147483647&Signature=v3msoOtMuWI2zmV99rEsSa~QCQh-5ipN6Ld9Wi2C8tV~YXgSh8lClWDuIjsX7YO8NzKiBL3qvFvzPN6eRO1Oearn6Yql7ayUARVTJ2B32zCVgXPHc13jvXibqWVNp9nhOReX14ObtZD6UHE~uyzBgq795uZ2SK7z5k1D4s52rtjKdFg3XHA7QxoZc56pKqdW60HoBqySufWyNdppN5mXKI0vUsgPstYA99BuP58Ul5xUU2kX1p~t~bLYmpZLEj8i9scC2ejr4C6aFEOtYstVjB6j6qnqBUkCujbrEN4Dck-FwJYffcuKh9FBcg~JQudCSoozq6Vn-S68rMIVSykitQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of specific classes of mutation. (A) Distribution of specific classes of mutagenesis-specific (black) and transformation-specific (white) base substitution mutation. Y-axis indicates the ratio of the number of each type of base change to that of total base changes. (B) Distribution of sizes of transformation-specific deletions (black), transformation-specific insertions (dark gray), mutagenesis-specific deletions (light grey), and mutagenesis-specific insertions (white). Y-axis indicates the ratio of the number of each length of deletion or insertion to that of total deletion or insertion. (C) Transgenic-specific deletion and insertion mutations. Locations and flanking sequences are shown. Mono- or di-nucleotide context are highlighted by black letters. Lower case and bold letters indicate deletion and insertion, respectively." path-from-xml="dst03603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6245109/dst03603.tiff" path-from-xml="dst03603.tiff" target="_blank">View large</a><a section="6245109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst03603.png?Expires=2147483647&Signature=csDHk-zfJQrk~1vIwyyKDnQXhwYplTuZmZlB~iM4X2HgBuq6g1k81uu7~ZEVXIFsCkPR3nqKsDoH1j3XWCZ1XUFULh6La4KmHfNWT2vzP9fVn0xXAmouo9kE0kBiA9vuA2ixNWJxgwTe1yIomwMcl6nfSFIPv7mLG-JgY48rgL7nz0b3zyISFXL9EBeEhOdojEODR-rT0bXNuVNQGVsjPcw8BoEhkJQ1f1QeoFj2TxO2BIKebapEHVPRsEMH~YNx45IzwfJiI0~2s3WQwdEQXk5mhs7O9jVPlVDF3IfbyCpq4UeGfUMc1j2LIPPh0w-8uGZjnpZpjoIke~18Pl1~Fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6245109&ar=421469&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of specific classes of mutation. (A) Distribution of specific classes of mutagenesis-specific (black) and transformation-specific (white) base substitution mutation. <em>Y</em>-axis indicates the ratio of the number of each type of base change to that of total base changes. (B) Distribution of sizes of transformation-specific deletions (black), transformation-specific insertions (dark gray), mutagenesis-specific deletions (light grey), and mutagenesis-specific insertions (white). <em>Y</em>-axis indicates the ratio of the number of each length of deletion or insertion to that of total deletion or insertion. (C) Transgenic-specific deletion and insertion mutations. Locations and flanking sequences are shown. Mono- or di-nucleotide context are highlighted by black letters. Lower case and bold letters indicate deletion and insertion, respectively.</p></div></div><p>We found 1386, 6903, and 6953 DIPs relative to the Nipponbare reference in Koshihikari, a123, and OSCR11, respectively (Table <a reveal-id="DST036TB1" data-open="DST036TB1" class="link link-reveal link-table xref-fig">1</a>). DIPs preferentially occurred in repetitive regions including homopolymers and microsatellites, as previously reported.<sup><a reveal-id="DST036C22" data-open="DST036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> We detected 28 transformation-induced DIPs and 3147 mutagenesis-induced DIPs, covering 95.3% and 93.1% of the genome, respectively (Tables <a reveal-id="DST036TB1" data-open="DST036TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DST036TB2" data-open="DST036TB2" class="link link-reveal link-table xref-fig">2</a>). Transformation-induced DIPs were 1 or 2 bp in size except for one 5-bp deletion (Fig. <a reveal-id="DST036F3" data-open="DST036F3" class="link link-reveal link-table xref-fig">3</a>B). Among them, all of the 1- or 2-bp DIPs occurred in a mono- or di-nucleotide context, but the 5-bp deletion (chr09: 13005724) did not occur in a polymeric context (Fig. <a reveal-id="DST036F3" data-open="DST036F3" class="link link-reveal link-table xref-fig">3</a>C). One or two bp DIPs may be generated by slippage during DNA replication. In <em>Arabidopsis</em>, somaclonal DIPs are 1 or 2 bp insertions or deletions in a polymeric context.<sup><a reveal-id="DST036C22" data-open="DST036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The 5-bp deletion might have been the result of the improper repair of a DSB caused by <em>Agrobacterium</em> infection, but without T-DNA integration. However, since 5-bp deletions in a non-polymeric context have been detected among mutagenesis-induced DIPs (Fig. <a reveal-id="DST036F3" data-open="DST036F3" class="link link-reveal link-table xref-fig">3</a>B), it might happen spontaneously.</p>                            <h3 scrollto-destination=6245120 id="6245120" class="section-title" data-legacy-id=s3d>3.4. Comparison of mutation rates</h3>
<p>We identified a total of 195 transformation-induced mutations and 4086 mutagenesis-induced mutations, and mutation rates (number of mutations/covered sites) were 5.5 × 10<sup>−7</sup> and 1.2 × 10<sup>−5</sup>, respectively. Both transformation-induced and mutagenesis-induced mutations were distributed across the genome in a largely uniform pattern, suggesting that they were randomly produced at a genome-wide level (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The calculated mutagenesis rates are 91- and 2000-fold higher than the spontaneous mutation rate of 6.0 × 10<sup>−9</sup> mutations/effective site calculated for <em>Arabidopsis</em>.<sup><a reveal-id="DST036C24" data-open="DST036C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Jiang <em>et al</em>.<sup><a reveal-id="DST036C22" data-open="DST036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> reported a theoretical mutation rate in <em>Arabidopsis</em> by multiplying the detected mutation rate by 4, resulting in a mutation rate of 1.1 × 10<sup>−6</sup>. The detected somaclonal mutation rate in <em>Arabidopsis</em> after 3 weeks of cell culture is ∼2.6 × 10<sup>−7</sup>. Miyao <em>et al</em>.<sup><a reveal-id="DST036C21" data-open="DST036C21" class="link link-ref link-reveal xref-bibr">21</a></sup> reported a somaclonal mutation rate of 1.7 × 10<sup>−6</sup> in rice after 5 months of cell culture. In the transformation process used for establishing the transgenic OSCR11 line, cell culture was performed for ∼2 months (callus induction: ∼1 month, <em>Agrobacterium</em> infection: 3 days, and selection: ∼1 month). Interestingly, somaclonal mutation rate per cell culture week (<em>Arabidopsis</em>: 0.86 × 10<sup>−7</sup> and rice: 0.85 × 10<sup>−7</sup>)<sup><a reveal-id="DST036C21" data-open="DST036C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST036C22" data-open="DST036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and the transformation mutation rate per cell culture week (0.68 × 10<sup>−7</sup>: in this study) were similar. Considering the base substitution rate of transformation-induced SNPs was highly comparable with the rate induced by somaclonal variation, transformation-induced SNPs cannot be readily distinguished from somaclonal variants. These results support previous reports, showing that cell culture is a primary mutation source during <em>Agrobacterium</em>-mediated transformation.<sup><a reveal-id="DST036C5" data-open="DST036C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>Among the transformation-induced mutations, 5.6% (11 SNPs) were non-synonymous, and altered the amino acid sequences of 11 genes (Tables <a reveal-id="DST036TB1" data-open="DST036TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DST036TB2" data-open="DST036TB2" class="link link-reveal link-table xref-fig">2</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Among the mutagenesis-induced mutations, 3.9% (40 SNPs, 44 deletions, and 74 insertions) were non-synonymous, and altered the amino acid sequences of 140 genes (Table <a reveal-id="DST036TB1" data-open="DST036TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Numbers of non-synonymous mutations caused by transformation and mutagenesis were much less than those between japonica rice cultivars Koshihikari (794 genes) or Omachi (1017 genes) and Nipponbare.<sup><a reveal-id="DST036C15" data-open="DST036C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST036C27" data-open="DST036C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Although we did not observe deleterious phenotypes in our rice lines, the non-synonymous mutations induced by transformation might cause some phenotypes under specific conditions. The mutations detected showed a largely uniform distribution and did not preferentially accumulate near the T-DNA insertion site or near loci involved in the glutelin-reduced trait (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Thus, undesired mutations can be easily eliminated by crossing.</p>                            <h3 scrollto-destination=6245132 id="6245132" class="section-title" data-legacy-id=s3e>3.5. Transcriptome analysis of the a123 and OSCR11 endosperms</h3>
<p>To investigate the influence of <em>Agrobacterium</em>-mediated transformation on the rice transcriptome, we performed strand-specific mRNA-Seq in the maturing endosperm of a123 and OSCR11 lines using Ion Torrent PGM. When OSCR11 seeds are provided to allergy patients as an edible vaccine, the embryo is removed by polishing; therefore, RNA was extracted from de-hulled endosperms whose embryos had been cut out. Since activities of promoters used for transgene expression are highest at 10–20 days after flowering, we chose this stage to analyse. We used single-end reads that mapped to known annotated exons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). We identified only 28 DEGs between OSCR11 and a123 using a FDR of &lt;0.05 (Table <a reveal-id="DST036TB3" data-open="DST036TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>), which is comparable with the small number of DEGs detected previously between transgenic rice and wild-type rice using microarray analyses. Among the DEGs, 11 were related to quality control proteins in the endoplasmic reticulum (ER), such as chaperones<sup><a reveal-id="DST036C28" data-open="DST036C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DST036C29" data-open="DST036C29" class="link link-ref link-reveal xref-bibr">29</a></sup> (Table <a reveal-id="DST036TB3" data-open="DST036TB3" class="link link-reveal link-table xref-fig">3</a>). The altered expression of these genes is likely caused by the accumulation of recombinant proteins (cedar pollen allergens) that induce ER stress. <em>OsbZIP50</em> is a cytoplasmic splice variant regulated by OsIRE1and is a molecular marker of strong ER stress.<sup><a reveal-id="DST036C28" data-open="DST036C28" class="link link-ref link-reveal xref-bibr">28</a></sup> However, we did not detect this spliced transcript, indicating that the OSCR11 endosperm is not under strong ER stress. In OSCR11, the T-DNA was inserted 27 bp upstream of the <em>Os08g0107400</em> 5′-UTR, suggesting that the upregulation of <em>Os08g0107400</em> was probably caused by the T-DNA insertion. Increased expression of <em>GluA-2/Os10g0400200</em> and <em>Os10g0207500</em>, components of T-DNA,<sup><a reveal-id="DST036C3" data-open="DST036C3" class="link link-ref link-reveal xref-bibr">3</a></sup> were directly derived from the transformed T-DNA. Half of DEGs, described above, could be explained by T-DNA insertion and introduced genes, or expected. </p>                        <a id="6245140" scrollto-destination="6245140"></a>
<div content-id="DST036TB3" class="table-modal"><div class="table-wrap"><div id="DST036TB3" class="table-wrap-title" data-id="DST036TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Differentially expressed genes between a123 and OSCR11</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">RAP-ID </th><th colspan="2">Average of RPKM<hr /> </th><th rowspan="2">Fold change </th><th rowspan="2">Annotation </th></tr><tr><th>a123 </th><th>OSCR11 </th></tr></thead><tbody><tr><td>Os03g0115800 </td><td>0.8 </td><td>155.0 </td><td>7.5 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os08g0107400 </td><td>1.0 </td><td>72.4 </td><td>6.1 </td><td>Similar to GDP-mannose transporter<sup>b</sup> </td></tr><tr><td>Os10g0400200 </td><td>132.3 </td><td>2293.5 </td><td>4.1 </td><td>GluA-2<sup>b</sup> </td></tr><tr><td>Os09g0538000 </td><td>0.0 </td><td>8.1 </td><td>N/A </td><td>Ribonuclease T2 family protein (OsRNS5) </td></tr><tr><td>Os01g0537250 </td><td>0.5 </td><td>115.1 </td><td>8.0 </td><td>Protein of unknown function DUF3778 domain-containing protein </td></tr><tr><td>Os09g0512700 </td><td>9.1 </td><td>40.1 </td><td>2.1 </td><td>Armadillo-like helical domain-containing protein (Fes1-like)<sup>a</sup> </td></tr><tr><td>Os01g0917100 </td><td>7.5 </td><td>0.3 </td><td>−4.7 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os04g0641101 </td><td>150.6 </td><td>48.1 </td><td>−1.7 </td><td>Non-protein coding transcript </td></tr><tr><td>Os06g0593100 </td><td>82.3 </td><td>219.7 </td><td>1.3 </td><td>Similar to UDP-galactose/UDP-glucose transporter<sup>a</sup> </td></tr><tr><td>Os01g0293000 </td><td>0.0 </td><td>4.3 </td><td>N/A </td><td>Similar to S-adenosylmethionine synthetase 1 (EC 2.5.1.6) </td></tr><tr><td>Os06g0622700 </td><td>63.8 </td><td>160.7 </td><td>1.3 </td><td>OsbZIP50<sup>a</sup> </td></tr><tr><td>Os01g0880800 </td><td>81.9 </td><td>32.2 </td><td>−1.4 </td><td>Acyl-[acyl-carrier-protein] desaturase </td></tr><tr><td>Os09g0451500 </td><td>109.7 </td><td>274.3 </td><td>1.3 </td><td>OsPDI2;3<sup>a</sup> </td></tr><tr><td>Os03g0182800 </td><td>30.4 </td><td>12.0 </td><td>−1.4 </td><td>OsERF3 </td></tr><tr><td>Os03g0293000 </td><td>19.4 </td><td>48.9 </td><td>1.3 </td><td>Similar to DnaJ domain-containing protein<sup>a</sup> </td></tr><tr><td>Os12g0504900 </td><td>15.0 </td><td>1.2 </td><td>−3.7 </td><td>Glucoamylase<sup>a</sup> </td></tr><tr><td>Os10g0207500 </td><td>0.0 </td><td>5.1 </td><td>N/A </td><td>Similar to TPD1<sup>a</sup> </td></tr><tr><td>Os09g0491100 </td><td>0.0 </td><td>2.0 </td><td>N/A </td><td>Similar to beta-primeverosidase (EC 3.2.1.149)<sup>a</sup> </td></tr><tr><td>Os07g0679400 </td><td>4.2 </td><td>0.8 </td><td>−2.3 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0693600 </td><td>0.1 </td><td>3.4 </td><td>4.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os02g0710900 </td><td>12.1 </td><td>26.8 </td><td>1.1 </td><td>Heat shock protein Hsp70 family protein<sup>a</sup> </td></tr><tr><td>Os04g0498900 </td><td>4.2 </td><td>0.5 </td><td>−3.0 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os05g0156500 </td><td>65.9 </td><td>145.2 </td><td>1.1 </td><td>Similar to Apobec-1-binding protein 2 (ERdj3B-like)<sup>a</sup> </td></tr><tr><td>Os09g0451000 </td><td>3.1 </td><td>17.0 </td><td>2.3 </td><td>OsACO1 </td></tr><tr><td>Os08g0142900 </td><td>64.6 </td><td>20.6 </td><td>−1.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0149200 </td><td>5.7 </td><td>13.3 </td><td>1.1 </td><td>Similar to Metallothionein-like protein type 2 </td></tr><tr><td>Os02g0565200 </td><td>50.3 </td><td>106.2 </td><td>1.0 </td><td>Microsomal signal peptidase 25-kDa subunit family protein<sup>a</sup> </td></tr><tr><td>Os06g0716700 </td><td>61.9 </td><td>129.8 </td><td>1.0 </td><td>Similar to heat shock protein 90<sup>a</sup> </td></tr></tbody></table></div><div content-id="DST036TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">RAP-ID </th><th colspan="2">Average of RPKM<hr /> </th><th rowspan="2">Fold change </th><th rowspan="2">Annotation </th></tr><tr><th>a123 </th><th>OSCR11 </th></tr></thead><tbody><tr><td>Os03g0115800 </td><td>0.8 </td><td>155.0 </td><td>7.5 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os08g0107400 </td><td>1.0 </td><td>72.4 </td><td>6.1 </td><td>Similar to GDP-mannose transporter<sup>b</sup> </td></tr><tr><td>Os10g0400200 </td><td>132.3 </td><td>2293.5 </td><td>4.1 </td><td>GluA-2<sup>b</sup> </td></tr><tr><td>Os09g0538000 </td><td>0.0 </td><td>8.1 </td><td>N/A </td><td>Ribonuclease T2 family protein (OsRNS5) </td></tr><tr><td>Os01g0537250 </td><td>0.5 </td><td>115.1 </td><td>8.0 </td><td>Protein of unknown function DUF3778 domain-containing protein </td></tr><tr><td>Os09g0512700 </td><td>9.1 </td><td>40.1 </td><td>2.1 </td><td>Armadillo-like helical domain-containing protein (Fes1-like)<sup>a</sup> </td></tr><tr><td>Os01g0917100 </td><td>7.5 </td><td>0.3 </td><td>−4.7 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os04g0641101 </td><td>150.6 </td><td>48.1 </td><td>−1.7 </td><td>Non-protein coding transcript </td></tr><tr><td>Os06g0593100 </td><td>82.3 </td><td>219.7 </td><td>1.3 </td><td>Similar to UDP-galactose/UDP-glucose transporter<sup>a</sup> </td></tr><tr><td>Os01g0293000 </td><td>0.0 </td><td>4.3 </td><td>N/A </td><td>Similar to S-adenosylmethionine synthetase 1 (EC 2.5.1.6) </td></tr><tr><td>Os06g0622700 </td><td>63.8 </td><td>160.7 </td><td>1.3 </td><td>OsbZIP50<sup>a</sup> </td></tr><tr><td>Os01g0880800 </td><td>81.9 </td><td>32.2 </td><td>−1.4 </td><td>Acyl-[acyl-carrier-protein] desaturase </td></tr><tr><td>Os09g0451500 </td><td>109.7 </td><td>274.3 </td><td>1.3 </td><td>OsPDI2;3<sup>a</sup> </td></tr><tr><td>Os03g0182800 </td><td>30.4 </td><td>12.0 </td><td>−1.4 </td><td>OsERF3 </td></tr><tr><td>Os03g0293000 </td><td>19.4 </td><td>48.9 </td><td>1.3 </td><td>Similar to DnaJ domain-containing protein<sup>a</sup> </td></tr><tr><td>Os12g0504900 </td><td>15.0 </td><td>1.2 </td><td>−3.7 </td><td>Glucoamylase<sup>a</sup> </td></tr><tr><td>Os10g0207500 </td><td>0.0 </td><td>5.1 </td><td>N/A </td><td>Similar to TPD1<sup>a</sup> </td></tr><tr><td>Os09g0491100 </td><td>0.0 </td><td>2.0 </td><td>N/A </td><td>Similar to beta-primeverosidase (EC 3.2.1.149)<sup>a</sup> </td></tr><tr><td>Os07g0679400 </td><td>4.2 </td><td>0.8 </td><td>−2.3 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0693600 </td><td>0.1 </td><td>3.4 </td><td>4.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os02g0710900 </td><td>12.1 </td><td>26.8 </td><td>1.1 </td><td>Heat shock protein Hsp70 family protein<sup>a</sup> </td></tr><tr><td>Os04g0498900 </td><td>4.2 </td><td>0.5 </td><td>−3.0 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os05g0156500 </td><td>65.9 </td><td>145.2 </td><td>1.1 </td><td>Similar to Apobec-1-binding protein 2 (ERdj3B-like)<sup>a</sup> </td></tr><tr><td>Os09g0451000 </td><td>3.1 </td><td>17.0 </td><td>2.3 </td><td>OsACO1 </td></tr><tr><td>Os08g0142900 </td><td>64.6 </td><td>20.6 </td><td>−1.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0149200 </td><td>5.7 </td><td>13.3 </td><td>1.1 </td><td>Similar to Metallothionein-like protein type 2 </td></tr><tr><td>Os02g0565200 </td><td>50.3 </td><td>106.2 </td><td>1.0 </td><td>Microsomal signal peptidase 25-kDa subunit family protein<sup>a</sup> </td></tr><tr><td>Os06g0716700 </td><td>61.9 </td><td>129.8 </td><td>1.0 </td><td>Similar to heat shock protein 90<sup>a</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Fold changes are represented as log2 phase (OSCR11 to a123).</p></div></div><div content-id=""><div class="fn"><p>Expression levels are normalized to Reads Per Kilobase of exon per Million mapped reads (RPKM).</p></div></div><div content-id=""><div class="fn"><p>N/A in Fold change indicating RPKM in a line was &gt;0, but the other was 0.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Genes associated with ER stress.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Genes directly related to integrated T-DNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6245140" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST036TB3" class="table-wrap-title" data-id="DST036TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Differentially expressed genes between a123 and OSCR11</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">RAP-ID </th><th colspan="2">Average of RPKM<hr /> </th><th rowspan="2">Fold change </th><th rowspan="2">Annotation </th></tr><tr><th>a123 </th><th>OSCR11 </th></tr></thead><tbody><tr><td>Os03g0115800 </td><td>0.8 </td><td>155.0 </td><td>7.5 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os08g0107400 </td><td>1.0 </td><td>72.4 </td><td>6.1 </td><td>Similar to GDP-mannose transporter<sup>b</sup> </td></tr><tr><td>Os10g0400200 </td><td>132.3 </td><td>2293.5 </td><td>4.1 </td><td>GluA-2<sup>b</sup> </td></tr><tr><td>Os09g0538000 </td><td>0.0 </td><td>8.1 </td><td>N/A </td><td>Ribonuclease T2 family protein (OsRNS5) </td></tr><tr><td>Os01g0537250 </td><td>0.5 </td><td>115.1 </td><td>8.0 </td><td>Protein of unknown function DUF3778 domain-containing protein </td></tr><tr><td>Os09g0512700 </td><td>9.1 </td><td>40.1 </td><td>2.1 </td><td>Armadillo-like helical domain-containing protein (Fes1-like)<sup>a</sup> </td></tr><tr><td>Os01g0917100 </td><td>7.5 </td><td>0.3 </td><td>−4.7 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os04g0641101 </td><td>150.6 </td><td>48.1 </td><td>−1.7 </td><td>Non-protein coding transcript </td></tr><tr><td>Os06g0593100 </td><td>82.3 </td><td>219.7 </td><td>1.3 </td><td>Similar to UDP-galactose/UDP-glucose transporter<sup>a</sup> </td></tr><tr><td>Os01g0293000 </td><td>0.0 </td><td>4.3 </td><td>N/A </td><td>Similar to S-adenosylmethionine synthetase 1 (EC 2.5.1.6) </td></tr><tr><td>Os06g0622700 </td><td>63.8 </td><td>160.7 </td><td>1.3 </td><td>OsbZIP50<sup>a</sup> </td></tr><tr><td>Os01g0880800 </td><td>81.9 </td><td>32.2 </td><td>−1.4 </td><td>Acyl-[acyl-carrier-protein] desaturase </td></tr><tr><td>Os09g0451500 </td><td>109.7 </td><td>274.3 </td><td>1.3 </td><td>OsPDI2;3<sup>a</sup> </td></tr><tr><td>Os03g0182800 </td><td>30.4 </td><td>12.0 </td><td>−1.4 </td><td>OsERF3 </td></tr><tr><td>Os03g0293000 </td><td>19.4 </td><td>48.9 </td><td>1.3 </td><td>Similar to DnaJ domain-containing protein<sup>a</sup> </td></tr><tr><td>Os12g0504900 </td><td>15.0 </td><td>1.2 </td><td>−3.7 </td><td>Glucoamylase<sup>a</sup> </td></tr><tr><td>Os10g0207500 </td><td>0.0 </td><td>5.1 </td><td>N/A </td><td>Similar to TPD1<sup>a</sup> </td></tr><tr><td>Os09g0491100 </td><td>0.0 </td><td>2.0 </td><td>N/A </td><td>Similar to beta-primeverosidase (EC 3.2.1.149)<sup>a</sup> </td></tr><tr><td>Os07g0679400 </td><td>4.2 </td><td>0.8 </td><td>−2.3 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0693600 </td><td>0.1 </td><td>3.4 </td><td>4.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os02g0710900 </td><td>12.1 </td><td>26.8 </td><td>1.1 </td><td>Heat shock protein Hsp70 family protein<sup>a</sup> </td></tr><tr><td>Os04g0498900 </td><td>4.2 </td><td>0.5 </td><td>−3.0 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os05g0156500 </td><td>65.9 </td><td>145.2 </td><td>1.1 </td><td>Similar to Apobec-1-binding protein 2 (ERdj3B-like)<sup>a</sup> </td></tr><tr><td>Os09g0451000 </td><td>3.1 </td><td>17.0 </td><td>2.3 </td><td>OsACO1 </td></tr><tr><td>Os08g0142900 </td><td>64.6 </td><td>20.6 </td><td>−1.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0149200 </td><td>5.7 </td><td>13.3 </td><td>1.1 </td><td>Similar to Metallothionein-like protein type 2 </td></tr><tr><td>Os02g0565200 </td><td>50.3 </td><td>106.2 </td><td>1.0 </td><td>Microsomal signal peptidase 25-kDa subunit family protein<sup>a</sup> </td></tr><tr><td>Os06g0716700 </td><td>61.9 </td><td>129.8 </td><td>1.0 </td><td>Similar to heat shock protein 90<sup>a</sup> </td></tr></tbody></table></div><div content-id="DST036TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">RAP-ID </th><th colspan="2">Average of RPKM<hr /> </th><th rowspan="2">Fold change </th><th rowspan="2">Annotation </th></tr><tr><th>a123 </th><th>OSCR11 </th></tr></thead><tbody><tr><td>Os03g0115800 </td><td>0.8 </td><td>155.0 </td><td>7.5 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os08g0107400 </td><td>1.0 </td><td>72.4 </td><td>6.1 </td><td>Similar to GDP-mannose transporter<sup>b</sup> </td></tr><tr><td>Os10g0400200 </td><td>132.3 </td><td>2293.5 </td><td>4.1 </td><td>GluA-2<sup>b</sup> </td></tr><tr><td>Os09g0538000 </td><td>0.0 </td><td>8.1 </td><td>N/A </td><td>Ribonuclease T2 family protein (OsRNS5) </td></tr><tr><td>Os01g0537250 </td><td>0.5 </td><td>115.1 </td><td>8.0 </td><td>Protein of unknown function DUF3778 domain-containing protein </td></tr><tr><td>Os09g0512700 </td><td>9.1 </td><td>40.1 </td><td>2.1 </td><td>Armadillo-like helical domain-containing protein (Fes1-like)<sup>a</sup> </td></tr><tr><td>Os01g0917100 </td><td>7.5 </td><td>0.3 </td><td>−4.7 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os04g0641101 </td><td>150.6 </td><td>48.1 </td><td>−1.7 </td><td>Non-protein coding transcript </td></tr><tr><td>Os06g0593100 </td><td>82.3 </td><td>219.7 </td><td>1.3 </td><td>Similar to UDP-galactose/UDP-glucose transporter<sup>a</sup> </td></tr><tr><td>Os01g0293000 </td><td>0.0 </td><td>4.3 </td><td>N/A </td><td>Similar to S-adenosylmethionine synthetase 1 (EC 2.5.1.6) </td></tr><tr><td>Os06g0622700 </td><td>63.8 </td><td>160.7 </td><td>1.3 </td><td>OsbZIP50<sup>a</sup> </td></tr><tr><td>Os01g0880800 </td><td>81.9 </td><td>32.2 </td><td>−1.4 </td><td>Acyl-[acyl-carrier-protein] desaturase </td></tr><tr><td>Os09g0451500 </td><td>109.7 </td><td>274.3 </td><td>1.3 </td><td>OsPDI2;3<sup>a</sup> </td></tr><tr><td>Os03g0182800 </td><td>30.4 </td><td>12.0 </td><td>−1.4 </td><td>OsERF3 </td></tr><tr><td>Os03g0293000 </td><td>19.4 </td><td>48.9 </td><td>1.3 </td><td>Similar to DnaJ domain-containing protein<sup>a</sup> </td></tr><tr><td>Os12g0504900 </td><td>15.0 </td><td>1.2 </td><td>−3.7 </td><td>Glucoamylase<sup>a</sup> </td></tr><tr><td>Os10g0207500 </td><td>0.0 </td><td>5.1 </td><td>N/A </td><td>Similar to TPD1<sup>a</sup> </td></tr><tr><td>Os09g0491100 </td><td>0.0 </td><td>2.0 </td><td>N/A </td><td>Similar to beta-primeverosidase (EC 3.2.1.149)<sup>a</sup> </td></tr><tr><td>Os07g0679400 </td><td>4.2 </td><td>0.8 </td><td>−2.3 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0693600 </td><td>0.1 </td><td>3.4 </td><td>4.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os02g0710900 </td><td>12.1 </td><td>26.8 </td><td>1.1 </td><td>Heat shock protein Hsp70 family protein<sup>a</sup> </td></tr><tr><td>Os04g0498900 </td><td>4.2 </td><td>0.5 </td><td>−3.0 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os05g0156500 </td><td>65.9 </td><td>145.2 </td><td>1.1 </td><td>Similar to Apobec-1-binding protein 2 (ERdj3B-like)<sup>a</sup> </td></tr><tr><td>Os09g0451000 </td><td>3.1 </td><td>17.0 </td><td>2.3 </td><td>OsACO1 </td></tr><tr><td>Os08g0142900 </td><td>64.6 </td><td>20.6 </td><td>−1.8 </td><td>Conserved hypothetical protein </td></tr><tr><td>Os01g0149200 </td><td>5.7 </td><td>13.3 </td><td>1.1 </td><td>Similar to Metallothionein-like protein type 2 </td></tr><tr><td>Os02g0565200 </td><td>50.3 </td><td>106.2 </td><td>1.0 </td><td>Microsomal signal peptidase 25-kDa subunit family protein<sup>a</sup> </td></tr><tr><td>Os06g0716700 </td><td>61.9 </td><td>129.8 </td><td>1.0 </td><td>Similar to heat shock protein 90<sup>a</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Fold changes are represented as log2 phase (OSCR11 to a123).</p></div></div><div content-id=""><div class="fn"><p>Expression levels are normalized to Reads Per Kilobase of exon per Million mapped reads (RPKM).</p></div></div><div content-id=""><div class="fn"><p>N/A in Fold change indicating RPKM in a line was &gt;0, but the other was 0.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Genes associated with ER stress.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Genes directly related to integrated T-DNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6245140" target="_blank">View Large</a></div></div><p>We detected no transformation-induced SNPs or DIPs within 2 kb upstream of any DEG, suggesting that chromosome damage caused by the transformation event had little effect on the endosperm transcriptome. Signalling pathways operating downstream of a few polymorphic genes, or the ER stress that accompanies transgene transformation, may be responsible for the differential gene expression in OSCR11. However, we cannot exclude the possibility that epigenetic memory of stress or other factors are responsible for the DEGs that were not directly or indirectly caused by the T-DNA insertion.</p><p>If there are significant damages on host genome by <em>Agrobacterium-</em>mediated transformation, there should be many base transversions and numerous DSB repair traces such as deletions in non-polymeric contexts, like gamma-ray treatment causes them. We demonstrated that mutations caused by the transformation event used in developing a transgenic OSCR11 rice line were mainly induced during the cell culture step and were almost indistinguishable from those associated with somaclonal variation, suggesting that there was an undetectable level of genomic damage, if any, caused by <em>Agrobacterium</em> infection other than T-DNA insertion. Furthermore, the calculated mutation rate between OSCR11 and the host rice line a123 was significantly lower than that between a123 and the original cultivar Koshihikari. Thus, the observed low frequency of mutation is consistent with only a subtle transcriptome difference between OSCR11 and a123. The transformation procedures employed in this study are standard practices in many laboratories, but the culture media were optimized for Koshihikari and its derivative cultivars. We did not perform any backcrosses or screening based on the agronomic traits of OSCR11, yet OSCR11 was almost identical to a123 at the genomic level. Although it will be necessary to analyse more lines to establish the general differences between GM crops and non-GM crops, we concluded that this transgenic line is almost identical to a123 at the genomic level.</p><p>Rapid progress has been made in gene targeting and recombination technologies to specifically edit target loci within the rice genome,<sup><a reveal-id="DST036C30  DST036C32" data-open="DST036C30  DST036C32" class="link link-ref link-reveal xref-bibr">30–32</a></sup> but the potential side effects of <em>Agrobacterium</em>-mediated transformation have not been fully explored. To obtain transgenic crops whose genome is identical to that of the host in every respect except for specific target loci, multiple backcrosses will be necessary. Our study provides valuable information for researchers who use transgenic approaches to improve crop yields and other traits. Classical molecular techniques such as Southern blotting are still useful and necessary to compare genetic identity, but NGS platforms have the added potential of whole-genome analysis of transgenic plants. Combining whole-genome sequencing with transcriptome analysis is a robust method to assess the genome integrity of a transgenic plant in relation to its host crop.</p>                            <h2 scrollto-destination=6245154 id="6245154" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6245166 id="6245166" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work is supported by grants from the Ministry of Agriculture, Forestry and Fisheries of Japan, ‘Agri-Health Translational Research Project’ grant to F.T., and ‘Development of Genome Information Database System for Innovation of Crop and Livestock Production’ to T.I.</p>                            <h2 scrollto-destination=6245170 id="6245170" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank E. Domon and Y. Katayose for helpful discussions of this work.</p>                            <h2 scrollto-destination=6245175 id="6245175" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST036C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Montagu</div> <div class="given-names">M.</div></div>. </div><div class="article-title">It is a long way to GM agriculture</div>, <div class="source">Ann. Rev. Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">1</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=It%20is%20a%20long%20way%20to%20GM%20agriculture&amp;author=M.%20Van%20Montagu&amp;publication_year=2011&amp;journal=Ann.%20Rev.%20Plant%20Biol.&amp;volume=62&amp;pages=1-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-042110-103906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-042110-103906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-042110-103906"> </span></div> </div></div></div></div></div><div content-id="DST036C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clive</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Global Status of Commercialized Biotech/GM Crops: 2011</div>, <div class="year">2011</div><div class="publisher-loc">Ithaca, NY</div><div class="publisher-name">ISAAA</div> <div class="comment"><em>ISAAA Brief</em></div></div></div></div></div><div content-id="DST036C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wakasa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hirose</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Oral immunotherapy with transgenic rice seed containing destructed Japanese cedar pollen allergens, Cry j 1 and Cry j 2, against Japanese cedar pollinosis</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2013</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">66</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oral%20immunotherapy%20with%20transgenic%20rice%20seed%20containing%20destructed%20Japanese%20cedar%20pollen%20allergens%2C%20Cry%20j%201%20and%20Cry%20j%202%2C%20against%20Japanese%20cedar%20pollinosis&amp;author=Y.%20Wakasa&amp;author=H.%20Takagi&amp;author=S.%20Hirose&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=11&amp;pages=66-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23066780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labra</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Savini</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bracale</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genomic changes in transgenic rice (<em>Oryza sativa</em> L.) plants produced by infecting calli with <em>Agrobacterium tumefaciens</em></div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2001</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">325</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20changes%20in%20transgenic%20rice%20(Oryza%20sativa%20L.)%20plants%20produced%20by%20infecting%20calli%20with%20Agrobacterium%20tumefaciens&amp;author=M.%20Labra&amp;author=C.%20Savini&amp;author=M.%20Bracale&amp;publication_year=2001&amp;journal=Plant%20Cell%20Rep.&amp;volume=20&amp;pages=325-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002990100329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002990100329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002990100329"> </span></div> </div></div></div></div></div><div content-id="DST036C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labra</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Vannini</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Grassi</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Genomic stability in <em>Arabidopsis thaliana</em> transgenic plants obtained by floral dip</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">1512</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20stability%20in%20Arabidopsis%20thaliana%20transgenic%20plants%20obtained%20by%20floral%20dip&amp;author=M.%20Labra&amp;author=C.%20Vannini&amp;author=F.%20Grassi&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=1512-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1773-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1773-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1773-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15300384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gelvin</div> <div class="given-names">S.B.</div></div>. </div><div class="article-title">Plant proteins involved in <em>Agrobacterium</em>-mediated genetic transformation</div>, <div class="source">Ann. Rev. Phytopathol.</div> , <div class="year">2010</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">45</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20proteins%20involved%20in%20Agrobacterium-mediated%20genetic%20transformation&amp;author=S.B.%20Gelvin&amp;publication_year=2010&amp;journal=Ann.%20Rev.%20Phytopathol.&amp;volume=48&amp;pages=45-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-phyto-080508-081852" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-phyto-080508-081852" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-phyto-080508-081852"> </span></div> </div></div></div></div></div><div content-id="DST036C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ming</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The draft genome of the transgenic tropical fruit tree papaya (<em>Carica papaya</em> Linnaeus)</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">452</div> (pg. <div class="fpage">991</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20the%20transgenic%20tropical%20fruit%20tree%20papaya%20(Carica%20papaya%20Linnaeus)&amp;author=R.%20Ming&amp;author=S.%20Hou&amp;author=Y.%20Feng&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=991-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06856" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06856" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06856"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18432245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Batista</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Saibo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lourenco</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Oliveira</div> <div class="given-names">M.M.</div></div>. </div><div class="article-title">Microarray analyses reveal that plant mutagenesis may induce more transcriptomic changes than transgene insertion</div>, <div class="source">Proc. Natl Acad. Sci. USA.</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">3640</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20analyses%20reveal%20that%20plant%20mutagenesis%20may%20induce%20more%20transcriptomic%20changes%20than%20transgene%20insertion&amp;author=R.%20Batista&amp;author=N.%20Saibo&amp;author=T.%20Lourenco&amp;author=M.M.%20Oliveira&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA.&amp;volume=105&amp;pages=3640-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0707881105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0707881105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0707881105"> </span></div> </div></div></div></div></div><div content-id="DST036C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Utsumi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takaiwa</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Foreign gene products can be enhanced by introduction into low storage protein mutants</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2003</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">411</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foreign%20gene%20products%20can%20be%20enhanced%20by%20introduction%20into%20low%20storage%20protein%20mutants&amp;author=Y.%20Tada&amp;author=S.%20Utsumi&amp;author=F.%20Takaiwa&amp;publication_year=2003&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=1&amp;pages=411-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1467-7652.2003.00038.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1467-7652.2003.00038.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1467-7652.2003.00038.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17134400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kusaba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Mutants lacking glutelin subunits in rice: mapping and combination of mutated glutelin genes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">177</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutants%20lacking%20glutelin%20subunits%20in%20rice%3A%20mapping%20and%20combination%20of%20mutated%20glutelin%20genes&amp;author=S.%20Iida&amp;author=M.%20Kusaba&amp;author=T.%20Nishio&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=94&amp;pages=177-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050397" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050397"> </span></div> </div></div></div></div></div><div content-id="DST036C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>, <div class="source">EMBnet J.</div> , <div class="year">2011</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">10</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=EMBnet%20J.&amp;volume=17&amp;pages=10-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="DST036C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takaiwa</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kikuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oono</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A rice glutelin gene family—a major type of glutelin messenger-RNAs can be divided into 2 classes</div>, <div class="source">Mol. Genet. Genomics.</div> , <div class="year">1987</div>, vol. <div class="volume">208</div> (pg. <div class="fpage">15</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rice%20glutelin%20gene%20family%E2%80%94a%20major%20type%20of%20glutelin%20messenger-RNAs%20can%20be%20divided%20into%202%20classes&amp;author=F.%20Takaiwa&amp;author=S.%20Kikuchi&amp;author=K.%20Oono&amp;publication_year=1987&amp;journal=Mol.%20Genet.%20Genomics.&amp;volume=208&amp;pages=15-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00330416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00330416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00330416"> </span></div> </div></div></div></div></div><div content-id="DST036C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Rice Annotation Project Database (RAP-DB): an integrative and interactive database for rice genomics</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2013</div>, vol. <div class="volume">54</div> pg. <div class="fpage">e6</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20Annotation%20Project%20Database%20(RAP-DB)%3A%20an%20integrative%20and%20interactive%20database%20for%20rice%20genomics&amp;author=H.%20Sakai&amp;author=S.S.%20Lee&amp;author=T.%20Tanaka&amp;publication_year=2013&amp;journal=Plant%20Cell%20Physiol.&amp;volume=54&amp;pages=e6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcs183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcs183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcs183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23299411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Differential expression analysis for sequence count data</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">R106</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-10-r106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-10-r106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-10-r106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yonemaru</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">267</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20definition%20of%20the%20pedigree%20haplotypes%20of%20closely%20related%20rice%20cultivars%20by%20means%20of%20genome-wide%20discovery%20of%20single-nucleotide%20polymorphisms&amp;author=T.%20Yamamoto&amp;author=H.%20Nagasaki&amp;author=J.%20Yonemaru&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20423466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ulker</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Rosso</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Logemann</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Somssich</div> <div class="given-names">I.E.</div></div>,  <div class="name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>. </div><div class="article-title">T-DNA-mediated transfer of <em>Agrobacterium tumefaciens</em> chromosomal DNA into plants</div>, <div class="source">Nature Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1015</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=T-DNA-mediated%20transfer%20of%20Agrobacterium%20tumefaciens%20chromosomal%20DNA%20into%20plants&amp;author=B.%20Ulker&amp;author=Y.%20Li&amp;author=M.G.%20Rosso&amp;author=E.%20Logemann&amp;author=I.E.%20Somssich&amp;author=B.%20Weisshaar&amp;publication_year=2008&amp;journal=Nature%20Biotechnol.&amp;volume=26&amp;pages=1015-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1491"> </span></div> </div></div></div></div></div><div content-id="DST036C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">An</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Bacterial transposons are co-transferred with T-DNA to rice chromosomes during <em>Agrobacterium</em>-mediated transformation</div>, <div class="source">Mol. Cells</div> , <div class="year">2012</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">583</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20transposons%20are%20co-transferred%20with%20T-DNA%20to%20rice%20chromosomes%20during%20Agrobacterium-mediated%20transformation&amp;author=S.R.%20Kim&amp;author=G.%20An&amp;publication_year=2012&amp;journal=Mol.%20Cells&amp;volume=33&amp;pages=583-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10059-012-0010-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10059-012-0010-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10059-012-0010-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22570148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wood</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Setubal</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Kaul</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The genome of the natural genetic engineer <em>Agrobacterium tumefaciens</em> C58</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">294</div> (pg. <div class="fpage">2317</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20natural%20genetic%20engineer%20Agrobacterium%20tumefaciens%20C58&amp;author=D.W.%20Wood&amp;author=J.C.%20Setubal&amp;author=R.%20Kaul&amp;publication_year=2001&amp;journal=Science&amp;volume=294&amp;pages=2317-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1066804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1066804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1066804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1754</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sabot</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Picault</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">El-Baidouri</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Transpositional landscape of the rice genome revealed by paired-end mapping of high-throughput re-sequencing data</div>, <div class="source">Plant J Cell Mol. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">241</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transpositional%20landscape%20of%20the%20rice%20genome%20revealed%20by%20paired-end%20mapping%20of%20high-throughput%20re-sequencing%20data&amp;author=F.%20Sabot&amp;author=N.%20Picault&amp;author=M.%20El-Baidouri&amp;publication_year=2011&amp;journal=Plant%20J%20Cell%20Mol.%20Biol.&amp;volume=66&amp;pages=241-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2011.66.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2011.66.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2011.66.issue-2"> </span></div> </div></div></div></div></div><div content-id="DST036C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyao</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nakagome</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohnuma</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Molecular spectrum of somaclonal variation in regenerated rice revealed by whole-genome sequencing</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2012</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">256</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20spectrum%20of%20somaclonal%20variation%20in%20regenerated%20rice%20revealed%20by%20whole-genome%20sequencing&amp;author=A.%20Miyao&amp;author=M.%20Nakagome&amp;author=T.%20Ohnuma&amp;publication_year=2012&amp;journal=Plant%20Cell%20Physiol.&amp;volume=53&amp;pages=256-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22156226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Mithani</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gan</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Regenerant Arabidopsis lineages display a distinct genome-wide spectrum of mutations conferring variant phenotypes</div>, <div class="source">Curr. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1385</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regenerant%20Arabidopsis%20lineages%20display%20a%20distinct%20genome-wide%20spectrum%20of%20mutations%20conferring%20variant%20phenotypes&amp;author=C.%20Jiang&amp;author=A.%20Mithani&amp;author=X.%20Gan&amp;publication_year=2011&amp;journal=Curr.%20Biol.&amp;volume=21&amp;pages=1385-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2011.07.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2011.07.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2011.07.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21802297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hurwitz</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Kudrna</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Rice structural variation: a comparative analysis of structural variation between rice and three of its closest relatives in the genus <em>Oryza</em></div>, <div class="source">Plant J Cell Mol Biol</div> , <div class="year">2010</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">990</div>-<div class="lpage">1003</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20structural%20variation%3A%20a%20comparative%20analysis%20of%20structural%20variation%20between%20rice%20and%20three%20of%20its%20closest%20relatives%20in%20the%20genus%20Oryza&amp;author=B.L.%20Hurwitz&amp;author=D.%20Kudrna&amp;author=Y.%20Yu&amp;publication_year=2010&amp;journal=Plant%20J%20Cell%20Mol%20Biol&amp;volume=63&amp;pages=990-1003" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2010.63.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.63.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2010.63.issue-6"> </span></div> </div></div></div></div></div><div content-id="DST036C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ossowski</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lucas-Lledo</div> <div class="given-names">J.I.</div></div>, et al. </div><div class="article-title">The rate and molecular spectrum of spontaneous mutations in <em>Arabidopsis thaliana</em></div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">92</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rate%20and%20molecular%20spectrum%20of%20spontaneous%20mutations%20in%20Arabidopsis%20thaliana&amp;author=S.%20Ossowski&amp;author=K.%20Schneeberger&amp;author=J.I.%20Lucas-Lledo&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=92-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1180677" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1180677" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1180677"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greene</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Codomo</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">N.E.</div></div>, et al. </div><div class="article-title">Spectrum of chemically induced mutations from a large-scale reverse-genetic screen in Arabidopsis</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">731</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spectrum%20of%20chemically%20induced%20mutations%20from%20a%20large-scale%20reverse-genetic%20screen%20in%20Arabidopsis&amp;author=E.A.%20Greene&amp;author=C.A.%20Codomo&amp;author=N.E.%20Taylor&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=731-40" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12807792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morita</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kusaba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Iida</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nishimura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Molecular characterization of mutations induced by gamma irradiation in rice</div>, <div class="source">Genes Genet. Syst.</div> , <div class="year">2009</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">361</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20mutations%20induced%20by%20gamma%20irradiation%20in%20rice&amp;author=R.%20Morita&amp;author=M.%20Kusaba&amp;author=S.%20Iida&amp;author=H.%20Yamaguchi&amp;author=T.%20Nishio&amp;author=M.%20Nishimura&amp;publication_year=2009&amp;journal=Genes%20Genet.%20Syst.&amp;volume=84&amp;pages=361-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.84.361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.84.361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.84.361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20154423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arai-Kichise</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shiwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Discovery of genome-wide DNA polymorphisms in a landrace cultivar of Japonica rice by whole-genome sequencing</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">274</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20genome-wide%20DNA%20polymorphisms%20in%20a%20landrace%20cultivar%20of%20Japonica%20rice%20by%20whole-genome%20sequencing&amp;author=Y.%20Arai-Kichise&amp;author=Y.%20Shiwa&amp;author=H.%20Nagasaki&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol.&amp;volume=52&amp;pages=274-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21258067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wakasa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kawakatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takaiwa</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Signal transduction by IRE1-mediated splicing of bZIP50 and other stress sensors in the endoplasmic reticulum stress response of rice</div>, <div class="source">Plant J.</div> , <div class="year">2012</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">946</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signal%20transduction%20by%20IRE1-mediated%20splicing%20of%20bZIP50%20and%20other%20stress%20sensors%20in%20the%20endoplasmic%20reticulum%20stress%20response%20of%20rice&amp;author=S.%20Hayashi&amp;author=Y.%20Wakasa&amp;author=H.%20Takahashi&amp;author=T.%20Kawakatsu&amp;author=F.%20Takaiwa&amp;publication_year=2012&amp;journal=Plant%20J.&amp;volume=69&amp;pages=946-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2011.04844.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2011.04844.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2011.04844.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22050533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oono</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wakasa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hirose</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sakuta</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Takaiwa</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Analysis of ER stress in developing rice endosperm accumulating beta-amyloid peptide</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2010</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">691</div>-<div class="lpage">718</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20ER%20stress%20in%20developing%20rice%20endosperm%20accumulating%20beta-amyloid%20peptide&amp;author=Y.%20Oono&amp;author=Y.%20Wakasa&amp;author=S.%20Hirose&amp;author=L.%20Yang&amp;author=C.%20Sakuta&amp;author=F.%20Takaiwa&amp;publication_year=2010&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=8&amp;pages=691-718" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2010.8.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2010.8.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2010.8.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20331531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Spalding</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Weeks</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">B.</div></div>. </div><div class="article-title">High-efficiency TALEN-based gene editing produces disease-resistant rice</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">390</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-efficiency%20TALEN-based%20gene%20editing%20produces%20disease-resistant%20rice&amp;author=T.%20Li&amp;author=B.%20Liu&amp;author=M.H.%20Spalding&amp;author=D.P.%20Weeks&amp;author=B.%20Yang&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=390-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22565958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Terada</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Urawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inagaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsugane</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Iida</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Efficient gene targeting by homologous recombination in rice</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2002</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1030</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20gene%20targeting%20by%20homologous%20recombination%20in%20rice&amp;author=R.%20Terada&amp;author=H.%20Urawa&amp;author=Y.%20Inagaki&amp;author=K.%20Tsugane&amp;author=S.%20Iida&amp;publication_year=2002&amp;journal=Nat.%20Biotechnol.&amp;volume=20&amp;pages=1030-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12219079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ozawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wakasa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ogo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Matsuo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kawahigashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takaiwa</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Development of an efficient <em>Agrobacterium</em>-mediated gene targeting system for rice and analysis of rice knockouts lacking granule-bound starch synthase (Waxy) and beta1,2-xylosyltransferase</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2012</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">755</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20an%20efficient%20Agrobacterium-mediated%20gene%20targeting%20system%20for%20rice%20and%20analysis%20of%20rice%20knockouts%20lacking%20granule-bound%20starch%20synthase%20(Waxy)%20and%20beta1%2C2-xylosyltransferase&amp;author=K.%20Ozawa&amp;author=Y.%20Wakasa&amp;author=Y.%20Ogo&amp;author=K.%20Matsuo&amp;author=H.%20Kawahigashi&amp;author=F.%20Takaiwa&amp;publication_year=2012&amp;journal=Plant%20Cell%20Physiol.&amp;volume=53&amp;pages=755-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcs016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcs016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcs016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22327484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST036C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST036C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robinson</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Thorvaldsdottir</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Winckler</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Integrative genomics viewer</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">24</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20genomics%20viewer&amp;author=J.T.%20Robinson&amp;author=H.%20Thorvaldsdottir&amp;author=W.%20Winckler&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=24-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1754" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21221095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=421469&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst036-dst036supp_tables.xlsx?Expires=2147483647&amp;Signature=hY20JJNb6I8G67Tux0JAIAE~Z0dKOt3rGikLmpqln1HQtvfVsMmgBAXn1W0w5QWdYb0CCsyCVrVnP0xzXn7uPsLs7asPK~v2nxViEWU-Ph1UfogQKLngkIP4mPRGym1HPhZBsIU5CC0AqCEyDuKdDqcRu~Ciw150~cFTOLha73DUdeRTaqcfEavXZu0zLcdcQrDcYXJ8ccvG-ODs74ORLIe3iXjMWfACY3YYunPYimqRbNTr24VFVo7QmmmzJypAZsYvfeR393W1pk-mQ3jrsIWJ7cDuhJhQoXnnitqkkcp2J9cbnvg8NoalNrIGCA0xNAxObSkL06LlmM-eByZv7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst036/2/dst036-dst036supp_figs.pdf?Expires=2147483647&amp;Signature=XVYkARTMs5Jn-Ev3bUkCxL4xTwlJJsDUg4coMq6vsnQvL88LFOpyZhok7qcFNgURhtQvZWcOdmVUFJPqXMtKksv6U-K3-SLx7up6kZYXU~iO2FXGVyNXVQI01kKz-zZ55xlWELf9DOKJVCIztKGfYazhoqx~RMEg1qYYx4eT-mQjNARlbOePbp6eY9Vyzq-MI1tRysi8RFNOMH7Z12aeKAmcj8GYEM64ZQgrF3aXf5PVrsqLVa0DzaF~5Pgmknly1WFO2dDtCpRHI5C5ouX~hlUiUfDJF~bDZ~0A6yBJOZtn9lHIb8xcJ1xcL5W96bgBcyNkjbnm8zbsHlmCQtyrNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">292</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst036"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="421469"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f6%2f623%2f421469">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330168100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F6%2F623%2F421469" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/421469'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Whole-Genome Analysis of a Transgenic Rice Seed-Based Edible Vaccine Against Cedar Pollen Allergy" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst036");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f623%2f421469">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

