

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/5/249/421165","@type":"ScholarlyArticle","name":"Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression","datePublished":"2009-09-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2009-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp016","keywords":["gene coexpression","Pearson's correlation coefficient","GeneChip summarization","Arabidopsis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/5/249/421165","author":[{"name":"Obayashi, Takeshi","affiliation":"1  Human Genome Center, Institute of Medical Science ,  The University of Tokyo ,  4-6-1 Shirokane-dai, Minato-ku, Tokyo 108-8639 ,  Japan","@type":"Person"},{"name":"Kinoshita, Kengo","affiliation":"1  Human Genome Center, Institute of Medical Science ,  The University of Tokyo ,  4-6-1 Shirokane-dai, Minato-ku, Tokyo 108-8639 ,  Japan    2  Institute for Bioinformatics Research and Development ,  Japan Science and Technology Agency ,  4-1-8 Honcho, Kawaguchi, Saitama 332-0012 ,  Japan","@type":"Person"}],"description":"Abstract.  Information regarding gene coexpression is useful to predict gene function. Several databases have been constructed for gene coexpression in model or","pageStart":"249","pageEnd":"260","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression" />
    <meta property="og:description" content="Abstract.  Information regarding gene coexpression is useful to predict gene function. Several databases have been constructed for gene coexpression in model or" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/5/249/421165" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-09-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Obayashi, Takeshi" /><meta name="citation_author_institution" content="Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1 Shirokane-dai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author" content="Kinoshita, Kengo" /><meta name="citation_author_institution" content="Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1 Shirokane-dai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author_institution" content="Institute for Bioinformatics Research and Development, Japan Science and Technology Agency, 4-1-8 Honcho, Kawaguchi, Saitama 332-0012, Japan" /><meta name="citation_title" content="Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression" /><meta name="citation_firstpage" content="249" /><meta name="citation_lastpage" content="260" /><meta name="citation_doi" content="10.1093/dnares/dsp016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2009/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=14863-8; " /><meta name="citation_reference" content="citation_title=Comprehensive identification of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization; Mol. Biol. Cell;  citation_year=1998;  citation_volume=9;  citation_pages=3273-97; " /><meta name="citation_reference" content="citation_title=Deciphering protein–protein interactions. Part I. Experimental techniques and databases; PLoS Comput. Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=e42" /><meta name="citation_reference" content="citation_title=KaPPA-view: a web-based analysis tool for integration of transcript and metabolite data on plant metabolic pathway maps; Plant Physiol.;  citation_year=2005;  citation_volume=138;  citation_pages=1289-300; " /><meta name="citation_reference" content="citation_title=Prediction of human disease genes by human–mouse conserved coexpression analysis; PLoS Comput. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000043" /><meta name="citation_reference" content="citation_title=Large-scale analysis of the human and mouse transcriptomes; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=4465-70; " /><meta name="citation_reference" content="citation_title=CSB.DB: a comprehensive systems-biology database; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3647-51; " /><meta name="citation_reference" content="citation_title=The botany array resource: e-Northerns, expression angling, and promoter analyses; Plant J.;  citation_year=2005;  citation_volume=43;  citation_pages=153-63; " /><meta name="citation_reference" content="citation_title=Arabidopsis Co-expression Tool (ACT): web server tools for microarray-based gene expression analysis; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W504-9; " /><meta name="citation_reference" content="citation_title=Annotating genes of known and unknown function by large-scale coexpression analysis; Plant Physiol.;  citation_year=2008;  citation_volume=147;  citation_pages=41-57; " /><meta name="citation_reference" content="citation_title=GeneCAT—novel webtools that combine BLAST and co-expression analyses; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=W320-6; " /><meta name="citation_reference" content="citation_title=CressExpress: a tool for large-scale mining of expression data from Arabidopsis; Plant Physiol.;  citation_year=2008;  citation_volume=147;  citation_pages=1004-16; " /><meta name="citation_reference" content="citation_title=COXPRESdb: a database of coexpressed gene networks in mammals; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D77-82; " /><meta name="citation_reference" content="citation_title=ATTED-II provides coexpressed gene networks for Arabidopsis; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D987-91; " /><meta name="citation_reference" content="citation_title=NCBI GEO: mining tens of millions of expression profiles—database and tools update; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D760-5; " /><meta name="citation_reference" content="citation_title=The Arabidopsis information resource (TAIR): gene structure and function annotation; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D1009-14; " /><meta name="citation_reference" content="citation_title=NASCArrays: a repository for microarray data generated by NASC's transcriptomics service; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D575-7; " /><meta name="citation_reference" content="citation_title=Robust estimators for expression analysis; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=1585-92; " /><meta name="citation_reference" content="citation_title=Effective similarity measures for expression profiles; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1616-22; " /><meta name="citation_reference" content="citation_title=A robust measure of correlation between two genes on a microarray; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=220" /><meta name="citation_reference" content="citation_title=Discovery of meaningful associations in genomic data using partial correlation coefficients; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3565-74; " /><meta name="citation_reference" content="citation_title=Approaches for extracting practical information from gene co-expression networks in plant biology; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=381-90; " /><meta name="citation_reference" content="citation_title=A gene-coexpression network for global discovery of conserved genetic modules; Science;  citation_year=2003;  citation_volume=302;  citation_pages=249-55; " /><meta name="citation_reference" content="citation_title=Wrestling with pleiotropy: genomic and topological analysis of the yeast gene expression network; Bioessays;  citation_year=2002;  citation_volume=24;  citation_pages=267-74; " /><meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data; Biostatistics;  citation_year=2003;  citation_volume=4;  citation_pages=249-64; " /><meta name="citation_reference" content="citation_title=A model based background adjustment for oligonucleotide expression arrays; J. Am. Stat. Assoc.;  citation_year=2004;  citation_volume=99;  citation_pages=909-17; " /><meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics; Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R80" /><meta name="citation_reference" content="citation_title=Interactively optimizing signal-to-noise ratios in expression profiling: project-specific algorithm selection and detection P-value weighting in Affymetrix microarrays; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2534-44; " /><meta name="citation_reference" content="citation_title=In silico gene function prediction using ontology-based pattern identification; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1237-45; " /><meta name="citation_reference" content="citation_title=Investigating semantic similarity measures across the Gene Ontology: the relationship between sequence and annotation; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=1275-83; " /><meta name="citation_reference" content="citation_title=Comparative analysis of microarray normalization procedures: effects on reverse engineering gene networks; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=i282-8; " /><meta name="citation_reference" content="citation_title=Identification of brassinosteroid-related genes by means of transcript co-response analyses; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=2685-96; " /><meta name="citation_reference" content="citation_title=Comparison of Affymetrix GeneChip expression measures; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=789-94; " /><meta name="citation_reference" content="citation_title=Using RNA sample titrations to assess microarray platform performance and normalization techniques; Nat. Biotechnol.;  citation_year=2006;  citation_volume=24;  citation_pages=1123-31; " /><meta name="citation_reference" content="citation_title=The utility of MAS5 expression summary and detection call algorithms; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=273" /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=A gene expression map of Arabidopsis thaliana development; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=501-6; " /><meta name="citation_reference" content="citation_title=The AtGenExpress global stress expression data set: protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses; Plant J.;  citation_year=2007;  citation_volume=50;  citation_pages=347-63; " /><meta name="citation_reference" content="citation_title=The AtGenExpress hormone and chemical treatment data set: experimental design, data evaluation, model data analysis and data access; Plant J.;  citation_year=2008;  citation_volume=55;  citation_pages=526-42; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/5/249/1166533/dsp016.pdf" /><meta name="description" content="Abstract.  Information regarding gene coexpression is useful to predict gene function. Several databases have been constructed for gene coexpression in model or" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/5/249/421165" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/5/249/421165" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/5/249/421165");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp016';
    </script>
<title>
	Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./421165" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYtUPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPYmF5YXNoaSwgVGFrZXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikh1bWFuIEdlbm9tZSBDZW50ZXIsIEluc3RpdHV0ZSBvZiBNZWRpY2FsIFNjaWVuY2UsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA0LTYtMSBTaGlyb2thbmUtZGFpLCBNaW5hdG8ta3UsIFRva3lvIDEwOC04NjM5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpbm9zaGl0YSwgS2VuZ28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJIdW1hbiBHZW5vbWUgQ2VudGVyLCBJbnN0aXR1dGUgb2YgTWVkaWNhbCBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNC02LTEgU2hpcm9rYW5lLWRhaSwgTWluYXRvLWt1LCBUb2t5byAxMDgtODYzOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIEJpb2luZm9ybWF0aWNzIFJlc2VhcmNoIGFuZCBEZXZlbG9wbWVudCwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBBZ2VuY3ksIDQtMS04IEhvbmNobywgS2F3YWd1Y2hpLCBTYWl0YW1hIDMzMi0wMDEyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUmFuayBvZiBDb3JyZWxhdGlvbiBDb2VmZmljaWVudCBhcyBhIENvbXBhcmFibGUgTWVhc3VyZSBmb3IgQmlvbG9naWNhbCBTaWduaWZpY2FuY2Ugb2YgR2VuZSBDb2V4cHJlc3Npb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcDAxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA5LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2x1c3RlciBhbmFseXNpcyBhbmQgZGlzcGxheSBvZiBnZW5vbWUtd2lkZSBleHByZXNzaW9uIHBhdHRlcm5zOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MTQ4NjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBpZGVudGlmaWNhdGlvbiBvZiBjZWxsIGN5Y2xlLXJlZ3VsYXRlZCBnZW5lcyBvZiB0aGUgeWVhc3QgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGJ5IG1pY3JvYXJyYXkgaHlicmlkaXphdGlvbjsgTW9sLiBCaW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMjczLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNpcGhlcmluZyBwcm90ZWlu4oCTcHJvdGVpbiBpbnRlcmFjdGlvbnMuIFBhcnQgSS4gRXhwZXJpbWVudGFsIHRlY2huaXF1ZXMgYW5kIGRhdGFiYXNlczsgUExvUyBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lNDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LYVBQQS12aWV3OiBhIHdlYi1iYXNlZCBhbmFseXNpcyB0b29sIGZvciBpbnRlZ3JhdGlvbiBvZiB0cmFuc2NyaXB0IGFuZCBtZXRhYm9saXRlIGRhdGEgb24gcGxhbnQgbWV0YWJvbGljIHBhdGh3YXkgbWFwczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM4OyAgY2l0YXRpb25fcGFnZXM9MTI4OS0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgaHVtYW4gZGlzZWFzZSBnZW5lcyBieSBodW1hbuKAk21vdXNlIGNvbnNlcnZlZCBjb2V4cHJlc3Npb24gYW5hbHlzaXM7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAwNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBhbmFseXNpcyBvZiB0aGUgaHVtYW4gYW5kIG1vdXNlIHRyYW5zY3JpcHRvbWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9NDQ2NS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1NCLkRCOiBhIGNvbXByZWhlbnNpdmUgc3lzdGVtcy1iaW9sb2d5IGRhdGFiYXNlOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTM2NDctNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBib3RhbnkgYXJyYXkgcmVzb3VyY2U6IGUtTm9ydGhlcm5zLCBleHByZXNzaW9uIGFuZ2xpbmcsIGFuZCBwcm9tb3RlciBhbmFseXNlczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xNTMtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIENvLWV4cHJlc3Npb24gVG9vbCAoQUNUKTogd2ViIHNlcnZlciB0b29scyBmb3IgbWljcm9hcnJheS1iYXNlZCBnZW5lIGV4cHJlc3Npb24gYW5hbHlzaXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPVc1MDQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5ub3RhdGluZyBnZW5lcyBvZiBrbm93biBhbmQgdW5rbm93biBmdW5jdGlvbiBieSBsYXJnZS1zY2FsZSBjb2V4cHJlc3Npb24gYW5hbHlzaXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTQxLTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lQ0FU4oCUbm92ZWwgd2VidG9vbHMgdGhhdCBjb21iaW5lIEJMQVNUIGFuZCBjby1leHByZXNzaW9uIGFuYWx5c2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1XMzIwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyZXNzRXhwcmVzczogYSB0b29sIGZvciBsYXJnZS1zY2FsZSBtaW5pbmcgb2YgZXhwcmVzc2lvbiBkYXRhIGZyb20gQXJhYmlkb3BzaXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTEwMDQtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNPWFBSRVNkYjogYSBkYXRhYmFzZSBvZiBjb2V4cHJlc3NlZCBnZW5lIG5ldHdvcmtzIGluIG1hbW1hbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQ3Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVRURUQtSUkgcHJvdmlkZXMgY29leHByZXNzZWQgZ2VuZSBuZXR3b3JrcyBmb3IgQXJhYmlkb3BzaXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQ5ODctOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5DQkkgR0VPOiBtaW5pbmcgdGVucyBvZiBtaWxsaW9ucyBvZiBleHByZXNzaW9uIHByb2ZpbGVz4oCUZGF0YWJhc2UgYW5kIHRvb2xzIHVwZGF0ZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDc2MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgaW5mb3JtYXRpb24gcmVzb3VyY2UgKFRBSVIpOiBnZW5lIHN0cnVjdHVyZSBhbmQgZnVuY3Rpb24gYW5ub3RhdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDEwMDktMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5BU0NBcnJheXM6IGEgcmVwb3NpdG9yeSBmb3IgbWljcm9hcnJheSBkYXRhIGdlbmVyYXRlZCBieSBOQVNDJ3MgdHJhbnNjcmlwdG9taWNzIHNlcnZpY2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPUQ1NzUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9idXN0IGVzdGltYXRvcnMgZm9yIGV4cHJlc3Npb24gYW5hbHlzaXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTU4NS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0aXZlIHNpbWlsYXJpdHkgbWVhc3VyZXMgZm9yIGV4cHJlc3Npb24gcHJvZmlsZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTYxNi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2J1c3QgbWVhc3VyZSBvZiBjb3JyZWxhdGlvbiBiZXR3ZWVuIHR3byBnZW5lcyBvbiBhIG1pY3JvYXJyYXk7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIG1lYW5pbmdmdWwgYXNzb2NpYXRpb25zIGluIGdlbm9taWMgZGF0YSB1c2luZyBwYXJ0aWFsIGNvcnJlbGF0aW9uIGNvZWZmaWNpZW50czsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0zNTY1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHByb2FjaGVzIGZvciBleHRyYWN0aW5nIHByYWN0aWNhbCBpbmZvcm1hdGlvbiBmcm9tIGdlbmUgY28tZXhwcmVzc2lvbiBuZXR3b3JrcyBpbiBwbGFudCBiaW9sb2d5OyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MzgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmUtY29leHByZXNzaW9uIG5ldHdvcmsgZm9yIGdsb2JhbCBkaXNjb3Zlcnkgb2YgY29uc2VydmVkIGdlbmV0aWMgbW9kdWxlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz0yNDktNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdyZXN0bGluZyB3aXRoIHBsZWlvdHJvcHk6IGdlbm9taWMgYW5kIHRvcG9sb2dpY2FsIGFuYWx5c2lzIG9mIHRoZSB5ZWFzdCBnZW5lIGV4cHJlc3Npb24gbmV0d29yazsgQmlvZXNzYXlzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjY3LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb3JhdGlvbiwgbm9ybWFsaXphdGlvbiwgYW5kIHN1bW1hcmllcyBvZiBoaWdoIGRlbnNpdHkgb2xpZ29udWNsZW90aWRlIGFycmF5IHByb2JlIGxldmVsIGRhdGE7IEJpb3N0YXRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI0OS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtb2RlbCBiYXNlZCBiYWNrZ3JvdW5kIGFkanVzdG1lbnQgZm9yIG9saWdvbnVjbGVvdGlkZSBleHByZXNzaW9uIGFycmF5czsgSi4gQW0uIFN0YXQuIEFzc29jLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTkwOS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvY29uZHVjdG9yOiBvcGVuIHNvZnR3YXJlIGRldmVsb3BtZW50IGZvciBjb21wdXRhdGlvbmFsIGJpb2xvZ3kgYW5kIGJpb2luZm9ybWF0aWNzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPVI4MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyYWN0aXZlbHkgb3B0aW1pemluZyBzaWduYWwtdG8tbm9pc2UgcmF0aW9zIGluIGV4cHJlc3Npb24gcHJvZmlsaW5nOiBwcm9qZWN0LXNwZWNpZmljIGFsZ29yaXRobSBzZWxlY3Rpb24gYW5kIGRldGVjdGlvbiBQLXZhbHVlIHdlaWdodGluZyBpbiBBZmZ5bWV0cml4IG1pY3JvYXJyYXlzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI1MzQtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHNpbGljbyBnZW5lIGZ1bmN0aW9uIHByZWRpY3Rpb24gdXNpbmcgb250b2xvZ3ktYmFzZWQgcGF0dGVybiBpZGVudGlmaWNhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMjM3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnZlc3RpZ2F0aW5nIHNlbWFudGljIHNpbWlsYXJpdHkgbWVhc3VyZXMgYWNyb3NzIHRoZSBHZW5lIE9udG9sb2d5OiB0aGUgcmVsYXRpb25zaGlwIGJldHdlZW4gc2VxdWVuY2UgYW5kIGFubm90YXRpb247IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTI3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgbWljcm9hcnJheSBub3JtYWxpemF0aW9uIHByb2NlZHVyZXM6IGVmZmVjdHMgb24gcmV2ZXJzZSBlbmdpbmVlcmluZyBnZW5lIG5ldHdvcmtzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPWkyODItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYnJhc3Npbm9zdGVyb2lkLXJlbGF0ZWQgZ2VuZXMgYnkgbWVhbnMgb2YgdHJhbnNjcmlwdCBjby1yZXNwb25zZSBhbmFseXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MjY4NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBBZmZ5bWV0cml4IEdlbmVDaGlwIGV4cHJlc3Npb24gbWVhc3VyZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9Nzg5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBSTkEgc2FtcGxlIHRpdHJhdGlvbnMgdG8gYXNzZXNzIG1pY3JvYXJyYXkgcGxhdGZvcm0gcGVyZm9ybWFuY2UgYW5kIG5vcm1hbGl6YXRpb24gdGVjaG5pcXVlczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTExMjMtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB1dGlsaXR5IG9mIE1BUzUgZXhwcmVzc2lvbiBzdW1tYXJ5IGFuZCBkZXRlY3Rpb24gY2FsbCBhbGdvcml0aG1zOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGZsb3dlcmluZyBwbGFudCBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgY2l0YXRpb25fYXV0aG9yPUFyYWJpZG9wc2lzIEdlbm9tZSBJbml0aWF0aXZlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lIGV4cHJlc3Npb24gbWFwIG9mIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGRldmVsb3BtZW50OyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTUwMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXRHZW5FeHByZXNzIGdsb2JhbCBzdHJlc3MgZXhwcmVzc2lvbiBkYXRhIHNldDogcHJvdG9jb2xzLCBldmFsdWF0aW9uIGFuZCBtb2RlbCBkYXRhIGFuYWx5c2lzIG9mIFVWLUIgbGlnaHQsIGRyb3VnaHQgYW5kIGNvbGQgc3RyZXNzIHJlc3BvbnNlczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0zNDctNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBdEdlbkV4cHJlc3MgaG9ybW9uZSBhbmQgY2hlbWljYWwgdHJlYXRtZW50IGRhdGEgc2V0OiBleHBlcmltZW50YWwgZGVzaWduLCBkYXRhIGV2YWx1YXRpb24sIG1vZGVsIGRhdGEgYW5hbHlzaXMgYW5kIGRhdGEgYWNjZXNzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTUyNi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzUvMjQ5LzExNjY1MzMvZHNwMDE2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBJbmZvcm1hdGlvbiByZWdhcmRpbmcgZ2VuZSBjb2V4cHJlc3Npb24gaXMgdXNlZnVsIHRvIHByZWRpY3QgZ2VuZSBmdW5jdGlvbi4gU2V2ZXJhbCBkYXRhYmFzZXMgaGF2ZSBiZWVuIGNvbnN0cnVjdGVkIGZvciBnZW5lIGNvZXhwcmVzc2lvbiBpbiBtb2RlbCBvciIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDIxMTY1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKuix5v4bg3HVxkHeHK0anFUUqo6" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f249%2f421165" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f249%2f421165" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&amp;Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/5">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6239790">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239790">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6239792">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239792">1 Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6239798">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239798">2 Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6239810">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239810">3 Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6239843">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239843">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6239845">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239845">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6239847">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6239847">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/16/5/261/420676">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Obayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Human Genome Center, Institute of Medical Science</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">4-6-1 Shirokane-dai, Minato-ku, Tokyo 108-8639</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Obayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Obayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Obayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeshi Obayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kengo Kinoshita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Human Genome Center, Institute of Medical Science</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">4-6-1 Shirokane-dai, Minato-ku, Tokyo 108-8639</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Institute for Bioinformatics Research and Development</div>, <div class="insititution">Japan Science and Technology Agency</div>, <div class="addr-line">4-1-8 Honcho, Kawaguchi, Saitama 332-0012</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 3-5449-5131. Fax. <div class="fax">+81 3-5449-5133</div>. E-mail: <a href="mailto:kino@ims.u-tokyo.ac.jp" target="_blank">kino@ims.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kengo+Kinoshita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kinoshita K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kinoshita K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kengo Kinoshita</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 249–260, <a href='https://doi.org/10.1093/dnares/dsp016'>https://doi.org/10.1093/dnares/dsp016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 September 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 March 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">14 August 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="421165" href="/dnaresearch/article-pdf/16/5/249/1166533/dsp016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takeshi Obayashi, Kengo Kinoshita;  Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression, <em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 249–260, <a href='https://doi.org/10.1093/dnares/dsp016'>https://doi.org/10.1093/dnares/dsp016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/421165?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/421165?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/421165?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/421165?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/421165?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/421165?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/421165?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Rank%20of%20Correlation%20Coefficient%20as%20a%20Comparable%20Measure%20for%20Biological%20Significance%20of%20Gene%20Coexpression&amp;publicationDate=2009-09-18&amp;volumeNum=16&amp;issueNum=5&amp;author=Obayashi%2C%20Takeshi%3B%20Kinoshita%2C%20Kengo&amp;startPage=249&amp;endPage=260&amp;contentId=10.1093%2Fdnares%2Fdsp016&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f249%2f421165" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6239790 id="6239790" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Information regarding gene coexpression is useful to predict gene function. Several databases have been constructed for gene coexpression in model organisms based on a large amount of publicly available gene expression data measured by GeneChip platforms. In these databases, Pearson's correlation coefficients (PCCs) of gene expression patterns are widely used as a measure of gene coexpression. Although the coexpression measure or GeneChip summarization method affects the performance of the gene coexpression database, previous studies for these calculation procedures were tested with only a small number of samples and a particular species. To evaluate the effectiveness of coexpression measures, assessments with large-scale microarray data are required. We first examined characteristics of PCC and found that the optimal PCC threshold to retrieve functionally related genes was affected by the method of gene expression database construction and the target gene function. In addition, we found that this problem could be overcome when we used correlation ranks instead of correlation values. This observation was evaluated by large-scale gene expression data for four species: Arabidopsis, human, mouse and rat.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene coexpression&quot;">gene coexpression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Pearson's correlation coefficient&quot;">Pearson's correlation coefficient</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;GeneChip summarization&quot;">GeneChip summarization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Arabidopsis">Arabidopsis</a></div>                            <h2 scrollto-destination=6239792 id="6239792" class="section-title" data-legacy-id=s1>1 Introduction</h2>
<p>The function of every gene depends on that of another gene(s). To predict gene partnerships, gene coexpression databases can be used, because coexpressed genes are generally expected to be involved in related cellular functions.<sup><a reveal-id="DSP016C1" data-open="DSP016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Many technical improvements have been achieved in microarray measurements, and thus coexpression databases are now widely used for various experimental objectives, such as gene targeting, regulatory investigations and/or identification of potential partners in protein–protein interactions.<sup><a reveal-id="DSP016C2" data-open="DSP016C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP016C3" data-open="DSP016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In addition to target-specified research, gene coexpression data provide a fundamental basis for omics studies such as the metabolome or phenome.<sup><a reveal-id="DSP016C4" data-open="DSP016C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSP016C5" data-open="DSP016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> To easily extract pertinent information from gene coexpression data, gene coexpression databases with various analysing tools have been constructed for model organisms.<sup><a reveal-id="DSP016C6  DSP016C14" data-open="DSP016C6  DSP016C14" class="link link-ref link-reveal xref-bibr">6–14</a></sup></p><p>In these databases, gene coexpression data, which are similarities of expression patterns of gene pairs over a number of samples, have been calculated using publicly available gene expression data produced using the Affymetrix GeneChip system, most of which are stored in primary databases such as GEO,<sup><a reveal-id="DSP016C15" data-open="DSP016C15" class="link link-ref link-reveal xref-bibr">15</a></sup> TAIR<sup><a reveal-id="DSP016C16" data-open="DSP016C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and NASCArray.<sup><a reveal-id="DSP016C17" data-open="DSP016C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Calculation of gene coexpression can be divided into the following three general steps: (i) selection of microarray samples, (ii) normalization of gene expression data and (iii) calculation of gene coexpression. In many coexpression databases, the MAS5 algorithm<sup><a reveal-id="DSP016C18" data-open="DSP016C18" class="link link-ref link-reveal xref-bibr">18</a></sup> is used for GeneChip summarization, and Pearson's correlation coefficients (PCCs) are used to measure gene coexpression. In this paper, we especially focus on coexpression measure for construction and usage of large-scale gene coexpression databases.</p><p>PCC is one of the most convenient measures to evaluate gene expression similarities<sup><a reveal-id="DSP016C1" data-open="DSP016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> because it is easy to calculate and is familiar to experimental biologists; still, certain caveats have been reported. Yona <em>et al</em>.<sup><a reveal-id="DSP016C19" data-open="DSP016C19" class="link link-ref link-reveal xref-bibr">19</a></sup> assessed five coexpression measures based on four types of yeast microarray data. The ‘mass–distance’ measure, which they proposed, showed stably higher performance than others, although the most effective measure differed between data sets. Hardin <em>et al</em>.<sup><a reveal-id="DSP016C20" data-open="DSP016C20" class="link link-ref link-reveal xref-bibr">20</a></sup> reported the insufficiency of PCC against outliers using a small data set based on 25 microarray slides. de la Fuente <em>et al</em>.<sup><a reveal-id="DSP016C21" data-open="DSP016C21" class="link link-ref link-reveal xref-bibr">21</a></sup> proposed the use of a partial correlation coefficient with PCC to improve causal properties. Although these studies are valuable for the calculation of gene coexpression, it is difficult to directly apply the results to large-scale database construction, because of its extremely high calculation cost for large-scale collections of gene expression data. Condition-independent gene coexpression data are constructed based on hundreds or thousands of GeneChip data to generalize specific experimental conditions; for example, for Arabidopsis, 1310, 1779 and 1388 GeneChips are used for PED,<sup><a reveal-id="DSP016C10" data-open="DSP016C10" class="link link-ref link-reveal xref-bibr">10</a></sup> CressExpress<sup><a reveal-id="DSP016C12" data-open="DSP016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and ATTED-II,<sup><a reveal-id="DSP016C14" data-open="DSP016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> respectively. To balance high performance and easy calculation for database construction, we re-examined coexpression measures using large-scale gene expression data. Note that we did not specially focus causal relationship, but effective retrieval of coexpressed genes to find functional partner of a gene(s) of interest.</p><p>To retrieve coexpressed gene sets, users have to set a threshold of coexpression value, because coexpression databases basically return continuous values indicating how strong the two genes of interest are coexpressed in selected samples. Aoki <em>et al</em>.<sup><a reveal-id="DSP016C22" data-open="DSP016C22" class="link link-ref link-reveal xref-bibr">22</a></sup> specified a minimal PCC value (0.55–0.66) for coexpressed gene retrieval to minimize false gene function relationships. Because the calculation was based on downloadable coexpression data from ATTED-II,<sup><a reveal-id="DSP016C14" data-open="DSP016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> this estimation is valuable for the users of this database. However, experimental biologists can now use many coexpression databases that were constructed using different samples and different normalization procedures. In addition, some databases provide their own options for sample selection and/or for coexpression measures other than PCC. Moreover, comparison of coexpression data among different species has been a powerful approach to investigate functional modules.<sup><a reveal-id="DSP016C23" data-open="DSP016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> A question arising from this situation is whether users can apply a particular threshold (e.g. PCC = 0.6) to retrieve coexpressed genes for any databases with any options as pointed out by Manfield <em>et al</em>.<sup><a reveal-id="DSP016C9" data-open="DSP016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Another question is whether the PCC threshold can be applied to any genes for various functions such as direct interactions for protein complex formation, successive reactions by enzymes in a particular metabolic pathway or gene regulatory relationships. The number of coexpressed genes under a particular PCC threshold follows a power-law distribution.<sup><a reveal-id="DSP016C24" data-open="DSP016C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Namely, a small number of genes are coexpressed with thousands of other genes, whereas a large number of genes are not coexpressed with any (or only a few) other genes. Although PCC is a convenient measure of expression similarity between gene pairs of interest, it may not directly indicate the strength of gene functional relationships. The fundamental question we address here is how the strength of gene functional relationships is affected by calculation procedures or gene functions. For simplicity, we hereafter refer to the strength of a gene functional relationship as ‘biological significance’.</p><p>In this study, we first investigated characteristics of PCC as a measure of gene coexpression. The problems we found for PCC fall into two categories, both of which lead low comparability of PCC value. The first case is derived from the construction of gene expression data, which includes the selection of microarray samples (a specific condition or a mixture of various conditions) and of GeneChip summarization methods [RMA,<sup><a reveal-id="DSP016C25" data-open="DSP016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> GCRMA,<sup><a reveal-id="DSP016C26" data-open="DSP016C26" class="link link-ref link-reveal xref-bibr">26</a></sup> MAS5<sup><a reveal-id="DSP016C18" data-open="DSP016C18" class="link link-ref link-reveal xref-bibr">18</a></sup> or PLIER (Affymetrix Technical Note, <a class="link link-uri" href="http://www.affymetrix.com/support/technical/technotes/plier_technote.pdf">http://www.affymetrix.com/support/technical/technotes/plier_technote.pdf</a>)]. The second case is derived from the variety of cellular functions that the gene of interest concerned. The biological significance of a PCC value depends on each type of coexpression data, and thus users cannot directly compare PCC values obtained from different types of coexpression data. To obtain a comparable coexpression measure, we tested several measures and found that correlation rank could normalize these differences and can be used as the comparable measure. Four model species—Arabidopsis, human, mouse and rat—were used to confirm that the correlation rank was useful to directly compare the coexpression level among different genes, conditions and species.</p>                            <h2 scrollto-destination=6239798 id="6239798" class="section-title" data-legacy-id=s2>2 Materials and methods</h2>
                            <h3 scrollto-destination=6239799 id="6239799" class="section-title" data-legacy-id=s2a>2.1 Data source of microarray experiments</h3>
<p>To calculate condition-independent gene coexpression data, we constructed gene expression profiles using as many genes and samples as possible. Towards this end, we selected the GeneChip platforms shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Because some samples were manually omitted due to different GeneChip usage, e.g. ChIP-on-chip or heterohybridization of close species, we used the following number of GeneChips; 1388 for Arabidopsis, 5188 for human, 2226 for mouse and 632 for rat. The samples used are shown in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>. GeneChip summarizations were performed for each experiment using BioConducter packages.<sup><a reveal-id="DSP016C27" data-open="DSP016C27" class="link link-ref link-reveal xref-bibr">27</a></sup> PLIER was run with quantile normalization, and an offset of 10 was added to expression values of PLIER and MAS5. If these options were not applied, GO prediction performance in this study significantly decreased (data not shown). All the expression values used in this study were in base-2 logarithm. For each experiment, an average expression level for each gene was subtracted to normalize differences in basal expression levels between experiments. Finally, all experiments were combined into one large expression matrix, which was constructed for each summarization method and for each species.</p>                            <h3 scrollto-destination=6239801 id="6239801" class="section-title" data-legacy-id=s2b>2.2 Calculation of gene-to-gene PCC for gene coexpression</h3>
<p>Even if technical replications are normalized, some data (e.g. a large series of time-course experiments under a single biological condition) are biologically redundant and result in unfairly biased gene expression data. Because these unexpected biases will affect the PCC values, we used a gene-to-gene PCC weighted by the following sample information scores <em>W</em><sub><em>S</em></sub>. The weighted PCC has been applied to evaluate the relationships between two variables with data samples by unbalance manner, and also used for microarray analyses.<sup><a reveal-id="DSP016C28" data-open="DSP016C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSP016C29" data-open="DSP016C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The weight <em>W</em><sub><em>Sa</em></sub> for a sample <em>Sa</em> of interest was derived from the sample-to-sample similarity <em>J</em><sub><em>Sa,Sx</em></sub> between the sample <em>Sa</em> and anyone of the sample <em>Sx</em>, which was calculated as PCC between the two samples. To focus on significantly similar samples, we introduced the cut-off threshold <em>C</em>. If the sample similarity <em>J</em><sub><em>Sa</em></sub><sub><em>,</em></sub><sub><em>Sx</em></sub> is smaller than the cut-off threshold <em>C</em>, <em>J</em>′<sub><em>Sa,Sx</em></sub> is set to 0. On the other hand, if <em>J</em><sub><em>Sa,Sx</em></sub> is larger than <em>C</em>, <em>J</em>′<sub><em>Sa,Sx</em></sub>= (<em>J</em>′<sub><em>Sa,Sx</em></sub> − <em>C</em>)/(1 − <em>C</em>), so that the range of sample-to-sample similarity <em>J</em>′<sub><em>Sa,Sx</em></sub> becomes 0 to 1. We roughly optimized the cut-off threshold <em>C</em> and used 0.4. We did not carry out a fine optimization because the results were not sensitive to this parameter. The sample redundancy <em>J</em><sub><em>Sa</em></sub> for the sample <em>Sa</em> is calculated as the summation of the <em>J</em>′<sub><em>Sa,Sx</em></sub>, namely <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01606.gif?Expires=2147483647&Signature=cp7Ss-s1od9gjzFYobU~vxC~goPP1eThhby0YXXk6l9ZGTB-eVNbShbLIL40tCqmOsSorzbiWp4rqGtaV3js1CqkofMk3Ho~GY8TtMyXC1ijJAIuy8jAzHK2jZ22Jbm8Zq4DQZVOkjiDOv7eAeT3Cnw9EEMHaY~4Fh18UlguXt2nKaZ9Q29Iko1-HTFuXAoHUhLYMlZmkEjbqXQUCKD0xL-pI1YKG~nSumPOPlZ7gJpXnuKd0ssxRGsBTMnGQPjJPPKIKlDp0TSwxiXuWzWSvJHAdrGjm2SnZAl9nBhBMIL~CCrp9u6UDxgQ-6zmvEmz8oyBUoinOVLYQYWYCVi3Fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp01606.gif" />⁠</span>. Because a large <em>J</em><sub><em>Sa</em></sub> value indicates highly redundant and thus a poorly informative sample, weight of the sample <em>Sa</em>, <em>W</em><sub><em>Sa</em></sub>, was defined as the inverse of the square root of the sample redundancy <em>J</em><sub><em>Sa</em></sub>. This procedure is analogous to the calculation of the standard error from the standard deviation, where the number of samples corresponds to the sample redundancy. If the sample <em>S</em> is replicated four times without experimental error, the reliability of the data for the sample <em>S</em> is doubled. Finally, the weighted PCC was calculated between two probes, according to the formula of weighted PCC<sup><a reveal-id="DSP016C28" data-open="DSP016C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with the weight described here. This weighted PCC was used as PCC in this study. Negative PCC values were used as is, namely treated as weaker relationships than zero PCC value, because negative correlations did not promote gene function prediction in our data (see also Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>C, where red dots indicating the same GO annotation to each of the reference gene did not significantly appear in right-bottom anti-correlation region). The effectiveness of the weight was not fully evaluated and it can be different with data set. In our data set, the weighted PCC gave slightly better performance than non-weighted PCC, but the differences were not so large in our data set.</p><p>In the case of Arabidopsis, genes with a single probe set are used, so that a probe-to-probe correlation directory indicates gene-to-gene correlation. For human, mouse and rat cases, the probe-to-probe correlations are transformed to gene-to-gene correlations using the maximum correlation value between all pairs of probes between the two genes, because most of genes in these three species are supported by multiple probe sets.</p>                            <h3 scrollto-destination=6239804 id="6239804" class="section-title" data-legacy-id=s2c>2.3 Calculation of PCC rank and mutual rank for gene coexpression</h3>
<p>When we focus one gene of interest, we can obtain coexpressed gene list sorted by PCC values between the gene of interest and all other genes on the microarray. ‘PCC rank’ used in this study is the rank of the gene in the PCC-ordered gene list. When gene A is third strongly coexpressed genes for gene B, PCC rank of gene A to gene B is 3. This PCC rank was used as one of coexpression measures. Since PCC rank between two genes of interest can be different, we introduced another coexpression measure, mutual rank (MR), by taking a geometric average of the PCC rank from gene A to gene B and that of gene B to gene A. The reason why we used ‘geometric average’ rather than arithmetic average is that we think that the difference of PCC ranks will change as logarithmic manner. For example, the impact of the difference of 1 and 3 in PCC rank can be similar with that of 100 and 300. Actually, the geometric-averaged MR showed slightly better gene prediction performances than arithmetic-averaged MR (data not shown).</p>                            <h3 scrollto-destination=6239806 id="6239806" class="section-title" data-legacy-id=s2d>2.4 Extraction of GO terms for the assessment of gene coexpression</h3>
<p>Each of the three categories of Gene Ontology (GO) Annotation,<sup><a reveal-id="DSP016C16" data-open="DSP016C16" class="link link-ref link-reveal xref-bibr">16</a></sup> namely biological process (BP), cellular component (CC) and molecular function (MF), was used to annotate gene function. Because GO terms have hierarchical topology with different importance, we selected appropriate GO terms to represent gene function. The selection was conducted based on the information content of GO terms.<sup><a reveal-id="DSP016C30" data-open="DSP016C30" class="link link-ref link-reveal xref-bibr">30</a></sup> All annotations were first mapped to all upper GO terms up to the root terms. Because terms that are associated with too many genes have less informative annotations, and thus could not be used to construct new experiments, we used GO terms associated with &gt;4 and &lt;20 genes. As a result, 729 BP terms, 147 CC terms and 391 MF terms were selected on average for the four species. Although we chose this range of gene numbers (i.e. &gt;4 and &lt;20) based on the characteristics of the randomized coexpressed gene lists to be AUC = 0.5 (see next section for AUC), we reached the same conclusion even using other ranges. The statistics of the selected GO terms are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p>                            <h3 scrollto-destination=6239808 id="6239808" class="section-title" data-legacy-id=s2e>2.5 Prediction of gene function</h3>
<p>We iteratively applied the nearest neighbour approach to predict GO annotations for each reference gene. The actual procedure was conducted as follows. First, we applied it to GO annotations of the most strongly coexpressed gene to the reference gene. Then, the GO annotations of the second-most strongly coexpressed gene were applied. In the same way, GO annotations of all coexpressed genes were iteratively applied. One of the characteristics of this prediction method is that it does not require parameter optimization. Another characteristic is that we can introduce any thresholds to define valid coexpressed genes. Using the predicted result on various thresholds, we generated receiver operating characteristic (ROC) curves, which is a plot of true-positive rate [TP/(TP + FN)] against false-positive rate [TN/(FP + TN)] with all possible threshold values, where TP, FN, TN and FP are the number of true positives, false negatives, true negatives and false positives, respectively. In gene function prediction, the number of positive gene-to-function relationships is far smaller than that of negative relationships. To evaluate such unbalanced data, evaluation measure such as overall accuracy, the Mathews correlation coefficient and F-measure is not adequate. A representative ROC curve is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. Because the ROC curves produced in this analysis showed standard convex-upward shapes, we simply showed the area under the ROC curve (i.e. AUC) to compare the effectiveness of the coexpression data. For example of BP in Arabidopsis, 376 GO annotations for 2280 genes were tested (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), where all possible associations were 857 280, and positive associations were 3705. We calculated true- and false-positive rates for all possible thresholds (857 280) to draw the ROC curve, and then AUC were obtained as the average of the true-positive rates for the false-positive rate ranging from 0 to 1 by 0.0001 steps.</p>                            <h2 scrollto-destination=6239810 id="6239810" class="section-title" data-legacy-id=s3>3 Results and discussion</h2>
                            <h3 scrollto-destination=6239811 id="6239811" class="section-title" data-legacy-id=s3a>3.1 Problems with PCC values</h3>
<p>We first used the PCC value as a measure of coexpression but found that the inferred biological significance varied depending on the type of coexpression data. This problem can be attributed to two main factors: (i) different gene expression data and data treatments were used in each database, and (ii) different kinds of cellular functions require differences in the strength of gene coexpression. Examples of these two possibilities for Arabidopsis are shown in Figs <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>, respectively.</p>                        <a id="6239813" scrollto-destination="6239813"></a>
<div id="DSP016F1" data-id="DSP016F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01601.jpeg?Expires=2147483647&Signature=Y5eYEuUEr7rxc1Mk2nV7cngqnIGzEShDRQFh8rR3Dl6w3cgN52gybsfxVjR9aaDt9NYvKhLJS4Q5QaZU~VYvnyf-AsZDWVm6HiY~D3FKXns2hKDix9B7ojmLInXYG9IBUjTSXSBgpShbW7pOvxpqymeZJyLgCPO5mkhTJw6FwOGf49~YOLwthSobXYFLDoKy47bFBExIFU0HLpyAng9H8sCBQ2tRe3vLkt~Fw64OHVb4mcLZlQuLC9oBcoIWz07LKUSSTX6f0vshaP3Tspm1wIt-gZ5ASemC5Tw-r-B1WqlszGq5iFEEEBvqQbg7-Gbn8Y4ZkwiGNN~SXOhgPkfJiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributions of PCC values between each gene and its strongest coexpressed gene in Arabidopsis. (A) Distribution of PCC values calculated from 1388 GeneChip slides (black), 237 slides (red) and 328 slides (green), both of which were summarized by RMA. (B) Distribution of PCC values calculated from 1388 GeneChip slides summarized by four methods (RMA, GCRMA, MAS5 and PLIER)." path-from-xml="dsp01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239813/dsp01601.jpeg" path-from-xml="dsp01601.jpeg" target="_blank">View large</a><a section="6239813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01601.jpeg?Expires=2147483647&Signature=sVquAH~OFRP2SbxusR9-asdlyeUw8QH0EGQ-Ug0sw-~HWrIR6wwcFlH7hHzveRKt6qVlzo9TgSMfyTE~qwdlhDJVKFu3a-bt10DBKoUlUd9Z2IkACxcWxT7CKWerTLmy9Wg~0vS4qMWc6J3CMRf7~13Cd8PXOPF06428WTXirdx1~4GtbawpSR8Z1MQh-U2HHq5GAnGfGRUOap3tv5cr-SDNXqHidgLvkXlLM9CksiTvOrSy7Gkv6SNssEnOi4BNbIZGRp25KksjYxV86BcTNb5zTdh~-LGP0rfqmFY-UgKKiserjhrsJBiL9C2knB~Dp8T3X9gt~IY3sZpz0W-zDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239813&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of PCC values between each gene and its strongest coexpressed gene in Arabidopsis. (A) Distribution of PCC values calculated from 1388 GeneChip slides (black), 237 slides (red) and 328 slides (green), both of which were summarized by RMA. (B) Distribution of PCC values calculated from 1388 GeneChip slides summarized by four methods (RMA, GCRMA, MAS5 and PLIER).</p></div></div><div content-id="DSP016F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01601.jpeg?Expires=2147483647&Signature=Y5eYEuUEr7rxc1Mk2nV7cngqnIGzEShDRQFh8rR3Dl6w3cgN52gybsfxVjR9aaDt9NYvKhLJS4Q5QaZU~VYvnyf-AsZDWVm6HiY~D3FKXns2hKDix9B7ojmLInXYG9IBUjTSXSBgpShbW7pOvxpqymeZJyLgCPO5mkhTJw6FwOGf49~YOLwthSobXYFLDoKy47bFBExIFU0HLpyAng9H8sCBQ2tRe3vLkt~Fw64OHVb4mcLZlQuLC9oBcoIWz07LKUSSTX6f0vshaP3Tspm1wIt-gZ5ASemC5Tw-r-B1WqlszGq5iFEEEBvqQbg7-Gbn8Y4ZkwiGNN~SXOhgPkfJiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributions of PCC values between each gene and its strongest coexpressed gene in Arabidopsis. (A) Distribution of PCC values calculated from 1388 GeneChip slides (black), 237 slides (red) and 328 slides (green), both of which were summarized by RMA. (B) Distribution of PCC values calculated from 1388 GeneChip slides summarized by four methods (RMA, GCRMA, MAS5 and PLIER)." path-from-xml="dsp01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239813/dsp01601.jpeg" path-from-xml="dsp01601.jpeg" target="_blank">View large</a><a section="6239813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01601.jpeg?Expires=2147483647&Signature=sVquAH~OFRP2SbxusR9-asdlyeUw8QH0EGQ-Ug0sw-~HWrIR6wwcFlH7hHzveRKt6qVlzo9TgSMfyTE~qwdlhDJVKFu3a-bt10DBKoUlUd9Z2IkACxcWxT7CKWerTLmy9Wg~0vS4qMWc6J3CMRf7~13Cd8PXOPF06428WTXirdx1~4GtbawpSR8Z1MQh-U2HHq5GAnGfGRUOap3tv5cr-SDNXqHidgLvkXlLM9CksiTvOrSy7Gkv6SNssEnOi4BNbIZGRp25KksjYxV86BcTNb5zTdh~-LGP0rfqmFY-UgKKiserjhrsJBiL9C2knB~Dp8T3X9gt~IY3sZpz0W-zDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239813&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of PCC values between each gene and its strongest coexpressed gene in Arabidopsis. (A) Distribution of PCC values calculated from 1388 GeneChip slides (black), 237 slides (red) and 328 slides (green), both of which were summarized by RMA. (B) Distribution of PCC values calculated from 1388 GeneChip slides summarized by four methods (RMA, GCRMA, MAS5 and PLIER).</p></div></div>                        <a id="6239814" scrollto-destination="6239814"></a>
<div id="DSP016F2" data-id="DSP016F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01602.jpeg?Expires=2147483647&Signature=Uan8ffP96WZ-JRK5C9vsDDFw8zF3fNxEEcPamUWaEvkNJ7JbFPLLkqojFDGkcZWX1QSkaI27rZazu49RGwcx9m-Glq2hFZi7no5Jx7nsfS-eIrMiP~PY-~c5cJ2BcT-AHF60oG5J4Eni8uexVDTAcvM3r8ZoCNlh0SWBdXBlMLIco0IUM9tMsDtJRnO5mND3RGNHYEU1RQfgdQNX8CbTomkYoKXrV-l5U8TU~tZVyUwdMZjEIc31OTanbDD-SOzJa97M~3dwWWg3u~xVMe4TrHM3U2QKjEFHGbsv-CoViPMYxKTh4PoJwz3qsGMHmrFQqlyIYcL57BaosrApg3i8sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationships of PCC values and GO term agreement. Decay curves of PCC values for gene coexpression are shown for Arabidopsis genes. Decay curves of PCC values of coexpressed genes from two particular genes: At3g20000 (A) and At5g06140 (B). Red dots indicate genes that have the same annotations of GO BP terms as the reference gene. Statistical significance of non-random distribution of the red dots was established by the Kolmogorov–Smirnov test. (C) The same curves in A and B are overlaid for all Arabidopsis genes. (D) Detailed view of the 0–400 range of C." path-from-xml="dsp01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239814/dsp01602.jpeg" path-from-xml="dsp01602.jpeg" target="_blank">View large</a><a section="6239814" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01602.jpeg?Expires=2147483647&Signature=wPiOj07ZZ1ImWqK5CpfFXaboS5Q6uMr0Nt2VdON6m-Ky16KpYtNNwHYyOYVhJ4ouBqKEHydZKw~OKOjVWi9jEKNPHt14FLkJdB1mZ4lnnynHMeirvM1xeL3PcurtyaZxSzAchLaV050D2deptlsVxF0g7iqxAjrmlJsuTEfE4NEP8xgQZN8NGrlUxtLpiBgu-drPrNP9MoU-GtQaSWpML1CiKAsgbxKDnwz3SYjypeniuRKmxnwesc7e7J3rUHP2akwS6GeHG7M2x52rqmgiSScSCj3utjwVnJzdSKnREUToV0CINrFyj9oRjYZIUGPDARHXAldri5TdBPnsPtF4-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239814&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationships of PCC values and GO term agreement. Decay curves of PCC values for gene coexpression are shown for Arabidopsis genes. Decay curves of PCC values of coexpressed genes from two particular genes: At3g20000 (A) and At5g06140 (B). Red dots indicate genes that have the same annotations of GO BP terms as the reference gene. Statistical significance of non-random distribution of the red dots was established by the Kolmogorov–Smirnov test. (C) The same curves in A and B are overlaid for all Arabidopsis genes. (D) Detailed view of the 0–400 range of C.</p></div></div><div content-id="DSP016F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01602.jpeg?Expires=2147483647&Signature=Uan8ffP96WZ-JRK5C9vsDDFw8zF3fNxEEcPamUWaEvkNJ7JbFPLLkqojFDGkcZWX1QSkaI27rZazu49RGwcx9m-Glq2hFZi7no5Jx7nsfS-eIrMiP~PY-~c5cJ2BcT-AHF60oG5J4Eni8uexVDTAcvM3r8ZoCNlh0SWBdXBlMLIco0IUM9tMsDtJRnO5mND3RGNHYEU1RQfgdQNX8CbTomkYoKXrV-l5U8TU~tZVyUwdMZjEIc31OTanbDD-SOzJa97M~3dwWWg3u~xVMe4TrHM3U2QKjEFHGbsv-CoViPMYxKTh4PoJwz3qsGMHmrFQqlyIYcL57BaosrApg3i8sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationships of PCC values and GO term agreement. Decay curves of PCC values for gene coexpression are shown for Arabidopsis genes. Decay curves of PCC values of coexpressed genes from two particular genes: At3g20000 (A) and At5g06140 (B). Red dots indicate genes that have the same annotations of GO BP terms as the reference gene. Statistical significance of non-random distribution of the red dots was established by the Kolmogorov–Smirnov test. (C) The same curves in A and B are overlaid for all Arabidopsis genes. (D) Detailed view of the 0–400 range of C." path-from-xml="dsp01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239814/dsp01602.jpeg" path-from-xml="dsp01602.jpeg" target="_blank">View large</a><a section="6239814" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01602.jpeg?Expires=2147483647&Signature=wPiOj07ZZ1ImWqK5CpfFXaboS5Q6uMr0Nt2VdON6m-Ky16KpYtNNwHYyOYVhJ4ouBqKEHydZKw~OKOjVWi9jEKNPHt14FLkJdB1mZ4lnnynHMeirvM1xeL3PcurtyaZxSzAchLaV050D2deptlsVxF0g7iqxAjrmlJsuTEfE4NEP8xgQZN8NGrlUxtLpiBgu-drPrNP9MoU-GtQaSWpML1CiKAsgbxKDnwz3SYjypeniuRKmxnwesc7e7J3rUHP2akwS6GeHG7M2x52rqmgiSScSCj3utjwVnJzdSKnREUToV0CINrFyj9oRjYZIUGPDARHXAldri5TdBPnsPtF4-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239814&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationships of PCC values and GO term agreement. Decay curves of PCC values for gene coexpression are shown for Arabidopsis genes. Decay curves of PCC values of coexpressed genes from two particular genes: At3g20000 (A) and At5g06140 (B). Red dots indicate genes that have the same annotations of GO BP terms as the reference gene. Statistical significance of non-random distribution of the red dots was established by the Kolmogorov–Smirnov test. (C) The same curves in A and B are overlaid for all Arabidopsis genes. (D) Detailed view of the 0–400 range of C.</p></div></div><p>The first factor is related to the expression data construction. Specifically, it depends on the choice of sample set (Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>A) and the choice of normalization method for the expression data (Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>B). Figure <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a> shows the distribution of the gene-to-gene PCCs between each gene and the most strongly coexpressed gene. In Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>A, the black line shows the distribution of the PCC values calculated from the 1388 Arabidopsis GeneChip slides downloaded from TAIR,<sup><a reveal-id="DSP016C16" data-open="DSP016C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and the red line was calculated from the 237 samples related to developmental experiments (TAIR-ME00319) and the green line was calculated from the 328 samples related to abiotic stresses (from TAIR-ME00325 to ME00330), both of which were a subset of the 1388 samples containing various experimental conditions. As seen in the figure, the red- and green-line distributions are shifted to the right compared with the black-line distribution, indicating that high coexpression around PCC = 0.8 was commonly observed in the PCCs from the developmental samples, whereas high coexpression pairs were rarely observed in the unselected samples (black-line distribution). Two possible reasons—one biological and the other mathematical—can account for the differences in PCC values. The biological reason is that gene expression changes among developmental samples were far larger than those elicited in response to environmental stresses. Such large changes in gene expression amplitude can decrease experimental noises that decrease any gene correlations. Therefore, absolute values of the PCC obtained from developmental experiments will be higher than those from experiments with unselected samples. On the other hand, the mathematical reason is that a smaller sample number tends to produce larger amplitude of correlation values between any two genes as discussed previously.<sup><a reveal-id="DSP016C12" data-open="DSP016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> This can easily be understood by considering an extreme case using only two microarray slides, for which all the PCC values between the most highly coexpressed gene pair should be 1.0 from the definition of PCC. The mathematical issue can be resolved by using statistical significances of the PCC values. In fact, some coexpression databases, such as ACT,<sup><a reveal-id="DSP016C9" data-open="DSP016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> CSB.DB<sup><a reveal-id="DSP016C7" data-open="DSP016C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and CressExpress,<sup><a reveal-id="DSP016C12" data-open="DSP016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> calculate the statistical significance of every PCC value. In the later part of this paper, we introduced rank of PCC. For a given expression matrix, the effect of rank is identical to PCC and to its <em>P</em>-value, because the order of PCC and its <em>P</em>-value of PCC were identical.</p><p>In the same way, the choice of the normalization method of microarray data affects the PCC values. Figure <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>B shows that distributions of the highest PCCs from RMA- and GCRMA-summarized data manifested as a single peak, whereas those of MAS5 and PLIER were bimodal. The peak around PCC = 0.2 observed in MAS5 and PLIER was derived from genes with low expression level (data not shown), where noises in microarray experiments may strongly affected the observed expression patterns to calculated gene coexpression. Similar to Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>A, high PCC values (e.g. PCC = 0.8) were more frequently observed in RMA- and GCRMA-summarized data, whereas those values were less frequently observed in MAS5- and PLIER-summarized data.</p><p>Gene function is another factor causing differences in biological significance of a PCC value. In Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>A and B, two examples show relationships between PCC values and gene function. The black line in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>A indicates PCC values of coexpressed genes to a gene, At3g20000 (mitochondria outer membrane protein TOM40), in descending order. Genes with the same function by GO BP annotation as the reference gene (At3g20000) are highlighted by red dots. Because genes with the same cellular function can be expected to be coexpressed, it is reasonable that the red dots accumulated at the top-left area in the graph. Figure <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>B is another example with a different reference gene, At5g06140 (PHOX domain-containing protein). Good accumulation of genes with the same function was again observed in the top-left area. However, the absolute PCC values of the red dots were different in these two graphs. Most of red dots in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>A accumulated around PCC = 0.6, whereas most red dots in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>B accumulated around PCC = 0.3. This observation suggested that the required strength of gene coexpression might be different for each reference gene. At3g20000 (Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>A) requires stronger gene coexpression than At5g06140 (Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>B) to realize its function. To check the generality of this observation, the correlation curves for all genes were overlaid in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>C and D (Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>D is a close-up view of Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>C). If the absolute value of PCC directly indicates biological significance, these red dots should accumulate in the upper area in these graphs. This was not the case, however, as they accumulated in the left-most region. Especially in the region of the top-most 20 genes, there is almost no relationship between the PCC values and functional relationship. These results suggest that the rank of PCC is a more effective measure of coexpression (to identify functionally related genes) than the PCC value itself. In the following sections, we focus on the characteristics of the rank of PCC value for each reference gene, which we call ‘PCC rank’ against normal ‘PCC value’, hereafter.</p>                            <h3 scrollto-destination=6239818 id="6239818" class="section-title" data-legacy-id=s3b>3.2 PCC rank can normalize inconsistencies caused by different sample compositions or different GeneChip summarization methods</h3>
<p>From the analyses of the relationship between PCC value and gene function, we proposed that the PCC rank might be a good measure of coexpression. Next, we assessed how PCC rank is affected by differences in expression data observed in Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>. We observed the distribution between developmental samples and all samples for PCC values (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>A, left) and PCC ranks (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>A, right), and that between RMA- and GCRMA-summarized data for PCC value (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>B, left) and PCC rank (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>B, right). The left panel of Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>A is a density plot between PCC values calculated from 1388 Arabidopsis GeneChips on the <em>x</em>-axis and PCC values calculated from 237 developmental samples on the <em>y</em>-axis. Each original dot was calculated for all gene pairs and shown as a density plot in logarithmic scale. The distribution was a broad S-shape, indicating weak correspondence of the two types of PCC values. On the other hand, when we used PCC rank instead of PCC value, the distribution aligned on the diagonal (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>A, right panel), meaning there was a linear correspondence of PCC ranks between the different samples. Notably, the genes in the highest (bottom-left) and the lowest (top-right) rank regions were highly populated between the two types of coexpression data, indicating that PCC ranks, as opposed to PCC values, were robust with respect to different sample compositions when the correlation was relatively high in each reference gene. In the same way, differences in PCC values caused by different GeneChip summarization methods shown in Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>B can be normalized by PCC rank (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>B). Although it was generally well correlated in both panels, some gene pairs showed exceptionally high values for GCRMA in PCC value (Fig. <a reveal-id="DSP016F3" data-open="DSP016F3" class="link link-reveal link-table xref-fig">3</a>B, left) as was the case in Fig. <a reveal-id="DSP016F1" data-open="DSP016F1" class="link link-reveal link-table xref-fig">1</a>B. Again, these PCC ranks showed very good agreement (Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>B, right). This result indicated that PCC rank was more stable than PCC value with respect to the selection of GeneChip summarization methods, and thus is suitable as a comparable coexpression measure. Correspondence between RMA-MAS5 and MAS5-PLIER showed similar results (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Our observation that PCC values obtained after GCRMA summarization were higher than those obtained after RMA summarization is consistent with the report for GCRMA problems by Lim <em>et al</em>.,<sup><a reveal-id="DSP016C31" data-open="DSP016C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and thus these problematic characteristics of GCRMA may be a general feature.</p>                        <a id="6239820" scrollto-destination="6239820"></a>
<div id="DSP016F3" data-id="DSP016F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01603.jpeg?Expires=2147483647&Signature=02sgae~BCTgXwbWG7Q~LCbqQV52NljA0YGzi8FwOaBDUJpen3tOTneLeYENJgeN6d3~pZQYFZYhl~K3dpctt3EFgahDf8dBqCfRyNr2H8pdUl2jlEg6gpdjiA4YSq-WzRyJVuCu1saOwezAanu1j7OEdIIHNRD2Qhs14UfS1c9ydD6Nl3So~Myu7W7ec2UROvVn1~xvBapMlPCVLjgvxB3AzvSHFe1BoQ7rzrAJjVUvz3LvU8zjIHGjfH~mNK5THm0uc0Q6OKJ0-hqhS~LF4XxfzvakaB0i0nnVH85F49HlPxRHDU6KxuiaZW~lmm73lVRaUQQtZqylKYJuOXCNz5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inconsistency of the PCC values and consistency of the PCC ranks for differently constructed expression data. Degree of coexpression is compared using PCC value (left panels) or PCC rank (right panels). (A) The x-axes indicate coexpression calculated from 1388 Arabidopsis GeneChip slides, whereas the y-axes indicate coexpression calculated from 237 developmental slides. (B) The x-axes indicate coexpression calculated from RMA-summarized data, whereas the y-axes indicate that from GCRMA-summarized data. All distributions are represented by density plots in logarithmic scale." path-from-xml="dsp01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239820/dsp01603.jpeg" path-from-xml="dsp01603.jpeg" target="_blank">View large</a><a section="6239820" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01603.jpeg?Expires=2147483647&Signature=csUEzAWz23r7EpcHU-cNWe9macPyOzdEu1hJcg2fHcJ~Ee1tPUeVcMXnLev2qLeozLBUCJNFI54J7m8h3MUnv0yktBEQa-8sKKZ-Z1GarnIROl-a9RNvB6qnOk1rTDmvOyrByUj3EhILW7S3tZNP0~2sE2oUxZdbxZmxtWkw8h0cQWvI-3xCI7C-e4QsYv9ejwDGEZFPfU~~qCSKB7JTrvzuSaOJYgRdRB9GlwAsPXSBjhYCQxTNUKudUTUQxlaTlXQBKKZj8Ig9DywWoOzJ6sDbTzrSjy1tfauDjfbhEVzGuqcws8-8wtP-UY3sr~CvGOY5WgClIC6y02~1eqWwdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239820&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inconsistency of the PCC values and consistency of the PCC ranks for differently constructed expression data. Degree of coexpression is compared using PCC value (left panels) or PCC rank (right panels). (A) The <em>x</em>-axes indicate coexpression calculated from 1388 Arabidopsis GeneChip slides, whereas the <em>y</em>-axes indicate coexpression calculated from 237 developmental slides. (B) The <em>x</em>-axes indicate coexpression calculated from RMA-summarized data, whereas the <em>y</em>-axes indicate that from GCRMA-summarized data. All distributions are represented by density plots in logarithmic scale.</p></div></div><div content-id="DSP016F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01603.jpeg?Expires=2147483647&Signature=02sgae~BCTgXwbWG7Q~LCbqQV52NljA0YGzi8FwOaBDUJpen3tOTneLeYENJgeN6d3~pZQYFZYhl~K3dpctt3EFgahDf8dBqCfRyNr2H8pdUl2jlEg6gpdjiA4YSq-WzRyJVuCu1saOwezAanu1j7OEdIIHNRD2Qhs14UfS1c9ydD6Nl3So~Myu7W7ec2UROvVn1~xvBapMlPCVLjgvxB3AzvSHFe1BoQ7rzrAJjVUvz3LvU8zjIHGjfH~mNK5THm0uc0Q6OKJ0-hqhS~LF4XxfzvakaB0i0nnVH85F49HlPxRHDU6KxuiaZW~lmm73lVRaUQQtZqylKYJuOXCNz5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inconsistency of the PCC values and consistency of the PCC ranks for differently constructed expression data. Degree of coexpression is compared using PCC value (left panels) or PCC rank (right panels). (A) The x-axes indicate coexpression calculated from 1388 Arabidopsis GeneChip slides, whereas the y-axes indicate coexpression calculated from 237 developmental slides. (B) The x-axes indicate coexpression calculated from RMA-summarized data, whereas the y-axes indicate that from GCRMA-summarized data. All distributions are represented by density plots in logarithmic scale." path-from-xml="dsp01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239820/dsp01603.jpeg" path-from-xml="dsp01603.jpeg" target="_blank">View large</a><a section="6239820" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01603.jpeg?Expires=2147483647&Signature=csUEzAWz23r7EpcHU-cNWe9macPyOzdEu1hJcg2fHcJ~Ee1tPUeVcMXnLev2qLeozLBUCJNFI54J7m8h3MUnv0yktBEQa-8sKKZ-Z1GarnIROl-a9RNvB6qnOk1rTDmvOyrByUj3EhILW7S3tZNP0~2sE2oUxZdbxZmxtWkw8h0cQWvI-3xCI7C-e4QsYv9ejwDGEZFPfU~~qCSKB7JTrvzuSaOJYgRdRB9GlwAsPXSBjhYCQxTNUKudUTUQxlaTlXQBKKZj8Ig9DywWoOzJ6sDbTzrSjy1tfauDjfbhEVzGuqcws8-8wtP-UY3sr~CvGOY5WgClIC6y02~1eqWwdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239820&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inconsistency of the PCC values and consistency of the PCC ranks for differently constructed expression data. Degree of coexpression is compared using PCC value (left panels) or PCC rank (right panels). (A) The <em>x</em>-axes indicate coexpression calculated from 1388 Arabidopsis GeneChip slides, whereas the <em>y</em>-axes indicate coexpression calculated from 237 developmental slides. (B) The <em>x</em>-axes indicate coexpression calculated from RMA-summarized data, whereas the <em>y</em>-axes indicate that from GCRMA-summarized data. All distributions are represented by density plots in logarithmic scale.</p></div></div><p>PCC values and its distribution on a given set of experiments can also be affected by the quality of microarray data, where more noise in microarray experiments causes lower PCC values. On the other hand, we can neglect the original PCC values and its distributions by taking the rank of the PCC values. This may be one of the reasons why rank-based values can compare different coexpression data. As inextricably linked aspect of this comparability, rank-based values do not include any information to estimate the quality of original microarray data. This phenomenon is quite similar with RMA summarization of GeneChip compared with MAS5, where the rank of probe intensity is used to reconstruct common distribution.</p>                            <h3 scrollto-destination=6239822 id="6239822" class="section-title" data-legacy-id=s3c>3.3 Large-scale assessment of PCC rank and MR</h3>
<p>For large-scale assessment of PCC rank, we predicted gene function using coexpressed gene lists sorted by PCC value or by PCC rank and compared the results, because many recent studies used gene coexpression data to identify gene function (see review<sup><a reveal-id="DSP016C22" data-open="DSP016C22" class="link link-ref link-reveal xref-bibr">22</a></sup>). After constructing a coexpressed gene list for each gene, we predicted GO annotations assigned to the reference gene using the following procedure (see Section 2 for details). First, the GO annotations for the gene that was most strongly coexpressed with the reference gene were applied as the first prediction. Then, the GO annotations of the second-most strongly coexpressed gene were applied, and so forth until all coexpressed genes under a given threshold of coexpression were applied. On the basis of ROC curves of this prediction, the AUCs were compared as prediction performance. AUC = 1.0 indicates a perfect prediction, whereas AUC = 0.5 means random prediction. We evaluated the performance for each GO category, i.e. BP, CC and MF.</p><p>We compared the performance of the GO annotation prediction based on PCC values and PCC ranks (Fig. <a reveal-id="DSP016F4" data-open="DSP016F4" class="link link-reveal link-table xref-fig">4</a>). In all the cases for GO BP, CC and MF predictions, PCC rank was more effective than PCC value. This result agreed well with the result in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>D, which showed that genes with same GO BP annotation accumulated in the high PCC rank region (the left-most region in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>D) rather than the high PCC value region (top region in Fig. <a reveal-id="DSP016F2" data-open="DSP016F2" class="link link-reveal link-table xref-fig">2</a>D). Note here that we introduced ‘MR’ by taking a geometric average of the PCC rank from gene A to gene B and that of gene B to gene A, because the two ranks between genes A and B are usually different and thus are not convenient to use like a PCC value. The performance of MR was almost the same as or slightly better than PCC rank (Fig. <a reveal-id="DSP016F4" data-open="DSP016F4" class="link link-reveal link-table xref-fig">4</a>). To further confirm the statistical significance of the difference between PCC value and the other two measures, we randomly selected half of the entire genes, and constructed PCC table and calculated PCC rank and MR, and then AUCs of gene function predictions were obtained as the same procedure in this study. This procedure was repeated 100 times and calculated the standard deviation of the difference of AUCs. The result showed clear difference in performances between PCC value and the rank-based measures (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Note that no difference between PCC rank and MR was observed.</p>                        <a id="6239825" scrollto-destination="6239825"></a>
<div id="DSP016F4" data-id="DSP016F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01604.jpeg?Expires=2147483647&Signature=V3SjZoSXvku3GGeo0NY7MI6wsQheEqXv9mgKKZ~o~Nha9OL5cB0~SLZsirXW~Pak5mzABWH5MJDqZ-C170Wr8aafxuqTR-qrR1IMu0PVboeOZi6ukbUAxdbJgUPAfmoJ8~bPpBOwWrLD5UcIdGpJz2jubrx~0gohbmPjvC5heNUNOK9SpGKXqXrHhe3r7hyF4Hhv0r0cJxGnItcU91SMY5j9Aqk8nUPoaCANzN0i5FOueH092Te7214gwD~mMAOztqJ4u1Q0QvHgJUWe2UiB418SG6WKmqD9R0BZwaJjlIC8eAoAlgnFcJn9LPbhPhbI3k7Aw0Rln1KK5v6p5VbjJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of rank-based coexpression measures to predict GO annotations. The y-axis indicates AUCs to predict three types of GO annotations (BP, CC and MF) from gene coexpression data represented by PCC value, PCC rank and MR. Note that the AUCs on the y-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences in AUCs rather than the absolute value of AUCs. BP, biological process; CC, cellular component; MF, molecular function." path-from-xml="dsp01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239825/dsp01604.jpeg" path-from-xml="dsp01604.jpeg" target="_blank">View large</a><a section="6239825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01604.jpeg?Expires=2147483647&Signature=QqiYFPy1vLfD57ReMcOOiTtwb~a5~Cw6Zf5BIfkaCH-btdUJB0ydiRBg8jf~9uPKqQzRQJjI393Jg0APUkxBdGMavni-9BKDss1gS-CebwCtHvZ5ZlBydqlo~wjTfftOf8N5jDnFUG-PqmXmMV-XHQjalXofdMkqcGgSQE2VfBhD8M1zQ~FcMWczDDM01qrHOjqESwiyEnbC3Yb1iDpRzEj-iqMTa6NdFtic6~ImY2aqNE4T18PrVGNvWw11oJzV4SqdcUe1mqQ-dz20CE7151Td-D65u~0gc-gS~LBL~CY2-Jk4yTujsMDM1BhBJ7vF-CfBU32ZK4cdnnB5UE13lA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239825&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effect of rank-based coexpression measures to predict GO annotations. The <em>y</em>-axis indicates AUCs to predict three types of GO annotations (BP, CC and MF) from gene coexpression data represented by PCC value, PCC rank and MR. Note that the AUCs on the <em>y</em>-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences in AUCs rather than the absolute value of AUCs. BP, biological process; CC, cellular component; MF, molecular function.</p></div></div><div content-id="DSP016F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01604.jpeg?Expires=2147483647&Signature=V3SjZoSXvku3GGeo0NY7MI6wsQheEqXv9mgKKZ~o~Nha9OL5cB0~SLZsirXW~Pak5mzABWH5MJDqZ-C170Wr8aafxuqTR-qrR1IMu0PVboeOZi6ukbUAxdbJgUPAfmoJ8~bPpBOwWrLD5UcIdGpJz2jubrx~0gohbmPjvC5heNUNOK9SpGKXqXrHhe3r7hyF4Hhv0r0cJxGnItcU91SMY5j9Aqk8nUPoaCANzN0i5FOueH092Te7214gwD~mMAOztqJ4u1Q0QvHgJUWe2UiB418SG6WKmqD9R0BZwaJjlIC8eAoAlgnFcJn9LPbhPhbI3k7Aw0Rln1KK5v6p5VbjJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of rank-based coexpression measures to predict GO annotations. The y-axis indicates AUCs to predict three types of GO annotations (BP, CC and MF) from gene coexpression data represented by PCC value, PCC rank and MR. Note that the AUCs on the y-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences in AUCs rather than the absolute value of AUCs. BP, biological process; CC, cellular component; MF, molecular function." path-from-xml="dsp01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239825/dsp01604.jpeg" path-from-xml="dsp01604.jpeg" target="_blank">View large</a><a section="6239825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01604.jpeg?Expires=2147483647&Signature=QqiYFPy1vLfD57ReMcOOiTtwb~a5~Cw6Zf5BIfkaCH-btdUJB0ydiRBg8jf~9uPKqQzRQJjI393Jg0APUkxBdGMavni-9BKDss1gS-CebwCtHvZ5ZlBydqlo~wjTfftOf8N5jDnFUG-PqmXmMV-XHQjalXofdMkqcGgSQE2VfBhD8M1zQ~FcMWczDDM01qrHOjqESwiyEnbC3Yb1iDpRzEj-iqMTa6NdFtic6~ImY2aqNE4T18PrVGNvWw11oJzV4SqdcUe1mqQ-dz20CE7151Td-D65u~0gc-gS~LBL~CY2-Jk4yTujsMDM1BhBJ7vF-CfBU32ZK4cdnnB5UE13lA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239825&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effect of rank-based coexpression measures to predict GO annotations. The <em>y</em>-axis indicates AUCs to predict three types of GO annotations (BP, CC and MF) from gene coexpression data represented by PCC value, PCC rank and MR. Note that the AUCs on the <em>y</em>-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences in AUCs rather than the absolute value of AUCs. BP, biological process; CC, cellular component; MF, molecular function.</p></div></div><p>Our observation that PCC value was less effective for predicting gene function than PCC rank is probably a consequence of the different requirements of coexpression for different biological functions. When we consider genes for a protein complex, strength of required gene coexpression may depend on the stability of monomers. Strong coexpression should be required for unstable monomers, whereas loose coexpression may be sufficient for stable monomers, the half-life of which may be regulated by phosphorylation and/or protein subcellular localization. In the same way, when we consider genes for enzymes of a particular metabolic pathway, strength of required gene coexpression may depend on the stability or toxicity of the metabolites. If the intermediate metabolite(s) between any reactions is unstable or toxic, strong coexpression of the enzyme genes will be required.</p><p>PCC is not the only measure of gene coexpression. For example, Spearman's correlation coefficient (SCC), mutual information content and partial correlation have also been used in gene coexpression studies.<sup><a reveal-id="DSP016C21" data-open="DSP016C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSP016C31" data-open="DSP016C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSP016C32" data-open="DSP016C32" class="link link-ref link-reveal xref-bibr">32</a></sup> According to our results, the performance of PCC and SCC values was almost the same for GO term prediction (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Although PCC may still have limitations with respect to outliers, as reported previously,<sup><a reveal-id="DSP016C20" data-open="DSP016C20" class="link link-ref link-reveal xref-bibr">20</a></sup> the outlier effect was smaller in large-scale microarray data. Compared with the difference between SCC and PCC values, PCC rank is very different from PCC value. It may be understood by considering that SCC value as well as PCC value is two-body relationship whereas PCC rank is multi-body relationship, i.e. PCC value can be calculated from just two genes of interest, and PCC rank reflects the distribution of PCC value around the two genes of interest. In this sense, the relationship between SCC and PCC values was more similar than that of PCC value and PCC rank. This viewpoint was also supported by our preliminary observation that SCC rank had a similar effect to PCC rank (data not shown).</p>                            <h3 scrollto-destination=6239828 id="6239828" class="section-title" data-legacy-id=s3d>3.4 Performance of PCC rank and MR in other species</h3>
<p>Although to this point we have focused solely on Arabidopsis data, we also tested the effectiveness of PCC rank and MR with respect to gene coexpression in other species to evaluate the generalities of PCC rank. We used RMA-summarized data for Arabidopsis (Fig. <a reveal-id="DSP016F4" data-open="DSP016F4" class="link link-reveal link-table xref-fig">4</a>), and here we show the results with different GeneChip summarization methods (GCRMA,<sup><a reveal-id="DSP016C25" data-open="DSP016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> MAS5<sup><a reveal-id="DSP016C18" data-open="DSP016C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and PLIER) for different model species (human, mouse and rat), which have been used in several gene coexpression databases. Because the pattern of the results was almost the same for the BP, CC and MF categories of GO (Fig. <a reveal-id="DSP016F4" data-open="DSP016F4" class="link link-reveal link-table xref-fig">4</a>), we show only the average AUC values for the other species and other normalization methods (Fig. <a reveal-id="DSP016F5" data-open="DSP016F5" class="link link-reveal link-table xref-fig">5</a>). Figure <a reveal-id="DSP016F5" data-open="DSP016F5" class="link link-reveal link-table xref-fig">5</a>A shows the results of the gene function prediction in Arabidopsis. AUC values on the <em>y</em>-axis were compared for three coexpression measures (PCC value, PCC rank and MR) and for four GeneChip summarization methods (RMA, GCRMA, MAS5 and PLIER). For all four GeneChip summarization methods, PCC rank and MR showed higher prediction efficiency than PCC values. Figure <a reveal-id="DSP016F4" data-open="DSP016F4" class="link link-reveal link-table xref-fig">4</a>B–D shows the results for human, mouse and rat, respectively. Although the gene expression data for the four species were completely different with respect to sample size and sample composition and the quality or density of GO terms is quite different in each species, PCC rank and MR were consistently more effective at predicting gene function than PCC value, strongly suggesting that the relatively higher performance of PCC rank is a general feature.</p>                        <a id="6239830" scrollto-destination="6239830"></a>
<div id="DSP016F5" data-id="DSP016F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01605.jpeg?Expires=2147483647&Signature=ynOoL~PkqiHn6nJvfKRpT0YeGZHVxdbTyJSTfhnAbV8f-Qs3zCdQYebiYLDBWCij8orqgIvq8yomWKyEGlsCpuua4TpXu9ZosE26irOyI9WS8hH6r~72MBAtQE25VzYgYvN8FYhlrH8KMC-lBMeKvJBk984Jtt4ASVOiyyMK15DmQvT41aTDGHaEOJf-4-s~kJ0nebpeJc7NmEXuTCBoa~L4j7Xf6dpBe4UV8NxgC-MzTPubLvOzdNgD0~TPb~acEbbHErcgSjEDDCA7rIFtqvJUVW8I5w31qEcougQUW19pQ5zF1OwIzpnnXo6bT~AhhzqpPtULz8ICmh0NHKNACQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of summarization methods for GO predictions. The four types of summarization methods were assessed for their ability to predict GO annotations. The y-axes indicate average AUC values of the ROC curves to predict GO BP, CC and MF. Note that the AUCs on the y-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences of AUCs rather than the absolute value of AUCs." path-from-xml="dsp01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239830/dsp01605.jpeg" path-from-xml="dsp01605.jpeg" target="_blank">View large</a><a section="6239830" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01605.jpeg?Expires=2147483647&Signature=5NagWUX3xMPlPdima9eWlX-8PGT3v9UzwvxIl4mSw54IYZGcYshcAwEM09t52i6fwe9-OWNjfR7mZLYZKFgPKdXCZvxr2a-c~pcwj2UE86DAAuDx2lGE-9RQBa4j44KBgZxQd6QDwO7jj59DmbJyFeCVEdbS5A9-lxr39O3FB63Gd9O6QYZoV017KYbQ6zhBiR435PjjqsIBHmONRV30uvHMgiCZctysfPC5rjhBSJz0ymzJahKXsoVLMjlWaj~mCUYPP2WNYAzLfqKKOJcec2BVpMsKbsWSgjO9isEc7xbMcMJpVGVXW856s6G9dpjYrn7o4r9nIVZjrDr5~W0gIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239830&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effect of summarization methods for GO predictions. The four types of summarization methods were assessed for their ability to predict GO annotations. The <em>y</em>-axes indicate average AUC values of the ROC curves to predict GO BP, CC and MF. Note that the AUCs on the <em>y</em>-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences of AUCs rather than the absolute value of AUCs.</p></div></div><div content-id="DSP016F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/m_dsp01605.jpeg?Expires=2147483647&Signature=ynOoL~PkqiHn6nJvfKRpT0YeGZHVxdbTyJSTfhnAbV8f-Qs3zCdQYebiYLDBWCij8orqgIvq8yomWKyEGlsCpuua4TpXu9ZosE26irOyI9WS8hH6r~72MBAtQE25VzYgYvN8FYhlrH8KMC-lBMeKvJBk984Jtt4ASVOiyyMK15DmQvT41aTDGHaEOJf-4-s~kJ0nebpeJc7NmEXuTCBoa~L4j7Xf6dpBe4UV8NxgC-MzTPubLvOzdNgD0~TPb~acEbbHErcgSjEDDCA7rIFtqvJUVW8I5w31qEcougQUW19pQ5zF1OwIzpnnXo6bT~AhhzqpPtULz8ICmh0NHKNACQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of summarization methods for GO predictions. The four types of summarization methods were assessed for their ability to predict GO annotations. The y-axes indicate average AUC values of the ROC curves to predict GO BP, CC and MF. Note that the AUCs on the y-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences of AUCs rather than the absolute value of AUCs." path-from-xml="dsp01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6239830/dsp01605.jpeg" path-from-xml="dsp01605.jpeg" target="_blank">View large</a><a section="6239830" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp01605.jpeg?Expires=2147483647&Signature=5NagWUX3xMPlPdima9eWlX-8PGT3v9UzwvxIl4mSw54IYZGcYshcAwEM09t52i6fwe9-OWNjfR7mZLYZKFgPKdXCZvxr2a-c~pcwj2UE86DAAuDx2lGE-9RQBa4j44KBgZxQd6QDwO7jj59DmbJyFeCVEdbS5A9-lxr39O3FB63Gd9O6QYZoV017KYbQ6zhBiR435PjjqsIBHmONRV30uvHMgiCZctysfPC5rjhBSJz0ymzJahKXsoVLMjlWaj~mCUYPP2WNYAzLfqKKOJcec2BVpMsKbsWSgjO9isEc7xbMcMJpVGVXW856s6G9dpjYrn7o4r9nIVZjrDr5~W0gIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6239830&ar=421165&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effect of summarization methods for GO predictions. The four types of summarization methods were assessed for their ability to predict GO annotations. The <em>y</em>-axes indicate average AUC values of the ROC curves to predict GO BP, CC and MF. Note that the AUCs on the <em>y</em>-axis do not start from 0.5, which corresponds to random prediction, because we focused on differences of AUCs rather than the absolute value of AUCs.</p></div></div>                            <h3 scrollto-destination=6239831 id="6239831" class="section-title" data-legacy-id=s3e>3.5 Effects of PCC rank with respect to different summarization methods of GeneChip data</h3>
<p>Figure <a reveal-id="DSP016F5" data-open="DSP016F5" class="link link-reveal link-table xref-fig">5</a> shows the performance of different GeneChip summarization methods. The selection of the GeneChip summarization method is one of the most analysed issues of GeneChip data. Although many studies have compared GeneChip summarizations, there has been no general consensus. For example, Irizarry <em>et al</em>.<sup><a reveal-id="DSP016C33" data-open="DSP016C33" class="link link-ref link-reveal xref-bibr">33</a></sup> showed superiority of GCRMA, whereas relatively low performance of GCRMA was shown by titration experiments.<sup><a reveal-id="DSP016C34" data-open="DSP016C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Srinivasasainagendra <em>et al</em>.<sup><a reveal-id="DSP016C12" data-open="DSP016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> discussed RMA superiority compared with GCRMA and MAS5 based on their unpublished result using redundant probeset pairs, which should be separated from the probeset pairs randomly selected. This disagreement is partially caused by different assessment systems. The focus of our assessment was to obtain more information from gene coexpression data. From this point of view, Lim <em>et al</em>.<sup><a reveal-id="DSP016C31" data-open="DSP016C31" class="link link-ref link-reveal xref-bibr">31</a></sup> reported MAS5 superiority and the requirement for GCRMA modification when using human GeneChip data. Although our results do not support the inferiority of GCRMA, MAS5 showed the highest performance for humans and, in that sense, supports the results by Lim <em>et al</em>.<sup><a reveal-id="DSP016C31" data-open="DSP016C31" class="link link-ref link-reveal xref-bibr">31</a></sup> As for basic properties of GeneChip summarization methods, MAS5 returns relatively high accuracy of gene expression values, but reproducibility of the value is relatively low compared with other summarization methods.<sup><a reveal-id="DSP016C33" data-open="DSP016C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Because the number of human samples was larger (5188 samples) than that for the other species (1388, 2226 and 632 samples for Arabidopsis, mouse and rat, respectively), the shortcoming of low reproducibility using MAS5 summarization may be overcome by a large number of samples. In fact, when we used 500 randomly selected human samples instead of all 5188 samples, MAS5 showed lower performance than RMA (data not shown). However, MAS5 still has merits to construct coexpression data, because it can be applied to a single GeneChip slide and because the number of available data from MAS5 in public database is larger than that from other methods which requires raw data registrations. Note that we did not use detection call of MAS5 (Present, Marginal and Absent), which provides significance of expression against background noise. Although the effectiveness of detection call of MAS5 has been reported,<sup><a reveal-id="DSP016C35" data-open="DSP016C35" class="link link-ref link-reveal xref-bibr">35</a></sup> we did not use it to avoid managements of missing values. This probably caused lower performance of MAS5.</p>                            <h3 scrollto-destination=6239833 id="6239833" class="section-title" data-legacy-id=s3f>3.6 Gene coexpression data for Arabidopsis are more powerful than those for mammalian species</h3>
<p>The AUC of GO term prediction for Arabidopsis genes was higher than for the mammalian genes (Fig. <a reveal-id="DSP016F5" data-open="DSP016F5" class="link link-reveal link-table xref-fig">5</a>). This may be because Arabidopsis has a simpler morphology and gene structure than mammals, i.e. Arabidopsis has fewer tissues and a lower degree of tissue differentiation. For example, flowering plant tissues have totipotency, and thus the developmental programme for each tissue can be maintained independently, e.g. cutting the bulb is not lethal. On the other hand, Arabidopsis has many paralogous genes to increase the variation of transcripts instead of using an alternative splicing strategy that is common in mammals.<sup><a reveal-id="DSP016C36" data-open="DSP016C36" class="link link-ref link-reveal xref-bibr">36</a></sup> This characteristic in Arabidopsis probably results in a more accurate measurement of the gene expression pattern, because gene expression microarrays cannot distinguish between splicing variants. Also, the sample variety of GeneChip data might be higher in Arabidopsis. The GeneChip data for Arabidopsis include many time-course experiments for external stimuli in addition to precise tissue samples collected by AtGenExpress,<sup><a reveal-id="DSP016C37  DSP016C39" data-open="DSP016C37  DSP016C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup> whereas this type of experiment may be more difficult to obtain for mammalian species. In fact, gene coexpression data are extensively used for gene targeting in Arabidopsis research, whereas this is not a major approach used in mammalian studies.</p>                            <h3 scrollto-destination=6239835 id="6239835" class="section-title" data-legacy-id=s3g>3.7 MR effects on hierarchical clustering</h3>
<p>To evaluate the effect of MR on multiple reference genes, we conducted hierarchical clustering using PCC or MR as the metric; (1 − PCC) was used as the PCC metric. Since the result of hierarchical clustering depends on a selected linkage rule, we applied seven methods available in ‘hclust’ function of statistical package R (<a class="link link-uri" href="http://www.r-project.org">http://www.r-project.org</a>). To assess hierarchical clustering result, we counted the number of junction whose nodes are filled with a single GO annotation, i.e. the nodes with the same function (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>A). As a result, MR metric showed generally better performance than PCC metric, and the difference between MR and PCC was remarkable for single, centroid and median linkage methods (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4B–D</a>). It may be noteworthy that MR showed very stable performance for every linkage method compared with the variable performances in PCC values. This result may suggest that MR already included some effects caused by multiple gene reference induced by some linkage rules as average, mcquitty, ward and complete, where all pairs of the cluster members are considered to calculate the distance between clusters.</p>                            <h3 scrollto-destination=6239837 id="6239837" class="section-title" data-legacy-id=s3h>3.8 Conclusion</h3>
<p>In this study, we investigated the characteristics of PCC (value and rank) as a coexpression measure. The biological significance of PCC could be altered by expression data construction and gene function. We found that PCC rank could normalize differences in biological significance, and we propose a new measure for coexpression analysis, MR, because a single index for a pair of gene is more convenient. MR is easy to calculate from PCC and can be directly compared among different coexpression data. This universality of MR enabled us to introduce a common threshold to all reference genes and to calculate average distances among multiple genes. As a consequence, hierarchical clustering and a combined coexpressed gene list for multiple reference genes can be available more effectively. Tools for these functions based on MR are available in our databases.<sup><a reveal-id="DSP016C13" data-open="DSP016C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP016C14" data-open="DSP016C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>Although selection of the GeneChip summarization method strongly affected the performance of coexpression data, MR could reduce the difference. This result applies directly to coexpression database construction, because much of the GeneChip expression data have been stored in public repositories as MAS5-summarized data. Our results provide a standard calculation procedure for condition-independent gene coexpression data to elucidate gene-to-gene functional relationships.</p><p>The generalities and easy calculation of PCC rank or MR will enable the users to directly compare results obtained from different coexpression databases, and this will strongly promote comparative transcriptomics using public databases. Searching conserved coexpression is one of the possible important applications, which is also provided in our databases.<sup><a reveal-id="DSP016C13" data-open="DSP016C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP016C14" data-open="DSP016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In this study, we did not pay attention to anti-correlation. To analyse anti-correlation using rank-based measures, some normalization such as taking percentile of PCC rank or MR may be useful to standardize the measures from zero to one.</p>                            <h3 scrollto-destination=6239841 id="6239841" class="section-title" data-legacy-id=s3i>3.9 Data availability</h3>
<p>Coexpression data represented by the PCC value and the MR based on RMA-summarized GeneChip data used in this study are available at (<a class="link link-uri" href="http://atted.jp/download.shtml">http://atted.jp/download.shtml</a>) for Arabidopsis and (<a class="link link-uri" href="http://coxpresdb.jp/download.shtml">http://coxpresdb.jp/download.shtml</a>) for human, mouse and rat.</p>                            <h2 scrollto-destination=6239843 id="6239843" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6239845 id="6239845" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by a grant-in-aid from the Institute for Scientific Research on Priority Areas from the Ministry of Education, Culture, Sports, Science and Technology of Japan to K.K. and by the Global COE Program (Center of Education and Research for Advanced Genome-Based Medicine), MEXT, Japan to T.O.</p>                            <h2 scrollto-destination=6239847 id="6239847" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP016C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Spellman</div> <div class="given-names">P.T.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.O.</div></div>,  <div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Cluster analysis and display of genome-wide expression patterns</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">14863</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns&amp;author=M.B.%20Eisen&amp;author=P.T.%20Spellman&amp;author=P.O.%20Brown&amp;author=D.%20Botstein&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=14863-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.25.14863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.25.14863"> </span></div> </div></div></div></div></div><div content-id="DSP016C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spellman</div> <div class="given-names">P.T.</div></div>,  <div class="name"><div class="surname">Sherlock</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">M.Q.</div></div>, et al. </div><div class="article-title">Comprehensive identification of cell cycle-regulated genes of the yeast <em>Saccharomyces cerevisiae</em> by microarray hybridization</div>, <div class="source">Mol. Biol. Cell</div> , <div class="year">1998</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">3273</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20identification%20of%20cell%20cycle-regulated%20genes%20of%20the%20yeast%20Saccharomyces%20cerevisiae%20by%20microarray%20hybridization&amp;author=P.T.%20Spellman&amp;author=G.%20Sherlock&amp;author=M.Q.%20Zhang&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Cell&amp;volume=9&amp;pages=3273-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.9.12.3273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.9.12.3273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.9.12.3273"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9843569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shoemaker</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Panchenko</div> <div class="given-names">A.R.</div></div>. </div><div class="article-title">Deciphering protein–protein interactions. Part I. Experimental techniques and databases</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e42</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20protein%E2%80%93protein%20interactions.%20Part%20I.%20Experimental%20techniques%20and%20databases&amp;author=B.A.%20Shoemaker&amp;author=A.R.%20Panchenko&amp;publication_year=2007&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=3&amp;pages=e42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0030042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0030042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0030042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17397251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tokimatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">KaPPA-view: a web-based analysis tool for integration of transcript and metabolite data on plant metabolic pathway maps</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">1289</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KaPPA-view%3A%20a%20web-based%20analysis%20tool%20for%20integration%20of%20transcript%20and%20metabolite%20data%20on%20plant%20metabolic%20pathway%20maps&amp;author=T.%20Tokimatsu&amp;author=N.%20Sakurai&amp;author=H.%20Suzuki&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=138&amp;pages=1289-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.060525" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.060525" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.060525"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16010003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ala</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Piro</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Grassi</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Prediction of human disease genes by human–mouse conserved coexpression analysis</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000043</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20human%20disease%20genes%20by%20human%E2%80%93mouse%20conserved%20coexpression%20analysis&amp;author=U.%20Ala&amp;author=R.M.%20Piro&amp;author=E.%20Grassi&amp;publication_year=2008&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=4&amp;pages=e1000043" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18369433" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Su</div> <div class="given-names">A.I.</div></div>,  <div class="name"><div class="surname">Cooke</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Ching</div> <div class="given-names">K.A.</div></div>, et al. </div><div class="article-title">Large-scale analysis of the human and mouse transcriptomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">4465</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20the%20human%20and%20mouse%20transcriptomes&amp;author=A.I.%20Su&amp;author=M.P.%20Cooke&amp;author=K.A.%20Ching&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=4465-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.012025199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.012025199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.012025199"> </span></div> </div></div></div></div></div><div content-id="DSP016C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steinhauser</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Luedemann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Thimm</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Kopka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">CSB.DB: a comprehensive systems-biology database</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3647</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CSB.DB%3A%20a%20comprehensive%20systems-biology%20database&amp;author=D.%20Steinhauser&amp;author=B.%20Usadel&amp;author=A.%20Luedemann&amp;author=O.%20Thimm&amp;author=J.%20Kopka&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3647-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toufighi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Brady</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Austin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ly</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Provart</div> <div class="given-names">N.J.</div></div>. </div><div class="article-title">The botany array resource: e-Northerns, expression angling, and promoter analyses</div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">153</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20botany%20array%20resource%3A%20e-Northerns%2C%20expression%20angling%2C%20and%20promoter%20analyses&amp;author=K.%20Toufighi&amp;author=S.M.%20Brady&amp;author=R.%20Austin&amp;author=E.%20Ly&amp;author=N.J.%20Provart&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=43&amp;pages=153-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.43.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.43.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.43.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15960624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manfield</div> <div class="given-names">I.W.</div></div>,  <div class="name"><div class="surname">Jen</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Pinney</div> <div class="given-names">J.W.</div></div>, et al. </div><div class="article-title">Arabidopsis Co-expression Tool (ACT): web server tools for microarray-based gene expression analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W504</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20Co-expression%20Tool%20(ACT)%3A%20web%20server%20tools%20for%20microarray-based%20gene%20expression%20analysis&amp;author=I.W.%20Manfield&amp;author=C.H.%20Jen&amp;author=J.W.%20Pinney&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W504-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Horan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bailey-Serres</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Annotating genes of known and unknown function by large-scale coexpression analysis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">41</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotating%20genes%20of%20known%20and%20unknown%20function%20by%20large-scale%20coexpression%20analysis&amp;author=K.%20Horan&amp;author=C.%20Jang&amp;author=J.%20Bailey-Serres&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=147&amp;pages=41-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.117366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.117366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.117366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18354039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mutwil</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Obro</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Willats</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Persson</div> <div class="given-names">S.</div></div>. </div><div class="article-title">GeneCAT—novel webtools that combine BLAST and co-expression analyses</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">W320</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneCAT%E2%80%94novel%20webtools%20that%20combine%20BLAST%20and%20co-expression%20analyses&amp;author=M.%20Mutwil&amp;author=J.%20Obro&amp;author=W.G.%20Willats&amp;author=S.%20Persson&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=W320-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn292" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn292" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn292"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18480120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Srinivasasainagendra</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Page</div> <div class="given-names">G.P.</div></div>,  <div class="name"><div class="surname">Mehta</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Coulibaly</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Loraine</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">CressExpress: a tool for large-scale mining of expression data from Arabidopsis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">1004</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CressExpress%3A%20a%20tool%20for%20large-scale%20mining%20of%20expression%20data%20from%20Arabidopsis&amp;author=V.%20Srinivasasainagendra&amp;author=G.P.%20Page&amp;author=T.%20Mehta&amp;author=I.%20Coulibaly&amp;author=A.E.%20Loraine&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=147&amp;pages=1004-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.115535" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.115535" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.115535"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18467456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shibaoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohta</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div>. </div><div class="article-title">COXPRESdb: a database of coexpressed gene networks in mammals</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D77</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=COXPRESdb%3A%20a%20database%20of%20coexpressed%20gene%20networks%20in%20mammals&amp;author=T.%20Obayashi&amp;author=S.%20Hayashi&amp;author=M.%20Shibaoka&amp;author=M.%20Saeki&amp;author=H.%20Ohta&amp;author=K.%20Kinoshita&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D77-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm840" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohta</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div>. </div><div class="article-title">ATTED-II provides coexpressed gene networks for Arabidopsis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D987</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ATTED-II%20provides%20coexpressed%20gene%20networks%20for%20Arabidopsis&amp;author=T.%20Obayashi&amp;author=S.%20Hayashi&amp;author=M.%20Saeki&amp;author=H.%20Ohta&amp;author=K.%20Kinoshita&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D987-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn807" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn807" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn807"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18953027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrett</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Troup</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Wilhite</div> <div class="given-names">S.E.</div></div>, et al. </div><div class="article-title">NCBI GEO: mining tens of millions of expression profiles—database and tools update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D760</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20GEO%3A%20mining%20tens%20of%20millions%20of%20expression%20profiles%E2%80%94database%20and%20tools%20update&amp;author=T.%20Barrett&amp;author=D.B.%20Troup&amp;author=S.E.%20Wilhite&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D760-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl887" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl887" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl887"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17099226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swarbreck</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wilks</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The Arabidopsis information resource (TAIR): gene structure and function annotation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D1009</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20information%20resource%20(TAIR)%3A%20gene%20structure%20and%20function%20annotation&amp;author=D.%20Swarbreck&amp;author=C.%20Wilks&amp;author=P.%20Lamesch&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D1009-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Craigon</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">James</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Okyere</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jotham</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">S.</div></div>. </div><div class="article-title">NASCArrays: a repository for microarray data generated by NASC's transcriptomics service</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D575</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NASCArrays%3A%20a%20repository%20for%20microarray%20data%20generated%20by%20NASC's%20transcriptomics%20service&amp;author=D.J.%20Craigon&amp;author=N.%20James&amp;author=J.%20Okyere&amp;author=J.%20Higgins&amp;author=J.%20Jotham&amp;author=S.%20May&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D575-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hubbell</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Mei</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Robust estimators for expression analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1585</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robust%20estimators%20for%20expression%20analysis&amp;author=E.%20Hubbell&amp;author=W.M.%20Liu&amp;author=R.%20Mei&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=1585-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.12.1585" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.12.1585" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.12.1585"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12490442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yona</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Dirks</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Rahman</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Effective similarity measures for expression profiles</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1616</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effective%20similarity%20measures%20for%20expression%20profiles&amp;author=G.%20Yona&amp;author=W.%20Dirks&amp;author=S.%20Rahman&amp;author=D.M.%20Lin&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1616-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16595558" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hardin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mitani</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hicks</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">VanKoten</div> <div class="given-names">B.</div></div>. </div><div class="article-title">A robust measure of correlation between two genes on a microarray</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">220</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%20measure%20of%20correlation%20between%20two%20genes%20on%20a%20microarray&amp;author=J.%20Hardin&amp;author=A.%20Mitani&amp;author=L.%20Hicks&amp;author=B.%20VanKoten&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17592643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de la Fuente</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bing</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hoeschele</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Mendes</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Discovery of meaningful associations in genomic data using partial correlation coefficients</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3565</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20meaningful%20associations%20in%20genomic%20data%20using%20partial%20correlation%20coefficients&amp;author=A.%20de%20la%20Fuente&amp;author=N.%20Bing&amp;author=I.%20Hoeschele&amp;author=P.%20Mendes&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3565-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth445" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth445" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth445"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15284096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aoki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Approaches for extracting practical information from gene co-expression networks in plant biology</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">381</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Approaches%20for%20extracting%20practical%20information%20from%20gene%20co-expression%20networks%20in%20plant%20biology&amp;author=K.%20Aoki&amp;author=Y.%20Ogata&amp;author=D.%20Shibata&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=381-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17251202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stuart</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Segal</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Koller</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title">A gene-coexpression network for global discovery of conserved genetic modules</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">249</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene-coexpression%20network%20for%20global%20discovery%20of%20conserved%20genetic%20modules&amp;author=J.M.%20Stuart&amp;author=E.%20Segal&amp;author=D.%20Koller&amp;author=S.K.%20Kim&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=249-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1087447" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1087447" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1087447"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12934013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Featherstone</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Broadie</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Wrestling with pleiotropy: genomic and topological analysis of the yeast gene expression network</div>, <div class="source">Bioessays</div> , <div class="year">2002</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">267</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wrestling%20with%20pleiotropy%3A%20genomic%20and%20topological%20analysis%20of%20the%20yeast%20gene%20expression%20network&amp;author=D.E.%20Featherstone&amp;author=K.%20Broadie&amp;publication_year=2002&amp;journal=Bioessays&amp;volume=24&amp;pages=267-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11891763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Hobbs</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Collin</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Exploration, normalization, and summaries of high density oligonucleotide array probe level data</div>, <div class="source">Biostatistics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">249</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploration%2C%20normalization%2C%20and%20summaries%20of%20high%20density%20oligonucleotide%20array%20probe%20level%20data&amp;author=R.A.%20Irizarry&amp;author=B.%20Hobbs&amp;author=F.%20Collin&amp;publication_year=2003&amp;journal=Biostatistics&amp;volume=4&amp;pages=249-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2F4.2.249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2F4.2.249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12925520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Gentleman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Murillo</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Spencer</div> <div class="given-names">F.</div></div>. </div><div class="article-title">A model based background adjustment for oligonucleotide expression arrays</div>, <div class="source">J. Am. Stat. Assoc.</div> , <div class="year">2004</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">909</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20model%20based%20background%20adjustment%20for%20oligonucleotide%20expression%20arrays&amp;author=Z.%20Wu&amp;author=R.A.%20Irizarry&amp;author=R.%20Gentleman&amp;author=F.M.%20Murillo&amp;author=F.%20Spencer&amp;publication_year=2004&amp;journal=J.%20Am.%20Stat.%20Assoc.&amp;volume=99&amp;pages=909-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1198/016214504000000683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1198%2F016214504000000683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1198%2F016214504000000683"> </span></div> </div></div></div></div></div><div content-id="DSP016C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gentleman</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Carey</div> <div class="given-names">V.J.</div></div>,  <div class="name"><div class="surname">Bates</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">Bioconductor: open software development for computational biology and bioinformatics</div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R80</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioconductor%3A%20open%20software%20development%20for%20computational%20biology%20and%20bioinformatics&amp;author=R.C.%20Gentleman&amp;author=V.J.%20Carey&amp;author=D.M.%20Bates&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-10-r80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-10-r80"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15461798" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seo</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bakay</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y-.W.</div></div>,  <div class="name"><div class="surname">Hilmer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shneiderman</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hoffman</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Interactively optimizing signal-to-noise ratios in expression profiling: project-specific algorithm selection and detection <em>P</em>-value weighting in Affymetrix microarrays</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2534</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactively%20optimizing%20signal-to-noise%20ratios%20in%20expression%20profiling%3A%20project-specific%20algorithm%20selection%20and%20detection%20P-value%20weighting%20in%20Affymetrix%20microarrays&amp;author=J.%20Seo&amp;author=M.%20Bakay&amp;author=Y-.W.%20Chen&amp;author=S.%20Hilmer&amp;author=B.%20Shneiderman&amp;author=E.%20Hoffman&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2534-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15117752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Santrosyan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Winzeler</div> <div class="given-names">E.</div></div>. </div><div class="article-title"><em>In silico</em> gene function prediction using ontology-based pattern identification</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1237</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20gene%20function%20prediction%20using%20ontology-based%20pattern%20identification&amp;author=Y.%20Zhou&amp;author=J.%20Young&amp;author=A.%20Santrosyan&amp;author=K.%20Chen&amp;author=F.%20Yan&amp;author=E.%20Winzeler&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1237-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15531612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lord</div> <div class="given-names">P.W.</div></div>,  <div class="name"><div class="surname">Stevens</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Brass</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Goble</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Investigating semantic similarity measures across the Gene Ontology: the relationship between sequence and annotation</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1275</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigating%20semantic%20similarity%20measures%20across%20the%20Gene%20Ontology%3A%20the%20relationship%20between%20sequence%20and%20annotation&amp;author=P.W.%20Lord&amp;author=R.D.%20Stevens&amp;author=A.%20Brass&amp;author=C.A.%20Goble&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1275-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12835272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lim</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lefebvre</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Califano</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Comparative analysis of microarray normalization procedures: effects on reverse engineering gene networks</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">i282</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20microarray%20normalization%20procedures%3A%20effects%20on%20reverse%20engineering%20gene%20networks&amp;author=W.K.%20Lim&amp;author=K.%20Wang&amp;author=C.%20Lefebvre&amp;author=A.%20Califano&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=i282-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lisso</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Steinhauser</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Altmann</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kopka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mussig</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Identification of brassinosteroid-related genes by means of transcript co-response analyses</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">2685</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20brassinosteroid-related%20genes%20by%20means%20of%20transcript%20co-response%20analyses&amp;author=J.%20Lisso&amp;author=D.%20Steinhauser&amp;author=T.%20Altmann&amp;author=J.%20Kopka&amp;author=C.%20Mussig&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=2685-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki566"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15891113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Jaffee</div> <div class="given-names">H.A.</div></div>. </div><div class="article-title">Comparison of Affymetrix GeneChip expression measures</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">789</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20Affymetrix%20GeneChip%20expression%20measures&amp;author=R.A.%20Irizarry&amp;author=Z.%20Wu&amp;author=H.A.%20Jaffee&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=789-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btk046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtk046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtk046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16410320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shippy</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fulmer-Smentek</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">R.V.</div></div>, et al. </div><div class="article-title">Using RNA sample titrations to assess microarray platform performance and normalization techniques</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1123</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20RNA%20sample%20titrations%20to%20assess%20microarray%20platform%20performance%20and%20normalization%20techniques&amp;author=R.%20Shippy&amp;author=S.%20Fulmer-Smentek&amp;author=R.V.%20Jensen&amp;publication_year=2006&amp;journal=Nat.%20Biotechnol.&amp;volume=24&amp;pages=1123-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16964226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pepper</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Saunders</div> <div class="given-names">E.K.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">L.E.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">The utility of MAS5 expression summary and detection call algorithms</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">273</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20utility%20of%20MAS5%20expression%20summary%20and%20detection%20call%20algorithms&amp;author=S.D.%20Pepper&amp;author=E.K.%20Saunders&amp;author=L.E.%20Edwards&amp;author=C.L.%20Wilson&amp;author=C.J.%20Miller&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=273" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-273"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17663764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="collab">Arabidopsis Genome Initiative</div><div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmid</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davison</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Henz</div> <div class="given-names">S.R.</div></div>, et al. </div><div class="article-title">A gene expression map of <em>Arabidopsis thaliana</em> development</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">501</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene%20expression%20map%20of%20Arabidopsis%20thaliana%20development&amp;author=M.%20Schmid&amp;author=T.S.%20Davison&amp;author=S.R.%20Henz&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=501-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kilian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Whitehead</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Horak</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The AtGenExpress global stress expression data set: protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses</div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">347</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20AtGenExpress%20global%20stress%20expression%20data%20set%3A%20protocols%2C%20evaluation%20and%20model%20data%20analysis%20of%20UV-B%20light%2C%20drought%20and%20cold%20stress%20responses&amp;author=J.%20Kilian&amp;author=D.%20Whitehead&amp;author=J.%20Horak&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=50&amp;pages=347-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03052.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03052.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03052.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17376166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP016C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP016C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Akiyama</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">The AtGenExpress hormone and chemical treatment data set: experimental design, data evaluation, model data analysis and data access</div>, <div class="source">Plant J.</div> , <div class="year">2008</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">526</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20AtGenExpress%20hormone%20and%20chemical%20treatment%20data%20set%3A%20experimental%20design%2C%20data%20evaluation%2C%20model%20data%20analysis%20and%20data%20access&amp;author=H.%20Goda&amp;author=E.%20Sasaki&amp;author=K.%20Akiyama&amp;publication_year=2008&amp;journal=Plant%20J.&amp;volume=55&amp;pages=526-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2008.55.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2008.55.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2008.55.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18419781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Kazuki Saito</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=421165&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp016-dsp016_Tables.xls?Expires=2147483647&amp;Signature=qR9ncbNxKAOrkurNZUgoE5SlIKFl5xuOczn8c17VJeSeOgEjEVD-D21kCUEjTTvopuBBdzhNipWGS8rcDe6HKgq9pc~biTfZApQH4CH4wWE53mrG8dxkuRyqMGSWEj-Tk2uXzehAI5n131OJg4y0Bk2G0u3sPZWbGjZRdJZixKyHTDypaTmUcHOubxfcomoFyyFx0ePFaiwHJqBBlzSfCmiA8CAcT036nr32VZHcLzVXa0xmIyfxWSavczAzVBdNeemihztpKdXCZL-jcZQ46FvaMF6mdr-oeUFUt6zY5wNasP5B1vePPyTo8gIkYxdkbka899RI9I3YC2fwIQW7hQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp016/2/dsp016-dsp016supp.pdf?Expires=2147483647&amp;Signature=ZBt11zaJCiT~8VMFmk53aXBKN84o~f0-Iw4X4QSxoD34YEADQnqaYQLQ3HkAjRGZ6paUvio1OU7r76t0YgNNXNoKqZHmD-zh98XXS9FIwXRkKxIYYZCHnWyLdxZSolkw3eEaohRuBUU43FcP-~62DDTqAC8JpBpl4LR1oOEgSgAGl2t1pcy~8a4H6EAgzNInlDBw9eFi5D~xnw4k8pjAArOf1DyeJOpcBxWfhn4uwA59q6TZpP1FNgU7W3IF2OZLxGyrMtSaPJFrNqX~F2nuDmy8-X1Rewuk3svUZ~grmJS-fhDy5oBJrdqXnykIBbERI0sTKaIoqpLGCLviTgRMYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">538</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">108</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="421165"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f5%2f249%2f421165">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000271610400001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (108)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F5%2F249%2F421165" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/421165'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Rank of Correlation Coefficient as a Comparable Measure for Biological Significance of Gene Coexpression" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f249%2f421165">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

