

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/2/109/333081","@type":"ScholarlyArticle","name":"A Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions","datePublished":"2012-12-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2013-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss037","keywords":["deep sequencing","barley","phosphorus","non-coding small RNAs","miRNAs","differential expression"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/2/109/333081","author":[{"name":"Hackenberg, Michael","affiliation":"1  Computational Genomics and Bioinformatics Group, Genetics Department ,  University of Granada ,  Granada 18071 ,  Spain","@type":"Person"},{"name":"Huang, Po-Jung","affiliation":"2  Bioinformatics Center ,  Chang Gung University ,  Taoyuan, Taiwan ,  China","@type":"Person"},{"name":"Huang, Chun-Yuan","affiliation":"3  Australian Centre for Plant Functional Genomics ,  The University of Adelaide ,  Adelaide, South Australia 5064 ,  Australia","@type":"Person"},{"name":"Shi, Bu-Jun","affiliation":"3  Australian Centre for Plant Functional Genomics ,  The University of Adelaide ,  Adelaide, South Australia 5064 ,  Australia","@type":"Person"},{"name":"Gustafson, Perry","affiliation":"4  USDA-ARS ,  University of Missouri ,  206 Curtis Hall, Columbia, MO 65211-7020 ,  USA","@type":"Person"},{"name":"Langridge, Peter","affiliation":"3  Australian Centre for Plant Functional Genomics ,  The University of Adelaide ,  Adelaide, South Australia 5064 ,  Australia","@type":"Person"}],"description":"Abstract.  Phosphorus (P) is essential for plant growth. MicroRNAs (miRNAs) play a key role in phosphate homeostasis. However, little is known about P effect on","pageStart":"109","pageEnd":"125","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions" />
    <meta property="og:description" content="Abstract.  Phosphorus (P) is essential for plant growth. MicroRNAs (miRNAs) play a key role in phosphate homeostasis. However, little is known about P effect on" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/2/109/333081" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-12-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hackenberg, Michael" /><meta name="citation_author_institution" content="Computational Genomics and Bioinformatics Group, Genetics Department, University of Granada, Granada 18071, Spain" /><meta name="citation_author" content="Huang, Po-Jung" /><meta name="citation_author_institution" content="Bioinformatics Center, Chang Gung University, Taoyuan, Taiwan, China" /><meta name="citation_author" content="Huang, Chun-Yuan" /><meta name="citation_author_institution" content="Australian Centre for Plant Functional Genomics, The University of Adelaide, Adelaide, South Australia 5064, Australia" /><meta name="citation_author" content="Shi, Bu-Jun" /><meta name="citation_author_institution" content="Australian Centre for Plant Functional Genomics, The University of Adelaide, Adelaide, South Australia 5064, Australia" /><meta name="citation_author" content="Gustafson, Perry" /><meta name="citation_author_institution" content="USDA-ARS, University of Missouri, 206 Curtis Hall, Columbia, MO 65211-7020, USA" /><meta name="citation_author" content="Langridge, Peter" /><meta name="citation_author_institution" content="Australian Centre for Plant Functional Genomics, The University of Adelaide, Adelaide, South Australia 5064, Australia" /><meta name="citation_title" content="A Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions" /><meta name="citation_firstpage" content="109" /><meta name="citation_lastpage" content="125" /><meta name="citation_doi" content="10.1093/dnares/dss037" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2013/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=MicroRNAs and their regulatory roles in plants; Ann. Rev. Plant Biol.;  citation_year=2006;  citation_volume=57;  citation_pages=19-53; " /><meta name="citation_reference" content="citation_title=A novel class of bacteria-induced small RNAs in Arabidopsis; Genes Dev.;  citation_year=2007;  citation_volume=21;  citation_pages=3123-34; " /><meta name="citation_reference" content="citation_title=Post-transcriptional small RNA pathways in plants: mechanisms and regulations; Genes Dev.;  citation_year=2006;  citation_volume=20;  citation_pages=759-71; " /><meta name="citation_reference" content="citation_title=A novel class of small RNAs: tRNA-derived RNA fragments (tRFs); Genes Dev.;  citation_year=2009;  citation_volume=23;  citation_pages=2639-49; " /><meta name="citation_reference" content="citation_title=Transfer RNA-derived fragments: origins, processing, and functions; RNA;  citation_year=2011;  citation_volume=2;  citation_pages=853-62; " /><meta name="citation_reference" content="citation_title=Phosphorus nutrition of spring wheat (Triticum aestivum L.). 1. Effects of phosphorus supply on plant symptoms, yield, components of yield, and plant phosphorus uptake; Aust. J. Agric. Res.;  citation_year=1997;  citation_volume=455;  citation_pages=855-67; " /><meta name="citation_reference" content="citation_title=Barley genotypes with long root hairs sustain high grain yields in low-P field; Plant Soil;  citation_year=2004;  citation_volume=262;  citation_pages=55-62; " /><meta name="citation_reference" content="citation_title=Phosphate utilization efficiency correlates with expression of low-affinity phosphate transporters and noncoding RNA, IPS1, in barley; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=1217-9; " /><meta name="citation_reference" content="citation_title=The role of microRNAs in phosphorus deficiency signalling; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=1016-24; " /><meta name="citation_reference" content="citation_title=MicroRNA expression profiles associated with phosphorus deficiency in white lupin (Lupinus albus L.); Plant Sci.;  citation_year=2010;  citation_volume=178;  citation_pages=23-9; " /><meta name="citation_reference" content="citation_title=Analysis of phosphorus deficient responsive miRNAs and cis-elements from soybean (Glycine max L.); J. Plant Physiol.;  citation_year=2010;  citation_volume=167;  citation_pages=1289-97; " /><meta name="citation_reference" content="citation_title=Uncovering small RNA-mediated responses to phosphate deficiency in Arabidopsis by deep sequencing; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=2120-32; " /><meta name="citation_reference" content="citation_title=Regulatory network of microRNA399 and PHO2 by systemic signalling; Plant Physiol.;  citation_year=2008;  citation_volume=147;  citation_pages=732-46; " /><meta name="citation_reference" content="citation_title=MicroRNA399 is a longdistance signal for the regulation of plant phosphate homeostasis; Plant J.;  citation_year=2008;  citation_volume=53;  citation_pages=731-8; " /><meta name="citation_reference" content="citation_title=Effects of the Pallas gene in barley: phene analysis, overdominance, variability; Hereditas;  citation_year=1977;  citation_volume=86;  citation_pages=251-66; " /><meta name="citation_reference" content="citation_title=A study of the role of root morphological traits in growth of barley in zinc-deficient soil; J. Exp. Bot.;  citation_year=2007;  citation_volume=58;  citation_pages=2775-84; " /><meta name="citation_reference" content="citation_publisher=CSIRO Publishing, Melbourne, Australia; Plant Analysis: An Interpretation Manual;  citation_year=1997;  citation_pages=86-92; " /><meta name="citation_reference" content="citation_title=Impact of soil tillage on the robustness of the genetic component of variation in phosphorus (P) use efficiency in barley (Hordeum vulgare L.); Plant Soil;  citation_year=2011;  citation_volume=339;  citation_pages=113-23; " /><meta name="citation_reference" content="citation_title=miRanalyzer: an update on the detection and analysis of microRNAs in high-throughput sequencing experiments; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=W132-138; " /><meta name="citation_reference" content="citation_title=The TIGR Plant Repeat Databases: a collective resource for the identification of repetitive sequences in plants; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D360-3; " /><meta name="citation_reference" content="citation_title=TREP: a database of Triticeae repetitive elements; Trends Plant Sci.;  citation_year=2002;  citation_volume=7;  citation_pages=561-2; " /><meta name="citation_reference" content="citation_title=Repbase update, a database of eukaryotic repetitive elements; Cytogenet. Genome Res.;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=Discovering microRNAs from deep sequencing data using miRDeep; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=407-15; " /><meta name="citation_reference" content="citation_title=Mfold web server for nucleic acid folding and hybridization prediction; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3406-15; " /><meta name="citation_reference" content="citation_title=psRNATarget: a plant small RNA target analysis server; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=W155-159; " /><meta name="citation_reference" content="citation_title=Endogenous siRNA and miRNA targets identified by sequencing of the Arabidopsis degradome; Curr. Biol.;  citation_year=2008;  citation_volume=18;  citation_pages=758-62; " /><meta name="citation_reference" content="citation_title=Construction of a barley bacterial artificial chromosome library suitable for cloning genes for boron tolerance, sodium exclusion and high grain zinc content; Plant Breeding;  citation_year=2010;  citation_volume=129;  citation_pages=291-6; " /><meta name="citation_reference" content="citation_title=Discovery of barley miRNAs through deep sequencing of short reads; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=129" /><meta name="citation_reference" content="citation_title=Identification and characterization of microRNAs from barley (Hordeum vulgare L.) by high-throughput sequencing; Int. J. Mol. Sci.;  citation_year=2012;  citation_volume=13;  citation_pages=2973-84; " /><meta name="citation_reference" content="citation_title=NERD, a plant-specific GW protein, defines an additional RNAi-dependent chromatin-based pathway in Arabidopsis; Mol. Cell;  citation_year=2012;  citation_volume=48;  citation_pages=121-132; " /><meta name="citation_reference" content="citation_title=RDR2 partially antagonizes the production of RDR6-dependent siRNA in sense transgene-mediated PTGS; PLoS One;  citation_year=2012;  citation_pages=e29785" /><meta name="citation_reference" content="citation_title=A novel class of heat-responsive small RNAs derived from the chloroplast genome of Chinese cabbage (Brassica rapa); BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=289" /><meta name="citation_reference" content="citation_title=Short non-coding RNA fragments accumulating in chloroplasts: footprints of RNA binding proteins?; Nucleic Acid Res.;  citation_year=2011;  citation_volume=40;  citation_pages=3106-3116; " /><meta name="citation_reference" content="citation_title=miR156-regulated SPL transcription factors define an endogenous flowering pathway in Arabidopsis thaliana; Cell;  citation_year=2009;  citation_volume=138;  citation_pages=738-49; " /><meta name="citation_reference" content="citation_title=A miRNA involved in phosphate-starvation response in Arabidopsis; Curr. Biol.;  citation_year=2005;  citation_volume=15;  citation_pages=2038-43; " /><meta name="citation_reference" content="citation_title=Identification of nutrient-responsive Arabidopsis and rapeseed microRNAs by comprehensive real-time polymerase chain reaction profiling and small RNA sequencing; Plant Physiol.;  citation_year=2009;  citation_volume=150;  citation_pages=1541-55; " /><meta name="citation_reference" content="citation_title=A family of microRNAs present in plants and animals; Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=3355-69; " /><meta name="citation_reference" content="citation_title=Evidence that miRNAs are different from other RNAs Cell; Mol. Life Sci.;  citation_year=2006a;  citation_volume=63;  citation_pages=246-54; " /><meta name="citation_reference" content="citation_title=Conservation and divergence of plant microRNA genes; Plant J.;  citation_year=2006b;  citation_volume=46;  citation_pages=243-59; " /><meta name="citation_reference" content="citation_title=Deep sequencing of Brachypodium small RNAs at the global genome level identifies microRNAs involved in cold stress response; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=449" /><meta name="citation_reference" content="citation_title=Identification of miniature inverted-repeat transposable elements (MITEs) and biogenesis of their siRNAs in the Solanaceae: new functional implications for MITEs; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=42-56; " /><meta name="citation_reference" content="citation_title=Epigenetic reprogramming and small RNA silencing of transposable elements in pollen; Cell;  citation_year=2009;  citation_volume=136;  citation_pages=461-72; " /><meta name="citation_reference" content="citation_title=Antisense transcription in the mammalian transcriptome; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1564-6; " /><meta name="citation_reference" content="citation_title=Endogenous siRNAs derived from transposons and mRNAs in Drosophila somatic cells; Science;  citation_year=2008;  citation_volume=320;  citation_pages=1077-81; " /><meta name="citation_reference" content="citation_title=Evidence for transcriptional regulation of plastid photosynthesis genes in Arabidopsis thaliana roots; Plant Physiol.;  citation_year=1997;  citation_volume=114;  citation_pages=623-30; " /><meta name="citation_reference" content="citation_title=Plant response to stress: biochemical adaptations to phosphate deficiency; citation_publisher=Marcel Dekker, New York, USA; Encyclopedia of Plant and Crop Science;  citation_year=2004;  citation_pages=976-80; " /><meta name="citation_reference" content="citation_title=Control of translation and mRNA degradation by miRNAs and siRNAs; Genes Dev.;  citation_year=2006;  citation_volume=20;  citation_pages=515-24; " /><meta name="citation_reference" content="citation_title=Trypanosoma cruzi produces a population of tRNA-derived small RNAs which are recruited to specific cytoplasmic granules and secreted to the extracellular medium; BMC Proc.;  citation_year=2011;  citation_volume=5;  citation_issue=Suppl. 1;  citation_pages=O7" /><meta name="citation_reference" content="citation_title=Decay of mRNAs targeted by RISC requires XRN1, the Ski complex, and the exosome; RNA;  citation_year=2005;  citation_volume=11;  citation_pages=459-69; " /><meta name="citation_reference" content="citation_title=Filtering of deep sequencing data reveals the existence of abundant Dicer-dependent small RNAs derived from tRNAs; RNA;  citation_year=2009;  citation_volume=15;  citation_pages=2147-60; " /><meta name="citation_reference" content="citation_title=AtXRN4 degrades mRNA in Arabidopsis and its substrates include selected miRNA targets; Mol. Cell;  citation_year=2004;  citation_volume=15;  citation_pages=173-83; " /><meta name="citation_reference" content="citation_title=Human tRNA-derived small RNAs in the global regulation of RNA silencing; RNA;  citation_year=2010;  citation_volume=16;  citation_pages=673-95; " /><meta name="citation_reference" content="citation_title=Extensive terminal and asymmetric processing of small RNAs from rRNAs, snoRNAs, snRNAs, and tRNAs; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=6787-99; " /><meta name="citation_reference" content="citation_title=Regulatory RNAs derived from transfer RNA; RNA;  citation_year=2010;  citation_volume=16;  citation_pages=1865-9; " /><meta name="citation_reference" content="citation_title=The RNase Rny1p cleaves tRNAs and promotes cell death during oxidative stress in Saccharomyces cerevisiae; J. Cell Biol.;  citation_year=2009;  citation_volume=185;  citation_pages=43-50; " /><meta name="citation_reference" content="citation_title=Angiogenin cleaves tRNA and promotes stress-induced translational repression; J. Cell Biol.;  citation_year=2009;  citation_volume=185;  citation_pages=35-42; " /><meta name="citation_reference" content="citation_title=Identification of small non-coding RNAs from mitochondria and chloroplasts; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=3842-52; " /><meta name="citation_reference" content="citation_title=Diversity of endogenous small non-coding RNAs in Oryza sativa; Genetica;  citation_year=2006;  citation_volume=128;  citation_pages=21-31; " /><meta name="citation_reference" content="citation_title=Elucidation of the small RNA component of the transcriptome; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1567-9; " /><meta name="citation_reference" content="citation_title=A diverse and evolutionarily fluid set of microRNAs in Arabidopsis thaliana; Genes Dev.;  citation_year=2006;  citation_volume=20;  citation_pages=3407-25; " /><meta name="citation_reference" content="citation_title=Site-specific binding of a PPR protein defines and stabilizes 50 and 30 mRNA termini in chloroplasts; EMBO J.;  citation_year=2009;  citation_volume=28;  citation_pages=2042-52; " /><meta name="citation_reference" content="citation_title=The cutting crew – ribonucleases are key players in the control of plastid gene expression; J. Exp. Bot.;  citation_year=2012;  citation_volume=63;  citation_pages=1663-73; " /><meta name="citation_reference" content="citation_title=Ribonuclease P processes polycistronic tRNA transcripts in Escherichia coli independent of ribonuclease E; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=7614-25; " /><meta name="citation_reference" content="citation_author=Role of phosphorus in photosynthetic carbon metabolism; citation_publisher=Taylor &amp; Francis Group, LLC, Tucson; Handbook of Photosynthesis;  citation_year=2005;  citation_pages=123-48; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/2/109/952454/dss037.pdf" /><meta name="description" content="Abstract.  Phosphorus (P) is essential for plant growth. MicroRNAs (miRNAs) play a key role in phosphate homeostasis. However, little is known about P effect on" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/2/109/333081" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/2/109/333081" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/2/109/333081");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss037';
    </script>
<title>
	Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./333081" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYaGATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFja2VuYmVyZywgTWljaGFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvbXB1dGF0aW9uYWwgR2Vub21pY3MgYW5kIEJpb2luZm9ybWF0aWNzIEdyb3VwLCBHZW5ldGljcyBEZXBhcnRtZW50LCBVbml2ZXJzaXR5IG9mIEdyYW5hZGEsIEdyYW5hZGEgMTgwNzEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVhbmcsIFBvLUp1bmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBDZW50ZXIsIENoYW5nIEd1bmcgVW5pdmVyc2l0eSwgVGFveXVhbiwgVGFpd2FuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1YW5nLCBDaHVuLVl1YW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBdXN0cmFsaWFuIENlbnRyZSBmb3IgUGxhbnQgRnVuY3Rpb25hbCBHZW5vbWljcywgVGhlIFVuaXZlcnNpdHkgb2YgQWRlbGFpZGUsIEFkZWxhaWRlLCBTb3V0aCBBdXN0cmFsaWEgNTA2NCwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpLCBCdS1KdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBdXN0cmFsaWFuIENlbnRyZSBmb3IgUGxhbnQgRnVuY3Rpb25hbCBHZW5vbWljcywgVGhlIFVuaXZlcnNpdHkgb2YgQWRlbGFpZGUsIEFkZWxhaWRlLCBTb3V0aCBBdXN0cmFsaWEgNTA2NCwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VzdGFmc29uLCBQZXJyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBVbml2ZXJzaXR5IG9mIE1pc3NvdXJpLCAyMDYgQ3VydGlzIEhhbGwsIENvbHVtYmlhLCBNTyA2NTIxMS03MDIwLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMYW5ncmlkZ2UsIFBldGVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQXVzdHJhbGlhbiBDZW50cmUgZm9yIFBsYW50IEZ1bmN0aW9uYWwgR2Vub21pY3MsIFRoZSBVbml2ZXJzaXR5IG9mIEFkZWxhaWRlLCBBZGVsYWlkZSwgU291dGggQXVzdHJhbGlhIDUwNjQsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSBDb21wcmVoZW5zaXZlIEV4cHJlc3Npb24gUHJvZmlsZSBvZiBNaWNyb1JOQXMgYW5kIE90aGVyIENsYXNzZXMgb2YgTm9uLUNvZGluZyBTbWFsbCBSTkFzIGluIEJhcmxleSBVbmRlciBQaG9zcGhvcm91cy1EZWZpY2llbnQgYW5kIC1TdWZmaWNpZW50IENvbmRpdGlvbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkFzIGFuZCB0aGVpciByZWd1bGF0b3J5IHJvbGVzIGluIHBsYW50czsgQW5uLiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MTktNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgY2xhc3Mgb2YgYmFjdGVyaWEtaW5kdWNlZCBzbWFsbCBSTkFzIGluIEFyYWJpZG9wc2lzOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzEyMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9zdC10cmFuc2NyaXB0aW9uYWwgc21hbGwgUk5BIHBhdGh3YXlzIGluIHBsYW50czogbWVjaGFuaXNtcyBhbmQgcmVndWxhdGlvbnM7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz03NTktNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgY2xhc3Mgb2Ygc21hbGwgUk5BczogdFJOQS1kZXJpdmVkIFJOQSBmcmFnbWVudHMgKHRSRnMpOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjYzOS00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXIgUk5BLWRlcml2ZWQgZnJhZ21lbnRzOiBvcmlnaW5zLCBwcm9jZXNzaW5nLCBhbmQgZnVuY3Rpb25zOyBSTkE7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTg1My02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvc3Bob3J1cyBudXRyaXRpb24gb2Ygc3ByaW5nIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLikuIDEuIEVmZmVjdHMgb2YgcGhvc3Bob3J1cyBzdXBwbHkgb24gcGxhbnQgc3ltcHRvbXMsIHlpZWxkLCBjb21wb25lbnRzIG9mIHlpZWxkLCBhbmQgcGxhbnQgcGhvc3Bob3J1cyB1cHRha2U7IEF1c3QuIEouIEFncmljLiBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTQ1NTsgIGNpdGF0aW9uX3BhZ2VzPTg1NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFybGV5IGdlbm90eXBlcyB3aXRoIGxvbmcgcm9vdCBoYWlycyBzdXN0YWluIGhpZ2ggZ3JhaW4geWllbGRzIGluIGxvdy1QIGZpZWxkOyBQbGFudCBTb2lsOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI2MjsgIGNpdGF0aW9uX3BhZ2VzPTU1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaG9zcGhhdGUgdXRpbGl6YXRpb24gZWZmaWNpZW5jeSBjb3JyZWxhdGVzIHdpdGggZXhwcmVzc2lvbiBvZiBsb3ctYWZmaW5pdHkgcGhvc3BoYXRlIHRyYW5zcG9ydGVycyBhbmQgbm9uY29kaW5nIFJOQSwgSVBTMSwgaW4gYmFybGV5OyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0xMjE3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIG1pY3JvUk5BcyBpbiBwaG9zcGhvcnVzIGRlZmljaWVuY3kgc2lnbmFsbGluZzsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9MTAxNi0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkEgZXhwcmVzc2lvbiBwcm9maWxlcyBhc3NvY2lhdGVkIHdpdGggcGhvc3Bob3J1cyBkZWZpY2llbmN5IGluIHdoaXRlIGx1cGluIChMdXBpbnVzIGFsYnVzIEwuKTsgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzg7ICBjaXRhdGlvbl9wYWdlcz0yMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBwaG9zcGhvcnVzIGRlZmljaWVudCByZXNwb25zaXZlIG1pUk5BcyBhbmQgY2lzLWVsZW1lbnRzIGZyb20gc295YmVhbiAoR2x5Y2luZSBtYXggTC4pOyBKLiBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz0xMjg5LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmNvdmVyaW5nIHNtYWxsIFJOQS1tZWRpYXRlZCByZXNwb25zZXMgdG8gcGhvc3BoYXRlIGRlZmljaWVuY3kgaW4gQXJhYmlkb3BzaXMgYnkgZGVlcCBzZXF1ZW5jaW5nOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0yMTIwLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0b3J5IG5ldHdvcmsgb2YgbWljcm9STkEzOTkgYW5kIFBITzIgYnkgc3lzdGVtaWMgc2lnbmFsbGluZzsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9NzMyLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb1JOQTM5OSBpcyBhIGxvbmdkaXN0YW5jZSBzaWduYWwgZm9yIHRoZSByZWd1bGF0aW9uIG9mIHBsYW50IHBob3NwaGF0ZSBob21lb3N0YXNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz03MzEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiB0aGUgUGFsbGFzIGdlbmUgaW4gYmFybGV5OiBwaGVuZSBhbmFseXNpcywgb3ZlcmRvbWluYW5jZSwgdmFyaWFiaWxpdHk7IEhlcmVkaXRhczsgIGNpdGF0aW9uX3llYXI9MTk3NzsgIGNpdGF0aW9uX3ZvbHVtZT04NjsgIGNpdGF0aW9uX3BhZ2VzPTI1MS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdHVkeSBvZiB0aGUgcm9sZSBvZiByb290IG1vcnBob2xvZ2ljYWwgdHJhaXRzIGluIGdyb3d0aCBvZiBiYXJsZXkgaW4gemluYy1kZWZpY2llbnQgc29pbDsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9Mjc3NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNTSVJPIFB1Ymxpc2hpbmcsIE1lbGJvdXJuZSwgQXVzdHJhbGlhOyBQbGFudCBBbmFseXNpczogQW4gSW50ZXJwcmV0YXRpb24gTWFudWFsOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fcGFnZXM9ODYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcGFjdCBvZiBzb2lsIHRpbGxhZ2Ugb24gdGhlIHJvYnVzdG5lc3Mgb2YgdGhlIGdlbmV0aWMgY29tcG9uZW50IG9mIHZhcmlhdGlvbiBpbiBwaG9zcGhvcnVzIChQKSB1c2UgZWZmaWNpZW5jeSBpbiBiYXJsZXkgKEhvcmRldW0gdnVsZ2FyZSBMLik7IFBsYW50IFNvaWw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzM5OyAgY2l0YXRpb25fcGFnZXM9MTEzLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1taVJhbmFseXplcjogYW4gdXBkYXRlIG9uIHRoZSBkZXRlY3Rpb24gYW5kIGFuYWx5c2lzIG9mIG1pY3JvUk5BcyBpbiBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBleHBlcmltZW50czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9VzEzMi0xMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBUSUdSIFBsYW50IFJlcGVhdCBEYXRhYmFzZXM6IGEgY29sbGVjdGl2ZSByZXNvdXJjZSBmb3IgdGhlIGlkZW50aWZpY2F0aW9uIG9mIHJlcGV0aXRpdmUgc2VxdWVuY2VzIGluIHBsYW50czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDM2MC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UUkVQOiBhIGRhdGFiYXNlIG9mIFRyaXRpY2VhZSByZXBldGl0aXZlIGVsZW1lbnRzOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTYxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGJhc2UgdXBkYXRlLCBhIGRhdGFiYXNlIG9mIGV1a2FyeW90aWMgcmVwZXRpdGl2ZSBlbGVtZW50czsgQ3l0b2dlbmV0LiBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz00NjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJpbmcgbWljcm9STkFzIGZyb20gZGVlcCBzZXF1ZW5jaW5nIGRhdGEgdXNpbmcgbWlSRGVlcDsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQwNy0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWZvbGQgd2ViIHNlcnZlciBmb3IgbnVjbGVpYyBhY2lkIGZvbGRpbmcgYW5kIGh5YnJpZGl6YXRpb24gcHJlZGljdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzQwNi0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9cHNSTkFUYXJnZXQ6IGEgcGxhbnQgc21hbGwgUk5BIHRhcmdldCBhbmFseXNpcyBzZXJ2ZXI7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPVcxNTUtMTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvZ2Vub3VzIHNpUk5BIGFuZCBtaVJOQSB0YXJnZXRzIGlkZW50aWZpZWQgYnkgc2VxdWVuY2luZyBvZiB0aGUgQXJhYmlkb3BzaXMgZGVncmFkb21lOyBDdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTc1OC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgYmFybGV5IGJhY3RlcmlhbCBhcnRpZmljaWFsIGNocm9tb3NvbWUgbGlicmFyeSBzdWl0YWJsZSBmb3IgY2xvbmluZyBnZW5lcyBmb3IgYm9yb24gdG9sZXJhbmNlLCBzb2RpdW0gZXhjbHVzaW9uIGFuZCBoaWdoIGdyYWluIHppbmMgY29udGVudDsgUGxhbnQgQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTI5OyAgY2l0YXRpb25fcGFnZXM9MjkxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiBiYXJsZXkgbWlSTkFzIHRocm91Z2ggZGVlcCBzZXF1ZW5jaW5nIG9mIHNob3J0IHJlYWRzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBtaWNyb1JOQXMgZnJvbSBiYXJsZXkgKEhvcmRldW0gdnVsZ2FyZSBMLikgYnkgaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmc7IEludC4gSi4gTW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9Mjk3My04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkVSRCwgYSBwbGFudC1zcGVjaWZpYyBHVyBwcm90ZWluLCBkZWZpbmVzIGFuIGFkZGl0aW9uYWwgUk5BaS1kZXBlbmRlbnQgY2hyb21hdGluLWJhc2VkIHBhdGh3YXkgaW4gQXJhYmlkb3BzaXM7IE1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEyMS0xMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJEUjIgcGFydGlhbGx5IGFudGFnb25pemVzIHRoZSBwcm9kdWN0aW9uIG9mIFJEUjYtZGVwZW5kZW50IHNpUk5BIGluIHNlbnNlIHRyYW5zZ2VuZS1tZWRpYXRlZCBQVEdTOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3BhZ2VzPWUyOTc4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgY2xhc3Mgb2YgaGVhdC1yZXNwb25zaXZlIHNtYWxsIFJOQXMgZGVyaXZlZCBmcm9tIHRoZSBjaGxvcm9wbGFzdCBnZW5vbWUgb2YgQ2hpbmVzZSBjYWJiYWdlIChCcmFzc2ljYSByYXBhKTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9Mjg5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hvcnQgbm9uLWNvZGluZyBSTkEgZnJhZ21lbnRzIGFjY3VtdWxhdGluZyBpbiBjaGxvcm9wbGFzdHM6IGZvb3RwcmludHMgb2YgUk5BIGJpbmRpbmcgcHJvdGVpbnM/OyBOdWNsZWljIEFjaWQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTMxMDYtMzExNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bWlSMTU2LXJlZ3VsYXRlZCBTUEwgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGRlZmluZSBhbiBlbmRvZ2Vub3VzIGZsb3dlcmluZyBwYXRod2F5IGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEzODsgIGNpdGF0aW9uX3BhZ2VzPTczOC00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaVJOQSBpbnZvbHZlZCBpbiBwaG9zcGhhdGUtc3RhcnZhdGlvbiByZXNwb25zZSBpbiBBcmFiaWRvcHNpczsgQ3Vyci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMDM4LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBudXRyaWVudC1yZXNwb25zaXZlIEFyYWJpZG9wc2lzIGFuZCByYXBlc2VlZCBtaWNyb1JOQXMgYnkgY29tcHJlaGVuc2l2ZSByZWFsLXRpbWUgcG9seW1lcmFzZSBjaGFpbiByZWFjdGlvbiBwcm9maWxpbmcgYW5kIHNtYWxsIFJOQSBzZXF1ZW5jaW5nOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTA7ICBjaXRhdGlvbl9wYWdlcz0xNTQxLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhbWlseSBvZiBtaWNyb1JOQXMgcHJlc2VudCBpbiBwbGFudHMgYW5kIGFuaW1hbHM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0zMzU1LTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSB0aGF0IG1pUk5BcyBhcmUgZGlmZmVyZW50IGZyb20gb3RoZXIgUk5BcyBDZWxsOyBNb2wuIExpZmUgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNmE7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0yNDYtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBhbmQgZGl2ZXJnZW5jZSBvZiBwbGFudCBtaWNyb1JOQSBnZW5lczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDZiOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MjQzLTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwIHNlcXVlbmNpbmcgb2YgQnJhY2h5cG9kaXVtIHNtYWxsIFJOQXMgYXQgdGhlIGdsb2JhbCBnZW5vbWUgbGV2ZWwgaWRlbnRpZmllcyBtaWNyb1JOQXMgaW52b2x2ZWQgaW4gY29sZCBzdHJlc3MgcmVzcG9uc2U7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIG1pbmlhdHVyZSBpbnZlcnRlZC1yZXBlYXQgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIChNSVRFcykgYW5kIGJpb2dlbmVzaXMgb2YgdGhlaXIgc2lSTkFzIGluIHRoZSBTb2xhbmFjZWFlOiBuZXcgZnVuY3Rpb25hbCBpbXBsaWNhdGlvbnMgZm9yIE1JVEVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQyLTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIHJlcHJvZ3JhbW1pbmcgYW5kIHNtYWxsIFJOQSBzaWxlbmNpbmcgb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIHBvbGxlbjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz00NjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFudGlzZW5zZSB0cmFuc2NyaXB0aW9uIGluIHRoZSBtYW1tYWxpYW4gdHJhbnNjcmlwdG9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDk7ICBjaXRhdGlvbl9wYWdlcz0xNTY0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZG9nZW5vdXMgc2lSTkFzIGRlcml2ZWQgZnJvbSB0cmFuc3Bvc29ucyBhbmQgbVJOQXMgaW4gRHJvc29waGlsYSBzb21hdGljIGNlbGxzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNzctODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdGlvbiBvZiBwbGFzdGlkIHBob3Rvc3ludGhlc2lzIGdlbmVzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHJvb3RzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz02MjMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHJlc3BvbnNlIHRvIHN0cmVzczogYmlvY2hlbWljYWwgYWRhcHRhdGlvbnMgdG8gcGhvc3BoYXRlIGRlZmljaWVuY3k7IGNpdGF0aW9uX3B1Ymxpc2hlcj1NYXJjZWwgRGVra2VyLCBOZXcgWW9yaywgVVNBOyBFbmN5Y2xvcGVkaWEgb2YgUGxhbnQgYW5kIENyb3AgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3BhZ2VzPTk3Ni04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbCBvZiB0cmFuc2xhdGlvbiBhbmQgbVJOQSBkZWdyYWRhdGlvbiBieSBtaVJOQXMgYW5kIHNpUk5BczsgR2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTUxNS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJ5cGFub3NvbWEgY3J1emkgcHJvZHVjZXMgYSBwb3B1bGF0aW9uIG9mIHRSTkEtZGVyaXZlZCBzbWFsbCBSTkFzIHdoaWNoIGFyZSByZWNydWl0ZWQgdG8gc3BlY2lmaWMgY3l0b3BsYXNtaWMgZ3JhbnVsZXMgYW5kIHNlY3JldGVkIHRvIHRoZSBleHRyYWNlbGx1bGFyIG1lZGl1bTsgQk1DIFByb2MuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbC4gMTsgIGNpdGF0aW9uX3BhZ2VzPU83IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVjYXkgb2YgbVJOQXMgdGFyZ2V0ZWQgYnkgUklTQyByZXF1aXJlcyBYUk4xLCB0aGUgU2tpIGNvbXBsZXgsIGFuZCB0aGUgZXhvc29tZTsgUk5BOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDU5LTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaWx0ZXJpbmcgb2YgZGVlcCBzZXF1ZW5jaW5nIGRhdGEgcmV2ZWFscyB0aGUgZXhpc3RlbmNlIG9mIGFidW5kYW50IERpY2VyLWRlcGVuZGVudCBzbWFsbCBSTkFzIGRlcml2ZWQgZnJvbSB0Uk5BczsgUk5BOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjE0Ny02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXRYUk40IGRlZ3JhZGVzIG1STkEgaW4gQXJhYmlkb3BzaXMgYW5kIGl0cyBzdWJzdHJhdGVzIGluY2x1ZGUgc2VsZWN0ZWQgbWlSTkEgdGFyZ2V0czsgTW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTczLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiB0Uk5BLWRlcml2ZWQgc21hbGwgUk5BcyBpbiB0aGUgZ2xvYmFsIHJlZ3VsYXRpb24gb2YgUk5BIHNpbGVuY2luZzsgUk5BOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NjczLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgdGVybWluYWwgYW5kIGFzeW1tZXRyaWMgcHJvY2Vzc2luZyBvZiBzbWFsbCBSTkFzIGZyb20gclJOQXMsIHNub1JOQXMsIHNuUk5BcywgYW5kIHRSTkFzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz02Nzg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0b3J5IFJOQXMgZGVyaXZlZCBmcm9tIHRyYW5zZmVyIFJOQTsgUk5BOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTg2NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUk5hc2UgUm55MXAgY2xlYXZlcyB0Uk5BcyBhbmQgcHJvbW90ZXMgY2VsbCBkZWF0aCBkdXJpbmcgb3hpZGF0aXZlIHN0cmVzcyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTg1OyAgY2l0YXRpb25fcGFnZXM9NDMtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuZ2lvZ2VuaW4gY2xlYXZlcyB0Uk5BIGFuZCBwcm9tb3RlcyBzdHJlc3MtaW5kdWNlZCB0cmFuc2xhdGlvbmFsIHJlcHJlc3Npb247IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTg1OyAgY2l0YXRpb25fcGFnZXM9MzUtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHNtYWxsIG5vbi1jb2RpbmcgUk5BcyBmcm9tIG1pdG9jaG9uZHJpYSBhbmQgY2hsb3JvcGxhc3RzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0zODQyLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgb2YgZW5kb2dlbm91cyBzbWFsbCBub24tY29kaW5nIFJOQXMgaW4gT3J5emEgc2F0aXZhOyBHZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0yMS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWx1Y2lkYXRpb24gb2YgdGhlIHNtYWxsIFJOQSBjb21wb25lbnQgb2YgdGhlIHRyYW5zY3JpcHRvbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA5OyAgY2l0YXRpb25fcGFnZXM9MTU2Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRpdmVyc2UgYW5kIGV2b2x1dGlvbmFyaWx5IGZsdWlkIHNldCBvZiBtaWNyb1JOQXMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0zNDA3LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaXRlLXNwZWNpZmljIGJpbmRpbmcgb2YgYSBQUFIgcHJvdGVpbiBkZWZpbmVzIGFuZCBzdGFiaWxpemVzIDUwIGFuZCAzMCBtUk5BIHRlcm1pbmkgaW4gY2hsb3JvcGxhc3RzOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjA0Mi01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGN1dHRpbmcgY3JldyDigJMgcmlib251Y2xlYXNlcyBhcmUga2V5IHBsYXllcnMgaW4gdGhlIGNvbnRyb2wgb2YgcGxhc3RpZCBnZW5lIGV4cHJlc3Npb247IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTE2NjMtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpYm9udWNsZWFzZSBQIHByb2Nlc3NlcyBwb2x5Y2lzdHJvbmljIHRSTkEgdHJhbnNjcmlwdHMgaW4gRXNjaGVyaWNoaWEgY29saSBpbmRlcGVuZGVudCBvZiByaWJvbnVjbGVhc2UgRTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NzYxNC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPVJvbGUgb2YgcGhvc3Bob3J1cyBpbiBwaG90b3N5bnRoZXRpYyBjYXJib24gbWV0YWJvbGlzbTsgY2l0YXRpb25fcHVibGlzaGVyPVRheWxvciAmYW1wOyBGcmFuY2lzIEdyb3VwLCBMTEMsIFR1Y3NvbjsgSGFuZGJvb2sgb2YgUGhvdG9zeW50aGVzaXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl9wYWdlcz0xMjMtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC8yLzEwOS85NTI0NTQvZHNzMDM3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQaG9zcGhvcnVzIChQKSBpcyBlc3NlbnRpYWwgZm9yIHBsYW50IGdyb3d0aC4gTWljcm9STkFzIChtaVJOQXMpIHBsYXkgYSBrZXkgcm9sZSBpbiBwaG9zcGhhdGUgaG9tZW9zdGFzaXMuIEhvd2V2ZXIsIGxpdHRsZSBpcyBrbm93biBhYm91dCBQIGVmZmVjdCBvbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzMzMDgxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJRnPgBQySKMib/cXmb2bnndb6RD" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f109%2f333081" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f109%2f333081" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12698" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&amp;Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/2">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4600048">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600048">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600054">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600054">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600071">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600071">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600149">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600149">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600364">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600364">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600388">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600388">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600394">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600394">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4600401">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4600401">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/20/2/127/333239">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michael Hackenberg</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Computational Genomics and Bioinformatics Group, Genetics Department</div>, <div class="insititution">University of Granada</div>, <div class="addr-line">Granada 18071</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michael+Hackenberg">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hackenberg M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hackenberg M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michael Hackenberg</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Po-Jung Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bioinformatics Center</div>, <div class="insititution">Chang Gung University</div>, <div class="addr-line">Taoyuan, Taiwan</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Po-Jung+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Po-Jung Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chun-Yuan Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Australian Centre for Plant Functional Genomics</div>, <div class="insititution">The University of Adelaide</div>, <div class="addr-line">Adelaide, South Australia 5064</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chun-Yuan+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chun-Yuan Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bu-Jun Shi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Australian Centre for Plant Functional Genomics</div>, <div class="insititution">The University of Adelaide</div>, <div class="addr-line">Adelaide, South Australia 5064</div>, <div class="country">Australia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +(61) 8-83037168. Fax. +(61) 8-83037102. Email: <a href="mailto:bujun.shi@acpfg.com.au" target="_blank">bujun.shi@acpfg.com.au</a> (B.-J.S.); Tel. +(61) 8-83037014. Fax. +(61) 8-83037102. Email: <a href="mailto:peter.langridge@acpfg.com.au" target="_blank">peter.langridge@acpfg.com.au</a> (P.L.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bu-Jun+Shi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shi B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shi B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bu-Jun Shi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Perry Gustafson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">USDA-ARS</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">206 Curtis Hall, Columbia, MO 65211-7020</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Perry+Gustafson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gustafson P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gustafson P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Perry Gustafson</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peter Langridge</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Australian Centre for Plant Functional Genomics</div>, <div class="insititution">The University of Adelaide</div>, <div class="addr-line">Adelaide, South Australia 5064</div>, <div class="country">Australia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +(61) 8-83037168. Fax. +(61) 8-83037102. Email: <a href="mailto:bujun.shi@acpfg.com.au" target="_blank">bujun.shi@acpfg.com.au</a> (B.-J.S.); Tel. +(61) 8-83037014. Fax. +(61) 8-83037102. Email: <a href="mailto:peter.langridge@acpfg.com.au" target="_blank">peter.langridge@acpfg.com.au</a> (P.L.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peter+Langridge">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Langridge P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Langridge P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Peter Langridge</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 109–125, <a href='https://doi.org/10.1093/dnares/dss037'>https://doi.org/10.1093/dnares/dss037</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 December 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 August 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="333081" href="/dnaresearch/article-pdf/20/2/109/952454/dss037.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Michael Hackenberg, Po-Jung Huang, Chun-Yuan Huang, Bu-Jun Shi, Perry Gustafson, Peter Langridge;  A Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions, <em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 109–125, <a href='https://doi.org/10.1093/dnares/dss037'>https://doi.org/10.1093/dnares/dss037</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/333081?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/333081?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/333081?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/333081?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/333081?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/333081?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/333081?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Comprehensive%20Expression%20Profile%20of%20MicroRNAs%20and%20Other%20Classes%20of%20Non-Coding%20Small%20RNAs%20in%20Barley%20Under%20Phosphorous-Deficient%20and%20-Sufficient%20Conditions&amp;publicationDate=2012-12-23&amp;volumeNum=20&amp;issueNum=2&amp;author=Hackenberg%2C%20Michael%3B%20Huang%2C%20Po-Jung&amp;startPage=109&amp;endPage=125&amp;contentId=10.1093%2Fdnares%2Fdss037&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f109%2f333081" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4600048 id="4600048" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Phosphorus (P) is essential for plant growth. MicroRNAs (miRNAs) play a key role in phosphate homeostasis. However, little is known about P effect on miRNA expression in barley (<em>Hordeum vulgare</em> L.). In this study, we used Illumina's next-generation sequencing technology to sequence small RNAs (sRNAs) in barley grown under P-deficient and P-sufficient conditions. We identified 221 conserved miRNAs and 12 novel miRNAs, of which 55 were only present in P-deficient treatment while 32 only existed in P-sufficient treatment. Total 47 miRNAs were significantly differentially expressed between the two P treatments (|log<sub>2</sub>| &gt; 1). We also identified many other classes of sRNAs, including sense and antisense sRNAs, repeat-associated sRNAs, transfer RNA (tRNA)-derived sRNAs and chloroplast-derived sRNAs, and some of which were also significantly differentially expressed between the two P treatments. Of all the sRNAs identified, antisense sRNAs were the most abundant sRNA class in both P treatments. Surprisingly, about one-fourth of sRNAs were derived from the chloroplast genome, and a chloroplast-encoded tRNA-derived sRNA was the most abundant sRNA of all the sRNAs sequenced. Our data provide valuable clues for understanding the properties of sRNAs and new insights into the potential roles of miRNAs and other classes of sRNAs in the control of phosphate homeostasis.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;deep sequencing&quot;">deep sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="barley">barley</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phosphorus">phosphorus</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-coding small RNAs&quot;">non-coding small RNAs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="miRNAs">miRNAs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;differential expression&quot;">differential expression</a></div>                            <h2 scrollto-destination=4600054 id="4600054" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>MicroRNAs (miRNAs) are a novel class of single-stranded, non-coding (nc) small RNAs (sRNAs). They are about 21–22 nucleotides (nt) in length, derived from hairpin precursors (pre-miRNAs) that in turn are formed from miRNA primary transcripts. MiRNAs play critical roles in the regulation of genes responsible for cell division, differentiation, development and signalling.<sup><a reveal-id="DSS037C1" data-open="DSS037C1" class="link link-ref link-reveal xref-bibr">1</a></sup> So far, 4053 miRNAs have been identified in plants, including 22 in barley (<em>Hordeum vulgare</em> L.) (miRBase version 18, November 2011).</p><p>Another important class of sRNAs is small interfering RNAs (siRNAs) that also regulate gene expression. Although siRNAs are similar in size and function to miRNAs, they are double-stranded (ds) and processed from perfectly long dsRNAs derived from viruses, transgenes or endogenous transcripts such as transposons. Based on the genomic origin and the interaction of distinct components, endogenous siRNAs can be further classified into trans-acting siRNAs, natural antisense transcript-derived siRNAs (natsiRNAs), repeat-associated siRNAs (rasiRNAs), long siRNAs<sup><a reveal-id="DSS037C2" data-open="DSS037C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and heterochromatin siRNAs.<sup><a reveal-id="DSS037C3" data-open="DSS037C3" class="link link-ref link-reveal xref-bibr">3</a></sup> All these subclasses of siRNAs are involved in biological processes.</p><p>Recently, several other classes of sRNAs were defined. They are tiny ncRNAs, 21U-RNAs, scan RNAs, promoter/termini-associated sRNAs, transcription initiation RNAs, transcription start site-associated RNAs, splice site RNAs and sRNAs derived from ribosomal RNAs (rRNAs), small nucleolar RNAs (snoRNAs) and transfer RNAs (tRNAs).<sup><a reveal-id="DSS037C4" data-open="DSS037C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS037C5" data-open="DSS037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Most of these sRNAs are generated through the miRNA/siRNA pathway.<sup><a reveal-id="DSS037C5" data-open="DSS037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Distinctively, tRNA-derived sRNAs (tsRNAs) are essentially restricted to the cytoplasm and associated with argonaute proteins.</p><p>Barley is one of the most important cereal crops worldwide, ranking fourth in terms of production and is widely used for brewing and animal feed. Phosphorus (P) is an essential nutrient for plant growth and development. In cereal crops such as barley and wheat (<em>Triticum aestivum</em>), sufficient P in plants increases while deficient P in plants decreases the plant growth and grain yields.<sup><a reveal-id="DSS037C6  DSS037C8" data-open="DSS037C6  DSS037C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> Unfortunately, P availability in soil is low because soil P is often present in the precipitated or organic form that cannot be taken up by plants. On the other hand, plant requirements are high, especially for wheat and barley, whose production uses approximately 46% of the P fertilizers applied to cereals according to Food and Agriculture Organization (FAO) of the United Nations (FAO Fertilizer and Plant Nutrition Bulletin 17; <a class="link link-uri" href="http://www.fao.org">http://www.fao.org</a>). Thus, the improvement of P uptake from low concentrations of P in soil by cereal crops is crucial for sustainable agriculture and food security. Recently, miRNAs were found to participate in P uptake into plants.<sup><a reveal-id="DSS037C9" data-open="DSS037C9" class="link link-ref link-reveal xref-bibr">9</a></sup> While miRNAs regulate P starvation-responsive genes at the post-transcriptional level, they are regulated by different P concentrations <em>per se</em>. In lupin and soybean, 167 out of 771 and 57 out of 853 miRNAs examined, respectively, are induced by P limitation.<sup><a reveal-id="DSS037C10" data-open="DSS037C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS037C11" data-open="DSS037C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In <em>Arabidopsis,</em> miR399, miR778, miR2111 and miR827 are specifically induced by deficient P.<sup><a reveal-id="DSS037C12" data-open="DSS037C12" class="link link-ref link-reveal xref-bibr">12</a></sup> miR399 has been demonstrated to be a component of the shoot-to-root P deficiency signalling pathway.<sup><a reveal-id="DSS037C13" data-open="DSS037C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS037C14" data-open="DSS037C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In contrast to miR399, the roles of miR778, miR2111 or miR827 in the regulation of phosphate (Pi) homeostasis are not clear. In this study, we used Illumina's next-generation sequencing technology to sequence sRNAs in the shoots of barley grown under P deficient and P sufficient conditions. Our data reveal a comprehensive profile of miRNAs and other classes of sRNAs under different P conditions and provide a valuable resource for identifying miRNAs and/or sRNAs important for the control of Pi homeostasis.</p>                            <h2 scrollto-destination=4600071 id="4600071" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4600076 id="4600076" class="section-title" data-legacy-id=s2a>2.1. Barley cultivar</h3>
<p>Pallas barley (<em>H. vulgare</em> L., cv Pallas) is an X-ray mutant widely used for plant breeding and evolution.<sup><a reveal-id="DSS037C15" data-open="DSS037C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Pallas barley was grown in soil supplied with either 22.5 mg (deficient P) or 75 mg (sufficient P) Pi (KH<sub>2</sub>PO<sub>4</sub>) kg<sup>−1</sup> dry soil. Basal nutrients and calcium carbonate were added into the soil, and growth conditions were the same as those described.<sup><a reveal-id="DSS037C16" data-open="DSS037C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The plants were harvested 16 days after seed imbibition. It is to note that the P supply of 22.5 mg Pi in the soil only results in moderate P deficiency in barley plants. The reason we adopted such Pi concentration is based on realistic soil P-deficient condition and previous studies<sup><a reveal-id="DSS037C8" data-open="DSS037C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS037C17" data-open="DSS037C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS037C18" data-open="DSS037C18" class="link link-ref link-reveal xref-bibr">18</a></sup> showing that the supply of 22.5 mg Pi in the soil leads to 30% reduction in P concentration in barley shoots when compared with the critical P concentration required by barley plants.</p>                            <h3 scrollto-destination=4600087 id="4600087" class="section-title" data-legacy-id=s2b>2.2. sRNA isolation and sequencing</h3>
<p>Total RNA was extracted from shoot material using TRIzol reagent (Invitrogen, Carlsbad, CA, USA). sRNAs were isolated from total RNA using the Purelink miRNA Isolation Kit (Invitrogen, Carlsbad, CA, USA) and then electrophoresed on a 15% polyacrylamide (30:0.8) gel containing 7 molar (M) urea in 1× Tris-borate-ethylenediaminetetraacetic acid (EDTA) buffer (TBE: 0.04 M Tris base, 0.04 M acetate, 0.001 M EDTA, pH 8.0). Eighteen- to 30-nt sRNAs excised and eluted from the gel were ligated with both 5′ (5′-GUUCAGAGUUCUACAGUCCGACGAUC-3′) and 3′ (5′-pUCGUAUGCCGUCUUCUGCUUGUidT-3′, p-Pi, idT-inverted deoxythymidine) adaptors using T4 RNA ligase. Sequencing was performed using the 36-base Illumina platform.</p>                            <h3 scrollto-destination=4600095 id="4600095" class="section-title" data-legacy-id=s2c>2.3. Trimming of sRNA sequences and bioinformatics analysis</h3>
<p>The sequence reads obtained from the 36-base Illumina platform were filtered and trimmed as follows: after removing poly A, T, C, G and/or N-containing reads, all reads with the same read sequences were grouped and termed ‘unique reads’. Next, the 3′ adapter sequence was removed as described.<sup><a reveal-id="DSS037C19" data-open="DSS037C19" class="link link-ref link-reveal xref-bibr">19</a></sup> After adapter detection, the reads were re-grouped to calculate the read count of unique read sequences. All trimmed reads were compared with the complete barley chloroplast genome sequence without mismatches and then separated into non-chloroplast reads (nuclear-derived reads) and chloroplast reads. The expression of known miRNAs and other transcribed sequences was analysed using the miRanalyzer standalone version.<sup><a reveal-id="DSS037C19" data-open="DSS037C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Zero mismatch was set for all libraries except for Rfam that allows two mismatches. For all other parameters, the default values were used. After the reads were mapped to a given library, the mapped reads were removed and the remaining reads were used to map to another library and so on. To identify sRNAs derived from repeats such as (retro)transposons, the reads with a minimum length of 20 bp were aligned to the following libraries: (i) barley and rice libraries from the TIGR repeat database,<sup><a reveal-id="DSS037C20" data-open="DSS037C20" class="link link-ref link-reveal xref-bibr">20</a></sup> (ii) the Triticeae repeat sequence database (TREP)<sup><a reveal-id="DSS037C21" data-open="DSS037C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and (iii) all RepBase repeats.<sup><a reveal-id="DSS037C22" data-open="DSS037C22" class="link link-ref link-reveal xref-bibr">22</a></sup> To identify sRNAs corresponding to rRNAs, tRNAs, small nuclear RNAs (snRNAs), snoRNAs, spliceosome RNAs (U1–U6) and other annotated ncRNAs, we compared the reads with sequences extracted from the Rfam database containing various ncRNA families and other structured RNA elements. To analyse antisense sRNAs (also called siRNAs), we first removed the reads mapped to known miRNAs and then aligned the remaining reads to the reverse strand of all barley genes available in the databases. Reads hitting the same genes were grouped into ‘read clusters’ based on their sequence similarity. The ‘anti-sense’ reads were then mapped to the forward strand of all other libraries (tRNA, genes, repeats, etc.). Because a gene could be hit by different reads (different length variants or from different genome elements), we clustered similar reads together (reads that are perfect inclusions of each other).</p>                            <h3 scrollto-destination=4600105 id="4600105" class="section-title" data-legacy-id=s2d>2.4. Prediction of novel miRNAs and their targets</h3>
<p>Prediction of novel miRNAs was performed with miRDeep<sup><a reveal-id="DSS037C23" data-open="DSS037C23" class="link link-ref link-reveal xref-bibr">23</a></sup> that uses a probabilistic model of miRNA biogenesis to score compatibility of the position and frequency of the deep-sequencing reads with the hairpin structure of the miRNA precursor. All the trimmed reads were mapped (without mismatch) to barley expressed sequence tag (EST) sequences retrieved from the HarvEST:Barley (version 1.83) database (<a class="link link-uri" href="http://harvest.ucr.edu">http://harvest.ucr.edu</a>). The predicted precursor sequences were examined for their stem-loop structures with MFOLD.<sup><a reveal-id="DSS037C24" data-open="DSS037C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Only putative miRNA precursors meeting a minimum free energy lower than –16 kcal/mol were retained for next analysis. Target genes of the putative novel miRNAs were predicted using psRNATarget,<sup><a reveal-id="DSS037C25" data-open="DSS037C25" class="link link-ref link-reveal xref-bibr">25</a></sup> a plant sRNA target analysis server (<a class="link link-uri" href="http://plantgrn.noble.org/psRNATarget">http://plantgrn.noble.org/psRNATarget</a>).</p>                            <h3 scrollto-destination=4600114 id="4600114" class="section-title" data-legacy-id=s2e>2.5. Degradome library construction</h3>
<p>A degradome library was constructed as described.<sup><a reveal-id="DSS037C26" data-open="DSS037C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Briefly, poly(A) RNA, extracted from total RNA of Pallas shoots and roots using the Oligotex Kit (Qiagen), was ligated with a 5′ RNA adaptor containing a <em>Mme</em>I restriction site using T4 RNA ligase (Invitrogen), followed by reverse transcription, second-strand synthesis, <em>Mme</em>I digestion, ligation of a 3′ dsDNA adaptor, gel purification, and polymerase chain reaction (PCR) amplification. Amplified PCR products were sequenced with the Illumina HiSeq platform.</p>                            <h3 scrollto-destination=4600123 id="4600123" class="section-title" data-legacy-id=s2f>2.6. Northern blot hybridization</h3>
<p>Fifty micrograms of total RNA extracted from barley were separated on a 15% polyacrylamide gel containing 7 M urea and transferred to Hybond-N membrane (Amersham Bioscience) using 20× saline-sodium citrate buffer (SSC: 3 M NaCl and 0.3 M sodium citrate, pH 7.0). The membrane was hybridized with <sup>32</sup>P-labelled DNA oligonucleotide probe reverse complementary to predicted miRNA sequence and made with γ-<sup>32</sup>P -adenosine triphosphate (ATP) using T4 polynucleotide kinase (New England Biolabs, Beverly, MA, USA). A U6 snRNA-specific probe served as a loading control. Prehybridization and hybridization were both performed at 37°C in 125 mM Na<sub>2</sub>HPO<sub>4</sub> (pH 7.2), 250 mM NaCl, 7% sodium dodecyl sulphate (SDS) and 50% formamide. Washing was also performed at 37°C using 2x SSC (0.3 M NaCl and 0.03 M sodium citrate), 0.2% SDS, twice followed by 1x SSC, 0.1% SDS once. The membrane was subjected to autoradiography at –80°C for 7 days.</p>                            <h3 scrollto-destination=4600133 id="4600133" class="section-title" data-legacy-id=s2g>2.7. Southern blot hybridization</h3>
<p>Southern blot hybridization was carried out as described.<sup><a reveal-id="DSS037C27" data-open="DSS037C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Briefly, barley genomic DNA was digested with appropriate restriction enzymes and separated via an electrophoresis gel. The DNA was then transferred to a membrane and hybridized with radiolabelled probes. After hybridization, the membrane was washed once at 65°C for 30–45 minutes (min) in 2x SSC containing 0.1% SDS and twice at 65°C for 30–45 min in 1x SSC containing 0.1% SDS. The membrane was subjected to autoradiography at −80°C as required until some background signal became visible.</p>                            <h3 scrollto-destination=4600141 id="4600141" class="section-title" data-legacy-id=s2h>2.8. Reverse transcription PCR (RT-PCR)</h3>
<p>Reverse transcription PCR (RT-PCR) was performed as described.<sup><a reveal-id="DSS037C28" data-open="DSS037C28" class="link link-ref link-reveal xref-bibr">28</a></sup> First-strand cDNA was synthesized using total RNA as template with a primer reverse complementary to a miRNA precursor sequence. The synthesized cDNAs of pre-miRNAs were amplified using specific primer pairs with the conditions: 7 min at 95°C, followed by 45 cycles of 10 seconds (s) at 95°C, 10 s at 55°C and 30 s at 72°C and final extension step at 72°C for 10 min. The amplified products were electrophoresed in a 2% agarose gel in 1× Tris-acetate-EDTA buffer (TAE: 40 mM Tris base, 40 mM acetate and 1 mM EDTA, pH 8.3).</p>                            <h2 scrollto-destination=4600149 id="4600149" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4600155 id="4600155" class="section-title" data-legacy-id=s3a>3.1. Phenotype of Pallas barley under P-deficient condition</h3>
<p>Pallas barley was grown in the soil supplied with 22.5 mg Pi kg<sup>−1</sup> soil (moderately deficient P, hereafter designated deficient P) and 75 mg Pi kg<sup>−1</sup> soil (sufficient P). Under deficient P, Pallas grew slowly when compared with plants grown with sufficient P, but showed no other symptoms of P deficiency (Fig. <a reveal-id="DSS037F1" data-open="DSS037F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4600164" scrollto-destination="4600164"></a>
<div id="DSS037F1" data-id="DSS037F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03701.jpeg?Expires=2147483647&Signature=YQFAKqKhgFNthjaFsdNSEBqZDvckFq3ZhDV2a1Qxgmd1V5uDGPADhSUPV~nmJ~RNCnVuRd1mzDZ~AMYFJdayg7pLMt3bin1HC8cwQWQr8osO8STMl9PrwEUMxB~syWpfRcL-pyMq5fCjAo7rSknWqpzTYHZieE~d368M9W0QRpomARXTQIaaSM8giaTF2YBiqvFs-r0dPlzrhlpaYMiJllH0RXKFiIQqwwK-aKGFFeGf2mHKkZj80xn-S8IMo1e78o6tlfcH7AuPTm4n7Lkevbt4Z4~pHnLkOsSwVi90y2mzs9NmfIjJMMDleMxRs6ghp9PrZAkWdE5F2KoZly6wRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenotypes of barley (H. vulgare L. cv Pallas) grown for 16 days in soil amended with either 22.5 mg (deficient P) or 75 mg (sufficient P) KH2PO4 kg−1 dry soil." path-from-xml="dss03701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600164/dss03701.jpeg" path-from-xml="dss03701.jpeg" target="_blank">View large</a><a section="4600164" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03701.jpeg?Expires=2147483647&Signature=SePa8ug3gAe17ALeHH0SEIW-nfVlHxFSxfcaNjpQP50shwNPzoSlwXCux-j4wKXIAY-VAuFYRtaR0A8uXzAIVwXz14tpk-tbiZn6xHMmH-DLDqFBLmZLSsSL-ZVGQ~kfigWkZeA1x-7NwVfXOZe5M12GprzvczLPSsgpUCZnHyeVO28VggkcR~kq94BzCFdnpPisCQko7ayw-LHTvdNGjAO-V7dEqfwyt3qpvEdptakc4rSbjXPsl~O3N1xczLsdCaTvSMN~cRCjqHjJD42PHDkEmB~f9HLA46HrHpzY6ibljV~JmgCvsRt-Ivvh07eCjgphfawKq5ovM6dT8s-iHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600164&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotypes of barley (<em>H. vulgare</em> L. cv Pallas) grown for 16 days in soil amended with either 22.5 mg (deficient P) or 75 mg (sufficient P) KH<sub>2</sub>PO<sub>4</sub> kg<sup>−1</sup> dry soil.</p></div></div><div content-id="DSS037F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03701.jpeg?Expires=2147483647&Signature=YQFAKqKhgFNthjaFsdNSEBqZDvckFq3ZhDV2a1Qxgmd1V5uDGPADhSUPV~nmJ~RNCnVuRd1mzDZ~AMYFJdayg7pLMt3bin1HC8cwQWQr8osO8STMl9PrwEUMxB~syWpfRcL-pyMq5fCjAo7rSknWqpzTYHZieE~d368M9W0QRpomARXTQIaaSM8giaTF2YBiqvFs-r0dPlzrhlpaYMiJllH0RXKFiIQqwwK-aKGFFeGf2mHKkZj80xn-S8IMo1e78o6tlfcH7AuPTm4n7Lkevbt4Z4~pHnLkOsSwVi90y2mzs9NmfIjJMMDleMxRs6ghp9PrZAkWdE5F2KoZly6wRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenotypes of barley (H. vulgare L. cv Pallas) grown for 16 days in soil amended with either 22.5 mg (deficient P) or 75 mg (sufficient P) KH2PO4 kg−1 dry soil." path-from-xml="dss03701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600164/dss03701.jpeg" path-from-xml="dss03701.jpeg" target="_blank">View large</a><a section="4600164" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03701.jpeg?Expires=2147483647&Signature=SePa8ug3gAe17ALeHH0SEIW-nfVlHxFSxfcaNjpQP50shwNPzoSlwXCux-j4wKXIAY-VAuFYRtaR0A8uXzAIVwXz14tpk-tbiZn6xHMmH-DLDqFBLmZLSsSL-ZVGQ~kfigWkZeA1x-7NwVfXOZe5M12GprzvczLPSsgpUCZnHyeVO28VggkcR~kq94BzCFdnpPisCQko7ayw-LHTvdNGjAO-V7dEqfwyt3qpvEdptakc4rSbjXPsl~O3N1xczLsdCaTvSMN~cRCjqHjJD42PHDkEmB~f9HLA46HrHpzY6ibljV~JmgCvsRt-Ivvh07eCjgphfawKq5ovM6dT8s-iHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600164&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotypes of barley (<em>H. vulgare</em> L. cv Pallas) grown for 16 days in soil amended with either 22.5 mg (deficient P) or 75 mg (sufficient P) KH<sub>2</sub>PO<sub>4</sub> kg<sup>−1</sup> dry soil.</p></div></div>                            <h3 scrollto-destination=4600174 id="4600174" class="section-title" data-legacy-id=s3b>3.2. Size distribution of sRNAs from P-deficient and P-sufficient barley shoots</h3>
<p>Totals of 7 006 282 sRNA sequencing reads of 36 bases from P-deficient barley shoots and 6 184 850 sRNA sequencing reads of 36 bases from P-sufficient barley shoots were obtained [the sequences have been deposited in the publicly accessible database, National Center for Biotechnology Information (NCBI) (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) with accession number SRA025074.1]. Figure <a reveal-id="DSS037F2" data-open="DSS037F2" class="link link-reveal link-table xref-fig">2</a>A shows the size distribution of sRNAs in P-deficient and P-sufficient shoots after the removal of poly A, T, C, G, N and adaptor sequences. Twenty and 21 nt were the two major size classes in each P treatment. Twenty four-nt sRNAs were abundant in P-deficient shoots, but not in P-sufficient shoots. Nineteen- and 22-nt sRNAs were slightly less abundant than the 20-, 21- and 24-nt sRNAs for both P treatments. Other sizes of sRNAs were found in low abundance in both P treatments. Overall, 19-, 20-, 21- and 24-nt sRNAs were more abundant in P-deficient shoots than in P-sufficient shoots (Fig. <a reveal-id="DSS037F2" data-open="DSS037F2" class="link link-reveal link-table xref-fig">2</a>A). These data are not very consistent with the size distribution of sRNAs from another barley cultivar Clipper (<em>H. vulgare</em> L. cv Clipper), where 24 nt was the only major size class.<sup><a reveal-id="DSS037C29" data-open="DSS037C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Clipper and Pallas are two distinct barley genotypes. Taken together, these suggest that the expression of 24-nt sRNAs is subject to conditions. </p>                        <a id="4600182" scrollto-destination="4600182"></a>
<div id="DSS037F2" data-id="DSS037F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03702.jpeg?Expires=2147483647&Signature=PPXFRU44wUTI~XUIdE5Ky8NAYWXwlCd5~MInCVakiWXILaArtZnZzRC5EIu-XjHqEvZ3M2rNMFmIhj5EfjeWLqg5T0Yoav97BxVB4XvqwA8nRvIu50gG43hTz1ksMpC17OiqNw8gfnPyaMSyBhIOu1TnUO3o0Hbe6myjnlHZsjO0-Hy4VwsX5iE583fvfSGCRk1UXorJfa8MaOGXgL9SjuXEPNColWAE0aJagL1PBLS2zHL2kcIMjHSkcJ2KoeoyHxvAUkbzu81Yc1zI8N4cgOc9QG~mY6I~tE8NwDgHZ06LhuvSJNGoI5I4W4SyBWyEAAu2mOghza1CE5vg0bedZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Size distribution of sRNA sequencing reads from P-deficient and P-sufficient barley shoots. (A) Size distribution of total reads. (B) Size distribution of unique reads. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600182/dss03702.jpeg" path-from-xml="dss03702.jpeg" target="_blank">View large</a><a section="4600182" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03702.jpeg?Expires=2147483647&Signature=PHZx2STmjD1mBEGoBI-OxKrFKgpSutraNf72QI8rQ24qnsjNPI3kf6LZ62jFy3azyhMKvonr9rR7OU7~sgqNBQ9ufnBvAR6xOSqcKGOpAo5185eqHPTeAWYyFavZB4qz6XH9u-j6GquEt0rwTSvnIBIbCNMmTqZzrM9j5Fok~p4KHGno9MYf04KxbCOI2iiVbjPIbKBRZJ5Sq3lcqiJX-g8naUGCX2EJQCCiME5OLLFs8LtvPjxMmRpHi4GJJt9U2RwFS6fCTWDlXaDgSuOfrM9NAkF6S1KHwHzu8RtLI3OMz6-oc4lIe6Bf7uXXGh23tm-kw4J1miy7bV2bBCHaBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600182&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Size distribution of sRNA sequencing reads from P-deficient and P-sufficient barley shoots. (A) Size distribution of total reads. (B) Size distribution of unique reads. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS037F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03702.jpeg?Expires=2147483647&Signature=PPXFRU44wUTI~XUIdE5Ky8NAYWXwlCd5~MInCVakiWXILaArtZnZzRC5EIu-XjHqEvZ3M2rNMFmIhj5EfjeWLqg5T0Yoav97BxVB4XvqwA8nRvIu50gG43hTz1ksMpC17OiqNw8gfnPyaMSyBhIOu1TnUO3o0Hbe6myjnlHZsjO0-Hy4VwsX5iE583fvfSGCRk1UXorJfa8MaOGXgL9SjuXEPNColWAE0aJagL1PBLS2zHL2kcIMjHSkcJ2KoeoyHxvAUkbzu81Yc1zI8N4cgOc9QG~mY6I~tE8NwDgHZ06LhuvSJNGoI5I4W4SyBWyEAAu2mOghza1CE5vg0bedZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Size distribution of sRNA sequencing reads from P-deficient and P-sufficient barley shoots. (A) Size distribution of total reads. (B) Size distribution of unique reads. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600182/dss03702.jpeg" path-from-xml="dss03702.jpeg" target="_blank">View large</a><a section="4600182" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03702.jpeg?Expires=2147483647&Signature=PHZx2STmjD1mBEGoBI-OxKrFKgpSutraNf72QI8rQ24qnsjNPI3kf6LZ62jFy3azyhMKvonr9rR7OU7~sgqNBQ9ufnBvAR6xOSqcKGOpAo5185eqHPTeAWYyFavZB4qz6XH9u-j6GquEt0rwTSvnIBIbCNMmTqZzrM9j5Fok~p4KHGno9MYf04KxbCOI2iiVbjPIbKBRZJ5Sq3lcqiJX-g8naUGCX2EJQCCiME5OLLFs8LtvPjxMmRpHi4GJJt9U2RwFS6fCTWDlXaDgSuOfrM9NAkF6S1KHwHzu8RtLI3OMz6-oc4lIe6Bf7uXXGh23tm-kw4J1miy7bV2bBCHaBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600182&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Size distribution of sRNA sequencing reads from P-deficient and P-sufficient barley shoots. (A) Size distribution of total reads. (B) Size distribution of unique reads. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>After the removal of duplicated reads, 24-nt sRNAs were the most dominant size class, followed by the 23-nt sRNAs for both P treatments (Fig. <a reveal-id="DSS037F2" data-open="DSS037F2" class="link link-reveal link-table xref-fig">2</a>B). This indicates that 20- and 21-nt sRNAs largely consisted of redundant sRNAs, whereas 24-nt sRNAs primarily comprised unique or low abundant sRNAs. Twenty four-nt sRNAs, which are implicated to derive from intergenic and repetitive genomic regions and mainly guide transcriptional gene silencing,<sup><a reveal-id="DSS037C30" data-open="DSS037C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSS037C31" data-open="DSS037C31" class="link link-ref link-reveal xref-bibr">31</a></sup> accounted for 48.5% of the total unique sRNAs in P-deficient shoots and 29.4% in P-sufficient shoots. The number of 24-nt sRNAs in P-deficient shoots was over twice that in P-sufficient shoots. In contrast, the numbers of other sizes of sRNAs were similar for each P treatment (Fig. <a reveal-id="DSS037F2" data-open="DSS037F2" class="link link-reveal link-table xref-fig">2</a>B). When a read count threshold ≥4 or 10 was applied, all sRNA sizes were significantly decreased in number in both P-deficient and P-sufficient shoots, except for 24-nt sRNAs in P-deficient shoots, indicating that most sRNAs are low copy.</p>                            <h3 scrollto-destination=4600195 id="4600195" class="section-title" data-legacy-id=s3c>3.3. sRNAs are generated from both the nuclear and chloroplast genomes</h3>
<p>Chloroplasts have their own genome, transcriptional and translational machineries. Many chloroplast-derived sRNAs (csRNAs) have recently been identified.<sup><a reveal-id="DSS037C32" data-open="DSS037C32" class="link link-ref link-reveal xref-bibr">32</a>–<a reveal-id="DSS037C34" data-open="DSS037C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We took advantage of the available barley chloroplast genome sequence to distinguish between sRNAs derived from the chloroplast and nuclear genomes in barley and found that about one-fourth of sRNAs were derived from the chloroplast genome in both P-deficient and P-sufficient shoots (Table <a reveal-id="DSS037TB1" data-open="DSS037TB1" class="link link-reveal link-table xref-fig">1</a>). In contrast to sRNAs derived from the nuclear genome, csRNAs were highly redundant in both P treatments. Further analysis showed that there were fewer differentially expressed sRNAs derived from the chloroplast genome in P-deficient shoots than in P-sufficient shoots, whereas the trend was reversed for differentially expressed sRNAs derived from the nuclear genome (Table <a reveal-id="DSS037TB1" data-open="DSS037TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4600203" scrollto-destination="4600203"></a>
<div content-id="DSS037TB1" class="table-modal"><div class="table-wrap"><div id="DSS037TB1" class="table-wrap-title" data-id="DSS037TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the sequencing reads in P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Non-chloroplast reads<hr /> </th><th colspan="3">Chloroplast reads (0 mismatch)<hr /> </th></tr></thead><tbody><tr><td>≥1 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas+</em>P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>811 530 </td><td>1 394 560 </td><td>0.58 </td><td>54 608 </td><td>62 942 </td><td>0.87 </td></tr><tr><td>Total read count </td><td>3 153 867 </td><td>4 137 948 </td><td>0.76 </td><td>1 153 778 </td><td>1 738 653 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>650 912 </td><td>1 233 942 </td><td>0.53 </td><td>31 604 </td><td>39 938 </td><td>0.79 </td></tr><tr><td>Common reads </td><td> </td><td>160 618 </td><td> </td><td> </td><td>23 004 </td><td> </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>25 981 </td><td> </td><td>27 132 </td><td>6327 </td><td> </td><td>2050 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="3">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">33.07 </td><td colspan="3">36.42 </td></tr><tr><td>≥4 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>36 309 </td><td>68 306 </td><td>0.53 </td><td>7575 </td><td>9857 </td><td>0.77 </td></tr><tr><td>Total read count </td><td>2 282 413 </td><td>2 533 554 </td><td>0.90 </td><td>1 091 929 </td><td>1 665 248 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>2314 </td><td>5193 </td><td>0.45 </td><td>497 </td><td>477 </td><td>1.04 </td></tr><tr><td>Common reads </td><td colspan="3">42 104 </td><td colspan="3">7446 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>9243 </td><td> </td><td>12 259 </td><td>2698 </td><td> </td><td>1194 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">51.07 </td><td colspan="3">52.27 </td></tr><tr><td>≥10 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>13 577 </td><td>19 148 </td><td>0.71 </td><td>2503 </td><td>3219 </td><td>0.78 </td></tr><tr><td>Total read count </td><td>2 157 072 </td><td>2 270 506 </td><td>0.95 </td><td>1 063 737 </td><td>1 628 357 </td><td>0.65 </td></tr><tr><td>Treatment-specific reads </td><td>96 </td><td>330 </td><td>0.29 </td><td>11 </td><td>62 </td><td>0.18 </td></tr><tr><td>Common reads </td><td colspan="2">17 140 </td><td> </td><td colspan="2">2938 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>4293 </td><td> </td><td>3753 </td><td>1032 </td><td> </td><td>487 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">46.94 </td><td colspan="3">51.70 </td></tr></tbody></table></div><div content-id="DSS037TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Non-chloroplast reads<hr /> </th><th colspan="3">Chloroplast reads (0 mismatch)<hr /> </th></tr></thead><tbody><tr><td>≥1 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas+</em>P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>811 530 </td><td>1 394 560 </td><td>0.58 </td><td>54 608 </td><td>62 942 </td><td>0.87 </td></tr><tr><td>Total read count </td><td>3 153 867 </td><td>4 137 948 </td><td>0.76 </td><td>1 153 778 </td><td>1 738 653 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>650 912 </td><td>1 233 942 </td><td>0.53 </td><td>31 604 </td><td>39 938 </td><td>0.79 </td></tr><tr><td>Common reads </td><td> </td><td>160 618 </td><td> </td><td> </td><td>23 004 </td><td> </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>25 981 </td><td> </td><td>27 132 </td><td>6327 </td><td> </td><td>2050 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="3">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">33.07 </td><td colspan="3">36.42 </td></tr><tr><td>≥4 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>36 309 </td><td>68 306 </td><td>0.53 </td><td>7575 </td><td>9857 </td><td>0.77 </td></tr><tr><td>Total read count </td><td>2 282 413 </td><td>2 533 554 </td><td>0.90 </td><td>1 091 929 </td><td>1 665 248 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>2314 </td><td>5193 </td><td>0.45 </td><td>497 </td><td>477 </td><td>1.04 </td></tr><tr><td>Common reads </td><td colspan="3">42 104 </td><td colspan="3">7446 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>9243 </td><td> </td><td>12 259 </td><td>2698 </td><td> </td><td>1194 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">51.07 </td><td colspan="3">52.27 </td></tr><tr><td>≥10 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>13 577 </td><td>19 148 </td><td>0.71 </td><td>2503 </td><td>3219 </td><td>0.78 </td></tr><tr><td>Total read count </td><td>2 157 072 </td><td>2 270 506 </td><td>0.95 </td><td>1 063 737 </td><td>1 628 357 </td><td>0.65 </td></tr><tr><td>Treatment-specific reads </td><td>96 </td><td>330 </td><td>0.29 </td><td>11 </td><td>62 </td><td>0.18 </td></tr><tr><td>Common reads </td><td colspan="2">17 140 </td><td> </td><td colspan="2">2938 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>4293 </td><td> </td><td>3753 </td><td>1032 </td><td> </td><td>487 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">46.94 </td><td colspan="3">51.70 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>≥1, ≥4 or ≥10 is the read count threshold. For the detection of down-regulated [log<sub>2</sub>(−P/+P) ≤−1] and up-regulated reads [log<sub>2</sub>(−P/+P) ≥1], only common reads (present in both P conditions) are used.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600203" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS037TB1" class="table-wrap-title" data-id="DSS037TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the sequencing reads in P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Non-chloroplast reads<hr /> </th><th colspan="3">Chloroplast reads (0 mismatch)<hr /> </th></tr></thead><tbody><tr><td>≥1 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas+</em>P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>811 530 </td><td>1 394 560 </td><td>0.58 </td><td>54 608 </td><td>62 942 </td><td>0.87 </td></tr><tr><td>Total read count </td><td>3 153 867 </td><td>4 137 948 </td><td>0.76 </td><td>1 153 778 </td><td>1 738 653 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>650 912 </td><td>1 233 942 </td><td>0.53 </td><td>31 604 </td><td>39 938 </td><td>0.79 </td></tr><tr><td>Common reads </td><td> </td><td>160 618 </td><td> </td><td> </td><td>23 004 </td><td> </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>25 981 </td><td> </td><td>27 132 </td><td>6327 </td><td> </td><td>2050 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="3">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">33.07 </td><td colspan="3">36.42 </td></tr><tr><td>≥4 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>36 309 </td><td>68 306 </td><td>0.53 </td><td>7575 </td><td>9857 </td><td>0.77 </td></tr><tr><td>Total read count </td><td>2 282 413 </td><td>2 533 554 </td><td>0.90 </td><td>1 091 929 </td><td>1 665 248 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>2314 </td><td>5193 </td><td>0.45 </td><td>497 </td><td>477 </td><td>1.04 </td></tr><tr><td>Common reads </td><td colspan="3">42 104 </td><td colspan="3">7446 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>9243 </td><td> </td><td>12 259 </td><td>2698 </td><td> </td><td>1194 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">51.07 </td><td colspan="3">52.27 </td></tr><tr><td>≥10 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>13 577 </td><td>19 148 </td><td>0.71 </td><td>2503 </td><td>3219 </td><td>0.78 </td></tr><tr><td>Total read count </td><td>2 157 072 </td><td>2 270 506 </td><td>0.95 </td><td>1 063 737 </td><td>1 628 357 </td><td>0.65 </td></tr><tr><td>Treatment-specific reads </td><td>96 </td><td>330 </td><td>0.29 </td><td>11 </td><td>62 </td><td>0.18 </td></tr><tr><td>Common reads </td><td colspan="2">17 140 </td><td> </td><td colspan="2">2938 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>4293 </td><td> </td><td>3753 </td><td>1032 </td><td> </td><td>487 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">46.94 </td><td colspan="3">51.70 </td></tr></tbody></table></div><div content-id="DSS037TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Non-chloroplast reads<hr /> </th><th colspan="3">Chloroplast reads (0 mismatch)<hr /> </th></tr></thead><tbody><tr><td>≥1 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas+</em>P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>811 530 </td><td>1 394 560 </td><td>0.58 </td><td>54 608 </td><td>62 942 </td><td>0.87 </td></tr><tr><td>Total read count </td><td>3 153 867 </td><td>4 137 948 </td><td>0.76 </td><td>1 153 778 </td><td>1 738 653 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>650 912 </td><td>1 233 942 </td><td>0.53 </td><td>31 604 </td><td>39 938 </td><td>0.79 </td></tr><tr><td>Common reads </td><td> </td><td>160 618 </td><td> </td><td> </td><td>23 004 </td><td> </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>25 981 </td><td> </td><td>27 132 </td><td>6327 </td><td> </td><td>2050 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="3">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">33.07 </td><td colspan="3">36.42 </td></tr><tr><td>≥4 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>36 309 </td><td>68 306 </td><td>0.53 </td><td>7575 </td><td>9857 </td><td>0.77 </td></tr><tr><td>Total read count </td><td>2 282 413 </td><td>2 533 554 </td><td>0.90 </td><td>1 091 929 </td><td>1 665 248 </td><td>0.66 </td></tr><tr><td>Treatment-specific reads </td><td>2314 </td><td>5193 </td><td>0.45 </td><td>497 </td><td>477 </td><td>1.04 </td></tr><tr><td>Common reads </td><td colspan="3">42 104 </td><td colspan="3">7446 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>9243 </td><td> </td><td>12 259 </td><td>2698 </td><td> </td><td>1194 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">51.07 </td><td colspan="3">52.27 </td></tr><tr><td>≥10 </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td><td><em>Pallas</em> + P </td><td><em>Pallas</em> − P </td><td>+P/−P </td></tr><tr><td>Unique reads </td><td>13 577 </td><td>19 148 </td><td>0.71 </td><td>2503 </td><td>3219 </td><td>0.78 </td></tr><tr><td>Total read count </td><td>2 157 072 </td><td>2 270 506 </td><td>0.95 </td><td>1 063 737 </td><td>1 628 357 </td><td>0.65 </td></tr><tr><td>Treatment-specific reads </td><td>96 </td><td>330 </td><td>0.29 </td><td>11 </td><td>62 </td><td>0.18 </td></tr><tr><td>Common reads </td><td colspan="2">17 140 </td><td> </td><td colspan="2">2938 </td></tr><tr><td> </td><td colspan="2">Down-regulated </td><td>Up-regulated </td><td colspan="2">Down-regulated </td><td>Up-regulated </td></tr><tr><td> </td><td>4293 </td><td> </td><td>3753 </td><td>1032 </td><td> </td><td>487 </td></tr><tr><td> </td><td colspan="3">Percentage of dysregulated reads </td><td colspan="2">Percentage of dysregulated reads </td></tr><tr><td> </td><td colspan="3">46.94 </td><td colspan="3">51.70 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>≥1, ≥4 or ≥10 is the read count threshold. For the detection of down-regulated [log<sub>2</sub>(−P/+P) ≤−1] and up-regulated reads [log<sub>2</sub>(−P/+P) ≥1], only common reads (present in both P conditions) are used.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600203" target="_blank">View Large</a></div></div><p>When a read count threshold ≥4 or 10 was applied, the numbers of P treatment-specific sRNAs derived from the nuclear or chloroplast genome were greatly reduced (Table <a reveal-id="DSS037TB1" data-open="DSS037TB1" class="link link-reveal link-table xref-fig">1</a>), indicating that many of the treatment-specific sRNAs are low in abundance. The abundance of sRNAs derived from the nuclear or chloroplast genome, or common to both P treatments, is also reduced, but not significantly. In contrast, the abundance of sRNAs derived from the nuclear or chloroplast genome, or differentially expressed under P deficiency and sufficiency, is less changed, indicating that most of these sRNAs are relatively abundant.</p>                            <h3 scrollto-destination=4600210 id="4600210" class="section-title" data-legacy-id=s3d>3.4. Expression profile of miRNAs in P-deficient and P-sufficient shoots</h3>
<p>To identify barley miRNAs, all non-chloroplast reads from P-deficient and P-sufficient shoots were aligned to known barley miRNAs from miRBase. Nineteen barley miRNAs were mapped in P-sufficient shoots and 18 in P-deficient shoots (Table <a reveal-id="DSS037TB2" data-open="DSS037TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The extra miRNA in P-sufficient shoots is miR444a that is a low-copy miRNA. It is unclear if this miRNA is a <em>bona fide</em> miRNA or resulted from a sequencing error. Three barley miRNAs, hvu-miR399, hvu-miR1126 and hvu-miR5051, were significantly up-regulated in P-deficient shoots (log<sub>2</sub> &gt; 1) (Table <a reveal-id="DSS037TB2" data-open="DSS037TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The expression pattern of miR399 is consistent with previous reports,<sup><a reveal-id="DSS037C13" data-open="DSS037C13" class="link link-ref link-reveal xref-bibr">13</a></sup> suggesting that these miRNAs may indeed be up-regulated by deficient P. Hvu-miR168 was the most abundant barley miRNA, followed by hvu-miR156 in both P treatments (Table <a reveal-id="DSS037TB2" data-open="DSS037TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The two miRNAs together accounted for 95.3% of the annotated barley miRNAs in P-sufficient shoots and 95.1% in P-deficient shoots. Such highly expressed miRNAs are likely to have important roles in barley. In <em>Arabidopsis</em>, miR168 has been shown to regulate argonaute-1 (AGO1) that is involved in both miRNA biogenesis and mRNA degradation,<sup><a reveal-id="DSS037C3" data-open="DSS037C3" class="link link-ref link-reveal xref-bibr">3</a></sup> whereas miR156 regulates squamosa promoter binding protein-like (SPL) transcription factors that define an endogenous flowering pathway.<sup><a reveal-id="DSS037C35" data-open="DSS037C35" class="link link-ref link-reveal xref-bibr">35</a></sup> However, neither of these two miRNAs was significantly differentially expressed between P-deficient and P-sufficient shoots, suggesting that the expression of these two miRNAs may be independent of P nutritional status. To identify conserved miRNAs, the remaining unmapped reads were aligned to miRNAs from other species in miRBase. One hundred and forty seven miRNAs in P-sufficient shoots and 171 in P-deficient shoots were mapped (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Of these conserved miRNAs, 55 were only present in P-deficient shoots while 32 only existed in P-sufficient shoots. MiR156 was the most abundant among all the conserved miRNAs identified in both P treatments. </p>                        <a id="4600221" scrollto-destination="4600221"></a>
<div content-id="DSS037TB2" class="table-modal"><div class="table-wrap"><div id="DSS037TB2" class="table-wrap-title" data-id="DSS037TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Barley miRNAs identified in P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>hvu-miR168-5p </td><td>511 386 </td><td>533 602 </td><td>406 701.5194 </td><td>−0.330442284 </td></tr><tr><td>hvu-miR156 </td><td>307 838 </td><td>223 346 </td><td>170 230.167 </td><td>−0.854684607 </td></tr><tr><td>hvu-miR166/b/c </td><td>34 661 </td><td>31 173 </td><td>23 759.48078 </td><td>−0.544809969 </td></tr><tr><td>hvu-miR5048 </td><td>2900 </td><td>3146 </td><td>2397.822685 </td><td>−0.274327923 </td></tr><tr><td>hvu-miR159a/b </td><td>2248 </td><td>3011 </td><td>2294.928196 </td><td>0.029806979 </td></tr><tr><td>hvu-miR168-3p </td><td>295 </td><td>327 </td><td>249.2333178 </td><td>−0.243218012 </td></tr><tr><td>hvu-miR1120 </td><td>172 </td><td>198 </td><td>150.9119172 </td><td>−0.188701828 </td></tr><tr><td>hvu-miR5049 </td><td>155 </td><td>134 </td><td>102.1323076 </td><td>−0.601828908 </td></tr><tr><td>hvu-miR171 </td><td>152 </td><td>182 </td><td>138.7170148 </td><td>−0.131926566 </td></tr><tr><td>hvu-miR444b </td><td>73 </td><td>51 </td><td>38.8712514 </td><td>−0.90919291 </td></tr><tr><td>hvu-miR5052 </td><td>51 </td><td>47 </td><td>35.8225258 </td><td>−0.509630184 </td></tr><tr><td>hvu-miR397 </td><td>48 </td><td>46 </td><td>35.0603444 </td><td>−0.453194238 </td></tr><tr><td>hvu-miR1126 </td><td>24 </td><td>64 </td><td>48.7796096 </td><td>1.023243806 </td></tr><tr><td>hvu-miR5050 </td><td>14 </td><td>13 </td><td>9.908358201 </td><td>−0.498708897 </td></tr><tr><td>hvu-miR1436 </td><td>10 </td><td>12 </td><td>9.146176801 </td><td>−0.128759287 </td></tr><tr><td>hvu-miR399 </td><td>6 </td><td>377 </td><td>287.3423878 </td><td>5.581664519 </td></tr><tr><td>hvu-miR5053 </td><td>3 </td><td>2 </td><td>1.5243628 </td><td>−0.976756194 </td></tr><tr><td>hvu-miR5051 </td><td>2 </td><td>8 </td><td>6.0974512 </td><td>1.608206307 </td></tr><tr><td>hvu-miR444a </td><td>1 </td><td>0 </td><td>0 </td><td>#NUM! </td></tr></tbody></table></div><div content-id="DSS037TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>hvu-miR168-5p </td><td>511 386 </td><td>533 602 </td><td>406 701.5194 </td><td>−0.330442284 </td></tr><tr><td>hvu-miR156 </td><td>307 838 </td><td>223 346 </td><td>170 230.167 </td><td>−0.854684607 </td></tr><tr><td>hvu-miR166/b/c </td><td>34 661 </td><td>31 173 </td><td>23 759.48078 </td><td>−0.544809969 </td></tr><tr><td>hvu-miR5048 </td><td>2900 </td><td>3146 </td><td>2397.822685 </td><td>−0.274327923 </td></tr><tr><td>hvu-miR159a/b </td><td>2248 </td><td>3011 </td><td>2294.928196 </td><td>0.029806979 </td></tr><tr><td>hvu-miR168-3p </td><td>295 </td><td>327 </td><td>249.2333178 </td><td>−0.243218012 </td></tr><tr><td>hvu-miR1120 </td><td>172 </td><td>198 </td><td>150.9119172 </td><td>−0.188701828 </td></tr><tr><td>hvu-miR5049 </td><td>155 </td><td>134 </td><td>102.1323076 </td><td>−0.601828908 </td></tr><tr><td>hvu-miR171 </td><td>152 </td><td>182 </td><td>138.7170148 </td><td>−0.131926566 </td></tr><tr><td>hvu-miR444b </td><td>73 </td><td>51 </td><td>38.8712514 </td><td>−0.90919291 </td></tr><tr><td>hvu-miR5052 </td><td>51 </td><td>47 </td><td>35.8225258 </td><td>−0.509630184 </td></tr><tr><td>hvu-miR397 </td><td>48 </td><td>46 </td><td>35.0603444 </td><td>−0.453194238 </td></tr><tr><td>hvu-miR1126 </td><td>24 </td><td>64 </td><td>48.7796096 </td><td>1.023243806 </td></tr><tr><td>hvu-miR5050 </td><td>14 </td><td>13 </td><td>9.908358201 </td><td>−0.498708897 </td></tr><tr><td>hvu-miR1436 </td><td>10 </td><td>12 </td><td>9.146176801 </td><td>−0.128759287 </td></tr><tr><td>hvu-miR399 </td><td>6 </td><td>377 </td><td>287.3423878 </td><td>5.581664519 </td></tr><tr><td>hvu-miR5053 </td><td>3 </td><td>2 </td><td>1.5243628 </td><td>−0.976756194 </td></tr><tr><td>hvu-miR5051 </td><td>2 </td><td>8 </td><td>6.0974512 </td><td>1.608206307 </td></tr><tr><td>hvu-miR444a </td><td>1 </td><td>0 </td><td>0 </td><td>#NUM! </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>‘#NUM’ means that the log<sub>2</sub> cannot be calculated with a read count of 0 in either sample. ‘−P norm’ is a linear normalization to get the total read counts equal in both samples. The same meaning below.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600221" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS037TB2" class="table-wrap-title" data-id="DSS037TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Barley miRNAs identified in P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>hvu-miR168-5p </td><td>511 386 </td><td>533 602 </td><td>406 701.5194 </td><td>−0.330442284 </td></tr><tr><td>hvu-miR156 </td><td>307 838 </td><td>223 346 </td><td>170 230.167 </td><td>−0.854684607 </td></tr><tr><td>hvu-miR166/b/c </td><td>34 661 </td><td>31 173 </td><td>23 759.48078 </td><td>−0.544809969 </td></tr><tr><td>hvu-miR5048 </td><td>2900 </td><td>3146 </td><td>2397.822685 </td><td>−0.274327923 </td></tr><tr><td>hvu-miR159a/b </td><td>2248 </td><td>3011 </td><td>2294.928196 </td><td>0.029806979 </td></tr><tr><td>hvu-miR168-3p </td><td>295 </td><td>327 </td><td>249.2333178 </td><td>−0.243218012 </td></tr><tr><td>hvu-miR1120 </td><td>172 </td><td>198 </td><td>150.9119172 </td><td>−0.188701828 </td></tr><tr><td>hvu-miR5049 </td><td>155 </td><td>134 </td><td>102.1323076 </td><td>−0.601828908 </td></tr><tr><td>hvu-miR171 </td><td>152 </td><td>182 </td><td>138.7170148 </td><td>−0.131926566 </td></tr><tr><td>hvu-miR444b </td><td>73 </td><td>51 </td><td>38.8712514 </td><td>−0.90919291 </td></tr><tr><td>hvu-miR5052 </td><td>51 </td><td>47 </td><td>35.8225258 </td><td>−0.509630184 </td></tr><tr><td>hvu-miR397 </td><td>48 </td><td>46 </td><td>35.0603444 </td><td>−0.453194238 </td></tr><tr><td>hvu-miR1126 </td><td>24 </td><td>64 </td><td>48.7796096 </td><td>1.023243806 </td></tr><tr><td>hvu-miR5050 </td><td>14 </td><td>13 </td><td>9.908358201 </td><td>−0.498708897 </td></tr><tr><td>hvu-miR1436 </td><td>10 </td><td>12 </td><td>9.146176801 </td><td>−0.128759287 </td></tr><tr><td>hvu-miR399 </td><td>6 </td><td>377 </td><td>287.3423878 </td><td>5.581664519 </td></tr><tr><td>hvu-miR5053 </td><td>3 </td><td>2 </td><td>1.5243628 </td><td>−0.976756194 </td></tr><tr><td>hvu-miR5051 </td><td>2 </td><td>8 </td><td>6.0974512 </td><td>1.608206307 </td></tr><tr><td>hvu-miR444a </td><td>1 </td><td>0 </td><td>0 </td><td>#NUM! </td></tr></tbody></table></div><div content-id="DSS037TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>hvu-miR168-5p </td><td>511 386 </td><td>533 602 </td><td>406 701.5194 </td><td>−0.330442284 </td></tr><tr><td>hvu-miR156 </td><td>307 838 </td><td>223 346 </td><td>170 230.167 </td><td>−0.854684607 </td></tr><tr><td>hvu-miR166/b/c </td><td>34 661 </td><td>31 173 </td><td>23 759.48078 </td><td>−0.544809969 </td></tr><tr><td>hvu-miR5048 </td><td>2900 </td><td>3146 </td><td>2397.822685 </td><td>−0.274327923 </td></tr><tr><td>hvu-miR159a/b </td><td>2248 </td><td>3011 </td><td>2294.928196 </td><td>0.029806979 </td></tr><tr><td>hvu-miR168-3p </td><td>295 </td><td>327 </td><td>249.2333178 </td><td>−0.243218012 </td></tr><tr><td>hvu-miR1120 </td><td>172 </td><td>198 </td><td>150.9119172 </td><td>−0.188701828 </td></tr><tr><td>hvu-miR5049 </td><td>155 </td><td>134 </td><td>102.1323076 </td><td>−0.601828908 </td></tr><tr><td>hvu-miR171 </td><td>152 </td><td>182 </td><td>138.7170148 </td><td>−0.131926566 </td></tr><tr><td>hvu-miR444b </td><td>73 </td><td>51 </td><td>38.8712514 </td><td>−0.90919291 </td></tr><tr><td>hvu-miR5052 </td><td>51 </td><td>47 </td><td>35.8225258 </td><td>−0.509630184 </td></tr><tr><td>hvu-miR397 </td><td>48 </td><td>46 </td><td>35.0603444 </td><td>−0.453194238 </td></tr><tr><td>hvu-miR1126 </td><td>24 </td><td>64 </td><td>48.7796096 </td><td>1.023243806 </td></tr><tr><td>hvu-miR5050 </td><td>14 </td><td>13 </td><td>9.908358201 </td><td>−0.498708897 </td></tr><tr><td>hvu-miR1436 </td><td>10 </td><td>12 </td><td>9.146176801 </td><td>−0.128759287 </td></tr><tr><td>hvu-miR399 </td><td>6 </td><td>377 </td><td>287.3423878 </td><td>5.581664519 </td></tr><tr><td>hvu-miR5053 </td><td>3 </td><td>2 </td><td>1.5243628 </td><td>−0.976756194 </td></tr><tr><td>hvu-miR5051 </td><td>2 </td><td>8 </td><td>6.0974512 </td><td>1.608206307 </td></tr><tr><td>hvu-miR444a </td><td>1 </td><td>0 </td><td>0 </td><td>#NUM! </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>‘#NUM’ means that the log<sub>2</sub> cannot be calculated with a read count of 0 in either sample. ‘−P norm’ is a linear normalization to get the total read counts equal in both samples. The same meaning below.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600221" target="_blank">View Large</a></div></div><p>Seven conserved miRNAs were up-regulated, while 19 were down-regulated (|log<sub>2</sub>| &gt; 1) in P-deficient shoots when a threshold was ≥10 (Table <a reveal-id="DSS037TB3" data-open="DSS037TB3" class="link link-reveal link-table xref-fig">3</a>). MiR399 was only detected in P-deficient shoots, but not in P-sufficient shoots (Table <a reveal-id="DSS037TB3" data-open="DSS037TB3" class="link link-reveal link-table xref-fig">3</a>). MiR827 was also significantly up-regulated, being 8 times more abundant in P-deficient shoots than in P-sufficient shoots, which is consistent with previous studies.<sup><a reveal-id="DSS037C12" data-open="DSS037C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS037C36" data-open="DSS037C36" class="link link-ref link-reveal xref-bibr">36</a></sup> In contrast to miR399, miR827 was much more abundant in both P treatments (Table <a reveal-id="DSS037TB3" data-open="DSS037TB3" class="link link-reveal link-table xref-fig">3</a>). In addition, miR827 has an additional role in drought tolerance (M. Aukerman and W. Park, unpublished data). These suggest that the two P-responsive miRNAs might be regulated by different mechanisms. Surprisingly, one miRNA, dme-miR-1–3p, from <em>Drosophila</em> was detected and significantly up-regulated in P-deficient shoots (Table <a reveal-id="DSS037TB3" data-open="DSS037TB3" class="link link-reveal link-table xref-fig">3</a>). MiRNAs conserved between plants and animals have been reported previously.<sup><a reveal-id="DSS037C37" data-open="DSS037C37" class="link link-ref link-reveal xref-bibr">37</a></sup> If this result is true, then dme-miR-1–3p could be used as a phylogenetic marker to help unravel the evolutionary history and relationships between plants and insects. </p>                        <a id="4600228" scrollto-destination="4600228"></a>
<div content-id="DSS037TB3" class="table-modal"><div class="table-wrap"><div id="DSS037TB3" class="table-wrap-title" data-id="DSS037TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Differentially expressed miRNAs between P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>zma-miR156k/sbi-miR156e </td><td>1334 </td><td>508 </td><td>387.18815 </td><td>−1.78465 </td></tr><tr><td>osa-miR827a/b/zma-miR827/bdi-miR827/ssp-miR827 </td><td>1152 </td><td>9344 </td><td>7121.823 </td><td>2.628106 </td></tr><tr><td>ptc-miR156k/vvi-miR156a </td><td>710 </td><td>455 </td><td>346.79254 </td><td>−1.03375 </td></tr><tr><td>bdi-miR408 </td><td>247 </td><td>85 </td><td>64.785419 </td><td>−1.93077 </td></tr><tr><td>rgl-miR5139 </td><td>155 </td><td>100 </td><td>76.21814 </td><td>−1.02406 </td></tr><tr><td>vvi-miR156e </td><td>100 </td><td>64 </td><td>48.77961 </td><td>−1.03565 </td></tr><tr><td>ath-miR165a/b </td><td>85 </td><td>22 </td><td>16.767991 </td><td>−2.34175 </td></tr><tr><td>mtr-miR156j </td><td>83 </td><td>48 </td><td>36.584707 </td><td>−1.18187 </td></tr><tr><td>vvi-miR167c </td><td>64 </td><td>34 </td><td>25.914168 </td><td>−1.30433 </td></tr><tr><td>bdi-miR166e </td><td>58 </td><td>34 </td><td>25.914168 </td><td>−1.16231 </td></tr><tr><td>zma-miR396a*/b* </td><td>51 </td><td>21 </td><td>16.005809 </td><td>−1.6719 </td></tr><tr><td>ath-miR399b/c/d/sbi-miR399d/i/zma-miR399e/i/j/mtr-miR399l/p/vvi-miR399b/c/pvu-miR399a/rco-miR399a/aly-miR399b/c/csi-miR399d/tcc-miR399i/vun-miR399a </td><td>46 </td><td>147 </td><td>112.04067 </td><td>1.284317 </td></tr><tr><td>osa-miR5072 </td><td>43 </td><td>21 </td><td>16.005809 </td><td>−1.42574 </td></tr><tr><td>csi-miR172b </td><td>28 </td><td>17 </td><td>12.957084 </td><td>−1.11169 </td></tr><tr><td>ghr-miR156c </td><td>21 </td><td>13 </td><td>9.9083582 </td><td>−1.08367 </td></tr><tr><td>zma-miR169i*/k*/j* </td><td>20 </td><td>13 </td><td>9.9083582 </td><td>−1.01328 </td></tr><tr><td>zma-miR156l* </td><td>19 </td><td>8 </td><td>6.0974512 </td><td>−1.63972 </td></tr><tr><td>bta-miR-2487 </td><td>16 </td><td>4 </td><td>3.0487256 </td><td>−2.39179 </td></tr><tr><td>ptc-miR164f/mtr-miR164d/vvi-miR164b/rco-miR164d/tcc-miR164c </td><td>15 </td><td>9 </td><td>6.8596326 </td><td>−1.12876 </td></tr><tr><td>osa-miR399a/b/c/sbi-miR399a/c/h/j/zma-miR399a/c/ h/ptc-miR399f/g/vvi-miR399a/h/csi-miR399e/c/tcc-miR399g/bdi-miR399b/vun-miR399b </td><td>11 </td><td>148 </td><td>112.80285 </td><td>3.358228 </td></tr><tr><td>osa-miR166e </td><td>11 </td><td>4 </td><td>3.0487256 </td><td>−1.85123 </td></tr><tr><td>zma-miR166b*/d* </td><td>10 </td><td>5 </td><td>3.810907 </td><td>−1.39179 </td></tr><tr><td>dme-miR-1-3p/dps-miR-1/ame-miR-1/aga-miR-1/bmo-miR-1a/tca-miR-1-3p/dan-miR-1/der-miR-1/dgr-miR-1/dmo-miR-1/dpe-miR-1/dse-miR-1/dsi-miR-1/dvi-miR-1/dwi-miR-1/dya-miR-1/dpu-miR-1/isc-miR-1/aae-miR-1/cqu-miR-1/api-miR-1/nvi-miR-1/ngi-miR-1 </td><td>7 </td><td>266 </td><td>202.74025 </td><td>4.856134 </td></tr><tr><td>ame-miR-3772 </td><td>4 </td><td>28 </td><td>21.341079 </td><td>2.415561 </td></tr><tr><td>tae-miR1126 </td><td>2 </td><td>12 </td><td>9.1461768 </td><td>2.193169 </td></tr><tr><td>zma-miR399h* </td><td>0 </td><td>145 </td><td>110.5163 </td><td>#NUM </td></tr></tbody></table></div><div content-id="DSS037TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>zma-miR156k/sbi-miR156e </td><td>1334 </td><td>508 </td><td>387.18815 </td><td>−1.78465 </td></tr><tr><td>osa-miR827a/b/zma-miR827/bdi-miR827/ssp-miR827 </td><td>1152 </td><td>9344 </td><td>7121.823 </td><td>2.628106 </td></tr><tr><td>ptc-miR156k/vvi-miR156a </td><td>710 </td><td>455 </td><td>346.79254 </td><td>−1.03375 </td></tr><tr><td>bdi-miR408 </td><td>247 </td><td>85 </td><td>64.785419 </td><td>−1.93077 </td></tr><tr><td>rgl-miR5139 </td><td>155 </td><td>100 </td><td>76.21814 </td><td>−1.02406 </td></tr><tr><td>vvi-miR156e </td><td>100 </td><td>64 </td><td>48.77961 </td><td>−1.03565 </td></tr><tr><td>ath-miR165a/b </td><td>85 </td><td>22 </td><td>16.767991 </td><td>−2.34175 </td></tr><tr><td>mtr-miR156j </td><td>83 </td><td>48 </td><td>36.584707 </td><td>−1.18187 </td></tr><tr><td>vvi-miR167c </td><td>64 </td><td>34 </td><td>25.914168 </td><td>−1.30433 </td></tr><tr><td>bdi-miR166e </td><td>58 </td><td>34 </td><td>25.914168 </td><td>−1.16231 </td></tr><tr><td>zma-miR396a*/b* </td><td>51 </td><td>21 </td><td>16.005809 </td><td>−1.6719 </td></tr><tr><td>ath-miR399b/c/d/sbi-miR399d/i/zma-miR399e/i/j/mtr-miR399l/p/vvi-miR399b/c/pvu-miR399a/rco-miR399a/aly-miR399b/c/csi-miR399d/tcc-miR399i/vun-miR399a </td><td>46 </td><td>147 </td><td>112.04067 </td><td>1.284317 </td></tr><tr><td>osa-miR5072 </td><td>43 </td><td>21 </td><td>16.005809 </td><td>−1.42574 </td></tr><tr><td>csi-miR172b </td><td>28 </td><td>17 </td><td>12.957084 </td><td>−1.11169 </td></tr><tr><td>ghr-miR156c </td><td>21 </td><td>13 </td><td>9.9083582 </td><td>−1.08367 </td></tr><tr><td>zma-miR169i*/k*/j* </td><td>20 </td><td>13 </td><td>9.9083582 </td><td>−1.01328 </td></tr><tr><td>zma-miR156l* </td><td>19 </td><td>8 </td><td>6.0974512 </td><td>−1.63972 </td></tr><tr><td>bta-miR-2487 </td><td>16 </td><td>4 </td><td>3.0487256 </td><td>−2.39179 </td></tr><tr><td>ptc-miR164f/mtr-miR164d/vvi-miR164b/rco-miR164d/tcc-miR164c </td><td>15 </td><td>9 </td><td>6.8596326 </td><td>−1.12876 </td></tr><tr><td>osa-miR399a/b/c/sbi-miR399a/c/h/j/zma-miR399a/c/ h/ptc-miR399f/g/vvi-miR399a/h/csi-miR399e/c/tcc-miR399g/bdi-miR399b/vun-miR399b </td><td>11 </td><td>148 </td><td>112.80285 </td><td>3.358228 </td></tr><tr><td>osa-miR166e </td><td>11 </td><td>4 </td><td>3.0487256 </td><td>−1.85123 </td></tr><tr><td>zma-miR166b*/d* </td><td>10 </td><td>5 </td><td>3.810907 </td><td>−1.39179 </td></tr><tr><td>dme-miR-1-3p/dps-miR-1/ame-miR-1/aga-miR-1/bmo-miR-1a/tca-miR-1-3p/dan-miR-1/der-miR-1/dgr-miR-1/dmo-miR-1/dpe-miR-1/dse-miR-1/dsi-miR-1/dvi-miR-1/dwi-miR-1/dya-miR-1/dpu-miR-1/isc-miR-1/aae-miR-1/cqu-miR-1/api-miR-1/nvi-miR-1/ngi-miR-1 </td><td>7 </td><td>266 </td><td>202.74025 </td><td>4.856134 </td></tr><tr><td>ame-miR-3772 </td><td>4 </td><td>28 </td><td>21.341079 </td><td>2.415561 </td></tr><tr><td>tae-miR1126 </td><td>2 </td><td>12 </td><td>9.1461768 </td><td>2.193169 </td></tr><tr><td>zma-miR399h* </td><td>0 </td><td>145 </td><td>110.5163 </td><td>#NUM </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Threshold ≥10 is applied. ‘−’ indicates that miRNAs are down-regulated, whereas the others are up-regulated.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600228" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS037TB3" class="table-wrap-title" data-id="DSS037TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Differentially expressed miRNAs between P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>zma-miR156k/sbi-miR156e </td><td>1334 </td><td>508 </td><td>387.18815 </td><td>−1.78465 </td></tr><tr><td>osa-miR827a/b/zma-miR827/bdi-miR827/ssp-miR827 </td><td>1152 </td><td>9344 </td><td>7121.823 </td><td>2.628106 </td></tr><tr><td>ptc-miR156k/vvi-miR156a </td><td>710 </td><td>455 </td><td>346.79254 </td><td>−1.03375 </td></tr><tr><td>bdi-miR408 </td><td>247 </td><td>85 </td><td>64.785419 </td><td>−1.93077 </td></tr><tr><td>rgl-miR5139 </td><td>155 </td><td>100 </td><td>76.21814 </td><td>−1.02406 </td></tr><tr><td>vvi-miR156e </td><td>100 </td><td>64 </td><td>48.77961 </td><td>−1.03565 </td></tr><tr><td>ath-miR165a/b </td><td>85 </td><td>22 </td><td>16.767991 </td><td>−2.34175 </td></tr><tr><td>mtr-miR156j </td><td>83 </td><td>48 </td><td>36.584707 </td><td>−1.18187 </td></tr><tr><td>vvi-miR167c </td><td>64 </td><td>34 </td><td>25.914168 </td><td>−1.30433 </td></tr><tr><td>bdi-miR166e </td><td>58 </td><td>34 </td><td>25.914168 </td><td>−1.16231 </td></tr><tr><td>zma-miR396a*/b* </td><td>51 </td><td>21 </td><td>16.005809 </td><td>−1.6719 </td></tr><tr><td>ath-miR399b/c/d/sbi-miR399d/i/zma-miR399e/i/j/mtr-miR399l/p/vvi-miR399b/c/pvu-miR399a/rco-miR399a/aly-miR399b/c/csi-miR399d/tcc-miR399i/vun-miR399a </td><td>46 </td><td>147 </td><td>112.04067 </td><td>1.284317 </td></tr><tr><td>osa-miR5072 </td><td>43 </td><td>21 </td><td>16.005809 </td><td>−1.42574 </td></tr><tr><td>csi-miR172b </td><td>28 </td><td>17 </td><td>12.957084 </td><td>−1.11169 </td></tr><tr><td>ghr-miR156c </td><td>21 </td><td>13 </td><td>9.9083582 </td><td>−1.08367 </td></tr><tr><td>zma-miR169i*/k*/j* </td><td>20 </td><td>13 </td><td>9.9083582 </td><td>−1.01328 </td></tr><tr><td>zma-miR156l* </td><td>19 </td><td>8 </td><td>6.0974512 </td><td>−1.63972 </td></tr><tr><td>bta-miR-2487 </td><td>16 </td><td>4 </td><td>3.0487256 </td><td>−2.39179 </td></tr><tr><td>ptc-miR164f/mtr-miR164d/vvi-miR164b/rco-miR164d/tcc-miR164c </td><td>15 </td><td>9 </td><td>6.8596326 </td><td>−1.12876 </td></tr><tr><td>osa-miR399a/b/c/sbi-miR399a/c/h/j/zma-miR399a/c/ h/ptc-miR399f/g/vvi-miR399a/h/csi-miR399e/c/tcc-miR399g/bdi-miR399b/vun-miR399b </td><td>11 </td><td>148 </td><td>112.80285 </td><td>3.358228 </td></tr><tr><td>osa-miR166e </td><td>11 </td><td>4 </td><td>3.0487256 </td><td>−1.85123 </td></tr><tr><td>zma-miR166b*/d* </td><td>10 </td><td>5 </td><td>3.810907 </td><td>−1.39179 </td></tr><tr><td>dme-miR-1-3p/dps-miR-1/ame-miR-1/aga-miR-1/bmo-miR-1a/tca-miR-1-3p/dan-miR-1/der-miR-1/dgr-miR-1/dmo-miR-1/dpe-miR-1/dse-miR-1/dsi-miR-1/dvi-miR-1/dwi-miR-1/dya-miR-1/dpu-miR-1/isc-miR-1/aae-miR-1/cqu-miR-1/api-miR-1/nvi-miR-1/ngi-miR-1 </td><td>7 </td><td>266 </td><td>202.74025 </td><td>4.856134 </td></tr><tr><td>ame-miR-3772 </td><td>4 </td><td>28 </td><td>21.341079 </td><td>2.415561 </td></tr><tr><td>tae-miR1126 </td><td>2 </td><td>12 </td><td>9.1461768 </td><td>2.193169 </td></tr><tr><td>zma-miR399h* </td><td>0 </td><td>145 </td><td>110.5163 </td><td>#NUM </td></tr></tbody></table></div><div content-id="DSS037TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>zma-miR156k/sbi-miR156e </td><td>1334 </td><td>508 </td><td>387.18815 </td><td>−1.78465 </td></tr><tr><td>osa-miR827a/b/zma-miR827/bdi-miR827/ssp-miR827 </td><td>1152 </td><td>9344 </td><td>7121.823 </td><td>2.628106 </td></tr><tr><td>ptc-miR156k/vvi-miR156a </td><td>710 </td><td>455 </td><td>346.79254 </td><td>−1.03375 </td></tr><tr><td>bdi-miR408 </td><td>247 </td><td>85 </td><td>64.785419 </td><td>−1.93077 </td></tr><tr><td>rgl-miR5139 </td><td>155 </td><td>100 </td><td>76.21814 </td><td>−1.02406 </td></tr><tr><td>vvi-miR156e </td><td>100 </td><td>64 </td><td>48.77961 </td><td>−1.03565 </td></tr><tr><td>ath-miR165a/b </td><td>85 </td><td>22 </td><td>16.767991 </td><td>−2.34175 </td></tr><tr><td>mtr-miR156j </td><td>83 </td><td>48 </td><td>36.584707 </td><td>−1.18187 </td></tr><tr><td>vvi-miR167c </td><td>64 </td><td>34 </td><td>25.914168 </td><td>−1.30433 </td></tr><tr><td>bdi-miR166e </td><td>58 </td><td>34 </td><td>25.914168 </td><td>−1.16231 </td></tr><tr><td>zma-miR396a*/b* </td><td>51 </td><td>21 </td><td>16.005809 </td><td>−1.6719 </td></tr><tr><td>ath-miR399b/c/d/sbi-miR399d/i/zma-miR399e/i/j/mtr-miR399l/p/vvi-miR399b/c/pvu-miR399a/rco-miR399a/aly-miR399b/c/csi-miR399d/tcc-miR399i/vun-miR399a </td><td>46 </td><td>147 </td><td>112.04067 </td><td>1.284317 </td></tr><tr><td>osa-miR5072 </td><td>43 </td><td>21 </td><td>16.005809 </td><td>−1.42574 </td></tr><tr><td>csi-miR172b </td><td>28 </td><td>17 </td><td>12.957084 </td><td>−1.11169 </td></tr><tr><td>ghr-miR156c </td><td>21 </td><td>13 </td><td>9.9083582 </td><td>−1.08367 </td></tr><tr><td>zma-miR169i*/k*/j* </td><td>20 </td><td>13 </td><td>9.9083582 </td><td>−1.01328 </td></tr><tr><td>zma-miR156l* </td><td>19 </td><td>8 </td><td>6.0974512 </td><td>−1.63972 </td></tr><tr><td>bta-miR-2487 </td><td>16 </td><td>4 </td><td>3.0487256 </td><td>−2.39179 </td></tr><tr><td>ptc-miR164f/mtr-miR164d/vvi-miR164b/rco-miR164d/tcc-miR164c </td><td>15 </td><td>9 </td><td>6.8596326 </td><td>−1.12876 </td></tr><tr><td>osa-miR399a/b/c/sbi-miR399a/c/h/j/zma-miR399a/c/ h/ptc-miR399f/g/vvi-miR399a/h/csi-miR399e/c/tcc-miR399g/bdi-miR399b/vun-miR399b </td><td>11 </td><td>148 </td><td>112.80285 </td><td>3.358228 </td></tr><tr><td>osa-miR166e </td><td>11 </td><td>4 </td><td>3.0487256 </td><td>−1.85123 </td></tr><tr><td>zma-miR166b*/d* </td><td>10 </td><td>5 </td><td>3.810907 </td><td>−1.39179 </td></tr><tr><td>dme-miR-1-3p/dps-miR-1/ame-miR-1/aga-miR-1/bmo-miR-1a/tca-miR-1-3p/dan-miR-1/der-miR-1/dgr-miR-1/dmo-miR-1/dpe-miR-1/dse-miR-1/dsi-miR-1/dvi-miR-1/dwi-miR-1/dya-miR-1/dpu-miR-1/isc-miR-1/aae-miR-1/cqu-miR-1/api-miR-1/nvi-miR-1/ngi-miR-1 </td><td>7 </td><td>266 </td><td>202.74025 </td><td>4.856134 </td></tr><tr><td>ame-miR-3772 </td><td>4 </td><td>28 </td><td>21.341079 </td><td>2.415561 </td></tr><tr><td>tae-miR1126 </td><td>2 </td><td>12 </td><td>9.1461768 </td><td>2.193169 </td></tr><tr><td>zma-miR399h* </td><td>0 </td><td>145 </td><td>110.5163 </td><td>#NUM </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Threshold ≥10 is applied. ‘−’ indicates that miRNAs are down-regulated, whereas the others are up-regulated.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600228" target="_blank">View Large</a></div></div><p>Previous studies showed that miR778 and miR2111 are up-regulated under P limitation in <em>Arabidopsis</em>.<sup><a reveal-id="DSS037C12" data-open="DSS037C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, these two miRNAs were not present in barley shoots, or in rice,<sup><a reveal-id="DSS037C12" data-open="DSS037C12" class="link link-ref link-reveal xref-bibr">12</a></sup> suggesting that they may be dicot specific. MiR398 was not present in barley shoots either, but it is present and down-regulated under P limitation in <em>Arabidopsis</em>.<sup><a reveal-id="DSS037C12" data-open="DSS037C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In addition, unlike in <em>Arabidopsis</em>,<sup><a reveal-id="DSS037C12" data-open="DSS037C12" class="link link-ref link-reveal xref-bibr">12</a></sup> miR169 and miR395 were not down-regulated under P limitation in barley shoots (Table <a reveal-id="DSS037TB3" data-open="DSS037TB3" class="link link-reveal link-table xref-fig">3</a>). These data indicate that miRNAs have different responses to P in different species.</p><p>To confirm the expression of conserved miRNAs, Northern hybridization was performed using probes reversely complementary to hvu-miR156, hvu-miR399, osa-miR528, bdi-miR827 and bdi-miR408. These miRNAs were selected because they were relatively abundant. Hvu-miR399 and bdi-miR408 were hardly detectable, but the three other miRNAs were detected in both P-deficient and P-sufficient shoots (Fig. <a reveal-id="DSS037F3" data-open="DSS037F3" class="link link-reveal link-table xref-fig">3</a>). Bdi-miR827 was expressed at a higher level in P-deficient shoots than in P-sufficient shoots, whereas osa-miR528 expression was slightly reduced under P deficiency (Fig. <a reveal-id="DSS037F3" data-open="DSS037F3" class="link link-reveal link-table xref-fig">3</a>). Both miRNAs were expressed at higher levels in leaves than in roots (Fig. <a reveal-id="DSS037F3" data-open="DSS037F3" class="link link-reveal link-table xref-fig">3</a>). The expression level of hvu-miR156 did not differ between P-deficient and P-sufficient leaves, but was higher in P-deficient roots than in P-sufficient roots (Fig. <a reveal-id="DSS037F3" data-open="DSS037F3" class="link link-reveal link-table xref-fig">3</a>).The expression levels of these miRNAs detected by the Northern hybridization were in agreement with the read abundances of the same miRNAs identified from the deep sequencing. </p>                        <a id="4600238" scrollto-destination="4600238"></a>
<div id="DSS037F3" data-id="DSS037F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03703.jpeg?Expires=2147483647&Signature=BFpEq4vqEbdnaQ5Om9MkNH4zJk4psig7rX6cX1mr2KUBVotqs-Py8sv6jgf2K2JhnbAs837w1IVYYDf0AVkhbA2gZ9kUXT8hnDTtgG1nY0d~wRirEcMLhdJmsuXVDct9P61o7OJ7KJn3kFf~6M4N6SEBybZ6nr3ELfuSuS-hBnAIz7ZSiEPuRkDgeIEaN5V7NDwWjv4QDeLVdHu3SO3heEgkfXuHRyv7RvBS5g7ZB0ZGlEas07oFsGfBzqRop89QIVTUuRqDEoYAyeK-nC~8IpwrnbUY-M5ISt9lfwtmItpxrG6EBHYobEcjvfWK0hxjBNa~8PKpHTSGsh1JfkDnMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Northern blot hybridization of miRNA expression in P-deficient and P-sufficient shoots of Pallas barley. Total RNA extracted from leaf and root tissues from P-deficient (−P) and P-sufficient (+P) barley was hybridized with probes reversely complementary to indicated known and novel miRNAs. Hybridization with U6 probe was used as a loading control. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600238/dss03703.jpeg" path-from-xml="dss03703.jpeg" target="_blank">View large</a><a section="4600238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03703.jpeg?Expires=2147483647&Signature=LPpHVzgKkAyZFPopb2D8Wnj7C7UptNsR7GCefIeJBwjchIQicz9AKMCF70Axid6KwY31QkrgviFUej2e176z968oKkRIIjb1~si2eFB3l380AvayXvoEHbf6stBFDb0n7eeDBEOI3kXoYva9BN7lQcdVqnqkba6a2uHNv3CQ9wtjH5AXSG-OiLD5QAxvR0zzrdAanA-I0bAntHkJbR8DmerYiFgzuCdFBq-G1wjcCGvUsh4Zn8wuekHRinsNW~SjWBEeTkUmhFH3NGEqyAC9sXCsR-wyApzPGJmr~d7clQpeV72Oh-Lh3HRsjEjQNvIF0MMpfdZdPkFH7BrPgua8lA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600238&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Northern blot hybridization of miRNA expression in P-deficient and P-sufficient shoots of Pallas barley. Total RNA extracted from leaf and root tissues from P-deficient (−P) and P-sufficient (+P) barley was hybridized with probes reversely complementary to indicated known and novel miRNAs. Hybridization with U6 probe was used as a loading control. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS037F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03703.jpeg?Expires=2147483647&Signature=BFpEq4vqEbdnaQ5Om9MkNH4zJk4psig7rX6cX1mr2KUBVotqs-Py8sv6jgf2K2JhnbAs837w1IVYYDf0AVkhbA2gZ9kUXT8hnDTtgG1nY0d~wRirEcMLhdJmsuXVDct9P61o7OJ7KJn3kFf~6M4N6SEBybZ6nr3ELfuSuS-hBnAIz7ZSiEPuRkDgeIEaN5V7NDwWjv4QDeLVdHu3SO3heEgkfXuHRyv7RvBS5g7ZB0ZGlEas07oFsGfBzqRop89QIVTUuRqDEoYAyeK-nC~8IpwrnbUY-M5ISt9lfwtmItpxrG6EBHYobEcjvfWK0hxjBNa~8PKpHTSGsh1JfkDnMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Northern blot hybridization of miRNA expression in P-deficient and P-sufficient shoots of Pallas barley. Total RNA extracted from leaf and root tissues from P-deficient (−P) and P-sufficient (+P) barley was hybridized with probes reversely complementary to indicated known and novel miRNAs. Hybridization with U6 probe was used as a loading control. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600238/dss03703.jpeg" path-from-xml="dss03703.jpeg" target="_blank">View large</a><a section="4600238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03703.jpeg?Expires=2147483647&Signature=LPpHVzgKkAyZFPopb2D8Wnj7C7UptNsR7GCefIeJBwjchIQicz9AKMCF70Axid6KwY31QkrgviFUej2e176z968oKkRIIjb1~si2eFB3l380AvayXvoEHbf6stBFDb0n7eeDBEOI3kXoYva9BN7lQcdVqnqkba6a2uHNv3CQ9wtjH5AXSG-OiLD5QAxvR0zzrdAanA-I0bAntHkJbR8DmerYiFgzuCdFBq-G1wjcCGvUsh4Zn8wuekHRinsNW~SjWBEeTkUmhFH3NGEqyAC9sXCsR-wyApzPGJmr~d7clQpeV72Oh-Lh3HRsjEjQNvIF0MMpfdZdPkFH7BrPgua8lA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600238&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Northern blot hybridization of miRNA expression in P-deficient and P-sufficient shoots of Pallas barley. Total RNA extracted from leaf and root tissues from P-deficient (−P) and P-sufficient (+P) barley was hybridized with probes reversely complementary to indicated known and novel miRNAs. Hybridization with U6 probe was used as a loading control. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div>                            <h3 scrollto-destination=4600247 id="4600247" class="section-title" data-legacy-id=s3e>3.5. Predicted novel miRNAs in P-deficient and P-sufficient shoots</h3>
<p>To identify novel miRNAs, we only selected miRNA precursor sequences with secondary structures having higher negative minimal free energies and minimal free energy index (MFEI) than other different type of RNAs and MFEI of greater than 0.85.<sup><a reveal-id="DSS037C38" data-open="DSS037C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Then, we aligned all reads to these selected miRNA precursor sequences. If the reads displayed one or two peaks without appearance in the intervening region, these reads would be taken on board as novel miRNA candidates for further analyses. This is based on the alignment patterns of known miRNAs from miRBase (data not shown). These two typical patterns can be explained by the stability of duplexes or the stability of miRNAs that are incorporated into the RNA-induced silencing complex (RISC). Accordingly, we identified 12 novel miRNA candidates present in both P-deficient and P-sufficient shoots (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>). One miRNA (hvu-miRX27) was found to exist in two mature sequences, but differing only by 1 nt extension at the ends (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>). It is unclear which mature sequence is a true miRNA sequence. </p>                        <a id="4600257" scrollto-destination="4600257"></a>
<div content-id="DSS037TB4" class="table-modal"><div class="table-wrap"><div id="DSS037TB4" class="table-wrap-title" data-id="DSS037TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Predicted novel miRNAs in P-deficient and P-sufficient shoots of Pallas barley by miRDeep</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>miRNA </th><th>Count –P </th><th>Count +P </th><th>Precursor miRNA </th><th>Target Accession </th><th>Action </th><th>Function </th></tr></thead><tbody><tr><td>hvu-miRX21 </td><td>3512 </td><td>3279 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03706.gif?Expires=2147483647&amp;Signature=uys9GEJPW13OXEw-uL6Z~H8oeHUcd1VlayHvcN3BLKDlJANTm~mI2ys3OEQuzYWtMqV4AnbgA0HHaoCaosWWMu1l5Ik~l9rhWkZHpQTccyg9xxZuxrb4UsktS61plc0SGWDd5pzPfN7E8BZmmUHB-6XdNW5fIG1jA6qvF503DONjlPWVpBF1ZEHTDWngneLew1czsgxvslK-W~e2wiSiNtCjNJfAed5~fVG~Qa0ZmYz4SLeq7TAXs4oYSGLGa3-e8rrWef37wvv1W-Jz97uFynm3yOmA1SHjYzT7F-goMtoqcmAtPFKLWYhLK7nHuF3jAlyviTUd8CNCMyigA4iN4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03706.gif" /> </td><td>TC207745 </td><td>Cleavage </td><td>E3 ubiquitin-protein ligase </td></tr><tr><td>hvu-miRX22 </td><td>3494 </td><td>2015 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03707.gif?Expires=2147483647&amp;Signature=0JR2rriAmRnGhIDR-BRjORZW8sUq74tYQIHrhXgOzWQnPN40lv8kS81rQ6c7o4z4~lHGwJLLSKpzwvMMBONF4JPijmVFig5fl5VPRpf-1ZC-lz8dOI2Ly5g3zwNOQn6ikJoL9dXhHRi5XDOA5UF4S38mN5HblYGSNTeGxPlcW3VaI7j~L6eYarpKtq8B7Ut-RKd-BY2YBq1UZxCRUBp2~IpBRNHkVJ4gLM2dEP5VrcGcgo0HmQ~wDoV7eRw9nxZkQq77Ee3A-jV3Od~UiN-RPDIa9sq0KUhGITkS0rrse6W9p42Nc6LScGaRKRpMgKlFLgX0eFMJMSZd4uX5atCiZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03707.gif" /> </td><td>TC236060 </td><td>Cleavage </td><td>unknown </td></tr><tr><td>hvu-miRX23 </td><td>85 </td><td>69 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03708.gif?Expires=2147483647&amp;Signature=WH-W~2HdPLWJhgK0zv5fdUVzbE1lVTwT9wZlayqht9XBY9x-GdlHFldf9jNPXPTqjRMtz-RK0dTYHeKztkGPs7yJm3fzYYv28Ma00Pb7X56wrpJrxJPZDOe3fcXorMumQyNQF0YuRi70DQZgRdqWLs0Xh3G7PXObShkS9-KfH0gXSVvEjxpwfmjxYV0v5QZJcRyS0PKova6LMMkwjC-gRq2gqjy08TzaYuegiCwumMbH6StEq8P7fTrSVMjdoB2qDfj9fP~8HiXUYkzdBaKDqsxtS2yC-CIq7EezBvOxaxgiYd9l4-CTV0BUZWjru6RpTX4dshSqhxN0-ojrCsP8LQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03708.gif" /> </td><td>TC224987 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage (two sites)</p></li></ul> </td><td>Homeodomain protein </td></tr><tr><td>hvu-miRX24 </td><td>212 </td><td>172 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03709.gif?Expires=2147483647&amp;Signature=ZN54B6u4SHqFqLsNO57Ra0jptQDFXX1XR1lxyeQabuQz4pX0qFy~CLBdrRd3qh90SjnjGQoNoCeyaSOv8udbKKOgJdca5qCWQ1WDdph9WS~LKrn860AWVwzvVY-69N1WWRpmUqM8mvldlEXZhS~zDQ~78geLu~X6-2nj6I20f8O4-YHIrd2VgPj7IDKLMkSPcoMIzCxHh9thcW7r5XFnTHJE9cYtGrlR01Y46fl05xRO7Z65JdnspnlFiibEV4Iy8QhJK1A958nuPeSPYga2x7ViXCEazxCObHSC-4p4raCZ1Z-KLnB1fjO3Us4mtvhEtEPpyeHLlgy4R1mpiKBP1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03709.gif" /> </td><td>TC195197 </td><td>Cleavage </td><td>Chloroplast Mg-chelatase subunit XANTHA-G precursor </td></tr><tr><td>hvu-miRX25 </td><td>28 </td><td>12 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03710.gif?Expires=2147483647&amp;Signature=vVBb53sbSuCSfaQSeckouhvLk0teWgr1pl-G5kor2-Q2zGcewcQovz-MxzeLmIUQ3ipAW0lSSG7aemB3X-rieKZTRMWB~sFCLeB35QdEWvtmWENRfBjpcNNX9nFKmSDpK4k8Wq3hJct842n5drOTcwf9m0VHIOH9GV-CLJPZEN43tN8tnXBwqmrQPvBS0JuE661-kHBsnMN3hXPrWizXCSebXrId3T0nBtLJlOkphKtvtnEz2OhgH8hRAwDLST6~hstZY4owKppONAxGkd5d6sYTMgSVToY33fzZdprlZjBhvyB0zJo-h4-asn2F70lkaTwNWzcbB86hxzR0MfMYDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03710.gif" /> </td><td>TC236049 </td><td>Cleavage </td><td>Transcription regulator protein </td></tr><tr><td>hvu-miRX26 </td><td>497 </td><td>543 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03711.gif?Expires=2147483647&amp;Signature=yBgmBVrff8od7pGXcS1cR2Xv8lwARORM8ThjVKIArBoQPRXHNo1eAkzx6tk7XKr8-ndVnPJAFrFFfiqkQk5ifqZOgWAZZrrpHWne3YvOhLXjkcKi-nkqbSfe07F5EHIqrqmJQdhjvXvW1mBbjCEebKcJVRfF1RvvokJRguB3aTuhzRmmIlT7EQ4RKKnDpbpzK4lngTaPfvnWB1jULJ0INp9D7gF1ECUMsJGaTMNFuB9MhKowheqKt0dZlj2HSEPa6fRPsmkH68OcDhQvz4nlJrGGV93Xa-I3dkuTBUPtaZRUNYRPbETypzRwXZ83yXhqpDnnUJDWPgF5u-XWoSIdSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03711.gif" /> </td><td>DN159928 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Adhesion protein PapA </td></tr><tr><td>hvu-miRX27 </td><td>230 </td><td>131 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03712.gif?Expires=2147483647&amp;Signature=VynHOfut~6Wq~tZmXDcSM9GvNfQLWWVEPTgSxk-BhWuc0nWE9TmiyXc-8rpjyc6OZtcZ6G0hJwF5vUyFXRX0CFuO03MoF1Ey2intmBHHLcIrR~hUSx~HvhQzCZil1nsbdL0wqoM-Q3gd7cOHfPpoOlxTdhHF2Sd98viXv1cKmmRDOBPJVuFQi8a9YcHX7xapT0Ztp-~eWM86faXviNqEkQpQ9-zs5scKI20wSCtZVFchGO6GaJBwSiB-gUcmCS2JBhWLlQFIYo5upR-v8yk0USWyz9ollqqWGylSBodqd~5BerdCtsq3Dl6B60ui9YJS7vd5xAcklOiRTeKHJgHkHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03712.gif" /> </td><td>BQ756036 </td><td>Cleavage </td><td>Terpene cyclase </td></tr><tr><td>hvu-miRX27 </td><td>257 </td><td>182 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03713.gif?Expires=2147483647&amp;Signature=U2X8bYVPPmrWNIXt~MOhWL-dsoguw2jj7ebcORObvIXp6Pdb4pel5I44ufH~b6YAP0R0qLdhQQyj~W0zD6JbieJOyYIhq9KnMbxgn9bJndKPNL32ETf6wlgayc-bGReBjavHokhT1WIoXKje9y9pWDrgLTu1YPYm70USbPcwNoPjRIowjuFtoPBApEQhKshOtj0lwNMBVGl9CgOloEIpNXudoSow1m9pYRyPHb27-rFFA2vj77ndbLrZOob5ArZYiKFDdd4~0vOoWCmQGam2UAHiBKiUct71BPe4zJYaEXNM4A0EVGGJGENw3jtErT4tnh0~zd74PmHu3pBqI2-WRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03713.gif" /> </td><td>TC196145 </td><td>Cleavage (two sites) </td><td>CTR1-like kinase kinase kinase-like </td></tr><tr><td>hvu-miRX28 </td><td>151 </td><td>133 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03714.gif?Expires=2147483647&amp;Signature=1qtw--gNItcowKi6ZXhOcEB5JJJ4egOLPKCQY9CeV8O~15K61pjBrSBPmslu093iKueF8UitvJuzjN51KBK3CDvIqJNgRY8wIdQnQ6n9fkQHcLBHCCucXiB-rCbfRcYeS6TIqzX5lG2ZonA7PuJFGHISvn9TUClCOT7bCRKtGzd~XwsnHJR~t4vVlC9hj7buMeg9~ineE7c7jHEsJMkM6eeybraGeEPFeyQy7j69QIPZLkj05s8KGdKpb276H-yEtX2igjRAl5BLGY1fqg7he5ujPVgEN1KqfAa6fnX0IYkvGkQLkGq6v5LdyIlhJwX4LwG4ThtN4E2yrb9aV~JiEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03714.gif" /> </td><td>TC229652 </td><td>Cleavage </td><td>Unknown </td></tr><tr><td>hvu-miRX29 </td><td>1463 </td><td>765 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03715.gif?Expires=2147483647&amp;Signature=1W1Zkk3nXfqA0grTu8W6P7XdKU1ZqGdMFCBsfhRF3VgHMC-ivIlxMU7rrIQqJ~mPRhBUj0IgN2ObgZa49tzhWXnquaIPhEN2aFgGX3D7YGWKYcV3J3udNhU3fcz4-Kguvqrtj1pwu1TLIDxNS9OHv8DBCt2BAht6SplTZVFjJJFulTCS5srER4QMP4VPoa6L3~F1noNRtKY-JNC4fYdcxlwbeqz59FMiGjhMUnPgvhM39QTkc9o7xekLvTaSe~EWYLy4ZD3dEZ9QNX9uYMhwLWwanil64udaAeL-jfXv8-Fl-NB7oRHuPIlY8veojRYD4y1CZfvwT23IHdiWkJeQJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03715.gif" /> </td><td>TC195723 </td><td>Cleavage </td><td>Pathogenesis-related protein 4 precursor </td></tr><tr><td>hvu-miRX30 </td><td>585 </td><td>392 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03716.gif?Expires=2147483647&amp;Signature=PDIA6A86AuUhpTggTyrkQNMHljc~IpUQq2mMrtyopdn41i3DmOFJhUIBH4aR5cFpO6eVjRHZH4h2YJpyqU7eHjjZr9BUbyEYflVoAd~A-Pg4FURZ0SYYoIYs~VCnPNzzqLEc3-73qbPsMAkl~j0bUKosfXSYqxg8BrfiuT6TsQace4OKpLrZih~8uwWcCuB416MfKTZvbSpxVT5oPqrycSfXIq3L-03rUzNgF2UxHmiuzaRZ3CeEpjs-0tIdDYQ59UERsIC7NRJbTv6AhZ7gTojOXT-fdfumz2Cdv78lT4pc7YasFrNFKYVRl60g8neK68RwUlOBJ2XDfD8-qFq14A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03716.gif" /> </td><td>TC214946 </td><td>Cleavage </td><td>Phospholipid hydroperoxide glutathione peroxidase-like protein </td></tr><tr><td>hvu-miRX31 </td><td>360 </td><td>269 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03717.gif?Expires=2147483647&amp;Signature=zkqD7VLpyF44-0Vv0J~UWULHHX3giXRqEiav7JkScHinth0rlZAi3jcEx5MRmUNeqZGdpqx3UKzG~i-tW~2s4rfpTkFmb5CfOK6gnXJdQ9lKS1buu4wZDloErnvc2amFImhrubx~Vx8LsX-Sr0p4XCarh3ao9Khaf2WTLSSoXBJ-DmLbcxMb1pOLHmiWGlwVdSX~kahi3a3Y3ZxqNSiyJq2sCMT6hoTcChRMQ~e9sn0JzAm18FpcKVj8Phjrsl2j4FVdw4WXNG8XJw3~KFOr4~gGjIolC0bZqRZda6OBNlrWUTUN31txAk5kqt0Ayz122U~7VI6A5mNw9PzIEtX75Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03717.gif" /> </td><td>TC196806 </td><td>Cleavage (four sites) </td><td>AMP-binding enzyme family protein expressed </td></tr><tr><td>hvu-miRX32 </td><td>193 </td><td>92 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03718.gif?Expires=2147483647&amp;Signature=Dmnaeye4Sv86u-Q3nB7gPrNduYrpdpT2L-g7Jbig3FVvzcG29Awzliuuth20IAg0zXgzyynvMY69xCU8WsDQ8n7jbCPxDBcJ13sc~nRojImNscz72-37DH9yOIHIIxpv55AZ5cDQqVh9TvccUqMyThpINT4vezrjhP9N5g9jPEgag7C0vt9OzMILzb8q02cFyx8XLQgHeh~a5LvFebTXeR4MoB3sOKmZjHsCvF8Sj6ANmT6njYtfxtcUMilZLcpZQt8tbPPCO9RyDxUhOvLZY9wtr4SGV4fKZjp01tZHWbh9-Lbg~U5d5TbkkrPe4sqwTTO1oc09OxvGMj-r1aC0eQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03718.gif" /> </td><td>EX593121 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Unknown </td></tr></tbody></table></div><div content-id="DSS037TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>miRNA </th><th>Count –P </th><th>Count +P </th><th>Precursor miRNA </th><th>Target Accession </th><th>Action </th><th>Function </th></tr></thead><tbody><tr><td>hvu-miRX21 </td><td>3512 </td><td>3279 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03706.gif?Expires=2147483647&amp;Signature=uys9GEJPW13OXEw-uL6Z~H8oeHUcd1VlayHvcN3BLKDlJANTm~mI2ys3OEQuzYWtMqV4AnbgA0HHaoCaosWWMu1l5Ik~l9rhWkZHpQTccyg9xxZuxrb4UsktS61plc0SGWDd5pzPfN7E8BZmmUHB-6XdNW5fIG1jA6qvF503DONjlPWVpBF1ZEHTDWngneLew1czsgxvslK-W~e2wiSiNtCjNJfAed5~fVG~Qa0ZmYz4SLeq7TAXs4oYSGLGa3-e8rrWef37wvv1W-Jz97uFynm3yOmA1SHjYzT7F-goMtoqcmAtPFKLWYhLK7nHuF3jAlyviTUd8CNCMyigA4iN4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03706.gif" /> </td><td>TC207745 </td><td>Cleavage </td><td>E3 ubiquitin-protein ligase </td></tr><tr><td>hvu-miRX22 </td><td>3494 </td><td>2015 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03707.gif?Expires=2147483647&amp;Signature=0JR2rriAmRnGhIDR-BRjORZW8sUq74tYQIHrhXgOzWQnPN40lv8kS81rQ6c7o4z4~lHGwJLLSKpzwvMMBONF4JPijmVFig5fl5VPRpf-1ZC-lz8dOI2Ly5g3zwNOQn6ikJoL9dXhHRi5XDOA5UF4S38mN5HblYGSNTeGxPlcW3VaI7j~L6eYarpKtq8B7Ut-RKd-BY2YBq1UZxCRUBp2~IpBRNHkVJ4gLM2dEP5VrcGcgo0HmQ~wDoV7eRw9nxZkQq77Ee3A-jV3Od~UiN-RPDIa9sq0KUhGITkS0rrse6W9p42Nc6LScGaRKRpMgKlFLgX0eFMJMSZd4uX5atCiZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03707.gif" /> </td><td>TC236060 </td><td>Cleavage </td><td>unknown </td></tr><tr><td>hvu-miRX23 </td><td>85 </td><td>69 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03708.gif?Expires=2147483647&amp;Signature=WH-W~2HdPLWJhgK0zv5fdUVzbE1lVTwT9wZlayqht9XBY9x-GdlHFldf9jNPXPTqjRMtz-RK0dTYHeKztkGPs7yJm3fzYYv28Ma00Pb7X56wrpJrxJPZDOe3fcXorMumQyNQF0YuRi70DQZgRdqWLs0Xh3G7PXObShkS9-KfH0gXSVvEjxpwfmjxYV0v5QZJcRyS0PKova6LMMkwjC-gRq2gqjy08TzaYuegiCwumMbH6StEq8P7fTrSVMjdoB2qDfj9fP~8HiXUYkzdBaKDqsxtS2yC-CIq7EezBvOxaxgiYd9l4-CTV0BUZWjru6RpTX4dshSqhxN0-ojrCsP8LQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03708.gif" /> </td><td>TC224987 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage (two sites)</p></li></ul> </td><td>Homeodomain protein </td></tr><tr><td>hvu-miRX24 </td><td>212 </td><td>172 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03709.gif?Expires=2147483647&amp;Signature=ZN54B6u4SHqFqLsNO57Ra0jptQDFXX1XR1lxyeQabuQz4pX0qFy~CLBdrRd3qh90SjnjGQoNoCeyaSOv8udbKKOgJdca5qCWQ1WDdph9WS~LKrn860AWVwzvVY-69N1WWRpmUqM8mvldlEXZhS~zDQ~78geLu~X6-2nj6I20f8O4-YHIrd2VgPj7IDKLMkSPcoMIzCxHh9thcW7r5XFnTHJE9cYtGrlR01Y46fl05xRO7Z65JdnspnlFiibEV4Iy8QhJK1A958nuPeSPYga2x7ViXCEazxCObHSC-4p4raCZ1Z-KLnB1fjO3Us4mtvhEtEPpyeHLlgy4R1mpiKBP1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03709.gif" /> </td><td>TC195197 </td><td>Cleavage </td><td>Chloroplast Mg-chelatase subunit XANTHA-G precursor </td></tr><tr><td>hvu-miRX25 </td><td>28 </td><td>12 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03710.gif?Expires=2147483647&amp;Signature=vVBb53sbSuCSfaQSeckouhvLk0teWgr1pl-G5kor2-Q2zGcewcQovz-MxzeLmIUQ3ipAW0lSSG7aemB3X-rieKZTRMWB~sFCLeB35QdEWvtmWENRfBjpcNNX9nFKmSDpK4k8Wq3hJct842n5drOTcwf9m0VHIOH9GV-CLJPZEN43tN8tnXBwqmrQPvBS0JuE661-kHBsnMN3hXPrWizXCSebXrId3T0nBtLJlOkphKtvtnEz2OhgH8hRAwDLST6~hstZY4owKppONAxGkd5d6sYTMgSVToY33fzZdprlZjBhvyB0zJo-h4-asn2F70lkaTwNWzcbB86hxzR0MfMYDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03710.gif" /> </td><td>TC236049 </td><td>Cleavage </td><td>Transcription regulator protein </td></tr><tr><td>hvu-miRX26 </td><td>497 </td><td>543 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03711.gif?Expires=2147483647&amp;Signature=yBgmBVrff8od7pGXcS1cR2Xv8lwARORM8ThjVKIArBoQPRXHNo1eAkzx6tk7XKr8-ndVnPJAFrFFfiqkQk5ifqZOgWAZZrrpHWne3YvOhLXjkcKi-nkqbSfe07F5EHIqrqmJQdhjvXvW1mBbjCEebKcJVRfF1RvvokJRguB3aTuhzRmmIlT7EQ4RKKnDpbpzK4lngTaPfvnWB1jULJ0INp9D7gF1ECUMsJGaTMNFuB9MhKowheqKt0dZlj2HSEPa6fRPsmkH68OcDhQvz4nlJrGGV93Xa-I3dkuTBUPtaZRUNYRPbETypzRwXZ83yXhqpDnnUJDWPgF5u-XWoSIdSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03711.gif" /> </td><td>DN159928 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Adhesion protein PapA </td></tr><tr><td>hvu-miRX27 </td><td>230 </td><td>131 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03712.gif?Expires=2147483647&amp;Signature=VynHOfut~6Wq~tZmXDcSM9GvNfQLWWVEPTgSxk-BhWuc0nWE9TmiyXc-8rpjyc6OZtcZ6G0hJwF5vUyFXRX0CFuO03MoF1Ey2intmBHHLcIrR~hUSx~HvhQzCZil1nsbdL0wqoM-Q3gd7cOHfPpoOlxTdhHF2Sd98viXv1cKmmRDOBPJVuFQi8a9YcHX7xapT0Ztp-~eWM86faXviNqEkQpQ9-zs5scKI20wSCtZVFchGO6GaJBwSiB-gUcmCS2JBhWLlQFIYo5upR-v8yk0USWyz9ollqqWGylSBodqd~5BerdCtsq3Dl6B60ui9YJS7vd5xAcklOiRTeKHJgHkHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03712.gif" /> </td><td>BQ756036 </td><td>Cleavage </td><td>Terpene cyclase </td></tr><tr><td>hvu-miRX27 </td><td>257 </td><td>182 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03713.gif?Expires=2147483647&amp;Signature=U2X8bYVPPmrWNIXt~MOhWL-dsoguw2jj7ebcORObvIXp6Pdb4pel5I44ufH~b6YAP0R0qLdhQQyj~W0zD6JbieJOyYIhq9KnMbxgn9bJndKPNL32ETf6wlgayc-bGReBjavHokhT1WIoXKje9y9pWDrgLTu1YPYm70USbPcwNoPjRIowjuFtoPBApEQhKshOtj0lwNMBVGl9CgOloEIpNXudoSow1m9pYRyPHb27-rFFA2vj77ndbLrZOob5ArZYiKFDdd4~0vOoWCmQGam2UAHiBKiUct71BPe4zJYaEXNM4A0EVGGJGENw3jtErT4tnh0~zd74PmHu3pBqI2-WRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03713.gif" /> </td><td>TC196145 </td><td>Cleavage (two sites) </td><td>CTR1-like kinase kinase kinase-like </td></tr><tr><td>hvu-miRX28 </td><td>151 </td><td>133 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03714.gif?Expires=2147483647&amp;Signature=1qtw--gNItcowKi6ZXhOcEB5JJJ4egOLPKCQY9CeV8O~15K61pjBrSBPmslu093iKueF8UitvJuzjN51KBK3CDvIqJNgRY8wIdQnQ6n9fkQHcLBHCCucXiB-rCbfRcYeS6TIqzX5lG2ZonA7PuJFGHISvn9TUClCOT7bCRKtGzd~XwsnHJR~t4vVlC9hj7buMeg9~ineE7c7jHEsJMkM6eeybraGeEPFeyQy7j69QIPZLkj05s8KGdKpb276H-yEtX2igjRAl5BLGY1fqg7he5ujPVgEN1KqfAa6fnX0IYkvGkQLkGq6v5LdyIlhJwX4LwG4ThtN4E2yrb9aV~JiEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03714.gif" /> </td><td>TC229652 </td><td>Cleavage </td><td>Unknown </td></tr><tr><td>hvu-miRX29 </td><td>1463 </td><td>765 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03715.gif?Expires=2147483647&amp;Signature=1W1Zkk3nXfqA0grTu8W6P7XdKU1ZqGdMFCBsfhRF3VgHMC-ivIlxMU7rrIQqJ~mPRhBUj0IgN2ObgZa49tzhWXnquaIPhEN2aFgGX3D7YGWKYcV3J3udNhU3fcz4-Kguvqrtj1pwu1TLIDxNS9OHv8DBCt2BAht6SplTZVFjJJFulTCS5srER4QMP4VPoa6L3~F1noNRtKY-JNC4fYdcxlwbeqz59FMiGjhMUnPgvhM39QTkc9o7xekLvTaSe~EWYLy4ZD3dEZ9QNX9uYMhwLWwanil64udaAeL-jfXv8-Fl-NB7oRHuPIlY8veojRYD4y1CZfvwT23IHdiWkJeQJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03715.gif" /> </td><td>TC195723 </td><td>Cleavage </td><td>Pathogenesis-related protein 4 precursor </td></tr><tr><td>hvu-miRX30 </td><td>585 </td><td>392 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03716.gif?Expires=2147483647&amp;Signature=PDIA6A86AuUhpTggTyrkQNMHljc~IpUQq2mMrtyopdn41i3DmOFJhUIBH4aR5cFpO6eVjRHZH4h2YJpyqU7eHjjZr9BUbyEYflVoAd~A-Pg4FURZ0SYYoIYs~VCnPNzzqLEc3-73qbPsMAkl~j0bUKosfXSYqxg8BrfiuT6TsQace4OKpLrZih~8uwWcCuB416MfKTZvbSpxVT5oPqrycSfXIq3L-03rUzNgF2UxHmiuzaRZ3CeEpjs-0tIdDYQ59UERsIC7NRJbTv6AhZ7gTojOXT-fdfumz2Cdv78lT4pc7YasFrNFKYVRl60g8neK68RwUlOBJ2XDfD8-qFq14A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03716.gif" /> </td><td>TC214946 </td><td>Cleavage </td><td>Phospholipid hydroperoxide glutathione peroxidase-like protein </td></tr><tr><td>hvu-miRX31 </td><td>360 </td><td>269 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03717.gif?Expires=2147483647&amp;Signature=zkqD7VLpyF44-0Vv0J~UWULHHX3giXRqEiav7JkScHinth0rlZAi3jcEx5MRmUNeqZGdpqx3UKzG~i-tW~2s4rfpTkFmb5CfOK6gnXJdQ9lKS1buu4wZDloErnvc2amFImhrubx~Vx8LsX-Sr0p4XCarh3ao9Khaf2WTLSSoXBJ-DmLbcxMb1pOLHmiWGlwVdSX~kahi3a3Y3ZxqNSiyJq2sCMT6hoTcChRMQ~e9sn0JzAm18FpcKVj8Phjrsl2j4FVdw4WXNG8XJw3~KFOr4~gGjIolC0bZqRZda6OBNlrWUTUN31txAk5kqt0Ayz122U~7VI6A5mNw9PzIEtX75Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03717.gif" /> </td><td>TC196806 </td><td>Cleavage (four sites) </td><td>AMP-binding enzyme family protein expressed </td></tr><tr><td>hvu-miRX32 </td><td>193 </td><td>92 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03718.gif?Expires=2147483647&amp;Signature=Dmnaeye4Sv86u-Q3nB7gPrNduYrpdpT2L-g7Jbig3FVvzcG29Awzliuuth20IAg0zXgzyynvMY69xCU8WsDQ8n7jbCPxDBcJ13sc~nRojImNscz72-37DH9yOIHIIxpv55AZ5cDQqVh9TvccUqMyThpINT4vezrjhP9N5g9jPEgag7C0vt9OzMILzb8q02cFyx8XLQgHeh~a5LvFebTXeR4MoB3sOKmZjHsCvF8Sj6ANmT6njYtfxtcUMilZLcpZQt8tbPPCO9RyDxUhOvLZY9wtr4SGV4fKZjp01tZHWbh9-Lbg~U5d5TbkkrPe4sqwTTO1oc09OxvGMj-r1aC0eQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03718.gif" /> </td><td>EX593121 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Unknown </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>‘Count–P’ and ‘count +P’ indicate read counts from P-deficient and P-sufficient barley shoots, respectively. Targets were predicted using psRNAtarget (<a class="link link-uri" href="http://plantgrn.noble.org/psRNATarget/">http://plantgrn.noble.org/psRNATarget/</a>). Cleavage sites were determined using degradome library.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600257" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS037TB4" class="table-wrap-title" data-id="DSS037TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Predicted novel miRNAs in P-deficient and P-sufficient shoots of Pallas barley by miRDeep</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>miRNA </th><th>Count –P </th><th>Count +P </th><th>Precursor miRNA </th><th>Target Accession </th><th>Action </th><th>Function </th></tr></thead><tbody><tr><td>hvu-miRX21 </td><td>3512 </td><td>3279 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03706.gif?Expires=2147483647&amp;Signature=uys9GEJPW13OXEw-uL6Z~H8oeHUcd1VlayHvcN3BLKDlJANTm~mI2ys3OEQuzYWtMqV4AnbgA0HHaoCaosWWMu1l5Ik~l9rhWkZHpQTccyg9xxZuxrb4UsktS61plc0SGWDd5pzPfN7E8BZmmUHB-6XdNW5fIG1jA6qvF503DONjlPWVpBF1ZEHTDWngneLew1czsgxvslK-W~e2wiSiNtCjNJfAed5~fVG~Qa0ZmYz4SLeq7TAXs4oYSGLGa3-e8rrWef37wvv1W-Jz97uFynm3yOmA1SHjYzT7F-goMtoqcmAtPFKLWYhLK7nHuF3jAlyviTUd8CNCMyigA4iN4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03706.gif" /> </td><td>TC207745 </td><td>Cleavage </td><td>E3 ubiquitin-protein ligase </td></tr><tr><td>hvu-miRX22 </td><td>3494 </td><td>2015 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03707.gif?Expires=2147483647&amp;Signature=0JR2rriAmRnGhIDR-BRjORZW8sUq74tYQIHrhXgOzWQnPN40lv8kS81rQ6c7o4z4~lHGwJLLSKpzwvMMBONF4JPijmVFig5fl5VPRpf-1ZC-lz8dOI2Ly5g3zwNOQn6ikJoL9dXhHRi5XDOA5UF4S38mN5HblYGSNTeGxPlcW3VaI7j~L6eYarpKtq8B7Ut-RKd-BY2YBq1UZxCRUBp2~IpBRNHkVJ4gLM2dEP5VrcGcgo0HmQ~wDoV7eRw9nxZkQq77Ee3A-jV3Od~UiN-RPDIa9sq0KUhGITkS0rrse6W9p42Nc6LScGaRKRpMgKlFLgX0eFMJMSZd4uX5atCiZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03707.gif" /> </td><td>TC236060 </td><td>Cleavage </td><td>unknown </td></tr><tr><td>hvu-miRX23 </td><td>85 </td><td>69 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03708.gif?Expires=2147483647&amp;Signature=WH-W~2HdPLWJhgK0zv5fdUVzbE1lVTwT9wZlayqht9XBY9x-GdlHFldf9jNPXPTqjRMtz-RK0dTYHeKztkGPs7yJm3fzYYv28Ma00Pb7X56wrpJrxJPZDOe3fcXorMumQyNQF0YuRi70DQZgRdqWLs0Xh3G7PXObShkS9-KfH0gXSVvEjxpwfmjxYV0v5QZJcRyS0PKova6LMMkwjC-gRq2gqjy08TzaYuegiCwumMbH6StEq8P7fTrSVMjdoB2qDfj9fP~8HiXUYkzdBaKDqsxtS2yC-CIq7EezBvOxaxgiYd9l4-CTV0BUZWjru6RpTX4dshSqhxN0-ojrCsP8LQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03708.gif" /> </td><td>TC224987 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage (two sites)</p></li></ul> </td><td>Homeodomain protein </td></tr><tr><td>hvu-miRX24 </td><td>212 </td><td>172 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03709.gif?Expires=2147483647&amp;Signature=ZN54B6u4SHqFqLsNO57Ra0jptQDFXX1XR1lxyeQabuQz4pX0qFy~CLBdrRd3qh90SjnjGQoNoCeyaSOv8udbKKOgJdca5qCWQ1WDdph9WS~LKrn860AWVwzvVY-69N1WWRpmUqM8mvldlEXZhS~zDQ~78geLu~X6-2nj6I20f8O4-YHIrd2VgPj7IDKLMkSPcoMIzCxHh9thcW7r5XFnTHJE9cYtGrlR01Y46fl05xRO7Z65JdnspnlFiibEV4Iy8QhJK1A958nuPeSPYga2x7ViXCEazxCObHSC-4p4raCZ1Z-KLnB1fjO3Us4mtvhEtEPpyeHLlgy4R1mpiKBP1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03709.gif" /> </td><td>TC195197 </td><td>Cleavage </td><td>Chloroplast Mg-chelatase subunit XANTHA-G precursor </td></tr><tr><td>hvu-miRX25 </td><td>28 </td><td>12 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03710.gif?Expires=2147483647&amp;Signature=vVBb53sbSuCSfaQSeckouhvLk0teWgr1pl-G5kor2-Q2zGcewcQovz-MxzeLmIUQ3ipAW0lSSG7aemB3X-rieKZTRMWB~sFCLeB35QdEWvtmWENRfBjpcNNX9nFKmSDpK4k8Wq3hJct842n5drOTcwf9m0VHIOH9GV-CLJPZEN43tN8tnXBwqmrQPvBS0JuE661-kHBsnMN3hXPrWizXCSebXrId3T0nBtLJlOkphKtvtnEz2OhgH8hRAwDLST6~hstZY4owKppONAxGkd5d6sYTMgSVToY33fzZdprlZjBhvyB0zJo-h4-asn2F70lkaTwNWzcbB86hxzR0MfMYDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03710.gif" /> </td><td>TC236049 </td><td>Cleavage </td><td>Transcription regulator protein </td></tr><tr><td>hvu-miRX26 </td><td>497 </td><td>543 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03711.gif?Expires=2147483647&amp;Signature=yBgmBVrff8od7pGXcS1cR2Xv8lwARORM8ThjVKIArBoQPRXHNo1eAkzx6tk7XKr8-ndVnPJAFrFFfiqkQk5ifqZOgWAZZrrpHWne3YvOhLXjkcKi-nkqbSfe07F5EHIqrqmJQdhjvXvW1mBbjCEebKcJVRfF1RvvokJRguB3aTuhzRmmIlT7EQ4RKKnDpbpzK4lngTaPfvnWB1jULJ0INp9D7gF1ECUMsJGaTMNFuB9MhKowheqKt0dZlj2HSEPa6fRPsmkH68OcDhQvz4nlJrGGV93Xa-I3dkuTBUPtaZRUNYRPbETypzRwXZ83yXhqpDnnUJDWPgF5u-XWoSIdSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03711.gif" /> </td><td>DN159928 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Adhesion protein PapA </td></tr><tr><td>hvu-miRX27 </td><td>230 </td><td>131 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03712.gif?Expires=2147483647&amp;Signature=VynHOfut~6Wq~tZmXDcSM9GvNfQLWWVEPTgSxk-BhWuc0nWE9TmiyXc-8rpjyc6OZtcZ6G0hJwF5vUyFXRX0CFuO03MoF1Ey2intmBHHLcIrR~hUSx~HvhQzCZil1nsbdL0wqoM-Q3gd7cOHfPpoOlxTdhHF2Sd98viXv1cKmmRDOBPJVuFQi8a9YcHX7xapT0Ztp-~eWM86faXviNqEkQpQ9-zs5scKI20wSCtZVFchGO6GaJBwSiB-gUcmCS2JBhWLlQFIYo5upR-v8yk0USWyz9ollqqWGylSBodqd~5BerdCtsq3Dl6B60ui9YJS7vd5xAcklOiRTeKHJgHkHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03712.gif" /> </td><td>BQ756036 </td><td>Cleavage </td><td>Terpene cyclase </td></tr><tr><td>hvu-miRX27 </td><td>257 </td><td>182 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03713.gif?Expires=2147483647&amp;Signature=U2X8bYVPPmrWNIXt~MOhWL-dsoguw2jj7ebcORObvIXp6Pdb4pel5I44ufH~b6YAP0R0qLdhQQyj~W0zD6JbieJOyYIhq9KnMbxgn9bJndKPNL32ETf6wlgayc-bGReBjavHokhT1WIoXKje9y9pWDrgLTu1YPYm70USbPcwNoPjRIowjuFtoPBApEQhKshOtj0lwNMBVGl9CgOloEIpNXudoSow1m9pYRyPHb27-rFFA2vj77ndbLrZOob5ArZYiKFDdd4~0vOoWCmQGam2UAHiBKiUct71BPe4zJYaEXNM4A0EVGGJGENw3jtErT4tnh0~zd74PmHu3pBqI2-WRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03713.gif" /> </td><td>TC196145 </td><td>Cleavage (two sites) </td><td>CTR1-like kinase kinase kinase-like </td></tr><tr><td>hvu-miRX28 </td><td>151 </td><td>133 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03714.gif?Expires=2147483647&amp;Signature=1qtw--gNItcowKi6ZXhOcEB5JJJ4egOLPKCQY9CeV8O~15K61pjBrSBPmslu093iKueF8UitvJuzjN51KBK3CDvIqJNgRY8wIdQnQ6n9fkQHcLBHCCucXiB-rCbfRcYeS6TIqzX5lG2ZonA7PuJFGHISvn9TUClCOT7bCRKtGzd~XwsnHJR~t4vVlC9hj7buMeg9~ineE7c7jHEsJMkM6eeybraGeEPFeyQy7j69QIPZLkj05s8KGdKpb276H-yEtX2igjRAl5BLGY1fqg7he5ujPVgEN1KqfAa6fnX0IYkvGkQLkGq6v5LdyIlhJwX4LwG4ThtN4E2yrb9aV~JiEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03714.gif" /> </td><td>TC229652 </td><td>Cleavage </td><td>Unknown </td></tr><tr><td>hvu-miRX29 </td><td>1463 </td><td>765 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03715.gif?Expires=2147483647&amp;Signature=1W1Zkk3nXfqA0grTu8W6P7XdKU1ZqGdMFCBsfhRF3VgHMC-ivIlxMU7rrIQqJ~mPRhBUj0IgN2ObgZa49tzhWXnquaIPhEN2aFgGX3D7YGWKYcV3J3udNhU3fcz4-Kguvqrtj1pwu1TLIDxNS9OHv8DBCt2BAht6SplTZVFjJJFulTCS5srER4QMP4VPoa6L3~F1noNRtKY-JNC4fYdcxlwbeqz59FMiGjhMUnPgvhM39QTkc9o7xekLvTaSe~EWYLy4ZD3dEZ9QNX9uYMhwLWwanil64udaAeL-jfXv8-Fl-NB7oRHuPIlY8veojRYD4y1CZfvwT23IHdiWkJeQJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03715.gif" /> </td><td>TC195723 </td><td>Cleavage </td><td>Pathogenesis-related protein 4 precursor </td></tr><tr><td>hvu-miRX30 </td><td>585 </td><td>392 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03716.gif?Expires=2147483647&amp;Signature=PDIA6A86AuUhpTggTyrkQNMHljc~IpUQq2mMrtyopdn41i3DmOFJhUIBH4aR5cFpO6eVjRHZH4h2YJpyqU7eHjjZr9BUbyEYflVoAd~A-Pg4FURZ0SYYoIYs~VCnPNzzqLEc3-73qbPsMAkl~j0bUKosfXSYqxg8BrfiuT6TsQace4OKpLrZih~8uwWcCuB416MfKTZvbSpxVT5oPqrycSfXIq3L-03rUzNgF2UxHmiuzaRZ3CeEpjs-0tIdDYQ59UERsIC7NRJbTv6AhZ7gTojOXT-fdfumz2Cdv78lT4pc7YasFrNFKYVRl60g8neK68RwUlOBJ2XDfD8-qFq14A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03716.gif" /> </td><td>TC214946 </td><td>Cleavage </td><td>Phospholipid hydroperoxide glutathione peroxidase-like protein </td></tr><tr><td>hvu-miRX31 </td><td>360 </td><td>269 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03717.gif?Expires=2147483647&amp;Signature=zkqD7VLpyF44-0Vv0J~UWULHHX3giXRqEiav7JkScHinth0rlZAi3jcEx5MRmUNeqZGdpqx3UKzG~i-tW~2s4rfpTkFmb5CfOK6gnXJdQ9lKS1buu4wZDloErnvc2amFImhrubx~Vx8LsX-Sr0p4XCarh3ao9Khaf2WTLSSoXBJ-DmLbcxMb1pOLHmiWGlwVdSX~kahi3a3Y3ZxqNSiyJq2sCMT6hoTcChRMQ~e9sn0JzAm18FpcKVj8Phjrsl2j4FVdw4WXNG8XJw3~KFOr4~gGjIolC0bZqRZda6OBNlrWUTUN31txAk5kqt0Ayz122U~7VI6A5mNw9PzIEtX75Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03717.gif" /> </td><td>TC196806 </td><td>Cleavage (four sites) </td><td>AMP-binding enzyme family protein expressed </td></tr><tr><td>hvu-miRX32 </td><td>193 </td><td>92 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03718.gif?Expires=2147483647&amp;Signature=Dmnaeye4Sv86u-Q3nB7gPrNduYrpdpT2L-g7Jbig3FVvzcG29Awzliuuth20IAg0zXgzyynvMY69xCU8WsDQ8n7jbCPxDBcJ13sc~nRojImNscz72-37DH9yOIHIIxpv55AZ5cDQqVh9TvccUqMyThpINT4vezrjhP9N5g9jPEgag7C0vt9OzMILzb8q02cFyx8XLQgHeh~a5LvFebTXeR4MoB3sOKmZjHsCvF8Sj6ANmT6njYtfxtcUMilZLcpZQt8tbPPCO9RyDxUhOvLZY9wtr4SGV4fKZjp01tZHWbh9-Lbg~U5d5TbkkrPe4sqwTTO1oc09OxvGMj-r1aC0eQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03718.gif" /> </td><td>EX593121 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Unknown </td></tr></tbody></table></div><div content-id="DSS037TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>miRNA </th><th>Count –P </th><th>Count +P </th><th>Precursor miRNA </th><th>Target Accession </th><th>Action </th><th>Function </th></tr></thead><tbody><tr><td>hvu-miRX21 </td><td>3512 </td><td>3279 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03706.gif?Expires=2147483647&amp;Signature=uys9GEJPW13OXEw-uL6Z~H8oeHUcd1VlayHvcN3BLKDlJANTm~mI2ys3OEQuzYWtMqV4AnbgA0HHaoCaosWWMu1l5Ik~l9rhWkZHpQTccyg9xxZuxrb4UsktS61plc0SGWDd5pzPfN7E8BZmmUHB-6XdNW5fIG1jA6qvF503DONjlPWVpBF1ZEHTDWngneLew1czsgxvslK-W~e2wiSiNtCjNJfAed5~fVG~Qa0ZmYz4SLeq7TAXs4oYSGLGa3-e8rrWef37wvv1W-Jz97uFynm3yOmA1SHjYzT7F-goMtoqcmAtPFKLWYhLK7nHuF3jAlyviTUd8CNCMyigA4iN4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03706.gif" /> </td><td>TC207745 </td><td>Cleavage </td><td>E3 ubiquitin-protein ligase </td></tr><tr><td>hvu-miRX22 </td><td>3494 </td><td>2015 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03707.gif?Expires=2147483647&amp;Signature=0JR2rriAmRnGhIDR-BRjORZW8sUq74tYQIHrhXgOzWQnPN40lv8kS81rQ6c7o4z4~lHGwJLLSKpzwvMMBONF4JPijmVFig5fl5VPRpf-1ZC-lz8dOI2Ly5g3zwNOQn6ikJoL9dXhHRi5XDOA5UF4S38mN5HblYGSNTeGxPlcW3VaI7j~L6eYarpKtq8B7Ut-RKd-BY2YBq1UZxCRUBp2~IpBRNHkVJ4gLM2dEP5VrcGcgo0HmQ~wDoV7eRw9nxZkQq77Ee3A-jV3Od~UiN-RPDIa9sq0KUhGITkS0rrse6W9p42Nc6LScGaRKRpMgKlFLgX0eFMJMSZd4uX5atCiZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03707.gif" /> </td><td>TC236060 </td><td>Cleavage </td><td>unknown </td></tr><tr><td>hvu-miRX23 </td><td>85 </td><td>69 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03708.gif?Expires=2147483647&amp;Signature=WH-W~2HdPLWJhgK0zv5fdUVzbE1lVTwT9wZlayqht9XBY9x-GdlHFldf9jNPXPTqjRMtz-RK0dTYHeKztkGPs7yJm3fzYYv28Ma00Pb7X56wrpJrxJPZDOe3fcXorMumQyNQF0YuRi70DQZgRdqWLs0Xh3G7PXObShkS9-KfH0gXSVvEjxpwfmjxYV0v5QZJcRyS0PKova6LMMkwjC-gRq2gqjy08TzaYuegiCwumMbH6StEq8P7fTrSVMjdoB2qDfj9fP~8HiXUYkzdBaKDqsxtS2yC-CIq7EezBvOxaxgiYd9l4-CTV0BUZWjru6RpTX4dshSqhxN0-ojrCsP8LQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03708.gif" /> </td><td>TC224987 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage (two sites)</p></li></ul> </td><td>Homeodomain protein </td></tr><tr><td>hvu-miRX24 </td><td>212 </td><td>172 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03709.gif?Expires=2147483647&amp;Signature=ZN54B6u4SHqFqLsNO57Ra0jptQDFXX1XR1lxyeQabuQz4pX0qFy~CLBdrRd3qh90SjnjGQoNoCeyaSOv8udbKKOgJdca5qCWQ1WDdph9WS~LKrn860AWVwzvVY-69N1WWRpmUqM8mvldlEXZhS~zDQ~78geLu~X6-2nj6I20f8O4-YHIrd2VgPj7IDKLMkSPcoMIzCxHh9thcW7r5XFnTHJE9cYtGrlR01Y46fl05xRO7Z65JdnspnlFiibEV4Iy8QhJK1A958nuPeSPYga2x7ViXCEazxCObHSC-4p4raCZ1Z-KLnB1fjO3Us4mtvhEtEPpyeHLlgy4R1mpiKBP1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03709.gif" /> </td><td>TC195197 </td><td>Cleavage </td><td>Chloroplast Mg-chelatase subunit XANTHA-G precursor </td></tr><tr><td>hvu-miRX25 </td><td>28 </td><td>12 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03710.gif?Expires=2147483647&amp;Signature=vVBb53sbSuCSfaQSeckouhvLk0teWgr1pl-G5kor2-Q2zGcewcQovz-MxzeLmIUQ3ipAW0lSSG7aemB3X-rieKZTRMWB~sFCLeB35QdEWvtmWENRfBjpcNNX9nFKmSDpK4k8Wq3hJct842n5drOTcwf9m0VHIOH9GV-CLJPZEN43tN8tnXBwqmrQPvBS0JuE661-kHBsnMN3hXPrWizXCSebXrId3T0nBtLJlOkphKtvtnEz2OhgH8hRAwDLST6~hstZY4owKppONAxGkd5d6sYTMgSVToY33fzZdprlZjBhvyB0zJo-h4-asn2F70lkaTwNWzcbB86hxzR0MfMYDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03710.gif" /> </td><td>TC236049 </td><td>Cleavage </td><td>Transcription regulator protein </td></tr><tr><td>hvu-miRX26 </td><td>497 </td><td>543 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03711.gif?Expires=2147483647&amp;Signature=yBgmBVrff8od7pGXcS1cR2Xv8lwARORM8ThjVKIArBoQPRXHNo1eAkzx6tk7XKr8-ndVnPJAFrFFfiqkQk5ifqZOgWAZZrrpHWne3YvOhLXjkcKi-nkqbSfe07F5EHIqrqmJQdhjvXvW1mBbjCEebKcJVRfF1RvvokJRguB3aTuhzRmmIlT7EQ4RKKnDpbpzK4lngTaPfvnWB1jULJ0INp9D7gF1ECUMsJGaTMNFuB9MhKowheqKt0dZlj2HSEPa6fRPsmkH68OcDhQvz4nlJrGGV93Xa-I3dkuTBUPtaZRUNYRPbETypzRwXZ83yXhqpDnnUJDWPgF5u-XWoSIdSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03711.gif" /> </td><td>DN159928 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Adhesion protein PapA </td></tr><tr><td>hvu-miRX27 </td><td>230 </td><td>131 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03712.gif?Expires=2147483647&amp;Signature=VynHOfut~6Wq~tZmXDcSM9GvNfQLWWVEPTgSxk-BhWuc0nWE9TmiyXc-8rpjyc6OZtcZ6G0hJwF5vUyFXRX0CFuO03MoF1Ey2intmBHHLcIrR~hUSx~HvhQzCZil1nsbdL0wqoM-Q3gd7cOHfPpoOlxTdhHF2Sd98viXv1cKmmRDOBPJVuFQi8a9YcHX7xapT0Ztp-~eWM86faXviNqEkQpQ9-zs5scKI20wSCtZVFchGO6GaJBwSiB-gUcmCS2JBhWLlQFIYo5upR-v8yk0USWyz9ollqqWGylSBodqd~5BerdCtsq3Dl6B60ui9YJS7vd5xAcklOiRTeKHJgHkHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03712.gif" /> </td><td>BQ756036 </td><td>Cleavage </td><td>Terpene cyclase </td></tr><tr><td>hvu-miRX27 </td><td>257 </td><td>182 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03713.gif?Expires=2147483647&amp;Signature=U2X8bYVPPmrWNIXt~MOhWL-dsoguw2jj7ebcORObvIXp6Pdb4pel5I44ufH~b6YAP0R0qLdhQQyj~W0zD6JbieJOyYIhq9KnMbxgn9bJndKPNL32ETf6wlgayc-bGReBjavHokhT1WIoXKje9y9pWDrgLTu1YPYm70USbPcwNoPjRIowjuFtoPBApEQhKshOtj0lwNMBVGl9CgOloEIpNXudoSow1m9pYRyPHb27-rFFA2vj77ndbLrZOob5ArZYiKFDdd4~0vOoWCmQGam2UAHiBKiUct71BPe4zJYaEXNM4A0EVGGJGENw3jtErT4tnh0~zd74PmHu3pBqI2-WRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03713.gif" /> </td><td>TC196145 </td><td>Cleavage (two sites) </td><td>CTR1-like kinase kinase kinase-like </td></tr><tr><td>hvu-miRX28 </td><td>151 </td><td>133 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03714.gif?Expires=2147483647&amp;Signature=1qtw--gNItcowKi6ZXhOcEB5JJJ4egOLPKCQY9CeV8O~15K61pjBrSBPmslu093iKueF8UitvJuzjN51KBK3CDvIqJNgRY8wIdQnQ6n9fkQHcLBHCCucXiB-rCbfRcYeS6TIqzX5lG2ZonA7PuJFGHISvn9TUClCOT7bCRKtGzd~XwsnHJR~t4vVlC9hj7buMeg9~ineE7c7jHEsJMkM6eeybraGeEPFeyQy7j69QIPZLkj05s8KGdKpb276H-yEtX2igjRAl5BLGY1fqg7he5ujPVgEN1KqfAa6fnX0IYkvGkQLkGq6v5LdyIlhJwX4LwG4ThtN4E2yrb9aV~JiEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03714.gif" /> </td><td>TC229652 </td><td>Cleavage </td><td>Unknown </td></tr><tr><td>hvu-miRX29 </td><td>1463 </td><td>765 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03715.gif?Expires=2147483647&amp;Signature=1W1Zkk3nXfqA0grTu8W6P7XdKU1ZqGdMFCBsfhRF3VgHMC-ivIlxMU7rrIQqJ~mPRhBUj0IgN2ObgZa49tzhWXnquaIPhEN2aFgGX3D7YGWKYcV3J3udNhU3fcz4-Kguvqrtj1pwu1TLIDxNS9OHv8DBCt2BAht6SplTZVFjJJFulTCS5srER4QMP4VPoa6L3~F1noNRtKY-JNC4fYdcxlwbeqz59FMiGjhMUnPgvhM39QTkc9o7xekLvTaSe~EWYLy4ZD3dEZ9QNX9uYMhwLWwanil64udaAeL-jfXv8-Fl-NB7oRHuPIlY8veojRYD4y1CZfvwT23IHdiWkJeQJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03715.gif" /> </td><td>TC195723 </td><td>Cleavage </td><td>Pathogenesis-related protein 4 precursor </td></tr><tr><td>hvu-miRX30 </td><td>585 </td><td>392 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03716.gif?Expires=2147483647&amp;Signature=PDIA6A86AuUhpTggTyrkQNMHljc~IpUQq2mMrtyopdn41i3DmOFJhUIBH4aR5cFpO6eVjRHZH4h2YJpyqU7eHjjZr9BUbyEYflVoAd~A-Pg4FURZ0SYYoIYs~VCnPNzzqLEc3-73qbPsMAkl~j0bUKosfXSYqxg8BrfiuT6TsQace4OKpLrZih~8uwWcCuB416MfKTZvbSpxVT5oPqrycSfXIq3L-03rUzNgF2UxHmiuzaRZ3CeEpjs-0tIdDYQ59UERsIC7NRJbTv6AhZ7gTojOXT-fdfumz2Cdv78lT4pc7YasFrNFKYVRl60g8neK68RwUlOBJ2XDfD8-qFq14A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03716.gif" /> </td><td>TC214946 </td><td>Cleavage </td><td>Phospholipid hydroperoxide glutathione peroxidase-like protein </td></tr><tr><td>hvu-miRX31 </td><td>360 </td><td>269 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03717.gif?Expires=2147483647&amp;Signature=zkqD7VLpyF44-0Vv0J~UWULHHX3giXRqEiav7JkScHinth0rlZAi3jcEx5MRmUNeqZGdpqx3UKzG~i-tW~2s4rfpTkFmb5CfOK6gnXJdQ9lKS1buu4wZDloErnvc2amFImhrubx~Vx8LsX-Sr0p4XCarh3ao9Khaf2WTLSSoXBJ-DmLbcxMb1pOLHmiWGlwVdSX~kahi3a3Y3ZxqNSiyJq2sCMT6hoTcChRMQ~e9sn0JzAm18FpcKVj8Phjrsl2j4FVdw4WXNG8XJw3~KFOr4~gGjIolC0bZqRZda6OBNlrWUTUN31txAk5kqt0Ayz122U~7VI6A5mNw9PzIEtX75Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03717.gif" /> </td><td>TC196806 </td><td>Cleavage (four sites) </td><td>AMP-binding enzyme family protein expressed </td></tr><tr><td>hvu-miRX32 </td><td>193 </td><td>92 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03718.gif?Expires=2147483647&amp;Signature=Dmnaeye4Sv86u-Q3nB7gPrNduYrpdpT2L-g7Jbig3FVvzcG29Awzliuuth20IAg0zXgzyynvMY69xCU8WsDQ8n7jbCPxDBcJ13sc~nRojImNscz72-37DH9yOIHIIxpv55AZ5cDQqVh9TvccUqMyThpINT4vezrjhP9N5g9jPEgag7C0vt9OzMILzb8q02cFyx8XLQgHeh~a5LvFebTXeR4MoB3sOKmZjHsCvF8Sj6ANmT6njYtfxtcUMilZLcpZQt8tbPPCO9RyDxUhOvLZY9wtr4SGV4fKZjp01tZHWbh9-Lbg~U5d5TbkkrPe4sqwTTO1oc09OxvGMj-r1aC0eQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dss03718.gif" /> </td><td>EX593121 </td><td><ul class="list-simple"><li><p>Translation inhibition</p></li><li><p>Cleavage</p></li></ul> </td><td>Unknown </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>‘Count–P’ and ‘count +P’ indicate read counts from P-deficient and P-sufficient barley shoots, respectively. Targets were predicted using psRNAtarget (<a class="link link-uri" href="http://plantgrn.noble.org/psRNATarget/">http://plantgrn.noble.org/psRNATarget/</a>). Cleavage sites were determined using degradome library.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600257" target="_blank">View Large</a></div></div><p>Most of the predicted novel miRNAs were 20–21 nt in length and had a preference for A at the 5′ end (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>), which is different from conserved miRNAs that have a preference for U at the 5′ end.<sup><a reveal-id="DSS037C28" data-open="DSS037C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSS037C39" data-open="DSS037C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSS037C40" data-open="DSS037C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Most of the predicted novel miRNAs were expressed at higher levels in P-deficient shoots than in P-sufficient shoots (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>). Three miRNAs (hvu-miRX25, hvu-miRX29 and hvu-miRX32) were significantly up-regulated in P-deficient shoots (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>).</p>                            <h3 scrollto-destination=4600304 id="4600304" class="section-title" data-legacy-id=s3f>3.6. Validation of predicted novel miRNAs</h3>
<p>Northern blot hybridization was used to validate the predicted novel miRNAs. Two (hvu-miRX21 and hvu-miRX22) out of 3 selected novel miRNA candidates (hvu-miRX21, hvu-miRX22 and hvu-miRX27) generated a clear hybridization signal of expected size in leaves, but not in roots, for both P treatments (Fig. <a reveal-id="DSS037F3" data-open="DSS037F3" class="link link-reveal link-table xref-fig">3</a>). RT-PCR was used to further confirm the miRNA candidates. All the candidates generated specific products of expected size in shoot tissue (Fig. <a reveal-id="DSS037F4" data-open="DSS037F4" class="link link-reveal link-table xref-fig">4</a>). Hvu-miRX27 appeared to be more highly expressed in P-sufficient shoots than in P-deficient shoots, whereas hvu-miRX21 showed similar yields between P-deficient and P–sufficient shoots. </p>                        <a id="4600310" scrollto-destination="4600310"></a>
<div id="DSS037F4" data-id="DSS037F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03704.jpeg?Expires=2147483647&Signature=Ptzvxchh22ntd9f4cqE0aY~CgqeqmOHLlqCQ36wt-OF--wwPrANmO73NOoGasjFA7r7aaUZelWPuQfMiz9V3a6wkVf9Q0T6N3QDvp0KROQLUEpSBXseZQUyBJ1uPSDswZTk5~gruGenwy9GJYNmx1wW0BkUB~lc-TLqt7obUDlzINQ0tR8LY-rOFcdKsUzNz4nyENlIHtLHe2LM5YZhb4NWOeLLUdJxSUCNTrfi5JyF2bfOirkHqAZIPlnI677FjEPUZ6AFfJ9z9~b0AeiAy0gNR-vnz~UgDG7j6Xf1CxcY3nwMlfO6IBry29Xa-0NCJnxI5Ay5eMASuoUrgK9XGqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT-PCR expression analysis of predicted novel miRNAs. (A) Sequences were used to amplify predicted novel miRNA precursor sequences with primers (underlined). Mature miRNA sequences are highlighted in red. P1 and P3 are forward primers while P2 and P4 are reverse primers indicated in Italics. (B) Amplification of the cDNAs synthesized from total RNA extracted from P-deficient (−P) and P-sufficient (+P) shoots was performed with primers as shown in (A). The samples, primers and sizes of RT-PCR products are all indicated. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600310/dss03704.jpeg" path-from-xml="dss03704.jpeg" target="_blank">View large</a><a section="4600310" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03704.jpeg?Expires=2147483647&Signature=1JTLV514nXm5BhWSs5Em2wwMv7~ed~vXMb~CPkAgfrpJeKLumsTINjm3BNJ8waBruvJ7G9ww9pRa54Rs7fgmztsE9-iLfe882n3r88q6560j~oc8yEpL~2XLA1x2vNMUUnG0cCvqOl3lDtKyMUU33rX82j6FoEhBv41P8CInN4FhOPHKFblsHPq6G9UdP-BWZ6bHINk9lobvvC7gIj21QhYapmywdWKFiyHpPfYwpLlPLz~PtBAAg-ncgUlQK4AMv60GRKFy3uKWN68BQ~Xb0oWFmlYeJzUf35Ur4umHHLx7pYONtCutz1vIxpxN3HDg5qrEUtuQnbaD3q6oIdx3TQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600310&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT-PCR expression analysis of predicted novel miRNAs. (A) Sequences were used to amplify predicted novel miRNA precursor sequences with primers (underlined). Mature miRNA sequences are highlighted in red. P1 and P3 are forward primers while P2 and P4 are reverse primers indicated in Italics. (B) Amplification of the cDNAs synthesized from total RNA extracted from P-deficient (−P) and P-sufficient (+P) shoots was performed with primers as shown in (A). The samples, primers and sizes of RT-PCR products are all indicated. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS037F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03704.jpeg?Expires=2147483647&Signature=Ptzvxchh22ntd9f4cqE0aY~CgqeqmOHLlqCQ36wt-OF--wwPrANmO73NOoGasjFA7r7aaUZelWPuQfMiz9V3a6wkVf9Q0T6N3QDvp0KROQLUEpSBXseZQUyBJ1uPSDswZTk5~gruGenwy9GJYNmx1wW0BkUB~lc-TLqt7obUDlzINQ0tR8LY-rOFcdKsUzNz4nyENlIHtLHe2LM5YZhb4NWOeLLUdJxSUCNTrfi5JyF2bfOirkHqAZIPlnI677FjEPUZ6AFfJ9z9~b0AeiAy0gNR-vnz~UgDG7j6Xf1CxcY3nwMlfO6IBry29Xa-0NCJnxI5Ay5eMASuoUrgK9XGqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT-PCR expression analysis of predicted novel miRNAs. (A) Sequences were used to amplify predicted novel miRNA precursor sequences with primers (underlined). Mature miRNA sequences are highlighted in red. P1 and P3 are forward primers while P2 and P4 are reverse primers indicated in Italics. (B) Amplification of the cDNAs synthesized from total RNA extracted from P-deficient (−P) and P-sufficient (+P) shoots was performed with primers as shown in (A). The samples, primers and sizes of RT-PCR products are all indicated. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600310/dss03704.jpeg" path-from-xml="dss03704.jpeg" target="_blank">View large</a><a section="4600310" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03704.jpeg?Expires=2147483647&Signature=1JTLV514nXm5BhWSs5Em2wwMv7~ed~vXMb~CPkAgfrpJeKLumsTINjm3BNJ8waBruvJ7G9ww9pRa54Rs7fgmztsE9-iLfe882n3r88q6560j~oc8yEpL~2XLA1x2vNMUUnG0cCvqOl3lDtKyMUU33rX82j6FoEhBv41P8CInN4FhOPHKFblsHPq6G9UdP-BWZ6bHINk9lobvvC7gIj21QhYapmywdWKFiyHpPfYwpLlPLz~PtBAAg-ncgUlQK4AMv60GRKFy3uKWN68BQ~Xb0oWFmlYeJzUf35Ur4umHHLx7pYONtCutz1vIxpxN3HDg5qrEUtuQnbaD3q6oIdx3TQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600310&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT-PCR expression analysis of predicted novel miRNAs. (A) Sequences were used to amplify predicted novel miRNA precursor sequences with primers (underlined). Mature miRNA sequences are highlighted in red. P1 and P3 are forward primers while P2 and P4 are reverse primers indicated in Italics. (B) Amplification of the cDNAs synthesized from total RNA extracted from P-deficient (−P) and P-sufficient (+P) shoots was performed with primers as shown in (A). The samples, primers and sizes of RT-PCR products are all indicated. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div>                            <h3 scrollto-destination=4600319 id="4600319" class="section-title" data-legacy-id=s3g>3.7. Predicted miRNA targets and validation</h3>
<p>MiRNA targets were predicted using psRNAtarget. All the novel miRNA candidates had their targets found in the HVGI database (Release 11). Some targets had known function, whereas others were not annotated (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>). In addition, some targets were targeted by abundant miRNAs, whereas others were targeted only by scarce miRNAs (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>). To validate the predicted targets, which could in turn confirm the predicted novel miRNAs, we constructed a degradome library based on the miRNA-guided cleavage of the target between the 10th and 11th nt of complementarity relative to the guiding miRNA. This library has advantages over the 5′-RACE approach that requires <em>a priori</em> miRNA target prediction and only tests one target at a time. This library has been deposited in NCBI with accession number SRS376178. Three predicted novel miRNAs, hvu-miR23, hvu-miR27 and hvu-miR31, out of 12 from P-deficient and P-sufficient shoots were perfectly reverse complementary to the first 10 nt of 3 sequence fragments in the library (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>), providing potential evidence of miRNA-mediated cleavage. Intriguingly, all these three target genes were cleaved more than twice at different positions (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>). This suggests that these miRNAs may have diverted cleavage sites. It is of notice that one target (DN159928) is regulated by the miRNA at both translational and transcriptional levels (Table <a reveal-id="DSS037TB4" data-open="DSS037TB4" class="link link-reveal link-table xref-fig">4</a>).</p>                            <h3 scrollto-destination=4600328 id="4600328" class="section-title" data-legacy-id=s3h>3.8. Profiles of other classes of nuclear-derived ncRNAs in P-deficient and P-sufficient shoots</h3>
<p>To explore the profiles of other classes of nuclear-derived ncRNAs in Pallas P-deficient and P-sufficient shoots, all the non-miRNA reads were aligned to the Rfam database. Seven hundred and ninety six and 715 ncRNAs were mapped in P-deficient and P-sufficient shoots, respectively, of which 114 ncRNAs were specific to P-deficient shoots, whereas 32 ncRNAs were specific to P-sufficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Of 683 shared ncRNAs, 176 were up-regulated whereas 51 were down-regulated in P-deficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The most abundant ncRNAs were small subunit rRNA 5-derived sRNAs, followed by tsRNAs in both P treatments. The two classes of sRNAs accounted for 80% of the mapped ncRNAs in P-deficient shoots and 76.3% in P-sufficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>tsRNAs were further dissected by aligning all the non-miRNA reads to tRNA databases. Fifty-six out of the total 61 tRNAs were found to generate tsRNAs in both P-deficient and P-sufficient shoots. These 56 tRNAs specify all essential 20 amino acids encoded by the standard genetic code (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Six tsRNAs were significantly up-regulated (log<sub>2</sub> &gt; 1), whereas 4 were significantly down-regulated (log<sub>2</sub> &lt;–1), in P-deficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Notably, the read count of tsRNAs was 2-fold greater in P-deficient shoots than in P-sufficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>tsRNAs derived from tRNA-Gly(TCC) were the most abundant tsRNA species in both P treatments, accounting for 58.6% of the total tsRNAs in P-deficient shoots and 58.2% in P-sufficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). tRNA-Ala(AGC)-derived sRNAs were the second most abundant species in P-sufficient shoots, but the third in P-deficient shoots. The second most abundant tsRNA in P-deficient shoots was tRNA-Arg(CCT)-derived sRNAs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The relative proportions of the other tsRNAs did not appear to be consistent between the two P treatments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>Next, we analysed rasiRNAs by aligning all the non-miRNA reads to the TREP database that contains 1716 transposable elements (TEs). TEs are a major source of siRNAs<sup><a reveal-id="DSS037C41" data-open="DSS037C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSS037C42" data-open="DSS037C42" class="link link-ref link-reveal xref-bibr">42</a></sup> that in turn suppress the mutagenic activity of TEs.<sup><a reveal-id="DSS037C43" data-open="DSS037C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The sRNAs derived from 685 TEs in P-deficient shoots and the sRNAs derived from 582 TEs in P-sufficient shoots were identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Of them, 581 TE-derived sRNAs were common between the two P treatments. Almost every TE-derived sRNA was less abundant in P-sufficient shoots than in P-deficient shoots, with half of these significantly down-regulated (log<sub>2</sub> &gt;–1) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). There are 59.8% of the TE-derived sRNAs being mapped to retrotransposons (Class I TEs), 32.6% to DNA transposons (Class II TEs) and 7.6% to unclassified TEs in P-sufficient shoots (Table <a reveal-id="DSS037TB5" data-open="DSS037TB5" class="link link-reveal link-table xref-fig">5</a>). Coincidentally, 53% of the TE-derived sRNAs were mapped to retrotransposons, 39.6% to DNA transposons and 7.3% to unclassified TEs in P-deficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The percentage of each class of TE-derived sRNAs was similar between the two P treatments. Similar results were also obtained when the reads were aligned to other repetitive databases such as Repbase, the TIGR rice repeat database and the TIGR barley repeat database (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). </p>                        <a id="4600340" scrollto-destination="4600340"></a>
<div content-id="DSS037TB5" class="table-modal"><div class="table-wrap"><div id="DSS037TB5" class="table-wrap-title" data-id="DSS037TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Summary of mapped TE-derived sRNAs in P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>Retrotransposon </td><td>4543 </td><td>8934 </td><td>6809.329 </td><td>0.583867 </td></tr><tr><td>DNA transposon </td><td>2482 </td><td>6683 </td><td>5093.658 </td><td>1.037199 </td></tr><tr><td>Unknown </td><td>577 </td><td>1234 </td><td>940.5318 </td><td>0.704905 </td></tr><tr><td>LTR </td><td>4366 </td><td>8473 </td><td>6457.963 </td><td>0.564767 </td></tr><tr><td>TIR </td><td>2469 </td><td>6660 </td><td>5076.128 </td><td>1.039802 </td></tr><tr><td>Unknown </td><td>582 </td><td>1238 </td><td>943.5806 </td><td>0.697127 </td></tr><tr><td>LINE </td><td>167 </td><td>436 </td><td>332.3111 </td><td>0.992686 </td></tr><tr><td>SINE </td><td>10 </td><td>25 </td><td>19.05454 </td><td>0.930134 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>Copia </td><td>2421 </td><td>4580 </td><td>3490.791 </td><td>0.527951 </td></tr><tr><td>Gypsy </td><td>1843 </td><td>3599 </td><td>2743.091 </td><td>0.573746 </td></tr><tr><td>CACTA </td><td>1387 </td><td>3316 </td><td>2527.394 </td><td>0.865683 </td></tr><tr><td>Unknown </td><td>861 </td><td>1993 </td><td>1519.028 </td><td>0.819063 </td></tr><tr><td>Mariner </td><td>559 </td><td>1432 </td><td>1091.444 </td><td>0.965318 </td></tr><tr><td>Harbinger </td><td>414 </td><td>1616 </td><td>1231.685 </td><td>1.572931 </td></tr><tr><td>Mutator </td><td>103 </td><td>282 </td><td>214.9352 </td><td>1.061257 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>HAT </td><td>6 </td><td>14 </td><td>10.67054 </td><td>0.830599 </td></tr></tbody></table></div><div content-id="DSS037TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>Retrotransposon </td><td>4543 </td><td>8934 </td><td>6809.329 </td><td>0.583867 </td></tr><tr><td>DNA transposon </td><td>2482 </td><td>6683 </td><td>5093.658 </td><td>1.037199 </td></tr><tr><td>Unknown </td><td>577 </td><td>1234 </td><td>940.5318 </td><td>0.704905 </td></tr><tr><td>LTR </td><td>4366 </td><td>8473 </td><td>6457.963 </td><td>0.564767 </td></tr><tr><td>TIR </td><td>2469 </td><td>6660 </td><td>5076.128 </td><td>1.039802 </td></tr><tr><td>Unknown </td><td>582 </td><td>1238 </td><td>943.5806 </td><td>0.697127 </td></tr><tr><td>LINE </td><td>167 </td><td>436 </td><td>332.3111 </td><td>0.992686 </td></tr><tr><td>SINE </td><td>10 </td><td>25 </td><td>19.05454 </td><td>0.930134 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>Copia </td><td>2421 </td><td>4580 </td><td>3490.791 </td><td>0.527951 </td></tr><tr><td>Gypsy </td><td>1843 </td><td>3599 </td><td>2743.091 </td><td>0.573746 </td></tr><tr><td>CACTA </td><td>1387 </td><td>3316 </td><td>2527.394 </td><td>0.865683 </td></tr><tr><td>Unknown </td><td>861 </td><td>1993 </td><td>1519.028 </td><td>0.819063 </td></tr><tr><td>Mariner </td><td>559 </td><td>1432 </td><td>1091.444 </td><td>0.965318 </td></tr><tr><td>Harbinger </td><td>414 </td><td>1616 </td><td>1231.685 </td><td>1.572931 </td></tr><tr><td>Mutator </td><td>103 </td><td>282 </td><td>214.9352 </td><td>1.061257 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>HAT </td><td>6 </td><td>14 </td><td>10.67054 </td><td>0.830599 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600340" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS037TB5" class="table-wrap-title" data-id="DSS037TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Summary of mapped TE-derived sRNAs in P-deficient and P-sufficient shoots of Pallas barley</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>Retrotransposon </td><td>4543 </td><td>8934 </td><td>6809.329 </td><td>0.583867 </td></tr><tr><td>DNA transposon </td><td>2482 </td><td>6683 </td><td>5093.658 </td><td>1.037199 </td></tr><tr><td>Unknown </td><td>577 </td><td>1234 </td><td>940.5318 </td><td>0.704905 </td></tr><tr><td>LTR </td><td>4366 </td><td>8473 </td><td>6457.963 </td><td>0.564767 </td></tr><tr><td>TIR </td><td>2469 </td><td>6660 </td><td>5076.128 </td><td>1.039802 </td></tr><tr><td>Unknown </td><td>582 </td><td>1238 </td><td>943.5806 </td><td>0.697127 </td></tr><tr><td>LINE </td><td>167 </td><td>436 </td><td>332.3111 </td><td>0.992686 </td></tr><tr><td>SINE </td><td>10 </td><td>25 </td><td>19.05454 </td><td>0.930134 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>Copia </td><td>2421 </td><td>4580 </td><td>3490.791 </td><td>0.527951 </td></tr><tr><td>Gypsy </td><td>1843 </td><td>3599 </td><td>2743.091 </td><td>0.573746 </td></tr><tr><td>CACTA </td><td>1387 </td><td>3316 </td><td>2527.394 </td><td>0.865683 </td></tr><tr><td>Unknown </td><td>861 </td><td>1993 </td><td>1519.028 </td><td>0.819063 </td></tr><tr><td>Mariner </td><td>559 </td><td>1432 </td><td>1091.444 </td><td>0.965318 </td></tr><tr><td>Harbinger </td><td>414 </td><td>1616 </td><td>1231.685 </td><td>1.572931 </td></tr><tr><td>Mutator </td><td>103 </td><td>282 </td><td>214.9352 </td><td>1.061257 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>HAT </td><td>6 </td><td>14 </td><td>10.67054 </td><td>0.830599 </td></tr></tbody></table></div><div content-id="DSS037TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>Retrotransposon </td><td>4543 </td><td>8934 </td><td>6809.329 </td><td>0.583867 </td></tr><tr><td>DNA transposon </td><td>2482 </td><td>6683 </td><td>5093.658 </td><td>1.037199 </td></tr><tr><td>Unknown </td><td>577 </td><td>1234 </td><td>940.5318 </td><td>0.704905 </td></tr><tr><td>LTR </td><td>4366 </td><td>8473 </td><td>6457.963 </td><td>0.564767 </td></tr><tr><td>TIR </td><td>2469 </td><td>6660 </td><td>5076.128 </td><td>1.039802 </td></tr><tr><td>Unknown </td><td>582 </td><td>1238 </td><td>943.5806 </td><td>0.697127 </td></tr><tr><td>LINE </td><td>167 </td><td>436 </td><td>332.3111 </td><td>0.992686 </td></tr><tr><td>SINE </td><td>10 </td><td>25 </td><td>19.05454 </td><td>0.930134 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>Copia </td><td>2421 </td><td>4580 </td><td>3490.791 </td><td>0.527951 </td></tr><tr><td>Gypsy </td><td>1843 </td><td>3599 </td><td>2743.091 </td><td>0.573746 </td></tr><tr><td>CACTA </td><td>1387 </td><td>3316 </td><td>2527.394 </td><td>0.865683 </td></tr><tr><td>Unknown </td><td>861 </td><td>1993 </td><td>1519.028 </td><td>0.819063 </td></tr><tr><td>Mariner </td><td>559 </td><td>1432 </td><td>1091.444 </td><td>0.965318 </td></tr><tr><td>Harbinger </td><td>414 </td><td>1616 </td><td>1231.685 </td><td>1.572931 </td></tr><tr><td>Mutator </td><td>103 </td><td>282 </td><td>214.9352 </td><td>1.061257 </td></tr><tr><td>Helitron </td><td>8 </td><td>19 </td><td>14.48145 </td><td>0.856134 </td></tr><tr><td>HAT </td><td>6 </td><td>14 </td><td>10.67054 </td><td>0.830599 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600340" target="_blank">View Large</a></div></div><p>To analyse natsiRNAs, another source of siRNAs that are capable of blocking the translation of mRNAs or degrading mRNAs through the siRNA pathway,<sup><a reveal-id="DSS037C44" data-open="DSS037C44" class="link link-ref link-reveal xref-bibr">44</a></sup> all non-miRNA reads were firstly aligned to the reverse strand of barley genes available in the databases, and the mapped reads were then aligned to the forward strand of all other libraries, including tRNA, genes, repeats and so on. In total, 290 234 and 303 953 natsiRNAs were identified in P-deficient and P-sufficient shoots, respectively, of which many were treatment specific (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).</p><p>Some natsiRNAs were found to target multiple genes, whereas others target only a single gene (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The latter accounted for 80.6% of the total mapped genes in P-deficient shoots and 78.2% in P-sufficient shoots. A great number of natsiRNAs were derived from TEs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>), which is consistent with previous studies.<sup><a reveal-id="DSS037C45" data-open="DSS037C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Many natsiRNAs formed duplexes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).</p><p>The discovery of siRNA duplexes prompted us to investigate sense siRNAs. Alignment of all the nuclear-derived reads to barley genes in the HVGI database identified 128 corresponding barley genes in P-sufficient shoots and 137 in P-deficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S5</a> under non-chloroplast sense sRNAs). Some genes had more associated reads than other genes. In addition, more sense siRNAs were present in P-deficient shoots than in P-sufficient shoots. The expression levels of sense siRNAs were also higher in P-deficient shoots than in P-sufficient shoots.</p>                            <h3 scrollto-destination=4600348 id="4600348" class="section-title" data-legacy-id=s3i>3.9. Profile of csRNAs in P-deficient and P-sufficient shoots</h3>
<p>CsRNAs in the two P treatments were further analysed in detail. The result showed that the csRNAs were derived from all regions of the chloroplast genome, including CDS (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> under chloroplast sRNAs_CDS), tRNA genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> under chloroplast sRNAs_tRNA) and repetitive sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> under chloroplast sRNAs_rep). The chloroplast-derived tsRNAs were the most abundant sRNA class, accounting for 22.7% of the total sRNAs in P-deficient shoots and 19% in P-sufficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> under chloroplast sRNAs_tRNA). One tsRNA derived from the tRNA-His(GUG) gene was the most abundant of all the sRNAs (including miRNAs) sequenced. This tsRNA accounted for 84.5% of the total tsRNAs identified and 51.2% of the total csRNAs. This result explains why csRNAs are highly redundant when compared with sRNAs derived from the nuclear genome. Curiously, this most abundant tsRNA was only present in barley leaves and not in roots (Fig. <a reveal-id="DSS037F5" data-open="DSS037F5" class="link link-reveal link-table xref-fig">5</a>A) and is perfectly reversely complementary to a gene involved in biological processes. The absence or low expression of this tsRNA in the roots may be due to reduced plastid copy numbers in roots compared to those in leaves.<sup><a reveal-id="DSS037C46" data-open="DSS037C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                        <a id="4600353" scrollto-destination="4600353"></a>
<div id="DSS037F5" data-id="DSS037F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03705.jpeg?Expires=2147483647&Signature=E~ZynujpgOMQu7Xp6tHzQcBGjnHXzVVTfU2aUAdmsfBdQtMxKHX954fu89pSw40Q6NNrARdjoWJvsI4CkjCE~kYTpt0CDu0uqAJOXYJU0L93dhWexewEJdM1A~OwJmS44OvedGdxUrcpgHQsSIxo69uV0w06OHxwbyq1qWnd5JuoizspCvtusL6J-I1TO8o-sx6nf~kJyuWwsWQPnHKKeEZCu1VsX~6bEe46nxIm9BZleMhJbqxlH0uzn2UtqsOJXFc~WjHA7iofAzZoZqeD88uDJMupasKxJGzgGMEmLGtaUMcBihJYYksm6lWulzCng3toxfeDEteL3CaWEex~5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Northern blot hybridization of tsRNAs derived from tRNA-His(GTG) and read distribution on four tRNA genes in P-deficient (−P) and P-sufficient (+P) shoots of barley. (A) Total RNA extracted from leaf and root tissues from P-deficient and P-sufficient barley was hybridized with probes reversely complementary to the most abundant tsRNA derived from tRNA-His(GTG). Hybridization with U6 probe is used as a loading control. (B) All the reads obtained were aligned with each of the four most abundant tRNA genes. The names of tRNA genes, positions of reads on the tRNA genes and read fraction are indicated. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600353/dss03705.jpeg" path-from-xml="dss03705.jpeg" target="_blank">View large</a><a section="4600353" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03705.jpeg?Expires=2147483647&Signature=Zh2uPW8x7G2ptQVUgK69uwHaFhnzMfolIjL575ustGytoQr1bKgfrNgIUdk-tUoNpIIPsyAXCpjjTVIHhlD5XLbOJ2~wVUIlAquwrRFrYOxXW~prlqfAhqZFuLmdfYCTAhbrc59IU4JCq4q7tkiRmxE56MR-v6e5t01H0eWNRFASVMT2ivHs2aKvF1AHpnbkFfNo9cPKR13-a1wVXZB1ovN9wsws1bxEtjGQ8hKtL6v6SMM6zoNLITXkwaSUXsz8C4aQQGdib0zHY9jFbMoMFBz8pUM7kwVkvZjHU3RjmvI72KC0JY2rCfKZcHinYYjRxC6sBCUeyADTEe9K57i55Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600353&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Northern blot hybridization of tsRNAs derived from tRNA-His(GTG) and read distribution on four tRNA genes in P-deficient (−P) and P-sufficient (+P) shoots of barley. (A) Total RNA extracted from leaf and root tissues from P-deficient and P-sufficient barley was hybridized with probes reversely complementary to the most abundant tsRNA derived from tRNA-His(GTG). Hybridization with U6 probe is used as a loading control. (B) All the reads obtained were aligned with each of the four most abundant tRNA genes. The names of tRNA genes, positions of reads on the tRNA genes and read fraction are indicated. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS037F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/m_dss03705.jpeg?Expires=2147483647&Signature=E~ZynujpgOMQu7Xp6tHzQcBGjnHXzVVTfU2aUAdmsfBdQtMxKHX954fu89pSw40Q6NNrARdjoWJvsI4CkjCE~kYTpt0CDu0uqAJOXYJU0L93dhWexewEJdM1A~OwJmS44OvedGdxUrcpgHQsSIxo69uV0w06OHxwbyq1qWnd5JuoizspCvtusL6J-I1TO8o-sx6nf~kJyuWwsWQPnHKKeEZCu1VsX~6bEe46nxIm9BZleMhJbqxlH0uzn2UtqsOJXFc~WjHA7iofAzZoZqeD88uDJMupasKxJGzgGMEmLGtaUMcBihJYYksm6lWulzCng3toxfeDEteL3CaWEex~5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Northern blot hybridization of tsRNAs derived from tRNA-His(GTG) and read distribution on four tRNA genes in P-deficient (−P) and P-sufficient (+P) shoots of barley. (A) Total RNA extracted from leaf and root tissues from P-deficient and P-sufficient barley was hybridized with probes reversely complementary to the most abundant tsRNA derived from tRNA-His(GTG). Hybridization with U6 probe is used as a loading control. (B) All the reads obtained were aligned with each of the four most abundant tRNA genes. The names of tRNA genes, positions of reads on the tRNA genes and read fraction are indicated. This figure appears in colour in the online version of DNA Research." path-from-xml="dss03705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4600353/dss03705.jpeg" path-from-xml="dss03705.jpeg" target="_blank">View large</a><a section="4600353" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss03705.jpeg?Expires=2147483647&Signature=Zh2uPW8x7G2ptQVUgK69uwHaFhnzMfolIjL575ustGytoQr1bKgfrNgIUdk-tUoNpIIPsyAXCpjjTVIHhlD5XLbOJ2~wVUIlAquwrRFrYOxXW~prlqfAhqZFuLmdfYCTAhbrc59IU4JCq4q7tkiRmxE56MR-v6e5t01H0eWNRFASVMT2ivHs2aKvF1AHpnbkFfNo9cPKR13-a1wVXZB1ovN9wsws1bxEtjGQ8hKtL6v6SMM6zoNLITXkwaSUXsz8C4aQQGdib0zHY9jFbMoMFBz8pUM7kwVkvZjHU3RjmvI72KC0JY2rCfKZcHinYYjRxC6sBCUeyADTEe9K57i55Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4600353&ar=333081&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Northern blot hybridization of tsRNAs derived from tRNA-His(GTG) and read distribution on four tRNA genes in P-deficient (−P) and P-sufficient (+P) shoots of barley. (A) Total RNA extracted from leaf and root tissues from P-deficient and P-sufficient barley was hybridized with probes reversely complementary to the most abundant tsRNA derived from tRNA-His(GTG). Hybridization with U6 probe is used as a loading control. (B) All the reads obtained were aligned with each of the four most abundant tRNA genes. The names of tRNA genes, positions of reads on the tRNA genes and read fraction are indicated. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>Alignment of all the csRNAs to the tRNA-HIS(GUG) gene revealed that the distribution of csRNAs over the gene was uneven. A peak was located at the 5′ end, where the sequence corresponded to that of the most abundant tsRNA (Fig. <a reveal-id="DSS037F5" data-open="DSS037F5" class="link link-reveal link-table xref-fig">5</a>B), suggesting that the most abundant tsRNA is not randomly generated. Other tsRNAs gave a similar distribution pattern as the tRNA-HIS(GUG)-derived sRNAs (Fig. <a reveal-id="DSS037F5" data-open="DSS037F5" class="link link-reveal link-table xref-fig">5</a>B).</p><p>tsRNAs derived from four tRNA genes, trnR-ACG, trnE-UUC, trnL-CAA and trnL-UAA, were significantly up-regulated (log<sub>2</sub> &gt; 1), whereas tsRNAs derived from the trnS-GCU gene were significantly down-regulated (log<sub>2</sub> &lt; −1), in P-deficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> under chloroplast sRNAs_tRNA). A few other csRNAs derived from the other regions were also found to be significantly differentially expressed (|log<sub>2</sub>| ≥ 1) between the two P treatments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). These results imply that P nutrition is associated with the expression of some csRNAs.</p><p>Unexpectedly, 95 barley genes were mapped by csRNAs in each of the P-deficient and P-sufficient shoots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> under chloroplast sRNAs_HVGI), although this number is lower than that mapped by the nuclear sRNAs (128 in P-sufficient shoots and 137 in P-deficient shoots). One barley gene (TC277237), encoding photosystem II reaction centre protein H, was mapped the most by csRNAs in both P treatments. sRNAs derived from seven barley genes were present in both the chloroplast genome and the nuclear genome but were differentially expressed between the two genomes (Table <a reveal-id="DSS037TB6" data-open="DSS037TB6" class="link link-reveal link-table xref-fig">6</a>). For example, sRNAs derived from the gene TC250483 were more abundant in the nuclear genome than in the chloroplast genome, whereas sRNAs derived from the gene BY843503 were more abundant in the chloroplast genome than in the nuclear genome (Table <a reveal-id="DSS037TB6" data-open="DSS037TB6" class="link link-reveal link-table xref-fig">6</a>). These data suggest that a genome or organelle-specific factor may be involved in the expression of each class of sRNAs. </p>                        <a id="4600362" scrollto-destination="4600362"></a>
<div content-id="DSS037TB6" class="table-modal"><div class="table-wrap"><div id="DSS037TB6" class="table-wrap-title" data-id="DSS037TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Shared gene-derived sRNAs between chloroplast and nucleus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="5">Mapped barley genes from nuclear<hr /> </th><th colspan="5">Mapped barley genes from chloroplast<hr /> </th></tr><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>TC250483 </td><td>235 </td><td>226 </td><td>172.253 </td><td>−0.44813 </td><td>TC250483 </td><td>20 </td><td>35 </td><td>23.22616 </td><td>0.215751 </td></tr><tr><td>BY848620 </td><td>31 </td><td>35 </td><td>26.67635 </td><td>−0.21671 </td><td>BY848620 </td><td>111 </td><td>73 </td><td>48.44313 </td><td>−1.1962 </td></tr><tr><td>BM097437 </td><td>20 </td><td>21 </td><td>16.00581 </td><td>−0.3214 </td><td>BM097437 </td><td>112 </td><td>84 </td><td>55.74278 </td><td>−1.00664 </td></tr><tr><td>BY841253 </td><td>17 </td><td>10 </td><td>7.621814 </td><td>−1.15733 </td><td>BY841253 </td><td>526 </td><td>312 </td><td>207.0446 </td><td>−1.34512 </td></tr><tr><td>BY843503 </td><td>14 </td><td>34 </td><td>25.91417 </td><td>0.888314 </td><td>BY843503 </td><td>1329 </td><td>2281 </td><td>1513.682 </td><td>0.187721 </td></tr><tr><td>TC272545 </td><td>10 </td><td>11 </td><td>8.383995 </td><td>−0.25429 </td><td>TC272545 </td><td>25 </td><td>265 </td><td>175.8552 </td><td>2.814388 </td></tr><tr><td>TC279709 </td><td>7 </td><td>10 </td><td>7.621814 </td><td>0.122779 </td><td>TC279709 </td><td>18 </td><td>21 </td><td>13.9357 </td><td>−0.36921 </td></tr></tbody></table></div><div content-id="DSS037TB6" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="5">Mapped barley genes from nuclear<hr /> </th><th colspan="5">Mapped barley genes from chloroplast<hr /> </th></tr><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>TC250483 </td><td>235 </td><td>226 </td><td>172.253 </td><td>−0.44813 </td><td>TC250483 </td><td>20 </td><td>35 </td><td>23.22616 </td><td>0.215751 </td></tr><tr><td>BY848620 </td><td>31 </td><td>35 </td><td>26.67635 </td><td>−0.21671 </td><td>BY848620 </td><td>111 </td><td>73 </td><td>48.44313 </td><td>−1.1962 </td></tr><tr><td>BM097437 </td><td>20 </td><td>21 </td><td>16.00581 </td><td>−0.3214 </td><td>BM097437 </td><td>112 </td><td>84 </td><td>55.74278 </td><td>−1.00664 </td></tr><tr><td>BY841253 </td><td>17 </td><td>10 </td><td>7.621814 </td><td>−1.15733 </td><td>BY841253 </td><td>526 </td><td>312 </td><td>207.0446 </td><td>−1.34512 </td></tr><tr><td>BY843503 </td><td>14 </td><td>34 </td><td>25.91417 </td><td>0.888314 </td><td>BY843503 </td><td>1329 </td><td>2281 </td><td>1513.682 </td><td>0.187721 </td></tr><tr><td>TC272545 </td><td>10 </td><td>11 </td><td>8.383995 </td><td>−0.25429 </td><td>TC272545 </td><td>25 </td><td>265 </td><td>175.8552 </td><td>2.814388 </td></tr><tr><td>TC279709 </td><td>7 </td><td>10 </td><td>7.621814 </td><td>0.122779 </td><td>TC279709 </td><td>18 </td><td>21 </td><td>13.9357 </td><td>−0.36921 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600362" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS037TB6" class="table-wrap-title" data-id="DSS037TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Shared gene-derived sRNAs between chloroplast and nucleus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="5">Mapped barley genes from nuclear<hr /> </th><th colspan="5">Mapped barley genes from chloroplast<hr /> </th></tr><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>TC250483 </td><td>235 </td><td>226 </td><td>172.253 </td><td>−0.44813 </td><td>TC250483 </td><td>20 </td><td>35 </td><td>23.22616 </td><td>0.215751 </td></tr><tr><td>BY848620 </td><td>31 </td><td>35 </td><td>26.67635 </td><td>−0.21671 </td><td>BY848620 </td><td>111 </td><td>73 </td><td>48.44313 </td><td>−1.1962 </td></tr><tr><td>BM097437 </td><td>20 </td><td>21 </td><td>16.00581 </td><td>−0.3214 </td><td>BM097437 </td><td>112 </td><td>84 </td><td>55.74278 </td><td>−1.00664 </td></tr><tr><td>BY841253 </td><td>17 </td><td>10 </td><td>7.621814 </td><td>−1.15733 </td><td>BY841253 </td><td>526 </td><td>312 </td><td>207.0446 </td><td>−1.34512 </td></tr><tr><td>BY843503 </td><td>14 </td><td>34 </td><td>25.91417 </td><td>0.888314 </td><td>BY843503 </td><td>1329 </td><td>2281 </td><td>1513.682 </td><td>0.187721 </td></tr><tr><td>TC272545 </td><td>10 </td><td>11 </td><td>8.383995 </td><td>−0.25429 </td><td>TC272545 </td><td>25 </td><td>265 </td><td>175.8552 </td><td>2.814388 </td></tr><tr><td>TC279709 </td><td>7 </td><td>10 </td><td>7.621814 </td><td>0.122779 </td><td>TC279709 </td><td>18 </td><td>21 </td><td>13.9357 </td><td>−0.36921 </td></tr></tbody></table></div><div content-id="DSS037TB6" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="5">Mapped barley genes from nuclear<hr /> </th><th colspan="5">Mapped barley genes from chloroplast<hr /> </th></tr><tr><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th><th>Name </th><th>Read count +P </th><th>Read count −P </th><th>−P norm </th><th>Log<sub>2</sub> </th></tr></thead><tbody><tr><td>TC250483 </td><td>235 </td><td>226 </td><td>172.253 </td><td>−0.44813 </td><td>TC250483 </td><td>20 </td><td>35 </td><td>23.22616 </td><td>0.215751 </td></tr><tr><td>BY848620 </td><td>31 </td><td>35 </td><td>26.67635 </td><td>−0.21671 </td><td>BY848620 </td><td>111 </td><td>73 </td><td>48.44313 </td><td>−1.1962 </td></tr><tr><td>BM097437 </td><td>20 </td><td>21 </td><td>16.00581 </td><td>−0.3214 </td><td>BM097437 </td><td>112 </td><td>84 </td><td>55.74278 </td><td>−1.00664 </td></tr><tr><td>BY841253 </td><td>17 </td><td>10 </td><td>7.621814 </td><td>−1.15733 </td><td>BY841253 </td><td>526 </td><td>312 </td><td>207.0446 </td><td>−1.34512 </td></tr><tr><td>BY843503 </td><td>14 </td><td>34 </td><td>25.91417 </td><td>0.888314 </td><td>BY843503 </td><td>1329 </td><td>2281 </td><td>1513.682 </td><td>0.187721 </td></tr><tr><td>TC272545 </td><td>10 </td><td>11 </td><td>8.383995 </td><td>−0.25429 </td><td>TC272545 </td><td>25 </td><td>265 </td><td>175.8552 </td><td>2.814388 </td></tr><tr><td>TC279709 </td><td>7 </td><td>10 </td><td>7.621814 </td><td>0.122779 </td><td>TC279709 </td><td>18 </td><td>21 </td><td>13.9357 </td><td>−0.36921 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4600362" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4600364 id="4600364" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>P is a key determinant of plant growth and development, but its availability is low in the soil. Therefore, finding an effective way to enable cereal crops to take up Pi from low P soils efficiently is desirable. MiRNAs have been shown to be important gene regulators in plant Pi homeostasis. In <em>Arabidopsis</em>, a few of miRNAs have been identified to be regulated by P,<sup><a reveal-id="DSS037C47" data-open="DSS037C47" class="link link-ref link-reveal xref-bibr">47</a></sup> suggesting a close relationship between miRNA and P. To investigate this relationship in barley, we comprehensively analysed sRNA sequence datasets generated by deep sequencing from Pallas barley grown under P-deficient and P-sufficient conditions. Our result showed that about 1 million sRNA sequences out of the total 7 million sequences were specific to either P-deficient or P-sufficient treatment. In addition, over 30% of common sequences were significantly differentially expressed between the two P treatments. These indicate that P has a great impact on the expression of sRNAs. How P affects the expression of sRNAs is unclear, but it likely involves its signalling pathway that results in some genes being activated, whereas others being degraded, under different P conditions, thus leading to different sRNA populations. We observed that more sRNAs were generated under P-deficient condition than under P-sufficient condition. A similar situation has been described previously.<sup><a reveal-id="DSS037C48" data-open="DSS037C48" class="link link-ref link-reveal xref-bibr">48</a></sup> In addition, we observed that the number of individual siRNAs or miRNAs is also more in P-deficient treatment than in P-sufficient treatment. Coincidentally, more mRNA-derived individual sRNAs were found in P-deficient treatment when compared with those in P-sufficient treatment. One model could be used to explain these phenomena: P deficiency causes the instability or degradation of RNA molecules firstly because P is a part of an RNA molecule, and the lack of P reduces its incorporation into the RNA molecules, and then the generated sRNAs are used as siRNAs to guide the cellular RNA degradation machinery.<sup><a reveal-id="DSS037C49" data-open="DSS037C49" class="link link-ref link-reveal xref-bibr">49</a></sup> This has been the case in <em>Drosophila</em><sup><a reveal-id="DSS037C50" data-open="DSS037C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and <em>Arabidopsis</em>.<sup><a reveal-id="DSS037C51" data-open="DSS037C51" class="link link-ref link-reveal xref-bibr">51</a></sup> However, it must be pointed out that such degradation mode is only for sRNAs derived from the 3′, but not 5′, terminal regions.<sup><a reveal-id="DSS037C45" data-open="DSS037C45" class="link link-ref link-reveal xref-bibr">45</a></sup> The 5′ terminal degradation mode from the miRNA/siRNA-induced cleavage has not yet been elucidated so far.</p><p>We have identified a total of 233 miRNAs in Pallas barley, of which some were significantly differentially expressed between the two P treatments (|log<sub>2</sub>| ≥ 1), whereas others were P treatment specific. A total of 214 miRNAs have been detected using cross-species comparison or prediction. In a recent study, 126 conserved miRNAs and 148 barley-specific miRNAs were identified in barley cultivar Clipper.<sup><a reveal-id="DSS037C29" data-open="DSS037C29" class="link link-ref link-reveal xref-bibr">29</a></sup> To add evidence to the identified miRNAs, we compared our set of miRNAs with the ones detected in Clipper. We found that of the total 233 miRNAs identified in Pallas, 94 (40.3%) have also been detected in Clipper (allowing at most 1 mismatch). There is a clear tendency between coexistence (expressed in both Pallas and Clipper) and expression level. The most expressed miRNAs in Pallas are also highly expressed in Clipper, whereas lower expressed miRNAs have a lower probability to co-exist.</p><p>Many miRNAs are known to show a strongly tissue-dependent expression profile. Because the Pallas and Clipper data were generated from different tissues (shoots only in Pallas and a mixed tissues of leaves, stems, roots and spikes in Clipper), the absence of coexistence does not necessarily imply a false positive miRNA detection/prediction. It might be due to the tissue-specific expression profile of many miRNAs. For example, hvu-miR156 is the second highest expressed miRNA in Pallas, whereas the same miRNA in Clipper is only ranked 14. In addition, in Pallas, hvu-miR156 is 0 order of magnitude from the highest expressed hvu-miR168, whereas in Clipper, this miRNA is over two orders of magnitude from the highest expressed hvu-miR167d (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Both the highest expressed hvu-miR168 in Pallas and the highest expressed hvu-miR167d in Clipper have roughly the same scale in expression level (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). This suggests that miR156 might be highly shoot specific. Interestingly, the two highly expressed novel miRNAs (hvu-miRX21 and hvu-miRX22) in Pallas, which have not been detected in any other species so far, are not detected in Clipper either. Another highly expressed miRNA that is absent in Clipper is miR827, which is strongly up-regulated under deficient P (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>) and might, therefore, not be in Clipper barley that grew under a normal (P) nutrition condition.</p><p>tsRNAs were the second most abundant sRNA class in both P treatments, accounting for 22.7% of the total sRNAs in P-deficient treatment and 19% in P-sufficient treatment. The high abundance of tsRNAs has been the case in epimastigotes and trypomastigotes, which accounts for 30 and 40% of the total sRNAs, respectively.<sup><a reveal-id="DSS037C50" data-open="DSS037C50" class="link link-ref link-reveal xref-bibr">50</a></sup> To our surprise, one tsRNA derived from the chloroplast-encoded tRNA-His(GTG) gene is the most abundant sRNA of all the sRNAs sequenced in both P treatments. Recent studies have shown that tsRNAs are Dicer dependent, can be incorporated into the RISC and direct Ago-mediated cleavage.<sup><a reveal-id="DSS037C4" data-open="DSS037C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS037C50" data-open="DSS037C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DSS037C52" data-open="DSS037C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSS037C53" data-open="DSS037C53" class="link link-ref link-reveal xref-bibr">53</a></sup> These suggest that tsRNAs can function in an miRNA-like pathway.<sup><a reveal-id="DSS037C5" data-open="DSS037C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS037C54" data-open="DSS037C54" class="link link-ref link-reveal xref-bibr">54</a></sup> This would then imply that tsRNAs could play a certain role in the cells. Indeed, a tsRNA (tRF-1001) from human prostate cancer cells was recently demonstrated to be required for cell viability through the siRNA/miRNA pathway.<sup><a reveal-id="DSS037C4" data-open="DSS037C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Although tsRNAs are processed by Dicer-like endonucleases and associated with argonautes as miRNAs, they are derived from RNA polymerase III-catalyzed transcripts. In contrast, miRNAs are derived from RNA polymerase II transcripts. In addition, the sizes of tsRNAs are generally larger than those of miRNAs. In bacteria, fungi, plants and animals, the size of tsRNAs ranges from 30 to 35 nt.<sup><a reveal-id="DSS037C4" data-open="DSS037C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Furthermore, tRNA genes are mostly cleaved at or around the anti-codon loop in both yeast and humans.<sup><a reveal-id="DSS037C55" data-open="DSS037C55" class="link link-ref link-reveal xref-bibr">55</a>,<a reveal-id="DSS037C56" data-open="DSS037C56" class="link link-ref link-reveal xref-bibr">56</a></sup> However, in barley, we found that the predominant size of tsRNAs was 20 nt, with the cleavage sites mainly occurring at the very 5′ ends of tRNA transcripts. This observation indicates that the biogenesis of tsRNAs in barley may adopt a mechanism different from tsRNAs in other organisms.</p><p>Chloroplast-derived sRNAs are a novel class of sRNAs. A large number of csRNAs have been identified in various species.<sup><a reveal-id="DSS037C32" data-open="DSS037C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS037C57  DSS037C60" data-open="DSS037C57  DSS037C60" class="link link-ref link-reveal xref-bibr">57–60</a></sup> CsRNAs have been proposed to protect RNA from degradation.<sup><a reveal-id="DSS037C61" data-open="DSS037C61" class="link link-ref link-reveal xref-bibr">61</a></sup> However, the biogenesis mechanism of these csRNAs is unclear. It is believed that normal RNA degradation processes and the miRNA/siRNA pathway may both be involved. RNA degradation uses ribonucleases to cleave RNA at exposed sites. However, the main enzyme responsible for initial cleavage in the course of RNA degradation has not been identified in plant chloroplasts.<sup><a reveal-id="DSS037C62" data-open="DSS037C62" class="link link-ref link-reveal xref-bibr">62</a></sup> The miRNA/siRNA pathway adopts a specific Dicer ribonuclease to initially generate hairpin precursors, resulting in 20–24 nt short duplexes. However, this process may not be accomplished in the chloroplast <em>per se</em>, but in the nucleus, because the Dicer ribonuclease is not present in the chloroplast.<sup><a reveal-id="DSS037C62" data-open="DSS037C62" class="link link-ref link-reveal xref-bibr">62</a></sup> This could explain why so far no miRNAs have been identified in the chloroplast. In the case of the chloroplast-derived tsRNAs, which are the most abundant csRNAs in barley, their generation may involve RNase P, an endoribonuclease essential for processing polycistronic precursor tRNAs into mature tRNA 5′ ends.<sup><a reveal-id="DSS037C63" data-open="DSS037C63" class="link link-ref link-reveal xref-bibr">63</a></sup> In addition, other tRNA-specific nucleases, such as colicin E5, colicin D, prrC, ranpirnase or onconase and angiogenin, could be involved.<sup><a reveal-id="DSS037C4" data-open="DSS037C4" class="link link-ref link-reveal xref-bibr">4</a></sup> However, the involvement of such enzymes cannot explain the preferential cleavage sites found in the tsRNAs.</p><p>P affects the expression of csRNAs (Table <a reveal-id="DSS037TB1" data-open="DSS037TB1" class="link link-reveal link-table xref-fig">1</a>). This suggests that P may be involved in degradative changes in the chloroplasts. Previous studies have shown that P deficiency decreases the chlorophyll content to levels below what is required for chloroplast stability and the ability of the chloroplast to synthesize ATP.<sup><a reveal-id="DSS037C64" data-open="DSS037C64" class="link link-ref link-reveal xref-bibr">64</a></sup> This implies that csRNAs may result from a lack of energy available for photosynthesis and cellular respiration. However, we must point out that it cannot be determined, if csRNAs are derived from the chloroplast genome or derived from chloroplast-derived sequences inserted in the nuclear genome. Therefore, the data for csRNAs identified in this study may not truly reflect the real situation in the plants.</p>                            <h2 scrollto-destination=4600388 id="4600388" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4600394 id="4600394" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors wish to thank Ursula Langridge, Hui Zhou, Bryce Shi and Petrus Tang for providing assistance. The authors also thank Dr Julie Hayes for her careful and critical reading of the manuscript.</p>                            <h2 scrollto-destination=4600401 id="4600401" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS037C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones-Rhoades</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Bartel</div> <div class="given-names">B.</div></div>. </div><div class="article-title">MicroRNAs and their regulatory roles in plants</div>, <div class="source">Ann. Rev. Plant Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">19</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%20and%20their%20regulatory%20roles%20in%20plants&amp;author=M.W.%20Jones-Rhoades&amp;author=D.P.%20Bartel&amp;author=B.%20Bartel&amp;publication_year=2006&amp;journal=Ann.%20Rev.%20Plant%20Biol.&amp;volume=57&amp;pages=19-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.57.032905.105218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.57.032905.105218"> </span></div> </div></div></div></div></div><div content-id="DSS037C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katiyar-Agarwal</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Vivian-Smith</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A novel class of bacteria-induced small RNAs in <em>Arabidopsis</em></div>, <div class="source">Genes Dev.</div> , <div class="year">2007</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3123</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20bacteria-induced%20small%20RNAs%20in%20Arabidopsis&amp;author=S.%20Katiyar-Agarwal&amp;author=S.%20Gao&amp;author=A.%20Vivian-Smith&amp;author=H.%20Jin&amp;publication_year=2007&amp;journal=Genes%20Dev.&amp;volume=21&amp;pages=3123-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1595107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1595107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1595107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18003861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vaucheret</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Post-transcriptional small RNA pathways in plants: mechanisms and regulations</div>, <div class="source">Genes Dev.</div> , <div class="year">2006</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">759</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Post-transcriptional%20small%20RNA%20pathways%20in%20plants%3A%20mechanisms%20and%20regulations&amp;author=H.%20Vaucheret&amp;publication_year=2006&amp;journal=Genes%20Dev.&amp;volume=20&amp;pages=759-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1410506" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1410506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1410506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16600909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Malhotra</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A novel class of small RNAs: tRNA-derived RNA fragments (tRFs)</div>, <div class="source">Genes Dev.</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2639</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20small%20RNAs%3A%20tRNA-derived%20RNA%20fragments%20(tRFs)&amp;author=Y.S.%20Lee&amp;author=Y.%20Shibata&amp;author=A.%20Malhotra&amp;author=A.%20Dutta&amp;publication_year=2009&amp;journal=Genes%20Dev.&amp;volume=23&amp;pages=2639-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1837609" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1837609" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1837609"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19933153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sobala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hutvagner</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Transfer RNA-derived fragments: origins, processing, and functions</div>, <div class="source">RNA</div> , <div class="year">2011</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">853</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/wrna.96" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/wrna.96</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20RNA-derived%20fragments%3A%20origins%2C%20processing%2C%20and%20functions&amp;author=A.%20Sobala&amp;author=G.%20Hutvagner&amp;publication_year=2011&amp;journal=RNA&amp;volume=2&amp;pages=853-62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21976287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elliott</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Reuter</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Reddy</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Abbott</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Phosphorus nutrition of spring wheat (Triticum aestivum L.). 1. Effects of phosphorus supply on plant symptoms, yield, components of yield, and plant phosphorus uptake</div>, <div class="source">Aust. J. Agric. Res.</div> , <div class="year">1997</div>, vol. <div class="volume">455</div> (pg. <div class="fpage">855</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphorus%20nutrition%20of%20spring%20wheat%20(Triticum%20aestivum%20L.).%201.%20Effects%20of%20phosphorus%20supply%20on%20plant%20symptoms%2C%20yield%2C%20components%20of%20yield%2C%20and%20plant%20phosphorus%20uptake&amp;author=D.E.%20Elliott&amp;author=D.J.%20Reuter&amp;author=G.D.%20Reddy&amp;author=R.%20Abbott&amp;publication_year=1997&amp;journal=Aust.%20J.%20Agric.%20Res.&amp;volume=455&amp;pages=855-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/A96159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FA96159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FA96159"> </span></div> </div></div></div></div></div><div content-id="DSS037C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gahoonia</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">N.E.</div></div>. </div><div class="article-title">Barley genotypes with long root hairs sustain high grain yields in low-P field</div>, <div class="source">Plant Soil</div> , <div class="year">2004</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">55</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Barley%20genotypes%20with%20long%20root%20hairs%20sustain%20high%20grain%20yields%20in%20low-P%20field&amp;author=T.S.%20Gahoonia&amp;author=N.E.%20Nielsen&amp;publication_year=2004&amp;journal=Plant%20Soil&amp;volume=262&amp;pages=55-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PLSO.0000037020.58002.ac" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PLSO.0000037020.58002.ac" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PLSO.0000037020.58002.ac"> </span></div> </div></div></div></div></div><div content-id="DSS037C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Shirley</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Genc</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Langridge</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Phosphate utilization efficiency correlates with expression of low-affinity phosphate transporters and noncoding RNA, IPS1, in barley</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1217</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphate%20utilization%20efficiency%20correlates%20with%20expression%20of%20low-affinity%20phosphate%20transporters%20and%20noncoding%20RNA%2C%20IPS1%2C%20in%20barley&amp;author=C.Y.%20Huang&amp;author=N.%20Shirley&amp;author=Y.%20Genc&amp;author=B.%20Shi&amp;author=P.%20Langridge&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1217-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.178459" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.178459" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.178459"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21606317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuo</div> <div class="given-names">H.F.</div></div>,  <div class="name"><div class="surname">Chiou</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">The role of microRNAs in phosphorus deficiency signalling</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1016</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20microRNAs%20in%20phosphorus%20deficiency%20signalling&amp;author=H.F.%20Kuo&amp;author=T.J.%20Chiou&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1016-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.175265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.175265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.175265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21562333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">H.Q.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">C.X.</div></div>,  <div class="name"><div class="surname">Yin</div> <div class="given-names">X.M.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Q.R.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.M.</div></div>. </div><div class="article-title">MicroRNA expression profiles associated with phosphorus deficiency in white lupin (Lupinus albus L.)</div>, <div class="source">Plant Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">178</div> (pg. <div class="fpage">23</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNA%20expression%20profiles%20associated%20with%20phosphorus%20deficiency%20in%20white%20lupin%20(Lupinus%20albus%20L.)&amp;author=Y.Y.%20Zhu&amp;author=H.Q.%20Zeng&amp;author=C.X.%20Dong&amp;author=X.M.%20Yin&amp;author=Q.R.%20Shen&amp;author=Z.M.%20Yang&amp;publication_year=2010&amp;journal=Plant%20Sci.&amp;volume=178&amp;pages=23-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2009.09.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2009.09.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2009.09.011"> </span></div> </div></div></div></div></div><div content-id="DSS037C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">H.Q.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">S.Q.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.M.</div></div>. </div><div class="article-title">Analysis of phosphorus deficient responsive miRNAs and cis-elements from soybean (Glycine max L.)</div>, <div class="source">J. Plant Physiol.</div> , <div class="year">2010</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">1289</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20phosphorus%20deficient%20responsive%20miRNAs%20and%20cis-elements%20from%20soybean%20(Glycine%20max%20L.)&amp;author=H.Q.%20Zeng&amp;author=Y.Y.%20Zhu&amp;author=S.Q.%20Huang&amp;author=Z.M.%20Yang&amp;publication_year=2010&amp;journal=J.%20Plant%20Physiol.&amp;volume=167&amp;pages=1289-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2010.04.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2010.04.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2010.04.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20591534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hsieh</div> <div class="given-names">L.C.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Shih</div> <div class="given-names">AChCh</div></div>, et al. </div><div class="article-title">Uncovering small RNA-mediated responses to phosphate deficiency in <em>Arabidopsis</em> by deep sequencing</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">2120</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uncovering%20small%20RNA-mediated%20responses%20to%20phosphate%20deficiency%20in%20Arabidopsis%20by%20deep%20sequencing&amp;author=L.C.%20Hsieh&amp;author=S.I.%20Lin&amp;author=AChCh%20Shih&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=2120-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.147280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.147280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.147280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Chiang</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">W.Y.</div></div>, et al. </div><div class="article-title">Regulatory network of microRNA399 and PHO2 by systemic signalling</div>, <div class="source">Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">732</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20network%20of%20microRNA399%20and%20PHO2%20by%20systemic%20signalling&amp;author=S.I.%20Lin&amp;author=S.F.%20Chiang&amp;author=W.Y.%20Lin&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=147&amp;pages=732-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.116269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.116269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.116269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18390805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pant</div> <div class="given-names">B.D.</div></div>,  <div class="name"><div class="surname">Buhtz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kehr</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Scheible</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">MicroRNA399 is a longdistance signal for the regulation of plant phosphate homeostasis</div>, <div class="source">Plant J.</div> , <div class="year">2008</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">731</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNA399%20is%20a%20longdistance%20signal%20for%20the%20regulation%20of%20plant%20phosphate%20homeostasis&amp;author=B.D.%20Pant&amp;author=A.%20Buhtz&amp;author=J.%20Kehr&amp;author=W.R.%20Scheible&amp;publication_year=2008&amp;journal=Plant%20J.&amp;volume=53&amp;pages=731-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2008.53.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2008.53.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2008.53.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17988220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gustafsson</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ekman</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Dormling</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Effects of the Pallas gene in barley: phene analysis, overdominance, variability</div>, <div class="source">Hereditas</div> , <div class="year">1977</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">251</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20the%20Pallas%20gene%20in%20barley%3A%20phene%20analysis%2C%20overdominance%2C%20variability&amp;author=A.%20Gustafsson&amp;author=G.%20Ekman&amp;author=I.%20Dormling&amp;publication_year=1977&amp;journal=Hereditas&amp;volume=86&amp;pages=251-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1601-5223.1977.tb01235.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1601-5223.1977.tb01235.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1601-5223.1977.tb01235.x"> </span></div> </div></div></div></div></div><div content-id="DSS037C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Genc</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Langridge</div> <div class="given-names">P.</div></div>. </div><div class="article-title">A study of the role of root morphological traits in growth of barley in zinc-deficient soil</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">2775</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20study%20of%20the%20role%20of%20root%20morphological%20traits%20in%20growth%20of%20barley%20in%20zinc-deficient%20soil&amp;author=Y.%20Genc&amp;author=C.Y.%20Huang&amp;author=P.%20Langridge&amp;publication_year=2007&amp;journal=J.%20Exp.%20Bot.&amp;volume=58&amp;pages=2775-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erm142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferm142"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17609531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reuter</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Robison</div> <div class="given-names">J.B.</div></div>. </div>, <div class="source">Plant Analysis: An Interpretation Manual</div> , <div class="year">1997</div><div class="publisher-loc">Melbourne, Australia</div><div class="publisher-name">CSIRO Publishing</div>(pg. <div class="fpage">86</div>-<div class="lpage">92</div>)</div></div></div></div><div content-id="DSS037C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">George</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">L.K.</div></div>,  <div class="name"><div class="surname">Newton</div> <div class="given-names">A.C.</div></div>, et al. </div><div class="article-title">Impact of soil tillage on the robustness of the genetic component of variation in phosphorus (P) use efficiency in barley (<em>Hordeum vulgare</em> L.)</div>, <div class="source">Plant Soil</div> , <div class="year">2011</div>, vol. <div class="volume">339</div> (pg. <div class="fpage">113</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20soil%20tillage%20on%20the%20robustness%20of%20the%20genetic%20component%20of%20variation%20in%20phosphorus%20(P)%20use%20efficiency%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=T.S.%20George&amp;author=L.K.%20Brown&amp;author=A.C.%20Newton&amp;publication_year=2011&amp;journal=Plant%20Soil&amp;volume=339&amp;pages=113-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11104-009-0209-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11104-009-0209-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11104-009-0209-5"> </span></div> </div></div></div></div></div><div content-id="DSS037C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hackenberg</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rodriguez-Ezpeleta</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Aransay</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">miRanalyzer: an update on the detection and analysis of microRNAs in high-throughput sequencing experiments</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">W132</div>-<div class="lpage">138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miRanalyzer%3A%20an%20update%20on%20the%20detection%20and%20analysis%20of%20microRNAs%20in%20high-throughput%20sequencing%20experiments&amp;author=M.%20Hackenberg&amp;author=N.%20Rodriguez-Ezpeleta&amp;author=A.M.%20Aransay&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=W132-138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21515631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ouyang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>. </div><div class="article-title">The TIGR Plant Repeat Databases: a collective resource for the identification of repetitive sequences in plants</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D360</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TIGR%20Plant%20Repeat%20Databases%3A%20a%20collective%20resource%20for%20the%20identification%20of%20repetitive%20sequences%20in%20plants&amp;author=S.%20Ouyang&amp;author=C.R.%20Buell&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D360-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh099"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wicker</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Matthews</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">B.</div></div>. </div><div class="article-title">TREP: a database of Triticeae repetitive elements</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">561</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TREP%3A%20a%20database%20of%20Triticeae%20repetitive%20elements&amp;author=T.%20Wicker&amp;author=D.E.%20Matthews&amp;author=B.%20Keller&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci.&amp;volume=7&amp;pages=561-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(02)02372-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(02)02372-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(02)02372-5"> </span></div> </div></div></div></div></div><div content-id="DSS037C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Repbase update, a database of eukaryotic repetitive elements</div>, <div class="source">Cytogenet. Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">462</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res.&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Friedlander</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Adamidi</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Discovering microRNAs from deep sequencing data using miRDeep</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">407</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovering%20microRNAs%20from%20deep%20sequencing%20data%20using%20miRDeep&amp;author=M.R.%20Friedlander&amp;author=W.%20Chen&amp;author=C.%20Adamidi&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=407-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18392026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zuker</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mfold web server for nucleic acid folding and hybridization prediction</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3406</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mfold%20web%20server%20for%20nucleic%20acid%20folding%20and%20hybridization%20prediction&amp;author=M.%20Zuker&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3406-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg595" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dai</div> <div class="given-names">X.B.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">P.X.</div></div>. </div><div class="article-title">psRNATarget: a plant small RNA target analysis server</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">W155</div>-<div class="lpage">159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=psRNATarget%3A%20a%20plant%20small%20RNA%20target%20analysis%20server&amp;author=X.B.%20Dai&amp;author=P.X.%20Zhao&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=W155-159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21622958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Addo-Quaye</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Eshoo</div> <div class="given-names">T.W.</div></div>,  <div class="name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Axtell</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Endogenous siRNA and miRNA targets identified by sequencing of the <em>Arabidopsis</em> degradome</div>, <div class="source">Curr. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">758</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endogenous%20siRNA%20and%20miRNA%20targets%20identified%20by%20sequencing%20of%20the%20Arabidopsis%20degradome&amp;author=C.%20Addo-Quaye&amp;author=T.W.%20Eshoo&amp;author=D.P.%20Bartel&amp;author=M.J.%20Axtell&amp;publication_year=2008&amp;journal=Curr.%20Biol.&amp;volume=18&amp;pages=758-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2008.04.042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2008.04.042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2008.04.042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18472421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Sutton</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Collins</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Pallotta</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Langridge</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Construction of a barley bacterial artificial chromosome library suitable for cloning genes for boron tolerance, sodium exclusion and high grain zinc content</div>, <div class="source">Plant Breeding</div> , <div class="year">2010</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">291</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20barley%20bacterial%20artificial%20chromosome%20library%20suitable%20for%20cloning%20genes%20for%20boron%20tolerance%2C%20sodium%20exclusion%20and%20high%20grain%20zinc%20content&amp;author=B.J.%20Shi&amp;author=T.%20Sutton&amp;author=N.C.%20Collins&amp;author=M.%20Pallotta&amp;author=P.%20Langridge&amp;publication_year=2010&amp;journal=Plant%20Breeding&amp;volume=129&amp;pages=291-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2010.129.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2010.129.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2010.129.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSS037C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schreiber</div> <div class="given-names">A.W.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Langridge</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Baumann</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Discovery of barley miRNAs through deep sequencing of short reads</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">129</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20barley%20miRNAs%20through%20deep%20sequencing%20of%20short%20reads&amp;author=A.W.%20Schreiber&amp;author=B.J.%20Shi&amp;author=C.Y.%20Huang&amp;author=P.%20Langridge&amp;author=U.%20Baumann&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21352554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lv</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nie</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Identification and characterization of microRNAs from barley (<em>Hordeum vulgare</em> L.) by high-throughput sequencing</div>, <div class="source">Int. J. Mol. Sci.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2973</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20microRNAs%20from%20barley%20(Hordeum%20vulgare%20L.)%20by%20high-throughput%20sequencing&amp;author=S.%20Lv&amp;author=X.%20Nie&amp;author=L.%20Wang&amp;publication_year=2012&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;volume=13&amp;pages=2973-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms13032973" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms13032973" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms13032973"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22489137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pontier</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Picart</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Roudier</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">NERD, a plant-specific GW protein, defines an additional RNAi-dependent chromatin-based pathway in <em>Arabidopsis</em></div>, <div class="source">Mol. Cell</div> , <div class="year">2012</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">121</div>-<div class="lpage">132</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NERD%2C%20a%20plant-specific%20GW%20protein%2C%20defines%20an%20additional%20RNAi-dependent%20chromatin-based%20pathway%20in%20Arabidopsis&amp;author=D.%20Pontier&amp;author=C.%20Picart&amp;author=F.%20Roudier&amp;publication_year=2012&amp;journal=Mol.%20Cell&amp;volume=48&amp;pages=121-132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.07.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.07.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.07.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22940247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jauvion</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Rivard</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bouteiller</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Elmayan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vaucheret</div> <div class="given-names">H.</div></div>. </div><div class="article-title">RDR2 partially antagonizes the production of RDR6-dependent siRNA in sense transgene-mediated PTGS</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>pg. <div class="fpage">e29785</div> </div></div></div></div><div content-id="DSS037C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">A novel class of heat-responsive small RNAs derived from the chloroplast genome of Chinese cabbage (Brassica rapa)</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">289</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20heat-responsive%20small%20RNAs%20derived%20from%20the%20chloroplast%20genome%20of%20Chinese%20cabbage%20(Brassica%20rapa)&amp;author=L.%20Wang&amp;author=X.%20Yu&amp;author=H.%20Wang&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=289" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-289" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-289" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-289"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21639890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruwe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Schmitz-Linnewebwe</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Short non-coding RNA fragments accumulating in chloroplasts: footprints of RNA binding proteins?</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">2011</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">3106</div>-<div class="lpage">3116</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20non-coding%20RNA%20fragments%20accumulating%20in%20chloroplasts%3A%20footprints%20of%20RNA%20binding%20proteins%3F&amp;author=K.%20Ruwe&amp;author=C.%20Schmitz-Linnewebwe&amp;publication_year=2011&amp;journal=Nucleic%20Acid%20Res.&amp;volume=40&amp;pages=3106-3116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1138" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1138" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1138"> </span></div> </div></div></div></div></div><div content-id="DSS037C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Czech</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div>. </div><div class="article-title">miR156-regulated SPL transcription factors define an endogenous flowering pathway in <em>Arabidopsis thaliana</em></div>, <div class="source">Cell</div> , <div class="year">2009</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">738</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miR156-regulated%20SPL%20transcription%20factors%20define%20an%20endogenous%20flowering%20pathway%20in%20Arabidopsis%20thaliana&amp;author=J.W.%20Wang&amp;author=B.%20Czech&amp;author=D.%20Weigel&amp;publication_year=2009&amp;journal=Cell&amp;volume=138&amp;pages=738-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.06.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.06.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.06.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19703399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujii</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Chiou</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Aung</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">A miRNA involved in phosphate-starvation response in <em>Arabidopsis</em></div>, <div class="source">Curr. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">2038</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20miRNA%20involved%20in%20phosphate-starvation%20response%20in%20Arabidopsis&amp;author=H.%20Fujii&amp;author=T.J.%20Chiou&amp;author=S.I.%20Lin&amp;author=K.%20Aung&amp;author=J.K.%20Zhu&amp;publication_year=2005&amp;journal=Curr.%20Biol.&amp;volume=15&amp;pages=2038-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2005.10.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2005.10.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2005.10.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pant</div> <div class="given-names">B.D.</div></div>,  <div class="name"><div class="surname">Musialak-Lange</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nuc</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Identification of nutrient-responsive <em>Arabidopsis</em> and rapeseed microRNAs by comprehensive real-time polymerase chain reaction profiling and small RNA sequencing</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">1541</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20nutrient-responsive%20Arabidopsis%20and%20rapeseed%20microRNAs%20by%20comprehensive%20real-time%20polymerase%20chain%20reaction%20profiling%20and%20small%20RNA%20sequencing&amp;author=B.D.%20Pant&amp;author=M.%20Musialak-Lange&amp;author=P.%20Nuc&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=150&amp;pages=1541-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.139139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.139139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.139139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19465578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arteaga-Vázquez</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Caballero-Pérez</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vielle-Calzada</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">A family of microRNAs present in plants and animals</div>, <div class="source">Plant Cell</div> , <div class="year">2006</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">3355</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20family%20of%20microRNAs%20present%20in%20plants%20and%20animals&amp;author=M.%20Arteaga-V%C3%A1zquez&amp;author=J.%20Caballero-P%C3%A9rez&amp;author=J.P.%20Vielle-Calzada&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=3355-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.044420" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.044420" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.044420"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17189346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">X.P.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Cobb</div> <div class="given-names">G.P.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">Evidence that miRNAs are different from other RNAs Cell</div>, <div class="source">Mol. Life Sci.</div> , <div class="year">2006a</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">246</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20that%20miRNAs%20are%20different%20from%20other%20RNAs%20Cell&amp;author=B.H.%20Zhang&amp;author=X.P.%20Pan&amp;author=S.B.%20Cox&amp;author=G.P.%20Cobb&amp;author=T.A.%20Anderson&amp;publication_year=2006a&amp;journal=Mol.%20Life%20Sci.&amp;volume=63&amp;pages=246-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-005-5467-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-005-5467-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-005-5467-7"> </span></div> </div></div></div></div></div><div content-id="DSS037C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">X.P.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Cobb</div> <div class="given-names">G.P.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">Conservation and divergence of plant microRNA genes</div>, <div class="source">Plant J.</div> , <div class="year">2006b</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">243</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20divergence%20of%20plant%20microRNA%20genes&amp;author=B.H.%20Zhang&amp;author=X.P.%20Pan&amp;author=C.H.%20Cannon&amp;author=G.P.%20Cobb&amp;author=T.A.%20Anderson&amp;publication_year=2006b&amp;journal=Plant%20J.&amp;volume=46&amp;pages=243-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2006.46.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2006.46.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2006.46.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSS037C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Huan</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chong</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Deep sequencing of <em>Brachypodium</em> small RNAs at the global genome level identifies microRNAs involved in cold stress response</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">449</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471–2164-10-449" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471–2164-10-449</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20of%20Brachypodium%20small%20RNAs%20at%20the%20global%20genome%20level%20identifies%20microRNAs%20involved%20in%20cold%20stress%20response&amp;author=J.%20Zhang&amp;author=Y.%20Xu&amp;author=Q.%20Huan&amp;author=K.%20Chong&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=449" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19772667" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Padmanabhan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Identification of miniature inverted-repeat transposable elements (MITEs) and biogenesis of their siRNAs in the Solanaceae: new functional implications for MITEs</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">42</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20miniature%20inverted-repeat%20transposable%20elements%20(MITEs)%20and%20biogenesis%20of%20their%20siRNAs%20in%20the%20Solanaceae%3A%20new%20functional%20implications%20for%20MITEs&amp;author=H.%20Kuang&amp;author=C.%20Padmanabhan&amp;author=F.%20Li&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=42-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.078196.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.078196.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.078196.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19037014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slotkin</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Vaughn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Borges</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Epigenetic reprogramming and small RNA silencing of transposable elements in pollen</div>, <div class="source">Cell</div> , <div class="year">2009</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">461</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20reprogramming%20and%20small%20RNA%20silencing%20of%20transposable%20elements%20in%20pollen&amp;author=R.K.%20Slotkin&amp;author=M.%20Vaughn&amp;author=F.%20Borges&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=461-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.12.038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.12.038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.12.038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19203581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tomaru</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kasukawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Antisense transcription in the mammalian transcriptome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1564</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antisense%20transcription%20in%20the%20mammalian%20transcriptome&amp;author=S.%20Katayama&amp;author=Y.%20Tomaru&amp;author=T.%20Kasukawa&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1564-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1112009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1112009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1112009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ghildiyal</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Seitz</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Horwich</div> <div class="given-names">M.D.</div></div>, et al. </div><div class="article-title">Endogenous siRNAs derived from transposons and mRNAs in <em>Drosophila</em> somatic cells</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">320</div> (pg. <div class="fpage">1077</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endogenous%20siRNAs%20derived%20from%20transposons%20and%20mRNAs%20in%20Drosophila%20somatic%20cells&amp;author=M.%20Ghildiyal&amp;author=H.%20Seitz&amp;author=M.D.%20Horwich&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=1077-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1157396" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1157396" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1157396"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18403677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Isono</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Niwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Satoh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Evidence for transcriptional regulation of plastid photosynthesis genes in <em>Arabidopsis thaliana</em> roots</div>, <div class="source">Plant Physiol.</div> , <div class="year">1997</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">623</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20transcriptional%20regulation%20of%20plastid%20photosynthesis%20genes%20in%20Arabidopsis%20thaliana%20roots&amp;author=K.%20Isono&amp;author=Y.%20Niwa&amp;author=K.%20Satoh&amp;author=H.%20Kobayashi&amp;publication_year=1997&amp;journal=Plant%20Physiol.&amp;volume=114&amp;pages=623-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.114.2.623" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.2.623" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.114.2.623"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9193094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Plaxton</div> <div class="given-names">W.C.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Goodman</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Plant response to stress: biochemical adaptations to phosphate deficiency</div>, <div class="source">Encyclopedia of Plant and Crop Science</div> , <div class="year">2004</div><div class="publisher-loc">New York, USA</div><div class="publisher-name">Marcel Dekker</div>(pg. <div class="fpage">976</div>-<div class="lpage">80</div>)</div></div></div></div><div content-id="DSS037C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Valencia-Sanchez</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hannon</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Parker</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Control of translation and mRNA degradation by miRNAs and siRNAs</div>, <div class="source">Genes Dev.</div> , <div class="year">2006</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">515</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20translation%20and%20mRNA%20degradation%20by%20miRNAs%20and%20siRNAs&amp;author=M.A.%20Valencia-Sanchez&amp;author=J.%20Liu&amp;author=G.J.%20Hannon&amp;author=R.%20Parker&amp;publication_year=2006&amp;journal=Genes%20Dev.&amp;volume=20&amp;pages=515-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1399806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1399806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1399806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16510870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia Silva</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Frugier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tosar</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title"><em>Trypanosoma cruzi</em> produces a population of tRNA-derived small RNAs which are recruited to specific cytoplasmic granules and secreted to the extracellular medium</div>, <div class="source">BMC Proc.</div> , <div class="year">2011</div>, vol. <div class="volume">5</div> <div class="issue">Suppl. 1</div>pg. <div class="fpage">O7</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1753-6561-5-S1-O7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1753-6561-5-S1-O7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trypanosoma%20cruzi%20produces%20a%20population%20of%20tRNA-derived%20small%20RNAs%20which%20are%20recruited%20to%20specific%20cytoplasmic%20granules%20and%20secreted%20to%20the%20extracellular%20medium&amp;author=M.R.%20Garcia%20Silva&amp;author=M.%20Frugier&amp;author=J.%20Tosar&amp;publication_year=2011&amp;journal=BMC%20Proc.&amp;volume=5&amp;pages=O7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1753-6561-5-s1-o7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1753-6561-5-s1-o7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1753-6561-5-s1-o7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22373374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orbanm</div> <div class="given-names">T.I.</div></div>,  <div class="name"><div class="surname">Izaurralde</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Decay of mRNAs targeted by RISC requires XRN1, the Ski complex, and the exosome</div>, <div class="source">RNA</div> , <div class="year">2005</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">459</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Decay%20of%20mRNAs%20targeted%20by%20RISC%20requires%20XRN1%2C%20the%20Ski%20complex%2C%20and%20the%20exosome&amp;author=T.I.%20Orbanm&amp;author=E.%20Izaurralde&amp;publication_year=2005&amp;journal=RNA&amp;volume=11&amp;pages=459-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.7231505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.7231505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.7231505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15703439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sobala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Filtering of deep sequencing data reveals the existence of abundant Dicer-dependent small RNAs derived from tRNAs</div>, <div class="source">RNA</div> , <div class="year">2009</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">2147</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filtering%20of%20deep%20sequencing%20data%20reveals%20the%20existence%20of%20abundant%20Dicer-dependent%20small%20RNAs%20derived%20from%20tRNAs&amp;author=C.%20Cole&amp;author=A.%20Sobala&amp;author=C.%20Lu&amp;publication_year=2009&amp;journal=RNA&amp;volume=15&amp;pages=2147-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.1738409" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.1738409" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.1738409"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19850906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Souret</div> <div class="given-names">F.F.</div></div>,  <div class="name"><div class="surname">Kastenmayer</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">AtXRN4 degrades mRNA in <em>Arabidopsis</em> and its substrates include selected miRNA targets</div>, <div class="source">Mol. Cell</div> , <div class="year">2004</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">173</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AtXRN4%20degrades%20mRNA%20in%20Arabidopsis%20and%20its%20substrates%20include%20selected%20miRNA%20targets&amp;author=F.F.%20Souret&amp;author=J.P.%20Kastenmayer&amp;author=P.J.%20Green&amp;publication_year=2004&amp;journal=Mol.%20Cell&amp;volume=15&amp;pages=173-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2004.06.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2004.06.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2004.06.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15260969" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haussecker</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lau</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Parameswaran</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Fire</div> <div class="given-names">A.Z.</div></div>,  <div class="name"><div class="surname">Kay</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Human tRNA-derived small RNAs in the global regulation of RNA silencing</div>, <div class="source">RNA</div> , <div class="year">2010</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">673</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20tRNA-derived%20small%20RNAs%20in%20the%20global%20regulation%20of%20RNA%20silencing&amp;author=D.%20Haussecker&amp;author=Y.%20Huang&amp;author=A.%20Lau&amp;author=P.%20Parameswaran&amp;author=A.Z.%20Fire&amp;author=M.A.%20Kay&amp;publication_year=2010&amp;journal=RNA&amp;volume=16&amp;pages=673-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.2000810" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.2000810" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.2000810"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20181738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Ender</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Meister</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">John</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Extensive terminal and asymmetric processing of small RNAs from rRNAs, snoRNAs, snRNAs, and tRNAs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">6787</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20terminal%20and%20asymmetric%20processing%20of%20small%20RNAs%20from%20rRNAs%2C%20snoRNAs%2C%20snRNAs%2C%20and%20tRNAs&amp;author=Z.%20Li&amp;author=C.%20Ender&amp;author=G.%20Meister&amp;author=P.S.%20Moore&amp;author=Y.%20Chang&amp;author=B.%20John&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=6787-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pederson</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Regulatory RNAs derived from transfer RNA</div>, <div class="source">RNA</div> , <div class="year">2010</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1865</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20RNAs%20derived%20from%20transfer%20RNA&amp;author=T.%20Pederson&amp;publication_year=2010&amp;journal=RNA&amp;volume=16&amp;pages=1865-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.2266510" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.2266510" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.2266510"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20719919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Parker</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The RNase Rny1p cleaves tRNAs and promotes cell death during oxidative stress in <em>Saccharomyces cerevisiae</em></div>, <div class="source">J. Cell Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">43</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RNase%20Rny1p%20cleaves%20tRNAs%20and%20promotes%20cell%20death%20during%20oxidative%20stress%20in%20Saccharomyces%20cerevisiae&amp;author=D.M.%20Thompson&amp;author=R.%20Parker&amp;publication_year=2009&amp;journal=J.%20Cell%20Biol.&amp;volume=185&amp;pages=43-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200811119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200811119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200811119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19332891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamasaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ivanov</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">G.F.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Angiogenin cleaves tRNA and promotes stress-induced translational repression</div>, <div class="source">J. Cell Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">35</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Angiogenin%20cleaves%20tRNA%20and%20promotes%20stress-induced%20translational%20repression&amp;author=S.%20Yamasaki&amp;author=P.%20Ivanov&amp;author=G.F.%20Hu&amp;author=P.%20Anderson&amp;publication_year=2009&amp;journal=J.%20Cell%20Biol.&amp;volume=185&amp;pages=35-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200811106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200811106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200811106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19332886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lung</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zemann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Madej</div> <div class="given-names">M.J.</div></div>, et al. </div><div class="article-title">Identification of small non-coding RNAs from mitochondria and chloroplasts</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">3842</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20small%20non-coding%20RNAs%20from%20mitochondria%20and%20chloroplasts&amp;author=B.%20Lung&amp;author=A.%20Zemann&amp;author=M.J.%20Madej&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=3842-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl448" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl448" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl448"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16899451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kong</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Diversity of endogenous small non-coding RNAs in <em>Oryza sativa</em></div>, <div class="source">Genetica</div> , <div class="year">2006</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">21</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20endogenous%20small%20non-coding%20RNAs%20in%20Oryza%20sativa&amp;author=Z.%20Chen&amp;author=J.%20Zhang&amp;author=J.%20Kong&amp;publication_year=2006&amp;journal=Genetica&amp;volume=128&amp;pages=21-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-005-2486-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-005-2486-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-005-2486-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17028937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tej</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Haudenschild</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Meyers</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">Elucidation of the small RNA component of the transcriptome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1567</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elucidation%20of%20the%20small%20RNA%20component%20of%20the%20transcriptome&amp;author=C.%20Lu&amp;author=S.S.%20Tej&amp;author=S.%20Luo&amp;author=C.D.%20Haudenschild&amp;author=B.C.%20Meyers&amp;author=P.J.%20Green&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1567-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1114112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1114112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1114112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rajagopalan</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Vaucheret</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Trejo</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div>. </div><div class="article-title">A diverse and evolutionarily fluid set of microRNAs in <em>Arabidopsis thaliana</em></div>, <div class="source">Genes Dev.</div> , <div class="year">2006</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3407</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20diverse%20and%20evolutionarily%20fluid%20set%20of%20microRNAs%20in%20Arabidopsis%20thaliana&amp;author=R.%20Rajagopalan&amp;author=H.%20Vaucheret&amp;author=J.%20Trejo&amp;author=D.P.%20Bartel&amp;publication_year=2006&amp;journal=Genes%20Dev.&amp;volume=20&amp;pages=3407-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1476406" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1476406" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1476406"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17182867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pfalz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bayraktar</div> <div class="given-names">O.A.</div></div>,  <div class="name"><div class="surname">Prikryl</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Barkan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Site-specific binding of a PPR protein defines and stabilizes 50 and 30 mRNA termini in chloroplasts</div>, <div class="source">EMBO J.</div> , <div class="year">2009</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2042</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site-specific%20binding%20of%20a%20PPR%20protein%20defines%20and%20stabilizes%2050%20and%2030%20mRNA%20termini%20in%20chloroplasts&amp;author=J.%20Pfalz&amp;author=O.A.%20Bayraktar&amp;author=J.%20Prikryl&amp;author=A.%20Barkan&amp;publication_year=2009&amp;journal=EMBO%20J.&amp;volume=28&amp;pages=2042-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/emboj.2009.121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Femboj.2009.121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Femboj.2009.121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19424177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoppel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Meurer</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The cutting crew – ribonucleases are key players in the control of plastid gene expression</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2012</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">1663</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cutting%20crew%20%E2%80%93%20ribonucleases%20are%20key%20players%20in%20the%20control%20of%20plastid%20gene%20expression&amp;author=R.%20Stoppel&amp;author=J.%20Meurer&amp;publication_year=2012&amp;journal=J.%20Exp.%20Bot.&amp;volume=63&amp;pages=1663-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140236" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mohanty</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Kushner</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Ribonuclease P processes polycistronic tRNA transcripts in Escherichia coli independent of ribonuclease E</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">7614</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribonuclease%20P%20processes%20polycistronic%20tRNA%20transcripts%20in%20Escherichia%20coli%20independent%20of%20ribonuclease%20E&amp;author=B.K.%20Mohanty&amp;author=S.R.%20Kushner&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=7614-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm917" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm917" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm917"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17981836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS037C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS037C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rychter</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Rao</div> <div class="given-names">I.M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Pessarakli</div> <div class="given-names">M.</div></div>. </div><div class="collab">Role of phosphorus in photosynthetic carbon metabolism</div>, <div class="source">Handbook of Photosynthesis</div> , <div class="year">2005</div><div class="publisher-loc">Tucson</div><div class="publisher-name">Taylor &amp; Francis Group, LLC</div>(pg. <div class="fpage">123</div>-<div class="lpage">48</div>)</div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=333081&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table8.xlsx?Expires=2147483647&amp;Signature=Jy1HE1x~NcOPy-GFrbIutrZ8~eekCDKOrkGW1D5VITM8RQZaKSP~VZy3YfAb5rFEYwEW6uj2LCQW72ECPcgmCHmpLJ8FLaoNkmTcBWYOEUe1SUw~yVKSu46nUm2hES8feiwkiMad0Nirdd6qBmuEKLg~gCH2doZisVpmIIyx1muT006n3Cpa5jSUXZrW2514iBGLnOHs5RtbXxXSAD3YMfuDb-iQKjF4V4pC6i--aiCthFl2XiE5Cm03CMfMGMt2qwtwVcPsNF-IsfdW3Dc5HJm0PfNLX0lK6uiFU~xqIPE2jQ9nfij8BmXXpJVKpe9fgXolr7PvEHYX~HjRpg0iog__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table7.xlsx?Expires=2147483647&amp;Signature=erCj0LnaST5N7jPg6YRNsyY0PIOQxzKUGn6L2U62l048CvPKbWqD~4PUi2YvBQZ2G1ZlGBdNXZLlprs3kEycyO8i7QGP6uRo-fkGWxHFqTYlZsE8qAJ6xqY-fa5njL7MpsuN9sdGZKe2yJmjYS2rG3vpG~LwRf7fy8xOPuaG15gUZNNpfSN0QBE6238fOvr9ZRmTYqHunGRoWRZFqz7T5jBxKRZ2uPbrb1gIj0SKL669Pv4WdRGpud4Brb4L0dA9lOhEeEqfKrQzBnPOx7uUm0QcvZQLdAcncULeVfVKXcZT1IOdwepEwG~H8l6OHPq18kZ~-XXQyUcnzaxfxPe8yw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table6.xlsx?Expires=2147483647&amp;Signature=rCKFacvn~~SD3ADdJN6UGANnd~druKdzJ1S~z~Wtq8szWV8rhEkr85CEtzuhqR29F9vgYBkb4utqKtLFdYrwosIZlyT7TIQIoAFmJ0Uzt6K1bmM63Lw-W7D5q0ugNNkLsxQkUoZmXD9Wh4ZnsAVlxLUbNOT113hiXinTJ23KSbDpfsokd1pWJbVSOEwS2QdIIoekTkS7TEwhazZy3Rypk-fx7aAfQIBfTnZym6kVXS92sZtM0eK8mwcsa0PjEfNSTxSfwqmsgq~lfF1UzOuBltBZIprH5B1nENLGII6ztvgEsv9dQjTaWuiKEFz~0pw4w0NhmzeN4Sgrb2kW0-7omw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table5.xlsx?Expires=2147483647&amp;Signature=09KkR2nCNTJDVr3TYZ~A84oXxm8~kVo5T0DkW4EJODKmidHtPhoyuiXWcgE-Nf-DTZ0zJN92CbaY7NK6qxcFmP~vH5yvkkQOXzgawa6ze3LKpuRzPFCTnSasvUr-0~ebvBlK0cjPeXJ-9pdBb8q26PEyLmaLorawa3Yyln~OgKOONcdI0P2RbK5Dx-MkX6qfLQKiuHpdaSh348dPYuVKmgInNBzmPckEsgzXxewnpnLR4ij-zu0rmMZNZonb5dzoZeWUkONhOhw1m0H9jCZ-yAVgH9JhY7pwBwNlTHZjk3V-hvU-K5eEg6zf84gSStF2U771itmal0FKEUeVVX5Mjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table4.xlsx?Expires=2147483647&amp;Signature=FD4JkwzWPFxlhveLtL6ZgrLglKoN~6LbY1DZpuyELID7ZL1AE2rEsV3ecprWL9OTi5sjvAcVLQ0JF9-mKmjj8abGaCzu5tCNH0OUB0pSsLODc4kFj3R7ZIa4thY~psPjtf-UhSBehQV3omdr336KkxMBEu7CY~iTOQlKaDnYlVunuerXaCS7ozWhjkrA~P24MzB5WWUUk-ihF5zB0soKSWKwvFTL7Py2USLUXMw0UNBmBvx8mi8hyhDr4f~y9lSdaHDFfbhAIEXTWbRmNVE~W8cwSH13eT7Ppiw9ui0bpal-N9Tams85SeMYzvJii7IUQ~AuXDQjLN794w7dDyRpnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table3.xlsx?Expires=2147483647&amp;Signature=w81N7l9GgHYY38C8EPstBL0ER80JKR6zyjowgqbQApYuHjJdaKCyTsuxXjQMKNAgy~Lp7TtCwgf8~qKH5x6SS8IpFFF8ST0CyFTti9ewdBg37kWw-XWTICd4yQrcwmBFLPbc-7o8aChw5v2Y-~u4cKOGWmT30C8G98gK7ChSgacjj0kuBrRPQleMHf7gJKvXpO0h-I~1ogK05rPFz-yIKtYfPOUji11HkIbP3VwOsbog6k-BKrXPPloMZ1tbXyZo4xbLVUNYg5Djr~bToOh-KgN4X3fw~7jR-Ou~k2bkVWETL597GcNtMDHMyOFbkxkyKUtSKp7Kess5NFy7py7cPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table2.xlsx?Expires=2147483647&amp;Signature=EnQ5YkIt4ioQQ9J0Qo-68XwmwvSdd9tn1KRJwETC6iLdz2Qez4ocuvvH3q6l~jmA2vZ2zNtNC3mtH-f~qHYhWp85QmZmdg8sgagpPn-svmzVhd7qZI4HdTTgfl7fUfHJlbVlp--tBQAgu0WecnkHh1wlPikr9sAr7g7JMZGZb-4ldbadWz85xYsOxX3dXIFldN2M6QBEcuGvIKZpyI4A8qShmtAfjNjCCjQ0MtNi7-osdDEfCQQ4anj0i8O58R7u1TxgQ6zxSxl4d5RCX13g4v1Veb94Z3gKtqDixmwK06B-Sl1nf0ok5W0Th0V810xhD7mV74xt~3v2BfwCKIyxsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss037/2/dss037-dss037supp_table1.xlsx?Expires=2147483647&amp;Signature=4-tpvs18X8nAXIoIKJx-JqDGKtH8-YHZbMpmjP~THhiw3UXsNgcLvprZIoR5ewkovdKQ43ZO9Hp2OP5wI5viSNikzS7HBM57v8Z9Ghy9me2wjmdbxj3nsLS6EN6lnZX7J5Y6zwHeOjlxsK3~LLG9ObZbttwI4e15FMp~xCYRL1mz-d8Q3Cp4mJqITRhP4Rnu0~82rt9~1jpmGCBj9ub~bKwLpOcjGQqQoX~cr66SPqUI0TJJnbDvyJO18qOOlqdTW6dhxcpBS~RYDJtLFupwqjv8jJco4m1Y1tE0Wr0eAH6SdP4JwgljKk4ReG3HEBajffz85kiv4eK4LJ7Esahl1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">317</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">37</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss037"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="333081"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f2%2f109%2f333081">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000318584600001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (37)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F2%2F109%2F333081" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/333081'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comprehensive Expression Profile of MicroRNAs and Other Classes of Non-Coding Small RNAs in Barley Under Phosphorous-Deficient and -Sufficient Conditions" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss037");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f109%2f333081">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

