

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/5/261/420676","@type":"ScholarlyArticle","name":"Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder","datePublished":"2009-09-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2009-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp014","keywords":["algorithm","software","transcription factor binding site","ChIP-seq","embryonic stem cells"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/5/261/420676","author":[{"name":"Sharov, Alexei A.","affiliation":"Developmental Genomics and Aging Section, Laboratory of Genetics ,  National Institute on Aging, NIH ,  Baltimore, MD 21224 ,  USA","@type":"Person"},{"name":"Ko, Minoru S.H.","affiliation":"Developmental Genomics and Aging Section, Laboratory of Genetics ,  National Institute on Aging, NIH ,  Baltimore, MD 21224 ,  USA","@type":"Person"}],"description":"Abstract.  We present CisFinder software, which generates a comprehensive list of motifs enriched in a set of DNA sequences and describes them with position fre","pageStart":"261","pageEnd":"273","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder" />
    <meta property="og:description" content="Abstract.  We present CisFinder software, which generates a comprehensive list of motifs enriched in a set of DNA sequences and describes them with position fre" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/5/261/420676" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-09-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sharov, Alexei A." /><meta name="citation_author_institution" content="Developmental Genomics and Aging Section, Laboratory of Genetics, National Institute on Aging, NIH, Baltimore, MD 21224, USA" /><meta name="citation_author" content="Ko, Minoru S.H." /><meta name="citation_author_institution" content="Developmental Genomics and Aging Section, Laboratory of Genetics, National Institute on Aging, NIH, Baltimore, MD 21224, USA" /><meta name="citation_title" content="Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder" /><meta name="citation_firstpage" content="261" /><meta name="citation_lastpage" content="273" /><meta name="citation_doi" content="10.1093/dnares/dsp014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2009/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=SELEX–a (r)evolutionary method to generate high-affinity nucleic acid ligands; Biomol. Eng.;  citation_year=2007;  citation_volume=24;  citation_pages=381-403; " /><meta name="citation_reference" content="citation_title=Diversity and complexity in DNA recognition by transcription factors; Science;  citation_year=2009;  citation_volume=324;  citation_pages=1720-3; " /><meta name="citation_reference" content="citation_title=High-resolution profiling of histone methylations in the human genome; Cell;  citation_year=2007;  citation_volume=129;  citation_pages=823-37; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of in vivo protein–DNA interactions; Science;  citation_year=2007;  citation_volume=316;  citation_pages=1497-502; " /><meta name="citation_reference" content="citation_title=Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=651-7; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of protein–DNA interactions by chromatin immunoprecipitation and DNA microarray hybridization; Methods Mol. Biol.;  citation_year=2003;  citation_volume=224;  citation_pages=99-109; " /><meta name="citation_reference" content="citation_title=Cross-species de novo identification of cis-regulatory modules with GibbsModule: application to gene regulation in embryonic stem cells; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1325-35; " /><meta name="citation_reference" content="citation_title=Variation in homeodomain DNA binding revealed by high-resolution analysis of sequence preferences; Cell;  citation_year=2008;  citation_volume=133;  citation_pages=1266-76; " /><meta name="citation_reference" content="citation_title=Integration of external signaling pathways with the core transcriptional network in embryonic stem cells; Cell;  citation_year=2008;  citation_volume=133;  citation_pages=1106-17; " /><meta name="citation_reference" content="citation_title=The Oct4 and Nanog transcription network regulates pluripotency in mouse embryonic stem cells; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=431-40; " /><meta name="citation_reference" content="citation_title=Computational epigenetics; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=A survey of DNA motif finding algorithms; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=S21" /><meta name="citation_reference" content="citation_title=Improved benchmarks for computational motif discovery; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=193" /><meta name="citation_reference" content="citation_title=Assessing computational tools for the discovery of transcription factor binding sites; Nat. Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=137-44; " /><meta name="citation_reference" content="citation_title=MEME: discovering and analyzing DNA and protein sequence motifs; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W369-73; " /><meta name="citation_reference" content="citation_title=Gibbs Recursive Sampler: finding transcription factor binding sites; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3580-5; " /><meta name="citation_reference" content="citation_title=GAME: detecting cis-regulatory elements using a genetic algorithm; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1577-84; " /><meta name="citation_reference" content="citation_title=A transdimensional Bayesian model for pattern recognition in DNA sequences; Biostatistics;  citation_year=2008;  citation_volume=9;  citation_pages=668-85; " /><meta name="citation_reference" content="citation_title=Extracting sequence features to predict protein–DNA interactions: a comparative study; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=4137-48; " /><meta name="citation_reference" content="citation_title=An algorithm for finding protein–DNA binding sites with applications to chromatin-immunoprecipitation microarray experiments; Nat. Biotechnol.;  citation_year=2002;  citation_volume=20;  citation_pages=835-9; " /><meta name="citation_reference" content="citation_publisher=Freeman, New York; Biometry. The Principles and Practice of Statistics in Biological Research;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Fisher's combined P-value for detecting differentially expressed genes using Affymetrix expression arrays; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=96" /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate—a practical and powerful approach to multiple testing; J. R. Stat. Soc. Ser. B;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=A novel Bayesian DNA motif comparison method for clustering and retrieval; PLoS Comput. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000010" /><meta name="citation_reference" content="citation_title=Quantifying similarity between motifs; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R24" /><meta name="citation_reference" content="citation_title=Extracting regulatory sites from the upstream region of yeast genes by computational analysis of oligonucleotide frequencies; J. Mol. Biol.;  citation_year=1998;  citation_volume=281;  citation_pages=827-42; " /><meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=14863-8; " /><meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6097-100; " /><meta name="citation_reference" content="citation_title=CisFinder;  citation_url=http://lgsun.grc.nia.nih.gov/CisFinder;  citation_URL_Access_Date=;  citation_year=2008; " /><meta name="citation_reference" content="citation_title=MATCH: a tool for searching transcription factor binding sites in DNA sequences; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3576-9; " /><meta name="citation_reference" content="citation_title=MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4878-84; " /><meta name="citation_reference" content="citation_title=Evolution of the mammalian transcription factor binding repertoire via transposable elements; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1752-62; " /><meta name="citation_reference" content="citation_title=JASPAR, the open access database of transcription factor-binding profiles: new content and tools in the 2008 update; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D102-6; " /><meta name="citation_reference" content="citation_title=CisView: a browser and database of cis-regulatory modules predicted in the mouse genome; DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=123-34; " /><meta name="citation_reference" content="citation_title=The UCSC genome browser database; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=51-4; " /><meta name="citation_reference" content="citation_title=WeederH: an algorithm for finding conserved regulatory motifs and regions in homologous sequences; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=46" /><meta name="citation_reference" content="citation_title=Synergism with the coactivator OBF-1 (OCA-B, BOB-1) is mediated by a specific POU dimer configuration; Cell;  citation_year=2000;  citation_volume=103;  citation_pages=853-64; " /><meta name="citation_reference" content="citation_title=New POU dimer configuration mediates antagonistic control of an osteopontin preimplantation enhancer by Oct-4 and Sox-2; Genes Dev.;  citation_year=1998;  citation_volume=12;  citation_pages=2073-90; " /><meta name="citation_reference" content="citation_title=High-throughput biochemical analysis of in vivo location data reveals novel distinct classes of POU5F1(Oct4)/DNA complexes; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=631-9; " /><meta name="citation_reference" content="citation_title=Developmental-specific activity of the FGF-4 enhancer requires the synergistic action of Sox2 and Oct-3; Genes Dev.;  citation_year=1995;  citation_volume=9;  citation_pages=2635-45; " /><meta name="citation_reference" content="citation_title=Target specificities of estrogen receptor-related receptors: analysis of binding sequences and identification of Rb1-inducible coiled-coil 1 (Rb1cc1) as a target gene; J. Biochem.;  citation_year=2008;  citation_volume=143;  citation_pages=395-406; " /><meta name="citation_reference" content="citation_title=The protein CTCF is required for the enhancer blocking activity of vertebrate insulators; Cell;  citation_year=1999;  citation_volume=98;  citation_pages=387-96; " /><meta name="citation_reference" content="citation_title=CTCF is conserved from Drosophila to humans and confers enhancer blocking of the Fab-8 insulator; EMBO Rep.;  citation_year=2005;  citation_volume=6;  citation_pages=165-70; " /><meta name="citation_reference" content="citation_title=Role of CTCF binding sites in the Igf2/H19 imprinting control region; Mol. Cell. Biol.;  citation_year=2004;  citation_volume=24;  citation_pages=4791-800; " /><meta name="citation_reference" content="citation_title=Systematic discovery of regulatory motifs in conserved regions of the human genome, including thousands of CTCF insulator sites; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=7145-50; " /><meta name="citation_reference" content="citation_title=Systematic discovery of regulatory motifs in human promoters and 3’ UTRs by comparison of several mammals; Nature;  citation_year=2005;  citation_volume=434;  citation_pages=338-45; " /><meta name="citation_reference" content="citation_title=TRANSFAC: transcriptional regulation, from patterns to profiles; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=374-8; " /><meta name="citation_reference" content="citation_title=Retroposition and evolution of the DNA-binding motifs of YY1, YY2 and REX1; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=3442-52; " /><meta name="citation_reference" content="citation_title=YY1's longer DNA-binding motifs; Genomics;  citation_year=2009;  citation_volume=93;  citation_pages=152-8; " /><meta name="citation_reference" content="citation_title=Octamer binding proteins confer transcriptional activity in early mouse embryogenesis; EMBO J.;  citation_year=1989;  citation_volume=8;  citation_pages=2551-7; " /><meta name="citation_reference" content="citation_title=Dynamic transcription programs during ES cell differentiation towards mesoderm in serum versus serum-freeBMP4 culture; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=365" /><meta name="citation_reference" content="citation_title=Of mice and men: phylogenetic footprinting aids the discovery of regulatory elements; J. Biol.;  citation_year=2003;  citation_volume=2;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Integrated biclustering of heterogeneous genome-wide datasets for the inference of global regulatory networks; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=280" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/5/261/1165499/dsp014.pdf" /><meta name="description" content="Abstract.  We present CisFinder software, which generates a comprehensive list of motifs enriched in a set of DNA sequences and describes them with position fre" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/5/261/420676" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/5/261/420676" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/5/261/420676");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp014';
    </script>
<title>
	Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./420676" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYZqPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGFyb3YsIEFsZXhlaSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRldmVsb3BtZW50YWwgR2Vub21pY3MgYW5kIEFnaW5nIFNlY3Rpb24sIExhYm9yYXRvcnkgb2YgR2VuZXRpY3MsIE5hdGlvbmFsIEluc3RpdHV0ZSBvbiBBZ2luZywgTklILCBCYWx0aW1vcmUsIE1EIDIxMjI0LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLbywgTWlub3J1IFMuSC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXZlbG9wbWVudGFsIEdlbm9taWNzIGFuZCBBZ2luZyBTZWN0aW9uLCBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzLCBOYXRpb25hbCBJbnN0aXR1dGUgb24gQWdpbmcsIE5JSCwgQmFsdGltb3JlLCBNRCAyMTIyNCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFeGhhdXN0aXZlIFNlYXJjaCBmb3IgT3Zlci1yZXByZXNlbnRlZCBETkEgU2VxdWVuY2UgTW90aWZzIHdpdGggQ2lzRmluZGVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3AwMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwOS8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNFTEVY4oCTYSAocilldm9sdXRpb25hcnkgbWV0aG9kIHRvIGdlbmVyYXRlIGhpZ2gtYWZmaW5pdHkgbnVjbGVpYyBhY2lkIGxpZ2FuZHM7IEJpb21vbC4gRW5nLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM4MS00MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBhbmQgY29tcGxleGl0eSBpbiBETkEgcmVjb2duaXRpb24gYnkgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTE3MjAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIHByb2ZpbGluZyBvZiBoaXN0b25lIG1ldGh5bGF0aW9ucyBpbiB0aGUgaHVtYW4gZ2Vub21lOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEyOTsgIGNpdGF0aW9uX3BhZ2VzPTgyMy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgbWFwcGluZyBvZiBpbiB2aXZvIHByb3RlaW7igJNETkEgaW50ZXJhY3Rpb25zOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNjsgIGNpdGF0aW9uX3BhZ2VzPTE0OTctNTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBwcm9maWxlcyBvZiBTVEFUMSBETkEgYXNzb2NpYXRpb24gdXNpbmcgY2hyb21hdGluIGltbXVub3ByZWNpcGl0YXRpb24gYW5kIG1hc3NpdmVseSBwYXJhbGxlbCBzZXF1ZW5jaW5nOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTY1MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtYXBwaW5nIG9mIHByb3RlaW7igJNETkEgaW50ZXJhY3Rpb25zIGJ5IGNocm9tYXRpbiBpbW11bm9wcmVjaXBpdGF0aW9uIGFuZCBETkEgbWljcm9hcnJheSBoeWJyaWRpemF0aW9uOyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjI0OyAgY2l0YXRpb25fcGFnZXM9OTktMTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Dcm9zcy1zcGVjaWVzIGRlIG5vdm8gaWRlbnRpZmljYXRpb24gb2YgY2lzLXJlZ3VsYXRvcnkgbW9kdWxlcyB3aXRoIEdpYmJzTW9kdWxlOiBhcHBsaWNhdGlvbiB0byBnZW5lIHJlZ3VsYXRpb24gaW4gZW1icnlvbmljIHN0ZW0gY2VsbHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTMyNS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWF0aW9uIGluIGhvbWVvZG9tYWluIEROQSBiaW5kaW5nIHJldmVhbGVkIGJ5IGhpZ2gtcmVzb2x1dGlvbiBhbmFseXNpcyBvZiBzZXF1ZW5jZSBwcmVmZXJlbmNlczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz0xMjY2LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGlvbiBvZiBleHRlcm5hbCBzaWduYWxpbmcgcGF0aHdheXMgd2l0aCB0aGUgY29yZSB0cmFuc2NyaXB0aW9uYWwgbmV0d29yayBpbiBlbWJyeW9uaWMgc3RlbSBjZWxsczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz0xMTA2LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgT2N0NCBhbmQgTmFub2cgdHJhbnNjcmlwdGlvbiBuZXR3b3JrIHJlZ3VsYXRlcyBwbHVyaXBvdGVuY3kgaW4gbW91c2UgZW1icnlvbmljIHN0ZW0gY2VsbHM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NDMxLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGVwaWdlbmV0aWNzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3VydmV5IG9mIEROQSBtb3RpZiBmaW5kaW5nIGFsZ29yaXRobXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9UzIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgYmVuY2htYXJrcyBmb3IgY29tcHV0YXRpb25hbCBtb3RpZiBkaXNjb3Zlcnk7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzaW5nIGNvbXB1dGF0aW9uYWwgdG9vbHMgZm9yIHRoZSBkaXNjb3Zlcnkgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3IgYmluZGluZyBzaXRlczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVNRTogZGlzY292ZXJpbmcgYW5kIGFuYWx5emluZyBETkEgYW5kIHByb3RlaW4gc2VxdWVuY2UgbW90aWZzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XMzY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HaWJicyBSZWN1cnNpdmUgU2FtcGxlcjogZmluZGluZyB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHNpdGVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNTgwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdBTUU6IGRldGVjdGluZyBjaXMtcmVndWxhdG9yeSBlbGVtZW50cyB1c2luZyBhIGdlbmV0aWMgYWxnb3JpdGhtOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE1NzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdHJhbnNkaW1lbnNpb25hbCBCYXllc2lhbiBtb2RlbCBmb3IgcGF0dGVybiByZWNvZ25pdGlvbiBpbiBETkEgc2VxdWVuY2VzOyBCaW9zdGF0aXN0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02NjgtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dHJhY3Rpbmcgc2VxdWVuY2UgZmVhdHVyZXMgdG8gcHJlZGljdCBwcm90ZWlu4oCTRE5BIGludGVyYWN0aW9uczogYSBjb21wYXJhdGl2ZSBzdHVkeTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NDEzNy00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWxnb3JpdGhtIGZvciBmaW5kaW5nIHByb3RlaW7igJNETkEgYmluZGluZyBzaXRlcyB3aXRoIGFwcGxpY2F0aW9ucyB0byBjaHJvbWF0aW4taW1tdW5vcHJlY2lwaXRhdGlvbiBtaWNyb2FycmF5IGV4cGVyaW1lbnRzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9ODM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1GcmVlbWFuLCBOZXcgWW9yazsgQmlvbWV0cnkuIFRoZSBQcmluY2lwbGVzIGFuZCBQcmFjdGljZSBvZiBTdGF0aXN0aWNzIGluIEJpb2xvZ2ljYWwgUmVzZWFyY2g7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpc2hlcidzIGNvbWJpbmVkIFAtdmFsdWUgZm9yIGRldGVjdGluZyBkaWZmZXJlbnRpYWxseSBleHByZXNzZWQgZ2VuZXMgdXNpbmcgQWZmeW1ldHJpeCBleHByZXNzaW9uIGFycmF5czsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz05NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2xsaW5nIHRoZSBmYWxzZSBkaXNjb3ZlcnkgcmF0ZeKAlGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyBKLiBSLiBTdGF0LiBTb2MuIFNlci4gQjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTI4OS0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgQmF5ZXNpYW4gRE5BIG1vdGlmIGNvbXBhcmlzb24gbWV0aG9kIGZvciBjbHVzdGVyaW5nIGFuZCByZXRyaWV2YWw7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGlmeWluZyBzaW1pbGFyaXR5IGJldHdlZW4gbW90aWZzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dHJhY3RpbmcgcmVndWxhdG9yeSBzaXRlcyBmcm9tIHRoZSB1cHN0cmVhbSByZWdpb24gb2YgeWVhc3QgZ2VuZXMgYnkgY29tcHV0YXRpb25hbCBhbmFseXNpcyBvZiBvbGlnb251Y2xlb3RpZGUgZnJlcXVlbmNpZXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjgxOyAgY2l0YXRpb25fcGFnZXM9ODI3LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyIGFuYWx5c2lzIGFuZCBkaXNwbGF5IG9mIGdlbm9tZS13aWRlIGV4cHJlc3Npb24gcGF0dGVybnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xNDg2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBsb2dvczogYSBuZXcgd2F5IHRvIGRpc3BsYXkgY29uc2Vuc3VzIHNlcXVlbmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjA5Ny0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpc0ZpbmRlcjsgIGNpdGF0aW9uX3VybD1odHRwOi8vbGdzdW4uZ3JjLm5pYS5uaWguZ292L0Npc0ZpbmRlcjsgIGNpdGF0aW9uX1VSTF9BY2Nlc3NfRGF0ZT07ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BVENIOiBhIHRvb2wgZm9yIHNlYXJjaGluZyB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHNpdGVzIGluIEROQSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM1NzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF0SW5kIGFuZCBNYXRJbnNwZWN0b3I6IG5ldyBmYXN0IGFuZCB2ZXJzYXRpbGUgdG9vbHMgZm9yIGRldGVjdGlvbiBvZiBjb25zZW5zdXMgbWF0Y2hlcyBpbiBudWNsZW90aWRlIHNlcXVlbmNlIGRhdGE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQ4NzgtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiB0aGUgbWFtbWFsaWFuIHRyYW5zY3JpcHRpb24gZmFjdG9yIGJpbmRpbmcgcmVwZXJ0b2lyZSB2aWEgdHJhbnNwb3NhYmxlIGVsZW1lbnRzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE3NTItNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUpBU1BBUiwgdGhlIG9wZW4gYWNjZXNzIGRhdGFiYXNlIG9mIHRyYW5zY3JpcHRpb24gZmFjdG9yLWJpbmRpbmcgcHJvZmlsZXM6IG5ldyBjb250ZW50IGFuZCB0b29scyBpbiB0aGUgMjAwOCB1cGRhdGU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQxMDItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lzVmlldzogYSBicm93c2VyIGFuZCBkYXRhYmFzZSBvZiBjaXMtcmVndWxhdG9yeSBtb2R1bGVzIHByZWRpY3RlZCBpbiB0aGUgbW91c2UgZ2Vub21lOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEyMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFVDU0MgZ2Vub21lIGJyb3dzZXIgZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTUxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdlZWRlckg6IGFuIGFsZ29yaXRobSBmb3IgZmluZGluZyBjb25zZXJ2ZWQgcmVndWxhdG9yeSBtb3RpZnMgYW5kIHJlZ2lvbnMgaW4gaG9tb2xvZ291cyBzZXF1ZW5jZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5lcmdpc20gd2l0aCB0aGUgY29hY3RpdmF0b3IgT0JGLTEgKE9DQS1CLCBCT0ItMSkgaXMgbWVkaWF0ZWQgYnkgYSBzcGVjaWZpYyBQT1UgZGltZXIgY29uZmlndXJhdGlvbjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz04NTMtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBQT1UgZGltZXIgY29uZmlndXJhdGlvbiBtZWRpYXRlcyBhbnRhZ29uaXN0aWMgY29udHJvbCBvZiBhbiBvc3Rlb3BvbnRpbiBwcmVpbXBsYW50YXRpb24gZW5oYW5jZXIgYnkgT2N0LTQgYW5kIFNveC0yOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjA3My05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGJpb2NoZW1pY2FsIGFuYWx5c2lzIG9mIGluIHZpdm8gbG9jYXRpb24gZGF0YSByZXZlYWxzIG5vdmVsIGRpc3RpbmN0IGNsYXNzZXMgb2YgUE9VNUYxKE9jdDQpL0ROQSBjb21wbGV4ZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50YWwtc3BlY2lmaWMgYWN0aXZpdHkgb2YgdGhlIEZHRi00IGVuaGFuY2VyIHJlcXVpcmVzIHRoZSBzeW5lcmdpc3RpYyBhY3Rpb24gb2YgU294MiBhbmQgT2N0LTM7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI2MzUtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldCBzcGVjaWZpY2l0aWVzIG9mIGVzdHJvZ2VuIHJlY2VwdG9yLXJlbGF0ZWQgcmVjZXB0b3JzOiBhbmFseXNpcyBvZiBiaW5kaW5nIHNlcXVlbmNlcyBhbmQgaWRlbnRpZmljYXRpb24gb2YgUmIxLWluZHVjaWJsZSBjb2lsZWQtY29pbCAxIChSYjFjYzEpIGFzIGEgdGFyZ2V0IGdlbmU7IEouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0MzsgIGNpdGF0aW9uX3BhZ2VzPTM5NS00MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcm90ZWluIENUQ0YgaXMgcmVxdWlyZWQgZm9yIHRoZSBlbmhhbmNlciBibG9ja2luZyBhY3Rpdml0eSBvZiB2ZXJ0ZWJyYXRlIGluc3VsYXRvcnM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0zODctOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNUQ0YgaXMgY29uc2VydmVkIGZyb20gRHJvc29waGlsYSB0byBodW1hbnMgYW5kIGNvbmZlcnMgZW5oYW5jZXIgYmxvY2tpbmcgb2YgdGhlIEZhYi04IGluc3VsYXRvcjsgRU1CTyBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNjUtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgQ1RDRiBiaW5kaW5nIHNpdGVzIGluIHRoZSBJZ2YyL0gxOSBpbXByaW50aW5nIGNvbnRyb2wgcmVnaW9uOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDc5MS04MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgZGlzY292ZXJ5IG9mIHJlZ3VsYXRvcnkgbW90aWZzIGluIGNvbnNlcnZlZCByZWdpb25zIG9mIHRoZSBodW1hbiBnZW5vbWUsIGluY2x1ZGluZyB0aG91c2FuZHMgb2YgQ1RDRiBpbnN1bGF0b3Igc2l0ZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NzE0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBkaXNjb3Zlcnkgb2YgcmVndWxhdG9yeSBtb3RpZnMgaW4gaHVtYW4gcHJvbW90ZXJzIGFuZCAz4oCZIFVUUnMgYnkgY29tcGFyaXNvbiBvZiBzZXZlcmFsIG1hbW1hbHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzQ7ICBjaXRhdGlvbl9wYWdlcz0zMzgtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRSQU5TRkFDOiB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdGlvbiwgZnJvbSBwYXR0ZXJucyB0byBwcm9maWxlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9Mzc0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldHJvcG9zaXRpb24gYW5kIGV2b2x1dGlvbiBvZiB0aGUgRE5BLWJpbmRpbmcgbW90aWZzIG9mIFlZMSwgWVkyIGFuZCBSRVgxOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0zNDQyLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ZWTEncyBsb25nZXIgRE5BLWJpbmRpbmcgbW90aWZzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTE1Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PY3RhbWVyIGJpbmRpbmcgcHJvdGVpbnMgY29uZmVyIHRyYW5zY3JpcHRpb25hbCBhY3Rpdml0eSBpbiBlYXJseSBtb3VzZSBlbWJyeW9nZW5lc2lzOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yNTUxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgdHJhbnNjcmlwdGlvbiBwcm9ncmFtcyBkdXJpbmcgRVMgY2VsbCBkaWZmZXJlbnRpYXRpb24gdG93YXJkcyBtZXNvZGVybSBpbiBzZXJ1bSB2ZXJzdXMgc2VydW0tZnJlZUJNUDQgY3VsdHVyZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PZiBtaWNlIGFuZCBtZW46IHBoeWxvZ2VuZXRpYyBmb290cHJpbnRpbmcgYWlkcyB0aGUgZGlzY292ZXJ5IG9mIHJlZ3VsYXRvcnkgZWxlbWVudHM7IEouIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0ZWQgYmljbHVzdGVyaW5nIG9mIGhldGVyb2dlbmVvdXMgZ2Vub21lLXdpZGUgZGF0YXNldHMgZm9yIHRoZSBpbmZlcmVuY2Ugb2YgZ2xvYmFsIHJlZ3VsYXRvcnkgbmV0d29ya3M7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzUvMjYxLzExNjU0OTkvZHNwMDE0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBwcmVzZW50IENpc0ZpbmRlciBzb2Z0d2FyZSwgd2hpY2ggZ2VuZXJhdGVzIGEgY29tcHJlaGVuc2l2ZSBsaXN0IG9mIG1vdGlmcyBlbnJpY2hlZCBpbiBhIHNldCBvZiBETkEgc2VxdWVuY2VzIGFuZCBkZXNjcmliZXMgdGhlbSB3aXRoIHBvc2l0aW9uIGZyZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDIwNjc2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKxTwhYGfnCutQdyK6OASIx3nRFW" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f261%2f420676" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f261%2f420676" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&amp;Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/5">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6232246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232246">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232248">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232248">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232252">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232252">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232273">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232273">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232310">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232310">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232312">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232312">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232314">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232314">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232316">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232316">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/5/249/421165">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/5/275/421566">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexei A. Sharov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Developmental Genomics and Aging Section, Laboratory of Genetics</div>, <div class="insititution">National Institute on Aging, NIH</div>, <div class="addr-line">Baltimore, MD 21224</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alexei+A.+Sharov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sharov A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sharov A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alexei A. Sharov</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Minoru S.H. Ko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Developmental Genomics and Aging Section, Laboratory of Genetics</div>, <div class="insititution">National Institute on Aging, NIH</div>, <div class="addr-line">Baltimore, MD 21224</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 410-558-8359. Fax. <div class="fax">+1 410-558-8331</div>. E-mail: <a href="mailto:kom@mail.nih.gov" target="_blank">kom@mail.nih.gov</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Minoru+S.H.+Ko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ko M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ko M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Minoru S.H. Ko</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 261–273, <a href='https://doi.org/10.1093/dnares/dsp014'>https://doi.org/10.1093/dnares/dsp014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 September 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 May 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 July 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="420676" href="/dnaresearch/article-pdf/16/5/261/1165499/dsp014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Alexei A. Sharov, Minoru S.H. Ko;  Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder, <em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 261–273, <a href='https://doi.org/10.1093/dnares/dsp014'>https://doi.org/10.1093/dnares/dsp014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/420676?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/420676?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/420676?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/420676?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/420676?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/420676?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/420676?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Exhaustive%20Search%20for%20Over-represented%20DNA%20Sequence%20Motifs%20with%20CisFinder&amp;publicationDate=2009-09-09&amp;volumeNum=16&amp;issueNum=5&amp;author=Sharov%2C%20Alexei%20A.%3B%20Ko%2C%20Minoru%20S.H.&amp;startPage=261&amp;endPage=273&amp;contentId=10.1093%2Fdnares%2Fdsp014&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f261%2f420676" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6232246 id="6232246" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We present CisFinder software, which generates a comprehensive list of motifs enriched in a set of DNA sequences and describes them with position frequency matrices (PFMs). A new algorithm was designed to estimate PFMs directly from counts of <em>n</em>-mer words with and without gaps; then PFMs are extended over gaps and flanking regions and clustered to generate non-redundant sets of motifs. The algorithm successfully identified binding motifs for 12 transcription factors (TFs) in embryonic stem cells based on published chromatin immunoprecipitation sequencing data. Furthermore, CisFinder successfully identified alternative binding motifs of TFs (e.g. POU5F1, ESRRB, and CTCF) and motifs for known and unknown co-factors of genes associated with the pluripotent state of ES cells. CisFinder also showed robust performance in the identification of motifs that were only slightly enriched in a set of DNA sequences.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="algorithm">algorithm</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="software">software</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factor binding site&quot;">transcription factor binding site</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ChIP-seq">ChIP-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;embryonic stem cells&quot;">embryonic stem cells</a></div>                            <h2 scrollto-destination=6232248 id="6232248" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Transcription factor (TF) binding motifs in eukaryotes have been identified by examining binding sequences of purified TFs (e.g. SELEX<sup><a reveal-id="DSP014C1" data-open="DSP014C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and Protein Binding Microarrays<sup><a reveal-id="DSP014C2" data-open="DSP014C2" class="link link-ref link-reveal xref-bibr">2</a></sup>) and by carrying out chromatin immunoprecipitation coupled with massively parallel sequencing (ChIP-seq<sup><a reveal-id="DSP014C3  DSP014C5" data-open="DSP014C3  DSP014C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup>) and microarray (ChIP-chip).<sup><a reveal-id="DSP014C6" data-open="DSP014C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The ChIP methods can account for biological context of TF binding<sup><a reveal-id="DSP014C7  DSP014C10" data-open="DSP014C7  DSP014C10" class="link link-ref link-reveal xref-bibr">7–10</a></sup> because many TFs require co-factors for sequence-specific binding to DNA, which are not present in <em>in vitro</em> assays. On the other hand, TF binding sites identified in the ChIP methods will include not only direct binding sites but also binding sites indirectly associated with the TF through the protein–protein interaction of other TFs that binds directly to DNA. Furthermore, ChIP-seq data often include several million sequence tags and &gt;10 000 binding locations.<sup><a reveal-id="DSP014C4" data-open="DSP014C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSP014C11" data-open="DSP014C11" class="link link-ref link-reveal xref-bibr">11</a></sup> These features of high-throughput genome-wide ChIP technology make the bioinformatic task of identifying TF binding motifs a great challenge.</p><p>Various software tools have been developed to identify over-represented DNA sequence motifs (reviewed in Das and Dai,<sup><a reveal-id="DSP014C12" data-open="DSP014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Sandve <em>et al</em>.,<sup><a reveal-id="DSP014C13" data-open="DSP014C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and Tompa <em>et al</em>.<sup><a reveal-id="DSP014C14" data-open="DSP014C14" class="link link-ref link-reveal xref-bibr">14</a></sup>). For example, traditional probabilistic methods include expectation maximization (MEME<sup><a reveal-id="DSP014C15" data-open="DSP014C15" class="link link-ref link-reveal xref-bibr">15</a></sup>), Gibbs sampling,<sup><a reveal-id="DSP014C7" data-open="DSP014C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP014C16" data-open="DSP014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> genetic algorithms (GAME<sup><a reveal-id="DSP014C17" data-open="DSP014C17" class="link link-ref link-reveal xref-bibr">17</a></sup>), integrated Bayesian models,<sup><a reveal-id="DSP014C18" data-open="DSP014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> neural networks, support vector machines, Bayesian additive regression trees,<sup><a reveal-id="DSP014C19" data-open="DSP014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and approximate maximum a posteriory (MAP) scoring functions.<sup><a reveal-id="DSP014C20" data-open="DSP014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> These methods work well when data sets are small, and thus, only a small fraction of top-scored binding sites is usually processed with these algorithms.<sup><a reveal-id="DSP014C10" data-open="DSP014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Weeder, which is based on counting matching patterns with a certain maximum number of mismatches, has been reported to outperform many other software tools.<sup><a reveal-id="DSP014C14" data-open="DSP014C14" class="link link-ref link-reveal xref-bibr">14</a></sup> However, most existing algorithms are limited to searching only for a single motif at a time. To find additional motifs, the software has to be run again after removing the first motif from the sequence.<sup><a reveal-id="DSP014C15" data-open="DSP014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> With this approach, results may be different depending on the order in which motifs are processed. For example, a composite motif that supports binding of two TFs (TF<sub>1</sub> and TF<sub>2</sub>) may be lost if a more abundant motif (TF<sub>1</sub>) is processed first and then removed from the sequence. Machine-learning algorithms, such as Gibbs sampler and neural networks, tend to fall into local maxima<sup><a reveal-id="DSP014C7" data-open="DSP014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and often fail to differentiate between similar motifs.</p><p>In this paper, we present a new algorithm for <em>de novo</em> identification of over-represented DNA motifs, which is implemented as the online software tool CisFinder (<a class="link link-uri" href="http://lgsun.grc.nia.nih.gov/CisFinder">http://lgsun.grc.nia.nih.gov/CisFinder</a>). It is a complementary method to existing probabilistic algorithms and has advantages in the exploratory analysis of large input files typical for ChIP-chip or ChIP-seq data sets. CisFinder can effectively process large sequences (up to 50 Mb), extract a comprehensive list of over-represented motifs in a single run, and analyze data with poor enrichment of DNA-binding motifs. Because of high processing speed (&lt;1 min for complete data analyses), the software can be used in an interactive manner to test many different parameter sets. The software has been tested using available ChIP-seq data on TFs expressed in ES cells.<sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                            <h2 scrollto-destination=6232252 id="6232252" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6232253 id="6232253" class="section-title" data-legacy-id=s2a>2.1 Estimating position frequency matrices from n-mer word counts</h3>
<div class="block-child-p">The proposed algorithm is based on estimating position frequency matrices (PFMs) directly from <em>n</em>-mer word counts in the test set and control set of sequences. To explain the algorithm, we first describe a numerical example and then present the formal justification of the method. Consider a specific <em>n</em>-mer word <em>W</em> (e.g. <em>W</em> = ‘ATGCAAAT’), which has <em>T</em>(<em>W</em>) = 200 matches (instances) in the set of test sequences and <em>C</em>(<em>W</em>) = 50 matches in the set of control sequences. For simplicity, we count only a total number of instances as if all sequences in a set are concatenated. (However, the CisFinder has an option to count only one match of each word per sequence.) In this example, the total length of both test and control sequences is 3 Mb. For a word <em>W</em>, we define a nucleotide substitution matrix [<em>W</em><sub><em>pi</em></sub>], which contains words that are derived from <em>W</em> by placing a nucleotide <em>i</em> in a position <em>p</em> (Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>A). The frequency of each word from the nucleotide substitution matrix counted in the same target sequence makes the frequency substitution matrix (Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>B). For convenience, we will use brief notations <em>T</em><sub><em>pi</em></sub> = <em>T</em>(<em>W</em><sub><em>pi</em></sub>) and <em>C</em><sub><em>pi</em></sub> = <em>C</em>(<em>W</em><sub><em>pi</em></sub>) for the frequency of word <em>W</em><sub><em>pi</em></sub> instances in the test and control sequence sets (elements of frequency substitution matrices). Then, the proposed method to estimate PFMs is <a id="DSP014eqn1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP014eqn1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp014eqn1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqn1.gif?Expires=2147483647&amp;Signature=A8crawGnK0wbeq5wj1ITO7O~T26ral9uViESGiwLhXgrDlpIf6ZF8jr0JO7pRkrHNRFnguLXfE93cg6mDkogcOdVr5o8MQ2McQrF3GH35ECL9hohKaQJAe6SLcCslqdZGwIgLdzeWsM2IRqwdHYTJynnPuRryAjluzokl2CH7jxzD71U7oUOK8tqJhY6j-i59eWfVL-8MXgJybjX9lX-BTHoxuvLq1DShxK78awUXMGB1DZETjl-hHYhT6rWFYI2oL8sOFjK~K0kfqY8~8GZAF1c5sQA2g9RBSKlO4qdXuJNHzNciUfVOX6HEI7wq0DKjRsEwasjPfRWf189rsM7FQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqn1.gif" /></a></div></div><div class="label">e1</div></div> where <em>φ</em><sub><em>pi</em></sub> is the estimate of PFM element, and <em>T</em><sub><em>pi</em></sub> and <em>C</em><sub><em>pi</em></sub> are the counts of word <em>W</em><sub><em>pi</em></sub>, in the test and control sequences, respectively. Because word counts are random variables, they may appear smaller in the test sequences than in the control sequences by chance, resulting in a negative PFM element. To avoid this, negative differences are replaced with zero and then normalized as shown in Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>C–E. Thus, the estimate of PFM element (<em>φ</em><sub><em>pi</em></sub>) can now be presented as follows:<a id="DSP014eqn2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP014eqn2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp014eqn2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqn2.gif?Expires=2147483647&amp;Signature=tyc70T~mwm0c12BAhAAY07EkCTAGov3iU4AujwJNmXINBpE00iyiYJqNrFI6uh5zkfx~hJy9QSF4FK29KG1-r-4OX8s-KgZNCvMST5uNVK41UzyvTM0fh~APsZ5iNVGzHkjWz5xoCxU5brwd4n-aP~5yP8BGkHM2jQ4yzkmQTLHachNUYwCT2cK2KYMlkNowbylI0daT8VUk1Zs-k8oerk-Pw0~RhztPZTsj569aHZF~mnT3d2AyhQWQ1jQU-BRV3PhZE2XgnnNir-AyGL6Bzi5EGLINH5CLBqEFF-VI5~Nv5lBRxkMCusYoMiKjN9xp39esCAPTb0BRQqULLuELAg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqn2.gif" /></a></div></div><div class="label">e2</div></div></div>                        <a id="6232255" scrollto-destination="6232255"></a>
<div id="DSP014F1" data-id="DSP014F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01401.jpeg?Expires=2147483647&Signature=NN~W6xjoXOxFFgZWinCJJSJhw~11CzZGxuV4TStLrnKSt9Z8MMm0uDgOj~yyY3vfZLZM2mzQYRiRsSuJKtpKlqLMdsGLuJfwAs2Cbz-l2MFJe3Ym9IX1fqJ~a75cAKXuEko1ZvD~8q0iDIZXucyU~HOu8uXhJXBhcph8J44Ho7rcMgxTIsJpmwOT~oRlly75KNIL8wAMT7jrYAEEK07iWGMGqffH5kBrH7ffHfex1nEqxldPCLcmdYanmuqO6UhyQCJUbma0YLK9kMlbSDewkz1u7SWEdLrnoNJVfKCqg-N2cuThhmxewfC4zLAkkqjvHlLgC-62gutlxAOJfliYzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CisFinder algorithm for de novo identification of DNA motifs. (A) Example of a nucleotide substitution matrix for word ATGCAAAT; (B) frequency substitution matrices for the test and control sequences; (C) subtraction of matrices; (D) negative values are replaced by zero; (E) normalized PFM; (F) position and width of gaps in the words; (G) extending the PFM over the gaps and flanking sequences; (H) clustering and combining of PFMs to generate a sequence logo." path-from-xml="dsp01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232255/dsp01401.jpeg" path-from-xml="dsp01401.jpeg" target="_blank">View large</a><a section="6232255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp01401.jpeg?Expires=2147483647&Signature=Yu72rQOboWqQlW7frEgDP7aH2C514oXO0H4qSiIGZJiF1Hz1B6y~ng8SaZJCxK4HJ977Q9UVKtMoYxOLk1XWTxMKk8Kri-G2wrXx4c8xu6~eTUgDUUf~3FjynmErv49D4tITFdoE7aFTwWLexNunKuMvYedKU6qF~K3htgIR~FUT6Sja-7IyH2JPr4AZl7sSGYDRroqhNRL3eLPW8ehUpdBen7HNhqJIFRokKFS9O~meCJp8fdnCRe0fhdnmmV7rY5ihJfnehiADs7KPH3QGFykYkP~3~3SYI4TR~1lTtj048KnwuCZlcVS~CKzl4oQ1lAuO4r5qjmfSXOUpm6ykQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232255&ar=420676&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CisFinder algorithm for <em>de novo</em> identification of DNA motifs. (A) Example of a nucleotide substitution matrix for word ATGCAAAT; (B) frequency substitution matrices for the test and control sequences; (C) subtraction of matrices; (D) negative values are replaced by zero; (E) normalized PFM; (F) position and width of gaps in the words; (G) extending the PFM over the gaps and flanking sequences; (H) clustering and combining of PFMs to generate a sequence logo.</p></div></div><div content-id="DSP014F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01401.jpeg?Expires=2147483647&Signature=NN~W6xjoXOxFFgZWinCJJSJhw~11CzZGxuV4TStLrnKSt9Z8MMm0uDgOj~yyY3vfZLZM2mzQYRiRsSuJKtpKlqLMdsGLuJfwAs2Cbz-l2MFJe3Ym9IX1fqJ~a75cAKXuEko1ZvD~8q0iDIZXucyU~HOu8uXhJXBhcph8J44Ho7rcMgxTIsJpmwOT~oRlly75KNIL8wAMT7jrYAEEK07iWGMGqffH5kBrH7ffHfex1nEqxldPCLcmdYanmuqO6UhyQCJUbma0YLK9kMlbSDewkz1u7SWEdLrnoNJVfKCqg-N2cuThhmxewfC4zLAkkqjvHlLgC-62gutlxAOJfliYzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CisFinder algorithm for de novo identification of DNA motifs. (A) Example of a nucleotide substitution matrix for word ATGCAAAT; (B) frequency substitution matrices for the test and control sequences; (C) subtraction of matrices; (D) negative values are replaced by zero; (E) normalized PFM; (F) position and width of gaps in the words; (G) extending the PFM over the gaps and flanking sequences; (H) clustering and combining of PFMs to generate a sequence logo." path-from-xml="dsp01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232255/dsp01401.jpeg" path-from-xml="dsp01401.jpeg" target="_blank">View large</a><a section="6232255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp01401.jpeg?Expires=2147483647&Signature=Yu72rQOboWqQlW7frEgDP7aH2C514oXO0H4qSiIGZJiF1Hz1B6y~ng8SaZJCxK4HJ977Q9UVKtMoYxOLk1XWTxMKk8Kri-G2wrXx4c8xu6~eTUgDUUf~3FjynmErv49D4tITFdoE7aFTwWLexNunKuMvYedKU6qF~K3htgIR~FUT6Sja-7IyH2JPr4AZl7sSGYDRroqhNRL3eLPW8ehUpdBen7HNhqJIFRokKFS9O~meCJp8fdnCRe0fhdnmmV7rY5ihJfnehiADs7KPH3QGFykYkP~3~3SYI4TR~1lTtj048KnwuCZlcVS~CKzl4oQ1lAuO4r5qjmfSXOUpm6ykQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232255&ar=420676&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CisFinder algorithm for <em>de novo</em> identification of DNA motifs. (A) Example of a nucleotide substitution matrix for word ATGCAAAT; (B) frequency substitution matrices for the test and control sequences; (C) subtraction of matrices; (D) negative values are replaced by zero; (E) normalized PFM; (F) position and width of gaps in the words; (G) extending the PFM over the gaps and flanking sequences; (H) clustering and combining of PFMs to generate a sequence logo.</p></div></div><p>If test and control sequence sets have different total lengths, then the number of word counts in the control sequences is adjusted by the total sequence length.</p><div class="block-child-p">This method is justified by the following model. Let us assume that a TF binds to a set of locations in the genome where corresponding DNA sequences can be aligned together. Using this alignment, we can estimate the frequency, <em>f</em><sub><em>pi</em></sub>, of each nucleotide <em>i</em> in each position of aligned sequences, <em>p</em>, which is the element of the PFM. We further assume that binding strength of the TF is additive with no interaction between positions. This simplification is justified by the fact that all existing databases use PFMs to describe TF motifs, and this strategy works reasonably well. Consider a word <em>W</em> with a sequence of nucleotides that corresponds to the maximum values of the PFM at each position. This word is then used to generate frequency substitution matrices [<em>T</em><sub><em>pi</em></sub>] and [<em>C</em><sub><em>pi</em></sub>] for the test and control sets of sequences, respectively. Each instance of word <em>W</em><sub><em>pi</em></sub> in the test or control sequences can either correspond to a true binding site of the TF (we call it functional) or not (non-functional). Factors determining the functionality of different instances of the same DNA word are largely unknown and may include sequence context and chromatin status. Because the probability of TF binding is proportional to PFM elements at each position (based on the assumption of additive contribution of each position to TF binding), the number of functional instances, FT(<em>W</em><sub><em>pi</em></sub>), of word <em>W</em><sub><em>pi</em></sub> in the test sequences is proportional to <em>f</em><sub><em>pi</em></sub>:<a id="DSP014eqn3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP014eqn3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp014eqn3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqn3.gif?Expires=2147483647&amp;Signature=uqWOdilJ7Cudy1rd1B32upI6VXSVrAK-iuNyDW6HEQz24hjJFKOryFHvSXn5Fd46l8aMUWv6ZpKfybvT8YTNHJQ2qqkfOzGkub0gZIdSo7I2h60VdzdS5M2cTdwUNI~i457yUdsDEtmumWqbaqOyTCI0WPfSj28SwTuUQPaukpyPGICXqzpMSysFMuBN5tsrmwx7JCxeGdaYzlW1LASjkIY67ci1u-LvN0K-nxIEX5VcpVVgGujFAPOPUBj2mwc8HV6GqYSKThSNLkuDjacBmi4UORwdE3L43i~jH0tYf8tVHhrdjVQcJlMXL6MjkYufkOTjtOojeNvWfDI~B3b7Bw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqn3.gif" /></a></div></div><div class="label">e3</div></div> The total number of instance of word <em>W</em><sub><em>pi</em></sub> in test sequences equals the sum of functional, FT(<em>W</em><sub><em>pi</em></sub>), and non-functional, NT(<em>W</em><sub><em>pi</em></sub>), instances:<a id="DSP014eqn4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP014eqn4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp014eqn4.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqn4.gif?Expires=2147483647&amp;Signature=cODBHdEr01k0Bdhh7tWtzvHTa72nZAzNnLkj40Fl~GHlemtOHWIB8c5Lf9e9wcz89pi1nR5Y0i8ZimlJQd6FKjoPeqXTE5YBZzbtlo97bK0x7DYLByGkzDckoUqcxbyLOY0z7btpbGH7S8-3EwmDPk5G6t~P~yK9qk8Rxh61jsOrrd2sgYiUnveJhRHhQz04LscC3GGTlG6wcnxRqn7haaDyDbLmcmC8BuEpib1956dp7RKhZxETOyVr-fikXCDTs-DI1f9jF4N7JPgseUZZzBnNuAwaypia2wI07np4bx97X~mBsDHZliBzh834Ty5xaNthYOBFxQG9RQ~L5ahDHg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqn4.gif" /></a></div></div><div class="label">e4</div></div></div><div class="block-child-p">Similarly, the total number of instance of word <em>W</em><sub><em>pi</em></sub> in control sequences equals the sum of functional, FC(<em>W</em><sub><em>pi</em></sub>), and non-functional, NC(<em>W</em><sub><em>pi</em></sub>), instances. Although the functional instances are enriched in the test sequences compared with control sequences, some functional instances may be present in the control sequences, because of possible false negatives in ChIP data. Because we can assume that non-functional instances of the word are not affected by the ChIP procedure, their counts are equal in the test and control sets of sequences: NT(<em>W</em><sub><em>pi</em></sub>) = NC(<em>W</em><sub><em>pi</em></sub>). Then, the numerator in Equation (<a href="#DSP014eqn1" class="sectionLink xref-sec">e1</a>) is <a id="DSP014eqn5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP014eqn5"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp014eqn5.gif?Expires=2147483647&Signature=wHQkt0rR~GG2GaZmaKlMBMjFfK6jdVd4eTncjDK3~anKIKPRyhlpXWmuxEZX3jl-h53o6RyehN2JLQuQoUsNs02XOD0ZKqcKyrXjRRJg47nljz7TM8EwGrN5RDDRI0zVcrHY49TjGnEVrGo6unAFMBa1~4ZqF5SJLPSwCnAnzQKqVSa3lk1c-5sH2IpUEliuw~sz385pMsqxYmsUsFAqMCQA7WMexitw8NnLWMoZFSd19Sv9XtcNeE0FtBCAKblqhFlkhPwDVnL8xBOAamvBn7I9pFdbc5w12uHcH-x4o7pRXvHZGDj6bCoZhumu3w1l4bIr-TNvR4PHmbkdV1IryA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp014eqn5.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqn5.gif?Expires=2147483647&Signature=0dULd2VOSAUrA9RSOSeZEF9Hx2iEXQUGChigyxVU~SWKL6GQlk37knaBBprSra3Ho7R~6ojSHrJ5t0QL81foJNEJvQDkuyt2IO0k1IA4gQk97HyR1BJjNB3Xc-bjqMlCCbiXXkPkuVnSWqvLl8eWvyuzxgNzgGjhGnCp1aNajMpH3xlrRywLIzEJTAUi2oq6LOfUZMZOK17CDxjAH2QdIu376GGsPa2Pf1DZYGCu6JluiYLpVu~iC6WBlRXlBBaR0Pyq8p7ZStgHTigscE9zV6I7Zi4p5cBJTM48iRysR-JL8ehRMbB-aicFT4WbqSMxc34SD2hEo1xGlRMXCKz2JA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqn5.gif" /></a></div></div><div class="label">e5</div></div> Because functional instances of word <em>W</em> are over-represented in the test set of sequences compared with control, the final sum in Equation (<a href="#DSP014eqn5" class="sectionLink xref-sec">e5</a>) is always positive and the difference (<em>T</em><sub><em>pi</em></sub> − <em>C</em><sub><em>pi</em></sub>) is proportional to <em>f</em><sub><em>pi</em></sub>. Thus, Equation (<a href="#DSP014eqn1" class="sectionLink xref-sec">e1</a>) gives a true estimate of <em>f</em><sub><em>pi</em></sub> in the PFM. This reasoning holds true, if the word <em>W</em> is shorter than the full binding motif or includes a gap. However, the word should be long enough to capture the informative portion of the motif so that it remains strongly over-represented in the set of test sequences compared with control.</div><p>Because the PFM is estimated as a difference between word counts in the test and control sets of sequences [Equations (<a href="#DSP014eqn1" class="sectionLink xref-sec">e1</a>) and (<a href="#DSP014eqn2" class="sectionLink xref-sec">e2</a>)], the variance of PFM elements is equal to the sum of variances of word counts in the test and control sequences. The variance of word counts is very close to the mean, which is expected from the Poisson distribution. This was also checked using pseudo-random sequences generated with the third order Markov process. For example, if word counts are 120 in the test set of sequences and 40 in the control set (i.e. 3-fold over-representation), then the relative error (accuracy) is equal to sqrt(120 + 40)/(120–40) = 0.158.</p>                            <h3 scrollto-destination=6232260 id="6232260" class="section-title" data-legacy-id=s2b>2.2 Implementation of the method for PFM estimation</h3>
<p>A successful ChIP-seq experiment generates a set of genome locations that are enriched in TF binding sites. For a test sequence set, we usually extract 200 bp sequence segments centered at a peak of projected TF binding sites. For a control sequence set, we usually extract 500 bp sequence segments starting from nucleotide positions 400 bp away from both ends of 200 bp test sequence segments. (However, the CisFinder allows users to choose different sequence lengths.) The CisFinder identifies binding motifs of TFs using direct counts of all possible 8-mer words with and without gaps in both test and control sequence sets (Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>F). This word length was selected experimentally based on the observation that longer words have too few matches in target sequences, whereas shorter words may fail to capture the most informative portion of the motif and show lower rates of over-representation. (Note: the command-line version of CisFinder allows the use of 6- and 10-mer words.) Word counts are stored in the array of integers. Although there are many different ways to insert gaps in the 8-mer words, we consider only 8 specific patterns of gap insertions (Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>F). We found that this limited set of gap insertion effectively helps to capture composite motifs with multiple functional elements. For example, search for a word ‘ATGCAAAT’ with a 2 bp gap in the middle is equivalent to the search for word ‘ATGCNNAAAT’. PFM is then estimated for each word based on &gt;1.5-fold (default threshold) enrichment in the test sequences compared with the control sequences using Equation (<a href="#DSP014eqn2" class="sectionLink xref-sec">e2</a>). The adjustable fold enrichment criterion is optional (it can be set to 1), which provides additional flexibility in the use of the program.</p><p>Over-representation of word counts in the test sequences compared with the control sequences is then evaluated using a <em>z</em>-score which is estimated based on the hypergeometric probability distribution. Let us first consider the case where only one instance of each word is counted per each sequence of equal (or approximately equal) length. The proportion of sequences, <em>q</em>, with a given word in the set of test sequences is compared with the proportion of sequences, <em>p</em>, with the same word in the combined set of test and control sequences (if the null-hypothesis is true, then test and control sets of sequences can be combined) with <em>z</em>-score: <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01404.gif?Expires=2147483647&Signature=NtdIhnHTEbnr06u0bBTutMOVkCo1KM0D25xvzKzmUjzQQeBgyWAvFSfhXnpmuCKwYXXeG5vzQ6B82UqtlEQNWIki3K1-qKhShju36LnKnXnXp9gBrFr5U81nx4XuruDWjgPPwLLldVkMr5z6ulXf0ahuhy-GKCSoTJobrUK~y8JL0X-~lBb2aBkDFM7yUdU0hpkXw7PnwbfIyDZUZem4xoRk51Bm7vdvgIcOjbaN0mjrTYNVjdgDaRdLVR-2~t1x4D0VfwRqPgFp6T8lxbCUJH2nMerOCS1TP~MoCFxj4Dt0FxNjQYy8Q4O5AFlzQrz1Bn~Erm7yn2k1lQFIzhucAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp01404.gif" /></span> where <em>n</em> is the number of test sequences, and <em>N</em> is the number of combined test and control sequences. If multiple instances of each word are counted per each sequence, then the method is modified as follows. The set of test sequences with the total length <em>T</em> is split into <em>T</em>/<em>m</em> segments of length <em>m</em>, where <em>m</em> is the actual length of the word including gaps. Each instance of the word is then associated with the segment where it starts. Because overlapping instances of the same word are counted as one instance, there is not more than one instance associated with the same segment. Similarly, the set of control sequences with the total length <em>C</em> is split into <em>C</em>/<em>m</em> segments of length <em>m</em>. We use the same equation for the <em>z</em>-score (see above) where <em>q</em> is the proportion of test segments with the word, <em>p</em> is the proportion of test and control segments with the word, <em>n</em> = <em>T</em>/<em>m</em>, and <em>N</em> = (<em>T</em> + <em>C</em>)/<em>m</em>. Although occurrences of word instances in adjacent segments may be weakly correlated, the hypergeometric distribution gives a reasonable approximation of the <em>z</em>-score.</p><p>To fill the gaps and extend the length of PFMs, the test and control sequences are searched again for the exact match to each word with <em>z</em> &gt; 1.643 (to satisfy the condition of <em>P</em> &lt; 0.05 for one-tail <em>z</em>-test). Each match of the word in the test (or control) sequence is then examined for nucleotides in the gaps and flanking sequences (2 bp at each side) that are not included in the word. In this way, we can count nucleotide frequencies in gaps and flanking regions and estimate the PFM for these positions using Equation (<a href="#DSP014eqn2" class="sectionLink xref-sec">e2</a>) (Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>G). The program is also designed to trim flanking sequences if they are not informative (if the ratio of maximum frequency to minimum frequency is &lt;3). To increase the information content of PFMs, we use the contrasting procedure: the median of minimum PFM values at each position is subtracted from all PFM values; negative values are then replaced by zero; and the PFM is re-normalized.</p><p>The frequency distribution of nucleotides in the flanking regions and in gaps of a certain word may differ substantially between the test and control sets of sequences. In such a case, this difference can be used to increase the statistical power for identification of significant motifs. To incorporate this factor into the statistical evaluation of motif significance, we compared frequency distributions of nucleotides (counted for each nucleotide and each flanking/gap position) in the test and control sequences using the G-test.<sup><a reveal-id="DSP014C21" data-open="DSP014C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Assuming that this test is independent from the <em>z</em>-test for over-representation of word counts (see above), we combined <em>P</em>-values from these tests using Fisher's method.<sup><a reveal-id="DSP014C22" data-open="DSP014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Finally, we used the false discovery rate (FDR) to account for simultaneous testing of multiple hypotheses.<sup><a reveal-id="DSP014C23" data-open="DSP014C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We designed the program to generate at least 100 top-scored motifs and additional motifs, if they satisfy the criterion of FDR &lt; 0.05.</p><p>PFMs are then clustered based on similarity and/or co-occurrence (Fig. <a reveal-id="DSP014F1" data-open="DSP014F1" class="link link-reveal link-table xref-fig">1</a>H). Various methods have been proposed to measure the similarity of PFMs, including Bayesian models.<sup><a reveal-id="DSP014C24" data-open="DSP014C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Here, we use a simpler method and measure similarity by Pearson correlation between elements of the corresponding position weight matrices (PWMs) for all overlapping positions, where PWM is derived from PFM by log-transformation: <em>x</em><sub><em>ij</em></sub> = log(<em>p</em><sub><em>ij</em></sub>/<em>q</em><sub><em>j</em></sub>), <em>x</em><sub><em>ij</em></sub> is the weight of nucleotide <em>j</em> in position <em>i</em>, <em>p</em><sub><em>ij</em></sub> the probability to find nucleotide <em>j</em> in position <em>i</em>, and <em>q</em><sub><em>j</em></sub> the background frequency of nucleotide <em>j</em>. For simplicity, here equal background frequencies (<em>q</em><sub><em>j</em></sub> = 0.25) are assumed and zero probabilities are avoided by adding pseudo-count = 1 to nucleotide counts in the PFM. Offset and orientation of motifs are selected based on the maximum correlation, restricted to the minimum overlap of 6 bp and maximum overhang of 2 bp. Because correlation is estimated for a minimum of six overlapping positions, there are at least 24 points (six positions × four nucleotides) for estimating correlation. Thus, even low correlation is significant (e.g. <em>r</em> = 0.5; d.f. = 22; <em>P</em> &lt; 0.05). Therefore, the default correlation threshold set in CisFinder (<em>r</em> = 0.7) is always significant (however, users can also adjust the correlation threshold to increase or decrease the size of clusters). As a measure of motif similarity, we use a correlation between PWM elements rather than a previously proposed correlation between PFM elements,<sup><a reveal-id="DSP014C25" data-open="DSP014C25" class="link link-ref link-reveal xref-bibr">25</a></sup> because the log-transformation increases the contribution of low values to the correlation and represents the binding strength of TFs better. For example, the difference between probabilities 0.99 and 0.7 corresponds to the 1.41-fold change in binding strength, whereas the same difference between probabilities 0.01 and 0.3 corresponds to the 30-fold change in binding strength. After log-transformation (log<sub>10</sub>), the difference in the second pair of probabilities (1.477) is greater than that in the first pair (0.151).</p><p>We use single-linkage clustering, and then each cluster is checked for homogeneity. If the cluster is not homogeneous, it is separated into subclusters using the second round of clustering. Subclustering is done iteratively starting from a pair of seed motifs, adding sequentially most similar motifs, and re-estimating the combined PFM for the subcluster. Each pair of motifs is characterized by the score =<em>r m</em><sub>1</sub><em>m</em><sub>2</sub>, where <em>r</em> is the correlation between PWMs, and <em>m</em><sub>1</sub> and <em>m</em><sub>2</sub> are the numbers of linked members for motifs 1 and 2, respectively. Then the pair with the highest score is selected as a seed for the subcluster. This procedure is different from the single-linkage clustering because motifs are added to the subcluster based on the similarity to the combined PFM of all motifs that are already included into the subcluster, whereas the single-linkage clustering is based on the similarity between individual (non-combined) motifs. Motifs are added until no motif within the cluster can be added to the subcluster using the given threshold of similarity. If all elements in the cluster appear to be in the same subcluster, then the cluster is considered homogeneous. Otherwise, the elements of the subcluster are removed from the cluster, and the same algorithm is applied to the remaining elements.</p><div class="block-child-p">The advantage of clustering PFMs compared with clustering words (as in RSAT<sup><a reveal-id="DSP014C26" data-open="DSP014C26" class="link link-ref link-reveal xref-bibr">26</a></sup>) is that PFMs contain more information than words alone. Words differ qualitatively (the nucleotide either matches or mismatches), whereas PFMs differ quantitatively (i.e. the probability of each nucleotide correlates between two PFMs). Motifs within the same cluster are then arranged using the hierarchical clustering with cluster flips to place similar motifs near each other.<sup><a reveal-id="DSP014C27" data-open="DSP014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Then, the PFM for the entire cluster is estimated as the weighted average of member PFMs using local information content at each position <em>p</em><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp014eqnU6.gif?Expires=2147483647&Signature=UpqSihpA2nSDNAFPr4JUf5kXCUHwUy-m4JW0KxL9WKjDR27NBplDDdg--NjgrUyfiE-oFPYUgEnrVRhxM8rdc8JVE-c-1RmenmLKp---qObK1pvYlks0DMURpThNDf7Ny~gMNPZrtw~aUFYcQdsXxJ7HwRVk~hJOnlaedpF56QbJoymDb~hdhQLNpxjVSltVjfcNJ~BXzbquQG69R42Z8aVDjkmYu7HoWW0u9d0Rk8Vubvsfo8Pc1rDoc9yYu5wq1kKr4EJaJFc79SgZNQkMk4aJiIMCW69baUaTeNDpMxrl-WYE5gOIcujQDxWIaoW5ZvHAcV~11MGkRr3LjGOgCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp014eqnU6.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqnU6.gif?Expires=2147483647&Signature=MPo6GKZrFd7z5eI3LXpVE6l-7Ff37fKq-5pwnQ~uro24zpfterJy61VKWTySwoSjfCYIXXE3CbYj4EhSBDD99ruS6mkFGh1slvWokMiHWkvLt~gxJYkLETF8Cgk6x3EwcQwvsNOcU5rvvHrmBFzbOIM3uV8ZUkanxpknyJTXdU68GlUwhzzhg2X3~JI-CX1VKOILDRYH1iYwTXkMfXFwUSxOeyJNUKKngNYPu9BTDx19X1bfg6nn1ONRMCHkQAvDFlxU1rDQJ4I-dNdY-kNbLizu6zhcTtHemMcCAUts0Z~nCy~1PEUGu~H2dUjYAiuvT-FmMdvbcIWKuG5x43M5ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqnU6.gif" /></a></div></div></div> where <em>f</em><sub><em>ki</em></sub> is the element of PFM, multiplied by motif abundance as a weight. Finally, a sequence logo<sup><a reveal-id="DSP014C28" data-open="DSP014C28" class="link link-ref link-reveal xref-bibr">28</a></sup> is generated from the PFM.</div><p>As an alternative criterion for clustering, the CisFinder also uses co-occurrence of word instances in the test sequences. This method is generally less accurate than the similarity-based method because of the limited number of word pairs in the sequence set. We, therefore, designed the program to use the correlation-based clustering method as a default. However, the co-occurrence method may help to cluster PFMs with a high level of self-similarity (after shifting a position by 1–4 bp), because their relative positions cannot be uniquely identified based on the correlation. Further details of the algorithm implementation are available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Text S1</a> and online (<a class="link link-uri" href="http://lgsun.grc.nia.nih.gov/CisFinder">http://lgsun.grc.nia.nih.gov/CisFinder</a>).<sup><a reveal-id="DSP014C29" data-open="DSP014C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h3 scrollto-destination=6232269 id="6232269" class="section-title" data-legacy-id=s2c>2.3 Implementation of additional tool to search for motifs that match to PFMs</h3>
<div class="block-child-p">Once PFMs are estimated by the CisFinder or given in the literature, it is often necessary to find DNA sites that match a specific PFM in a given sequence (e.g. in promoters of certain genes). This task is computationally intensive if a matching score is estimated sequentially at each position of the sequence as in MatInspector or MATCH.<sup><a reveal-id="DSP014C30" data-open="DSP014C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSP014C31" data-open="DSP014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We implemented as additional tool in the CisFinder website, a faster method to identify DNA sites, which is based on a lookup table. For each motif represented by a PFM, we selected the most informative stretch of eight nucleotides, which is used as a core. Then, a lookup table is generated that specifies all PFMs from the list whose cores match sufficiently well to each possible 8-mer word. The length of 8 bp is selected for the core because the number of all 8-mers is small enough to keep the lookup table in the computer memory, and 8-mer words are specific enough to be linked with only a few PFMs that match them. A match score is defined as log likelihood that a specific sequence matches a matrix and is equal to the sum of those elements of the PWM (log-transformed PFM) that corresponds to nucleotides at each position of the sequence. The match score for the full matrix, <em>T</em><sub>full</sub> = <em>T</em><sub>8</sub> + <em>T</em><sub>resid</sub>, where <em>T</em><sub>8</sub> is the match score for 8-mer core and <em>T</em><sub>resid</sub> is the match score for the residual of the matrix. The program finds the threshold value <em>R</em><sub>8</sub> for the match score <em>T</em><sub>8</sub>, which ensures that the match score for the full matrix exceeds the given threshold <em>R</em><sub>full</sub> with probability 0.999 if <em>T</em><sub>8</sub> &gt; <em>R</em><sub>8</sub>: <a id="DSP014eqn6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP014eqn6"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp014eqn7.gif?Expires=2147483647&Signature=PNxhcxjNvmfcqzjzssk0niHCK5XE~R-tf1W-PSy6VGWxOC2hgY9HpQXPPauG0NIx9WOOEs~4reN-o8b1qHt9FyJp1ebIiOyx2H7lE2Bmz3sqwaHa68FmwC9F631cOBZAU83ZLIc6N4KFMJGWGdOZ6zycplM36qYQll-N~nJayDW4dWcHE9TGAmpAzHMTXQK7Hsp7KnzcjwQ0j7MmARCX7gS~iyFdshUekid2e5jES7wmPSzztp1QB0fgoS5mqn7oArvDz8rmFdZsnKeCl~qdSyLuCYgh90HJ9GTAlRzfVLDC7fKSqoZeHNPIsO5c5Hf8Vpp2ys2ZWZsig6RLOm-frg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp014eqn7.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp014eqn7.gif?Expires=2147483647&Signature=Beotn70LRRUGyXLqeFjaLdV8gxFHgTQBZzyeswoUcqQ7D2F3rHVP71TCPlwlHUaNz8-A9XbhLK1wn1elgyi~U8dXoMzCSxUVhVoKEZiFfioNAfoL~AmoKEVs~7Mra9CVoQE0G8jF1rv~PLEwuRzi5a~SQECKOpBGVzFtMo7CuNBeHD46puPne2wtrGQK~FNaWOKgRBffWozRF4jAJCW81d9m-cKaFwn9zyYJ5YQ8nXASeRlPn0SSzXljnPHb5Te4cmEsWYzDDy6IFmE36q0OQG6wVSv~Z0f6gwAHELUEePdTV9~Ut1XwrD-mMLAHxuhIvAEETSivX0-1SA4WthTmUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp014eqn7.gif" /></a></div></div><div class="label">e6</div></div> where <em>F</em> is the cumulative probability distribution of <em>T</em><sub>resid</sub> when matching to a random sequence. The value of <em>F</em><sup>−1</sup>(0.999) is estimated by Monte-Carlo simulation. A PFM is included into the lookup table for the 8-mer core word if <em>T</em><sub>8</sub> &gt; <em>R</em><sub>8</sub>. The query sequence is scanned sequentially, and for each position, only those matrices are tested that are in the lookup table for the specific 8-mer word that starts at this position. Although this method may miss up to 0.1% of matching sites, we consider it a reasonable trade-off for the increase in computation speed by several orders. On the basis of Equation (<a href="#DSP014eqn6" class="sectionLink xref-sec">e6</a>), these missed sites always have a poor match to the core motif. Although the match score of missed sites formally exceeds the threshold, the quality of these sites is low from a biological point of view because of the poor match to the core motif.</div>                            <h3 scrollto-destination=6232271 id="6232271" class="section-title" data-legacy-id=s2d>2.4 Data sets used in this study</h3>
<p>CisFinder was tested using published ChIP-seq data on binding of 14 TFs [CTCF, ESRRB, KLF4, MYC, POU5F1 (also known as OCT4 or OCT3/4), SMAD1, SOX2, STAT3, TCFCP2L2, ZFX, P300, NMYC, NANOG, E2F1] in ES cells<sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). We also used a deliberately selected low-quality subset of ChIP-PET data<sup><a reveal-id="DSP014C10" data-open="DSP014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> on binding of POU5F1 to test if CisFinder can process sequences with low enrichment of binding motifs. We used genome locations with 2 (<em>N</em> = 19 803) and 3 ditags (<em>N</em> = 3361) from POU5F1 ChIP-PET that did not include loci with additional NANOG ditags to avoid indirect binding effects (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). All binding regions were mapped to the latest mouse genome (mm9, NCBI/NIH) using the UCSC coordinate conversion tool (<a class="link link-uri" href="http://genome.ucsc.edu/cgi-bin/hgLiftOver">http://genome.ucsc.edu/cgi-bin/hgLiftOver</a>).</p>                            <h2 scrollto-destination=6232273 id="6232273" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6232274 id="6232274" class="section-title" data-legacy-id=s3a>3.1 CisFinder algorithm and its main features</h3>
<p>The proposed CisFinder algorithm, which is implemented as an online software tool,<sup><a reveal-id="DSP014C29" data-open="DSP014C29" class="link link-ref link-reveal xref-bibr">29</a></sup> is described in detail in Section 2. In brief, CisFinder has the following features. </p><ul class="roman-lower"><li><p>CisFinder algorithm is based on detecting over-represented short words (i.e. nucleotide sequences) in a sequence and clustering them. Unlike oligo-analysis (RSAT)<sup><a reveal-id="DSP014C26" data-open="DSP014C26" class="link link-ref link-reveal xref-bibr">26</a></sup>, which is also based on the same concept but clusters exact words, CisFinder clusters PFMs that represent binding motifs more accurately than exact words.</p></li><li><p>CisFinder algorithm analyzes words with gaps and expands PFMs over the gaps and flanking regions.</p></li><li><p>CisFinder uses real control sequences to compare against test sequences. This helps to process repeat regions, because motifs that are specific to repeat sequences are expected to be equally abundant in the test and control sets of sequences [thus, the difference of motif frequencies (e1) is close to zero]. Because mammalian functional TF binding sites are often located in repeat regions,<sup><a reveal-id="DSP014C32" data-open="DSP014C32" class="link link-ref link-reveal xref-bibr">32</a></sup> the ability to search for motifs without removing repeat sequences is useful. An option to use randomized model sequence (a third-order Markov process with probabilities extracted from the test sequences) as control is also provided.</p></li><li><p>CisFinder is designed to carry out exhaustive searches for all over-represented DNA motifs in a single run. It combines motifs only at the clustering step, and users can adjust the correlation threshold used for clustering to make clusters bigger or smaller.</p></li><li><p>CisFinder includes auxiliary functions: comparison of DNA motifs with databases of known binding motifs of TFs,<sup><a reveal-id="DSP014C33" data-open="DSP014C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSP014C34" data-open="DSP014C34" class="link link-ref link-reveal xref-bibr">34</a></sup> search for motifs that match to PFMs, visualization of sequences and TF binding motifs with a CisView browser<sup><a reveal-id="DSP014C34" data-open="DSP014C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and UCSC genome browser,<sup><a reveal-id="DSP014C35" data-open="DSP014C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and extraction of sequence fractions and subsets of sequences.</p></li></ul>                            <h3 scrollto-destination=6232277 id="6232277" class="section-title" data-legacy-id=s3b>3.2 CisFinder algorithm accurately identifies PFMs of TF binding motifs</h3>
<p>To test the performance of the new algorithm, we used ChIP-seq data for 12 TFs associated with the pluripotent state of ES cells.<sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> We extracted 200 bp sequence segments centered at TF binding locations identified with ChIP-seq and compared them with control sequences (i.e. 500 bp sequence segments starting from nucleotide positions 400 bp away from both ends of 200 bp test sequence segments). Clustering of PFMs generates highly consistent TF binding motifs that were independent from the correlation threshold used for clustering (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>A). In contrast, clustering over-represented 8-mer words using the RSAT<sup><a reveal-id="DSP014C26" data-open="DSP014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> resulted in long aberrant motifs because of the chain effect of clustering. All 12 motifs identified with CisFinder matched well with motifs found by Chen <em>et al.</em><sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> with Weeder (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>B), indicating that the quality of results is comparable. Unlike other existing tools, CisFinder has also generated PFMs for additional over-represented motifs at the same time. Utility of such additional motifs will be presented and discussed below (Sections 3.3–3.5).</p>                        <a id="6232279" scrollto-destination="6232279"></a>
<div id="DSP014F2" data-id="DSP014F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01402.jpeg?Expires=2147483647&Signature=0Sn7ByG3CSbs0pS0a4oCHCZVj9A2cgBll0PMv26U1V5J1HOgPUV5805D35duB7KkMUumaVIfugMb1zsY3X4M4QZb5RDw7ZZQ1mQ-CX3HOmClk3OP6vT79x0Z2vGDoJeGtF6xSy8rSAU3NXGzZu6Cmfbv022ltQOPxoX9Azm6HdAh3vAox-yA~dlwKEz5vPxwe9AhAUcpeB9pcL20ScRzrEvlDeoqIXhZfB329P-C2Upe2KdrVBcK1b5vErzYk9tnCldDqJ68Nu32vAdQ0TqlmJ9pkYbVbMyJ5eU0S~3sjkFicQZsDx-p1x40nkzK7fGBLIZaSUu4vdOmvR9ualoWjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Testing CisFinder algorithm. (A) Binding motifs of POU5F1 generated by clustering of PFMs (with CisFinder) and over-represented 8-mer words. Binding motifs of TFs in ES cells identified with CisFinder. (B) Comparison of TF binding motifs generated by Chen et al.9 using Weeder and motifs generated with CisFinder. (C–E) Binding motifs of POU5F1, ESRRB, and CTCF, respectively, identified with CisFinder." path-from-xml="dsp01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232279/dsp01402.jpeg" path-from-xml="dsp01402.jpeg" target="_blank">View large</a><a section="6232279" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp01402.jpeg?Expires=2147483647&Signature=DrW1zOSfpHKvfRd9OyWldUSG6GMnTU-y6N1HNsWrUJAzV0RKZ6GpWzjqCB94gKTZpwBJV842-hGNLRhB76kkIIRcvkoMWFj9fJ2ReLFV~GaG7V7yfnQ04BY6x-K~LGDZGFsU5b93XWdSDTUB1CpyPMVSIosLmG1FYFiMuKSubTC2mJh53saCYO7rXdevljRz~deGdC4OhwUHvV99CF8vgPNYVI7AODunuQOFt2hMXGGQl-HGDQTFoqgeZTP8bGwmXaYXs4kxTt1rUJw2GHSYLeQ2og97dNsXydN4rOITbuUSzO2949G1Rk3D0MUDrIyN2WntWrXEwITuuaWt3ToRIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232279&ar=420676&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Testing CisFinder algorithm. (A) Binding motifs of POU5F1 generated by clustering of PFMs (with CisFinder) and over-represented 8-mer words. Binding motifs of TFs in ES cells identified with CisFinder. (B) Comparison of TF binding motifs generated by Chen <em>et al.</em><sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> using Weeder and motifs generated with CisFinder. (C–E) Binding motifs of POU5F1, ESRRB, and CTCF, respectively, identified with CisFinder.</p></div></div><div content-id="DSP014F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01402.jpeg?Expires=2147483647&Signature=0Sn7ByG3CSbs0pS0a4oCHCZVj9A2cgBll0PMv26U1V5J1HOgPUV5805D35duB7KkMUumaVIfugMb1zsY3X4M4QZb5RDw7ZZQ1mQ-CX3HOmClk3OP6vT79x0Z2vGDoJeGtF6xSy8rSAU3NXGzZu6Cmfbv022ltQOPxoX9Azm6HdAh3vAox-yA~dlwKEz5vPxwe9AhAUcpeB9pcL20ScRzrEvlDeoqIXhZfB329P-C2Upe2KdrVBcK1b5vErzYk9tnCldDqJ68Nu32vAdQ0TqlmJ9pkYbVbMyJ5eU0S~3sjkFicQZsDx-p1x40nkzK7fGBLIZaSUu4vdOmvR9ualoWjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Testing CisFinder algorithm. (A) Binding motifs of POU5F1 generated by clustering of PFMs (with CisFinder) and over-represented 8-mer words. Binding motifs of TFs in ES cells identified with CisFinder. (B) Comparison of TF binding motifs generated by Chen et al.9 using Weeder and motifs generated with CisFinder. (C–E) Binding motifs of POU5F1, ESRRB, and CTCF, respectively, identified with CisFinder." path-from-xml="dsp01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232279/dsp01402.jpeg" path-from-xml="dsp01402.jpeg" target="_blank">View large</a><a section="6232279" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp01402.jpeg?Expires=2147483647&Signature=DrW1zOSfpHKvfRd9OyWldUSG6GMnTU-y6N1HNsWrUJAzV0RKZ6GpWzjqCB94gKTZpwBJV842-hGNLRhB76kkIIRcvkoMWFj9fJ2ReLFV~GaG7V7yfnQ04BY6x-K~LGDZGFsU5b93XWdSDTUB1CpyPMVSIosLmG1FYFiMuKSubTC2mJh53saCYO7rXdevljRz~deGdC4OhwUHvV99CF8vgPNYVI7AODunuQOFt2hMXGGQl-HGDQTFoqgeZTP8bGwmXaYXs4kxTt1rUJw2GHSYLeQ2og97dNsXydN4rOITbuUSzO2949G1Rk3D0MUDrIyN2WntWrXEwITuuaWt3ToRIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232279&ar=420676&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Testing CisFinder algorithm. (A) Binding motifs of POU5F1 generated by clustering of PFMs (with CisFinder) and over-represented 8-mer words. Binding motifs of TFs in ES cells identified with CisFinder. (B) Comparison of TF binding motifs generated by Chen <em>et al.</em><sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> using Weeder and motifs generated with CisFinder. (C–E) Binding motifs of POU5F1, ESRRB, and CTCF, respectively, identified with CisFinder.</p></div></div><p>Computation time for all steps of the CisFinder algorithm ranged from 5 to 120 s (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), with median time 38 s needed to process a 7.5 Mb sequence (sum of test and control sequences). Our estimate is that our software works &gt;1000 times faster than both MEME<sup><a reveal-id="DSP014C15" data-open="DSP014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and Weeder<sup><a reveal-id="DSP014C36" data-open="DSP014C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and &gt;100 times faster than RSAT.<sup><a reveal-id="DSP014C26" data-open="DSP014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The MDscan<sup><a reveal-id="DSP014C20" data-open="DSP014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> works fast; however, it is designed to process a small number of sequences (from 20 to 400, see <a class="link link-uri" href="http://ai.stanford.edu/~xsliu/MDscan">http://ai.stanford.edu/~xsliu/MDscan</a>), and the online version of the software accepts only 200 sequences.</p><p>Taken together, the data indicate that CisFinder works faster than existing tools without sacrificing sensitivity. It is, however, difficult to make a fair comparison between tools for sensitivities and calculation speeds because each tool was designed to process different types of data. CisFinder was developed to process ChIP-seq or ChIP-chip data, which typically include several thousands of sequences (i.e. a few megabase) and cannot be effectively processed by probabilistic methods (e.g. MEME and Weeder). On the other hand, the probabilistic methods works efficiently on relatively small numbers of sequences (e.g. a typical benchmark sequence set is &lt;32 kb<sup><a reveal-id="DSP014C13" data-open="DSP014C13" class="link link-ref link-reveal xref-bibr">13</a></sup>), as they are designed to process a small data set by selecting only high-scoring sequences. However, the reduction in the data set often leads to the loss of useful information, as we describe below (Section 3.3).</p>                            <h3 scrollto-destination=6232282 id="6232282" class="section-title" data-legacy-id=s3c>3.3 Cisfinder algorithm detects alternative binding motifs</h3>
<p>Eukaryotic transcription regulation is extremely complex, and most TFs have multiple binding motifs, which correspond to direct binding of single TFs, tandems of identical TFs in various orientations and spacings, binding with various co-factors, and finally, indirect binding via protein–protein interactions with other TFs. Analysis from 50 to 200 high-score binding sites (which is a typical data size for MEME or Weeder) is usually sufficient to extract the main motif, but it is often not sufficient to examine alternative motifs. For example, Chen <em>et al.</em><sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> used Weeder<sup><a reveal-id="DSP014C36" data-open="DSP014C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and reported only a single motif for each TF. In contrast, using the same data set, CisFinder was able to find multiple motifs for each TF, e.g. POU5F1 (also known as OCT4 or OCT3/4), ESRRB, and CTCF (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>C–E).</p><p>For the POU5F1, predicted alternative binding motifs included several palindromes (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>C, b–f). Previous studies have already shown that these motifs are also functional: (b) is a ‘MORE’ motif,<sup><a reveal-id="DSP014C37" data-open="DSP014C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (e) is a ‘PORE’ motif,<sup><a reveal-id="DSP014C38" data-open="DSP014C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and (c) is a part of two motifs identified by Tantin <em>et al.</em><sup><a reveal-id="DSP014C39" data-open="DSP014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> On the other hand, CisFinder could not detect a well-known OCT–3N–SOX composite motif with a 3 bp spacer between OCT and SOX motifs, which is located in the enhancer of <em>Fgf4</em>.<sup><a reveal-id="DSP014C40" data-open="DSP014C40" class="link link-ref link-reveal xref-bibr">40</a></sup> To investigate this issue, we searched for this motif in ChIP-selected sequences using a PFM derived from the regular OCT–SOX composite motif after adding a 3 bp spacer. Because OCT–SOX and OCT–3N–SOX motifs are similar, we counted sites only if they matched more strongly to OCT–3N–SOX than to OCT–SOX motif. We found that the OCT–3N–SOX motif was indeed present in ChIP–POU5F1 sites (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), but its abundance was too low (20-fold less abundant than OCT–SOX motif) to be detected <em>de novo</em> with a statistical confidence.</p><p>For the estrogen-related receptor beta (ESRRB), CisFinder predicted 12 alternative binding motifs (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>D), all of which represented different repeat configurations of the same elementary motif AGGTCA. In direct repeats (i.e. repeats in the same orientation) (a–g), monomers were spaced by either 0, 1, 2, 3, 4, or 5 bp. In inverted repeats, the spacing between monomers was less flexible. When the first monomer had a positive orientation (h–j), then inverted repeats were spaced by either −3, 0, or 3 bp (−3 means 3 bp overlap). However, when the first monomer had negative orientation (k and l), then motifs were spaced by either 2 or 6 bp. Akter <em>et al.</em><sup><a reveal-id="DSP014C41" data-open="DSP014C41" class="link link-ref link-reveal xref-bibr">41</a></sup> tested 12 paired motifs (direct and inverted) with the competitive EMSA and found the increased binding of estrogen-related receptors to direct repeats with 0, 2, and 4 bp spacing and to inverted repeats with 0 and 3 bp spacing. Thus, the <em>in vivo</em> ChIP-seq data confirmed <em>in vitro</em> EMSA data by Akter <em>et al.</em>, although the motifs (h), (k), and (l) found in the ChIP-seq data (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>D) were not tested by Akter <em>et al.</em> Our results indicate that ESRRB can bind <em>in vivo</em> to direct repeats spaced by 1, 3, and 5 bp despite weak competitiveness in EMSA, presenting the largest set of alternative binding motifs detected for the ESRRB.</p><p>For the CTCF (an insulator in the regulation of transcription<sup><a reveal-id="DSP014C42" data-open="DSP014C42" class="link link-ref link-reveal xref-bibr">42</a></sup>), several alternative binding motifs were detected. The main DNA motif enriched in ChIP–CTCF loci (Fig. <a reveal-id="DSP014F2" data-open="DSP014F2" class="link link-reveal link-table xref-fig">2</a>E, a) matched well to the motifs identified in earlier studies<sup><a reveal-id="DSP014C43  DSP014C45" data-open="DSP014C43  DSP014C45" class="link link-ref link-reveal xref-bibr">43–45</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Furthermore, CisFinder identified several alternative binding motifs for CTCF (b–e), including three palindromes (b–d). However, further experimental validation is needed to prove that these motifs are indeed functional.</p>                            <h3 scrollto-destination=6232296 id="6232296" class="section-title" data-legacy-id=s3d>3.4 CisFinder algorithm detects binding motifs of potential co-factors</h3>
<p>Genome locations identified with ChIP for a specific TF often do not carry the primary or alternative binding motifs, but are enriched with binding motifs for other TFs (cofactors). The most likely interpretation of this phenomenon is that the TF used for the immunoprecipitation binds to DNAs indirectly through binding to a co-factor that directly binds to DNA. Thus, the analysis of co-factor binding motifs may help to infer potential mechanisms of transcription regulation.</p><p>To explore this issue, we first selected 22 motifs that were over-represented in ChIP loci for single or multiple TFs reported by Chen <em>et al.</em><sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and used the corresponding PFMs generated by CisFinder to search for these motifs in 200 bp DNA segments centered at ChIP loci (Fig. <a reveal-id="DSP014F3" data-open="DSP014F3" class="link link-reveal link-table xref-fig">3</a>). Some of these motifs were well characterized and supported the bindings of known TFs (e.g. ESRRB, GABP, ATF1, and TEF). A motif MIT-008 was shown to be over-represented in mammalian promoters,<sup><a reveal-id="DSP014C46" data-open="DSP014C46" class="link link-ref link-reveal xref-bibr">46</a></sup> although a TF binding to these sites remains unknown. We also found a novel motif (AP4-L) which is similar to the V$AP4_01 binding motif in TRANSFAC.<sup><a reveal-id="DSP014C47" data-open="DSP014C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The YY1 motif may correspond to ZFP42 (=REX1) binding because both TFs have nearly identical motifs.<sup><a reveal-id="DSP014C48" data-open="DSP014C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSP014C49" data-open="DSP014C49" class="link link-ref link-reveal xref-bibr">49</a></sup></p>                        <a id="6232299" scrollto-destination="6232299"></a>
<div id="DSP014F3" data-id="DSP014F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01403.jpeg?Expires=2147483647&Signature=L9MvN-M1taeWSTJt2FhOEZsipLOguMarx0CVG4ZSOdrMZsi1O~edHalSfgMb01irXGedrcxiS5YZL6YfOBiPHWXtxQRj23HRdDFBQR-I1ytXkE8hgFoeerSXMXem3xSOgMVsp-OngOwaIjJSRkBNQYoSve~oZBiin-DWEbOwBxYSUZqjvaItAyj45Wu8~c5w~20nsa8cKjSTxcCTJiUNkbLHnAO4VYNF-L1bmI2nCO1IQhnjj~ktTFxE7-HW3h2NI64jw9t2Nt9hnTNjuCYOLgtL0UcEQ9PI9PuE~SfpwGyUkyOEPLTm4taepepaTG49FgqM1rQGu47qzD3FXf5Qyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Motifs of TFs and their co-factors over-represented in ChIP-seq (data from Chen et al.9) distal binding sites (200 bp segments centered at binding sites and located 500–100 000 bp away from transcription start sites) compared with flanking regions 500–1000 bp away from binding sites. Motifs were selected if they were over-represented by &gt;2-fold for at least one TF; search was done with CisFinder using the option of one false positive match per 10 kb. Groups of TFs and binding motifs with high over-representation rate are outlined." path-from-xml="dsp01403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232299/dsp01403.jpeg" path-from-xml="dsp01403.jpeg" target="_blank">View large</a><a section="6232299" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp01403.jpeg?Expires=2147483647&Signature=oh~BSkbOhSVIbGELG9LuGL65dnPlFQ6UkO9E8anBLI0UfMjASa9nTI-VgTM8ygCtf3J2vb2aFXS0xOufbQjOK20K95t1uqyUf7d9zQ-uvNN~S5K~KGBR2V2nHaUwHJmSYD1mNgWVDNvQNXsWM7i77o0VfYSqSsbkslaZ5edfGo5~T0eTWAAQG0yKSuLTml7OLLSPdPq0xCRcYF4302F-AGM3PbunM~7J2vZTdhaun2kgCa8TxomyX9cxd-95iesGH2YNiIg5vhBTEV4p8Bub0xz5FrQ0lXb7cnoJM0578EKHuUYR1bzm0dxOxr~v2dKi6x-BFqBeLEknZV8zITrt8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232299&ar=420676&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Motifs of TFs and their co-factors over-represented in ChIP-seq (data from Chen <em>et al</em>.<sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup>) distal binding sites (200 bp segments centered at binding sites and located 500–100 000 bp away from transcription start sites) compared with flanking regions 500–1000 bp away from binding sites. Motifs were selected if they were over-represented by &gt;2-fold for at least one TF; search was done with CisFinder using the option of one false positive match per 10 kb. Groups of TFs and binding motifs with high over-representation rate are outlined.</p></div></div><div content-id="DSP014F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/m_dsp01403.jpeg?Expires=2147483647&Signature=L9MvN-M1taeWSTJt2FhOEZsipLOguMarx0CVG4ZSOdrMZsi1O~edHalSfgMb01irXGedrcxiS5YZL6YfOBiPHWXtxQRj23HRdDFBQR-I1ytXkE8hgFoeerSXMXem3xSOgMVsp-OngOwaIjJSRkBNQYoSve~oZBiin-DWEbOwBxYSUZqjvaItAyj45Wu8~c5w~20nsa8cKjSTxcCTJiUNkbLHnAO4VYNF-L1bmI2nCO1IQhnjj~ktTFxE7-HW3h2NI64jw9t2Nt9hnTNjuCYOLgtL0UcEQ9PI9PuE~SfpwGyUkyOEPLTm4taepepaTG49FgqM1rQGu47qzD3FXf5Qyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Motifs of TFs and their co-factors over-represented in ChIP-seq (data from Chen et al.9) distal binding sites (200 bp segments centered at binding sites and located 500–100 000 bp away from transcription start sites) compared with flanking regions 500–1000 bp away from binding sites. Motifs were selected if they were over-represented by &gt;2-fold for at least one TF; search was done with CisFinder using the option of one false positive match per 10 kb. Groups of TFs and binding motifs with high over-representation rate are outlined." path-from-xml="dsp01403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232299/dsp01403.jpeg" path-from-xml="dsp01403.jpeg" target="_blank">View large</a><a section="6232299" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp01403.jpeg?Expires=2147483647&Signature=oh~BSkbOhSVIbGELG9LuGL65dnPlFQ6UkO9E8anBLI0UfMjASa9nTI-VgTM8ygCtf3J2vb2aFXS0xOufbQjOK20K95t1uqyUf7d9zQ-uvNN~S5K~KGBR2V2nHaUwHJmSYD1mNgWVDNvQNXsWM7i77o0VfYSqSsbkslaZ5edfGo5~T0eTWAAQG0yKSuLTml7OLLSPdPq0xCRcYF4302F-AGM3PbunM~7J2vZTdhaun2kgCa8TxomyX9cxd-95iesGH2YNiIg5vhBTEV4p8Bub0xz5FrQ0lXb7cnoJM0578EKHuUYR1bzm0dxOxr~v2dKi6x-BFqBeLEknZV8zITrt8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232299&ar=420676&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Motifs of TFs and their co-factors over-represented in ChIP-seq (data from Chen <em>et al</em>.<sup><a reveal-id="DSP014C9" data-open="DSP014C9" class="link link-ref link-reveal xref-bibr">9</a></sup>) distal binding sites (200 bp segments centered at binding sites and located 500–100 000 bp away from transcription start sites) compared with flanking regions 500–1000 bp away from binding sites. Motifs were selected if they were over-represented by &gt;2-fold for at least one TF; search was done with CisFinder using the option of one false positive match per 10 kb. Groups of TFs and binding motifs with high over-representation rate are outlined.</p></div></div><p>Next, we compared the abundance of these motifs in the 200 bp DNA segments centered at ChIP loci with the control sequences (i.e. 500 bp sequence segments starting from nucleotide positions 400 bp away from both ends of 200 bp test sequence segments). To obtain a homogeneous data set, we used only the ChIP loci that were located at &gt;500 bp away from the transcription start sites of genes (distal ChIP loci). Another reason to focus on the distal ChIP loci was that pluripotency-related TFs, such as POU5F1 and NANOG, are active mostly at distal locations rather than at proximal promoters.<sup><a reveal-id="DSP014C10" data-open="DSP014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> We then tabulated the motif abundance data and found that TFs and corresponding binding motifs formed three distinctive groups (Fig. <a reveal-id="DSP014F3" data-open="DSP014F3" class="link link-reveal link-table xref-fig">3</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The first group (group #1) included the major pluripotency-related TFs (POU5F1, SOX2, and NANOG) as well as SMAD1 and P300. As expected, the strongest binding motif in this group was the OCT–SOX composite motif. OCT motif alone was associated mostly with POU5F1 binding, whereas SOX2 motif alone, which was known previously as SOX9 (V$SOX9_B1) in TRANSFAC,<sup><a reveal-id="DSP014C47" data-open="DSP014C47" class="link link-ref link-reveal xref-bibr">47</a></sup> was associated mostly with binding of SOX2, NANOG, and SMAD1. A novel motif AP4-L was associated with binding of all TFs in the group #1, but the association was strongest for SMAD1 and NANOG. A TEF motif was most abundant in P300 binding locations. The second group (group #2) included STAT3, KLF4, ESRRB, and TCFCP2L1. The third group (group #3) included MYC, NMYC, ZFX, and E2F1 (Fig. <a reveal-id="DSP014F3" data-open="DSP014F3" class="link link-reveal link-table xref-fig">3</a>). Although it is tempting to speculate that these TFs in each group form a protein complex, drawing such a conclusion requires further evidence for the presence of such protein complexes in the ES cells.</p><p>We also noticed that some DNA motifs were negatively associated with binding of some TFs, which may indicate the inhibition of DNA binding. For example, the major OCT4 palindrome motifs, OCT4-GCGC and OCT4-MORE, were strongly under-represented in many ChIP loci including binding sites of pluripotency-related TFs (NANOG, SOX2, STAT3, KLF4) (green color), except for POU5F1 that bound to these motifs (Fig. <a reveal-id="DSP014F3" data-open="DSP014F3" class="link link-reveal link-table xref-fig">3</a>). This suggests that palindrome POU5F1 motifs are likely to be involved in a different cellular function than supporting ES cell pluripotency. The OCT–SOX and SOX2 motifs alone were negatively associated with the binding of SUZ12, which may explain why Polycomb protein complexes cannot inactivate pluripotency-related genes in ES cells.</p>                            <h3 scrollto-destination=6232302 id="6232302" class="section-title" data-legacy-id=s3e>3.5 CisFinder algorithm can find motifs with a low level of enrichment</h3>
<p>We tested whether the CisFinder algorithm was robust enough to identify motifs that were only slightly enriched in the set of DNA sequences. According to Loh <em>et al.</em>,<sup><a reveal-id="DSP014C10" data-open="DSP014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> ChIP loci with at least 4 ditags (ChIP-PET data) were reliable enough to infer binding of POU5F1 and NANOG. Thus, we used ChIP loci with 2 or 3 ditags for POU5F1 as examples of data with a low level of motif enrichment. To evaluate the over-representation of binding motifs, we searched for the OCT–SOX motif in 200 bp test DNA segments centered at ChIP loci and in control sequences (i.e. two 500 bp sequence segments starting from nucleotide positions 400 bp away from both ends of 200 bp test sequence segments). To avoid a circular reference, we took the PFM for the OCT–SOX motif from an independent source, where the PFM was estimated on the basis of ChIP-PET loci with at least 4 ditags for POU5F1.<sup><a reveal-id="DSP014C10" data-open="DSP014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The over-representation ratios of the OCT–SOX motif density were only 1.57 and 0.99 in ChIP-PET data sets with 3 and 2 ditags, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). They were substantially lower than the over-representation ratio (7.10) of the OCT–SOX motif in the ChIP-seq data, which confirms the low level of motif enrichment. The CisFinder algorithm was successful in finding the OCT–SOX composite motif ATTGTTATGCAAAT as the top-scored consensus sequence for the set of 3361 ChIP loci with 3 ditags. Similarly, in the set of 19 803 genome loci with 2 ditags for POU5F1, CisFinder identified a canonical POU-motif ATGCAAAT.<sup><a reveal-id="DSP014C50" data-open="DSP014C50" class="link link-ref link-reveal xref-bibr">50</a></sup> However, this motif was not the top-scored one (rank = 11), which may be the result of a large proportion of false positives in the data set. The OCT–SOX composite motif was not found, which can be explained by no enrichment of this motif (over-representation ratio = 0.99) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Thus, we hypothesized that the weak binding of POU5F1 does not require SOX2 as a co-factor. Top-scored motifs over-represented in ChIP loci with 2 ditags were also meaningful: they corresponded to NRF1 and KLF motifs, which were associated with POU5F1 binding as shown above (Fig. <a reveal-id="DSP014F3" data-open="DSP014F3" class="link link-reveal link-table xref-fig">3</a>) and reported in the literature.<sup><a reveal-id="DSP014C51" data-open="DSP014C51" class="link link-ref link-reveal xref-bibr">51</a></sup> In comparison, neither MEME<sup><a reveal-id="DSP014C15" data-open="DSP014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> nor Weeder<sup><a reveal-id="DSP014C36" data-open="DSP014C36" class="link link-ref link-reveal xref-bibr">36</a></sup> found any meaningful motifs in both data sets with 3 or 2 ditags of POU5F1.</p>                            <h3 scrollto-destination=6232304 id="6232304" class="section-title" data-legacy-id=s3f>3.6 Other potential applications and limitations of CisFinder</h3>
<p>Although CisFinder was designed specifically for the analysis of ChIP experiments on TF binding, it can be used for other purposes. For example, it can be used to find over-represented motifs in promoters of co-regulated genes, in introns of alternatively spliced genes, or in 3′-untranslated regions of genes with high or low rates of mRNA degradation. The search for over-represented motifs can be improved by limiting the search to evolutionarily conserved regulatory regions because functional sequences have a tendency to be conserved during evolution.<sup><a reveal-id="DSP014C52" data-open="DSP014C52" class="link link-ref link-reveal xref-bibr">52</a></sup> [However, recent findings indicate that many regulatory regions are located in transposable elements, which are usually not conserved.<sup><a reveal-id="DSP014C32" data-open="DSP014C32" class="link link-ref link-reveal xref-bibr">32</a></sup>]</p><p>Because of its high processing speed, CisFinder can be used interactively by adjusting parameters of motif detection. Also, it can be utilized effectively as a component of systems for reconstructing gene regulatory networks. For example, Reiss <em>et al.</em><sup><a reveal-id="DSP014C53" data-open="DSP014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> used <em>de novo</em> motif discovery in promoters of co-regulated genes, which were clustered using the data on gene expression in various conditions. Because the identification of motifs is repeated many times in this analysis, the use of CisFinder algorithm can increase the processing speed.</p><p>The main limitation of CisFinder algorithm is that its performance decreases if the input sequence is too short. For example, if the length of sequence is 32 kb, then it contains only one 8-mer word on average (based on the random model). In this case, the CisFinder can detect only highly over-represented motifs (e.g. with &gt;10-fold enrichment) and, thus, other software (e.g. MEME<sup><a reveal-id="DSP014C15" data-open="DSP014C15" class="link link-ref link-reveal xref-bibr">15</a></sup>) should be used instead.</p>                            <h3 scrollto-destination=6232308 id="6232308" class="section-title" data-legacy-id=s3g>3.7 Conclusion</h3>
<p>CisFinder implements an express method for <em>de novo</em> identification of over-represented DNA motifs and is specifically designed to process ChIP-chip and ChIP-seq data. It is a complementary method to existing motif-finding tools, which are highly efficient in processing short input sequences. Unique features of CisFinder are: (i) it extracts all over-represented motifs in a single run and describes them with PFMs; (ii) it can effectively process large sequences (up to 50 Mb); (iii) because of its high processing speed, it can be used in an interactive manner by running the analyses multiple times after re-adjusting parameters; and (iv) it can process data with a low-level enrichment of DNA motifs.</p>                            <h2 scrollto-destination=6232310 id="6232310" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6232312 id="6232312" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This research was supported entirely by the Intramural Research Program of the NIH, National Institute on Aging.</p>                            <h2 scrollto-destination=6232314 id="6232314" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dawood Dudekula for help with configuration of web server and program testing. We thank Dr Huck-Hui Ng at the Genome Institute of Singapore for kindly providing raw data of their published ChIP-PET study.</p>                            <h2 scrollto-destination=6232316 id="6232316" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP014C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoltenburg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Reinemann</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Strehlitz</div> <div class="given-names">B.</div></div>. </div><div class="article-title">SELEX–a (r)evolutionary method to generate high-affinity nucleic acid ligands</div>, <div class="source">Biomol. Eng.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">381</div>-<div class="lpage">403</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SELEX%E2%80%93a%20(r)evolutionary%20method%20to%20generate%20high-affinity%20nucleic%20acid%20ligands&amp;author=R.%20Stoltenburg&amp;author=C.%20Reinemann&amp;author=B.%20Strehlitz&amp;publication_year=2007&amp;journal=Biomol.%20Eng.&amp;volume=24&amp;pages=381-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bioeng.2007.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bioeng.2007.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bioeng.2007.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17627883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Badis</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Berger</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Philippakis</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Diversity and complexity in DNA recognition by transcription factors</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">324</div> (pg. <div class="fpage">1720</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20and%20complexity%20in%20DNA%20recognition%20by%20transcription%20factors&amp;author=G.%20Badis&amp;author=M.F.%20Berger&amp;author=A.A.%20Philippakis&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=1720-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1162327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1162327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1162327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19443739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">High-resolution profiling of histone methylations in the human genome</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">823</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20profiling%20of%20histone%20methylations%20in%20the%20human%20genome&amp;author=A.%20Barski&amp;author=S.%20Cuddapah&amp;author=K.%20Cui&amp;publication_year=2007&amp;journal=Cell&amp;volume=129&amp;pages=823-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.05.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.05.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.05.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17512414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genome-wide mapping of in vivo protein–DNA interactions</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">316</div> (pg. <div class="fpage">1497</div>-<div class="lpage">502</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20in%20vivo%20protein%E2%80%93DNA%20interactions&amp;author=D.S.%20Johnson&amp;author=A.%20Mortazavi&amp;author=R.M.%20Myers&amp;author=B.%20Wold&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1497-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1141319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1141319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1141319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robertson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hirst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bainbridge</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">651</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20profiles%20of%20STAT1%20DNA%20association%20using%20chromatin%20immunoprecipitation%20and%20massively%20parallel%20sequencing&amp;author=G.%20Robertson&amp;author=M.%20Hirst&amp;author=M.%20Bainbridge&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=651-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17558387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lieb</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Genome-wide mapping of protein–DNA interactions by chromatin immunoprecipitation and DNA microarray hybridization</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">224</div> (pg. <div class="fpage">99</div>-<div class="lpage">109</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20protein%E2%80%93DNA%20interactions%20by%20chromatin%20immunoprecipitation%20and%20DNA%20microarray%20hybridization&amp;author=J.D.%20Lieb&amp;publication_year=2003&amp;journal=Methods%20Mol.%20Biol.&amp;volume=224&amp;pages=99-109" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12710669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chia</div> <div class="given-names">N.Y.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Zhong</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Cross-species de novo identification of cis-regulatory modules with GibbsModule: application to gene regulation in embryonic stem cells</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1325</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-species%20de%20novo%20identification%20of%20cis-regulatory%20modules%20with%20GibbsModule%3A%20application%20to%20gene%20regulation%20in%20embryonic%20stem%20cells&amp;author=D.%20Xie&amp;author=J.%20Cai&amp;author=N.Y.%20Chia&amp;author=H.H.%20Ng&amp;author=S.%20Zhong&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1325-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.072769.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.072769.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.072769.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18490265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Badis</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gehrke</div> <div class="given-names">A.R.</div></div>, et al. </div><div class="article-title">Variation in homeodomain DNA binding revealed by high-resolution analysis of sequence preferences</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">1266</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20homeodomain%20DNA%20binding%20revealed%20by%20high-resolution%20analysis%20of%20sequence%20preferences&amp;author=M.F.%20Berger&amp;author=G.%20Badis&amp;author=A.R.%20Gehrke&amp;publication_year=2008&amp;journal=Cell&amp;volume=133&amp;pages=1266-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.05.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.05.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.05.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18585359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Integration of external signaling pathways with the core transcriptional network in embryonic stem cells</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">1106</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20external%20signaling%20pathways%20with%20the%20core%20transcriptional%20network%20in%20embryonic%20stem%20cells&amp;author=X.%20Chen&amp;author=H.%20Xu&amp;author=P.%20Yuan&amp;publication_year=2008&amp;journal=Cell&amp;volume=133&amp;pages=1106-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.04.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.04.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.04.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18555785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loh</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chew</div> <div class="given-names">J.L.</div></div>, et al. </div><div class="article-title">The Oct4 and Nanog transcription network regulates pluripotency in mouse embryonic stem cells</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">431</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Oct4%20and%20Nanog%20transcription%20network%20regulates%20pluripotency%20in%20mouse%20embryonic%20stem%20cells&amp;author=Y.H.%20Loh&amp;author=Q.%20Wu&amp;author=J.L.%20Chew&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=431-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1760" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1760" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1760"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16518401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bock</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lengauer</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Computational epigenetics</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20epigenetics&amp;author=C.%20Bock&amp;author=T.%20Lengauer&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm546" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm546" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm546"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18024971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Dai</div> <div class="given-names">H.K.</div></div>. </div><div class="article-title">A survey of DNA motif finding algorithms</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> <div class="supplement">Suppl 7</div>pg. <div class="fpage">S21</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20DNA%20motif%20finding%20algorithms&amp;author=M.K.%20Das&amp;author=H.K.%20Dai&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=S21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-S7-S21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-S7-S21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-S7-S21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18047721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandve</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Abul</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Walseng</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Drablos</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Improved benchmarks for computational motif discovery</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">193</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20benchmarks%20for%20computational%20motif%20discovery&amp;author=G.K.%20Sandve&amp;author=O.%20Abul&amp;author=V.%20Walseng&amp;author=F.%20Drablos&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=193" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-193" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-193" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-193"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17559676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tompa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>, et al. </div><div class="article-title">Assessing computational tools for the discovery of transcription factor binding sites</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">137</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20computational%20tools%20for%20the%20discovery%20of%20transcription%20factor%20binding%20sites&amp;author=M.%20Tompa&amp;author=N.%20Li&amp;author=T.L.%20Bailey&amp;publication_year=2005&amp;journal=Nat.%20Biotechnol.&amp;volume=23&amp;pages=137-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15637633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Misleh</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.W.</div></div>. </div><div class="article-title">MEME: discovering and analyzing DNA and protein sequence motifs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W369</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEME%3A%20discovering%20and%20analyzing%20DNA%20and%20protein%20sequence%20motifs&amp;author=T.L.%20Bailey&amp;author=N.%20Williams&amp;author=C.%20Misleh&amp;author=W.W.%20Li&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W369-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Rouchka</div> <div class="given-names">E.C.</div></div>,  <div class="name"><div class="surname">Lawrence</div> <div class="given-names">C.E.</div></div>. </div><div class="article-title">Gibbs Recursive Sampler: finding transcription factor binding sites</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3580</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gibbs%20Recursive%20Sampler%3A%20finding%20transcription%20factor%20binding%20sites&amp;author=W.%20Thompson&amp;author=E.C.%20Rouchka&amp;author=C.E.%20Lawrence&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3580-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg608" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg608" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg608"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">S.T.</div></div>. </div><div class="article-title">GAME: detecting cis-regulatory elements using a genetic algorithm</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1577</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GAME%3A%20detecting%20cis-regulatory%20elements%20using%20a%20genetic%20algorithm&amp;author=Z.%20Wei&amp;author=S.T.%20Jensen&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1577-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Wakefield</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Self</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A transdimensional Bayesian model for pattern recognition in DNA sequences</div>, <div class="source">Biostatistics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">668</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20transdimensional%20Bayesian%20model%20for%20pattern%20recognition%20in%20DNA%20sequences&amp;author=S.M.%20Li&amp;author=J.%20Wakefield&amp;author=S.%20Self&amp;publication_year=2008&amp;journal=Biostatistics&amp;volume=9&amp;pages=668-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/kxm058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2Fkxm058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2Fkxm058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Extracting sequence features to predict protein–DNA interactions: a comparative study</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">4137</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extracting%20sequence%20features%20to%20predict%20protein%E2%80%93DNA%20interactions%3A%20a%20comparative%20study&amp;author=Q.%20Zhou&amp;author=J.S.%20Liu&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=4137-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18556756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">X.S.</div></div>,  <div class="name"><div class="surname">Brutlag</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">An algorithm for finding protein–DNA binding sites with applications to chromatin-immunoprecipitation microarray experiments</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2002</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">835</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20algorithm%20for%20finding%20protein%E2%80%93DNA%20binding%20sites%20with%20applications%20to%20chromatin-immunoprecipitation%20microarray%20experiments&amp;author=X.S.%20Liu&amp;author=D.L.%20Brutlag&amp;author=J.S.%20Liu&amp;publication_year=2002&amp;journal=Nat.%20Biotechnol.&amp;volume=20&amp;pages=835-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt717" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt717" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt717"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12101404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sokal</div> <div class="given-names">R.R.</div></div>,  <div class="name"><div class="surname">Rohlf</div> <div class="given-names">F.J.</div></div>. </div>, <div class="source">Biometry. The Principles and Practice of Statistics in Biological Research</div> , <div class="year">2001</div><div class="publisher-loc">New York</div><div class="publisher-name">Freeman</div></div></div></div></div><div content-id="DSP014C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hess</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Iyer</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Fisher's combined <em>P</em>-value for detecting differentially expressed genes using Affymetrix expression arrays</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">96</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fisher's%20combined%20P-value%20for%20detecting%20differentially%20expressed%20genes%20using%20Affymetrix%20expression%20arrays&amp;author=A.%20Hess&amp;author=H.%20Iyer&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-96" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-96" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-96"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17419876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Controlling the false discovery rate—a practical and powerful approach to multiple testing</div>, <div class="source">J. R. Stat. Soc. Ser. B</div> , <div class="year">1995</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="DSP014C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Habib</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kaplan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Margalit</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Friedman</div> <div class="given-names">N.</div></div>. </div><div class="article-title">A novel Bayesian DNA motif comparison method for clustering and retrieval</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000010</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20Bayesian%20DNA%20motif%20comparison%20method%20for%20clustering%20and%20retrieval&amp;author=N.%20Habib&amp;author=T.%20Kaplan&amp;author=H.%20Margalit&amp;author=N.%20Friedman&amp;publication_year=2008&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=4&amp;pages=e1000010" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Stamatoyannopoulos</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Noble</div> <div class="given-names">W.S.</div></div>. </div><div class="article-title">Quantifying similarity between motifs</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R24</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20similarity%20between%20motifs&amp;author=S.%20Gupta&amp;author=J.A.%20Stamatoyannopoulos&amp;author=T.L.%20Bailey&amp;author=W.S.%20Noble&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-2-r24" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-2-r24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-2-r24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17324271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Helden</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Andre</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Extracting regulatory sites from the upstream region of yeast genes by computational analysis of oligonucleotide frequencies</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">827</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extracting%20regulatory%20sites%20from%20the%20upstream%20region%20of%20yeast%20genes%20by%20computational%20analysis%20of%20oligonucleotide%20frequencies&amp;author=J.%20van%20Helden&amp;author=B.%20Andre&amp;author=J.%20Collado-Vides&amp;publication_year=1998&amp;journal=J.%20Mol.%20Biol.&amp;volume=281&amp;pages=827-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1998.1947" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1998.1947" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1998.1947"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9719638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Spellman</div> <div class="given-names">P.T.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.O.</div></div>,  <div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Cluster analysis and display of genome-wide expression patterns</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">14863</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns&amp;author=M.B.%20Eisen&amp;author=P.T.%20Spellman&amp;author=P.O.%20Brown&amp;author=D.%20Botstein&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=14863-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.25.14863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.25.14863"> </span></div> </div></div></div></div></div><div content-id="DSP014C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schneider</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Sequence logos: a new way to display consensus sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6097</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20logos%3A%20a%20new%20way%20to%20display%20consensus%20sequences&amp;author=T.D.%20Schneider&amp;author=R.M.%20Stephens&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6097-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.20.6097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.20.6097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2172928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharov</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Ko</div> <div class="given-names">M.S.H.</div></div>. </div><div class="article-title">CisFinder</div>, <div class="year">2008</div> <div class="comment"><a class="link link-uri" href="http://lgsun.grc.nia.nih.gov/CisFinder">http://lgsun.grc.nia.nih.gov/CisFinder</a>.</div></div></div></div></div><div content-id="DSP014C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kel</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Gossling</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Reuter</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Cheremushkin</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kel-Margoulis</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Wingender</div> <div class="given-names">E.</div></div>. </div><div class="article-title">MATCH: a tool for searching transcription factor binding sites in DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3576</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MATCH%3A%20a%20tool%20for%20searching%20transcription%20factor%20binding%20sites%20in%20DNA%20sequences&amp;author=A.E.%20Kel&amp;author=E.%20Gossling&amp;author=I.%20Reuter&amp;author=E.%20Cheremushkin&amp;author=O.V.%20Kel-Margoulis&amp;author=E.%20Wingender&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3576-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg585" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg585" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg585"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quandt</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Frech</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Karas</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wingender</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Werner</div> <div class="given-names">T.</div></div>. </div><div class="article-title">MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">4878</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MatInd%20and%20MatInspector%3A%20new%20fast%20and%20versatile%20tools%20for%20detection%20of%20consensus%20matches%20in%20nucleotide%20sequence%20data&amp;author=K.%20Quandt&amp;author=K.%20Frech&amp;author=H.%20Karas&amp;author=E.%20Wingender&amp;author=T.%20Werner&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=4878-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.23.4878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.23.4878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.23.4878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8532532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bourque</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Leong</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Vega</div> <div class="given-names">V.B.</div></div>, et al. </div><div class="article-title">Evolution of the mammalian transcription factor binding repertoire via transposable elements</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1752</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20mammalian%20transcription%20factor%20binding%20repertoire%20via%20transposable%20elements&amp;author=G.%20Bourque&amp;author=B.%20Leong&amp;author=V.B.%20Vega&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1752-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.080663.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.080663.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.080663.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18682548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bryne</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Valen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">M.H.</div></div>, et al. </div><div class="article-title">JASPAR, the open access database of transcription factor-binding profiles: new content and tools in the 2008 update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D102</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=JASPAR%2C%20the%20open%20access%20database%20of%20transcription%20factor-binding%20profiles%3A%20new%20content%20and%20tools%20in%20the%202008%20update&amp;author=J.C.%20Bryne&amp;author=E.%20Valen&amp;author=M.H.%20Tang&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D102-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharov</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Dudekula</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Ko</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">CisView: a browser and database of cis-regulatory modules predicted in the mouse genome</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">123</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CisView%3A%20a%20browser%20and%20database%20of%20cis-regulatory%20modules%20predicted%20in%20the%20mouse%20genome&amp;author=A.A.%20Sharov&amp;author=D.B.%20Dudekula&amp;author=M.S.%20Ko&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=123-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16980320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karolchik</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">The UCSC genome browser database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">51</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20UCSC%20genome%20browser%20database&amp;author=D.%20Karolchik&amp;author=R.%20Baertsch&amp;author=M.%20Diekhans&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=51-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pavesi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zambelli</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Pesole</div> <div class="given-names">G.</div></div>. </div><div class="article-title">WeederH: an algorithm for finding conserved regulatory motifs and regions in homologous sequences</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">46</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WeederH%3A%20an%20algorithm%20for%20finding%20conserved%20regulatory%20motifs%20and%20regions%20in%20homologous%20sequences&amp;author=G.%20Pavesi&amp;author=F.%20Zambelli&amp;author=G.%20Pesole&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-46" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-46" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-46"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17286865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tomilin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Remenyi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lins</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Synergism with the coactivator OBF-1 (OCA-B, BOB-1) is mediated by a specific POU dimer configuration</div>, <div class="source">Cell</div> , <div class="year">2000</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">853</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synergism%20with%20the%20coactivator%20OBF-1%20(OCA-B%2C%20BOB-1)%20is%20mediated%20by%20a%20specific%20POU%20dimer%20configuration&amp;author=A.%20Tomilin&amp;author=A.%20Remenyi&amp;author=K.%20Lins&amp;publication_year=2000&amp;journal=Cell&amp;volume=103&amp;pages=853-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)00189-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)00189-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)00189-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11136971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botquin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Hess</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fuhrmann</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">New POU dimer configuration mediates antagonistic control of an osteopontin preimplantation enhancer by Oct-4 and Sox-2</div>, <div class="source">Genes Dev.</div> , <div class="year">1998</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">2073</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20POU%20dimer%20configuration%20mediates%20antagonistic%20control%20of%20an%20osteopontin%20preimplantation%20enhancer%20by%20Oct-4%20and%20Sox-2&amp;author=V.%20Botquin&amp;author=H.%20Hess&amp;author=G.%20Fuhrmann&amp;publication_year=1998&amp;journal=Genes%20Dev.&amp;volume=12&amp;pages=2073-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.12.13.2073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.12.13.2073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.12.13.2073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9649510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tantin</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gemberling</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Callister</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fairbrother</div> <div class="given-names">W.</div></div>. </div><div class="article-title">High-throughput biochemical analysis of in vivo location data reveals novel distinct classes of POU5F1(Oct4)/DNA complexes</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">631</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20biochemical%20analysis%20of%20in%20vivo%20location%20data%20reveals%20novel%20distinct%20classes%20of%20POU5F1(Oct4)%2FDNA%20complexes&amp;author=D.%20Tantin&amp;author=M.%20Gemberling&amp;author=C.%20Callister&amp;author=W.%20Fairbrother&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=631-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.072942.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.072942.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.072942.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18212089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yuan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Corbi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Basilico</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dailey</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Developmental-specific activity of the FGF-4 enhancer requires the synergistic action of Sox2 and Oct-3</div>, <div class="source">Genes Dev.</div> , <div class="year">1995</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">2635</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental-specific%20activity%20of%20the%20FGF-4%20enhancer%20requires%20the%20synergistic%20action%20of%20Sox2%20and%20Oct-3&amp;author=H.%20Yuan&amp;author=N.%20Corbi&amp;author=C.%20Basilico&amp;author=L.%20Dailey&amp;publication_year=1995&amp;journal=Genes%20Dev.&amp;volume=9&amp;pages=2635-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.9.21.2635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.9.21.2635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.9.21.2635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7590241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akter</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Chano</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okabe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hirose</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Osumi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Target specificities of estrogen receptor-related receptors: analysis of binding sequences and identification of Rb1-inducible coiled-coil 1 (Rb1cc1) as a target gene</div>, <div class="source">J. Biochem.</div> , <div class="year">2008</div>, vol. <div class="volume">143</div> (pg. <div class="fpage">395</div>-<div class="lpage">406</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target%20specificities%20of%20estrogen%20receptor-related%20receptors%3A%20analysis%20of%20binding%20sequences%20and%20identification%20of%20Rb1-inducible%20coiled-coil%201%20(Rb1cc1)%20as%20a%20target%20gene&amp;author=M.H.%20Akter&amp;author=T.%20Chano&amp;author=H.%20Okabe&amp;author=T.%20Yamaguchi&amp;author=F.%20Hirose&amp;author=T.%20Osumi&amp;publication_year=2008&amp;journal=J.%20Biochem.&amp;volume=143&amp;pages=395-406" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jb/mvm231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjb%2Fmvm231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjb%2Fmvm231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bell</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">West</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Felsenfeld</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The protein CTCF is required for the enhancer blocking activity of vertebrate insulators</div>, <div class="source">Cell</div> , <div class="year">1999</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">387</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20protein%20CTCF%20is%20required%20for%20the%20enhancer%20blocking%20activity%20of%20vertebrate%20insulators&amp;author=A.C.%20Bell&amp;author=A.G.%20West&amp;author=G.%20Felsenfeld&amp;publication_year=1999&amp;journal=Cell&amp;volume=98&amp;pages=387-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81967-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81967-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81967-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10458613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moon</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Filippova</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Loukinov</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">CTCF is conserved from Drosophila to humans and confers enhancer blocking of the Fab-8 insulator</div>, <div class="source">EMBO Rep.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">165</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CTCF%20is%20conserved%20from%20Drosophila%20to%20humans%20and%20confers%20enhancer%20blocking%20of%20the%20Fab-8%20insulator&amp;author=H.%20Moon&amp;author=G.%20Filippova&amp;author=D.%20Loukinov&amp;publication_year=2005&amp;journal=EMBO%20Rep.&amp;volume=6&amp;pages=165-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.embor.7400334" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.embor.7400334" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.embor.7400334"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15678159" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szabo</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Silva</div> <div class="given-names">F.J.</div></div>,  <div class="name"><div class="surname">Tsark</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Mann</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Role of CTCF binding sites in the Igf2/H19 imprinting control region</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">4791</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20CTCF%20binding%20sites%20in%20the%20Igf2%2FH19%20imprinting%20control%20region&amp;author=P.E.%20Szabo&amp;author=S.H.%20Tang&amp;author=F.J.%20Silva&amp;author=W.M.%20Tsark&amp;author=J.R.%20Mann&amp;publication_year=2004&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=24&amp;pages=4791-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.24.11.4791-4800.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.24.11.4791-4800.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.24.11.4791-4800.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15143173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Mikkelsen</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Gnirke</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lindblad-Toh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kellis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">Systematic discovery of regulatory motifs in conserved regions of the human genome, including thousands of CTCF insulator sites</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">7145</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20discovery%20of%20regulatory%20motifs%20in%20conserved%20regions%20of%20the%20human%20genome%2C%20including%20thousands%20of%20CTCF%20insulator%20sites&amp;author=X.%20Xie&amp;author=T.S.%20Mikkelsen&amp;author=A.%20Gnirke&amp;author=K.%20Lindblad-Toh&amp;author=M.%20Kellis&amp;author=E.S.%20Lander&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=7145-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0701811104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0701811104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0701811104"> </span></div> </div></div></div></div></div><div content-id="DSP014C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kulbokas</div> <div class="given-names">E.J.</div></div>, et al. </div><div class="article-title">Systematic discovery of regulatory motifs in human promoters and 3’ UTRs by comparison of several mammals</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">434</div> (pg. <div class="fpage">338</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20discovery%20of%20regulatory%20motifs%20in%20human%20promoters%20and%203%E2%80%99%20UTRs%20by%20comparison%20of%20several%20mammals&amp;author=X.%20Xie&amp;author=J.%20Lu&amp;author=E.J.%20Kulbokas&amp;publication_year=2005&amp;journal=Nature&amp;volume=434&amp;pages=338-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15735639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matys</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fricke</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Geffers</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">TRANSFAC: transcriptional regulation, from patterns to profiles</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">374</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TRANSFAC%3A%20transcriptional%20regulation%2C%20from%20patterns%20to%20profiles&amp;author=V.%20Matys&amp;author=E.%20Fricke&amp;author=R.%20Geffers&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=374-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Faulk</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Retroposition and evolution of the DNA-binding motifs of YY1, YY2 and REX1</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">3442</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retroposition%20and%20evolution%20of%20the%20DNA-binding%20motifs%20of%20YY1%2C%20YY2%20and%20REX1&amp;author=J.D.%20Kim&amp;author=C.%20Faulk&amp;author=J.%20Kim&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=3442-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm235"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17478514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">J.</div></div>. </div><div class="article-title">YY1's longer DNA-binding motifs</div>, <div class="source">Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">152</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=YY1's%20longer%20DNA-binding%20motifs&amp;author=J.%20Kim&amp;publication_year=2009&amp;journal=Genomics&amp;volume=93&amp;pages=152-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2008.09.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2008.09.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2008.09.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18950698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scholer</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Balling</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hatzopoulos</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gruss</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Octamer binding proteins confer transcriptional activity in early mouse embryogenesis</div>, <div class="source">EMBO J.</div> , <div class="year">1989</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">2551</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Octamer%20binding%20proteins%20confer%20transcriptional%20activity%20in%20early%20mouse%20embryogenesis&amp;author=H.R.%20Scholer&amp;author=R.%20Balling&amp;author=A.K.%20Hatzopoulos&amp;author=N.%20Suzuki&amp;author=P.%20Gruss&amp;publication_year=1989&amp;journal=EMBO%20J.&amp;volume=8&amp;pages=2551-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2573524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bruce</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Gardiner</div> <div class="given-names">B.B.</div></div>,  <div class="name"><div class="surname">Burke</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Gongora</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Grimmond</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Perkins</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Dynamic transcription programs during ES cell differentiation towards mesoderm in serum versus serum-freeBMP4 culture</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">365</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20transcription%20programs%20during%20ES%20cell%20differentiation%20towards%20mesoderm%20in%20serum%20versus%20serum-freeBMP4%20culture&amp;author=S.J.%20Bruce&amp;author=B.B.%20Gardiner&amp;author=L.J.%20Burke&amp;author=M.M.%20Gongora&amp;author=S.M.%20Grimmond&amp;author=A.C.%20Perkins&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=365" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-365" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-365" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-365"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17925037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Of mice and men: phylogenetic footprinting aids the discovery of regulatory elements</div>, <div class="source">J. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">2</div> pg. <div class="fpage">11</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Of%20mice%20and%20men%3A%20phylogenetic%20footprinting%20aids%20the%20discovery%20of%20regulatory%20elements&amp;author=Z.%20Zhang&amp;author=M.%20Gerstein&amp;publication_year=2003&amp;journal=J.%20Biol.&amp;volume=2&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-4924-2-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-4924-2-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-4924-2-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12814519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP014C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP014C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reiss</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Baliga</div> <div class="given-names">N.S.</div></div>,  <div class="name"><div class="surname">Bonneau</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Integrated biclustering of heterogeneous genome-wide datasets for the inference of global regulatory networks</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">280</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20biclustering%20of%20heterogeneous%20genome-wide%20datasets%20for%20the%20inference%20of%20global%20regulatory%20networks&amp;author=D.J.%20Reiss&amp;author=N.S.%20Baliga&amp;author=R.%20Bonneau&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=280" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16749936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Kenta Nakai</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=420676&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp014/2/dsp014-dsp014supp.pdf?Expires=2147483647&amp;Signature=TPzyLNAhKTzBlCTVNfRzfr0mGeu0NSA49YXiv1GB3M6DPzsMYcsAVbrBHmMHPhovNt8oq7MWfmb81kCh8VDd2NQTYxZJB~tE-n-MNF~qFUJcmRWkLkh0QKWjr5bIFvbNonDWstB1gn8LlPJ8kcB8SGY2cc7345epmHiKKIJRC0TLkB-YdkuuAeuiJZbqZTqLAU8eR42z5nVWZTh4mORrLu2bhhF5s9NofNej4bAJAJuObXym4dbrY59yPR3W5qMgfXPIFO2dW5-ZI6G1R2Xjn3c8Uup4nWRUK9QZE5ljWLfhQT7zaFHAuobFUUSe51FsPXNE~jcH~S9QU6oQaLjv3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">279</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">75</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="420676"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f5%2f261%2f420676">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000271610400002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (75)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F5%2F261%2F420676" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/420676'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Exhaustive Search for Over-represented DNA Sequence Motifs with CisFinder" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f261%2f420676">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

