

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/1/23/347368","@type":"ScholarlyArticle","name":"Determination of the  in vivo  structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach","datePublished":"2010-01-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2010-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp027","keywords":["DNA topology","loop attachment regions","matrix attachment regions","nuclear matrix","nucleotype"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/1/23/347368","author":[{"name":"Rivera-Mulia, Juan Carlos","affiliation":"1  Laboratorio de Biología Molecular, Facultad de Medicina ,  Universidad Autónoma del Estado de México ,  Apartado Postal 428, C.P. 50000, Toluca, Edo. Méx. ,  México    2  Programa de Doctorado en Ciencias Biomédicas ,  IFC-UNAM, Apartado Postal 70-243, C.P. 04510, D.F. ,  México","@type":"Person"},{"name":"Aranda-Anzaldo, Armando","affiliation":"1  Laboratorio de Biología Molecular, Facultad de Medicina ,  Universidad Autónoma del Estado de México ,  Apartado Postal 428, C.P. 50000, Toluca, Edo. Méx. ,  México    2  Programa de Doctorado en Ciencias Biomédicas ,  IFC-UNAM, Apartado Postal 70-243, C.P. 04510, D.F. ,  México","@type":"Person"}],"description":"Abstract.  Nuclear DNA of metazoans is organized in supercoiled loops anchored to a proteinaceous substructure known as the nuclear matrix (NM). DNA is anchored","pageStart":"23","pageEnd":"35","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/1/1/m_cover.gif?Expires=2147483647&Signature=gmtBwQKOCxNij4txyJ-Agd3ex2K7zADLjZgB4yoyDPCuCBxm8rNqvqkhW5k6zWBTEHDsBFHh9yD-hWwwdQR9j5ewB01pfPET55PUMFnQxV39XGIpAiyOJgm-CjrYsQ7Tz~Arn77OBKRlDKReDeozBw~0n-G7uI3SEANuwJK-8WBdyocI7TAaLcJkMyRKHt5P5LZnOwNVGCz8AWDCrjmjcbJFlGWIUbXJLzq4Nh7xOjprYqGF9FDg3nxz1zD3Snjv6oCeeIactT6vD6Ghh7KyKs-edLg7X8Az-CDyI0Hk2p9cS~WPWLe7yUhnUsOYP2HXUj-RQ-PGFInBtBwboH1Esg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Determination of the  in vivo  structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/1/1/m_cover.gif?Expires=2147483647&Signature=gmtBwQKOCxNij4txyJ-Agd3ex2K7zADLjZgB4yoyDPCuCBxm8rNqvqkhW5k6zWBTEHDsBFHh9yD-hWwwdQR9j5ewB01pfPET55PUMFnQxV39XGIpAiyOJgm-CjrYsQ7Tz~Arn77OBKRlDKReDeozBw~0n-G7uI3SEANuwJK-8WBdyocI7TAaLcJkMyRKHt5P5LZnOwNVGCz8AWDCrjmjcbJFlGWIUbXJLzq4Nh7xOjprYqGF9FDg3nxz1zD3Snjv6oCeeIactT6vD6Ghh7KyKs-edLg7X8Az-CDyI0Hk2p9cS~WPWLe7yUhnUsOYP2HXUj-RQ-PGFInBtBwboH1Esg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Determination of the  in vivo  structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach" />
    <meta property="og:description" content="Abstract.  Nuclear DNA of metazoans is organized in supercoiled loops anchored to a proteinaceous substructure known as the nuclear matrix (NM). DNA is anchored" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/1/23/347368" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-01-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Rivera-Mulia, Juan Carlos" /><meta name="citation_author_institution" content="Laboratorio de Biología Molecular, Facultad de Medicina, Universidad Autónoma del Estado de México, Apartado Postal 428, C.P. 50000, Toluca, Edo. Méx., México" /><meta name="citation_author_institution" content="Programa de Doctorado en Ciencias Biomédicas, IFC-UNAM, Apartado Postal 70-243, C.P. 04510, D.F., México" /><meta name="citation_author" content="Aranda-Anzaldo, Armando" /><meta name="citation_author_institution" content="Laboratorio de Biología Molecular, Facultad de Medicina, Universidad Autónoma del Estado de México, Apartado Postal 428, C.P. 50000, Toluca, Edo. Méx., México" /><meta name="citation_author_institution" content="Programa de Doctorado en Ciencias Biomédicas, IFC-UNAM, Apartado Postal 70-243, C.P. 04510, D.F., México" /><meta name="citation_title" content="Determination of the in vivo structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach" /><meta name="citation_firstpage" content="23" /><meta name="citation_lastpage" content="35" /><meta name="citation_doi" content="10.1093/dnares/dsp027" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2010/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Experimental observations of a nuclear matrix; J. Cell Sci.;  citation_year=2001;  citation_volume=114;  citation_pages=463-74; " /><meta name="citation_reference" content="citation_title=Dynamic view of the nuclear matrix; Acta Med. Okayama;  citation_year=2005;  citation_volume=59;  citation_pages=113-20; " /><meta name="citation_reference" content="citation_title=NMPdb: database of nuclear matrix proteins; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D160-3; " /><meta name="citation_reference" content="citation_title=Anchoring the genome; Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=2011" /><meta name="citation_reference" content="citation_title=The nuclear matrix and chromosomal DNA loops: is there any correlation between partitioning of the genome into loops and functional domains?; Cell Mol. Biol. Lett.;  citation_year=2001;  citation_volume=6;  citation_pages=59-69; " /><meta name="citation_reference" content="citation_title=Gene positional changes relative to the nuclear substructure correlate with the proliferating status of hepatocytes during liver regeneration; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6168-79; " /><meta name="citation_reference" content="citation_title=Chromatin loops are selectively anchored using scaffold/matrix-attachment regions; J. Cell Sci.;  citation_year=2004;  citation_volume=117;  citation_pages=999-1008; " /><meta name="citation_reference" content="citation_title=Specific radial positions of human chromosomes X, 1 and 19 remain unchanged in chromatin depleted nuclei of primary human fibroblasts: evidence for the organizing role of the nuclear matrix; J. Cell. Biochem.;  citation_year=2005;  citation_volume=96;  citation_pages=850-7; " /><meta name="citation_reference" content="citation_title=Dynamic properties of the nuclear matrix; citation_publisher=Academic Press, Orlando; The Cell Nucleus;  citation_year=1979;  citation_pages=413-56; " /><meta name="citation_reference" content="citation_title=On the role of chromatin higher-order structure and mechanical interactions in the regulation of gene expression; Speculations Sci. Technol.;  citation_year=1989;  citation_volume=12;  citation_pages=163-76; " /><meta name="citation_reference" content="citation_title=DNA binding properties of the nuclear matrix and individual nuclear matrix proteins; J. Biol. Chem.;  citation_year=1991;  citation_volume=266;  citation_pages=11131-40; " /><meta name="citation_reference" content="citation_title=Dynamic association of replicating DNA fragments with the nuclear matrix of regenerating liver; Exp. Cell Res.;  citation_year=1981;  citation_volume=132;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Natural ageing in the rat liver correlates with progressive stabilisation of DNA-nuclear matrix interactions and withdrawal of genes from the nuclear substructure; Mech. Ageing Dev.;  citation_year=2005;  citation_volume=126;  citation_pages=767-82; " /><meta name="citation_reference" content="citation_title=Scaffold/matrix-attached regions: structural properties creating transcriptionally active loci; Int. Rev. Cytol.;  citation_year=1995;  citation_volume=162A;  citation_pages=389-454; " /><meta name="citation_reference" content="citation_title=Predicting indirect readout effects in protein-DNA interactions; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=8337-41; " /><meta name="citation_reference" content="citation_title=The nuclear scaffold exhibits DNA-binding sites selective for supercoiled DNA; J. Biol. Chem.;  citation_year=1988;  citation_volume=263;  citation_pages=7235-41; " /><meta name="citation_reference" content="citation_title=Segregation of transcription and replication sites into higher order domains; Science;  citation_year=1998;  citation_volume=281;  citation_pages=1502-5; " /><meta name="citation_reference" content="citation_title=The organization of replication and transcription; Science;  citation_year=1999;  citation_volume=282;  citation_pages=1790-5; " /><meta name="citation_reference" content="citation_title=Nuclear matrix support of DNA replication; J. Cell Biochem.;  citation_year=2005;  citation_volume=96;  citation_pages=951-61; " /><meta name="citation_reference" content="citation_title=Loss of DNA-loop supercoiling and organization in cells infected by herpes simplex virus type 1; Res. Virol.;  citation_year=1997;  citation_volume=148;  citation_pages=397-408; " /><meta name="citation_reference" content="citation_title=p53 is a rate-limiting factor in the repair of higher-order DNA structure; Biochim. Biophys. Acta;  citation_year=1999;  citation_volume=1446;  citation_pages=181-92; " /><meta name="citation_reference" content="citation_title=The c-myc insulator element and matrix attachment region define the c-myc chromosomal domain; Mol. Cell Biol.;  citation_year=2003;  citation_volume=23;  citation_pages=9338-48; " /><meta name="citation_reference" content="citation_title=Visualization of individual DNA loops and a map of loop domains in the human dystrophin gene; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=2079-86; " /><meta name="citation_reference" content="citation_title=Chromatin loop domain organization within the 4q35 locus in facioscapulohumeral dystrophy patients versus normal human myoblasts; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=6982-7; " /><meta name="citation_reference" content="citation_title=Determination of the chromatin domain structure in arrayed repeated regions: organization of the somatic 5S RNA domain during embryogenesis in Xenopus laevis; J. Cell Biochem.;  citation_year=2007;  citation_volume=102;  citation_pages=1140-8; " /><meta name="citation_reference" content="citation_title=Functional interaction between PML and SATB1 regulates chromatin-loop architecture and transcription of the MHC class 1 locus; Nat. Cell Biol.;  citation_year=2007;  citation_volume=9;  citation_pages=45-56; " /><meta name="citation_reference" content="citation_title=In silico prediction of scaffold/matrix attachment regions in large genomic sequences; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=349-54; " /><meta name="citation_reference" content="citation_title=A comparative study of S/MAR prediction tools; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=71" /><meta name="citation_reference" content="citation_title=Reconfiguration of genomic anchors upon transcriptional activation of the human major histocompatibility complex; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1778-1786; " /><meta name="citation_reference" content="citation_title=Differential nuclear scaffold/matrix attachment marks expressed genes; Hum. Mol. Genet.;  citation_year=2009;  citation_volume=18;  citation_pages=645-54; " /><meta name="citation_reference" content="citation_title=Positional mapping of specific DNA sequences relative to the nuclear substructure by direct polymerase chain reaction on nuclear matrix-bound templates; Anal. Biochem.;  citation_year=2003;  citation_volume=313;  citation_pages=196-207; " /><meta name="citation_reference" content="citation_title=Mathematical model to predict regions of chromatin attachment to the nuclear matrix; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=1419-25; " /><meta name="citation_reference" content="citation_title=A bipartite sequence element associated with matrix/scaffold attachment regions; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=2924-30; " /><meta name="citation_reference" content="citation_title=Comparative study and prediction of DNA fragments associated with various elements of the nuclear matrix; Biochim. Biophys. Acta;  citation_year=2001;  citation_volume=1517;  citation_pages=351-64; " /><meta name="citation_reference" content=" citation_edition=3rd edition; citation_publisher=Worth Publishers, New York; Lehninger Principles of Biochemistry;  citation_year=2000;  citation_pages=915-8; " /><meta name="citation_reference" content=" citation_edition=3rd edition; citation_publisher=Elsevier-Academic Press, London; Understanding DNA;  citation_year=2004;  citation_pages=94-138; " /><meta name="citation_reference" content="citation_title=Biological significance of unwinding capability of nuclear matrix-associating DNAs; Science;  citation_year=1992;  citation_volume=255;  citation_pages=195-7; " /><meta name="citation_reference" content="citation_title=Stress-induced duplex destabilization in scaffold/matrix attachment regions; J. Mol. Biol.;  citation_year=1997;  citation_volume=2774;  citation_pages=181-96; " /><meta name="citation_reference" content="citation_title=Gene positional changes relative to the nuclear substructure during carbon tetrachloride-induced hepatic fibrosis in rats; J. Cell Biochem.;  citation_year=2004;  citation_volume=93;  citation_pages=1084-98; " /><meta name="citation_reference" content=" citation_edition=2nd edition; citation_publisher=John Wiley and Sons, New York; Gene Expression 2;  citation_year=1980;  citation_pages=360-2; " /><meta name="citation_reference" content="citation_title=Long-range fragmentation of the eukaryotic genome by exogenous and endogenous nucleases proceeds in a specific fashion via preferential DNA cleavage at matrix attachment sites; J. Biol. Chem.;  citation_year=1995;  citation_volume=270;  citation_pages=18685-90; " /><meta name="citation_reference" content="citation_title=The alpha-fetoprotein enhancer region activates the albumin and alpha-fetoprotein promoters during liver development; citation_text=doi:10.1016/j.dbio.2009.09.026; Dev. Biol.;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=DNA interaction with the nuclear matrix and spatial organization of replication and transcription; BioEssays;  citation_year=1987;  citation_volume=6;  citation_pages=19-23; " /><meta name="citation_reference" content="citation_title=Identification and characterization of nuclear matrix-attachment regions in the human serpin gene cluster at 14q32.1; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=3779-91; " /><meta name="citation_reference" content="citation_title=Different topoisomerase II antitumor drugs direct similar specific long-range fragmentation of an amplified c-myc gene locus in living cells and in high-salt-extracted nuclei; Proc. Natl Acad. Sci. USA;  citation_year=1995;  citation_volume=92;  citation_pages=102-6; " /><meta name="citation_reference" content="citation_title=Histone modifications, chromatin structure and the nuclear matrix; J. Cell Biochem.;  citation_year=1996;  citation_volume=62;  citation_pages=149-57; " /><meta name="citation_reference" content="citation_title=Chromatin loops, illegitimate recombination and genome evolution; BioEssays;  citation_year=2009;  citation_volume=31;  citation_pages=278-86; " /><meta name="citation_reference" content="citation_title=Molecular mechanisms of alpha-fetoprotein gene expression; Biochemistry (Mosc.);  citation_year=2000;  citation_volume=65;  citation_pages=117-33; " /><meta name="citation_reference" content="citation_title=An albumin enhancer located 10 kb upstream functions along with its promoter to direct efficient, liver-specific expression in transgenic mice; Genes Dev.;  citation_year=1987;  citation_volume=1;  citation_pages=268-76; " /><meta name="citation_reference" content="citation_title=Transcription factories are nuclear subcompartments that remain in the absence of transcription; Genes Dev.;  citation_year=2008;  citation_volume=22;  citation_pages=20-5; " /><meta name="citation_reference" content="citation_title=Replicon clusters are stable units of chromosome structure: evidence that nuclear organization contributes to the efficient activation and propagation of S phase in human cells; J. Cell Biol.;  citation_year=1998;  citation_volume=140;  citation_pages=1285-95; " /><meta name="citation_reference" content="citation_title=The spatio-temporal organization of DNA replication sites is identical in primary, immortalized and transformed mammalian cells; J. Cell Sci.;  citation_year=2002;  citation_volume=115;  citation_pages=4037-51; " /><meta name="citation_reference" content="citation_title=Myc dynamically and preferentially relocates to a transcription factory occupied by Igh; PLoS Biol.;  citation_year=2007;  citation_volume=5;  citation_pages=e192" /><meta name="citation_reference" content="citation_title=Maintenance of a functional higher order chromatin structure: the role of the nuclear matrix in normal and disease states; Gene Ther. Mol. Biol.;  citation_year=2009;  citation_volume=13;  citation_pages=231-43; " /><meta name="citation_reference" content="citation_title=A global but stable change in HeLa cell morphology induces reorganization of DNA structural loop domains within the cell nucleus; J. Cell Biochem.;  citation_year=2005;  citation_volume=96;  citation_pages=79-88; " /><meta name="citation_reference" content="citation_title=A structural basis for cellular senescence; Aging;  citation_year=2009;  citation_volume=1;  citation_pages=598-607; " /><meta name="citation_reference" content="citation_title=Mitotic remodelling of the replicon and chromosome structure; Cell;  citation_year=2005;  citation_volume=123;  citation_pages=787-801; " /><meta name="citation_reference" content="citation_title=The time and duration of meiosis; Philos. Trans. R. Soc. Lond. B, Biol. Sci.;  citation_year=1977;  citation_volume=277;  citation_pages=201-26; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/1/23/994565/dsp027.pdf" /><meta name="description" content="Abstract.  Nuclear DNA of metazoans is organized in supercoiled loops anchored to a proteinaceous substructure known as the nuclear matrix (NM). DNA is anchored" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/1/23/347368" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/1/23/347368" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/1/23/347368");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp027';
    </script>
<title>
	Determination of the in vivo structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347368" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bdl7PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSaXZlcmEtTXVsaWEsIEp1YW4gQ2FybG9zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yaW8gZGUgQmlvbG9nw61hIE1vbGVjdWxhciwgRmFjdWx0YWQgZGUgTWVkaWNpbmEsIFVuaXZlcnNpZGFkIEF1dMOzbm9tYSBkZWwgRXN0YWRvIGRlIE3DqXhpY28sIEFwYXJ0YWRvIFBvc3RhbCA0MjgsIEMuUC4gNTAwMDAsIFRvbHVjYSwgRWRvLiBNw6l4LiwgTcOpeGljbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlByb2dyYW1hIGRlIERvY3RvcmFkbyBlbiBDaWVuY2lhcyBCaW9tw6lkaWNhcywgSUZDLVVOQU0sIEFwYXJ0YWRvIFBvc3RhbCA3MC0yNDMsIEMuUC4gMDQ1MTAsIEQuRi4sIE3DqXhpY28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmFuZGEtQW56YWxkbywgQXJtYW5kbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcmlvIGRlIEJpb2xvZ8OtYSBNb2xlY3VsYXIsIEZhY3VsdGFkIGRlIE1lZGljaW5hLCBVbml2ZXJzaWRhZCBBdXTDs25vbWEgZGVsIEVzdGFkbyBkZSBNw6l4aWNvLCBBcGFydGFkbyBQb3N0YWwgNDI4LCBDLlAuIDUwMDAwLCBUb2x1Y2EsIEVkby4gTcOpeC4sIE3DqXhpY28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQcm9ncmFtYSBkZSBEb2N0b3JhZG8gZW4gQ2llbmNpYXMgQmlvbcOpZGljYXMsIElGQy1VTkFNLCBBcGFydGFkbyBQb3N0YWwgNzAtMjQzLCBDLlAuIDA0NTEwLCBELkYuLCBNw6l4aWNvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXRlcm1pbmF0aW9uIG9mIHRoZSBpbiB2aXZvIHN0cnVjdHVyYWwgRE5BIGxvb3Agb3JnYW5pemF0aW9uIGluIHRoZSBnZW5vbWljIHJlZ2lvbiBvZiB0aGUgcmF0IGFsYnVtaW4gbG9jdXMgYnkgbWVhbnMgb2YgYSB0b3BvbG9naWNhbCBhcHByb2FjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcDAyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwZXJpbWVudGFsIG9ic2VydmF0aW9ucyBvZiBhIG51Y2xlYXIgbWF0cml4OyBKLiBDZWxsIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9NDYzLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIHZpZXcgb2YgdGhlIG51Y2xlYXIgbWF0cml4OyBBY3RhIE1lZC4gT2theWFtYTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTExMy0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tk1QZGI6IGRhdGFiYXNlIG9mIG51Y2xlYXIgbWF0cml4IHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1EMTYwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuY2hvcmluZyB0aGUgZ2Vub21lOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIwMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbnVjbGVhciBtYXRyaXggYW5kIGNocm9tb3NvbWFsIEROQSBsb29wczogaXMgdGhlcmUgYW55IGNvcnJlbGF0aW9uIGJldHdlZW4gcGFydGl0aW9uaW5nIG9mIHRoZSBnZW5vbWUgaW50byBsb29wcyBhbmQgZnVuY3Rpb25hbCBkb21haW5zPzsgQ2VsbCBNb2wuIEJpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz01OS02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwb3NpdGlvbmFsIGNoYW5nZXMgcmVsYXRpdmUgdG8gdGhlIG51Y2xlYXIgc3Vic3RydWN0dXJlIGNvcnJlbGF0ZSB3aXRoIHRoZSBwcm9saWZlcmF0aW5nIHN0YXR1cyBvZiBoZXBhdG9jeXRlcyBkdXJpbmcgbGl2ZXIgcmVnZW5lcmF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz02MTY4LTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbWF0aW4gbG9vcHMgYXJlIHNlbGVjdGl2ZWx5IGFuY2hvcmVkIHVzaW5nIHNjYWZmb2xkL21hdHJpeC1hdHRhY2htZW50IHJlZ2lvbnM7IEouIENlbGwgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz05OTktMTAwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2lmaWMgcmFkaWFsIHBvc2l0aW9ucyBvZiBodW1hbiBjaHJvbW9zb21lcyBYLCAxIGFuZCAxOSByZW1haW4gdW5jaGFuZ2VkIGluIGNocm9tYXRpbiBkZXBsZXRlZCBudWNsZWkgb2YgcHJpbWFyeSBodW1hbiBmaWJyb2JsYXN0czogZXZpZGVuY2UgZm9yIHRoZSBvcmdhbml6aW5nIHJvbGUgb2YgdGhlIG51Y2xlYXIgbWF0cml4OyBKLiBDZWxsLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTg1MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIHByb3BlcnRpZXMgb2YgdGhlIG51Y2xlYXIgbWF0cml4OyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIE9ybGFuZG87IFRoZSBDZWxsIE51Y2xldXM7ICBjaXRhdGlvbl95ZWFyPTE5Nzk7ICBjaXRhdGlvbl9wYWdlcz00MTMtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHRoZSByb2xlIG9mIGNocm9tYXRpbiBoaWdoZXItb3JkZXIgc3RydWN0dXJlIGFuZCBtZWNoYW5pY2FsIGludGVyYWN0aW9ucyBpbiB0aGUgcmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb247IFNwZWN1bGF0aW9ucyBTY2kuIFRlY2hub2wuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTYzLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgYmluZGluZyBwcm9wZXJ0aWVzIG9mIHRoZSBudWNsZWFyIG1hdHJpeCBhbmQgaW5kaXZpZHVhbCBudWNsZWFyIG1hdHJpeCBwcm90ZWluczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9MTExMzEtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgYXNzb2NpYXRpb24gb2YgcmVwbGljYXRpbmcgRE5BIGZyYWdtZW50cyB3aXRoIHRoZSBudWNsZWFyIG1hdHJpeCBvZiByZWdlbmVyYXRpbmcgbGl2ZXI7IEV4cC4gQ2VsbCBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgYWdlaW5nIGluIHRoZSByYXQgbGl2ZXIgY29ycmVsYXRlcyB3aXRoIHByb2dyZXNzaXZlIHN0YWJpbGlzYXRpb24gb2YgRE5BLW51Y2xlYXIgbWF0cml4IGludGVyYWN0aW9ucyBhbmQgd2l0aGRyYXdhbCBvZiBnZW5lcyBmcm9tIHRoZSBudWNsZWFyIHN1YnN0cnVjdHVyZTsgTWVjaC4gQWdlaW5nIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9NzY3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY2FmZm9sZC9tYXRyaXgtYXR0YWNoZWQgcmVnaW9uczogc3RydWN0dXJhbCBwcm9wZXJ0aWVzIGNyZWF0aW5nIHRyYW5zY3JpcHRpb25hbGx5IGFjdGl2ZSBsb2NpOyBJbnQuIFJldi4gQ3l0b2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE2MkE7ICBjaXRhdGlvbl9wYWdlcz0zODktNDU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIGluZGlyZWN0IHJlYWRvdXQgZWZmZWN0cyBpbiBwcm90ZWluLUROQSBpbnRlcmFjdGlvbnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9ODMzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG51Y2xlYXIgc2NhZmZvbGQgZXhoaWJpdHMgRE5BLWJpbmRpbmcgc2l0ZXMgc2VsZWN0aXZlIGZvciBzdXBlcmNvaWxlZCBETkE7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTI2MzsgIGNpdGF0aW9uX3BhZ2VzPTcyMzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlZ3JlZ2F0aW9uIG9mIHRyYW5zY3JpcHRpb24gYW5kIHJlcGxpY2F0aW9uIHNpdGVzIGludG8gaGlnaGVyIG9yZGVyIGRvbWFpbnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjgxOyAgY2l0YXRpb25fcGFnZXM9MTUwMi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JnYW5pemF0aW9uIG9mIHJlcGxpY2F0aW9uIGFuZCB0cmFuc2NyaXB0aW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI4MjsgIGNpdGF0aW9uX3BhZ2VzPTE3OTAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBtYXRyaXggc3VwcG9ydCBvZiBETkEgcmVwbGljYXRpb247IEouIENlbGwgQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz05NTEtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvc3Mgb2YgRE5BLWxvb3Agc3VwZXJjb2lsaW5nIGFuZCBvcmdhbml6YXRpb24gaW4gY2VsbHMgaW5mZWN0ZWQgYnkgaGVycGVzIHNpbXBsZXggdmlydXMgdHlwZSAxOyBSZXMuIFZpcm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz0zOTctNDA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1wNTMgaXMgYSByYXRlLWxpbWl0aW5nIGZhY3RvciBpbiB0aGUgcmVwYWlyIG9mIGhpZ2hlci1vcmRlciBETkEgc3RydWN0dXJlOyBCaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE0NDY7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjLW15YyBpbnN1bGF0b3IgZWxlbWVudCBhbmQgbWF0cml4IGF0dGFjaG1lbnQgcmVnaW9uIGRlZmluZSB0aGUgYy1teWMgY2hyb21vc29tYWwgZG9tYWluOyBNb2wuIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz05MzM4LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WaXN1YWxpemF0aW9uIG9mIGluZGl2aWR1YWwgRE5BIGxvb3BzIGFuZCBhIG1hcCBvZiBsb29wIGRvbWFpbnMgaW4gdGhlIGh1bWFuIGR5c3Ryb3BoaW4gZ2VuZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MjA3OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21hdGluIGxvb3AgZG9tYWluIG9yZ2FuaXphdGlvbiB3aXRoaW4gdGhlIDRxMzUgbG9jdXMgaW4gZmFjaW9zY2FwdWxvaHVtZXJhbCBkeXN0cm9waHkgcGF0aWVudHMgdmVyc3VzIG5vcm1hbCBodW1hbiBteW9ibGFzdHM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9Njk4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlcm1pbmF0aW9uIG9mIHRoZSBjaHJvbWF0aW4gZG9tYWluIHN0cnVjdHVyZSBpbiBhcnJheWVkIHJlcGVhdGVkIHJlZ2lvbnM6IG9yZ2FuaXphdGlvbiBvZiB0aGUgc29tYXRpYyA1UyBSTkEgZG9tYWluIGR1cmluZyBlbWJyeW9nZW5lc2lzIGluIFhlbm9wdXMgbGFldmlzOyBKLiBDZWxsIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTExNDAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBpbnRlcmFjdGlvbiBiZXR3ZWVuIFBNTCBhbmQgU0FUQjEgcmVndWxhdGVzIGNocm9tYXRpbi1sb29wIGFyY2hpdGVjdHVyZSBhbmQgdHJhbnNjcmlwdGlvbiBvZiB0aGUgTUhDIGNsYXNzIDEgbG9jdXM7IE5hdC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDUtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHNpbGljbyBwcmVkaWN0aW9uIG9mIHNjYWZmb2xkL21hdHJpeCBhdHRhY2htZW50IHJlZ2lvbnMgaW4gbGFyZ2UgZ2Vub21pYyBzZXF1ZW5jZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzQ5LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmF0aXZlIHN0dWR5IG9mIFMvTUFSIHByZWRpY3Rpb24gdG9vbHM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbmZpZ3VyYXRpb24gb2YgZ2Vub21pYyBhbmNob3JzIHVwb24gdHJhbnNjcmlwdGlvbmFsIGFjdGl2YXRpb24gb2YgdGhlIGh1bWFuIG1ham9yIGhpc3RvY29tcGF0aWJpbGl0eSBjb21wbGV4OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE3NzgtMTc4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIG51Y2xlYXIgc2NhZmZvbGQvbWF0cml4IGF0dGFjaG1lbnQgbWFya3MgZXhwcmVzc2VkIGdlbmVzOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjQ1LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3NpdGlvbmFsIG1hcHBpbmcgb2Ygc3BlY2lmaWMgRE5BIHNlcXVlbmNlcyByZWxhdGl2ZSB0byB0aGUgbnVjbGVhciBzdWJzdHJ1Y3R1cmUgYnkgZGlyZWN0IHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb24gb24gbnVjbGVhciBtYXRyaXgtYm91bmQgdGVtcGxhdGVzOyBBbmFsLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz0xOTYtMjA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXRoZW1hdGljYWwgbW9kZWwgdG8gcHJlZGljdCByZWdpb25zIG9mIGNocm9tYXRpbiBhdHRhY2htZW50IHRvIHRoZSBudWNsZWFyIG1hdHJpeDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTQxOS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBiaXBhcnRpdGUgc2VxdWVuY2UgZWxlbWVudCBhc3NvY2lhdGVkIHdpdGggbWF0cml4L3NjYWZmb2xkIGF0dGFjaG1lbnQgcmVnaW9uczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjkyNC0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgc3R1ZHkgYW5kIHByZWRpY3Rpb24gb2YgRE5BIGZyYWdtZW50cyBhc3NvY2lhdGVkIHdpdGggdmFyaW91cyBlbGVtZW50cyBvZiB0aGUgbnVjbGVhciBtYXRyaXg7IEJpb2NoaW0uIEJpb3BoeXMuIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTUxNzsgIGNpdGF0aW9uX3BhZ2VzPTM1MS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249M3JkIGVkaXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1Xb3J0aCBQdWJsaXNoZXJzLCBOZXcgWW9yazsgTGVobmluZ2VyIFByaW5jaXBsZXMgb2YgQmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fcGFnZXM9OTE1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBjaXRhdGlvbl9lZGl0aW9uPTNyZCBlZGl0aW9uOyBjaXRhdGlvbl9wdWJsaXNoZXI9RWxzZXZpZXItQWNhZGVtaWMgUHJlc3MsIExvbmRvbjsgVW5kZXJzdGFuZGluZyBETkE7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl9wYWdlcz05NC0xMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2xvZ2ljYWwgc2lnbmlmaWNhbmNlIG9mIHVud2luZGluZyBjYXBhYmlsaXR5IG9mIG51Y2xlYXIgbWF0cml4LWFzc29jaWF0aW5nIEROQXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjU1OyAgY2l0YXRpb25fcGFnZXM9MTk1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmVzcy1pbmR1Y2VkIGR1cGxleCBkZXN0YWJpbGl6YXRpb24gaW4gc2NhZmZvbGQvbWF0cml4IGF0dGFjaG1lbnQgcmVnaW9uczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzc0OyAgY2l0YXRpb25fcGFnZXM9MTgxLTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHBvc2l0aW9uYWwgY2hhbmdlcyByZWxhdGl2ZSB0byB0aGUgbnVjbGVhciBzdWJzdHJ1Y3R1cmUgZHVyaW5nIGNhcmJvbiB0ZXRyYWNobG9yaWRlLWluZHVjZWQgaGVwYXRpYyBmaWJyb3NpcyBpbiByYXRzOyBKLiBDZWxsIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9MTA4NC05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249Mm5kIGVkaXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1Kb2huIFdpbGV5IGFuZCBTb25zLCBOZXcgWW9yazsgR2VuZSBFeHByZXNzaW9uIDI7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl9wYWdlcz0zNjAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZy1yYW5nZSBmcmFnbWVudGF0aW9uIG9mIHRoZSBldWthcnlvdGljIGdlbm9tZSBieSBleG9nZW5vdXMgYW5kIGVuZG9nZW5vdXMgbnVjbGVhc2VzIHByb2NlZWRzIGluIGEgc3BlY2lmaWMgZmFzaGlvbiB2aWEgcHJlZmVyZW50aWFsIEROQSBjbGVhdmFnZSBhdCBtYXRyaXggYXR0YWNobWVudCBzaXRlczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjcwOyAgY2l0YXRpb25fcGFnZXM9MTg2ODUtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhbHBoYS1mZXRvcHJvdGVpbiBlbmhhbmNlciByZWdpb24gYWN0aXZhdGVzIHRoZSBhbGJ1bWluIGFuZCBhbHBoYS1mZXRvcHJvdGVpbiBwcm9tb3RlcnMgZHVyaW5nIGxpdmVyIGRldmVsb3BtZW50OyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDE2L2ouZGJpby4yMDA5LjA5LjAyNjsgRGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGludGVyYWN0aW9uIHdpdGggdGhlIG51Y2xlYXIgbWF0cml4IGFuZCBzcGF0aWFsIG9yZ2FuaXphdGlvbiBvZiByZXBsaWNhdGlvbiBhbmQgdHJhbnNjcmlwdGlvbjsgQmlvRXNzYXlzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xOS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgbnVjbGVhciBtYXRyaXgtYXR0YWNobWVudCByZWdpb25zIGluIHRoZSBodW1hbiBzZXJwaW4gZ2VuZSBjbHVzdGVyIGF0IDE0cTMyLjE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTM3NzktOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudCB0b3BvaXNvbWVyYXNlIElJIGFudGl0dW1vciBkcnVncyBkaXJlY3Qgc2ltaWxhciBzcGVjaWZpYyBsb25nLXJhbmdlIGZyYWdtZW50YXRpb24gb2YgYW4gYW1wbGlmaWVkIGMtbXljIGdlbmUgbG9jdXMgaW4gbGl2aW5nIGNlbGxzIGFuZCBpbiBoaWdoLXNhbHQtZXh0cmFjdGVkIG51Y2xlaTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT05MjsgIGNpdGF0aW9uX3BhZ2VzPTEwMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0b25lIG1vZGlmaWNhdGlvbnMsIGNocm9tYXRpbiBzdHJ1Y3R1cmUgYW5kIHRoZSBudWNsZWFyIG1hdHJpeDsgSi4gQ2VsbCBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTE0OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21hdGluIGxvb3BzLCBpbGxlZ2l0aW1hdGUgcmVjb21iaW5hdGlvbiBhbmQgZ2Vub21lIGV2b2x1dGlvbjsgQmlvRXNzYXlzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9Mjc4LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWVjaGFuaXNtcyBvZiBhbHBoYS1mZXRvcHJvdGVpbiBnZW5lIGV4cHJlc3Npb247IEJpb2NoZW1pc3RyeSAoTW9zYy4pOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9MTE3LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbGJ1bWluIGVuaGFuY2VyIGxvY2F0ZWQgMTAga2IgdXBzdHJlYW0gZnVuY3Rpb25zIGFsb25nIHdpdGggaXRzIHByb21vdGVyIHRvIGRpcmVjdCBlZmZpY2llbnQsIGxpdmVyLXNwZWNpZmljIGV4cHJlc3Npb24gaW4gdHJhbnNnZW5pYyBtaWNlOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yNjgtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gZmFjdG9yaWVzIGFyZSBudWNsZWFyIHN1YmNvbXBhcnRtZW50cyB0aGF0IHJlbWFpbiBpbiB0aGUgYWJzZW5jZSBvZiB0cmFuc2NyaXB0aW9uOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwbGljb24gY2x1c3RlcnMgYXJlIHN0YWJsZSB1bml0cyBvZiBjaHJvbW9zb21lIHN0cnVjdHVyZTogZXZpZGVuY2UgdGhhdCBudWNsZWFyIG9yZ2FuaXphdGlvbiBjb250cmlidXRlcyB0byB0aGUgZWZmaWNpZW50IGFjdGl2YXRpb24gYW5kIHByb3BhZ2F0aW9uIG9mIFMgcGhhc2UgaW4gaHVtYW4gY2VsbHM7IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQwOyAgY2l0YXRpb25fcGFnZXM9MTI4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNwYXRpby10ZW1wb3JhbCBvcmdhbml6YXRpb24gb2YgRE5BIHJlcGxpY2F0aW9uIHNpdGVzIGlzIGlkZW50aWNhbCBpbiBwcmltYXJ5LCBpbW1vcnRhbGl6ZWQgYW5kIHRyYW5zZm9ybWVkIG1hbW1hbGlhbiBjZWxsczsgSi4gQ2VsbCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMzctNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU15YyBkeW5hbWljYWxseSBhbmQgcHJlZmVyZW50aWFsbHkgcmVsb2NhdGVzIHRvIGEgdHJhbnNjcmlwdGlvbiBmYWN0b3J5IG9jY3VwaWVkIGJ5IElnaDsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTE5MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1haW50ZW5hbmNlIG9mIGEgZnVuY3Rpb25hbCBoaWdoZXIgb3JkZXIgY2hyb21hdGluIHN0cnVjdHVyZTogdGhlIHJvbGUgb2YgdGhlIG51Y2xlYXIgbWF0cml4IGluIG5vcm1hbCBhbmQgZGlzZWFzZSBzdGF0ZXM7IEdlbmUgVGhlci4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIzMS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnbG9iYWwgYnV0IHN0YWJsZSBjaGFuZ2UgaW4gSGVMYSBjZWxsIG1vcnBob2xvZ3kgaW5kdWNlcyByZW9yZ2FuaXphdGlvbiBvZiBETkEgc3RydWN0dXJhbCBsb29wIGRvbWFpbnMgd2l0aGluIHRoZSBjZWxsIG51Y2xldXM7IEouIENlbGwgQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz03OS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdHJ1Y3R1cmFsIGJhc2lzIGZvciBjZWxsdWxhciBzZW5lc2NlbmNlOyBBZ2luZzsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NTk4LTYwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWl0b3RpYyByZW1vZGVsbGluZyBvZiB0aGUgcmVwbGljb24gYW5kIGNocm9tb3NvbWUgc3RydWN0dXJlOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTc4Ny04MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0aW1lIGFuZCBkdXJhdGlvbiBvZiBtZWlvc2lzOyBQaGlsb3MuIFRyYW5zLiBSLiBTb2MuIExvbmQuIEIsIEJpb2wuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5Nzc7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9MjAxLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTcvMS8yMy85OTQ1NjUvZHNwMDI3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBOdWNsZWFyIEROQSBvZiBtZXRhem9hbnMgaXMgb3JnYW5pemVkIGluIHN1cGVyY29pbGVkIGxvb3BzIGFuY2hvcmVkIHRvIGEgcHJvdGVpbmFjZW91cyBzdWJzdHJ1Y3R1cmUga25vd24gYXMgdGhlIG51Y2xlYXIgbWF0cml4IChOTSkuIEROQSBpcyBhbmNob3JlZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ3MzY4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZPUyuRDLCi+58k7ElidTxl2C7oTl" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f23%2f347368" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f23%2f347368" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13350" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/1/1/m_cover.gif?Expires=2147483647&amp;Signature=gmtBwQKOCxNij4txyJ-Agd3ex2K7zADLjZgB4yoyDPCuCBxm8rNqvqkhW5k6zWBTEHDsBFHh9yD-hWwwdQR9j5ewB01pfPET55PUMFnQxV39XGIpAiyOJgm-CjrYsQ7Tz~Arn77OBKRlDKReDeozBw~0n-G7uI3SEANuwJK-8WBdyocI7TAaLcJkMyRKHt5P5LZnOwNVGCz8AWDCrjmjcbJFlGWIUbXJLzq4Nh7xOjprYqGF9FDg3nxz1zD3Snjv6oCeeIactT6vD6Ghh7KyKs-edLg7X8Az-CDyI0Hk2p9cS~WPWLe7yUhnUsOYP2HXUj-RQ-PGFInBtBwboH1Esg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/1">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4914815">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914815">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4914817">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914817">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4914820">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914820">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4914836">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914836">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4914876">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914876">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4914880">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914880">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4914885">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4914885">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/1/11/346409">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/1/37/347240">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Determination of the <em>in vivo</em> structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juan Carlos Rivera-Mulia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratorio de Biología Molecular, Facultad de Medicina</div>, <div class="insititution">Universidad Autónoma del Estado de México</div>, <div class="addr-line">Apartado Postal 428, C.P. 50000, Toluca, Edo. Méx.</div>, <div class="country">México</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Programa de Doctorado en Ciencias Biomédicas</div>, <div class="addr-line">IFC-UNAM, Apartado Postal 70-243, C.P. 04510, D.F.</div>, <div class="country">México</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juan+Carlos+Rivera-Mulia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rivera-Mulia J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rivera-Mulia J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Juan Carlos Rivera-Mulia</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Armando Aranda-Anzaldo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratorio de Biología Molecular, Facultad de Medicina</div>, <div class="insititution">Universidad Autónoma del Estado de México</div>, <div class="addr-line">Apartado Postal 428, C.P. 50000, Toluca, Edo. Méx.</div>, <div class="country">México</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Programa de Doctorado en Ciencias Biomédicas</div>, <div class="addr-line">IFC-UNAM, Apartado Postal 70-243, C.P. 04510, D.F.</div>, <div class="country">México</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +52-722-2173552 ext: 222. Fax. <div class="fax">+52-722-2174142</div>. E-mail: <a href="mailto:aaa@uaemex.mx" target="_blank">aaa@uaemex.mx</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Armando+Aranda-Anzaldo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aranda-Anzaldo A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aranda-Anzaldo A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Armando Aranda-Anzaldo</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 1, 1 February 2010, Pages 23–35, <a href='https://doi.org/10.1093/dnares/dsp027'>https://doi.org/10.1093/dnares/dsp027</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 January 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 September 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 December 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347368" href="/dnaresearch/article-pdf/17/1/23/994565/dsp027.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Juan Carlos Rivera-Mulia, Armando Aranda-Anzaldo;  Determination of the <em>in vivo</em> structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach, <em>DNA Research</em>, Volume 17, Issue 1, 1 February 2010, Pages 23–35, <a href='https://doi.org/10.1093/dnares/dsp027'>https://doi.org/10.1093/dnares/dsp027</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347368?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347368?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347368?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347368?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347368?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347368?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347368?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Determination%20of%20the%20%26lt%3Bem%26gt%3Bin%20vivo%26lt%3B%2Fem%26gt%3B%20structural%20DNA%20loop%20organization%20in%20the%20genomic%20region%20of%20the%20rat%20albumin%20locus%20by%20means%20of%20a%20topological%20approach&amp;publicationDate=2010-01-04&amp;volumeNum=17&amp;issueNum=1&amp;author=Rivera-Mulia%2C%20Juan%20Carlos%3B%20Aranda-Anzaldo%2C%20Armando&amp;startPage=23&amp;endPage=35&amp;contentId=10.1093%2Fdnares%2Fdsp027&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f23%2f347368" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4914815 id="4914815" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Nuclear DNA of metazoans is organized in supercoiled loops anchored to a proteinaceous substructure known as the nuclear matrix (NM). DNA is anchored to the NM by non-coding sequences known as matrix attachment regions (MARs). There are no consensus sequences for identification of MARs and not all potential MARs are actually bound to the NM constituting loop attachment regions (LARs). Fundamental processes of nuclear physiology occur at macromolecular complexes organized on the NM; thus, the topological organization of DNA loops must be important. Here, we describe a general method for determining the structural DNA loop organization in any large genomic region with a known sequence. The method exploits the topological properties of loop DNA attached to the NM and elementary topological principles such as that points in a deformable string (DNA) can be positionally mapped relative to a position-reference invariant (NM), and from such mapping, the configuration of the string in third dimension can be deduced. Therefore, it is possible to determine the specific DNA loop configuration without previous characterization of the LARs involved. We determined in hepatocytes and B-lymphocytes of the rat the DNA loop organization of a genomic region that contains four members of the albumin gene family.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA topology&quot;">DNA topology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;loop attachment regions&quot;">loop attachment regions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;matrix attachment regions&quot;">matrix attachment regions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;nuclear matrix&quot;">nuclear matrix</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="nucleotype">nucleotype</a></div>                            <h2 scrollto-destination=4914817 id="4914817" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>In the interphase nucleus of metazoan cells, the DNA is organized in supercoiled loops anchored to a nuclear substructure commonly known as the nuclear matrix (NM) which is a non-soluble complex of ribonucleoproteins obtained after extracting the nucleus with non-ionic detergents, high salt and treatment with DNase.<sup><a reveal-id="DSP027C1" data-open="DSP027C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP027C2" data-open="DSP027C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The exact composition of the NM is a matter of debate as some 400 proteins have been associated with this structure.<sup><a reveal-id="DSP027C3" data-open="DSP027C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The DNA is anchored to the NM by means of non-coding sequences of variable length known as matrix attachment regions (MARs). Yet, there is no consensus sequence for <em>a priori</em> identification of MARs, although they are generally rich in AT and repetitive sequences, and map to regions where the DNA is intrinsically curved or kinked and has a propensity for base unpairing.<sup><a reveal-id="DSP027C4" data-open="DSP027C4" class="link link-ref link-reveal xref-bibr">4</a></sup> MARs are operationally classified into structural-constitutive, resistant to high-salt extraction and functional-facultative, non-resistant to high-salt extraction.<sup><a reveal-id="DSP027C5" data-open="DSP027C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSP027C6" data-open="DSP027C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Therefore, not all potential MARs are actually bound to the NM constituting true loop attachment regions (LARs).<sup><a reveal-id="DSP027C5" data-open="DSP027C5" class="link link-ref link-reveal xref-bibr">5</a></sup> There is evidence that when multiple copies of a specific MAR are present, these are used in a selective fashion, indicating the adaptability of the MAR sequence to serve as anchor only under certain conditions.<sup><a reveal-id="DSP027C7" data-open="DSP027C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The higher-order structure of interphase and metaphase chromosomes is likely to be maintained by constitutive MARs.<sup><a reveal-id="DSP027C4" data-open="DSP027C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSP027C8" data-open="DSP027C8" class="link link-ref link-reveal xref-bibr">8</a></sup> It has been suggested that the dynamic selectivity in the use of MARs as DNA anchors would modulate both the DNA loop average length and the stability of the topological relationships between DNA and the NM during development and cell differentiation.<sup><a reveal-id="DSP027C9" data-open="DSP027C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSP027C10" data-open="DSP027C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Only a very limited number of proteins have been identified that participate in sequence-specific binding of DNA to the NM.<sup><a reveal-id="DSP027C2" data-open="DSP027C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP027C4" data-open="DSP027C4" class="link link-ref link-reveal xref-bibr">4</a></sup> However, saturation experiments indicate the existence of some 150 000 salt-resistant DNA-binding sites per NM in rat hepatocytes,<sup><a reveal-id="DSP027C11" data-open="DSP027C11" class="link link-ref link-reveal xref-bibr">11</a></sup> whereas the average DNA loop size in adult rat hepatocytes is 80–90 kb,<sup><a reveal-id="DSP027C12" data-open="DSP027C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and this figure is compatible with roughly 66 000 DNA loops per rat diploid genome.<sup><a reveal-id="DSP027C13" data-open="DSP027C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Therefore, given that there are no MAR consensus sequences, these facts imply that such DNA–NM interactions are the result of indirect readout effects between DNA and NM proteins and thus not equivalent to the direct readout interactions between transcription factors and specific DNA sequences.<sup><a reveal-id="DSP027C14" data-open="DSP027C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Such sequence-independent DNA–protein interactions depend on DNA shape and overall DNA mechanical properties such as curvature, helical twist and bending and torsional flexibilities,<sup><a reveal-id="DSP027C15" data-open="DSP027C15" class="link link-ref link-reveal xref-bibr">15</a></sup> therefore the local topology (spatial configuration) of the DNA involved must be a determining factor.<sup><a reveal-id="DSP027C16" data-open="DSP027C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Fundamental processes of nuclear physiology such as DNA replication, transcription and processing of primary transcripts occur at macromolecular complexes or factories organized on the NM.<sup><a reveal-id="DSP027C17  DSP027C19" data-open="DSP027C17  DSP027C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> Thus, the topological relationship between the DNA loops and the NM appears to be very important for appropriate nuclear physiology. Indeed, correct repair of DNA damage must include the recovery of both the double helix integrity and the complex three-dimensional DNA topology; otherwise, the cell will not survive.<sup><a reveal-id="DSP027C20" data-open="DSP027C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP027C21" data-open="DSP027C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>Varied approaches have been used for establishing the DNA loop organization in a given region of the genome, based on the experimental<sup><a reveal-id="DSP027C22  DSP027C26" data-open="DSP027C22  DSP027C26" class="link link-ref link-reveal xref-bibr">22–26</a></sup> or the <em>in silico</em> identification of MARs<sup><a reveal-id="DSP027C27" data-open="DSP027C27" class="link link-ref link-reveal xref-bibr">27</a></sup> which has been shown to be quite unreliable in the absence of experimental confirmation,<sup><a reveal-id="DSP027C28" data-open="DSP027C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and more recently on a combination of both approaches coupled to high-throughput analytical techniques.<sup><a reveal-id="DSP027C29" data-open="DSP027C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSP027C30" data-open="DSP027C30" class="link link-ref link-reveal xref-bibr">30</a></sup> All these methods have their specific advantages and pitfalls, but all of them have as priority the identification of the actual MARs (LARs) in a given DNA region so as to infer from such data the possible DNA loop organization. However, considering that DNA is a continuous helical string that becomes looped by binding to the NM, thus establishing a topological higher-order in third-dimensional space, it follows from elementary topological considerations that by mapping the relative positions to the NM of several points (i.e. DNA sequences) along a given string of DNA whose whole sequence is known, it will be possible to establish the spatial configuration of the loops formed by such a DNA string without the need of characterizing the LARs involved. We have previously developed a method for mapping the position of any DNA sequence relative to the NM based on the direct use of PCR on NM-bound templates.<sup><a reveal-id="DSP027C31" data-open="DSP027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> An extension of this method, described hereunder, was used for determining the structural DNA loop organization in a region of 162 kb in chromosome 14 of the rat where four closely related genes that are members of the albumin gene family are located. Such a characterization was done in nucleoids which consist of the NM plus the naked DNA loops attached to it, obtained from freshly isolated hepatocytes and naïve B-lymphocytes from the rat and so the resulting DNA loop organization very likely corresponds to the actual <em>in vivo</em> organization.</p>                            <h2 scrollto-destination=4914820 id="4914820" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4914821 id="4914821" class="section-title" data-legacy-id=s2a>2.1. Animals</h3>
<p>Male Wistar rats weighing 200–250 g were used in accordance with the official Mexican norm for production, care and use of laboratory animals (NOM-062-ZOO-1999).</p>                            <h3 scrollto-destination=4914823 id="4914823" class="section-title" data-legacy-id=s2b>2.2. Primary cells</h3>
<p>Primary rat hepatocytes were obtained from livers of male Wistar rats, using the protocol described previously.<sup><a reveal-id="DSP027C6" data-open="DSP027C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Briefly, the livers were washed <em>in situ</em> by perfusion with PBS without Ca<sup>2+</sup> and Mg<sup>2+</sup> (PBS-A) at 37°C for 5 min at 15 ml/min. Next, the tissue was perfused with a solution of collagenase IV, Sigma (0.025% collagenase with 0.075% CaCl<sub>2</sub> in HEPES buffer, pH 7.6) for 8 min. Viable hepatocytes were counted in a haemocytometer and used immediately for preparing nucleoids (see below). Primary naïve B-lymphocytes were obtained from the spleen of male Wistar rats. Briefly, the spleen was fragmented and filtered in PBS-A. Total lymphocytes were isolated in preformed gradients of isotonic Percoll (25%) by centrifugation at 2100 rpm 15 min at room temperature. Subsequently, naïve B-lymphocytes were purified by magnetic separation (MACS Separation columns, Miltenyi Biotec) with anti-CD43-specific beads (rat CD43 microbeads, Miltenyi Biotec) according to the manufacturer's instructions. Viable B-lymphocytes were counted in a haemocytometer and used immediately for preparing nucleoids.</p>                            <h3 scrollto-destination=4914825 id="4914825" class="section-title" data-legacy-id=s2c>2.3. Preparation of nucleoids</h3>
<p>The DNA loops plus the nuclear substructure constitute a ‘nucleoid’, a very large nucleoprotein aggregate generated by gentle lysis of a cell at pH 8 in non-ionic detergent and the presence of high-salt concentration. Nucleoids were prepared as described previously.<sup><a reveal-id="DSP027C21" data-open="DSP027C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSP027C31" data-open="DSP027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Briefly, freshly isolated and washed hepatocytes and naïve B-lymphocytes are suspended in ice-cold PBS-A, and aliquots of 50 µl containing 3 × 10<sup>5</sup> hepatocytes or 5 × 10<sup>5</sup> lymphocytes are gently mixed with 150 µl of a lysis solution containing 2.6 M NaCl, 1.3 mM EDTA, 2.6 mM Tris, 0.6% Triton-X 100 (pH 8.0). After 20 min for hepatocytes and 15 min for native B-lymphocytes (at 4°C), the mixture is washed in 14 ml of PBS-A at 4°C for 4 min at 3000 rpm (1500 g). The pellet is recovered in a volume ranging from 200 to 300 µl.</p>                            <h3 scrollto-destination=4914827 id="4914827" class="section-title" data-legacy-id=s2d>2.4. DNase I digestion of nucleoid samples</h3>
<p>The washed nucleoids are pooled for setting up the DNase I digestion curves (1.8 × 10<sup>6</sup> nucleoids of hepatocytes or 2.5 × 10<sup>6</sup> nucleoids of naïve B-lymphocytes in 1.2 ml of PBS-A) and mixed with 5 ml of DNase I digestion buffer (10 mM MgCl<sub>2</sub>, 0.1 mM dithiothreitol, 50 mM Tris at pH 7.2). Digestions were carried out at 37°C with 0.5 U/ml DNase I (Sigma). Each digestion time-point aliquot contains 3 × 10<sup>5</sup> nucleoids of hepatocytes or 5 × 10<sup>5</sup> nucleoids of B-lymphocytes. Digestion reactions were stopped by adding 200 µl of stop buffer (final EDTA concentration of 30 mM). The stop buffer contains 0.2 M EDTA and 10 mM Tris at pH 7.5. After digestion with DNase I, the NM-bound DNA was determined by spectrometry on aliquots of partially digested nucleoid samples that were washed and further handled as described previously.<sup><a reveal-id="DSP027C31" data-open="DSP027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The final nucleoid pellet was re-suspended in 200 µl of double-distilled H<sub>2</sub>O to be used directly as a template for PCR.</p>                            <h3 scrollto-destination=4914829 id="4914829" class="section-title" data-legacy-id=s2e>2.5. Genomic DNA primers</h3>
<p>Distinct sets of primers were designed for establishing the topological positions relative to the NM of 15 small DNA sequences located along 162 kb of the genomic region containing four members of the rat albumin gene family. Primer pairs were designed approximately 10 kb each in order to establish rather regular intervals along the region studied. We took into account important considerations in the oligonucleotide design so as to get primers with high specificity but able to perform efficient DNA amplification under the same PCR conditions (reactants concentrations, temperatures, times of each step and number of cycles). It is important to remark that MARs are not suitable targets for high-quality primer design, because their characteristic composition (richness of A-T tracks and repetitive DNA) prevents an acceptable design. Therefore, both potential and actual MARs within the chosen region are likely to be excluded by our primer-design algorithms. All primer sets were designed with a length of 20–25 bp, G-C content between 50% and 55% (with a difference &lt;3%), Tm of 55–60°C (with a difference &lt;2°C), and PCR products of 250–550 bp (Table <a reveal-id="DSP027TB1" data-open="DSP027TB1" class="link link-reveal link-table xref-fig">1</a>). Secondary structures with ΔG &lt; −1 kcal/mol and dimmers/duplexes with ΔG &lt; −2 kcal/mol were avoided. Additionally, the specificity of each primer set was confirmed by the NCBI BLAST algorithm.</p>                        <a id="4914831" scrollto-destination="4914831"></a>
<div content-id="DSP027TB1" class="table-modal"><div class="table-wrap"><div id="DSP027TB1" class="table-wrap-title" data-id="DSP027TB1"><div class="label -label">Table 1</div><div class="caption"><p>Sets of primers and corresponding amplicon sizes defining 15 target sequences along the 162-kb genomic region containing members of the rat albumin gene family</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th>Forward primer </th><th>Reverse primer </th><th>Amplicon length (bp) </th></tr></thead><tbody><tr><td>a </td><td>TGGCAAACATACGCAAGGGA </td><td>GCGAAACACACCCCTGGAAA </td><td>275 </td></tr><tr><td>b </td><td>GAGGACAGTTAGTGCTGTAGGGTTG </td><td>CCTCCAACGAAGTTCCCAGAAT </td><td>547 </td></tr><tr><td>c </td><td>TCCTTTGTAACCAGGCAAGTGG </td><td>CCCATTTCCCAGATCCTTCACTCT </td><td>374 </td></tr><tr><td>d </td><td>CCCAGGGTCAGAGTATATCAGTGC </td><td>CGCTGAACGTATGTCTGAGTCA </td><td>305 </td></tr><tr><td>e </td><td>TGGTAGGCAGAGATGTGAGGAAAG </td><td>CCTGTTGTCCTAATGCTGGTCCTA </td><td>382 </td></tr><tr><td>f </td><td>CTGATCTTCAGGCAATATGGCAGG </td><td>TTGGCTGATGTCGTCTGGACA </td><td>393 </td></tr><tr><td>g </td><td>AAGGATAGGTGCTTGGCTGACA </td><td>GCCCTAACCCTGTGTGTGTATCTTG </td><td>504 </td></tr><tr><td>h </td><td>GATCACGTAACAACCCTGTCAGCT </td><td>CCTTCACAGCACCCGTCATACA </td><td>263 </td></tr><tr><td>i </td><td>GGTGCTGGGAATTTGACTAAGGC </td><td>TAAACTCAGGTGACAGGCTACGGC </td><td>465 </td></tr><tr><td>j </td><td>AGGAACCAGGGAATCGAGTGCT </td><td>AACTTGCGGGTGTTCTCTCCTT </td><td>392 </td></tr><tr><td>k </td><td>TGTCAGCATGATGGTGGCTCA </td><td>CTCGATTTGCCATGTCCTGTCT </td><td>252 </td></tr><tr><td>l </td><td>GGGCTGGGTCCATATTGCTTGA </td><td>ATGCTTTGGGCTTGCCTGAAG </td><td>373 </td></tr><tr><td>m </td><td>ACGACTTCCCTTCCTATCCACAG </td><td>GTAGAAAGTCGTTCTGGTTGCCAC </td><td>234 </td></tr><tr><td>n </td><td>CCCTAATCTTGCTGTGGTTTGG </td><td>TGAGAGCTGGGCAGACATCAA </td><td>355 </td></tr><tr><td>o </td><td>GGTGACAGTTGACAGAGAGCCTTC </td><td>GCTCCATGCTGACCTTGAAGTCTA </td><td>272 </td></tr></tbody></table></div><div content-id="DSP027TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th>Forward primer </th><th>Reverse primer </th><th>Amplicon length (bp) </th></tr></thead><tbody><tr><td>a </td><td>TGGCAAACATACGCAAGGGA </td><td>GCGAAACACACCCCTGGAAA </td><td>275 </td></tr><tr><td>b </td><td>GAGGACAGTTAGTGCTGTAGGGTTG </td><td>CCTCCAACGAAGTTCCCAGAAT </td><td>547 </td></tr><tr><td>c </td><td>TCCTTTGTAACCAGGCAAGTGG </td><td>CCCATTTCCCAGATCCTTCACTCT </td><td>374 </td></tr><tr><td>d </td><td>CCCAGGGTCAGAGTATATCAGTGC </td><td>CGCTGAACGTATGTCTGAGTCA </td><td>305 </td></tr><tr><td>e </td><td>TGGTAGGCAGAGATGTGAGGAAAG </td><td>CCTGTTGTCCTAATGCTGGTCCTA </td><td>382 </td></tr><tr><td>f </td><td>CTGATCTTCAGGCAATATGGCAGG </td><td>TTGGCTGATGTCGTCTGGACA </td><td>393 </td></tr><tr><td>g </td><td>AAGGATAGGTGCTTGGCTGACA </td><td>GCCCTAACCCTGTGTGTGTATCTTG </td><td>504 </td></tr><tr><td>h </td><td>GATCACGTAACAACCCTGTCAGCT </td><td>CCTTCACAGCACCCGTCATACA </td><td>263 </td></tr><tr><td>i </td><td>GGTGCTGGGAATTTGACTAAGGC </td><td>TAAACTCAGGTGACAGGCTACGGC </td><td>465 </td></tr><tr><td>j </td><td>AGGAACCAGGGAATCGAGTGCT </td><td>AACTTGCGGGTGTTCTCTCCTT </td><td>392 </td></tr><tr><td>k </td><td>TGTCAGCATGATGGTGGCTCA </td><td>CTCGATTTGCCATGTCCTGTCT </td><td>252 </td></tr><tr><td>l </td><td>GGGCTGGGTCCATATTGCTTGA </td><td>ATGCTTTGGGCTTGCCTGAAG </td><td>373 </td></tr><tr><td>m </td><td>ACGACTTCCCTTCCTATCCACAG </td><td>GTAGAAAGTCGTTCTGGTTGCCAC </td><td>234 </td></tr><tr><td>n </td><td>CCCTAATCTTGCTGTGGTTTGG </td><td>TGAGAGCTGGGCAGACATCAA </td><td>355 </td></tr><tr><td>o </td><td>GGTGACAGTTGACAGAGAGCCTTC </td><td>GCTCCATGCTGACCTTGAAGTCTA </td><td>272 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914831" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP027TB1" class="table-wrap-title" data-id="DSP027TB1"><div class="label -label">Table 1</div><div class="caption"><p>Sets of primers and corresponding amplicon sizes defining 15 target sequences along the 162-kb genomic region containing members of the rat albumin gene family</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th>Forward primer </th><th>Reverse primer </th><th>Amplicon length (bp) </th></tr></thead><tbody><tr><td>a </td><td>TGGCAAACATACGCAAGGGA </td><td>GCGAAACACACCCCTGGAAA </td><td>275 </td></tr><tr><td>b </td><td>GAGGACAGTTAGTGCTGTAGGGTTG </td><td>CCTCCAACGAAGTTCCCAGAAT </td><td>547 </td></tr><tr><td>c </td><td>TCCTTTGTAACCAGGCAAGTGG </td><td>CCCATTTCCCAGATCCTTCACTCT </td><td>374 </td></tr><tr><td>d </td><td>CCCAGGGTCAGAGTATATCAGTGC </td><td>CGCTGAACGTATGTCTGAGTCA </td><td>305 </td></tr><tr><td>e </td><td>TGGTAGGCAGAGATGTGAGGAAAG </td><td>CCTGTTGTCCTAATGCTGGTCCTA </td><td>382 </td></tr><tr><td>f </td><td>CTGATCTTCAGGCAATATGGCAGG </td><td>TTGGCTGATGTCGTCTGGACA </td><td>393 </td></tr><tr><td>g </td><td>AAGGATAGGTGCTTGGCTGACA </td><td>GCCCTAACCCTGTGTGTGTATCTTG </td><td>504 </td></tr><tr><td>h </td><td>GATCACGTAACAACCCTGTCAGCT </td><td>CCTTCACAGCACCCGTCATACA </td><td>263 </td></tr><tr><td>i </td><td>GGTGCTGGGAATTTGACTAAGGC </td><td>TAAACTCAGGTGACAGGCTACGGC </td><td>465 </td></tr><tr><td>j </td><td>AGGAACCAGGGAATCGAGTGCT </td><td>AACTTGCGGGTGTTCTCTCCTT </td><td>392 </td></tr><tr><td>k </td><td>TGTCAGCATGATGGTGGCTCA </td><td>CTCGATTTGCCATGTCCTGTCT </td><td>252 </td></tr><tr><td>l </td><td>GGGCTGGGTCCATATTGCTTGA </td><td>ATGCTTTGGGCTTGCCTGAAG </td><td>373 </td></tr><tr><td>m </td><td>ACGACTTCCCTTCCTATCCACAG </td><td>GTAGAAAGTCGTTCTGGTTGCCAC </td><td>234 </td></tr><tr><td>n </td><td>CCCTAATCTTGCTGTGGTTTGG </td><td>TGAGAGCTGGGCAGACATCAA </td><td>355 </td></tr><tr><td>o </td><td>GGTGACAGTTGACAGAGAGCCTTC </td><td>GCTCCATGCTGACCTTGAAGTCTA </td><td>272 </td></tr></tbody></table></div><div content-id="DSP027TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th>Forward primer </th><th>Reverse primer </th><th>Amplicon length (bp) </th></tr></thead><tbody><tr><td>a </td><td>TGGCAAACATACGCAAGGGA </td><td>GCGAAACACACCCCTGGAAA </td><td>275 </td></tr><tr><td>b </td><td>GAGGACAGTTAGTGCTGTAGGGTTG </td><td>CCTCCAACGAAGTTCCCAGAAT </td><td>547 </td></tr><tr><td>c </td><td>TCCTTTGTAACCAGGCAAGTGG </td><td>CCCATTTCCCAGATCCTTCACTCT </td><td>374 </td></tr><tr><td>d </td><td>CCCAGGGTCAGAGTATATCAGTGC </td><td>CGCTGAACGTATGTCTGAGTCA </td><td>305 </td></tr><tr><td>e </td><td>TGGTAGGCAGAGATGTGAGGAAAG </td><td>CCTGTTGTCCTAATGCTGGTCCTA </td><td>382 </td></tr><tr><td>f </td><td>CTGATCTTCAGGCAATATGGCAGG </td><td>TTGGCTGATGTCGTCTGGACA </td><td>393 </td></tr><tr><td>g </td><td>AAGGATAGGTGCTTGGCTGACA </td><td>GCCCTAACCCTGTGTGTGTATCTTG </td><td>504 </td></tr><tr><td>h </td><td>GATCACGTAACAACCCTGTCAGCT </td><td>CCTTCACAGCACCCGTCATACA </td><td>263 </td></tr><tr><td>i </td><td>GGTGCTGGGAATTTGACTAAGGC </td><td>TAAACTCAGGTGACAGGCTACGGC </td><td>465 </td></tr><tr><td>j </td><td>AGGAACCAGGGAATCGAGTGCT </td><td>AACTTGCGGGTGTTCTCTCCTT </td><td>392 </td></tr><tr><td>k </td><td>TGTCAGCATGATGGTGGCTCA </td><td>CTCGATTTGCCATGTCCTGTCT </td><td>252 </td></tr><tr><td>l </td><td>GGGCTGGGTCCATATTGCTTGA </td><td>ATGCTTTGGGCTTGCCTGAAG </td><td>373 </td></tr><tr><td>m </td><td>ACGACTTCCCTTCCTATCCACAG </td><td>GTAGAAAGTCGTTCTGGTTGCCAC </td><td>234 </td></tr><tr><td>n </td><td>CCCTAATCTTGCTGTGGTTTGG </td><td>TGAGAGCTGGGCAGACATCAA </td><td>355 </td></tr><tr><td>o </td><td>GGTGACAGTTGACAGAGAGCCTTC </td><td>GCTCCATGCTGACCTTGAAGTCTA </td><td>272 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914831" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4914832 id="4914832" class="section-title" data-legacy-id=s2f>2.6. PCR amplification</h3>
<p>Fifty nanograms of NM-bound DNA was used as a template for PCR. PCR was carried out using 0.7 U GoTaq DNA polymerase (Promega), 2.5 mM MgCl<sub>2</sub>, 0.2 mM of each dNTP and 0.1 µM of each primer. Amplification was performed in an Applied Biosystems 2720 thermocycler and the same amplification program was used for all pairs of primers: initial denaturising step at 94°C for 5 min, denaturising step at 94°C for 45 s, annealing at 56°C for 30 s and extension at 72°C for 1 min for 35 cycles, with a final extension at 72°C for 10 min. The identity of all the amplicons was confirmed by restriction analysis with the appropriate restriction enzymes. Amplicons were electrophoresed on 2% agarose gels and visualized using ethidium bromide staining (0.5 µl/ml), recorded and analyzed using a Kodak 1D Image Analysis Software 3.5 system. Amplicons were scored as positive or negative on partially digested nucleoid samples, depending on whether they are detectable by the software using the default settings.</p>                            <h3 scrollto-destination=4914834 id="4914834" class="section-title" data-legacy-id=s2g>2.7. Computer-aided prediction of MAR sequences</h3>
<p>The potential MARs in the region studied were identified by <em>in silico</em> prediction using four different freely available software: MarFinder,<sup><a reveal-id="DSP027C32" data-open="DSP027C32" class="link link-ref link-reveal xref-bibr">32</a></sup> MARScan,<sup><a reveal-id="DSP027C33" data-open="DSP027C33" class="link link-ref link-reveal xref-bibr">33</a></sup> ChrClass<sup><a reveal-id="DSP027C34" data-open="DSP027C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and SMARtest.<sup><a reveal-id="DSP027C27" data-open="DSP027C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h2 scrollto-destination=4914836 id="4914836" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4914837 id="4914837" class="section-title" data-legacy-id=s3a>3.1. Rationale of the method</h3>
<p>Our method exploits the topological properties of the average DNA loop that result from the fact that such loops are topologically constrained by being anchored to the NM, thus being equivalent to closed DNA circles. Under such a condition, the DNA molecule undergoes structural stress resulting from two factors: the covalently linked backbones of the DNA strands are helicoidal but rigid, and the low-energy hydrogen bonds between the stacked bases are quasi-statistical unions that continuously break apart and form again; such a situation poses the risk that the nucleotide bases may gyrate away from the double-helix axis and become exposed. DNA naturally solves this structural-stress problem by further coiling upon its own axis, thus avoiding the exposure of the nucleotide bases, but becoming negatively supercoiled in a similar fashion to a pulled house-telephone cord.<sup><a reveal-id="DSP027C35" data-open="DSP027C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSP027C36" data-open="DSP027C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Thus, the naked DNA loops display a gradient of supercoiling that goes from lower to higher from tip to base of the loop,<sup><a reveal-id="DSP027C36" data-open="DSP027C36" class="link link-ref link-reveal xref-bibr">36</a></sup> save for the fact that the structural properties of MARs are such that they also function as buffers or sinks of negative supercoiling<sup><a reveal-id="DSP027C37" data-open="DSP027C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSP027C38" data-open="DSP027C38" class="link link-ref link-reveal xref-bibr">38</a></sup> thus avoiding maximal supercoiling at the base of the loops. The NM plus the naked DNA loops anchored to it constitute a nucleoid. Under the conditions of lysis employed to generate nucleoids, the DNA remains essentially intact, although it lacks the nucleosome structure because of the dissociation of histones and most other nuclear proteins usually associated with DNA; yet, the DNA loops remain topologically constrained and supercoiled as depicted in Fig. <a reveal-id="DSP027F1" data-open="DSP027F1" class="link link-reveal link-table xref-fig">1</a>. Indeed, nucleoids are also known as nuclear halos since the exposure of such structures to DNA-intercalating agents like ethidium bromide leads to unwinding of the DNA loops that form a DNA halo around the NM periphery (Fig. <a reveal-id="DSP027F1" data-open="DSP027F1" class="link link-reveal link-table xref-fig">1</a>C). A typical DNA loop can be divided into four topological zones according to their relative proximity to the NM. Each of these zones would manifest an identifiable behaviour when exposed to non-specific nucleases that are sensitive to the local DNA topology (Fig. <a reveal-id="DSP027F1" data-open="DSP027F1" class="link link-reveal link-table xref-fig">1</a>A). We have previously shown that in nucleoid preparations, the relative resistance of a given loop-DNA sequence to a limited concentration of DNase I is directly proportional to its proximity to the NM anchoring point.<sup><a reveal-id="DSP027C31" data-open="DSP027C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSP027C39" data-open="DSP027C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Two main factors determine this property. (i) Steric hindrance resulting from the proteinaceous NM that acts as a physical barrier relatively protecting the naked loop DNA that is closer to the NM from endonuclease action. (ii) The local degree of loop DNA supercoiling that is lower in the distal portions of the loop and higher in the regions proximal to the NM. Supercoiling is a structural barrier against the action of non-specific endonucleases, such as DNase I, that hydrolyze the DNA backbone by a single-strand cleavage (nicking) mechanism.<sup><a reveal-id="DSP027C40" data-open="DSP027C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Both factors only confer relative but not absolute DNase I resistance to loop DNA. However, in a large sample of nucleoids exposed to a limited concentration of DNase I, there is a consistent trend in which the loop-DNA sensitivity to the enzyme is inversely proportional to its distance relative to the NM and so distal regions of the loop are digested first whereas the regions closer to the NM are digested later. Indeed, it is known that the DNA embedded within the NM is very resistant to DNase I action, and there is a fraction corresponding to some 2% the total DNA that is basically non-digestible even when exposed to high concentrations of the enzyme. This fraction corresponds to fragments with an average length of 1.6 kb in rat hepatocytes,<sup><a reveal-id="DSP027C12" data-open="DSP027C12" class="link link-ref link-reveal xref-bibr">12</a></sup> likely to represent the regions that include the actual MARs (LARs) anchored to the NM. This pattern of sensitivity to DNase I holds provided that the DNA is basically devoid of histones and most other proteins that form chromatin. Indeed, whole chromatin attached to the NM shows an inverse pattern of nuclease sensitivity compared with that of naked loop DNA. In looped chromatin, those sequences closer to the matrix attachment point are preferentially cleaved by nucleases.<sup><a reveal-id="DSP027C41" data-open="DSP027C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p>                        <a id="4914839" scrollto-destination="4914839"></a>
<div id="DSP027F1" data-id="DSP027F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02701.jpeg?Expires=2147483647&Signature=OjqNiS2eW9oGSMzhvYxJV8lKqkUMkx1SWuYP3nfN6GxSFYjFNyH8yBbwO0ez9HadH6ToRHDug3LJcYQgNj11UPfFPNXGT91NQzsN4tu0e5aiHxzLr3MPA75LfEf6LdPVpoOn3h1gS7IviDk2Zt4BZOoTyrIZ6TCSD8oaFhd1eRHozmYbBonOnjhwJ5GAZ6hFXitt2MvugC80DpfjSjBFOkNs8-rEEG-eZ5pGeVKLS-ppEfz7Ac62cWJE96N0h1QJWSeQBe0K5200T5XcXUSt-LgJwvDeKmam2SPQ91bslNaZ~G9aJ2VQJXzE7YRKY-Iqv8FLAbmT3qHjbuGFAHUqhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Properties of naked DNA loops attached to the NM. (A) Drawing illustrating the local topology along a typical supercoiled DNA loop that correlates with both distance relative to the NM and sensitivity to DNase I. (B) Phase contrast micrograph showing the NM of a hepatocyte nucleoid. (C) Fluorescence micrograph showing the DNA halo around the NM of a hepatocyte nucleoid caused by the unwinding of the supercoiled DNA loops by treatment of the nucleoid with the DNA-intercalating agent ethidium bromide (80 µg/ml). Scale bars 10 µm." path-from-xml="dsp02701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914839/dsp02701.jpeg" path-from-xml="dsp02701.jpeg" target="_blank">View large</a><a section="4914839" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02701.jpeg?Expires=2147483647&Signature=nZWEBXYxErfP9M26Mab4sy-9~rtSC1eiQlk0wUQ-u5DrFKd7YpBTuhcK9vmw5nlrqa6FjrV7sGHc0Qh5eaAOSRkt~uFD0yEHv3iyKqdLm35f1Tx7Qv6qqBGTTDnv0cNYTpWj-vdvNLv3UGiLOxzFcCv6xE~nTldnirM2NKOVAI9xxSXhJcAAEgThIdt99yHtBGj~QOeTnAlxI34GRMZYNaR0rFxNeLDzTEbSIXAiWCSCWgmpCHnn16LP-zxQPxhU2qxxck3pJQLB14On4Tx2gZ2~vzdHnAGGUl4ImlMIi-kV1isa7o7D9sbkwK332VThg4wxaKWe6NjqmfWTe2Jw6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914839&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Properties of naked DNA loops attached to the NM. (<strong>A</strong>) Drawing illustrating the local topology along a typical supercoiled DNA loop that correlates with both distance relative to the NM and sensitivity to DNase I. (<strong>B</strong>) Phase contrast micrograph showing the NM of a hepatocyte nucleoid. (<strong>C</strong>) Fluorescence micrograph showing the DNA halo around the NM of a hepatocyte nucleoid caused by the unwinding of the supercoiled DNA loops by treatment of the nucleoid with the DNA-intercalating agent ethidium bromide (80 µg/ml). Scale bars 10 µm.</p></div></div><div content-id="DSP027F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02701.jpeg?Expires=2147483647&Signature=OjqNiS2eW9oGSMzhvYxJV8lKqkUMkx1SWuYP3nfN6GxSFYjFNyH8yBbwO0ez9HadH6ToRHDug3LJcYQgNj11UPfFPNXGT91NQzsN4tu0e5aiHxzLr3MPA75LfEf6LdPVpoOn3h1gS7IviDk2Zt4BZOoTyrIZ6TCSD8oaFhd1eRHozmYbBonOnjhwJ5GAZ6hFXitt2MvugC80DpfjSjBFOkNs8-rEEG-eZ5pGeVKLS-ppEfz7Ac62cWJE96N0h1QJWSeQBe0K5200T5XcXUSt-LgJwvDeKmam2SPQ91bslNaZ~G9aJ2VQJXzE7YRKY-Iqv8FLAbmT3qHjbuGFAHUqhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Properties of naked DNA loops attached to the NM. (A) Drawing illustrating the local topology along a typical supercoiled DNA loop that correlates with both distance relative to the NM and sensitivity to DNase I. (B) Phase contrast micrograph showing the NM of a hepatocyte nucleoid. (C) Fluorescence micrograph showing the DNA halo around the NM of a hepatocyte nucleoid caused by the unwinding of the supercoiled DNA loops by treatment of the nucleoid with the DNA-intercalating agent ethidium bromide (80 µg/ml). Scale bars 10 µm." path-from-xml="dsp02701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914839/dsp02701.jpeg" path-from-xml="dsp02701.jpeg" target="_blank">View large</a><a section="4914839" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02701.jpeg?Expires=2147483647&Signature=nZWEBXYxErfP9M26Mab4sy-9~rtSC1eiQlk0wUQ-u5DrFKd7YpBTuhcK9vmw5nlrqa6FjrV7sGHc0Qh5eaAOSRkt~uFD0yEHv3iyKqdLm35f1Tx7Qv6qqBGTTDnv0cNYTpWj-vdvNLv3UGiLOxzFcCv6xE~nTldnirM2NKOVAI9xxSXhJcAAEgThIdt99yHtBGj~QOeTnAlxI34GRMZYNaR0rFxNeLDzTEbSIXAiWCSCWgmpCHnn16LP-zxQPxhU2qxxck3pJQLB14On4Tx2gZ2~vzdHnAGGUl4ImlMIi-kV1isa7o7D9sbkwK332VThg4wxaKWe6NjqmfWTe2Jw6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914839&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Properties of naked DNA loops attached to the NM. (<strong>A</strong>) Drawing illustrating the local topology along a typical supercoiled DNA loop that correlates with both distance relative to the NM and sensitivity to DNase I. (<strong>B</strong>) Phase contrast micrograph showing the NM of a hepatocyte nucleoid. (<strong>C</strong>) Fluorescence micrograph showing the DNA halo around the NM of a hepatocyte nucleoid caused by the unwinding of the supercoiled DNA loops by treatment of the nucleoid with the DNA-intercalating agent ethidium bromide (80 µg/ml). Scale bars 10 µm.</p></div></div>                            <h3 scrollto-destination=4914840 id="4914840" class="section-title" data-legacy-id=s3b>3.2. Differential kinetics of loop DNA digestion as a function of proximity to the NM</h3>
<p>Treatment of nucleoids from quiescent, freshly isolated rat hepatocytes, with a limited concentration of DNase I (0.5 U/ml) produces a highly reproducible kinetics of digestion of loop DNA (Fig. <a reveal-id="DSP027F2" data-open="DSP027F2" class="link link-reveal link-table xref-fig">2</a>A). It is possible to identify three different phases in the digestion curve: the first corresponds to a very fast kinetics of digestion that removes almost 60% the total DNA associated with the NM within the first 5 min. Such a DNA corresponds to the loop fraction distal to the NM; in such a fraction, relatively minor DNA supercoiling is the only barrier to the endonuclease action. Moreover, as the DNase I nicks the loop DNA, each nick becomes a point of further DNA unwinding, thus increasing the reduction in loop supercoiling in time and so making more accessible the loop DNA to the action of the endonuclease. The second phase that lasts ∼10 min shows a reduced kinetics of digestion in which some further 10% loop DNA is removed. This slower kinetics is the result of the slower rate of supercoiling loss in the loop DNA proximal to the NM (originally highly supercoiled) that slows down the action of the enzyme. The third and slowest phase, lasting 45 min, shows the removal of some further 10% total DNA associated with the NM. This very slow kinetics results from two factors: the effect of any residual DNA supercoiling and principally of the proximity between the loop DNA and the NM proteins that act as physical barriers against the action of DNase I on the loop DNA that is very close to the NM. Finally, there is about 20% total DNA that remains bound to the NM even after 60 min of treatment with DNase I. This fraction corresponds to the DNA that is actually embedded within the NM, and so it is rather inaccessible to the limited concentration of DNase I used.</p>                        <a id="4914842" scrollto-destination="4914842"></a>
<div id="DSP027F2" data-id="DSP027F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02702.jpeg?Expires=2147483647&Signature=L5jYHqF5437yAIhkLgAhnRCNYyhtt3WHA65BmvLqI0oLecS85Y2twACT7Yu0WxYh~W4N-usdhFJxeUOpicvppU6TMvvbC5KBmSNEJAJIbUwVaQuCtCALhDw~Z1Xx4vn89M-SluYxhz74oJffr52qNoDA9ILQheMoOA0o624ecrJxIX39E0~kFwJ8VbcWRdRuFPvjDKcFBQEWqLlKzT0b1iKcjexQFMmmaUoXYlMRuuOPrCCiF6j~PIalfzXypHMZ~071ywZNUSjvgHXlYm7F7Eysu0VKDfFJ48mR6JYsVbN~zVuRqOr2fKGas63qrwboT0Mgn0M3wVM7eiCzUUu-Hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Kinetics of nucleoid DNA digestion. Nucleoids were digested with DNase-I (0.5 U/ml). Each time-point value is the average of separate experiments with samples obtained from separate animals (n = 9 for hepatocytes, n = 4 for naïve B-lymphocytes). Bars indicate the corresponding SD. The topological zones relative to the NM correspond to decreasing percentages of total DNA bound to the NM. (A) For hepatocytes: distal to NM (100–42% total DNA), proximal to NM (42–30% total DNA), very close to NM (30–19% total DNA) and embedded within NM (19–0% total DNA). The corresponding slopes are the following: 0–5 min = −9.80; 5–15 min = −1.70; 15–30 min = −0.47 and 30–60 min = −0.17. Hence, the slope of the digestion curve became close to zero by 60 min of DNase I treatment and remained like that even after 120 min of incubation with the enzyme (slope 60–120 min = −0.17). (B) For naïve B-lymphocytes: distal to NM (100–33% total DNA), proximal to NM (33–7% total DNA), very close to NM (7–4% total DNA) and embedded within NM (4–0% total DNA). The corresponding slopes are the following: 0–5 min = −11.46; 5–15 min = −2.84; 15–30 min = −0.61 and 30–60 min = −0.06 (slope 60–120 min = −0.03). Hence, the slope of the digestion curve also became close to zero by 60 min of DNase I treatment." path-from-xml="dsp02702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914842/dsp02702.jpeg" path-from-xml="dsp02702.jpeg" target="_blank">View large</a><a section="4914842" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02702.jpeg?Expires=2147483647&Signature=cmjRibradVoZ3n-3i0hCFVv6w-47Szdc5MAcaPRmhy0y084LoqhDO1Pq-i-I-74J3bBZXEGHCDbNMxUCXXVbLBN1y7PlBLtZ2RLizmEzpE9zQ~U0SeeqY4JJqfrrOPCueEmNmJFPsL9dLK9qyRhpV~Gd5ahKsvg-z8DPL~OchwjRdtfXZwi8SAGoN7ONWLzQ0N5fQbVN3klBYMeLmvnTDZLDDvwEP6-LpuhPCpLJnwp6p0puz1ZVZX0jMdtsiEsHjqsD~npxA85yAhmLJ5lCQpb8kKNZq57I9mklkcrQgmIAlBlM5Zj9b98qhHRKOJ7MAKmwpaBmYV~f~nHbmiVxYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914842&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Kinetics of nucleoid DNA digestion. Nucleoids were digested with DNase-I (0.5 U/ml). Each time-point value is the average of separate experiments with samples obtained from separate animals (<em>n</em> = 9 for hepatocytes, <em>n</em> = 4 for naïve B-lymphocytes). Bars indicate the corresponding SD. The topological zones relative to the NM correspond to decreasing percentages of total DNA bound to the NM. (<strong>A</strong>) For hepatocytes: distal to NM (100–42% total DNA), proximal to NM (42–30% total DNA), very close to NM (30–19% total DNA) and embedded within NM (19–0% total DNA). The corresponding slopes are the following: 0–5 min = −9.80; 5–15 min = −1.70; 15–30 min = −0.47 and 30–60 min = −0.17. Hence, the slope of the digestion curve became close to zero by 60 min of DNase I treatment and remained like that even after 120 min of incubation with the enzyme (slope 60–120 min = −0.17). (<strong>B</strong>) For naïve B-lymphocytes: distal to NM (100–33% total DNA), proximal to NM (33–7% total DNA), very close to NM (7–4% total DNA) and embedded within NM (4–0% total DNA). The corresponding slopes are the following: 0–5 min = −11.46; 5–15 min = −2.84; 15–30 min = −0.61 and 30–60 min = −0.06 (slope 60–120 min = −0.03). Hence, the slope of the digestion curve also became close to zero by 60 min of DNase I treatment.</p></div></div><div content-id="DSP027F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02702.jpeg?Expires=2147483647&Signature=L5jYHqF5437yAIhkLgAhnRCNYyhtt3WHA65BmvLqI0oLecS85Y2twACT7Yu0WxYh~W4N-usdhFJxeUOpicvppU6TMvvbC5KBmSNEJAJIbUwVaQuCtCALhDw~Z1Xx4vn89M-SluYxhz74oJffr52qNoDA9ILQheMoOA0o624ecrJxIX39E0~kFwJ8VbcWRdRuFPvjDKcFBQEWqLlKzT0b1iKcjexQFMmmaUoXYlMRuuOPrCCiF6j~PIalfzXypHMZ~071ywZNUSjvgHXlYm7F7Eysu0VKDfFJ48mR6JYsVbN~zVuRqOr2fKGas63qrwboT0Mgn0M3wVM7eiCzUUu-Hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Kinetics of nucleoid DNA digestion. Nucleoids were digested with DNase-I (0.5 U/ml). Each time-point value is the average of separate experiments with samples obtained from separate animals (n = 9 for hepatocytes, n = 4 for naïve B-lymphocytes). Bars indicate the corresponding SD. The topological zones relative to the NM correspond to decreasing percentages of total DNA bound to the NM. (A) For hepatocytes: distal to NM (100–42% total DNA), proximal to NM (42–30% total DNA), very close to NM (30–19% total DNA) and embedded within NM (19–0% total DNA). The corresponding slopes are the following: 0–5 min = −9.80; 5–15 min = −1.70; 15–30 min = −0.47 and 30–60 min = −0.17. Hence, the slope of the digestion curve became close to zero by 60 min of DNase I treatment and remained like that even after 120 min of incubation with the enzyme (slope 60–120 min = −0.17). (B) For naïve B-lymphocytes: distal to NM (100–33% total DNA), proximal to NM (33–7% total DNA), very close to NM (7–4% total DNA) and embedded within NM (4–0% total DNA). The corresponding slopes are the following: 0–5 min = −11.46; 5–15 min = −2.84; 15–30 min = −0.61 and 30–60 min = −0.06 (slope 60–120 min = −0.03). Hence, the slope of the digestion curve also became close to zero by 60 min of DNase I treatment." path-from-xml="dsp02702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914842/dsp02702.jpeg" path-from-xml="dsp02702.jpeg" target="_blank">View large</a><a section="4914842" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02702.jpeg?Expires=2147483647&Signature=cmjRibradVoZ3n-3i0hCFVv6w-47Szdc5MAcaPRmhy0y084LoqhDO1Pq-i-I-74J3bBZXEGHCDbNMxUCXXVbLBN1y7PlBLtZ2RLizmEzpE9zQ~U0SeeqY4JJqfrrOPCueEmNmJFPsL9dLK9qyRhpV~Gd5ahKsvg-z8DPL~OchwjRdtfXZwi8SAGoN7ONWLzQ0N5fQbVN3klBYMeLmvnTDZLDDvwEP6-LpuhPCpLJnwp6p0puz1ZVZX0jMdtsiEsHjqsD~npxA85yAhmLJ5lCQpb8kKNZq57I9mklkcrQgmIAlBlM5Zj9b98qhHRKOJ7MAKmwpaBmYV~f~nHbmiVxYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914842&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Kinetics of nucleoid DNA digestion. Nucleoids were digested with DNase-I (0.5 U/ml). Each time-point value is the average of separate experiments with samples obtained from separate animals (<em>n</em> = 9 for hepatocytes, <em>n</em> = 4 for naïve B-lymphocytes). Bars indicate the corresponding SD. The topological zones relative to the NM correspond to decreasing percentages of total DNA bound to the NM. (<strong>A</strong>) For hepatocytes: distal to NM (100–42% total DNA), proximal to NM (42–30% total DNA), very close to NM (30–19% total DNA) and embedded within NM (19–0% total DNA). The corresponding slopes are the following: 0–5 min = −9.80; 5–15 min = −1.70; 15–30 min = −0.47 and 30–60 min = −0.17. Hence, the slope of the digestion curve became close to zero by 60 min of DNase I treatment and remained like that even after 120 min of incubation with the enzyme (slope 60–120 min = −0.17). (<strong>B</strong>) For naïve B-lymphocytes: distal to NM (100–33% total DNA), proximal to NM (33–7% total DNA), very close to NM (7–4% total DNA) and embedded within NM (4–0% total DNA). The corresponding slopes are the following: 0–5 min = −11.46; 5–15 min = −2.84; 15–30 min = −0.61 and 30–60 min = −0.06 (slope 60–120 min = −0.03). Hence, the slope of the digestion curve also became close to zero by 60 min of DNase I treatment.</p></div></div><p>Nucleoids from naïve B-lymphocyte treated with the same limited concentration of DNase I display a faster kinetics of loop-DNA digestion, and so after 60 min of treatment with the enzyme, the amount of total DNA embedded within the NM is very small (≤4.0%), yet the digestion of further loop DNA stalls and becomes negligible afterwards (Fig. <a reveal-id="DSP027F2" data-open="DSP027F2" class="link link-reveal link-table xref-fig">2</a>B). The average DNA loop size in naïve B-lymphocytes is rather similar to that in hepatocytes, yet the DNA–NM interactions are more fragile in B-lymphocytes when compared with hepatocytes, and comparative PAGE analysis shows that the relative amount of typical NM proteins is significantly reduced in naïve B-lymphocytes, in particular the three nuclear lamins (A, B and C) that are major components of standard NM preparations (our unpublished results). Hence, the presence of a less dense NM in the nucleoids from naïve B-lymphocytes explains why the fraction of total DNA that is relatively protected from DNase I action is significantly smaller than that in nucleoids from hepatocytes. Depending on the cell type of the nucleoids studied, preliminary experiments must be carried out so as to define the concentration of DNase I suitable to produce consistent tri-phasic digestion curves (Fig. <a reveal-id="DSP027F2" data-open="DSP027F2" class="link link-reveal link-table xref-fig">2</a>A and B). On the basis of our experience working with several cell lines<sup><a reveal-id="DSP027C31" data-open="DSP027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and different types of primary cells, the range of DNase I concentrations to be considered is between 1.0 and 0.05 U/ml. Indeed, extensive damage to DNA due to incorrect nucleoid preparation and handling, or the natural presence of nicked DNA in some terminally differentiated cell types (such as non-naïve, activated lymphocytes) produce bi-phasic nucleoid-DNA digestion curves since the loop-DNA supercoiling is either absent or severely reduced. Nevertheless, in all nucleoid preparations treated with a limited amount of DNase I, there is a fraction of total DNA that is resistant to the enzyme by being embedded within the NM, such a DNA is easily identifiable in the digestion curves as the fraction that is digested very slowly so that the local slope becomes close to zero and remains like that even after very long incubation times (Fig. <a reveal-id="DSP027F2" data-open="DSP027F2" class="link link-reveal link-table xref-fig">2</a>A and B).</p>                            <h3 scrollto-destination=4914844 id="4914844" class="section-title" data-legacy-id=s3c>3.3. Mapping the relative position to the NM of DNA sequences along the genomic region containing the rat albumin locus</h3>
<p>On the basis of the above considerations and results, we used the general strategy depicted in Fig. <a reveal-id="DSP027F3" data-open="DSP027F3" class="link link-reveal link-table xref-fig">3</a> in order to map the position relative to the NM of 15 small DNA sequences located along a region corresponding to 162 kb of chromosome 14 in the rat, which includes the loci of the following gene members of the albumin gene family: albumin (<em>Alb</em>), alpha-fetoprotein (<em>Afp</em>), afamin (<em>Afm</em>) and the pseudo-gene similar to <em>Afp</em> (<em>Afp-L</em>). Such small DNA sequences were spaced some 10 kb apart (Fig. <a reveal-id="DSP027F4" data-open="DSP027F4" class="link link-reveal link-table xref-fig">4</a>). Therefore, nucleoids prepared from freshly isolated hepatocytes or naïve B-lymphocytes (Fig. <a reveal-id="DSP027F3" data-open="DSP027F3" class="link link-reveal link-table xref-fig">3</a>A) were incubated with DNase I (0.5 U/ml) for different times so as to obtain nucleoid samples with differential amounts of DNA associated with the NM (Fig. <a reveal-id="DSP027F3" data-open="DSP027F3" class="link link-reveal link-table xref-fig">3</a>B and C), and such samples with partially digested NM-bound DNA were used for PCR amplification of the chosen target sequences located along the 162-kb genomic region under study (Fig. <a reveal-id="DSP027F3" data-open="DSP027F3" class="link link-reveal link-table xref-fig">3</a>A). By correlating the amplification data (Fig. <a reveal-id="DSP027F3" data-open="DSP027F3" class="link link-reveal link-table xref-fig">3</a>A) with the kinetics of the nucleoid-DNA digestion (Fig. <a reveal-id="DSP027F3" data-open="DSP027F3" class="link link-reveal link-table xref-fig">3</a>B), it is possible to locate each target sequence within a topological zone relative to the NM (Fig. <a reveal-id="DSP027F2" data-open="DSP027F2" class="link link-reveal link-table xref-fig">2</a>). Previous studies have shown that the average size of the nuclear DNA fragments liberated by non-specific nucleases in rat hepatocytes is 0.8 kb.<sup><a reveal-id="DSP027C12" data-open="DSP027C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Thus, the DNA sequences to be mapped are &lt;550 bp in length (Table <a reveal-id="DSP027TB1" data-open="DSP027TB1" class="link link-reveal link-table xref-fig">1</a>), and so likely to be cut as whole units by the endonuclease instead of being progressively eroded by partial digestions. Therefore, in our mapping protocol, we score the specific templates as either present (amplifiable) or absent (non-amplifiable) as a function of endonuclease-digestion time (Fig. <a reveal-id="DSP027F5" data-open="DSP027F5" class="link link-reveal link-table xref-fig">5</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). This was scored without considering the intensity of the amplicon signals, but just whether such signals are detected or not by an image-analysis program (Kodak 1D Image Analysis Software 3.5), using the default settings. We established these criteria because in our topological-mapping approach, it is the average relative position to the NM anchoring point and not the actual template length the critical parameter that determines the average sensitivity to DNase I of each sequence mapped. Nevertheless, the chosen amplicons were also selected on the basis that all of them may be amplified with similar efficiency using the same amplification program (see Section 2). Thus, the absence of amplified product at a given digestion time-point indicates that the relative abundance of the target template has fallen to a non-amplifiable level within the large nucleoid population analysed in each sample.<sup><a reveal-id="DSP027C6" data-open="DSP027C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p>                        <a id="4914846" scrollto-destination="4914846"></a>
<div id="DSP027F3" data-id="DSP027F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02703.jpeg?Expires=2147483647&Signature=Oz9L06bGikh2vvk3WCN9A5WnWpV1EbIJ3OAOt-fki9dIXUMA77QuTkVkABZdIR0vMv9K5IlwZBIe9qKGQgc2x23Hneu5bD8bNMzTIOU8bjgWYOjHd5SeajQ6hSrzzVVAR41feMj~0ZrCRbzGU59lLQvCciidct7nMND1AIQ0Zlj7iTpoVLAqc5l5Te30Cneo6vDINzDZRxDJevh16ZuKbq93-9F08Ubod-LrpM5m3s25zoqGgr-LU3HI6hR4lFWqPNc-A4WpQwNTMR8OMV0~4OD5B7iOICbuhyT7VWNALiAsEz-8tjNVdpaWAy~0dA7M-YZ32JOO9Pp0I9hchvengg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Procedure for mapping the position of DNA sequences relative to the NM. (A) Nucleoids prepared from freshly isolated rat primary cells were incubated with DNase I so as to progressively digest the loop DNA, obtaining a tri-phasic kinetics of digestion (B). Nucleoid samples with partially digested NM-bound DNA, as shown in the drawing and fluorescent micrographs, (C) were used for PCR amplification of target DNA sequences located along the genomic region containing members of the rat albumin gene family. The amplicons were run in agarose gels and scored as present (+) or absent (−) by an image-analysis software (A) as a function of nucleoid-sample digestion time (C)." path-from-xml="dsp02703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914846/dsp02703.jpeg" path-from-xml="dsp02703.jpeg" target="_blank">View large</a><a section="4914846" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02703.jpeg?Expires=2147483647&Signature=P3uYPLTpUEs5Us~hAxZ7IHwgWUn0OOMdQ~vaqKa9rC30xw1BFqt4Kna0pIzM-fRzDMAkj436hobTPeb~eiOyfRFSyfu2VXxaWXBIOwxDME6JKNCNgVaEbL-g9~zNdgf8ri7JrtCeomLeP0HGAYE-~VSv7gglgsurUc844xAhX2l8LRSuJu~8gqF2MUgxFnlg3ti7iJSUtY9HxRQtdi28g6NQOZXj9JmWxNOYjLafyRwyHtGTBJXLAtOn9d~XzDIGpsEwI0gLhqZbsgjw-Ql6b87rGELeryPy9m2t9vTjeGB0raQXwpg1UPNquvzaX5rFyH8m7WKhYdazLaP-v-U06A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914846&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Procedure for mapping the position of DNA sequences relative to the NM. (<strong>A</strong>) Nucleoids prepared from freshly isolated rat primary cells were incubated with DNase I so as to progressively digest the loop DNA, obtaining a tri-phasic kinetics of digestion (<strong>B</strong>). Nucleoid samples with partially digested NM-bound DNA, as shown in the drawing and fluorescent micrographs, (<strong>C</strong>) were used for PCR amplification of target DNA sequences located along the genomic region containing members of the rat albumin gene family. The amplicons were run in agarose gels and scored as present (+) or absent (−) by an image-analysis software (A) as a function of nucleoid-sample digestion time (C).</p></div></div><div content-id="DSP027F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02703.jpeg?Expires=2147483647&Signature=Oz9L06bGikh2vvk3WCN9A5WnWpV1EbIJ3OAOt-fki9dIXUMA77QuTkVkABZdIR0vMv9K5IlwZBIe9qKGQgc2x23Hneu5bD8bNMzTIOU8bjgWYOjHd5SeajQ6hSrzzVVAR41feMj~0ZrCRbzGU59lLQvCciidct7nMND1AIQ0Zlj7iTpoVLAqc5l5Te30Cneo6vDINzDZRxDJevh16ZuKbq93-9F08Ubod-LrpM5m3s25zoqGgr-LU3HI6hR4lFWqPNc-A4WpQwNTMR8OMV0~4OD5B7iOICbuhyT7VWNALiAsEz-8tjNVdpaWAy~0dA7M-YZ32JOO9Pp0I9hchvengg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Procedure for mapping the position of DNA sequences relative to the NM. (A) Nucleoids prepared from freshly isolated rat primary cells were incubated with DNase I so as to progressively digest the loop DNA, obtaining a tri-phasic kinetics of digestion (B). Nucleoid samples with partially digested NM-bound DNA, as shown in the drawing and fluorescent micrographs, (C) were used for PCR amplification of target DNA sequences located along the genomic region containing members of the rat albumin gene family. The amplicons were run in agarose gels and scored as present (+) or absent (−) by an image-analysis software (A) as a function of nucleoid-sample digestion time (C)." path-from-xml="dsp02703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914846/dsp02703.jpeg" path-from-xml="dsp02703.jpeg" target="_blank">View large</a><a section="4914846" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02703.jpeg?Expires=2147483647&Signature=P3uYPLTpUEs5Us~hAxZ7IHwgWUn0OOMdQ~vaqKa9rC30xw1BFqt4Kna0pIzM-fRzDMAkj436hobTPeb~eiOyfRFSyfu2VXxaWXBIOwxDME6JKNCNgVaEbL-g9~zNdgf8ri7JrtCeomLeP0HGAYE-~VSv7gglgsurUc844xAhX2l8LRSuJu~8gqF2MUgxFnlg3ti7iJSUtY9HxRQtdi28g6NQOZXj9JmWxNOYjLafyRwyHtGTBJXLAtOn9d~XzDIGpsEwI0gLhqZbsgjw-Ql6b87rGELeryPy9m2t9vTjeGB0raQXwpg1UPNquvzaX5rFyH8m7WKhYdazLaP-v-U06A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914846&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Procedure for mapping the position of DNA sequences relative to the NM. (<strong>A</strong>) Nucleoids prepared from freshly isolated rat primary cells were incubated with DNase I so as to progressively digest the loop DNA, obtaining a tri-phasic kinetics of digestion (<strong>B</strong>). Nucleoid samples with partially digested NM-bound DNA, as shown in the drawing and fluorescent micrographs, (<strong>C</strong>) were used for PCR amplification of target DNA sequences located along the genomic region containing members of the rat albumin gene family. The amplicons were run in agarose gels and scored as present (+) or absent (−) by an image-analysis software (A) as a function of nucleoid-sample digestion time (C).</p></div></div>                        <a id="4914847" scrollto-destination="4914847"></a>
<div id="DSP027F4" data-id="DSP027F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02704.jpeg?Expires=2147483647&Signature=V7tPW6Mk6trtUeqRg0R1Wc6os-sOH~YyJGB9ntWXI~NA-DhecRhSvmlTnj2i5GS380JvSWiMWglEcaIyTDOD3POADuVdq0mN6vf1ahYWARLkir~8dyAZCJ1hzfv7Vy4xhW96kAM9Xl4DzIv8tHe3c7HJ0ClYysPEldsBWnTWt7AtP2TXQVCNH5zTUsTkC9Z0wt7gnXmOAk9nAqF7BCW5j~eVF~70H-6ePjIwZ6PtgtGhcx04QOqaJqUmH1lFdQ~X6EzFXnLJCR1P9T4A4bWv75-NvOjAnW4ym1D4qL28mMI~8iekpXaI4b~yJ3MyaUpaKzS97gzBKv788ADXuHGlww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The 162-kb genomic region containing members of the rat albumin gene family (chromosome 14): Alb (albumin), Afp (alpha-fetoprotein), Afm (afamin), Afp-L (pseudo-gene similar to Afp). The letters indicate the location of the target DNA sequences to be positionally mapped relative to the NM." path-from-xml="dsp02704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914847/dsp02704.jpeg" path-from-xml="dsp02704.jpeg" target="_blank">View large</a><a section="4914847" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02704.jpeg?Expires=2147483647&Signature=zeJgZoPTOR~SYan7p2~4xTldiq4VrSuh34kPGE-S-hg4LgtPhA1SS-egWKBmlSr5L8dp~7FYEEEUKbw4INZlZcW5p58yXBR7Qb2Ac3Xhn~USPQAGCYSEW1BpPmLanPsXH2vehqH4vMhZKpg2kgffS99aCvmqTPrztFtp3LEik5LBGURqhf7viHwS5Mcn5SwCT1nRXLScPgGz6LGd0b0u6oRLFQzNEcwfxHOLVoZP~8QJjOTPOKv3eOd0cf2hEgJcDJpuVe9aJZziEaFEN8pOizHqyrfiT5QflDtG59mjQkYeOvpoWCsReiAbHZf8iXRgnl80hEhecxco9SEeodhL1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914847&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The 162-kb genomic region containing members of the rat albumin gene family (chromosome 14): <em>Alb</em> (albumin), <em>Afp</em> (alpha-fetoprotein), <em>Afm</em> (afamin), <em>Afp-L</em> (pseudo-gene similar to <em>Afp</em>). The letters indicate the location of the target DNA sequences to be positionally mapped relative to the NM.</p></div></div><div content-id="DSP027F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02704.jpeg?Expires=2147483647&Signature=V7tPW6Mk6trtUeqRg0R1Wc6os-sOH~YyJGB9ntWXI~NA-DhecRhSvmlTnj2i5GS380JvSWiMWglEcaIyTDOD3POADuVdq0mN6vf1ahYWARLkir~8dyAZCJ1hzfv7Vy4xhW96kAM9Xl4DzIv8tHe3c7HJ0ClYysPEldsBWnTWt7AtP2TXQVCNH5zTUsTkC9Z0wt7gnXmOAk9nAqF7BCW5j~eVF~70H-6ePjIwZ6PtgtGhcx04QOqaJqUmH1lFdQ~X6EzFXnLJCR1P9T4A4bWv75-NvOjAnW4ym1D4qL28mMI~8iekpXaI4b~yJ3MyaUpaKzS97gzBKv788ADXuHGlww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The 162-kb genomic region containing members of the rat albumin gene family (chromosome 14): Alb (albumin), Afp (alpha-fetoprotein), Afm (afamin), Afp-L (pseudo-gene similar to Afp). The letters indicate the location of the target DNA sequences to be positionally mapped relative to the NM." path-from-xml="dsp02704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914847/dsp02704.jpeg" path-from-xml="dsp02704.jpeg" target="_blank">View large</a><a section="4914847" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02704.jpeg?Expires=2147483647&Signature=zeJgZoPTOR~SYan7p2~4xTldiq4VrSuh34kPGE-S-hg4LgtPhA1SS-egWKBmlSr5L8dp~7FYEEEUKbw4INZlZcW5p58yXBR7Qb2Ac3Xhn~USPQAGCYSEW1BpPmLanPsXH2vehqH4vMhZKpg2kgffS99aCvmqTPrztFtp3LEik5LBGURqhf7viHwS5Mcn5SwCT1nRXLScPgGz6LGd0b0u6oRLFQzNEcwfxHOLVoZP~8QJjOTPOKv3eOd0cf2hEgJcDJpuVe9aJZziEaFEN8pOizHqyrfiT5QflDtG59mjQkYeOvpoWCsReiAbHZf8iXRgnl80hEhecxco9SEeodhL1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914847&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The 162-kb genomic region containing members of the rat albumin gene family (chromosome 14): <em>Alb</em> (albumin), <em>Afp</em> (alpha-fetoprotein), <em>Afm</em> (afamin), <em>Afp-L</em> (pseudo-gene similar to <em>Afp</em>). The letters indicate the location of the target DNA sequences to be positionally mapped relative to the NM.</p></div></div>                        <a id="4914848" scrollto-destination="4914848"></a>
<div id="DSP027F5" data-id="DSP027F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02705.jpeg?Expires=2147483647&Signature=qtzShTHMwjCLKI17FhqYNh5V8W4xo6CZqVql7GwwM~GqP1Dr4~WZ5QmOaAO4vId50s4-C7QvErMkiczgpzWhRca82MVv1jaY8c7sL0WONHb31xczCLgPllPK3v3i5n5ynnLGhjV7XE~Ky01Kwmk5hyB6iq0M9FcKAnfT7~7fxCgQJbaNlor7I0-MyUPIUgeBZGqKKMNTUJBt6WuCZKB6snlo8dVG9blqpuZWXER2zIBBdOhYG7Y9MSwFXUdVkfdZJfRlkl54yYvTbuFgGJ3f3ZCeSzP9PWUqp8w-LfVSUQdvB2qiWAVp5wDOL6K0GHWNnU30on4O0tsvlUdviBGv-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positional mapping relative to the NM of specific target DNA sequences by PCR. Rat-hepatocyte nucleoids were treated with DNase-I (0.5 U/ml) for different times. The residual NM-bound DNA was directly used as template for PCR amplification of the target sequences (a–o). The specific amplicons were resolved in 2% agarose gels stained with ethidium bromide (0.5 µl/ml). C, control. Topological zones with respect to NM: D, distal; P, proximal; VC, very close; E, embedded within NM. (−), negative control (no template); (+), positive control (pure genomic DNA as template). The amplification patterns were consistently reproduced in separate experiments with samples from independent animals (n = 4)." path-from-xml="dsp02705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914848/dsp02705.jpeg" path-from-xml="dsp02705.jpeg" target="_blank">View large</a><a section="4914848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02705.jpeg?Expires=2147483647&Signature=J8G9PEgTjvwezRqSjJACsX6r1v5-WutjYwU66SdAoy1jKMi34o3a3SRLD0yIZ2pXU5I2gPHzdvbMGuuE-DzZwgJJZ8kPLwJgObR-TSUjZPQ6I~kXvcs4JF~J~Y-mtFfhyAMZRCECXt9S-O9wgiK2U34xI5qSzQCkgni2oU99cphhYXFYXyQj84UAThqYcjRJsQvTMdDP5gWmuGqI5nf7JmB48aGFQI~ESdVH40rmp45UNd9RAA9G6emriJv3988vit5Wj6t6Z~ksGPT-kTXrwPkKQAnssEzlHxm~dsTOOWLeiS71RceFiurYax2dvjaX32VsVxWtCnW9m~8iZrGgjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914848&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positional mapping relative to the NM of specific target DNA sequences by PCR. Rat-hepatocyte nucleoids were treated with DNase-I (0.5 U/ml) for different times. The residual NM-bound DNA was directly used as template for PCR amplification of the target sequences (a–o). The specific amplicons were resolved in 2% agarose gels stained with ethidium bromide (0.5 µl/ml). C, control. Topological zones with respect to NM: D, distal; P, proximal; VC, very close; E, embedded within NM. (−), negative control (no template); (+), positive control (pure genomic DNA as template). The amplification patterns were consistently reproduced in separate experiments with samples from independent animals (<em>n</em> = 4).</p></div></div><div content-id="DSP027F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02705.jpeg?Expires=2147483647&Signature=qtzShTHMwjCLKI17FhqYNh5V8W4xo6CZqVql7GwwM~GqP1Dr4~WZ5QmOaAO4vId50s4-C7QvErMkiczgpzWhRca82MVv1jaY8c7sL0WONHb31xczCLgPllPK3v3i5n5ynnLGhjV7XE~Ky01Kwmk5hyB6iq0M9FcKAnfT7~7fxCgQJbaNlor7I0-MyUPIUgeBZGqKKMNTUJBt6WuCZKB6snlo8dVG9blqpuZWXER2zIBBdOhYG7Y9MSwFXUdVkfdZJfRlkl54yYvTbuFgGJ3f3ZCeSzP9PWUqp8w-LfVSUQdvB2qiWAVp5wDOL6K0GHWNnU30on4O0tsvlUdviBGv-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positional mapping relative to the NM of specific target DNA sequences by PCR. Rat-hepatocyte nucleoids were treated with DNase-I (0.5 U/ml) for different times. The residual NM-bound DNA was directly used as template for PCR amplification of the target sequences (a–o). The specific amplicons were resolved in 2% agarose gels stained with ethidium bromide (0.5 µl/ml). C, control. Topological zones with respect to NM: D, distal; P, proximal; VC, very close; E, embedded within NM. (−), negative control (no template); (+), positive control (pure genomic DNA as template). The amplification patterns were consistently reproduced in separate experiments with samples from independent animals (n = 4)." path-from-xml="dsp02705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914848/dsp02705.jpeg" path-from-xml="dsp02705.jpeg" target="_blank">View large</a><a section="4914848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02705.jpeg?Expires=2147483647&Signature=J8G9PEgTjvwezRqSjJACsX6r1v5-WutjYwU66SdAoy1jKMi34o3a3SRLD0yIZ2pXU5I2gPHzdvbMGuuE-DzZwgJJZ8kPLwJgObR-TSUjZPQ6I~kXvcs4JF~J~Y-mtFfhyAMZRCECXt9S-O9wgiK2U34xI5qSzQCkgni2oU99cphhYXFYXyQj84UAThqYcjRJsQvTMdDP5gWmuGqI5nf7JmB48aGFQI~ESdVH40rmp45UNd9RAA9G6emriJv3988vit5Wj6t6Z~ksGPT-kTXrwPkKQAnssEzlHxm~dsTOOWLeiS71RceFiurYax2dvjaX32VsVxWtCnW9m~8iZrGgjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914848&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positional mapping relative to the NM of specific target DNA sequences by PCR. Rat-hepatocyte nucleoids were treated with DNase-I (0.5 U/ml) for different times. The residual NM-bound DNA was directly used as template for PCR amplification of the target sequences (a–o). The specific amplicons were resolved in 2% agarose gels stained with ethidium bromide (0.5 µl/ml). C, control. Topological zones with respect to NM: D, distal; P, proximal; VC, very close; E, embedded within NM. (−), negative control (no template); (+), positive control (pure genomic DNA as template). The amplification patterns were consistently reproduced in separate experiments with samples from independent animals (<em>n</em> = 4).</p></div></div><p>It is important to emphasize that the method described here is not intended for identification and characterization of actual MARs (LARs) in a given genomic region. Indeed, some general properties of MARs such as the presence of A-T tracts and repetitive sequences make them unsuitable for efficient PCR amplification under stringent conditions for achieving highly specific amplification of most other non-repetitive genomic sequences. Thus, primer-design programs will grant a low score to any set of primers targeting amplicons containing potential MARs, even more if such amplicons are meant to be highly specific and small-sized (≤500 bp), as those targeted in the present protocol.</p>                            <h3 scrollto-destination=4914850 id="4914850" class="section-title" data-legacy-id=s3d>3.4. Determination of the structural DNA loop organization in the region of the rat albumin locus</h3>
<p>By coupling the highly reproducible amplification experiments (Fig. <a reveal-id="DSP027F5" data-open="DSP027F5" class="link link-reveal link-table xref-fig">5</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), together with the consistent kinetics of loop DNA digestion by DNase I (Fig. <a reveal-id="DSP027F2" data-open="DSP027F2" class="link link-reveal link-table xref-fig">2</a>), we were able to locate each mapped sequence within a given topological zone relative to the NM (Tables <a reveal-id="DSP027TB2" data-open="DSP027TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSP027TB3" data-open="DSP027TB3" class="link link-reveal link-table xref-fig">3</a>). Such data were used to draw the most likely DNA loop organization relative to the NM of the 162-kb region studied, by considering the distance (in kb) between the separate sequences mapped and the topological positions of such sequences relative to the NM. The data suggest that in the hepatocytes, the region is organized in five loops anchored to the NM (Fig. <a reveal-id="DSP027F6" data-open="DSP027F6" class="link link-reveal link-table xref-fig">6</a>A). The resulting size from tip to base of such loops is not bigger than 20 kb; thus, the largest loop of the set, containing both <em>Afp</em> and <em>Afm</em> sequences, would be some 40 kb in length. The genomic region studied is a high-gene-density region. Through evolution, a series of duplications formed the genes within the <em>Alb</em> family that includes the vitamin D-binding protein (<em>DBP</em>) gene located almost 1 Mb upstream of the <em>Alb</em> gene.<sup><a reveal-id="DSP027C42" data-open="DSP027C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Nevertheless, such a gene family has been travelling as a cluster within the mammalian genomes. The fact that the region studied is organized into five rather compact structural DNA loops whose size (40–20 kb) is smaller than the estimated average rat-hepatocyte DNA loop size (80 kb) may be relevant to the functionality of such a genomic region, since fundamental processes of nuclear physiology occur at macromolecular complexes organized on the NM.<sup><a reveal-id="DSP027C17" data-open="DSP027C17" class="link link-ref link-reveal xref-bibr">17</a>–<a reveal-id="DSP027C19" data-open="DSP027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> It remains to be explored whether high-gene-density regions are usually organized into relatively small-sized structural DNA loops; at least another report suggests that such may be the case.<sup><a reveal-id="DSP027C26" data-open="DSP027C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, despite the fact that the average DNA loop size in naïve B-lymphocytes is quite similar to that in hepatocytes (our unpublished results), in naïve B-lymphocytes the corresponding 162 kb region is very close to the NM as most sequences map within the VC topological region. Moreover, the target sequence h consistently maps to the DNase-resistant fraction of NM-bound DNA, and so it must be embedded within the NM. Only the l sequence remains proximal to the NM (Table <a reveal-id="DSP027TB3" data-open="DSP027TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). This means that in naïve B-lymphocytes the 162-kb region is organized in numerous but tiny (&lt;14 kb) DNA loops (Table <a reveal-id="DSP027TB3" data-open="DSP027TB3" class="link link-reveal link-table xref-fig">3</a> and Fig. <a reveal-id="DSP027F6" data-open="DSP027F6" class="link link-reveal link-table xref-fig">6</a>B). This is quite interesting since the genes within the region studied are not known to be expressed in B-lymphocytes as it was confirmed by RT–PCR (data not shown), suggesting that packaging of genes in truly small DNA loops may be correlated with the stable transcriptional silencing of such genes (see below).</p>                        <a id="4914852" scrollto-destination="4914852"></a>
<div id="DSP027F6" data-id="DSP027F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02706.jpeg?Expires=2147483647&Signature=KRJcVU~0n29GGuMj7Cd6VNPed2nxykz2oItbJ3jD8svNgcmFAfHjSdBjpjzMz4a10HeS921Nisxz228kkH6eXE4gVkYCeb9KahL6C0nF5Dvqti229eXq5acWB6BoOitW~qND9LI9Oew1lCGI2EOezyf8eticrWaCW0oN5OdASDq7BoZ0XZpYLIP80QMdEvJUl9x~LGhSqUla6bTp2kyO0AArC163HhA0AoH7hu5QkmgsSs1LPmwdgEauLLAV1frrYHNQfVCuiYoSNLkiBsUd~610hlkSMUfXLOdnyUCTb~tgIu-9MLTu8-guV5LgvBBfCSOSEecHYIhXlEmcCC-uEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimentally determined structural DNA loop organization of the 162-kb region containing members of the rat albumin gene family in hepatocytes (A) and naïve B-lymphocytes (B). The letters indicate the position of the target sequences within the DNA loops. The bold lines indicate the position of the whole named genes. The dashed lines indicate projected loop regions outside of the region studied. The left bar indicates the relative DNA loop size from tip to base; hence the whole DNA loop length is approximately the double of such value. The right bar indicates the topological zones relative to the NM according to the kinetics of nucleoid DNA digestion. The illustration is according to scale in kb along the x and y axis." path-from-xml="dsp02706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914852/dsp02706.jpeg" path-from-xml="dsp02706.jpeg" target="_blank">View large</a><a section="4914852" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02706.jpeg?Expires=2147483647&Signature=zPIVVXGgYQR-jKar2SQ0-C0~1TcKi5eyx4HXTYwVLT8tZG3wrJWjz6LRXOLKla9~61u5MRgofJzkaRI1ahaRqm77apn5y8KyZxOLny95Rrw4tz8FSav4FhQq8ROMle8LqWrpbnUP3Vylt5nKQC0smYSz7nTyxD35txvyjWsCTINe850k~fd255OhHFfcxns0Pc0dbzRuynOIJJ2ow2ZbKnjTVc7Pn9GgF2RBOEy4BWkvPRAkL5NxLbVSx3LVrtFh-OMhOOYpA2gPrtAV~DKuIvBhojeSPZCZYxBrxEK5ClK3SpZnLezzJFgcWs4Lo8qF8wjAOqSMp3JYjQm7T~~elQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914852&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimentally determined structural DNA loop organization of the 162-kb region containing members of the rat albumin gene family in hepatocytes (<strong>A</strong>) and naïve B-lymphocytes (<strong>B</strong>). The letters indicate the position of the target sequences within the DNA loops. The bold lines indicate the position of the whole named genes. The dashed lines indicate projected loop regions outside of the region studied. The left bar indicates the relative DNA loop size from tip to base; hence the whole DNA loop length is approximately the double of such value. The right bar indicates the topological zones relative to the NM according to the kinetics of nucleoid DNA digestion. The illustration is according to scale in kb along the <em>x</em> and <em>y</em> axis.</p></div></div><div content-id="DSP027F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/m_dsp02706.jpeg?Expires=2147483647&Signature=KRJcVU~0n29GGuMj7Cd6VNPed2nxykz2oItbJ3jD8svNgcmFAfHjSdBjpjzMz4a10HeS921Nisxz228kkH6eXE4gVkYCeb9KahL6C0nF5Dvqti229eXq5acWB6BoOitW~qND9LI9Oew1lCGI2EOezyf8eticrWaCW0oN5OdASDq7BoZ0XZpYLIP80QMdEvJUl9x~LGhSqUla6bTp2kyO0AArC163HhA0AoH7hu5QkmgsSs1LPmwdgEauLLAV1frrYHNQfVCuiYoSNLkiBsUd~610hlkSMUfXLOdnyUCTb~tgIu-9MLTu8-guV5LgvBBfCSOSEecHYIhXlEmcCC-uEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimentally determined structural DNA loop organization of the 162-kb region containing members of the rat albumin gene family in hepatocytes (A) and naïve B-lymphocytes (B). The letters indicate the position of the target sequences within the DNA loops. The bold lines indicate the position of the whole named genes. The dashed lines indicate projected loop regions outside of the region studied. The left bar indicates the relative DNA loop size from tip to base; hence the whole DNA loop length is approximately the double of such value. The right bar indicates the topological zones relative to the NM according to the kinetics of nucleoid DNA digestion. The illustration is according to scale in kb along the x and y axis." path-from-xml="dsp02706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4914852/dsp02706.jpeg" path-from-xml="dsp02706.jpeg" target="_blank">View large</a><a section="4914852" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp02706.jpeg?Expires=2147483647&Signature=zPIVVXGgYQR-jKar2SQ0-C0~1TcKi5eyx4HXTYwVLT8tZG3wrJWjz6LRXOLKla9~61u5MRgofJzkaRI1ahaRqm77apn5y8KyZxOLny95Rrw4tz8FSav4FhQq8ROMle8LqWrpbnUP3Vylt5nKQC0smYSz7nTyxD35txvyjWsCTINe850k~fd255OhHFfcxns0Pc0dbzRuynOIJJ2ow2ZbKnjTVc7Pn9GgF2RBOEy4BWkvPRAkL5NxLbVSx3LVrtFh-OMhOOYpA2gPrtAV~DKuIvBhojeSPZCZYxBrxEK5ClK3SpZnLezzJFgcWs4Lo8qF8wjAOqSMp3JYjQm7T~~elQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4914852&ar=347368&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimentally determined structural DNA loop organization of the 162-kb region containing members of the rat albumin gene family in hepatocytes (<strong>A</strong>) and naïve B-lymphocytes (<strong>B</strong>). The letters indicate the position of the target sequences within the DNA loops. The bold lines indicate the position of the whole named genes. The dashed lines indicate projected loop regions outside of the region studied. The left bar indicates the relative DNA loop size from tip to base; hence the whole DNA loop length is approximately the double of such value. The right bar indicates the topological zones relative to the NM according to the kinetics of nucleoid DNA digestion. The illustration is according to scale in kb along the <em>x</em> and <em>y</em> axis.</p></div></div>                        <a id="4914853" scrollto-destination="4914853"></a>
<div content-id="DSP027TB2" class="table-modal"><div class="table-wrap"><div id="DSP027TB2" class="table-wrap-title" data-id="DSP027TB2"><div class="label -label">Table 2</div><div class="caption"><p>Location of the target sequences (amplicons) in the specific topological zones relative to the NM from rat hepatocytes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to the NM<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>i </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr></tbody></table></div><div content-id="DSP027TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to the NM<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>i </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The amplicons were scored either as positive or as negative as a function of nuclease digestion time and for each topological zone relative to the NM, depending on whether or not they were detected by a digital image-analysis system (Kodak 1D Image Analysis Software 3.5) using the default settings.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914853" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP027TB2" class="table-wrap-title" data-id="DSP027TB2"><div class="label -label">Table 2</div><div class="caption"><p>Location of the target sequences (amplicons) in the specific topological zones relative to the NM from rat hepatocytes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to the NM<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>i </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr></tbody></table></div><div content-id="DSP027TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to the NM<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>i </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>− </td><td>− </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The amplicons were scored either as positive or as negative as a function of nuclease digestion time and for each topological zone relative to the NM, depending on whether or not they were detected by a digital image-analysis system (Kodak 1D Image Analysis Software 3.5) using the default settings.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914853" target="_blank">View Large</a></div></div>                        <a id="4914854" scrollto-destination="4914854"></a>
<div content-id="DSP027TB3" class="table-modal"><div class="table-wrap"><div id="DSP027TB3" class="table-wrap-title" data-id="DSP027TB3"><div class="label -label">Table 3</div><div class="caption"><p>Location of the target sequences (amplicons) in the specific topological zones relative to the NM in naïve B-lymphocytes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to NM in B-lymphocytes<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>i </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div content-id="DSP027TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to NM in B-lymphocytes<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>i </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The amplicons were scored either as positive or as negative as a function of nuclease digestion time and for each topological zone relative to the NM, depending on whether or not they were detected by a digital image-analysis system (Kodak 1D Image Analysis Software 3.5) using the default settings.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914854" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP027TB3" class="table-wrap-title" data-id="DSP027TB3"><div class="label -label">Table 3</div><div class="caption"><p>Location of the target sequences (amplicons) in the specific topological zones relative to the NM in naïve B-lymphocytes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to NM in B-lymphocytes<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>i </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div content-id="DSP027TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="4">Topological zones relative to NM in B-lymphocytes<hr /> </th></tr><tr><th> </th><th>Distal to NM </th><th>Proximal to NM </th><th>Very close to NM </th><th>Embedded within NM </th></tr></thead><tbody><tr><td>a </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>b </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>c </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>d </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>e </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>f </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>g </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>h </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>i </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>j </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>k </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>l </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td>m </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>n </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>o </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The amplicons were scored either as positive or as negative as a function of nuclease digestion time and for each topological zone relative to the NM, depending on whether or not they were detected by a digital image-analysis system (Kodak 1D Image Analysis Software 3.5) using the default settings.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914854" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4914857 id="4914857" class="section-title" data-legacy-id=s3e>3.5. Predicted MARs in silico do not fully correspond to actual MARs in vivo</h3>
<p>We used four different available computer programs for predicting the location of potential MARs along the 162-kb region studied. All these programs have been quoted as MAR-prediction or even as MAR-identifying devices in several published papers.<sup><a reveal-id="DSP027C27" data-open="DSP027C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSP027C28" data-open="DSP027C28" class="link link-ref link-reveal xref-bibr">28</a></sup> However, each program uses different criteria so as to define the respective MAR-searching algorithms. This can be appreciated as not all programs coincide in their predictions (Table <a reveal-id="DSP027TB4" data-open="DSP027TB4" class="link link-reveal link-table xref-fig">4</a>). Moreover, the mapping results in hepatocytes indicate that loop-DNA regions containing actual MARs (LARs) are located next to the following sequences mapped: a, d, h, k, m, o. However, only the region next to the d sequence is consistently identified as a MAR-containing region by the four programs, whereas the other experimentally defined LAR-containing regions are only partially identified by some of the four MAR-searching programs used and all programs predict rather inconsistently the presence of MARs in regions that according to the experimental results are unlikely to be bound to the NM in hepatocytes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The programs detect potential MARs close to most of the sequences mapped along the region studied, but even sequences such as b and c that lack any program-identified MAR nearby (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>) are located very close to the NM in naïve B-lymphocytes, suggesting that there are actual MARs (LARs) that cannot be detected by any of the MAR-identifying programs. Only the sequence l that also lacks any program-identified MAR nearby is consistently far from the NM in naïve B-lymphocytes. These results suggest that MAR-finding programs are highly unreliable tools for actual MAR identification, a fact that has already been shown by others.<sup><a reveal-id="DSP027C28" data-open="DSP027C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                        <a id="4914861" scrollto-destination="4914861"></a>
<div content-id="DSP027TB4" class="table-modal"><div class="table-wrap"><div id="DSP027TB4" class="table-wrap-title" data-id="DSP027TB4"><div class="label -label">Table 4</div><div class="caption"><p>Location of potential MARs in the 162-kb genomic region studied as predicted by four different MAR-searching computer programs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="3">MarFinder<hr /> </th><th colspan="3">MARScan<hr /> </th><th colspan="3">ChrClass<hr /> </th><th colspan="3">SMARtest<hr /> </th></tr><tr><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th></tr></thead><tbody><tr><td>36 380 </td><td>36 580 </td><td>201 </td><td>1762 </td><td>1824 </td><td>63 </td><td>5035 </td><td>5595 </td><td>561 </td><td>36 686 </td><td>37 120 </td><td>435 </td></tr><tr><td>62 080 </td><td>62 580 </td><td>501 </td><td>2980 </td><td>3002 </td><td>23 </td><td>35 884 </td><td>36 384 </td><td>501 </td><td>47 786 </td><td>48 090 </td><td>305 </td></tr><tr><td>76 480 </td><td>76 880 </td><td>401 </td><td>36 939 </td><td>37 003 </td><td>65 </td><td>51 284 </td><td>51 584 </td><td>301 </td><td>51 306 </td><td>51 640 </td><td>335 </td></tr><tr><td>88 480 </td><td>89 580 </td><td>1101 </td><td>55 573 </td><td>55 589 </td><td>17 </td><td>59 884 </td><td>60 284 </td><td>401 </td><td>78 281 </td><td>78 580 </td><td>300 </td></tr><tr><td>102 380 </td><td>102 980 </td><td>601 </td><td>69 392 </td><td>69 513 </td><td>122 </td><td>88 984 </td><td>89 884 </td><td>901 </td><td>88 811 </td><td>89 300 </td><td>490 </td></tr><tr><td>141 901 </td><td>142 601 </td><td>701 </td><td>84 791 </td><td>84 806 </td><td>16 </td><td>90 784 </td><td>91 084 </td><td>301 </td><td>108 916 </td><td>109 295 </td><td>380 </td></tr><tr><td> </td><td> </td><td> </td><td>88 179 </td><td>88 232 </td><td>54 </td><td>102 561 </td><td>102 861 </td><td>301 </td><td>138 866 </td><td>139 280 </td><td>415 </td></tr><tr><td> </td><td> </td><td> </td><td>102 643 </td><td>102 669 </td><td>27 </td><td>141 301 </td><td>141 801 </td><td>501 </td><td>141 341 </td><td>141 650 </td><td>310 </td></tr><tr><td> </td><td> </td><td> </td><td>131 451 </td><td>131 487 </td><td>37 </td><td>142 101 </td><td>142 601 </td><td>501 </td><td>142 106 </td><td>142 570 </td><td>465 </td></tr><tr><td> </td><td> </td><td> </td><td>142 475 </td><td>142 491 </td><td>17 </td><td>153 801 </td><td>154 601 </td><td>801 </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>142 773 </td><td>142 788 </td><td>16 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSP027TB4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="3">MarFinder<hr /> </th><th colspan="3">MARScan<hr /> </th><th colspan="3">ChrClass<hr /> </th><th colspan="3">SMARtest<hr /> </th></tr><tr><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th></tr></thead><tbody><tr><td>36 380 </td><td>36 580 </td><td>201 </td><td>1762 </td><td>1824 </td><td>63 </td><td>5035 </td><td>5595 </td><td>561 </td><td>36 686 </td><td>37 120 </td><td>435 </td></tr><tr><td>62 080 </td><td>62 580 </td><td>501 </td><td>2980 </td><td>3002 </td><td>23 </td><td>35 884 </td><td>36 384 </td><td>501 </td><td>47 786 </td><td>48 090 </td><td>305 </td></tr><tr><td>76 480 </td><td>76 880 </td><td>401 </td><td>36 939 </td><td>37 003 </td><td>65 </td><td>51 284 </td><td>51 584 </td><td>301 </td><td>51 306 </td><td>51 640 </td><td>335 </td></tr><tr><td>88 480 </td><td>89 580 </td><td>1101 </td><td>55 573 </td><td>55 589 </td><td>17 </td><td>59 884 </td><td>60 284 </td><td>401 </td><td>78 281 </td><td>78 580 </td><td>300 </td></tr><tr><td>102 380 </td><td>102 980 </td><td>601 </td><td>69 392 </td><td>69 513 </td><td>122 </td><td>88 984 </td><td>89 884 </td><td>901 </td><td>88 811 </td><td>89 300 </td><td>490 </td></tr><tr><td>141 901 </td><td>142 601 </td><td>701 </td><td>84 791 </td><td>84 806 </td><td>16 </td><td>90 784 </td><td>91 084 </td><td>301 </td><td>108 916 </td><td>109 295 </td><td>380 </td></tr><tr><td> </td><td> </td><td> </td><td>88 179 </td><td>88 232 </td><td>54 </td><td>102 561 </td><td>102 861 </td><td>301 </td><td>138 866 </td><td>139 280 </td><td>415 </td></tr><tr><td> </td><td> </td><td> </td><td>102 643 </td><td>102 669 </td><td>27 </td><td>141 301 </td><td>141 801 </td><td>501 </td><td>141 341 </td><td>141 650 </td><td>310 </td></tr><tr><td> </td><td> </td><td> </td><td>131 451 </td><td>131 487 </td><td>37 </td><td>142 101 </td><td>142 601 </td><td>501 </td><td>142 106 </td><td>142 570 </td><td>465 </td></tr><tr><td> </td><td> </td><td> </td><td>142 475 </td><td>142 491 </td><td>17 </td><td>153 801 </td><td>154 601 </td><td>801 </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>142 773 </td><td>142 788 </td><td>16 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers indicate the nucleotide start and ending position number along the genomic region studied. The length of the predicted MARs is indicated in bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914861" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP027TB4" class="table-wrap-title" data-id="DSP027TB4"><div class="label -label">Table 4</div><div class="caption"><p>Location of potential MARs in the 162-kb genomic region studied as predicted by four different MAR-searching computer programs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="3">MarFinder<hr /> </th><th colspan="3">MARScan<hr /> </th><th colspan="3">ChrClass<hr /> </th><th colspan="3">SMARtest<hr /> </th></tr><tr><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th></tr></thead><tbody><tr><td>36 380 </td><td>36 580 </td><td>201 </td><td>1762 </td><td>1824 </td><td>63 </td><td>5035 </td><td>5595 </td><td>561 </td><td>36 686 </td><td>37 120 </td><td>435 </td></tr><tr><td>62 080 </td><td>62 580 </td><td>501 </td><td>2980 </td><td>3002 </td><td>23 </td><td>35 884 </td><td>36 384 </td><td>501 </td><td>47 786 </td><td>48 090 </td><td>305 </td></tr><tr><td>76 480 </td><td>76 880 </td><td>401 </td><td>36 939 </td><td>37 003 </td><td>65 </td><td>51 284 </td><td>51 584 </td><td>301 </td><td>51 306 </td><td>51 640 </td><td>335 </td></tr><tr><td>88 480 </td><td>89 580 </td><td>1101 </td><td>55 573 </td><td>55 589 </td><td>17 </td><td>59 884 </td><td>60 284 </td><td>401 </td><td>78 281 </td><td>78 580 </td><td>300 </td></tr><tr><td>102 380 </td><td>102 980 </td><td>601 </td><td>69 392 </td><td>69 513 </td><td>122 </td><td>88 984 </td><td>89 884 </td><td>901 </td><td>88 811 </td><td>89 300 </td><td>490 </td></tr><tr><td>141 901 </td><td>142 601 </td><td>701 </td><td>84 791 </td><td>84 806 </td><td>16 </td><td>90 784 </td><td>91 084 </td><td>301 </td><td>108 916 </td><td>109 295 </td><td>380 </td></tr><tr><td> </td><td> </td><td> </td><td>88 179 </td><td>88 232 </td><td>54 </td><td>102 561 </td><td>102 861 </td><td>301 </td><td>138 866 </td><td>139 280 </td><td>415 </td></tr><tr><td> </td><td> </td><td> </td><td>102 643 </td><td>102 669 </td><td>27 </td><td>141 301 </td><td>141 801 </td><td>501 </td><td>141 341 </td><td>141 650 </td><td>310 </td></tr><tr><td> </td><td> </td><td> </td><td>131 451 </td><td>131 487 </td><td>37 </td><td>142 101 </td><td>142 601 </td><td>501 </td><td>142 106 </td><td>142 570 </td><td>465 </td></tr><tr><td> </td><td> </td><td> </td><td>142 475 </td><td>142 491 </td><td>17 </td><td>153 801 </td><td>154 601 </td><td>801 </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>142 773 </td><td>142 788 </td><td>16 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSP027TB4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="3">MarFinder<hr /> </th><th colspan="3">MARScan<hr /> </th><th colspan="3">ChrClass<hr /> </th><th colspan="3">SMARtest<hr /> </th></tr><tr><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th><th>Start </th><th>End </th><th>Length (bp) </th></tr></thead><tbody><tr><td>36 380 </td><td>36 580 </td><td>201 </td><td>1762 </td><td>1824 </td><td>63 </td><td>5035 </td><td>5595 </td><td>561 </td><td>36 686 </td><td>37 120 </td><td>435 </td></tr><tr><td>62 080 </td><td>62 580 </td><td>501 </td><td>2980 </td><td>3002 </td><td>23 </td><td>35 884 </td><td>36 384 </td><td>501 </td><td>47 786 </td><td>48 090 </td><td>305 </td></tr><tr><td>76 480 </td><td>76 880 </td><td>401 </td><td>36 939 </td><td>37 003 </td><td>65 </td><td>51 284 </td><td>51 584 </td><td>301 </td><td>51 306 </td><td>51 640 </td><td>335 </td></tr><tr><td>88 480 </td><td>89 580 </td><td>1101 </td><td>55 573 </td><td>55 589 </td><td>17 </td><td>59 884 </td><td>60 284 </td><td>401 </td><td>78 281 </td><td>78 580 </td><td>300 </td></tr><tr><td>102 380 </td><td>102 980 </td><td>601 </td><td>69 392 </td><td>69 513 </td><td>122 </td><td>88 984 </td><td>89 884 </td><td>901 </td><td>88 811 </td><td>89 300 </td><td>490 </td></tr><tr><td>141 901 </td><td>142 601 </td><td>701 </td><td>84 791 </td><td>84 806 </td><td>16 </td><td>90 784 </td><td>91 084 </td><td>301 </td><td>108 916 </td><td>109 295 </td><td>380 </td></tr><tr><td> </td><td> </td><td> </td><td>88 179 </td><td>88 232 </td><td>54 </td><td>102 561 </td><td>102 861 </td><td>301 </td><td>138 866 </td><td>139 280 </td><td>415 </td></tr><tr><td> </td><td> </td><td> </td><td>102 643 </td><td>102 669 </td><td>27 </td><td>141 301 </td><td>141 801 </td><td>501 </td><td>141 341 </td><td>141 650 </td><td>310 </td></tr><tr><td> </td><td> </td><td> </td><td>131 451 </td><td>131 487 </td><td>37 </td><td>142 101 </td><td>142 601 </td><td>501 </td><td>142 106 </td><td>142 570 </td><td>465 </td></tr><tr><td> </td><td> </td><td> </td><td>142 475 </td><td>142 491 </td><td>17 </td><td>153 801 </td><td>154 601 </td><td>801 </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td>142 773 </td><td>142 788 </td><td>16 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers indicate the nucleotide start and ending position number along the genomic region studied. The length of the predicted MARs is indicated in bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4914861" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4914863 id="4914863" class="section-title" data-legacy-id=s3f>3.6. Conclusions</h3>
<p>The method described in this paper makes it possible to determine the organization in structural DNA loops of any long segment of genomic DNA with known sequence, provided that such a DNA is attached to the NM as is the case in nucleoid preparations of freshly isolated hepatocytes and naïve B-lymphocytes. The procedure used for isolating the nucleoids only preserves DNA–NM interactions that are resistant to high-salt extraction, therefore, facultative non-structural attachments to the NM that may result from some transient functional process such as active transcription are eliminated.<sup><a reveal-id="DSP027C43" data-open="DSP027C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Our method exploits the topological properties of naked DNA attached to the NM as well as elementary topological principles such as that points in a deformable string (DNA) can be positionally mapped relative to a position-reference invariant (NM), and from such mapping, it can be deduced the configuration of the string in third dimension. Moreover, both the intrinsic persistence length of mixed-sequence DNA, estimated at some 240 bp, and negative supercoiling confer relative rigidity to the naked DNA loops,<sup><a reveal-id="DSP027C36" data-open="DSP027C36" class="link link-ref link-reveal xref-bibr">36</a></sup> thus limiting their deformability in such a way that the determined positions of loop-DNA sequences relative to the NM are highly reproducible among experiments, indicating limited fluctuation of the DNA loop morphology over time. Nevertheless, some sequences that happen to lie close to the boundary between two topological zones may display some variability in their relative position to the NM, such is the case of sequence e in hepatocytes that in one of four experiments it was detected in the proximal zone, whereas in three experiments it was only detectable in the distal zone relative to the NM (Fig. <a reveal-id="DSP027F5" data-open="DSP027F5" class="link link-reveal link-table xref-fig">5</a>). It must be stressed that the presence of the corresponding amplicons in a given topological zone was determined by an image-analysis system using its default settings independently of the apparent intensity of the amplicon signal in the corresponding gel photograph. Thus, for example, in one of four experiments with hepatocytes, the k amplicon was barely apparent in the proximal region; nevertheless, it was consistently detected in such a region by the image-analysis software in the four separate experiments (Fig. <a reveal-id="DSP027F5" data-open="DSP027F5" class="link link-reveal link-table xref-fig">5</a>).</p><p>The present protocol works with NM-bound DNA templates and so the actual MARs (LARs) present in such preparations are deeply embedded within the NM, thus being poorly available to the action of any externally added enzyme. Most LAR-identifying protocols depend on the proteolytic destruction of the NM so as to liberate the NM-bound DNA fragments that may be further characterized by a number of techniques.<sup><a reveal-id="DSP027C22" data-open="DSP027C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSP027C24" data-open="DSP027C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSP027C26" data-open="DSP027C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSP027C30" data-open="DSP027C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSP027C44" data-open="DSP027C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Thus, our protocol addresses the problem of determining the configuration of structural DNA loops in a given, large enough genomic region without the need of mapping or characterizing the actual MARs (LARs) present in such a region as a precondition for determining the possible loop configuration. Nevertheless, our protocol delimits the genomic regions where such LARs are more likely to be found. Yet, pinpointing the actual extension of such LARs depends on applying further techniques beyond the present protocol. So far, there is only one published protocol that directly maps LARs on NM-bound DNA; this protocol depends on the activation of endogenous, NM-bound topoisomerase II that releases DNA loops by cutting along MARs embedded within the NM.<sup><a reveal-id="DSP027C23" data-open="DSP027C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSP027C45" data-open="DSP027C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p><p>The <em>in vivo</em>, local spatial configuration of DNA into structural loops, as determined by the present protocol, constitutes the starting point for any further refinement or modification of such a configuration by chromatin proteins and epigenetic mechanisms affecting chromatin structure. However, chromatin as such plays no role in determining the structural DNA loop organization since chromatin proteins are completely eliminated by the high-salt extraction, and yet stable DNA–NM interactions persist. In any case, the organization of the genome into structural DNA loops must be highly relevant for the nuclear physiology. Indeed, current evidence suggest that the structural DNA loops may correspond to the actual replicons,<sup><a reveal-id="DSP027C5" data-open="DSP027C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSP027C6" data-open="DSP027C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP027C19" data-open="DSP027C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSP027C39" data-open="DSP027C39" class="link link-ref link-reveal xref-bibr">39</a></sup> yet the pattern of such structural loops may also determine the limits of further local structural changes that may be related to transient chromatin modifications resulting in functional interactions (related to transcription) between loop DNA and protein complexes located upon the NM.<sup><a reveal-id="DSP027C18" data-open="DSP027C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSP027C46" data-open="DSP027C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSP027C47" data-open="DSP027C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The genomic region studied contains classical examples of developmentally regulated genes. Indeed, <em>Alb</em>, <em>Afp</em> and <em>Afm</em> are primarily expressed in the liver, yet while <em>Afp</em> and <em>Alb</em> are highly expressed in the fetal liver, the transcription of <em>Afp</em> is rapidly reduced after birth, but <em>Alb</em> and <em>Afm</em> continue to be expressed in the adult liver.<sup><a reveal-id="DSP027C42" data-open="DSP027C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSP027C48" data-open="DSP027C48" class="link link-ref link-reveal xref-bibr">48</a></sup> There are three enhancers in the intergenic region between <em>Alb</em> and <em>Afp</em> whose action is required for <em>Afp</em> and <em>Alb</em> activation early in liver development and for <em>Afp</em> reactivation during liver regeneration, yet later during fetal development <em>Alb</em> expression becomes independent of such enhancers due to the action of an <em>Alb</em>-specific enhancer that lies between 8.5 and 10 kb upstream the <em>Alb</em> transcription-start site.<sup><a reveal-id="DSP027C42" data-open="DSP027C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSP027C49" data-open="DSP027C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Interestingly, the intergenic enhancers are located in the region between the sequences corresponding to amplicons c and d<sup><a reveal-id="DSP027C48" data-open="DSP027C48" class="link link-ref link-reveal xref-bibr">48</a></sup> which is clearly detached from the NM in the adult rat (Fig. <a reveal-id="DSP027F6" data-open="DSP027F6" class="link link-reveal link-table xref-fig">6</a>A). We actually mapped the position relative to the NM of enhancer 1 (E1) which is located 2 kb upstream of sequence d that belongs to the <em>Afp</em> promoter region (Fig. <a reveal-id="DSP027F4" data-open="DSP027F4" class="link link-reveal link-table xref-fig">4</a>). E1 mapped within the region proximal to the NM (data not shown) the same as d (Table <a reveal-id="DSP027TB2" data-open="DSP027TB2" class="link link-reveal link-table xref-fig">2</a>). So far the <em>Alb-</em>specific enhancer has not been formally characterized in the rat. However, comparison of the mouse <em>Alb</em>-enhancer sequence indicates that a closely related sequence is present in the rat in a 1.3-kb region located 8.3 kb upstream of the <em>Alb</em> transcription-start site. Our results suggest the presence of a LAR some 5 kb upstream of sequence a that belongs to the <em>Alb</em> promoter (Figs <a reveal-id="DSP027F4" data-open="DSP027F4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSP027F6" data-open="DSP027F6" class="link link-reveal link-table xref-fig">6</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Hence the putative rat <em>Alb</em>-enhancer is located some 2 kb upstream of the possible LAR and so the enhancer could be very close to the NM, although this needs to be experimentally verified.</p><p>It is known that replication and transcription occur at factories organized upon the NM.<sup><a reveal-id="DSP027C18" data-open="DSP027C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSP027C50" data-open="DSP027C50" class="link link-ref link-reveal xref-bibr">50</a></sup> However, replication and transcription factories never co-localize since they occupy distinct but rather fixed sites that are conserved throughout several cell generations.<sup><a reveal-id="DSP027C17" data-open="DSP027C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP027C51" data-open="DSP027C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSP027C52" data-open="DSP027C52" class="link link-ref link-reveal xref-bibr">52</a></sup> In mammalian lymphocytes it is known that genes separated by tens of mega-bases in <em>cis</em> and genes that belong to separate chromosomes somehow migrate to preassembled transcription factories for their actual transcription.<sup><a reveal-id="DSP027C50" data-open="DSP027C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DSP027C53" data-open="DSP027C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Such a process requires that the genes undergoing active transcription loop out away from their respective chromosome territories so that distant genomic regions are brought together within one nuclear complex for transcription.<sup><a reveal-id="DSP027C54" data-open="DSP027C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Hence the fact that in naïve B-lymphocytes the genomic region studied is very close to the NM, suggests that the genes present in the region are prevented from looping out towards a transcription factory, since the genes are included into numerous tiny loops tightly bound to the NM (Fig. <a reveal-id="DSP027F6" data-open="DSP027F6" class="link link-reveal link-table xref-fig">6</a>B).</p><p>The high-salt-resistant LARs that determine structural attachments to the NM constitute long-term interactions between DNA and the NM. Nevertheless, as predicted by some models, there is evidence that under the influence of a combination of mechanical, biochemical and thermodynamical factors the pattern of such attachments may be modified during development and aging leading to stable changes in cell differentiation.<sup><a reveal-id="DSP027C10" data-open="DSP027C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP027C13" data-open="DSP027C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP027C55" data-open="DSP027C55" class="link link-ref link-reveal xref-bibr">55</a>,<a reveal-id="DSP027C56" data-open="DSP027C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Varied and important evidence suggests that the high-salt resistant structural DNA loops correspond to the actual replicons <em>in vivo</em>.<sup><a reveal-id="DSP027C5" data-open="DSP027C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSP027C19" data-open="DSP027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Thus it is remarkable that in totipotent blastomeres the cell cycle consists of overlapping S and M phases with no G1 or G2, the rapid DNA replication observed involves the use of high numbers of closely spaced attachments between DNA and the NM and so the average DNA loops are very small, yet this is compatible with very active DNA replication but not with active transcription as genes in the chromosomes of blastomeres are largely inactive during cleavage.<sup><a reveal-id="DSP027C57" data-open="DSP027C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Therefore in naïve B-lymphocytes the presence of tiny loops in the region studied may contribute to the transcriptional silencing of the corresponding genes without preventing such a region from being replicated after B-lymphocyte activation.</p><p>The nucleotype has been defined as those characters of nuclear DNA that may affect the phenotype independently of the information encoded in such a DNA,<sup><a reveal-id="DSP027C58" data-open="DSP027C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and it has been suggested that besides species-specific nucleotypic characters such as karyotype, and the C-value for nuclear DNA that correlates with overall cell-cycle length, there could be tissue-specific nucleotypic characters, such as the pattern of structural DNA loops, that correlate with cell differentiation.<sup><a reveal-id="DSP027C10" data-open="DSP027C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Our results support the existence of such tissue specific nucleotypic characters.</p>                            <h2 scrollto-destination=4914876 id="4914876" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4914880 id="4914880" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was sponsored by CONACYT, Mexico, grant 48447-Q (25506) and UAEM, Mexico, grant 2212/2006, both awarded to A.A.-A. J.C.R.-M. is a CONACYT Research Scholar within the Graduate Program in Biomedical Sciences at IFC-UNAM, México.</p>                            <h2 scrollto-destination=4914885 id="4914885" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP027C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nickerson</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Experimental observations of a nuclear matrix</div>, <div class="source">J. Cell Sci.</div> , <div class="year">2001</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">463</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20observations%20of%20a%20nuclear%20matrix&amp;author=J.A.%20Nickerson&amp;publication_year=2001&amp;journal=J.%20Cell%20Sci.&amp;volume=114&amp;pages=463-74" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11171316" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsutsui</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsutsui</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Dynamic view of the nuclear matrix</div>, <div class="source">Acta Med. Okayama</div> , <div class="year">2005</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">113</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20view%20of%20the%20nuclear%20matrix&amp;author=K.M.%20Tsutsui&amp;author=K.%20Sano&amp;author=K.%20Tsutsui&amp;publication_year=2005&amp;journal=Acta%20Med.%20Okayama&amp;volume=59&amp;pages=113-20" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16155636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mika</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rost</div> <div class="given-names">B.</div></div>. </div><div class="article-title">NMPdb: database of nuclear matrix proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D160</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NMPdb%3A%20database%20of%20nuclear%20matrix%20proteins&amp;author=S.%20Mika&amp;author=B.%20Rost&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D160-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ottaviani</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lever</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Takousis</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sheer</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Anchoring the genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">2011</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anchoring%20the%20genome&amp;author=D.%20Ottaviani&amp;author=E.%20Lever&amp;author=P.%20Takousis&amp;author=D.%20Sheer&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=2011" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-201"> </span></div> </div></div></div></div></div><div content-id="DSP027C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">The nuclear matrix and chromosomal DNA loops: is there any correlation between partitioning of the genome into loops and functional domains?</div>, <div class="source">Cell Mol. Biol. Lett.</div> , <div class="year">2001</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">59</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nuclear%20matrix%20and%20chromosomal%20DNA%20loops%3A%20is%20there%20any%20correlation%20between%20partitioning%20of%20the%20genome%20into%20loops%20and%20functional%20domains%3F&amp;author=S.V.%20Razin&amp;publication_year=2001&amp;journal=Cell%20Mol.%20Biol.%20Lett.&amp;volume=6&amp;pages=59-69" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11544631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maya-Mendoza</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hernández-Muñoz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gariglio</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Gene positional changes relative to the nuclear substructure correlate with the proliferating status of hepatocytes during liver regeneration</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">6168</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20positional%20changes%20relative%20to%20the%20nuclear%20substructure%20correlate%20with%20the%20proliferating%20status%20of%20hepatocytes%20during%20liver%20regeneration&amp;author=A.%20Maya-Mendoza&amp;author=R.%20Hern%C3%A1ndez-Mu%C3%B1oz&amp;author=P.%20Gariglio&amp;author=A.%20Aranda-Anzaldo&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6168-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg825" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg825" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg825"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14576303" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heng</div> <div class="given-names">H.H.Q.</div></div>,  <div class="name"><div class="surname">Goetze</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ye</div> <div class="given-names">J.C.</div></div>, et al. </div><div class="article-title">Chromatin loops are selectively anchored using scaffold/matrix-attachment regions</div>, <div class="source">J. Cell Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">999</div>-<div class="lpage">1008</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20loops%20are%20selectively%20anchored%20using%20scaffold%2Fmatrix-attachment%20regions&amp;author=H.H.Q.%20Heng&amp;author=S.%20Goetze&amp;author=J.C.%20Ye&amp;publication_year=2004&amp;journal=J.%20Cell%20Sci.&amp;volume=117&amp;pages=999-1008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.00976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.00976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.00976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14996931" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petrova</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Iarovaia</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Verbovoy</div> <div class="given-names">V.A.</div></div>,  <div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">Specific radial positions of human chromosomes X, 1 and 19 remain unchanged in chromatin depleted nuclei of primary human fibroblasts: evidence for the organizing role of the nuclear matrix</div>, <div class="source">J. Cell. Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">850</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specific%20radial%20positions%20of%20human%20chromosomes%20X%2C%201%20and%2019%20remain%20unchanged%20in%20chromatin%20depleted%20nuclei%20of%20primary%20human%20fibroblasts%3A%20evidence%20for%20the%20organizing%20role%20of%20the%20nuclear%20matrix&amp;author=N.V.%20Petrova&amp;author=O.V.%20Iarovaia&amp;author=V.A.%20Verbovoy&amp;author=S.V.%20Razin&amp;publication_year=2005&amp;journal=J.%20Cell.%20Biochem.&amp;volume=96&amp;pages=850-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16149066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Busch</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Dynamic properties of the nuclear matrix</div>, <div class="source">The Cell Nucleus</div> , <div class="year">1979</div><div class="publisher-loc">Orlando</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">413</div>-<div class="lpage">56</div>)</div></div></div></div><div content-id="DSP027C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">On the role of chromatin higher-order structure and mechanical interactions in the regulation of gene expression</div>, <div class="source">Speculations Sci. Technol.</div> , <div class="year">1989</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">163</div>-<div class="lpage">76</div>)</div></div></div></div><div content-id="DSP027C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hakes</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>. </div><div class="article-title">DNA binding properties of the nuclear matrix and individual nuclear matrix proteins</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1991</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">11131</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20binding%20properties%20of%20the%20nuclear%20matrix%20and%20individual%20nuclear%20matrix%20proteins&amp;author=D.J.%20Hakes&amp;author=R.%20Berezney&amp;publication_year=1991&amp;journal=J.%20Biol.%20Chem.&amp;volume=266&amp;pages=11131-40" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2040622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Buchholtz</div> <div class="given-names">L.A.</div></div>. </div><div class="article-title">Dynamic association of replicating DNA fragments with the nuclear matrix of regenerating liver</div>, <div class="source">Exp. Cell Res.</div> , <div class="year">1981</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">1</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20association%20of%20replicating%20DNA%20fragments%20with%20the%20nuclear%20matrix%20of%20regenerating%20liver&amp;author=R.%20Berezney&amp;author=L.A.%20Buchholtz&amp;publication_year=1981&amp;journal=Exp.%20Cell%20Res.&amp;volume=132&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-4827(81)90076-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-4827(81)90076-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-4827(81)90076-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7202556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maya-Mendoza</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hernández-Muñoz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gariglio</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Natural ageing in the rat liver correlates with progressive stabilisation of DNA-nuclear matrix interactions and withdrawal of genes from the nuclear substructure</div>, <div class="source">Mech. Ageing Dev.</div> , <div class="year">2005</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">767</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20ageing%20in%20the%20rat%20liver%20correlates%20with%20progressive%20stabilisation%20of%20DNA-nuclear%20matrix%20interactions%20and%20withdrawal%20of%20genes%20from%20the%20nuclear%20substructure&amp;author=A.%20Maya-Mendoza&amp;author=R.%20Hern%C3%A1ndez-Mu%C3%B1oz&amp;author=P.%20Gariglio&amp;author=A.%20Aranda-Anzaldo&amp;publication_year=2005&amp;journal=Mech.%20Ageing%20Dev.&amp;volume=126&amp;pages=767-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mad.2005.01.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mad.2005.01.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mad.2005.01.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15888332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bode</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ríos-Ramírez</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mielke</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Stengert</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kay</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Klehr-Wirth</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Scaffold/matrix-attached regions: structural properties creating transcriptionally active loci</div>, <div class="source">Int. Rev. Cytol.</div> , <div class="year">1995</div>, vol. <div class="volume">162A</div> (pg. <div class="fpage">389</div>-<div class="lpage">454</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffold%2Fmatrix-attached%20regions%3A%20structural%20properties%20creating%20transcriptionally%20active%20loci&amp;author=J.%20Bode&amp;author=M.%20R%C3%ADos-Ram%C3%ADrez&amp;author=C.%20Mielke&amp;author=M.%20Stengert&amp;author=V.%20Kay&amp;author=D.%20Klehr-Wirth&amp;publication_year=1995&amp;journal=Int.%20Rev.%20Cytol.&amp;volume=162A&amp;pages=389-454" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8575884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xi</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Hedge</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Shakked</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Crothers</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Predicting indirect readout effects in protein-DNA interactions</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">8337</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20indirect%20readout%20effects%20in%20protein-DNA%20interactions&amp;author=Y.%20Zhang&amp;author=Z.%20Xi&amp;author=R.S.%20Hedge&amp;author=Z.%20Shakked&amp;author=D.M.%20Crothers&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=8337-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402319101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402319101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402319101"> </span></div> </div></div></div></div></div><div content-id="DSP027C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsutsui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsutsui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muller</div> <div class="given-names">M.T.</div></div>. </div><div class="article-title">The nuclear scaffold exhibits DNA-binding sites selective for supercoiled DNA</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1988</div>, vol. <div class="volume">263</div> (pg. <div class="fpage">7235</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nuclear%20scaffold%20exhibits%20DNA-binding%20sites%20selective%20for%20supercoiled%20DNA&amp;author=K.%20Tsutsui&amp;author=K.%20Tsutsui&amp;author=M.T.%20Muller&amp;publication_year=1988&amp;journal=J.%20Biol.%20Chem.&amp;volume=263&amp;pages=7235-41" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3366776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Samarabandu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Devdhar</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Siegel</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Acharya</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Segregation of transcription and replication sites into higher order domains</div>, <div class="source">Science</div> , <div class="year">1998</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">1502</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segregation%20of%20transcription%20and%20replication%20sites%20into%20higher%20order%20domains&amp;author=X.%20Wei&amp;author=J.%20Samarabandu&amp;author=R.S.%20Devdhar&amp;author=A.J.%20Siegel&amp;author=R.%20Acharya&amp;author=R.%20Berezney&amp;publication_year=1998&amp;journal=Science&amp;volume=281&amp;pages=1502-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.281.5382.1502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.281.5382.1502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.281.5382.1502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9727975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cook</div> <div class="given-names">P.R.</div></div>. </div><div class="article-title">The organization of replication and transcription</div>, <div class="source">Science</div> , <div class="year">1999</div>, vol. <div class="volume">282</div> (pg. <div class="fpage">1790</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20organization%20of%20replication%20and%20transcription&amp;author=P.R.%20Cook&amp;publication_year=1999&amp;journal=Science&amp;volume=282&amp;pages=1790-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.284.5421.1790" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.284.5421.1790" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.284.5421.1790"> </span></div> </div></div></div></div></div><div content-id="DSP027C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anachkova</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Djeliova</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Russev</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Nuclear matrix support of DNA replication</div>, <div class="source">J. Cell Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">951</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20matrix%20support%20of%20DNA%20replication&amp;author=B.%20Anachkova&amp;author=V.%20Djeliova&amp;author=G.%20Russev&amp;publication_year=2005&amp;journal=J.%20Cell%20Biochem.&amp;volume=96&amp;pages=951-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16167334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dent</div> <div class="given-names">M.A.R.</div></div>. </div><div class="article-title">Loss of DNA-loop supercoiling and organization in cells infected by herpes simplex virus type 1</div>, <div class="source">Res. Virol.</div> , <div class="year">1997</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">397</div>-<div class="lpage">408</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss%20of%20DNA-loop%20supercoiling%20and%20organization%20in%20cells%20infected%20by%20herpes%20simplex%20virus%20type%201&amp;author=A.%20Aranda-Anzaldo&amp;author=M.A.R.%20Dent&amp;publication_year=1997&amp;journal=Res.%20Virol.&amp;volume=148&amp;pages=397-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0923-2516(97)83629-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0923-2516(97)83629-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0923-2516(97)83629-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9498011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Orozco-Velasco</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">García-Villa</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Gariglio</div> <div class="given-names">P.</div></div>. </div><div class="article-title">p53 is a rate-limiting factor in the repair of higher-order DNA structure</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="year">1999</div>, vol. <div class="volume">1446</div> (pg. <div class="fpage">181</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=p53%20is%20a%20rate-limiting%20factor%20in%20the%20repair%20of%20higher-order%20DNA%20structure&amp;author=A.%20Aranda-Anzaldo&amp;author=F.%20Orozco-Velasco&amp;author=E.%20Garc%C3%ADa-Villa&amp;author=P.%20Gariglio&amp;publication_year=1999&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1446&amp;pages=181-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-4781(99)00086-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-4781(99)00086-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-4781(99)00086-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10524193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gombert</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Farris</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Rubio</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Morey-Rosler</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Schubach</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Krumm</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The c-myc insulator element and matrix attachment region define the c-myc chromosomal domain</div>, <div class="source">Mol. Cell Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">9338</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20c-myc%20insulator%20element%20and%20matrix%20attachment%20region%20define%20the%20c-myc%20chromosomal%20domain&amp;author=W.M.%20Gombert&amp;author=S.D.%20Farris&amp;author=E.D.%20Rubio&amp;author=K.M.%20Morey-Rosler&amp;author=W.H.%20Schubach&amp;author=A.%20Krumm&amp;publication_year=2003&amp;journal=Mol.%20Cell%20Biol.&amp;volume=23&amp;pages=9338-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.23.24.9338-9348.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.23.24.9338-9348.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.23.24.9338-9348.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14645543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iarovaia</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Bystritsky</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ravcheev</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hancock</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">Visualization of individual DNA loops and a map of loop domains in the human dystrophin gene</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">2079</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Visualization%20of%20individual%20DNA%20loops%20and%20a%20map%20of%20loop%20domains%20in%20the%20human%20dystrophin%20gene&amp;author=O.V.%20Iarovaia&amp;author=A.%20Bystritsky&amp;author=D.%20Ravcheev&amp;author=R.%20Hancock&amp;author=S.V.%20Razin&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=2079-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh532" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh532" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh532"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15087487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petrov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pirozhkova</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Carnac</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Laoudj</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lipinski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Vassetzky</div> <div class="given-names">Y.S.</div></div>. </div><div class="article-title">Chromatin loop domain organization within the 4q35 locus in facioscapulohumeral dystrophy patients versus normal human myoblasts</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">6982</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20loop%20domain%20organization%20within%20the%204q35%20locus%20in%20facioscapulohumeral%20dystrophy%20patients%20versus%20normal%20human%20myoblasts&amp;author=A.%20Petrov&amp;author=I.%20Pirozhkova&amp;author=G.%20Carnac&amp;author=D.%20Laoudj&amp;author=M.%20Lipinski&amp;author=Y.S.%20Vassetzky&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=6982-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0511235103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0511235103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0511235103"> </span></div> </div></div></div></div></div><div content-id="DSP027C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hair</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vassetzky</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Determination of the chromatin domain structure in arrayed repeated regions: organization of the somatic 5S RNA domain during embryogenesis in <em>Xenopus laevis</em></div>, <div class="source">J. Cell Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">1140</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determination%20of%20the%20chromatin%20domain%20structure%20in%20arrayed%20repeated%20regions%3A%20organization%20of%20the%20somatic%205S%20RNA%20domain%20during%20embryogenesis%20in%20Xenopus%20laevis&amp;author=A.%20Hair&amp;author=Y.%20Vassetzky&amp;publication_year=2007&amp;journal=J.%20Cell%20Biochem.&amp;volume=102&amp;pages=1140-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17577215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bischof</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Purbey</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Functional interaction between PML and SATB1 regulates chromatin-loop architecture and transcription of the MHC class 1 locus</div>, <div class="source">Nat. Cell Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">45</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20interaction%20between%20PML%20and%20SATB1%20regulates%20chromatin-loop%20architecture%20and%20transcription%20of%20the%20MHC%20class%201%20locus&amp;author=P.%20Kumar&amp;author=O.%20Bischof&amp;author=P.%20Purbey&amp;publication_year=2007&amp;journal=Nat.%20Cell%20Biol.&amp;volume=9&amp;pages=45-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncb1516" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncb1516" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncb1516"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17173041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frisch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Frech</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Klingenhoff</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cartharius</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Liebich</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Werner</div> <div class="given-names">T.</div></div>. </div><div class="article-title">In silico prediction of scaffold/matrix attachment regions in large genomic sequences</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">349</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20prediction%20of%20scaffold%2Fmatrix%20attachment%20regions%20in%20large%20genomic%20sequences&amp;author=M.%20Frisch&amp;author=K.%20Frech&amp;author=A.%20Klingenhoff&amp;author=K.%20Cartharius&amp;author=I.%20Liebich&amp;author=T.%20Werner&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=349-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.206602. Article published online before print in January 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.206602. Article published online before print in January 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.206602. Article published online before print in January 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11827955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evans</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ott</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hansen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Koentges</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div>. </div><div class="article-title">A comparative study of S/MAR prediction tools</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">71</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20study%20of%20S%2FMAR%20prediction%20tools&amp;author=K.%20Evans&amp;author=S.%20Ott&amp;author=A.%20Hansen&amp;author=G.%20Koentges&amp;author=L.%20Wernisch&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-71" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-71" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-71"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17335576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ottaviani</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lever</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Mitter</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Reconfiguration of genomic anchors upon transcriptional activation of the human major histocompatibility complex</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1778</div>-<div class="lpage">1786</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconfiguration%20of%20genomic%20anchors%20upon%20transcriptional%20activation%20of%20the%20human%20major%20histocompatibility%20complex&amp;author=D.%20Ottaviani&amp;author=E.%20Lever&amp;author=R.%20Mitter&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1778-1786" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.082313.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.082313.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.082313.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18849521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Linnemann</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Platts</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Krawetz</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Differential nuclear scaffold/matrix attachment marks expressed genes</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">645</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20nuclear%20scaffold%2Fmatrix%20attachment%20marks%20expressed%20genes&amp;author=A.K.%20Linnemann&amp;author=A.E.%20Platts&amp;author=S.A.%20Krawetz&amp;publication_year=2009&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=18&amp;pages=645-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddn394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddn394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddn394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19017725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maya-Mendoza</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Positional mapping of specific DNA sequences relative to the nuclear substructure by direct polymerase chain reaction on nuclear matrix-bound templates</div>, <div class="source">Anal. Biochem.</div> , <div class="year">2003</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">196</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positional%20mapping%20of%20specific%20DNA%20sequences%20relative%20to%20the%20nuclear%20substructure%20by%20direct%20polymerase%20chain%20reaction%20on%20nuclear%20matrix-bound%20templates&amp;author=A.%20Maya-Mendoza&amp;author=A.%20Aranda-Anzaldo&amp;publication_year=2003&amp;journal=Anal.%20Biochem.&amp;volume=313&amp;pages=196-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0003-2697(02)00611-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0003-2697(02)00611-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0003-2697(02)00611-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12605856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Kramer</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Krawetz</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Mathematical model to predict regions of chromatin attachment to the nuclear matrix</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1419</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mathematical%20model%20to%20predict%20regions%20of%20chromatin%20attachment%20to%20the%20nuclear%20matrix&amp;author=G.B.%20Singh&amp;author=J.A.%20Kramer&amp;author=S.A.%20Krawetz&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=1419-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.7.1419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.7.1419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.7.1419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9060438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Drunen</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Sewalt</div> <div class="given-names">R.G.A.B.</div></div>,  <div class="name"><div class="surname">Oosterling</div> <div class="given-names">R.W.</div></div>,  <div class="name"><div class="surname">Weisbeek</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Smeekens</div> <div class="given-names">S.C.M.</div></div>,  <div class="name"><div class="surname">van Driel</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A bipartite sequence element associated with matrix/scaffold attachment regions</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">2924</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20bipartite%20sequence%20element%20associated%20with%20matrix%2Fscaffold%20attachment%20regions&amp;author=C.M.%20van%20Drunen&amp;author=R.G.A.B.%20Sewalt&amp;author=R.W.%20Oosterling&amp;author=P.J.%20Weisbeek&amp;author=S.C.M.%20Smeekens&amp;author=R.%20van%20Driel&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=2924-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.14.2924" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.14.2924" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.14.2924"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10390535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glazko</div> <div class="given-names">G.V.</div></div>,  <div class="name"><div class="surname">Rogozin</div> <div class="given-names">I.B.</div></div>,  <div class="name"><div class="surname">Glazkov</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">Comparative study and prediction of DNA fragments associated with various elements of the nuclear matrix</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="year">2001</div>, vol. <div class="volume">1517</div> (pg. <div class="fpage">351</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20study%20and%20prediction%20of%20DNA%20fragments%20associated%20with%20various%20elements%20of%20the%20nuclear%20matrix&amp;author=G.V.%20Glazko&amp;author=I.B.%20Rogozin&amp;author=M.V.%20Glazkov&amp;publication_year=2001&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1517&amp;pages=351-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-4781(00)00297-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-4781(00)00297-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-4781(00)00297-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11342213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">M.M.</div></div>. </div>, <div class="source">Lehninger Principles of Biochemistry</div> , <div class="year">2000</div><div class="edition">3rd edition</div><div class="publisher-loc">New York</div><div class="publisher-name">Worth Publishers</div>(pg. <div class="fpage">915</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSP027C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Calladine</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Drew</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Luisi</div> <div class="given-names">B.F.</div></div>,  <div class="name"><div class="surname">Travers</div> <div class="given-names">A.A.</div></div>. </div>, <div class="source">Understanding DNA</div> , <div class="year">2004</div><div class="edition">3rd edition</div><div class="publisher-loc">London</div><div class="publisher-name">Elsevier-Academic Press</div>(pg. <div class="fpage">94</div>-<div class="lpage">138</div>)</div></div></div></div><div content-id="DSP027C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bode</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kowhi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Dickinson</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Biological significance of unwinding capability of nuclear matrix-associating DNAs</div>, <div class="source">Science</div> , <div class="year">1992</div>, vol. <div class="volume">255</div> (pg. <div class="fpage">195</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biological%20significance%20of%20unwinding%20capability%20of%20nuclear%20matrix-associating%20DNAs&amp;author=J.%20Bode&amp;author=Y.%20Kowhi&amp;author=L.%20Dickinson&amp;publication_year=1992&amp;journal=Science&amp;volume=255&amp;pages=195-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1553545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1553545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1553545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1553545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benham</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kohwi-Shigematsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bode</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Stress-induced duplex destabilization in scaffold/matrix attachment regions</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">2774</div> (pg. <div class="fpage">181</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stress-induced%20duplex%20destabilization%20in%20scaffold%2Fmatrix%20attachment%20regions&amp;author=C.%20Benham&amp;author=T.%20Kohwi-Shigematsu&amp;author=J.%20Bode&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=2774&amp;pages=181-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.1385" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.1385" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.1385"> </span></div> </div></div></div></div></div><div content-id="DSP027C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maya-Mendoza</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hernández-Muñoz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gariglio</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Gene positional changes relative to the nuclear substructure during carbon tetrachloride-induced hepatic fibrosis in rats</div>, <div class="source">J. Cell Biochem.</div> , <div class="year">2004</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">1084</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20positional%20changes%20relative%20to%20the%20nuclear%20substructure%20during%20carbon%20tetrachloride-induced%20hepatic%20fibrosis%20in%20rats&amp;author=A.%20Maya-Mendoza&amp;author=R.%20Hern%C3%A1ndez-Mu%C3%B1oz&amp;author=P.%20Gariglio&amp;author=A.%20Aranda-Anzaldo&amp;publication_year=2004&amp;journal=J.%20Cell%20Biochem.&amp;volume=93&amp;pages=1084-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jcb.20264" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjcb.20264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjcb.20264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15449316" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewin</div> <div class="given-names">B.</div></div>. </div>, <div class="source">Gene Expression 2</div> , <div class="year">1980</div><div class="edition">2nd edition</div><div class="publisher-loc">New York</div><div class="publisher-name">John Wiley and Sons</div>(pg. <div class="fpage">360</div>-<div class="lpage">2</div>)</div></div></div></div><div content-id="DSP027C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gromova</div> <div class="given-names">I.I.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">O.F.</div></div>,  <div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">Long-range fragmentation of the eukaryotic genome by exogenous and endogenous nucleases proceeds in a specific fashion via preferential DNA cleavage at matrix attachment sites</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">18685</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-range%20fragmentation%20of%20the%20eukaryotic%20genome%20by%20exogenous%20and%20endogenous%20nucleases%20proceeds%20in%20a%20specific%20fashion%20via%20preferential%20DNA%20cleavage%20at%20matrix%20attachment%20sites&amp;author=I.I.%20Gromova&amp;author=O.F.%20Nielsen&amp;author=S.V.%20Razin&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem.&amp;volume=270&amp;pages=18685-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.31.18685" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.31.18685" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.31.18685"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7629201" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jin</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Spear</div> <div class="given-names">B.T.</div></div>. </div><div class="article-title">The alpha-fetoprotein enhancer region activates the albumin and alpha-fetoprotein promoters during liver development</div>, <div class="source">Dev. Biol.</div> , <div class="year">2009</div> <div class="comment">doi:10.1016/j.dbio.2009.09.026</div></div></div></div></div><div content-id="DSP027C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">DNA interaction with the nuclear matrix and spatial organization of replication and transcription</div>, <div class="source">BioEssays</div> , <div class="year">1987</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">19</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20interaction%20with%20the%20nuclear%20matrix%20and%20spatial%20organization%20of%20replication%20and%20transcription&amp;author=S.V.%20Razin&amp;publication_year=1987&amp;journal=BioEssays&amp;volume=6&amp;pages=19-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3551943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rollini</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Namciu</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Marsden</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Fournier</div> <div class="given-names">R.E.K.</div></div>. </div><div class="article-title">Identification and characterization of nuclear matrix-attachment regions in the human serpin gene cluster at 14q32.1</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">3779</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20nuclear%20matrix-attachment%20regions%20in%20the%20human%20serpin%20gene%20cluster%20at%2014q32.1&amp;author=P.%20Rollini&amp;author=S.J.%20Namciu&amp;author=M.D.%20Marsden&amp;author=R.E.K.%20Fournier&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=3779-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.19.3779" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.19.3779" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.19.3779"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10481016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gromova</div> <div class="given-names">I.I.</div></div>,  <div class="name"><div class="surname">Thomsen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">Different topoisomerase II antitumor drugs direct similar specific long-range fragmentation of an amplified c-myc gene locus in living cells and in high-salt-extracted nuclei</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1995</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">102</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20topoisomerase%20II%20antitumor%20drugs%20direct%20similar%20specific%20long-range%20fragmentation%20of%20an%20amplified%20c-myc%20gene%20locus%20in%20living%20cells%20and%20in%20high-salt-extracted%20nuclei&amp;author=I.I.%20Gromova&amp;author=B.%20Thomsen&amp;author=S.V.%20Razin&amp;publication_year=1995&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=92&amp;pages=102-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.92.1.102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.92.1.102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.92.1.102"> </span></div> </div></div></div></div></div><div content-id="DSP027C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davie</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Histone modifications, chromatin structure and the nuclear matrix</div>, <div class="source">J. Cell Biochem.</div> , <div class="year">1996</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">149</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Histone%20modifications%2C%20chromatin%20structure%20and%20the%20nuclear%20matrix&amp;author=J.R.%20Davie&amp;publication_year=1996&amp;journal=J.%20Cell%20Biochem.&amp;volume=62&amp;pages=149-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8844394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kantidze</div> <div class="given-names">O.L.</div></div>,  <div class="name"><div class="surname">Razin</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">Chromatin loops, illegitimate recombination and genome evolution</div>, <div class="source">BioEssays</div> , <div class="year">2009</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">278</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20loops%2C%20illegitimate%20recombination%20and%20genome%20evolution&amp;author=O.L.%20Kantidze&amp;author=S.V.%20Razin&amp;publication_year=2009&amp;journal=BioEssays&amp;volume=31&amp;pages=278-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bies.v31:3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbies.v31:3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbies.v31:3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19260023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lazarevich</div> <div class="given-names">N.L.</div></div>. </div><div class="article-title">Molecular mechanisms of alpha-fetoprotein gene expression</div>, <div class="source">Biochemistry (Mosc.)</div> , <div class="year">2000</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">117</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20alpha-fetoprotein%20gene%20expression&amp;author=N.L.%20Lazarevich&amp;publication_year=2000&amp;journal=Biochemistry%20(Mosc.)&amp;volume=65&amp;pages=117-33" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10702646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pinkert</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Ornitz</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Brinster</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Palmiter</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">An albumin enhancer located 10 kb upstream functions along with its promoter to direct efficient, liver-specific expression in transgenic mice</div>, <div class="source">Genes Dev.</div> , <div class="year">1987</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">268</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20albumin%20enhancer%20located%2010%20kb%20upstream%20functions%20along%20with%20its%20promoter%20to%20direct%20efficient%2C%20liver-specific%20expression%20in%20transgenic%20mice&amp;author=C.A.%20Pinkert&amp;author=D.M.%20Ornitz&amp;author=R.L.%20Brinster&amp;author=R.D.%20Palmiter&amp;publication_year=1987&amp;journal=Genes%20Dev.&amp;volume=1&amp;pages=268-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1.3.268" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1.3.268" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1.3.268"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3678824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitchell</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Transcription factories are nuclear subcompartments that remain in the absence of transcription</div>, <div class="source">Genes Dev.</div> , <div class="year">2008</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">20</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factories%20are%20nuclear%20subcompartments%20that%20remain%20in%20the%20absence%20of%20transcription&amp;author=J.A.%20Mitchell&amp;author=P.%20Fraser&amp;publication_year=2008&amp;journal=Genes%20Dev.&amp;volume=22&amp;pages=20-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.454008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.454008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.454008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18172162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Pombo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Replicon clusters are stable units of chromosome structure: evidence that nuclear organization contributes to the efficient activation and propagation of S phase in human cells</div>, <div class="source">J. Cell Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">140</div> (pg. <div class="fpage">1285</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replicon%20clusters%20are%20stable%20units%20of%20chromosome%20structure%3A%20evidence%20that%20nuclear%20organization%20contributes%20to%20the%20efficient%20activation%20and%20propagation%20of%20S%20phase%20in%20human%20cells&amp;author=D.A.%20Jackson&amp;author=A.%20Pombo&amp;publication_year=1998&amp;journal=J.%20Cell%20Biol.&amp;volume=140&amp;pages=1285-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.140.6.1285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.140.6.1285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.140.6.1285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9508763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dimitrova</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The spatio-temporal organization of DNA replication sites is identical in primary, immortalized and transformed mammalian cells</div>, <div class="source">J. Cell Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">4037</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20spatio-temporal%20organization%20of%20DNA%20replication%20sites%20is%20identical%20in%20primary%2C%20immortalized%20and%20transformed%20mammalian%20cells&amp;author=D.S.%20Dimitrova&amp;author=R.%20Berezney&amp;publication_year=2002&amp;journal=J.%20Cell%20Sci.&amp;volume=115&amp;pages=4037-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.00087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.00087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.00087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12356909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osborne</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Chakalova</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Mitchell</div> <div class="given-names">J.A.</div></div>, et al. </div><div class="article-title">Myc dynamically and preferentially relocates to a transcription factory occupied by Igh</div>, <div class="source">PLoS Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e192</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Myc%20dynamically%20and%20preferentially%20relocates%20to%20a%20transcription%20factory%20occupied%20by%20Igh&amp;author=C.S.%20Osborne&amp;author=L.%20Chakalova&amp;author=J.A.%20Mitchell&amp;publication_year=2007&amp;journal=PLoS%20Biol.&amp;volume=5&amp;pages=e192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0050192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0050192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17622196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Linnemann</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Krawetz</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Maintenance of a functional higher order chromatin structure: the role of the nuclear matrix in normal and disease states</div>, <div class="source">Gene Ther. Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">231</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maintenance%20of%20a%20functional%20higher%20order%20chromatin%20structure%3A%20the%20role%20of%20the%20nuclear%20matrix%20in%20normal%20and%20disease%20states&amp;author=A.K.%20Linnemann&amp;author=S.A.%20Krawetz&amp;publication_year=2009&amp;journal=Gene%20Ther.%20Mol.%20Biol.&amp;volume=13&amp;pages=231-43" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20948980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martínez-Ramos</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Maya-Mendoza</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gariglio</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A global but stable change in HeLa cell morphology induces reorganization of DNA structural loop domains within the cell nucleus</div>, <div class="source">J. Cell Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">79</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20but%20stable%20change%20in%20HeLa%20cell%20morphology%20induces%20reorganization%20of%20DNA%20structural%20loop%20domains%20within%20the%20cell%20nucleus&amp;author=I.%20Mart%C3%ADnez-Ramos&amp;author=A.%20Maya-Mendoza&amp;author=P.%20Gariglio&amp;author=A.%20Aranda-Anzaldo&amp;publication_year=2005&amp;journal=J.%20Cell%20Biochem.&amp;volume=96&amp;pages=79-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16052476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aranda-Anzaldo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A structural basis for cellular senescence</div>, <div class="source">Aging</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">598</div>-<div class="lpage">607</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20structural%20basis%20for%20cellular%20senescence&amp;author=A.%20Aranda-Anzaldo&amp;publication_year=2009&amp;journal=Aging&amp;volume=1&amp;pages=598-607" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.18632/aging.v1i7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.18632%2Faging.v1i7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.18632%2Faging.v1i7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20157542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lemaitre</div> <div class="given-names">J-M.</div></div>,  <div class="name"><div class="surname">Danis</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pasero</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Vassetzky</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mechali</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mitotic remodelling of the replicon and chromosome structure</div>, <div class="source">Cell</div> , <div class="year">2005</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">787</div>-<div class="lpage">801</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitotic%20remodelling%20of%20the%20replicon%20and%20chromosome%20structure&amp;author=J-M.%20Lemaitre&amp;author=E.%20Danis&amp;author=P.%20Pasero&amp;author=Y.%20Vassetzky&amp;author=M.%20Mechali&amp;publication_year=2005&amp;journal=Cell&amp;volume=123&amp;pages=787-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2005.08.045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2005.08.045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2005.08.045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16325575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP027C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP027C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennett</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">The time and duration of meiosis</div>, <div class="source">Philos. Trans. R. Soc. Lond. B, Biol. Sci.</div> , <div class="year">1977</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">201</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20time%20and%20duration%20of%20meiosis&amp;author=M.D.%20Bennett&amp;publication_year=1977&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%2C%20Biol.%20Sci.&amp;volume=277&amp;pages=201-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.1977.0012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1977.0012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.1977.0012"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Mitsuo Oshimura</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347368&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp027/2/dsp027-dsp027supp.pdf?Expires=2147483647&amp;Signature=rLEFsjjmoB41-FLB1TjjxjkPmXP2zlkuwLy1130nMQfOm6uVFR28nBF8OIS8OF4r46eqBj4UIDJw~tOcDROkAADVj996bn0dNod9DCnpyVZcMJNaRjyZV72TjGw9676E2jghxpTj7~0uUCHqgN6sx0o20jDIotCBZxleklipdv~sYFW~275Gi2fqcAgJjdsUroksto3b6Tn45BhylyPJrC-JrWUH8EKke6~mpT60UiWxGGtsPFJuZXNiDf1Ql3nQlFqVt-U7zgYGezv914zBfEGgeBXfKviTry5rcdFSyeo32X8aAhDQtmJVIlKO6yIsKXOyxKLt5Io7NZ4KMyYd1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">205</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000275264300003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp027"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347368"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000275264300003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f1%2f23%2f347368">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000275264300003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F1%2F23%2F347368" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347368'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Determination of the in vivo structural DNA loop organization in the genomic region of the rat albumin locus by means of a topological approach" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp027");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f23%2f347368">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

