

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/2/115/1745337","@type":"ScholarlyArticle","name":"High-density linkage mapping and distribution of segregation distortion regions in the oak genome","datePublished":"2016-03-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw001","keywords":["high-density linkage map","SNP","segregation distortion","reproductive barriers","<italic>Quercus</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/2/115/1745337","author":[{"name":"Bodénès, Catherine","affiliation":"1  INRA, UMR1202 BIOGECO ,  F-33610 Cestas ,  France    2  Université de Bordeaux, UMR1202 BIOGECO ,  F-33610 Talence ,  France","@type":"Person"},{"name":"Chancerel, Emilie","affiliation":"1  INRA, UMR1202 BIOGECO ,  F-33610 Cestas ,  France    2  Université de Bordeaux, UMR1202 BIOGECO ,  F-33610 Talence ,  France","@type":"Person"},{"name":"Ehrenmann, François","affiliation":"1  INRA, UMR1202 BIOGECO ,  F-33610 Cestas ,  France    2  Université de Bordeaux, UMR1202 BIOGECO ,  F-33610 Talence ,  France","@type":"Person"},{"name":"Kremer, Antoine","affiliation":"1  INRA, UMR1202 BIOGECO ,  F-33610 Cestas ,  France    2  Université de Bordeaux, UMR1202 BIOGECO ,  F-33610 Talence ,  France","@type":"Person"},{"name":"Plomion, Christophe","affiliation":"1  INRA, UMR1202 BIOGECO ,  F-33610 Cestas ,  France    2  Université de Bordeaux, UMR1202 BIOGECO ,  F-33610 Talence ,  France","@type":"Person"}],"description":"Abstract.  We developed the densest single-nucleotide polymorphism (SNP)-based linkage genetic map to date for the genus Quercus. An 8k gene-based SNP array was","pageStart":"115","pageEnd":"124","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-density linkage mapping and distribution of segregation distortion regions in the oak genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-density linkage mapping and distribution of segregation distortion regions in the oak genome" />
    <meta property="og:description" content="Abstract.  We developed the densest single-nucleotide polymorphism (SNP)-based linkage genetic map to date for the genus Quercus. An 8k gene-based SNP array was" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/2/115/1745337" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-03-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Bodénès, Catherine" /><meta name="citation_author_institution" content="INRA, UMR1202 BIOGECO, F-33610 Cestas, France" /><meta name="citation_author_institution" content="Université de Bordeaux, UMR1202 BIOGECO, F-33610 Talence, France" /><meta name="citation_author" content="Chancerel, Emilie" /><meta name="citation_author_institution" content="INRA, UMR1202 BIOGECO, F-33610 Cestas, France" /><meta name="citation_author_institution" content="Université de Bordeaux, UMR1202 BIOGECO, F-33610 Talence, France" /><meta name="citation_author" content="Ehrenmann, François" /><meta name="citation_author_institution" content="INRA, UMR1202 BIOGECO, F-33610 Cestas, France" /><meta name="citation_author_institution" content="Université de Bordeaux, UMR1202 BIOGECO, F-33610 Talence, France" /><meta name="citation_author" content="Kremer, Antoine" /><meta name="citation_author_institution" content="INRA, UMR1202 BIOGECO, F-33610 Cestas, France" /><meta name="citation_author_institution" content="Université de Bordeaux, UMR1202 BIOGECO, F-33610 Talence, France" /><meta name="citation_author" content="Plomion, Christophe" /><meta name="citation_author_institution" content="INRA, UMR1202 BIOGECO, F-33610 Cestas, France" /><meta name="citation_author_institution" content="Université de Bordeaux, UMR1202 BIOGECO, F-33610 Talence, France" /><meta name="citation_title" content="High-density linkage mapping and distribution of segregation distortion regions in the oak genome" /><meta name="citation_firstpage" content="115" /><meta name="citation_lastpage" content="124" /><meta name="citation_doi" content="10.1093/dnares/dsw001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2016/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Evidence for hybridization and introgression within a species-rich oak (Quercus spp.) community;  citation_journal_title=BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=218" /><meta name="citation_reference" content="citation_title=Outlier loci highlight the direction of introgression in oaks;  citation_journal_title=Mol. Ecol.;  citation_year=2013;  citation_volume=22;  citation_pages=450-62; " /><meta name="citation_reference" content="citation_title=Strength and variability of postmating reproductive isolating barriers between four European white oak species;  citation_journal_title=Tree Genet. Genomics;  citation_year=2013;  citation_volume=9;  citation_pages=841-53; " /><meta name="citation_reference" content="citation_title=A linkage disequilibrium perspective on the genetic mosaic of speciation in two hybridizing Mediterranean white oaks;  citation_journal_title=Heredity;  citation_year=2015;  citation_volume=114;  citation_pages=373-86; " /><meta name="citation_reference" content="citation_title=Leaf morphological differentiation between Quercus robur and Quercus petraea is stable across western European mixed oak stands;  citation_journal_title=Ann. For. Sci.;  citation_year=2002;  citation_volume=59;  citation_pages=777-87; " /><meta name="citation_reference" content="citation_title=Differences in morphological and physiological responses to water-logging between two sympatric oak species (Quercus petraea [Matt.] Liebl., Quercus robur L.);  citation_journal_title=Ann. Sci. For.;  citation_year=2006;  citation_volume=63;  citation_pages=849-59; " /><meta name="citation_reference" content="citation_title=Le point sur les chênes blancs européens;  citation_journal_title=Forêt Entreprise;  citation_year=1996;  citation_volume=112;  citation_pages=49-56; " /><meta name="citation_reference" content="citation_title=Carbon isotope discrimination and wood anatomy variations in mixed stands of Quercus robur and Quercus petraea;  citation_journal_title=Plant Cell Environment;  citation_year=2001;  citation_volume=24;  citation_pages=861-8; " /><meta name="citation_reference" content="citation_title=Hybridization as a mechanism of invasion in oaks;  citation_journal_title=New Phytol.;  citation_year=2003;  citation_volume=161;  citation_pages=151-64; " /><meta name="citation_reference" content="citation_title=Mating system and asymetric hybridization in a mixed stand of European oaks;  citation_journal_title=Evolution;  citation_year=1996;  citation_volume=50;  citation_pages=900-8; " /><meta name="citation_reference" content="citation_title=Comparative mapping between Quercus and Castanea using simple-sequence repeats (SSRs);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=558-66; " /><meta name="citation_reference" content="citation_title=Genome scanning for interspecific differentiation between two closely related oak species [Quercus robur L. and Q. petraea (Matt.) Liebl.];  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=1615-26; " /><meta name="citation_reference" content="citation_title=Comparative mapping in the Fagaceae and beyond using EST-SSRs;  citation_journal_title=BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=153" /><meta name="citation_reference" content="citation_title=Single-nucleotide polymorphism discovery and validation in high density SNP array for genetic analysis in European white oaks;  citation_journal_title=Mol. Ecol. Res.;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=Construction of a high-density composite map and comparative mapping of segregation distortion regions in barley;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2010;  citation_volume=284;  citation_pages=319-31; " /><meta name="citation_reference" content="citation_title=A reference genetic map of C. clementina hort. ex Tan.; citrus evolution inferences from comparative mapping;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=593" /><meta name="citation_reference" content="citation_title=Genetics of postzygotic isolation in Eucalyptus: whole-genome analysis of barriers to introgression in a wide interspecific cross of Eucalyptus grandis and E. globulus;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=1405-18; " /><meta name="citation_reference" content="citation_title=Chromosomal regions associated with segregation distortion in maize;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2002;  citation_volume=105;  citation_pages=622-8; " /><meta name="citation_reference" content="citation_title=Chromosomal regions associated with segregation distortion of molecular markers in F2, backcross, doubled haploid, and recombinant inbred populations in rice (Oryza sativa L.);  citation_journal_title=Mol. Gen. Genet.;  citation_year=1997;  citation_volume=253;  citation_pages=535-45; " /><meta name="citation_reference" content="citation_title=Evidence for Dobzhansky-Muller incompatibilities contributing to the sterility of hybrids between Mimulus guttatus and M. nasatus;  citation_journal_title=Evolution;  citation_year=2001;  citation_volume=55;  citation_pages=1932-42; " /><meta name="citation_reference" content="citation_title=Large-scale heterospecific segregation distortion in Populus revealed by a dense genetic map;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=451-63; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato;  citation_journal_title=DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Genetic analysis of segregation distortion of SSR markers in F2 population of barley;  citation_journal_title=J. Agric. Sci.;  citation_year=2010;  citation_volume=3;  citation_pages=172-7; " /><meta name="citation_reference" content="citation_title=Patterns of reproductive isolation in three Angiosperm genera;  citation_journal_title=Evolution;  citation_year=2015;  citation_volume=58;  citation_pages=1195-208; " /><meta name="citation_reference" content="citation_title=Hybrid sterility in plant: stories from rice;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2010;  citation_volume=13;  citation_pages=186-92; " /><meta name="citation_reference" content="citation_title=Molecular evolution and genetics of postzygotic reproductive isolation in plants;  citation_journal_title=Biol. Rep.;  citation_year=2012;  citation_volume=4;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=Genetic load and its causes in long-lived plants;  citation_journal_title=Trees;  citation_year=1988;  citation_volume=2;  citation_pages=195-203; " /><meta name="citation_reference" content="citation_title=Inbreeding depression in conifers: implications for breeding strategy;  citation_journal_title=For. Sci.;  citation_year=1996;  citation_volume=42;  citation_pages=102-17; " /><meta name="citation_reference" content="citation_title=A novel meiotic drive locus almost completely distorts segregation in Mimulus (Monkeyflower) hybrids;  citation_journal_title=Genetics;  citation_year=2005;  citation_volume=169;  citation_pages=347-53; " /><meta name="citation_reference" content="citation_title=Results of species hybridization with Quercus robur L. and Quercus petraea (Matt.) Liebl.;  citation_journal_title=Ann. Sci. For.;  citation_year=1993;  citation_volume=50;  citation_pages=137s-43s; " /><meta name="citation_reference" content="citation_title=Bioinformatic analysis of ESTs collected by Sanger and pyrosequencing methods for a keustone forest tree species: oak;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=650" /><meta name="citation_reference" content="citation_publisher=Wageningen, Netherlands;  citation_title=Joinmap® 4, Software for the Calculation of Genetic Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs;  citation_journal_title=J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=LPmerge: an R package for merging genetic maps by linear programming;  citation_journal_title=BioInformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1623-4; " /><meta name="citation_reference" content="citation_title=Estimation of recombination frequencies and construction of RFLP linkage maps in plants from crosses between heterozygous parents;  citation_journal_title=Genetics;  citation_year=1990;  citation_volume=125;  citation_pages=645-54; " /><meta name="citation_reference" content="citation_title=An integrated genetic map based on four mapping populations and quantitative trait loci associated with economically important traits in watermelon (Citrullus lanatus);  citation_journal_title=BMC Plant Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=33" /><meta name="citation_reference" content="citation_title=Detection of segregation distortion loci in triticale (x Triticosecale Wittmack) based on a high-density DArT marker consensus genetic linkage map;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=380" /><meta name="citation_reference" content="citation_title=Development of a 10,000 locus genetic map of the sunflower genome based on multiple crosses;  citation_journal_title=G3;  citation_year=2012;  citation_volume=2;  citation_pages=721-9; " /><meta name="citation_reference" content="citation_title=Genome-wide distribution of genetic diversity and linkage disequilibrium in a mass-selected population of maritime pine;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=171" /><meta name="citation_reference" content="citation_title=Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish;  citation_journal_title=DNA Res.;  citation_year=2015;  citation_volume=22;  citation_pages=39-52; " /><meta name="citation_reference" content="citation_title=An improved consensus linkage map of Barley based on flow-sorted chromosomes and Single Nucleotide Polymorphism markers;  citation_journal_title=Plant Genome;  citation_year=2011;  citation_volume=4;  citation_pages=238-49; " /><meta name="citation_reference" content="citation_title=The USDA barley core collection: genetic diversity, population structure, and potential for genome-wide association studies;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Evidence of intense chromosomal shuffling during conifer evolution;  citation_journal_title=Genome Biol. Evol.;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=A consensus map for Ug99 stem rust resistance loci in wheat;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2014;  citation_volume=127;  citation_pages=1561-81; " /><meta name="citation_reference" content="citation_title=The oak gene expression atlas: insights into Fagaceae genome evolution and the discovery of genes regulated during bud dormancy release;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=112" /><meta name="citation_reference" content="citation_title=Decoding the oak genome: public release of sequence data, assembly, annotation and publication strategies;  citation_journal_title=Mol. Ecol. Res.;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discover y and genotyping for constructing a saturated link age map of Chickpea (Cicer arietinum L.);  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=357-73; " /><meta name="citation_reference" content="citation_title=Saturated linkage map construction in Rubus idaeus using genotyping by sequencing and genome-independent imputation;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=2" /><meta name="citation_reference" content="citation_title=Genomic characterization of DArT markers based on high-density linkage analysis and physical mapping to the Eucalyptus genome;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=9" /><meta name="citation_reference" content="citation_title=Comparative genomic and population genetic analysis indicate highly porous genomes and high levels of gene flow between divergent Helianthus species;  citation_journal_title=Evolution;  citation_year=2009;  citation_volume=63;  citation_pages=2061-75; " /><meta name="citation_reference" content="citation_title=Whole genome comparisons of Fragaria, Prunus and Malus reveal different modes of evolution between Rosaceous subfamilies;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=129" /><meta name="citation_reference" content="citation_title=High-density linkage mapping in a pine tree reveals a genomic region associated with inbreeding depression and provides clues to the extent and distribution of meiotic recombination;  citation_journal_title=BMC Biol.;  citation_year=2013;  citation_volume=11;  citation_pages=50" /><meta name="citation_reference" content="citation_title=Construction of an integrated high density Simple Sequence Repeat linkage map in cultivated strawberry (Fragaria 3 ananassa) and its applicability;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=79-92; " /><meta name="citation_reference" content="citation_title=High-resolution genetic linkage maps of Eucalyptus improve Eucalyptus grandis genome assembly;  citation_journal_title=New Phytol.;  citation_year=2014;  citation_volume=206;  citation_pages=1283-96; " /><meta name="citation_reference" content="citation_title=High density SNP mapping and QTL analysis for fruit quality characteristics in peach (Prunus persica L.);  citation_journal_title=Tree Genet. Genomes;  citation_year=2013;  citation_volume=9;  citation_pages=19-36; " /><meta name="citation_reference" content="citation_title=Gains in QTL detection using an ultra-high density SNPmap based on population sequencing relative to traditional RFLP/SSR markers;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e17595" /><meta name="citation_reference" content="citation_title=Chromosomal rearrangements and the genetics of reproductive barriers in Mimulus (Monkeyflowers);  citation_journal_title=Evolution;  citation_year=2013;  citation_volume=6;  citation_pages=2547-56; " /><meta name="citation_reference" content="citation_title=Comparison of quantitative trait loci for adaptive traits between oak and chestnut based on an expressed sequence tag consensus map;  citation_journal_title=Genetics;  citation_year=2006;  citation_volume=172;  citation_pages=533-46; " /><meta name="citation_reference" content="citation_title=Decoding plant and animal genome plasticity from differential paleo-evolutionary patterns and processes;  citation_journal_title=Genome Biol. Evol.;  citation_year=2012;  citation_volume=4;  citation_pages=917-28; " /><meta name="citation_reference" content="citation_title=Gene diversity in natural populations of oak species;  citation_journal_title=Ann. Sci. For.;  citation_year=1993;  citation_volume=50;  citation_issue=(Suppl. 1);  citation_pages=186s-202s; " /><meta name="citation_reference" content="citation_title=High rates of gene flow by pollen and seed in oak populations across Europe;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e85130" /><meta name="citation_reference" content="citation_title=High genetic load in the pacific oyster Crassostrea gigas;  citation_journal_title=Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=255-65; " /><meta name="citation_reference" content="citation_title=High-density genetic linkage maps with over 2,400 sequence-anchored DArT markers for genetic dissection in an F2 pseudo-backcross of Eucalyptus grandis × E. urophylla;  citation_journal_title=Tree Genet. Genomes;  citation_year=2012;  citation_volume=8;  citation_pages=163-75; " /><meta name="citation_reference" content="citation_title=Strength, diversity and plasticity of postmating reproductive barriers between two hybridizing oak species (Quercus robur L. and Quercus petraea (Matt) Liebl.);  citation_journal_title=J. Evol. Biol.;  citation_year=2012;  citation_volume=25;  citation_pages=157-73; " /><meta name="citation_reference" content="citation_title=Species relative abundance and direction of introgression in oaks;  citation_journal_title=Mol. Ecol.;  citation_year=2008;  citation_volume=18;  citation_pages=2228-42; " /><meta name="citation_reference" content="citation_title=Reproductive patterns shape introgression dynamics and species succession within the European white oak species complex;  citation_journal_title=Evolution;  citation_year=2011;  citation_volume=1;  citation_pages=156-70; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci controlling water use efficiency and related traits in Quercus robur L.;  citation_journal_title=Tree Genet. Genomes;  citation_year=2008;  citation_volume=4;  citation_pages=263-78; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci of tolerance to waterlogging in a European oak (Quercus robur L.): physiological relevance and temporal effect patterns;  citation_journal_title=Plant, Cell Environment;  citation_year=2007;  citation_volume=30;  citation_pages=422-34; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/2/115/17462972/dsw001.pdf" /><meta name="description" content="Abstract.  We developed the densest single-nucleotide polymorphism (SNP)-based linkage genetic map to date for the genus Quercus. An 8k gene-based SNP array was" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/2/115/1745337" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/2/115/1745337" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/2/115/1745337");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw001';
    </script>
<title>
	High-density linkage mapping and distribution of segregation distortion regions in the oak genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745337" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbChATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm9kw6luw6hzLCBDYXRoZXJpbmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBVTVIxMjAyIEJJT0dFQ08sIEYtMzM2MTAgQ2VzdGFzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaXTDqSBkZSBCb3JkZWF1eCwgVU1SMTIwMiBCSU9HRUNPLCBGLTMzNjEwIFRhbGVuY2UsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoYW5jZXJlbCwgRW1pbGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVU1SMTIwMiBCSU9HRUNPLCBGLTMzNjEwIENlc3RhcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0w6kgZGUgQm9yZGVhdXgsIFVNUjEyMDIgQklPR0VDTywgRi0zMzYxMCBUYWxlbmNlLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJFaHJlbm1hbm4sIEZyYW7Dp29pcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIFVNUjEyMDIgQklPR0VDTywgRi0zMzYxMCBDZXN0YXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdMOpIGRlIEJvcmRlYXV4LCBVTVIxMjAyIEJJT0dFQ08sIEYtMzM2MTAgVGFsZW5jZSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3JlbWVyLCBBbnRvaW5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVU1SMTIwMiBCSU9HRUNPLCBGLTMzNjEwIENlc3RhcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0w6kgZGUgQm9yZGVhdXgsIFVNUjEyMDIgQklPR0VDTywgRi0zMzYxMCBUYWxlbmNlLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQbG9taW9uLCBDaHJpc3RvcGhlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVU1SMTIwMiBCSU9HRUNPLCBGLTMzNjEwIENlc3RhcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0w6kgZGUgQm9yZGVhdXgsIFVNUjEyMDIgQklPR0VDTywgRi0zMzYxMCBUYWxlbmNlLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2gtZGVuc2l0eSBsaW5rYWdlIG1hcHBpbmcgYW5kIGRpc3RyaWJ1dGlvbiBvZiBzZWdyZWdhdGlvbiBkaXN0b3J0aW9uIHJlZ2lvbnMgaW4gdGhlIG9hayBnZW5vbWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZm9yIGh5YnJpZGl6YXRpb24gYW5kIGludHJvZ3Jlc3Npb24gd2l0aGluIGEgc3BlY2llcy1yaWNoIG9hayAoUXVlcmN1cyBzcHAuKSBjb21tdW5pdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3V0bGllciBsb2NpIGhpZ2hsaWdodCB0aGUgZGlyZWN0aW9uIG9mIGludHJvZ3Jlc3Npb24gaW4gb2FrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQ1MC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyZW5ndGggYW5kIHZhcmlhYmlsaXR5IG9mIHBvc3RtYXRpbmcgcmVwcm9kdWN0aXZlIGlzb2xhdGluZyBiYXJyaWVycyBiZXR3ZWVuIGZvdXIgRXVyb3BlYW4gd2hpdGUgb2FrIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz04NDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbGlua2FnZSBkaXNlcXVpbGlicml1bSBwZXJzcGVjdGl2ZSBvbiB0aGUgZ2VuZXRpYyBtb3NhaWMgb2Ygc3BlY2lhdGlvbiBpbiB0d28gaHlicmlkaXppbmcgTWVkaXRlcnJhbmVhbiB3aGl0ZSBvYWtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz0zNzMtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlYWYgbW9ycGhvbG9naWNhbCBkaWZmZXJlbnRpYXRpb24gYmV0d2VlbiBRdWVyY3VzIHJvYnVyIGFuZCBRdWVyY3VzIHBldHJhZWEgaXMgc3RhYmxlIGFjcm9zcyB3ZXN0ZXJuIEV1cm9wZWFuIG1peGVkIG9hayBzdGFuZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gRm9yLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9Nzc3LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbmNlcyBpbiBtb3JwaG9sb2dpY2FsIGFuZCBwaHlzaW9sb2dpY2FsIHJlc3BvbnNlcyB0byB3YXRlci1sb2dnaW5nIGJldHdlZW4gdHdvIHN5bXBhdHJpYyBvYWsgc3BlY2llcyAoUXVlcmN1cyBwZXRyYWVhIFtNYXR0Ll0gTGllYmwuLCBRdWVyY3VzIHJvYnVyIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBTY2kuIEZvci47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz04NDktNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlIHBvaW50IHN1ciBsZXMgY2jDqm5lcyBibGFuY3MgZXVyb3DDqWVuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rm9yw6p0IEVudHJlcHJpc2U7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9NDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcmJvbiBpc290b3BlIGRpc2NyaW1pbmF0aW9uIGFuZCB3b29kIGFuYXRvbXkgdmFyaWF0aW9ucyBpbiBtaXhlZCBzdGFuZHMgb2YgUXVlcmN1cyByb2J1ciBhbmQgUXVlcmN1cyBwZXRyYWVhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIEVudmlyb25tZW50OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9ODYxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5YnJpZGl6YXRpb24gYXMgYSBtZWNoYW5pc20gb2YgaW52YXNpb24gaW4gb2FrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MTUxLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXRpbmcgc3lzdGVtIGFuZCBhc3ltZXRyaWMgaHlicmlkaXphdGlvbiBpbiBhIG1peGVkIHN0YW5kIG9mIEV1cm9wZWFuIG9ha3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTkwMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBtYXBwaW5nIGJldHdlZW4gUXVlcmN1cyBhbmQgQ2FzdGFuZWEgdXNpbmcgc2ltcGxlLXNlcXVlbmNlIHJlcGVhdHMgKFNTUnMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTU1OC02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNjYW5uaW5nIGZvciBpbnRlcnNwZWNpZmljIGRpZmZlcmVudGlhdGlvbiBiZXR3ZWVuIHR3byBjbG9zZWx5IHJlbGF0ZWQgb2FrIHNwZWNpZXMgW1F1ZXJjdXMgcm9idXIgTC4gYW5kIFEuIHBldHJhZWEgKE1hdHQuKSBMaWVibC5dOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjg7ICBjaXRhdGlvbl9wYWdlcz0xNjE1LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBtYXBwaW5nIGluIHRoZSBGYWdhY2VhZSBhbmQgYmV5b25kIHVzaW5nIEVTVC1TU1JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IGFuZCB2YWxpZGF0aW9uIGluIGhpZ2ggZGVuc2l0eSBTTlAgYXJyYXkgZm9yIGdlbmV0aWMgYW5hbHlzaXMgaW4gRXVyb3BlYW4gd2hpdGUgb2FrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBoaWdoLWRlbnNpdHkgY29tcG9zaXRlIG1hcCBhbmQgY29tcGFyYXRpdmUgbWFwcGluZyBvZiBzZWdyZWdhdGlvbiBkaXN0b3J0aW9uIHJlZ2lvbnMgaW4gYmFybGV5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbmV0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODQ7ICBjaXRhdGlvbl9wYWdlcz0zMTktMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmVmZXJlbmNlIGdlbmV0aWMgbWFwIG9mIEMuIGNsZW1lbnRpbmEgaG9ydC4gZXggVGFuLjsgY2l0cnVzIGV2b2x1dGlvbiBpbmZlcmVuY2VzIGZyb20gY29tcGFyYXRpdmUgbWFwcGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NTkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2YgcG9zdHp5Z290aWMgaXNvbGF0aW9uIGluIEV1Y2FseXB0dXM6IHdob2xlLWdlbm9tZSBhbmFseXNpcyBvZiBiYXJyaWVycyB0byBpbnRyb2dyZXNzaW9uIGluIGEgd2lkZSBpbnRlcnNwZWNpZmljIGNyb3NzIG9mIEV1Y2FseXB0dXMgZ3JhbmRpcyBhbmQgRS4gZ2xvYnVsdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MDUtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIHJlZ2lvbnMgYXNzb2NpYXRlZCB3aXRoIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gaW4gbWFpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9NjIyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIHJlZ2lvbnMgYXNzb2NpYXRlZCB3aXRoIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gb2YgbW9sZWN1bGFyIG1hcmtlcnMgaW4gRjIsIGJhY2tjcm9zcywgZG91YmxlZCBoYXBsb2lkLCBhbmQgcmVjb21iaW5hbnQgaW5icmVkIHBvcHVsYXRpb25zIGluIHJpY2UgKE9yeXphIHNhdGl2YSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjUzOyAgY2l0YXRpb25fcGFnZXM9NTM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgRG9iemhhbnNreS1NdWxsZXIgaW5jb21wYXRpYmlsaXRpZXMgY29udHJpYnV0aW5nIHRvIHRoZSBzdGVyaWxpdHkgb2YgaHlicmlkcyBiZXR3ZWVuIE1pbXVsdXMgZ3V0dGF0dXMgYW5kIE0uIG5hc2F0dXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTE5MzItNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGhldGVyb3NwZWNpZmljIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gaW4gUG9wdWx1cyByZXZlYWxlZCBieSBhIGRlbnNlIGdlbmV0aWMgbWFwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTQ1MS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSBhbmQgbGlua2FnZSBtYXAgY29uc3RydWN0aW9uIGluIGN1bHRpdmF0ZWQgdG9tYXRvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmFseXNpcyBvZiBzZWdyZWdhdGlvbiBkaXN0b3J0aW9uIG9mIFNTUiBtYXJrZXJzIGluIEYyIHBvcHVsYXRpb24gb2YgYmFybGV5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBZ3JpYy4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTcyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIG9mIHJlcHJvZHVjdGl2ZSBpc29sYXRpb24gaW4gdGhyZWUgQW5naW9zcGVybSBnZW5lcmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTExOTUtMjA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IeWJyaWQgc3RlcmlsaXR5IGluIHBsYW50OiBzdG9yaWVzIGZyb20gcmljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xODYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldm9sdXRpb24gYW5kIGdlbmV0aWNzIG9mIHBvc3R6eWdvdGljIHJlcHJvZHVjdGl2ZSBpc29sYXRpb24gaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbG9hZCBhbmQgaXRzIGNhdXNlcyBpbiBsb25nLWxpdmVkIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZXM7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluYnJlZWRpbmcgZGVwcmVzc2lvbiBpbiBjb25pZmVyczogaW1wbGljYXRpb25zIGZvciBicmVlZGluZyBzdHJhdGVneTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rm9yLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTAyLTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIG1laW90aWMgZHJpdmUgbG9jdXMgYWxtb3N0IGNvbXBsZXRlbHkgZGlzdG9ydHMgc2VncmVnYXRpb24gaW4gTWltdWx1cyAoTW9ua2V5Zmxvd2VyKSBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0zNDctNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc3VsdHMgb2Ygc3BlY2llcyBoeWJyaWRpemF0aW9uIHdpdGggUXVlcmN1cyByb2J1ciBMLiBhbmQgUXVlcmN1cyBwZXRyYWVhIChNYXR0LikgTGllYmwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFNjaS4gRm9yLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTEzN3MtNDNzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9pbmZvcm1hdGljIGFuYWx5c2lzIG9mIEVTVHMgY29sbGVjdGVkIGJ5IFNhbmdlciBhbmQgcHlyb3NlcXVlbmNpbmcgbWV0aG9kcyBmb3IgYSBrZXVzdG9uZSBmb3Jlc3QgdHJlZSBzcGVjaWVzOiBvYWs7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTY1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1XYWdlbmluZ2VuLCBOZXRoZXJsYW5kczsgIGNpdGF0aW9uX3RpdGxlPUpvaW5tYXDCriA0LCBTb2Z0d2FyZSBmb3IgdGhlIENhbGN1bGF0aW9uIG9mIEdlbmV0aWMgTWFwcyBpbiBFeHBlcmltZW50YWwgUG9wdWxhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MUG1lcmdlOiBhbiBSIHBhY2thZ2UgZm9yIG1lcmdpbmcgZ2VuZXRpYyBtYXBzIGJ5IGxpbmVhciBwcm9ncmFtbWluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvSW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNjIzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpb24gb2YgcmVjb21iaW5hdGlvbiBmcmVxdWVuY2llcyBhbmQgY29uc3RydWN0aW9uIG9mIFJGTFAgbGlua2FnZSBtYXBzIGluIHBsYW50cyBmcm9tIGNyb3NzZXMgYmV0d2VlbiBoZXRlcm96eWdvdXMgcGFyZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTI1OyAgY2l0YXRpb25fcGFnZXM9NjQ1LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIGdlbmV0aWMgbWFwIGJhc2VkIG9uIGZvdXIgbWFwcGluZyBwb3B1bGF0aW9ucyBhbmQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYXNzb2NpYXRlZCB3aXRoIGVjb25vbWljYWxseSBpbXBvcnRhbnQgdHJhaXRzIGluIHdhdGVybWVsb24gKENpdHJ1bGx1cyBsYW5hdHVzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2Ygc2VncmVnYXRpb24gZGlzdG9ydGlvbiBsb2NpIGluIHRyaXRpY2FsZSAoeCBUcml0aWNvc2VjYWxlIFdpdHRtYWNrKSBiYXNlZCBvbiBhIGhpZ2gtZGVuc2l0eSBEQXJUIG1hcmtlciBjb25zZW5zdXMgZ2VuZXRpYyBsaW5rYWdlIG1hcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYSAxMCwwMDAgbG9jdXMgZ2VuZXRpYyBtYXAgb2YgdGhlIHN1bmZsb3dlciBnZW5vbWUgYmFzZWQgb24gbXVsdGlwbGUgY3Jvc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTcyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkaXN0cmlidXRpb24gb2YgZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gYSBtYXNzLXNlbGVjdGVkIHBvcHVsYXRpb24gb2YgbWFyaXRpbWUgcGluZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5LCBoaWdoLXJlc29sdXRpb24gZ2VuZXRpYyBtYXAgYW5kIGl0cyBpbnRlZ3JhdGlvbiB3aXRoIEJBQy1iYXNlZCBwaHlzaWNhbCBtYXAgaW4gY2hhbm5lbCBjYXRmaXNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTM5LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbXByb3ZlZCBjb25zZW5zdXMgbGlua2FnZSBtYXAgb2YgQmFybGV5IGJhc2VkIG9uIGZsb3ctc29ydGVkIGNocm9tb3NvbWVzIGFuZCBTaW5nbGUgTnVjbGVvdGlkZSBQb2x5bW9ycGhpc20gbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yMzgtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVU0RBIGJhcmxleSBjb3JlIGNvbGxlY3Rpb246IGdlbmV0aWMgZGl2ZXJzaXR5LCBwb3B1bGF0aW9uIHN0cnVjdHVyZSwgYW5kIHBvdGVudGlhbCBmb3IgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBvZiBpbnRlbnNlIGNocm9tb3NvbWFsIHNodWZmbGluZyBkdXJpbmcgY29uaWZlciBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgbWFwIGZvciBVZzk5IHN0ZW0gcnVzdCByZXNpc3RhbmNlIGxvY2kgaW4gd2hlYXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI3OyAgY2l0YXRpb25fcGFnZXM9MTU2MS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9hayBnZW5lIGV4cHJlc3Npb24gYXRsYXM6IGluc2lnaHRzIGludG8gRmFnYWNlYWUgZ2Vub21lIGV2b2x1dGlvbiBhbmQgdGhlIGRpc2NvdmVyeSBvZiBnZW5lcyByZWd1bGF0ZWQgZHVyaW5nIGJ1ZCBkb3JtYW5jeSByZWxlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNvZGluZyB0aGUgb2FrIGdlbm9tZTogcHVibGljIHJlbGVhc2Ugb2Ygc2VxdWVuY2UgZGF0YSwgYXNzZW1ibHksIGFubm90YXRpb24gYW5kIHB1YmxpY2F0aW9uIHN0cmF0ZWdpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlciB5IGFuZCBnZW5vdHlwaW5nIGZvciBjb25zdHJ1Y3RpbmcgYSBzYXR1cmF0ZWQgbGluayBhZ2UgbWFwIG9mIENoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2F0dXJhdGVkIGxpbmthZ2UgbWFwIGNvbnN0cnVjdGlvbiBpbiBSdWJ1cyBpZGFldXMgdXNpbmcgZ2Vub3R5cGluZyBieSBzZXF1ZW5jaW5nIGFuZCBnZW5vbWUtaW5kZXBlbmRlbnQgaW1wdXRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgY2hhcmFjdGVyaXphdGlvbiBvZiBEQXJUIG1hcmtlcnMgYmFzZWQgb24gaGlnaC1kZW5zaXR5IGxpbmthZ2UgYW5hbHlzaXMgYW5kIHBoeXNpY2FsIG1hcHBpbmcgdG8gdGhlIEV1Y2FseXB0dXMgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWMgYW5kIHBvcHVsYXRpb24gZ2VuZXRpYyBhbmFseXNpcyBpbmRpY2F0ZSBoaWdobHkgcG9yb3VzIGdlbm9tZXMgYW5kIGhpZ2ggbGV2ZWxzIG9mIGdlbmUgZmxvdyBiZXR3ZWVuIGRpdmVyZ2VudCBIZWxpYW50aHVzIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTIwNjEtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlIGdlbm9tZSBjb21wYXJpc29ucyBvZiBGcmFnYXJpYSwgUHJ1bnVzIGFuZCBNYWx1cyByZXZlYWwgZGlmZmVyZW50IG1vZGVzIG9mIGV2b2x1dGlvbiBiZXR3ZWVuIFJvc2FjZW91cyBzdWJmYW1pbGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwcGluZyBpbiBhIHBpbmUgdHJlZSByZXZlYWxzIGEgZ2Vub21pYyByZWdpb24gYXNzb2NpYXRlZCB3aXRoIGluYnJlZWRpbmcgZGVwcmVzc2lvbiBhbmQgcHJvdmlkZXMgY2x1ZXMgdG8gdGhlIGV4dGVudCBhbmQgZGlzdHJpYnV0aW9uIG9mIG1laW90aWMgcmVjb21iaW5hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYW4gaW50ZWdyYXRlZCBoaWdoIGRlbnNpdHkgU2ltcGxlIFNlcXVlbmNlIFJlcGVhdCBsaW5rYWdlIG1hcCBpbiBjdWx0aXZhdGVkIHN0cmF3YmVycnkgKEZyYWdhcmlhIDMgYW5hbmFzc2EpIGFuZCBpdHMgYXBwbGljYWJpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz03OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIGdlbmV0aWMgbGlua2FnZSBtYXBzIG9mIEV1Y2FseXB0dXMgaW1wcm92ZSBFdWNhbHlwdHVzIGdyYW5kaXMgZ2Vub21lIGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDY7ICBjaXRhdGlvbl9wYWdlcz0xMjgzLTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIGRlbnNpdHkgU05QIG1hcHBpbmcgYW5kIFFUTCBhbmFseXNpcyBmb3IgZnJ1aXQgcXVhbGl0eSBjaGFyYWN0ZXJpc3RpY3MgaW4gcGVhY2ggKFBydW51cyBwZXJzaWNhIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTktMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhaW5zIGluIFFUTCBkZXRlY3Rpb24gdXNpbmcgYW4gdWx0cmEtaGlnaCBkZW5zaXR5IFNOUG1hcCBiYXNlZCBvbiBwb3B1bGF0aW9uIHNlcXVlbmNpbmcgcmVsYXRpdmUgdG8gdHJhZGl0aW9uYWwgUkZMUC9TU1IgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxNzU5NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIHJlYXJyYW5nZW1lbnRzIGFuZCB0aGUgZ2VuZXRpY3Mgb2YgcmVwcm9kdWN0aXZlIGJhcnJpZXJzIGluIE1pbXVsdXMgKE1vbmtleWZsb3dlcnMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fdm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTI1NDctNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIGFkYXB0aXZlIHRyYWl0cyBiZXR3ZWVuIG9hayBhbmQgY2hlc3RudXQgYmFzZWQgb24gYW4gZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyBjb25zZW5zdXMgbWFwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzI7ICBjaXRhdGlvbl9wYWdlcz01MzMtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlY29kaW5nIHBsYW50IGFuZCBhbmltYWwgZ2Vub21lIHBsYXN0aWNpdHkgZnJvbSBkaWZmZXJlbnRpYWwgcGFsZW8tZXZvbHV0aW9uYXJ5IHBhdHRlcm5zIGFuZCBwcm9jZXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9OTE3LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGRpdmVyc2l0eSBpbiBuYXR1cmFsIHBvcHVsYXRpb25zIG9mIG9hayBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFNjaS4gRm9yLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX2lzc3VlPShTdXBwbC4gMSk7ICBjaXRhdGlvbl9wYWdlcz0xODZzLTIwMnM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggcmF0ZXMgb2YgZ2VuZSBmbG93IGJ5IHBvbGxlbiBhbmQgc2VlZCBpbiBvYWsgcG9wdWxhdGlvbnMgYWNyb3NzIEV1cm9wZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWU4NTEzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggZ2VuZXRpYyBsb2FkIGluIHRoZSBwYWNpZmljIG95c3RlciBDcmFzc29zdHJlYSBnaWdhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTU5OyAgY2l0YXRpb25fcGFnZXM9MjU1LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWRlbnNpdHkgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgd2l0aCBvdmVyIDIsNDAwIHNlcXVlbmNlLWFuY2hvcmVkIERBclQgbWFya2VycyBmb3IgZ2VuZXRpYyBkaXNzZWN0aW9uIGluIGFuIEYyIHBzZXVkby1iYWNrY3Jvc3Mgb2YgRXVjYWx5cHR1cyBncmFuZGlzIMOXIEUuIHVyb3BoeWxsYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTYzLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJlbmd0aCwgZGl2ZXJzaXR5IGFuZCBwbGFzdGljaXR5IG9mIHBvc3RtYXRpbmcgcmVwcm9kdWN0aXZlIGJhcnJpZXJzIGJldHdlZW4gdHdvIGh5YnJpZGl6aW5nIG9hayBzcGVjaWVzIChRdWVyY3VzIHJvYnVyIEwuIGFuZCBRdWVyY3VzIHBldHJhZWEgKE1hdHQpIExpZWJsLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTU3LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjaWVzIHJlbGF0aXZlIGFidW5kYW5jZSBhbmQgZGlyZWN0aW9uIG9mIGludHJvZ3Jlc3Npb24gaW4gb2FrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTIyMjgtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcHJvZHVjdGl2ZSBwYXR0ZXJucyBzaGFwZSBpbnRyb2dyZXNzaW9uIGR5bmFtaWNzIGFuZCBzcGVjaWVzIHN1Y2Nlc3Npb24gd2l0aGluIHRoZSBFdXJvcGVhbiB3aGl0ZSBvYWsgc3BlY2llcyBjb21wbGV4OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fdm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE1Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgY29udHJvbGxpbmcgd2F0ZXIgdXNlIGVmZmljaWVuY3kgYW5kIHJlbGF0ZWQgdHJhaXRzIGluIFF1ZXJjdXMgcm9idXIgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI2My03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgb2YgdG9sZXJhbmNlIHRvIHdhdGVybG9nZ2luZyBpbiBhIEV1cm9wZWFuIG9hayAoUXVlcmN1cyByb2J1ciBMLik6IHBoeXNpb2xvZ2ljYWwgcmVsZXZhbmNlIGFuZCB0ZW1wb3JhbCBlZmZlY3QgcGF0dGVybnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50LCBDZWxsIEVudmlyb25tZW50OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NDIyLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvMi8xMTUvMTc0NjI5NzIvZHN3MDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBkZXZlbG9wZWQgdGhlIGRlbnNlc3Qgc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIChTTlApLWJhc2VkIGxpbmthZ2UgZ2VuZXRpYyBtYXAgdG8gZGF0ZSBmb3IgdGhlIGdlbnVzIFF1ZXJjdXMuIEFuIDhrIGdlbmUtYmFzZWQgU05QIGFycmF5IHdhcyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MTc0NTMzNyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSW1UItty7/OLpaswLzEzZvpr7leg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f115%2f1745337" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f115%2f1745337" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69060" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&amp;Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/2">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197902">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197902">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197907">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197907">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197915">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197915">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197944">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197944">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197971">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197971">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197991">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197991">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198000">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198000">Authors' contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198003">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198003">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198007">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198007">Web portal</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198020">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198020">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198029">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198029">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/2/101/1745328">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/2/125/1745357">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-density linkage mapping and distribution of segregation distortion regions in the oak genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Catherine Bod&#233;n&#232;s</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Cestas</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Université de Bordeaux, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Talence</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +33 557122842, Fax. +33 557122881; Email: <a href="mailto:catherine.bodenes@pierroton.inra.fr" target="_blank">catherine.bodenes@pierroton.inra.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Catherine+Bod%c3%a9n%c3%a8s">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bod&#233;n&#232;s C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bod&#233;n&#232;s C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Catherine Bodénès</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Emilie Chancerel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Cestas</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Université de Bordeaux, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Talence</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Emilie+Chancerel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chancerel E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chancerel E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Emilie Chancerel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fran&#231;ois Ehrenmann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Cestas</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Université de Bordeaux, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Talence</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fran%c3%a7ois+Ehrenmann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ehrenmann F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ehrenmann F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">François Ehrenmann</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antoine Kremer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Cestas</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Université de Bordeaux, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Talence</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antoine+Kremer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kremer A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kremer A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Antoine Kremer</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christophe Plomion</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Cestas</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Université de Bordeaux, UMR1202 BIOGECO</div>, <div class="addr-line">F-33610 Talence</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christophe+Plomion">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Plomion C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Plomion C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christophe Plomion</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 115–124, <a href='https://doi.org/10.1093/dnares/dsw001'>https://doi.org/10.1093/dnares/dsw001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 March 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 July 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 January 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745337" href="/dnaresearch/article-pdf/23/2/115/17462972/dsw001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Catherine Bodénès, Emilie Chancerel, François Ehrenmann, Antoine Kremer, Christophe Plomion;  High-density linkage mapping and distribution of segregation distortion regions in the oak genome, <em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 115–124, <a href='https://doi.org/10.1093/dnares/dsw001'>https://doi.org/10.1093/dnares/dsw001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745337?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-density%20linkage%20mapping%20and%20distribution%20of%20segregation%20distortion%20regions%20in%20the%20oak%20genome&amp;publicationDate=2016-03-23&amp;volumeNum=23&amp;issueNum=2&amp;author=Bod%C3%A9n%C3%A8s%2C%20Catherine%3B%20Chancerel%2C%20Emilie&amp;startPage=115&amp;endPage=124&amp;contentId=10.1093%2Fdnares%2Fdsw001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f115%2f1745337" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87197902 id="87197902" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We developed the densest single-nucleotide polymorphism (SNP)-based linkage genetic map to date for the genus <em>Quercus</em>. An 8k gene-based SNP array was used to genotype more than 1,000 full-sibs from two intraspecific and two interspecific full-sib families of <em>Quercus petraea</em> and <em>Quercus robur</em>. A high degree of collinearity was observed between the eight parental maps of the two species. A composite map was then established with 4,261 SNP markers spanning 742 cM over the 12 linkage groups (LGs) of the oak genome. Nine genomic regions from six LGs displayed highly significant distortions of segregation. Two main hypotheses concerning the mechanisms underlying segregation distortion are discussed: genetic load vs. reproductive barriers. Our findings suggest a predominance of pre-zygotic to post-zygotic barriers.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-density linkage map&quot;">high-density linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;segregation distortion&quot;">segregation distortion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reproductive barriers&quot;">reproductive barriers</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Quercus"><em>Quercus</em></a></div>                            <h2 scrollto-destination=87197907 id="87197907" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>`The genus <em>Quercus</em> is a major forest tree genus with 300–600 species spread throughout the world (<a class="link link-uri" href="http://www.mobot.org/MOBOT/research/APweb/orders/fagalesweb.htm#Fagales">http://www.mobot.org/MOBOT/research/APweb/orders/fagalesweb.htm#Fagales</a>). The European white oak species complex—including <em>Quercus robur</em> (Qr), pedunculate oak and <em>Quercus petraea</em> (Qp), sessile oak, in particular—is an exceptional model for studies of the genetics of speciation in sympatry.<sup><a reveal-id="DSW001C1  DSW001C4" data-open="DSW001C1  DSW001C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> These two main oak species have similar distribution areas, but display morphological differences for many traits, including leaf shape, acorn peduncle length, and hairiness.<sup><a reveal-id="DSW001C5" data-open="DSW001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> They also have different ecophysiological requirements: Qr tolerates waterlogging and light exposure more effectively than Qp,<sup><a reveal-id="DSW001C6" data-open="DSW001C6" class="link link-ref link-reveal xref-bibr">6</a></sup> whereas Qp has a higher water use efficiency and is more shade-tolerant, consistent with its role as a post-pioneer species.<sup><a reveal-id="DSW001C7  DSW001C9" data-open="DSW001C7  DSW001C9" class="link link-ref link-reveal xref-bibr">7–9</a></sup> These two sympatric species are interfertile, forming natural hybrids in mixed stands and displaying only partial reproductive isolation (RI).<sup><a reveal-id="DSW001C3" data-open="DSW001C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW001C10" data-open="DSW001C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>Low-density genetic linkage maps in oaks have been constructed with various genotyping techniques: random amplified polmorphic DNA, amplified fragment length polymorphism and simple sequence repeat (SSR).<sup><a reveal-id="DSW001C11  DSW001C13" data-open="DSW001C11  DSW001C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> Progress has recently been made, with the development of a single-nucleotide polymorphism (SNP) assay for Qr and Qp.<sup><a reveal-id="DSW001C14" data-open="DSW001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Using this resource, we were able in this present study to construct high-density gene-based maps with a large number of SNP markers, providing a detailed scan of segregation distortion (SD) within the genome. SD, corresponding to deviation from the expected Mendelian proportion of individuals in a given genotypic class within a segregating population, has been widely documented in plant species, within the framework of genetic mapping studies, e.g. in barley,<sup><a reveal-id="DSW001C15" data-open="DSW001C15" class="link link-ref link-reveal xref-bibr">15</a></sup> clementine,<sup><a reveal-id="DSW001C16" data-open="DSW001C16" class="link link-ref link-reveal xref-bibr">16</a></sup> eucalyptus,<sup><a reveal-id="DSW001C17" data-open="DSW001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> maize,<sup><a reveal-id="DSW001C18" data-open="DSW001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> rice,<sup><a reveal-id="DSW001C19" data-open="DSW001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> monkeyflower,<sup><a reveal-id="DSW001C20" data-open="DSW001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> poplar<sup><a reveal-id="DSW001C21" data-open="DSW001C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and tomato.<sup><a reveal-id="DSW001C22" data-open="DSW001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The detection of SD depends heavily on the type of cross. In most cases, SD is stronger and, therefore, easier to detect, in interspecific crosses than in intraspecific crosses.<sup><a reveal-id="DSW001C23" data-open="DSW001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Some of these studies showed that distortion spreads over linked markers (and even over large genetic distances in some cases), creating SD regions (SDR). SD may result from three underlying causes. Gametic incompatibility (one of the many types of prezygotic barrier)<sup><a reveal-id="DSW001C24" data-open="DSW001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and/or reduced hybrid viability (one of the many forms of post-zygotic selection)<sup><a reveal-id="DSW001C25" data-open="DSW001C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSW001C26" data-open="DSW001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> may be involved. Indeed, various developmental processes in the pollen, embryo and seedling offer opportunities for differential selection at the gamete (pollen tube competition, pollen tube growth) and/or zygote (embryo competition, ovule abortion) level. The accumulation of deleterious mutations is an alternative mechanism that may give rise to SD. Genetic load has often been advocated as a source of SD, particularly in trees, which form large populations likely to have accumulated large number of deleterious mutations.<sup><a reveal-id="DSW001C27" data-open="DSW001C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSW001C28" data-open="DSW001C28" class="link link-ref link-reveal xref-bibr">28</a></sup> SD may also arise due to asymmetric allelic inheritance in heterozygotes (i.e. meiotic drive),<sup><a reveal-id="DSW001C29" data-open="DSW001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> but this process is not considered here as it relates to hybrid fertility rather than hybrid viability.</p><p>In this context, the first objective of this study was to establish a high-density gene-based genetic map of oak by genotyping more than 1,000 full-sibs from two intraspecific and two interspecific full-sib families of <em>Q. petraea</em> and <em>Q. robur</em>. A single composite map was constructed by merging the eight parental maps. This map includes 4,261 gene-based markers and is the densest linkage map ever produced for oak. The second objective of our study was to carry out a detailed analysis of the SDRs based on the patterns of SD observed for intraspecific and interspecific crosses. The SDRs identified on the eight linkage maps were compared to investigate the causes of the observed SDRs that span 6 of the 12 chromosomes. More precisely, we describe the extent of SD along linkage groups (LGs) and depict the distribution of SDRs, compare the consistency of SDRs across different genetic backgrounds and draw inferences about the potential sources of the widespread SD in the oak genome.</p>                            <h2 scrollto-destination=87197915 id="87197915" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87197917 id="87197917" class="section-title" data-legacy-id=s2a>2.1. Plant material and DNA extraction</h3>
<p>Six parental trees, three <em>Q. robur</em> (3P, A4, 11P) and three <em>Q. petraea</em> (QS28, QS21, QS29), were used to generate four full-sib families (F1) (Fig. <a reveal-id="DSW001F1" data-open="DSW001F1" class="link link-reveal link-table xref-fig">1</a>): </p><ul class="bullet"><li><p>One <em>Q. robur</em> intraspecific family (3P × A4) referred to as P1.</p></li><li><p>One <em>Q. petraea</em> intraspecific family (QS28f × QS21) referred to as P4.</p></li><li><p>Two interspecific families (11P1 × QS28m, 11P2 × QS29) referred to as P2 and P3.</p></li></ul>                        <a id="87197924" scrollto-destination="87197924"></a>
<div id="DSW001F1" data-id="DSW001F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00101.jpeg?Expires=2147483647&Signature=eDY4EBDcmejXZ2uF26fO61t4UrMb1~UjoBnWMT4XwXfn0WgmDXFjmhA8G2rWaDrAkDrOvulc5S-xeBFux5G~4JYNJ~aeXJwM-NNWoYaMdHUIMbgtLENKRvoDcoya1vJ7OYgBYxcXUfEk8wKNuHvpDNh7AmuJG~vN7DNjwkSZZHPBMDPM4yONxsBBmByO3dBwDuTCLXYSyegv~GJe6C-dRj3W5DzdMmv6SZUn1XltXxDCYVysSKOItqUuwiBIqlGnmCRrvcYmPC8~N2Ch3L6XtAPnroGdF7V4iSH5p6MmqbqH~knthPUTgowHNKpwJTBTNGJGvwW225KKmOPciLeuHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Description of the four pedigrees (3P, A4, 11P, QS28, QS29 and QS21 referred to the name of the parents used for the controlled crosses. P1, P2, P3 and P4 correspond to the name of the pedigree; n represents the number of offspring in each pedigree)." path-from-xml="dsw00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197924/dsw00101.jpeg" path-from-xml="dsw00101.jpeg" target="_blank">View large</a><a section="87197924" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00101.jpeg?Expires=2147483647&Signature=YpYojJU9kCbDdLYyouUJlTY72~Iv5jAi0nNPUl9oW0KPWGapBrzgWmb9SjWDGlCmXVOLX52tlSzN7AAd7KBu2Uch5nebSa7~ceIdyr1PqVz6F3QbnF0Bxarbn-km0L17Y3d8ZquvP3K0obg4gxGHqgRh9XMU0n1qjYEzTfzb91RxSPLE5y-VilqjZhK~0aatj5QvoZfU63EVrTv-SRUvKn5QMxeVBGuBZRpwenlme0cNXSo3BCUay5-9frVIGFFV~60dsYpmufy0V9wvZCACpMtgGwPR~l3Dxw~-OQbzEg9LPuw09GAUnIVunpxPqFTjMXyu4CU-LMitvNUHIdW2xQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197924&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Description of the four pedigrees (3P, A4, 11P, QS28, QS29 and QS21 referred to the name of the parents used for the controlled crosses. P1, P2, P3 and P4 correspond to the name of the pedigree; <em>n</em> represents the number of offspring in each pedigree).</p></div></div><div content-id="DSW001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00101.jpeg?Expires=2147483647&Signature=eDY4EBDcmejXZ2uF26fO61t4UrMb1~UjoBnWMT4XwXfn0WgmDXFjmhA8G2rWaDrAkDrOvulc5S-xeBFux5G~4JYNJ~aeXJwM-NNWoYaMdHUIMbgtLENKRvoDcoya1vJ7OYgBYxcXUfEk8wKNuHvpDNh7AmuJG~vN7DNjwkSZZHPBMDPM4yONxsBBmByO3dBwDuTCLXYSyegv~GJe6C-dRj3W5DzdMmv6SZUn1XltXxDCYVysSKOItqUuwiBIqlGnmCRrvcYmPC8~N2Ch3L6XtAPnroGdF7V4iSH5p6MmqbqH~knthPUTgowHNKpwJTBTNGJGvwW225KKmOPciLeuHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Description of the four pedigrees (3P, A4, 11P, QS28, QS29 and QS21 referred to the name of the parents used for the controlled crosses. P1, P2, P3 and P4 correspond to the name of the pedigree; n represents the number of offspring in each pedigree)." path-from-xml="dsw00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197924/dsw00101.jpeg" path-from-xml="dsw00101.jpeg" target="_blank">View large</a><a section="87197924" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00101.jpeg?Expires=2147483647&Signature=YpYojJU9kCbDdLYyouUJlTY72~Iv5jAi0nNPUl9oW0KPWGapBrzgWmb9SjWDGlCmXVOLX52tlSzN7AAd7KBu2Uch5nebSa7~ceIdyr1PqVz6F3QbnF0Bxarbn-km0L17Y3d8ZquvP3K0obg4gxGHqgRh9XMU0n1qjYEzTfzb91RxSPLE5y-VilqjZhK~0aatj5QvoZfU63EVrTv-SRUvKn5QMxeVBGuBZRpwenlme0cNXSo3BCUay5-9frVIGFFV~60dsYpmufy0V9wvZCACpMtgGwPR~l3Dxw~-OQbzEg9LPuw09GAUnIVunpxPqFTjMXyu4CU-LMitvNUHIdW2xQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197924&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Description of the four pedigrees (3P, A4, 11P, QS28, QS29 and QS21 referred to the name of the parents used for the controlled crosses. P1, P2, P3 and P4 correspond to the name of the pedigree; <em>n</em> represents the number of offspring in each pedigree).</p></div></div><p>Crosses are denoted ‘female parent × male parent’ below. The same parental tree of <em>Q. petraea</em> (QS28) was used as the male parent in one cross (QS28m) and the female parent (QS28f) in another cross. Furthermore, the <em>Q. robur</em> parent 11P was used as a female parent in two different crosses and is therefore named 11P1 and 11P2.</p><p><em>Quercus robur</em> had to be used as the female parent in all interspecific crosses, due to the strong asymmetry of hybridization success<sup><a reveal-id="DSW001C3" data-open="DSW001C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW001C30" data-open="DSW001C30" class="link link-ref link-reveal xref-bibr">30</a></sup> between these two species. The sizes of the progenies of the full-sib crosses varied from 398 (QS28f × QS21) to 114 (11P2 × QS29) individuals. Intraspecific and interspecific controlled crosses were implemented over the years since 1993. The offspring were installed in stool beds at the nursery of the INRA Forestry Research Station Pierroton (latitude 44.44°N, longitude 0.46°W) in South West of France, before subsequent vegetative propagation of the offspring. Bud and leaf material was collected on the stool beds and DNA was extracted as described by Bodénès et al.<sup><a reveal-id="DSW001C13" data-open="DSW001C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p>                            <h3 scrollto-destination=87197928 id="87197928" class="section-title" data-legacy-id=s2b>2.2. SNP genotyping</h3>
<p>SNP genotyping was carried out on the four mapping populations (1,059 samples in total), with the Illumina<sup>®</sup> Infinium iSelect Custom Genotyping Array (Illumina Inc., San Diego, CA, USA), according to the manufacturer's standard protocol, using 200 ng of genomic DNA per sample. The selected SNPs correspond to two different data sets, the first one was obtained from the resequencing of six trees used as parents of the four controlled crosses and the second one was obtained from the oak Unigene established by Ueno <em>et al</em>.<sup><a reveal-id="DSW001C31" data-open="DSW001C31" class="link link-ref link-reveal xref-bibr">31</a></sup> In total, 7,913 SNPs were selected for array design. We deliberately selected a limited number of markers (from 1 to 3) per contig, to ensure a broad distribution of SNPs across the genome.<sup><a reveal-id="DSW001C14" data-open="DSW001C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p>                            <h3 scrollto-destination=87197934 id="87197934" class="section-title" data-legacy-id=s2c>2.3. Linkage map construction</h3>
<p>For each cross, two data sets were generated, each containing the meiotic segregation information from one of the parents. Markers segregating in a 1:1 ratio (testcross configuration) were mapped with JoinMap 4.0 (Kyazma, Wageningen, NL).<sup><a reveal-id="DSW001C32" data-open="DSW001C32" class="link link-ref link-reveal xref-bibr">32</a></sup> For each population, the minimum logarithm of odds (LOD) threshold for grouping was determined by identifying grouping tree branches with stable marker numbers over increasing consecutive LOD values for a total number of groups of 12 which corresponds to the number of chromosome. The minimum LOD threshold for most of the linkage maps was 5 and increased to 6 for P4m map or 7 for P2m map. The two parental linkage maps for each mapping population were constructed with the maximum likelihood (ML) algorithm, with a minimum LOD of 5 and the default parameters (recombination frequency of 0.4 and a maximum threshold value of 1 for the jump). Recombination frequencies were converted to map distances in cM with the Kosambi's mapping function. The maps shown were plotted with MapChart 2.0<sup><a reveal-id="DSW001C33" data-open="DSW001C33" class="link link-ref link-reveal xref-bibr">33</a></sup> or using an R script. LG numbering was based on SSR markers, as in the study by Bodénès <em>et al</em>.<sup><a reveal-id="DSW001C13" data-open="DSW001C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The cosegregation of SSRs and SNPs (data not shown) made it possible to identify homologous LGs unambiguously.</p>                            <h3 scrollto-destination=87197937 id="87197937" class="section-title" data-legacy-id=s2d>2.4. Composite map construction</h3>
<p>We constructed a composite map from the eight parental maps, with the LPmerge software developed by Endelman and Plomion,<sup><a reveal-id="DSW001C34" data-open="DSW001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> which is available as an R package <a class="link link-uri" href="https://cran.r-project.org/web/packages/LPmerge/index.html">https://cran.r-project.org/web/packages/LPmerge/index.html</a>. This approach is based on the integration of linkage map data rather than observed recombination between markers, with linear programming used to minimize the mean absolute error between the composite map and the linkage map for each population as efficiently as possible. For assessment of the goodness-of-fit for the composite map, LPmerge computes a root-mean-square error (RMSE) per LG by comparing the position (in cM) of all markers on the composite map with that on the component maps (<a class="link link-uri" href="http://w3.pierroton.inra.fr/cgi-bin/cmap/viewer?ref_map_set_acc=51&amp;ref_map_accs=-1">http://w3.pierroton.inra.fr/cgi-bin/cmap/viewer?ref_map_set_acc=51&amp;ref_map_accs=-1</a>). We calculated this metric for different maximum interval sizes (<em>k</em> in the algorithm), ranging from 1 to 8. The value of <em>k</em> minimizing the mean RMSE per LG was selected for construction of the composite map (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw001_Supplementary_Data.zip?Expires=2147483647&Signature=NmivpNrUO-LgVSeMRXc4bQUwZvSC5jnBdoWFQGCXBFaBGk3dN9q2jJr9gQX8i75DhN4EBlVhWXpPza189X~BfsPkFIs9bBGJndENLkuzwBPTL7Sl5Gr-Q8kBkUAtXN7A7umF6N~Des8mg4ECXksZ51WsP~Y0EUVTwX00dcXbRI5HVaVwGrF2bHoZ9Dism3irHcolFemd-sOBkSZKOhR4UuTx-D-gBnuU0k0nOmW7dsgmGmUluO98CiKTl0bJWOaGR7ywe9OziUmdtaNv8P8Uqbrh81fTZYxe7j7YUUO9lyCGi2mDMkqxatbdDkR4aMN7CisXR6iSdYtFdZtsqNMidA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>). Intercross markers (segregating in a 1:2:1 ratio) were added as accessory markers in a second step taking the rate of recombination between these loci and the closest linked test-cross framework marker (FM) into account.</p>                            <h3 scrollto-destination=87197939 id="87197939" class="section-title" data-legacy-id=s2e>2.5. SD analysis</h3>
<p>We tested each marker for significant deviation from the expected Mendelian genotype frequencies (<em>χ</em><sup>2</sup> with 1 degree of freedom for codominant markers, <em>α</em>= 0.05 calculated with JoinMap software) to detect SD. Assuming that each LG corresponds to one chromosome (<em>n</em>= 12 in <em>Quercus</em>) and that each chromosome contains at least two independent regions (the mean length of LGs was 66 cM, this paper), we expected there to be at least 24 independent genomic regions. A threshold of at least 0.05/24 ≈ 0.002 would therefore be required to obtain a genome wide error rate of <em>α</em> = 0.05. However, we applied a more stringent threshold (<em>α</em> = 0.001) and only considered distorted regions with more than three tightly linked distorted loci, to decrease the false-positive rate and to ensure that only biologically meaningful SDRs were detected. Markers displaying SD were conserved and integrated into the map. We investigated the patterns of distortion on the eight parental maps, by plotting the <em>χ</em><sup>2</sup> value of each marker along the 12 LG of the composite map obtained with the LPmerge programme (Fig. <a reveal-id="DSW001F2" data-open="DSW001F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="87197943" scrollto-destination="87197943"></a>
<div id="DSW001F2" data-id="DSW001F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00102.jpeg?Expires=2147483647&Signature=CgkIeuiBPCLyi0-YW63gcY3b-PpfiTmjXA1wKP4E29Mbtspjl4GNbAxVW7eaKfTPwl21nwPyyTNlK4SpZ7zLzLYGlS~coMQ2cle9NHRiShRlHGPGb-pIjnqgAQ~j6uTz20RqQm9mHRSm3A5CihOyamr5MjVPlMMkIT0jX63lYDOXJbQvggNenzaMPK9j6GmLzoYz2sVVjON6z3Zvsx7A6rsjWCyMIJxYhHrJ5sGfyAuRFnxSaVEUkVfrMNEwn5Ujn8wql02N7ZgRcU~f1QEePJvwNCA99fufaIkqPbuN5oXmqe6h0sgVBBXwOfbEv0u11MGw8aNNnHhcqASebspMDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of SD for the 12 LGs for the 8 parental maps, α = 0.001. The vertical line corresponds to a type I error of 0.001. The x-axis corresponds to the χ2 value and the y-axis corresponds to the map length of the LG in cM. The black vertical line corresponds to the χ2 threshold value, orange line corresponds to P1f, yellow line: P1m, light green: P2f, dark green: P2m, pink: P3f, violet: P3m, light blue: P4f and dark blue: P4m." path-from-xml="dsw00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197943/dsw00102.jpeg" path-from-xml="dsw00102.jpeg" target="_blank">View large</a><a section="87197943" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00102.jpeg?Expires=2147483647&Signature=bZYwt497gpMTyPgvaDJoJTTfP3v-01-eI4kIWHsI3f0F231F~xOe1FjLz0z47hzPHAAJK6XRi9ilCdw25f2Z1uUHM8a7~b8S3cLDPCDgRFdyGVBFfBon5HQDT1yuHMiZvz1zdKFZl66esoWmgXAyO76MOvcbSBaH0XWCke7O0vyxmdDXIsV6KzenJZdhmtIe9TOFLVWpw63eI36FSzITjdnQVKHm2IxQweqVlU7PSr3lhpr28qcqlYMpf4dsRObwLI2H3k-KZiMexhosPXnsbSafsOfvW~jm0u93JUlO-Tqs2Id2lqVcz1BgOu-UBjlRkQcQdn-LW-sNuiVxmWUbcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197943&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of SD for the 12 LGs for the 8 parental maps, <em>α</em> = 0.001. The vertical line corresponds to a type I error of 0.001. The <em>x</em>-axis corresponds to the <em>χ</em><sup>2</sup> value and the <em>y</em>-axis corresponds to the map length of the LG in cM. The black vertical line corresponds to the <em>χ</em><sup>2</sup> threshold value, orange line corresponds to P1f, yellow line: P1m, light green: P2f, dark green: P2m, pink: P3f, violet: P3m, light blue: P4f and dark blue: P4m.</p></div></div><div content-id="DSW001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00102.jpeg?Expires=2147483647&Signature=CgkIeuiBPCLyi0-YW63gcY3b-PpfiTmjXA1wKP4E29Mbtspjl4GNbAxVW7eaKfTPwl21nwPyyTNlK4SpZ7zLzLYGlS~coMQ2cle9NHRiShRlHGPGb-pIjnqgAQ~j6uTz20RqQm9mHRSm3A5CihOyamr5MjVPlMMkIT0jX63lYDOXJbQvggNenzaMPK9j6GmLzoYz2sVVjON6z3Zvsx7A6rsjWCyMIJxYhHrJ5sGfyAuRFnxSaVEUkVfrMNEwn5Ujn8wql02N7ZgRcU~f1QEePJvwNCA99fufaIkqPbuN5oXmqe6h0sgVBBXwOfbEv0u11MGw8aNNnHhcqASebspMDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of SD for the 12 LGs for the 8 parental maps, α = 0.001. The vertical line corresponds to a type I error of 0.001. The x-axis corresponds to the χ2 value and the y-axis corresponds to the map length of the LG in cM. The black vertical line corresponds to the χ2 threshold value, orange line corresponds to P1f, yellow line: P1m, light green: P2f, dark green: P2m, pink: P3f, violet: P3m, light blue: P4f and dark blue: P4m." path-from-xml="dsw00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197943/dsw00102.jpeg" path-from-xml="dsw00102.jpeg" target="_blank">View large</a><a section="87197943" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00102.jpeg?Expires=2147483647&Signature=bZYwt497gpMTyPgvaDJoJTTfP3v-01-eI4kIWHsI3f0F231F~xOe1FjLz0z47hzPHAAJK6XRi9ilCdw25f2Z1uUHM8a7~b8S3cLDPCDgRFdyGVBFfBon5HQDT1yuHMiZvz1zdKFZl66esoWmgXAyO76MOvcbSBaH0XWCke7O0vyxmdDXIsV6KzenJZdhmtIe9TOFLVWpw63eI36FSzITjdnQVKHm2IxQweqVlU7PSr3lhpr28qcqlYMpf4dsRObwLI2H3k-KZiMexhosPXnsbSafsOfvW~jm0u93JUlO-Tqs2Id2lqVcz1BgOu-UBjlRkQcQdn-LW-sNuiVxmWUbcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197943&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of SD for the 12 LGs for the 8 parental maps, <em>α</em> = 0.001. The vertical line corresponds to a type I error of 0.001. The <em>x</em>-axis corresponds to the <em>χ</em><sup>2</sup> value and the <em>y</em>-axis corresponds to the map length of the LG in cM. The black vertical line corresponds to the <em>χ</em><sup>2</sup> threshold value, orange line corresponds to P1f, yellow line: P1m, light green: P2f, dark green: P2m, pink: P3f, violet: P3m, light blue: P4f and dark blue: P4m.</p></div></div>                            <h2 scrollto-destination=87197944 id="87197944" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87197946 id="87197946" class="section-title" data-legacy-id=s3a>3.1. SNP genotyping</h3>
<p>The 7,913 SNPs were submitted to Illumina for Oligo Pool All (OPA) design for use in the Infinium assay. From the initial set of 7,913 SNPs, 903 (11.4%) did not pass Illumina production quality control and were eliminated. The genotyping data, across the four mapping populations, for the 7,010 SNPs retained were analyzed with Illumina GenomeStudio software, which clusters and calls data automatically, making it possible to visualize the data directly.<sup><a reveal-id="DSW001C14" data-open="DSW001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> For each SNP, the representation of the genotyping data included three main clusters, corresponding to the AA homozygote, the AB heterozygote and the BB homozygote. We obtained four different segregating configurations in the F1 mapping populations: AB×AA (heterozygous in the female parent), AA×AB (heterozygous in the male parent) (Fig. <a reveal-id="DSW001F3" data-open="DSW001F3" class="link link-reveal link-table xref-fig">3</a>a), AB×AB (both parents heterozygous) (Fig. <a reveal-id="DSW001F3" data-open="DSW001F3" class="link link-reveal link-table xref-fig">3</a>b) and AA×AA, BB×BB or AA×BB (both parents homozygous, not informative for genetic mapping). We inspected all the SNPs by eye on the Illumina clusters, making use of the distribution of the segregating full-sibs relative to the parental positions. Observations were carried out individually for each mapping pedigree. The results for the four mapping populations were merged, and 5,726 SNPs were retained (Table <a reveal-id="DSW001TB1" data-open="DSW001TB1" class="link link-reveal link-table xref-fig">1</a>). We discarded SNPs that did not yield well defined, clearly separated clusters (i.e. for which the genotype could not be called unambiguously). We optimized the positions of the segregating loci, by mapping as FMs only the SNPs segregating in a testcross configuration (1:1 ratio). Intercross markers (1:2:1 ratio) are less informative for linkage analysis.<sup><a reveal-id="DSW001C35" data-open="DSW001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> They were therefore excluded from construction of the framework map. </p>                        <a id="87197950" scrollto-destination="87197950"></a>
<div content-id="DSW001TB1" class="table-modal"><div class="table-wrap"><div id="DSW001TB1" class="table-wrap-title" data-id="DSW001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of polymorphic (P), monomorphic (M) and failed (NA) SNPs in the four pedigrees, P1, P2, P3 and P4</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>P1 </th><th>P2 </th><th>P3 </th><th>P4 </th><th>All pedigrees </th></tr></thead><tbody><tr><td>P </td><td>3,515 </td><td>3,734 </td><td>3,399 </td><td>3,418 </td><td>5,726 </td></tr><tr><td>M </td><td>2,012 </td><td>2,120 </td><td>2,488 </td><td>2,222 </td><td>637 </td></tr><tr><td>NA </td><td>1,483 </td><td>1,156 </td><td>1,123 </td><td>1,370 </td><td>647 </td></tr></tbody></table></div><div content-id="DSW001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>P1 </th><th>P2 </th><th>P3 </th><th>P4 </th><th>All pedigrees </th></tr></thead><tbody><tr><td>P </td><td>3,515 </td><td>3,734 </td><td>3,399 </td><td>3,418 </td><td>5,726 </td></tr><tr><td>M </td><td>2,012 </td><td>2,120 </td><td>2,488 </td><td>2,222 </td><td>637 </td></tr><tr><td>NA </td><td>1,483 </td><td>1,156 </td><td>1,123 </td><td>1,370 </td><td>647 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197950" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW001TB1" class="table-wrap-title" data-id="DSW001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of polymorphic (P), monomorphic (M) and failed (NA) SNPs in the four pedigrees, P1, P2, P3 and P4</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>P1 </th><th>P2 </th><th>P3 </th><th>P4 </th><th>All pedigrees </th></tr></thead><tbody><tr><td>P </td><td>3,515 </td><td>3,734 </td><td>3,399 </td><td>3,418 </td><td>5,726 </td></tr><tr><td>M </td><td>2,012 </td><td>2,120 </td><td>2,488 </td><td>2,222 </td><td>637 </td></tr><tr><td>NA </td><td>1,483 </td><td>1,156 </td><td>1,123 </td><td>1,370 </td><td>647 </td></tr></tbody></table></div><div content-id="DSW001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>P1 </th><th>P2 </th><th>P3 </th><th>P4 </th><th>All pedigrees </th></tr></thead><tbody><tr><td>P </td><td>3,515 </td><td>3,734 </td><td>3,399 </td><td>3,418 </td><td>5,726 </td></tr><tr><td>M </td><td>2,012 </td><td>2,120 </td><td>2,488 </td><td>2,222 </td><td>637 </td></tr><tr><td>NA </td><td>1,483 </td><td>1,156 </td><td>1,123 </td><td>1,370 </td><td>647 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197950" target="_blank">View Large</a></div></div>                        <a id="87197952" scrollto-destination="87197952"></a>
<div id="DSW001F3" data-id="DSW001F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00103.jpeg?Expires=2147483647&Signature=o8OMxEFemxkij8OoWJwPrrIERRYcz1eluUVp50LVJ4eq86x7RaLF0L7vZ0Qdy9Ls6tNZerOxU54aYurDz~twfceUQza5S0bLCNgcr2SgM5nxVf5MB6rI1enymWSjBu4SbVJlTp6uh4viksPKZmAq6-ZPrjzeKeZqPT00nix7foam8NHbOZ8J3tgk~-PfE~bdRHH6LTg7izCGnVNQebAbbmt-iK-UlzYmg6upzdjiwJB9bzr67J8tLVKso46~WeZFVhcIfh5WZjcr0y02NFn1USBLV9z2vi251t8hE9eqf5Ii23mnyxny~Pj4tHiFj9gNQPVbwFriOESEa~bfAJ8qDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) AA×AB configuration [one parent is homozygous, the other, heterozygous, two classes of genotypes are observed among the progeny, AA (in red, left part) and AB (violet part, middle part)]. (b) AB×AB configuration [both parents are heterozygous, three classes of genotypes are observed among the progeny, AA (in red, left part) AB (in violet, middle part) and BB (in blue, right part)]. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197952/dsw00103.jpeg" path-from-xml="dsw00103.jpeg" target="_blank">View large</a><a section="87197952" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00103.jpeg?Expires=2147483647&Signature=Qe8h0OM4a1Mt0UiC2~GI0S548kJU~VW7unZLtUAPh29nilwiDeNsMFazvp3mC7AmWRUs~Vjkl7Mint82D-Cl~nMuzbd1nwkdBrJd6BByMcNjRf~tV~uXcveDPI1kDE-BJPekt8DS6YLVUiiMAgSXQtCH-T8W5WhM3nSusHyJT~JVf7-FrlRH0ooq3goC4VYc5e~YhAFkwkLX0x3JbSfVzFrLbjYiA14fV7UvTgTMAZ-s0vvlytNpOiS263uF8ojL2r-gvDJvVbLSvTXPSjyfE7Qmzsa~8DEt8rPAr8uOJck3WgFU3JT4SiaPoWIvDitYdVbvtnIEDk2jpKgSGu6Gtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197952&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) AA×AB configuration [one parent is homozygous, the other, heterozygous, two classes of genotypes are observed among the progeny, AA (in red, left part) and AB (violet part, middle part)]. (b) AB×AB configuration [both parents are heterozygous, three classes of genotypes are observed among the progeny, AA (in red, left part) AB (in violet, middle part) and BB (in blue, right part)]. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00103.jpeg?Expires=2147483647&Signature=o8OMxEFemxkij8OoWJwPrrIERRYcz1eluUVp50LVJ4eq86x7RaLF0L7vZ0Qdy9Ls6tNZerOxU54aYurDz~twfceUQza5S0bLCNgcr2SgM5nxVf5MB6rI1enymWSjBu4SbVJlTp6uh4viksPKZmAq6-ZPrjzeKeZqPT00nix7foam8NHbOZ8J3tgk~-PfE~bdRHH6LTg7izCGnVNQebAbbmt-iK-UlzYmg6upzdjiwJB9bzr67J8tLVKso46~WeZFVhcIfh5WZjcr0y02NFn1USBLV9z2vi251t8hE9eqf5Ii23mnyxny~Pj4tHiFj9gNQPVbwFriOESEa~bfAJ8qDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) AA×AB configuration [one parent is homozygous, the other, heterozygous, two classes of genotypes are observed among the progeny, AA (in red, left part) and AB (violet part, middle part)]. (b) AB×AB configuration [both parents are heterozygous, three classes of genotypes are observed among the progeny, AA (in red, left part) AB (in violet, middle part) and BB (in blue, right part)]. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197952/dsw00103.jpeg" path-from-xml="dsw00103.jpeg" target="_blank">View large</a><a section="87197952" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00103.jpeg?Expires=2147483647&Signature=Qe8h0OM4a1Mt0UiC2~GI0S548kJU~VW7unZLtUAPh29nilwiDeNsMFazvp3mC7AmWRUs~Vjkl7Mint82D-Cl~nMuzbd1nwkdBrJd6BByMcNjRf~tV~uXcveDPI1kDE-BJPekt8DS6YLVUiiMAgSXQtCH-T8W5WhM3nSusHyJT~JVf7-FrlRH0ooq3goC4VYc5e~YhAFkwkLX0x3JbSfVzFrLbjYiA14fV7UvTgTMAZ-s0vvlytNpOiS263uF8ojL2r-gvDJvVbLSvTXPSjyfE7Qmzsa~8DEt8rPAr8uOJck3WgFU3JT4SiaPoWIvDitYdVbvtnIEDk2jpKgSGu6Gtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197952&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) AA×AB configuration [one parent is homozygous, the other, heterozygous, two classes of genotypes are observed among the progeny, AA (in red, left part) and AB (violet part, middle part)]. (b) AB×AB configuration [both parents are heterozygous, three classes of genotypes are observed among the progeny, AA (in red, left part) AB (in violet, middle part) and BB (in blue, right part)]. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87197955 id="87197955" class="section-title" data-legacy-id=s3b>3.2. Construction of eight parental maps and one composite map</h3>
<p>From the 7,010 SNPs passing Illumina production quality control testing, 6,363 SNPs were ‘scorable’ and were used to genotype the four mapping populations.<sup><a reveal-id="DSW001C14" data-open="DSW001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The number of SNPs mapped differed between parental trees, ranging from 1,421 SNPs for P3f to 889 for P4m (Table <a reveal-id="DSW001TB2" data-open="DSW001TB2" class="link link-reveal link-table xref-fig">2</a>). The 12 expected LGs were retrieved for all the parental trees. The size of the genetic maps varied from 684 cM (P4f) to 840 cM (P4m) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw001_Supplementary_Data.zip?Expires=2147483647&Signature=NmivpNrUO-LgVSeMRXc4bQUwZvSC5jnBdoWFQGCXBFaBGk3dN9q2jJr9gQX8i75DhN4EBlVhWXpPza189X~BfsPkFIs9bBGJndENLkuzwBPTL7Sl5Gr-Q8kBkUAtXN7A7umF6N~Des8mg4ECXksZ51WsP~Y0EUVTwX00dcXbRI5HVaVwGrF2bHoZ9Dism3irHcolFemd-sOBkSZKOhR4UuTx-D-gBnuU0k0nOmW7dsgmGmUluO98CiKTl0bJWOaGR7ywe9OziUmdtaNv8P8Uqbrh81fTZYxe7j7YUUO9lyCGi2mDMkqxatbdDkR4aMN7CisXR6iSdYtFdZtsqNMidA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). The number of markers per LG varied from 259 (LG2 for P4f) to 40 (LG4 for P1m) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw001_Supplementary_Data.zip?Expires=2147483647&Signature=NmivpNrUO-LgVSeMRXc4bQUwZvSC5jnBdoWFQGCXBFaBGk3dN9q2jJr9gQX8i75DhN4EBlVhWXpPza189X~BfsPkFIs9bBGJndENLkuzwBPTL7Sl5Gr-Q8kBkUAtXN7A7umF6N~Des8mg4ECXksZ51WsP~Y0EUVTwX00dcXbRI5HVaVwGrF2bHoZ9Dism3irHcolFemd-sOBkSZKOhR4UuTx-D-gBnuU0k0nOmW7dsgmGmUluO98CiKTl0bJWOaGR7ywe9OziUmdtaNv8P8Uqbrh81fTZYxe7j7YUUO9lyCGi2mDMkqxatbdDkR4aMN7CisXR6iSdYtFdZtsqNMidA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). The LGs were all of similar size (mean of 62 ± 11 cM, suggesting the presence of at least one chiasma per chromosome), except for LG2, which was ∼1.5 times longer than the other LGs. Alignment of the eight parental maps obtained for the two species (<em>Q. robur</em> and <em>Q. petraea</em>) revealed a high degree of collinearity between the maps, making it possible to construct a composite map with LPmerge software composed of 4,261 FMs and 129 accessory markers (provided by intercross markers). We noticed that markers at the end of some LGs (LG3, LG4, LG5, LG7, LG8 and LG11) were around 5–14 cM distant from adjacent markers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw001_Supplementary_Data.zip?Expires=2147483647&Signature=NmivpNrUO-LgVSeMRXc4bQUwZvSC5jnBdoWFQGCXBFaBGk3dN9q2jJr9gQX8i75DhN4EBlVhWXpPza189X~BfsPkFIs9bBGJndENLkuzwBPTL7Sl5Gr-Q8kBkUAtXN7A7umF6N~Des8mg4ECXksZ51WsP~Y0EUVTwX00dcXbRI5HVaVwGrF2bHoZ9Dism3irHcolFemd-sOBkSZKOhR4UuTx-D-gBnuU0k0nOmW7dsgmGmUluO98CiKTl0bJWOaGR7ywe9OziUmdtaNv8P8Uqbrh81fTZYxe7j7YUUO9lyCGi2mDMkqxatbdDkR4aMN7CisXR6iSdYtFdZtsqNMidA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>). These markers were found to be present in only one (LG3, LG4, LG7, LG8) or two (for LG5 and LG11) contributing maps and distorted the merged map distances calculated by LPmerge. Therefore, these markers were moved at the position of their nearest adjacent marker calculated by JoinMap on the parental map. This URL allows us to compare LGs from different parental linkage maps: <a class="link link-uri" href="http://w3.pierroton.inra.fr/cgi-bin/cmap/map_details?ref_pmap_set_acc=47;ref_map_accs=1251;comparative_maps=1%3dmap_acc%3d1287;highlight=%22s_1BHPHQ_1326%22">http://w3.pierroton.inra.fr/cgi-bin/cmap/map_details?ref_pmap_set_acc=47;ref_map_accs=1251;comparative_maps=1%3dmap_acc%3d1287;highlight=%22s_1BHPHQ_1326%22</a>. The 12 composite LGs constructed from testcross markers covered 742 cM in total, with individual LG lengths of 51 cM (LG1) to 93 cM (LG2) and a mean density of 1 SNP marker per 0.2 cM (Table <a reveal-id="DSW001TB3" data-open="DSW001TB3" class="link link-reveal link-table xref-fig">3</a>, Fig. <a reveal-id="DSW001F4" data-open="DSW001F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="87197959" scrollto-destination="87197959"></a>
<div content-id="DSW001TB2" class="table-modal"><div class="table-wrap"><div id="DSW001TB2" class="table-wrap-title" data-id="DSW001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Genotyping results for each parental map (testcross markers only) of the four pedigrees (P1–P4), f and m referred to the female and male maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>P1f </th><th>P1m </th><th>P2f </th><th>P2m </th><th>P3f </th><th>P3m </th><th>P4f </th><th>P4m </th></tr></thead><tbody><tr><td>Number of individuals </td><td>369 </td><td>369 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>398 </td><td>398 </td></tr><tr><td>Individuals with &gt;10% missing data </td><td>45 </td><td>45 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>48 </td><td>48 </td></tr><tr><td>Number of individuals considered </td><td>324 </td><td>324 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>350 </td><td>350 </td></tr><tr><td>Total number of SNPs </td><td>1,303 </td><td>1,272 </td><td>1,401 </td><td>1,375 </td><td>1,622 </td><td>1,030 </td><td>1,556 </td><td>969 </td></tr><tr><td>SNPs same contig, different LGs </td><td>0 </td><td>1 </td><td>0 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td></tr><tr><td>SNPs same contig, same map position (replicated SNP) </td><td>131 </td><td>133 </td><td>125 </td><td>142 </td><td>178 </td><td>101 </td><td>159 </td><td>73 </td></tr><tr><td>Analysed SNPs </td><td>1,172 </td><td>1,138 </td><td>1,276 </td><td>1,231 </td><td>1,442 </td><td>928 </td><td>1,395 </td><td>896 </td></tr><tr><td>Ungrouped SNPs </td><td>1 </td><td>0 </td><td>10 </td><td>26 </td><td>21 </td><td>22 </td><td>1 </td><td>6 </td></tr><tr><td>Excluded SNPs </td><td>1 </td><td>2 </td><td>37 </td><td>0 </td><td>0 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>Mapped SNPs </td><td>1,170 </td><td>1,136 </td><td>1,229 </td><td>1,205 </td><td>1,421 </td><td>905 </td><td>1,393 </td><td>889 </td></tr></tbody></table></div><div content-id="DSW001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>P1f </th><th>P1m </th><th>P2f </th><th>P2m </th><th>P3f </th><th>P3m </th><th>P4f </th><th>P4m </th></tr></thead><tbody><tr><td>Number of individuals </td><td>369 </td><td>369 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>398 </td><td>398 </td></tr><tr><td>Individuals with &gt;10% missing data </td><td>45 </td><td>45 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>48 </td><td>48 </td></tr><tr><td>Number of individuals considered </td><td>324 </td><td>324 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>350 </td><td>350 </td></tr><tr><td>Total number of SNPs </td><td>1,303 </td><td>1,272 </td><td>1,401 </td><td>1,375 </td><td>1,622 </td><td>1,030 </td><td>1,556 </td><td>969 </td></tr><tr><td>SNPs same contig, different LGs </td><td>0 </td><td>1 </td><td>0 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td></tr><tr><td>SNPs same contig, same map position (replicated SNP) </td><td>131 </td><td>133 </td><td>125 </td><td>142 </td><td>178 </td><td>101 </td><td>159 </td><td>73 </td></tr><tr><td>Analysed SNPs </td><td>1,172 </td><td>1,138 </td><td>1,276 </td><td>1,231 </td><td>1,442 </td><td>928 </td><td>1,395 </td><td>896 </td></tr><tr><td>Ungrouped SNPs </td><td>1 </td><td>0 </td><td>10 </td><td>26 </td><td>21 </td><td>22 </td><td>1 </td><td>6 </td></tr><tr><td>Excluded SNPs </td><td>1 </td><td>2 </td><td>37 </td><td>0 </td><td>0 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>Mapped SNPs </td><td>1,170 </td><td>1,136 </td><td>1,229 </td><td>1,205 </td><td>1,421 </td><td>905 </td><td>1,393 </td><td>889 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197959" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW001TB2" class="table-wrap-title" data-id="DSW001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Genotyping results for each parental map (testcross markers only) of the four pedigrees (P1–P4), f and m referred to the female and male maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>P1f </th><th>P1m </th><th>P2f </th><th>P2m </th><th>P3f </th><th>P3m </th><th>P4f </th><th>P4m </th></tr></thead><tbody><tr><td>Number of individuals </td><td>369 </td><td>369 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>398 </td><td>398 </td></tr><tr><td>Individuals with &gt;10% missing data </td><td>45 </td><td>45 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>48 </td><td>48 </td></tr><tr><td>Number of individuals considered </td><td>324 </td><td>324 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>350 </td><td>350 </td></tr><tr><td>Total number of SNPs </td><td>1,303 </td><td>1,272 </td><td>1,401 </td><td>1,375 </td><td>1,622 </td><td>1,030 </td><td>1,556 </td><td>969 </td></tr><tr><td>SNPs same contig, different LGs </td><td>0 </td><td>1 </td><td>0 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td></tr><tr><td>SNPs same contig, same map position (replicated SNP) </td><td>131 </td><td>133 </td><td>125 </td><td>142 </td><td>178 </td><td>101 </td><td>159 </td><td>73 </td></tr><tr><td>Analysed SNPs </td><td>1,172 </td><td>1,138 </td><td>1,276 </td><td>1,231 </td><td>1,442 </td><td>928 </td><td>1,395 </td><td>896 </td></tr><tr><td>Ungrouped SNPs </td><td>1 </td><td>0 </td><td>10 </td><td>26 </td><td>21 </td><td>22 </td><td>1 </td><td>6 </td></tr><tr><td>Excluded SNPs </td><td>1 </td><td>2 </td><td>37 </td><td>0 </td><td>0 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>Mapped SNPs </td><td>1,170 </td><td>1,136 </td><td>1,229 </td><td>1,205 </td><td>1,421 </td><td>905 </td><td>1,393 </td><td>889 </td></tr></tbody></table></div><div content-id="DSW001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>P1f </th><th>P1m </th><th>P2f </th><th>P2m </th><th>P3f </th><th>P3m </th><th>P4f </th><th>P4m </th></tr></thead><tbody><tr><td>Number of individuals </td><td>369 </td><td>369 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>398 </td><td>398 </td></tr><tr><td>Individuals with &gt;10% missing data </td><td>45 </td><td>45 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>48 </td><td>48 </td></tr><tr><td>Number of individuals considered </td><td>324 </td><td>324 </td><td>178 </td><td>178 </td><td>114 </td><td>114 </td><td>350 </td><td>350 </td></tr><tr><td>Total number of SNPs </td><td>1,303 </td><td>1,272 </td><td>1,401 </td><td>1,375 </td><td>1,622 </td><td>1,030 </td><td>1,556 </td><td>969 </td></tr><tr><td>SNPs same contig, different LGs </td><td>0 </td><td>1 </td><td>0 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td></tr><tr><td>SNPs same contig, same map position (replicated SNP) </td><td>131 </td><td>133 </td><td>125 </td><td>142 </td><td>178 </td><td>101 </td><td>159 </td><td>73 </td></tr><tr><td>Analysed SNPs </td><td>1,172 </td><td>1,138 </td><td>1,276 </td><td>1,231 </td><td>1,442 </td><td>928 </td><td>1,395 </td><td>896 </td></tr><tr><td>Ungrouped SNPs </td><td>1 </td><td>0 </td><td>10 </td><td>26 </td><td>21 </td><td>22 </td><td>1 </td><td>6 </td></tr><tr><td>Excluded SNPs </td><td>1 </td><td>2 </td><td>37 </td><td>0 </td><td>0 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>Mapped SNPs </td><td>1,170 </td><td>1,136 </td><td>1,229 </td><td>1,205 </td><td>1,421 </td><td>905 </td><td>1,393 </td><td>889 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197959" target="_blank">View Large</a></div></div>                        <a id="87197961" scrollto-destination="87197961"></a>
<div content-id="DSW001TB3" class="table-modal"><div class="table-wrap"><div id="DSW001TB3" class="table-wrap-title" data-id="DSW001TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Features of the composite map constructed with testcross markers. <em>k</em> is a parameter of LPmerge software, varying from 1 to 8, corresponding to the maximum size of the interval</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Number of loci </th><th><em>k</em> var/min RMSE </th><th>LG size (in cM) </th><th>SNP/cM </th></tr></thead><tbody><tr><td>LG1 </td><td>310 </td><td>1 </td><td>51 </td><td>6.1 </td></tr><tr><td>LG2 </td><td>712 </td><td>8 </td><td>93 </td><td>7.7 </td></tr><tr><td>LG3 </td><td>302 </td><td>3 </td><td>63 </td><td>4.8 </td></tr><tr><td>LG4 </td><td>229 </td><td>3 </td><td>63 </td><td>3.6 </td></tr><tr><td>LG5 </td><td>300 </td><td>1 </td><td>64 </td><td>4.7 </td></tr><tr><td>LG6 </td><td>406 </td><td>5 </td><td>63 </td><td>6.4 </td></tr><tr><td>LG7 </td><td>329 </td><td>3 </td><td>56 </td><td>5.9 </td></tr><tr><td>LG8 </td><td>437 </td><td>4 </td><td>67 </td><td>6.5 </td></tr><tr><td>LG9 </td><td>299 </td><td>3 </td><td>58 </td><td>5.2 </td></tr><tr><td>LG10 </td><td>289 </td><td>5 </td><td>58 </td><td>5,0 </td></tr><tr><td>LG11 </td><td>298 </td><td>1 </td><td>54 </td><td>5.6 </td></tr><tr><td>LG12 </td><td>350 </td><td>1 </td><td>53 </td><td>6.6 </td></tr><tr><td>Total </td><td>4,261 </td><td> </td><td>742 </td><td> </td></tr><tr><td>Mean </td><td>355 </td><td> </td><td>62 </td><td>5.7 </td></tr></tbody></table></div><div content-id="DSW001TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Number of loci </th><th><em>k</em> var/min RMSE </th><th>LG size (in cM) </th><th>SNP/cM </th></tr></thead><tbody><tr><td>LG1 </td><td>310 </td><td>1 </td><td>51 </td><td>6.1 </td></tr><tr><td>LG2 </td><td>712 </td><td>8 </td><td>93 </td><td>7.7 </td></tr><tr><td>LG3 </td><td>302 </td><td>3 </td><td>63 </td><td>4.8 </td></tr><tr><td>LG4 </td><td>229 </td><td>3 </td><td>63 </td><td>3.6 </td></tr><tr><td>LG5 </td><td>300 </td><td>1 </td><td>64 </td><td>4.7 </td></tr><tr><td>LG6 </td><td>406 </td><td>5 </td><td>63 </td><td>6.4 </td></tr><tr><td>LG7 </td><td>329 </td><td>3 </td><td>56 </td><td>5.9 </td></tr><tr><td>LG8 </td><td>437 </td><td>4 </td><td>67 </td><td>6.5 </td></tr><tr><td>LG9 </td><td>299 </td><td>3 </td><td>58 </td><td>5.2 </td></tr><tr><td>LG10 </td><td>289 </td><td>5 </td><td>58 </td><td>5,0 </td></tr><tr><td>LG11 </td><td>298 </td><td>1 </td><td>54 </td><td>5.6 </td></tr><tr><td>LG12 </td><td>350 </td><td>1 </td><td>53 </td><td>6.6 </td></tr><tr><td>Total </td><td>4,261 </td><td> </td><td>742 </td><td> </td></tr><tr><td>Mean </td><td>355 </td><td> </td><td>62 </td><td>5.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The composite map with the lowest RMSE (obtained with various values of <em>k</em>) was retained.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197961" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW001TB3" class="table-wrap-title" data-id="DSW001TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Features of the composite map constructed with testcross markers. <em>k</em> is a parameter of LPmerge software, varying from 1 to 8, corresponding to the maximum size of the interval</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Number of loci </th><th><em>k</em> var/min RMSE </th><th>LG size (in cM) </th><th>SNP/cM </th></tr></thead><tbody><tr><td>LG1 </td><td>310 </td><td>1 </td><td>51 </td><td>6.1 </td></tr><tr><td>LG2 </td><td>712 </td><td>8 </td><td>93 </td><td>7.7 </td></tr><tr><td>LG3 </td><td>302 </td><td>3 </td><td>63 </td><td>4.8 </td></tr><tr><td>LG4 </td><td>229 </td><td>3 </td><td>63 </td><td>3.6 </td></tr><tr><td>LG5 </td><td>300 </td><td>1 </td><td>64 </td><td>4.7 </td></tr><tr><td>LG6 </td><td>406 </td><td>5 </td><td>63 </td><td>6.4 </td></tr><tr><td>LG7 </td><td>329 </td><td>3 </td><td>56 </td><td>5.9 </td></tr><tr><td>LG8 </td><td>437 </td><td>4 </td><td>67 </td><td>6.5 </td></tr><tr><td>LG9 </td><td>299 </td><td>3 </td><td>58 </td><td>5.2 </td></tr><tr><td>LG10 </td><td>289 </td><td>5 </td><td>58 </td><td>5,0 </td></tr><tr><td>LG11 </td><td>298 </td><td>1 </td><td>54 </td><td>5.6 </td></tr><tr><td>LG12 </td><td>350 </td><td>1 </td><td>53 </td><td>6.6 </td></tr><tr><td>Total </td><td>4,261 </td><td> </td><td>742 </td><td> </td></tr><tr><td>Mean </td><td>355 </td><td> </td><td>62 </td><td>5.7 </td></tr></tbody></table></div><div content-id="DSW001TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Number of loci </th><th><em>k</em> var/min RMSE </th><th>LG size (in cM) </th><th>SNP/cM </th></tr></thead><tbody><tr><td>LG1 </td><td>310 </td><td>1 </td><td>51 </td><td>6.1 </td></tr><tr><td>LG2 </td><td>712 </td><td>8 </td><td>93 </td><td>7.7 </td></tr><tr><td>LG3 </td><td>302 </td><td>3 </td><td>63 </td><td>4.8 </td></tr><tr><td>LG4 </td><td>229 </td><td>3 </td><td>63 </td><td>3.6 </td></tr><tr><td>LG5 </td><td>300 </td><td>1 </td><td>64 </td><td>4.7 </td></tr><tr><td>LG6 </td><td>406 </td><td>5 </td><td>63 </td><td>6.4 </td></tr><tr><td>LG7 </td><td>329 </td><td>3 </td><td>56 </td><td>5.9 </td></tr><tr><td>LG8 </td><td>437 </td><td>4 </td><td>67 </td><td>6.5 </td></tr><tr><td>LG9 </td><td>299 </td><td>3 </td><td>58 </td><td>5.2 </td></tr><tr><td>LG10 </td><td>289 </td><td>5 </td><td>58 </td><td>5,0 </td></tr><tr><td>LG11 </td><td>298 </td><td>1 </td><td>54 </td><td>5.6 </td></tr><tr><td>LG12 </td><td>350 </td><td>1 </td><td>53 </td><td>6.6 </td></tr><tr><td>Total </td><td>4,261 </td><td> </td><td>742 </td><td> </td></tr><tr><td>Mean </td><td>355 </td><td> </td><td>62 </td><td>5.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The composite map with the lowest RMSE (obtained with various values of <em>k</em>) was retained.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197961" target="_blank">View Large</a></div></div>                        <a id="87197962" scrollto-destination="87197962"></a>
<div id="DSW001F4" data-id="DSW001F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00104.jpeg?Expires=2147483647&Signature=O8y38dThY2lthy62qeeLF2zBklZMW47D5u1ndcUV19g15aqcuPEH1AVvxzoCRPuOD3cwpZSb8gFHYp2vUDpOdOq6jioR26jPcNxQ3IeBDC09AC2ls7WGwyTLvOhEetE023GaCteqynmjBXQSYdGYWaKLSu60fjIlItwIudrmggd4zpOEKJbovUP8QFQfhAtxiq12~a1uta9DWtXY45naRxNpbF0cynR9jXLTRyddQMtrDsbf7XVb9JF-5~kVg-C7arCjc0kBqGMtqGN0RXOdi22naU1K4B7yFD8XgiduST1ibvGyp77Egon4Rk-Aidwf9bKNeEeTPEA59GVXvyYthw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Composite linkage map for the 12 LGs and localization of the 9 segregation distorted regions (SDRs). The SDRs are represented by a vertical colour trait along the LG2, in orange: P1f, LG4, in dark green:, P2m, LG6, left to right, in light blue: P4f, dark blue: P4m, orange: P1f, LG8, in dark green: P2m, LG10, in yellow: P1m, LG11, in dark green: P2m, in violet: P3m). The LG size in cM is given in the LG name. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197962/dsw00104.jpeg" path-from-xml="dsw00104.jpeg" target="_blank">View large</a><a section="87197962" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00104.jpeg?Expires=2147483647&Signature=Ml2~tyZMV-OGSLNb1Oy-h1I78Ah16fBrQIsRPfpOoFf5AMIpLMb2SEp-GC8T4vOCpzpDXbIU2K6NeByx280eAM5wkyCoKlZoGBS9epZfDxcpGYCKHjwMijAqAjvCqmdPPYxooqlfzkXo0oheexseHgBNM6esRgA0Lm7ix1T8zZlTiJ6fqCXlWb8sF0rRsmN-s1nmkpSkc0RozvKLLzICe0jioGTcDKT3zCwZ82p8Am4WBYrPFSMojvO0m5AtneAGoiQyeT3L~p8MhuHFPbSmzeKRPoURa6WMKhydAVvzBkiVU5IWsnRwzPPPJ9riHBoGC85VXPX7~~QxQ2yvQuUqOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197962&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Composite linkage map for the 12 LGs and localization of the 9 segregation distorted regions (SDRs). The SDRs are represented by a vertical colour trait along the LG2, in orange: P1f, LG4, in dark green:, P2m, LG6, left to right, in light blue: P4f, dark blue: P4m, orange: P1f, LG8, in dark green: P2m, LG10, in yellow: P1m, LG11, in dark green: P2m, in violet: P3m). The LG size in cM is given in the LG name. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW001F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/m_dsw00104.jpeg?Expires=2147483647&Signature=O8y38dThY2lthy62qeeLF2zBklZMW47D5u1ndcUV19g15aqcuPEH1AVvxzoCRPuOD3cwpZSb8gFHYp2vUDpOdOq6jioR26jPcNxQ3IeBDC09AC2ls7WGwyTLvOhEetE023GaCteqynmjBXQSYdGYWaKLSu60fjIlItwIudrmggd4zpOEKJbovUP8QFQfhAtxiq12~a1uta9DWtXY45naRxNpbF0cynR9jXLTRyddQMtrDsbf7XVb9JF-5~kVg-C7arCjc0kBqGMtqGN0RXOdi22naU1K4B7yFD8XgiduST1ibvGyp77Egon4Rk-Aidwf9bKNeEeTPEA59GVXvyYthw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Composite linkage map for the 12 LGs and localization of the 9 segregation distorted regions (SDRs). The SDRs are represented by a vertical colour trait along the LG2, in orange: P1f, LG4, in dark green:, P2m, LG6, left to right, in light blue: P4f, dark blue: P4m, orange: P1f, LG8, in dark green: P2m, LG10, in yellow: P1m, LG11, in dark green: P2m, in violet: P3m). The LG size in cM is given in the LG name. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197962/dsw00104.jpeg" path-from-xml="dsw00104.jpeg" target="_blank">View large</a><a section="87197962" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw00104.jpeg?Expires=2147483647&Signature=Ml2~tyZMV-OGSLNb1Oy-h1I78Ah16fBrQIsRPfpOoFf5AMIpLMb2SEp-GC8T4vOCpzpDXbIU2K6NeByx280eAM5wkyCoKlZoGBS9epZfDxcpGYCKHjwMijAqAjvCqmdPPYxooqlfzkXo0oheexseHgBNM6esRgA0Lm7ix1T8zZlTiJ6fqCXlWb8sF0rRsmN-s1nmkpSkc0RozvKLLzICe0jioGTcDKT3zCwZ82p8Am4WBYrPFSMojvO0m5AtneAGoiQyeT3L~p8MhuHFPbSmzeKRPoURa6WMKhydAVvzBkiVU5IWsnRwzPPPJ9riHBoGC85VXPX7~~QxQ2yvQuUqOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197962&ar=1745337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Composite linkage map for the 12 LGs and localization of the 9 segregation distorted regions (SDRs). The SDRs are represented by a vertical colour trait along the LG2, in orange: P1f, LG4, in dark green:, P2m, LG6, left to right, in light blue: P4f, dark blue: P4m, orange: P1f, LG8, in dark green: P2m, LG10, in yellow: P1m, LG11, in dark green: P2m, in violet: P3m). The LG size in cM is given in the LG name. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87197964 id="87197964" class="section-title" data-legacy-id=s3c>3.3. Segregation distortion</h3>
<p>The genome-wide patterns of SD for the eight parental maps are presented in Fig. <a reveal-id="DSW001F2" data-open="DSW001F2" class="link link-reveal link-table xref-fig">2</a>. Overall, 0 (P2f, P3f) to 15% (P2m) of SNPs, depending on the mapping population and the parental tree, displayed significant SD (<em>α</em> = 0.001) (Table <a reveal-id="DSW001TB4" data-open="DSW001TB4" class="link link-reveal link-table xref-fig">4</a>). SD was non-randomly distributed along LGs: nine SDRs were identified on the eight parental maps. These SDRs were unevenly distributed on six LGs. Three types of SDR were observed: (i) SDRs at the ends of LGs (LG2, LG6, LG10), (ii) SDRs in the middle of the LG (LG4, LG8) and (iii) SDRs encompassing the whole LG (LG11) (Figs <a reveal-id="DSW001F2" data-open="DSW001F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSW001F4" data-open="DSW001F4" class="link link-reveal link-table xref-fig">4</a>). The most significant distortions were observed on LG8 (P2m, 96% of markers displayed SD), LG11 (P2m and P3m, 60–88% of loci displaying SD, respectively), LG4 (P2m, 44% of loci displaying SD), LG6 (P1f, P4f and P4m, 20 to 32% of loci displaying SD) and LG10 (P1m, 40% of loci displaying SD) (Table <a reveal-id="DSW001TB5" data-open="DSW001TB5" class="link link-reveal link-table xref-fig">5</a>). We found that 79% of the 359 loci displaying SD belonged to the male parent (<em>χ</em>² test, <em>P-</em>value of 9 × 10<sup>−5</sup>) and 62% belonged to the two interspecific crosses (<em>χ</em>² test, <em>P-</em>value of 1 × 10<sup>−3</sup>). </p>                        <a id="87197968" scrollto-destination="87197968"></a>
<div content-id="DSW001TB4" class="table-modal"><div class="table-wrap"><div id="DSW001TB4" class="table-wrap-title" data-id="DSW001TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Number of loci displaying SD for the eight parental maps, <em>α</em> = 0.001</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pedigree name </th><th>Cross type </th><th>Name of the parents </th><th>Mapping population size </th><th>SNP genotyped </th><th>Distorded loci <em>α</em> = 0.001 </th><th>LG (number of distorted loci) </th><th>Percentage of distorted SNPs </th></tr></thead><tbody><tr><td>P1 </td><td>Intraspecific </td><td>P1f </td><td>324 </td><td>1,170 </td><td>38 </td><td>LG2 (11); LG6 (27) </td><td>3 </td></tr><tr><td>3P×A4 </td><td>F1 </td><td>P1m </td><td> </td><td>1,136 </td><td>25 </td><td>LG7 (4); LG10 (23) </td><td>2 </td></tr><tr><td>P2 </td><td>Interspecific </td><td>P2f </td><td>178 </td><td>1,229 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS28 </td><td>F1 </td><td>P2m </td><td> </td><td>1,205 </td><td>177 </td><td>LG4 (34); LG8 (95); LG11 (48) </td><td>15 </td></tr><tr><td>P3 </td><td>Interspecific </td><td>P3f </td><td>114 </td><td>1,421 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS29 </td><td>F1 </td><td>P3m </td><td> </td><td>905 </td><td>47 </td><td>LG11 (47) </td><td>5 </td></tr><tr><td>P4 </td><td>Intraspecific </td><td>P4f </td><td>350 </td><td>1,393 </td><td>38 </td><td>LG6 (38) </td><td>3 </td></tr><tr><td>QS28×QS21 </td><td>F1 </td><td>P4m </td><td> </td><td>889 </td><td>34 </td><td>LG6 (34) </td><td>4 </td></tr></tbody></table></div><div content-id="DSW001TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Pedigree name </th><th>Cross type </th><th>Name of the parents </th><th>Mapping population size </th><th>SNP genotyped </th><th>Distorded loci <em>α</em> = 0.001 </th><th>LG (number of distorted loci) </th><th>Percentage of distorted SNPs </th></tr></thead><tbody><tr><td>P1 </td><td>Intraspecific </td><td>P1f </td><td>324 </td><td>1,170 </td><td>38 </td><td>LG2 (11); LG6 (27) </td><td>3 </td></tr><tr><td>3P×A4 </td><td>F1 </td><td>P1m </td><td> </td><td>1,136 </td><td>25 </td><td>LG7 (4); LG10 (23) </td><td>2 </td></tr><tr><td>P2 </td><td>Interspecific </td><td>P2f </td><td>178 </td><td>1,229 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS28 </td><td>F1 </td><td>P2m </td><td> </td><td>1,205 </td><td>177 </td><td>LG4 (34); LG8 (95); LG11 (48) </td><td>15 </td></tr><tr><td>P3 </td><td>Interspecific </td><td>P3f </td><td>114 </td><td>1,421 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS29 </td><td>F1 </td><td>P3m </td><td> </td><td>905 </td><td>47 </td><td>LG11 (47) </td><td>5 </td></tr><tr><td>P4 </td><td>Intraspecific </td><td>P4f </td><td>350 </td><td>1,393 </td><td>38 </td><td>LG6 (38) </td><td>3 </td></tr><tr><td>QS28×QS21 </td><td>F1 </td><td>P4m </td><td> </td><td>889 </td><td>34 </td><td>LG6 (34) </td><td>4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>LG for linkage group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197968" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW001TB4" class="table-wrap-title" data-id="DSW001TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Number of loci displaying SD for the eight parental maps, <em>α</em> = 0.001</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pedigree name </th><th>Cross type </th><th>Name of the parents </th><th>Mapping population size </th><th>SNP genotyped </th><th>Distorded loci <em>α</em> = 0.001 </th><th>LG (number of distorted loci) </th><th>Percentage of distorted SNPs </th></tr></thead><tbody><tr><td>P1 </td><td>Intraspecific </td><td>P1f </td><td>324 </td><td>1,170 </td><td>38 </td><td>LG2 (11); LG6 (27) </td><td>3 </td></tr><tr><td>3P×A4 </td><td>F1 </td><td>P1m </td><td> </td><td>1,136 </td><td>25 </td><td>LG7 (4); LG10 (23) </td><td>2 </td></tr><tr><td>P2 </td><td>Interspecific </td><td>P2f </td><td>178 </td><td>1,229 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS28 </td><td>F1 </td><td>P2m </td><td> </td><td>1,205 </td><td>177 </td><td>LG4 (34); LG8 (95); LG11 (48) </td><td>15 </td></tr><tr><td>P3 </td><td>Interspecific </td><td>P3f </td><td>114 </td><td>1,421 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS29 </td><td>F1 </td><td>P3m </td><td> </td><td>905 </td><td>47 </td><td>LG11 (47) </td><td>5 </td></tr><tr><td>P4 </td><td>Intraspecific </td><td>P4f </td><td>350 </td><td>1,393 </td><td>38 </td><td>LG6 (38) </td><td>3 </td></tr><tr><td>QS28×QS21 </td><td>F1 </td><td>P4m </td><td> </td><td>889 </td><td>34 </td><td>LG6 (34) </td><td>4 </td></tr></tbody></table></div><div content-id="DSW001TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Pedigree name </th><th>Cross type </th><th>Name of the parents </th><th>Mapping population size </th><th>SNP genotyped </th><th>Distorded loci <em>α</em> = 0.001 </th><th>LG (number of distorted loci) </th><th>Percentage of distorted SNPs </th></tr></thead><tbody><tr><td>P1 </td><td>Intraspecific </td><td>P1f </td><td>324 </td><td>1,170 </td><td>38 </td><td>LG2 (11); LG6 (27) </td><td>3 </td></tr><tr><td>3P×A4 </td><td>F1 </td><td>P1m </td><td> </td><td>1,136 </td><td>25 </td><td>LG7 (4); LG10 (23) </td><td>2 </td></tr><tr><td>P2 </td><td>Interspecific </td><td>P2f </td><td>178 </td><td>1,229 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS28 </td><td>F1 </td><td>P2m </td><td> </td><td>1,205 </td><td>177 </td><td>LG4 (34); LG8 (95); LG11 (48) </td><td>15 </td></tr><tr><td>P3 </td><td>Interspecific </td><td>P3f </td><td>114 </td><td>1,421 </td><td>0 </td><td> </td><td>0 </td></tr><tr><td>11P×QS29 </td><td>F1 </td><td>P3m </td><td> </td><td>905 </td><td>47 </td><td>LG11 (47) </td><td>5 </td></tr><tr><td>P4 </td><td>Intraspecific </td><td>P4f </td><td>350 </td><td>1,393 </td><td>38 </td><td>LG6 (38) </td><td>3 </td></tr><tr><td>QS28×QS21 </td><td>F1 </td><td>P4m </td><td> </td><td>889 </td><td>34 </td><td>LG6 (34) </td><td>4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>LG for linkage group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197968" target="_blank">View Large</a></div></div>                        <a id="87197970" scrollto-destination="87197970"></a>
<div content-id="DSW001TB5" class="table-modal"><div class="table-wrap"><div id="DSW001TB5" class="table-wrap-title" data-id="DSW001TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Number and coverage of SDRs over the LGs, <em>α</em> = 0.001</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Parental genotype </th><th>LG </th><th>Number of distorted SNP loci </th><th>Proportion of distorted loci (%) </th><th>SDR (in cM) </th><th>LG size (in cM) </th><th>Proportion of distorted LG (%) </th></tr></thead><tbody><tr><td>P1f </td><td>LG2 </td><td>11 </td><td>6 </td><td>1 </td><td>89 </td><td>1 </td></tr><tr><td> </td><td>LG6 </td><td>28 </td><td>28 </td><td>10 </td><td>52 </td><td>20 </td></tr><tr><td>P1m </td><td>LG10 </td><td>23 </td><td>28 </td><td>24 </td><td>59 </td><td>40 </td></tr><tr><td>P2f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P2m </td><td>LG4 </td><td>34 </td><td>61 </td><td>31 </td><td>70 </td><td>44 </td></tr><tr><td> </td><td>LG8 </td><td>95 </td><td>91 </td><td>55 </td><td>58 </td><td>96 </td></tr><tr><td> </td><td>LG11 </td><td>48 </td><td>64 </td><td>34 </td><td>57 </td><td>60 </td></tr><tr><td>P3f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P3m </td><td>LG11 </td><td>47 </td><td>87 </td><td>34 </td><td>38 </td><td>88 </td></tr><tr><td>P4f </td><td>LG6 </td><td>38 </td><td>30 </td><td>17 </td><td>52 </td><td>32 </td></tr><tr><td>P4m </td><td>LG6 </td><td>34 </td><td>33 </td><td>15 </td><td>71 </td><td>21 </td></tr></tbody></table></div><div content-id="DSW001TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Parental genotype </th><th>LG </th><th>Number of distorted SNP loci </th><th>Proportion of distorted loci (%) </th><th>SDR (in cM) </th><th>LG size (in cM) </th><th>Proportion of distorted LG (%) </th></tr></thead><tbody><tr><td>P1f </td><td>LG2 </td><td>11 </td><td>6 </td><td>1 </td><td>89 </td><td>1 </td></tr><tr><td> </td><td>LG6 </td><td>28 </td><td>28 </td><td>10 </td><td>52 </td><td>20 </td></tr><tr><td>P1m </td><td>LG10 </td><td>23 </td><td>28 </td><td>24 </td><td>59 </td><td>40 </td></tr><tr><td>P2f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P2m </td><td>LG4 </td><td>34 </td><td>61 </td><td>31 </td><td>70 </td><td>44 </td></tr><tr><td> </td><td>LG8 </td><td>95 </td><td>91 </td><td>55 </td><td>58 </td><td>96 </td></tr><tr><td> </td><td>LG11 </td><td>48 </td><td>64 </td><td>34 </td><td>57 </td><td>60 </td></tr><tr><td>P3f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P3m </td><td>LG11 </td><td>47 </td><td>87 </td><td>34 </td><td>38 </td><td>88 </td></tr><tr><td>P4f </td><td>LG6 </td><td>38 </td><td>30 </td><td>17 </td><td>52 </td><td>32 </td></tr><tr><td>P4m </td><td>LG6 </td><td>34 </td><td>33 </td><td>15 </td><td>71 </td><td>21 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197970" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW001TB5" class="table-wrap-title" data-id="DSW001TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Number and coverage of SDRs over the LGs, <em>α</em> = 0.001</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Parental genotype </th><th>LG </th><th>Number of distorted SNP loci </th><th>Proportion of distorted loci (%) </th><th>SDR (in cM) </th><th>LG size (in cM) </th><th>Proportion of distorted LG (%) </th></tr></thead><tbody><tr><td>P1f </td><td>LG2 </td><td>11 </td><td>6 </td><td>1 </td><td>89 </td><td>1 </td></tr><tr><td> </td><td>LG6 </td><td>28 </td><td>28 </td><td>10 </td><td>52 </td><td>20 </td></tr><tr><td>P1m </td><td>LG10 </td><td>23 </td><td>28 </td><td>24 </td><td>59 </td><td>40 </td></tr><tr><td>P2f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P2m </td><td>LG4 </td><td>34 </td><td>61 </td><td>31 </td><td>70 </td><td>44 </td></tr><tr><td> </td><td>LG8 </td><td>95 </td><td>91 </td><td>55 </td><td>58 </td><td>96 </td></tr><tr><td> </td><td>LG11 </td><td>48 </td><td>64 </td><td>34 </td><td>57 </td><td>60 </td></tr><tr><td>P3f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P3m </td><td>LG11 </td><td>47 </td><td>87 </td><td>34 </td><td>38 </td><td>88 </td></tr><tr><td>P4f </td><td>LG6 </td><td>38 </td><td>30 </td><td>17 </td><td>52 </td><td>32 </td></tr><tr><td>P4m </td><td>LG6 </td><td>34 </td><td>33 </td><td>15 </td><td>71 </td><td>21 </td></tr></tbody></table></div><div content-id="DSW001TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Parental genotype </th><th>LG </th><th>Number of distorted SNP loci </th><th>Proportion of distorted loci (%) </th><th>SDR (in cM) </th><th>LG size (in cM) </th><th>Proportion of distorted LG (%) </th></tr></thead><tbody><tr><td>P1f </td><td>LG2 </td><td>11 </td><td>6 </td><td>1 </td><td>89 </td><td>1 </td></tr><tr><td> </td><td>LG6 </td><td>28 </td><td>28 </td><td>10 </td><td>52 </td><td>20 </td></tr><tr><td>P1m </td><td>LG10 </td><td>23 </td><td>28 </td><td>24 </td><td>59 </td><td>40 </td></tr><tr><td>P2f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P2m </td><td>LG4 </td><td>34 </td><td>61 </td><td>31 </td><td>70 </td><td>44 </td></tr><tr><td> </td><td>LG8 </td><td>95 </td><td>91 </td><td>55 </td><td>58 </td><td>96 </td></tr><tr><td> </td><td>LG11 </td><td>48 </td><td>64 </td><td>34 </td><td>57 </td><td>60 </td></tr><tr><td>P3f </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>P3m </td><td>LG11 </td><td>47 </td><td>87 </td><td>34 </td><td>38 </td><td>88 </td></tr><tr><td>P4f </td><td>LG6 </td><td>38 </td><td>30 </td><td>17 </td><td>52 </td><td>32 </td></tr><tr><td>P4m </td><td>LG6 </td><td>34 </td><td>33 </td><td>15 </td><td>71 </td><td>21 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197970" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87197971 id="87197971" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=87197973 id="87197973" class="section-title" data-legacy-id=s4a>4.1. Construction of a high-density gene-based linkage map and application in genetic studies in oak and beyond</h3>
<p>We report here the development and validation of the first high-throughput Illumina SNP genotyping assay for oaks, with a success rate of 88.6%, i.e. 7,010 of the 7,913 SNPs were successfully genotyped. Overall, 82% of the SNPs successfully genotyped were polymorphic in at least one of the four pedigrees and 63% were mapped as FMs on the parental genetic maps.</p><p>The use of multiple segregating populations of diverse genetic backgrounds made it possible to map a larger number of markers and to achieve greater genome coverage. This has already been illustrated in watermelon, for example, in which the genotyping of four mapping populations increased the proportion of mapped loci from the most polymorphic pedigree by 29%.<sup><a reveal-id="DSW001C36" data-open="DSW001C36" class="link link-ref link-reveal xref-bibr">36</a></sup> In this study, the gain in terms of the number of newly segregating test-cross markers from the pedigree with the largest number of SNPs (2,472 in P2) with respect to the other three pedigrees was remarkably high, reaching +1,756 SNPs (i.e. +41% newly mapped markers).</p><p>The development of array-based genotyping technologies has made it possible to generate high-density linkage maps rapidly, but the integration of independent maps containing thousands of loci remains a real challenge.<sup><a reveal-id="DSW001C37  DSW001C39" data-open="DSW001C37  DSW001C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup> Two main approaches have been developed for the construction of combined maps. The first involves pooling genotypic data and minimizing the sum of recombination frequencies, as proposed in the ML method implemented in JoinMap (e.g. as used in triticale by Alheit <em>et al</em>.<sup><a reveal-id="DSW001C37" data-open="DSW001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and in sunflower by Bowers <em>et al</em>.).<sup><a reveal-id="DSW001C38" data-open="DSW001C38" class="link link-ref link-reveal xref-bibr">38</a></sup> However, this method is time-consuming and may be not appropriate when thousands of markers are used, due to the computational time required. The second approach involves the direct integration of independent linkage maps, as proposed in MergeMap<sup><a reveal-id="DSW001C40" data-open="DSW001C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSW001C41" data-open="DSW001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and LPmerge<sup><a reveal-id="DSW001C34" data-open="DSW001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and recently used in pine,<sup><a reveal-id="DSW001C39" data-open="DSW001C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSW001C42" data-open="DSW001C42" class="link link-ref link-reveal xref-bibr">42</a></sup> barley<sup><a reveal-id="DSW001C43" data-open="DSW001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and wheat.<sup><a reveal-id="DSW001C44" data-open="DSW001C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Using this second approach, we merged eight linkage maps each containing 889–1,421 markers, to obtain a composite map including 4,261 loci corresponding to 4,239 different contigs of the oak UniGene database<sup><a reveal-id="DSW001C45" data-open="DSW001C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and covering 742 cM on the 12 LGs. This map is considerably denser than a previously published linkage map based on 397 EST and genomic SSRs.<sup><a reveal-id="DSW001C13" data-open="DSW001C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Framework maps were constructed exclusively from markers with a testcross configuration. All intercross markers were excluded from construction of the framework map because they provide little information about linkage.<sup><a reveal-id="DSW001C35" data-open="DSW001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> We added markers segregating in the intercross configuration at a later stage, as accessory markers. The eight parental maps displayed remarkably high degrees of collinearity and no chromosomal rearrangements were observed, providing support for our approach of merging maps to construct a unified composite map for the genus <em>Quercus,</em> including both species maps. In the framework of the oak genome sequencing project,<sup><a reveal-id="DSW001C46" data-open="DSW001C46" class="link link-ref link-reveal xref-bibr">46</a></sup> this genomic resource will be crucial for the assembly of genome scaffolds into chromosomal pseudomolecules. High-density sequence-based linkage maps have been used to anchor and orient scaffolds in many plant species, including <em>Cicer arietinum</em> (chickpea),<sup><a reveal-id="DSW001C47" data-open="DSW001C47" class="link link-ref link-reveal xref-bibr">47</a></sup><em>Rubus</em> (raspberry)<sup><a reveal-id="DSW001C48" data-open="DSW001C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and <em>Eucalyptus</em>.<sup><a reveal-id="DSW001C49" data-open="DSW001C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The oak composite linkage map also provides a framework for genome-wide analysis at the centimorgan scale, for genomic scans of species/population divergence,<sup><a reveal-id="DSW001C50" data-open="DSW001C50" class="link link-ref link-reveal xref-bibr">50</a></sup> studies of the evolutionary relationships between related species,<sup><a reveal-id="DSW001C16" data-open="DSW001C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW001C51" data-open="DSW001C51" class="link link-ref link-reveal xref-bibr">51</a></sup> the detection of recombination hot and cold spots,<sup><a reveal-id="DSW001C52" data-open="DSW001C52" class="link link-ref link-reveal xref-bibr">52</a></sup> studies of the extent of long-distance linkage disequilibrium and genetic diversity,<sup><a reveal-id="DSW001C39" data-open="DSW001C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSW001C53" data-open="DSW001C53" class="link link-ref link-reveal xref-bibr">53</a></sup> the detection and positional characterization of QTLs through co-localization with gene-based markers (e.g.<sup><a reveal-id="DSW001C54  DSW001C56" data-open="DSW001C54  DSW001C56" class="link link-ref link-reveal xref-bibr">54–56</a></sup>), and the identification of chromosomal rearrangements.<sup><a reveal-id="DSW001C57" data-open="DSW001C57" class="link link-ref link-reveal xref-bibr">57</a></sup> This unified linkage map for the genus <em>Quercus</em> will also be useful for analyses of synteny and collinearity within the Fagaceae at a much higher resolution than previously reported (e.g. between <em>Quercus</em> and <em>Castanea</em>),<sup><a reveal-id="DSW001C13" data-open="DSW001C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSW001C14" data-open="DSW001C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSW001C58" data-open="DSW001C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and for extending such analyses to other Eurosids, providing insight into genome evolution<sup><a reveal-id="DSW001C59" data-open="DSW001C59" class="link link-ref link-reveal xref-bibr">59</a></sup> and a framework for the transfer of genetic information between species.</p>                            <h3 scrollto-destination=87197982 id="87197982" class="section-title" data-legacy-id=s4b>4.2. Patterns of SD identify gametic incompatibility as a major RI barrier in oak</h3>
<p>Deviation from the ratios expected for Mendelian inheritance reveal disturbances in the transmission of genetic information from one generation to the next, generating interesting hypotheses about the mechanisms underlying RI for exploration in further studies. Overall, our results revealed that SD was widespread in the oak genome. Regardless of the species or cross, half of the 12 LGs displayed SD, for 6–91% of markers, depending on the LG. We observed large differences in SD values between intra- and interspecific crosses, and between male and female parents.</p><p>These SDRs may result from chromosome loss or rearrangements, genetic load or pre- or post-zygotic selection, and interpretation may differ between intra- and interspecific crosses.</p><p>Peculiar life history traits of trees may raise their genetic load and result in substantial SD. Oaks, like most forest trees species, form very large populations, and generally outcross through wind pollination, resulting in high levels of gene flow.<sup><a reveal-id="DSW001C60" data-open="DSW001C60" class="link link-ref link-reveal xref-bibr">60</a>,<a reveal-id="DSW001C61" data-open="DSW001C61" class="link link-ref link-reveal xref-bibr">61</a></sup> These characteristics lead to the accumulation of deleterious mutations and the build-up of a large genetic load.<sup><a reveal-id="DSW001C27" data-open="DSW001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The accumulation of recessive deleterious mutations in fitness genes (pollen fertility, anther receptivity, seed fertility) or loci closely linked to fitness genes decreases the viability of plants with homozygous at these loci.<sup><a reveal-id="DSW001C62" data-open="DSW001C62" class="link link-ref link-reveal xref-bibr">62</a></sup> Genetic load may be partially purged at early embryonic stages, by the death or sterility of hybrids carrying homozygous recessive deleterious mutations. Genetic load varies considerably between plant species and may be a major source of SDRs.<sup><a reveal-id="DSW001C17" data-open="DSW001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> For the intraspecific pedigree P4, we observed SDRs on LG6 that were conserved between the two parental maps. This could indicate the presence of lethal or sublethal genes compromising seedling survival in both parental genotypes.<sup><a reveal-id="DSW001C48" data-open="DSW001C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p><p>Gametic incompatibility and/or reduced hybrid viability can also contribute to SD. In this study, the proportion of loci displaying SD differed between the eight linkage maps (0–14.5%). The frequency of SD was higher on three LGs: LG6 (28% of loci displaying SD distributed over 16 cM, for three parents), LG 8 (26.5%, 55 cM, one parent) and LG11 (26.5%, 33 cM, two parents). In several previous studies (eucalyptus,<sup><a reveal-id="DSW001C63" data-open="DSW001C63" class="link link-ref link-reveal xref-bibr">63</a></sup> rice<sup><a reveal-id="DSW001C19" data-open="DSW001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and monkeyflower),<sup><a reveal-id="DSW001C20" data-open="DSW001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> clusters of distorted loci were found to extend over all or most of the LG. In rice, for example, distortion gradually decreases with increasing distance from the markers displaying the highest levels of SD, located at or near previously reported gametophytic gene loci or sterility loci. In our study, 79% of the loci presenting SD were of paternal origin whatever the type of cross (intra- or inter-specific). LG11 in the cross with QS29 as the male parent (P3m) provides an extreme example, with an SDR encompassing 88% of the SNPs on this LG, as previously reported in a study with far fewer EST–SSR markers and a much smaller number of offspring.<sup><a reveal-id="DSW001C13" data-open="DSW001C13" class="link link-ref link-reveal xref-bibr">13</a></sup> These figures may reflect strong pollen incompatibilities between the parents of the different crosses, as previously reported by Abadie <em>et al</em>.<sup><a reveal-id="DSW001C64" data-open="DSW001C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Indeed, the results obtained for one genotype (QS28), used as either the male or the female parent in controlled crosses, strongly support the observed trend: the number of markers displaying SD was five times higher when this genotype was used as the male parent (P2m) than when it was used as the female parent (P4f). A confounding effect of the type of cross may also have contributed to the observed pattern (see next paragraph) because QS28 was used as the male in the interspecific cross and as the female in the intraspecific cross. Male gametophytic selection has been identified as the phenomenon most frequently causing skewed segregation, due to selective influences of the gynoecium, resulting in genetic incompatibility.<sup><a reveal-id="DSW001C19" data-open="DSW001C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>Interestingly, 62% of the loci displaying SD were derived from interspecific crosses and were only detected in the male LGs. SD is frequently observed in interspecific crosses, and has been attributed to biological factors, such as pollen–pistil incompatibility, hybrid viability, sterility due to gametophytic competition, negative epistatic interactions between alleles or positive introgression.<sup><a reveal-id="DSW001C17" data-open="DSW001C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSW001C21" data-open="DSW001C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Pre-reproductive barriers play a major role in the directionality of introgression: genetically based pollen discrimination is a major barrier, as it greatly increases assortative mating within species and the parental species fidelity of hybrids.<sup><a reveal-id="DSW001C65" data-open="DSW001C65" class="link link-ref link-reveal xref-bibr">65</a></sup> Alternatively, natural selection may play a key role by acting against unfit genetic combinations. Lepais and Gerber<sup><a reveal-id="DSW001C66" data-open="DSW001C66" class="link link-ref link-reveal xref-bibr">66</a></sup> observed lower levels of mating success for interspecific crosses compared with intraspecific crosses or backcross mating events, in a mixed stand of four European white oak species (including Qr and Qp). They clearly showed that the different species contributed unequally to reproduction success through differences in pollen efficacy. Most pure-bred plants reproduce preferentially with conspecific individuals. These findings were confirmed by another independent hybridization study conducted in a mixed stand of four European white oak species, Qr, Qp, <em>Quercus pubescens</em> and <em>Quercus frainetto</em>.<sup><a reveal-id="DSW001C1" data-open="DSW001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> This previous study highlighted the importance of selection against hybrids, resulting in the maintenance of four distinct parental gene pools in sympatry. Based on these observations, we suggest that gametic incompatibility could lead to SD in chromosomal regions on either the female or the male map, whereas reduced hybrid variability would be likely to cause SD in the corresponding regions of both parental maps. Most of the observed SDRs were identified in only one parent, generally the male. Thus, a large fraction of the SDRs were sex-specific, suggesting that gametic selection plays an important role in shaping SDRs in oak.</p>                            <h2 scrollto-destination=87197991 id="87197991" class="section-title" data-legacy-id=s5>5. Conclusion</h2>
<p>Our study demonstrates the relevance of Illumina technology for SNP genotyping for multipedigree studies in oaks. The high rate of successful SNP development for the six different parental genotypes used in the four controlled crosses provided us with a very large set of mappable SNPs, which was essential for comparative mapping and construction of the oak high-density composite linkage map.</p><p>We established a high-density composite linkage map based on more than 4,261 SNP loci suitable for use as a reference gene-based map for the genus <em>Quercus</em> and for the Fagaceae in general. Genomic resources are available for very few species within the Fagaceae family and the map developed here could be useful for studies of species from the same genus or for related genera belonging to this family.</p><p>Finally, we identified regions of SD potentially related to RI or genetic load. Further studies assessing seed abortion and the viability of young hybrid seedlings during juvenile development are required to shed light on the causal mechanisms underlying RI. Additional investigations of the co-localization between SDRs, QTLs for adaptive traits<sup><a reveal-id="DSW001C67" data-open="DSW001C67" class="link link-ref link-reveal xref-bibr">67</a>,<a reveal-id="DSW001C68" data-open="DSW001C68" class="link link-ref link-reveal xref-bibr">68</a></sup> and species divergence hot spots should also be carried out. The recent availability of a whole-genome sequence for <em>Quercus</em><sup><a reveal-id="DSW001C46" data-open="DSW001C46" class="link link-ref link-reveal xref-bibr">46</a></sup> will finally help to identify genes located in and underlying SDRs and the genes involved in RI.</p>                            <h2 scrollto-destination=87198000 id="87198000" class="section-title" data-legacy-id=s6>Authors' contributions</h2>
<p>C.B. performed the genetic analysis, E.C. extracted the DNA, E.C. and C.B. carried out the genotyping analysis, F.E. updated the databases, C.B. and C.P. wrote the manuscript. C.B., C.P. and A.K. conceived and designed the project. All authors read and approved the final manuscript.</p>                            <h2 scrollto-destination=87198003 id="87198003" class="section-title" data-legacy-id=s8>Funding</h2>
<p>The study was funded by the European Commission, as part of the FP5 OAKFLOW project (Intra and interspecific geneflow in oaks as mechanisms promoting genetic diversity and adaptive potential, N°QLK5-2000-00960), the FP6 program (FP6-2004-GLOBAL-3, Network of Excellence EVOLTREE ‘Evolution of Trees as Drivers of Terrestrial Biodiversity’, No. 016322). Funding to pay the Open Access publication charges for this article was provided by Treepeace (European Research Council Advanced Grant FP7-339728).</p>                            <h2 scrollto-destination=87198007 id="87198007" class="section-title" data-legacy-id=s9>Web portal</h2>
<p><em>Quercus</em> portal:</p><p>https://w3.pierroton.inra.fr/QuercusPortal/</p><p>CMap Comparative Map Viewer for the composite LGs:</p><p><a class="link link-uri" href="http://w3.pierroton.inra.fr/cgi-bin/cmap/viewer?ref_map_set_acc=51&amp;ref_map_accs=-1">http://w3.pierroton.inra.fr/cgi-bin/cmap/viewer?ref_map_set_acc=51&amp;ref_map_accs=-1</a></p><p>CMap Comparative Map Viewer for different parental LGs:</p><p><a class="link link-uri" href="http://w3.pierroton.inra.fr/cgi-bin/cmap/map_details?ref_map_set_acc=47;ref_map_accs=1251;comparative_maps=1%3dmap_acc%3d1287;highlight=%22s_1BHPHQ_1326%22">http://w3.pierroton.inra.fr/cgi-bin/cmap/map_details?ref_map_set_acc=47;ref_map_accs=1251;comparative_maps=1%3dmap_acc%3d1287;highlight=%22s_1BHPHQ_1326%22</a></p>                            <h2 scrollto-destination=87198020 id="87198020" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Benjamin Dencausse and Guy Roussel for technical support for the creation and monitoring of the controlled crosses and the team of the INRA experimental unit for assistance with field work. We thank Cyril Firmat, Jérôme Bartholomé and Hélène Lagraulet for their help for the development of R scripts.</p>                            <h2 scrollto-destination=87198029 id="87198029" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Curtu</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Gailing</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Finkeldey</div> <div class="given-names">R.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Evidence for hybridization and introgression within a species-rich oak (Quercus spp.) community</div>, <div class="source">BMC Evol. Biol.</div> , <div class="volume">7</div>, <div class="fpage">218</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20hybridization%20and%20introgression%20within%20a%20species-rich%20oak%20(Quercus%20spp.)%20community&amp;author=A.%20Curtu&amp;author=O.%20Gailing&amp;author=R.%20Finkeldey&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-218"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guichoux</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Garnier-Géré</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Lagache</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Lang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Boury</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Petit</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Outlier loci highlight the direction of introgression in oaks</div>, <div class="source">Mol. Ecol.</div> , <div class="volume">22</div>, <div class="fpage">450</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Outlier%20loci%20highlight%20the%20direction%20of%20introgression%20in%20oaks&amp;author=E.%20Guichoux&amp;author=P.%20Garnier-G%C3%A9r%C3%A9&amp;author=L.%20Lagache&amp;author=T.%20Lang&amp;author=C.%20Boury&amp;author=R.J.%20Petit&amp;publication_year=2013&amp;journal=Mol.%20Ecol.&amp;volume=22&amp;pages=450-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23190431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lepais</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Roussel</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Hubert</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gerber</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Strength and variability of postmating reproductive isolating barriers between four European white oak species</div>, <div class="source">Tree Genet. Genomics</div> , <div class="volume">9</div>, <div class="fpage">841</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strength%20and%20variability%20of%20postmating%20reproductive%20isolating%20barriers%20between%20four%20European%20white%20oak%20species&amp;author=O.%20Lepais&amp;author=G.%20Roussel&amp;author=F.%20Hubert&amp;author=A.%20Kremer&amp;author=S.%20Gerber&amp;publication_year=2013&amp;journal=Tree%20Genet.%20Genomics&amp;volume=9&amp;pages=841-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-013-0602-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-013-0602-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-013-0602-3"> </span></div> </div></div></div></div></div><div content-id="DSW001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goicoechea</div>
               <div class="given-names">P.G.</div>
            </div>, <div class="name string-name"><div class="surname">Herrán</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Durand</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Plomion</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">A linkage disequilibrium perspective on the genetic mosaic of speciation in two hybridizing Mediterranean white oaks</div>, <div class="source">Heredity</div> , <div class="volume">114</div>, <div class="fpage">373</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20linkage%20disequilibrium%20perspective%20on%20the%20genetic%20mosaic%20of%20speciation%20in%20two%20hybridizing%20Mediterranean%20white%20oaks&amp;author=P.G.%20Goicoechea&amp;author=A.%20Herr%C3%A1n&amp;author=J.%20Durand&amp;author=C.%20Bod%C3%A9n%C3%A8s&amp;author=C.%20Plomion&amp;author=A.%20Kremer&amp;publication_year=2015&amp;journal=Heredity&amp;volume=114&amp;pages=373-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2014.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2014.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2014.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25515016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kremer</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Dupouey</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Deans</div> <div class="given-names">J.D.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Leaf morphological differentiation between <em>Quercus robur</em> and <em>Quercus petraea</em> is stable across western European mixed oak stands</div>, <div class="source">Ann. For. Sci.</div> , <div class="volume">59</div>, <div class="fpage">777</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Leaf%20morphological%20differentiation%20between%20Quercus%20robur%20and%20Quercus%20petraea%20is%20stable%20across%20western%20European%20mixed%20oak%20stands&amp;author=A.%20Kremer&amp;author=J.L.%20Dupouey&amp;author=J.D.%20Deans&amp;publication_year=2002&amp;journal=Ann.%20For.%20Sci.&amp;volume=59&amp;pages=777-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/forest:2002065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fforest:2002065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fforest:2002065"> </span></div> </div></div></div></div></div><div content-id="DSW001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parelle</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Brendel</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Differences in morphological and physiological responses to water-logging between two sympatric oak species (<em>Quercus petraea</em> [Matt.] Liebl., <em>Quercus robur</em> L.)</div>, <div class="source">Ann. Sci. For.</div> , <div class="volume">63</div>, <div class="fpage">849</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20morphological%20and%20physiological%20responses%20to%20water-logging%20between%20two%20sympatric%20oak%20species%20(Quercus%20petraea%20%5BMatt.%5D%20Liebl.%2C%20Quercus%20robur%20L.)&amp;author=J.%20Parelle&amp;author=O.%20Brendel&amp;author=C.%20Bod%C3%A9n%C3%A8s&amp;publication_year=2006&amp;journal=Ann.%20Sci.%20For.&amp;volume=63&amp;pages=849-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/forest:2006068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fforest:2006068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fforest:2006068"> </span></div> </div></div></div></div></div><div content-id="DSW001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ducousso</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Petit</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Le point sur les chênes blancs européens</div>, <div class="source">Forêt Entreprise</div> , <div class="volume">112</div>, <div class="fpage">49</div>–<div class="lpage">56</div>.</div></div></div></div><div content-id="DSW001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ponton</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Dupouey</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Bréda</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Feuillat</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dreyer</div> <div class="given-names">E.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Carbon isotope discrimination and wood anatomy variations in mixed stands of <em>Quercus robur</em> and <em>Quercus petraea</em></div><em>,</em><div class="source">Plant Cell Environment</div> , <div class="volume">24</div>, <div class="fpage">861</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbon%20isotope%20discrimination%20and%20wood%20anatomy%20variations%20in%20mixed%20stands%20of%20Quercus%20robur%20and%20Quercus%20petraea&amp;author=S.%20Ponton&amp;author=J.L.%20Dupouey&amp;author=N.%20Br%C3%A9da&amp;author=F.%20Feuillat&amp;author=C.%20Bod%C3%A9n%C3%A8s&amp;author=E.%20Dreyer&amp;publication_year=2001&amp;journal=Plant%20Cell%20Environment&amp;volume=24&amp;pages=861-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.0016-8025.2001.00733.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.0016-8025.2001.00733.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.0016-8025.2001.00733.x"> </span></div> </div></div></div></div></div><div content-id="DSW001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Petit</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ducousso</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Roussel</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Hybridization as a mechanism of invasion in oaks</div>, <div class="source">New Phytol.</div> , <div class="volume">161</div>, <div class="fpage">151</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybridization%20as%20a%20mechanism%20of%20invasion%20in%20oaks&amp;author=R.J.%20Petit&amp;author=C.%20Bod%C3%A9n%C3%A8s&amp;author=A.%20Ducousso&amp;author=G.%20Roussel&amp;author=A.%20Kremer&amp;publication_year=2003&amp;journal=New%20Phytol.&amp;volume=161&amp;pages=151-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1469-8137.2003.00944.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1469-8137.2003.00944.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1469-8137.2003.00944.x"> </span></div> </div></div></div></div></div><div content-id="DSW001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bacilieri</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Ducousso</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Petit</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Mating system and asymetric hybridization in a mixed stand of European oaks</div>, <div class="source">Evolution</div> , <div class="volume">50</div>, <div class="fpage">900</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mating%20system%20and%20asymetric%20hybridization%20in%20a%20mixed%20stand%20of%20European%20oaks&amp;author=R.%20Bacilieri&amp;author=A.%20Ducousso&amp;author=R.J.%20Petit&amp;author=A.%20Kremer&amp;publication_year=1996&amp;journal=Evolution&amp;volume=50&amp;pages=900-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2410861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2410861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2410861"> </span></div> </div></div></div></div></div><div content-id="DSW001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barreneche</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Casasoli</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Comparative mapping between <em>Quercus</em> and Castanea using simple-sequence repeats (SSRs)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">108</div>, <div class="fpage">558</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20between%20Quercus%20and%20Castanea%20using%20simple-sequence%20repeats%20(SSRs)&amp;author=T.%20Barreneche&amp;author=M.%20Casasoli&amp;author=K.%20Russell&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=558-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1462-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1462-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1462-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14564395" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Scotti-Saintagne</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Mariette</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Porth</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Genome scanning for interspecific differentiation between two closely related oak species [<em>Quercus robur</em> L. and <em>Q. petraea</em> (Matt.) Liebl.]</div>, <div class="source">Genetics</div> , <div class="volume">168</div>, <div class="fpage">1615</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20scanning%20for%20interspecific%20differentiation%20between%20two%20closely%20related%20oak%20species%20%5BQuercus%20robur%20L.%20and%20Q.%20petraea%20(Matt.)%20Liebl.%5D&amp;author=C.%20Scotti-Saintagne&amp;author=S.%20Mariette&amp;author=I.%20Porth&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=1615-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.026849" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.026849" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.026849"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15579711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bodénès</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Chancerel</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Murat</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Comparative mapping in the Fagaceae and beyond using EST-SSRs</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">12</div>, <div class="fpage">153</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20in%20the%20Fagaceae%20and%20beyond%20using%20EST-SSRs&amp;author=C.%20Bod%C3%A9n%C3%A8s&amp;author=E.%20Chancerel&amp;author=F.%20Murat&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22931513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lepoittevin</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Chancerel</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Single-nucleotide polymorphism discovery and validation in high density SNP array for genetic analysis in European white oaks</div>, <div class="source">Mol. Ecol. Res.</div> , <div class="pub-id-doi"><a href="https://doi.org/10.1111/1755-0998.12407" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/1755-0998.12407</a></div>.</div></div></div></div><div content-id="DSW001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Kilian</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Construction of a high-density composite map and comparative mapping of segregation distortion regions in barley</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">284</div>, <div class="fpage">319</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20composite%20map%20and%20comparative%20mapping%20of%20segregation%20distortion%20regions%20in%20barley&amp;author=H.%20Li&amp;author=A.%20Kilian&amp;author=M.%20Zhou&amp;publication_year=2010&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=284&amp;pages=319-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-010-0570-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-010-0570-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-010-0570-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20803217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ollitraut</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">A reference genetic map of <em>C. clementina</em> hort. ex Tan.; citrus evolution inferences from comparative mapping</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">593</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20reference%20genetic%20map%20of%20C.%20clementina%20hort.%20ex%20Tan.%3B%20citrus%20evolution%20inferences%20from%20comparative%20mapping&amp;author=P.%20Ollitraut&amp;author=J.%20Terol&amp;author=C.%20Chen&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=593" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-593" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-593" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-593"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23126659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Myburg</div>
               <div class="given-names">A.A.</div>
            </div>, <div class="name string-name"><div class="surname">Vogl</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Griffin</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Sederoff</div> <div class="given-names">R.R.</div></div>, <div class="name string-name"><div class="surname">Whetten</div> <div class="given-names">R.W.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Genetics of postzygotic isolation in Eucalyptus: whole-genome analysis of barriers to introgression in a wide interspecific cross of <em>Eucalyptus grandis</em> and <em>E. globulus</em></div><em>,</em><div class="source">Genetics</div> , <div class="volume">166</div>, <div class="fpage">1405</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20postzygotic%20isolation%20in%20Eucalyptus%3A%20whole-genome%20analysis%20of%20barriers%20to%20introgression%20in%20a%20wide%20interspecific%20cross%20of%20Eucalyptus%20grandis%20and%20E.%20globulus&amp;author=A.A.%20Myburg&amp;author=C.%20Vogl&amp;author=A.R.%20Griffin&amp;author=R.R.%20Sederoff&amp;author=R.W.%20Whetten&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=1405-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.166.3.1405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.3.1405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.166.3.1405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15082559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Romero-Severson</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bernardo</div> <div class="given-names">R.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Chromosomal regions associated with segregation distortion in maize</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">105</div>, <div class="fpage">622</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20regions%20associated%20with%20segregation%20distortion%20in%20maize&amp;author=H.%20Lu&amp;author=J.%20Romero-Severson&amp;author=R.%20Bernardo&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=105&amp;pages=622-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0970-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0970-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0970-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">McCouch</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Chromosomal regions associated with segregation distortion of molecular markers in F2, backcross, doubled haploid, and recombinant inbred populations in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="volume">253</div>, <div class="fpage">535</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20regions%20associated%20with%20segregation%20distortion%20of%20molecular%20markers%20in%20F2%2C%20backcross%2C%20doubled%20haploid%2C%20and%20recombinant%20inbred%20populations%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=Y.%20Xu&amp;author=L.%20Zhu&amp;author=J.%20Xiao&amp;author=N.%20Huang&amp;author=S.R.%20McCouch&amp;publication_year=1997&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=253&amp;pages=535-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380050355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380050355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380050355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9065686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fishman</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Willis</div> <div class="given-names">J.H.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Evidence for Dobzhansky-Muller incompatibilities contributing to the sterility of hybrids between <em>Mimulus guttatus</em> and <em>M. nasatus</em></div>, <div class="source">Evolution</div> , <div class="volume">55</div>, <div class="fpage">1932</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20Dobzhansky-Muller%20incompatibilities%20contributing%20to%20the%20sterility%20of%20hybrids%20between%20Mimulus%20guttatus%20and%20M.%20nasatus&amp;author=L.%20Fishman&amp;author=J.H.%20Willis&amp;publication_year=2001&amp;journal=Evolution&amp;volume=55&amp;pages=1932-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.2001.55.issue-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.2001.55.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.2001.55.issue-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11761055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yin</div>
               <div class="given-names">T.M.</div>
            </div>, <div class="name string-name"><div class="surname">DiFazio</div> <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Gunter</div> <div class="given-names">L.E.</div></div>, <div class="name string-name"><div class="surname">Riemenschneider</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Large-scale heterospecific segregation distortion in <em>Populus</em> revealed by a dense genetic map</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">109</div>, <div class="fpage">451</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20heterospecific%20segregation%20distortion%20in%20Populus%20revealed%20by%20a%20dense%20genetic%20map&amp;author=T.M.%20Yin&amp;author=S.P.%20DiFazio&amp;author=L.E.%20Gunter&amp;author=D.%20Riemenschneider&amp;author=G.A.%20Tuskan&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=451-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1653-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1653-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1653-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15168022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shirasawa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="volume">17</div>, <div class="fpage">381</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">You</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Genetic analysis of segregation distortion of SSR markers in F2 population of barley</div>, <div class="source">J. Agric. Sci.</div> , <div class="volume">3</div>, <div class="fpage">172</div>–<div class="lpage">7</div>.</div></div></div></div><div content-id="DSW001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Moyle</div>
               <div class="given-names">L.C.</div>
            </div>, <div class="name string-name"><div class="surname">Olson</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Tiffin</div> <div class="given-names">P.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Patterns of reproductive isolation in three Angiosperm genera</div>, <div class="source">Evolution</div> , <div class="volume">58</div>, <div class="fpage">1195</div>–<div class="lpage">208</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20reproductive%20isolation%20in%20three%20Angiosperm%20genera&amp;author=L.C.%20Moyle&amp;author=M.S.%20Olson&amp;author=P.%20Tiffin&amp;publication_year=2015&amp;journal=Evolution&amp;volume=58&amp;pages=1195-208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.2004.58.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.2004.58.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.2004.58.issue-6"> </span></div> </div></div></div></div></div><div content-id="DSW001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ouyang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y-G.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Hybrid sterility in plant: stories from rice</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">13</div>, <div class="fpage">186</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybrid%20sterility%20in%20plant%3A%20stories%20from%20rice&amp;author=Y.%20Ouyang&amp;author=Y-G.%20Liu&amp;author=Q.%20Zhang&amp;publication_year=2010&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=13&amp;pages=186-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2010.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2010.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2010.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20153244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sweigart</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Willis</div> <div class="given-names">J.H.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Molecular evolution and genetics of postzygotic reproductive isolation in plants</div>, <div class="source">Biol. Rep.</div> , <div class="volume">4</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.</div></div></div></div><div content-id="DSW001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Klekowski</div>
               <div class="given-names">E.J.</div>
            </div>
          </div>
          <div class="year">1988</div>, <div class="article-title">Genetic load and its causes in long-lived plants</div>, <div class="source">Trees</div> , <div class="volume">2</div>, <div class="fpage">195</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20load%20and%20its%20causes%20in%20long-lived%20plants&amp;author=E.J.%20Klekowski&amp;publication_year=1988&amp;journal=Trees&amp;volume=2&amp;pages=195-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00202374" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00202374" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00202374"> </span></div> </div></div></div></div></div><div content-id="DSW001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Williams</div>
               <div class="given-names">C.G.</div>
            </div>, <div class="name string-name"><div class="surname">Savolainen</div> <div class="given-names">O.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Inbreeding depression in conifers: implications for breeding strategy</div>, <div class="source">For. Sci.</div> , <div class="volume">42</div>, <div class="fpage">102</div>–<div class="lpage">17</div>.</div></div></div></div><div content-id="DSW001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fishman</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Willis</div> <div class="given-names">J.H.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">A novel meiotic drive locus almost completely distorts segregation in <em>Mimulus</em> (Monkeyflower) hybrids</div>, <div class="source">Genetics</div> , <div class="volume">169</div>, <div class="fpage">347</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20meiotic%20drive%20locus%20almost%20completely%20distorts%20segregation%20in%20Mimulus%20(Monkeyflower)%20hybrids&amp;author=L.%20Fishman&amp;author=J.H.%20Willis&amp;publication_year=2005&amp;journal=Genetics&amp;volume=169&amp;pages=347-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.032789" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.032789" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.032789"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Steinhoff</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1993</div>, <div class="article-title">Results of species hybridization with <em>Quercus robur</em> L. and <em>Quercus petraea</em> (Matt.) Liebl.</div>, <div class="source">Ann. Sci. For.</div> , <div class="volume">50</div>, <div class="fpage">137s</div>–<div class="lpage">43s</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Results%20of%20species%20hybridization%20with%20Quercus%20robur%20L.%20and%20Quercus%20petraea%20(Matt.)%20Liebl.&amp;author=S.%20Steinhoff&amp;publication_year=1993&amp;journal=Ann.%20Sci.%20For.&amp;volume=50&amp;pages=137s-43s" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/forest:19930713" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fforest:19930713" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fforest:19930713"> </span></div> </div></div></div></div></div><div content-id="DSW001C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ueno</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Le Provost</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Léger</div> <div class="given-names">V.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Bioinformatic analysis of ESTs collected by Sanger and pyrosequencing methods for a keustone forest tree species: oak</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">650</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20analysis%20of%20ESTs%20collected%20by%20Sanger%20and%20pyrosequencing%20methods%20for%20a%20keustone%20forest%20tree%20species%3A%20oak&amp;author=S.%20Ueno&amp;author=G.%20Le%20Provost&amp;author=V.%20L%C3%A9ger&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=650" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21092232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">van Ooijen</div>
               <div class="given-names">J.W.</div>
            </div>
          </div>, In <div class="source">Joinmap® 4, Software for the Calculation of Genetic Maps in Experimental Populations</div> . Edited by <div class="person-group"><div class="name string-name"><div class="surname">Kyazma</div> <div class="given-names">B.V</div></div></div>. <div class="publisher-loc">Wageningen, Netherlands</div>; <div class="year">2006</div>.</div></div></div></div><div content-id="DSW001C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voorrips</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">2002</div>, <div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="volume">93</div>, <div class="fpage">77</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Endelman</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Plomion</div> <div class="given-names">C.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">LPmerge: an R package for merging genetic maps by linear programming</div>, <div class="source">BioInformatics</div> , <div class="volume">30</div>, <div class="fpage">1623</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LPmerge%3A%20an%20R%20package%20for%20merging%20genetic%20maps%20by%20linear%20programming&amp;author=J.%20Endelman&amp;author=C.%20Plomion&amp;publication_year=2014&amp;journal=BioInformatics&amp;volume=30&amp;pages=1623-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24532720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ritter</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Gebhardt</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Salamini</div> <div class="given-names">F.</div></div></div>
          <div class="year">1990</div>, <div class="article-title">Estimation of recombination frequencies and construction of RFLP linkage maps in plants from crosses between heterozygous parents</div>, <div class="source">Genetics</div> , <div class="volume">125</div>, <div class="fpage">645</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20recombination%20frequencies%20and%20construction%20of%20RFLP%20linkage%20maps%20in%20plants%20from%20crosses%20between%20heterozygous%20parents&amp;author=E.%20Ritter&amp;author=C.%20Gebhardt&amp;author=F.%20Salamini&amp;publication_year=1990&amp;journal=Genetics&amp;volume=125&amp;pages=645-54" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1974227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ren</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">McGregor</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">An integrated genetic map based on four mapping populations and quantitative trait loci associated with economically important traits in watermelon (<em>Citrullus lanatus</em>)</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">14</div>, <div class="fpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%20map%20based%20on%20four%20mapping%20populations%20and%20quantitative%20trait%20loci%20associated%20with%20economically%20important%20traits%20in%20watermelon%20(Citrullus%20lanatus)&amp;author=Y.%20Ren&amp;author=C.%20McGregor&amp;author=Y.%20Zhang&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol.&amp;volume=14&amp;pages=33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24443961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alheit</div>
               <div class="given-names">K.V.</div>
            </div>, <div class="name string-name"><div class="surname">Reif</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Maurer</div> <div class="given-names">H.P.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Detection of segregation distortion loci in triticale (x <em>Triticosecale</em> Wittmack) based on a high-density DArT marker consensus genetic linkage map</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">380</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20segregation%20distortion%20loci%20in%20triticale%20(x%20Triticosecale%20Wittmack)%20based%20on%20a%20high-density%20DArT%20marker%20consensus%20genetic%20linkage%20map&amp;author=K.V.%20Alheit&amp;author=J.C.%20Reif&amp;author=H.P.%20Maurer&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=380" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-380" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-380" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-380"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21798064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bowers</div>
               <div class="given-names">J.E.</div>
            </div>, <div class="name string-name"><div class="surname">Bachlava</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Brunick</div> <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Rieseberg</div> <div class="given-names">L.H.</div></div>, <div class="name string-name"><div class="surname">Knapp</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Burke</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Development of a 10,000 locus genetic map of the sunflower genome based on multiple crosses</div>, <div class="source">G3</div> , <div class="volume">2</div>, <div class="fpage">721</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%2010%2C000%20locus%20genetic%20map%20of%20the%20sunflower%20genome%20based%20on%20multiple%20crosses&amp;author=J.E.%20Bowers&amp;author=E.%20Bachlava&amp;author=R.L.%20Brunick&amp;author=L.H.%20Rieseberg&amp;author=S.J.%20Knapp&amp;author=J.M.%20Burke&amp;publication_year=2012&amp;journal=G3&amp;volume=2&amp;pages=721-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.002659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.002659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.002659"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22870395" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plomion</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Chancerel</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Endelman</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide distribution of genetic diversity and linkage disequilibrium in a mass-selected population of maritime pine</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">171</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20distribution%20of%20genetic%20diversity%20and%20linkage%20disequilibrium%20in%20a%20mass-selected%20population%20of%20maritime%20pine&amp;author=C.%20Plomion&amp;author=E.%20Chancerel&amp;author=J.%20Endelman&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=171" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-171"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24581176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Qin</div> <div class="given-names">Z.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish</div>, <div class="source">DNA Res.</div> , <div class="volume">22</div>, <div class="fpage">39</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%2C%20high-resolution%20genetic%20map%20and%20its%20integration%20with%20BAC-based%20physical%20map%20in%20channel%20catfish&amp;author=Y.%20Li&amp;author=S.%20Liu&amp;author=Z.%20Qin&amp;publication_year=2015&amp;journal=DNA%20Res.&amp;volume=22&amp;pages=39-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Muñoz-Amatriaín</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Moscou</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">An improved consensus linkage map of Barley based on flow-sorted chromosomes and Single Nucleotide Polymorphism markers</div>, <div class="source">Plant Genome</div> , <div class="volume">4</div>, <div class="fpage">238</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20consensus%20linkage%20map%20of%20Barley%20based%20on%20flow-sorted%20chromosomes%20and%20Single%20Nucleotide%20Polymorphism%20markers&amp;author=M.%20Mu%C3%B1oz-Amatria%C3%ADn&amp;author=M.J.%20Moscou&amp;author=P.R.%20Bhat&amp;publication_year=2011&amp;journal=Plant%20Genome&amp;volume=4&amp;pages=238-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2011.08.0023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2011.08.0023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2011.08.0023"> </span></div> </div></div></div></div></div><div content-id="DSW001C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Muñoz-Amatriaian</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Cuesta-Marcos</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Endelman</div> <div class="given-names">J.B.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">The USDA barley core collection: genetic diversity, population structure, and potential for genome-wide association studies</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">4</div>.</div></div></div></div><div content-id="DSW001C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">de Miguel</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Bartholomé</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ehrenmann</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Evidence of intense chromosomal shuffling during conifer evolution</div>, <div class="source">Genome Biol. Evol.</div> , <div class="pub-id-doi"><a href="https://doi.org/10.1093/gbe/evv185" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/gbe/evv185</a></div>.</div></div></div></div><div content-id="DSW001C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">L.X.</div>
            </div>, <div class="name string-name"><div class="surname">Barbier</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Rouse</div> <div class="given-names">M.N.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">A consensus map for Ug99 stem rust resistance loci in wheat</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">127</div>, <div class="fpage">1561</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20map%20for%20Ug99%20stem%20rust%20resistance%20loci%20in%20wheat&amp;author=L.X.%20Yu&amp;author=H.%20Barbier&amp;author=M.N.%20Rouse&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=127&amp;pages=1561-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-014-2326-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-014-2326-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-014-2326-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24903979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lesur</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Le Provost</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bento</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">The oak gene expression atlas: insights into Fagaceae genome evolution and the discovery of genes regulated during bud dormancy release</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">112</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20oak%20gene%20expression%20atlas%3A%20insights%20into%20Fagaceae%20genome%20evolution%20and%20the%20discovery%20of%20genes%20regulated%20during%20bud%20dormancy%20release&amp;author=I.%20Lesur&amp;author=G.%20Le%20Provost&amp;author=P.%20Bento&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1331-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1331-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1331-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25765701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plomion</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Anselem</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Decoding the oak genome: public release of sequence data, assembly, annotation and publication strategies</div>, <div class="source">Mol. Ecol. Res.</div> , <div class="pub-id-doi"><a href="https://doi.org/10.1111/1755-0998.12425" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/1755-0998.12425</a></div>.</div></div></div></div><div content-id="DSW001C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gaur</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Azam</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jeena</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">High-throughput SNP discover y and genotyping for constructing a saturated link age map of Chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">357</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discover%20y%20and%20genotyping%20for%20constructing%20a%20saturated%20link%20age%20map%20of%20Chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Gaur&amp;author=S.%20Azam&amp;author=G.%20Jeena&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=357-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ward</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Bhangoo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fernández-Fernández</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Saturated linkage map construction in <em>Rubus idaeus</em> using genotyping by sequencing and genome-independent imputation</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saturated%20linkage%20map%20construction%20in%20Rubus%20idaeus%20using%20genotyping%20by%20sequencing%20and%20genome-independent%20imputation&amp;author=J.%20Ward&amp;author=J.%20Bhangoo&amp;author=F.%20Fern%C3%A1ndez-Fern%C3%A1ndez&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23324311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Petroli</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Sansaloni</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Carling</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genomic characterization of DArT markers based on high-density linkage analysis and physical mapping to the Eucalyptus genome</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20characterization%20of%20DArT%20markers%20based%20on%20high-density%20linkage%20analysis%20and%20physical%20mapping%20to%20the%20Eucalyptus%20genome&amp;author=C.%20Petroli&amp;author=C.P.%20Sansaloni&amp;author=J.%20Carling&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0044684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0044684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0044684"> </span></div> </div></div></div></div></div><div content-id="DSW001C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kane</div>
               <div class="given-names">N.C.</div>
            </div>, <div class="name string-name"><div class="surname">King</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Barker</div> <div class="given-names">M.S.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Comparative genomic and population genetic analysis indicate highly porous genomes and high levels of gene flow between divergent <em>Helianthus</em> species</div>, <div class="source">Evolution</div> , <div class="volume">63</div>, <div class="fpage">2061</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20and%20population%20genetic%20analysis%20indicate%20highly%20porous%20genomes%20and%20high%20levels%20of%20gene%20flow%20between%20divergent%20Helianthus%20species&amp;author=N.C.%20Kane&amp;author=M.G.%20King&amp;author=M.S.%20Barker&amp;publication_year=2009&amp;journal=Evolution&amp;volume=63&amp;pages=2061-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.2009.63.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.2009.63.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.2009.63.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19473382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jung</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Cestaro</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Whole genome comparisons of <em>Fragaria</em>, <em>Prunus</em> and <em>Malus</em> reveal different modes of evolution between Rosaceous subfamilies</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">129</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20comparisons%20of%20Fragaria%2C%20Prunus%20and%20Malus%20reveal%20different%20modes%20of%20evolution%20between%20Rosaceous%20subfamilies&amp;author=S.%20Jung&amp;author=A.%20Cestaro&amp;author=M.%20Troggio&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22475018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chancerel</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Lamy</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Lesur</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">High-density linkage mapping in a pine tree reveals a genomic region associated with inbreeding depression and provides clues to the extent and distribution of meiotic recombination</div>, <div class="source">BMC Biol.</div> , <div class="volume">11</div>, <div class="fpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20linkage%20mapping%20in%20a%20pine%20tree%20reveals%20a%20genomic%20region%20associated%20with%20inbreeding%20depression%20and%20provides%20clues%20to%20the%20extent%20and%20distribution%20of%20meiotic%20recombination&amp;author=E.%20Chancerel&amp;author=J.B.%20Lamy&amp;author=I.%20Lesur&amp;publication_year=2013&amp;journal=BMC%20Biol.&amp;volume=11&amp;pages=50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1741-7007-11-50" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1741-7007-11-50" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1741-7007-11-50"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23597128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Isobe</div>
               <div class="given-names">S.N.</div>
            </div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Construction of an integrated high density Simple Sequence Repeat linkage map in cultivated strawberry <em>(Fragaria 3 ananassa</em>) and its applicability</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">79</div>–<div class="lpage">92</div>, <div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dss035" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dss035</a></div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20integrated%20high%20density%20Simple%20Sequence%20Repeat%20linkage%20map%20in%20cultivated%20strawberry%20(Fragaria%203%20ananassa)%20and%20its%20applicability&amp;author=S.N.%20Isobe&amp;author=H.%20Hirakawa&amp;author=S.%20Sato&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=79-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23248204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bartholomé</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Mandrou</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Mabiala</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">High-resolution genetic linkage maps of Eucalyptus improve <em>Eucalyptus grandis</em> genome assembly</div>, <div class="source">New Phytol.</div> , <div class="volume">206</div>, <div class="fpage">1283</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20genetic%20linkage%20maps%20of%20Eucalyptus%20improve%20Eucalyptus%20grandis%20genome%20assembly&amp;author=J.%20Bartholom%C3%A9&amp;author=E.%20Mandrou&amp;author=A.%20Mabiala&amp;publication_year=2014&amp;journal=New%20Phytol.&amp;volume=206&amp;pages=1283-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25385325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martínez-García</div>
               <div class="given-names">P.J.</div>
            </div>, <div class="name string-name"><div class="surname">Parfitt</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Ogundiwin</div> <div class="given-names">E.A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">High density SNP mapping and QTL analysis for fruit quality characteristics in peach (<em>Prunus persica</em> L.)</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">9</div>, <div class="fpage">19</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20density%20SNP%20mapping%20and%20QTL%20analysis%20for%20fruit%20quality%20characteristics%20in%20peach%20(Prunus%20persica%20L.)&amp;author=P.J.%20Mart%C3%ADnez-Garc%C3%ADa&amp;author=D.E.%20Parfitt&amp;author=E.A.%20Ogundiwin&amp;publication_year=2013&amp;journal=Tree%20Genet.%20Genomes&amp;volume=9&amp;pages=19-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-012-0522-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-012-0522-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-012-0522-7"> </span></div> </div></div></div></div></div><div content-id="DSW001C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">H.H.</div>
            </div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">W.B.</div></div>, <div class="name string-name"><div class="surname">Wand</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Gains in QTL detection using an ultra-high density SNPmap based on population sequencing relative to traditional RFLP/SSR markers</div>, <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e17595</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gains%20in%20QTL%20detection%20using%20an%20ultra-high%20density%20SNPmap%20based%20on%20population%20sequencing%20relative%20to%20traditional%20RFLP%2FSSR%20markers&amp;author=H.H.%20Yu&amp;author=W.B.%20Xie&amp;author=J.%20Wand&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e17595" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0017595" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0017595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0017595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21390234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fishman</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Stathos</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Beardsley</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">C.F.</div></div>, <div class="name string-name"><div class="surname">Hill</div> <div class="given-names">J.P.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Chromosomal rearrangements and the genetics of reproductive barriers in <em>Mimulus</em> (Monkeyflowers)</div>, <div class="source">Evolution</div> , <div class="volume">6</div>, <div class="fpage">2547</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20rearrangements%20and%20the%20genetics%20of%20reproductive%20barriers%20in%20Mimulus%20(Monkeyflowers)&amp;author=L.%20Fishman&amp;author=A.%20Stathos&amp;author=P.M.%20Beardsley&amp;author=C.F.%20Williams&amp;author=J.P.%20Hill&amp;publication_year=2013&amp;journal=Evolution&amp;volume=6&amp;pages=2547-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.2013.67.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.2013.67.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.2013.67.issue-9"> </span></div> </div></div></div></div></div><div content-id="DSW001C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Casasoli</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Derory</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Morera-Dutrey</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Comparison of quantitative trait loci for adaptive traits between oak and chestnut based on an expressed sequence tag consensus map</div>, <div class="source">Genetics</div> , <div class="volume">172</div>, <div class="fpage">533</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20quantitative%20trait%20loci%20for%20adaptive%20traits%20between%20oak%20and%20chestnut%20based%20on%20an%20expressed%20sequence%20tag%20consensus%20map&amp;author=M.%20Casasoli&amp;author=J.%20Derory&amp;author=C.%20Morera-Dutrey&amp;publication_year=2006&amp;journal=Genetics&amp;volume=172&amp;pages=533-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.048439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.048439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.048439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16204213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Murat</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Salse</div> <div class="given-names">J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Decoding plant and animal genome plasticity from differential paleo-evolutionary patterns and processes</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">4</div>, <div class="fpage">917</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Decoding%20plant%20and%20animal%20genome%20plasticity%20from%20differential%20paleo-evolutionary%20patterns%20and%20processes&amp;author=F.%20Murat&amp;author=Y.%20Van%20de%20Peer&amp;author=J.%20Salse&amp;publication_year=2012&amp;journal=Genome%20Biol.%20Evol.&amp;volume=4&amp;pages=917-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22833223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kremer</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Petit</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">1993</div>, <div class="article-title">Gene diversity in natural populations of oak species</div>, <div class="source">Ann. Sci. For.</div> , <div class="volume">50</div><div class="issue">(Suppl. 1)</div>, <div class="fpage">186s</div>–<div class="lpage">202s</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20diversity%20in%20natural%20populations%20of%20oak%20species&amp;author=A.%20Kremer&amp;author=R.J.%20Petit&amp;publication_year=1993&amp;journal=Ann.%20Sci.%20For.&amp;volume=50&amp;pages=186s-202s" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/forest:19930717" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fforest:19930717" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fforest:19930717"> </span></div> </div></div></div></div></div><div content-id="DSW001C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gerber</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Chadoeuf</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gugerli</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">High rates of gene flow by pollen and seed in oak populations across Europe</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e85130</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20rates%20of%20gene%20flow%20by%20pollen%20and%20seed%20in%20oak%20populations%20across%20Europe&amp;author=S.%20Gerber&amp;author=J.%20Chadoeuf&amp;author=F.%20Gugerli&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e85130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0085130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0085130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0085130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24454802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Launey</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Hedgecok</div> <div class="given-names">D.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">High genetic load in the pacific oyster <em>Crassostrea gigas</em></div>, <div class="source">Genetics</div> , <div class="volume">159</div>, <div class="fpage">255</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20genetic%20load%20in%20the%20pacific%20oyster%20Crassostrea%20gigas&amp;author=S.%20Launey&amp;author=D.%20Hedgecok&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=255-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11560902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kullan</div>
               <div class="given-names">A.R.K.</div>
            </div>, <div class="name string-name"><div class="surname">van Dyk</div> <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kanzler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bayley</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Myburg</div> <div class="given-names">A.A.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">High-density genetic linkage maps with over 2,400 sequence-anchored DArT markers for genetic dissection in an F2 pseudo-backcross of <em>Eucalyptus grandis</em> × <em>E. urophylla</em></div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">8</div>, <div class="fpage">163</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20genetic%20linkage%20maps%20with%20over%202%2C400%20sequence-anchored%20DArT%20markers%20for%20genetic%20dissection%20in%20an%20F2%20pseudo-backcross%20of%20Eucalyptus%20grandis%20%C3%97%20E.%20urophylla&amp;author=A.R.K.%20Kullan&amp;author=M.M.%20van%20Dyk&amp;author=N.%20Jones&amp;author=A.%20Kanzler&amp;author=A.%20Bayley&amp;author=A.A.%20Myburg&amp;publication_year=2012&amp;journal=Tree%20Genet.%20Genomes&amp;volume=8&amp;pages=163-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-011-0430-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-011-0430-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-011-0430-2"> </span></div> </div></div></div></div></div><div content-id="DSW001C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Abadie</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Roussel</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Dencausse</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Strength, diversity and plasticity of postmating reproductive barriers between two hybridizing oak species (<em>Quercus robur</em> L. and <em>Quercus petraea</em> (Matt) Liebl.)</div>, <div class="source">J. Evol. Biol.</div> , <div class="volume">25</div>, <div class="fpage">157</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strength%2C%20diversity%20and%20plasticity%20of%20postmating%20reproductive%20barriers%20between%20two%20hybridizing%20oak%20species%20(Quercus%20robur%20L.%20and%20Quercus%20petraea%20(Matt)%20Liebl.)&amp;author=P.%20Abadie&amp;author=G.%20Roussel&amp;author=B.%20Dencausse&amp;publication_year=2012&amp;journal=J.%20Evol.%20Biol.&amp;volume=25&amp;pages=157-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeb.2011.25.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeb.2011.25.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeb.2011.25.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22092648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW001C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lepais</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Petit</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Guichoux</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Species relative abundance and direction of introgression in oaks</div>, <div class="source">Mol. Ecol.</div> , <div class="volume">18</div>, <div class="fpage">2228</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20relative%20abundance%20and%20direction%20of%20introgression%20in%20oaks&amp;author=O.%20Lepais&amp;author=R.J.%20Petit&amp;author=E.%20Guichoux&amp;publication_year=2008&amp;journal=Mol.%20Ecol.&amp;volume=18&amp;pages=2228-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2009.18.issue-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2009.18.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2009.18.issue-10"> </span></div> </div></div></div></div></div><div content-id="DSW001C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lepais</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Gerber</div> <div class="given-names">S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Reproductive patterns shape introgression dynamics and species succession within the European white oak species complex</div>, <div class="source">Evolution</div> , <div class="volume">1</div>, <div class="fpage">156</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reproductive%20patterns%20shape%20introgression%20dynamics%20and%20species%20succession%20within%20the%20European%20white%20oak%20species%20complex&amp;author=O.%20Lepais&amp;author=S.%20Gerber&amp;publication_year=2011&amp;journal=Evolution&amp;volume=1&amp;pages=156-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1558-5646.2010.01101.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1558-5646.2010.01101.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1558-5646.2010.01101.x"> </span></div> </div></div></div></div></div><div content-id="DSW001C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brendel</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Le Thiec</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Scotti-Saintagne</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bodénès</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Guehl</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Quantitative trait loci controlling water use efficiency and related traits in <em>Quercus robur</em> L.</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">4</div>, <div class="fpage">263</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20controlling%20water%20use%20efficiency%20and%20related%20traits%20in%20Quercus%20robur%20L.&amp;author=O.%20Brendel&amp;author=D.%20Le%20Thiec&amp;author=C.%20Scotti-Saintagne&amp;author=C.%20Bod%C3%A9n%C3%A8s&amp;author=A.%20Kremer&amp;author=J.M.%20Guehl&amp;publication_year=2008&amp;journal=Tree%20Genet.%20Genomes&amp;volume=4&amp;pages=263-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-007-0107-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-007-0107-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-007-0107-z"> </span></div> </div></div></div></div></div><div content-id="DSW001C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW001C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parelle</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Zapater</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Scotti-Saintagne</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Quantitative trait loci of tolerance to waterlogging in a European oak (<em>Quercus robur</em> L.): physiological relevance and temporal effect patterns</div>, <div class="source">Plant, Cell Environment</div> , <div class="volume">30</div>, <div class="fpage">422</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20of%20tolerance%20to%20waterlogging%20in%20a%20European%20oak%20(Quercus%20robur%20L.)%3A%20physiological%20relevance%20and%20temporal%20effect%20patterns&amp;author=J.%20Parelle&amp;author=M.%20Zapater&amp;author=C.%20Scotti-Saintagne&amp;publication_year=2007&amp;journal=Plant%2C%20Cell%20Environment&amp;volume=30&amp;pages=422-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2007.30.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2007.30.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2007.30.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17324229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745337&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw001/2/dsw001_Supplementary_Data.zip?Expires=2147483647&amp;Signature=NmivpNrUO-LgVSeMRXc4bQUwZvSC5jnBdoWFQGCXBFaBGk3dN9q2jJr9gQX8i75DhN4EBlVhWXpPza189X~BfsPkFIs9bBGJndENLkuzwBPTL7Sl5Gr-Q8kBkUAtXN7A7umF6N~Des8mg4ECXksZ51WsP~Y0EUVTwX00dcXbRI5HVaVwGrF2bHoZ9Dism3irHcolFemd-sOBkSZKOhR4UuTx-D-gBnuU0k0nOmW7dsgmGmUluO98CiKTl0bJWOaGR7ywe9OziUmdtaNv8P8Uqbrh81fTZYxe7j7YUUO9lyCGi2mDMkqxatbdDkR4aMN7CisXR6iSdYtFdZtsqNMidA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,241</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">15</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745337"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f2%2f115%2f1745337">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000375088800004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (15)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F2%2F115%2F1745337" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745337'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-density linkage mapping and distribution of segregation distortion regions in the oak genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f115%2f1745337">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

