

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/87/4282306","@type":"ScholarlyArticle","name":"Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of  Botryosphaeriaceae ","datePublished":"2017-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx040","keywords":["<italic>de novo </italic>assembly","<italic>Lasiodiplodia theobromae</italic>","low-depth sequencing","RNASeq","virulence factors"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/87/4282306","author":[{"name":"Yan, Ji Ye","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Zhao, Wen Sheng","affiliation":"College of Plant Protection, China Agricultural University, Beijing, China","@type":"Person"},{"name":"Chen, Zhen","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Xing, Qi Kai","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Zhang, Wei","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China  College of Plant Protection, China Agricultural University, Beijing, China","@type":"Person"},{"name":"Chethana, K W Thilini","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China  Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand","@type":"Person"},{"name":"Xue, Min Feng","affiliation":"Institute of Plant Protection, Hubei Academy of Agricultural Sciences, Wuhan, China","@type":"Person"},{"name":"Xu, Jian Ping","affiliation":"Department of Biology, McMaster University, ON, Canada","@type":"Person"},{"name":"Phillips, Alan J L","affiliation":"University of Lisbon, Faculty of Sciences, Bio Systems and Integrative Sciences Institute (BioISI), Campo Grande, Lisbon, Portugal","@type":"Person"},{"name":"Wang, Yong","affiliation":"Department of Plant Pathology, Guizhou University, Guiyang, Guizhou, China","@type":"Person"},{"name":"Liu, Jian Hua","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Liu, Mei","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Zhou, Ying","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Jayawardena, Ruvishika S","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China  Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand","@type":"Person"},{"name":"Manawasinghe, Ishara S","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China  Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand","@type":"Person"},{"name":"Huang, Jin Bao","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Qiao, Guang Hang","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Fu, Chun Yuan","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Guo, Fei Fei","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"},{"name":"Dissanayake, Asha J","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China  Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand","@type":"Person"},{"name":"Peng, You Liang","affiliation":"College of Plant Protection, China Agricultural University, Beijing, China","@type":"Person"},{"name":"Hyde, Kevin D","affiliation":"Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand","@type":"Person"},{"name":"Li, Xing Hong","affiliation":"Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China","@type":"Person"}],"description":"Abstract.  Botryosphaeriaceae are an important fungal family that cause woody plant diseases worldwide. Recent studies have established a correlation between en","pageStart":"87","pageEnd":"102","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of  Botryosphaeriaceae ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of  Botryosphaeriaceae " />
    <meta property="og:description" content="Abstract.  Botryosphaeriaceae are an important fungal family that cause woody plant diseases worldwide. Recent studies have established a correlation between en" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/87/4282306" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-10-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yan, Ji Ye" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Zhao, Wen Sheng" /><meta name="citation_author_institution" content="College of Plant Protection, China Agricultural University, Beijing, China" /><meta name="citation_author" content="Chen, Zhen" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Xing, Qi Kai" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Zhang, Wei" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author_institution" content="College of Plant Protection, China Agricultural University, Beijing, China" /><meta name="citation_author" content="Chethana, K W Thilini" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author_institution" content="Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand" /><meta name="citation_author" content="Xue, Min Feng" /><meta name="citation_author_institution" content="Institute of Plant Protection, Hubei Academy of Agricultural Sciences, Wuhan, China" /><meta name="citation_author" content="Xu, Jian Ping" /><meta name="citation_author_institution" content="Department of Biology, McMaster University, ON, Canada" /><meta name="citation_author" content="Phillips, Alan J L" /><meta name="citation_author_institution" content="University of Lisbon, Faculty of Sciences, Bio Systems and Integrative Sciences Institute (BioISI), Campo Grande, Lisbon, Portugal" /><meta name="citation_author" content="Wang, Yong" /><meta name="citation_author_institution" content="Department of Plant Pathology, Guizhou University, Guiyang, Guizhou, China" /><meta name="citation_author" content="Liu, Jian Hua" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Liu, Mei" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Zhou, Ying" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Jayawardena, Ruvishika S" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author_institution" content="Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand" /><meta name="citation_author" content="Manawasinghe, Ishara S" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author_institution" content="Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand" /><meta name="citation_author" content="Huang, Jin Bao" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Qiao, Guang Hang" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Fu, Chun Yuan" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Guo, Fei Fei" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author" content="Dissanayake, Asha J" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_author_institution" content="Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand" /><meta name="citation_author" content="Peng, You Liang" /><meta name="citation_author_institution" content="College of Plant Protection, China Agricultural University, Beijing, China" /><meta name="citation_author" content="Hyde, Kevin D" /><meta name="citation_author_institution" content="Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand" /><meta name="citation_author" content="Li, Xing Hong" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China" /><meta name="citation_title" content="Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of Botryosphaeriaceae" /><meta name="citation_firstpage" content="87" /><meta name="citation_lastpage" content="102" /><meta name="citation_doi" content="10.1093/dnares/dsx040" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The Botryosphaeriaceae: genera and species known from culture,;  citation_journal_title=Stud. Mycol;  citation_year=2013;  citation_volume=76;  citation_pages=51-167; " /><meta name="citation_reference" content="citation_title=Species of Botryosphaeriaceae involved in grapevine dieback in China,;  citation_journal_title=Fungal Divers;  citation_year=2013;  citation_volume=61;  citation_pages=221-36; " /><meta name="citation_reference" content="citation_title=Distinctive expansion of gene families associated with plant cell wall degradation, secondary metabolism, and nutrient uptake in the genomes of grapevine trunk pathogens,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1-22; " /><meta name="citation_reference" content="citation_title=Trail decryption of molecular research on Botryosphaeriaceae in woody plants;  citation_journal_title=Phytopathol. Mediterr;  citation_year=2016;  citation_volume=55;  citation_pages=147-71; " /><meta name="citation_reference" content="citation_title=Global transcriptional analysis suggests Lasiodiplodia theobromae pathogenicity factors involved in modulation of grapevine defensive response,;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=615" /><meta name="citation_reference" content="citation_title=Effect of Neofusicoccum parvum and Diplodia seriata extra-cellular compounds on defence gene expression in Vitis vinifera cv. Chardonnay and cv. Gewurztraminer,;  citation_journal_title=Phytopathol. Mediterr;  citation_year=2012;  citation_volume=51;  citation_pages=410-52; " /><meta name="citation_reference" content="citation_title=Secretome analysis identifies potential virulence factors of Diplodia corticola, a fungal pathogen involved in cork oak (Quercus suber) decline,;  citation_journal_title=Fungal Biol;  citation_year=2014;  citation_volume=118;  citation_pages=516-23; " /><meta name="citation_reference" content="citation_title=Phytotoxic metabolites from Neofusicoccum parvum, a pathogen of Botryosphaeria dieback of grapevine,;  citation_journal_title=Phytochemistry;  citation_year=2015;  citation_volume=115;  citation_pages=207-15; " /><meta name="citation_reference" content="citation_title=Phytotoxins produced by fungi associated with grapevine trunk diseases,;  citation_journal_title=Toxins;  citation_year=2011;  citation_volume=3;  citation_pages=1569-605; " /><meta name="citation_reference" content="citation_title=Grapevine trunk diseases: complex and still poorly understood;  citation_journal_title=Plant Pathol;  citation_year=2013;  citation_volume=62;  citation_pages=243-65; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Neofusicoccum parvum isolate UCR-NP2, a fungal vascular pathogen associated with grapevine cankers,;  citation_journal_title=Genome Announc;  citation_year=2013;  citation_volume=1;  citation_pages=e00339-132013; " /><meta name="citation_reference" content="citation_title=Friend or foe? Evolutionary history of glycoside hydrolase family 32 genes encoding for sucrolytic activity in fungi and its implications for plant-fungal symbioses,;  citation_journal_title=BMC Evol. Biol;  citation_year=2009;  citation_volume=9;  citation_pages=1-16; " /><meta name="citation_reference" content="citation_title=A Phytophthora sojae glycoside hydrolase 12 protein is a major virulence factor during soybean infection and is recognized as a PAMP;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=2057-72; " /><meta name="citation_reference" content="citation_title=Draft genome sequences of Diplodia sapinea, Ceratocystis manginecans, and Ceratocystis moniliformis,;  citation_journal_title=IMA Fungus;  citation_year=2014;  citation_volume=5;  citation_pages=135-40; " /><meta name="citation_reference" content="citation_title=Genes determining pathogenicity to pea are clustered on a supernumerary chromosome in the fungal plant pathogen Nectria haematococca;  citation_journal_title=Plant J;  citation_year=2014;  citation_volume=25;  citation_pages=305-14; " /><meta name="citation_reference" content="citation_title=Insights into the oxidative degradation of cellulose by a copper metalloenzyme that exploits biomass components,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=15079-084; " /><meta name="citation_reference" content="citation_title=Microbial genome-enabled insights into plant-microorganism interactions,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=5;  citation_pages=797-813; " /><meta name="citation_reference" content="citation_title=Chemistry and molecular organization of plant cell walls;  citation_journal_title=Water Res;  citation_year=2008;  citation_volume=45;  citation_pages=1587-96; " /><meta name="citation_reference" content="citation_title=An Ustilago maydis gene involved in H2O2 detoxification is required for virulence,;  citation_journal_title=Plant Cell;  citation_year=2007;  citation_volume=19;  citation_pages=2293-309; " /><meta name="citation_reference" content="citation_title=Karyotype variation within clonal lineages of the rice blast fungus Magnaporthe grisea;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1993;  citation_volume=59;  citation_pages=585-93; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes,;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=351-8; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of repetitive elements in eukaryotic genomes,;  citation_journal_title=Mob. DNA;  citation_year=2015;  citation_volume=6;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes,;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=1269-76; " /><meta name="citation_reference" content="citation_title=AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints,;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W465-7; " /><meta name="citation_reference" content="citation_title=Gene identification in novel eukaryotic genomes by self-training algorithm,;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=6494-506; " /><meta name="citation_reference" content="citation_title=GenBlastA: enabling BLAST to identify homologous gene sequences,;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=143-9; " /><meta name="citation_reference" content="citation_title=GeneWise and genomewise,;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=988-95; " /><meta name="citation_reference" content="citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions,;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=785-6; " /><meta name="citation_reference" content="citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq,;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=621-8; " /><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using evidencemodeler and the program to assemble spliced alignments,;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7" /><meta name="citation_reference" content="citation_title=KAAS: an automatic genome annotation and pathway reconstruction server,;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W182-5; " /><meta name="citation_reference" content="citation_title=The genome sequence of the rice blast fungus Magnaporthe grisea,;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=434;  citation_pages=980-6; " /><meta name="citation_reference" content="citation_title=Tools to kill: genome of one of the most destructive plant pathogenic fungi Macrophomina phaseolina,;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1-16; " /><meta name="citation_reference" content="citation_title=The transporter classification database,;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=251-8; " /><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform,;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-66; " /><meta name="citation_reference" content="citation_title=MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=1870-74; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland;  citation_title=PAUP* 4.0: phylogenetic analysis using parsimony (* and other methods);  citation_year=2002; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=The genome sequence of the filamentous fungus Neurospora crassa,;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=422;  citation_pages=859-68; " /><meta name="citation_reference" content="citation_title=Genome expansion and gene loss in powdery mildew fungi reveal tradeoffs in extreme parasitism;  citation_journal_title=Science;  citation_year=2010;  citation_volume=330;  citation_pages=1543-6; " /><meta name="citation_reference" content="citation_title=Selection on the protein-coding genome,;  citation_journal_title=Methods Mol. Biol;  citation_year=2012;  citation_volume=856;  citation_pages=113-40; " /><meta name="citation_reference" content="citation_title=The life and death of gene families,;  citation_journal_title=Bioessays;  citation_year=2012;  citation_volume=31;  citation_pages=29-39; " /><meta name="citation_reference" content="citation_title=Altered patterns of gene duplication and differential gene gain and loss in fungal pathogens,;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=95-101; " /><meta name="citation_reference" content="citation_title=CAFE: a computational tool for the study of gene family evolution,;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1269-71; " /><meta name="citation_reference" content="citation_title=Regulation of fungal secondary metabolism,;  citation_journal_title=Nat. Rev. Microbiol;  citation_year=2013;  citation_volume=11;  citation_pages=21-32; " /><meta name="citation_reference" content="citation_title=Accurate prediction of secondary metabolite gene clusters in filamentous fungi,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=E99-107; " /><meta name="citation_reference" content="citation_title=SMURF: genomic mapping of fungal secondary metabolite clusters,;  citation_journal_title=Fungal Genet. Biol;  citation_year=2010;  citation_volume=47;  citation_pages=736-41; " /><meta name="citation_reference" content="citation_title=antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences,;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W339-46; " /><meta name="citation_reference" content="citation_title=Membrane transporters in self resistance of Cercospora nicotianae to the photo-activated toxin cercosporin,;  citation_journal_title=Curr. Genet;  citation_year=2015;  citation_volume=61;  citation_pages=601-20; " /><meta name="citation_reference" content="citation_title=Aspergillus niger mstA encodes a high-affinity sugar/H+ symporter which is regulated in response to extracellular pH,;  citation_journal_title=Biochem J;  citation_year=2004;  citation_volume=379;  citation_pages=375-83; " /><meta name="citation_reference" content="citation_title=CDR4 is the major contributor to azole resistance among four Pdr5p-like ABC transporters in Neurospora crassa,;  citation_journal_title=Fungal Biol;  citation_year=2012;  citation_volume=116;  citation_pages=848-54; " /><meta name="citation_reference" content="citation_title=The predicted secretome of the plant pathogenic fungus Fusarium graminearum: a refined comparative analysis,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e33731" /><meta name="citation_reference" content="citation_title=Locating proteins in the cell using TargetP, SignalP and related tools,;  citation_journal_title=Nat. Protoc;  citation_year=2007;  citation_volume=2;  citation_pages=953-71; " /><meta name="citation_reference" content="Sonnhammer EL, von Heijne HG, Krogh A A hidden Markov model for predicting transmembrane helices in protein sequences. In: Glasgow, TL, Major, F, Lathrop, D, Sankoff, D, Sensen, C (eds) Proceedings of the Sixth International Conference on Intelligent Systems for Molecular Biology , 1998 , p. 175–82. AAAI Press, Menlo Park, CA, USA. " /><meta name="citation_reference" content="citation_title=Lifestyle transitions in plant pathogenic Colletotrichum fungi deciphered by genome and transcriptome analyses;  citation_journal_title=Nat. Genet;  citation_year=2012;  citation_volume=44;  citation_pages=1060-5; " /><meta name="citation_reference" content="citation_title=Carbohydrate-binding modules: fine tuning polysaccharide recognition,;  citation_journal_title=Biochem. J;  citation_year=2004;  citation_volume=382;  citation_pages=769-81; " /><meta name="citation_reference" content="citation_title=Expansion of the enzymatic repertoire of the CAZy database to integrate auxiliary redox enzymes,;  citation_journal_title=Biotechnol. Biofuels;  citation_year=2013;  citation_volume=6;  citation_pages=41" /><meta name="citation_reference" content="citation_title=Candidate effector proteins of the necrotrophic apple canker pathogen Valsa mali can suppress BAX-induced PCD,;  citation_journal_title=Front Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=579" /><meta name="citation_reference" content="citation_title=Genome sequence of Valsa canker pathogens uncovers a potential adaptation of colonization of woody bark,;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=208;  citation_pages=1202-16; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the genomes of two field isolates of the rice blast fungus Magnaporthe oryzae,;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=1002869" /><meta name="citation_reference" content="citation_title=Exploring genome characteristics and sequence quality without a reference,;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1228-35; " /><meta name="citation_reference" content="citation_title=Extensive error in the number of genes inferred from draft genome assemblies,;  citation_journal_title=PLoS Comput. Biol;  citation_year=2014;  citation_volume=10;  citation_pages=e1003998" /><meta name="citation_reference" content="citation_title=Specific adaptation of Ustilaginoidea virens in occupying host florets revealed by comparative and functional genomics,;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=3849" /><meta name="citation_reference" content="citation_title=Genetic and pathogenic diversity of Neofusicoccum parvum in new Zealand vineyards,;  citation_journal_title=Fungal Biol;  citation_year=2012;  citation_volume=116;  citation_pages=276-88; " /><meta name="citation_reference" content="citation_title=dbPAF: an integrative database of protein phosphorylation in animals and fungi,;  citation_journal_title=Sci. Rep.;  citation_year=2016;  citation_volume=6;  citation_pages=e23534" /><meta name="citation_reference" content="citation_title=The role of mitogen activated protein (MAP) kinase signalling components and the Ste12 transcription factor in germination and pathogenicity of Botrytis cinerea,;  citation_journal_title=Mol. Plant Pathol.;  citation_year=2010;  citation_volume=11;  citation_pages=105-19; " /><meta name="citation_reference" content="citation_title=MAP kinase signalling pathway components and targets conserved between the distantly related plant pathogenic fungi Mycosphaerella graminicola and Magnaporthe grisea,;  citation_journal_title=Fungal Genet. Biol.;  citation_year=2009;  citation_volume=46;  citation_pages=667-81; " /><meta name="citation_reference" content="citation_title=Virulence regulation of phytopathogenic fungi by pH,;  citation_journal_title=Antioxid. Redox Signal;  citation_year=2013;  citation_volume=19;  citation_pages=1012-25; " /><meta name="citation_reference" content="citation_title=Functional analysis of the cytochrome P450 monooxygenase gene bcbot1 of Botrytis cinerea indicates that botrydial is a strain-specific virulence factor,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=2005;  citation_volume=18;  citation_pages=137-49; " /><meta name="citation_reference" content="citation_title=Genomic sequence of the pathogenic and allergenic filamentous fungus Aspergillus fumigatus;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=438;  citation_pages=1092-3; " /><meta name="citation_reference" content="citation_title=Inhibitory effects of polypeptides secreted by the grapevine pathogens Phaeomoniella chlamydospora and Phaeoacremonium aleophilum on plant cell activities,;  citation_journal_title=Physiol. Mol. Plant Pathol.;  citation_year=2010;  citation_volume=74;  citation_pages=403-11; " /><meta name="citation_reference" content="citation_title=pH regulation of electrogenic sugar/H+ symport in MFS sugar permeases,;  citation_journal_title=PLoS ONE;  citation_year=2016;  citation_volume=1111;  citation_pages=e0156392" /><meta name="citation_reference" content="citation_title=MFS transportome of the human pathogenic yeast Candida albicans,;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=99;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Analysis of two in planta expressed LysM effector homologs from the fungus Mycosphaerella graminicola reveals novel functional properties and varying contributions to virulence on wheat,;  citation_journal_title=Plant Physiol;  citation_year=2011;  citation_volume=156;  citation_pages=756-69; " /><meta name="citation_reference" content="citation_title=In planta effector competition assays detect Hyaloperonospora arabidopsidis effectors that contribute to virulence and localize to different plant subcellular compartments,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=2013;  citation_volume=26;  citation_pages=745-57; " /><meta name="citation_reference" content="citation_title=Deployment of the Burkholderia glumae type III secretion system as an efficient tool for translocating pathogen effectors to monocot cells,;  citation_journal_title=Plant J;  citation_year=2013;  citation_volume=74;  citation_pages=701-12; " /><meta name="citation_reference" content="citation_title=Comparative genome structure, secondary metabolite, and effector coding capacity across Cochliobolus pathogens,;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003233" /><meta name="citation_reference" content="citation_title=Comparative genomics of the white-rot fungi, Phanerochaete carnosa and P. chrysosporium, to elucidate the genetic basis of the distinct wood types they colonize,;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=579" /><meta name="citation_reference" content="citation_title=The paleozoic origin of enzymatic lignin decomposition reconstructed from 31 fungal genomes,;  citation_journal_title=Science;  citation_year=2012;  citation_volume=336;  citation_pages=1715-19; " /><meta name="citation_reference" content="citation_title=The endopolygalacturonase gene Bcpg1 is required for full virulence of Botrytis cinerea,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=1998;  citation_volume=11;  citation_pages=1009-16; " /><meta name="citation_reference" content="citation_title=Comparative analysis of fungal genomes reveals different plant cell wall degrading capacity in fungi,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=54-64; " /><meta name="citation_reference" content="citation_title=Extensive sampling of basidiomycete genomes demonstrates inadequacy of the white-rot/brown-rot paradigm for wood decay fungi,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=9923-8; " /><meta name="citation_reference" content="citation_title=Characterization of the carbohydrate binding module 18 gene family in the amphibian pathogen Batrachochytrium dendrobatidis,;  citation_journal_title=Fungal Genet. Biol;  citation_year=2015;  citation_volume=77;  citation_pages=31-9; " /><meta name="citation_reference" content="citation_title=Bacteria establish an aqueous living space in plants crucial for virulence;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=539;  citation_pages=524-530; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/87/24151625/dsx040.pdf" /><meta name="description" content="Abstract.  Botryosphaeriaceae are an important fungal family that cause woody plant diseases worldwide. Recent studies have established a correlation between en" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/87/4282306" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/87/4282306" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/87/4282306");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx040';
    </script>
<title>
	Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of Botryosphaeriaceae | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4282306" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfPwATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuLCBKaSBZZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgRW52aXJvbm1lbnQgRnJpZW5kbHkgTWFuYWdlbWVudCBvbiBGcnVpdCBEaXNlYXNlcyBhbmQgUGVzdHMgaW4gTm9ydGggQ2hpbmEsIEluc3RpdHV0ZSBvZiBQbGFudCBhbmQgRW52aXJvbm1lbnQgUHJvdGVjdGlvbiwgQmVpamluZyBBY2FkZW15IG9mIEFncmljdWx0dXJlIGFuZCBGb3Jlc3RyeSBTY2llbmNlcywgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFvLCBXZW4gU2hlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb2xsZWdlIG9mIFBsYW50IFByb3RlY3Rpb24sIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIFpoZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWGluZywgUWkgS2FpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBFbnZpcm9ubWVudCBGcmllbmRseSBNYW5hZ2VtZW50IG9uIEZydWl0IERpc2Vhc2VzIGFuZCBQZXN0cyBpbiBOb3J0aCBDaGluYSwgSW5zdGl0dXRlIG9mIFBsYW50IGFuZCBFbnZpcm9ubWVudCBQcm90ZWN0aW9uLCBCZWlqaW5nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBXZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBQbGFudCBQcm90ZWN0aW9uLCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGV0aGFuYSwgSyBXIFRoaWxpbmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEV4Y2VsbGVuY2UgaW4gRnVuZ2FsIFJlc2VhcmNoLCBNYWUgRmFoIEx1YW5nIFVuaXZlcnNpdHksIENoaWFuZyBSYWksIFRoYWlsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWHVlLCBNaW4gRmVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBQbGFudCBQcm90ZWN0aW9uLCBIdWJlaSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgV3VoYW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWHUsIEppYW4gUGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgTWNNYXN0ZXIgVW5pdmVyc2l0eSwgT04sIENhbmFkYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBoaWxsaXBzLCBBbGFuIEogTCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgTGlzYm9uLCBGYWN1bHR5IG9mIFNjaWVuY2VzLCBCaW8gU3lzdGVtcyBhbmQgSW50ZWdyYXRpdmUgU2NpZW5jZXMgSW5zdGl0dXRlIChCaW9JU0kpLCBDYW1wbyBHcmFuZGUsIExpc2JvbiwgUG9ydHVnYWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBZb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBQYXRob2xvZ3ksIEd1aXpob3UgVW5pdmVyc2l0eSwgR3VpeWFuZywgR3VpemhvdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaXUsIEppYW4gSHVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBFbnZpcm9ubWVudCBGcmllbmRseSBNYW5hZ2VtZW50IG9uIEZydWl0IERpc2Vhc2VzIGFuZCBQZXN0cyBpbiBOb3J0aCBDaGluYSwgSW5zdGl0dXRlIG9mIFBsYW50IGFuZCBFbnZpcm9ubWVudCBQcm90ZWN0aW9uLCBCZWlqaW5nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgTWVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBFbnZpcm9ubWVudCBGcmllbmRseSBNYW5hZ2VtZW50IG9uIEZydWl0IERpc2Vhc2VzIGFuZCBQZXN0cyBpbiBOb3J0aCBDaGluYSwgSW5zdGl0dXRlIG9mIFBsYW50IGFuZCBFbnZpcm9ubWVudCBQcm90ZWN0aW9uLCBCZWlqaW5nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilpob3UsIFlpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmF5YXdhcmRlbmEsIFJ1dmlzaGlrYSBTIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBFbnZpcm9ubWVudCBGcmllbmRseSBNYW5hZ2VtZW50IG9uIEZydWl0IERpc2Vhc2VzIGFuZCBQZXN0cyBpbiBOb3J0aCBDaGluYSwgSW5zdGl0dXRlIG9mIFBsYW50IGFuZCBFbnZpcm9ubWVudCBQcm90ZWN0aW9uLCBCZWlqaW5nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFeGNlbGxlbmNlIGluIEZ1bmdhbCBSZXNlYXJjaCwgTWFlIEZhaCBMdWFuZyBVbml2ZXJzaXR5LCBDaGlhbmcgUmFpLCBUaGFpbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hbmF3YXNpbmdoZSwgSXNoYXJhIFMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEV4Y2VsbGVuY2UgaW4gRnVuZ2FsIFJlc2VhcmNoLCBNYWUgRmFoIEx1YW5nIFVuaXZlcnNpdHksIENoaWFuZyBSYWksIFRoYWlsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVhbmcsIEppbiBCYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUWlhbywgR3VhbmcgSGFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgRW52aXJvbm1lbnQgRnJpZW5kbHkgTWFuYWdlbWVudCBvbiBGcnVpdCBEaXNlYXNlcyBhbmQgUGVzdHMgaW4gTm9ydGggQ2hpbmEsIEluc3RpdHV0ZSBvZiBQbGFudCBhbmQgRW52aXJvbm1lbnQgUHJvdGVjdGlvbiwgQmVpamluZyBBY2FkZW15IG9mIEFncmljdWx0dXJlIGFuZCBGb3Jlc3RyeSBTY2llbmNlcywgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdSwgQ2h1biBZdWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBFbnZpcm9ubWVudCBGcmllbmRseSBNYW5hZ2VtZW50IG9uIEZydWl0IERpc2Vhc2VzIGFuZCBQZXN0cyBpbiBOb3J0aCBDaGluYSwgSW5zdGl0dXRlIG9mIFBsYW50IGFuZCBFbnZpcm9ubWVudCBQcm90ZWN0aW9uLCBCZWlqaW5nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1bywgRmVpIEZlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgRW52aXJvbm1lbnQgRnJpZW5kbHkgTWFuYWdlbWVudCBvbiBGcnVpdCBEaXNlYXNlcyBhbmQgUGVzdHMgaW4gTm9ydGggQ2hpbmEsIEluc3RpdHV0ZSBvZiBQbGFudCBhbmQgRW52aXJvbm1lbnQgUHJvdGVjdGlvbiwgQmVpamluZyBBY2FkZW15IG9mIEFncmljdWx0dXJlIGFuZCBGb3Jlc3RyeSBTY2llbmNlcywgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEaXNzYW5heWFrZSwgQXNoYSBKIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBFbnZpcm9ubWVudCBGcmllbmRseSBNYW5hZ2VtZW50IG9uIEZydWl0IERpc2Vhc2VzIGFuZCBQZXN0cyBpbiBOb3J0aCBDaGluYSwgSW5zdGl0dXRlIG9mIFBsYW50IGFuZCBFbnZpcm9ubWVudCBQcm90ZWN0aW9uLCBCZWlqaW5nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFeGNlbGxlbmNlIGluIEZ1bmdhbCBSZXNlYXJjaCwgTWFlIEZhaCBMdWFuZyBVbml2ZXJzaXR5LCBDaGlhbmcgUmFpLCBUaGFpbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBlbmcsIFlvdSBMaWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvbGxlZ2Ugb2YgUGxhbnQgUHJvdGVjdGlvbiwgQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHlkZSwgS2V2aW4gRCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFeGNlbGxlbmNlIGluIEZ1bmdhbCBSZXNlYXJjaCwgTWFlIEZhaCBMdWFuZyBVbml2ZXJzaXR5LCBDaGlhbmcgUmFpLCBUaGFpbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBYaW5nIEhvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEVudmlyb25tZW50IEZyaWVuZGx5IE1hbmFnZW1lbnQgb24gRnJ1aXQgRGlzZWFzZXMgYW5kIFBlc3RzIGluIE5vcnRoIENoaW5hLCBJbnN0aXR1dGUgb2YgUGxhbnQgYW5kIEVudmlyb25tZW50IFByb3RlY3Rpb24sIEJlaWppbmcgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBhbmQgRm9yZXN0cnkgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb21wYXJhdGl2ZSBnZW5vbWUgYW5kIHRyYW5zY3JpcHRvbWUgYW5hbHlzZXMgcmV2ZWFsIGFkYXB0YXRpb25zIHRvIG9wcG9ydHVuaXN0aWMgaW5mZWN0aW9ucyBpbiB3b29keSBwbGFudCBkZWdyYWRpbmcgcGF0aG9nZW5zIG9mIEJvdHJ5b3NwaGFlcmlhY2VhZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9Ijg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwNDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCb3RyeW9zcGhhZXJpYWNlYWU6IGdlbmVyYSBhbmQgc3BlY2llcyBrbm93biBmcm9tIGN1bHR1cmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TdHVkLiBNeWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTUxLTE2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2llcyBvZiBCb3RyeW9zcGhhZXJpYWNlYWUgaW52b2x2ZWQgaW4gZ3JhcGV2aW5lIGRpZWJhY2sgaW4gQ2hpbmEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgRGl2ZXJzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MjIxLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0aW5jdGl2ZSBleHBhbnNpb24gb2YgZ2VuZSBmYW1pbGllcyBhc3NvY2lhdGVkIHdpdGggcGxhbnQgY2VsbCB3YWxsIGRlZ3JhZGF0aW9uLCBzZWNvbmRhcnkgbWV0YWJvbGlzbSwgYW5kIG51dHJpZW50IHVwdGFrZSBpbiB0aGUgZ2Vub21lcyBvZiBncmFwZXZpbmUgdHJ1bmsgcGF0aG9nZW5zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhaWwgZGVjcnlwdGlvbiBvZiBtb2xlY3VsYXIgcmVzZWFyY2ggb24gQm90cnlvc3BoYWVyaWFjZWFlIGluIHdvb2R5IHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9wYXRob2wuIE1lZGl0ZXJyOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MTQ3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgdHJhbnNjcmlwdGlvbmFsIGFuYWx5c2lzIHN1Z2dlc3RzIExhc2lvZGlwbG9kaWEgdGhlb2Jyb21hZSBwYXRob2dlbmljaXR5IGZhY3RvcnMgaW52b2x2ZWQgaW4gbW9kdWxhdGlvbiBvZiBncmFwZXZpbmUgZGVmZW5zaXZlIHJlc3BvbnNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0IG9mIE5lb2Z1c2ljb2NjdW0gcGFydnVtIGFuZCBEaXBsb2RpYSBzZXJpYXRhIGV4dHJhLWNlbGx1bGFyIGNvbXBvdW5kcyBvbiBkZWZlbmNlIGdlbmUgZXhwcmVzc2lvbiBpbiBWaXRpcyB2aW5pZmVyYSBjdi4gQ2hhcmRvbm5heSBhbmQgY3YuIEdld3VyenRyYW1pbmVyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9wYXRob2wuIE1lZGl0ZXJyOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9NDEwLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWNyZXRvbWUgYW5hbHlzaXMgaWRlbnRpZmllcyBwb3RlbnRpYWwgdmlydWxlbmNlIGZhY3RvcnMgb2YgRGlwbG9kaWEgY29ydGljb2xhLCBhIGZ1bmdhbCBwYXRob2dlbiBpbnZvbHZlZCBpbiBjb3JrIG9hayAoUXVlcmN1cyBzdWJlcikgZGVjbGluZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmdhbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTUxNi0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5dG90b3hpYyBtZXRhYm9saXRlcyBmcm9tIE5lb2Z1c2ljb2NjdW0gcGFydnVtLCBhIHBhdGhvZ2VuIG9mIEJvdHJ5b3NwaGFlcmlhIGRpZWJhY2sgb2YgZ3JhcGV2aW5lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9MjA3LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHl0b3RveGlucyBwcm9kdWNlZCBieSBmdW5naSBhc3NvY2lhdGVkIHdpdGggZ3JhcGV2aW5lIHRydW5rIGRpc2Vhc2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VG94aW5zOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xNTY5LTYwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JhcGV2aW5lIHRydW5rIGRpc2Vhc2VzOiBjb21wbGV4IGFuZCBzdGlsbCBwb29ybHkgdW5kZXJzdG9vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGF0aG9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MjQzLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgTmVvZnVzaWNvY2N1bSBwYXJ2dW0gaXNvbGF0ZSBVQ1ItTlAyLCBhIGZ1bmdhbCB2YXNjdWxhciBwYXRob2dlbiBhc3NvY2lhdGVkIHdpdGggZ3JhcGV2aW5lIGNhbmtlcnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQW5ub3VuYzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9ZTAwMzM5LTEzMjAxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJpZW5kIG9yIGZvZT8gRXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgZ2x5Y29zaWRlIGh5ZHJvbGFzZSBmYW1pbHkgMzIgZ2VuZXMgZW5jb2RpbmcgZm9yIHN1Y3JvbHl0aWMgYWN0aXZpdHkgaW4gZnVuZ2kgYW5kIGl0cyBpbXBsaWNhdGlvbnMgZm9yIHBsYW50LWZ1bmdhbCBzeW1iaW9zZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgRXZvbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBQaHl0b3BodGhvcmEgc29qYWUgZ2x5Y29zaWRlIGh5ZHJvbGFzZSAxMiBwcm90ZWluIGlzIGEgbWFqb3IgdmlydWxlbmNlIGZhY3RvciBkdXJpbmcgc295YmVhbiBpbmZlY3Rpb24gYW5kIGlzIHJlY29nbml6ZWQgYXMgYSBQQU1QOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjA1Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJhZnQgZ2Vub21lIHNlcXVlbmNlcyBvZiBEaXBsb2RpYSBzYXBpbmVhLCBDZXJhdG9jeXN0aXMgbWFuZ2luZWNhbnMsIGFuZCBDZXJhdG9jeXN0aXMgbW9uaWxpZm9ybWlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SU1BIEZ1bmd1czsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTM1LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcyBkZXRlcm1pbmluZyBwYXRob2dlbmljaXR5IHRvIHBlYSBhcmUgY2x1c3RlcmVkIG9uIGEgc3VwZXJudW1lcmFyeSBjaHJvbW9zb21lIGluIHRoZSBmdW5nYWwgcGxhbnQgcGF0aG9nZW4gTmVjdHJpYSBoYWVtYXRvY29jY2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMDUtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGludG8gdGhlIG94aWRhdGl2ZSBkZWdyYWRhdGlvbiBvZiBjZWxsdWxvc2UgYnkgYSBjb3BwZXIgbWV0YWxsb2VuenltZSB0aGF0IGV4cGxvaXRzIGJpb21hc3MgY29tcG9uZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTE1MDc5LTA4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGdlbm9tZS1lbmFibGVkIGluc2lnaHRzIGludG8gcGxhbnQtbWljcm9vcmdhbmlzbSBpbnRlcmFjdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTc5Ny04MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoZW1pc3RyeSBhbmQgbW9sZWN1bGFyIG9yZ2FuaXphdGlvbiBvZiBwbGFudCBjZWxsIHdhbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1XYXRlciBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0xNTg3LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBVc3RpbGFnbyBtYXlkaXMgZ2VuZSBpbnZvbHZlZCBpbiBIMk8yIGRldG94aWZpY2F0aW9uIGlzIHJlcXVpcmVkIGZvciB2aXJ1bGVuY2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MjI5My0zMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUthcnlvdHlwZSB2YXJpYXRpb24gd2l0aGluIGNsb25hbCBsaW5lYWdlcyBvZiB0aGUgcmljZSBibGFzdCBmdW5ndXMgTWFnbmFwb3J0aGUgZ3Jpc2VhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz01ODUtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gaWRlbnRpZmljYXRpb24gb2YgcmVwZWF0IGZhbWlsaWVzIGluIGxhcmdlIGdlbm9tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM1MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBiYXNlIFVwZGF0ZSwgYSBkYXRhYmFzZSBvZiByZXBldGl0aXZlIGVsZW1lbnRzIGluIGV1a2FyeW90aWMgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vYi4gRE5BOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBkZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBzZXF1ZW5jZSBmYW1pbGllcyBpbiBzZXF1ZW5jZWQgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjY5LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BVUdVU1RVUzogYSB3ZWIgc2VydmVyIGZvciBnZW5lIHByZWRpY3Rpb24gaW4gZXVrYXJ5b3RlcyB0aGF0IGFsbG93cyB1c2VyLWRlZmluZWQgY29uc3RyYWludHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVc0NjUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBpZGVudGlmaWNhdGlvbiBpbiBub3ZlbCBldWthcnlvdGljIGdlbm9tZXMgYnkgc2VsZi10cmFpbmluZyBhbGdvcml0aG0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTY0OTQtNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5CbGFzdEE6IGVuYWJsaW5nIEJMQVNUIHRvIGlkZW50aWZ5IGhvbW9sb2dvdXMgZ2VuZSBzZXF1ZW5jZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTQzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVXaXNlIGFuZCBnZW5vbWV3aXNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTk4OC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lnbmFsUCA0LjA6IGRpc2NyaW1pbmF0aW5nIHNpZ25hbCBwZXB0aWRlcyBmcm9tIHRyYW5zbWVtYnJhbmUgcmVnaW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Nzg1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYW5kIHF1YW50aWZ5aW5nIG1hbW1hbGlhbiB0cmFuc2NyaXB0b21lcyBieSBSTkEtU2VxLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz02MjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGV1a2FyeW90aWMgZ2VuZSBzdHJ1Y3R1cmUgYW5ub3RhdGlvbiB1c2luZyBldmlkZW5jZW1vZGVsZXIgYW5kIHRoZSBwcm9ncmFtIHRvIGFzc2VtYmxlIHNwbGljZWQgYWxpZ25tZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtBQVM6IGFuIGF1dG9tYXRpYyBnZW5vbWUgYW5ub3RhdGlvbiBhbmQgcGF0aHdheSByZWNvbnN0cnVjdGlvbiBzZXJ2ZXIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVcxODItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcmljZSBibGFzdCBmdW5ndXMgTWFnbmFwb3J0aGUgZ3Jpc2VhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNDsgIGNpdGF0aW9uX3BhZ2VzPTk4MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub29scyB0byBraWxsOiBnZW5vbWUgb2Ygb25lIG9mIHRoZSBtb3N0IGRlc3RydWN0aXZlIHBsYW50IHBhdGhvZ2VuaWMgZnVuZ2kgTWFjcm9waG9taW5hIHBoYXNlb2xpbmEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xLTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdHJhbnNwb3J0ZXIgY2xhc3NpZmljYXRpb24gZGF0YWJhc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTI1MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVDogYSBub3ZlbCBtZXRob2QgZm9yIHJhcGlkIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBiYXNlZCBvbiBmYXN0IEZvdXJpZXIgdHJhbnNmb3JtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zMDU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNzogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB2ZXJzaW9uIDcuMCBmb3IgYmlnZ2VyIGRhdGFzZXRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTg3MC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNpbmF1ZXIgQXNzb2NpYXRlcywgU3VuZGVybGFuZDsgIGNpdGF0aW9uX3RpdGxlPVBBVVAqIDQuMDogcGh5bG9nZW5ldGljIGFuYWx5c2lzIHVzaW5nIHBhcnNpbW9ueSAoKiBhbmQgb3RoZXIgbWV0aG9kcyk7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTUwgNDogcGh5bG9nZW5ldGljIGFuYWx5c2lzIGJ5IG1heGltdW0gbGlrZWxpaG9vZCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE1ODYtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFzdCwgbG9jay1mcmVlIGFwcHJvYWNoIGZvciBlZmZpY2llbnQgcGFyYWxsZWwgY291bnRpbmcgb2Ygb2NjdXJyZW5jZXMgb2Ygay1tZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NjQtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGZpbGFtZW50b3VzIGZ1bmd1cyBOZXVyb3Nwb3JhIGNyYXNzYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjI7ICBjaXRhdGlvbl9wYWdlcz04NTktNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBleHBhbnNpb24gYW5kIGdlbmUgbG9zcyBpbiBwb3dkZXJ5IG1pbGRldyBmdW5naSByZXZlYWwgdHJhZGVvZmZzIGluIGV4dHJlbWUgcGFyYXNpdGlzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzA7ICBjaXRhdGlvbl9wYWdlcz0xNTQzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBvbiB0aGUgcHJvdGVpbi1jb2RpbmcgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODU2OyAgY2l0YXRpb25fcGFnZXM9MTEzLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbGlmZSBhbmQgZGVhdGggb2YgZ2VuZSBmYW1pbGllcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcmVkIHBhdHRlcm5zIG9mIGdlbmUgZHVwbGljYXRpb24gYW5kIGRpZmZlcmVudGlhbCBnZW5lIGdhaW4gYW5kIGxvc3MgaW4gZnVuZ2FsIHBhdGhvZ2Vucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9OTUtMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DQUZFOiBhIGNvbXB1dGF0aW9uYWwgdG9vbCBmb3IgdGhlIHN0dWR5IG9mIGdlbmUgZmFtaWx5IGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTI2OS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBmdW5nYWwgc2Vjb25kYXJ5IG1ldGFib2xpc20sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjEtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VyYXRlIHByZWRpY3Rpb24gb2Ygc2Vjb25kYXJ5IG1ldGFib2xpdGUgZ2VuZSBjbHVzdGVycyBpbiBmaWxhbWVudG91cyBmdW5naSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPUU5OS0xMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNNVVJGOiBnZW5vbWljIG1hcHBpbmcgb2YgZnVuZ2FsIHNlY29uZGFyeSBtZXRhYm9saXRlIGNsdXN0ZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NzM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1hbnRpU01BU0g6IHJhcGlkIGlkZW50aWZpY2F0aW9uLCBhbm5vdGF0aW9uIGFuZCBhbmFseXNpcyBvZiBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBiaW9zeW50aGVzaXMgZ2VuZSBjbHVzdGVycyBpbiBiYWN0ZXJpYWwgYW5kIGZ1bmdhbCBnZW5vbWUgc2VxdWVuY2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMzM5LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZW1icmFuZSB0cmFuc3BvcnRlcnMgaW4gc2VsZiByZXNpc3RhbmNlIG9mIENlcmNvc3BvcmEgbmljb3RpYW5hZSB0byB0aGUgcGhvdG8tYWN0aXZhdGVkIHRveGluIGNlcmNvc3BvcmluLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz02MDEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzcGVyZ2lsbHVzIG5pZ2VyIG1zdEEgZW5jb2RlcyBhIGhpZ2gtYWZmaW5pdHkgc3VnYXIvSCsgc3ltcG9ydGVyIHdoaWNoIGlzIHJlZ3VsYXRlZCBpbiByZXNwb25zZSB0byBleHRyYWNlbGx1bGFyIHBILDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbSBKOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM3OTsgIGNpdGF0aW9uX3BhZ2VzPTM3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0RSNCBpcyB0aGUgbWFqb3IgY29udHJpYnV0b3IgdG8gYXpvbGUgcmVzaXN0YW5jZSBhbW9uZyBmb3VyIFBkcjVwLWxpa2UgQUJDIHRyYW5zcG9ydGVycyBpbiBOZXVyb3Nwb3JhIGNyYXNzYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmdhbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTExNjsgIGNpdGF0aW9uX3BhZ2VzPTg0OC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHByZWRpY3RlZCBzZWNyZXRvbWUgb2YgdGhlIHBsYW50IHBhdGhvZ2VuaWMgZnVuZ3VzIEZ1c2FyaXVtIGdyYW1pbmVhcnVtOiBhIHJlZmluZWQgY29tcGFyYXRpdmUgYW5hbHlzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTMzNzMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYXRpbmcgcHJvdGVpbnMgaW4gdGhlIGNlbGwgdXNpbmcgVGFyZ2V0UCwgU2lnbmFsUCBhbmQgcmVsYXRlZCB0b29scyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUHJvdG9jOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz05NTMtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlNvbm5oYW1tZXIgRUwsIHZvbiBIZWlqbmUgSEcsIEtyb2doIEEgQSBoaWRkZW4gTWFya292IG1vZGVsIGZvciBwcmVkaWN0aW5nIHRyYW5zbWVtYnJhbmUgaGVsaWNlcyBpbiBwcm90ZWluIHNlcXVlbmNlcy4gSW46IEdsYXNnb3csIFRMLCBNYWpvciwgRiwgTGF0aHJvcCwgRCwgU2Fua29mZiwgRCwgU2Vuc2VuLCBDIChlZHMpIFByb2NlZWRpbmdzIG9mIHRoZSBTaXh0aCBJbnRlcm5hdGlvbmFsIENvbmZlcmVuY2Ugb24gSW50ZWxsaWdlbnQgU3lzdGVtcyBmb3IgTW9sZWN1bGFyIEJpb2xvZ3kgLCAxOTk4ICwgcC4gMTc14oCTODIuIEFBQUkgUHJlc3MsIE1lbmxvIFBhcmssIENBLCBVU0EuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpZmVzdHlsZSB0cmFuc2l0aW9ucyBpbiBwbGFudCBwYXRob2dlbmljIENvbGxldG90cmljaHVtIGZ1bmdpIGRlY2lwaGVyZWQgYnkgZ2Vub21lIGFuZCB0cmFuc2NyaXB0b21lIGFuYWx5c2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTA2MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXJib2h5ZHJhdGUtYmluZGluZyBtb2R1bGVzOiBmaW5lIHR1bmluZyBwb2x5c2FjY2hhcmlkZSByZWNvZ25pdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEo7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzgyOyAgY2l0YXRpb25fcGFnZXM9NzY5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBhbnNpb24gb2YgdGhlIGVuenltYXRpYyByZXBlcnRvaXJlIG9mIHRoZSBDQVp5IGRhdGFiYXNlIHRvIGludGVncmF0ZSBhdXhpbGlhcnkgcmVkb3ggZW56eW1lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEJpb2Z1ZWxzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz00MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbmRpZGF0ZSBlZmZlY3RvciBwcm90ZWlucyBvZiB0aGUgbmVjcm90cm9waGljIGFwcGxlIGNhbmtlciBwYXRob2dlbiBWYWxzYSBtYWxpIGNhbiBzdXBwcmVzcyBCQVgtaW5kdWNlZCBQQ0QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udCBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTU3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBWYWxzYSBjYW5rZXIgcGF0aG9nZW5zIHVuY292ZXJzIGEgcG90ZW50aWFsIGFkYXB0YXRpb24gb2YgY29sb25pemF0aW9uIG9mIHdvb2R5IGJhcmssOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIwODsgIGNpdGF0aW9uX3BhZ2VzPTEyMDItMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWVzIG9mIHR3byBmaWVsZCBpc29sYXRlcyBvZiB0aGUgcmljZSBibGFzdCBmdW5ndXMgTWFnbmFwb3J0aGUgb3J5emFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTAwMjg2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyBnZW5vbWUgY2hhcmFjdGVyaXN0aWNzIGFuZCBzZXF1ZW5jZSBxdWFsaXR5IHdpdGhvdXQgYSByZWZlcmVuY2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEyMjgtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBlcnJvciBpbiB0aGUgbnVtYmVyIG9mIGdlbmVzIGluZmVycmVkIGZyb20gZHJhZnQgZ2Vub21lIGFzc2VtYmxpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzOTk4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2lmaWMgYWRhcHRhdGlvbiBvZiBVc3RpbGFnaW5vaWRlYSB2aXJlbnMgaW4gb2NjdXB5aW5nIGhvc3QgZmxvcmV0cyByZXZlYWxlZCBieSBjb21wYXJhdGl2ZSBhbmQgZnVuY3Rpb25hbCBnZW5vbWljcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zODQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmQgcGF0aG9nZW5pYyBkaXZlcnNpdHkgb2YgTmVvZnVzaWNvY2N1bSBwYXJ2dW0gaW4gbmV3IFplYWxhbmQgdmluZXlhcmRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTE2OyAgY2l0YXRpb25fcGFnZXM9Mjc2LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1kYlBBRjogYW4gaW50ZWdyYXRpdmUgZGF0YWJhc2Ugb2YgcHJvdGVpbiBwaG9zcGhvcnlsYXRpb24gaW4gYW5pbWFscyBhbmQgZnVuZ2ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMzUzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIG1pdG9nZW4gYWN0aXZhdGVkIHByb3RlaW4gKE1BUCkga2luYXNlIHNpZ25hbGxpbmcgY29tcG9uZW50cyBhbmQgdGhlIFN0ZTEyIHRyYW5zY3JpcHRpb24gZmFjdG9yIGluIGdlcm1pbmF0aW9uIGFuZCBwYXRob2dlbmljaXR5IG9mIEJvdHJ5dGlzIGNpbmVyZWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50IFBhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDUtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BUCBraW5hc2Ugc2lnbmFsbGluZyBwYXRod2F5IGNvbXBvbmVudHMgYW5kIHRhcmdldHMgY29uc2VydmVkIGJldHdlZW4gdGhlIGRpc3RhbnRseSByZWxhdGVkIHBsYW50IHBhdGhvZ2VuaWMgZnVuZ2kgTXljb3NwaGFlcmVsbGEgZ3JhbWluaWNvbGEgYW5kIE1hZ25hcG9ydGhlIGdyaXNlYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmdhbCBHZW5ldC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz02NjctODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpcnVsZW5jZSByZWd1bGF0aW9uIG9mIHBoeXRvcGF0aG9nZW5pYyBmdW5naSBieSBwSCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFudGlveGlkLiBSZWRveCBTaWduYWw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMDEyLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGFuYWx5c2lzIG9mIHRoZSBjeXRvY2hyb21lIFA0NTAgbW9ub294eWdlbmFzZSBnZW5lIGJjYm90MSBvZiBCb3RyeXRpcyBjaW5lcmVhIGluZGljYXRlcyB0aGF0IGJvdHJ5ZGlhbCBpcyBhIHN0cmFpbi1zcGVjaWZpYyB2aXJ1bGVuY2UgZmFjdG9yLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTM3LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHNlcXVlbmNlIG9mIHRoZSBwYXRob2dlbmljIGFuZCBhbGxlcmdlbmljIGZpbGFtZW50b3VzIGZ1bmd1cyBBc3BlcmdpbGx1cyBmdW1pZ2F0dXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMDkyLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluaGliaXRvcnkgZWZmZWN0cyBvZiBwb2x5cGVwdGlkZXMgc2VjcmV0ZWQgYnkgdGhlIGdyYXBldmluZSBwYXRob2dlbnMgUGhhZW9tb25pZWxsYSBjaGxhbXlkb3Nwb3JhIGFuZCBQaGFlb2FjcmVtb25pdW0gYWxlb3BoaWx1bSBvbiBwbGFudCBjZWxsIGFjdGl2aXRpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHlzaW9sLiBNb2wuIFBsYW50IFBhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXBIIHJlZ3VsYXRpb24gb2YgZWxlY3Ryb2dlbmljIHN1Z2FyL0grIHN5bXBvcnQgaW4gTUZTIHN1Z2FyIHBlcm1lYXNlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExMTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE1NjM5MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1GUyB0cmFuc3BvcnRvbWUgb2YgdGhlIGh1bWFuIHBhdGhvZ2VuaWMgeWVhc3QgQ2FuZGlkYSBhbGJpY2Fucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHR3byBpbiBwbGFudGEgZXhwcmVzc2VkIEx5c00gZWZmZWN0b3IgaG9tb2xvZ3MgZnJvbSB0aGUgZnVuZ3VzIE15Y29zcGhhZXJlbGxhIGdyYW1pbmljb2xhIHJldmVhbHMgbm92ZWwgZnVuY3Rpb25hbCBwcm9wZXJ0aWVzIGFuZCB2YXJ5aW5nIGNvbnRyaWJ1dGlvbnMgdG8gdmlydWxlbmNlIG9uIHdoZWF0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz03NTYtNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHBsYW50YSBlZmZlY3RvciBjb21wZXRpdGlvbiBhc3NheXMgZGV0ZWN0IEh5YWxvcGVyb25vc3BvcmEgYXJhYmlkb3BzaWRpcyBlZmZlY3RvcnMgdGhhdCBjb250cmlidXRlIHRvIHZpcnVsZW5jZSBhbmQgbG9jYWxpemUgdG8gZGlmZmVyZW50IHBsYW50IHN1YmNlbGx1bGFyIGNvbXBhcnRtZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTc0NS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVwbG95bWVudCBvZiB0aGUgQnVya2hvbGRlcmlhIGdsdW1hZSB0eXBlIElJSSBzZWNyZXRpb24gc3lzdGVtIGFzIGFuIGVmZmljaWVudCB0b29sIGZvciB0cmFuc2xvY2F0aW5nIHBhdGhvZ2VuIGVmZmVjdG9ycyB0byBtb25vY290IGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTcwMS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21lIHN0cnVjdHVyZSwgc2Vjb25kYXJ5IG1ldGFib2xpdGUsIGFuZCBlZmZlY3RvciBjb2RpbmcgY2FwYWNpdHkgYWNyb3NzIENvY2hsaW9ib2x1cyBwYXRob2dlbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzIzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIG9mIHRoZSB3aGl0ZS1yb3QgZnVuZ2ksIFBoYW5lcm9jaGFldGUgY2Fybm9zYSBhbmQgUC4gY2hyeXNvc3Bvcml1bSwgdG8gZWx1Y2lkYXRlIHRoZSBnZW5ldGljIGJhc2lzIG9mIHRoZSBkaXN0aW5jdCB3b29kIHR5cGVzIHRoZXkgY29sb25pemUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01NzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGFsZW96b2ljIG9yaWdpbiBvZiBlbnp5bWF0aWMgbGlnbmluIGRlY29tcG9zaXRpb24gcmVjb25zdHJ1Y3RlZCBmcm9tIDMxIGZ1bmdhbCBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzY7ICBjaXRhdGlvbl9wYWdlcz0xNzE1LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZW5kb3BvbHlnYWxhY3R1cm9uYXNlIGdlbmUgQmNwZzEgaXMgcmVxdWlyZWQgZm9yIGZ1bGwgdmlydWxlbmNlIG9mIEJvdHJ5dGlzIGNpbmVyZWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3Q7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDA5LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBmdW5nYWwgZ2Vub21lcyByZXZlYWxzIGRpZmZlcmVudCBwbGFudCBjZWxsIHdhbGwgZGVncmFkaW5nIGNhcGFjaXR5IGluIGZ1bmdpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTQtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBzYW1wbGluZyBvZiBiYXNpZGlvbXljZXRlIGdlbm9tZXMgZGVtb25zdHJhdGVzIGluYWRlcXVhY3kgb2YgdGhlIHdoaXRlLXJvdC9icm93bi1yb3QgcGFyYWRpZ20gZm9yIHdvb2QgZGVjYXkgZnVuZ2ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz05OTIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIGNhcmJvaHlkcmF0ZSBiaW5kaW5nIG1vZHVsZSAxOCBnZW5lIGZhbWlseSBpbiB0aGUgYW1waGliaWFuIHBhdGhvZ2VuIEJhdHJhY2hvY2h5dHJpdW0gZGVuZHJvYmF0aWRpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmdhbCBHZW5ldC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhIGVzdGFibGlzaCBhbiBhcXVlb3VzIGxpdmluZyBzcGFjZSBpbiBwbGFudHMgY3J1Y2lhbCBmb3IgdmlydWxlbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTM5OyAgY2l0YXRpb25fcGFnZXM9NTI0LTUzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzEvODcvMjQxNTE2MjUvZHN4MDQwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBCb3RyeW9zcGhhZXJpYWNlYWUgYXJlIGFuIGltcG9ydGFudCBmdW5nYWwgZmFtaWx5IHRoYXQgY2F1c2Ugd29vZHkgcGxhbnQgZGlzZWFzZXMgd29ybGR3aWRlLiBSZWNlbnQgc3R1ZGllcyBoYXZlIGVzdGFibGlzaGVkIGEgY29ycmVsYXRpb24gYmV0d2VlbiBlbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDI4MjMwNiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSCEcF8Ve6LkOauzwIDYosVUBu6hw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f87%2f4282306" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f87%2f4282306" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112483231">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483231">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483233">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483233">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483238">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483238">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483264">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483264">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483302">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483302">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483313">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483313">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483315">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483315">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483325">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483325">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/71/4265514">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/103/4562529">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ji Ye Yan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx040-cor1" data-open="dsx040-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 1051503334. Fax. +86 1051503899. Email: <a href="mailto:jiyeyan@vip.163.com" target="_blank">jiyeyan@vip.163.com</a> (J.Y.Y.); <a href="mailto:lixinghong1962@163.com" target="_blank">lixinghong1962@163.com</a> (X.H.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ji+Ye+Yan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yan J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yan J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ji Ye Yan</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wen Sheng Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">College of Plant Protection, China Agricultural University, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wen+Sheng+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wen Sheng Zhao</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhen Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhen+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen Z&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Zhen Chen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qi Kai Xing</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qi+Kai+Xing">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xing Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xing Q&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Qi Kai Xing</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div><div class="aff">College of Plant Protection, China Agricultural University, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wei+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wei Zhang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">K W Thilini Chethana</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div><div class="aff">Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=K+W+Thilini+Chethana">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chethana K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chethana K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">K W Thilini Chethana</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Min Feng Xue</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Plant Protection, Hubei Academy of Agricultural Sciences, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Min+Feng+Xue">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xue M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xue M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Min Feng Xue</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jian Ping Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, McMaster University, ON, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jian+Ping+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jian Ping Xu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alan J L Phillips</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">University of Lisbon, Faculty of Sciences, Bio Systems and Integrative Sciences Institute (BioISI), Campo Grande, Lisbon, Portugal</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alan+J+L+Phillips">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Phillips A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Phillips A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Alan J L Phillips</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Plant Pathology, Guizhou University, Guiyang, Guizhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yong Wang</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jian Hua Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jian+Hua+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jian Hua Liu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mei Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mei+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mei Liu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ying Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ying+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ying Zhou</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ruvishika S Jayawardena</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div><div class="aff">Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ruvishika+S+Jayawardena">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jayawardena R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jayawardena R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ruvishika S Jayawardena</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ishara S Manawasinghe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div><div class="aff">Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ishara+S+Manawasinghe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Manawasinghe I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Manawasinghe I&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ishara S Manawasinghe</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jin Bao Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jin+Bao+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jin Bao Huang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guang Hang Qiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guang+Hang+Qiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qiao G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qiao G&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Guang Hang Qiao</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chun Yuan Fu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chun+Yuan+Fu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fu C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chun Yuan Fu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fei Fei Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fei+Fei+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Fei Fei Guo</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asha J Dissanayake</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div><div class="aff">Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asha+J+Dissanayake">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dissanayake A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dissanayake A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Asha J Dissanayake</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">You Liang Peng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">College of Plant Protection, China Agricultural University, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=You+Liang+Peng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Peng Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Peng Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">You Liang Peng</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kevin D Hyde</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Excellence in Fungal Research, Mae Fah Luang University, Chiang Rai, Thailand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kevin+D+Hyde">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hyde K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hyde K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kevin D Hyde</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xing Hong Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Environment Friendly Management on Fruit Diseases and Pests in North China, Institute of Plant and Environment Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx040-cor1" data-open="dsx040-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 1051503334. Fax. +86 1051503899. Email: <a href="mailto:jiyeyan@vip.163.com" target="_blank">jiyeyan@vip.163.com</a> (J.Y.Y.); <a href="mailto:lixinghong1962@163.com" target="_blank">lixinghong1962@163.com</a> (X.H.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xing+Hong+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xing Hong Li</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx040-FM2"><div class="fn"><div class="footnote-content">Ji Ye Yan, Wen Sheng Zhao, Zhen Chen and Qi Kai Xing contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx040-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 87–102, <a href='https://doi.org/10.1093/dnares/dsx040'>https://doi.org/10.1093/dnares/dsx040</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 October 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 December 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 September 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4282306" href="/dnaresearch/article-pdf/25/1/87/24151625/dsx040.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ji Ye Yan, Wen Sheng Zhao, Zhen Chen, Qi Kai Xing, Wei Zhang, K W Thilini Chethana, Min Feng Xue, Jian Ping Xu, Alan J L Phillips, Yong Wang, Jian Hua Liu, Mei Liu, Ying Zhou, Ruvishika S Jayawardena, Ishara S Manawasinghe, Jin Bao Huang, Guang Hang Qiao, Chun Yuan Fu, Fei Fei Guo, Asha J Dissanayake, You Liang Peng, Kevin D Hyde, Xing Hong Li;  Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em>, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 87–102, <a href='https://doi.org/10.1093/dnares/dsx040'>https://doi.org/10.1093/dnares/dsx040</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4282306?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparative%20genome%20and%20transcriptome%20analyses%20reveal%20adaptations%20to%20opportunistic%20infections%20in%20woody%20plant%20degrading%20pathogens%20of%20%26lt%3Bem%26gt%3BBotryosphaeriaceae%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-10-03&amp;volumeNum=25&amp;issueNum=1&amp;author=Yan%2C%20Ji%20Ye%3B%20Zhao%2C%20Wen%20Sheng&amp;startPage=87&amp;endPage=102&amp;contentId=10.1093%2Fdnares%2Fdsx040&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f87%2f4282306" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112483231 id="112483231" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Botryosphaeriaceae</em> are an important fungal family that cause woody plant diseases worldwide. Recent studies have established a correlation between environmental factors and disease expression; however, less is known about factors that trigger these diseases. The current study reports on the 43.3 Mb <em>de novo</em> genome of <em>Lasiodiplodia theobromae</em> and five other genomes of <em>Botryosphaeriaceae</em> pathogens. Botryosphaeriaceous genomes showed an expansion of gene families associated with cell wall degradation, nutrient uptake, secondary metabolism and membrane transport, which contribute to adaptations for wood degradation. Transcriptome analysis revealed that genes involved in carbohydrate catabolism, pectin, starch and sucrose metabolism, and pentose and glucuronate interconversion pathways were induced during infection. Furthermore, genes in carbohydrate-binding modules, lysine motif domain and the glycosyl hydrolase gene families were induced by high temperature. Among these genes, overexpression of two selected putative lignocellulase genes led to increased virulence in the transformants. These results demonstrate the importance of high temperatures in opportunistic infections. This study also presents a set of <em>Botryosphaeriaceae</em>-specific effectors responsible for the identification of virulence-related pathogen-associated molecular patterns and demonstrates their active participation in suppressing hypersensitive responses. Together, these findings significantly expand our understanding of the determinants of pathogenicity or virulence in <em>Botryosphaeriaceae</em> and provide new insights for developing management strategies against them.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;de novo assembly&quot;"><em>de novo </em>assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lasiodiplodia theobromae&quot;"><em>Lasiodiplodia theobromae</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;low-depth sequencing&quot;">low-depth sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNASeq">RNASeq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;virulence factors&quot;">virulence factors</a></div>                            <h2 scrollto-destination=112483233 id="112483233" class="section-title" >1. Introduction</h2>
<p>The fungal family <em>Botryosphaeriaceae</em> (Ascomycetes) includes saprobes, endophytes and phytopathogens.<a reveal-id="dsx040-B1 dsx040-B2 dsx040-B3 dsx040-B4" data-open="dsx040-B1 dsx040-B2 dsx040-B3 dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> Members of this family are among the most widely geographically distributed opportunistic plant pathogens<a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> with an extensive host range, infecting many economically important plants worldwide.<a reveal-id="dsx040-B1 dsx040-B2 dsx040-B3 dsx040-B4 dsx040-B5" data-open="dsx040-B1 dsx040-B2 dsx040-B3 dsx040-B4 dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>1–5</sup></a> These pathogens can establish successful infections via wounds or natural openings, such as lenticels and stomata.<a reveal-id="dsx040-B2" data-open="dsx040-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Once inside the host tissues, they may remain latent, live as endophytes and then shift into pathogens when the host is stressed. Hence, taxa in this family have been described as latent pathogens or plant opportunistic fungal pathogens.<a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> These pathogens can cause serious diseases, such as dieback, cankers, shoot blights, leaf spots, fruit and seed rots, and witches’ brooms, in a broad range of woody plants.<a reveal-id="dsx040-B1" data-open="dsx040-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx040-B2" data-open="dsx040-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> This shift from an endophyte to a pathogen is related to host biochemical and epigenetic changes in response to environmental stresses.<a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p><p>Among the currently accepted 23 genera of <em>Botryosphaeriaceae</em>, <em>Botryosphaeria</em> Ces. &amp; De Not., <em>Diplodia</em> Fr., <em>Lasiodiplodia</em> Ellis &amp; Everh. and <em>Neofusicoccum</em> Crous, Slippers &amp; A.J.L. Phillips are considered the most virulent.<a reveal-id="dsx040-B1" data-open="dsx040-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx040-B2" data-open="dsx040-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> These botryosphaeriaceous pathogens attack the perennial organs of important woody plants, including apples, eucalyptus, grapes, peaches, pears and poplars, compromising the translocation of water and nutrients, and ultimately leading to death.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> To establish a successful infection, these woody pathogens overcome both preformed and inducible host defences by producing enzymes and/or toxins.<a reveal-id="dsx040-B5 dsx040-B6 dsx040-B7 dsx040-B8 dsx040-B9 dsx040-B10" data-open="dsx040-B5 dsx040-B6 dsx040-B7 dsx040-B8 dsx040-B9 dsx040-B10" class="link link-ref link-reveal xref-bibr"><sup>5–10</sup></a> During the last decade, several studies have confirmed the functions and effects of virulence factors, such as cell wall–degrading enzymes, effectors, secreted proteins and phytotoxins of these pathogens on their hosts.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3–</sup></a><a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> For example, most of these botryosphaeriaceous pathogens produce a wide array of secreted proteins that target the components in the primary and secondary cell walls of plants, such as endo-β-1,4-cellulases [glycosyl hydrolase family 5 (GH5)], β-glucosidases (GH3), xyloglucan transglucosylase/hydrolases (GH16) and β-xylosidases (GH43).<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B7" data-open="dsx040-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> These glycosyl hydrolase family proteins assist in the hydrolysis of glycosidic bonds in complex sugars and are involved in diverse functions, including the degradation of biomass, antibacterial defence strategies, pathogenesis mechanisms and a few typical cellular functions.<a reveal-id="dsx040-B12" data-open="dsx040-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx040-B13" data-open="dsx040-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Even though this knowledge can be used to identify proteins related to pathogenesis and virulence, a better understanding of their potential virulence functions is required to develop effective disease management strategies.</p><p>Although many genomic and transcriptomic studies have been conducted for different phytopathogens, only a few have focused on botryosphaeriaceous pathogens. These genomic studies, combined with transcriptome analyses, have significantly advanced our knowledge on the pathogenesis of these organisms. Recent studies on common botryosphaeriaceous woody plant pathogens have presented draft genomes for <em>Diplodia sapinea</em>,<a reveal-id="dsx040-B14" data-open="dsx040-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><em>Diplodia seriata</em><a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and <em>Neofusicoccum parvum</em>.<a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Functional annotation of sequenced genomes has provided a wide range of potential virulence factors, many of which are associated with lignocellulose degradation, host colonization, cellular transporters, nutrient uptake, secondary metabolism and toxin production. Some of the virulence genes predicted by Morales-Cruz et al.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and Paolinelli-Alfonso et al.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> have been identified as virulence factors in other fungal pathogens, such as <em>Eutypa lata</em>, <em>Phaeoacremonium aleophilum</em>, <em>Phytophthora</em> sp. and <em>Togninia minima</em>.<a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx040-B15" data-open="dsx040-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx040-B16" data-open="dsx040-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Although single-genome analysis facilitates better insights into the biology of a pathogen, comparative analysis of multiple genomes can often reveal a significantly greater amount of information on the physiology and evolution of a pathogen.<a reveal-id="dsx040-B17" data-open="dsx040-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Indeed, comparative genomic analyses of related taxa have facilitated the identification of both unique and shared genes and pathways related to virulence in fungal pathogens.</p><p>In this study, we sequenced and analysed the genomes of three botryosphaeriaceous pathogens, <em>Botryosphaeria dothidea</em>, <em>Lasiodiplodia theobromae</em> and <em>N.</em><em>parvum</em>. Of these, <em>L. theobromae</em> has been recorded as the most virulent on grapes in China. Compared with <em>L. theobromae</em>, <em>N. parvum</em> and <em>B. dothidea</em> are considered to be mildly virulent and weakly virulent, respectively, on grapes in China.<a reveal-id="dsx040-B2" data-open="dsx040-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The main objectives of this study were to (i) identify the genomic characteristics of these fungi, (ii) understand the genetic variation among the sequenced species and genera, (iii) identify genes or gene families involved in virulence, with a special focus on genes that are unique to each of these genera, as well as those shared among these genera, and (iv) understand the opportunistic infections identified in these genera. Whole-genome, transcriptome and comparative genome analyses revealed that their genomes encode a diverse range of genes related to virulence, including carbohydrate-active enzymes (CAZymes), peroxidases, cytochrome P450s, effector proteins, transcription factor genes and genes involved in secondary metabolism. Our analyses identified obvious expansion of gene families related to transporters and the synthesis of secondary metabolites in <em>Botryosphaeriaceae</em>. We also identified different infection strategies utilized by <em>L. theobromae</em> to adapt to different lifestyles. Functional characterization of several genes confirmed their roles in pathogenicity as candidate virulence factors. Our study provides a solid foundation from which to explore the molecular processes responsible for virulence in <em>L. theobromae</em> as well as in other botryosphaeriaceous taxa.</p>                            <h2 scrollto-destination=112483238 id="112483238" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=112483239 id="112483239" class="section-title" >2.1. Strains, culture conditions and DNA isolation</h3>
<p>Fungal strains were collected from <em>Vitis vinifera</em> in different provinces and used for sequencing (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The strains were deposited at the Beijing Academy of Agricultural and Forestry Sciences, Beijing, China. Cultures were grown in potato dextrose agar (PDA: 200 g potato, 20 g dextrose and 20 g agar) medium at 28 °C and stored at −80 °C. Total DNA was isolated from the collected mycelium using the Cetyltrimethylammonium bromide method.<a reveal-id="dsx040-B18" data-open="dsx040-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p>                            <h3 scrollto-destination=112483241 id="112483241" class="section-title" >2.2. Genome sequencing and assembly</h3>
<p>The genomic DNA of <em>L.</em><em>theobromae</em> CSS-01s was sequenced with the Illumina platforms at the Beijing Genomics Institute (BGI) in Shenzhen, China, and a 500-bp paired-end DNA library and a 6-kb mate-pair DNA library were constructed. Data from the Illumina libraries were first filtered by removing low-quality sequences, base-calling duplicates and adaptor contamination. A total of 3.1 Gb of paired-end reads, with an average read length of 90 bp and a sequencing coverage of approximately 90×, was achieved for <em>L</em>. <em>theobromae</em> strain CSS-01s. The genome sequence was assembled using SOAPdenovo software. First, short reads from the paired-end library (500 bp) were assembled into contigs using sequence overlap information. Then, information from the mate-pair library was used to join the contigs into scaffolds. Finally, SOAP Gapcloser version 1.12 (<a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html" target="_blank">http://soap.genomics.org.cn/soapdenovo.html</a> (10 May 2016, date last accessed)) was used to close the gaps inside the scaffolds.</p><p>Two other botryosphaeriaceous taxa (<em>B. dothidea</em> and <em>N. parvum</em>) with two isolates for each species and another <em>L. theobromae</em> isolate (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were subjected to low-depth sequencing. Genomic DNA extracted from all of the above isolates was sequenced using Solexa sequencing technology (Beijing Novo Gene Bioinformatics Technology Co., Ltd, Beijing, China). These libraries, with 500-bp inserts, were assembled using SOAPdenovo software (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p>                            <h3 scrollto-destination=112483244 id="112483244" class="section-title" >2.3. Transcriptome sequencing and analyses</h3>
<p><em>Lasiodiplodia theobromae</em> conidial suspension (1 ml, 1 × 10<sup>5</sup> conidial·ml<sup>−1</sup>) was inoculated onto the 27 wounded stems of <em>V.</em><em>vinifera</em> var. Summer Black and maintained at two temperatures (25 °C and 35 °C) for 48 h in the green house at Beijing Academy of Agricultural and Forestry Sciences, Beijing, China. Wounded stems inoculated with sterile water were maintained as controls at the same conditions. Simultaneously, <em>L. theobromae</em> cultures were maintained on PDA at same conditions. Then, infected host and fungal samples were collected from hosts at 8- and 12-h post-inoculation (hpi) at 25 °C. Simultaneously, fungal mycelia were collected from PDA at 8- and 12-h post-culturing at both temperatures. All the collected samples were immediately frozen in liquid nitrogen and then stored at −70 °C for RNA isolation. Total RNA was isolated from <em>in vitro</em> cultures and from diseased samples using Trizol according to the manufacturer’s instructions (Invitrogen, USA). Total RNA from nine stems with three biological replicates for each sample at 8 and 12 hpi were used for library construction. All procedures including mRNA purification, cDNA preparation, end repair of cDNA, adapter ligation and cDNA amplification were carried out by following Novo gene company methodologies for preparing Illumina RNASeq libraries. The paired-end reads with 150 bp were generated via Illumina HiSeq<sup>™</sup> by Novogene company. All the sequence data were filtered to remove adaptor sequences and low-quality sequences. The quality trimmed reads were mapped to <em>L. theobromae</em> genome using Botwie2, allowing up to two base mismatches per read. The reads per kb per million reads was calculated to reflect the expression level. The differentially expressed analysis was done by DESeq and DEGseq software. In addition, RNASeq data assembled by Trinity were used to confirm the completeness of the draft genome of <em>L. theobromae</em> strain CSS-01s. RNASeq reads were assembled into 24,537 contigs longer than 300 bp, and all the contigs were then searched using blast against the <em>L. theobromae</em> genome.</p>                            <h3 scrollto-destination=112483246 id="112483246" class="section-title" >2.4. Annotation of repetitive elements</h3>
<p>The Tandem Repeats Finder<a reveal-id="dsx040-B19" data-open="dsx040-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> was used to search for tandem repeats in the <em>L. theobromae</em> genome. Transposable elements were predicted by combining the results from RepeatMasker (version 4.0), RepeatProteinMasker (version 4.0) and RepeatModeler version (1.0.4).<a reveal-id="dsx040-B20" data-open="dsx040-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Homology-based discovery of transposable elements was conducted by RepeatMasker and RepeatProteinMasker against the sequences in the RepBase database version 19.10 (RM database version 20140131). RepeatModeler integrated two <em>de novo</em> repeat finding programs, RECON (version 1.08)<a reveal-id="dsx040-B21" data-open="dsx040-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and RepeatScout (version 1.0.5),<a reveal-id="dsx040-B20" data-open="dsx040-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> to identify repeat element boundaries and family relationships from the sequence data.</p>                            <h3 scrollto-destination=112483248 id="112483248" class="section-title" >2.5. Annotation of protein-coding genes</h3>
<p>Protein-coding genes in <em>L. theobromae</em> strain CSS-01s were identified by integrating <em>ab initio</em> prediction with homology-based gene prediction and transcriptome sequencing approaches. Repeat sequences were masked throughout the genome by RepeatModeler and RepeatMasker. Then, gene models were predicted from these repeat-masked genomes using three <em>ab initio</em> predictors, Augustus (version 3.0.3),<a reveal-id="dsx040-B22" data-open="dsx040-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> SNAP<a reveal-id="dsx040-B23" data-open="dsx040-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and GeneMark-ES (version 2.3f)<a reveal-id="dsx040-B24" data-open="dsx040-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a>. The accuracy of the <em>ab initio</em> annotation was improved using the proteomes of ten other fungi (<em>Alternaria brassicicola</em>, <em>Aspergillus fumigatus</em>, <em>Blumeria graminis</em>, <em>Colletotrichum graminicola</em>, <em>C</em>. <em>higginsianum</em>, <em>Fusarium graminearum</em>, <em>Neurospora crassa</em>, <em>Pyricularia oryzae</em>, <em>Saccharomyces cerevisiae</em> and <em>Schizosaccharomyces pombe</em>) and the RNASeq data. These 10 downloaded fungal proteomes were aligned to the <em>L. theobromae</em> genome by genBlastA (version 1.0.4). Proteins with more than 70% sequence identity were used as templates to predict the gene structure of <em>L. theobromae</em> with Genewise (version 2.4.1).<a reveal-id="dsx040-B25" data-open="dsx040-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Then, predicted genes with incomplete open reading frames were removed. The RNASeq data were aligned to the <em>L. theobromae</em> genome by TopHat (version 2.0.11),<a reveal-id="dsx040-B26" data-open="dsx040-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and potential transcripts in the <em>L. theobromae</em> genome were predicted using Cufflinks (version 2.2.1).<a reveal-id="dsx040-B27" data-open="dsx040-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> TransDecoder (version 2.0.1) was used to identify candidate coding regions within the predicted transcript sequences. Finally, all the outputs of the above methods were converted to Gene Finding Format (GFF) files with local PERL scripts. The resultant GFF files from each of the prediction programs were used as inputs into the EVM (version 2012-06-25) program to integrate GFF files obtained from the three gene prediction methods. Homology gene prediction was used to predict genes in five other sequenced botryosphaeriaceous taxa as listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, using the predicted genes of <em>L. theobromae</em> CSS-01s as templates.</p>                            <h3 scrollto-destination=112483250 id="112483250" class="section-title" >2.6. Functional annotation of predicted genes</h3>
<p>Basic functional annotations of all predicted proteins were searched against the NCBI non-redundant sequence database and UniProt/Swiss-Prot databases under the threshold parameter of E-value cut-off ≤1e<sup>−5</sup>. Secreted proteins were predicted using a combination of tools. Potential secreted proteins should contain a signal peptide without any transmembrane domain. The availability of a secretion signal peptide evaluated by both SignalP (version 4.1)<a reveal-id="dsx040-B28" data-open="dsx040-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and TargetP (version 1.1),<a reveal-id="dsx040-B29" data-open="dsx040-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and the unavailability of a transmembrane domain predicted by TMHMM (version 2.0)<a reveal-id="dsx040-B30" data-open="dsx040-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> were used to define potential secreted proteins. SignalP predicts the presence of signal peptide cleavage sites and their locations in the amino acid sequence. TargetP predicts the subcellular location of eukaryotic proteins based on the predicted presence of any N-terminal pre-sequences such as chloroplast transit peptides, mitochondrial targeting peptides or secretory pathway signal peptides. Among the predicted secretory proteins, secreted proteins ≤400 amino-acids in size and with ≥4 cysteine residues were defined as putative effectors.</p><p>Searches against the protein family database PFAM and Gene Ontology (GO) annotation of each genome were performed using InterProScan (<a class="link link-uri" href="http://www.ebi.ac.uk/Tools/pfa/iprscan5/" target="_blank">http://www.ebi.ac.uk/Tools/pfa/iprscan5/</a> (1 June 2016, date last accessed)) analysis. Kyoto Encyclopaedia of Genes and Genomes (KEGG) annotation was performed to assign molecular interaction networks and metabolic pathways using the online KEGG Automatic Annotation Server (KAAS) (<a class="link link-uri" href="http://www.genome.jp/kegg/kaas/" target="_blank">http://www.genome.jp/kegg/kaas/</a> (1 June 2016, date last accessed)). The transcription factors in the genome were annotated by mapping the IPRSCAN terms to the Fungal Transcription Factor database (<a class="link link-uri" href="http://ftfd.snu.ac.kr/tf.php?a=interpro" target="_blank">http://ftfd.snu.ac.kr/tf.php?a=interpro</a> (10 June 2016, date last accessed)), and transporters were annotated by searching against the TCDB database.</p><p>The family-specific HMM profiles of CAZymes were downloaded from the dbCAN database, and the hmmscan program in the HMMER 3.0 package<a reveal-id="dsx040-B31" data-open="dsx040-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> was used to search the fungal proteomes against the downloaded HMM profiles. Then, the primary results were parsed by the hmmscan-parser scripts supplied by the dbCAN database.</p>                            <h3 scrollto-destination=112483254 id="112483254" class="section-title" >2.7. Comparative genomics</h3>
<p>The protein sequences of the <em>L. theobromae</em> CSS-01s genome were compared with the protein sequences of 10 other fungi genomes, including five pathogenic taxa (<em>Blumeria graminis</em>, <em>Colletotrichum graminicola, C. higginsianum</em>, <em>F.</em><em>graminearum</em> and <em>P.</em><em>oryzae</em>), two opportunistic pathogens (<em>A.</em><em>brassicicola</em> and <em>A.</em><em>fumigatus</em>) and three non-pathogenic fungi (<em>N.</em><em>crassa</em>, <em>S.</em><em>cerevisiae</em> and <em>S.</em><em>pombe</em>), as well as five other isolates of the <em>Botryosphaeriaceae</em> family.<a reveal-id="dsx040-B32" data-open="dsx040-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsx040-B33" data-open="dsx040-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> OrthoMCL (version 2.0)<a reveal-id="dsx040-B34" data-open="dsx040-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was used to find the orthologues, co-orthologues and in-paralogue pairs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), as well as reciprocal best hits for each proteome. The tools were executed with an E-value &lt;10<sup>−5</sup> with the All-versus-All BLASTP search and the inflation parameter 2.0 in the MCL algorithm.</p>                            <h3 scrollto-destination=112483256 id="112483256" class="section-title" >2.8. Ka/Ks calculation</h3>
<p>For Ka/Ks calculation, 1:1 orthologous genes were identified among each isolate pair of each <em>Botryosphaeriaceae</em> species. Subsequently, the proteins between the two isolates were aligned in MUSCLE (version 3.8) using the default parameters and then converted to the corresponding nucleotide sequence alignment. The FASTA files containing the aligned sequences were converted into AXT files by a local PERL script, and the ratios of Ka and Ks were calculated between orthologous coding regions with Ka/Ks_Calculator v2.0 using a maximum-likelihood YN model.</p>                            <h3 scrollto-destination=112483258 id="112483258" class="section-title" >2.9. Divergence time estimate</h3>
<p>OrthoMCL was used to identify 1,029 single-copy gene families. The proteins of these single-copy gene families were aligned using MAFFT (version 7.187) with the default parameters and then converted to the corresponding nucleotide sequence alignment. A phylogenomic tree was constructed using MEGA7 based on the concatenated alignments of these 1,029 single-copy orthologous families. The phylogenomic tree consists of 15 other fungal taxa including eight pathogenic taxa (<em>Blumeria graminis</em>, <em>Botrytis cinera</em>, <em>Colletotrichum graminicola, C. higginsianum</em>, <em>D.</em><em>seriata</em>, <em>E.</em><em>lata</em>, <em>Fusarium graminearum</em> and <em>Pyricularia oryzae</em>), three opportunistic pathogens (<em>Alternaria alternata</em>, <em>Alternaria brassicicola</em> and <em>Aspergillus fumigatus</em>) and three non-pathogenic fungi (<em>Neurospora crassa, Saccharomyces cerevisiae</em> and <em>S.</em><em>pombe</em>), as well as five other isolates of the <em>Botryosphaeriaceae</em> family. Bootstrap support for internal branches was evaluated from 1,000 full heuristic searches. Divergence times between species were calculated using the MCMC tree program in the PAML.</p>                            <h3 scrollto-destination=112483260 id="112483260" class="section-title" >2.10. Functional studies on putative effector proteins</h3>
<p>To understand the functions of these putative effector genes identified from the current study, 27 randomly selected effectors, of which 18 are specific, were translocated into plant cells to identify effectors that suppress hypersensitive responses (HRs) in the host. These putative effector genes (without signal peptide regions) of <em>Botryosphaeriaceae</em> were amplified using the primer sets listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. The PCR products were digested with the corresponding restriction enzymes and subcloned into effector detector vector (pEDV).<a reveal-id="dsx040-B35" data-open="dsx040-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><em>Burkholderia glumae</em> competent cells were prepared as described, with minor modifications.<a reveal-id="dsx040-B36" data-open="dsx040-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Each pEDV construct was transformed into <em>B. glumae</em> using electroporation (Bio-Rad MicroPulser<sup>TM</sup>). Transformed strains were spread on LB agar medium containing 25 mg·l<sup>−1</sup> gentamycin. At least four glycerol stocks were prepared for each transformant and stored at −80 °C for further use. Leaves of 4-week-old <em>Nicotiana benthamiana</em> plants were infiltrated with bacterial inocula of OD<sub>600</sub>=0.1 using a 1 ml needleless syringe. A half-section of each leaf was injected with <em>B. glumae</em> containing the pEDV empty vector, and the other half-section was injected with <em>B. glumae</em> containing the effector gene construct. Plants were maintained at 25 °C. The timing of the hyposensitive responses was recorded at 2 days post-inoculation.</p>                            <h3 scrollto-destination=112483262 id="112483262" class="section-title" >2.11. Functional studies on putative lignocellulase encoding genes</h3>
<p>Maximum parsimony analysis was conducted for all the annotated glycosyl hydrolase family genes in PAUP (phylogenetic analysis using parsimony) v.4.0b10<a reveal-id="dsx040-B37" data-open="dsx040-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> using heuristic search option with tree bisection–reconnection branch swapping and 1,000 random sequence additions. The full-length coding sequences of candidate genes were isolated by PCR using the specific primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. The PCR products were digested with <em>HindIII</em> and <em>EcoRI</em> and inserted into the same sites behind the PtrpC promoter in a modified pBluescript II KS vector. The wild-type isolate, CSS-01s, was cultured at 28 °C on PDA plates. Mycelia collected from the culture were used to isolate fungal genomic DNA and protoplasts. Protoplasts were isolated and transformed using the PEG/CaCl<sub>2</sub> method.<a reveal-id="dsx040-B38" data-open="dsx040-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Selection medium was supplemented with 1,100 μg/ml neomycin (Amresco, USA) to screen for neomycin-resistant transformants. Putative overexpression mutants were identified by PCR and further confirmed by qRT-PCR. Pathogenicity tests were conducted on detached ‘Summer Black’ grape shoots as described previously.<a reveal-id="dsx040-B2" data-open="dsx040-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The inoculated shoots were maintained on an alternating dark/light cycle under moist conditions in the greenhouse for 8–10 days at 25 °C. All treatments were performed with at least five replicates, and all experiments were repeated three times. Data were analysed by Student’s <em>t</em>-test using SAS software.</p>                            <h2 scrollto-destination=112483264 id="112483264" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=112483265 id="112483265" class="section-title" >3.1. Genome sequencing, assembly and annotation</h3>
<p>The genome of the highly virulent <em>L.</em><em>theobromae</em> strain CSS-01s (as shown in <a reveal-id="dsx040-F1" data-open="dsx040-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) was assembled using sequencing data generated by Illumina technology (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Approximately 342 million Illumina reads (∼90× coverage of the genome) were used for assembly in SOAPdenovo software, resulting in a genome sequence of 43.3 Mb. The assembled genome accounts for 95.7% of the total genome size (∼45.2 Mb), as estimated by JELLYFISH (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).<a reveal-id="dsx040-B39" data-open="dsx040-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The current genome assembly of <em>L. theobromae</em> strain CSS-01s consists of 29 scaffolds (≥ 1 kb) and 60 contigs with N50 lengths of 4.7 and 1.7 Mb, respectively. The overall GC content of the <em>L. theobromae</em> genome is approximately 54.77%. Furthermore, all RNASeq reads were assembled into 24,210 contigs longer than 300 bp, and 95.8% of these aligned with the predicted gene models, confirming the completeness and accuracy of the genome assembly. Compared with the virulence of <em>L. theobromae</em>, <em>N.</em><em>parvum</em> and <em>B.</em><em>dothidea</em> were mildly virulent and weakly virulent, respectively, in China (<a reveal-id="dsx040-F1" data-open="dsx040-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). The genome assembly results for all botryosphaeriaceous isolates are shown in <a reveal-id="dsx040-T1" data-open="dsx040-T1" class="link link-reveal link-table xref-fig">Table 1</a>. <div content-id="dsx040-T1" class="table-modal"><div class="table-wrap"><div id="dsx040-T1" data-id="dsx040-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Major features of <em>L. theobromae</em>, <em>B. dothidea</em> and <em>N. parvum</em> genome assemblies from Illumina data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>L. theobromae</em><hr /> </th><th colspan="4"><em>B. dothidea</em><hr /> </th><th colspan="4"><em>N. parvum</em><hr /> </th></tr><tr><th colspan="2">CSS-01S<hr /> </th><th colspan="2">HBQJZ-02S<hr /> </th><th colspan="2">Sxg01s<hr /> </th><th colspan="2">LNXCEYGS-1<hr /> </th><th colspan="2"><em>CDZ1M1s3</em><hr /> </th><th colspan="2"><em>Gxg-01s</em><hr /> </th></tr><tr><th> </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th></tr></thead><tbody><tr><td>Total number </td><td>29 </td><td>60 </td><td>338 </td><td>1,594 </td><td>960 </td><td>2,203 </td><td>1,010 </td><td>2,190 </td><td>1,520 </td><td>2,757 </td><td>1,149 </td><td>2,534 </td></tr><tr><td>Total length (bp) </td><td>43,283,415 </td><td>43,280,627 </td><td>43,092,220 </td><td>42,975,261 </td><td>43,081,153 </td><td>43,048,860 </td><td>43,005,517 </td><td>42,846,751 </td><td>41,659,988 </td><td>41,636,448 </td><td>42,600,755 </td><td>42,560,320 </td></tr><tr><td>N50 (bp) </td><td>4,775,223 </td><td>1,738,941 </td><td>272,785 </td><td>48,918 </td><td>134,448 </td><td>67,019 </td><td>123,568 </td><td>71,603 </td><td>52,961 </td><td>39,937 </td><td>102,721 </td><td>57,603 </td></tr><tr><td>N90 (bp) </td><td>1,285,992 </td><td>409,362 </td><td>68,335 </td><td>14,359 </td><td>30,311 </td><td>15,690 </td><td>25,957 </td><td>16,444 </td><td>13,679 </td><td>10,166 </td><td>22,655 </td><td>14,005 </td></tr><tr><td>Maximum length (bp) </td><td>6,723,813 </td><td>3,030,172 </td><td>1,438,296 </td><td>232,370 </td><td>476,208 </td><td>428,425 </td><td>620,772 </td><td>400,854 </td><td>282,031 </td><td>179,773 </td><td>703,251 </td><td>276,000 </td></tr><tr><td>Minimum length (bp) </td><td>1,230 </td><td>410 </td><td>1,005 </td><td>500 </td><td>1,000 </td><td>31 </td><td>1,000 </td><td>39 </td><td>1,001 </td><td>25 </td><td>1,001 </td><td>31 </td></tr><tr><td>Sequence GC (%) </td><td>54.77 </td><td>54.77 </td><td>54.93 </td><td>54.94 </td><td>54.77 </td><td>54.77 </td><td>54.76 </td><td>54.76 </td><td>57.05 </td><td>57.05 </td><td>56.91 </td><td>56.91 </td></tr></tbody></table></div><div content-id="dsx040-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>L. theobromae</em><hr /> </th><th colspan="4"><em>B. dothidea</em><hr /> </th><th colspan="4"><em>N. parvum</em><hr /> </th></tr><tr><th colspan="2">CSS-01S<hr /> </th><th colspan="2">HBQJZ-02S<hr /> </th><th colspan="2">Sxg01s<hr /> </th><th colspan="2">LNXCEYGS-1<hr /> </th><th colspan="2"><em>CDZ1M1s3</em><hr /> </th><th colspan="2"><em>Gxg-01s</em><hr /> </th></tr><tr><th> </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th></tr></thead><tbody><tr><td>Total number </td><td>29 </td><td>60 </td><td>338 </td><td>1,594 </td><td>960 </td><td>2,203 </td><td>1,010 </td><td>2,190 </td><td>1,520 </td><td>2,757 </td><td>1,149 </td><td>2,534 </td></tr><tr><td>Total length (bp) </td><td>43,283,415 </td><td>43,280,627 </td><td>43,092,220 </td><td>42,975,261 </td><td>43,081,153 </td><td>43,048,860 </td><td>43,005,517 </td><td>42,846,751 </td><td>41,659,988 </td><td>41,636,448 </td><td>42,600,755 </td><td>42,560,320 </td></tr><tr><td>N50 (bp) </td><td>4,775,223 </td><td>1,738,941 </td><td>272,785 </td><td>48,918 </td><td>134,448 </td><td>67,019 </td><td>123,568 </td><td>71,603 </td><td>52,961 </td><td>39,937 </td><td>102,721 </td><td>57,603 </td></tr><tr><td>N90 (bp) </td><td>1,285,992 </td><td>409,362 </td><td>68,335 </td><td>14,359 </td><td>30,311 </td><td>15,690 </td><td>25,957 </td><td>16,444 </td><td>13,679 </td><td>10,166 </td><td>22,655 </td><td>14,005 </td></tr><tr><td>Maximum length (bp) </td><td>6,723,813 </td><td>3,030,172 </td><td>1,438,296 </td><td>232,370 </td><td>476,208 </td><td>428,425 </td><td>620,772 </td><td>400,854 </td><td>282,031 </td><td>179,773 </td><td>703,251 </td><td>276,000 </td></tr><tr><td>Minimum length (bp) </td><td>1,230 </td><td>410 </td><td>1,005 </td><td>500 </td><td>1,000 </td><td>31 </td><td>1,000 </td><td>39 </td><td>1,001 </td><td>25 </td><td>1,001 </td><td>31 </td></tr><tr><td>Sequence GC (%) </td><td>54.77 </td><td>54.77 </td><td>54.93 </td><td>54.94 </td><td>54.77 </td><td>54.77 </td><td>54.76 </td><td>54.76 </td><td>57.05 </td><td>57.05 </td><td>56.91 </td><td>56.91 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx040-T1" data-id="dsx040-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Major features of <em>L. theobromae</em>, <em>B. dothidea</em> and <em>N. parvum</em> genome assemblies from Illumina data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>L. theobromae</em><hr /> </th><th colspan="4"><em>B. dothidea</em><hr /> </th><th colspan="4"><em>N. parvum</em><hr /> </th></tr><tr><th colspan="2">CSS-01S<hr /> </th><th colspan="2">HBQJZ-02S<hr /> </th><th colspan="2">Sxg01s<hr /> </th><th colspan="2">LNXCEYGS-1<hr /> </th><th colspan="2"><em>CDZ1M1s3</em><hr /> </th><th colspan="2"><em>Gxg-01s</em><hr /> </th></tr><tr><th> </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th></tr></thead><tbody><tr><td>Total number </td><td>29 </td><td>60 </td><td>338 </td><td>1,594 </td><td>960 </td><td>2,203 </td><td>1,010 </td><td>2,190 </td><td>1,520 </td><td>2,757 </td><td>1,149 </td><td>2,534 </td></tr><tr><td>Total length (bp) </td><td>43,283,415 </td><td>43,280,627 </td><td>43,092,220 </td><td>42,975,261 </td><td>43,081,153 </td><td>43,048,860 </td><td>43,005,517 </td><td>42,846,751 </td><td>41,659,988 </td><td>41,636,448 </td><td>42,600,755 </td><td>42,560,320 </td></tr><tr><td>N50 (bp) </td><td>4,775,223 </td><td>1,738,941 </td><td>272,785 </td><td>48,918 </td><td>134,448 </td><td>67,019 </td><td>123,568 </td><td>71,603 </td><td>52,961 </td><td>39,937 </td><td>102,721 </td><td>57,603 </td></tr><tr><td>N90 (bp) </td><td>1,285,992 </td><td>409,362 </td><td>68,335 </td><td>14,359 </td><td>30,311 </td><td>15,690 </td><td>25,957 </td><td>16,444 </td><td>13,679 </td><td>10,166 </td><td>22,655 </td><td>14,005 </td></tr><tr><td>Maximum length (bp) </td><td>6,723,813 </td><td>3,030,172 </td><td>1,438,296 </td><td>232,370 </td><td>476,208 </td><td>428,425 </td><td>620,772 </td><td>400,854 </td><td>282,031 </td><td>179,773 </td><td>703,251 </td><td>276,000 </td></tr><tr><td>Minimum length (bp) </td><td>1,230 </td><td>410 </td><td>1,005 </td><td>500 </td><td>1,000 </td><td>31 </td><td>1,000 </td><td>39 </td><td>1,001 </td><td>25 </td><td>1,001 </td><td>31 </td></tr><tr><td>Sequence GC (%) </td><td>54.77 </td><td>54.77 </td><td>54.93 </td><td>54.94 </td><td>54.77 </td><td>54.77 </td><td>54.76 </td><td>54.76 </td><td>57.05 </td><td>57.05 </td><td>56.91 </td><td>56.91 </td></tr></tbody></table></div><div content-id="dsx040-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>L. theobromae</em><hr /> </th><th colspan="4"><em>B. dothidea</em><hr /> </th><th colspan="4"><em>N. parvum</em><hr /> </th></tr><tr><th colspan="2">CSS-01S<hr /> </th><th colspan="2">HBQJZ-02S<hr /> </th><th colspan="2">Sxg01s<hr /> </th><th colspan="2">LNXCEYGS-1<hr /> </th><th colspan="2"><em>CDZ1M1s3</em><hr /> </th><th colspan="2"><em>Gxg-01s</em><hr /> </th></tr><tr><th> </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th><th>Scaffold </th><th>Contig </th></tr></thead><tbody><tr><td>Total number </td><td>29 </td><td>60 </td><td>338 </td><td>1,594 </td><td>960 </td><td>2,203 </td><td>1,010 </td><td>2,190 </td><td>1,520 </td><td>2,757 </td><td>1,149 </td><td>2,534 </td></tr><tr><td>Total length (bp) </td><td>43,283,415 </td><td>43,280,627 </td><td>43,092,220 </td><td>42,975,261 </td><td>43,081,153 </td><td>43,048,860 </td><td>43,005,517 </td><td>42,846,751 </td><td>41,659,988 </td><td>41,636,448 </td><td>42,600,755 </td><td>42,560,320 </td></tr><tr><td>N50 (bp) </td><td>4,775,223 </td><td>1,738,941 </td><td>272,785 </td><td>48,918 </td><td>134,448 </td><td>67,019 </td><td>123,568 </td><td>71,603 </td><td>52,961 </td><td>39,937 </td><td>102,721 </td><td>57,603 </td></tr><tr><td>N90 (bp) </td><td>1,285,992 </td><td>409,362 </td><td>68,335 </td><td>14,359 </td><td>30,311 </td><td>15,690 </td><td>25,957 </td><td>16,444 </td><td>13,679 </td><td>10,166 </td><td>22,655 </td><td>14,005 </td></tr><tr><td>Maximum length (bp) </td><td>6,723,813 </td><td>3,030,172 </td><td>1,438,296 </td><td>232,370 </td><td>476,208 </td><td>428,425 </td><td>620,772 </td><td>400,854 </td><td>282,031 </td><td>179,773 </td><td>703,251 </td><td>276,000 </td></tr><tr><td>Minimum length (bp) </td><td>1,230 </td><td>410 </td><td>1,005 </td><td>500 </td><td>1,000 </td><td>31 </td><td>1,000 </td><td>39 </td><td>1,001 </td><td>25 </td><td>1,001 </td><td>31 </td></tr><tr><td>Sequence GC (%) </td><td>54.77 </td><td>54.77 </td><td>54.93 </td><td>54.94 </td><td>54.77 </td><td>54.77 </td><td>54.76 </td><td>54.76 </td><td>57.05 </td><td>57.05 </td><td>56.91 </td><td>56.91 </td></tr></tbody></table></div></div></p>                        <a id="112483267" scrollto-destination="112483267"></a>
<div id="dsx040-F1" data-id="dsx040-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f1.png?Expires=2147483647&Signature=SpH1bgqlJkp2XDjzskjq1bWguyNNhDi4j1xlJ5jZtDGzGBFFbt1WvRyjsNhkAVETnmwwoHngOJ0KkKYsrBaH6OWjVeS6T3nMapJLWznFurmKYFwgYmiiqGnf~gA9ySJJTIxtwMDLsaq6OwyHo0eS5f05xgevLZmg0kK4dLPgpRv3bjgF3q6m0bKOfK6lR~8-~z~0MKtQUqAcrPEvxNtEbNuXIxxSJx5S-xzTBkyUpMFjAxy5kQsbsiHD~2I9eNHKzzo2SNan3X0KHumMnEgLNb5ew5DIfQUr0p~83RhB55CTop-XoOFoOgJdcuvXMNqEYYHj2xh4PDArxIaRXh5CHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differences in the virulence and the phylogenomic analyses of the fungi used in the current study. (A) Differences in the virulence of isolates used for whole genome analyses on grapevine. Results indicate Lasiodiplodia theobromae as the most virulent among the three species. CK indicate the control for which normal PDA plug was used without any fungal inoculum. Figures showed the results of the pathogenicity test conducted for all the isolates used in the study. (B) Phylogenomic analyses for evolutionary relationships among different fungi. Time scale of the neighbour-joining phylogenomic tree was shown by million years ago at each node. Estimates of divergence times (millions of years) calculated from the rate of sequence similarity are indicated at each node. The red dots are the fossil calibrations from the Time Tree database." path-from-xml="dsx040f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483267/dsx040f1.tif" path-from-xml="dsx040f1.tif" target="_blank">View large</a><a section="112483267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f1.png?Expires=2147483647&Signature=N3ho4shbsusMq6xrHjwnfci5KDvqDptlFTO6PO2avjmu12PECEeNaWxaTV6KfPBZ8bKNANtR-y5schJ1pYRU~yL2huU9HQr-xTMwqNYb-0I5clLf5m1En1gpLeV-SG4iWQwq71X2jAu9TBKw4OTnvvU-gpSRPRIEcOy-BcrgfEy4xonQdobXwIZrerYJ9PTUYiVXu4xO3khx0ujl4xJSFhaQJAeO9Kc28pT6N4jUZMfbimh4epyEYwOyH7FwJeTGxrZbfbyXCUNK0VwV8GcHylh5GYiFbVzZTci660PwR90D0d9t6lSmza9p~PLOqTaC7mvGbUzoYFqjR9Q9QABYqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483267&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differences in the virulence and the phylogenomic analyses of the fungi used in the current study. (A) Differences in the virulence of isolates used for whole genome analyses on grapevine. Results indicate <em>Lasiodiplodia theobromae</em> as the most virulent among the three species. CK indicate the control for which normal PDA plug was used without any fungal inoculum. Figures showed the results of the pathogenicity test conducted for all the isolates used in the study. (B) Phylogenomic analyses for evolutionary relationships among different fungi. Time scale of the neighbour-joining phylogenomic tree was shown by million years ago at each node. Estimates of divergence times (millions of years) calculated from the rate of sequence similarity are indicated at each node. The red dots are the fossil calibrations from the Time Tree database.</p></div></div><div content-id="dsx040-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f1.png?Expires=2147483647&Signature=SpH1bgqlJkp2XDjzskjq1bWguyNNhDi4j1xlJ5jZtDGzGBFFbt1WvRyjsNhkAVETnmwwoHngOJ0KkKYsrBaH6OWjVeS6T3nMapJLWznFurmKYFwgYmiiqGnf~gA9ySJJTIxtwMDLsaq6OwyHo0eS5f05xgevLZmg0kK4dLPgpRv3bjgF3q6m0bKOfK6lR~8-~z~0MKtQUqAcrPEvxNtEbNuXIxxSJx5S-xzTBkyUpMFjAxy5kQsbsiHD~2I9eNHKzzo2SNan3X0KHumMnEgLNb5ew5DIfQUr0p~83RhB55CTop-XoOFoOgJdcuvXMNqEYYHj2xh4PDArxIaRXh5CHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differences in the virulence and the phylogenomic analyses of the fungi used in the current study. (A) Differences in the virulence of isolates used for whole genome analyses on grapevine. Results indicate Lasiodiplodia theobromae as the most virulent among the three species. CK indicate the control for which normal PDA plug was used without any fungal inoculum. Figures showed the results of the pathogenicity test conducted for all the isolates used in the study. (B) Phylogenomic analyses for evolutionary relationships among different fungi. Time scale of the neighbour-joining phylogenomic tree was shown by million years ago at each node. Estimates of divergence times (millions of years) calculated from the rate of sequence similarity are indicated at each node. The red dots are the fossil calibrations from the Time Tree database." path-from-xml="dsx040f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483267/dsx040f1.tif" path-from-xml="dsx040f1.tif" target="_blank">View large</a><a section="112483267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f1.png?Expires=2147483647&Signature=N3ho4shbsusMq6xrHjwnfci5KDvqDptlFTO6PO2avjmu12PECEeNaWxaTV6KfPBZ8bKNANtR-y5schJ1pYRU~yL2huU9HQr-xTMwqNYb-0I5clLf5m1En1gpLeV-SG4iWQwq71X2jAu9TBKw4OTnvvU-gpSRPRIEcOy-BcrgfEy4xonQdobXwIZrerYJ9PTUYiVXu4xO3khx0ujl4xJSFhaQJAeO9Kc28pT6N4jUZMfbimh4epyEYwOyH7FwJeTGxrZbfbyXCUNK0VwV8GcHylh5GYiFbVzZTci660PwR90D0d9t6lSmza9p~PLOqTaC7mvGbUzoYFqjR9Q9QABYqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483267&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differences in the virulence and the phylogenomic analyses of the fungi used in the current study. (A) Differences in the virulence of isolates used for whole genome analyses on grapevine. Results indicate <em>Lasiodiplodia theobromae</em> as the most virulent among the three species. CK indicate the control for which normal PDA plug was used without any fungal inoculum. Figures showed the results of the pathogenicity test conducted for all the isolates used in the study. (B) Phylogenomic analyses for evolutionary relationships among different fungi. Time scale of the neighbour-joining phylogenomic tree was shown by million years ago at each node. Estimates of divergence times (millions of years) calculated from the rate of sequence similarity are indicated at each node. The red dots are the fossil calibrations from the Time Tree database.</p></div></div><p>Integrated data from the <em>ab initio</em> prediction, homology-based search and RNASeq facilitated the prediction of 12,902 high-confidence protein-coding genes in <em>L. theobromae</em> (CSS-01s), with an average sequence length of 1,629 bp. The number of predicted genes in the <em>L. theobromae</em> genome is similar to those in other fungal genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). On average, each predicted gene contains 2.8 exons, with 9,902 of the predicted genes containing introns. The coding regions of these predicted genes constitute 43.6% of the genome. The genomic features for <em>L. theobromae</em> CSS-01s is shown in <a reveal-id="dsx040-T2" data-open="dsx040-T2" class="link link-reveal link-table xref-fig">Table 2</a>. The genome features of the other five <em>Botryosphaeriaceae</em> isolates are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>. <div content-id="dsx040-T2" class="table-modal"><div class="table-wrap"><div id="dsx040-T2" data-id="dsx040-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Genome features of <em>L. theobromae</em> CSS<em>-</em>01S</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Features </th><th><em>L. theobromae</em> CSS-01s </th></tr></thead><tbody><tr><td>Size (Mb) </td><td>43.3 </td></tr><tr><td>Coverage </td><td>90 x </td></tr><tr><td>% G+C content </td><td>54.77 </td></tr><tr><td>% Repeat </td><td>3.02 </td></tr><tr><td>Protein-coding genes </td><td>12,902 </td></tr><tr><td>Average gene length (bp) </td><td>1,629 </td></tr><tr><td>Gene density (# gene per Mb) </td><td>297 </td></tr><tr><td>Exons per gene </td><td>2.8 </td></tr><tr><td>Average exon length (bp) </td><td>506 </td></tr><tr><td>Introns per gene </td><td>1.8 </td></tr><tr><td>Average intron length (bp) </td><td>88 </td></tr><tr><td>tRNA </td><td>177 </td></tr></tbody></table></div><div content-id="dsx040-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Features </th><th><em>L. theobromae</em> CSS-01s </th></tr></thead><tbody><tr><td>Size (Mb) </td><td>43.3 </td></tr><tr><td>Coverage </td><td>90 x </td></tr><tr><td>% G+C content </td><td>54.77 </td></tr><tr><td>% Repeat </td><td>3.02 </td></tr><tr><td>Protein-coding genes </td><td>12,902 </td></tr><tr><td>Average gene length (bp) </td><td>1,629 </td></tr><tr><td>Gene density (# gene per Mb) </td><td>297 </td></tr><tr><td>Exons per gene </td><td>2.8 </td></tr><tr><td>Average exon length (bp) </td><td>506 </td></tr><tr><td>Introns per gene </td><td>1.8 </td></tr><tr><td>Average intron length (bp) </td><td>88 </td></tr><tr><td>tRNA </td><td>177 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx040-T2" data-id="dsx040-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Genome features of <em>L. theobromae</em> CSS<em>-</em>01S</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Features </th><th><em>L. theobromae</em> CSS-01s </th></tr></thead><tbody><tr><td>Size (Mb) </td><td>43.3 </td></tr><tr><td>Coverage </td><td>90 x </td></tr><tr><td>% G+C content </td><td>54.77 </td></tr><tr><td>% Repeat </td><td>3.02 </td></tr><tr><td>Protein-coding genes </td><td>12,902 </td></tr><tr><td>Average gene length (bp) </td><td>1,629 </td></tr><tr><td>Gene density (# gene per Mb) </td><td>297 </td></tr><tr><td>Exons per gene </td><td>2.8 </td></tr><tr><td>Average exon length (bp) </td><td>506 </td></tr><tr><td>Introns per gene </td><td>1.8 </td></tr><tr><td>Average intron length (bp) </td><td>88 </td></tr><tr><td>tRNA </td><td>177 </td></tr></tbody></table></div><div content-id="dsx040-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Features </th><th><em>L. theobromae</em> CSS-01s </th></tr></thead><tbody><tr><td>Size (Mb) </td><td>43.3 </td></tr><tr><td>Coverage </td><td>90 x </td></tr><tr><td>% G+C content </td><td>54.77 </td></tr><tr><td>% Repeat </td><td>3.02 </td></tr><tr><td>Protein-coding genes </td><td>12,902 </td></tr><tr><td>Average gene length (bp) </td><td>1,629 </td></tr><tr><td>Gene density (# gene per Mb) </td><td>297 </td></tr><tr><td>Exons per gene </td><td>2.8 </td></tr><tr><td>Average exon length (bp) </td><td>506 </td></tr><tr><td>Introns per gene </td><td>1.8 </td></tr><tr><td>Average intron length (bp) </td><td>88 </td></tr><tr><td>tRNA </td><td>177 </td></tr></tbody></table></div></div></p><p>The predicted genes were further annotated based on comparisons with several databases, as indicated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. An interproScan analysis identified 7,733 proteins belonging to 3,570 protein families in <em>L. theobromae</em>. Functional annotation showed that among the 12,902 predicted genes, 7,149 (55.4%) could be assigned with GO terms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 2</a></span>), and 5,520 genes could be mapped to the KEGG pathway database. Our analyses predicted 937 secreted proteins, accounting for 6.9% of the proteome, with 359 proteins identified as potential effectors. Similar numbers of secreted proteins were found in <em>D.</em><em>seriata</em><a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> (910) and <em>N.</em><em>parvum</em><a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> (1097). However, the corresponding number of secreted proteins found in the <em>Pyricularia oryzae</em><a reveal-id="dsx040-B32" data-open="dsx040-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> (739), <em>Valsa mali</em><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (782) and <em>V. pyri</em><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (774) genomes were lower than that of <em>L</em>. <em>theobromae</em>. Of all the secreted proteins identified, 269 (including 101 potential effectors) were specific to <em>L. theobromae.</em> Among the protein-coding genes revealed by the different annotation systems, the major functional groups in <em>L. theobromae</em> include degrading CAZymes (820 genes, of which 198 are specific to <em>L</em>. <em>theobromae</em>), cellular transporters (2,419 genes, of which 307 are specific), transcription factors (528 genes, of which 118 are specific) and secondary metabolites (54 genes, none were specific) (see <a reveal-id="dsx040-F2" data-open="dsx040-F2" class="link link-reveal link-table xref-fig">Fig. 2C and D</a>). Genome synteny analysis revealed an overall correlation between the homologous chromosomes of the six isolates (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Repetitive elements identified in this study are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>. </p>                        <a id="112483270" scrollto-destination="112483270"></a>
<div id="dsx040-F2" data-id="dsx040-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f2.png?Expires=2147483647&Signature=Z8DHTeV0TSZO9-W1wSIeqYAXcOcw-AGDEz2-OEXkDmw6ZxKrkWpNYKvG8OECK~fFgr2y8zB9AAS3gj6L1SrxK1zaPUr0qp3n5JT11LlMOrOJmK9TTGJlV-mY6QeZj4qaV7we6qTnbxsxmnLHRxyNpVEsOIA03iQsBP9oFAiIUnTKPZhsEgUE0k11Dmiw7mOyU2RAC5Gr2bo4MeNqKTupENGSdCfzNysQPvDJ5WWZgWMp8spK4EEvy0HvHsolKaXRy3F7u-JOQjz2Bd~hXDnNATYggfZ9eEfyZ4l6q-QbY0IHvJYQWK9jiD26USv2kEsgvjVpVn6T233nVLq4aJtJWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the gene family size comparison between the L. theobromae (CSS-01s) and other fungi. (A and B) Venn diagram of predicted gene families in (A) L. theobromae, B. dothidea and N. parvum and (B) Botryosphaeriaceae family with other fungi. The values explain the union counts of orthologous gene families shared among different fungi groups. Botryosphaeriaceae taxa share the highest number of unique gene families with opportunistic pathogens. (C) Top 20 Pfam domains in L. theobromae CSS-01s genome, compared with those in other fungi used in the comparative analysis. (D) Pair-wise comparison of Pfam gene family sizes between L. theobromae CSS-01s and other fungi used in the comparative analysis." path-from-xml="dsx040f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483270/dsx040f2.tif" path-from-xml="dsx040f2.tif" target="_blank">View large</a><a section="112483270" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f2.png?Expires=2147483647&Signature=leDhzGvO5W~I6rtohfQw3l8OEd-w0ZXxGGfgLjm1sskix-pKbk22Fr0G1dCA-LmJM7j4HlcB7azCcWageMz3teCVExuNk9lEDrFt6w1tXJUDV-ozOTq2ellEzji6vegLPqEO~kmXayL0h8OsAPcQCsdHCi~SBsrOfo~W6lIxOQ33XdYT0l6yGaNs2QrKAPwi5AcNVJ0WoTz-MV1FclgK~PaqC2nT-6UdV7X56JW4FJeD13W51NZpwYwYXh8R3GiWhPgoPDNBogCAj3ZdSN0qjAKUzBwhsIJtIa06TqcDU5ohRhbInCL13kIC1wZNiDsEDtEtGfBrNWwJ7p3cEA1OjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483270&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the gene family size comparison between the <em>L. theobromae</em> (CSS-01s) and other fungi. (A and B) Venn diagram of predicted gene families in (A) <em>L. theobromae, B. dothidea</em> and <em>N. parvum</em> and (B) <em>Botryosphaeriaceae</em> family with other fungi. The values explain the union counts of orthologous gene families shared among different fungi groups. <em>Botryosphaeriaceae</em> taxa share the highest number of unique gene families with opportunistic pathogens. (C) Top 20 Pfam domains in <em>L. theobromae</em> CSS-01s genome, compared with those in other fungi used in the comparative analysis. (D) Pair-wise comparison of Pfam gene family sizes between <em>L. theobromae</em> CSS-01s and other fungi used in the comparative analysis.</p></div></div><div content-id="dsx040-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f2.png?Expires=2147483647&Signature=Z8DHTeV0TSZO9-W1wSIeqYAXcOcw-AGDEz2-OEXkDmw6ZxKrkWpNYKvG8OECK~fFgr2y8zB9AAS3gj6L1SrxK1zaPUr0qp3n5JT11LlMOrOJmK9TTGJlV-mY6QeZj4qaV7we6qTnbxsxmnLHRxyNpVEsOIA03iQsBP9oFAiIUnTKPZhsEgUE0k11Dmiw7mOyU2RAC5Gr2bo4MeNqKTupENGSdCfzNysQPvDJ5WWZgWMp8spK4EEvy0HvHsolKaXRy3F7u-JOQjz2Bd~hXDnNATYggfZ9eEfyZ4l6q-QbY0IHvJYQWK9jiD26USv2kEsgvjVpVn6T233nVLq4aJtJWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the gene family size comparison between the L. theobromae (CSS-01s) and other fungi. (A and B) Venn diagram of predicted gene families in (A) L. theobromae, B. dothidea and N. parvum and (B) Botryosphaeriaceae family with other fungi. The values explain the union counts of orthologous gene families shared among different fungi groups. Botryosphaeriaceae taxa share the highest number of unique gene families with opportunistic pathogens. (C) Top 20 Pfam domains in L. theobromae CSS-01s genome, compared with those in other fungi used in the comparative analysis. (D) Pair-wise comparison of Pfam gene family sizes between L. theobromae CSS-01s and other fungi used in the comparative analysis." path-from-xml="dsx040f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483270/dsx040f2.tif" path-from-xml="dsx040f2.tif" target="_blank">View large</a><a section="112483270" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f2.png?Expires=2147483647&Signature=leDhzGvO5W~I6rtohfQw3l8OEd-w0ZXxGGfgLjm1sskix-pKbk22Fr0G1dCA-LmJM7j4HlcB7azCcWageMz3teCVExuNk9lEDrFt6w1tXJUDV-ozOTq2ellEzji6vegLPqEO~kmXayL0h8OsAPcQCsdHCi~SBsrOfo~W6lIxOQ33XdYT0l6yGaNs2QrKAPwi5AcNVJ0WoTz-MV1FclgK~PaqC2nT-6UdV7X56JW4FJeD13W51NZpwYwYXh8R3GiWhPgoPDNBogCAj3ZdSN0qjAKUzBwhsIJtIa06TqcDU5ohRhbInCL13kIC1wZNiDsEDtEtGfBrNWwJ7p3cEA1OjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483270&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the gene family size comparison between the <em>L. theobromae</em> (CSS-01s) and other fungi. (A and B) Venn diagram of predicted gene families in (A) <em>L. theobromae, B. dothidea</em> and <em>N. parvum</em> and (B) <em>Botryosphaeriaceae</em> family with other fungi. The values explain the union counts of orthologous gene families shared among different fungi groups. <em>Botryosphaeriaceae</em> taxa share the highest number of unique gene families with opportunistic pathogens. (C) Top 20 Pfam domains in <em>L. theobromae</em> CSS-01s genome, compared with those in other fungi used in the comparative analysis. (D) Pair-wise comparison of Pfam gene family sizes between <em>L. theobromae</em> CSS-01s and other fungi used in the comparative analysis.</p></div></div>                            <h3 scrollto-destination=112483271 id="112483271" class="section-title" >3.2. Selection pressures on orthologues of <em>L. theobromae</em> and other botryosphaeriaceous isolates</h3>
<p>Adaptive evolution is often associated with changes in protein-coding genes in the genome<a reveal-id="dsx040-B41" data-open="dsx040-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a>, and the Ka/Ks ratio is commonly used to reveal the direction and magnitude of selection upon these genes. Among the orthologous pairs between the <em>L. theobromae</em> strains, there were significant divergences for 809 orthologues. The Ka and Ks rates as well as their ratios were all calculated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Among these, four pairs of orthologues have Ka/Ks ratios &gt; 1, and 13 have Ka/Ks rates ranging from 0.5 to 1. The Ka/Ks ratios between <em>L. theobromae</em> CSS-01s and the four other isolates of botryosphaeriaceous taxa are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>. Orthologous pairs with a Ka/Ks value &gt; 1 indicate positive selection, whereas Ka/Ks value between 0.5 and 1 indicate weak purifying selection. Interestingly, all four genes (evm.model.scaffold_15.78, evm.model.scaffold_4.947, evm.model.scaffold_9.354 and evm.model.scaffold_9.168) showing positive selection were newly discovered genes with unknown functions. However, two of the genes have similar sequences in <em>N. parvum</em><a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and <em>Macrophoma phaseolina</em><a reveal-id="dsx040-B42" data-open="dsx040-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>. The similar gene found in <em>N. parvum</em> contains a W40 domain with a wide variety of functions involved in regulatory modules in signal transduction, pre-mRNA processing and cytoskeleton assembly.<a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p>                            <h3 scrollto-destination=112483273 id="112483273" class="section-title" >3.3. Comparative genomics and evolution</h3>
<p>Comparative genomics revealed a broad range of gene families available in the genome with potential functions related to virulence. In addition to the six botryosphaeriaceous isolates sequenced in this study, 15 other fungal genomes were also included for comparative analyses.<a reveal-id="dsx040-B32" data-open="dsx040-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsx040-B33" data-open="dsx040-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> These taxa includes pathogens, opportunistic pathogens and non-pathogenic fungi. According to the analysis, these species shared 1,029 single-copy orthologous groups. Phylogenetic analysis (<a reveal-id="dsx040-F1" data-open="dsx040-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>) revealed that the species belonging to <em>Botryosphaeriaceae</em> are more closely related to <em>A. brassicicola</em> and <em>A. alternata</em>, than to the other taxa. This is acceptable, because they all belong to the same class, Dothideomycetes. When compared with all other genomes, 2,813 gene families containing 1,129 species-specific groups (1,156 genes) were found to be specific to <em>Botryosphaeriaceae</em>, most of which were single-copy genes (<a reveal-id="dsx040-F2" data-open="dsx040-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>). These genes are enriched in cytochrome P450s and GO functional categories, such as hydrolase activity, lyase activity, metabolic processes, oxidation-reduction processes, oxidoreductase activity, iron ion binding, heme binding and transporter activity (FDR &lt;0.05, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>).</p><p>Based on their lifestyles, botryosphaeriaceous taxa share 221 unique orthologous gene families with pathogenic fungi, 1,049 with opportunistic fungal pathogens and 36 with non-pathogenic fungi (<a reveal-id="dsx040-F2" data-open="dsx040-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>). Orthologous families shared between pathogenic fungi and <em>Botryosphaeriaceae</em> are enriched in phosphorylation, signal transduction and enzyme inhibitor activity functions (FDR &lt;0.05, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>), whereas no significant functional enrichment was found in the gene families shared between the non-pathogenic taxa and botryosphaeriaceous taxa. Among all other fungal groups, <em>Botryosphaeriaceae</em> taxa shared the most orthologous gene families (1,049) with opportunistic fungal pathogens (<a reveal-id="dsx040-F2" data-open="dsx040-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>), with enriched hydrolase, catalytic, cellulose binding, oxidoreductase, transmembrane transporter, and transcription factor activities, among many others (FDR &lt;0.05, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). These genes are mainly involved in the metabolism of starch, sucrose, fructose and mannose, as well as secondary metabolite synthesis.</p>                            <h3 scrollto-destination=112483276 id="112483276" class="section-title" >3.4. Gene family expansions and contractions</h3>
<p>The expansion of gene families in fungal genomes is mainly associated with host adaptations<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B43" data-open="dsx040-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> and functions linked to virulence, such as host cell wall degradation, transport functions and secondary metabolite synthesis.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B44" data-open="dsx040-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> This study used the Computational Analysis of gene Family Evolution (CAFÉ)<a reveal-id="dsx040-B45" data-open="dsx040-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> program to identify gene families that have potentially undergone significant expansions or contractions in botryosphaeriaceous taxa. When compared with all other fungal genomes analysed in this study, five gene families (51 genes) in <em>L. theobromae</em> have significantly expanded (<em>P</em> ≤0.05, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>), whereas one family has contracted. According to Clusters of Orthologous Group (COG) annotation, these expanded gene families in botryosphaeriaceous taxa are functionally related to secondary metabolism, plant cell wall degradation and cellular transporters such as non-ribosomal peptide synthetases (NRPS), cytochrome P450, polyketide synthase (PKS), ATP-binding cassette (ABC)–type multidrug transport system, ATPase component, and sulphite reductase and alpha subunit (flavoprotein).</p>                            <h3 scrollto-destination=112483278 id="112483278" class="section-title" >3.5. Secondary metabolism genes</h3>
<p>Secondary metabolites are believed to be involved in the development of certain disease symptoms and, in some cases, cause the host to die.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B6" data-open="dsx040-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx040-B46" data-open="dsx040-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Genes encoding a biosynthetic pathway are co-regulated at the transcriptional level and are often physically clustered on fungal chromosomes.<a reveal-id="dsx040-B46" data-open="dsx040-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> A biosynthetic gene is often surrounded by genes encoding transporters and others involved in post-synthesis modification of the metabolite.<a reveal-id="dsx040-B46" data-open="dsx040-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsx040-B47" data-open="dsx040-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> In this study, two computational programs, SMURF (<a class="link link-uri" href="http://jcvi.org/smurf/index.php" target="_blank">http://jcvi.org/smurf/index.php</a> (20 June 2016, date last accessed))<a reveal-id="dsx040-B48" data-open="dsx040-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and antiSAMSH (<a class="link link-uri" href="http://antismash.secondarymetabolites.org/" target="_blank">http://antismash.secondarymetabolites.org/</a> (20 June 2016, date last accessed)),<a reveal-id="dsx040-B49" data-open="dsx040-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> were used to identify genes involved in the biosynthesis of secondary metabolites. The <em>L. theobromae</em> genome contains a total of 58 secondary metabolite gene clusters, categorized into eight main groups. The majority of the clusters belong to type 1 polyketide synthases (t1 PKS, 11 clusters), followed by 12 non-ribosomal peptide synthetases (NRPS), 10 terpene synthases, four non-ribosomal peptide synthetase–t1 PKS hybrids, three putative fatty acid clusters, a type 1 PKS-non-ribosomal peptide synthetase hybrid, a non-ribosomal peptide synthetase–putative fatty acid hybrid and another putative saccharide cluster. In addition to these eight known clusters, there were nine other clusters encoding secondary metabolite related proteins.</p>                            <h3 scrollto-destination=112483280 id="112483280" class="section-title" >3.6. Host-pathogen interactions revealed during infection</h3>
<p>A comprehensive understanding of the <em>L. theobromae</em> transcriptome should significantly facilitate the discovery of new genes and their expression profiles. To understand pathogenicity-related genes, the transcript profiles of <em>L. theobromae</em> in <em>V.</em><em>vinifera</em> were analysed at two early infection stages (<a reveal-id="dsx040-F3" data-open="dsx040-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). When compared with axenic cultures, 75 and 283 genes were up-regulated, whereas 89 and 232 genes were down-regulated, at 8 and 12 hpi, respectively (<a reveal-id="dsx040-F3" data-open="dsx040-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). During the early infection stages of <em>L. theobromae</em>, a total of 285 up-regulated genes and 243 down-regulated genes were identified. Among these genes, genes that exhibited significant expression during infection are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S14 and S15</a></span>. According to GO enrichment analysis, the up-regulated genes at 8 hpi were enriched in molecular function and biological process terms, concentrated in ferric iron binding, the hydrolysis of O-glycosyl compounds and polygalacturonase activity. However, more genes were up-regulated at 12 hpi than at 8 hpi. At 12 hpi, most of the up-regulated genes were involved in pectin degradation, polygalacturonase activities and carbohydrate catabolic processes. This finding indicates that during infection, these up-regulated genes facilitate the degradation of cell walls by <em>L. theobromae</em>. KEGG pathway analysis confirmed that the genes up-regulated at 8 and 12 hpi were significantly enriched in starch and sucrose metabolism pathways, as well as in the pentose and glucuronate interconversion pathways. </p>                        <a id="112483282" scrollto-destination="112483282"></a>
<div id="dsx040-F3" data-id="dsx040-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f3.png?Expires=2147483647&Signature=YxzcWTyPUukhpUsy8lD0JTz4yL1-KNmRpxoBJQcXq-6jeQJBEBo37O1EpkhhT0YURRIRj08kL6RRU-di~nFAem8fMP~0qRHMbotwDA3InO8PhxrZuq4k1rMn3sQ9~IxLsRrxtMjf3idtOKufXvRNsPmPEq0K1t~SAraE~DvKGYWYUN-1g4Wmx~6GvpL3JMnJztbSVxcQBMgmvk6hbSZJzxbbtGE2zlhl5DKoeg553zuauUPn9WCJ5-E32HVidO~cauCuE~3vdTMCkg36QUje0gdGURgGC-ac7kXggqIKPtPXwlqFd9UNzdszq0WLgFu2wZ~vlWUMD6ev2meFBNw5YQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Difference in the virulence of Lasiodiplodia theobromae and their gene expression at different stages of growth and at different temperatures. (A) Difference in the virulence of L. theobromae at different infection stages on grape shoots. Figure shows the progress of infection of L. theobromae isolate CSS-01s at (i) 8, (ii) 12 and (iii) 24 hpi. (B) Differences in the virulence of L. theobromae isolate CSS-01s at different temperatures on grape shoots and their growth differences on the axenic cultures at respective temperatures: (i) control test with PDA without inoculum, (ii) virulence at 25 °C (iii) virulence at 35 °C, (iv) axenic culture growth at 25 °C and (v) culture growth at 35 °C. (C) Venn diagram showing in planta differentially expressed genes of L. theobromae at different time points after inoculation. Numbers indicate differentially expressed genes that are specific to each time point, or shared by two different time points. Differentially expressed genes at 8 and 12 hpi are represented with circles. (D) Up-regulation of LysM domain genes of L. theobromae during transcriptome analysis. L indicates samples collected from axenic cultures, M indicates samples of fungus and host tissues. Two different temperatures of 25 °C and 35 °C indicated by numbers 25 and 35. Two sample collection time points were indicated by 8 and 12 hpi." path-from-xml="dsx040f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483282/dsx040f3.tif" path-from-xml="dsx040f3.tif" target="_blank">View large</a><a section="112483282" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f3.png?Expires=2147483647&Signature=crxrG7ypHMWklV4BKfXYYi~q~cNNC2UWMeykq0WncJCS5iQPntQOMqYIkHLo93vyT9KDZrqgkHijNY5PYAhGEsEO6vv4BTOJLRtYEBLBJReW0WceiLfP7ChgZRMNk0hLo9krAHoPDxjr7i01Cwb~qGzs5AuaZtTqUxKBdbHVeP~M74cgxEFP~6bHbuY2uMUlduqtpM08tvtRwiTs9ZkMikuyapY2XGjA-ZUw4CLzL9ClWbxQmaxAlB9nXs4L6fR7p6bBD-tjsFUFRAiKgSOhm-zy2fJKSncMg6LXp-rCM~LXMyr0DNTub1M7JuLf97Whj1AfpTnm7kvSlkguqpP98A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483282&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Difference in the virulence of <em>Lasiodiplodia theobromae</em> and their gene expression at different stages of growth and at different temperatures. (A) Difference in the virulence of <em>L. theobromae</em> at different infection stages on grape shoots. Figure shows the progress of infection of <em>L. theobromae</em> isolate CSS-01s at (i) 8, (ii) 12 and (iii) 24 hpi. (B) Differences in the virulence of <em>L. theobromae</em> isolate CSS-01s at different temperatures on grape shoots and their growth differences on the axenic cultures at respective temperatures: (i) control test with PDA without inoculum, (ii) virulence at 25 °C (iii) virulence at 35 °C, (iv) axenic culture growth at 25 °C and (v) culture growth at 35 °C. (C) Venn diagram showing <em>in planta</em> differentially expressed genes of <em>L</em>. <em>theobromae</em> at different time points after inoculation. Numbers indicate differentially expressed genes that are specific to each time point, or shared by two different time points. Differentially expressed genes at 8 and 12 hpi are represented with circles. (D) Up-regulation of LysM domain genes of <em>L. theobromae</em> during transcriptome analysis. L indicates samples collected from axenic cultures, M indicates samples of fungus and host tissues. Two different temperatures of 25 °C and 35 °C indicated by numbers 25 and 35. Two sample collection time points were indicated by 8 and 12 hpi.</p></div></div><div content-id="dsx040-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f3.png?Expires=2147483647&Signature=YxzcWTyPUukhpUsy8lD0JTz4yL1-KNmRpxoBJQcXq-6jeQJBEBo37O1EpkhhT0YURRIRj08kL6RRU-di~nFAem8fMP~0qRHMbotwDA3InO8PhxrZuq4k1rMn3sQ9~IxLsRrxtMjf3idtOKufXvRNsPmPEq0K1t~SAraE~DvKGYWYUN-1g4Wmx~6GvpL3JMnJztbSVxcQBMgmvk6hbSZJzxbbtGE2zlhl5DKoeg553zuauUPn9WCJ5-E32HVidO~cauCuE~3vdTMCkg36QUje0gdGURgGC-ac7kXggqIKPtPXwlqFd9UNzdszq0WLgFu2wZ~vlWUMD6ev2meFBNw5YQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Difference in the virulence of Lasiodiplodia theobromae and their gene expression at different stages of growth and at different temperatures. (A) Difference in the virulence of L. theobromae at different infection stages on grape shoots. Figure shows the progress of infection of L. theobromae isolate CSS-01s at (i) 8, (ii) 12 and (iii) 24 hpi. (B) Differences in the virulence of L. theobromae isolate CSS-01s at different temperatures on grape shoots and their growth differences on the axenic cultures at respective temperatures: (i) control test with PDA without inoculum, (ii) virulence at 25 °C (iii) virulence at 35 °C, (iv) axenic culture growth at 25 °C and (v) culture growth at 35 °C. (C) Venn diagram showing in planta differentially expressed genes of L. theobromae at different time points after inoculation. Numbers indicate differentially expressed genes that are specific to each time point, or shared by two different time points. Differentially expressed genes at 8 and 12 hpi are represented with circles. (D) Up-regulation of LysM domain genes of L. theobromae during transcriptome analysis. L indicates samples collected from axenic cultures, M indicates samples of fungus and host tissues. Two different temperatures of 25 °C and 35 °C indicated by numbers 25 and 35. Two sample collection time points were indicated by 8 and 12 hpi." path-from-xml="dsx040f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483282/dsx040f3.tif" path-from-xml="dsx040f3.tif" target="_blank">View large</a><a section="112483282" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f3.png?Expires=2147483647&Signature=crxrG7ypHMWklV4BKfXYYi~q~cNNC2UWMeykq0WncJCS5iQPntQOMqYIkHLo93vyT9KDZrqgkHijNY5PYAhGEsEO6vv4BTOJLRtYEBLBJReW0WceiLfP7ChgZRMNk0hLo9krAHoPDxjr7i01Cwb~qGzs5AuaZtTqUxKBdbHVeP~M74cgxEFP~6bHbuY2uMUlduqtpM08tvtRwiTs9ZkMikuyapY2XGjA-ZUw4CLzL9ClWbxQmaxAlB9nXs4L6fR7p6bBD-tjsFUFRAiKgSOhm-zy2fJKSncMg6LXp-rCM~LXMyr0DNTub1M7JuLf97Whj1AfpTnm7kvSlkguqpP98A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483282&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Difference in the virulence of <em>Lasiodiplodia theobromae</em> and their gene expression at different stages of growth and at different temperatures. (A) Difference in the virulence of <em>L. theobromae</em> at different infection stages on grape shoots. Figure shows the progress of infection of <em>L. theobromae</em> isolate CSS-01s at (i) 8, (ii) 12 and (iii) 24 hpi. (B) Differences in the virulence of <em>L. theobromae</em> isolate CSS-01s at different temperatures on grape shoots and their growth differences on the axenic cultures at respective temperatures: (i) control test with PDA without inoculum, (ii) virulence at 25 °C (iii) virulence at 35 °C, (iv) axenic culture growth at 25 °C and (v) culture growth at 35 °C. (C) Venn diagram showing <em>in planta</em> differentially expressed genes of <em>L</em>. <em>theobromae</em> at different time points after inoculation. Numbers indicate differentially expressed genes that are specific to each time point, or shared by two different time points. Differentially expressed genes at 8 and 12 hpi are represented with circles. (D) Up-regulation of LysM domain genes of <em>L. theobromae</em> during transcriptome analysis. L indicates samples collected from axenic cultures, M indicates samples of fungus and host tissues. Two different temperatures of 25 °C and 35 °C indicated by numbers 25 and 35. Two sample collection time points were indicated by 8 and 12 hpi.</p></div></div><p>From a total of 75 genes up-regulated at 8 hpi, 40 were secreted proteins, of which 20 were predicted to be putative effectors. Among the 283 up-regulated genes at 12 hpi, 119 were secreted proteins, of which 51 were predicted to be putative effectors. These putative effectors contain conserved functional domains such as glycoside hydrolase, pectate lyase or fungal-specific extracellular EGF-like domains (CFEM) to enhance their invasion and pathogenesis. Unlike in other pathogenic taxa,<a reveal-id="dsx040-B42" data-open="dsx040-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> secondary metabolism genes were not up-regulated at both 8 and 12 hpi.</p>                            <h3 scrollto-destination=112483284 id="112483284" class="section-title" >3.7. Expansion and expression of membrane transporters</h3>
<p>Among the wide array of membrane transporters, two main families are abundant in fungal genomes, the ABC family and the major facilitator super (MFS) family.<a reveal-id="dsx040-B50" data-open="dsx040-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> The <em>L. theobromae</em> genome encodes 917 membrane transporters. Searches against the transporter classification database confirm that <em>L. theobromae</em> has the largest number of membrane transporters compared with the other taxa (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S16</a></span>). The number of membrane transporters in the <em>L. theobromae</em> genome is similar to that in <em>A. fumigatus</em>. Several membrane transporters in the <em>L. theobromae</em> genome are significantly expanded, including amino acid/choline transporters, anion:cation symporters, sugar porters, drug:H+ antiporter-2, multidrug resistance exporters (a.k.a. ABCB) and the pleiotropic drug resistance (PDR) transporters (<a reveal-id="dsx040-F4" data-open="dsx040-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Upon host contact, 17 membrane transporters were significantly up-regulated, including amino acid transporters, PDRs, sugar porters and triose-phosphate transporters. The multidrug resistance proteins in the PDR family confer resistance to cycloheximide and antifungal agents such as azoles and terbinafine<a reveal-id="dsx040-B51" data-open="dsx040-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a>, and both were up-regulated. Two genes in the sugar porter family were significantly up-regulated; one of the two genes is a glucose/xylose: H+ symporter and the other is an MFS permease. These sugar transporters may allow the fungus to grow under different pH conditions.<a reveal-id="dsx040-B52" data-open="dsx040-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a></p>                        <a id="112483286" scrollto-destination="112483286"></a>
<div id="dsx040-F4" data-id="dsx040-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f4.png?Expires=2147483647&Signature=R5OG4N-nPsScquwKtKLcHrnQC9X-49Peo8R2a2r9A8oHIrmilePnVMT0gXEflblt-DgnIW~zk9NcZv15Dcee~XVHgiXM0WD9yxfQd3xN3xuvBTTDptOdyuOPJxsORWR-O2ARXATxa24r4DQvikylbwD24EkC8nXQz13XhzRz0eNjuXb88MMS7G-EHt1YXcowpZZrJvTerLtLFusdMQn8PhJh9Wg88tKw8hFJjdouOAdi6x9VvBg82G8XyLbCgiigd4A09brtr0r5Cos6XbhNzlxeMdJ32UwRrXEIjnoBhvUwu9Adpla0JuKJ8GjRGbrwHdI1Ubx7cUGDYV9chbySoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene family expansions and contractions of L. theobromae, and up-regulation of gene expressions during infection. (A) Comparison of membrane transporter families in L. theobromae (CSS-01s). The families with red colour mean expansion in the L. theobromae genome, whereas the blue means contraction. (B) Plots of up-regulated CWDE gene expressions in each family during the infections of L. theobromae. Transcriptomic analysis revealed several CWDE belonging to different families up-regulated during infection. These gene families were involved in pathogenicity processes of the fungi. PL indicates pectin lyases." path-from-xml="dsx040f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483286/dsx040f4.tif" path-from-xml="dsx040f4.tif" target="_blank">View large</a><a section="112483286" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f4.png?Expires=2147483647&Signature=xRy9nN13ZYBWWsvfGSx7bo6gIg4dJS6u0gybq3OcFqzPiJbBNBacXwgTe0QQAI~CNcGVcE05SoBIx5ShpitALr7-Y5E5TnZgvT-swS9QcDWCY8C1fIf2xqp46h4LCC8mr~e~DWyTE56gCp5JluUoT9~nByOBy4lAwvM4q~XFZa6j8osy7OH78Febj-spPSADU-FeXY4C~-OFZ4ueZ9Sm15FXmZccIXZNjGs4EvCdbWrb011hpA5TvnXsHoC4NQOqrr3l7vxD1BN5pFSCCnGGn7emJJywNm87MWv8~-4lK7Gz6sJ-sq8RpkAnnDPSirf~qJ4GvYj63hIjX0JgxE9TzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483286&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene family expansions and contractions of <em>L. theobromae</em>, and up-regulation of gene expressions during infection. (A) Comparison of membrane transporter families in <em>L. theobromae</em> (CSS-01s). The families with red colour mean expansion in the <em>L. theobromae</em> genome, whereas the blue means contraction. (B) Plots of up-regulated CWDE gene expressions in each family during the infections of <em>L. theobromae.</em> Transcriptomic analysis revealed several CWDE belonging to different families up-regulated during infection. These gene families were involved in pathogenicity processes of the fungi. PL indicates pectin lyases.</p></div></div><div content-id="dsx040-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f4.png?Expires=2147483647&Signature=R5OG4N-nPsScquwKtKLcHrnQC9X-49Peo8R2a2r9A8oHIrmilePnVMT0gXEflblt-DgnIW~zk9NcZv15Dcee~XVHgiXM0WD9yxfQd3xN3xuvBTTDptOdyuOPJxsORWR-O2ARXATxa24r4DQvikylbwD24EkC8nXQz13XhzRz0eNjuXb88MMS7G-EHt1YXcowpZZrJvTerLtLFusdMQn8PhJh9Wg88tKw8hFJjdouOAdi6x9VvBg82G8XyLbCgiigd4A09brtr0r5Cos6XbhNzlxeMdJ32UwRrXEIjnoBhvUwu9Adpla0JuKJ8GjRGbrwHdI1Ubx7cUGDYV9chbySoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene family expansions and contractions of L. theobromae, and up-regulation of gene expressions during infection. (A) Comparison of membrane transporter families in L. theobromae (CSS-01s). The families with red colour mean expansion in the L. theobromae genome, whereas the blue means contraction. (B) Plots of up-regulated CWDE gene expressions in each family during the infections of L. theobromae. Transcriptomic analysis revealed several CWDE belonging to different families up-regulated during infection. These gene families were involved in pathogenicity processes of the fungi. PL indicates pectin lyases." path-from-xml="dsx040f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483286/dsx040f4.tif" path-from-xml="dsx040f4.tif" target="_blank">View large</a><a section="112483286" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f4.png?Expires=2147483647&Signature=xRy9nN13ZYBWWsvfGSx7bo6gIg4dJS6u0gybq3OcFqzPiJbBNBacXwgTe0QQAI~CNcGVcE05SoBIx5ShpitALr7-Y5E5TnZgvT-swS9QcDWCY8C1fIf2xqp46h4LCC8mr~e~DWyTE56gCp5JluUoT9~nByOBy4lAwvM4q~XFZa6j8osy7OH78Febj-spPSADU-FeXY4C~-OFZ4ueZ9Sm15FXmZccIXZNjGs4EvCdbWrb011hpA5TvnXsHoC4NQOqrr3l7vxD1BN5pFSCCnGGn7emJJywNm87MWv8~-4lK7Gz6sJ-sq8RpkAnnDPSirf~qJ4GvYj63hIjX0JgxE9TzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483286&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene family expansions and contractions of <em>L. theobromae</em>, and up-regulation of gene expressions during infection. (A) Comparison of membrane transporter families in <em>L. theobromae</em> (CSS-01s). The families with red colour mean expansion in the <em>L. theobromae</em> genome, whereas the blue means contraction. (B) Plots of up-regulated CWDE gene expressions in each family during the infections of <em>L. theobromae.</em> Transcriptomic analysis revealed several CWDE belonging to different families up-regulated during infection. These gene families were involved in pathogenicity processes of the fungi. PL indicates pectin lyases.</p></div></div>                            <h3 scrollto-destination=112483287 id="112483287" class="section-title" >3.8. <em>Lasiodiplodia theobromae</em> secretome</h3>
<p>Proteins secreted by pathogenic fungi are essential for host–pathogen interactions, including the successful establishment of infections.<a reveal-id="dsx040-B53" data-open="dsx040-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Therefore, identification and functional annotation of the secretome is important for identifying potential virulence factors. The <em>L. theobromae</em> genome encodes a total of 937 secreted proteins, accounting for 7.2% of its proteome, as predicted by SignalP-4.1,<a reveal-id="dsx040-B28" data-open="dsx040-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> TargetP-1.1,<a reveal-id="dsx040-B54" data-open="dsx040-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and TMHMM-2.0.<a reveal-id="dsx040-B55" data-open="dsx040-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Among the secreted proteins identified, 417 were assigned with GO terms. Among the 283 genes up-regulated during infection, 105 are secreted proteins. Functional enrichment analysis showed that proteins involved in carbohydrate metabolism are significantly overrepresented in the up-regulated <em>L. theobromae</em> secretome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S17</a></span>).</p>                            <h3 scrollto-destination=112483289 id="112483289" class="section-title" >3.9. Prediction of putative effectors and their roles in suppressing HRs</h3>
<p>Plant pathogenic fungi secrete effectors that suppress the first layer of inducible defences, known as pathogen-associated molecular pattern (PAMP) triggered immunity, which detects conserved pathogen molecules, such as flagellin, Elongation Factor Thermo unstable, peptidoglycan and chitin by host receptors.<a reveal-id="dsx040-B56" data-open="dsx040-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> However, very little is known about the effector proteins that contribute to <em>Botryosphaeriaceae</em> virulence. Among the 937 secreted proteins predicted from the <em>L. theobromae</em> genome, we identified 359 putative effectors. Searches for conserved motifs among these effectors showed that 157 contain Y/F/WxC motifs, 45 contain L/IxAR motifs, 1 contains YxSL[R/K] motifs, and 13 contain a single RxLR motif each. Among these, 77 putative effectors were specific to <em>L. theobromae</em>. Five effectors (BR_1.280, BR_2.1361, BR_2.1420, BR_4.711 and BR_10.166) exhibited a strong suppressive effect, whereas BR_1.397, BR_1.899 and BR_5.292 had a weak effect in <em>N. benthamiana</em> (<a reveal-id="dsx040-F5" data-open="dsx040-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Two of the effectors, BR_1.280 and BR_2.1361, encode hypothetical proteins (<a reveal-id="dsx040-F5" data-open="dsx040-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). These results significantly expand our understanding of the determinants of pathogenicity or virulence in <em>Botryosphaeriaceae</em>. </p>                        <a id="112483291" scrollto-destination="112483291"></a>
<div id="dsx040-F5" data-id="dsx040-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f5.png?Expires=2147483647&Signature=xJ7ifUBPc4aVC5LHiHrtvef~Jn0mFMm~75KlttBS3pCTaLyrAbVbKnVItoXmBZGJm36mhmwXbKHQymNut4Ikr~KxRhxQyV1qxsco5DuF0MNLdV72vV1dHqJ-lVnzwgJFlwkbBwDjm3s5EG2lz1N6-LlwkxYyPfVrjg8xVNwi636ThLh0iOWTcMghRQ0FbwE3qqANlrDHh4lrjhZ2AfAC70pvJR6JtmKfGFKBci-lzjYoMseImcrxMaYc-iH1QQ9SEGuBsgg329k4Q5sOcX1fP0ef6~ah6mWa7bEbf65lI1ozzzbKKpCQLBf-Tka1fZFGiSGJUW63784KqNHnMNO6ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative effectors and their hypersensitive responses in N. benthamiana. Putative Botryosphaeriaceae effectors suppressed B. glumae-induced hyposensitive response in N. benthamiana. Representative cell death symptoms were photographed at 2 days after B. glumae inoculation. The right half leaf sections were injected with B. glumae with the pEDV empty vector and the left half sections were injected with B. glumae with effector gene constructs. Black circle represents pEDV empty vector, red circle represents effectors specific to Botryosphaeriaceae, orange circle represents commons effectors in Botryosphaeriaceae and the purple circle represents leaves injected with 0.9%NaCl." path-from-xml="dsx040f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483291/dsx040f5.tif" path-from-xml="dsx040f5.tif" target="_blank">View large</a><a section="112483291" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f5.png?Expires=2147483647&Signature=0Nl~FEzrjD3Vg-NnFKH04qdC8AIbL20gxbAc0GMb8OR-ep4MeoT4i6O15Nq1PxDiBR7gS13uq6ytY26LHvH3XF85oKVGGDfBJIETnnahtgBTMAIpALZ2VlxTBHw7ny0ufrCeWtPr3BMScb0BxAERSP9CBcTtWvxy03129u5hGqwS4X~O-VgYk9LornXIrP4xNCHg1bkQSXKMm8VDKjfvgscikuh7DJ0dtkxlkW7Vd5i1RTvRjVM99Zc8IXsDiiW4lC31WYKOfR2cd13-xb2JqajQ74FEY9vXDAQ7aHtrdNSRNYnsYidTegqati2K3rR-WIOwsdH2ONzy-HsCs~9LTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483291&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative effectors and their hypersensitive responses in <em>N. benthamiana</em>. Putative <em>Botryosphaeriaceae</em> effectors suppressed <em>B. glumae</em>-induced hyposensitive response in <em>N. benthamiana</em>. Representative cell death symptoms were photographed at 2 days after <em>B. glumae</em> inoculation. The right half leaf sections were injected with <em>B. glumae</em> with the pEDV empty vector and the left half sections were injected with <em>B. glumae</em> with effector gene constructs. Black circle represents pEDV empty vector, red circle represents effectors specific to <em>Botryosphaeriaceae</em>, orange circle represents commons effectors in <em>Botryosphaeriaceae</em> and the purple circle represents leaves injected with 0.9%NaCl.</p></div></div><div content-id="dsx040-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f5.png?Expires=2147483647&Signature=xJ7ifUBPc4aVC5LHiHrtvef~Jn0mFMm~75KlttBS3pCTaLyrAbVbKnVItoXmBZGJm36mhmwXbKHQymNut4Ikr~KxRhxQyV1qxsco5DuF0MNLdV72vV1dHqJ-lVnzwgJFlwkbBwDjm3s5EG2lz1N6-LlwkxYyPfVrjg8xVNwi636ThLh0iOWTcMghRQ0FbwE3qqANlrDHh4lrjhZ2AfAC70pvJR6JtmKfGFKBci-lzjYoMseImcrxMaYc-iH1QQ9SEGuBsgg329k4Q5sOcX1fP0ef6~ah6mWa7bEbf65lI1ozzzbKKpCQLBf-Tka1fZFGiSGJUW63784KqNHnMNO6ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative effectors and their hypersensitive responses in N. benthamiana. Putative Botryosphaeriaceae effectors suppressed B. glumae-induced hyposensitive response in N. benthamiana. Representative cell death symptoms were photographed at 2 days after B. glumae inoculation. The right half leaf sections were injected with B. glumae with the pEDV empty vector and the left half sections were injected with B. glumae with effector gene constructs. Black circle represents pEDV empty vector, red circle represents effectors specific to Botryosphaeriaceae, orange circle represents commons effectors in Botryosphaeriaceae and the purple circle represents leaves injected with 0.9%NaCl." path-from-xml="dsx040f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483291/dsx040f5.tif" path-from-xml="dsx040f5.tif" target="_blank">View large</a><a section="112483291" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f5.png?Expires=2147483647&Signature=0Nl~FEzrjD3Vg-NnFKH04qdC8AIbL20gxbAc0GMb8OR-ep4MeoT4i6O15Nq1PxDiBR7gS13uq6ytY26LHvH3XF85oKVGGDfBJIETnnahtgBTMAIpALZ2VlxTBHw7ny0ufrCeWtPr3BMScb0BxAERSP9CBcTtWvxy03129u5hGqwS4X~O-VgYk9LornXIrP4xNCHg1bkQSXKMm8VDKjfvgscikuh7DJ0dtkxlkW7Vd5i1RTvRjVM99Zc8IXsDiiW4lC31WYKOfR2cd13-xb2JqajQ74FEY9vXDAQ7aHtrdNSRNYnsYidTegqati2K3rR-WIOwsdH2ONzy-HsCs~9LTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483291&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative effectors and their hypersensitive responses in <em>N. benthamiana</em>. Putative <em>Botryosphaeriaceae</em> effectors suppressed <em>B. glumae</em>-induced hyposensitive response in <em>N. benthamiana</em>. Representative cell death symptoms were photographed at 2 days after <em>B. glumae</em> inoculation. The right half leaf sections were injected with <em>B. glumae</em> with the pEDV empty vector and the left half sections were injected with <em>B. glumae</em> with effector gene constructs. Black circle represents pEDV empty vector, red circle represents effectors specific to <em>Botryosphaeriaceae</em>, orange circle represents commons effectors in <em>Botryosphaeriaceae</em> and the purple circle represents leaves injected with 0.9%NaCl.</p></div></div>                            <h3 scrollto-destination=112483292 id="112483292" class="section-title" >3.10. Potential adaptations of the <em>L. theobromae</em> genome and transcriptome for plant cell wall degradation</h3>
<p>Phytopathogenic fungi encode many CAZymes. During colonization and infection, CAZymes play an important role in the disassembly of plant cell walls, including the breakdown, biosynthesis or modification of glycoconjugates and oligo- and polysaccharides.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> CAZymes are grouped into five classes (glycoside hydrolases, GH; glycosyl transferases, GT; polysaccharide lyases; carbohydrate esterases, CE and auxiliary activities, AA) and an associated module (carbohydrate-binding modules, CBMs).</p><p>To understand the potential genes involved in the adaptation of <em>L. theobromae</em> to different habitats and substrates, we identified the repertoire of CAZymes known to be involved in organic matter degradation. A total of 820 CAZymes (mapped to 763 predicted <em>L. theobromae</em> genes) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S18</a></span>) were identified (see <a reveal-id="dsx040-F2" data-open="dsx040-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). From the identified CAZymes, 10 (AA3, AA7, CBM18, CE1, CE10, GH109, GH3 GH43, GH78 and GT1) were expanded in the <em>L. theobromae</em> genome. Among the 283 genes up-regulated during infection, 66 are annotated as cell wall-degrading enzymes (CWDEs), involved in lignin degeneration (6 genes), cellulose degeneration (11 genes), hemicellulose degeneration (8 genes) and pectin degeneration (13 genes) (<a reveal-id="dsx040-F4" data-open="dsx040-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). GHs represented the largest super family involved in carbohydrate degradation processes.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> For a successful infection, pathogens must encode pectin degrading enzymes to break down plant pectin. The GH28 (polygalacturonases) family plays a critical role in pectin degradation in fungal pathogens such as <em>C. higginsianum, V. mali</em> and <em>V. pyri</em><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx040-B57" data-open="dsx040-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and has expanded in the genomes of the two <em>Valsa</em> species.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> When compared with the other fungal genomes, the genome of <em>L. theobromae</em> has the most GH28 members (14 genes, of which 4 were up-regulated during infection), suggesting that <em>L. theobromae</em> has a high capacity for pectin degradation.</p><p>Lignocellulose, which is composed of cellulose, hemicellulose and lignin, is one of the barriers that pathogens must surpass in order to colonize host tissues. Since <em>L. theobromae</em> is mainly a trunk pathogen, lignocellulose degrading enzymes likely play a major role during infection.<a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Indeed, GH subfamilies involved in the degradation of cellulose and hemicellulose were the most abundant among all the fungal genomes. The degradation of cellulose requires the collaboration of endoglucanases (GH6, GH9, GH12, GH44, GH45, GH51, GH74, GH124, CBM65 and CBM72), cellobiohydrolases (GH6, GH7, GH19, GH48 and GH74) and β-1,4-glucosidase (GH3). The <em>L. theobromae</em> genome encodes many of these CWDEs. Furthermore, among all the fungal pathogens studied, <em>L</em>. <em>theobromae</em> possesses the highest number of GH3s in its genome. Xylanases in the GH3 sub-family degrade xylan, and the <em>L. theobromae</em> genome contains a complete suite of enzymes to degrade xylan (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). During infection, three GH12, one GH7 and three GH3 genes encoding lignocellulose-degrading enzymes were up-regulated in <em>L. theobromae</em>, indicating their ability to effectively degrade cellulose, hemicellulose and lignin. Similar up-regulation of these genes were observed in <em>L. theobromae</em>.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Compared with <em>L. theobromae</em>, the GH12 family is expanded in <em>C. higginsianum</em>, and similar CAZyme modules are available in <em>L. theobromae</em>, <em>A. fumigatus</em> and <em>F. graminearum</em>.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B6" data-open="dsx040-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Among the fungi analysed so far, the <em>L. theobromae</em> genome encodes the highest number of xyloglucan transglucosylases (GH16, 24 genes), followed by <em>F. graminearum</em>, <em>C. higginsianum</em> and <em>C. graminicola</em>. Endo-β-1,4-cellulases (GH5) represent another gene family with a high number of encoded genes in <em>L. theobromae</em>, and these genes are commonly found in highly pathogenic species, such as <em>E.</em><em>lata</em> and <em>N. parvum</em>.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B6" data-open="dsx040-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Our analyses identified the highest number of GH5s in <em>C. higginsianum</em>, followed by <em>L. theobromae</em> and <em>C. graminicola</em>.</p><p>CBMs are an important gene family with carbohydrate binding activity.<a reveal-id="dsx040-B58" data-open="dsx040-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> According to our comparative analysis, the highest number of CBMs was found in the highly pathogenic <em>P. oryzae</em> (99 genes), followed by <em>C. graminicola</em> (70 genes), <em>C. higginsianum</em> (80 genes), <em>F. graminearum</em> (68 genes) and <em>L. theobromae</em> (54 genes). In contrast, all the non-pathogenic species, including <em>S. pombe</em> (8 genes) and <em>S. cerevisiae</em> (8 genes), have fewer reported CBMs in their genomes.</p><p>AAs are families of redox enzymes that act in conjunction with CAZymes to degrade lignocellulosic material. They represent ligninolytic enzymes and lytic polysaccharide mono-oxygenases involved in lignin breakdown.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B59" data-open="dsx040-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> Our analyses found that both AA3 (38 genes) and AA7 (50 genes) families were expanded in the <em>L. theobromae</em> genome. The AA3 family is related to cellobiose dehydrogenase activity, and AA7 oxidizes glycosyl residues.<a reveal-id="dsx040-B59" data-open="dsx040-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> These AA3 genes were also abundant in the <em>N. parvum</em> and <em>Stereum hirsutum</em> genomes.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, only one or two AA family genes were present in the non-pathogenic fungal genomes.</p>                            <h3 scrollto-destination=112483298 id="112483298" class="section-title" >3.11. <em>Lasiodiplodia theobromae</em> gene expression induced by infection and high temperature exhibits characteristics of opportunistic phytopathogens</h3>
<p>Transcriptome analysis conducted at two temperatures and at two time points after host inoculation identified a large number of differentially expressed genes. Differences in virulence at the different temperatures on grape shoots and growth on axenic cultures are shown in <a reveal-id="dsx040-F3" data-open="dsx040-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>. According to the transcriptomic data of samples collected from <em>L. theobromae</em> cultures on PDA plates, 361 genes were up-regulated, whereas 248 were down-regulated at the higher temperature. These up-regulated genes include genes with LysM (see <a reveal-id="dsx040-F3" data-open="dsx040-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>), allergens, GH 61 family genes, cytochrome P450s and heat shock proteins. Of these, few were related to primary pathogenicity mechanisms.</p><p>Phylogenetic analysis was conducted for randomly selected genes belong to glycosyl hydrolase family (<a reveal-id="dsx040-F6" data-open="dsx040-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>). According to the analysis, all of the genes clade together with related GH family genes identified from previous studies. <a reveal-id="dsx040-F6" data-open="dsx040-F6" class="link link-reveal link-table xref-fig">Figure 6B</a> demonstrates the gene expression of these genes. To investigate the roles of these up-regulated genes in the virulence of <em>Botryosphaeriaceae</em>, a GH 5 family putative cellulase gene (GH 7.188) and a GH 61 family putative lignocellulase gene (GH 2.1698) were selected for further study. These two genes were overexpressed using a strong PtrpC promoter. The overexpressing transformants showed no effect on vegetative growth and conidiation on PDA medium. We confirmed the elevated expression levels of these over-expressing transformants of GH 2.1698 and GH 7.188. Intriguingly, pathogenicity assays showed that both overexpressing transformants had significantly enhanced virulence on grape shoots compared with the wild-type strain CSS-01 (<a reveal-id="dsx040-F6" data-open="dsx040-F6" class="link link-reveal link-table xref-fig">Fig. 6C and D</a>). These results suggest that lignocellulose degrading enzymes play an important role during the colonization of <em>Botryosphaeriaceae</em> canker development. Furthermore, our results suggest that virulence in this family is temperature dependent, a typical characteristic of opportunistic infection. </p>                        <a id="112483301" scrollto-destination="112483301"></a>
<div id="dsx040-F6" data-id="dsx040-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f6.png?Expires=2147483647&Signature=Eeq9M75manHbDoblysFfPwhecK7DT0K6sYznsWuZiiOBUsuuYVc2jx8PVJljhfJuWHRePpuZvjswZ4m~jBESFMB2CEO3ZlH~-u6IYBTmY3T8OoBEjbmyl~H3f96JI6Dq2mhAfhQmbBqTzsOZ9rynePEd~U-d91KWVFWI4b~nFt~4htF9OnzKNCA8Dw7LyDsyuNiMLX9GzVV13FrjSrfbX1sBXmleK3Ci1lcOAx-tfD9-d44AaDo5qN-ieLmiPVFMFrlU01PbZYF90VABgFIZPWl9XbBYRttOgXJ1FCfnCSVwJDDMwLnyAJew9VUTdAnfCezSiTcHd9COvxK8WIqR6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogentic analyses, gene expression and pathogenicity assays conducted for glycosyl hydrolase family genes. (A) One of the 1000 most parsimonious trees obtained from a heuristic search of glycosyl hydrolase family genes and 36 related sequences from closely related species. Parsimony bootstrap support values ≥ 50% are indicated at the nodes and branches and the sequences from this study are given in bold. (B) Gene expression analysis of selected glycosyl hydrolase family genes. This includes the gene expression at different conditions (on axenic cultures and on grape shoots after inoculation at 25 °C and at 35 °C) and at different infection stages (8 and 12 hpi). (C) Lesions on stems were photographed at 3 days post-inoculation of the above L. theobromae transformants. WT indicates the wild-type strain of L. theobromae CSS-01s and numbers 1–10 represent the replicates of each overexpressing transformants conducted for each gene. First 10 stems were inoculated with the GH 2.1698 overexpressing transformants of L. theobromae, and the second 10 stems were inoculated with the GH 7.188 overexpressing transformants of L. theobromae. (D) Bar chart showing the mean lesion lengths of the overexpressing transformants. WT indicates the wild-type strain of L. theobromae CSS-01s. H2-OV1–H2-OV10 represents the mean lesion lengths of the GH 2.1698 overexpressing transformants of L. theobromae, and H7-OV1–H7-OV10 represents the GH 7.188 overexpressing transformants of L. theobromae." path-from-xml="dsx040f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483301/dsx040f6.tif" path-from-xml="dsx040f6.tif" target="_blank">View large</a><a section="112483301" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f6.png?Expires=2147483647&Signature=LIvxUxd7rZOn4YilJhENuoa6Z7rsGFfhuI0AWQnQt11SFpk5F7BNtdY95OA0xXKBDnEwNNYbXb4140Jq51BGXVfYT-oK1SfZgY0VH85z0jiBRlUg~nmWNSeDbhYt7n8Hk3BEpYEN1zO7AJR-cq3tVeuTNb6owKLBp2Eo4mPAuYUcISIRnl4zEst-aDgjSeLCyPOlvQxgBkp3E8kaSd-TfooJo2wfh3UUNrS6ER9RRKta658GYvYSohOa~0QZAa0HoQpVEzr6-vnuL0d-oem5cq0cOHLuDGOHp3wzuCKC~qFH~JHXq5uBd9Bj9Q-YXq59wPzatsdi8Zhxq7PC-sJHiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483301&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogentic analyses, gene expression and pathogenicity assays conducted for glycosyl hydrolase family genes. (A) One of the 1000 most parsimonious trees obtained from a heuristic search of glycosyl hydrolase family genes and 36 related sequences from closely related species. Parsimony bootstrap support values ≥ 50% are indicated at the nodes and branches and the sequences from this study are given in bold. (B) Gene expression analysis of selected glycosyl hydrolase family genes. This includes the gene expression at different conditions (on axenic cultures and on grape shoots after inoculation at 25 °C and at 35 °C) and at different infection stages (8 and 12 hpi). (C) Lesions on stems were photographed at 3 days post-inoculation of the above <em>L. theobromae</em> transformants. WT indicates the wild-type strain of <em>L. theobromae</em> CSS-01s and numbers 1–10 represent the replicates of each overexpressing transformants conducted for each gene. First 10 stems were inoculated with the GH 2.1698 overexpressing transformants of <em>L. theobromae</em>, and the second 10 stems were inoculated with the GH 7.188 overexpressing transformants of <em>L. theobromae</em>. (D) Bar chart showing the mean lesion lengths of the overexpressing transformants. WT indicates the wild-type strain of <em>L. theobromae</em> CSS-01s. H2-OV1–H2-OV10 represents the mean lesion lengths of the GH 2.1698 overexpressing transformants of <em>L. theobromae</em>, and H7-OV1–H7-OV10 represents the GH 7.188 overexpressing transformants of <em>L. theobromae</em>.</p></div></div><div content-id="dsx040-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/m_dsx040f6.png?Expires=2147483647&Signature=Eeq9M75manHbDoblysFfPwhecK7DT0K6sYznsWuZiiOBUsuuYVc2jx8PVJljhfJuWHRePpuZvjswZ4m~jBESFMB2CEO3ZlH~-u6IYBTmY3T8OoBEjbmyl~H3f96JI6Dq2mhAfhQmbBqTzsOZ9rynePEd~U-d91KWVFWI4b~nFt~4htF9OnzKNCA8Dw7LyDsyuNiMLX9GzVV13FrjSrfbX1sBXmleK3Ci1lcOAx-tfD9-d44AaDo5qN-ieLmiPVFMFrlU01PbZYF90VABgFIZPWl9XbBYRttOgXJ1FCfnCSVwJDDMwLnyAJew9VUTdAnfCezSiTcHd9COvxK8WIqR6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogentic analyses, gene expression and pathogenicity assays conducted for glycosyl hydrolase family genes. (A) One of the 1000 most parsimonious trees obtained from a heuristic search of glycosyl hydrolase family genes and 36 related sequences from closely related species. Parsimony bootstrap support values ≥ 50% are indicated at the nodes and branches and the sequences from this study are given in bold. (B) Gene expression analysis of selected glycosyl hydrolase family genes. This includes the gene expression at different conditions (on axenic cultures and on grape shoots after inoculation at 25 °C and at 35 °C) and at different infection stages (8 and 12 hpi). (C) Lesions on stems were photographed at 3 days post-inoculation of the above L. theobromae transformants. WT indicates the wild-type strain of L. theobromae CSS-01s and numbers 1–10 represent the replicates of each overexpressing transformants conducted for each gene. First 10 stems were inoculated with the GH 2.1698 overexpressing transformants of L. theobromae, and the second 10 stems were inoculated with the GH 7.188 overexpressing transformants of L. theobromae. (D) Bar chart showing the mean lesion lengths of the overexpressing transformants. WT indicates the wild-type strain of L. theobromae CSS-01s. H2-OV1–H2-OV10 represents the mean lesion lengths of the GH 2.1698 overexpressing transformants of L. theobromae, and H7-OV1–H7-OV10 represents the GH 7.188 overexpressing transformants of L. theobromae." path-from-xml="dsx040f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483301/dsx040f6.tif" path-from-xml="dsx040f6.tif" target="_blank">View large</a><a section="112483301" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/dsx040f6.png?Expires=2147483647&Signature=LIvxUxd7rZOn4YilJhENuoa6Z7rsGFfhuI0AWQnQt11SFpk5F7BNtdY95OA0xXKBDnEwNNYbXb4140Jq51BGXVfYT-oK1SfZgY0VH85z0jiBRlUg~nmWNSeDbhYt7n8Hk3BEpYEN1zO7AJR-cq3tVeuTNb6owKLBp2Eo4mPAuYUcISIRnl4zEst-aDgjSeLCyPOlvQxgBkp3E8kaSd-TfooJo2wfh3UUNrS6ER9RRKta658GYvYSohOa~0QZAa0HoQpVEzr6-vnuL0d-oem5cq0cOHLuDGOHp3wzuCKC~qFH~JHXq5uBd9Bj9Q-YXq59wPzatsdi8Zhxq7PC-sJHiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483301&ar=4282306&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx040f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogentic analyses, gene expression and pathogenicity assays conducted for glycosyl hydrolase family genes. (A) One of the 1000 most parsimonious trees obtained from a heuristic search of glycosyl hydrolase family genes and 36 related sequences from closely related species. Parsimony bootstrap support values ≥ 50% are indicated at the nodes and branches and the sequences from this study are given in bold. (B) Gene expression analysis of selected glycosyl hydrolase family genes. This includes the gene expression at different conditions (on axenic cultures and on grape shoots after inoculation at 25 °C and at 35 °C) and at different infection stages (8 and 12 hpi). (C) Lesions on stems were photographed at 3 days post-inoculation of the above <em>L. theobromae</em> transformants. WT indicates the wild-type strain of <em>L. theobromae</em> CSS-01s and numbers 1–10 represent the replicates of each overexpressing transformants conducted for each gene. First 10 stems were inoculated with the GH 2.1698 overexpressing transformants of <em>L. theobromae</em>, and the second 10 stems were inoculated with the GH 7.188 overexpressing transformants of <em>L. theobromae</em>. (D) Bar chart showing the mean lesion lengths of the overexpressing transformants. WT indicates the wild-type strain of <em>L. theobromae</em> CSS-01s. H2-OV1–H2-OV10 represents the mean lesion lengths of the GH 2.1698 overexpressing transformants of <em>L. theobromae</em>, and H7-OV1–H7-OV10 represents the GH 7.188 overexpressing transformants of <em>L. theobromae</em>.</p></div></div>                            <h2 scrollto-destination=112483302 id="112483302" class="section-title" >4. Discussion</h2>
<p>In this study, we developed fundamental genomic and transcriptomic resources for functional characterization of three botryosphaeriaceous pathogens, <em>B.</em><em>dothidea</em>, <em>L.</em><em>theobromae</em> and <em>N.</em><em>parvum</em>. These three species have similar genome sizes (42.1 Mb to 43.3 Mb), and the assemblies of <em>B. dothidea</em> and <em>N. parvum</em> were similar to those reported in previous studies. Indeed, all the sequenced genomes in <em>Botryosphaeriaceae</em> have similar sizes, ranging from 36.9 to 43.5 Mb, with similar genome coverage.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B11" data-open="dsx040-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx040-B14" data-open="dsx040-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> In addition to these botryosphaeriaceous taxa, the genome sizes for other pathogenic fungi compared here were similar to other common plant pathogens with similar lifestyles (<a reveal-id="dsx040-T3" data-open="dsx040-T3" class="link link-reveal link-table xref-fig">Table 3</a>).<a reveal-id="dsx040-B32" data-open="dsx040-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx040-B60" data-open="dsx040-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx040-B61" data-open="dsx040-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Both the JELLYFISH analysis,<a reveal-id="dsx040-B39" data-open="dsx040-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> which estimated genome size based on k-mer distributions,<a reveal-id="dsx040-B62" data-open="dsx040-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> and the tRNA analyses indicated a high degree of completeness in the assembled genomes. Furthermore, transcriptomic data obtained through RNASeq confirmed the accuracy and completeness of the assembled genome and facilitated the refinement of the predicted gene models of <em>L. theobromae</em>.<a reveal-id="dsx040-B63" data-open="dsx040-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><div content-id="dsx040-T3" class="table-modal"><div class="table-wrap"><div id="dsx040-T3" data-id="dsx040-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Comparison of genome sizes between <em>L. theobromae</em> and other pathogenic fungi</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pathogenic taxa </th><th>Genome size (Mb) </th></tr></thead><tbody><tr><td><em>Neofusicoccum parvum</em><sup><strong>26</strong></sup> </td><td>42.50 </td></tr><tr><td><em>Neofusicoccum parvum</em> (Current study) </td><td>42.60 </td></tr><tr><td><em>Botryosphaeria dothidea</em> (Current study) </td><td>43.10 </td></tr><tr><td><em>Botryosphaeria dothidea</em><sup><strong>30</strong></sup> </td><td>43.50 </td></tr><tr><td><em>Diplodia sapinea</em><sup><strong>30</strong></sup> </td><td>36.97 </td></tr><tr><td><em>Diplodia seriata</em><sup><strong>5</strong></sup> </td><td>37.10 </td></tr><tr><td><em>Lasiodiplodia theobromae</em> (Current study) </td><td>43.30 </td></tr><tr><td><em>Eutypa lata</em><sup><strong>25</strong></sup> </td><td>54.40 </td></tr><tr><td><em>Pyricularia oryzae</em><sup><strong>39</strong></sup> </td><td>40.30 </td></tr><tr><td><em>Diaporthe ampelina</em><sup><strong>5</strong></sup> </td><td>47.40 </td></tr><tr><td><em>Neurospora crassa</em><sup><strong>40</strong></sup> </td><td>39.90 </td></tr><tr><td><em>Ustilaginoidea virens</em><sup><strong>41</strong></sup> </td><td>39.40 </td></tr><tr><td><em>Valsa pyri</em><sup><strong>21</strong></sup> </td><td>35.74 </td></tr><tr><td><em>Valsa mali</em><sup><strong>21</strong></sup> </td><td>44.74 </td></tr></tbody></table></div><div content-id="dsx040-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Pathogenic taxa </th><th>Genome size (Mb) </th></tr></thead><tbody><tr><td><em>Neofusicoccum parvum</em><sup><strong>26</strong></sup> </td><td>42.50 </td></tr><tr><td><em>Neofusicoccum parvum</em> (Current study) </td><td>42.60 </td></tr><tr><td><em>Botryosphaeria dothidea</em> (Current study) </td><td>43.10 </td></tr><tr><td><em>Botryosphaeria dothidea</em><sup><strong>30</strong></sup> </td><td>43.50 </td></tr><tr><td><em>Diplodia sapinea</em><sup><strong>30</strong></sup> </td><td>36.97 </td></tr><tr><td><em>Diplodia seriata</em><sup><strong>5</strong></sup> </td><td>37.10 </td></tr><tr><td><em>Lasiodiplodia theobromae</em> (Current study) </td><td>43.30 </td></tr><tr><td><em>Eutypa lata</em><sup><strong>25</strong></sup> </td><td>54.40 </td></tr><tr><td><em>Pyricularia oryzae</em><sup><strong>39</strong></sup> </td><td>40.30 </td></tr><tr><td><em>Diaporthe ampelina</em><sup><strong>5</strong></sup> </td><td>47.40 </td></tr><tr><td><em>Neurospora crassa</em><sup><strong>40</strong></sup> </td><td>39.90 </td></tr><tr><td><em>Ustilaginoidea virens</em><sup><strong>41</strong></sup> </td><td>39.40 </td></tr><tr><td><em>Valsa pyri</em><sup><strong>21</strong></sup> </td><td>35.74 </td></tr><tr><td><em>Valsa mali</em><sup><strong>21</strong></sup> </td><td>44.74 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx040-T3" data-id="dsx040-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Comparison of genome sizes between <em>L. theobromae</em> and other pathogenic fungi</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pathogenic taxa </th><th>Genome size (Mb) </th></tr></thead><tbody><tr><td><em>Neofusicoccum parvum</em><sup><strong>26</strong></sup> </td><td>42.50 </td></tr><tr><td><em>Neofusicoccum parvum</em> (Current study) </td><td>42.60 </td></tr><tr><td><em>Botryosphaeria dothidea</em> (Current study) </td><td>43.10 </td></tr><tr><td><em>Botryosphaeria dothidea</em><sup><strong>30</strong></sup> </td><td>43.50 </td></tr><tr><td><em>Diplodia sapinea</em><sup><strong>30</strong></sup> </td><td>36.97 </td></tr><tr><td><em>Diplodia seriata</em><sup><strong>5</strong></sup> </td><td>37.10 </td></tr><tr><td><em>Lasiodiplodia theobromae</em> (Current study) </td><td>43.30 </td></tr><tr><td><em>Eutypa lata</em><sup><strong>25</strong></sup> </td><td>54.40 </td></tr><tr><td><em>Pyricularia oryzae</em><sup><strong>39</strong></sup> </td><td>40.30 </td></tr><tr><td><em>Diaporthe ampelina</em><sup><strong>5</strong></sup> </td><td>47.40 </td></tr><tr><td><em>Neurospora crassa</em><sup><strong>40</strong></sup> </td><td>39.90 </td></tr><tr><td><em>Ustilaginoidea virens</em><sup><strong>41</strong></sup> </td><td>39.40 </td></tr><tr><td><em>Valsa pyri</em><sup><strong>21</strong></sup> </td><td>35.74 </td></tr><tr><td><em>Valsa mali</em><sup><strong>21</strong></sup> </td><td>44.74 </td></tr></tbody></table></div><div content-id="dsx040-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Pathogenic taxa </th><th>Genome size (Mb) </th></tr></thead><tbody><tr><td><em>Neofusicoccum parvum</em><sup><strong>26</strong></sup> </td><td>42.50 </td></tr><tr><td><em>Neofusicoccum parvum</em> (Current study) </td><td>42.60 </td></tr><tr><td><em>Botryosphaeria dothidea</em> (Current study) </td><td>43.10 </td></tr><tr><td><em>Botryosphaeria dothidea</em><sup><strong>30</strong></sup> </td><td>43.50 </td></tr><tr><td><em>Diplodia sapinea</em><sup><strong>30</strong></sup> </td><td>36.97 </td></tr><tr><td><em>Diplodia seriata</em><sup><strong>5</strong></sup> </td><td>37.10 </td></tr><tr><td><em>Lasiodiplodia theobromae</em> (Current study) </td><td>43.30 </td></tr><tr><td><em>Eutypa lata</em><sup><strong>25</strong></sup> </td><td>54.40 </td></tr><tr><td><em>Pyricularia oryzae</em><sup><strong>39</strong></sup> </td><td>40.30 </td></tr><tr><td><em>Diaporthe ampelina</em><sup><strong>5</strong></sup> </td><td>47.40 </td></tr><tr><td><em>Neurospora crassa</em><sup><strong>40</strong></sup> </td><td>39.90 </td></tr><tr><td><em>Ustilaginoidea virens</em><sup><strong>41</strong></sup> </td><td>39.40 </td></tr><tr><td><em>Valsa pyri</em><sup><strong>21</strong></sup> </td><td>35.74 </td></tr><tr><td><em>Valsa mali</em><sup><strong>21</strong></sup> </td><td>44.74 </td></tr></tbody></table></div></div></p><p>Compared with the protein-coding genes predicted for other pathogens,<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B14" data-open="dsx040-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx040-B60" data-open="dsx040-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx040-B61" data-open="dsx040-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> our combined approach allowed for the prediction of the highest number of protein-coding genes in any fungal pathogen. Identifying repetitive elements is important for understanding the genome architecture of a pathogen and, ultimately, its evolution.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Repetitive elements present in the genomes of <em>Botryosphaeriaceae</em> taxa were much lower, ranging from 3.02% to 3.84%. This ratio is much lower than those reported for other pathogens such as <em>P. oryzae</em> (11.75%),<a reveal-id="dsx040-B64" data-open="dsx040-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><em>N. crassa</em> (12.72%),<a reveal-id="dsx040-B60" data-open="dsx040-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><em>Ustilago virens</em> (25%)<a reveal-id="dsx040-B61" data-open="dsx040-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> and <em>V. mali</em> (14.05%).<a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> This finding confirms the high genetic similarity observed among <em>Botryosphaeriaceae</em> species in previous studies.<a reveal-id="dsx040-B65" data-open="dsx040-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a></p><p>Comparative genome analysis revealed data on genetic variation, differential gene expression, and evolutionary dynamics between species and their specific adaptations to different environmental conditions and lifestyles. This study revealed that <em>Botryosphaeriaceae</em> species are phylogenetically closer to opportunistic pathogens, indicating that they may share traits common to opportunistic fungal pathogens. This was clearly evidenced from the phylogenetic study as well as from <a reveal-id="dsx040-F2" data-open="dsx040-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>, which shows that <em>Botryosphaeriaceae</em> fungi share the highest number of unique orthologous gene families with opportunistic fungal pathogens (1,049) than to pathogenic (221) and non-pathogenic fungi (36). It was further confirmed by the inventory of genes identified by comparative genome analysis, which are involved in the infection and the colonization of the host by <em>L. theobromae</em> and other <em>Botryosphaeriaceae</em> pathogens. Similar results were discovered in a recent transcriptomic study conducted for <em>L</em>. <em>theobromae</em>.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Functional annotation of these genomes provides additional evidence for the hypothesis that <em>Botryosphaeriaceae</em> fungi are more closely related to opportunistic fungi. According to the GO categories of the overrepresented genes, <em>Botryosphaeriaceae</em> species shared only three overrepresented genes with primary fungal pathogens (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>), whereas they shared 27 with opportunistic phytopathogens (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). The categories of genes shared with opportunistic pathogens include those related to cell wall degradation, carbohydrate and protein metabolism, gene expression regulation and the biosynthesis of secondary metabolites. Many of these genes have been shown to be involved in the pathogenicity of many fungal pathogens.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B14" data-open="dsx040-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx040-B61" data-open="dsx040-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Based on the phylogenetic analysis, the highest number of unique orthologous gene families shared with other opportunistic fungal pathogens, and the highest number of overrepresented gene categories shared with other opportunistic pathogens, we confirmed that <em>Botryosphaeriaceae</em> fungi are more closely related to opportunistic pathogens than to other pathogens. Gene categories shared between <em>Botryosphaeriaceae</em> species and other pathogenic fungi were also enriched in core elements essential for virulence in many pathogens. Phosphorylation is an essential post-translational modification that regulates a broad spectrum of biological processes through the activity of numerous protein kinases.<a reveal-id="dsx040-B66" data-open="dsx040-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Signal transduction is another important gene category for a pathogen to respond to environmental changes during host infection and regulate a network of cellular pathways.<a reveal-id="dsx040-B66" data-open="dsx040-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a>. All of these virulence-related genetic elements are commonly found in the genomes of fungal pathogens such as <em>P. oryzae</em>, <em>Fusarium oxysporum</em>,<a reveal-id="dsx040-B66" data-open="dsx040-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a><sup>,</sup><em>Botrytis cinera</em><a reveal-id="dsx040-B67" data-open="dsx040-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> and many other pathogenic fungi.<a reveal-id="dsx040-B68" data-open="dsx040-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a></p><p>During the comparative genome analysis, we observed significant expansions and contractions in many gene families. Such changes can provide adaptive advantages for the organisms at certain ecological niches during natural selection.<a reveal-id="dsx040-B69" data-open="dsx040-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> These expanded gene families include those responsible for virulence and nutrient uptake. In addition to the cellular transporters and cytochrome P450s, several gene families related to secondary metabolism have also undergone significant expansion. Among the 58 secondary metabolite clusters observed, NPRS gene clusters are responsible for the synthesis of toxic polypeptides, whereas PKS clusters are responsible for the production of naphthalenone pentaketides in botryosphaeriaceous fungi, <em>D. seriata</em>, <em>L. theobromae</em><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and <em>N. parvum</em>, as well as other pathogens such as <em>A. fumigatus</em>, <em>A. nidulans</em>, <em>D. ampelina</em>, <em>E. lata</em>, <em>Phaeomoniella chlamydospora</em> and <em>T.</em><em>minima</em>.<a reveal-id="dsx040-B9" data-open="dsx040-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Even though secondary metabolite gene clusters are not subjected to regulation during infection, their significant expansion in the genome suggests that they may be involved in producing disease symptoms and host adaptations. Cytochrome P450s are a super family of mono-oxygenases involved in the post-synthesis modification of diverse metabolites.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B9" data-open="dsx040-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx040-B46" data-open="dsx040-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsx040-B70" data-open="dsx040-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Furthermore, these proteins facilitate fungal adaptation to specific ecological niches by modifying potentially harmful environmental chemicals.<a reveal-id="dsx040-B70" data-open="dsx040-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Having a genome enriched in species-specific cytochrome P450s may help explain the broad geographic distribution of botryosphaeriaceous taxa. Expansion of this gene family observed in this study further confirms their high capacity for adapting to different hosts, which may explain the wide host range of <em>Botryosphaeriaceae</em> species. Because these genes are also related to several physiological traits, their expansion has likely facilitated the evolution of pathogenicity.<a reveal-id="dsx040-B71" data-open="dsx040-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a></p><p>Cellular transporters have significant roles in fungal pathogenesis as well as in protecting against plant defence compounds.<a reveal-id="dsx040-B72" data-open="dsx040-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Comparative genome analysis revealed that <em>L. theobromae</em> encode the highest number of cellular transporters among all the analysed genomes. Genes encoding (ABC)–type transport systems showed significant expansion in <em>L. theobromae</em>, and their expression is significantly up-regulated during infection. The importance of these multidrug facilitator transporters in drug resistance and pathogenesis has been confirmed in <em>B. cinera</em>, <em>Gibberella pulicaris, P. oryzae, Nectria haematococca and other pathogenic fungi.</em><a reveal-id="dsx040-B15" data-open="dsx040-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> They are also responsible for polysaccharide, lipid and amino acid transport. Similarly, the expanded and up-regulated transporter family genes identified in this study could also play an important role in the fungal pathogenesis of <em>L. theobromae</em> during infection by providing protection through the detoxification of antifungal phytotoxins. Sugar transporters also showed a high level of regulation during host infection in <em>L. theobromae</em>, allowing it to adapt to different host environments with varying pH conditions<a reveal-id="dsx040-B52" data-open="dsx040-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and various plant–pathogen interactions.<a reveal-id="dsx040-B52" data-open="dsx040-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsx040-B60" data-open="dsx040-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx040-B73" data-open="dsx040-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> These sugar symporters belong to the MFS family responsible for sporulation, cell-to-cell communication and pathogenesis in fungi.<a reveal-id="dsx040-B74" data-open="dsx040-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> Because these families are up-regulated during infection, both ABC-type transporters and MFS transporters are considered to be virulence or pathogenicity factors in <em>L</em>. <em>theobromae</em>.</p><p>Transcriptome profiling identified a set of putative effectors that were up-regulated during <em>L</em>. <em>theobromae</em> infection. These include conserved functional domains important for fungal pathogenesis mechanisms and other effector domains such as CFEM. Effectors enable pathogens to evade or suppress plant defence systems.<a reveal-id="dsx040-B56" data-open="dsx040-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><a reveal-id="dsx040-B75" data-open="dsx040-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> Genetic manipulation of fungal pathogens for candidate effector genes is difficult and cumbersome. This makes the whole process of effector functional analysis time consuming and laborious. Translocation of effector proteins to host using a type III secretion system (T3SS) of bacterial pathogens proven to be a fast alternative to the traditional method of generating fungal transformants. This efficient effector delivery (pEDV) system utilizes the N-terminal portion of AvrRps4, which is required for Type III secretion system (T3SS)-dependent delivery and <em>in planta</em> processing. In this study, we have experimentally shown that several randomly selected putative effectors can suppress <em>B.</em><em>glumae</em> triggered hypersensitivity in <em>N.</em><em>benthamiana</em>. These results are consistent with the hypothesis that these putative effectors play crucial roles in the pathogenesis of <em>L</em>. <em>theobromae</em> on its hosts by identifying PAMPs. This system has been successfully used in screening candidate non-bacterial effectors predicted in the genome <em>Hyaloperonospora arabidopsidis</em>,<a reveal-id="dsx040-B76" data-open="dsx040-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a><sup>,</sup><em>P. oryzae</em>,<a reveal-id="dsx040-B77" data-open="dsx040-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> and <em>Ustilaginoidea virens.</em><a reveal-id="dsx040-B64" data-open="dsx040-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a></p><p>In addition to the biotrophic stage, most phytopathogens also have hemi-biotrophic or necrotrophic stages, in which the fungi destroy the cell wall, enter the cell, and kill and destroy host cell structures.<a reveal-id="dsx040-B72" data-open="dsx040-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Proteins secreted by these pathogens play crucial roles in early colonization and pathogenesis by targeting all components of the primary and secondary plant cell walls.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The current study reveals a broad range of gene families with potential virulence functions in <em>L. theobromae</em>, similar to the botryosphaeriaceous taxa <em>D. seriata</em><a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a>, <em>L. theobromae</em><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and <em>N. parvum</em><a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a>. Previous studies have identified a broad range of key genes responsible for wood degradation and host colonization in phytopathogens.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B78" data-open="dsx040-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a><sup>,</sup><a reveal-id="dsx040-B79" data-open="dsx040-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> Similar to highly pathogenic <em>B. cinera</em>,<a reveal-id="dsx040-B80" data-open="dsx040-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><em>L.</em><em>theobromae</em> possesses the highest number of genes encoding pectolytic enzymes among the sequenced <em>Botryosphaeriaceae</em> genomes. The significant induction of these genes during infection is consistent with their critical roles in fungal pathogenesis. Because most of the dicotyledon cell walls consist of ∼35% pectin, these polygalacturonases facilitate the breakdown of cell wall.<a reveal-id="dsx040-B81" data-open="dsx040-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> This localized degradation of cell wall is required for the fungus to access the plant cytoplasm and to spread across host tissues. <a reveal-id="dsx040-B81" data-open="dsx040-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a><sup>,</sup><a reveal-id="dsx040-B82" data-open="dsx040-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> Their importance has been experimentally confirmed in many other species, including <em>Colletotrichum gloeosporioides</em>, <em>C. higginsianum</em>, <em>C. lindemuthianum</em>, <em>V. mali</em> and <em>V. pyri</em>.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B40" data-open="dsx040-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx040-B57" data-open="dsx040-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><a reveal-id="dsx040-B82" data-open="dsx040-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> In addition, the <em>L. theobromae</em> genome encodes a wider array of enzymes (GH5, GH3, GH16, GH43) targeting cellulose and hemicellulose, similar to other pathogens such as <em>D. seriata</em>, <em>E. lata</em>, <em>N. parvum</em> and <em>P. chlamydospora</em>.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B78" data-open="dsx040-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a><sup>,</sup><a reveal-id="dsx040-B80" data-open="dsx040-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><a reveal-id="dsx040-B82" data-open="dsx040-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> Their expression levels were significantly induced during infection, and the current study confirmed the enhanced pathogenicity of several putative lignocellulase genes by overexpressing them in <em>L. theobromae</em>. Together, the up-regulation of genes encoding CAZymes targeting pectin, cellulose, hemicellulose and xylan during infection explains the rapid colonization and infection of <em>L. theobromae</em> on woody plants.</p><p>Carbohydrate binding modules have been linked to the evolution of fungal pathogenicity in many fungi. During the infection of <em>L. theobromae</em>, CBM 18 family genes were significantly up-regulated. This subclass domain of chitin recognition has been confirmed for fungal protection from host defences by preventing plant factors from degrading fungal chitin.<a reveal-id="dsx040-B83" data-open="dsx040-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> Hence, during fungal infection, this family together with auxiliary enzymes participates in fungal pathogenesis. In this study, the AA3 and AA7 families showed significant expansion in the <em>L. theobromae</em> genome. AA7-domain families are responsible for oxidizing a variety of carbohydrates and contributing to lignin degradation,<a reveal-id="dsx040-B59" data-open="dsx040-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> whereas AA3-domain families participate in the degradation of cellulose, hemicellulose and lignin.<a reveal-id="dsx040-B45" data-open="dsx040-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Similar expansions of AA7s and AA3s have been found in many other fungal pathogens, such as <em>B. cinera</em>, <em>C. graminicola</em>, <em>C. higginsianum</em>, <em>D. seriata</em>, <em>E. lata</em>, <em>F. graminearum</em>, <em>F. solani</em>, <em>P. oryzae</em>, <em>N. parvum</em> and <em>S. hirsutum</em>.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B59" data-open="dsx040-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><a reveal-id="dsx040-B84" data-open="dsx040-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> In addition, this study showed the significantly induced expression of peroxidases and oxidoreductases in <em>L. theobromae</em> during infection. These two classes of enzymes are known to be involved in pathogenicity-related processes such as the degradation of lignin and the detoxification of reactive oxygen species produced by the host.<a reveal-id="dsx040-B3" data-open="dsx040-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B78" data-open="dsx040-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a></p><p>Our pathogenicity analysis revealed that a higher temperature increased the virulence of <em>L</em>. <em>theobromae</em> on grapevine. Consistent with this result, transcriptome analysis also revealed that several genes were up-regulated at a higher temperature. Furthermore, overexpression of two genes belonging to GH 5 and GH 61 families in <em>L. theobromae</em> significantly enhanced its virulence on <em>Vitis</em> shoots. The GH 5 and GH 61 families encode cellulases and lignocellulases, respectively. These genes are involved in cell wall degradation during infection and colonization. Transcriptome analysis revealed several genes corresponding to fungal pathogenicity, such as chitin-binding CBM family genes, cytochrome P450s and heat shock proteins, which were induced at high temperature. This increased virulence at a higher temperature is a typical characteristic of an opportunistic infection.<a reveal-id="dsx040-B4" data-open="dsx040-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Hence, we can confirm that <em>L. theobromae</em> can cause opportunistic infections at high temperatures. Transcriptomic study conducted on <em>L. theobromae</em> also confirmed the up-regulation of virulence related genes under heat stress conditions.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Recently few studies have been conducted to determine the influence of environmental conditions such as high temperature and high humidity, on the establishment of the pathogen and the development of numerous diseases on plants.<a reveal-id="dsx040-B5" data-open="dsx040-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx040-B85" data-open="dsx040-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a></p><p>In summary, this study revealed a high similarity between <em>L. theobromae</em> and other opportunistic fungal phytopathogens. We established genomic and transcriptomic resources for <em>L. theobromae</em> and incorporated the data into a comparative analysis to catalogue genes and gene families with putative virulence functions. <em>L.</em><em>theobromae</em> employs multiple virulence strategies to adapt to its diverse lifestyles, including CWDEs, secreted proteases, secondary metabolites, putative effectors and membrane transporters. Specific gene families of pectinases, secondary metabolism enzymes and transporters related to multidrug resistance are uniquely expanded in the <em>L</em>. <em>theobromae</em> genome and are frequently up-regulated during infection. In addition, large gene inventories were involved in cell wall degradation, nutrient uptake and secondary metabolism in <em>L. theobromae</em>. These results suggest potential adaptations for colonization and infection of the host and reflect their hemi-biotrophic lifestyle. Functional studies, together with transcriptomic data, indicate the active role of putative effectors and some lignocellulases in virulence, pathogenicity and host adaptation. Transcriptomic analysis revealed that several pathogenicity-related genes were induced at high temperature. This discovery, together with the phylogenomic analysis, indicates that <em>L.</em><em>theobromae</em> is an opportunistic phytopathogen. This study provides valuable genomic and transcriptomic resources for future studies of the evolution and pathogenesis of <em>L. theobromae</em>.</p>                            <h2 scrollto-destination=112483313 id="112483313" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://DNARES online" target="_blank">DNARES online</a>.</p>                            <h2 scrollto-destination=112483315 id="112483315" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The work is supported by CARS-30, NSFC (31501567), KJCX20140402 and Beijing Nova Program (Z141105001814047). We are grateful to Dr Ricardo Dias for comments on the manuscript and Dr Wenxian Sun (China Agricultural University, China) for providing the vectors for the study.</p>                            <h3 scrollto-destination=112483317 id="112483317" class="section-title" >Accession numbers</h3>
<p>MDYX00000000, SRP108461, SRP107819.</p>                            <h3 scrollto-destination=112483319 id="112483319" class="section-title" >Data availability</h3>
<p>The whole genome shotgun project for <em>Lasiodiplodia theobromae</em> CSS-01S has been deposited at DDBJ/ENA/GenBank under the accession code MDYX00000000. The version described in this paper is version MDYX01000000. Whole genome sequences of the other five isolates of <em>Lasiodiplodia theobromae</em> (HBQJZ-02S), <em>Botryosphaeria dothidea</em> (Sxg01s, LNXCEYGS-1), <em>Neofusicoccum parvum</em> (CDZ1M1s3, Gxg-01s) has been deposited at GenBank under the accession code SRP108461. All Total mRNA and Polysome-bound mRNA sequencing data generated in this study for <em>Lasiodiplodia theobromae</em> CSS-01S have been submitted to the NCBI Sequence Read Archive (SRA; <a class="link link-uri" href="http://www.ncbi.nlm. nih.gov/sra/" target="_blank">http://www.ncbi.nlm. nih.gov/sra/</a>) under accession number SRP107819.</p>                            <h3 scrollto-destination=112483321 id="112483321" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=112483323 id="112483323" class="section-title" >Author contributions</h3>
<p>J.Y.Y., W.S.Z., Z.C. and Q.K.X. contributed equally to this work as first authors. J.Y.Y., W.S.Z., Y.L.P. and X.H.L. managed the project and designed the experiment; J.Y.Y., Q.K.X., W.Z., and K.W.T.C. conducted the experiments; J.Y.Y., W.S.Z., Z.C., M.F.X., K.D.H., X.H.L. and J.P.X. analysed the data; Z.C. and K.W.T.C. wrote the paper. J.Y.Y., Z.C., K.W.T.C., K.D.H., J.P.X. and A.J.L.P. revised the paper. All of the other authors contributed by assisting in molecular experiments and pathogenicity tests.</p>                            <h2 scrollto-destination=112483325 id="112483325" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx040-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Phillips</div> <div class="given-names">A. J. L.</div></div>, <div class="name string-name"><div class="surname">Alves</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Abdollahzadeh</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The <em>Botryosphaeriaceae</em>: genera and species known from culture,</div> <div class="source">Stud. Mycol</div> ., <div class="volume">76</div>, <div class="fpage">51</div>–<div class="lpage">167</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Botryosphaeriaceae%3A%20genera%20and%20species%20known%20from%20culture%2C&amp;author=A.%20J.%20L.%20Phillips&amp;author=A.%20Alves&amp;author=J.%20Abdollahzadeh&amp;publication_year=2013&amp;journal=Stud.%20Mycol&amp;volume=76&amp;pages=51-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3114/sim0021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3114%2Fsim0021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3114%2Fsim0021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24302790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Species of <em>Botryosphaeriaceae</em> involved in grapevine dieback in China,</div> <div class="source">Fungal Divers</div> ., <div class="volume">61</div>, <div class="fpage">221</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20of%20Botryosphaeriaceae%20involved%20in%20grapevine%20dieback%20in%20China%2C&amp;author=J.%20Yan&amp;author=Y.%20Xie&amp;author=W.%20Zhang&amp;publication_year=2013&amp;journal=Fungal%20Divers&amp;volume=61&amp;pages=221-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13225-013-0251-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13225-013-0251-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13225-013-0251-8"> </span></div> </div></div></div></div></div><div content-id="dsx040-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morales-Cruz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Amrine</div> <div class="given-names">K. C. H.</div></div>, <div class="name string-name"><div class="surname">Blanco-Ulate</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Distinctive expansion of gene families associated with plant cell wall degradation, secondary metabolism, and nutrient uptake in the genomes of grapevine trunk pathogens,</div> <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">1</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinctive%20expansion%20of%20gene%20families%20associated%20with%20plant%20cell%20wall%20degradation%2C%20secondary%20metabolism%2C%20and%20nutrient%20uptake%20in%20the%20genomes%20of%20grapevine%20trunk%20pathogens%2C&amp;author=A.%20Morales-Cruz&amp;author=K.%20C.%20H.%20Amrine&amp;author=B.%20Blanco-Ulate&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1624-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1624-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1624-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25553907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chethana</div> <div class="given-names">K. W. T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X. H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Hyde</div> <div class="given-names">K. D.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">J. Y.</div></div></div> <div class="year">2016</div>. 
<div class="article-title">Trail decryption of molecular research on <em>Botryosphaeriaceae</em> in woody plants</div>, <div class="source">Phytopathol. Mediterr</div> ., <div class="volume">55</div>, <div class="fpage">147</div>–<div class="lpage">71</div>.</div></div></div></div><div content-id="dsx040-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paolinelli-Alfonso</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Villalobos-Escobedo</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Rolshausen</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Global transcriptional analysis suggests <em>Lasiodiplodia theobromae</em> pathogenicity factors involved in modulation of grapevine defensive response,</div> <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">615</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptional%20analysis%20suggests%20Lasiodiplodia%20theobromae%20pathogenicity%20factors%20involved%20in%20modulation%20of%20grapevine%20defensive%20response%2C&amp;author=M.%20Paolinelli-Alfonso&amp;author=J.%20M.%20Villalobos-Escobedo&amp;author=P.%20Rolshausen&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=615" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-2952-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-2952-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-2952-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27514986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramírez-Suero</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chong</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Farine</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Effect of <em>Neofusicoccum parvum</em> and <em>Diplodia seriata</em> extra-cellular compounds on defence gene expression in <em>Vitis vinifera</em> cv. Chardonnay and cv. Gewurztraminer,</div> <div class="source">Phytopathol. Mediterr</div> ., <div class="volume">51</div>, <div class="fpage">410</div>–<div class="lpage">52</div>.</div></div></div></div><div content-id="dsx040-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fernandes</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Alves</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Correia</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Devreese</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Esteves</div> <div class="given-names">A. C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Secretome analysis identifies potential virulence factors of <em>Diplodia corticola</em>, a fungal pathogen involved in cork oak (<em>Quercus suber</em>) decline,</div> <div class="source">Fungal Biol</div> ., <div class="volume">118</div>, <div class="fpage">516</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Secretome%20analysis%20identifies%20potential%20virulence%20factors%20of%20Diplodia%20corticola%2C%20a%20fungal%20pathogen%20involved%20in%20cork%20oak%20(Quercus%20suber)%20decline%2C&amp;author=I.%20Fernandes&amp;author=A.%20Alves&amp;author=A.%20Correia&amp;author=B.%20Devreese&amp;author=A.%20C.%20Esteves&amp;publication_year=2014&amp;journal=Fungal%20Biol&amp;volume=118&amp;pages=516-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.funbio.2014.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.funbio.2014.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.funbio.2014.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24863480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abou-Mansour</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Débieux</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Ramírez-Suero</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Phytotoxic metabolites from <em>Neofusicoccum parvum</em>, a pathogen of Botryosphaeria dieback of grapevine,</div> <div class="source">Phytochemistry</div> , <div class="volume">115</div>, <div class="fpage">207</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytotoxic%20metabolites%20from%20Neofusicoccum%20parvum%2C%20a%20pathogen%20of%20Botryosphaeria%20dieback%20of%20grapevine%2C&amp;author=E.%20Abou-Mansour&amp;author=J.%20L.%20D%C3%A9bieux&amp;author=M.%20Ram%C3%ADrez-Suero&amp;publication_year=2015&amp;journal=Phytochemistry&amp;volume=115&amp;pages=207-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.phytochem.2015.01.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.phytochem.2015.01.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.phytochem.2015.01.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25747381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andolfi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mugnai</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Luque</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Surico</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Cimmino</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Evidente</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Phytotoxins produced by fungi associated with grapevine trunk diseases,</div> <div class="source">Toxins</div> , <div class="volume">3</div>, <div class="fpage">1569</div>–<div class="lpage">605</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytotoxins%20produced%20by%20fungi%20associated%20with%20grapevine%20trunk%20diseases%2C&amp;author=A.%20Andolfi&amp;author=L.%20Mugnai&amp;author=J.%20Luque&amp;author=G.%20Surico&amp;author=A.%20Cimmino&amp;author=A.%20Evidente&amp;publication_year=2011&amp;journal=Toxins&amp;volume=3&amp;pages=1569-605" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/toxins3121569" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Ftoxins3121569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Ftoxins3121569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22295177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bertsch</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ramírez‐Suero</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Magnin‐Robert</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Grapevine trunk diseases: complex and still poorly understood</div>, <div class="source">Plant Pathol</div> ., <div class="volume">62</div>, <div class="fpage">243</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Grapevine%20trunk%20diseases%3A%20complex%20and%20still%20poorly%20understood&amp;author=C.%20Bertsch&amp;author=M.%20Ram%C3%ADrez%E2%80%90Suero&amp;author=M.%20Magnin%E2%80%90Robert&amp;publication_year=2013&amp;journal=Plant%20Pathol&amp;volume=62&amp;pages=243-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppa.2013.62.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppa.2013.62.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppa.2013.62.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsx040-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blanco-Ulate</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Rolshausen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cantu</div> <div class="given-names">D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence of <em>Neofusicoccum parvum</em> isolate UCR-NP2, a fungal vascular pathogen associated with grapevine cankers,</div> <div class="source">Genome Announc</div> ., <div class="volume">1</div>, <div class="fpage">e00339</div>–<div class="lpage">132013</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Neofusicoccum%20parvum%20isolate%20UCR-NP2%2C%20a%20fungal%20vascular%20pathogen%20associated%20with%20grapevine%20cankers%2C&amp;author=B.%20Blanco-Ulate&amp;author=P.%20Rolshausen&amp;author=D.%20Cantu&amp;publication_year=2013&amp;journal=Genome%20Announc&amp;volume=1&amp;pages=e00339-132013" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23766404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parrent</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">James</div> <div class="given-names">T. Y.</div></div>, <div class="name string-name"><div class="surname">Vasaitis</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">A. F.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Friend or foe? Evolutionary history of glycoside hydrolase family 32 genes encoding for sucrolytic activity in fungi and its implications for plant-fungal symbioses,</div> <div class="source">BMC Evol. Biol</div> ., <div class="volume">9</div>, <div class="fpage">1</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Friend%20or%20foe%3F%20Evolutionary%20history%20of%20glycoside%20hydrolase%20family%2032%20genes%20encoding%20for%20sucrolytic%20activity%20in%20fungi%20and%20its%20implications%20for%20plant-fungal%20symbioses%2C&amp;author=J.%20L.%20Parrent&amp;author=T.%20Y.%20James&amp;author=R.%20Vasaitis&amp;author=A.%20F.%20Taylor&amp;publication_year=2009&amp;journal=BMC%20Evol.%20Biol&amp;volume=9&amp;pages=1-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-9-148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-9-148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-9-148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">A <em>Phytophthora sojae</em> glycoside hydrolase 12 protein is a major virulence factor during soybean infection and is recognized as a PAMP</div>, <div class="source">Plant Cell</div> , <div class="volume">27</div>, <div class="fpage">2057</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Phytophthora%20sojae%20glycoside%20hydrolase%2012%20protein%20is%20a%20major%20virulence%20factor%20during%20soybean%20infection%20and%20is%20recognized%20as%20a%20PAMP&amp;author=Z.%20Ma&amp;author=T.%20Song&amp;author=L.%20Zhu&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=2057-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.15.00390" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.15.00390" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.15.00390"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26163574" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van der Nest</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Bihon</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">De Vos</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Draft genome sequences of <em>Diplodia sapinea</em>, <em>Ceratocystis manginecans</em>, and <em>Ceratocystis moniliformis</em>,</div> <div class="source">IMA Fungus</div> , <div class="volume">5</div>, <div class="fpage">135</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequences%20of%20Diplodia%20sapinea%2C%20Ceratocystis%20manginecans%2C%20and%20Ceratocystis%20moniliformis%2C&amp;author=M.%20A.%20van%20der%20Nest&amp;author=W.%20Bihon&amp;author=L.%20De%20Vos&amp;publication_year=2014&amp;journal=IMA%20Fungus&amp;volume=5&amp;pages=135-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5598/imafungus.2014.05.01.13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5598%2Fimafungus.2014.05.01.13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5598%2Fimafungus.2014.05.01.13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25083413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Benny</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Kistler</div> <div class="given-names">H.C.</div></div>, <div class="name string-name"><div class="surname">VanEtten</div> <div class="given-names">H. D.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genes determining pathogenicity to pea are clustered on a supernumerary chromosome in the fungal plant pathogen <em>Nectria haematococca</em></div>, <div class="source">Plant J</div> ., <div class="volume">25</div>, <div class="fpage">305</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20determining%20pathogenicity%20to%20pea%20are%20clustered%20on%20a%20supernumerary%20chromosome%20in%20the%20fungal%20plant%20pathogen%20Nectria%20haematococca&amp;author=Y.%20Han&amp;author=X.%20Liu&amp;author=U.%20Benny&amp;author=H.C.%20Kistler&amp;author=H.%20D.%20VanEtten&amp;publication_year=2014&amp;journal=Plant%20J&amp;volume=25&amp;pages=305-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.2001.00969.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.2001.00969.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.2001.00969.x"> </span></div> </div></div></div></div></div><div content-id="dsx040-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Sweeney</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Lo Leggio</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Insights into the oxidative degradation of cellulose by a copper metalloenzyme that exploits biomass components,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">15079</div>–<div class="lpage">084</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20oxidative%20degradation%20of%20cellulose%20by%20a%20copper%20metalloenzyme%20that%20exploits%20biomass%20components%2C&amp;author=R.%20J.%20Quinlan&amp;author=M.%20D.%20Sweeney&amp;author=L.%20Lo%20Leggio&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=15079-084" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1105776108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1105776108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1105776108"> </span></div> </div></div></div></div></div><div content-id="dsx040-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">McHardy</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">Schulze-Lefert</div> <div class="given-names">P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Microbial genome-enabled insights into plant-microorganism interactions,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">5</div>, <div class="fpage">797</div>–<div class="lpage">813</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20genome-enabled%20insights%20into%20plant-microorganism%20interactions%2C&amp;author=D.%20Guttman&amp;author=A.%20C.%20McHardy&amp;author=P.%20Schulze-Lefert&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=5&amp;pages=797-813" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3748"> </span></div> </div></div></div></div></div><div content-id="dsx040-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harris</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Stone</div> <div class="given-names">B. A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Chemistry and molecular organization of plant cell walls</div>, <div class="source">Water Res</div> ., <div class="volume">45</div>, <div class="fpage">1587</div>–<div class="lpage">96</div>.</div></div></div></div><div content-id="dsx040-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Molina</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">An <em>Ustilago maydis</em> gene involved in H<sub>2</sub>O<sub>2</sub> detoxification is required for virulence,</div> <div class="source">Plant Cell</div> , <div class="volume">19</div>, <div class="fpage">2293</div>–<div class="lpage">309</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20Ustilago%20maydis%20gene%20involved%20in%20H2O2%20detoxification%20is%20required%20for%20virulence%2C&amp;author=L.%20Molina&amp;author=R.%20Kahmann&amp;publication_year=2007&amp;journal=Plant%20Cell&amp;volume=19&amp;pages=2293-309" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.107.052332" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.107.052332" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.107.052332"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17616735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Talbot</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Ebbole</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Hamer</div> <div class="given-names">J. E.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Karyotype variation within clonal lineages of the rice blast fungus <em>Magnaporthe grisea</em></div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">59</div>, <div class="fpage">585</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Karyotype%20variation%20within%20clonal%20lineages%20of%20the%20rice%20blast%20fungus%20Magnaporthe%20grisea&amp;author=N.%20J.%20Talbot&amp;author=D.%20J.%20Ebbole&amp;author=J.%20E.%20Hamer&amp;publication_year=1993&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=59&amp;pages=585-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16348876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Price</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N. C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P. A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">De novo identification of repeat families in large genomes,</div> <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes%2C&amp;author=A.%20L.%20Price&amp;author=N.%20C.%20Jones&amp;author=P.%20A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div> </div></div></div></div></div><div content-id="dsx040-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kojima</div> <div class="given-names">K. K.</div></div>, <div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Repbase Update, a database of repetitive elements in eukaryotic genomes,</div> <div class="source">Mob. DNA</div> , <div class="volume">6</div>. <div class="fpage">1</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20repetitive%20elements%20in%20eukaryotic%20genomes%2C&amp;author=W.%20Bao&amp;author=K.%20K.%20Kojima&amp;author=O.%20Kohany&amp;publication_year=2015&amp;journal=Mob.%20DNA&amp;volume=6&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13100-015-0041-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13100-015-0041-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13100-015-0041-9"> </span></div> </div></div></div></div></div><div content-id="dsx040-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Automated de novo identification of repeat sequence families in sequenced genomes,</div> <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">1269</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes%2C&amp;author=Z.%20Bao&amp;author=S.%20R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">W465</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20a%20web%20server%20for%20gene%20prediction%20in%20eukaryotes%20that%20allows%20user-defined%20constraints%2C&amp;author=M.%20Stanke&amp;author=B.%20Morgenstern&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W465-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki458" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki458" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki458"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ter-Hovhannisyan</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Chernoff</div> <div class="given-names">Y. O.</div></div>, <div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Gene identification in novel eukaryotic genomes by self-training algorithm,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">6494</div>–<div class="lpage">506</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20identification%20in%20novel%20eukaryotic%20genomes%20by%20self-training%20algorithm%2C&amp;author=A.%20Lomsadze&amp;author=V.%20Ter-Hovhannisyan&amp;author=Y.%20O.%20Chernoff&amp;author=M.%20Borodovsky&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=6494-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki937" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki937" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki937"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16314312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">She</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chu</div> <div class="given-names">J. S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Pei</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">N.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">GenBlastA: enabling BLAST to identify homologous gene sequences,</div> <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">143</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenBlastA%3A%20enabling%20BLAST%20to%20identify%20homologous%20gene%20sequences%2C&amp;author=R.%20She&amp;author=J.%20S.%20Chu&amp;author=K.%20Wang&amp;author=J.%20Pei&amp;author=N.%20Chen&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=143-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.082081.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.082081.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.082081.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18838612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Clamp</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">GeneWise and genomewise,</div> <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">988</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneWise%20and%20genomewise%2C&amp;author=E.%20Birney&amp;author=M.%20Clamp&amp;author=R.%20Durbin&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=988-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1865504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1865504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1865504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Petersen</div> <div class="given-names">T. N.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions,</div> <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">785</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SignalP%204.0%3A%20discriminating%20signal%20peptides%20from%20transmembrane%20regions%2C&amp;author=T.%20N.%20Petersen&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=785-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21959131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B. A.</div></div>, <div class="name string-name"><div class="surname">McCue</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Schaeffer</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wold</div> <div class="given-names">B.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq,</div> <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">621</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq%2C&amp;author=A.%20Mortazavi&amp;author=B.%20A.%20Williams&amp;author=K.%20McCue&amp;author=L.%20Schaeffer&amp;author=B.%20Wold&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=621-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using evidencemodeler and the program to assemble spliced alignments,</div> <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20evidencemodeler%20and%20the%20program%20to%20assemble%20spliced%20alignments%2C&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Okuda</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yoshizawa</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">KAAS: an automatic genome annotation and pathway reconstruction server,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W182</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KAAS%3A%20an%20automatic%20genome%20annotation%20and%20pathway%20reconstruction%20server%2C&amp;author=Y.%20Moriya&amp;author=M.%20Itoh&amp;author=S.%20Okuda&amp;author=A.%20C.%20Yoshizawa&amp;author=M.%20Kanehisa&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W182-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dean</div> <div class="given-names">R. A.</div></div>, <div class="name string-name"><div class="surname">Talbot</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Ebbole</div> <div class="given-names">D. J.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">The genome sequence of the rice blast fungus <em>Magnaporthe grisea</em>,</div> <div class="source">Nature</div> , <div class="volume">434</div>, <div class="fpage">980</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20rice%20blast%20fungus%20Magnaporthe%20grisea%2C&amp;author=R.%20A.%20Dean&amp;author=N.%20J.%20Talbot&amp;author=D.%20J.%20Ebbole&amp;publication_year=2005&amp;journal=Nature&amp;volume=434&amp;pages=980-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15846337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Islam</div> <div class="given-names">Md. S.</div></div>, <div class="name string-name"><div class="surname">Haque</div> <div class="given-names">Md. S.</div></div>, <div class="name string-name"><div class="surname">Islam</div> <div class="given-names">M. M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Tools to kill: genome of one of the most destructive plant pathogenic fungi <em>Macrophomina phaseolina</em>,</div> <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">1</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tools%20to%20kill%3A%20genome%20of%20one%20of%20the%20most%20destructive%20plant%20pathogenic%20fungi%20Macrophomina%20phaseolina%2C&amp;author=Md.%20S.%20Islam&amp;author=Md.%20S.%20Haque&amp;author=M.%20M.%20Islam&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-493" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-493" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-493"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saier</div> <div class="given-names">M. H.</div>Jr.</div>, <div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">V. S.</div></div>, <div class="name string-name"><div class="surname">Tamang</div> <div class="given-names">D. G.</div></div>, <div class="name string-name"><div class="surname">Vastermark</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The transporter classification database,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">251</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transporter%20classification%20database%2C&amp;author=M.%20H.%20Saier&amp;author=V.%20S.%20Reddy&amp;author=D.%20G.%20Tamang&amp;author=A.%20Vastermark&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=251-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1097"> </span></div> </div></div></div></div></div><div content-id="dsx040-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Misawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">3059</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform%2C&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">1870</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA7%3A%20molecular%20evolutionary%20genetics%20analysis%20version%207.0%20for%20bigger%20datasets%2C&amp;author=S.%20Kumar&amp;author=G.%20Stecher&amp;author=K.%20Tamura&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=1870-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Swofford</div> <div class="given-names">D. L.</div></div></div> <div class="year">2002</div>, <div class="source">PAUP* 4.0: phylogenetic analysis using parsimony (* and other methods)</div> . 
<div class="publisher-name">Sinauer Associates</div>, 
<div class="publisher-loc">Sunderland</div>.</div></div></div></div><div content-id="dsx040-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood%2C&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers%2C&amp;author=G.%20Marcais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Galagan</div> <div class="given-names">J. E.</div></div>, <div class="name string-name"><div class="surname">Calvo</div> <div class="given-names">S. E.</div></div>, <div class="name string-name"><div class="surname">Borkovich</div> <div class="given-names">K. A.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">The genome sequence of the filamentous fungus <em>Neurospora crassa</em>,</div> <div class="source">Nature</div> , <div class="volume">422</div>, <div class="fpage">859</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20filamentous%20fungus%20Neurospora%20crassa%2C&amp;author=J.%20E.%20Galagan&amp;author=S.%20E.%20Calvo&amp;author=K.%20A.%20Borkovich&amp;publication_year=2003&amp;journal=Nature&amp;volume=422&amp;pages=859-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01554" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01554" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01554"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12712197" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Spanu</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Abbott</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Amselem</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Genome expansion and gene loss in powdery mildew fungi reveal tradeoffs in extreme parasitism</div>, <div class="source">Science</div> , <div class="volume">330</div>, <div class="fpage">1543</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20expansion%20and%20gene%20loss%20in%20powdery%20mildew%20fungi%20reveal%20tradeoffs%20in%20extreme%20parasitism&amp;author=P.%20D.%20Spanu&amp;author=J.%20C.%20Abbott&amp;author=J.%20Amselem&amp;publication_year=2010&amp;journal=Science&amp;volume=330&amp;pages=1543-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1194573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1194573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1194573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21148392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kosiol</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Anisimova</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Selection on the protein-coding genome,</div> <div class="source">Methods Mol. Biol</div> ., <div class="volume">856</div>, <div class="fpage">113</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20the%20protein-coding%20genome%2C&amp;author=C.%20Kosiol&amp;author=M.%20Anisimova&amp;publication_year=2012&amp;journal=Methods%20Mol.%20Biol&amp;volume=856&amp;pages=113-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-1-61779-585-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-61779-585-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-1-61779-585-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22399457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Demuth</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M. W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The life and death of gene families,</div> <div class="source">Bioessays</div> , <div class="volume">31</div>, <div class="fpage">29</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20life%20and%20death%20of%20gene%20families%2C&amp;author=J.%20P.%20Demuth&amp;author=M.%20W.%20Hahn&amp;publication_year=2012&amp;journal=Bioessays&amp;volume=31&amp;pages=29-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bies.v31:1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbies.v31:1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbies.v31:1"> </span></div> </div></div></div></div></div><div content-id="dsx040-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Powell</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Conant</div> <div class="given-names">G. C.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Carbone</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">R. A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Altered patterns of gene duplication and differential gene gain and loss in fungal pathogens,</div> <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">95</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20patterns%20of%20gene%20duplication%20and%20differential%20gene%20gain%20and%20loss%20in%20fungal%20pathogens%2C&amp;author=A.%20J.%20Powell&amp;author=G.%20C.%20Conant&amp;author=D.%20E.%20Brown&amp;author=I.%20Carbone&amp;author=R.%20A.%20Dean&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=95-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18298839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Bie</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cristianini</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Demuth</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M. W.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">CAFE: a computational tool for the study of gene family evolution,</div> <div class="source">Bioinformatics</div> , <div class="volume">22</div>, <div class="fpage">1269</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAFE%3A%20a%20computational%20tool%20for%20the%20study%20of%20gene%20family%20evolution%2C&amp;author=T.%20De%20Bie&amp;author=N.%20Cristianini&amp;author=J.%20P.%20Demuth&amp;author=M.%20W.%20Hahn&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1269-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16543274" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brakhage</div> <div class="given-names">A. A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Regulation of fungal secondary metabolism,</div> <div class="source">Nat. Rev. Microbiol</div> ., <div class="volume">11</div>, <div class="fpage">21</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20fungal%20secondary%20metabolism%2C&amp;author=A.%20A.%20Brakhage&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Microbiol&amp;volume=11&amp;pages=21-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2916" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2916" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2916"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23178386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Klitgaard</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Accurate prediction of secondary metabolite gene clusters in filamentous fungi,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">E99</div>–<div class="lpage">107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20prediction%20of%20secondary%20metabolite%20gene%20clusters%20in%20filamentous%20fungi%2C&amp;author=M.%20R.%20Anderson&amp;author=J.%20B.%20Nielsen&amp;author=A.%20Klitgaard&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=E99-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1205532110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1205532110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1205532110"> </span></div> </div></div></div></div></div><div content-id="dsx040-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khaldi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Seifuddin</div> <div class="given-names">F. T.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">SMURF: genomic mapping of fungal secondary metabolite clusters,</div> <div class="source">Fungal Genet. Biol</div> ., <div class="volume">47</div>, <div class="fpage">736</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMURF%3A%20genomic%20mapping%20of%20fungal%20secondary%20metabolite%20clusters%2C&amp;author=N.%20Khaldi&amp;author=F.%20T.%20Seifuddin&amp;author=G.%20Turner&amp;publication_year=2010&amp;journal=Fungal%20Genet.%20Biol&amp;volume=47&amp;pages=736-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20554054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Medema</div> <div class="given-names">M. H.</div></div>, <div class="name string-name"><div class="surname">Blin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Cimermancic</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W339</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=antiSMASH%3A%20rapid%20identification%2C%20annotation%20and%20analysis%20of%20secondary%20metabolite%20biosynthesis%20gene%20clusters%20in%20bacterial%20and%20fungal%20genome%20sequences%2C&amp;author=M.%20H.%20Medema&amp;author=K.%20Blin&amp;author=P.%20Cimermancic&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W339-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beseli</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Amnuaykanjanasin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Herrero</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Daub</div> <div class="given-names">M. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Membrane transporters in self resistance of <em>Cercospora nicotianae</em> to the photo-activated toxin cercosporin,</div> <div class="source">Curr. Genet</div> ., <div class="volume">61</div>, <div class="fpage">601</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Membrane%20transporters%20in%20self%20resistance%20of%20Cercospora%20nicotianae%20to%20the%20photo-activated%20toxin%20cercosporin%2C&amp;author=A.%20Beseli&amp;author=A.%20Amnuaykanjanasin&amp;author=S.%20Herrero&amp;author=E.%20Thomas&amp;author=M.%20E.%20Daub&amp;publication_year=2015&amp;journal=Curr.%20Genet&amp;volume=61&amp;pages=601-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00294-015-0486-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00294-015-0486-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00294-015-0486-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25862648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vankuyk</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Diderich</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">MacCabe</div> <div class="given-names">A. P.</div></div>, <div class="name string-name"><div class="surname">Hererro</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ruijter</div> <div class="given-names">G. J.</div></div>, <div class="name string-name"><div class="surname">Visser</div> <div class="given-names">J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title"><em>Aspergillus niger</em> mstA encodes a high-affinity sugar/H+ symporter which is regulated in response to extracellular pH,</div> <div class="source">Biochem J</div> ., <div class="volume">379</div>, <div class="fpage">375</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aspergillus%20niger%20mstA%20encodes%20a%20high-affinity%20sugar%2FH%2B%20symporter%20which%20is%20regulated%20in%20response%20to%20extracellular%20pH%2C&amp;author=P.%20A.%20Vankuyk&amp;author=J.%20A.%20Diderich&amp;author=A.%20P.%20MacCabe&amp;author=O.%20Hererro&amp;author=G.%20J.%20Ruijter&amp;author=J.%20Visser&amp;publication_year=2004&amp;journal=Biochem%20J&amp;volume=379&amp;pages=375-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj20030624" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj20030624" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj20030624"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14717659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title"><em>CDR4 is the major contributor to azole resistance among four Pdr5p-like ABC transporters in Neurospora crassa</em>,</div> <div class="source">Fungal Biol</div> ., <div class="volume">116</div>, <div class="fpage">848</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDR4%20is%20the%20major%20contributor%20to%20azole%20resistance%20among%20four%20Pdr5p-like%20ABC%20transporters%20in%20Neurospora%20crassa%2C&amp;author=Y.%20Zhang&amp;author=Z.%20Zhang&amp;author=X.%20Zhang&amp;publication_year=2012&amp;journal=Fungal%20Biol&amp;volume=116&amp;pages=848-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.funbio.2012.05.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.funbio.2012.05.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.funbio.2012.05.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22749171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brown</div> <div class="given-names">N. A.</div></div>, <div class="name string-name"><div class="surname">Antoniw</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hammond-Kosack</div> <div class="given-names">K. E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The predicted secretome of the plant pathogenic fungus <em>Fusarium graminearum</em>: a refined comparative analysis,</div> <div class="source">PLoS One</div> , <div class="volume">7</div>, article <div class="fpage">e33731</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20predicted%20secretome%20of%20the%20plant%20pathogenic%20fungus%20Fusarium%20graminearum%3A%20a%20refined%20comparative%20analysis%2C&amp;author=N.%20A.%20Brown&amp;author=J.%20Antoniw&amp;author=K.%20E.%20Hammond-Kosack&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e33731" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0033731" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0033731" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0033731"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22493673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Emanuelsson</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Locating proteins in the cell using TargetP, SignalP and related tools,</div> <div class="source">Nat. Protoc</div> ., <div class="volume">2</div>, <div class="fpage">953</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20proteins%20in%20the%20cell%20using%20TargetP%2C%20SignalP%20and%20related%20tools%2C&amp;author=O.%20Emanuelsson&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2007&amp;journal=Nat.%20Protoc&amp;volume=2&amp;pages=953-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2007.131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2007.131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2007.131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sonnhammer</div> <div class="given-names">EL</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">HG</div></div>, <div class="name string-name"><div class="surname">Krogh</div> <div class="given-names">A</div></div></div> A hidden Markov model for predicting transmembrane helices in protein sequences. In: Glasgow, TL, Major, F, Lathrop, D, Sankoff, D, Sensen, C (eds) <em>Proceedings of the Sixth International Conference on Intelligent Systems for Molecular Biology</em>, <div class="year">1998</div>, p. 175–82. AAAI Press, Menlo Park, CA, USA.</div></div></div></div><div content-id="dsx040-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">O'Connell</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Thon</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Hacquard</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Lifestyle transitions in plant pathogenic <em>Colletotrichum</em> fungi deciphered by genome and transcriptome analyses</div>, <div class="source">Nat. Genet</div> ., <div class="volume">44</div>, <div class="fpage">1060</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lifestyle%20transitions%20in%20plant%20pathogenic%20Colletotrichum%20fungi%20deciphered%20by%20genome%20and%20transcriptome%20analyses&amp;author=R.%20J.%20O'Connell&amp;author=M.%20R.%20Thon&amp;author=S.%20Hacquard&amp;publication_year=2012&amp;journal=Nat.%20Genet&amp;volume=44&amp;pages=1060-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2372" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22885923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boraston</div> <div class="given-names">A. B.</div></div>, <div class="name string-name"><div class="surname">Bolam</div> <div class="given-names">D. N.</div></div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">H. J.</div></div>, <div class="name string-name"><div class="surname">Davies</div> <div class="given-names">G. J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Carbohydrate-binding modules: fine tuning polysaccharide recognition,</div> <div class="source">Biochem. J</div> ., <div class="volume">382</div>, <div class="fpage">769</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbohydrate-binding%20modules%3A%20fine%20tuning%20polysaccharide%20recognition%2C&amp;author=A.%20B.%20Boraston&amp;author=D.%20N.%20Bolam&amp;author=H.%20J.%20Gilbert&amp;author=G.%20J.%20Davies&amp;publication_year=2004&amp;journal=Biochem.%20J&amp;volume=382&amp;pages=769-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BJ20040892" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20040892" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBJ20040892"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15214846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Levasseur</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Drula</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Lombard</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Coutinho</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Henrissat</div> <div class="given-names">B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Expansion of the enzymatic repertoire of the CAZy database to integrate auxiliary redox enzymes,</div> <div class="source">Biotechnol. Biofuels</div> , <div class="volume">6</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expansion%20of%20the%20enzymatic%20repertoire%20of%20the%20CAZy%20database%20to%20integrate%20auxiliary%20redox%20enzymes%2C&amp;author=A.%20Levasseur&amp;author=E.%20Drula&amp;author=V.%20Lombard&amp;author=P.%20M.%20Coutinho&amp;author=B.%20Henrissat&amp;publication_year=2013&amp;journal=Biotechnol.%20Biofuels&amp;volume=6&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1754-6834-6-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1754-6834-6-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1754-6834-6-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23514094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yin</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Candidate effector proteins of the necrotrophic apple canker pathogen <em>Valsa mali</em> can suppress BAX-induced PCD,</div> <div class="source">Front Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">579</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Candidate%20effector%20proteins%20of%20the%20necrotrophic%20apple%20canker%20pathogen%20Valsa%20mali%20can%20suppress%20BAX-induced%20PCD%2C&amp;author=Z.%20Li&amp;author=Z.%20Yin&amp;author=Y.%20Fan&amp;author=M.%20Xu&amp;author=Z.%20Kang&amp;author=L.%20Huang&amp;publication_year=2015&amp;journal=Front%20Plant%20Sci&amp;volume=6&amp;pages=579" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26284095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yin</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome sequence of Valsa canker pathogens uncovers a potential adaptation of colonization of woody bark,</div> <div class="source">New Phytol</div> ., <div class="volume">208</div>, <div class="fpage">1202</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20Valsa%20canker%20pathogens%20uncovers%20a%20potential%20adaptation%20of%20colonization%20of%20woody%20bark%2C&amp;author=Z.%20Yin&amp;author=H.%20Liu&amp;author=Z.%20Li&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=208&amp;pages=1202-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26137988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Comparative analysis of the genomes of two field isolates of the rice blast fungus <em>Magnaporthe oryzae</em>,</div> <div class="source">PLoS Genet</div> ., <div class="volume">8</div>, article <div class="fpage">1002869</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20genomes%20of%20two%20field%20isolates%20of%20the%20rice%20blast%20fungus%20Magnaporthe%20oryzae%2C&amp;author=M.%20Xue&amp;author=J.%20Yang&amp;author=Z.%20Li&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=1002869" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002869" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002869"> </span></div> </div></div></div></div></div><div content-id="dsx040-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">J. T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Exploring genome characteristics and sequence quality without a reference,</div> <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1228</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20genome%20characteristics%20and%20sequence%20quality%20without%20a%20reference%2C&amp;author=J.%20T.%20Simpson&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1228-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24443382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Denton</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">Lugo-Martinez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tucker</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Schrider</div> <div class="given-names">D. R.</div></div>, <div class="name string-name"><div class="surname">Warren</div> <div class="given-names">W. C.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M. W.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Extensive error in the number of genes inferred from draft genome assemblies,</div> <div class="source">PLoS Comput. Biol</div> ., <div class="volume">10</div>, article <div class="fpage">e1003998</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20error%20in%20the%20number%20of%20genes%20inferred%20from%20draft%20genome%20assemblies%2C&amp;author=J.%20F.%20Denton&amp;author=J.%20Lugo-Martinez&amp;author=A.%20E.%20Tucker&amp;author=D.%20R.%20Schrider&amp;author=W.%20C.%20Warren&amp;author=M.%20W.%20Hahn&amp;publication_year=2014&amp;journal=PLoS%20Comput.%20Biol&amp;volume=10&amp;pages=e1003998" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003998"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25474019" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Specific adaptation of <em>Ustilaginoidea virens</em> in occupying host florets revealed by comparative and functional genomics,</div> <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">3849</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specific%20adaptation%20of%20Ustilaginoidea%20virens%20in%20occupying%20host%20florets%20revealed%20by%20comparative%20and%20functional%20genomics%2C&amp;author=Y.%20Zhang&amp;author=K.%20Zhang&amp;author=A.%20Fang&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=3849" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24846013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baskarathevan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jaspers</div> <div class="given-names">M. V.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">E. E.</div></div>, <div class="name string-name"><div class="surname">Cruickshank</div> <div class="given-names">R. H.</div></div>, <div class="name string-name"><div class="surname">Ridgway</div> <div class="given-names">H. J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genetic and pathogenic diversity of <em>Neofusicoccum parvum</em> in new Zealand vineyards,</div> <div class="source">Fungal Biol</div> ., <div class="volume">116</div>, <div class="fpage">276</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20pathogenic%20diversity%20of%20Neofusicoccum%20parvum%20in%20new%20Zealand%20vineyards%2C&amp;author=J.%20Baskarathevan&amp;author=M.%20V.%20Jaspers&amp;author=E.%20E.%20Jones&amp;author=R.%20H.%20Cruickshank&amp;author=H.%20J.%20Ridgway&amp;publication_year=2012&amp;journal=Fungal%20Biol&amp;volume=116&amp;pages=276-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.funbio.2011.11.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.funbio.2011.11.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.funbio.2011.11.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22289773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ullah</div> <div class="given-names">S.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">dbPAF: an integrative database of protein phosphorylation in animals and fungi,</div> <div class="source">Sci. Rep.</div>  <div class="volume">6</div>, article <div class="fpage">e23534</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dbPAF%3A%20an%20integrative%20database%20of%20protein%20phosphorylation%20in%20animals%20and%20fungi%2C&amp;author=S.%20Ullah&amp;publication_year=2016&amp;journal=Sci.%20Rep.&amp;volume=6&amp;pages=e23534" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep23534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep23534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep23534"> </span></div> </div></div></div></div></div><div content-id="dsx040-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schamber</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Leroch</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diwo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Mendgen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>. 
<div class="article-title">The role of mitogen activated protein (MAP) kinase signalling components and the Ste12 transcription factor in germination and pathogenicity of <em>Botrytis cinerea</em>,</div> <div class="source">Mol. Plant Pathol.</div>  <div class="volume">11</div>, <div class="fpage">105</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20mitogen%20activated%20protein%20(MAP)%20kinase%20signalling%20components%20and%20the%20Ste12%20transcription%20factor%20in%20germination%20and%20pathogenicity%20of%20Botrytis%20cinerea%2C&amp;author=A.%20Schamber&amp;author=M.%20Leroch&amp;author=J.%20Diwo&amp;author=K.%20Mendgen&amp;author=M.%20Hahn&amp;publication_year=2010&amp;journal=Mol.%20Plant%20Pathol.&amp;volume=11&amp;pages=105-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mpp.2010.11.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmpp.2010.11.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmpp.2010.11.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20078780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kramer</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Thines</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Foster</div> <div class="given-names">A. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">MAP kinase signalling pathway components and targets conserved between the distantly related plant pathogenic fungi <em>Mycosphaerella graminicola</em> and <em>Magnaporthe grisea</em>,</div> <div class="source">Fungal Genet. Biol.</div>  <div class="volume">46</div>, <div class="fpage">667</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAP%20kinase%20signalling%20pathway%20components%20and%20targets%20conserved%20between%20the%20distantly%20related%20plant%20pathogenic%20fungi%20Mycosphaerella%20graminicola%20and%20Magnaporthe%20grisea%2C&amp;author=B.%20Kramer&amp;author=E.%20Thines&amp;author=A.%20J.%20Foster&amp;publication_year=2009&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=46&amp;pages=667-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2009.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2009.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2009.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19520179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alkan</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Espeso</div> <div class="given-names">E. A.</div></div>, <div class="name string-name"><div class="surname">Prusky</div> <div class="given-names">D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Virulence regulation of phytopathogenic fungi by pH,</div> <div class="source">Antioxid. Redox Signal</div> , <div class="volume">19</div>, <div class="fpage">1012</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Virulence%20regulation%20of%20phytopathogenic%20fungi%20by%20pH%2C&amp;author=N.%20Alkan&amp;author=E.%20A.%20Espeso&amp;author=D.%20Prusky&amp;publication_year=2013&amp;journal=Antioxid.%20Redox%20Signal&amp;volume=19&amp;pages=1012-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/ars.2012.5062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fars.2012.5062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fars.2012.5062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23249178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Siewers</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Viaud</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jimenez-Teja</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Functional analysis of the cytochrome P450 monooxygenase gene bcbot1 of <em>Botrytis cinerea</em> indicates that botrydial is a strain-specific virulence factor,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">18</div>, <div class="fpage">137</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20analysis%20of%20the%20cytochrome%20P450%20monooxygenase%20gene%20bcbot1%20of%20Botrytis%20cinerea%20indicates%20that%20botrydial%20is%20a%20strain-specific%20virulence%20factor%2C&amp;author=V.%20Siewers&amp;author=M.%20Viaud&amp;author=D.%20Jimenez-Teja&amp;publication_year=2005&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=18&amp;pages=137-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-18-0602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-18-0602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-18-0602"> </span></div> </div></div></div></div></div><div content-id="dsx040-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nierman</div> <div class="given-names">W. C.</div></div>, <div class="name string-name"><div class="surname">Pain</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">M. J.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Genomic sequence of the pathogenic and allergenic filamentous fungus <em>Aspergillus fumigatus</em></div>, <div class="source">Nature</div> , <div class="volume">438</div>, <div class="fpage">1092</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20sequence%20of%20the%20pathogenic%20and%20allergenic%20filamentous%20fungus%20Aspergillus%20fumigatus&amp;author=W.%20C.%20Nierman&amp;author=A.%20Pain&amp;author=M.%20J.%20Anderson&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=1092-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04332" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04332" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04332"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16371993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luini</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Fleurat-Lessard</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Rousseau</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Roblin</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Berjeaud</div> <div class="given-names">J.-M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Inhibitory effects of polypeptides secreted by the grapevine pathogens <em>Phaeomoniella chlamydospora</em> and <em>Phaeoacremonium aleophilum</em> on plant cell activities,</div> <div class="source">Physiol. Mol. Plant Pathol.</div>  <div class="volume">74</div>, <div class="fpage">403</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inhibitory%20effects%20of%20polypeptides%20secreted%20by%20the%20grapevine%20pathogens%20Phaeomoniella%20chlamydospora%20and%20Phaeoacremonium%20aleophilum%20on%20plant%20cell%20activities%2C&amp;author=E.%20Luini&amp;author=P.%20Fleurat-Lessard&amp;author=L.%20Rousseau&amp;author=G.%20Roblin&amp;author=J.-M.%20Berjeaud&amp;publication_year=2010&amp;journal=Physiol.%20Mol.%20Plant%20Pathol.&amp;volume=74&amp;pages=403-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pmpp.2010.06.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pmpp.2010.06.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pmpp.2010.06.007"> </span></div> </div></div></div></div></div><div content-id="dsx040-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bazzone</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Madej</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Kaback</div> <div class="given-names">H. R.</div></div>, <div class="name string-name"><div class="surname">Fendler</div> <div class="given-names">K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title"><em>pH regulation of electrogenic sugar/H+ symport in MFS sugar permeases</em>,</div> <div class="source">PLoS ONE</div> , <div class="volume">1111</div>, article <div class="fpage">e0156392</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=pH%20regulation%20of%20electrogenic%20sugar%2FH%2B%20symport%20in%20MFS%20sugar%20permeases%2C&amp;author=A.%20Bazzone&amp;author=M.%20G.%20Madej&amp;author=H.%20R.%20Kaback&amp;author=K.%20Fendler&amp;publication_year=2016&amp;journal=PLoS%20ONE&amp;volume=1111&amp;pages=e0156392" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0156392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0156392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0156392"> </span></div> </div></div></div></div></div><div content-id="dsx040-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Puri</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Manoharlal</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title"><em>MFS transportome of the human pathogenic yeast Candida albicans</em>,</div> <div class="source">BMC Genomics</div> , <div class="volume">99</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.</div></div></div></div><div content-id="dsx040-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marshall</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kombrink</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Motteram</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Analysis of two in planta expressed LysM effector homologs from the fungus <em>Mycosphaerella graminicola</em> reveals novel functional properties and varying contributions to virulence on wheat,</div> <div class="source">Plant Physiol</div> ., <div class="volume">156</div>, <div class="fpage">756</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20two%20in%20planta%20expressed%20LysM%20effector%20homologs%20from%20the%20fungus%20Mycosphaerella%20graminicola%20reveals%20novel%20functional%20properties%20and%20varying%20contributions%20to%20virulence%20on%20wheat%2C&amp;author=R.%20Marshall&amp;author=A.%20Kombrink&amp;author=J.%20Motteram&amp;publication_year=2011&amp;journal=Plant%20Physiol&amp;volume=156&amp;pages=756-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.176347" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.176347" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.176347"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21467214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Badel</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Piquerez</div> <div class="given-names">S. J. M.</div></div>, <div class="name string-name"><div class="surname">Greenshields</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">In planta effector competition assays detect <em>Hyaloperonospora arabidopsidis</em> effectors that contribute to virulence and localize to different plant subcellular compartments,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">26</div>, <div class="fpage">745</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20planta%20effector%20competition%20assays%20detect%20Hyaloperonospora%20arabidopsidis%20effectors%20that%20contribute%20to%20virulence%20and%20localize%20to%20different%20plant%20subcellular%20compartments%2C&amp;author=J.%20L.%20Badel&amp;author=S.%20J.%20M.%20Piquerez&amp;author=D.%20Greenshields&amp;publication_year=2013&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=26&amp;pages=745-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-06-12-0154-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-06-12-0154-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-06-12-0154-R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23734779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hirabuchi</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Deployment of the <em>Burkholderia glumae</em> type III secretion system as an efficient tool for translocating pathogen effectors to monocot cells,</div> <div class="source">Plant J</div> ., <div class="volume">74</div>, <div class="fpage">701</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deployment%20of%20the%20Burkholderia%20glumae%20type%20III%20secretion%20system%20as%20an%20efficient%20tool%20for%20translocating%20pathogen%20effectors%20to%20monocot%20cells%2C&amp;author=S.%20Sharma&amp;author=S.%20Sharma&amp;author=A.%20Hirabuchi&amp;publication_year=2013&amp;journal=Plant%20J&amp;volume=74&amp;pages=701-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.74.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.74.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.74.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23451734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Condon</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Leng</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Comparative genome structure, secondary metabolite, and effector coding capacity across <em>Cochliobolus</em> pathogens,</div> <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, article <div class="fpage">e1003233</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20structure%2C%20secondary%20metabolite%2C%20and%20effector%20coding%20capacity%20across%20Cochliobolus%20pathogens%2C&amp;author=B.%20J.%20Condon&amp;author=Y.%20Leng&amp;author=D.%20Wu&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23357949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">MacDonald</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Syed</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Comparative genomics of the white-rot fungi, <em>Phanerochaete carnosa</em> and <em>P</em>. <em>chrysosporium</em>, to elucidate the genetic basis of the distinct wood types they colonize,</div> <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">579</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20the%20white-rot%20fungi%2C%20Phanerochaete%20carnosa%20and%20P.%20chrysosporium%2C%20to%20elucidate%20the%20genetic%20basis%20of%20the%20distinct%20wood%20types%20they%20colonize%2C&amp;author=H.%20Suzuki&amp;author=J.%20MacDonald&amp;author=K.%20Syed&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=579" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23114097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Floudas</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Binder</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Riley</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The paleozoic origin of enzymatic lignin decomposition reconstructed from 31 fungal genomes,</div> <div class="source">Science</div> , <div class="volume">336</div>, <div class="fpage">1715</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20paleozoic%20origin%20of%20enzymatic%20lignin%20decomposition%20reconstructed%20from%2031%20fungal%20genomes%2C&amp;author=D.%20Floudas&amp;author=M.%20Binder&amp;author=R.%20Riley&amp;publication_year=2012&amp;journal=Science&amp;volume=336&amp;pages=1715-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1221748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1221748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1221748"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">ten Have</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mulder</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Visser</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">van Kan</div> <div class="given-names">J. A. L.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">The endopolygalacturonase gene Bcpg1 is required for full virulence of <em>Botrytis cinerea</em>,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">11</div>, <div class="fpage">1009</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20endopolygalacturonase%20gene%20Bcpg1%20is%20required%20for%20full%20virulence%20of%20Botrytis%20cinerea%2C&amp;author=A.%20ten%20Have&amp;author=W.%20Mulder&amp;author=J.%20Visser&amp;author=J.%20A.%20L.%20van%20Kan&amp;publication_year=1998&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=11&amp;pages=1009-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.1998.11.10.1009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.1998.11.10.1009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.1998.11.10.1009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9768518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J. -R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Comparative analysis of fungal genomes reveals different plant cell wall degrading capacity in fungi,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">54</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20fungal%20genomes%20reveals%20different%20plant%20cell%20wall%20degrading%20capacity%20in%20fungi%2C&amp;author=Z.%20Zhao&amp;author=H.%20Liu&amp;author=C.%20Wang&amp;author=J.%20-R.%20Xu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=54-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23350846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Riley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salamov</div> <div class="given-names">A. A.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">D. W.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Extensive sampling of basidiomycete genomes demonstrates inadequacy of the white-rot/brown-rot paradigm for wood decay fungi,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">9923</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20sampling%20of%20basidiomycete%20genomes%20demonstrates%20inadequacy%20of%20the%20white-rot%2Fbrown-rot%20paradigm%20for%20wood%20decay%20fungi%2C&amp;author=R.%20Riley&amp;author=A.%20A.%20Salamov&amp;author=D.%20W.%20Brown&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=9923-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1400592111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1400592111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1400592111"> </span></div> </div></div></div></div></div><div content-id="dsx040-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Stajich</div> <div class="given-names">J. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Characterization of the carbohydrate binding module 18 gene family in the amphibian pathogen <em>Batrachochytrium dendrobatidis</em>,</div> <div class="source">Fungal Genet. Biol</div> ., <div class="volume">77</div>, <div class="fpage">31</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20carbohydrate%20binding%20module%2018%20gene%20family%20in%20the%20amphibian%20pathogen%20Batrachochytrium%20dendrobatidis%2C&amp;author=P.%20Liu&amp;author=J.%20E.%20Stajich&amp;publication_year=2015&amp;journal=Fungal%20Genet.%20Biol&amp;volume=77&amp;pages=31-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2015.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2015.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2015.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25819009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx040-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx040-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xin</div> <div class="given-names">X.-F.</div></div>, <div class="name string-name"><div class="surname">Nomura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Aung</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Bacteria establish an aqueous living space in plants crucial for virulence</div>. <div class="source">Nature</div> , <div class="volume">539</div>, <div class="fpage">524</div>–<div class="lpage">530</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteria%20establish%20an%20aqueous%20living%20space%20in%20plants%20crucial%20for%20virulence&amp;author=X.-F.%20Xin&amp;author=K.%20Nomura&amp;author=K.%20Aung&amp;publication_year=2016&amp;journal=Nature&amp;volume=539&amp;pages=524-530" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature20166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature20166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature20166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27882964" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4282306&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx040/3/suppl_data_dsx040.zip?Expires=2147483647&amp;Signature=FOH0CPQfsCcQ5pdI~pNNT8aYnj5TXpQe63dUXvPWdS94cqpwp~dLr5Jl~yGmlZjJlkfdT4DAh~JK-TmiD-DRVF82MEuXhixCFKnvZtqodV0v8hD4YrmTyIOOT6ndwo~Bv2-cpgTJb~TvM3QxLYEhaq89-XgtPXBZ~iF4TbSZQPCyennpdQxCTKx2cR8Di~arvzsPdst7pD3LAHADLYbahnHPLg369JyFQbsna5OTjo-QZQJVMdi5DZ-0SBgd8Nw1O5IKajo3mq05CUBXwjVklhEc85KCvYRRcANHNdPi7VY6r4cKQsqZi6dAUnvD8HiCDI1CCMRDlY3k849unRF2-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,382</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx040"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4282306"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f87%2f4282306">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000425954100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F87%2F4282306" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4282306'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of Botryosphaeriaceae" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx040");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f87%2f4282306">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

