

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/217/4683712","@type":"ScholarlyArticle","name":"Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi","datePublished":"2017-12-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx051","keywords":["arbuscular mycorrhizal fungi","transcriptomics","phylogenomics","orthology clustering "],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/217/4683712","author":[{"name":"Beaudet, Denis","affiliation":"Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada","@type":"Person"},{"name":"Chen, Eric C H","affiliation":"Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada","@type":"Person"},{"name":"Mathieu, Stephanie","affiliation":"Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada","@type":"Person"},{"name":"Yildirir, Gokalp","affiliation":"Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada","@type":"Person"},{"name":"Ndikumana, Steve","affiliation":"Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada","@type":"Person"},{"name":"Dalpé, Yolande","affiliation":"Agriculture and Agri-Food Canada, 960 Carling Ave, Ottawa, Ontario K1A 0C6, Canada","@type":"Person"},{"name":"Séguin, Sylvie","affiliation":"Agriculture and Agri-Food Canada, 960 Carling Ave, Ottawa, Ontario K1A 0C6, Canada","@type":"Person"},{"name":"Farinelli, Laurent","affiliation":"Fasteris SA, Chemin du Pont-du-Centenaire 109, Geneva 1228, Switzerland","@type":"Person"},{"name":"Stajich, Jason E","affiliation":"Department of Plant Pathology &amp; Microbiology and Institute for Integrative Genome Biology, University of California, Riverside, Riverside, CA 92521, USA","@type":"Person"},{"name":"Corradi, Nicolas","affiliation":"Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada","@type":"Person"}],"description":"Abstract.  Arbuscular mycorrhizal fungi (AMF) are a group of soil microorganisms that establish symbioses with the vast majority of land plants. To date, genera","pageStart":"217","pageEnd":"227","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi" />
    <meta property="og:description" content="Abstract.  Arbuscular mycorrhizal fungi (AMF) are a group of soil microorganisms that establish symbioses with the vast majority of land plants. To date, genera" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/217/4683712" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-12-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Beaudet, Denis" /><meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada" /><meta name="citation_author" content="Chen, Eric C H" /><meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada" /><meta name="citation_author" content="Mathieu, Stephanie" /><meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada" /><meta name="citation_author" content="Yildirir, Gokalp" /><meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada" /><meta name="citation_author" content="Ndikumana, Steve" /><meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada" /><meta name="citation_author" content="Dalpé, Yolande" /><meta name="citation_author_institution" content="Agriculture and Agri-Food Canada, 960 Carling Ave, Ottawa, Ontario K1A 0C6, Canada" /><meta name="citation_author" content="Séguin, Sylvie" /><meta name="citation_author_institution" content="Agriculture and Agri-Food Canada, 960 Carling Ave, Ottawa, Ontario K1A 0C6, Canada" /><meta name="citation_author" content="Farinelli, Laurent" /><meta name="citation_author_institution" content="Fasteris SA, Chemin du Pont-du-Centenaire 109, Geneva 1228, Switzerland" /><meta name="citation_author" content="Stajich, Jason E" /><meta name="citation_author_institution" content="Department of Plant Pathology &amp; Microbiology and Institute for Integrative Genome Biology, University of California, Riverside, Riverside, CA 92521, USA" /><meta name="citation_author" content="Corradi, Nicolas" /><meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada" /><meta name="citation_title" content="Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi" /><meta name="citation_firstpage" content="217" /><meta name="citation_lastpage" content="227" /><meta name="citation_doi" content="10.1093/dnares/dsx051" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content=" citation_title=Mycorrhizal Symbiosis;  citation_year=2008; " /><meta name="citation_reference" content="citation_title=Mechanisms underlying beneficial plant–fungus interactions in mycorrhizal symbiosis;  citation_journal_title=Nat. Commun;  citation_year=2010;  citation_volume=1;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Mutualistic mycorrhiza-like symbiosis in the most ancient group of land plants;  citation_journal_title=Nat. Commun;  citation_year=2010;  citation_volume=1;  citation_pages=103" /><meta name="citation_reference" content="citation_title=Arbuscular mycorrhiza: the mother of plant root endosymbioses;  citation_journal_title=Nat. Rev. Microbiol;  citation_year=2008;  citation_volume=6;  citation_pages=763-75; " /><meta name="citation_reference" content="citation_title=Mycorrhizal fungal diversity determines plant biodiversity, ecosystem variability and productivity;  citation_journal_title=Nature;  citation_year=1998;  citation_volume=396;  citation_pages=69-72; " /><meta name="citation_reference" content="citation_title=Arbuscular mycorrhizal fungi as natural biofertilizers: let’s benefit from past successes;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=1559" /><meta name="citation_reference" content="citation_title=Genome diversity in arbuscular mycorrhizal fungi;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2015;  citation_volume=26;  citation_pages=113-9; " /><meta name="citation_reference" content="citation_title=Evidence for the evolution of multiple genomes in arbuscular mycorrhizal fungi;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=414;  citation_pages=745-8; " /><meta name="citation_reference" content="citation_title=Low gene copy number shows that arbuscular mycorrhizal fungi inherit genetically different nuclei;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=433;  citation_pages=160-3; " /><meta name="citation_reference" content="citation_title=Organization of genetic variation in individuals of arbuscular mycorrhizal fungi;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=427;  citation_pages=733-7; " /><meta name="citation_reference" content="Schussler A., Walker C.  2010 , The Glomeromycota: a species list with new families. http// www.amf-phylogeny.com (14 November 2017, date last accessed). " /><meta name="citation_reference" content="citation_title=Phylogenetic reference data for systematics and phylotaxonomy of arbuscular mycorrhizal fungi from phylum to species level;  citation_journal_title=New Phytol;  citation_year=2012;  citation_volume=193;  citation_pages=970-84; " /><meta name="citation_reference" content="citation_title=Fungal Mating in the Most Widespread Plant Symbionts?;  citation_journal_title=Trends Plant Sci;  citation_volume=64;  citation_pages=175-83; " /><meta name="citation_reference" content="citation_title=Evidence for the sexual origin of heterokaryosis in arbuscular mycorrhizal fungi;  citation_journal_title=Nat. Microbiol;  citation_year=2016;  citation_volume=1;  citation_pages=16033." /><meta name="citation_reference" content="citation_title=DNA-based species level detection of Glomeromycota: one PCR primer set for all arbuscular mycorrhizal fungi;  citation_journal_title=New Phytol;  citation_year=2009;  citation_volume=183;  citation_pages=212-23; " /><meta name="citation_reference" content="citation_title=DNA-based detection and identification of Glomeromycota: the virtual taxonomy of environmental sequences;  citation_journal_title=Botany;  citation_year=2013;  citation_volume=92;  citation_pages=135-47; " /><meta name="citation_reference" content="citation_title=A new fungal phylum, the Glomeromycota: phylogeny and evolution;  citation_journal_title=Mycol. Res;  citation_year=2001;  citation_volume=105;  citation_pages=1413-21; " /><meta name="citation_reference" content="citation_title=A phylum-level phylogenetic classification of zygomycete fungi based on genome-scale data;  citation_journal_title=Mycologia;  citation_year=2016;  citation_volume=108;  citation_pages=1028-46; " /><meta name="citation_reference" content="citation_title=Uniting species- and community-oriented approaches to understand arbuscular mycorrhizal fungal diversity;  citation_journal_title=Fungal Ecol;  citation_year=2016;  citation_volume=24;  citation_pages=106-13; " /><meta name="citation_reference" content="citation_title=Advances in Glomeromycota taxonomy and classification;  citation_journal_title=IMA Fungus;  citation_year=2011;  citation_volume=2;  citation_pages=191-9; " /><meta name="citation_reference" content="citation_title=An evidence-based consensus for the classification of arbuscular mycorrhizal fungi (Glomeromycota);  citation_journal_title=Mycorrhiza;  citation_year=2013;  citation_volume=23;  citation_pages=515-31; " /><meta name="citation_reference" content="citation_title=Molecular trait indicators: moving beyond phylogeny in arbuscular mycorrhizal ecology;  citation_journal_title=New Phytol;  citation_year=2010;  citation_volume=185;  citation_pages=67-82; " /><meta name="citation_reference" content="citation_title=The transcriptome of the arbuscular mycorrhizal fungus Glomus intraradices (DAOM 197198) reveals functional tradeoffs in an obligate symbiont;  citation_journal_title=New Phytol;  citation_year=2012;  citation_volume=193;  citation_pages=755-69; " /><meta name="citation_reference" content="citation_title=Symbiosis with an endobacterium increases the fitness of a mycorrhizal fungus, raising its bioenergetic potential;  citation_journal_title=ISME J;  citation_year=2016;  citation_volume=10;  citation_pages=130-44; " /><meta name="citation_reference" content="citation_title=A survey of the gene repertoire of gigaspora rosea unravels conserved features among glomeromycota for obligate biotrophy;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=233" /><meta name="citation_reference" content="citation_title=Polyphosphate accumulation is driven by transcriptome alterations that lead to near-synchronous and near-equivalent uptake of inorganic cations in an arbuscular mycorrhizal fungus;  citation_journal_title=New Phytol;  citation_year=2014;  citation_volume=204;  citation_pages=638-49; " /><meta name="citation_reference" content="citation_title=The effector candidate repertoire of the arbuscular mycorrhizal fungus Rhizophagus clarus;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=101" /><meta name="citation_reference" content="citation_title=Genome of an arbuscular mycorrhizal fungus provides insight into the oldest plant symbiosis;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=2013;  citation_volume=110;  citation_pages=20117-22; " /><meta name="citation_reference" content="citation_title=Single nucleus genome sequencing reveals high similarity among nuclei of an endomycorrhizal fungus;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004078" /><meta name="citation_reference" content="citation_title=Single-cell sequencing-based technologies will revolutionize whole-organism science;  citation_journal_title=Nat. Rev. Genet;  citation_year=2013;  citation_volume=14;  citation_pages=618-30; " /><meta name="citation_reference" content="citation_title=Nitrogen transfer and assimilation between the arbuscular mycorrhizal fungus Glomus intraradices Schenck &amp; Smith and Ri T-DNA roots of Daucus carota L. in an in vitro compartmented system;  citation_journal_title=Can. J. Microbiol;  citation_year=2004;  citation_volume=50;  citation_pages=251-60; " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=HMMER web server: interactive sequence similarity searching;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W29-37; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=D281-8; " /><meta name="citation_reference" content="citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=785-6; " /><meta name="citation_reference" content="citation_title=Predicting transmembrane protein topology with a hidden markov model: application to complete genomes1;  citation_journal_title=J. Mol. Biol;  citation_year=2001;  citation_volume=305;  citation_pages=567-80; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology;  citation_journal_title=Nat. Genet;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=KEGG: Kyoto encyclopedia of genes and genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=W3-10; " /><meta name="citation_reference" content="citation_title=WEGO: a web tool for plotting GO annotations;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W293-7; " /><meta name="citation_reference" content="citation_title=WebMGA: a customizable web server for fast metagenomic sequence analysis;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=444" /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=Phylogenetic and functional assessment of orthologs inference projects and methods;  citation_journal_title=PLoS Comput. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=e1000262" /><meta name="citation_reference" content="citation_title=Phylogenomics supports microsporidia as the earliest diverging clade of sequenced fungi;  citation_journal_title=BMC Biol;  citation_year=2012;  citation_volume=10;  citation_pages=47" /><meta name="citation_reference" content="citation_title=The RAxML 7.0.4 manual;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2688-90; " /><meta name="citation_reference" content="citation_title=PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2286-8; " /><meta name="citation_reference" content="citation_title=Molecular evolution in bacterial endosymbionts of fungi;  citation_journal_title=Mol. Biol. Evol;  citation_year=2010;  citation_volume=27;  citation_pages=622-36; " /><meta name="citation_reference" content="citation_publisher=Springer New York, New York, NY;  citation_title=Microbial Environmental Genomics (MEG);  citation_year=2016;  citation_pages=29-53; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=Arabidopsis thaliana BTB/POZ-MATH proteins interact with members of the ERF/AP2 transcription factor family;  citation_journal_title=FEBS J;  citation_year=2009;  citation_volume=276;  citation_pages=6624-35; " /><meta name="citation_reference" content="citation_title=Expansion and function of repeat domain proteins during stress and development in plants;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=6;  citation_pages=1218" /><meta name="citation_reference" content="citation_title=Overview of fungal lipase: a review;  citation_journal_title=Appl. Biochem. Biotechnol;  citation_year=2012;  citation_volume=166;  citation_pages=486-520; " /><meta name="citation_reference" content="citation_title=Curation of characterized glycoside hydrolases of fungal origin;  citation_journal_title=Database;  citation_year=2011;  citation_volume=2011;  citation_pages=bar020" /><meta name="citation_reference" content="citation_title=Who lives in a fungus[quest] the diversity, origins and functions of fungal endobacteria living in Mucoromycota;  citation_journal_title=ISME J;  citation_year=2017;  citation_volume=11;  citation_pages=1727-35; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/217/24662239/dsx051.pdf" /><meta name="description" content="Abstract.  Arbuscular mycorrhizal fungi (AMF) are a group of soil microorganisms that establish symbioses with the vast majority of land plants. To date, genera" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/217/4683712" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/217/4683712" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/217/4683712");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx051';
    </script>
<title>
	Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4683712" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdyEATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmVhdWRldCwgRGVuaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgT3R0YXdhLCBPdHRhd2EsIE9udGFyaW8gSzFOIDZONSwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlbiwgRXJpYyBDIEgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgT3R0YXdhLCBPdHRhd2EsIE9udGFyaW8gSzFOIDZONSwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWF0aGlldSwgU3RlcGhhbmllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIE90dGF3YSwgT3R0YXdhLCBPbnRhcmlvIEsxTiA2TjUsIENhbmFkYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllpbGRpcmlyLCBHb2thbHAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgT3R0YXdhLCBPdHRhd2EsIE9udGFyaW8gSzFOIDZONSwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmRpa3VtYW5hLCBTdGV2ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBPdHRhd2EsIE90dGF3YSwgT250YXJpbyBLMU4gNk41LCBDYW5hZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEYWxww6ksIFlvbGFuZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZ3JpY3VsdHVyZSBhbmQgQWdyaS1Gb29kIENhbmFkYSwgOTYwIENhcmxpbmcgQXZlLCBPdHRhd2EsIE9udGFyaW8gSzFBIDBDNiwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU8OpZ3VpbiwgU3lsdmllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQWdyaWN1bHR1cmUgYW5kIEFncmktRm9vZCBDYW5hZGEsIDk2MCBDYXJsaW5nIEF2ZSwgT3R0YXdhLCBPbnRhcmlvIEsxQSAwQzYsIENhbmFkYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZhcmluZWxsaSwgTGF1cmVudCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhc3RlcmlzIFNBLCBDaGVtaW4gZHUgUG9udC1kdS1DZW50ZW5haXJlIDEwOSwgR2VuZXZhIDEyMjgsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3RhamljaCwgSmFzb24gRSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgUGF0aG9sb2d5ICZhbXA7IE1pY3JvYmlvbG9neSBhbmQgSW5zdGl0dXRlIGZvciBJbnRlZ3JhdGl2ZSBHZW5vbWUgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBDYWxpZm9ybmlhLCBSaXZlcnNpZGUsIFJpdmVyc2lkZSwgQ0EgOTI1MjEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNvcnJhZGksIE5pY29sYXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgT3R0YXdhLCBPdHRhd2EsIE9udGFyaW8gSzFOIDZONSwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJVbHRyYS1sb3cgaW5wdXQgdHJhbnNjcmlwdG9taWNzIHJldmVhbCB0aGUgc3BvcmUgZnVuY3Rpb25hbCBjb250ZW50IGFuZCBwaHlsb2dlbmV0aWMgYWZmaWxpYXRpb25zIG9mIHBvb3JseSBzdHVkaWVkIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDA1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX3RpdGxlPU15Y29ycmhpemFsIFN5bWJpb3NpczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyB1bmRlcmx5aW5nIGJlbmVmaWNpYWwgcGxhbnTigJNmdW5ndXMgaW50ZXJhY3Rpb25zIGluIG15Y29ycmhpemFsIHN5bWJpb3NpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dHVhbGlzdGljIG15Y29ycmhpemEtbGlrZSBzeW1iaW9zaXMgaW4gdGhlIG1vc3QgYW5jaWVudCBncm91cCBvZiBsYW5kIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTEwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYnVzY3VsYXIgbXljb3JyaGl6YTogdGhlIG1vdGhlciBvZiBwbGFudCByb290IGVuZG9zeW1iaW9zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTc2My03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXljb3JyaGl6YWwgZnVuZ2FsIGRpdmVyc2l0eSBkZXRlcm1pbmVzIHBsYW50IGJpb2RpdmVyc2l0eSwgZWNvc3lzdGVtIHZhcmlhYmlsaXR5IGFuZCBwcm9kdWN0aXZpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zOTY7ICBjaXRhdGlvbl9wYWdlcz02OS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5naSBhcyBuYXR1cmFsIGJpb2ZlcnRpbGl6ZXJzOiBsZXTigJlzIGJlbmVmaXQgZnJvbSBwYXN0IHN1Y2Nlc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTU1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBkaXZlcnNpdHkgaW4gYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTExMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgdGhlIGV2b2x1dGlvbiBvZiBtdWx0aXBsZSBnZW5vbWVzIGluIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MTQ7ICBjaXRhdGlvbl9wYWdlcz03NDUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG93IGdlbmUgY29weSBudW1iZXIgc2hvd3MgdGhhdCBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIGZ1bmdpIGluaGVyaXQgZ2VuZXRpY2FsbHkgZGlmZmVyZW50IG51Y2xlaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzMzsgIGNpdGF0aW9uX3BhZ2VzPTE2MC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Pcmdhbml6YXRpb24gb2YgZ2VuZXRpYyB2YXJpYXRpb24gaW4gaW5kaXZpZHVhbHMgb2YgYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQyNzsgIGNpdGF0aW9uX3BhZ2VzPTczMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJTY2h1c3NsZXIgQS4sIFdhbGtlciBDLiAgMjAxMCAsIFRoZSBHbG9tZXJvbXljb3RhOiBhIHNwZWNpZXMgbGlzdCB3aXRoIG5ldyBmYW1pbGllcy4gaHR0cC8vIHd3dy5hbWYtcGh5bG9nZW55LmNvbSAoMTQgTm92ZW1iZXIgMjAxNywgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIHJlZmVyZW5jZSBkYXRhIGZvciBzeXN0ZW1hdGljcyBhbmQgcGh5bG90YXhvbm9teSBvZiBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIGZ1bmdpIGZyb20gcGh5bHVtIHRvIHNwZWNpZXMgbGV2ZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTkzOyAgY2l0YXRpb25fcGFnZXM9OTcwLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5nYWwgTWF0aW5nIGluIHRoZSBNb3N0IFdpZGVzcHJlYWQgUGxhbnQgU3ltYmlvbnRzPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZm9yIHRoZSBzZXh1YWwgb3JpZ2luIG9mIGhldGVyb2thcnlvc2lzIGluIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xNjAzMy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEtYmFzZWQgc3BlY2llcyBsZXZlbCBkZXRlY3Rpb24gb2YgR2xvbWVyb215Y290YTogb25lIFBDUiBwcmltZXIgc2V0IGZvciBhbGwgYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz0yMTItMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQS1iYXNlZCBkZXRlY3Rpb24gYW5kIGlkZW50aWZpY2F0aW9uIG9mIEdsb21lcm9teWNvdGE6IHRoZSB2aXJ0dWFsIHRheG9ub215IG9mIGVudmlyb25tZW50YWwgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Cb3Rhbnk7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz0xMzUtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IGZ1bmdhbCBwaHlsdW0sIHRoZSBHbG9tZXJvbXljb3RhOiBwaHlsb2dlbnkgYW5kIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TXljb2wuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0xNDEzLTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeWx1bS1sZXZlbCBwaHlsb2dlbmV0aWMgY2xhc3NpZmljYXRpb24gb2Ygenlnb215Y2V0ZSBmdW5naSBiYXNlZCBvbiBnZW5vbWUtc2NhbGUgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TXljb2xvZ2lhOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTEwMjgtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaXRpbmcgc3BlY2llcy0gYW5kIGNvbW11bml0eS1vcmllbnRlZCBhcHByb2FjaGVzIHRvIHVuZGVyc3RhbmQgYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5nYWwgZGl2ZXJzaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEwNi0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4gR2xvbWVyb215Y290YSB0YXhvbm9teSBhbmQgY2xhc3NpZmljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlNQSBGdW5ndXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE5MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldmlkZW5jZS1iYXNlZCBjb25zZW5zdXMgZm9yIHRoZSBjbGFzc2lmaWNhdGlvbiBvZiBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIGZ1bmdpIChHbG9tZXJvbXljb3RhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TXljb3JyaGl6YTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTUxNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHRyYWl0IGluZGljYXRvcnM6IG1vdmluZyBiZXlvbmQgcGh5bG9nZW55IGluIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZWNvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz02Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRvbWUgb2YgdGhlIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ3VzIEdsb211cyBpbnRyYXJhZGljZXMgKERBT00gMTk3MTk4KSByZXZlYWxzIGZ1bmN0aW9uYWwgdHJhZGVvZmZzIGluIGFuIG9ibGlnYXRlIHN5bWJpb250OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3ltYmlvc2lzIHdpdGggYW4gZW5kb2JhY3Rlcml1bSBpbmNyZWFzZXMgdGhlIGZpdG5lc3Mgb2YgYSBteWNvcnJoaXphbCBmdW5ndXMsIHJhaXNpbmcgaXRzIGJpb2VuZXJnZXRpYyBwb3RlbnRpYWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlTTUUgSjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEzMC00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdXJ2ZXkgb2YgdGhlIGdlbmUgcmVwZXJ0b2lyZSBvZiBnaWdhc3BvcmEgcm9zZWEgdW5yYXZlbHMgY29uc2VydmVkIGZlYXR1cmVzIGFtb25nIGdsb21lcm9teWNvdGEgZm9yIG9ibGlnYXRlIGJpb3Ryb3BoeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seXBob3NwaGF0ZSBhY2N1bXVsYXRpb24gaXMgZHJpdmVuIGJ5IHRyYW5zY3JpcHRvbWUgYWx0ZXJhdGlvbnMgdGhhdCBsZWFkIHRvIG5lYXItc3luY2hyb25vdXMgYW5kIG5lYXItZXF1aXZhbGVudCB1cHRha2Ugb2YgaW5vcmdhbmljIGNhdGlvbnMgaW4gYW4gYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5ndXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjA0OyAgY2l0YXRpb25fcGFnZXM9NjM4LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZWZmZWN0b3IgY2FuZGlkYXRlIHJlcGVydG9pcmUgb2YgdGhlIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ3VzIFJoaXpvcGhhZ3VzIGNsYXJ1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG9mIGFuIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ3VzIHByb3ZpZGVzIGluc2lnaHQgaW50byB0aGUgb2xkZXN0IHBsYW50IHN5bWJpb3NpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0yMDExNy0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIG51Y2xldXMgZ2Vub21lIHNlcXVlbmNpbmcgcmV2ZWFscyBoaWdoIHNpbWlsYXJpdHkgYW1vbmcgbnVjbGVpIG9mIGFuIGVuZG9teWNvcnJoaXphbCBmdW5ndXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDA3OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1jZWxsIHNlcXVlbmNpbmctYmFzZWQgdGVjaG5vbG9naWVzIHdpbGwgcmV2b2x1dGlvbml6ZSB3aG9sZS1vcmdhbmlzbSBzY2llbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02MTgtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5pdHJvZ2VuIHRyYW5zZmVyIGFuZCBhc3NpbWlsYXRpb24gYmV0d2VlbiB0aGUgYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5ndXMgR2xvbXVzIGludHJhcmFkaWNlcyBTY2hlbmNrICZhbXA7IFNtaXRoIGFuZCBSaSBULUROQSByb290cyBvZiBEYXVjdXMgY2Fyb3RhIEwuIGluIGFuIGluIHZpdHJvIGNvbXBhcnRtZW50ZWQgc3lzdGVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYW4uIEouIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTI1MS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVsbC1sZW5ndGggdHJhbnNjcmlwdG9tZSBhc3NlbWJseSBmcm9tIFJOQS1TZXEgZGF0YSB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9NjQ0LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SE1NRVIgd2ViIHNlcnZlcjogaW50ZXJhY3RpdmUgc2VxdWVuY2Ugc2ltaWxhcml0eSBzZWFyY2hpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9VzI5LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQyODEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lnbmFsUCA0LjA6IGRpc2NyaW1pbmF0aW5nIHNpZ25hbCBwZXB0aWRlcyBmcm9tIHRyYW5zbWVtYnJhbmUgcmVnaW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz03ODUtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyB0cmFuc21lbWJyYW5lIHByb3RlaW4gdG9wb2xvZ3kgd2l0aCBhIGhpZGRlbiBtYXJrb3YgbW9kZWw6IGFwcGxpY2F0aW9uIHRvIGNvbXBsZXRlIGdlbm9tZXMxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzA1OyAgY2l0YXRpb25fcGFnZXM9NTY3LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtFR0c6IEt5b3RvIGVuY3ljbG9wZWRpYSBvZiBnZW5lcyBhbmQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdhbGF4eSBwbGF0Zm9ybSBmb3IgYWNjZXNzaWJsZSwgcmVwcm9kdWNpYmxlIGFuZCBjb2xsYWJvcmF0aXZlIGJpb21lZGljYWwgYW5hbHlzZXM6IDIwMTYgdXBkYXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPVczLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XRUdPOiBhIHdlYiB0b29sIGZvciBwbG90dGluZyBHTyBhbm5vdGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XMjkzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdlYk1HQTogYSBjdXN0b21pemFibGUgd2ViIHNlcnZlciBmb3IgZmFzdCBtZXRhZ2Vub21pYyBzZXF1ZW5jZSBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3J0aG9NQ0w6IGlkZW50aWZpY2F0aW9uIG9mIG9ydGhvbG9nIGdyb3VwcyBmb3IgZXVrYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjE3OC04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGFuZCBmdW5jdGlvbmFsIGFzc2Vzc21lbnQgb2Ygb3J0aG9sb2dzIGluZmVyZW5jZSBwcm9qZWN0cyBhbmQgbWV0aG9kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwMjYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5vbWljcyBzdXBwb3J0cyBtaWNyb3Nwb3JpZGlhIGFzIHRoZSBlYXJsaWVzdCBkaXZlcmdpbmcgY2xhZGUgb2Ygc2VxdWVuY2VkIGZ1bmdpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJBeE1MIDcuMC40IG1hbnVhbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yNjg4LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb0JheWVzIDM6IGEgQmF5ZXNpYW4gc29mdHdhcmUgcGFja2FnZSBmb3IgcGh5bG9nZW5ldGljIHJlY29uc3RydWN0aW9uIGFuZCBtb2xlY3VsYXIgZGF0aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIyODYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGV2b2x1dGlvbiBpbiBiYWN0ZXJpYWwgZW5kb3N5bWJpb250cyBvZiBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NjIyLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIgTmV3IFlvcmssIE5ldyBZb3JrLCBOWTsgIGNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBFbnZpcm9ubWVudGFsIEdlbm9taWNzIChNRUcpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fcGFnZXM9MjktNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJVU0NPOiBhc3Nlc3NpbmcgZ2Vub21lIGFzc2VtYmx5IGFuZCBhbm5vdGF0aW9uIGNvbXBsZXRlbmVzcyB3aXRoIHNpbmdsZS1jb3B5IG9ydGhvbG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMjEwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIHRoYWxpYW5hIEJUQi9QT1otTUFUSCBwcm90ZWlucyBpbnRlcmFjdCB3aXRoIG1lbWJlcnMgb2YgdGhlIEVSRi9BUDIgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZmFtaWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIEo7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc2OyAgY2l0YXRpb25fcGFnZXM9NjYyNC0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwYW5zaW9uIGFuZCBmdW5jdGlvbiBvZiByZXBlYXQgZG9tYWluIHByb3RlaW5zIGR1cmluZyBzdHJlc3MgYW5kIGRldmVsb3BtZW50IGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU92ZXJ2aWV3IG9mIGZ1bmdhbCBsaXBhc2U6IGEgcmV2aWV3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBCaW9jaGVtLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE2NjsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ni01MjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1cmF0aW9uIG9mIGNoYXJhY3Rlcml6ZWQgZ2x5Y29zaWRlIGh5ZHJvbGFzZXMgb2YgZnVuZ2FsIG9yaWdpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGF0YWJhc2U7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjAxMTsgIGNpdGF0aW9uX3BhZ2VzPWJhcjAyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdobyBsaXZlcyBpbiBhIGZ1bmd1c1txdWVzdF0gdGhlIGRpdmVyc2l0eSwgb3JpZ2lucyBhbmQgZnVuY3Rpb25zIG9mIGZ1bmdhbCBlbmRvYmFjdGVyaWEgbGl2aW5nIGluIE11Y29yb215Y290YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SVNNRSBKOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTcyNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzIvMjE3LzI0NjYyMjM5L2RzeDA1MS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5naSAoQU1GKSBhcmUgYSBncm91cCBvZiBzb2lsIG1pY3Jvb3JnYW5pc21zIHRoYXQgZXN0YWJsaXNoIHN5bWJpb3NlcyB3aXRoIHRoZSB2YXN0IG1ham9yaXR5IG9mIGxhbmQgcGxhbnRzLiBUbyBkYXRlLCBnZW5lcmEiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ2ODM3MTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkJO0YTGrbapwPCJrpISa24uvu7KE=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f217%2f4683712" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f217%2f4683712" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115733959">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733959">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733961">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733961">Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733967">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733967">Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733984">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733984">Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115734013">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115734013">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115734023">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115734023">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/2/207/4675083">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Denis Beaudet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Denis+Beaudet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Beaudet D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Beaudet D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Denis Beaudet</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eric C H Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eric+C+H+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eric C H Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephanie Mathieu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephanie+Mathieu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mathieu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mathieu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stephanie Mathieu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gokalp Yildirir</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gokalp+Yildirir">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yildirir G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yildirir G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gokalp Yildirir</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Steve Ndikumana</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Steve+Ndikumana">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ndikumana S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ndikumana S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Steve Ndikumana</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yolande Dalp&#233;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agriculture and Agri-Food Canada, 960 Carling Ave, Ottawa, Ontario K1A 0C6, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yolande+Dalp%c3%a9">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dalp&#233; Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dalp&#233; Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yolande Dalpé</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sylvie S&#233;guin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agriculture and Agri-Food Canada, 960 Carling Ave, Ottawa, Ontario K1A 0C6, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sylvie+S%c3%a9guin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=S&#233;guin S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;S&#233;guin S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sylvie Séguin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Laurent Farinelli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Fasteris SA, Chemin du Pont-du-Centenaire 109, Geneva 1228, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Laurent+Farinelli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Farinelli L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Farinelli L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Laurent Farinelli</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jason E Stajich</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Plant Pathology &amp; Microbiology and Institute for Integrative Genome Biology, University of California, Riverside, Riverside, CA 92521, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jason+E+Stajich">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stajich J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Stajich J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jason E Stajich</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicolas Corradi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology, University of Ottawa, Ottawa, Ontario K1N 6N5, Canada</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx051-cor1" data-open="dsx051-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 613 562 5800. Fax. +1 613-562-5486. Email: <a href="mailto:ncorradi@uottawa.ca" target="_blank">ncorradi@uottawa.ca</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicolas+Corradi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Corradi N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Corradi N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicolas Corradi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx051-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 217–227, <a href='https://doi.org/10.1093/dnares/dsx051'>https://doi.org/10.1093/dnares/dsx051</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 December 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 June 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 November 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4683712" href="/dnaresearch/article-pdf/25/2/217/24662239/dsx051.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Denis Beaudet, Eric C H Chen, Stephanie Mathieu, Gokalp Yildirir, Steve Ndikumana, Yolande Dalpé, Sylvie Séguin, Laurent Farinelli, Jason E Stajich, Nicolas Corradi;  Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi, <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 217–227, <a href='https://doi.org/10.1093/dnares/dsx051'>https://doi.org/10.1093/dnares/dsx051</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4683712?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Ultra-low%20input%20transcriptomics%20reveal%20the%20spore%20functional%20content%20and%20phylogenetic%20affiliations%20of%20poorly%20studied%20arbuscular%20mycorrhizal%20fungi&amp;publicationDate=2017-12-02&amp;volumeNum=25&amp;issueNum=2&amp;author=Beaudet%2C%20Denis%3B%20Chen%2C%20Eric%20C%20H&amp;startPage=217&amp;endPage=227&amp;contentId=10.1093%2Fdnares%2Fdsx051&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f217%2f4683712" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=115733959 id="115733959" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Arbuscular mycorrhizal fungi (AMF) are a group of soil microorganisms that establish symbioses with the vast majority of land plants. To date, generation of AMF coding information has been limited to model genera that grow well axenically; <em>Rhizoglomus</em> and <em>Gigaspora</em>. Meanwhile, data on the functional gene repertoire of most AMF families is non-existent. Here, we provide primary large-scale transcriptome data from eight poorly studied AMF species (<em>Acaulospora morrowiae, Diversispora versiforme, Scutellospora calospora, Racocetra castanea, Paraglomus brasilianum, Ambispora leptoticha, Claroideoglomus claroideum</em> and <em>Funneliformis mosseae</em>) using ultra-low input ribonucleic acid (RNA)-seq approaches. Our analyses reveals that quiescent spores of many AMF species harbour a diverse functional diversity and solidify known evolutionary relationships within the group. Our findings demonstrate that RNA-seq data obtained from low-input RNA are reliable in comparison to conventional RNA-seq experiments. Thus, our methodology can potentially be used to deepen our understanding of fungal microbial function and phylogeny using minute amounts of RNA material.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;arbuscular mycorrhizal fungi&quot;">arbuscular mycorrhizal fungi</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptomics">transcriptomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogenomics">phylogenomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;orthology clustering &quot;">orthology clustering </a></div>                            <h2 scrollto-destination=115733961 id="115733961" class="section-title" >Introduction</h2>
<p>Arbuscular mycorrhizal fungi (AMF) are peculiar organisms that evolved an obligate symbiotic relationship with plant roots. This mutually beneficial association occurs in more than 80% of plant species<a reveal-id="dsx051-B1" data-open="dsx051-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and is suggested to have facilitated the colonization of land by plants.<a reveal-id="dsx051-B2" data-open="dsx051-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx051-B3" data-open="dsx051-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> In this partnership, plants provide the fungal partner with photosynthetic carbon sources and lipids in exchange for micro- and macro-elements (e.g. phosphates and nitrates) scavenged by AMF from the surrounding soil environment.<a reveal-id="dsx051-B4" data-open="dsx051-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> AMF also interact with other soil microorganisms and are known drivers of ecosystem biodiversity, as they increase plant nutrient uptake and growth, along with resistance to pathogens.<a reveal-id="dsx051-B5" data-open="dsx051-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx051-B6" data-open="dsx051-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> AMF spores and hyphae also harbour thousands of nuclei flowing in a common cytoplasm (a coenocyte) and were thought to be asexual organisms. The genomic organization of this nuclear population has long remained a puzzle,<a reveal-id="dsx051-B7 dsx051-B8 dsx051-B9 dsx051-B10" data-open="dsx051-B7 dsx051-B8 dsx051-B9 dsx051-B10" class="link link-ref link-reveal xref-bibr"><sup>7–10</sup></a> but recent data from the model species <em>Rhizoglomus irregulare</em> (also known as <em>Rhizophagus irregularis</em> and misidentified as <em>Glomus intraradices</em><a reveal-id="dsx051-B11" data-open="dsx051-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx051-B12" data-open="dsx051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>) showed the existence of conventional homokaryotic/dikaryotic patterns in this species that drive mating in sexual fungi.<a reveal-id="dsx051-B13" data-open="dsx051-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx051-B14" data-open="dsx051-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>To date, AMF taxonomy has been based on spore morphology (presence/absence of several contrasting layers, textures and shape) and analyses of ribosomal ribonucleic acid genes (rRNA).<a reveal-id="dsx051-B15" data-open="dsx051-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx051-B16" data-open="dsx051-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Using the small subunit rRNA sequences (SSU), AMF were included in a monophyletic phylum called Glomeromycota,<a reveal-id="dsx051-B17" data-open="dsx051-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> but recent phylogenomic data led to the erection of a new phylum called Mucoromycota, in which AMF now have their own subphyla, the Glomeromycotina.<a reveal-id="dsx051-B18" data-open="dsx051-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> A total of 288 AMF species have been described, of which 60% were sequenced for at least one of ribosomal marker.<a reveal-id="dsx051-B19" data-open="dsx051-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> The taxonomy of this group experience frequent reshufflings of nomenclature amendments.<a reveal-id="dsx051-B20" data-open="dsx051-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx051-B21" data-open="dsx051-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> To alleviate this, it was proposed that AMF taxonomy should move towards a functional era<a reveal-id="dsx051-B22" data-open="dsx051-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> or that phylogenomics should be used to elucidate their clade affiliations.<a reveal-id="dsx051-B19" data-open="dsx051-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p><p>Advances in sequencing technologies have geared us towards a functional and genome-based Glomeromycotina research, as evidenced by the recent publication of transcriptomes<a reveal-id="dsx051-B23 dsx051-B24 dsx051-B25 dsx051-B26 dsx051-B27" data-open="dsx051-B23 dsx051-B24 dsx051-B25 dsx051-B26 dsx051-B27" class="link link-ref link-reveal xref-bibr"><sup>23–27</sup></a> and genome drafts<a reveal-id="dsx051-B28" data-open="dsx051-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsx051-B29" data-open="dsx051-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> from two model AMF genera, namely those of <em>R. irregulare and R. clarus</em>, and those of sister-species <em>G. rosea</em> and <em>G. margarita</em>. Available data have provided essential information on AMF symbiosis, revealing conserved features for obligate biotrophy, effector proteins and genes involved in their molecular interactions with hosts and endosymbiotic bacteria. However, coding sequence data from hundreds of pot-cultured AMF species is virtually non-existent.</p><p>The goal of our study is to acquire primary functional and phylogenetic information from quiescent spores of poorly studied AMF. To this end, we used an approach originally designed for single cell transcriptomics.<a reveal-id="dsx051-B30" data-open="dsx051-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> In principle, this methodology could generate coding sequence information from minute amounts of RNA extracted from quiescent spores cultured in pots, while simultaneously leaving-out most of the bacterial contamination.</p><p>We show that this method can lead to successful acquisition of large-scale coding data from spores of eight AMF genera (<em>Claroideoglomus</em>, <em>Funneliformis</em>, <em>Paraglomus</em>, <em>Ambispora</em>, <em>Acaulospora</em>, <em>Diversispora</em>, <em>Dentiscutata</em> and <em>Scutellospora</em>) using ultra-low amounts of RNA. This data provides a first insight into the functional diversity of species for which no coding sequence is available, and is readily usable for mapping, orthology assignment and phylogenetic analyses. Our data compares favourably against reference transcriptomes generated from <em>in vitro</em> cultured species, suggesting that this method could be used for comparing expression levels among different AMF life-stages or across spatial locations within cultures, including <em>in plantae</em>.</p>                            <h2 scrollto-destination=115733967 id="115733967" class="section-title" >Materials and methods</h2>
                            <h3 scrollto-destination=115733968 id="115733968" class="section-title" >Isolation and preparation of the biological material</h3>
<p>In this study, quiescent spores from a total of nine AMF species were harvested, including <em>Rhizoglomus irregulare</em> (DAOM-234181)<em>, Funneliformis mosseae</em> (DAOM-236685)<em>, Acaulospora morrowiae</em> (INVAM-CR315B)<em>, Diversispora versiforme</em> (INVAM-W475-40)<em>, Scutellospora calospora</em> (INVAM-IL209)<em>, Racocetra castanea</em> (BEG-1)<em>, Paraglomus brasilianum</em> (DAOM-240472) and <em>Ambispora leptoticha</em> (INVAM-JA116). Spores were thoroughly identified and maintained in Agriculture and Agri-food Canada (AAC) Glomeromycotina <em>in vivo</em> pot culture collections. This fungal collection is commonly used by international researchers interested in all aspects of AMF biology, and are regularly checked for contamination using both morphological and molecular techniques (i.e. small subunit of the rRNA gene, SSU; H<sup>+</sup> ATPAses).</p><p>For all species, only quiescent spores with intact morphology and lipid content were used for downstream analyses using an inverted microscope. Spores were originally collected by using a sieve cascade of descending size ranging from 500 µm to 63 µm. Lowest fractions were extracted, placed in a 50 ml Falcon tube with the addition of 50% w/w sucrose solution. After centrifugation at 5,000 g for 5 min, supernatant was collected for single spore isolation, and species identification was confirmed using morphological descriptions detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figures S1.1–S1.9</a></span>, and by sequencing the SSU using DNA isolated from random spores. <em>Claroideoglomus claroideum (DAOM-234280)</em> was obtained from an <em>in vitro</em> culture on M medium<a reveal-id="dsx051-B31" data-open="dsx051-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> in symbiosis with Agrobacterium-transformed <em>Daucus carota</em> roots. Spore surface sterilization was performed three times for each spore using successive baths of Chloramine-T 2% Tween 20 for 2 min and sterile distilled water for 1 min. Spores were incubated overnight in a solution of streptomycin 0.2 mg/ml and gentamycin 0.1 mg/ml and washed in a final bath of sterile distilled water. Sterilized spores were stored at 4°C.</p>                            <h3 scrollto-destination=115733971 id="115733971" class="section-title" >RNA extraction, quantification, and cDNA synthesis</h3>
<p>RNA was extracted using the NucleoSpin RNA XS kit (Macherey-Nagel, Germany) under a sterilized laminar flow hood. Spores were crushed in sterile nuclease-free 1.5 ml Eppendorf tubes with a micro-pestle and the RNA isolation step followed manufacturer’s recommendations. The cDNA was produced using the SMARTer Ultra Low Input RNA Kit for sequencing v4 used for single cell transcriptomic (SCT) (Takara Bio USA Inc.) following manufacturer’s recommendations. Adapters were used as template for cDNA synthesis and downstream PCR for the cDNA amplification. Two positive controls (Diluted Control RNA) provided with the kit and one negative control (no sample) were performed. The resulting cDNA was purified using the PCR-Purification kit (QIAGEN), and quantification and quality assessment was performed using a Qbit 2.0 fluorimeter with the dsDNA HS (High Sensitivity) Assay Kit.</p>                            <h3 scrollto-destination=115733973 id="115733973" class="section-title" >cDNA illumina sequencing and reads assembly</h3>
<p>The cDNA of all species was processed using the Nextera-XT DNA library preparation kit (Illumina). The resulting libraries were quality checked using a high-throughput bioanalyzer (Caliper LabChip GX instrument), and sequenced using one full lane of Illumina HiSeq 2500 instrument, using the High-Output V4 mode and paired-reads 2x125 bp (Fasteris SA, Switzerland). The generated paired-end reads were trimmed using the TrimGalore v.0.4.1 software with a Phred score cut-off of 20 and TruSeq Illumina adapter sequences were removed with CutAdapt v.1.8.3. Trimmed reads were assembled <em>de novo</em> using Trinity v.2.1.1.<a reveal-id="dsx051-B32" data-open="dsx051-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> A genome-guided assembly was performed for <em>R. irregulare</em> reads obtained by SCT. In this case, paired-end reads were mapped back against the reference genome assembly and all the CDS, respectively, using the Burrows–Wheeler alignment tool (BWA v0.7.10),<a reveal-id="dsx051-B33" data-open="dsx051-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with the BWA-MEM algorithm. SAMtools v0.1.19<a reveal-id="dsx051-B34" data-open="dsx051-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was used to convert SAM files into sorted BAM. The assembly results and statistics are available in supporting information (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Raw reads are deposited in GenBank (accession number SRX2583204-SRX2583220). Assemblies and annotations are available at <a class="link link-uri" href="https://github.com/zygolife/AMF_Phylogenomics" target="_blank">https://github.com/zygolife/AMF_Phylogenomics</a> (14 November 2017, date last accessed).</p>                            <h3 scrollto-destination=115733975 id="115733975" class="section-title" >Functional annotation pipeline</h3>
<p>SCT data and reference transcriptomes of <em>R. irregulare</em> and <em>G. rosea</em><a reveal-id="dsx051-B23" data-open="dsx051-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsx051-B25" data-open="dsx051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> annotated using the Trinotate v.2.0.2 annotation pipeline. The pipeline makes use of a wide range of algorithms, methods and databases for functional annotation. These include blastx and blastp<a reveal-id="dsx051-B35" data-open="dsx051-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> against UniRef and SwissProt databases, HMMER3,<a reveal-id="dsx051-B36" data-open="dsx051-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> the Pfam database,<a reveal-id="dsx051-B37" data-open="dsx051-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> SignalP<a reveal-id="dsx051-B38" data-open="dsx051-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> tmHMM,<a reveal-id="dsx051-B39" data-open="dsx051-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> eggnog, GO<a reveal-id="dsx051-B40" data-open="dsx051-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and KEGG.<a reveal-id="dsx051-B41" data-open="dsx051-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Non-redundant virtual transcripts (NRVTs) for each strain were obtained by filtering the predicted Trinity genes for unique occurrences using the text manipulation tool of the Galaxy webserver.<a reveal-id="dsx051-B42" data-open="dsx051-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The annotated data set were filtered by separating NRVTs into taxonomic-affiliated groups (i.e. Glomeromycotina, Other Fungi, Other Eukaryotes, Bacteria, Archaea, Plantae and Virus). The Glomeromycotina NRVTs predicted GO-terms were used as input in the Web Gene Ontology Annotation Plot (WEGO)<a reveal-id="dsx051-B43" data-open="dsx051-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> to obtain a hierarchical histogram of the GO ontology related functions. All taxonomic-sorted NRVTs were fed into the TransDecoder v.3.0.1 software to predict their ORFs (&gt;100 aa). ORFs were used as input in the webMGA webserver<a reveal-id="dsx051-B44" data-open="dsx051-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> to obtain KOG classes’ functions and protein domain families.</p>                            <h3 scrollto-destination=115733977 id="115733977" class="section-title" >Orthology clustering analysis</h3>
<p>Orthologous affiliation and shared functionality of NRVTs was obtained using FastOrtho, an OrthoMCL-based program.<a reveal-id="dsx051-B45" data-open="dsx051-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Transdecoder v.3.0.1 predicted protein sequences (&gt;100 aa) of four NRVT classes (Glomeromycotina-, Unknown-, Other Eukaryotes-, Prokaryotes-annotated) were used as inputs with an <em>e</em>-value cut-off of 1<em>e</em>−05. The output was filtered with the Galaxy webtool<a reveal-id="dsx051-B42" data-open="dsx051-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> to identify orthogroups shared among AMF strains and the number of genes present in these groups. BUSCO analyses (v.2.0) were performed on the Glomeromycota-annotated NRVTs, orthologous shared Unknown- and Other Eukaryotes-annotated NRVTs using the fungi_odb9 database and the species <em>Rhizopus oryzae</em> (now known as <em>R. delemar</em>) as reference. In total, 290 core orthologues were assessed for their presence in each transcriptome, respectively.</p>                            <h3 scrollto-destination=115733979 id="115733979" class="section-title" >RNA extraction for qPCR</h3>
<p>Quantitative real-time PCR (qPCR) was used to verify the expression of random regions of the transcriptome obtained using ultra-low input RNA methods. Expression of each region was measured in three biological replicates of <em>R. irregulare</em> consisting of cDNA produced using RNA from approximately 250 spores. Total RNA was extracted using the Qiagen RNeasy Plant Mini Kit (Qiagen, Venio, Netherlands) and DNA contamination was removed using the RapidOut DNA Removal Kit (ThermoFisher Scientific). RNA was subjected to RT-PCR using the iScript cDNA synthesis kit (Bio-Rad Laboratories, Hercules, CA, USA) following manufacturer’s recommendation. Six regions of the AMF transcriptome were selected (6230, 8946, 1110, 9312, 910 and α-tubulin) for analysis using qPCR and corresponding primers were designed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Real-time PCR reactions were carried out in a CFX 96 thermal cycler (Bio-Rad Laboratories) in technical duplicates and consisted of 4.5 µl of H2O, 1 µl each of forward and reverse primers, 7.5 µL of SsoFast 2X master mix (Bio-Rad Laboratories) and 1 µl of undiluted cDNA. The qPCR protocol consisted of an initial incubation at 95°C for 30 s followed by 40 cycles of 95°C for 5 s and 59.3°C for 5 s and a final melt curve from 60 to 95°C with 0.5°C increments. The primer efficiency set was determined using a serial dilution, temperature gradient and efficiency between 90% and 110%.</p>                            <h3 scrollto-destination=115733981 id="115733981" class="section-title" >Phylogenomic analysis</h3>
<p>A phylogenomic pipeline was applied to a set of 434 conserved genes.<a reveal-id="dsx051-B18" data-open="dsx051-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The 40 species selected include the 8 newly sequenced AMF species, 2 previously published references of <em>R. irregulare</em> and <em>G. rosea</em>, 8 other Mucoromycota representatives, 8 Dikarya fungi, 7 Zoopagomycota, four Chytridiomycota, 2 Blastocladiomycota and 1 Cryptomycota (<em>Rozella</em>). For AMF, the following number of partial sequences were recovered as best reciprocal hit, an approach that was shown to often outperformed projects with more complex algorithms including OrthoMCL,<a reveal-id="dsx051-B46" data-open="dsx051-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> to an hmmsearch reflecting the relative completeness of these transcriptomes for these query proteins: <em>Acaulospora morrowiae</em>, 337 genes; <em>Ambispora leptoticha</em>, 206 genes; <em>Claroideoglomus_claroideum</em>, 313 genes; <em>Diversispora versiforme</em>, 222 genes; <em>Funneliformis mosseae</em>, 426 genes; <em>Gigaspora rosea</em>, 385 genes; <em>Paraglomus occultum</em>, 334 genes; <em>Racocetra castanea</em>, 428; <em>Scutellospora calospora</em>, 427 genes; <em>Rhizophagus irregularis</em>, 423 genes.</p><p>The tree was rooted with the choanoflagellate <em>Monosiga brevicolis</em>. The pipeline, including all associated data is archived in the github repository <a class="link link-uri" href="https://github.com/zygolife/AMF_Phylogenomics" target="_blank">https://github.com/zygolife/AMF_Phylogenomics</a> (14 November 2017, date last accessed). The approach uses HMMER3<a reveal-id="dsx051-B36" data-open="dsx051-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> to search the predicted proteins of each strain with profile Hidden Markov Models (HMMs) constructed from a set of conserved markers denoted ‘JGI_1086’ (<a class="link link-uri" href="https://github.com/zygolife/AMF_Phylogenomics" target="_blank">https://github.com/zygolife/AMF_Phylogenomics</a> (14 November 2017, date last accessed)). A table of the best protein match for each HMM query in each proteome is recorded. A multiple sequence alignment (MSA) is constructed for each marker gene with the single best copy protein from each strain or omitted if no sequence is of sufficient similarity. These MSAs were trimmed with trimAl<a reveal-id="dsx051-B47" data-open="dsx051-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> using the automated1 option and requiring all sequences to match at least 50% of residues with an overlap with at least 60% of the rest of the sequences (-resoverlap 0.50 -seqoverlap 60). The resulting individual protein alignments were concatenated into a single alignment with one sequence per species. A bootstrapped maximum likelihood phylogenetic tree was inferred from this alignment using RAxML v8.2.8<a reveal-id="dsx051-B48" data-open="dsx051-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> with rapid bootstrapping (-f a -o Mbre -m PROTGAMMALG -s AMF.2016_Oct_06.JGI1086.41sp.fasaln -n Standard.AMF.2016_Oct_06.JGI1086.41sp -N autoMRE –d). The resolved tree was visualized in FigTree (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">http://tree.bio.ed.ac.uk/software/figtree/</a> (14 November 2017, date last accessed)). In parallel, the CAT+Γ4 model of amino acid substitution implemented in PhyloBayes 3 was also used.<a reveal-id="dsx051-B49" data-open="dsx051-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> In this case, two concurrent chains were run in parallel and were terminated manually after 1,116 cycles. Both methodologies varied only in their placement of Chytidiomycota and Blastocladiomycota as the most basal fungal phylum.</p>                            <h2 scrollto-destination=115733984 id="115733984" class="section-title" >Results and discussion</h2>
                            <h3 scrollto-destination=115733985 id="115733985" class="section-title" >Acquisition of transcriptome data from poorly studied AMF species</h3>
<p>Initial attempts to obtain sufficient RNA from 1 to 20 single AMF spores resulted in unsuccessful cDNA amplifications. However, subsequent RNA extractions from more spores (<em>n</em> = 22–175, <a reveal-id="dsx051-F1" data-open="dsx051-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) resulted in successful extractions and sequencing. The size of spores appeared to play a major role in the amount of RNA extracted, with taxa in the Gigasporaceae harbouring more RNAs than relatives with smaller spores (for variation in spore shape and size see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span>). The RNA extraction ranged from a mere 3 ng for <em>P. brasilianum</em> to a maximum of 720 ng for <em>S. calospora</em>. Assembly statistics vary substantially among the data sets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), but high RNA yield did not necessarily result in better assemblies. The lowest and highest number of transcripts and gene predictions were found for the <em>S. calospora</em> and <em>D. versiforme</em> related data, respectively. These also show the highest and lowest assembly sizes and predictions, while GC% was found to be lowest for transcriptome isolated from <em>C. claroideum</em> spores (30%). SCT transcriptomes from <em>R. irregulare</em> (260 bp) and <em>S. calospora</em> (1,091 bp) had the smallest and largest N<sub>50,</sub> respectively. </p>                        <a id="115733987" scrollto-destination="115733987"></a>
<div id="dsx051-F1" data-id="dsx051-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f1.png?Expires=2147483647&Signature=ykaW3sFNi3854Xe8TmNaml4zkT4fh~jyftFGt9CAJs3jAhyU2v4pHil5pTuWIr4g4EWpPUPU8WjXlNI8xbw7SVGlEzMwvTL~r9h-zYDx5FohsFbwfz-K3B2mPcnww7Qzd98kW5R~E04ph1sP3HNEIg-y7tIv6gfRJ2DItnCNKrgoyEBAlm0Lhz80EQgqCy3sOOEUSFLPM4qaYse7ndg7QyTqXAill4mTXdvQK66DjDALp-aCRQgLhiKz3Pgh8G4lW06~n1ysi7ne7cNEp4KlnfnY89pzGznxs5nlTFpXJ8JGdJoW1UMkUEjBm~DkaXEDlmG4RiQCmzfKbvLJz~lmGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary table of the growth conditions, characteristics, phylogenetic affiliations, collection identifiers and SCT protocol status of all the AMF species selected in this study. The phylogeny is based on the SSU (full)-5.8S-LSU rDNA subunit and is a representation of the tree published by Kruger et al.12 The circles correspond to successful cDNA amplification according to the range expected by the SMARTer Ultra Low Input RNA Kit for Sequencing v4 (Takara Bio USA Inc.)." path-from-xml="dsx051f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733987/dsx051f1.tif" path-from-xml="dsx051f1.tif" target="_blank">View large</a><a section="115733987" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f1.png?Expires=2147483647&Signature=fdh4O37vIDKR~op8-VNsK1UeU8FzaepVS8MUOzqILkVgDvE1M3En0uDRqVXzD0iNic3gzyigIXsap0A5jmX6dmlXnHb3TRKEufAt35XSwK57c6MsmrwEnb2z~dBHvn~t8Us9WjobH-C6vUHI2t3G3~pqyCjiT7I25T8yLiuskIZPrGAcen7y7G1uAu8BwMrDNKa--fkaT4PMbmkY1InywFFf4gK4~bcx8GNrS5VSx1By6rw2csXJP-z-NZQNygXd8IkgG3qp0z0SIlflZvHmlj5n7BYWYBN4gtWoeLdpi7DJQUZ8AQY3ELeoWmygx4SQXsMZgGNAf4VRuln6mg9SqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733987&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary table of the growth conditions, characteristics, phylogenetic affiliations, collection identifiers and SCT protocol status of all the AMF species selected in this study. The phylogeny is based on the SSU (full)-5.8S-LSU rDNA subunit and is a representation of the tree published by Kruger <em>et al.</em><a reveal-id="dsx051-B12" data-open="dsx051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The circles correspond to successful cDNA amplification according to the range expected by the SMARTer Ultra Low Input RNA Kit for Sequencing v4 (Takara Bio USA Inc.).</p></div></div><div content-id="dsx051-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f1.png?Expires=2147483647&Signature=ykaW3sFNi3854Xe8TmNaml4zkT4fh~jyftFGt9CAJs3jAhyU2v4pHil5pTuWIr4g4EWpPUPU8WjXlNI8xbw7SVGlEzMwvTL~r9h-zYDx5FohsFbwfz-K3B2mPcnww7Qzd98kW5R~E04ph1sP3HNEIg-y7tIv6gfRJ2DItnCNKrgoyEBAlm0Lhz80EQgqCy3sOOEUSFLPM4qaYse7ndg7QyTqXAill4mTXdvQK66DjDALp-aCRQgLhiKz3Pgh8G4lW06~n1ysi7ne7cNEp4KlnfnY89pzGznxs5nlTFpXJ8JGdJoW1UMkUEjBm~DkaXEDlmG4RiQCmzfKbvLJz~lmGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary table of the growth conditions, characteristics, phylogenetic affiliations, collection identifiers and SCT protocol status of all the AMF species selected in this study. The phylogeny is based on the SSU (full)-5.8S-LSU rDNA subunit and is a representation of the tree published by Kruger et al.12 The circles correspond to successful cDNA amplification according to the range expected by the SMARTer Ultra Low Input RNA Kit for Sequencing v4 (Takara Bio USA Inc.)." path-from-xml="dsx051f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733987/dsx051f1.tif" path-from-xml="dsx051f1.tif" target="_blank">View large</a><a section="115733987" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f1.png?Expires=2147483647&Signature=fdh4O37vIDKR~op8-VNsK1UeU8FzaepVS8MUOzqILkVgDvE1M3En0uDRqVXzD0iNic3gzyigIXsap0A5jmX6dmlXnHb3TRKEufAt35XSwK57c6MsmrwEnb2z~dBHvn~t8Us9WjobH-C6vUHI2t3G3~pqyCjiT7I25T8yLiuskIZPrGAcen7y7G1uAu8BwMrDNKa--fkaT4PMbmkY1InywFFf4gK4~bcx8GNrS5VSx1By6rw2csXJP-z-NZQNygXd8IkgG3qp0z0SIlflZvHmlj5n7BYWYBN4gtWoeLdpi7DJQUZ8AQY3ELeoWmygx4SQXsMZgGNAf4VRuln6mg9SqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733987&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary table of the growth conditions, characteristics, phylogenetic affiliations, collection identifiers and SCT protocol status of all the AMF species selected in this study. The phylogeny is based on the SSU (full)-5.8S-LSU rDNA subunit and is a representation of the tree published by Kruger <em>et al.</em><a reveal-id="dsx051-B12" data-open="dsx051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The circles correspond to successful cDNA amplification according to the range expected by the SMARTer Ultra Low Input RNA Kit for Sequencing v4 (Takara Bio USA Inc.).</p></div></div>                            <h3 scrollto-destination=115733988 id="115733988" class="section-title" >SCT from single spores vs conventional culture-based procedures</h3>
<p>To determine the suitability of the SCT approach for reference-based transcriptomics, we mapped our SCT-based ‘pot-cultured’ <em>R. irregulare</em> reads obtained from single spores against an available reference genome from this species.<a reveal-id="dsx051-B28" data-open="dsx051-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> This analysis compares the diversity of sequence reads obtained from a few quiescent spores to those previously obtained by others on the same species using much larger amounts of RNA and conditions (axenic growth, germinating spores, <em>in Plantae</em>, etc.).<a reveal-id="dsx051-B23" data-open="dsx051-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Our analyses showed that 86% of SCT reads from single spores map against the reference, 72% of which are properly paired with an average coverage of 42X. Moreover, 89.6% of known <em>R. irregularis</em> CDS<a reveal-id="dsx051-B28" data-open="dsx051-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> are mapped by our SCT reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) with coverage ranging from 11 to 50. Read mapping considerably improved our original <em>R. irregularis de novo</em> assembly (from 61,050 to 36,848 predicted genes).</p><p>When the longest isoform of each predicted gene is considered, the taxonomic distribution of NRVTs from SCT and the reference data set are largely comparable (<a reveal-id="dsx051-F2" data-open="dsx051-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The NRVTs and Glomeromycotina-annotated genes are 23,753 vs 25,906 and 51.8 vs 55.7% for the SCT and reference based data, respectively, and proportion of prokaryotic NRVTs is very low in both cases (0.8% and 1.3%). When transcriptomes are annotated using the same methodology, both conditions show identical GO-term functions with surprisingly comparable percentages for the vast majority of categories, around the 50% parity threshold (<a reveal-id="dsx051-F2" data-open="dsx051-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Mapping <em>S. calospora</em> and <em>R. castanea</em> SCT reads against an assembly and CDS from a similar Gigasporacea taxon (kindly provided by Christophe Roux) leads to similar findings. Overall, these analyses indicate that the SCT method does not introduce strong biases in functional annotation and read mapping, and could be applicable to transcriptomic analyses of other AMF stages, tissues etc. </p>                        <a id="115733991" scrollto-destination="115733991"></a>
<div id="dsx051-F2" data-id="dsx051-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f2.png?Expires=2147483647&Signature=aCZiOYZb1K2yPYL5d5sdpmz-xZwVZlTqb2qaXuulXzlZavC8hkdlhv3e1g33~aU~y5NHmAvlXp5z6GG4~CvAmuwsixjPL74foqFFRwe8z6UTpIn72wWk~yf5aL7semKI8atwHUJhC~m5HMrned3QJRw~mYksCSQhkwXF6oH7-Udiy5nJSz2UkkrMzW7sKayTBetHuN7B97p~I7uBN9rwQyZqvtKXqk8aWBoiLA7DjjhIx5sX~wTFPL305fRCbyzlIUz6TR0tTTkgZq0MMh7pShN5cmDDd95XXgR~8fFMueS7fkserdpB9vEJotQSRihWr3bhkqWKBRbXGQ3KfMy7Fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Taxonomic distribution comparison of the annotated NRVTs between the R. irregulare reference transcriptome28 and the genome-guided assembly of the in vivo R. irregulare transcriptome obtained from 150 spores. Regions corresponding to the percentage of glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs. (B). Histogram representing the % of GO-terms hierarchical functions (cellular component, molecular functions and biological process) associated to the glomeromycotina with GO-terms annotated-NRVTs. The comparison is made between the reference R. irregulare transcriptome23 and the genome-guided assembly of the in vivo R. irregulare transcriptome obtained by SCT from 150 spores." path-from-xml="dsx051f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733991/dsx051f2.tif" path-from-xml="dsx051f2.tif" target="_blank">View large</a><a section="115733991" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f2.png?Expires=2147483647&Signature=kABilC08-SfmmWhPuh0j5oAM-FqpeWLRlvanSMSyEEuDIK8tA2wdBjV23tQmAiycok4dx3ATIaMbYkPhvTWDMgdnzDM9XE2vGHEBkgccjBBS-T7rFVUeQVxorM5cI3iOfVznoJB-juBkI2QZEsGPXVxJZRv590mGwmWYitImWunOetk8rBaVEyt4tz547rZHVgdfnxCLyp938~I3E4wuDDMSfGOka6k2KLg8~90rOAho4qVgiS7Fpl76B67r3fQSWitoVg6F5ypQ391TceYAnrF76h-gKPx2Tq5eRqey8nXJio2jbJAIRsmFTZVKhiJGgZIIkV9PEcUk9gKUWuQ0UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733991&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Taxonomic distribution comparison of the annotated NRVTs between the <em>R. irregulare</em> reference transcriptome<a reveal-id="dsx051-B28" data-open="dsx051-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and the genome-guided assembly of the <em>in vivo R. irregulare</em> transcriptome obtained from 150 spores. Regions corresponding to the percentage of glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs. (B<strong>)</strong>. Histogram representing the % of GO-terms hierarchical functions (cellular component, molecular functions and biological process) associated to the glomeromycotina with GO-terms annotated-NRVTs. The comparison is made between the reference <em>R. irregulare</em> transcriptome<a reveal-id="dsx051-B23" data-open="dsx051-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and the genome-guided assembly of the <em>in vivo R. irregulare</em> transcriptome obtained by SCT from 150 spores.</p></div></div><div content-id="dsx051-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f2.png?Expires=2147483647&Signature=aCZiOYZb1K2yPYL5d5sdpmz-xZwVZlTqb2qaXuulXzlZavC8hkdlhv3e1g33~aU~y5NHmAvlXp5z6GG4~CvAmuwsixjPL74foqFFRwe8z6UTpIn72wWk~yf5aL7semKI8atwHUJhC~m5HMrned3QJRw~mYksCSQhkwXF6oH7-Udiy5nJSz2UkkrMzW7sKayTBetHuN7B97p~I7uBN9rwQyZqvtKXqk8aWBoiLA7DjjhIx5sX~wTFPL305fRCbyzlIUz6TR0tTTkgZq0MMh7pShN5cmDDd95XXgR~8fFMueS7fkserdpB9vEJotQSRihWr3bhkqWKBRbXGQ3KfMy7Fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Taxonomic distribution comparison of the annotated NRVTs between the R. irregulare reference transcriptome28 and the genome-guided assembly of the in vivo R. irregulare transcriptome obtained from 150 spores. Regions corresponding to the percentage of glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs. (B). Histogram representing the % of GO-terms hierarchical functions (cellular component, molecular functions and biological process) associated to the glomeromycotina with GO-terms annotated-NRVTs. The comparison is made between the reference R. irregulare transcriptome23 and the genome-guided assembly of the in vivo R. irregulare transcriptome obtained by SCT from 150 spores." path-from-xml="dsx051f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733991/dsx051f2.tif" path-from-xml="dsx051f2.tif" target="_blank">View large</a><a section="115733991" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f2.png?Expires=2147483647&Signature=kABilC08-SfmmWhPuh0j5oAM-FqpeWLRlvanSMSyEEuDIK8tA2wdBjV23tQmAiycok4dx3ATIaMbYkPhvTWDMgdnzDM9XE2vGHEBkgccjBBS-T7rFVUeQVxorM5cI3iOfVznoJB-juBkI2QZEsGPXVxJZRv590mGwmWYitImWunOetk8rBaVEyt4tz547rZHVgdfnxCLyp938~I3E4wuDDMSfGOka6k2KLg8~90rOAho4qVgiS7Fpl76B67r3fQSWitoVg6F5ypQ391TceYAnrF76h-gKPx2Tq5eRqey8nXJio2jbJAIRsmFTZVKhiJGgZIIkV9PEcUk9gKUWuQ0UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733991&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Taxonomic distribution comparison of the annotated NRVTs between the <em>R. irregulare</em> reference transcriptome<a reveal-id="dsx051-B28" data-open="dsx051-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and the genome-guided assembly of the <em>in vivo R. irregulare</em> transcriptome obtained from 150 spores. Regions corresponding to the percentage of glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs. (B<strong>)</strong>. Histogram representing the % of GO-terms hierarchical functions (cellular component, molecular functions and biological process) associated to the glomeromycotina with GO-terms annotated-NRVTs. The comparison is made between the reference <em>R. irregulare</em> transcriptome<a reveal-id="dsx051-B23" data-open="dsx051-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and the genome-guided assembly of the <em>in vivo R. irregulare</em> transcriptome obtained by SCT from 150 spores.</p></div></div>                            <h3 scrollto-destination=115733992 id="115733992" class="section-title" >Taxonomical and functional analysis of SCT-based transcriptomes</h3>
<p>Using identical pipelines and procedures, we compared the NRVTs taxonomic distribution and functional annotations of all SCT data with a reference transcriptome from <em>G. rosea.</em><a reveal-id="dsx051-B25" data-open="dsx051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> To avoid the analysis of potential contaminants, all functional analyses were performed exclusively on Glomeromycota orthologues shared among samples analysed in this study, and no evidence of cross-contamination among our samples was found by inspecting 100 random orthologue alignments. These analyses represent a first snapshot of the coding diversity of an AMF life-stage (the spore) typically seen as dormant, from eight species for which no coding sequence is currently available. However, this data does not represent an analysis of species-specific transcript due to lack of intra-sample replication. Reference <em>G. rosea</em> data used for comparisons comes from a variety of conditions (<em>in vitro, in vivo</em>, spores, extra-radical mycelium, etc.). The <em>G. margarita</em> transcriptome<a reveal-id="dsx051-B24" data-open="dsx051-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> was not analysed here to avoid redundancy—i.e. both taxa represent sister-species and data sets harbour essentially the same number of NRVTs; ≈ 80,000<a reveal-id="dsx051-B25" data-open="dsx051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>).</p><p>Overall taxonomic distributions of SCT transcriptomes showed remarkable similarities with that of <em>G. rosea</em>; particularly for the best assembly (<em>S. calospora, R. castanea, C. claroideum, F. mossea</em>) (<a reveal-id="dsx051-F3" data-open="dsx051-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). When only Glomeromycotina-annotated NRVTs are considered, the predicted proteins can be classified into 25 KOG functional classes. In support of recent genome analyses, the most represented KOG classes for all transcriptomes were the signal transduction mechanisms and post-translational modifications, with most represented molecular functions being binding and catalytic activity. The majority of biological processes related to GO-terms were for the cellular and metabolic process (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S4 and S5</a></span>). Unsurprisingly, most NRVTs have no known function and/or taxonomic affiliation. Prokaryotic NRVTs are abundant in four species (<em>Dve, Pbr, Ale</em> and <em>Amo</em>) (40.3, 18.9, 24 and 26.4%, respectively) (<a reveal-id="dsx051-F3" data-open="dsx051-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>), and many of these show high sequence similarity with known AMF or fungal prokaryotic endosymbionts,<a reveal-id="dsx051-B24" data-open="dsx051-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx051-B50" data-open="dsx051-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><a reveal-id="dsx051-B51" data-open="dsx051-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> especially to <em>Glomeribacter gigasporarum</em>. Most putative bacterial transcripts are involved in translation structure and biogenesis, as well as amino-acid and energy metabolism, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). </p>                        <a id="115733995" scrollto-destination="115733995"></a>
<div id="dsx051-F3" data-id="dsx051-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f3.png?Expires=2147483647&Signature=THzNz7wqB5HWpL82~icbWnnG8xa9xENec6RgA6CtNZ~7fWIuBPy5JfZKpq0nsI92BAUvO4d~T7ax2UuYzymmp9mKOqGJik97~XyWBnlm-09ZRsTUtITr9xY98-FTW8LlVjOmvugZXT5O72BB2UQSsvJNTM1KqeWI35HP9ePCiUlAcHAB2ZIAHsPHs2TA1OzWbCI3I53ZG0TMQTM6ZbHW~64Mg5gmq1Eax4EOF7DEpJCJG9sfOWtRTm6hbNKrWR-EBmnTQfivpaqruU3sNddflh-i1lahHgkU8vUJdXMlSROD2b5g1kXqwch2dINDiEaj8Q1er--ZlRyeoCM~MuWxGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Taxonomic distribution and comparison of the annotated NRVTs between all AMF SCT transcriptomes and the G. rosea reference transcriptome.25 Regions corresponding to the percentage of Glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs." path-from-xml="dsx051f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733995/dsx051f3.tif" path-from-xml="dsx051f3.tif" target="_blank">View large</a><a section="115733995" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f3.png?Expires=2147483647&Signature=iZAAlOtQY5Q~DSwq1Tu6qHdZC76vdKZsIfidtsSLX2a7R5fTaYFINikeble7qzb~4-g3DuYJwTdCXFU25vgQ3a9mrnTdglSO8lm0fTlyFns1ez6TbFNdKl~qLeIH~ZNEYc7FUDuxh-bEe-8fKQg0SMVIpeSKt0XC6wsVEqiQ1nqUUI7wEdUQOzbfF87q1tAWEWdW9g0Y2TxiRWI6bL-3JPxvbcS80Cho6j4l2al-Xyb3epzUki~OfyfWOEM4IIweeUTsA-IUBiE4FvfQKBiFfOmXdU4NLLLhmGy~-5tXEZ8b6GIBwlASbxcmkHFa4Squqt00rmFM4yP3s9ZB6efaww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733995&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Taxonomic distribution and comparison of the annotated NRVTs between all AMF SCT transcriptomes and the <em>G. rosea</em> reference transcriptome.<a reveal-id="dsx051-B25" data-open="dsx051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Regions corresponding to the percentage of Glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs.</p></div></div><div content-id="dsx051-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f3.png?Expires=2147483647&Signature=THzNz7wqB5HWpL82~icbWnnG8xa9xENec6RgA6CtNZ~7fWIuBPy5JfZKpq0nsI92BAUvO4d~T7ax2UuYzymmp9mKOqGJik97~XyWBnlm-09ZRsTUtITr9xY98-FTW8LlVjOmvugZXT5O72BB2UQSsvJNTM1KqeWI35HP9ePCiUlAcHAB2ZIAHsPHs2TA1OzWbCI3I53ZG0TMQTM6ZbHW~64Mg5gmq1Eax4EOF7DEpJCJG9sfOWtRTm6hbNKrWR-EBmnTQfivpaqruU3sNddflh-i1lahHgkU8vUJdXMlSROD2b5g1kXqwch2dINDiEaj8Q1er--ZlRyeoCM~MuWxGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Taxonomic distribution and comparison of the annotated NRVTs between all AMF SCT transcriptomes and the G. rosea reference transcriptome.25 Regions corresponding to the percentage of Glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs." path-from-xml="dsx051f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733995/dsx051f3.tif" path-from-xml="dsx051f3.tif" target="_blank">View large</a><a section="115733995" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f3.png?Expires=2147483647&Signature=iZAAlOtQY5Q~DSwq1Tu6qHdZC76vdKZsIfidtsSLX2a7R5fTaYFINikeble7qzb~4-g3DuYJwTdCXFU25vgQ3a9mrnTdglSO8lm0fTlyFns1ez6TbFNdKl~qLeIH~ZNEYc7FUDuxh-bEe-8fKQg0SMVIpeSKt0XC6wsVEqiQ1nqUUI7wEdUQOzbfF87q1tAWEWdW9g0Y2TxiRWI6bL-3JPxvbcS80Cho6j4l2al-Xyb3epzUki~OfyfWOEM4IIweeUTsA-IUBiE4FvfQKBiFfOmXdU4NLLLhmGy~-5tXEZ8b6GIBwlASbxcmkHFa4Squqt00rmFM4yP3s9ZB6efaww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733995&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Taxonomic distribution and comparison of the annotated NRVTs between all AMF SCT transcriptomes and the <em>G. rosea</em> reference transcriptome.<a reveal-id="dsx051-B25" data-open="dsx051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Regions corresponding to the percentage of Glomeromycotina with GO-terms, glomeromycotina, other fungi, other eukaryotes, prokaryotes and unknown affiliation annotated NRVTs, respectively are shown. The numbers located at the end of the histogram bars represents the total number of NRVTs.</p></div></div>                            <h3 scrollto-destination=115733996 id="115733996" class="section-title" >Orthology analysis and SCT completeness</h3>
<p>To identify conservation across data set, an orthology clustering analysis was performed on the Glomeromycotina-, Unknown- and Eukaryote-annotated NRVTs that clustered within orthogroups (<a reveal-id="dsx051-F4" data-open="dsx051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S2</a></span>). The Glomeromycotina-annotated subgroups are surprisingly conserved among transcriptomes. Specifically, a total of 232 orthogroups were found to be shared by all transcriptomes, and the percentage of shared transcripts among SCT data sets ranged from 61.4% to 87.2%. The function of these shared AMF genes broadly reflects that of the Glomeromycotina-annotated NRVT data set. Orthogroup conservation also shows evidence of correlation with phylogenetic relationships (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The presence of 6 orthologues involved in different biological categories was further validated using RT-qPCRs procedures using RNA extracted from <em>R. irregularis</em> spores (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S3</a></span>). Interestingly, conservation across data sets extends to NRVTs with unknown functions and Eukaryotic homologues, although none of these were found to be shared by all 10 data sets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S2</a></span> and Table S8). </p>                        <a id="115733998" scrollto-destination="115733998"></a>
<div id="dsx051-F4" data-id="dsx051-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f4.png?Expires=2147483647&Signature=UpuYYSnXwAa5ik3YgOnQ6Xx-HZsOZC7UlmGIoXyPvdIwG-8lOpZ3j9IvC7fe0CAmqScWwZxK5MUKYBx2Y5EhBA72kaZnKdvStGLo8w0ZonyJXHrjv0blDg4zyuVmD5S-cVQwVOxN8EgojBnNyKq9HrMG5f3ElqxUonVoYo4eU-8Yei7G5Z4PfAil7~ZNhWnkjWsQmVpaogqmf7JjBMmnNARnIet0hSKLE~1ic01E5PBIpn739-3EJ-bAjAvJk7iwU6ZwhY0UDY48WE9jmMMPhOyTVhpahlccnjigqxrrnjinePf7zLB5dJg94gFXFb4MSfhj4WqUgQrzdWJatkE0Xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene orthology affiliations where each petal represents a single species. The total number of proteins included in the orthology analysis and the percentage of total shared protein is shown within the flower petals, along with the species identifier (Gro: Gigaspora rosea Reference; Rir: Rhizoglomus irregulare Reference; Ccl: Claroideoglomus claroideum; Fmo: Funneliformis mosseae; Ale: Ambispora leptoticha; Pbr: Paraglomus brasilianum; Amo: Acaulospora morrowiae; Dve: Diversispora versiforme; Rca: Racocetra castanea; Sca: Scutellospora calospora). The flower button shows the number of orthogroups shared among all species. The extruding spirals show the number of orthogroups shared between the individual species, where the coloured dots above the petals represent the starting point to follow the relationships, with the corresponding values always below the guiding line (towards the flower). (A) The flower shows the orthologous relationships for the Glomeromycotina-annotated proteins (&gt;100 aa) in a clockwise manner. (B) Orthologous protein shared % matrix—linked to the orthology flowers above them. Each dot varies in size proportionally to the percentage of shared proteins between individual species in a reciprocal fashion, depending on which side of the oblique lines, the value is read. (C). Distribution of high level functional annotations from glomeromycota-shared orthogroups. Bars represent orthogroups involved in ‘Cellular Processes and Signaling’, ‘Information Storage and Processing’, ‘Metabolism’ and those that are ‘Poorly Characterized’." path-from-xml="dsx051f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733998/dsx051f4.tif" path-from-xml="dsx051f4.tif" target="_blank">View large</a><a section="115733998" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f4.png?Expires=2147483647&Signature=tLq0AKYho99ZLDBo4lH4kPgHIGaHQCJRhTpEpqUpHqpdTav9yd4Xfbkwzx2JyIsgp8NNyyirUAnwdxN1H~RmurUzMH0GrGlruCuGgsEojDLQZwnEe~PYlD2dtPNf23xpBQI0vz5DDJqPd59EVEGeNVeRgrKbfwi8eGrn-crZvrQ5C4iaEXfTUfhbkmEj1bW~z9fYKQMmwgdhrIfD0I0Cd8a2xwVZnYWn5FqoNLEfgTBQpK-VzM24pvun3vu5ygjr6k7WICYL0A5-CgwHZeiT-uZ-UVzMuotuao~EtIYRwtobAHTqoFB45aDOCNyTRibWVF-dApBLPUnrDVFGnrieaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733998&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene orthology affiliations where each petal represents a single species. The total number of proteins included in the orthology analysis and the percentage of total shared protein is shown within the flower petals, along with the species identifier (<em>Gro</em>: <em>Gigaspora rosea</em> Reference; <em>Rir</em>: <em>Rhizoglomus irregulare</em> Reference; <em>Ccl</em>: <em>Claroideoglomus claroideum</em>; <em>Fmo</em>: <em>Funneliformis mosseae</em>; <em>Ale</em>: <em>Ambispora leptoticha</em>; <em>Pbr</em>: <em>Paraglomus brasilianum</em>; <em>Amo</em>: <em>Acaulospora morrowiae</em>; <em>Dve</em>: <em>Diversispora versiforme</em>; <em>Rca</em>: <em>Racocetra castanea</em>; <em>Sca</em>: <em>Scutellospora calospora</em>). The flower button shows the number of orthogroups shared among all species. The extruding spirals show the number of orthogroups shared between the individual species, where the coloured dots above the petals represent the starting point to follow the relationships, with the corresponding values always below the guiding line (towards the flower). (A) The flower shows the orthologous relationships for the Glomeromycotina-annotated proteins (&gt;100 aa) in a clockwise manner. (B) Orthologous protein shared % matrix—linked to the orthology flowers above them. Each dot varies in size proportionally to the percentage of shared proteins between individual species in a reciprocal fashion, depending on which side of the oblique lines, the value is read. (C). Distribution of high level functional annotations from glomeromycota-shared orthogroups. Bars represent orthogroups involved in ‘Cellular Processes and Signaling’, ‘Information Storage and Processing’, ‘Metabolism’ and those that are ‘Poorly Characterized’.</p></div></div><div content-id="dsx051-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f4.png?Expires=2147483647&Signature=UpuYYSnXwAa5ik3YgOnQ6Xx-HZsOZC7UlmGIoXyPvdIwG-8lOpZ3j9IvC7fe0CAmqScWwZxK5MUKYBx2Y5EhBA72kaZnKdvStGLo8w0ZonyJXHrjv0blDg4zyuVmD5S-cVQwVOxN8EgojBnNyKq9HrMG5f3ElqxUonVoYo4eU-8Yei7G5Z4PfAil7~ZNhWnkjWsQmVpaogqmf7JjBMmnNARnIet0hSKLE~1ic01E5PBIpn739-3EJ-bAjAvJk7iwU6ZwhY0UDY48WE9jmMMPhOyTVhpahlccnjigqxrrnjinePf7zLB5dJg94gFXFb4MSfhj4WqUgQrzdWJatkE0Xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene orthology affiliations where each petal represents a single species. The total number of proteins included in the orthology analysis and the percentage of total shared protein is shown within the flower petals, along with the species identifier (Gro: Gigaspora rosea Reference; Rir: Rhizoglomus irregulare Reference; Ccl: Claroideoglomus claroideum; Fmo: Funneliformis mosseae; Ale: Ambispora leptoticha; Pbr: Paraglomus brasilianum; Amo: Acaulospora morrowiae; Dve: Diversispora versiforme; Rca: Racocetra castanea; Sca: Scutellospora calospora). The flower button shows the number of orthogroups shared among all species. The extruding spirals show the number of orthogroups shared between the individual species, where the coloured dots above the petals represent the starting point to follow the relationships, with the corresponding values always below the guiding line (towards the flower). (A) The flower shows the orthologous relationships for the Glomeromycotina-annotated proteins (&gt;100 aa) in a clockwise manner. (B) Orthologous protein shared % matrix—linked to the orthology flowers above them. Each dot varies in size proportionally to the percentage of shared proteins between individual species in a reciprocal fashion, depending on which side of the oblique lines, the value is read. (C). Distribution of high level functional annotations from glomeromycota-shared orthogroups. Bars represent orthogroups involved in ‘Cellular Processes and Signaling’, ‘Information Storage and Processing’, ‘Metabolism’ and those that are ‘Poorly Characterized’." path-from-xml="dsx051f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733998/dsx051f4.tif" path-from-xml="dsx051f4.tif" target="_blank">View large</a><a section="115733998" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f4.png?Expires=2147483647&Signature=tLq0AKYho99ZLDBo4lH4kPgHIGaHQCJRhTpEpqUpHqpdTav9yd4Xfbkwzx2JyIsgp8NNyyirUAnwdxN1H~RmurUzMH0GrGlruCuGgsEojDLQZwnEe~PYlD2dtPNf23xpBQI0vz5DDJqPd59EVEGeNVeRgrKbfwi8eGrn-crZvrQ5C4iaEXfTUfhbkmEj1bW~z9fYKQMmwgdhrIfD0I0Cd8a2xwVZnYWn5FqoNLEfgTBQpK-VzM24pvun3vu5ygjr6k7WICYL0A5-CgwHZeiT-uZ-UVzMuotuao~EtIYRwtobAHTqoFB45aDOCNyTRibWVF-dApBLPUnrDVFGnrieaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733998&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene orthology affiliations where each petal represents a single species. The total number of proteins included in the orthology analysis and the percentage of total shared protein is shown within the flower petals, along with the species identifier (<em>Gro</em>: <em>Gigaspora rosea</em> Reference; <em>Rir</em>: <em>Rhizoglomus irregulare</em> Reference; <em>Ccl</em>: <em>Claroideoglomus claroideum</em>; <em>Fmo</em>: <em>Funneliformis mosseae</em>; <em>Ale</em>: <em>Ambispora leptoticha</em>; <em>Pbr</em>: <em>Paraglomus brasilianum</em>; <em>Amo</em>: <em>Acaulospora morrowiae</em>; <em>Dve</em>: <em>Diversispora versiforme</em>; <em>Rca</em>: <em>Racocetra castanea</em>; <em>Sca</em>: <em>Scutellospora calospora</em>). The flower button shows the number of orthogroups shared among all species. The extruding spirals show the number of orthogroups shared between the individual species, where the coloured dots above the petals represent the starting point to follow the relationships, with the corresponding values always below the guiding line (towards the flower). (A) The flower shows the orthologous relationships for the Glomeromycotina-annotated proteins (&gt;100 aa) in a clockwise manner. (B) Orthologous protein shared % matrix—linked to the orthology flowers above them. Each dot varies in size proportionally to the percentage of shared proteins between individual species in a reciprocal fashion, depending on which side of the oblique lines, the value is read. (C). Distribution of high level functional annotations from glomeromycota-shared orthogroups. Bars represent orthogroups involved in ‘Cellular Processes and Signaling’, ‘Information Storage and Processing’, ‘Metabolism’ and those that are ‘Poorly Characterized’.</p></div></div><p>BUSCO benchmark analyses, based on 290 core Eukaryotic fungal genes used by the program to infer genome completeness,<a reveal-id="dsx051-B52" data-open="dsx051-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> shows strong correlation between the N50 and the performance on the benchmark genome (<em>Rhizopus oryzae</em>, <a reveal-id="dsx051-F5" data-open="dsx051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Reference transcriptomes lack between 13.1% and 22.1% of genes, compared with 21.4–77.6% for the SCT samples. Generally, reference RNA-seq data are more complete, and less fragmented, with the exception of data from <em>S. castanea</em>. Each transcriptome contained between 126 and 1444 (<em>Amo</em> and <em>Ale</em>, respectively) unique glomeromycotan transcripts. Their AMF origin indicates that homologues of these genes are also present in other members of the phylum. The 30 most represented data set-specific families include homologues of known AMF genes (tyrosine kinases, extracellular proteins SEL-1, proteins containing BTB/POZ and Kelch domains, TRAP230 subunit hormone receptors, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). All these homologues are also found in publicly available AMF genome data. Importantly, our data also confirms the absence of most homologues of Glomeromycotina core genes (MGCGs; a term recently coined by Tang et al. 2016<a reveal-id="dsx051-B25" data-open="dsx051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). </p>                        <a id="115734000" scrollto-destination="115734000"></a>
<div id="dsx051-F5" data-id="dsx051-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f5.png?Expires=2147483647&Signature=mt7tuP0CTYwcBrBVb5JHjxaX4KOBkn~Y-YfemR0kHoUWYPcwKpBI4EgccATK4rFINt2KiedJSnjU9z3NK3wh7vU8tVizMsVuDiFBdKB7w9FGaQPnKZtutzc4BIL9Ppi7cAkjm4QYPHSXQt8CNj9W8LSZPRTKJGYrCFxGZ7EyfyrOobtXzKN1oUFeTjy8R-Hc06~sakLMBEl99UG2mE3PUPRd-yzyc8ECmiZE3EQd7zhytSP01MobXtbitUiDuj4t8WeAtQ6qNDqdV-SVtHL1ottra9QxL5m1GB6vc2vrGcpuH~~CYIXmhR~trnrKM~UZJ7ViXLD90AVaC12ArHJB-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transcriptome representativeness comparison among the AMF SCT data sets and the reference R. irregulare and G. rosea RNA-seq, performed using a BUSCO benchmark. In total, 290 core fungal genes were search across these data sets and evaluated for completeness, single-copy (S), duplication (D), fragmentation (F) and absence (M). The histogram representations shows the percentage of the genes (and number indicated above the bars) belonging in these categories, for each species." path-from-xml="dsx051f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115734000/dsx051f5.tif" path-from-xml="dsx051f5.tif" target="_blank">View large</a><a section="115734000" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f5.png?Expires=2147483647&Signature=ARB8SV5uTlfYE8OdL6nCFS6uFNi2E9ALqvDJ47sz8Nh5aLnr5MdbaqzFsPZ-RPAlYhjuoRHho6ZA5B8Jki2NfdrNxaUVa6W~dUAZzjgcqJl9W5SDDR2dd2AgKMuUP3oEYeWdfZUfPcq-XiIcV5Nx0Nn1fe7NpWWw8FQ~axj36oQgv15~SiC4-nuRYUgO-xsXhfAQhYazii5Ib16ofbrqX72C8821sxG~79fkv0F4eI-QtRp8VECjglqVyeRavo0NCZYdoQ0-UgNZhHi0CCA52fiwA7S4H5bNSQxZoQc2KKTRlFa8pw9aemf~FnAEwHLEHcc4BgjJokDg~C-vwPDsOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115734000&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transcriptome representativeness comparison among the AMF SCT data sets and the reference <em>R. irregulare</em> and <em>G. rosea</em> RNA-seq, performed using a BUSCO benchmark. In total, 290 core fungal genes were search across these data sets and evaluated for completeness, single-copy (S), duplication (D), fragmentation (F) and absence (M). The histogram representations shows the percentage of the genes (and number indicated above the bars) belonging in these categories, for each species.</p></div></div><div content-id="dsx051-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f5.png?Expires=2147483647&Signature=mt7tuP0CTYwcBrBVb5JHjxaX4KOBkn~Y-YfemR0kHoUWYPcwKpBI4EgccATK4rFINt2KiedJSnjU9z3NK3wh7vU8tVizMsVuDiFBdKB7w9FGaQPnKZtutzc4BIL9Ppi7cAkjm4QYPHSXQt8CNj9W8LSZPRTKJGYrCFxGZ7EyfyrOobtXzKN1oUFeTjy8R-Hc06~sakLMBEl99UG2mE3PUPRd-yzyc8ECmiZE3EQd7zhytSP01MobXtbitUiDuj4t8WeAtQ6qNDqdV-SVtHL1ottra9QxL5m1GB6vc2vrGcpuH~~CYIXmhR~trnrKM~UZJ7ViXLD90AVaC12ArHJB-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transcriptome representativeness comparison among the AMF SCT data sets and the reference R. irregulare and G. rosea RNA-seq, performed using a BUSCO benchmark. In total, 290 core fungal genes were search across these data sets and evaluated for completeness, single-copy (S), duplication (D), fragmentation (F) and absence (M). The histogram representations shows the percentage of the genes (and number indicated above the bars) belonging in these categories, for each species." path-from-xml="dsx051f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115734000/dsx051f5.tif" path-from-xml="dsx051f5.tif" target="_blank">View large</a><a section="115734000" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f5.png?Expires=2147483647&Signature=ARB8SV5uTlfYE8OdL6nCFS6uFNi2E9ALqvDJ47sz8Nh5aLnr5MdbaqzFsPZ-RPAlYhjuoRHho6ZA5B8Jki2NfdrNxaUVa6W~dUAZzjgcqJl9W5SDDR2dd2AgKMuUP3oEYeWdfZUfPcq-XiIcV5Nx0Nn1fe7NpWWw8FQ~axj36oQgv15~SiC4-nuRYUgO-xsXhfAQhYazii5Ib16ofbrqX72C8821sxG~79fkv0F4eI-QtRp8VECjglqVyeRavo0NCZYdoQ0-UgNZhHi0CCA52fiwA7S4H5bNSQxZoQc2KKTRlFa8pw9aemf~FnAEwHLEHcc4BgjJokDg~C-vwPDsOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115734000&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transcriptome representativeness comparison among the AMF SCT data sets and the reference <em>R. irregulare</em> and <em>G. rosea</em> RNA-seq, performed using a BUSCO benchmark. In total, 290 core fungal genes were search across these data sets and evaluated for completeness, single-copy (S), duplication (D), fragmentation (F) and absence (M). The histogram representations shows the percentage of the genes (and number indicated above the bars) belonging in these categories, for each species.</p></div></div>                            <h3 scrollto-destination=115734001 id="115734001" class="section-title" >Reconstruction of the AMF phylogeny using new transcriptome data</h3>
<p>Aligning 434 putative single-copy genes (139,586 sites) found in all SCT data and distant fungal relatives in the Zoopagomycota, Blastocladiomycota, Chytridiomycota, Cryptomycota resulted in a highly supported fungal phylogeny that is consistent with recently published results based on large phylogenomic data set (<a reveal-id="dsx051-F6" data-open="dsx051-F6" class="link link-reveal link-table xref-fig">Fig. 6</a><a reveal-id="dsx051-B18" data-open="dsx051-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a>) Monophyly of all recently proposed phyla is supported by both Maximum Likelihood and Bayesian methodologies. The Zoopagomycota is sister to the Dikarya and Mucoromycota clade. Within Mucoromyota, the Glomeromycotina is monophyletic and sister to the Mucoromycotina and Mortierellomycotina clade, though there is no strong support for their specific branching. </p>                        <a id="115734003" scrollto-destination="115734003"></a>
<div id="dsx051-F6" data-id="dsx051-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f6.png?Expires=2147483647&Signature=10WunFSQ92EkDDNKBrU-jHcyoI6d~aYlRILHaQoOvb792DnxbyyVpY2RoZYyfXkDa1a9l6G2vWBCXNssTywqX1o~IZeIC4Xml1JqKEZFlL-g4aAibuNXgzQ3I~ewE9wvv6iW4bxYsqghAiQMX6wbatPMV2GJK12Jl-QJmItZ-KDckbP~PoqJupLM6k6BthajWBwbw538ZXuVW8HtzqsIhXCbOBlZo8ZrnKJ4R4uq5PL-biIL4evFKT4FhaN7bFbbCEOkKlOVALMWMU3U96E2efnPZ198yv0B6tvpGcxQEPL5KrH6j-2NNh~uC-2gp7CUo8qxe-2GpJ1I4NlOIhZROw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic placement of the eight AMF families within Fungi. Bayesian phylogenetic tree of selected fungal representatives of the Kingdom Fungi is based on the concatenated alignment of 434 conserved orthologous proteins and more than 120 K trimmed amino acid positions. Numbers below or above branches correspond to Bayesian inference value obtained with PhyloBayes (two concurrent chains were run for 1,116 cycles using the CAT model) and bootstrap support values obtained using Maximum likelihood, respectively. The corresponding fungal phylum of each main clades are identified on the right side of the figure and is highlighted by the gray shaded area." path-from-xml="dsx051f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115734003/dsx051f6.tif" path-from-xml="dsx051f6.tif" target="_blank">View large</a><a section="115734003" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f6.png?Expires=2147483647&Signature=3aXmi26fSuwy8UhR4yRqx1Udi0j59hlMFFD7gQO0gc7G5bp4M9tCCaCISLRoYpdKZsopzVTbMm0~Ne2ybQW0PsEMEFGcE9k3WfmxD0zhe~ZUMXdp5K9pfwbt4Oe8eJtOg08hbalq0cc3T6vlpsm~s~y7YAjNkktIiILNHW7NE1AYGjVKHGkTdym5MowEoCiG-83NSkt9YIW-EtJiPoHgLI9BS3LO22jw2bLCOPWlpv68cJDdCzZIgn9xppzkIbbvVW5XfxIthf~ZW2PACb0FAFQzCYERP7JP-jzCrMI7bWbDZeolSlOvDyEdLb6uWq-bLMh7Shyx4TpkVN3J7fpmVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115734003&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic placement of the eight AMF families within Fungi. Bayesian phylogenetic tree of selected fungal representatives of the Kingdom Fungi is based on the concatenated alignment of 434 conserved orthologous proteins and more than 120 K trimmed amino acid positions. Numbers below or above branches correspond to Bayesian inference value obtained with PhyloBayes (two concurrent chains were run for 1,116 cycles using the CAT model) and bootstrap support values obtained using Maximum likelihood, respectively. The corresponding fungal phylum of each main clades are identified on the right side of the figure and is highlighted by the gray shaded area.</p></div></div><div content-id="dsx051-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/m_dsx051f6.png?Expires=2147483647&Signature=10WunFSQ92EkDDNKBrU-jHcyoI6d~aYlRILHaQoOvb792DnxbyyVpY2RoZYyfXkDa1a9l6G2vWBCXNssTywqX1o~IZeIC4Xml1JqKEZFlL-g4aAibuNXgzQ3I~ewE9wvv6iW4bxYsqghAiQMX6wbatPMV2GJK12Jl-QJmItZ-KDckbP~PoqJupLM6k6BthajWBwbw538ZXuVW8HtzqsIhXCbOBlZo8ZrnKJ4R4uq5PL-biIL4evFKT4FhaN7bFbbCEOkKlOVALMWMU3U96E2efnPZ198yv0B6tvpGcxQEPL5KrH6j-2NNh~uC-2gp7CUo8qxe-2GpJ1I4NlOIhZROw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic placement of the eight AMF families within Fungi. Bayesian phylogenetic tree of selected fungal representatives of the Kingdom Fungi is based on the concatenated alignment of 434 conserved orthologous proteins and more than 120 K trimmed amino acid positions. Numbers below or above branches correspond to Bayesian inference value obtained with PhyloBayes (two concurrent chains were run for 1,116 cycles using the CAT model) and bootstrap support values obtained using Maximum likelihood, respectively. The corresponding fungal phylum of each main clades are identified on the right side of the figure and is highlighted by the gray shaded area." path-from-xml="dsx051f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115734003/dsx051f6.tif" path-from-xml="dsx051f6.tif" target="_blank">View large</a><a section="115734003" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051f6.png?Expires=2147483647&Signature=3aXmi26fSuwy8UhR4yRqx1Udi0j59hlMFFD7gQO0gc7G5bp4M9tCCaCISLRoYpdKZsopzVTbMm0~Ne2ybQW0PsEMEFGcE9k3WfmxD0zhe~ZUMXdp5K9pfwbt4Oe8eJtOg08hbalq0cc3T6vlpsm~s~y7YAjNkktIiILNHW7NE1AYGjVKHGkTdym5MowEoCiG-83NSkt9YIW-EtJiPoHgLI9BS3LO22jw2bLCOPWlpv68cJDdCzZIgn9xppzkIbbvVW5XfxIthf~ZW2PACb0FAFQzCYERP7JP-jzCrMI7bWbDZeolSlOvDyEdLb6uWq-bLMh7Shyx4TpkVN3J7fpmVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115734003&ar=4683712&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx051f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic placement of the eight AMF families within Fungi. Bayesian phylogenetic tree of selected fungal representatives of the Kingdom Fungi is based on the concatenated alignment of 434 conserved orthologous proteins and more than 120 K trimmed amino acid positions. Numbers below or above branches correspond to Bayesian inference value obtained with PhyloBayes (two concurrent chains were run for 1,116 cycles using the CAT model) and bootstrap support values obtained using Maximum likelihood, respectively. The corresponding fungal phylum of each main clades are identified on the right side of the figure and is highlighted by the gray shaded area.</p></div></div><p>Within the Glomeromycotina, most known phylogenetic relationships based morphology and SSU trees are strongly supported. Specifically, our analyses confirms the basal position of the Ambisporaceae and Paraglomeraceae in the AMF tree, groups together members of the Gigasporaceae (<em>Scutellospora</em>, <em>Gigaspora</em>, <em>Racocentra</em>) and Glomeraceae (<em>Rhizophagus</em> and <em>Funneliformis</em>), and supports the evolutionary relationship between Acaulosporacea and Diversisporaceae.<a reveal-id="dsx051-B12" data-open="dsx051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The only conflict with past phylogenies is represented by <em>C. claroideum</em>, which clusters with <em>A. leptothica</em> and <em>P. brasiliensis</em> with strong support.</p>                            <h3 scrollto-destination=115734005 id="115734005" class="section-title" >SCT exposes the RNA diversity of overlooked AMF spores and new phylogenetic relationships within the glomeromycotina</h3>
<p>Our approach based on low-input RNA produced transcriptomes with high conservation in sequence and function across the AMF phylogeny, and revealed a diverse functional diversity expressed in life-stages that are supposedly dormant. We confirm that MGCGs orthologues involved in invertase activity (SUC2 homolog) are likely absent in this phylum. The absence of the invertase orthologues in 10 transcriptomes comforts the ‘Glomeromycotina core symbiotic concept’, which proposes that the host plant is responsible for sucrose production and hydrolysis.</p><p>Spores of all species express many copies of BTB/POZ and vprBP homolog domain, which are known facilitators of the ubiquitin-protein ligase complexes.<a reveal-id="dsx051-B53" data-open="dsx051-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> ‘Kelch domains’, which form general protein–protein interaction modules, are also very abundant in all transcriptomes.<a reveal-id="dsx051-B54" data-open="dsx051-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Spores also express many alpha amylase related proteins in all data sets, as well as several glycoside hydrolase, lipases and proteases. These possibly confer AMF the molecular toolkit to enzymatically breakdown complex carbohydrates (polysaccharides), multiple protein and lipid substrates, either from the environment, their host or cell energy reserves.<a reveal-id="dsx051-B55" data-open="dsx051-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsx051-B56" data-open="dsx051-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Our study also revealed thousands of ‘unknown’ glomeromycotan coding sequences shared among all transcriptomes. Their abundance and high conservation across data sets highlights the importance of this ‘uncategorized’ realm of genes for the AMF biology.</p><p>AMF spores also share a number of transcripts encoded by putative AMF symbionts, including <em>G. gigasporarum</em> and mollicutes-related endosymbionts (MREs). Their genomes harbour genes linked to their host metabolism, in the intricate molecular dialogue between AMF and their intra-cellular inhabitants; particularly the expression of AMF genes involved in growth, development and transport in its fungal host.<a reveal-id="dsx051-B24" data-open="dsx051-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx051-B51" data-open="dsx051-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsx051-B57" data-open="dsx051-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Our data shows that endosymbionts are transcriptionally active in AMF spores, expressing a variety of functions, including translation, amino-acid metabolism or energy production.</p><p>The multigene fungal phylogeny we report also show that acquisition of larger coding data set could result in the reorganization of certain AMF families. This is particularly true for the Claroideoglomeraceae, Ambisporaceae and Paraglomeraceae, which group together with full support in our analyses but not in rRNA based phylogenies.<a reveal-id="dsx051-B11" data-open="dsx051-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> It will be interesting to see if the addition of new taxa to this multi-gene alignment—e.g. <em>Geosiphon</em>, <em>Archeospora, Pacispora</em>—will reshuffle this novel evolutionary affiliation, as the genus <em>Claroideoglomus</em> has been typically associated with the Glomeraceae, though with much lower support. Our low-input RNA method could also represent a stepping stone for producing future solid and durable AMF classifications,<a reveal-id="dsx051-B19" data-open="dsx051-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and could help selecting suitable protein-coding markers that are representative of the global phylogenetic signal. Such data is crucial to enter a new era of AMF classification based on functionality, as proposed by Gamper <em>et al.</em><a reveal-id="dsx051-B22" data-open="dsx051-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p>                            <h3 scrollto-destination=115734010 id="115734010" class="section-title" >Ultra-low input RNA methods as a tool to deepen our understanding of AMF biology and evolution</h3>
<p>Here, we showed that the methodologies based on ultra-low input RNA represents reliable avenues to obtain large AMF coding sequence data and important functional insights from minute amounts of RNA extracted from spores of recalcitrant AMF species. These results are a solid foundation to build future knowledge on other species in the group, and deepen our understanding of this ecologically important group. The data we collected also revealed that spores from many under sampled AMF taxa express a wide panel of transcripts. Some of these could be ‘ready-made’ for germination and/or plant colonization, although it is still unclear whether all spores express these transcripts at significant levels for biological function.</p><p>From an evolutionary perspective, the wealth of data we provide uncovered phylogenetic relationships that were previously shadowed by a focus on single rRNA gene sequences. Within this context, this transcriptomics data opens the door to phylogenetic classifications that develops markers representative of the global classification and suited for functional ecological studies. In the future, the methods we used could prove useful to assess tissue-specific functionalities, and may offer further insights into the plant–fungus dialogue based on RNA-seq data. Indeed, we have shown that sequencing reads obtained from few spores results, in many cases, in information comparable to that obtained from <em>in vitro</em> cultures. If the trend holds, our method could be applied to study, for instance, how transcription changes along different portions of the mycelium or <em>in plantae</em>. Ultimately, such advances will be essential to better understand how AMF benefit the health of plants or terrestrial ecosystems; processes that are linked to crucial societal challenges such as climate change, sustainable management, pest invasion and food security.</p>                            <h2 scrollto-destination=115734013 id="115734013" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank anonymous reviewers for their helpful comments on a previous version of the manuscript, and Timea Marton for assistance with RNA extraction procedures.</p>                            <h3 scrollto-destination=115734015 id="115734015" class="section-title" >Funding</h3>
<p>This work was supported by the Discovery program from the Natural Sciences and Engineering Research Council of Canada (NSERC-Discovery) and an Early Researcher Award from the Ontario Ministry of Research and Innovation (ER13-09-190) to N.C. This work was partially supported by US NSF project grant ZyGoLife DEB-1441715 to J.E.S. and DEB-1441677 to T.J. Phylogenomic analyses were performed on the high performance computing resources at the Institute for Integrative Genome Biology at University of California-Riverside supported by NSF DBI-1429826 and NIH S10-OD016290. Many of the available genomes were generated under the 1,000 Fungal Genomes project at the Joint Genome Institute. Any opinions, findings, conclusions, or recommendations expressed in this publication are those of the author(s) and do not necessarily reflect the view of the National Science Foundation. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>                            <h3 scrollto-destination=115734017 id="115734017" class="section-title" >Accession numbers</h3>
<p>The raw sequencing reads have been deposited in GenBank under these accession number SRX2583204-SRX2583220. The assemblies and annotations are available through <a class="link link-uri" href="https://github.com/zygolife/AMF_Phylogenomics/tree/master/SCT_ALL_DATA" target="_blank">https://github.com/zygolife/AMF_Phylogenomics/tree/master/SCT_ALL_DATA</a> (14 November 2017, date last accessed).</p>                            <h3 scrollto-destination=115734019 id="115734019" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=115734021 id="115734021" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=115734023 id="115734023" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx051-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Read</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, <div class="source">Mycorrhizal Symbiosis</div> , <div class="edition">3rd edition</div> AP (Academic Press), Cambridge, Massachusetts.</div></div></div></div><div content-id="dsx051-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bonfante</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Genre</div> <div class="given-names">A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Mechanisms underlying beneficial plant–fungus interactions in mycorrhizal symbiosis</div>, <div class="source">Nat. Commun</div> ., <div class="volume">1</div>, <div class="fpage">1</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20underlying%20beneficial%20plant%E2%80%93fungus%20interactions%20in%20mycorrhizal%20symbiosis&amp;author=P.%20Bonfante&amp;author=A.%20Genre&amp;publication_year=2010&amp;journal=Nat.%20Commun&amp;volume=1&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms1046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms1046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms1046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20975674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Humphreys</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Franks</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Rees</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bidartondo</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Leake</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Beerling</div> <div class="given-names">D.J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Mutualistic mycorrhiza-like symbiosis in the most ancient group of land plants</div>, <div class="source">Nat. Commun</div> ., <div class="volume">1</div>, <div class="fpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutualistic%20mycorrhiza-like%20symbiosis%20in%20the%20most%20ancient%20group%20of%20land%20plants&amp;author=C.P.%20Humphreys&amp;author=P.J.%20Franks&amp;author=M.%20Rees&amp;author=M.I.%20Bidartondo&amp;author=J.R.%20Leake&amp;author=D.J.%20Beerling&amp;publication_year=2010&amp;journal=Nat.%20Commun&amp;volume=1&amp;pages=103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms1105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms1105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms1105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21045821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parniske</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Arbuscular mycorrhiza: the mother of plant root endosymbioses</div>, <div class="source">Nat. Rev. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">763</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arbuscular%20mycorrhiza%3A%20the%20mother%20of%20plant%20root%20endosymbioses&amp;author=M.%20Parniske&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Microbiol&amp;volume=6&amp;pages=763-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18794914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van der Heijden</div> <div class="given-names">M.G.A.</div></div>, <div class="name string-name"><div class="surname">Klironomos</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Ursic</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">1998</div>, 
<div class="article-title">Mycorrhizal fungal diversity determines plant biodiversity, ecosystem variability and productivity</div>, <div class="source">Nature</div> , <div class="volume">396</div>, <div class="fpage">69</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mycorrhizal%20fungal%20diversity%20determines%20plant%20biodiversity%2C%20ecosystem%20variability%20and%20productivity&amp;author=M.G.A.%20van%20der%20Heijden&amp;author=J.N.%20Klironomos&amp;author=M.%20Ursic&amp;publication_year=1998&amp;journal=Nature&amp;volume=396&amp;pages=69-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/23932" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F23932" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F23932"> </span></div> </div></div></div></div></div><div content-id="dsx051-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berruti</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lumini</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Balestrini</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bianciotto</div> <div class="given-names">V.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Arbuscular mycorrhizal fungi as natural biofertilizers: let’s benefit from past successes</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">1559</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arbuscular%20mycorrhizal%20fungi%20as%20natural%20biofertilizers%3A%20let%E2%80%99s%20benefit%20from%20past%20successes&amp;author=A.%20Berruti&amp;author=E.%20Lumini&amp;author=R.%20Balestrini&amp;author=V.%20Bianciotto&amp;publication_year=2015&amp;journal=Front.%20Microbiol&amp;volume=6&amp;pages=1559" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26834714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Young</div> <div class="given-names">J. P.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genome diversity in arbuscular mycorrhizal fungi</div>, <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">26</div>, <div class="fpage">113</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20diversity%20in%20arbuscular%20mycorrhizal%20fungi&amp;author=J.%20P.%20Young&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=26&amp;pages=113-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2015.06.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2015.06.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2015.06.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26190590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kuhn</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Hijri</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sanders</div> <div class="given-names">I.R.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Evidence for the evolution of multiple genomes in arbuscular mycorrhizal fungi</div>, <div class="source">Nature</div> , <div class="volume">414</div>, <div class="fpage">745</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20evolution%20of%20multiple%20genomes%20in%20arbuscular%20mycorrhizal%20fungi&amp;author=G.%20Kuhn&amp;author=M.%20Hijri&amp;author=I.R.%20Sanders&amp;publication_year=2001&amp;journal=Nature&amp;volume=414&amp;pages=745-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/414745a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F414745a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F414745a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11742398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hijri</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sanders</div> <div class="given-names">I.R.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Low gene copy number shows that arbuscular mycorrhizal fungi inherit genetically different nuclei</div>, <div class="source">Nature</div> , <div class="volume">433</div>, <div class="fpage">160</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low%20gene%20copy%20number%20shows%20that%20arbuscular%20mycorrhizal%20fungi%20inherit%20genetically%20different%20nuclei&amp;author=M.%20Hijri&amp;author=I.R.%20Sanders&amp;publication_year=2005&amp;journal=Nature&amp;volume=433&amp;pages=160-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15650740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pawlowska</div> <div class="given-names">T.E.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">J.W.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Organization of genetic variation in individuals of arbuscular mycorrhizal fungi</div>, <div class="source">Nature</div> , <div class="volume">427</div>, <div class="fpage">733</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organization%20of%20genetic%20variation%20in%20individuals%20of%20arbuscular%20mycorrhizal%20fungi&amp;author=T.E.%20Pawlowska&amp;author=J.W.%20Taylor&amp;publication_year=2004&amp;journal=Nature&amp;volume=427&amp;pages=733-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14973485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schussler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">C.</div></div></div> <div class="year">2010</div>, The Glomeromycota: a species list with new families. http//<a class="link link-uri" href="http://www.amf-phylogeny.com" target="_blank">www.amf-phylogeny.com</a> (14 November 2017, date last accessed).</div></div></div></div><div content-id="dsx051-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kruger</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kruger</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Stockinger</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Schussler</div> <div class="given-names">A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Phylogenetic reference data for systematics and phylotaxonomy of arbuscular mycorrhizal fungi from phylum to species level</div>, <div class="source">New Phytol</div> ., <div class="volume">193</div>, <div class="fpage">970</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20reference%20data%20for%20systematics%20and%20phylotaxonomy%20of%20arbuscular%20mycorrhizal%20fungi%20from%20phylum%20to%20species%20level&amp;author=M.%20Kruger&amp;author=C.%20Kruger&amp;author=C.%20Walker&amp;author=H.%20Stockinger&amp;author=A.%20Schussler&amp;publication_year=2012&amp;journal=New%20Phytol&amp;volume=193&amp;pages=970-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.03962.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03962.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.03962.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22150759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Corradi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Brachmann</div> <div class="given-names">A.</div></div></div> 
<div class="article-title">Fungal Mating in the Most Widespread Plant Symbionts?</div> <div class="source">Trends Plant Sci</div> ., <div class="volume">64</div>, <div class="fpage">175</div>–<div class="lpage">83</div>.</div></div></div></div><div content-id="dsx051-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ropars</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Toro</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Noel</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Evidence for the sexual origin of heterokaryosis in arbuscular mycorrhizal fungi</div>, <div class="source">Nat. Microbiol</div> ., <div class="volume">1</div>, <div class="fpage">16033.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20sexual%20origin%20of%20heterokaryosis%20in%20arbuscular%20mycorrhizal%20fungi&amp;author=J.%20Ropars&amp;author=K.S.%20Toro&amp;author=J.%20Noel&amp;publication_year=2016&amp;journal=Nat.%20Microbiol&amp;volume=1&amp;pages=16033." target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmicrobiol.2016.33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmicrobiol.2016.33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmicrobiol.2016.33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27572831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krüger</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stockinger</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Krüger</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Schüßler</div> <div class="given-names">A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">DNA-based species level detection of Glomeromycota: one PCR primer set for all arbuscular mycorrhizal fungi</div>, <div class="source">New Phytol</div> ., <div class="volume">183</div>, <div class="fpage">212</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-based%20species%20level%20detection%20of%20Glomeromycota%3A%20one%20PCR%20primer%20set%20for%20all%20arbuscular%20mycorrhizal%20fungi&amp;author=M.%20Kr%C3%BCger&amp;author=H.%20Stockinger&amp;author=C.%20Kr%C3%BCger&amp;author=A.%20Sch%C3%BC%C3%9Fler&amp;publication_year=2009&amp;journal=New%20Phytol&amp;volume=183&amp;pages=212-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2009.183.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2009.183.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2009.183.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19368665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Öpik</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Davison</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Moora</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zobel</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">DNA-based detection and identification of Glomeromycota: the virtual taxonomy of environmental sequences</div>, <div class="source">Botany</div> , <div class="volume">92</div>, <div class="fpage">135</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-based%20detection%20and%20identification%20of%20Glomeromycota%3A%20the%20virtual%20taxonomy%20of%20environmental%20sequences&amp;author=M.%20%C3%96pik&amp;author=J.%20Davison&amp;author=M.%20Moora&amp;author=M.%20Zobel&amp;publication_year=2013&amp;journal=Botany&amp;volume=92&amp;pages=135-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/cjb-2013-0110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fcjb-2013-0110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fcjb-2013-0110"> </span></div> </div></div></div></div></div><div content-id="dsx051-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schussler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Schwarzott</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">C.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">A new fungal phylum, the Glomeromycota: phylogeny and evolution</div>, <div class="source">Mycol. Res</div> ., <div class="volume">105</div>, <div class="fpage">1413</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20fungal%20phylum%2C%20the%20Glomeromycota%3A%20phylogeny%20and%20evolution&amp;author=A.%20Schussler&amp;author=D.%20Schwarzott&amp;author=C.%20Walker&amp;publication_year=2001&amp;journal=Mycol.%20Res&amp;volume=105&amp;pages=1413-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0953756201005196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0953756201005196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0953756201005196"> </span></div> </div></div></div></div></div><div content-id="dsx051-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Spatafora</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Benny</div> <div class="given-names">G.L.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A phylum-level phylogenetic classification of zygomycete fungi based on genome-scale data</div>, <div class="source">Mycologia</div> , <div class="volume">108</div>, <div class="fpage">1028</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylum-level%20phylogenetic%20classification%20of%20zygomycete%20fungi%20based%20on%20genome-scale%20data&amp;author=J.W.%20Spatafora&amp;author=Y.%20Chang&amp;author=G.L.%20Benny&amp;publication_year=2016&amp;journal=Mycologia&amp;volume=108&amp;pages=1028-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3852/16-042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3852%2F16-042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3852%2F16-042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27738200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Öpik</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Davison</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Uniting species- and community-oriented approaches to understand arbuscular mycorrhizal fungal diversity</div>, <div class="source">Fungal Ecol</div> ., <div class="volume">24</div>, <div class="fpage">106</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uniting%20species-%20and%20community-oriented%20approaches%20to%20understand%20arbuscular%20mycorrhizal%20fungal%20diversity&amp;author=M.%20%C3%96pik&amp;author=J.%20Davison&amp;publication_year=2016&amp;journal=Fungal%20Ecol&amp;volume=24&amp;pages=106-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.funeco.2016.07.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.funeco.2016.07.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.funeco.2016.07.005"> </span></div> </div></div></div></div></div><div content-id="dsx051-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oehl</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sieverding</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Palenzuela</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ineichen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Alves da Silva</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Advances in Glomeromycota taxonomy and classification</div>, <div class="source">IMA Fungus</div> , <div class="volume">2</div>, <div class="fpage">191</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Glomeromycota%20taxonomy%20and%20classification&amp;author=F.%20Oehl&amp;author=E.%20Sieverding&amp;author=J.%20Palenzuela&amp;author=K.%20Ineichen&amp;author=G.%20Alves%20da%20Silva&amp;publication_year=2011&amp;journal=IMA%20Fungus&amp;volume=2&amp;pages=191-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5598/imafungus.2011.02.02.10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5598%2Fimafungus.2011.02.02.10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5598%2Fimafungus.2011.02.02.10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22679604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Redecker</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Schussler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Stockinger</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sturmer</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Morton</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">An evidence-based consensus for the classification of arbuscular mycorrhizal fungi (Glomeromycota)</div>, <div class="source">Mycorrhiza</div> , <div class="volume">23</div>, <div class="fpage">515</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evidence-based%20consensus%20for%20the%20classification%20of%20arbuscular%20mycorrhizal%20fungi%20(Glomeromycota)&amp;author=D.%20Redecker&amp;author=A.%20Schussler&amp;author=H.%20Stockinger&amp;author=S.L.%20Sturmer&amp;author=J.B.%20Morton&amp;author=C.%20Walker&amp;publication_year=2013&amp;journal=Mycorrhiza&amp;volume=23&amp;pages=515-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00572-013-0486-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00572-013-0486-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00572-013-0486-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23558516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gamper</div> <div class="given-names">H.A.</div></div>, <div class="name string-name"><div class="surname">van der Heijden</div> <div class="given-names">M.G.A.</div></div>, <div class="name string-name"><div class="surname">Kowalchuk</div> <div class="given-names">G.A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Molecular trait indicators: moving beyond phylogeny in arbuscular mycorrhizal ecology</div>, <div class="source">New Phytol</div> ., <div class="volume">185</div>, <div class="fpage">67</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20trait%20indicators%3A%20moving%20beyond%20phylogeny%20in%20arbuscular%20mycorrhizal%20ecology&amp;author=H.A.%20Gamper&amp;author=M.G.A.%20van%20der%20Heijden&amp;author=G.A.%20Kowalchuk&amp;publication_year=2010&amp;journal=New%20Phytol&amp;volume=185&amp;pages=67-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2009.03058.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2009.03058.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2009.03058.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19863727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tisserant</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kohler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dozolme-Seddas</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The transcriptome of the arbuscular mycorrhizal fungus Glomus intraradices (DAOM 197198) reveals functional tradeoffs in an obligate symbiont</div>, <div class="source">New Phytol</div> ., <div class="volume">193</div>, <div class="fpage">755</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptome%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Glomus%20intraradices%20(DAOM%20197198)%20reveals%20functional%20tradeoffs%20in%20an%20obligate%20symbiont&amp;author=E.%20Tisserant&amp;author=A.%20Kohler&amp;author=P.%20Dozolme-Seddas&amp;publication_year=2012&amp;journal=New%20Phytol&amp;volume=193&amp;pages=755-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.03948.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03948.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.03948.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22092242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salvioli</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ghignone</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Novero</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Symbiosis with an endobacterium increases the fitness of a mycorrhizal fungus, raising its bioenergetic potential</div>, <div class="source">ISME J</div> ., <div class="volume">10</div>, <div class="fpage">130</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Symbiosis%20with%20an%20endobacterium%20increases%20the%20fitness%20of%20a%20mycorrhizal%20fungus%2C%20raising%20its%20bioenergetic%20potential&amp;author=A.%20Salvioli&amp;author=S.%20Ghignone&amp;author=M.%20Novero&amp;publication_year=2016&amp;journal=ISME%20J&amp;volume=10&amp;pages=130-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2015.91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2015.91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2015.91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26046255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">San Clemente</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Roy</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Becard</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Roux</div> <div class="given-names">C.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">A survey of the gene repertoire of gigaspora rosea unravels conserved features among glomeromycota for obligate biotrophy</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">233</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20the%20gene%20repertoire%20of%20gigaspora%20rosea%20unravels%20conserved%20features%20among%20glomeromycota%20for%20obligate%20biotrophy&amp;author=N.%20Tang&amp;author=H.%20San%20Clemente&amp;author=S.%20Roy&amp;author=G.%20Becard&amp;author=B.%20Zhao&amp;author=C.%20Roux&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=233" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26973612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kikuchi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hijikata</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Yokoyama</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Polyphosphate accumulation is driven by transcriptome alterations that lead to near-synchronous and near-equivalent uptake of inorganic cations in an arbuscular mycorrhizal fungus</div>, <div class="source">New Phytol</div> ., <div class="volume">204</div>, <div class="fpage">638</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyphosphate%20accumulation%20is%20driven%20by%20transcriptome%20alterations%20that%20lead%20to%20near-synchronous%20and%20near-equivalent%20uptake%20of%20inorganic%20cations%20in%20an%20arbuscular%20mycorrhizal%20fungus&amp;author=Y.%20Kikuchi&amp;author=N.%20Hijikata&amp;author=K.%20Yokoyama&amp;publication_year=2014&amp;journal=New%20Phytol&amp;volume=204&amp;pages=638-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12937" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12937" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12937"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25039900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sędzielewska Toro</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Brachmann</div> <div class="given-names">A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The effector candidate repertoire of the arbuscular mycorrhizal fungus Rhizophagus clarus</div>, <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effector%20candidate%20repertoire%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Rhizophagus%20clarus&amp;author=K.%20S%C4%99dzielewska%20Toro&amp;author=A.%20Brachmann&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-2422-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-2422-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-2422-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26861502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tisserant</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Malbreil</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kuo</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Genome of an arbuscular mycorrhizal fungus provides insight into the oldest plant symbiosis</div>, <div class="source">Proc. Natl. Acad. Sci. U S A</div> , <div class="volume">110</div>, <div class="fpage">20117</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20of%20an%20arbuscular%20mycorrhizal%20fungus%20provides%20insight%20into%20the%20oldest%20plant%20symbiosis&amp;author=E.%20Tisserant&amp;author=M.%20Malbreil&amp;author=A.%20Kuo&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=110&amp;pages=20117-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1313452110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1313452110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1313452110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24277808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Limpens</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Single nucleus genome sequencing reveals high similarity among nuclei of an endomycorrhizal fungus</div>, <div class="source">PLoS Genet</div> , <div class="volume">10</div>, <div class="fpage">e1004078</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleus%20genome%20sequencing%20reveals%20high%20similarity%20among%20nuclei%20of%20an%20endomycorrhizal%20fungus&amp;author=K.%20Lin&amp;author=E.%20Limpens&amp;author=Z.%20Zhang&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004078" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24415955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shapiro</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Biezuner</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Linnarsson</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Single-cell sequencing-based technologies will revolutionize whole-organism science</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">14</div>, <div class="fpage">618</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20sequencing-based%20technologies%20will%20revolutionize%20whole-organism%20science&amp;author=E.%20Shapiro&amp;author=T.%20Biezuner&amp;author=S.%20Linnarsson&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Genet&amp;volume=14&amp;pages=618-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3542" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3542" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3542"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23897237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Toussaint</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">St-Arnaud</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Charest</div> <div class="given-names">C.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Nitrogen transfer and assimilation between the arbuscular mycorrhizal fungus Glomus intraradices Schenck &amp; Smith and Ri T-DNA roots of Daucus carota L. in an in vitro compartmented system</div>, <div class="source">Can. J. Microbiol</div> ., <div class="volume">50</div>, <div class="fpage">251</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrogen%20transfer%20and%20assimilation%20between%20the%20arbuscular%20mycorrhizal%20fungus%20Glomus%20intraradices%20Schenck%20%26%20Smith%20and%20Ri%20T-DNA%20roots%20of%20Daucus%20carota%20L.%20in%20an%20in%20vitro%20compartmented%20system&amp;author=J.P.%20Toussaint&amp;author=M.%20St-Arnaud&amp;author=C.%20Charest&amp;publication_year=2004&amp;journal=Can.%20J.%20Microbiol&amp;volume=50&amp;pages=251-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/w04-009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fw04-009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fw04-009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15213749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">HMMER web server: interactive sequence similarity searching</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W29</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HMMER%20web%20server%3A%20interactive%20sequence%20similarity%20searching&amp;author=R.D.%20Finn&amp;author=J.%20Clements&amp;author=S.R.%20Eddy&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W29-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">D281</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Tate&amp;author=J.%20Mistry&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D281-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm960"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039703" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Petersen</div> <div class="given-names">T.N.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions</div>, <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">785</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SignalP%204.0%3A%20discriminating%20signal%20peptides%20from%20transmembrane%20regions&amp;author=T.N.%20Petersen&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=785-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21959131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Larsson</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Sonnhammer</div> <div class="given-names">E.L.L.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Predicting transmembrane protein topology with a hidden markov model: application to complete genomes1</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">305</div>, <div class="fpage">567</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20transmembrane%20protein%20topology%20with%20a%20hidden%20markov%20model%3A%20application%20to%20complete%20genomes1&amp;author=A.%20Krogh&amp;author=B.%20Larsson&amp;author=G.%20von%20Heijne&amp;author=E.L.L.%20Sonnhammer&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol&amp;volume=305&amp;pages=567-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.4315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.4315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.4315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11152613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div></div>, et al.   <div class="year">2000</div>, 
<div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet</div> ., <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Bono</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">KEGG: Kyoto encyclopedia of genes and genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">29</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Afgan</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Baker</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">van den Beek</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">W3</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Galaxy%20platform%20for%20accessible%2C%20reproducible%20and%20collaborative%20biomedical%20analyses%3A%202016%20update&amp;author=E.%20Afgan&amp;author=D.%20Baker&amp;author=M.%20van%20den%20Beek&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=W3-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27137889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">WEGO: a web tool for plotting GO annotations</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W293</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WEGO%3A%20a%20web%20tool%20for%20plotting%20GO%20annotations&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W293-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Niu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">WebMGA: a customizable web server for fast metagenomic sequence analysis</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">444</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WebMGA%3A%20a%20customizable%20web%20server%20for%20fast%20metagenomic%20sequence%20analysis&amp;author=S.%20Wu&amp;author=Z.%20Zhu&amp;author=L.%20Fu&amp;author=B.%20Niu&amp;author=W.%20Li&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=444" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-444"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21899761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Stoeckert</div> <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D.S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.J.%20Stoeckert&amp;author=D.S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altenhoff</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Dessimoz</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Eisen</div> <div class="given-names">J.A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Phylogenetic and functional assessment of orthologs inference projects and methods</div>, <div class="source">PLoS Comput. Biol</div> ., <div class="volume">5</div>, <div class="fpage">e1000262</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20and%20functional%20assessment%20of%20orthologs%20inference%20projects%20and%20methods&amp;author=A.M.%20Altenhoff&amp;author=C.%20Dessimoz&amp;author=J.A.%20Eisen&amp;publication_year=2009&amp;journal=PLoS%20Comput.%20Biol&amp;volume=5&amp;pages=e1000262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19148271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Capella-Gutierrez</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Marcet-Houben</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gabaldon</div> <div class="given-names">T.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Phylogenomics supports microsporidia as the earliest diverging clade of sequenced fungi</div>, <div class="source">BMC Biol</div> ., <div class="volume">10</div>, <div class="fpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics%20supports%20microsporidia%20as%20the%20earliest%20diverging%20clade%20of%20sequenced%20fungi&amp;author=S.%20Capella-Gutierrez&amp;author=M.%20Marcet-Houben&amp;author=T.%20Gabaldon&amp;publication_year=2012&amp;journal=BMC%20Biol&amp;volume=10&amp;pages=47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1741-7007-10-47" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1741-7007-10-47" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1741-7007-10-47"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22651672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The RAxML 7.0.4 manual</div>, <div class="source">Bioinformatics</div> , <div class="volume">22</div>, <div class="fpage">2688</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RAxML%207.0.4%20manual&amp;author=A.%20Stamatakis&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2688-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lartillot</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Lepage</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Blanquart</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2286</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PhyloBayes%203%3A%20a%20Bayesian%20software%20package%20for%20phylogenetic%20reconstruction%20and%20molecular%20dating&amp;author=N.%20Lartillot&amp;author=T.%20Lepage&amp;author=S.%20Blanquart&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2286-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp368" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp368" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp368"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19535536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Castillo</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Pawlowska</div> <div class="given-names">T.E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Molecular evolution in bacterial endosymbionts of fungi</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">622</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20in%20bacterial%20endosymbionts%20of%20fungi&amp;author=D.M.%20Castillo&amp;author=T.E.%20Pawlowska&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol&amp;volume=27&amp;pages=622-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19923192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Desirò</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Salvioli</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bonfante</div> <div class="given-names">P.</div></div></div> <div class="year">2016</div>, Investigating the endobacteria which thrive in arbuscular mycorrhizal fungi. In: <div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Uroz</div> <div class="given-names">S.</div></div></div>, (eds.), <div class="source">Microbial Environmental Genomics (MEG)</div> . 
<div class="publisher-name">Springer New York</div>, 
<div class="publisher-loc">New York, NY</div>, pp. <div class="fpage">29</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20Environmental%20Genomics%20(MEG)&amp;author=A.%20Desir%C3%B2&amp;author=A.%20Salvioli&amp;author=P.%20Bonfante&amp;author=F.%20Martin&amp;author=S.%20Uroz&amp;publication_year=2016&amp;book=Microbial%20Environmental%20Genomics%20(MEG)" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-1-4939-3369-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-4939-3369-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-1-4939-3369-3"> </span></div> </div></div></div></div></div><div content-id="dsx051-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simão</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.A.%20Sim%C3%A3o&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weber</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hellmann</div> <div class="given-names">H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Arabidopsis thaliana BTB/POZ-MATH proteins interact with members of the ERF/AP2 transcription factor family</div>, <div class="source">FEBS J</div> ., <div class="volume">276</div>, <div class="fpage">6624</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20thaliana%20BTB%2FPOZ-MATH%20proteins%20interact%20with%20members%20of%20the%20ERF%2FAP2%20transcription%20factor%20family&amp;author=H.%20Weber&amp;author=H.%20Hellmann&amp;publication_year=2009&amp;journal=FEBS%20J&amp;volume=276&amp;pages=6624-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2009.07373.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2009.07373.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2009.07373.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19843165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pandey</div> <div class="given-names">G. K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Expansion and function of repeat domain proteins during stress and development in plants</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">1218</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expansion%20and%20function%20of%20repeat%20domain%20proteins%20during%20stress%20and%20development%20in%20plants&amp;author=M.%20Sharma&amp;author=G.%20K.%20Pandey&amp;publication_year=2016&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=1218" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26793205" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singh</div> <div class="given-names">A. K.</div></div>, <div class="name string-name"><div class="surname">Mukhopadhyay</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Overview of fungal lipase: a review</div>, <div class="source">Appl. Biochem. Biotechnol</div> ., <div class="volume">166</div>, <div class="fpage">486</div>–<div class="lpage">520</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overview%20of%20fungal%20lipase%3A%20a%20review&amp;author=A.%20K.%20Singh&amp;author=M.%20Mukhopadhyay&amp;publication_year=2012&amp;journal=Appl.%20Biochem.%20Biotechnol&amp;volume=166&amp;pages=486-520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12010-011-9444-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12010-011-9444-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12010-011-9444-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22072143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murphy</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Powlowski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tsang</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Curation of characterized glycoside hydrolases of fungal origin</div>, <div class="source">Database</div> , <div class="volume">2011</div>, <div class="fpage">bar020</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Curation%20of%20characterized%20glycoside%20hydrolases%20of%20fungal%20origin&amp;author=C.%20Murphy&amp;author=J.%20Powlowski&amp;author=M.%20Wu&amp;author=G.%20Butler&amp;author=A.%20Tsang&amp;publication_year=2011&amp;journal=Database&amp;volume=2011&amp;pages=bar020" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/database/bar020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdatabase%2Fbar020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdatabase%2Fbar020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21622642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx051-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx051-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bonfante</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Desiro</div> <div class="given-names">A.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Who lives in a fungus[quest] the diversity, origins and functions of fungal endobacteria living in Mucoromycota</div>, <div class="source">ISME J</div> ., <div class="volume">11</div>, <div class="fpage">1727</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Who%20lives%20in%20a%20fungus%5Bquest%5D%20the%20diversity%2C%20origins%20and%20functions%20of%20fungal%20endobacteria%20living%20in%20Mucoromycota&amp;author=P.%20Bonfante&amp;author=A.%20Desiro&amp;publication_year=2017&amp;journal=ISME%20J&amp;volume=11&amp;pages=1727-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2017.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2017.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2017.21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28387771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4683712&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx051/1/dsx051_supp.zip?Expires=2147483647&amp;Signature=i6yFhOjy3T0HO51-13jdLQbEE4HUloMGVQg36lPWM8WUtv-YfBqx0yXMsYmBkOHUiZ5WVWjQcGDTbVmTRR2IHysUC76Q~CFZ74B0jn-XiSOQ-VcZpnNj7liHyjyblOWYIXhh7SQvHZukMe83vVQvfs4654MI42rb7o0dWXAPvCkaVP6uoFECNserCmrJZK2H9d7l8l7vmpmCjZSLBGlFOkrcmtOZAcpXad42qn~C8mR1RYoGTsnCH7pJ51nROYIhO5MFxBp-t~HUP3To3nFFVeI68tLDoxnGvfYMzVPZh1zX01qNXg6qD~W2NY9D5eUUhRmiOiuwL2NMhGS4WQKHDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">4,161</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx051"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4683712"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f217%2f4683712">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F217%2F4683712" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4683712'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Ultra-low input transcriptomics reveal the spore functional content and phylogenetic affiliations of poorly studied arbuscular mycorrhizal fungi" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx051");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f217%2f4683712">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

