

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/4/407/3744590","@type":"ScholarlyArticle","name":"Genetic variants associated with the root system architecture of oilseed rape ( Brassica napus  L.) under contrasting phosphate supply","datePublished":"2017-04-19","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2017-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx013","keywords":["oilseed rape","root system architecture","single-nucleotide polymorphism","genome-wide association study","BnA3Hap"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/4/407/3744590","author":[{"name":"Wang, Xiaohua","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Chen, Yanling","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Thomas, Catherine L.","affiliation":"3 Plant and Crop Sciences Division, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough LE12 5RD, UK","@type":"Person"},{"name":"Ding, Guangda","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Xu, Ping","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Shi, Dexu","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Grandke, Fabian","affiliation":"4 Department of Plant Breeding, IFZ Research Centre for BioSystems, Land Use and Nutrition, Justus Liebig University, Giessen 35392, Germany","@type":"Person"},{"name":"Jin, Kemo","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Cai, Hongmei","affiliation":"2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Xu, Fangsen","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Yi, Bin","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Broadley, Martin R.","affiliation":"3 Plant and Crop Sciences Division, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough LE12 5RD, UK","@type":"Person"},{"name":"Shi, Lei","affiliation":"1 National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China   2 Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"}],"description":"Abstract.  Breeding crops with ideal root system architecture for efficient absorption of phosphorus is an important strategy to reduce the use of phosphate fer","pageStart":"407","pageEnd":"417","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genetic variants associated with the root system architecture of oilseed rape ( Brassica napus  L.) under contrasting phosphate supply","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genetic variants associated with the root system architecture of oilseed rape ( Brassica napus  L.) under contrasting phosphate supply" />
    <meta property="og:description" content="Abstract.  Breeding crops with ideal root system architecture for efficient absorption of phosphorus is an important strategy to reduce the use of phosphate fer" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/4/407/3744590" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-04-19" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wang, Xiaohua" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Chen, Yanling" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Thomas, Catherine L." /><meta name="citation_author_institution" content="Plant and Crop Sciences Division, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough LE12 5RD, UK" /><meta name="citation_author" content="Ding, Guangda" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Xu, Ping" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Shi, Dexu" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Grandke, Fabian" /><meta name="citation_author_institution" content="Department of Plant Breeding, IFZ Research Centre for BioSystems, Land Use and Nutrition, Justus Liebig University, Giessen 35392, Germany" /><meta name="citation_author" content="Jin, Kemo" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Cai, Hongmei" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Xu, Fangsen" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Yi, Bin" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Broadley, Martin R." /><meta name="citation_author_institution" content="Plant and Crop Sciences Division, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough LE12 5RD, UK" /><meta name="citation_author" content="Shi, Lei" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_title" content="Genetic variants associated with the root system architecture of oilseed rape (Brassica napus L.) under contrasting phosphate supply" /><meta name="citation_firstpage" content="407" /><meta name="citation_lastpage" content="417" /><meta name="citation_doi" content="10.1093/dnares/dsx013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2017/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Plant nutrition, rooting for more phosphorus;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=488;  citation_pages=466-7; " /><meta name="citation_reference" content="citation_title=Root phenes that reduce the metabolic costs of soil exploration, opportunities for 21st century agriculture;  citation_journal_title=Plant Cell Environ;  citation_year=2015;  citation_volume=38;  citation_pages=1775-84; " /><meta name="citation_reference" content="citation_title=High-throughput root phenotyping screens identify genetic loci associated with root architectural traits in Brassica napus under contrasting phosphate availabilities;  citation_journal_title=Ann. Bot-London;  citation_year=2013;  citation_volume=112;  citation_pages=381-9; " /><meta name="citation_reference" content="citation_title=Shoot yield drives phosphorus use efficiency in Brassica oleracea and correlates with root architecture traits;  citation_journal_title=J. Exp. Bot;  citation_year=2009;  citation_volume=60;  citation_pages=1953-68; " /><meta name="citation_reference" content="citation_title=Root tip contact with low-phosphate media reprograms plant root architecture;  citation_journal_title=Nat. Genet;  citation_year=2007;  citation_volume=39;  citation_pages=792-6; " /><meta name="citation_reference" content="citation_title=Substitution mapping of Pup1, a major QTL increasing phosphorus uptake of rice from a phosphorus-deficient soil;  citation_journal_title=Theor. Appl. Genet;  citation_year=2002;  citation_volume=105;  citation_pages=890-7; " /><meta name="citation_reference" content="citation_title=The protein kinase Pstol1 from traditional rice confers tolerance of phosphorus deficiency;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=488;  citation_pages=535-9; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci for root morphology in response to low phosphorus stress in Brassica napus;  citation_journal_title=Theor. Appl. Genet;  citation_year=2010;  citation_volume=121;  citation_pages=181-93; " /><meta name="citation_reference" content="citation_title=QTL meta-analysis of root traits in Brassica napus under contrasting phosphorus supply in two growth systems;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=33113" /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces;  citation_journal_title=Nat. Genet;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping of root traits in a japonica rice panel;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e78037" /><meta name="citation_reference" content="citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=11479-84; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=159-62; " /><meta name="citation_reference" content="citation_title=Genome-wide association study reveals the genetic architecture of flowering time in rapeseed (Brassica napus L.);  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=43-52; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis of symbiotic nitrogen fixation in common bean;  citation_journal_title=Theor. Appl. Genet;  citation_year=2015;  citation_volume=128;  citation_pages=1999-2017; " /><meta name="citation_reference" content="citation_title=The role of root size versus root efficiency in phosphorus acquisition in rice;  citation_journal_title=J. Exp. Bot;  citation_year=2016;  citation_volume=67;  citation_pages=1179-89; " /><meta name="citation_reference" content="citation_title=The acid phosphatase-encoding gene GmACP1 contributes to soybean tolerance to low-phosphorus stress;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=229-31; " /><meta name="citation_reference" content="citation_title=Integration of responses within and across arabidopsis natural accessions uncovers loci controlling root systems architecture;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=15133-8; " /><meta name="citation_reference" content="citation_title=Genetic variation in ZmVPP1 contributes to drought tolerance in maize seedlings;  citation_journal_title=Nat. Genet;  citation_volume=48;  citation_pages=1233-41; " /><meta name="citation_reference" content="citation_title=Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (Brassica napus L.);  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=A genome-wide association study reveals novel elite allelic variations in seed oil content of Brassica napus;  citation_journal_title=Theor. Appl. Genet;  citation_year=2016;  citation_volume=129;  citation_pages=1203-15; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis and differential expression analysis of resistance to Sclerotinia stem rot in Brassica napus;  citation_journal_title=Plant Biotechnol. J;  citation_year=2016;  citation_volume=14;  citation_pages=1368-80; " /><meta name="citation_reference" content="citation_title=Species- and genome-wide dissection of the shoot ionome in Brassica napus and its relationship to seedling development;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5;  citation_pages=485" /><meta name="citation_reference" content="citation_publisher=Wiley-Blackwell, ;  citation_title=Biogeography: An Ecological and Evolutionary Approach;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=High-throughput phenotyping HTP, identifies seedling root traits linked to variation in seed yield and nutrient capture in field-grown oilseed rape (Brassica napus L.);  citation_journal_title=Ann. Bot-London;  citation_year=2016;  citation_volume=118;  citation_pages=655-65; " /><meta name="citation_reference" content="citation_title=Phenotyping pipeline reveals major seedling root growth QTL in hexaploid wheat;  citation_journal_title=J. Exp. Bot;  citation_year=2015;  citation_volume=66;  citation_pages=2283-92; " /><meta name="citation_reference" content="citation_title=Identification of molecular markers linked to trilocular gene (mc1) in Brassica juncea L;  citation_journal_title=Mol. Breeding;  citation_year=2013;  citation_volume=33;  citation_pages=425-34; " /><meta name="citation_reference" content="citation_title=Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=950-3; " /><meta name="citation_reference" content="citation_title=Association mapping in structured populations;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2000;  citation_volume=67;  citation_pages=170-81; " /><meta name="citation_reference" content="citation_title=Detecting the number of clusters of individuals using the software STRUCTURE, a simulation study;  citation_journal_title=Mol. Ecol;  citation_year=2005;  citation_volume=14;  citation_pages=2611-20; " /><meta name="citation_reference" content="citation_title=Identification of Phosphorous Efficient Germplasm in Oilseed Rape;  citation_journal_title=J. Plant Nutr;  citation_year=2009;  citation_volume=32;  citation_pages=1148-63; " /><meta name="citation_reference" content="citation_title=Genetic structure and linkage disequilibrium pattern of a rapeseed Brassica napus L., association mapping panel revealed by microsatellites;  citation_journal_title=Theor. Appl. Genet;  citation_year=2012;  citation_volume=125;  citation_pages=437-47; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa;  citation_journal_title=Nat. Commun;  citation_year=2011;  citation_volume=2;  citation_pages=467" /><meta name="citation_reference" content="citation_title=The extent of linkage disequilibrium in Arabidopsis thaliana;  citation_journal_title=Nat. Genet;  citation_year=2002;  citation_volume=30;  citation_pages=190-3; " /><meta name="citation_reference" content="citation_title=Population structure and genetic differentiation associated with breeding history and selection in tomato (Solanum lycopersicum L.);  citation_journal_title=Heredity;  citation_year=2011;  citation_volume=106;  citation_pages=927-35; " /><meta name="citation_reference" content="citation_title=Genome-wide investigation of genetic changes during modern breeding of Brassica napus;  citation_journal_title=Theor. Appl. Genet;  citation_year=2014;  citation_volume=127;  citation_pages=1817-29; " /><meta name="citation_reference" content="citation_title=gsrc-an R package for genome structure rearrangement calling;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_pages=545-6; " /><meta name="citation_reference" content="citation_title=Superior root hair formation confers root efficiency in some, but not all, rice genotypes upon p deficiency;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7; " /><meta name="citation_reference" content="citation_title=Identification of QTL controlling root growth response to phosphate starvation in Arabidopsis thaliana;  citation_journal_title=Plant Cell Environ;  citation_year=2006;  citation_volume=29;  citation_pages=115-25; " /><meta name="citation_reference" content="citation_title=RACK1 is a negative regulator of ABA responses in Arabidopsis;  citation_journal_title=J. Exp. Bot;  citation_year=2009;  citation_volume=60;  citation_pages=3819-33; " /><meta name="citation_reference" content="citation_title=Endogenous arabidopsis messenger RNAs transported to distant tissues;  citation_journal_title=Nat. Plants;  citation_year=2015;  citation_volume=1; " /><meta name="citation_reference" content="citation_title=PlantTFDB 3.0, a portal for the functional and evolutionary study of plant transcription factors;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=1182-7; " /><meta name="citation_reference" content="citation_title=Expression profiling of auxin-treated Arabidopsis roots, toward a molecular analysis of lateral root emergence;  citation_journal_title=Plant Cell Physiol;  citation_year=2006;  citation_volume=47;  citation_pages=788-92; " /><meta name="citation_reference" content="citation_title=Agronomic phosphorus imbalances across the world's croplands;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=3086-91; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/4/407/22893223/dsx013.pdf" /><meta name="description" content="Abstract.  Breeding crops with ideal root system architecture for efficient absorption of phosphorus is an important strategy to reduce the use of phosphate fer" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/4/407/3744590" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/4/407/3744590" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/4/407/3744590");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx013';
    </script>
<title>
	Genetic variants associated with the root system architecture of oilseed rape (Brassica napus L.) under contrasting phosphate supply | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3744590" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcWHATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgWGlhb2h1YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCBhbmQgTmF0aW9uYWwgQ2VudHJlIG9mIFBsYW50IEdlbmUgUmVzZWFyY2gsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQXJhYmxlIExhbmQgQ29uc2VydmF0aW9uIChNaWRkbGUgYW5kIExvd2VyIFJlYWNoZXMgb2YgWWFuZ3R6ZSBSaXZlciksIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIFlhbmxpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBLZXkgTGFib3JhdG9yeSBvZiBDcm9wIEdlbmV0aWMgSW1wcm92ZW1lbnQgYW5kIE5hdGlvbmFsIENlbnRyZSBvZiBQbGFudCBHZW5lIFJlc2VhcmNoLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEFyYWJsZSBMYW5kIENvbnNlcnZhdGlvbiAoTWlkZGxlIGFuZCBMb3dlciBSZWFjaGVzIG9mIFlhbmd0emUgUml2ZXIpLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUaG9tYXMsIENhdGhlcmluZSBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBsYW50IGFuZCBDcm9wIFNjaWVuY2VzIERpdmlzaW9uLCBTY2hvb2wgb2YgQmlvc2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTm90dGluZ2hhbSwgU3V0dG9uIEJvbmluZ3RvbiBDYW1wdXMsIExvdWdoYm9yb3VnaCBMRTEy4oCJNVJELCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRpbmcsIEd1YW5nZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBLZXkgTGFib3JhdG9yeSBvZiBDcm9wIEdlbmV0aWMgSW1wcm92ZW1lbnQgYW5kIE5hdGlvbmFsIENlbnRyZSBvZiBQbGFudCBHZW5lIFJlc2VhcmNoLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEFyYWJsZSBMYW5kIENvbnNlcnZhdGlvbiAoTWlkZGxlIGFuZCBMb3dlciBSZWFjaGVzIG9mIFlhbmd0emUgUml2ZXIpLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJYdSwgUGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCBhbmQgTmF0aW9uYWwgQ2VudHJlIG9mIFBsYW50IEdlbmUgUmVzZWFyY2gsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpLCBEZXh1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50IGFuZCBOYXRpb25hbCBDZW50cmUgb2YgUGxhbnQgR2VuZSBSZXNlYXJjaCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBcmFibGUgTGFuZCBDb25zZXJ2YXRpb24gKE1pZGRsZSBhbmQgTG93ZXIgUmVhY2hlcyBvZiBZYW5ndHplIFJpdmVyKSwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3JhbmRrZSwgRmFiaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCcmVlZGluZywgSUZaIFJlc2VhcmNoIENlbnRyZSBmb3IgQmlvU3lzdGVtcywgTGFuZCBVc2UgYW5kIE51dHJpdGlvbiwgSnVzdHVzIExpZWJpZyBVbml2ZXJzaXR5LCBHaWVzc2VuIDM1MzkyLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmluLCBLZW1vIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50IGFuZCBOYXRpb25hbCBDZW50cmUgb2YgUGxhbnQgR2VuZSBSZXNlYXJjaCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBcmFibGUgTGFuZCBDb25zZXJ2YXRpb24gKE1pZGRsZSBhbmQgTG93ZXIgUmVhY2hlcyBvZiBZYW5ndHplIFJpdmVyKSwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FpLCBIb25nbWVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQXJhYmxlIExhbmQgQ29uc2VydmF0aW9uIChNaWRkbGUgYW5kIExvd2VyIFJlYWNoZXMgb2YgWWFuZ3R6ZSBSaXZlciksIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilh1LCBGYW5nc2VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50IGFuZCBOYXRpb25hbCBDZW50cmUgb2YgUGxhbnQgR2VuZSBSZXNlYXJjaCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBcmFibGUgTGFuZCBDb25zZXJ2YXRpb24gKE1pZGRsZSBhbmQgTG93ZXIgUmVhY2hlcyBvZiBZYW5ndHplIFJpdmVyKSwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWksIEJpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCBhbmQgTmF0aW9uYWwgQ2VudHJlIG9mIFBsYW50IEdlbmUgUmVzZWFyY2gsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQnJvYWRsZXksIE1hcnRpbiBSLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBsYW50IGFuZCBDcm9wIFNjaWVuY2VzIERpdmlzaW9uLCBTY2hvb2wgb2YgQmlvc2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTm90dGluZ2hhbSwgU3V0dG9uIEJvbmluZ3RvbiBDYW1wdXMsIExvdWdoYm9yb3VnaCBMRTEy4oCJNVJELCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaSwgTGVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50IGFuZCBOYXRpb25hbCBDZW50cmUgb2YgUGxhbnQgR2VuZSBSZXNlYXJjaCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBcmFibGUgTGFuZCBDb25zZXJ2YXRpb24gKE1pZGRsZSBhbmQgTG93ZXIgUmVhY2hlcyBvZiBZYW5ndHplIFJpdmVyKSwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5ldGljIHZhcmlhbnRzIGFzc29jaWF0ZWQgd2l0aCB0aGUgcm9vdCBzeXN0ZW0gYXJjaGl0ZWN0dXJlIG9mIG9pbHNlZWQgcmFwZSAoQnJhc3NpY2EgbmFwdXMgTC4pIHVuZGVyIGNvbnRyYXN0aW5nIHBob3NwaGF0ZSBzdXBwbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0MDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgbnV0cml0aW9uLCByb290aW5nIGZvciBtb3JlIHBob3NwaG9ydXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODg7ICBjaXRhdGlvbl9wYWdlcz00NjYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9vdCBwaGVuZXMgdGhhdCByZWR1Y2UgdGhlIG1ldGFib2xpYyBjb3N0cyBvZiBzb2lsIGV4cGxvcmF0aW9uLCBvcHBvcnR1bml0aWVzIGZvciAyMXN0IGNlbnR1cnkgYWdyaWN1bHR1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgRW52aXJvbjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTE3NzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCByb290IHBoZW5vdHlwaW5nIHNjcmVlbnMgaWRlbnRpZnkgZ2VuZXRpYyBsb2NpIGFzc29jaWF0ZWQgd2l0aCByb290IGFyY2hpdGVjdHVyYWwgdHJhaXRzIGluIEJyYXNzaWNhIG5hcHVzIHVuZGVyIGNvbnRyYXN0aW5nIHBob3NwaGF0ZSBhdmFpbGFiaWxpdGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBCb3QtTG9uZG9uOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG9vdCB5aWVsZCBkcml2ZXMgcGhvc3Bob3J1cyB1c2UgZWZmaWNpZW5jeSBpbiBCcmFzc2ljYSBvbGVyYWNlYSBhbmQgY29ycmVsYXRlcyB3aXRoIHJvb3QgYXJjaGl0ZWN0dXJlIHRyYWl0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXhwLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0xOTUzLTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb290IHRpcCBjb250YWN0IHdpdGggbG93LXBob3NwaGF0ZSBtZWRpYSByZXByb2dyYW1zIHBsYW50IHJvb3QgYXJjaGl0ZWN0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NzkyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YnN0aXR1dGlvbiBtYXBwaW5nIG9mIFB1cDEsIGEgbWFqb3IgUVRMIGluY3JlYXNpbmcgcGhvc3Bob3J1cyB1cHRha2Ugb2YgcmljZSBmcm9tIGEgcGhvc3Bob3J1cy1kZWZpY2llbnQgc29pbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTg5MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcHJvdGVpbiBraW5hc2UgUHN0b2wxIGZyb20gdHJhZGl0aW9uYWwgcmljZSBjb25mZXJzIHRvbGVyYW5jZSBvZiBwaG9zcGhvcnVzIGRlZmljaWVuY3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODg7ICBjaXRhdGlvbl9wYWdlcz01MzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIHJvb3QgbW9ycGhvbG9neSBpbiByZXNwb25zZSB0byBsb3cgcGhvc3Bob3J1cyBzdHJlc3MgaW4gQnJhc3NpY2EgbmFwdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBtZXRhLWFuYWx5c2lzIG9mIHJvb3QgdHJhaXRzIGluIEJyYXNzaWNhIG5hcHVzIHVuZGVyIGNvbnRyYXN0aW5nIHBob3NwaG9ydXMgc3VwcGx5IGluIHR3byBncm93dGggc3lzdGVtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMzMTEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyBvZiAxNCBhZ3Jvbm9taWMgdHJhaXRzIGluIHJpY2UgbGFuZHJhY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTYxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIG1hcHBpbmcgb2Ygcm9vdCB0cmFpdHMgaW4gYSBqYXBvbmljYSByaWNlIHBhbmVsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTc4MDM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gYW5kIHBoZW5vdHlwaWMgYXNzb2NpYXRpb25zIGluIHRoZSBtYWl6ZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTE0NzktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IG9mIGxlYWYgYXJjaGl0ZWN0dXJlIGluIHRoZSBtYWl6ZSBuZXN0ZWQgYXNzb2NpYXRpb24gbWFwcGluZyBwb3B1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTU5LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeSByZXZlYWxzIHRoZSBnZW5ldGljIGFyY2hpdGVjdHVyZSBvZiBmbG93ZXJpbmcgdGltZSBpbiByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDMtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIGFuYWx5c2lzIG9mIHN5bWJpb3RpYyBuaXRyb2dlbiBmaXhhdGlvbiBpbiBjb21tb24gYmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTE5OTktMjAxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2Ygcm9vdCBzaXplIHZlcnN1cyByb290IGVmZmljaWVuY3kgaW4gcGhvc3Bob3J1cyBhY3F1aXNpdGlvbiBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTExNzktODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhY2lkIHBob3NwaGF0YXNlLWVuY29kaW5nIGdlbmUgR21BQ1AxIGNvbnRyaWJ1dGVzIHRvIHNveWJlYW4gdG9sZXJhbmNlIHRvIGxvdy1waG9zcGhvcnVzIHN0cmVzczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTIyOS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpb24gb2YgcmVzcG9uc2VzIHdpdGhpbiBhbmQgYWNyb3NzIGFyYWJpZG9wc2lzIG5hdHVyYWwgYWNjZXNzaW9ucyB1bmNvdmVycyBsb2NpIGNvbnRyb2xsaW5nIHJvb3Qgc3lzdGVtcyBhcmNoaXRlY3R1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTE1MTMzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWF0aW9uIGluIFptVlBQMSBjb250cmlidXRlcyB0byBkcm91Z2h0IHRvbGVyYW5jZSBpbiBtYWl6ZSBzZWVkbGluZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xMjMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeSBkaXNzZWN0cyB0aGUgZ2VuZXRpYyBhcmNoaXRlY3R1cmUgb2Ygc2VlZCB3ZWlnaHQgYW5kIHNlZWQgcXVhbGl0eSBpbiByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzU1LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IHJldmVhbHMgbm92ZWwgZWxpdGUgYWxsZWxpYyB2YXJpYXRpb25zIGluIHNlZWQgb2lsIGNvbnRlbnQgb2YgQnJhc3NpY2EgbmFwdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz0xMjAzLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBhbmFseXNpcyBhbmQgZGlmZmVyZW50aWFsIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgcmVzaXN0YW5jZSB0byBTY2xlcm90aW5pYSBzdGVtIHJvdCBpbiBCcmFzc2ljYSBuYXB1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQmlvdGVjaG5vbC4gSjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEzNjgtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZXMtIGFuZCBnZW5vbWUtd2lkZSBkaXNzZWN0aW9uIG9mIHRoZSBzaG9vdCBpb25vbWUgaW4gQnJhc3NpY2EgbmFwdXMgYW5kIGl0cyByZWxhdGlvbnNoaXAgdG8gc2VlZGxpbmcgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQ4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1XaWxleS1CbGFja3dlbGwsIDsgIGNpdGF0aW9uX3RpdGxlPUJpb2dlb2dyYXBoeTogQW4gRWNvbG9naWNhbCBhbmQgRXZvbHV0aW9uYXJ5IEFwcHJvYWNoOyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgcGhlbm90eXBpbmcgSFRQLCBpZGVudGlmaWVzIHNlZWRsaW5nIHJvb3QgdHJhaXRzIGxpbmtlZCB0byB2YXJpYXRpb24gaW4gc2VlZCB5aWVsZCBhbmQgbnV0cmllbnQgY2FwdHVyZSBpbiBmaWVsZC1ncm93biBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBCb3QtTG9uZG9uOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTY1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhlbm90eXBpbmcgcGlwZWxpbmUgcmV2ZWFscyBtYWpvciBzZWVkbGluZyByb290IGdyb3d0aCBRVEwgaW4gaGV4YXBsb2lkIHdoZWF0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTIyODMtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIG1vbGVjdWxhciBtYXJrZXJzIGxpbmtlZCB0byB0cmlsb2N1bGFyIGdlbmUgKG1jMSkgaW4gQnJhc3NpY2EganVuY2VhIEw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz00MjUtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVhcmx5IGFsbG9wb2x5cGxvaWQgZXZvbHV0aW9uIGluIHRoZSBwb3N0LU5lb2xpdGhpYyBCcmFzc2ljYSBuYXB1cyBvaWxzZWVkIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDU7ICBjaXRhdGlvbl9wYWdlcz05NTAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gbWFwcGluZyBpbiBzdHJ1Y3R1cmVkIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gSHVtLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTE3MC04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHRoZSBudW1iZXIgb2YgY2x1c3RlcnMgb2YgaW5kaXZpZHVhbHMgdXNpbmcgdGhlIHNvZnR3YXJlIFNUUlVDVFVSRSwgYSBzaW11bGF0aW9uIHN0dWR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNjExLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBQaG9zcGhvcm91cyBFZmZpY2llbnQgR2VybXBsYXNtIGluIE9pbHNlZWQgUmFwZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgTnV0cjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTExNDgtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgc3RydWN0dXJlIGFuZCBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIHBhdHRlcm4gb2YgYSByYXBlc2VlZCBCcmFzc2ljYSBuYXB1cyBMLiwgYXNzb2NpYXRpb24gbWFwcGluZyBwYW5lbCByZXZlYWxlZCBieSBtaWNyb3NhdGVsbGl0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz00MzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIG1hcHBpbmcgcmV2ZWFscyBhIHJpY2ggZ2VuZXRpYyBhcmNoaXRlY3R1cmUgb2YgY29tcGxleCB0cmFpdHMgaW4gT3J5emEgc2F0aXZhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NDY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV4dGVudCBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTkwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gc3RydWN0dXJlIGFuZCBnZW5ldGljIGRpZmZlcmVudGlhdGlvbiBhc3NvY2lhdGVkIHdpdGggYnJlZWRpbmcgaGlzdG9yeSBhbmQgc2VsZWN0aW9uIGluIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz05MjctMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGludmVzdGlnYXRpb24gb2YgZ2VuZXRpYyBjaGFuZ2VzIGR1cmluZyBtb2Rlcm4gYnJlZWRpbmcgb2YgQnJhc3NpY2EgbmFwdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz0xODE3LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1nc3JjLWFuIFIgcGFja2FnZSBmb3IgZ2Vub21lIHN0cnVjdHVyZSByZWFycmFuZ2VtZW50IGNhbGxpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTQ1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cGVyaW9yIHJvb3QgaGFpciBmb3JtYXRpb24gY29uZmVycyByb290IGVmZmljaWVuY3kgaW4gc29tZSwgYnV0IG5vdCBhbGwsIHJpY2UgZ2Vub3R5cGVzIHVwb24gcCBkZWZpY2llbmN5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFFUTCBjb250cm9sbGluZyByb290IGdyb3d0aCByZXNwb25zZSB0byBwaG9zcGhhdGUgc3RhcnZhdGlvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBFbnZpcm9uOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SQUNLMSBpcyBhIG5lZ2F0aXZlIHJlZ3VsYXRvciBvZiBBQkEgcmVzcG9uc2VzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTM4MTktMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZG9nZW5vdXMgYXJhYmlkb3BzaXMgbWVzc2VuZ2VyIFJOQXMgdHJhbnNwb3J0ZWQgdG8gZGlzdGFudCB0aXNzdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFBsYW50czsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudFRGREIgMy4wLCBhIHBvcnRhbCBmb3IgdGhlIGZ1bmN0aW9uYWwgYW5kIGV2b2x1dGlvbmFyeSBzdHVkeSBvZiBwbGFudCB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTE4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIHByb2ZpbGluZyBvZiBhdXhpbi10cmVhdGVkIEFyYWJpZG9wc2lzIHJvb3RzLCB0b3dhcmQgYSBtb2xlY3VsYXIgYW5hbHlzaXMgb2YgbGF0ZXJhbCByb290IGVtZXJnZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9Nzg4LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ3Jvbm9taWMgcGhvc3Bob3J1cyBpbWJhbGFuY2VzIGFjcm9zcyB0aGUgd29ybGQncyBjcm9wbGFuZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTMwODYtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC80LzQwNy8yMjg5MzIyMy9kc3gwMTMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEJyZWVkaW5nIGNyb3BzIHdpdGggaWRlYWwgcm9vdCBzeXN0ZW0gYXJjaGl0ZWN0dXJlIGZvciBlZmZpY2llbnQgYWJzb3JwdGlvbiBvZiBwaG9zcGhvcnVzIGlzIGFuIGltcG9ydGFudCBzdHJhdGVneSB0byByZWR1Y2UgdGhlIHVzZSBvZiBwaG9zcGhhdGUgZmVyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNzQ0NTkwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZOr9Y2t9MJpBN3mMvErTlcjLVjde" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f407%2f3744590" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f407%2f3744590" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=98073" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&amp;Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/4">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106357209">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357209">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357211">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357211">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357217">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357217">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357237">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357237">3 Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357260">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357260">4 Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357272">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357272">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357274">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357274">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357276">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357276">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357278">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357278">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/4/397/3815695">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/4/419/3752474">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genetic variants associated with the root system architecture of oilseed rape (<em>Brassica napus</em> L.) under contrasting phosphate supply
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaohua Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaohua+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xiaohua Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanling Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanling+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yanling Chen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Catherine L. Thomas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Plant and Crop Sciences Division, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough LE12 5RD, UK</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Catherine+L.+Thomas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Thomas C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Thomas C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Catherine L. Thomas</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guangda Ding</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guangda+Ding">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ding G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ding G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Guangda Ding</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ping Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ping+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ping Xu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dexu Shi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dexu+Shi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shi D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shi D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Dexu Shi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fabian Grandke</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Department of Plant Breeding, IFZ Research Centre for BioSystems, Land Use and Nutrition, Justus Liebig University, Giessen 35392, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fabian+Grandke">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Grandke F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Grandke F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fabian Grandke</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kemo Jin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kemo+Jin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jin K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jin K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kemo Jin</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hongmei Cai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hongmei+Cai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cai H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cai H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hongmei Cai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fangsen Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fangsen+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fangsen Xu</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bin Yi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors jointly supervised and contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bin+Yi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yi B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yi B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Bin Yi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin R. Broadley</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Plant and Crop Sciences Division, School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughborough LE12 5RD, UK</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors jointly supervised and contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+R.+Broadley">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Broadley M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Broadley M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Martin R. Broadley</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lei Shi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Key Laboratory of Crop Genetic Improvement and National Centre of Plant Gene Research, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff"><div class="label -label">2</div>Key Laboratory of Arable Land Conservation (Middle and Lower Reaches of Yangtze River), Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors jointly supervised and contributed equally to this work.</p>
                </div>
                <div class="info-card-note">
                    <a reveal-id="dsx013-cor1" data-open="dsx013-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lei+Shi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shi L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shi L&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Lei Shi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx013-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors jointly supervised and contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx013-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 407–417, <a href='https://doi.org/10.1093/dnares/dsx013'>https://doi.org/10.1093/dnares/dsx013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">19 April 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 November 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">15 March 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 March 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3744590" href="/dnaresearch/article-pdf/24/4/407/22893223/dsx013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xiaohua Wang, Yanling Chen, Catherine L. Thomas, Guangda Ding, Ping Xu, Dexu Shi, Fabian Grandke, Kemo Jin, Hongmei Cai, Fangsen Xu, Bin Yi, Martin R. Broadley, Lei Shi;  Genetic variants associated with the root system architecture of oilseed rape (<em>Brassica napus</em> L.) under contrasting phosphate supply, <em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 407–417, <a href='https://doi.org/10.1093/dnares/dsx013'>https://doi.org/10.1093/dnares/dsx013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3744590?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genetic%20variants%20associated%20with%20the%20root%20system%20architecture%20of%20oilseed%20rape%20(%26lt%3Bem%26gt%3BBrassica%20napus%26lt%3B%2Fem%26gt%3B%20L.)%20under%20contrasting%20phosphate%20supply&amp;publicationDate=2017-04-19&amp;volumeNum=24&amp;issueNum=4&amp;author=Wang%2C%20Xiaohua%3B%20Chen%2C%20Yanling&amp;startPage=407&amp;endPage=417&amp;contentId=10.1093%2Fdnares%2Fdsx013&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f407%2f3744590" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106357209 id="106357209" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Breeding crops with ideal root system architecture for efficient absorption of phosphorus is an important strategy to reduce the use of phosphate fertilizers. To investigate genetic variants leading to changes in root system architecture, 405 oilseed rape cultivars were genotyped with a 60K <em>Brassica</em> Infinium SNP array in low and high P environments. A total of 285 single-nucleotide polymorphisms were associated with root system architecture traits at varying phosphorus levels. Nine single-nucleotide polymorphisms corroborate a previous linkage analysis of root system architecture quantitative trait loci in the <em>Bna</em>TNDH population. One peak single-nucleotide polymorphism region on A3 was associated with all root system architecture traits and co-localized with a quantitative trait locus for primary root length at low phosphorus. Two more single-nucleotide polymorphism peaks on A5 for root dry weight at low phosphorus were detected in both growth systems and co-localized with a quantitative trait locus for the same trait. The candidate genes identified on A3 form a haplotype ‘<em>BnA3Hap</em>’, that will be important for understanding the phosphorus/root system interaction and for the incorporation into <em>Brassica napus</em> breeding programs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;oilseed rape&quot;">oilseed rape</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;root system architecture&quot;">root system architecture</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single-nucleotide polymorphism&quot;">single-nucleotide polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome-wide association study&quot;">genome-wide association study</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="BnA3Hap">BnA3Hap</a></div>                            <h2 scrollto-destination=106357211 id="106357211" class="section-title" >1. Introduction</h2>
<p>The demand for oilseed rape (Canola; <em>Brassica napus</em> L.) as a food and fuel crop has increased global production to 24.6 Mt yr<sup>−1</sup> in 2013 (FAO 2013; <a class="link link-uri" href="http://faostat.fao.org/" target="_blank">http://faostat.fao.org/</a>). Phosphorus (P) is critical in the production of oilseed crops, and with worldwide demand for P fertilizer rising,<a reveal-id="dsx013-B1" data-open="dsx013-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> breeding P use efficiency into crops can be an effective strategy. Root system architecture (RSA) plays an important role in P utilization. This is due to low soil P mobility resulting in heteromorphic spatial variation within the soil, but also from variation in bio-availability due to varying soil pH, microbial activity, and colloid chemistry.<a reveal-id="dsx013-B2" data-open="dsx013-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a></p><p>Improved RSA traits have been identified in <em>Brassica</em> which deliver greater resistance to P starvation.<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> For example, under low P in <em>B. napus</em><a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and <em>B. oleracea</em><a reveal-id="dsx013-B4" data-open="dsx013-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> primary roots become shorter and lateral roots increase in number and length. In <em>A. thaliana</em>, quantitative trait locus (QTL) <em>LPR1</em> (<em>Low Phosphate Root 1</em>) is mapped to a 36-kb region on chromosome 1 in a recombinant inbred line (RIL) population.<a reveal-id="dsx013-B5" data-open="dsx013-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In rice (<em>Oryza sativa</em>), a major QTL for P uptake (<em>Pup1</em>) is mapped to a 150-kb region on chromosome 12 in an F<sub>3</sub> near-isogenic line (NIL) population.<a reveal-id="dsx013-B6" data-open="dsx013-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> The gene conferring the phenotype (<em>PSTOL1</em>) is attributed to this region, and is a protein kinase in this QTL region that acts as an enhancer of early root growth, thereby enabling plants to acquire more phosphorus and other nutrients.<a reveal-id="dsx013-B7" data-open="dsx013-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In <em>B. napus</em>, a total of 62 signiﬁcant QTL for total root length, root surface area, root volume, total dry weight and plant P uptake at two P levels were detected in three independent experiments with a F<sub>10</sub> RIL population under low and high P conditions. Two QTL clusters, <em>uq.C3a</em> and <em>uq.C3b</em> were identified as specifically for root traits and P uptake under LP stress.<a reveal-id="dsx013-B8" data-open="dsx013-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> A total of 131 QTL are identified for RSA traits under low and normal phosphate supply using a ‘pouch and wick’ system and an agar based system. QTLs for root and shoot biomass co-localized on chromosome A3, and a QTL for lateral root emergence colocalized with chromosomes A4/C4 and C8/C9 in a doubled haploid population (<em>Bna</em>TNDH).<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p><p>Genome-wide association studies (GWAS) have been successfully applied to the genetic dissection of complex traits in crops, such as drought resistance in <em>Oryza sativa</em><a reveal-id="dsx013-B10" data-open="dsx013-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx013-B11" data-open="dsx013-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and flowering time and leaf architecture in <em>Zea mays</em>.<a reveal-id="dsx013-B12" data-open="dsx013-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx013-B13" data-open="dsx013-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Compared to QTL mapping, GWAS can achieve a higher resolution of the underlying genetic loci and does not require a mapping population.<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> A total of 40 SNP peaks were detected for symbiotic nitrogen fixation traits in bean (<em>Phaseolus vulgaris</em>).<a reveal-id="dsx013-B15" data-open="dsx013-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Six SNPs were attributed to root efficiency of P uptake in traditional rice.<a reveal-id="dsx013-B16" data-open="dsx013-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Seventy-four significant SNPs associated with P-efficiency were detected in soybean (<em>Glycine max</em>).<a reveal-id="dsx013-B17" data-open="dsx013-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Candidate genes for P efficiency in Arabidopsis,<a reveal-id="dsx013-B18" data-open="dsx013-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> rice<a reveal-id="dsx013-B11" data-open="dsx013-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and maize<a reveal-id="dsx013-B19" data-open="dsx013-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> have been identified. GWAS has also been applied to the dissection of complex traits in <em>B. napus</em>, e.g. flowering time,<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> seed weight and seed quality,<a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> seed oil content,<a reveal-id="dsx013-B21" data-open="dsx013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> stem resistance to <em>Sclerotinia sclerotiorum</em><a reveal-id="dsx013-B22" data-open="dsx013-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and seedling shoot ionomic traits.<a reveal-id="dsx013-B23" data-open="dsx013-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p><p>Haplotypes comprise SNPs or other markers on the same chromosome that are typically inherited together with little chance of recombination.<a reveal-id="dsx013-B24" data-open="dsx013-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> To understand the effects of allelic variation on seed oil content, 324 accessions of <em>B. napus</em> with favourable and undesirable haplotypes at ≥ 15 loci were grouped into four classes.<a reveal-id="dsx013-B21" data-open="dsx013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Other important haplotypes were discovered for resistance to <em>Sclerotina stem</em> rot in <em>B. napus</em><a reveal-id="dsx013-B22" data-open="dsx013-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and P efficiency in soybean.<a reveal-id="dsx013-B17" data-open="dsx013-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p><p>In the present study, an association panel of 405 <em>B. napus</em> cultivars was genotyped with the 60K <em>Brassica</em> Infinium SNP array. RSA traits were investigated in ‘pouch and wick’ and hydroponic system experiments,<a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx013-B25" data-open="dsx013-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> under low and high P treatments. The aims of this study were (i) to determine the genetic diversity and RSA of the population, (ii) to investigate the genetics of RSA under high and low P, and (iii) to predict candidate genes associated with the genetic loci to (iv) reveal a favourable haplotype for breeding P-efficient <em>B. napus</em>.</p>                            <h2 scrollto-destination=106357217 id="106357217" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=106357218 id="106357218" class="section-title" >2.1 Plant material</h3>
<p>The association panel comprised 405 <em>B. napus</em> cultivars and inbred lines, including 342 semi-winter, 34 spring, 26 winter and 3 unknown types, collected from major breeding centers across China.<a reveal-id="dsx013-B21" data-open="dsx013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> 370 lines originated in China, 20 from Europe, 5 from Canada, 4 from Australia, 4 from Korea and 2 from Japan (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s1_dsx013.xlsx?Expires=2147483647&Signature=OxQkLsitoMyFFYOhWCHthnOq49K6I8y6sLJepzaLkFzRDL4WQdzPBivTVrO~5oM-QI-32whWtq1Wsv6h76RlfWXLqiOnI5tba2e0xHKo7LHwAgxMe9HusWwAGLBN~rsNmga1zb5ruq6K-3wKjnh74exh9l5592ey1PP7ntpRAUsmMfc2B5jvaxvg0fKCaXt8SUqxIlmo04DIhJMrzmwDbDA1-c9LaW~OpiIaqYXgDJD4YjweRKRANGUjMSWhiNtEGE7q6BUGM07JsDOx-GzBbpnSugIW3I8Kp97paef-V0Q1VCexKSMntQzPWHJuslTiQSVt1PQi9LhZHLgwSulK2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p>                            <h3 scrollto-destination=106357220 id="106357220" class="section-title" >2.2 Phenotyping root system architecture</h3>
<p>The root system architecture (RSA) of the panel was screened at low P (LP: 5 μmol l<sup>−1</sup> P) and normal P (NP: 250 μmol l<sup>−1</sup> P) using a modified standard Hoagland’s solution in 2013-14. The seeds were sterilized using 70% (v/v) ethanol and NaOCl (2.5% active chlorine), and then germinated on gauze with pure water for three days. The seedlings were transplanted to a hydroponic growth tank and grown for the first week with a 25% standard Hoagland’s solution, and then with a 50% standard Hoagland’s solution followed by a 100% standard Hoagland’s solution for a further five days. After 12 days the primary root length (PRL) was measured by a ruler. Seedling samples were dried at 80 °C for seven days for dry weight measurements (shoot dry weight − SDW; root dry weight − RDW). The panel was also screened at LP (LP: 0 μmol l<sup>−1</sup> P) using a modified Hoagland’s solution whereby the KH<sub>2</sub>PO<sub>4</sub> was replaced with KCl. The seedlings were grown using a ‘pouch and wick’ system,<a reveal-id="dsx013-B26" data-open="dsx013-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> adapted by Thomas.<a reveal-id="dsx013-B25" data-open="dsx013-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Briefly, this method comprises growth pouches composed of germination paper down with the seedling growing between sheets of black polythene, the pouch is hung in a tank and suspended in nutrient solution, and each tank can accommodate 96 pouches (192 seedlings, one on each side of the paper). For each line, 16 seeds were sown across eight independent replicates distributed across four different aluminium frames. The nutrient solution was changed every 3 days. The RSA of samples were imaged at 14 days as previously.<a reveal-id="dsx013-B25" data-open="dsx013-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> The RSA including PRL (primary root length), LRN (lateral root number), MLRL (mean lateral root length = LRL/LRN), LRL (lateral root length), TRL (total root length = PRL + LRL) and LRD (lateral root density = LRN/PRL) was measured using RootReader 2D (<a class="link link-uri" href="http://www.plantmineralnutrition.net/rr2d.php" target="_blank">http://www.plantmineralnutrition.net/rr2d.php</a>). Shoot and root samples were dried at 80 °C and dry weights (shoot, SDW; root, RDW) were determined (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s1_dsx013.docx?Expires=2147483647&Signature=fv5qu4w438o~SjSQHf-QJtbVQWJlaQPsr9RD6T1TZAe1MTBBPcT9RVt418eorgHnjgxjq~z43OKJWKTDF~-rAEY63fcA8pMwzb9sAFDMEoX2HWOtjug6xgh6ZWUr9wrN2FRivwxqklZhb-R0DoIqa2V0jXB-uyEhlcc3msisUKCg26nNRWSTCVt-IzFmSkCxRI6UtLeTuIhJJQSbqOD8qces2yUw-V0I-UcBMZ2Y1KMQkdxHM0GV22KlwmdkCRACmJIixiWSWl03T1QJq2aKuTkq6Fimqy7cE5djMMMjk8mEhbrfgq0e6dr2DrDQfWxaWqeJCZh~egVpJ6feF7mIaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).</p>                            <h3 scrollto-destination=106357222 id="106357222" class="section-title" >2.3 Data analysis</h3>
<p>Restricted maximum likelihood (REML) procedures were used to estimate line means and sources of variation in root traits.<a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx013-B25" data-open="dsx013-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Means were estimated using the [Genotype] term as a fixed factor, retaining [(Replicate/Side of paper/Tray/Frame/Room)] as random factors.<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Random terms and no defined fixed factor were used to estimate sources of variation. Statistical analysis was completed using GenStat15<sup>th</sup> Edition (VSN International, Oxford, UK).</p>                            <h3 scrollto-destination=106357224 id="106357224" class="section-title" >2.4 Genotyping of the association panel and <em>in silico</em> mapping of SNPs</h3>
<p>Four young leaves from different individuals of each variety were planted in Wuhan and were sampled. DNA was extracted using a modified CTAB method<a reveal-id="dsx013-B27" data-open="dsx013-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and then adjusted to 50 ng μl<sup>−1</sup>. The 60K <em>Brassica</em> Infinium SNP array was employed and the SNP data were analyzed using Illumina BeadStudio (Illumina Inc. San Diego, California, USA).<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx013-B21" data-open="dsx013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The SNPs with call frequencies &gt; 0.8 or minor allele frequencies (MAF) &gt; 0.05 and homozygous genotype frequency &gt; 0 were selected for association mapping analysis.<a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The physical position of the SNPs was identified by aligning the sequence of a 50 bp SNP probe attached to each SNP with the genome sequences of <em>B. napus</em><a reveal-id="dsx013-B28" data-open="dsx013-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> using local BLASTn (BLAST: Basic Local Alignment Search Tool, <a class="link link-uri" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). If the SNP probe matched two or more locations in the reference genome, the SNPs were regarded as non-specific markers and abandoned.<a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a></p>                            <h3 scrollto-destination=106357226 id="106357226" class="section-title" >2.5 Population structure, relative kinship and linkage disequilibrium analysis of the association panel</h3>
<p>The population structure was estimated using the Bayesian Markov chain Monte Carlo model (MCMC) in STRUCTURE V. 2.3.4.<a reveal-id="dsx013-B29" data-open="dsx013-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsx013-B30" data-open="dsx013-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Each K value (the number of real populations) was obtained with five independent runs with a putative number of populations set from 1 to 10, based on a model for admixture and correlated allele frequencies. The length of the burn-in period and number of MCMC replications after burn-in were each set to 100,000. Based on the rate of change in log probability of the data (LnP(D)) and an <em>adhoc</em> statistic Δ<em>K</em> between successive K values, the true K value was determined.<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The Q matrix, which integrates the cluster membership coefficient matrices of replicate runs from STRUCTURE, was determined using CLUMPP (<a class="link link-uri" href="http://www.stanford.edu/group/rosenberglab/clumpp.html" target="_blank">http://www.stanford.edu/group/rosenberglab/clumpp.html</a>). Genetic distances among varieties were calculated by PowerMarker version 3.25 (<a class="link link-uri" href="http://statgen.ncsu.edu/powermarker/downloads.htm" target="_blank">http://statgen.ncsu.edu/powermarker/downloads.htm</a>). Based on the analysis of molecular variance, AMOVA, the subgroup differentiation, pairwise <em>F<sub>ST</sub></em>, was measured using ArlequinV. 3.5.2.2 (<a class="link link-uri" href="http://cmpg.unibe.ch/software/arlequin35" target="_blank">http://cmpg.unibe.ch/software/arlequin35</a><span class="underline">/</span>). After double centering, distance matrices were used to obtain ‘eigenvectors’ in NTSYS V. 2.2 (<a class="link link-uri" href="http://www.exetersoftware.com/cat/ntsyspc/ntsyspc.html" target="_blank">http://www.exetersoftware.com/cat/ntsyspc/ntsyspc.html</a>). The relative kinship matrix was calculated by SPAGeDi (<a class="link link-uri" href="http://ebe.ulb.ac.be/ebe/SPAGeDi.html" target="_blank">http://ebe.ulb.ac.be/ebe/SPAGeDi.html</a>) when all negative kinship values between two individuals were set to 0. The polymorphism information content (PIC) and gene diversity of the SNP markers were estimated using PowerMarker version 3.25 (<a class="link link-uri" href="http://statgen.ncsu.edu/powermarker" target="_blank">http://statgen.ncsu.edu/powermarker</a><span class="underline">/</span>). The parameter <em>r</em><sup>2</sup> between each pair of markers was used to estimate the linkage disequilibrium (LD) using TASSEL V. 4.0.<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p>                            <h3 scrollto-destination=106357228 id="106357228" class="section-title" >2.6 Genome-wide association analysis of RSA traits</h3>
<p>Population structure and kinship were used to correct for spurious association created by genotype-phenotype covariance. GLM (generalized linear model) and mixed linear model (MLM) calculations were performed in TASSEL to determine the best model for association analyses. The GLM included a naïve model without controlling for population structure, and the Q model which controlled for population structure using the Q matrix to identify populations. The MLM included the K model which assesses inter-individual relative kinship using the kinship matrix obtained in SPAGeDi software, and the Q + K model which controlled for population structure with the Q matrix, and kinship with the kinship matrix. Quantile–quantile plots were created with a negative log<sub>10</sub> (P) value of the expected <em>P</em>-value from the genotype–phenotype association and the expected <em>P</em>-value from the assumption that no association exists between genotype and phenotype. The threshold of significance was set to <em>P &lt;</em> 5.15 × 10<sup>−5</sup> (1/total SNPs used).<a reveal-id="dsx013-B22" data-open="dsx013-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p>                            <h3 scrollto-destination=106357230 id="106357230" class="section-title" >2.7 Differential expression genes underlying the QTL and significant SNP loci</h3>
<p>The <em>B. napus</em> cultivar ‘Eyou Changjia’ were planted at normal P (250 μmol l<sup>−1</sup>) for ten days, and then half of the plants were transferred to a nutrition solution with zero P (0 μmol l<sup>−1</sup>) for five days. The shoot (S) and root (R) tissues of three plants at normal P and zero P were separated and immediately placed in liquid nitrogen and stored at 80 °C. The sequencing libraries of twelve RNA samples: S1_LP, S2_LP, S3_LP, S1_NP, S2_NP, S3_NP, R1_LP, R2_LP, R3_LP, R1_NP, R2_NP and R3_NP, were generated using the Illumina RNA Library Prep Kit and sequenced on an Illumina Hiseq 2000 platform with 100-bp paired-end reads. The <em>B. napus</em> reference genome was constructed using bowtie 2 (Broad Institute, Cambridge, MA, USA). Sequencing reads were aligned to the <em>B. napus</em> reference genome with bowtie2 and then assembled using TopHat 2.0.0 and Cufflinks.<a reveal-id="dsx013-B22" data-open="dsx013-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Gene expression was estimated using FPKM (fragments per kilobase of exon per million mapped fragments). Differentially expressed genes (DEGs) between two samples were identified with Cuffdiff, based on criteria False Discovery Rate (FDR) &lt; 0.01 and |log<sub>2</sub> (FPKM_LP/FPKM_NP)|&gt;2 (Ding, unpublished data).</p><p>The QTLs for root traits previously detected in the Tapidor x Ningyou7 doubled haploid (<em>Bna</em>TNDH) population<a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> were explored for colocalization with SNP marker associations in the present study. The SNP markers flanking co-located QTL and SNP regions on chromosomes A3 were mapped onto the genome of cultivar Darmor-<em>bzh</em>, respectively. The intervals of a significant locus ranged from loci minus LD to loci plus LD. Genes/transcription factors which were located in the homologous physical regions of the reference map and previously identified as DEGs (differentially expressed genes) responding to high and low P in the <em>B. napus</em> cultivar ‘Eyou changjia’, were selected as putative candidate genes: <em>BnaA03g05800D, BnaA03g05820D</em>, <em>BnaA03g05830D</em>, <em>BnaA03g05850D</em>, <em>BnaA05g22460D</em>, <em>BnaA05g22070D</em>.</p>                            <h3 scrollto-destination=106357233 id="106357233" class="section-title" >2.8 Expression profile of putative candidate genes in P-efficient and P-inefficient <em>B. napus</em> cultivars</h3>
<p>The P efficiency of the varieties was evaluated using a P efficiency coefficient of the ratio of SDW under LP/SDW under HP.<a reveal-id="dsx013-B31" data-open="dsx013-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Two P-efficient (121 and 246) and two P-inefficient (292 and 453) varieties were selected for gene expression analysis of putative candidate genes. Plants were grown as above in the hydroponic system under a modified standard Hoagland’s solution for 10 days, and then transplanted to zero P for a further 5 days. Leaves and root were sampled and immediately snap-frozen in liquid nitrogen and stored at −80 °C for downstream RNA extraction. Total cDNA was synthesized from 1 μg RNA with RevertAid First Stand cDNA Synthesis kit (Cat No: K1621, Thermo Scientific, Lithuania, EU). Primers are listed in <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s2_dsx013.xlsx?Expires=2147483647&Signature=FWe7Yrdg9FWib4z0QCNID9TyUJtu4MQT25X7FHJzkKfGk5OoD7WfDwwAHSKQ-d6rKb9Od0-a8GZXfCeEx3NVWTbZ24Uj-VG9XE4sB11Ov~jsfOhU4r1yiP3JGujjmfjZQ1nUGk0B7IdqtrO-GOKnM~CgknkzB01N74FD0MXf4XZKU~7AVZdV9J2HJCNiaAC5uTa2Uez1kN1v8CALREHEg2p1qpJuhH4T~0W1ajszUcFYEjJKuazdCngX49WPMg5sqlECdWVo1ERR4C7tRaRBqJYMAWy14V9JAWpo3~g9ISNTefKdyBhCkt88pN3roAbD26bVErKNRd10a86AuNTVtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The final reaction mixture for RT-PCR contained 10 μl SSo Advanced SYBR Green Supermix (Bio-Rad), 2.0 μl cDNA, 1 μl primer and 7 μl distilled water. Real-time PCR was carried out in triplicate on each sample using Bio-Rad CFX Real-Time PCR System.<a reveal-id="dsx013-B22" data-open="dsx013-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Relative expression levels were evaluated using the 2<sup>-△△CT</sup> method. Actin was used as an internal control for normalization. The relative expression of genes was normalized to the level of a reference gene <em>BnPAP17</em> in the P- efficient cultivar (246) at low P.</p>                            <h3 scrollto-destination=106357235 id="106357235" class="section-title" >2.9 Geographical distribution of <em>B. napus</em> haplotypes</h3>
<p>There were 21 significant SNPs for the RSA traits at LP, which formed a genetic cluster on A3 within a target QTL region. TASSEL version 4.0 was used to analyze the sliding window of LD on chromosome A3. The SNPs located in the LD intervals in the A3 cluster region were employed to identify the haplotype blocks in the accessions. The geographical distribution of haplotypes was predicted through a joint analysis of correlations between haplotypes and RSA traits, and origin of the panel varieties.</p>                            <h2 scrollto-destination=106357237 id="106357237" class="section-title" >3 Results</h2>
                            <h3 scrollto-destination=106357238 id="106357238" class="section-title" >3.1 Phenotypic variation in root system architecture at low and normal P</h3>
<p>Extensive phenotypic variation was observed in all traits under the two growth systems (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s1_dsx013.docx?Expires=2147483647&Signature=fv5qu4w438o~SjSQHf-QJtbVQWJlaQPsr9RD6T1TZAe1MTBBPcT9RVt418eorgHnjgxjq~z43OKJWKTDF~-rAEY63fcA8pMwzb9sAFDMEoX2HWOtjug6xgh6ZWUr9wrN2FRivwxqklZhb-R0DoIqa2V0jXB-uyEhlcc3msisUKCg26nNRWSTCVt-IzFmSkCxRI6UtLeTuIhJJQSbqOD8qces2yUw-V0I-UcBMZ2Y1KMQkdxHM0GV22KlwmdkCRACmJIixiWSWl03T1QJq2aKuTkq6Fimqy7cE5djMMMjk8mEhbrfgq0e6dr2DrDQfWxaWqeJCZh~egVpJ6feF7mIaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). For example, in the hydroponic system (HS) under normal P (NP) in 2013, mean values ranged as follows: SDW 0.086–0.834 g, RDW 0.0080–0.081 g, R/S ratio 0.023–0.213 and PRL 13.73–37.67 cm; and under low P (LP), SDW 0.063–0.213 g, RDW 0.022–0.065 g, R/S ratio 0.198–0.502 and PRL 23.02–42.05 cm. In the HS the RDW, R/S ratio and PRL were all greater at LP as compared with NP in both experimental years (HSI; HSII) (<a reveal-id="dsx013-T1" data-open="dsx013-T1" class="link link-reveal link-table xref-fig">Table 1</a>), and the coefficient of variation (CV) at NP was higher. In the ‘pouch and wick’ system (PS) experiment at LP, the panel phenotypes are shown in <a reveal-id="dsx013-T2" data-open="dsx013-T2" class="link link-reveal link-table xref-fig">Table 2</a>. All traits showed continuous variation with an approximate normal distribution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s1_dsx013.docx?Expires=2147483647&Signature=fv5qu4w438o~SjSQHf-QJtbVQWJlaQPsr9RD6T1TZAe1MTBBPcT9RVt418eorgHnjgxjq~z43OKJWKTDF~-rAEY63fcA8pMwzb9sAFDMEoX2HWOtjug6xgh6ZWUr9wrN2FRivwxqklZhb-R0DoIqa2V0jXB-uyEhlcc3msisUKCg26nNRWSTCVt-IzFmSkCxRI6UtLeTuIhJJQSbqOD8qces2yUw-V0I-UcBMZ2Y1KMQkdxHM0GV22KlwmdkCRACmJIixiWSWl03T1QJq2aKuTkq6Fimqy7cE5djMMMjk8mEhbrfgq0e6dr2DrDQfWxaWqeJCZh~egVpJ6feF7mIaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). <div content-id="dsx013-T1" class="table-modal"><div class="table-wrap"><div id="dsx013-T1" data-id="dsx013-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Mean, standard deviation (SD), minimum(min), maximum (max) and Coefficient of Variation (CV, %) of the shoot dry weight (SDW, g), root dry weight (RDW, g), R/S ratio and primary root length (PRL) in the OSR panel, in the hydroponic system experiments in 2013 (HSI) and 2014 (HSII), under low phosphorus (LP) and normal phosphorus (NP) treatments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">P Level </th><th colspan="2">Mean ± SD<hr /> </th><th colspan="2">Min<hr /> </th><th colspan="2">Max<hr /> </th><th colspan="2">CV (%)<hr /> </th></tr><tr><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th></tr></thead><tbody><tr><td rowspan="2">SDW (g) </td><td>LP </td><td>0.129<sup>b</sup><strong>±</strong> 0.026 </td><td>0.105<sup>ab</sup><strong>±</strong> 0.028 </td><td>0.063 </td><td>0.030 </td><td>0.213 </td><td>0.209 </td><td>20.0 </td><td>26.3 </td></tr><tr><td>NP </td><td>0.357<sup>a</sup><strong>±</strong> 0.136 </td><td>0.372<sup>a</sup><strong>±</strong> 0.144 </td><td>0.086 </td><td>0.069 </td><td>0.834 </td><td>1.266 </td><td>38.1 </td><td>38.8 </td></tr><tr><td rowspan="2">RDW (g) </td><td>LP </td><td>0.041<sup>a</sup><strong>±</strong> 0.008 </td><td>0.032<sup>a</sup><strong>±</strong> 0.008 </td><td>0.022 </td><td>0.012 </td><td>0.065 </td><td>0.0623 </td><td>19.7 </td><td>25.1 </td></tr><tr><td>NP </td><td>0.032<sup>ab</sup><strong>±</strong> 0.012 </td><td>0.030<sup>ab</sup><strong>±</strong> 0.014 </td><td>0.008 </td><td>0.006 </td><td>0.081 </td><td>0.147 </td><td>36.4 </td><td>41.9 </td></tr><tr><td rowspan="2">R/S ratio </td><td>LP </td><td>0.322<sup>a</sup><strong>±</strong> 0.049 </td><td>0.312<sup>a</sup><strong>±</strong> 0.055 </td><td>0.198 </td><td>0.178 </td><td>0.502 </td><td>0.499 </td><td>15.2 </td><td>17.5 </td></tr><tr><td>NP </td><td>0.096<sup>b</sup><strong>±</strong> 0.031 </td><td>0.091<sup>c</sup><strong>±</strong> 0.028 </td><td>0.023 </td><td>0.038 </td><td>0.213 </td><td>0.249 </td><td>32.2 </td><td>31.1 </td></tr><tr><td rowspan="2">PRL (cm) </td><td>LP </td><td>31.00<sup>a</sup><strong>±</strong> 2.55 </td><td>32.40<sup>a</sup><strong>±</strong> 3.33 </td><td>23.02 </td><td>21.69 </td><td>42.05 </td><td>41.78 </td><td>8.2 </td><td>10.38 </td></tr><tr><td>NP </td><td>25.8<sup>ab</sup><strong>±</strong> 3.75 </td><td>24.2<sup>ab</sup><strong>±</strong> 2.85 </td><td>13.73 </td><td>15.61 </td><td>37.67 </td><td>37.36 </td><td>14.5 </td><td>11.8 </td></tr></tbody></table></div><div content-id="dsx013-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">P Level </th><th colspan="2">Mean ± SD<hr /> </th><th colspan="2">Min<hr /> </th><th colspan="2">Max<hr /> </th><th colspan="2">CV (%)<hr /> </th></tr><tr><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th></tr></thead><tbody><tr><td rowspan="2">SDW (g) </td><td>LP </td><td>0.129<sup>b</sup><strong>±</strong> 0.026 </td><td>0.105<sup>ab</sup><strong>±</strong> 0.028 </td><td>0.063 </td><td>0.030 </td><td>0.213 </td><td>0.209 </td><td>20.0 </td><td>26.3 </td></tr><tr><td>NP </td><td>0.357<sup>a</sup><strong>±</strong> 0.136 </td><td>0.372<sup>a</sup><strong>±</strong> 0.144 </td><td>0.086 </td><td>0.069 </td><td>0.834 </td><td>1.266 </td><td>38.1 </td><td>38.8 </td></tr><tr><td rowspan="2">RDW (g) </td><td>LP </td><td>0.041<sup>a</sup><strong>±</strong> 0.008 </td><td>0.032<sup>a</sup><strong>±</strong> 0.008 </td><td>0.022 </td><td>0.012 </td><td>0.065 </td><td>0.0623 </td><td>19.7 </td><td>25.1 </td></tr><tr><td>NP </td><td>0.032<sup>ab</sup><strong>±</strong> 0.012 </td><td>0.030<sup>ab</sup><strong>±</strong> 0.014 </td><td>0.008 </td><td>0.006 </td><td>0.081 </td><td>0.147 </td><td>36.4 </td><td>41.9 </td></tr><tr><td rowspan="2">R/S ratio </td><td>LP </td><td>0.322<sup>a</sup><strong>±</strong> 0.049 </td><td>0.312<sup>a</sup><strong>±</strong> 0.055 </td><td>0.198 </td><td>0.178 </td><td>0.502 </td><td>0.499 </td><td>15.2 </td><td>17.5 </td></tr><tr><td>NP </td><td>0.096<sup>b</sup><strong>±</strong> 0.031 </td><td>0.091<sup>c</sup><strong>±</strong> 0.028 </td><td>0.023 </td><td>0.038 </td><td>0.213 </td><td>0.249 </td><td>32.2 </td><td>31.1 </td></tr><tr><td rowspan="2">PRL (cm) </td><td>LP </td><td>31.00<sup>a</sup><strong>±</strong> 2.55 </td><td>32.40<sup>a</sup><strong>±</strong> 3.33 </td><td>23.02 </td><td>21.69 </td><td>42.05 </td><td>41.78 </td><td>8.2 </td><td>10.38 </td></tr><tr><td>NP </td><td>25.8<sup>ab</sup><strong>±</strong> 3.75 </td><td>24.2<sup>ab</sup><strong>±</strong> 2.85 </td><td>13.73 </td><td>15.61 </td><td>37.67 </td><td>37.36 </td><td>14.5 </td><td>11.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Values with different letters show significant differences at <em>P</em>&lt;0.05.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx013-T1" data-id="dsx013-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Mean, standard deviation (SD), minimum(min), maximum (max) and Coefficient of Variation (CV, %) of the shoot dry weight (SDW, g), root dry weight (RDW, g), R/S ratio and primary root length (PRL) in the OSR panel, in the hydroponic system experiments in 2013 (HSI) and 2014 (HSII), under low phosphorus (LP) and normal phosphorus (NP) treatments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">P Level </th><th colspan="2">Mean ± SD<hr /> </th><th colspan="2">Min<hr /> </th><th colspan="2">Max<hr /> </th><th colspan="2">CV (%)<hr /> </th></tr><tr><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th></tr></thead><tbody><tr><td rowspan="2">SDW (g) </td><td>LP </td><td>0.129<sup>b</sup><strong>±</strong> 0.026 </td><td>0.105<sup>ab</sup><strong>±</strong> 0.028 </td><td>0.063 </td><td>0.030 </td><td>0.213 </td><td>0.209 </td><td>20.0 </td><td>26.3 </td></tr><tr><td>NP </td><td>0.357<sup>a</sup><strong>±</strong> 0.136 </td><td>0.372<sup>a</sup><strong>±</strong> 0.144 </td><td>0.086 </td><td>0.069 </td><td>0.834 </td><td>1.266 </td><td>38.1 </td><td>38.8 </td></tr><tr><td rowspan="2">RDW (g) </td><td>LP </td><td>0.041<sup>a</sup><strong>±</strong> 0.008 </td><td>0.032<sup>a</sup><strong>±</strong> 0.008 </td><td>0.022 </td><td>0.012 </td><td>0.065 </td><td>0.0623 </td><td>19.7 </td><td>25.1 </td></tr><tr><td>NP </td><td>0.032<sup>ab</sup><strong>±</strong> 0.012 </td><td>0.030<sup>ab</sup><strong>±</strong> 0.014 </td><td>0.008 </td><td>0.006 </td><td>0.081 </td><td>0.147 </td><td>36.4 </td><td>41.9 </td></tr><tr><td rowspan="2">R/S ratio </td><td>LP </td><td>0.322<sup>a</sup><strong>±</strong> 0.049 </td><td>0.312<sup>a</sup><strong>±</strong> 0.055 </td><td>0.198 </td><td>0.178 </td><td>0.502 </td><td>0.499 </td><td>15.2 </td><td>17.5 </td></tr><tr><td>NP </td><td>0.096<sup>b</sup><strong>±</strong> 0.031 </td><td>0.091<sup>c</sup><strong>±</strong> 0.028 </td><td>0.023 </td><td>0.038 </td><td>0.213 </td><td>0.249 </td><td>32.2 </td><td>31.1 </td></tr><tr><td rowspan="2">PRL (cm) </td><td>LP </td><td>31.00<sup>a</sup><strong>±</strong> 2.55 </td><td>32.40<sup>a</sup><strong>±</strong> 3.33 </td><td>23.02 </td><td>21.69 </td><td>42.05 </td><td>41.78 </td><td>8.2 </td><td>10.38 </td></tr><tr><td>NP </td><td>25.8<sup>ab</sup><strong>±</strong> 3.75 </td><td>24.2<sup>ab</sup><strong>±</strong> 2.85 </td><td>13.73 </td><td>15.61 </td><td>37.67 </td><td>37.36 </td><td>14.5 </td><td>11.8 </td></tr></tbody></table></div><div content-id="dsx013-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">P Level </th><th colspan="2">Mean ± SD<hr /> </th><th colspan="2">Min<hr /> </th><th colspan="2">Max<hr /> </th><th colspan="2">CV (%)<hr /> </th></tr><tr><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th><th>2013 </th><th>2014 </th></tr></thead><tbody><tr><td rowspan="2">SDW (g) </td><td>LP </td><td>0.129<sup>b</sup><strong>±</strong> 0.026 </td><td>0.105<sup>ab</sup><strong>±</strong> 0.028 </td><td>0.063 </td><td>0.030 </td><td>0.213 </td><td>0.209 </td><td>20.0 </td><td>26.3 </td></tr><tr><td>NP </td><td>0.357<sup>a</sup><strong>±</strong> 0.136 </td><td>0.372<sup>a</sup><strong>±</strong> 0.144 </td><td>0.086 </td><td>0.069 </td><td>0.834 </td><td>1.266 </td><td>38.1 </td><td>38.8 </td></tr><tr><td rowspan="2">RDW (g) </td><td>LP </td><td>0.041<sup>a</sup><strong>±</strong> 0.008 </td><td>0.032<sup>a</sup><strong>±</strong> 0.008 </td><td>0.022 </td><td>0.012 </td><td>0.065 </td><td>0.0623 </td><td>19.7 </td><td>25.1 </td></tr><tr><td>NP </td><td>0.032<sup>ab</sup><strong>±</strong> 0.012 </td><td>0.030<sup>ab</sup><strong>±</strong> 0.014 </td><td>0.008 </td><td>0.006 </td><td>0.081 </td><td>0.147 </td><td>36.4 </td><td>41.9 </td></tr><tr><td rowspan="2">R/S ratio </td><td>LP </td><td>0.322<sup>a</sup><strong>±</strong> 0.049 </td><td>0.312<sup>a</sup><strong>±</strong> 0.055 </td><td>0.198 </td><td>0.178 </td><td>0.502 </td><td>0.499 </td><td>15.2 </td><td>17.5 </td></tr><tr><td>NP </td><td>0.096<sup>b</sup><strong>±</strong> 0.031 </td><td>0.091<sup>c</sup><strong>±</strong> 0.028 </td><td>0.023 </td><td>0.038 </td><td>0.213 </td><td>0.249 </td><td>32.2 </td><td>31.1 </td></tr><tr><td rowspan="2">PRL (cm) </td><td>LP </td><td>31.00<sup>a</sup><strong>±</strong> 2.55 </td><td>32.40<sup>a</sup><strong>±</strong> 3.33 </td><td>23.02 </td><td>21.69 </td><td>42.05 </td><td>41.78 </td><td>8.2 </td><td>10.38 </td></tr><tr><td>NP </td><td>25.8<sup>ab</sup><strong>±</strong> 3.75 </td><td>24.2<sup>ab</sup><strong>±</strong> 2.85 </td><td>13.73 </td><td>15.61 </td><td>37.67 </td><td>37.36 </td><td>14.5 </td><td>11.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Values with different letters show significant differences at <em>P</em>&lt;0.05.</p></div></div></div></div><div content-id="dsx013-T2" class="table-modal"><div class="table-wrap"><div id="dsx013-T2" data-id="dsx013-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Mean, standard deviation (SD), minimum (min), maximum (max) and coefficient of variation (CV, %) of the shoot dry weight (SDW, g), root dry weight (RDW, g), R/S ratio, primary root length (PRL), lateral root length (LRL), total root length (TRL), lateral root density (LRD), lateral root number (LRN) and mean lateral root length (MLRL) in the OSR panel, in the ‘pouch and wick’ system (PS) experiment, under a low phosphorus (LP) treatment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Mean ± SD </th><th>Min </th><th>Max </th><th>CV (%) </th></tr></thead><tbody><tr><td>SDW (g) </td><td>0.006±0.002 </td><td>0.002 </td><td>0.011 </td><td>28.4 </td></tr><tr><td>RDW (g) </td><td>0.002±0.001 </td><td>0.001 </td><td>0.005 </td><td>31.3 </td></tr><tr><td>R/S ratio </td><td>0.322±0.068 </td><td>0.177 </td><td>0.585 </td><td>21.2 </td></tr><tr><td>PRL (cm) </td><td>12.80±2.70 </td><td>4.50 </td><td>22.70 </td><td>21.2 </td></tr><tr><td>LRL (cm) </td><td>34.50±9.63 </td><td>10.51 </td><td>71.16 </td><td>27.9 </td></tr><tr><td>TRL (cm) </td><td>47.30±11.70 </td><td>18.45 </td><td>90.49 </td><td>24.7 </td></tr><tr><td>LRD (cm/cm<sup>2</sup>) </td><td>1.30±0.210 </td><td>0.53 </td><td>2.17 </td><td>16.3 </td></tr><tr><td>LRN (N) </td><td>16.4±4.3 </td><td>5.0 </td><td>33.5 </td><td>26.4 </td></tr><tr><td>MLRL (cm) </td><td>0.65±0.050 </td><td>0.46 </td><td>0.81 </td><td>7.2 </td></tr></tbody></table></div><div content-id="dsx013-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Mean ± SD </th><th>Min </th><th>Max </th><th>CV (%) </th></tr></thead><tbody><tr><td>SDW (g) </td><td>0.006±0.002 </td><td>0.002 </td><td>0.011 </td><td>28.4 </td></tr><tr><td>RDW (g) </td><td>0.002±0.001 </td><td>0.001 </td><td>0.005 </td><td>31.3 </td></tr><tr><td>R/S ratio </td><td>0.322±0.068 </td><td>0.177 </td><td>0.585 </td><td>21.2 </td></tr><tr><td>PRL (cm) </td><td>12.80±2.70 </td><td>4.50 </td><td>22.70 </td><td>21.2 </td></tr><tr><td>LRL (cm) </td><td>34.50±9.63 </td><td>10.51 </td><td>71.16 </td><td>27.9 </td></tr><tr><td>TRL (cm) </td><td>47.30±11.70 </td><td>18.45 </td><td>90.49 </td><td>24.7 </td></tr><tr><td>LRD (cm/cm<sup>2</sup>) </td><td>1.30±0.210 </td><td>0.53 </td><td>2.17 </td><td>16.3 </td></tr><tr><td>LRN (N) </td><td>16.4±4.3 </td><td>5.0 </td><td>33.5 </td><td>26.4 </td></tr><tr><td>MLRL (cm) </td><td>0.65±0.050 </td><td>0.46 </td><td>0.81 </td><td>7.2 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx013-T2" data-id="dsx013-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Mean, standard deviation (SD), minimum (min), maximum (max) and coefficient of variation (CV, %) of the shoot dry weight (SDW, g), root dry weight (RDW, g), R/S ratio, primary root length (PRL), lateral root length (LRL), total root length (TRL), lateral root density (LRD), lateral root number (LRN) and mean lateral root length (MLRL) in the OSR panel, in the ‘pouch and wick’ system (PS) experiment, under a low phosphorus (LP) treatment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Mean ± SD </th><th>Min </th><th>Max </th><th>CV (%) </th></tr></thead><tbody><tr><td>SDW (g) </td><td>0.006±0.002 </td><td>0.002 </td><td>0.011 </td><td>28.4 </td></tr><tr><td>RDW (g) </td><td>0.002±0.001 </td><td>0.001 </td><td>0.005 </td><td>31.3 </td></tr><tr><td>R/S ratio </td><td>0.322±0.068 </td><td>0.177 </td><td>0.585 </td><td>21.2 </td></tr><tr><td>PRL (cm) </td><td>12.80±2.70 </td><td>4.50 </td><td>22.70 </td><td>21.2 </td></tr><tr><td>LRL (cm) </td><td>34.50±9.63 </td><td>10.51 </td><td>71.16 </td><td>27.9 </td></tr><tr><td>TRL (cm) </td><td>47.30±11.70 </td><td>18.45 </td><td>90.49 </td><td>24.7 </td></tr><tr><td>LRD (cm/cm<sup>2</sup>) </td><td>1.30±0.210 </td><td>0.53 </td><td>2.17 </td><td>16.3 </td></tr><tr><td>LRN (N) </td><td>16.4±4.3 </td><td>5.0 </td><td>33.5 </td><td>26.4 </td></tr><tr><td>MLRL (cm) </td><td>0.65±0.050 </td><td>0.46 </td><td>0.81 </td><td>7.2 </td></tr></tbody></table></div><div content-id="dsx013-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Mean ± SD </th><th>Min </th><th>Max </th><th>CV (%) </th></tr></thead><tbody><tr><td>SDW (g) </td><td>0.006±0.002 </td><td>0.002 </td><td>0.011 </td><td>28.4 </td></tr><tr><td>RDW (g) </td><td>0.002±0.001 </td><td>0.001 </td><td>0.005 </td><td>31.3 </td></tr><tr><td>R/S ratio </td><td>0.322±0.068 </td><td>0.177 </td><td>0.585 </td><td>21.2 </td></tr><tr><td>PRL (cm) </td><td>12.80±2.70 </td><td>4.50 </td><td>22.70 </td><td>21.2 </td></tr><tr><td>LRL (cm) </td><td>34.50±9.63 </td><td>10.51 </td><td>71.16 </td><td>27.9 </td></tr><tr><td>TRL (cm) </td><td>47.30±11.70 </td><td>18.45 </td><td>90.49 </td><td>24.7 </td></tr><tr><td>LRD (cm/cm<sup>2</sup>) </td><td>1.30±0.210 </td><td>0.53 </td><td>2.17 </td><td>16.3 </td></tr><tr><td>LRN (N) </td><td>16.4±4.3 </td><td>5.0 </td><td>33.5 </td><td>26.4 </td></tr><tr><td>MLRL (cm) </td><td>0.65±0.050 </td><td>0.46 </td><td>0.81 </td><td>7.2 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=106357240 id="106357240" class="section-title" >3.2 LD decay, population structure and relative kinship</h3>
<p>The 60K <em>Brassica</em> Infinium SNP array contains 52,157 SNPs. A total of 30,976 matched to a unique location in the reference genome of cultivar darmor<em>-bzh</em> and 19,397 high-quality SNPs with MAF &gt; 0.05 and call frequencies of SNPs &lt; 0.8 (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s3_dsx013.xlsx?Expires=2147483647&amp;Signature=QWtqdeRgICY7seGk2jdhsvJM1V6lkYXNLV7ECip7XYso~9i3UOu1wRuRD~xZ0xywmHDQTfP3DwrBryIMIQB48KSFLwr4h5gNm4qEZJe67IbbGCIOlLLazwqFLQYL4nCvLah3C52JOvglYDbwUhWO1f5OXs8x9VTyvxeXg1b-RiywSy-~viLL~w8BWVMjoBL5lN7~wPALtt8bAl3L9XD754gZ9limivgh9lULF2UOBwV2RA4QYzSbYF5-ed8vsbX845oAoqsyUxylQwuo1YGBIR5Ma8qeqyCEuG5TFHwVigGPl~6nB4wP8PD4Xc3pY9OOIJPTQjY0v0uCBTMXDij23Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) were selected to assess the population structure, relative kinship, LD, and to conduct association analyses. SNP number on each chromosome ranged from 482 on C9 to 2131 on C4. The density of SNPs on each chromosome ranged from 98.31 kb/SNP on C9 to 19.23 kb/SNP on A10 (<a reveal-id="dsx013-T3" data-open="dsx013-T3" class="link link-reveal link-table xref-fig">Table 3</a>). LD decay on each chromosome ranged from 8250 kb on C1 to 125 kb on A3, A4 and A6 when <em>r</em><sup>2</sup> was 0.2. SNP density in the LD region ranged from 4 SNP/LD<sub>chrA2</sub> on A2 to 347 SNP/LD<sub>chrC1</sub> on C1 (<a reveal-id="dsx013-T3" data-open="dsx013-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The LD decay on the A genome, C genome and the whole genome was 1065 kb, 250 kb and 1100 kb, respectively (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). SNP density on chromosomes and in LD regions on the A genome were 27 SNP/kb and 35 SNP/LD<sub>A</sub>, respectively, on the C genome were 36.97 SNP/kb and 10 SNP/LD<sub>C</sub>, respectively and on the whole genome were 33 SNP/kb and 18 SNP/LD<sub>C</sub>, respectively (<a reveal-id="dsx013-T3" data-open="dsx013-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <div content-id="dsx013-T3" class="table-modal"><div class="table-wrap"><div id="dsx013-T3" data-id="dsx013-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>SNP number, LD decay (physical distance on the genome when the value of <em>r</em><sup>2</sup> is 0.2) and SNP density each chromosome in the OSR panel</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>No. of SNPs </th><th>LD decay (kb) </th><th>SNP density per chromosome (SNP/kb) </th><th>SNP density per LD region (SNP/LD<sub>chr</sub>) </th></tr></thead><tbody><tr><td>A1 </td><td>815 </td><td>150 </td><td>28 </td><td>5 </td></tr><tr><td>A2 </td><td>555 </td><td>165 </td><td>45 </td><td>4 </td></tr><tr><td>A3 </td><td>1224 </td><td>125 </td><td>24 </td><td>5 </td></tr><tr><td>A4 </td><td>837 </td><td>125 </td><td>23 </td><td>5 </td></tr><tr><td>A5 </td><td>886 </td><td>200 </td><td>25 </td><td>8 </td></tr><tr><td>A6 </td><td>898 </td><td>125 </td><td>27 </td><td>5 </td></tr><tr><td>A7 </td><td>1051 </td><td>130 </td><td>23 </td><td>6 </td></tr><tr><td>A8 </td><td>565 </td><td>3300 </td><td>33 </td><td>100 </td></tr><tr><td>A9 </td><td>916 </td><td>600 </td><td>37 </td><td>16 </td></tr><tr><td>A10 </td><td>895 </td><td>475 </td><td>19 </td><td>25 </td></tr><tr><td>C1 </td><td>1626 </td><td>8250 </td><td>24 </td><td>347 </td></tr><tr><td>C2 </td><td>1586 </td><td>7125 </td><td>29 </td><td>250 </td></tr><tr><td>C3 </td><td>1754 </td><td>400 </td><td>35 </td><td>12 </td></tr><tr><td>C4 </td><td>2131 </td><td>6000 </td><td>23 </td><td>262 </td></tr><tr><td>C5 </td><td>505 </td><td>575 </td><td>84 </td><td>7 </td></tr><tr><td>C6 </td><td>675 </td><td>840 </td><td>55 </td><td>15 </td></tr><tr><td>C7 </td><td>1016 </td><td>775 </td><td>39 </td><td>20 </td></tr><tr><td>C8 </td><td>980 </td><td>1800 </td><td>38 </td><td>47 </td></tr><tr><td>C9 </td><td>482 </td><td>1350 </td><td>98 </td><td>14 </td></tr><tr><td>A genome </td><td>8642 </td><td>250 </td><td>27 </td><td>35 </td></tr><tr><td>C genome </td><td>10755 </td><td>1100 </td><td>37 </td><td>10 </td></tr><tr><td>Whole genome </td><td>19397 </td><td>1065 </td><td>33 </td><td>18 </td></tr></tbody></table></div><div content-id="dsx013-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>No. of SNPs </th><th>LD decay (kb) </th><th>SNP density per chromosome (SNP/kb) </th><th>SNP density per LD region (SNP/LD<sub>chr</sub>) </th></tr></thead><tbody><tr><td>A1 </td><td>815 </td><td>150 </td><td>28 </td><td>5 </td></tr><tr><td>A2 </td><td>555 </td><td>165 </td><td>45 </td><td>4 </td></tr><tr><td>A3 </td><td>1224 </td><td>125 </td><td>24 </td><td>5 </td></tr><tr><td>A4 </td><td>837 </td><td>125 </td><td>23 </td><td>5 </td></tr><tr><td>A5 </td><td>886 </td><td>200 </td><td>25 </td><td>8 </td></tr><tr><td>A6 </td><td>898 </td><td>125 </td><td>27 </td><td>5 </td></tr><tr><td>A7 </td><td>1051 </td><td>130 </td><td>23 </td><td>6 </td></tr><tr><td>A8 </td><td>565 </td><td>3300 </td><td>33 </td><td>100 </td></tr><tr><td>A9 </td><td>916 </td><td>600 </td><td>37 </td><td>16 </td></tr><tr><td>A10 </td><td>895 </td><td>475 </td><td>19 </td><td>25 </td></tr><tr><td>C1 </td><td>1626 </td><td>8250 </td><td>24 </td><td>347 </td></tr><tr><td>C2 </td><td>1586 </td><td>7125 </td><td>29 </td><td>250 </td></tr><tr><td>C3 </td><td>1754 </td><td>400 </td><td>35 </td><td>12 </td></tr><tr><td>C4 </td><td>2131 </td><td>6000 </td><td>23 </td><td>262 </td></tr><tr><td>C5 </td><td>505 </td><td>575 </td><td>84 </td><td>7 </td></tr><tr><td>C6 </td><td>675 </td><td>840 </td><td>55 </td><td>15 </td></tr><tr><td>C7 </td><td>1016 </td><td>775 </td><td>39 </td><td>20 </td></tr><tr><td>C8 </td><td>980 </td><td>1800 </td><td>38 </td><td>47 </td></tr><tr><td>C9 </td><td>482 </td><td>1350 </td><td>98 </td><td>14 </td></tr><tr><td>A genome </td><td>8642 </td><td>250 </td><td>27 </td><td>35 </td></tr><tr><td>C genome </td><td>10755 </td><td>1100 </td><td>37 </td><td>10 </td></tr><tr><td>Whole genome </td><td>19397 </td><td>1065 </td><td>33 </td><td>18 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx013-T3" data-id="dsx013-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>SNP number, LD decay (physical distance on the genome when the value of <em>r</em><sup>2</sup> is 0.2) and SNP density each chromosome in the OSR panel</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>No. of SNPs </th><th>LD decay (kb) </th><th>SNP density per chromosome (SNP/kb) </th><th>SNP density per LD region (SNP/LD<sub>chr</sub>) </th></tr></thead><tbody><tr><td>A1 </td><td>815 </td><td>150 </td><td>28 </td><td>5 </td></tr><tr><td>A2 </td><td>555 </td><td>165 </td><td>45 </td><td>4 </td></tr><tr><td>A3 </td><td>1224 </td><td>125 </td><td>24 </td><td>5 </td></tr><tr><td>A4 </td><td>837 </td><td>125 </td><td>23 </td><td>5 </td></tr><tr><td>A5 </td><td>886 </td><td>200 </td><td>25 </td><td>8 </td></tr><tr><td>A6 </td><td>898 </td><td>125 </td><td>27 </td><td>5 </td></tr><tr><td>A7 </td><td>1051 </td><td>130 </td><td>23 </td><td>6 </td></tr><tr><td>A8 </td><td>565 </td><td>3300 </td><td>33 </td><td>100 </td></tr><tr><td>A9 </td><td>916 </td><td>600 </td><td>37 </td><td>16 </td></tr><tr><td>A10 </td><td>895 </td><td>475 </td><td>19 </td><td>25 </td></tr><tr><td>C1 </td><td>1626 </td><td>8250 </td><td>24 </td><td>347 </td></tr><tr><td>C2 </td><td>1586 </td><td>7125 </td><td>29 </td><td>250 </td></tr><tr><td>C3 </td><td>1754 </td><td>400 </td><td>35 </td><td>12 </td></tr><tr><td>C4 </td><td>2131 </td><td>6000 </td><td>23 </td><td>262 </td></tr><tr><td>C5 </td><td>505 </td><td>575 </td><td>84 </td><td>7 </td></tr><tr><td>C6 </td><td>675 </td><td>840 </td><td>55 </td><td>15 </td></tr><tr><td>C7 </td><td>1016 </td><td>775 </td><td>39 </td><td>20 </td></tr><tr><td>C8 </td><td>980 </td><td>1800 </td><td>38 </td><td>47 </td></tr><tr><td>C9 </td><td>482 </td><td>1350 </td><td>98 </td><td>14 </td></tr><tr><td>A genome </td><td>8642 </td><td>250 </td><td>27 </td><td>35 </td></tr><tr><td>C genome </td><td>10755 </td><td>1100 </td><td>37 </td><td>10 </td></tr><tr><td>Whole genome </td><td>19397 </td><td>1065 </td><td>33 </td><td>18 </td></tr></tbody></table></div><div content-id="dsx013-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>No. of SNPs </th><th>LD decay (kb) </th><th>SNP density per chromosome (SNP/kb) </th><th>SNP density per LD region (SNP/LD<sub>chr</sub>) </th></tr></thead><tbody><tr><td>A1 </td><td>815 </td><td>150 </td><td>28 </td><td>5 </td></tr><tr><td>A2 </td><td>555 </td><td>165 </td><td>45 </td><td>4 </td></tr><tr><td>A3 </td><td>1224 </td><td>125 </td><td>24 </td><td>5 </td></tr><tr><td>A4 </td><td>837 </td><td>125 </td><td>23 </td><td>5 </td></tr><tr><td>A5 </td><td>886 </td><td>200 </td><td>25 </td><td>8 </td></tr><tr><td>A6 </td><td>898 </td><td>125 </td><td>27 </td><td>5 </td></tr><tr><td>A7 </td><td>1051 </td><td>130 </td><td>23 </td><td>6 </td></tr><tr><td>A8 </td><td>565 </td><td>3300 </td><td>33 </td><td>100 </td></tr><tr><td>A9 </td><td>916 </td><td>600 </td><td>37 </td><td>16 </td></tr><tr><td>A10 </td><td>895 </td><td>475 </td><td>19 </td><td>25 </td></tr><tr><td>C1 </td><td>1626 </td><td>8250 </td><td>24 </td><td>347 </td></tr><tr><td>C2 </td><td>1586 </td><td>7125 </td><td>29 </td><td>250 </td></tr><tr><td>C3 </td><td>1754 </td><td>400 </td><td>35 </td><td>12 </td></tr><tr><td>C4 </td><td>2131 </td><td>6000 </td><td>23 </td><td>262 </td></tr><tr><td>C5 </td><td>505 </td><td>575 </td><td>84 </td><td>7 </td></tr><tr><td>C6 </td><td>675 </td><td>840 </td><td>55 </td><td>15 </td></tr><tr><td>C7 </td><td>1016 </td><td>775 </td><td>39 </td><td>20 </td></tr><tr><td>C8 </td><td>980 </td><td>1800 </td><td>38 </td><td>47 </td></tr><tr><td>C9 </td><td>482 </td><td>1350 </td><td>98 </td><td>14 </td></tr><tr><td>A genome </td><td>8642 </td><td>250 </td><td>27 </td><td>35 </td></tr><tr><td>C genome </td><td>10755 </td><td>1100 </td><td>37 </td><td>10 </td></tr><tr><td>Whole genome </td><td>19397 </td><td>1065 </td><td>33 </td><td>18 </td></tr></tbody></table></div></div></p>                        <a id="106357242" scrollto-destination="106357242"></a>
<div id="dsx013-F1" data-id="dsx013-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f1.png?Expires=2147483647&Signature=Q-85PGncGXigvLt1urEn3VAfdZ5buMpFWNOpLjnXOakVLCE5iUvHLM7eaHZpo2owfuwRK67L7b2~gaIg8qIa~xFnTHvpNHC6nWU2WiKm36iNzHlbPMbwHraKkqGaBMoFYOJN5RnqR7aLaGtziSHugJi7ocecLLJzGBSTDBZPmNPNOA~MFekMNgexWAEBGmW4Tcx5Fhzs0GI-faDoPKRQ4eZA9vY-AWysR8Q6mzJApBLqwQQD7iqiNkwcQgyU~oPRZryug8iS1yebgzxYqwNAsS33WE25VwIVN0t1WaRI07o4CAihpC~HjJaJXPAgd88yRz0ygMVwGC8dBHIAribtbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) The rate of change in log probability (LnP(D)) and adhoc statistic ΔK (delta K) of population structure in the 405 B. napus population association panel. (b) Linkage disequilibrium (LD) of the B. napus panel and A, C subgenomes. (c) The relative kinship of the B. napus panel and two subgroups. (d) LD, gene diversity, polymorphism information content (PIC) and kinship coefficient of the B. napus panel and two subgroups. (e) Two subgroups of the B. napus panel when K equals 2." path-from-xml="dsx013f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357242/dsx013f1.png" path-from-xml="dsx013f1.png" target="_blank">View large</a><a section="106357242" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f1.png?Expires=2147483647&Signature=qSADbeipJZ1BPbhF5yHiQ~MYWpUet0i1n1Z3S6EFlnkrWg4Mrtu3QlebGCJAk~oAT4jyspnsAGJDDdnkpgFMtc2a~7FratOX7wDRNVxX~rxo4Bn0ZXksSMT4onnNjo~3rTkoZJBcWvuW9JMT5xt-Eqa~DmsdH93AjhU3HFOKZqWNlGnSjgimh~rznMHO5Doxa0KqbEGz7KlaIs1W3JdNSWaN7e2mhxqP-22aPVExRRV-2UgNSUA96-sU~gAj~QKIVuw6-K3Sjme1xV-liStdxPlC-JKWZBu4etx8ywTLJ505X8ghfWgtjw4N3Lt7hX4p8MhOKEybtUUGtsH9qjowgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357242&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The rate of change in log probability (LnP(D)) and adhoc statistic Δ<em>K</em> (delta K) of population structure in the 405 <em>B. napus</em> population association panel. (b) Linkage disequilibrium (LD) of the <em>B. napus</em> panel and A, C subgenomes. (c) The relative kinship of the <em>B. napus</em> panel and two subgroups. (d) LD, gene diversity, polymorphism information content (PIC) and kinship coefficient of the <em>B. napus</em> panel and two subgroups. (e) Two subgroups of the <em>B. napus</em> panel when K equals 2.</p></div></div><div content-id="dsx013-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f1.png?Expires=2147483647&Signature=Q-85PGncGXigvLt1urEn3VAfdZ5buMpFWNOpLjnXOakVLCE5iUvHLM7eaHZpo2owfuwRK67L7b2~gaIg8qIa~xFnTHvpNHC6nWU2WiKm36iNzHlbPMbwHraKkqGaBMoFYOJN5RnqR7aLaGtziSHugJi7ocecLLJzGBSTDBZPmNPNOA~MFekMNgexWAEBGmW4Tcx5Fhzs0GI-faDoPKRQ4eZA9vY-AWysR8Q6mzJApBLqwQQD7iqiNkwcQgyU~oPRZryug8iS1yebgzxYqwNAsS33WE25VwIVN0t1WaRI07o4CAihpC~HjJaJXPAgd88yRz0ygMVwGC8dBHIAribtbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) The rate of change in log probability (LnP(D)) and adhoc statistic ΔK (delta K) of population structure in the 405 B. napus population association panel. (b) Linkage disequilibrium (LD) of the B. napus panel and A, C subgenomes. (c) The relative kinship of the B. napus panel and two subgroups. (d) LD, gene diversity, polymorphism information content (PIC) and kinship coefficient of the B. napus panel and two subgroups. (e) Two subgroups of the B. napus panel when K equals 2." path-from-xml="dsx013f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357242/dsx013f1.png" path-from-xml="dsx013f1.png" target="_blank">View large</a><a section="106357242" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f1.png?Expires=2147483647&Signature=qSADbeipJZ1BPbhF5yHiQ~MYWpUet0i1n1Z3S6EFlnkrWg4Mrtu3QlebGCJAk~oAT4jyspnsAGJDDdnkpgFMtc2a~7FratOX7wDRNVxX~rxo4Bn0ZXksSMT4onnNjo~3rTkoZJBcWvuW9JMT5xt-Eqa~DmsdH93AjhU3HFOKZqWNlGnSjgimh~rznMHO5Doxa0KqbEGz7KlaIs1W3JdNSWaN7e2mhxqP-22aPVExRRV-2UgNSUA96-sU~gAj~QKIVuw6-K3Sjme1xV-liStdxPlC-JKWZBu4etx8ywTLJ505X8ghfWgtjw4N3Lt7hX4p8MhOKEybtUUGtsH9qjowgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357242&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The rate of change in log probability (LnP(D)) and adhoc statistic Δ<em>K</em> (delta K) of population structure in the 405 <em>B. napus</em> population association panel. (b) Linkage disequilibrium (LD) of the <em>B. napus</em> panel and A, C subgenomes. (c) The relative kinship of the <em>B. napus</em> panel and two subgroups. (d) LD, gene diversity, polymorphism information content (PIC) and kinship coefficient of the <em>B. napus</em> panel and two subgroups. (e) Two subgroups of the <em>B. napus</em> panel when K equals 2.</p></div></div><p>The population structure of the <em>B. napus</em> panel was assessed. The K values increased continuously with no obvious inflexion points of LnP(K) values (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). The ΔK values indicated that the rate of change <em>adhoc</em> statistic was equal to 2, therefore the population could be divided into two clusters; Subgroup1 and Subgroup2 (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1e</a>). The <em>F<sub>ST</sub></em> of Subgroup1 and Subgroup2 was 0.027 (<em>P &lt;</em> 0.05). A total of 66 lines were in Subgroup1; 56, 3, 2, 2, 2 and 1 line(s) originated from China, Korea, Australia, Denmark, Germany and Sweden, respectively. A total of 339 lines were in Subgroup 2; 314, 2, 1, 2, 5, 9, 2, 2, 1 and 1 line originated from China, Japan, Korea, Australia, Canada, Germany, France, Sweden, Poland and Denmark, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s1_dsx013.xlsx?Expires=2147483647&Signature=OxQkLsitoMyFFYOhWCHthnOq49K6I8y6sLJepzaLkFzRDL4WQdzPBivTVrO~5oM-QI-32whWtq1Wsv6h76RlfWXLqiOnI5tba2e0xHKo7LHwAgxMe9HusWwAGLBN~rsNmga1zb5ruq6K-3wKjnh74exh9l5592ey1PP7ntpRAUsmMfc2B5jvaxvg0fKCaXt8SUqxIlmo04DIhJMrzmwDbDA1-c9LaW~OpiIaqYXgDJD4YjweRKRANGUjMSWhiNtEGE7q6BUGM07JsDOx-GzBbpnSugIW3I8Kp97paef-V0Q1VCexKSMntQzPWHJuslTiQSVt1PQi9LhZHLgwSulK2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The LD decay distance in Subgroup2 (875 kb) was longer than in Subgroup1 (812.5 kb) and shorter than in the whole panel (1065 kb). Gene diversity of Subgroup2 (0.4010) was higher than in Subgroup1 (0.3847). The PIC in Subgroup2 (0.3175) was higher than in Subgroup1 (0.3057) (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1d</a>). The Kinship coefficient of Subgroup2 (0.0697) was higher than Subgroup1 (0.0540). The <em>r</em><sup>2</sup> in Subgroup 2 (0.3452) was higher than in Subgroup1 (0.3122) (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1d</a>).</p><p>The relative kinships of the association panel were evaluated. The lines with kinship coefficients &lt; 0.05 accounted for 67.9, 69.5 and 70.6%, and the lines with kinship coefficients equal to 0 accounted for 57.4, 51.3 and 62.4% in the whole panel, Subgroup 1 and Subgroup 2, respectively (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig 1d</a>), indicating that the majority of the accessions have a weak relationship with one another (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1c</a>).</p>                            <h3 scrollto-destination=106357245 id="106357245" class="section-title" >3.3 Genome wide association mapping of RSA</h3>
<p>The Naïve model, Q model, K model and Q + K model were used for association mapping. The deviations of observed values from the expected values were shown in the QQ plots (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s2_dsx013.docx?Expires=2147483647&amp;Signature=2drZJUKGZofUqCrVJagbdRaMB62Fp9lmw8O-DH4wPA8Yh6s7dIhoKmfyGVmBiqtzkrjdD86GkcqzmWAiq-luY4hbIi87xlbMRNFg9JLg7TFHSqgD2y5KK0RhTnoZ5q0raV3P4j8OHP6GVv3o~h5KvQ7YPOxhYYwocW3TWZrEr4W2qmGxBT3jeb5Q0O-xtWjlvbMurVh9cx1jlo-vACjPeINxm5x8ED8YEAmAo-61K4nNKaL4BCzYu0aAqCJImnz4sx0JGPlnRP1jccraZ0qLWkHJNU14a9qM~J4UOe~~M0zIUVxC6zM5JoeFT-O0YPGSSdbhjGncqXiRxvIowaTUtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and were applied to select the most suitable model for each trait in the different experiments. Significant associations between SNPs and traits were identified by scanning the genome with a <em>P &lt;</em> 5.15 × 10<sup>−5</sup> threshold. In the hydroponic experiment I (HSI) there were 41 significant SNPs at LP and 9 at NP. Under LP; 7 SNPs for PRL were detected with the Q model; 4 SNPs for RDW were detected with the Q + K model; 6 SNPs for R/S ratio and 24 SNPs for SDW were detected with the Q model. Under NP; 1 SNP for PRL, 2 for RDW, 3 for R/S ratio and 3 for SDW were detected with the Q model. In the hydroponic experiment II (HSII), there were 31 significant SNPs at LP and 58 at NP. Under LP; 6 SNPs for PRL were detected with the Q + K model; 7 SNPs for RDW were detected with the naïve model; 6 SNPs for R/S ratio were detected with Q model and 12 SNPs for SDW detected with the Q model. At NP; 9 SNPs for PRL, 2 SNPs for R/S ratio and 37 SNPs for SDW were detected with the Q model; and 10 SNPs for RDW detected with the naïve model (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s4_dsx013.xlsx?Expires=2147483647&amp;Signature=IbUcupsxUPT5cq30LZeuuLat6tZpW~hlnNB-rQe5C7NY6C28TDppw8OWOaLGVKgGOY3NWfOWblitIWUiNgAHrj3wynGS7Lp9DVyFyvb9mEnOs8nBjEnxgBtB-LCdGYqXrwxLzB1oGiIyYle7u0u6D3gNollYGRXSssDY2gy7MyXwGcVnDE6obvgEjZI8ThDlFBHE1WYYmpRZC7sKQQEzWUHlkURrSCk9V1F2JiqffLrMV3zaWfw1yBO-FlO-taTKBHqskUwt7PpphroBgXqvRhs8m2XGhvoCxfwI9oT4yrTAn0Ftiw3ajn1y4-r7grkwl8C9qWD~vGpBpdvUpcOlSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). In the ‘pouch and wick’ system (PS) 165 significant SNPs were identified at LP; 28 SNPs for PRL, 33 for RDW, 22 for R/S ratio, 32 for TRL, 5 for LRD, 31 for LRL and 7 for MLRL were detected with the Q model and 3 SNPs for SDW and 4 SNPs for LRN were detected with the Q + K model (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s3_dsx013.docx?Expires=2147483647&amp;Signature=OFxDF3GY~0YyAh3E5wS~Jreb1lFs2aRxzyVMVBsCQkcW-wPIz1n7jJd~J8wi6yV5mSxtc3hvUYB8EfCdVf8MnAOuH58kVnzl8Xu~F5Oc2fWtqPwH6jQfAZ6DHnmkY6YW-sm8uP4MLb4H2EthxuX2VlGXkSR-IjsizV3h3edDfG86uy2OpZPpzvjVqF3bcxvcOX1RVjQ0omehg8-KnfaRroAAR38vRVDpar1gISyZvCcQM0rGGh1-nIHO~c8-tU6ix0ZnOuFolVKYMMTREbJ6H16NmeRqguA2rGE9ogCzCPTQCVqSL1M80ZstmwxQUrIsR00j7sNEsu2TcdzizUi~uQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> and Table S4).</p>                            <h3 scrollto-destination=106357247 id="106357247" class="section-title" >3.4 Co-locating SNPs and QTLs</h3>
<p>Nine significant SNPs detected with GWAS analyses co-localized with the intervals of the QTLs for the same traits in a previous linkage analyses of the <em>Bna</em>TNDH population,<a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> including one SNP on A3 for PRL, two SNPs on A5 for RDW (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s4_dsx013.docx?Expires=2147483647&Signature=mhLawzkdURKP6ru1iy3EPJg9sZ7SLIM~4QkBMkKLNKR7atsrbCXiDTkH3TeWDbbmBWLQEX73P2zYS0rbkf3q3wluW62F-lO~890AD1IkQS0w9ObIntb3A~EuOlpUbAzY~bh7uqSn-M~TEqKns9Ckhljy3LuaYk~ekAXFJeuY~V41O0k6uR6Dt1IcFUtsRW-0RD2kTnHXBDH7opJjLd5-6C5wIHMZfyqA0T12NNHIYUoyzd-Y-hRFcfRcozWUu~mESQpcwWrKBPaav9tKPquysvJk6IQxMJ9zEmAq9DPZJTR3OvQ6tVni-uzAHsNzEVbfB8PQbGTtupDU90NXE-YLlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>a–c), three SNPs on A7 and C4 for SDW, and one SNP on A3 for TRL detected at LP; two SNPs on A3 for SDW detected at NP (<a reveal-id="dsx013-T4" data-open="dsx013-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Therefore, the co-localized loci were detected for the same traits under the same P treatment in the two studies. The two nearly significant SNP markers on A5; Bn-scaff_16893_1-p69538 and Bn-A05-p18820057 at 16.76–17.25 Mb for RDW at LP, were detected in both systems employed in this study (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s4_dsx013.docx?Expires=2147483647&Signature=mhLawzkdURKP6ru1iy3EPJg9sZ7SLIM~4QkBMkKLNKR7atsrbCXiDTkH3TeWDbbmBWLQEX73P2zYS0rbkf3q3wluW62F-lO~890AD1IkQS0w9ObIntb3A~EuOlpUbAzY~bh7uqSn-M~TEqKns9Ckhljy3LuaYk~ekAXFJeuY~V41O0k6uR6Dt1IcFUtsRW-0RD2kTnHXBDH7opJjLd5-6C5wIHMZfyqA0T12NNHIYUoyzd-Y-hRFcfRcozWUu~mESQpcwWrKBPaav9tKPquysvJk6IQxMJ9zEmAq9DPZJTR3OvQ6tVni-uzAHsNzEVbfB8PQbGTtupDU90NXE-YLlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>b and c). Moreover, there was a significant SNP marker Bn-A03-p3001467 at 2.49–2.72 Mb on A3 detected for all root traits measured in the PS experiment at LP (<a reveal-id="dsx013-F2" data-open="dsx013-F2" class="link link-reveal link-table xref-fig">Fig. 2b–h</a>). Bn-A03-p3001467 could explain phenotypic variation (PVE) of PRL (9.2%), LRN (6.0%), LRL (12.4%), MLRL (6.3%), TRL (13.8%), RDW (14.5%) and SDW (5.9%) at LP (<a reveal-id="dsx013-T3" data-open="dsx013-T3" class="link link-reveal link-table xref-fig">Table 3</a>), and thus was the main locus responding to P deficiency. <div content-id="dsx013-T4" class="table-modal"><div class="table-wrap"><div id="dsx013-T4" data-id="dsx013-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>SNPs detected by GWAS in the OSR panel; chromosomes (Chr.); physical position (Pos. Mb); P value (significance); phenotypic variation explained (PVE %) and QTLs detected in the <em>Bna</em>TN DH mapping population<sup>9</sup>; LOD score (significance); marker interval; confidence interval (CI cM). co-located for root system architecture (RSA) traits, under low P (LP) and normal P (NP) treatments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Significant SNPs<hr /> </th><th colspan="4">Significant QTLs<hr /> </th></tr><tr><th>Trait/treatment </th><th>SNP </th><th>Chr. </th><th>Pos. (Mb) </th><th>P value </th><th>PVE (%) </th><th>Experiment </th><th>QTLs </th><th>LOD score </th><th>Marker interval </th><th>Cl (cM) </th></tr></thead><tbody><tr><td>PRL_LP </td><td>Bn-A03-p3001467 </td><td>A3 </td><td>2.56 </td><td>1.91E-07 </td><td>9.20 </td><td>2014PS </td><td>qPRL_LP_A3a </td><td>3.24 </td><td>Bn-A03-p2491346/Bn-A03-p4473205 </td><td>28.8-43.5 </td></tr><tr><td>RDW_LP </td><td>Bn-A05-p18820057 </td><td>A5 </td><td>17.08 </td><td>1.35E-05 </td><td>5.63 </td><td>2014PS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>RDW_LP </td><td>Bn-scaff_16893_1-p69538 </td><td>A5 </td><td>6.91 </td><td>1.28E-04 </td><td>4.79 </td><td>2014HS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p11068231 </td><td>A7 </td><td>12.31 </td><td>2.07E-05 </td><td>5.94 </td><td>2013HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A02-p126006 </td><td>31.7-39.2 </td></tr><tr><td>SDW_LP </td><td>Bn-scaff_16394_1-p124017 </td><td>C4 </td><td>31.52 </td><td>3.31E-05 </td><td>5.74 </td><td>2013HS </td><td>qSDW_LP_C4a </td><td>3.10 </td><td>Bn-scaff_17180_1-p59560/Bn-scaff_15798_1-p699979 </td><td>60.6-70.3 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p10557165 </td><td>A7 </td><td>11.74 </td><td>7.01E-05 </td><td>5.28 </td><td>2014HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A07-p10891272 </td><td>31.7-39.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A03-p7688578 </td><td>A3 </td><td>6.99 </td><td>1.02E-05 </td><td>6.48 </td><td>2014HS </td><td>qSDW_HP_A3b </td><td>6.00 </td><td>Bn-A03-p7420835/Bn-A03-p7814324 </td><td>48.9-51.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A04-p18962378 </td><td>A4 </td><td>19.01 </td><td>5.25E-05 </td><td>5.52 </td><td>2014HS </td><td>qSDW_HP_A4a </td><td>4.62 </td><td>Bn-scaff_18903_1-p856363/Bn-scaff_27676_1-p191772 </td><td>78.0-81.6 </td></tr><tr><td>TRL_LP </td><td>Bn-A03-p4408895 </td><td>A3 </td><td>3.94 </td><td>1.38E-07 </td><td>7.90 </td><td>2014PS </td><td>qTRL_LP_A3a </td><td>3.93 </td><td>Bn-A03-p4473205/Bn-A03-p5045095 </td><td>45.4-46.9 </td></tr></tbody></table></div><div content-id="dsx013-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Significant SNPs<hr /> </th><th colspan="4">Significant QTLs<hr /> </th></tr><tr><th>Trait/treatment </th><th>SNP </th><th>Chr. </th><th>Pos. (Mb) </th><th>P value </th><th>PVE (%) </th><th>Experiment </th><th>QTLs </th><th>LOD score </th><th>Marker interval </th><th>Cl (cM) </th></tr></thead><tbody><tr><td>PRL_LP </td><td>Bn-A03-p3001467 </td><td>A3 </td><td>2.56 </td><td>1.91E-07 </td><td>9.20 </td><td>2014PS </td><td>qPRL_LP_A3a </td><td>3.24 </td><td>Bn-A03-p2491346/Bn-A03-p4473205 </td><td>28.8-43.5 </td></tr><tr><td>RDW_LP </td><td>Bn-A05-p18820057 </td><td>A5 </td><td>17.08 </td><td>1.35E-05 </td><td>5.63 </td><td>2014PS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>RDW_LP </td><td>Bn-scaff_16893_1-p69538 </td><td>A5 </td><td>6.91 </td><td>1.28E-04 </td><td>4.79 </td><td>2014HS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p11068231 </td><td>A7 </td><td>12.31 </td><td>2.07E-05 </td><td>5.94 </td><td>2013HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A02-p126006 </td><td>31.7-39.2 </td></tr><tr><td>SDW_LP </td><td>Bn-scaff_16394_1-p124017 </td><td>C4 </td><td>31.52 </td><td>3.31E-05 </td><td>5.74 </td><td>2013HS </td><td>qSDW_LP_C4a </td><td>3.10 </td><td>Bn-scaff_17180_1-p59560/Bn-scaff_15798_1-p699979 </td><td>60.6-70.3 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p10557165 </td><td>A7 </td><td>11.74 </td><td>7.01E-05 </td><td>5.28 </td><td>2014HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A07-p10891272 </td><td>31.7-39.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A03-p7688578 </td><td>A3 </td><td>6.99 </td><td>1.02E-05 </td><td>6.48 </td><td>2014HS </td><td>qSDW_HP_A3b </td><td>6.00 </td><td>Bn-A03-p7420835/Bn-A03-p7814324 </td><td>48.9-51.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A04-p18962378 </td><td>A4 </td><td>19.01 </td><td>5.25E-05 </td><td>5.52 </td><td>2014HS </td><td>qSDW_HP_A4a </td><td>4.62 </td><td>Bn-scaff_18903_1-p856363/Bn-scaff_27676_1-p191772 </td><td>78.0-81.6 </td></tr><tr><td>TRL_LP </td><td>Bn-A03-p4408895 </td><td>A3 </td><td>3.94 </td><td>1.38E-07 </td><td>7.90 </td><td>2014PS </td><td>qTRL_LP_A3a </td><td>3.93 </td><td>Bn-A03-p4473205/Bn-A03-p5045095 </td><td>45.4-46.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>SDW, shoot dry weight; RDW, root dry weight; PRL, primary root length; TRL, total root length.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx013-T4" data-id="dsx013-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>SNPs detected by GWAS in the OSR panel; chromosomes (Chr.); physical position (Pos. Mb); P value (significance); phenotypic variation explained (PVE %) and QTLs detected in the <em>Bna</em>TN DH mapping population<sup>9</sup>; LOD score (significance); marker interval; confidence interval (CI cM). co-located for root system architecture (RSA) traits, under low P (LP) and normal P (NP) treatments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Significant SNPs<hr /> </th><th colspan="4">Significant QTLs<hr /> </th></tr><tr><th>Trait/treatment </th><th>SNP </th><th>Chr. </th><th>Pos. (Mb) </th><th>P value </th><th>PVE (%) </th><th>Experiment </th><th>QTLs </th><th>LOD score </th><th>Marker interval </th><th>Cl (cM) </th></tr></thead><tbody><tr><td>PRL_LP </td><td>Bn-A03-p3001467 </td><td>A3 </td><td>2.56 </td><td>1.91E-07 </td><td>9.20 </td><td>2014PS </td><td>qPRL_LP_A3a </td><td>3.24 </td><td>Bn-A03-p2491346/Bn-A03-p4473205 </td><td>28.8-43.5 </td></tr><tr><td>RDW_LP </td><td>Bn-A05-p18820057 </td><td>A5 </td><td>17.08 </td><td>1.35E-05 </td><td>5.63 </td><td>2014PS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>RDW_LP </td><td>Bn-scaff_16893_1-p69538 </td><td>A5 </td><td>6.91 </td><td>1.28E-04 </td><td>4.79 </td><td>2014HS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p11068231 </td><td>A7 </td><td>12.31 </td><td>2.07E-05 </td><td>5.94 </td><td>2013HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A02-p126006 </td><td>31.7-39.2 </td></tr><tr><td>SDW_LP </td><td>Bn-scaff_16394_1-p124017 </td><td>C4 </td><td>31.52 </td><td>3.31E-05 </td><td>5.74 </td><td>2013HS </td><td>qSDW_LP_C4a </td><td>3.10 </td><td>Bn-scaff_17180_1-p59560/Bn-scaff_15798_1-p699979 </td><td>60.6-70.3 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p10557165 </td><td>A7 </td><td>11.74 </td><td>7.01E-05 </td><td>5.28 </td><td>2014HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A07-p10891272 </td><td>31.7-39.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A03-p7688578 </td><td>A3 </td><td>6.99 </td><td>1.02E-05 </td><td>6.48 </td><td>2014HS </td><td>qSDW_HP_A3b </td><td>6.00 </td><td>Bn-A03-p7420835/Bn-A03-p7814324 </td><td>48.9-51.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A04-p18962378 </td><td>A4 </td><td>19.01 </td><td>5.25E-05 </td><td>5.52 </td><td>2014HS </td><td>qSDW_HP_A4a </td><td>4.62 </td><td>Bn-scaff_18903_1-p856363/Bn-scaff_27676_1-p191772 </td><td>78.0-81.6 </td></tr><tr><td>TRL_LP </td><td>Bn-A03-p4408895 </td><td>A3 </td><td>3.94 </td><td>1.38E-07 </td><td>7.90 </td><td>2014PS </td><td>qTRL_LP_A3a </td><td>3.93 </td><td>Bn-A03-p4473205/Bn-A03-p5045095 </td><td>45.4-46.9 </td></tr></tbody></table></div><div content-id="dsx013-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Significant SNPs<hr /> </th><th colspan="4">Significant QTLs<hr /> </th></tr><tr><th>Trait/treatment </th><th>SNP </th><th>Chr. </th><th>Pos. (Mb) </th><th>P value </th><th>PVE (%) </th><th>Experiment </th><th>QTLs </th><th>LOD score </th><th>Marker interval </th><th>Cl (cM) </th></tr></thead><tbody><tr><td>PRL_LP </td><td>Bn-A03-p3001467 </td><td>A3 </td><td>2.56 </td><td>1.91E-07 </td><td>9.20 </td><td>2014PS </td><td>qPRL_LP_A3a </td><td>3.24 </td><td>Bn-A03-p2491346/Bn-A03-p4473205 </td><td>28.8-43.5 </td></tr><tr><td>RDW_LP </td><td>Bn-A05-p18820057 </td><td>A5 </td><td>17.08 </td><td>1.35E-05 </td><td>5.63 </td><td>2014PS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>RDW_LP </td><td>Bn-scaff_16893_1-p69538 </td><td>A5 </td><td>6.91 </td><td>1.28E-04 </td><td>4.79 </td><td>2014HS </td><td>qRDW_LP_A5a </td><td>3.77 </td><td>Bn-A09-p24520612/Bn-A05-p18958083 </td><td>54.5-63.0 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p11068231 </td><td>A7 </td><td>12.31 </td><td>2.07E-05 </td><td>5.94 </td><td>2013HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A02-p126006 </td><td>31.7-39.2 </td></tr><tr><td>SDW_LP </td><td>Bn-scaff_16394_1-p124017 </td><td>C4 </td><td>31.52 </td><td>3.31E-05 </td><td>5.74 </td><td>2013HS </td><td>qSDW_LP_C4a </td><td>3.10 </td><td>Bn-scaff_17180_1-p59560/Bn-scaff_15798_1-p699979 </td><td>60.6-70.3 </td></tr><tr><td>SDW_LP </td><td>Bn-A07-p10557165 </td><td>A7 </td><td>11.74 </td><td>7.01E-05 </td><td>5.28 </td><td>2014HS </td><td>qSDW_LP_A7a </td><td>5.87 </td><td>Bn-A07-p9849702/Bn-A07-p10891272 </td><td>31.7-39.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A03-p7688578 </td><td>A3 </td><td>6.99 </td><td>1.02E-05 </td><td>6.48 </td><td>2014HS </td><td>qSDW_HP_A3b </td><td>6.00 </td><td>Bn-A03-p7420835/Bn-A03-p7814324 </td><td>48.9-51.2 </td></tr><tr><td>SDW_NP </td><td>Bn-A04-p18962378 </td><td>A4 </td><td>19.01 </td><td>5.25E-05 </td><td>5.52 </td><td>2014HS </td><td>qSDW_HP_A4a </td><td>4.62 </td><td>Bn-scaff_18903_1-p856363/Bn-scaff_27676_1-p191772 </td><td>78.0-81.6 </td></tr><tr><td>TRL_LP </td><td>Bn-A03-p4408895 </td><td>A3 </td><td>3.94 </td><td>1.38E-07 </td><td>7.90 </td><td>2014PS </td><td>qTRL_LP_A3a </td><td>3.93 </td><td>Bn-A03-p4473205/Bn-A03-p5045095 </td><td>45.4-46.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>SDW, shoot dry weight; RDW, root dry weight; PRL, primary root length; TRL, total root length.</p></div></div></div></div></p>                        <a id="106357249" scrollto-destination="106357249"></a>
<div id="dsx013-F2" data-id="dsx013-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f2.png?Expires=2147483647&Signature=EoYfAkeNF6Bdjk~-FCpjUDOcvEPHAdftYAxnnjG8HzhQ7gw5AyFzTPObIoYk9gsdPbR1~xv7B51OPfvyOs5GCV5leu-FtLt8ETmoRdv7Hn8NgReajg06l1bK-hkGWP02OPe0dhYkmj7kdUeBZZXNw7ijV-zQVHflXkCeHKyk6YM3NhJk2FpfGyaJu-WYVeVESxru~QOmFj3jWYtXvNvxosVECFPVfXN~rLrJ0AwcEmY2r1dfcx2GUunsvK0gXpxdponQl4KaJjzRqa-CLjpyL~lIg6rNTfvDVcQDtaUSnlijzzalKBPvarNMVMJ~hzlCxUgzA53qyvubZZbBvpYlbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Colocalized locus on chromosome A3 for root traits of B. napus at LP. (a) QTL detected for PRL in the BnaTNDH population linkage analysis (Zhang et al. 2016). (b) peak SNP for PRL. (c) peak SNP for LRN. (d) peak SNP for LRL. (e) peak SNP for MLRL. (f) peak SNP for TRL. (g) peak SNP for RDW. (h) peak SNP for SDW detected with GWAS in the ‘pouch and wick’ (PS) experiment. i) expression profile detected with transcriptome sequencing of candidate genes located in the QTL linkage disequilibrium intervals; ①BnaA03g05800D, ②BnaA03g05820D, ③BnaA03g05830D and ④BnaA03g05850D. primary root length (PRL); lateral root number (LRN); lateral root length (LRL); mean lateral root length (MLRL); total root length (TRL)." path-from-xml="dsx013f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357249/dsx013f2.png" path-from-xml="dsx013f2.png" target="_blank">View large</a><a section="106357249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f2.png?Expires=2147483647&Signature=Ow3JsbRfSvtRLO1xW3lO-GgP8L3IhLZYW1Ylks2DEgEPWHFWSX~lbKFiBU-UiU-7yGRdmEHqk3J~tL8pzpEjMoS2I9zA03ItcFLTUGFy9yvUZ5-iElReEbD7iROVqljYhDgD~gDdjMdERU-Dp5ZHOzt~EIG6JBA~E0D-tmfrsh-MJfhTdUbXp55~cVMCn8Y1BbaK4sJiA0k-rmKAnWwPMwKUbhy0TZBCJ2hRKR0-yyk1H11Bv5Z3lXs9PjxvlSXkTVh0T1-An~v1XWwtiECj1lfTRigPbT~QT8OqDJMyVeQuH-fvyiISASrgIRCDUYNaWL3xo4sTbTDk~PNucfFagw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357249&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Colocalized locus on chromosome A3 for root traits of <em>B. napus</em> at LP. (a) QTL detected for PRL in the <em>Bna</em>TNDH population linkage analysis (Zhang <em>et al.</em> 2016). (b) peak SNP for PRL. (c) peak SNP for LRN. (d) peak SNP for LRL. (e) peak SNP for MLRL. (f) peak SNP for TRL. (g) peak SNP for RDW. (h) peak SNP for SDW detected with GWAS in the ‘pouch and wick’ (PS) experiment. i) expression profile detected with transcriptome sequencing of candidate genes located in the QTL linkage disequilibrium intervals; ①<em>BnaA03g05800D</em>, ②<em>BnaA03g05820D</em>, ③<em>BnaA03g05830D</em> and ④<em>BnaA03g05850D</em>. primary root length (PRL); lateral root number (LRN); lateral root length (LRL); mean lateral root length (MLRL); total root length (TRL).</p></div></div><div content-id="dsx013-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f2.png?Expires=2147483647&Signature=EoYfAkeNF6Bdjk~-FCpjUDOcvEPHAdftYAxnnjG8HzhQ7gw5AyFzTPObIoYk9gsdPbR1~xv7B51OPfvyOs5GCV5leu-FtLt8ETmoRdv7Hn8NgReajg06l1bK-hkGWP02OPe0dhYkmj7kdUeBZZXNw7ijV-zQVHflXkCeHKyk6YM3NhJk2FpfGyaJu-WYVeVESxru~QOmFj3jWYtXvNvxosVECFPVfXN~rLrJ0AwcEmY2r1dfcx2GUunsvK0gXpxdponQl4KaJjzRqa-CLjpyL~lIg6rNTfvDVcQDtaUSnlijzzalKBPvarNMVMJ~hzlCxUgzA53qyvubZZbBvpYlbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Colocalized locus on chromosome A3 for root traits of B. napus at LP. (a) QTL detected for PRL in the BnaTNDH population linkage analysis (Zhang et al. 2016). (b) peak SNP for PRL. (c) peak SNP for LRN. (d) peak SNP for LRL. (e) peak SNP for MLRL. (f) peak SNP for TRL. (g) peak SNP for RDW. (h) peak SNP for SDW detected with GWAS in the ‘pouch and wick’ (PS) experiment. i) expression profile detected with transcriptome sequencing of candidate genes located in the QTL linkage disequilibrium intervals; ①BnaA03g05800D, ②BnaA03g05820D, ③BnaA03g05830D and ④BnaA03g05850D. primary root length (PRL); lateral root number (LRN); lateral root length (LRL); mean lateral root length (MLRL); total root length (TRL)." path-from-xml="dsx013f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357249/dsx013f2.png" path-from-xml="dsx013f2.png" target="_blank">View large</a><a section="106357249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f2.png?Expires=2147483647&Signature=Ow3JsbRfSvtRLO1xW3lO-GgP8L3IhLZYW1Ylks2DEgEPWHFWSX~lbKFiBU-UiU-7yGRdmEHqk3J~tL8pzpEjMoS2I9zA03ItcFLTUGFy9yvUZ5-iElReEbD7iROVqljYhDgD~gDdjMdERU-Dp5ZHOzt~EIG6JBA~E0D-tmfrsh-MJfhTdUbXp55~cVMCn8Y1BbaK4sJiA0k-rmKAnWwPMwKUbhy0TZBCJ2hRKR0-yyk1H11Bv5Z3lXs9PjxvlSXkTVh0T1-An~v1XWwtiECj1lfTRigPbT~QT8OqDJMyVeQuH-fvyiISASrgIRCDUYNaWL3xo4sTbTDk~PNucfFagw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357249&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Colocalized locus on chromosome A3 for root traits of <em>B. napus</em> at LP. (a) QTL detected for PRL in the <em>Bna</em>TNDH population linkage analysis (Zhang <em>et al.</em> 2016). (b) peak SNP for PRL. (c) peak SNP for LRN. (d) peak SNP for LRL. (e) peak SNP for MLRL. (f) peak SNP for TRL. (g) peak SNP for RDW. (h) peak SNP for SDW detected with GWAS in the ‘pouch and wick’ (PS) experiment. i) expression profile detected with transcriptome sequencing of candidate genes located in the QTL linkage disequilibrium intervals; ①<em>BnaA03g05800D</em>, ②<em>BnaA03g05820D</em>, ③<em>BnaA03g05830D</em> and ④<em>BnaA03g05850D</em>. primary root length (PRL); lateral root number (LRN); lateral root length (LRL); mean lateral root length (MLRL); total root length (TRL).</p></div></div>                            <h3 scrollto-destination=106357250 id="106357250" class="section-title" >3.5 Gene expression profile of candidate genes</h3>
<p>A total of 88 candidate genes were within the QTL confidence interval on A3 (<a reveal-id="dsx013-F2" data-open="dsx013-F2" class="link link-reveal link-table xref-fig">Fig. 2i</a> and <span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s5_dsx013.xlsx?Expires=2147483647&amp;Signature=PdWR4~nIX8~729uYfN3uNVfXxYYhLa1ze9N8OgXEzN0WeA~p~a9n5mrzUiwEUcK~2xxu4nxIfV-3aXzosdTaTZSdAyMl3rGjDMajzxPx19WsvSsyuPI8F5jgjUaIo4g3L~UppGF3mybRmfIb-ibBm~mkpGgNT-u86Va1vnuLuUpHuSaqMyjyUrL6pe5sQ7rvRgl6CRPdUUftZ5ckglydFQu-pUCv~aMzprLs2HrF9IOKh8ID-sVQI2mVShmIWfAy0PD-APw5B~61geHHzSS8db4Kjn5YltlWtNazm0ezu5S-dLQ8nJkogq29P3~5PxRpGkpu1fDxik~bd~CDhtR0bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>); <em>BnaA03g05800D</em>, <em>BnaA03g05820D</em>, <em>BnaA03g05830D</em> and <em>BnaA03g05850D</em> were located in the LD decay region of the peak SNP Bn-A03-p3001467 at 2.49–2.72 Mb, and formed the haplotype blocks on A3 (<em>BnA3Hap</em>). In the cultivar ‘Eyou changjia’, the relative expression of <em>BnaA03g05800D</em> and <em>BnaA03g05830D</em> increased, and <em>BnaA03g05820D</em> and <em>BnaA03g05850D</em> decreased, under P-deficiency in the shoot. In the root, the relative expression of <em>BnaA03g05820D</em> increased, and the <em>BnaA03g05800D</em> and <em>BnaA03g05830D</em> decreased, under P-deficiency (<a reveal-id="dsx013-F2" data-open="dsx013-F2" class="link link-reveal link-table xref-fig">Fig. 2i</a>).</p><p>Two P-efficient varieties- 121 and 246 and two P-inefficient varieties- 292 and 453 were selected to measure gene expression of <em>BnaA03g05800D</em>, <em>BnaA03g05820D</em>, <em>BnaA03g05830D and BnaA03g05850D</em> by RT-qPCR. In comparison to P-inefficient varieties, the P-efficient varieties had a higher relative expression of <em>BnPAP17</em> (<em>BnaA05g22460D</em>) in both root and shoot at LP. The relative expression of <em>BnRACK1C</em> (<em>BnaA05g22070D</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s5_dsx013.docx?Expires=2147483647&Signature=2R0mKRdmdoV6oZiGGEBHkfQQB1z9ed3AZvmR~YiG60BMDmce1kx8Vb6tmcOtxHSY5pfRN224s45KIyjJJt4DH6JMOTP6XEORxns0~6qY~lsZNa3gzrk173wU~D-kWOC8zyMcmr64cMSjAMjHeHz7UVyXC66B7YfrbMKo5x11Bzz5Ih648O4ueBjf1b7UFI7N5fLajlbGsKK9Lhyb66ib00RIN0fsE-u~1Mc8KZl89-sJehOKEggeSjGIAwywye3y0G71jfTnT-PceqJ4RYA8fh4-7JZupWAd6uHI4eEyWqKM0a9QhLH6M~SVR1CxhLO4KvIxJu27LRhz9eLgMsXxDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>) was higher in the root at both P levels but lower in shoot at NP. <em>BnaA03g05800D</em> expression was higher in the root and shoot at both P levels, the relative expression of <em>BnaA03g05820D</em> was higher in the shoot at NP and in the root at LP, the relative expression of <em>BnaA03g05830D</em> was higher in the root at LP, and the relative expression of <em>BnaA03g05850D</em> was lower in shoot at LP (<a reveal-id="dsx013-F3" data-open="dsx013-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). This provides further evidence that the four putative genes were closely associated with P efficiency. </p>                        <a id="106357253" scrollto-destination="106357253"></a>
<div id="dsx013-F3" data-id="dsx013-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f3.png?Expires=2147483647&Signature=audTe0NA2PQCmYHVZJB4nlugYmlP5cEEgdtXMb8ygtCYTIf6N~fpte9FNJgl~XKETn0GKxHPxFAO1tEZ9zPj0eQIWlcRwDbkZP3gFPmI2GnUJhWDmWsOF7KLjnUQQsEBtAXTjEmazXTpyk4ay-uwI66IVmAYQLoi2xRSrOxuoLYdUICwX5rB1qScBgLF15yVS-RTg9XIxj0fhME1YmdDBAZft4ptYV8QSPlULQDGIV1sNHxtBR82zRK08~7OYpED21WdGoE2bHOFpqHNDErrUhciwPnTu4sZJkJ58Kcu7109o4aZghyOetwK-x8DJPJt6A8E8rSkcqUn4GhXAu-f9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative expression of differentially expressed genes (DEGs) in the BnA3Hap haplotype region located on chromosomes A3. (a) BnaA03g05800D. (b) BnaA03g05820D. (c) BnaA03g05830D. (d) BnaA03g05850D *P &lt; 0.05; **P &lt; 0.01. Root at NP (R + P); Root at LP (R-P); Shoot at NP (S + P); Shoot at LP (S-P). P –efficient- 121 and 246 and –inefficient varieties- 292 and 453." path-from-xml="dsx013f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357253/dsx013f3.png" path-from-xml="dsx013f3.png" target="_blank">View large</a><a section="106357253" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f3.png?Expires=2147483647&Signature=vd66Z9EBWdW80u8UnSiWkflof0j~svPvUULz8A089DaQ9TttGxn9aJcrnQCFcQux52Qnp6104bumBeFRQlDKxmRm08JfLu-~hz7tEOrm4xTevszBmwPori~v3cZ3X5vqjMp77~ngLQhcJKjZcClKXpdiDY3npzHTPLDR9m3n8BhmTys37gpCQG7QUnmsbCQw8DRnAkfyFMwSBte675pz3GgZzKQ5maGUDpfq0QhYpz0yZOhHuFcq3iXEzPi7cwRXbq66cJl0C3kLeSzuMW-BjG9K8x98gE1ADRQ-tBF6Hy0~1j8IMtyuX-7XsyDeIZswvQ5nPnC8yNPOjGbFUz~lAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357253&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative expression of differentially expressed genes (DEGs) in the <em>BnA3Hap</em> haplotype region located on chromosomes A3. (a) <em>BnaA03g05800D</em>. (b) <em>BnaA03g05820D</em>. (c) <em>BnaA03g05830D.</em> (d) <em>BnaA03g05850D</em> *<em>P &lt;</em> 0.05; **<em>P &lt;</em> 0.01. Root at NP (R + P); Root at LP (R-P); Shoot at NP (S + P); Shoot at LP (S-P). <em>P</em> –efficient- 121 and 246 and –inefficient varieties- 292 and 453.</p></div></div><div content-id="dsx013-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f3.png?Expires=2147483647&Signature=audTe0NA2PQCmYHVZJB4nlugYmlP5cEEgdtXMb8ygtCYTIf6N~fpte9FNJgl~XKETn0GKxHPxFAO1tEZ9zPj0eQIWlcRwDbkZP3gFPmI2GnUJhWDmWsOF7KLjnUQQsEBtAXTjEmazXTpyk4ay-uwI66IVmAYQLoi2xRSrOxuoLYdUICwX5rB1qScBgLF15yVS-RTg9XIxj0fhME1YmdDBAZft4ptYV8QSPlULQDGIV1sNHxtBR82zRK08~7OYpED21WdGoE2bHOFpqHNDErrUhciwPnTu4sZJkJ58Kcu7109o4aZghyOetwK-x8DJPJt6A8E8rSkcqUn4GhXAu-f9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative expression of differentially expressed genes (DEGs) in the BnA3Hap haplotype region located on chromosomes A3. (a) BnaA03g05800D. (b) BnaA03g05820D. (c) BnaA03g05830D. (d) BnaA03g05850D *P &lt; 0.05; **P &lt; 0.01. Root at NP (R + P); Root at LP (R-P); Shoot at NP (S + P); Shoot at LP (S-P). P –efficient- 121 and 246 and –inefficient varieties- 292 and 453." path-from-xml="dsx013f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357253/dsx013f3.png" path-from-xml="dsx013f3.png" target="_blank">View large</a><a section="106357253" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f3.png?Expires=2147483647&Signature=vd66Z9EBWdW80u8UnSiWkflof0j~svPvUULz8A089DaQ9TttGxn9aJcrnQCFcQux52Qnp6104bumBeFRQlDKxmRm08JfLu-~hz7tEOrm4xTevszBmwPori~v3cZ3X5vqjMp77~ngLQhcJKjZcClKXpdiDY3npzHTPLDR9m3n8BhmTys37gpCQG7QUnmsbCQw8DRnAkfyFMwSBte675pz3GgZzKQ5maGUDpfq0QhYpz0yZOhHuFcq3iXEzPi7cwRXbq66cJl0C3kLeSzuMW-BjG9K8x98gE1ADRQ-tBF6Hy0~1j8IMtyuX-7XsyDeIZswvQ5nPnC8yNPOjGbFUz~lAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357253&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative expression of differentially expressed genes (DEGs) in the <em>BnA3Hap</em> haplotype region located on chromosomes A3. (a) <em>BnaA03g05800D</em>. (b) <em>BnaA03g05820D</em>. (c) <em>BnaA03g05830D.</em> (d) <em>BnaA03g05850D</em> *<em>P &lt;</em> 0.05; **<em>P &lt;</em> 0.01. Root at NP (R + P); Root at LP (R-P); Shoot at NP (S + P); Shoot at LP (S-P). <em>P</em> –efficient- 121 and 246 and –inefficient varieties- 292 and 453.</p></div></div>                            <h3 scrollto-destination=106357254 id="106357254" class="section-title" >3.6 <em>BnA3Hap</em> haplotypes associated RSA traits and geographical distribution</h3>
<p>There were 21 SNPs located in the QTL confidence intervals on A3 (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4h</a>). Eight successive SNPs located near SNP Bn-A03-p3052882 at 25.6 kb and the SNP Bn-A03-p3001467 at 75.2 kb formed nine haplotypes (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4g</a>), designated ‘<em>BnA3Hap</em> haplotypes’. The varieties with haplotype 6 (GAAAAAGG) had higher average values in all the investigated RSA traits than those with the other haplotypes (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4a–f</a>); two of the three varieties with haplotype 6 originated from European countries. Whereas, the varieties with haplotype 2 (AGCCGAGG) had lower average values in all of the investigated RSA traits than those with the other haplotypes (<a reveal-id="dsx013-F5" data-open="dsx013-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Fifty-one of the 53 varieties with haplotype 2 originated from China. Haplotype 7 (GAAAACTA) was the most common haplotype (<em>n =</em> 183) and the RSA parameters of Haplotype 7 varieties were intermediate to haplotypes 6 and 2 varieties, respectively (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4a–f</a>). The geographic distribution of the varieties was East Asia (285), Oceania (3), Europe (18) and North America (Canada) (4). Moreover, in China, the varieties distributed in Sichuan (10), Gansu (4), Shanxi (18), Hunan (17), Jiangsu (41), Chongqing (55), Hubei (131) and Qinghai (9) provinces (<a reveal-id="dsx013-F5" data-open="dsx013-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Tajima’s D test of each SNP loci of <em>BnA3Hap</em> in <em>B. napus</em> indicated varieties from Gansu, Shanxi and Jiangsu provinces of China and Canada showed haplotype- <em>BnA3Hap</em> selection (<a reveal-id="dsx013-F6" data-open="dsx013-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). </p>                        <a id="106357256" scrollto-destination="106357256"></a>
<div id="dsx013-F4" data-id="dsx013-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f4.png?Expires=2147483647&Signature=SkV1yCaaMZA~GqaOh11F6HqJSlr-x8CyNrhnPDJllaQVXr~dYgPlg4w5vBh2WvM4Mi0-DyzLFVgqjfg9R-mERK0RC-FAidykohHxIZgrb0-yM5STJdAv8kj1b0IsddiWcNycQSsuZh45HEzpJWz4-nPXHQlnRuNoeeCIKAYVuKrNXbRA55OsISfKRdKW7pQq46dQT3l0pFuxiEYTOsLoqCNSchQKYHQZOHFT~hLz--Q6ypKQebrUpVjM~CwMi1sP2BjhGGlbZNYFJKNwgHNEIWgPpuikkvn0Ea-2HN3a1JpM1QeVmg~IBJryUXH3ROaseiyQ-zPbKXCOsJncjz4hjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The haplotypes associated with RSA (root system architecture) traits in the ‘pouch and wick’ system (PS) at low P (LP). (a). Lateral root length (LRL). (b) Lateral root number (LRN). (c) Mean lateral root length (MLRL). (d) Primary root length (PRL). (e) Root dry weight (RDW). (f) Total root length (TRL). (g) BnA3Hap haplotypes. (h) The BnA3Hap haplotypes on A3 (darker colour trilateral)." path-from-xml="dsx013f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357256/dsx013f4.png" path-from-xml="dsx013f4.png" target="_blank">View large</a><a section="106357256" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f4.png?Expires=2147483647&Signature=GdRjPNlwKEqlKAtrSrYnRGHDdpnL1XUf35niCLCG~Sn6b4MjLEkHgiB0vlSKOF0VmaxwvosnQmoHvgYjVNGRTOU9fI~HCgPJ7W3yNvFpjlyejavKY56PMfYf95eVbPNMA0j5-Ot2wZGYa88JZQYLyYYdJ37R4ZNqoYVUj~NCwJX0ET~tYMGjcjMGrp02BNWI0aC3-Hx7SX03w6cqpqxmFn95IVGnlhyYd~J1gJtJ7KcA1HyYubEJrs26j0M~AjAeuoVNK3xgET5Xw1HQisKxxupXybQ9xoWuJE0dP-kO0J1IWRCxkHQ0GkUP88~ex9SLSrvXvwJ6jOgCOuJxjwwv1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357256&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The haplotypes associated with RSA (root system architecture) traits in the ‘pouch and wick’ system (PS) at low P (LP). (a). Lateral root length (LRL). (b) Lateral root number (LRN). (c) Mean lateral root length (MLRL). (d) Primary root length (PRL). (e) Root dry weight (RDW). (f) Total root length (TRL). (g) <em>BnA3Hap</em> haplotypes. (h) The <em>BnA3Hap</em> haplotypes on A3 (darker colour trilateral).</p></div></div><div content-id="dsx013-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f4.png?Expires=2147483647&Signature=SkV1yCaaMZA~GqaOh11F6HqJSlr-x8CyNrhnPDJllaQVXr~dYgPlg4w5vBh2WvM4Mi0-DyzLFVgqjfg9R-mERK0RC-FAidykohHxIZgrb0-yM5STJdAv8kj1b0IsddiWcNycQSsuZh45HEzpJWz4-nPXHQlnRuNoeeCIKAYVuKrNXbRA55OsISfKRdKW7pQq46dQT3l0pFuxiEYTOsLoqCNSchQKYHQZOHFT~hLz--Q6ypKQebrUpVjM~CwMi1sP2BjhGGlbZNYFJKNwgHNEIWgPpuikkvn0Ea-2HN3a1JpM1QeVmg~IBJryUXH3ROaseiyQ-zPbKXCOsJncjz4hjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The haplotypes associated with RSA (root system architecture) traits in the ‘pouch and wick’ system (PS) at low P (LP). (a). Lateral root length (LRL). (b) Lateral root number (LRN). (c) Mean lateral root length (MLRL). (d) Primary root length (PRL). (e) Root dry weight (RDW). (f) Total root length (TRL). (g) BnA3Hap haplotypes. (h) The BnA3Hap haplotypes on A3 (darker colour trilateral)." path-from-xml="dsx013f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357256/dsx013f4.png" path-from-xml="dsx013f4.png" target="_blank">View large</a><a section="106357256" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f4.png?Expires=2147483647&Signature=GdRjPNlwKEqlKAtrSrYnRGHDdpnL1XUf35niCLCG~Sn6b4MjLEkHgiB0vlSKOF0VmaxwvosnQmoHvgYjVNGRTOU9fI~HCgPJ7W3yNvFpjlyejavKY56PMfYf95eVbPNMA0j5-Ot2wZGYa88JZQYLyYYdJ37R4ZNqoYVUj~NCwJX0ET~tYMGjcjMGrp02BNWI0aC3-Hx7SX03w6cqpqxmFn95IVGnlhyYd~J1gJtJ7KcA1HyYubEJrs26j0M~AjAeuoVNK3xgET5Xw1HQisKxxupXybQ9xoWuJE0dP-kO0J1IWRCxkHQ0GkUP88~ex9SLSrvXvwJ6jOgCOuJxjwwv1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357256&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The haplotypes associated with RSA (root system architecture) traits in the ‘pouch and wick’ system (PS) at low P (LP). (a). Lateral root length (LRL). (b) Lateral root number (LRN). (c) Mean lateral root length (MLRL). (d) Primary root length (PRL). (e) Root dry weight (RDW). (f) Total root length (TRL). (g) <em>BnA3Hap</em> haplotypes. (h) The <em>BnA3Hap</em> haplotypes on A3 (darker colour trilateral).</p></div></div>                        <a id="106357257" scrollto-destination="106357257"></a>
<div id="dsx013-F5" data-id="dsx013-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f5.png?Expires=2147483647&Signature=STAGEDglE9u6x72ECfFF3-jqwZbLkYMTqfOTKZo6x6dlrMoJMK2dVk1iicOsrYuxSmJ3uJSc5varTAoixiDTDlXWdWTGt4UuKe22gzA0Siz5BuEeuHnYWsBbaW2sjCXYAKl0mSnIuE1za6RcQjYpQyGx0TgZfZSxzwGeQMutVQ59BZAmeHF3NyttzyOjzvCJP8xoGZqckGJXT-lvUs5HhLWaMuKRq2Xvgg0TNqums6EShyPFIjDhEoFX7EEApnGA6Y0Vlmy34f3nvq-cZwMzzGXzZc-AseK37I2A-OjoD4mC25iidk0E-b9~yESY5K8ySY5fdBrxk40z40mTpkXseg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The geographical distribution of the BnA3Hap haplotypes in the B. napus panel. The percentage of the varieties in a region accounted for in the panel is shown in the circle." path-from-xml="dsx013f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357257/dsx013f5.png" path-from-xml="dsx013f5.png" target="_blank">View large</a><a section="106357257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f5.png?Expires=2147483647&Signature=VZomE-QCxrOn86--yxL~BiV-ZH0sOpfSWWeZUvBbrHltyqn46psYMNYakJXj7~3TO0qxbtPaTWZrPofgJrl3or8xaS8qGVgP2pGrdOUpBjbNyM5VVzTScDxSvt6rTZ8tMlq4jRXrI8q5nq-E7RHIi-3yKfwQqQyLvSzVtubnDO7MiP3VWiUGJ8ZUNJslNqyvnfPMHcTI8X25vjK3fqWHmOlRI~4i31WTuMNcvGE2nR7mFfNLH9mtMqu1VE8tk-M5yTWee-LfCRX-pzyChypf5Xn~Tr7K122Izy6kgD~BVew2drwIND-M2fhPdh1YoSgYsD4UkLu01ePUKeA~DjJ1~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357257&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The geographical distribution of the <em>BnA3Hap</em> haplotypes in the <em>B. napus</em> panel. The percentage of the varieties in a region accounted for in the panel is shown in the circle.</p></div></div><div content-id="dsx013-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f5.png?Expires=2147483647&Signature=STAGEDglE9u6x72ECfFF3-jqwZbLkYMTqfOTKZo6x6dlrMoJMK2dVk1iicOsrYuxSmJ3uJSc5varTAoixiDTDlXWdWTGt4UuKe22gzA0Siz5BuEeuHnYWsBbaW2sjCXYAKl0mSnIuE1za6RcQjYpQyGx0TgZfZSxzwGeQMutVQ59BZAmeHF3NyttzyOjzvCJP8xoGZqckGJXT-lvUs5HhLWaMuKRq2Xvgg0TNqums6EShyPFIjDhEoFX7EEApnGA6Y0Vlmy34f3nvq-cZwMzzGXzZc-AseK37I2A-OjoD4mC25iidk0E-b9~yESY5K8ySY5fdBrxk40z40mTpkXseg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The geographical distribution of the BnA3Hap haplotypes in the B. napus panel. The percentage of the varieties in a region accounted for in the panel is shown in the circle." path-from-xml="dsx013f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357257/dsx013f5.png" path-from-xml="dsx013f5.png" target="_blank">View large</a><a section="106357257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f5.png?Expires=2147483647&Signature=VZomE-QCxrOn86--yxL~BiV-ZH0sOpfSWWeZUvBbrHltyqn46psYMNYakJXj7~3TO0qxbtPaTWZrPofgJrl3or8xaS8qGVgP2pGrdOUpBjbNyM5VVzTScDxSvt6rTZ8tMlq4jRXrI8q5nq-E7RHIi-3yKfwQqQyLvSzVtubnDO7MiP3VWiUGJ8ZUNJslNqyvnfPMHcTI8X25vjK3fqWHmOlRI~4i31WTuMNcvGE2nR7mFfNLH9mtMqu1VE8tk-M5yTWee-LfCRX-pzyChypf5Xn~Tr7K122Izy6kgD~BVew2drwIND-M2fhPdh1YoSgYsD4UkLu01ePUKeA~DjJ1~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357257&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The geographical distribution of the <em>BnA3Hap</em> haplotypes in the <em>B. napus</em> panel. The percentage of the varieties in a region accounted for in the panel is shown in the circle.</p></div></div>                        <a id="106357258" scrollto-destination="106357258"></a>
<div id="dsx013-F6" data-id="dsx013-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f6.png?Expires=2147483647&Signature=Zjhzqz8quTIlzjYdnFOaNUfczHAmBypTeNUvru16K0AK9R~5rrURkKBB8ajwCncQqO03niKkrqKzxwTmRTAXHr36Lgxf0hRPrTN22CsUnwgsjmqr598bKh5KvD5MRT4Be-GD3H-2nqtYxn8LrSw9qn9aUHAZADVas-Y9w61UsCBfSWLYLt3uRGFBVKHz1g1FBwiesq8SBMXfLjqqTYWyXbYBtp3MflJnaURg3HNIAzjCPecmEa5NuwBvfTVJFeEIgjaXcKXb3NY6SYfPu2mJtT6UL6T38Npdg7RX~g3LSK12FQSIuX33dimJH7kCyLk8~uHi16iWSPR~NNiPetVh7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The Tajima’s D values of geographical distribution in BnA3Hap haplotypes." path-from-xml="dsx013f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357258/dsx013f6.png" path-from-xml="dsx013f6.png" target="_blank">View large</a><a section="106357258" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f6.png?Expires=2147483647&Signature=YUd-M6Bv9gOQqIzwDMMuQKjL94XQqjSiqS6hXO82OoPvKhlk-QykFif~XZMIPITj2pKYiR4yIRQp79xCn7BeE3NMbDe0eTSHUH9I-jWhTDqCbsvtgf8xKvjhZ0t8mNM~8mY1jwCjHQY1N4DupJeW10Snx3Co-LbmXmFmHhKCTITMyryTEjNUj2NZGItuuubK0~wi9MJ6j2KD0Ni9ei6ovUBloE59FF-0BMnNE03UY-VJnyQRQPfJCdk~osxrO7K2~fbWUavvhIy9XQLyeX-SsPWkXQwAD4b0ltOqlBf8248v~VSFSNzfQvxDbBNvT7-w98QRRguwpk0oCqKqEB2H2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357258&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The Tajima’s D values of geographical distribution in <em>BnA3Hap</em> haplotypes.</p></div></div><div content-id="dsx013-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/m_dsx013f6.png?Expires=2147483647&Signature=Zjhzqz8quTIlzjYdnFOaNUfczHAmBypTeNUvru16K0AK9R~5rrURkKBB8ajwCncQqO03niKkrqKzxwTmRTAXHr36Lgxf0hRPrTN22CsUnwgsjmqr598bKh5KvD5MRT4Be-GD3H-2nqtYxn8LrSw9qn9aUHAZADVas-Y9w61UsCBfSWLYLt3uRGFBVKHz1g1FBwiesq8SBMXfLjqqTYWyXbYBtp3MflJnaURg3HNIAzjCPecmEa5NuwBvfTVJFeEIgjaXcKXb3NY6SYfPu2mJtT6UL6T38Npdg7RX~g3LSK12FQSIuX33dimJH7kCyLk8~uHi16iWSPR~NNiPetVh7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The Tajima’s D values of geographical distribution in BnA3Hap haplotypes." path-from-xml="dsx013f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357258/dsx013f6.png" path-from-xml="dsx013f6.png" target="_blank">View large</a><a section="106357258" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/dsx013f6.png?Expires=2147483647&Signature=YUd-M6Bv9gOQqIzwDMMuQKjL94XQqjSiqS6hXO82OoPvKhlk-QykFif~XZMIPITj2pKYiR4yIRQp79xCn7BeE3NMbDe0eTSHUH9I-jWhTDqCbsvtgf8xKvjhZ0t8mNM~8mY1jwCjHQY1N4DupJeW10Snx3Co-LbmXmFmHhKCTITMyryTEjNUj2NZGItuuubK0~wi9MJ6j2KD0Ni9ei6ovUBloE59FF-0BMnNE03UY-VJnyQRQPfJCdk~osxrO7K2~fbWUavvhIy9XQLyeX-SsPWkXQwAD4b0ltOqlBf8248v~VSFSNzfQvxDbBNvT7-w98QRRguwpk0oCqKqEB2H2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357258&ar=3744590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx013f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The Tajima’s D values of geographical distribution in <em>BnA3Hap</em> haplotypes.</p></div></div><p>The alignment of <em>BnaA03g05800D, BnaA03g05820D, BnaA03g05830D</em> and <em>BnaA03g05850D</em> in <em>B. napus</em> was identical to that of their orthologous genes in <em>B. rapa</em>, however, they differed from orthologs genes in <em>A. thaliana</em>. One gene was located on chromosome 3, and three were located on chromosome 5 of <em>A. thaliana</em>, suggesting hybridization between <em>B. rapa</em> and <em>B. oleracea</em>, while the <em>BnA3Hap</em> was conserved in the A genome (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s6_dsx013.docx?Expires=2147483647&Signature=Y~eThUjWExtw1YoopHiODzuZUfC7eSNqvQvSbWetO6ygn~cCyP7gfNu-BdY0xpFm7l8uJK37d8L1s8RK5zspuRC2ssmtzi9GQFEQOqmcirdZnYX24CTNUaRF3rjRLNyXFPg58Xmn0Mf5JvTgRuif-u5SvipV2kO7AkbiscGstkfcaak7C41M~9gWegyyLcAKHWngNY-OnaMRZI-zM9jb9f8D57VjKRAX2bMbdHCi19LQFWC7aPtD20nmRFCouGWY2Wl60gfbbTFro4gMrrPR9b1YO8xJZRbG~MMFQMmWLYJtfT0Z84Q4qC5qk~NshwNPRU24BvQWPfUy7Z0J9vtYIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>).</p>                            <h2 scrollto-destination=106357260 id="106357260" class="section-title" >4 Discussion</h2>
                            <h3 scrollto-destination=106357261 id="106357261" class="section-title" >4.1 Genetic diversity, relative kinship and LD</h3>
<p>In previous <em>B. napus</em> association panels, two subgroups were identified in 192 <em>B. napus</em> accessions<a reveal-id="dsx013-B32" data-open="dsx013-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and 523 <em>B. napus</em> accessions,<a reveal-id="dsx013-B21" data-open="dsx013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> but three subgroups in 472 <em>B. napus</em> accessions.<a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> In the present study, 405 varieties of the panel of Liu<a reveal-id="dsx013-B21" data-open="dsx013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> were used and similarly divided into two subgroups (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). The LD, gene diversity, PIC, and relative kinship were slightly different between Subgroup1 and Subgroup2 (<a reveal-id="dsx013-F1" data-open="dsx013-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). As compared with European <em>B. napus</em>, the Chinese <em>B. napus</em> accessions studied had a more diverse genetic background (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s1_dsx013.xlsx?Expires=2147483647&Signature=OxQkLsitoMyFFYOhWCHthnOq49K6I8y6sLJepzaLkFzRDL4WQdzPBivTVrO~5oM-QI-32whWtq1Wsv6h76RlfWXLqiOnI5tba2e0xHKo7LHwAgxMe9HusWwAGLBN~rsNmga1zb5ruq6K-3wKjnh74exh9l5592ey1PP7ntpRAUsmMfc2B5jvaxvg0fKCaXt8SUqxIlmo04DIhJMrzmwDbDA1-c9LaW~OpiIaqYXgDJD4YjweRKRANGUjMSWhiNtEGE7q6BUGM07JsDOx-GzBbpnSugIW3I8Kp97paef-V0Q1VCexKSMntQzPWHJuslTiQSVt1PQi9LhZHLgwSulK2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), due to greater recombination in the A subgenome introduced through inter-specific hybridizations between <em>B. napus</em> and <em>B. rapa.</em> The LD in the C genome is greater than in the A genome (<a reveal-id="dsx013-T3" data-open="dsx013-T3" class="link link-reveal link-table xref-fig">Table 3</a>), but lower in PIC and gene diversity, indicating that the C genome may contribute more valuable alleles to adapt to different growth environments. In addition, the LD decay observed here- 1.06 Mb was different from previous <em>B. napus</em> panels, which observed 0.5–1 cM in 192 inbred lines,<a reveal-id="dsx013-B32" data-open="dsx013-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> 2.0 Mb in 472 inbred lines<a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and 6.5 Mb in 523 inbred lines,<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and larger than that of other species, e.g. 100 kb–1 Mb in rice,<a reveal-id="dsx013-B33" data-open="dsx013-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> 1–100 kb in maize<a reveal-id="dsx013-B12" data-open="dsx013-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and 250 kb in <em>Arabidopsis</em>.<a reveal-id="dsx013-B34" data-open="dsx013-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The comparatively high resolution provided by association mapping is dependent upon the LD of genome.<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The low LD detected in this study indicated that this <em>B. napus</em> panel is suitable for association analysis and has the potential to identify SNPs in a narrow intervals equivalent to the distance of LD decay of 1.06 Mb. In addition, the number of SNPs, the length of LD decay, and the SNP density on each chromosome were different in this study (<a reveal-id="dsx013-T3" data-open="dsx013-T3" class="link link-reveal link-table xref-fig">Table 3</a>), similar to the studies with 523<a reveal-id="dsx013-B14" data-open="dsx013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and 472<a reveal-id="dsx013-B20" data-open="dsx013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> accessions.</p><p>In this study, we did not investigate the association between the population structure and the breeding history, as the cultivars in this panel lack breeding information history. The subpopulations in a panel of 70 tomato lines corresponded to historical patterns of breeding conducted for specific production environments.<a reveal-id="dsx013-B35" data-open="dsx013-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> However, the population structure in a 472 accessions <em>Brassica napus</em> population showed only weak association with breeding history.<a reveal-id="dsx013-B36" data-open="dsx013-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a></p><p>Structural variation of this <em>B. napus</em> panel was evaluated by gsrc, an R package for genome structure rearrangement calling.<a reveal-id="dsx013-B37" data-open="dsx013-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> The C subgenome had more deletions and duplications than the A subgenome (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s7_dsx013.docx?Expires=2147483647&Signature=GnIY2kco8Zppr3Gmn9H~K2D-lKwDEkhbDO1n~yq4WW~F~3hGPcNffODmQQR0fMVPlyq7uJC5nk1-GV4PvvCCRGFnRpUdy2XcTQOIvh2X13AANRX9Xd3fRLeU99NKVRJIlAETKwJbnnC00nG9A8h3FsBq8OZu34HbB6AgndfydOXAoefUXrfQkTIy1nxX2HzOAucH8HMgGqwQunOUOs45vzheoa~cbjq~aJQQwpjsO9XUnY~YxH-0yQiwLf~bD6vK7mb~BeWF5bmik6zLuIYBxVqrpjh3OW4BhStUc3VYk7dIumrddT8eke6J668-PY3roMLZapBAGL-gtn~qXLuO9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>a). The number of CNVs in the region of ‘<em>BnA3Hap</em>’ was particularly low and therefore they have been not included in our association model. Genome structure rearrangements are a common phenomenon in allopolyploid species, which have been shown to affect phenotypic traits, and therefore it is of interest to monitor them on a genome wide scale in the future to detect further QTL. Subgenome C contains many CNVs and requires a deeper analysis to validate these and account for them in the association. For some regions (e.g. A10) CNVs were detected in the majority of lines in the panel and indicate a systematic effect.</p>                            <h3 scrollto-destination=106357265 id="106357265" class="section-title" >4.2 Candidate genes identified for RSA</h3>
<p>In this study, only 9 out of 285 SNPs colocalized with previously identified QTLs for RSA traits, which can be attributed to three reasons. First, as compared with QTL mapping, GWAS can identify relatively higher resolution genetic loci from underlying traits because natural populations incorporate ancient recombination events. Second, RSA traits in the <em>Bna</em>TNDH population were screened at low and high P with a ‘pouch and wick’<a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and an agar culture systems,<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> respectively. RSA traits of GWAS population were screened using a solution culture system at both low and high P and a ‘pouch and wick’ system at low P. The growth conditions influenced genotypic ranking and the range of variability.<a reveal-id="dsx013-B38" data-open="dsx013-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Third, uncorrelated data may be the result different environmental parameters between experiments.</p><p>The response of RSA traits to low Pi availability in <em>B. napus</em> has been well characterized.<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx013-B8" data-open="dsx013-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> With low P availability, typically a reduction in the development of the primary roots occur,<a reveal-id="dsx013-B4" data-open="dsx013-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx013-B8" data-open="dsx013-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx013-B9" data-open="dsx013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and increases the number and length of lateral roots are observed.<a reveal-id="dsx013-B3" data-open="dsx013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx013-B35" data-open="dsx013-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><em>LPR2</em> (<em>At1g71040</em>) was shown to regulate primary root development in <em>Arabidopsis</em> under low Pi availability. It is a paralogue of <em>Arabidopsis LPR1</em> (<em>At1g23010</em>), which encodes a multicopper oxidase protein, which are essential for the reducing root growth observed when root tips contact low-Pi media.<a reveal-id="dsx013-B5" data-open="dsx013-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx013-B39" data-open="dsx013-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The <em>RACK1</em> orthologous gene in <em>Arabidopsis</em> regulates lateral root formation by changing the sensitivity of the plant to gibberellin, brassinosteroid, abscisic acid, and auxin.<a reveal-id="dsx013-B40" data-open="dsx013-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p><p>The genes associated with significant SNPs for root traits in the loci of target QTL were predicted to be the candidate genes for P efficiency. <em>BnaA03g05800D</em>, <em>BnaA03g05820D</em>, <em>BnaA03g05830D</em> and <em>BnaA03g05850D</em> were predicted to be closely associated with P efficiency, which were also located in the confidence interval of the QTL for primary root length at low P as well as the LD decay region of the peak SNP Bn-A03-p3001467 at 2.49–2.72 Mb on A3 (<a reveal-id="dsx013-F2" data-open="dsx013-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). All genes had significantly different expression in the P-efficient cultivar ‘Eyou Changjia’ between low P and high P (<a reveal-id="dsx013-F2" data-open="dsx013-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The differential expression of the putative candidate genes in response to low-P stress was also confirmed in the P-efficient and P-inefficient accessions (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). <em>BnaA03g05800D</em> in <em>B. napus</em> is orthologous to <em>AT5G16010</em> in <em>Arabidopsis</em>, which encodes a 3-oxo-5-alpha-steroid 4-dehydrogenase family protein that moves between various organs under normal or nutrient-limiting conditions.<a reveal-id="dsx013-B41" data-open="dsx013-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><em>BnaA03g05820D</em> is orthologous to <em>AT3G02530</em> encoding a TCP family transcription factor.<a reveal-id="dsx013-B42" data-open="dsx013-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><em>BnaA03g05830D</em> is homologous to <em>AT5G16110</em>, which regulates lateral root development.<a reveal-id="dsx013-B43" data-open="dsx013-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Along with <em>MEX1</em>, the <em>AT5G16150- plastidic GLC translocator</em> (<em>PGLCT</em>) is essential for the growth and development of starch degradation products from chloroplasts and starch-mediated pathway for photoassimilate export in <em>Arabidopsis</em>.</p><p><em>BnaA03g05800D</em>, <em>BnaA03g05820D</em>, <em>BnaA03g05830D</em> and <em>BnaA03g05850D</em> had differential SNPs within the 5’ UTR or introns; the A/G alleles of lead SNP Bn-A03-p3073797 were located in the introns of <em>BnaA03g5800D</em>, A/C alleles of lead SNP Bn-A03-p3084593 were located in the introns of <em>BnaA03g5820D</em>, A/G alleles of lead SNP Bn-A03-p3091011 were located in the introns of <em>BnaA03g5830D</em>, and the A/G alleles of lead SNP Bn-A03-p3097703 were located in introns of <em>BnaA03g5850D.</em> These polymorphisms may cause differential gene expression between P-efficient and P-inefficient accessions (<a reveal-id="dsx013-F3" data-open="dsx013-F3" class="link link-reveal link-table xref-fig">Figs. 3 and 4</a>h). The characterization of the genetic elements associated with these traits could therefore provide elite alleles for the breeding of crops adapted to low P.</p>                            <h3 scrollto-destination=106357270 id="106357270" class="section-title" >4.3 <em>BnA3Hap</em> haplotypes</h3>
<p>Breeders have matched functionally different P-tolerance alleles to specific environments during selection. Oilseed rape was selected as a crop 300–400 years ago.<a reveal-id="dsx013-B36" data-open="dsx013-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In this study, the SNPs within candidate genes (<em>BnaA03g5800D, BnaA03g5820D, BnaA03g5830D</em> and <em>BnaA03g5850D</em>) on A3 formed a haplotype ‘<em>BnA3Hap</em>’, contributes to <em>B. napus</em> tolerance to low P stress (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4g and h</a>). <em>BnA3Hap</em> haplotypes were detected at low P under the ‘pouch and wick’ system (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4a–f</a>), and verified under the hydroponic system for two years. In addition, the Tajima’s D values of <em>BnA3Hap</em> haplotypes in <em>B. napus</em> varieties were strongly associated with the distribution of the agronomic P surpluses in soil<a reveal-id="dsx013-B44" data-open="dsx013-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> (<a reveal-id="dsx013-F5" data-open="dsx013-F5" class="link link-reveal link-table xref-fig">Figs. 5 and 6</a>). Compared to ancestor species (<em>B. rapa</em> and <em>A. thaliana</em>), the order of the four orthologous genes of <em>BnA3Hap</em> haplotypes in A3 in <em>B. napus</em> showed collinearity with that in <em>B. rapa</em>, and were slightly different to that in <em>A. thaliana</em> (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s6_dsx013.docx?Expires=2147483647&Signature=Y~eThUjWExtw1YoopHiODzuZUfC7eSNqvQvSbWetO6ygn~cCyP7gfNu-BdY0xpFm7l8uJK37d8L1s8RK5zspuRC2ssmtzi9GQFEQOqmcirdZnYX24CTNUaRF3rjRLNyXFPg58Xmn0Mf5JvTgRuif-u5SvipV2kO7AkbiscGstkfcaak7C41M~9gWegyyLcAKHWngNY-OnaMRZI-zM9jb9f8D57VjKRAX2bMbdHCi19LQFWC7aPtD20nmRFCouGWY2Wl60gfbbTFro4gMrrPR9b1YO8xJZRbG~MMFQMmWLYJtfT0Z84Q4qC5qk~NshwNPRU24BvQWPfUy7Z0J9vtYIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>), which indicates that this haplotype contributed to six RSA traits at LP (<a reveal-id="dsx013-F4" data-open="dsx013-F4" class="link link-reveal link-table xref-fig">Fig. 4a–f</a>) that were conserved during <em>B. napus</em> breeding. The discovery of the optimal haplotype of <em>BnA3Hap</em> can now enable the accurate selection of <em>B. napus</em> with higher P efficiencies and improve our understanding of the molecular mechanisms underlying P efficiency in plants.</p>                            <h2 scrollto-destination=106357272 id="106357272" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s1_dsx013.docx?Expires=2147483647&Signature=fv5qu4w438o~SjSQHf-QJtbVQWJlaQPsr9RD6T1TZAe1MTBBPcT9RVt418eorgHnjgxjq~z43OKJWKTDF~-rAEY63fcA8pMwzb9sAFDMEoX2HWOtjug6xgh6ZWUr9wrN2FRivwxqklZhb-R0DoIqa2V0jXB-uyEhlcc3msisUKCg26nNRWSTCVt-IzFmSkCxRI6UtLeTuIhJJQSbqOD8qces2yUw-V0I-UcBMZ2Y1KMQkdxHM0GV22KlwmdkCRACmJIixiWSWl03T1QJq2aKuTkq6Fimqy7cE5djMMMjk8mEhbrfgq0e6dr2DrDQfWxaWqeJCZh~egVpJ6feF7mIaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> Online.</p>                            <h2 scrollto-destination=106357274 id="106357274" class="section-title" >Conflict of interest</h2>
<p>None declared. </p>                            <h2 scrollto-destination=106357276 id="106357276" class="section-title" >Funding</h2>
<p>The authors acknowledge the financial support from the National Nature Science Foundation of China (Grant No. 31471933), the UK Biotechnology and Biological Sciences Research Council (BBSRC) (Crop Improvement Research Club, CIRC, Project BB/J019631/1; International Partnering Award BB/J020443/1), New Century Excellent Talents in University of Ministry of Education of China (Grant No. NCET-13-0809), Natural and Fundamental Research Funds for the Central Universities of China (Grant No. 2014PY020; 2662015PY105).</p>                            <h2 scrollto-destination=106357278 id="106357278" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx013-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kochian</div>
               <div class="given-names">L.V.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Plant nutrition, rooting for more phosphorus</div>. <div class="source">Nature</div> , <div class="volume">488</div>, <div class="fpage">466</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20nutrition%2C%20rooting%20for%20more%20phosphorus&amp;author=L.V.%20Kochian&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=466-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/488466a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F488466a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F488466a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22914160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lynch</div>
               <div class="given-names">J.P.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Root phenes that reduce the metabolic costs of soil exploration, opportunities for 21<sup>st</sup> century agriculture</div>. <div class="source">Plant Cell Environ</div> ., <div class="volume">38</div>, <div class="fpage">1775</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%20phenes%20that%20reduce%20the%20metabolic%20costs%20of%20soil%20exploration%2C%20opportunities%20for%2021st%20century%20agriculture&amp;author=J.P.%20Lynch&amp;publication_year=2015&amp;journal=Plant%20Cell%20Environ&amp;volume=38&amp;pages=1775-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2015.38.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2015.38.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2015.38.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25255708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Broadley</div> <div class="given-names">M.R.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">High-throughput root phenotyping screens identify genetic loci associated with root architectural traits in <em>Brassica napus</em> under contrasting phosphate availabilities</div>. <div class="source">Ann. Bot-London</div> ., <div class="volume">112</div><strong>,</strong><div class="fpage">381</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20root%20phenotyping%20screens%20identify%20genetic%20loci%20associated%20with%20root%20architectural%20traits%20in%20Brassica%20napus%20under%20contrasting%20phosphate%20availabilities&amp;author=L.%20Shi&amp;author=T.%20Shi&amp;author=M.R.%20Broadley&amp;publication_year=2013&amp;journal=Ann.%20Bot-London&amp;volume=112&amp;pages=381-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcs245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcs245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcs245"> </span></div> </div></div></div></div></div><div content-id="dsx013-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hammond</div>
               <div class="given-names">J.P.</div>
            </div>, <div class="name string-name"><div class="surname">Broadley</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">White</div> <div class="given-names">P.J.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Shoot yield drives phosphorus use efficiency in <em>Brassica oleracea</em> and correlates with root architecture traits</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">60</div>, <div class="fpage">1953</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shoot%20yield%20drives%20phosphorus%20use%20efficiency%20in%20Brassica%20oleracea%20and%20correlates%20with%20root%20architecture%20traits&amp;author=J.P.%20Hammond&amp;author=M.R.%20Broadley&amp;author=P.J.%20White&amp;publication_year=2009&amp;journal=J.%20Exp.%20Bot&amp;volume=60&amp;pages=1953-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erp083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferp083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferp083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19346243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Svistoonoff</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Creff</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Reymond</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Root tip contact with low-phosphate media reprograms plant root architecture</div>. <div class="source">Nat. Genet</div> ., <div class="volume">39</div>, <div class="fpage">792</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%20tip%20contact%20with%20low-phosphate%20media%20reprograms%20plant%20root%20architecture&amp;author=S.%20Svistoonoff&amp;author=A.%20Creff&amp;author=M.%20Reymond&amp;publication_year=2007&amp;journal=Nat.%20Genet&amp;volume=39&amp;pages=792-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17496893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wissuwa</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Wegner</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ae</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Yano</div> <div class="given-names">M.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">Substitution mapping of <em>Pup1</em>, a major QTL increasing phosphorus uptake of rice from a phosphorus-deficient soil</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">105</div>, <div class="fpage">890</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substitution%20mapping%20of%20Pup1%2C%20a%20major%20QTL%20increasing%20phosphorus%20uptake%20of%20rice%20from%20a%20phosphorus-deficient%20soil&amp;author=M.%20Wissuwa&amp;author=J.%20Wegner&amp;author=N.%20Ae&amp;author=M.%20Yano&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet&amp;volume=105&amp;pages=890-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1051-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1051-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1051-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gamuyao</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Chin</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Pariasca-Tanaka</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">The protein kinase Pstol1 from traditional rice confers tolerance of phosphorus deficiency</div>. <div class="source">Nature</div> , <div class="volume">488</div>, <div class="fpage">535</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20protein%20kinase%20Pstol1%20from%20traditional%20rice%20confers%20tolerance%20of%20phosphorus%20deficiency&amp;author=R.%20Gamuyao&amp;author=J.H.%20Chin&amp;author=J.%20Pariasca-Tanaka&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=535-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22914168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Meng</div> <div class="given-names">J.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Quantitative trait loci for root morphology in response to low phosphorus stress in <em>Brassica napus</em></div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">121</div>, <div class="fpage">181</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20for%20root%20morphology%20in%20response%20to%20low%20phosphorus%20stress%20in%20Brassica%20napus&amp;author=M.%20Yang&amp;author=G.%20Ding&amp;author=L.%20Shi&amp;author=J.%20Feng&amp;author=F.%20Xu&amp;author=J.%20Meng&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet&amp;volume=121&amp;pages=181-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1301-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1301-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1301-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20217384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">C.L.</div></div>, <div class="name string-name"><div class="surname">Xiang</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2016</div>. 
<div class="article-title">QTL meta-analysis of root traits in <em>Brassica napus</em> under contrasting phosphorus supply in two growth systems</div>. <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">33113</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20meta-analysis%20of%20root%20traits%20in%20Brassica%20napus%20under%20contrasting%20phosphorus%20supply%20in%20two%20growth%20systems&amp;author=Y.%20Zhang&amp;author=C.L.%20Thomas&amp;author=J.%20Xiang&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=33113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep33113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep33113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep33113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27624881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Sang</div> <div class="given-names">T.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>. <div class="source">Nat. Genet</div> ., <div class="volume">42</div>, <div class="fpage">961</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Courtois</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Audebert</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dardou</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genome-wide association mapping of root traits in a japonica rice panel</div>. <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e78037</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20of%20root%20traits%20in%20a%20japonica%20rice%20panel&amp;author=B.%20Courtois&amp;author=A.%20Audebert&amp;author=A.%20Dardou&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e78037" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0078037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0078037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0078037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24223758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Remington</div>
               <div class="given-names">D.L.</div>
            </div>, <div class="name string-name"><div class="surname">Thornsberry</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2001</div>, 
<div class="article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">98</div>, <div class="fpage">11479</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20and%20phenotypic%20associations%20in%20the%20maize%20genome&amp;author=D.L.%20Remington&amp;author=J.M.%20Thornsberry&amp;author=Y.%20Matsuoka&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=11479-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.201394398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.201394398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.201394398"> </span></div> </div></div></div></div></div><div content-id="dsx013-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tian</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P.J.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population</div>. <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">159</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20leaf%20architecture%20in%20the%20maize%20nested%20association%20mapping%20population&amp;author=F.%20Tian&amp;author=P.J.%20Bradbury&amp;author=P.J.%20Brown&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=159-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.746" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.746" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.746"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Genome-wide association study reveals the genetic architecture of flowering time in rapeseed (<em>Brassica napus</em> L.)</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">43</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20reveals%20the%20genetic%20architecture%20of%20flowering%20time%20in%20rapeseed%20(Brassica%20napus%20L.)&amp;author=L.%20Xu&amp;author=K.%20Hu&amp;author=Z.%20Zhang&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=43-52" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26659471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kamfwa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Cichy</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">Kelly</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">2015</div>. 
<div class="article-title">Genome-wide association analysis of symbiotic nitrogen fixation in common bean</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">128</div>, <div class="fpage">1999</div>–<div class="lpage">2017</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20of%20symbiotic%20nitrogen%20fixation%20in%20common%20bean&amp;author=K.%20Kamfwa&amp;author=K.A.%20Cichy&amp;author=J.D.%20Kelly&amp;publication_year=2015&amp;journal=Theor.%20Appl.%20Genet&amp;volume=128&amp;pages=1999-2017" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-015-2562-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-015-2562-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-015-2562-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26133733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mori</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Fukuda</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Vejchasarn</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Nestler</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pariasca-Tanaka</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wissuwa</div> <div class="given-names">M.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">The role of root size versus root efficiency in phosphorus acquisition in rice</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">67</div>, <div class="fpage">1179</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20root%20size%20versus%20root%20efficiency%20in%20phosphorus%20acquisition%20in%20rice&amp;author=A.%20Mori&amp;author=T.%20Fukuda&amp;author=P.%20Vejchasarn&amp;author=J.%20Nestler&amp;author=J.%20Pariasca-Tanaka&amp;author=M.%20Wissuwa&amp;publication_year=2016&amp;journal=J.%20Exp.%20Bot&amp;volume=67&amp;pages=1179-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erv557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferv557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferv557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26842979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">The acid phosphatase-encoding gene <em>GmACP1</em> contributes to soybean tolerance to low-phosphorus stress</div>. <div class="source">PLoS Genet</div> ., <div class="volume">10</div>, <div class="fpage">229</div>–<div class="lpage">31</div>.</div></div></div></div><div content-id="dsx013-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rosas</div>
               <div class="given-names">U.</div>
            </div>, <div class="name string-name"><div class="surname">Cibrian-Jaramillo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ristova</div> <div class="given-names">D.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Integration of responses within and across arabidopsis natural accessions uncovers loci controlling root systems architecture</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">15133</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20responses%20within%20and%20across%20arabidopsis%20natural%20accessions%20uncovers%20loci%20controlling%20root%20systems%20architecture&amp;author=U.%20Rosas&amp;author=A.%20Cibrian-Jaramillo&amp;author=D.%20Ristova&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=15133-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1305883110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1305883110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1305883110"> </span></div> </div></div></div></div></div><div content-id="dsx013-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div></div>, et al.  , 
<div class="article-title">Genetic variation in <em>ZmVPP1</em> contributes to drought tolerance in maize seedlings</div>. <div class="source">Nat. Genet</div> ., <div class="volume">48</div>, <div class="fpage">1233</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27526320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (<em>Brassica napus</em> L.)</div>. <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">355</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20dissects%20the%20genetic%20architecture%20of%20seed%20weight%20and%20seed%20quality%20in%20rapeseed%20(Brassica%20napus%20L.)&amp;author=F.%20Li&amp;author=B.%20Chen&amp;author=K.%20Xu&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24510440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">A genome-wide association study reveals novel elite allelic variations in seed oil content of <em>Brassica napus</em></div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">129</div>, <div class="fpage">1203</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20association%20study%20reveals%20novel%20elite%20allelic%20variations%20in%20seed%20oil%20content%20of%20Brassica%20napus&amp;author=S.%20Liu&amp;author=C.%20Fan&amp;author=J.%20Li&amp;publication_year=2016&amp;journal=Theor.%20Appl.%20Genet&amp;volume=129&amp;pages=1203-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-016-2697-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-016-2697-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-016-2697-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26912143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wei</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Jian</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Genome-wide association analysis and differential expression analysis of resistance to <em>Sclerotinia</em> stem rot in <em>Brassica napus</em></div>. <div class="source">Plant Biotechnol. J</div> ., <div class="volume">14</div>, <div class="fpage">1368</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20and%20differential%20expression%20analysis%20of%20resistance%20to%20Sclerotinia%20stem%20rot%20in%20Brassica%20napus&amp;author=L.%20Wei&amp;author=H.%20Jian&amp;author=K.%20Lu&amp;publication_year=2016&amp;journal=Plant%20Biotechnol.%20J&amp;volume=14&amp;pages=1368-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12501"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26563848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bus</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Korber</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Parkin</div> <div class="given-names">I.A.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Species- and genome-wide dissection of the shoot ionome in <em>Brassica napus</em> and its relationship to seedling development</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">485</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species-%20and%20genome-wide%20dissection%20of%20the%20shoot%20ionome%20in%20Brassica%20napus%20and%20its%20relationship%20to%20seedling%20development&amp;author=A.%20Bus&amp;author=N.%20Korber&amp;author=I.A.%20Parkin&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci&amp;volume=5&amp;pages=485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25324847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cox</div>
               <div class="given-names">C.B.</div>
            </div>, <div class="name string-name"><div class="surname">Moore</div> <div class="given-names">P.D.</div></div>, <div class="name string-name"><div class="surname">Ladle</div> <div class="given-names">R.</div></div></div>
          <div class="year">2016</div>, <div class="source">Biogeography: An Ecological and Evolutionary Approach</div> . 
<div class="publisher-name">Wiley-Blackwell</div>., ISBN 978-1-118-96858-1, pp.106.</div></div></div></div><div content-id="dsx013-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thomas</div>
               <div class="given-names">C.L.</div>
            </div>, <div class="name string-name"><div class="surname">Graham</div> <div class="given-names">N.S.</div></div>, <div class="name string-name"><div class="surname">Hayden</div> <div class="given-names">R.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">High-throughput phenotyping HTP, identifies seedling root traits linked to variation in seed yield and nutrient capture in field-grown oilseed rape (<em>Brassica napus</em> L.)</div>. <div class="source">Ann. Bot-London</div> ., <div class="volume">118</div>, <div class="fpage">655</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20phenotyping%20HTP%2C%20identifies%20seedling%20root%20traits%20linked%20to%20variation%20in%20seed%20yield%20and%20nutrient%20capture%20in%20field-grown%20oilseed%20rape%20(Brassica%20napus%20L.)&amp;author=C.L.%20Thomas&amp;author=N.S.%20Graham&amp;author=R.%20Hayden&amp;publication_year=2016&amp;journal=Ann.%20Bot-London&amp;volume=118&amp;pages=655-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcw046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcw046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcw046"> </span></div> </div></div></div></div></div><div content-id="dsx013-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Atkinson</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Wingen</div> <div class="given-names">L.U.</div></div>, <div class="name string-name"><div class="surname">Griffiths</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Phenotyping pipeline reveals major seedling root growth QTL in hexaploid wheat</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">66</div>, <div class="fpage">2283</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotyping%20pipeline%20reveals%20major%20seedling%20root%20growth%20QTL%20in%20hexaploid%20wheat&amp;author=J.A.%20Atkinson&amp;author=L.U.%20Wingen&amp;author=M.%20Griffiths&amp;publication_year=2015&amp;journal=J.%20Exp.%20Bot&amp;volume=66&amp;pages=2283-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erv006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferv006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferv006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25740921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Lv</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Identification of molecular markers linked to trilocular gene (<em>mc1</em>) in <em>Brassica juncea</em> L</div>. <div class="source">Mol. Breeding</div> , <div class="volume">33</div>, <div class="fpage">425</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20molecular%20markers%20linked%20to%20trilocular%20gene%20(mc1)%20in%20Brassica%20juncea%20L&amp;author=P.%20Xu&amp;author=Z.%20Lv&amp;author=X.%20Zhang&amp;publication_year=2013&amp;journal=Mol.%20Breeding&amp;volume=33&amp;pages=425-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-013-9960-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-013-9960-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-013-9960-7"> </span></div> </div></div></div></div></div><div content-id="dsx013-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chalhoub</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Denoeud</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Early allopolyploid evolution in the post-Neolithic <em>Brassica napus</em> oilseed genome</div>. <div class="source">Science</div> , <div class="volume">345</div>, <div class="fpage">950</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Early%20allopolyploid%20evolution%20in%20the%20post-Neolithic%20Brassica%20napus%20oilseed%20genome&amp;author=B.%20Chalhoub&amp;author=F.%20Denoeud&amp;author=S.%20Liu&amp;publication_year=2014&amp;journal=Science&amp;volume=345&amp;pages=950-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1253435" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1253435" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1253435"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25146293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pritchard</div>
               <div class="given-names">J.K.</div>
            </div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rosenberg</div> <div class="given-names">N.A.</div></div>, <div class="name string-name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">Association mapping in structured populations</div>. <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">67</div>, <div class="fpage">170</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20in%20structured%20populations&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=N.A.%20Rosenberg&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=67&amp;pages=170-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/302959" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F302959" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F302959"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Evanno</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Regnaut</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Goudet</div> <div class="given-names">J.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Detecting the number of clusters of individuals using the software STRUCTURE, a simulation study</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">14</div>, <div class="fpage">2611</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%2C%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;publication_year=2005&amp;journal=Mol.%20Ecol&amp;volume=14&amp;pages=2611-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2005.14.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2005.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2005.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duan</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">F.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Identification of Phosphorous Efficient Germplasm in Oilseed Rape</div>. <div class="source">J. Plant Nutr</div> ., <div class="volume">32</div>, <div class="fpage">1148</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Phosphorous%20Efficient%20Germplasm%20in%20Oilseed%20Rape&amp;author=H.%20Duan&amp;author=L.%20Shi&amp;author=X.%20Ye&amp;author=Y.%20Wang&amp;author=F.%20Xu&amp;publication_year=2009&amp;journal=J.%20Plant%20Nutr&amp;volume=32&amp;pages=1148-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/01904160902943171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F01904160902943171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F01904160902943171"> </span></div> </div></div></div></div></div><div content-id="dsx013-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xiao</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">W.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Genetic structure and linkage disequilibrium pattern of a rapeseed <em>Brassica napus</em> L., association mapping panel revealed by microsatellites</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">125</div>, <div class="fpage">437</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20linkage%20disequilibrium%20pattern%20of%20a%20rapeseed%20Brassica%20napus%20L.%2C%20association%20mapping%20panel%20revealed%20by%20microsatellites&amp;author=Y.%20Xiao&amp;author=D.%20Cai&amp;author=W.%20Yang&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet&amp;volume=125&amp;pages=437-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1843-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1843-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1843-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22437490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Tung</div> <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Eizenga</div> <div class="given-names">G.C.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em></div>. <div class="source">Nat. Commun</div> ., <div class="volume">2</div>, <div class="fpage">467</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20reveals%20a%20rich%20genetic%20architecture%20of%20complex%20traits%20in%20Oryza%20sativa&amp;author=K.%20Zhao&amp;author=C.W.%20Tung&amp;author=G.C.%20Eizenga&amp;publication_year=2011&amp;journal=Nat.%20Commun&amp;volume=2&amp;pages=467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms1467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms1467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms1467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21915109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nordborg</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Borevitz</div> <div class="given-names">J.O.</div></div>, <div class="name string-name"><div class="surname">Bergelson</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2002</div>, 
<div class="article-title">The extent of linkage disequilibrium in <em>Arabidopsis thaliana</em></div>. <div class="source">Nat. Genet</div> ., <div class="volume">30</div>, <div class="fpage">190</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20extent%20of%20linkage%20disequilibrium%20in%20Arabidopsis%20thaliana&amp;author=M.%20Nordborg&amp;author=J.O.%20Borevitz&amp;author=J.%20Bergelson&amp;publication_year=2002&amp;journal=Nat.%20Genet&amp;volume=30&amp;pages=190-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng813" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng813" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng813"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11780140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sim</div>
               <div class="given-names">S.C.</div>
            </div>, <div class="name string-name"><div class="surname">Robbins</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div></div>,et al.  <div class="year">2011</div>, 
<div class="article-title">Population structure and genetic differentiation associated with breeding history and selection in tomato (<em>Solanum lycopersicum</em> L.)</div>. <div class="source">Heredity</div> , <div class="volume">106</div>, <div class="fpage">927</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20genetic%20differentiation%20associated%20with%20breeding%20history%20and%20selection%20in%20tomato%20(Solanum%20lycopersicum%20L.)&amp;author=S.C.%20Sim&amp;author=M.D.%20Robbins&amp;author=A.%20Van%20Deynze&amp;publication_year=2011&amp;journal=Heredity&amp;volume=106&amp;pages=927-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2010.139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2010.139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2010.139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21081965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">B.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide investigation of genetic changes during modern breeding of <em>Brassica napus</em></div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">127</div>, <div class="fpage">1817</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20investigation%20of%20genetic%20changes%20during%20modern%20breeding%20of%20Brassica%20napus&amp;author=N.%20Wang&amp;author=F.%20Li&amp;author=B.%20Chen&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet&amp;volume=127&amp;pages=1817-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-014-2343-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-014-2343-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-014-2343-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24947439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grandke</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Snowdon</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Samans</div> <div class="given-names">B.</div></div></div>
          <div class="year">2017</div>
          <div class="article-title">gsrc-an R package for genome structure rearrangement calling</div>. <div class="source">Bioinformatics</div> , <div class="volume">33</div>, <div class="fpage">545</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=gsrc-an%20R%20package%20for%20genome%20structure%20rearrangement%20calling&amp;author=F.%20Grandke&amp;author=R.%20Snowdon&amp;author=B.%20Samans&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=545-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27797758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nestler</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wissuwa</div> <div class="given-names">M.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">Superior root hair formation confers root efficiency in some, but not all, rice genotypes upon p deficiency</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>, e75997.</div></div></div></div><div content-id="dsx013-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Reymond</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Svistoonoff</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Loudet</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Nussaume</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Desnos</div> <div class="given-names">T.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Identification of QTL controlling root growth response to phosphate starvation in <em>Arabidopsis thaliana</em></div>. <div class="source">Plant Cell Environ</div> ., <div class="volume">29</div>, <div class="fpage">115</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTL%20controlling%20root%20growth%20response%20to%20phosphate%20starvation%20in%20Arabidopsis%20thaliana&amp;author=M.%20Reymond&amp;author=S.%20Svistoonoff&amp;author=O.%20Loudet&amp;author=L.%20Nussaume&amp;author=T.%20Desnos&amp;publication_year=2006&amp;journal=Plant%20Cell%20Environ&amp;volume=29&amp;pages=115-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/PCE.2006.29.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2FPCE.2006.29.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2FPCE.2006.29.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17086758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">W.D.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.G.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title"><em>RACK1</em> is a negative regulator of ABA responses in Arabidopsis</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">60</div>, <div class="fpage">3819</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RACK1%20is%20a%20negative%20regulator%20of%20ABA%20responses%20in%20Arabidopsis&amp;author=J.%20Guo&amp;author=J.%20Wang&amp;author=L.%20Xi&amp;author=W.D.%20Huang&amp;author=J.%20Liang&amp;author=J.G.%20Chen&amp;publication_year=2009&amp;journal=J.%20Exp.%20Bot&amp;volume=60&amp;pages=3819-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erp221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferp221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferp221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19584117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thieme</div>
               <div class="given-names">C.J.</div>
            </div>, <div class="name string-name"><div class="surname">Rojas-Triana</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stecyk</div> <div class="given-names">E.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Endogenous arabidopsis messenger RNAs transported to distant tissues</div>. <div class="source">Nat. Plants</div> , <div class="volume">1</div>.</div></div></div></div><div content-id="dsx013-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jin</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">PlantTFDB 3.0, a portal for the functional and evolutionary study of plant transcription factors</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">1182</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlantTFDB%203.0%2C%20a%20portal%20for%20the%20functional%20and%20evolutionary%20study%20of%20plant%20transcription%20factors&amp;author=J.%20Jin&amp;author=H.%20Zhang&amp;author=L.%20Kong&amp;author=G.%20Gao&amp;author=J.%20Luo&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=1182-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1016"> </span></div> </div></div></div></div></div><div content-id="dsx013-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Laskowski</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Biller</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Stanley</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kajstura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Prusty</div> <div class="given-names">R.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Expression profiling of auxin-treated <em>Arabidopsis</em> roots, toward a molecular analysis of lateral root emergence</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">47</div>, <div class="fpage">788</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20profiling%20of%20auxin-treated%20Arabidopsis%20roots%2C%20toward%20a%20molecular%20analysis%20of%20lateral%20root%20emergence&amp;author=M.%20Laskowski&amp;author=S.%20Biller&amp;author=K.%20Stanley&amp;author=T.%20Kajstura&amp;author=R.%20Prusty&amp;publication_year=2006&amp;journal=Plant%20Cell%20Physiol&amp;volume=47&amp;pages=788-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcj043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcj043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcj043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16621846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx013-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx013-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">MacDonald</div>
               <div class="given-names">G.K.</div>
            </div>, <div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Potter</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Ramankutty</div> <div class="given-names">N.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Agronomic phosphorus imbalances across the world's croplands</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">3086</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Agronomic%20phosphorus%20imbalances%20across%20the%20world's%20croplands&amp;author=G.K.%20MacDonald&amp;author=E.M.%20Bennett&amp;author=P.A.%20Potter&amp;author=N.%20Ramankutty&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=3086-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1010808108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1010808108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1010808108"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3744590&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s1_dsx013.docx?Expires=2147483647&amp;Signature=fv5qu4w438o~SjSQHf-QJtbVQWJlaQPsr9RD6T1TZAe1MTBBPcT9RVt418eorgHnjgxjq~z43OKJWKTDF~-rAEY63fcA8pMwzb9sAFDMEoX2HWOtjug6xgh6ZWUr9wrN2FRivwxqklZhb-R0DoIqa2V0jXB-uyEhlcc3msisUKCg26nNRWSTCVt-IzFmSkCxRI6UtLeTuIhJJQSbqOD8qces2yUw-V0I-UcBMZ2Y1KMQkdxHM0GV22KlwmdkCRACmJIixiWSWl03T1QJq2aKuTkq6Fimqy7cE5djMMMjk8mEhbrfgq0e6dr2DrDQfWxaWqeJCZh~egVpJ6feF7mIaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s2_dsx013.docx?Expires=2147483647&amp;Signature=2drZJUKGZofUqCrVJagbdRaMB62Fp9lmw8O-DH4wPA8Yh6s7dIhoKmfyGVmBiqtzkrjdD86GkcqzmWAiq-luY4hbIi87xlbMRNFg9JLg7TFHSqgD2y5KK0RhTnoZ5q0raV3P4j8OHP6GVv3o~h5KvQ7YPOxhYYwocW3TWZrEr4W2qmGxBT3jeb5Q0O-xtWjlvbMurVh9cx1jlo-vACjPeINxm5x8ED8YEAmAo-61K4nNKaL4BCzYu0aAqCJImnz4sx0JGPlnRP1jccraZ0qLWkHJNU14a9qM~J4UOe~~M0zIUVxC6zM5JoeFT-O0YPGSSdbhjGncqXiRxvIowaTUtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s3_dsx013.docx?Expires=2147483647&amp;Signature=OFxDF3GY~0YyAh3E5wS~Jreb1lFs2aRxzyVMVBsCQkcW-wPIz1n7jJd~J8wi6yV5mSxtc3hvUYB8EfCdVf8MnAOuH58kVnzl8Xu~F5Oc2fWtqPwH6jQfAZ6DHnmkY6YW-sm8uP4MLb4H2EthxuX2VlGXkSR-IjsizV3h3edDfG86uy2OpZPpzvjVqF3bcxvcOX1RVjQ0omehg8-KnfaRroAAR38vRVDpar1gISyZvCcQM0rGGh1-nIHO~c8-tU6ix0ZnOuFolVKYMMTREbJ6H16NmeRqguA2rGE9ogCzCPTQCVqSL1M80ZstmwxQUrIsR00j7sNEsu2TcdzizUi~uQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s4_dsx013.docx?Expires=2147483647&amp;Signature=mhLawzkdURKP6ru1iy3EPJg9sZ7SLIM~4QkBMkKLNKR7atsrbCXiDTkH3TeWDbbmBWLQEX73P2zYS0rbkf3q3wluW62F-lO~890AD1IkQS0w9ObIntb3A~EuOlpUbAzY~bh7uqSn-M~TEqKns9Ckhljy3LuaYk~ekAXFJeuY~V41O0k6uR6Dt1IcFUtsRW-0RD2kTnHXBDH7opJjLd5-6C5wIHMZfyqA0T12NNHIYUoyzd-Y-hRFcfRcozWUu~mESQpcwWrKBPaav9tKPquysvJk6IQxMJ9zEmAq9DPZJTR3OvQ6tVni-uzAHsNzEVbfB8PQbGTtupDU90NXE-YLlw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S4</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s5_dsx013.docx?Expires=2147483647&amp;Signature=2R0mKRdmdoV6oZiGGEBHkfQQB1z9ed3AZvmR~YiG60BMDmce1kx8Vb6tmcOtxHSY5pfRN224s45KIyjJJt4DH6JMOTP6XEORxns0~6qY~lsZNa3gzrk173wU~D-kWOC8zyMcmr64cMSjAMjHeHz7UVyXC66B7YfrbMKo5x11Bzz5Ih648O4ueBjf1b7UFI7N5fLajlbGsKK9Lhyb66ib00RIN0fsE-u~1Mc8KZl89-sJehOKEggeSjGIAwywye3y0G71jfTnT-PceqJ4RYA8fh4-7JZupWAd6uHI4eEyWqKM0a9QhLH6M~SVR1CxhLO4KvIxJu27LRhz9eLgMsXxDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S5</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s6_dsx013.docx?Expires=2147483647&amp;Signature=Y~eThUjWExtw1YoopHiODzuZUfC7eSNqvQvSbWetO6ygn~cCyP7gfNu-BdY0xpFm7l8uJK37d8L1s8RK5zspuRC2ssmtzi9GQFEQOqmcirdZnYX24CTNUaRF3rjRLNyXFPg58Xmn0Mf5JvTgRuif-u5SvipV2kO7AkbiscGstkfcaak7C41M~9gWegyyLcAKHWngNY-OnaMRZI-zM9jb9f8D57VjKRAX2bMbdHCi19LQFWC7aPtD20nmRFCouGWY2Wl60gfbbTFro4gMrrPR9b1YO8xJZRbG~MMFQMmWLYJtfT0Z84Q4qC5qk~NshwNPRU24BvQWPfUy7Z0J9vtYIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S6</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/figure_s7_dsx013.docx?Expires=2147483647&amp;Signature=GnIY2kco8Zppr3Gmn9H~K2D-lKwDEkhbDO1n~yq4WW~F~3hGPcNffODmQQR0fMVPlyq7uJC5nk1-GV4PvvCCRGFnRpUdy2XcTQOIvh2X13AANRX9Xd3fRLeU99NKVRJIlAETKwJbnnC00nG9A8h3FsBq8OZu34HbB6AgndfydOXAoefUXrfQkTIy1nxX2HzOAucH8HMgGqwQunOUOs45vzheoa~cbjq~aJQQwpjsO9XUnY~YxH-0yQiwLf~bD6vK7mb~BeWF5bmik6zLuIYBxVqrpjh3OW4BhStUc3VYk7dIumrddT8eke6J668-PY3roMLZapBAGL-gtn~qXLuO9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S7</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s1_dsx013.xlsx?Expires=2147483647&amp;Signature=OxQkLsitoMyFFYOhWCHthnOq49K6I8y6sLJepzaLkFzRDL4WQdzPBivTVrO~5oM-QI-32whWtq1Wsv6h76RlfWXLqiOnI5tba2e0xHKo7LHwAgxMe9HusWwAGLBN~rsNmga1zb5ruq6K-3wKjnh74exh9l5592ey1PP7ntpRAUsmMfc2B5jvaxvg0fKCaXt8SUqxIlmo04DIhJMrzmwDbDA1-c9LaW~OpiIaqYXgDJD4YjweRKRANGUjMSWhiNtEGE7q6BUGM07JsDOx-GzBbpnSugIW3I8Kp97paef-V0Q1VCexKSMntQzPWHJuslTiQSVt1PQi9LhZHLgwSulK2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s2_dsx013.xlsx?Expires=2147483647&amp;Signature=FWe7Yrdg9FWib4z0QCNID9TyUJtu4MQT25X7FHJzkKfGk5OoD7WfDwwAHSKQ-d6rKb9Od0-a8GZXfCeEx3NVWTbZ24Uj-VG9XE4sB11Ov~jsfOhU4r1yiP3JGujjmfjZQ1nUGk0B7IdqtrO-GOKnM~CgknkzB01N74FD0MXf4XZKU~7AVZdV9J2HJCNiaAC5uTa2Uez1kN1v8CALREHEg2p1qpJuhH4T~0W1ajszUcFYEjJKuazdCngX49WPMg5sqlECdWVo1ERR4C7tRaRBqJYMAWy14V9JAWpo3~g9ISNTefKdyBhCkt88pN3roAbD26bVErKNRd10a86AuNTVtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s3_dsx013.xlsx?Expires=2147483647&amp;Signature=QWtqdeRgICY7seGk2jdhsvJM1V6lkYXNLV7ECip7XYso~9i3UOu1wRuRD~xZ0xywmHDQTfP3DwrBryIMIQB48KSFLwr4h5gNm4qEZJe67IbbGCIOlLLazwqFLQYL4nCvLah3C52JOvglYDbwUhWO1f5OXs8x9VTyvxeXg1b-RiywSy-~viLL~w8BWVMjoBL5lN7~wPALtt8bAl3L9XD754gZ9limivgh9lULF2UOBwV2RA4QYzSbYF5-ed8vsbX845oAoqsyUxylQwuo1YGBIR5Ma8qeqyCEuG5TFHwVigGPl~6nB4wP8PD4Xc3pY9OOIJPTQjY0v0uCBTMXDij23Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s4_dsx013.xlsx?Expires=2147483647&amp;Signature=IbUcupsxUPT5cq30LZeuuLat6tZpW~hlnNB-rQe5C7NY6C28TDppw8OWOaLGVKgGOY3NWfOWblitIWUiNgAHrj3wynGS7Lp9DVyFyvb9mEnOs8nBjEnxgBtB-LCdGYqXrwxLzB1oGiIyYle7u0u6D3gNollYGRXSssDY2gy7MyXwGcVnDE6obvgEjZI8ThDlFBHE1WYYmpRZC7sKQQEzWUHlkURrSCk9V1F2JiqffLrMV3zaWfw1yBO-FlO-taTKBHqskUwt7PpphroBgXqvRhs8m2XGhvoCxfwI9oT4yrTAn0Ftiw3ajn1y4-r7grkwl8C9qWD~vGpBpdvUpcOlSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx013/2/table_s5_dsx013.xlsx?Expires=2147483647&amp;Signature=PdWR4~nIX8~729uYfN3uNVfXxYYhLa1ze9N8OgXEzN0WeA~p~a9n5mrzUiwEUcK~2xxu4nxIfV-3aXzosdTaTZSdAyMl3rGjDMajzxPx19WsvSsyuPI8F5jgjUaIo4g3L~UppGF3mybRmfIb-ibBm~mkpGgNT-u86Va1vnuLuUpHuSaqMyjyUrL6pe5sQ7rvRgl6CRPdUUftZ5ckglydFQu-pUCv~aMzprLs2HrF9IOKh8ID-sVQI2mVShmIWfAy0PD-APw5B~61geHHzSS8db4Kjn5YltlWtNazm0ezu5S-dLQ8nJkogq29P3~5PxRpGkpu1fDxik~bd~CDhtR0bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">923</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000407965100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3744590"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000407965100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f4%2f407%2f3744590">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000407965100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F4%2F407%2F3744590" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3744590'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genetic variants associated with the root system architecture of oilseed rape (Brassica napus L.) under contrasting phosphate supply" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f407%2f3744590">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

