

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/4/185/376012","@type":"ScholarlyArticle","name":"The Whole-genome Sequencing of the Obligate Intracellular Bacterium  Orientia tsutsugamushi  Revealed Massive Gene Amplification During Reductive Genome Evolution","datePublished":"2008-05-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2008-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn011","keywords":["<italic>Orientia tsutsugamushi</italic>","genome sequencing","obligate intracellular bacterium","repetitive sequence","IS element","integrative and conjugative element","gene amplification","genome reduction"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/4/185/376012","author":[{"name":"Nakayama, Keisuke","affiliation":"1  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Yamashita, Atsushi","affiliation":"2  Kitasato Institute for Life Science, Kitasato University ,  5-9-1 Shirogane, Minato-ku Tokyo 108-8641 ,  Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"3  Laboratory of Comparative Genomics ,  Graduate School of Information Science, Nara Institute of Science and Technology ,  8916-5 Takayama-cho, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Morimoto, Takuya","affiliation":"1  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Ogawa, Michihiro","affiliation":"3  Laboratory of Comparative Genomics ,  Graduate School of Information Science, Nara Institute of Science and Technology ,  8916-5 Takayama-cho, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Fukuhara, Masahiro","affiliation":"4  Laboratory of Microbiology, Faculty of Pharmacy ,  Niigata University of Pharmacy and Applied Life Sciences ,  265-1 Higashijima, Akiha-ku, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Urakami, Hiroshi","affiliation":"5  Laboratory of Food Microbiology and Food Safety, Faculty of Applied Life Sciences ,  Niigata University of Pharmacy and Applied Life Sciences ,  265-1 Higashijima, Akiha-ku, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Ohnishi, Makoto","affiliation":"6  Department of Bacteriology ,  National Institute for Infectious Diseases, Toyama 1-23-1, Shinjuku-ku, Tokyo 162-8640 ,  Japan","@type":"Person"},{"name":"Uchiyama, Ikuo","affiliation":"7  National Institute for Basic Biology, National Institutes of Natural Sciences ,  Nishigonaka 38, Myodaiji, Okazaki 444-8585 ,  Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"8  Frontier Science Research Center, University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Ooka, Tadasuke","affiliation":"1  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Oshima, Kenshiro","affiliation":"2  Kitasato Institute for Life Science, Kitasato University ,  5-9-1 Shirogane, Minato-ku Tokyo 108-8641 ,  Japan    9  Graduate School of Frontier Sciences, The University of Tokyo ,  5-1-5 Kasiwanoha, Kashiwa, Chiba, 277-8561 ,  Japan","@type":"Person"},{"name":"Tamura, Akira","affiliation":"10  Niigata University of Pharmacy and Applied Life Sciences ,  265-1 Higashijima, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"9  Graduate School of Frontier Sciences, The University of Tokyo ,  5-1-5 Kasiwanoha, Kashiwa, Chiba, 277-8561 ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"1  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan    8  Frontier Science Research Center, University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"}],"description":"Abstract.  Scrub typhus (‘Tsutsugamushi’ disease in Japanese) is a mite-borne infectious disease. The causative agent is Orientia tsutsugamushi, an obligate int","pageStart":"185","pageEnd":"199","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The Whole-genome Sequencing of the Obligate Intracellular Bacterium  Orientia tsutsugamushi  Revealed Massive Gene Amplification During Reductive Genome Evolution","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Whole-genome Sequencing of the Obligate Intracellular Bacterium  Orientia tsutsugamushi  Revealed Massive Gene Amplification During Reductive Genome Evolution" />
    <meta property="og:description" content="Abstract.  Scrub typhus (‘Tsutsugamushi’ disease in Japanese) is a mite-borne infectious disease. The causative agent is Orientia tsutsugamushi, an obligate int" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/4/185/376012" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-05-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nakayama, Keisuke" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Yamashita, Atsushi" /><meta name="citation_author_institution" content="Kitasato Institute for Life Science, Kitasato University, 5-9-1 Shirogane, Minato-ku Tokyo 108-8641, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Morimoto, Takuya" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Ogawa, Michihiro" /><meta name="citation_author_institution" content="Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Fukuhara, Masahiro" /><meta name="citation_author_institution" content="Laboratory of Microbiology, Faculty of Pharmacy, Niigata University of Pharmacy and Applied Life Sciences, 265-1 Higashijima, Akiha-ku, Niigata 956-8603, Japan" /><meta name="citation_author" content="Urakami, Hiroshi" /><meta name="citation_author_institution" content="Laboratory of Food Microbiology and Food Safety, Faculty of Applied Life Sciences, Niigata University of Pharmacy and Applied Life Sciences, 265-1 Higashijima, Akiha-ku, Niigata 956-8603, Japan" /><meta name="citation_author" content="Ohnishi, Makoto" /><meta name="citation_author_institution" content="Department of Bacteriology, National Institute for Infectious Diseases, Toyama 1-23-1, Shinjuku-ku, Tokyo 162-8640, Japan" /><meta name="citation_author" content="Uchiyama, Ikuo" /><meta name="citation_author_institution" content="National Institute for Basic Biology, National Institutes of Natural Sciences, Nishigonaka 38, Myodaiji, Okazaki 444-8585, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Ooka, Tadasuke" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Oshima, Kenshiro" /><meta name="citation_author_institution" content="Kitasato Institute for Life Science, Kitasato University, 5-9-1 Shirogane, Minato-ku Tokyo 108-8641, Japan" /><meta name="citation_author_institution" content="Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kasiwanoha, Kashiwa, Chiba, 277-8561, Japan" /><meta name="citation_author" content="Tamura, Akira" /><meta name="citation_author_institution" content="Niigata University of Pharmacy and Applied Life Sciences, 265-1 Higashijima, Niigata 956-8603, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kasiwanoha, Kashiwa, Chiba, 277-8561, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_title" content="The Whole-genome Sequencing of the Obligate Intracellular Bacterium Orientia tsutsugamushi Revealed Massive Gene Amplification During Reductive Genome Evolution" /><meta name="citation_firstpage" content="185" /><meta name="citation_lastpage" content="199" /><meta name="citation_doi" content="10.1093/dnares/dsn011" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2008/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Classification of Rickettsia tsutsugamushi in a new genus, Orientia gen. nov., as Orientia tsutsugamushi comb. nov; Int. J. Syst. Bacteriol.;  citation_year=1995;  citation_volume=45;  citation_pages=589-591; " /><meta name="citation_reference" content="citation_title=Vertical transmission in chigger borne rickettsiosis; Indian J. Med. Res.;  citation_year=1978;  citation_volume=68;  citation_pages=31-38; " /><meta name="citation_reference" content="citation_title=Transovarial transmission of Rickettsia tsutsugamushi in Leptotrombidium (Leptotrombidium) arenicola Traub (Acarina: Trombiculidae); J. Med. Entomol.;  citation_year=1972;  citation_volume=9;  citation_pages=71-72; " /><meta name="citation_reference" content="citation_title=Transovarial development of scrub typhus rickettsiae in a colony of vector mites; Trans. R. Soc. Trop. Med. Hyg.;  citation_year=1969;  citation_volume=63;  citation_pages=251-258; " /><meta name="citation_reference" content="citation_title=Transovarial transmission of Rickettsia tsutsugamushi in Leptotrombidium pallidum successively reared in the laboratory; Jpn. J. Exp. Med.;  citation_year=1988;  citation_volume=58;  citation_pages=213-218; " /><meta name="citation_reference" content="citation_title=An ultrastructural study of vertical transmission of Rickettsia tsutsugamushi during oogenesis and spermatogenesis in Leptortombidium pallidum; Am. J. Trop. Med. Hyg.;  citation_year=1994;  citation_volume=50;  citation_pages=219-228; " /><meta name="citation_reference" content="citation_title=Failed vertical transmission of Rickettsia tsutsugamushi (Rickettsiales: Rickettsiaceae) acquired from rickettsemic mice by Leptotrombidium pallidum (Acari: trombiculidae); J. Med. Entomol.;  citation_year=1994;  citation_volume=31;  citation_pages=212-216; " /><meta name="citation_reference" content="citation_title=Attempts to infect and demonstrate transovarial transmission of R. tsutsugamushi in three species of Leptotrombidium mites; Ann. N Y. Acad. Sci.;  citation_year=1975;  citation_volume=266;  citation_pages=80-90; " /><meta name="citation_reference" content="citation_title=The acquisition of Rickettsia tsutsugamushi by chiggers (trombiculid mites) during the feeding process; Ann. N Y. Acad. Sci.;  citation_year=1975;  citation_volume=266;  citation_pages=91-114; " /><meta name="citation_reference" content="citation_title=Scrub typhus and tropical rickettsioses; Curr. Opin. Infect. Dis.;  citation_year=2003;  citation_volume=16;  citation_pages=429-436; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of Orientia tsutsugamushi strains based on the sequence homologies of 56-kDa type-specific antigen genes; FEMS Microbiol. Lett.;  citation_year=1999;  citation_volume=15;  citation_pages=163-169; " /><meta name="citation_reference" content="citation_title=Phylogenetic characterization of Orientia tsutsugamushi isolated in Taiwan according to the sequence homologies of 56-kDa type-specific antigen genes; Microbiol. Immunol.;  citation_year=2003;  citation_volume=47;  citation_pages=577-583; " /><meta name="citation_reference" content="citation_title=Epidemiological survey of Orientia tsutsugamushi distribution in field rodents in Saitama Prefecture, Japan, and discovery of a new type; Microbiol. Immunol.;  citation_year=2001;  citation_volume=45;  citation_pages=439-446; " /><meta name="citation_reference" content="citation_title=The Orientia tsutsugamushi genome reveals massive proliferation of conjugative type IV secretion system and host–cell interaction genes; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=8;  citation_pages=7981-7986; " /><meta name="citation_reference" content="citation_title=Demonstration of antigenic and genotypic variation in Orientia tsutsugamushi which were isolated in Japan, and their classification into type and subtype; Microbiol. Immunol.;  citation_year=1996;  citation_volume=40;  citation_pages=627-638; " /><meta name="citation_reference" content="citation_title=Epidemiology of Tsutsugamushi disease in relation to the serotypes of Rickettsia tsutsugamushi isolated from patients, field mice, and unfed chiggers on the eastern slope of Mount Fuji, Shizuoka Prefecture, Japan; J. Clin. Microbiol.;  citation_year=1992;  citation_volume=30;  citation_pages=2842-2846; " /><meta name="citation_reference" content="citation_title=Epidemiological survey by means of complement fixation test on scrub typhus in Japan; Acta. Med. Biol (Niigata).;  citation_year=1967;  citation_volume=15;  citation_pages=69-85; " /><meta name="citation_reference" content="citation_title=Strain variation of rickettsia orientalis in the complement fixation test; Jpn. J. Med. Sci. Biol.;  citation_year=1964;  citation_volume=17;  citation_pages=59-72; " /><meta name="citation_reference" content="citation_title=Electron microscopic studies on intracellular multiplication of Rickettsia tsutsugamushi in L cells; Microbiol. Immunol.;  citation_year=1984;  citation_volume=28;  citation_pages=1191-1201; " /><meta name="citation_reference" content="citation_title=Purification of Rickettsia tsutsugamushi by Percoll density gradient centrifugation; Microbiol. Immunol.;  citation_year=1982;  citation_volume=26;  citation_pages=321-328; " /><meta name="citation_reference" content="citation_title=DNA base composition of Rickettsia tsutsugamushi determined by reversed-phase high-performance liquid chromatography; Int. J. Syst. Bacteriol.;  citation_year=1991;  citation_volume=41;  citation_pages=247-248; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=186-194; " /><meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=195-202; " /><meta name="citation_reference" content="citation_title=Chromosome of the enterohemorrhagic Escherichia coli O157:H7; comparative analysis with K-12 MG1655 revealed the acquisition of a large amount of foreign DNAs; DNA Res.;  citation_year=1999;  citation_volume=31;  citation_pages=361-368; " /><meta name="citation_reference" content="citation_title=Genomic diversity of enterohemorrhagic Escherichia coli O157 revealed by whole genome PCR scanning; Proc. Natl Acad. Sci. USA.;  citation_year=2002;  citation_volume=99;  citation_pages=17043-17048; " /><meta name="citation_reference" content="citation_title=An automated system for genome analysis to support microbial whole-genome shotgun sequencing; Biosci. Biotechnol. Biochem.;  citation_year=2000;  citation_volume=64;  citation_pages=670-673; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=MBGD: a platform for microbial comparative genomics based on the automated construction of orthologous groups; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_issue=Database issue;  citation_pages=D343-D346; " /><meta name="citation_reference" content="citation_title=Flexible sequence similarity searching with the FASTA3 program package; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=185-219; " /><meta name="citation_reference" content="citation_title=Improved tools for biological sequence comparison; Proc. Natl Acad. Sci. USA.;  citation_year=1988;  citation_volume=85;  citation_pages=2444-2448; " /><meta name="citation_reference" content="citation_title=A database for post-genome analysis; Trends Genet.;  citation_year=1997;  citation_volume=13;  citation_pages=375-376; " /><meta name="citation_reference" content="citation_title=Analyzing genomes with cumulative skew diagrams; Nucleic Acids Res.;  citation_year=1998;  citation_volume=15;  citation_pages=2286-2290; " /><meta name="citation_reference" content="citation_title=Genome sequence of Rickettsia bellii illuminates the role of Amoebae in gene exchanges between intracellular pathogens; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=733-744; " /><meta name="citation_reference" content="citation_title=The genome sequence of Rickettsia felis identifies the first putative conjugative plasmid in an obligate intracellular parasite; PLoS Biol.;  citation_year=2005;  citation_volume=3;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Mechanisms of evolution in Rickettsia conorii and R. prowazekii; Science;  citation_year=2001;  citation_volume=293;  citation_pages=2093-2098; " /><meta name="citation_reference" content="citation_title=The genome sequence of Rickettsia prowazekii and the origin of mitochondria; Nature;  citation_year=1998;  citation_volume=396;  citation_pages=133-140; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Rickettsia typhi and comparison with sequences of other rickettsiae; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=5842-5855; " /><meta name="citation_reference" content="citation_title=Phylogenomics of the reproductive parasite Wolbachia pipientis wMel: a streamlined genome overrun by mobile genetic elements; PLoS Biol.;  citation_year=2004;  citation_volume=2;  citation_pages=327-341; " /><meta name="citation_reference" content="citation_title=Shaping bacterial genomes with integrative and conjugative elements; Res. Microbiol.;  citation_year=2004;  citation_volume=155;  citation_pages=376-386; " /><meta name="citation_reference" content="citation_title=Lateral gene transfer between obligate intracellular bacteria: evidence from the Rickettsia massiliae genome; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1657-1664; " /><meta name="citation_reference" content="citation_title=Entry exclusion in F-like plasmids requires intact TraG in the donor that recognizes its cognate TraS in the recipient; Microbiology;  citation_year=2007;  citation_volume=153;  citation_pages=442-451; " /><meta name="citation_reference" content="citation_title=TraT lipoprotein, a plasmid-specified mediator of interactions between gram-negative bacteria and their environment; Microbiol. Rev.;  citation_year=1990;  citation_volume=54;  citation_pages=331-341; " /><meta name="citation_reference" content="citation_title=Illuminating the evolutionary history of chlamydiae; Science;  citation_year=2004;  citation_volume=304;  citation_pages=728-730; " /><meta name="citation_reference" content="citation_title=Genome dynamics and diversity of Shigella species, the etiologic agents of bacillary dysentery; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=6445-6458; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Francisella tularensis, the causative agent of tularemia; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=153-159; " /><meta name="citation_reference" content="citation_title=Plant transposable elements: where genetics meets genomics; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=329-341; " /><meta name="citation_reference" content="citation_title=Complete genome structure of the thermophilic cyanobacterium Thermosynechococcus elongatus BP-1; DNA Res.;  citation_year=2002;  citation_volume=31;  citation_pages=123-130; " /><meta name="citation_reference" content="citation_title=Deficiency of peptidoglycan and lipopolysaccharide components in Rickettsia tsutsugamushi; Infect. Immun.;  citation_year=1987;  citation_volume=55;  citation_pages=2290-2292; " /><meta name="citation_reference" content="citation_title=Comparative ultrastructural study on the cell envelopes of Rickettsia prowazekii, Rickettsia rickettsii, and Rickettsia tsutsugamushi; Infect. Immun.;  citation_year=1978;  citation_volume=21;  citation_pages=1020-1023; " /><meta name="citation_reference" content="citation_title=Two nucleotide transport proteins in Chlamydia trachomatis, One for net nucleoside triphosphate uptake and the other for transport of energy; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=1196-1202; " /><meta name="citation_reference" content="citation_title=A candidate NAD+ transporter in an intracellular bacterial symbiont related to Chlamydiae; Nature;  citation_year=2004;  citation_volume=432;  citation_pages=622-625; " /><meta name="citation_reference" content="citation_title=Ankyrin repeat: a unique motif mediating protein–protein interactions; Biochemistry;  citation_year=2006;  citation_volume=45;  citation_pages=15168-15178; " /><meta name="citation_reference" content="citation_title=MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment; Brief. Bioinform.;  citation_year=2004;  citation_volume=5;  citation_pages=150-163; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/4/185/1068831/dsn011.pdf" /><meta name="description" content="Abstract.  Scrub typhus (‘Tsutsugamushi’ disease in Japanese) is a mite-borne infectious disease. The causative agent is Orientia tsutsugamushi, an obligate int" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/4/185/376012" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/4/185/376012" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/4/185/376012");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn011';
    </script>
<title>
	Whole-genome Sequencing of the Obligate Intracellular Bacterium Orientia tsutsugamushi Revealed Massive Gene Amplification During Reductive Genome Evolution | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./376012" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaePATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXlhbWEsIEtlaXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBNaWNyb2Jpb2xvZ3ksIERlcGFydG1lbnQgb2YgSW5mZWN0aW91cyBEaXNlYXNlcywgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgNTIwMCBLaXlvdGFrZSwgTWl5YXpha2kgODk5LTE2OTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYXNoaXRhLCBBdHN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2UsIEtpdGFzYXRvIFVuaXZlcnNpdHksIDUtOS0xIFNoaXJvZ2FuZSwgTWluYXRvLWt1IFRva3lvIDEwOC04NjQxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1cm9rYXdhLCBLZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIENvbXBhcmF0aXZlIEdlbm9taWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLWNobywgSWtvbWEsIE5hcmEgNjMwLTAxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaW1vdG8sIFRha3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ2F3YSwgTWljaGloaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBDb21wYXJhdGl2ZSBHZW5vbWljcywgR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYS1jaG8sIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3VoYXJhLCBNYXNhaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTWljcm9iaW9sb2d5LCBGYWN1bHR5IG9mIFBoYXJtYWN5LCBOaWlnYXRhIFVuaXZlcnNpdHkgb2YgUGhhcm1hY3kgYW5kIEFwcGxpZWQgTGlmZSBTY2llbmNlcywgMjY1LTEgSGlnYXNoaWppbWEsIEFraWhhLWt1LCBOaWlnYXRhIDk1Ni04NjAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVyYWthbWksIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEZvb2QgTWljcm9iaW9sb2d5IGFuZCBGb29kIFNhZmV0eSwgRmFjdWx0eSBvZiBBcHBsaWVkIExpZmUgU2NpZW5jZXMsIE5paWdhdGEgVW5pdmVyc2l0eSBvZiBQaGFybWFjeSBhbmQgQXBwbGllZCBMaWZlIFNjaWVuY2VzLCAyNjUtMSBIaWdhc2hpamltYSwgQWtpaGEta3UsIE5paWdhdGEgOTU2LTg2MDMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2huaXNoaSwgTWFrb3RvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCYWN0ZXJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgSW5mZWN0aW91cyBEaXNlYXNlcywgVG95YW1hIDEtMjMtMSwgU2hpbmp1a3Uta3UsIFRva3lvIDE2Mi04NjQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVjaGl5YW1hLCBJa3VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBCYXNpYyBCaW9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGVzIG9mIE5hdHVyYWwgU2NpZW5jZXMsIE5pc2hpZ29uYWthIDM4LCBNeW9kYWlqaSwgT2themFraSA0NDQtODU4NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ3VyYSwgWW9zaGl0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPb2thLCBUYWRhc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIEtlbnNoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2UsIEtpdGFzYXRvIFVuaXZlcnNpdHksIDUtOS0xIFNoaXJvZ2FuZSwgTWluYXRvLWt1IFRva3lvIDEwOC04NjQxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01IEthc2l3YW5vaGEsIEthc2hpd2EsIENoaWJhLCAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYW11cmEsIEFraXJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmlpZ2F0YSBVbml2ZXJzaXR5IG9mIFBoYXJtYWN5IGFuZCBBcHBsaWVkIExpZmUgU2NpZW5jZXMsIDI2NS0xIEhpZ2FzaGlqaW1hLCBOaWlnYXRhIDk1Ni04NjAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhdHRvcmksIE1hc2FoaXJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEsIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhheWFzaGksIFRldHN1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBNaWNyb2Jpb2xvZ3ksIERlcGFydG1lbnQgb2YgSW5mZWN0aW91cyBEaXNlYXNlcywgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgNTIwMCBLaXlvdGFrZSwgTWl5YXpha2kgODk5LTE2OTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnJvbnRpZXIgU2NpZW5jZSBSZXNlYXJjaCBDZW50ZXIsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDUyMDAgS2l5b3Rha2UsIE1peWF6YWtpIDg5OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIFdob2xlLWdlbm9tZSBTZXF1ZW5jaW5nIG9mIHRoZSBPYmxpZ2F0ZSBJbnRyYWNlbGx1bGFyIEJhY3Rlcml1bSBPcmllbnRpYSB0c3V0c3VnYW11c2hpIFJldmVhbGVkIE1hc3NpdmUgR2VuZSBBbXBsaWZpY2F0aW9uIER1cmluZyBSZWR1Y3RpdmUgR2Vub21lIEV2b2x1dGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbGFzc2lmaWNhdGlvbiBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgaW4gYSBuZXcgZ2VudXMsIE9yaWVudGlhIGdlbi4gbm92LiwgYXMgT3JpZW50aWEgdHN1dHN1Z2FtdXNoaSBjb21iLiBub3Y7IEludC4gSi4gU3lzdC4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTU4OS01OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlcnRpY2FsIHRyYW5zbWlzc2lvbiBpbiBjaGlnZ2VyIGJvcm5lIHJpY2tldHRzaW9zaXM7IEluZGlhbiBKLiBNZWQuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0zMS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNvdmFyaWFsIHRyYW5zbWlzc2lvbiBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgaW4gTGVwdG90cm9tYmlkaXVtIChMZXB0b3Ryb21iaWRpdW0pIGFyZW5pY29sYSBUcmF1YiAoQWNhcmluYTogVHJvbWJpY3VsaWRhZSk7IEouIE1lZC4gRW50b21vbC47ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTcxLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc292YXJpYWwgZGV2ZWxvcG1lbnQgb2Ygc2NydWIgdHlwaHVzIHJpY2tldHRzaWFlIGluIGEgY29sb255IG9mIHZlY3RvciBtaXRlczsgVHJhbnMuIFIuIFNvYy4gVHJvcC4gTWVkLiBIeWcuOyAgY2l0YXRpb25feWVhcj0xOTY5OyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MjUxLTI1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNvdmFyaWFsIHRyYW5zbWlzc2lvbiBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgaW4gTGVwdG90cm9tYmlkaXVtIHBhbGxpZHVtIHN1Y2Nlc3NpdmVseSByZWFyZWQgaW4gdGhlIGxhYm9yYXRvcnk7IEpwbi4gSi4gRXhwLiBNZWQuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MjEzLTIxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gdWx0cmFzdHJ1Y3R1cmFsIHN0dWR5IG9mIHZlcnRpY2FsIHRyYW5zbWlzc2lvbiBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgZHVyaW5nIG9vZ2VuZXNpcyBhbmQgc3Blcm1hdG9nZW5lc2lzIGluIExlcHRvcnRvbWJpZGl1bSBwYWxsaWR1bTsgQW0uIEouIFRyb3AuIE1lZC4gSHlnLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTIxOS0yMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhaWxlZCB2ZXJ0aWNhbCB0cmFuc21pc3Npb24gb2YgUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpIChSaWNrZXR0c2lhbGVzOiBSaWNrZXR0c2lhY2VhZSkgYWNxdWlyZWQgZnJvbSByaWNrZXR0c2VtaWMgbWljZSBieSBMZXB0b3Ryb21iaWRpdW0gcGFsbGlkdW0gKEFjYXJpOiB0cm9tYmljdWxpZGFlKTsgSi4gTWVkLiBFbnRvbW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTIxMi0yMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF0dGVtcHRzIHRvIGluZmVjdCBhbmQgZGVtb25zdHJhdGUgdHJhbnNvdmFyaWFsIHRyYW5zbWlzc2lvbiBvZiBSLiB0c3V0c3VnYW11c2hpIGluIHRocmVlIHNwZWNpZXMgb2YgTGVwdG90cm9tYmlkaXVtIG1pdGVzOyBBbm4uIE4gWS4gQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjY7ICBjaXRhdGlvbl9wYWdlcz04MC05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFjcXVpc2l0aW9uIG9mIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaSBieSBjaGlnZ2VycyAodHJvbWJpY3VsaWQgbWl0ZXMpIGR1cmluZyB0aGUgZmVlZGluZyBwcm9jZXNzOyBBbm4uIE4gWS4gQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjY7ICBjaXRhdGlvbl9wYWdlcz05MS0xMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjcnViIHR5cGh1cyBhbmQgdHJvcGljYWwgcmlja2V0dHNpb3NlczsgQ3Vyci4gT3Bpbi4gSW5mZWN0LiBEaXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NDI5LTQzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGFuYWx5c2lzIG9mIE9yaWVudGlhIHRzdXRzdWdhbXVzaGkgc3RyYWlucyBiYXNlZCBvbiB0aGUgc2VxdWVuY2UgaG9tb2xvZ2llcyBvZiA1Ni1rRGEgdHlwZS1zcGVjaWZpYyBhbnRpZ2VuIGdlbmVzOyBGRU1TIE1pY3JvYmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNjMtMTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgY2hhcmFjdGVyaXphdGlvbiBvZiBPcmllbnRpYSB0c3V0c3VnYW11c2hpIGlzb2xhdGVkIGluIFRhaXdhbiBhY2NvcmRpbmcgdG8gdGhlIHNlcXVlbmNlIGhvbW9sb2dpZXMgb2YgNTYta0RhIHR5cGUtc3BlY2lmaWMgYW50aWdlbiBnZW5lczsgTWljcm9iaW9sLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTU3Ny01ODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWRlbWlvbG9naWNhbCBzdXJ2ZXkgb2YgT3JpZW50aWEgdHN1dHN1Z2FtdXNoaSBkaXN0cmlidXRpb24gaW4gZmllbGQgcm9kZW50cyBpbiBTYWl0YW1hIFByZWZlY3R1cmUsIEphcGFuLCBhbmQgZGlzY292ZXJ5IG9mIGEgbmV3IHR5cGU7IE1pY3JvYmlvbC4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz00MzktNDQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgT3JpZW50aWEgdHN1dHN1Z2FtdXNoaSBnZW5vbWUgcmV2ZWFscyBtYXNzaXZlIHByb2xpZmVyYXRpb24gb2YgY29uanVnYXRpdmUgdHlwZSBJViBzZWNyZXRpb24gc3lzdGVtIGFuZCBob3N04oCTY2VsbCBpbnRlcmFjdGlvbiBnZW5lczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Nzk4MS03OTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZW1vbnN0cmF0aW9uIG9mIGFudGlnZW5pYyBhbmQgZ2Vub3R5cGljIHZhcmlhdGlvbiBpbiBPcmllbnRpYSB0c3V0c3VnYW11c2hpIHdoaWNoIHdlcmUgaXNvbGF0ZWQgaW4gSmFwYW4sIGFuZCB0aGVpciBjbGFzc2lmaWNhdGlvbiBpbnRvIHR5cGUgYW5kIHN1YnR5cGU7IE1pY3JvYmlvbC4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz02MjctNjM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlkZW1pb2xvZ3kgb2YgVHN1dHN1Z2FtdXNoaSBkaXNlYXNlIGluIHJlbGF0aW9uIHRvIHRoZSBzZXJvdHlwZXMgb2YgUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpIGlzb2xhdGVkIGZyb20gcGF0aWVudHMsIGZpZWxkIG1pY2UsIGFuZCB1bmZlZCBjaGlnZ2VycyBvbiB0aGUgZWFzdGVybiBzbG9wZSBvZiBNb3VudCBGdWppLCBTaGl6dW9rYSBQcmVmZWN0dXJlLCBKYXBhbjsgSi4gQ2xpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI4NDItMjg0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZGVtaW9sb2dpY2FsIHN1cnZleSBieSBtZWFucyBvZiBjb21wbGVtZW50IGZpeGF0aW9uIHRlc3Qgb24gc2NydWIgdHlwaHVzIGluIEphcGFuOyBBY3RhLiBNZWQuIEJpb2wgKE5paWdhdGEpLjsgIGNpdGF0aW9uX3llYXI9MTk2NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTY5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhaW4gdmFyaWF0aW9uIG9mIHJpY2tldHRzaWEgb3JpZW50YWxpcyBpbiB0aGUgY29tcGxlbWVudCBmaXhhdGlvbiB0ZXN0OyBKcG4uIEouIE1lZC4gU2NpLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk2NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTU5LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbGVjdHJvbiBtaWNyb3Njb3BpYyBzdHVkaWVzIG9uIGludHJhY2VsbHVsYXIgbXVsdGlwbGljYXRpb24gb2YgUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpIGluIEwgY2VsbHM7IE1pY3JvYmlvbC4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xMTkxLTEyMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1cmlmaWNhdGlvbiBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgYnkgUGVyY29sbCBkZW5zaXR5IGdyYWRpZW50IGNlbnRyaWZ1Z2F0aW9uOyBNaWNyb2Jpb2wuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzIxLTMyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGJhc2UgY29tcG9zaXRpb24gb2YgUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpIGRldGVybWluZWQgYnkgcmV2ZXJzZWQtcGhhc2UgaGlnaC1wZXJmb3JtYW5jZSBsaXF1aWQgY2hyb21hdG9ncmFwaHk7IEludC4gSi4gU3lzdC4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTI0Ny0yNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UtY2FsbGluZyBvZiBhdXRvbWF0ZWQgc2VxdWVuY2VyIHRyYWNlcyB1c2luZyBwaHJlZC4gSUkuIEVycm9yIHByb2JhYmlsaXRpZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xODYtMTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZWQ6IGEgZ3JhcGhpY2FsIHRvb2wgZm9yIHNlcXVlbmNlIGZpbmlzaGluZzsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWUgb2YgdGhlIGVudGVyb2hlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgTzE1NzpINzsgY29tcGFyYXRpdmUgYW5hbHlzaXMgd2l0aCBLLTEyIE1HMTY1NSByZXZlYWxlZCB0aGUgYWNxdWlzaXRpb24gb2YgYSBsYXJnZSBhbW91bnQgb2YgZm9yZWlnbiBETkFzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM2MS0zNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgZGl2ZXJzaXR5IG9mIGVudGVyb2hlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgTzE1NyByZXZlYWxlZCBieSB3aG9sZSBnZW5vbWUgUENSIHNjYW5uaW5nOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTE3MDQzLTE3MDQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhdXRvbWF0ZWQgc3lzdGVtIGZvciBnZW5vbWUgYW5hbHlzaXMgdG8gc3VwcG9ydCBtaWNyb2JpYWwgd2hvbGUtZ2Vub21lIHNob3RndW4gc2VxdWVuY2luZzsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTY3MC02NzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1CR0Q6IGEgcGxhdGZvcm0gZm9yIG1pY3JvYmlhbCBjb21wYXJhdGl2ZSBnZW5vbWljcyBiYXNlZCBvbiB0aGUgYXV0b21hdGVkIGNvbnN0cnVjdGlvbiBvZiBvcnRob2xvZ291cyBncm91cHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX2lzc3VlPURhdGFiYXNlIGlzc3VlOyAgY2l0YXRpb25fcGFnZXM9RDM0My1EMzQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbGV4aWJsZSBzZXF1ZW5jZSBzaW1pbGFyaXR5IHNlYXJjaGluZyB3aXRoIHRoZSBGQVNUQTMgcHJvZ3JhbSBwYWNrYWdlOyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MTg1LTIxOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgdG9vbHMgZm9yIGJpb2xvZ2ljYWwgc2VxdWVuY2UgY29tcGFyaXNvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQS47ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNDQ0LTI0NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGF0YWJhc2UgZm9yIHBvc3QtZ2Vub21lIGFuYWx5c2lzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9Mzc1LTM3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHl6aW5nIGdlbm9tZXMgd2l0aCBjdW11bGF0aXZlIHNrZXcgZGlhZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIyODYtMjI5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIFJpY2tldHRzaWEgYmVsbGlpIGlsbHVtaW5hdGVzIHRoZSByb2xlIG9mIEFtb2ViYWUgaW4gZ2VuZSBleGNoYW5nZXMgYmV0d2VlbiBpbnRyYWNlbGx1bGFyIHBhdGhvZ2VuczsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTczMy03NDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgUmlja2V0dHNpYSBmZWxpcyBpZGVudGlmaWVzIHRoZSBmaXJzdCBwdXRhdGl2ZSBjb25qdWdhdGl2ZSBwbGFzbWlkIGluIGFuIG9ibGlnYXRlIGludHJhY2VsbHVsYXIgcGFyYXNpdGU7IFBMb1MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgb2YgZXZvbHV0aW9uIGluIFJpY2tldHRzaWEgY29ub3JpaSBhbmQgUi4gcHJvd2F6ZWtpaTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTM7ICBjaXRhdGlvbl9wYWdlcz0yMDkzLTIwOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgUmlja2V0dHNpYSBwcm93YXpla2lpIGFuZCB0aGUgb3JpZ2luIG9mIG1pdG9jaG9uZHJpYTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM5NjsgIGNpdGF0aW9uX3BhZ2VzPTEzMy0xNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBSaWNrZXR0c2lhIHR5cGhpIGFuZCBjb21wYXJpc29uIHdpdGggc2VxdWVuY2VzIG9mIG90aGVyIHJpY2tldHRzaWFlOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTU4NDItNTg1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5vbWljcyBvZiB0aGUgcmVwcm9kdWN0aXZlIHBhcmFzaXRlIFdvbGJhY2hpYSBwaXBpZW50aXMgd01lbDogYSBzdHJlYW1saW5lZCBnZW5vbWUgb3ZlcnJ1biBieSBtb2JpbGUgZ2VuZXRpYyBlbGVtZW50czsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MzI3LTM0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hhcGluZyBiYWN0ZXJpYWwgZ2Vub21lcyB3aXRoIGludGVncmF0aXZlIGFuZCBjb25qdWdhdGl2ZSBlbGVtZW50czsgUmVzLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni0zODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhdGVyYWwgZ2VuZSB0cmFuc2ZlciBiZXR3ZWVuIG9ibGlnYXRlIGludHJhY2VsbHVsYXIgYmFjdGVyaWE6IGV2aWRlbmNlIGZyb20gdGhlIFJpY2tldHRzaWEgbWFzc2lsaWFlIGdlbm9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNjU3LTE2NjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVudHJ5IGV4Y2x1c2lvbiBpbiBGLWxpa2UgcGxhc21pZHMgcmVxdWlyZXMgaW50YWN0IFRyYUcgaW4gdGhlIGRvbm9yIHRoYXQgcmVjb2duaXplcyBpdHMgY29nbmF0ZSBUcmFTIGluIHRoZSByZWNpcGllbnQ7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTM7ICBjaXRhdGlvbl9wYWdlcz00NDItNDUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFUIGxpcG9wcm90ZWluLCBhIHBsYXNtaWQtc3BlY2lmaWVkIG1lZGlhdG9yIG9mIGludGVyYWN0aW9ucyBiZXR3ZWVuIGdyYW0tbmVnYXRpdmUgYmFjdGVyaWEgYW5kIHRoZWlyIGVudmlyb25tZW50OyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0zMzEtMzQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbGx1bWluYXRpbmcgdGhlIGV2b2x1dGlvbmFyeSBoaXN0b3J5IG9mIGNobGFteWRpYWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA0OyAgY2l0YXRpb25fcGFnZXM9NzI4LTczMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGR5bmFtaWNzIGFuZCBkaXZlcnNpdHkgb2YgU2hpZ2VsbGEgc3BlY2llcywgdGhlIGV0aW9sb2dpYyBhZ2VudHMgb2YgYmFjaWxsYXJ5IGR5c2VudGVyeTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NjQ0NS02NDU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIEZyYW5jaXNlbGxhIHR1bGFyZW5zaXMsIHRoZSBjYXVzYXRpdmUgYWdlbnQgb2YgdHVsYXJlbWlhOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTE1My0xNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHRyYW5zcG9zYWJsZSBlbGVtZW50czogd2hlcmUgZ2VuZXRpY3MgbWVldHMgZ2Vub21pY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMyOS0zNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzdHJ1Y3R1cmUgb2YgdGhlIHRoZXJtb3BoaWxpYyBjeWFub2JhY3Rlcml1bSBUaGVybW9zeW5lY2hvY29jY3VzIGVsb25nYXR1cyBCUC0xOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTEyMy0xMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmljaWVuY3kgb2YgcGVwdGlkb2dseWNhbiBhbmQgbGlwb3BvbHlzYWNjaGFyaWRlIGNvbXBvbmVudHMgaW4gUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpOyBJbmZlY3QuIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTIyOTAtMjI5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgdWx0cmFzdHJ1Y3R1cmFsIHN0dWR5IG9uIHRoZSBjZWxsIGVudmVsb3BlcyBvZiBSaWNrZXR0c2lhIHByb3dhemVraWksIFJpY2tldHRzaWEgcmlja2V0dHNpaSwgYW5kIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaTsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMDIwLTEwMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBudWNsZW90aWRlIHRyYW5zcG9ydCBwcm90ZWlucyBpbiBDaGxhbXlkaWEgdHJhY2hvbWF0aXMsIE9uZSBmb3IgbmV0IG51Y2xlb3NpZGUgdHJpcGhvc3BoYXRlIHVwdGFrZSBhbmQgdGhlIG90aGVyIGZvciB0cmFuc3BvcnQgb2YgZW5lcmd5OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE4MTsgIGNpdGF0aW9uX3BhZ2VzPTExOTYtMTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjYW5kaWRhdGUgTkFEKyB0cmFuc3BvcnRlciBpbiBhbiBpbnRyYWNlbGx1bGFyIGJhY3RlcmlhbCBzeW1iaW9udCByZWxhdGVkIHRvIENobGFteWRpYWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzI7ICBjaXRhdGlvbl9wYWdlcz02MjItNjI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bbmt5cmluIHJlcGVhdDogYSB1bmlxdWUgbW90aWYgbWVkaWF0aW5nIHByb3RlaW7igJNwcm90ZWluIGludGVyYWN0aW9uczsgQmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MTUxNjgtMTUxNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0EzOiBpbnRlZ3JhdGVkIHNvZnR3YXJlIGZvciBtb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIGFuZCBzZXF1ZW5jZSBhbGlnbm1lbnQ7IEJyaWVmLiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNTAtMTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvNC8xODUvMTA2ODgzMS9kc24wMTEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFNjcnViIHR5cGh1cyAo4oCYVHN1dHN1Z2FtdXNoaeKAmSBkaXNlYXNlIGluIEphcGFuZXNlKSBpcyBhIG1pdGUtYm9ybmUgaW5mZWN0aW91cyBkaXNlYXNlLiBUaGUgY2F1c2F0aXZlIGFnZW50IGlzIE9yaWVudGlhIHRzdXRzdWdhbXVzaGksIGFuIG9ibGlnYXRlIGludCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzc2MDEyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZL9xLjG/M3hWdHLW/KdRB05XnTpj" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f185%2f376012" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f185%2f376012" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14644" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&amp;Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/4">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5474240">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474240">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474242">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474242">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474251">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474251">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474276">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474276">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474395">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474395">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474398">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474398">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474400">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474400">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474402">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474402">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/4/173/376904">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/4/201/376260">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The Whole-genome Sequencing of the Obligate Intracellular Bacterium <em>Orientia tsutsugamushi</em> Revealed Massive Gene Amplification During Reductive Genome Evolution
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keisuke Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keisuke+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Keisuke Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kitasato Institute for Life Science, Kitasato University</div>, <div class="addr-line">5-9-1 Shirogane, Minato-ku Tokyo 108-8641</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamashita A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Atsushi Yamashita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Laboratory of Comparative Genomics</div>, <div class="insititution">Graduate School of Information Science, Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama-cho, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuya Morimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuya+Morimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morimoto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morimoto T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takuya Morimoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michihiro Ogawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Laboratory of Comparative Genomics</div>, <div class="insititution">Graduate School of Information Science, Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama-cho, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michihiro+Ogawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogawa M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Michihiro Ogawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahiro Fukuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Microbiology, Faculty of Pharmacy</div>, <div class="insititution">Niigata University of Pharmacy and Applied Life Sciences</div>, <div class="addr-line">265-1 Higashijima, Akiha-ku, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahiro+Fukuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuhara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuhara M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masahiro Fukuhara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Urakami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Laboratory of Food Microbiology and Food Safety, Faculty of Applied Life Sciences</div>, <div class="insititution">Niigata University of Pharmacy and Applied Life Sciences</div>, <div class="addr-line">265-1 Higashijima, Akiha-ku, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Urakami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Urakami H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Urakami H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroshi Urakami</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Ohnishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Bacteriology</div>, <div class="insititution">National Institute for Infectious Diseases, Toyama 1-23-1, Shinjuku-ku, Tokyo 162-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Ohnishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohnishi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohnishi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Makoto Ohnishi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ikuo Uchiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">National Institute for Basic Biology, National Institutes of Natural Sciences</div>, <div class="addr-line">Nishigonaka 38, Myodaiji, Okazaki 444-8585</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ikuo+Uchiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uchiyama I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Uchiyama I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ikuo Uchiyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">Frontier Science Research Center, University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadasuke Ooka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadasuke+Ooka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ooka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ooka T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tadasuke Ooka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kitasato Institute for Life Science, Kitasato University</div>, <div class="addr-line">5-9-1 Shirogane, Minato-ku Tokyo 108-8641</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">9</div><div class="insititution">Graduate School of Frontier Sciences, The University of Tokyo</div>, <div class="addr-line">5-1-5 Kasiwanoha, Kashiwa, Chiba, 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenshiro+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kenshiro Oshima</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akira Tamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">10</div><div class="insititution">Niigata University of Pharmacy and Applied Life Sciences</div>, <div class="addr-line">265-1 Higashijima, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akira+Tamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tamura A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tamura A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akira Tamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="insititution">Graduate School of Frontier Sciences, The University of Tokyo</div>, <div class="addr-line">5-1-5 Kasiwanoha, Kashiwa, Chiba, 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">8</div><div class="insititution">Frontier Science Research Center, University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. E-mail: <a href="mailto:thayash@med.miyazaki-u.ac.jp" target="_blank">thayash@med.miyazaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Biological Science Laboratories, Kao Corporation, 2606 Akabane Ichikai Haga, Tochigi 321-3497, Japan; Graduate School of Information Science, and Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma, Nara 630-0192, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Office of Biologics I, Pharmaceutical and Medical Devices Agency, 3-3-2 Kasumigaseki, Chiyoda-ku, Tokyo 100-0013, Japan</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 185–199, <a href='https://doi.org/10.1093/dnares/dsn011'>https://doi.org/10.1093/dnares/dsn011</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 May 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 December 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 May 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="376012" href="/dnaresearch/article-pdf/15/4/185/1068831/dsn011.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Keisuke Nakayama, Atsushi Yamashita, Ken Kurokawa, Takuya Morimoto, Michihiro Ogawa, Masahiro Fukuhara, Hiroshi Urakami, Makoto Ohnishi, Ikuo Uchiyama, Yoshitoshi Ogura, Tadasuke Ooka, Kenshiro Oshima, Akira Tamura, Masahira Hattori, Tetsuya Hayashi;  The Whole-genome Sequencing of the Obligate Intracellular Bacterium <em>Orientia tsutsugamushi</em> Revealed Massive Gene Amplification During Reductive Genome Evolution, <em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 185–199, <a href='https://doi.org/10.1093/dnares/dsn011'>https://doi.org/10.1093/dnares/dsn011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/376012?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/376012?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/376012?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/376012?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/376012?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/376012?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/376012?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20Whole-genome%20Sequencing%20of%20the%20Obligate%20Intracellular%20Bacterium%20%26lt%3Bem%26gt%3BOrientia%20tsutsugamushi%26lt%3B%2Fem%26gt%3B%20Revealed%20Massive%20Gene%20Amplification%20During%20Reductive%20Genome%20Evolution&amp;publicationDate=2008-05-28&amp;volumeNum=15&amp;issueNum=4&amp;author=Nakayama%2C%20Keisuke%3B%20Yamashita%2C%20Atsushi&amp;startPage=185&amp;endPage=199&amp;contentId=10.1093%2Fdnares%2Fdsn011&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f185%2f376012" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5474240 id="5474240" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Scrub typhus (‘<em>Tsutsugamushi</em>’ disease in Japanese) is a mite-borne infectious disease. The causative agent is <em>Orientia tsutsugamushi</em>, an obligate intracellular bacterium belonging to the family <em>Rickettsiaceae</em> of the subdivision alpha-Proteobacteria. In this study, we determined the complete genome sequence of <em>O. tsutsugamushi</em> strain Ikeda, which comprises a single chromosome of 2 008 987 bp and contains 1967 protein coding sequences (CDSs). The chromosome is much larger than those of other members of <em>Rickettsiaceae</em>, and 46.7% of the sequence was occupied by repetitive sequences derived from an integrative and conjugative element, 10 types of transposable elements, and seven types of short repeats of unknown origins. The massive amplification and degradation of these elements have generated a huge number of repeated genes (1196 CDSs, categorized into 85 families), many of which are pseudogenes (766 CDSs), and also induced intensive genome shuffling. By comparing the gene content with those of other family members of <em>Rickettsiacea</em>, we identified the core gene set of the family <em>Rickettsiaceae</em> and found that, while much more extensive gene loss has taken place among the housekeeping genes of <em>Orientia</em> than those of <em>Rickettsia</em>, <em>O. tsutsugamushi</em> has acquired a large number of foreign genes. The <em>O. tsutsugamushi</em> genome sequence is thus a prominent example of the high plasticity of bacterial genomes, and provides the genetic basis for a better understanding of the biology of <em>O. tsutsugamushi</em> and the pathogenesis of ‘<em>Tsutsugamushi</em>’ disease.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Orientia tsutsugamushi&quot;"><em>Orientia tsutsugamushi</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;obligate intracellular bacterium&quot;">obligate intracellular bacterium</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;repetitive sequence&quot;">repetitive sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;IS element&quot;">IS element</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;integrative and conjugative element&quot;">integrative and conjugative element</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene amplification&quot;">gene amplification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome reduction&quot;">genome reduction</a></div>                            <h2 scrollto-destination=5474242 id="5474242" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Orientia tsutsugamushi</em>, a causative agent of scrub typhus or ‘<em>Tsutsugamushi</em>’ disease, is a Gram-negative bacterium belonging to the order <em>Rickettsiales</em> from the alpha-subdivision of Proteobacteria. The bacterium had been classified into the genus <em>Rickettsia</em>, but has recently been transferred to the genus <em>Orientia</em>, a genus newly created in the family <em>Rickettsiaceae</em>, based on the differences in the 16S ribosomal RNA (rRNA) sequence (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>) and in several morphological and biochemical features.<sup><a reveal-id="DSN011C1" data-open="DSN011C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p><p>Like other members of the order <em>Rickettsiales</em>, <em>O. tsutsugamushi</em> is an obligate intracellular parasite. In nature, the bacterium lives in trombiculid mites, which are called ‘<em>tsutsugamushi</em>’ in Japanese. In infected mites, the bacterium is found in the host cell cytosol of various organs including the oocytes, and is efficiently passed on to the offspring via transovarial transmission.<sup><a reveal-id="DSN011C2  DSN011C6" data-open="DSN011C2  DSN011C6" class="link link-ref link-reveal xref-bibr">2–6</a></sup> Trombiculid mites suck the tissue fluid of mammals only once in their life cycle at the larval stage, when the bacterium is transferred to the animal to induce scrub typhus. Reverse transfer, from infected animals to mites, occurs inefficiently,<sup><a reveal-id="DSN011C7" data-open="DSN011C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSN011C8" data-open="DSN011C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and the bacterium transmitted in such a way is rarely passed on to the offspring.<sup><a reveal-id="DSN011C7  DSN011C9" data-open="DSN011C7  DSN011C9" class="link link-ref link-reveal xref-bibr">7–9</a></sup> Accordingly, limited numbers of mite lines retain <em>O. tsutsugamushi</em>. However, <em>O. tsutsugamushi</em> or mites carrying this bacterium are widely distributed in Asia and the Western Pacific islands; one billion people are estimated to be at risk for scrub typhus, and one million cases occur annually.<sup><a reveal-id="DSN011C10" data-open="DSN011C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>In infected animals and humans, <em>O. tsutsugamushi</em> invades the macrophages and vascular endothelial cells, escapes from the phagosomes, and propagates in the cytosol like other <em>Rickettsiaceae</em> family members, inducing an acute febrile state. It often causes a deadly illness if not treated with appropriate antibiotics, but the molecular pathogenicity is still poorly understood and no vaccine is currently available.</p><p><em>Orientia tsutsugamushi</em> strains are serologically classified into several subgroups based on the antigenic variations of a major surface protein called ‘56-kDa protein’.<sup><a reveal-id="DSN011C11" data-open="DSN011C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The results of our pulsed-field gel electrophoresis (PFGE) analysis indicate that the genome size of <em>O. tsutsugamushi</em> is much larger than those of other members of the order <em>Rickettsiales</em>, and that the genome exhibits a significant strain-to-strain size variation, ranging from 2.0 to 2.7 Mb (Tamura <em>et al.</em>, unpublished data). However, no known techniques for genetic manipulation are applicable to this bacterium, and only a few pieces of genetic information have so far been accumulated; only the DNA sequences of the genes for the 56-kDa protein from various serotypes have been intensively analyzed.<sup><a reveal-id="DSN011C12" data-open="DSN011C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN011C13" data-open="DSN011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In order to better understand the genetic features of this bacterium, we determined the complete genome sequence of <em>O. tsutsugamushi</em> strain Ikeda. Here, we present the genomic features of the microorganism revealed by genome sequence analysis of the strain Ikeda and genomic comparison with other sequenced members of the family <em>Rickettsiacea</em>. Some of our results obtained by genomic comparison of the strain Ikeda and other recently sequenced <em>O. tsutsugamushi</em> strain Boryong<sup><a reveal-id="DSN011C14" data-open="DSN011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> are also presented.</p>                            <h2 scrollto-destination=5474251 id="5474251" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5474254 id="5474254" class="section-title" data-legacy-id=s2a>2.1. Bacterial strain</h3>
<p><em>Orientia tsutsugamushi</em> strain Ikeda was originally isolated from a patient in 1979 in Niigata Prefecture, Japan. The strain belongs to the Japanese Gilliam serotype and is highly virulent in mice.<sup><a reveal-id="DSN011C15" data-open="DSN011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The mite vector of the strain was not identified, but Japanese Gilliam serotype strains are usually carried by <em>Leptotrombidium pallidum</em> or <em>Leptotrombidium scutellare</em>.<sup><a reveal-id="DSN011C12" data-open="DSN011C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN011C15  DSN011C18" data-open="DSN011C15  DSN011C18" class="link link-ref link-reveal xref-bibr">15–18</a></sup> The strain has been kept in the laboratory by repeated propagation on L929 cells and passages in mice since the initial isolation, which may have generated multiple sub-strains containing some genomic alterations. To avoid the usage of heterogeneous genomic DNA derived from such sub-strains for genome sequencing, we performed single plaque purification on L929 cells as previously described.<sup><a reveal-id="DSN011C19" data-open="DSN011C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The cloned strain was propagated in L929 cells, and bacterial cells were purified on a percoll gradient as described previously.<sup><a reveal-id="DSN011C20" data-open="DSN011C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                            <h3 scrollto-destination=5474258 id="5474258" class="section-title" data-legacy-id=s2b>2.2. DNA preparation and sequencing</h3>
<p>Bacterial DNA was obtained from the purified cells by standard procedures.<sup><a reveal-id="DSN011C21" data-open="DSN011C21" class="link link-ref link-reveal xref-bibr">21</a></sup> A single batch of DNA preparation was used for all the experiments described hereafter. The genomic DNA was randomly sheared by means of a Hydroshear (GeneMachines) and used for the construction of genomic libraries. We prepared two pUC18-based random genomic libraries with insert sizes of 1–1.5 or 4–5 kb. A BAC library with an insert size of 11–15 kb was also constructed using the pIndigoBAC-5 vector (Epicentre). Sequencing was carried out using the BigDye v3.1 chemistry on ABI370 or ABI3730 sequencers (ABI), or the ET chemistry on MegaBACE4500 sequencers (GE Healthcare).</p><p>Because the genome sequence of <em>O. tsutsugamushi</em> was extremely rich in repetitive sequences, we employed a sequencing strategy comprising the following four steps. We first produced 32 000 shotgun reads from the 1–1.5 kb library and 14 000 reads from the 4–5 kb library using the forward sequencing primer. The Phred/Phrap software package<sup><a reveal-id="DSN011C22" data-open="DSN011C22" class="link link-ref link-reveal xref-bibr">22</a></sup> was used for base-calling, quality assessment, and sequence assembly. The assemblies were visualized in order to count the base variations and detect misassembly using the Consed software.<sup><a reveal-id="DSN011C23" data-open="DSN011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Based on the assembled data from 46 000 shotgun reads, we selected 6700 informative clones from both libraries, which were predicted to be located at the contig ends. The reverse end-sequences of these clones were determined using the reverse primer, and were assembled with the 46 000 forward sequences. The redundancy of the random shotgun sequences we obtained up to this stage was ∼13. At the third stage, the end sequences of 2383 BAC clones were determined, and 142 clones which were predicted to bridge the contigs were selected. These clones were individually sequenced and used as scaffolds in the sequence assembly. After re-assembly, 26 additional BAC clones were selected, sequenced individually, and added to the assembly. This assembly yielded 15 major contigs ranging in length from 20.4 to 296.0 kb. Finally, in order to close the 15 gaps, genomic PCR was carried out for five gaps using an LA long PCR kit (Takara), and 14 BAC clones were retrieved to close the remaining 10 gaps. The sequences of the five PCR products (3.7–25.4 kb) and the 14 BAC clones were individually determined. By adding these sequences to the assembly, we obtained the complete genome sequence of <em>O. tsutsugamushi</em> strain Ikeda.</p>                            <h3 scrollto-destination=5474265 id="5474265" class="section-title" data-legacy-id=s2c>2. 3. Validation of the sequence assembly</h3>
<p>To validate the sequence assembly, we digested the chromosomal DNA of strain Ikeda with <em>Sma</em>I or <em>Eag</em>I, or double-digested it with these two enzymes, and analyzed the digested DNA by PFGE as described previously.<sup><a reveal-id="DSN011C24" data-open="DSN011C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The digestion patterns we obtained were in good agreement with those deduced from the assembled sequence (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). In addition, we designed 18 pairs of PCR primers that amplified the 18-genomic segments that covered a 69-kb region of the chromosome (nucleotide positions: 611 095–670 870 bp) and performed a PCR scanning analysis.<sup><a reveal-id="DSN011C25" data-open="DSN011C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In this analysis, all primer pairs yielded PCR products of expected sizes (data not shown).</p>                            <h3 scrollto-destination=5474269 id="5474269" class="section-title" data-legacy-id=s2d>2. 4. Gene identification, sequence annotation, and data analyses</h3>
<p>The identification of potential protein coding sequences (CDSs) and their annotation was done by using GenomeGambler, version 1.5.<sup><a reveal-id="DSN011C26" data-open="DSN011C26" class="link link-ref link-reveal xref-bibr">26</a></sup> We first identified CDSs longer than 50 amino acids. Then, by searching all the intergenic regions for CDSs showing sequence similarities to known proteins with the BLASTP program,<sup><a reveal-id="DSN011C27" data-open="DSN011C27" class="link link-ref link-reveal xref-bibr">27</a></sup> we identified 69 CDSs shorter than 50 amino acids. Orthologous genes were identified using the MBGD (Microbial Genome Database for Comparative Analysis) database (<a class="link link-uri" href="http://mbgd.genome.ad.jp/">http://mbgd.genome.ad.jp/</a>).<sup><a reveal-id="DSN011C28" data-open="DSN011C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>Repetitive sequences were identified through sliding window analysis (window size, 100 bp; step size, 3 bp). All 100-bp sequence windows were searched against the entire genome sequence using the BLASTN program.<sup><a reveal-id="DSN011C27" data-open="DSN011C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The ssearch3 program<sup><a reveal-id="DSN011C29" data-open="DSN011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> was also used to define the borders of each repetitive sequence. The determination of the consensus sequences of each repetitive sequence and their classification were carried out by manual inspection based on the multiple alignments generated by the ClustalW program.<sup><a reveal-id="DSN011C30" data-open="DSN011C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Repeated gene families, which were defined in this study as the genes whose gene products exhibited at least 90% amino acid sequence identity over 60% of the alignment length, were identified by all-to-all BLASTP analysis of the <em>O. tsutsugamushi</em> gene products. The KEGG (Kyoto Encyclopedia of Genes and Genomes) database (<a class="link link-uri" href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>)<sup><a reveal-id="DSN011C31" data-open="DSN011C31" class="link link-ref link-reveal xref-bibr">31</a></sup> was used to analyze the metabolic pathways of <em>O. tsutsugamushi</em>. The annotated genome sequence of <em>O. tsutsugamushi</em> strain Ikeda has been deposited at the DDBJ/EMBL/GenBank databanks under accession number AP008981.</p>                            <h2 scrollto-destination=5474276 id="5474276" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5474281 id="5474281" class="section-title" data-legacy-id=s3a>3. 1. General features of the genome</h3>
<p><em>Orientia tsutsugamushi</em> has a circular chromosome consisting of 2 008 987 bp (Table <a reveal-id="DSN011TB1" data-open="DSN011TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSN011F1" data-open="DSN011F1" class="link link-reveal link-table xref-fig">1</a>). No plasmids or prophages were detected. A putative origin of replication (<em>ori</em>) was assigned through cumulative GC skew analysis, because no clear skew in the leading and lagging strands was observed through conventional GC skew analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>).<sup><a reveal-id="DSN011C32" data-open="DSN011C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The overall features of the <em>O. tsutsugamushi</em> genome are summarized in Table <a reveal-id="DSN011TB1" data-open="DSN011TB1" class="link link-reveal link-table xref-fig">1</a>, and a comparison with selected members of the order <em>Rickettsiales</em> is given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                        <a id="5474290" scrollto-destination="5474290"></a>
<div id="DSN011F1" data-id="DSN011F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01101.jpeg?Expires=2147483647&Signature=voSF3zrRATBEnWT3jievncAh0~YWxmMgsKUrkKqHAJQlwockbQyj8ch4~TV8MFdjkGuvlxhGanmvX-JdCoU-jyobY5oCuI2XijcGG4onpOpteYD2CWHXGueSTl5g-Xrb-th~paqegZwjAo242f4O0-McNr4FNrUXxYMXGRVQFsPk~P5TQMJP9vZta6i87dxj7MSxbwB8iolYHUcGIn1NVUdhlKL6B2L8TV19QvS7~iwqX9ArSy42F4jlYNNHBQOZsGr8u8VfDFzatpiZNDp2N~Zd3Ug8GoDLzrQZyUnEAcmgrpoEtZ3HLbMaf881qCv0dcg3G2lpLUOhoFY6OHI7RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structural features of the O. tsutsugamushi genome. The putative origin of replication is located at the top. The outermost red rays show the repetition rates of each chromosomal region. The repetition rates were determined by sliding window analysis (window size, 100 bp; step size, 3 bp). The total number of 100-bp windows retrieved from the entire genome sequence was 669 663. All the 100-bp window sequences were searched for homologous regions using the BLASTN program. The repetition rate was defined as the total number of BLASTN hits (threshold: &gt;120 BLASTN similarity score, excluding hits to their own sequences). The longest ray corresponds to the 108-folds repetition. The outermost black circle is a scale where the short dashes mark 100-kb intervals. The circles inside the scale show, from the outside in: (i) genes on the forward strand (singleton genes in black and repeated genes in red), (ii) genes on the reverse strand (singletons in black and repeated genes in red), (iii) OtAGE sequences on the forward (orange) and reverse (greenish yellow) strands, (iv) transposable elements (ISOt1 and mISOt1, green; ISOt2 and mISOt2, light blue; ISOt3, blue; ISOt4 and mISOt4, orange; ISOt5, dark blue; GIIOt1, red), (v) singleton genes (gray) and repeated genes (light red) whose homologues are present in R. bellii, (vi) R. felis, (vii) R. conorii, (viii) R. prowazekii, (ix) G + C content [%: blue, higher than the average of the whole genome (30.5%); orange, lower than the average], (x) GC skew [(G–C)/(G + C): blue, ≥0; orange, &lt;0], and (xi) tRNA (green) and rRNA (orange) genes." path-from-xml="dsn01101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474290/dsn01101.jpeg" path-from-xml="dsn01101.jpeg" target="_blank">View large</a><a section="5474290" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01101.jpeg?Expires=2147483647&Signature=sJUapU4iaLKHHmbKCndtjgtEu8fgFM~qSvKjIBavVESRXcTIL9xCRp9~~eQPbJwZqiag3K6uehzDO9Qqfq2kzovaUoLLUv5ib7v9zN1y4NZEWqyUizmn4T2ruZviQ121JtJXQ6DdpxARV3M4OugAuYLHOHR1jYuXwaEGQc0ETTzkhwjf~05vo46OKKPRoGzp9VQDNGUXhGvYZ41CPaMPZ54yWB0WUgQs7QLROcbfCzv8T9DDEzxQb8nF06I63NPIXHByEYXJkz9xW7p4yTvP8FC99R9wKVH8CGyzXV3TV4fUdjdLo2v9WI~ctUOJzBHjpeROK4kgQPSmkAZ7DI30UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474290&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structural features of the <em>O. tsutsugamushi</em> genome. The putative origin of replication is located at the top. The outermost red rays show the repetition rates of each chromosomal region. The repetition rates were determined by sliding window analysis (window size, 100 bp; step size, 3 bp). The total number of 100-bp windows retrieved from the entire genome sequence was 669 663. All the 100-bp window sequences were searched for homologous regions using the BLASTN program. The repetition rate was defined as the total number of BLASTN hits (threshold: &gt;120 BLASTN similarity score, excluding hits to their own sequences). The longest ray corresponds to the 108-folds repetition. The outermost black circle is a scale where the short dashes mark 100-kb intervals. The circles inside the scale show, from the outside in: (i) genes on the forward strand (singleton genes in black and repeated genes in red), (ii) genes on the reverse strand (singletons in black and repeated genes in red), (iii) OtAGE sequences on the forward (orange) and reverse (greenish yellow) strands, (iv) transposable elements (IS<em>Ot1</em> and mIS<em>Ot1</em>, green; IS<em>Ot2</em> and mIS<em>Ot2</em>, light blue; IS<em>Ot3</em>, blue; IS<em>Ot4</em> and mIS<em>Ot4</em>, orange; IS<em>Ot5</em>, dark blue; GII<em>Ot1</em>, red), (v) singleton genes (gray) and repeated genes (light red) whose homologues are present in <em>R. bellii</em>, (vi) <em>R. felis</em>, (vii) <em>R. conorii</em>, (viii) <em>R. prowazekii</em>, (ix) G + C content [%: blue, higher than the average of the whole genome (30.5%); orange, lower than the average], (x) GC skew [(G–C)/(G + C): blue, ≥0; orange, &lt;0], and (xi) tRNA (green) and rRNA (orange) genes.</p></div></div><div content-id="DSN011F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01101.jpeg?Expires=2147483647&Signature=voSF3zrRATBEnWT3jievncAh0~YWxmMgsKUrkKqHAJQlwockbQyj8ch4~TV8MFdjkGuvlxhGanmvX-JdCoU-jyobY5oCuI2XijcGG4onpOpteYD2CWHXGueSTl5g-Xrb-th~paqegZwjAo242f4O0-McNr4FNrUXxYMXGRVQFsPk~P5TQMJP9vZta6i87dxj7MSxbwB8iolYHUcGIn1NVUdhlKL6B2L8TV19QvS7~iwqX9ArSy42F4jlYNNHBQOZsGr8u8VfDFzatpiZNDp2N~Zd3Ug8GoDLzrQZyUnEAcmgrpoEtZ3HLbMaf881qCv0dcg3G2lpLUOhoFY6OHI7RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structural features of the O. tsutsugamushi genome. The putative origin of replication is located at the top. The outermost red rays show the repetition rates of each chromosomal region. The repetition rates were determined by sliding window analysis (window size, 100 bp; step size, 3 bp). The total number of 100-bp windows retrieved from the entire genome sequence was 669 663. All the 100-bp window sequences were searched for homologous regions using the BLASTN program. The repetition rate was defined as the total number of BLASTN hits (threshold: &gt;120 BLASTN similarity score, excluding hits to their own sequences). The longest ray corresponds to the 108-folds repetition. The outermost black circle is a scale where the short dashes mark 100-kb intervals. The circles inside the scale show, from the outside in: (i) genes on the forward strand (singleton genes in black and repeated genes in red), (ii) genes on the reverse strand (singletons in black and repeated genes in red), (iii) OtAGE sequences on the forward (orange) and reverse (greenish yellow) strands, (iv) transposable elements (ISOt1 and mISOt1, green; ISOt2 and mISOt2, light blue; ISOt3, blue; ISOt4 and mISOt4, orange; ISOt5, dark blue; GIIOt1, red), (v) singleton genes (gray) and repeated genes (light red) whose homologues are present in R. bellii, (vi) R. felis, (vii) R. conorii, (viii) R. prowazekii, (ix) G + C content [%: blue, higher than the average of the whole genome (30.5%); orange, lower than the average], (x) GC skew [(G–C)/(G + C): blue, ≥0; orange, &lt;0], and (xi) tRNA (green) and rRNA (orange) genes." path-from-xml="dsn01101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474290/dsn01101.jpeg" path-from-xml="dsn01101.jpeg" target="_blank">View large</a><a section="5474290" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01101.jpeg?Expires=2147483647&Signature=sJUapU4iaLKHHmbKCndtjgtEu8fgFM~qSvKjIBavVESRXcTIL9xCRp9~~eQPbJwZqiag3K6uehzDO9Qqfq2kzovaUoLLUv5ib7v9zN1y4NZEWqyUizmn4T2ruZviQ121JtJXQ6DdpxARV3M4OugAuYLHOHR1jYuXwaEGQc0ETTzkhwjf~05vo46OKKPRoGzp9VQDNGUXhGvYZ41CPaMPZ54yWB0WUgQs7QLROcbfCzv8T9DDEzxQb8nF06I63NPIXHByEYXJkz9xW7p4yTvP8FC99R9wKVH8CGyzXV3TV4fUdjdLo2v9WI~ctUOJzBHjpeROK4kgQPSmkAZ7DI30UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474290&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structural features of the <em>O. tsutsugamushi</em> genome. The putative origin of replication is located at the top. The outermost red rays show the repetition rates of each chromosomal region. The repetition rates were determined by sliding window analysis (window size, 100 bp; step size, 3 bp). The total number of 100-bp windows retrieved from the entire genome sequence was 669 663. All the 100-bp window sequences were searched for homologous regions using the BLASTN program. The repetition rate was defined as the total number of BLASTN hits (threshold: &gt;120 BLASTN similarity score, excluding hits to their own sequences). The longest ray corresponds to the 108-folds repetition. The outermost black circle is a scale where the short dashes mark 100-kb intervals. The circles inside the scale show, from the outside in: (i) genes on the forward strand (singleton genes in black and repeated genes in red), (ii) genes on the reverse strand (singletons in black and repeated genes in red), (iii) OtAGE sequences on the forward (orange) and reverse (greenish yellow) strands, (iv) transposable elements (IS<em>Ot1</em> and mIS<em>Ot1</em>, green; IS<em>Ot2</em> and mIS<em>Ot2</em>, light blue; IS<em>Ot3</em>, blue; IS<em>Ot4</em> and mIS<em>Ot4</em>, orange; IS<em>Ot5</em>, dark blue; GII<em>Ot1</em>, red), (v) singleton genes (gray) and repeated genes (light red) whose homologues are present in <em>R. bellii</em>, (vi) <em>R. felis</em>, (vii) <em>R. conorii</em>, (viii) <em>R. prowazekii</em>, (ix) G + C content [%: blue, higher than the average of the whole genome (30.5%); orange, lower than the average], (x) GC skew [(G–C)/(G + C): blue, ≥0; orange, &lt;0], and (xi) tRNA (green) and rRNA (orange) genes.</p></div></div>                        <a id="5474294" scrollto-destination="5474294"></a>
<div content-id="DSN011TB1" class="table-modal"><div class="table-wrap"><div id="DSN011TB1" class="table-wrap-title" data-id="DSN011TB1"><div class="label -label">Table 1</div><div class="caption"><p>General features of the genome of <em>O. tsutsugamushi</em> strain IKEDA</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Chromosome (bp) </td><td>2 008 987 </td></tr><tr><td>G + C content (%) </td><td>30.5 </td></tr><tr><td>tRNA genes </td><td>34 </td></tr><tr><td>rRNA genes<sup>a</sup> </td><td>3 </td></tr><tr><td>Other RNA genes </td><td>1 </td></tr><tr><td>CDSs<sup>b</sup> </td><td>1967 (803) </td></tr><tr><td> Function assigned </td><td>1445 (651) </td></tr><tr><td> Hypothetical </td><td>522 (152) </td></tr><tr><td> Repeated genes </td><td>1196 (766) </td></tr><tr><td>  Function assigned </td><td>895 (619) </td></tr><tr><td>  Hypothetical </td><td>301 (147) </td></tr><tr><td> Singleton genes </td><td>771 (37) </td></tr><tr><td>  Function assigned </td><td>550 (32) </td></tr><tr><td>  Hypothetical </td><td>221 (5) </td></tr><tr><td>Repetitive elements </td><td>969 </td></tr><tr><td> 1. OtAGEs </td><td>185 </td></tr><tr><td> 2. Transposable elements </td><td>619 </td></tr><tr><td>  IS<em>Ot1</em> </td><td>113 </td></tr><tr><td>  mIS<em>Ot1</em> </td><td>61 </td></tr><tr><td>  IS<em>Ot2</em> </td><td>17 </td></tr><tr><td>  mIS<em>Ot2a</em> </td><td>30 </td></tr><tr><td>  mIS<em>Ot2b</em> </td><td>65 </td></tr><tr><td>  IS<em>Ot3</em> </td><td>33 </td></tr><tr><td>  IS<em>Ot4</em> </td><td>124 </td></tr><tr><td>  mIS<em>Ot4</em> </td><td>55 </td></tr><tr><td>  IS<em>Ot5</em> </td><td>70 </td></tr><tr><td>  GII<em>Ot1</em> </td><td>51 </td></tr><tr><td> 3. Short repeats </td><td>165 </td></tr><tr><td>  SR<em>Ot1</em> </td><td>77 </td></tr><tr><td>  SR<em>Ot2</em> </td><td>50 </td></tr><tr><td>  SR<em>Ot3</em> </td><td>23 </td></tr><tr><td>  SR<em>Ot4</em> </td><td>6 </td></tr><tr><td>  SR<em>Ot5</em> </td><td>5 </td></tr><tr><td>  SR<em>Ot6</em> </td><td>2 </td></tr><tr><td>  SR<em>Ot7</em> </td><td>2 </td></tr></tbody></table></div><div content-id="DSN011TB1" class="table-modal"><table><tbody><tr><td>Chromosome (bp) </td><td>2 008 987 </td></tr><tr><td>G + C content (%) </td><td>30.5 </td></tr><tr><td>tRNA genes </td><td>34 </td></tr><tr><td>rRNA genes<sup>a</sup> </td><td>3 </td></tr><tr><td>Other RNA genes </td><td>1 </td></tr><tr><td>CDSs<sup>b</sup> </td><td>1967 (803) </td></tr><tr><td> Function assigned </td><td>1445 (651) </td></tr><tr><td> Hypothetical </td><td>522 (152) </td></tr><tr><td> Repeated genes </td><td>1196 (766) </td></tr><tr><td>  Function assigned </td><td>895 (619) </td></tr><tr><td>  Hypothetical </td><td>301 (147) </td></tr><tr><td> Singleton genes </td><td>771 (37) </td></tr><tr><td>  Function assigned </td><td>550 (32) </td></tr><tr><td>  Hypothetical </td><td>221 (5) </td></tr><tr><td>Repetitive elements </td><td>969 </td></tr><tr><td> 1. OtAGEs </td><td>185 </td></tr><tr><td> 2. Transposable elements </td><td>619 </td></tr><tr><td>  IS<em>Ot1</em> </td><td>113 </td></tr><tr><td>  mIS<em>Ot1</em> </td><td>61 </td></tr><tr><td>  IS<em>Ot2</em> </td><td>17 </td></tr><tr><td>  mIS<em>Ot2a</em> </td><td>30 </td></tr><tr><td>  mIS<em>Ot2b</em> </td><td>65 </td></tr><tr><td>  IS<em>Ot3</em> </td><td>33 </td></tr><tr><td>  IS<em>Ot4</em> </td><td>124 </td></tr><tr><td>  mIS<em>Ot4</em> </td><td>55 </td></tr><tr><td>  IS<em>Ot5</em> </td><td>70 </td></tr><tr><td>  GII<em>Ot1</em> </td><td>51 </td></tr><tr><td> 3. Short repeats </td><td>165 </td></tr><tr><td>  SR<em>Ot1</em> </td><td>77 </td></tr><tr><td>  SR<em>Ot2</em> </td><td>50 </td></tr><tr><td>  SR<em>Ot3</em> </td><td>23 </td></tr><tr><td>  SR<em>Ot4</em> </td><td>6 </td></tr><tr><td>  SR<em>Ot5</em> </td><td>5 </td></tr><tr><td>  SR<em>Ot6</em> </td><td>2 </td></tr><tr><td>  SR<em>Ot7</em> </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup><em>O. tsutsugamushi</em> contains one set of rRNA genes (5S, 16S, and 23S), but the 16S rRNA gene is located separately from the others.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The numbers of pseudogenes are indicated in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5474294" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN011TB1" class="table-wrap-title" data-id="DSN011TB1"><div class="label -label">Table 1</div><div class="caption"><p>General features of the genome of <em>O. tsutsugamushi</em> strain IKEDA</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Chromosome (bp) </td><td>2 008 987 </td></tr><tr><td>G + C content (%) </td><td>30.5 </td></tr><tr><td>tRNA genes </td><td>34 </td></tr><tr><td>rRNA genes<sup>a</sup> </td><td>3 </td></tr><tr><td>Other RNA genes </td><td>1 </td></tr><tr><td>CDSs<sup>b</sup> </td><td>1967 (803) </td></tr><tr><td> Function assigned </td><td>1445 (651) </td></tr><tr><td> Hypothetical </td><td>522 (152) </td></tr><tr><td> Repeated genes </td><td>1196 (766) </td></tr><tr><td>  Function assigned </td><td>895 (619) </td></tr><tr><td>  Hypothetical </td><td>301 (147) </td></tr><tr><td> Singleton genes </td><td>771 (37) </td></tr><tr><td>  Function assigned </td><td>550 (32) </td></tr><tr><td>  Hypothetical </td><td>221 (5) </td></tr><tr><td>Repetitive elements </td><td>969 </td></tr><tr><td> 1. OtAGEs </td><td>185 </td></tr><tr><td> 2. Transposable elements </td><td>619 </td></tr><tr><td>  IS<em>Ot1</em> </td><td>113 </td></tr><tr><td>  mIS<em>Ot1</em> </td><td>61 </td></tr><tr><td>  IS<em>Ot2</em> </td><td>17 </td></tr><tr><td>  mIS<em>Ot2a</em> </td><td>30 </td></tr><tr><td>  mIS<em>Ot2b</em> </td><td>65 </td></tr><tr><td>  IS<em>Ot3</em> </td><td>33 </td></tr><tr><td>  IS<em>Ot4</em> </td><td>124 </td></tr><tr><td>  mIS<em>Ot4</em> </td><td>55 </td></tr><tr><td>  IS<em>Ot5</em> </td><td>70 </td></tr><tr><td>  GII<em>Ot1</em> </td><td>51 </td></tr><tr><td> 3. Short repeats </td><td>165 </td></tr><tr><td>  SR<em>Ot1</em> </td><td>77 </td></tr><tr><td>  SR<em>Ot2</em> </td><td>50 </td></tr><tr><td>  SR<em>Ot3</em> </td><td>23 </td></tr><tr><td>  SR<em>Ot4</em> </td><td>6 </td></tr><tr><td>  SR<em>Ot5</em> </td><td>5 </td></tr><tr><td>  SR<em>Ot6</em> </td><td>2 </td></tr><tr><td>  SR<em>Ot7</em> </td><td>2 </td></tr></tbody></table></div><div content-id="DSN011TB1" class="table-modal"><table><tbody><tr><td>Chromosome (bp) </td><td>2 008 987 </td></tr><tr><td>G + C content (%) </td><td>30.5 </td></tr><tr><td>tRNA genes </td><td>34 </td></tr><tr><td>rRNA genes<sup>a</sup> </td><td>3 </td></tr><tr><td>Other RNA genes </td><td>1 </td></tr><tr><td>CDSs<sup>b</sup> </td><td>1967 (803) </td></tr><tr><td> Function assigned </td><td>1445 (651) </td></tr><tr><td> Hypothetical </td><td>522 (152) </td></tr><tr><td> Repeated genes </td><td>1196 (766) </td></tr><tr><td>  Function assigned </td><td>895 (619) </td></tr><tr><td>  Hypothetical </td><td>301 (147) </td></tr><tr><td> Singleton genes </td><td>771 (37) </td></tr><tr><td>  Function assigned </td><td>550 (32) </td></tr><tr><td>  Hypothetical </td><td>221 (5) </td></tr><tr><td>Repetitive elements </td><td>969 </td></tr><tr><td> 1. OtAGEs </td><td>185 </td></tr><tr><td> 2. Transposable elements </td><td>619 </td></tr><tr><td>  IS<em>Ot1</em> </td><td>113 </td></tr><tr><td>  mIS<em>Ot1</em> </td><td>61 </td></tr><tr><td>  IS<em>Ot2</em> </td><td>17 </td></tr><tr><td>  mIS<em>Ot2a</em> </td><td>30 </td></tr><tr><td>  mIS<em>Ot2b</em> </td><td>65 </td></tr><tr><td>  IS<em>Ot3</em> </td><td>33 </td></tr><tr><td>  IS<em>Ot4</em> </td><td>124 </td></tr><tr><td>  mIS<em>Ot4</em> </td><td>55 </td></tr><tr><td>  IS<em>Ot5</em> </td><td>70 </td></tr><tr><td>  GII<em>Ot1</em> </td><td>51 </td></tr><tr><td> 3. Short repeats </td><td>165 </td></tr><tr><td>  SR<em>Ot1</em> </td><td>77 </td></tr><tr><td>  SR<em>Ot2</em> </td><td>50 </td></tr><tr><td>  SR<em>Ot3</em> </td><td>23 </td></tr><tr><td>  SR<em>Ot4</em> </td><td>6 </td></tr><tr><td>  SR<em>Ot5</em> </td><td>5 </td></tr><tr><td>  SR<em>Ot6</em> </td><td>2 </td></tr><tr><td>  SR<em>Ot7</em> </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup><em>O. tsutsugamushi</em> contains one set of rRNA genes (5S, 16S, and 23S), but the 16S rRNA gene is located separately from the others.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The numbers of pseudogenes are indicated in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5474294" target="_blank">View Large</a></div></div><p>Although the G + C content and the number of rRNA and tRNA genes do not largely differ from those of other <em>Rickettsiales</em>, the <em>O. tsutsugamushi</em> genome contains a much higher number of protein-coding genes (1967 CDSs) and has the largest genome size in the order <em>Rickettsiales</em>. Of the 1967 CDSs, as many as 1196 are repeated genes constituting 85 families (<em><span class="underline">O</span>. <span class="underline">t</span>sutsugamushi</em><span class="underline">R</span>epeated <span class="underline">G</span>ene families: OtRG1–OtRG85), where the members in each family exhibit more than 90% amino acid sequence identity (see Section 2 for the definition of repeated genes; all other genes that do not belong to any of the repeated gene families are referred to as ‘singleton genes’ in this study). This unusual gene duplication is associated with the explosive amplification of several repetitive sequences, as described below. Since massive gene decay has also taken place in these repeated genes, as much as 64% of the 1196 repeated CDSs (766) are pseudogenes (truncated, split, or degraded). In contrast, smaller numbers of pseudogenes were identified among the singleton genes (4.8%, or 37 out of the 771 CDSs). The proportion of pseudogenes among the singleton genes is, however, within a range similar to those of other <em>Rickettsia</em> genomes<sup><a reveal-id="DSN011C33  DSN011C37" data-open="DSN011C33  DSN011C37" class="link link-ref link-reveal xref-bibr">33–37</a></sup> known to harbor relatively larger numbers of split genes, a signature of progressive genome reduction.</p><p>Another striking feature of the <em>O. tsutsugamushi</em> genome is that it exhibits very poor colinearity to any of the previously sequenced <em>Rickettsia</em> genomes including that of <em>R. bellii</em>, which exhibits exceptionally little colinearity to other <em>Rickettsia</em> genomes<sup><a reveal-id="DSN011C33" data-open="DSN011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). This suggests that extensive genome shuffling has taken place in the evolutionary course of <em>O. tsutsugamushi</em>. The massively amplified repetitive sequences may have mediated this genome shuffling.</p>                            <h3 scrollto-destination=5474301 id="5474301" class="section-title" data-legacy-id=s3b>3.2. Explosive amplification of repetitive sequences</h3>
<p>We identified 18 types of repetitive sequences in the <em>O. tsutsugamushi</em> genome. They have explosively been amplified and scattered throughout the genome (Fig. <a reveal-id="DSN011F1" data-open="DSN011F1" class="link link-reveal link-table xref-fig">1</a>). Although massive decay has also taken place in these repetitive sequences, their total length now reaches 933.8 kb (46.5% of the entire genome). Among the bacteria so far sequenced, <em>Wolbachia pipientis w</em>Mel and <em>Anaplasma phagocytophilum</em>, both of which are obligate intracellular bacteria belonging to the order <em>Rickettsiales</em>, contain the highest proportion of repetitive sequences (14.2 and 12.7%, respectively),<sup><a reveal-id="DSN011C38" data-open="DSN011C38" class="link link-ref link-reveal xref-bibr">38</a></sup> but much lower than <em>O. tsutsugamushi</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Of interest is that the size of the <em>O. tsutsugamushi</em> genome divested of these repetitive sequences is almost the same as that of the <em>R. prowazekii</em> genome (1070 and 1116 kb, respectively).</p><p>The identified repetitive sequences were categorized into three types: (i) a genetic element which we named ‘<em>O. tsutsugamushi</em> amplified genetic element’ (OtAGE), (ii) transposable elements, and (iii) others (Table <a reveal-id="DSN011TB1" data-open="DSN011TB1" class="link link-reveal link-table xref-fig">1</a>).</p>                            <h4 scrollto-destination=5474308 id="5474308" class="section-title" data-legacy-id=s3b1>3.2.1. OtAGE</h4>
<p>The OtAGE is a large genetic element about 33 kb in size and encodes 33–38 genes. This element has explosively been amplified in the <em>O. tsutsugamushi</em> genome, and each copy has extensively been degenerated by various types of deletion and insertion. However, we could reconstitute a probable intact form of the OtAGE by comparing the structures of relatively large copies (Fig. <a reveal-id="DSN011F2" data-open="DSN011F2" class="link link-reveal link-table xref-fig">2</a>). Based on the reconstituted structure, we identified a total of 185 remnants of the OtAGE in the <em>O. tsutsugamushi</em> genome. The sum of their lengths is ∼694 kb (34.6% of the entire genome). We could not define their integration junctions because of the high sequence variation and accumulation of transposable elements in the possible junction regions.</p>                        <a id="5474314" scrollto-destination="5474314"></a>
<div id="DSN011F2" data-id="DSN011F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01102.jpeg?Expires=2147483647&Signature=X26bDCERqNyMufUW~qbserIPlev5hVBQCxZbAaGSnXwaCkwdIvDa0eeIFumBR3UejJYvR8SWX1HSw66CVxHibjQU4Dnh2JaTRYC9-IkbCQ--e1rjPpDNBd8fwj4COxXoOgI-A2j4xpZZWZ8ajH~ToVJME8PvIqJSE1rKfNbG3JkLz45HWAe9R0hvSWuFzlkEV2rFq1~Y~4QT2wSl94Iw0-pAdmYD~kt8GPKIFZDDZg4UiVw~wXgbqhK~oHt~jny2DgPiSOKN9t4j5km510Q964CD6~fhFwaW-Uee1hENGNm1j7h7unkR0Ji6oAiifIqArO2-eOx5D12UZacs6OwpiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene organization of OtAGEs (O. tsutsugamushi amplified genetic elements). The gene organizations of the longest 30 OtAGEs identified in the O. tsutsugamushi genome are shown. The structure of the intact OtAGE that we deduced from the structures of these 30 OtAGEs is shown at the top. Arrows depicted by solid lines indicate intact genes and those depicted by dotted lines indicate degraded ones. Genes for conjugal transfer are depicted in green, integrase genes in red, genes for ATPase domain-containing proteins in blue, genes for other functions in light orange, genes for transposases in yellow, conserved hypothetical genes in white, O. tsutsugamushi-specific hypothetical genes whose gene products exhibited no significant sequence similarities to known protein sequences in gray, and genes that were probably trapped or inserted in OtAGEs in black. Four types of integrase genes and three types of gene cassettes encoding ATPase domain-containing proteins are present. Balloon marks indicate the insertion sites of IS elements (the ISOt number is labeled for each IS element). Deleted regions are indicated by broken dotted lines." path-from-xml="dsn01102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474314/dsn01102.jpeg" path-from-xml="dsn01102.jpeg" target="_blank">View large</a><a section="5474314" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01102.jpeg?Expires=2147483647&Signature=KL-13LmqRYHcRT3ckkqxg~4GFw-G~doOg6cF16oVbX13P2nXx~4fv30GxahxO6gV47qwizUUYUTw3huZ00aeX8RPQucdvLNcsX9q4-TL9M-4GuQsjtjaztmxX5d5X51uIgR9BhYwCk9XlfHOg-kFy1JJvHEf545PtcIvUF-LfmZ3P~NJDpD5qYcD1an-rZnMa90nfsUoxqxHa-j7JWY~enEqvpBxEXr1cDxV~hUe7~Inq2X8lo221sTmfuMoHbh5fgXIHJbBezn4Kd0elfAEzDYPFwyVLjMiMZLWKkXC5yhEetIHiGKAnhZBtoFzJbiCM~voGMP3157DPjlsMtKEzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474314&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene organization of OtAGEs (<em>O. tsutsugamushi</em> amplified genetic elements). The gene organizations of the longest 30 OtAGEs identified in the <em>O. tsutsugamushi</em> genome are shown. The structure of the intact OtAGE that we deduced from the structures of these 30 OtAGEs is shown at the top. Arrows depicted by solid lines indicate intact genes and those depicted by dotted lines indicate degraded ones. Genes for conjugal transfer are depicted in green, integrase genes in red, genes for ATPase domain-containing proteins in blue, genes for other functions in light orange, genes for transposases in yellow, conserved hypothetical genes in white, <em>O. tsutsugamushi</em>-specific hypothetical genes whose gene products exhibited no significant sequence similarities to known protein sequences in gray, and genes that were probably trapped or inserted in OtAGEs in black. Four types of integrase genes and three types of gene cassettes encoding ATPase domain-containing proteins are present. Balloon marks indicate the insertion sites of IS elements (the IS<em>Ot</em> number is labeled for each IS element). Deleted regions are indicated by broken dotted lines.</p></div></div><div content-id="DSN011F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01102.jpeg?Expires=2147483647&Signature=X26bDCERqNyMufUW~qbserIPlev5hVBQCxZbAaGSnXwaCkwdIvDa0eeIFumBR3UejJYvR8SWX1HSw66CVxHibjQU4Dnh2JaTRYC9-IkbCQ--e1rjPpDNBd8fwj4COxXoOgI-A2j4xpZZWZ8ajH~ToVJME8PvIqJSE1rKfNbG3JkLz45HWAe9R0hvSWuFzlkEV2rFq1~Y~4QT2wSl94Iw0-pAdmYD~kt8GPKIFZDDZg4UiVw~wXgbqhK~oHt~jny2DgPiSOKN9t4j5km510Q964CD6~fhFwaW-Uee1hENGNm1j7h7unkR0Ji6oAiifIqArO2-eOx5D12UZacs6OwpiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene organization of OtAGEs (O. tsutsugamushi amplified genetic elements). The gene organizations of the longest 30 OtAGEs identified in the O. tsutsugamushi genome are shown. The structure of the intact OtAGE that we deduced from the structures of these 30 OtAGEs is shown at the top. Arrows depicted by solid lines indicate intact genes and those depicted by dotted lines indicate degraded ones. Genes for conjugal transfer are depicted in green, integrase genes in red, genes for ATPase domain-containing proteins in blue, genes for other functions in light orange, genes for transposases in yellow, conserved hypothetical genes in white, O. tsutsugamushi-specific hypothetical genes whose gene products exhibited no significant sequence similarities to known protein sequences in gray, and genes that were probably trapped or inserted in OtAGEs in black. Four types of integrase genes and three types of gene cassettes encoding ATPase domain-containing proteins are present. Balloon marks indicate the insertion sites of IS elements (the ISOt number is labeled for each IS element). Deleted regions are indicated by broken dotted lines." path-from-xml="dsn01102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474314/dsn01102.jpeg" path-from-xml="dsn01102.jpeg" target="_blank">View large</a><a section="5474314" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01102.jpeg?Expires=2147483647&Signature=KL-13LmqRYHcRT3ckkqxg~4GFw-G~doOg6cF16oVbX13P2nXx~4fv30GxahxO6gV47qwizUUYUTw3huZ00aeX8RPQucdvLNcsX9q4-TL9M-4GuQsjtjaztmxX5d5X51uIgR9BhYwCk9XlfHOg-kFy1JJvHEf545PtcIvUF-LfmZ3P~NJDpD5qYcD1an-rZnMa90nfsUoxqxHa-j7JWY~enEqvpBxEXr1cDxV~hUe7~Inq2X8lo221sTmfuMoHbh5fgXIHJbBezn4Kd0elfAEzDYPFwyVLjMiMZLWKkXC5yhEetIHiGKAnhZBtoFzJbiCM~voGMP3157DPjlsMtKEzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474314&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene organization of OtAGEs (<em>O. tsutsugamushi</em> amplified genetic elements). The gene organizations of the longest 30 OtAGEs identified in the <em>O. tsutsugamushi</em> genome are shown. The structure of the intact OtAGE that we deduced from the structures of these 30 OtAGEs is shown at the top. Arrows depicted by solid lines indicate intact genes and those depicted by dotted lines indicate degraded ones. Genes for conjugal transfer are depicted in green, integrase genes in red, genes for ATPase domain-containing proteins in blue, genes for other functions in light orange, genes for transposases in yellow, conserved hypothetical genes in white, <em>O. tsutsugamushi</em>-specific hypothetical genes whose gene products exhibited no significant sequence similarities to known protein sequences in gray, and genes that were probably trapped or inserted in OtAGEs in black. Four types of integrase genes and three types of gene cassettes encoding ATPase domain-containing proteins are present. Balloon marks indicate the insertion sites of IS elements (the IS<em>Ot</em> number is labeled for each IS element). Deleted regions are indicated by broken dotted lines.</p></div></div><p>The OtAGE encodes an integrase gene (<em>int</em>) at the left end, which is followed by a set of genes for conjugative transfer (<em>tra</em> genes) similar to those in various conjugative plasmids represented by the F plasmid (Fig. <a reveal-id="DSN011F3" data-open="DSN011F3" class="link link-reveal link-table xref-fig">3</a>). The OtAGE can thus be regarded as a member of the integrative and conjugative element (ICE) group, a recently recognized group of mobile genetic elements that disseminate by conjugative transfer like conjugative plasmids and integrate themselves into the genome like temperate bacteriophages.<sup><a reveal-id="DSN011C39" data-open="DSN011C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The <em>tra</em> gene product sequences are highly conserved between OtAGEs (&gt;90% amino-acid sequence identity), but four types of <em>int</em> genes were clearly distinguished (OtRG10–OtRG14; OtRG12 and OtRG13 represent one split gene, Fig. <a reveal-id="DSN011F2" data-open="DSN011F2" class="link link-reveal link-table xref-fig">2</a>). These integrases have significantly diverged in sequence (36–38% identity to each other), indicating that they have different origins.</p>                        <a id="5474322" scrollto-destination="5474322"></a>
<div id="DSN011F3" data-id="DSN011F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01103.jpeg?Expires=2147483647&Signature=j5J40-5qBviaDYSSLOiBuzyVeSdG5Bcae74R4GIu3Mnaiuko5LHoCISO2BLnCgcA5L4zCgxIADn7h0hdqquzBOAzl6fZAN9ACjcGmUxwln7hH06TQ85iXOIubPmKozZXJIQNWvp43~dnkmDix51RXveIggTHM~5Pff8sVkvhKZkfgoG5DEeoh0~AYSN8DevVnORMlpJx0PMDEwT6aeQ7Kn7xyXarLwO-xfzRodO0j7Hre7yHAEPh7uJtwhBgOi22zFDKlgpw5TbXxWrSan2bgfq-yTvDVaE5LEwtsNQswngUzIqCS2TTZG8TSpDfZbzZAvsce0gSqh9klsS-j1fiJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the OtAGE with other integrative and conjugative elements (ICEs). The genetic organizations of the OtAGE and five other ICEs are drawn to scale. Only the regions required for conjugal transfer are shown, except for the OtAGE and an OtAGE-like ICE of R. bellii. The R. bellii element is integrated into a valine tRNA gene with 54-bp sequence duplication (indicated by red rhomboids). The R. felis plasmid contains only few genes related to conjugation. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white, and O. tsutsugamushi-specific hypothetical genes by gray. The homologous regions are indicated by light green shading. Homologous regions were identified by the BLASTP program (e-value threshold: ≤1e−10). To identify the homologous regions in the traA, traE, and traV gene products, which showed higher sequence diversity between elements, we used the ClustalW program.30" path-from-xml="dsn01103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474322/dsn01103.jpeg" path-from-xml="dsn01103.jpeg" target="_blank">View large</a><a section="5474322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01103.jpeg?Expires=2147483647&Signature=RmKiGQivpuT5fs9gyHeRbPGbZZF-n1a72zBbG57IWZ3qKFP4Id7A~7lq4XTK0IWHaGLB7i9V8TVLaWiO3ZtVpLAv2N7oBR9BwD64Mxba7xhTxEg0QHCw9DuiR8ODeENCJOZy8O8Gj0l0xVm79FznAPb4AoqrfajLqJCAoQdpUEKfHW5La82idCHCgI7wHR9y2b~WBSO8iYyxyBQNMvRB~pi3ar1LL6Ncplr7yeQMduTL0~Ju8lY1RtS4b01riPlH4blKbhY8UyLAQ1qq6fbUSoKpe2WE1RIZtpobIXMnCBiFVqcCwTvJhtEJ8-kXyrp89VwasFq1T6dHxTds~2L0ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474322&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the OtAGE with other integrative and conjugative elements (ICEs). The genetic organizations of the OtAGE and five other ICEs are drawn to scale. Only the regions required for conjugal transfer are shown, except for the OtAGE and an OtAGE-like ICE of <em>R. bellii</em>. The <em>R. bellii</em> element is integrated into a valine tRNA gene with 54-bp sequence duplication (indicated by red rhomboids). The <em>R. felis</em> plasmid contains only few genes related to conjugation. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white, and <em>O. tsutsugamushi</em>-specific hypothetical genes by gray. The homologous regions are indicated by light green shading. Homologous regions were identified by the BLASTP program (e-value threshold: ≤1e−10). To identify the homologous regions in the <em>traA</em>, <em>traE</em>, and <em>traV</em> gene products, which showed higher sequence diversity between elements, we used the ClustalW program.<sup><a reveal-id="DSN011C30" data-open="DSN011C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p></div></div><div content-id="DSN011F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01103.jpeg?Expires=2147483647&Signature=j5J40-5qBviaDYSSLOiBuzyVeSdG5Bcae74R4GIu3Mnaiuko5LHoCISO2BLnCgcA5L4zCgxIADn7h0hdqquzBOAzl6fZAN9ACjcGmUxwln7hH06TQ85iXOIubPmKozZXJIQNWvp43~dnkmDix51RXveIggTHM~5Pff8sVkvhKZkfgoG5DEeoh0~AYSN8DevVnORMlpJx0PMDEwT6aeQ7Kn7xyXarLwO-xfzRodO0j7Hre7yHAEPh7uJtwhBgOi22zFDKlgpw5TbXxWrSan2bgfq-yTvDVaE5LEwtsNQswngUzIqCS2TTZG8TSpDfZbzZAvsce0gSqh9klsS-j1fiJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the OtAGE with other integrative and conjugative elements (ICEs). The genetic organizations of the OtAGE and five other ICEs are drawn to scale. Only the regions required for conjugal transfer are shown, except for the OtAGE and an OtAGE-like ICE of R. bellii. The R. bellii element is integrated into a valine tRNA gene with 54-bp sequence duplication (indicated by red rhomboids). The R. felis plasmid contains only few genes related to conjugation. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white, and O. tsutsugamushi-specific hypothetical genes by gray. The homologous regions are indicated by light green shading. Homologous regions were identified by the BLASTP program (e-value threshold: ≤1e−10). To identify the homologous regions in the traA, traE, and traV gene products, which showed higher sequence diversity between elements, we used the ClustalW program.30" path-from-xml="dsn01103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474322/dsn01103.jpeg" path-from-xml="dsn01103.jpeg" target="_blank">View large</a><a section="5474322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01103.jpeg?Expires=2147483647&Signature=RmKiGQivpuT5fs9gyHeRbPGbZZF-n1a72zBbG57IWZ3qKFP4Id7A~7lq4XTK0IWHaGLB7i9V8TVLaWiO3ZtVpLAv2N7oBR9BwD64Mxba7xhTxEg0QHCw9DuiR8ODeENCJOZy8O8Gj0l0xVm79FznAPb4AoqrfajLqJCAoQdpUEKfHW5La82idCHCgI7wHR9y2b~WBSO8iYyxyBQNMvRB~pi3ar1LL6Ncplr7yeQMduTL0~Ju8lY1RtS4b01riPlH4blKbhY8UyLAQ1qq6fbUSoKpe2WE1RIZtpobIXMnCBiFVqcCwTvJhtEJ8-kXyrp89VwasFq1T6dHxTds~2L0ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474322&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the OtAGE with other integrative and conjugative elements (ICEs). The genetic organizations of the OtAGE and five other ICEs are drawn to scale. Only the regions required for conjugal transfer are shown, except for the OtAGE and an OtAGE-like ICE of <em>R. bellii</em>. The <em>R. bellii</em> element is integrated into a valine tRNA gene with 54-bp sequence duplication (indicated by red rhomboids). The <em>R. felis</em> plasmid contains only few genes related to conjugation. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white, and <em>O. tsutsugamushi</em>-specific hypothetical genes by gray. The homologous regions are indicated by light green shading. Homologous regions were identified by the BLASTP program (e-value threshold: ≤1e−10). To identify the homologous regions in the <em>traA</em>, <em>traE</em>, and <em>traV</em> gene products, which showed higher sequence diversity between elements, we used the ClustalW program.<sup><a reveal-id="DSN011C30" data-open="DSN011C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p></div></div><p>In the right one-third of the OtAGE, genes for various functions reside (Fig. <a reveal-id="DSN011F2" data-open="DSN011F2" class="link link-reveal link-table xref-fig">2</a>). These include two genes encoding almost identical SpoT (ppGpp hydrolase)-like proteins, a gene cassette encoding multiple ATPase domain-containing proteins, and genes for DNA methylase and DNA helicase. As for the gene cassette encoding ATPase domain-containing proteins, three types of cassettes, each encoding two to four ATPase domain-containing proteins, were identified. The encoded ATPase domain-containing proteins exhibited significant sequence diversities, not only between the three types of cassettes but also within the same cassette (up to 49% amino-acid sequence identity, but in most cases less than 40%), indicating a complex evolutionary history of the gene cassettes. In addition, several genes, whose gene products are also apparently not related to conjugative transfer, such as peroxiredoxin and ankyrin repeat (AR)-proteins, were found to be inserted into various sites in the OtAGE.</p><p>Among the <em>tra</em> gene clusters so far identified in various conjugative plasmids and ICEs, the gene cluster in the OtAGE is most similar in sequence and gene organization to that identified in the <em>R. bellii</em> genome<sup><a reveal-id="DSN011C33" data-open="DSN011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> (Fig. <a reveal-id="DSN011F3" data-open="DSN011F3" class="link link-reveal link-table xref-fig">3</a>). The <em>R. bellii tra</em> gene cluster is also preceded by an <em>int</em> gene. Furthermore, the <em>R. bellii int</em>/<em>tra</em> gene cluster is located in a 38.7-kb element that is integrated into a valine tRNA gene with a terminal sequence duplication of 54 bp.<sup><a reveal-id="DSN011C40" data-open="DSN011C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The <em>R. bellii</em> element also contains a set of genes similar but not identical to those on the right part of the OtAGE, indicating that it is an ICE closely related to the OtAGE. Among the <em>Rickettsia</em> species so far sequenced, <em>R. massiliae</em>, which has very recently been sequenced,<sup><a reveal-id="DSN011C40" data-open="DSN011C40" class="link link-ref link-reveal xref-bibr">40</a></sup> also contains an OtAGE-like element (data not shown) and <em>R. felis</em> contains a few <em>tra</em> genes or their remnants in a plasmid.<sup><a reveal-id="DSN011C34" data-open="DSN011C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We could not estimate the time point when the OtAGEs invaded <em>O. tsutsugamushi</em>. However, the presence of four distinct <em>int</em> genes and three different types of gene cassettes encoding ATPase domain-containing proteins among the OtAGEs may suggest that invasion of <em>O. tsutsugamushi</em> by similar but different types of OtAGEs took place at least four times.</p><p>A sex pilus-like structure was observed on the cell surface of <em>R. bellii</em>, suggesting that the <em>R. bellii</em> ICE may still be active in conjugation.<sup><a reveal-id="DSN011C33" data-open="DSN011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In contrast, all the OtAGEs contained various types of deletions, and many of the remaining <em>tra</em> genes have become pseudogenes by base substitution and deletion, or insertion of transposable elements (Fig. <a reveal-id="DSN011F2" data-open="DSN011F2" class="link link-reveal link-table xref-fig">2</a>). Thus, all the OtAGEs currently present in <em>O. tsutsugamushi</em> strain Ikeda appear to be no longer transmissible by themselves.</p><p>Although an increasing number of ICE have been identified in a wide range of prokaryotes, the amplification of ICEs in a single genome has never been reported so far. Thus, the amplification of the OtAGE is a quite unique phenomenon. The mechanism of this unusual amplification is not known. However, once a mutation affecting the inhibition mechanism of invasion by the same type of ICE, similar to TraT-mediated surface exclusion and TraS-mediated entry exclusion in the F plasmid,<sup><a reveal-id="DSN011C41" data-open="DSN011C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSN011C42" data-open="DSN011C42" class="link link-ref link-reveal xref-bibr">42</a></sup> was introduced, reciprocal transfer of the OtAGE could have taken place between the <em>O. tsutsugamushi</em> cells densely inhabiting the host cell cytosol. Such reciprocal transfers may have induced the unusual amplification of the OtAGE in <em>O. tsutsugamushi</em>.</p>                            <h4 scrollto-destination=5474337 id="5474337" class="section-title" data-legacy-id=s3b2>3.2.2. Transposable elements</h4>
<p>We identified five types of insertion sequence (IS) elements (named IS<em>Ot1</em>–IS<em>Ot5</em>), four types of miniature inverted-repeat transposable elements (MITEs), and a Group II (GII) intron (Fig. <a reveal-id="DSN011F4" data-open="DSN011F4" class="link link-reveal link-table xref-fig">4</a>). None of these transposable elements is conserved in other sequenced <em>Rickettsia</em> genomes. Each type of transposable element has also been explosively amplified, and a total of 621 copies including their fragments were identified, altogether representing 13.0% of the genome (261 kb in total). The amplification of transposable elements has been described also in other obligate intracellular bacteria with reduced genomes, such as <em>Wolbachia pipientis w</em>Mel [seven types of IS elements (51 copies in total) and four types of GII introns (17 copies)],<sup><a reveal-id="DSN011C38" data-open="DSN011C38" class="link link-ref link-reveal xref-bibr">38</a></sup><em>Parachlamydia</em> sp. UWE25 [82 IS transposases (TPases)],<sup><a reveal-id="DSN011C43" data-open="DSN011C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>R. felis</em> (82 TPases),<sup><a reveal-id="DSN011C34" data-open="DSN011C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and <em>R. bellii</em> (39 TPases).<sup><a reveal-id="DSN011C33" data-open="DSN011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> However, the number of copies in <em>O. tsutsugamushi</em> is more than 10 times higher than those in these bacteria, and comparable to that in <em>Shigella dysenteriae</em>, which contains the highest number of IS elements among the prokaryotes sequenced to date (701 copies in a 4469-kb chromosome and a 183-kb plasmid).<sup><a reveal-id="DSN011C44" data-open="DSN011C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                        <a id="5474343" scrollto-destination="5474343"></a>
<div id="DSN011F4" data-id="DSN011F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01104.jpeg?Expires=2147483647&Signature=2zFwORTb9xT8qXRBwPt1xP5VlLVKjXwgnPZfUhvk6C-tDXz~tmNk2Q-LCeWpuZAinKlz3qfBR5G4izn6zuztfgntY1BmBtQyvsac8fBufd1QStnBlZ0dCD85hkrDB4ReKgUOcx6m2vFpFAQNE8e0e8DnqVU-AKEmidh9BSI-F~NAR~kFkTDupO16j1WrNChHykWROzEX2ZdMHVz09X6Qe6WIgfYK4Z4XhqAZfKN7FMIjk3GA~xS3WCTHKfDnwZyvjsajCi0ezF18omLPAPYFZl1UH45bZClDxxfd8S9kgSo~T1ddnXgKKkMctqcBYO0wl57NrWn04b-3vwORlxv84Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transposable elements of O. tsutsugamushi strain Ikeda. The genetic organizations of the ransposable elements identified in the genome of O. tsutsugamushi strain Ikeda are drawn to scale. Family names, lengths and other information for each type of transposable element are also shown. ISOt1, ISOt2, and ISOt4 have cognate miniature inverted-repeat transposable elements (MITEs) namely mISOt1, mISOt2, and mISOt4, respectively. There are two variants of mISOt2 (mISOt2a and ISOt2b). Regions with almost identical sequences between IS elements and MITEs are indicated by dotted lines. Transposases and reverse transcriptases are depicted by orange arrows, and terminal inverted repeats by grey arrow heads. ISOt1 is most closely related to IS1381 among the members of the IS5 family, and ISOt5 to IS621 among the members of the IS630 family." path-from-xml="dsn01104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474343/dsn01104.jpeg" path-from-xml="dsn01104.jpeg" target="_blank">View large</a><a section="5474343" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01104.jpeg?Expires=2147483647&Signature=JrJF3ZK~w7kR74CNkrbt~-CggeCJ1ySGFS28rSW10sD4Eogf7we-v4jbdhj9irMGgi1oAT5Z2qx0JCG7h~HiyG9ybz9TykqsyBzUHjZF6U0YgtBpFZp3qH1LuN53tg2rc0W1jHq0zDrO0HHnftOLpXRi7K13tGix-pVl-ayivPA9cl7b9GiYsHtmPPKCIUCL7oJJQ1n-viJWk8uBI9vrjTFxaGQa4yGvFgloCLrNre349mzb4Bj6Wi~gsK5jlQQp31JFrFcjnxDRekZ9Q45KovVuYKAsC90l8XM1JXw3muLE-O~0lLxG87S37qOSM9k~dd2jC9-iuk2bRqShE4sn4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474343&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transposable elements of <em>O. tsutsugamushi</em> strain Ikeda. The genetic organizations of the ransposable elements identified in the genome of <em>O. tsutsugamushi</em> strain Ikeda are drawn to scale. Family names, lengths and other information for each type of transposable element are also shown. IS<em>Ot1</em>, IS<em>Ot2</em>, and IS<em>Ot4</em> have cognate miniature inverted-repeat transposable elements (MITEs) namely mIS<em>Ot1</em>, mIS<em>Ot2</em>, and mIS<em>Ot4</em>, respectively. There are two variants of mIS<em>Ot2</em> (mIS<em>Ot2</em>a and IS<em>Ot2</em>b). Regions with almost identical sequences between IS elements and MITEs are indicated by dotted lines. Transposases and reverse transcriptases are depicted by orange arrows, and terminal inverted repeats by grey arrow heads. IS<em>Ot1</em> is most closely related to IS<em>1381</em> among the members of the IS<em>5</em> family, and IS<em>Ot5</em> to IS<em>621</em> among the members of the IS<em>630</em> family.</p></div></div><div content-id="DSN011F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01104.jpeg?Expires=2147483647&Signature=2zFwORTb9xT8qXRBwPt1xP5VlLVKjXwgnPZfUhvk6C-tDXz~tmNk2Q-LCeWpuZAinKlz3qfBR5G4izn6zuztfgntY1BmBtQyvsac8fBufd1QStnBlZ0dCD85hkrDB4ReKgUOcx6m2vFpFAQNE8e0e8DnqVU-AKEmidh9BSI-F~NAR~kFkTDupO16j1WrNChHykWROzEX2ZdMHVz09X6Qe6WIgfYK4Z4XhqAZfKN7FMIjk3GA~xS3WCTHKfDnwZyvjsajCi0ezF18omLPAPYFZl1UH45bZClDxxfd8S9kgSo~T1ddnXgKKkMctqcBYO0wl57NrWn04b-3vwORlxv84Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transposable elements of O. tsutsugamushi strain Ikeda. The genetic organizations of the ransposable elements identified in the genome of O. tsutsugamushi strain Ikeda are drawn to scale. Family names, lengths and other information for each type of transposable element are also shown. ISOt1, ISOt2, and ISOt4 have cognate miniature inverted-repeat transposable elements (MITEs) namely mISOt1, mISOt2, and mISOt4, respectively. There are two variants of mISOt2 (mISOt2a and ISOt2b). Regions with almost identical sequences between IS elements and MITEs are indicated by dotted lines. Transposases and reverse transcriptases are depicted by orange arrows, and terminal inverted repeats by grey arrow heads. ISOt1 is most closely related to IS1381 among the members of the IS5 family, and ISOt5 to IS621 among the members of the IS630 family." path-from-xml="dsn01104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474343/dsn01104.jpeg" path-from-xml="dsn01104.jpeg" target="_blank">View large</a><a section="5474343" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01104.jpeg?Expires=2147483647&Signature=JrJF3ZK~w7kR74CNkrbt~-CggeCJ1ySGFS28rSW10sD4Eogf7we-v4jbdhj9irMGgi1oAT5Z2qx0JCG7h~HiyG9ybz9TykqsyBzUHjZF6U0YgtBpFZp3qH1LuN53tg2rc0W1jHq0zDrO0HHnftOLpXRi7K13tGix-pVl-ayivPA9cl7b9GiYsHtmPPKCIUCL7oJJQ1n-viJWk8uBI9vrjTFxaGQa4yGvFgloCLrNre349mzb4Bj6Wi~gsK5jlQQp31JFrFcjnxDRekZ9Q45KovVuYKAsC90l8XM1JXw3muLE-O~0lLxG87S37qOSM9k~dd2jC9-iuk2bRqShE4sn4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474343&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transposable elements of <em>O. tsutsugamushi</em> strain Ikeda. The genetic organizations of the ransposable elements identified in the genome of <em>O. tsutsugamushi</em> strain Ikeda are drawn to scale. Family names, lengths and other information for each type of transposable element are also shown. IS<em>Ot1</em>, IS<em>Ot2</em>, and IS<em>Ot4</em> have cognate miniature inverted-repeat transposable elements (MITEs) namely mIS<em>Ot1</em>, mIS<em>Ot2</em>, and mIS<em>Ot4</em>, respectively. There are two variants of mIS<em>Ot2</em> (mIS<em>Ot2</em>a and IS<em>Ot2</em>b). Regions with almost identical sequences between IS elements and MITEs are indicated by dotted lines. Transposases and reverse transcriptases are depicted by orange arrows, and terminal inverted repeats by grey arrow heads. IS<em>Ot1</em> is most closely related to IS<em>1381</em> among the members of the IS<em>5</em> family, and IS<em>Ot5</em> to IS<em>621</em> among the members of the IS<em>630</em> family.</p></div></div><p>Among the five IS elements, two (IS<em>Ot2</em> and IS<em>Ot3</em>) belong to the IS<em>630</em> and Tc-1/mariner family. Tc-1/mariner elements are often found in the genomes of nematodes and insects, but rarely in bacteria.<sup><a reveal-id="DSN011C45" data-open="DSN011C45" class="link link-ref link-reveal xref-bibr">45</a></sup> This may imply that IS<em>Ot2</em> and IS<em>Ot3</em> were transferred from chiggers. Among the four types of MITEs, one contains the same terminal inverted repeat sequences (TIRs) as IS<em>Ot1</em>, and is therefore named ‘mIS<em>Ot1’</em> (miniature IS<em>Ot1</em>). Two (namely ‘mIS<em>Ot2a’</em> and ‘mIS<em>Ot2b</em>’) share the same or very similar TIRs with IS<em>Ot2</em>, and the remaining one (mIS<em>Ot4</em>) with IS<em>Ot4</em>. The massive amplification of these MITEs (201 copies in total) suggests that they are (or at least were) very actively mobilized by transposases encoded <em>in trans</em> in the cognate IS elements. Although numerous MITEs have been found in many eukaryotic genomes,<sup><a reveal-id="DSN011C46" data-open="DSN011C46" class="link link-ref link-reveal xref-bibr">46</a></sup> no such amplification of MITEs has been observed in prokaryotes. Similarly, to the best of our knowledge, <em>O. tsutsugamushi</em> contains the highest number of GII introns (51 copies) among the prokaryotes thus far sequenced, with <em>Thermosynechococcus elongates</em> BP-1 being the next with 28 copies (<a class="link link-uri" href="http://www.fp.ucalgary.ca/group2introns/">http://www.fp.ucalgary.ca/group2introns/</a>).<sup><a reveal-id="DSN011C47" data-open="DSN011C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Many of the transposable elements identified in the <em>O. tsutsugamushi</em> genome have been truncated by deletion or fragmented by insertion of other transposable elements (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>).</p>                            <h4 scrollto-destination=5474348 id="5474348" class="section-title" data-legacy-id=s3b3>3.2.3. Other repetitive sequences</h4>
<p>We additionally identified seven types of repetitive sequences in the <em>O. tsutsugamushi</em> genome (165 copies in total) as summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. These repetitive sequences, namely ‘Short Repeats of <em>O. tsutsugamushi’</em> (SROt1–SROt7), are not related to the <em>Rickettsia</em> palindromic elements (RPEs) widely distributed in <em>Rickettsia</em> genomes.<sup><a reveal-id="DSN011C35" data-open="DSN011C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Among the seven SRs, only SROt2 (110 bp in length) was found to contain TIR-like sequences of 21 bp, suggesting that it may also be a type of MITE, although no cognate IS element was identified. The functions and origins of other SRs are unknown.</p>                            <h3 scrollto-destination=5474353 id="5474353" class="section-title" data-legacy-id=s3c>3.3. Repeated genes</h3>
<p>The massive amplification of repetitive sequences in the <em>O. tsutsugamushi</em> genome is directly or indirectly linked to the generation of a large number of repeated genes, which were classified into 85 OtRG families (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Among these, nine are TPases and a reverse transcriptase/maturase in the IS elements and the GII intron, and 45 are the core components of the OtAGE mentioned above. The remaining 31 OtRG families are located at various sites in the chromosome, but are frequently associated with the ISs/MITEs and/or OtAGEs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). This suggests that these mobile elements have been involved in the proliferation of many of these OtRG families. In fact, eight of the 31 OtRG families have a single orthologue in other rickettsial species. These genes may have been trapped in or between these mobile elements, and duplicated along with the amplification of the mobile elements. The eight OtRGs include a unique two-component system sensor histidine kinase/response regulator hybrid protein fused with a sodium/proton transporter domain (OtRG55) and a peroxiredoxin (OtRG59). These proteins may be involved in sensing and/or coping with osmotic or oxidative stresses which the bacterium encounters in the mite cells or during the infection of mammals. Most of the other OtRG families are hypothetical protein families with unknown functions, but their amplification may also have conferred some advantages to <em>O. tsutsugamushi</em>, by adapting them to the host environments. In this regard, of particular interest are several intramolecular repeat-containing proteins that have also been amplified (see Section 3.5).</p>                            <h3 scrollto-destination=5474358 id="5474358" class="section-title" data-legacy-id=s3d>3.4. Singleton genes and the metabolic capacity of O. tsutsugamushi</h3>
<p>The number of singleton genes (771 genes) identified in the <em>O. tsutsugamushi</em> genome is lower than the total number of CDSs (834) in <em>R. prowazekii</em>, which has the smallest genome among the <em>Rickettsia</em> species so far sequenced.<sup><a reveal-id="DSN011C36" data-open="DSN011C36" class="link link-ref link-reveal xref-bibr">36</a></sup> As shown in Table <a reveal-id="DSN011TB2" data-open="DSN011TB2" class="link link-reveal link-table xref-fig">2</a>, the genome comparison of <em>O. tsutsugamushi</em> and <em>R. prowazekii</em> revealed 542 genes that are shared by the two species. Most of them (519 genes, 95.8%) are conserved in all other sequenced <em>Rickettsia</em> species (Table <a reveal-id="DSN011TB2" data-open="DSN011TB2" class="link link-reveal link-table xref-fig">2</a>), and are therefore likely to represent the core gene set of the family <em>Rickettsiaceae</em>. This set includes various housekeeping genes and also a set of genes for the type IV secretion system (T4SS) similar to the <em>virB</em> system of <em>Agrobacterium tumefaciens</em>. Despite the extensive genome shuffling, the organization of T4SS genes is highly conserved between <em>O. tsutsugamushi</em> and <em>Rickettsia</em>. A similar set of T4SS genes is also conserved in the family <em>Anaplasmataceae</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). This strongly suggests that T4SS plays an essential role in maintaining the intracellular life style of the members of the order <em>Rickettsiales</em>.</p>                        <a id="5474362" scrollto-destination="5474362"></a>
<div content-id="DSN011TB2" class="table-modal"><div class="table-wrap"><div id="DSN011TB2" class="table-wrap-title" data-id="DSN011TB2"><div class="label -label">Table 2</div><div class="caption"><p>Conservation of the <em>O. tsutsugamushi</em> singleton genes in the sequenced five <em>Rickettsia</em> species<sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Conservation patterns<sup>b</sup> </th><th>Number of genes<sup>c</sup> </th></tr></thead><tbody><tr><td>Ot (+)/Rp (−) </td><td>195 (149) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> </td><td>161 (131) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>34 (18) </td></tr><tr><td>Ot (+)/Rp (+) </td><td>542 (84) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>519 (80) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>23 (4) </td></tr><tr><td>Ot (−)/Rp (+) </td><td>257 (125) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>178 (75) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>79 (50) </td></tr></tbody></table></div><div content-id="DSN011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Conservation patterns<sup>b</sup> </th><th>Number of genes<sup>c</sup> </th></tr></thead><tbody><tr><td>Ot (+)/Rp (−) </td><td>195 (149) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> </td><td>161 (131) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>34 (18) </td></tr><tr><td>Ot (+)/Rp (+) </td><td>542 (84) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>519 (80) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>23 (4) </td></tr><tr><td>Ot (−)/Rp (+) </td><td>257 (125) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>178 (75) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>79 (50) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The five <em>Rickettsia</em> species examined here include <em>R. prowazekii, R. conorii, R. felis, R. typhi</em>, and <em>R. bellii</em>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Ot(+)/Rp(−), <em>O. tsutsugamushi</em> singleton genes that are not present in <em>R. prowazekii</em>; Ot(+)/Rp(+), <em>O. tsutsugamushi</em> singleton genes that are present in <em>R. prowazekii</em>; Ot(−)/Rp(+), <em>R. prowazekii</em> genes that are not present in <em>O. tsutsugamushi</em>.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The numbers of genes encoding hypothetical proteins of unknown functions are indicated in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5474362" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN011TB2" class="table-wrap-title" data-id="DSN011TB2"><div class="label -label">Table 2</div><div class="caption"><p>Conservation of the <em>O. tsutsugamushi</em> singleton genes in the sequenced five <em>Rickettsia</em> species<sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Conservation patterns<sup>b</sup> </th><th>Number of genes<sup>c</sup> </th></tr></thead><tbody><tr><td>Ot (+)/Rp (−) </td><td>195 (149) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> </td><td>161 (131) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>34 (18) </td></tr><tr><td>Ot (+)/Rp (+) </td><td>542 (84) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>519 (80) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>23 (4) </td></tr><tr><td>Ot (−)/Rp (+) </td><td>257 (125) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>178 (75) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>79 (50) </td></tr></tbody></table></div><div content-id="DSN011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Conservation patterns<sup>b</sup> </th><th>Number of genes<sup>c</sup> </th></tr></thead><tbody><tr><td>Ot (+)/Rp (−) </td><td>195 (149) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> </td><td>161 (131) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>34 (18) </td></tr><tr><td>Ot (+)/Rp (+) </td><td>542 (84) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>519 (80) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>23 (4) </td></tr><tr><td>Ot (−)/Rp (+) </td><td>257 (125) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> </td><td>178 (75) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> </td><td>79 (50) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The five <em>Rickettsia</em> species examined here include <em>R. prowazekii, R. conorii, R. felis, R. typhi</em>, and <em>R. bellii</em>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Ot(+)/Rp(−), <em>O. tsutsugamushi</em> singleton genes that are not present in <em>R. prowazekii</em>; Ot(+)/Rp(+), <em>O. tsutsugamushi</em> singleton genes that are present in <em>R. prowazekii</em>; Ot(−)/Rp(+), <em>R. prowazekii</em> genes that are not present in <em>O. tsutsugamushi</em>.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The numbers of genes encoding hypothetical proteins of unknown functions are indicated in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5474362" target="_blank">View Large</a></div></div><p>It has been previously reported that <em>O. tsutsugamushi</em> cells do not contain lipopolysaccharide (LPS) and peptidoglycan, while the members of genus <em>Rickettsia</em> contain these cellular components.<sup><a reveal-id="DSN011C48" data-open="DSN011C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Consistent with this, the electron-microscopic appearance of the outer membrane of <em>O. tsutsugamushi</em> is very different from those of the members of the genus <em>Rickettsia</em>, and these differences were the major reasons to separate <em>O. tsutsugamushi</em> from the genus <em>Rickettsia</em>.<sup><a reveal-id="DSN011C49" data-open="DSN011C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The higher sensitivity to various physical stresses observed in <em>O. tsutsugamushi</em> may also be attributable to these differences. Unexpectedly, a set of genes required for the biosynthesis and degradation of peptidoglycan was identified in <em>O. tsutsugamushi</em>, although those for LPS biosynthesis were completely missing. In our preliminary proteome analysis, we confirmed that at least 10 of these peptidoglycan biosynthesis-related genes were expressed in the <em>O. tsutsugamushi</em> cells grown in L929 cells (data not shown).</p><p>Through the genomic comparison of <em>O. tsutsugamushi</em> and <em>R. prowazekii</em>, we also identified 195 <em>O. tsutsugamushi</em> singleton genes that are absent or degraded in <em>R. prowazekii</em> (referred to as ‘<em>O. tsutsugamushi</em>-specific’ genes), and 257 <em>R. prowazekii</em> genes that are absent or degraded in <em>O. tsutsugamushi</em> (referred to as ‘<em>R. prowazekii</em>-specific’ genes) (Table <a reveal-id="DSN011TB2" data-open="DSN011TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Of the 195 <em>O. tsutsugamushi</em>-specific genes, 161 are absent in all other sequenced <em>Rickettsia</em> species, and 94 genes exhibit no significant homology to known proteins. This finding suggests that a significant portion of the <em>O. tsutsugamushi</em>-specific genes have foreign origins. It is noteworthy that these genes are often clustered in the <em>O. tsutsugamushi</em> genome (data not shown), suggesting that they have been brought into <em>O. tsutsugamushi</em> as blocks. A prominent example is a genomic region about 19 kb in size located around the 550-kb position in the <em>O. tsutsugamushi</em> chromosome (Fig. <a reveal-id="DSN011F1" data-open="DSN011F1" class="link link-reveal link-table xref-fig">1</a>). This region exhibits a higher G + C composition than the average of the <em>O. tsutsugamushi</em> genome, and 23 <em>Orientia</em>-specific genes are accumulated. Two major surface antigens of <em>O. tsutsugamushi</em>, i.e. the 56- and 22-kDa proteins (OTT_0945 and OTT_1548, respectively), and a 56-kDa protein-like protein (OTT_0946) are also <em>O. tsutsugamushi</em> specific.</p><p>Among the 257 <em>R. prowazekii</em>-specific genes, 178 are conserved in all the <em>Rickettsia</em> species so far sequenced. Of the 178 genes, 139 have their orthologues in a wide range of alpha-Proteobacteria. Thus, at least these 139 genes can be regarded as genes that have been deleted or degraded in the lineage leading to <em>O. tsutsugamushi</em> after separation from the <em>Rickettsia</em> family. It is noteworthy that they include a significant number of genes for various housekeeping functions, such as the formation of a cell envelope, energy metabolism, fatty-acid metabolism, nucleotide metabolism, and DNA recombination and repair (indicated by red arrows in Fig. <a reveal-id="DSN011F5" data-open="DSN011F5" class="link link-reveal link-table xref-fig">5</a>; see also <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Several genes for RNA synthesis and modification, such as the RNA polymerase omega subunit and RNA helicase RhlE, those for stress response, such as the HslUV protease, ClpB chaperone, BipA GTPase and the OsmY protein, and those for drug sensitivity, such as the EmrA and EmrB proteins, are also specifically missing in <em>O. tsutsugamushi</em>. In addition, the genes for hemolysin A and patatin-like phospholipase, both of which are thought to be involved in the pathogenesis of <em>Rickettsia</em> species, are missing in <em>O. tsutsugamushi</em>.</p>                        <a id="5474370" scrollto-destination="5474370"></a>
<div id="DSN011F5" data-id="DSN011F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01105.jpeg?Expires=2147483647&Signature=LmZJg5sIoHYxPe8wya6hqSeQn-beREilCO5t2z3g7bqg-P9NNxjYF15oTZ~HlGw3S4Is5e8~m1Kcd6~4oSBeiI94QYpMo7LuT5rYJ6xbEct62YoR7NFedYhaovGySjFJbrIfu7qKBB2f2F69lI93t3AmMyiPSEJ9UaCTdmQZ8RRvJVQD8Q3atV8yY3op5Z9WXlKqeOf0X~2vU~oJRiqP04sDfaIN6GDxxk-Pm7tDiEXIkwfz1uBso7o2XbsCcXtCtI8FQV70w6AOxlgcYgdlRNARUhUkrpedX9B95ZB~4CvodvPsD~Bg2wNoLn9rZJSSDzW1yyHDxmEQz8ALdlXktw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Metabolic pathways of O. tsutsugamushi and R. prowazekii. The metabolic pathways of O. tsutsugamushi and R. prowazekii are compared. Pathways identified in both bacteria are indicated by black arrows, those identified in neither bacterium by gray broken-lined arrows, those identified only in O. tsutsugamushi by green arrows, and those identified only in R. prowazekii by red arrows. Pink arrows indicate the pathways that exhibit variable distribution in the genus Rickettsia but are absent in O. tsutsugamushi. Genes for DNA repair functions that were identified in O. tsutsugamushi and R. prowazekii are listed in the box." path-from-xml="dsn01105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474370/dsn01105.jpeg" path-from-xml="dsn01105.jpeg" target="_blank">View large</a><a section="5474370" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01105.jpeg?Expires=2147483647&Signature=jaRQ4QZWxlEeqz4qYMKvlGU4d6sM30IVJN-odsQrdGHkKnk1UWZBeY3eOQqMPsaEA0uwyNEhmvpGskugyMpuhpY-s4FPOH2PbYo5BRTT1Qi3QzFoeT6zdn0uoTNDHKE7DYICr1l1awAShatv1IvcGvkWK8OmZ7nwsgYqqmCEUXhJHwyuxRwa15cTN9CFzMPXZ3N5v425bncjcYpBZMKN7c2IqwvKUO4ldY0yGWnqgV-1SRbrxRssyX7a7muooyX3YbW-GvLyeYBK9JCQIbPMKkqrE3adVmRI5YZoTgbpqG~ZMqAmH3Gn81K458MFDW9RvESwNTqNGM17B7Ku7Ar5nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474370&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Metabolic pathways of <em>O. tsutsugamushi</em> and <em>R. prowazekii</em>. The metabolic pathways of <em>O. tsutsugamushi</em> and <em>R. prowazekii</em> are compared. Pathways identified in both bacteria are indicated by black arrows, those identified in neither bacterium by gray broken-lined arrows, those identified only in <em>O. tsutsugamushi</em> by green arrows, and those identified only in <em>R. prowazekii</em> by red arrows. Pink arrows indicate the pathways that exhibit variable distribution in the genus <em>Rickettsia</em> but are absent in <em>O. tsutsugamushi</em>. Genes for DNA repair functions that were identified in <em>O. tsutsugamushi</em> and <em>R. prowazekii</em> are listed in the box.</p></div></div><div content-id="DSN011F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01105.jpeg?Expires=2147483647&Signature=LmZJg5sIoHYxPe8wya6hqSeQn-beREilCO5t2z3g7bqg-P9NNxjYF15oTZ~HlGw3S4Is5e8~m1Kcd6~4oSBeiI94QYpMo7LuT5rYJ6xbEct62YoR7NFedYhaovGySjFJbrIfu7qKBB2f2F69lI93t3AmMyiPSEJ9UaCTdmQZ8RRvJVQD8Q3atV8yY3op5Z9WXlKqeOf0X~2vU~oJRiqP04sDfaIN6GDxxk-Pm7tDiEXIkwfz1uBso7o2XbsCcXtCtI8FQV70w6AOxlgcYgdlRNARUhUkrpedX9B95ZB~4CvodvPsD~Bg2wNoLn9rZJSSDzW1yyHDxmEQz8ALdlXktw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Metabolic pathways of O. tsutsugamushi and R. prowazekii. The metabolic pathways of O. tsutsugamushi and R. prowazekii are compared. Pathways identified in both bacteria are indicated by black arrows, those identified in neither bacterium by gray broken-lined arrows, those identified only in O. tsutsugamushi by green arrows, and those identified only in R. prowazekii by red arrows. Pink arrows indicate the pathways that exhibit variable distribution in the genus Rickettsia but are absent in O. tsutsugamushi. Genes for DNA repair functions that were identified in O. tsutsugamushi and R. prowazekii are listed in the box." path-from-xml="dsn01105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474370/dsn01105.jpeg" path-from-xml="dsn01105.jpeg" target="_blank">View large</a><a section="5474370" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01105.jpeg?Expires=2147483647&Signature=jaRQ4QZWxlEeqz4qYMKvlGU4d6sM30IVJN-odsQrdGHkKnk1UWZBeY3eOQqMPsaEA0uwyNEhmvpGskugyMpuhpY-s4FPOH2PbYo5BRTT1Qi3QzFoeT6zdn0uoTNDHKE7DYICr1l1awAShatv1IvcGvkWK8OmZ7nwsgYqqmCEUXhJHwyuxRwa15cTN9CFzMPXZ3N5v425bncjcYpBZMKN7c2IqwvKUO4ldY0yGWnqgV-1SRbrxRssyX7a7muooyX3YbW-GvLyeYBK9JCQIbPMKkqrE3adVmRI5YZoTgbpqG~ZMqAmH3Gn81K458MFDW9RvESwNTqNGM17B7Ku7Ar5nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474370&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Metabolic pathways of <em>O. tsutsugamushi</em> and <em>R. prowazekii</em>. The metabolic pathways of <em>O. tsutsugamushi</em> and <em>R. prowazekii</em> are compared. Pathways identified in both bacteria are indicated by black arrows, those identified in neither bacterium by gray broken-lined arrows, those identified only in <em>O. tsutsugamushi</em> by green arrows, and those identified only in <em>R. prowazekii</em> by red arrows. Pink arrows indicate the pathways that exhibit variable distribution in the genus <em>Rickettsia</em> but are absent in <em>O. tsutsugamushi</em>. Genes for DNA repair functions that were identified in <em>O. tsutsugamushi</em> and <em>R. prowazekii</em> are listed in the box.</p></div></div><p>Many genes for nucleotide metabolism, including those for <em>de novo</em> synthesis pathways, are absent in <em>O. tsutsugamushi</em>, as in <em>Rickettsia</em> species (Fig. <a reveal-id="DSN011F5" data-open="DSN011F5" class="link link-reveal link-table xref-fig">5</a>). The genes for CTP synthase and cytidylate kinase are additionally missing in <em>O. tsutsugamushi</em>. This may have rendered the bacterium more dependent on the host cell functions for the pyrimidine nucleotides. All these defects are probably compensated by the presence of five types of nucleotide transporters, as shown in <em>Chlamydia trachomatis</em>.<sup><a reveal-id="DSN011C50" data-open="DSN011C50" class="link link-ref link-reveal xref-bibr">50</a></sup> This may also be the case for the nicotinamido adenine dinucleotide (NAD) metabolism.<sup><a reveal-id="DSN011C51" data-open="DSN011C51" class="link link-ref link-reveal xref-bibr">51</a></sup></p><p>DNA recombination and repair systems are relatively well conserved in <em>Rickettsia</em> species (Fig. <a reveal-id="DSN011F5" data-open="DSN011F5" class="link link-reveal link-table xref-fig">5</a>). But a significant number of genes for these functions have specifically been deleted in <em>O. tsutsugamushi</em>, such as those for the UvrB, UvrC, RecN and RecG proteins and the transcription-repair coupling factor Mfd. This may be related to the acquisition and amplification of mobile genetic elements, as well as to the maintenance of an extremely repeat-rich genome.</p><p>Of particular interest is the lack of pyruvate dehydrogenase due to the deletion of the <em>pdhABC</em> genes and a second copy of <em>pdhD</em>. Other pathways generating acetyl-CoA are also missing (Fig. <a reveal-id="DSN011F5" data-open="DSN011F5" class="link link-reveal link-table xref-fig">5</a>). Thus, the bacterium appears to be unable to synthesize acetyl-CoA. This defect is probably linked to the incomplete tricarboxylic acid (TCA) cycle of the bacterium, where the citrate synthase and aconitate hydratase genes are inactivated by an authentic frame shift mutation and a premature stop codon, respectively (Fig. <a reveal-id="DSN011F5" data-open="DSN011F5" class="link link-reveal link-table xref-fig">5</a>). However, these defects may be bypassed by asparate aminotransferase, which catalyzes the oxaloacetate/alpha-ketoglutarate interconversion. Pyruvate may also be introduced into the TCA cycle by malate oxidoreductase. In addition, the glycoxylate cycle is missing in <em>O. tsutsugamushi</em>, as in <em>Rickettsia</em> species.</p><p>The inability to synthesize acetyl-CoA probably is further linked to the deletion of the genes for acetyl-CoA carboxylase, which catalyzes the transfer of carboxyl groups to acetyl-CoA to generate malonyl-CoA, the first step in fatty-acid biosynthesis (Fig. <a reveal-id="DSN011F5" data-open="DSN011F5" class="link link-reveal link-table xref-fig">5</a>). The genes for biotin-apoprotein ligase (<em>birA</em>) and biotin transporter (<em>bioY</em>), which are required for the formation of biotin carboxyl carrier protein, are also specifically missing in <em>O. tsutsugamushi</em>. In addition, the <em>fabH</em> gene encoding KAS III, which catalyzes the condensation of the acetyl-CoA starter unit with malonyl-ACP to yield acetoacetyl-CoA, the first step in the fatty-acid elongation pathway, is missing. Although the gene for KAS I is also absent as in <em>Rickettsia species</em>, the genes for malonyl-CoA:ACP transacylase and KAS II (<em>fabD</em> and <em>fabF</em>) are conserved. It is thus most likely that fatty-acid biosynthesis in <em>O. tsutsugamushi</em> is initiated by these two enzymes using host-derived malonyl-CoA.</p><p>Another distinguishing feature of <em>O. tsutsugamushi</em> is the lack of genes for lipoprotein biosynthesis. Although a gene for lipoprotein-specific leader peptidase (<em>lsp</em>) was identified, other genes for the modification and localization of lipoproteins (<em>lgt</em>, <em>lnt</em>, <em>lolA/B/C/E</em>/<em>D</em>) are missing (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). This may raise the possibility that <em>O. tsutsugamushi</em> lacks lipoproteins or has a very unique lipoprotein biosynthesis system. Consistent with this, of the five <em>O. tsutsugamushi</em> proteins whose homologues have been predicted to be lipoproteins in <em>Rickettsia species</em>, none contains the typical lipobox required for processing and lipid-modification of lipoproteins. This finding, together with the absence of LPS, suggests that a simple cell envelope may be advantageous to the intracellular life of <em>O. tsutsugamushi</em>.</p><p>All these genomic features of <em>O. tsutsugamushi</em> indicate that high levels of horizontal gene transfer (HGT) and gene amplification have occurred in <em>O. tsutsugamushi</em> after it diverged from the genus <em>Rickettsia</em>. On the other hand, <em>O. tsutsugamushi</em> has also undergone further reductive genome evolution and established an intracellular lifestyle that is more dependent on its host cell functions than <em>Rickettsia</em> species.</p>                            <h3 scrollto-destination=5474378 id="5474378" class="section-title" data-legacy-id=s3e>3.5. Repeat-containing proteins</h3>
<p>As briefly mentioned in Section 3.3, the <em>Ot</em>RG families include eight types of ankyrin repeat-containing proteins (AR proteins), four tetratricopeptide repeat-containing proteins (TPR proteins), and two novel repeat-containing proteins (Fig. <a reveal-id="DSN011F6" data-open="DSN011F6" class="link link-reveal link-table xref-fig">6</a>). We additionally identified 11 singleton genes encoding different types of AR proteins and five singleton genes encoding novel repeat-containing proteins. Ankyrin repeats have been found in many eukaryotic proteins and are known to mediate various protein–protein interactions.<sup><a reveal-id="DSN011C52" data-open="DSN011C52" class="link link-ref link-reveal xref-bibr">52</a></sup> However, they are rarely found in prokaryotes. Several bacteria, most of which are intracellular bacteria, such as <em>R. felis</em>, <em>R. bellii</em>, <em>W. pipientis</em>, <em>Legionella pneumonia</em>, and <em>Coxiella burnetii</em>, are exceptionally enriched in AR proteins (18–25 proteins), and these proteins are suspected to be involved in host–bacterial interactions. <em>Orientia tsutsugamushi</em> contains 20 types of genes for AR proteins, and the total number of copies (<a reveal-id="DSN011C46" data-open="DSN011C46" class="link link-ref link-reveal xref-bibr">46</a>) greatly exceeds those of other intracellular bacteria. As all these AR proteins have no signal sequence for the general secretion pathway, they could be candidates for the effecter proteins secreted by T4SS. And they may be involved in the modulation of host-cell functions by <em>O. tsutsugamushi</em>, which is required for the bacterium to develop and maintain a symbiotic relationship in mite cells and/or survive and propagate in mammals. Other repeat-containing proteins also do not possess apparent signal sequences, suggesting that they would also be T4SS effector candidates.</p>                        <a id="5474381" scrollto-destination="5474381"></a>
<div id="DSN011F6" data-id="DSN011F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01106.jpeg?Expires=2147483647&Signature=SKLlOkXMNDu4JAamgTUHXw5jGInVouPMoUCuQneE6C2gUykFBTTYu08QNXW9QMSzjwvjbYI4Vyzcg4JGQ2blWtKTlFLhkoBUxrQhtQFt8GS-iUNhVI6p8ikD2NeoP~DgXhpVzSayEhMs-XQNRPvvIroKtHwTX0BffG2WOuGh31Rmasnw95fKsi3ElAqLfuOEpJhGjMUs1T4MO--UU8nKoH7lfWPAXJrvFV9S86gad3aIPnNU2fITuBNzrU-p9ZTGEKoOhxwmMsXnE2MHqT~l99RN6Na7RIwFxHSJ5q-kWzhFf16H5aepvhzj00~ij9wRXm0lcIsozfqy7OPGXgUdlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repeat-containing proteins of O. tsutsugamushi. The repeat-containing proteins identified in the O. tsutsugamushi genome and their characteristics are shown. The structure of each repeat-containing protein is drawn to scale. Repeating units identified in each protein or family are depicted by colored boxes. The same repeating units are indicated by the same color." path-from-xml="dsn01106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474381/dsn01106.jpeg" path-from-xml="dsn01106.jpeg" target="_blank">View large</a><a section="5474381" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01106.jpeg?Expires=2147483647&Signature=bLs0ZuSNLUAUYDk4YI5bNxcRBT5IgrE9UWL~GIf70rPZph8R0tqytj27CS2VY4cj9vSJSMqqwUdvtuxrAsoCYliNbwurFpbgh0eXtBg75SC6guFrIzfiIfqRbB76hl2eqD92w8NMKdmcZ0ZQCZDATNyPV4c0NzLkZgi3YdyPIbquJ~MR08glKnv92NyhrfjZy4t-R~n7qhCSeCbE3JOjtkVdAPG7yYJuVpIfdaUbLMHENqjvAZAN93If~tsXjpxdqjJpDqT1u6hHKyAB~VhpTg2gYNjFhW-Sa1TDcBZYsbavrZercmuQnLJXkVRsHyPXVncPLLmgEw2tPYHMCRXqDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474381&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Repeat-containing proteins of <em>O. tsutsugamushi</em>. The repeat-containing proteins identified in the <em>O. tsutsugamushi</em> genome and their characteristics are shown. The structure of each repeat-containing protein is drawn to scale. Repeating units identified in each protein or family are depicted by colored boxes. The same repeating units are indicated by the same color.</p></div></div><div content-id="DSN011F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/m_dsn01106.jpeg?Expires=2147483647&Signature=SKLlOkXMNDu4JAamgTUHXw5jGInVouPMoUCuQneE6C2gUykFBTTYu08QNXW9QMSzjwvjbYI4Vyzcg4JGQ2blWtKTlFLhkoBUxrQhtQFt8GS-iUNhVI6p8ikD2NeoP~DgXhpVzSayEhMs-XQNRPvvIroKtHwTX0BffG2WOuGh31Rmasnw95fKsi3ElAqLfuOEpJhGjMUs1T4MO--UU8nKoH7lfWPAXJrvFV9S86gad3aIPnNU2fITuBNzrU-p9ZTGEKoOhxwmMsXnE2MHqT~l99RN6Na7RIwFxHSJ5q-kWzhFf16H5aepvhzj00~ij9wRXm0lcIsozfqy7OPGXgUdlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repeat-containing proteins of O. tsutsugamushi. The repeat-containing proteins identified in the O. tsutsugamushi genome and their characteristics are shown. The structure of each repeat-containing protein is drawn to scale. Repeating units identified in each protein or family are depicted by colored boxes. The same repeating units are indicated by the same color." path-from-xml="dsn01106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474381/dsn01106.jpeg" path-from-xml="dsn01106.jpeg" target="_blank">View large</a><a section="5474381" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn01106.jpeg?Expires=2147483647&Signature=bLs0ZuSNLUAUYDk4YI5bNxcRBT5IgrE9UWL~GIf70rPZph8R0tqytj27CS2VY4cj9vSJSMqqwUdvtuxrAsoCYliNbwurFpbgh0eXtBg75SC6guFrIzfiIfqRbB76hl2eqD92w8NMKdmcZ0ZQCZDATNyPV4c0NzLkZgi3YdyPIbquJ~MR08glKnv92NyhrfjZy4t-R~n7qhCSeCbE3JOjtkVdAPG7yYJuVpIfdaUbLMHENqjvAZAN93If~tsXjpxdqjJpDqT1u6hHKyAB~VhpTg2gYNjFhW-Sa1TDcBZYsbavrZercmuQnLJXkVRsHyPXVncPLLmgEw2tPYHMCRXqDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474381&ar=376012&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Repeat-containing proteins of <em>O. tsutsugamushi</em>. The repeat-containing proteins identified in the <em>O. tsutsugamushi</em> genome and their characteristics are shown. The structure of each repeat-containing protein is drawn to scale. Repeating units identified in each protein or family are depicted by colored boxes. The same repeating units are indicated by the same color.</p></div></div>                            <h3 scrollto-destination=5474385 id="5474385" class="section-title" data-legacy-id=s3f>3.6. Genome comparison with O. tsutsugamushi strain Boryong</h3>
<p>While we were preparing this report, the genome sequence of <em>O. tsutsugamushi</em> strain Boryong isolated in Korea was reported.<sup><a reveal-id="DSN011C14" data-open="DSN011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The genome size of strain Boryong (2 127 051 bp) is 118 kb longer than that of IKEDA, but the G + C content and the numbers of rRNA and tRNA genes are the same as those of IKEDA. Homology search of the 542 genes that are shared by strain IKEDA and <em>R. prowazekii</em> using BLASTN revealed that 540 genes are also conserved in strain Boryong. This indicates that the genome backbone is highly conserved between the two strains. The homologues exhibited high level of sequence conservation (97.5% identity on average) while notable sequence variations were observed in several genes for surface proteins.</p><p>Search of the mobile elements identified in IKEDA using BLASTN revealed that all the elements, including OtAGE, are also present and have explosively been amplified in strain Boryong. Although exact numbers of each element in Boryong are yet to be determined, this result suggests that these elements invaded into <em>O. tsutsugamushi</em> before the separation of the two strains. More importantly, the result of our preliminary dot-plot analysis of the two genomes suggested that intensive genome shuffling has also taken place between the two <em>O. tsutsugamushi</em> strains. Although more careful and intensive comparison of the two genomes is required, the amplified mobile elements seem to have been deeply involved in the genome shuffling.</p>                            <h3 scrollto-destination=5474390 id="5474390" class="section-title" data-legacy-id=s3g>3.7. Conclusion</h3>
<p>In <em>O. tsutsugamushi</em>, high level of gene loss has taken place like in other obligate intracellular bacteria, but massive amplification of various mobile elements has also taken place, which has induced intensive genome shuffling and generated a large number of repeated genes. Although the timing of the acquisition and/or amplification of these elements are yet to be elucidated, the extremely narrow population bottleneck created by the unique life cycle of the bacterium appears to have allowed such a very unique genome evolution. Thus, the genome sequence of <em>O. tsutsugamushi</em> strain Ikeda will be not only the genetic basis for a better understanding of the biology of the bacterium and pathogenesis of the ‘<em>Tsutsugamushi</em>’ disease, but also a very attractive material for studying the processes of genome evolution and the high plasticity of bacterial genomes.</p>                            <h2 scrollto-destination=5474395 id="5474395" class="section-title" data-legacy-id=s4>Funding</h2>
<p>This work was supported by the Research for the Future Program of the Japan Society for the Promotion of Science (JSPS-RFTF00L01411) and Grants-in-Aid from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=5474398 id="5474398" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5474400 id="5474400" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Eiichi Ohtsubo and Kenji Ichiyanagi for their help in our analysis of the IS elements and GII introns. We also thank Seigo Yamamoto, Yoshiro Terawaki and Hiroshi Yoshikawa for their advice and encouragement, Hidehiro Toh, Akemi Yoshida, Yumiko Takeshita, Noriko Kanemaru and Nobuko Fujii for their technical assistance, and Yumiko Hayashi for her linguistic assistance.</p>                            <h2 scrollto-destination=5474402 id="5474402" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN011C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Classification of <em>Rickettsia tsutsugamushi</em> in a new genus, <em>Orientia</em> gen. nov., as <em>Orientia tsutsugamushi</em> comb. nov</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1995</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">589</div>-<div class="lpage">591</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classification%20of%20Rickettsia%20tsutsugamushi%20in%20a%20new%20genus%2C%20Orientia%20gen.%20nov.%2C%20as%20Orientia%20tsutsugamushi%20comb.%20nov&amp;author=A.%20Tamura&amp;author=N.%20Ohashi&amp;author=H.%20Urakami&amp;publication_year=1995&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=45&amp;pages=589-591" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-45-3-589" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-45-3-589" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-45-3-589"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8590688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bandopadhyay</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Vertical transmission in chigger borne rickettsiosis</div>, <div class="source">Indian J. Med. Res.</div> , <div class="year">1978</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">31</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vertical%20transmission%20in%20chigger%20borne%20rickettsiosis&amp;author=J.%20Rai&amp;author=D.%20Bandopadhyay&amp;publication_year=1978&amp;journal=Indian%20J.%20Med.%20Res.&amp;volume=68&amp;pages=31-38" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/581279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rapmund</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Dohany</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Manikumaran</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Transovarial transmission of <em>Rickettsia tsutsugamushi</em> in <em>Leptotrombidium</em> (<em>Leptotrombidium</em>) <em>arenicola Traub</em> (Acarina: Trombiculidae)</div>, <div class="source">J. Med. Entomol.</div> , <div class="year">1972</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">71</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transovarial%20transmission%20of%20Rickettsia%20tsutsugamushi%20in%20Leptotrombidium%20(Leptotrombidium)%20arenicola%20Traub%20(Acarina%3A%20Trombiculidae)&amp;author=G.%20Rapmund&amp;author=A.%20L.%20Dohany&amp;author=C.%20Manikumaran&amp;publication_year=1972&amp;journal=J.%20Med.%20Entomol.&amp;volume=9&amp;pages=71-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jmedent/9.1.71" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjmedent%2F9.1.71" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjmedent%2F9.1.71"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4623054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rapmund</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Upham</div> <div class="given-names">R. W.</div>Jr.</div>,  <div class="name"><div class="surname">Kundin</div> <div class="given-names">W. D.</div></div>, et al. </div><div class="article-title">Transovarial development of scrub typhus rickettsiae in a colony of vector mites</div>, <div class="source">Trans. R. Soc. Trop. Med. Hyg.</div> , <div class="year">1969</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">251</div>-<div class="lpage">258</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transovarial%20development%20of%20scrub%20typhus%20rickettsiae%20in%20a%20colony%20of%20vector%20mites&amp;author=G.%20Rapmund&amp;author=R.%20W.%20Upham&amp;author=W.%20D.%20Kundin&amp;publication_year=1969&amp;journal=Trans.%20R.%20Soc.%20Trop.%20Med.%20Hyg.&amp;volume=63&amp;pages=251-258" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0035-9203(69)90155-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0035-9203(69)90155-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0035-9203(69)90155-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4978468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nogami</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Transovarial transmission of <em>Rickettsia tsutsugamushi</em> in <em>Leptotrombidium pallidum</em> successively reared in the laboratory</div>, <div class="source">Jpn. J. Exp. Med.</div> , <div class="year">1988</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">213</div>-<div class="lpage">218</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transovarial%20transmission%20of%20Rickettsia%20tsutsugamushi%20in%20Leptotrombidium%20pallidum%20successively%20reared%20in%20the%20laboratory&amp;author=M.%20Takahashi&amp;author=M.%20Murata&amp;author=S.%20Nogami&amp;publication_year=1988&amp;journal=Jpn.%20J.%20Exp.%20Med.&amp;volume=58&amp;pages=213-218" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3149693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takahasi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hori</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">An ultrastructural study of vertical transmission of <em>Rickettsia tsutsugamushi</em> during oogenesis and spermatogenesis in <em>Leptortombidium pallidum</em></div>, <div class="source">Am. J. Trop. Med. Hyg.</div> , <div class="year">1994</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">219</div>-<div class="lpage">228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ultrastructural%20study%20of%20vertical%20transmission%20of%20Rickettsia%20tsutsugamushi%20during%20oogenesis%20and%20spermatogenesis%20in%20Leptortombidium%20pallidum&amp;author=H.%20Urakami&amp;author=M.%20Takahasi&amp;author=E.%20Hori&amp;publication_year=1994&amp;journal=Am.%20J.%20Trop.%20Med.%20Hyg.&amp;volume=50&amp;pages=219-228" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8116816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Misumi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Failed vertical transmission of <em>Rickettsia tsutsugamushi</em> (Rickettsiales: Rickettsiaceae) acquired from rickettsemic mice by <em>Leptotrombidium pallidum</em> (Acari: trombiculidae)</div>, <div class="source">J. Med. Entomol.</div> , <div class="year">1994</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">212</div>-<div class="lpage">216</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Failed%20vertical%20transmission%20of%20Rickettsia%20tsutsugamushi%20(Rickettsiales%3A%20Rickettsiaceae)%20acquired%20from%20rickettsemic%20mice%20by%20Leptotrombidium%20pallidum%20(Acari%3A%20trombiculidae)&amp;author=M.%20Takahashi&amp;author=M.%20Murata&amp;author=H.%20Misumi&amp;publication_year=1994&amp;journal=J.%20Med.%20Entomol.&amp;volume=31&amp;pages=212-216" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jmedent/31.2.212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjmedent%2F31.2.212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjmedent%2F31.2.212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8189412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walker</div> <div class="given-names">J. S.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">C. T.</div></div>,  <div class="name"><div class="surname">Manikumaran</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Attempts to infect and demonstrate transovarial transmission of <em>R. tsutsugamushi</em> in three species of <em>Leptotrombidium</em> mites</div>, <div class="source">Ann. N Y. Acad. Sci.</div> , <div class="year">1975</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">80</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Attempts%20to%20infect%20and%20demonstrate%20transovarial%20transmission%20of%20R.%20tsutsugamushi%20in%20three%20species%20of%20Leptotrombidium%20mites&amp;author=J.%20S.%20Walker&amp;author=C.%20T.%20Chan&amp;author=C.%20Manikumaran&amp;publication_year=1975&amp;journal=Ann.%20N%20Y.%20Acad.%20Sci.&amp;volume=266&amp;pages=80-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nyas.1975.266.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnyas.1975.266.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnyas.1975.266.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/829478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Traub</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wisseman</div> <div class="given-names">C. L.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">M. R.</div></div>, et al. </div><div class="article-title">The acquisition of <em>Rickettsia tsutsugamushi</em> by chiggers (trombiculid mites) during the feeding process</div>, <div class="source">Ann. N Y. Acad. Sci.</div> , <div class="year">1975</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">91</div>-<div class="lpage">114</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20acquisition%20of%20Rickettsia%20tsutsugamushi%20by%20chiggers%20(trombiculid%20mites)%20during%20the%20feeding%20process&amp;author=R.%20Traub&amp;author=C.%20L.%20Wisseman&amp;author=M.%20R.%20Jones&amp;publication_year=1975&amp;journal=Ann.%20N%20Y.%20Acad.%20Sci.&amp;volume=266&amp;pages=91-114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nyas.1975.266.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnyas.1975.266.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnyas.1975.266.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/829479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watt</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Parola</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Scrub typhus and tropical rickettsioses</div>, <div class="source">Curr. Opin. Infect. Dis.</div> , <div class="year">2003</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">429</div>-<div class="lpage">436</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scrub%20typhus%20and%20tropical%20rickettsioses&amp;author=G.%20Watt&amp;author=P.%20Parola&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Infect.%20Dis.&amp;volume=16&amp;pages=429-436" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1097/00001432-200310000-00009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1097%2F00001432-200310000-00009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1097%2F00001432-200310000-00009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14501995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Enatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Phylogenetic analysis of <em>Orientia tsutsugamushi</em> strains based on the sequence homologies of 56-kDa type-specific antigen genes</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">163</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20Orientia%20tsutsugamushi%20strains%20based%20on%20the%20sequence%20homologies%20of%2056-kDa%20type-specific%20antigen%20genes&amp;author=T.%20Enatsu&amp;author=H.%20Urakami&amp;author=A.%20Tamura&amp;publication_year=1999&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=15&amp;pages=163-169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.1999.180.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.1999.180.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.1999.180.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSN011C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Phylogenetic characterization of <em>Orientia tsutsugamushi</em> isolated in Taiwan according to the sequence homologies of 56-kDa type-specific antigen genes</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">2003</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">577</div>-<div class="lpage">583</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20characterization%20of%20Orientia%20tsutsugamushi%20isolated%20in%20Taiwan%20according%20to%20the%20sequence%20homologies%20of%2056-kDa%20type-specific%20antigen%20genes&amp;author=Y.%20Qiang&amp;author=A.%20Tamura&amp;author=H.%20Urakami&amp;publication_year=2003&amp;journal=Microbiol.%20Immunol.&amp;volume=47&amp;pages=577-583" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.2003.47.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.2003.47.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.2003.47.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14524618" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Koyama</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Epidemiological survey of <em>Orientia tsutsugamushi</em> distribution in field rodents in Saitama Prefecture, Japan, and discovery of a new type</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">2001</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">439</div>-<div class="lpage">446</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiological%20survey%20of%20Orientia%20tsutsugamushi%20distribution%20in%20field%20rodents%20in%20Saitama%20Prefecture%2C%20Japan%2C%20and%20discovery%20of%20a%20new%20type&amp;author=A.%20Tamura&amp;author=N.%20Yamamoto&amp;author=S.%20Koyama&amp;publication_year=2001&amp;journal=Microbiol.%20Immunol.&amp;volume=45&amp;pages=439-446" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.2001.45.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.2001.45.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.2001.45.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11497219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">N. H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">H. R.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J. H.</div></div>, et al. </div><div class="article-title">The <em>Orientia tsutsugamushi</em> genome reveals massive proliferation of conjugative type IV secretion system and host–cell interaction genes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">7981</div>-<div class="lpage">7986</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Orientia%20tsutsugamushi%20genome%20reveals%20massive%20proliferation%20of%20conjugative%20type%20IV%20secretion%20system%20and%20host%E2%80%93cell%20interaction%20genes&amp;author=N.%20H.%20Cho&amp;author=H.%20R.%20Kim&amp;author=J.%20H.%20Lee&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=8&amp;pages=7981-7986" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611553104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611553104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611553104"> </span></div> </div></div></div></div></div><div content-id="DSN011C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Koyama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Demonstration of antigenic and genotypic variation in <em>Orientia tsutsugamushi</em> which were isolated in Japan, and their classification into type and subtype</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1996</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">627</div>-<div class="lpage">638</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Demonstration%20of%20antigenic%20and%20genotypic%20variation%20in%20Orientia%20tsutsugamushi%20which%20were%20isolated%20in%20Japan%2C%20and%20their%20classification%20into%20type%20and%20subtype&amp;author=N.%20Ohashi&amp;author=Y.%20Koyama&amp;author=H.%20Urakami&amp;publication_year=1996&amp;journal=Microbiol.%20Immunol.&amp;volume=40&amp;pages=627-638" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1996.40.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1996.40.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1996.40.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8908607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawamori</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Akiyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sugieda</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Epidemiology of Tsutsugamushi disease in relation to the serotypes of <em>Rickettsia tsutsugamushi</em> isolated from patients, field mice, and unfed chiggers on the eastern slope of Mount Fuji, Shizuoka Prefecture, Japan</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="year">1992</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">2842</div>-<div class="lpage">2846</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiology%20of%20Tsutsugamushi%20disease%20in%20relation%20to%20the%20serotypes%20of%20Rickettsia%20tsutsugamushi%20isolated%20from%20patients%2C%20field%20mice%2C%20and%20unfed%20chiggers%20on%20the%20eastern%20slope%20of%20Mount%20Fuji%2C%20Shizuoka%20Prefecture%2C%20Japan&amp;author=F.%20Kawamori&amp;author=M.%20Akiyama&amp;author=M.%20Sugieda&amp;publication_year=1992&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=30&amp;pages=2842-2846" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1452653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kitaoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okubo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asanuma</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Epidemiological survey by means of complement fixation test on scrub typhus in Japan</div>, <div class="source">Acta. Med. Biol (Niigata).</div> , <div class="year">1967</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">69</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiological%20survey%20by%20means%20of%20complement%20fixation%20test%20on%20scrub%20typhus%20in%20Japan&amp;author=M.%20Kitaoka&amp;author=K.%20Okubo&amp;author=K.%20Asanuma&amp;publication_year=1967&amp;journal=Acta.%20Med.%20Biol%20(Niigata).&amp;volume=15&amp;pages=69-85" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4968953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shishido</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Strain variation of <em>rickettsia orientalis</em> in the complement fixation test</div>, <div class="source">Jpn. J. Med. Sci. Biol.</div> , <div class="year">1964</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">59</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strain%20variation%20of%20rickettsia%20orientalis%20in%20the%20complement%20fixation%20test&amp;author=A.%20Shishido&amp;publication_year=1964&amp;journal=Jpn.%20J.%20Med.%20Sci.%20Biol.&amp;volume=17&amp;pages=59-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7883/yoken1952.17.59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7883%2Fyoken1952.17.59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7883%2Fyoken1952.17.59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14206509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsuruhara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Electron microscopic studies on intracellular multiplication of <em>Rickettsia tsutsugamushi</em> in L cells</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1984</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1191</div>-<div class="lpage">1201</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Electron%20microscopic%20studies%20on%20intracellular%20multiplication%20of%20Rickettsia%20tsutsugamushi%20in%20L%20cells&amp;author=H.%20Urakami&amp;author=T.%20Tsuruhara&amp;author=A.%20Tamura&amp;publication_year=1984&amp;journal=Microbiol.%20Immunol.&amp;volume=28&amp;pages=1191-1201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1984.28.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1984.28.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1984.28.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6441107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Uramaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsuruhara</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Purification of <em>Rickettsia tsutsugamushi</em> by Percoll density gradient centrifugation</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1982</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">321</div>-<div class="lpage">328</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20of%20Rickettsia%20tsutsugamushi%20by%20Percoll%20density%20gradient%20centrifugation&amp;author=A.%20Tamura&amp;author=H.%20Uramaki&amp;author=T.%20Tsuruhara&amp;publication_year=1982&amp;journal=Microbiol.%20Immunol.&amp;volume=26&amp;pages=321-328" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1982.26.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1982.26.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1982.26.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6287181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Minamishima</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">DNA base composition of <em>Rickettsia tsutsugamushi</em> determined by reversed-phase high-performance liquid chromatography</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1991</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">247</div>-<div class="lpage">248</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20base%20composition%20of%20Rickettsia%20tsutsugamushi%20determined%20by%20reversed-phase%20high-performance%20liquid%20chromatography&amp;author=K.%20Kumura&amp;author=Y.%20Minamishima&amp;author=S.%20Yamamoto&amp;author=N.%20Ohashi&amp;author=A.%20Tamura&amp;publication_year=1991&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=41&amp;pages=247-248" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-41-2-247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-41-2-247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-41-2-247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1906730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using <em>phred.</em> II. Error probabilities</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">186</div>-<div class="lpage">194</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities&amp;author=B.%20Ewing&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=186-194" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gordon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Abajian</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title"><em>Consed:</em> a graphical tool for sequence finishing</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">195</div>-<div class="lpage">202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consed%3A%20a%20graphical%20tool%20for%20sequence%20finishing&amp;author=D.%20Gordon&amp;author=C.%20Abajian&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=195-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kuhara</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Chromosome of the enterohemorrhagic <em>Escherichia coli</em> O157:H7; comparative analysis with K-12 MG1655 revealed the acquisition of a large amount of foreign DNAs</div>, <div class="source">DNA Res.</div> , <div class="year">1999</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">361</div>-<div class="lpage">368</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20of%20the%20enterohemorrhagic%20Escherichia%20coli%20O157%3AH7%3B%20comparative%20analysis%20with%20K-12%20MG1655%20revealed%20the%20acquisition%20of%20a%20large%20amount%20of%20foreign%20DNAs&amp;author=M.%20Ohnishi&amp;author=C.%20Tanaka&amp;author=S.%20Kuhara&amp;publication_year=1999&amp;journal=DNA%20Res.&amp;volume=31&amp;pages=361-368" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/6.6.361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F6.6.361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F6.6.361"> </span></div> </div></div></div></div></div><div content-id="DSN011C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Terajima</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genomic diversity of enterohemorrhagic <em>Escherichia coli</em> O157 revealed by whole genome PCR scanning</div>, <div class="source">Proc. Natl Acad. Sci. USA.</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">17043</div>-<div class="lpage">17048</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20diversity%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%20revealed%20by%20whole%20genome%20PCR%20scanning&amp;author=M.%20Ohnishi&amp;author=J.%20Terajima&amp;author=K.%20Kurokawa&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA.&amp;volume=99&amp;pages=17043-17048" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.262441699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.262441699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.262441699"> </span></div> </div></div></div></div></div><div content-id="DSN011C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakiyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">An automated system for genome analysis to support microbial whole-genome shotgun sequencing</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">670</div>-<div class="lpage">673</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20automated%20system%20for%20genome%20analysis%20to%20support%20microbial%20whole-genome%20shotgun%20sequencing&amp;author=T.%20Sakiyama&amp;author=H.%20Takami&amp;author=N.%20Ogasawara&amp;publication_year=2000&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=64&amp;pages=670-673" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.64.670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.64.670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.64.670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10803979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>. </div><div class="article-title">MBGD: a platform for microbial comparative genomics based on the automated construction of orthologous groups</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> <div class="issue">Database issue</div>(pg. <div class="fpage">D343</div>-<div class="lpage">D346</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MBGD%3A%20a%20platform%20for%20microbial%20comparative%20genomics%20based%20on%20the%20automated%20construction%20of%20orthologous%20groups&amp;author=I.%20Uchiyama&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D343-D346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17135196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">W. R.</div></div>. </div><div class="article-title">Flexible sequence similarity searching with the FASTA3 program package</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">185</div>-<div class="lpage">219</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flexible%20sequence%20similarity%20searching%20with%20the%20FASTA3%20program%20package&amp;author=W.%20R.%20Pearson&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=185-219" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">W. R.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Improved tools for biological sequence comparison</div>, <div class="source">Proc. Natl Acad. Sci. USA.</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2444</div>-<div class="lpage">2448</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20tools%20for%20biological%20sequence%20comparison&amp;author=W.%20R.%20Pearson&amp;author=D.%20J.%20Lipman&amp;publication_year=1988&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA.&amp;volume=85&amp;pages=2444-2448" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2444"> </span></div> </div></div></div></div></div><div content-id="DSN011C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A database for post-genome analysis</div>, <div class="source">Trends Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">375</div>-<div class="lpage">376</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20database%20for%20post-genome%20analysis&amp;author=M.%20Kanehisa&amp;publication_year=1997&amp;journal=Trends%20Genet.&amp;volume=13&amp;pages=375-376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(97)01223-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(97)01223-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(97)01223-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9287494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grigoriev</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Analyzing genomes with cumulative skew diagrams</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">2286</div>-<div class="lpage">2290</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20genomes%20with%20cumulative%20skew%20diagrams&amp;author=A.%20Grigoriev&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=15&amp;pages=2286-2290" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.10.2286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.10.2286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.10.2286"> </span></div> </div></div></div></div></div><div content-id="DSN011C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">La Scola</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome sequence of <em>Rickettsia bellii</em> illuminates the role of Amoebae in gene exchanges between intracellular pathogens</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">733</div>-<div class="lpage">744</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20Rickettsia%20bellii%20illuminates%20the%20role%20of%20Amoebae%20in%20gene%20exchanges%20between%20intracellular%20pathogens&amp;author=H.%20Ogata&amp;author=B.%20La%20Scola&amp;author=S.%20Audic&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=733-744" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020076"> </span></div> </div></div></div></div></div><div content-id="DSN011C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Renesto</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome sequence of <em>Rickettsia felis</em> identifies the first putative conjugative plasmid in an obligate intracellular parasite</div>, <div class="source">PLoS Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Rickettsia%20felis%20identifies%20the%20first%20putative%20conjugative%20plasmid%20in%20an%20obligate%20intracellular%20parasite&amp;author=H.%20Ogata&amp;author=P.%20Renesto&amp;author=S.%20Audic&amp;publication_year=2005&amp;journal=PLoS%20Biol.&amp;volume=3&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030248"> </span></div> </div></div></div></div></div><div content-id="DSN011C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Renesto-Audiffren</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Mechanisms of evolution in <em>Rickettsia conorii</em> and <em>R. prowazekii</em></div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">293</div> (pg. <div class="fpage">2093</div>-<div class="lpage">2098</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20evolution%20in%20Rickettsia%20conorii%20and%20R.%20prowazekii&amp;author=H.%20Ogata&amp;author=S.%20Audic&amp;author=P.%20Renesto-Audiffren&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=2093-2098" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1061471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1061471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1061471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11557893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson</div> <div class="given-names">S. G.</div></div>,  <div class="name"><div class="surname">Zomorodipour</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">J. O.</div></div>, et al. </div><div class="article-title">The genome sequence of <em>Rickettsia prowazekii</em> and the origin of mitochondria</div>, <div class="source">Nature</div> , <div class="year">1998</div>, vol. <div class="volume">396</div> (pg. <div class="fpage">133</div>-<div class="lpage">140</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Rickettsia%20prowazekii%20and%20the%20origin%20of%20mitochondria&amp;author=S.%20G.%20Andersson&amp;author=A.%20Zomorodipour&amp;author=J.%20O.%20Andersson&amp;publication_year=1998&amp;journal=Nature&amp;volume=396&amp;pages=133-140" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/24094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F24094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F24094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9823893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLeod</div> <div class="given-names">M. P.</div></div>,  <div class="name"><div class="surname">Qin</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Karpathy</div> <div class="given-names">S. E.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Rickettsia typhi</em> and comparison with sequences of other <em>rickettsiae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">5842</div>-<div class="lpage">5855</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Rickettsia%20typhi%20and%20comparison%20with%20sequences%20of%20other%20rickettsiae&amp;author=M.%20P.%20McLeod&amp;author=X.%20Qin&amp;author=S.%20E.%20Karpathy&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=5842-5855" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.17.5842-5855.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.17.5842-5855.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.17.5842-5855.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15317790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">L. V.</div></div>,  <div class="name"><div class="surname">Vamathevan</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Phylogenomics of the reproductive parasite <em>Wolbachia pipientis</em> wMel: a streamlined genome overrun by mobile genetic elements</div>, <div class="source">PLoS Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">327</div>-<div class="lpage">341</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics%20of%20the%20reproductive%20parasite%20Wolbachia%20pipientis%20wMel%3A%20a%20streamlined%20genome%20overrun%20by%20mobile%20genetic%20elements&amp;author=M.%20Wu&amp;author=L.%20V.%20Sun&amp;author=J.%20Vamathevan&amp;publication_year=2004&amp;journal=PLoS%20Biol.&amp;volume=2&amp;pages=327-341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020327"> </span></div> </div></div></div></div></div><div content-id="DSN011C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burrus</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Waldor</div> <div class="given-names">M. K.</div></div>. </div><div class="article-title">Shaping bacterial genomes with integrative and conjugative elements</div>, <div class="source">Res. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">376</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shaping%20bacterial%20genomes%20with%20integrative%20and%20conjugative%20elements&amp;author=V.%20Burrus&amp;author=M.%20K.%20Waldor&amp;publication_year=2004&amp;journal=Res.%20Microbiol.&amp;volume=155&amp;pages=376-386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.resmic.2004.01.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.resmic.2004.01.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.resmic.2004.01.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15207870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blanc</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Robert</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Lateral gene transfer between obligate intracellular bacteria: evidence from the <em>Rickettsia massiliae</em> genome</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1657</div>-<div class="lpage">1664</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lateral%20gene%20transfer%20between%20obligate%20intracellular%20bacteria%3A%20evidence%20from%20the%20Rickettsia%20massiliae%20genome&amp;author=G.%20Blanc&amp;author=H.%20Ogata&amp;author=C.%20Robert&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1657-1664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6742107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6742107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6742107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17916642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Audette</div> <div class="given-names">G. F.</div></div>,  <div class="name"><div class="surname">Manchak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Beatty</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Entry exclusion in F-like plasmids requires intact TraG in the donor that recognizes its cognate TraS in the recipient</div>, <div class="source">Microbiology</div> , <div class="year">2007</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">442</div>-<div class="lpage">451</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Entry%20exclusion%20in%20F-like%20plasmids%20requires%20intact%20TraG%20in%20the%20donor%20that%20recognizes%20its%20cognate%20TraS%20in%20the%20recipient&amp;author=G.%20F.%20Audette&amp;author=J.%20Manchak&amp;author=P.%20Beatty&amp;publication_year=2007&amp;journal=Microbiology&amp;volume=153&amp;pages=442-451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2006/001917-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2006%2F001917-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2006%2F001917-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17259615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sukupolvi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">O'Connor</div> <div class="given-names">C. D.</div></div>. </div><div class="article-title">TraT lipoprotein, a plasmid-specified mediator of interactions between gram-negative bacteria and their environment</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1990</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">331</div>-<div class="lpage">341</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TraT%20lipoprotein%2C%20a%20plasmid-specified%20mediator%20of%20interactions%20between%20gram-negative%20bacteria%20and%20their%20environment&amp;author=S.%20Sukupolvi&amp;author=C.%20D.%20O'Connor&amp;publication_year=1990&amp;journal=Microbiol.%20Rev.&amp;volume=54&amp;pages=331-341" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2087219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Horn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Collingro</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schmitz-Esser</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Illuminating the evolutionary history of chlamydiae</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">728</div>-<div class="lpage">730</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Illuminating%20the%20evolutionary%20history%20of%20chlamydiae&amp;author=M.%20Horn&amp;author=A.%20Collingro&amp;author=S.%20Schmitz-Esser&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=728-730" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1096330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1096330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1096330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15073324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Jian</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xiaobing</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Genome dynamics and diversity of <em>Shigella</em> species, the etiologic agents of bacillary dysentery</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">6445</div>-<div class="lpage">6458</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20dynamics%20and%20diversity%20of%20Shigella%20species%2C%20the%20etiologic%20agents%20of%20bacillary%20dysentery&amp;author=Y.%20Fan&amp;author=Y.%20Jian&amp;author=Z.%20Xiaobing&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=6445-6458" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16275786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larsson</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Oyston</div> <div class="given-names">P. C.</div></div>,  <div class="name"><div class="surname">Chain</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The complete genome sequence of <em>Francisella tularensis</em>, the causative agent of tularemia</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">153</div>-<div class="lpage">159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Francisella%20tularensis%2C%20the%20causative%20agent%20of%20tularemia&amp;author=P.%20Larsson&amp;author=P.%20C.%20Oyston&amp;author=P.%20Chain&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=153-159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1499"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15640799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">Plant transposable elements: where genetics meets genomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">329</div>-<div class="lpage">341</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transposable%20elements%3A%20where%20genetics%20meets%20genomics&amp;author=C.%20Feschotte&amp;author=N.%20Jiang&amp;author=S.%20R.%20Wessler&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=329-341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Complete genome structure of the thermophilic cyanobacterium <em>Thermosynechococcus elongatus</em> BP-1</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">123</div>-<div class="lpage">130</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20structure%20of%20the%20thermophilic%20cyanobacterium%20Thermosynechococcus%20elongatus%20BP-1&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;author=S.%20Sato&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=31&amp;pages=123-130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.4.123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.4.123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.4.123"> </span></div> </div></div></div></div></div><div content-id="DSN011C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Deficiency of peptidoglycan and lipopolysaccharide components in <em>Rickettsia tsutsugamushi</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">1987</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">2290</div>-<div class="lpage">2292</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deficiency%20of%20peptidoglycan%20and%20lipopolysaccharide%20components%20in%20Rickettsia%20tsutsugamushi&amp;author=K.%20Amano&amp;author=A.%20Tamura&amp;author=N.%20Ohashi&amp;publication_year=1987&amp;journal=Infect.%20Immun.&amp;volume=55&amp;pages=2290-2292" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3114150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Silverman</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Wisseman</div> <div class="given-names">C. L.</div></div>. </div><div class="article-title">Comparative ultrastructural study on the cell envelopes of <em>Rickettsia prowazekii</em>, <em>Rickettsia rickettsii</em>, and <em>Rickettsia tsutsugamushi</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">1978</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1020</div>-<div class="lpage">1023</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20ultrastructural%20study%20on%20the%20cell%20envelopes%20of%20Rickettsia%20prowazekii%2C%20Rickettsia%20rickettsii%2C%20and%20Rickettsia%20tsutsugamushi&amp;author=D.%20J.%20Silverman&amp;author=C.%20L.%20Wisseman&amp;publication_year=1978&amp;journal=Infect.%20Immun.&amp;volume=21&amp;pages=1020-1023" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/101465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tjaden</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Winkler</div> <div class="given-names">H. H.</div></div>,  <div class="name"><div class="surname">Schwöppe</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Two nucleotide transport proteins in <em>Chlamydia trachomatis</em>, One for net nucleoside triphosphate uptake and the other for transport of energy</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">1196</div>-<div class="lpage">1202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20nucleotide%20transport%20proteins%20in%20Chlamydia%20trachomatis%2C%20One%20for%20net%20nucleoside%20triphosphate%20uptake%20and%20the%20other%20for%20transport%20of%20energy&amp;author=J.%20Tjaden&amp;author=H.%20H.%20Winkler&amp;author=C.%20Schw%C3%B6ppe&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=1196-1202" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9973346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haferkamp</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Schmitz-Esser</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Linka</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A candidate NAD+ transporter in an intracellular bacterial symbiont related to Chlamydiae</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">432</div> (pg. <div class="fpage">622</div>-<div class="lpage">625</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20candidate%20NAD%2B%20transporter%20in%20an%20intracellular%20bacterial%20symbiont%20related%20to%20Chlamydiae&amp;author=I.%20Haferkamp&amp;author=S.%20Schmitz-Esser&amp;author=N.%20Linka&amp;publication_year=2004&amp;journal=Nature&amp;volume=432&amp;pages=622-625" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15577910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mahajan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tsai</div> <div class="given-names">M. D.</div></div>. </div><div class="article-title">Ankyrin repeat: a unique motif mediating protein–protein interactions</div>, <div class="source">Biochemistry</div> , <div class="year">2006</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">15168</div>-<div class="lpage">15178</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ankyrin%20repeat%3A%20a%20unique%20motif%20mediating%20protein%E2%80%93protein%20interactions&amp;author=J.%20Li&amp;author=A.%20Mahajan&amp;author=M.%20D.%20Tsai&amp;publication_year=2006&amp;journal=Biochemistry&amp;volume=45&amp;pages=15168-15178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi062188q" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi062188q" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi062188q"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17176038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN011C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN011C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment</div>, <div class="source">Brief. Bioinform.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">150</div>-<div class="lpage">163</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA3%3A%20integrated%20software%20for%20molecular%20evolutionary%20genetics%20analysis%20and%20sequence%20alignment&amp;author=S.%20Kumar&amp;author=K.%20Tamura&amp;author=M.%20Nei&amp;publication_year=2004&amp;journal=Brief.%20Bioinform.&amp;volume=5&amp;pages=150-163" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2F5.2.150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2F5.2.150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15260895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=376012&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn011-dsn011_Tables.xls?Expires=2147483647&amp;Signature=rkSFErCIoCsNL-dxap5TqEFYv-qtgeU~mUGXmm93Flafx5rpDd~bseonqNlfoMw5cYxfqW86nBOG0inX9IgHHJUehnbAyvlAa--k13aF2v31JbchmVlgwwjo9GrHrxXzqX8MUAi6jjRna2ieUWiDBam-D13JFE4SgGrb~RUdJG3Z~A6jplfYxrOHT2dgEdjZj0BUyAydzTeBTb3eTRdaI3KIM11eIgsTT6RvB1bXeCa7FbrmZfXTO3Ek6Fo51QxhRfQ4qcVt~38mM3dcxKIQiFafbtBjRm9-4eU0GuGOtD8SH6a2K4mDLP9h6VDbYx7jO6ccYk-G-s6P5M4Wze-5xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn011/2/dsn011-dsn011_Figures.pdf?Expires=2147483647&amp;Signature=ufRYGXffv3bXjT2I1qgvR65jq8mm5LBcjFb3-ap1~Zo709f8z6c9RVo7ZTVPZqqjjUhmvPK5-HJIoBUgCFt0NnAc3IQfaFDTPj8rJEoRSmEDB3FjaUUw~UvQG~StlG0ZUpzcy5MWy6mvrAQsn7htBuQ4mV7rvpvL8lSvnjaIJhV7NlUGyMEIQS~djnti6JsoczDdWdda3ZHdaxOd0ayj0hNS9k2mH-uvl5mnWWbHGZ~~KgomcH9I5DDupiVjKCYVejCAdgOTmjsuphZL0sPz59id9SlG97zbf2XuwHeoBK4ve1GxC-C6erJPRVKHzo3-KwIJ2xW5z6YORm-zQQwWgQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">358</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">79</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn011"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="376012"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f4%2f185%2f376012">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260564600002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (79)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F4%2F185%2F376012" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/376012'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Whole-genome Sequencing of the Obligate Intracellular Bacterium Orientia tsutsugamushi Revealed Massive Gene Amplification During Reductive Genome Evolution" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn011");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f185%2f376012">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

