

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/123/4657153","@type":"ScholarlyArticle","name":"Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei","datePublished":"2017-11-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx043","keywords":["glycine max","SSR","SNP","CSSL"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/123/4657153","author":[{"name":"Watanabe, Satoshi","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Shimizu, Takehiko","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Machita, Kayo","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Tsubokura, Yasutaka","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Xia, Zhengjun","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Yamada, Tetsuya","affiliation":"Soybean Breeding Unit, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8517, Japan","@type":"Person"},{"name":"Hajika, Makita","affiliation":"Soybean Breeding Unit, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8517, Japan","@type":"Person"},{"name":"Ishimoto, Masao","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Katayose, Yuichi","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Harada, Kyuya","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Kaga, Akito","affiliation":"Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"}],"description":"Abstract.  Using progeny of a cross between Japanese soybean Enrei and Chinese soybean Peking, we developed a high-density linkage map and chromosomal segment s","pageStart":"123","pageEnd":"136","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei" />
    <meta property="og:description" content="Abstract.  Using progeny of a cross between Japanese soybean Enrei and Chinese soybean Peking, we developed a high-density linkage map and chromosomal segment s" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/123/4657153" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-11-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Watanabe, Satoshi" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Shimizu, Takehiko" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Machita, Kayo" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Tsubokura, Yasutaka" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Xia, Zhengjun" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Yamada, Tetsuya" /><meta name="citation_author_institution" content="Soybean Breeding Unit, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8517, Japan" /><meta name="citation_author" content="Hajika, Makita" /><meta name="citation_author_institution" content="Soybean Breeding Unit, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8517, Japan" /><meta name="citation_author" content="Ishimoto, Masao" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Katayose, Yuichi" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Harada, Kyuya" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Kaga, Akito" /><meta name="citation_author_institution" content="Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_title" content="Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei" /><meta name="citation_firstpage" content="123" /><meta name="citation_lastpage" content="136" /><meta name="citation_doi" content="10.1093/dnares/dsx043" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=408-14; " /><meta name="citation_reference" content="citation_title=Construction of a high-density mutant library in soybean and development of a mutant retrieval method using amplicon sequencing;  citation_journal_title=BMC Genom;  citation_year=2015;  citation_volume=16;  citation_pages=1014" /><meta name="citation_reference" content="citation_title=An integrated genetic linkage map of the soybean genome;  citation_journal_title=Crop Sci;  citation_year=1999;  citation_volume=39;  citation_pages=1464-90; " /><meta name="citation_reference" content="citation_title=Abundance of SSR Motifs and development of candidate polymorphic SSR Markers (BARCSOYSSR_1.0) in soybean;  citation_journal_title=Crop Sci;  citation_year=2010;  citation_volume=50;  citation_pages=1950" /><meta name="citation_reference" content="citation_title=Construction of high resolution genetic linkage maps to improve the soybean genome sequence assembly Glyma1.01;  citation_journal_title=BMC Genom;  citation_year=2016;  citation_volume=17;  citation_pages=33" /><meta name="citation_reference" content="citation_title=Cytogenetic analysis of translocations in soybean;  citation_journal_title=J. Heredity;  citation_year=1999;  citation_volume=90;  citation_pages=648-53; " /><meta name="citation_reference" content="citation_title=Fluorescence in situ hybridization-based karyotyping of soybean translocation lines;  citation_journal_title=G3;  citation_year=2011;  citation_volume=1;  citation_pages=117-29; " /><meta name="citation_reference" content="citation_title=The Glycine max cv. Enrei genome for improvement of Japanese soybean cultivars;  citation_journal_title=Int. J. Genom;  citation_year=2015;  citation_volume=2015;  citation_pages=358127" /><meta name="citation_reference" content="citation_title=Resistance of soybean to the soybean-cyst nematode as determined by a double-row method;  citation_journal_title=Plant Dis. Reporter;  citation_year=1957;  citation_volume=41;  citation_pages=923-4; " /><meta name="citation_reference" content="Lorenzen L.L.  1994 , Soybean cultivar development: a genome perspective. Ph.D. Dissertation, Iowa State University, Ames, IA. Available at: http://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=11625&amp;context=rtd (1 November 2017, date last accessed). " /><meta name="citation_reference" content="citation_title=Molecular marker mapping of Rsv4, a gene conferring resistance to all known strains of soybean mosaic virus;  citation_journal_title=Crop Sci;  citation_year=2000;  citation_volume=40;  citation_pages=1434-7; " /><meta name="citation_reference" content="citation_title=2nd gene for resistance to peanut mottle virus in soybeans;  citation_journal_title=Crop Sci;  citation_year=1979;  citation_volume=19;  citation_pages=656-8; " /><meta name="citation_reference" content="citation_title=Cloned avirulence gene of Pseudomonas syringae pv. glycinea determines race-specific incompatibility on Glycine max (L.) Merr;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1984;  citation_volume=81;  citation_pages=6024-8; " /><meta name="citation_reference" content="citation_title=Genetic analysis of frogeye leaf spot resistance in PI54610 and Peking soybean;  citation_journal_title=Crop Sci;  citation_year=1999;  citation_volume=39;  citation_pages=1021-5; " /><meta name="citation_reference" content="citation_title=Breeding for resistance to soybean dwarf disease 1. Resistance of soybean varieties to soybean dwarf disease;  citation_journal_title=Bull. Hokkaido Prefect Agric. Exp. Stn;  citation_year=1976;  citation_volume=35;  citation_pages=8-17; " /><meta name="citation_reference" content="citation_title=A seedling test for resistance to soybean stem canker caused by Diaporthe phaseolorum var caulivora;  citation_journal_title=Phytopathology;  citation_year=1982;  citation_volume=72;  citation_pages=807-9; " /><meta name="citation_reference" content="citation_title=Resistance to the reniform nematode in selected soybean cultivars and germplasm lines;  citation_journal_title=J. Nematol;  citation_year=1996;  citation_volume=28;  citation_pages=612-5; " /><meta name="citation_reference" content="citation_title=Effect of 2 commercial fungicides on incidence of Diaporthe phaseolorum var caulivora on susceptible soybean cultivars;  citation_journal_title=Phytopathology;  citation_year=1984;  citation_volume=74;  citation_pages=827" /><meta name="citation_reference" content="citation_title=The role of seed structure and oxygen responsiveness in pre-germination flooding tolerance of soybean cultivars;  citation_journal_title=Plant. Prod. Sci;  citation_year=2005;  citation_volume=8;  citation_pages=157-65; " /><meta name="citation_reference" content="citation_title=Relation of seed structures to soybean cultivar difference in pre-germination flooding tolerance;  citation_journal_title=Plant. Prod. Sci;  citation_year=2008;  citation_volume=11;  citation_pages=434-9; " /><meta name="citation_reference" content="citation_title=Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci;  citation_journal_title=Theor. Appl. Genet;  citation_year=1997;  citation_volume=95;  citation_pages=1005-11; " /><meta name="citation_reference" content="citation_title=Fine mapping of the FT1 locus for soybean flowering time using a residual heterozygous line derived from a recombinant inbred line;  citation_journal_title=Theor. Appl. Genet;  citation_year=2005;  citation_volume=110;  citation_pages=634-9; " /><meta name="citation_reference" content="citation_title=An introgression line population of Lycopersicon pennellii in the cultivated tomato enables the identification and fine mapping of yield-associated qtl;  citation_journal_title=Genetics;  citation_year=1995;  citation_volume=141;  citation_pages=1147-62; " /><meta name="citation_reference" content="citation_title=STAIRS: a new genetic resource for functional genomic studies of Arabidopsis;  citation_journal_title=Plant J. Cell Mol. Biol;  citation_year=2002;  citation_volume=31;  citation_pages=355-64; " /><meta name="citation_reference" content="citation_title=Chromosome segment substitution lines: a powerful tool for the introgression of valuable genes from Oryza wild species into cultivated rice (O. sativa);  citation_journal_title=Rice;  citation_year=2010;  citation_volume=3;  citation_pages=218-34; " /><meta name="citation_reference" content="citation_title=Development of candidate introgression lines using an exotic barley accession (Hordeum vulgare ssp. spontaneum) as donor;  citation_journal_title=Theor. Appl. Genet;  citation_year=2004;  citation_volume=109;  citation_pages=1736-45; " /><meta name="citation_reference" content="citation_title=Construction of chromosome segment substitution lines in peanut (Arachis hypogaea L.) using a wild synthetic and QTL mapping for plant morphology;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e48642" /><meta name="citation_reference" content="citation_title=Establishment of introgression libraries in hybrid rye (Secale cereale L.) from an Iranian primitive accession as a new tool for rye breeding and genomics;  citation_journal_title=Theor. Appl. Genet;  citation_year=2008;  citation_volume=117;  citation_pages=641-52; " /><meta name="citation_reference" content="citation_title=The development of lettuce backcross inbred lines (BILs) for exploitation of the Lactuca saligna (wild lettuce) germplasm;  citation_journal_title=Theor. Appl. Genet;  citation_year=2004;  citation_volume=109;  citation_pages=394-401; " /><meta name="citation_reference" content="citation_title=Development of a chromosome segment substitution line population with wild soybean (Glycine soja Sieb. et Zucc.) as donor parent;  citation_journal_title=Euphytica;  citation_year=2012;  citation_volume=189;  citation_pages=293-307; " /><meta name="citation_reference" content="citation_title=Introgression of a quantitative trait locus for yield from Glycine soja into commercial soybean cultivars;  citation_journal_title=Theor. Appl. Genet;  citation_year=2003;  citation_volume=106;  citation_pages=575-82; " /><meta name="citation_reference" content="citation_title=De novo assembly of soybean wild relatives for pan-genome analysis of diversity and agronomic traits;  citation_journal_title=Nat. Biotechnol;  citation_year=2014;  citation_volume=32;  citation_pages=1045-52; " /><meta name="citation_reference" content="citation_title=Evaluation of soybean germplasm conserved in NIAS genebank and development of mini core collections;  citation_journal_title=Breed. Sci;  citation_year=2012;  citation_volume=61;  citation_pages=566-92; " /><meta name="citation_reference" content="citation_title=Distinct copy number, coding sequence, and locus methylation patterns underlie Rhg1-mediated soybean resistance to soybean cyst nematode;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=165;  citation_pages=630-47; " /><meta name="citation_reference" content="citation_title=Natural variation in the genes responsible for maturity loci E1, E2, E3 and E4 in soybean;  citation_journal_title=Ann. Botany;  citation_year=2014;  citation_volume=113;  citation_pages=429-41; " /><meta name="citation_reference" content="citation_title=Stage of development descriptions for soybeans, Glycine max (L.) Merrill;  citation_journal_title=Crop Sci;  citation_year=1971;  citation_volume=11;  citation_pages=929" /><meta name="citation_reference" content="citation_title=A 300 kilobase interval genetic map of rice including 883 expressed sequences;  citation_journal_title=Nat. Genet;  citation_year=1994;  citation_volume=8;  citation_pages=365-72; " /><meta name="citation_reference" content="citation_title=Sex-specific organisation of middle repetitive DNA sequences in the mealybug Planococcus lilacinus;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=3745-51; " /><meta name="citation_reference" content="citation_title=Read2Marker: a data processing tool for microsatellite marker development from a large data set;  citation_journal_title=Biotechnology;  citation_year=2005;  citation_volume=39;  citation_pages=472-6; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers;  citation_journal_title=Methods Mol. Biol;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=GENOMEMASKER package for designing unique genomic PCR primers;  citation_journal_title=BMC Bioinform;  citation_year=2006;  citation_volume=7;  citation_pages=172" /><meta name="citation_reference" content="citation_title=Characterization of the soybean genome using EST-derived microsatellite markers;  citation_journal_title=DNA Res;  citation_year=2007;  citation_volume=14;  citation_pages=271-81; " /><meta name="citation_reference" content="citation_title=An economic method for the fluorescent labeling of PCR fragments;  citation_journal_title=Nat. Biotechnol;  citation_year=2000;  citation_volume=18;  citation_pages=233-4; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=685-96; " /><meta name="citation_reference" content="citation_title=Qualitative and quantitative genotyping using single base primer extension coupled with matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MassARRAY);  citation_journal_title=Methods Mol. Biol;  citation_year=2009;  citation_volume=578;  citation_pages=307-43; " /><meta name="citation_reference" content="Van Ooijen J.W., Voorrips R.E.  2001 , JoinMap®Version 4.0: Software for the Calculation of Genetic Linkage Maps. Wageningen: Plant Research International. " /><meta name="citation_reference" content="citation_title=R/qtl: QTL mapping in experimental crosses;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=889-90; " /><meta name="citation_reference" content="R Development Core Team. 2008 , R: a language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.r-project.org/ (1 November 2017, date last accessed). " /><meta name="citation_reference" content="citation_title=A graphical representation of genetic and physical maps: the Marey map;  citation_journal_title=Genomics;  citation_year=1991;  citation_volume=11;  citation_pages=219-22; " /><meta name="citation_reference" content="citation_title=MareyMap: an R-based tool with graphical interface for estimating recombination rates;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2188-9; " /><meta name="citation_reference" content="citation_title=Genetic linkage map of the nucleolus organizer region in the soybean;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=178;  citation_pages=605-8; " /><meta name="citation_reference" content="citation_title=A Fluorescence in situ hybridization system for karyotyping soybean;  citation_journal_title=Genetics;  citation_year=2010;  citation_volume=185;  citation_pages=727-44; " /><meta name="citation_reference" content="citation_title=SoyTEdb: a comprehensive database of transposable elements in the soybean genome;  citation_journal_title=BMC Genom;  citation_year=2010;  citation_volume=11;  citation_pages=113" /><meta name="citation_reference" content="citation_title=Evaluation of the USDA soybean germplasm collection: maturity groups 000 to IV (PI 273.483 to PI 427.107);  citation_journal_title=T. Bull. USDA;  citation_year=1987;  citation_volume=1718;  citation_pages=44-123; " /><meta name="citation_reference" content="citation_title=Map-based cloning of the gene associated with the soybean maturity locus E3;  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=182;  citation_pages=1251-62; " /><meta name="citation_reference" content="citation_title=A map-based cloning strategy employing a residual heterozygous line reveals that the GIGANTEA gene is involved in soybean maturity and flowering;  citation_journal_title=Genetics;  citation_year=2011;  citation_volume=188;  citation_pages=395-407; " /><meta name="citation_reference" content="citation_title=Field evaluation of green stem disorder in soybean cultivars;  citation_journal_title=Crop Sci;  citation_year=2006;  citation_volume=46;  citation_pages=879-85; " /><meta name="citation_reference" content="citation_title=A recessive allele for delayed flowering at the soybean maturity locus E9 is a leaky allele of FT2a, a FLOWERING LOCUS T ortholog;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=20" /><meta name="citation_reference" content="citation_title=A soybean quantitative trait locus that promotes flowering under long days is identified as FT5a, a FLOWERING LOCUS T ortholog;  citation_journal_title=Exbotj;  citation_year=2016;  citation_volume=67;  citation_pages=5247-58; " /><meta name="citation_reference" content="citation_title=The soybean stem growth habit gene Dt1 is an ortholog of Arabidopsis TERMINAL FLOWER1;  citation_journal_title=Plant Physiol;  citation_year=2010;  citation_volume=153;  citation_pages=198-210; " /><meta name="citation_reference" content="citation_title=A third allele at the soybean dt1 locus;  citation_journal_title=Crop Sci;  citation_year=1997;  citation_volume=37;  citation_pages=757-62; " /><meta name="citation_reference" content="citation_title=SNP identification and marker assay development for high-throughput selection of soybean cyst nematode resistance;  citation_journal_title=BMC Genom;  citation_year=2015;  citation_volume=16;  citation_pages=314" /><meta name="citation_reference" content="citation_title=A major and stable QTL associated with seed weight in soybean across multiple environments and genetic backgrounds;  citation_journal_title=Theor. Appl. Genet;  citation_year=2014;  citation_volume=127;  citation_pages=1365-74; " /><meta name="citation_reference" content="citation_title=Comparison of narrow and broad leaflet isolines of soybean;  citation_journal_title=Crop Sci;  citation_year=1981;  citation_volume=21;  citation_pages=25-7; " /><meta name="citation_reference" content="citation_title=Ln is a hey regulator of leaflet shape and number of seeds per pod in soybean;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=4807-18; " /><meta name="citation_reference" content="citation_title=An informative linkage map of soybean reveals QTLs for flowering time, leaflet morphology and regions of segregation distortion;  citation_journal_title=DNA Res;  citation_year=2001;  citation_volume=8;  citation_pages=61-72; " /><meta name="citation_reference" content="citation_title=Positional cloning and characterization reveal the molecular basis for soybean maturity locus E1 that regulates photoperiodic flowering;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=E2155-64; " /><meta name="citation_reference" content="citation_title=Genetic variation in four maturity genes affects photoperiod insensitivity and PHYA-regulated post-flowering responses of soybean;  citation_journal_title=BMC Plant Biol;  citation_year=2013;  citation_volume=13;  citation_pages=91" /><meta name="citation_reference" content="citation_title=A new soybean maturity and photoperiod-sensitivity locus linked to E1 and T;  citation_journal_title=Crop Sci;  citation_year=2001;  citation_volume=41;  citation_pages=698-701; " /><meta name="citation_reference" content="citation_title=QTL affecting fitness of hybrids between wild and cultivated soybeans in experimental fields;  citation_journal_title=Ecol. Evol;  citation_year=2013;  citation_volume=3;  citation_pages=2150-68; " /><meta name="citation_reference" content="citation_title=Identification of QTL controlling post-flowering period in soybean;  citation_journal_title=Breed. Sci;  citation_year=2012;  citation_volume=61;  citation_pages=646-52; " /><meta name="citation_reference" content="citation_title=Resistance of soybeans to the soybean cyst nematode as determined by a double-row method;  citation_journal_title=Plant Dis. Rep;  citation_year=1957;  citation_volume=41;  citation_pages=923-4; " /><meta name="citation_reference" content="citation_title=The soybean-Phytophthora resistance locus Rps1-k encompasses coiled coil-nucleotide binding-leucine rich repeat-like genes and repetitive sequences;  citation_journal_title=BMC Plant Biol;  citation_year=2008;  citation_volume=8;  citation_pages=29" /><meta name="citation_reference" content="citation_title=Sequence analysis of a DNA fragment from Sinorhizobium fredii USDA257 which extends the nitrogen fixation host range of Rhizobium species NGR234 to soybean, Glycine max (L.) Merr cultivar Peking;  citation_journal_title=Symbiosis;  citation_year=2009;  citation_volume=48;  citation_pages=110-9; " /><meta name="citation_reference" content="citation_title=Genetic allelism of nodulation response genes Rj1, Rj2, and Rj4 in soybean;  citation_journal_title=Crop Sci;  citation_year=1989;  citation_volume=29;  citation_pages=1347-50; " /><meta name="citation_reference" content="citation_title=R gene-controlled host specificity in the legume-rhizobia symbiosis;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=18735-40; " /><meta name="citation_reference" content="citation_title=Rj4, a gene controlling nodulation specificity in soybeans, encodes a thaumatin-like protein but not the one previously reported;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=170;  citation_pages=26-32; " /><meta name="citation_reference" content="citation_title=Molecular mapping of a new gene for resistance to frogeye leaf spot of soya bean in Peking;  citation_journal_title=Plant Breed;  citation_year=2001;  citation_volume=120;  citation_pages=73-8; " /><meta name="citation_reference" content="citation_title=Research Notes: evaluation of soybean germplasm for resistance to corn earworm–IV;  citation_journal_title=Soybean Genet. Newslett;  citation_year=1980;  citation_volume=7;  citation_pages=26" /><meta name="citation_reference" content="citation_title=QTL analysis of seed-flooding tolerance in soybean (Glycine max [L.] Merr.);  citation_journal_title=Plant Sci;  citation_year=2009;  citation_volume=176;  citation_pages=514-21; " /><meta name="citation_reference" content="citation_title=Genotype effects on proliferative embryogenesis and plant regeneration of soybean;  citation_journal_title=In Vitro Cell. Dev. Biol.–Plant;  citation_year=1993;  citation_volume=29;  citation_pages=102-8; " /><meta name="citation_reference" content="citation_title=Identification of QTL underlying somatic embryogenesis capacity of immature embryos in soybean (Glycine max (L.) Merr.);  citation_journal_title=Plant Cell Rep;  citation_year=2010;  citation_volume=29;  citation_pages=125-31; " /><meta name="citation_reference" content="citation_title=Inheritance of soybean susceptibility to Agrobacterium tumefaciens and its relationship to transformation;  citation_journal_title=Crop Sci;  citation_year=1995;  citation_volume=35;  citation_pages=1152-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/123/24662147/dsx043.pdf" /><meta name="description" content="Abstract.  Using progeny of a cross between Japanese soybean Enrei and Chinese soybean Peking, we developed a high-density linkage map and chromosomal segment s" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/123/4657153" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/123/4657153" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/123/4657153");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx043';
    </script>
<title>
	Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4657153" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb3LATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2F0YW5hYmUsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTb3liZWFuIEFwcGxpZWQgR2Vub21pY3MgUmVzZWFyY2ggVW5pdCwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzIChOSUFTKSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGltaXp1LCBUYWtlaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNveWJlYW4gQXBwbGllZCBHZW5vbWljcyBSZXNlYXJjaCBVbml0LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMgKE5JQVMpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hY2hpdGEsIEtheW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTb3liZWFuIEFwcGxpZWQgR2Vub21pY3MgUmVzZWFyY2ggVW5pdCwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzIChOSUFTKSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3Vib2t1cmEsIFlhc3V0YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU295YmVhbiBBcHBsaWVkIEdlbm9taWNzIFJlc2VhcmNoIFVuaXQsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcyAoTklBUyksIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWGlhLCBaaGVuZ2p1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNveWJlYW4gQXBwbGllZCBHZW5vbWljcyBSZXNlYXJjaCBVbml0LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMgKE5JQVMpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFkYSwgVGV0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNveWJlYW4gQnJlZWRpbmcgVW5pdCwgSW5zdGl0dXRlIG9mIENyb3AgU2NpZW5jZSwgTkFSTywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODUxNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYWppa2EsIE1ha2l0YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNveWJlYW4gQnJlZWRpbmcgVW5pdCwgSW5zdGl0dXRlIG9mIENyb3AgU2NpZW5jZSwgTkFSTywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODUxNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpbW90bywgTWFzYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTb3liZWFuIEFwcGxpZWQgR2Vub21pY3MgUmVzZWFyY2ggVW5pdCwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzIChOSUFTKSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRheW9zZSwgWXVpY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU295YmVhbiBBcHBsaWVkIEdlbm9taWNzIFJlc2VhcmNoIFVuaXQsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcyAoTklBUyksIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFyYWRhLCBLeXV5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNveWJlYW4gQXBwbGllZCBHZW5vbWljcyBSZXNlYXJjaCBVbml0LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMgKE5JQVMpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthZ2EsIEFraXRvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU295YmVhbiBBcHBsaWVkIEdlbm9taWNzIFJlc2VhcmNoIFVuaXQsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcyAoTklBUyksIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXZlbG9wbWVudCBvZiBhIGhpZ2gtZGVuc2l0eSBsaW5rYWdlIG1hcCBhbmQgY2hyb21vc29tZSBzZWdtZW50IHN1YnN0aXR1dGlvbiBsaW5lcyBmb3IgSmFwYW5lc2Ugc295YmVhbiBjdWx0aXZhciBFbnJlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTgvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHBhbGFlb3BvbHlwbG9pZCBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDYzOyAgY2l0YXRpb25fcGFnZXM9MTc4LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNlcXVlbmNpbmcgMzAyIHdpbGQgYW5kIGN1bHRpdmF0ZWQgYWNjZXNzaW9ucyBpZGVudGlmaWVzIGdlbmVzIHJlbGF0ZWQgdG8gZG9tZXN0aWNhdGlvbiBhbmQgaW1wcm92ZW1lbnQgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NDA4LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBoaWdoLWRlbnNpdHkgbXV0YW50IGxpYnJhcnkgaW4gc295YmVhbiBhbmQgZGV2ZWxvcG1lbnQgb2YgYSBtdXRhbnQgcmV0cmlldmFsIG1ldGhvZCB1c2luZyBhbXBsaWNvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub207ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMDE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIHRoZSBzb3liZWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xNDY0LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnVuZGFuY2Ugb2YgU1NSIE1vdGlmcyBhbmQgZGV2ZWxvcG1lbnQgb2YgY2FuZGlkYXRlIHBvbHltb3JwaGljIFNTUiBNYXJrZXJzIChCQVJDU09ZU1NSXzEuMCkgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0xOTUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGhpZ2ggcmVzb2x1dGlvbiBnZW5ldGljIGxpbmthZ2UgbWFwcyB0byBpbXByb3ZlIHRoZSBzb3liZWFuIGdlbm9tZSBzZXF1ZW5jZSBhc3NlbWJseSBHbHltYTEuMDE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b2dlbmV0aWMgYW5hbHlzaXMgb2YgdHJhbnNsb2NhdGlvbnMgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz02NDgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsdW9yZXNjZW5jZSBpbiBzaXR1IGh5YnJpZGl6YXRpb24tYmFzZWQga2FyeW90eXBpbmcgb2Ygc295YmVhbiB0cmFuc2xvY2F0aW9uIGxpbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMzsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTE3LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2x5Y2luZSBtYXggY3YuIEVucmVpIGdlbm9tZSBmb3IgaW1wcm92ZW1lbnQgb2YgSmFwYW5lc2Ugc295YmVhbiBjdWx0aXZhcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gR2Vub207ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjAxNTsgIGNpdGF0aW9uX3BhZ2VzPTM1ODEyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc2lzdGFuY2Ugb2Ygc295YmVhbiB0byB0aGUgc295YmVhbi1jeXN0IG5lbWF0b2RlIGFzIGRldGVybWluZWQgYnkgYSBkb3VibGUtcm93IG1ldGhvZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgRGlzLiBSZXBvcnRlcjsgIGNpdGF0aW9uX3llYXI9MTk1NzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTkyMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJMb3JlbnplbiBMLkwuICAxOTk0ICwgU295YmVhbiBjdWx0aXZhciBkZXZlbG9wbWVudDogYSBnZW5vbWUgcGVyc3BlY3RpdmUuIFBoLkQuIERpc3NlcnRhdGlvbiwgSW93YSBTdGF0ZSBVbml2ZXJzaXR5LCBBbWVzLCBJQS4gQXZhaWxhYmxlIGF0OiBodHRwOi8vbGliLmRyLmlhc3RhdGUuZWR1L2NnaS92aWV3Y29udGVudC5jZ2k/YXJ0aWNsZT0xMTYyNSZhbXA7Y29udGV4dD1ydGQgKDEgTm92ZW1iZXIgMjAxNywgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1hcmtlciBtYXBwaW5nIG9mIFJzdjQsIGEgZ2VuZSBjb25mZXJyaW5nIHJlc2lzdGFuY2UgdG8gYWxsIGtub3duIHN0cmFpbnMgb2Ygc295YmVhbiBtb3NhaWMgdmlydXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTQzNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT0ybmQgZ2VuZSBmb3IgcmVzaXN0YW5jZSB0byBwZWFudXQgbW90dGxlIHZpcnVzIGluIHNveWJlYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk3OTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uZWQgYXZpcnVsZW5jZSBnZW5lIG9mIFBzZXVkb21vbmFzIHN5cmluZ2FlIHB2LiBnbHljaW5lYSBkZXRlcm1pbmVzIHJhY2Utc3BlY2lmaWMgaW5jb21wYXRpYmlsaXR5IG9uIEdseWNpbmUgbWF4IChMLikgTWVycjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz02MDI0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgb2YgZnJvZ2V5ZSBsZWFmIHNwb3QgcmVzaXN0YW5jZSBpbiBQSTU0NjEwIGFuZCBQZWtpbmcgc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xMDIxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJyZWVkaW5nIGZvciByZXNpc3RhbmNlIHRvIHNveWJlYW4gZHdhcmYgZGlzZWFzZSAxLiBSZXNpc3RhbmNlIG9mIHNveWJlYW4gdmFyaWV0aWVzIHRvIHNveWJlYW4gZHdhcmYgZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnVsbC4gSG9ra2FpZG8gUHJlZmVjdCBBZ3JpYy4gRXhwLiBTdG47ICBjaXRhdGlvbl95ZWFyPTE5NzY7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz04LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNlZWRsaW5nIHRlc3QgZm9yIHJlc2lzdGFuY2UgdG8gc295YmVhbiBzdGVtIGNhbmtlciBjYXVzZWQgYnkgRGlhcG9ydGhlIHBoYXNlb2xvcnVtIHZhciBjYXVsaXZvcmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvcGF0aG9sb2d5OyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9ODA3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc2lzdGFuY2UgdG8gdGhlIHJlbmlmb3JtIG5lbWF0b2RlIGluIHNlbGVjdGVkIHNveWJlYW4gY3VsdGl2YXJzIGFuZCBnZXJtcGxhc20gbGluZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE5lbWF0b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz02MTItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0IG9mIDIgY29tbWVyY2lhbCBmdW5naWNpZGVzIG9uIGluY2lkZW5jZSBvZiBEaWFwb3J0aGUgcGhhc2VvbG9ydW0gdmFyIGNhdWxpdm9yYSBvbiBzdXNjZXB0aWJsZSBzb3liZWFuIGN1bHRpdmFyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9wYXRob2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz04MjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBzZWVkIHN0cnVjdHVyZSBhbmQgb3h5Z2VuIHJlc3BvbnNpdmVuZXNzIGluIHByZS1nZXJtaW5hdGlvbiBmbG9vZGluZyB0b2xlcmFuY2Ugb2Ygc295YmVhbiBjdWx0aXZhcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50LiBQcm9kLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE1Ny02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVsYXRpb24gb2Ygc2VlZCBzdHJ1Y3R1cmVzIHRvIHNveWJlYW4gY3VsdGl2YXIgZGlmZmVyZW5jZSBpbiBwcmUtZ2VybWluYXRpb24gZmxvb2RpbmcgdG9sZXJhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudC4gUHJvZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDM0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhldGVyb2dlbmVvdXMgaW5icmVkIGZhbWlseSAoSElGKSBhbmFseXNpczogYSBtZXRob2QgZm9yIGRldmVsb3BpbmcgbmVhci1pc29nZW5pYyBsaW5lcyB0aGF0IGRpZmZlciBhdCBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MTAwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZSBtYXBwaW5nIG9mIHRoZSBGVDEgbG9jdXMgZm9yIHNveWJlYW4gZmxvd2VyaW5nIHRpbWUgdXNpbmcgYSByZXNpZHVhbCBoZXRlcm96eWdvdXMgbGluZSBkZXJpdmVkIGZyb20gYSByZWNvbWJpbmFudCBpbmJyZWQgbGluZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTYzNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRyb2dyZXNzaW9uIGxpbmUgcG9wdWxhdGlvbiBvZiBMeWNvcGVyc2ljb24gcGVubmVsbGlpIGluIHRoZSBjdWx0aXZhdGVkIHRvbWF0byBlbmFibGVzIHRoZSBpZGVudGlmaWNhdGlvbiBhbmQgZmluZSBtYXBwaW5nIG9mIHlpZWxkLWFzc29jaWF0ZWQgcXRsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0xMTQ3LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVEFJUlM6IGEgbmV3IGdlbmV0aWMgcmVzb3VyY2UgZm9yIGZ1bmN0aW9uYWwgZ2Vub21pYyBzdHVkaWVzIG9mIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLiBDZWxsIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZSBzZWdtZW50IHN1YnN0aXR1dGlvbiBsaW5lczogYSBwb3dlcmZ1bCB0b29sIGZvciB0aGUgaW50cm9ncmVzc2lvbiBvZiB2YWx1YWJsZSBnZW5lcyBmcm9tIE9yeXphIHdpbGQgc3BlY2llcyBpbnRvIGN1bHRpdmF0ZWQgcmljZSAoTy4gc2F0aXZhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmljZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjE4LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBjYW5kaWRhdGUgaW50cm9ncmVzc2lvbiBsaW5lcyB1c2luZyBhbiBleG90aWMgYmFybGV5IGFjY2Vzc2lvbiAoSG9yZGV1bSB2dWxnYXJlIHNzcC4gc3BvbnRhbmV1bSkgYXMgZG9ub3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xNzM2LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgY2hyb21vc29tZSBzZWdtZW50IHN1YnN0aXR1dGlvbiBsaW5lcyBpbiBwZWFudXQgKEFyYWNoaXMgaHlwb2dhZWEgTC4pIHVzaW5nIGEgd2lsZCBzeW50aGV0aWMgYW5kIFFUTCBtYXBwaW5nIGZvciBwbGFudCBtb3JwaG9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ4NjQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0YWJsaXNobWVudCBvZiBpbnRyb2dyZXNzaW9uIGxpYnJhcmllcyBpbiBoeWJyaWQgcnllIChTZWNhbGUgY2VyZWFsZSBMLikgZnJvbSBhbiBJcmFuaWFuIHByaW1pdGl2ZSBhY2Nlc3Npb24gYXMgYSBuZXcgdG9vbCBmb3IgcnllIGJyZWVkaW5nIGFuZCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNzsgIGNpdGF0aW9uX3BhZ2VzPTY0MS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRldmVsb3BtZW50IG9mIGxldHR1Y2UgYmFja2Nyb3NzIGluYnJlZCBsaW5lcyAoQklMcykgZm9yIGV4cGxvaXRhdGlvbiBvZiB0aGUgTGFjdHVjYSBzYWxpZ25hICh3aWxkIGxldHR1Y2UpIGdlcm1wbGFzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTM5NC00MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgY2hyb21vc29tZSBzZWdtZW50IHN1YnN0aXR1dGlvbiBsaW5lIHBvcHVsYXRpb24gd2l0aCB3aWxkIHNveWJlYW4gKEdseWNpbmUgc29qYSBTaWViLiBldCBadWNjLikgYXMgZG9ub3IgcGFyZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTg5OyAgY2l0YXRpb25fcGFnZXM9MjkzLTMwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9ncmVzc2lvbiBvZiBhIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBmb3IgeWllbGQgZnJvbSBHbHljaW5lIHNvamEgaW50byBjb21tZXJjaWFsIHNveWJlYW4gY3VsdGl2YXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9NTc1LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IG9mIHNveWJlYW4gd2lsZCByZWxhdGl2ZXMgZm9yIHBhbi1nZW5vbWUgYW5hbHlzaXMgb2YgZGl2ZXJzaXR5IGFuZCBhZ3Jvbm9taWMgdHJhaXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMDQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIHNveWJlYW4gZ2VybXBsYXNtIGNvbnNlcnZlZCBpbiBOSUFTIGdlbmViYW5rIGFuZCBkZXZlbG9wbWVudCBvZiBtaW5pIGNvcmUgY29sbGVjdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyZWVkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz01NjYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RpbmN0IGNvcHkgbnVtYmVyLCBjb2Rpbmcgc2VxdWVuY2UsIGFuZCBsb2N1cyBtZXRoeWxhdGlvbiBwYXR0ZXJucyB1bmRlcmxpZSBSaGcxLW1lZGlhdGVkIHNveWJlYW4gcmVzaXN0YW5jZSB0byBzb3liZWFuIGN5c3QgbmVtYXRvZGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9NjMwLTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIHZhcmlhdGlvbiBpbiB0aGUgZ2VuZXMgcmVzcG9uc2libGUgZm9yIG1hdHVyaXR5IGxvY2kgRTEsIEUyLCBFMyBhbmQgRTQgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBCb3Rhbnk7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NDI5LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFnZSBvZiBkZXZlbG9wbWVudCBkZXNjcmlwdGlvbnMgZm9yIHNveWJlYW5zLCBHbHljaW5lIG1heCAoTC4pIE1lcnJpbGw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpOyAgY2l0YXRpb25feWVhcj0xOTcxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OTI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSAzMDAga2lsb2Jhc2UgaW50ZXJ2YWwgZ2VuZXRpYyBtYXAgb2YgcmljZSBpbmNsdWRpbmcgODgzIGV4cHJlc3NlZCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM2NS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2V4LXNwZWNpZmljIG9yZ2FuaXNhdGlvbiBvZiBtaWRkbGUgcmVwZXRpdGl2ZSBETkEgc2VxdWVuY2VzIGluIHRoZSBtZWFseWJ1ZyBQbGFub2NvY2N1cyBsaWxhY2ludXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9Mzc0NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVhZDJNYXJrZXI6IGEgZGF0YSBwcm9jZXNzaW5nIHRvb2wgZm9yIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlciBkZXZlbG9wbWVudCBmcm9tIGEgbGFyZ2UgZGF0YSBzZXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz00NzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyMyBvbiB0aGUgV1dXIGZvciBnZW5lcmFsIHVzZXJzIGFuZCBmb3IgYmlvbG9naXN0IHByb2dyYW1tZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZXRob2RzIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0zNjUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdFTk9NRU1BU0tFUiBwYWNrYWdlIGZvciBkZXNpZ25pbmcgdW5pcXVlIGdlbm9taWMgUENSIHByaW1lcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIHNveWJlYW4gZ2Vub21lIHVzaW5nIEVTVC1kZXJpdmVkIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVjb25vbWljIG1ldGhvZCBmb3IgdGhlIGZsdW9yZXNjZW50IGxhYmVsaW5nIG9mIFBDUiBmcmFnbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTIzMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzb3liZWFuIHRyYW5zY3JpcHQgbWFwOiBnZW5lIGRpc3RyaWJ1dGlvbiwgaGFwbG90eXBlIGFuZCBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTY4NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0YXRpdmUgYW5kIHF1YW50aXRhdGl2ZSBnZW5vdHlwaW5nIHVzaW5nIHNpbmdsZSBiYXNlIHByaW1lciBleHRlbnNpb24gY291cGxlZCB3aXRoIG1hdHJpeC1hc3Npc3RlZCBsYXNlciBkZXNvcnB0aW9uL2lvbml6YXRpb24gdGltZS1vZi1mbGlnaHQgbWFzcyBzcGVjdHJvbWV0cnkgKE1hc3NBUlJBWSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHMgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU3ODsgIGNpdGF0aW9uX3BhZ2VzPTMwNy00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVmFuIE9vaWplbiBKLlcuLCBWb29ycmlwcyBSLkUuICAyMDAxICwgSm9pbk1hcMKuVmVyc2lvbiA0LjA6IFNvZnR3YXJlIGZvciB0aGUgQ2FsY3VsYXRpb24gb2YgR2VuZXRpYyBMaW5rYWdlIE1hcHMuIFdhZ2VuaW5nZW46IFBsYW50IFJlc2VhcmNoIEludGVybmF0aW9uYWwuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVIvcXRsOiBRVEwgbWFwcGluZyBpbiBleHBlcmltZW50YWwgY3Jvc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz04ODktOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlIgRGV2ZWxvcG1lbnQgQ29yZSBUZWFtLiAyMDA4ICwgUjogYSBsYW5ndWFnZSBhbmQgZW52aXJvbm1lbnQgZm9yIHN0YXRpc3RpY2FsIGNvbXB1dGluZy4gUiBGb3VuZGF0aW9uIGZvciBTdGF0aXN0aWNhbCBDb21wdXRpbmcsIFZpZW5uYSwgQXVzdHJpYS4gaHR0cHM6Ly93d3cuci1wcm9qZWN0Lm9yZy8gKDEgTm92ZW1iZXIgMjAxNywgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBncmFwaGljYWwgcmVwcmVzZW50YXRpb24gb2YgZ2VuZXRpYyBhbmQgcGh5c2ljYWwgbWFwczogdGhlIE1hcmV5IG1hcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMTktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcmV5TWFwOiBhbiBSLWJhc2VkIHRvb2wgd2l0aCBncmFwaGljYWwgaW50ZXJmYWNlIGZvciBlc3RpbWF0aW5nIHJlY29tYmluYXRpb24gcmF0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjE4OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGxpbmthZ2UgbWFwIG9mIHRoZSBudWNsZW9sdXMgb3JnYW5pemVyIHJlZ2lvbiBpbiB0aGUgc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc4OyAgY2l0YXRpb25fcGFnZXM9NjA1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgRmx1b3Jlc2NlbmNlIGluIHNpdHUgaHlicmlkaXphdGlvbiBzeXN0ZW0gZm9yIGthcnlvdHlwaW5nIHNveWJlYW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPTcyNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U295VEVkYjogYSBjb21wcmVoZW5zaXZlIGRhdGFiYXNlIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiB0aGUgc295YmVhbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2YgdGhlIFVTREEgc295YmVhbiBnZXJtcGxhc20gY29sbGVjdGlvbjogbWF0dXJpdHkgZ3JvdXBzIDAwMCB0byBJViAoUEkgMjczLjQ4MyB0byBQSSA0MjcuMTA3KTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VC4gQnVsbC4gVVNEQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE4OyAgY2l0YXRpb25fcGFnZXM9NDQtMTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXAtYmFzZWQgY2xvbmluZyBvZiB0aGUgZ2VuZSBhc3NvY2lhdGVkIHdpdGggdGhlIHNveWJlYW4gbWF0dXJpdHkgbG9jdXMgRTM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MjsgIGNpdGF0aW9uX3BhZ2VzPTEyNTEtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWFwLWJhc2VkIGNsb25pbmcgc3RyYXRlZ3kgZW1wbG95aW5nIGEgcmVzaWR1YWwgaGV0ZXJvenlnb3VzIGxpbmUgcmV2ZWFscyB0aGF0IHRoZSBHSUdBTlRFQSBnZW5lIGlzIGludm9sdmVkIGluIHNveWJlYW4gbWF0dXJpdHkgYW5kIGZsb3dlcmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9Mzk1LTQwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmllbGQgZXZhbHVhdGlvbiBvZiBncmVlbiBzdGVtIGRpc29yZGVyIGluIHNveWJlYW4gY3VsdGl2YXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTg3OS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZWNlc3NpdmUgYWxsZWxlIGZvciBkZWxheWVkIGZsb3dlcmluZyBhdCB0aGUgc295YmVhbiBtYXR1cml0eSBsb2N1cyBFOSBpcyBhIGxlYWt5IGFsbGVsZSBvZiBGVDJhLCBhIEZMT1dFUklORyBMT0NVUyBUIG9ydGhvbG9nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzb3liZWFuIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyB0aGF0IHByb21vdGVzIGZsb3dlcmluZyB1bmRlciBsb25nIGRheXMgaXMgaWRlbnRpZmllZCBhcyBGVDVhLCBhIEZMT1dFUklORyBMT0NVUyBUIG9ydGhvbG9nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FeGJvdGo7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz01MjQ3LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc295YmVhbiBzdGVtIGdyb3d0aCBoYWJpdCBnZW5lIER0MSBpcyBhbiBvcnRob2xvZyBvZiBBcmFiaWRvcHNpcyBURVJNSU5BTCBGTE9XRVIxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1MzsgIGNpdGF0aW9uX3BhZ2VzPTE5OC0yMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGhpcmQgYWxsZWxlIGF0IHRoZSBzb3liZWFuIGR0MSBsb2N1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz03NTctNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBpZGVudGlmaWNhdGlvbiBhbmQgbWFya2VyIGFzc2F5IGRldmVsb3BtZW50IGZvciBoaWdoLXRocm91Z2hwdXQgc2VsZWN0aW9uIG9mIHNveWJlYW4gY3lzdCBuZW1hdG9kZSByZXNpc3RhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub207ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1ham9yIGFuZCBzdGFibGUgUVRMIGFzc29jaWF0ZWQgd2l0aCBzZWVkIHdlaWdodCBpbiBzb3liZWFuIGFjcm9zcyBtdWx0aXBsZSBlbnZpcm9ubWVudHMgYW5kIGdlbmV0aWMgYmFja2dyb3VuZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz0xMzY1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIG5hcnJvdyBhbmQgYnJvYWQgbGVhZmxldCBpc29saW5lcyBvZiBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxuIGlzIGEgaGV5IHJlZ3VsYXRvciBvZiBsZWFmbGV0IHNoYXBlIGFuZCBudW1iZXIgb2Ygc2VlZHMgcGVyIHBvZCBpbiBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDgwNy0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW5mb3JtYXRpdmUgbGlua2FnZSBtYXAgb2Ygc295YmVhbiByZXZlYWxzIFFUTHMgZm9yIGZsb3dlcmluZyB0aW1lLCBsZWFmbGV0IG1vcnBob2xvZ3kgYW5kIHJlZ2lvbnMgb2Ygc2VncmVnYXRpb24gZGlzdG9ydGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvc2l0aW9uYWwgY2xvbmluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiByZXZlYWwgdGhlIG1vbGVjdWxhciBiYXNpcyBmb3Igc295YmVhbiBtYXR1cml0eSBsb2N1cyBFMSB0aGF0IHJlZ3VsYXRlcyBwaG90b3BlcmlvZGljIGZsb3dlcmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9RTIxNTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWF0aW9uIGluIGZvdXIgbWF0dXJpdHkgZ2VuZXMgYWZmZWN0cyBwaG90b3BlcmlvZCBpbnNlbnNpdGl2aXR5IGFuZCBQSFlBLXJlZ3VsYXRlZCBwb3N0LWZsb3dlcmluZyByZXNwb25zZXMgb2Ygc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz05MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IHNveWJlYW4gbWF0dXJpdHkgYW5kIHBob3RvcGVyaW9kLXNlbnNpdGl2aXR5IGxvY3VzIGxpbmtlZCB0byBFMSBhbmQgVDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz02OTgtNzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgYWZmZWN0aW5nIGZpdG5lc3Mgb2YgaHlicmlkcyBiZXR3ZWVuIHdpbGQgYW5kIGN1bHRpdmF0ZWQgc295YmVhbnMgaW4gZXhwZXJpbWVudGFsIGZpZWxkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWNvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjE1MC02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgUVRMIGNvbnRyb2xsaW5nIHBvc3QtZmxvd2VyaW5nIHBlcmlvZCBpbiBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmVlZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NjQ2LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNpc3RhbmNlIG9mIHNveWJlYW5zIHRvIHRoZSBzb3liZWFuIGN5c3QgbmVtYXRvZGUgYXMgZGV0ZXJtaW5lZCBieSBhIGRvdWJsZS1yb3cgbWV0aG9kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBEaXMuIFJlcDsgIGNpdGF0aW9uX3llYXI9MTk1NzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTkyMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc295YmVhbi1QaHl0b3BodGhvcmEgcmVzaXN0YW5jZSBsb2N1cyBScHMxLWsgZW5jb21wYXNzZXMgY29pbGVkIGNvaWwtbnVjbGVvdGlkZSBiaW5kaW5nLWxldWNpbmUgcmljaCByZXBlYXQtbGlrZSBnZW5lcyBhbmQgcmVwZXRpdGl2ZSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFuYWx5c2lzIG9mIGEgRE5BIGZyYWdtZW50IGZyb20gU2lub3JoaXpvYml1bSBmcmVkaWkgVVNEQTI1NyB3aGljaCBleHRlbmRzIHRoZSBuaXRyb2dlbiBmaXhhdGlvbiBob3N0IHJhbmdlIG9mIFJoaXpvYml1bSBzcGVjaWVzIE5HUjIzNCB0byBzb3liZWFuLCBHbHljaW5lIG1heCAoTC4pIE1lcnIgY3VsdGl2YXIgUGVraW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeW1iaW9zaXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xMTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbGxlbGlzbSBvZiBub2R1bGF0aW9uIHJlc3BvbnNlIGdlbmVzIFJqMSwgUmoyLCBhbmQgUmo0IGluIHNveWJlYW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTM0Ny01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UiBnZW5lLWNvbnRyb2xsZWQgaG9zdCBzcGVjaWZpY2l0eSBpbiB0aGUgbGVndW1lLXJoaXpvYmlhIHN5bWJpb3NpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTg3MzUtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJqNCwgYSBnZW5lIGNvbnRyb2xsaW5nIG5vZHVsYXRpb24gc3BlY2lmaWNpdHkgaW4gc295YmVhbnMsIGVuY29kZXMgYSB0aGF1bWF0aW4tbGlrZSBwcm90ZWluIGJ1dCBub3QgdGhlIG9uZSBwcmV2aW91c2x5IHJlcG9ydGVkOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3MDsgIGNpdGF0aW9uX3BhZ2VzPTI2LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWFwcGluZyBvZiBhIG5ldyBnZW5lIGZvciByZXNpc3RhbmNlIHRvIGZyb2dleWUgbGVhZiBzcG90IG9mIHNveWEgYmVhbiBpbiBQZWtpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTczLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc2VhcmNoIE5vdGVzOiBldmFsdWF0aW9uIG9mIHNveWJlYW4gZ2VybXBsYXNtIGZvciByZXNpc3RhbmNlIHRvIGNvcm4gZWFyd29ybeKAk0lWOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Tb3liZWFuIEdlbmV0LiBOZXdzbGV0dDsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgYW5hbHlzaXMgb2Ygc2VlZC1mbG9vZGluZyB0b2xlcmFuY2UgaW4gc295YmVhbiAoR2x5Y2luZSBtYXggW0wuXSBNZXJyLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz01MTQtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBlIGVmZmVjdHMgb24gcHJvbGlmZXJhdGl2ZSBlbWJyeW9nZW5lc2lzIGFuZCBwbGFudCByZWdlbmVyYXRpb24gb2Ygc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW4gVml0cm8gQ2VsbC4gRGV2LiBCaW9sLuKAk1BsYW50OyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTAyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFFUTCB1bmRlcmx5aW5nIHNvbWF0aWMgZW1icnlvZ2VuZXNpcyBjYXBhY2l0eSBvZiBpbW1hdHVyZSBlbWJyeW9zIGluIHNveWJlYW4gKEdseWNpbmUgbWF4IChMLikgTWVyci4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5oZXJpdGFuY2Ugb2Ygc295YmVhbiBzdXNjZXB0aWJpbGl0eSB0byBBZ3JvYmFjdGVyaXVtIHR1bWVmYWNpZW5zIGFuZCBpdHMgcmVsYXRpb25zaGlwIHRvIHRyYW5zZm9ybWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTExNTItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzIvMTIzLzI0NjYyMTQ3L2RzeDA0My5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVXNpbmcgcHJvZ2VueSBvZiBhIGNyb3NzIGJldHdlZW4gSmFwYW5lc2Ugc295YmVhbiBFbnJlaSBhbmQgQ2hpbmVzZSBzb3liZWFuIFBla2luZywgd2UgZGV2ZWxvcGVkIGEgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIGFuZCBjaHJvbW9zb21hbCBzZWdtZW50IHMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ2NTcxNTMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkCpcCPSErc5m8bGDrpFTcpdZSWJk=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f123%2f4657153" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f123%2f4657153" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115733271">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733271">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733273">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733273">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733281">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733281">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733302">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733302">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733337">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733337">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733340">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733340">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733346">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733346">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/2/113/4554776">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/2/137/4583475">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoshi Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takehiko Shimizu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kayo Machita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kayo+Machita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Machita K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Machita K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kayo Machita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasutaka Tsubokura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasutaka+Tsubokura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsubokura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsubokura Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasutaka Tsubokura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhengjun Xia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhengjun+Xia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xia Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xia Z&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Zhengjun Xia</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Breeding Unit, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8517, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tetsuya Yamada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makita Hajika</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Breeding Unit, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8517, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makita+Hajika">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hajika M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hajika M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Makita Hajika</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masao Ishimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masao+Ishimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishimoto M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishimoto M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masao Ishimoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuichi Katayose</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuichi+Katayose">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katayose Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Katayose Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuichi Katayose</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyuya Harada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyuya+Harada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harada K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harada K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kyuya Harada</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akito Kaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Soybean Applied Genomics Research Unit, National Institute of Agrobiological Sciences (NIAS), Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx043-cor1" data-open="dsx043-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 29 838 7452. Fax. +81 29 838 7408. Email: <a href="mailto:kaga@affrc.go.jp" target="_blank">kaga@affrc.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akito+Kaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kaga A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kaga A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akito Kaga</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM2"><div class="fn"><div class="footnote-content">Present address: Faculty of Agriculture, Saga University, Honjyo-10 machi 1, Saga 840-8502, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM3"><div class="fn"><div class="footnote-content">Present address: Soybean and Field Crop Applied Genomics Research Unit, Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM4"><div class="fn"><div class="footnote-content">Present address: Snow Brand Seed Co., Ltd, Chiba Research Station, Inage, Chiba 263-0001, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM5"><div class="fn"><div class="footnote-content">Present address: Key Laboratory of Soybean Molecular Design Breeding, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Harbin 150081, China</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM6"><div class="fn"><div class="footnote-content">Present address: Advanced Genomics Breeding Section, Institute of Crop Science, NARO, Tsukuba, Ibaraki 305-8634, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM7"><div class="fn"><div class="footnote-content">Present address: Department of Biotechnology, Graduate School of Engineering, Osaka University, Suita, Osaka 565-0871, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx043-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 123–136, <a href='https://doi.org/10.1093/dnares/dsx043'>https://doi.org/10.1093/dnares/dsx043</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 November 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 April 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 September 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4657153" href="/dnaresearch/article-pdf/25/2/123/24662147/dsx043.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Satoshi Watanabe, Takehiko Shimizu, Kayo Machita, Yasutaka Tsubokura, Zhengjun Xia, Tetsuya Yamada, Makita Hajika, Masao Ishimoto, Yuichi Katayose, Kyuya Harada, Akito Kaga;  Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei, <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 123–136, <a href='https://doi.org/10.1093/dnares/dsx043'>https://doi.org/10.1093/dnares/dsx043</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4657153?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20of%20a%20high-density%20linkage%20map%20and%20chromosome%20segment%20substitution%20lines%20for%20Japanese%20soybean%20cultivar%20Enrei&amp;publicationDate=2017-11-24&amp;volumeNum=25&amp;issueNum=2&amp;author=Watanabe%2C%20Satoshi%3B%20Shimizu%2C%20Takehiko&amp;startPage=123&amp;endPage=136&amp;contentId=10.1093%2Fdnares%2Fdsx043&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f123%2f4657153" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=115733271 id="115733271" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Using progeny of a cross between Japanese soybean Enrei and Chinese soybean Peking, we developed a high-density linkage map and chromosomal segment substitution lines (CSSLs). The map consists of 2,177 markers with polymorphism information for 32 accessions and provides a detailed genetic framework for these markers. The marker order on the linkage map revealed close agreement with that on the chromosome-scale assembly, Wm82.a2.v1. The differences, especially on Chr. 5 and Chr. 11, in the present map provides information to identify regions in the genome assembly where additional information is required to resolve marker order and assign remaining scaffolds. To cover the entire soybean genome, we used 999 BC<sub>3</sub>F<sub>2</sub> backcross plants and selected 103 CSSLs carrying chromosomal segments from Peking in the genetic background of Enrei. Using these low-genetic-complexity resources, we dissected variation in traits related to flowering, maturity and yield into approximately 50 reproducible quantitative trait loci (QTLs) and evaluated QTLs with small genetic effects as single genetic factors in a uniform genetic background. CSSLs developed in this study may be good starting material for removing the unfavourable characteristics of Peking during pre-breeding and for isolation of genes conferring disease and stress resistance that have not yet been characterized.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;glycine max&quot;">glycine max</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SSR">SSR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="CSSL">CSSL</a></div>                            <h2 scrollto-destination=115733273 id="115733273" class="section-title" >1. Introduction</h2>
<p>Soybean, <em>Glycine max</em> (L.) Merr., is the most important legume and is the fourth crop next to rice, wheat and maize in terms of world crop production. The estimated size of the soybean genome is 1.1 Gb.<a reveal-id="dsx043-B1" data-open="dsx043-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The genome sequence of the US cultivar Williams 82, Glyma0, became available on the Phytozome in January 2008. The first chromosome-scale assembly, Glyma1.01, became available in December 2008, and a new assembly, Wm82.a2.v1, was released in January 2014 (<a class="link link-uri" href="https://phytozome.jgi.doe.gov/pz/portal.html" target="_blank">https://phytozome.jgi.doe.gov/pz/portal.html</a> (1 November 2017, date last accessed)). According to the Phytozome web site, annotations of some genes have been improved on the basis of RNA-seq data and the number of unmapped scaffolds has been reduced by using new assembly methods and constructing high-density linkage maps. Two other soybean chromosome-scale assemblies are available from the NCBI web site (Glycine_max_v1.1 and Glycine_max_v2.0). However, gene names, numbers and genomic positions are not comparable among different assemblies and are sometimes very confusing to users. In the present study, we discuss only the Glyma1.01 and Wm82.a2.v1 assemblies at the Phytozome web site. </p><p>A reference genome sequence is a versatile tool with which to characterize the relationships between genes and agronomically important traits. However, additional genome information and experimental materials suitable for genetic characterization are also needed. New next-generation sequencing (NGS) technologies are expected to yield genomic sequences of a wide variety of soybean germplasms.<a reveal-id="dsx043-B2" data-open="dsx043-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In addition, the development of soybean mutant libraries and the identification of mutants related to agronomically important traits from their phenotypes and reverse-genetic approaches based on NGS are expected to provide new genetic resources.<a reveal-id="dsx043-B3" data-open="dsx043-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The efficient use of this information and experimental materials is necessary for further soybean breeding.</p><p>Recent re-sequencing information has enabled the development of molecular markers for soybean gene discovery and breeding. Among molecular markers, simple sequence repeats (SSRs) and microsatellites are useful tools because of their abundance, multi-allelic features, co-dominant inheritance, high variability and ease of analysis. SSR markers developed by previous studies<a reveal-id="dsx043-B4" data-open="dsx043-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx043-B5" data-open="dsx043-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> have been widely used in soybean breeding programs worldwide and are available from SoyBase (<a class="link link-uri" href="https://www.soybase.org/" target="_blank">https://www.soybase.org/</a> (1 November 2017, date last accessed)). A genetic linkage map is essential for soybean chromosome assembly. The reference sequence Glyma1.01 was assembled on the basis of a consensus linkage map,<a reveal-id="dsx043-B1" data-open="dsx043-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and Wm82.a2.v1 (Glyma2) on the basis of high-density linkage maps.<a reveal-id="dsx043-B6" data-open="dsx043-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> However, chromosomal translocation and inversion in soybean have been reported with cytogenic analysis<a reveal-id="dsx043-B7" data-open="dsx043-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and karyotyping based on fluorescence <em>in situ</em> hybridization has also identified translocations and inversions in several soybean cultivars.<a reveal-id="dsx043-B8" data-open="dsx043-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Comparison of high-density linkage maps is expected to be deepen with our knowledge about chromosomal structure in soybean cultivars.</p><p>Soybean is an important source of traditional staple foods such as tofu, natto, miso and soy sauce in Japan. The unique cuisine, geographical and historical isolation of Japan probably shaped distinct agro-morphological characteristics of Japanese soybeans from those of continental soybeans. Since Enrei is a major cultivar (9% of total soybean cultivation area in Japan in 2014) with seeds of high quality for food processing, previous study<a reveal-id="dsx043-B9" data-open="dsx043-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> sequenced the whole genome of Enrei as a representative Japanese cultivar. The sequencing average read coverage at a locus was 22.2×, and 1,659,041 SNPs and 344,418 insertions/deletions between the Enrei assembly and the reference sequence of Williams 82 were identified. One 4-coumaroyl-CoA-ligase gene (out of 10 genes in the William 82 reference genome), seven chalcone synthase genes (out of 24 genes), three chalcone isomerase genes (out of 16 genes), one flavonol synthase gene (out of 4 genes) and six dihydroflavonol 4-reductase genes (out of 10 genes), those predicted as anthocyanin and flavonoid biosynthesis genes in Williams 82 genome, were not found in the Enrei genome. Hence, the accumulation of genome and marker information for a wide range of soybean elite cultivars and the development of experimental resources to facilitate the evaluation of useful genes that differ from Williams 82 orthologs are necessary to utilize a wide range of the genetic diversity in soybean breeding.</p><p>Peking is a landrace that has been extensively used in a breeding programs because of its resistance to the soybean cyst nematode <em>Heterodera glycines</em>.<a reveal-id="dsx043-B10" data-open="dsx043-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In 1906, the accession was introduced from Beijing, China, into the USA by the US Department of Agriculture (USDA).<a reveal-id="dsx043-B11" data-open="dsx043-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Peking is also resistant to soybean mosaic virus,<a reveal-id="dsx043-B12" data-open="dsx043-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> peanut mottle virus,<a reveal-id="dsx043-B13" data-open="dsx043-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> bacterial blight,<a reveal-id="dsx043-B14" data-open="dsx043-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> frogeye leaf spot (<em>Cercospora sojina</em>),<a reveal-id="dsx043-B15" data-open="dsx043-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> soybean dwarf disease,<a reveal-id="dsx043-B16" data-open="dsx043-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> soybean stem canker,<a reveal-id="dsx043-B17" data-open="dsx043-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> reniform nematode <em>Rotylenchulus reniformis</em>,<a reveal-id="dsx043-B18" data-open="dsx043-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><em>Phytophthora</em> stem and root rot<a reveal-id="dsx043-B19" data-open="dsx043-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and germinates well under wet conditions.<a reveal-id="dsx043-B20" data-open="dsx043-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx043-B21" data-open="dsx043-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> However, the genes responsible for these traits, except for resistance to cyst nematode and phytophthora stem and root rot, remain uncharacterized.</p><p>Many quantitative trait loci (QTLs) related to important agronomic traits such as flowering time, plant height, maturity, seed weight, yield, seed nutrients, seed oil contents and seed protein are summarized in SoyBase. Isolation and characterization of genes for these traits are very important in facilitating MAS with DNA markers tightly linked to a locus or nucleotide polymorphism suitable for distinguishing functional alleles. However, the development of experimental material suitable for positional cloning is time consuming. Among such materials, ‘heterozygous inbred families<a reveal-id="dsx043-B22" data-open="dsx043-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><sup>’</sup> and ‘residual heterozygous lines<a reveal-id="dsx043-B23" data-open="dsx043-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><sup>’</sup> have been used to analyze QTLs as single Mendelian factors for fine mapping. In addition, series of near-isogenic lines (NILs), which have a common genetic background, are advantageous for QTL evaluation. First NIL library, which consisted of a series of NILs, currently referred to as chromosomal segment substitution lines (CSSLs), each having different chromosomal segments originating from a wild donor parent in the genetic background of cultivated tomato, was developed and used for identification of a yield-related QTL.<a reveal-id="dsx043-B24" data-open="dsx043-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> CSSLs were developed in many plant species, including tomato,<a reveal-id="dsx043-B24" data-open="dsx043-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Arabidopsis,<a reveal-id="dsx043-B25" data-open="dsx043-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> rice<a reveal-id="dsx043-B26" data-open="dsx043-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a>), barley,<a reveal-id="dsx043-B27" data-open="dsx043-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> peanut<a reveal-id="dsx043-B28" data-open="dsx043-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> rye,<a reveal-id="dsx043-B29" data-open="dsx043-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> lettuce<a reveal-id="dsx043-B30" data-open="dsx043-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and wild soybean.<a reveal-id="dsx043-B31" data-open="dsx043-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> In soybean, one genetic locus from wild soybean increasing yield was also reported.<a reveal-id="dsx043-B32" data-open="dsx043-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Genomic sequences indicate that the genetic diversity of cultivated soybean is narrow compared with that of landraces and wild soybean (<em>G. soja</em>).<a reveal-id="dsx043-B33" data-open="dsx043-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Although CSSLs carrying segments of the wild soybean genome have been developed,<a reveal-id="dsx043-B31" data-open="dsx043-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> the availability of CSSLs from out of Japan is limited, and the development of CSSLs using various combinations of soybean germplasms would provide novel breeding materials and increase genetic diversity.</p><p>In the present study, using progeny from a cross between the leading Japanese cultivar, Enrei, and the Chinese landrace Peking, we developed a high-density linkage map and CSSLs. Enrei is a representative Japanese cultivar with high quality for food processing, whereas Peking is an excellent Chinese germplasm resistant to various diseases and stresses. The genetic distance between these parents is much larger than that between Japanese landraces and Enrei but smaller than wild soybean and Enrei.<a reveal-id="dsx043-B34" data-open="dsx043-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> In addition to re-sequencing information,<a reveal-id="dsx043-B9" data-open="dsx043-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx043-B35" data-open="dsx043-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> genes controlling flowering time and growth habit have been characterized for these parents.<a reveal-id="dsx043-B36" data-open="dsx043-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Therefore, genomic resources developed in the present study may help to characterize agronomically important genes.</p>                            <h2 scrollto-destination=115733281 id="115733281" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=115733282 id="115733282" class="section-title" >2.1. Plant materials</h3>
<p>A total of 32 Soybean lines including 16 Japanese breeding varieties, two USA breeding varieties, six Japanese landraces, four exotic landraces and four wild soybean accessions were used to identify SSR polymorphisms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). A cross between Enrei [<em>G. max</em>; accession number in National Agriculture and Food Research Organization (NARO) Genebank, Japan: GmJMC025] as the female parent and Peking (GmWMC084) as the male parent was performed in 2005, and an F<sub>2</sub> mapping population was developed. The F<sub>2</sub> population (189 plants) and 20 plants of each parent were grown with an inter-row spacing of 80 cm and a hill spacing of 30 cm in the field at NARO in Tsukuba, Ibaraki, Japan (36°01'25.6”N 140°06'59.1”E). All experimental populations were evaluated at the same location. Seeds were sown on 30 May 2007. Images of the parent plants are shown in <a reveal-id="dsx043-F1" data-open="dsx043-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. The breeding scheme is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>. </p>                        <a id="115733284" scrollto-destination="115733284"></a>
<div id="dsx043-F1" data-id="dsx043-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f1.png?Expires=2147483647&Signature=xahFwcryiNdIJbtmahqX8fUj1jRuSFRKt~TjdryoOUcvEGDjZ9sTxNLqtXxmBw5ocBtqLIOwkXJXta~AVSERWjlx0eC5s9BoKnpjk0a843Bm57fi4JLzt~QPpSDr4XY5wIj44YR~UvLvU8KBTXxhahdCfkClNv4xVYjRiZw5Zk-J0Wmii7-TihsDVqvumIPmL5VVTk~s038KO~TFClaCXAqr44~I~-mmUivFQLXBcP3qfUlpRtzQgWryySl2ztj7qR4pv7vH~v5pG4w7TJs6cLsvX1QKJJK1Z8jYoaCuagTCPoxxehKjP8MtYywr~-4L2CYGODLNjozp1p3laaaUng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Seeds and mature plants of Enrei and Peking. The ruler is in centimeters." path-from-xml="dsx043f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733284/dsx043f1.tif" path-from-xml="dsx043f1.tif" target="_blank">View large</a><a section="115733284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f1.png?Expires=2147483647&Signature=qGpLaRN1syI2V8Q3wxGubLcawJoJFlQs6rGMaY-GMvHyvU~wXOqfD5XnKUsSaRJ9b0ehenzavq0n7KGShxoSpD0OpotHhUhZ5Rj4mXzb97DMBEdg4W7AfxusRG75Rv57Ee~bFrdQsRkXnJouBIPYQIx26KQ6DBpcDe-MUgXzh4YbguyUyULnT-4Aa3t8MvHQcvicDkn4PuFrWIglXRGi33exJbaUUB59MK19jlAHKTj3HJms0mXfACeb~s51wV5SJVgXMmR9ax5vtiubUxwPm7ebOaBoExLWaMtUx25Lvp5OEBGBfVKeCOHqbYwLkIM1-VSprYK7amOfY8WrXxUQXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733284&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Seeds and mature plants of Enrei and Peking. The ruler is in centimeters.</p></div></div><div content-id="dsx043-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f1.png?Expires=2147483647&Signature=xahFwcryiNdIJbtmahqX8fUj1jRuSFRKt~TjdryoOUcvEGDjZ9sTxNLqtXxmBw5ocBtqLIOwkXJXta~AVSERWjlx0eC5s9BoKnpjk0a843Bm57fi4JLzt~QPpSDr4XY5wIj44YR~UvLvU8KBTXxhahdCfkClNv4xVYjRiZw5Zk-J0Wmii7-TihsDVqvumIPmL5VVTk~s038KO~TFClaCXAqr44~I~-mmUivFQLXBcP3qfUlpRtzQgWryySl2ztj7qR4pv7vH~v5pG4w7TJs6cLsvX1QKJJK1Z8jYoaCuagTCPoxxehKjP8MtYywr~-4L2CYGODLNjozp1p3laaaUng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Seeds and mature plants of Enrei and Peking. The ruler is in centimeters." path-from-xml="dsx043f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733284/dsx043f1.tif" path-from-xml="dsx043f1.tif" target="_blank">View large</a><a section="115733284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f1.png?Expires=2147483647&Signature=qGpLaRN1syI2V8Q3wxGubLcawJoJFlQs6rGMaY-GMvHyvU~wXOqfD5XnKUsSaRJ9b0ehenzavq0n7KGShxoSpD0OpotHhUhZ5Rj4mXzb97DMBEdg4W7AfxusRG75Rv57Ee~bFrdQsRkXnJouBIPYQIx26KQ6DBpcDe-MUgXzh4YbguyUyULnT-4Aa3t8MvHQcvicDkn4PuFrWIglXRGi33exJbaUUB59MK19jlAHKTj3HJms0mXfACeb~s51wV5SJVgXMmR9ax5vtiubUxwPm7ebOaBoExLWaMtUx25Lvp5OEBGBfVKeCOHqbYwLkIM1-VSprYK7amOfY8WrXxUQXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733284&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Seeds and mature plants of Enrei and Peking. The ruler is in centimeters.</p></div></div>                            <h3 scrollto-destination=115733285 id="115733285" class="section-title" >2.2. Development of a backcross population and CSSLs</h3>
<p>Enrei was used as the recurrent parent and Peking as the donor parent. Pollen from F<sub>1</sub> plants was used to pollinate Enrei flowers before blooming, and &gt;1,000 BC<sub>1</sub>F<sub>1</sub> seeds were obtained in 2006. All BC<sub>1</sub>F<sub>1</sub> seeds were sown, and 999 BC<sub>1</sub>F<sub>1</sub> plants were crossed with Enrei to produce BC<sub>2</sub>F<sub>1</sub> in 2007 and BC<sub>3</sub>F<sub>1</sub> in 2008. BC<sub>3</sub>F<sub>2</sub> seeds were obtained by self-pollinating each BC<sub>3</sub>F<sub>1</sub> plant; the BC<sub>3</sub>F<sub>2</sub> population (999 plants) was sown on 16 June 2010. Next year, CSSLs (103 lines) were selected from the BC<sub>3</sub>F<sub>2</sub> population (as described below) and grown under natural day length in the same field. CSSLs were sown on 24 June 2011 (10 plants per line) and on 27 June 2012 (4–20 plants per line). Selection of CSSLs was based on the data obtained from 320 SSR markers that covered all chromosomes evenly. The proportion and length of donor chromosomal segments were calculated from genetic distances between DNA markers, and the positions of recombination breakpoints in all chromosomes were calculated in individual lines. Candidate lines with low proportions of donor chromosomal segments per chromosome were first selected, and then 103 CSSLs were selected so that the donor segments of 4–6 lines covered each chromosome.</p>                            <h3 scrollto-destination=115733287 id="115733287" class="section-title" >2.3. Phenotypic evaluation</h3>
<div class="block-child-p">Agronomic traits of each plant in the F<sub>2</sub>, BC<sub>3</sub>F<sub>2</sub> and CSSL populations were evaluated (<a reveal-id="dsx043-T1" data-open="dsx043-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Days to first flowering (DFF) corresponded to the R1 stage.<a reveal-id="dsx043-B37" data-open="dsx043-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Days to flowering of the top raceme (DFT) corresponded to the R2 stage. Days to harvest (DH) corresponded to the R8 stage. Flowering period (FP = DFT − DFF), reproductive period (RP = DH − DFF) were calculated. Plant height (PH), number of pods (NP), one-hundred-seed weight (SWH) and total seed weight (TSW) were measured in each BC<sub>3</sub>F<sub>2</sub> plant and CSSL population. Genetic variance (heritability) was calculated from the phenotypic variance of each population, F<sub>2</sub> and BC<sub>3</sub>F<sub>2,</sub> with that of the parents as follows. <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">B</mi><mi mathvariant="normal" xmlns="">r</mi><mi mathvariant="normal" xmlns="">o</mi><mi mathvariant="normal" xmlns="">a</mi><mi mathvariant="normal" xmlns="">d</mi><mo xmlns="">-</mo><mi mathvariant="normal" xmlns="">s</mi><mi mathvariant="normal" xmlns="">e</mi><mi mathvariant="normal" xmlns="">n</mi><mi mathvariant="normal" xmlns="">s</mi><mi mathvariant="normal" xmlns="">e</mi><mi mathvariant="normal" xmlns=""> </mi><mi mathvariant="normal" xmlns="">h</mi><mi mathvariant="normal" xmlns="">e</mi><mi mathvariant="normal" xmlns="">r</mi><mi mathvariant="normal" xmlns="">i</mi><mi mathvariant="normal" xmlns="">t</mi><mi mathvariant="normal" xmlns="">a</mi><mi mathvariant="normal" xmlns="">b</mi><mi mathvariant="normal" xmlns="">i</mi><mi mathvariant="normal" xmlns="">l</mi><mi mathvariant="normal" xmlns="">i</mi><mi mathvariant="normal" xmlns="">t</mi><mi mathvariant="normal" xmlns="">y</mi><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mo>(</mo><mi mathvariant="normal">p</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">y</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">v</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mo>-</mo><mi mathvariant="normal">p</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">y</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">v</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi><mo>)</mo></mrow><mrow><mi mathvariant="normal">p</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">y</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">v</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal"> </mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow></mfrac></math></div></div></div><div content-id="dsx043-T1" class="table-modal"><div class="table-wrap"><div id="dsx043-T1" data-id="dsx043-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Description of investigated traits, broard sence heritability and number of QTLs detected in the present study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">Trait abbreviation </th><th rowspan="2">QTL abbreviation </th><th rowspan="2">Trait evaluation </th><th colspan="4">Broad-sense heritability<hr /> </th><th colspan="3">Number of QTLs detected in each population<hr /> </th><th colspan="3">Number detected QTLs common between populations<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSLs (2011) </th><th>CSSLs (2012) </th><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSL </th><th>F<sub>2</sub> vs BC<sub>3</sub>F<sub>2</sub> </th><th>F<sub>2</sub> vs CSSL </th><th>BC<sub>3</sub>F<sub>2</sub> vs CSSL </th></tr></thead><tbody><tr><td>Days to first flowering (R1) </td><td>DFF </td><td><em>qDFF</em> </td><td>Days from sowing to first flowering </td><td>97.6% </td><td>92.0% </td><td>77.9% </td><td>89.0% </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td></tr><tr><td>Days to flowering of top raceme (R2) </td><td>DFT </td><td><em>qDFT</em> </td><td>Days from sowing to flowering of the top raceme </td><td>NA </td><td>96.3% </td><td>94.3% </td><td>79.9% </td><td>Not evaluated </td><td>6 </td><td>6 </td><td>Not evaluated </td><td>Not evaluated </td><td>6 </td></tr><tr><td>Flowering period (days from R1 to R2) </td><td>FP </td><td><em>qFP</em> </td><td>R2−R1 </td><td>NA </td><td>89.0% </td><td>55.8% </td><td>51.0% </td><td>Not evaluated </td><td>2 </td><td>2 </td><td>Not evaluated </td><td>Not evaluated </td><td>2 </td></tr><tr><td>Days to harvest (R8) </td><td>DH </td><td><em>qDH</em> </td><td>Days from sowing to harvest (maturity stage) </td><td>89.7% </td><td>35.9% </td><td>94.5% </td><td>61.8% </td><td>5 </td><td>9 </td><td>7 </td><td>5 </td><td>5 </td><td>7 </td></tr><tr><td>Reproductive period (days from R1 to harvest) </td><td>RP </td><td><em>qRP</em> </td><td>Harvesting day–first flowering day </td><td>74.9% </td><td>5.2% </td><td>85.3% </td><td>48.4% </td><td>2 </td><td>6 </td><td>4 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Plant height </td><td>PH </td><td><em>qPH</em> </td><td>Distance from the primary leaf node to the base of the top raceme </td><td>89.8% </td><td>87.7% </td><td>93.7% </td><td>87.1% </td><td>3 </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>4 </td></tr><tr><td>Number of pods </td><td>NP </td><td><em>qNP</em> </td><td>Number of harvested total pods </td><td>90.6% </td><td>76.5% </td><td>81.9% </td><td>66.9% </td><td>2 </td><td>8 </td><td>5 </td><td>2 </td><td>2 </td><td>5 </td></tr><tr><td>Seed weight per plant (total seed weight) </td><td>TSW </td><td><em>qTSW</em> </td><td>Weight of harvested total seed </td><td>NA </td><td>72.3% </td><td>64.5% </td><td>51.8% </td><td>2 </td><td>8 </td><td>8 </td><td>2 </td><td>2 </td><td>8 </td></tr><tr><td>Seed weight (100 seeds) </td><td>SWH </td><td><em>qSWH</em> </td><td>Weight of 100 seeds </td><td>60.3% </td><td>63.3% </td><td>63.6% </td><td>71.2% </td><td>Not evaluated </td><td>7 </td><td>7 </td><td>Not evaluated </td><td>Not evaluated </td><td>7 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td><td>56 </td><td>50 </td><td>20 </td><td>21 </td><td>49 </td></tr></tbody></table></div><div content-id="dsx043-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">Trait abbreviation </th><th rowspan="2">QTL abbreviation </th><th rowspan="2">Trait evaluation </th><th colspan="4">Broad-sense heritability<hr /> </th><th colspan="3">Number of QTLs detected in each population<hr /> </th><th colspan="3">Number detected QTLs common between populations<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSLs (2011) </th><th>CSSLs (2012) </th><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSL </th><th>F<sub>2</sub> vs BC<sub>3</sub>F<sub>2</sub> </th><th>F<sub>2</sub> vs CSSL </th><th>BC<sub>3</sub>F<sub>2</sub> vs CSSL </th></tr></thead><tbody><tr><td>Days to first flowering (R1) </td><td>DFF </td><td><em>qDFF</em> </td><td>Days from sowing to first flowering </td><td>97.6% </td><td>92.0% </td><td>77.9% </td><td>89.0% </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td></tr><tr><td>Days to flowering of top raceme (R2) </td><td>DFT </td><td><em>qDFT</em> </td><td>Days from sowing to flowering of the top raceme </td><td>NA </td><td>96.3% </td><td>94.3% </td><td>79.9% </td><td>Not evaluated </td><td>6 </td><td>6 </td><td>Not evaluated </td><td>Not evaluated </td><td>6 </td></tr><tr><td>Flowering period (days from R1 to R2) </td><td>FP </td><td><em>qFP</em> </td><td>R2−R1 </td><td>NA </td><td>89.0% </td><td>55.8% </td><td>51.0% </td><td>Not evaluated </td><td>2 </td><td>2 </td><td>Not evaluated </td><td>Not evaluated </td><td>2 </td></tr><tr><td>Days to harvest (R8) </td><td>DH </td><td><em>qDH</em> </td><td>Days from sowing to harvest (maturity stage) </td><td>89.7% </td><td>35.9% </td><td>94.5% </td><td>61.8% </td><td>5 </td><td>9 </td><td>7 </td><td>5 </td><td>5 </td><td>7 </td></tr><tr><td>Reproductive period (days from R1 to harvest) </td><td>RP </td><td><em>qRP</em> </td><td>Harvesting day–first flowering day </td><td>74.9% </td><td>5.2% </td><td>85.3% </td><td>48.4% </td><td>2 </td><td>6 </td><td>4 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Plant height </td><td>PH </td><td><em>qPH</em> </td><td>Distance from the primary leaf node to the base of the top raceme </td><td>89.8% </td><td>87.7% </td><td>93.7% </td><td>87.1% </td><td>3 </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>4 </td></tr><tr><td>Number of pods </td><td>NP </td><td><em>qNP</em> </td><td>Number of harvested total pods </td><td>90.6% </td><td>76.5% </td><td>81.9% </td><td>66.9% </td><td>2 </td><td>8 </td><td>5 </td><td>2 </td><td>2 </td><td>5 </td></tr><tr><td>Seed weight per plant (total seed weight) </td><td>TSW </td><td><em>qTSW</em> </td><td>Weight of harvested total seed </td><td>NA </td><td>72.3% </td><td>64.5% </td><td>51.8% </td><td>2 </td><td>8 </td><td>8 </td><td>2 </td><td>2 </td><td>8 </td></tr><tr><td>Seed weight (100 seeds) </td><td>SWH </td><td><em>qSWH</em> </td><td>Weight of 100 seeds </td><td>60.3% </td><td>63.3% </td><td>63.6% </td><td>71.2% </td><td>Not evaluated </td><td>7 </td><td>7 </td><td>Not evaluated </td><td>Not evaluated </td><td>7 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td><td>56 </td><td>50 </td><td>20 </td><td>21 </td><td>49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Partially confirmed QTLs are included.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx043-T1" data-id="dsx043-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Description of investigated traits, broard sence heritability and number of QTLs detected in the present study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">Trait abbreviation </th><th rowspan="2">QTL abbreviation </th><th rowspan="2">Trait evaluation </th><th colspan="4">Broad-sense heritability<hr /> </th><th colspan="3">Number of QTLs detected in each population<hr /> </th><th colspan="3">Number detected QTLs common between populations<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSLs (2011) </th><th>CSSLs (2012) </th><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSL </th><th>F<sub>2</sub> vs BC<sub>3</sub>F<sub>2</sub> </th><th>F<sub>2</sub> vs CSSL </th><th>BC<sub>3</sub>F<sub>2</sub> vs CSSL </th></tr></thead><tbody><tr><td>Days to first flowering (R1) </td><td>DFF </td><td><em>qDFF</em> </td><td>Days from sowing to first flowering </td><td>97.6% </td><td>92.0% </td><td>77.9% </td><td>89.0% </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td></tr><tr><td>Days to flowering of top raceme (R2) </td><td>DFT </td><td><em>qDFT</em> </td><td>Days from sowing to flowering of the top raceme </td><td>NA </td><td>96.3% </td><td>94.3% </td><td>79.9% </td><td>Not evaluated </td><td>6 </td><td>6 </td><td>Not evaluated </td><td>Not evaluated </td><td>6 </td></tr><tr><td>Flowering period (days from R1 to R2) </td><td>FP </td><td><em>qFP</em> </td><td>R2−R1 </td><td>NA </td><td>89.0% </td><td>55.8% </td><td>51.0% </td><td>Not evaluated </td><td>2 </td><td>2 </td><td>Not evaluated </td><td>Not evaluated </td><td>2 </td></tr><tr><td>Days to harvest (R8) </td><td>DH </td><td><em>qDH</em> </td><td>Days from sowing to harvest (maturity stage) </td><td>89.7% </td><td>35.9% </td><td>94.5% </td><td>61.8% </td><td>5 </td><td>9 </td><td>7 </td><td>5 </td><td>5 </td><td>7 </td></tr><tr><td>Reproductive period (days from R1 to harvest) </td><td>RP </td><td><em>qRP</em> </td><td>Harvesting day–first flowering day </td><td>74.9% </td><td>5.2% </td><td>85.3% </td><td>48.4% </td><td>2 </td><td>6 </td><td>4 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Plant height </td><td>PH </td><td><em>qPH</em> </td><td>Distance from the primary leaf node to the base of the top raceme </td><td>89.8% </td><td>87.7% </td><td>93.7% </td><td>87.1% </td><td>3 </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>4 </td></tr><tr><td>Number of pods </td><td>NP </td><td><em>qNP</em> </td><td>Number of harvested total pods </td><td>90.6% </td><td>76.5% </td><td>81.9% </td><td>66.9% </td><td>2 </td><td>8 </td><td>5 </td><td>2 </td><td>2 </td><td>5 </td></tr><tr><td>Seed weight per plant (total seed weight) </td><td>TSW </td><td><em>qTSW</em> </td><td>Weight of harvested total seed </td><td>NA </td><td>72.3% </td><td>64.5% </td><td>51.8% </td><td>2 </td><td>8 </td><td>8 </td><td>2 </td><td>2 </td><td>8 </td></tr><tr><td>Seed weight (100 seeds) </td><td>SWH </td><td><em>qSWH</em> </td><td>Weight of 100 seeds </td><td>60.3% </td><td>63.3% </td><td>63.6% </td><td>71.2% </td><td>Not evaluated </td><td>7 </td><td>7 </td><td>Not evaluated </td><td>Not evaluated </td><td>7 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td><td>56 </td><td>50 </td><td>20 </td><td>21 </td><td>49 </td></tr></tbody></table></div><div content-id="dsx043-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th rowspan="2">Trait abbreviation </th><th rowspan="2">QTL abbreviation </th><th rowspan="2">Trait evaluation </th><th colspan="4">Broad-sense heritability<hr /> </th><th colspan="3">Number of QTLs detected in each population<hr /> </th><th colspan="3">Number detected QTLs common between populations<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSLs (2011) </th><th>CSSLs (2012) </th><th>F<sub>2</sub> </th><th>BC<sub>3</sub>F<sub>2</sub> </th><th>CSSL </th><th>F<sub>2</sub> vs BC<sub>3</sub>F<sub>2</sub> </th><th>F<sub>2</sub> vs CSSL </th><th>BC<sub>3</sub>F<sub>2</sub> vs CSSL </th></tr></thead><tbody><tr><td>Days to first flowering (R1) </td><td>DFF </td><td><em>qDFF</em> </td><td>Days from sowing to first flowering </td><td>97.6% </td><td>92.0% </td><td>77.9% </td><td>89.0% </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td><td>7 </td><td>6 </td></tr><tr><td>Days to flowering of top raceme (R2) </td><td>DFT </td><td><em>qDFT</em> </td><td>Days from sowing to flowering of the top raceme </td><td>NA </td><td>96.3% </td><td>94.3% </td><td>79.9% </td><td>Not evaluated </td><td>6 </td><td>6 </td><td>Not evaluated </td><td>Not evaluated </td><td>6 </td></tr><tr><td>Flowering period (days from R1 to R2) </td><td>FP </td><td><em>qFP</em> </td><td>R2−R1 </td><td>NA </td><td>89.0% </td><td>55.8% </td><td>51.0% </td><td>Not evaluated </td><td>2 </td><td>2 </td><td>Not evaluated </td><td>Not evaluated </td><td>2 </td></tr><tr><td>Days to harvest (R8) </td><td>DH </td><td><em>qDH</em> </td><td>Days from sowing to harvest (maturity stage) </td><td>89.7% </td><td>35.9% </td><td>94.5% </td><td>61.8% </td><td>5 </td><td>9 </td><td>7 </td><td>5 </td><td>5 </td><td>7 </td></tr><tr><td>Reproductive period (days from R1 to harvest) </td><td>RP </td><td><em>qRP</em> </td><td>Harvesting day–first flowering day </td><td>74.9% </td><td>5.2% </td><td>85.3% </td><td>48.4% </td><td>2 </td><td>6 </td><td>4 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Plant height </td><td>PH </td><td><em>qPH</em> </td><td>Distance from the primary leaf node to the base of the top raceme </td><td>89.8% </td><td>87.7% </td><td>93.7% </td><td>87.1% </td><td>3 </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>4 </td></tr><tr><td>Number of pods </td><td>NP </td><td><em>qNP</em> </td><td>Number of harvested total pods </td><td>90.6% </td><td>76.5% </td><td>81.9% </td><td>66.9% </td><td>2 </td><td>8 </td><td>5 </td><td>2 </td><td>2 </td><td>5 </td></tr><tr><td>Seed weight per plant (total seed weight) </td><td>TSW </td><td><em>qTSW</em> </td><td>Weight of harvested total seed </td><td>NA </td><td>72.3% </td><td>64.5% </td><td>51.8% </td><td>2 </td><td>8 </td><td>8 </td><td>2 </td><td>2 </td><td>8 </td></tr><tr><td>Seed weight (100 seeds) </td><td>SWH </td><td><em>qSWH</em> </td><td>Weight of 100 seeds </td><td>60.3% </td><td>63.3% </td><td>63.6% </td><td>71.2% </td><td>Not evaluated </td><td>7 </td><td>7 </td><td>Not evaluated </td><td>Not evaluated </td><td>7 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td><td>56 </td><td>50 </td><td>20 </td><td>21 </td><td>49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Partially confirmed QTLs are included.</p></div></div></div></div></div>                            <h3 scrollto-destination=115733289 id="115733289" class="section-title" >2.4. Total genomic DNA extraction</h3>
<p>DNA of parents and F<sub>2</sub> plants was extracted from fresh leaves (3 g) by a Cetyltrimethylammonium Bromide (CTAB) protocol.<a reveal-id="dsx043-B38" data-open="dsx043-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> DNA of cultivars and BC<sub>3</sub>F<sub>2</sub> plants was isolated as described previously.<a reveal-id="dsx043-B39" data-open="dsx043-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> DNA was quantified on a fluorescence microplate reader (ARVO; Perkin Elmer, Boston, MA, USA) according to the manufacturer‘s instructions and was adjusted to 50 and 20 ng/μl for SSR and SNP analyses, respectively.</p>                            <h3 scrollto-destination=115733291 id="115733291" class="section-title" >2.5. SSR marker analysis</h3>
<p><em>Marker design</em>: SSR core motifs in Glyma1.01 and in large scaffolds (&gt;100 kb) of Glyma0 were extracted in the read2Marker program with default parameters.<a reveal-id="dsx043-B40" data-open="dsx043-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Primer pairs to amplify the core motifs were designed in Primer3 software.<a reveal-id="dsx043-B41" data-open="dsx043-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Three types of primer pairs with different amplicon sizes (small, 80–210 bp; medium, 211–340 bp; large, 341–500 bp) were designed; the parameter settings were Opt_Tm = 60, Min_Tm = 53, Max_Tm = 70 and Max_Poly_X = 3. Sequences of the primer pairs were searched against Glyma1.01 to determine the number of binding sites, amplicon sizes and locations in Genome tester software<a reveal-id="dsx043-B42" data-open="dsx043-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> with default parameters until a single amplicon was obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The SSR motifs that were consistent with BARC soybean potential SSR markers in SoyBase and EST-SSR markers<a reveal-id="dsx043-B43" data-open="dsx043-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> are denoted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p><p><em>Detection with fluorescently labelled primers</em>: Primers were labelled with 6-FAM, HEX or NED fluorescent dyes. Multiplex PCR mixture (6 µl) contained DNA (10 ng/µl), 0.1 unit/µl of <em>Taq</em> DNA polymerase (Finnzymes), 1× Optimized <em>Taq</em> buffer, primer mix (0.02–0.2 µM each), 200 µM dNTPs and 1.6 M betaine. PCR was performed on a GeneAmp 9700 amplifier (Applied Biosystems, Foster City, USA) as follows: 1 cycle of initial denaturation at 95 °C for 2 min; 40 cycles of denaturation at 94 °C for 30 s, annealing at 55 °C for 2 min and extension at 72 °C for 1 min; and final extension at 72 °C for 3 min. The PCR product was diluted with water (1 : 10), and 1 µl was added to a mixture of Hi-Di formamide (10 µl) and GeneScan 400HD ROX size standard (0.2 µl). The sample was separated on an ABI3730 capillary sequencer (Applied Biosystems). PCR fragments detected simultaneously with three dyes were resolved in ABI GeneMapper v. 4.0 software (Applied Biosystems). To measure allelic size, relative fluorescent units and sizes (bp) of the highest stutter peaks of the 32 accessions were sorted by peak size rounded to integer numbers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>).</p><p><em>Detection with modified universal fluorescently labelled (UFL) primer</em>: The UFL method<a reveal-id="dsx043-B44" data-open="dsx043-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> was modified to attain multiplex amplification. Short polylinker sequences from the pBluescript vector were used as queries in blastn searches<a reveal-id="dsx043-B45" data-open="dsx043-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> against Glyma1.01; the parameter settings were word_size = 7, evalue = 0.1 and perc_identity = 90. Three oligonucleotide tag sequences that did not show any hits in the soybean genome were used instead of the original M13 universal primer and were fluorescently labelled: 5′-CCACCGACGTGTCGCAC with 6-FAM, CCGTGCAGTCCGTCAGC with HEX and GGTGGCGACTCCTGGAG with NED (all dyes from Applied Biosystems). The concentrations of the oligonucleotide tag sequence (0.04 µM), 5′-tagged forward primer (0.04 µM) and unlabelled reverse primer (0.4 µM each) per marker were optimized to attain multiplex amplification. PCR conditions, detection of PCR fragments, genotyping and allele sizing were as described above.</p><p><em>Genotyping of the mapping population:</em> PCR conditions were optimized to attain multiplex amplification for 12–18 fluorescently labelled SSR markers and 9–12 UFL SSR markers. Multiplex PCR mixture (3–5 µl) contained total DNA (50 ng/µl), 1× Multiplex PCR Mix (Qiagen), 1× Q solution and primer mix (described above). The following conditions were used: initial denaturation at 95 °C for 15 min; 18 cycles total of 94 °C for 30 s and 3 cycles each of 68, 66, 64, 62, 60 and 58 °C for 3 min; then 40 cycles of 30 s at 94 °C, 3 min at 55 °C and 1 min at 72 °C; and final extension at 72 °C for 10 min. The primer concentration for fluorescently labelled SSR markers can be reduced to 1/10. Genotyping conditions were as for SSR marker detection.</p>                            <h3 scrollto-destination=115733296 id="115733296" class="section-title" >2.6. SNP marker analysis</h3>
<p>Sequence-tagged sites containing SNP information developed<a reveal-id="dsx043-B46" data-open="dsx043-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> were used as queries in blastn searches against Glyma1.01 with default parameters. Multiplex assays for 1,000 randomly selected SNPs distributed throughout the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) were designed to amplify low-copy sequences in Sequenom Assay Design 3.1 software (Sequenom). The Sequenom MassARRAY system<a reveal-id="dsx043-B47" data-open="dsx043-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> was used for SNP genotyping. Multiplex PCR followed by template-directed single base extension at each SNP site was conducted with a MassARRAY iPLEX Gold kit (Sequenom) following the manufacturer‘s protocol. The genotypes were determined in MassARRAY Typer 4.0 software (Sequenom).</p>                            <h3 scrollto-destination=115733298 id="115733298" class="section-title" >2.7. Linkage map construction and QTL detection</h3>
<p>The linkage maps of F<sub>2</sub> and BC<sub>3</sub>F<sub>2</sub> populations were constructed by using JoinMap v. 4.0 software.<a reveal-id="dsx043-B48" data-open="dsx043-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> The logarithm of odds (LOD) threshold for grouping of DNA markers was 4.0. The marker order was determined using the maximum likelihood mapping algorithm. The recombination frequency was converted into genetic distance (cM) using the Haldane mapping function. Marker genotypes were examined by eye to check whether the pattern of marker segregation changed gradually at each round of map construction in order to ensure the correct marker order. When a discrepancy between the marker order and chromosome assembly was found, additional markers were integrated into the position to confirm the discrepancies. QTLs were analyzed by composite interval mapping implemented in the R/qtl package<a reveal-id="dsx043-B49" data-open="dsx043-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> with threshold values (<em>P </em>&lt; 0.1) that were calculated by the 1,000-permutation test for each trait. The detected peak positions were used for the ‘refineqtl’ and ‘fitqtl’ functions to estimate the maximum-likelihood position for each QTL model, the effects of each QTL and genetic variance. The phenotypic values of CSSLs and Enrei were evaluated in 2011 and 2012 using the Tukey-Kramer test in R v. 3.3.1 software<a reveal-id="dsx043-B50" data-open="dsx043-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> with significance level, <em>P </em>&lt;<em> </em>0.01.</p>                            <h3 scrollto-destination=115733300 id="115733300" class="section-title" >2.8. Comparison of the linkage map with the chromosome assembly</h3>
<p>The positions of primer sequences of mapped markers on Glyma1.01 and Wm82.a2.v1 were estimated by similarity searches using Genome tester and/or blastn described above. The first base position of either forward or reverse primer sequences on the coordinates of the chromosome assembly and the size and number of expected PCR products are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. When multiple polymorphic fragments are amplified by a primer set, we added the 0.1, 0.2, 0.3 suffixes after name of the marker mapped to different linkage groups. In the comparison of the linkage map with the chromosome assembly, we omitted markers being far from the expected position based on information of surrounding markers and then used only information of the marker position of chromosome assembly which marker designed in Glyma1.01 and Wm82.a2.v1. Thirty-one public SSR markers for which only information on primer sequences but difficult to locate precise position on chromosome assemblies were not included. The Marey map approach<a reveal-id="dsx043-B51" data-open="dsx043-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> was used to visualize the corresponding positions between the linkage map and the genome assemblies. Genetic positions of the markers were interpolated using the cubic spline method with default parameters settings in MareyMap version 1.3.3.<a reveal-id="dsx043-B52" data-open="dsx043-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a></p>                            <h2 scrollto-destination=115733302 id="115733302" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=115733303 id="115733303" class="section-title" >3.1. DNA markers</h3>
<p>SSR markers has been used in MAS because of multi allelic behavior, easy handling and low cost. Information on their polymorphism, genotyping quality and genetic positions is useful for marker selection. We identified 171,915 SSR loci (62,739 di-, 41, 696 tri-, 154 tetranucleotide and 67,326 compound loci) in the soybean reference sequence Glyma1.01.<a reveal-id="dsx043-B1" data-open="dsx043-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Because extended regions in the soybean genome are duplicated as a result of ancient polyploidization,<a reveal-id="dsx043-B1" data-open="dsx043-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> novel primer pairs for 148,569 SSR loci were designed to amplify single PCR products to avoid analytical complexity (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Polymorphisms of 2,235 SSR loci and that of previously developed 982 SNP markers<a reveal-id="dsx043-B46" data-open="dsx043-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> were evaluated in 32 soybean germplasms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Sizes of amplified PCR product containing SSRs and nucleotides for SNPs, the number of PCR products, polymorphic information content and genetic positions estimated from the high-density linkage map are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2</a></span> and S3.</p>                            <h3 scrollto-destination=115733305 id="115733305" class="section-title" >3.2. High-density linkage map</h3>
<p>A high-density linkage map was constructed by using a single F<sub>2</sub> mapping population. Genomic SSR markers,<a reveal-id="dsx043-B4" data-open="dsx043-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx043-B5" data-open="dsx043-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> EST-SSR markers<a reveal-id="dsx043-B43" data-open="dsx043-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> and the new SSR markers described above were incorporated into the map to cover the entire genome. The map spans 2885.7 cM and contains 1,667 SSR and 510 SNP markers (<a reveal-id="dsx043-F2" data-open="dsx043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, left white bars); the average marker distance is 1.3 cM and maximum distance is 6.8 cM. Severe segregation distortion (<em>P</em> ≤ 0.001) was observed for the markers on Gm06 (115.5–118.4 cM, corresponding to 46.7–47.3 Mbp) and Gm11 (116.3–119.3 cM, 36.1–36.2 Mbp) (<a reveal-id="dsx043-F2" data-open="dsx043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, blue lines). In these regions, the frequency of Peking alleles was less than that of Enrei alleles. </p>                        <a id="115733307" scrollto-destination="115733307"></a>
<div id="dsx043-F2" data-id="dsx043-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f2.png?Expires=2147483647&Signature=Zo-Z-dSg6JmfcxExsq2ucmNbTbmElhU9UTQbPjIGO~Olyf7JfQGByu3y53iZYfGDHyozHMoV~aI7mp9Bxe1qONDoRh~qS7FGUBNPqfA~8B2vYOq-hkHOESVqLVz1SozuxHu3UV1g0NSL1AiIt9kQBt4nVprvo7HzILdkzw37TCc2Fc~RRNg8gIK00eqNL2t0wyfaLG8q8RqA4kJaJIJBKHJQ7CmIscP6B03Wkfm-2ksW9aQegHiP22ctzqRCgmZAkkXYeTnw6GZcYL9~Nn4VQzeKEOXoAv3beK580AKSKBNAto73sObS-O3j5LRBPL6eL5uuT04rscehSNeGerZl3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A high-density genetic linkage map between Enrei and Peking (left) aligned with a physical map of Williams 82 (Glyma1.01) (right), with marker locations connected by black lines. The regions with suppressed recombination, shown on the same scale on the right, are connected with the linkage map by orange lines and may correspond to the pericentromeric regions. Red trapezoids indicate discrepancies between genetic and physical maps; blue bars indicate regions with severe segregation distortion. The discrepancies that have been resolved in the genome assembly Wm82.a2.v1 are indicated by red dot trapezoids." path-from-xml="dsx043f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733307/dsx043f2.tif" path-from-xml="dsx043f2.tif" target="_blank">View large</a><a section="115733307" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f2.png?Expires=2147483647&Signature=3UccgRGC2jkmyvcXnl6WpTWvuCAUJcmuqVb8qZIE1Yj7sXk~0yFWUsZQkU8wZaU4Awr~68IzAfB9NUmo58EVXKqRIfsxAa4sy25SCCmmAOT8OICYlYkAsD4m0Ly88ASqa5TGxpSPhcDCo2Vaf9a4a4kP0sPH2QymDT6gp0pJy-gCEJuYcU8v~W5qh1Ki6gydFTZDd4IrICUYxs6V3rqktyS-2bJANatrMHe39BLBf8Vrmq9kWZ6enUUlki4BnldQKB2k4vHuCmOGWSi~sv3D8PXncujIbyZmDbDCQT-wmMXDUMDAyomE1Fa22s4~etEHueFPHXxDhGNA3sOpxY3tyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733307&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A high-density genetic linkage map between Enrei and Peking (left) aligned with a physical map of Williams 82 (Glyma1.01) (right), with marker locations connected by black lines. The regions with suppressed recombination, shown on the same scale on the right, are connected with the linkage map by orange lines and may correspond to the pericentromeric regions. Red trapezoids indicate discrepancies between genetic and physical maps; blue bars indicate regions with severe segregation distortion. The discrepancies that have been resolved in the genome assembly Wm82.a2.v1 are indicated by red dot trapezoids.</p></div></div><div content-id="dsx043-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f2.png?Expires=2147483647&Signature=Zo-Z-dSg6JmfcxExsq2ucmNbTbmElhU9UTQbPjIGO~Olyf7JfQGByu3y53iZYfGDHyozHMoV~aI7mp9Bxe1qONDoRh~qS7FGUBNPqfA~8B2vYOq-hkHOESVqLVz1SozuxHu3UV1g0NSL1AiIt9kQBt4nVprvo7HzILdkzw37TCc2Fc~RRNg8gIK00eqNL2t0wyfaLG8q8RqA4kJaJIJBKHJQ7CmIscP6B03Wkfm-2ksW9aQegHiP22ctzqRCgmZAkkXYeTnw6GZcYL9~Nn4VQzeKEOXoAv3beK580AKSKBNAto73sObS-O3j5LRBPL6eL5uuT04rscehSNeGerZl3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A high-density genetic linkage map between Enrei and Peking (left) aligned with a physical map of Williams 82 (Glyma1.01) (right), with marker locations connected by black lines. The regions with suppressed recombination, shown on the same scale on the right, are connected with the linkage map by orange lines and may correspond to the pericentromeric regions. Red trapezoids indicate discrepancies between genetic and physical maps; blue bars indicate regions with severe segregation distortion. The discrepancies that have been resolved in the genome assembly Wm82.a2.v1 are indicated by red dot trapezoids." path-from-xml="dsx043f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733307/dsx043f2.tif" path-from-xml="dsx043f2.tif" target="_blank">View large</a><a section="115733307" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f2.png?Expires=2147483647&Signature=3UccgRGC2jkmyvcXnl6WpTWvuCAUJcmuqVb8qZIE1Yj7sXk~0yFWUsZQkU8wZaU4Awr~68IzAfB9NUmo58EVXKqRIfsxAa4sy25SCCmmAOT8OICYlYkAsD4m0Ly88ASqa5TGxpSPhcDCo2Vaf9a4a4kP0sPH2QymDT6gp0pJy-gCEJuYcU8v~W5qh1Ki6gydFTZDd4IrICUYxs6V3rqktyS-2bJANatrMHe39BLBf8Vrmq9kWZ6enUUlki4BnldQKB2k4vHuCmOGWSi~sv3D8PXncujIbyZmDbDCQT-wmMXDUMDAyomE1Fa22s4~etEHueFPHXxDhGNA3sOpxY3tyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733307&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A high-density genetic linkage map between Enrei and Peking (left) aligned with a physical map of Williams 82 (Glyma1.01) (right), with marker locations connected by black lines. The regions with suppressed recombination, shown on the same scale on the right, are connected with the linkage map by orange lines and may correspond to the pericentromeric regions. Red trapezoids indicate discrepancies between genetic and physical maps; blue bars indicate regions with severe segregation distortion. The discrepancies that have been resolved in the genome assembly Wm82.a2.v1 are indicated by red dot trapezoids.</p></div></div><p>We compared marker locations on the linkage map (<a reveal-id="dsx043-F2" data-open="dsx043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, left white bars) with those on Glyma1.01 (<a reveal-id="dsx043-F2" data-open="dsx043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, right black bars) and Wm82.a2.v1 assemblies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Because important flowering- and yield-related QTLs in the region of Gm11 (11.0–14.8 Mb) of Glyma1.01 assembly are absent on Chr. 11 of Wm82.a2.v1, we discussed based on the Glyma1.01 assembly. The corresponding region was found to be included in two unassigned scaffolds of Wm82.a2.v1 assembly, scaffold_21 and scaffold_32 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). In total, 2,155 of 2,177 markers (99%) were anchored on Glyma1.01. The short-range order of most markers on the linkage map closely agreed with that of their physical positions in the genome assembly, but a wide range of regions differed at the distal ends of chromosomes. For example, reverse marker order on Gm05, Gm11, Gm13, Gm14 and Gm19 was observed at the distal ends of the corresponding linkage groups (<a reveal-id="dsx043-F2" data-open="dsx043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, red rectangles). In particular, the orientation of the top of Gm13 (14.8–15.6 Mb) containing nucleolar organizer region (NOR)<a reveal-id="dsx043-B53" data-open="dsx043-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> was reversed in our map. Similarly, by using fluorescent CentGm probes, the reverse orientation of this region was previously detected in a cytogenetic study of Peking.<a reveal-id="dsx043-B54" data-open="dsx043-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> These large discrepancies have been resolved in Wm82.a2.v1,<a reveal-id="dsx043-B6" data-open="dsx043-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> although some differences, especially on Chr. 05 (8.6–22.0 Mb) and Chr. 11 (11.1–28.0 Mb), remain. As for Chr. 11, the two unassigned scaffolds described above are not integrated into the two high density linkage maps,<a reveal-id="dsx043-B6" data-open="dsx043-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> because no SNP marker to anchor the scaffolds is not available. In contrast, order of SNP markers in the region of Chr. 05 are consistent among the two maps, therefore, the discrepancy with information of the present map might reflect the genomic differences in different accessions. Thus, the differences in the present map provides information to identify regions in the genome assembly where additional information is required to resolve marker order and assign remaining scaffolds.</p>                            <h3 scrollto-destination=115733309 id="115733309" class="section-title" >3.3. Relationships between physical and genetic distances</h3>
<p>Integration of markers that have been used for MAS and QTL mapping by breeders and researchers into the high-density linkage map allowed us to determine relative genetic and physical relationships through marker positions. Corresponding position between the linkage map and the genome assembly was visualized by the Marey map approach.<a reveal-id="dsx043-B51" data-open="dsx043-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> The corresponding physical distance to genetic distance was ∼360 kb/cM, assuming a genome size of ∼1.1 Gb, although positional biases (50 kb/cM–7 Mb/cM) were found. The ratio of physical to genetic distance varied considerably depending on the chromosomal region, and looked like a sigmoid curve. For example, the ratio in the middle regions of Gm06 (100–110 cM) and Gm11 (94–103 cM) was &gt;10 times that in the other regions (<a reveal-id="dsx043-F3" data-open="dsx043-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Such regions with highly suppressed recombination flanking the centromeres are termed the pericentromeres. In each chromosome, highly suppressed recombination between markers was observed in the pericentromere, but the extent of suppression differed among chromosomes (<a reveal-id="dsx043-F2" data-open="dsx043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). A pair of markers in such regions would provide insufficient information on recombination for genetic mapping even if they are located physically far from each other in the reference sequence. Therefore, information on recombination frequency across the genome is useful for genome-wide association study and for marker choice for MAS and QTL mapping. We estimated the genetic positions of all markers from the Marey map (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S3</a></span>). The choice of markers based on their genetic positions rather than their physical positions in the reference sequences would reduce the cost of MAS for breeding and QTL mapping. </p>                        <a id="115733311" scrollto-destination="115733311"></a>
<div id="dsx043-F3" data-id="dsx043-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f3.png?Expires=2147483647&Signature=KI~bcD5G6XeF2VY0-LAO5BlzWNrpdY7wV~W20CB4DN6sdELF64JPWZw3HqCtE3yia3L5Fq2Kw8HOZvtcKs7QgXXy2AakORpJaFEEfLe4McHjnxI3XZ-3S32uKqttefGS4NreW3oG~a3fVpKvePxoAafvS5IFasLrBOsliHp1ncwITJI0ODadNvMd1Qewhivl6u3hWirc5cuVlPLcKhgVbOw7yzme~xaPqoTcVH2IsqUjev8TNxHqIZ-dr2Rl-B5lb6Pj8JjcspHS~IMbiz4zq95Z-6Zh-fI3VrAGDqjbcpHAxfbSTzHMk9L~KsiTvoEGE598x4PwuvYQv2u0Kq~yTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Marey maps51 for chromosomes Gm06 and Gm11. Small dots indicate marker locations. Black boxes indicate the locations of the centromere repeat sequences. The recombination rate apparently decreases in the pericentromeric region. Arrow indicates the location of the E1 gene." path-from-xml="dsx043f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733311/dsx043f3.tif" path-from-xml="dsx043f3.tif" target="_blank">View large</a><a section="115733311" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f3.png?Expires=2147483647&Signature=Ka4USSK2ku7ONqxp4weO8i2NlZ7n5UGASSBguXwh4KM9S6CK5eGW58bsXhA3AxMnXKe7h0sX~~mCV533kUeTb-BrMEgB-RxgCfSI2G-0JiFuCmGBsCQfrB241xyoWTOcKzan4r7VT0DQj2q~K9wgWUuH4GcX0CY7Jci746WdCW2AlgybtRBfvjH85P9RuBgnoYYRssr5X72T2dD8EnUsa6MHgzfAbvqlWgNQuuNG7mIrFSCwp~8wQUJ3Xn1Lw1peQwJZGOVSqvJZMLh1skrPQz1Cxr3aWmbPmq1eIUtj6SInte4V0qdgzWB-HkuTSBqujCngH2j-u~EIh2grXF4clA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733311&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Marey maps<a reveal-id="dsx043-B51" data-open="dsx043-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> for chromosomes Gm06 and Gm11. Small dots indicate marker locations. Black boxes indicate the locations of the centromere repeat sequences. The recombination rate apparently decreases in the pericentromeric region. Arrow indicates the location of the <em>E1</em> gene.</p></div></div><div content-id="dsx043-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f3.png?Expires=2147483647&Signature=KI~bcD5G6XeF2VY0-LAO5BlzWNrpdY7wV~W20CB4DN6sdELF64JPWZw3HqCtE3yia3L5Fq2Kw8HOZvtcKs7QgXXy2AakORpJaFEEfLe4McHjnxI3XZ-3S32uKqttefGS4NreW3oG~a3fVpKvePxoAafvS5IFasLrBOsliHp1ncwITJI0ODadNvMd1Qewhivl6u3hWirc5cuVlPLcKhgVbOw7yzme~xaPqoTcVH2IsqUjev8TNxHqIZ-dr2Rl-B5lb6Pj8JjcspHS~IMbiz4zq95Z-6Zh-fI3VrAGDqjbcpHAxfbSTzHMk9L~KsiTvoEGE598x4PwuvYQv2u0Kq~yTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Marey maps51 for chromosomes Gm06 and Gm11. Small dots indicate marker locations. Black boxes indicate the locations of the centromere repeat sequences. The recombination rate apparently decreases in the pericentromeric region. Arrow indicates the location of the E1 gene." path-from-xml="dsx043f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733311/dsx043f3.tif" path-from-xml="dsx043f3.tif" target="_blank">View large</a><a section="115733311" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f3.png?Expires=2147483647&Signature=Ka4USSK2ku7ONqxp4weO8i2NlZ7n5UGASSBguXwh4KM9S6CK5eGW58bsXhA3AxMnXKe7h0sX~~mCV533kUeTb-BrMEgB-RxgCfSI2G-0JiFuCmGBsCQfrB241xyoWTOcKzan4r7VT0DQj2q~K9wgWUuH4GcX0CY7Jci746WdCW2AlgybtRBfvjH85P9RuBgnoYYRssr5X72T2dD8EnUsa6MHgzfAbvqlWgNQuuNG7mIrFSCwp~8wQUJ3Xn1Lw1peQwJZGOVSqvJZMLh1skrPQz1Cxr3aWmbPmq1eIUtj6SInte4V0qdgzWB-HkuTSBqujCngH2j-u~EIh2grXF4clA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733311&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Marey maps<a reveal-id="dsx043-B51" data-open="dsx043-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> for chromosomes Gm06 and Gm11. Small dots indicate marker locations. Black boxes indicate the locations of the centromere repeat sequences. The recombination rate apparently decreases in the pericentromeric region. Arrow indicates the location of the <em>E1</em> gene.</p></div></div><p>Surprisingly, low-recombination regions covered ∼555 Mb (∼60%) of the published soybean genome sequence and sometimes they formed patches in euchromatic regions (e.g. on Gm11 76.5–78.6 cM; <a reveal-id="dsx043-F3" data-open="dsx043-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Interestingly, the distribution of low-recombination regions in all chromosomes coincided with the abundance of long terminal repeat retrotransposons reported in soybean.<a reveal-id="dsx043-B55" data-open="dsx043-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Gm07 and Gm16 had more than one apparent peak of suppressed recombination in addition to the pericentromeric regions.</p>                            <h3 scrollto-destination=115733313 id="115733313" class="section-title" >3.4. QTLs in F<sub>2</sub> population</h3>
<p>High-density genetic linkage maps have made it possible to genetically dissect flowering time differences between parents, and thus to better understand the genetic basis for soybean flowering by comparing previously reported flowering- and/or maturity-related QTLs. Both parents belong to soybean maturity group IV<a reveal-id="dsx043-B56" data-open="dsx043-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a>; DFF was 48.2 for Enrei and 60.7 for Peking. The F<sub>2</sub> population showed transgressive segregation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), and the broad-sense heritability of DFF was 75.5%. Seven QTLs (<em>qDFF</em>, <a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>) for DFF were identified. Both parents had alleles that accelerated flowering; Enrei had such alleles on Gm10, Gm12, Gm16 and Gm19 and Peking on Gm06, Gm11 and Gm13. Despite the high genetic complexity of the F<sub>2</sub> population, almost all phenotypic variation in DFF was explained (91.4%) by the four major QTLs (on Gm06, Gm10, Gm12 and Gm19) and three minor QTLs (on Gm11, Gm13, Gm16). Only <em>qDFF</em>_<em>Gm06</em> was located close to the low-recombination region. In F<sub>2</sub> population, 19 out of 21 QTLs (expect for <em>qDFF_Gm13</em> and <em>qPH</em>_<em>Gm10</em>) showed higher LOD score than 1% significant level threshold value. Surprisingly, most QTLs for other traits (PH, NP, SWH, DH and RP) were clustered with QTLs for DFF (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>), suggesting two possibilities that the latter have pleiotropic effects on other traits or genes for these traits are clustered together. <div content-id="dsx043-T2" class="table-modal"><div class="table-wrap"><div id="dsx043-T2" data-id="dsx043-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>QTLs detected in F<sub>2</sub> and BC<sub>3</sub>F<sub>2</sub> populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="10">QTLs detected in F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th colspan="10">QTLs detected in BC<sub>3</sub>F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th>Trait </th><th>QTL name </th><th>QTL position<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th> </th><th>QTL position<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th>Confirmation of QTL with CSSL<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th><th>Confirmed QTLs between F<sub>2</sub> and BC<sub>3</sub> F<sub>2</sub> population </th><th>Candidate genes or related QTLs </th><th>ID of related QTLs reported in Soybase </th><th>References </th></tr></thead><tbody><tr><td>DFF </td><td><em>qDFF_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>51.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>23.9 </td><td>4.9 </td><td>0.2 </td><td>1.6 </td><td>0.3 </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>174.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.5 </td><td>3.3 </td><td>0.1 </td><td>0.5 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>Liu et al. 2009, Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFF_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>57.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.6 </td><td>–5.5 </td><td>0.2 </td><td>2.6 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>145.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>22.7 </td><td>–3.2 </td><td>0.1 </td><td>1.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>Xia et al. 2012 </td></tr><tr><td> </td><td><em>qDFF_Gm12</em> </td><td><a href="mailto:Gm12@33.0" target="_blank">Gm12@33.0</a> </td><td>C12-BARC- 015603-02006 </td><td>s024200450 </td><td>s024200450 </td><td>36.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>13.6 </td><td>3.7 </td><td>0.2 </td><td>0.3 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.8 </td><td>2.0 </td><td>0.1 </td><td>–0.1 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td>First flower 25–2 </td><td>Kuroda et al. 2013 </td></tr><tr><td> </td><td><em>qDFF_Gm10</em> </td><td><a href="mailto:Gm10@116.0" target="_blank">Gm10@116.0</a> </td><td>C10-BARC- 015003-01948 </td><td>C10-BARC- 015925-02017 </td><td>C10-BARC- 015925-02017 </td><td>37.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.3 </td><td>4.0 </td><td>0.2 </td><td>1.2 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>70.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.2 </td><td>1.9 </td><td>0.1 </td><td>0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>Watanabe et al. 2011 </td></tr><tr><td> </td><td><em>qDFF_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>18.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.6 </td><td>–2.4 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>38.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.7 </td><td>–1.2 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>First flower 8–4 </td><td>Yamanaka et al. 2001 </td></tr><tr><td> </td><td><em>qDFF_Gm16</em> </td><td><a href="mailto:Gm16@33.0" target="_blank">Gm16@33.0</a> </td><td>C16-BARC- 020505-04644 </td><td>GMES0775 </td><td>GMES0775 </td><td>10.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>1.7 </td><td>0.2 </td><td>0.8 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>16.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–0.9 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>GmFT5a</em> </td><td> </td><td>Takeshima et al. 2016 </td></tr><tr><td> </td><td><em>qDFF_Gm13</em> </td><td><a href="mailto:Gm13@7.0" target="_blank">Gm13@7.0</a> </td><td>s000702358-2 </td><td>s000701869-2 </td><td>s000701869-2 </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.7 </td><td>–1.5 </td><td>0.3 </td><td>–0.1 </td><td>0.4 </td><td> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>Confirmed </td><td><em>-</em> </td><td><em>qDFF_Gm13</em> </td><td>First flower 11–4 </td><td>Gai et al. 2007 </td></tr><tr><td>DFT </td><td><em>qDFT_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>285.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>55.3 </td><td>8.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>Liu et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>193.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.9 </td><td>4.8 </td><td>0.2 </td><td>2.8 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E3</em> </td><td> </td><td>Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.6" target="_blank">Gm06@105.6</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>78.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>11.5 </td><td>–3.5 </td><td>0.2 </td><td>1.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.7 </td><td>3.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>68.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.7 </td><td>3.0 </td><td>0.2 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@60.0" target="_blank">Gm11@60.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>35.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–2.1 </td><td>0.2 </td><td>0.4 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td>FP </td><td><em>qFP_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@100.0" target="_blank">Gm19@100.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>242.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>67.4 </td><td>5.4 </td><td>0.1 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qFP_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>131.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.8 </td><td>2.7 </td><td>0.2 </td><td>2.6 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td>DH </td><td><em>qDH_Gm19</em> </td><td><a href="mailto:Gm19@118.0" target="_blank">Gm19@118.0</a> </td><td>Satt229 </td><td>FT3-751 </td><td>FT3-751 </td><td>22.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.2 </td><td>8.1 </td><td>0.7 </td><td>2.5 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>39.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>12.1 </td><td>6.1 </td><td>0.5 </td><td>–0.6 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>34.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.2 </td><td>–11.2 </td><td>0.7 </td><td>2.1 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>30.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.1 </td><td>–5.4 </td><td>0.5 </td><td>0.8 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>29.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>21.1 </td><td>–10.0 </td><td>0.7 </td><td>0.0 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm11@41.0" target="_blank">Gm11@41.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>Satt197 </td><td>18.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.5 </td><td>–4.3 </td><td>0.5 </td><td>0.5 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qDH_Gm10</em> </td><td><a href="mailto:Gm10@119.0" target="_blank">Gm10@119.0</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>s005205820-2 </td><td>17.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.5 </td><td>7.3 </td><td>0.8 </td><td>1.9 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>13.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.8 </td><td>3.4 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm12</em> </td><td><a href="mailto:Gm12@33.5" target="_blank">Gm12@33.5</a> </td><td>s024200450 </td><td>s024200389 </td><td>s024200450 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.6 </td><td>6.4 </td><td>0.7 </td><td>1.3 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>11.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.4 </td><td>3.0 </td><td>0.5 </td><td>0.0 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@48.0" target="_blank">Gm02@48.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>8.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>2.3 </td><td>0.5 </td><td>1.6 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td>Pod maturity 22–4 </td><td>Reinprecht et al. 2006 </td></tr><tr><td> </td><td><em>qDH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@125.0" target="_blank">Gm13@125.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>5.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–2.3 </td><td>0.5 </td><td>2.7 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qDH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>5.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–2.5 </td><td>0.6 </td><td>1.3 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td>Reproductive period 1–7 </td><td>Wang et al. 2015 (position is slightly different) </td></tr><tr><td> </td><td><em>qDH_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@43.8" target="_blank">Gm01@43.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.3 </td><td>2.2 </td><td>0.5 </td><td>–2.2 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td>Pod maturity 1–2 </td><td>Keim et al. 1990B </td></tr><tr><td>RP </td><td><em>qRP_Gm11</em> </td><td><a href="mailto:Gm11@70.0" target="_blank">Gm11@70.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>s008000014-2 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.4 </td><td>–7.0 </td><td>0.7 </td><td>0.1 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm11@38.0" target="_blank">Gm11@38.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>GMES0045 </td><td>12.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–3.4 </td><td>0.5 </td><td>0.2 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qRP_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@49.0" target="_blank">Gm02@49.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>9.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.5 </td><td>2.6 </td><td>0.6 </td><td>1.2 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@124.0" target="_blank">Gm13@124.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>7.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.8 </td><td>–2.6 </td><td>0.5 </td><td>2.8 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qRP_Gm19</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>7.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>2.7 </td><td>0.5 </td><td>–1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>10.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>16.9 </td><td>–5.4 </td><td>0.7 </td><td>–0.9 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>–2.2 </td><td>0.5 </td><td>–0.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@54.5" target="_blank">Gm16@54.5</a> </td><td>Satt596 </td><td>s014300091-2 </td><td>Satt596 </td><td>5.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.1 </td><td>2.3 </td><td>0.5 </td><td>–0.5 </td><td>1.0 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>4.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>–2.3 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@44.8" target="_blank">Gm01@44.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>2.1 </td><td>0.5 </td><td>–2.0 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>PH </td><td><em>qPH_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><a href="mailto:Gm19@114.0" target="_blank">Gm19@114.0</a> </td><td>Satt664 </td><td>C19-BARC- 030101-06809 </td><td>Satt664 </td><td>30.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>42.7 </td><td>15.5 </td><td>1.1 </td><td>0.5 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>129.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.7 </td><td>14.4 </td><td>0.5 </td><td>–3.5 </td><td>1.0 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>78.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.6 </td><td>7.0 </td><td>0.6 </td><td>7.2 </td><td>1.1 </td><td>Confirmed </td><td>–– </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm10</em> </td><td><a href="mailto:Gm10@117.2" target="_blank">Gm10@117.2</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>FT2-300H01d </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>6.4 </td><td>5.8 </td><td>1.1 </td><td>4.1 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm10@102.0" target="_blank">Gm10@102.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>17.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.3 </td><td>4.0 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>16.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>–10.5 </td><td>1.1 </td><td>3.8 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>15.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.8 </td><td>–3.8 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td>NP </td><td><em>qNP_Gm19</em> </td><td><a href="mailto:Gm19@116.0" target="_blank">Gm19@116.0</a> </td><td>C19-BARC- 030101-06809 </td><td>Satt229 </td><td>C19-BARC- 030101-06809 </td><td>15.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.4 </td><td>84.4 </td><td>9.7 </td><td>17.0 </td><td>## </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>148.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>39.3 </td><td>80.0 </td><td>3.5 </td><td>30.0 </td><td>6.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm12</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>33.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.8 </td><td>31.6 </td><td>3.0 </td><td>6.7 </td><td>6.0 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm10</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm10@109.0" target="_blank">Gm10@109.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>27.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.4 </td><td>28.3 </td><td>3.1 </td><td>5.3 </td><td>5.6 </td><td>–– </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>11.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>−73.0 </td><td>9.5 </td><td>9.2 </td><td>## </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>23.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–29.7 </td><td>3.2 </td><td>5.1 </td><td>5.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>15.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.9 </td><td>–21.2 </td><td>2.8 </td><td>1.7 </td><td>5.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@69.5" target="_blank">Gm16@69.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>s026800043 </td><td>7.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–18.5 </td><td>3.2 </td><td>16.3 </td><td>6.1 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td>TSW </td><td><em>qTSW_Gm19</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>155.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.3 </td><td>37.9 </td><td>1.7 </td><td>18.3 </td><td>3.2 </td><td>–– </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>44.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.0 </td><td>18.1 </td><td>1.5 </td><td>1.7 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@61.0" target="_blank">Gm11@61.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>36.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.5 </td><td>–16.3 </td><td>1.4 </td><td>1.4 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.4" target="_blank">Gm06@105.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>35.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.4 </td><td>–17.4 </td><td>1.5 </td><td>3.2 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@110.0" target="_blank">Gm10@110.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>20.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.6 </td><td>10.7 </td><td>1.5 </td><td>5.2 </td><td>2.8 </td><td>Confirmed </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm16</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>8.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–9.7 </td><td>1.5 </td><td>9.4 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm20</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm20@99.0" target="_blank">Gm20@99.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.1 </td><td>–9.4 </td><td>2.0 </td><td>1.5 </td><td>3.9 </td><td>Confirmed </td><td>–– </td><td><em>qTSW_Gm20</em> </td><td>Seed yield 15–15 </td><td>Kabelka et al. 2004 </td></tr><tr><td>SWH </td><td><em>qSWH_Gm08</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm08@46.0" target="_blank">Gm08@46.0</a> </td><td>GMES1620 </td><td>AY262686B </td><td>AY262686B </td><td>33.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.1 </td><td>–2.8 </td><td>0.2 </td><td>1.9 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm08</em> </td><td>Seed weight 4–5 </td><td>Maughan et al. 1996 (close to I locus) </td></tr><tr><td> </td><td><em>qSWH_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>8.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>17.1 </td><td>–2.4 </td><td>0.4 </td><td>1.7 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>29.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.8 </td><td>–2.3 </td><td>0.2 </td><td>1.1 </td><td>0.4 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@59.0" target="_blank">Gm11@59.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>22.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.6 </td><td>–1.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@51.0" target="_blank">Gm17@51.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001710954m </td><td>15.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.4 </td><td>–2.0 </td><td>0.3 </td><td>0.5 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm17</em> </td><td>Seed yield 5-2 </td><td>Renyna and Sneller 2001 </td></tr><tr><td> </td><td><em>qSWH_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>8.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>15.9 </td><td>2.4 </td><td>0.4 </td><td>1.1 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm19@107.1" target="_blank">Gm19@107.1</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>10.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.1 </td><td>1.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm20</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm20@98.0" target="_blank">Gm20@98.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.5 </td><td>0.3 </td><td>1.1 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qSWH_Gm20</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@16.0" target="_blank">Gm13@16.0</a> </td><td>Satt146 </td><td>Satt252 </td><td>Satt252 </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.1 </td><td>0.2 </td><td>0.0 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qDFF_Gm13</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@22.0" target="_blank">Gm02@22.0</a> </td><td>Sat_096 </td><td>Sat_351 </td><td>Sat_351 </td><td>5.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.7 </td><td>–0.9 </td><td>0.2 </td><td>–0.4 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm02</em> </td><td> </td><td>“ </td></tr></tbody></table></div><div content-id="dsx043-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="10">QTLs detected in F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th colspan="10">QTLs detected in BC<sub>3</sub>F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th>Trait </th><th>QTL name </th><th>QTL position<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th> </th><th>QTL position<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th>Confirmation of QTL with CSSL<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th><th>Confirmed QTLs between F<sub>2</sub> and BC<sub>3</sub> F<sub>2</sub> population </th><th>Candidate genes or related QTLs </th><th>ID of related QTLs reported in Soybase </th><th>References </th></tr></thead><tbody><tr><td>DFF </td><td><em>qDFF_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>51.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>23.9 </td><td>4.9 </td><td>0.2 </td><td>1.6 </td><td>0.3 </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>174.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.5 </td><td>3.3 </td><td>0.1 </td><td>0.5 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>Liu et al. 2009, Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFF_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>57.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.6 </td><td>–5.5 </td><td>0.2 </td><td>2.6 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>145.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>22.7 </td><td>–3.2 </td><td>0.1 </td><td>1.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>Xia et al. 2012 </td></tr><tr><td> </td><td><em>qDFF_Gm12</em> </td><td><a href="mailto:Gm12@33.0" target="_blank">Gm12@33.0</a> </td><td>C12-BARC- 015603-02006 </td><td>s024200450 </td><td>s024200450 </td><td>36.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>13.6 </td><td>3.7 </td><td>0.2 </td><td>0.3 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.8 </td><td>2.0 </td><td>0.1 </td><td>–0.1 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td>First flower 25–2 </td><td>Kuroda et al. 2013 </td></tr><tr><td> </td><td><em>qDFF_Gm10</em> </td><td><a href="mailto:Gm10@116.0" target="_blank">Gm10@116.0</a> </td><td>C10-BARC- 015003-01948 </td><td>C10-BARC- 015925-02017 </td><td>C10-BARC- 015925-02017 </td><td>37.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.3 </td><td>4.0 </td><td>0.2 </td><td>1.2 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>70.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.2 </td><td>1.9 </td><td>0.1 </td><td>0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>Watanabe et al. 2011 </td></tr><tr><td> </td><td><em>qDFF_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>18.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.6 </td><td>–2.4 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>38.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.7 </td><td>–1.2 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>First flower 8–4 </td><td>Yamanaka et al. 2001 </td></tr><tr><td> </td><td><em>qDFF_Gm16</em> </td><td><a href="mailto:Gm16@33.0" target="_blank">Gm16@33.0</a> </td><td>C16-BARC- 020505-04644 </td><td>GMES0775 </td><td>GMES0775 </td><td>10.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>1.7 </td><td>0.2 </td><td>0.8 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>16.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–0.9 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>GmFT5a</em> </td><td> </td><td>Takeshima et al. 2016 </td></tr><tr><td> </td><td><em>qDFF_Gm13</em> </td><td><a href="mailto:Gm13@7.0" target="_blank">Gm13@7.0</a> </td><td>s000702358-2 </td><td>s000701869-2 </td><td>s000701869-2 </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.7 </td><td>–1.5 </td><td>0.3 </td><td>–0.1 </td><td>0.4 </td><td> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>Confirmed </td><td><em>-</em> </td><td><em>qDFF_Gm13</em> </td><td>First flower 11–4 </td><td>Gai et al. 2007 </td></tr><tr><td>DFT </td><td><em>qDFT_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>285.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>55.3 </td><td>8.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>Liu et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>193.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.9 </td><td>4.8 </td><td>0.2 </td><td>2.8 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E3</em> </td><td> </td><td>Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.6" target="_blank">Gm06@105.6</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>78.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>11.5 </td><td>–3.5 </td><td>0.2 </td><td>1.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.7 </td><td>3.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>68.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.7 </td><td>3.0 </td><td>0.2 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@60.0" target="_blank">Gm11@60.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>35.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–2.1 </td><td>0.2 </td><td>0.4 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td>FP </td><td><em>qFP_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@100.0" target="_blank">Gm19@100.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>242.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>67.4 </td><td>5.4 </td><td>0.1 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qFP_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>131.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.8 </td><td>2.7 </td><td>0.2 </td><td>2.6 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td>DH </td><td><em>qDH_Gm19</em> </td><td><a href="mailto:Gm19@118.0" target="_blank">Gm19@118.0</a> </td><td>Satt229 </td><td>FT3-751 </td><td>FT3-751 </td><td>22.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.2 </td><td>8.1 </td><td>0.7 </td><td>2.5 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>39.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>12.1 </td><td>6.1 </td><td>0.5 </td><td>–0.6 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>34.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.2 </td><td>–11.2 </td><td>0.7 </td><td>2.1 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>30.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.1 </td><td>–5.4 </td><td>0.5 </td><td>0.8 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>29.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>21.1 </td><td>–10.0 </td><td>0.7 </td><td>0.0 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm11@41.0" target="_blank">Gm11@41.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>Satt197 </td><td>18.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.5 </td><td>–4.3 </td><td>0.5 </td><td>0.5 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qDH_Gm10</em> </td><td><a href="mailto:Gm10@119.0" target="_blank">Gm10@119.0</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>s005205820-2 </td><td>17.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.5 </td><td>7.3 </td><td>0.8 </td><td>1.9 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>13.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.8 </td><td>3.4 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm12</em> </td><td><a href="mailto:Gm12@33.5" target="_blank">Gm12@33.5</a> </td><td>s024200450 </td><td>s024200389 </td><td>s024200450 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.6 </td><td>6.4 </td><td>0.7 </td><td>1.3 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>11.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.4 </td><td>3.0 </td><td>0.5 </td><td>0.0 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@48.0" target="_blank">Gm02@48.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>8.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>2.3 </td><td>0.5 </td><td>1.6 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td>Pod maturity 22–4 </td><td>Reinprecht et al. 2006 </td></tr><tr><td> </td><td><em>qDH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@125.0" target="_blank">Gm13@125.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>5.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–2.3 </td><td>0.5 </td><td>2.7 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qDH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>5.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–2.5 </td><td>0.6 </td><td>1.3 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td>Reproductive period 1–7 </td><td>Wang et al. 2015 (position is slightly different) </td></tr><tr><td> </td><td><em>qDH_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@43.8" target="_blank">Gm01@43.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.3 </td><td>2.2 </td><td>0.5 </td><td>–2.2 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td>Pod maturity 1–2 </td><td>Keim et al. 1990B </td></tr><tr><td>RP </td><td><em>qRP_Gm11</em> </td><td><a href="mailto:Gm11@70.0" target="_blank">Gm11@70.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>s008000014-2 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.4 </td><td>–7.0 </td><td>0.7 </td><td>0.1 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm11@38.0" target="_blank">Gm11@38.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>GMES0045 </td><td>12.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–3.4 </td><td>0.5 </td><td>0.2 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qRP_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@49.0" target="_blank">Gm02@49.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>9.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.5 </td><td>2.6 </td><td>0.6 </td><td>1.2 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@124.0" target="_blank">Gm13@124.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>7.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.8 </td><td>–2.6 </td><td>0.5 </td><td>2.8 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qRP_Gm19</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>7.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>2.7 </td><td>0.5 </td><td>–1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>10.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>16.9 </td><td>–5.4 </td><td>0.7 </td><td>–0.9 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>–2.2 </td><td>0.5 </td><td>–0.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@54.5" target="_blank">Gm16@54.5</a> </td><td>Satt596 </td><td>s014300091-2 </td><td>Satt596 </td><td>5.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.1 </td><td>2.3 </td><td>0.5 </td><td>–0.5 </td><td>1.0 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>4.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>–2.3 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@44.8" target="_blank">Gm01@44.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>2.1 </td><td>0.5 </td><td>–2.0 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>PH </td><td><em>qPH_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><a href="mailto:Gm19@114.0" target="_blank">Gm19@114.0</a> </td><td>Satt664 </td><td>C19-BARC- 030101-06809 </td><td>Satt664 </td><td>30.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>42.7 </td><td>15.5 </td><td>1.1 </td><td>0.5 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>129.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.7 </td><td>14.4 </td><td>0.5 </td><td>–3.5 </td><td>1.0 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>78.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.6 </td><td>7.0 </td><td>0.6 </td><td>7.2 </td><td>1.1 </td><td>Confirmed </td><td>–– </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm10</em> </td><td><a href="mailto:Gm10@117.2" target="_blank">Gm10@117.2</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>FT2-300H01d </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>6.4 </td><td>5.8 </td><td>1.1 </td><td>4.1 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm10@102.0" target="_blank">Gm10@102.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>17.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.3 </td><td>4.0 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>16.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>–10.5 </td><td>1.1 </td><td>3.8 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>15.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.8 </td><td>–3.8 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td>NP </td><td><em>qNP_Gm19</em> </td><td><a href="mailto:Gm19@116.0" target="_blank">Gm19@116.0</a> </td><td>C19-BARC- 030101-06809 </td><td>Satt229 </td><td>C19-BARC- 030101-06809 </td><td>15.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.4 </td><td>84.4 </td><td>9.7 </td><td>17.0 </td><td>## </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>148.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>39.3 </td><td>80.0 </td><td>3.5 </td><td>30.0 </td><td>6.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm12</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>33.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.8 </td><td>31.6 </td><td>3.0 </td><td>6.7 </td><td>6.0 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm10</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm10@109.0" target="_blank">Gm10@109.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>27.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.4 </td><td>28.3 </td><td>3.1 </td><td>5.3 </td><td>5.6 </td><td>–– </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>11.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>−73.0 </td><td>9.5 </td><td>9.2 </td><td>## </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>23.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–29.7 </td><td>3.2 </td><td>5.1 </td><td>5.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>15.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.9 </td><td>–21.2 </td><td>2.8 </td><td>1.7 </td><td>5.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@69.5" target="_blank">Gm16@69.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>s026800043 </td><td>7.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–18.5 </td><td>3.2 </td><td>16.3 </td><td>6.1 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td>TSW </td><td><em>qTSW_Gm19</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>155.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.3 </td><td>37.9 </td><td>1.7 </td><td>18.3 </td><td>3.2 </td><td>–– </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>44.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.0 </td><td>18.1 </td><td>1.5 </td><td>1.7 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@61.0" target="_blank">Gm11@61.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>36.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.5 </td><td>–16.3 </td><td>1.4 </td><td>1.4 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.4" target="_blank">Gm06@105.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>35.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.4 </td><td>–17.4 </td><td>1.5 </td><td>3.2 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@110.0" target="_blank">Gm10@110.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>20.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.6 </td><td>10.7 </td><td>1.5 </td><td>5.2 </td><td>2.8 </td><td>Confirmed </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm16</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>8.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–9.7 </td><td>1.5 </td><td>9.4 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm20</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm20@99.0" target="_blank">Gm20@99.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.1 </td><td>–9.4 </td><td>2.0 </td><td>1.5 </td><td>3.9 </td><td>Confirmed </td><td>–– </td><td><em>qTSW_Gm20</em> </td><td>Seed yield 15–15 </td><td>Kabelka et al. 2004 </td></tr><tr><td>SWH </td><td><em>qSWH_Gm08</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm08@46.0" target="_blank">Gm08@46.0</a> </td><td>GMES1620 </td><td>AY262686B </td><td>AY262686B </td><td>33.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.1 </td><td>–2.8 </td><td>0.2 </td><td>1.9 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm08</em> </td><td>Seed weight 4–5 </td><td>Maughan et al. 1996 (close to I locus) </td></tr><tr><td> </td><td><em>qSWH_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>8.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>17.1 </td><td>–2.4 </td><td>0.4 </td><td>1.7 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>29.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.8 </td><td>–2.3 </td><td>0.2 </td><td>1.1 </td><td>0.4 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@59.0" target="_blank">Gm11@59.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>22.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.6 </td><td>–1.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@51.0" target="_blank">Gm17@51.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001710954m </td><td>15.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.4 </td><td>–2.0 </td><td>0.3 </td><td>0.5 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm17</em> </td><td>Seed yield 5-2 </td><td>Renyna and Sneller 2001 </td></tr><tr><td> </td><td><em>qSWH_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>8.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>15.9 </td><td>2.4 </td><td>0.4 </td><td>1.1 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm19@107.1" target="_blank">Gm19@107.1</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>10.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.1 </td><td>1.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm20</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm20@98.0" target="_blank">Gm20@98.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.5 </td><td>0.3 </td><td>1.1 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qSWH_Gm20</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@16.0" target="_blank">Gm13@16.0</a> </td><td>Satt146 </td><td>Satt252 </td><td>Satt252 </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.1 </td><td>0.2 </td><td>0.0 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qDFF_Gm13</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@22.0" target="_blank">Gm02@22.0</a> </td><td>Sat_096 </td><td>Sat_351 </td><td>Sat_351 </td><td>5.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.7 </td><td>–0.9 </td><td>0.2 </td><td>–0.4 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm02</em> </td><td> </td><td>“ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>NA, the trait was not evaluated in the F<sub>2</sub> population; LOD, logarithm of odds of the presence of a QTL; PVE, percentage of variance explained by the QTL.</p></div></div><div content-id="tblfn3"><div class="fn"><div class="label -label">a</div><p>Sowing date of F<sub>2</sub> population and BC<sub>3</sub>F<sub>3</sub> population were 30 May 2007 and 24 June2011, respectively.</p></div></div><div content-id="tblfn4"><div class="fn"><div class="label -label">b</div><p>Indicated as chromosome name and position in cM separated by @.</p></div></div><div content-id="tblfn5"><div class="fn"><div class="label -label">c</div><p>‘Confirmed partially’ indicates that a singificant difference was observed between Enrei and a CSSL habouring several donor choromosomal fragments.</p></div></div><div content-id="tblfn6"><div class="fn"><div class="label -label">d</div><p>Effects of the QTLs corresponding to <em>E3</em> and <em>Dt1</em> loci were calculated separately.</p></div></div><div content-id="tblfn7"><div class="fn"><div class="label -label">*</div><p>Significant at <em>P &lt; </em>0.05 after 1,000 permutation; **Significant at <em>P &lt; </em>0.01 after 1,000 permutation.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx043-T2" data-id="dsx043-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>QTLs detected in F<sub>2</sub> and BC<sub>3</sub>F<sub>2</sub> populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="10">QTLs detected in F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th colspan="10">QTLs detected in BC<sub>3</sub>F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th>Trait </th><th>QTL name </th><th>QTL position<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th> </th><th>QTL position<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th>Confirmation of QTL with CSSL<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th><th>Confirmed QTLs between F<sub>2</sub> and BC<sub>3</sub> F<sub>2</sub> population </th><th>Candidate genes or related QTLs </th><th>ID of related QTLs reported in Soybase </th><th>References </th></tr></thead><tbody><tr><td>DFF </td><td><em>qDFF_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>51.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>23.9 </td><td>4.9 </td><td>0.2 </td><td>1.6 </td><td>0.3 </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>174.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.5 </td><td>3.3 </td><td>0.1 </td><td>0.5 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>Liu et al. 2009, Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFF_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>57.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.6 </td><td>–5.5 </td><td>0.2 </td><td>2.6 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>145.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>22.7 </td><td>–3.2 </td><td>0.1 </td><td>1.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>Xia et al. 2012 </td></tr><tr><td> </td><td><em>qDFF_Gm12</em> </td><td><a href="mailto:Gm12@33.0" target="_blank">Gm12@33.0</a> </td><td>C12-BARC- 015603-02006 </td><td>s024200450 </td><td>s024200450 </td><td>36.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>13.6 </td><td>3.7 </td><td>0.2 </td><td>0.3 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.8 </td><td>2.0 </td><td>0.1 </td><td>–0.1 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td>First flower 25–2 </td><td>Kuroda et al. 2013 </td></tr><tr><td> </td><td><em>qDFF_Gm10</em> </td><td><a href="mailto:Gm10@116.0" target="_blank">Gm10@116.0</a> </td><td>C10-BARC- 015003-01948 </td><td>C10-BARC- 015925-02017 </td><td>C10-BARC- 015925-02017 </td><td>37.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.3 </td><td>4.0 </td><td>0.2 </td><td>1.2 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>70.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.2 </td><td>1.9 </td><td>0.1 </td><td>0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>Watanabe et al. 2011 </td></tr><tr><td> </td><td><em>qDFF_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>18.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.6 </td><td>–2.4 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>38.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.7 </td><td>–1.2 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>First flower 8–4 </td><td>Yamanaka et al. 2001 </td></tr><tr><td> </td><td><em>qDFF_Gm16</em> </td><td><a href="mailto:Gm16@33.0" target="_blank">Gm16@33.0</a> </td><td>C16-BARC- 020505-04644 </td><td>GMES0775 </td><td>GMES0775 </td><td>10.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>1.7 </td><td>0.2 </td><td>0.8 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>16.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–0.9 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>GmFT5a</em> </td><td> </td><td>Takeshima et al. 2016 </td></tr><tr><td> </td><td><em>qDFF_Gm13</em> </td><td><a href="mailto:Gm13@7.0" target="_blank">Gm13@7.0</a> </td><td>s000702358-2 </td><td>s000701869-2 </td><td>s000701869-2 </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.7 </td><td>–1.5 </td><td>0.3 </td><td>–0.1 </td><td>0.4 </td><td> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>Confirmed </td><td><em>-</em> </td><td><em>qDFF_Gm13</em> </td><td>First flower 11–4 </td><td>Gai et al. 2007 </td></tr><tr><td>DFT </td><td><em>qDFT_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>285.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>55.3 </td><td>8.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>Liu et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>193.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.9 </td><td>4.8 </td><td>0.2 </td><td>2.8 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E3</em> </td><td> </td><td>Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.6" target="_blank">Gm06@105.6</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>78.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>11.5 </td><td>–3.5 </td><td>0.2 </td><td>1.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.7 </td><td>3.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>68.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.7 </td><td>3.0 </td><td>0.2 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@60.0" target="_blank">Gm11@60.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>35.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–2.1 </td><td>0.2 </td><td>0.4 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td>FP </td><td><em>qFP_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@100.0" target="_blank">Gm19@100.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>242.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>67.4 </td><td>5.4 </td><td>0.1 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qFP_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>131.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.8 </td><td>2.7 </td><td>0.2 </td><td>2.6 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td>DH </td><td><em>qDH_Gm19</em> </td><td><a href="mailto:Gm19@118.0" target="_blank">Gm19@118.0</a> </td><td>Satt229 </td><td>FT3-751 </td><td>FT3-751 </td><td>22.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.2 </td><td>8.1 </td><td>0.7 </td><td>2.5 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>39.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>12.1 </td><td>6.1 </td><td>0.5 </td><td>–0.6 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>34.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.2 </td><td>–11.2 </td><td>0.7 </td><td>2.1 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>30.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.1 </td><td>–5.4 </td><td>0.5 </td><td>0.8 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>29.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>21.1 </td><td>–10.0 </td><td>0.7 </td><td>0.0 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm11@41.0" target="_blank">Gm11@41.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>Satt197 </td><td>18.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.5 </td><td>–4.3 </td><td>0.5 </td><td>0.5 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qDH_Gm10</em> </td><td><a href="mailto:Gm10@119.0" target="_blank">Gm10@119.0</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>s005205820-2 </td><td>17.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.5 </td><td>7.3 </td><td>0.8 </td><td>1.9 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>13.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.8 </td><td>3.4 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm12</em> </td><td><a href="mailto:Gm12@33.5" target="_blank">Gm12@33.5</a> </td><td>s024200450 </td><td>s024200389 </td><td>s024200450 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.6 </td><td>6.4 </td><td>0.7 </td><td>1.3 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>11.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.4 </td><td>3.0 </td><td>0.5 </td><td>0.0 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@48.0" target="_blank">Gm02@48.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>8.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>2.3 </td><td>0.5 </td><td>1.6 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td>Pod maturity 22–4 </td><td>Reinprecht et al. 2006 </td></tr><tr><td> </td><td><em>qDH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@125.0" target="_blank">Gm13@125.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>5.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–2.3 </td><td>0.5 </td><td>2.7 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qDH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>5.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–2.5 </td><td>0.6 </td><td>1.3 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td>Reproductive period 1–7 </td><td>Wang et al. 2015 (position is slightly different) </td></tr><tr><td> </td><td><em>qDH_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@43.8" target="_blank">Gm01@43.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.3 </td><td>2.2 </td><td>0.5 </td><td>–2.2 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td>Pod maturity 1–2 </td><td>Keim et al. 1990B </td></tr><tr><td>RP </td><td><em>qRP_Gm11</em> </td><td><a href="mailto:Gm11@70.0" target="_blank">Gm11@70.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>s008000014-2 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.4 </td><td>–7.0 </td><td>0.7 </td><td>0.1 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm11@38.0" target="_blank">Gm11@38.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>GMES0045 </td><td>12.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–3.4 </td><td>0.5 </td><td>0.2 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qRP_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@49.0" target="_blank">Gm02@49.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>9.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.5 </td><td>2.6 </td><td>0.6 </td><td>1.2 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@124.0" target="_blank">Gm13@124.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>7.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.8 </td><td>–2.6 </td><td>0.5 </td><td>2.8 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qRP_Gm19</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>7.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>2.7 </td><td>0.5 </td><td>–1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>10.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>16.9 </td><td>–5.4 </td><td>0.7 </td><td>–0.9 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>–2.2 </td><td>0.5 </td><td>–0.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@54.5" target="_blank">Gm16@54.5</a> </td><td>Satt596 </td><td>s014300091-2 </td><td>Satt596 </td><td>5.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.1 </td><td>2.3 </td><td>0.5 </td><td>–0.5 </td><td>1.0 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>4.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>–2.3 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@44.8" target="_blank">Gm01@44.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>2.1 </td><td>0.5 </td><td>–2.0 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>PH </td><td><em>qPH_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><a href="mailto:Gm19@114.0" target="_blank">Gm19@114.0</a> </td><td>Satt664 </td><td>C19-BARC- 030101-06809 </td><td>Satt664 </td><td>30.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>42.7 </td><td>15.5 </td><td>1.1 </td><td>0.5 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>129.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.7 </td><td>14.4 </td><td>0.5 </td><td>–3.5 </td><td>1.0 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>78.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.6 </td><td>7.0 </td><td>0.6 </td><td>7.2 </td><td>1.1 </td><td>Confirmed </td><td>–– </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm10</em> </td><td><a href="mailto:Gm10@117.2" target="_blank">Gm10@117.2</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>FT2-300H01d </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>6.4 </td><td>5.8 </td><td>1.1 </td><td>4.1 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm10@102.0" target="_blank">Gm10@102.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>17.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.3 </td><td>4.0 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>16.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>–10.5 </td><td>1.1 </td><td>3.8 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>15.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.8 </td><td>–3.8 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td>NP </td><td><em>qNP_Gm19</em> </td><td><a href="mailto:Gm19@116.0" target="_blank">Gm19@116.0</a> </td><td>C19-BARC- 030101-06809 </td><td>Satt229 </td><td>C19-BARC- 030101-06809 </td><td>15.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.4 </td><td>84.4 </td><td>9.7 </td><td>17.0 </td><td>## </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>148.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>39.3 </td><td>80.0 </td><td>3.5 </td><td>30.0 </td><td>6.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm12</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>33.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.8 </td><td>31.6 </td><td>3.0 </td><td>6.7 </td><td>6.0 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm10</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm10@109.0" target="_blank">Gm10@109.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>27.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.4 </td><td>28.3 </td><td>3.1 </td><td>5.3 </td><td>5.6 </td><td>–– </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>11.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>−73.0 </td><td>9.5 </td><td>9.2 </td><td>## </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>23.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–29.7 </td><td>3.2 </td><td>5.1 </td><td>5.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>15.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.9 </td><td>–21.2 </td><td>2.8 </td><td>1.7 </td><td>5.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@69.5" target="_blank">Gm16@69.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>s026800043 </td><td>7.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–18.5 </td><td>3.2 </td><td>16.3 </td><td>6.1 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td>TSW </td><td><em>qTSW_Gm19</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>155.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.3 </td><td>37.9 </td><td>1.7 </td><td>18.3 </td><td>3.2 </td><td>–– </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>44.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.0 </td><td>18.1 </td><td>1.5 </td><td>1.7 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@61.0" target="_blank">Gm11@61.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>36.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.5 </td><td>–16.3 </td><td>1.4 </td><td>1.4 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.4" target="_blank">Gm06@105.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>35.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.4 </td><td>–17.4 </td><td>1.5 </td><td>3.2 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@110.0" target="_blank">Gm10@110.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>20.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.6 </td><td>10.7 </td><td>1.5 </td><td>5.2 </td><td>2.8 </td><td>Confirmed </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm16</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>8.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–9.7 </td><td>1.5 </td><td>9.4 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm20</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm20@99.0" target="_blank">Gm20@99.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.1 </td><td>–9.4 </td><td>2.0 </td><td>1.5 </td><td>3.9 </td><td>Confirmed </td><td>–– </td><td><em>qTSW_Gm20</em> </td><td>Seed yield 15–15 </td><td>Kabelka et al. 2004 </td></tr><tr><td>SWH </td><td><em>qSWH_Gm08</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm08@46.0" target="_blank">Gm08@46.0</a> </td><td>GMES1620 </td><td>AY262686B </td><td>AY262686B </td><td>33.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.1 </td><td>–2.8 </td><td>0.2 </td><td>1.9 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm08</em> </td><td>Seed weight 4–5 </td><td>Maughan et al. 1996 (close to I locus) </td></tr><tr><td> </td><td><em>qSWH_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>8.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>17.1 </td><td>–2.4 </td><td>0.4 </td><td>1.7 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>29.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.8 </td><td>–2.3 </td><td>0.2 </td><td>1.1 </td><td>0.4 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@59.0" target="_blank">Gm11@59.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>22.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.6 </td><td>–1.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@51.0" target="_blank">Gm17@51.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001710954m </td><td>15.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.4 </td><td>–2.0 </td><td>0.3 </td><td>0.5 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm17</em> </td><td>Seed yield 5-2 </td><td>Renyna and Sneller 2001 </td></tr><tr><td> </td><td><em>qSWH_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>8.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>15.9 </td><td>2.4 </td><td>0.4 </td><td>1.1 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm19@107.1" target="_blank">Gm19@107.1</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>10.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.1 </td><td>1.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm20</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm20@98.0" target="_blank">Gm20@98.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.5 </td><td>0.3 </td><td>1.1 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qSWH_Gm20</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@16.0" target="_blank">Gm13@16.0</a> </td><td>Satt146 </td><td>Satt252 </td><td>Satt252 </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.1 </td><td>0.2 </td><td>0.0 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qDFF_Gm13</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@22.0" target="_blank">Gm02@22.0</a> </td><td>Sat_096 </td><td>Sat_351 </td><td>Sat_351 </td><td>5.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.7 </td><td>–0.9 </td><td>0.2 </td><td>–0.4 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm02</em> </td><td> </td><td>“ </td></tr></tbody></table></div><div content-id="dsx043-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="10">QTLs detected in F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th colspan="10">QTLs detected in BC<sub>3</sub>F<sub>2</sub> population<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th>Trait </th><th>QTL name </th><th>QTL position<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th> </th><th>QTL position<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Flanking marker 1 </th><th>Flanking marker 2 </th><th>Closest DNA marker </th><th>LOD </th><th>PVE (%) </th><th>Additive effect of Peking allele </th><th>s.d. </th><th>Dominance effect </th><th>s.d. </th><th>Confirmation of QTL with CSSL<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th><th>Confirmed QTLs between F<sub>2</sub> and BC<sub>3</sub> F<sub>2</sub> population </th><th>Candidate genes or related QTLs </th><th>ID of related QTLs reported in Soybase </th><th>References </th></tr></thead><tbody><tr><td>DFF </td><td><em>qDFF_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>51.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>23.9 </td><td>4.9 </td><td>0.2 </td><td>1.6 </td><td>0.3 </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>174.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.5 </td><td>3.3 </td><td>0.1 </td><td>0.5 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>Liu et al. 2009, Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFF_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>57.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>29.6 </td><td>–5.5 </td><td>0.2 </td><td>2.6 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>145.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>22.7 </td><td>–3.2 </td><td>0.1 </td><td>1.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>Xia et al. 2012 </td></tr><tr><td> </td><td><em>qDFF_Gm12</em> </td><td><a href="mailto:Gm12@33.0" target="_blank">Gm12@33.0</a> </td><td>C12-BARC- 015603-02006 </td><td>s024200450 </td><td>s024200450 </td><td>36.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>13.6 </td><td>3.7 </td><td>0.2 </td><td>0.3 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.8 </td><td>2.0 </td><td>0.1 </td><td>–0.1 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td>First flower 25–2 </td><td>Kuroda et al. 2013 </td></tr><tr><td> </td><td><em>qDFF_Gm10</em> </td><td><a href="mailto:Gm10@116.0" target="_blank">Gm10@116.0</a> </td><td>C10-BARC- 015003-01948 </td><td>C10-BARC- 015925-02017 </td><td>C10-BARC- 015925-02017 </td><td>37.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.3 </td><td>4.0 </td><td>0.2 </td><td>1.2 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>70.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.2 </td><td>1.9 </td><td>0.1 </td><td>0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>Watanabe et al. 2011 </td></tr><tr><td> </td><td><em>qDFF_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>18.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.6 </td><td>–2.4 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>38.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.7 </td><td>–1.2 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>First flower 8–4 </td><td>Yamanaka et al. 2001 </td></tr><tr><td> </td><td><em>qDFF_Gm16</em> </td><td><a href="mailto:Gm16@33.0" target="_blank">Gm16@33.0</a> </td><td>C16-BARC- 020505-04644 </td><td>GMES0775 </td><td>GMES0775 </td><td>10.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>1.7 </td><td>0.2 </td><td>0.8 </td><td>0.4 </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>16.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–0.9 </td><td>0.1 </td><td>–0.2 </td><td>0.2 </td><td>Confirmed </td><td>Confirmed </td><td><em>GmFT5a</em> </td><td> </td><td>Takeshima et al. 2016 </td></tr><tr><td> </td><td><em>qDFF_Gm13</em> </td><td><a href="mailto:Gm13@7.0" target="_blank">Gm13@7.0</a> </td><td>s000702358-2 </td><td>s000701869-2 </td><td>s000701869-2 </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.7 </td><td>–1.5 </td><td>0.3 </td><td>–0.1 </td><td>0.4 </td><td> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>Confirmed </td><td><em>-</em> </td><td><em>qDFF_Gm13</em> </td><td>First flower 11–4 </td><td>Gai et al. 2007 </td></tr><tr><td>DFT </td><td><em>qDFT_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>285.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>55.3 </td><td>8.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>Liu et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@106.0" target="_blank">Gm19@106.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>193.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.9 </td><td>4.8 </td><td>0.2 </td><td>2.8 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E3</em> </td><td> </td><td>Watanabe et al. 2009 </td></tr><tr><td> </td><td><em>qDFT_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.6" target="_blank">Gm06@105.6</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>78.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>11.5 </td><td>–3.5 </td><td>0.2 </td><td>1.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>74.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.7 </td><td>3.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>68.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.7 </td><td>3.0 </td><td>0.2 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDFT_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@60.0" target="_blank">Gm11@60.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>35.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–2.1 </td><td>0.2 </td><td>0.4 </td><td>0.3 </td><td>Confirmed </td><td>NA </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td>FP </td><td><em>qFP_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@100.0" target="_blank">Gm19@100.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>242.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>67.4 </td><td>5.4 </td><td>0.1 </td><td>–0.2 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qFP_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>131.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.8 </td><td>2.7 </td><td>0.2 </td><td>2.6 </td><td>0.3 </td><td>Confirmed(partially) </td><td>NA </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td>DH </td><td><em>qDH_Gm19</em> </td><td><a href="mailto:Gm19@118.0" target="_blank">Gm19@118.0</a> </td><td>Satt229 </td><td>FT3-751 </td><td>FT3-751 </td><td>22.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>14.2 </td><td>8.1 </td><td>0.7 </td><td>2.5 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>39.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>12.1 </td><td>6.1 </td><td>0.5 </td><td>–0.6 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm06</em> </td><td><a href="mailto:Gm06@102.0" target="_blank">Gm06@102.0</a> </td><td>sF3H </td><td>C06-BARC- 041345-07969 </td><td>C06-BARC- 041345-07969 </td><td>34.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.2 </td><td>–11.2 </td><td>0.7 </td><td>2.1 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>30.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.1 </td><td>–5.4 </td><td>0.5 </td><td>0.8 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm11</em> </td><td><a href="mailto:Gm11@71.0" target="_blank">Gm11@71.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>T001111280m </td><td>29.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>21.1 </td><td>–10.0 </td><td>0.7 </td><td>0.0 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm11@41.0" target="_blank">Gm11@41.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>Satt197 </td><td>18.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.5 </td><td>–4.3 </td><td>0.5 </td><td>0.5 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qDH_Gm10</em> </td><td><a href="mailto:Gm10@119.0" target="_blank">Gm10@119.0</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>s005205820-2 </td><td>17.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.5 </td><td>7.3 </td><td>0.8 </td><td>1.9 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm10@103.0" target="_blank">Gm10@103.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>13.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.8 </td><td>3.4 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm12</em> </td><td><a href="mailto:Gm12@33.5" target="_blank">Gm12@33.5</a> </td><td>s024200450 </td><td>s024200389 </td><td>s024200450 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>9.6 </td><td>6.4 </td><td>0.7 </td><td>1.3 </td><td>1.0 </td><td> </td><td><a href="mailto:Gm12@49.0" target="_blank">Gm12@49.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>11.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.4 </td><td>3.0 </td><td>0.5 </td><td>0.0 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qDH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@48.0" target="_blank">Gm02@48.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>8.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>2.3 </td><td>0.5 </td><td>1.6 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td>Pod maturity 22–4 </td><td>Reinprecht et al. 2006 </td></tr><tr><td> </td><td><em>qDH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@125.0" target="_blank">Gm13@125.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>5.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–2.3 </td><td>0.5 </td><td>2.7 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qDH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>5.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–2.5 </td><td>0.6 </td><td>1.3 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td>Reproductive period 1–7 </td><td>Wang et al. 2015 (position is slightly different) </td></tr><tr><td> </td><td><em>qDH_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@43.8" target="_blank">Gm01@43.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.3 </td><td>2.2 </td><td>0.5 </td><td>–2.2 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td>Pod maturity 1–2 </td><td>Keim et al. 1990B </td></tr><tr><td>RP </td><td><em>qRP_Gm11</em> </td><td><a href="mailto:Gm11@70.0" target="_blank">Gm11@70.0</a> </td><td>s008000014-2 </td><td>T001111280m </td><td>s008000014-2 </td><td>16.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.4 </td><td>–7.0 </td><td>0.7 </td><td>0.1 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm11@38.0" target="_blank">Gm11@38.0</a> </td><td>GMES0045 </td><td>Satt197 </td><td>GMES0045 </td><td>12.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–3.4 </td><td>0.5 </td><td>0.2 </td><td>1.1 </td><td>Confirmed </td><td>Confirmed </td><td><em>qDFF_Gm11</em> </td><td>Reproductive stage length 8–1 </td><td>Komatsu et al. 2012 </td></tr><tr><td> </td><td><em>qRP_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@49.0" target="_blank">Gm02@49.0</a> </td><td>Satt701 </td><td>Satt296 </td><td>Satt701 </td><td>9.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.5 </td><td>2.6 </td><td>0.6 </td><td>1.2 </td><td>1.1 </td><td>–– </td><td>–– </td><td><em>qRP_Gm02</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@124.0" target="_blank">Gm13@124.0</a> </td><td>Satt362 </td><td>s003004926-2 </td><td>Satt362 </td><td>7.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.8 </td><td>–2.6 </td><td>0.5 </td><td>2.8 </td><td>1.0 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm13</em> </td><td> </td><td>Novel </td></tr><tr><td> </td><td><em>qRP_Gm19</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@105.0" target="_blank">Gm19@105.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>7.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.7 </td><td>2.7 </td><td>0.5 </td><td>–1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>10.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>16.9 </td><td>–5.4 </td><td>0.7 </td><td>–0.9 </td><td>1.1 </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.3 </td><td>–2.2 </td><td>0.5 </td><td>–0.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@54.5" target="_blank">Gm16@54.5</a> </td><td>Satt596 </td><td>s014300091-2 </td><td>Satt596 </td><td>5.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.1 </td><td>2.3 </td><td>0.5 </td><td>–0.5 </td><td>1.0 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@46.0" target="_blank">Gm17@46.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001709449l </td><td>4.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>–2.3 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>–– </td><td><em>qRP_Gm17</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qRP_Gm01</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm01@44.8" target="_blank">Gm01@44.8</a> </td><td>Satt221 </td><td>Satt370 </td><td>Satt221 </td><td>4.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.6 </td><td>2.1 </td><td>0.5 </td><td>–2.0 </td><td>0.9 </td><td>–– </td><td>–– </td><td><em>qRP_Gm01</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>PH </td><td><em>qPH_Gm19_1</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td><a href="mailto:Gm19@114.0" target="_blank">Gm19@114.0</a> </td><td>Satt664 </td><td>C19-BARC- 030101-06809 </td><td>Satt664 </td><td>30.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>42.7 </td><td>15.5 </td><td>1.1 </td><td>0.5 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>129.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>45.7 </td><td>14.4 </td><td>0.5 </td><td>–3.5 </td><td>1.0 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm19_2</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>78.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>27.6 </td><td>7.0 </td><td>0.6 </td><td>7.2 </td><td>1.1 </td><td>Confirmed </td><td>–– </td><td><em>E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm10</em> </td><td><a href="mailto:Gm10@117.2" target="_blank">Gm10@117.2</a> </td><td>FT2-300H01d </td><td>s005205820-2 </td><td>FT2-300H01d </td><td>6.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>6.4 </td><td>5.8 </td><td>1.1 </td><td>4.1 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm10@102.0" target="_blank">Gm10@102.0</a> </td><td>Satt331 </td><td>FT2-300H01d </td><td>FT2-300H01d </td><td>17.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.3 </td><td>4.0 </td><td>0.5 </td><td>–0.4 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qPH_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>16.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>–10.5 </td><td>1.1 </td><td>3.8 </td><td>1.6 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>15.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.8 </td><td>–3.8 </td><td>0.5 </td><td>1.1 </td><td>0.9 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td>NP </td><td><em>qNP_Gm19</em> </td><td><a href="mailto:Gm19@116.0" target="_blank">Gm19@116.0</a> </td><td>C19-BARC- 030101-06809 </td><td>Satt229 </td><td>C19-BARC- 030101-06809 </td><td>15.0<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>26.4 </td><td>84.4 </td><td>9.7 </td><td>17.0 </td><td>## </td><td> </td><td><a href="mailto:Gm19@101.0" target="_blank">Gm19@101.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>148.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>39.3 </td><td>80.0 </td><td>3.5 </td><td>30.0 </td><td>6.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm12</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>33.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.8 </td><td>31.6 </td><td>3.0 </td><td>6.7 </td><td>6.0 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm10</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm10@109.0" target="_blank">Gm10@109.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>27.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>5.4 </td><td>28.3 </td><td>3.1 </td><td>5.3 </td><td>5.6 </td><td>–– </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm06</em> </td><td><a href="mailto:Gm06@103.0" target="_blank">Gm06@103.0</a> </td><td>s005700017 </td><td>s007100484 </td><td>s005700017 </td><td>11.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>19.3 </td><td>−73.0 </td><td>9.5 </td><td>9.2 </td><td>## </td><td> </td><td><a href="mailto:Gm06@104.4" target="_blank">Gm06@104.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>23.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.6 </td><td>–29.7 </td><td>3.2 </td><td>5.1 </td><td>5.8 </td><td>Confirmed </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@61.5" target="_blank">Gm11@61.5</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>15.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>2.9 </td><td>–21.2 </td><td>2.8 </td><td>1.7 </td><td>5.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qNP_Gm16</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm16@69.5" target="_blank">Gm16@69.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>s026800043 </td><td>7.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">*</a> </td><td>1.4 </td><td>–18.5 </td><td>3.2 </td><td>16.3 </td><td>6.1 </td><td>–– </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td>TSW </td><td><em>qTSW_Gm19</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm19@102.0" target="_blank">Gm19@102.0</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>155.5<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>37.3 </td><td>37.9 </td><td>1.7 </td><td>18.3 </td><td>3.2 </td><td>–– </td><td>–– </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm12</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm12@50.0" target="_blank">Gm12@50.0</a> </td><td>s024200450 </td><td>s023500372 </td><td>s024200450 </td><td>44.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.0 </td><td>18.1 </td><td>1.5 </td><td>1.7 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm12</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm11</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm11@61.0" target="_blank">Gm11@61.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>36.2<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.5 </td><td>–16.3 </td><td>1.4 </td><td>1.4 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm06</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm06@105.4" target="_blank">Gm06@105.4</a> </td><td>Satt277 </td><td>sF3H </td><td>sF3H </td><td>35.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.4 </td><td>–17.4 </td><td>1.5 </td><td>3.2 </td><td>2.6 </td><td>Confirmed </td><td>–– </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm10</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm10@110.0" target="_blank">Gm10@110.0</a> </td><td>FT2-300H01d </td><td>T001045537m </td><td>FT2-300H01d </td><td>20.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.6 </td><td>10.7 </td><td>1.5 </td><td>5.2 </td><td>2.8 </td><td>Confirmed </td><td>–– </td><td><em>E2</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm16</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm16@68.5" target="_blank">Gm16@68.5</a> </td><td>GMES4751 </td><td>s026800043 </td><td>GMES4751 </td><td>8.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.5 </td><td>–9.7 </td><td>1.5 </td><td>9.4 </td><td>2.9 </td><td>Confirmed </td><td>–– </td><td><em>GmFT5a</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qTSW_Gm20</em> </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td>NA </td><td> </td><td><a href="mailto:Gm20@99.0" target="_blank">Gm20@99.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.8<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.1 </td><td>–9.4 </td><td>2.0 </td><td>1.5 </td><td>3.9 </td><td>Confirmed </td><td>–– </td><td><em>qTSW_Gm20</em> </td><td>Seed yield 15–15 </td><td>Kabelka et al. 2004 </td></tr><tr><td>SWH </td><td><em>qSWH_Gm08</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm08@46.0" target="_blank">Gm08@46.0</a> </td><td>GMES1620 </td><td>AY262686B </td><td>AY262686B </td><td>33.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>10.1 </td><td>–2.8 </td><td>0.2 </td><td>1.9 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm08</em> </td><td>Seed weight 4–5 </td><td>Maughan et al. 1996 (close to I locus) </td></tr><tr><td> </td><td><em>qSWH_Gm06</em> </td><td><a href="mailto:Gm06@98.4" target="_blank">Gm06@98.4</a> </td><td>Satt277 </td><td>C06-BARC- 014491-01562 </td><td>Satt277 </td><td>8.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>17.1 </td><td>–2.4 </td><td>0.4 </td><td>1.7 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm06@106.4" target="_blank">Gm06@106.4</a> </td><td>sF3H </td><td>FT1SSR9 </td><td>FT1SSR9 </td><td>29.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>8.8 </td><td>–2.3 </td><td>0.2 </td><td>1.1 </td><td>0.4 </td><td>Confirmed(partially) </td><td>Confirmed </td><td><em>E1</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm11</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm11@59.0" target="_blank">Gm11@59.0</a> </td><td>GMES0766 </td><td>T001111280m </td><td>T001111280m </td><td>22.3<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>6.6 </td><td>–1.8 </td><td>0.2 </td><td>0.0 </td><td>0.4 </td><td>Confirmed </td><td>–– </td><td><em>qDFF_Gm11</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm17</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm17@51.0" target="_blank">Gm17@51.0</a> </td><td>T001709449l </td><td>T001710954m </td><td>T001710954m </td><td>15.1<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>4.4 </td><td>–2.0 </td><td>0.3 </td><td>0.5 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm17</em> </td><td>Seed yield 5-2 </td><td>Renyna and Sneller 2001 </td></tr><tr><td> </td><td><em>qSWH_Gm19</em> </td><td><a href="mailto:Gm19@118.9" target="_blank">Gm19@118.9</a> </td><td>FT3-751 </td><td>C19-BARC- 018175-02533 </td><td>FT3-751 </td><td>8.4<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>15.9 </td><td>2.4 </td><td>0.4 </td><td>1.1 </td><td>0.5 </td><td> </td><td><a href="mailto:Gm19@107.1" target="_blank">Gm19@107.1</a> </td><td>Sat_286 </td><td>FT3-751 </td><td>FT3-751 </td><td>10.7<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>3.1 </td><td>1.1 </td><td>0.2 </td><td>0.1 </td><td>0.3 </td><td>Confirmed </td><td>Confirmed </td><td><em>Dt1/E3</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm20</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm20@98.0" target="_blank">Gm20@98.0</a> </td><td>T002042675s </td><td>Sct_189 </td><td>T002042675s </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.5 </td><td>0.3 </td><td>1.1 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qSWH_Gm20</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm13</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm13@16.0" target="_blank">Gm13@16.0</a> </td><td>Satt146 </td><td>Satt252 </td><td>Satt252 </td><td>6.6<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.9 </td><td>–1.1 </td><td>0.2 </td><td>0.0 </td><td>0.5 </td><td>Confirmed(partially) </td><td>–– </td><td><em>qDFF_Gm13</em> </td><td> </td><td>“ </td></tr><tr><td> </td><td><em>qSWH_Gm02</em> </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td>–– </td><td> </td><td><a href="mailto:Gm02@22.0" target="_blank">Gm02@22.0</a> </td><td>Sat_096 </td><td>Sat_351 </td><td>Sat_351 </td><td>5.9<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn">**</a> </td><td>1.7 </td><td>–0.9 </td><td>0.2 </td><td>–0.4 </td><td>0.5 </td><td>Confirmed </td><td>–– </td><td><em>qSWH_Gm02</em> </td><td> </td><td>“ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>NA, the trait was not evaluated in the F<sub>2</sub> population; LOD, logarithm of odds of the presence of a QTL; PVE, percentage of variance explained by the QTL.</p></div></div><div content-id="tblfn3"><div class="fn"><div class="label -label">a</div><p>Sowing date of F<sub>2</sub> population and BC<sub>3</sub>F<sub>3</sub> population were 30 May 2007 and 24 June2011, respectively.</p></div></div><div content-id="tblfn4"><div class="fn"><div class="label -label">b</div><p>Indicated as chromosome name and position in cM separated by @.</p></div></div><div content-id="tblfn5"><div class="fn"><div class="label -label">c</div><p>‘Confirmed partially’ indicates that a singificant difference was observed between Enrei and a CSSL habouring several donor choromosomal fragments.</p></div></div><div content-id="tblfn6"><div class="fn"><div class="label -label">d</div><p>Effects of the QTLs corresponding to <em>E3</em> and <em>Dt1</em> loci were calculated separately.</p></div></div><div content-id="tblfn7"><div class="fn"><div class="label -label">*</div><p>Significant at <em>P &lt; </em>0.05 after 1,000 permutation; **Significant at <em>P &lt; </em>0.01 after 1,000 permutation.</p></div></div></div></div></p><p>Our high-density linkage map offers an opportunity to explore candidate genes for flowering time and related QTLs. For instance, the physical location of two major QTLs, <em>qDFF_Gm19</em> and <em>qDFF_Gm10</em>, estimated from the Marey map was 47.6 Mb and 44.9 Mb, respectively. Genes responsible for <em>E3</em><a reveal-id="dsx043-B57" data-open="dsx043-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and <em>E2</em><a reveal-id="dsx043-B58" data-open="dsx043-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> are located at ∼47.5 Mb on Gm19 and ∼44.7 Mb on Gm10, respectively. The estimated physical locations of the two QTLs differ from those of <em>E2</em> and <em>E3</em> by only ∼200 kb. Thus, the positions of SSR markers and interpolated genetic distance provide great potential to narrow down the boundaries of genomic regions that include candidate genes.</p>                            <h3 scrollto-destination=115733316 id="115733316" class="section-title" >3.5. QTLs in BC<sub>3</sub>F<sub>2</sub> population</h3>
<p>Nine agronomic traits (<a reveal-id="dsx043-T1" data-open="dsx043-T1" class="link link-reveal link-table xref-fig">Table 1</a>) of 999 BC<sub>3</sub>F<sub>2</sub> individuals were evaluated and the average values of each trait was compared with those of the recurrent parent (Enrei); the average value of each trait of BC<sub>3</sub>F<sub>2</sub> were almost the same as those of Enrei (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4, Fig. S4</a></span>). Whereas the effects of Peking alleles increase the phenotypic variance observed in BC<sub>3</sub>F<sub>2.</sub> Among the genetic variances (heritability) of all traits, some traits such as DFF, DFT and PH were highly heritable, whereas heritability of yield-related traits, such as TSW and SWH, were intermediate, and that of DH and RP was very low (<a reveal-id="dsx043-T1" data-open="dsx043-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Enrei tends to suffer from green stem disorder, caused by biotic and abiotic stresses<a reveal-id="dsx043-B59" data-open="dsx043-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a>; this disorder, which delays harvesting, would broaden the range of the values of maturity-related traits of the recurrent parent and the BC<sub>3</sub>F<sub>2</sub> population and decrease the heritability of these traits.</p><p>The linkage map was constructed for the BC<sub>3</sub>F<sub>2</sub> population to perform QTL mapping for these nine traits. The genotype of each BC<sub>3</sub>F<sub>2</sub> line was determined by using 320 SSR markers selected from the high-density genetic linkage map described above, so that they covered the whole genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The ratios of Peking-homozygous (3.1%), heterozygous (6.3%) and Enrei-homozygous (90.6%) loci in the population coincided well with the theoretical ratios (3.3%, 6.0% and 89.1%, respectively). The map consisted of 20 linkage groups covering 2475.3 cM with an average interval of 8.3 cM, and showed a genetic order of DNA markers similar to that expected from the high-density linkage map of the F<sub>2</sub> population.</p><p>In total, 56 QTLs scattered over 12 chromosomes were identified from the positions of LOD peaks for the nine traits. Six QTLs for DFF, six for DFT, two for FP, nine for DH, eight for RP, four for PH, six for NP, seven for TSW and eight for SWH showed significant LOD scores. Among them, 53 QTLs (expect for <em>qDH_Gm01</em>, <em>qDH_Gm17</em> and <em>qNP</em>_<em>Gm16</em>) showed higher LOD score than 1% significant level of threshold value (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Six out of seven QTLs for DFF detected in the F<sub>2</sub> population were reproducible in the BC<sub>3</sub>F<sub>2</sub> population. As in the F<sub>2</sub> population, many QTLs for other traits were clustered with QTLs for DFF, suggesting that QTLs for flowering time strongly affect other traits. However, additional QTLs for maturity and yield traits were separate from DFF QTLs in the BC<sub>3</sub>F<sub>2</sub> population.</p><p>We previously characterized the sequences of the known flowering genes, <em>E1–E4</em>, in many soybean cultivars<a reveal-id="dsx043-B36" data-open="dsx043-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a>; on this information, Enrei has <em>E1</em>, <em>e2</em>, <em>e3</em> and <em>E4</em> alleles, and Peking has <em>E1</em>, <em>E2</em>, <em>E3</em> and <em>E4</em> alleles. Therefore, we expected that only the <em>E2</em> and <em>E3</em> loci would segregate in the population; however, <em>qDFF_Gm06</em> was identified in a region close to the <em>E1</em> locus (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Allelic effects of <em>qDFF</em>s agreed well with the results of QTL analysis in the F<sub>2</sub> population. In addition, two soybean orthologs of <em>Arabidopsis Flowering locus T</em> (FT), <em>GmFT2a</em><a reveal-id="dsx043-B60" data-open="dsx043-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and <em>GmFT5a</em><a reveal-id="dsx043-B61" data-open="dsx043-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> delay flowering in cultivars adapted to northern Japan. The location of <em>qDFF_Gm16</em> was very close to the <em>GmFT5a</em> locus, and the Peking allele delayed flowering.</p><p>A major QTL for DFT, <em>qDFT_Gm19</em>, was identified close to the <em>Dt1</em> locus. The Peking allele significantly delayed flowering of the top raceme. The dominant allele of <em>Dt1</em>, which is a homolog of Arabidopsis <em>TERMINAL FLOWER 1</em>, controls indeterminate growth habit in soybean.<a reveal-id="dsx043-B62" data-open="dsx043-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> The growth habit of Peking is closer to that of a semi-determinate isoline (<em>Dt1</em>, <em>Dt2</em>) than to that of a determinate isoline (<em>dt1</em>, <em>dt2</em>) in top-leaf and stem traits, but the Peking allele is controlled by <em>dt1-t</em>, which is allelic to <em>dt1</em>.<a reveal-id="dsx043-B63" data-open="dsx043-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Together with the semi-determinate growth of Peking, delaying flowering of the top raceme will increase both the number of nodes on the main stem and plant height. The corresponding traits, NP and PH, were controlled by QTLs detected near the <em>Dt1</em> locus (<a reveal-id="dsx043-F4" data-open="dsx043-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The genetic variance of DFT and PH explained by the <em>Dt1</em> locus accounted up to 55.3% and 45.7%, respectively, of total variance (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>), indicating that the <em>Dt1</em> locus strongly affects these traits. </p>                        <a id="115733322" scrollto-destination="115733322"></a>
<div id="dsx043-F4" data-id="dsx043-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f4.png?Expires=2147483647&Signature=KJ-nxXl4~aYsr-peYWTwBzGXRDBAsicA5NqhOySFNp0~N9fhiDGaLZyAVurV3eIElu5LPmu51aYdf7vd6-1ldugTXRpv4Ngfcw5Cd7ziFqCKEPYxoUAMWi43QRBzxxwYhTnKjtXc7R3p7xNO0uNisZLLG~Z-h-T2dzRLpi8GDdTuYy671jv~bEGE0z8jq6eFLFg4mcCmpkK9168zovggjEahO~sh1OIEG0UxDvshvlCp5aknttDsmZVy5Ci5BQqUk6n3glQIO67NatRLRSWcyjDT4k2I8wG~DwOXQRwr8tZRegThHw2TgVkGJD8UQNNtLiI-Vu7FM2kX3P63dUulEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Whole-chromosome view of the graphical genotypes of the CSSLs and locations of QTLs identified in the BC3F2 population. Marker genotypes: black, Peking homozygous; white, Enrei homozygous; gray, heterozygous. QTL positions on the right side are in black." path-from-xml="dsx043f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733322/dsx043f4.tif" path-from-xml="dsx043f4.tif" target="_blank">View large</a><a section="115733322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f4.png?Expires=2147483647&Signature=zF4knXOvEGXBztWP945YqjbDP7sVo59deac7WibOZT7fcOPtSVC1VI0mPYclBoHbqnN5b8s1Z97gvZWQT~btftMnKUBEjmYQq98CJKE7lQxQzSuDsBZInI~pfQ5A2-jBlP28jLOmLs55ruRyxmomu~Ll~t5-ryCGc46y5Q5W8zU1JKeKOGBUcZTGqfifmLL38Gs2UDMt9TYj5TauJXVAzhY5WU3UpGrv-x4O1-GB0VUHNAJ068v33um-fUKxQ8GUzvDJhpIH-sx8s00CEks7NFIoN21vp-EHvDwm7wgaGnwI0GxhgZ0l3ASjo45x9zE7axrK3A6abnm97qT6k0Xn4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733322&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Whole-chromosome view of the graphical genotypes of the CSSLs and locations of QTLs identified in the BC<sub>3</sub>F<sub>2</sub> population. Marker genotypes: black, Peking homozygous; white, Enrei homozygous; gray, heterozygous. QTL positions on the right side are in black.</p></div></div><div content-id="dsx043-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f4.png?Expires=2147483647&Signature=KJ-nxXl4~aYsr-peYWTwBzGXRDBAsicA5NqhOySFNp0~N9fhiDGaLZyAVurV3eIElu5LPmu51aYdf7vd6-1ldugTXRpv4Ngfcw5Cd7ziFqCKEPYxoUAMWi43QRBzxxwYhTnKjtXc7R3p7xNO0uNisZLLG~Z-h-T2dzRLpi8GDdTuYy671jv~bEGE0z8jq6eFLFg4mcCmpkK9168zovggjEahO~sh1OIEG0UxDvshvlCp5aknttDsmZVy5Ci5BQqUk6n3glQIO67NatRLRSWcyjDT4k2I8wG~DwOXQRwr8tZRegThHw2TgVkGJD8UQNNtLiI-Vu7FM2kX3P63dUulEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Whole-chromosome view of the graphical genotypes of the CSSLs and locations of QTLs identified in the BC3F2 population. Marker genotypes: black, Peking homozygous; white, Enrei homozygous; gray, heterozygous. QTL positions on the right side are in black." path-from-xml="dsx043f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733322/dsx043f4.tif" path-from-xml="dsx043f4.tif" target="_blank">View large</a><a section="115733322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f4.png?Expires=2147483647&Signature=zF4knXOvEGXBztWP945YqjbDP7sVo59deac7WibOZT7fcOPtSVC1VI0mPYclBoHbqnN5b8s1Z97gvZWQT~btftMnKUBEjmYQq98CJKE7lQxQzSuDsBZInI~pfQ5A2-jBlP28jLOmLs55ruRyxmomu~Ll~t5-ryCGc46y5Q5W8zU1JKeKOGBUcZTGqfifmLL38Gs2UDMt9TYj5TauJXVAzhY5WU3UpGrv-x4O1-GB0VUHNAJ068v33um-fUKxQ8GUzvDJhpIH-sx8s00CEks7NFIoN21vp-EHvDwm7wgaGnwI0GxhgZ0l3ASjo45x9zE7axrK3A6abnm97qT6k0Xn4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733322&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Whole-chromosome view of the graphical genotypes of the CSSLs and locations of QTLs identified in the BC<sub>3</sub>F<sub>2</sub> population. Marker genotypes: black, Peking homozygous; white, Enrei homozygous; gray, heterozygous. QTL positions on the right side are in black.</p></div></div><p>The locations of QTLs for yield-related traits NP and TSW in the BC<sub>3</sub>F<sub>2</sub> population were very similar to those of QTLs for DFF in the F<sub>2</sub> population. The ratios of genetic variance explained by six flowering-time QTLs and the <em>Dt1</em> locus were examined but the order of effects of <em>qDFF</em>s did not coincide with that of post-flowering including yield-related traits. For example, <em>qRP_Gm11</em>, detected at the same locus as <em>qDFF_Gm11</em>, had a larger additive effect on RP (−3.4 days) than on DFF (−1.2 days). Each QTL for DFF had a different effect on the phenotypic variance of other traits, likely reflecting functional differences between the underlying genes. Hence, dissection of these QTLs as single Mendelian factors would provide more precise information about their effects.</p><p>Four QTLs for SWH were identified on Gm02, Gm08, Gm17 and Gm20, where no QTLs for DFF were identified. None of the QTLs from small-seeded Peking increased SWH. The Peking allele of <em>qSWH_Gm08</em> had the strongest effect of decreasing SWH (10% of phenotypic variance in the BC<sub>3</sub>F<sub>2</sub> population explained). Because this QTL is close to <em>Rhg4</em>, an important locus for cyst nematode resistance of Peking,<a reveal-id="dsx043-B64" data-open="dsx043-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> pre-breeding of an <em>Rhg4</em> NIL to remove such an unfavourable Peking allele is important for breeding of large-seeded cultivars. Previous study<a reveal-id="dsx043-B65" data-open="dsx043-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> identified a QTL (<em>qSW17-1OA</em>) with stable effects on seed weight in diverse environments over several years. The genetic position of <em>qSWH_Gm17</em> is likely to be the same as that of <em>qSW17-1OA</em> and some other QTLs in previous study (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The locations of <em>qSWH_Gm20</em> and <em>qTSW_Gm20</em> were close to that of <em>Ln</em>, which controls leaflet shape and seed size.<a reveal-id="dsx043-B66" data-open="dsx043-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> The gene responsible for <em>Ln</em> is homologous to Arabidopsis <em>JAGGED</em>.<a reveal-id="dsx043-B67" data-open="dsx043-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Although most Peking alleles had negative effects on yield-related traits in the Enrei genetic background, two QTLs related to late flowering (q<em>DFF_Gm10</em>, <em>qDFF_Gm19</em>) and one QTL extending the reproductive period (<em>qRP_Gm19</em>) associated with increased TSW in this genetic background (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>).</p>                            <h3 scrollto-destination=115733325 id="115733325" class="section-title" >3.6. Development of CSSLs and confirmation of QTLs</h3>
<p>The graphical genotypes of the selected 103 CSSLs are shown in <a reveal-id="dsx043-F4" data-open="dsx043-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>. The average length (± standard deviation) of the donor chromosomal segment was 85.4 ± 42.1 cM for homozygous and 130.8 cM ± 59.6 cM for heterozygous alleles. A few donor segments other than the target segment remained in the genetic background of the recurrent parent; therefore, further backcrossing with MAS would be necessary to eliminate these extra segments. However, CSSLs harboring different donor chromosomal segments from Peking were still useful to dissect QTLs as single genetic factors and to evaluate the genetic effects of individual QTLs.</p><p>Two-year evaluations of nine agronomic traits using 103 CSSLs revealed that 50 of these lines had at least one trait that was significantly different from that of Enrei by the Tukey-Kramer test (<em>P</em> &lt; 0.01). Most lines whose donor segments contained QTLs originating from Peking showed significantly different phenotypes. These differences agreed well with the effects of QTLs identified in the BC<sub>3</sub>F<sub>2</sub> population (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Of the 57 QTLs detected in the F<sub>2</sub> and BC<sub>3</sub>F<sub>2</sub> populations, 49 were also detected in CSSLs (<a reveal-id="dsx043-T1" data-open="dsx043-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Therefore, we considered that majority of QTLs reported in this study are highly reliable. Interestingly, four CSSLs B0704, B0804, B0816 and B0879 had the same genotype as Enrei at all DFF QTLs but had a significantly (<em>P</em> &lt;0.01) different DFF from Enrei (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The results indicate that the effect of a novel QTL appeared in the simple genetic background of a CSSL but was undetectable in the BC<sub>3</sub>F<sub>2</sub> population. In 2011 and 2012, 36 CSSLs showed significant differences in TSW from Enrei; 27 of them had QTLs for DFF, RP and SWH, but genetic factors in the remaining lines were not identified. Increasing the number of plants used for evaluation or crossing with Enrei would be necessary to uncover the cause of the increase in TSW and to use this QTL in soybean breeding programs.</p><p>We further evaluated the progeny of each line heterozygous for each of six flowering-time QTLs (<em>E1</em>, <em>E2</em>, <em>E3</em>, <em>qDFF_Gm11</em>, <em>qDFF_Gm12</em> and <em>GmFT5a</em>) to evaluate whether the chromosomal segments from Peking contained flowering time QTLs. Progeny classification according to the genotypes of SSR markers near each QTL showed significant association between phenotypic values and QTL genotypes (data not shown). Representative graphical genotypes of two CSSLs harboring donor chromosomal segments that included <em>qDFF_Gm11</em> (B0015 and B0676) and the effect of <em>qDFF_Gm11</em> on the phenotype are shown in <a reveal-id="dsx043-F5" data-open="dsx043-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>. Both lines differed significantly (<em>P </em>&lt;0.01) from Enrei in RP (<a reveal-id="dsx043-F5" data-open="dsx043-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). <em>qDFF_Gm11</em> had the fifth strongest effect among seven DFF QTLs (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>), whereas <em>qDFF_Gm11</em> had the strongest effect on RP among all RP QTLs. The decreased RP resulted in a clear difference in the maturity phenotype (<a reveal-id="dsx043-F5" data-open="dsx043-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>), and this QTL also affected TSW and SWH (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Comparison of the effects of <em>qDFF</em>(<em>qRP</em>)<em>_Gm11</em> on different traits suggests that the gene responsible for this QTL probably extends the pod-filling period. The results described above indicate that CSSLs developed in this study may help to dissect the genes underlying the detected QTLs into single genetic factors and provide breeding materials with the genetic background of an elite Japanese cultivar. </p>                        <a id="115733329" scrollto-destination="115733329"></a>
<div id="dsx043-F5" data-id="dsx043-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f5.png?Expires=2147483647&Signature=R2bOHVpUgC80XrulK902VJtM~Tyftwa~E9~-6c65sWVLlRvmFNJqbXLUuBpc2-uo7f8-HGMwTmYiJo7Vz4oFOnmNxq-hZV1h9TOaIJkeZksxcPBpH4HQhTzAFe93EaL6POoWxWL3tY~H4cd9HVUfQ6C1wVO98-W0Lq5uv2n8BaCUUAnB0z7-cU0FflOdJG6uCxviMErPP7kd6o0Ie2QhSsSnsv-wSzfUfMaEBfQKM717hkpQfKxyhl84XLRwmmjyXcBWUVVUuh1S5grg8wPHHctgnfU7Jq2~UhCmrOsOVG0xXTPUqO0OqwHO2UNxd~Vzs5qNYFDBBtadueY~djhRSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Confirmation of QTL effects on the reproductive period using CSSL lines. A, Graphical genotypes of the CSSLs B0676 and B0015. These lines have donor (Peking) chromosomal segments containing qDFF_Gm11, which affects the length of the reproductive period (arrowheads). B, Reproductive periods of the two CSSLs and the recurrent parent Enrei (En). C, Representative images of CSSL B0676 (right) in comparison with the recurrent parent Enrei (left). Early maturity in B0676 is caused by a chromosome segment that includes qDFF_Gm11 from Peking. " path-from-xml="dsx043f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733329/dsx043f5.tif" path-from-xml="dsx043f5.tif" target="_blank">View large</a><a section="115733329" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f5.png?Expires=2147483647&Signature=Nk3-IYgHJEVxqCvF7ZR~96~utJ6yM9eJCiFQPzrodKMfphTIqVE1b7Too0Qt7RyFTkcxoDH~bXj-CEi~9Tr5gFy1fuQ8fr-vGLqG3ot3j~S2Fl8nDDlfVZp8rSwbWYw6Ky2skntkXkarv6LcCa8AhxM~VMRCRIy~X-1Bb1jCSzdh1ESlRx9CIqbvcRh9XnfgwZGLyfRR8kWVjNkTkK~SH9XESxqhZszEvGNzHAgqFhpW8tCLzsBE9LOm1F9b6t9q43MVwyTnYKR-fp5fihgXOJYaHOng-Ir8nnt8PH-XqxpSuTkp08BpMvvVq6e2fAoNW6t48i6W2GHYpeqJoYhGPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733329&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Confirmation of QTL effects on the reproductive period using CSSL lines. A, Graphical genotypes of the CSSLs B0676 and B0015. These lines have donor (Peking) chromosomal segments containing <em>qDFF_Gm11</em>, which affects the length of the reproductive period (arrowheads). B, Reproductive periods of the two CSSLs and the recurrent parent Enrei (En). C, Representative images of CSSL B0676 (right) in comparison with the recurrent parent Enrei (left). Early maturity in B0676 is caused by a chromosome segment that includes <em>qDFF_Gm11</em> from Peking. </p></div></div><div content-id="dsx043-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/m_dsx043f5.png?Expires=2147483647&Signature=R2bOHVpUgC80XrulK902VJtM~Tyftwa~E9~-6c65sWVLlRvmFNJqbXLUuBpc2-uo7f8-HGMwTmYiJo7Vz4oFOnmNxq-hZV1h9TOaIJkeZksxcPBpH4HQhTzAFe93EaL6POoWxWL3tY~H4cd9HVUfQ6C1wVO98-W0Lq5uv2n8BaCUUAnB0z7-cU0FflOdJG6uCxviMErPP7kd6o0Ie2QhSsSnsv-wSzfUfMaEBfQKM717hkpQfKxyhl84XLRwmmjyXcBWUVVUuh1S5grg8wPHHctgnfU7Jq2~UhCmrOsOVG0xXTPUqO0OqwHO2UNxd~Vzs5qNYFDBBtadueY~djhRSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Confirmation of QTL effects on the reproductive period using CSSL lines. A, Graphical genotypes of the CSSLs B0676 and B0015. These lines have donor (Peking) chromosomal segments containing qDFF_Gm11, which affects the length of the reproductive period (arrowheads). B, Reproductive periods of the two CSSLs and the recurrent parent Enrei (En). C, Representative images of CSSL B0676 (right) in comparison with the recurrent parent Enrei (left). Early maturity in B0676 is caused by a chromosome segment that includes qDFF_Gm11 from Peking. " path-from-xml="dsx043f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733329/dsx043f5.tif" path-from-xml="dsx043f5.tif" target="_blank">View large</a><a section="115733329" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043f5.png?Expires=2147483647&Signature=Nk3-IYgHJEVxqCvF7ZR~96~utJ6yM9eJCiFQPzrodKMfphTIqVE1b7Too0Qt7RyFTkcxoDH~bXj-CEi~9Tr5gFy1fuQ8fr-vGLqG3ot3j~S2Fl8nDDlfVZp8rSwbWYw6Ky2skntkXkarv6LcCa8AhxM~VMRCRIy~X-1Bb1jCSzdh1ESlRx9CIqbvcRh9XnfgwZGLyfRR8kWVjNkTkK~SH9XESxqhZszEvGNzHAgqFhpW8tCLzsBE9LOm1F9b6t9q43MVwyTnYKR-fp5fihgXOJYaHOng-Ir8nnt8PH-XqxpSuTkp08BpMvvVq6e2fAoNW6t48i6W2GHYpeqJoYhGPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733329&ar=4657153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx043f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Confirmation of QTL effects on the reproductive period using CSSL lines. A, Graphical genotypes of the CSSLs B0676 and B0015. These lines have donor (Peking) chromosomal segments containing <em>qDFF_Gm11</em>, which affects the length of the reproductive period (arrowheads). B, Reproductive periods of the two CSSLs and the recurrent parent Enrei (En). C, Representative images of CSSL B0676 (right) in comparison with the recurrent parent Enrei (left). Early maturity in B0676 is caused by a chromosome segment that includes <em>qDFF_Gm11</em> from Peking. </p></div></div>                            <h3 scrollto-destination=115733330 id="115733330" class="section-title" >3.7. Novel flowering-related genes</h3>
<p>We identified seven QTLs related to DFF in F<sub>2</sub>, BC<sub>3</sub>F<sub>2</sub> populations and selected CSSLs (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Among soybean maturity genes, the classical <em>E1</em> locus has the strongest effect on flowering time.<a reveal-id="dsx043-B68" data-open="dsx043-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> In response to photoperiod, a putative transcription factor encoded by <em>E1</em> controls flowering time by regulating the expression of <em>FT</em> genes.<a reveal-id="dsx043-B69" data-open="dsx043-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> The detection of <em>qDFF_Gm06</em> close to <em>E1</em> in the F<sub>2</sub> and BC<sub>3</sub>F<sub>2</sub> populations suggests that the <em>E1</em> allele differs between Peking and Enrei. We assume that Peking has a recessive allele because of an early flowering effect of this QTL (<a reveal-id="dsx043-T2" data-open="dsx043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). However, the coding region of Peking <em>E1</em> is reported to be identical to that of Enrei <em>E1</em>.<a reveal-id="dsx043-B36" data-open="dsx043-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Thus, Peking may have a recessive allele different from the known one, because several different types of recessive alleles have been identified for soybean maturity genes, such as the <em>E4</em> locus,<a reveal-id="dsx043-B70" data-open="dsx043-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> and the promoter region of Peking has a high similarity to those of other cultivars with recessive <em>e1</em> alleles.<a reveal-id="dsx043-B36" data-open="dsx043-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Alternatively, a novel gene affecting soybean flowering time and maturity may be located in this region. The <em>qDFF_Gm06</em> was located in the pericentromeric region (<a reveal-id="dsx043-F3" data-open="dsx043-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The other maturity gene <em>E7</em> locus for flowering is located on the same chromosome and is genetically tightly linked to the <em>E1</em> locus.<a reveal-id="dsx043-B71" data-open="dsx043-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> Fine mapping would be needed to resolve these possibilities.</p><p>Genes responsible for <em>qDFF_Gm11</em>, <em>qDFF_Gm12</em> and <em>qDFF_Gm13</em> (and nearby QTLs for other traits) have not been characterized. QTLs corresponding to <em>qDFF_Gm12</em> have been identified in two different populations derived from crosses between wild and cultivated soybeans.<a reveal-id="dsx043-B72" data-open="dsx043-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Satt442 (the closest DNA marker in the above study) was located close to the marker s024200450 in the present study. Previous study<a reveal-id="dsx043-B73" data-open="dsx043-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> reported QTLs at positions similar to that of <em>qDFF_Gm11</em>. They identified a strong QTL, named <em>qDfm1</em> (<em>Duration from flowering to maturity</em>), using RILs derived from a cross between Fukuyutaka and Ippon-sango in a low-latitude region of Japan (Kyushu, 32°52′N). Although the positions of <em>qDFF_Gm11</em> and <em>qDfm1</em> are close to each other on the linkage map, the effect of qDfm1 was not associated with DFF.<a reveal-id="dsx043-B73" data-open="dsx043-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> The difference might be related to population size (192 F<sub>2</sub> or 999 BC<sub>3</sub>F<sub>2</sub> plants vs. 143 RILs) and a short photoperiod at low latitudes. In any case, both QTLs control RP under different photoperiod and temperature conditions (in the middle and southern parts of Japan). If the same gene is responsible for these QTLs, the Peking allele, which shortens RP, would be useful in wide latitudinal range. The evaluation of genetic interactions between <em>E</em> genes and <em>qDFF_Gm11</em> by crossing CSSLs having different qDFF alleles would be important for controlling maturity.</p>                            <h3 scrollto-destination=115733333 id="115733333" class="section-title" >3.8. Future characterization of useful traits in Peking</h3>
<p>Peking was first identified as material resistant to the soybean cyst nematode <em>H. glycines</em>.<a reveal-id="dsx043-B74" data-open="dsx043-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> Peking was introduced into the USA from Beijing, China, in 1906 (previous accession number: PI17852B; current USDA accession number: PI548402). However, other USDA accessions named Peking differ from PI548402 by RFLP analysis<a reveal-id="dsx043-B11" data-open="dsx043-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a>: PI297543 (introduced from Hungary in 1964), PI438496 and PI438497 (from the Russian Federation in 1979). The Peking accession used in the present study revealed an SSR genotype patterns very similar to those of PI548402, but quite different from the patterns of PI438496A, PI438496B, PI438496C, PI438497 and two pure lines, PI548205 and PI548359, derived from Peking in the USDA germplasm collection, and from the pattern of JP28432 in NARO Genebank (data not shown).</p><p>DNA markers for the soybean cyst nematode resistance genes <em>Rhg4</em> and <em>Rhg1</em>, based on functional SNPs in Peking, have been reported.<a reveal-id="dsx043-B64" data-open="dsx043-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Previous study<a reveal-id="dsx043-B75" data-open="dsx043-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> isolated the Peking gene <em>Rps1k</em>, which promotes resistance to stem rot disease caused by <em>Phytophthora sojae</em> races 2 and 11.<a reveal-id="dsx043-B19" data-open="dsx043-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Peking has also been used in studies of interactions with nitrogen-fixing rhizobia (<em>Rhizobium</em> and <em>Sinorhizobium</em> species) owing to their host specificity and genetic control of symbiosis by a host gene.<a reveal-id="dsx043-B76" data-open="dsx043-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> Peking carries <em>rfg1</em>, which controls effective nodulation of fast-growing rhizobia, and <em>Rj4</em>, which controls nodulation of slow-growing bradyrhizobia.<a reveal-id="dsx043-B77" data-open="dsx043-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> The responsible genes were isolated for <em>rfg1</em><a reveal-id="dsx043-B78" data-open="dsx043-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> and that for <em>Rj4</em>.<a reveal-id="dsx043-B79" data-open="dsx043-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> Peking is also resistant to the reniform nematode <em>Rotylenchulus reniformis</em>.<a reveal-id="dsx043-B18" data-open="dsx043-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p><p>Peking has a <em>Rsv4</em> gene for resistance to soybean mosaic virus and DNA markers have been developed for this gene.<a reveal-id="dsx043-B12" data-open="dsx043-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> A recessive gene, <em>rpv</em><sub>2</sub>, for resistance to peanut mottle virus has been reported in Peking.<a reveal-id="dsx043-B13" data-open="dsx043-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Peking is resistant to many isolates of the frogeye leaf spot pathogen, <em>Cercospora sojina</em>,<a reveal-id="dsx043-B15" data-open="dsx043-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> which is controlled by a single dominant gene, <em>Rcs<sub>Peking</sub></em>, mapped near the SSR marker Satt244, and another resistance gene, <em>Rcs3</em>.<a reveal-id="dsx043-B80" data-open="dsx043-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> Peking is tolerant to <em>Pseudomonas syringae</em> pv. <em>glycinea</em> race 6, which cause bacterial blight disease,<a reveal-id="dsx043-B14" data-open="dsx043-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and to soybean dwarf disease,<a reveal-id="dsx043-B16" data-open="dsx043-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> soybean stem canker<a reveal-id="dsx043-B17" data-open="dsx043-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and corn earworm.<a reveal-id="dsx043-B81" data-open="dsx043-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> Its seed-flooding tolerance at the germination stage has been explained by the structural characteristics of its seeds<a reveal-id="dsx043-B20" data-open="dsx043-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx043-B21" data-open="dsx043-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and is controlled by four QTLs.<a reveal-id="dsx043-B82" data-open="dsx043-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> The success rate of somatic embryogenesis from immature embryo cultures of Peking is reportedly high,<a reveal-id="dsx043-B83" data-open="dsx043-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> and six QTLs associated with somatic embryogenesis have been identified.<a reveal-id="dsx043-B84" data-open="dsx043-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> Susceptibility of Peking to tumor formation by <em>Agrobacterium tumefaciens</em> is controlled by several genes,<a reveal-id="dsx043-B85" data-open="dsx043-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> but these genes have not been isolated. The evaluation of CSSLs for resistance to these diseases and to stress would help to identify and characterize the resistance genes.</p>                            <h2 scrollto-destination=115733337 id="115733337" class="section-title" >4. Conclusion</h2>
<p>We developed a high-density linkage map and CSSLs carrying chromosomal segments from the Chinese soybean Peking in the background of the Japanese soybean Enrei. The map provides a detailed genetic framework within which to use molecular markers for breeding and to obtain a precise assembly of the genome sequence in the Japanese soybean genetic background. The marker order on the linkage map agreed well with the new genome assembly Wm82.a2.v1, but large differences were identified on Chrs. 05 and 11. These CSSLs are a unique resource that would be useful for evaluating minor QTLs as single genetic factors in a uniform genetic background. We demonstrated that many QTLs related to basic agronomic traits detected in the F<sub>2</sub> and BC<sub>3</sub>F<sub>2</sub> populations were reproducible in the CSSLs.</p><p>Peking has many useful genes for resistance to diseases, pests and stress, but most of these genes have not yet been characterized. CSSLs developed in this study would be a good resource for us in developing new cultivars harboring resistance genes and for gene isolation by positional cloning. To increase the genetic diversity of soybean for breeding, finding and confirmation of QTLs associated with agronomic traits using a mapping population derived from these lines would be necessary. CSSLs developed in this study would also be good starting materials for removing the unfavourable characteristics of Peking.</p>                            <h2 scrollto-destination=115733340 id="115733340" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Kazuhiro Yagasaki, Nagano Prefecture Vegetable and Ornamental Crops Experiment Station, for supplying soybean materials used in this study. We thank Koji Takahashi, Naohiro Yamada, Nobuhiko Oki, Kaori Hirata, Benitez Eduardo of Institute of Crop Science for their kind assistance for backcrossing. We also appreciate the technical support in field work from the staff of the National Institute of Agrobiological Sciences: T. Nobori, N. Karino, T. Ohmizu, T. Taguchi, Y. Tsubokura and K. Sugimoto. This work was supported by a grant from the Ministry of Agriculture, Forestry and Fisheries of Japan (Genomics for Agricultural Innovation, DD-1010, SOY1002, SOY2003).</p>                            <h3 scrollto-destination=115733342 id="115733342" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=115733344 id="115733344" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=115733346 id="115733346" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx043-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">178</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">408</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%20302%20wild%20and%20cultivated%20accessions%20identifies%20genes%20related%20to%20domestication%20and%20improvement%20in%20soybean&amp;author=Z.%20Zhou&amp;author=Y.%20Jiang&amp;author=Z.%20Wang&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=408-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25643055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kaga</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Anai</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Construction of a high-density mutant library in soybean and development of a mutant retrieval method using amplicon sequencing</div>, <div class="source">BMC Genom</div> ., <div class="volume">16</div>, <div class="fpage">1014</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20mutant%20library%20in%20soybean%20and%20development%20of%20a%20mutant%20retrieval%20method%20using%20amplicon%20sequencing&amp;author=M.%20Tsuda&amp;author=A.%20Kaga&amp;author=T.%20Anai&amp;publication_year=2015&amp;journal=BMC%20Genom&amp;volume=16&amp;pages=1014" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2079-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2079-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2079-y"> </span></div> </div></div></div></div></div><div content-id="dsx043-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cregan</div> <div class="given-names">P.B.</div></div>, <div class="name string-name"><div class="surname">Jarvik</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Bush</div> <div class="given-names">A.L.</div></div></div>, et al.   <div class="year">1999</div>, 
<div class="article-title">An integrated genetic linkage map of the soybean genome</div>, <div class="source">Crop Sci</div> ., <div class="volume">39</div>, <div class="fpage">1464</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%20linkage%20map%20of%20the%20soybean%20genome&amp;author=P.B.%20Cregan&amp;author=T.%20Jarvik&amp;author=A.L.%20Bush&amp;publication_year=1999&amp;journal=Crop%20Sci&amp;volume=39&amp;pages=1464-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1999.3951464x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1999.3951464x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1999.3951464x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Jia</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Abundance of SSR Motifs and development of candidate polymorphic SSR Markers (BARCSOYSSR_1.0) in soybean</div>, <div class="source">Crop Sci</div> ., <div class="volume">50</div>, <div class="fpage">1950</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%20of%20SSR%20Motifs%20and%20development%20of%20candidate%20polymorphic%20SSR%20Markers%20(BARCSOYSSR_1.0)%20in%20soybean&amp;author=Q.%20Song&amp;author=G.%20Jia&amp;author=Y.%20Zhu&amp;publication_year=2010&amp;journal=Crop%20Sci&amp;volume=50&amp;pages=1950" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2009.10.0607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2009.10.0607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2009.10.0607"> </span></div> </div></div></div></div></div><div content-id="dsx043-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">Q.J.</div></div>, <div class="name string-name"><div class="surname">Jenkins</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jia</div> <div class="given-names">G.F.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Construction of high resolution genetic linkage maps to improve the soybean genome sequence assembly Glyma1.01</div>, <div class="source">BMC Genom</div> ., <div class="volume">17</div>, <div class="fpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20high%20resolution%20genetic%20linkage%20maps%20to%20improve%20the%20soybean%20genome%20sequence%20assembly%20Glyma1.01&amp;author=Q.J.%20Song&amp;author=J.%20Jenkins&amp;author=G.F.%20Jia&amp;publication_year=2016&amp;journal=BMC%20Genom&amp;volume=17&amp;pages=33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2344-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2344-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2344-0"> </span></div> </div></div></div></div></div><div content-id="dsx043-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mahama</div> <div class="given-names">A.A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Cytogenetic analysis of translocations in soybean</div>, <div class="source">J. Heredity</div> , <div class="volume">90</div>, <div class="fpage">648</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytogenetic%20analysis%20of%20translocations%20in%20soybean&amp;author=A.A.%20Mahama&amp;publication_year=1999&amp;journal=J.%20Heredity&amp;volume=90&amp;pages=648-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/90.6.648" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F90.6.648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F90.6.648"> </span></div> </div></div></div></div></div><div content-id="dsx043-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Findley</div> <div class="given-names">S.D.</div></div>, <div class="name string-name"><div class="surname">Pappas</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Birchler</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">R.G.</div></div>, <div class="name string-name"><div class="surname">Stacey</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Fluorescence in situ hybridization-based karyotyping of soybean translocation lines</div>, <div class="source">G3</div> , <div class="volume">1</div>, <div class="fpage">117</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fluorescence%20in%20situ%20hybridization-based%20karyotyping%20of%20soybean%20translocation%20lines&amp;author=S.D.%20Findley&amp;author=A.L.%20Pappas&amp;author=Y.%20Cui&amp;author=J.A.%20Birchler&amp;author=R.G.%20Palmer&amp;author=G.%20Stacey&amp;publication_year=2011&amp;journal=G3&amp;volume=1&amp;pages=117-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.000034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.000034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.000034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shimomura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kanamori</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Komatsu</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The <em>Glycine max</em> cv. Enrei genome for improvement of Japanese soybean cultivars</div>, <div class="source">Int. J. Genom</div> ., <div class="volume">2015</div>, <div class="fpage">358127</div>.</div></div></div></div><div content-id="dsx043-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ross</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Brim</div> <div class="given-names">C.A.</div></div></div> <div class="year">1957</div>, 
<div class="article-title">Resistance of soybean to the soybean-cyst nematode as determined by a double-row method</div>, <div class="source">Plant Dis. Reporter</div> , <div class="volume">41</div>, <div class="fpage">923</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="dsx043-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lorenzen</div> <div class="given-names">L.L.</div></div></div> <div class="year">1994</div>, Soybean cultivar development: a genome perspective. Ph.D. Dissertation, Iowa State University, Ames, IA. Available at: <a class="link link-uri" href="http://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=11625&amp;context=rtd" target="_blank">http://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=11625&amp;context=rtd</a> (1 November 2017, date last accessed).</div></div></div></div><div content-id="dsx043-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hayes</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Buss</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Maroof</div> <div class="given-names">M.A.S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Molecular marker mapping of <em>Rsv4</em>, a gene conferring resistance to all known strains of soybean mosaic virus</div>, <div class="source">Crop Sci</div> ., <div class="volume">40</div>, <div class="fpage">1434</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20marker%20mapping%20of%20Rsv4%2C%20a%20gene%20conferring%20resistance%20to%20all%20known%20strains%20of%20soybean%20mosaic%20virus&amp;author=A.J.%20Hayes&amp;author=G.R.%20Ma&amp;author=G.R.%20Buss&amp;author=M.A.S.%20Maroof&amp;publication_year=2000&amp;journal=Crop%20Sci&amp;volume=40&amp;pages=1434-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2000.4051434x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2000.4051434x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2000.4051434x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shipe</div> <div class="given-names">E.R.</div></div>, <div class="name string-name"><div class="surname">Buss</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Tolin</div> <div class="given-names">S.A.</div></div></div> <div class="year">1979</div>, 
<div class="article-title">2nd gene for resistance to peanut mottle virus in soybeans</div>, <div class="source">Crop Sci</div> ., <div class="volume">19</div>, <div class="fpage">656</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=2nd%20gene%20for%20resistance%20to%20peanut%20mottle%20virus%20in%20soybeans&amp;author=E.R.%20Shipe&amp;author=G.R.%20Buss&amp;author=S.A.%20Tolin&amp;publication_year=1979&amp;journal=Crop%20Sci&amp;volume=19&amp;pages=656-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1979.0011183X001900050028x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1979.0011183X001900050028x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1979.0011183X001900050028x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Staskawicz</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Dahlbeck</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Keen</div> <div class="given-names">N.T.</div></div></div> <div class="year">1984</div>, 
<div class="article-title">Cloned avirulence gene of <em>Pseudomonas syringae</em> pv. <em>glycinea</em> determines race-specific incompatibility on <em>Glycine max</em> (L.) Merr</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">81</div>, <div class="fpage">6024</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloned%20avirulence%20gene%20of%20Pseudomonas%20syringae%20pv.%20glycinea%20determines%20race-specific%20incompatibility%20on%20Glycine%20max%20(L.)%20Merr&amp;author=B.J.%20Staskawicz&amp;author=D.%20Dahlbeck&amp;author=N.T.%20Keen&amp;publication_year=1984&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=81&amp;pages=6024-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.81.19.6024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.81.19.6024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.81.19.6024"> </span></div> </div></div></div></div></div><div content-id="dsx043-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baker</div> <div class="given-names">W.A.</div></div>, <div class="name string-name"><div class="surname">Weaver</div> <div class="given-names">D.B.</div></div>, <div class="name string-name"><div class="surname">Qui</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pace</div> <div class="given-names">P.F.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Genetic analysis of frogeye leaf spot resistance in PI54610 and Peking soybean</div>, <div class="source">Crop Sci</div> ., <div class="volume">39</div>, <div class="fpage">1021</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20frogeye%20leaf%20spot%20resistance%20in%20PI54610%20and%20Peking%20soybean&amp;author=W.A.%20Baker&amp;author=D.B.%20Weaver&amp;author=J.%20Qui&amp;author=P.F.%20Pace&amp;publication_year=1999&amp;journal=Crop%20Sci&amp;volume=39&amp;pages=1021-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1999.0011183X003900040010x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1999.0011183X003900040010x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1999.0011183X003900040010x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tanimura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tamada</div> <div class="given-names">T.</div></div></div> <div class="year">1976</div>, 
<div class="article-title">Breeding for resistance to soybean dwarf disease 1. Resistance of soybean varieties to soybean dwarf disease</div>, <div class="source">Bull. Hokkaido Prefect Agric. Exp. Stn</div> ., <div class="volume">35</div>, <div class="fpage">8</div>–<div class="lpage">17</div>.</div></div></div></div><div content-id="dsx043-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keeling</div> <div class="given-names">B.L.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">A seedling test for resistance to soybean stem canker caused by <em>Diaporthe phaseolorum</em> var <em>caulivora</em></div>, <div class="source">Phytopathology</div> , <div class="volume">72</div>, <div class="fpage">807</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20seedling%20test%20for%20resistance%20to%20soybean%20stem%20canker%20caused%20by%20Diaporthe%20phaseolorum%20var%20caulivora&amp;author=B.L.%20Keeling&amp;publication_year=1982&amp;journal=Phytopathology&amp;volume=72&amp;pages=807-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/Phyto-72-807" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPhyto-72-807" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPhyto-72-807"> </span></div> </div></div></div></div></div><div content-id="dsx043-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robbins</div> <div class="given-names">R.T.</div></div>, <div class="name string-name"><div class="surname">Rakes</div> <div class="given-names">L.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Resistance to the reniform nematode in selected soybean cultivars and germplasm lines</div>, <div class="source">J. Nematol</div> ., <div class="volume">28</div>, <div class="fpage">612</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resistance%20to%20the%20reniform%20nematode%20in%20selected%20soybean%20cultivars%20and%20germplasm%20lines&amp;author=R.T.%20Robbins&amp;author=L.%20Rakes&amp;publication_year=1996&amp;journal=J.%20Nematol&amp;volume=28&amp;pages=612-5" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19277183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pacumbaba</div> <div class="given-names">R.P.</div></div>, <div class="name string-name"><div class="surname">Sapra</div> <div class="given-names">V.T.</div></div>, <div class="name string-name"><div class="surname">Prom</div> <div class="given-names">L.K.</div></div></div> <div class="year">1984</div>, 
<div class="article-title">Effect of 2 commercial fungicides on incidence of <em>Diaporthe phaseolorum</em> var <em>caulivora</em> on susceptible soybean cultivars</div>, <div class="source">Phytopathology</div> , <div class="volume">74</div>, <div class="fpage">827</div>.</div></div></div></div><div content-id="dsx043-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tian</div> <div class="given-names">X.-H.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kokubun</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The role of seed structure and oxygen responsiveness in pre-germination flooding tolerance of soybean cultivars</div>, <div class="source">Plant. Prod. Sci</div> ., <div class="volume">8</div>, <div class="fpage">157</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20seed%20structure%20and%20oxygen%20responsiveness%20in%20pre-germination%20flooding%20tolerance%20of%20soybean%20cultivars&amp;author=X.-H.%20Tian&amp;author=T.%20Nakamura&amp;author=M.%20Kokubun&amp;publication_year=2005&amp;journal=Plant.%20Prod.%20Sci&amp;volume=8&amp;pages=157-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1626/pps.8.157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1626%2Fpps.8.157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1626%2Fpps.8.157"> </span></div> </div></div></div></div></div><div content-id="dsx043-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Muramatsu</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kokubun</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Horigane</div> <div class="given-names">A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Relation of seed structures to soybean cultivar difference in pre-germination flooding tolerance</div>, <div class="source">Plant. Prod. Sci</div> ., <div class="volume">11</div>, <div class="fpage">434</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relation%20of%20seed%20structures%20to%20soybean%20cultivar%20difference%20in%20pre-germination%20flooding%20tolerance&amp;author=N.%20Muramatsu&amp;author=M.%20Kokubun&amp;author=A.%20Horigane&amp;publication_year=2008&amp;journal=Plant.%20Prod.%20Sci&amp;volume=11&amp;pages=434-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1626/pps.11.434" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1626%2Fpps.11.434" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1626%2Fpps.11.434"> </span></div> </div></div></div></div></div><div content-id="dsx043-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuinstra</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Ejeta</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Goldsbrough</div> <div class="given-names">P.B.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">95</div>, <div class="fpage">1005</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneous%20inbred%20family%20(HIF)%20analysis%3A%20a%20method%20for%20developing%20near-isogenic%20lines%20that%20differ%20at%20quantitative%20trait%20loci&amp;author=M.R.%20Tuinstra&amp;author=G.%20Ejeta&amp;author=P.B.%20Goldsbrough&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet&amp;volume=95&amp;pages=1005-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050654"> </span></div> </div></div></div></div></div><div content-id="dsx043-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Toda</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Fine mapping of the <em>FT1</em> locus for soybean flowering time using a residual heterozygous line derived from a recombinant inbred line</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">110</div>, <div class="fpage">634</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20the%20FT1%20locus%20for%20soybean%20flowering%20time%20using%20a%20residual%20heterozygous%20line%20derived%20from%20a%20recombinant%20inbred%20line&amp;author=N.%20Yamanaka&amp;author=S.%20Watanabe&amp;author=K.%20Toda&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet&amp;volume=110&amp;pages=634-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1886-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1886-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1886-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15657740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eshed</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">An introgression line population of <em>Lycopersicon pennellii</em> in the cultivated tomato enables the identification and fine mapping of yield-associated qtl</div>, <div class="source">Genetics</div> , <div class="volume">141</div>, <div class="fpage">1147</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20introgression%20line%20population%20of%20Lycopersicon%20pennellii%20in%20the%20cultivated%20tomato%20enables%20the%20identification%20and%20fine%20mapping%20of%20yield-associated%20qtl&amp;author=Y.%20Eshed&amp;author=D.%20Zamir&amp;publication_year=1995&amp;journal=Genetics&amp;volume=141&amp;pages=1147-62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8582620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koumproglou</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wilkes</div> <div class="given-names">T.M.</div></div>, <div class="name string-name"><div class="surname">Townson</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">STAIRS: a new genetic resource for functional genomic studies of Arabidopsis</div>, <div class="source">Plant J. Cell Mol. Biol</div> ., <div class="volume">31</div>, <div class="fpage">355</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAIRS%3A%20a%20new%20genetic%20resource%20for%20functional%20genomic%20studies%20of%20Arabidopsis&amp;author=R.%20Koumproglou&amp;author=T.M.%20Wilkes&amp;author=P.%20Townson&amp;publication_year=2002&amp;journal=Plant%20J.%20Cell%20Mol.%20Biol&amp;volume=31&amp;pages=355-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2002.01353.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2002.01353.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2002.01353.x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ali</div> <div class="given-names">M.L.</div></div>, <div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lorieux</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Eizenga</div> <div class="given-names">G.C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Chromosome segment substitution lines: a powerful tool for the introgression of valuable genes from <em>Oryza</em> wild species into cultivated rice (<em>O. sativa</em>)</div>, <div class="source">Rice</div> , <div class="volume">3</div>, <div class="fpage">218</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20segment%20substitution%20lines%3A%20a%20powerful%20tool%20for%20the%20introgression%20of%20valuable%20genes%20from%20Oryza%20wild%20species%20into%20cultivated%20rice%20(O.%20sativa)&amp;author=M.L.%20Ali&amp;author=P.L.%20Sanchez&amp;author=S.%20Yu&amp;author=M.%20Lorieux&amp;author=G.C.%20Eizenga&amp;publication_year=2010&amp;journal=Rice&amp;volume=3&amp;pages=218-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12284-010-9058-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12284-010-9058-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12284-010-9058-3"> </span></div> </div></div></div></div></div><div content-id="dsx043-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">von Korff</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Léon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pillen</div> <div class="given-names">K.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Development of candidate introgression lines using an exotic barley accession (<em>Hordeum vulgare</em> ssp. <em>spontaneum</em>) as donor</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">109</div>, <div class="fpage">1736</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20candidate%20introgression%20lines%20using%20an%20exotic%20barley%20accession%20(Hordeum%20vulgare%20ssp.%20spontaneum)%20as%20donor&amp;author=M.%20von%20Korff&amp;author=H.%20Wang&amp;author=J.%20L%C3%A9on&amp;author=K.%20Pillen&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet&amp;volume=109&amp;pages=1736-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1818-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1818-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1818-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15502912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fonceka</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tossim</div> <div class="given-names">H.-A.</div></div>, <div class="name string-name"><div class="surname">Rivallan</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Construction of chromosome segment substitution lines in peanut (<em>Arachis hypogaea</em> L.) using a wild synthetic and QTL mapping for plant morphology</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e48642</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20chromosome%20segment%20substitution%20lines%20in%20peanut%20(Arachis%20hypogaea%20L.)%20using%20a%20wild%20synthetic%20and%20QTL%20mapping%20for%20plant%20morphology&amp;author=D.%20Fonceka&amp;author=H.-A.%20Tossim&amp;author=R.%20Rivallan&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e48642" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0048642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0048642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0048642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23185268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Falke</div> <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Sušić</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Hackauf</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Establishment of introgression libraries in hybrid rye (Secale cereale L.) from an Iranian primitive accession as a new tool for rye breeding and genomics</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">117</div>, <div class="fpage">641</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20introgression%20libraries%20in%20hybrid%20rye%20(Secale%20cereale%20L.)%20from%20an%20Iranian%20primitive%20accession%20as%20a%20new%20tool%20for%20rye%20breeding%20and%20genomics&amp;author=K.C.%20Falke&amp;author=Z.%20Su%C5%A1i%C4%87&amp;author=B.%20Hackauf&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet&amp;volume=117&amp;pages=641-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0808-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0808-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0808-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18535814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jeuken</div> <div class="given-names">M.J.W.</div></div>, <div class="name string-name"><div class="surname">Lindhout</div> <div class="given-names">P.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The development of lettuce backcross inbred lines (BILs) for exploitation of the <em>Lactuca saligna</em> (wild lettuce) germplasm</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">109</div>, <div class="fpage">394</div>–<div class="lpage">401</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%20lettuce%20backcross%20inbred%20lines%20(BILs)%20for%20exploitation%20of%20the%20Lactuca%20saligna%20(wild%20lettuce)%20germplasm&amp;author=M.J.W.%20Jeuken&amp;author=P.%20Lindhout&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet&amp;volume=109&amp;pages=394-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1643-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1643-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1643-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15103409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Xiang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gai</div> <div class="given-names">J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Development of a chromosome segment substitution line population with wild soybean (<em>Glycine soja</em> Sieb. et Zucc.) as donor parent</div>, <div class="source">Euphytica</div> , <div class="volume">189</div>, <div class="fpage">293</div>–<div class="lpage">307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20chromosome%20segment%20substitution%20line%20population%20with%20wild%20soybean%20(Glycine%20soja%20Sieb.%20et%20Zucc.)%20as%20donor%20parent&amp;author=W.%20Wang&amp;author=Q.%20He&amp;author=H.%20Yang&amp;author=S.%20Xiang&amp;author=T.%20Zhao&amp;author=J.%20Gai&amp;publication_year=2012&amp;journal=Euphytica&amp;volume=189&amp;pages=293-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-012-0817-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-012-0817-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-012-0817-7"> </span></div> </div></div></div></div></div><div content-id="dsx043-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Concibido</div> <div class="given-names">V.C.</div></div>, <div class="name string-name"><div class="surname">La Vallee</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">McLaird</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Introgression of a quantitative trait locus for yield from Glycine soja into commercial soybean cultivars</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">106</div>, <div class="fpage">575</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introgression%20of%20a%20quantitative%20trait%20locus%20for%20yield%20from%20Glycine%20soja%20into%20commercial%20soybean%20cultivars&amp;author=V.C.%20Concibido&amp;author=B.%20La%20Vallee&amp;author=P.%20McLaird&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet&amp;volume=106&amp;pages=575-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1071-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1071-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1071-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12595984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.H.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">G.Y.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">J.X.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">De novo assembly of soybean wild relatives for pan-genome analysis of diversity and agronomic traits</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">32</div>, <div class="fpage">1045</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20soybean%20wild%20relatives%20for%20pan-genome%20analysis%20of%20diversity%20and%20agronomic%20traits&amp;author=Y.H.%20Li&amp;author=G.Y.%20Zhou&amp;author=J.X.%20Ma&amp;publication_year=2014&amp;journal=Nat.%20Biotechnol&amp;volume=32&amp;pages=1045-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25218520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kaga</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Evaluation of soybean germplasm conserved in NIAS genebank and development of mini core collections</div>, <div class="source">Breed. Sci</div> ., <div class="volume">61</div>, <div class="fpage">566</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20soybean%20germplasm%20conserved%20in%20NIAS%20genebank%20and%20development%20of%20mini%20core%20collections&amp;author=A.%20Kaga&amp;author=T.%20Shimizu&amp;author=S.%20Watanabe&amp;publication_year=2012&amp;journal=Breed.%20Sci&amp;volume=61&amp;pages=566-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.61.566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.61.566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.61.566"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cook</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Bayless</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Distinct copy number, coding sequence, and locus methylation patterns underlie <em>Rhg1</em>-mediated soybean resistance to soybean cyst nematode</div>, <div class="source">Plant Physiol</div> ., <div class="volume">165</div>, <div class="fpage">630</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20copy%20number%2C%20coding%20sequence%2C%20and%20locus%20methylation%20patterns%20underlie%20Rhg1-mediated%20soybean%20resistance%20to%20soybean%20cyst%20nematode&amp;author=D.E.%20Cook&amp;author=A.M.%20Bayless&amp;author=K.%20Wang&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=165&amp;pages=630-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.114.235952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.235952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.114.235952"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24733883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsubokura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Natural variation in the genes responsible for maturity loci <em>E1</em>, <em>E2</em>, <em>E3</em> and <em>E4</em> in soybean</div>, <div class="source">Ann. Botany</div> , <div class="volume">113</div>, <div class="fpage">429</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20in%20the%20genes%20responsible%20for%20maturity%20loci%20E1%2C%20E2%2C%20E3%20and%20E4%20in%20soybean&amp;author=Y.%20Tsubokura&amp;author=S.%20Watanabe&amp;author=Z.%20Xia&amp;publication_year=2014&amp;journal=Ann.%20Botany&amp;volume=113&amp;pages=429-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mct269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmct269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmct269"> </span></div> </div></div></div></div></div><div content-id="dsx043-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fehr</div> <div class="given-names">W.R.</div></div>, <div class="name string-name"><div class="surname">Caviness</div> <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Burmood</div> <div class="given-names">D.T.</div></div>, <div class="name string-name"><div class="surname">Pennington</div> <div class="given-names">J.S.</div></div></div> <div class="year">1971</div>, 
<div class="article-title">Stage of development descriptions for soybeans, <em>Glycine max</em> (L.) Merrill</div>, <div class="source">Crop Sci</div> ., <div class="volume">11</div>, <div class="fpage">929</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stage%20of%20development%20descriptions%20for%20soybeans%2C%20Glycine%20max%20(L.)%20Merrill&amp;author=W.R.%20Fehr&amp;author=C.E.%20Caviness&amp;author=D.T.%20Burmood&amp;author=J.S.%20Pennington&amp;publication_year=1971&amp;journal=Crop%20Sci&amp;volume=11&amp;pages=929" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1971.0011183X001100060051x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1971.0011183X001100060051x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1971.0011183X001100060051x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurata</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Nagamura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">1994</div>, 
<div class="article-title">A 300 kilobase interval genetic map of rice including 883 expressed sequences</div>, <div class="source">Nat. Genet</div> ., <div class="volume">8</div>, <div class="fpage">365</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20300%20kilobase%20interval%20genetic%20map%20of%20rice%20including%20883%20expressed%20sequences&amp;author=N.%20Kurata&amp;author=Y.%20Nagamura&amp;author=K.%20Yamamoto&amp;publication_year=1994&amp;journal=Nat.%20Genet&amp;volume=8&amp;pages=365-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1294-365" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1294-365" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1294-365"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7894488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khosla</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Augustus</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Brahmachari</div> <div class="given-names">V.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Sex-specific organisation of middle repetitive DNA sequences in the mealybug <em>Planococcus lilacinus</em></div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">3745</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex-specific%20organisation%20of%20middle%20repetitive%20DNA%20sequences%20in%20the%20mealybug%20Planococcus%20lilacinus&amp;author=S.%20Khosla&amp;author=M.%20Augustus&amp;author=V.%20Brahmachari&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=3745-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.18.3745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.18.3745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.18.3745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10471745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nunome</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Minamiyama</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kono</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Namiki</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kojima</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Read2Marker: a data processing tool for microsatellite marker development from a large data set</div>, <div class="source">Biotechnology</div> , <div class="volume">39</div>, <div class="fpage">472</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Read2Marker%3A%20a%20data%20processing%20tool%20for%20microsatellite%20marker%20development%20from%20a%20large%20data%20set&amp;author=H.%20Fukuoka&amp;author=T.%20Nunome&amp;author=Y.%20Minamiyama&amp;author=I.%20Kono&amp;author=N.%20Namiki&amp;author=A.%20Kojima&amp;publication_year=2005&amp;journal=Biotechnology&amp;volume=39&amp;pages=472-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2144/000112025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000112025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2144%2F000112025"> </span></div> </div></div></div></div></div><div content-id="dsx043-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol</div> ., <div class="volume">132</div>, <div class="fpage">365</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andreson</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Reppo</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kaplinski</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Remm</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">GENOMEMASKER package for designing unique genomic PCR primers</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">7</div>, <div class="fpage">172</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GENOMEMASKER%20package%20for%20designing%20unique%20genomic%20PCR%20primers&amp;author=R.%20Andreson&amp;author=E.%20Reppo&amp;author=L.%20Kaplinski&amp;author=M.%20Remm&amp;publication_year=2006&amp;journal=BMC%20Bioinform&amp;volume=7&amp;pages=172" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-172"> </span></div> </div></div></div></div></div><div content-id="dsx043-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hisano</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Characterization of the soybean genome using EST-derived microsatellite markers</div>, <div class="source">DNA Res</div> ., <div class="volume">14</div>, <div class="fpage">271</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20soybean%20genome%20using%20EST-derived%20microsatellite%20markers&amp;author=H.%20Hisano&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;publication_year=2007&amp;journal=DNA%20Res&amp;volume=14&amp;pages=271-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schuelke</div> <div class="given-names">M.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">An economic method for the fluorescent labeling of PCR fragments</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">18</div>, <div class="fpage">233</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20economic%20method%20for%20the%20fluorescent%20labeling%20of%20PCR%20fragments&amp;author=M.%20Schuelke&amp;publication_year=2000&amp;journal=Nat.%20Biotechnol&amp;volume=18&amp;pages=233-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/72708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F72708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F72708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10657137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;author=J.%20Zhang&amp;author=W.%20Miller&amp;author=D.J.%20Lipman&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">I.Y.</div></div>, <div class="name string-name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>, <div class="name string-name"><div class="surname">Matukumalli</div> <div class="given-names">L.K.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis</div>, <div class="source">Genetics</div> , <div class="volume">176</div>, <div class="fpage">685</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20soybean%20transcript%20map%3A%20gene%20distribution%2C%20haplotype%20and%20single-nucleotide%20polymorphism%20analysis&amp;author=I.Y.%20Choi&amp;author=D.L.%20Hyten&amp;author=L.K.%20Matukumalli&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=685-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.070821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.070821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.070821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17339218" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oeth</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">del Mistro</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Marnellos</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">van den Boom</div> <div class="given-names">D.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Qualitative and quantitative genotyping using single base primer extension coupled with matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MassARRAY)</div>, <div class="source">Methods Mol. Biol</div> ., <div class="volume">578</div>, <div class="fpage">307</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Qualitative%20and%20quantitative%20genotyping%20using%20single%20base%20primer%20extension%20coupled%20with%20matrix-assisted%20laser%20desorption%2Fionization%20time-of-flight%20mass%20spectrometry%20(MassARRAY)&amp;author=P.%20Oeth&amp;author=G.%20del%20Mistro&amp;author=G.%20Marnellos&amp;author=T.%20Shi&amp;author=D.%20van%20den%20Boom&amp;publication_year=2009&amp;journal=Methods%20Mol.%20Biol&amp;volume=578&amp;pages=307-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-1-60327-411-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-60327-411-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-1-60327-411-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19768603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div></div> <div class="year">2001</div>, <em>JoinMap<sup>®</sup> Version 4.0: Software for the Calculation of Genetic Linkage Maps</em>. Wageningen: Plant Research International.</div></div></div></div><div content-id="dsx043-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Broman</div> <div class="given-names">K.W.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Churchill</div> <div class="given-names">G.A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">R/qtl: QTL mapping in experimental crosses</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">889</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=R%2Fqtl%3A%20QTL%20mapping%20in%20experimental%20crosses&amp;author=K.W.%20Broman&amp;author=H.%20Wu&amp;author=S.%20Sen&amp;author=G.A.%20Churchill&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=889-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12724300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="collab">R Development Core Team</div>. <div class="year">2008</div>, R: a language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a class="link link-uri" href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a> (1 November 2017, date last accessed).</div></div></div></div><div content-id="dsx043-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chakravarti</div> <div class="given-names">A.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">A graphical representation of genetic and physical maps: the Marey map</div>, <div class="source">Genomics</div> , <div class="volume">11</div>, <div class="fpage">219</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20graphical%20representation%20of%20genetic%20and%20physical%20maps%3A%20the%20Marey%20map&amp;author=A.%20Chakravarti&amp;publication_year=1991&amp;journal=Genomics&amp;volume=11&amp;pages=219-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(91)90123-V" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(91)90123-V" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(91)90123-V"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1765381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rezvoy</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Charif</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gueguen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Marais</div> <div class="given-names">G.A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">MareyMap: an R-based tool with graphical interface for estimating recombination rates</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2188</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MareyMap%3A%20an%20R-based%20tool%20with%20graphical%20interface%20for%20estimating%20recombination%20rates&amp;author=C.%20Rezvoy&amp;author=D.%20Charif&amp;author=L.%20Gueguen&amp;author=G.A.%20Marais&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2188-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jeong</div> <div class="given-names">S.C.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Genetic linkage map of the nucleolus organizer region in the soybean</div>, <div class="source">Genetics</div> , <div class="volume">178</div>, <div class="fpage">605</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20of%20the%20nucleolus%20organizer%20region%20in%20the%20soybean&amp;author=K.%20Yang&amp;author=S.C.%20Jeong&amp;publication_year=2008&amp;journal=Genetics&amp;volume=178&amp;pages=605-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.081620" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.081620" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.081620"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18202401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Findley</div> <div class="given-names">S.D.</div></div>, <div class="name string-name"><div class="surname">Cannon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Varala</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">A Fluorescence in situ hybridization system for karyotyping soybean</div>, <div class="source">Genetics</div> , <div class="volume">185</div>, <div class="fpage">727</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Fluorescence%20in%20situ%20hybridization%20system%20for%20karyotyping%20soybean&amp;author=S.D.%20Findley&amp;author=S.%20Cannon&amp;author=K.%20Varala&amp;publication_year=2010&amp;journal=Genetics&amp;volume=185&amp;pages=727-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.113753" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.113753" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.113753"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20421607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Grant</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tian</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">SoyTEdb: a comprehensive database of transposable elements in the soybean genome</div>, <div class="source">BMC Genom</div> ., <div class="volume">11</div>, <div class="fpage">113</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SoyTEdb%3A%20a%20comprehensive%20database%20of%20transposable%20elements%20in%20the%20soybean%20genome&amp;author=J.%20Du&amp;author=D.%20Grant&amp;author=Z.%20Tian&amp;publication_year=2010&amp;journal=BMC%20Genom&amp;volume=11&amp;pages=113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-113"> </span></div> </div></div></div></div></div><div content-id="dsx043-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Amdor</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Orf</div> <div class="given-names">J.H.</div></div></div>, et al.   <div class="year">1987</div>, 
<div class="article-title">Evaluation of the USDA soybean germplasm collection: maturity groups 000 to IV (PI 273.483 to PI 427.107)</div>, <div class="source">T. Bull. USDA</div> , <div class="volume">1718</div>, <div class="fpage">44</div>–<div class="lpage">123</div>.</div></div></div></div><div content-id="dsx043-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hideshima</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Map-based cloning of the gene associated with the soybean maturity locus <em>E3</em></div>, <div class="source">Genetics</div> , <div class="volume">182</div>, <div class="fpage">1251</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Map-based%20cloning%20of%20the%20gene%20associated%20with%20the%20soybean%20maturity%20locus%20E3&amp;author=S.%20Watanabe&amp;author=R.%20Hideshima&amp;author=Z.%20Xia&amp;publication_year=2009&amp;journal=Genetics&amp;volume=182&amp;pages=1251-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.098772" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.098772" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.098772"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19474204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Hideshima</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A map-based cloning strategy employing a residual heterozygous line reveals that the <em>GIGANTEA</em> gene is involved in soybean maturity and flowering</div>, <div class="source">Genetics</div> , <div class="volume">188</div>, <div class="fpage">395</div>–<div class="lpage">407</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20map-based%20cloning%20strategy%20employing%20a%20residual%20heterozygous%20line%20reveals%20that%20the%20GIGANTEA%20gene%20is%20involved%20in%20soybean%20maturity%20and%20flowering&amp;author=S.%20Watanabe&amp;author=Z.%20Xia&amp;author=R.%20Hideshima&amp;publication_year=2011&amp;journal=Genetics&amp;volume=188&amp;pages=395-407" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.125062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.125062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.125062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21406680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hill</div> <div class="given-names">C.B.</div></div>, <div class="name string-name"><div class="surname">Hartman</div> <div class="given-names">G.L.</div></div>, <div class="name string-name"><div class="surname">Esgar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hobbs</div> <div class="given-names">H.A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Field evaluation of green stem disorder in soybean cultivars</div>, <div class="source">Crop Sci</div> ., <div class="volume">46</div>, <div class="fpage">879</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Field%20evaluation%20of%20green%20stem%20disorder%20in%20soybean%20cultivars&amp;author=C.B.%20Hill&amp;author=G.L.%20Hartman&amp;author=R.%20Esgar&amp;author=H.A.%20Hobbs&amp;publication_year=2006&amp;journal=Crop%20Sci&amp;volume=46&amp;pages=879-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2005.0207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2005.0207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2005.0207"> </span></div> </div></div></div></div></div><div content-id="dsx043-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Takeshima</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A recessive allele for delayed flowering at the soybean maturity locus <em>E9</em> is a leaky allele of <em>FT2a</em>, a <em>FLOWERING LOCUS T</em> ortholog</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20recessive%20allele%20for%20delayed%20flowering%20at%20the%20soybean%20maturity%20locus%20E9%20is%20a%20leaky%20allele%20of%20FT2a%2C%20a%20FLOWERING%20LOCUS%20T%20ortholog&amp;author=C.%20Zhao&amp;author=R.%20Takeshima&amp;author=J.%20Zhu&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-016-0704-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0704-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-016-0704-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26786479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takeshima</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A soybean quantitative trait locus that promotes flowering under long days is identified as <em>FT5a</em>, a <em>FLOWERING LOCUS T</em> ortholog</div>, <div class="source">Exbotj</div> , <div class="volume">67</div>, <div class="fpage">5247</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20soybean%20quantitative%20trait%20locus%20that%20promotes%20flowering%20under%20long%20days%20is%20identified%20as%20FT5a%2C%20a%20FLOWERING%20LOCUS%20T%20ortholog&amp;author=R.%20Takeshima&amp;author=T.%20Hayashi&amp;author=J.%20Zhu&amp;publication_year=2016&amp;journal=Exbotj&amp;volume=67&amp;pages=5247-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erw283" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferw283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferw283"> </span></div> </div></div></div></div></div><div content-id="dsx043-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Uchiyama</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The soybean stem growth habit gene <em>Dt1</em> is an ortholog of Arabidopsis <em>TERMINAL FLOWER1</em></div>, <div class="source">Plant Physiol</div> ., <div class="volume">153</div>, <div class="fpage">198</div>–<div class="lpage">210</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20soybean%20stem%20growth%20habit%20gene%20Dt1%20is%20an%20ortholog%20of%20Arabidopsis%20TERMINAL%20FLOWER1&amp;author=B.%20Liu&amp;author=S.%20Watanabe&amp;author=T.%20Uchiyama&amp;publication_year=2010&amp;journal=Plant%20Physiol&amp;volume=153&amp;pages=198-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.150607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.150607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.150607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20219831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Bernard</div> <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">R.L.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">A third allele at the soybean <em>dt1</em> locus</div>, <div class="source">Crop Sci</div> ., <div class="volume">37</div>, <div class="fpage">757</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20third%20allele%20at%20the%20soybean%20dt1%20locus&amp;author=J.A.%20Thompson&amp;author=R.L.%20Bernard&amp;author=R.L.%20Nelson&amp;publication_year=1997&amp;journal=Crop%20Sci&amp;volume=37&amp;pages=757-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1997.0011183X003700030011x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1997.0011183X003700030011x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1997.0011183X003700030011x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Noe</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Arelli</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Meksem</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">SNP identification and marker assay development for high-throughput selection of soybean cyst nematode resistance</div>, <div class="source">BMC Genom</div> ., <div class="volume">16</div>, <div class="fpage">314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20identification%20and%20marker%20assay%20development%20for%20high-throughput%20selection%20of%20soybean%20cyst%20nematode%20resistance&amp;author=Z.%20Shi&amp;author=S.M.%20Liu&amp;author=J.%20Noe&amp;author=P.%20Arelli&amp;author=K.%20Meksem&amp;author=Z.L.%20Li&amp;publication_year=2015&amp;journal=BMC%20Genom&amp;volume=16&amp;pages=314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1531-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1531-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1531-3"> </span></div> </div></div></div></div></div><div content-id="dsx043-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sayama</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A major and stable QTL associated with seed weight in soybean across multiple environments and genetic backgrounds</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">127</div>, <div class="fpage">1365</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20major%20and%20stable%20QTL%20associated%20with%20seed%20weight%20in%20soybean%20across%20multiple%20environments%20and%20genetic%20backgrounds&amp;author=S.%20Kato&amp;author=T.%20Sayama&amp;author=K.%20Fujii&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet&amp;volume=127&amp;pages=1365-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-014-2304-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-014-2304-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-014-2304-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24718925" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mandl</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Buss</div> <div class="given-names">G.R.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Comparison of narrow and broad leaflet isolines of soybean</div>, <div class="source">Crop Sci</div> ., <div class="volume">21</div>, <div class="fpage">25</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20narrow%20and%20broad%20leaflet%20isolines%20of%20soybean&amp;author=F.A.%20Mandl&amp;author=G.R.%20Buss&amp;publication_year=1981&amp;journal=Crop%20Sci&amp;volume=21&amp;pages=25-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1981.0011183X002100010007x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1981.0011183X002100010007x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1981.0011183X002100010007x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jeong</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Suh</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">M.H.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title"><em>Ln</em> is a hey regulator of leaflet shape and number of seeds per pod in soybean</div>, <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">4807</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ln%20is%20a%20hey%20regulator%20of%20leaflet%20shape%20and%20number%20of%20seeds%20per%20pod%20in%20soybean&amp;author=N.%20Jeong&amp;author=S.J.%20Suh&amp;author=M.H.%20Kim&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=4807-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.104968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.104968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.104968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23243125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ninomiya</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hoshi</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2001</div>, 
<div class="article-title">An informative linkage map of soybean reveals QTLs for flowering time, leaflet morphology and regions of segregation distortion</div>, <div class="source">DNA Res</div> ., <div class="volume">8</div>, <div class="fpage">61</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20informative%20linkage%20map%20of%20soybean%20reveals%20QTLs%20for%20flowering%20time%2C%20leaflet%20morphology%20and%20regions%20of%20segregation%20distortion&amp;author=N.%20Yamanaka&amp;author=S.%20Ninomiya&amp;author=M.%20Hoshi&amp;publication_year=2001&amp;journal=DNA%20Res&amp;volume=8&amp;pages=61-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.2.61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.2.61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.2.61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11347903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xia</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Positional cloning and characterization reveal the molecular basis for soybean maturity locus <em>E1</em> that regulates photoperiodic flowering</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">E2155</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positional%20cloning%20and%20characterization%20reveal%20the%20molecular%20basis%20for%20soybean%20maturity%20locus%20E1%20that%20regulates%20photoperiodic%20flowering&amp;author=Z.%20Xia&amp;author=S.%20Watanabe&amp;author=T.%20Yamada&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=E2155-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1117982109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1117982109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1117982109"> </span></div> </div></div></div></div></div><div content-id="dsx043-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">M.L.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.H.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Genetic variation in four maturity genes affects photoperiod insensitivity and PHYA-regulated post-flowering responses of soybean</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">13</div>, <div class="fpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20four%20maturity%20genes%20affects%20photoperiod%20insensitivity%20and%20PHYA-regulated%20post-flowering%20responses%20of%20soybean&amp;author=M.L.%20Xu&amp;author=Z.H.%20Xu&amp;author=B.H.%20Liu&amp;publication_year=2013&amp;journal=BMC%20Plant%20Biol&amp;volume=13&amp;pages=91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-13-91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-13-91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-13-91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23799885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cober</div> <div class="given-names">E.R.</div></div>, <div class="name string-name"><div class="surname">Voldeng</div> <div class="given-names">H.D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">A new soybean maturity and photoperiod-sensitivity locus linked to <em>E1</em> and <em>T</em></div>, <div class="source">Crop Sci</div> ., <div class="volume">41</div>, <div class="fpage">698</div>–<div class="lpage">701</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20soybean%20maturity%20and%20photoperiod-sensitivity%20locus%20linked%20to%20E1%20and%20T&amp;author=E.R.%20Cober&amp;author=H.D.%20Voldeng&amp;publication_year=2001&amp;journal=Crop%20Sci&amp;volume=41&amp;pages=698-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2001.413698x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2001.413698x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2001.413698x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kuroda</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kaga</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tomooka</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">QTL affecting fitness of hybrids between wild and cultivated soybeans in experimental fields</div>, <div class="source">Ecol. Evol</div> ., <div class="volume">3</div>, <div class="fpage">2150</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20affecting%20fitness%20of%20hybrids%20between%20wild%20and%20cultivated%20soybeans%20in%20experimental%20fields&amp;author=Y.%20Kuroda&amp;author=A.%20Kaga&amp;author=N.%20Tomooka&amp;publication_year=2013&amp;journal=Ecol.%20Evol&amp;volume=3&amp;pages=2150-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ece3.606" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fece3.606" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fece3.606"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23919159" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Komatsu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hwang</div> <div class="given-names">T.Y.</div></div>, <div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Identification of QTL controlling post-flowering period in soybean</div>, <div class="source">Breed. Sci</div> ., <div class="volume">61</div>, <div class="fpage">646</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTL%20controlling%20post-flowering%20period%20in%20soybean&amp;author=K.%20Komatsu&amp;author=T.Y.%20Hwang&amp;author=M.%20Takahashi&amp;publication_year=2012&amp;journal=Breed.%20Sci&amp;volume=61&amp;pages=646-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.61.646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.61.646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.61.646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ross</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Brim</div> <div class="given-names">C.A.</div></div></div> <div class="year">1957</div>, 
<div class="article-title">Resistance of soybeans to the soybean cyst nematode as determined by a double-row method</div>, <div class="source">Plant Dis. Rep</div> ., <div class="volume">41</div>, <div class="fpage">923</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="dsx043-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gao</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bhattacharyya</div> <div class="given-names">M.K.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The soybean-<em>Phytophthora</em> resistance locus <em>Rps1-k</em> encompasses coiled coil-nucleotide binding-leucine rich repeat-like genes and repetitive sequences</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">8</div>, <div class="fpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20soybean-Phytophthora%20resistance%20locus%20Rps1-k%20encompasses%20coiled%20coil-nucleotide%20binding-leucine%20rich%20repeat-like%20genes%20and%20repetitive%20sequences&amp;author=H.%20Gao&amp;author=M.K.%20Bhattacharyya&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol&amp;volume=8&amp;pages=29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18366691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boboye</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Nyakatura</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Rosenthal</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Perret</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Broughton</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Boller</div> <div class="given-names">T.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Sequence analysis of a DNA fragment from <em>Sinorhizobium fredii</em> USDA257 which extends the nitrogen fixation host range of <em>Rhizobium</em> species NGR234 to soybean, <em>Glycine max</em> (L.) Merr cultivar Peking</div>, <div class="source">Symbiosis</div> , <div class="volume">48</div>, <div class="fpage">110</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20a%20DNA%20fragment%20from%20Sinorhizobium%20fredii%20USDA257%20which%20extends%20the%20nitrogen%20fixation%20host%20range%20of%20Rhizobium%20species%20NGR234%20to%20soybean%2C%20Glycine%20max%20(L.)%20Merr%20cultivar%20Peking&amp;author=B.%20Boboye&amp;author=G.%20Nyakatura&amp;author=A.%20Rosenthal&amp;author=X.%20Perret&amp;author=W.J.%20Broughton&amp;author=T.%20Boller&amp;publication_year=2009&amp;journal=Symbiosis&amp;volume=48&amp;pages=110-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF03179990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF03179990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF03179990"> </span></div> </div></div></div></div></div><div content-id="dsx043-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Devine</div> <div class="given-names">T.E.</div></div>, <div class="name string-name"><div class="surname">O'Neill</div> <div class="given-names">J.J.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Genetic allelism of nodulation response genes <em>Rj1</em>, <em>Rj2</em>, and <em>Rj4</em> in soybean</div>, <div class="source">Crop Sci</div> ., <div class="volume">29</div>, <div class="fpage">1347</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20allelism%20of%20nodulation%20response%20genes%20Rj1%2C%20Rj2%2C%20and%20Rj4%20in%20soybean&amp;author=T.E.%20Devine&amp;author=J.J.%20O'Neill&amp;publication_year=1989&amp;journal=Crop%20Sci&amp;volume=29&amp;pages=1347-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1989.0011183X002900060003x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1989.0011183X002900060003x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1989.0011183X002900060003x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">M.Q.</div></div>, <div class="name string-name"><div class="surname">Krishnan</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">H.Y.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">R gene-controlled host specificity in the legume-rhizobia symbiosis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">18735</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=R%20gene-controlled%20host%20specificity%20in%20the%20legume-rhizobia%20symbiosis&amp;author=S.M.%20Yang&amp;author=F.%20Tang&amp;author=M.Q.%20Gao&amp;author=H.B.%20Krishnan&amp;author=H.Y.%20Zhu&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=18735-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1011957107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1011957107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1011957107"> </span></div> </div></div></div></div></div><div content-id="dsx043-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">H.Y.</div></div></div> <div class="year">2016</div>, 
<div class="article-title"><em>Rj4</em>, a gene controlling nodulation specificity in soybeans, encodes a thaumatin-like protein but not the one previously reported</div>, <div class="source">Plant Physiol</div> ., <div class="volume">170</div>, <div class="fpage">26</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rj4%2C%20a%20gene%20controlling%20nodulation%20specificity%20in%20soybeans%2C%20encodes%20a%20thaumatin-like%20protein%20but%20not%20the%20one%20previously%20reported&amp;author=F.%20Tang&amp;author=S.M.%20Yang&amp;author=J.G.%20Liu&amp;author=H.Y.%20Zhu&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=170&amp;pages=26-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.15.01661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.15.01661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.15.01661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26582727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Weaver</div> <div class="given-names">D.B.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">B.L.</div></div>, <div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">J.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Molecular mapping of a new gene for resistance to frogeye leaf spot of soya bean in Peking</div>, <div class="source">Plant Breed</div> ., <div class="volume">120</div>, <div class="fpage">73</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20of%20a%20new%20gene%20for%20resistance%20to%20frogeye%20leaf%20spot%20of%20soya%20bean%20in%20Peking&amp;author=W.%20Yang&amp;author=D.B.%20Weaver&amp;author=B.L.%20Nielsen&amp;author=J.%20Qiu&amp;publication_year=2001&amp;journal=Plant%20Breed&amp;volume=120&amp;pages=73-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1439-0523.2001.00563.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1439-0523.2001.00563.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1439-0523.2001.00563.x"> </span></div> </div></div></div></div></div><div content-id="dsx043-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Joshi</div> <div class="given-names">J.M.</div></div></div> <div class="year">1980</div>, 
<div class="article-title">Research Notes: evaluation of soybean germplasm for resistance to corn earworm–IV</div>, <div class="source">Soybean Genet. Newslett</div> ., <div class="volume">7</div>, <div class="fpage">26</div>.</div></div></div></div><div content-id="dsx043-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sayama</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nakazaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">QTL analysis of seed-flooding tolerance in soybean (<em>Glycine max</em> [L.] Merr.)</div>, <div class="source">Plant Sci</div> ., <div class="volume">176</div>, <div class="fpage">514</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20seed-flooding%20tolerance%20in%20soybean%20(Glycine%20max%20%5BL.%5D%20Merr.)&amp;author=T.%20Sayama&amp;author=T.%20Nakazaki&amp;author=G.%20Ishikawa&amp;publication_year=2009&amp;journal=Plant%20Sci&amp;volume=176&amp;pages=514-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2009.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2009.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2009.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26493141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Boerma</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Parrott</div> <div class="given-names">W.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Genotype effects on proliferative embryogenesis and plant regeneration of soybean</div>, <div class="source">In Vitro Cell. Dev. Biol.–Plant</div> , <div class="volume">29</div>, <div class="fpage">102</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotype%20effects%20on%20proliferative%20embryogenesis%20and%20plant%20regeneration%20of%20soybean&amp;author=M.%20Bailey&amp;author=H.%20Boerma&amp;author=W.%20Parrott&amp;publication_year=1993&amp;journal=In%20Vitro%20Cell.%20Dev.%20Biol.%E2%80%93Plant&amp;volume=29&amp;pages=102-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02632279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02632279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02632279"> </span></div> </div></div></div></div></div><div content-id="dsx043-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Teng</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Identification of QTL underlying somatic embryogenesis capacity of immature embryos in soybean (<em>Glycine max</em> (L.) Merr.)</div>, <div class="source">Plant Cell Rep</div> ., <div class="volume">29</div>, <div class="fpage">125</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTL%20underlying%20somatic%20embryogenesis%20capacity%20of%20immature%20embryos%20in%20soybean%20(Glycine%20max%20(L.)%20Merr.)&amp;author=X.%20Song&amp;author=Y.%20Han&amp;author=W.%20Teng&amp;author=G.%20Sun&amp;author=W.%20Li&amp;publication_year=2010&amp;journal=Plant%20Cell%20Rep&amp;volume=29&amp;pages=125-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-009-0804-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-009-0804-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-009-0804-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19960195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx043-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx043-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mauro</div> <div class="given-names">A.O.</div></div>, <div class="name string-name"><div class="surname">Pfeiffer</div> <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Collins</div> <div class="given-names">G.B.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Inheritance of soybean susceptibility to <em>Agrobacterium tumefaciens</em> and its relationship to transformation</div>, <div class="source">Crop Sci</div> ., <div class="volume">35</div>, <div class="fpage">1152</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20of%20soybean%20susceptibility%20to%20Agrobacterium%20tumefaciens%20and%20its%20relationship%20to%20transformation&amp;author=A.O.%20Mauro&amp;author=T.W.%20Pfeiffer&amp;author=G.B.%20Collins&amp;publication_year=1995&amp;journal=Crop%20Sci&amp;volume=35&amp;pages=1152-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1995.0011183X003500040040x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1995.0011183X003500040040x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1995.0011183X003500040040x"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4657153&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx043/1/dsx043_supp.zip?Expires=2147483647&amp;Signature=A9bIzZDpRO1URlK-7kkb7xM7V303Vnhqu19DFS0EpKqUw3gFmBB~pp~nN8vHVktqJgalboaarQKTVsfWjjnr1Zh5HS5mY6FMzvql-MZv5A8pnfE5Jgb7q20AWyFm46MJbMqmv34ITkTEN1hizjQE0RL49x52e1r69e-Qre0miGa8ZMuzKEkNe8Saug4FvF0jDWpAg74RH8igFn-qZNJa9xrv3puYcEVVaeAgux2bMYChpEkCCJI1Z5GVUEHCGeAmbWWZB3hp8thM~92Q-Po-KvYonyfJ1CgS07Uzl5NL2r34toaobUGEh-KyUZscbEubtegdbHha~77zLraQgmF3Jg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">902</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx043"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4657153"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f123%2f4657153">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F123%2F4657153" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4657153'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx043");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f123%2f4657153">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

