

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/5/343/347949","@type":"ScholarlyArticle","name":"Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing","datePublished":"2015-09-10","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2015-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv017","keywords":["pig","DNA methylation","methylome","epigenetics","RRBS"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/5/343/347949","author":[{"name":"Choi, Minkyeung","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Lee, Jongin","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Le, Min Thong","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Nguyen, Dinh Truong","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Park, Suhyun","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Soundrarajan, Nagasundarapandian","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Schachtschneider, Kyle M.","affiliation":"2  Department of Animal Sciences ,  University of Illinois ,  Urbana, IL ,  USA    3  Animal Breeding and Genomics Center ,  Wageningen University ,  Wageningen ,  The Netherlands","@type":"Person"},{"name":"Kim, Jaebum","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Park, Jin-Ki","affiliation":"4  Animal Biotechnology Division ,  National Institute of Animal Science ,  Suwon ,  Korea","@type":"Person"},{"name":"Kim, Jin-Hoi","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"},{"name":"Park, Chankyu","affiliation":"1  Department of Animal Biotechnology ,  Konkuk University ,  Kwangjin-gu, Seoul 143-701 ,  Korea","@type":"Person"}],"description":"Abstract.  DNA methylation plays a major role in the epigenetic regulation of gene expression. Although a few DNA methylation profiling studies of porcine genom","pageStart":"343","pageEnd":"355","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing" />
    <meta property="og:description" content="Abstract.  DNA methylation plays a major role in the epigenetic regulation of gene expression. Although a few DNA methylation profiling studies of porcine genom" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/5/343/347949" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-09-10" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Choi, Minkyeung" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Lee, Jongin" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Le, Min Thong" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Nguyen, Dinh Truong" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Park, Suhyun" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Soundrarajan, Nagasundarapandian" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Schachtschneider, Kyle M." /><meta name="citation_author_institution" content="Department of Animal Sciences, University of Illinois, Urbana, IL, USA" /><meta name="citation_author_institution" content="Animal Breeding and Genomics Center, Wageningen University, Wageningen, The Netherlands" /><meta name="citation_author" content="Kim, Jaebum" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Park, Jin-Ki" /><meta name="citation_author_institution" content="Animal Biotechnology Division, National Institute of Animal Science, Suwon, Korea" /><meta name="citation_author" content="Kim, Jin-Hoi" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_author" content="Park, Chankyu" /><meta name="citation_author_institution" content="Department of Animal Biotechnology, Konkuk University, Kwangjin-gu, Seoul 143-701, Korea" /><meta name="citation_title" content="Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing" /><meta name="citation_firstpage" content="343" /><meta name="citation_lastpage" content="355" /><meta name="citation_doi" content="10.1093/dnares/dsv017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2015/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=DNA methylation and structural and functional bimodality of vertebrate promoters;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2008;  citation_volume=25;  citation_pages=1602-8; " /><meta name="citation_reference" content="citation_title=Quantitative comparison of genome-wide DNA methylation mapping technologies;  citation_journal_title=Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=1106-14; " /><meta name="citation_reference" content="citation_title=Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome;  citation_journal_title=Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=457-66; " /><meta name="citation_reference" content="citation_title=DNA methylation landscapes: provocative insights from epigenomics;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2008;  citation_volume=9;  citation_pages=465-76; " /><meta name="citation_reference" content="citation_title=X-inactivation profile reveals extensive variability in X-linked gene expression in females;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=434;  citation_pages=400-4; " /><meta name="citation_reference" content="citation_title=DNA hypomethylation in cancer cells;  citation_journal_title=Epigenomics;  citation_year=2009;  citation_volume=1;  citation_pages=239-59; " /><meta name="citation_reference" content="citation_title=Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer;  citation_journal_title=Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=246-58; " /><meta name="citation_reference" content="citation_title=Epigenetic reprogramming in mammals;  citation_journal_title=Hum. Mol. Genet.;  citation_year=2005;  citation_volume=14;  citation_pages=R47-58; " /><meta name="citation_reference" content="citation_title=Epigenetic events in mammalian germ-cell development: reprogramming and beyond;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2008;  citation_volume=9;  citation_pages=129-40; " /><meta name="citation_reference" content="citation_title=Quantitative analysis of human tissue-specific differences in methylation;  citation_journal_title=Biochem. Biophys. Res. Commun.;  citation_year=2008;  citation_volume=376;  citation_pages=658-64; " /><meta name="citation_reference" content="citation_title=Genome-wide survey reveals dynamic widespread tissue-specific changes in DNA methylation during development;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=231" /><meta name="citation_reference" content="citation_title=Genomic distribution and inter-sample variation of non-CpG methylation across human cell types;  citation_journal_title=PLoS Genet.;  citation_year=2011;  citation_volume=7;  citation_pages=e1002389" /><meta name="citation_reference" content="citation_title=Tissue specific DNA methylation of CpG islands in normal human adult somatic tissues distinguishes neural from non-neural tissues;  citation_journal_title=Epigenetics;  citation_year=2010;  citation_volume=5;  citation_pages=527-38; " /><meta name="citation_reference" content="citation_title=DNA methylation dynamics during in vivo differentiation of blood and skin stem cells;  citation_journal_title=Mol. Cell;  citation_year=2012;  citation_volume=47;  citation_pages=633-47; " /><meta name="citation_reference" content="citation_title=DNA methylome profiling of human tissues identifies global and tissue-specific methylation patterns;  citation_journal_title=Genome Biol.;  citation_year=2014;  citation_volume=15;  citation_pages=r54" /><meta name="citation_reference" content="citation_title=Dynamic DNA methylation across diverse human cell lines and tissues;  citation_journal_title=Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=555-67; " /><meta name="citation_reference" content="citation_title=Immortality, but not oncogenic transformation, of primary human cells leads to epigenetic reprogramming of DNA methylation and gene expression;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=3529-41; " /><meta name="citation_reference" content="citation_title=CpG islands in vertebrate genomes;  citation_journal_title=J. Mol. Biol.;  citation_year=1987;  citation_volume=196;  citation_pages=261-82; " /><meta name="citation_reference" content="citation_title=Genome-scale DNA methylation maps of pluripotent and differentiated cells;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=454;  citation_pages=766-70; " /><meta name="citation_reference" content="citation_title=Diametrically opposite methylome-transcriptome relationships in high- and low-CpG promoter genes in postmitotic neural rat tissue;  citation_journal_title=Epigenetics;  citation_year=2012;  citation_volume=7;  citation_pages=421-8; " /><meta name="citation_reference" content="citation_title=Epigenetics: a landscape takes shape;  citation_journal_title=Cell;  citation_year=2007;  citation_volume=128;  citation_pages=635-8; " /><meta name="citation_reference" content="citation_title=Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5868-77; " /><meta name="citation_reference" content="citation_title=A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis;  citation_journal_title=Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=779-85; " /><meta name="citation_reference" content="citation_title=Purification of CpG islands using a methylated DNA binding column;  citation_journal_title=Nat. Genet.;  citation_year=1994;  citation_volume=6;  citation_pages=236-44; " /><meta name="citation_reference" content="citation_title=Whole-genome DNA methylation profiling using MethylCap-seq;  citation_journal_title=Methods;  citation_year=2010;  citation_volume=52;  citation_pages=232-6; " /><meta name="citation_reference" content="citation_title=MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=391-9; " /><meta name="citation_reference" content="citation_title=Evaluation of the Infinium Methylation 450K technology;  citation_journal_title=Epigenomics;  citation_year=2011;  citation_volume=3;  citation_pages=771-84; " /><meta name="citation_reference" content="citation_title=Preparation of reduced representation bisulfite sequencing libraries for genome-scale DNA methylation profiling;  citation_journal_title=Nat. Protoc.;  citation_year=2011;  citation_volume=6;  citation_pages=468-81; " /><meta name="citation_reference" content="citation_title=High-throughput bisulfite sequencing in mammalian genomes;  citation_journal_title=Methods;  citation_year=2009;  citation_volume=48;  citation_pages=226-32; " /><meta name="citation_reference" content="citation_title=DNA methylation and human disease;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=597-610; " /><meta name="citation_reference" content="citation_title=Diet, methyl donors and DNA methylation: interactions between dietary folate, methionine and choline;  citation_journal_title=J. Nutr.;  citation_year=2002;  citation_volume=132;  citation_pages=2333s-5s; " /><meta name="citation_reference" content="citation_title=Maternal dietary protein affects transcriptional regulation of myostatin gene distinctively at weaning and finishing stages in skeletal muscle of Meishan pigs;  citation_journal_title=Epigenetics;  citation_year=2011;  citation_volume=6;  citation_pages=899-907; " /><meta name="citation_reference" content="citation_title=Maternal dietary protein restriction and excess affects offspring gene expression and methylation of non-SMC subunits of condensin I in liver and skeletal muscle;  citation_journal_title=Epigenetics;  citation_year=2012;  citation_volume=7;  citation_pages=239-52; " /><meta name="citation_reference" content="citation_title=Genome-wide DNA methylation profiles of small intestine and liver in fast-growing and slow-growing weaning piglets;  citation_journal_title=Asian Australas. J. Anim. Sci.;  citation_year=2014;  citation_volume=27;  citation_pages=1532-9; " /><meta name="citation_reference" content="citation_title=DNA methylation biomarkers for lung cancer;  citation_journal_title=Tumour Biol.;  citation_year=2012;  citation_volume=33;  citation_pages=287-96; " /><meta name="citation_reference" content="citation_title=Linkage of DNA methylation quantitative trait loci to human cancer risk;  citation_journal_title=Cell Rep.;  citation_year=2014;  citation_volume=7;  citation_pages=331-8; " /><meta name="citation_reference" content="citation_title=Integrative genome-wide expression and promoter DNA methylation profiling identifies a potential novel panel of ovarian cancer epigenetic biomarkers;  citation_journal_title=Cancer Lett.;  citation_year=2012;  citation_volume=318;  citation_pages=76-85; " /><meta name="citation_reference" content="citation_title=DNA methylation profiling reveals novel biomarkers and important roles for DNA methyltransferases in prostate cancer;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=1017-27; " /><meta name="citation_reference" content="citation_title=Alzheimer's disease: early alterations in brain DNA methylation at ANK1, BIN1, RHBDF2 and other loci;  citation_journal_title=Nat. Neurosci.;  citation_year=2014;  citation_volume=17;  citation_pages=1156-63; " /><meta name="citation_reference" content="citation_title=Genome-wide DNA methylation patterns and transcription analysis in sheep muscle;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e101853" /><meta name="citation_reference" content="citation_title=Genome-wide analysis of DNA methylation in bovine placentas;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=12" /><meta name="citation_reference" content="citation_title=Genome-wide DNA methylation profiles and their relationships with mRNA and the microRNA transcriptome in bovine muscle tissue (Bos taurine);  citation_journal_title=Sci Rep-Uk;  citation_year=2014;  citation_volume=4;  citation_pages=1-17; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of DNA methylation patterns in horse;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=598" /><meta name="citation_reference" content="citation_title=Marked methylation changes in intestinal genes during the perinatal period of preterm neonates;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=716" /><meta name="citation_reference" content="Tumbleson S.  1997 , Advances in swine in biomedical research. Edited by M.E. Tumbleson and L.B. Schook. Plenum Publishing Corporation, 1996. Volume 1, pp. 462. Volume 2, pp. 499 , Exp. Physiol. , 82 , 803 – 804. " /><meta name="citation_reference" content="citation_title=Analyses of pig genomes provide insight into porcine demography and evolution;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=393-8; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of copy number variation in pig genomes in the context of adaptation and domestication;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=449" /><meta name="citation_reference" content="citation_title=The complete swine olfactory subgenome: expansion of the olfactory gene repertoire in the pig genome;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=584" /><meta name="citation_reference" content="citation_title=Strong signatures of selection in the domestic pig genome;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=19529-36; " /><meta name="citation_reference" content="citation_title=An atlas of DNA methylomes in porcine adipose and muscle tissues;  citation_journal_title=Nat. Commun.;  citation_year=2012;  citation_volume=3;  citation_pages=850" /><meta name="citation_reference" content="citation_title=BS-Seeker2: a versatile aligning pipeline for bisulfite sequencing data;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=774" /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles;  citation_journal_title=Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=R87" /><meta name="citation_reference" content="citation_title=jvenn: an interactive Venn diagram viewer;  citation_journal_title=BMC Bioinformatics;  citation_year=2014;  citation_volume=15;  citation_pages=293" /><meta name="citation_reference" content="citation_title=Microarray expression profiles of 20.000 genes across 23 healthy porcine tissues;  citation_journal_title=PLoS ONE;  citation_year=2007;  citation_volume=2;  citation_pages=e1203" /><meta name="citation_reference" content="citation_publisher=R Foundation for Statistical Computing: Vienna, Austria, ;  citation_title=R: a language and environment for statistical computing;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Transcriptional regulation and function during the human cell cycle;  citation_journal_title=Nat. Genet.;  citation_year=2001;  citation_volume=27;  citation_pages=48-54; " /><meta name="citation_reference" content="citation_title=Double restriction-enzyme digestion improves the coverage and accuracy of genome-wide CpG methylation profiling by reduced representation bisulfite sequencing;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Mapping the zebrafish brain methylome using reduced representation bisulfite sequencing;  citation_journal_title=Epigenetics;  citation_year=2013;  citation_volume=8;  citation_pages=979-89; " /><meta name="citation_reference" content="citation_title=Principles and challenges of genomewide DNA methylation analysis;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=191-203; " /><meta name="citation_reference" content="citation_title=A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=1412-7; " /><meta name="citation_reference" content="citation_title=Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications;  citation_journal_title=Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=1097-105; " /><meta name="citation_reference" content="citation_title=Measuring the methylome in clinical samples: improved processing of the Infinium Human Methylation450 BeadChip Array;  citation_journal_title=Epigenetics.;  citation_year=2012;  citation_volume=7;  citation_pages=1173-87; " /><meta name="citation_reference" content="citation_title=Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain;  citation_journal_title=Nat. Neurosci.;  citation_year=2014;  citation_volume=17;  citation_pages=215-22; " /><meta name="citation_reference" content="citation_title=Dynamic CpG island methylation landscape in oocytes and preimplantation embryos;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=811-4; " /><meta name="citation_reference" content="citation_title=Activation of nuclear receptor NR5A2 increases Glut4 expression and glucose metabolism in muscle cells;  citation_journal_title=Biochem. Biophys. Res. Commun.;  citation_year=2014;  citation_volume=446;  citation_pages=614-9; " /><meta name="citation_reference" content="citation_title=Genome-wide DNA methylation changes between the superficial and deep backfat tissues of the pig;  citation_journal_title=Int. J. Mol. Sci.;  citation_year=2012;  citation_volume=13;  citation_pages=7098-108; " /><meta name="citation_reference" content="citation_title=CpG islands as gene markers in the human genome;  citation_journal_title=Genomics;  citation_year=1992;  citation_volume=13;  citation_pages=1095-107; " /><meta name="citation_reference" content="citation_title=Differential methylation of tissue- and cancer-specific CpG island shores distinguishes human induced pluripotent stem cells, embryonic stem cells and fibroblasts;  citation_journal_title=Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=1350-3; " /><meta name="citation_reference" content="citation_title=The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores;  citation_journal_title=Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=178-86; " /><meta name="citation_reference" content="citation_title=A methylation profile of in vitro immortalized human cell lines;  citation_journal_title=Int. J. Oncol.;  citation_year=2005;  citation_volume=26;  citation_pages=275-85; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/5/343/997057/dsv017.pdf" /><meta name="description" content="Abstract.  DNA methylation plays a major role in the epigenetic regulation of gene expression. Although a few DNA methylation profiling studies of porcine genom" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/5/343/347949" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/5/343/347949" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/5/343/347949");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv017';
    </script>
<title>
	Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347949" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYqnATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvaSwgTWlua3lldW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgQmlvdGVjaG5vbG9neSwgS29ua3VrIFVuaXZlcnNpdHksIEt3YW5namluLWd1LCBTZW91bCAxNDMtNzAxLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlZSwgSm9uZ2luIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgQmlvdGVjaG5vbG9neSwgS29ua3VrIFVuaXZlcnNpdHksIEt3YW5namluLWd1LCBTZW91bCAxNDMtNzAxLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlLCBNaW4gVGhvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBCaW90ZWNobm9sb2d5LCBLb25rdWsgVW5pdmVyc2l0eSwgS3dhbmdqaW4tZ3UsIFNlb3VsIDE0My03MDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmd1eWVuLCBEaW5oIFRydW9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEJpb3RlY2hub2xvZ3ksIEtvbmt1ayBVbml2ZXJzaXR5LCBLd2FuZ2ppbi1ndSwgU2VvdWwgMTQzLTcwMSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJrLCBTdWh5dW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBCaW90ZWNobm9sb2d5LCBLb25rdWsgVW5pdmVyc2l0eSwgS3dhbmdqaW4tZ3UsIFNlb3VsIDE0My03MDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU291bmRyYXJhamFuLCBOYWdhc3VuZGFyYXBhbmRpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBCaW90ZWNobm9sb2d5LCBLb25rdWsgVW5pdmVyc2l0eSwgS3dhbmdqaW4tZ3UsIFNlb3VsIDE0My03MDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2NoYWNodHNjaG5laWRlciwgS3lsZSBNLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIElsbGlub2lzLCBVcmJhbmEsIElMLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbmltYWwgQnJlZWRpbmcgYW5kIEdlbm9taWNzIENlbnRlciwgV2FnZW5pbmdlbiBVbml2ZXJzaXR5LCBXYWdlbmluZ2VuLCBUaGUgTmV0aGVybGFuZHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIEphZWJ1bSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEJpb3RlY2hub2xvZ3ksIEtvbmt1ayBVbml2ZXJzaXR5LCBLd2FuZ2ppbi1ndSwgU2VvdWwgMTQzLTcwMSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJrLCBKaW4tS2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbmltYWwgQmlvdGVjaG5vbG9neSBEaXZpc2lvbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFuaW1hbCBTY2llbmNlLCBTdXdvbiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIEppbi1Ib2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBCaW90ZWNobm9sb2d5LCBLb25rdWsgVW5pdmVyc2l0eSwgS3dhbmdqaW4tZ3UsIFNlb3VsIDE0My03MDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyaywgQ2hhbmt5dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEJpb3RlY2hub2xvZ3ksIEtvbmt1ayBVbml2ZXJzaXR5LCBLd2FuZ2ppbi1ndSwgU2VvdWwgMTQzLTcwMSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIEROQSBtZXRoeWxhdGlvbiBpbiBwaWdzIHVzaW5nIHJlZHVjZWQgcmVwcmVzZW50YXRpb24gYmlzdWxmaXRlIHNlcXVlbmNpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdjAxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIGFuZCBzdHJ1Y3R1cmFsIGFuZCBmdW5jdGlvbmFsIGJpbW9kYWxpdHkgb2YgdmVydGVicmF0ZSBwcm9tb3RlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNjAyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSBjb21wYXJpc29uIG9mIGdlbm9tZS13aWRlIEROQSBtZXRoeWxhdGlvbiBtYXBwaW5nIHRlY2hub2xvZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTExMDYtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RyaWJ1dGlvbiwgc2lsZW5jaW5nIHBvdGVudGlhbCBhbmQgZXZvbHV0aW9uYXJ5IGltcGFjdCBvZiBwcm9tb3RlciBETkEgbWV0aHlsYXRpb24gaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz00NTctNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBsYW5kc2NhcGVzOiBwcm92b2NhdGl2ZSBpbnNpZ2h0cyBmcm9tIGVwaWdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00NjUtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVgtaW5hY3RpdmF0aW9uIHByb2ZpbGUgcmV2ZWFscyBleHRlbnNpdmUgdmFyaWFiaWxpdHkgaW4gWC1saW5rZWQgZ2VuZSBleHByZXNzaW9uIGluIGZlbWFsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzQ7ICBjaXRhdGlvbl9wYWdlcz00MDAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGh5cG9tZXRoeWxhdGlvbiBpbiBjYW5jZXIgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVwaWdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yMzktNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBETkEgaHlwb21ldGh5bGF0aW9uIGNvdXBsZWQgdG8gcmVwcmVzc2l2ZSBjaHJvbWF0aW4gZG9tYWluIGZvcm1hdGlvbiBhbmQgZ2VuZSBzaWxlbmNpbmcgaW4gYnJlYXN0IGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yNDYtNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgcmVwcm9ncmFtbWluZyBpbiBtYW1tYWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjQ3LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIGV2ZW50cyBpbiBtYW1tYWxpYW4gZ2VybS1jZWxsIGRldmVsb3BtZW50OiByZXByb2dyYW1taW5nIGFuZCBiZXlvbmQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEyOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIGFuYWx5c2lzIG9mIGh1bWFuIHRpc3N1ZS1zcGVjaWZpYyBkaWZmZXJlbmNlcyBpbiBtZXRoeWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM3NjsgIGNpdGF0aW9uX3BhZ2VzPTY1OC02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgc3VydmV5IHJldmVhbHMgZHluYW1pYyB3aWRlc3ByZWFkIHRpc3N1ZS1zcGVjaWZpYyBjaGFuZ2VzIGluIEROQSBtZXRoeWxhdGlvbiBkdXJpbmcgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTIzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgZGlzdHJpYnV0aW9uIGFuZCBpbnRlci1zYW1wbGUgdmFyaWF0aW9uIG9mIG5vbi1DcEcgbWV0aHlsYXRpb24gYWNyb3NzIGh1bWFuIGNlbGwgdHlwZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjM4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRpc3N1ZSBzcGVjaWZpYyBETkEgbWV0aHlsYXRpb24gb2YgQ3BHIGlzbGFuZHMgaW4gbm9ybWFsIGh1bWFuIGFkdWx0IHNvbWF0aWMgdGlzc3VlcyBkaXN0aW5ndWlzaGVzIG5ldXJhbCBmcm9tIG5vbi1uZXVyYWwgdGlzc3VlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXBpZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTUyNy0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIGR5bmFtaWNzIGR1cmluZyBpbiB2aXZvIGRpZmZlcmVudGlhdGlvbiBvZiBibG9vZCBhbmQgc2tpbiBzdGVtIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz02MzMtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxvbWUgcHJvZmlsaW5nIG9mIGh1bWFuIHRpc3N1ZXMgaWRlbnRpZmllcyBnbG9iYWwgYW5kIHRpc3N1ZS1zcGVjaWZpYyBtZXRoeWxhdGlvbiBwYXR0ZXJuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9cjU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyBETkEgbWV0aHlsYXRpb24gYWNyb3NzIGRpdmVyc2UgaHVtYW4gY2VsbCBsaW5lcyBhbmQgdGlzc3VlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz01NTUtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltbW9ydGFsaXR5LCBidXQgbm90IG9uY29nZW5pYyB0cmFuc2Zvcm1hdGlvbiwgb2YgcHJpbWFyeSBodW1hbiBjZWxscyBsZWFkcyB0byBlcGlnZW5ldGljIHJlcHJvZ3JhbW1pbmcgb2YgRE5BIG1ldGh5bGF0aW9uIGFuZCBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTM1MjktNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNwRyBpc2xhbmRzIGluIHZlcnRlYnJhdGUgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTY7ICBjaXRhdGlvbl9wYWdlcz0yNjEtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS1zY2FsZSBETkEgbWV0aHlsYXRpb24gbWFwcyBvZiBwbHVyaXBvdGVudCBhbmQgZGlmZmVyZW50aWF0ZWQgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTQ7ICBjaXRhdGlvbl9wYWdlcz03NjYtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpYW1ldHJpY2FsbHkgb3Bwb3NpdGUgbWV0aHlsb21lLXRyYW5zY3JpcHRvbWUgcmVsYXRpb25zaGlwcyBpbiBoaWdoLSBhbmQgbG93LUNwRyBwcm9tb3RlciBnZW5lcyBpbiBwb3N0bWl0b3RpYyBuZXVyYWwgcmF0IHRpc3N1ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXBpZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQyMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljczogYSBsYW5kc2NhcGUgdGFrZXMgc2hhcGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI4OyAgY2l0YXRpb25fcGFnZXM9NjM1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZHVjZWQgcmVwcmVzZW50YXRpb24gYmlzdWxmaXRlIHNlcXVlbmNpbmcgZm9yIGNvbXBhcmF0aXZlIGhpZ2gtcmVzb2x1dGlvbiBETkEgbWV0aHlsYXRpb24gYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NjgtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgQmF5ZXNpYW4gZGVjb252b2x1dGlvbiBzdHJhdGVneSBmb3IgaW1tdW5vcHJlY2lwaXRhdGlvbi1iYXNlZCBETkEgbWV0aHlsb21lIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Nzc5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QdXJpZmljYXRpb24gb2YgQ3BHIGlzbGFuZHMgdXNpbmcgYSBtZXRoeWxhdGVkIEROQSBiaW5kaW5nIGNvbHVtbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIzNi00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIEROQSBtZXRoeWxhdGlvbiBwcm9maWxpbmcgdXNpbmcgTWV0aHlsQ2FwLXNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTIzMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQkQtaXNvbGF0ZWQgR2Vub21lIFNlcXVlbmNpbmcgcHJvdmlkZXMgYSBoaWdoLXRocm91Z2hwdXQgYW5kIGNvbXByZWhlbnNpdmUgc3VydmV5IG9mIEROQSBtZXRoeWxhdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0zOTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiB0aGUgSW5maW5pdW0gTWV0aHlsYXRpb24gNDUwSyB0ZWNobm9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FcGlnZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NzcxLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVwYXJhdGlvbiBvZiByZWR1Y2VkIHJlcHJlc2VudGF0aW9uIGJpc3VsZml0ZSBzZXF1ZW5jaW5nIGxpYnJhcmllcyBmb3IgZ2Vub21lLXNjYWxlIEROQSBtZXRoeWxhdGlvbiBwcm9maWxpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUHJvdG9jLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDY4LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgYmlzdWxmaXRlIHNlcXVlbmNpbmcgaW4gbWFtbWFsaWFuIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0yMjYtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBhbmQgaHVtYW4gZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NTk3LTYxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlldCwgbWV0aHlsIGRvbm9ycyBhbmQgRE5BIG1ldGh5bGF0aW9uOiBpbnRlcmFjdGlvbnMgYmV0d2VlbiBkaWV0YXJ5IGZvbGF0ZSwgbWV0aGlvbmluZSBhbmQgY2hvbGluZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTnV0ci47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MjMzM3MtNXM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hdGVybmFsIGRpZXRhcnkgcHJvdGVpbiBhZmZlY3RzIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIG9mIG15b3N0YXRpbiBnZW5lIGRpc3RpbmN0aXZlbHkgYXQgd2VhbmluZyBhbmQgZmluaXNoaW5nIHN0YWdlcyBpbiBza2VsZXRhbCBtdXNjbGUgb2YgTWVpc2hhbiBwaWdzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FcGlnZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODk5LTkwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF0ZXJuYWwgZGlldGFyeSBwcm90ZWluIHJlc3RyaWN0aW9uIGFuZCBleGNlc3MgYWZmZWN0cyBvZmZzcHJpbmcgZ2VuZSBleHByZXNzaW9uIGFuZCBtZXRoeWxhdGlvbiBvZiBub24tU01DIHN1YnVuaXRzIG9mIGNvbmRlbnNpbiBJIGluIGxpdmVyIGFuZCBza2VsZXRhbCBtdXNjbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVwaWdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMzktNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIEROQSBtZXRoeWxhdGlvbiBwcm9maWxlcyBvZiBzbWFsbCBpbnRlc3RpbmUgYW5kIGxpdmVyIGluIGZhc3QtZ3Jvd2luZyBhbmQgc2xvdy1ncm93aW5nIHdlYW5pbmcgcGlnbGV0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXNpYW4gQXVzdHJhbGFzLiBKLiBBbmltLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTUzMi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gYmlvbWFya2VycyBmb3IgbHVuZyBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVR1bW91ciBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBvZiBETkEgbWV0aHlsYXRpb24gcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgdG8gaHVtYW4gY2FuY2VyIHJpc2s7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzMxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aXZlIGdlbm9tZS13aWRlIGV4cHJlc3Npb24gYW5kIHByb21vdGVyIEROQSBtZXRoeWxhdGlvbiBwcm9maWxpbmcgaWRlbnRpZmllcyBhIHBvdGVudGlhbCBub3ZlbCBwYW5lbCBvZiBvdmFyaWFuIGNhbmNlciBlcGlnZW5ldGljIGJpb21hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbmNlciBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTg7ICBjaXRhdGlvbl9wYWdlcz03Ni04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIHByb2ZpbGluZyByZXZlYWxzIG5vdmVsIGJpb21hcmtlcnMgYW5kIGltcG9ydGFudCByb2xlcyBmb3IgRE5BIG1ldGh5bHRyYW5zZmVyYXNlcyBpbiBwcm9zdGF0ZSBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTAxNy0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx6aGVpbWVyJ3MgZGlzZWFzZTogZWFybHkgYWx0ZXJhdGlvbnMgaW4gYnJhaW4gRE5BIG1ldGh5bGF0aW9uIGF0IEFOSzEsIEJJTjEsIFJIQkRGMiBhbmQgb3RoZXIgbG9jaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBOZXVyb3NjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMTU2LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBETkEgbWV0aHlsYXRpb24gcGF0dGVybnMgYW5kIHRyYW5zY3JpcHRpb24gYW5hbHlzaXMgaW4gc2hlZXAgbXVzY2xlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMTg1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIEROQSBtZXRoeWxhdGlvbiBpbiBib3ZpbmUgcGxhY2VudGFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIEROQSBtZXRoeWxhdGlvbiBwcm9maWxlcyBhbmQgdGhlaXIgcmVsYXRpb25zaGlwcyB3aXRoIG1STkEgYW5kIHRoZSBtaWNyb1JOQSB0cmFuc2NyaXB0b21lIGluIGJvdmluZSBtdXNjbGUgdGlzc3VlIChCb3MgdGF1cmluZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaSBSZXAtVWs7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIEROQSBtZXRoeWxhdGlvbiBwYXR0ZXJucyBpbiBob3JzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTk4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFya2VkIG1ldGh5bGF0aW9uIGNoYW5nZXMgaW4gaW50ZXN0aW5hbCBnZW5lcyBkdXJpbmcgdGhlIHBlcmluYXRhbCBwZXJpb2Qgb2YgcHJldGVybSBuZW9uYXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NzE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVHVtYmxlc29uIFMuICAxOTk3ICwgQWR2YW5jZXMgaW4gc3dpbmUgaW4gYmlvbWVkaWNhbCByZXNlYXJjaC4gRWRpdGVkIGJ5IE0uRS4gVHVtYmxlc29uIGFuZCBMLkIuIFNjaG9vay4gUGxlbnVtIFB1Ymxpc2hpbmcgQ29ycG9yYXRpb24sIDE5OTYuIFZvbHVtZSAxLCBwcC4gNDYyLiBWb2x1bWUgMiwgcHAuIDQ5OSAsIEV4cC4gUGh5c2lvbC4gLCA4MiAsIDgwMyDigJMgODA0LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNlcyBvZiBwaWcgZ2Vub21lcyBwcm92aWRlIGluc2lnaHQgaW50byBwb3JjaW5lIGRlbW9ncmFwaHkgYW5kIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MTsgIGNpdGF0aW9uX3BhZ2VzPTM5My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgY29weSBudW1iZXIgdmFyaWF0aW9uIGluIHBpZyBnZW5vbWVzIGluIHRoZSBjb250ZXh0IG9mIGFkYXB0YXRpb24gYW5kIGRvbWVzdGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBzd2luZSBvbGZhY3Rvcnkgc3ViZ2Vub21lOiBleHBhbnNpb24gb2YgdGhlIG9sZmFjdG9yeSBnZW5lIHJlcGVydG9pcmUgaW4gdGhlIHBpZyBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cm9uZyBzaWduYXR1cmVzIG9mIHNlbGVjdGlvbiBpbiB0aGUgZG9tZXN0aWMgcGlnIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTk1MjktMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGF0bGFzIG9mIEROQSBtZXRoeWxvbWVzIGluIHBvcmNpbmUgYWRpcG9zZSBhbmQgbXVzY2xlIHRpc3N1ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ODUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QlMtU2Vla2VyMjogYSB2ZXJzYXRpbGUgYWxpZ25pbmcgcGlwZWxpbmUgZm9yIGJpc3VsZml0ZSBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTc3NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1ldGh5bEtpdDogYSBjb21wcmVoZW5zaXZlIFIgcGFja2FnZSBmb3IgdGhlIGFuYWx5c2lzIG9mIGdlbm9tZS13aWRlIEROQSBtZXRoeWxhdGlvbiBwcm9maWxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9Ujg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9anZlbm46IGFuIGludGVyYWN0aXZlIFZlbm4gZGlhZ3JhbSB2aWV3ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYXJyYXkgZXhwcmVzc2lvbiBwcm9maWxlcyBvZiAyMC4wMDAgZ2VuZXMgYWNyb3NzIDIzIGhlYWx0aHkgcG9yY2luZSB0aXNzdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTEyMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9UiBGb3VuZGF0aW9uIGZvciBTdGF0aXN0aWNhbCBDb21wdXRpbmc6IFZpZW5uYSwgQXVzdHJpYSwgOyAgY2l0YXRpb25fdGl0bGU9UjogYSBsYW5ndWFnZSBhbmQgZW52aXJvbm1lbnQgZm9yIHN0YXRpc3RpY2FsIGNvbXB1dGluZzsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gYW5kIGZ1bmN0aW9uIGR1cmluZyB0aGUgaHVtYW4gY2VsbCBjeWNsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz00OC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG91YmxlIHJlc3RyaWN0aW9uLWVuenltZSBkaWdlc3Rpb24gaW1wcm92ZXMgdGhlIGNvdmVyYWdlIGFuZCBhY2N1cmFjeSBvZiBnZW5vbWUtd2lkZSBDcEcgbWV0aHlsYXRpb24gcHJvZmlsaW5nIGJ5IHJlZHVjZWQgcmVwcmVzZW50YXRpb24gYmlzdWxmaXRlIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyB0aGUgemVicmFmaXNoIGJyYWluIG1ldGh5bG9tZSB1c2luZyByZWR1Y2VkIHJlcHJlc2VudGF0aW9uIGJpc3VsZml0ZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FcGlnZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTc5LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmluY2lwbGVzIGFuZCBjaGFsbGVuZ2VzIG9mIGdlbm9tZXdpZGUgRE5BIG1ldGh5bGF0aW9uIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTkxLTIwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBDcEcgZGludWNsZW90aWRlcyBpbiB0aGUgaHVtYW4gZ2Vub21lIGRpc3Rpbmd1aXNoZXMgdHdvIGRpc3RpbmN0IGNsYXNzZXMgb2YgcHJvbW90ZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0xNDEyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2Ygc2VxdWVuY2luZy1iYXNlZCBtZXRob2RzIHRvIHByb2ZpbGUgRE5BIG1ldGh5bGF0aW9uIGFuZCBpZGVudGlmaWNhdGlvbiBvZiBtb25vYWxsZWxpYyBlcGlnZW5ldGljIG1vZGlmaWNhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xMDk3LTEwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVhc3VyaW5nIHRoZSBtZXRoeWxvbWUgaW4gY2xpbmljYWwgc2FtcGxlczogaW1wcm92ZWQgcHJvY2Vzc2luZyBvZiB0aGUgSW5maW5pdW0gSHVtYW4gTWV0aHlsYXRpb240NTAgQmVhZENoaXAgQXJyYXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVwaWdlbmV0aWNzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTE3My04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdHJpYnV0aW9uLCByZWNvZ25pdGlvbiBhbmQgcmVndWxhdGlvbiBvZiBub24tQ3BHIG1ldGh5bGF0aW9uIGluIHRoZSBhZHVsdCBtYW1tYWxpYW4gYnJhaW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTmV1cm9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjE1LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIENwRyBpc2xhbmQgbWV0aHlsYXRpb24gbGFuZHNjYXBlIGluIG9vY3l0ZXMgYW5kIHByZWltcGxhbnRhdGlvbiBlbWJyeW9zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTgxMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3RpdmF0aW9uIG9mIG51Y2xlYXIgcmVjZXB0b3IgTlI1QTIgaW5jcmVhc2VzIEdsdXQ0IGV4cHJlc3Npb24gYW5kIGdsdWNvc2UgbWV0YWJvbGlzbSBpbiBtdXNjbGUgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00NDY7ICBjaXRhdGlvbl9wYWdlcz02MTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgRE5BIG1ldGh5bGF0aW9uIGNoYW5nZXMgYmV0d2VlbiB0aGUgc3VwZXJmaWNpYWwgYW5kIGRlZXAgYmFja2ZhdCB0aXNzdWVzIG9mIHRoZSBwaWc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gTW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NzA5OC0xMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNwRyBpc2xhbmRzIGFzIGdlbmUgbWFya2VycyBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEwOTUtMTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgbWV0aHlsYXRpb24gb2YgdGlzc3VlLSBhbmQgY2FuY2VyLXNwZWNpZmljIENwRyBpc2xhbmQgc2hvcmVzIGRpc3Rpbmd1aXNoZXMgaHVtYW4gaW5kdWNlZCBwbHVyaXBvdGVudCBzdGVtIGNlbGxzLCBlbWJyeW9uaWMgc3RlbSBjZWxscyBhbmQgZmlicm9ibGFzdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTM1MC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaHVtYW4gY29sb24gY2FuY2VyIG1ldGh5bG9tZSBzaG93cyBzaW1pbGFyIGh5cG8tIGFuZCBoeXBlcm1ldGh5bGF0aW9uIGF0IGNvbnNlcnZlZCB0aXNzdWUtc3BlY2lmaWMgQ3BHIGlzbGFuZCBzaG9yZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTc4LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1ldGh5bGF0aW9uIHByb2ZpbGUgb2YgaW4gdml0cm8gaW1tb3J0YWxpemVkIGh1bWFuIGNlbGwgbGluZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gT25jb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Mjc1LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvNS8zNDMvOTk3MDU3L2RzdjAxNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRE5BIG1ldGh5bGF0aW9uIHBsYXlzIGEgbWFqb3Igcm9sZSBpbiB0aGUgZXBpZ2VuZXRpYyByZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbi4gQWx0aG91Z2ggYSBmZXcgRE5BIG1ldGh5bGF0aW9uIHByb2ZpbGluZyBzdHVkaWVzIG9mIHBvcmNpbmUgZ2Vub20iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0Nzk0OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSpakSHRtcDCgU+hXPCiISbOrXwSw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f343%2f347949" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f343%2f347949" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&amp;Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/5">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4936920">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4936920">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4936922">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4936922">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4936929">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4936929">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4936950">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4936950">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4937034">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4937034">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4937046">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4937046">Authors’ contribution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4937050">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4937050">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4937055">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4937055">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4937060">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4937060">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/5/331/347913">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/5/357/348184">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Minkyeung Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Minkyeung+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Minkyeung Choi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jongin Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jongin+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jongin Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Min Thong Le</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Min+Thong+Le">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Le M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Le M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Min Thong Le</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dinh Truong Nguyen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dinh+Truong+Nguyen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nguyen D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nguyen D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Dinh Truong Nguyen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suhyun Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suhyun+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Park S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Suhyun Park</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nagasundarapandian Soundrarajan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nagasundarapandian+Soundrarajan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Soundrarajan N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Soundrarajan N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nagasundarapandian Soundrarajan</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyle M. Schachtschneider</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Animal Sciences</div>, <div class="insititution">University of Illinois</div>, <div class="addr-line">Urbana, IL</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Animal Breeding and Genomics Center</div>, <div class="insititution">Wageningen University</div>, <div class="addr-line">Wageningen</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyle+M.+Schachtschneider">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schachtschneider K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schachtschneider K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kyle M. Schachtschneider</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jaebum Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jaebum+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jaebum Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jin-Ki Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Animal Biotechnology Division</div>, <div class="insititution">National Institute of Animal Science</div>, <div class="addr-line">Suwon</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jin-Ki+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Park J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jin-Ki Park</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jin-Hoi Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jin-Hoi+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jin-Hoi Kim</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chankyu Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Biotechnology</div>, <div class="insititution">Konkuk University</div>, <div class="addr-line">Kwangjin-gu, Seoul 143-701</div>, <div class="country">Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 2-450-3697. Fax. +82 2-457-8488. E-mail: <a href="mailto:chankyu@konkuk.ac.kr" target="_blank">chankyu@konkuk.ac.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chankyu+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Park C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chankyu Park</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: School of Biotechnology, Tan Tao University, Duc Hoa, Vietnam.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="TN8"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 343–355, <a href='https://doi.org/10.1093/dnares/dsv017'>https://doi.org/10.1093/dnares/dsv017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">10 September 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 March 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">31 July 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347949" href="/dnaresearch/article-pdf/22/5/343/997057/dsv017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Minkyeung Choi, Jongin Lee, Min Thong Le, Dinh Truong Nguyen, Suhyun Park, Nagasundarapandian Soundrarajan, Kyle M. Schachtschneider, Jaebum Kim, Jin-Ki Park, Jin-Hoi Kim, Chankyu Park;  Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing, <em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 343–355, <a href='https://doi.org/10.1093/dnares/dsv017'>https://doi.org/10.1093/dnares/dsv017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347949?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347949?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347949?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347949?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347949?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347949?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347949?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20analysis%20of%20DNA%20methylation%20in%20pigs%20using%20reduced%20representation%20bisulfite%20sequencing&amp;publicationDate=2015-09-10&amp;volumeNum=22&amp;issueNum=5&amp;author=Choi%2C%20Minkyeung%3B%20Lee%2C%20Jongin&amp;startPage=343&amp;endPage=355&amp;contentId=10.1093%2Fdnares%2Fdsv017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f343%2f347949" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4936920 id="4936920" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>DNA methylation plays a major role in the epigenetic regulation of gene expression. Although a few DNA methylation profiling studies of porcine genome which is one of the important biomedical models for human diseases have been reported, the available data are still limited. We tried to study methylation patterns of diverse pig tissues as a study of the International Swine Methylome Consortium to generate the swine reference methylome map to extensively evaluate the methylation profile of the pig genome at a single base resolution. We generated and analysed the DNA methylome profiles of five different tissues and a cell line originated from pig. On average, 39.85 and 62.1% of cytosine and guanine dinucleotides (CpGs) of CpG islands and 2 kb upstream of transcription start sites were covered, respectively. We detected a low rate (an average of 1.67%) of non-CpG methylation in the six samples except for the neocortex (2.3%). The observed global CpG methylation patterns of pigs indicated high similarity to other mammals including humans. The percentage of CpG methylation associated with gene features was similar among the tissues but not for a 3D4/2 cell line. Our results provide essential information for future studies of the porcine epigenome.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="pig">pig</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="methylome">methylome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetics">epigenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RRBS">RRBS</a></div>                            <h2 scrollto-destination=4936922 id="4936922" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>DNA methylation is a mechanism behind epigenetic changes in gene expression in diverse species including mammals.<sup><a reveal-id="DSV017C1" data-open="DSV017C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV017C2" data-open="DSV017C2" class="link link-ref link-reveal xref-bibr">2</a></sup> 5-Methylcytosine plays an important role in gene silencing, genomic imprinting,<sup><a reveal-id="DSV017C3" data-open="DSV017C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSV017C4" data-open="DSV017C4" class="link link-ref link-reveal xref-bibr">4</a></sup> X-chromosome inactivation,<sup><a reveal-id="DSV017C5" data-open="DSV017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> cancer progression,<sup><a reveal-id="DSV017C6" data-open="DSV017C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV017C7" data-open="DSV017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> embryonic development and tissue differentiation.<sup><a reveal-id="DSV017C8  DSV017C11" data-open="DSV017C8  DSV017C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> Recent studies reported the tissue- or cell line-specific difference of DNA methylation which contributes to regulating tissue- or cell-specific gene expression.<sup><a reveal-id="DSV017C12  DSV017C16" data-open="DSV017C12  DSV017C16" class="link link-ref link-reveal xref-bibr">12–16</a></sup> Cell immortalization can lead to the accumulation of DNA methylation at promoter regions, resulting in corresponding changes in gene expression.<sup><a reveal-id="DSV017C17" data-open="DSV017C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>In mammals, DNA methylation occurs preferentially at cytosine and guanine dinucleotides (CpGs), which tend to cluster in highly enriched short regions (0.2–2 kb) known as CpG islands (CGIs); ∼50–60% of promoter regions are associated with CGIs.<sup><a reveal-id="DSV017C18" data-open="DSV017C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The levels of CpG methylation at CGI and promoter regions are lower than those at other CpG sites<sup><a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV017C20" data-open="DSV017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and play an important role in regulating gene expression.<sup><a reveal-id="DSV017C21" data-open="DSV017C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>Several methods including bisulfite sequencing, reduced representation bisulfite sequencing (RRBS),<sup><a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV017C22" data-open="DSV017C22" class="link link-ref link-reveal xref-bibr">22</a></sup> methylated DNA immunoprecipitation (MeDIP-seq),<sup><a reveal-id="DSV017C23" data-open="DSV017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> methyl-binding domain sequencing (MBD-seq, MethylCap-seq)<sup><a reveal-id="DSV017C24  DSV017C26" data-open="DSV017C24  DSV017C26" class="link link-ref link-reveal xref-bibr">24–26</a></sup> and BeadChip array (Infinium<sup><a reveal-id="DSV017C27" data-open="DSV017C27" class="link link-ref link-reveal xref-bibr">27</a></sup>) have been developed to analyse patterns of DNA methylation across diverse genomes.<sup><a reveal-id="DSV017C2" data-open="DSV017C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV017C28" data-open="DSV017C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV017C29" data-open="DSV017C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Aberrant methylation patterns on chromosomes or a part of genome can cause developmental abnormalities or diseases.<sup><a reveal-id="DSV017C30" data-open="DSV017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Several large-scale projects including the International Human Epigenome Consortium (<a class="link link-uri" href="http://www.ihec-epigenomes.org/">http://www.ihec-epigenomes.org/</a>), human ENCODE project (<a class="link link-uri" href="https://www.encodeproject.org/">https://www.encodeproject.org/</a>) and Blueprint Epigenome (<a class="link link-uri" href="http://www.blueprint-epigenome.eu/">http://www.blueprint-epigenome.eu/</a>) are in progress for a deeper understanding of the functional importance of epigenetic changes including DNA methylation. Recently, the International Swine Methylome Consortium (ISMC) was organized to generate the swine reference methylome map to illuminate the importance of epigenetic changes to phenotypic variations in production traits<sup><a reveal-id="DSV017C31  DSV017C34" data-open="DSV017C31  DSV017C34" class="link link-ref link-reveal xref-bibr">31–34</a></sup> and disease progression.<sup><a reveal-id="DSV017C30" data-open="DSV017C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSV017C35  DSV017C39" data-open="DSV017C35  DSV017C39" class="link link-ref link-reveal xref-bibr">35–39</a></sup> Although genome-wide methylation studies of farm animals have recently been published for sheep,<sup><a reveal-id="DSV017C40" data-open="DSV017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> cattle,<sup><a reveal-id="DSV017C41" data-open="DSV017C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSV017C42" data-open="DSV017C42" class="link link-ref link-reveal xref-bibr">42</a></sup> horses<sup><a reveal-id="DSV017C43" data-open="DSV017C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and pigs,<sup><a reveal-id="DSV017C44" data-open="DSV017C44" class="link link-ref link-reveal xref-bibr">44</a></sup> the depth of data and tissue coverage are still limited compared with that of humans and mice.</p><p>The importance of domestic pigs (<em>Sus scrofa domesticus</em>), as large animal biomedical models for human diseases, has recently been recognized for its similarity to humans in organ size and structure, general physiology and disease phenotypes.<sup><a reveal-id="DSV017C45" data-open="DSV017C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Advantages in animal husbandry, large litter size and short generation interval of pigs make them more suitable as animal models over primate models, although the phylogenetic relationship of pigs to humans is more distant than that of primates to humans.</p><p>The completion of the draft reference genome of the pig<sup><a reveal-id="DSV017C46" data-open="DSV017C46" class="link link-ref link-reveal xref-bibr">46</a></sup> has allowed production of previously unachievable results using sequence-based genome-wide studies on variations or evolutionary changes.<sup><a reveal-id="DSV017C47  DSV017C49" data-open="DSV017C47  DSV017C49" class="link link-ref link-reveal xref-bibr">47–49</a></sup> Although the analysis of global genome methylation using MeDIP-seq<sup><a reveal-id="DSV017C50" data-open="DSV017C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and RRBS<sup><a reveal-id="DSV017C44" data-open="DSV017C44" class="link link-ref link-reveal xref-bibr">44</a></sup> was reported in pigs, the results from more diverse tissues at single base-pair resolution are necessary to have better understanding of the methylation profiles of the pig genome related to chromosomal variations, gene features, CGI association and non-biparental expression.</p><p>In this study, we analysed pig methylome of five different tissues including the neocortex, spleen, muscle, liver and olfactory epithelium, and a pulmonary alveolar macrophage (PAM) cell line (3D4/2) by using the RRBS method. We compared the methylation profiles across different tissues and described the similarity and differences of the pig methylome from human and mice methylomes in previously reported RRBS analysis. Further, we included the results of methylome analysis from several previously unreported tissues and demonstrated the successful use of RRBS on pig genome.</p>                            <h2 scrollto-destination=4936929 id="4936929" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4936930 id="4936930" class="section-title" data-legacy-id=s2a>2.1. Ethics statement</h3>
<p>The experimental procedure was approved and supervised by the Institute of Animal Care and Use Committee of Konkuk University, Seoul, Korea. Three individuals of 9-week-old Landrace × Yorkshire crossbreed pigs in good health were purchased from local farm, deeply anaesthetized and sacrificed.</p>                            <h3 scrollto-destination=4936933 id="4936933" class="section-title" data-legacy-id=s2b>2.2. Sample preparation and genomic DNA isolation</h3>
<p>Target tissues including the neocortex, spleen, liver, femoral muscle and olfactory epithelium were dissected out, snap frozen in liquid nitrogen and kept at −80°C until use. Pig pulmonary alveolar macrophages cell line 3D4/2 (ATCC<sup>®</sup> CRL-2845™) was cultured in T75 cell culture flasks (SPL Life Sciences, Pocheon, Korea) using an RPMI 1640 culture medium (ATCC modification) (Life Technologies, Carlsbad, CA, USA) supplemented with 10% fetal bovine serum and 1% Penicillin–Streptomycin (Life Technologies) and 0.1 mM 2-mercaptoethanol. Genomic DNA was extracted from 0.5 g tissue or from 3 × 10<sup>7</sup> cells. Briefly, tissues were incubated in 400 µl of tissue lysis buffer (0.1 M Tris–HCl, 200 mM NaCl, 5 mM EDTA and 0.2% SDS) with 250 µg/ml proteinase K at 55°C for 6 h, and cell pellets were incubated in 400 µl lysis buffer (10 mM Tris–HCl, 100 mM EDTA and 0.5% SDS) at 37°C for 6 h, followed by phenol extraction. Isolated DNA was incubated with 20 µg/ml DNase-free RNase and purified using a PowerClean<sup>®</sup> DNA Clean-Up Kit (MO BIO, Carlsbad, CA, USA) according to the manufacturer's protocol.</p>                            <h3 scrollto-destination=4936935 id="4936935" class="section-title" data-legacy-id=s2c>2.3. Sequencing and reads alignment</h3>
<p>The construction of RRBS libraries and paired-end sequencing using a HiSeq2000 analyzer (Illumina, San Diego, CA, USA) was performed at BGI-Shenzhen (Shenzhen, China) according to the manufacturer's instructions. Briefly, 100 ng of pooled genomic DNA with an equal amount from three individuals was used for the construction of each RRBS library. Genomic DNA was digested with <em>Msp</em>I enzyme (recognition site C^CGG, NEB, Ipswich, MA, USA) at 37°C for 16 h. After purification, single A nucleotides were added to the digested blunt-end products, followed by ligation to methylated adapter with T overhang. Ligated products corresponding to DNA fragments 40–220 bp long were purified by 2% agarose gel electrophoresis. Bisulfite conversion was conducted using a ZYMO EZ DNA Methylation-Gold™ Kit (ZYMO, Irvine, CA, USA). The final libraries were generated by PCR amplification. RRBS libraries were analysed by an Agilent 2100 Bioanalyzer (Agilent Technologies) and quantified by real-time PCR. The sequenced paired-end read length was 49 bp. Raw sequencing data were processed by an Illumina base-calling pipeline. Clean reads were aligned to the modified pig reference genome (Sscrofa10.2) in an unbiased way for bisulfite sequencing data. After removing adaptor sequences, clean reads were aligned to two modified references with BS-seeker 2 v. 2.0.8<sup><a reveal-id="DSV017C51" data-open="DSV017C51" class="link link-ref link-reveal xref-bibr">51</a></sup> using Bowtie 2 v. 2.1.0<sup><a reveal-id="DSV017C52" data-open="DSV017C52" class="link link-ref link-reveal xref-bibr">52</a></sup> in local alignment mode and allowing no more than four mismatches per read. Methylation status was determined using the bs_seeker2-call_methylation.py script and only uniquely aligned reads. For the simulation analysis of pig genome RRBS, we performed an <em>in silico</em> restriction analysis of the current pig reference genome (Sus scrofa 10.2) for <em>Msp</em>I or <em>Msp</em>I/<em>ApeK</em>I and set the fragment size to 40–220 bp.</p>                            <h3 scrollto-destination=4936938 id="4936938" class="section-title" data-legacy-id=s2d>2.4. RRBS data analysis</h3>
<p>Methylation levels of cytosines were analysed by methylKit.<sup><a reveal-id="DSV017C53" data-open="DSV017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Briefly, the number of methylated and unmethylated CpG and non-CpG (CHG and CHH, H representing A/C/T) sites was counted for each region. CGIs were defined as regions &gt;200 bp with a GC fraction &gt;0.5 and an observed-to-expected ratio of CpG &gt;0.6. CGI shores were defined as regions 2 kb in length adjacent to CGIs.<sup><a reveal-id="DSV017C18" data-open="DSV017C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The porcine CGIs annotation was downloaded from USCS (<a class="link link-uri" href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>, susScr3). The porcine gene annotation was downloaded from the Ensembl (<a class="link link-uri" href="http://www.ensembl.org/">http://www.ensembl.org/</a>, protein-coding genes). To define the differentially methylated cytosines (DMCs), multiple pairwise comparisons were performed against CpG methylation information of six samples and filtered (<em>q</em> &lt; 0.01) using methylKit.<sup><a reveal-id="DSV017C53" data-open="DSV017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Tissue-specific differentially methylated CpG sites (tDMC) were identified by a script which we developed to select the CpGs showing tissue-specific methylation levels in comparison to other samples. The result of each comparison among different tissues was compiled. Filtered DMCs were annotated based on genes and CGI features including gene bodies and upstream 2, 5 and 10 kb regions from the TSS. The number of DMCs was visualized by Venn Diagrams using the Jvenn (<a class="link link-uri" href="http://bioinfo.genotoul.fr/jvenn/index.html">http://bioinfo.genotoul.fr/jvenn/index.html</a>).<sup><a reveal-id="DSV017C54" data-open="DSV017C54" class="link link-ref link-reveal xref-bibr">54</a></sup></p>                            <h3 scrollto-destination=4936941 id="4936941" class="section-title" data-legacy-id=s2e>2.5. Functional annotation of tDMC and correlation analysis between the methylation level and gene expression</h3>
<p>Gene ontological (GO) analysis was carried out for tDMC using an in-house developed script. The swine GO terms were downloaded from the Ensembl (<a class="link link-uri" href="http://www.ensembl.org/">http://www.ensembl.org/</a>) and the gene sets obtained from tDMC analysis for each tissue. Gene expression profiles were downloaded from the Gene Expression Omnibus (GEO). The gene expression profiles created with 27k cDNAs probes and 23 healthy porcine tissues (GEO accession ID: GSE4918)<sup><a reveal-id="DSV017C55" data-open="DSV017C55" class="link link-ref link-reveal xref-bibr">55</a></sup> were downloaded. The tissues and accession numbers used in this study are frontal cortex (GSM110567 and GSM110545), liver (GSM110543 and GSM110569), biceps femoris (GSM110549 and GSM110577) and spleen (GSM110571 and GSM110546), respectively. Differentially expressed genes (DEGs) across four tissues were identified using the GEO2R plugin (<em>P</em> &lt; 0.05) after the adjustment for multiple testing. The correlation between the levels of gene expression and methylation in tDMCs was calculated by the Pearson's correlation test.</p>                            <h3 scrollto-destination=4936945 id="4936945" class="section-title" data-legacy-id=s2f>2.6. Statistical analysis</h3>
<p>All statistical analyses were performed using R programming language (version 3.1.1).<sup><a reveal-id="DSV017C56" data-open="DSV017C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Statistical difference in methylation rates of CpG and non-CpG sites across tissues was determined using the Wilcoxon signed-rank test. The Pearson's correlation test was used to compare the global methylation patterns and the specific methylation patterns for CGI and CGI shore across tissues which were implemented using methylKit package (version 0.9.2).<sup><a reveal-id="DSV017C53" data-open="DSV017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The clustering analysis with the methylation levels and patterns of different samples was carried out using Ward's minimum variance method implemented in methylKit package.<sup><a reveal-id="DSV017C53" data-open="DSV017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The statistical significance of enrichments and/or depletions of GO was calculated by the hypergeometric test<sup><a reveal-id="DSV017C57" data-open="DSV017C57" class="link link-ref link-reveal xref-bibr">57</a></sup> and multiple test correction using R language.</p>                            <h2 scrollto-destination=4936950 id="4936950" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4936952 id="4936952" class="section-title" data-legacy-id=s3a>3.1. <em>In silico</em> analysis of pig genome to predict possible coverage of CpG sites from RRBS</h3>
<p><em>In silico</em> restriction analysis of the pig genome using enzymes <em>Msp</em>I was carried out to predict the possible coverage of CpG sites from the RRBS analysis of the pig genome. The <em>in silico</em> pig genome RRBS library consisting of DNA fragments in a size range of 40–220 bp resulted in a total of 2,371,480 fragments for <em>Msp</em>I digestion.</p><p>We performed <em>in silico</em> analyses of the distribution pattern of restricted fragments of the pig genome in the window of 200 bp with a 20-bp increment, starting from 200 and up to 500 bp for <em>Msp</em>I digestion. In addition, 40–220 bp window was analysed as a comparison to the result of humans and mice (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The results were similar to those of humans and mice in which the 40- to 220-bp window covers 37.8% of total fragments of <em>Msp</em>I-digested libraries.<sup><a reveal-id="DSV017C58" data-open="DSV017C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p><p>Our <em>in silico</em> analysis showed that 28% of generated fragments digested by <em>Msp</em>I was expected to be distributed in the 40- to 220-bp window, which corresponds to 2.6% of the pig genome. For CpG sites, 11.9% of the total CpG sites of the pig genome were predicted in <em>Msp</em>I libraries (Table <a reveal-id="DSV017TB1" data-open="DSV017TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The size of the pig reduced representation (RR) genome was the closest to that of the human. </p>                        <a id="4936961" scrollto-destination="4936961"></a>
<div content-id="DSV017TB1" class="table-modal"><div class="table-wrap"><div id="DSV017TB1" class="table-wrap-title" data-id="DSV017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison between porcine and other reduced representation (RR) genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Genome </th><th colspan="4"><em>Msp</em>I<hr /> </th></tr><tr><th>Human </th><th>Mouse </th><th>Zebrafish<sup><a reveal-id="DSV017C59" data-open="DSV017C59" class="link link-ref link-reveal xref-bibr">59</a></sup> </th><th>Pig </th></tr></thead><tbody><tr><td>Size of genome (Gb) </td><td>3.2 </td><td>2.8 </td><td>1.41 </td><td>2.8 </td></tr><tr><td>Size of RR genome (Mb) </td><td>74 </td><td>38 </td><td>31 </td><td>73 </td></tr><tr><td>Per cent of whole genome </td><td>2.3 </td><td>1.4 </td><td>2.2 </td><td>2.6 </td></tr><tr><td>GC content of whole genome (%) </td><td>40.9 </td><td>41.7 </td><td>36.5 </td><td>42.3 </td></tr><tr><td>Number of fragments (RR genome) </td><td>647,626 </td><td>333,104 </td><td>264,598 </td><td>664,080 </td></tr><tr><td>Number of CpG sites (RR genome) </td><td>4,068,947 </td><td>1,506,712 </td><td>1,430,390 </td><td>5,586,410 </td></tr><tr><td>Per cent of total genomic CpG sites </td><td>13.5 </td><td>7 </td><td>5.3 </td><td>18.30 </td></tr></tbody></table></div><div content-id="DSV017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Genome </th><th colspan="4"><em>Msp</em>I<hr /> </th></tr><tr><th>Human </th><th>Mouse </th><th>Zebrafish<sup><a reveal-id="DSV017C59" data-open="DSV017C59" class="link link-ref link-reveal xref-bibr">59</a></sup> </th><th>Pig </th></tr></thead><tbody><tr><td>Size of genome (Gb) </td><td>3.2 </td><td>2.8 </td><td>1.41 </td><td>2.8 </td></tr><tr><td>Size of RR genome (Mb) </td><td>74 </td><td>38 </td><td>31 </td><td>73 </td></tr><tr><td>Per cent of whole genome </td><td>2.3 </td><td>1.4 </td><td>2.2 </td><td>2.6 </td></tr><tr><td>GC content of whole genome (%) </td><td>40.9 </td><td>41.7 </td><td>36.5 </td><td>42.3 </td></tr><tr><td>Number of fragments (RR genome) </td><td>647,626 </td><td>333,104 </td><td>264,598 </td><td>664,080 </td></tr><tr><td>Number of CpG sites (RR genome) </td><td>4,068,947 </td><td>1,506,712 </td><td>1,430,390 </td><td>5,586,410 </td></tr><tr><td>Per cent of total genomic CpG sites </td><td>13.5 </td><td>7 </td><td>5.3 </td><td>18.30 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>All RR genomes were simulated by selecting DNA fragment in a 40- to 220-bp window after <em>in silico</em> restriction enzyme digestion.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4936961" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV017TB1" class="table-wrap-title" data-id="DSV017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison between porcine and other reduced representation (RR) genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Genome </th><th colspan="4"><em>Msp</em>I<hr /> </th></tr><tr><th>Human </th><th>Mouse </th><th>Zebrafish<sup><a reveal-id="DSV017C59" data-open="DSV017C59" class="link link-ref link-reveal xref-bibr">59</a></sup> </th><th>Pig </th></tr></thead><tbody><tr><td>Size of genome (Gb) </td><td>3.2 </td><td>2.8 </td><td>1.41 </td><td>2.8 </td></tr><tr><td>Size of RR genome (Mb) </td><td>74 </td><td>38 </td><td>31 </td><td>73 </td></tr><tr><td>Per cent of whole genome </td><td>2.3 </td><td>1.4 </td><td>2.2 </td><td>2.6 </td></tr><tr><td>GC content of whole genome (%) </td><td>40.9 </td><td>41.7 </td><td>36.5 </td><td>42.3 </td></tr><tr><td>Number of fragments (RR genome) </td><td>647,626 </td><td>333,104 </td><td>264,598 </td><td>664,080 </td></tr><tr><td>Number of CpG sites (RR genome) </td><td>4,068,947 </td><td>1,506,712 </td><td>1,430,390 </td><td>5,586,410 </td></tr><tr><td>Per cent of total genomic CpG sites </td><td>13.5 </td><td>7 </td><td>5.3 </td><td>18.30 </td></tr></tbody></table></div><div content-id="DSV017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Genome </th><th colspan="4"><em>Msp</em>I<hr /> </th></tr><tr><th>Human </th><th>Mouse </th><th>Zebrafish<sup><a reveal-id="DSV017C59" data-open="DSV017C59" class="link link-ref link-reveal xref-bibr">59</a></sup> </th><th>Pig </th></tr></thead><tbody><tr><td>Size of genome (Gb) </td><td>3.2 </td><td>2.8 </td><td>1.41 </td><td>2.8 </td></tr><tr><td>Size of RR genome (Mb) </td><td>74 </td><td>38 </td><td>31 </td><td>73 </td></tr><tr><td>Per cent of whole genome </td><td>2.3 </td><td>1.4 </td><td>2.2 </td><td>2.6 </td></tr><tr><td>GC content of whole genome (%) </td><td>40.9 </td><td>41.7 </td><td>36.5 </td><td>42.3 </td></tr><tr><td>Number of fragments (RR genome) </td><td>647,626 </td><td>333,104 </td><td>264,598 </td><td>664,080 </td></tr><tr><td>Number of CpG sites (RR genome) </td><td>4,068,947 </td><td>1,506,712 </td><td>1,430,390 </td><td>5,586,410 </td></tr><tr><td>Per cent of total genomic CpG sites </td><td>13.5 </td><td>7 </td><td>5.3 </td><td>18.30 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>All RR genomes were simulated by selecting DNA fragment in a 40- to 220-bp window after <em>in silico</em> restriction enzyme digestion.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4936961" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4936963 id="4936963" class="section-title" data-legacy-id=s3b>3.2. Mapping and analysis of RRBS reads from five tissues and a cell line</h3>
<p>We constructed <em>Msp</em>I single restriction RRBS libraries as an initial study of the RRBS analysis of the pig genome. According to the <em>in silico</em> analysis, the coverage of nearly 54.71% of total CpGs within CGI (1,295,120 of 2,367,139 total CpGs within CGI) and ∼83.49% of CpGs within the upstream 2 kb region from the transcription start sites (TSS) (250,631 of 300,187 protein-coding genes) was expected in the <em>Msp</em>I-digested pig RRBS library.</p><p>A total of six libraries were constructed with 40- to 220-bp insert fragments, which is identical to RRBS studies in humans and mice,<sup><a reveal-id="DSV017C58" data-open="DSV017C58" class="link link-ref link-reveal xref-bibr">58</a></sup> from five tissues including the neocortex, spleen, liver, muscle, olfactory epithelium and a 3D4/2 (PAM) cell line. We generated a minimum of 3.25Gb clean reads from each RRBS library with the maximum of 5.48Gb from the liver (Table <a reveal-id="DSV017TB2" data-open="DSV017TB2" class="link link-reveal link-table xref-fig">2</a>). The sequences were submitted to European Nucleotide Archive (ENA) under the study accession number, PRJEB9561. An average of 59.56% of total paired-end reads were aligned to the reference genome according to the criteria described in Materials and methods (Table <a reveal-id="DSV017TB2" data-open="DSV017TB2" class="link link-reveal link-table xref-fig">2</a>). The reads with a single placement, minimum number of mismatches (4 mismatches per read) and clear strand assignment were defined as uniquely mapped reads. The percentage of uniquely mapped reads was 56.89, 57.16, 50.77, 51.76, 53.56 and 55.08% for the neocortex, spleen, liver, muscle, olfactory epithelium and 3D4/2 cells, respectively, with the average of 54.2%. The frequency of multi-mapped reads ranged from 4.92 to 5.74% for different samples (Table <a reveal-id="DSV017TB2" data-open="DSV017TB2" class="link link-reveal link-table xref-fig">2</a>). The coverage of CpGs detected within CGIs and the upstream 2 kb region from TSS ranged from 38.3 to 43.32% and 58.75 to 67.8% of total CpGs in the genome, respectively. This is somewhat lower than the theoretical predicted value (54.71% for CGI and 83.49% for upstream 2 kb from TSS) of the simulation because of the efficiency difference between the experiment and theoretical analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The results from our experimental analysis are in accord with the expected values from the simulation, indicating the validity of our data. </p>                        <a id="4936969" scrollto-destination="4936969"></a>
<div content-id="DSV017TB2" class="table-modal"><div class="table-wrap"><div id="DSV017TB2" class="table-wrap-title" data-id="DSV017TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Details of mapping of porcine RRBS libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Clean read (M) </th><th>Clean read (Gb) </th><th>Mapped rate (%) </th><th>Uniquely mapped rate (%) </th><th>Multiple mapping (%) </th><th>CpG methylation (%) </th><th>Non-CpG methylation (%) </th></tr></thead><tbody><tr><td>3D4/2 </td><td>66.3 </td><td>3.25 </td><td>60.83 </td><td>55.08 </td><td>5.74 </td><td>53.05 </td><td>1.71 </td></tr><tr><td>Neocortex </td><td>76.8 </td><td>3.76 </td><td>62.29 </td><td>56.89 </td><td>5.40 </td><td>50.87 </td><td>2.30 </td></tr><tr><td>OE </td><td>67.6 </td><td>3.31 </td><td>58.48 </td><td>53.56 </td><td>4.92 </td><td>49.19 </td><td>1.63 </td></tr><tr><td>Spleen </td><td>72.6 </td><td>3.56 </td><td>62.38 </td><td>57.16 </td><td>5.22 </td><td>52.84 </td><td>1.66 </td></tr><tr><td>Liver </td><td>111.84 </td><td>5.48 </td><td>56.17 </td><td>50.77 </td><td>5.40 </td><td>47.83 </td><td>1.67 </td></tr><tr><td>Muscle </td><td>85.1 </td><td>4.17 </td><td>57.20 </td><td>51.76 </td><td>5.45 </td><td>48.06 </td><td>1.69 </td></tr></tbody></table></div><div content-id="DSV017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Clean read (M) </th><th>Clean read (Gb) </th><th>Mapped rate (%) </th><th>Uniquely mapped rate (%) </th><th>Multiple mapping (%) </th><th>CpG methylation (%) </th><th>Non-CpG methylation (%) </th></tr></thead><tbody><tr><td>3D4/2 </td><td>66.3 </td><td>3.25 </td><td>60.83 </td><td>55.08 </td><td>5.74 </td><td>53.05 </td><td>1.71 </td></tr><tr><td>Neocortex </td><td>76.8 </td><td>3.76 </td><td>62.29 </td><td>56.89 </td><td>5.40 </td><td>50.87 </td><td>2.30 </td></tr><tr><td>OE </td><td>67.6 </td><td>3.31 </td><td>58.48 </td><td>53.56 </td><td>4.92 </td><td>49.19 </td><td>1.63 </td></tr><tr><td>Spleen </td><td>72.6 </td><td>3.56 </td><td>62.38 </td><td>57.16 </td><td>5.22 </td><td>52.84 </td><td>1.66 </td></tr><tr><td>Liver </td><td>111.84 </td><td>5.48 </td><td>56.17 </td><td>50.77 </td><td>5.40 </td><td>47.83 </td><td>1.67 </td></tr><tr><td>Muscle </td><td>85.1 </td><td>4.17 </td><td>57.20 </td><td>51.76 </td><td>5.45 </td><td>48.06 </td><td>1.69 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>OE: olfactory epithelium.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4936969" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV017TB2" class="table-wrap-title" data-id="DSV017TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Details of mapping of porcine RRBS libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Clean read (M) </th><th>Clean read (Gb) </th><th>Mapped rate (%) </th><th>Uniquely mapped rate (%) </th><th>Multiple mapping (%) </th><th>CpG methylation (%) </th><th>Non-CpG methylation (%) </th></tr></thead><tbody><tr><td>3D4/2 </td><td>66.3 </td><td>3.25 </td><td>60.83 </td><td>55.08 </td><td>5.74 </td><td>53.05 </td><td>1.71 </td></tr><tr><td>Neocortex </td><td>76.8 </td><td>3.76 </td><td>62.29 </td><td>56.89 </td><td>5.40 </td><td>50.87 </td><td>2.30 </td></tr><tr><td>OE </td><td>67.6 </td><td>3.31 </td><td>58.48 </td><td>53.56 </td><td>4.92 </td><td>49.19 </td><td>1.63 </td></tr><tr><td>Spleen </td><td>72.6 </td><td>3.56 </td><td>62.38 </td><td>57.16 </td><td>5.22 </td><td>52.84 </td><td>1.66 </td></tr><tr><td>Liver </td><td>111.84 </td><td>5.48 </td><td>56.17 </td><td>50.77 </td><td>5.40 </td><td>47.83 </td><td>1.67 </td></tr><tr><td>Muscle </td><td>85.1 </td><td>4.17 </td><td>57.20 </td><td>51.76 </td><td>5.45 </td><td>48.06 </td><td>1.69 </td></tr></tbody></table></div><div content-id="DSV017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Clean read (M) </th><th>Clean read (Gb) </th><th>Mapped rate (%) </th><th>Uniquely mapped rate (%) </th><th>Multiple mapping (%) </th><th>CpG methylation (%) </th><th>Non-CpG methylation (%) </th></tr></thead><tbody><tr><td>3D4/2 </td><td>66.3 </td><td>3.25 </td><td>60.83 </td><td>55.08 </td><td>5.74 </td><td>53.05 </td><td>1.71 </td></tr><tr><td>Neocortex </td><td>76.8 </td><td>3.76 </td><td>62.29 </td><td>56.89 </td><td>5.40 </td><td>50.87 </td><td>2.30 </td></tr><tr><td>OE </td><td>67.6 </td><td>3.31 </td><td>58.48 </td><td>53.56 </td><td>4.92 </td><td>49.19 </td><td>1.63 </td></tr><tr><td>Spleen </td><td>72.6 </td><td>3.56 </td><td>62.38 </td><td>57.16 </td><td>5.22 </td><td>52.84 </td><td>1.66 </td></tr><tr><td>Liver </td><td>111.84 </td><td>5.48 </td><td>56.17 </td><td>50.77 </td><td>5.40 </td><td>47.83 </td><td>1.67 </td></tr><tr><td>Muscle </td><td>85.1 </td><td>4.17 </td><td>57.20 </td><td>51.76 </td><td>5.45 </td><td>48.06 </td><td>1.69 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>OE: olfactory epithelium.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4936969" target="_blank">View Large</a></div></div><p>Subsequently, we selected CpG sites that were present &gt;10 times in the mapping, namely CpG10, and subjected them to further analyses. The mean coverage of CpG10 was 37- to 56.5-fold among tissues (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The frequency of CpG10 was similar across the six samples, and the average coverage rates of CpG10 for gene-related regions including exon, intron and 10 kb upstream of TSS were 50.88, 44.38 and 42.58%, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=4936973 id="4936973" class="section-title" data-legacy-id=s3c>3.3. Global methylation profile of the pig RR genome</h3>
<p>RRBS analysis of mammalian genome showed the enrichment of CpG-rich regions<sup><a reveal-id="DSV017C2" data-open="DSV017C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV017C60" data-open="DSV017C60" class="link link-ref link-reveal xref-bibr">60</a></sup> and the presence of a large amount of unmethylated CpG sites in CGIs.<sup><a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV017C61" data-open="DSV017C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The percentage of methylated CpG sites in the pig RR genome ranged from 47.83 to 53.05% in the six different samples (Table <a reveal-id="DSV017TB2" data-open="DSV017TB2" class="link link-reveal link-table xref-fig">2</a>). The distribution of read coverage per base for each sample shows that the results did not much suffer from PCR duplication bias, which forms a secondary peak towards the right side of the histograms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>).<sup><a reveal-id="DSV017C53" data-open="DSV017C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p><p>Methylated CpG sites can be classified by the methylation levels to hypo- (&lt;10%), intermediate (20–80%) and hyper-methylated CpGs (&gt;90%), and the frequencies of three differently methylated CpG10 groups ranged from 34.5 to 40.2%, 18.3 to 23.8% and 21.4 to 28.5%, respectively, of total CpG sites, depending on the type of tissue, showing that hypo-methylated CpG sites were the largest group (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Interestingly, relatively higher hyper- and heavily methylated CpGs (&gt;95%) were detected from 3D4/2 cells than from other tissues (<em>P</em> = 0.0035 and 0.0008 for hyper- and heavy methylation, respectively; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4 and Table S3</a>). The distribution of percentage methylation from our study using five tissues and a cell line showed a pattern analogous to the bimodal distribution (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>), which is similar to the previous mammalian RRBS studies.<sup><a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV017C62" data-open="DSV017C62" class="link link-ref link-reveal xref-bibr">62</a>,<a reveal-id="DSV017C63" data-open="DSV017C63" class="link link-ref link-reveal xref-bibr">63</a></sup></p><p>Our analysis showed that the global CpG methylation is similar among the neocortex, spleen, liver, femoral muscle and olfactory epithelium (Pearson's correlation score: 0.95–0.96). However, comparisons of tissues versus 3D4/2 cell line showed significant difference in the distribution of the CpG methylation (Pearson's correlation score: 0.69–0.7, Fig. <a reveal-id="DSV017F1" data-open="DSV017F1" class="link link-reveal link-table xref-fig">1</a>A). The distance among different tissues based on the global methylation level is shown in Fig. <a reveal-id="DSV017F2" data-open="DSV017F2" class="link link-reveal link-table xref-fig">2</a>. CpG methylation among six different samples showed that the levels of methylation of 2 kb upstream of TSS and CGIs of the PAM cell line (3D4/2) were significantly higher than those in other tissues (Fig. <a reveal-id="DSV017F1" data-open="DSV017F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Regarding the results of the PAM cell line, we manually confirmed the methylation patterns of several randomly selected CpG sites detected in our RRBS by independent bisulfite sequencing. A total of 7 DMRs were amplified from the bisulfite-treated genomic DNA template and sequenced. The seven DMRs were located on four hypo-methylated (<em>ABLIM2, CBFA2T3, SDK1</em> and <em>TBCD</em>)- and three hyper-methylated genes (<em>CD14, DKC1</em> and <em>GPC3</em>)-associated regions in the PAM cell line. All results were consistent with those from the RRBS analysis of 3D4/2 (data not shown). </p>                        <a id="4936981" scrollto-destination="4936981"></a>
<div id="DSV017F1" data-id="DSV017F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01701.png?Expires=2147483647&Signature=v-E8wSAmiYE-e8WIVhlpUh6NH5YDSRI6AAQvf9IkwUybuIfqq8zX3XXNg~ylyjv0wupxntR3f18OHb~cPbiaEKg04~0tZmjXNRkUwWxkOhIeVXRPY1BoDMviO25i~b8R3vwA2BH0eBXX1XcOSRM-z4jOvlUaYY1JBOX15-NYfrMsk3KzHN-z-fwaKguIwxv0ufmsToVBWMgHC4V6NRDenO6PYmFnua9rhmtfxYo4O~UvfcBKOxf1PJroEe39AXdoBMNRGNU20QwvuBIeeCm7hrO~hKVQqPLktsasOYv7za0mG0QosUSLuvOFWIcev4TkE4OTHBIMvBlyrbMGJKwXfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation analysis of the CpG methylation patterns among different tissues. Scatter plots of percentage methylation values for pairwise comparisons in six libraries [pulmonary alveolar macrophage (PAM) cells, neocortex, spleen, liver, muscle and olfactory epithelium (OE)]. Colours in the scatter plot indicate the number of CpG sites with identical methylation pattern (methylated or non-methylated); yellow denotes large number of correlations, blue denotes lack of correlation and green denotes different methylation patterns. Numbers in the upper right side represent the pairwise Pearson's correlation scores. Histograms on the diagonal are methylation distribution of CpG sites for each sample. Comparisons of global CpG methylation level (A), methylation level of CpG in CGI regions (B) and CGI shore (C)." path-from-xml="dsv01701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4936981/dsv01701.tiff" path-from-xml="dsv01701.tiff" target="_blank">View large</a><a section="4936981" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01701.png?Expires=2147483647&Signature=tTjzMhbDmL7LV-L6NIWHNFCc6hwLlowplOWxXVxLbNm2WkkSzbjcPhtoKzFWBqf52VkTf8mI1PQMU3e7rCpiAYpJ6DWb90kdOd6Tdu51s69~vfq~2PudWuFWDCMpcZkC1YtxleIWQ3qw6OLsPnpoKSGHlgMLIs16KoJoSD4pXzVE8FBIh5yNBbsaxu2K8a9wSgRQQBXo4Dcp5kn8M20UTItYQyyZT6awonUHqV6eSy5dtHul6B8HcIeEDAZpU28OlzIXKIkdKjsAiVLAoGuHMcmt6sZqMUKAGPI64qufa9DWQix1MGMlPedxvWtOepN3lzYg3dpOyuKFvdwAp0zsHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4936981&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation analysis of the CpG methylation patterns among different tissues. Scatter plots of percentage methylation values for pairwise comparisons in six libraries [pulmonary alveolar macrophage (PAM) cells, neocortex, spleen, liver, muscle and olfactory epithelium (OE)]. Colours in the scatter plot indicate the number of CpG sites with identical methylation pattern (methylated or non-methylated); yellow denotes large number of correlations, blue denotes lack of correlation and green denotes different methylation patterns. Numbers in the upper right side represent the pairwise Pearson's correlation scores. Histograms on the diagonal are methylation distribution of CpG sites for each sample. Comparisons of global CpG methylation level (A), methylation level of CpG in CGI regions (B) and CGI shore (C).</p></div></div><div content-id="DSV017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01701.png?Expires=2147483647&Signature=v-E8wSAmiYE-e8WIVhlpUh6NH5YDSRI6AAQvf9IkwUybuIfqq8zX3XXNg~ylyjv0wupxntR3f18OHb~cPbiaEKg04~0tZmjXNRkUwWxkOhIeVXRPY1BoDMviO25i~b8R3vwA2BH0eBXX1XcOSRM-z4jOvlUaYY1JBOX15-NYfrMsk3KzHN-z-fwaKguIwxv0ufmsToVBWMgHC4V6NRDenO6PYmFnua9rhmtfxYo4O~UvfcBKOxf1PJroEe39AXdoBMNRGNU20QwvuBIeeCm7hrO~hKVQqPLktsasOYv7za0mG0QosUSLuvOFWIcev4TkE4OTHBIMvBlyrbMGJKwXfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation analysis of the CpG methylation patterns among different tissues. Scatter plots of percentage methylation values for pairwise comparisons in six libraries [pulmonary alveolar macrophage (PAM) cells, neocortex, spleen, liver, muscle and olfactory epithelium (OE)]. Colours in the scatter plot indicate the number of CpG sites with identical methylation pattern (methylated or non-methylated); yellow denotes large number of correlations, blue denotes lack of correlation and green denotes different methylation patterns. Numbers in the upper right side represent the pairwise Pearson's correlation scores. Histograms on the diagonal are methylation distribution of CpG sites for each sample. Comparisons of global CpG methylation level (A), methylation level of CpG in CGI regions (B) and CGI shore (C)." path-from-xml="dsv01701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4936981/dsv01701.tiff" path-from-xml="dsv01701.tiff" target="_blank">View large</a><a section="4936981" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01701.png?Expires=2147483647&Signature=tTjzMhbDmL7LV-L6NIWHNFCc6hwLlowplOWxXVxLbNm2WkkSzbjcPhtoKzFWBqf52VkTf8mI1PQMU3e7rCpiAYpJ6DWb90kdOd6Tdu51s69~vfq~2PudWuFWDCMpcZkC1YtxleIWQ3qw6OLsPnpoKSGHlgMLIs16KoJoSD4pXzVE8FBIh5yNBbsaxu2K8a9wSgRQQBXo4Dcp5kn8M20UTItYQyyZT6awonUHqV6eSy5dtHul6B8HcIeEDAZpU28OlzIXKIkdKjsAiVLAoGuHMcmt6sZqMUKAGPI64qufa9DWQix1MGMlPedxvWtOepN3lzYg3dpOyuKFvdwAp0zsHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4936981&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation analysis of the CpG methylation patterns among different tissues. Scatter plots of percentage methylation values for pairwise comparisons in six libraries [pulmonary alveolar macrophage (PAM) cells, neocortex, spleen, liver, muscle and olfactory epithelium (OE)]. Colours in the scatter plot indicate the number of CpG sites with identical methylation pattern (methylated or non-methylated); yellow denotes large number of correlations, blue denotes lack of correlation and green denotes different methylation patterns. Numbers in the upper right side represent the pairwise Pearson's correlation scores. Histograms on the diagonal are methylation distribution of CpG sites for each sample. Comparisons of global CpG methylation level (A), methylation level of CpG in CGI regions (B) and CGI shore (C).</p></div></div>                        <a id="4936983" scrollto-destination="4936983"></a>
<div id="DSV017F2" data-id="DSV017F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01702.png?Expires=2147483647&Signature=q07aPRHQK4tSt00qHT1OreCHWifBa2HuLRrNm-4MScE0jxMRW3TW~x5owqNOAhE63INaYtHYagqLaKhIJiC3ZGEJLioow8DL2O5vgG52fMMLkbs2OKd4IL6SCmmtFIlofyvzY~Rb0RzPKtpbyjHDvnPyKVVNHzg0XKiU3yJmL-mTvHqFfAfD~QVH5OjhCiNtFBylAZmywn2aQ6kyQhblcPu89G7keEoyy6JSjdWrQkL-PQ5y-5RcJLS9pQo0tmqX8b2LWu2YnJR~FySOG0kygMPgtlkCYpJx1PlAfD3WAVq4qEIRtlnGNGbX58ciXWwt~aAot3VpcXXC~S8JZGnZXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical cluster analysis according to methylation patterns across different tissues. Distance between tissues according to their methylation patterns was estimated by the Ward's minimum variance method." path-from-xml="dsv01702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4936983/dsv01702.tiff" path-from-xml="dsv01702.tiff" target="_blank">View large</a><a section="4936983" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01702.png?Expires=2147483647&Signature=mN6HKEd4EGbOpFIg-ojSWBrChvpey6SEb2H5fCpGWr2HRIhS7syuYdVIMZFT7etqdBKpew3DQGjEh--WykcGXYmjGcqVcPvvQsCqaQBNIfe1QRjeH5cBkdhP1ZXRBejqQaVNsfyq2CIhIDtYlSUGhSQ2WihM1l38tJRwiG9EuFwZ4UwlrgCSXdUVXANoS9f77vzB1xMvvsLIRv04x7yCLSD0oW-7biLMgvFkz2LTdpG87Afp82gCP94gzRLpRMa4ALuRCiJ6XHPAyAcQbhNIitoLtyfrilGVqt8iE1jsCTa8VjhQqyKl-c4OzIhqp90yGZ8Cc7gFs7wHYxaw4r7nUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4936983&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical cluster analysis according to methylation patterns across different tissues. Distance between tissues according to their methylation patterns was estimated by the Ward's minimum variance method.</p></div></div><div content-id="DSV017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01702.png?Expires=2147483647&Signature=q07aPRHQK4tSt00qHT1OreCHWifBa2HuLRrNm-4MScE0jxMRW3TW~x5owqNOAhE63INaYtHYagqLaKhIJiC3ZGEJLioow8DL2O5vgG52fMMLkbs2OKd4IL6SCmmtFIlofyvzY~Rb0RzPKtpbyjHDvnPyKVVNHzg0XKiU3yJmL-mTvHqFfAfD~QVH5OjhCiNtFBylAZmywn2aQ6kyQhblcPu89G7keEoyy6JSjdWrQkL-PQ5y-5RcJLS9pQo0tmqX8b2LWu2YnJR~FySOG0kygMPgtlkCYpJx1PlAfD3WAVq4qEIRtlnGNGbX58ciXWwt~aAot3VpcXXC~S8JZGnZXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical cluster analysis according to methylation patterns across different tissues. Distance between tissues according to their methylation patterns was estimated by the Ward's minimum variance method." path-from-xml="dsv01702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4936983/dsv01702.tiff" path-from-xml="dsv01702.tiff" target="_blank">View large</a><a section="4936983" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01702.png?Expires=2147483647&Signature=mN6HKEd4EGbOpFIg-ojSWBrChvpey6SEb2H5fCpGWr2HRIhS7syuYdVIMZFT7etqdBKpew3DQGjEh--WykcGXYmjGcqVcPvvQsCqaQBNIfe1QRjeH5cBkdhP1ZXRBejqQaVNsfyq2CIhIDtYlSUGhSQ2WihM1l38tJRwiG9EuFwZ4UwlrgCSXdUVXANoS9f77vzB1xMvvsLIRv04x7yCLSD0oW-7biLMgvFkz2LTdpG87Afp82gCP94gzRLpRMa4ALuRCiJ6XHPAyAcQbhNIitoLtyfrilGVqt8iE1jsCTa8VjhQqyKl-c4OzIhqp90yGZ8Cc7gFs7wHYxaw4r7nUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4936983&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical cluster analysis according to methylation patterns across different tissues. Distance between tissues according to their methylation patterns was estimated by the Ward's minimum variance method.</p></div></div><p>The rates of methylation for non-CpG sites (CHG and CHH; H representing A/C/T) varied from 1.63 to 1.71% of the total non-CpGs except for the neocortex, which showed slightly higher amount of methylated non-CpGs (2.3% of the total) compared with other tissues (Table <a reveal-id="DSV017TB2" data-open="DSV017TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSV017F3" data-open="DSV017F3" class="link link-reveal link-table xref-fig">3</a>). This result is consistent with that in a previous report in humans.<sup><a reveal-id="DSV017C64" data-open="DSV017C64" class="link link-ref link-reveal xref-bibr">64</a></sup> We also observed differences in the level of methylation of 2 kb upstream of TSS and CGI regions of non-CpG sites (CHG and CHH) from the PAM and neocortex (Fig. <a reveal-id="DSV017F3" data-open="DSV017F3" class="link link-reveal link-table xref-fig">3</a>B, C and D). Also, the percentage of methylated non-CpG sites in the 2 kb upstream region of TSS of the liver was higher over the spleen, muscle and olfactory epithelium, which was contributed by higher CHG methylation rate of the 2 kb upstream region of TSS in the liver than others. </p>                        <a id="4936988" scrollto-destination="4936988"></a>
<div id="DSV017F3" data-id="DSV017F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01703.png?Expires=2147483647&Signature=X9wEkovsfnpviXLzOoqYgyKK-AEIhVHqlIGR1qIO-0zB0586jd38Ek~Si0eX9R2I8B-mUFvlb4nDmbIqoA4DweDIef3GpDymhPF33oE~Wigt57b6Ju2xTrOYW63BOypoXQlNAERiBgBup78L5ENlxzzzICTl-KmrN2vsSuox-jKbl2g6UKbx8cttHiAP-bRrGKXh3OBDyhUO3arN-bZ6IV7X2iXZFp0QUlwjkPBNITNbpxu7VB8bfxCyWzNgjpr0XLZw01I6rxwfe8ZttqKRwSMowmAjM6yvIuPowhOdMhGULOjH3v352Do9j85plLtw7fOvHJcHNCvAUmlbup2h0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the percentage methylation level of CpG and non-CpG sites among different tissues. Methylation level was indicated according to different criteria including global, 2 kb upstream of TSS (up2 kb) and CGI. Non-CpG methylation was separately evaluated for mCHG and mCHH. PAM, pulmonary alveolar macrophage; OE, olfactory epithelium. Percentage methylation of CpG sites (A), non-CpG (B), CHG (C) and CHH (D) for global, up2 kb and CGI. ** denotes P &lt; 1 × 10−5, * denotes P &lt; 1 × 10−3." path-from-xml="dsv01703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4936988/dsv01703.tiff" path-from-xml="dsv01703.tiff" target="_blank">View large</a><a section="4936988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01703.png?Expires=2147483647&Signature=pFSoKmvLXDFIV-PcNem8g1P19UaR3n0lqqVxYo29KHCHRoLl9m8vgjw9Al9bAv4h8rZwY-OZYy7ueJwjv0F1p2qo2nRHCwqlStZB3dhZIuOHLDtm37~vpYv3r3TgRSUcv-UduPeHZVFCAUHz6I7ouDOcMXMVpKozAEc5pBxVdAaONm0Pzt5ATvL-SN9-kMVeigJK0HDvHv-QKEyxafXRHVtzmiDQGPws0blEnrkfSgnG8G08jqpsRfzgW9a92T4Rb8AGx-saEaFE85UIoLPsvcLRrm-yuCIBQAFfWaUJ0SuGxOlg6zgQ2BJaOrWALalK6zrU7B6Rga0jRdvweodrSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4936988&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the percentage methylation level of CpG and non-CpG sites among different tissues. Methylation level was indicated according to different criteria including global, 2 kb upstream of TSS (up2 kb) and CGI. Non-CpG methylation was separately evaluated for mCHG and mCHH. PAM, pulmonary alveolar macrophage; OE, olfactory epithelium. Percentage methylation of CpG sites (A), non-CpG (B), CHG (C) and CHH (D) for global, up2 kb and CGI. ** denotes <em>P</em> &lt; 1 × 10<sup>−5</sup>, * denotes <em>P</em> &lt; 1 × 10<sup>−3</sup>.</p></div></div><div content-id="DSV017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01703.png?Expires=2147483647&Signature=X9wEkovsfnpviXLzOoqYgyKK-AEIhVHqlIGR1qIO-0zB0586jd38Ek~Si0eX9R2I8B-mUFvlb4nDmbIqoA4DweDIef3GpDymhPF33oE~Wigt57b6Ju2xTrOYW63BOypoXQlNAERiBgBup78L5ENlxzzzICTl-KmrN2vsSuox-jKbl2g6UKbx8cttHiAP-bRrGKXh3OBDyhUO3arN-bZ6IV7X2iXZFp0QUlwjkPBNITNbpxu7VB8bfxCyWzNgjpr0XLZw01I6rxwfe8ZttqKRwSMowmAjM6yvIuPowhOdMhGULOjH3v352Do9j85plLtw7fOvHJcHNCvAUmlbup2h0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the percentage methylation level of CpG and non-CpG sites among different tissues. Methylation level was indicated according to different criteria including global, 2 kb upstream of TSS (up2 kb) and CGI. Non-CpG methylation was separately evaluated for mCHG and mCHH. PAM, pulmonary alveolar macrophage; OE, olfactory epithelium. Percentage methylation of CpG sites (A), non-CpG (B), CHG (C) and CHH (D) for global, up2 kb and CGI. ** denotes P &lt; 1 × 10−5, * denotes P &lt; 1 × 10−3." path-from-xml="dsv01703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4936988/dsv01703.tiff" path-from-xml="dsv01703.tiff" target="_blank">View large</a><a section="4936988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01703.png?Expires=2147483647&Signature=pFSoKmvLXDFIV-PcNem8g1P19UaR3n0lqqVxYo29KHCHRoLl9m8vgjw9Al9bAv4h8rZwY-OZYy7ueJwjv0F1p2qo2nRHCwqlStZB3dhZIuOHLDtm37~vpYv3r3TgRSUcv-UduPeHZVFCAUHz6I7ouDOcMXMVpKozAEc5pBxVdAaONm0Pzt5ATvL-SN9-kMVeigJK0HDvHv-QKEyxafXRHVtzmiDQGPws0blEnrkfSgnG8G08jqpsRfzgW9a92T4Rb8AGx-saEaFE85UIoLPsvcLRrm-yuCIBQAFfWaUJ0SuGxOlg6zgQ2BJaOrWALalK6zrU7B6Rga0jRdvweodrSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4936988&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the percentage methylation level of CpG and non-CpG sites among different tissues. Methylation level was indicated according to different criteria including global, 2 kb upstream of TSS (up2 kb) and CGI. Non-CpG methylation was separately evaluated for mCHG and mCHH. PAM, pulmonary alveolar macrophage; OE, olfactory epithelium. Percentage methylation of CpG sites (A), non-CpG (B), CHG (C) and CHH (D) for global, up2 kb and CGI. ** denotes <em>P</em> &lt; 1 × 10<sup>−5</sup>, * denotes <em>P</em> &lt; 1 × 10<sup>−3</sup>.</p></div></div>                            <h3 scrollto-destination=4936990 id="4936990" class="section-title" data-legacy-id=s3d>3.4. Methylation of CGI and CGI shore</h3>
<p>Correlation of methylation levels among tissues was performed at CGIs and CGI shores (2 kb regions on either side of CGIs), which constitute the largest portion of CpG10 with its CGI features such as CGI, CGI shore and CGI shelf (regions 2 kb further from CGI shores) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). More than half of CpG10 (60.2–62.4%) were mapped to CGIs and the CGI shores. Non-CGI regions within CpG10 ranged from 31.59 to 33.47% among the tested tissues. The rest of CpG10 were mapped to the CGI shelf. The density of CpG10 gradually decreased when moving away from CGIs as reported in humans and mouse.<sup><a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV017C65" data-open="DSV017C65" class="link link-ref link-reveal xref-bibr">65</a></sup> The per cent methylation of CpG10 in CGI features was compared across tissues. The CGI shores showed a much higher degree of methylation than CGI (Fig. <a reveal-id="DSV017F1" data-open="DSV017F1" class="link link-reveal link-table xref-fig">1</a>B and C), which is consistent with previous reports that the methylation level of CGI was lower than that of the CGI shore and global regions.<sup><a reveal-id="DSV017C19" data-open="DSV017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV017C65" data-open="DSV017C65" class="link link-ref link-reveal xref-bibr">65</a></sup> The Pearson's correlation values among the methylation rates of the neocortex, spleen, liver, muscle and olfactory epithelium within CGI shore were from 0.91 to 0.93, which is somewhat lower than those of CGI (0.96–0.97), indicating the presence of tissue-specific methylation despite the small difference (Fig. <a reveal-id="DSV017F1" data-open="DSV017F1" class="link link-reveal link-table xref-fig">1</a>). Results of individual DMCs were summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p>                            <h3 scrollto-destination=4936995 id="4936995" class="section-title" data-legacy-id=s3e>3.5. CpG association to gene features in pig methylome</h3>
<p>The current annotation of the Ensembl gene set for the pig genome shows 21,607 protein-coding genes (<a class="link link-uri" href="http://www.ensembl.org/Sus_scrofa/">www.ensembl.org/Sus_scrofa/</a>). Excluding genes on unplaced scaffolds and mitochondrial DNA, 19,429 genes with chromosomal assignment were used to define the gene features including exon, intron and upstream 2, 5 and 10 kb of TSS (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). On average, 24.1% of total CpG10 was mapped to the gene body (8.08% to exons and 16.2% to introns), and this covered 77.74% of the expected gene body regions which according to the RRBS simulation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Depending on the type of tissues, 6.29–6.52% and 10.24–10.49% of CpG10 were localized 5 and 10 kb upstream of TSS, respectively. The patterns of CpG10 association to gene features were consistent across different tissues (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=4936998 id="4936998" class="section-title" data-legacy-id=s3f>3.6. Comparison of gene number and methylation rate in pig chromosomes</h3>
<p>To address the influence of gene density on methylation rates of pig chromosomes, we compared the observed methylation rates with the number of expected CpG sites and the number of genes for each chromosome (Fig. <a reveal-id="DSV017F4" data-open="DSV017F4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). The percentage methylation rate was similar across chromosomes except for chromosome Y. In most of the chromosomes, there was a strong positive correlation between the total number of CpG sites and genes. The exception was the ratio of CpG sites to gene numbers in chromosomes 3, 6 and 12, which was higher than the ratio in other chromosomes and was reversed for chromosome 13. However, as expected, the GC percentage of the chromosomes was correlated with the CpG methylation for each chromosome (Fig. <a reveal-id="DSV017F4" data-open="DSV017F4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). We hypothesized that chromosomes with higher gene numbers could result in lower methylation level than chromosomes with lower gene density, because promoter and CGI regions showed lower levels of CpG methylation than other genomic features (Fig. <a reveal-id="DSV017F4" data-open="DSV017F4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>) and the CpG density of those regions was higher than the average. However, no correlation was observed between gene number and CpG methylation levels of chromosomes in our analysis. The observed methylation rates were similar among different tissues except for 3D4/2 cells, especially for the upstream 2 and 5 kb regions of TSS (Fig. <a reveal-id="DSV017F4" data-open="DSV017F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="4937002" scrollto-destination="4937002"></a>
<div id="DSV017F4" data-id="DSV017F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01704.png?Expires=2147483647&Signature=IK8tNlzLXgbroHs2wZ2uxtq1I~O1yAn1oR4w25iqd1pT2d9UbIV-jUThY5R0IaJ0R-aBtBs7bu0QqyiuXa4QZeUJkxNAi-K0Hni~EvePez1nw7y5gBu2rd~aHHCcL5OUmHBgeRj~5ICoU-k6289kj-N9yGt2DGX4nlXPmhOyFatTaGnPlfuguWJBm4Tvh2B5GKIbElGgpnkJS~1ydqKQioMbjGfctKWzj9gXg2j5ir7GiODB5Eb1IV-LHTlArJuDr1oP6OjVQ0P22MYoyxp~Bhe-qxpfXpFUArMF~QjsYPmhO5pDR6~Q3EGGE1LxuPpmaKF9uatGcMlqAYhRXZ4Wjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the methylation levels in gene-related regions in relation to the numbers of genes and CpGs on individual pig chromosomes. The gene annotation data of the pig genome were obtained from Ensemble. The total number of CpG for each chromosome was obtained from the current pig genome assembly (Sscrofa10.2). Gene body (A), 2 kb (B) and 5 kb (C) upstream regions of TSS. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4937002/dsv01704.tiff" path-from-xml="dsv01704.tiff" target="_blank">View large</a><a section="4937002" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01704.png?Expires=2147483647&Signature=dTbMpH~U-ai9HIQi~LvrEhK~wcJ9U6QqjEeo7W1e9DrzBLQxrICb4BNUprYwLJe~d3LKwwrILcq2qlRlKD7HbO~BtVjNSre3j7iQAHZ4jehHHqo79XC~yupSKZreEm8HrfT8BhGYiirOxz6AxN1Y3UDEEvnbYuGkBo2JlqN9NJo1gTuE4TcOdBwdpcBfkDX9wg4FDHId1hWeR7oxymUmMmhEnSRK6BtxtijLXJCGrViGtbtIEiZT9gdjJ9Vrwiqknhj2yw2LzbQlrILFxF2mbCHOf~n5ffnLpaMEBj~RIUZupoICwxAFElNCkid6e5QtHy2Q7WjLWi29GluAqRFleg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4937002&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the methylation levels in gene-related regions in relation to the numbers of genes and CpGs on individual pig chromosomes. The gene annotation data of the pig genome were obtained from Ensemble. The total number of CpG for each chromosome was obtained from the current pig genome assembly (Sscrofa10.2). Gene body (A), 2 kb (B) and 5 kb (C) upstream regions of TSS. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV017F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01704.png?Expires=2147483647&Signature=IK8tNlzLXgbroHs2wZ2uxtq1I~O1yAn1oR4w25iqd1pT2d9UbIV-jUThY5R0IaJ0R-aBtBs7bu0QqyiuXa4QZeUJkxNAi-K0Hni~EvePez1nw7y5gBu2rd~aHHCcL5OUmHBgeRj~5ICoU-k6289kj-N9yGt2DGX4nlXPmhOyFatTaGnPlfuguWJBm4Tvh2B5GKIbElGgpnkJS~1ydqKQioMbjGfctKWzj9gXg2j5ir7GiODB5Eb1IV-LHTlArJuDr1oP6OjVQ0P22MYoyxp~Bhe-qxpfXpFUArMF~QjsYPmhO5pDR6~Q3EGGE1LxuPpmaKF9uatGcMlqAYhRXZ4Wjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the methylation levels in gene-related regions in relation to the numbers of genes and CpGs on individual pig chromosomes. The gene annotation data of the pig genome were obtained from Ensemble. The total number of CpG for each chromosome was obtained from the current pig genome assembly (Sscrofa10.2). Gene body (A), 2 kb (B) and 5 kb (C) upstream regions of TSS. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4937002/dsv01704.tiff" path-from-xml="dsv01704.tiff" target="_blank">View large</a><a section="4937002" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01704.png?Expires=2147483647&Signature=dTbMpH~U-ai9HIQi~LvrEhK~wcJ9U6QqjEeo7W1e9DrzBLQxrICb4BNUprYwLJe~d3LKwwrILcq2qlRlKD7HbO~BtVjNSre3j7iQAHZ4jehHHqo79XC~yupSKZreEm8HrfT8BhGYiirOxz6AxN1Y3UDEEvnbYuGkBo2JlqN9NJo1gTuE4TcOdBwdpcBfkDX9wg4FDHId1hWeR7oxymUmMmhEnSRK6BtxtijLXJCGrViGtbtIEiZT9gdjJ9Vrwiqknhj2yw2LzbQlrILFxF2mbCHOf~n5ffnLpaMEBj~RIUZupoICwxAFElNCkid6e5QtHy2Q7WjLWi29GluAqRFleg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4937002&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the methylation levels in gene-related regions in relation to the numbers of genes and CpGs on individual pig chromosomes. The gene annotation data of the pig genome were obtained from Ensemble. The total number of CpG for each chromosome was obtained from the current pig genome assembly (Sscrofa10.2). Gene body (A), 2 kb (B) and 5 kb (C) upstream regions of TSS. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=4937005 id="4937005" class="section-title" data-legacy-id=s3g>3.7. Methylation patterns of imprinted genes from RRBS analysis of the pig genome</h3>
<p>We analysed the RRBS reads corresponding to 22 out of 30 genes known for genomic imprinting in pigs (<a class="link link-uri" href="http://www.geneimprint.com/">http://www.geneimprint.com/</a>), which had annotation of genomic location in Sscrofa10.2 (data not shown). Among them, eight genes, <em>DIO3, DIRAS3, MEG3, MEST, NAP1L5, NECD, NNAT, PHLDA2</em>, satisfied our criteria to be selected as candidates for the analysis of imprinted genes, which includes clear genomic locations including gene bodies and the upstream 2 and 2–10 kb regions from TSS in the current pig genome assembly (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Interestingly, the levels of CpG methylation of the eight known imprinted genes were significantly higher than those of randomly selected non-imprinted genes (<em>GAPDH, COX5B</em> and <em>ACTA1</em>) at gene bodies and upstream 2 kb of TSS regions in pig tissue samples (Fig. <a reveal-id="DSV017F5" data-open="DSV017F5" class="link link-reveal link-table xref-fig">5</a>). The difference was less consistent in the upstream 2–10 kb region of TSS. In addition, we found a difference in the level of methylation among different tissues. Especially, the difference was the largest in 3D4/2 cells. For example, CpGs in the gene bodies of <em>DIO3, DIRAS3, MEG3</em> and <em>NNAT</em> were highly methylated comparing with other tissues. The methylation patterns of upstream 2 and 2–10 kb regions from TSS were also similar (Fig. <a reveal-id="DSV017F5" data-open="DSV017F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="4937009" scrollto-destination="4937009"></a>
<div id="DSV017F5" data-id="DSV017F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01705.png?Expires=2147483647&Signature=P99l1AvAwKZ2AKbR~JI9cpAx69tXYGtcNDFbu8j1IeL11MnW9nxU0MRUDvBFxuQD24keZr3zPJY1JXsFiKsMVrgkdI6BmkImHvTnKwEOk4jod2gZR7garwU66RqDTkP4uaFcZBmtc1I43qqa96DMm4thLPcW4fQuydmWmgcNlZCda8Z9K8ZPUtTGKPUjZV4SwR9tuISA1QYsfXVsznBwKKL8N37Mm21G5LuQRxtUYyvnLrDl9NKbM2yYc4O9ZHbX8K3GOIhBvERhthmhPmi34vV~P6RcbI5A30GBzx-S-ZMEqIMPaPrnuGezwt2rxxCMtIFWbll8vnfy7zAkM7MvXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The pattern of CpG methylation of imprinted genes from six tissues in pigs. The result was visualized by heat map. Gene body, exon and intron; up2 kb and up2–10 kb, upstream 2 kb and 2–10 kb regions from TSS, respectively." path-from-xml="dsv01705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4937009/dsv01705.tiff" path-from-xml="dsv01705.tiff" target="_blank">View large</a><a section="4937009" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01705.png?Expires=2147483647&Signature=WZl-RPGjCcgComShMo-GsM1PI5on4GQcWSHlVdzZYuoC5jikUSPC2iURlQsFqoZONQkukiD0Q2ZG1LgiUSKpgdX1cvJqnpoCZ3RuBcrTheLetH4b2JM5pv7pZXZFgx6N1BOipQrYeJB2Cf4J2vvXGaxa8vBCOTtIB0GAr00teGBCxHQIZt~6UtOYdaPLT7BnORDNa49C5BTv2uo8KSlgxnRDJW7RHlLCzJnhT7ZfSVsqmitVw9ur5LZmMDzZo45i2au3-VjycEf-3EIAzS-JHDdHaTID3qGGvJ5OoIuTxox7fU7UoiK9-8iAfrVsE~U1I-1dNxaypcIiEduJC03r7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4937009&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The pattern of CpG methylation of imprinted genes from six tissues in pigs. The result was visualized by heat map. Gene body, exon and intron; up2 kb and up2–10 kb, upstream 2 kb and 2–10 kb regions from TSS, respectively.</p></div></div><div content-id="DSV017F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01705.png?Expires=2147483647&Signature=P99l1AvAwKZ2AKbR~JI9cpAx69tXYGtcNDFbu8j1IeL11MnW9nxU0MRUDvBFxuQD24keZr3zPJY1JXsFiKsMVrgkdI6BmkImHvTnKwEOk4jod2gZR7garwU66RqDTkP4uaFcZBmtc1I43qqa96DMm4thLPcW4fQuydmWmgcNlZCda8Z9K8ZPUtTGKPUjZV4SwR9tuISA1QYsfXVsznBwKKL8N37Mm21G5LuQRxtUYyvnLrDl9NKbM2yYc4O9ZHbX8K3GOIhBvERhthmhPmi34vV~P6RcbI5A30GBzx-S-ZMEqIMPaPrnuGezwt2rxxCMtIFWbll8vnfy7zAkM7MvXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The pattern of CpG methylation of imprinted genes from six tissues in pigs. The result was visualized by heat map. Gene body, exon and intron; up2 kb and up2–10 kb, upstream 2 kb and 2–10 kb regions from TSS, respectively." path-from-xml="dsv01705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4937009/dsv01705.tiff" path-from-xml="dsv01705.tiff" target="_blank">View large</a><a section="4937009" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01705.png?Expires=2147483647&Signature=WZl-RPGjCcgComShMo-GsM1PI5on4GQcWSHlVdzZYuoC5jikUSPC2iURlQsFqoZONQkukiD0Q2ZG1LgiUSKpgdX1cvJqnpoCZ3RuBcrTheLetH4b2JM5pv7pZXZFgx6N1BOipQrYeJB2Cf4J2vvXGaxa8vBCOTtIB0GAr00teGBCxHQIZt~6UtOYdaPLT7BnORDNa49C5BTv2uo8KSlgxnRDJW7RHlLCzJnhT7ZfSVsqmitVw9ur5LZmMDzZo45i2au3-VjycEf-3EIAzS-JHDdHaTID3qGGvJ5OoIuTxox7fU7UoiK9-8iAfrVsE~U1I-1dNxaypcIiEduJC03r7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4937009&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The pattern of CpG methylation of imprinted genes from six tissues in pigs. The result was visualized by heat map. Gene body, exon and intron; up2 kb and up2–10 kb, upstream 2 kb and 2–10 kb regions from TSS, respectively.</p></div></div>                            <h3 scrollto-destination=4937017 id="4937017" class="section-title" data-legacy-id=s3h>3.8. Identification of tDMCs</h3>
<p>DMCs were identified using methylKit from six samples (<em>q</em>-value &lt;0.01, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7 and Table S7</a>). Among the DMCs, tissue specifically methylated CpGs were selected. A total of 63,482 and 32,410 tissue-specific DMCs (tDMCs) were identified from gene bodies and upstream 10 kb region from TSS, respectively, under the cut-off value of 30% methylation difference. Excluding the PAM cell library, only 4,780 and 1,680 tDMCs were identified from gene bodies and upstream 10 kb regions from TSS, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). Among the tissue samples, spleen-specific DMCs were the largest and occupied 28.7% of the total and followed by the liver (24.1%) and olfactory epithelium (23.1%). Neocortex- and muscle-specific libraries constitute 9.3 and 14.8% of the total tDMCs, respectively (Fig. <a reveal-id="DSV017F6" data-open="DSV017F6" class="link link-reveal link-table xref-fig">6</a>). The tDMC profile including chromosomal positions was summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>. </p>                        <a id="4937021" scrollto-destination="4937021"></a>
<div id="DSV017F6" data-id="DSV017F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01706.png?Expires=2147483647&Signature=xsYkrg6ZVx32i0eNCYbYBfnLjt1kDZGrBzQHVrGOs9Af4POQmrwNOHx6OGSzDDa4yroRTHn9mVmOLQ6IABGUuPph-Ku~bwkpZBOa9C8mTtMPdM7u87BynbVyT1oN-lAHODR4Ibg0O9Jjd7Jv32J9-Ly6SQvYWYT0yWvzsEhHG3RXphJ5LfNbLjmI04rw2EjHAl9XGN82NkcDxSs7sAl-DE89DigAEIyMExO-913SyULtHOItbx92bp6b2cxkNZ97-e5VPkcgbdKcS3LGIqAN-0Mk~DLFd7RGXAhqVnLXisN7yzsgR3Yyi8-zBRbecUxmKEFahKDBVuW4aYrwOy7zuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number of tissue-specific differentially methylated cytosine (tDMCs) from five pig tissues. Venn diagrams showing the number of positions of tDMCs from each tissue. The positions of tDMCs depend on tissue type were visualized by Venn diagram. NC: neocortex; OE: olfactory epithelium; LV: liver; MS: muscle; SP: spleen. The 3D4/2 cells were not represented here. Detailed positions of tDMC were summarized in the Supplementary Table S8. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01706.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4937021/dsv01706.tiff" path-from-xml="dsv01706.tiff" target="_blank">View large</a><a section="4937021" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01706.png?Expires=2147483647&Signature=lJNGYQ95H9XdVSp4ejoOJYxrpWIIjSl4WA3y6uo5nqmr0zKlO6TNkvHeFa7i3e2dtu8ulZcFpXy48g8utm1AIOP7gJ56DqC4ZCIjCZSh2Xyd~uwsIr6HbjIFJW0a8Bd0g5ygYKUaauf5BaTy3REv~X6VU52nzudtqvDCEZW5823fQ9lTN4vR2Hn5q2ia~Du2chBObhmAVprIq-sHoABLDd5ZelQBqv6w54YmAu~qbBbSYE-sKHWhveAr0hcuGdPa7~8Zc3A2v3jWXK3S2PCd6eGHBaxVL7jTAbQT8EvIjj~zJMF9Zt89JFRVaiNj0j2ogrnt7rybkTd0nE2rtZATdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4937021&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01706.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number of tissue-specific differentially methylated cytosine (tDMCs) from five pig tissues. Venn diagrams showing the number of positions of tDMCs from each tissue. The positions of tDMCs depend on tissue type were visualized by Venn diagram. NC: neocortex; OE: olfactory epithelium; LV: liver; MS: muscle; SP: spleen. The 3D4/2 cells were not represented here. Detailed positions of tDMC were summarized in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV017F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/m_dsv01706.png?Expires=2147483647&Signature=xsYkrg6ZVx32i0eNCYbYBfnLjt1kDZGrBzQHVrGOs9Af4POQmrwNOHx6OGSzDDa4yroRTHn9mVmOLQ6IABGUuPph-Ku~bwkpZBOa9C8mTtMPdM7u87BynbVyT1oN-lAHODR4Ibg0O9Jjd7Jv32J9-Ly6SQvYWYT0yWvzsEhHG3RXphJ5LfNbLjmI04rw2EjHAl9XGN82NkcDxSs7sAl-DE89DigAEIyMExO-913SyULtHOItbx92bp6b2cxkNZ97-e5VPkcgbdKcS3LGIqAN-0Mk~DLFd7RGXAhqVnLXisN7yzsgR3Yyi8-zBRbecUxmKEFahKDBVuW4aYrwOy7zuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number of tissue-specific differentially methylated cytosine (tDMCs) from five pig tissues. Venn diagrams showing the number of positions of tDMCs from each tissue. The positions of tDMCs depend on tissue type were visualized by Venn diagram. NC: neocortex; OE: olfactory epithelium; LV: liver; MS: muscle; SP: spleen. The 3D4/2 cells were not represented here. Detailed positions of tDMC were summarized in the Supplementary Table S8. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01706.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4937021/dsv01706.tiff" path-from-xml="dsv01706.tiff" target="_blank">View large</a><a section="4937021" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv01706.png?Expires=2147483647&Signature=lJNGYQ95H9XdVSp4ejoOJYxrpWIIjSl4WA3y6uo5nqmr0zKlO6TNkvHeFa7i3e2dtu8ulZcFpXy48g8utm1AIOP7gJ56DqC4ZCIjCZSh2Xyd~uwsIr6HbjIFJW0a8Bd0g5ygYKUaauf5BaTy3REv~X6VU52nzudtqvDCEZW5823fQ9lTN4vR2Hn5q2ia~Du2chBObhmAVprIq-sHoABLDd5ZelQBqv6w54YmAu~qbBbSYE-sKHWhveAr0hcuGdPa7~8Zc3A2v3jWXK3S2PCd6eGHBaxVL7jTAbQT8EvIjj~zJMF9Zt89JFRVaiNj0j2ogrnt7rybkTd0nE2rtZATdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4937021&ar=347949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01706.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number of tissue-specific differentially methylated cytosine (tDMCs) from five pig tissues. Venn diagrams showing the number of positions of tDMCs from each tissue. The positions of tDMCs depend on tissue type were visualized by Venn diagram. NC: neocortex; OE: olfactory epithelium; LV: liver; MS: muscle; SP: spleen. The 3D4/2 cells were not represented here. Detailed positions of tDMC were summarized in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=4937024 id="4937024" class="section-title" data-legacy-id=s3i>3.9. Functional annotation of tDMCs</h3>
<p>Identified tDMCs were mapped to 1,897 genes on the basis of their genomic location (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). GO analysis was performed for tDMCs in each tissue (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S9 and S10</a>). Statistical significance of GO terms to tDMCs was evaluated by the hypergeometric test with a multiple correction. The results showed that some of highly ranked GO were matched with characteristics of the tissues. In the neocortex, ‘olfactory behaviour’, ‘positive regulation of neural precursor cell proliferation’, ‘norepinephrine biosynthetic process’, ‘brain development’ and other neuronal signal transduction-related GO terms were significant (<em>q</em> &lt; 0.05). The GO terms for functions of the liver were significantly associated, including ‘enzyme regulator activity’, ‘glycolytic process’, ‘positive regulation of heparan sulfate proteoglycan biosynthetic process’, ‘carbohydrate metabolic process’ and other metabolic activity-related terms. Similarly, ‘actin cytoskeleton’ and ‘mesodermal cell fate specification’ were specific GO terms for the muscle. ‘Leucocyte-mediated cytotoxicity’, ‘focal adhesion’ and ‘cell migration’ were significant for spleen-specific, DMCs-related genes. For PAM cells, GO terms were enriched with transcription regulation-associated terms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). For the olfactory epithelium, we were unable to find significant GO terms matching to the biological characteristics of the tissue, which may be due to the reason that the tissue constitutes of only a small number of olfactory function-related cells together with epithelial cells.</p>                            <h3 scrollto-destination=4937028 id="4937028" class="section-title" data-legacy-id=s3j>3.10. Correlation between tDMCs and gene expression</h3>
<p>To evaluate the influence of DNA methylation on gene expression, we analysed the association between tDMCs and gene expression profiles of the frontal cortex, liver, biceps femoris and spleen. DEGs across four tissues were identified using the GEO2R plugin (<em>P</em> &lt; 0.05) from a microarrary dataset of the GEO database. First, we selected genes (<em>n</em> = 42) present in both tDMCs-associated groups from RRBS and DEG groups from microarrary transcriptome analyses. The Pearson's correlation test showed that a total of 55 tDMCs which associated with 27 genes showed strong negative correlation between the levels of gene expression and CpG methylation (Table <a reveal-id="DSV017TB3" data-open="DSV017TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="4937032" scrollto-destination="4937032"></a>
<div content-id="DSV017TB3" class="table-modal"><div class="table-wrap"><div id="DSV017TB3" class="table-wrap-title" data-id="DSV017TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Correlation between tDMCs and the level of gene expression from the neocortex, muscle and spleen</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissue type </th><th>Gene </th><th>Chr<sup>a</sup> </th><th>Position </th><th colspan="2">Class<sup>b</sup><hr /> </th><th>M<sup>c</sup> </th><th>E<sup>d</sup> </th><th>Pearson's corr.<sup>e</sup> </th></tr></thead><tbody><tr><td rowspan="15">Neocortex </td><td rowspan="11">NRXN2 </td><td>2 </td><td>6587361 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.897000684 </td></tr><tr><td>2 </td><td>6635041 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.838015353 </td></tr><tr><td>2 </td><td>6635090 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.886097969 </td></tr><tr><td>2 </td><td>6635002 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.828919384 </td></tr><tr><td>2 </td><td>6634986 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.902681422 </td></tr><tr><td>2 </td><td>6634955 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729923 </td></tr><tr><td>2 </td><td>6634980 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.874609127 </td></tr><tr><td>2 </td><td>6634930 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.720944026 </td></tr><tr><td>2 </td><td>6634961 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.837225041 </td></tr><tr><td>2 </td><td>6634934 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.785419851 </td></tr><tr><td>2 </td><td>6634951 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.865604511 </td></tr><tr><td rowspan="2">MAPT </td><td>12 </td><td>17133831 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.926451406 </td></tr><tr><td>12 </td><td>17133834 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.922834214 </td></tr><tr><td rowspan="2">SYNE1 </td><td>1 </td><td>16124463 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.80723621 </td></tr><tr><td>1 </td><td>16124489 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.750224969 </td></tr><tr><td rowspan="18">Muscle </td><td>CCDC88C </td><td>7 </td><td>119491407 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.936174813 </td></tr><tr><td rowspan="2">EXOC2 </td><td>7 </td><td>75292 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.851066176 </td></tr><tr><td>7 </td><td>75340 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.832158353 </td></tr><tr><td rowspan="2">FGR </td><td>6 </td><td>78329214 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.802870198 </td></tr><tr><td>6 </td><td>78336406 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.755515526 </td></tr><tr><td>MAP2K7 </td><td>2 </td><td>71712421 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.857718538 </td></tr><tr><td rowspan="4">NR5A2 </td><td>10 </td><td>27361479 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957673269 </td></tr><tr><td>10 </td><td>27361475 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.95434547 </td></tr><tr><td>10 </td><td>27361468 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.921209644 </td></tr><tr><td>10 </td><td>27361450 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.960461808 </td></tr><tr><td>PCOLCE </td><td>3 </td><td>7921803 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.995207087 </td></tr><tr><td>RAB22A </td><td>17 </td><td>65753356 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.884084122 </td></tr><tr><td>SLA-7 </td><td>7 </td><td>27608918 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.914258584 </td></tr><tr><td>TUBGCP6 </td><td>5 </td><td>110682055 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.90531831 </td></tr><tr><td rowspan="2">VGLL2 </td><td>1 </td><td>49880654 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.753634499 </td></tr><tr><td>1 </td><td>49880666 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.805639146 </td></tr><tr><td rowspan="2">XPO4 </td><td>11 </td><td>268954 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.909143259 </td></tr><tr><td>11 </td><td>269035 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.914121809 </td></tr><tr><td rowspan="22">Spleen </td><td>ICAM-1 </td><td>2 </td><td>69496888 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.755261243 </td></tr><tr><td rowspan="2">ALDH3A1 </td><td>12 </td><td>62650896 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.788071349 </td></tr><tr><td>12 </td><td>62650898 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.752480115 </td></tr><tr><td>KIRREL3 </td><td>9 </td><td>59344774 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957798851 </td></tr><tr><td>GTF2IRD1 </td><td>3 </td><td>11271020 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.917047466 </td></tr><tr><td>UBA52 </td><td>2 </td><td>59052151 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.953385635 </td></tr><tr><td>CD163 </td><td>14 </td><td>153630638 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.860727255 </td></tr><tr><td>HMHA1 </td><td>2 </td><td>77641888 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.861581753 </td></tr><tr><td rowspan="4">RPS21 </td><td>17 </td><td>69165427 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.829232698 </td></tr><tr><td>17 </td><td>69165425 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.811905734 </td></tr><tr><td>17 </td><td>69169540 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.852778602 </td></tr><tr><td>17 </td><td>69169556 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.743101278 </td></tr><tr><td>SLC9A3R2 </td><td>3 </td><td>42231210 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.752184061 </td></tr><tr><td>ZNF646 </td><td>3 </td><td>17732629 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.828529507 </td></tr><tr><td rowspan="5">ADAMTS4 </td><td>4 </td><td>97117737 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729052 </td></tr><tr><td>4 </td><td>97117751 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.8029031 </td></tr><tr><td>4 </td><td>97117778 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.806509102 </td></tr><tr><td>4 </td><td>97117512 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.763107181 </td></tr><tr><td>4 </td><td>97117534 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.814870957 </td></tr><tr><td rowspan="3">TBKBP1 </td><td>12 </td><td>23838338 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.89415394 </td></tr><tr><td>12 </td><td>23838110 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.975048371 </td></tr><tr><td>12 </td><td>23838349 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.821418354 </td></tr></tbody></table></div><div content-id="DSV017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissue type </th><th>Gene </th><th>Chr<sup>a</sup> </th><th>Position </th><th colspan="2">Class<sup>b</sup><hr /> </th><th>M<sup>c</sup> </th><th>E<sup>d</sup> </th><th>Pearson's corr.<sup>e</sup> </th></tr></thead><tbody><tr><td rowspan="15">Neocortex </td><td rowspan="11">NRXN2 </td><td>2 </td><td>6587361 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.897000684 </td></tr><tr><td>2 </td><td>6635041 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.838015353 </td></tr><tr><td>2 </td><td>6635090 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.886097969 </td></tr><tr><td>2 </td><td>6635002 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.828919384 </td></tr><tr><td>2 </td><td>6634986 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.902681422 </td></tr><tr><td>2 </td><td>6634955 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729923 </td></tr><tr><td>2 </td><td>6634980 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.874609127 </td></tr><tr><td>2 </td><td>6634930 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.720944026 </td></tr><tr><td>2 </td><td>6634961 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.837225041 </td></tr><tr><td>2 </td><td>6634934 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.785419851 </td></tr><tr><td>2 </td><td>6634951 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.865604511 </td></tr><tr><td rowspan="2">MAPT </td><td>12 </td><td>17133831 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.926451406 </td></tr><tr><td>12 </td><td>17133834 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.922834214 </td></tr><tr><td rowspan="2">SYNE1 </td><td>1 </td><td>16124463 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.80723621 </td></tr><tr><td>1 </td><td>16124489 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.750224969 </td></tr><tr><td rowspan="18">Muscle </td><td>CCDC88C </td><td>7 </td><td>119491407 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.936174813 </td></tr><tr><td rowspan="2">EXOC2 </td><td>7 </td><td>75292 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.851066176 </td></tr><tr><td>7 </td><td>75340 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.832158353 </td></tr><tr><td rowspan="2">FGR </td><td>6 </td><td>78329214 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.802870198 </td></tr><tr><td>6 </td><td>78336406 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.755515526 </td></tr><tr><td>MAP2K7 </td><td>2 </td><td>71712421 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.857718538 </td></tr><tr><td rowspan="4">NR5A2 </td><td>10 </td><td>27361479 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957673269 </td></tr><tr><td>10 </td><td>27361475 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.95434547 </td></tr><tr><td>10 </td><td>27361468 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.921209644 </td></tr><tr><td>10 </td><td>27361450 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.960461808 </td></tr><tr><td>PCOLCE </td><td>3 </td><td>7921803 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.995207087 </td></tr><tr><td>RAB22A </td><td>17 </td><td>65753356 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.884084122 </td></tr><tr><td>SLA-7 </td><td>7 </td><td>27608918 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.914258584 </td></tr><tr><td>TUBGCP6 </td><td>5 </td><td>110682055 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.90531831 </td></tr><tr><td rowspan="2">VGLL2 </td><td>1 </td><td>49880654 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.753634499 </td></tr><tr><td>1 </td><td>49880666 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.805639146 </td></tr><tr><td rowspan="2">XPO4 </td><td>11 </td><td>268954 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.909143259 </td></tr><tr><td>11 </td><td>269035 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.914121809 </td></tr><tr><td rowspan="22">Spleen </td><td>ICAM-1 </td><td>2 </td><td>69496888 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.755261243 </td></tr><tr><td rowspan="2">ALDH3A1 </td><td>12 </td><td>62650896 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.788071349 </td></tr><tr><td>12 </td><td>62650898 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.752480115 </td></tr><tr><td>KIRREL3 </td><td>9 </td><td>59344774 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957798851 </td></tr><tr><td>GTF2IRD1 </td><td>3 </td><td>11271020 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.917047466 </td></tr><tr><td>UBA52 </td><td>2 </td><td>59052151 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.953385635 </td></tr><tr><td>CD163 </td><td>14 </td><td>153630638 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.860727255 </td></tr><tr><td>HMHA1 </td><td>2 </td><td>77641888 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.861581753 </td></tr><tr><td rowspan="4">RPS21 </td><td>17 </td><td>69165427 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.829232698 </td></tr><tr><td>17 </td><td>69165425 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.811905734 </td></tr><tr><td>17 </td><td>69169540 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.852778602 </td></tr><tr><td>17 </td><td>69169556 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.743101278 </td></tr><tr><td>SLC9A3R2 </td><td>3 </td><td>42231210 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.752184061 </td></tr><tr><td>ZNF646 </td><td>3 </td><td>17732629 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.828529507 </td></tr><tr><td rowspan="5">ADAMTS4 </td><td>4 </td><td>97117737 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729052 </td></tr><tr><td>4 </td><td>97117751 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.8029031 </td></tr><tr><td>4 </td><td>97117778 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.806509102 </td></tr><tr><td>4 </td><td>97117512 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.763107181 </td></tr><tr><td>4 </td><td>97117534 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.814870957 </td></tr><tr><td rowspan="3">TBKBP1 </td><td>12 </td><td>23838338 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.89415394 </td></tr><tr><td>12 </td><td>23838110 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.975048371 </td></tr><tr><td>12 </td><td>23838349 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.821418354 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Chromosome.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>b</sup>Class represents genomic location of tDMC; exon and intron region, 10 kb upstream region from TSS and CGI Shore were marked as Genebody, UP10K and CGI shore, respectively.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>c</sup>Methylation pattern, + and − represent higher and lower methylation over other tissues, respectively.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>d</sup>Expression level, + and − represent higher and lower expression levels over other tissues, respectively.</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>e</sup>Pearson's correlation score.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4937032" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV017TB3" class="table-wrap-title" data-id="DSV017TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Correlation between tDMCs and the level of gene expression from the neocortex, muscle and spleen</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissue type </th><th>Gene </th><th>Chr<sup>a</sup> </th><th>Position </th><th colspan="2">Class<sup>b</sup><hr /> </th><th>M<sup>c</sup> </th><th>E<sup>d</sup> </th><th>Pearson's corr.<sup>e</sup> </th></tr></thead><tbody><tr><td rowspan="15">Neocortex </td><td rowspan="11">NRXN2 </td><td>2 </td><td>6587361 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.897000684 </td></tr><tr><td>2 </td><td>6635041 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.838015353 </td></tr><tr><td>2 </td><td>6635090 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.886097969 </td></tr><tr><td>2 </td><td>6635002 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.828919384 </td></tr><tr><td>2 </td><td>6634986 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.902681422 </td></tr><tr><td>2 </td><td>6634955 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729923 </td></tr><tr><td>2 </td><td>6634980 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.874609127 </td></tr><tr><td>2 </td><td>6634930 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.720944026 </td></tr><tr><td>2 </td><td>6634961 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.837225041 </td></tr><tr><td>2 </td><td>6634934 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.785419851 </td></tr><tr><td>2 </td><td>6634951 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.865604511 </td></tr><tr><td rowspan="2">MAPT </td><td>12 </td><td>17133831 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.926451406 </td></tr><tr><td>12 </td><td>17133834 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.922834214 </td></tr><tr><td rowspan="2">SYNE1 </td><td>1 </td><td>16124463 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.80723621 </td></tr><tr><td>1 </td><td>16124489 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.750224969 </td></tr><tr><td rowspan="18">Muscle </td><td>CCDC88C </td><td>7 </td><td>119491407 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.936174813 </td></tr><tr><td rowspan="2">EXOC2 </td><td>7 </td><td>75292 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.851066176 </td></tr><tr><td>7 </td><td>75340 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.832158353 </td></tr><tr><td rowspan="2">FGR </td><td>6 </td><td>78329214 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.802870198 </td></tr><tr><td>6 </td><td>78336406 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.755515526 </td></tr><tr><td>MAP2K7 </td><td>2 </td><td>71712421 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.857718538 </td></tr><tr><td rowspan="4">NR5A2 </td><td>10 </td><td>27361479 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957673269 </td></tr><tr><td>10 </td><td>27361475 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.95434547 </td></tr><tr><td>10 </td><td>27361468 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.921209644 </td></tr><tr><td>10 </td><td>27361450 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.960461808 </td></tr><tr><td>PCOLCE </td><td>3 </td><td>7921803 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.995207087 </td></tr><tr><td>RAB22A </td><td>17 </td><td>65753356 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.884084122 </td></tr><tr><td>SLA-7 </td><td>7 </td><td>27608918 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.914258584 </td></tr><tr><td>TUBGCP6 </td><td>5 </td><td>110682055 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.90531831 </td></tr><tr><td rowspan="2">VGLL2 </td><td>1 </td><td>49880654 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.753634499 </td></tr><tr><td>1 </td><td>49880666 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.805639146 </td></tr><tr><td rowspan="2">XPO4 </td><td>11 </td><td>268954 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.909143259 </td></tr><tr><td>11 </td><td>269035 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.914121809 </td></tr><tr><td rowspan="22">Spleen </td><td>ICAM-1 </td><td>2 </td><td>69496888 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.755261243 </td></tr><tr><td rowspan="2">ALDH3A1 </td><td>12 </td><td>62650896 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.788071349 </td></tr><tr><td>12 </td><td>62650898 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.752480115 </td></tr><tr><td>KIRREL3 </td><td>9 </td><td>59344774 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957798851 </td></tr><tr><td>GTF2IRD1 </td><td>3 </td><td>11271020 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.917047466 </td></tr><tr><td>UBA52 </td><td>2 </td><td>59052151 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.953385635 </td></tr><tr><td>CD163 </td><td>14 </td><td>153630638 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.860727255 </td></tr><tr><td>HMHA1 </td><td>2 </td><td>77641888 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.861581753 </td></tr><tr><td rowspan="4">RPS21 </td><td>17 </td><td>69165427 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.829232698 </td></tr><tr><td>17 </td><td>69165425 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.811905734 </td></tr><tr><td>17 </td><td>69169540 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.852778602 </td></tr><tr><td>17 </td><td>69169556 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.743101278 </td></tr><tr><td>SLC9A3R2 </td><td>3 </td><td>42231210 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.752184061 </td></tr><tr><td>ZNF646 </td><td>3 </td><td>17732629 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.828529507 </td></tr><tr><td rowspan="5">ADAMTS4 </td><td>4 </td><td>97117737 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729052 </td></tr><tr><td>4 </td><td>97117751 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.8029031 </td></tr><tr><td>4 </td><td>97117778 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.806509102 </td></tr><tr><td>4 </td><td>97117512 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.763107181 </td></tr><tr><td>4 </td><td>97117534 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.814870957 </td></tr><tr><td rowspan="3">TBKBP1 </td><td>12 </td><td>23838338 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.89415394 </td></tr><tr><td>12 </td><td>23838110 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.975048371 </td></tr><tr><td>12 </td><td>23838349 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.821418354 </td></tr></tbody></table></div><div content-id="DSV017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissue type </th><th>Gene </th><th>Chr<sup>a</sup> </th><th>Position </th><th colspan="2">Class<sup>b</sup><hr /> </th><th>M<sup>c</sup> </th><th>E<sup>d</sup> </th><th>Pearson's corr.<sup>e</sup> </th></tr></thead><tbody><tr><td rowspan="15">Neocortex </td><td rowspan="11">NRXN2 </td><td>2 </td><td>6587361 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.897000684 </td></tr><tr><td>2 </td><td>6635041 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.838015353 </td></tr><tr><td>2 </td><td>6635090 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.886097969 </td></tr><tr><td>2 </td><td>6635002 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.828919384 </td></tr><tr><td>2 </td><td>6634986 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.902681422 </td></tr><tr><td>2 </td><td>6634955 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729923 </td></tr><tr><td>2 </td><td>6634980 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.874609127 </td></tr><tr><td>2 </td><td>6634930 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.720944026 </td></tr><tr><td>2 </td><td>6634961 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.837225041 </td></tr><tr><td>2 </td><td>6634934 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.785419851 </td></tr><tr><td>2 </td><td>6634951 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.865604511 </td></tr><tr><td rowspan="2">MAPT </td><td>12 </td><td>17133831 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.926451406 </td></tr><tr><td>12 </td><td>17133834 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.922834214 </td></tr><tr><td rowspan="2">SYNE1 </td><td>1 </td><td>16124463 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.80723621 </td></tr><tr><td>1 </td><td>16124489 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.750224969 </td></tr><tr><td rowspan="18">Muscle </td><td>CCDC88C </td><td>7 </td><td>119491407 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.936174813 </td></tr><tr><td rowspan="2">EXOC2 </td><td>7 </td><td>75292 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.851066176 </td></tr><tr><td>7 </td><td>75340 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.832158353 </td></tr><tr><td rowspan="2">FGR </td><td>6 </td><td>78329214 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.802870198 </td></tr><tr><td>6 </td><td>78336406 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.755515526 </td></tr><tr><td>MAP2K7 </td><td>2 </td><td>71712421 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.857718538 </td></tr><tr><td rowspan="4">NR5A2 </td><td>10 </td><td>27361479 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957673269 </td></tr><tr><td>10 </td><td>27361475 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.95434547 </td></tr><tr><td>10 </td><td>27361468 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.921209644 </td></tr><tr><td>10 </td><td>27361450 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.960461808 </td></tr><tr><td>PCOLCE </td><td>3 </td><td>7921803 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.995207087 </td></tr><tr><td>RAB22A </td><td>17 </td><td>65753356 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.884084122 </td></tr><tr><td>SLA-7 </td><td>7 </td><td>27608918 </td><td>UP10K </td><td> </td><td>+ </td><td>− </td><td>−0.914258584 </td></tr><tr><td>TUBGCP6 </td><td>5 </td><td>110682055 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.90531831 </td></tr><tr><td rowspan="2">VGLL2 </td><td>1 </td><td>49880654 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.753634499 </td></tr><tr><td>1 </td><td>49880666 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.805639146 </td></tr><tr><td rowspan="2">XPO4 </td><td>11 </td><td>268954 </td><td>Genebody </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.909143259 </td></tr><tr><td>11 </td><td>269035 </td><td>Genebody </td><td> </td><td>+ </td><td>− </td><td>−0.914121809 </td></tr><tr><td rowspan="22">Spleen </td><td>ICAM-1 </td><td>2 </td><td>69496888 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.755261243 </td></tr><tr><td rowspan="2">ALDH3A1 </td><td>12 </td><td>62650896 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.788071349 </td></tr><tr><td>12 </td><td>62650898 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.752480115 </td></tr><tr><td>KIRREL3 </td><td>9 </td><td>59344774 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.957798851 </td></tr><tr><td>GTF2IRD1 </td><td>3 </td><td>11271020 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.917047466 </td></tr><tr><td>UBA52 </td><td>2 </td><td>59052151 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.953385635 </td></tr><tr><td>CD163 </td><td>14 </td><td>153630638 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.860727255 </td></tr><tr><td>HMHA1 </td><td>2 </td><td>77641888 </td><td>UP10K </td><td>CGI shore </td><td>+ </td><td>− </td><td>−0.861581753 </td></tr><tr><td rowspan="4">RPS21 </td><td>17 </td><td>69165427 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.829232698 </td></tr><tr><td>17 </td><td>69165425 </td><td>UP10K </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.811905734 </td></tr><tr><td>17 </td><td>69169540 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.852778602 </td></tr><tr><td>17 </td><td>69169556 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.743101278 </td></tr><tr><td>SLC9A3R2 </td><td>3 </td><td>42231210 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.752184061 </td></tr><tr><td>ZNF646 </td><td>3 </td><td>17732629 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.828529507 </td></tr><tr><td rowspan="5">ADAMTS4 </td><td>4 </td><td>97117737 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.802729052 </td></tr><tr><td>4 </td><td>97117751 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.8029031 </td></tr><tr><td>4 </td><td>97117778 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.806509102 </td></tr><tr><td>4 </td><td>97117512 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.763107181 </td></tr><tr><td>4 </td><td>97117534 </td><td>UP10K </td><td> </td><td>− </td><td>+ </td><td>−0.814870957 </td></tr><tr><td rowspan="3">TBKBP1 </td><td>12 </td><td>23838338 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.89415394 </td></tr><tr><td>12 </td><td>23838110 </td><td>Genebody </td><td>CGI shore </td><td>− </td><td>+ </td><td>−0.975048371 </td></tr><tr><td>12 </td><td>23838349 </td><td>Genebody </td><td> </td><td>− </td><td>+ </td><td>−0.821418354 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Chromosome.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>b</sup>Class represents genomic location of tDMC; exon and intron region, 10 kb upstream region from TSS and CGI Shore were marked as Genebody, UP10K and CGI shore, respectively.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>c</sup>Methylation pattern, + and − represent higher and lower methylation over other tissues, respectively.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>d</sup>Expression level, + and − represent higher and lower expression levels over other tissues, respectively.</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>e</sup>Pearson's correlation score.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4937032" target="_blank">View Large</a></div></div><p>From GO analysis of the neocortex, synapse-related genes (<em>NRXN2</em> and <em>SYNE1</em>) and regulation of nerve system-related gene (<em>MAPT</em>) showed negative correlation between the levels of gene expression and CpG methylation at 15 tDMCs (average Pearson's correlation score less than −0.843). The expression levels of those three genes were higher in neocortex than in other tissues (adjusted <em>P</em>-value &lt;0.05). Fourteen out of 15 tDMCs were located on gene bodies, but only a single tDMC that associated with <em>NRXN2</em> was mapped to the upstream 10 kb region from TSS. In the muscle, 18 tDMCs were associated with 11 genes and showed negative correlation between the levels of expression and methylation (average Pearson's correlation score less than −0.884). The tDMCs of <em>NR5A2,</em><sup><a reveal-id="DSV017C66" data-open="DSV017C66" class="link link-ref link-reveal xref-bibr">66</a></sup><em>PCOLCE and VGLL2</em> which promote muscle development and activities were up-regulated and hypo-methylated. For the spleen, tDMCs of three immune-related genes <em>ICAM1, CD163</em> and <em>TBKBP1</em> were hypo-methylated (Table <a reveal-id="DSV017TB3" data-open="DSV017TB3" class="link link-reveal link-table xref-fig">3</a>).</p>                            <h2 scrollto-destination=4937034 id="4937034" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The completion of the pig genome assembly (Sscrofa10.2) provides a basis for understanding the epigenetic modifications of the pig genome including DNA methylation, which is another important issue in modern genetics. In this study, we generated the global methylation profiles from the porcine neocortex, spleen, liver, muscle, olfactory epithelium and a PAM cell line and analysed the RR genome of the pig. Although we did not analyse the methylation patterns of a complete list of pig tissues and the entire genome of the pig such as bisulfite sequencing, our results provide valuable input to the characteristics of the pig methylome considering that there have not been sufficient studies on the genome level of the profiles of the pig methylome from various tissues in a single nucleotide resolution.</p><p>Previously, Li et al.<sup><a reveal-id="DSV017C50" data-open="DSV017C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DSV017C67" data-open="DSV017C67" class="link link-ref link-reveal xref-bibr">67</a></sup> reported the results on the methylation pattern of the pig genome using MeDIP-seq, which investigated the genome-wide relationship between DNA methylation and fat deposition using adipose tissues of breeds with different phenotypes. Recently, Gao et al.<sup><a reveal-id="DSV017C44" data-open="DSV017C44" class="link link-ref link-reveal xref-bibr">44</a></sup> presented the RRBS analysis of the pig genome as the first report using the intestinal tissue. However, both studies showed limitations of providing information on only specific tissues or mapping resolution of the study.</p><p>In this study, we reported the global methylation patterns of five different tissues (the neocortex, spleen, liver, muscle and olfactory epithelium) and a cell line (PAM; 3D4/2) of the pig, which are part of quite diverse biological systems. The results of the global DNA methylation patterns corroborated the results of the previous studies<sup><a reveal-id="DSV017C44" data-open="DSV017C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSV017C58" data-open="DSV017C58" class="link link-ref link-reveal xref-bibr">58</a></sup> on the same regions.</p><p>Because the design of our RRBS analysis of the pig genome including the restriction enzyme and fragment size selection was identical to that in the studies of humans and mice,<sup><a reveal-id="DSV017C58" data-open="DSV017C58" class="link link-ref link-reveal xref-bibr">58</a></sup> it was appropriate to compare our results to the available data from those species. The size of the pig RR genome was similar to that of human (73 vs. 74 M, respectively, for <em>Msp</em>I, Table <a reveal-id="DSV017TB1" data-open="DSV017TB1" class="link link-reveal link-table xref-fig">1</a>). The RRBS reads of the pig exhibited a slightly lower frequency of multiple mapped reads than human (5.35 vs. 7.7%, respectively), and the percentage of total CpG sites in the pig RR genome (18.34%) was 4.84% higher than in the human, which may be attributed to slightly lower amount of repeat elements (∼40% of the genome) in the pig compared with those in the human (nearly 50%).</p><p>As expected, we observed the enrichment of CpG sites in the RR genome, and the rate was higher in pig (7.1-fold) than in humans (5.8-fold) and mice (5-fold) (Table <a reveal-id="DSV017TB1" data-open="DSV017TB1" class="link link-reveal link-table xref-fig">1</a>). A drawback of the RRBS is that CpGs within CpG-poor regions are scarcely covered because of the limitations of the RRBS method, which relies on the CpG-associated restriction enzymes. However, ∼60% of human gene contain a CGI near the TSS.<sup><a reveal-id="DSV017C68" data-open="DSV017C68" class="link link-ref link-reveal xref-bibr">68</a></sup> Therefore, RRBS studies can provide information of CGI and gene-associated CpG sites. Our results given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a> were consistent with the average of 44.21% of CpGs that belonged to the CGI observed in our RRBS analysis. CGIs consisted of only 1.11% of the pig genome and contained 8.44% of the total CpGs in the pig genome.</p><p>For future RRBS studies of the pig genome, it is necessary to improve the genome coverage to reduce the number of missing CpGs. Comparison of the <em>Msp</em>I-digested to the <em>Msp</em>I<em>/ApeK</em>I–double-digested RR genomes showed significant improvement in the percentage genome (2.6–12.12 Mb) and the total CpG coverage (18.3–28.4%), which were similar to those of studies in humans and mice.<sup><a reveal-id="DSV017C58" data-open="DSV017C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Further studies using <em>Msp</em>I<em>/ApeK</em>I should significantly improve the genome coverage of RRBS for the pig genome.</p><p>For GO analysis of the pig genome, gene enrichment analysis using web-based software is difficult to perform because of unavailability of enough functional annotation data. The amount of information on the functional annotation of the pig genome is still limited. As an attempt to overcome the difficulty, we performed a gene enrichment test against our tDMR-related gene dataset using the GeneMANIA (<a class="link link-uri" href="http://www.genemania.org/">http://www.genemania.org/</a>) software which allows the use of annotation data of the human genome. Our pilot analysis showed that the results were more informative than just relying on pig GO terms (data not shown).</p><p>Recent studies recognized the importance of CGI shores which might be associated with tissue-specific DMRs (tDMRs) and cancer-specific DMRs (cDMRs).<sup><a reveal-id="DSV017C69" data-open="DSV017C69" class="link link-ref link-reveal xref-bibr">69</a>,<a reveal-id="DSV017C70" data-open="DSV017C70" class="link link-ref link-reveal xref-bibr">70</a></sup> We detected a difference in the per cent methylation of CGI shore among the six samples. Lower correlation values between CpG methylation levels on CGI shore from different tissues might suggest a correlation with tissue-specific differential methylation (Fig. <a reveal-id="DSV017F1" data-open="DSV017F1" class="link link-reveal link-table xref-fig">1</a>). In this study, we detected 18 tDMCs located in the CGI shore region and they showed strong correlation with the level of gene expression using the transcriptome profile of a microarray experiment. The use of RNA-seq data should improve our understanding between tDMCs and the regulation of gene expression in pigs. The global difference in the level of non-CpG methylation was observed in the neocortex, which showed ∼35% more methylated non-CpG sites than other samples (Fig. <a reveal-id="DSV017F3" data-open="DSV017F3" class="link link-reveal link-table xref-fig">3</a>). The amount of methylated non-CpG sites was &gt;1.71% in all samples except in the neocortex (2.3%). The total amount of non-CpG sites (CHG and CHH) was almost four times larger than that of CpG sites (data not shown). Although the difference (0.6%) is small, the actual number of CpGs corresponding to that difference may not be as small and warrants further studies.</p><p>Recent studies showed that the SV40 large T antigen gene or <em>hTERT</em> transfected immortalized cell line results in DNA hyper-methylation and large-scale changes in gene expression.<sup><a reveal-id="DSV017C17" data-open="DSV017C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The result also showed the accumulation of DNA methylation at promoter regions according to doubling times or replicative lifespan. This is consistent with our finding of hyper-methylation in 3D4/2 cells which were established by transfecting the SV40 large T antigen (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). It was reported that a large part of immortality-associated changes in gene expression were related with RNA processing and regulation of transcription factor activity in immortalized cells.<sup><a reveal-id="DSV017C71" data-open="DSV017C71" class="link link-ref link-reveal xref-bibr">71</a></sup> The GO analysis of tDMCs among different tissues also showed that many of the 3D4/2 cell-specific tDMCs were related to transcriptional regulation. Thus, the uniqueness of the methlylome profile of 3D4/2 cells in this study is consistent to the results of previous methylome analyses using permanent cell lines. However, it still remains inconclusive whether the difference between 3D4/2 cells and other tissues was due to the characteristics of the permanent cell lines or the cell culture conditions or the natural methylation patters of pulmonary alveolar macrophage cells, because only one cell line was included in this study.</p><p>To create a suitable reference map of porcine methylome to biological studies, DNA methylation profiles from diverse tissues and diverse developmental stages are necessary. Although the list of samples and developmental stages is still far from being complete, our study contributes to the knowledge of DNA methylation in pig genome, which could provide information regarding biological processes in response to environmental changes, development and disease. To our knowledge, our results are among the initial studies to extensively evaluate the general methylation profile of the pig genome at a single base resolution.</p>                            <h2 scrollto-destination=4937046 id="4937046" class="section-title" data-legacy-id=s5>Authors’ contribution</h2>
<p>M.K.C. was responsible for sample preparation, genomic DNA isolation, RRBS data analysis using methylKit and involved in drafting the manuscript. J.I.L. performed the read alignment and overall RRBS data analysis. M.T.L. and D.T.N. collected pig samples. S.P. and N.S. performed the statistical analysis. K.M.S., J.B.K. and J.H.K. provided helpful ideas and discussion for the experiment. C.P. was involved in project planning, discussion and writing of the manuscript as a project principle investigator. All authors read and approved the final manuscript.</p>                            <h2 scrollto-destination=4937050 id="4937050" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4937055 id="4937055" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the fund from the Cooperative Research Program for Agriculture Science &amp; Technology Development (No. PJ009103), Rural Development Administration, Republic of Korea and by the 2013 KU Brain Pool of Konkuk University. Funding to pay the Open Access publication charges for this article was provided by Konkuk University.</p>                            <h2 scrollto-destination=4937060 id="4937060" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elango</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Yi</div>  <div class="given-names">S.V.</div></div></div> <div class="year">2008</div>, <div class="article-title">DNA methylation and structural and functional bimodality of vertebrate promoters</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">25</div>, <div class="fpage">1602</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20structural%20and%20functional%20bimodality%20of%20vertebrate%20promoters&amp;author=N.%20Elango&amp;author=S.V.%20Yi&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=25&amp;pages=1602-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18469331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bock</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Tomazou</div>  <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Brinkman</div>  <div class="given-names">A.B.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">Quantitative comparison of genome-wide DNA methylation mapping technologies</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">28</div>, <div class="fpage">1106</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20comparison%20of%20genome-wide%20DNA%20methylation%20mapping%20technologies&amp;author=C.%20Bock&amp;author=E.M.%20Tomazou&amp;author=A.B.%20Brinkman&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=1106-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1681" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1681" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1681"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weber</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hellmann</div>  <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Stadler</div>  <div class="given-names">M.B.</div></div> et al.  </div>. <div class="year">2007</div>, <div class="article-title">Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="volume">39</div>, <div class="fpage">457</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%2C%20silencing%20potential%20and%20evolutionary%20impact%20of%20promoter%20DNA%20methylation%20in%20the%20human%20genome&amp;author=M.%20Weber&amp;author=I.%20Hellmann&amp;author=M.B.%20Stadler&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=457-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17334365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div>  <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Bird</div>  <div class="given-names">A.</div></div></div> <div class="year">2008</div>, <div class="article-title">DNA methylation landscapes: provocative insights from epigenomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">9</div>, <div class="fpage">465</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20landscapes%3A%20provocative%20insights%20from%20epigenomics&amp;author=M.M.%20Suzuki&amp;author=A.%20Bird&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=9&amp;pages=465-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carrel</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Willard</div>  <div class="given-names">H.F.</div></div></div> <div class="year">2005</div>, <div class="article-title">X-inactivation profile reveals extensive variability in X-linked gene expression in females</div>, <div class="source">Nature</div> , <div class="volume">434</div>, <div class="fpage">400</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=X-inactivation%20profile%20reveals%20extensive%20variability%20in%20X-linked%20gene%20expression%20in%20females&amp;author=L.%20Carrel&amp;author=H.F.%20Willard&amp;publication_year=2005&amp;journal=Nature&amp;volume=434&amp;pages=400-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15772666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ehrlich</div>  <div class="given-names">M.</div></div></div> <div class="year">2009</div>, <div class="article-title">DNA hypomethylation in cancer cells</div>, <div class="source">Epigenomics</div> , <div class="volume">1</div>, <div class="fpage">239</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20hypomethylation%20in%20cancer%20cells&amp;author=M.%20Ehrlich&amp;publication_year=2009&amp;journal=Epigenomics&amp;volume=1&amp;pages=239-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2217/epi.09.33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2217%2Fepi.09.33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2217%2Fepi.09.33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20495664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hon</div>  <div class="given-names">G.C.</div></div>, <div class="name string-name"><div class="surname">Hawkins</div>  <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Caballero</div>  <div class="given-names">O.L.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer</div>, <div class="source">Genome Res.</div> , <div class="volume">22</div>, <div class="fpage">246</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20DNA%20hypomethylation%20coupled%20to%20repressive%20chromatin%20domain%20formation%20and%20gene%20silencing%20in%20breast%20cancer&amp;author=G.C.%20Hon&amp;author=R.D.%20Hawkins&amp;author=O.L.%20Caballero&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=246-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.125872.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.125872.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.125872.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22156296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morgan</div>  <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Santos</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Green</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Dean</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Reik</div>  <div class="given-names">W.</div></div></div> <div class="year">2005</div>, <div class="article-title">Epigenetic reprogramming in mammals</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="volume">14</div> (<div class="comment">Spec No</div>), <div class="fpage">R47</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20reprogramming%20in%20mammals&amp;author=H.D.%20Morgan&amp;author=F.%20Santos&amp;author=K.%20Green&amp;author=W.%20Dean&amp;author=W.%20Reik&amp;publication_year=2005&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=14&amp;pages=R47-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddi114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddi114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddi114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15809273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sasaki</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsui</div>  <div class="given-names">Y.</div></div></div> <div class="year">2008</div>, <div class="article-title">Epigenetic events in mammalian germ-cell development: reprogramming and beyond</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">9</div>, <div class="fpage">129</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20events%20in%20mammalian%20germ-cell%20development%3A%20reprogramming%20and%20beyond&amp;author=H.%20Sasaki&amp;author=Y.%20Matsui&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=9&amp;pages=129-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2295" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2295" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2295"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18197165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Igarashi</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Muroi</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kawashima</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Quantitative analysis of human tissue-specific differences in methylation</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="volume">376</div>, <div class="fpage">658</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20analysis%20of%20human%20tissue-specific%20differences%20in%20methylation&amp;author=J.%20Igarashi&amp;author=S.%20Muroi&amp;author=H.%20Kawashima&amp;publication_year=2008&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=376&amp;pages=658-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2008.09.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2008.09.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2008.09.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18805397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liang</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Song</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Ghosh</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Genome-wide survey reveals dynamic widespread tissue-specific changes in DNA methylation during development</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">231</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20reveals%20dynamic%20widespread%20tissue-specific%20changes%20in%20DNA%20methylation%20during%20development&amp;author=P.%20Liang&amp;author=F.%20Song&amp;author=S.%20Ghosh&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=231" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21569359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ziller</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Muller</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Liao</div>  <div class="given-names">J.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Genomic distribution and inter-sample variation of non-CpG methylation across human cell types</div>, <div class="source">PLoS Genet.</div> , <div class="volume">7</div>, <div class="fpage">e1002389</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20distribution%20and%20inter-sample%20variation%20of%20non-CpG%20methylation%20across%20human%20cell%20types&amp;author=M.J.%20Ziller&amp;author=F.%20Muller&amp;author=J.%20Liao&amp;publication_year=2011&amp;journal=PLoS%20Genet.&amp;volume=7&amp;pages=e1002389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ghosh</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yates</div>  <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Frühwald</div>  <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Miecznikowski</div>  <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Plass</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Smiraglia</div>  <div class="given-names">D.</div></div></div> <div class="year">2010</div>, <div class="article-title">Tissue specific DNA methylation of CpG islands in normal human adult somatic tissues distinguishes neural from non-neural tissues</div>, <div class="source">Epigenetics</div> , <div class="volume">5</div>, <div class="fpage">527</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue%20specific%20DNA%20methylation%20of%20CpG%20islands%20in%20normal%20human%20adult%20somatic%20tissues%20distinguishes%20neural%20from%20non-neural%20tissues&amp;author=S.%20Ghosh&amp;author=A.J.%20Yates&amp;author=M.C.%20Fr%C3%BChwald&amp;author=J.C.%20Miecznikowski&amp;author=C.%20Plass&amp;author=D.%20Smiraglia&amp;publication_year=2010&amp;journal=Epigenetics&amp;volume=5&amp;pages=527-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.5.6.12228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.5.6.12228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.5.6.12228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20505344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bock</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Beerman</div>  <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Lien</div>  <div class="given-names">W.H.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">DNA methylation dynamics during in vivo differentiation of blood and skin stem cells</div>, <div class="source">Mol. Cell</div> , <div class="volume">47</div>, <div class="fpage">633</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20dynamics%20during%20in%20vivo%20differentiation%20of%20blood%20and%20skin%20stem%20cells&amp;author=C.%20Bock&amp;author=I.%20Beerman&amp;author=W.H.%20Lien&amp;publication_year=2012&amp;journal=Mol.%20Cell&amp;volume=47&amp;pages=633-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.06.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.06.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.06.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22841485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lokk</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Modhukur</div>  <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Rajashekar</div>  <div class="given-names">B.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">DNA methylome profiling of human tissues identifies global and tissue-specific methylation patterns</div>, <div class="source">Genome Biol.</div> , <div class="volume">15</div>, <div class="fpage">r54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylome%20profiling%20of%20human%20tissues%20identifies%20global%20and%20tissue-specific%20methylation%20patterns&amp;author=K.%20Lokk&amp;author=V.%20Modhukur&amp;author=B.%20Rajashekar&amp;publication_year=2014&amp;journal=Genome%20Biol.&amp;volume=15&amp;pages=r54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-4-r54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-4-r54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-4-r54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24690455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varley</div>  <div class="given-names">K.E.</div></div>, <div class="name string-name"><div class="surname">Gertz</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bowling</div>  <div class="given-names">K.M.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Dynamic DNA methylation across diverse human cell lines and tissues</div>, <div class="source">Genome Res.</div> , <div class="volume">23</div>, <div class="fpage">555</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20DNA%20methylation%20across%20diverse%20human%20cell%20lines%20and%20tissues&amp;author=K.E.%20Varley&amp;author=J.%20Gertz&amp;author=K.M.%20Bowling&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=555-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.147942.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.147942.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.147942.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23325432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gordon</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Clouaire</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Bao</div>  <div class="given-names">X.X.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Immortality, but not oncogenic transformation, of primary human cells leads to epigenetic reprogramming of DNA methylation and gene expression</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">3529</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immortality%2C%20but%20not%20oncogenic%20transformation%2C%20of%20primary%20human%20cells%20leads%20to%20epigenetic%20reprogramming%20of%20DNA%20methylation%20and%20gene%20expression&amp;author=K.%20Gordon&amp;author=T.%20Clouaire&amp;author=X.X.%20Bao&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=3529-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24371281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gardiner-Garden</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Frommer</div>  <div class="given-names">M.</div></div></div> <div class="year">1987</div>, <div class="article-title">CpG islands in vertebrate genomes</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">196</div>, <div class="fpage">261</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20islands%20in%20vertebrate%20genomes&amp;author=M.%20Gardiner-Garden&amp;author=M.%20Frommer&amp;publication_year=1987&amp;journal=J.%20Mol.%20Biol.&amp;volume=196&amp;pages=261-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(87)90689-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(87)90689-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(87)90689-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3656447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meissner</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mikkelsen</div>  <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Gu</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Genome-scale DNA methylation maps of pluripotent and differentiated cells</div>, <div class="source">Nature</div> , <div class="volume">454</div>, <div class="fpage">766</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20DNA%20methylation%20maps%20of%20pluripotent%20and%20differentiated%20cells&amp;author=A.%20Meissner&amp;author=T.S.%20Mikkelsen&amp;author=H.%20Gu&amp;publication_year=2008&amp;journal=Nature&amp;volume=454&amp;pages=766-70" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18600261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hartung</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kanwar</div>  <div class="given-names">R.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Diametrically opposite methylome-transcriptome relationships in high- and low-CpG promoter genes in postmitotic neural rat tissue</div>, <div class="source">Epigenetics</div> , <div class="volume">7</div>, <div class="fpage">421</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diametrically%20opposite%20methylome-transcriptome%20relationships%20in%20high-%20and%20low-CpG%20promoter%20genes%20in%20postmitotic%20neural%20rat%20tissue&amp;author=T.%20Hartung&amp;author=L.%20Zhang&amp;author=R.%20Kanwar&amp;publication_year=2012&amp;journal=Epigenetics&amp;volume=7&amp;pages=421-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.19565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.19565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.19565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22415013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goldberg</div>  <div class="given-names">A.D.</div></div>, <div class="name string-name"><div class="surname">Allis</div>  <div class="given-names">C.D.</div></div>, <div class="name string-name"><div class="surname">Bernstein</div>  <div class="given-names">E.</div></div></div> <div class="year">2007</div>, <div class="article-title">Epigenetics: a landscape takes shape</div>, <div class="source">Cell</div> , <div class="volume">128</div>, <div class="fpage">635</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetics%3A%20a%20landscape%20takes%20shape&amp;author=A.D.%20Goldberg&amp;author=C.D.%20Allis&amp;author=E.%20Bernstein&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=635-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17320500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meissner</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gnirke</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bell</div>  <div class="given-names">G.W.</div></div>, <div class="name string-name"><div class="surname">Ramsahoye</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lander</div>  <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Jaenisch</div>  <div class="given-names">R.</div></div></div> <div class="year">2005</div>, <div class="article-title">Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">5868</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20representation%20bisulfite%20sequencing%20for%20comparative%20high-resolution%20DNA%20methylation%20analysis&amp;author=A.%20Meissner&amp;author=A.%20Gnirke&amp;author=G.W.%20Bell&amp;author=B.%20Ramsahoye&amp;author=E.S.%20Lander&amp;author=R.%20Jaenisch&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki901" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki901" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki901"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16224102" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Down</div>  <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Rakyan</div>  <div class="given-names">V.K.</div></div>, <div class="name string-name"><div class="surname">Turner</div>  <div class="given-names">D.J.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">26</div>, <div class="fpage">779</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Bayesian%20deconvolution%20strategy%20for%20immunoprecipitation-based%20DNA%20methylome%20analysis&amp;author=T.A.%20Down&amp;author=V.K.%20Rakyan&amp;author=D.J.%20Turner&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=779-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18612301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cross</div>  <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Charlton</div>  <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Nan</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Bird</div>  <div class="given-names">A.P.</div></div></div> <div class="year">1994</div>, <div class="article-title">Purification of CpG islands using a methylated DNA binding column</div>, <div class="source">Nat. Genet.</div> , <div class="volume">6</div>, <div class="fpage">236</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20of%20CpG%20islands%20using%20a%20methylated%20DNA%20binding%20column&amp;author=S.H.%20Cross&amp;author=J.A.%20Charlton&amp;author=X.%20Nan&amp;author=A.P.%20Bird&amp;publication_year=1994&amp;journal=Nat.%20Genet.&amp;volume=6&amp;pages=236-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0394-236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0394-236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0394-236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8012384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brinkman</div>  <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Simmer</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Ma</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kaan</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zhu</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Stunnenberg</div>  <div class="given-names">H.G.</div></div></div> <div class="year">2010</div>, <div class="article-title">Whole-genome DNA methylation profiling using MethylCap-seq</div>, <div class="source">Methods</div> , <div class="volume">52</div>, <div class="fpage">232</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20DNA%20methylation%20profiling%20using%20MethylCap-seq&amp;author=A.B.%20Brinkman&amp;author=F.%20Simmer&amp;author=K.%20Ma&amp;author=A.%20Kaan&amp;author=J.%20Zhu&amp;author=H.G.%20Stunnenberg&amp;publication_year=2010&amp;journal=Methods&amp;volume=52&amp;pages=232-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymeth.2010.06.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymeth.2010.06.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymeth.2010.06.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20542119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Serre</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Lee</div>  <div class="given-names">B.H.</div></div>, <div class="name string-name"><div class="surname">Ting</div>  <div class="given-names">A.H.</div></div></div> <div class="year">2010</div>, <div class="article-title">MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">391</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MBD-isolated%20Genome%20Sequencing%20provides%20a%20high-throughput%20and%20comprehensive%20survey%20of%20DNA%20methylation%20in%20the%20human%20genome&amp;author=D.%20Serre&amp;author=B.H.%20Lee&amp;author=A.H.%20Ting&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=391-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dedeurwaerder</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Defrance</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Calonne</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Denis</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sotiriou</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Fuks</div>  <div class="given-names">F.</div></div></div> <div class="year">2011</div>, <div class="article-title">Evaluation of the Infinium Methylation 450K technology</div>, <div class="source">Epigenomics</div> , <div class="volume">3</div>, <div class="fpage">771</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20the%20Infinium%20Methylation%20450K%20technology&amp;author=S.%20Dedeurwaerder&amp;author=M.%20Defrance&amp;author=E.%20Calonne&amp;author=H.%20Denis&amp;author=C.%20Sotiriou&amp;author=F.%20Fuks&amp;publication_year=2011&amp;journal=Epigenomics&amp;volume=3&amp;pages=771-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2217/epi.11.105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2217%2Fepi.11.105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2217%2Fepi.11.105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22126295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gu</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Smith</div>  <div class="given-names">Z.D.</div></div>, <div class="name string-name"><div class="surname">Bock</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Boyle</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gnirke</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Meissner</div>  <div class="given-names">A.</div></div></div> <div class="year">2011</div>, <div class="article-title">Preparation of reduced representation bisulfite sequencing libraries for genome-scale DNA methylation profiling</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">6</div>, <div class="fpage">468</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preparation%20of%20reduced%20representation%20bisulfite%20sequencing%20libraries%20for%20genome-scale%20DNA%20methylation%20profiling&amp;author=H.%20Gu&amp;author=Z.D.%20Smith&amp;author=C.%20Bock&amp;author=P.%20Boyle&amp;author=A.%20Gnirke&amp;author=A.%20Meissner&amp;publication_year=2011&amp;journal=Nat.%20Protoc.&amp;volume=6&amp;pages=468-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2010.190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2010.190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2010.190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21412275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div>  <div class="given-names">Z.D.</div></div>, <div class="name string-name"><div class="surname">Gu</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bock</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gnirke</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Meissner</div>  <div class="given-names">A.</div></div></div> <div class="year">2009</div>, <div class="article-title">High-throughput bisulfite sequencing in mammalian genomes</div>, <div class="source">Methods</div> , <div class="volume">48</div>, <div class="fpage">226</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20bisulfite%20sequencing%20in%20mammalian%20genomes&amp;author=Z.D.%20Smith&amp;author=H.%20Gu&amp;author=C.%20Bock&amp;author=A.%20Gnirke&amp;author=A.%20Meissner&amp;publication_year=2009&amp;journal=Methods&amp;volume=48&amp;pages=226-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymeth.2009.05.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymeth.2009.05.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymeth.2009.05.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19442738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robertson</div>  <div class="given-names">K.D.</div></div></div> <div class="year">2005</div>, <div class="article-title">DNA methylation and human disease</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">6</div>, <div class="fpage">597</div>–<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20human%20disease&amp;author=K.D.%20Robertson&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=597-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16136652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Niculescu</div>  <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Zeisel</div>  <div class="given-names">S.H.</div></div></div> <div class="year">2002</div>, <div class="article-title">Diet, methyl donors and DNA methylation: interactions between dietary folate, methionine and choline</div>, <div class="source">J. Nutr.</div> , <div class="volume">132</div>, <div class="fpage">2333s</div>–<div class="lpage">5s</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diet%2C%20methyl%20donors%20and%20DNA%20methylation%3A%20interactions%20between%20dietary%20folate%2C%20methionine%20and%20choline&amp;author=M.D.%20Niculescu&amp;author=S.H.%20Zeisel&amp;publication_year=2002&amp;journal=J.%20Nutr.&amp;volume=132&amp;pages=2333s-5s" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12163687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">X.J.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">J.Q.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">R.S.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Maternal dietary protein affects transcriptional regulation of myostatin gene distinctively at weaning and finishing stages in skeletal muscle of Meishan pigs</div>, <div class="source">Epigenetics</div> , <div class="volume">6</div>, <div class="fpage">899</div>–<div class="lpage">907</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maternal%20dietary%20protein%20affects%20transcriptional%20regulation%20of%20myostatin%20gene%20distinctively%20at%20weaning%20and%20finishing%20stages%20in%20skeletal%20muscle%20of%20Meishan%20pigs&amp;author=X.J.%20Liu&amp;author=J.Q.%20Wang&amp;author=R.S.%20Li&amp;publication_year=2011&amp;journal=Epigenetics&amp;volume=6&amp;pages=899-907" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.6.7.16005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.6.7.16005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.6.7.16005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21597337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altmann</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Murani</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Schwerin</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Metges</div>  <div class="given-names">C.C.</div></div>, <div class="name string-name"><div class="surname">Wimmers</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ponsuksili</div>  <div class="given-names">S.</div></div></div> <div class="year">2012</div>, <div class="article-title">Maternal dietary protein restriction and excess affects offspring gene expression and methylation of non-SMC subunits of condensin I in liver and skeletal muscle</div>, <div class="source">Epigenetics</div> , <div class="volume">7</div>, <div class="fpage">239</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maternal%20dietary%20protein%20restriction%20and%20excess%20affects%20offspring%20gene%20expression%20and%20methylation%20of%20non-SMC%20subunits%20of%20condensin%20I%20in%20liver%20and%20skeletal%20muscle&amp;author=S.%20Altmann&amp;author=E.%20Murani&amp;author=M.%20Schwerin&amp;author=C.C.%20Metges&amp;author=K.%20Wimmers&amp;author=S.%20Ponsuksili&amp;publication_year=2012&amp;journal=Epigenetics&amp;volume=7&amp;pages=239-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.7.3.19183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.7.3.19183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.7.3.19183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22430800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kwak</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">D.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide DNA methylation profiles of small intestine and liver in fast-growing and slow-growing weaning piglets</div>, <div class="source">Asian Australas. J. Anim. Sci.</div> , <div class="volume">27</div>, <div class="fpage">1532</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20DNA%20methylation%20profiles%20of%20small%20intestine%20and%20liver%20in%20fast-growing%20and%20slow-growing%20weaning%20piglets&amp;author=W.%20Kwak&amp;author=J.N.%20Kim&amp;author=D.%20Kim&amp;publication_year=2014&amp;journal=Asian%20Australas.%20J.%20Anim.%20Sci.&amp;volume=27&amp;pages=1532-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5713/ajas.2014.14309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5713%2Fajas.2014.14309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5713%2Fajas.2014.14309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25358311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rauch</div>  <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wu</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Kernstine</div>  <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Riggs</div>  <div class="given-names">A.D.</div></div>, <div class="name string-name"><div class="surname">Pfeifer</div>  <div class="given-names">G.P.</div></div></div> <div class="year">2012</div>, <div class="article-title">DNA methylation biomarkers for lung cancer</div>, <div class="source">Tumour Biol.</div> , <div class="volume">33</div>, <div class="fpage">287</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20biomarkers%20for%20lung%20cancer&amp;author=T.A.%20Rauch&amp;author=Z.%20Wang&amp;author=X.%20Wu&amp;author=K.H.%20Kernstine&amp;author=A.D.%20Riggs&amp;author=G.P.%20Pfeifer&amp;publication_year=2012&amp;journal=Tumour%20Biol.&amp;volume=33&amp;pages=287-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13277-011-0282-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13277-011-0282-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13277-011-0282-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22143938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heyn</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sayols</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Moutinho</div>  <div class="given-names">C.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Linkage of DNA methylation quantitative trait loci to human cancer risk</div>, <div class="source">Cell Rep.</div> , <div class="volume">7</div>, <div class="fpage">331</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20of%20DNA%20methylation%20quantitative%20trait%20loci%20to%20human%20cancer%20risk&amp;author=H.%20Heyn&amp;author=S.%20Sayols&amp;author=C.%20Moutinho&amp;publication_year=2014&amp;journal=Cell%20Rep.&amp;volume=7&amp;pages=331-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2014.03.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2014.03.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2014.03.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24703846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gloss</div>  <div class="given-names">B.S.</div></div>, <div class="name string-name"><div class="surname">Patterson</div>  <div class="given-names">K.I.</div></div>, <div class="name string-name"><div class="surname">Barton</div>  <div class="given-names">C.A.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Integrative genome-wide expression and promoter DNA methylation profiling identifies a potential novel panel of ovarian cancer epigenetic biomarkers</div>, <div class="source">Cancer Lett.</div> , <div class="volume">318</div>, <div class="fpage">76</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20genome-wide%20expression%20and%20promoter%20DNA%20methylation%20profiling%20identifies%20a%20potential%20novel%20panel%20of%20ovarian%20cancer%20epigenetic%20biomarkers&amp;author=B.S.%20Gloss&amp;author=K.I.%20Patterson&amp;author=C.A.%20Barton&amp;publication_year=2012&amp;journal=Cancer%20Lett.&amp;volume=318&amp;pages=76-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.canlet.2011.12.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.canlet.2011.12.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.canlet.2011.12.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22155104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobayashi</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Absher</div>  <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Gulzar</div>  <div class="given-names">Z.G.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">DNA methylation profiling reveals novel biomarkers and important roles for DNA methyltransferases in prostate cancer</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">1017</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20profiling%20reveals%20novel%20biomarkers%20and%20important%20roles%20for%20DNA%20methyltransferases%20in%20prostate%20cancer&amp;author=Y.%20Kobayashi&amp;author=D.M.%20Absher&amp;author=Z.G.%20Gulzar&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=1017-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.119487.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.119487.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.119487.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21521786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Jager</div>  <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Srivastava</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Lunnon</div>  <div class="given-names">K.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Alzheimer's disease: early alterations in brain DNA methylation at ANK1, BIN1, RHBDF2 and other loci</div>, <div class="source">Nat. Neurosci.</div> , <div class="volume">17</div>, <div class="fpage">1156</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alzheimer's%20disease%3A%20early%20alterations%20in%20brain%20DNA%20methylation%20at%20ANK1%2C%20BIN1%2C%20RHBDF2%20and%20other%20loci&amp;author=P.L.%20De%20Jager&amp;author=G.%20Srivastava&amp;author=K.%20Lunnon&amp;publication_year=2014&amp;journal=Nat.%20Neurosci.&amp;volume=17&amp;pages=1156-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nn.3786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnn.3786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnn.3786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25129075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Couldrey</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Brauning</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bracegirdle</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Maclean</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Henderson</div>  <div class="given-names">H.V.</div></div>, <div class="name string-name"><div class="surname">McEwan</div>  <div class="given-names">J.C.</div></div></div> <div class="year">2014</div>, <div class="article-title">Genome-wide DNA methylation patterns and transcription analysis in sheep muscle</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e101853</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20DNA%20methylation%20patterns%20and%20transcription%20analysis%20in%20sheep%20muscle&amp;author=C.%20Couldrey&amp;author=R.%20Brauning&amp;author=J.%20Bracegirdle&amp;author=P.%20Maclean&amp;author=H.V.%20Henderson&amp;author=J.C.%20McEwan&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e101853" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0101853" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0101853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0101853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25010796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Su</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xing</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">Y.</div></div></div> <div class="year">2014</div>, <div class="article-title">Genome-wide analysis of DNA methylation in bovine placentas</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20DNA%20methylation%20in%20bovine%20placentas&amp;author=J.%20Su&amp;author=Y.%20Wang&amp;author=X.%20Xing&amp;author=J.%20Liu&amp;author=Y.%20Zhang&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24397284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">Y.Z.</div></div>, <div class="name string-name"><div class="surname">Sun</div>  <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">L.Z.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide DNA methylation profiles and their relationships with mRNA and the microRNA transcriptome in bovine muscle tissue (Bos taurine)</div>, <div class="source">Sci Rep-Uk</div> , <div class="volume">4</div>, <div class="fpage">1</div>–<div class="lpage">17</div>.</div></div></div></div><div content-id="DSV017C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div>  <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Hong</div>  <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Moon</div>  <div class="given-names">J.W.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide analysis of DNA methylation patterns in horse</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">598</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20DNA%20methylation%20patterns%20in%20horse&amp;author=J.R.%20Lee&amp;author=C.P.%20Hong&amp;author=J.W.%20Moon&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=598" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25027854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gao</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jiang</div>  <div class="given-names">P.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Marked methylation changes in intestinal genes during the perinatal period of preterm neonates</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">716</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Marked%20methylation%20changes%20in%20intestinal%20genes%20during%20the%20perinatal%20period%20of%20preterm%20neonates&amp;author=F.%20Gao&amp;author=J.%20Zhang&amp;author=P.%20Jiang&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=716" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-716" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-716" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-716"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25163507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tumbleson</div>  <div class="given-names">S.</div></div></div> <div class="year">1997</div>, <div class="article-title">Advances in swine in biomedical research</div>. <div class="comment">Edited by M.E. Tumbleson and L.B. Schook. Plenum Publishing Corporation, 1996. Volume 1, pp. 462. Volume 2, pp. 499</div>, <div class="source">Exp. Physiol.</div> , <div class="volume">82</div>, <div class="fpage">803</div>–<div class="lpage">804</div>.</div></div></div></div><div content-id="DSV017C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Groenen</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Archibald</div>  <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Uenishi</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Analyses of pig genomes provide insight into porcine demography and evolution</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">393</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyses%20of%20pig%20genomes%20provide%20insight%20into%20porcine%20demography%20and%20evolution&amp;author=M.A.%20Groenen&amp;author=A.L.%20Archibald&amp;author=H.%20Uenishi&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=393-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23151582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paudel</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Madsen</div>  <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Megens</div>  <div class="given-names">H.J.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Evolutionary dynamics of copy number variation in pig genomes in the context of adaptation and domestication</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">449</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20copy%20number%20variation%20in%20pig%20genomes%20in%20the%20context%20of%20adaptation%20and%20domestication&amp;author=Y.%20Paudel&amp;author=O.%20Madsen&amp;author=H.J.%20Megens&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=449" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23829399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nguyen</div>  <div class="given-names">D.T.</div></div>, <div class="name string-name"><div class="surname">Lee</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Choi</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">The complete swine olfactory subgenome: expansion of the olfactory gene repertoire in the pig genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">584</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20swine%20olfactory%20subgenome%3A%20expansion%20of%20the%20olfactory%20gene%20repertoire%20in%20the%20pig%20genome&amp;author=D.T.%20Nguyen&amp;author=K.%20Lee&amp;author=H.%20Choi&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=584" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-584" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-584" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-584"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23153364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rubin</div>  <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Megens</div>  <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Martinez Barrio</div>  <div class="given-names">A.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Strong signatures of selection in the domestic pig genome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">19529</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20signatures%20of%20selection%20in%20the%20domestic%20pig%20genome&amp;author=C.J.%20Rubin&amp;author=H.J.%20Megens&amp;author=A.%20Martinez%20Barrio&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=19529-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1217149109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1217149109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1217149109"> </span></div> </div></div></div></div></div><div content-id="DSV017C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wu</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Luo</div>  <div class="given-names">Z.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">An atlas of DNA methylomes in porcine adipose and muscle tissues</div>, <div class="source">Nat. Commun.</div> , <div class="volume">3</div>, <div class="fpage">850</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20atlas%20of%20DNA%20methylomes%20in%20porcine%20adipose%20and%20muscle%20tissues&amp;author=M.%20Li&amp;author=H.%20Wu&amp;author=Z.%20Luo&amp;publication_year=2012&amp;journal=Nat.%20Commun.&amp;volume=3&amp;pages=850" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms1854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms1854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms1854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22617290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Fiziev</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Yan</div>  <div class="given-names">W.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">BS-Seeker2: a versatile aligning pipeline for bisulfite sequencing data</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">774</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BS-Seeker2%3A%20a%20versatile%20aligning%20pipeline%20for%20bisulfite%20sequencing%20data&amp;author=W.%20Guo&amp;author=P.%20Fiziev&amp;author=W.%20Yan&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=774" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-774" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-774" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-774"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24206606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div>  <div class="given-names">S.L.</div></div></div> <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akalin</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kormaksson</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles</div>, <div class="source">Genome Biol.</div> , <div class="volume">13</div>, <div class="fpage">R87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=methylKit%3A%20a%20comprehensive%20R%20package%20for%20the%20analysis%20of%20genome-wide%20DNA%20methylation%20profiles&amp;author=A.%20Akalin&amp;author=M.%20Kormaksson&amp;author=S.%20Li&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=R87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-10-r87" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-10-r87" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-10-r87"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23034086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bardou</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mariette</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Escudié</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Djemiel</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Klopp</div>  <div class="given-names">C.</div></div></div> <div class="year">2014</div>, <div class="article-title">jvenn: an interactive Venn diagram viewer</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">15</div>, <div class="fpage">293</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=jvenn%3A%20an%20interactive%20Venn%20diagram%20viewer&amp;author=P.%20Bardou&amp;author=J.%20Mariette&amp;author=F.%20Escudi%C3%A9&amp;author=C.%20Djemiel&amp;author=C.%20Klopp&amp;publication_year=2014&amp;journal=BMC%20Bioinformatics&amp;volume=15&amp;pages=293" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-15-293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-15-293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-15-293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25176396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hornshoj</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Conley</div>  <div class="given-names">L.N.</div></div>, <div class="name string-name"><div class="surname">Hedegaard</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sorensen</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Panitz</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Bendixen</div>  <div class="given-names">C.</div></div></div> <div class="year">2007</div>, <div class="article-title">Microarray expression profiles of 20.000 genes across 23 healthy porcine tissues</div>, <div class="source">PLoS ONE</div> , <div class="volume">2</div>, <div class="fpage">e1203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20expression%20profiles%20of%2020.000%20genes%20across%2023%20healthy%20porcine%20tissues&amp;author=H.%20Hornshoj&amp;author=L.N.%20Conley&amp;author=J.%20Hedegaard&amp;author=P.%20Sorensen&amp;author=F.%20Panitz&amp;author=C.%20Bendixen&amp;publication_year=2007&amp;journal=PLoS%20ONE&amp;volume=2&amp;pages=e1203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0001203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0001203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0001203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18030337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Team</div>  <div class="given-names">R.C.</div></div></div> <div class="year">2014</div>, <div class="source">R: a language and environment for statistical computing</div> . <div class="publisher-name">R Foundation for Statistical Computing: Vienna, Austria</div>.</div></div></div></div><div content-id="DSV017C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cho</div>  <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Campbell</div>  <div class="given-names">M.J.</div></div> et al.  </div>. <div class="year">2001</div>, <div class="article-title">Transcriptional regulation and function during the human cell cycle</div>, <div class="source">Nat. Genet.</div> , <div class="volume">27</div>, <div class="fpage">48</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulation%20and%20function%20during%20the%20human%20cell%20cycle&amp;author=R.J.%20Cho&amp;author=M.%20Huang&amp;author=M.J.%20Campbell&amp;publication_year=2001&amp;journal=Nat.%20Genet.&amp;volume=27&amp;pages=48-54" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11137997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xia</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">L.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Double restriction-enzyme digestion improves the coverage and accuracy of genome-wide CpG methylation profiling by reduced representation bisulfite sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double%20restriction-enzyme%20digestion%20improves%20the%20coverage%20and%20accuracy%20of%20genome-wide%20CpG%20methylation%20profiling%20by%20reduced%20representation%20bisulfite%20sequencing&amp;author=J.%20Wang&amp;author=Y.%20Xia&amp;author=L.%20Li&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23324053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chatterjee</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ozaki</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Stockwell</div>  <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Horsfield</div>  <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Morison</div>  <div class="given-names">I.M.</div></div>, <div class="name string-name"><div class="surname">Nakagawa</div>  <div class="given-names">S.</div></div></div> <div class="year">2013</div>, <div class="article-title">Mapping the zebrafish brain methylome using reduced representation bisulfite sequencing</div>, <div class="source">Epigenetics</div> , <div class="volume">8</div>, <div class="fpage">979</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20the%20zebrafish%20brain%20methylome%20using%20reduced%20representation%20bisulfite%20sequencing&amp;author=A.%20Chatterjee&amp;author=Y.%20Ozaki&amp;author=P.A.%20Stockwell&amp;author=J.A.%20Horsfield&amp;author=I.M.%20Morison&amp;author=S.%20Nakagawa&amp;publication_year=2013&amp;journal=Epigenetics&amp;volume=8&amp;pages=979-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.25797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.25797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.25797"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23975027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laird</div>  <div class="given-names">P.W.</div></div></div> <div class="year">2010</div>, <div class="article-title">Principles and challenges of genomewide DNA methylation analysis</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">11</div>, <div class="fpage">191</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Principles%20and%20challenges%20of%20genomewide%20DNA%20methylation%20analysis&amp;author=P.W.%20Laird&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=191-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2732" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20125086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saxonov</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Berg</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Brutlag</div>  <div class="given-names">D.L.</div></div></div> <div class="year">2006</div>, <div class="article-title">A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">1412</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20analysis%20of%20CpG%20dinucleotides%20in%20the%20human%20genome%20distinguishes%20two%20distinct%20classes%20of%20promoters&amp;author=S.%20Saxonov&amp;author=P.%20Berg&amp;author=D.L.%20Brutlag&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=1412-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0510310103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0510310103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0510310103"> </span></div> </div></div></div></div></div><div content-id="DSV017C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harris</div>  <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Coarfa</div>  <div class="given-names">C.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">28</div>, <div class="fpage">1097</div>–<div class="lpage">105</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20sequencing-based%20methods%20to%20profile%20DNA%20methylation%20and%20identification%20of%20monoallelic%20epigenetic%20modifications&amp;author=R.A.%20Harris&amp;author=T.%20Wang&amp;author=C.%20Coarfa&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=1097-105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1682" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1682" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1682"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pan</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chen</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Dogra</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Measuring the methylome in clinical samples: improved processing of the Infinium Human Methylation450 BeadChip Array</div>, <div class="source">Epigenetics.</div> , <div class="volume">7</div>, <div class="fpage">1173</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measuring%20the%20methylome%20in%20clinical%20samples%3A%20improved%20processing%20of%20the%20Infinium%20Human%20Methylation450%20BeadChip%20Array&amp;author=H.%20Pan&amp;author=L.%20Chen&amp;author=S.%20Dogra&amp;publication_year=2012&amp;journal=Epigenetics.&amp;volume=7&amp;pages=1173-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.22102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.22102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.22102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22964528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div>  <div class="given-names">J.U.</div></div>, <div class="name string-name"><div class="surname">Su</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shin</div>  <div class="given-names">J.H.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain</div>, <div class="source">Nat. Neurosci.</div> , <div class="volume">17</div>, <div class="fpage">215</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%2C%20recognition%20and%20regulation%20of%20non-CpG%20methylation%20in%20the%20adult%20mammalian%20brain&amp;author=J.U.%20Guo&amp;author=Y.%20Su&amp;author=J.H.%20Shin&amp;publication_year=2014&amp;journal=Nat.%20Neurosci.&amp;volume=17&amp;pages=215-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nn.3607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnn.3607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnn.3607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24362762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smallwood</div>  <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Tomizawa</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Krueger</div>  <div class="given-names">F.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Dynamic CpG island methylation landscape in oocytes and preimplantation embryos</div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">811</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20CpG%20island%20methylation%20landscape%20in%20oocytes%20and%20preimplantation%20embryos&amp;author=S.A.%20Smallwood&amp;author=S.%20Tomizawa&amp;author=F.%20Krueger&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=811-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21706000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolado-Carrancio</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Riancho</div>  <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Sainz</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Rodriguez-Rey</div>  <div class="given-names">J.C.</div></div></div> <div class="year">2014</div>, <div class="article-title">Activation of nuclear receptor NR5A2 increases Glut4 expression and glucose metabolism in muscle cells</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="volume">446</div>, <div class="fpage">614</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activation%20of%20nuclear%20receptor%20NR5A2%20increases%20Glut4%20expression%20and%20glucose%20metabolism%20in%20muscle%20cells&amp;author=A.%20Bolado-Carrancio&amp;author=J.A.%20Riancho&amp;author=J.%20Sainz&amp;author=J.C.%20Rodriguez-Rey&amp;publication_year=2014&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=446&amp;pages=614-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2014.03.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2014.03.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2014.03.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24632207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wu</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genome-wide DNA methylation changes between the superficial and deep backfat tissues of the pig</div>, <div class="source">Int. J. Mol. Sci.</div> , <div class="volume">13</div>, <div class="fpage">7098</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20DNA%20methylation%20changes%20between%20the%20superficial%20and%20deep%20backfat%20tissues%20of%20the%20pig&amp;author=M.%20Li&amp;author=T.%20Wang&amp;author=H.%20Wu&amp;publication_year=2012&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;volume=13&amp;pages=7098-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms13067098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms13067098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms13067098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22837681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larsen</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Gundersen</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Lopez</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Prydz</div>  <div class="given-names">H.</div></div></div> <div class="year">1992</div>, <div class="article-title">CpG islands as gene markers in the human genome</div>, <div class="source">Genomics</div> , <div class="volume">13</div>, <div class="fpage">1095</div>–<div class="lpage">107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20islands%20as%20gene%20markers%20in%20the%20human%20genome&amp;author=F.%20Larsen&amp;author=G.%20Gundersen&amp;author=R.%20Lopez&amp;author=H.%20Prydz&amp;publication_year=1992&amp;journal=Genomics&amp;volume=13&amp;pages=1095-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(92)90024-M" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(92)90024-M" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(92)90024-M"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1505946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doi</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Park</div>  <div class="given-names">I.H.</div></div>, <div class="name string-name"><div class="surname">Wen</div>  <div class="given-names">B.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">Differential methylation of tissue- and cancer-specific CpG island shores distinguishes human induced pluripotent stem cells, embryonic stem cells and fibroblasts</div>, <div class="source">Nat. Genet.</div> , <div class="volume">41</div>, <div class="fpage">1350</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20methylation%20of%20tissue-%20and%20cancer-specific%20CpG%20island%20shores%20distinguishes%20human%20induced%20pluripotent%20stem%20cells%2C%20embryonic%20stem%20cells%20and%20fibroblasts&amp;author=A.%20Doi&amp;author=I.H.%20Park&amp;author=B.%20Wen&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=1350-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Irizarry</div>  <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Ladd-Acosta</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wen</div>  <div class="given-names">B.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores</div>, <div class="source">Nat. Genet.</div> , <div class="volume">41</div>, <div class="fpage">178</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20colon%20cancer%20methylome%20shows%20similar%20hypo-%20and%20hypermethylation%20at%20conserved%20tissue-specific%20CpG%20island%20shores&amp;author=R.A.%20Irizarry&amp;author=C.%20Ladd-Acosta&amp;author=B.%20Wen&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=178-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV017C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV017C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bates</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2005</div>, <div class="article-title">A methylation profile of in vitro immortalized human cell lines</div>, <div class="source">Int. J. Oncol.</div> , <div class="volume">26</div>, <div class="fpage">275</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20methylation%20profile%20of%20in%20vitro%20immortalized%20human%20cell%20lines&amp;author=L.%20Liu&amp;author=J.%20Zhang&amp;author=S.%20Bates&amp;publication_year=2005&amp;journal=Int.%20J.%20Oncol.&amp;volume=26&amp;pages=275-85" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15586250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347949&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv017-dsv017supp_tables.xlsx?Expires=2147483647&amp;Signature=MOQ-JwbSfDiBRhwjYzkKQXGhmxywrEtjJKk4khyk7u0tlJ4BSZOVImGyvsbQr~Pf11u-VW-YePrN1cIYEjiq4Jzs9tC~m6~UZ2q2CqLLPOnqIgNPfs2BHh4UU1c2s--DsewA6p7MvEY4~inG8Bqn61iZ1sMOH0Mc5Zh~STKUVFDbjnFvnm2haUIDPzKHCqDQy1y2y~CNFYnRfFmlEfydJ17L5Uw9LI~ii6erRDtNvq7XRJmF0k1FR3WBJZ~gVBrZwR7ygq05Z6oknrUxiav9KJhdSax0O6QlbdQQdNeZ-cKbgtk3CCodoZ9yDuwtycT2H4V~uKBHVuzssCYSUubypw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv017/2/dsv017-dsv017supp_data.pdf?Expires=2147483647&amp;Signature=eSp-j8PsCyepBRPJMi79avudn3rc4u-aFcFXv8kxhhhTBdZ5UITgLMOyElbda4icWM2-9mOG0zlESTG1~azD1ZjaqPQwOvZstdrxNADbMbbuKMIc4CHx9CMCc6ybAxgEsiCnWB1XQ~ol~eA4ZHzhs5VI1zx8-tqTbdb0MlMGYdCcN5G7gyG57MOLBxmY3UcJOqt1QP5MUM6A0R0jVxZ1Rx6dEPlba04ww4oLaKUXSFpabo9BUWvzAZGsPiLrtCFG31-ckVPTFlGsD4JCVjFf2Fu5b0tDBV29P903a~XfXvvMsaDztgl5aQRtg2rsolkhNzJOjGdvzP2esCji2OgIsQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">784</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">20</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347949"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f5%2f343%2f347949">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362820100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (20)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F5%2F343%2F347949" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347949'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide analysis of DNA methylation in pigs using reduced representation bisulfite sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f343%2f347949">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

