

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/387/2462399","@type":"ScholarlyArticle","name":"Genome-wide analysis of tandem repeats in  Tribolium castaneum  genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms","datePublished":"2015-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv021","keywords":["Tandem repeats","euchromatic regions","evolutionary trends","transposition","<italic>Tribolium castaneum</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/387/2462399","author":[{"name":"Pavlek, Martina","affiliation":"1  Ruđer Bošković Institute ,  Bijenička 54, Zagreb HR-10002 ,  Croatia","@type":"Person"},{"name":"Gelfand, Yevgeniy","affiliation":"2  Laboratory for Biocomputing and Informatics ,  Boston University ,  Boston, MA 02215 ,  USA","@type":"Person"},{"name":"Plohl, Miroslav","affiliation":"1  Ruđer Bošković Institute ,  Bijenička 54, Zagreb HR-10002 ,  Croatia","@type":"Person"},{"name":"Meštrović, Nevenka","affiliation":"1  Ruđer Bošković Institute ,  Bijenička 54, Zagreb HR-10002 ,  Croatia","@type":"Person"}],"description":"Abstract.  Although satellite DNAs are well-explored components of heterochromatin and centromeres, little is known about emergence, dispersal and possible impa","pageStart":"387","pageEnd":"401","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide analysis of tandem repeats in  Tribolium castaneum  genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide analysis of tandem repeats in  Tribolium castaneum  genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms" />
    <meta property="og:description" content="Abstract.  Although satellite DNAs are well-explored components of heterochromatin and centromeres, little is known about emergence, dispersal and possible impa" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/387/2462399" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-10-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Pavlek, Martina" /><meta name="citation_author_institution" content="Ruđer Bošković Institute, Bijenička 54, Zagreb HR-10002, Croatia" /><meta name="citation_author" content="Gelfand, Yevgeniy" /><meta name="citation_author_institution" content="Laboratory for Biocomputing and Informatics, Boston University, Boston, MA 02215, USA" /><meta name="citation_author" content="Plohl, Miroslav" /><meta name="citation_author_institution" content="Ruđer Bošković Institute, Bijenička 54, Zagreb HR-10002, Croatia" /><meta name="citation_author" content="Meštrović, Nevenka" /><meta name="citation_author_institution" content="Ruđer Bošković Institute, Bijenička 54, Zagreb HR-10002, Croatia" /><meta name="citation_title" content="Genome-wide analysis of tandem repeats in Tribolium castaneum genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms" /><meta name="citation_firstpage" content="387" /><meta name="citation_lastpage" content="401" /><meta name="citation_doi" content="10.1093/dnares/dsv021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Satellite DNA evolution; citation_publisher=Karger Publishers, Basel;  citation_title=Repetitive DNA. Genome Dyn 7;  citation_year=2012;  citation_pages=126-52; " /><meta name="citation_reference" content="citation_title=Centromeres convert but don't cross;  citation_journal_title=PLoS Biol.;  citation_year=2010;  citation_volume=8;  citation_pages=1-5; " /><meta name="citation_reference" content="citation_title=Comparative analysis of tandem repeats from hundreds of species reveals unique insights into centromere evolution;  citation_journal_title=Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R10" /><meta name="citation_reference" content="citation_title=Nucleosomes and centromeric DNA packaging;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=1-2; " /><meta name="citation_reference" content="citation_title=Molecular drive in multigene families: how biological novelties arise, spread and are assimilated;  citation_journal_title=Trends Genet.;  citation_year=1986;  citation_volume=2;  citation_pages=159-65; " /><meta name="citation_reference" content="citation_title=Physical and genetic mapping of the human×chromosome centromere: repression of recombination;  citation_journal_title=Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=100-10; " /><meta name="citation_reference" content="citation_title=Evolution of repeated DNA sequences by unequal crossover;  citation_journal_title=Science;  citation_year=1976;  citation_volume=191;  citation_pages=528-35; " /><meta name="citation_reference" content="citation_title=The evolutionary dynamics of alpha-satellite;  citation_journal_title=Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=88-96; " /><meta name="citation_reference" content="citation_title=Tetris is a foldback transposon that provided the building blocks for an emerging satellite DNA of Drosophila virilis;  citation_journal_title=Genome Biol. Evol.;  citation_year=2014;  citation_volume=6;  citation_pages=1302-13; " /><meta name="citation_reference" content="citation_title=Tandem repeat-containing MITE elements in the clam Donax trunculus;  citation_journal_title=Genome Biol. Evol.;  citation_year=2013;  citation_volume=5;  citation_pages=2549-59; " /><meta name="citation_reference" content="citation_title=Extrachromosomal circles of satellite repeats and 5S ribosomal DNA in human cells;  citation_journal_title=Mob. DNA;  citation_year=2010;  citation_volume=1;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Analysis of the largest tandemly repeated DNA families in the human genome;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=533" /><meta name="citation_reference" content="citation_title=Tandemly repeated DNA families in the mouse genome;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=531" /><meta name="citation_reference" content="citation_title=The 1. 688 repetitive DNA of Drosophila: concerted evolution at different genomic scales and association with genes;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2012;  citation_volume=29;  citation_pages=7-11; " /><meta name="citation_reference" content="citation_title=Genomic characterization of large heterochromatic gaps in the human genome assembly;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2014;  citation_volume=10;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=The regulatory regions required for B paramutation and expression are located far upstream of the maize b1 transcribed sequences;  citation_journal_title=Genetics;  citation_year=2002;  citation_volume=162;  citation_pages=917-30; " /><meta name="citation_reference" content="citation_title=BRCA1 tumour suppression occurs via heterochromatin-mediated silencing;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=477;  citation_pages=179-84; " /><meta name="citation_reference" content="citation_title=Insertion of beta-satellite repeats identifies a transmembrane protease causing both congenital and childhood onset autosomal recessive deafness;  citation_journal_title=Nat. Genet.;  citation_year=2001;  citation_volume=27;  citation_pages=59-63; " /><meta name="citation_reference" content="citation_title=Intragene higher order repeats in neuroblastoma breakpoint family genes distinguish humans from chimpanzees;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=1877-92; " /><meta name="citation_reference" content="citation_title=The genome of the model beetle and pest Tribolium castaneum;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=452;  citation_pages=949-55; " /><meta name="citation_reference" content="citation_title=Satellite DNA of the red flour beetle Tribolium castaneum – comparative study of satellites from the genus Tribolium;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=1059-66; " /><meta name="citation_reference" content="citation_title=Conserved patterns in the evolution of Tribolium satellite DNAs.pdf;  citation_journal_title=Gene;  citation_year=2004;  citation_volume=332;  citation_pages=169-77; " /><meta name="citation_reference" content="citation_title=Long inversely oriented subunits form a complex monomer of Tribolium brevicornis satellite DNA;  citation_journal_title=J. Mol. Evol.;  citation_year=2005;  citation_volume=60;  citation_pages=513-25; " /><meta name="citation_reference" content="citation_title=Satellite DNA junctions identify the potential origin of new repetitive elements in the beetle Tribolium madens;  citation_journal_title=Gene;  citation_year=2007;  citation_volume=394;  citation_pages=45-52; " /><meta name="citation_reference" content="citation_title=Parallelism in evolution of highly repetitive DNAs in sibling species;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=1857-67; " /><meta name="citation_reference" content="citation_title=Structure and population dynamics of the major satellite DNA in the red flour beetle Tribolium castaneum;  citation_journal_title=Genetica;  citation_year=2011;  citation_volume=139;  citation_pages=999-1008; " /><meta name="citation_reference" content="citation_title=Satellite DNA-like elements associated with genes within euchromatin of the beetle Tribolium castaneum;  citation_journal_title=G3;  citation_year=2012;  citation_volume=2;  citation_pages=931-41; " /><meta name="citation_reference" content="citation_title=Analysis of repetitive DNA distribution patterns in the Tribolium castaneum genome;  citation_journal_title=Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=14" /><meta name="citation_reference" content="citation_title=Cytogenetics of chromosome rearrangements in Tribolium castaneum;  citation_journal_title=Genome;  citation_year=1995;  citation_volume=38;  citation_pages=673-80; " /><meta name="citation_reference" content="citation_title=TRDB – the Tandem Repeats Database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2006;  citation_volume=35;  citation_pages=D80-7; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis;  citation_journal_title=Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=174-5; " /><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood;  citation_journal_title=Syst. Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=696-704; " /><meta name="citation_reference" content="citation_title=jModelTest 2: more models, new heuristics and parallel computing;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=772" /><meta name="citation_reference" content="citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0;  citation_journal_title=Syst. Biol.;  citation_year=2010;  citation_volume=59;  citation_pages=307-21; " /><meta name="citation_reference" content="citation_title=Repbase update, a database of eukaryotic repetitive elements;  citation_journal_title=Cytogenet. Genome Res.;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=.; citation_publisher=Auckland, New Zealand;  citation_title=Geneious. Version 5.4. Biomatters Ltd;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Chromosome number and sex-determining systems in Tenebrionidae (Coleoptera); citation_publisher=AEC, Barcelona;  citation_title=Advances in Coleopterology;  citation_year=1991;  citation_pages=167-76; " /><meta name="citation_reference" content="citation_title=Centromere identity from the DNA point of view;  citation_journal_title=Chromosoma;  citation_year=2014;  citation_volume=4;  citation_pages=313-25; " /><meta name="citation_reference" content="citation_title=Satellite DNA as a driver of population divergence in the red flour beetle Tribolium castaneum;  citation_journal_title=Genome Biol. Evol.;  citation_year=2014;  citation_volume=7;  citation_pages=228-39; " /><meta name="citation_reference" content="citation_title=The organization and evolution of the responder satellite in species of the Drosophila melanogaster group: dynamic evolution of a target of meiotic drive;  citation_journal_title=BMC Evol. Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Progressive proximal expansion of the primate X chromosome centromere;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=10563-8; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=The structure of DNA in the nucleosome core;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=423;  citation_pages=145-50; " /><meta name="citation_reference" content="citation_title=The octamer is the major form of CENP-A nucleosomes at human centromeres;  citation_journal_title=Nat. Struct. Mol. Biol.;  citation_year=2013;  citation_volume=20;  citation_pages=687-95; " /><meta name="citation_reference" content="citation_title=The CentO satellite confers translational and rotational phasing on cenH3 nucleosomes in rice centromeres;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=E4875-83; " /><meta name="citation_reference" content="citation_title=The roles of the monomer length and nucleotide context of plant tandem repeats in nucleosome positioning;  citation_journal_title=J. Biomol. Struct. Dyn.;  citation_year=2014;  citation_volume=32;  citation_pages=115-26; " /><meta name="citation_reference" content="citation_title=Comparative study of satellite sequences and phylogeny of five species from the genus Palorus (Insecta, Coleoptera);  citation_journal_title=Genome;  citation_year=2000;  citation_volume=43;  citation_pages=776-85; " /><meta name="citation_reference" content="citation_title=A theoretical perusal of the satellite DNA curvature in tenebrionid beetles;  citation_journal_title=J. Biomol. Struct. Dyn.;  citation_year=1998;  citation_volume=16;  citation_pages=41-50; " /><meta name="citation_reference" content="citation_title=Composition and organization of active centromere sequences in complex genomes;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=324" /><meta name="citation_reference" content="citation_title=Transcription and evolutionary dynamics of the centromeric satellite repeat CentO in rice;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=2505-20; " /><meta name="citation_reference" content="citation_title=Global sequence characterization of rice centromeric satellite based on oligomer frequency analysis in large-scale sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2101-8; " /><meta name="citation_reference" content="citation_title=The evolution of African great ape subtelomeric heterochromatin and the fusion of human chromosome 2;  citation_journal_title=Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=1036-49; " /><meta name="citation_reference" content="citation_title=Sequence homogenization and chromosomal localization of VicTR-B satellites differ between closely related Vicia species;  citation_journal_title=Chromosoma;  citation_year=2006;  citation_volume=115;  citation_pages=437-47; " /><meta name="citation_reference" content="citation_title=A novel interspersed type of organization of satellite DNAs in Tribolium madens heterochromatin;  citation_journal_title=Chromosome Res.;  citation_year=2000;  citation_volume=8;  citation_pages=201-12; " /><meta name="citation_reference" content="citation_title=The role of unequal crossover in alpha-satellite DNA evolution: a computational analysis;  citation_journal_title=J. Comput. Biol.;  citation_year=2004;  citation_volume=11;  citation_pages=933-44; " /><meta name="citation_reference" content="citation_title=Tandem repeats derived from centromeric retrotransposons;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=142" /><meta name="citation_reference" content="citation_title=Survey of extrachromosomal circular DNA derived from plant satellite repeats;  citation_journal_title=BMC Plant Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=90" /><meta name="citation_reference" content="citation_title=Major evolutionary transitions in centromere complexity;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=138;  citation_pages=1067-82; " /><meta name="citation_reference" content="citation_title=Species-specific heterochromatin prevents mitotic chromosome segregation to cause hybrid lethality in Drosophila;  citation_journal_title=PLoS Biol.;  citation_year=2009;  citation_volume=7;  citation_pages=e1000234" /><meta name="citation_reference" content="citation_title=Altered heterochromatin binding by a hybrid sterility protein in Drosophila sibling species;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1538-41; " /><meta name="citation_reference" content="citation_title=Correlated variation and population differentiation in satellite DNA abundance among lines of Drosophila melanogaster;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=18793-8; " /><meta name="citation_reference" content="citation_title=Epigenetic effects of polymorphic Y chromosomes modulate chromatin components, immune response, and sexual conflict;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=15826-31; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/387/17462804/dsv021.pdf" /><meta name="description" content="Abstract.  Although satellite DNAs are well-explored components of heterochromatin and centromeres, little is known about emergence, dispersal and possible impa" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/387/2462399" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/387/2462399" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/387/2462399");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv021';
    </script>
<title>
	Genome-wide analysis of tandem repeats in Tribolium castaneum genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462399" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaCKATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGF2bGVrLCBNYXJ0aW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUnXEkWVyIEJvxaFrb3ZpxIcgSW5zdGl0dXRlLCBCaWplbmnEjWthIDU0LCBaYWdyZWIgSFItMTAwMDIsIENyb2F0aWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHZWxmYW5kLCBZZXZnZW5peSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgZm9yIEJpb2NvbXB1dGluZyBhbmQgSW5mb3JtYXRpY3MsIEJvc3RvbiBVbml2ZXJzaXR5LCBCb3N0b24sIE1BIDAyMjE1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQbG9obCwgTWlyb3NsYXYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSdcSRZXIgQm/FoWtvdmnEhyBJbnN0aXR1dGUsIEJpamVuacSNa2EgNTQsIFphZ3JlYiBIUi0xMDAwMiwgQ3JvYXRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1lxaF0cm92acSHLCBOZXZlbmthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUnXEkWVyIEJvxaFrb3ZpxIcgSW5zdGl0dXRlLCBCaWplbmnEjWthIDU0LCBaYWdyZWIgSFItMTAwMDIsIENyb2F0aWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIHRhbmRlbSByZXBlYXRzIGluIFRyaWJvbGl1bSBjYXN0YW5ldW0gZ2Vub21lIHJldmVhbHMgYWJ1bmRhbnQgYW5kIGhpZ2hseSBkeW5hbWljIHRhbmRlbSByZXBlYXQgZmFtaWxpZXMgd2l0aCBzYXRlbGxpdGUgRE5BIGZlYXR1cmVzIGluIGV1Y2hyb21hdGljIGNocm9tb3NvbWFsIGFybXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdjAyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2F0ZWxsaXRlIEROQSBldm9sdXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1LYXJnZXIgUHVibGlzaGVycywgQmFzZWw7ICBjaXRhdGlvbl90aXRsZT1SZXBldGl0aXZlIEROQS4gR2Vub21lIER5biA3OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fcGFnZXM9MTI2LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZW50cm9tZXJlcyBjb252ZXJ0IGJ1dCBkb24ndCBjcm9zczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0YW5kZW0gcmVwZWF0cyBmcm9tIGh1bmRyZWRzIG9mIHNwZWNpZXMgcmV2ZWFscyB1bmlxdWUgaW5zaWdodHMgaW50byBjZW50cm9tZXJlIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvc29tZXMgYW5kIGNlbnRyb21lcmljIEROQSBwYWNrYWdpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGRyaXZlIGluIG11bHRpZ2VuZSBmYW1pbGllczogaG93IGJpb2xvZ2ljYWwgbm92ZWx0aWVzIGFyaXNlLCBzcHJlYWQgYW5kIGFyZSBhc3NpbWlsYXRlZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlzaWNhbCBhbmQgZ2VuZXRpYyBtYXBwaW5nIG9mIHRoZSBodW1hbsOXY2hyb21vc29tZSBjZW50cm9tZXJlOiByZXByZXNzaW9uIG9mIHJlY29tYmluYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMDAtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiByZXBlYXRlZCBETkEgc2VxdWVuY2VzIGJ5IHVuZXF1YWwgY3Jvc3NvdmVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTE5MTsgIGNpdGF0aW9uX3BhZ2VzPTUyOC0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV2b2x1dGlvbmFyeSBkeW5hbWljcyBvZiBhbHBoYS1zYXRlbGxpdGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODgtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRldHJpcyBpcyBhIGZvbGRiYWNrIHRyYW5zcG9zb24gdGhhdCBwcm92aWRlZCB0aGUgYnVpbGRpbmcgYmxvY2tzIGZvciBhbiBlbWVyZ2luZyBzYXRlbGxpdGUgRE5BIG9mIERyb3NvcGhpbGEgdmlyaWxpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xMzAyLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gcmVwZWF0LWNvbnRhaW5pbmcgTUlURSBlbGVtZW50cyBpbiB0aGUgY2xhbSBEb25heCB0cnVuY3VsdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjU0OS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0cmFjaHJvbW9zb21hbCBjaXJjbGVzIG9mIHNhdGVsbGl0ZSByZXBlYXRzIGFuZCA1UyByaWJvc29tYWwgRE5BIGluIGh1bWFuIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2IuIEROQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgbGFyZ2VzdCB0YW5kZW1seSByZXBlYXRlZCBETkEgZmFtaWxpZXMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01MzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW1seSByZXBlYXRlZCBETkEgZmFtaWxpZXMgaW4gdGhlIG1vdXNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NTMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIDEuIDY4OCByZXBldGl0aXZlIEROQSBvZiBEcm9zb3BoaWxhOiBjb25jZXJ0ZWQgZXZvbHV0aW9uIGF0IGRpZmZlcmVudCBnZW5vbWljIHNjYWxlcyBhbmQgYXNzb2NpYXRpb24gd2l0aCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTctMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgY2hhcmFjdGVyaXphdGlvbiBvZiBsYXJnZSBoZXRlcm9jaHJvbWF0aWMgZ2FwcyBpbiB0aGUgaHVtYW4gZ2Vub21lIGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVndWxhdG9yeSByZWdpb25zIHJlcXVpcmVkIGZvciBCIHBhcmFtdXRhdGlvbiBhbmQgZXhwcmVzc2lvbiBhcmUgbG9jYXRlZCBmYXIgdXBzdHJlYW0gb2YgdGhlIG1haXplIGIxIHRyYW5zY3JpYmVkIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYyOyAgY2l0YXRpb25fcGFnZXM9OTE3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CUkNBMSB0dW1vdXIgc3VwcHJlc3Npb24gb2NjdXJzIHZpYSBoZXRlcm9jaHJvbWF0aW4tbWVkaWF0ZWQgc2lsZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc3OyAgY2l0YXRpb25fcGFnZXM9MTc5LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNlcnRpb24gb2YgYmV0YS1zYXRlbGxpdGUgcmVwZWF0cyBpZGVudGlmaWVzIGEgdHJhbnNtZW1icmFuZSBwcm90ZWFzZSBjYXVzaW5nIGJvdGggY29uZ2VuaXRhbCBhbmQgY2hpbGRob29kIG9uc2V0IGF1dG9zb21hbCByZWNlc3NpdmUgZGVhZm5lc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NTktNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJhZ2VuZSBoaWdoZXIgb3JkZXIgcmVwZWF0cyBpbiBuZXVyb2JsYXN0b21hIGJyZWFrcG9pbnQgZmFtaWx5IGdlbmVzIGRpc3Rpbmd1aXNoIGh1bWFucyBmcm9tIGNoaW1wYW56ZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTg3Ny05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgbW9kZWwgYmVldGxlIGFuZCBwZXN0IFRyaWJvbGl1bSBjYXN0YW5ldW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTI7ICBjaXRhdGlvbl9wYWdlcz05NDktNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhdGVsbGl0ZSBETkEgb2YgdGhlIHJlZCBmbG91ciBiZWV0bGUgVHJpYm9saXVtIGNhc3RhbmV1bSDigJMgY29tcGFyYXRpdmUgc3R1ZHkgb2Ygc2F0ZWxsaXRlcyBmcm9tIHRoZSBnZW51cyBUcmlib2xpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xMDU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2ZWQgcGF0dGVybnMgaW4gdGhlIGV2b2x1dGlvbiBvZiBUcmlib2xpdW0gc2F0ZWxsaXRlIEROQXMucGRmOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMzMjsgIGNpdGF0aW9uX3BhZ2VzPTE2OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZyBpbnZlcnNlbHkgb3JpZW50ZWQgc3VidW5pdHMgZm9ybSBhIGNvbXBsZXggbW9ub21lciBvZiBUcmlib2xpdW0gYnJldmljb3JuaXMgc2F0ZWxsaXRlIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTUxMy0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2F0ZWxsaXRlIEROQSBqdW5jdGlvbnMgaWRlbnRpZnkgdGhlIHBvdGVudGlhbCBvcmlnaW4gb2YgbmV3IHJlcGV0aXRpdmUgZWxlbWVudHMgaW4gdGhlIGJlZXRsZSBUcmlib2xpdW0gbWFkZW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5NDsgIGNpdGF0aW9uX3BhZ2VzPTQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbGxlbGlzbSBpbiBldm9sdXRpb24gb2YgaGlnaGx5IHJlcGV0aXRpdmUgRE5BcyBpbiBzaWJsaW5nIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xODU3LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgYW5kIHBvcHVsYXRpb24gZHluYW1pY3Mgb2YgdGhlIG1ham9yIHNhdGVsbGl0ZSBETkEgaW4gdGhlIHJlZCBmbG91ciBiZWV0bGUgVHJpYm9saXVtIGNhc3RhbmV1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTM5OyAgY2l0YXRpb25fcGFnZXM9OTk5LTEwMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhdGVsbGl0ZSBETkEtbGlrZSBlbGVtZW50cyBhc3NvY2lhdGVkIHdpdGggZ2VuZXMgd2l0aGluIGV1Y2hyb21hdGluIG9mIHRoZSBiZWV0bGUgVHJpYm9saXVtIGNhc3RhbmV1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTkzMS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgcmVwZXRpdGl2ZSBETkEgZGlzdHJpYnV0aW9uIHBhdHRlcm5zIGluIHRoZSBUcmlib2xpdW0gY2FzdGFuZXVtIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9nZW5ldGljcyBvZiBjaHJvbW9zb21lIHJlYXJyYW5nZW1lbnRzIGluIFRyaWJvbGl1bSBjYXN0YW5ldW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTY3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VFJEQiDigJMgdGhlIFRhbmRlbSBSZXBlYXRzIERhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EODAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFuZGVtIHJlcGVhdHMgZmluZGVyOiBhIHByb2dyYW0gdG8gYW5hbHl6ZSBETkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURuYVNQIHZlcnNpb24gMzogYW4gaW50ZWdyYXRlZCBwcm9ncmFtIGZvciBtb2xlY3VsYXIgcG9wdWxhdGlvbiBnZW5ldGljcyBhbmQgbW9sZWN1bGFyIGV2b2x1dGlvbiBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNzQtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUsIGZhc3QsIGFuZCBhY2N1cmF0ZSBhbGdvcml0aG0gdG8gZXN0aW1hdGUgbGFyZ2UgcGh5bG9nZW5pZXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTY5Ni03MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWpNb2RlbFRlc3QgMjogbW9yZSBtb2RlbHMsIG5ldyBoZXVyaXN0aWNzIGFuZCBwYXJhbGxlbCBjb21wdXRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NzcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IGFsZ29yaXRobXMgYW5kIG1ldGhvZHMgdG8gZXN0aW1hdGUgbWF4aW11bS1saWtlbGlob29kIHBoeWxvZ2VuaWVzOiBhc3Nlc3NpbmcgdGhlIHBlcmZvcm1hbmNlIG9mIFBoeU1MIDMuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0zMDctMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGJhc2UgdXBkYXRlLCBhIGRhdGFiYXNlIG9mIGV1a2FyeW90aWMgcmVwZXRpdGl2ZSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b2dlbmV0LiBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz00NjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9LjsgY2l0YXRpb25fcHVibGlzaGVyPUF1Y2tsYW5kLCBOZXcgWmVhbGFuZDsgIGNpdGF0aW9uX3RpdGxlPUdlbmVpb3VzLiBWZXJzaW9uIDUuNC4gQmlvbWF0dGVycyBMdGQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWUgbnVtYmVyIGFuZCBzZXgtZGV0ZXJtaW5pbmcgc3lzdGVtcyBpbiBUZW5lYnJpb25pZGFlIChDb2xlb3B0ZXJhKTsgY2l0YXRpb25fcHVibGlzaGVyPUFFQywgQmFyY2Vsb25hOyAgY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4gQ29sZW9wdGVyb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl9wYWdlcz0xNjctNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbnRyb21lcmUgaWRlbnRpdHkgZnJvbSB0aGUgRE5BIHBvaW50IG9mIHZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWE7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTMxMy0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2F0ZWxsaXRlIEROQSBhcyBhIGRyaXZlciBvZiBwb3B1bGF0aW9uIGRpdmVyZ2VuY2UgaW4gdGhlIHJlZCBmbG91ciBiZWV0bGUgVHJpYm9saXVtIGNhc3RhbmV1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMjgtMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBvcmdhbml6YXRpb24gYW5kIGV2b2x1dGlvbiBvZiB0aGUgcmVzcG9uZGVyIHNhdGVsbGl0ZSBpbiBzcGVjaWVzIG9mIHRoZSBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBncm91cDogZHluYW1pYyBldm9sdXRpb24gb2YgYSB0YXJnZXQgb2YgbWVpb3RpYyBkcml2ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZ3Jlc3NpdmUgcHJveGltYWwgZXhwYW5zaW9uIG9mIHRoZSBwcmltYXRlIFggY2hyb21vc29tZSBjZW50cm9tZXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMDU2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3RydWN0dXJlIG9mIEROQSBpbiB0aGUgbnVjbGVvc29tZSBjb3JlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDIzOyAgY2l0YXRpb25fcGFnZXM9MTQ1LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb2N0YW1lciBpcyB0aGUgbWFqb3IgZm9ybSBvZiBDRU5QLUEgbnVjbGVvc29tZXMgYXQgaHVtYW4gY2VudHJvbWVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9Njg3LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ2VudE8gc2F0ZWxsaXRlIGNvbmZlcnMgdHJhbnNsYXRpb25hbCBhbmQgcm90YXRpb25hbCBwaGFzaW5nIG9uIGNlbkgzIG51Y2xlb3NvbWVzIGluIHJpY2UgY2VudHJvbWVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPUU0ODc1LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZXMgb2YgdGhlIG1vbm9tZXIgbGVuZ3RoIGFuZCBudWNsZW90aWRlIGNvbnRleHQgb2YgcGxhbnQgdGFuZGVtIHJlcGVhdHMgaW4gbnVjbGVvc29tZSBwb3NpdGlvbmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbW9sLiBTdHJ1Y3QuIER5bi47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMTUtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHN0dWR5IG9mIHNhdGVsbGl0ZSBzZXF1ZW5jZXMgYW5kIHBoeWxvZ2VueSBvZiBmaXZlIHNwZWNpZXMgZnJvbSB0aGUgZ2VudXMgUGFsb3J1cyAoSW5zZWN0YSwgQ29sZW9wdGVyYSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTc3Ni04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB0aGVvcmV0aWNhbCBwZXJ1c2FsIG9mIHRoZSBzYXRlbGxpdGUgRE5BIGN1cnZhdHVyZSBpbiB0ZW5lYnJpb25pZCBiZWV0bGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9tb2wuIFN0cnVjdC4gRHluLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQxLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wb3NpdGlvbiBhbmQgb3JnYW5pemF0aW9uIG9mIGFjdGl2ZSBjZW50cm9tZXJlIHNlcXVlbmNlcyBpbiBjb21wbGV4IGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTMyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gYW5kIGV2b2x1dGlvbmFyeSBkeW5hbWljcyBvZiB0aGUgY2VudHJvbWVyaWMgc2F0ZWxsaXRlIHJlcGVhdCBDZW50TyBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjUwNS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHNlcXVlbmNlIGNoYXJhY3Rlcml6YXRpb24gb2YgcmljZSBjZW50cm9tZXJpYyBzYXRlbGxpdGUgYmFzZWQgb24gb2xpZ29tZXIgZnJlcXVlbmN5IGFuYWx5c2lzIGluIGxhcmdlLXNjYWxlIHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0yMTAxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb24gb2YgQWZyaWNhbiBncmVhdCBhcGUgc3VidGVsb21lcmljIGhldGVyb2Nocm9tYXRpbiBhbmQgdGhlIGZ1c2lvbiBvZiBodW1hbiBjaHJvbW9zb21lIDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTAzNi00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgaG9tb2dlbml6YXRpb24gYW5kIGNocm9tb3NvbWFsIGxvY2FsaXphdGlvbiBvZiBWaWNUUi1CIHNhdGVsbGl0ZXMgZGlmZmVyIGJldHdlZW4gY2xvc2VseSByZWxhdGVkIFZpY2lhIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWE7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9NDM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGludGVyc3BlcnNlZCB0eXBlIG9mIG9yZ2FuaXphdGlvbiBvZiBzYXRlbGxpdGUgRE5BcyBpbiBUcmlib2xpdW0gbWFkZW5zIGhldGVyb2Nocm9tYXRpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMDEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIHVuZXF1YWwgY3Jvc3NvdmVyIGluIGFscGhhLXNhdGVsbGl0ZSBETkEgZXZvbHV0aW9uOiBhIGNvbXB1dGF0aW9uYWwgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz05MzMtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhbmRlbSByZXBlYXRzIGRlcml2ZWQgZnJvbSBjZW50cm9tZXJpYyByZXRyb3RyYW5zcG9zb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdXJ2ZXkgb2YgZXh0cmFjaHJvbW9zb21hbCBjaXJjdWxhciBETkEgZGVyaXZlZCBmcm9tIHBsYW50IHNhdGVsbGl0ZSByZXBlYXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFqb3IgZXZvbHV0aW9uYXJ5IHRyYW5zaXRpb25zIGluIGNlbnRyb21lcmUgY29tcGxleGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzg7ICBjaXRhdGlvbl9wYWdlcz0xMDY3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjaWVzLXNwZWNpZmljIGhldGVyb2Nocm9tYXRpbiBwcmV2ZW50cyBtaXRvdGljIGNocm9tb3NvbWUgc2VncmVnYXRpb24gdG8gY2F1c2UgaHlicmlkIGxldGhhbGl0eSBpbiBEcm9zb3BoaWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDIzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVyZWQgaGV0ZXJvY2hyb21hdGluIGJpbmRpbmcgYnkgYSBoeWJyaWQgc3RlcmlsaXR5IHByb3RlaW4gaW4gRHJvc29waGlsYSBzaWJsaW5nIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI2OyAgY2l0YXRpb25fcGFnZXM9MTUzOC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRlZCB2YXJpYXRpb24gYW5kIHBvcHVsYXRpb24gZGlmZmVyZW50aWF0aW9uIGluIHNhdGVsbGl0ZSBETkEgYWJ1bmRhbmNlIGFtb25nIGxpbmVzIG9mIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xODc5My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIGVmZmVjdHMgb2YgcG9seW1vcnBoaWMgWSBjaHJvbW9zb21lcyBtb2R1bGF0ZSBjaHJvbWF0aW4gY29tcG9uZW50cywgaW1tdW5lIHJlc3BvbnNlLCBhbmQgc2V4dWFsIGNvbmZsaWN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xNTgyNi0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzYvMzg3LzE3NDYyODA0L2RzdjAyMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWx0aG91Z2ggc2F0ZWxsaXRlIEROQXMgYXJlIHdlbGwtZXhwbG9yZWQgY29tcG9uZW50cyBvZiBoZXRlcm9jaHJvbWF0aW4gYW5kIGNlbnRyb21lcmVzLCBsaXR0bGUgaXMga25vd24gYWJvdXQgZW1lcmdlbmNlLCBkaXNwZXJzYWwgYW5kIHBvc3NpYmxlIGltcGEiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI0NjIzOTkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk65MFV8CsmOvXqU66kYzRgBOWHn4=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f387%2f2462399" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f387%2f2462399" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196552">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196552">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196565">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196565">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196596">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196596">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196644">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196644">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196700">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196700">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196720">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196720">Authors’ contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196724">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196724">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196727">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196727">Acknowledgments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196730">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196730">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/22/6/403/2462415">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide analysis of tandem repeats in <em>Tribolium castaneum</em> genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martina Pavlek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Ruđer Bošković Institute</div>, <div class="addr-line">Bijenička 54, Zagreb HR-10002</div>, <div class="country">Croatia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martina+Pavlek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pavlek M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pavlek M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martina Pavlek</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yevgeniy Gelfand</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory for Biocomputing and Informatics</div>, <div class="insititution">Boston University</div>, <div class="addr-line">Boston, MA 02215</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yevgeniy+Gelfand">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gelfand Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gelfand Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yevgeniy Gelfand</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Miroslav Plohl</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Ruđer Bošković Institute</div>, <div class="addr-line">Bijenička 54, Zagreb HR-10002</div>, <div class="country">Croatia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Miroslav+Plohl">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Plohl M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Plohl M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Miroslav Plohl</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nevenka Meštrović</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Ruđer Bošković Institute</div>, <div class="addr-line">Bijenička 54, Zagreb HR-10002</div>, <div class="country">Croatia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +385 1-4571-273. Fax. +385 1-4561-177. E-mail: <a href="mailto:nevenka@irb.hr" target="_blank">nevenka@irb.hr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nevenka+Me%c5%a1trovi%c4%87">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Meštrović N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Meštrović N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nevenka Meštrović</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 387–401, <a href='https://doi.org/10.1093/dnares/dsv021'>https://doi.org/10.1093/dnares/dsv021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 October 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 June 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 August 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462399" href="/dnaresearch/article-pdf/22/6/387/17462804/dsv021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Martina Pavlek, Yevgeniy Gelfand, Miroslav Plohl, Nevenka Meštrović;  Genome-wide analysis of tandem repeats in <em>Tribolium castaneum</em> genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 387–401, <a href='https://doi.org/10.1093/dnares/dsv021'>https://doi.org/10.1093/dnares/dsv021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462399?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20analysis%20of%20tandem%20repeats%20in%20%26lt%3Bem%26gt%3BTribolium%20castaneum%26lt%3B%2Fem%26gt%3B%20genome%20reveals%20abundant%20and%20highly%20dynamic%20tandem%20repeat%20families%20with%20satellite%20DNA%20features%20in%20euchromatic%20chromosomal%20arms&amp;publicationDate=2015-10-01&amp;volumeNum=22&amp;issueNum=6&amp;author=Pavlek%2C%20Martina%3B%20Gelfand%2C%20Yevgeniy&amp;startPage=387&amp;endPage=401&amp;contentId=10.1093%2Fdnares%2Fdsv021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f387%2f2462399" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87196552 id="87196552" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Although satellite DNAs are well-explored components of heterochromatin and centromeres, little is known about emergence, dispersal and possible impact of comparably structured tandem repeats (TRs) on the genome-wide scale. Our bioinformatics analysis of assembled <em>Tribolium castaneum</em> genome disclosed significant contribution of TRs in euchromatic chromosomal arms and clear predominance of satellite DNA-typical 170 bp monomers in arrays of ≥5 repeats. By applying different experimental approaches, we revealed that the nine most prominent TR families Cast1–Cast9 extracted from the assembly comprise ∼4.3% of the entire genome and reside almost exclusively in euchromatic regions. Among them, seven families that build ∼3.9% of the genome are based on ∼170 and ∼340 bp long monomers. Results of phylogenetic analyses of 2500 monomers originating from these families show high-sequence dynamics, evident by extensive exchanges between arrays on non-homologous chromosomes. In addition, our analysis shows that concerted evolution acts more efficiently on longer than on shorter arrays. Efficient genome-wide distribution of nine TR families implies the role of transposition only in expansion of the most dispersed family, and involvement of other mechanisms is anticipated. Despite similarities in sequence features, FISH experiments indicate high-level compartmentalization of centromeric and euchromatic tandem repeats.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Tandem repeats&quot;">Tandem repeats</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;euchromatic regions&quot;">euchromatic regions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;evolutionary trends&quot;">evolutionary trends</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transposition">transposition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Tribolium castaneum&quot;"><em>Tribolium castaneum</em></a></div>                            <h2 scrollto-destination=87196565 id="87196565" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Eukaryotes typically display high proportions of repetitive elements that outmatch 50% of the genome. Among these repetitive elements, satellite DNAs (satDNAs) are a class of diverse tandemly repeated DNA sequences that build long arrays located in heterochromatin and often represent the most abundant genome fraction.<sup><a reveal-id="DSV021C1" data-open="DSV021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> They are usually highly prevalent at and around centromeres, which are regions with suppressed recombination.<sup><a reveal-id="DSV021C2" data-open="DSV021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Centromeric satDNAs change rapidly during evolution despite their conserved function at the centromeric locus.<sup><a reveal-id="DSV021C3" data-open="DSV021C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In addition to extreme diversity in nucleotide sequences between species, centromeric satDNAs are typically characterized by sequential arrangement of monomers in the form of long arrays and by preferential monomer length corresponding to the size of nucleosomal DNA.<sup><a reveal-id="DSV021C4" data-open="DSV021C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Our current knowledge about satDNA evolution is mostly based on studies of centromeric satDNAs. SatDNAs evolve in concert as a result of molecular drive, a process by which mutations are homogenized throughout a family of monomers in a genome, and are fixed in a population.<sup><a reveal-id="DSV021C5" data-open="DSV021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Theoretical models predict unequal crossing-over and gene conversion as the most widespread mechanisms involved in dynamics of satDNAs evolution.<sup><a reveal-id="DSV021C2" data-open="DSV021C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV021C6" data-open="DSV021C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV021C7" data-open="DSV021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Studies on human centromeric alpha satDNA show that these mechanisms act more efficiently within arrays than between arrays, and decrease progressively between arrays on homologous and on heterologous chromosomes.<sup><a reveal-id="DSV021C8" data-open="DSV021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Internal tandem repeats (TRs) found in some transposable elements (TEs) boost them to propagate satellite sequences throughout the genome.<sup><a reveal-id="DSV021C9" data-open="DSV021C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSV021C10" data-open="DSV021C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In addition, recent results strongly support the idea that rolling circle amplification and reinsertion of extrachromosomal circles can be important for efficient dispersion of satellite arrays through a genome.<sup><a reveal-id="DSV021C11" data-open="DSV021C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>Genome-wide annotation and study of TR-rich regions from assemblies of complex genomes represent a challenge. Due to the long arrays composed of nearly identical monomers, these genome fractions remain the most poorly mapped in assembled genomes. Moreover, centromeric regions are commonly omitted from the assembly of a genome. Comprehensive bioinformatics analysis of large arrays of TRs (&gt;10 kb) located in the euchromatic part of the human genome showed a wide range of monomer size variations, from several nucleotides to several kilobases.<sup><a reveal-id="DSV021C12" data-open="DSV021C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Among them, one of the largest non-centromeric arrays is 600 kb to 1.7 Mbp long, being located on human chromosome 8 and composed of 12 kb long monomers. Bioinformatics analysis of two mouse whole-genome shotgun assemblies revealed eight new satDNAs including some chromosome-specific satDNA subfamilies, which serve as a unique chromosome bar code.<sup><a reveal-id="DSV021C13" data-open="DSV021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Concerning evolutionary dynamics, comparative analysis of the most abundant satDNA in <em>D. melanogaster</em>, known as 1.688 satDNA, shows differential rates of concerted evolution in distinct genomic regions, from euchromatin to centromeric heterochromatin.<sup><a reveal-id="DSV021C14" data-open="DSV021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In addition to variations in chromosomal distribution, a novel alignment-free algorithm applied to Human Satellite 3 estimates extreme array size variation (7–98 Mb) of the HSat3 between individuals on the Y chromosome, thus confirming considerable satDNA array size polymorphisms.<sup><a reveal-id="DSV021C15" data-open="DSV021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Recent reports on satDNA impact on euchromatic genome suggests roles in modulation of gene regulation,<sup><a reveal-id="DSV021C16" data-open="DSV021C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV021C17" data-open="DSV021C17" class="link link-ref link-reveal xref-bibr">17</a></sup> in disease-associated gene mutations<sup><a reveal-id="DSV021C18" data-open="DSV021C18" class="link link-ref link-reveal xref-bibr">18</a></sup> as well as in accumulation of differences within genes between human and chimpanzee.<sup><a reveal-id="DSV021C19" data-open="DSV021C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p><em>Tribolium castaneum</em> is considered to be the most important model organism after <em>Drosophila</em> in studies of insect development, population genetics, and comparative genomics. Its genome is also the first coleopteran genome to be sequenced.<sup><a reveal-id="DSV021C20" data-open="DSV021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The genomes of <em>Tribolium</em> beetles are characterized by massive blocks of one or two abundant species-specific satDNAs localized in centromeric regions.<sup><a reveal-id="DSV021C21  DSV021C25" data-open="DSV021C21  DSV021C25" class="link link-ref link-reveal xref-bibr">21–25</a></sup> In <em>T. castaneum</em>, two related subfamilies of TCAST satDNA, estimated to comprise up to 35% of the whole genome, encompass the centromeric heterochromatin.<sup><a reveal-id="DSV021C26" data-open="DSV021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Furthermore, the two types of TCAST-like elements are found dispersed within euchromatin. The first represents TCAST satellite-like elements in the form of short arrays (up to tetramers), whereas the second consists of TCAST-like elements embedded within a complex unit similar to DNA transposon.<sup><a reveal-id="DSV021C27" data-open="DSV021C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>Three complementary approaches, used for <em>de novo</em> genome-wide identification of repetitive DNA, recovered &gt;30% of repetitive DNA in the assembled part of the <em>T. castaneum</em> genome.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> All <em>T. castaneum</em> chromosomes are characterized by large blocks of heterochromatin surrounding centromeres, while no prominent heterochromatic blocks could be detected cytogenetically on chromosome arms.<sup><a reveal-id="DSV021C29" data-open="DSV021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Although centromeric satDNA is underrepresented in the assembled genome of <em>T. castaneum</em> analyses performed by Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> revealed significant portion of TRs with monomer over 100 bp in length. These results indicate the existence of a number of satDNA candidate sequences distributed outside the centromeric regions, in euchromatic chromosomal arms. The availability of a genome assembly based on whole-genome shotgun (WGS) reads as well as on Fosmid and BAC end-sequences offers a good platform for genome-wide study of TRs. A similar study of evolutionary trends of 1.688 repetitive DNA has already been performed for the <em>Drosophila</em> genome<sup><a reveal-id="DSV021C14" data-open="DSV021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> where sequencing and assembly approaches were used in a way similar to that in <em>T. castaneum</em>.<sup><a reveal-id="DSV021C20" data-open="DSV021C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p><p>In this work, combining bioinformatics and experimental approaches, we identified and studied content, distribution, and structural features of TRs in the <em>T. castaneum</em> genome. Given the general lack of knowledge about TRs in euchromatic regions, we further focused on the nine most abundant TR families detected in the assembled genome. FISH experiments confirmed their almost exclusive localization in euchromatic chromosomal arms, while phylogenetic analyses revealed their highly dynamic evolution, particularly evident in their extensive exchanges between non-homologous chromosomes. Our results also suggest that in addition to other mechanisms, transposition may play an important role in the efficient spread of the most expanded TR family.</p>                            <h2 scrollto-destination=87196596 id="87196596" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196609 id="87196609" class="section-title" data-legacy-id=s2a>2.1 Satellite DNA database construction and phylogenetic analyses</h3>
<p>The <em>T. castaneum</em> (Tcas3.0) genome was assembled using the gold-standard Sanger assembly strategy with the benefit of extensive genetic maps obtained by the Tribolium Genome Sequencing Consortium.<sup><a reveal-id="DSV021C20" data-open="DSV021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> More detailed, high-quality sequence reads were produced from WGS sequencing libraries of ∼3 and 6 kb in pUC18 subclones, as well as additional reads from fosmids (40 kb) and BACs (130 kb). These reads were assembled using the Atlas suite of assembly tools.<sup><a reveal-id="DSV021C20" data-open="DSV021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The assembled genome of <em>T. castanem</em> was downloaded in the fasta format from the web page <a class="link link-uri" href="http://ftp://ftp.bioinformatics.ksu.edu/pub/BeetleBase/3.0/">ftp://ftp.bioinformatics.ksu.edu/pub/BeetleBase/3.0/</a> (The third version of the assembly: Tcas_3.0) in the form of 10 chromosomes. The sequence of each chromosome was uploaded to Tandem Repeats Database (TRDB),<sup><a reveal-id="DSV021C30" data-open="DSV021C30" class="link link-ref link-reveal xref-bibr">30</a></sup><a class="link link-uri" href="https://tandem.bu.edu/cgibin/trdb/trdb.exe">https://tandem.bu.edu/cgibin/trdb/trdb.exe</a> and analysed using Tandem Repeats Finder (TRF) algorithm.<sup><a reveal-id="DSV021C31" data-open="DSV021C31" class="link link-ref link-reveal xref-bibr">31</a></sup> All chromosomes have been processed with TRF using alignment parameters 2, 7, 7 for match, mismatch, and indels, respectively, and a minimum alignment score of 50. The range of the period size 100–500 bp was selected to exclude the micro/minisatellite fraction. The initial raw TRF output included TR arrays with overlapping genome coordinates. Using the redundancy elimination tool in TRDB and additional manual elimination, arrays with the shorter monomer units were selected for further analyses. Arrays with unspecified (N) nucleotides in both flanking regions were removed. Analysis of monomer length trends in extracted arrays was performed using the filtering option for copy number in arrays. For further analyses arrays with ≥5 monomers were selected. They were merged and analysed with the clustering tool integrated in TRDB. Conditions were as follows: <em>P</em>-value excluded (set to 0), cut-off value set at 70%, heuristical and DUST algorithm excluded, PAM algorithm included with default values (0.7 and 0.3). The result of these analyses was formation of clusters, i.e. groups of arrays that represent TR families. Monomer sequences of all arrays from selected clusters were downloaded in the fasta format. Left and right flanking regions (extending up to 4000 bp) of all selected clusters were also downloaded. Multiple sequence alignments of monomers were done using Clustal W for each TR family. Alignments without truncated monomers from the beginning and the end of the array were used for phylogenetic analyses. The Lasergene software package v.7.0.0 (DnaStar) was used in dot plot analyses and PCR primer design. Monomer sequence variability was analysed using DnaSP v.4.10.9.<sup><a reveal-id="DSV021C32" data-open="DSV021C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Maximum likelihood (ML) trees based on Clustal W alignment were obtained with the PhyML 3.0 software<sup><a reveal-id="DSV021C33" data-open="DSV021C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using best-fit models calculated by the jModelTest 2.1.3.<sup><a reveal-id="DSV021C34" data-open="DSV021C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Due to the large number of monomers, branch support was evaluated using the approximate likelihood ratio test.<sup><a reveal-id="DSV021C35" data-open="DSV021C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Trees were displayed with FigTree v1.40 and adjusted in Corel11 software. All new TR families were blasted against the NCBI GenBank Database and Repbase<sup><a reveal-id="DSV021C36" data-open="DSV021C36" class="link link-ref link-reveal xref-bibr">36</a></sup> to check similarity with published sequences. Sequence editing, selection of restriction enzymes (REs) and MUSCLE alignments of flanking regions were performed using the Geneious 5.4.3 program.<sup><a reveal-id="DSV021C37" data-open="DSV021C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In order to extract TCAST variants from unassembled reads, TRF and clustering analyses were done on 2153 unassembled reads (<a class="link link-uri" href="http://ftp://ftp.bioinformatics.ksu.edu/pub/BeetleBase/3.0/">ftp://ftp.bioinformatics.ksu.edu/pub/BeetleBase/3.0/</a>) under the TRF parameters described above. The consensus sequences of five TCAST subfamilies were constructed based on multiple alignments of all extracted variants (available upon request). Recombinant clones with monomers or dimers of satellite DNA were sequenced by the Macrogen Europe Laboratory (Amsterdam, The Netherlands). Monomers of 9 TR families were deposited in EMBL databank under Accession Numbers: Cast1 (KP846079-KP846566), Cast2 (KP846568-KP847079), Cast3 (KP847080-KP847309), Cast4 (KP847310-KP847735), Cast5 (KP847736-KP848112), Cast6 (KP848113-KP848269), Cast7 (KP848352-KP848270), Cast8 (KP848353-KP848472), Cast9 (KP848473-KP848536) and consensus sequence of new TCAST subfamilies (KR046220-KR046222).</p>                            <h3 scrollto-destination=87196623 id="87196623" class="section-title" data-legacy-id=s2b>2.2 DNA isolation, cloning and sequencing</h3>
<p>A <em>T. castaneum</em> culture (laboratory strain) was obtained from the Central Science Laboratory (Sand Hutton, York, UK). Insects were maintained on flour and kept in glass jars at room temperature, in a laboratory at the Ruđer Bošković Institute. Genomic DNA was isolated from adults by standard phenol–chloroform extraction. Primers were constructed based on the consensus sequences of each TR families as well as on the R66-like flanking region. Primer sequences and their positions on monomer consensus sequences are indicated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4 and Table S1</a></span>. The reaction mixture consisted of a reaction buffer, 1.5 mM MgCl2, 0.2 mM dNTPs, 0.5 U GoTaq DNA polymerase (Promega), 0.4 mM of each primer and 20 ng of genomic DNA. The PCR cycling parameters used were as follows: 2 min initial denaturation at 94°C, followed by 30 cycles of: 95°C for 30 s, 55°C for 30 s and 72°C for 1 min. Final extension was at 72°C for 10 min. PCR products were purified using the QIAquick PCR Purification Kit (Qiagen). PCR products were ligated in a pGEM T-Easy vector (Promega) and transformed in <em>Escherichia coli</em> DH5a-competent cells (Invitrogene). Recombinant clones with monomers or dimers of satellite DNA were sequenced by the Macrogen Europe Laboratory (Amsterdam, The Netherlands).</p>                            <h3 scrollto-destination=87196629 id="87196629" class="section-title" data-legacy-id=s2c>2.3 Southern and dot blot analyses</h3>
<p>Standard procedures were used for restriction endonuclease digestions, electrophoresis and transfer to nylon membranes. For genomic Southern hybridization analysis, 8 µg of genomic DNA was digested with REs which cut once in most monomer sequences and REs with recognition sites only in some monomers; HinfI and HaeIII (Cast1), EcoRI and HaeIII (Cast2), HaeIII and HinfI (Cast4), RsaI and DraI (Cast 5) and HaeIII and HinfI (Cast7). Digested DNA was separated by electrophoresis in a 0.8% agarose, denatured and DNA transferred to Hybond N+ membrane (Amersham). Cloned satellite monomers labelled with biotin-16-dUTP by PCR were used as hybridization probes. Hybridizations were performed overnight under moderate stringency conditions (65°C) in buffer containing 250 mM Na<sub>2</sub>HPO<sub>4</sub> (pH 7.2), 20% SDS, 1 mM EDTA, 0.5% blocking reagent and 50 ng/ml of the probe. Post-hybridization washes were done in 20 mM Na<sub>2</sub>HPO<sub>4</sub>/1 mM EDTA/1% SDS at a temperature 2°C lower than the hybridization temperature. Chemiluminescent detection was carried out using the alkaline phosphatase substrate CDP-Star (Roche Applied Science). The abundance of TR families was estimated by quantitative dot-blot analysis using a series of genomic DNA dilutions ranging from 50 to 200 ng. Satellite monomers, excised from a plasmid, were dot-blotted in the range between 0.05 and 1 ng, and used as a calibration curve.</p>                            <h3 scrollto-destination=87196639 id="87196639" class="section-title" data-legacy-id=s2d>2.4 Chromosome preparations and two-colour fluorescence <em>in situ</em> hybridization</h3>
<p>Male gonads were isolated from adults and chromosome spreads were prepared by the ‘squash’ technique as described previously.<sup><a reveal-id="DSV021C38" data-open="DSV021C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Detailed mapping of nine new TR families on chromosomes in meiotic pro metaphase was not possible due to extremely rare observation of this phase. The condensation state in this phase also causes lower FISH sensitivity especially in the case of low copy families. Technical difficulties were overcome by using chromosomes in mitotic pro metaphase which enabled detection of centromere regions together with FISH signals of newly detected TR families. Two-colour FISH was carried out to determine the positions of new TR families related to centromeric regions. A TCAST-specific probe was generated by nick translation labelling of cloned dimer sequences with Cy3-dUTP using the Nick Translation Mix (Roche Applied Science). FISH probes for TR families detected in this work (Cast1–Cast9) were obtained by PCR labelling of cloned monomers with biotin-16-dUTP (Roche Applied Science) by using M13 forward and reverse primers (Invitrogen). In order to investigate <em>in situ</em> localization of Cast5 arrays with respect to their flanking regions, two-colour FISH was used. Probes were generated by PCR labelling of cloned Cast5 with Cy3-dUTP and R66-like flanking sequences with biotin-16-dUTP. Hybridizations were performed for 18 h at 37°C in a solution containing 60% formamide, 2× SSC, 10% dextran sulphate, 20 mM sodium phosphate, and 10 ng/μl of each probe. Post-hybridization washes were done in 50% formamide/2× SSC at 37°C. Biotin-labelled probes were detected with fluorescein avidin D and biotinylated antiavidin D (Vector Laboratories). The chromosomes were counter-stained with DAPI 4′6-diamidino-2-phenylindole (Invitrogen) and analysed with appropriate filters on an Olympus BX51 epifluorescence microscope equipped with an Olympus DP70 digital camera system. Merging of images was performed using Adobe Photoshop CS5 Extended Version 12.0 software.</p>                            <h2 scrollto-destination=87196644 id="87196644" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87196646 id="87196646" class="section-title" data-legacy-id=s3a>3.1 Distribution and monomer length of TRs in the assembled <em>T. castaneum</em> genome</h3>
<p>In order to identify features and distribution of TRs in the output of the assembled <em>T. castaneum</em> genome, we analysed all 10 assembled chromosomes individually using the TRF software. Two thousand nine hundred and sixty arrays of TRs comprising a total length of 3.25 Mb were obtained. These repeats constitute 2.1% of the 156 Mb long <em>T. castaneum</em> genome assembly. To determine whether TRs identified in our study might show any preferential genome localization with respect to the repeat copy number, distributions of short (&lt;5 monomers) and long (≥5 monomers) arrays were analysed separately and presented along 10 <em>T. castaneum</em> chromosomes (Fig. <a reveal-id="DSV021F1" data-open="DSV021F1" class="link link-reveal link-table xref-fig">1</a>). The 300 kb long uncaptured gaps between scaffolds were taken into account to mark dominant gaps in the assembled genome. The abundance of short and long TR arrays is evidently higher in CH3, CH6, CH8, CH9 and CH10 than in other chromosomes. The obtained results are in accordance with the previous study of Wang et al.,<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> where RepeatScout and TEpipe tools revealed higher accumulation of the repetitive class named HighA and of TEs in the same chromosomal subset. Here, short TR arrays (&lt;5 monomers) showed almost uniform distribution along the assembled chromosomes (Fig. <a reveal-id="DSV021F1" data-open="DSV021F1" class="link link-reveal link-table xref-fig">1</a>). Interestingly, long arrays (≥5 monomers) displayed a higher tendency to reside in the euchromatic chromosomal compartment, being less represented in the putatively heterochromatic domains proposed by Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Although the observed trend of long array distribution could be affected by gaps in the assembly of tandemly repeated sequences, marked uncaptured gaps do not seem to be more frequent in putative heterochromatic domains than in euchromatic chromosomal segments. Declared heterochromatic domains in the assembled genome represent chromosomal regions with high proportion of HighA and TE repetitive classes. Taking into account that the HighA sequence library includes 30% of TEs obtained by the TEpipe algorithm, and only a small fraction (∼6%) of TRs obtained by TRF,<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> we conclude that HighA is mostly built of dispersed repetitive sequences, for example, non-autonomous TEs. </p>                        <a id="87196648" scrollto-destination="87196648"></a>
<div id="DSV021F1" data-id="DSV021F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02101.png?Expires=2147483647&Signature=V9yX22JlMOwIerpf2mtFv5Ii8g1RfIjrLsA6EWaLc-RAgP5NhRIMamJVPEHP3zgUsAc-SIVRrE8CW9vkledVkaG0Yq3ZBCOA370vGluz369JIgg9uBv3jhLV1Y6rfXSKQDa4SNHQ3Kip70fPVOJyrP37VVqF6uzHXgGD4BNWZuWhrxfUB5FWDY4s5nMgBUnSgYt0HIi2r8uDV0aOCXHqmhXnqGte0VHwNf21etg870isDuJKaOdFZuSE-iaNAuwk~V2jlRTISOcPb7NZf9mU0iYY3RA~t5cu9DOwhPQw6e0gH-sSevG95kU5Uer3ZhyN7~3Wpci7uXaWSxiQq0TpFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic distribution of arrays of TRs on T. castaneum assembled chromosomes (CH1–CH10). Vertical bars represent short arrays (&lt;5 monomers/array, upper line) and long arrays (≥5 monomers/array, lower line). The actual number of arrays per Mb for short and long arrays is indicated above each chromosome. Red dots correspond to centromeric TCAST satDNA found in the assembled genome. Horizontal bar represents putative euchromatin (white) and heterochromatin (HighA domain, grey) regions as identified in Wang et al.28 Locations of the 300 kb placeholders were included to define uncaptured gaps (yellow). Red triangles indicate assumed position of the centromere and large blocks of centromeric heterochromatin based on our FISH analyses (Fig. 4), chromosome banding29 and HighA domain (putative heterochromatin) defined in Wang et al.28" path-from-xml="dsv02101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196648/dsv02101.tiff" path-from-xml="dsv02101.tiff" target="_blank">View large</a><a section="87196648" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02101.png?Expires=2147483647&Signature=yYfxBSRvv4xqWcgHol-GGXCta6ojnULEAaNuIo2e5TbVsfq4ESVq2d64-9Axouq92bR8Tpouxi-EAeTBQdLviW7-s-9ida7VLZqgLvmgtyncx4D5jSjMAeWVJCIbvXxgiziKa5S0mwOFdbknyWxggv6zgNVIB4uIuZUm1zgYomR2m~NXRdaCw8SRbB2hMonbbgf4AajhgmmWzmLPn5pDiINojvVETaymkiKbp4jILxtBMGJ0ekYTM476BWnFNlHb1uRI2-zv4tEvsWFezru2d0vWWQWbEJxbOp5asgmFjSD8yCRd9Y7wAioJF2xHmXIAQwVdIPexJIY8yi2auvjfTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196648&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic distribution of arrays of TRs on <em>T. castaneum</em> assembled chromosomes (CH1–CH10). Vertical bars represent short arrays (&lt;5 monomers/array, upper line) and long arrays (≥5 monomers/array, lower line). The actual number of arrays per Mb for short and long arrays is indicated above each chromosome. Red dots correspond to centromeric TCAST satDNA found in the assembled genome. Horizontal bar represents putative euchromatin (white) and heterochromatin (HighA domain, grey) regions as identified in Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Locations of the 300 kb placeholders were included to define uncaptured gaps (yellow). Red triangles indicate assumed position of the centromere and large blocks of centromeric heterochromatin based on our FISH analyses (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>), chromosome banding<sup><a reveal-id="DSV021C29" data-open="DSV021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and HighA domain (putative heterochromatin) defined in Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p></div></div><div content-id="DSV021F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02101.png?Expires=2147483647&Signature=V9yX22JlMOwIerpf2mtFv5Ii8g1RfIjrLsA6EWaLc-RAgP5NhRIMamJVPEHP3zgUsAc-SIVRrE8CW9vkledVkaG0Yq3ZBCOA370vGluz369JIgg9uBv3jhLV1Y6rfXSKQDa4SNHQ3Kip70fPVOJyrP37VVqF6uzHXgGD4BNWZuWhrxfUB5FWDY4s5nMgBUnSgYt0HIi2r8uDV0aOCXHqmhXnqGte0VHwNf21etg870isDuJKaOdFZuSE-iaNAuwk~V2jlRTISOcPb7NZf9mU0iYY3RA~t5cu9DOwhPQw6e0gH-sSevG95kU5Uer3ZhyN7~3Wpci7uXaWSxiQq0TpFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic distribution of arrays of TRs on T. castaneum assembled chromosomes (CH1–CH10). Vertical bars represent short arrays (&lt;5 monomers/array, upper line) and long arrays (≥5 monomers/array, lower line). The actual number of arrays per Mb for short and long arrays is indicated above each chromosome. Red dots correspond to centromeric TCAST satDNA found in the assembled genome. Horizontal bar represents putative euchromatin (white) and heterochromatin (HighA domain, grey) regions as identified in Wang et al.28 Locations of the 300 kb placeholders were included to define uncaptured gaps (yellow). Red triangles indicate assumed position of the centromere and large blocks of centromeric heterochromatin based on our FISH analyses (Fig. 4), chromosome banding29 and HighA domain (putative heterochromatin) defined in Wang et al.28" path-from-xml="dsv02101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196648/dsv02101.tiff" path-from-xml="dsv02101.tiff" target="_blank">View large</a><a section="87196648" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02101.png?Expires=2147483647&Signature=yYfxBSRvv4xqWcgHol-GGXCta6ojnULEAaNuIo2e5TbVsfq4ESVq2d64-9Axouq92bR8Tpouxi-EAeTBQdLviW7-s-9ida7VLZqgLvmgtyncx4D5jSjMAeWVJCIbvXxgiziKa5S0mwOFdbknyWxggv6zgNVIB4uIuZUm1zgYomR2m~NXRdaCw8SRbB2hMonbbgf4AajhgmmWzmLPn5pDiINojvVETaymkiKbp4jILxtBMGJ0ekYTM476BWnFNlHb1uRI2-zv4tEvsWFezru2d0vWWQWbEJxbOp5asgmFjSD8yCRd9Y7wAioJF2xHmXIAQwVdIPexJIY8yi2auvjfTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196648&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic distribution of arrays of TRs on <em>T. castaneum</em> assembled chromosomes (CH1–CH10). Vertical bars represent short arrays (&lt;5 monomers/array, upper line) and long arrays (≥5 monomers/array, lower line). The actual number of arrays per Mb for short and long arrays is indicated above each chromosome. Red dots correspond to centromeric TCAST satDNA found in the assembled genome. Horizontal bar represents putative euchromatin (white) and heterochromatin (HighA domain, grey) regions as identified in Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Locations of the 300 kb placeholders were included to define uncaptured gaps (yellow). Red triangles indicate assumed position of the centromere and large blocks of centromeric heterochromatin based on our FISH analyses (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>), chromosome banding<sup><a reveal-id="DSV021C29" data-open="DSV021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and HighA domain (putative heterochromatin) defined in Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p></div></div><p>To explore trends of monomer length in extracted arrays and a possible correlation with copy number of monomers, all 2960 arrays were subdivided into three classes: arrays with ≤2 monomers (634 arrays), 3–4 monomers (1563 arrays) and ≥5 monomers (763 arrays). Each class was analysed separately (Fig. <a reveal-id="DSV021F2" data-open="DSV021F2" class="link link-reveal link-table xref-fig">2</a>). Arrays with only two repeat units are predominantly built of monomers having a length between 100 and 180 bp, while the number of array drops with increased monomer size (Fig. <a reveal-id="DSV021F2" data-open="DSV021F2" class="link link-reveal link-table xref-fig">2</a>A). Arrays with 3–4 monomers as well as arrays ≥5 are predominantly built of 160–180 bp long monomers, while the relative contribution of arrays with &lt;160 bp long monomers drops significantly (Fig. <a reveal-id="DSV021F2" data-open="DSV021F2" class="link link-reveal link-table xref-fig">2</a>B and C). In the same time, the profile of arrays is enriched with 200–220 and 320–340 bp long monomers (Fig. <a reveal-id="DSV021F2" data-open="DSV021F2" class="link link-reveal link-table xref-fig">2</a>B and C). Furthermore, a dramatic decrease in the number of long arrays is evident when monomer length exceeds 340 bp (Fig. <a reveal-id="DSV021F2" data-open="DSV021F2" class="link link-reveal link-table xref-fig">2</a>C). </p>                        <a id="87196653" scrollto-destination="87196653"></a>
<div id="DSV021F2" data-id="DSV021F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02102.png?Expires=2147483647&Signature=WVc7EWg4dEDwVToAj3jUm4r8UmXGbFoF1YTp4JLmd~hGq~PpKmIx0u2uuWNoUFrlswBeATgsBDTtIfqsMT8F3mYHI7CrXwjzeJ4tJU0bnnzeVTNRE8AtJde3RE1QgHagtUkHOu~rjwD-f1Pk5rplHclQAe37qHxnvcxXBxzxGbZFeWqOGKUzrYVAuhIRNawz9RdeH1-wJBoeHJM68lezx03mBZVud-lBhRNe5D5kWQHbjcAsGjiNQ1tBqxIPT9RYgKic0LSF-C~rNUAZVhkk5FnhoDFRThMUwp9uRYm1fcszjhEzXmTTJFyN~KzATC9YtOyvlfLcMKy~wzJrxA10uA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation of monomer number in extracted TRF arrays and monomer length. Plotted is number of arrays as a function of monomer length for arrays with ≤2 monomers (A) 3–4 monomers (B) and ≥5 monomers (C). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196653/dsv02102.tiff" path-from-xml="dsv02102.tiff" target="_blank">View large</a><a section="87196653" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02102.png?Expires=2147483647&Signature=iZgX15z2iZKyPOD19v4G1sTGyoAL63LjzPxMzI4qh~u8Z0bw-4X-KejqYHBCzu6JyPk2KZWMGfGjcvCViLT8iwQ133zKJRbl3XKhCFMywEQ6gKJBBVSARWBvzknwTgu43i6ZSRzwZUg7IBMc9DO3i5eTwI~cR61fEnWVPvlOfA9e0RhBCwOqUsdg4YvRbgc6eAPPTkqVVjES-c-3LL997W8YGXh03BV8HRtqdTmmLLvdgzH6neuETQMWCV5GSrWFTfV8GAcGv5y1cwke9eRnojjL7XdxI5m2L~RrZZ3lVrBBcCrteH4u6dgB9I~WAXCs-NiHWQrlMsMZmz7IWYuJIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196653&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation of monomer number in extracted TRF arrays and monomer length. Plotted is number of arrays as a function of monomer length for arrays with ≤2 monomers (A) 3–4 monomers (B) and ≥5 monomers (C). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV021F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02102.png?Expires=2147483647&Signature=WVc7EWg4dEDwVToAj3jUm4r8UmXGbFoF1YTp4JLmd~hGq~PpKmIx0u2uuWNoUFrlswBeATgsBDTtIfqsMT8F3mYHI7CrXwjzeJ4tJU0bnnzeVTNRE8AtJde3RE1QgHagtUkHOu~rjwD-f1Pk5rplHclQAe37qHxnvcxXBxzxGbZFeWqOGKUzrYVAuhIRNawz9RdeH1-wJBoeHJM68lezx03mBZVud-lBhRNe5D5kWQHbjcAsGjiNQ1tBqxIPT9RYgKic0LSF-C~rNUAZVhkk5FnhoDFRThMUwp9uRYm1fcszjhEzXmTTJFyN~KzATC9YtOyvlfLcMKy~wzJrxA10uA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation of monomer number in extracted TRF arrays and monomer length. Plotted is number of arrays as a function of monomer length for arrays with ≤2 monomers (A) 3–4 monomers (B) and ≥5 monomers (C). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196653/dsv02102.tiff" path-from-xml="dsv02102.tiff" target="_blank">View large</a><a section="87196653" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02102.png?Expires=2147483647&Signature=iZgX15z2iZKyPOD19v4G1sTGyoAL63LjzPxMzI4qh~u8Z0bw-4X-KejqYHBCzu6JyPk2KZWMGfGjcvCViLT8iwQ133zKJRbl3XKhCFMywEQ6gKJBBVSARWBvzknwTgu43i6ZSRzwZUg7IBMc9DO3i5eTwI~cR61fEnWVPvlOfA9e0RhBCwOqUsdg4YvRbgc6eAPPTkqVVjES-c-3LL997W8YGXh03BV8HRtqdTmmLLvdgzH6neuETQMWCV5GSrWFTfV8GAcGv5y1cwke9eRnojjL7XdxI5m2L~RrZZ3lVrBBcCrteH4u6dgB9I~WAXCs-NiHWQrlMsMZmz7IWYuJIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196653&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation of monomer number in extracted TRF arrays and monomer length. Plotted is number of arrays as a function of monomer length for arrays with ≤2 monomers (A) 3–4 monomers (B) and ≥5 monomers (C). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87196656 id="87196656" class="section-title" data-legacy-id=s3b>3.2 Revision of centromeric TCAST satDNA</h3>
<p>Centromeric TCAST satDNA was identified experimentally.<sup><a reveal-id="DSV021C21" data-open="DSV021C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV021C26" data-open="DSV021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Bioinformatic identification of repetitive DNA by RepeatScout revealed only 0.3% of TCAST satDNA in the assembled genome,<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> while the majority of sequenced TCAST satDNA was retained in unassembled reads. We extracted TCAST monomers from unassembled reads to define all sequence variants (data not shown). The alignment of all monomers revealed five subfamilies of TCAST satDNA with mutual sequence similarity of 70% and monomer length variation from 332 to 384 bp (Fig. <a reveal-id="DSV021F3" data-open="DSV021F3" class="link link-reveal link-table xref-fig">3</a>). Two subfamilies (subf1 and subf3) were previously described,<sup><a reveal-id="DSV021C21" data-open="DSV021C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV021C26" data-open="DSV021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> while others are identified in this work for the first time. Dramatic monomer length variation is mainly due to insertion/deletion events in one region of extracted monomers. In order to explore the distribution of this satDNA in the assembled genome, we mapped TCAST satDNA on assembled chromosomes by BLAST search using consensus sequences of all subfamilies as queries (red spots in Fig. <a reveal-id="DSV021F1" data-open="DSV021F1" class="link link-reveal link-table xref-fig">1</a>). We determined 130 short arrays of mainly 1–2 monomers, distributed randomly along chromosomes. </p>                        <a id="87196658" scrollto-destination="87196658"></a>
<div id="DSV021F3" data-id="DSV021F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02103.png?Expires=2147483647&Signature=unfQRd8KwTtj6XQtQTpL23ZKe2XVyLJeBOwTGKTANGesdIbjN1S-6tbVmbrJsviM42c7PS14i5RlGLxdEtsoffY1GJNjsUyVapv84pvjr54F0vvEYwZS915aNkmXLXq3cQLQ59~IxTUZLLf4wS2Lj-Y1dFhSyHL8ZbgICopalEByeDQqsnd6z9XaxbwnAR82gFI65rU4ZpsUqXmiYk2AQdIOi7bXMzwPigNimd~ygIP8rBSRdSI3orbYq-c1TRjP3oMuxiQiG-Xwvrwyy63Ew7aY5nrjsc8mdj8OAgeQ~1DRyWYfJIGMvgpcLNBbIniJExPo29Cz6NaBPwVI~gVZ-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of consensus monomer of five TCAST satDNA subfamilies from unassembled reads of T. castaneum genome. Consensus sequences of subfamilies are derived according to the majority principle in alignment of variants from unassembled reads. Monomer length variation in the central region is highlighted in grey." path-from-xml="dsv02103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196658/dsv02103.tiff" path-from-xml="dsv02103.tiff" target="_blank">View large</a><a section="87196658" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02103.png?Expires=2147483647&Signature=YpqzFnDELSPbNSUrXuDdwkgFjBnH7SFDSJfdXWaxsQgDqqzhGEulCVxQFB3MZCFtyIpgqmp2fuMca7WByspijhY6aUF6iV9LBO6W9Mc0qmnZRpV7X4tuxdRScO5jHPe0IkNwSSO-8N5wPxuzo7r3BK6CFEOQNCdjln~wTBlqiHnxXCTMqvbh8w0-SZrPmlO1Mfy3LQmR6TVszqfGaXeCaUCXpefDdnq6MHEjSXMUazV2wqPEKSxnInxYwgLjXDSgbSucsFKv4GHyqD~BHNaLisEE3M-xQnSKYqcdAHFB0PPyxJb2YOdH6zv~v9qI13affFUwhUpsAieKCenap0X04Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196658&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of consensus monomer of five TCAST satDNA subfamilies from unassembled reads of <em>T. castaneum</em> genome. Consensus sequences of subfamilies are derived according to the majority principle in alignment of variants from unassembled reads. Monomer length variation in the central region is highlighted in grey.</p></div></div><div content-id="DSV021F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02103.png?Expires=2147483647&Signature=unfQRd8KwTtj6XQtQTpL23ZKe2XVyLJeBOwTGKTANGesdIbjN1S-6tbVmbrJsviM42c7PS14i5RlGLxdEtsoffY1GJNjsUyVapv84pvjr54F0vvEYwZS915aNkmXLXq3cQLQ59~IxTUZLLf4wS2Lj-Y1dFhSyHL8ZbgICopalEByeDQqsnd6z9XaxbwnAR82gFI65rU4ZpsUqXmiYk2AQdIOi7bXMzwPigNimd~ygIP8rBSRdSI3orbYq-c1TRjP3oMuxiQiG-Xwvrwyy63Ew7aY5nrjsc8mdj8OAgeQ~1DRyWYfJIGMvgpcLNBbIniJExPo29Cz6NaBPwVI~gVZ-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of consensus monomer of five TCAST satDNA subfamilies from unassembled reads of T. castaneum genome. Consensus sequences of subfamilies are derived according to the majority principle in alignment of variants from unassembled reads. Monomer length variation in the central region is highlighted in grey." path-from-xml="dsv02103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196658/dsv02103.tiff" path-from-xml="dsv02103.tiff" target="_blank">View large</a><a section="87196658" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02103.png?Expires=2147483647&Signature=YpqzFnDELSPbNSUrXuDdwkgFjBnH7SFDSJfdXWaxsQgDqqzhGEulCVxQFB3MZCFtyIpgqmp2fuMca7WByspijhY6aUF6iV9LBO6W9Mc0qmnZRpV7X4tuxdRScO5jHPe0IkNwSSO-8N5wPxuzo7r3BK6CFEOQNCdjln~wTBlqiHnxXCTMqvbh8w0-SZrPmlO1Mfy3LQmR6TVszqfGaXeCaUCXpefDdnq6MHEjSXMUazV2wqPEKSxnInxYwgLjXDSgbSucsFKv4GHyqD~BHNaLisEE3M-xQnSKYqcdAHFB0PPyxJb2YOdH6zv~v9qI13affFUwhUpsAieKCenap0X04Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196658&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of consensus monomer of five TCAST satDNA subfamilies from unassembled reads of <em>T. castaneum</em> genome. Consensus sequences of subfamilies are derived according to the majority principle in alignment of variants from unassembled reads. Monomer length variation in the central region is highlighted in grey.</p></div></div><p>Due to small chromosome sizes (only 1–3 μm), previous FISH analysis of TCAST satDNA had very poor resolution.<sup><a reveal-id="DSV021C21" data-open="DSV021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> To evaluate the centromeric localization of all TCAST subfamilies as well as to reexamine the karyotype with respect to the centromeric region, we performed individual FISH analyses with monomers specific for each of five subfamilies. Given that all FISH analyses show identical localization of TCAST subfamilies, we present only one of them in Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>a. <em>Tribolium castaneum</em> has a 2<em>n</em> = 20 complement of chromosomes and a meioformula 9+XYp.<sup><a reveal-id="DSV021C29" data-open="DSV021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Detailed cytogenetic analyses confirmed localization of TCAST subfamilies to centromeric regions of all chromosomes without any significant signals outside of centromeric regions. It must be noted that short TCAST arrays dispersed along the assembled genome are evidently below the detection level by FISH. FISH analyses also enabled identification of Yp, two metacentric chromosomes CH2 and CH3, and the largest telocentric chromosome CH4 (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>a). The remaining chromosomes are mostly small telocentrics. </p>                        <a id="87196663" scrollto-destination="87196663"></a>
<div id="DSV021F4" data-id="DSV021F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02104.png?Expires=2147483647&Signature=I30kNHIxn10aL21RbCYKCA2eDYxEGWRsH4t9X8TWJ4Pnoj5dNM-TpVaXl8IDaRfIbSR5tZnuoXmkSiLsKlY5r1PdMe-X56RXOk0sp-obLMZYdWiEw-YkT~8Jr7VbVZlTW~eMXz-VX2MLqvWOemBc2pfQNnDoU4-~bqQIr5M7y66AOjw68np6-KqSOjW5q6Gdpt9LzYc98pDK3cbo2gUZei9YzSVVDbNNO-8IqZibRkC6RR~DrEPRWx7qUpgsszRWdRrT5QvwmnF9r3W0itQuzgbZlMqYNwul8vxsH6ec1npCQHJoDmj27R8onU17tXGiUGHIVcI2zRjun4~DD6mKlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Fluorescence in situ hybridization of centromeric TCAST satDNA and TR families determined in this work by TRF analysis. Chromosomes are counter-stained with DAPI. FISH showing centromeric TCAST satDNA (TCAST subf3 as a probe; red signals) on T. castaneum chromosomes in meiotic pro metaphase (A). Arrows point to chromosomes CH2, CH3, CH4 and Yp. Chromosomes were named according to the karyotype analysis provided by Stuart and Mocelin.29 Two-coloured FISH performed on chromosomes in mitotic pro metaphase show localization of new TR families (green): Cast1 (B), Cast2 (C), Cast3 (D), Cast4 (E), Cast5 (F), Cast6 (G), Cast7 (H), Cast8 (I) and Cast9 (J) with respect to centromeric regions marked with TCAST satDNA (red). The bar represents 1 μm. Aside to chromosome spreads are shown Southern blot analyses of genomic DNA digested with REs and hybridized with Cast1 (B), Cast2 (C), Cast4 (E), Cast 5 (F) and Cast6 (G). Only TR families with &gt;0.5% of genomic DNA are presented." path-from-xml="dsv02104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196663/dsv02104.tiff" path-from-xml="dsv02104.tiff" target="_blank">View large</a><a section="87196663" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02104.png?Expires=2147483647&Signature=UL7Y269OHfgosXNm9nxIDixghD12CB9UQmTfggiXNS95MqMTHt2XQ0XmqgXsMk7hcn5~qZKYi1EfYMnFoK8~NkfzC9vnciGpPQD0PXj6UXlSxJmRuCUMI695FlYgz3R4Q-emLs1s3auxv4XVukLALPvOahHV~P80fC8eeQGVN6QmEOTvjTFMlyygPsNoEhYPwybK4WsngUKWWd7IPtEiT34LByPLHnOJl01dMAwYd6-cXQ8DR0B0lgBZP8B8fCvp7UP~MatfOHwTAvsPKThV8jux-f5vcRnOfVfNAzrZyThK0z9KyjV-RbdL4x4VJ1ei~~IblfSRBnZee2NUtfzBbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196663&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fluorescence <em>in situ</em> hybridization of centromeric TCAST satDNA and TR families determined in this work by TRF analysis. Chromosomes are counter-stained with DAPI. FISH showing centromeric TCAST satDNA (TCAST subf3 as a probe; red signals) on <em>T. castaneum</em> chromosomes in meiotic pro metaphase (A). Arrows point to chromosomes CH2, CH3, CH4 and Yp. Chromosomes were named according to the karyotype analysis provided by Stuart and Mocelin.<sup><a reveal-id="DSV021C29" data-open="DSV021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Two-coloured FISH performed on chromosomes in mitotic pro metaphase show localization of new TR families (green): Cast1 (B), Cast2 (C), Cast3 (D), Cast4 (E), Cast5 (F), Cast6 (G), Cast7 (H), Cast8 (I) and Cast9 (J) with respect to centromeric regions marked with TCAST satDNA (red). The bar represents 1 μm. Aside to chromosome spreads are shown Southern blot analyses of genomic DNA digested with REs and hybridized with Cast1 (B), Cast2 (C), Cast4 (E), Cast 5 (F) and Cast6 (G). Only TR families with &gt;0.5% of genomic DNA are presented.</p></div></div><div content-id="DSV021F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02104.png?Expires=2147483647&Signature=I30kNHIxn10aL21RbCYKCA2eDYxEGWRsH4t9X8TWJ4Pnoj5dNM-TpVaXl8IDaRfIbSR5tZnuoXmkSiLsKlY5r1PdMe-X56RXOk0sp-obLMZYdWiEw-YkT~8Jr7VbVZlTW~eMXz-VX2MLqvWOemBc2pfQNnDoU4-~bqQIr5M7y66AOjw68np6-KqSOjW5q6Gdpt9LzYc98pDK3cbo2gUZei9YzSVVDbNNO-8IqZibRkC6RR~DrEPRWx7qUpgsszRWdRrT5QvwmnF9r3W0itQuzgbZlMqYNwul8vxsH6ec1npCQHJoDmj27R8onU17tXGiUGHIVcI2zRjun4~DD6mKlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Fluorescence in situ hybridization of centromeric TCAST satDNA and TR families determined in this work by TRF analysis. Chromosomes are counter-stained with DAPI. FISH showing centromeric TCAST satDNA (TCAST subf3 as a probe; red signals) on T. castaneum chromosomes in meiotic pro metaphase (A). Arrows point to chromosomes CH2, CH3, CH4 and Yp. Chromosomes were named according to the karyotype analysis provided by Stuart and Mocelin.29 Two-coloured FISH performed on chromosomes in mitotic pro metaphase show localization of new TR families (green): Cast1 (B), Cast2 (C), Cast3 (D), Cast4 (E), Cast5 (F), Cast6 (G), Cast7 (H), Cast8 (I) and Cast9 (J) with respect to centromeric regions marked with TCAST satDNA (red). The bar represents 1 μm. Aside to chromosome spreads are shown Southern blot analyses of genomic DNA digested with REs and hybridized with Cast1 (B), Cast2 (C), Cast4 (E), Cast 5 (F) and Cast6 (G). Only TR families with &gt;0.5% of genomic DNA are presented." path-from-xml="dsv02104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196663/dsv02104.tiff" path-from-xml="dsv02104.tiff" target="_blank">View large</a><a section="87196663" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02104.png?Expires=2147483647&Signature=UL7Y269OHfgosXNm9nxIDixghD12CB9UQmTfggiXNS95MqMTHt2XQ0XmqgXsMk7hcn5~qZKYi1EfYMnFoK8~NkfzC9vnciGpPQD0PXj6UXlSxJmRuCUMI695FlYgz3R4Q-emLs1s3auxv4XVukLALPvOahHV~P80fC8eeQGVN6QmEOTvjTFMlyygPsNoEhYPwybK4WsngUKWWd7IPtEiT34LByPLHnOJl01dMAwYd6-cXQ8DR0B0lgBZP8B8fCvp7UP~MatfOHwTAvsPKThV8jux-f5vcRnOfVfNAzrZyThK0z9KyjV-RbdL4x4VJ1ei~~IblfSRBnZee2NUtfzBbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196663&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fluorescence <em>in situ</em> hybridization of centromeric TCAST satDNA and TR families determined in this work by TRF analysis. Chromosomes are counter-stained with DAPI. FISH showing centromeric TCAST satDNA (TCAST subf3 as a probe; red signals) on <em>T. castaneum</em> chromosomes in meiotic pro metaphase (A). Arrows point to chromosomes CH2, CH3, CH4 and Yp. Chromosomes were named according to the karyotype analysis provided by Stuart and Mocelin.<sup><a reveal-id="DSV021C29" data-open="DSV021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Two-coloured FISH performed on chromosomes in mitotic pro metaphase show localization of new TR families (green): Cast1 (B), Cast2 (C), Cast3 (D), Cast4 (E), Cast5 (F), Cast6 (G), Cast7 (H), Cast8 (I) and Cast9 (J) with respect to centromeric regions marked with TCAST satDNA (red). The bar represents 1 μm. Aside to chromosome spreads are shown Southern blot analyses of genomic DNA digested with REs and hybridized with Cast1 (B), Cast2 (C), Cast4 (E), Cast 5 (F) and Cast6 (G). Only TR families with &gt;0.5% of genomic DNA are presented.</p></div></div>                            <h3 scrollto-destination=87196665 id="87196665" class="section-title" data-legacy-id=s3c>3.3 Characterization of the most abundant TR families detected in the assembled genome</h3>
<p>In order to explore the most abundant TR families in the assembled <em>T. castaneum</em> genome we further focused on arrays with ≥5 monomers obtained in the TRF analysis output. This cutoff level was also selected to prevent blurring the results of phylogenetic analyses on selected most abundant TR families, that would be caused by a large number of branches in phylogenetic trees derived from monomers in shorter arrays. This analysis recovered 763 arrays with ≥5 TRs, cumulatively making up 1.63 Mb that constitute 1.04% of the assembled genome. Extracted arrays were clustered using tools implemented in the TRDB.<sup><a reveal-id="DSV021C30" data-open="DSV021C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Sequence identity &gt;70% was selected as the clustering requirement in order to keep the identity level similar to that detected within the majority of satDNA families.<sup><a reveal-id="DSV021C39" data-open="DSV021C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Using this parameter, 371 arrays were further classified into 56 clusters; 23 containing 3–42 arrays, while others were those with only two related arrays. In order to explore in detail the most prominent TR families, we selected nine largest clusters which were represented with at least two arrays on at least two chromosomes. These criteria were set to enable comparative studies of monomers from different arrays located on a single chromosome as well as comparisons of arrays among chromosomes.</p><p>Nine selected clusters comprise about two-thirds of arrays with ≥5 monomers (Table <a reveal-id="DSV021TB1" data-open="DSV021TB1" class="link link-reveal link-table xref-fig">1</a>, clustering results are available on the TRDB website upon request). Monomers belonging to each cluster were further exported and named with letters and numbers related to the chromosome number, genomic position and position in the particular array. Each cluster thus represents the most abundant TR families hereafter indicated as Cast1–Cast9. Monomers were aligned with ClustalW (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) and structural characteristics of the nine TR families are summarized in Table <a reveal-id="DSV021TB1" data-open="DSV021TB1" class="link link-reveal link-table xref-fig">1</a>. In agreement with the estimated correlation between monomer size and array length, 5 of 9 families are based on ∼170 while two have ∼340 bp long monomers. The remaining two families have monomers of ∼110 and 210 bp, respectively. The number of tandemly repeated monomers in the obtained arrays is up to 54. The nucleotide sequences of all families show high AT content (≥60%) and 9–28% nucleotide diversity within the family. The abundance of newly detected TRs in the assembled genome ranges from 0.006% up to 0.075% (9–117 kb). We also observed that periodicity of AT tracts is a prominent feature of all analysed TRs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). </p>                        <a id="87196668" scrollto-destination="87196668"></a>
<div content-id="DSV021TB1" class="table-modal"><div class="table-wrap"><div id="DSV021TB1" class="table-wrap-title" data-id="DSV021TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Structural features of euchromatic TR families</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TR families </th><th>Number of arrays </th><th colspan="10">Number of arrays per chromosome<hr /> </th><th rowspan="2">Max. number of monomers per array </th><th rowspan="2">Avarage number of monomers per array </th><th rowspan="2">Nucleotide diversity (Pi) of monomers in cluster ± standard deviation </th><th rowspan="2">The monomer length (bp) </th><th rowspan="2">AT content </th><th rowspan="2">Number of monomers </th><th rowspan="2">Total repeat family length (kb) </th><th rowspan="2">% of assembled genome </th><th rowspan="2">% of genome estimated by dot blot </th></tr><tr><th> </th><th> </th><th>CH1 </th><th>CH2 </th><th>CH3 </th><th>CH4 </th><th>CH5 </th><th>CH6 </th><th>CH7 </th><th>CH8 </th><th>CH9 </th><th>CH10 </th></tr></thead><tbody><tr><td>Cast1 </td><td>46 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>1 </td><td>8 </td><td>3 </td><td>4 </td><td>10 </td><td>5 </td><td>31 </td><td>10.63 </td><td>0.20143 ± 0.00320 </td><td>166–173 </td><td>66.2 </td><td>489 </td><td>83 </td><td>0.053 </td><td>&gt;1 </td></tr><tr><td>Cast2 </td><td>42 </td><td>6 </td><td>1 </td><td>4 </td><td>5 </td><td>4 </td><td>4 </td><td>1 </td><td>10 </td><td>1 </td><td>3 </td><td>33 </td><td>12.19 </td><td>0.12138 ± 0.00303 </td><td>166–172 </td><td>72.3 </td><td>512 </td><td>87 </td><td>0.056 </td><td>0.5 </td></tr><tr><td>Cast3 </td><td>35 </td><td>2 </td><td>1 </td><td>3 </td><td>6 </td><td>3 </td><td>7 </td><td>2 </td><td>5 </td><td>2 </td><td>4 </td><td>17 </td><td>6.57 </td><td>0.11734 ± 0.006 </td><td>205–219 </td><td>74.8 </td><td>230 </td><td>48 </td><td>0.031 </td><td>0.2 </td></tr><tr><td>Cast4 </td><td>33 </td><td>0 </td><td>1 </td><td>5 </td><td>0 </td><td>2 </td><td>3 </td><td>8 </td><td>8 </td><td>4 </td><td>0 </td><td>31 </td><td>12.91 </td><td>0.21765 ± 0.00296 </td><td>168–176 </td><td>69.7 </td><td>426 </td><td>73 </td><td>0.047 </td><td>0.5 </td></tr><tr><td>Cast5 </td><td>30 </td><td>0 </td><td>1 </td><td>6 </td><td>3 </td><td>3 </td><td>5 </td><td>1 </td><td>7 </td><td>1 </td><td>3 </td><td>28 </td><td>11.6 </td><td>0.09620 ± 0.00182 </td><td>270–338 </td><td>73.1 </td><td>384 </td><td>117 </td><td>0.075 </td><td>&gt;1 </td></tr><tr><td>Cast6 </td><td>6 </td><td>0 </td><td>0 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>54 </td><td>21.17 </td><td>0.16874 ± 0.00385 </td><td>179–181 </td><td>66 </td><td>157 </td><td>28 </td><td>0.018 </td><td>0.5 </td></tr><tr><td>Cast7 </td><td>10 </td><td>0 </td><td>0 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>12 </td><td>8.3 </td><td>0.28310 ± 0.00129 </td><td>109–114 </td><td>68.9 </td><td>83 </td><td>9 </td><td>0.006 </td><td>0.2 </td></tr><tr><td>Cast8 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>4 </td><td>2 </td><td>24 </td><td>12 </td><td>0.16321 ± 0.00733 </td><td>161–167 </td><td>67.3 </td><td>120 </td><td>20 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Cast9 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>0 </td><td>1 </td><td>0 </td><td>3 </td><td>10 </td><td>6.4 </td><td>0.25299 ± 0.00481 </td><td>311–346 </td><td>72.6 </td><td>64 </td><td>21 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Total </td><td>222 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2501 </td><td>486 </td><td>0.312 </td><td>&gt;4.3 </td></tr></tbody></table></div><div content-id="DSV021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>TR families </th><th>Number of arrays </th><th colspan="10">Number of arrays per chromosome<hr /> </th><th rowspan="2">Max. number of monomers per array </th><th rowspan="2">Avarage number of monomers per array </th><th rowspan="2">Nucleotide diversity (Pi) of monomers in cluster ± standard deviation </th><th rowspan="2">The monomer length (bp) </th><th rowspan="2">AT content </th><th rowspan="2">Number of monomers </th><th rowspan="2">Total repeat family length (kb) </th><th rowspan="2">% of assembled genome </th><th rowspan="2">% of genome estimated by dot blot </th></tr><tr><th> </th><th> </th><th>CH1 </th><th>CH2 </th><th>CH3 </th><th>CH4 </th><th>CH5 </th><th>CH6 </th><th>CH7 </th><th>CH8 </th><th>CH9 </th><th>CH10 </th></tr></thead><tbody><tr><td>Cast1 </td><td>46 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>1 </td><td>8 </td><td>3 </td><td>4 </td><td>10 </td><td>5 </td><td>31 </td><td>10.63 </td><td>0.20143 ± 0.00320 </td><td>166–173 </td><td>66.2 </td><td>489 </td><td>83 </td><td>0.053 </td><td>&gt;1 </td></tr><tr><td>Cast2 </td><td>42 </td><td>6 </td><td>1 </td><td>4 </td><td>5 </td><td>4 </td><td>4 </td><td>1 </td><td>10 </td><td>1 </td><td>3 </td><td>33 </td><td>12.19 </td><td>0.12138 ± 0.00303 </td><td>166–172 </td><td>72.3 </td><td>512 </td><td>87 </td><td>0.056 </td><td>0.5 </td></tr><tr><td>Cast3 </td><td>35 </td><td>2 </td><td>1 </td><td>3 </td><td>6 </td><td>3 </td><td>7 </td><td>2 </td><td>5 </td><td>2 </td><td>4 </td><td>17 </td><td>6.57 </td><td>0.11734 ± 0.006 </td><td>205–219 </td><td>74.8 </td><td>230 </td><td>48 </td><td>0.031 </td><td>0.2 </td></tr><tr><td>Cast4 </td><td>33 </td><td>0 </td><td>1 </td><td>5 </td><td>0 </td><td>2 </td><td>3 </td><td>8 </td><td>8 </td><td>4 </td><td>0 </td><td>31 </td><td>12.91 </td><td>0.21765 ± 0.00296 </td><td>168–176 </td><td>69.7 </td><td>426 </td><td>73 </td><td>0.047 </td><td>0.5 </td></tr><tr><td>Cast5 </td><td>30 </td><td>0 </td><td>1 </td><td>6 </td><td>3 </td><td>3 </td><td>5 </td><td>1 </td><td>7 </td><td>1 </td><td>3 </td><td>28 </td><td>11.6 </td><td>0.09620 ± 0.00182 </td><td>270–338 </td><td>73.1 </td><td>384 </td><td>117 </td><td>0.075 </td><td>&gt;1 </td></tr><tr><td>Cast6 </td><td>6 </td><td>0 </td><td>0 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>54 </td><td>21.17 </td><td>0.16874 ± 0.00385 </td><td>179–181 </td><td>66 </td><td>157 </td><td>28 </td><td>0.018 </td><td>0.5 </td></tr><tr><td>Cast7 </td><td>10 </td><td>0 </td><td>0 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>12 </td><td>8.3 </td><td>0.28310 ± 0.00129 </td><td>109–114 </td><td>68.9 </td><td>83 </td><td>9 </td><td>0.006 </td><td>0.2 </td></tr><tr><td>Cast8 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>4 </td><td>2 </td><td>24 </td><td>12 </td><td>0.16321 ± 0.00733 </td><td>161–167 </td><td>67.3 </td><td>120 </td><td>20 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Cast9 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>0 </td><td>1 </td><td>0 </td><td>3 </td><td>10 </td><td>6.4 </td><td>0.25299 ± 0.00481 </td><td>311–346 </td><td>72.6 </td><td>64 </td><td>21 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Total </td><td>222 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2501 </td><td>486 </td><td>0.312 </td><td>&gt;4.3 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196668" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV021TB1" class="table-wrap-title" data-id="DSV021TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Structural features of euchromatic TR families</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TR families </th><th>Number of arrays </th><th colspan="10">Number of arrays per chromosome<hr /> </th><th rowspan="2">Max. number of monomers per array </th><th rowspan="2">Avarage number of monomers per array </th><th rowspan="2">Nucleotide diversity (Pi) of monomers in cluster ± standard deviation </th><th rowspan="2">The monomer length (bp) </th><th rowspan="2">AT content </th><th rowspan="2">Number of monomers </th><th rowspan="2">Total repeat family length (kb) </th><th rowspan="2">% of assembled genome </th><th rowspan="2">% of genome estimated by dot blot </th></tr><tr><th> </th><th> </th><th>CH1 </th><th>CH2 </th><th>CH3 </th><th>CH4 </th><th>CH5 </th><th>CH6 </th><th>CH7 </th><th>CH8 </th><th>CH9 </th><th>CH10 </th></tr></thead><tbody><tr><td>Cast1 </td><td>46 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>1 </td><td>8 </td><td>3 </td><td>4 </td><td>10 </td><td>5 </td><td>31 </td><td>10.63 </td><td>0.20143 ± 0.00320 </td><td>166–173 </td><td>66.2 </td><td>489 </td><td>83 </td><td>0.053 </td><td>&gt;1 </td></tr><tr><td>Cast2 </td><td>42 </td><td>6 </td><td>1 </td><td>4 </td><td>5 </td><td>4 </td><td>4 </td><td>1 </td><td>10 </td><td>1 </td><td>3 </td><td>33 </td><td>12.19 </td><td>0.12138 ± 0.00303 </td><td>166–172 </td><td>72.3 </td><td>512 </td><td>87 </td><td>0.056 </td><td>0.5 </td></tr><tr><td>Cast3 </td><td>35 </td><td>2 </td><td>1 </td><td>3 </td><td>6 </td><td>3 </td><td>7 </td><td>2 </td><td>5 </td><td>2 </td><td>4 </td><td>17 </td><td>6.57 </td><td>0.11734 ± 0.006 </td><td>205–219 </td><td>74.8 </td><td>230 </td><td>48 </td><td>0.031 </td><td>0.2 </td></tr><tr><td>Cast4 </td><td>33 </td><td>0 </td><td>1 </td><td>5 </td><td>0 </td><td>2 </td><td>3 </td><td>8 </td><td>8 </td><td>4 </td><td>0 </td><td>31 </td><td>12.91 </td><td>0.21765 ± 0.00296 </td><td>168–176 </td><td>69.7 </td><td>426 </td><td>73 </td><td>0.047 </td><td>0.5 </td></tr><tr><td>Cast5 </td><td>30 </td><td>0 </td><td>1 </td><td>6 </td><td>3 </td><td>3 </td><td>5 </td><td>1 </td><td>7 </td><td>1 </td><td>3 </td><td>28 </td><td>11.6 </td><td>0.09620 ± 0.00182 </td><td>270–338 </td><td>73.1 </td><td>384 </td><td>117 </td><td>0.075 </td><td>&gt;1 </td></tr><tr><td>Cast6 </td><td>6 </td><td>0 </td><td>0 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>54 </td><td>21.17 </td><td>0.16874 ± 0.00385 </td><td>179–181 </td><td>66 </td><td>157 </td><td>28 </td><td>0.018 </td><td>0.5 </td></tr><tr><td>Cast7 </td><td>10 </td><td>0 </td><td>0 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>12 </td><td>8.3 </td><td>0.28310 ± 0.00129 </td><td>109–114 </td><td>68.9 </td><td>83 </td><td>9 </td><td>0.006 </td><td>0.2 </td></tr><tr><td>Cast8 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>4 </td><td>2 </td><td>24 </td><td>12 </td><td>0.16321 ± 0.00733 </td><td>161–167 </td><td>67.3 </td><td>120 </td><td>20 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Cast9 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>0 </td><td>1 </td><td>0 </td><td>3 </td><td>10 </td><td>6.4 </td><td>0.25299 ± 0.00481 </td><td>311–346 </td><td>72.6 </td><td>64 </td><td>21 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Total </td><td>222 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2501 </td><td>486 </td><td>0.312 </td><td>&gt;4.3 </td></tr></tbody></table></div><div content-id="DSV021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>TR families </th><th>Number of arrays </th><th colspan="10">Number of arrays per chromosome<hr /> </th><th rowspan="2">Max. number of monomers per array </th><th rowspan="2">Avarage number of monomers per array </th><th rowspan="2">Nucleotide diversity (Pi) of monomers in cluster ± standard deviation </th><th rowspan="2">The monomer length (bp) </th><th rowspan="2">AT content </th><th rowspan="2">Number of monomers </th><th rowspan="2">Total repeat family length (kb) </th><th rowspan="2">% of assembled genome </th><th rowspan="2">% of genome estimated by dot blot </th></tr><tr><th> </th><th> </th><th>CH1 </th><th>CH2 </th><th>CH3 </th><th>CH4 </th><th>CH5 </th><th>CH6 </th><th>CH7 </th><th>CH8 </th><th>CH9 </th><th>CH10 </th></tr></thead><tbody><tr><td>Cast1 </td><td>46 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>1 </td><td>8 </td><td>3 </td><td>4 </td><td>10 </td><td>5 </td><td>31 </td><td>10.63 </td><td>0.20143 ± 0.00320 </td><td>166–173 </td><td>66.2 </td><td>489 </td><td>83 </td><td>0.053 </td><td>&gt;1 </td></tr><tr><td>Cast2 </td><td>42 </td><td>6 </td><td>1 </td><td>4 </td><td>5 </td><td>4 </td><td>4 </td><td>1 </td><td>10 </td><td>1 </td><td>3 </td><td>33 </td><td>12.19 </td><td>0.12138 ± 0.00303 </td><td>166–172 </td><td>72.3 </td><td>512 </td><td>87 </td><td>0.056 </td><td>0.5 </td></tr><tr><td>Cast3 </td><td>35 </td><td>2 </td><td>1 </td><td>3 </td><td>6 </td><td>3 </td><td>7 </td><td>2 </td><td>5 </td><td>2 </td><td>4 </td><td>17 </td><td>6.57 </td><td>0.11734 ± 0.006 </td><td>205–219 </td><td>74.8 </td><td>230 </td><td>48 </td><td>0.031 </td><td>0.2 </td></tr><tr><td>Cast4 </td><td>33 </td><td>0 </td><td>1 </td><td>5 </td><td>0 </td><td>2 </td><td>3 </td><td>8 </td><td>8 </td><td>4 </td><td>0 </td><td>31 </td><td>12.91 </td><td>0.21765 ± 0.00296 </td><td>168–176 </td><td>69.7 </td><td>426 </td><td>73 </td><td>0.047 </td><td>0.5 </td></tr><tr><td>Cast5 </td><td>30 </td><td>0 </td><td>1 </td><td>6 </td><td>3 </td><td>3 </td><td>5 </td><td>1 </td><td>7 </td><td>1 </td><td>3 </td><td>28 </td><td>11.6 </td><td>0.09620 ± 0.00182 </td><td>270–338 </td><td>73.1 </td><td>384 </td><td>117 </td><td>0.075 </td><td>&gt;1 </td></tr><tr><td>Cast6 </td><td>6 </td><td>0 </td><td>0 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>54 </td><td>21.17 </td><td>0.16874 ± 0.00385 </td><td>179–181 </td><td>66 </td><td>157 </td><td>28 </td><td>0.018 </td><td>0.5 </td></tr><tr><td>Cast7 </td><td>10 </td><td>0 </td><td>0 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>12 </td><td>8.3 </td><td>0.28310 ± 0.00129 </td><td>109–114 </td><td>68.9 </td><td>83 </td><td>9 </td><td>0.006 </td><td>0.2 </td></tr><tr><td>Cast8 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>4 </td><td>2 </td><td>24 </td><td>12 </td><td>0.16321 ± 0.00733 </td><td>161–167 </td><td>67.3 </td><td>120 </td><td>20 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Cast9 </td><td>10 </td><td>0 </td><td>0 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>0 </td><td>1 </td><td>0 </td><td>3 </td><td>10 </td><td>6.4 </td><td>0.25299 ± 0.00481 </td><td>311–346 </td><td>72.6 </td><td>64 </td><td>21 </td><td>0.013 </td><td>0.2 </td></tr><tr><td>Total </td><td>222 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2501 </td><td>486 </td><td>0.312 </td><td>&gt;4.3 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196668" target="_blank">View Large</a></div></div><p>BLAST search of both GenBank and RepBase with the consensus monomer sequence of each TR family as a query did not reveal significant similarity with any other sequence. However, local BLAST search with new TR families on repetitive classes obtained previously by RepeatScout<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> recovered their significant homology with Cast4 and Cast5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). In particular, high homology is shown between the Cast4 family and 7 repetitive elements previously identified by RepeatScout which mostly represent dispersed dimers of Cast4 family as well as parts of monomers associated with different flanking regions. Similarly, three RepeatScout-defined elements show homology with Cast5. They are composed of Cast5 monomer and flanking regions (for details, see below).</p><p>In addition to bioinformatic predictions of abundance and genome distribution, the newly found TR families were also examined by FISH, Southern analyses and dot-blot experiments (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a> and Table <a reveal-id="DSV021TB1" data-open="DSV021TB1" class="link link-reveal link-table xref-fig">1</a>). Individual TR family members were amplified with specific primers derived from consensus monomer sequences obtained in this work (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Cloned and sequenced monomers or dimers were further used as a template for the probe in hybridization experiments. Taking into account that genome segments rich in TRs remain poorly assembled and underestimated in outputs of genome projects, dot-blot hybridization was performed for each <em>in silico</em> detected TR family. Cast1 and Cast5 are the most abundant, each comprising &gt;1% of the genome (Table <a reveal-id="DSV021TB1" data-open="DSV021TB1" class="link link-reveal link-table xref-fig">1</a>). Cast2, Cast4 and Cast6 are less represented, each making up 0.5% of the genome, while other TR families comprise ∼0.2% of the genome. In total, according to the results of dot-blot assay, estimated abundance of all analysed TR families is &gt;4% of the genome. The experimentally determined genome content of these TR families is ∼10 times higher than the representativeness in the assembled genome. For further support of abundance, we also analysed the presence of these TR families in <em>T. castaneum</em> unassembled reads, and these estimations confirmed the highest relative proportion of Cast1, Cast2, Cast5 and Cast6 (data not shown). It is important to observe that among the experimentally estimated contribution of TR families that make &gt;4% of the genome, the most abundant (&gt;2.7%) are those based on ∼170 bp monomer, while families with ∼340 bp monomer comprise &gt;1.2% of the genome.</p><p>To validate the TR profile of the sequence sets generated <em>in silico</em>, Southern blot hybridization analyses were carried out on the most prominent TR families (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>). Genomic DNA was digested using REs which cut once in the majority of monomer sequences as well as with REs having a recognition site only in some monomers. In addition to a strong signal of predicted monomer size, typical satellite ladder pattern was observed in all hybridization analyses.</p><p>In order to detect chromosomal localization of Cast1–Cast9 families relative to centromeric regions, two-colour FISH was done combining each detected TR family and the centromeric TCAST satDNA. FISH analyses on chromosomes in mitotic pro metaphase enabled detection of centromeric regions together with signals of newly detected TR families (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>). As already shown by dot-blot experiments, signals obtained after FISH with the most abundant Cast1 and Cast5 were significantly stronger relative to other Cast families. Interestingly, FISH results show localization of all nine TR families almost exclusively on non-centromeric/euchromatic chromosome parts. Only a few overlapping signals with TCAST could be observed in centromeric regions, particularly in the case of Cast5.</p>                            <h3 scrollto-destination=87196674 id="87196674" class="section-title" data-legacy-id=s3d>3.4 Study of TR family evolution in euchromatic regions</h3>
<p>In order to assess evolutionary trends of dominant TR families located outside of centromeric regions in <em>T. castaneum</em>, we examined phylogenetic relationships between their monomers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Two thousand five hundred monomers originating from the nine selected families (Cast1–Cast9) were included in the analyses. Annotation with respect to their actual position allows detailed identification of phylogenetic relationships among monomers within and between arrays and chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Simplified forms of ML trees with marked subgroups which show recent exchange events within a chromosome and between non-homologs are presented in Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>. In all ML trees, the monomer groups show a significant level of exchange events which occurred between non-homologous chromosomes. </p>                        <a id="87196677" scrollto-destination="87196677"></a>
<div id="DSV021F5" data-id="DSV021F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02105.png?Expires=2147483647&Signature=xFOVp-CMgPBTbS0oBigtjHOBDFfjX9g~yPXcPQSPwvDe7ksb0AfUVEax5GUePe4NWdn25MFB6L5NEIOtLqFYg~Jo4jR0x4aquyJenzSR1QFiByZZcHlK6uP2bmFsVTbaDLdFuwrGtS3ZKegfRIZnLwSlLNaojCiz3mHdQ3BPjLHSU02hp5E8Oe2jlBsMcwMEDsTA6b0RBWCMY-q9A7MK5cMnNnm0thH1n6CAkMLp19k7prSUF4VcY70RN3zoub8NBwS5h2xPi6Ov91kMLCh4-H4a1HCgD1jaTHLIDPtKODJzcLD9SfaAAMahtnG9-pdE7Zcbgt7~tHrCajk2ubmKpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships of monomers of Cast1–Cast6 families presented in ML trees (A–F). Terminal branches are coloured according to the chromosome of origin. Monomers originating from a single array and grouped in a single branch of the tree are not specifically marked. Dominant groups which display putative recent exchange events are indicated with arches. Black arches mark monomers that group together in the tree, although they originate from arrays located on non-homologous chromosomes. Grey arches represent monomers in chromosome-specific arrays distant one from the other (&gt;20 kb). Blank arches indicate monomers located in dominant chromosome-specific arrays positioned &lt;20 kb apart. Symbols indicate monomers originating from the same array but dispersed in the tree. The branch support values are indicated at major branch points. (G) FISH of Cast6 family on T. castaneum chromosomes. Arrow points to the CH3 chromosome." path-from-xml="dsv02105.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196677/dsv02105.tiff" path-from-xml="dsv02105.tiff" target="_blank">View large</a><a section="87196677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02105.png?Expires=2147483647&Signature=iLJ00XvLawmaeRrpc78d6dAL6Er9wVFm7X2q9VzzePeJNC3tusFNDcjGz1dOzf24p3gq63Pv6gE11B9LnvGHWAIdIkPt0RG-50nuqrF1MbFalDtCsHjSx~QbWxHNYChO4zztWbqgcs-n9i0uC~mHW-bgSu9K9njfHzBEg57bN5~DpP~gDV0KR91Dwwo9pH4OGGIJVZmxHwzVBuZXrVPswdH7XDrXO0xZnZBPhq9pbV92kmuxQEegeHZqZV-lq0R~Acn4QBwtfhJBCl5Z4tMTbE9zjyCa27PqBmOzRkkbQrv7yzyAwryLwhMKfJZkMxMzZCTMqyZIIfk9K6ATI2posw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196677&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02105.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships of monomers of Cast1–Cast6 families presented in ML trees (A–F). Terminal branches are coloured according to the chromosome of origin. Monomers originating from a single array and grouped in a single branch of the tree are not specifically marked. Dominant groups which display putative recent exchange events are indicated with arches. Black arches mark monomers that group together in the tree, although they originate from arrays located on non-homologous chromosomes. Grey arches represent monomers in chromosome-specific arrays distant one from the other (&gt;20 kb). Blank arches indicate monomers located in dominant chromosome-specific arrays positioned &lt;20 kb apart. Symbols indicate monomers originating from the same array but dispersed in the tree. The branch support values are indicated at major branch points. (G) FISH of Cast6 family on <em>T. castaneum</em> chromosomes. Arrow points to the CH3 chromosome.</p></div></div><div content-id="DSV021F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02105.png?Expires=2147483647&Signature=xFOVp-CMgPBTbS0oBigtjHOBDFfjX9g~yPXcPQSPwvDe7ksb0AfUVEax5GUePe4NWdn25MFB6L5NEIOtLqFYg~Jo4jR0x4aquyJenzSR1QFiByZZcHlK6uP2bmFsVTbaDLdFuwrGtS3ZKegfRIZnLwSlLNaojCiz3mHdQ3BPjLHSU02hp5E8Oe2jlBsMcwMEDsTA6b0RBWCMY-q9A7MK5cMnNnm0thH1n6CAkMLp19k7prSUF4VcY70RN3zoub8NBwS5h2xPi6Ov91kMLCh4-H4a1HCgD1jaTHLIDPtKODJzcLD9SfaAAMahtnG9-pdE7Zcbgt7~tHrCajk2ubmKpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships of monomers of Cast1–Cast6 families presented in ML trees (A–F). Terminal branches are coloured according to the chromosome of origin. Monomers originating from a single array and grouped in a single branch of the tree are not specifically marked. Dominant groups which display putative recent exchange events are indicated with arches. Black arches mark monomers that group together in the tree, although they originate from arrays located on non-homologous chromosomes. Grey arches represent monomers in chromosome-specific arrays distant one from the other (&gt;20 kb). Blank arches indicate monomers located in dominant chromosome-specific arrays positioned &lt;20 kb apart. Symbols indicate monomers originating from the same array but dispersed in the tree. The branch support values are indicated at major branch points. (G) FISH of Cast6 family on T. castaneum chromosomes. Arrow points to the CH3 chromosome." path-from-xml="dsv02105.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196677/dsv02105.tiff" path-from-xml="dsv02105.tiff" target="_blank">View large</a><a section="87196677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02105.png?Expires=2147483647&Signature=iLJ00XvLawmaeRrpc78d6dAL6Er9wVFm7X2q9VzzePeJNC3tusFNDcjGz1dOzf24p3gq63Pv6gE11B9LnvGHWAIdIkPt0RG-50nuqrF1MbFalDtCsHjSx~QbWxHNYChO4zztWbqgcs-n9i0uC~mHW-bgSu9K9njfHzBEg57bN5~DpP~gDV0KR91Dwwo9pH4OGGIJVZmxHwzVBuZXrVPswdH7XDrXO0xZnZBPhq9pbV92kmuxQEegeHZqZV-lq0R~Acn4QBwtfhJBCl5Z4tMTbE9zjyCa27PqBmOzRkkbQrv7yzyAwryLwhMKfJZkMxMzZCTMqyZIIfk9K6ATI2posw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196677&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02105.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships of monomers of Cast1–Cast6 families presented in ML trees (A–F). Terminal branches are coloured according to the chromosome of origin. Monomers originating from a single array and grouped in a single branch of the tree are not specifically marked. Dominant groups which display putative recent exchange events are indicated with arches. Black arches mark monomers that group together in the tree, although they originate from arrays located on non-homologous chromosomes. Grey arches represent monomers in chromosome-specific arrays distant one from the other (&gt;20 kb). Blank arches indicate monomers located in dominant chromosome-specific arrays positioned &lt;20 kb apart. Symbols indicate monomers originating from the same array but dispersed in the tree. The branch support values are indicated at major branch points. (G) FISH of Cast6 family on <em>T. castaneum</em> chromosomes. Arrow points to the CH3 chromosome.</p></div></div><p>The ML tree of Cast1 shows three groups of monomers originating from distant arrays (&gt;20 kb) on homologous chromosomes and a recent exchange between non-homologous chromosomes (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>A). However, according to our FISH results, it can be concluded that the Cast1 family makes very long arrays which are not represented in the assembled genome.</p><p>The Cast2 phylogenetic tree displays two distinct branches with monomers originating from arrays located on 6 or 7 non-homologous chromosomes (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>B). This dispersion pattern together with short branches grouped in subclusters indicates a relatively recent expansion of arrays between non-homologous chromosomes which occurred in two independent events. In addition, monomers of this family form one chromosome-specific cluster composed of monomers originating from three distantly located arrays (separated by at least 20 kb). In addition, the Cast2 tree reveals exchange events between the sex-chromosome (X, marked as CH1) and autosomes.</p><p>The Cast3 family is characterized by low monomer divergence (11%; Table <a reveal-id="DSV021TB1" data-open="DSV021TB1" class="link link-reveal link-table xref-fig">1</a>) and short arrays mainly comprised of 5–7 monomers (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>C). Cast3 monomers derived from the majority of short arrays show a scattered distribution in the ML tree (see symbols on the tree), while those from longer arrays (up to 19 monomers) group together.</p><p>The Cast4 family is characterized by significant monomer divergence (21%; Table <a reveal-id="DSV021TB1" data-open="DSV021TB1" class="link link-reveal link-table xref-fig">1</a>) observed even among monomers within arrays, as evident by long branches in the tree. Nevertheless, monomers from the same array tend to group together (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>D). Two modes of chromosome-specific clustering are distinctive: (i) arrays are located close to each other (&lt;20 kb) and probably homogenized together, and (ii) arrays are distant (&gt;20 kb) and show intra-chromosomal exchanges.</p><p>Regardless the lowest overall monomer sequence divergence (9%), Cast5 shows clear grouping of monomers originating from the same arrays (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>E). In addition, monomers from the same chromosome show a tendency to group together even when arrays are positioned at distant locations. The tree also suggests recent inter-chromosomal exchange in a fraction of monomers. Cast7, Cast8 and Cast9 trees are not shown here because they follow similar evolutionary trends as for the TR families described above.</p><p>Interestingly, phylogenetic analysis of Cast6 (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>F) revealed that its long arrays are predominantly located on the chromosome CH3. Monomers in arrays evidence significant intra-chromosomal exchange. An almost exclusive localization of the Cast6 family on CH3 was also confirmed by FISH with the Cast6 monomer as a probe on the meiotic pro metaphase plate (Fig. <a reveal-id="DSV021F5" data-open="DSV021F5" class="link link-reveal link-table xref-fig">5</a>G). The coherence between sequence data for Cast6 from the assembled genome and FISH experiments further confirms the authenticity of the genome assembly.</p><p>Although each family shows its own pattern in the phylogenetic tree, common features can be deduced. The presented tree topologies show predominant clustering of monomers derived from the same array. This is particularly true for monomers derived from long arrays while those from short arrays generally do not show any consistent grouping. A general observation deduced from all extracted TR families is that monomers in arrays positioned on the same chromosome do not group with a frequency significantly higher than monomers located on non-homologous chromosomes. In support of this, but with exception of Cast6, all of the TR families show grouping of monomers from arrays located on nearly all non-homologous chromosomes, thus suggesting extensive inter-chromosomal exchanges.</p>                            <h3 scrollto-destination=87196689 id="87196689" class="section-title" data-legacy-id=s3e>3.5 Possible mechanisms of propagation</h3>
<p>In order to investigate putative propagation mechanisms for the analysed TR families we first wanted to check if they are just passively carried by expansion of other DNA segments in the genome. If this is so, it would be expected that array flanking regions or at least some of them are mutually homologous. To address this question, we generated 4 kb long left and right array flanking regions from the TRF output and compared them separately for each family. Our results show that among the nine TR families, only Cast2 and Cast5 display homologies in their flanking regions. The Cast2 family show homology only in a small number of left flanking regions (7 of 36 arrays), while the vast majority of Cast5 arrays (22 of 28) could be grouped according to similarities in left and right flanking regions. Flanking regions of Cast5 are further similar to R66 and R140 repetitive sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) extracted previously from the sequenced <em>T. castaneum</em> genome by RepeatScout analyses.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Detailed analyses of R66 and R140 show that both sequences are composed of Cast5 monomer parts together with parts of the array flanking region. Alignments of R66 and R140 (without the monomer part) with the flanking sequences of Cast5 family are presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>. In both cases, flanking regions show homology in 1 kb long segments, although R140-like flanking regions in some variants extend up to 2 kb. Significant variations were detected in positions of both left and right junctions of flanking sequences with respect to the TR array. A search of RepBase with R66-like and R140-like Cast5 flanking regions as a query shows stretch of 140 bp with a high homology (84%) of R140-like sequence to the non-autonomous Tc1/Mariner transposon defined in <em>T. castaneum</em>. This result suggests possibly a transposon property of Cast5 flanking regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In support, two-colour FISH with Cast5 monomer and centromeric satDNA (Fig. <a reveal-id="DSV021F4" data-open="DSV021F4" class="link link-reveal link-table xref-fig">4</a>F) indicates higher expansion of Cast5 family in comparison with all other families.</p><p>A schematic presentation of flanking regions and arrays of Cast5 family members with respect to monomer composition and orientation is presented in Fig. <a reveal-id="DSV021F6" data-open="DSV021F6" class="link link-reveal link-table xref-fig">6</a>A. The same orientation of monomers relative to flanking regions can be observed in almost all arrays, while flanking modules and monomers together are positioned in both directions with respect to the orientation of genomic sequences. </p>                        <a id="87196694" scrollto-destination="87196694"></a>
<div id="DSV021F6" data-id="DSV021F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02106.png?Expires=2147483647&Signature=HPVrq85~US20nRGriTkmgbzl1AFgpimCeNZtvdv0hEJ70JtcYhe0-Lys1zvSfijme0DKU1bx7CeiG0uZYWhGB~AZViTeB9jzvhanD~AERYDwbIYarWLiAfiLBNPoUUePvQquDAu2RQ4gtun6rHlRk3n66G1SWBB6R2tZ2oqLMwGLuohyHdzLDM8wLmev3uMz8XkUKQYzo7QNQmE5q2U5LrWdCpv2dpa4VS7~VXGxMPEErR2hF-fPIzxKeetrVI6zXAKvVMmjp1rVf2GaCJtq4VFUkw9rQtFyBMwhAEaa6lHBK4mvndP8C2EqeXIMQPyRFr6rHdmI3sSZt5NWQ8JZuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic presentation of flanking regions composed of R66-like and R140-like sequences associated with arrays of Cast5 (A). Two-colour FISH of R66-like (green) and Cast5 (red) probes hybridized on T. castaneum chromosomes (B)." path-from-xml="dsv02106.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196694/dsv02106.tiff" path-from-xml="dsv02106.tiff" target="_blank">View large</a><a section="87196694" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02106.png?Expires=2147483647&Signature=wkCDTUK9vxC851Vl5P~8D7-tjNqo-EldtiJfXshsoEv9hW5MfcHPhmHDFD~ifuSQ5H~3zcA4kAaAcBU7BxZRWkkEHkrCTJfttFxdegbiS5QSXt1Uiz8lw0gIPNEXo7lA2zjumzAfxWtH4qGA4W9vSDkeTVpJ0zh~LVZvAcGcycLSI2toRNLtLcjbem9kqZOsCWJ2~6RGGZ1zhlourKQO-xq4ilCewxUtOGA5OcY0fHIwPrRDGEBJAnRbgZUvWm6I2VpRXVfJpYt-umRFu-lPNvZgSxCKCoLpF7mpZu9FyC-hFvLFNiwdpqg-S2snMAtpfrt4rrMkmYP0l5Gx5iMBAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196694&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02106.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic presentation of flanking regions composed of R66-like and R140-like sequences associated with arrays of Cast5 (A). Two-colour FISH of R66-like (green) and Cast5 (red) probes hybridized on <em>T. castaneum</em> chromosomes (B).</p></div></div><div content-id="DSV021F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/m_dsv02106.png?Expires=2147483647&Signature=HPVrq85~US20nRGriTkmgbzl1AFgpimCeNZtvdv0hEJ70JtcYhe0-Lys1zvSfijme0DKU1bx7CeiG0uZYWhGB~AZViTeB9jzvhanD~AERYDwbIYarWLiAfiLBNPoUUePvQquDAu2RQ4gtun6rHlRk3n66G1SWBB6R2tZ2oqLMwGLuohyHdzLDM8wLmev3uMz8XkUKQYzo7QNQmE5q2U5LrWdCpv2dpa4VS7~VXGxMPEErR2hF-fPIzxKeetrVI6zXAKvVMmjp1rVf2GaCJtq4VFUkw9rQtFyBMwhAEaa6lHBK4mvndP8C2EqeXIMQPyRFr6rHdmI3sSZt5NWQ8JZuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic presentation of flanking regions composed of R66-like and R140-like sequences associated with arrays of Cast5 (A). Two-colour FISH of R66-like (green) and Cast5 (red) probes hybridized on T. castaneum chromosomes (B)." path-from-xml="dsv02106.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196694/dsv02106.tiff" path-from-xml="dsv02106.tiff" target="_blank">View large</a><a section="87196694" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv02106.png?Expires=2147483647&Signature=wkCDTUK9vxC851Vl5P~8D7-tjNqo-EldtiJfXshsoEv9hW5MfcHPhmHDFD~ifuSQ5H~3zcA4kAaAcBU7BxZRWkkEHkrCTJfttFxdegbiS5QSXt1Uiz8lw0gIPNEXo7lA2zjumzAfxWtH4qGA4W9vSDkeTVpJ0zh~LVZvAcGcycLSI2toRNLtLcjbem9kqZOsCWJ2~6RGGZ1zhlourKQO-xq4ilCewxUtOGA5OcY0fHIwPrRDGEBJAnRbgZUvWm6I2VpRXVfJpYt-umRFu-lPNvZgSxCKCoLpF7mpZu9FyC-hFvLFNiwdpqg-S2snMAtpfrt4rrMkmYP0l5Gx5iMBAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196694&ar=2462399&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02106.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic presentation of flanking regions composed of R66-like and R140-like sequences associated with arrays of Cast5 (A). Two-colour FISH of R66-like (green) and Cast5 (red) probes hybridized on <em>T. castaneum</em> chromosomes (B).</p></div></div><p>To investigate the extent of genomic co-localization of Cast5 repeats and flanking regions, we further performed two-colour FISH experiment with probes specific for the flanking sequence similar to R66 and for the Cast5 monomer (Fig. <a reveal-id="DSV021F6" data-open="DSV021F6" class="link link-reveal link-table xref-fig">6</a>B). The results obtained on meiotic pro metaphase chromosome spreads produced mostly co-localized hybridization signals, although individual signals corresponding to the flanking region as well as to TR family alone can be seen. This result is in accordance with bioinformatic analysis of Cast5 arrays in which some flanking sequence diverges significantly from R66-like.</p><p>Given that the other analysed TR families do not show any regularity in flanking regions, we inspected junction position in monomer sequences in order to asses if there is any preferential monomer part that might be linked with the transition site. Inspection of these junctions did not reveal any evident correlation between the sequence and junction position (data not shown).</p>                            <h2 scrollto-destination=87196700 id="87196700" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>A global survey of TRs throughout the entire genome is important for correct annotation of these sequences in genome assemblies, and also to improve insights into their evolutionary dynamics and mechanisms of emergence and expansion. The availability of a whole-genome assembly of <em>T. castaneum</em> mapped to chromosomes was the most important prerequisite in addressing these issues, making possible examination of TR evolution at the chromosomal and at the repeat-array level. To reveal the distribution profile and structural features of TRs as well as to build a database of highly abundant TRs, we first applied TRF analysis to the <em>T. castaneum</em> genome assembly, and focused on arrays with repeats in the range of 100–500 bp. The total amount of detected TRs is 1.63 Mb, which constitutes ∼1.04% of the assembled genome. Arrays are not uniformly distributed between chromosomes, showing higher density on five chromosomes. This result is in agreement with a previous report of overall repeat families and TE classes in the assembled genome.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> However, our combined FISH and bioinformatics studies of TCAST, the most abundant <em>T. castaneum</em> satDNA,<sup><a reveal-id="DSV021C26" data-open="DSV021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> confirmed the complete absence of centromeric regions in the genome assembly. The distribution profile of TCAST shows dominant localization in centromeric regions and arrays with only a few copies outside of centromeric regions. Another centromeric satDNA from <em>T. castaneum</em>, TCAST2, is similar to TCAST, and found in diverse euchromatic locations only in the monomeric form.<sup><a reveal-id="DSV021C40" data-open="DSV021C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Distribution profiles with large blocks of satDNA in centromeric regions and short arrays of centromeric satDNA (up to five monomers) located in the euchromatin were also defined for 1.688 and Rsp satDNAs in the <em>Drosophila melanogaster</em> genome.<sup><a reveal-id="DSV021C14" data-open="DSV021C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV021C41" data-open="DSV021C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p><p>In contrast to the centromeric region with dominant satDNA sequences, domains rich in dispearsed repeats and proclaimed as putative heterochromatin regions in Wang et al.<sup><a reveal-id="DSV021C28" data-open="DSV021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> flank large blocks of centromeric heterochromatin in <em>T. castaneum</em>. A similar architecture with alpha satDNAs predominantly located in centromeric regions and non-LTR transposons which colonize pericentromeric regions is observed in the human genome.<sup><a reveal-id="DSV021C42" data-open="DSV021C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV021C43" data-open="DSV021C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>Regardless of insignificant presence of TCAST and TCAST2 outside of centromeric regions, the distribution profile of the entire TR content indicates their significant contribution in the assembled genome. Interestingly, in addition to the arrays composed of several repeats (&lt;5 copies), there is a significant number of arrays based on ≥5 repeats.</p><p>Moreover, FISH and quantitative analysis of the new, most abundant TR families (Cast1–Cast9) characterized in the assembled genome established them as euchromatic and recovered their existence in significantly higher amounts than predicted by the assembled genome data. This was particularly obvious in Cast1 FISH analysis, where strong signals suggest the presence of long unassembled arrays almost exclusively located outside of centromeric regions. The estimated genome size of 204 Mb is 44 Mb larger than the assembled genome sequence. Based on our results, it can be assumed that the dominant fraction of unassembled genome constitute highly abundant TRs, including centromeric satDNAs TCAST and euchromatic TR families Cast1–Cast9.</p><p>Structural analysis of overall TRs in the assembled <em>T. castaneum</em> genome revealed a correlation between the monomer length and the number of monomers in arrays. There is an obvious predominance of ∼170 bp monomer length if the number of monomers in an array increases. In agreement with the experimentally estimated contribution, the most abundant TR families detected in euchromatic regions have the monomer lengths of ∼170 bp and of its double length. Similarly, all five subfamilies of centromeric TCAST satDNA are based on monomers of ∼360 bp. The preferential length of repeat units in abundant TR families (both euchromatic and centromeric) can be linked to requirements for efficient DNA packing of long arrays in chromatin structures. It is well known that chromatin in eukaryotic genomes is organized into the nucleosome with 1.67 turns of DNA around the histone octamer (147 bp) plus the linker DNA with a variable distance (10–70 bp).<sup><a reveal-id="DSV021C44" data-open="DSV021C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Similar rules govern the human and plant centromeres where the periodicity of centromeric nucleosomes (CenH3) is found to be in accord with the satDNA monomer length,<sup><a reveal-id="DSV021C45  DSV021C47" data-open="DSV021C45  DSV021C47" class="link link-ref link-reveal xref-bibr">45–47</a></sup> being predominantly of ∼170 bp and of its double length.<sup><a reveal-id="DSV021C4" data-open="DSV021C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The phenomenon observed in this work can be thus explained by the length of DNA wrapped around 1 or 2 nucleosomes as a requirement that may facilitate regular phasing of nucleosomes, and it can be a necessary prerequisite for dramatic amplification of TRs in both, centromeric and euchromatic regions. It has also been proposed for centromeric satDNAs that the monomer length longer than two nucleosomes is rare because longer sequences are unlikely to stabilize nucleosomes.<sup><a reveal-id="DSV021C3" data-open="DSV021C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSV021C46" data-open="DSV021C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Our overall analysis of TRs in the <em>T. castaneum</em> assembled genome is in accordance with this idea, showing that the proportion of arrays built of monomers &gt;380 bp dramatically decreases.</p><p>In addition to preferential monomer length in the nine TR families structural analyses further revealed a periodic distribution of A or T tracts (4–10 nucleotides), in the same manner as observed in TCAST and other centromeric satDNAs found in several other <em>Tribolium</em> species.<sup><a reveal-id="DSV021C22" data-open="DSV021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The periodic appearance of AA/TT dimers along eukaryotic DNA sequences, including those in centromeric regions, promote this nucleotide sequence pattern as a feature that may facilitate nucleosome formation.<sup><a reveal-id="DSV021C45" data-open="DSV021C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSV021C46" data-open="DSV021C46" class="link link-ref link-reveal xref-bibr">46</a></sup> In this regard, periodically distributed tracts of As and/or Ts, present in many centromeric satDNAs of tenebrionid beetles were shown to define the sequence-induced curvature of the DNA helix axis and could facilitate the tight packing of DNA in centromeric heterochromatin.<sup><a reveal-id="DSV021C48" data-open="DSV021C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSV021C49" data-open="DSV021C49" class="link link-ref link-reveal xref-bibr">49</a></sup> We hypothesize that, similar to the centromeric regions, monomer length and other sequence features (e.g. AT tracts) could be equally important for expansion of TRs in euchromatic regions and may be linked to the formation of hypothetical micro-heterochromatic regions embedded within the euchromatic chromosomal arms. Regardless of their similar structural features, our analyses show a high level of chromosomal compartmentalization of TRs in the <em>T. castaneum</em> genome, where prominent arrays of Cast1–Cast9 families are located almost exclusively in non-centromeric, i.e. euchromatic domains, in contrast to centromeric distribution of TCAST satDNA. Clear chromosomal compartmentalization of centromeric and different euchromatic TRs might suggest the existence of some additional requirements which could be important for expansion of TRs in different domains, for example, in functional centromeres.<sup><a reveal-id="DSV021C50" data-open="DSV021C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p><p>Within this study, we also focused on the nine most prominent non-centromeric TR families (Cast1–Cast9) in order to define evolutionary trends and mechanisms of TR dispersion throughout the genome. Phylogenetic analyses of monomers show a similar evolutionary scheme for all analysed TR families. Clustering of repeats from the same array is mainly observed in longer arrays, while monomers originating from arrays built of 5–7 monomers are often found dispersed in phylogenetic trees. These data suggest that the efficiency of homogenization mechanisms and concerted evolution in euchromatic regions depends on array length. In support of this, early computer simulations showed that homogenization mechanisms work better in long arrays, but poorly in those containing only few repeats.<sup><a reveal-id="DSV021C7" data-open="DSV021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In addition, our results show that monomers originating from the same chromosome do not group with significantly higher frequency than monomers located on heterologous chromosomes. Moreover, phylogenetic analyses recovered extensive exchanges between non-homologous chromosomes in almost all analysed TR families. This trend is particularly obvious in Cast2, Cast3, Cast4 and Cast5 families, where dominant clusters include arrays originating from all chromosomes. The presence of several subclusters originating from non-homologous chromosomes in almost each analysed tree allows us to propose that these families were spread in several rounds of inter-chromosomal exchange and subsequent amplification. Genome-wide expansion events suggest efficient mechanism(s) of TR propagation in euchromatic genome regions. Previous studies on evolutionary trends involving TRs were focused on centromeric satDNAs. Studies in plants and human thus showed higher sequence divergence between satDNAs located on different chromosomes than within a chromosome, and revealed the preferential occurrence of sequence exchange and homogenization at the intra-chromosomal level.<sup><a reveal-id="DSV021C51" data-open="DSV021C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSV021C52" data-open="DSV021C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The most prominent example is human alpha satDNA whose higher order repeat units show chromosome-specific differences in monomer composition and length.<sup><a reveal-id="DSV021C8" data-open="DSV021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, this homogenization pattern is not universal, and TRs in subtelomeric regions share a high degree of sequence identity despite being located on non-homologous chromosomes.<sup><a reveal-id="DSV021C53" data-open="DSV021C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSV021C54" data-open="DSV021C54" class="link link-ref link-reveal xref-bibr">54</a></sup> It has been postulated that exchanges between non-homologous chromosome ends occur during meiotic prophase, when all chromatids are interconnected.<sup><a reveal-id="DSV021C53" data-open="DSV021C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Similarly, enhanced efficiency in the spread of centromeric satDNA between non-homologous chromosomes noticed in many <em>Tribolium</em> species could be facilitated by a bouquet formation which occurs during the first meiotic division.<sup><a reveal-id="DSV021C25" data-open="DSV021C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSV021C55" data-open="DSV021C55" class="link link-ref link-reveal xref-bibr">55</a></sup></p><p>To assess mechanisms underlying the spread of TR sequences throughout the <em>T. castaneum</em> genome, we analysed TR array flanking regions, and recovered homologous flanking regions which resemble TEs only for the extremely dispersed Cast5 family. This suggests that repetitive sequences of the Cast5 family were initially distributed by a certain transposition activity and additionally amplified in longer arrays. SatDNA transposition has been advocated in some studies as alternative mechanism in the evolutionary dynamics of human centromeric region<sup><a reveal-id="DSV021C56" data-open="DSV021C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Similarly, enrichment in TRs derived from LTRs in <em>Zea mays</em> centromeres raises the possibility that centromeric satDNA can be renewed or replaced by novel satDNA repeats derived from retrotransposons.<sup><a reveal-id="DSV021C57" data-open="DSV021C57" class="link link-ref link-reveal xref-bibr">57</a></sup> In <em>T. castaneum</em>, TCAST-like elements have been identified within complex units that resemble a DNA transposon and some of them were inserted into introns.<sup><a reveal-id="DSV021C27" data-open="DSV021C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>It has also been proposed that the mechanisms of dispersion of TR families in human may be related to duplication of large segments in which arrays are embedded.<sup><a reveal-id="DSV021C12" data-open="DSV021C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Flanking regions of other analysed non-centromeric TR families in <em>T. castaneum</em> do not display mutual homology thus eliminating mechanisms of segmental duplication in the spread of these sequences. In addition, their junctions and flanking regions do not show any specific feature in the form of inverted repeats or motifs which could act as mediators in the mechanism of dispersion. However, taking into account that the <em>T. castaneum</em> genome is as AT-rich (67% A+T) as analysed TR families (66–74% A + T), we suspect that rolling circle amplification could be the mechanism of dispersion, while regions of micro-homologies in the form of AT tracts could be promoters of insertions at different locations. Recent data suggest rolling circle replication and reinsertion of extrachromosomal circular DNA as the mechanism of TR propagation in various organisms, including humans.<sup><a reveal-id="DSV021C11" data-open="DSV021C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV021C58" data-open="DSV021C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p><p>It has been proposed that recombination in centromeric regions is suppressed to prevent the deleterious effects of crossing over between megabase-sized arrays.<sup><a reveal-id="DSV021C59" data-open="DSV021C59" class="link link-ref link-reveal xref-bibr">59</a></sup> However, the extremely efficient distribution and rapid expansion of different TR families found in euchromatic regions can induce intensive rearrangements between diverse genome loci and thus contribute to genomic instability. In addition, TRs dispersed throughout the genome have exceptional potential to evolve independently, creating lineage-specific changes in the structure, sequence, or chromosomal localization, and thus generating incompatibilities between populations/species. Such rapid changes in genomes have been implicated in the post-zygotic isolation of several <em>Drosophila</em> species demonstrating the critical role of satDNA in hybrid incompatibility.<sup><a reveal-id="DSV021C60" data-open="DSV021C60" class="link link-ref link-reveal xref-bibr">60</a>,<a reveal-id="DSV021C61" data-open="DSV021C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Recent data based on analysis of TR profiles in multiple populations of <em>D. melanogaster</em> show significant differentiation of many analysed simple TRs at the population level.<sup><a reveal-id="DSV021C62" data-open="DSV021C62" class="link link-ref link-reveal xref-bibr">62</a></sup></p><p>Recent studies indicated that TRs can have remarkable effects on the euchromatic part of the genome. For example, length variation in multimegabase stretches of satDNA repeats of the <em>Drosophila</em> Y chromosome could be the major source of epigenetic variation which can modulate gene expression and cause variable phenotypes including differences in immune response.<sup><a reveal-id="DSV021C63" data-open="DSV021C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Our analysis of non-centromeric TRs in <em>T. castaneum</em> shows extremely efficient propagation in the euchromatic regions and suggest that they could be important factors in the modulation of gene expression. Their impact could be the result of direct insertion of arrays in introns/regulatory elements of genes, or through the formation of micro-heterochromatic environment in the vicinity of genes which may lead to the position effect variegation, and thus modulating the gene expression.</p><p>In conclusion, using a combination of bioinformatics and experimental approaches, we have delineated structural and evolutionary trends of TRs in euchromatic regions. The euchromatic chromosomal regions of the <em>T. castaneum</em> genome are replete with different highly abundant TRs which are prone to amplify into long arrays if they meet the preferential monomer length. In contrast to similar structural features between centromeric and euchromatic TRs, these two categories show clear chromosomal compartmentalization, suggesting that additional requirements may be imposed on sequences belonging to different regions. The observed evolutionary pattern of euchromatic TR families suggests intensive impact of concerted evolution on longer arrays, while homogenization of short arrays remains limited. Evolutionary trends suggest that efficient inter-chromosomal exchanges were followed by amplification on almost all chromosomes. Our results suggest efficient dispersion of nine TR families through the euchromatin regions and possible role of a transposition-like mechanism only in the case of the most expanded family. We propose that recombination between homologous TRs dispersed among euchromatic sequences can affect both expression of coding information as well as lead to deleterious chromosomal rearrangements.</p>                            <h2 scrollto-destination=87196720 id="87196720" class="section-title" data-legacy-id=s5>Authors’ contributions</h2>
<p>N.M. conceived the study, designed the experiments and analyses, and wrote the manuscript. M. Pavlek performed experiments and analyses, and participated in data interpretation. Y.G. participated in TRF analyses. M. Plohl participated in data interpretation, and critical revision of the manuscript. All authors read, and approved the manuscript.</p>                            <h2 scrollto-destination=87196724 id="87196724" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by Research Fund of Republic of Croatia, project no. 098-0982913-2756 and by funds received from the European Union's Seventh Framework Programme for Research and Technological Development under grant agreement No 316289 - InnoMol, FP7-REGPOT-2012-2013-1. Funding to pay the Open Access publication charges for this article was provided by Research Fund of Republic of Croatia, project no. 098-0982913-2756.</p>                            <h2 scrollto-destination=87196727 id="87196727" class="backacknowledgements-title" >Acknowledgments</h2>
<p>The authors thank Brankica Mravinac for critical reading of the manuscript. We are indebted to the colleagues from the Department of Molecular Biology (Faculty of Science, University of Zagreb) for using their fluorescence microscope. We thank Mary Sopta for critical reading of the manuscript.</p>                            <h2 scrollto-destination=87196730 id="87196730" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plohl</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Meštrović</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Mravinac</div> <div class="given-names">B.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Satellite DNA evolution</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">Garrido-Ramos</div> <div class="given-names">M.</div></div></div> (ed.), <div class="source">Repetitive DNA. Genome Dyn 7</div> . <div class="publisher-name">Karger Publishers</div>, <div class="publisher-loc">Basel</div>, pp. <div class="fpage">126</div>–<div class="lpage">52</div>.</div></div></div></div><div content-id="DSV021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Talbert</div>
               <div class="given-names">P.B.</div>
            </div>, <div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Centromeres convert but don't cross</div>, <div class="source">PLoS Biol.</div> , <div class="volume">8</div>, <div class="fpage">1</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Centromeres%20convert%20but%20don't%20cross&amp;author=P.B.%20Talbert&amp;author=S.%20Henikoff&amp;publication_year=2010&amp;journal=PLoS%20Biol.&amp;volume=8&amp;pages=1-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1000326" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1000326" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1000326"> </span></div> </div></div></div></div></div><div content-id="DSV021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Melters</div>
               <div class="given-names">D.P.</div>
            </div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.R.</div></div>, <div class="name string-name"><div class="surname">Young</div> <div class="given-names">H.A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Comparative analysis of tandem repeats from hundreds of species reveals unique insights into centromere evolution</div>, <div class="source">Genome Biol.</div> , <div class="volume">14</div>, <div class="fpage">R10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20tandem%20repeats%20from%20hundreds%20of%20species%20reveals%20unique%20insights%20into%20centromere%20evolution&amp;author=D.P.%20Melters&amp;author=K.R.%20Bradnam&amp;author=H.A.%20Young&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-1-r10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-1-r10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-1-r10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23363705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heslop-Harrison</div>
               <div class="given-names">J.S.P.</div>
            </div>, <div class="name string-name"><div class="surname">Schwarzacher</div> <div class="given-names">T.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Nucleosomes and centromeric DNA packaging</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">1</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosomes%20and%20centromeric%20DNA%20packaging&amp;author=J.S.P.%20Heslop-Harrison&amp;author=T.%20Schwarzacher&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=1-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1319945110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1319945110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1319945110"> </span></div> </div></div></div></div></div><div content-id="DSV021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dover</div>
               <div class="given-names">G.A.</div>
            </div>
          </div>
          <div class="year">1986</div>, <div class="article-title">Molecular drive in multigene families: how biological novelties arise, spread and are assimilated</div>, <div class="source">Trends Genet.</div> , <div class="volume">2</div>, <div class="fpage">159</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20drive%20in%20multigene%20families%3A%20how%20biological%20novelties%20arise%2C%20spread%20and%20are%20assimilated&amp;author=G.A.%20Dover&amp;publication_year=1986&amp;journal=Trends%20Genet.&amp;volume=2&amp;pages=159-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(86)90211-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(86)90211-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(86)90211-8"> </span></div> </div></div></div></div></div><div content-id="DSV021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mahtani</div>
               <div class="given-names">M.M.</div>
            </div>, <div class="name string-name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Physical and genetic mapping of the human×chromosome centromere: repression of recombination</div>, <div class="source">Genome Res.</div> , <div class="volume">8</div>, <div class="fpage">100</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20and%20genetic%20mapping%20of%20the%20human%C3%97chromosome%20centromere%3A%20repression%20of%20recombination&amp;author=M.M.%20Mahtani&amp;author=H.F.%20Willard&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=100-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.2.100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.2.100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.2.100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9477338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">G.P.</div>
            </div>
          </div>
          <div class="year">1976</div>, <div class="article-title">Evolution of repeated DNA sequences by unequal crossover</div>, <div class="source">Science</div> , <div class="volume">191</div>, <div class="fpage">528</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20repeated%20DNA%20sequences%20by%20unequal%20crossover&amp;author=G.P.%20Smith&amp;publication_year=1976&amp;journal=Science&amp;volume=191&amp;pages=528-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1251186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1251186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1251186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1251186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rudd</div>
               <div class="given-names">M.K.</div>
            </div>, <div class="name string-name"><div class="surname">Wray</div> <div class="given-names">G.A.</div></div>, <div class="name string-name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">The evolutionary dynamics of alpha-satellite</div>, <div class="source">Genome Res.</div> , <div class="volume">16</div>, <div class="fpage">88</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20dynamics%20of%20alpha-satellite&amp;author=M.K.%20Rudd&amp;author=G.A.%20Wray&amp;author=H.F.%20Willard&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=88-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3810906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3810906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3810906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dias</div>
               <div class="given-names">G.B.</div>
            </div>, <div class="name string-name"><div class="surname">Svartman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Delprat</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ruiz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kuhn</div> <div class="given-names">G.C.S.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Tetris is a foldback transposon that provided the building blocks for an emerging satellite DNA of <em>Drosophila virilis</em></div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">6</div>, <div class="fpage">1302</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tetris%20is%20a%20foldback%20transposon%20that%20provided%20the%20building%20blocks%20for%20an%20emerging%20satellite%20DNA%20of%20Drosophila%20virilis&amp;author=G.B.%20Dias&amp;author=M.%20Svartman&amp;author=A.%20Delprat&amp;author=A.%20Ruiz&amp;author=G.C.S.%20Kuhn&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol.&amp;volume=6&amp;pages=1302-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24858539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Šatović</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Tandem repeat-containing MITE elements in the clam <em>Donax trunculus</em></div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">5</div>, <div class="fpage">2549</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeat-containing%20MITE%20elements%20in%20the%20clam%20Donax%20trunculus&amp;author=E.%20%C5%A0atovi%C4%87&amp;author=M.%20Plohl&amp;publication_year=2013&amp;journal=Genome%20Biol.%20Evol.&amp;volume=5&amp;pages=2549-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evt202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevt202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevt202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24317975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cohen</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Agmon</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Sobol</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Segal</div> <div class="given-names">D.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Extrachromosomal circles of satellite repeats and 5S ribosomal DNA in human cells</div>, <div class="source">Mob. DNA</div> , <div class="volume">1</div>, <div class="fpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extrachromosomal%20circles%20of%20satellite%20repeats%20and%205S%20ribosomal%20DNA%20in%20human%20cells&amp;author=S.%20Cohen&amp;author=N.%20Agmon&amp;author=O.%20Sobol&amp;author=D.%20Segal&amp;publication_year=2010&amp;journal=Mob.%20DNA&amp;volume=1&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1759-8753-1-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1759-8753-1-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1759-8753-1-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20226008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Warburton</div>
               <div class="given-names">P.E.</div>
            </div>, <div class="name string-name"><div class="surname">Hasson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Guillem</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Lescale</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Abrusan</div> <div class="given-names">G.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Analysis of the largest tandemly repeated DNA families in the human genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">533</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20largest%20tandemly%20repeated%20DNA%20families%20in%20the%20human%20genome&amp;author=P.E.%20Warburton&amp;author=D.%20Hasson&amp;author=F.%20Guillem&amp;author=C.%20Lescale&amp;author=X.%20Jin&amp;author=G.%20Abrusan&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=533" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-533" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-533" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-533"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18992157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Komissarov</div>
               <div class="given-names">A.S.</div>
            </div>, <div class="name string-name"><div class="surname">Gavrilova</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Demin</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Ishov</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Podgornaya</div> <div class="given-names">O.I.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Tandemly repeated DNA families in the mouse genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">531</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandemly%20repeated%20DNA%20families%20in%20the%20mouse%20genome&amp;author=A.S.%20Komissarov&amp;author=E.V.%20Gavrilova&amp;author=S.J.%20Demin&amp;author=A.M.%20Ishov&amp;author=O.I.%20Podgornaya&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=531" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22035034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kuhn</div>
               <div class="given-names">G.C.S.</div>
            </div>, <div class="name string-name"><div class="surname">Küttler</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Moreira-Filho</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Heslop-Harrison</div> <div class="given-names">J.S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">The 1. 688 repetitive DNA of <em>Drosophila</em>: concerted evolution at different genomic scales and association with genes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">29</div>, <div class="fpage">7</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%201.%20688%20repetitive%20DNA%20of%20Drosophila%3A%20concerted%20evolution%20at%20different%20genomic%20scales%20and%20association%20with%20genes&amp;author=G.C.S.%20Kuhn&amp;author=H.%20K%C3%BCttler&amp;author=O.%20Moreira-Filho&amp;author=J.S.%20Heslop-Harrison&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=29&amp;pages=7-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21712468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altemose</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Miga</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Maggioni</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Genomic characterization of large heterochromatic gaps in the human genome assembly</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">10</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20characterization%20of%20large%20heterochromatic%20gaps%20in%20the%20human%20genome%20assembly&amp;author=N.%20Altemose&amp;author=K.H.%20Miga&amp;author=M.%20Maggioni&amp;author=H.F.%20Willard&amp;publication_year=2014&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=10&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003628" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003628" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003628"> </span></div> </div></div></div></div></div><div content-id="DSV021C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stam</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Belele</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ramakrishna</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Dorweiler</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Chandler</div> <div class="given-names">V.L.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">The regulatory regions required for B paramutation and expression are located far upstream of the maize b1 transcribed sequences</div>, <div class="source">Genetics</div> , <div class="volume">162</div>, <div class="fpage">917</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20regulatory%20regions%20required%20for%20B%20paramutation%20and%20expression%20are%20located%20far%20upstream%20of%20the%20maize%20b1%20transcribed%20sequences&amp;author=M.%20Stam&amp;author=C.%20Belele&amp;author=W.%20Ramakrishna&amp;author=J.E.%20Dorweiler&amp;author=J.L.%20Bennetzen&amp;author=V.L.%20Chandler&amp;publication_year=2002&amp;journal=Genetics&amp;volume=162&amp;pages=917-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhu</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Pao</div> <div class="given-names">G.M.</div></div>, <div class="name string-name"><div class="surname">Huynh</div> <div class="given-names">A.M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">BRCA1 tumour suppression occurs via heterochromatin-mediated silencing</div>, <div class="source">Nature</div> , <div class="volume">477</div>, <div class="fpage">179</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BRCA1%20tumour%20suppression%20occurs%20via%20heterochromatin-mediated%20silencing&amp;author=Q.%20Zhu&amp;author=G.M.%20Pao&amp;author=A.M.%20Huynh&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=179-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21901007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Scott</div>
               <div class="given-names">H.S.</div>
            </div>, <div class="name string-name"><div class="surname">Kudoh</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wattenhofer</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Insertion of beta-satellite repeats identifies a transmembrane protease causing both congenital and childhood onset autosomal recessive deafness</div>, <div class="source">Nat. Genet.</div> , <div class="volume">27</div>, <div class="fpage">59</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion%20of%20beta-satellite%20repeats%20identifies%20a%20transmembrane%20protease%20causing%20both%20congenital%20and%20childhood%20onset%20autosomal%20recessive%20deafness&amp;author=H.S.%20Scott&amp;author=J.%20Kudoh&amp;author=M.%20Wattenhofer&amp;publication_year=2001&amp;journal=Nat.%20Genet.&amp;volume=27&amp;pages=59-63" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11137999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paar</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Gluncić</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rosandić</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Basar</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Vlahović</div> <div class="given-names">I.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Intragene higher order repeats in neuroblastoma breakpoint family genes distinguish humans from chimpanzees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">28</div>, <div class="fpage">1877</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intragene%20higher%20order%20repeats%20in%20neuroblastoma%20breakpoint%20family%20genes%20distinguish%20humans%20from%20chimpanzees&amp;author=V.%20Paar&amp;author=M.%20Glunci%C4%87&amp;author=M.%20Rosandi%C4%87&amp;author=I.%20Basar&amp;author=I.%20Vlahovi%C4%87&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=1877-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21273634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Richards</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Gibbs</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Weinstock</div> <div class="given-names">G.M.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">The genome of the model beetle and pest <em>Tribolium castaneum</em></div>, <div class="source">Nature</div> , <div class="volume">452</div>, <div class="fpage">949</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20model%20beetle%20and%20pest%20Tribolium%20castaneum&amp;author=S.%20Richards&amp;author=R.A.%20Gibbs&amp;author=G.M.%20Weinstock&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=949-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06784" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06784" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06784"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18362917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ugarković</div>
               <div class="given-names">Đ.</div>
            </div>, <div class="name string-name"><div class="surname">Podnar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Satellite DNA of the red flour beetle <em>Tribolium castaneum</em> – comparative study of satellites from the genus Tribolium</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">13</div>, <div class="fpage">1059</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Satellite%20DNA%20of%20the%20red%20flour%20beetle%20Tribolium%20castaneum%20%E2%80%93%20comparative%20study%20of%20satellites%20from%20the%20genus%20Tribolium&amp;author=%C4%90.%20Ugarkovi%C4%87&amp;author=M.%20Podnar&amp;author=M.%20Plohl&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=13&amp;pages=1059-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025668" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8865660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mravinac</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ugarković</div> <div class="given-names">Ð.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Conserved patterns in the evolution of <em>Tribolium</em> satellite DNAs.pdf</div>, <div class="source">Gene</div> , <div class="volume">332</div>, <div class="fpage">169</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20patterns%20in%20the%20evolution%20of%20Tribolium%20satellite%20DNAs.pdf&amp;author=B.%20Mravinac&amp;author=M.%20Plohl&amp;author=%C3%90.%20Ugarkovi%C4%87&amp;publication_year=2004&amp;journal=Gene&amp;volume=332&amp;pages=169-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.02.055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.02.055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.02.055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mravinac</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Ugarković</div> <div class="given-names">Đ.</div></div>, <div class="name string-name"><div class="surname">Franjević</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Long inversely oriented subunits form a complex monomer of <em>Tribolium brevicornis</em> satellite DNA</div>, <div class="source">J. Mol. Evol.</div> , <div class="volume">60</div>, <div class="fpage">513</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20inversely%20oriented%20subunits%20form%20a%20complex%20monomer%20of%20Tribolium%20brevicornis%20satellite%20DNA&amp;author=B.%20Mravinac&amp;author=%C4%90.%20Ugarkovi%C4%87&amp;author=D.%20Franjevi%C4%87&amp;author=M.%20Plohl&amp;publication_year=2005&amp;journal=J.%20Mol.%20Evol.&amp;volume=60&amp;pages=513-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-004-0236-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-004-0236-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-004-0236-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15883886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mravinac</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Satellite DNA junctions identify the potential origin of new repetitive elements in the beetle <em>Tribolium madens</em></div>, <div class="source">Gene</div> , <div class="volume">394</div>, <div class="fpage">45</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Satellite%20DNA%20junctions%20identify%20the%20potential%20origin%20of%20new%20repetitive%20elements%20in%20the%20beetle%20Tribolium%20madens&amp;author=B.%20Mravinac&amp;author=M.%20Plohl&amp;publication_year=2007&amp;journal=Gene&amp;volume=394&amp;pages=45-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.01.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.01.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.01.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mravinac</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Parallelism in evolution of highly repetitive DNAs in sibling species</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">27</div>, <div class="fpage">1857</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallelism%20in%20evolution%20of%20highly%20repetitive%20DNAs%20in%20sibling%20species&amp;author=B.%20Mravinac&amp;author=M.%20Plohl&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=1857-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20203289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Feliciello</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Chinali</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Ugarković</div> <div class="given-names">Đ.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Structure and population dynamics of the major satellite DNA in the red flour beetle <em>Tribolium castaneum</em></div>, <div class="source">Genetica</div> , <div class="volume">139</div>, <div class="fpage">999</div>–<div class="lpage">1008</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20population%20dynamics%20of%20the%20major%20satellite%20DNA%20in%20the%20red%20flour%20beetle%20Tribolium%20castaneum&amp;author=I.%20Feliciello&amp;author=G.%20Chinali&amp;author=%C4%90.%20Ugarkovi%C4%87&amp;publication_year=2011&amp;journal=Genetica&amp;volume=139&amp;pages=999-1008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-011-9601-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-011-9601-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-011-9601-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21837441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brajković</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Feliciello</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Bruvo-Mađarić</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Satellite DNA-like elements associated with genes within euchromatin of the beetle <em>Tribolium castaneum</em></div>, <div class="source">G3</div> , <div class="volume">2</div>, <div class="fpage">931</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Satellite%20DNA-like%20elements%20associated%20with%20genes%20within%20euchromatin%20of%20the%20beetle%20Tribolium%20castaneum&amp;author=J.%20Brajkovi%C4%87&amp;author=I.%20Feliciello&amp;author=B.%20Bruvo-Ma%C4%91ari%C4%87&amp;publication_year=2012&amp;journal=G3&amp;volume=2&amp;pages=931-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.003467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.003467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.003467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22908042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lorenzen</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Beeman</div> <div class="given-names">R.W.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">S.J.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Analysis of repetitive DNA distribution patterns in the <em>Tribolium castaneum</em> genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">9</div>, <div class="fpage">14</div>.</div></div></div></div><div content-id="DSV021C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stuart</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Mocelin</div> <div class="given-names">G.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">Cytogenetics of chromosome rearrangements in <em>Tribolium castaneum</em></div>, <div class="source">Genome</div> , <div class="volume">38</div>, <div class="fpage">673</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytogenetics%20of%20chromosome%20rearrangements%20in%20Tribolium%20castaneum&amp;author=J.%20Stuart&amp;author=G.%20Mocelin&amp;publication_year=1995&amp;journal=Genome&amp;volume=38&amp;pages=673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g95-085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg95-085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg95-085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18470196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gelfand</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Rodriguez</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Benson</div> <div class="given-names">G.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">TRDB – the Tandem Repeats Database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">35</div>, <div class="fpage">D80</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TRDB%20%E2%80%93%20the%20Tandem%20Repeats%20Database&amp;author=Y.%20Gelfand&amp;author=A.%20Rodriguez&amp;author=G.%20Benson&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D80-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl1013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl1013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl1013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17175540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Benson</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">1999</div>, <div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">27</div>, <div class="fpage">573</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rozas</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Rozas</div> <div class="given-names">R.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis</div>, <div class="source">Bioinformatics</div> , <div class="volume">15</div>, <div class="fpage">174</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%20version%203%3A%20an%20integrated%20program%20for%20molecular%20population%20genetics%20and%20molecular%20evolution%20analysis&amp;author=J.%20Rozas&amp;author=R.%20Rozas&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=174-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.2.174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.2.174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.2.174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10089204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guindon</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>, <div class="source">Syst. Biol.</div> , <div class="volume">52</div>, <div class="fpage">696</div>–<div class="lpage">704</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2003&amp;journal=Syst.%20Biol.&amp;volume=52&amp;pages=696-704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150390235520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Darriba</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Taboada</div> <div class="given-names">G.L.</div></div>, <div class="name string-name"><div class="surname">Doallo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">jModelTest 2: more models, new heuristics and parallel computing</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">772</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=jModelTest%202%3A%20more%20models%2C%20new%20heuristics%20and%20parallel%20computing&amp;author=D.%20Darriba&amp;author=G.L.%20Taboada&amp;author=R.%20Doallo&amp;author=D.%20Posada&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=772" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22847109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guindon</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Dufayard</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Lefort</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Anisimova</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hordijk</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</div>, <div class="source">Syst. Biol.</div> , <div class="volume">59</div>, <div class="fpage">307</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=J.F.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Syst.%20Biol.&amp;volume=59&amp;pages=307-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syq010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fsysbio%2Fsyq010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jurka</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>, <div class="name string-name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Walichiewicz</div> <div class="given-names">J.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Repbase update, a database of eukaryotic repetitive elements</div>, <div class="source">Cytogenet. Genome Res.</div> , <div class="volume">110</div>, <div class="fpage">462</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res.&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Drummond</div>
               <div class="given-names">A.J.</div>
            </div>, <div class="name string-name"><div class="surname">Ashton</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Buxton</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="source">Geneious. Version 5.4. Biomatters Ltd</div> <div class="article-title">.</div>, <div class="publisher-loc">Auckland, New Zealand</div>.</div></div></div></div><div content-id="DSV021C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Juan</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Petitpierre</div> <div class="given-names">E.</div></div></div>
          <div class="year">1991</div>, <div class="article-title">Chromosome number and sex-determining systems in Tenebrionidae (Coleoptera)</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">Zunino</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Belles</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Blas</div> <div class="given-names">M.</div></div></div> (ed.), <div class="source">Advances in Coleopterology</div> . <div class="publisher-name">AEC</div>, <div class="publisher-loc">Barcelona</div>, pp. <div class="fpage">167</div>–<div class="lpage">76</div>.</div></div></div></div><div content-id="DSV021C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plohl</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Meštrović</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Mravinac</div> <div class="given-names">B.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Centromere identity from the DNA point of view</div>, <div class="source">Chromosoma</div> , <div class="volume">4</div>, <div class="fpage">313</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Centromere%20identity%20from%20the%20DNA%20point%20of%20view&amp;author=M.%20Plohl&amp;author=N.%20Me%C5%A1trovi%C4%87&amp;author=B.%20Mravinac&amp;publication_year=2014&amp;journal=Chromosoma&amp;volume=4&amp;pages=313-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00412-014-0462-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00412-014-0462-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00412-014-0462-0"> </span></div> </div></div></div></div></div><div content-id="DSV021C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Feliciello</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Akrap</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Brajković</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zlatar</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Ugarković</div> <div class="given-names">Đ.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Satellite DNA as a driver of population divergence in the red flour beetle <em>Tribolium castaneum</em></div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">7</div>, <div class="fpage">228</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Satellite%20DNA%20as%20a%20driver%20of%20population%20divergence%20in%20the%20red%20flour%20beetle%20Tribolium%20castaneum&amp;author=I.%20Feliciello&amp;author=I.%20Akrap&amp;author=J.%20Brajkovi%C4%87&amp;author=I.%20Zlatar&amp;author=%C4%90.%20Ugarkovi%C4%87&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol.&amp;volume=7&amp;pages=228-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25527837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larracuente</div>
               <div class="given-names">A.M.</div>
            </div>
          </div>
          <div class="year">2014</div>, <div class="article-title">The organization and evolution of the responder satellite in species of the <em>Drosophila melanogaster</em> group: dynamic evolution of a target of meiotic drive</div>, <div class="source">BMC Evol. Biol.</div> , <div class="volume">14</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20organization%20and%20evolution%20of%20the%20responder%20satellite%20in%20species%20of%20the%20Drosophila%20melanogaster%20group%3A%20dynamic%20evolution%20of%20a%20target%20of%20meiotic%20drive&amp;author=A.M.%20Larracuente&amp;publication_year=2014&amp;journal=BMC%20Evol.%20Biol.&amp;volume=14&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-014-0233-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-014-0233-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-014-0233-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24382122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schueler</div>
               <div class="given-names">M.G.</div>
            </div>, <div class="name string-name"><div class="surname">Dunn</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Bird</div> <div class="given-names">C.P.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Progressive proximal expansion of the primate X chromosome centromere</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">102</div>, <div class="fpage">10563</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progressive%20proximal%20expansion%20of%20the%20primate%20X%20chromosome%20centromere&amp;author=M.G.%20Schueler&amp;author=J.M.%20Dunn&amp;author=C.P.%20Bird&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=10563-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0503346102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0503346102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0503346102"> </span></div> </div></div></div></div></div><div content-id="DSV021C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lander</div>
               <div class="given-names">E.S.</div>
            </div>, <div class="name string-name"><div class="surname">Linton</div> <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="volume">409</div>, <div class="fpage">860</div>–<div class="lpage">921</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.S.%20Lander&amp;author=L.M.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Richmond</div>
               <div class="given-names">T.J.</div>
            </div>, <div class="name string-name"><div class="surname">Davey</div> <div class="given-names">C.A.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">The structure of DNA in the nucleosome core</div>, <div class="source">Nature</div> , <div class="volume">423</div>, <div class="fpage">145</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20DNA%20in%20the%20nucleosome%20core&amp;author=T.J.%20Richmond&amp;author=C.A.%20Davey&amp;publication_year=2003&amp;journal=Nature&amp;volume=423&amp;pages=145-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01595" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12736678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hasson</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Panchenko</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Salimian</div> <div class="given-names">K.J.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The octamer is the major form of CENP-A nucleosomes at human centromeres</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="volume">20</div>, <div class="fpage">687</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20octamer%20is%20the%20major%20form%20of%20CENP-A%20nucleosomes%20at%20human%20centromeres&amp;author=D.%20Hasson&amp;author=T.%20Panchenko&amp;author=K.J.%20Salimian&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=20&amp;pages=687-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Talbert</div> <div class="given-names">P.B.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The CentO satellite confers translational and rotational phasing on cenH3 nucleosomes in rice centromeres</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">E4875</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CentO%20satellite%20confers%20translational%20and%20rotational%20phasing%20on%20cenH3%20nucleosomes%20in%20rice%20centromeres&amp;author=T.%20Zhang&amp;author=P.B.%20Talbert&amp;author=W.%20Zhang&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=E4875-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1319548110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1319548110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1319548110"> </span></div> </div></div></div></div></div><div content-id="DSV021C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Levitsky</div>
               <div class="given-names">V.G.</div>
            </div>, <div class="name string-name"><div class="surname">Babenko</div> <div class="given-names">V.N.</div></div>, <div class="name string-name"><div class="surname">Vershinin</div> <div class="given-names">A.V.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">The roles of the monomer length and nucleotide context of plant tandem repeats in nucleosome positioning</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="volume">32</div>, <div class="fpage">115</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20roles%20of%20the%20monomer%20length%20and%20nucleotide%20context%20of%20plant%20tandem%20repeats%20in%20nucleosome%20positioning&amp;author=V.G.%20Levitsky&amp;author=V.N.%20Babenko&amp;author=A.V.%20Vershinin&amp;publication_year=2014&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=32&amp;pages=115-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2012.755796" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2012.755796" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2012.755796"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23384242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Meštrović</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Mravinac</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Juan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ugarković</div> <div class="given-names">Đ.</div></div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Comparative study of satellite sequences and phylogeny of five species from the genus <em>Palorus</em> (Insecta, Coleoptera)</div>, <div class="source">Genome</div> , <div class="volume">43</div>, <div class="fpage">776</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20study%20of%20satellite%20sequences%20and%20phylogeny%20of%20five%20species%20from%20the%20genus%20Palorus%20(Insecta%2C%20Coleoptera)&amp;author=N.%20Me%C5%A1trovi%C4%87&amp;author=B.%20Mravinac&amp;author=C.%20Juan&amp;author=%C4%90.%20Ugarkovi%C4%87&amp;author=M.%20Plohl&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=776-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-43-5-776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-43-5-776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-43-5-776"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11081967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barceló</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Gutiérrez</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Barjau</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Portugal</div> <div class="given-names">J.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">A theoretical perusal of the satellite DNA curvature in tenebrionid beetles</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="volume">16</div>, <div class="fpage">41</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20theoretical%20perusal%20of%20the%20satellite%20DNA%20curvature%20in%20tenebrionid%20beetles&amp;author=F.%20Barcel%C3%B3&amp;author=F.%20Guti%C3%A9rrez&amp;author=I.%20Barjau&amp;author=J.%20Portugal&amp;publication_year=1998&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=16&amp;pages=41-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1998.10508225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1998.10508225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1998.10508225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9745893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hayden</div>
               <div class="given-names">K.E.</div>
            </div>, <div class="name string-name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Composition and organization of active centromere sequences in complex genomes</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">324</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Composition%20and%20organization%20of%20active%20centromere%20sequences%20in%20complex%20genomes&amp;author=K.E.%20Hayden&amp;author=H.F.%20Willard&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=324" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22817545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">H.-R.R.</div>
            </div>, <div class="name string-name"><div class="surname">Neumann</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Macas</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">J.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Transcription and evolutionary dynamics of the centromeric satellite repeat CentO in rice</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">23</div>, <div class="fpage">2505</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20and%20evolutionary%20dynamics%20of%20the%20centromeric%20satellite%20repeat%20CentO%20in%20rice&amp;author=H.-R.R.%20Lee&amp;author=P.%20Neumann&amp;author=J.%20Macas&amp;author=J.%20Jiang&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=2505-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16987952" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Macas</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Neumann</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Novák</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">J.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Global sequence characterization of rice centromeric satellite based on oligomer frequency analysis in large-scale sequencing data</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">2101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20sequence%20characterization%20of%20rice%20centromeric%20satellite%20based%20on%20oligomer%20frequency%20analysis%20in%20large-scale%20sequencing%20data&amp;author=J.%20Macas&amp;author=P.%20Neumann&amp;author=P.%20Nov%C3%A1k&amp;author=J.%20Jiang&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20616383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ventura</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Catacchio</div> <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Sajjadian</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">The evolution of African great ape subtelomeric heterochromatin and the fusion of human chromosome 2</div>, <div class="source">Genome Res.</div> , <div class="volume">22</div>, <div class="fpage">1036</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20African%20great%20ape%20subtelomeric%20heterochromatin%20and%20the%20fusion%20of%20human%20chromosome%202&amp;author=M.%20Ventura&amp;author=C.R.%20Catacchio&amp;author=S.%20Sajjadian&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=1036-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.136556.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.136556.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.136556.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22419167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Macas</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Navrátilová</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Koblížková</div> <div class="given-names">A.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Sequence homogenization and chromosomal localization of VicTR-B satellites differ between closely related <em>Vicia</em> species</div>, <div class="source">Chromosoma</div> , <div class="volume">115</div>, <div class="fpage">437</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20homogenization%20and%20chromosomal%20localization%20of%20VicTR-B%20satellites%20differ%20between%20closely%20related%20Vicia%20species&amp;author=J.%20Macas&amp;author=A.%20Navr%C3%A1tilov%C3%A1&amp;author=A.%20Kobl%C3%AD%C5%BEkov%C3%A1&amp;publication_year=2006&amp;journal=Chromosoma&amp;volume=115&amp;pages=437-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00412-006-0070-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00412-006-0070-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00412-006-0070-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16788823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Žinić</div>
               <div class="given-names">S.D.</div>
            </div>, <div class="name string-name"><div class="surname">Ugarković</div> <div class="given-names">Đ.</div></div>, <div class="name string-name"><div class="surname">Cornudella</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">A novel interspersed type of organization of satellite DNAs in <em>Tribolium madens</em> heterochromatin</div>, <div class="source">Chromosome Res.</div> , <div class="volume">8</div>, <div class="fpage">201</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20interspersed%20type%20of%20organization%20of%20satellite%20DNAs%20in%20Tribolium%20madens%20heterochromatin&amp;author=S.D.%20%C5%BDini%C4%87&amp;author=%C4%90.%20Ugarkovi%C4%87&amp;author=L.%20Cornudella&amp;author=M.%20Plohl&amp;publication_year=2000&amp;journal=Chromosome%20Res.&amp;volume=8&amp;pages=201-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1009244711527" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1009244711527" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1009244711527"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10841047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alkan</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Sahinalp</div> <div class="given-names">S.C.</div></div>, <div class="name string-name"><div class="surname">Tüzün</div> <div class="given-names">E.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">The role of unequal crossover in alpha-satellite DNA evolution: a computational analysis</div>, <div class="source">J. Comput. Biol.</div> , <div class="volume">11</div>, <div class="fpage">933</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20unequal%20crossover%20in%20alpha-satellite%20DNA%20evolution%3A%20a%20computational%20analysis&amp;author=C.%20Alkan&amp;author=E.E.%20Eichler&amp;author=J.A.%20Bailey&amp;author=S.C.%20Sahinalp&amp;author=E.%20T%C3%BCz%C3%BCn&amp;publication_year=2004&amp;journal=J.%20Comput.%20Biol.&amp;volume=11&amp;pages=933-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2004.11.933" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2004.11.933" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.2004.11.933"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15700410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharma</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Wolfgruber</div> <div class="given-names">T.K.</div></div>, <div class="name string-name"><div class="surname">Presting</div> <div class="given-names">G.G.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Tandem repeats derived from centromeric retrotransposons</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">142</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20derived%20from%20centromeric%20retrotransposons&amp;author=A.%20Sharma&amp;author=T.K.%20Wolfgruber&amp;author=G.G.%20Presting&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=142" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-142"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23452340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Navrátilová</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Koblízková</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Macas</div> <div class="given-names">J.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Survey of extrachromosomal circular DNA derived from plant satellite repeats</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">8</div>, <div class="fpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20of%20extrachromosomal%20circular%20DNA%20derived%20from%20plant%20satellite%20repeats&amp;author=A.%20Navr%C3%A1tilov%C3%A1&amp;author=A.%20Kobl%C3%ADzkov%C3%A1&amp;author=J.%20Macas&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol.&amp;volume=8&amp;pages=90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-90" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-90" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-90"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18721471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Malik</div>
               <div class="given-names">H.S.</div>
            </div>, <div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Major evolutionary transitions in centromere complexity</div>, <div class="source">Cell</div> , <div class="volume">138</div>, <div class="fpage">1067</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Major%20evolutionary%20transitions%20in%20centromere%20complexity&amp;author=H.S.%20Malik&amp;author=S.%20Henikoff&amp;publication_year=2009&amp;journal=Cell&amp;volume=138&amp;pages=1067-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.08.036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.08.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.08.036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19766562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ferree</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Barbash</div> <div class="given-names">D.A.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Species-specific heterochromatin prevents mitotic chromosome segregation to cause hybrid lethality in <em>Drosophila</em></div>, <div class="source">PLoS Biol.</div> , <div class="volume">7</div>, <div class="fpage">e1000234</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species-specific%20heterochromatin%20prevents%20mitotic%20chromosome%20segregation%20to%20cause%20hybrid%20lethality%20in%20Drosophila&amp;author=P.M.%20Ferree&amp;author=D.A.%20Barbash&amp;publication_year=2009&amp;journal=PLoS%20Biol.&amp;volume=7&amp;pages=e1000234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1000234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1000234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1000234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19859525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bayes</div>
               <div class="given-names">J.J.</div>
            </div>, <div class="name string-name"><div class="surname">Malik</div> <div class="given-names">H.S.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Altered heterochromatin binding by a hybrid sterility protein in <em>Drosophila</em> sibling species</div>, <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">1538</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20heterochromatin%20binding%20by%20a%20hybrid%20sterility%20protein%20in%20Drosophila%20sibling%20species&amp;author=J.J.%20Bayes&amp;author=H.S.%20Malik&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1538-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19933102" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV021C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wei</div>
               <div class="given-names">K.H.-C.</div>
            </div>, <div class="name string-name"><div class="surname">Grenier</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Barbash</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">A.G.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Correlated variation and population differentiation in satellite DNA abundance among lines of <em>Drosophila melanogaster</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">18793</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlated%20variation%20and%20population%20differentiation%20in%20satellite%20DNA%20abundance%20among%20lines%20of%20Drosophila%20melanogaster&amp;author=K.H.-C.%20Wei&amp;author=J.K.%20Grenier&amp;author=D.A.%20Barbash&amp;author=A.G.%20Clark&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=18793-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1421951112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1421951112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1421951112"> </span></div> </div></div></div></div></div><div content-id="DSV021C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV021C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lemos</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Branco</div> <div class="given-names">A.T.</div></div>, <div class="name string-name"><div class="surname">Hartl</div> <div class="given-names">D.L.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Epigenetic effects of polymorphic Y chromosomes modulate chromatin components, immune response, and sexual conflict</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">15826</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20effects%20of%20polymorphic%20Y%20chromosomes%20modulate%20chromatin%20components%2C%20immune%20response%2C%20and%20sexual%20conflict&amp;author=B.%20Lemos&amp;author=A.T.%20Branco&amp;author=D.L.%20Hartl&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=15826-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1010383107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1010383107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1010383107"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462399&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv021/1/dsv021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=2Z3v8GXXOjlirATviRsUCP6c7PZorKvaG99XqyNL1-u94DGfve5ZV2xlWuP7jwlCf5IRdjL9dH2jTfiTcf-iio3eBih9y~oL8tKTXz~2tc5yCC9VkV8FfMGbg9QijOLHqEMXbBgcTxqkq4diw9GYeYaA0Ybp5evKLTA7ThSsZFg430h2VKbeG3P3PR~r4E3Jum0JA133J30k2HL38ThQSJzM5hlP0GuXVM-NgsrwhlaCy~UO0my~PuYUUGymd~Sm80WHESQxfwaJgtbt58ue6~TxxG4gcao~zP~0hG8xZF3M1dCwFLxZlCpdsAq9mvvRBssJ0GQpmzYJg9PHp3lzZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">411</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">15</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462399"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f387%2f2462399">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (15)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F387%2F2462399" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462399'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide analysis of tandem repeats in Tribolium castaneum genome reveals abundant and highly dynamic tandem repeat families with satellite DNA features in euchromatic chromosomal arms" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f387%2f2462399">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

