

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/6/369/484698","@type":"ScholarlyArticle","name":"Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis","datePublished":"2010-11-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2010-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq025","keywords":["focused microarray","Takayasu arteritis","spearman","RNA diagnostics","transcriptome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/6/369/484698","author":[{"name":"Okuzaki, Daisuke","affiliation":"1  Department of Molecular Genetics ,  Research Institute for Microbial Diseases ,  Suita, Osaka ,  Japan    2  DNA-Chip Development Center for Infectious Diseases ,  Research Institute for Microbial Diseases ,  Suita, Osaka ,  Japan","@type":"Person"},{"name":"Fukushima, Tatsunobu","affiliation":"3  Mitsubishi Rayon Co. Ltd, 1-6-41 Kounann, Minato-ku, Tokyo 108-8506 ,  Japan","@type":"Person"},{"name":"Tougan, Takahiro","affiliation":"2  DNA-Chip Development Center for Infectious Diseases ,  Research Institute for Microbial Diseases ,  Suita, Osaka ,  Japan","@type":"Person"},{"name":"Ishii, Tomonori","affiliation":"4  Rheumatology and Hematology ,  Tohoku University Hospital ,  1-1 Seiryo, Aoba-ku, Sendai 980-8574 ,  Japan","@type":"Person"},{"name":"Kobayashi, Shigeto","affiliation":"5  Juntendo Koshigaya Hospital ,  560 Fukuroyama, Koshigaya, Saitama 343-0032 ,  Japan","@type":"Person"},{"name":"Yoshizaki, Kazuyuki","affiliation":"3  Mitsubishi Rayon Co. Ltd, 1-6-41 Kounann, Minato-ku, Tokyo 108-8506 ,  Japan","@type":"Person"},{"name":"Akita, Takashi","affiliation":"6  Center for Advanced Science and Innovation ,  Osaka University ,  3-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Nojima, Hiroshi","affiliation":"1  Department of Molecular Genetics ,  Research Institute for Microbial Diseases ,  Suita, Osaka ,  Japan    2  DNA-Chip Development Center for Infectious Diseases ,  Research Institute for Microbial Diseases ,  Suita, Osaka ,  Japan","@type":"Person"}],"description":"Abstract.  Expression profiling of target genes in patient blood is a powerful tool for RNA diagnosis. Here, we describe Genopal™, a novel platform ideal for ef","pageStart":"369","pageEnd":"379","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis" />
    <meta property="og:description" content="Abstract.  Expression profiling of target genes in patient blood is a powerful tool for RNA diagnosis. Here, we describe Genopal™, a novel platform ideal for ef" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/6/369/484698" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-11-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Okuzaki, Daisuke" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Research Institute for Microbial Diseases, Suita, Osaka, Japan" /><meta name="citation_author_institution" content="DNA-Chip Development Center for Infectious Diseases, Research Institute for Microbial Diseases, Suita, Osaka, Japan" /><meta name="citation_author" content="Fukushima, Tatsunobu" /><meta name="citation_author_institution" content="Mitsubishi Rayon Co. Ltd, 1-6-41 Kounann, Minato-ku, Tokyo 108-8506, Japan" /><meta name="citation_author" content="Tougan, Takahiro" /><meta name="citation_author_institution" content="DNA-Chip Development Center for Infectious Diseases, Research Institute for Microbial Diseases, Suita, Osaka, Japan" /><meta name="citation_author" content="Ishii, Tomonori" /><meta name="citation_author_institution" content="Rheumatology and Hematology, Tohoku University Hospital, 1-1 Seiryo, Aoba-ku, Sendai 980-8574, Japan" /><meta name="citation_author" content="Kobayashi, Shigeto" /><meta name="citation_author_institution" content="Juntendo Koshigaya Hospital, 560 Fukuroyama, Koshigaya, Saitama 343-0032, Japan" /><meta name="citation_author" content="Yoshizaki, Kazuyuki" /><meta name="citation_author_institution" content="Mitsubishi Rayon Co. Ltd, 1-6-41 Kounann, Minato-ku, Tokyo 108-8506, Japan" /><meta name="citation_author" content="Akita, Takashi" /><meta name="citation_author_institution" content="Center for Advanced Science and Innovation, Osaka University, 3-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Nojima, Hiroshi" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Research Institute for Microbial Diseases, Suita, Osaka, Japan" /><meta name="citation_author_institution" content="DNA-Chip Development Center for Infectious Diseases, Research Institute for Microbial Diseases, Suita, Osaka, Japan" /><meta name="citation_title" content="Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis" /><meta name="citation_firstpage" content="369" /><meta name="citation_lastpage" content="379" /><meta name="citation_doi" content="10.1093/dnares/dsq025" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2010/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Focused microarray analysis; Methods;  citation_year=2003;  citation_volume=31;  citation_pages=306-16; " /><meta name="citation_reference" content="citation_title=Development of a focused microarray to assess human embryonic stem cell differentiation; Stem Cells Dev.;  citation_year=2005;  citation_volume=14;  citation_pages=270-84; " /><meta name="citation_reference" content="citation_title=A focused microarray to assess dopaminergic and glial cell differentiation from fetal tissue or embryonic stem cells; Stem Cells;  citation_year=2006;  citation_volume=24;  citation_pages=865-75; " /><meta name="citation_reference" content="citation_title=A focused microarray approach to functional glycomics: transcriptional regulation of the glycome; Glycobiology;  citation_year=2006;  citation_volume=16;  citation_pages=117-31; " /><meta name="citation_reference" content="citation_title=Focused microarray analysis of glyco-gene expression in human glioblastomas; J. Neurochem.;  citation_year=2007;  citation_volume=103;  citation_pages=14-24; " /><meta name="citation_reference" content="citation_title=A focused microarray to study human mitochondrial and nuclear gene expression; Biol. Res. Nurs.;  citation_year=2008;  citation_volume=9;  citation_pages=272-9; " /><meta name="citation_reference" content="citation_title=Evaluation of estrogenic activity of phthalate esters by gene expression profiling using a focused microarray (EstrArray); Environ. Toxicol. Chem.;  citation_year=2008;  citation_volume=27;  citation_pages=1416-25; " /><meta name="citation_reference" content="citation_title=Generation of a non-small cell lung cancer transcriptome microarray; BMC Med. Genomics;  citation_year=2008;  citation_volume=1;  citation_pages=20" /><meta name="citation_reference" content="citation_title=Overview of late outcome of medical and surgical treatment for Takayasu arteritis; Circulation;  citation_year=2008;  citation_volume=18;  citation_pages=2738-47; " /><meta name="citation_reference" content="citation_title=Fever of unknown origin; Semin. Nucl. Med.;  citation_year=2009;  citation_volume=39;  citation_pages=81-7; " /><meta name="citation_reference" content="citation_title=Isolation and expression profiling of genes upregulated in the peripheral blood cells of systemic lupus erythematosus patients; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=429-39; " /><meta name="citation_reference" content="citation_title=Isolation and expression profiling of genes upregulated in bone marrow-derived mononuclear cells of rheumatoid arthritis patients; DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=169-83; " /><meta name="citation_reference" content="citation_title=Expression profiling of PBMC-based diagnostic gene markers isolated from vasculitis patients; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=253-65; " /><meta name="citation_reference" content="citation_title=Focused microarray analysis of peripheral mononuclear blood cells from Churg-Strauss syndrome patients; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=103-14; " /><meta name="citation_reference" content="citation_title=Direct multiplexed measurement of gene expression with color-coded probe pairs; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=317-25; " /><meta name="citation_reference" content="citation_title=Marked change in microRNA expression during neuronal differentiation of human teratocarcinoma NTera2D1 and mouse embryonal carcinoma P19 cells; Biochem. Biophys. Res. Commun.;  citation_year=2007;  citation_volume=362;  citation_pages=360-7; " /><meta name="citation_reference" content="citation_title=miR-302b maintains “stemness” of human embryonal carcinoma cells by post-transcriptional regulation of Cyclin D2 expression; Biochem. Biophys. Res. Commun.;  citation_year=2008;  citation_volume=377;  citation_pages=434-40; " /><meta name="citation_reference" content="citation_title=Bitter gourd suppresses lipopolysaccharide-induced inflammatory responses; J. Agric. Food. Chem.;  citation_year=2008;  citation_volume=56;  citation_pages=4004-11; " /><meta name="citation_reference" content="citation_title=Gene-expression profiles of peripheral blood mononuclear cell subpopulations in acute graft-vs-host disease following cord blood transplantation; Exp. Hematol.;  citation_year=2008;  citation_volume=36;  citation_pages=1760-70; " /><meta name="citation_reference" content="citation_title=Characteristic gene expression in stromal cells of gastric cancers among atomic-bomb survivors; Int. J. Cancer;  citation_year=2009;  citation_volume=124;  citation_pages=1112-21; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/6/369/1310377/dsq025.pdf" /><meta name="description" content="Abstract.  Expression profiling of target genes in patient blood is a powerful tool for RNA diagnosis. Here, we describe Genopal™, a novel platform ideal for ef" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/6/369/484698" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/6/369/484698" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/6/369/484698");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq025';
    </script>
<title>
	Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484698" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd88PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPa3V6YWtpLCBEYWlzdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgR2VuZXRpY3MsIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgTWljcm9iaWFsIERpc2Vhc2VzLCBTdWl0YSwgT3Nha2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRE5BLUNoaXAgRGV2ZWxvcG1lbnQgQ2VudGVyIGZvciBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlcywgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3VzaGltYSwgVGF0c3Vub2J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWl0c3ViaXNoaSBSYXlvbiBDby4gTHRkLCAxLTYtNDEgS291bmFubiwgTWluYXRvLWt1LCBUb2t5byAxMDgtODUwNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb3VnYW4sIFRha2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRE5BLUNoaXAgRGV2ZWxvcG1lbnQgQ2VudGVyIGZvciBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlcywgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklzaGlpLCBUb21vbm9yaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJoZXVtYXRvbG9neSBhbmQgSGVtYXRvbG9neSwgVG9ob2t1IFVuaXZlcnNpdHkgSG9zcGl0YWwsIDEtMSBTZWlyeW8sIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2JheWFzaGksIFNoaWdldG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKdW50ZW5kbyBLb3NoaWdheWEgSG9zcGl0YWwsIDU2MCBGdWt1cm95YW1hLCBLb3NoaWdheWEsIFNhaXRhbWEgMzQzLTAwMzIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWW9zaGl6YWtpLCBLYXp1eXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pdHN1YmlzaGkgUmF5b24gQ28uIEx0ZCwgMS02LTQxIEtvdW5hbm4sIE1pbmF0by1rdSwgVG9reW8gMTA4LTg1MDYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWtpdGEsIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIEFkdmFuY2VkIFNjaWVuY2UgYW5kIElubm92YXRpb24sIE9zYWthIFVuaXZlcnNpdHksIDMtMSBZYW1hZGFva2EsIFN1aXRhLCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb2ppbWEsIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBHZW5ldGljcywgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciBNaWNyb2JpYWwgRGlzZWFzZXMsIFN1aXRhLCBPc2FrYSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJETkEtQ2hpcCBEZXZlbG9wbWVudCBDZW50ZXIgZm9yIEluZmVjdGlvdXMgRGlzZWFzZXMsIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgTWljcm9iaWFsIERpc2Vhc2VzLCBTdWl0YSwgT3Nha2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vcGFs4oSiOiBBIE5vdmVsIEhvbGxvdyBGaWJyZSBBcnJheSBmb3IgRm9jdXNlZCBNaWNyb2FycmF5IEFuYWx5c2lzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzY5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3EwMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMC8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvY3VzZWQgbWljcm9hcnJheSBhbmFseXNpczsgTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTMwNi0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYSBmb2N1c2VkIG1pY3JvYXJyYXkgdG8gYXNzZXNzIGh1bWFuIGVtYnJ5b25pYyBzdGVtIGNlbGwgZGlmZmVyZW50aWF0aW9uOyBTdGVtIENlbGxzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNzAtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZm9jdXNlZCBtaWNyb2FycmF5IHRvIGFzc2VzcyBkb3BhbWluZXJnaWMgYW5kIGdsaWFsIGNlbGwgZGlmZmVyZW50aWF0aW9uIGZyb20gZmV0YWwgdGlzc3VlIG9yIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBTdGVtIENlbGxzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9ODY1LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZvY3VzZWQgbWljcm9hcnJheSBhcHByb2FjaCB0byBmdW5jdGlvbmFsIGdseWNvbWljczogdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gb2YgdGhlIGdseWNvbWU7IEdseWNvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExNy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9jdXNlZCBtaWNyb2FycmF5IGFuYWx5c2lzIG9mIGdseWNvLWdlbmUgZXhwcmVzc2lvbiBpbiBodW1hbiBnbGlvYmxhc3RvbWFzOyBKLiBOZXVyb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTE0LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZvY3VzZWQgbWljcm9hcnJheSB0byBzdHVkeSBodW1hbiBtaXRvY2hvbmRyaWFsIGFuZCBudWNsZWFyIGdlbmUgZXhwcmVzc2lvbjsgQmlvbC4gUmVzLiBOdXJzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjcyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2YgZXN0cm9nZW5pYyBhY3Rpdml0eSBvZiBwaHRoYWxhdGUgZXN0ZXJzIGJ5IGdlbmUgZXhwcmVzc2lvbiBwcm9maWxpbmcgdXNpbmcgYSBmb2N1c2VkIG1pY3JvYXJyYXkgKEVzdHJBcnJheSk7IEVudmlyb24uIFRveGljb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTQxNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBvZiBhIG5vbi1zbWFsbCBjZWxsIGx1bmcgY2FuY2VyIHRyYW5zY3JpcHRvbWUgbWljcm9hcnJheTsgQk1DIE1lZC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3ZlcnZpZXcgb2YgbGF0ZSBvdXRjb21lIG9mIG1lZGljYWwgYW5kIHN1cmdpY2FsIHRyZWF0bWVudCBmb3IgVGFrYXlhc3UgYXJ0ZXJpdGlzOyBDaXJjdWxhdGlvbjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTI3MzgtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZldmVyIG9mIHVua25vd24gb3JpZ2luOyBTZW1pbi4gTnVjbC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTgxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgZXhwcmVzc2lvbiBwcm9maWxpbmcgb2YgZ2VuZXMgdXByZWd1bGF0ZWQgaW4gdGhlIHBlcmlwaGVyYWwgYmxvb2QgY2VsbHMgb2Ygc3lzdGVtaWMgbHVwdXMgZXJ5dGhlbWF0b3N1cyBwYXRpZW50czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00MjktMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgZXhwcmVzc2lvbiBwcm9maWxpbmcgb2YgZ2VuZXMgdXByZWd1bGF0ZWQgaW4gYm9uZSBtYXJyb3ctZGVyaXZlZCBtb25vbnVjbGVhciBjZWxscyBvZiByaGV1bWF0b2lkIGFydGhyaXRpcyBwYXRpZW50czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNjktODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gcHJvZmlsaW5nIG9mIFBCTUMtYmFzZWQgZGlhZ25vc3RpYyBnZW5lIG1hcmtlcnMgaXNvbGF0ZWQgZnJvbSB2YXNjdWxpdGlzIHBhdGllbnRzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI1My02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9jdXNlZCBtaWNyb2FycmF5IGFuYWx5c2lzIG9mIHBlcmlwaGVyYWwgbW9ub251Y2xlYXIgYmxvb2QgY2VsbHMgZnJvbSBDaHVyZy1TdHJhdXNzIHN5bmRyb21lIHBhdGllbnRzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IG11bHRpcGxleGVkIG1lYXN1cmVtZW50IG9mIGdlbmUgZXhwcmVzc2lvbiB3aXRoIGNvbG9yLWNvZGVkIHByb2JlIHBhaXJzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzE3LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXJrZWQgY2hhbmdlIGluIG1pY3JvUk5BIGV4cHJlc3Npb24gZHVyaW5nIG5ldXJvbmFsIGRpZmZlcmVudGlhdGlvbiBvZiBodW1hbiB0ZXJhdG9jYXJjaW5vbWEgTlRlcmEyRDEgYW5kIG1vdXNlIGVtYnJ5b25hbCBjYXJjaW5vbWEgUDE5IGNlbGxzOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzYyOyAgY2l0YXRpb25fcGFnZXM9MzYwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1pUi0zMDJiIG1haW50YWlucyDigJxzdGVtbmVzc+KAnSBvZiBodW1hbiBlbWJyeW9uYWwgY2FyY2lub21hIGNlbGxzIGJ5IHBvc3QtdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gb2YgQ3ljbGluIEQyIGV4cHJlc3Npb247IEJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNzc7ICBjaXRhdGlvbl9wYWdlcz00MzQtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpdHRlciBnb3VyZCBzdXBwcmVzc2VzIGxpcG9wb2x5c2FjY2hhcmlkZS1pbmR1Y2VkIGluZmxhbW1hdG9yeSByZXNwb25zZXM7IEouIEFncmljLiBGb29kLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTQwMDQtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUtZXhwcmVzc2lvbiBwcm9maWxlcyBvZiBwZXJpcGhlcmFsIGJsb29kIG1vbm9udWNsZWFyIGNlbGwgc3VicG9wdWxhdGlvbnMgaW4gYWN1dGUgZ3JhZnQtdnMtaG9zdCBkaXNlYXNlIGZvbGxvd2luZyBjb3JkIGJsb29kIHRyYW5zcGxhbnRhdGlvbjsgRXhwLiBIZW1hdG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTE3NjAtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3RlcmlzdGljIGdlbmUgZXhwcmVzc2lvbiBpbiBzdHJvbWFsIGNlbGxzIG9mIGdhc3RyaWMgY2FuY2VycyBhbW9uZyBhdG9taWMtYm9tYiBzdXJ2aXZvcnM7IEludC4gSi4gQ2FuY2VyOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTExMTItMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy82LzM2OS8xMzEwMzc3L2RzcTAyNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRXhwcmVzc2lvbiBwcm9maWxpbmcgb2YgdGFyZ2V0IGdlbmVzIGluIHBhdGllbnQgYmxvb2QgaXMgYSBwb3dlcmZ1bCB0b29sIGZvciBSTkEgZGlhZ25vc2lzLiBIZXJlLCB3ZSBkZXNjcmliZSBHZW5vcGFs4oSiLCBhIG5vdmVsIHBsYXRmb3JtIGlkZWFsIGZvciBlZiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDg0Njk4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZAjaKOHf9MFh8oQb1/JsyDgLjXk5" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f369%2f484698" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f369%2f484698" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19064" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&amp;Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/6">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7364949">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7364949">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7364952">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7364952">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7364959">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7364959">2. Patients, Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365008">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365008">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365045">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365045">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365057">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365057">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365061">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365061">Authors’ contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365065">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365065">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365070">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365070">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365074">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365074">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/6/353/484440">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/6/381/484555">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daisuke Okuzaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Research Institute for Microbial Diseases</div>, <div class="addr-line">Suita, Osaka</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">DNA-Chip Development Center for Infectious Diseases</div>, <div class="insititution">Research Institute for Microbial Diseases</div>, <div class="addr-line">Suita, Osaka</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daisuke+Okuzaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okuzaki D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okuzaki D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daisuke Okuzaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tatsunobu Fukushima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Mitsubishi Rayon Co. Ltd, 1-6-41 Kounann, Minato-ku, Tokyo 108-8506</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tatsunobu+Fukushima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukushima T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukushima T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tatsunobu Fukushima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Tougan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">DNA-Chip Development Center for Infectious Diseases</div>, <div class="insititution">Research Institute for Microbial Diseases</div>, <div class="addr-line">Suita, Osaka</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Tougan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tougan T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tougan T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiro Tougan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomonori Ishii</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Rheumatology and Hematology</div>, <div class="insititution">Tohoku University Hospital</div>, <div class="addr-line">1-1 Seiryo, Aoba-ku, Sendai 980-8574</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomonori+Ishii">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishii T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishii T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomonori Ishii</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeto Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Juntendo Koshigaya Hospital</div>, <div class="addr-line">560 Fukuroyama, Koshigaya, Saitama 343-0032</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeto+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeto Kobayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuyuki Yoshizaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Mitsubishi Rayon Co. Ltd, 1-6-41 Kounann, Minato-ku, Tokyo 108-8506</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuyuki+Yoshizaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshizaki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshizaki K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuyuki Yoshizaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Akita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Center for Advanced Science and Innovation</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">3-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Akita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Akita T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Akita T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Akita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Nojima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Research Institute for Microbial Diseases</div>, <div class="addr-line">Suita, Osaka</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">DNA-Chip Development Center for Infectious Diseases</div>, <div class="insititution">Research Institute for Microbial Diseases</div>, <div class="addr-line">Suita, Osaka</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 6-6875-3980. Fax. <div class="fax">+81 6-6875-5192</div>. E-mail: <a href="mailto:snj-0212@biken.osaka-u.ac.jp" target="_blank">snj-0212@biken.osaka-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Nojima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nojima H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nojima H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Nojima</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These two authors equally contributed to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 369–379, <a href='https://doi.org/10.1093/dnares/dsq025'>https://doi.org/10.1093/dnares/dsq025</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 November 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 July 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 September 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484698" href="/dnaresearch/article-pdf/17/6/369/1310377/dsq025.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Daisuke Okuzaki, Tatsunobu Fukushima, Takahiro Tougan, Tomonori Ishii, Shigeto Kobayashi, Kazuyuki Yoshizaki, Takashi Akita, Hiroshi Nojima;  Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis, <em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 369–379, <a href='https://doi.org/10.1093/dnares/dsq025'>https://doi.org/10.1093/dnares/dsq025</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484698?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484698?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484698?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484698?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484698?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484698?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484698?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genopal%E2%84%A2%3A%20A%20Novel%20Hollow%20Fibre%20Array%20for%20Focused%20Microarray%20Analysis&amp;publicationDate=2010-11-08&amp;volumeNum=17&amp;issueNum=6&amp;author=Okuzaki%2C%20Daisuke%3B%20Fukushima%2C%20Tatsunobu&amp;startPage=369&amp;endPage=379&amp;contentId=10.1093%2Fdnares%2Fdsq025&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f369%2f484698" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7364949 id="7364949" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Expression profiling of target genes in patient blood is a powerful tool for RNA diagnosis. Here, we describe Genopal™, a novel platform ideal for efficient focused microarray analysis. Genopal™, which consists of gel-filled fibres, is advantageous for high-quality mass production via large-scale slicing of the Genopal™ block. We prepared two arrays, infectant and autoimmunity, that provided highly reliable data in terms of repetitive scanning of the same and/or distinct microarrays. Moreover, we demonstrated that Genopal™ had sensitivity sufficient to yield signals in short hybridization times (0.5 h). Application of the autoimmunity array to blood samples allowed us to identify an expression pattern specific to Takayasu arteritis based on the Spearman rank correlation by comparing the reference profile with those of several autoimmune diseases and healthy volunteers (HVs). The comparison of these data with those obtained by other methods revealed that they exhibited similar expression profiles of many target genes. Taken together, these data demonstrate that Genopal™ is an advantageous platform for focused microarrays with regard to its low cost, rapid results and reliable quality.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;focused microarray&quot;">focused microarray</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Takayasu arteritis&quot;">Takayasu arteritis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="spearman">spearman</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA diagnostics&quot;">RNA diagnostics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a></div>                            <h2 scrollto-destination=7364952 id="7364952" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genome-wide expression profiling based on microarray technology is a powerful tool for both basic and clinical research focused on understanding genetic interactions and changes in gene pathways. It also facilitates the identification of genomic biomarkers that correlate with disease classification and with patient responses to drug treatment. For practical diagnostic purposes, however, the currently available general-purpose microarrays have significant drawbacks. Large volumes of data, such as from 40 000 genes or more, are not feasibly managed or analysed. Moreover, the high cost of the microarrays and the compound scanning tools has also hampered the clinical application of microarray systems. In contrast, customized, focused microarrays are advantageous for clinical use because they harbour a substantial, but not overwhelming, number of specific genes that allow researchers and clinicians to evaluate the state of target cells in a relatively inexpensive, rapid, and reliable way.<sup><a reveal-id="DSQ025C1" data-open="DSQ025C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Focused microarrays have been successfully applied to the analysis of human embryonic stem cell differentiation,<sup><a reveal-id="DSQ025C2" data-open="DSQ025C2" class="link link-ref link-reveal xref-bibr">2</a></sup> dopaminergic and glial cell differentiation,<sup><a reveal-id="DSQ025C3" data-open="DSQ025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> transcriptional regulation of the glycome,<sup><a reveal-id="DSQ025C4" data-open="DSQ025C4" class="link link-ref link-reveal xref-bibr">4</a></sup> glyco gene targets with therapeutic potential,<sup><a reveal-id="DSQ025C5" data-open="DSQ025C5" class="link link-ref link-reveal xref-bibr">5</a></sup> gene expression in the mitochondria,<sup><a reveal-id="DSQ025C6" data-open="DSQ025C6" class="link link-ref link-reveal xref-bibr">6</a></sup> oestrogenic activity of phthalate esters<sup><a reveal-id="DSQ025C7" data-open="DSQ025C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and the cell state of non-small cell lung cancer patients.<sup><a reveal-id="DSQ025C8" data-open="DSQ025C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Unfortunately, the focused microarrays currently available, which are based on the two-dimensional display of oligonucleotides on a glass-based platform, do not offer a substantial economic advantage, reliable quality or the possibility for large-scale production.</p><p>We report here the development and practical application of a novel platform for focused microarray, termed Genopal™. The performance of two types of focused microarrays were validated using fluorescently labelled antisense oligonucleotides selected from the genomes of infectious bacteria and viruses for the former, and complementary RNAs (cRNAs) derived from peripheral blood mononuclear cells (PBMCs) of Takayasu arteritis (TA) patients for the latter. TA is a rare autoimmune disease of unknown origin that predominantly affects women and leads to inflammation of the large arteries, resulting in various types of aortoarterial stenosis/occlusion or dilatation.<sup><a reveal-id="DSQ025C9" data-open="DSQ025C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSQ025C10" data-open="DSQ025C10" class="link link-ref link-reveal xref-bibr">10</a></sup>. Here, microarray analysis based on the Spearman rank correlation identified an expression pattern specific to TA, which suggests that the Genopal™ is useful for the diagnosis of TA.</p>                            <h2 scrollto-destination=7364959 id="7364959" class="section-title" data-legacy-id=s2>2. Patients, Materials and Methods</h2>
                            <h3 scrollto-destination=7364962 id="7364962" class="section-title" data-legacy-id=s2a>2.1. Human subjects and ethical considerations</h3>
<p>All systemic vasculitis patients used in this study were diagnosed according to a previously documented proposal (the ACR criteria and the CHCC criteria). This study was reviewed and approved by the Internal Review Board of the Research Institute for Microbial Diseases, Osaka University. Accordingly, written informed consent was obtained from all participants before their PBMCs were obtained. Serum samples were consecutively obtained regardless of the patient's symptom, active or inactive phase.</p>                            <h3 scrollto-destination=7364967 id="7364967" class="section-title" data-legacy-id=s2b>2.2. Production of hollow fibre arrays</h3>
<p>Genopal™ consists of hollow fibres impregnated with gel that firmly adheres to the inner walls of the fibres via the introduction of comb polymers (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>A). Detailed technical aspects of Genopal™ are described in the following patents: US7122378, US20040234114, US20040258897, US20050063877 and EP1627839. Briefly, methyl methacrylate (MMA, 100 parts), glycidyl methacrylate (GMA, 100 parts) and azobisisobutyronitrile (AIBN, 0.5 parts) were added dropwise to methyl ethyl ketone (MEK, 50 parts) at 80°C in a nitrogen gas stream (50 ml/min) for 3 h. AIBN (0.1 parts) and MEK (70 parts) were then added, and the mixture was allowed to stand for 1 h. Subsequently, AIBN (0.1 parts) and MEK (10 parts) were added, and the resulting product was allowed to stand for 3 h before the addition of MEK (50 parts). Thereafter, methylhydroquinone (0.5 parts), triphenylphosphine (2.5 parts) and acrylic acid (99 mol % GMA) were added. During these procedures, reactions were aerated (100 ml/min) and allowed to proceed for 30 h. Comb polymers with vinyl groups added to their side-chain termini were obtained. These copolymers contained 25 mol % vinyl groups (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>B). </p>                        <a id="7364972" scrollto-destination="7364972"></a>
<div id="DSQ025F1" data-id="DSQ025F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02501.jpeg?Expires=2147483647&Signature=F9s~HXMfNe3jUMZI~q9HFXH7dVKXR1AfodB~SaT4Ca2J0b~L9fLUHvMAPJtWhDlPjjJn9fr~CJvkqIpu6hEF1xWZmqOewXVHMRn5IzMqK3kAk3VuhgDKYABqYfr9~9jUoRFYfjei-NxgPP9ml7ze9MYdWB4aIc0KNYI~LFQfLv~rnA4QDopS8T3gzZb2YNAvJsh7w9H6CnCOzvWvIeNgvVY0DELFiq~2xFy0pm1E8nlR1UuFKt0nbSaXxQVtjWCy27asSZTmAx-4pHBoECViHW9VaOR43~RlwTjCxQsV0iaf0a7hlBwgUwsAcSfS2ytYRb6v2fiDo1nc~ImH6a9fRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structure of Genopal™ and its mass production application via slicing of the Genopal™ block. (A) Schematic representation of the Genopal™ microarray, which is composed of plastic hollow fibres. Oligonucleotide DNA capture probes were attached to a hydrophilic gel within the three-dimensional space of each hollow fibre. (B) The chemical structure of the comb polymers containing polymer-forming functional groups on their side chains. These polymers were inserted into the hollow fibres. (C) Comb polymers with poly HEMA added to their side chains firmly adhered to the inner walls of the hollow fibres. (D) Schematic representation of a Genopal™ block: the hollow fibres were bundled in an orderly arrangement, hardened with resin to form a Genopal™ block and sliced into thin microarrays." path-from-xml="dsq02501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7364972/dsq02501.jpeg" path-from-xml="dsq02501.jpeg" target="_blank">View large</a><a section="7364972" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02501.jpeg?Expires=2147483647&Signature=k-A2cYlCuaHu525eWs33oCHK-qVUIIWLYNwH0gya4ZmmIIMLa4OFMxUSVSEKtD5yyfWlTOTZrOrdhrMYwM~MO6fgT9OjsrOaWiUBVplfe1UOMolGS8Iagb6KcNundx6LufxeN2o1Yvj6kgcTMS5mO3w9LnY4GVN1URuo9wsOiywX8rk5E1FVcPe5yEEuE3tuSLY5Zib3SJ8wfFj9WP2U1Qzjf8TvK-BgOcJ4e--ZlhfmCmcfc5Jvsrsb8~-GqZHIawAXVuc8Rxc45vTTbrXVkW6J0I-AE2THCQOnFX23R~zv4p6tuAVy06VYnW2g1ZIdmzNgGZIiQIaXVx8KJRZa7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7364972&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structure of Genopal™ and its mass production application via slicing of the Genopal™ block. (A) Schematic representation of the Genopal™ microarray, which is composed of plastic hollow fibres. Oligonucleotide DNA capture probes were attached to a hydrophilic gel within the three-dimensional space of each hollow fibre. (B) The chemical structure of the comb polymers containing polymer-forming functional groups on their side chains. These polymers were inserted into the hollow fibres. (C) Comb polymers with poly HEMA added to their side chains firmly adhered to the inner walls of the hollow fibres. (D) Schematic representation of a Genopal™ block: the hollow fibres were bundled in an orderly arrangement, hardened with resin to form a Genopal™ block and sliced into thin microarrays.</p></div></div><div content-id="DSQ025F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02501.jpeg?Expires=2147483647&Signature=F9s~HXMfNe3jUMZI~q9HFXH7dVKXR1AfodB~SaT4Ca2J0b~L9fLUHvMAPJtWhDlPjjJn9fr~CJvkqIpu6hEF1xWZmqOewXVHMRn5IzMqK3kAk3VuhgDKYABqYfr9~9jUoRFYfjei-NxgPP9ml7ze9MYdWB4aIc0KNYI~LFQfLv~rnA4QDopS8T3gzZb2YNAvJsh7w9H6CnCOzvWvIeNgvVY0DELFiq~2xFy0pm1E8nlR1UuFKt0nbSaXxQVtjWCy27asSZTmAx-4pHBoECViHW9VaOR43~RlwTjCxQsV0iaf0a7hlBwgUwsAcSfS2ytYRb6v2fiDo1nc~ImH6a9fRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structure of Genopal™ and its mass production application via slicing of the Genopal™ block. (A) Schematic representation of the Genopal™ microarray, which is composed of plastic hollow fibres. Oligonucleotide DNA capture probes were attached to a hydrophilic gel within the three-dimensional space of each hollow fibre. (B) The chemical structure of the comb polymers containing polymer-forming functional groups on their side chains. These polymers were inserted into the hollow fibres. (C) Comb polymers with poly HEMA added to their side chains firmly adhered to the inner walls of the hollow fibres. (D) Schematic representation of a Genopal™ block: the hollow fibres were bundled in an orderly arrangement, hardened with resin to form a Genopal™ block and sliced into thin microarrays." path-from-xml="dsq02501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7364972/dsq02501.jpeg" path-from-xml="dsq02501.jpeg" target="_blank">View large</a><a section="7364972" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02501.jpeg?Expires=2147483647&Signature=k-A2cYlCuaHu525eWs33oCHK-qVUIIWLYNwH0gya4ZmmIIMLa4OFMxUSVSEKtD5yyfWlTOTZrOrdhrMYwM~MO6fgT9OjsrOaWiUBVplfe1UOMolGS8Iagb6KcNundx6LufxeN2o1Yvj6kgcTMS5mO3w9LnY4GVN1URuo9wsOiywX8rk5E1FVcPe5yEEuE3tuSLY5Zib3SJ8wfFj9WP2U1Qzjf8TvK-BgOcJ4e--ZlhfmCmcfc5Jvsrsb8~-GqZHIawAXVuc8Rxc45vTTbrXVkW6J0I-AE2THCQOnFX23R~zv4p6tuAVy06VYnW2g1ZIdmzNgGZIiQIaXVx8KJRZa7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7364972&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structure of Genopal™ and its mass production application via slicing of the Genopal™ block. (A) Schematic representation of the Genopal™ microarray, which is composed of plastic hollow fibres. Oligonucleotide DNA capture probes were attached to a hydrophilic gel within the three-dimensional space of each hollow fibre. (B) The chemical structure of the comb polymers containing polymer-forming functional groups on their side chains. These polymers were inserted into the hollow fibres. (C) Comb polymers with poly HEMA added to their side chains firmly adhered to the inner walls of the hollow fibres. (D) Schematic representation of a Genopal™ block: the hollow fibres were bundled in an orderly arrangement, hardened with resin to form a Genopal™ block and sliced into thin microarrays.</p></div></div><p>Two hundred and twenty-five hollow poly-MMA fibres (outer diameter, 300 µm; inner diameter, 200 µm; length, 60 cm; Mitsubishi Rayon Co. Ltd, Tokyo, Japan) were bundled, and comb polymers were introduced into the hollow portions. Part of the solution introduced by suction into the hollow portions was transferred to the trap tube, and the solvent remaining on the inner walls of the hollow portions was allowed to evaporate by air drying. Subsequently, two porous plates (thickness, 0.1 mm) comprising 49 pores each arranged in seven rows in both lengthwise and breadthwise directions, with a pore diameter of 0.32 mm and centre-to-centre distances between neighbouring pores of 0.42 mm, were stacked. Forty-nine hollow fibres of poly-MMA were allowed to pass through each pore of these two porous plates. The distance between the two porous plates was 50 mm, and both ends were immobilized while the threads were stretched between them. The starting material for the resin was then made to flow into the hollow fibre array and allowed to harden. Polyurethane resin adhesives (Nippolan 4276, Coronate 4403; by Nihon Polyurethane Industry Co. Ltd, Shunan-shi, Japan) were used. The porous plates were removed after the resin hardened, producing the resin block containing hollow fibres.</p><p>In the final step of the oligonucleotide synthesis, the reaction was performed using Aminolink II (Applied Biosystems, Foster City, CA, USA) to synthesize an oligonucleotide with aminated termini. The resulting GCAT oligo with aminated termini (50 µl, 500 nmol/ml), GMA (5 µl) and dimethylformamide (5 µl) were mixed, and the mixture was allowed to react at 70°C for 2 h before the addition of 190 µl of water. Thus, we obtained a GCAT oligo with 100 nmol/ml methacrylate groups (MA-GCAT). Subsequently, we prepared a starting solution for a gel comprise the monomer and the initiator with the following mass ratios: 9 parts acrylamide, 1 part <em>N</em>,<em>N</em>′-methylenebisacrylamide, 0.1 part 2,2′-azobis(2-methylpropionamidine)dihydrochloride (V-50) and 90 parts water. MA-GCAT was then added to the above solution to a final concentration of 0.5 nmol/l. The hollow portions of the hollow fibres in the resin block were filled with this solution, the block was transferred into a humidified hermetically sealed glass vessel, and the content of the vessel was allowed to polymerize at 70°C for 3 h. Thus, a block of hollow fibre array in which the hollow portion was impregnated with gel was obtained. This block was sliced to a thickness of 500 µm in a direction perpendicular to the direction of fibres.</p>                            <h3 scrollto-destination=7364979 id="7364979" class="section-title" data-legacy-id=s2c>2.3. Preparation of infectant and autoimmunity arrays</h3>
<p>The infectant Genopal™ probe (iGp) array were comprise 60 nucleotides (nts) from the genome sequences of 40 types of infectious bacteria and viruses (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), taking into consideration the specificity, melting temperature, GC content (40–60%) and secondary structure predicted with ProbeQuest software (Dynacom Co., Chiba, Japan). The generated synthetic DNA oligonucleotides were installed onto a Genopal™ platform to keep them immobilized in the hydrophilic gel within the three-dimensional space of each hollow fibre (Mitsubishi Rayon Co. Ltd). To prepare autoimmunity Genopal™ probe (aGp) array, 216 genes were selected based on their relevance to autoimmune diseases (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2A and Table S2</a>), 65 nt Genopal™ probes with similar annealing temperatures were designed, and the oligonucleotides were synthesized and installed onto the Genopal™ platform. Here, we also considered the melting temperature, GC content, specificity, secondary structure and low complexity of the sequences, which were primarily located within 1000 bases of the 3′ end of mRNA sequences.</p>                            <h3 scrollto-destination=7364984 id="7364984" class="section-title" data-legacy-id=s2d>2.4. RNA isolation</h3>
<p>Total RNA from fresh PBMCs of patients and pooled healthy volunteers (HV pools) was isolated using the PAXgene Blood RNA Kit and RNeasy Mini Kit with the optimal on-column DNase digestion, according to the manufacturer's instructions (Qiagen KK, Hilden, Germany). The quality of the RNA samples used for the microarray analysis was examined using the RNA 6000 Nano LabChip Kit (p/n 5067-1511) on the Agilent 2100 Bioanalyzer (G2938C; Agilent Technologies Inc., Palo Alto, CA, USA).</p>                            <h3 scrollto-destination=7364987 id="7364987" class="section-title" data-legacy-id=s2e>2.5. Microarray analysis on Genopal™</h3>
<p>Hybridizations on the infectant and autoimmunity arrays were carried out in 150 µl of hybridization buffer (0.12 M Tris–HCl, pH 7.5, 0.12 M NaCl, 0.05% Tween-20 and the indicated amount of antisense-oligonucleotides or 1125 ng of cRNA) at 65°C overnight or for the indicated times (for time-course measurements; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1B</a>) in a Genopal™-specific hybridization chamber (Mitsubishi Rayon Co. Ltd). After hybridization, the Genopal™ arrays were washed twice in 0.12 M Tris–HCl (pH 7.5), 0.12 M NaCl and 0.05% Tween-20 at 65°C for 20 min, followed by washing in 0.12 M Tris–HCl (pH 7.5), 0.12 M NaCl at room temperature for 10 min. The Genopal™ array was then scanned, and the image was captured using a cooled charge-coupled device type Microarray Image Analyzer equipped with multibeam excitation technology (Yokogawa Electric Co., Tokyo, Japan). For infectant array data, Microsoft Excel was used to calculate the coefficient of variation (CV) between arrays, between scans and between sections. For autoimmunity array data, signal intensities were processed by Subio Basic Plug-in (v1.6; Subio Inc., Aichi, Japan) as follows. First, all values &lt;1 were replaced by 1 and log<sub>2</sub> transformed. Next, ratios against intensities of the HV pools were calculated. Processed signals from 13 TA samples were averaged by 216 genes to generate a reference TA profile. Finally, the Spearman rank correlation between each sample and the reference TA profile by R were calculated. The details of the microarray data have been deposited in the Gene Expression Omnibus (GEO; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo">www.ncbi.nlm.nih.gov/geo</a>) database (accession number GSE18247).</p>                            <h3 scrollto-destination=7364992 id="7364992" class="section-title" data-legacy-id=s2f>2.6. Data analysis of aGp array</h3>
<p>We obtained raw data by measuring the signal intensity with exposure time of 0.1, 1.0, 4.0, 40 and 120 s. Then, we calculated the values of average, median and standard deviation (SD) for 216 probe and 12 blank wells (background). In most cases, the subtracted value (median value of 216 probes minus median value of background) exceeded the SD value of background. Thus, we employed it as the expression value. If not, we employed the median value of background as the expression value.</p>                            <h3 scrollto-destination=7364998 id="7364998" class="section-title" data-legacy-id=s2g>2.7. Microarray analysis on Agilent's microarray</h3>
<p>Microarray analyses were performed as two-colour hybridizations as described previously.<sup><a reveal-id="DSQ025C11" data-open="DSQ025C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Total RNAs obtained from PBMCs were independently reverse transcribed using oligo-dT primers containing the T7 RNA polymerase promoter sequence to generate cDNAs, which were then subjected to <em>in vitro</em> transcription using T7 RNA polymerase to label the cRNAs with Cy3-CTP or Cy5-CTP (Amersham Pharmacia Biotech, Piscataway, NJ, USA) using a Fluorescent Linear Amplification Kit (Agilent Technologies Inc.). Purified Cy5-labelled cRNAs (825 ng) from TA patients were then mixed with an equal amount of reverse-colour Cy3-labelled cRNAs from HV pools. Hybridizations, washing, scanning and gene analysis with Agilent's Whole Human Genome Microarray 4 × 44K G4112F were conducted according to the manufacturer's protocol (Agilent Technologies Inc.). Agilent Feature Extraction software (v. 9.5.1) was used to assess spot quality and extract feature intensity statistics. The Subio Platform and the Subio Basic Plug-in (v1.6; Subio Inc.) were then used to calculate the log ratio of rProcessedSignal over gProcessedSignal (HV pools). The details of the microarray data have been deposited in the GEO (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo">www.ncbi.nlm.nih.gov/geo</a>) database (accession number GSE18245). Genopal™ and Agilent's Whole Human Genome Microarray 4 × 44K data reported in this paper have been also deposited as superSeries GSE18316.</p>                            <h3 scrollto-destination=7365003 id="7365003" class="section-title" data-legacy-id=s2h>2.8. Expression profiling using quantitative reverse transcription–polymerase chain reaction and nCounter™ analyses</h3>
<p>Assay-on-Demand TaqMan probes with relevant primers were used for quantitative reverse transcription–polymerase chain reaction (qPCR) analysis using the ABI PRISM 7900 according to the manufacturer's instructions (PE Applied Biosystems). Total RNA (500 ng) obtained using the PAXgene Blood RNA Kit was reverse transcribed with the High Capacity cDNA Archive Kit (ABI). PCR consisted of an initial denaturation (95°C, 10 min) followed by 40 cycles of denaturation (95°C, 15 s) and annealing/extension (60°C, 1 min). A standard curve was generated from the amplification data for each primer using a dilution series of PBMC RNA as the template. Fold-change values were normalized to GAPDH expression levels using the standard curve method according to the manufacturer's protocol. Measurements performed using nCounter™ were conducted by NanoString Technologies Inc<em>.</em> (Seattle, WA, USA).</p>                            <h2 scrollto-destination=7365008 id="7365008" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7365010 id="7365010" class="section-title" data-legacy-id=s3a>3.1. Basic design of the Genopal™ hollow fibre array</h3>
<p>To provide a useful, focused microarray system for RNA diagnostics, we developed a novel platform termed Genopal™, which is comprise a hollow fibre array (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>). Different from typical glass slide DNA microarrays, which harbour two-dimensionally captured oligonucleotide probes attached to a solid surface, such as glass, Genopal™ consists of hollow gel-filled fibres in which oligonucleotides are uniquely embedded in a three-dimensionally arranged manner (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>A). The gel, which consists of copolymers containing 25 mol % vinyl groups (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>B), are firmly adhered to the inner walls of the hollow fibres via the introduction of comb polymers (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>C). Moreover, thin slices obtained by bundling these hollow fibres and then slicing the bundle exhibited sufficient adhesion between the gel and the inner walls of the hollow fibres (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>D). A full description of this platform is available online at <a class="link link-uri" href="http://www.mrc.co.jp/genome/e/index.html">http://www.mrc.co.jp/genome/e/index.html</a>.</p><p>Genopal™ offers a number of advantages in terms of its low-cost, ease of storage, speed and convenience compared with other microarray devices. First, it generates hybridization signals that are suitable for high-precision analysis. The oligonucleotides embedded in Genopal™ are maintained in wet conditions throughout all steps of the experiment, even during scanning, thereby allowing for rapid and easy microarray assay. Second, it produces a large number of microarrays with comparable quality, because the hollow fibres bundled into a single resin block are sliced perpendicular to the longitudinal direction of the fibres; this property is particularly advantageous for mass production at low cost when commercialized as a clinical tool for RNA diagnostics, reducing the cost down to nearly 10% of the conventional microarrays. Third, it is durable, robust and portable, as the sliced and packaged Genopal™ microarrays are maintained under wet conditions. The comb polymer, in particular, prevents the gel from detaching from the inner walls at the time of slicing or during procedures such as hybridization and washing.</p>                            <h3 scrollto-destination=7365014 id="7365014" class="section-title" data-legacy-id=s3b>3.2. Preparation and testing of an infectant array</h3>
<p>To examine the quality of the Genopal™ platform, a focused DNA microarray termed the ‘infectant array’ was designed based on 40 genes (probes) from infectious bacteria and viruses. To design the microarray probes, oligonucleotides 60 nts in length were selected from the genome sequence of each infectant, taking into consideration the melting temperature, GC content, specificity and secondary structure (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The probes were then chemically synthesized and integrated into four independent sections of the Genopal™ gel to test reproducibility of the assay (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1A</a>). Henceforth, we will refer to each 60 nt oligonucleotide as an iGp and each Genopal™ array with these probes as an iGp array. In parallel, antisense DNA (60 nts) was synthesized for use as target DNA corresponding to each iGp and labelled with a fluorescent dye (Cy5) for efficient detection when hybridized with the infectant array (Fig. <a reveal-id="DSQ025F2" data-open="DSQ025F2" class="link link-reveal link-table xref-fig">2</a>A). </p>                        <a id="7365016" scrollto-destination="7365016"></a>
<div id="DSQ025F2" data-id="DSQ025F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02502.jpeg?Expires=2147483647&Signature=X0zzxKOIYaQFrN0UK4WUSs6obKqu~0yD3ZjCB9j-q93sloOZlqMNQMrFDH74QJW8NH5mm-4voweLZ1g5BcvxQdhFYEZ1fp7NK-c1IfGhpAsbUXkOomfk7P9qe2v1MkfLnqRYXoGVAN2CSO5zXUF5bBPkyljZr5Z20RV9wDMvtbofeF1k1C2u4R0HG9-4~l35Y1ac-A1S83v8LhtW1mpY0FXwusrsOAiI7lRNql8dOziVwHa8xPLgJv-OuAZzE0y1EsY5mRN3x193dNrjHMq2u6LwDyFDABO4eFD-v0ox1WAkPofRHc8QeGndeqhMckQNIhwumysXHScyDrHDmpjK-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="iGp array set-up and data analysis. (A) Experimental design for the use of the iGp array. Each iGp array harboured four identical sets of 40 iGps that contained oligonucleotides (60 nt) corresponding to various infectious bacteria and viruses (Supplementary Table S1). Cy5-labelled antisense DNA (60 nt) was used as a target sample that emitted fluorescent signals when hybridized with the iGps. (B) Scatter plot of averaged signal intensities (horizontal) versus the CV of overall variation (vertical). Four replicate samples were independently prepared for hybridization to four iGp arrays. Each array was scanned four times to obtain 64 measured values for each probe in total. The average signal intensities and CVs were calculated from these 64 values. For each 40 iGps (black diamonds), the CV of the overall 64 average signal intensities (%) was plotted against the original mean value of the calculated average signal intensities. The 75th, 50th and 25th percentile of the CVs (10.1, 11.6 and 11.9%) on the vertical axis and the 75th, 50th and 25th percentile of the average signal intensity (205 032.6, 191 424.1 and 185 480.5 arbitrary units) on the horizontal axis were calculated. (C) Quantification of the sources of variation in the replicated iGp array. Sources of variance between arrays, between scans and between sections, composed of array 1 to array 4, 1st scan to 4th scan, and section 1 to section 4, were calculated to be 44.7 ± 17.00, 17.3 ± 4.57 and 38.1 ± 16.44% as the percent overall variance, respectively." path-from-xml="dsq02502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365016/dsq02502.jpeg" path-from-xml="dsq02502.jpeg" target="_blank">View large</a><a section="7365016" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02502.jpeg?Expires=2147483647&Signature=0-cDHnKw-jskZzexwtWMRnzwfSKGjv-5VOYfdbIEKpufhIDQ8lPDgZBbb6SFSD7sDkNEMgaFPPbdHUFSBdmu5N8doDoBYR-6tEElhGTl~1K7WoJ0VcST0T6UrF4PGnaNv6fdfqng~tjTbJ25SJ6uc14F7hUU4o1wh9puGUh0Xm59g1jw0tURc3dOGzT4uYo7PRJpHrTxpCZbQNCtML9yztfL6sbssl~KW6vitM25CpiQVodfj7H7zAT3olydpl-0wWxZ7jDdAmdeMXT3iWY4yAmrycmHqASU1XTWXhnTgX4kp6O3agwOIdXTZslRCVp98LLsN3cf~B0YPTla5lRpEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365016&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>iGp array set-up and data analysis. (A) Experimental design for the use of the iGp array. Each iGp array harboured four identical sets of 40 iGps that contained oligonucleotides (60 nt) corresponding to various infectious bacteria and viruses (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Cy5-labelled antisense DNA (60 nt) was used as a target sample that emitted fluorescent signals when hybridized with the iGps. (B) Scatter plot of averaged signal intensities (horizontal) versus the CV of overall variation (vertical). Four replicate samples were independently prepared for hybridization to four iGp arrays. Each array was scanned four times to obtain 64 measured values for each probe in total. The average signal intensities and CVs were calculated from these 64 values. For each 40 iGps (black diamonds), the CV of the overall 64 average signal intensities (%) was plotted against the original mean value of the calculated average signal intensities. The 75th, 50th and 25th percentile of the CVs (10.1, 11.6 and 11.9%) on the vertical axis and the 75th, 50th and 25th percentile of the average signal intensity (205 032.6, 191 424.1 and 185 480.5 arbitrary units) on the horizontal axis were calculated. (C) Quantification of the sources of variation in the replicated iGp array. Sources of variance between arrays, between scans and between sections, composed of array 1 to array 4, 1st scan to 4th scan, and section 1 to section 4, were calculated to be 44.7 ± 17.00, 17.3 ± 4.57 and 38.1 ± 16.44% as the percent overall variance, respectively.</p></div></div><div content-id="DSQ025F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02502.jpeg?Expires=2147483647&Signature=X0zzxKOIYaQFrN0UK4WUSs6obKqu~0yD3ZjCB9j-q93sloOZlqMNQMrFDH74QJW8NH5mm-4voweLZ1g5BcvxQdhFYEZ1fp7NK-c1IfGhpAsbUXkOomfk7P9qe2v1MkfLnqRYXoGVAN2CSO5zXUF5bBPkyljZr5Z20RV9wDMvtbofeF1k1C2u4R0HG9-4~l35Y1ac-A1S83v8LhtW1mpY0FXwusrsOAiI7lRNql8dOziVwHa8xPLgJv-OuAZzE0y1EsY5mRN3x193dNrjHMq2u6LwDyFDABO4eFD-v0ox1WAkPofRHc8QeGndeqhMckQNIhwumysXHScyDrHDmpjK-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="iGp array set-up and data analysis. (A) Experimental design for the use of the iGp array. Each iGp array harboured four identical sets of 40 iGps that contained oligonucleotides (60 nt) corresponding to various infectious bacteria and viruses (Supplementary Table S1). Cy5-labelled antisense DNA (60 nt) was used as a target sample that emitted fluorescent signals when hybridized with the iGps. (B) Scatter plot of averaged signal intensities (horizontal) versus the CV of overall variation (vertical). Four replicate samples were independently prepared for hybridization to four iGp arrays. Each array was scanned four times to obtain 64 measured values for each probe in total. The average signal intensities and CVs were calculated from these 64 values. For each 40 iGps (black diamonds), the CV of the overall 64 average signal intensities (%) was plotted against the original mean value of the calculated average signal intensities. The 75th, 50th and 25th percentile of the CVs (10.1, 11.6 and 11.9%) on the vertical axis and the 75th, 50th and 25th percentile of the average signal intensity (205 032.6, 191 424.1 and 185 480.5 arbitrary units) on the horizontal axis were calculated. (C) Quantification of the sources of variation in the replicated iGp array. Sources of variance between arrays, between scans and between sections, composed of array 1 to array 4, 1st scan to 4th scan, and section 1 to section 4, were calculated to be 44.7 ± 17.00, 17.3 ± 4.57 and 38.1 ± 16.44% as the percent overall variance, respectively." path-from-xml="dsq02502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365016/dsq02502.jpeg" path-from-xml="dsq02502.jpeg" target="_blank">View large</a><a section="7365016" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02502.jpeg?Expires=2147483647&Signature=0-cDHnKw-jskZzexwtWMRnzwfSKGjv-5VOYfdbIEKpufhIDQ8lPDgZBbb6SFSD7sDkNEMgaFPPbdHUFSBdmu5N8doDoBYR-6tEElhGTl~1K7WoJ0VcST0T6UrF4PGnaNv6fdfqng~tjTbJ25SJ6uc14F7hUU4o1wh9puGUh0Xm59g1jw0tURc3dOGzT4uYo7PRJpHrTxpCZbQNCtML9yztfL6sbssl~KW6vitM25CpiQVodfj7H7zAT3olydpl-0wWxZ7jDdAmdeMXT3iWY4yAmrycmHqASU1XTWXhnTgX4kp6O3agwOIdXTZslRCVp98LLsN3cf~B0YPTla5lRpEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365016&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>iGp array set-up and data analysis. (A) Experimental design for the use of the iGp array. Each iGp array harboured four identical sets of 40 iGps that contained oligonucleotides (60 nt) corresponding to various infectious bacteria and viruses (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Cy5-labelled antisense DNA (60 nt) was used as a target sample that emitted fluorescent signals when hybridized with the iGps. (B) Scatter plot of averaged signal intensities (horizontal) versus the CV of overall variation (vertical). Four replicate samples were independently prepared for hybridization to four iGp arrays. Each array was scanned four times to obtain 64 measured values for each probe in total. The average signal intensities and CVs were calculated from these 64 values. For each 40 iGps (black diamonds), the CV of the overall 64 average signal intensities (%) was plotted against the original mean value of the calculated average signal intensities. The 75th, 50th and 25th percentile of the CVs (10.1, 11.6 and 11.9%) on the vertical axis and the 75th, 50th and 25th percentile of the average signal intensity (205 032.6, 191 424.1 and 185 480.5 arbitrary units) on the horizontal axis were calculated. (C) Quantification of the sources of variation in the replicated iGp array. Sources of variance between arrays, between scans and between sections, composed of array 1 to array 4, 1st scan to 4th scan, and section 1 to section 4, were calculated to be 44.7 ± 17.00, 17.3 ± 4.57 and 38.1 ± 16.44% as the percent overall variance, respectively.</p></div></div><p>The hybridized signals were observed with high sensitivity; even signals that were very weak after 1 s of scanning became strong enough for analytical purposes at 4 s, and these signals were saturated at 40 s of scanning (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1A</a>). Four arrays, composed of four independent sections, were scanned four times each. In total, 64 measured values for each probe were analysed (Fig. <a reveal-id="DSQ025F2" data-open="DSQ025F2" class="link link-reveal link-table xref-fig">2</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1A</a>). The overall CV of signal intensities was very low (10.6% on average), indicating that the iGp array is a highly reproducible platform comparable with other existing microarray systems (Fig. <a reveal-id="DSQ025F2" data-open="DSQ025F2" class="link link-reveal link-table xref-fig">2</a>B). Thus, the sources of variance were analysed in detail (Fig. <a reveal-id="DSQ025F2" data-open="DSQ025F2" class="link link-reveal link-table xref-fig">2</a>C) and the largest source was between arrays (44.7%), as expected. The second largest source was between sections (38.1%), whereas variations between scans were slight (17.3%). Taken together, these results indicate that Genopal™ is a reliable platform for focused microarray that produces reproducible results from synthetic RNA samples.</p><p>To examine whether the iGp array was reliable in terms of speed, it was determined whether the hybridization time, which is the most tedious step in any DNA array, could be shortened. Thus, the span of hybridization time was varied from 0.5 h to overnight and compared with the resulting signal intensities (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1B</a>). We found that a 120-s scan after hybridization for only 0.5 h yielded signals that were sufficiently strong for signal profiling, even though the signal was much reduced compared with that of overnight hybridization (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1C</a>). This result further supported the commercial usefulness of Genopal™.</p>                            <h3 scrollto-destination=7365019 id="7365019" class="section-title" data-legacy-id=s3c>3.3. Preparation and testing of an autoimmunity array</h3>
<p>To evaluate the clinical applicability of Genopal™, we designed an ‘autoimmunity array’ based on genes related to autoimmune diseases. For this purpose, we selected 133 autoimmunity-related genes that were previously identified as being up- or down-regulated in PBMCs from patients with various autoimmune diseases (Fig. <a reveal-id="DSQ025F3" data-open="DSQ025F3" class="link link-reveal link-table xref-fig">3</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), such as systemic lupus erythematosus (SLE),<sup><a reveal-id="DSQ025C11" data-open="DSQ025C11" class="link link-ref link-reveal xref-bibr">11</a></sup> idiopathic thrombocytopenic purpura (ITP),<sup><a reveal-id="DSQ025C11" data-open="DSQ025C11" class="link link-ref link-reveal xref-bibr">11</a></sup>  rheumatoid arthritis (RA)<sup><a reveal-id="DSQ025C12" data-open="DSQ025C12" class="link link-ref link-reveal xref-bibr">12</a></sup>  and/or vasculitis.<sup><a reveal-id="DSQ025C13" data-open="DSQ025C13" class="link link-ref link-reveal xref-bibr">13</a></sup>  In addition, 12 genes from the group of PBMC-specific genes termed PREP (predominantly expressed in PBMC) genes, which are highly expressed in normal PBMCs but not in normal human fibroblasts,<sup><a reveal-id="DSQ025C14" data-open="DSQ025C14" class="link link-ref link-reveal xref-bibr">14</a></sup>  and 68 immunity-related genes known to be involved in immune regulation were also used. Genes for α-tubulin, β-tubulin and GAPDH were used as control probes, but not as normalization probes (Fig. <a reveal-id="DSQ025F3" data-open="DSQ025F3" class="link link-reveal link-table xref-fig">3</a>A). In total, 216 probes were selected, and 65 nt-long oligonucleotides were designed and synthesized from the mRNA sequence of each gene. These probes were termed aGp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>: aGp #1–216), and the Genopal™ array performed with these probes was called an aGp array. </p>                        <a id="7365021" scrollto-destination="7365021"></a>
<div id="DSQ025F3" data-id="DSQ025F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02503.jpeg?Expires=2147483647&Signature=0wzSzCFtEnaO4jzJQ8BAJrXtbnHxu2J0cUp96JWbmoGRDujXd1NsPgQdDCut---Xa-Dy5f9psKNNCigAk-cBonsrxgocRhgSEGh01jVooAiu~K5gM6shk~sS~Rf~IncfBRMLltZdF~UvOdwV0crzl-fxB-45CgcdGQRpQHlFZarpsVrSIj7r5ATcrjh9qYTkJWE17q3vtuGP9n3BHPd6GOyY~pR34YWi18sEopFRlFUjALTOvonCvDiUmlAxxng7VYEt0zSxFMJQmtClRjSaEqpATRD4albpMkMEhDqbx1mS45YzJ5psCw0alnbFOQf5SOiN2jG6uWEPcxBwMv3nxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Probe design and execution of aGp arrays. (A) Distribution of the probe groups (autoimmunity-related genes, immunity-related genes, PREP genes and control genes) installed into the aGp array. Each group is indicated by purple, green, red or blue sections, respectively. Among the autoimmunity-related genes, the numbers of up- or down-regulated genes in autoimmune diseases are shown in parenthesis (see the leftmost column of Supplementary Table S2 for details). The probe set of the aGp array is comprise one-third of experimental probes (data not shown) and two-thirds of genes associated with immunity. (B) Scatter plot of average signal intensities (horizontal) and CV (vertical) of the HV pool measured by the aGp array. RNA was extracted from the PBMCs of the HV pool and hybridized onto the aGp array. For 216 aGps on the aGp arrays, the CVs of the average signal intensities measured four times were plotted against the mean calculated average signal intensities. The 216 aGps are represented by grey diamonds. The average, 75th, 50th and 25th percentiles of the CVs (31.3, 22.5, 8.2 and 4.4%) on the vertical axis and the average, 75th, 50th and 25th percentiles of the average signal intensity (288.7, 49.3, 11.3 and 5.3) on the horizontal axis were calculated." path-from-xml="dsq02503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365021/dsq02503.jpeg" path-from-xml="dsq02503.jpeg" target="_blank">View large</a><a section="7365021" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02503.jpeg?Expires=2147483647&Signature=hPC4NqNQB3C5Ol6pwptPpW3DLa0bhdpwb8-XWtr4JPVUnJ-WrHpvnwVmgvjhCuAdqxBclzWUL5dW4ESyFVKtdNvj9hEwv4aMFjN9LfxMVo9nTres5~vQLPJK2sqQu9qnScb4aoAz-fJvB0XdEpkWAd7WVsjpyx-dC~OP4rPGN0Ppt4-DuqvoOiQvPCNJGVOG3WCxc2kCDEtINIAeierZjDBQ2QJb1VT4qmHu-8VKqqAwgrAJA6dLqivv9z-H0qzjgC~bZ4VPZRT4CSYjVKTUSXz9jDX0InHYycluke~60m-nDhPdFHzXmiYtXfcDIBiWCvJHRxSlIbypnot3Gj5PWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365021&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Probe design and execution of aGp arrays. (A) Distribution of the probe groups (autoimmunity-related genes, immunity-related genes, PREP genes and control genes) installed into the aGp array. Each group is indicated by purple, green, red or blue sections, respectively. Among the autoimmunity-related genes, the numbers of up- or down-regulated genes in autoimmune diseases are shown in parenthesis (see the leftmost column of <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> for details). The probe set of the aGp array is comprise one-third of experimental probes (data not shown) and two-thirds of genes associated with immunity. (B) Scatter plot of average signal intensities (horizontal) and CV (vertical) of the HV pool measured by the aGp array. RNA was extracted from the PBMCs of the HV pool and hybridized onto the aGp array. For 216 aGps on the aGp arrays, the CVs of the average signal intensities measured four times were plotted against the mean calculated average signal intensities. The 216 aGps are represented by grey diamonds. The average, 75th, 50th and 25th percentiles of the CVs (31.3, 22.5, 8.2 and 4.4%) on the vertical axis and the average, 75th, 50th and 25th percentiles of the average signal intensity (288.7, 49.3, 11.3 and 5.3) on the horizontal axis were calculated.</p></div></div><div content-id="DSQ025F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02503.jpeg?Expires=2147483647&Signature=0wzSzCFtEnaO4jzJQ8BAJrXtbnHxu2J0cUp96JWbmoGRDujXd1NsPgQdDCut---Xa-Dy5f9psKNNCigAk-cBonsrxgocRhgSEGh01jVooAiu~K5gM6shk~sS~Rf~IncfBRMLltZdF~UvOdwV0crzl-fxB-45CgcdGQRpQHlFZarpsVrSIj7r5ATcrjh9qYTkJWE17q3vtuGP9n3BHPd6GOyY~pR34YWi18sEopFRlFUjALTOvonCvDiUmlAxxng7VYEt0zSxFMJQmtClRjSaEqpATRD4albpMkMEhDqbx1mS45YzJ5psCw0alnbFOQf5SOiN2jG6uWEPcxBwMv3nxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Probe design and execution of aGp arrays. (A) Distribution of the probe groups (autoimmunity-related genes, immunity-related genes, PREP genes and control genes) installed into the aGp array. Each group is indicated by purple, green, red or blue sections, respectively. Among the autoimmunity-related genes, the numbers of up- or down-regulated genes in autoimmune diseases are shown in parenthesis (see the leftmost column of Supplementary Table S2 for details). The probe set of the aGp array is comprise one-third of experimental probes (data not shown) and two-thirds of genes associated with immunity. (B) Scatter plot of average signal intensities (horizontal) and CV (vertical) of the HV pool measured by the aGp array. RNA was extracted from the PBMCs of the HV pool and hybridized onto the aGp array. For 216 aGps on the aGp arrays, the CVs of the average signal intensities measured four times were plotted against the mean calculated average signal intensities. The 216 aGps are represented by grey diamonds. The average, 75th, 50th and 25th percentiles of the CVs (31.3, 22.5, 8.2 and 4.4%) on the vertical axis and the average, 75th, 50th and 25th percentiles of the average signal intensity (288.7, 49.3, 11.3 and 5.3) on the horizontal axis were calculated." path-from-xml="dsq02503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365021/dsq02503.jpeg" path-from-xml="dsq02503.jpeg" target="_blank">View large</a><a section="7365021" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02503.jpeg?Expires=2147483647&Signature=hPC4NqNQB3C5Ol6pwptPpW3DLa0bhdpwb8-XWtr4JPVUnJ-WrHpvnwVmgvjhCuAdqxBclzWUL5dW4ESyFVKtdNvj9hEwv4aMFjN9LfxMVo9nTres5~vQLPJK2sqQu9qnScb4aoAz-fJvB0XdEpkWAd7WVsjpyx-dC~OP4rPGN0Ppt4-DuqvoOiQvPCNJGVOG3WCxc2kCDEtINIAeierZjDBQ2QJb1VT4qmHu-8VKqqAwgrAJA6dLqivv9z-H0qzjgC~bZ4VPZRT4CSYjVKTUSXz9jDX0InHYycluke~60m-nDhPdFHzXmiYtXfcDIBiWCvJHRxSlIbypnot3Gj5PWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365021&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Probe design and execution of aGp arrays. (A) Distribution of the probe groups (autoimmunity-related genes, immunity-related genes, PREP genes and control genes) installed into the aGp array. Each group is indicated by purple, green, red or blue sections, respectively. Among the autoimmunity-related genes, the numbers of up- or down-regulated genes in autoimmune diseases are shown in parenthesis (see the leftmost column of <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> for details). The probe set of the aGp array is comprise one-third of experimental probes (data not shown) and two-thirds of genes associated with immunity. (B) Scatter plot of average signal intensities (horizontal) and CV (vertical) of the HV pool measured by the aGp array. RNA was extracted from the PBMCs of the HV pool and hybridized onto the aGp array. For 216 aGps on the aGp arrays, the CVs of the average signal intensities measured four times were plotted against the mean calculated average signal intensities. The 216 aGps are represented by grey diamonds. The average, 75th, 50th and 25th percentiles of the CVs (31.3, 22.5, 8.2 and 4.4%) on the vertical axis and the average, 75th, 50th and 25th percentiles of the average signal intensity (288.7, 49.3, 11.3 and 5.3) on the horizontal axis were calculated.</p></div></div><p>Reference cRNA for fluorescence (Cy5) labelling was amplified using a total mRNA pool extracted from 16 HV pools, which was used for hybridization to the aGp array. Each hybridized aGp array was scanned four times to estimate the variability of the data. The average intensities of 54 of the 216 probes were greater than 50 arbitrary units (au), and the CVs of these probes were very small (median = 4.8%); this is surprising considering the high mRNA complexity of clinical samples. In contrast, the CVs of the rest of probes were larger (median was 14.7%), as expected (Fig. <a reveal-id="DSQ025F3" data-open="DSQ025F3" class="link link-reveal link-table xref-fig">3</a>B). These results indicate that the aGp arrays also yielded reliable data, although aGp arrays that were hybridized with clinical mRNA samples showed lower intensities and larger CVs than iGp arrays that were hybridized with synthetic antisense DNAs.</p>                            <h3 scrollto-destination=7365023 id="7365023" class="section-title" data-legacy-id=s3d>3.4. Application of the autoimmunity array to clinical samples</h3>
<p>Next, an aGp array was applied to patient samples. Fluorescently labelled cRNAs were prepared using PBMCs collected from 13 patients suffering from TA, a rare autoimmune disease of the large arteries, and used as target samples. These patients (TA1–13) were under medical care (being treated with anti-inflammatory drugs such as steroids) and were clinically inactive (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> for the patients’ morbid states). The 13 TA cRNA samples were hybridized to 13 aGp arrays to generate 13 biological replicates. Additionally, the TA aGp arrays were also tested with other autoimmune diseases such as Behçet's syndrome (BD), Churg–Strauss syndrome (CSS), microscopic polyangiitis (MPA), malignant RA (MRA), polyarteritis nodosa (PN) and Wegener's granulomatosis (WG), as well as with normal volunteers (Nor) as comparable target samples (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>) in order to determine whether the expression profiles of the 13 TA patients were specific and useful for the diagnosis of TA.</p><p>To validate the reference TA profile described above by averaging 13 TA samples, the Spearman rank correlation for 216 aGp genes was calculated between each sample and the reference TA profile. Notably, all TA samples (TA1–13) were highly correlated (0.85 ± 0.04) with the TA reference (black circles in Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>A). The maximum and the minimum values of the Spearman rank correlation were computed to be 0.91 and 0.77, respectively, revealing an extremely tight correlation between all TA samples (TA1–13). Moreover, as shown by grey and white symbols in the middle and the right areas of Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>A, other patient samples showed conspicuously low Spearman's rank correlation coefficients (0.29 ± 0.12). This indicates that the aGp array is useful for the diagnosis of TA even if the patient's morbid state is inactive. </p>                        <a id="7365026" scrollto-destination="7365026"></a>
<div id="DSQ025F4" data-id="DSQ025F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02504.jpeg?Expires=2147483647&Signature=kVxiwGs5kfZSf57wHmtwN4chnkmqJ2C1GLYIbEH~ZOsivBucTHOoqxvxGyWTRyaMYoAGiRIyKyB~jQFxOT57eco2AVb-1vKsyuCSqfJNJN0pA~x2BSGRb4P042vS0LU0DbpTnstnFv2maMnd4af8o3n1UxBqK1WeFfAWPnKh5innAtWlNOtTCwF2VkuvLHltXgXzGf7lL52I4jtiE8wxlFlCnJ43G~fd4eUrbDcXkitDo14v0iG8tZ2yPEdEO1TnMu~5sLwfA9bSeCDUX1vGtAXcsNnpJ~HN880iGCUCnH66mmSxR1kP8hifxu9pkVztOBln2UTqGOkv5leinq6bRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiling with the aGp array. (A) Comparison of the Spearman rank correlation (vertical) between various autoimmune diseases and normal individuals. Black circles, TA; white squares, Nor (normal); black crosses, BD; white diamonds, CSS; grey circles, MPA; a grey bar, MRA; a white circle, PN; a grey square, WG. The number of tested individuals is indicated in parentheses. Plots are grouped into eight autoimmune disease groups and a HV (Nor) group. All of the 13 TA patients (black circles) are highly correlated by the Spearman rank correlation (r = 0.85 ± 0.04), whereas other autoimmune disease groups and the HV pool group showed distinctively lower correlation coefficients (Nor, 0.24; BD, 0.23; CSS, 0.43; MPA, 0.26; MRA, 0.38; PN, 0.37; WG, 0.31). (B) Line graphs showing the expression profiles of 59 characteristic genes in patients with TA,13 BD,  CSS4  and MPA.3 The number of examined patients is indicated in parentheses. The colour bar at the right side of the panel indicates the log2 ratio for each patient versus the HV pool sample. Line colours with intensity gradients indicate the mean values of relative expression: blue (down-regulation) corresponds to a log2 ratio −2, and crimson (up-regulation) corresponds to a log2 ratio 2 when each patient and HV pool are compared. Black lines signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared to the HV pool sample (see Supplementary Table S4 for details). (C) Scatter plot of the average signal intensities (in log scale) of the 13 TA patients versus the HV pool sample, plotted along the vertical and horizontal axes, respectively. Black spots indicate the 59 characteristic genes described above. The colour bar indicates the signal intensity of the 13 TA patients versus the HV pool sample. Circle colours with intensity gradients indicate the mean value of the expression level: blue (down-regulation) corresponds to a signal intensity of 10, and crimson (up-regulation) corresponds to a signal intensity of 250 when the average value of the 13 TA patients and the HV pool are compared. Black circles signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared with the HV pool sample (see Supplementary Table S4 for details)." path-from-xml="dsq02504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365026/dsq02504.jpeg" path-from-xml="dsq02504.jpeg" target="_blank">View large</a><a section="7365026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02504.jpeg?Expires=2147483647&Signature=rNV4w6~s62SNlAJC3ZpXmPSmsQIKyxn03mot8SRWNE37Z7kbOEPQ-mYyKHLeWQCq-Et0nr90mTkItRoKqL9DxWfJ6RhVJWkoYPe0~V8-YfhlYnP1e09bYjgD~52xGuMEqp85G2OoYM6g7snUSyGqeVinbzIJLk~MhUXBSPHLTvvWZrSgLUB7STlC~fnlKuSTGsBN4UtFtgIeFxLUVpKnPRDSPAVzm-nl9M1Z~G0bC0it-DzRK~~MQXab-vd2jYIVjhHVT5jC4YZp9yPWM0CE7JEgkCSiLAWoCErFITqNt6Jh-8jbmC~38Su6LuIMsEIDV0tXyMZOZfCe7tbSECtxsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365026&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiling with the aGp array. (A) Comparison of the Spearman rank correlation (vertical) between various autoimmune diseases and normal individuals. Black circles, TA; white squares, Nor (normal); black crosses, BD; white diamonds, CSS; grey circles, MPA; a grey bar, MRA; a white circle, PN; a grey square, WG. The number of tested individuals is indicated in parentheses. Plots are grouped into eight autoimmune disease groups and a HV (Nor) group. All of the 13 TA patients (black circles) are highly correlated by the Spearman rank correlation (<em>r</em> = 0.85 ± 0.04), whereas other autoimmune disease groups and the HV pool group showed distinctively lower correlation coefficients (Nor, 0.24; BD, 0.23; CSS, 0.43; MPA, 0.26; MRA, 0.38; PN, 0.37; WG, 0.31). (B) Line graphs showing the expression profiles of 59 characteristic genes in patients with TA,<sup><a reveal-id="DSQ025C13" data-open="DSQ025C13" class="link link-ref link-reveal xref-bibr">13</a></sup> BD,  CSS<sup><a reveal-id="DSQ025C4" data-open="DSQ025C4" class="link link-ref link-reveal xref-bibr">4</a></sup>  and MPA.<sup><a reveal-id="DSQ025C3" data-open="DSQ025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The number of examined patients is indicated in parentheses. The colour bar at the right side of the panel indicates the log<sub>2</sub> ratio for each patient versus the HV pool sample. Line colours with intensity gradients indicate the mean values of relative expression: blue (down-regulation) corresponds to a log<sub>2</sub> ratio −2, and crimson (up-regulation) corresponds to a log<sub>2</sub> ratio 2 when each patient and HV pool are compared. Black lines signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared to the HV pool sample (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> for details). (C) Scatter plot of the average signal intensities (in log scale) of the 13 TA patients versus the HV pool sample, plotted along the vertical and horizontal axes, respectively. Black spots indicate the 59 characteristic genes described above. The colour bar indicates the signal intensity of the 13 TA patients versus the HV pool sample. Circle colours with intensity gradients indicate the mean value of the expression level: blue (down-regulation) corresponds to a signal intensity of 10, and crimson (up-regulation) corresponds to a signal intensity of 250 when the average value of the 13 TA patients and the HV pool are compared. Black circles signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared with the HV pool sample (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> for details).</p></div></div><div content-id="DSQ025F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02504.jpeg?Expires=2147483647&Signature=kVxiwGs5kfZSf57wHmtwN4chnkmqJ2C1GLYIbEH~ZOsivBucTHOoqxvxGyWTRyaMYoAGiRIyKyB~jQFxOT57eco2AVb-1vKsyuCSqfJNJN0pA~x2BSGRb4P042vS0LU0DbpTnstnFv2maMnd4af8o3n1UxBqK1WeFfAWPnKh5innAtWlNOtTCwF2VkuvLHltXgXzGf7lL52I4jtiE8wxlFlCnJ43G~fd4eUrbDcXkitDo14v0iG8tZ2yPEdEO1TnMu~5sLwfA9bSeCDUX1vGtAXcsNnpJ~HN880iGCUCnH66mmSxR1kP8hifxu9pkVztOBln2UTqGOkv5leinq6bRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiling with the aGp array. (A) Comparison of the Spearman rank correlation (vertical) between various autoimmune diseases and normal individuals. Black circles, TA; white squares, Nor (normal); black crosses, BD; white diamonds, CSS; grey circles, MPA; a grey bar, MRA; a white circle, PN; a grey square, WG. The number of tested individuals is indicated in parentheses. Plots are grouped into eight autoimmune disease groups and a HV (Nor) group. All of the 13 TA patients (black circles) are highly correlated by the Spearman rank correlation (r = 0.85 ± 0.04), whereas other autoimmune disease groups and the HV pool group showed distinctively lower correlation coefficients (Nor, 0.24; BD, 0.23; CSS, 0.43; MPA, 0.26; MRA, 0.38; PN, 0.37; WG, 0.31). (B) Line graphs showing the expression profiles of 59 characteristic genes in patients with TA,13 BD,  CSS4  and MPA.3 The number of examined patients is indicated in parentheses. The colour bar at the right side of the panel indicates the log2 ratio for each patient versus the HV pool sample. Line colours with intensity gradients indicate the mean values of relative expression: blue (down-regulation) corresponds to a log2 ratio −2, and crimson (up-regulation) corresponds to a log2 ratio 2 when each patient and HV pool are compared. Black lines signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared to the HV pool sample (see Supplementary Table S4 for details). (C) Scatter plot of the average signal intensities (in log scale) of the 13 TA patients versus the HV pool sample, plotted along the vertical and horizontal axes, respectively. Black spots indicate the 59 characteristic genes described above. The colour bar indicates the signal intensity of the 13 TA patients versus the HV pool sample. Circle colours with intensity gradients indicate the mean value of the expression level: blue (down-regulation) corresponds to a signal intensity of 10, and crimson (up-regulation) corresponds to a signal intensity of 250 when the average value of the 13 TA patients and the HV pool are compared. Black circles signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared with the HV pool sample (see Supplementary Table S4 for details)." path-from-xml="dsq02504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365026/dsq02504.jpeg" path-from-xml="dsq02504.jpeg" target="_blank">View large</a><a section="7365026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02504.jpeg?Expires=2147483647&Signature=rNV4w6~s62SNlAJC3ZpXmPSmsQIKyxn03mot8SRWNE37Z7kbOEPQ-mYyKHLeWQCq-Et0nr90mTkItRoKqL9DxWfJ6RhVJWkoYPe0~V8-YfhlYnP1e09bYjgD~52xGuMEqp85G2OoYM6g7snUSyGqeVinbzIJLk~MhUXBSPHLTvvWZrSgLUB7STlC~fnlKuSTGsBN4UtFtgIeFxLUVpKnPRDSPAVzm-nl9M1Z~G0bC0it-DzRK~~MQXab-vd2jYIVjhHVT5jC4YZp9yPWM0CE7JEgkCSiLAWoCErFITqNt6Jh-8jbmC~38Su6LuIMsEIDV0tXyMZOZfCe7tbSECtxsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365026&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiling with the aGp array. (A) Comparison of the Spearman rank correlation (vertical) between various autoimmune diseases and normal individuals. Black circles, TA; white squares, Nor (normal); black crosses, BD; white diamonds, CSS; grey circles, MPA; a grey bar, MRA; a white circle, PN; a grey square, WG. The number of tested individuals is indicated in parentheses. Plots are grouped into eight autoimmune disease groups and a HV (Nor) group. All of the 13 TA patients (black circles) are highly correlated by the Spearman rank correlation (<em>r</em> = 0.85 ± 0.04), whereas other autoimmune disease groups and the HV pool group showed distinctively lower correlation coefficients (Nor, 0.24; BD, 0.23; CSS, 0.43; MPA, 0.26; MRA, 0.38; PN, 0.37; WG, 0.31). (B) Line graphs showing the expression profiles of 59 characteristic genes in patients with TA,<sup><a reveal-id="DSQ025C13" data-open="DSQ025C13" class="link link-ref link-reveal xref-bibr">13</a></sup> BD,  CSS<sup><a reveal-id="DSQ025C4" data-open="DSQ025C4" class="link link-ref link-reveal xref-bibr">4</a></sup>  and MPA.<sup><a reveal-id="DSQ025C3" data-open="DSQ025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The number of examined patients is indicated in parentheses. The colour bar at the right side of the panel indicates the log<sub>2</sub> ratio for each patient versus the HV pool sample. Line colours with intensity gradients indicate the mean values of relative expression: blue (down-regulation) corresponds to a log<sub>2</sub> ratio −2, and crimson (up-regulation) corresponds to a log<sub>2</sub> ratio 2 when each patient and HV pool are compared. Black lines signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared to the HV pool sample (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> for details). (C) Scatter plot of the average signal intensities (in log scale) of the 13 TA patients versus the HV pool sample, plotted along the vertical and horizontal axes, respectively. Black spots indicate the 59 characteristic genes described above. The colour bar indicates the signal intensity of the 13 TA patients versus the HV pool sample. Circle colours with intensity gradients indicate the mean value of the expression level: blue (down-regulation) corresponds to a signal intensity of 10, and crimson (up-regulation) corresponds to a signal intensity of 250 when the average value of the 13 TA patients and the HV pool are compared. Black circles signify genes that are 2-fold up- or down-regulated in at least 7 of the 13 TA patients compared with the HV pool sample (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> for details).</p></div></div><p>Next, 34 genes were isolated that showed at least a 2-fold up-regulation and 25 genes that showed at least a 2-fold down-regulation in at least 7 of 13 TA patients (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). We also found that the expression profiles of 59 characteristic genes represented by line graphs in 13 TA, 17 BD, 4 CSS and 3 MPA patients distinguished TA from other autoimmune diseases. The black lines signifying genes that were 2-fold up- or down-regulated in at least 7 of the 13 TA patients randomly fluctuated in other patients (Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>B). Because we did not normalize the aGp array data, systematic shifts in log-ratio distributions in proportion to the signal intensities were observed in the scatter plot (Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>C). Nonetheless, these 59 genes formed distinctive groupings in terms of correlation across the 216 genes. Namely, up- or down-regulated genes (black circles in Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>C) localized outside the diagonal bundles of dots (purple and crimson circles in Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>C) over a wide range of intensities. In contrast, these 59 genes showed no obvious correlation patterns in other patient samples (data not shown).</p><p>Notably, the aGp array has additional practical advantages for clinical use because the obtained data were found to be very stable after scanning, even without sealing or other direct countermeasures against exposure to ozone (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Moreover, the background level of the data after 19–27 days were largely reduced compared with 1 day data, which confirmed the reliability of the data even with low signal intensity. Taken together, these results indicate that aGp arrays are clinically useful because they can clearly distinguish TA patient profiles from other patients by the Spearman rank correlation using all 216 probes.</p>                            <h3 scrollto-destination=7365029 id="7365029" class="section-title" data-legacy-id=s3e>3.5. Comparison between aGp array, Agilent's microarray and qPCR</h3>
<p>Next, the performance of aGp arrays was compared with other tools to examine the variation of the expression patterns of the genes in the 13 TA patient samples. First, all TA samples (TA1–13) were applied to Agilent's Whole Human Genome Microarray (4 × 44K). DEFA3, interleukin (IL)-4 and IL-10 were selected for further analysis because they are known to be highly associated with various autoimmune diseases.<sup><a reveal-id="DSQ025C14" data-open="DSQ025C14" class="link link-ref link-reveal xref-bibr">14</a></sup> When the expression profiles of each gene from aGp array, Agilent's microarray and qPCR are compared, represented by green, purple and blue lines, respectively, it is apparent that DEFA3 achieved almost complete correlation among the three measurement systems (Fig. <a reveal-id="DSQ025F5" data-open="DSQ025F5" class="link link-reveal link-table xref-fig">5</a>A). This correlation is surprising considering the distinct sequences of the probes used in each measurement and further confirmed the reliability of the data obtained by aGp arrays. In contrast, IL-4 and IL-10 showed distinct expression patterns depending on the system (Fig. <a reveal-id="DSQ025F5" data-open="DSQ025F5" class="link link-reveal link-table xref-fig">5</a>Bi and ii). </p>                        <a id="7365031" scrollto-destination="7365031"></a>
<div id="DSQ025F5" data-id="DSQ025F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02505.jpeg?Expires=2147483647&Signature=CW~mUxtKzqAzGzb2XYj2Awc6Buf1HIQoYu309~MSplXQ3-18gJ54PHAMf6diVtG62u~qrSYi9Z7vbq7XR7gccKmWITfwrGxrURkTDEGUeih-XHKKXSOYP2QR4mAVPWXlq5mZczFSJW9vZ-xhlJykKgYadIkkzzEf6Wg4Ymsqtj0k994oJL1lL18rQkC-a-qkYw-AkFmYqkmBW6aXd3ZM2gOJcQ73ZY0pwGhpiVp3W8ZcwjH4ftQbeloZYrhzpqNcyj504n64C55RWNXxdGwWohQA8Enc~D~ofpzOc4Ng3DYWc0piD8vIUqlnDjsbmdHCDpzLERE3Cg1MOZSezlE6xQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of DEFA3 (A), IL-4 (Bi) and IL-10 (Bii) in the 13 TA patients. The vertical axis indicates the log2 ratio measured by qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple). The horizontal axis indicates the patient number. (A) The comparison of the values of peaks and valleys for DEFA3 revealed that these three methods yielded very similar expression profiles. (Bi and ii) The expression profiles of the IL-4 and IL-10 genes for the 13 TA patients were different between qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple)." path-from-xml="dsq02505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365031/dsq02505.jpeg" path-from-xml="dsq02505.jpeg" target="_blank">View large</a><a section="7365031" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02505.jpeg?Expires=2147483647&Signature=Y3ETyqTsijfRzN2U8gfUGZ1b60z3G1WMbE5q3DzOXZR-jy89OEAAODQjVaNvSOG9Jksb5F4qV~Mrhh5qi82zITqW5noHuaA7sk0xX6F5GzQLuuNxpN4M9xEqcWwg~dMfXUwVd37nNI8s6R3KB6VCVZMrXdJ5RZ2cMtUWa3kN2uyYf4DxNjyl2wNQskHm0EzWMA3uPYdMOm5DBmRa75lFdbAg7bkJSWjiiikLmxJI6HTYvpl9bL7KSQJtHiDZWmzewpYtgC~zVjWjdYBWJR1PzbqSA7Pmh7fE~IXqN3EK09pBsoO08K-hDaiQxccuu0Ztja-9x3g1hdOaYz4Dibshnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365031&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of DEFA3 (A), IL-4 (Bi) and IL-10 (Bii) in the 13 TA patients. The vertical axis indicates the log<sub>2</sub> ratio measured by qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple). The horizontal axis indicates the patient number. (A) The comparison of the values of peaks and valleys for DEFA3 revealed that these three methods yielded very similar expression profiles. (Bi and ii) The expression profiles of the IL-4 and IL-10 genes for the 13 TA patients were different between qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple).</p></div></div><div content-id="DSQ025F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02505.jpeg?Expires=2147483647&Signature=CW~mUxtKzqAzGzb2XYj2Awc6Buf1HIQoYu309~MSplXQ3-18gJ54PHAMf6diVtG62u~qrSYi9Z7vbq7XR7gccKmWITfwrGxrURkTDEGUeih-XHKKXSOYP2QR4mAVPWXlq5mZczFSJW9vZ-xhlJykKgYadIkkzzEf6Wg4Ymsqtj0k994oJL1lL18rQkC-a-qkYw-AkFmYqkmBW6aXd3ZM2gOJcQ73ZY0pwGhpiVp3W8ZcwjH4ftQbeloZYrhzpqNcyj504n64C55RWNXxdGwWohQA8Enc~D~ofpzOc4Ng3DYWc0piD8vIUqlnDjsbmdHCDpzLERE3Cg1MOZSezlE6xQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of DEFA3 (A), IL-4 (Bi) and IL-10 (Bii) in the 13 TA patients. The vertical axis indicates the log2 ratio measured by qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple). The horizontal axis indicates the patient number. (A) The comparison of the values of peaks and valleys for DEFA3 revealed that these three methods yielded very similar expression profiles. (Bi and ii) The expression profiles of the IL-4 and IL-10 genes for the 13 TA patients were different between qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple)." path-from-xml="dsq02505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365031/dsq02505.jpeg" path-from-xml="dsq02505.jpeg" target="_blank">View large</a><a section="7365031" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02505.jpeg?Expires=2147483647&Signature=Y3ETyqTsijfRzN2U8gfUGZ1b60z3G1WMbE5q3DzOXZR-jy89OEAAODQjVaNvSOG9Jksb5F4qV~Mrhh5qi82zITqW5noHuaA7sk0xX6F5GzQLuuNxpN4M9xEqcWwg~dMfXUwVd37nNI8s6R3KB6VCVZMrXdJ5RZ2cMtUWa3kN2uyYf4DxNjyl2wNQskHm0EzWMA3uPYdMOm5DBmRa75lFdbAg7bkJSWjiiikLmxJI6HTYvpl9bL7KSQJtHiDZWmzewpYtgC~zVjWjdYBWJR1PzbqSA7Pmh7fE~IXqN3EK09pBsoO08K-hDaiQxccuu0Ztja-9x3g1hdOaYz4Dibshnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365031&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of DEFA3 (A), IL-4 (Bi) and IL-10 (Bii) in the 13 TA patients. The vertical axis indicates the log<sub>2</sub> ratio measured by qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple). The horizontal axis indicates the patient number. (A) The comparison of the values of peaks and valleys for DEFA3 revealed that these three methods yielded very similar expression profiles. (Bi and ii) The expression profiles of the IL-4 and IL-10 genes for the 13 TA patients were different between qPCR–GAPDH (blue), aGp array (green) and Agilent's microarray (purple).</p></div></div><p>Notably, IL-4 and IL-10 were expressed at remarkably low levels compared with DEFA3 (indicated by arrows in Fig. <a reveal-id="DSQ025F6" data-open="DSQ025F6" class="link link-reveal link-table xref-fig">6</a>A). The scatter plot of log signal intensities from the HV pool (horizontal) and patient TA8 (vertical) show that the signal intensities of IL-4 and IL-10 are considerably lower than that of DEFA3. In addition, the correlation coefficients of DEFA3 for the averaged signal intensities of the aGp array and Agilent's microarray against qPCR normalized to GAPDH (0.93 and 0.96) are significantly higher than those of IL-4 (−0.52 and 0.52) and IL-10 (0.12 and −0.06), respectively (Fig. <a reveal-id="DSQ025F6" data-open="DSQ025F6" class="link link-reveal link-table xref-fig">6</a>B). </p>                        <a id="7365033" scrollto-destination="7365033"></a>
<div id="DSQ025F6" data-id="DSQ025F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02506.jpeg?Expires=2147483647&Signature=rntuyvoQoKucVByCxsB~tItGOeA1hVDMzWs8~f8vbebNLM5vTeviB6-5uTpvifOmeTrEakkBfF3c3SgW2zocilUgW8K1913yhTyEQ3FyGmfSYWsu~b6ujBvK70r8diMmtnxEQXhOjKqdLbaaUSbxbFCcOBsgbXpyCm5dKoICY9cCJbkeMtW4PCr4nPcEE0Mz7swRDwLEcIulX52eLm5h~ZzODrZnmiNXQOPhBj81m5VL6nd7FbOysM--lF3vcucweujddGg4YTGMtUMKm7IdjFiJJR-ofo-gQDddW3pxZ0TvNNk~IS1c8t-r5CTJP8kfQgNUhUsR~SkYPk1XJ-1GKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The aGp array provides sufficient sensitivity for diagnostic application. (A) A scatter plot of the log of signal intensity of the HV pool versus patient TA8 is plotted along the horizontal and vertical axes, respectively. DEFA3, IL-4 and IL-10 genes are indicated by orange arrows. The attached colour bar indicates a relative intensity scale. (B) The correlation coefficients of the Genopal™ autoimmunity array and Agilent's microarray against qPCR are shown. The correlation coefficient of DEFA3 for both Genopal™ and Agilent's arrays was high. On the other hand, because the signal intensities of IL-4 and IL-10 were within two orders of magnitude, as shown in the columns on the right, the correlation coefficient of IL-4 or IL-10 is lower than DEFA3." path-from-xml="dsq02506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365033/dsq02506.jpeg" path-from-xml="dsq02506.jpeg" target="_blank">View large</a><a section="7365033" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02506.jpeg?Expires=2147483647&Signature=VB7r5YlEmkzS63~MFubKivJhzVbDuiYa7migfiSAmd2AyHl0iLoeaCL2b9k0AR0-yCu~3e6D0eQag6QElQjc6~SjsolNan1csXDZcy74CKFuDINRjz70ybAvnAE6EzF2efI8yELNR-h2CCevJ1h6~pUz~22xi5PWR7e9KYblUcBI7tffCAMvyEhwjxEXtyc3CSCDCuWFSuKZtUpEHDgFTSwmyYLybPhTh75RP6g-RtzJArFHQhji~iDQQCkUW7faDS3F3An-t~ZLqAo4FdEaYCLfbEmRKUbbBh~uuvEbwiFnoS1StuI3dELQ-U19pG7Nq5Xc-gSDCyHyX3VeEyYfkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365033&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The aGp array provides sufficient sensitivity for diagnostic application. (A) A scatter plot of the log of signal intensity of the HV pool versus patient TA8 is plotted along the horizontal and vertical axes, respectively. DEFA3, IL-4 and IL-10 genes are indicated by orange arrows. The attached colour bar indicates a relative intensity scale. (B) The correlation coefficients of the Genopal™ autoimmunity array and Agilent's microarray against qPCR are shown. The correlation coefficient of DEFA3 for both Genopal™ and Agilent's arrays was high. On the other hand, because the signal intensities of IL-4 and IL-10 were within two orders of magnitude, as shown in the columns on the right, the correlation coefficient of IL-4 or IL-10 is lower than DEFA3.</p></div></div><div content-id="DSQ025F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02506.jpeg?Expires=2147483647&Signature=rntuyvoQoKucVByCxsB~tItGOeA1hVDMzWs8~f8vbebNLM5vTeviB6-5uTpvifOmeTrEakkBfF3c3SgW2zocilUgW8K1913yhTyEQ3FyGmfSYWsu~b6ujBvK70r8diMmtnxEQXhOjKqdLbaaUSbxbFCcOBsgbXpyCm5dKoICY9cCJbkeMtW4PCr4nPcEE0Mz7swRDwLEcIulX52eLm5h~ZzODrZnmiNXQOPhBj81m5VL6nd7FbOysM--lF3vcucweujddGg4YTGMtUMKm7IdjFiJJR-ofo-gQDddW3pxZ0TvNNk~IS1c8t-r5CTJP8kfQgNUhUsR~SkYPk1XJ-1GKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The aGp array provides sufficient sensitivity for diagnostic application. (A) A scatter plot of the log of signal intensity of the HV pool versus patient TA8 is plotted along the horizontal and vertical axes, respectively. DEFA3, IL-4 and IL-10 genes are indicated by orange arrows. The attached colour bar indicates a relative intensity scale. (B) The correlation coefficients of the Genopal™ autoimmunity array and Agilent's microarray against qPCR are shown. The correlation coefficient of DEFA3 for both Genopal™ and Agilent's arrays was high. On the other hand, because the signal intensities of IL-4 and IL-10 were within two orders of magnitude, as shown in the columns on the right, the correlation coefficient of IL-4 or IL-10 is lower than DEFA3." path-from-xml="dsq02506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365033/dsq02506.jpeg" path-from-xml="dsq02506.jpeg" target="_blank">View large</a><a section="7365033" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02506.jpeg?Expires=2147483647&Signature=VB7r5YlEmkzS63~MFubKivJhzVbDuiYa7migfiSAmd2AyHl0iLoeaCL2b9k0AR0-yCu~3e6D0eQag6QElQjc6~SjsolNan1csXDZcy74CKFuDINRjz70ybAvnAE6EzF2efI8yELNR-h2CCevJ1h6~pUz~22xi5PWR7e9KYblUcBI7tffCAMvyEhwjxEXtyc3CSCDCuWFSuKZtUpEHDgFTSwmyYLybPhTh75RP6g-RtzJArFHQhji~iDQQCkUW7faDS3F3An-t~ZLqAo4FdEaYCLfbEmRKUbbBh~uuvEbwiFnoS1StuI3dELQ-U19pG7Nq5Xc-gSDCyHyX3VeEyYfkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365033&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The aGp array provides sufficient sensitivity for diagnostic application. (A) A scatter plot of the log of signal intensity of the HV pool versus patient TA8 is plotted along the horizontal and vertical axes, respectively. DEFA3, IL-4 and IL-10 genes are indicated by orange arrows. The attached colour bar indicates a relative intensity scale. (B) The correlation coefficients of the Genopal™ autoimmunity array and Agilent's microarray against qPCR are shown. The correlation coefficient of DEFA3 for both Genopal™ and Agilent's arrays was high. On the other hand, because the signal intensities of IL-4 and IL-10 were within two orders of magnitude, as shown in the columns on the right, the correlation coefficient of IL-4 or IL-10 is lower than DEFA3.</p></div></div>                            <h3 scrollto-destination=7365037 id="7365037" class="section-title" data-legacy-id=s3f>3.6. Comparison with nCounter™</h3>
<p>To investigate whether aGp or Agilent provided more reliable results, we employed nCounter™ by NanoString<sup><a reveal-id="DSQ025C15" data-open="DSQ025C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and measured the expression levels of 25 relevant genes (aGps) using one patient sample (TA8). The raw data revealed that the expression levels of IL-4 and IL-10 in this patient were very low and that 10 of the 25 genes were up-regulated more than 2-fold compared with the HV sample (green arrows in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Comparison of the relative induction rates between aGp, Agilent and nCounter™ revealed that 12 of the 25 genes (yellow arrows in Fig. <a reveal-id="DSQ025F7" data-open="DSQ025F7" class="link link-reveal link-table xref-fig">7</a>) had similar results in all three methods, whereas nCounter™ favoured aGp arrays (turquoise arrows) or Agilent (pink arrows) for three and the six genes, respectively. The other four genes (purple arrows) showed distinct results among these three methods. As both nCounter™ and qPCR favoured aGp arrays for IL-10 and Agilent for IL-4 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>), it is difficult to determine at present which microarray provided more reliable results. Taken together, these results led us to conclude that aGp arrays provide reliable data at a level similar to Agilent's microarray, particularly when the expression levels of the target gene are not very low. </p>                        <a id="7365043" scrollto-destination="7365043"></a>
<div id="DSQ025F7" data-id="DSQ025F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02507.jpeg?Expires=2147483647&Signature=a3i62NSLjxNtEWQeSpEeY6tuRWC9jD42qsedzsYP5c1lds6ECoQMmHxFsE~ryelE5LIODmJD6XbAHyHe6esvJKl0DR04BlUOP1-nqhDNGRHbVdepSkZ5oCGgfQ7TacBMX7tNK0BULwnoSpCotc1CS1ZsAcB8r5vsQSQ129seLZNWoiqTHn-f4RLYCLnDlW7V0RnsRXq9u6g7w93YfIHs5Qi3GhlKn8I9hTtYcf1LT77TmaQ06oZkLhWW8KiiOQF4EZ-7HEK71HsF9DvCUNeHNsUgooR9ELKPwLXI36cuKmujJ3buKQTBJKD4ZrWm5Py4~vqWgDVFCfEgIm~ZDKjjCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the mRNA expression levels (y-axis) of 25 selected genes (x-axis) in PBMCs of patient TA8 using nCounter™ (blue bars), aGp array (Genopal™; red bars) or Agilent's microarray (green bars). Yellow arrows indicate genes that had similar expression levels in the three methods. Turquoise or pink arrows indicate the genes used by nCounter™ to favour aGp or Agilent in terms of their expression levels. Purple arrows indicate the genes that exhibited different expression levels among the three methods." path-from-xml="dsq02507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365043/dsq02507.jpeg" path-from-xml="dsq02507.jpeg" target="_blank">View large</a><a section="7365043" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02507.jpeg?Expires=2147483647&Signature=ZksLoxLI0OjTe~-HVbV5kYW6~URUR1XWOh6G9RtUkTJ2mz5N7pU4eb85K7ClcGb-0O4HwrI7QkJrxKwuEHwbjncVpk1G1CvaQ5xXttqIfUeVbY-ISiyCmK2JlbVIX5sCwe1OeoZhV1JolWbAVO4nUaoEoA7qlj2bcnDytHSWYot2fmRFi3qVfXjgO8A4sMOlRXk87UmhrLV~PrsEeYQIyTgeAc53SjjllAqGFZ6XnV7L~WIIMjmJeapP~H2dltt2u7ILXWzn5Hq9tA7FrOI2kk3sdyLAgbgVi36dHLDleoB6NYVW538-uMR48bgdVXIikV6Z2PkUmYsvHSozyRj1~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365043&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the mRNA expression levels (<em>y</em>-axis) of 25 selected genes (<em>x</em>-axis) in PBMCs of patient TA8 using nCounter™ (blue bars), aGp array (Genopal™; red bars) or Agilent's microarray (green bars). Yellow arrows indicate genes that had similar expression levels in the three methods. Turquoise or pink arrows indicate the genes used by nCounter™ to favour aGp or Agilent in terms of their expression levels. Purple arrows indicate the genes that exhibited different expression levels among the three methods.</p></div></div><div content-id="DSQ025F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/m_dsq02507.jpeg?Expires=2147483647&Signature=a3i62NSLjxNtEWQeSpEeY6tuRWC9jD42qsedzsYP5c1lds6ECoQMmHxFsE~ryelE5LIODmJD6XbAHyHe6esvJKl0DR04BlUOP1-nqhDNGRHbVdepSkZ5oCGgfQ7TacBMX7tNK0BULwnoSpCotc1CS1ZsAcB8r5vsQSQ129seLZNWoiqTHn-f4RLYCLnDlW7V0RnsRXq9u6g7w93YfIHs5Qi3GhlKn8I9hTtYcf1LT77TmaQ06oZkLhWW8KiiOQF4EZ-7HEK71HsF9DvCUNeHNsUgooR9ELKPwLXI36cuKmujJ3buKQTBJKD4ZrWm5Py4~vqWgDVFCfEgIm~ZDKjjCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the mRNA expression levels (y-axis) of 25 selected genes (x-axis) in PBMCs of patient TA8 using nCounter™ (blue bars), aGp array (Genopal™; red bars) or Agilent's microarray (green bars). Yellow arrows indicate genes that had similar expression levels in the three methods. Turquoise or pink arrows indicate the genes used by nCounter™ to favour aGp or Agilent in terms of their expression levels. Purple arrows indicate the genes that exhibited different expression levels among the three methods." path-from-xml="dsq02507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365043/dsq02507.jpeg" path-from-xml="dsq02507.jpeg" target="_blank">View large</a><a section="7365043" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq02507.jpeg?Expires=2147483647&Signature=ZksLoxLI0OjTe~-HVbV5kYW6~URUR1XWOh6G9RtUkTJ2mz5N7pU4eb85K7ClcGb-0O4HwrI7QkJrxKwuEHwbjncVpk1G1CvaQ5xXttqIfUeVbY-ISiyCmK2JlbVIX5sCwe1OeoZhV1JolWbAVO4nUaoEoA7qlj2bcnDytHSWYot2fmRFi3qVfXjgO8A4sMOlRXk87UmhrLV~PrsEeYQIyTgeAc53SjjllAqGFZ6XnV7L~WIIMjmJeapP~H2dltt2u7ILXWzn5Hq9tA7FrOI2kk3sdyLAgbgVi36dHLDleoB6NYVW538-uMR48bgdVXIikV6Z2PkUmYsvHSozyRj1~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7365043&ar=484698&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the mRNA expression levels (<em>y</em>-axis) of 25 selected genes (<em>x</em>-axis) in PBMCs of patient TA8 using nCounter™ (blue bars), aGp array (Genopal™; red bars) or Agilent's microarray (green bars). Yellow arrows indicate genes that had similar expression levels in the three methods. Turquoise or pink arrows indicate the genes used by nCounter™ to favour aGp or Agilent in terms of their expression levels. Purple arrows indicate the genes that exhibited different expression levels among the three methods.</p></div></div>                            <h2 scrollto-destination=7365045 id="7365045" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In the present study, we describe a novel platform termed Genopal™ that is ideal for the preparation of focused microarray (Fig. <a reveal-id="DSQ025F1" data-open="DSQ025F1" class="link link-reveal link-table xref-fig">1</a>). We first designed the iGp array, examined its performance using Cy5-labelled synthetic oligonucleotides as hybridization probes (Fig. <a reveal-id="DSQ025F2" data-open="DSQ025F2" class="link link-reveal link-table xref-fig">2</a>A) and determined that it yielded highly reproducible data (Fig. <a reveal-id="DSQ025F2" data-open="DSQ025F2" class="link link-reveal link-table xref-fig">2</a>B; CV was 10.6% on average). We also prepared the aGp array and tested their clinical applicability using RNA samples from a small volume of PBMCs obtained from TA patients (Fig. <a reveal-id="DSQ025F3" data-open="DSQ025F3" class="link link-reveal link-table xref-fig">3</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). We do not exclude the usefulness of other genes that were not installed into the aGp array. Nonetheless, we showed that the aGp array is able to distinguish TA patient profiles from HVs and patients with other autoimmune diseases based on the Spearman rank correlation using all 216 probes (Fig. <a reveal-id="DSQ025F4" data-open="DSQ025F4" class="link link-reveal link-table xref-fig">4</a>). Notably, this data-processing protocol is quite simple and extensible; all that is needed is to calculate the ratio of the Spearman rank correlation for the patients’ and HV pool samples. This protocol is useful only for analysis of small number of genes and cannot be applied to conventional genome-wide microarrays, which is another advantageous point of Genopal™. Comparison of the data obtained by an aGp array, Agilent's microarray, qPCR and nCounter™ indicated that these methods exhibited similar expression profiles with regard to target genes with high signal intensities (Figs <a reveal-id="DSQ025F5  DSQ025F7" data-open="DSQ025F5  DSQ025F7" class="link link-reveal link-table xref-fig">5–7</a>).</p><p>Using aGp arrays, we also identified here over 10 genes that were up- or down-regulated in almost all TA patients tested (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Among the up-regulated genes, S100A8, S100A9 and S10012 are inflammation-related genes. This is not unexpected because TA is an autoimmune disease that involves enhanced inflammation of the aorta. The up-regulation of amphiregulin (AREG), which is a member of the epidermal growth factor family, may be related to the ventricular hypertrophy often observed in TA patients. Alternatively, it may be involved in the maintenance of inflammatory situations by inducing the proliferation of regulatory cells in these phenomena. Enhanced expression of IL-1 receptor, type II (IL-1R2), a decoy receptor that traps pro-inflammatory IL-1β and does not initiate subsequent signalling events, may be due to a negative feedback response to the hyperactive inflammation in the aorta of TA patients (which results partially from steroid therapy). Defensins, which include DEFA3, are a family of microbicidal and cytotoxic peptides that are abundant in the granules of neutrophils and are thought to be involved in host Defence, which suggest the involvement of infection in the pathogenesis of TA. In contrast, interferon gamma (IFN-G) and interferon-induced genes (G1P2, IFI44, IFIT1, IFIT2 and IFI27) were down-regulated in many TA patients, which distinguish TA from SLE and ITP, as these genes are up-regulated in the latter autoimmune diseases. As there is no reliable diagnostic tool for TA at present, these genes may also be useful as diagnostic markers of the disease when used in combination with the Spearman rank correlation method described above.</p><p>One of the practical applications of iGp and aGp arrays would be their use in the diagnosis of patients afflicted with fever of unknown origin (FUO). Our proposal is presented schematically in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>. Namely, a 2-ml blood sample may be collected for every new FUO patient using PAXgene Blood RNA Kits of Quiagen, total RNA prepared, labelled with Cy5 and applied to an aGp array. As the aGp array harbours genes that are either up- or down-regulated in the PBMCs of several autoimmune diseases, such as SLE,<sup><a reveal-id="DSQ025C11" data-open="DSQ025C11" class="link link-ref link-reveal xref-bibr">11</a></sup>  ITP,<sup><a reveal-id="DSQ025C11" data-open="DSQ025C11" class="link link-ref link-reveal xref-bibr">11</a></sup> RA<sup><a reveal-id="DSQ025C12" data-open="DSQ025C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and/or vasculitis,<sup><a reveal-id="DSQ025C13" data-open="DSQ025C13" class="link link-ref link-reveal xref-bibr">13</a></sup> we expect to observe positive signals if the patient's FUO is related to any of these autoimmune diseases. In this case, the FUO patient may be directly referred to an autoimmune disease specialist to receive appropriate care. If the screen yielded negative results, other sample sources, such as phlegm and/or excrement, depending on the dubious infectants, may be collected for RNA preparation and screening using the iGp array. If a positive signal is detected, the patients could then be treated appropriately at the bedside. Recent reports indicate that Genopal™ is useful not only for cRNA but also for miRNA samples.<sup><a reveal-id="DSQ025C16  DSQ025C20" data-open="DSQ025C16  DSQ025C20" class="link link-ref link-reveal xref-bibr">16–20</a></sup></p><p>Taken together, we conclude that Genopal™ is an advantageous platform for focused microarrays with regard to its low cost, reliability, ease of storage, speed and suitability for large-scale production. The data-processing protocol we developed here might be quite useful for the diseases that lack diagnostic tools using conventional clinical methods.</p>                            <h2 scrollto-destination=7365057 id="7365057" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7365061 id="7365061" class="section-title" data-legacy-id=s6>Authors’ contributions</h2>
<p>D.O. and T.F. designed and performed research, T.T. performed research on the infectant array, T.I. and S.K. provided TA patient samples and monitored the results, T.A. developed the Genopal™ system and H.N. designed the research and wrote the manuscript. K.Y. provided BD samples.</p>                            <h2 scrollto-destination=7365065 id="7365065" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported in part by grants-in-aid from the Bio-Medical Cluster Project In Saito, Innovation Plaza Osaka and the Regional Research and Development Resources Utilization Program of the Japan Science and Technology Agency (JST); and Scientific Research on Priority Areas Applied Genomics, Scientific Research (S), Exploratory Research and the Science and Technology Incubation Program in Advanced Regions from the Ministry of Education, Culture, Sports, Science and Technology of Japan to H.N.</p>                            <h2 scrollto-destination=7365070 id="7365070" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr Claudia Gaspar of Bioedit Ltd. for critically reading this manuscript. We also thank Dr Akio Tanabe of Subio Inc. for technical advice in data analysis, Ms Maki Fukuda and Ms Chiharu Nakashima of our laboratory for technical assistance in microarray experiments, Mr Kazuhiko Yuyama of Gene Design Inc. for the synthesis of Genopal™ probes and Dr Nathan Elliott of NanoString Technologies Inc. (Seattle, WA, USA) for the nCounter™ analysis</p>                            <h2 scrollto-destination=7365074 id="7365074" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ025C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wurmbach</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Yuen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sealfon</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Focused microarray analysis</div>, <div class="source">Methods</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">306</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Focused%20microarray%20analysis&amp;author=E.%20Wurmbach&amp;author=T.%20Yuen&amp;author=S.C.%20Sealfon&amp;publication_year=2003&amp;journal=Methods&amp;volume=31&amp;pages=306-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1046-2023(03)00161-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1046-2023(03)00161-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1046-2023(03)00161-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">A.X.</div></div>,  <div class="name"><div class="surname">Mejido</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Development of a focused microarray to assess human embryonic stem cell differentiation</div>, <div class="source">Stem Cells Dev.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">270</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20focused%20microarray%20to%20assess%20human%20embryonic%20stem%20cell%20differentiation&amp;author=A.X.%20Yang&amp;author=J.%20Mejido&amp;author=Y.%20Luo&amp;publication_year=2005&amp;journal=Stem%20Cells%20Dev.&amp;volume=14&amp;pages=270-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/scd.2005.14.270" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fscd.2005.14.270" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fscd.2005.14.270"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Schwartz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rao</div> <div class="given-names">M.S.</div></div>, et al. </div><div class="article-title">A focused microarray to assess dopaminergic and glial cell differentiation from fetal tissue or embryonic stem cells</div>, <div class="source">Stem Cells</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">865</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20focused%20microarray%20to%20assess%20dopaminergic%20and%20glial%20cell%20differentiation%20from%20fetal%20tissue%20or%20embryonic%20stem%20cells&amp;author=Y.%20Luo&amp;author=C.%20Schwartz&amp;author=M.S.%20Rao&amp;publication_year=2006&amp;journal=Stem%20Cells&amp;volume=24&amp;pages=865-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1634/stemcells.2005-0392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1634%2Fstemcells.2005-0392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1634%2Fstemcells.2005-0392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16357341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Comelli</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Head</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Gilmartin</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A focused microarray approach to functional glycomics: transcriptional regulation of the glycome</div>, <div class="source">Glycobiology</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">117</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20focused%20microarray%20approach%20to%20functional%20glycomics%3A%20transcriptional%20regulation%20of%20the%20glycome&amp;author=E.M.%20Comelli&amp;author=S.R.%20Head&amp;author=T.%20Gilmartin&amp;publication_year=2006&amp;journal=Glycobiology&amp;volume=16&amp;pages=117-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/glycob/cwj048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fglycob%2Fcwj048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fglycob%2Fcwj048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16237199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kroes</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Dawson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Moskal</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Focused microarray analysis of glyco-gene expression in human glioblastomas</div>, <div class="source">J. Neurochem.</div> , <div class="year">2007</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">14</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Focused%20microarray%20analysis%20of%20glyco-gene%20expression%20in%20human%20glioblastomas&amp;author=R.A.%20Kroes&amp;author=G.%20Dawson&amp;author=J.R.%20Moskal&amp;publication_year=2007&amp;journal=J.%20Neurochem.&amp;volume=103&amp;pages=14-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jnc.2007.103.issue-s1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjnc.2007.103.issue-s1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjnc.2007.103.issue-s1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voss</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Raju</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Logun</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A focused microarray to study human mitochondrial and nuclear gene expression</div>, <div class="source">Biol. Res. Nurs.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">272</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20focused%20microarray%20to%20study%20human%20mitochondrial%20and%20nuclear%20gene%20expression&amp;author=J.G.%20Voss&amp;author=R.%20Raju&amp;author=C.%20Logun&amp;publication_year=2008&amp;journal=Biol.%20Res.%20Nurs.&amp;volume=9&amp;pages=272-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/1099800408315160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F1099800408315160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F1099800408315160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18398222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parveen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Inoue</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ise</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tanji</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kiyama</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Evaluation of estrogenic activity of phthalate esters by gene expression profiling using a focused microarray (EstrArray)</div>, <div class="source">Environ. Toxicol. Chem.</div> , <div class="year">2008</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1416</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20estrogenic%20activity%20of%20phthalate%20esters%20by%20gene%20expression%20profiling%20using%20a%20focused%20microarray%20(EstrArray)&amp;author=M.%20Parveen&amp;author=A.%20Inoue&amp;author=R.%20Ise&amp;author=M.%20Tanji&amp;author=R.%20Kiyama&amp;publication_year=2008&amp;journal=Environ.%20Toxicol.%20Chem.&amp;volume=27&amp;pages=1416-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1897/07-399.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1897%2F07-399.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1897%2F07-399.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18260696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanney</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">G.R.</div></div>,  <div class="name"><div class="surname">Farztdinov</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Generation of a non-small cell lung cancer transcriptome microarray</div>, <div class="source">BMC Med. Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> pg. <div class="fpage">20</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20a%20non-small%20cell%20lung%20cancer%20transcriptome%20microarray&amp;author=A.%20Tanney&amp;author=G.R.%20Oliver&amp;author=V.%20Farztdinov&amp;publication_year=2008&amp;journal=BMC%20Med.%20Genomics&amp;volume=1&amp;pages=20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1755-8794-1-20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1755-8794-1-20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1755-8794-1-20"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18513400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogino</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Matsuda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Minatoya</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Overview of late outcome of medical and surgical treatment for Takayasu arteritis</div>, <div class="source">Circulation</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">2738</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overview%20of%20late%20outcome%20of%20medical%20and%20surgical%20treatment%20for%20Takayasu%20arteritis&amp;author=H.%20Ogino&amp;author=H.%20Matsuda&amp;author=K.%20Minatoya&amp;publication_year=2008&amp;journal=Circulation&amp;volume=18&amp;pages=2738-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1161/CIRCULATIONAHA.107.759589" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1161%2FCIRCULATIONAHA.107.759589" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1161%2FCIRCULATIONAHA.107.759589"> </span></div> </div></div></div></div></div><div content-id="DSQ025C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bleeker-Rovers</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">van der Meer</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Oyen</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">Fever of unknown origin</div>, <div class="source">Semin. Nucl. Med.</div> , <div class="year">2009</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">81</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fever%20of%20unknown%20origin&amp;author=C.P.%20Bleeker-Rovers&amp;author=J.W.%20van%20der%20Meer&amp;author=W.J.%20Oyen&amp;publication_year=2009&amp;journal=Semin.%20Nucl.%20Med.&amp;volume=39&amp;pages=81-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1053/j.semnuclmed.2008.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1053%2Fj.semnuclmed.2008.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1053%2Fj.semnuclmed.2008.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19187801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Onda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tanigawa</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Isolation and expression profiling of genes upregulated in the peripheral blood cells of systemic lupus erythematosus patients</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">429</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20expression%20profiling%20of%20genes%20upregulated%20in%20the%20peripheral%20blood%20cells%20of%20systemic%20lupus%20erythematosus%20patients&amp;author=T.%20Ishii&amp;author=H.%20Onda&amp;author=A.%20Tanigawa&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=429-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shimaoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tougan</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Isolation and expression profiling of genes upregulated in bone marrow-derived mononuclear cells of rheumatoid arthritis patients</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">169</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20expression%20profiling%20of%20genes%20upregulated%20in%20bone%20marrow-derived%20mononuclear%20cells%20of%20rheumatoid%20arthritis%20patients&amp;author=N.%20Nakamura&amp;author=Y.%20Shimaoka&amp;author=T.%20Tougan&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=169-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17082220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Okuzaki</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Nojima</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Expression profiling of PBMC-based diagnostic gene markers isolated from vasculitis patients</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">253</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20profiling%20of%20PBMC-based%20diagnostic%20gene%20markers%20isolated%20from%20vasculitis%20patients&amp;author=S.%20Kobayashi&amp;author=D.%20Okuzaki&amp;author=H.%20Nojima&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=253-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18562305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tougan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Onda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Okuzaki</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hashimoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nojima</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Focused microarray analysis of peripheral mononuclear blood cells from Churg-Strauss syndrome patients</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">103</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Focused%20microarray%20analysis%20of%20peripheral%20mononuclear%20blood%20cells%20from%20Churg-Strauss%20syndrome%20patients&amp;author=T.%20Tougan&amp;author=H.%20Onda&amp;author=D.%20Okuzaki&amp;author=S.%20Kobayashi&amp;author=H.%20Hashimoto&amp;author=H.%20Nojima&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=103-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18263571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Geiss</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Bumgarner</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Birditt</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Direct multiplexed measurement of gene expression with color-coded probe pairs</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">317</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20multiplexed%20measurement%20of%20gene%20expression%20with%20color-coded%20probe%20pairs&amp;author=G.K.%20Geiss&amp;author=R.E.%20Bumgarner&amp;author=B.%20Birditt&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=317-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1385" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1385" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1385"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hohjoh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fukushima</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Marked change in microRNA expression during neuronal differentiation of human teratocarcinoma NTera2D1 and mouse embryonal carcinoma P19 cells</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2007</div>, vol. <div class="volume">362</div> (pg. <div class="fpage">360</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Marked%20change%20in%20microRNA%20expression%20during%20neuronal%20differentiation%20of%20human%20teratocarcinoma%20NTera2D1%20and%20mouse%20embryonal%20carcinoma%20P19%20cells&amp;author=H.%20Hohjoh&amp;author=T.%20Fukushima&amp;publication_year=2007&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=362&amp;pages=360-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2007.07.189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2007.07.189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2007.07.189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17716626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">N.S.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">W.J.</div></div>, et al. </div><div class="article-title">miR-302b maintains “stemness” of human embryonal carcinoma cells by post-transcriptional regulation of Cyclin D2 expression</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2008</div>, vol. <div class="volume">377</div> (pg. <div class="fpage">434</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miR-302b%20maintains%20%E2%80%9Cstemness%E2%80%9D%20of%20human%20embryonal%20carcinoma%20cells%20by%20post-transcriptional%20regulation%20of%20Cyclin%20D2%20expression&amp;author=N.S.%20Lee&amp;author=J.S.%20Kim&amp;author=W.J.%20Cho&amp;publication_year=2008&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=377&amp;pages=434-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2008.09.159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2008.09.159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2008.09.159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18930031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fukushima</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Bitter gourd suppresses lipopolysaccharide-induced inflammatory responses</div>, <div class="source">J. Agric. Food. Chem.</div> , <div class="year">2008</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">4004</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bitter%20gourd%20suppresses%20lipopolysaccharide-induced%20inflammatory%20responses&amp;author=M.%20Kobori&amp;author=H.%20Nakayama&amp;author=K.%20Fukushima&amp;publication_year=2008&amp;journal=J.%20Agric.%20Food.%20Chem.&amp;volume=56&amp;pages=4004-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf800052y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf800052y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf800052y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18489106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tojo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Gene-expression profiles of peripheral blood mononuclear cell subpopulations in acute graft-vs-host disease following cord blood transplantation</div>, <div class="source">Exp. Hematol.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">1760</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene-expression%20profiles%20of%20peripheral%20blood%20mononuclear%20cell%20subpopulations%20in%20acute%20graft-vs-host%20disease%20following%20cord%20blood%20transplantation&amp;author=N.%20Takahashi&amp;author=N.%20Sato&amp;author=S.%20Takahashi&amp;author=A.%20Tojo&amp;publication_year=2008&amp;journal=Exp.%20Hematol.&amp;volume=36&amp;pages=1760-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.exphem.2008.07.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.exphem.2008.07.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.exphem.2008.07.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18814951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ025C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ025C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oue</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sentani</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Characteristic gene expression in stromal cells of gastric cancers among atomic-bomb survivors</div>, <div class="source">Int. J. Cancer</div> , <div class="year">2009</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1112</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characteristic%20gene%20expression%20in%20stromal%20cells%20of%20gastric%20cancers%20among%20atomic-bomb%20survivors&amp;author=N.%20Oue&amp;author=K.%20Sentani&amp;author=N.%20Sakamoto&amp;publication_year=2009&amp;journal=Int.%20J.%20Cancer&amp;volume=124&amp;pages=1112-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ijc.v124:5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fijc.v124:5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fijc.v124:5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19058179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484698&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq025/2/dsq025-dsq025supp.pdf?Expires=2147483647&amp;Signature=H86C1Haz6W~xcsBTXwCMi~ME~6kD4zi-pQfsmaNq~qUzkULumnTbnqyW5oL2TFl-TDDZVSsOoA70PQ7aWDPD-cS3f4uLFut9fQ5nhBVZxzdBZC4Ll5HOHWmQlnplVDu1pfINtkpvNOJ5-erch93Czq0ltvMnDco1kzqPWFq0-m2hgp-3bLr2ZEqWOgN79hsB9olo7whq27pyXvLOPLpS4epkmbs9Hp69qVW8zrttVOVILhj7H7vScqEqeS52UkcKMlqPuiw61~E40E7EtVchR4g6NCowR37AsrcS4lW77cmN1YKnpQQ5aEB0bg6sqJMocxbCmQjvTaiP~7~H329TpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">119</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq025"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484698"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f6%2f369%2f484698">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000285800200004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F6%2F369%2F484698" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484698'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genopal™: A Novel Hollow Fibre Array for Focused Microarray Analysis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq025");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f369%2f484698">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

