

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/4/221/501057","@type":"ScholarlyArticle","name":"An EST-SSR Linkage Map of  Raphanus sativus  and Comparative Genomics of the Brassicaceae  †  ","datePublished":"2011-06-13","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2011-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr013","keywords":["comparative map","expressed sequence tag (EST)","genetic linkage map","<italic>Raphanus sativus</italic>","simple sequence repeat (SSR)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/4/221/501057","author":[{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Oyama, Maki","affiliation":"2  Mikado Kyowa Seed Co., Ltd. ,  2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujioka, Takashi","affiliation":"2  Mikado Kyowa Seed Co., Ltd. ,  2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202 ,  Japan","@type":"Person"},{"name":"Kimizuka-Takagi, Chiaki","affiliation":"3  Kimizuka Vegetable Farm ,  1023 Funako, Otaki, Isumi, Chiba 297-0204 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kato, Midori","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Takahashi, Chika","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tsuruoka, Hisano","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Wada, Tsuyuko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sakai, Takako","affiliation":"2  Mikado Kyowa Seed Co., Ltd. ,  2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Raphanus sativus (2n = 2x = 18) is a widely cultivated member of the family Brassicaceae, for which genomic resources are available only to a limited","pageStart":"221","pageEnd":"232","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"An EST-SSR Linkage Map of  Raphanus sativus  and Comparative Genomics of the Brassicaceae  †  ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="EST-SSR Linkage Map of  Raphanus sativus  and Comparative Genomics of the Brassicaceae  †  " />
    <meta property="og:description" content="Abstract.  Raphanus sativus (2n = 2x = 18) is a widely cultivated member of the family Brassicaceae, for which genomic resources are available only to a limited" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/4/221/501057" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-06-13" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Oyama, Maki" /><meta name="citation_author_institution" content="Mikado Kyowa Seed Co., Ltd., 2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujioka, Takashi" /><meta name="citation_author_institution" content="Mikado Kyowa Seed Co., Ltd., 2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202, Japan" /><meta name="citation_author" content="Kimizuka-Takagi, Chiaki" /><meta name="citation_author_institution" content="Kimizuka Vegetable Farm, 1023 Funako, Otaki, Isumi, Chiba 297-0204, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kato, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Takahashi, Chika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tsuruoka, Hisano" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Wada, Tsuyuko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sakai, Takako" /><meta name="citation_author_institution" content="Mikado Kyowa Seed Co., Ltd., 2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="An EST-SSR Linkage Map of Raphanus sativus and Comparative Genomics of the Brassicaceae" /><meta name="citation_firstpage" content="221" /><meta name="citation_lastpage" content="232" /><meta name="citation_doi" content="10.1093/dnares/dsr013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2011/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=468" /><meta name="citation_reference" content="citation_title=High-throughput genotyping with the GoldenGate assay in the complex genome of soybean; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=945-52; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and assay development in common bean; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=475" /><meta name="citation_reference" content="citation_title=A consensus genetic map of cowpea [Vigna unguiculata (L) Walp.] and synteny based on EST-derived SNPs; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=18159-64; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Microsatellite markers: an overview of the recent progress in plants; Euphytica;  citation_year=2011;  citation_volume=117;  citation_pages=309-34; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=The Arabidopsis Genome Initiative; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=Genome-analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization; Jpn. J. Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=Comparative mapping between Arabidopsis thaliana and Brassica nigra indicates that Brassica genomes have evolved through extensive genome replication accompanied by chromosome fusions and frequent rearrangements; Genetics;  citation_year=1998;  citation_volume=150;  citation_pages=1217-28; " /><meta name="citation_reference" content="citation_title=An EST-enriched comparative map of Brassica oleracea and Arabidopsis thaliana; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=776-88; " /><meta name="citation_reference" content="citation_title=A sequence-tagged linkage map of Brassica rapa; Genetics;  citation_year=2006;  citation_volume=174;  citation_pages=29-39; " /><meta name="citation_reference" content="citation_title=A Brassica rapa linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=311-23; " /><meta name="citation_reference" content="citation_title=Simple sequence repeat-based comparative genomics between Brassica rapa and Arabidopsis thaliana: the genetic origin of clubroot resistance; Genetics;  citation_year=2006;  citation_volume=173;  citation_pages=309-19; " /><meta name="citation_reference" content="citation_title=Radish; citation_publisher=Springer, New York; Genome Mapping and Molecular Breeding in Plants;  citation_year=2007;  citation_volume=vol. 5;  citation_pages=141-60; " /><meta name="citation_reference" content="citation_title=Cultivars of radish and their change; citation_text=(in Japanese); citation_publisher=Japan Society for the Promotion of Science, Tokyo; Japanese Radish;  citation_year=1958;  citation_pages=1-19; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content of some important plant; Plant Mol. Biol. Rep.;  citation_year=1991;  citation_volume=9;  citation_pages=208-18; " /><meta name="citation_reference" content="citation_title=Molecular mapping in oil radish (Raphanus sativus L.) and QTL analysis of resistance against beet cyst nematode (Heterodera schachtii); Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=775-82; " /><meta name="citation_reference" content="citation_title=QTL mapping of clubroot resistance in radish (Raphanus sativus L.); Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=1021-7; " /><meta name="citation_reference" content="citation_title=Mapping of QTLs controlling root shape and red pigmentation in radish, Raphanus sativus L.; Breed. Sci.;  citation_year=2008;  citation_volume=58;  citation_pages=55-61; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover (Trifolium pratense L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-64; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=An interspecific linkage map of SSR and intronic polymorphism markers in tomato; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=731-9; " /><meta name="citation_reference" content="citation_title=Map positions of 47 Arabidopsis sequences with sequence similarity to disease resistance genes; Plant J.;  citation_year=1997;  citation_volume=12;  citation_pages=1197-211; " /><meta name="citation_reference" content="citation_title=From Arabidopsis thaliana to Brassica napus: development of amplified consensus genetic markers (ACGM) for construction of a gene map; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=105;  citation_pages=1196-206; " /><meta name="citation_reference" content="citation_title=JoinMap®4; citation_publisher=Kyazma, B.V, Wageningen, The Netherlands; Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Genetic analysis and genome mapping in Raphanus; Genome;  citation_year=2003;  citation_volume=46;  citation_pages=423-30; " /><meta name="citation_reference" content="citation_title=Construction of a molecular linkage map of radish (Raphanus sativus L.), based on AFLP and Brassica-SSR markers; Breed. Sci.;  citation_year=2005;  citation_volume=55;  citation_pages=107-11; " /><meta name="citation_reference" content="citation_title=Megagametophyte-derived linkage maps of white spruce (Picea glauca) based on RAPD, SCAR and ESTP markers; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=987-97; " /><meta name="citation_reference" content="citation_title=Evidence for genetic restriction of recombination in the lima bean; Genetics;  citation_year=1963;  citation_volume=48;  citation_pages=1389-95; " /><meta name="citation_reference" content="citation_title=Amphidiploid Brassica juncea contains conserved progenitor genomes; Genome;  citation_year=2000;  citation_volume=43;  citation_pages=679-88; " /><meta name="citation_reference" content="citation_title=Rates of nucleotide substitution in angiosperm mitochondrial DNA sequences and dates of divergence between Brassica and other angiosperm lineages; J. Mol. Evol.;  citation_year=1999;  citation_volume=48;  citation_pages=597-604; " /><meta name="citation_reference" content="citation_title=Comparative evolutionary analysis of chalcone synthase and alcohol dehydrogenase loci in Arabidopsis, Arabis, and related genera (Brassicaceae); Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1483-98; " /><meta name="citation_reference" content="citation_title=Poliploid hybrids Raphanus sativus L. times Brassica oleracea L.; Bull. Appl. Bot.;  citation_year=1927;  citation_volume=17;  citation_pages=305-410; " /><meta name="citation_reference" content="citation_title=Cytogenetics and genome analysis in Brassica crops; Chromosome Res.;  citation_year=2007;  citation_volume=15;  citation_pages=85-95; " /><meta name="citation_reference" content="citation_title=Genetic control of the cytologically diploid behaviour of hexaploid wheat; Nature;  citation_year=1959;  citation_volume=182;  citation_pages=713-5; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of NBS-LRR-encoding genes in Arabidopsis; Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=809-34; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/4/221/1349046/dsr013.pdf" /><meta name="description" content="Abstract.  Raphanus sativus (2n = 2x = 18) is a widely cultivated member of the family Brassicaceae, for which genomic resources are available only to a limited" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/4/221/501057" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/4/221/501057" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/4/221/501057");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr013';
    </script>
<title>
	EST-SSR Linkage Map of Raphanus sativus and Comparative Genomics of the Brassicaceae† | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./501057" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BatkPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlyYXNhd2EsIEtlbnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik95YW1hLCBNYWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWlrYWRvIEt5b3dhIFNlZWQgQ28uLCBMdGQuLCAyNzg5LTEgU2hpbW8tT3Rha2ksIE90YWtpLCBJc3VtaSwgQ2hpYmEgMjk4LTAyMDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1amlva2EsIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWthZG8gS3lvd2EgU2VlZCBDby4sIEx0ZC4sIDI3ODktMSBTaGltby1PdGFraSwgT3Rha2ksIElzdW1pLCBDaGliYSAyOTgtMDIwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW1penVrYS1UYWthZ2ksIENoaWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktpbWl6dWthIFZlZ2V0YWJsZSBGYXJtLCAxMDIzIEZ1bmFrbywgT3Rha2ksIElzdW1pLCBDaGliYSAyOTctMDIwNCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXNhbW90bywgU2hpZ2VtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgTWlkb3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpc2hpZGEsIFlvc2hpZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hhcmEsIE1pdHN1eW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWhhc2hpLCBDaGlrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VydW9rYSwgSGlzYW5vIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhZGEsIFRzdXl1a28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrYWksIFRha2FrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pa2FkbyBLeW93YSBTZWVkIENvLiwgTHRkLiwgMjc4OS0xIFNoaW1vLU90YWtpLCBPdGFraSwgSXN1bWksIENoaWJhIDI5OC0wMjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklzb2JlLCBTYWNoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQW4gRVNULVNTUiBMaW5rYWdlIE1hcCBvZiBSYXBoYW51cyBzYXRpdnVzIGFuZCBDb21wYXJhdGl2ZSBHZW5vbWljcyBvZiB0aGUgQnJhc3NpY2FjZWFlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseS1tdWx0aXBsZXhlZCBTTlAgZ2Vub3R5cGluZyBmb3IgZ2VuZXRpYyBtYXBwaW5nIGFuZCBnZXJtcGxhc20gZGl2ZXJzaXR5IHN0dWRpZXMgaW4gcGVhOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00NjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgZ2Vub3R5cGluZyB3aXRoIHRoZSBHb2xkZW5HYXRlIGFzc2F5IGluIHRoZSBjb21wbGV4IGdlbm9tZSBvZiBzb3liZWFuOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNjsgIGNpdGF0aW9uX3BhZ2VzPTk0NS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGFzc2F5IGRldmVsb3BtZW50IGluIGNvbW1vbiBiZWFuOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbnNlbnN1cyBnZW5ldGljIG1hcCBvZiBjb3dwZWEgW1ZpZ25hIHVuZ3VpY3VsYXRhIChMKSBXYWxwLl0gYW5kIHN5bnRlbnkgYmFzZWQgb24gRVNULWRlcml2ZWQgU05QczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTgxNTktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkaXNjb3ZlcnkgYW5kIGxpbmthZ2UgbWFwIGNvbnN0cnVjdGlvbiBpbiBjdWx0aXZhdGVkIHRvbWF0bzsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zODEtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnM6IGFuIG92ZXJ2aWV3IG9mIHRoZSByZWNlbnQgcHJvZ3Jlc3MgaW4gcGxhbnRzOyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9MzA5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBmbG93ZXJpbmcgcGxhbnQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IGNpdGF0aW9uX2F1dGhvcj1UaGUgQXJhYmlkb3BzaXMgR2Vub21lIEluaXRpYXRpdmU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYtODE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtYW5hbHlzaXMgaW4gQnJhc3NpY2Egd2l0aCBzcGVjaWFsIHJlZmVyZW5jZSB0byB0aGUgZXhwZXJpbWVudGFsIGZvcm1hdGlvbiBvZiBCLiBuYXB1cyBhbmQgcGVjdWxpYXIgbW9kZSBvZiBmZXJ0aWxpemF0aW9uOyBKcG4uIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTE5MzU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00NTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1hcHBpbmcgYmV0d2VlbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBhbmQgQnJhc3NpY2EgbmlncmEgaW5kaWNhdGVzIHRoYXQgQnJhc3NpY2EgZ2Vub21lcyBoYXZlIGV2b2x2ZWQgdGhyb3VnaCBleHRlbnNpdmUgZ2Vub21lIHJlcGxpY2F0aW9uIGFjY29tcGFuaWVkIGJ5IGNocm9tb3NvbWUgZnVzaW9ucyBhbmQgZnJlcXVlbnQgcmVhcnJhbmdlbWVudHM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTEyMTctMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIEVTVC1lbnJpY2hlZCBjb21wYXJhdGl2ZSBtYXAgb2YgQnJhc3NpY2Egb2xlcmFjZWEgYW5kIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTc3Ni04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzZXF1ZW5jZS10YWdnZWQgbGlua2FnZSBtYXAgb2YgQnJhc3NpY2EgcmFwYTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTc0OyAgY2l0YXRpb25fcGFnZXM9MjktMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgQnJhc3NpY2EgcmFwYSBsaW5rYWdlIG1hcCBvZiBFU1QtYmFzZWQgU05QIG1hcmtlcnMgZm9yIGlkZW50aWZpY2F0aW9uIG9mIGNhbmRpZGF0ZSBnZW5lcyBjb250cm9sbGluZyBmbG93ZXJpbmcgdGltZSBhbmQgbGVhZiBtb3JwaG9sb2dpY2FsIHRyYWl0czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzZXF1ZW5jZSByZXBlYXQtYmFzZWQgY29tcGFyYXRpdmUgZ2Vub21pY3MgYmV0d2VlbiBCcmFzc2ljYSByYXBhIGFuZCBBcmFiaWRvcHNpcyB0aGFsaWFuYTogdGhlIGdlbmV0aWMgb3JpZ2luIG9mIGNsdWJyb290IHJlc2lzdGFuY2U7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3MzsgIGNpdGF0aW9uX3BhZ2VzPTMwOS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFkaXNoOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrOyBHZW5vbWUgTWFwcGluZyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nIGluIFBsYW50czsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT12b2wuIDU7ICBjaXRhdGlvbl9wYWdlcz0xNDEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1bHRpdmFycyBvZiByYWRpc2ggYW5kIHRoZWlyIGNoYW5nZTsgY2l0YXRpb25fdGV4dD0oaW4gSmFwYW5lc2UpOyBjaXRhdGlvbl9wdWJsaXNoZXI9SmFwYW4gU29jaWV0eSBmb3IgdGhlIFByb21vdGlvbiBvZiBTY2llbmNlLCBUb2t5bzsgSmFwYW5lc2UgUmFkaXNoOyAgY2l0YXRpb25feWVhcj0xOTU4OyAgY2l0YXRpb25fcGFnZXM9MS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCBvZiBzb21lIGltcG9ydGFudCBwbGFudDsgUGxhbnQgTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMDgtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBtYXBwaW5nIGluIG9pbCByYWRpc2ggKFJhcGhhbnVzIHNhdGl2dXMgTC4pIGFuZCBRVEwgYW5hbHlzaXMgb2YgcmVzaXN0YW5jZSBhZ2FpbnN0IGJlZXQgY3lzdCBuZW1hdG9kZSAoSGV0ZXJvZGVyYSBzY2hhY2h0aWkpOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTc3NS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIG1hcHBpbmcgb2YgY2x1YnJvb3QgcmVzaXN0YW5jZSBpbiByYWRpc2ggKFJhcGhhbnVzIHNhdGl2dXMgTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBvZiBRVExzIGNvbnRyb2xsaW5nIHJvb3Qgc2hhcGUgYW5kIHJlZCBwaWdtZW50YXRpb24gaW4gcmFkaXNoLCBSYXBoYW51cyBzYXRpdnVzIEwuOyBCcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTU1LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHN0cnVjdHVyYWwgYW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBvZiByZWQgY2xvdmVyIChUcmlmb2xpdW0gcHJhdGVuc2UgTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMwMS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENPRyBkYXRhYmFzZTogYW4gdXBkYXRlZCB2ZXJzaW9uIGluY2x1ZGVzIGV1a2FyeW90ZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlcnNwZWNpZmljIGxpbmthZ2UgbWFwIG9mIFNTUiBhbmQgaW50cm9uaWMgcG9seW1vcnBoaXNtIG1hcmtlcnMgaW4gdG9tYXRvOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXAgcG9zaXRpb25zIG9mIDQ3IEFyYWJpZG9wc2lzIHNlcXVlbmNlcyB3aXRoIHNlcXVlbmNlIHNpbWlsYXJpdHkgdG8gZGlzZWFzZSByZXNpc3RhbmNlIGdlbmVzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTExOTctMjExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gcm9tIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHRvIEJyYXNzaWNhIG5hcHVzOiBkZXZlbG9wbWVudCBvZiBhbXBsaWZpZWQgY29uc2Vuc3VzIGdlbmV0aWMgbWFya2VycyAoQUNHTSkgZm9yIGNvbnN0cnVjdGlvbiBvZiBhIGdlbmUgbWFwOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTExOTYtMjA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Kb2luTWFwwq40OyBjaXRhdGlvbl9wdWJsaXNoZXI9S3lhem1hLCBCLlYsIFdhZ2VuaW5nZW4sIFRoZSBOZXRoZXJsYW5kczsgU29mdHdhcmUgZm9yIHRoZSBDYWxjdWxhdGlvbiBvZiBHZW5ldGljIExpbmthZ2UgTWFwcyBpbiBFeHBlcmltZW50YWwgUG9wdWxhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgYW5kIGdlbm9tZSBtYXBwaW5nIGluIFJhcGhhbnVzOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz00MjMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIG1vbGVjdWxhciBsaW5rYWdlIG1hcCBvZiByYWRpc2ggKFJhcGhhbnVzIHNhdGl2dXMgTC4pLCBiYXNlZCBvbiBBRkxQIGFuZCBCcmFzc2ljYS1TU1IgbWFya2VyczsgQnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0xMDctMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZ2FnYW1ldG9waHl0ZS1kZXJpdmVkIGxpbmthZ2UgbWFwcyBvZiB3aGl0ZSBzcHJ1Y2UgKFBpY2VhIGdsYXVjYSkgYmFzZWQgb24gUkFQRCwgU0NBUiBhbmQgRVNUUCBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTk4Ny05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZm9yIGdlbmV0aWMgcmVzdHJpY3Rpb24gb2YgcmVjb21iaW5hdGlvbiBpbiB0aGUgbGltYSBiZWFuOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk2MzsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEzODktOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtcGhpZGlwbG9pZCBCcmFzc2ljYSBqdW5jZWEgY29udGFpbnMgY29uc2VydmVkIHByb2dlbml0b3IgZ2Vub21lczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9Njc5LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXRlcyBvZiBudWNsZW90aWRlIHN1YnN0aXR1dGlvbiBpbiBhbmdpb3NwZXJtIG1pdG9jaG9uZHJpYWwgRE5BIHNlcXVlbmNlcyBhbmQgZGF0ZXMgb2YgZGl2ZXJnZW5jZSBiZXR3ZWVuIEJyYXNzaWNhIGFuZCBvdGhlciBhbmdpb3NwZXJtIGxpbmVhZ2VzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NTk3LTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIGNoYWxjb25lIHN5bnRoYXNlIGFuZCBhbGNvaG9sIGRlaHlkcm9nZW5hc2UgbG9jaSBpbiBBcmFiaWRvcHNpcywgQXJhYmlzLCBhbmQgcmVsYXRlZCBnZW5lcmEgKEJyYXNzaWNhY2VhZSk7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNDgzLTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2xpcGxvaWQgaHlicmlkcyBSYXBoYW51cyBzYXRpdnVzIEwuIHRpbWVzIEJyYXNzaWNhIG9sZXJhY2VhIEwuOyBCdWxsLiBBcHBsLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTI3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzA1LTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b2dlbmV0aWNzIGFuZCBnZW5vbWUgYW5hbHlzaXMgaW4gQnJhc3NpY2EgY3JvcHM7IENocm9tb3NvbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGNvbnRyb2wgb2YgdGhlIGN5dG9sb2dpY2FsbHkgZGlwbG9pZCBiZWhhdmlvdXIgb2YgaGV4YXBsb2lkIHdoZWF0OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5NTk7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9NzEzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIE5CUy1MUlItZW5jb2RpbmcgZ2VuZXMgaW4gQXJhYmlkb3BzaXM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz04MDktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOC80LzIyMS8xMzQ5MDQ2L2RzcjAxMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUmFwaGFudXMgc2F0aXZ1cyAoMm4gPSAyeCA9IDE4KSBpcyBhIHdpZGVseSBjdWx0aXZhdGVkIG1lbWJlciBvZiB0aGUgZmFtaWx5IEJyYXNzaWNhY2VhZSwgZm9yIHdoaWNoIGdlbm9taWMgcmVzb3VyY2VzIGFyZSBhdmFpbGFibGUgb25seSB0byBhIGxpbWl0ZWQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwMTA1NyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRcjH7gp3peWvmsxPqXlbyomxODnQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f221%2f501057" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f221%2f501057" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19820" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&amp;Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/4">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7725038">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725038">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725040">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725040">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725045">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725045">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725056">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725056">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725074">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725074">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725082">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725082">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725084">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725084">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725086">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725086">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7725088">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7725088">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/4/211/500841">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/4/233/501892">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">An EST-SSR Linkage Map of <em>Raphanus sativus</em> and Comparative Genomics of the Brassicaceae<sup><a reveal-id="AN1" data-open="AN1" class="link link-ref link-reveal xref-fn">†</a></sup>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. +81 438-52-3934. Email: <a href="mailto:shirasaw@kazusa.or.jp" target="_blank">shirasaw@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maki Oyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Mikado Kyowa Seed Co., Ltd.</div>, <div class="addr-line">2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maki+Oyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oyama M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oyama M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Maki Oyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Fujioka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Mikado Kyowa Seed Co., Ltd.</div>, <div class="addr-line">2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Fujioka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujioka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujioka T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takashi Fujioka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiaki Kimizuka-Takagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Kimizuka Vegetable Farm</div>, <div class="addr-line">1023 Funako, Otaki, Isumi, Chiba 297-0204</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiaki+Kimizuka-Takagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kimizuka-Takagi C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kimizuka-Takagi C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chiaki Kimizuka-Takagi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Midori+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Midori Kato</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chika Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chika+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chika Takahashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisano Tsuruoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisano+Tsuruoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuruoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuruoka H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hisano Tsuruoka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyuko Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyuko+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsuyuko Wada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takako Sakai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Mikado Kyowa Seed Co., Ltd.</div>, <div class="addr-line">2789-1 Shimo-Otaki, Otaki, Isumi, Chiba 298-0202</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takako+Sakai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakai T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakai T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Takako Sakai</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Information for the EST-SSR markers and the genetic map is available at <a class="link link-uri" href="http://marker.kazusa.or.jp/Daikon">http://marker.kazusa.or.jp/Daikon</a>.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 221–232, <a href='https://doi.org/10.1093/dnares/dsr013'>https://doi.org/10.1093/dnares/dsr013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">13 June 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 April 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 May 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="501057" href="/dnaresearch/article-pdf/18/4/221/1349046/dsr013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenta Shirasawa, Maki Oyama, Hideki Hirakawa, Shusei Sato, Satoshi Tabata, Takashi Fujioka, Chiaki Kimizuka-Takagi, Shigemi Sasamoto, Akiko Watanabe, Midori Kato, Yoshie Kishida, Mitsuyo Kohara, Chika Takahashi, Hisano Tsuruoka, Tsuyuko Wada, Takako Sakai, Sachiko Isobe;  An EST-SSR Linkage Map of <em>Raphanus sativus</em> and Comparative Genomics of the Brassicaceae<sup></sup>, <em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 221–232, <a href='https://doi.org/10.1093/dnares/dsr013'>https://doi.org/10.1093/dnares/dsr013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/501057?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/501057?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/501057?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/501057?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/501057?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/501057?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/501057?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=An%20EST-SSR%20Linkage%20Map%20of%20%26lt%3Bem%26gt%3BRaphanus%20sativus%26lt%3B%2Fem%26gt%3B%20and%20Comparative%20Genomics%20of%20the%20Brassicaceae%26lt%3Bsup%26gt%3B%26lt%3Ba%20reveal-id%3D%26quot%3BAN1%26quot%3B%20data-open%3D%26quot%3BAN1%26quot%3B%20class%3D%26quot%3Blink%20link-ref%20link-reveal%20xref-fn%26quot%3B%26gt%3B%E2%80%A0%26lt%3B%2Fa%26gt%3B%26lt%3B%2Fsup%26gt%3B&amp;publicationDate=2011-06-13&amp;volumeNum=18&amp;issueNum=4&amp;author=Shirasawa%2C%20Kenta%3B%20Oyama%2C%20Maki&amp;startPage=221&amp;endPage=232&amp;contentId=10.1093%2Fdnares%2Fdsr013&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f221%2f501057" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7725038 id="7725038" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Raphanus sativus</em> (2<em>n</em> = 2<em>x</em> = 18) is a widely cultivated member of the family Brassicaceae, for which genomic resources are available only to a limited extent in comparison to many other members of the family. To promote more genetic and genomic studies and to enhance breeding programmes of <em>R. sativus</em>, we have prepared genetic resources such as complementary DNA libraries, expressed sequences tags (ESTs), simple sequence repeat (SSR) markers and a genetic linkage map. A total of 26 606 ESTs have been collected from seedlings, roots, leaves, and flowers, and clustered into 10 381 unigenes. Similarities were observed between the expression patterns of transcripts from <em>R. sativus</em> and those from representative members of the genera <em>Arabidopsis</em> and <em>Brassica</em>, indicating their functional relatedness. The EST sequence data were used to design 3800 SSR markers and consequently 630 polymorphic SSR loci and 213 reported marker loci have been mapped onto nine linkage groups, covering 1129.2 cM with an average distance of 1.3 cM between loci. Comparison of the mapped EST-SSR marker positions in <em>R. sativus</em> with the genome sequence of <em>A. thaliana</em> indicated that the Brassicaceae members have evolved from a common ancestor. It appears that genomic fragments corresponding to those of <em>A. thaliana</em> have been doubled and tripled in <em>R. sativus</em>. The genetic map developed here is expected to provide a standard map for the genetics, genomics, and molecular breeding of <em>R. sativus</em> as well as of related species. The resources are available at <a class="link link-uri" href="http://marker.kazusa.or.jp/Daikon">http://marker.kazusa.or.jp/Daikon</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative map&quot;">comparative map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;expressed sequence tag (EST)&quot;">expressed sequence tag (EST)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage map&quot;">genetic linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Raphanus sativus&quot;"><em>Raphanus sativus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;simple sequence repeat (SSR)&quot;">simple sequence repeat (SSR)</a></div>                            <h2 scrollto-destination=7725040 id="7725040" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>DNA markers are essential tools for plant genetics. In early studies, restriction fragment length polymorphism (RFLP), amplified fragment length polymorphism (AFLP) and random amplified polymorphic DNA (RAPD) techniques were widely used since no sequence information was required for their development. Single nucleotide polymorphism (SNP) is the most abundant source of genomic variation, and genetic studies using large-scale genotyping of SNPs have been performed with several plant species.<sup><a reveal-id="DSR013C1  DSR013C5" data-open="DSR013C1  DSR013C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> However, SNP markers only detect two alleles and biases in allelic frequency reflect the targeted population structure,<sup><a reveal-id="DSR013C5" data-open="DSR013C5" class="link link-ref link-reveal xref-bibr">5</a></sup> making it difficult to develop SNP markers that widely function across species. Although no similar high-throughput genotyping system has been developed for simple sequence repeats (SSR), these markers have several advantages over SNPs, including multi-allelic detection, high-transferability across species and flexibility with various laboratory systems.<sup><a reveal-id="DSR013C6" data-open="DSR013C6" class="link link-ref link-reveal xref-bibr">6</a></sup> SSR markers can be classified into two categories: genomic SSRs and expressed sequence tag (EST)-SSRs, which are designed from intergenic and intragenic sequences, respectively. Even though the polymorphic ratio of EST-SSR markers is sometimes lower than that of genomic SSRs, EST-SSRs can be expected to have greater transferability between species than genomic SSRs, since genic regions are more likely to be conserved among related species/genera.<sup><a reveal-id="DSR013C5" data-open="DSR013C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>Comparative genomics has had a significant impact on the fields of plant genetics and genomics. It has provided information on genomic evolution through the identification of chromosomal rearrangements and duplication/deletion events, as well as enabling large amounts of knowledge gained by studying model plants to be transferred to crops. The dicot species <em>Arabidopsis thaliana</em> (2<em>n</em> = 10) is a member of the family Brassicaceae. <em>A. thaliana</em> is the most studied model plant and extensive genomic resources have been developed in this species, including genomic sequences, EST information, mutant lines, DNA libraries and molecular markers, all of which are available to the research community (<a class="link link-uri" href="http://www.arabidopsis.org)">http://www.arabidopsis.org)</a>.<sup><a reveal-id="DSR013C7" data-open="DSR013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Similarly, the <em>Brassica</em> belong to the Brassicaceae and many <em>Brassica</em> species are cultivated as vegetable and oil crops. Commercially important <em>Brassica</em> species include broccoli, cabbage, cauliflower, Chinese cabbage, turnip, oilseed, and black mustard. Therefore, it is not surprising that the genomes of many members of the <em>Brassica</em> have been well studied. In particular, karyotype analysis has shown that <em>B. rapa</em> (2<em>n</em> = 20), <em>B. nigra</em> (2<em>n</em> = 16), and <em>B. oleracea</em> (2<em>n</em> = 18) are diploids with AA, BB, and CC genomes, respectively. In contrast, <em>B. juncea</em> (AABB: 2<em>n</em> = 36), <em>B. carinata</em> (BBCC: 2<em>n</em> = 34) and <em>B. napus</em> (AACC: 2<em>n</em> = 38) are amphidiploids, which were derived from combinations of two of the diploids.<sup><a reveal-id="DSR013C8" data-open="DSR013C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Comparative genomics between <em>A. thaliana</em> and different members of the <em>Brassica</em> have revealed syntenies in which regions of the <em>A. thaliana</em> genome have doubled or tripled.<sup><a reveal-id="DSR013C9  DSR013C11" data-open="DSR013C9  DSR013C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> In <em>B. rapa</em>, comparative studies have enabled map-based or candidate-gene cloning strategies to identify several quantitative trait loci (QTLs) and genes for agronomically important traits, such as flowering time, leaf morphology, and disease resistance.<sup><a reveal-id="DSR013C12" data-open="DSR013C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR013C13" data-open="DSR013C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p><em>Raphanus sativus</em> (2<em>n</em> = 2<em>x</em> = 18) is also a member of the Brassicaceae and it is used around the world as a vegetable crop, i.e. the radish or Japanese daikon. In particular, <em>R. sativus</em> is a common crop in East Asia and is the second most produced vegetable in Japan (<a class="link link-uri" href="http://www.e-stat.go.jp">www.e-stat.go.jp</a>). The genus <em>Raphanus</em> is originated from coastal regions along the Mediterranean and Black Seas and it is classified into two sections that comprise six species.<sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a></sup><em>R. sativus</em> was thought to have derived by hybridization between <em>R. maritimus</em> and <em>R. landra</em>, exhibiting an RR genome of 468–662 Mb.<sup><a reveal-id="DSR013C14  DSR013C16" data-open="DSR013C14  DSR013C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup><em>Raphanus sativus</em> genomic research has not progressed as far as for members of the <em>Brassica</em>, possibly because the genus <em>Raphanus</em> is less speciose and less economically important. Several genetic maps of <em>R. sativus</em> have been constructed using RFLP, AFLP, and RAPD markers, and these have been applied to QTL identification of disease and pest resistance, the shape and pigmentation of roots, and the flowering time.<sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSR013C17  DSR013C19" data-open="DSR013C17  DSR013C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> However, few studies have reported comparative genomics between <em>R. sativus</em> and <em>A. thaliana</em> because the <em>R. sativus</em> maps have no sequence-tagged markers. Kamei <em>et al.</em><sup><a reveal-id="DSR013C18" data-open="DSR013C18" class="link link-ref link-reveal xref-bibr">18</a></sup> used genomic SSR markers for <em>B. rapa</em> to perform a genetic analysis of <em>R. sativus</em>, but found low transferability of markers between the two species. Therefore, further advances in the genetic and genomic analyses of <em>R. sativus</em> will require the development of genic sequence-based markers.</p><p>In the present study, genomic resources were developed for <em>R. sativus</em>. These resources include cDNA libraries, EST sequences, EST-SSR markers, and a genetic map, all of which have great advantages to subsequent genomic studies in comparison with genomic SSR markers. Comparison with the <em>A. thaliana</em> genome indicated that the two genomes differentiated from a common ancestor and that <em>A. thaliana</em> genomic fragments had been doubled and tripled in the <em>R. sativus</em> genome. This comparative map could be used for the identification of candidate genes in QTLs from <em>R. sativus</em>.</p>                            <h2 scrollto-destination=7725045 id="7725045" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7725046 id="7725046" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Previously, an F<sub>2</sub> mapping population derived from a cross between two inbred lines, ‘GSK3-1’ and ‘HA2’, was used for construction of a genetic map of <em>R. sativus</em>.<sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> ‘GSK3-1’ is a selfed progeny from a leading Japanese variety of <em>R. sativus</em> known as Utsugi-Gensuke and ‘HA2’ is a Tokinashi type that exhibits late bolting. In the present study, F<sub>8–10</sub> progeny (<em>n</em> = 155) generated by single seed descent were used as recombinant inbred lines for linkage analysis. Genomic DNA from each line was extracted using the DNeasy Plant Mini kit (Qiagen, Germany).</p>                            <h3 scrollto-destination=7725048 id="7725048" class="section-title" data-legacy-id=s2b>2.2. Development of EST-derived SSR markers</h3>
<p>Plant RNA Purification Reagent (Invitrogen, USA) was used to extract total RNA from 5 g samples of seedlings, roots, leaves, and flowers of <em>R. sativus</em> ‘GSK3-1’. cDNA libraries were constructed for each organ and then sequencing analysis and data processing were performed as described previously.<sup><a reveal-id="DSR013C20" data-open="DSR013C20" class="link link-ref link-reveal xref-bibr">20</a></sup> High-quality reads that comprised &gt;50 bp of contiguous sequence were submitted to the DDBJ/EMBL/GenBank databases with the accession numbers FY428055 to FY454660. BLASTN was used to perform similarity searches between non-redundant <em>R. sativus</em> ESTs that clustered using the PHRAD program and the UniGene data sets for <em>R. sativus</em>, <em>A. thaliana</em>, <em>B. napus</em>, <em>B. oleracea</em>, <em>B. rapa</em>, <em>Lotus japonicas</em>, and <em>Oryza sativa</em> (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/unigene">http://www.ncbi.nlm.nih.gov/unigene</a>). BLASTX searches against amino acid sequences in the KOG set (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/COG">http://www.ncbi.nlm.nih.gov/COG</a>) enabled EST contigs to be classified into KOG categories.<sup><a reveal-id="DSR013C21" data-open="DSR013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> These sequence similarities were judged to be significant when the <em>E</em>-value was less than 1e−10.</p><p>Microsatellites or SSRs of ≥15 nucleotides in length were identified. All possible combinations of di- (NN), tri- (NNN), and tetra-nucleotide (NNNN) repeats were represented. Primer pairs were designed against the flanking sequences of each SSR, as described previously.<sup><a reveal-id="DSR013C22" data-open="DSR013C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>PCR reactions were performed using 0.5 ng genomic DNA in each 5 µl reaction. In addition to template, PCR reaction mixes contained 1 × PCR buffer (BIOLINE, UK), 3 mM MgCl<sub>2</sub>, 0.04U BIOTAQ™ DNA polymerase (BIOLINE, UK), 0.2 mM dNTPs, and 0.8 µM of each primer. The thermal cycling conditions were as follows: 1 min denaturation at 94°C; 35 cycles of 30 s denaturation at 94°C, 30 s annealing at 60°C and 1 min extension at 72°C; and a final 3 min extension at 72°C. The PCR products were separated by 10% polyacrylamide gel electrophoresis in TBE buffer, according to the standard protocol, or with a Type 3730 DNA fragment analyzer (Applied Biosystems, USA). In the latter case, the data were analysed using GeneMapper software (Applied Biosystems, USA).</p>                            <h3 scrollto-destination=7725052 id="7725052" class="section-title" data-legacy-id=s2c>2.3. Genotyping and map construction</h3>
<p>Linkage analysis was performed with genotypic data derived using the EST-SSR markers developed in this study, as well as segregation data generated from 223 published RAPD, RFLP, and trait marker loci,<sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSR013C23" data-open="DSR013C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSR013C24" data-open="DSR013C24" class="link link-ref link-reveal xref-bibr">24</a></sup> using JoinMap<sup>®</sup> software version 4.<sup><a reveal-id="DSR013C25" data-open="DSR013C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The marker loci were classified roughly into nine linkage groups using the JoinMap<sup>®</sup> grouping module and LOD scores of 4.0–10.0. The mapped positions of the RAPD, RFLP, and trait marker loci were used as a frame for linkage analysis. Marker order and genetic distance were calculated using a regression mapping algorithm and the following parameters: Haldane's mapping function, recombination frequency ≤0.35 and LOD score ≥2.0.</p>                            <h3 scrollto-destination=7725054 id="7725054" class="section-title" data-legacy-id=s2d>2.4. Synteny analysis</h3>
<p><em>R. sativus</em> amino acid sequences were deduced from mapped ESTs. BLASTX was used to compare predicted <em>R. sativus</em> sequences against the TAIR9-predicted protein database (<a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>) with a threshold <em>E</em>-value of 1e−50. Because Kim <em>et al</em>.<sup><a reveal-id="DSR013C11" data-open="DSR013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> reported a <em>B. rapa</em> genetic map constructed with EST-based markers, a comparative map between <em>A. thaliana</em> and <em>B. rapa</em> was also constructed by locating nucleotide sequences of marker loci from <em>B. rapa</em> on the <em>A. thaliana</em> genome.</p>                            <h2 scrollto-destination=7725056 id="7725056" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7725057 id="7725057" class="section-title" data-legacy-id=s3a>3.1. Features of <em>R. sativus</em> ESTs</h3>
<p>A total of 32 256 cDNA clones, including 9216 clones from a seedling library, 7680 clones from a root library, 7680 clones from a leaf library, and 7680 clones from a flower library, were sequenced from the 5′end. A total of 26 606 sequences (8458, 6320, 6721, and 5107 from seedling, root, leaf, and flower libraries, respectively) consisting of 19 391 246 qualified bases were obtained, and the average EST length was 729 bp.</p><p>The PHRAD program was used to cluster EST sequences in order to identify the number of independent EST species. These analyses indicated 10 381 potential non-redundant EST sequences, which included 6625 contigs and 3756 singletons with an average GC content of 45.6%. Non-redundant EST sequences were compared against the Unigene data of <em>R. sativus</em>, <em>A. thaliana</em>, <em>B. napus</em>, <em>B. oleracea</em>, <em>B. rapa</em>, <em>L. japonicas</em>, and <em>O. sativa</em> (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/unigene">http://www.ncbi.nlm.nih.gov/unigene</a>). These comparisons indicated 9758 non-redundant ESTs with significant similarity (<em>E</em>-value &lt;1e−10) to registered sequences of the seven species and the remaining 623 non-redundant ESTs were unique to the data from this study.</p><p>To investigate the functional classification of <em>R. sativus</em> ESTs, BLASTX was used to compare non-redundant EST sequences with eukaryotic clusters of orthologous groups (KOGs) and these ESTs were then classified into KOG categories.<sup><a reveal-id="DSR013C21" data-open="DSR013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Among the 10 381 non-redundant <em>R. sativus</em> EST sequences, 6265 showed similarities to KOG sequences that have functional classification, and their distribution is similar to those of the seven species (Fig. <a reveal-id="DSR013F1" data-open="DSR013F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="7725061" scrollto-destination="7725061"></a>
<div id="DSR013F1" data-id="DSR013F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01301.jpeg?Expires=2147483647&Signature=wjJp5vbLOJblQrnqCEFABz0I5oZqSICTauT6ewQ43O8UEkgDsjVsjfhV0DY~S4frVp5mmdWBjVkDrDWlhjGgNXHVQ1pGTULtNdTJuoMg6-T9wEJN-rwo4xvXA6mE09KHaI9OvFWvbvAtYAJCMN6bdmbptMExNsBIetMSGXpYDGpvTMxvwr593mufZFJYXMZEiIKIApTU-~5nYi0-sPj~QDERAN58VMOwarvoNEwNz1EeJq2-9qvHXa46mf1UXGXeNrv~~wU0fH7Pom-TPSVqFIGoDy1o2Slb4DaCO4eMfwrx-3evdee9zsZVV0jO8zE0r8sz~4neYodeJzO1OKHHrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of non-redundant R. sativus EST sequences and unigenes of the seven plant species into KOG categories. BLASTX was used to compare non-redundant EST sequences with the KOG sequence set. EST sequences were then classified into the KOG categories with the most similar sequences." path-from-xml="dsr01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725061/dsr01301.jpeg" path-from-xml="dsr01301.jpeg" target="_blank">View large</a><a section="7725061" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01301.jpeg?Expires=2147483647&Signature=1f8X-B9qTixN3gf2Er2vwj~BPCtZuAztnq0ixyd1lNbjFChdvh~yCC3wXyV77qSo~~3iGOP1-utDGVDLhwIeKMlFYpHGLzbBYPPCAlXlq1-OnV5hIQTrzZKMUp2eB77MpVxOjZLm1EWd5sSSVyjl~R6652-e4Gj4OPMes6dsh3yQzKkSNuALRGvh3V3RhBLGdbxdRojbntlbaq~sPCkSLKf-g3rQPxWaiNRsC-44fi1Fe-NOJIXIZ6DW6amXhUnRZRKWzaja4S23bbwe7KTLqxNV8TzUzmaWFzKMIcnWSxJb-IQkIYxCWj3qtFo8p8M2RxYgd9NsIS9sEGPst0ohTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7725061&ar=501057&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of non-redundant <em>R. sativus</em> EST sequences and unigenes of the seven plant species into KOG categories. BLASTX was used to compare non-redundant EST sequences with the KOG sequence set. EST sequences were then classified into the KOG categories with the most similar sequences.</p></div></div><div content-id="DSR013F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01301.jpeg?Expires=2147483647&Signature=wjJp5vbLOJblQrnqCEFABz0I5oZqSICTauT6ewQ43O8UEkgDsjVsjfhV0DY~S4frVp5mmdWBjVkDrDWlhjGgNXHVQ1pGTULtNdTJuoMg6-T9wEJN-rwo4xvXA6mE09KHaI9OvFWvbvAtYAJCMN6bdmbptMExNsBIetMSGXpYDGpvTMxvwr593mufZFJYXMZEiIKIApTU-~5nYi0-sPj~QDERAN58VMOwarvoNEwNz1EeJq2-9qvHXa46mf1UXGXeNrv~~wU0fH7Pom-TPSVqFIGoDy1o2Slb4DaCO4eMfwrx-3evdee9zsZVV0jO8zE0r8sz~4neYodeJzO1OKHHrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of non-redundant R. sativus EST sequences and unigenes of the seven plant species into KOG categories. BLASTX was used to compare non-redundant EST sequences with the KOG sequence set. EST sequences were then classified into the KOG categories with the most similar sequences." path-from-xml="dsr01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725061/dsr01301.jpeg" path-from-xml="dsr01301.jpeg" target="_blank">View large</a><a section="7725061" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01301.jpeg?Expires=2147483647&Signature=1f8X-B9qTixN3gf2Er2vwj~BPCtZuAztnq0ixyd1lNbjFChdvh~yCC3wXyV77qSo~~3iGOP1-utDGVDLhwIeKMlFYpHGLzbBYPPCAlXlq1-OnV5hIQTrzZKMUp2eB77MpVxOjZLm1EWd5sSSVyjl~R6652-e4Gj4OPMes6dsh3yQzKkSNuALRGvh3V3RhBLGdbxdRojbntlbaq~sPCkSLKf-g3rQPxWaiNRsC-44fi1Fe-NOJIXIZ6DW6amXhUnRZRKWzaja4S23bbwe7KTLqxNV8TzUzmaWFzKMIcnWSxJb-IQkIYxCWj3qtFo8p8M2RxYgd9NsIS9sEGPst0ohTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7725061&ar=501057&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of non-redundant <em>R. sativus</em> EST sequences and unigenes of the seven plant species into KOG categories. BLASTX was used to compare non-redundant EST sequences with the KOG sequence set. EST sequences were then classified into the KOG categories with the most similar sequences.</p></div></div>                            <h3 scrollto-destination=7725062 id="7725062" class="section-title" data-legacy-id=s3b>3.2. SSR motifs in <em>R. sativus</em> ESTs</h3>
<p><em>In silico</em> data mining of the 26 606 sequences yielded 3800 SSR markers, which were designated as RSS (<em><span class="underline">R</span>aphanus <span class="underline">s</span>ativus</em> EST-derived <span class="underline">S</span>SR) markers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Out of the 3800 SSR markers, 3733 markers (98.2%) had motif lengths ranging from 20 to 35 bases (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The RSS marker motifs contained 3094 (81.4%), 381 (10.0%), and 325 (8.6%) tri-, di-, and tetra-nucleotide repeats, respectively (Table <a reveal-id="DSR013TB1" data-open="DSR013TB1" class="link link-reveal link-table xref-fig">1</a>). The most abundant trinucleotide repeats were poly (AAG)n (21.4%), poly (GGA)n (14.2%), and poly (ATC)n (10.1%). Four types of dinucleotide repeats were observed and the poly (AG)n motifs were the most abundant (6.8%). The tetranucleotide repeats poly (AAAG)n (2.7%), poly (AAAC)n (2.1%) and poly (AAAT)n (1.4%) were more frequently observed than the other motifs. Among the 3800 SSR motifs, 1604 and 242 were located on coding and untranscribed regions, respectively (Table <a reveal-id="DSR013TB1" data-open="DSR013TB1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). SSRs on coding regions consisted of 30 di-, 1533 tri-, and 41 tetra-nucleotide repeat motifs, while those on untranscribed regions included 96 di-, 104 tri-, and 42 terta-nucleotide repeat motifs. The other 1954 SSRs were not assigned to the regions because the length of each EST was not enough to predict open reading frames. </p>                        <a id="7725064" scrollto-destination="7725064"></a>
<div content-id="DSR013TB1" class="table-modal"><div class="table-wrap"><div id="DSR013TB1" class="table-wrap-title" data-id="DSR013TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of SSR motifs in the RSS markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="8">Designed SSR markers<hr /> </th><th colspan="8">Polymorphic SSR markers<hr /> </th></tr><tr><th>Total </th><th>%<sup>a</sup> </th><th>Coding region </th><th>%<sup>a</sup> </th><th>UTR </th><th>%<sup>a</sup> </th><th>Unassigned region<sup>b</sup> </th><th>%<sup>a</sup> </th><th>Total </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Coding region </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>UTR </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Unassigned region<sup>b</sup> </th><th>Polymorphic ratio (%)<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="17">Dinucleotide </td></tr><tr><td> AC </td><td>60 </td><td>1.6 </td><td>3 </td><td>0.1 </td><td>7 </td><td>0.2 </td><td>50 </td><td>1.3 </td><td>14 </td><td>23.3 </td><td>2 </td><td>66.7 </td><td>1 </td><td>14.3 </td><td>11 </td><td>22.0 </td></tr><tr><td> AG </td><td>257 </td><td>6.8 </td><td>25 </td><td>0.7 </td><td>83 </td><td>2.2 </td><td>149 </td><td>3.9 </td><td>48 </td><td>18.7 </td><td>4 </td><td>16.0 </td><td>22 </td><td>26.5 </td><td>22 </td><td>14.8 </td></tr><tr><td> AT </td><td>63 </td><td>1.7 </td><td>2 </td><td>0.1 </td><td>6 </td><td>0.2 </td><td>55 </td><td>1.4 </td><td>14 </td><td>22.2 </td><td>1 </td><td>50.0 </td><td>1 </td><td>16.7 </td><td>12 </td><td>21.8 </td></tr><tr><td> GC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> Total </td><td>381 </td><td>10.0 </td><td>30 </td><td>0.8 </td><td>96 </td><td>2.5 </td><td>255 </td><td>6.7 </td><td>76 </td><td>19.9 </td><td>7 </td><td>23.3 </td><td>24 </td><td>25.0 </td><td>45 </td><td>17.6 </td></tr><tr><td colspan="17">Trinucleotide </td></tr><tr><td> AAC </td><td>305 </td><td>8.0 </td><td>140 </td><td>3.7 </td><td>9 </td><td>0.2 </td><td>156 </td><td>4.1 </td><td>53 </td><td>17.4 </td><td>27 </td><td>19.3 </td><td>1 </td><td>11.1 </td><td>25 </td><td>16.0 </td></tr><tr><td> AAG </td><td>815 </td><td>21.4 </td><td>374 </td><td>9.8 </td><td>49 </td><td>1.3 </td><td>392 </td><td>10.3 </td><td>119 </td><td>14.6 </td><td>55 </td><td>14.7 </td><td>7 </td><td>14.3 </td><td>57 </td><td>14.5 </td></tr><tr><td> AAT </td><td>103 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>10 </td><td>0.3 </td><td>80 </td><td>2.1 </td><td>20 </td><td>19.4 </td><td>2 </td><td>15.4 </td><td>2 </td><td>20.0 </td><td>16 </td><td>20.0 </td></tr><tr><td> ACG </td><td>158 </td><td>4.2 </td><td>74 </td><td>1.9 </td><td>6 </td><td>0.2 </td><td>78 </td><td>2.1 </td><td>23 </td><td>14.6 </td><td>9 </td><td>12.2 </td><td>1 </td><td>16.7 </td><td>13 </td><td>16.7 </td></tr><tr><td> ACT </td><td>65 </td><td>1.7 </td><td>22 </td><td>0.6 </td><td>0 </td><td>0.0 </td><td>43 </td><td>1.1 </td><td>11 </td><td>16.9 </td><td>3 </td><td>13.6 </td><td>0 </td><td>— </td><td>8 </td><td>18.6 </td></tr><tr><td> AGC </td><td>293 </td><td>7.7 </td><td>180 </td><td>4.7 </td><td>1 </td><td>0.0 </td><td>112 </td><td>2.9 </td><td>58 </td><td>19.8 </td><td>34 </td><td>18.9 </td><td>0 </td><td>0.0 </td><td>24 </td><td>21.4 </td></tr><tr><td> ATC </td><td>385 </td><td>10.1 </td><td>192 </td><td>5.1 </td><td>14 </td><td>0.4 </td><td>179 </td><td>4.7 </td><td>69 </td><td>17.9 </td><td>37 </td><td>19.3 </td><td>4 </td><td>28.6 </td><td>28 </td><td>15.6 </td></tr><tr><td> GGA </td><td>541 </td><td>14.2 </td><td>310 </td><td>8.2 </td><td>8 </td><td>0.2 </td><td>223 </td><td>5.9 </td><td>89 </td><td>16.5 </td><td>53 </td><td>17.1 </td><td>1 </td><td>12.5 </td><td>35 </td><td>15.7 </td></tr><tr><td> GGC </td><td>154 </td><td>4.1 </td><td>95 </td><td>2.5 </td><td>1 </td><td>0.0 </td><td>58 </td><td>1.5 </td><td>22 </td><td>14.3 </td><td>17 </td><td>17.9 </td><td>0 </td><td>0.0 </td><td>5 </td><td>8.6 </td></tr><tr><td> GGT </td><td>275 </td><td>7.2 </td><td>133 </td><td>3.5 </td><td>6 </td><td>0.2 </td><td>136 </td><td>3.6 </td><td>40 </td><td>14.5 </td><td>18 </td><td>13.5 </td><td>1 </td><td>16.7 </td><td>21 </td><td>15.4 </td></tr><tr><td> Total </td><td>3094 </td><td>81.4 </td><td>1533 </td><td>40.3 </td><td>104 </td><td>2.7 </td><td>1457 </td><td>38.3 </td><td>504 </td><td>16.3 </td><td>255 </td><td>16.6 </td><td>17 </td><td>16.3 </td><td>232 </td><td>15.9 </td></tr><tr><td colspan="17">Tetranucleotide </td></tr><tr><td> AAAC </td><td>81 </td><td>2.1 </td><td>3 </td><td>0.1 </td><td>11 </td><td>0.3 </td><td>67 </td><td>1.8 </td><td>17 </td><td>21.0 </td><td>1 </td><td>33.3 </td><td>2 </td><td>18.2 </td><td>14 </td><td>20.9 </td></tr><tr><td> AAAG </td><td>101 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>15 </td><td>0.4 </td><td>73 </td><td>1.9 </td><td>14 </td><td>13.9 </td><td>2 </td><td>15.4 </td><td>2 </td><td>13.3 </td><td>10 </td><td>13.7 </td></tr><tr><td> AAAT </td><td>53 </td><td>1.4 </td><td>3 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>46 </td><td>1.2 </td><td>14 </td><td>26.4 </td><td>1 </td><td>33.3 </td><td>1 </td><td>25.0 </td><td>12 </td><td>26.1 </td></tr><tr><td> AACG </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> AAGC </td><td>11 </td><td>0.3 </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>2 </td><td>18.2 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATC </td><td>24 </td><td>0.6 </td><td>4 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>16 </td><td>0.4 </td><td>6 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>3 </td><td>75.0 </td><td>3 </td><td>18.8 </td></tr><tr><td> AATG </td><td>9 </td><td>0.2 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>1 </td><td>11.1 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATT </td><td>7 </td><td>0.2 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>1 </td><td>14.3 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>1 </td><td>16.7 </td></tr><tr><td> AGGC </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td></tr><tr><td> GACG </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td></tr><tr><td> GAGC </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>100.0 </td></tr><tr><td> GATC </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAT </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>3 </td><td>37.5 </td><td>1 </td><td>50.0 </td><td>0 </td><td>— </td><td>2 </td><td>33.3 </td></tr><tr><td> GGCT </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGGA </td><td>18 </td><td>0.5 </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>8 </td><td>0.2 </td><td>3 </td><td>16.7 </td><td>2 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>12.5 </td></tr><tr><td> GGGT </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td>Total </td><td>325 </td><td>8.6 </td><td>41 </td><td>1.1 </td><td>42 </td><td>1.1 </td><td>242 </td><td>6.4 </td><td>62 </td><td>19.1 </td><td>8 </td><td>19.5 </td><td>8 </td><td>19.0 </td><td>46 </td><td>19.0 </td></tr><tr><td>Total </td><td>3800 </td><td>100.0 </td><td>1604 </td><td>42.2 </td><td>242 </td><td>6.4 </td><td>1954 </td><td>51.4 </td><td>642 </td><td>16.9 </td><td>270 </td><td>16.8 </td><td>49 </td><td>20.2 </td><td>323 </td><td>16.5 </td></tr></tbody></table></div><div content-id="DSR013TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="8">Designed SSR markers<hr /> </th><th colspan="8">Polymorphic SSR markers<hr /> </th></tr><tr><th>Total </th><th>%<sup>a</sup> </th><th>Coding region </th><th>%<sup>a</sup> </th><th>UTR </th><th>%<sup>a</sup> </th><th>Unassigned region<sup>b</sup> </th><th>%<sup>a</sup> </th><th>Total </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Coding region </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>UTR </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Unassigned region<sup>b</sup> </th><th>Polymorphic ratio (%)<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="17">Dinucleotide </td></tr><tr><td> AC </td><td>60 </td><td>1.6 </td><td>3 </td><td>0.1 </td><td>7 </td><td>0.2 </td><td>50 </td><td>1.3 </td><td>14 </td><td>23.3 </td><td>2 </td><td>66.7 </td><td>1 </td><td>14.3 </td><td>11 </td><td>22.0 </td></tr><tr><td> AG </td><td>257 </td><td>6.8 </td><td>25 </td><td>0.7 </td><td>83 </td><td>2.2 </td><td>149 </td><td>3.9 </td><td>48 </td><td>18.7 </td><td>4 </td><td>16.0 </td><td>22 </td><td>26.5 </td><td>22 </td><td>14.8 </td></tr><tr><td> AT </td><td>63 </td><td>1.7 </td><td>2 </td><td>0.1 </td><td>6 </td><td>0.2 </td><td>55 </td><td>1.4 </td><td>14 </td><td>22.2 </td><td>1 </td><td>50.0 </td><td>1 </td><td>16.7 </td><td>12 </td><td>21.8 </td></tr><tr><td> GC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> Total </td><td>381 </td><td>10.0 </td><td>30 </td><td>0.8 </td><td>96 </td><td>2.5 </td><td>255 </td><td>6.7 </td><td>76 </td><td>19.9 </td><td>7 </td><td>23.3 </td><td>24 </td><td>25.0 </td><td>45 </td><td>17.6 </td></tr><tr><td colspan="17">Trinucleotide </td></tr><tr><td> AAC </td><td>305 </td><td>8.0 </td><td>140 </td><td>3.7 </td><td>9 </td><td>0.2 </td><td>156 </td><td>4.1 </td><td>53 </td><td>17.4 </td><td>27 </td><td>19.3 </td><td>1 </td><td>11.1 </td><td>25 </td><td>16.0 </td></tr><tr><td> AAG </td><td>815 </td><td>21.4 </td><td>374 </td><td>9.8 </td><td>49 </td><td>1.3 </td><td>392 </td><td>10.3 </td><td>119 </td><td>14.6 </td><td>55 </td><td>14.7 </td><td>7 </td><td>14.3 </td><td>57 </td><td>14.5 </td></tr><tr><td> AAT </td><td>103 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>10 </td><td>0.3 </td><td>80 </td><td>2.1 </td><td>20 </td><td>19.4 </td><td>2 </td><td>15.4 </td><td>2 </td><td>20.0 </td><td>16 </td><td>20.0 </td></tr><tr><td> ACG </td><td>158 </td><td>4.2 </td><td>74 </td><td>1.9 </td><td>6 </td><td>0.2 </td><td>78 </td><td>2.1 </td><td>23 </td><td>14.6 </td><td>9 </td><td>12.2 </td><td>1 </td><td>16.7 </td><td>13 </td><td>16.7 </td></tr><tr><td> ACT </td><td>65 </td><td>1.7 </td><td>22 </td><td>0.6 </td><td>0 </td><td>0.0 </td><td>43 </td><td>1.1 </td><td>11 </td><td>16.9 </td><td>3 </td><td>13.6 </td><td>0 </td><td>— </td><td>8 </td><td>18.6 </td></tr><tr><td> AGC </td><td>293 </td><td>7.7 </td><td>180 </td><td>4.7 </td><td>1 </td><td>0.0 </td><td>112 </td><td>2.9 </td><td>58 </td><td>19.8 </td><td>34 </td><td>18.9 </td><td>0 </td><td>0.0 </td><td>24 </td><td>21.4 </td></tr><tr><td> ATC </td><td>385 </td><td>10.1 </td><td>192 </td><td>5.1 </td><td>14 </td><td>0.4 </td><td>179 </td><td>4.7 </td><td>69 </td><td>17.9 </td><td>37 </td><td>19.3 </td><td>4 </td><td>28.6 </td><td>28 </td><td>15.6 </td></tr><tr><td> GGA </td><td>541 </td><td>14.2 </td><td>310 </td><td>8.2 </td><td>8 </td><td>0.2 </td><td>223 </td><td>5.9 </td><td>89 </td><td>16.5 </td><td>53 </td><td>17.1 </td><td>1 </td><td>12.5 </td><td>35 </td><td>15.7 </td></tr><tr><td> GGC </td><td>154 </td><td>4.1 </td><td>95 </td><td>2.5 </td><td>1 </td><td>0.0 </td><td>58 </td><td>1.5 </td><td>22 </td><td>14.3 </td><td>17 </td><td>17.9 </td><td>0 </td><td>0.0 </td><td>5 </td><td>8.6 </td></tr><tr><td> GGT </td><td>275 </td><td>7.2 </td><td>133 </td><td>3.5 </td><td>6 </td><td>0.2 </td><td>136 </td><td>3.6 </td><td>40 </td><td>14.5 </td><td>18 </td><td>13.5 </td><td>1 </td><td>16.7 </td><td>21 </td><td>15.4 </td></tr><tr><td> Total </td><td>3094 </td><td>81.4 </td><td>1533 </td><td>40.3 </td><td>104 </td><td>2.7 </td><td>1457 </td><td>38.3 </td><td>504 </td><td>16.3 </td><td>255 </td><td>16.6 </td><td>17 </td><td>16.3 </td><td>232 </td><td>15.9 </td></tr><tr><td colspan="17">Tetranucleotide </td></tr><tr><td> AAAC </td><td>81 </td><td>2.1 </td><td>3 </td><td>0.1 </td><td>11 </td><td>0.3 </td><td>67 </td><td>1.8 </td><td>17 </td><td>21.0 </td><td>1 </td><td>33.3 </td><td>2 </td><td>18.2 </td><td>14 </td><td>20.9 </td></tr><tr><td> AAAG </td><td>101 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>15 </td><td>0.4 </td><td>73 </td><td>1.9 </td><td>14 </td><td>13.9 </td><td>2 </td><td>15.4 </td><td>2 </td><td>13.3 </td><td>10 </td><td>13.7 </td></tr><tr><td> AAAT </td><td>53 </td><td>1.4 </td><td>3 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>46 </td><td>1.2 </td><td>14 </td><td>26.4 </td><td>1 </td><td>33.3 </td><td>1 </td><td>25.0 </td><td>12 </td><td>26.1 </td></tr><tr><td> AACG </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> AAGC </td><td>11 </td><td>0.3 </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>2 </td><td>18.2 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATC </td><td>24 </td><td>0.6 </td><td>4 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>16 </td><td>0.4 </td><td>6 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>3 </td><td>75.0 </td><td>3 </td><td>18.8 </td></tr><tr><td> AATG </td><td>9 </td><td>0.2 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>1 </td><td>11.1 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATT </td><td>7 </td><td>0.2 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>1 </td><td>14.3 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>1 </td><td>16.7 </td></tr><tr><td> AGGC </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td></tr><tr><td> GACG </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td></tr><tr><td> GAGC </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>100.0 </td></tr><tr><td> GATC </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAT </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>3 </td><td>37.5 </td><td>1 </td><td>50.0 </td><td>0 </td><td>— </td><td>2 </td><td>33.3 </td></tr><tr><td> GGCT </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGGA </td><td>18 </td><td>0.5 </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>8 </td><td>0.2 </td><td>3 </td><td>16.7 </td><td>2 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>12.5 </td></tr><tr><td> GGGT </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td>Total </td><td>325 </td><td>8.6 </td><td>41 </td><td>1.1 </td><td>42 </td><td>1.1 </td><td>242 </td><td>6.4 </td><td>62 </td><td>19.1 </td><td>8 </td><td>19.5 </td><td>8 </td><td>19.0 </td><td>46 </td><td>19.0 </td></tr><tr><td>Total </td><td>3800 </td><td>100.0 </td><td>1604 </td><td>42.2 </td><td>242 </td><td>6.4 </td><td>1954 </td><td>51.4 </td><td>642 </td><td>16.9 </td><td>270 </td><td>16.8 </td><td>49 </td><td>20.2 </td><td>323 </td><td>16.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Percentage to a total of 3800 SSR markers.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The position of SSRs was not assigned because of the length of each EST was not enough to predict open reading frames.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Percentage to the designed SSR markers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7725064" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR013TB1" class="table-wrap-title" data-id="DSR013TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of SSR motifs in the RSS markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="8">Designed SSR markers<hr /> </th><th colspan="8">Polymorphic SSR markers<hr /> </th></tr><tr><th>Total </th><th>%<sup>a</sup> </th><th>Coding region </th><th>%<sup>a</sup> </th><th>UTR </th><th>%<sup>a</sup> </th><th>Unassigned region<sup>b</sup> </th><th>%<sup>a</sup> </th><th>Total </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Coding region </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>UTR </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Unassigned region<sup>b</sup> </th><th>Polymorphic ratio (%)<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="17">Dinucleotide </td></tr><tr><td> AC </td><td>60 </td><td>1.6 </td><td>3 </td><td>0.1 </td><td>7 </td><td>0.2 </td><td>50 </td><td>1.3 </td><td>14 </td><td>23.3 </td><td>2 </td><td>66.7 </td><td>1 </td><td>14.3 </td><td>11 </td><td>22.0 </td></tr><tr><td> AG </td><td>257 </td><td>6.8 </td><td>25 </td><td>0.7 </td><td>83 </td><td>2.2 </td><td>149 </td><td>3.9 </td><td>48 </td><td>18.7 </td><td>4 </td><td>16.0 </td><td>22 </td><td>26.5 </td><td>22 </td><td>14.8 </td></tr><tr><td> AT </td><td>63 </td><td>1.7 </td><td>2 </td><td>0.1 </td><td>6 </td><td>0.2 </td><td>55 </td><td>1.4 </td><td>14 </td><td>22.2 </td><td>1 </td><td>50.0 </td><td>1 </td><td>16.7 </td><td>12 </td><td>21.8 </td></tr><tr><td> GC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> Total </td><td>381 </td><td>10.0 </td><td>30 </td><td>0.8 </td><td>96 </td><td>2.5 </td><td>255 </td><td>6.7 </td><td>76 </td><td>19.9 </td><td>7 </td><td>23.3 </td><td>24 </td><td>25.0 </td><td>45 </td><td>17.6 </td></tr><tr><td colspan="17">Trinucleotide </td></tr><tr><td> AAC </td><td>305 </td><td>8.0 </td><td>140 </td><td>3.7 </td><td>9 </td><td>0.2 </td><td>156 </td><td>4.1 </td><td>53 </td><td>17.4 </td><td>27 </td><td>19.3 </td><td>1 </td><td>11.1 </td><td>25 </td><td>16.0 </td></tr><tr><td> AAG </td><td>815 </td><td>21.4 </td><td>374 </td><td>9.8 </td><td>49 </td><td>1.3 </td><td>392 </td><td>10.3 </td><td>119 </td><td>14.6 </td><td>55 </td><td>14.7 </td><td>7 </td><td>14.3 </td><td>57 </td><td>14.5 </td></tr><tr><td> AAT </td><td>103 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>10 </td><td>0.3 </td><td>80 </td><td>2.1 </td><td>20 </td><td>19.4 </td><td>2 </td><td>15.4 </td><td>2 </td><td>20.0 </td><td>16 </td><td>20.0 </td></tr><tr><td> ACG </td><td>158 </td><td>4.2 </td><td>74 </td><td>1.9 </td><td>6 </td><td>0.2 </td><td>78 </td><td>2.1 </td><td>23 </td><td>14.6 </td><td>9 </td><td>12.2 </td><td>1 </td><td>16.7 </td><td>13 </td><td>16.7 </td></tr><tr><td> ACT </td><td>65 </td><td>1.7 </td><td>22 </td><td>0.6 </td><td>0 </td><td>0.0 </td><td>43 </td><td>1.1 </td><td>11 </td><td>16.9 </td><td>3 </td><td>13.6 </td><td>0 </td><td>— </td><td>8 </td><td>18.6 </td></tr><tr><td> AGC </td><td>293 </td><td>7.7 </td><td>180 </td><td>4.7 </td><td>1 </td><td>0.0 </td><td>112 </td><td>2.9 </td><td>58 </td><td>19.8 </td><td>34 </td><td>18.9 </td><td>0 </td><td>0.0 </td><td>24 </td><td>21.4 </td></tr><tr><td> ATC </td><td>385 </td><td>10.1 </td><td>192 </td><td>5.1 </td><td>14 </td><td>0.4 </td><td>179 </td><td>4.7 </td><td>69 </td><td>17.9 </td><td>37 </td><td>19.3 </td><td>4 </td><td>28.6 </td><td>28 </td><td>15.6 </td></tr><tr><td> GGA </td><td>541 </td><td>14.2 </td><td>310 </td><td>8.2 </td><td>8 </td><td>0.2 </td><td>223 </td><td>5.9 </td><td>89 </td><td>16.5 </td><td>53 </td><td>17.1 </td><td>1 </td><td>12.5 </td><td>35 </td><td>15.7 </td></tr><tr><td> GGC </td><td>154 </td><td>4.1 </td><td>95 </td><td>2.5 </td><td>1 </td><td>0.0 </td><td>58 </td><td>1.5 </td><td>22 </td><td>14.3 </td><td>17 </td><td>17.9 </td><td>0 </td><td>0.0 </td><td>5 </td><td>8.6 </td></tr><tr><td> GGT </td><td>275 </td><td>7.2 </td><td>133 </td><td>3.5 </td><td>6 </td><td>0.2 </td><td>136 </td><td>3.6 </td><td>40 </td><td>14.5 </td><td>18 </td><td>13.5 </td><td>1 </td><td>16.7 </td><td>21 </td><td>15.4 </td></tr><tr><td> Total </td><td>3094 </td><td>81.4 </td><td>1533 </td><td>40.3 </td><td>104 </td><td>2.7 </td><td>1457 </td><td>38.3 </td><td>504 </td><td>16.3 </td><td>255 </td><td>16.6 </td><td>17 </td><td>16.3 </td><td>232 </td><td>15.9 </td></tr><tr><td colspan="17">Tetranucleotide </td></tr><tr><td> AAAC </td><td>81 </td><td>2.1 </td><td>3 </td><td>0.1 </td><td>11 </td><td>0.3 </td><td>67 </td><td>1.8 </td><td>17 </td><td>21.0 </td><td>1 </td><td>33.3 </td><td>2 </td><td>18.2 </td><td>14 </td><td>20.9 </td></tr><tr><td> AAAG </td><td>101 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>15 </td><td>0.4 </td><td>73 </td><td>1.9 </td><td>14 </td><td>13.9 </td><td>2 </td><td>15.4 </td><td>2 </td><td>13.3 </td><td>10 </td><td>13.7 </td></tr><tr><td> AAAT </td><td>53 </td><td>1.4 </td><td>3 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>46 </td><td>1.2 </td><td>14 </td><td>26.4 </td><td>1 </td><td>33.3 </td><td>1 </td><td>25.0 </td><td>12 </td><td>26.1 </td></tr><tr><td> AACG </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> AAGC </td><td>11 </td><td>0.3 </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>2 </td><td>18.2 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATC </td><td>24 </td><td>0.6 </td><td>4 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>16 </td><td>0.4 </td><td>6 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>3 </td><td>75.0 </td><td>3 </td><td>18.8 </td></tr><tr><td> AATG </td><td>9 </td><td>0.2 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>1 </td><td>11.1 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATT </td><td>7 </td><td>0.2 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>1 </td><td>14.3 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>1 </td><td>16.7 </td></tr><tr><td> AGGC </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td></tr><tr><td> GACG </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td></tr><tr><td> GAGC </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>100.0 </td></tr><tr><td> GATC </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAT </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>3 </td><td>37.5 </td><td>1 </td><td>50.0 </td><td>0 </td><td>— </td><td>2 </td><td>33.3 </td></tr><tr><td> GGCT </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGGA </td><td>18 </td><td>0.5 </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>8 </td><td>0.2 </td><td>3 </td><td>16.7 </td><td>2 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>12.5 </td></tr><tr><td> GGGT </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td>Total </td><td>325 </td><td>8.6 </td><td>41 </td><td>1.1 </td><td>42 </td><td>1.1 </td><td>242 </td><td>6.4 </td><td>62 </td><td>19.1 </td><td>8 </td><td>19.5 </td><td>8 </td><td>19.0 </td><td>46 </td><td>19.0 </td></tr><tr><td>Total </td><td>3800 </td><td>100.0 </td><td>1604 </td><td>42.2 </td><td>242 </td><td>6.4 </td><td>1954 </td><td>51.4 </td><td>642 </td><td>16.9 </td><td>270 </td><td>16.8 </td><td>49 </td><td>20.2 </td><td>323 </td><td>16.5 </td></tr></tbody></table></div><div content-id="DSR013TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="8">Designed SSR markers<hr /> </th><th colspan="8">Polymorphic SSR markers<hr /> </th></tr><tr><th>Total </th><th>%<sup>a</sup> </th><th>Coding region </th><th>%<sup>a</sup> </th><th>UTR </th><th>%<sup>a</sup> </th><th>Unassigned region<sup>b</sup> </th><th>%<sup>a</sup> </th><th>Total </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Coding region </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>UTR </th><th>Polymorphic ratio (%)<sup>c</sup> </th><th>Unassigned region<sup>b</sup> </th><th>Polymorphic ratio (%)<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="17">Dinucleotide </td></tr><tr><td> AC </td><td>60 </td><td>1.6 </td><td>3 </td><td>0.1 </td><td>7 </td><td>0.2 </td><td>50 </td><td>1.3 </td><td>14 </td><td>23.3 </td><td>2 </td><td>66.7 </td><td>1 </td><td>14.3 </td><td>11 </td><td>22.0 </td></tr><tr><td> AG </td><td>257 </td><td>6.8 </td><td>25 </td><td>0.7 </td><td>83 </td><td>2.2 </td><td>149 </td><td>3.9 </td><td>48 </td><td>18.7 </td><td>4 </td><td>16.0 </td><td>22 </td><td>26.5 </td><td>22 </td><td>14.8 </td></tr><tr><td> AT </td><td>63 </td><td>1.7 </td><td>2 </td><td>0.1 </td><td>6 </td><td>0.2 </td><td>55 </td><td>1.4 </td><td>14 </td><td>22.2 </td><td>1 </td><td>50.0 </td><td>1 </td><td>16.7 </td><td>12 </td><td>21.8 </td></tr><tr><td> GC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> Total </td><td>381 </td><td>10.0 </td><td>30 </td><td>0.8 </td><td>96 </td><td>2.5 </td><td>255 </td><td>6.7 </td><td>76 </td><td>19.9 </td><td>7 </td><td>23.3 </td><td>24 </td><td>25.0 </td><td>45 </td><td>17.6 </td></tr><tr><td colspan="17">Trinucleotide </td></tr><tr><td> AAC </td><td>305 </td><td>8.0 </td><td>140 </td><td>3.7 </td><td>9 </td><td>0.2 </td><td>156 </td><td>4.1 </td><td>53 </td><td>17.4 </td><td>27 </td><td>19.3 </td><td>1 </td><td>11.1 </td><td>25 </td><td>16.0 </td></tr><tr><td> AAG </td><td>815 </td><td>21.4 </td><td>374 </td><td>9.8 </td><td>49 </td><td>1.3 </td><td>392 </td><td>10.3 </td><td>119 </td><td>14.6 </td><td>55 </td><td>14.7 </td><td>7 </td><td>14.3 </td><td>57 </td><td>14.5 </td></tr><tr><td> AAT </td><td>103 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>10 </td><td>0.3 </td><td>80 </td><td>2.1 </td><td>20 </td><td>19.4 </td><td>2 </td><td>15.4 </td><td>2 </td><td>20.0 </td><td>16 </td><td>20.0 </td></tr><tr><td> ACG </td><td>158 </td><td>4.2 </td><td>74 </td><td>1.9 </td><td>6 </td><td>0.2 </td><td>78 </td><td>2.1 </td><td>23 </td><td>14.6 </td><td>9 </td><td>12.2 </td><td>1 </td><td>16.7 </td><td>13 </td><td>16.7 </td></tr><tr><td> ACT </td><td>65 </td><td>1.7 </td><td>22 </td><td>0.6 </td><td>0 </td><td>0.0 </td><td>43 </td><td>1.1 </td><td>11 </td><td>16.9 </td><td>3 </td><td>13.6 </td><td>0 </td><td>— </td><td>8 </td><td>18.6 </td></tr><tr><td> AGC </td><td>293 </td><td>7.7 </td><td>180 </td><td>4.7 </td><td>1 </td><td>0.0 </td><td>112 </td><td>2.9 </td><td>58 </td><td>19.8 </td><td>34 </td><td>18.9 </td><td>0 </td><td>0.0 </td><td>24 </td><td>21.4 </td></tr><tr><td> ATC </td><td>385 </td><td>10.1 </td><td>192 </td><td>5.1 </td><td>14 </td><td>0.4 </td><td>179 </td><td>4.7 </td><td>69 </td><td>17.9 </td><td>37 </td><td>19.3 </td><td>4 </td><td>28.6 </td><td>28 </td><td>15.6 </td></tr><tr><td> GGA </td><td>541 </td><td>14.2 </td><td>310 </td><td>8.2 </td><td>8 </td><td>0.2 </td><td>223 </td><td>5.9 </td><td>89 </td><td>16.5 </td><td>53 </td><td>17.1 </td><td>1 </td><td>12.5 </td><td>35 </td><td>15.7 </td></tr><tr><td> GGC </td><td>154 </td><td>4.1 </td><td>95 </td><td>2.5 </td><td>1 </td><td>0.0 </td><td>58 </td><td>1.5 </td><td>22 </td><td>14.3 </td><td>17 </td><td>17.9 </td><td>0 </td><td>0.0 </td><td>5 </td><td>8.6 </td></tr><tr><td> GGT </td><td>275 </td><td>7.2 </td><td>133 </td><td>3.5 </td><td>6 </td><td>0.2 </td><td>136 </td><td>3.6 </td><td>40 </td><td>14.5 </td><td>18 </td><td>13.5 </td><td>1 </td><td>16.7 </td><td>21 </td><td>15.4 </td></tr><tr><td> Total </td><td>3094 </td><td>81.4 </td><td>1533 </td><td>40.3 </td><td>104 </td><td>2.7 </td><td>1457 </td><td>38.3 </td><td>504 </td><td>16.3 </td><td>255 </td><td>16.6 </td><td>17 </td><td>16.3 </td><td>232 </td><td>15.9 </td></tr><tr><td colspan="17">Tetranucleotide </td></tr><tr><td> AAAC </td><td>81 </td><td>2.1 </td><td>3 </td><td>0.1 </td><td>11 </td><td>0.3 </td><td>67 </td><td>1.8 </td><td>17 </td><td>21.0 </td><td>1 </td><td>33.3 </td><td>2 </td><td>18.2 </td><td>14 </td><td>20.9 </td></tr><tr><td> AAAG </td><td>101 </td><td>2.7 </td><td>13 </td><td>0.3 </td><td>15 </td><td>0.4 </td><td>73 </td><td>1.9 </td><td>14 </td><td>13.9 </td><td>2 </td><td>15.4 </td><td>2 </td><td>13.3 </td><td>10 </td><td>13.7 </td></tr><tr><td> AAAT </td><td>53 </td><td>1.4 </td><td>3 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>46 </td><td>1.2 </td><td>14 </td><td>26.4 </td><td>1 </td><td>33.3 </td><td>1 </td><td>25.0 </td><td>12 </td><td>26.1 </td></tr><tr><td> AACG </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> AAGC </td><td>11 </td><td>0.3 </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>2 </td><td>18.2 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATC </td><td>24 </td><td>0.6 </td><td>4 </td><td>0.1 </td><td>4 </td><td>0.1 </td><td>16 </td><td>0.4 </td><td>6 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>3 </td><td>75.0 </td><td>3 </td><td>18.8 </td></tr><tr><td> AATG </td><td>9 </td><td>0.2 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>7 </td><td>0.2 </td><td>1 </td><td>11.1 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>14.3 </td></tr><tr><td> AATT </td><td>7 </td><td>0.2 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>1 </td><td>14.3 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>1 </td><td>16.7 </td></tr><tr><td> AGGC </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td></tr><tr><td> GACG </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td></tr><tr><td> GAGC </td><td>3 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>33.3 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>100.0 </td></tr><tr><td> GATC </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAC </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGAT </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>0 </td><td>0.0 </td><td>6 </td><td>0.2 </td><td>3 </td><td>37.5 </td><td>1 </td><td>50.0 </td><td>0 </td><td>— </td><td>2 </td><td>33.3 </td></tr><tr><td> GGCT </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td> GGGA </td><td>18 </td><td>0.5 </td><td>8 </td><td>0.2 </td><td>2 </td><td>0.1 </td><td>8 </td><td>0.2 </td><td>3 </td><td>16.7 </td><td>2 </td><td>25.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>12.5 </td></tr><tr><td> GGGT </td><td>2 </td><td>0.1 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>1 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>0.0 </td><td>0 </td><td>— </td><td>0 </td><td>0.0 </td></tr><tr><td>Total </td><td>325 </td><td>8.6 </td><td>41 </td><td>1.1 </td><td>42 </td><td>1.1 </td><td>242 </td><td>6.4 </td><td>62 </td><td>19.1 </td><td>8 </td><td>19.5 </td><td>8 </td><td>19.0 </td><td>46 </td><td>19.0 </td></tr><tr><td>Total </td><td>3800 </td><td>100.0 </td><td>1604 </td><td>42.2 </td><td>242 </td><td>6.4 </td><td>1954 </td><td>51.4 </td><td>642 </td><td>16.9 </td><td>270 </td><td>16.8 </td><td>49 </td><td>20.2 </td><td>323 </td><td>16.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Percentage to a total of 3800 SSR markers.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The position of SSRs was not assigned because of the length of each EST was not enough to predict open reading frames.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Percentage to the designed SSR markers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7725064" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7725065 id="7725065" class="section-title" data-legacy-id=s3c>3.3. Linkage analysis and map construction</h3>
<p>All 3800 SSR markers were used for polymorphic analysis of parental lines from the mapping population and 642 (16.9%) SSR markers were selected (Table <a reveal-id="DSR013TB1" data-open="DSR013TB1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These markers identified 18 double polymorphic loci and one triple polymorphic locus and, consequently, 662 polymorphic loci were generated from the 642 SSR markers. Among the 662 SSR loci, 545 and 117 identified co-dominant and dominant loci, respectively. Null alleles were observed in 98 and 19 of the 117 dominant markers from ‘HA2’ and ‘GSK3-1’, respectively. The polymorphic ratios of the di-, tri-, and tetra-nucleotide repeats were 19.9, 16.3, and 19.1%, respectively, and the polymorphic ratio of SSRs on UTRs (20.2%) were higher than that on coding regions (16.8%) (Table <a reveal-id="DSR013TB1" data-open="DSR013TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>Segregation data were generated from the 662 SSR and 223 RAPD, RFLP, and trait marker loci in the mapping population. Following linkage analysis, 843 loci (630 SSR and 213 RAPD, RFLP and trait loci) were mapped into nine linkage groups, while the remaining 42 loci (32 SSR and 10 RAPD, RFLP, and trait loci) were excluded from the analysis. Linkage groups were named according to the previous report.<sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The linkage groups covered 1129.2 cM (Table <a reveal-id="DSR013TB2" data-open="DSR013TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSR013F2" data-open="DSR013F2" class="link link-reveal link-table xref-fig">2</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) and the average distance between neighbouring loci was 1.3 cM, with distances ranging from 1.0 cM in linkage group 3 (LG3) to 2.0 cM in LG6. Segregation distortion was observed for 8.9% of the mapped marker loci (Table <a reveal-id="DSR013TB2" data-open="DSR013TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The segregation distortion ratios varied among linkage groups; LG5 showed distortions with 3.7% of the mapped loci, while LG2 showed distortions for 20.2% of the loci. </p>                        <a id="7725068" scrollto-destination="7725068"></a>
<div content-id="DSR013TB2" class="table-modal"><div class="table-wrap"><div id="DSR013TB2" class="table-wrap-title" data-id="DSR013TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Length, number of mapped loci, and segregation distortion of the genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th rowspan="2">Distance (cM) </th><th colspan="3">The number of marker loci<hr /> </th><th rowspan="2">cM/marker loci </th><th rowspan="2">Segregation distortion ratio (%) </th></tr><tr><th>Total </th><th>EST-SSR </th><th>RAPD </th></tr></thead><tbody><tr><td>1 </td><td>120.8 </td><td>82 </td><td>56 </td><td>26 </td><td>1.5 </td><td>7.3 </td></tr><tr><td>2 </td><td>149.9 </td><td>109 </td><td>88 </td><td>21 </td><td>1.4 </td><td>20.2 </td></tr><tr><td>3 </td><td>147.9 </td><td>155 </td><td>124 </td><td>31 </td><td>1.0 </td><td>7.7 </td></tr><tr><td>4 </td><td>148.0 </td><td>114 </td><td>82 </td><td>32 </td><td>1.3 </td><td>4.4 </td></tr><tr><td>5 </td><td>147.5 </td><td>107 </td><td>79 </td><td>28 </td><td>1.4 </td><td>3.7 </td></tr><tr><td>6 </td><td>153.2 </td><td>77 </td><td>50 </td><td>27 </td><td>2.0 </td><td>5.2 </td></tr><tr><td>7 </td><td>100.3 </td><td>84 </td><td>61 </td><td>23 </td><td>1.2 </td><td>10.7 </td></tr><tr><td>8 </td><td>85.4 </td><td>65 </td><td>49 </td><td>16 </td><td>1.3 </td><td>7.7 </td></tr><tr><td>9 </td><td>76.2 </td><td>50 </td><td>41 </td><td>9 </td><td>1.6 </td><td>16.0 </td></tr><tr><td>Total </td><td>1129.2 </td><td>843 </td><td>630 </td><td>213 </td><td>1.3 </td><td>8.9 </td></tr></tbody></table></div><div content-id="DSR013TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th rowspan="2">Distance (cM) </th><th colspan="3">The number of marker loci<hr /> </th><th rowspan="2">cM/marker loci </th><th rowspan="2">Segregation distortion ratio (%) </th></tr><tr><th>Total </th><th>EST-SSR </th><th>RAPD </th></tr></thead><tbody><tr><td>1 </td><td>120.8 </td><td>82 </td><td>56 </td><td>26 </td><td>1.5 </td><td>7.3 </td></tr><tr><td>2 </td><td>149.9 </td><td>109 </td><td>88 </td><td>21 </td><td>1.4 </td><td>20.2 </td></tr><tr><td>3 </td><td>147.9 </td><td>155 </td><td>124 </td><td>31 </td><td>1.0 </td><td>7.7 </td></tr><tr><td>4 </td><td>148.0 </td><td>114 </td><td>82 </td><td>32 </td><td>1.3 </td><td>4.4 </td></tr><tr><td>5 </td><td>147.5 </td><td>107 </td><td>79 </td><td>28 </td><td>1.4 </td><td>3.7 </td></tr><tr><td>6 </td><td>153.2 </td><td>77 </td><td>50 </td><td>27 </td><td>2.0 </td><td>5.2 </td></tr><tr><td>7 </td><td>100.3 </td><td>84 </td><td>61 </td><td>23 </td><td>1.2 </td><td>10.7 </td></tr><tr><td>8 </td><td>85.4 </td><td>65 </td><td>49 </td><td>16 </td><td>1.3 </td><td>7.7 </td></tr><tr><td>9 </td><td>76.2 </td><td>50 </td><td>41 </td><td>9 </td><td>1.6 </td><td>16.0 </td></tr><tr><td>Total </td><td>1129.2 </td><td>843 </td><td>630 </td><td>213 </td><td>1.3 </td><td>8.9 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7725068" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR013TB2" class="table-wrap-title" data-id="DSR013TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Length, number of mapped loci, and segregation distortion of the genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th rowspan="2">Distance (cM) </th><th colspan="3">The number of marker loci<hr /> </th><th rowspan="2">cM/marker loci </th><th rowspan="2">Segregation distortion ratio (%) </th></tr><tr><th>Total </th><th>EST-SSR </th><th>RAPD </th></tr></thead><tbody><tr><td>1 </td><td>120.8 </td><td>82 </td><td>56 </td><td>26 </td><td>1.5 </td><td>7.3 </td></tr><tr><td>2 </td><td>149.9 </td><td>109 </td><td>88 </td><td>21 </td><td>1.4 </td><td>20.2 </td></tr><tr><td>3 </td><td>147.9 </td><td>155 </td><td>124 </td><td>31 </td><td>1.0 </td><td>7.7 </td></tr><tr><td>4 </td><td>148.0 </td><td>114 </td><td>82 </td><td>32 </td><td>1.3 </td><td>4.4 </td></tr><tr><td>5 </td><td>147.5 </td><td>107 </td><td>79 </td><td>28 </td><td>1.4 </td><td>3.7 </td></tr><tr><td>6 </td><td>153.2 </td><td>77 </td><td>50 </td><td>27 </td><td>2.0 </td><td>5.2 </td></tr><tr><td>7 </td><td>100.3 </td><td>84 </td><td>61 </td><td>23 </td><td>1.2 </td><td>10.7 </td></tr><tr><td>8 </td><td>85.4 </td><td>65 </td><td>49 </td><td>16 </td><td>1.3 </td><td>7.7 </td></tr><tr><td>9 </td><td>76.2 </td><td>50 </td><td>41 </td><td>9 </td><td>1.6 </td><td>16.0 </td></tr><tr><td>Total </td><td>1129.2 </td><td>843 </td><td>630 </td><td>213 </td><td>1.3 </td><td>8.9 </td></tr></tbody></table></div><div content-id="DSR013TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th rowspan="2">Distance (cM) </th><th colspan="3">The number of marker loci<hr /> </th><th rowspan="2">cM/marker loci </th><th rowspan="2">Segregation distortion ratio (%) </th></tr><tr><th>Total </th><th>EST-SSR </th><th>RAPD </th></tr></thead><tbody><tr><td>1 </td><td>120.8 </td><td>82 </td><td>56 </td><td>26 </td><td>1.5 </td><td>7.3 </td></tr><tr><td>2 </td><td>149.9 </td><td>109 </td><td>88 </td><td>21 </td><td>1.4 </td><td>20.2 </td></tr><tr><td>3 </td><td>147.9 </td><td>155 </td><td>124 </td><td>31 </td><td>1.0 </td><td>7.7 </td></tr><tr><td>4 </td><td>148.0 </td><td>114 </td><td>82 </td><td>32 </td><td>1.3 </td><td>4.4 </td></tr><tr><td>5 </td><td>147.5 </td><td>107 </td><td>79 </td><td>28 </td><td>1.4 </td><td>3.7 </td></tr><tr><td>6 </td><td>153.2 </td><td>77 </td><td>50 </td><td>27 </td><td>2.0 </td><td>5.2 </td></tr><tr><td>7 </td><td>100.3 </td><td>84 </td><td>61 </td><td>23 </td><td>1.2 </td><td>10.7 </td></tr><tr><td>8 </td><td>85.4 </td><td>65 </td><td>49 </td><td>16 </td><td>1.3 </td><td>7.7 </td></tr><tr><td>9 </td><td>76.2 </td><td>50 </td><td>41 </td><td>9 </td><td>1.6 </td><td>16.0 </td></tr><tr><td>Total </td><td>1129.2 </td><td>843 </td><td>630 </td><td>213 </td><td>1.3 </td><td>8.9 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7725068" target="_blank">View Large</a></div></div>                        <a id="7725069" scrollto-destination="7725069"></a>
<div id="DSR013F2" data-id="DSR013F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01302a.jpeg?Expires=2147483647&amp;Signature=WmSNT~X3ZlcWG2bFetWVn~ZTgl3Xq~yw9tKJhtAxUkHvukTI0--JeZ9NRjFyQXaS95755SSJUxdza4WIo5T3a0SBEBb1k4oACRb5qU3t6IQD9Qk~X8HlEWbXuXCvb0eG0op071SJv7kE3GtchjrvTnVOCsm1uEB5NlASoFPgvGCCdNFqbqfwU0rkjhvk6mx0elLeDYOAr56oYdbk5KMvEpp5Kxm0eV483MG6FOTr-RvB3kVFgrSxZxq8dhNJPsZOtP9bLg5Qh-GbSKF~PfIIc6VaBYL1FtZCqIbKSoQCBJBEaXoPur0yAZ-IbSAXGUEIDT5XQTdHJ8gd9kYfL2E6fQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of R. sativus. The red, blue, yellow, green and pink lines on each linkage group correspond to chromosomes 1, 2, 3, 4 and 5 of A. thaliana. Detailed information about the map and markers is shown in Supplementary Table S2 and is also available at http://marker.kazusa.or.jp/Daikon." path-from-xml="dsr01302a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725069/dsr01302a.jpeg" path-from-xml="dsr01302a.jpeg" target="_blank">View large</a><a section="7725069" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01302a.jpeg?Expires=2147483647&amp;Signature=tmPZW1sTyOk~3eDNXtNwok69lvnyN45tVCqANBK35SZQWM6Q~ngz2Y8fkuv30VmLx24vx~o4HNvDwxlGUm8mCMeM35FlWrLGcI-iLUZ0ElQsm~4pLgaWb54x~wh6c1qJDRtoxcMx1oz0eAe1c-T-G9Y4E57oONC42nUTguDWfE-ybxF6FKIS-n-g34Ppwvd2Tcu2wcDPmWvc1Mu9dY6nkPs7zAiMIQryhF7Ej4M~CA6ib1FFoLoocySKlmzR-h0S8fMrSn-NsSBQFPC1oIiHLgV98aUr4GE~zg2LPssczFOMUzlgz0ThL8WqDSQorXi0kVU8vOfH5yw2JC8EvSG2Rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7725069&amp;ar=501057&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01302a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01302b.jpeg?Expires=2147483647&amp;Signature=YvNbC1uWqpdD11rp6gSlh0X5VkwlUHwzPCZinGb7zNjIIqJQq0uo0jREHFn5Z-HsD~nPzY1S6VFD-QMlN6RGENu-yxcxKxJ6aNnOcwSYDgzsv4D3-oToqjFE8F0soRkYQwwspl6oK6g7j3ceOFUXAySLDqsMbgXhfMlbrGgsiKkQYe1bAFGU7WNp1KhNTlIkN~eRKo8abAfJgRmrewhcnAmDWTNl0pjAJ-N54q923NmFzNrCdBrXd~FgL0wwgqyx92vTcn4FApXfQUDcpP~nK~Y8TctGnr1ZUJiq5N6AFo-aQDg018FezvVa5t3AHxIAgrUh2q5gTW31lhP9cY~Drw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of R. sativus. The red, blue, yellow, green and pink lines on each linkage group correspond to chromosomes 1, 2, 3, 4 and 5 of A. thaliana. Detailed information about the map and markers is shown in Supplementary Table S2 and is also available at http://marker.kazusa.or.jp/Daikon." path-from-xml="dsr01302b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725069/dsr01302b.jpeg" path-from-xml="dsr01302b.jpeg" target="_blank">View large</a><a section="7725069" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01302b.jpeg?Expires=2147483647&amp;Signature=wHaB4BQeB4Zdx2BOyT232QBXbwB6WE~nMLvZwl1u5dn6zjqpo12Nvmrd~WjGT33dDlhqXDYyTEb-tD~qnqTww3HcCmBZOcP1Jd6Yj~qBwRD~Y-O1XBlrNMUYNZTbmLg-K2FvfDN3hITtfG8Adkm9PCMzjDR2IQuoTU6KKNzD1Om4brvkJucnmkrmJug2ffr4pX973dxX-W9fszedo7GppO8BhiIv~BB2coHBzDE52tNiTap3CjY70NFDXFM0~tdsyUp1SpsPgLkjR52DRIqJJWUFDDSUfov8~Y3nG3ZS-prp~T-dA1dG7JZiV-yqHTz-ozISP2xzpGQd~1N~aywuyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7725069&amp;ar=501057&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01302b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage map of <em>R. sativus</em>. The red, blue, yellow, green and pink lines on each linkage group correspond to chromosomes 1, 2, 3, 4 and 5 of <em>A. thaliana</em>. Detailed information about the map and markers is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> and is also available at <a class="link link-uri" href="http://marker.kazusa.or.jp/Daikon">http://marker.kazusa.or.jp/Daikon</a>.</p></div></div><div content-id="DSR013F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01302a.jpeg?Expires=2147483647&amp;Signature=WmSNT~X3ZlcWG2bFetWVn~ZTgl3Xq~yw9tKJhtAxUkHvukTI0--JeZ9NRjFyQXaS95755SSJUxdza4WIo5T3a0SBEBb1k4oACRb5qU3t6IQD9Qk~X8HlEWbXuXCvb0eG0op071SJv7kE3GtchjrvTnVOCsm1uEB5NlASoFPgvGCCdNFqbqfwU0rkjhvk6mx0elLeDYOAr56oYdbk5KMvEpp5Kxm0eV483MG6FOTr-RvB3kVFgrSxZxq8dhNJPsZOtP9bLg5Qh-GbSKF~PfIIc6VaBYL1FtZCqIbKSoQCBJBEaXoPur0yAZ-IbSAXGUEIDT5XQTdHJ8gd9kYfL2E6fQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of R. sativus. The red, blue, yellow, green and pink lines on each linkage group correspond to chromosomes 1, 2, 3, 4 and 5 of A. thaliana. Detailed information about the map and markers is shown in Supplementary Table S2 and is also available at http://marker.kazusa.or.jp/Daikon." path-from-xml="dsr01302a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725069/dsr01302a.jpeg" path-from-xml="dsr01302a.jpeg" target="_blank">View large</a><a section="7725069" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01302a.jpeg?Expires=2147483647&amp;Signature=tmPZW1sTyOk~3eDNXtNwok69lvnyN45tVCqANBK35SZQWM6Q~ngz2Y8fkuv30VmLx24vx~o4HNvDwxlGUm8mCMeM35FlWrLGcI-iLUZ0ElQsm~4pLgaWb54x~wh6c1qJDRtoxcMx1oz0eAe1c-T-G9Y4E57oONC42nUTguDWfE-ybxF6FKIS-n-g34Ppwvd2Tcu2wcDPmWvc1Mu9dY6nkPs7zAiMIQryhF7Ej4M~CA6ib1FFoLoocySKlmzR-h0S8fMrSn-NsSBQFPC1oIiHLgV98aUr4GE~zg2LPssczFOMUzlgz0ThL8WqDSQorXi0kVU8vOfH5yw2JC8EvSG2Rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7725069&amp;ar=501057&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01302a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01302b.jpeg?Expires=2147483647&amp;Signature=YvNbC1uWqpdD11rp6gSlh0X5VkwlUHwzPCZinGb7zNjIIqJQq0uo0jREHFn5Z-HsD~nPzY1S6VFD-QMlN6RGENu-yxcxKxJ6aNnOcwSYDgzsv4D3-oToqjFE8F0soRkYQwwspl6oK6g7j3ceOFUXAySLDqsMbgXhfMlbrGgsiKkQYe1bAFGU7WNp1KhNTlIkN~eRKo8abAfJgRmrewhcnAmDWTNl0pjAJ-N54q923NmFzNrCdBrXd~FgL0wwgqyx92vTcn4FApXfQUDcpP~nK~Y8TctGnr1ZUJiq5N6AFo-aQDg018FezvVa5t3AHxIAgrUh2q5gTW31lhP9cY~Drw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of R. sativus. The red, blue, yellow, green and pink lines on each linkage group correspond to chromosomes 1, 2, 3, 4 and 5 of A. thaliana. Detailed information about the map and markers is shown in Supplementary Table S2 and is also available at http://marker.kazusa.or.jp/Daikon." path-from-xml="dsr01302b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725069/dsr01302b.jpeg" path-from-xml="dsr01302b.jpeg" target="_blank">View large</a><a section="7725069" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01302b.jpeg?Expires=2147483647&amp;Signature=wHaB4BQeB4Zdx2BOyT232QBXbwB6WE~nMLvZwl1u5dn6zjqpo12Nvmrd~WjGT33dDlhqXDYyTEb-tD~qnqTww3HcCmBZOcP1Jd6Yj~qBwRD~Y-O1XBlrNMUYNZTbmLg-K2FvfDN3hITtfG8Adkm9PCMzjDR2IQuoTU6KKNzD1Om4brvkJucnmkrmJug2ffr4pX973dxX-W9fszedo7GppO8BhiIv~BB2coHBzDE52tNiTap3CjY70NFDXFM0~tdsyUp1SpsPgLkjR52DRIqJJWUFDDSUfov8~Y3nG3ZS-prp~T-dA1dG7JZiV-yqHTz-ozISP2xzpGQd~1N~aywuyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7725069&amp;ar=501057&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01302b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage map of <em>R. sativus</em>. The red, blue, yellow, green and pink lines on each linkage group correspond to chromosomes 1, 2, 3, 4 and 5 of <em>A. thaliana</em>. Detailed information about the map and markers is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> and is also available at <a class="link link-uri" href="http://marker.kazusa.or.jp/Daikon">http://marker.kazusa.or.jp/Daikon</a>.</p></div></div>                            <h3 scrollto-destination=7725070 id="7725070" class="section-title" data-legacy-id=s3d>3.4. Genomic comparison of <em>R. sativus</em>, <em>A. thaliana</em>, and <em>B. rapa</em></h3>
<p>BLASTX was used to compare the nucleotide sequences of 630 ESTs mapped into linkage groups with the predicted-protein database of <em>A. thaliana</em>. Of the 630 ESTs, 358 showed significant homology (≤1e−50 <em>E</em>-value) to 500 <em>A. thaliana</em> genes, and 230, 87, and 22 ESTs mapped onto one, two, and three regions of the <em>A. thaliana</em> genome, respectively, while 19 other ESTs mapped onto four or five regions and 273 ESTs showed no significant homology to the <em>A. thaliana</em> genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). To compare the genome structures of <em>R. sativus</em> and <em>A. thaliana</em>, syntenic regions were deduced when more than three continuous EST loci on <em>R. sativus</em> linkage groups aligned to a single portion of the genome of <em>A. thaliana</em>. Chromosomal segments of <em>A. thaliana</em> were assigned onto the <em>R. sativus</em> genetic map (Fig. <a reveal-id="DSR013F2" data-open="DSR013F2" class="link link-reveal link-table xref-fig">2</a>). Approximately 65% (739.1 cM) of the genetic map was covered by 35 <em>A. thaliana</em> chromosome segments, which ranged from one segment on LG9 to six segments on LG3 and LG4. The longest segment (84.4 cM) was found on LG2 and it had derived from <em>A. thaliana</em> Chr2. The shortest segment (2.1 cM) was detected on LG7 and it arose from Chr1. The average length of a single segment was 21.1 cM.</p><p>Conversely, the <em>R. sativus</em> genetic map could be assigned to almost the whole <em>A. thaliana</em> genome, with the exception of the ribosomal DNA repeat regions on Chr2 and Chr4 (Fig. <a reveal-id="DSR013F3" data-open="DSR013F3" class="link link-reveal link-table xref-fig">3</a>). <em>R. sativus</em> linkage groups showed average mapped depths of 2.0 on the <em>A. thaliana</em> genome and depths ranged from 1 to 4. Two <em>R. sativus</em> linkage groups were redundantly assigned to 48% of the <em>A. thaliana</em> genome and the remaining 18, 24, and 4% of the <em>A. thaliana</em> genome were covered by single, triple, and quadruple <em>R. sativus</em> linkage groups, respectively. <em>A. thaliana</em> Chr3 was covered with two <em>R. sativus</em> LG3s, each of which was derived from both distal parts of LG3, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In addition to the <em>R. sativus</em> map, a published genetic map of <em>B. rapa</em> was also assigned onto the <em>A. thaliana</em> genome. This analysis confirmed that the <em>B. rapa</em> genome contains two or three copies of at least 24 genomic segments from <em>A. thaliana</em>, as suggested by a previous report.<sup><a reveal-id="DSR013C11" data-open="DSR013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The average mapped depth of <em>B. rapa</em> linkage groups on the <em>A. thaliana</em> genome was 1.4 and ranged from 1 to 3. Forty-eight per cent of the <em>A. thaliana</em> genome was covered by two <em>B. rapa</em> linkage groups<em>,</em> while 24 and 7% of the genome was covered by single and triple <em>B. rapa</em> linkage groups. Large syntenic segments were identified between the three species on the following <em>A. thaliana</em> (<em>At</em>) chromosomes: <em>At</em> Chr1, <em>B. rapa</em> (<em>Br</em>) LG9, <em>R. sativus</em> (<em>Rs</em>) LG7, and <em>Rs</em> LG9; <em>At</em> Chr2, <em>Br</em> LG3, <em>Br</em> LG4, <em>Rs</em> LG2, and LG4; <em>At</em> Chr3, <em>Br</em> LG3 and <em>Rs</em> LG3; <em>At</em> Chr4, <em>Br</em> LG 3, <em>Br</em> LG8, <em>Rs</em> LG4, <em>Rs</em> LG5, and <em>Rs</em> LG7; and <em>At</em> Ch5, <em>Br</em> LG10, and <em>Rs</em> LG2. Scattered syntenic segments were also observed on several genomic regions of <em>A. thaliana</em>. The most overlapping and disrupted syntenic region was identified on a distal part of the long arm of <em>At</em> Chr5. </p>                        <a id="7725073" scrollto-destination="7725073"></a>
<div id="DSR013F3" data-id="DSR013F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01303.jpeg?Expires=2147483647&Signature=B34dYSFa0Bh4GxXQ87SQGgFBqfVas5WYn02BZdIkzazdZVBu04-9zgLOzHJYp2rTIPEOncIBt6dABLuTOwKaXgmd~QEoEVrYzXho4ldqJK1KkEvyJjCBj~wSq0HwoYzacrWG6IhIiezdI2GO1cR-E0h5NHbG~qMpikfu9V-RLieCHV6JHh8soz-GDUg2Gs1He01p4TsuAXkqW2Xx3aJbMWD0-9n2PHDK~j1yx4evA5U9haioziMsAqiygOOqxgltxuiz--7iY33647OfvfHSBHIz9ny9RSK5WzZ-8YCBDoADKwjQz0PMrzAncP7HOxfqf5LWSTgGt87enfER0t0LHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny map of A. thaliana and R. sativus or B. rapa. A. thaliana chromosomes are shown with white bars. Telomeric and centromeric regions are indicated in black. Heterochromatic knobs and rDNA repeat regions are identified by grey circles and horizontal lines, respectively. The colored bars on the right and left sides of the A. thaliana chromosomes indicate linkage groups from R. sativus and B. rapa, respectively." path-from-xml="dsr01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725073/dsr01303.jpeg" path-from-xml="dsr01303.jpeg" target="_blank">View large</a><a section="7725073" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01303.jpeg?Expires=2147483647&Signature=sKb8GjHNcr4gZzl9MgW2txMiErR6kyLfsf~QOtX~6FAsuotNLGnccqr45i06bygmGHJ0-n3-BvYNYnrhEES9pPZhVRhLZ3e7AZiVp50Cg7Ku3nqlx0xp7GxNN-UgHogQdCcFYElpchyGpUdDicoRKmTL8Tnesnoze7gp88wAbZ95czNP0hoiRPQfXYyqr66wL8Iuiv7i-u4T~TPlRCgi9AxvzWWs~5kEZj02Gq51xgDotuKL7-JK9n9VqDENUftec7ohRvUdg8KR3Cq3HoXWjmZ6stAqHD1l3ZBRsUUcLJphcZRkmhNmK0FYUgt8WcraQFup2lLIn2M9Lf0on5wdHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7725073&ar=501057&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny map of <em>A. thaliana</em> and <em>R. sativus</em> or <em>B. rapa. A. thaliana</em> chromosomes are shown with white bars. Telomeric and centromeric regions are indicated in black. Heterochromatic knobs and rDNA repeat regions are identified by grey circles and horizontal lines, respectively. The colored bars on the right and left sides of the <em>A. thaliana</em> chromosomes indicate linkage groups from <em>R. sativus</em> and <em>B. rapa</em>, respectively.</p></div></div><div content-id="DSR013F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/m_dsr01303.jpeg?Expires=2147483647&Signature=B34dYSFa0Bh4GxXQ87SQGgFBqfVas5WYn02BZdIkzazdZVBu04-9zgLOzHJYp2rTIPEOncIBt6dABLuTOwKaXgmd~QEoEVrYzXho4ldqJK1KkEvyJjCBj~wSq0HwoYzacrWG6IhIiezdI2GO1cR-E0h5NHbG~qMpikfu9V-RLieCHV6JHh8soz-GDUg2Gs1He01p4TsuAXkqW2Xx3aJbMWD0-9n2PHDK~j1yx4evA5U9haioziMsAqiygOOqxgltxuiz--7iY33647OfvfHSBHIz9ny9RSK5WzZ-8YCBDoADKwjQz0PMrzAncP7HOxfqf5LWSTgGt87enfER0t0LHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny map of A. thaliana and R. sativus or B. rapa. A. thaliana chromosomes are shown with white bars. Telomeric and centromeric regions are indicated in black. Heterochromatic knobs and rDNA repeat regions are identified by grey circles and horizontal lines, respectively. The colored bars on the right and left sides of the A. thaliana chromosomes indicate linkage groups from R. sativus and B. rapa, respectively." path-from-xml="dsr01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7725073/dsr01303.jpeg" path-from-xml="dsr01303.jpeg" target="_blank">View large</a><a section="7725073" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr01303.jpeg?Expires=2147483647&Signature=sKb8GjHNcr4gZzl9MgW2txMiErR6kyLfsf~QOtX~6FAsuotNLGnccqr45i06bygmGHJ0-n3-BvYNYnrhEES9pPZhVRhLZ3e7AZiVp50Cg7Ku3nqlx0xp7GxNN-UgHogQdCcFYElpchyGpUdDicoRKmTL8Tnesnoze7gp88wAbZ95czNP0hoiRPQfXYyqr66wL8Iuiv7i-u4T~TPlRCgi9AxvzWWs~5kEZj02Gq51xgDotuKL7-JK9n9VqDENUftec7ohRvUdg8KR3Cq3HoXWjmZ6stAqHD1l3ZBRsUUcLJphcZRkmhNmK0FYUgt8WcraQFup2lLIn2M9Lf0on5wdHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7725073&ar=501057&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny map of <em>A. thaliana</em> and <em>R. sativus</em> or <em>B. rapa. A. thaliana</em> chromosomes are shown with white bars. Telomeric and centromeric regions are indicated in black. Heterochromatic knobs and rDNA repeat regions are identified by grey circles and horizontal lines, respectively. The colored bars on the right and left sides of the <em>A. thaliana</em> chromosomes indicate linkage groups from <em>R. sativus</em> and <em>B. rapa</em>, respectively.</p></div></div>                            <h2 scrollto-destination=7725074 id="7725074" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>A genetic map of <em>R. sativus</em> was constructed using 630 EST-SSR and 213 RAPD, RFLP, and trait marker loci. This map comprised nine linkage groups, which corresponded to the number of chromosomes in haploid <em>R. sativus</em> (<em>n</em> = 9). The genetic map was calculated to cover 1129.2 cM and the average interval between marker loci was 1.3 cM. To our knowledge, this map comprises the highest number and density of marker loci of all the published <em>R. sativus</em> genetic maps.<sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSR013C17  DSR013C19" data-open="DSR013C17  DSR013C19" class="link link-ref link-reveal xref-bibr">17–19</a>,<a reveal-id="DSR013C26" data-open="DSR013C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR013C27" data-open="DSR013C27" class="link link-ref link-reveal xref-bibr">27</a></sup> However, the genetic map generated in this study was shorter than the map reported by Budahn <em>et al.</em>,<sup><a reveal-id="DSR013C17" data-open="DSR013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> which was 1517 cM. That map was constructed using 245 F<sub>2</sub> plants and most of mapped markers were dominant, i.e. AFLPs and RAPDs. In contrast, the map from this study was developed using 155 RILs, with a large number of co-dominant SSR markers. These differences in length can be accounted for by the differences in size of mapping population, as well as in the number and types of DNA markers.<sup><a reveal-id="DSR013C28" data-open="DSR013C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Another contributing factor could be differences in the frequency of chromosome recombination caused by environmental factors and genetic diversity of the mapping parents.<sup><a reveal-id="DSR013C29" data-open="DSR013C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSR013C30" data-open="DSR013C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>EST-SSR markers are often preferable to RFLP, AFLP, and RAPD markers, since EST nucleotide sequences generally show higher levels of similarity across different species, genera, and families than sequences from intergenic regions. Therefore, the primers of EST-SSR markers are often directly applicable to PCR analyses of related species, although the efficiency of transferability depends upon the genetic distance between the species.<sup><a reveal-id="DSR013C6" data-open="DSR013C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Moreover, the multi-allelic nature of SSRs makes them more efficient than SNP markers for polymorphic analysis of diverse accessions. A preliminary experiment showed that <em>R. sativus</em> EST-SSR markers could be transferred to a mapping population derived from a cross between a Japanese daikon and a rat-tailed radish. PCR amplified specific DNA fragments in 665 (98%) of 676 tested EST-SSR markers and 291 (43%) markers showed polymorphism between the two lines. The order of mapped markers on this linkage map matched the order on the high-density linkage map from this study (data not shown).</p><p>EST-based markers are also valuable for comparisons of genome structure between different plant species. To demonstrate their utility, <em>R. sativus</em> and <em>A. thaliana</em> were compared using homoeologies between amino acid sequences predicted from the EST sequences and their locations on the present linkage map. The genetic map of <em>R. sativus</em> was covered by at least 35 disrupted <em>A. thaliana</em> genomic fragments and 90% of the genetic map corresponded to regions of the <em>A. thaliana</em> genome that had been doubled, tripled, or quadrupled. All ESTs that mapped onto LG9 in the <em>R. sativus</em> map corresponded to ca. 70% of <em>A. thaliana</em> Chr5. However, no perfectly homologous linkage groups were observed within the genetic map of <em>R. sativus</em>. These results suggest that the tripling of the <em>R. sativus</em> genome occurred at some point after divergence from its common ancestor with <em>A. thaliana</em>, and that this amplification was followed by structural rearrangements, including deletion of parts of the chromosomes. Alternatively, one part of the genome may have undergone a duplication event, while another part tripled. These events may have occurred at or after the time of divergence between <em>R. sativus</em> and <em>A. thaliana</em>, ca. 20 million years ago.<sup><a reveal-id="DSR013C31" data-open="DSR013C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR013C32" data-open="DSR013C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>In addition to comparative analysis between <em>R. sativus</em> and <em>A. thaliana</em>, the genome structure of <em>B. rapa</em> was compared with <em>R. sativus</em> via the <em>A. thaliana</em> genome. The two species shared large syntenic regions across the <em>A. thaliana</em> genome, which suggests that large regions of the <em>R. sativus</em> and <em>B. rapa</em> genomes are conserved. Several scattered syntenic regions were also observed on the <em>A. thaliana</em> genome, e.g. the distal part of the long arm of Chr5. However, there was no conservation in the order or composition of the corresponding genomic segments between <em>R. sativus</em> and <em>B. rapa</em> (Fig. <a reveal-id="DSR013F3" data-open="DSR013F3" class="link link-reveal link-table xref-fig">3</a>),<sup><a reveal-id="DSR013C11" data-open="DSR013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> which suggests no chromosomal synteny in this region among the three species. Absence of chromosomal synteny was also reported between <em>A. thaliana</em> and <em>B. oleracea</em> or <em>B. nigra</em>.<sup><a reveal-id="DSR013C9" data-open="DSR013C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR013C10" data-open="DSR013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> It has been suggested that the three <em>Brassica</em> diploids, i.e. <em>B. rapa</em>, <em>B. oleracea</em>, and <em>B. nigra</em>, evolved via chromosomal structural rearrangements and/or polyploidy.<sup><a reveal-id="DSR013C9  DSR013C11" data-open="DSR013C9  DSR013C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> The results from this study confirm the hypothesis that <em>R. sativus</em> and the three <em>Brassica</em> diploids differentiated independently from a common ancestor with <em>A. thaliana</em> and that this divergence occurred via genome rearrangement and/or polyploidy events at and/or after differentiation.</p><p>Since the first report for artificial amphidiploids between <em>R. sativus</em> and <em>B. oleracea</em>,<sup><a reveal-id="DSR013C33" data-open="DSR013C33" class="link link-ref link-reveal xref-bibr">33</a></sup> amphidiploids derived from crosses between several combinations of Brassicaceae have been synthesized.<sup><a reveal-id="DSR013C34" data-open="DSR013C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Because the amphidiploids were important materials for plant breeding due to their vigorous growths and adaptability for alien chromosomes, the mechanisms for the hybridization and the amphidiploid formation have been investigated by cytogenetic, epigenetic, and genomic approaches.<sup><a reveal-id="DSR013C33" data-open="DSR013C33" class="link link-ref link-reveal xref-bibr">33</a></sup> According to our results of the comparative genomics of the Brassicaceae, chromosomal synteny is unnecessary for hybridization and the amphidiploid formation, suggesting that genetic factors like <em>Ph1</em> reported in wheat,<sup><a reveal-id="DSR013C35" data-open="DSR013C35" class="link link-ref link-reveal xref-bibr">35</a></sup> micro-syntenies, and/or nuclear-organella interactions might be required.</p><p>As has been observed with the genetic and genomic studies of <em>Brassica</em> species,<sup><a reveal-id="DSR013C12" data-open="DSR013C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR013C13" data-open="DSR013C13" class="link link-ref link-reveal xref-bibr">13</a></sup> a comparative map between <em>R. sativus</em> and the <em>A. thaliana</em> genome will accelerate gene identification in <em>Raphanus</em> species. In particular, it will aid candidate gene isolation since a large number of genes/traits have already been characterized and mapped onto the <em>A. thaliana</em> genome. Kaneko <em>et al.</em><sup><a reveal-id="DSR013C14" data-open="DSR013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> identified a QTL associated with resistance to yellow disease caused by <em>Fusarium oxysporum</em> f. sp. <em>raphani</em>, and this QTL was linked to the OPJ14 locus on LG1. According to our comparative map between <em>R. sativus</em> and <em>A. thaliana</em>, this locus is predicted to correspond to the long arm of <em>A. thaliana</em> Chr1, 27 Mb from the top of the chromosome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). TIR-NBS genes have been implicated in disease resistance in many plant species.<sup><a reveal-id="DSR013C36" data-open="DSR013C36" class="link link-ref link-reveal xref-bibr">36</a></sup> A cluster of these genes was identified in the chromosomal region of <em>A. thaliana</em> that corresponds to resistance to yellow disease in the comparative map, a finding which suggests that a member of the TIR-NBS gene family is responsible for <em>F. oxysporum</em> resistance in <em>R. sativus</em>.</p><p>The <em>R. sativus</em> EST data, SSR markers and genetic map developed in this study can be used for various genetic analyses including gene mapping, QTL analysis, population genetics, marker-assisted breeding, and whole-genome sequencing studies. This study has provided a standard map for genomics, genetics, and breeding of <em>R. sativus</em> and related species.</p>                            <h2 scrollto-destination=7725082 id="7725082" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7725084 id="7725084" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute and the Mikado Kyowa Seed Co., Ltd.</p>                            <h2 scrollto-destination=7725086 id="7725086" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Prof. Kazunari Nomura (Nihon University, Japan) for providing the rat-tailed radish.</p>                            <h2 scrollto-destination=7725088 id="7725088" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR013C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deulvot</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Charrel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Marty</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">468</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-468" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-468</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly-multiplexed%20SNP%20genotyping%20for%20genetic%20mapping%20and%20germplasm%20diversity%20studies%20in%20pea&amp;author=C.%20Deulvot&amp;author=H.%20Charrel&amp;author=A.%20Marty&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20701750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.Y.</div></div>, et al. </div><div class="article-title">High-throughput genotyping with the GoldenGate assay in the complex genome of soybean</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">945</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-008-0726-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-008-0726-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20with%20the%20GoldenGate%20assay%20in%20the%20complex%20genome%20of%20soybean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=I.Y.%20Choi&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=945-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0726-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0726-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0726-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Fickus</div> <div class="given-names">E.W.</div></div>, et al. </div><div class="article-title">High-throughput SNP discovery and assay development in common bean</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">475</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-475" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-475</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20assay%20development%20in%20common%20bean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=E.W.%20Fickus&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=475" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20712881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muchero</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Diop</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>, et al. </div><div class="article-title">A consensus genetic map of cowpea [<em>Vigna unguiculata</em> (L) Walp.] and synteny based on EST-derived SNPs</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">18159</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0905886106" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0905886106</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20genetic%20map%20of%20cowpea%20%5BVigna%20unguiculata%20(L)%20Walp.%5D%20and%20synteny%20based%20on%20EST-derived%20SNPs&amp;author=W.%20Muchero&amp;author=N.N.%20Diop&amp;author=P.R.%20Bhat&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=18159-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905886106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905886106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905886106"> </span></div> </div></div></div></div></div><div content-id="DSR013C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">381</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsq024" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsq024</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kalia</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Rai</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Kalia</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dhawan</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Microsatellite markers: an overview of the recent progress in plants</div>, <div class="source">Euphytica</div> , <div class="year">2011</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">309</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10681-010-0286-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10681-010-0286-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%3A%20an%20overview%20of%20the%20recent%20progress%20in%20plants&amp;author=R.K.%20Kalia&amp;author=M.K.%20Rai&amp;author=S.%20Kalia&amp;author=R.%20Singh&amp;author=A.K.%20Dhawan&amp;publication_year=2011&amp;journal=Euphytica&amp;volume=117&amp;pages=309-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-010-0286-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-010-0286-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-010-0286-9"> </span></div> </div></div></div></div></div><div content-id="DSR013C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="collab">The Arabidopsis Genome Initiative</div><div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/35048692" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/35048692</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">U</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Genome-analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization</div>, <div class="source">Jpn. J. Bot.</div> , <div class="year">1935</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">389</div>-<div class="lpage">452</div>)</div></div></div></div><div content-id="DSR013C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagercrantz</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Comparative mapping between <em>Arabidopsis thaliana</em> and <em>Brassica nigra</em> indicates that Brassica genomes have evolved through extensive genome replication accompanied by chromosome fusions and frequent rearrangements</div>, <div class="source">Genetics</div> , <div class="year">1998</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">1217</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20between%20Arabidopsis%20thaliana%20and%20Brassica%20nigra%20indicates%20that%20Brassica%20genomes%20have%20evolved%20through%20extensive%20genome%20replication%20accompanied%20by%20chromosome%20fusions%20and%20frequent%20rearrangements&amp;author=U.%20Lagercrantz&amp;publication_year=1998&amp;journal=Genetics&amp;volume=150&amp;pages=1217-28" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9799273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lan</div> <div class="given-names">T.H.</div></div>,  <div class="name"><div class="surname">DelMonte</div> <div class="given-names">T.A.</div></div>,  <div class="name"><div class="surname">Reischmann</div> <div class="given-names">K.P.</div></div>, et al. </div><div class="article-title">An EST-enriched comparative map of <em>Brassica oleracea</em> and <em>Arabidopsis thaliana</em></div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">776</div>-<div class="lpage">88</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.10.6.776" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.10.6.776</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20EST-enriched%20comparative%20map%20of%20Brassica%20oleracea%20and%20Arabidopsis%20thaliana&amp;author=T.H.%20Lan&amp;author=T.A.%20DelMonte&amp;author=K.P.%20Reischmann&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=776-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.6.776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.6.776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.6.776"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10854410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Chung</div> <div class="given-names">T.Y.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">G.J.</div></div>, et al. </div><div class="article-title">A sequence-tagged linkage map of <em>Brassica rapa</em></div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">29</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.106.060152" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.106.060152</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sequence-tagged%20linkage%20map%20of%20Brassica%20rapa&amp;author=J.S.%20Kim&amp;author=T.Y.%20Chung&amp;author=G.J.%20King&amp;publication_year=2006&amp;journal=Genetics&amp;volume=174&amp;pages=29-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.060152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.060152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.060152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16988107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kitashiba</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inaba</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A <em>Brassica rapa</em> linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">311</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsp020" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsp020</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Brassica%20rapa%20linkage%20map%20of%20EST-based%20SNP%20markers%20for%20identification%20of%20candidate%20genes%20controlling%20flowering%20time%20and%20leaf%20morphological%20traits&amp;author=F.%20Li&amp;author=H.%20Kitashiba&amp;author=K.%20Inaba&amp;author=T.%20Nishio&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=311-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19884167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsukazaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Simple sequence repeat-based comparative genomics between <em>Brassica rapa</em> and <em>Arabidopsis thaliana</em>: the genetic origin of clubroot resistance</div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">173</div> (pg. <div class="fpage">309</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.104.038968" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.104.038968</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat-based%20comparative%20genomics%20between%20Brassica%20rapa%20and%20Arabidopsis%20thaliana%3A%20the%20genetic%20origin%20of%20clubroot%20resistance&amp;author=K.%20Suwabe&amp;author=H.%20Tsukazaki&amp;author=H.%20Iketani&amp;publication_year=2006&amp;journal=Genetics&amp;volume=173&amp;pages=309-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.038968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.038968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.038968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16723420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kimizuka-Takagi</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bang</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Matsuzawa</div> <div class="given-names">Y.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Kole</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Radish</div>, <div class="source">Genome Mapping and Molecular Breeding in Plants</div> , <div class="year">2007</div>, vol. <div class="volume">vol. 5</div> <div class="publisher-loc">New York</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">141</div>-<div class="lpage">60</div>)</div></div></div></div><div content-id="DSR013C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kitamura</div> <div class="given-names">S.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Nishiyama</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Cultivars of radish and their change</div>, <div class="source">Japanese Radish</div> , <div class="year">1958</div><div class="publisher-loc">Tokyo</div><div class="publisher-name">Japan Society for the Promotion of Science</div>(pg. <div class="fpage">1</div>-<div class="lpage">19</div>) <div class="comment">(in Japanese)</div></div></div></div></div><div content-id="DSR013C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arumuganathan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Earle</div> <div class="given-names">E.D.</div></div>. </div><div class="article-title">Nuclear DNA content of some important plant</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="year">1991</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">208</div>-<div class="lpage">18</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF02672069" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF02672069</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20of%20some%20important%20plant&amp;author=K.%20Arumuganathan&amp;author=E.D.%20Earle&amp;publication_year=1991&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=9&amp;pages=208-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02672069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02672069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02672069"> </span></div> </div></div></div></div></div><div content-id="DSR013C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Budahn</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Peterka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Mousa</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Ding</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Molecular mapping in oil radish (<em>Raphanus sativus</em> L.) and QTL analysis of resistance against beet cyst nematode (<em>Heterodera schachtii</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">775</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-008-0937-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-008-0937-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20in%20oil%20radish%20(Raphanus%20sativus%20L.)%20and%20QTL%20analysis%20of%20resistance%20against%20beet%20cyst%20nematode%20(Heterodera%20schachtii)&amp;author=H.%20Budahn&amp;author=H.%20Peterka&amp;author=M.A.%20Mousa&amp;author=Y.%20Ding&amp;author=S.%20Zhang&amp;author=J.%20Li&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=775-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0937-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0937-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0937-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19050847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kamei</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tsuro</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">QTL mapping of clubroot resistance in radish (<em>Raphanus sativus</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">1021</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-009-1230-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-009-1230-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20of%20clubroot%20resistance%20in%20radish%20(Raphanus%20sativus%20L.)&amp;author=A.%20Kamei&amp;author=M.%20Tsuro&amp;author=N.%20Kubo&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=1021-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1230-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1230-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1230-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20012934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuro</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mapping of QTLs controlling root shape and red pigmentation in radish, <em>Raphanus sativus</em> L.</div>, <div class="source">Breed. Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">55</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1270/jsbbs.58.55" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1270/jsbbs.58.55</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20QTLs%20controlling%20root%20shape%20and%20red%20pigmentation%20in%20radish%2C%20Raphanus%20sativus%20L.&amp;author=M.%20Tsuro&amp;author=K.%20Suwabe&amp;author=N.%20Kubo&amp;author=S.%20Matsumoto&amp;author=M.%20Hirai&amp;publication_year=2008&amp;journal=Breed.%20Sci.&amp;volume=58&amp;pages=55-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.58.55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.58.55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.58.55"> </span></div> </div></div></div></div></div><div content-id="DSR013C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pratense</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">301</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsi018" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsi018</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(Trifolium%20pratense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>, et al. </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">41</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-4-41" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-4-41</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">An interspecific linkage map of SSR and intronic polymorphism markers in tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">731</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-010-1344-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-010-1344-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20interspecific%20linkage%20map%20of%20SSR%20and%20intronic%20polymorphism%20markers%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=E.%20Asamizu&amp;author=H.%20Fukuoka&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1344-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1344-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1344-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20431859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botella</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Coleman</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Nishimura</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Somerville</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Map positions of 47 <em>Arabidopsis</em> sequences with sequence similarity to disease resistance genes</div>, <div class="source">Plant J.</div> , <div class="year">1997</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1197</div>-<div class="lpage">211</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.1997.12051197.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-313X.1997.12051197.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Map%20positions%20of%2047%20Arabidopsis%20sequences%20with%20sequence%20similarity%20to%20disease%20resistance%20genes&amp;author=M.A.%20Botella&amp;author=M.J.%20Coleman&amp;author=D.E.%20Hughes&amp;author=M.T.%20Nishimura&amp;author=J.D.%20Jones&amp;author=S.C.%20Somerville&amp;publication_year=1997&amp;journal=Plant%20J.&amp;volume=12&amp;pages=1197-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1997.12051197.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1997.12051197.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1997.12051197.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9418057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fourmann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Barret</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Froger</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">From <em>Arabidopsis thaliana</em> to <em>Brassica napus</em>: development of amplified consensus genetic markers (ACGM) for construction of a gene map</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">1196</div>-<div class="lpage">206</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-002-1040-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-002-1040-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20Arabidopsis%20thaliana%20to%20Brassica%20napus%3A%20development%20of%20amplified%20consensus%20genetic%20markers%20(ACGM)%20for%20construction%20of%20a%20gene%20map&amp;author=M.%20Fourmann&amp;author=P.%20Barret&amp;author=N.%20Froger&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=105&amp;pages=1196-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1040-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1040-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1040-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">JoinMap<sup>®</sup>4</div>, <div class="source">Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div> , <div class="year">2006</div><div class="publisher-loc">Wageningen, The Netherlands</div><div class="publisher-name">Kyazma, B.V</div></div></div></div></div><div content-id="DSR013C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bett</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Genetic analysis and genome mapping in <em>Raphanus</em></div>, <div class="source">Genome</div> , <div class="year">2003</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">423</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/g03-026" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/g03-026</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20and%20genome%20mapping%20in%20Raphanus&amp;author=K.E.%20Bett&amp;author=D.J.%20Lydiate&amp;publication_year=2003&amp;journal=Genome&amp;volume=46&amp;pages=423-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12834058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuro</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Construction of a molecular linkage map of radish (<em>Raphanus sativus</em> L.), based on AFLP and <em>Brassica</em>-SSR markers</div>, <div class="source">Breed. Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">107</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1270/jsbbs.55.107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1270/jsbbs.55.107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20molecular%20linkage%20map%20of%20radish%20(Raphanus%20sativus%20L.)%2C%20based%20on%20AFLP%20and%20Brassica-SSR%20markers&amp;author=M.%20Tsuro&amp;author=K.%20Suwabe&amp;author=N.%20Kubo&amp;author=S.%20Matsumoto&amp;author=M.%20Hirai&amp;publication_year=2005&amp;journal=Breed.%20Sci.&amp;volume=55&amp;pages=107-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.55.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.55.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.55.107"> </span></div> </div></div></div></div></div><div content-id="DSR013C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gosselin</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bousquet</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Isabel</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Megagametophyte-derived linkage maps of white spruce (<em>Picea glauca</em>) based on RAPD, SCAR and ESTP markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">987</div>-<div class="lpage">97</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-001-0823-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-001-0823-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Megagametophyte-derived%20linkage%20maps%20of%20white%20spruce%20(Picea%20glauca)%20based%20on%20RAPD%2C%20SCAR%20and%20ESTP%20markers&amp;author=I.%20Gosselin&amp;author=Y.%20Zhou&amp;author=J.%20Bousquet&amp;author=N.%20Isabel&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=987-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-001-0823-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-001-0823-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-001-0823-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allard</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Evidence for genetic restriction of recombination in the lima bean</div>, <div class="source">Genetics</div> , <div class="year">1963</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1389</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20genetic%20restriction%20of%20recombination%20in%20the%20lima%20bean&amp;author=R.W.%20Allard&amp;publication_year=1963&amp;journal=Genetics&amp;volume=48&amp;pages=1389-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17248187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Axelsson</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bowman</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Lagercrantz</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Amphidiploid <em>Brassica juncea</em> contains conserved progenitor genomes</div>, <div class="source">Genome</div> , <div class="year">2000</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">679</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amphidiploid%20Brassica%20juncea%20contains%20conserved%20progenitor%20genomes&amp;author=T.%20Axelsson&amp;author=C.M.%20Bowman&amp;author=A.G.%20Sharpe&amp;author=D.J.%20Lydiate&amp;author=U.%20Lagercrantz&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=679-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-43-4-679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-43-4-679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-43-4-679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10984181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Y.W.</div></div>,  <div class="name"><div class="surname">Lai</div> <div class="given-names">K.N.</div></div>,  <div class="name"><div class="surname">Tai</div> <div class="given-names">P.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Rates of nucleotide substitution in angiosperm mitochondrial DNA sequences and dates of divergence between <em>Brassica</em> and other angiosperm lineages</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">597</div>-<div class="lpage">604</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/PL00006502" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/PL00006502</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rates%20of%20nucleotide%20substitution%20in%20angiosperm%20mitochondrial%20DNA%20sequences%20and%20dates%20of%20divergence%20between%20Brassica%20and%20other%20angiosperm%20lineages&amp;author=Y.W.%20Yang&amp;author=K.N.%20Lai&amp;author=P.Y.%20Tai&amp;author=W.H.%20Li&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol.&amp;volume=48&amp;pages=597-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10198125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koch</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Haubold</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Comparative evolutionary analysis of chalcone synthase and alcohol dehydrogenase loci in <em>Arabidopsis</em>, <em>Arabis</em>, and related genera (Brassicaceae)</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1483</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20evolutionary%20analysis%20of%20chalcone%20synthase%20and%20alcohol%20dehydrogenase%20loci%20in%20Arabidopsis%2C%20Arabis%2C%20and%20related%20genera%20(Brassicaceae)&amp;author=M.A.%20Koch&amp;author=B.%20Haubold&amp;author=T.%20Mitchell-Olds&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1483-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11018155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karpechenko</div> <div class="given-names">G.D.</div></div>. </div><div class="article-title">Poliploid hybrids <em>Raphanus sativus</em> L. times <em>Brassica oleracea</em> L.</div>, <div class="source">Bull. Appl. Bot.</div> , <div class="year">1927</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">305</div>-<div class="lpage">410</div>)</div></div></div></div><div content-id="DSR013C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snowdon</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Cytogenetics and genome analysis in <em>Brassica</em> crops</div>, <div class="source">Chromosome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">85</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10577-006-1105-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10577-006-1105-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytogenetics%20and%20genome%20analysis%20in%20Brassica%20crops&amp;author=R.J.%20Snowdon&amp;publication_year=2007&amp;journal=Chromosome%20Res.&amp;volume=15&amp;pages=85-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-006-1105-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-006-1105-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-006-1105-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17295129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR013C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riley</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chapman</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Genetic control of the cytologically diploid behaviour of hexaploid wheat</div>, <div class="source">Nature</div> , <div class="year">1959</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">713</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/182713a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/182713a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20the%20cytologically%20diploid%20behaviour%20of%20hexaploid%20wheat&amp;author=R.%20Riley&amp;author=V.%20Chapman&amp;publication_year=1959&amp;journal=Nature&amp;volume=182&amp;pages=713-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/182713a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F182713a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F182713a0"> </span></div> </div></div></div></div></div><div content-id="DSR013C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR013C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyers</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Kozik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Griego</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kuang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Michelmore</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Genome-wide analysis of NBS-LRR-encoding genes in Arabidopsis</div>, <div class="source">Plant Cell</div> , <div class="year">2003</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">809</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.009308" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.009308</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20NBS-LRR-encoding%20genes%20in%20Arabidopsis&amp;author=B.C.%20Meyers&amp;author=A.%20Kozik&amp;author=A.%20Griego&amp;author=H.%20Kuang&amp;author=R.W.%20Michelmore&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=809-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.009308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.009308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.009308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=501057&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr013-dsr013supp_table2.xls?Expires=2147483647&amp;Signature=v0Pzv41hXQ0lELZAzlOW90A-Yjv82TgEa65LusA-mg6hNB2IivVdabdEMO2w101J9VBY3Ll03YzdHw33EsPYY9x5PFMEZ7wBmyFKntfBffR65RTUXzSvaV6ai2b4ZtIiHRfglOKtEfH2agLnrgDCiGbnUovl9y9YT6Jo1z7DIIlS3RW~g1e~DfqOVuS0hMT0FDAaUQnDnBNqNHDA9xtCOM~YLcaOaeAYTDOgn4hkFUUGgv~sFiEFfJyPiXMIP8FSxz-T63Urrx-Wxq6eE8BeqFkfFMBhkXRp2sAjDbDCFy6E51AyyxlLE8-RghupMT75ES3bwCt2mI32Ku~mC2NHFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr013/2/dsr013-dsr013supp_table1.xls?Expires=2147483647&amp;Signature=19q~Qj14i9UOYTgSCbf9vjTzhBrvKQSV5QsKEFPiXKxkzSMGpOSy3rLlHZwJ1Oos5OIdRXy8hnn2iur-rI5IgJXB0yMBGJLIpnAr-Z~2Ji-bWqToQtfwNvRETp5NIZqiuzaSDflR3g7phjU5PqVze2ioMx-uhFEf5-M4VWzCfnDUrHHYD1Zvv9Rt~Ogztu7dW~yR3H75vSPBGXIHjlaUI4yedEegWO~OkLwxtdBicx1s9iPQG-SUie90BZaQyItpg~ldOVG-WoOtGvqwUle7d28vafplhJG5Z5jHXJsquPKIJgY8tG4v0VGUqBLbwrPzcLqR3uXjiZ3EbkHB9DNjqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">271</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">45</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="501057"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f4%2f221%2f501057">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000294936100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (45)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F4%2F221%2F501057" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/501057'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="EST-SSR Linkage Map of Raphanus sativus and Comparative Genomics of the Brassicaceae†" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f221%2f501057">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

