

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/3/205/591312","@type":"ScholarlyArticle","name":"Functional classification of 15 million SNPs detected from diverse chicken populations","datePublished":"2015-04-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2015-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv005","keywords":["SNP","next-generation sequencing","chicken","selection signature","functional variants"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/3/205/591312","author":[{"name":"Gheyas, Almas A.","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"},{"name":"Boschiero, Clarissa","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"},{"name":"Eory, Lel","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"},{"name":"Ralph, Hannah","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"},{"name":"Kuo, Richard","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"},{"name":"Woolliams, John A.","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"},{"name":"Burt, David W.","affiliation":"The Roslin Institute and Royal (Dick) School of Veterinary Studies ,  University of Edinburgh ,  Easter Bush Campus, Midlothian EH25 9RG ,  UK","@type":"Person"}],"description":"Abstract.  Next-generation sequencing has prompted a surge of discovery of millions of genetic variants from vertebrate genomes. Besides applications in genetic","pageStart":"205","pageEnd":"217","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Functional classification of 15 million SNPs detected from diverse chicken populations","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Functional classification of 15 million SNPs detected from diverse chicken populations" />
    <meta property="og:description" content="Abstract.  Next-generation sequencing has prompted a surge of discovery of millions of genetic variants from vertebrate genomes. Besides applications in genetic" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/3/205/591312" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-04-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Gheyas, Almas A." /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_author" content="Boschiero, Clarissa" /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_author" content="Eory, Lel" /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_author" content="Ralph, Hannah" /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_author" content="Kuo, Richard" /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_author" content="Woolliams, John A." /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_author" content="Burt, David W." /><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Easter Bush Campus, Midlothian EH25 9RG, UK" /><meta name="citation_title" content="Functional classification of 15 million SNPs detected from diverse chicken populations" /><meta name="citation_firstpage" content="205" /><meta name="citation_lastpage" content="217" /><meta name="citation_doi" content="10.1093/dnares/dsv005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2015/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=SIFT: predicting amino acid changes that affect protein function; citation_author=Ng  P.C.; citation_author=Henikoff  S.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3812-4; " /><meta name="citation_reference" content="citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm; citation_author=Kumar  P.; citation_author=Henikoff  S.; citation_author=Ng  P.C.;  citation_journal_title=Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=1073-82; " /><meta name="citation_reference" content="citation_title=Predicting the functional effect of amino acid substitutions and indels; citation_author=Choi  Y.; citation_author=Sims  G.E.; citation_author=Murphy  S.; citation_author=Miller  J.R.; citation_author=Chan  A.P.;  citation_journal_title=Plos ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e46688" /><meta name="citation_reference" content="citation_title=Needles in stacks of needles: finding disease-causal variants in a wealth of genomic data; citation_author=Cooper  G.M.; citation_author=Shendure  J.;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=628-40; " /><meta name="citation_reference" content="citation_title=The Human Gene Mutation Database: building a comprehensive mutation repository for clinical and molecular genetics, diagnostic testing and personalized genomic medicine; citation_author=Stenson  P.D.; citation_author=Mort  M.; citation_author=Ball  E.V.; citation_author=Shaw  K.; citation_author=Phillips  A.D.; citation_author=Cooper  D.N.;  citation_journal_title=Hum. Genet.;  citation_year=2014;  citation_volume=133;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; citation_author=International Human Genome sequencing Consortium,; citation_author=Lander  E.S.; citation_author=Linton  L.M.;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=An integrated encyclopedia of DNA elements in the human genome; citation_author=ENCODE-Project-Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=489;  citation_pages=57-74; " /><meta name="citation_reference" content="citation_title=Potential etiologic and functional implications of genome-wide association loci for human diseases and traits; citation_author=Hindorff  L.A.; citation_author=Sethupathy  P.; citation_author=Junkins  H.A.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=9362-7; " /><meta name="citation_reference" content="citation_title=Interpreting noncoding genetic variation in complex traits and human disease; citation_author=Ward  L.D.; citation_author=Kellis  M.;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=1095-106; " /><meta name="citation_reference" content="citation_title=Deleterious mutation prediction in the secondary structure of RNAs; citation_author=Barash  D.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6578-84; " /><meta name="citation_reference" content="citation_title=RDMAS: a web server for RNA deleterious mutation analysis; citation_author=Shu  W.J.; citation_author=Bo  X.C.; citation_author=Liu  R.J.; citation_author=Zhao  D.S.; citation_author=Zheng  Z.Q.; citation_author=Wang  S.Q.;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=404" /><meta name="citation_reference" content="citation_title=RNAsnp: efficient detection of local RNA secondary structure changes induced by SNPs (vol 34, pg 546, 2013); citation_author=Sabarinathan  R.; citation_author=Tafer  H.; citation_author=Seemann  S.E.; citation_author=Hofacker  I.L.; citation_author=Stadler  P.F.; citation_author=Gorodkin  J.;  citation_journal_title=Hum. Mutat.;  citation_year=2013;  citation_volume=34;  citation_pages=546-56; " /><meta name="citation_reference" content="citation_title=Disease-associated mutations that alter the RNA structural ensemble; citation_author=Halvorsen  M.; citation_author=Martin  J.S.; citation_author=Broadaway  S.; citation_author=Laederach  A.;  citation_journal_title=PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1001074" /><meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes; citation_author=Siepel  A.; citation_author=Bejerano  G.; citation_author=Pedersen  J.S.;  citation_journal_title=Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1034-50; " /><meta name="citation_reference" content="citation_title=Identifying a high fraction of the human genome to be under selective constraint using GERP plus; citation_author=Davydov  E.V.; citation_author=Goode  D.L.; citation_author=Sirota  M.; citation_author=Cooper  G.M.; citation_author=Sidow  A.; citation_author=Batzoglou  S.;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2010;  citation_volume=6;  citation_pages=e1001025" /><meta name="citation_reference" content="citation_title=Prospects for identifying functional variation across the genome; citation_author=Macdonald  S.J.; citation_author=Long  A.D.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=6614-21; " /><meta name="citation_reference" content="citation_title=Positive natural selection in the human lineage; citation_author=Sabeti  P.C.; citation_author=Schaffner  S.F.; citation_author=Fry  B.;  citation_journal_title=Science;  citation_year=2006;  citation_volume=312;  citation_pages=1614-20; " /><meta name="citation_reference" content="citation_title=Molecular signatures of natural selection; citation_author=Nielsen  R.;  citation_journal_title=Annu. Rev. Genet.;  citation_year=2005;  citation_volume=39;  citation_pages=197-218; " /><meta name="citation_reference" content="citation_title=Development of a high density 600 K SNP genotyping array for chicken; citation_author=Kranis  A.; citation_author=Gheyas  A.A.; citation_author=Boschiero  C.;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=59" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=Li  H.; citation_author=Durbin  R.;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=Li  H.; citation_author=Handsaker  B.; citation_author=Wysoker  A.;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data; citation_author=Wang  K.; citation_author=Li  M.Y.; citation_author=Hakonarson  H.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e164" /><meta name="citation_reference" content="UCSC Multiple alignments of 6 vertebrate genomes with chicken. http://hgdownload.cse.ucsc.edu/goldenPath/galGal3/multiz7way/(14 April 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=The RNAsnp web server: predicting SNP effects on local RNA secondary structure; citation_author=Sabarinathan  R.; citation_author=Tafer  H.; citation_author=Seemann  S.E.; citation_author=Hofacker  I.L.; citation_author=Stadler  P.F.; citation_author=Gorodkin  J.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=W475-9; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate - a practical and powerful approach to multiple testing; citation_author=Benjamini  Y.; citation_author=Hochberg  Y.;  citation_journal_title=J. Roy. Stat. Soc. B Met.;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication; citation_author=Rubin  C.J.; citation_author=Zody  M.C.; citation_author=Eriksson  J.;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=587-91; " /><meta name="citation_reference" content="citation_title=Empirical threshold values for quantitative trait mapping; citation_author=Churchill  G.A.; citation_author=Doerge  R.W.;  citation_journal_title=Genetics;  citation_year=1994;  citation_volume=138;  citation_pages=963-71; " /><meta name="citation_reference" content="citation_title=A high resolution genome-wide scan for significant selective sweeps: an application to pooled sequence data in laying chickens; citation_author=Qanbari  S.; citation_author=Strom  T.M.; citation_author=Haberer  G.;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e49525" /><meta name="citation_reference" content="citation_title=Whole genome SNP discovery and analysis of genetic diversity in Turkey (Meleagris gallopavo); citation_author=Aslam  M.L.; citation_author=Bastiaansen  J.W.M.; citation_author=Elferink  M.G.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=391" /><meta name="citation_reference" content="citation_title=Conservation and losses of non-coding RNAs in avian genomes; citation_author=Gardner  P.P.; citation_author=Fasold  M.; citation_author=Burge  S.W.;  citation_journal_title=PLoS ONE;  citation_year=2015;  citation_volume=10;  citation_pages=e0121797" /><meta name="citation_reference" content="citation_title=Differentiating protein-coding and noncoding RNA: challenges and ambiguities; citation_author=Dinger  M.E.; citation_author=Pang  K.C.; citation_author=Mercer  T.R.; citation_author=Mattick  J.S.;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000176" /><meta name="citation_reference" content="citation_title=Mutation-Rates differ among regions of the mammalian genome; citation_author=Wolfe  K.H.; citation_author=Sharp  P.M.; citation_author=Li  W.H.;  citation_journal_title=Nature;  citation_year=1989;  citation_volume=337;  citation_pages=283-5; " /><meta name="citation_reference" content="citation_title=A systematic analysis of disease-associated variants in the 3′ regulatory regions of human protein-coding genes II: the importance of mRNA secondary structure in assessing the functionality of 3′ UTR variants; citation_author=Chen  J.M.; citation_author=Ferec  C.; citation_author=Cooper  D.N.;  citation_journal_title=Hum. Genet.;  citation_year=2006;  citation_volume=120;  citation_pages=301-33; " /><meta name="citation_reference" content="citation_title=Transcriptome-wide analysis of UTRs in non-small cell lung cancer reveals cancer-related genes with SNV-induced changes on RNA secondary structure and miRNA target sites; citation_author=Sabarinathan  R.; citation_author=Wenzel  A.; citation_author=Novotny  P.; citation_author=Tang  X.J.; citation_author=Kalari  K.R.; citation_author=Gorodkin  J.;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e82699" /><meta name="citation_reference" content="citation_title=Synonymous mutations in the human dopamine receptor D2 (DRD2) affect mRNA stability and synthesis of the receptor; citation_author=Duan  J.B.; citation_author=Wainwright  M.S.; citation_author=Comeron  J.M.;  citation_journal_title=Hum. Mol. Genet.;  citation_year=2003;  citation_volume=12;  citation_pages=205-16; " /><meta name="citation_reference" content="citation_title=Deep sequencing to reveal new variants in pooled DNA samples; citation_author=Out  A.A.; citation_author=van Minderhout  I.J.H.M.; citation_author=Goeman  J.J.;  citation_journal_title=Hum. Mutat.;  citation_year=2009;  citation_volume=30;  citation_pages=1703-12; " /><meta name="citation_reference" content="citation_title=To pool, or not to pool?; citation_author=Cutler  D.J.; citation_author=Jensen  J.D.;  citation_journal_title=Genetics;  citation_year=2010;  citation_volume=186;  citation_pages=41-3; " /><meta name="citation_reference" content="citation_title=Genetic variation in an individual human exome; citation_author=Ng  P.C.; citation_author=Levy  S.; citation_author=Huang  J.;  citation_journal_title=PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000160" /><meta name="citation_reference" content="citation_title=Chicken genome: current status and future opportunities; citation_author=Burt  D.W.;  citation_journal_title=Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1692-8; " /><meta name="citation_reference" content="citation_title=Identification of the Yellow skin gene reveals a hybrid origin of the domestic chicken; citation_author=Eriksson  J.; citation_author=Larson  G.; citation_author=Gunnarsson  U.;  citation_journal_title=PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000010" /><meta name="citation_reference" content="citation_title=Signatures of selection in the genomes of commercial and non-commercial chicken breeds; citation_author=Elferink  M.G.; citation_author=Megens  H.J.; citation_author=Vereijken  A.; citation_author=Hu  X.X.; citation_author=Crooijmans  R.P.M.A.; citation_author=Groenen  M.A.M.;  citation_journal_title=PLoS ONE.;  citation_year=2012;  citation_volume=7;  citation_pages=e32720" /><meta name="citation_reference" content="citation_title=Mice lacking melanin-concentrating hormone are hypophagic and lean; citation_author=Shimada  M.; citation_author=Tritos  N.A.; citation_author=Lowell  B.B.; citation_author=Flier  J.S.; citation_author=Maratos-Flier  E.;  citation_journal_title=Nature;  citation_year=1998;  citation_volume=396;  citation_pages=670-4; " /><meta name="citation_reference" content="citation_title=Establishing the validity of domestication genes using DNA from ancient chickens; citation_author=Flink  L.G.; citation_author=Allen  R.; citation_author=Barnett  R.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=6184-9; " /><meta name="citation_reference" content="citation_title=Involvement of thyrotropin in photoperiodic signal transduction in mice; citation_author=Ono  H.; citation_author=Hoshino  Y.; citation_author=Yasuo  S.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=18238-42; " /><meta name="citation_reference" content="citation_title=Light-induced hormone conversion of T-4 to T-3 regulates photoperiodic response of gonads in birds; citation_author=Yoshimura  T.; citation_author=Yasuo  S.; citation_author=Watanabe  M.;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=426;  citation_pages=178-81; " /><meta name="citation_reference" content="citation_title=Functional intronic polymorphisms: buried treasure awaiting discovery within our genes; citation_author=Cooper  D.N.;  citation_journal_title=Hum. Genomics;  citation_year=2010;  citation_volume=4;  citation_pages=284-8; " /><meta name="citation_reference" content="citation_title=Identification of quantitative trait loci associated with bone traits and body weight in an F2 resource population of chickens; citation_author=Schreiweis  M.A.; citation_author=Hester  P.Y.; citation_author=Moody  D.E.;  citation_journal_title=Genet. Sel. Evol.;  citation_year=2005;  citation_volume=37;  citation_pages=677-98; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci for BMD and bone strength in an intercross between domestic and wildtype chickens; citation_author=Rubin  C.J.; citation_author=Brandstrom  H.; citation_author=Wright  D.;  citation_journal_title=J. Bone Miner. Res.;  citation_year=2007;  citation_volume=22;  citation_pages=375-84; " /><meta name="citation_reference" content="citation_title=Characterization of a QTL affecting skeletal size in mice; citation_author=Christians  J.K.; citation_author=Bingham  V.; citation_author=Oliver  F.; citation_author=Heath  T.T.; citation_author=Keightley  P.D.;  citation_journal_title=Mamm. Genome.;  citation_year=2003;  citation_volume=14;  citation_pages=175-83; " /><meta name="citation_reference" content="citation_title=Protection against malaria by abnormalities in red cell surface antigens and cytoskeletal proteins; citation_author=Roberts  D; citation_author=Schrier  S.L.; citation_author=Daily  J.; " /><meta name="citation_reference" content="citation_title=Diabetes mellitus - a thrifty genotype rendered detrimental by progress; citation_author=Neel  J.V.;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=1962;  citation_volume=14;  citation_pages=353-62; " /><meta name="citation_reference" content="citation_title=Evidence for hitchhiking of deleterious mutations within the human genome; citation_author=Chun  S.; citation_author=Fay  J.C.;  citation_journal_title=PLoS Genet.;  citation_year=2011;  citation_volume=7;  citation_pages=e1002240" /><meta name="citation_reference" content="GCNT3-GeneCard. http://www.genecards.org/cgi-bin/carddisp.pl?gene=GCNT3(27 August 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=Mucins in cancer: function, prognosis and therapy; citation_author=Kufe  D.W.;  citation_journal_title=Nat. Rev. Cancer;  citation_year=2009;  citation_volume=9;  citation_pages=874-85; " /><meta name="citation_reference" content="PLA2R1-GeneCard. http://www.genecards.org/cgi-bin/carddisp.pl?gene=PLA2R1(27 August 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=The Forkhead Transcription Factor Foxi1 is a master regulator of vacuolar H+-ATPase proton pump subunits in the inner ear, kidney and epididymis; citation_author=Vidarsson  H.; citation_author=Westergren  R.; citation_author=Heglind  M.; citation_author=Blomqvist  S.R.; citation_author=Breton  S.; citation_author=Enerback  S.;  citation_journal_title=PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e4471" /><meta name="citation_reference" content="citation_title=African signatures of recent positive selection in human FOXI1; citation_author=Moreno-Estrada  A.; citation_author=Aparicio-Prat  E.; citation_author=Sikora  M.;  citation_journal_title=BMC Evol. Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=267" /><meta name="citation_reference" content="ZNF507-GeneCard. http://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNF507(27 August 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=Myosin IXB variant increases the risk of celiac disease and points toward a primary intestinal barrier defect; citation_author=Monsuur  A.J.; citation_author=de Bakker  P.I.W.; citation_author=Alizadeh  B.Z.;  citation_journal_title=Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=1341-4; " /><meta name="citation_reference" content="citation_title=Genetic variation in myosin IXB is associated with ulcerative colitis; citation_author=Van Bodegraven  A.A.; citation_author=Curley  C.R.; citation_author=Hunt  K.A.;  citation_journal_title=Gastroenterology;  citation_year=2006;  citation_volume=131;  citation_pages=1768-74; " /><meta name="citation_reference" content="citation_title=Hen uterine gene expression profiling during eggshell formation reveals putative proteins involved in the supply of minerals or in the shell mineralization process; citation_author=Brionne  A.; citation_author=Nys  Y.; citation_author=Hennequet-Antier  C.; citation_author=Gautron  J.;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=220" /><meta name="citation_reference" content="citation_title=Quantitative trait loci affecting eggshell traits in an F(2) population; citation_author=Tuiskula-Haavisto  M.; citation_author=Honkatukia  M.; citation_author=Preisinger  R.;  citation_journal_title=Anim. Genet.;  citation_year=2011;  citation_volume=42;  citation_pages=293-9; " /><meta name="citation_reference" content="ADAMTS-GeneFamily. http://ghr.nlm.nih.gov/geneFamily/adamts(14 April 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=Overlap of quantitative trait loci for early growth rate, and for body weight and age at onset of sexual maturity in chickens; citation_author=Podisi  B.K.; citation_author=Knott  S.A.; citation_author=Dunn  I.C.; citation_author=Law  A.S.; citation_author=Burt  D.W.; citation_author=Hocking  P.M.;  citation_journal_title=Reproduction;  citation_year=2011;  citation_volume=141;  citation_pages=381-9; " /><meta name="citation_reference" content="citation_title=Genetic mapping of quantitative trait loci affecting body weight, egg character and egg production in F2 intercross chickens; citation_author=Sasaki  O.; citation_author=Odawara  S.; citation_author=Takahashi  H.;  citation_journal_title=Anim. Genet.;  citation_year=2004;  citation_volume=35;  citation_pages=188-94; " /><meta name="citation_reference" content="citation_title=Eggshell pigments-from formation to deposition; citation_author=Sparks  N.H.C.;  citation_journal_title=Avian Biol. Res.;  citation_year=2011;  citation_volume=4;  citation_pages=162-7; " /><meta name="citation_reference" content="citation_title=Quantitative expression of candidate genes affecting eggshell color; citation_author=Zheng  C.W.; citation_author=Li  Z.S.; citation_author=Yang  N.; citation_author=Ning  Z.H.;  citation_journal_title=Anim. Sci. J.;  citation_year=2014;  citation_volume=85;  citation_pages=506-10; " /><meta name="citation_reference" content="FRMPD2-GeneCard. http://www.genecards.org/cgi-bin/carddisp.pl?gene=FRMPD2(27 August 2014, date last accessed). " /><meta name="citation_reference" content="Wikipedia. http://en.wikipedia.org/wiki/Phosphatidylinositol(27 August 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=Rare and common variants: twenty arguments; citation_author=Gibson  G.;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2012;  citation_volume=13;  citation_pages=135-45; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/3/205/1593656/dsv005.pdf" /><meta name="description" content="Abstract.  Next-generation sequencing has prompted a surge of discovery of millions of genetic variants from vertebrate genomes. Besides applications in genetic" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/3/205/591312" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/3/205/591312" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/3/205/591312");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv005';
    </script>
<title>
	Functional classification of 15 million SNPs detected from diverse chicken populations | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./591312" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZzHATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2hleWFzLCBBbG1hcyBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBSb3NsaW4gSW5zdGl0dXRlIGFuZCBSb3lhbCAoRGljaykgU2Nob29sIG9mIFZldGVyaW5hcnkgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBFZGluYnVyZ2gsIEVhc3RlciBCdXNoIENhbXB1cywgTWlkbG90aGlhbiBFSDI1IDlSRywgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb3NjaGllcm8sIENsYXJpc3NhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIFJvc2xpbiBJbnN0aXR1dGUgYW5kIFJveWFsIChEaWNrKSBTY2hvb2wgb2YgVmV0ZXJpbmFyeSBTdHVkaWVzLCBVbml2ZXJzaXR5IG9mIEVkaW5idXJnaCwgRWFzdGVyIEJ1c2ggQ2FtcHVzLCBNaWRsb3RoaWFuIEVIMjUgOVJHLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkVvcnksIExlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBSb3NsaW4gSW5zdGl0dXRlIGFuZCBSb3lhbCAoRGljaykgU2Nob29sIG9mIFZldGVyaW5hcnkgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBFZGluYnVyZ2gsIEVhc3RlciBCdXNoIENhbXB1cywgTWlkbG90aGlhbiBFSDI1IDlSRywgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYWxwaCwgSGFubmFoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIFJvc2xpbiBJbnN0aXR1dGUgYW5kIFJveWFsIChEaWNrKSBTY2hvb2wgb2YgVmV0ZXJpbmFyeSBTdHVkaWVzLCBVbml2ZXJzaXR5IG9mIEVkaW5idXJnaCwgRWFzdGVyIEJ1c2ggQ2FtcHVzLCBNaWRsb3RoaWFuIEVIMjUgOVJHLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1bywgUmljaGFyZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBSb3NsaW4gSW5zdGl0dXRlIGFuZCBSb3lhbCAoRGljaykgU2Nob29sIG9mIFZldGVyaW5hcnkgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBFZGluYnVyZ2gsIEVhc3RlciBCdXNoIENhbXB1cywgTWlkbG90aGlhbiBFSDI1IDlSRywgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXb29sbGlhbXMsIEpvaG4gQS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgUm9zbGluIEluc3RpdHV0ZSBhbmQgUm95YWwgKERpY2spIFNjaG9vbCBvZiBWZXRlcmluYXJ5IFN0dWRpZXMsIFVuaXZlcnNpdHkgb2YgRWRpbmJ1cmdoLCBFYXN0ZXIgQnVzaCBDYW1wdXMsIE1pZGxvdGhpYW4gRUgyNSA5UkcsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQnVydCwgRGF2aWQgVy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgUm9zbGluIEluc3RpdHV0ZSBhbmQgUm95YWwgKERpY2spIFNjaG9vbCBvZiBWZXRlcmluYXJ5IFN0dWRpZXMsIFVuaXZlcnNpdHkgb2YgRWRpbmJ1cmdoLCBFYXN0ZXIgQnVzaCBDYW1wdXMsIE1pZGxvdGhpYW4gRUgyNSA5UkcsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJGdW5jdGlvbmFsIGNsYXNzaWZpY2F0aW9uIG9mIDE1IG1pbGxpb24gU05QcyBkZXRlY3RlZCBmcm9tIGRpdmVyc2UgY2hpY2tlbiBwb3B1bGF0aW9ucyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TSUZUOiBwcmVkaWN0aW5nIGFtaW5vIGFjaWQgY2hhbmdlcyB0aGF0IGFmZmVjdCBwcm90ZWluIGZ1bmN0aW9uOyBjaXRhdGlvbl9hdXRob3I9TmcgIFAuQy47IGNpdGF0aW9uX2F1dGhvcj1IZW5pa29mZiAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM4MTItNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyB0aGUgZWZmZWN0cyBvZiBjb2Rpbmcgbm9uLXN5bm9ueW1vdXMgdmFyaWFudHMgb24gcHJvdGVpbiBmdW5jdGlvbiB1c2luZyB0aGUgU0lGVCBhbGdvcml0aG07IGNpdGF0aW9uX2F1dGhvcj1LdW1hciAgUC47IGNpdGF0aW9uX2F1dGhvcj1IZW5pa29mZiAgUy47IGNpdGF0aW9uX2F1dGhvcj1OZyAgUC5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDczLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHRoZSBmdW5jdGlvbmFsIGVmZmVjdCBvZiBhbWlubyBhY2lkIHN1YnN0aXR1dGlvbnMgYW5kIGluZGVsczsgY2l0YXRpb25fYXV0aG9yPUNob2kgIFkuOyBjaXRhdGlvbl9hdXRob3I9U2ltcyAgRy5FLjsgY2l0YXRpb25fYXV0aG9yPU11cnBoeSAgUy47IGNpdGF0aW9uX2F1dGhvcj1NaWxsZXIgIEouUi47IGNpdGF0aW9uX2F1dGhvcj1DaGFuICBBLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9zIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ2Njg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmVlZGxlcyBpbiBzdGFja3Mgb2YgbmVlZGxlczogZmluZGluZyBkaXNlYXNlLWNhdXNhbCB2YXJpYW50cyBpbiBhIHdlYWx0aCBvZiBnZW5vbWljIGRhdGE7IGNpdGF0aW9uX2F1dGhvcj1Db29wZXIgIEcuTS47IGNpdGF0aW9uX2F1dGhvcj1TaGVuZHVyZSAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02MjgtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBIdW1hbiBHZW5lIE11dGF0aW9uIERhdGFiYXNlOiBidWlsZGluZyBhIGNvbXByZWhlbnNpdmUgbXV0YXRpb24gcmVwb3NpdG9yeSBmb3IgY2xpbmljYWwgYW5kIG1vbGVjdWxhciBnZW5ldGljcywgZGlhZ25vc3RpYyB0ZXN0aW5nIGFuZCBwZXJzb25hbGl6ZWQgZ2Vub21pYyBtZWRpY2luZTsgY2l0YXRpb25fYXV0aG9yPVN0ZW5zb24gIFAuRC47IGNpdGF0aW9uX2F1dGhvcj1Nb3J0ICBNLjsgY2l0YXRpb25fYXV0aG9yPUJhbGwgIEUuVi47IGNpdGF0aW9uX2F1dGhvcj1TaGF3ICBLLjsgY2l0YXRpb25fYXV0aG9yPVBoaWxsaXBzICBBLkQuOyBjaXRhdGlvbl9hdXRob3I9Q29vcGVyICBELk4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluaXRpYWwgc2VxdWVuY2luZyBhbmQgYW5hbHlzaXMgb2YgdGhlIGh1bWFuIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUludGVybmF0aW9uYWwgSHVtYW4gR2Vub21lIHNlcXVlbmNpbmcgQ29uc29ydGl1bSw7IGNpdGF0aW9uX2F1dGhvcj1MYW5kZXIgIEUuUy47IGNpdGF0aW9uX2F1dGhvcj1MaW50b24gIEwuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIGVuY3ljbG9wZWRpYSBvZiBETkEgZWxlbWVudHMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUVOQ09ERS1Qcm9qZWN0LUNvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODk7ICBjaXRhdGlvbl9wYWdlcz01Ny03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG90ZW50aWFsIGV0aW9sb2dpYyBhbmQgZnVuY3Rpb25hbCBpbXBsaWNhdGlvbnMgb2YgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gbG9jaSBmb3IgaHVtYW4gZGlzZWFzZXMgYW5kIHRyYWl0czsgY2l0YXRpb25fYXV0aG9yPUhpbmRvcmZmICBMLkEuOyBjaXRhdGlvbl9hdXRob3I9U2V0aHVwYXRoeSAgUC47IGNpdGF0aW9uX2F1dGhvcj1KdW5raW5zICBILkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTkzNjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJwcmV0aW5nIG5vbmNvZGluZyBnZW5ldGljIHZhcmlhdGlvbiBpbiBjb21wbGV4IHRyYWl0cyBhbmQgaHVtYW4gZGlzZWFzZTsgY2l0YXRpb25fYXV0aG9yPVdhcmQgIEwuRC47IGNpdGF0aW9uX2F1dGhvcj1LZWxsaXMgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTA5NS0xMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlbGV0ZXJpb3VzIG11dGF0aW9uIHByZWRpY3Rpb24gaW4gdGhlIHNlY29uZGFyeSBzdHJ1Y3R1cmUgb2YgUk5BczsgY2l0YXRpb25fYXV0aG9yPUJhcmFzaCAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTY1NzgtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJETUFTOiBhIHdlYiBzZXJ2ZXIgZm9yIFJOQSBkZWxldGVyaW91cyBtdXRhdGlvbiBhbmFseXNpczsgY2l0YXRpb25fYXV0aG9yPVNodSAgVy5KLjsgY2l0YXRpb25fYXV0aG9yPUJvICBYLkMuOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBSLkouOyBjaXRhdGlvbl9hdXRob3I9WmhhbyAgRC5TLjsgY2l0YXRpb25fYXV0aG9yPVpoZW5nICBaLlEuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgUy5RLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkFzbnA6IGVmZmljaWVudCBkZXRlY3Rpb24gb2YgbG9jYWwgUk5BIHNlY29uZGFyeSBzdHJ1Y3R1cmUgY2hhbmdlcyBpbmR1Y2VkIGJ5IFNOUHMgKHZvbCAzNCwgcGcgNTQ2LCAyMDEzKTsgY2l0YXRpb25fYXV0aG9yPVNhYmFyaW5hdGhhbiAgUi47IGNpdGF0aW9uX2F1dGhvcj1UYWZlciAgSC47IGNpdGF0aW9uX2F1dGhvcj1TZWVtYW5uICBTLkUuOyBjaXRhdGlvbl9hdXRob3I9SG9mYWNrZXIgIEkuTC47IGNpdGF0aW9uX2F1dGhvcj1TdGFkbGVyICBQLkYuOyBjaXRhdGlvbl9hdXRob3I9R29yb2RraW4gIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIE11dGF0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTU0Ni01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzZWFzZS1hc3NvY2lhdGVkIG11dGF0aW9ucyB0aGF0IGFsdGVyIHRoZSBSTkEgc3RydWN0dXJhbCBlbnNlbWJsZTsgY2l0YXRpb25fYXV0aG9yPUhhbHZvcnNlbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1NYXJ0aW4gIEouUy47IGNpdGF0aW9uX2F1dGhvcj1Ccm9hZGF3YXkgIFMuOyBjaXRhdGlvbl9hdXRob3I9TGFlZGVyYWNoICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMDc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJpbHkgY29uc2VydmVkIGVsZW1lbnRzIGluIHZlcnRlYnJhdGUsIGluc2VjdCwgd29ybSwgYW5kIHllYXN0IGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1TaWVwZWwgIEEuOyBjaXRhdGlvbl9hdXRob3I9QmVqZXJhbm8gIEcuOyBjaXRhdGlvbl9hdXRob3I9UGVkZXJzZW4gIEouUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTAzNC01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgYSBoaWdoIGZyYWN0aW9uIG9mIHRoZSBodW1hbiBnZW5vbWUgdG8gYmUgdW5kZXIgc2VsZWN0aXZlIGNvbnN0cmFpbnQgdXNpbmcgR0VSUCBwbHVzOyBjaXRhdGlvbl9hdXRob3I9RGF2eWRvdiAgRS5WLjsgY2l0YXRpb25fYXV0aG9yPUdvb2RlICBELkwuOyBjaXRhdGlvbl9hdXRob3I9U2lyb3RhICBNLjsgY2l0YXRpb25fYXV0aG9yPUNvb3BlciAgRy5NLjsgY2l0YXRpb25fYXV0aG9yPVNpZG93ICBBLjsgY2l0YXRpb25fYXV0aG9yPUJhdHpvZ2xvdSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDEwMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9zcGVjdHMgZm9yIGlkZW50aWZ5aW5nIGZ1bmN0aW9uYWwgdmFyaWF0aW9uIGFjcm9zcyB0aGUgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9TWFjZG9uYWxkICBTLkouOyBjaXRhdGlvbl9hdXRob3I9TG9uZyAgQS5ELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz02NjE0LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3NpdGl2ZSBuYXR1cmFsIHNlbGVjdGlvbiBpbiB0aGUgaHVtYW4gbGluZWFnZTsgY2l0YXRpb25fYXV0aG9yPVNhYmV0aSAgUC5DLjsgY2l0YXRpb25fYXV0aG9yPVNjaGFmZm5lciAgUy5GLjsgY2l0YXRpb25fYXV0aG9yPUZyeSAgQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzEyOyAgY2l0YXRpb25fcGFnZXM9MTYxNC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHNpZ25hdHVyZXMgb2YgbmF0dXJhbCBzZWxlY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1OaWVsc2VuICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIGhpZ2ggZGVuc2l0eSA2MDAgSyBTTlAgZ2Vub3R5cGluZyBhcnJheSBmb3IgY2hpY2tlbjsgY2l0YXRpb25fYXV0aG9yPUtyYW5pcyAgQS47IGNpdGF0aW9uX2F1dGhvcj1HaGV5YXMgIEEuQS47IGNpdGF0aW9uX2F1dGhvcj1Cb3NjaGllcm8gIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93cy1XaGVlbGVyIHRyYW5zZm9ybTsgY2l0YXRpb25fYXV0aG9yPUxpICBILjsgY2l0YXRpb25fYXV0aG9yPUR1cmJpbiAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIEFsaWdubWVudC9NYXAgZm9ybWF0IGFuZCBTQU10b29sczsgY2l0YXRpb25fYXV0aG9yPUxpICBILjsgY2l0YXRpb25fYXV0aG9yPUhhbmRzYWtlciAgQi47IGNpdGF0aW9uX2F1dGhvcj1XeXNva2VyICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFOTk9WQVI6IGZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBvZiBnZW5ldGljIHZhcmlhbnRzIGZyb20gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgZGF0YTsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEsuOyBjaXRhdGlvbl9hdXRob3I9TGkgIE0uWS47IGNpdGF0aW9uX2F1dGhvcj1IYWtvbmFyc29uICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9ZTE2NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlVDU0MgTXVsdGlwbGUgYWxpZ25tZW50cyBvZiA2IHZlcnRlYnJhdGUgZ2Vub21lcyB3aXRoIGNoaWNrZW4uIGh0dHA6Ly9oZ2Rvd25sb2FkLmNzZS51Y3NjLmVkdS9nb2xkZW5QYXRoL2dhbEdhbDMvbXVsdGl6N3dheS8oMTQgQXByaWwgMjAxNCwgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJOQXNucCB3ZWIgc2VydmVyOiBwcmVkaWN0aW5nIFNOUCBlZmZlY3RzIG9uIGxvY2FsIFJOQSBzZWNvbmRhcnkgc3RydWN0dXJlOyBjaXRhdGlvbl9hdXRob3I9U2FiYXJpbmF0aGFuICBSLjsgY2l0YXRpb25fYXV0aG9yPVRhZmVyICBILjsgY2l0YXRpb25fYXV0aG9yPVNlZW1hbm4gIFMuRS47IGNpdGF0aW9uX2F1dGhvcj1Ib2ZhY2tlciAgSS5MLjsgY2l0YXRpb25fYXV0aG9yPVN0YWRsZXIgIFAuRi47IGNpdGF0aW9uX2F1dGhvcj1Hb3JvZGtpbiAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPVc0NzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbGxpbmcgdGhlIGZhbHNlIGRpc2NvdmVyeSByYXRlIC0gYSBwcmFjdGljYWwgYW5kIHBvd2VyZnVsIGFwcHJvYWNoIHRvIG11bHRpcGxlIHRlc3Rpbmc7IGNpdGF0aW9uX2F1dGhvcj1CZW5qYW1pbmkgIFkuOyBjaXRhdGlvbl9hdXRob3I9SG9jaGJlcmcgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBSb3kuIFN0YXQuIFNvYy4gQiBNZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9Mjg5LTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHJlc2VxdWVuY2luZyByZXZlYWxzIGxvY2kgdW5kZXIgc2VsZWN0aW9uIGR1cmluZyBjaGlja2VuIGRvbWVzdGljYXRpb247IGNpdGF0aW9uX2F1dGhvcj1SdWJpbiAgQy5KLjsgY2l0YXRpb25fYXV0aG9yPVpvZHkgIE0uQy47IGNpdGF0aW9uX2F1dGhvcj1Fcmlrc3NvbiAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz01ODctOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVtcGlyaWNhbCB0aHJlc2hvbGQgdmFsdWVzIGZvciBxdWFudGl0YXRpdmUgdHJhaXQgbWFwcGluZzsgY2l0YXRpb25fYXV0aG9yPUNodXJjaGlsbCAgRy5BLjsgY2l0YXRpb25fYXV0aG9yPURvZXJnZSAgUi5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM4OyAgY2l0YXRpb25fcGFnZXM9OTYzLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggcmVzb2x1dGlvbiBnZW5vbWUtd2lkZSBzY2FuIGZvciBzaWduaWZpY2FudCBzZWxlY3RpdmUgc3dlZXBzOiBhbiBhcHBsaWNhdGlvbiB0byBwb29sZWQgc2VxdWVuY2UgZGF0YSBpbiBsYXlpbmcgY2hpY2tlbnM7IGNpdGF0aW9uX2F1dGhvcj1RYW5iYXJpICBTLjsgY2l0YXRpb25fYXV0aG9yPVN0cm9tICBULk0uOyBjaXRhdGlvbl9hdXRob3I9SGFiZXJlciAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDk1MjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBnZW5vbWUgU05QIGRpc2NvdmVyeSBhbmQgYW5hbHlzaXMgb2YgZ2VuZXRpYyBkaXZlcnNpdHkgaW4gVHVya2V5IChNZWxlYWdyaXMgZ2FsbG9wYXZvKTsgY2l0YXRpb25fYXV0aG9yPUFzbGFtICBNLkwuOyBjaXRhdGlvbl9hdXRob3I9QmFzdGlhYW5zZW4gIEouVy5NLjsgY2l0YXRpb25fYXV0aG9yPUVsZmVyaW5rICBNLkcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gYW5kIGxvc3NlcyBvZiBub24tY29kaW5nIFJOQXMgaW4gYXZpYW4gZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUdhcmRuZXIgIFAuUC47IGNpdGF0aW9uX2F1dGhvcj1GYXNvbGQgIE0uOyBjaXRhdGlvbl9hdXRob3I9QnVyZ2UgIFMuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTAxMjE3OTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYXRpbmcgcHJvdGVpbi1jb2RpbmcgYW5kIG5vbmNvZGluZyBSTkE6IGNoYWxsZW5nZXMgYW5kIGFtYmlndWl0aWVzOyBjaXRhdGlvbl9hdXRob3I9RGluZ2VyICBNLkUuOyBjaXRhdGlvbl9hdXRob3I9UGFuZyAgSy5DLjsgY2l0YXRpb25fYXV0aG9yPU1lcmNlciAgVC5SLjsgY2l0YXRpb25fYXV0aG9yPU1hdHRpY2sgIEouUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAxNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbi1SYXRlcyBkaWZmZXIgYW1vbmcgcmVnaW9ucyBvZiB0aGUgbWFtbWFsaWFuIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPVdvbGZlICBLLkguOyBjaXRhdGlvbl9hdXRob3I9U2hhcnAgIFAuTS47IGNpdGF0aW9uX2F1dGhvcj1MaSAgVy5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTMzNzsgIGNpdGF0aW9uX3BhZ2VzPTI4My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN5c3RlbWF0aWMgYW5hbHlzaXMgb2YgZGlzZWFzZS1hc3NvY2lhdGVkIHZhcmlhbnRzIGluIHRoZSAz4oCyIHJlZ3VsYXRvcnkgcmVnaW9ucyBvZiBodW1hbiBwcm90ZWluLWNvZGluZyBnZW5lcyBJSTogdGhlIGltcG9ydGFuY2Ugb2YgbVJOQSBzZWNvbmRhcnkgc3RydWN0dXJlIGluIGFzc2Vzc2luZyB0aGUgZnVuY3Rpb25hbGl0eSBvZiAz4oCyIFVUUiB2YXJpYW50czsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIEouTS47IGNpdGF0aW9uX2F1dGhvcj1GZXJlYyAgQy47IGNpdGF0aW9uX2F1dGhvcj1Db29wZXIgIEQuTi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTMwMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZS13aWRlIGFuYWx5c2lzIG9mIFVUUnMgaW4gbm9uLXNtYWxsIGNlbGwgbHVuZyBjYW5jZXIgcmV2ZWFscyBjYW5jZXItcmVsYXRlZCBnZW5lcyB3aXRoIFNOVi1pbmR1Y2VkIGNoYW5nZXMgb24gUk5BIHNlY29uZGFyeSBzdHJ1Y3R1cmUgYW5kIG1pUk5BIHRhcmdldCBzaXRlczsgY2l0YXRpb25fYXV0aG9yPVNhYmFyaW5hdGhhbiAgUi47IGNpdGF0aW9uX2F1dGhvcj1XZW56ZWwgIEEuOyBjaXRhdGlvbl9hdXRob3I9Tm92b3RueSAgUC47IGNpdGF0aW9uX2F1dGhvcj1UYW5nICBYLkouOyBjaXRhdGlvbl9hdXRob3I9S2FsYXJpICBLLlIuOyBjaXRhdGlvbl9hdXRob3I9R29yb2RraW4gIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTgyNjk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBtdXRhdGlvbnMgaW4gdGhlIGh1bWFuIGRvcGFtaW5lIHJlY2VwdG9yIEQyIChEUkQyKSBhZmZlY3QgbVJOQSBzdGFiaWxpdHkgYW5kIHN5bnRoZXNpcyBvZiB0aGUgcmVjZXB0b3I7IGNpdGF0aW9uX2F1dGhvcj1EdWFuICBKLkIuOyBjaXRhdGlvbl9hdXRob3I9V2FpbndyaWdodCAgTS5TLjsgY2l0YXRpb25fYXV0aG9yPUNvbWVyb24gIEouTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMDUtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZXAgc2VxdWVuY2luZyB0byByZXZlYWwgbmV3IHZhcmlhbnRzIGluIHBvb2xlZCBETkEgc2FtcGxlczsgY2l0YXRpb25fYXV0aG9yPU91dCAgQS5BLjsgY2l0YXRpb25fYXV0aG9yPXZhbiBNaW5kZXJob3V0ICBJLkouSC5NLjsgY2l0YXRpb25fYXV0aG9yPUdvZW1hbiAgSi5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SHVtLiBNdXRhdC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNzAzLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UbyBwb29sLCBvciBub3QgdG8gcG9vbD87IGNpdGF0aW9uX2F1dGhvcj1DdXRsZXIgIEQuSi47IGNpdGF0aW9uX2F1dGhvcj1KZW5zZW4gIEouRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTQxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWF0aW9uIGluIGFuIGluZGl2aWR1YWwgaHVtYW4gZXhvbWU7IGNpdGF0aW9uX2F1dGhvcj1OZyAgUC5DLjsgY2l0YXRpb25fYXV0aG9yPUxldnkgIFMuOyBjaXRhdGlvbl9hdXRob3I9SHVhbmcgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAxNjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGlja2VuIGdlbm9tZTogY3VycmVudCBzdGF0dXMgYW5kIGZ1dHVyZSBvcHBvcnR1bml0aWVzOyBjaXRhdGlvbl9hdXRob3I9QnVydCAgRC5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNjkyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHRoZSBZZWxsb3cgc2tpbiBnZW5lIHJldmVhbHMgYSBoeWJyaWQgb3JpZ2luIG9mIHRoZSBkb21lc3RpYyBjaGlja2VuOyBjaXRhdGlvbl9hdXRob3I9RXJpa3Nzb24gIEouOyBjaXRhdGlvbl9hdXRob3I9TGFyc29uICBHLjsgY2l0YXRpb25fYXV0aG9yPUd1bm5hcnNzb24gIFUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYXR1cmVzIG9mIHNlbGVjdGlvbiBpbiB0aGUgZ2Vub21lcyBvZiBjb21tZXJjaWFsIGFuZCBub24tY29tbWVyY2lhbCBjaGlja2VuIGJyZWVkczsgY2l0YXRpb25fYXV0aG9yPUVsZmVyaW5rICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9TWVnZW5zICBILkouOyBjaXRhdGlvbl9hdXRob3I9VmVyZWlqa2VuICBBLjsgY2l0YXRpb25fYXV0aG9yPUh1ICBYLlguOyBjaXRhdGlvbl9hdXRob3I9Q3Jvb2lqbWFucyAgUi5QLk0uQS47IGNpdGF0aW9uX2F1dGhvcj1Hcm9lbmVuICBNLkEuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTMyNzIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljZSBsYWNraW5nIG1lbGFuaW4tY29uY2VudHJhdGluZyBob3Jtb25lIGFyZSBoeXBvcGhhZ2ljIGFuZCBsZWFuOyBjaXRhdGlvbl9hdXRob3I9U2hpbWFkYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Ucml0b3MgIE4uQS47IGNpdGF0aW9uX2F1dGhvcj1Mb3dlbGwgIEIuQi47IGNpdGF0aW9uX2F1dGhvcj1GbGllciAgSi5TLjsgY2l0YXRpb25fYXV0aG9yPU1hcmF0b3MtRmxpZXIgIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mzk2OyAgY2l0YXRpb25fcGFnZXM9NjcwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGFibGlzaGluZyB0aGUgdmFsaWRpdHkgb2YgZG9tZXN0aWNhdGlvbiBnZW5lcyB1c2luZyBETkEgZnJvbSBhbmNpZW50IGNoaWNrZW5zOyBjaXRhdGlvbl9hdXRob3I9RmxpbmsgIEwuRy47IGNpdGF0aW9uX2F1dGhvcj1BbGxlbiAgUi47IGNpdGF0aW9uX2F1dGhvcj1CYXJuZXR0ICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz02MTg0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludm9sdmVtZW50IG9mIHRoeXJvdHJvcGluIGluIHBob3RvcGVyaW9kaWMgc2lnbmFsIHRyYW5zZHVjdGlvbiBpbiBtaWNlOyBjaXRhdGlvbl9hdXRob3I9T25vICBILjsgY2l0YXRpb25fYXV0aG9yPUhvc2hpbm8gIFkuOyBjaXRhdGlvbl9hdXRob3I9WWFzdW8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTE4MjM4LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaWdodC1pbmR1Y2VkIGhvcm1vbmUgY29udmVyc2lvbiBvZiBULTQgdG8gVC0zIHJlZ3VsYXRlcyBwaG90b3BlcmlvZGljIHJlc3BvbnNlIG9mIGdvbmFkcyBpbiBiaXJkczsgY2l0YXRpb25fYXV0aG9yPVlvc2hpbXVyYSAgVC47IGNpdGF0aW9uX2F1dGhvcj1ZYXN1byAgUy47IGNpdGF0aW9uX2F1dGhvcj1XYXRhbmFiZSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjY7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgaW50cm9uaWMgcG9seW1vcnBoaXNtczogYnVyaWVkIHRyZWFzdXJlIGF3YWl0aW5nIGRpc2NvdmVyeSB3aXRoaW4gb3VyIGdlbmVzOyBjaXRhdGlvbl9hdXRob3I9Q29vcGVyICBELk4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yODQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYXNzb2NpYXRlZCB3aXRoIGJvbmUgdHJhaXRzIGFuZCBib2R5IHdlaWdodCBpbiBhbiBGMiByZXNvdXJjZSBwb3B1bGF0aW9uIG9mIGNoaWNrZW5zOyBjaXRhdGlvbl9hdXRob3I9U2NocmVpd2VpcyAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPUhlc3RlciAgUC5ZLjsgY2l0YXRpb25fYXV0aG9yPU1vb2R5ICBELkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gU2VsLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTY3Ny05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIEJNRCBhbmQgYm9uZSBzdHJlbmd0aCBpbiBhbiBpbnRlcmNyb3NzIGJldHdlZW4gZG9tZXN0aWMgYW5kIHdpbGR0eXBlIGNoaWNrZW5zOyBjaXRhdGlvbl9hdXRob3I9UnViaW4gIEMuSi47IGNpdGF0aW9uX2F1dGhvcj1CcmFuZHN0cm9tICBILjsgY2l0YXRpb25fYXV0aG9yPVdyaWdodCAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJvbmUgTWluZXIuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zNzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBRVEwgYWZmZWN0aW5nIHNrZWxldGFsIHNpemUgaW4gbWljZTsgY2l0YXRpb25fYXV0aG9yPUNocmlzdGlhbnMgIEouSy47IGNpdGF0aW9uX2F1dGhvcj1CaW5naGFtICBWLjsgY2l0YXRpb25fYXV0aG9yPU9saXZlciAgRi47IGNpdGF0aW9uX2F1dGhvcj1IZWF0aCAgVC5ULjsgY2l0YXRpb25fYXV0aG9yPUtlaWdodGxleSAgUC5ELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWFtbS4gR2Vub21lLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVjdGlvbiBhZ2FpbnN0IG1hbGFyaWEgYnkgYWJub3JtYWxpdGllcyBpbiByZWQgY2VsbCBzdXJmYWNlIGFudGlnZW5zIGFuZCBjeXRvc2tlbGV0YWwgcHJvdGVpbnM7IGNpdGF0aW9uX2F1dGhvcj1Sb2JlcnRzICBEOyBjaXRhdGlvbl9hdXRob3I9U2NocmllciAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPURhaWx5ICBKLjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlhYmV0ZXMgbWVsbGl0dXMgLSBhIHRocmlmdHkgZ2Vub3R5cGUgcmVuZGVyZWQgZGV0cmltZW50YWwgYnkgcHJvZ3Jlc3M7IGNpdGF0aW9uX2F1dGhvcj1OZWVsICBKLlYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5NjI7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zNTMtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBoaXRjaGhpa2luZyBvZiBkZWxldGVyaW91cyBtdXRhdGlvbnMgd2l0aGluIHRoZSBodW1hbiBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1DaHVuICBTLjsgY2l0YXRpb25fYXV0aG9yPUZheSAgSi5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyMjQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iR0NOVDMtR2VuZUNhcmQuIGh0dHA6Ly93d3cuZ2VuZWNhcmRzLm9yZy9jZ2ktYmluL2NhcmRkaXNwLnBsP2dlbmU9R0NOVDMoMjcgQXVndXN0IDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11Y2lucyBpbiBjYW5jZXI6IGZ1bmN0aW9uLCBwcm9nbm9zaXMgYW5kIHRoZXJhcHk7IGNpdGF0aW9uX2F1dGhvcj1LdWZlICBELlcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gQ2FuY2VyOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz04NzQtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlBMQTJSMS1HZW5lQ2FyZC4gaHR0cDovL3d3dy5nZW5lY2FyZHMub3JnL2NnaS1iaW4vY2FyZGRpc3AucGw/Z2VuZT1QTEEyUjEoMjcgQXVndXN0IDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBGb3JraGVhZCBUcmFuc2NyaXB0aW9uIEZhY3RvciBGb3hpMSBpcyBhIG1hc3RlciByZWd1bGF0b3Igb2YgdmFjdW9sYXIgSCstQVRQYXNlIHByb3RvbiBwdW1wIHN1YnVuaXRzIGluIHRoZSBpbm5lciBlYXIsIGtpZG5leSBhbmQgZXBpZGlkeW1pczsgY2l0YXRpb25fYXV0aG9yPVZpZGFyc3NvbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1XZXN0ZXJncmVuICBSLjsgY2l0YXRpb25fYXV0aG9yPUhlZ2xpbmQgIE0uOyBjaXRhdGlvbl9hdXRob3I9QmxvbXF2aXN0ICBTLlIuOyBjaXRhdGlvbl9hdXRob3I9QnJldG9uICBTLjsgY2l0YXRpb25fYXV0aG9yPUVuZXJiYWNrICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU0NDcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWZyaWNhbiBzaWduYXR1cmVzIG9mIHJlY2VudCBwb3NpdGl2ZSBzZWxlY3Rpb24gaW4gaHVtYW4gRk9YSTE7IGNpdGF0aW9uX2F1dGhvcj1Nb3Jlbm8tRXN0cmFkYSAgQS47IGNpdGF0aW9uX2F1dGhvcj1BcGFyaWNpby1QcmF0ICBFLjsgY2l0YXRpb25fYXV0aG9yPVNpa29yYSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTI2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlpORjUwNy1HZW5lQ2FyZC4gaHR0cDovL3d3dy5nZW5lY2FyZHMub3JnL2NnaS1iaW4vY2FyZGRpc3AucGw/Z2VuZT1aTkY1MDcoMjcgQXVndXN0IDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU15b3NpbiBJWEIgdmFyaWFudCBpbmNyZWFzZXMgdGhlIHJpc2sgb2YgY2VsaWFjIGRpc2Vhc2UgYW5kIHBvaW50cyB0b3dhcmQgYSBwcmltYXJ5IGludGVzdGluYWwgYmFycmllciBkZWZlY3Q7IGNpdGF0aW9uX2F1dGhvcj1Nb25zdXVyICBBLkouOyBjaXRhdGlvbl9hdXRob3I9ZGUgQmFra2VyICBQLkkuVy47IGNpdGF0aW9uX2F1dGhvcj1BbGl6YWRlaCAgQi5aLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xMzQxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWF0aW9uIGluIG15b3NpbiBJWEIgaXMgYXNzb2NpYXRlZCB3aXRoIHVsY2VyYXRpdmUgY29saXRpczsgY2l0YXRpb25fYXV0aG9yPVZhbiBCb2RlZ3JhdmVuICBBLkEuOyBjaXRhdGlvbl9hdXRob3I9Q3VybGV5ICBDLlIuOyBjaXRhdGlvbl9hdXRob3I9SHVudCAgSy5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2FzdHJvZW50ZXJvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz0xNzY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZW4gdXRlcmluZSBnZW5lIGV4cHJlc3Npb24gcHJvZmlsaW5nIGR1cmluZyBlZ2dzaGVsbCBmb3JtYXRpb24gcmV2ZWFscyBwdXRhdGl2ZSBwcm90ZWlucyBpbnZvbHZlZCBpbiB0aGUgc3VwcGx5IG9mIG1pbmVyYWxzIG9yIGluIHRoZSBzaGVsbCBtaW5lcmFsaXphdGlvbiBwcm9jZXNzOyBjaXRhdGlvbl9hdXRob3I9QnJpb25uZSAgQS47IGNpdGF0aW9uX2F1dGhvcj1OeXMgIFkuOyBjaXRhdGlvbl9hdXRob3I9SGVubmVxdWV0LUFudGllciAgQy47IGNpdGF0aW9uX2F1dGhvcj1HYXV0cm9uICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYWZmZWN0aW5nIGVnZ3NoZWxsIHRyYWl0cyBpbiBhbiBGKDIpIHBvcHVsYXRpb247IGNpdGF0aW9uX2F1dGhvcj1UdWlza3VsYS1IYWF2aXN0byAgTS47IGNpdGF0aW9uX2F1dGhvcj1Ib25rYXR1a2lhICBNLjsgY2l0YXRpb25fYXV0aG9yPVByZWlzaW5nZXIgIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbmltLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0yOTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQURBTVRTLUdlbmVGYW1pbHkuIGh0dHA6Ly9naHIubmxtLm5paC5nb3YvZ2VuZUZhbWlseS9hZGFtdHMoMTQgQXByaWwgMjAxNCwgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3ZlcmxhcCBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBmb3IgZWFybHkgZ3Jvd3RoIHJhdGUsIGFuZCBmb3IgYm9keSB3ZWlnaHQgYW5kIGFnZSBhdCBvbnNldCBvZiBzZXh1YWwgbWF0dXJpdHkgaW4gY2hpY2tlbnM7IGNpdGF0aW9uX2F1dGhvcj1Qb2Rpc2kgIEIuSy47IGNpdGF0aW9uX2F1dGhvcj1Lbm90dCAgUy5BLjsgY2l0YXRpb25fYXV0aG9yPUR1bm4gIEkuQy47IGNpdGF0aW9uX2F1dGhvcj1MYXcgIEEuUy47IGNpdGF0aW9uX2F1dGhvcj1CdXJ0ICBELlcuOyBjaXRhdGlvbl9hdXRob3I9SG9ja2luZyAgUC5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmVwcm9kdWN0aW9uOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYWZmZWN0aW5nIGJvZHkgd2VpZ2h0LCBlZ2cgY2hhcmFjdGVyIGFuZCBlZ2cgcHJvZHVjdGlvbiBpbiBGMiBpbnRlcmNyb3NzIGNoaWNrZW5zOyBjaXRhdGlvbl9hdXRob3I9U2FzYWtpICBPLjsgY2l0YXRpb25fYXV0aG9yPU9kYXdhcmEgIFMuOyBjaXRhdGlvbl9hdXRob3I9VGFrYWhhc2hpICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5pbS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MTg4LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZ2dzaGVsbCBwaWdtZW50cy1mcm9tIGZvcm1hdGlvbiB0byBkZXBvc2l0aW9uOyBjaXRhdGlvbl9hdXRob3I9U3BhcmtzICBOLkguQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUF2aWFuIEJpb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE2Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgZXhwcmVzc2lvbiBvZiBjYW5kaWRhdGUgZ2VuZXMgYWZmZWN0aW5nIGVnZ3NoZWxsIGNvbG9yOyBjaXRhdGlvbl9hdXRob3I9WmhlbmcgIEMuVy47IGNpdGF0aW9uX2F1dGhvcj1MaSAgWi5TLjsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIE4uOyBjaXRhdGlvbl9hdXRob3I9TmluZyAgWi5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5pbS4gU2NpLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTUwNi0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iRlJNUEQyLUdlbmVDYXJkLiBodHRwOi8vd3d3LmdlbmVjYXJkcy5vcmcvY2dpLWJpbi9jYXJkZGlzcC5wbD9nZW5lPUZSTVBEMigyNyBBdWd1c3QgMjAxNCwgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iV2lraXBlZGlhLiBodHRwOi8vZW4ud2lraXBlZGlhLm9yZy93aWtpL1Bob3NwaGF0aWR5bGlub3NpdG9sKDI3IEF1Z3VzdCAyMDE0LCBkYXRlIGxhc3QgYWNjZXNzZWQpLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXJlIGFuZCBjb21tb24gdmFyaWFudHM6IHR3ZW50eSBhcmd1bWVudHM7IGNpdGF0aW9uX2F1dGhvcj1HaWJzb24gIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvMy8yMDUvMTU5MzY1Ni9kc3YwMDUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGhhcyBwcm9tcHRlZCBhIHN1cmdlIG9mIGRpc2NvdmVyeSBvZiBtaWxsaW9ucyBvZiBnZW5ldGljIHZhcmlhbnRzIGZyb20gdmVydGVicmF0ZSBnZW5vbWVzLiBCZXNpZGVzIGFwcGxpY2F0aW9ucyBpbiBnZW5ldGljIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01OTEzMTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkcjO1leSmDnMlkdg5qvUJjVs4trA=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f205%2f591312" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f205%2f591312" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23905" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&amp;Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/3">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="9632139">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632139">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632143">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632143">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632150">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632150">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632212">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632212">3. Results and discussions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632317">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632317">4. Concluding remarks</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632322">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632322">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632324">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632324">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632326">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632326">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9632328">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9632328">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/3/193/591209">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/3/219/591394">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Functional classification of 15 million SNPs detected from diverse chicken populations
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Almas A. Gheyas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +44 (0)131-651-9216. Fax. +44 (0)131-651-9105. E-mail: <a href="mailto:dave.burt@roslin.ed.ac.uk" target="_blank">dave.burt@roslin.ed.ac.uk</a> (D.W.B.); Tel. +44(0)131-651-9100. Fax. +44(0)131-651-9105. E-mail: <a href="mailto:almas.gheyas@roslin.ed.ac.uk" target="_blank">almas.gheyas@roslin.ed.ac.uk</a> (A.A.G.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Almas+A.+Gheyas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gheyas A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gheyas A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Almas A. Gheyas</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Clarissa Boschiero</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Clarissa+Boschiero">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boschiero C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boschiero C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Clarissa Boschiero</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lel Eory</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lel+Eory">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Eory L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Eory L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lel Eory</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hannah Ralph</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hannah+Ralph">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ralph H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ralph H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hannah Ralph</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Richard Kuo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Richard+Kuo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuo R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuo R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Richard Kuo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John A. Woolliams</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=John+A.+Woolliams">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Woolliams J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Woolliams J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">John A. Woolliams</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David W. Burt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">The Roslin Institute and Royal (Dick) School of Veterinary Studies</div>, <div class="insititution">University of Edinburgh</div>, <div class="addr-line">Easter Bush Campus, Midlothian EH25 9RG</div>, <div class="country">UK</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +44 (0)131-651-9216. Fax. +44 (0)131-651-9105. E-mail: <a href="mailto:dave.burt@roslin.ed.ac.uk" target="_blank">dave.burt@roslin.ed.ac.uk</a> (D.W.B.); Tel. +44(0)131-651-9100. Fax. +44(0)131-651-9105. E-mail: <a href="mailto:almas.gheyas@roslin.ed.ac.uk" target="_blank">almas.gheyas@roslin.ed.ac.uk</a> (A.A.G.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+W.+Burt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Burt D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Burt D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">David W. Burt</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Current Address: USP/ESALQ, Dept. de Zootecnia, Piracicaba, SP 13418-900, Brazil.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 205–217, <a href='https://doi.org/10.1093/dnares/dsv005'>https://doi.org/10.1093/dnares/dsv005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 April 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 January 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 March 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="591312" href="/dnaresearch/article-pdf/22/3/205/1593656/dsv005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Almas A. Gheyas, Clarissa Boschiero, Lel Eory, Hannah Ralph, Richard Kuo, John A. Woolliams, David W. Burt;  Functional classification of 15 million SNPs detected from diverse chicken populations, <em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 205–217, <a href='https://doi.org/10.1093/dnares/dsv005'>https://doi.org/10.1093/dnares/dsv005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/591312?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/591312?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/591312?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/591312?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/591312?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/591312?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/591312?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Functional%20classification%20of%2015%20million%20SNPs%20detected%20from%20diverse%20chicken%20populations&amp;publicationDate=2015-04-29&amp;volumeNum=22&amp;issueNum=3&amp;author=Gheyas%2C%20Almas%20A.%3B%20Boschiero%2C%20Clarissa&amp;startPage=205&amp;endPage=217&amp;contentId=10.1093%2Fdnares%2Fdsv005&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f205%2f591312" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=9632139 id="9632139" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Next-generation sequencing has prompted a surge of discovery of millions of genetic variants from vertebrate genomes. Besides applications in genetic association and linkage studies, a fraction of these variants will have functional consequences. This study describes detection and characterization of 15 million SNPs from chicken genome with the goal to predict variants with potential functional implications (pfVars) from both coding and non-coding regions. The study reports: 183K amino acid-altering SNPs of which 48% predicted as evolutionary intolerant, 13K splicing variants, 51K likely to alter RNA secondary structures, 500K within most conserved elements and 3K from non-coding RNAs. Regions of local fixation within commercial broiler and layer lines were investigated as potential selective sweeps using genome-wide SNP data. Relationships with phenotypes, if any, of the pfVars were explored by overlaying the sweep regions with known QTLs. Based on this, the candidate genes and/or causal mutations for a number of important traits are discussed. Although the fixed variants within sweep regions were enriched with non-coding SNPs, some non-synonymous-intolerant mutations reached fixation, suggesting their possible adaptive advantage. The results presented in this study are expected to have important implications for future genomic research to identify candidate causal mutations and in poultry breeding.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next-generation sequencing&quot;">next-generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chicken">chicken</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;selection signature&quot;">selection signature</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;functional variants&quot;">functional variants</a></div>                            <h2 scrollto-destination=9632143 id="9632143" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>With the advent of next-generation sequencing (NGS), we have seen a surge of discovery of millions of genetic variants, particularly single-nucleotide polymorphisms (SNPs), from different species. Characterization of these variants with the goal to delineate those with potential functional implications (pfVars) is often challenging, especially in the absence of comprehensive annotation of genomes for functional elements. Today for some model species, e.g. chicken, the genomes are well annotated for protein-coding regions. As a consequence, our ability to characterize coding variants is much more mature. Additionally, many methods are available to predict the effects of amino acid (AA)-altering variants on protein functions by using evolutionary and biochemical information.<sup><a reveal-id="DSV005C1  DSV005C4" data-open="DSV005C1  DSV005C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> These resources have allowed extensive study of the effects of protein-coding variants on disease and other traits.<sup><a reveal-id="DSV005C5" data-open="DSV005C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The protein-coding regions, however, occupy only a small portion of the genome (∼1.5%) while the rest is non-coding.<sup><a reveal-id="DSV005C6" data-open="DSV005C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In spite of this, biological research on non-coding mutations has been limited partly due to the difficulty in interpreting their effects and partly due to the poor annotation of most genomes for functional non-coding elements (fNCEs). The situation, however, is improving rapidly. For instance, a great deal of insight on fNCEs has been generated by the human ENCODE project.<sup><a reveal-id="DSV005C7" data-open="DSV005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The importance of non-coding variants is further highlighted from the evidence gathered from meta-analysis of genome-wide association studies, which found that 88% of the trait-associated mutations are non-coding.<sup><a reveal-id="DSV005C8" data-open="DSV005C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Complex traits, in particular, are overwhelmingly associated with non-coding mutations, suggesting that these mutations have primarily regulatory roles.<sup><a reveal-id="DSV005C9" data-open="DSV005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Some predictive models have been developed to assess the effects of SNPs on secondary structures of RNAs, as structural characteristics are often crucial for the functioning of various non-coding RNAs (ncRNA) and <em>cis</em>-regulatory elements in mRNAs.<sup><a reveal-id="DSV005C10  DSV005C13" data-open="DSV005C10  DSV005C13" class="link link-ref link-reveal xref-bibr">10–13</a></sup></p><p>Due to scarcity of information on functional elements, particularly the fNCEs, researchers have often adopted evolutionary analyses as a means to locating these elements in genomes, in which, regions conserved across species are assumed to be under purifying selection and functionally important.<sup><a reveal-id="DSV005C14" data-open="DSV005C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV005C15" data-open="DSV005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Mutations within such regions are therefore considered deleterious.<sup><a reveal-id="DSV005C4" data-open="DSV005C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Although these approaches are able to identify regions under purifying selection, they are not suitable for detecting regions evolving rapidly<sup><a reveal-id="DSV005C16" data-open="DSV005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> that are subject to recent positive selections. A number of approaches based on allele frequency spectra and linkage disequilibrium (LD) structure have been proposed for detecting signatures of positive selection by looking for regions exhibiting local fixation, unusually long LD and/or population differentiation that appear in a non-neutral manner compared with the rest of the genome.<sup><a reveal-id="DSV005C17" data-open="DSV005C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV005C18" data-open="DSV005C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>The aim of the present study was to characterize millions of SNPs detected from the chicken genome in a large NGS project with the goal to identify pfVars. A number of complementary approaches were used to delineate pfVars from both coding and non-coding regions: (i) annotation of the variants against known genes, (ii) predicting effects of the AA-altering variants on protein function, (iii) predicting effects of variants on the secondary structures of mRNAs and ncRNAs, (iv) finding variants that coincided with most conserved elements and (v) detecting regions of local fixation as putative signatures of recent positive selection and considering the fixed pfVars within these as sources of candidate causal mutations of various phenotypes. Since chicken is a major farm animal and an important model organism for genetic and genomic studies, detection and characterization of genetic variants, especially the pfVars, have major incentives for and implications in both research and breeding.</p>                            <h2 scrollto-destination=9632150 id="9632150" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=9632152 id="9632152" class="section-title" data-legacy-id=s2a>2.1. Resequencing of chicken samples</h3>
<p>Details about the sequenced samples, method of sequencing and alignment of sequence reads to reference genome can be found in a previous study by Kranis <em>et al.</em><sup><a reveal-id="DSV005C19" data-open="DSV005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> In brief, 243 chickens were sequenced and these originated from 24 lines: four commercial broilers (B1–B4), six commercial white egg layers (WEL 1–6), five commercial brown egg layers (BEL 1–5), eight experimental inbred layers (I1–8) and one unselected layer line (RI-J). For 23 lines, DNAs from 10–15 individuals were pooled for creating the libraries, whereas for a single line (WEL6) three individuals were sequenced separately. Sequencing was performed on Illumina GAIIx platform using a paired-end protocol and the sequencing reads were mapped to the reference genome using the Burrows–Wheeler Aligner (BWA)<sup><a reveal-id="DSV005C20" data-open="DSV005C20" class="link link-ref link-reveal xref-bibr">20</a></sup> v0.5.7 using default setting.</p>                            <h3 scrollto-destination=9632157 id="9632157" class="section-title" data-legacy-id=s2b>2.2. Variant calling</h3>
<p>Variant calling was performed using the ‘<em>mpilup’</em> function of Samtools<sup><a reveal-id="DSV005C21" data-open="DSV005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (v0.1.18) package. The minimum thresholds for base and map qualities were set to Phred-based scores of 20. Following the initial calling of SNPs, filtrations were performed using the criteria: (i) SNP quality (SNPQ) ≥40, (ii) coverage at SNP position ≥5 and ≤ mean line coverage ± 3SD (Standard deviation), (iii) evidence of alternative (ALT) allele supported by at least two reads: one on forward strand and one on reverse, and (iv) distance between adjacent markers is &gt;1 base. Regions with too high density of SNPs (&gt;10 SNPs/50 bases) were excluded. The coordinates of the SNPs reported in this paper are on the latest chicken reference genome build, <em>Gallus_gallus_4.</em> Further details on variant calling are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Information</a>. All the filtered SNPs have been submitted to the dbSNP with the Handle ID ‘DWBURT’.</p>                            <h3 scrollto-destination=9632161 id="9632161" class="section-title" data-legacy-id=s2c>2.3. Estimation of false discovery rates</h3>
<p>False discovery rates (FDRs) of variant calls for the present study were estimated by calling SNPs from Sanger sequencing of 25 random genomic regions and comparing those with SNPs detected from the NGS data from the same regions (further details in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Information</a>). For each region, only the part that had good-quality Sanger sequence in each individual was used for FDR estimation, and as a result, the lengths of the sequences spanned between 150 and 500 bp.</p><p>Based on the comparison of variants from Sanger and NGS data, a SNP was called true positive (TP) when it was detected by both the method, a false positive (FP) when it was detected only by NGS and a false negative (FN) when it was detected only by Sanger. Any sequenced bases that were not called as SNPs by either of the two methods were considered to be true negatives (TNs). Based on these, sensitivity or true positive rate (TPR) was defined as the proportion of actual SNPs that were detected by NGS and was calculated as: TP/(TP + FN). Specificity was defined as the proportion of actual non-SNPs not detected as SNPs by NGS and calculated as: TN/(TN + FP). FP and FN rates were calculated as (1−specificity) and (1−sensitivity), respectively. The proportion of FP SNPs in the NGS data at different filtration criteria was calculated as: FP/(total NGS SNPs retained by the criteria).</p>                            <h3 scrollto-destination=9632168 id="9632168" class="section-title" data-legacy-id=s2d>2.4. Functional annotation and characterization of SNPs</h3>
<p>The 15 million SNPs were annotated against the chicken gene database from ENSEMBL (release 71) and most conserved element (MCE) database from UCSC using the software package ANNOVAR (version July 06, 2012).<sup><a reveal-id="DSV005C22" data-open="DSV005C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The MCEs on chicken genome were predicted by PhastCons package by aligning the genomes of six distant species, namely, human, mouse, rat, oppssum, <em>X. tropicalis</em> and zebrafish to chicken genome.<sup><a reveal-id="DSV005C23" data-open="DSV005C23" class="link link-ref link-reveal xref-bibr">23</a></sup> To annotate against the MCEs, the co-ordinates of the SNPs were first converted to <em>Gallus_gallus_3</em> genome build as the MCEs are mapped onto this build.</p><p>The effects of non-synonymous SNPs on protein function were predicted based on evolutionary conservation using the SIFT (jcvi-sift 1.03)<sup><a reveal-id="DSV005C1" data-open="DSV005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV005C2" data-open="DSV005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and PROVEAN (v1.1.3)<sup><a reveal-id="DSV005C3" data-open="DSV005C3" class="link link-ref link-reveal xref-bibr">3</a></sup> packages. SIFT prediction depends on the degree of conservation at individual AA positions, assuming that functionally important positions have been conserved over evolutionary timescale. Using multiple alignment of homologous but distantly related peptide sequences, SIFT calculates normalized probabilities (SIFT score) of observing all possible AA residues at a position. If the SIFT score of an altered AA is below certain threshold, the variant is considered evolutionary intolerant (INTOL) while above the threshold the variants are considered tolerant (TOL). In contrast, PROVEAN computes an unbiased averaged delta alignment score from multiple alignments of homologous but distantly related peptides as a metric to predict the effects of coding mutations. Below certain threshold of delta score, the variant is declared INTOL. For the present study, SIFT was run using locally generated peptide alignments consisting of sequences from the UniRef90 non-redundant peptide database and predicted sequences from additional 47 bird genomes recently generated by BGI. Sequences were filtered to remove those peptides with over 90% sequence identity to a representative sample using the CD-HIT package and then aligned using Muscle (v.3.8.31). Default scores (&lt;0.05 for SIFT score and &gt;3.25 for conservation score) were used to classify the SNPs as either evolutionary intolerant (INTOL) or tolerant (TOL). PROVEAN was run with the default parameters on the NCBI non-redundant protein database. CD-HIT package was used to remove redundancy and Muscle package was used to generate the alignment. Default threshold (−2.28) of the delta score—defined as the change in alignment score due to introduction of a mutation—was used to differentiate INTOL variants from TOLs.</p><p>The effects of SNPs on the secondary structures of mRNAs and ncRNAs were predicted using RNAsnp<sup><a reveal-id="DSV005C12" data-open="DSV005C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV005C24" data-open="DSV005C24" class="link link-ref link-reveal xref-bibr">24</a></sup> package. Predictions were performed using two Modes (1 and 2) using default parameters as detailed in the package. The Mode 1 calculates the base-pair probabilities of the wild-type and mutant RNA sequences using a ‘global folding’ algorithm and computes the structural difference between wild and mutant types using Euclidian distance or Pearson's correlation co-efficient for all sequence intervals or local regions. As outputs, the interval with maximum base pairing distance (<em>d_max</em>) or minimum correlation co-efficient (<em>r_min</em>) is reported along with the corresponding <em>P</em>-values. The Mode 2, on the other hand, uses a ‘local folding’ algorithm to calculate base pairing probabilities and is designed to predict the effects of SNPs on large RNA sequences. In a two-step process, this mode first calculates the structural difference using Euclidian distance for all sequence intervals of fixed window length. In the second step, the interval with maximum base pair distance and the corresponding <em>P</em>-value is reported. The <em>P</em>-values were corrected for multiple testing using Benjamini–Hotchberg (B–H) method.<sup><a reveal-id="DSV005C25" data-open="DSV005C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>Ingenuity Pathway Analyses (IPAs) were performed for the genes harbouring potentially deleterious sets of SNPs to gain insight into the biological pathways that are affected. For these analyses, the genes with target group of SNPs were set as up-regulated molecules against the rest of the chicken genes. B–H multiple testing correction was performed and pathways were considered to be enriched with target molecules only when they remained significant (at <em>P</em> ≤ 0.05) after the correction.</p><p>Allele frequency of each SNP was estimated based on the proportion of good-quality reads supporting the alternative or non-reference allele. Mean frequencies of individual variants were calculated based on the frequency estimates from the populations where the SNP was detected.</p>                            <h3 scrollto-destination=9632187 id="9632187" class="section-title" data-legacy-id=s2e>2.5. Selection signature analysis</h3>
<div class="block-child-p">Analyses to detect signatures of selection were performed by calculating pooled heterozygosity (Hp) from autosomal SNPs following the method described by Rubin <em>et al.</em><sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Hp values were calculated for sliding windows of 40 kb size with step size of 20 kb using the following formula.<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">Hp</mtext><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mn>2</mn><mo>∑</mo><mtext>nMAJ</mtext><mo>∑</mo><mtext>nMIN</mtext></mrow><mrow><mrow><msup><mrow><mo stretchy="false">(</mo><mo>∑</mo><mtext>nMAJ</mtext><mo>+</mo><mo>∑</mo><mtext>nMIN</mtext><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></mrow></mfrac></mrow></mstyle></math></div></div></div></div><p>Here ∑nMAJ and ∑nMIN are the sum of major and minor allele counts within a window, respectively. Only windows with &gt;10 SNPs were analysed. BEDTools v2.17.0 was used for creating and intersecting the windows with SNP data, and Hp calculation was performed using custom scripts. The Hp calculation was performed within each commercial broiler (B1–B4), BEL (BEL1–BEL5) and WEL (WEL1–WEL5) lines where pooled samples were sequenced. Significance cut-offs for the Hp values were decided empirically based on chromosome-wise permutation analysis following the methods described by Churchill and Doerge<sup><a reveal-id="DSV005C27" data-open="DSV005C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and Qanbari <em>et al.</em><sup><a reveal-id="DSV005C28" data-open="DSV005C28" class="link link-ref link-reveal xref-bibr">28</a></sup> for individual lines. For the permutation tests, the allele count data were shuffled for 10,000 times within chromosomes, but the SNP positions were kept fixed.</p>                            <h2 scrollto-destination=9632212 id="9632212" class="section-title" data-legacy-id=s3>3. Results and discussions</h2>
                            <h3 scrollto-destination=9632224 id="9632224" class="section-title" data-legacy-id=s3a>3.1. Detection of SNPs and FDRs</h3>
<p>Initial calling of variants using the criteria of base quality ≥20 and map quality ≥20 detected ∼48 million putative SNPs, which were either segregating or fixed for a non-reference or alternative allele within a line. Upon filtration based on several criteria viz. minimum SNPQ, acceptable coverage, number of reads supporting the alternative allele, inter-marker gap and removal of regions with too high density of variants—over 15 million (<em>n</em> = 15,310,407) high-quality SNPs were retained. These formed the basis of subsequent analyses described in this paper. About 48% of the 15 million SNPs are common with known variants in dbSNP (build140) which currently contains ∼8.9 million SNPs for chicken.</p><p>The filtered variants were distributed in the genome with an average density of 15 SNPs (± 9.54) per kb or 1 SNP per every 68 bases, although the number of variants in each 1 kb window varied widely ranging from 0 to 112. The number of SNPs detected from each line varied between 3.6 and 5 million. The proportions of segregating variants within lines varied widely ranging from 8 to 77%. For inbred lines, this proportion was particularly low with an average of 21% while the corresponding averages for the commercial lines were ∼74% for broilers, 53% for WELs and 48% for BELs. The smaller proportion of segregating SNPs in the inbred lines was expected as these lines have been developed through many generations of close-mating.</p><p>Even though stringent filtration criteria are often applied to minimize the FPs from NGS data, uncertainties in variant calling may still persist due to alignment errors, inaccurate base calling and insufficient coverage. FDRs for the present study were, therefore, estimated by comparing SNP calls from Sanger Sequence of 25 random regions with that from NGS for the same regions. In total, 97 SNPs were detected from Sanger sequencing of the 11,801 bases while the number of NGS variants from these regions varied depending on the stringency of filtration criteria used (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Before filtration, there were 200 NGS SNPs including all the 97 SNPs detected by Sanger. The Fig. <a reveal-id="DSV005F1" data-open="DSV005F1" class="link link-reveal link-table xref-fig">1</a>a and b compare the FDRs of two filtration methods: (i) using only the SNPQ as filtration criterion and (ii) using SNPQ in combination with the criterion of support of alternative allele by at least two reads—one on the forward strand and one on the reverse strand. Figure <a reveal-id="DSV005F1" data-open="DSV005F1" class="link link-reveal link-table xref-fig">1</a>a shows the trade-off between the FPRs and TPRs for the NGS data using Sanger sequencing as the gold standard. It emphasizes several points. First, the TPRs were very high for NGS although not perfect when high-quality scores were imposed. Second, the FPRs were small in relation to the number of sequenced bases and reduced as higher quality scores were imposed. Third, filtration using Method2 was more effective than Method1 in reducing the FPR. </p>                        <a id="9632247" scrollto-destination="9632247"></a>
<div id="DSV005F1" data-id="DSV005F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00501.jpeg?Expires=2147483647&Signature=lTOwLfP6mttzRYxem8izTKppp9mO54FyVIWdtBUIFLS0EfqVxg9W8~F16jOMms71t9YCDw5HmgHdd3Fg2z26skEzc53p-drMsO3MJ~NP53IzZMgFnWYAXeO4ttnBTT4GseIuEXWnlVvxmMNg0p9aWwOD7L8QA90bazIy1nszL7upqypAaBQ3vtP~eJXEh4oswZbU7o4xGfWnSshDKNGK7I3VOi-QJP8DhIFzjN-ehKlNPaGL~m8zFPCwBGt7ceZ4Xv4u8sVs6V25fcFO6iCH7IEZUiC5Ozf3lL3AxFVk1dXegywXGjGRQMs4UpzZbISIhPAzkqFrdWnZNJY9jQWDrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of two methods of filtration on false discovery rates (FDRs) of SNP call. Method 1 represents filtration based on SNP quality scores (SNPQ) alone and Method 2 represents filtration based on SNPQ and evidence of non-reference or alternative allele by at least one forward- and one reverse-stranded reads. (a) ROC (receiver operating characteristic) curve plotting the true positive rate (TPR) against false positive rate (FPR). TPR is the sensitivity and is defined as the proportion of actual SNPs detected by Next Generation Sequencing (NGS). FPR is (1−specificity) and is defined as the fraction of non-SNP bases wrongly called as SNP by NGS. (b) Error rates or proportion of false positives (FPs) within NGS SNP set at different SNPQ. Both the figures indicate Method 2 as a better filtration approach in terms of FDR." path-from-xml="dsv00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632247/dsv00501.jpeg" path-from-xml="dsv00501.jpeg" target="_blank">View large</a><a section="9632247" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00501.jpeg?Expires=2147483647&Signature=3UhS1~kJRhgtWcIqkYKXbT36SAiT3B1Hsc7w9ZVdmh-XxPVAAHpzLl9519rA~CnL0n-rMN~4kQ5fEcbyR8wu6FcRCwRbuscdNFkdvIWfiOs7Y0YNne79jIHAX1sizrLC8WBHzrh3g2HNFOROPBxC-CPuMbu9~GJ5s72y~YREbc20o6mvL5IfX~jt3w6C0KILp78C4JbGx5zJy9PYtwlgIylEnpXNFX51hU2YL-cT1MiA3whgNKO0ogAqWe3rB1rpspyA~g6ySiUnsiKYjAsoQGPEf5iHBMbdNJpf16OTHV4lThlU5ZtVhxVZVvqdQFRvNdvgJW~fdK1YIaDAxg79KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632247&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of two methods of filtration on false discovery rates (FDRs) of SNP call. Method 1 represents filtration based on SNP quality scores (SNPQ) alone and Method 2 represents filtration based on SNPQ and evidence of non-reference or alternative allele by at least one forward- and one reverse-stranded reads. (a) ROC (receiver operating characteristic) curve plotting the true positive rate (TPR) against false positive rate (FPR). TPR is the sensitivity and is defined as the proportion of actual SNPs detected by Next Generation Sequencing (NGS). FPR is (1−specificity) and is defined as the fraction of non-SNP bases wrongly called as SNP by NGS. (b) Error rates or proportion of false positives (FPs) within NGS SNP set at different SNPQ. Both the figures indicate Method 2 as a better filtration approach in terms of FDR.</p></div></div><div content-id="DSV005F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00501.jpeg?Expires=2147483647&Signature=lTOwLfP6mttzRYxem8izTKppp9mO54FyVIWdtBUIFLS0EfqVxg9W8~F16jOMms71t9YCDw5HmgHdd3Fg2z26skEzc53p-drMsO3MJ~NP53IzZMgFnWYAXeO4ttnBTT4GseIuEXWnlVvxmMNg0p9aWwOD7L8QA90bazIy1nszL7upqypAaBQ3vtP~eJXEh4oswZbU7o4xGfWnSshDKNGK7I3VOi-QJP8DhIFzjN-ehKlNPaGL~m8zFPCwBGt7ceZ4Xv4u8sVs6V25fcFO6iCH7IEZUiC5Ozf3lL3AxFVk1dXegywXGjGRQMs4UpzZbISIhPAzkqFrdWnZNJY9jQWDrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of two methods of filtration on false discovery rates (FDRs) of SNP call. Method 1 represents filtration based on SNP quality scores (SNPQ) alone and Method 2 represents filtration based on SNPQ and evidence of non-reference or alternative allele by at least one forward- and one reverse-stranded reads. (a) ROC (receiver operating characteristic) curve plotting the true positive rate (TPR) against false positive rate (FPR). TPR is the sensitivity and is defined as the proportion of actual SNPs detected by Next Generation Sequencing (NGS). FPR is (1−specificity) and is defined as the fraction of non-SNP bases wrongly called as SNP by NGS. (b) Error rates or proportion of false positives (FPs) within NGS SNP set at different SNPQ. Both the figures indicate Method 2 as a better filtration approach in terms of FDR." path-from-xml="dsv00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632247/dsv00501.jpeg" path-from-xml="dsv00501.jpeg" target="_blank">View large</a><a section="9632247" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00501.jpeg?Expires=2147483647&Signature=3UhS1~kJRhgtWcIqkYKXbT36SAiT3B1Hsc7w9ZVdmh-XxPVAAHpzLl9519rA~CnL0n-rMN~4kQ5fEcbyR8wu6FcRCwRbuscdNFkdvIWfiOs7Y0YNne79jIHAX1sizrLC8WBHzrh3g2HNFOROPBxC-CPuMbu9~GJ5s72y~YREbc20o6mvL5IfX~jt3w6C0KILp78C4JbGx5zJy9PYtwlgIylEnpXNFX51hU2YL-cT1MiA3whgNKO0ogAqWe3rB1rpspyA~g6ySiUnsiKYjAsoQGPEf5iHBMbdNJpf16OTHV4lThlU5ZtVhxVZVvqdQFRvNdvgJW~fdK1YIaDAxg79KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632247&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of two methods of filtration on false discovery rates (FDRs) of SNP call. Method 1 represents filtration based on SNP quality scores (SNPQ) alone and Method 2 represents filtration based on SNPQ and evidence of non-reference or alternative allele by at least one forward- and one reverse-stranded reads. (a) ROC (receiver operating characteristic) curve plotting the true positive rate (TPR) against false positive rate (FPR). TPR is the sensitivity and is defined as the proportion of actual SNPs detected by Next Generation Sequencing (NGS). FPR is (1−specificity) and is defined as the fraction of non-SNP bases wrongly called as SNP by NGS. (b) Error rates or proportion of false positives (FPs) within NGS SNP set at different SNPQ. Both the figures indicate Method 2 as a better filtration approach in terms of FDR.</p></div></div><p>Figure <a reveal-id="DSV005F1" data-open="DSV005F1" class="link link-reveal link-table xref-fig">1</a>b compares the error rates for the NGS SNPs between the two filtration methods mentioned above. It shows that when the criterion of support of alternative allele by at least one forward- and one reverse-stranded reads alone was applied without any filtration on SNPQ (i.e. when SNPQ = 0), the error rate dropped drastically from 51 to 26%. At SNPQ ≥ 40, imposing this criterion reduced the error rate from 13 to 3% but at the same time increased the FNR (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). This error rate of 3% in detecting SNPs from NGS data is comparable to the rates reported in other studies such as 4.5% reported for chicken<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and 2.6% for turkey.<sup><a reveal-id="DSV005C29" data-open="DSV005C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>Prediction of FNR, however, is more difficult as it may be affected by a range of factors such as missing regions in the genome assembly where variant calling was not possible, extent of repeat regions in the genome, presence of regions that are difficult to sequence due to their biochemical properties and attempt to reduce FPs resulting from alignment artefact due to presence of indels or duplicated regions. As a consequence, even though the above analysis estimated FNR to be only 2% with Method 2 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), the actual rate is expected to be much higher. Since the goal of the present study was to characterize a set of high-quality SNPs by minimizing the FPs, the application of stringent criteria was justified even with some compromise on the FNR.</p>                            <h3 scrollto-destination=9632264 id="9632264" class="section-title" data-legacy-id=s3b>3.2. Coding and non-coding variants</h3>
<p>Only 2.2% of the 15 million SNPs was predicted to be within protein-coding regions and 1.2% as AA-altering (non-synonymous and stop-gain/loss) when annotated against the ENSEMBL gene database for chicken. The rest of the SNPs were classified as non-coding (Table <a reveal-id="DSV005TB1" data-open="DSV005TB1" class="link link-reveal link-table xref-fig">1</a>). Apart from the AA-altering variants, the other potentially functional categories included: variants with the potential to disrupt splicing events (0.09%); variants in 3′and 5′ UTRs with the potential to regulate protein translation (1.3%); those within 1 kb up- or downstream of transcription start or end sites with possible roles on transcriptional regulation (2.8%); and finally, the SNPs belonging to ncRNAs (0.02%). In addition to the ENSEMBL gene database, all the SNPs were checked against 1,608 ncRNA transcripts, which have recently been characterized.<sup><a reveal-id="DSV005C30" data-open="DSV005C30" class="link link-ref link-reveal xref-bibr">30</a></sup> A total of 2,062 (0.01%) variants fell within these transcripts: 131 in CD-snoRNAs, 155 in HACA-snoRNAs, 1,073 in microRNAs, 241 in regulatory regions, 82 in tRNAs, and the remaining 380 were in unclassified transcripts. Some of these ncRNA variants (<em>n</em> = 284) were previously annotated to be within protein-coding genes (exonic, UTR and intronic categories) when the ENSEMBL database was used. Major proportions of these SNPs with dual annotation were enriched for three ncRNA categories: regulatory region (34% of the 284), microRNA (27%) and tRNA (14%) types. Such overlaps could be due to potential difficulty in distinguishing ncRNA from mRNA or might truly reflect regions having dual functions as both non-coding and coding.<sup><a reveal-id="DSV005C31" data-open="DSV005C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                        <a id="9632266" scrollto-destination="9632266"></a>
<div content-id="DSV005TB1" class="table-modal"><div class="table-wrap"><div id="DSV005TB1" class="table-wrap-title" data-id="DSV005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>SNPs belonging to different annotation categories</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Annotation categories </th><th>Number (%)<sup>a</sup> </th><th>Mean AAF (± S.D.)<sup>b</sup> </th><th>No. detected from &gt;10 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. detected from all 24 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. of private SNPs with AAF &gt; 0.9<sup>c</sup> </th></tr></thead><tbody><tr><td>Intergenic </td><td>7,688,980 (50.22) </td><td>0.63 (0.24) </td><td>584,504 (7.60%) </td><td>223,114 (2.90%) </td><td>172,139 (2.24%) </td></tr><tr><td>Up/downstream </td><td>433,726 (2.83) </td><td>0.60 (0.25) </td><td>31,710 (7.30%) </td><td>10,348 (2.38%) </td><td>8,377 (1.93%) </td></tr><tr><td>Intronic </td><td>6,637,278 (43.35) </td><td>0.63 (0.24) </td><td>503,093 (7.57%) </td><td>190,587 (2.87%) </td><td>134,161 (2.02%) </td></tr><tr><td>Exonic </td><td>335,331 (2.19) </td><td>0.46 (0.26) </td><td>15,686 (4.48%) </td><td>5,846 (1.67%) </td><td>3,293 (0.94%) </td></tr><tr><td> Non-synonymous INTOL </td><td>88,655 (0.58) </td><td>0.28 (0.16) </td><td>416 (0.47%) </td><td>129 (0.15%) </td><td>288 (0.32%) </td></tr><tr><td> Non-synonymous TOL (or no prediction) </td><td>93,596 (0.61) </td><td>0.42 (0.25) </td><td>4,114 (4.38%) </td><td>159 (1.70%) </td><td>850 (0.90%) </td></tr><tr><td> Stop-gain/loss </td><td>1,071 (0.01) </td><td>0.38 (0.24) </td><td>21 (1.96%) </td><td>7 (0.65%) </td><td>11 (1.03%) </td></tr><tr><td> Synonymous </td><td>151,999 (0.99) </td><td>0.58 (0.25) </td><td>11,135 (7.31%) </td><td>4,116 (2.70%) </td><td>2,144 (1.41%) </td></tr><tr><td>Splicing </td><td>13,147 (0.09) </td><td>0.35 (0.22) </td><td>319 (2.42%) </td><td>115 (0.87%) </td><td>73 (0.55%) </td></tr><tr><td>UTR3′/UTR5′ </td><td>204,693 (1.34) </td><td>0.58 (0.25) </td><td>14,299 (6.98%) </td><td>4,974 (2.43%) </td><td>3,504 (1.71%) </td></tr><tr><td>ncRNA </td><td>3,122 (0.02) </td><td>0.54 (0.28) </td><td>136 (4.36%) </td><td>47 (1.51%) </td><td>88 (2.82%) </td></tr><tr><td>Within MCE </td><td>537,966 (3.51) </td><td>0.55 (0.26) </td><td>35,856 (6.66%) </td><td>13,997 (2.60%) </td><td>7,721 (1.43%) </td></tr></tbody></table></div><div content-id="DSV005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Annotation categories </th><th>Number (%)<sup>a</sup> </th><th>Mean AAF (± S.D.)<sup>b</sup> </th><th>No. detected from &gt;10 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. detected from all 24 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. of private SNPs with AAF &gt; 0.9<sup>c</sup> </th></tr></thead><tbody><tr><td>Intergenic </td><td>7,688,980 (50.22) </td><td>0.63 (0.24) </td><td>584,504 (7.60%) </td><td>223,114 (2.90%) </td><td>172,139 (2.24%) </td></tr><tr><td>Up/downstream </td><td>433,726 (2.83) </td><td>0.60 (0.25) </td><td>31,710 (7.30%) </td><td>10,348 (2.38%) </td><td>8,377 (1.93%) </td></tr><tr><td>Intronic </td><td>6,637,278 (43.35) </td><td>0.63 (0.24) </td><td>503,093 (7.57%) </td><td>190,587 (2.87%) </td><td>134,161 (2.02%) </td></tr><tr><td>Exonic </td><td>335,331 (2.19) </td><td>0.46 (0.26) </td><td>15,686 (4.48%) </td><td>5,846 (1.67%) </td><td>3,293 (0.94%) </td></tr><tr><td> Non-synonymous INTOL </td><td>88,655 (0.58) </td><td>0.28 (0.16) </td><td>416 (0.47%) </td><td>129 (0.15%) </td><td>288 (0.32%) </td></tr><tr><td> Non-synonymous TOL (or no prediction) </td><td>93,596 (0.61) </td><td>0.42 (0.25) </td><td>4,114 (4.38%) </td><td>159 (1.70%) </td><td>850 (0.90%) </td></tr><tr><td> Stop-gain/loss </td><td>1,071 (0.01) </td><td>0.38 (0.24) </td><td>21 (1.96%) </td><td>7 (0.65%) </td><td>11 (1.03%) </td></tr><tr><td> Synonymous </td><td>151,999 (0.99) </td><td>0.58 (0.25) </td><td>11,135 (7.31%) </td><td>4,116 (2.70%) </td><td>2,144 (1.41%) </td></tr><tr><td>Splicing </td><td>13,147 (0.09) </td><td>0.35 (0.22) </td><td>319 (2.42%) </td><td>115 (0.87%) </td><td>73 (0.55%) </td></tr><tr><td>UTR3′/UTR5′ </td><td>204,693 (1.34) </td><td>0.58 (0.25) </td><td>14,299 (6.98%) </td><td>4,974 (2.43%) </td><td>3,504 (1.71%) </td></tr><tr><td>ncRNA </td><td>3,122 (0.02) </td><td>0.54 (0.28) </td><td>136 (4.36%) </td><td>47 (1.51%) </td><td>88 (2.82%) </td></tr><tr><td>Within MCE </td><td>537,966 (3.51) </td><td>0.55 (0.26) </td><td>35,856 (6.66%) </td><td>13,997 (2.60%) </td><td>7,721 (1.43%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>The percentages are in relation to 15 million SNPs.</p></div></div><div content-id="TN2"><div class="fn"><p><sup>b</sup>AAF refers to Alternative allele frequency.</p></div></div><div content-id="TN3"><div class="fn"><p><sup>c</sup>The percentages are in relation to the number of SNPs within the annotation category.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9632266" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV005TB1" class="table-wrap-title" data-id="DSV005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>SNPs belonging to different annotation categories</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Annotation categories </th><th>Number (%)<sup>a</sup> </th><th>Mean AAF (± S.D.)<sup>b</sup> </th><th>No. detected from &gt;10 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. detected from all 24 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. of private SNPs with AAF &gt; 0.9<sup>c</sup> </th></tr></thead><tbody><tr><td>Intergenic </td><td>7,688,980 (50.22) </td><td>0.63 (0.24) </td><td>584,504 (7.60%) </td><td>223,114 (2.90%) </td><td>172,139 (2.24%) </td></tr><tr><td>Up/downstream </td><td>433,726 (2.83) </td><td>0.60 (0.25) </td><td>31,710 (7.30%) </td><td>10,348 (2.38%) </td><td>8,377 (1.93%) </td></tr><tr><td>Intronic </td><td>6,637,278 (43.35) </td><td>0.63 (0.24) </td><td>503,093 (7.57%) </td><td>190,587 (2.87%) </td><td>134,161 (2.02%) </td></tr><tr><td>Exonic </td><td>335,331 (2.19) </td><td>0.46 (0.26) </td><td>15,686 (4.48%) </td><td>5,846 (1.67%) </td><td>3,293 (0.94%) </td></tr><tr><td> Non-synonymous INTOL </td><td>88,655 (0.58) </td><td>0.28 (0.16) </td><td>416 (0.47%) </td><td>129 (0.15%) </td><td>288 (0.32%) </td></tr><tr><td> Non-synonymous TOL (or no prediction) </td><td>93,596 (0.61) </td><td>0.42 (0.25) </td><td>4,114 (4.38%) </td><td>159 (1.70%) </td><td>850 (0.90%) </td></tr><tr><td> Stop-gain/loss </td><td>1,071 (0.01) </td><td>0.38 (0.24) </td><td>21 (1.96%) </td><td>7 (0.65%) </td><td>11 (1.03%) </td></tr><tr><td> Synonymous </td><td>151,999 (0.99) </td><td>0.58 (0.25) </td><td>11,135 (7.31%) </td><td>4,116 (2.70%) </td><td>2,144 (1.41%) </td></tr><tr><td>Splicing </td><td>13,147 (0.09) </td><td>0.35 (0.22) </td><td>319 (2.42%) </td><td>115 (0.87%) </td><td>73 (0.55%) </td></tr><tr><td>UTR3′/UTR5′ </td><td>204,693 (1.34) </td><td>0.58 (0.25) </td><td>14,299 (6.98%) </td><td>4,974 (2.43%) </td><td>3,504 (1.71%) </td></tr><tr><td>ncRNA </td><td>3,122 (0.02) </td><td>0.54 (0.28) </td><td>136 (4.36%) </td><td>47 (1.51%) </td><td>88 (2.82%) </td></tr><tr><td>Within MCE </td><td>537,966 (3.51) </td><td>0.55 (0.26) </td><td>35,856 (6.66%) </td><td>13,997 (2.60%) </td><td>7,721 (1.43%) </td></tr></tbody></table></div><div content-id="DSV005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Annotation categories </th><th>Number (%)<sup>a</sup> </th><th>Mean AAF (± S.D.)<sup>b</sup> </th><th>No. detected from &gt;10 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. detected from all 24 lines with mean AAF &gt; 0.9<sup>c</sup> </th><th>No. of private SNPs with AAF &gt; 0.9<sup>c</sup> </th></tr></thead><tbody><tr><td>Intergenic </td><td>7,688,980 (50.22) </td><td>0.63 (0.24) </td><td>584,504 (7.60%) </td><td>223,114 (2.90%) </td><td>172,139 (2.24%) </td></tr><tr><td>Up/downstream </td><td>433,726 (2.83) </td><td>0.60 (0.25) </td><td>31,710 (7.30%) </td><td>10,348 (2.38%) </td><td>8,377 (1.93%) </td></tr><tr><td>Intronic </td><td>6,637,278 (43.35) </td><td>0.63 (0.24) </td><td>503,093 (7.57%) </td><td>190,587 (2.87%) </td><td>134,161 (2.02%) </td></tr><tr><td>Exonic </td><td>335,331 (2.19) </td><td>0.46 (0.26) </td><td>15,686 (4.48%) </td><td>5,846 (1.67%) </td><td>3,293 (0.94%) </td></tr><tr><td> Non-synonymous INTOL </td><td>88,655 (0.58) </td><td>0.28 (0.16) </td><td>416 (0.47%) </td><td>129 (0.15%) </td><td>288 (0.32%) </td></tr><tr><td> Non-synonymous TOL (or no prediction) </td><td>93,596 (0.61) </td><td>0.42 (0.25) </td><td>4,114 (4.38%) </td><td>159 (1.70%) </td><td>850 (0.90%) </td></tr><tr><td> Stop-gain/loss </td><td>1,071 (0.01) </td><td>0.38 (0.24) </td><td>21 (1.96%) </td><td>7 (0.65%) </td><td>11 (1.03%) </td></tr><tr><td> Synonymous </td><td>151,999 (0.99) </td><td>0.58 (0.25) </td><td>11,135 (7.31%) </td><td>4,116 (2.70%) </td><td>2,144 (1.41%) </td></tr><tr><td>Splicing </td><td>13,147 (0.09) </td><td>0.35 (0.22) </td><td>319 (2.42%) </td><td>115 (0.87%) </td><td>73 (0.55%) </td></tr><tr><td>UTR3′/UTR5′ </td><td>204,693 (1.34) </td><td>0.58 (0.25) </td><td>14,299 (6.98%) </td><td>4,974 (2.43%) </td><td>3,504 (1.71%) </td></tr><tr><td>ncRNA </td><td>3,122 (0.02) </td><td>0.54 (0.28) </td><td>136 (4.36%) </td><td>47 (1.51%) </td><td>88 (2.82%) </td></tr><tr><td>Within MCE </td><td>537,966 (3.51) </td><td>0.55 (0.26) </td><td>35,856 (6.66%) </td><td>13,997 (2.60%) </td><td>7,721 (1.43%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>The percentages are in relation to 15 million SNPs.</p></div></div><div content-id="TN2"><div class="fn"><p><sup>b</sup>AAF refers to Alternative allele frequency.</p></div></div><div content-id="TN3"><div class="fn"><p><sup>c</sup>The percentages are in relation to the number of SNPs within the annotation category.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9632266" target="_blank">View Large</a></div></div><p>The AA-altering SNPs (<em>n</em> = 183,320) were detected from 13,286 genes representing 74% of the known genes in chicken. Their density in terms of length of coding sequence varied widely across the genes ranging from 1 SNP per 19 bases to 1 per 731 kb (mean 1 SNP per 5.5 kb). A high degree of correlation (<em>r</em> = 0.75, <em>P</em>&lt;0.001), however, was observed between the densities of AA-altering and synonymous mutations. This correlation combined with the large variation in the densities of coding SNPs indicate that some genes are more prone to accumulate variants either due to differential mutation rates or uneven selection pressure.<sup><a reveal-id="DSV005C32" data-open="DSV005C32" class="link link-ref link-reveal xref-bibr">32</a></sup> IPA demonstrated that the genes with high density of AA-altering variants (&gt;10 SNPs/kb coding sequence) were significantly enriched in nine biological pathways (<em>P</em> ≤ 0.05 after correction for multiple testing). These pathways were associated with a DNA replication, recombination and repair, cell-to-cell signalling and interaction, metabolism of lipids, carbohydrates and amino acids, and cancer (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). It is difficult to explain why certain genes within important pathways harbour high density of AA-altering variants. One explanation is that not all of these variants were predicted to have radical effect on the function of the genes, and when harmful, they were generally present at low frequencies; these aspects have been further explored in the next two sections. Closer inspection of these genes further reveals that majority of these (∼70%) have one or more paralogs in the genome, indicating that the loss of function of the genes due to deleterious variants may partly be rescued by their paralogous genes. There were, however, 20 genes that showed extreme SNP density (≥50 SNPs/kb coding sequence). While these genes may represent regions with rapid mutation rates, it is possible that the high variant density in these genes is actually a result of error in the reference genome assembly and/or error in mapping of sequence reads due to high level of duplications. Eleven of these 20 genes had no paralog in the genome and may represent assembly errors. The other 9 genes had one or more paralogs, as many as 58, and the high SNP density may be a result of mapping errors.</p>                            <h3 scrollto-destination=9632268 id="9632268" class="section-title" data-legacy-id=s3c>3.3. Negative impact of non-synonymous substitutions on protein function</h3>
<p>Even though non-synonymous SNPs change amino acid sequence within a protein, the effects are not always harmful or radical on protein function. Using SIFT, 23% of the non-synonymous variants (<em>n</em> = 41,980) were predicted as ‘intolerant’ (INTOL) having radical effect and 51% were predicted ‘tolerant’ (TOL) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1a</a>). The remaining 26% did not have any predictions either due to lack of sufficient aligned sequences or poor confidence in predictions. Contrary to this, PROVEAN provided predictions for &gt;98% of the SNPs and much higher proportion of the variants (45%; <em>n</em> = 82,564) were predicted as INTOL (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1b</a>). This can be attributed to two factors. First, PROVEAN takes into account the sequence context of a variant rather than only individual amino acid position.<sup><a reveal-id="DSV005C3" data-open="DSV005C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Second, in the absence of homologous peptides, PROVEAN can still perform analysis by comparing the alignment of the query sequence to itself before and after the introduction of the mutation, which is not possible with SIFT. Together 49% of the non-synonymous SNPs (<em>n</em> = 88,655) were predicted INTOL by at least one of the algorithms while only 20% (<em>n</em> = 35,889) were predicted as INTOL by both.</p><p>IPA analysis of genes harbouring INTOL non-synonymous SNPs from broiler, WEL and BEL groups suggested many pathways (<em>n</em> = 162–211 after multiple testing correction), especially signalling pathways (87–92%), to be significantly enriched for these variants (See the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> for the top five pathways from each group). Most of the INTOL variants were found to have low frequency and were detected from few lines only (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>), thereby limiting their harmful effect (see next section for further information on allele frequency). This can explain how important pathways could harbour INTOL variants. IPA analysis on inbred lines found only two pathways as significant of which the ‘Role of <em>BRCA1</em> in DNA Damage Response’ was found highly significant (<em>P</em> &lt; 0.001; with 20 of the 71 genes passing the cut-off criteria). This pathway consists of genes encoding for tumour suppressors and DNA damage repair proteins. Since, the inbred lines have been bred for experimental purposes for susceptibility/resistance to different viral pathogens, such as Marek's Disease Virus (MDV), Avian Leucosis Virus (ALV) and Lymphoid Leucosis Virus (LLV), it is likely that these genes within these pathways are candidates for explaining the susceptibility/resistance towards these pathogens.</p>                            <h3 scrollto-destination=9632271 id="9632271" class="section-title" data-legacy-id=s3d>3.4. Effects of SNPs on RNA secondary structure</h3>
<p>The functions of many ncRNAs and <em>cis</em>-regulatory elements in mRNAs often rely on their distinctive secondary structures.<sup><a reveal-id="DSV005C12" data-open="DSV005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Previous studies have shown that mutations within untranslated regions (UTR) of mRNA transcripts can alter the structures of <em>cis</em>-acting regulatory elements that reside within these regions and thereby affect the translational and transcriptional efficiency and stability of mRNAs.<sup><a reveal-id="DSV005C33" data-open="DSV005C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSV005C34" data-open="DSV005C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Even apparently silent synonymous mutations have been shown to affect mRNA folding and stability, altering protein translation.<sup><a reveal-id="DSV005C35" data-open="DSV005C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Therefore, in the present study, the putative effects of all UTR, synonymous, non-synonymous and ncRNA variants on the secondary structures of RNAs were investigated using the RNAsnp package.</p><p>About 9% (<em>n</em> = 19,296) of the UTR, 11% (<em>n</em> = 19,603) of the non-synonymous, 8% (<em>n</em> = 12,001) of synonymous and 7% (<em>n</em> = 218) of the ncRNA variants were predicted to affect RNA secondary structure by at least one of the parameters and models used (<em>P</em> ≤ 0.05 before correcting for multiple testing) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Although after correcting for multiple testing, only 15 variants (4 UTR3′, 7 ncRNA and 4 non-synonymous) remained significant (<em>P</em> ≤ 0.05), the other variants, however, can still be considered suggestive and be used for shortlisting candidate functional variants. These variants including the suggestive ones were located within 12,296 protein-coding genes and 176 ncRNA transcripts.</p>                            <h3 scrollto-destination=9632274 id="9632274" class="section-title" data-legacy-id=s3e>3.5. SNPs within evolutionary conserved elements</h3>
<p>Genomic regions conserved across distantly related species are assumed to be under purifying selection, and hence variants within these regions are likely to be harmful. The 15 million SNPs were annotated against the 950,084 known ‘most conserved elements’ (MCEs; UCSC<sup><a reveal-id="DSV005C23" data-open="DSV005C23" class="link link-ref link-reveal xref-bibr">23</a></sup>), covering 6.5% of the genome and with their sizes varying from 1 base to 4.3 kb (median 43 bases).</p><p>About 3.5% (<em>n</em> = 537,966) of the 15 million variants fell within only 2.5% of the MCEs, while the rest of the conserved elements were devoid of any SNPs. The density of variants within MCEs was 8 SNPs/kb, which was much lower compared with the genome-wide density of 15 SNPs/kb, supporting their conserved nature. Major proportions of the MCE variants belonged to intronic (38%), exonic (33%) and intergenic (23%) categories while the remaining 5% fell within other categories such as UTR (2.2%), up/downstream (1.8%) and splicing (0.9%) and ncRNA (0.1%). Among the MCE-exonic variants, 51% (<em>n</em> = 91,789) were AA altering and majority of these (<em>n</em> = 60,016) were predicted as INTOL as was expected due to the evolutionary conserved nature of these regions.</p>                            <h3 scrollto-destination=9632277 id="9632277" class="section-title" data-legacy-id=s3f>3.6. Allele frequency distributions of SNPs in different annotation categories</h3>
<p>Frequency distributions of non-reference or alternative alleles (AAF) of pfVar categories were compared with those of potential neutral variants to investigate differences in their patterns (Figs <a reveal-id="DSV005F2" data-open="DSV005F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSV005F3" data-open="DSV005F3" class="link link-reveal link-table xref-fig">3</a>, Table <a reveal-id="DSV005TB1" data-open="DSV005TB1" class="link link-reveal link-table xref-fig">1</a>). This revealed a number of aspects. First, except in the Inbred group, in commercial chickens the non-synonymous, stop-gain/loss and splicing variants were heavily skewed to the left, i.e. fell within lower frequency bins compared with other SNP categories. About 50–61% of the AA-altering and splicing variants had AAF ≤ 0.3, whereas synonymous, intronic and intergenic variants, which are largely assumed to be neutral, showed more or less even distributions across the frequency bins (Figs <a reveal-id="DSV005F2" data-open="DSV005F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSV005F3" data-open="DSV005F3" class="link link-reveal link-table xref-fig">3</a>). The overall mean frequency for AA-altering and splicing variants was 0.35 (SD: 0.22) compared with 0.58 (SD: 0.25) for synonymous or 0.63 (SD: 0.24) for intergenic and intronic variants (<em>t</em>-test, <em>P</em> &lt; 0.001). Apart from the AA-altering and splicing variants, ncRNA SNPs also showed a left skew in all commercial groups with ∼33–37% belonging to frequency bins ≤0.3. </p>                        <a id="9632279" scrollto-destination="9632279"></a>
<div id="DSV005F2" data-id="DSV005F2" class="fig fig-section"><div class="label fig-label">Figures 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00502.jpeg?Expires=2147483647&Signature=D3MpQ4-wRKdvcArsyp6lQRbUT6O30bQ6pNIvPLtLX9~2nVzDhpdRIMlmTzt~fRA-f9l5SmXKRmJ4W4FM5xJ80T77kSJnY3eXMbAUCSUH2nVfecV8MJlQgtKOvQUm1BKPQ0gMwQWI6-UFGJ0nWO3ZX60zNeoY2h45Yc8olSH2WYOHTHPrMReknwsSsj5WL8yhP06dOQDcqzxdIN6xEl2ml9a73lwjFbH5ObVA4CLLeAI8JiFqmz6b0v1hPEu~d0T-4UojKH8DLu18e8r~8DxAzRb4mQGT0mYn9mcxMzdnkXII9oU27azyU988ReDyXyZfdT2TQqy2v2tPKqyzo6mCHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a–d) Frequency distributions of alternative alleles of SNPs from protein-coding regions. X-axis represents the allele frequency (ranges from 0 to 1) and Y-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the Y-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the Y-axis is scaled from 0 to 50 (in percentage)." path-from-xml="dsv00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632279/dsv00502.jpeg" path-from-xml="dsv00502.jpeg" target="_blank">View large</a><a section="9632279" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00502.jpeg?Expires=2147483647&Signature=ewD6IndjaCZ1QtDba-7BAbUZS9pgoD7hzH4MX8N9PEsHza1cGuAbhRwQQ0Dww~LP23pGoSf6Gdjh-nlgrhmsdBoqVBBMv-ghkLzrsUStrX155wy~YRSRFBO9Qrq4glXxiMgr4nCvfdO8SWXG6izR3kDt5yLWghlC-4cx5U~~H2AuyvHIEFjcP7zE3bflJl8IkOPOrIPC1m4DfF9hp0yGVCqFWFvgmmdKKRmx018MDw8UEhhF5dtJgzsGENkHht~unGRLVuoeithmJFt5kQk6mCFexDym1Z7STgTRlnj1J-MEQeOx64RsqzQ4NzAEmjcFBAmwEHbVgXTmd-uwQ8-SbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632279&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a–d) Frequency distributions of alternative alleles of SNPs from protein-coding regions. <em>X</em>-axis represents the allele frequency (ranges from 0 to 1) and <em>Y</em>-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the <em>Y</em>-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the <em>Y</em>-axis is scaled from 0 to 50 (in percentage).</p></div></div><div content-id="DSV005F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figures 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00502.jpeg?Expires=2147483647&Signature=D3MpQ4-wRKdvcArsyp6lQRbUT6O30bQ6pNIvPLtLX9~2nVzDhpdRIMlmTzt~fRA-f9l5SmXKRmJ4W4FM5xJ80T77kSJnY3eXMbAUCSUH2nVfecV8MJlQgtKOvQUm1BKPQ0gMwQWI6-UFGJ0nWO3ZX60zNeoY2h45Yc8olSH2WYOHTHPrMReknwsSsj5WL8yhP06dOQDcqzxdIN6xEl2ml9a73lwjFbH5ObVA4CLLeAI8JiFqmz6b0v1hPEu~d0T-4UojKH8DLu18e8r~8DxAzRb4mQGT0mYn9mcxMzdnkXII9oU27azyU988ReDyXyZfdT2TQqy2v2tPKqyzo6mCHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a–d) Frequency distributions of alternative alleles of SNPs from protein-coding regions. X-axis represents the allele frequency (ranges from 0 to 1) and Y-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the Y-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the Y-axis is scaled from 0 to 50 (in percentage)." path-from-xml="dsv00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632279/dsv00502.jpeg" path-from-xml="dsv00502.jpeg" target="_blank">View large</a><a section="9632279" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00502.jpeg?Expires=2147483647&Signature=ewD6IndjaCZ1QtDba-7BAbUZS9pgoD7hzH4MX8N9PEsHza1cGuAbhRwQQ0Dww~LP23pGoSf6Gdjh-nlgrhmsdBoqVBBMv-ghkLzrsUStrX155wy~YRSRFBO9Qrq4glXxiMgr4nCvfdO8SWXG6izR3kDt5yLWghlC-4cx5U~~H2AuyvHIEFjcP7zE3bflJl8IkOPOrIPC1m4DfF9hp0yGVCqFWFvgmmdKKRmx018MDw8UEhhF5dtJgzsGENkHht~unGRLVuoeithmJFt5kQk6mCFexDym1Z7STgTRlnj1J-MEQeOx64RsqzQ4NzAEmjcFBAmwEHbVgXTmd-uwQ8-SbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632279&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a–d) Frequency distributions of alternative alleles of SNPs from protein-coding regions. <em>X</em>-axis represents the allele frequency (ranges from 0 to 1) and <em>Y</em>-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the <em>Y</em>-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the <em>Y</em>-axis is scaled from 0 to 50 (in percentage).</p></div></div>                        <a id="9632280" scrollto-destination="9632280"></a>
<div id="DSV005F3" data-id="DSV005F3" class="fig fig-section"><div class="label fig-label">Figures 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00503.jpeg?Expires=2147483647&Signature=TkFarJoDt1uWeT4J5unL-Q2V8t754O8zRYX3YqCR5Sp6iGFYX5lSZAw44sFdDyyJSmrcs7S~XAE77QV1lPtSTF8Xoe2~TReyyKmlkc3VdTcCVl0qge3DECTk1S2QeoDSnST4NqptaLZTJmabtZWxoGuh8z9U3vjRrHDQV8ivZTqt7gbGtdgoNi7s0KNH72~RHf02GFZv-X7vBYFuWQp5oZK8TSrMcPzYpNX6vcouVVO4ZwhGVFNpEWSE~Qy0rO5WENr3-4yK23xOnff8M4cPqQWF25LwPSXDHOjLqeCixw7yvO4x1QddbFu4p2Us~jYgvs63G~Sm57StII99JPAX0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a–-d) Frequency distributions of alternative alleles of SNPs from intergenic, intronic, up/downstream, splicing, ncRNA and UTR categories. X-axis represents the allele frequency (ranges from 0 to 1) and Y-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the Y-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the Y-axis is scaled from 0 to 50 (in percentage)." path-from-xml="dsv00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632280/dsv00503.jpeg" path-from-xml="dsv00503.jpeg" target="_blank">View large</a><a section="9632280" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00503.jpeg?Expires=2147483647&Signature=CFN9JR38e9mQZdHjHxL9VgbkVKo4oUc3Uam7ZjBwgHEYZOcvf8SuVrWeDzIDs4ugFoMcg5C32QkmWzlaD~eg9dCqH8H0ip~sbnotUOWZK2n4N~NcIpzc2O9Gmv9Lk5PH07sKcux9vSKg8nmKNzxtX8xYqUgAf3W0IndQQ28tC-XKhitZdAYMnUxhti-xwrN5BoVYO5hne2-mYaRompUMBe3gEbys-HSLcvwzBEsXBuQ-iHeugi~oWtKYFNUkddsWeplCX1lUVgfDWFJH3EPLhasclPWWiFKlIrPqbXmi7GyYS0NhqWOrYaxnXnXqfABx1ijJc-9YF6~fFjrhUNP23A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632280&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a–-d) Frequency distributions of alternative alleles of SNPs from intergenic, intronic, up/downstream, splicing, ncRNA and UTR categories. <em>X</em>-axis represents the allele frequency (ranges from 0 to 1) and <em>Y</em>-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the <em>Y</em>-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the <em>Y</em>-axis is scaled from 0 to 50 (in percentage).</p></div></div><div content-id="DSV005F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figures 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00503.jpeg?Expires=2147483647&Signature=TkFarJoDt1uWeT4J5unL-Q2V8t754O8zRYX3YqCR5Sp6iGFYX5lSZAw44sFdDyyJSmrcs7S~XAE77QV1lPtSTF8Xoe2~TReyyKmlkc3VdTcCVl0qge3DECTk1S2QeoDSnST4NqptaLZTJmabtZWxoGuh8z9U3vjRrHDQV8ivZTqt7gbGtdgoNi7s0KNH72~RHf02GFZv-X7vBYFuWQp5oZK8TSrMcPzYpNX6vcouVVO4ZwhGVFNpEWSE~Qy0rO5WENr3-4yK23xOnff8M4cPqQWF25LwPSXDHOjLqeCixw7yvO4x1QddbFu4p2Us~jYgvs63G~Sm57StII99JPAX0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a–-d) Frequency distributions of alternative alleles of SNPs from intergenic, intronic, up/downstream, splicing, ncRNA and UTR categories. X-axis represents the allele frequency (ranges from 0 to 1) and Y-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the Y-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the Y-axis is scaled from 0 to 50 (in percentage)." path-from-xml="dsv00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632280/dsv00503.jpeg" path-from-xml="dsv00503.jpeg" target="_blank">View large</a><a section="9632280" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00503.jpeg?Expires=2147483647&Signature=CFN9JR38e9mQZdHjHxL9VgbkVKo4oUc3Uam7ZjBwgHEYZOcvf8SuVrWeDzIDs4ugFoMcg5C32QkmWzlaD~eg9dCqH8H0ip~sbnotUOWZK2n4N~NcIpzc2O9Gmv9Lk5PH07sKcux9vSKg8nmKNzxtX8xYqUgAf3W0IndQQ28tC-XKhitZdAYMnUxhti-xwrN5BoVYO5hne2-mYaRompUMBe3gEbys-HSLcvwzBEsXBuQ-iHeugi~oWtKYFNUkddsWeplCX1lUVgfDWFJH3EPLhasclPWWiFKlIrPqbXmi7GyYS0NhqWOrYaxnXnXqfABx1ijJc-9YF6~fFjrhUNP23A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632280&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a–-d) Frequency distributions of alternative alleles of SNPs from intergenic, intronic, up/downstream, splicing, ncRNA and UTR categories. <em>X</em>-axis represents the allele frequency (ranges from 0 to 1) and <em>Y</em>-axis represents percentage of SNPs within a category. For the broiler, brown egg layer (BEL) and white egg layer (WEL) groups, the <em>Y</em>-axis is scaled from 0 to 25 (in percentage), and for Inbred group, the <em>Y</em>-axis is scaled from 0 to 50 (in percentage).</p></div></div><p>Second, although both non-synonymous INTOL and TOL variants showed left-skewed distribution, much greater proportion of INTOL SNPs (69–73%) fell within lower frequency bins (≤0.3) compared with TOLs (43–47%) in the commercial groups. This pattern provides confirmation to the assumption that INTOLs are more deleterious than the TOLs, but this does not equate the TOL variants with the neutral ones. Rather the left-skewed distributions of both TOL and INTOL variants indicate that non-synonymous SNPs in general are selected against.</p><p>Third, comparison of frequency distribution of SNPs within MCE and non-MCE revealed that only those MCE-SNPs belonging to AA-altering, splicing or ncRNA categories had a marked difference in the allele frequency distribution compared with non-MCE categories with the former group clustering around lower frequency bins (AAF ≤ 0.30) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3a</a>–<a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsv005/-/DC1" target="_blank">d</a>). MCE-SNPs belonging to other annotation categories such as intergenic, intronic, etc. all showed the same pattern of frequency distributions as of neutral categories.</p><p>Fourth, within the inbred group, all types of variants, irrespective of their categories, showed very high level of fixation. However, the number of AA-altering and splicing variants in inbred lines were much fewer compared with those from commercial groups (Figs <a reveal-id="DSV005F2" data-open="DSV005F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSV005F3" data-open="DSV005F3" class="link link-reveal link-table xref-fig">3</a>). Especially, the number of non-syonymous INTOL variants in inbred lines was 8–11 times less than those in commercial groups. This observation indicates that the negative selection has prevented highly deleterious mutations from reaching fixation in the inbred lines and the existing INTOL variants are possibly not severely harmful under the environmental conditions these birds are raised.</p><p>Finally, Figs <a reveal-id="DSV005F2" data-open="DSV005F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSV005F3" data-open="DSV005F3" class="link link-reveal link-table xref-fig">3</a> reveal that there were no SNPs within low-frequency bin (≤0.05). This result was expected due to a number of reasons. First of all, the allele frequency in the present study was calculated only based on 10–15 individuals per line. This meant that estimated frequency within line could not be below 0.03–0.05. Second, the sequencing of pooled DNA (Pool-seq) is known to be extremely prone to losing rare and low-frequency variants when performed at a low depth of sequencing coverage.<sup><a reveal-id="DSV005C36" data-open="DSV005C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSV005C37" data-open="DSV005C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In the present study, the sequence coverage was 8–17X, which was often lower than the number of individuals per pool (10–15) and the corresponding number of chromosomes (20–30) that needed to be sampled. This was compounded by the use of various filtration parameters such as quality scores, minimum coverage and the evidence of non-reference allele in both forward and reverse strands, which resulted in the loss of many true SNPs, especially from low-frequency categories. Cutler and Jensen<sup><a reveal-id="DSV005C37" data-open="DSV005C37" class="link link-ref link-reveal xref-bibr">37</a></sup> predicted that filtering SNPs from Pool-seq data with a Phred quality score of 20 (predicted error rate 1%) would lead to the loss of 29% of the variants when the coverage is between 11 and 50X, and majority of these variants will be rare and of low frequency. They also showed that if the error rate was very low (e.g. 0.1%) and only a small number of samples were pooled (e.g. 10 samples) that would require sequencing to a depth of 300X to detect all the variants including the rare ones.</p><p>The proportions of pfVars that appeared in multiple lines (&gt;10 lines) at high frequencies (&gt;0.9) were investigated as these might represent variants with adaptive advantage (Table <a reveal-id="DSV005TB1" data-open="DSV005TB1" class="link link-reveal link-table xref-fig">1</a>). Alternatively, this is also possible that these variants are only moderately deleterious or neutral and have increased in frequency through genetic drift or by hitchhiking with linked variants under positive selection. A very small proportions (0.5–2%) of stop-gain/loss, INTOL and splicing variants were detected from multiple lines at high frequency, while for categories like synonymous, intergenic, intronic, up/downstream and UTR, the corresponding proportions were much higher, above 7% (Table <a reveal-id="DSV005TB1" data-open="DSV005TB1" class="link link-reveal link-table xref-fig">1</a>). This agrees with the general expectation that the former group of variants are selected against due to their potential harmful effects. Since stop-gain variants result in the premature termination of proteins, they were further investigated which revealed that in most cases (85%) they were located: (i) in genes that have other paralogs in the genome, which might minimize their impact (64%); (ii) in uncharacterized hypothetical genes for which no functions are yet known and these genes may be prediction artefacts (25%); (iii) near the end of coding sequence thus allowing the translation of at least 90% of the length of the protein retaining most of its functionality (16%) or at the very beginning of the protein (12%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). A similar observation was reported by Ng <em>et al.</em><sup><a reveal-id="DSV005C38" data-open="DSV005C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Yet there were a few stop-gain variants that did not conform to any of these rules, yet were detected at very high frequencies (&gt;0.9) from multiple lines (&gt;10). These included three stop-gain mutations from the genes: <em>C1ORF101</em> (Chr3_34083273), <em>CCDC60</em> (Chr15_9679384) and <em>GLOD4</em> (Chr19_6926808). Their relative position within the respective proteins (calculated by dividing the position of the variant with the protein length) varied from 0.33 to 0.67. These variants may therefore have some selective advantage in domesticated chicken lines. Similarly, over 80% of the INTOL variants detected from multiple lines at high frequencies were located within genes with one or more paralogs or in novel genes with unknown function as discussed above.</p><p>Over 14% (<em>n</em> = 2,248,437) of the 15 million SNPs were private to individual lines. Of these, only 12.5% SNPs were fixed or nearly fixed. Although much smaller proportions of private fixed variants belonged to AA-altering and splicing categories (Table <a reveal-id="DSV005TB1" data-open="DSV005TB1" class="link link-reveal link-table xref-fig">1</a>), these could be important for the lines in question.</p>                            <h3 scrollto-destination=9632287 id="9632287" class="section-title" data-legacy-id=s3g>3.7. SNPs within regions of selection signature</h3>
<p>One of the signatures of positive selection is that it creates regions of local fixation compared with the overall pattern of diversity in the entire genome—a phenomenon known as a ‘selective sweep’.<sup><a reveal-id="DSV005C17" data-open="DSV005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The regions of local fixation were investigated by calculating pooled heterozygosities (Hp) in sliding windows using the SNP data for individual broiler, BEL and WEL lines. Since only the windows with &gt;10 SNPs were analysed, the number of analysable windows varied from 44,237 to 45,378 per line (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The average Hp values varied not only among the lines (0.22–0.36) but also among chromosomes (0.16–0.43), which is likely due to the wide variation in recombination rates among chromosomes.<sup><a reveal-id="DSV005C39" data-open="DSV005C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Therefore, in deciding the threshold Hps for detecting putative selective sweep (pSS) regions, chromosome-specific permutation analysis was performed within each line. Proportion of the windows with fixation signal varied considerably across the lines ranging from 0.7 to 9.8% at <em>P</em> &lt; 0.001 while with relaxed thresholds (at <em>P</em> &lt; <em>0.01</em> or <em>P</em> &lt; <em>0.05)</em>, the number increased considerably.</p><p>Differentiating the signatures generated by true selective sweep is challenging, because similar effects might be left on the genome by demographic events and population structure.<sup><a reveal-id="DSV005C17" data-open="DSV005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> While the permutation test improves the confidence in detecting pSS, comparison of the signals across populations can further confirm the results and minimize the incidence of false signals from the confounding effects of demographic events.<sup><a reveal-id="DSV005C18" data-open="DSV005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Therefore, in the following sections, we focus only on the common windows that passed critical Hps (at <em>P</em> &lt; 0.05) in all the lines within a group (broiler, BEL or WEL) or across groups. Sex chromosomes were excluded from analyses to avoid interpretational problems as the lines were differentially consisted of either male or female samples.</p><p>Common signals were detected only from 143 widows in broilers, 163 in BEL and 49 in WEL. Many of these windows were adjacent or overlapping and hence were joined to obtain non-overlapping regions that resulted in 60 discrete pSS regions for broiler, 66 for BEL and 25 for WEL (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). The relatively lower number of shared regions in the WEL group was due to a single line—WEL5, which elicited very few fixation signals (only 4% windows compared with ∼14% in other WEL lines). This is due to the fact the WEL5 had higher level of fixation relative to other lines and as a result had low critical Hps from the chromosome-wise permutation analyses causing very few windows to elicit selective sweep signal. If WEL5 is excluded from the comparison, 208 windows and 100 non-overlapping pSS regions are found to be shared among the other four WEL lines. Across different groups, we discovered only seven windows shared between broilers and BELs, five between BEL and WEL (excluding WEL5) and only one shared between broiler and WEL (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>).</p><p>As a proof that our approach could detect sweep regions, our results were compared with previous studies. This demonstrated that many of the genes that were partly or fully covered within our pSS regions had also been detected by other studies. Some notable genes among these are the following: <em>BCDO2</em> (β-carotene oxygenase 2) that has been found associated with yellow skin in domestic chickens;<sup><a reveal-id="DSV005C40" data-open="DSV005C40" class="link link-ref link-reveal xref-bibr">40</a></sup><em>TSHR</em> (thyroid-stimulating hormone receptor) playing important roles in regulation of metabolic functions and reproduction in commercial chickens;<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a></sup><em>IGF1</em>(insulin-like growth factor-1) and <em>HNF4G</em> (hepatocyte nuclear factor 4 gamma) which are candidates for growth;<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV005C41" data-open="DSV005C41" class="link link-ref link-reveal xref-bibr">41</a></sup><em>PMCH</em> (pro-melanin concentrating hormone) that plays important role in regulating appetite and metabolic functions;<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV005C42" data-open="DSV005C42" class="link link-ref link-reveal xref-bibr">42</a></sup><em>INSR</em> (insulin receptor), which affects growth traits with a central role in insulin signalling;<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and <em>NELL1</em>(<em>NEL-like 1</em>) gene, potentially associated with skeletal integrity in broiler.<sup><a reveal-id="DSV005C41" data-open="DSV005C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Moreover, over 91% of the pSS regions also overlapped with one or more QTLs (in chicken QTLdb) associated with production, behavioural- and health-related traits such as body weight, growth performance, meat quantity and qualities, fat content, egg production and qualities, feather pecking and antibody response to various pathogens. This provided further support in favour of predicted selection pressure on these regions.</p><p>The pSS regions harboured a large number of variants (6K–12K depending on groups), either fixed or segregating. While it is expected that the variants directly under selection will be fixed or be at high frequency (AAF &gt; 0.9), the other functionally active segregating variants within the pSS regions may be selectively advantageous or deleterious and will exert their effects quantitatively on the associated phenotype. About 38–56% of the variants in pSS regions were fixed in different groups. Many of the fixed variants, however, are likely to be neutral which might have reached fixation through hitchhiking just by being closely linked to causal mutations. Therefore, we decided to focus mainly on the pfVars viz. AA-altering, splicing, RNA structure-altering, MCE variants or those in ncRNAs. This resulted in 4,679 variants from all the pSS regions of which only 17% (<em>n</em> = 815) were fixed in at least one chicken group and 2% (<em>n</em> = 106) were fixed across the groups (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). We hypothesized that if a window elicited selection signal from more than one group and if variants therein were present ubiquitously and were fixed across the groups, this most likely represented old selection—possibly associated with traits that promote high production rates or better performance as a result of commercialization of the chicken or are relevant to adaptation in domesticated situations. On the other hand, if a pSS region was detected within a single group and variants were fixed only within that group, it was probably associated with group-specific traits. The major findings are discussed in the following sections.</p>                            <h3 scrollto-destination=9632297 id="9632297" class="section-title" data-legacy-id=s3h>3.7.1. Ubiquitously present pfVars in the BCDO2 gene are candidates for yellow skin trait</h3>
<p>Yellow skin colour is a ubiquitous feature in all Western commercial chicken.<sup><a reveal-id="DSV005C40" data-open="DSV005C40" class="link link-ref link-reveal xref-bibr">40</a></sup> It has been postulated that in chicken the trait is expressed due to the presence of one or more <em>cis</em>-acting and tissue-specific regulatory variants that inhibit(s) the expression of the <em>BCDO2</em> gene, which otherwise is responsible for degradation of carotenoid. In a previous study, Eriksson <em>et al.</em><sup><a reveal-id="DSV005C40" data-open="DSV005C40" class="link link-ref link-reveal xref-bibr">40</a></sup> suggested a minimum haplotype of 23.8 kb on chromosome 24 to be shared across diverse domesticated chicken breeds with yellow skin, and this haplotype is expected to harbour the causal mutation for the trait.</p><p>In the present study, the 40-kb window (Chr24:6120000–6160000) that covered the minimum haplotype of yellow skin trait elicited fixation signals in all the broiler and BEL lines. Within the WEL group, however, only two of the five lines produced signal for this window and as a result it was not detected as a common pSS region from the WEL group. Closer inspection of the variants within the window revealed that there were several pfVars (<em>n</em> = 13), of which seven were located directly within the yellow skin haplotype and were completely fixed across all the commercial lines including the WELs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). All of these variants are located either within the <em>BCD02</em> gene or nearby this gene and hence can be considered as candidate causal mutations. These include: 1 intergenic-MCE, 3 intronic-MCE, 2 non-synonymous TOL and 1 UTR5′ SNP that was suggestively RNA structure altering. In spite of complete fixation of these variants, the failure to detect a signal for the region from certain WEL lines suggests that the sweep region is smaller than the 40-kb window size used in this study, which is consistent with the size of shared haplotype (23.8 kb). Erriksson <em>et al.</em><sup><a reveal-id="DSV005C40" data-open="DSV005C40" class="link link-ref link-reveal xref-bibr">40</a></sup> speculated that the causal mutation(s) of the yellow skin trait will be within evolutionary conserved region(s) and will be regulatory in nature. Considering these, the variants within MCEs appear to be the most likely candidates.</p>                            <h3 scrollto-destination=9632300 id="9632300" class="section-title" data-legacy-id=s3i>3.7.2. Ubiquitously present non-synonymous SNPs in the TSHR gene are potential causal mutations of a known sweep locus</h3>
<p>Two recent studies have, respectively, detected and validated a strong signature of selection overlapping the <em>TSHR</em> gene.<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV005C43" data-open="DSV005C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Although the first study speculated the gene to be associated with domestication in chicken, the later study found a more recent origin of this signature and concluded that the <em>TSHR</em> mutation was neither a pre-requisite nor a critical factor in domestication process; instead it is most probably related to the improvement of chickens that began during the industrial revolution. Although the causal mutation(s) within this region and its phenotypic effects are yet to be established, the biological significance of the <em>TSHR</em> gene on the regulation of metabolic activities and reproduction, particularly in extending the mating season, is well known.<sup><a reveal-id="DSV005C44" data-open="DSV005C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSV005C45" data-open="DSV005C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Rubin <em>et al.</em><sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> detected a non-synonymous variation within the <em>TSHR</em> gene that they suspected was the causal mutation.</p><p>The present study also detected a strong signal covering part of the <em>TSHR</em> gene from the analyses of broilers and BEL lines. Within the WEL group, four of the five lines were significant for the same <em>TSHR</em> window but due to failure in getting signal from a single line, it was not ultimately picked up as a common signal. About 62% (<em>n</em> = 221) of the variants within the gene had average frequency &gt;0.90 suggesting a very strong selection pressure. There were only two fixed pfVars in this region though (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>): a non-synonymous TOL mutation and a non-synonymous INTOL SNP, the same variant that had also been detected by Rubin <em>et al.</em><sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> This INTOL variant also coincided with an MCE. Both the variants were completely fixed in all the commercial lines except in a single WEL line, although in this line the variants prevailed at high frequency (0.6–0.8). These variants were also fixed in non-commercial lines such as RI-J and inbred lines. These could therefore be the candidate causal mutations.</p>                            <h3 scrollto-destination=9632303 id="9632303" class="section-title" data-legacy-id=s3j>3.7.3. The putative sweep regions show enrichment of non-coding mutations within higher frequency classes</h3>
<p>Plotting the ratio of the number of non-coding and coding variants against allele frequency reveals that there is a significant enrichment of non-coding mutations at higher frequencies compared with coding variants (<em>P</em> &lt; 0.001 in <em>χ</em><sup>2</sup> test) (Fig. <a reveal-id="DSV005F4" data-open="DSV005F4" class="link link-reveal link-table xref-fig">4</a>). This is expected as non-synonymous, splicing or stop-gain/loss variants can have radical impacts on the translated protein, whereas the functional non-coding variants may be much more tolerated due their regulatory roles on gene expression.<sup><a reveal-id="DSV005C9" data-open="DSV005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> This is consistent with the observations from various genome-wide association studies that revealed that majority of the genetic variants associated with complex traits lie within non-coding regions of the genome.<sup><a reveal-id="DSV005C8" data-open="DSV005C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSV005C9" data-open="DSV005C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                        <a id="9632305" scrollto-destination="9632305"></a>
<div id="DSV005F4" data-id="DSV005F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00504.jpeg?Expires=2147483647&Signature=noI3cHn3QPZEZjG953ywGJnlcLHu2QBCHKq2g1E3vV4Psa~UeTof0jI-efQ8JShXaZrzYTh-zPMPiL~AuqVDtA5dGKlRzmk4PJeo9qIRCTmjhel1DsNmN2JesazdWnts1tfgYQRM-zO5B3ns~6CiWI6idPEqfFsSuRdY710Qbp13fBNbZ1-Qo~xgs7ZJcYmsXVEhaK6HTPtPGKVK5PKH2RYK~DKHvMQu5~jyZOBeXfwiQZICFtPDYgSXvW73YXf7JM4fh4dz5WULGs5zXs4srABCI-nSfhwkVfDg8pteIio~nHdBcHukNP0uBKGcKTKEFIBjlBDSSBER2LCp8p30zg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ratio of non-coding putatively functional variants (pfVars) with AA altering and splicing types at different allele frequency bins within selective sweep regions. The non-coding pfVars included intergenic, intronic, UTR and down/upstream variants from most conserved elements, variants located within known ncRNAs and variants that were predicted to alter RNA secondary structure. The graph indicates that high-frequency variants are enriched with non-coding types. BR, BEL and WEL refer to broiler, brown egg layer and white egg layer groups, respectively." path-from-xml="dsv00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632305/dsv00504.jpeg" path-from-xml="dsv00504.jpeg" target="_blank">View large</a><a section="9632305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00504.jpeg?Expires=2147483647&Signature=SyEDqcbrTpdXjAm5Tx7TUx2n8LHDZrl1AQayGHhY3INMq6veFsrLpOVDIs2Ecxfft3BfLNnHpujDiRGNYAro3mdQhVoPgWWQ~vDtiCQro6R9vfuGYaF2YwBEPhDLD8KZOwbax9GmbPyesO1c0IvgdEJnzDhum9VkvmA9NCBzy9GMptnu-TAQUqZz4YgnxMvpaT6tLXdgLxxOdDdJM2a2CkQp95wAQd27AwT4rZtKVm7kRzw~MBPp70bHvRWFpMyzJ1GzZU~mlJPt4jtPSW5u6rbdbOlPCz7Y4D-Z6s59tB8idoLa-vgq24eNZvF~wTIepm6cSFjHgcofIs-t7uUlcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632305&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Ratio of non-coding putatively functional variants (pfVars) with AA altering and splicing types at different allele frequency bins within selective sweep regions. The non-coding pfVars included intergenic, intronic, UTR and down/upstream variants from most conserved elements, variants located within known ncRNAs and variants that were predicted to alter RNA secondary structure. The graph indicates that high-frequency variants are enriched with non-coding types. BR, BEL and WEL refer to broiler, brown egg layer and white egg layer groups, respectively.</p></div></div><div content-id="DSV005F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/m_dsv00504.jpeg?Expires=2147483647&Signature=noI3cHn3QPZEZjG953ywGJnlcLHu2QBCHKq2g1E3vV4Psa~UeTof0jI-efQ8JShXaZrzYTh-zPMPiL~AuqVDtA5dGKlRzmk4PJeo9qIRCTmjhel1DsNmN2JesazdWnts1tfgYQRM-zO5B3ns~6CiWI6idPEqfFsSuRdY710Qbp13fBNbZ1-Qo~xgs7ZJcYmsXVEhaK6HTPtPGKVK5PKH2RYK~DKHvMQu5~jyZOBeXfwiQZICFtPDYgSXvW73YXf7JM4fh4dz5WULGs5zXs4srABCI-nSfhwkVfDg8pteIio~nHdBcHukNP0uBKGcKTKEFIBjlBDSSBER2LCp8p30zg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ratio of non-coding putatively functional variants (pfVars) with AA altering and splicing types at different allele frequency bins within selective sweep regions. The non-coding pfVars included intergenic, intronic, UTR and down/upstream variants from most conserved elements, variants located within known ncRNAs and variants that were predicted to alter RNA secondary structure. The graph indicates that high-frequency variants are enriched with non-coding types. BR, BEL and WEL refer to broiler, brown egg layer and white egg layer groups, respectively." path-from-xml="dsv00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9632305/dsv00504.jpeg" path-from-xml="dsv00504.jpeg" target="_blank">View large</a><a section="9632305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv00504.jpeg?Expires=2147483647&Signature=SyEDqcbrTpdXjAm5Tx7TUx2n8LHDZrl1AQayGHhY3INMq6veFsrLpOVDIs2Ecxfft3BfLNnHpujDiRGNYAro3mdQhVoPgWWQ~vDtiCQro6R9vfuGYaF2YwBEPhDLD8KZOwbax9GmbPyesO1c0IvgdEJnzDhum9VkvmA9NCBzy9GMptnu-TAQUqZz4YgnxMvpaT6tLXdgLxxOdDdJM2a2CkQp95wAQd27AwT4rZtKVm7kRzw~MBPp70bHvRWFpMyzJ1GzZU~mlJPt4jtPSW5u6rbdbOlPCz7Y4D-Z6s59tB8idoLa-vgq24eNZvF~wTIepm6cSFjHgcofIs-t7uUlcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9632305&ar=591312&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Ratio of non-coding putatively functional variants (pfVars) with AA altering and splicing types at different allele frequency bins within selective sweep regions. The non-coding pfVars included intergenic, intronic, UTR and down/upstream variants from most conserved elements, variants located within known ncRNAs and variants that were predicted to alter RNA secondary structure. The graph indicates that high-frequency variants are enriched with non-coding types. BR, BEL and WEL refer to broiler, brown egg layer and white egg layer groups, respectively.</p></div></div><p>Between 37 and 45% of the pSS regions across all the groups were completely devoid of genes and coincided entirely with intergenic regions and 54–70% of all the pSS regions showed fixation of only non-coding variants including intergenic, intronic, up/downstream and UTR SNPs. For instance, some of the genes found within the pSS regions were particularly enriched for fixed intronic pfVars (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>) and most of these genes have previously been detected in other selection signature studies<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV005C41" data-open="DSV005C41" class="link link-ref link-reveal xref-bibr">41</a></sup> as well, thereby strengthening their likelihood to be within true sweep regions. It is possible that these intronic variants exert their effects through regulation or disruption of transcriptional activities, splicing efficiency of their host genes or expression of alternative transcripts.<sup><a reveal-id="DSV005C46" data-open="DSV005C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p><p>Overlaying the known QTLs with the pSS regions reveals that the regions harbouring only non-coding fixed pfVars overlapped with QTLs of complex traits. For example, a large number of pSS regions detected in our study overlapped with QTLs for bone traits such as bone strength, bone-mineral content, bone-density, etc., suggesting a multi-gene hypothesis for regulation of these traits. Most of these pSS regions also coincided with QTLs for other traits as well, which further indicates possible pleiotropic effects of the same genes on multiple traits.<sup><a reveal-id="DSV005C47" data-open="DSV005C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSV005C48" data-open="DSV005C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Only one of the pSS regions (B-R47 in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>) from broiler lines, however, overlapped with only bone-related QTL(s) viz. Tibia width<sup><a reveal-id="DSV005C47" data-open="DSV005C47" class="link link-ref link-reveal xref-bibr">47</a></sup> and mineral density in femur bone,<sup><a reveal-id="DSV005C48" data-open="DSV005C48" class="link link-ref link-reveal xref-bibr">48</a></sup> thereby allowing its closer inspection. Both the QTLs, however, were detected with only suggestive significance and as a result their effect size is expected to be small, which is an indication of involvement of regulatory causal variant(s). This pSS region overlaps with the <em>PAPPA2 (Pappalysin 2)</em> gene. The protein encoded by this gene cleaves the <em>IGFBP-5</em> (insulin-like growth factor binding protein 5), which is an important stimulator of bone formation.<sup><a reveal-id="DSV005C49" data-open="DSV005C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Among all the variants detected from and around this gene, only non-coding SNPs reached fixation including 46 intronic, 2 upstream and 37 intergenic variants. Only one of the fixed intronic variant (Chr8:6817157) coincided with an MCE. This particular variant was found to be fixed in all the broiler and WEL lines and also in two of the five BEL lines. It is possible that this is a functional variant with a regulatory role.</p>                            <h3 scrollto-destination=9632308 id="9632308" class="section-title" data-legacy-id=s3k>3.7.4. Evolutionary intolerant non-synonymous SNPs prevailing at high frequencies may confer adaptive advantage</h3>
<p>From the pSS regions, 11 non-synonymous INTOL variants were detected that reached fixation in one or more groups. Apart from the one INTOL SNP from the <em>TSHR</em> gene that has been discussed above, the other 10 variants were detected from the following genes: <em>GCNT3</em> (<em>n</em> = 2), <em>PLA2R1</em> (<em>n</em> = 1), <em>FOXI1</em> (<em>n</em> = 1), <em>ZNF507</em> (<em>n</em> = 1), <em>ADAMTS7</em> (<em>n</em> = 1) and <em>ENSGALG00000023731</em> (novel chicken gene; <em>n</em> = 4). Even though the pSS regions overlapping these genes were detected from specific groups (either BEL or WEL), a number of these INTOL (<em>n</em> = 5) variants prevailed ubiquitously at high frequency or at complete fixation in all or most of the lines irrespective of groups. Some of the variants also coincided with MCEs. Their high frequency implies that even though evolutionary intolerant, these variants possibly confer important adaptive advantages. Classic examples of genetic defects being selected as adaptive advantage includes the prevalence of certain blood-related disorders in malaria-infected regions as a means to confer resistance to the latter disease<sup><a reveal-id="DSV005C50" data-open="DSV005C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and the increase in frequency of the risk allele of diabetes mellitus during food scarcity in the past.<sup><a reveal-id="DSV005C51" data-open="DSV005C51" class="link link-ref link-reveal xref-bibr">51</a></sup> It is, however, also possible that the fixed INTOL variants detected in the present study are not actually beneficial but have been hitchhiked to fixation with linked causal mutation.<sup><a reveal-id="DSV005C52" data-open="DSV005C52" class="link link-ref link-reveal xref-bibr">52</a></sup></p><p>Exploring the functions of the above genes harbouring the INTOL variants reveals that they perform a variety of functions which may be related to commercially important traits in chicken. The <em>GCNT3</em> (<em>glucosaminyl N-acetyl transferase 3, Mucin Type</em>) gene, for instance, plays an important role in the immune system through the synthesis of mucins in mucus-secretary epithelial tissues.<sup><a reveal-id="DSV005C53" data-open="DSV005C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Mucin hyper-secretion can be a common response to many pathogenic infections and cancers.<sup><a reveal-id="DSV005C54" data-open="DSV005C54" class="link link-ref link-reveal xref-bibr">54</a></sup> The proteins encoded by the <em>PLA2R1</em> (<em>phospholipase A2 receptor 1</em>) gene exert various effects in different tissues including inducement of cell proliferation and production of lipid mediators by serving as a receptor of phospholipase.<sup><a reveal-id="DSV005C55" data-open="DSV005C55" class="link link-ref link-reveal xref-bibr">55</a></sup> The region harbouring this gene overlapped with QTLs associated with several traits most notably for fat deposition, body weights and egg production performance. The <em>FOXI1</em> (<em>Forkhead box I1</em>) gene codes for a transcription factor that in human is involved in the physiology of several organs viz. inner ear, testis and kidney.<sup><a reveal-id="DSV005C56" data-open="DSV005C56" class="link link-ref link-reveal xref-bibr">56</a></sup> A recent study in human has detected a sweep region covering this gene and predicted that positive selection of the <em>FOXI1</em> gene imparts adaptive advantage in kidney-mediated water-electrolyte homeostasis in African population in response to climatic condition.<sup><a reveal-id="DSV005C57" data-open="DSV005C57" class="link link-ref link-reveal xref-bibr">57</a></sup> The pSS region covering this gene, in our study, overlapped with QTL(s) for growth, age-at-first egg and fear-related behaviour. The functions of the <em>ZNF507</em> and <em>ENSGALG00000023731</em> genes are not yet well studied, although <em>ZNF507</em> possibly plays role in transcriptional regulations affecting multiple functions.<sup><a reveal-id="DSV005C58" data-open="DSV005C58" class="link link-ref link-reveal xref-bibr">58</a></sup> The possible effect of the <em>ADAMTS7</em> gene and the fixed pfVars therein is discussed in the following section.</p>                            <h3 scrollto-destination=9632311 id="9632311" class="section-title" data-legacy-id=s3l>3.7.5 Broiler or layer-specific variants</h3>
<p>About 12% (<em>n</em> = 94) of the pfVars detected from various pSS regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>) were fixed exclusively in broilers while they were either undetected or segregating in layer groups indicating their possible association with broiler-related traits. Almost one-third (<em>n</em> = 28) of these variants resided within the largest broiler pSS region (B-R09 in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>) on chromosome1 and represented genes: <em>PARPBP, PMCH, NUP37, CCDC53, GNPTAB, SYCP3 and CHPT1</em>. This region also harboured the <em>IGF1</em> gene, which has known effect on growth, but the single pfVar from this gene (an intronic-MCE variant) was prevalent not only in broilers but also in layers in the present study. All these genes from B-R09 region were also detected within sweep regions in previous studies analysing commercial broiler lines.<sup><a reveal-id="DSV005C26" data-open="DSV005C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV005C41" data-open="DSV005C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p><p>The rest of the broiler-specific SNPs were either intergenic or belonged to a few genes, namely, <em>ROBO2, AMMECR1, BMPR1B, SPRED1, MEIS2, ARFGEF2, USE-1 and MYO9B.</em> The <em>MYO9B</em> (<em>Myosin IXB</em>) gene is interesting as, in human, variants within this gene have been found associated with inflammatory bowel disease, ulcerative colitis and coeliac disease.<sup><a reveal-id="DSV005C59" data-open="DSV005C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DSV005C60" data-open="DSV005C60" class="link link-ref link-reveal xref-bibr">60</a></sup> In our study, the pSS region (B-R60 in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>) harbouring this gene coincided with QTLs associated with antibody responses to <em>Escherichia coli</em> and <em>Salmonella enteritidis</em>, and growth<em>.</em> We detected one broiler-specific non-synonymous TOL SNP and several other ubiquitously present intronic-MCE variants from this gene. Based on the information from human studies, we speculate that this gene might have roles in triggering antibody responses to <em>E. coli</em> and <em>Salmonella</em> in chicken. Moreover, being a myosin encoding gene, the broiler-specific variant(s) within this may also affect muscle growth.</p><p>A number of layer-specific variants were also discovered—some of which were fixed exclusively in either BEL or WEL lines while the rest were fixed in both the layer groups. The variants fixed in both the layer groups (<em>n</em> = 20) came from nine genes only. Since direct effects of most of these genes are yet to be established, we investigated the overlapping QTLs to establish potential association of these genes with layer-related traits. For instance, a number of pfVars were detected from the gene <em>ADAMTS7</em> (<em>ADAM metallopeptidase with thrombospondin type 1 motif, 7</em>) including one non-synonymous INTOL mutation. We speculate that the variants within this gene may be associated with egg-shell strength in chicken as one member of the <em>ADAMTS</em> gene family (<em>ADAMTS1</em>) has previously been shown to be up-regulated in the uterus of hens laying hard-shelled eggs compared with those laying soft-shelled eggs.<sup><a reveal-id="DSV005C61" data-open="DSV005C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The region harbouring the <em>ADAMTS7</em> gene overlapped with QTLs for several traits including two for egg-shell strength and stiffness.<sup><a reveal-id="DSV005C62" data-open="DSV005C62" class="link link-ref link-reveal xref-bibr">62</a></sup> Nevertheless, since the members of this gene family are known to perform a variety of physiological roles by encoding enzymes with protease activities, the <em>ADAMTS7</em> genes may have wider effects on multiple traits.<sup><a reveal-id="DSV005C63" data-open="DSV005C63" class="link link-ref link-reveal xref-bibr">63</a></sup></p><p>The genes, <em>DOCK2</em> (<em>dedicator of cytokinesis 2</em>) and <em>GFPT2</em> (<em>glutamine-fructose-6-phosphate transaminase 2</em>), were detected from two pSS regions on Chromosome13 and coincided with QTL(s) associated with age-at-first egg and body weight.<sup><a reveal-id="DSV005C64" data-open="DSV005C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Both the genes contained several layer-specific and some ubiquitous variants. Since age-at-sexual- maturation is correlated to body weight, these genes possibly have pleiotropic effects on these traits, with the fixed pfVars probably conferring later maturity in layers.</p><p>A number of genes with BEL-specific variants on chromosome 6 coincided with a QTL suggestively associated with egg shell colours<sup><a reveal-id="DSV005C65" data-open="DSV005C65" class="link link-ref link-reveal xref-bibr">65</a></sup> and these include: <em>FRMPD2</em> (containing 1 stop-gain, 2 non-syonymous TOL and 11 intronic-MCE-SNPs)<em>, HIF1AN</em> (1 intronic-MCE variant), <em>MYPN</em> (1 non-syonymous TOL SNP) and <em>ATOH7</em> (2 RNA structure-altering SNPs). The brown colour of egg shell derives from a major pigment called protoporphyrin, which is secreted from the surface epithelial cells of the uterus and is deposited onto the shell.<sup><a reveal-id="DSV005C66" data-open="DSV005C66" class="link link-ref link-reveal xref-bibr">66</a></sup> The shell colouration can be affected by a number of mechanisms that control either the production or precipitation of the protoporphyrin.<sup><a reveal-id="DSV005C67" data-open="DSV005C67" class="link link-ref link-reveal xref-bibr">67</a></sup> One recent study has suggested that the brown and white egg layers are not dissimilar in the protoporphyrin content, but it is the control mechanism affecting the secretion and deposition of the pigment that causes the difference in egg colour.<sup><a reveal-id="DSV005C66" data-open="DSV005C66" class="link link-ref link-reveal xref-bibr">66</a></sup> Although no direct evidence was found in the extant literature linking any of the above genes with egg shell colour, but the <em>FRMPD2</em> (<em>FERM and PDZ domain containing 2</em>) appeared to be a strong candidate as a regulator of protoporphyrin precipitation. This is because it encodes a peripheral membrane protein that binds phosphatidylinositol 3,4-bisphosphate, which in turn plays important role in membrane trafficking.<sup><a reveal-id="DSV005C68" data-open="DSV005C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DSV005C69" data-open="DSV005C69" class="link link-ref link-reveal xref-bibr">69</a></sup> This gene, can therefore, play a regulatory role in the secretion and deposition of protoporphyrin on egg shell.</p>                            <h2 scrollto-destination=9632317 id="9632317" class="section-title" data-legacy-id=s4>4. Concluding remarks</h2>
<p>This paper reports the discovery and characterization of over 15 million SNPs from the chicken genome with the goal to delineate those with potential functional consequences—either having adaptive advantage or deleterious effect. To our knowledge, this is so far the largest study of its kind on chicken not only because a huge number of variants have been detected but also because a large number of populations and individuals that have been screened. Moreover, a number of complementary approaches were adopted to identify putatively functional variants from both coding and non-coding part of the genome. By investigating local fixation in the genome, this study has also detected a number of putative sweep regions and candidate casual variants therein. This analysis has confirmed a number of previously known sweep regions such as those involving the <em>BCDO2</em> gene for yellow skin colour and the <em>TSHR</em> gene and has identified novel regions, e.g. those potentially associated with bone strength and morphology, antibody response to <em>E. coli and Salmonella</em>, egg-shell colour and shell strength.</p><p>In spite of the discovery of millions of SNPs, the present study failed to capture the rare and low-frequency variants (AAF &lt; 0.05)—a major limitation of adopting Pool-seq approach with low coverage. Low-frequency variants typically represent recent mutations and are often private to specific populations. Their detection requires screening of hundreds of individuals from each population, which was beyond the scope of the present study. Unlike in human, where discovery of rare disease-causing variants has medical significance, for farm animals it is unlikely to be a priority, unless a common disease is caused by many rare variants as proposed by ‘rare allele model’.<sup><a reveal-id="DSV005C70" data-open="DSV005C70" class="link link-ref link-reveal xref-bibr">70</a></sup> Instead, for farm animals the priority is to improve commercially important traits through selective breeding and unless a disease is common enough it is unlikely to be a major focus in breeding programmes. Therefore, we argue that even though, low-frequency variants could not be characterized in the present study, it is the common variants that would have major application in breeding programmes of chicken.</p><p>Although the present study made efforts to characterize as many non-coding pfVars as possible using various approaches—viz. finding variants within conserved loci and known ncRNAs, predicting their effects on RNA structure and characterizing high-frequency variants within putative sweep regions—still the effort fell short due to incomplete annotation of chicken genome for functional non-coding elements. Knowledge of these elements, however, is rapidly improving due to advancements in transcriptome profiling (e.g. RNA-seq) and our predictive abilities of their functions through <em>in silico</em> analyses. Better characterization of these variants will therefore be possible in near future.</p><p>In conclusion, it is expected that by providing a large catalogue of variants in general and potentially functional variants in particular, this study will have major implications in breeding programmes and future research in chicken in the areas of genomic selection, genome-wide association studies, fine mapping of QTL, haplotype mapping and locating candidate causal variants.</p>                            <h2 scrollto-destination=9632322 id="9632322" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=9632324 id="9632324" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was funded by a BBSRC/DEFRA LINK grant. C.B.'s contribution was sponsored by a fellowship from the National Council for Scientific and Technological Development – Brazil (CNPq, grant 201306/2010-7). Funding to pay the Open Access publication charges for this article was provided by the Research Council UK (RCUK).</p>                            <h2 scrollto-destination=9632326 id="9632326" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the Aviagen Ltd, the Hy-line International, the Synbreed Consortium of Germany and the Pirbright Institute for providing us with chicken samples and/or sequenced data.</p>                            <h2 scrollto-destination=9632328 id="9632328" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div> <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, <div class="article-title">SIFT: predicting amino acid changes that affect protein function</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">31</div>, <div class="fpage">3812</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SIFT%3A%20predicting%20amino%20acid%20changes%20that%20affect%20protein%20function&amp;author=P.C.%20Ng&amp;author=S.%20Henikoff&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3812-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div></div> <div class="year">2009</div>, <div class="article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">4</div>, <div class="fpage">1073</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20effects%20of%20coding%20non-synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm&amp;author=P.%20Kumar&amp;author=S.%20Henikoff&amp;author=P.C.%20Ng&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=1073-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2009.86" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2009.86" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2009.86"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19561590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Sims</div> <div class="given-names">G.E.</div></div> <div class="name"><div class="surname">Murphy</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Miller</div> <div class="given-names">J.R.</div></div> <div class="name"><div class="surname">Chan</div> <div class="given-names">A.P.</div></div></div> <div class="year">2012</div>, <div class="article-title">Predicting the functional effect of amino acid substitutions and indels</div>, <div class="source">Plos ONE</div> , <div class="volume">7</div>, <div class="fpage">e46688</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20functional%20effect%20of%20amino%20acid%20substitutions%20and%20indels&amp;author=Y.%20Choi&amp;author=G.E.%20Sims&amp;author=S.%20Murphy&amp;author=J.R.%20Miller&amp;author=A.P.%20Chan&amp;publication_year=2012&amp;journal=Plos%20ONE&amp;volume=7&amp;pages=e46688" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0046688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0046688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0046688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23056405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cooper</div> <div class="given-names">G.M.</div></div> <div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, <div class="article-title">Needles in stacks of needles: finding disease-causal variants in a wealth of genomic data</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">628</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Needles%20in%20stacks%20of%20needles%3A%20finding%20disease-causal%20variants%20in%20a%20wealth%20of%20genomic%20data&amp;author=G.M.%20Cooper&amp;author=J.%20Shendure&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=628-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21850043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Stenson</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Mort</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Ball</div> <div class="given-names">E.V.</div></div> <div class="name"><div class="surname">Shaw</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Phillips</div> <div class="given-names">A.D.</div></div> <div class="name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div></div> <div class="year">2014</div>, <div class="article-title">The Human Gene Mutation Database: building a comprehensive mutation repository for clinical and molecular genetics, diagnostic testing and personalized genomic medicine</div>, <div class="source">Hum. Genet.</div> , <div class="volume">133</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Human%20Gene%20Mutation%20Database%3A%20building%20a%20comprehensive%20mutation%20repository%20for%20clinical%20and%20molecular%20genetics%2C%20diagnostic%20testing%20and%20personalized%20genomic%20medicine&amp;author=P.D.%20Stenson&amp;author=M.%20Mort&amp;author=E.V.%20Ball&amp;author=K.%20Shaw&amp;author=A.D.%20Phillips&amp;author=D.N.%20Cooper&amp;publication_year=2014&amp;journal=Hum.%20Genet.&amp;volume=133&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-013-1358-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-013-1358-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-013-1358-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24077912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div> <div class="name"><div class="surname">Linton</div> <div class="given-names">L.M.</div></div>et al.  </div>, <div class="collab">International Human Genome sequencing Consortium,</div> <div class="year">2001</div>, <div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="volume">409</div>, <div class="fpage">860</div>–<div class="lpage">921</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.S.%20Lander&amp;author=L.M.%20Linton&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="collab">ENCODE-Project-Consortium</div>. <div class="year">2012</div>, <div class="article-title">An integrated encyclopedia of DNA elements in the human genome</div>, <div class="source">Nature</div> , <div class="volume">489</div>, <div class="fpage">57</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955616" class="link link-pub-id" target="_blank">PubMed</a></div> </div><div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=ENCODE-Project-Consortium.+2012,+An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome,+Nature,+489,+57–74." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hindorff</div> <div class="given-names">L.A.</div></div> <div class="name"><div class="surname">Sethupathy</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Junkins</div> <div class="given-names">H.A.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">Potential etiologic and functional implications of genome-wide association loci for human diseases and traits</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">106</div>, <div class="fpage">9362</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Potential%20etiologic%20and%20functional%20implications%20of%20genome-wide%20association%20loci%20for%20human%20diseases%20and%20traits&amp;author=L.A.%20Hindorff&amp;author=P.%20Sethupathy&amp;author=H.A.%20Junkins&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=9362-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0903103106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0903103106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0903103106"> </span></div> </div></div></div></div></div><div content-id="DSV005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ward</div> <div class="given-names">L.D.</div></div> <div class="name"><div class="surname">Kellis</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, <div class="article-title">Interpreting noncoding genetic variation in complex traits and human disease</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">1095</div>–<div class="lpage">106</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interpreting%20noncoding%20genetic%20variation%20in%20complex%20traits%20and%20human%20disease&amp;author=L.D.%20Ward&amp;author=M.%20Kellis&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=1095-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23138309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Barash</div> <div class="given-names">D.</div></div></div> <div class="year">2003</div>, <div class="article-title">Deleterious mutation prediction in the secondary structure of RNAs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">31</div>, <div class="fpage">6578</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deleterious%20mutation%20prediction%20in%20the%20secondary%20structure%20of%20RNAs&amp;author=D.%20Barash&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6578-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg872" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg872" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg872"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14602917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Shu</div> <div class="given-names">W.J.</div></div> <div class="name"><div class="surname">Bo</div> <div class="given-names">X.C.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">R.J.</div></div> <div class="name"><div class="surname">Zhao</div> <div class="given-names">D.S.</div></div> <div class="name"><div class="surname">Zheng</div> <div class="given-names">Z.Q.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.Q.</div></div></div> <div class="year">2006</div>, <div class="article-title">RDMAS: a web server for RNA deleterious mutation analysis</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">7</div>, <div class="fpage">404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RDMAS%3A%20a%20web%20server%20for%20RNA%20deleterious%20mutation%20analysis&amp;author=W.J.%20Shu&amp;author=X.C.%20Bo&amp;author=R.J.%20Liu&amp;author=D.S.%20Zhao&amp;author=Z.Q.%20Zheng&amp;author=S.Q.%20Wang&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16956394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sabarinathan</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Tafer</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Seemann</div> <div class="given-names">S.E.</div></div> <div class="name"><div class="surname">Hofacker</div> <div class="given-names">I.L.</div></div> <div class="name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div> <div class="name"><div class="surname">Gorodkin</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, <div class="article-title">RNAsnp: efficient detection of local RNA secondary structure changes induced by SNPs (vol 34, pg 546, 2013)</div>, <div class="source">Hum. Mutat.</div> , <div class="volume">34</div>, <div class="fpage">546</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAsnp%3A%20efficient%20detection%20of%20local%20RNA%20secondary%20structure%20changes%20induced%20by%20SNPs%20(vol%2034%2C%20pg%20546%2C%202013)&amp;author=R.%20Sabarinathan&amp;author=H.%20Tafer&amp;author=S.E.%20Seemann&amp;author=I.L.%20Hofacker&amp;author=P.F.%20Stadler&amp;author=J.%20Gorodkin&amp;publication_year=2013&amp;journal=Hum.%20Mutat.&amp;volume=34&amp;pages=546-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.2013.34.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.2013.34.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.2013.34.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23315997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Halvorsen</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Martin</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Broadaway</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Laederach</div> <div class="given-names">A.</div></div></div> <div class="year">2010</div>, <div class="article-title">Disease-associated mutations that alter the RNA structural ensemble</div>, <div class="source">PLoS Genet.</div> , <div class="volume">6</div>, <div class="fpage">e1001074</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disease-associated%20mutations%20that%20alter%20the%20RNA%20structural%20ensemble&amp;author=M.%20Halvorsen&amp;author=J.S.%20Martin&amp;author=S.%20Broadaway&amp;author=A.%20Laederach&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1001074" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20808897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Siepel</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Bejerano</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Pedersen</div> <div class="given-names">J.S.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">15</div>, <div class="fpage">1034</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionarily%20conserved%20elements%20in%20vertebrate%2C%20insect%2C%20worm%2C%20and%20yeast%20genomes&amp;author=A.%20Siepel&amp;author=G.%20Bejerano&amp;author=J.S.%20Pedersen&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1034-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3715005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3715005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3715005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16024819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Davydov</div> <div class="given-names">E.V.</div></div> <div class="name"><div class="surname">Goode</div> <div class="given-names">D.L.</div></div> <div class="name"><div class="surname">Sirota</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Cooper</div> <div class="given-names">G.M.</div></div> <div class="name"><div class="surname">Sidow</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Batzoglou</div> <div class="given-names">S.</div></div></div> <div class="year">2010</div>, <div class="article-title">Identifying a high fraction of the human genome to be under selective constraint using GERP plus</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">6</div>, <div class="fpage">e1001025</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20a%20high%20fraction%20of%20the%20human%20genome%20to%20be%20under%20selective%20constraint%20using%20GERP%20plus&amp;author=E.V.%20Davydov&amp;author=D.L.%20Goode&amp;author=M.%20Sirota&amp;author=G.M.%20Cooper&amp;author=A.%20Sidow&amp;author=S.%20Batzoglou&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=6&amp;pages=e1001025" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1001025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1001025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1001025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21152010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Macdonald</div> <div class="given-names">S.J.</div></div> <div class="name"><div class="surname">Long</div> <div class="given-names">A.D.</div></div></div> <div class="year">2005</div>, <div class="article-title">Prospects for identifying functional variation across the genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">102</div>, <div class="fpage">6614</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prospects%20for%20identifying%20functional%20variation%20across%20the%20genome&amp;author=S.J.%20Macdonald&amp;author=A.D.%20Long&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=6614-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0501990102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0501990102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0501990102"> </span></div> </div></div></div></div></div><div content-id="DSV005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sabeti</div> <div class="given-names">P.C.</div></div> <div class="name"><div class="surname">Schaffner</div> <div class="given-names">S.F.</div></div> <div class="name"><div class="surname">Fry</div> <div class="given-names">B.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">Positive natural selection in the human lineage</div>, <div class="source">Science</div> , <div class="volume">312</div>, <div class="fpage">1614</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positive%20natural%20selection%20in%20the%20human%20lineage&amp;author=P.C.%20Sabeti&amp;author=S.F.%20Schaffner&amp;author=B.%20Fry&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1614-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1124309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1124309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1124309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16778047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div></div> <div class="year">2005</div>, <div class="article-title">Molecular signatures of natural selection</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="volume">39</div>, <div class="fpage">197</div>–<div class="lpage">218</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20signatures%20of%20natural%20selection&amp;author=R.%20Nielsen&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=39&amp;pages=197-218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112420" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.39.073003.112420" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.39.073003.112420"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16285858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kranis</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Gheyas</div> <div class="given-names">A.A.</div></div> <div class="name"><div class="surname">Boschiero</div> <div class="given-names">C.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Development of a high density 600 K SNP genotyping array for chicken</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20high%20density%20600%20K%20SNP%20genotyping%20array%20for%20chicken&amp;author=A.%20Kranis&amp;author=A.A.%20Gheyas&amp;author=C.%20Boschiero&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23356797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">M.Y.</div></div> <div class="name"><div class="surname">Hakonarson</div> <div class="given-names">H.</div></div></div> <div class="year">2010</div>, <div class="article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">e164</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ANNOVAR%3A%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&amp;author=K.%20Wang&amp;author=M.Y.%20Li&amp;author=H.%20Hakonarson&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20601685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="article-title">UCSC Multiple alignments of 6 vertebrate genomes with chicken</div>. <div class="comment"><a class="link link-uri" href="http://hgdownload.cse.ucsc.edu/goldenPath/galGal3/multiz7way/">http://hgdownload.cse.ucsc.edu/goldenPath/galGal3/multiz7way/</a> (14 April 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=UCSC+Multiple+alignments+of+6+vertebrate+genomes+with+chicken.+http://hgdownload.cse.ucsc.edu/goldenPath/galGal3/multiz7way/+(14+April+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sabarinathan</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Tafer</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Seemann</div> <div class="given-names">S.E.</div></div> <div class="name"><div class="surname">Hofacker</div> <div class="given-names">I.L.</div></div> <div class="name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div> <div class="name"><div class="surname">Gorodkin</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, <div class="article-title">The RNAsnp web server: predicting SNP effects on local RNA secondary structure</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">W475</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RNAsnp%20web%20server%3A%20predicting%20SNP%20effects%20on%20local%20RNA%20secondary%20structure&amp;author=R.%20Sabarinathan&amp;author=H.%20Tafer&amp;author=S.E.%20Seemann&amp;author=I.L.%20Hofacker&amp;author=P.F.%20Stadler&amp;author=J.%20Gorodkin&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=W475-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23630321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div></div> <div class="year">1995</div>, <div class="article-title">Controlling the false discovery rate - a practical and powerful approach to multiple testing</div>, <div class="source">J. Roy. Stat. Soc. B Met.</div> , <div class="volume">57</div>, <div class="fpage">289</div>–<div class="lpage">300</div>.</div></div></div></div><div content-id="DSV005C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rubin</div> <div class="given-names">C.J.</div></div> <div class="name"><div class="surname">Zody</div> <div class="given-names">M.C.</div></div> <div class="name"><div class="surname">Eriksson</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">Whole-genome resequencing reveals loci under selection during chicken domestication</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">587</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20resequencing%20reveals%20loci%20under%20selection%20during%20chicken%20domestication&amp;author=C.J.%20Rubin&amp;author=M.C.%20Zody&amp;author=J.%20Eriksson&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=587-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08832"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20220755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Churchill</div> <div class="given-names">G.A.</div></div> <div class="name"><div class="surname">Doerge</div> <div class="given-names">R.W.</div></div></div> <div class="year">1994</div>, <div class="article-title">Empirical threshold values for quantitative trait mapping</div>, <div class="source">Genetics</div> , <div class="volume">138</div>, <div class="fpage">963</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Empirical%20threshold%20values%20for%20quantitative%20trait%20mapping&amp;author=G.A.%20Churchill&amp;author=R.W.%20Doerge&amp;publication_year=1994&amp;journal=Genetics&amp;volume=138&amp;pages=963-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7851788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Qanbari</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Strom</div> <div class="given-names">T.M.</div></div> <div class="name"><div class="surname">Haberer</div> <div class="given-names">G.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">A high resolution genome-wide scan for significant selective sweeps: an application to pooled sequence data in laying chickens</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e49525</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20resolution%20genome-wide%20scan%20for%20significant%20selective%20sweeps%3A%20an%20application%20to%20pooled%20sequence%20data%20in%20laying%20chickens&amp;author=S.%20Qanbari&amp;author=T.M.%20Strom&amp;author=G.%20Haberer&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e49525" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0049525" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0049525" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0049525"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23209582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Aslam</div> <div class="given-names">M.L.</div></div> <div class="name"><div class="surname">Bastiaansen</div> <div class="given-names">J.W.M.</div></div> <div class="name"><div class="surname">Elferink</div> <div class="given-names">M.G.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Whole genome SNP discovery and analysis of genetic diversity in Turkey (Meleagris gallopavo)</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">391</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20SNP%20discovery%20and%20analysis%20of%20genetic%20diversity%20in%20Turkey%20(Meleagris%20gallopavo)&amp;author=M.L.%20Aslam&amp;author=J.W.M.%20Bastiaansen&amp;author=M.G.%20Elferink&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=391" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22891612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Gardner</div> <div class="given-names">P.P.</div></div> <div class="name"><div class="surname">Fasold</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Burge</div> <div class="given-names">S.W.</div></div>et al.  </div> <div class="year">2015</div>, <div class="article-title">Conservation and losses of non-coding RNAs in avian genomes</div>, <div class="source">PLoS ONE</div> , <div class="volume">10</div>, <div class="fpage">e0121797</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20losses%20of%20non-coding%20RNAs%20in%20avian%20genomes&amp;author=P.P.%20Gardner&amp;author=M.%20Fasold&amp;author=S.W.%20Burge&amp;publication_year=2015&amp;journal=PLoS%20ONE&amp;volume=10&amp;pages=e0121797" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0121797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0121797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0121797"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25822729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dinger</div> <div class="given-names">M.E.</div></div> <div class="name"><div class="surname">Pang</div> <div class="given-names">K.C.</div></div> <div class="name"><div class="surname">Mercer</div> <div class="given-names">T.R.</div></div> <div class="name"><div class="surname">Mattick</div> <div class="given-names">J.S.</div></div></div> <div class="year">2008</div>, <div class="article-title">Differentiating protein-coding and noncoding RNA: challenges and ambiguities</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">4</div>, <div class="fpage">e1000176</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differentiating%20protein-coding%20and%20noncoding%20RNA%3A%20challenges%20and%20ambiguities&amp;author=M.E.%20Dinger&amp;author=K.C.%20Pang&amp;author=T.R.%20Mercer&amp;author=J.S.%20Mattick&amp;publication_year=2008&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=4&amp;pages=e1000176" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19043537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div> <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div></div> <div class="year">1989</div>, <div class="article-title">Mutation-Rates differ among regions of the mammalian genome</div>, <div class="source">Nature</div> , <div class="volume">337</div>, <div class="fpage">283</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation-Rates%20differ%20among%20regions%20of%20the%20mammalian%20genome&amp;author=K.H.%20Wolfe&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1989&amp;journal=Nature&amp;volume=337&amp;pages=283-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/337283a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F337283a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F337283a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2911369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.M.</div></div> <div class="name"><div class="surname">Ferec</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div></div> <div class="year">2006</div>, <div class="article-title">A systematic analysis of disease-associated variants in the 3′ regulatory regions of human protein-coding genes II: the importance of mRNA secondary structure in assessing the functionality of 3′ UTR variants</div>, <div class="source">Hum. Genet.</div> , <div class="volume">120</div>, <div class="fpage">301</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20systematic%20analysis%20of%20disease-associated%20variants%20in%20the%203%E2%80%B2%20regulatory%20regions%20of%20human%20protein-coding%20genes%20II%3A%20the%20importance%20of%20mRNA%20secondary%20structure%20in%20assessing%20the%20functionality%20of%203%E2%80%B2%20UTR%20variants&amp;author=J.M.%20Chen&amp;author=C.%20Ferec&amp;author=D.N.%20Cooper&amp;publication_year=2006&amp;journal=Hum.%20Genet.&amp;volume=120&amp;pages=301-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-006-0218-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-006-0218-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-006-0218-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16807757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sabarinathan</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Wenzel</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Novotny</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Tang</div> <div class="given-names">X.J.</div></div> <div class="name"><div class="surname">Kalari</div> <div class="given-names">K.R.</div></div> <div class="name"><div class="surname">Gorodkin</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, <div class="article-title">Transcriptome-wide analysis of UTRs in non-small cell lung cancer reveals cancer-related genes with SNV-induced changes on RNA secondary structure and miRNA target sites</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e82699</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome-wide%20analysis%20of%20UTRs%20in%20non-small%20cell%20lung%20cancer%20reveals%20cancer-related%20genes%20with%20SNV-induced%20changes%20on%20RNA%20secondary%20structure%20and%20miRNA%20target%20sites&amp;author=R.%20Sabarinathan&amp;author=A.%20Wenzel&amp;author=P.%20Novotny&amp;author=X.J.%20Tang&amp;author=K.R.%20Kalari&amp;author=J.%20Gorodkin&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e82699" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0082699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0082699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0082699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24416147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Duan</div> <div class="given-names">J.B.</div></div> <div class="name"><div class="surname">Wainwright</div> <div class="given-names">M.S.</div></div> <div class="name"><div class="surname">Comeron</div> <div class="given-names">J.M.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">Synonymous mutations in the human dopamine receptor D2 (DRD2) affect mRNA stability and synthesis of the receptor</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="volume">12</div>, <div class="fpage">205</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20mutations%20in%20the%20human%20dopamine%20receptor%20D2%20(DRD2)%20affect%20mRNA%20stability%20and%20synthesis%20of%20the%20receptor&amp;author=J.B.%20Duan&amp;author=M.S.%20Wainwright&amp;author=J.M.%20Comeron&amp;publication_year=2003&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=12&amp;pages=205-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddg055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddg055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddg055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12554675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Out</div> <div class="given-names">A.A.</div></div> <div class="name"><div class="surname">van Minderhout</div> <div class="given-names">I.J.H.M.</div></div> <div class="name"><div class="surname">Goeman</div> <div class="given-names">J.J.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">Deep sequencing to reveal new variants in pooled DNA samples</div>, <div class="source">Hum. Mutat.</div> , <div class="volume">30</div>, <div class="fpage">1703</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20to%20reveal%20new%20variants%20in%20pooled%20DNA%20samples&amp;author=A.A.%20Out&amp;author=I.J.H.M.%20van%20Minderhout&amp;author=J.J.%20Goeman&amp;publication_year=2009&amp;journal=Hum.%20Mutat.&amp;volume=30&amp;pages=1703-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v30:12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v30:12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v30:12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19842214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cutler</div> <div class="given-names">D.J.</div></div> <div class="name"><div class="surname">Jensen</div> <div class="given-names">J.D.</div></div></div> <div class="year">2010</div>, <div class="article-title">To pool, or not to pool?</div> <div class="source">Genetics</div> , <div class="volume">186</div>, <div class="fpage">41</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=To%20pool%2C%20or%20not%20to%20pool%3F&amp;author=D.J.%20Cutler&amp;author=J.D.%20Jensen&amp;publication_year=2010&amp;journal=Genetics&amp;volume=186&amp;pages=41-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.121012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.121012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.121012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20855575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div> <div class="name"><div class="surname">Levy</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Genetic variation in an individual human exome</div>, <div class="source">PLoS Genet.</div> , <div class="volume">4</div>, <div class="fpage">e1000160</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20an%20individual%20human%20exome&amp;author=P.C.%20Ng&amp;author=S.%20Levy&amp;author=J.%20Huang&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000160" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18704161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Burt</div> <div class="given-names">D.W.</div></div></div> <div class="year">2005</div>, <div class="article-title">Chicken genome: current status and future opportunities</div>, <div class="source">Genome Res.</div> , <div class="volume">15</div>, <div class="fpage">1692</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chicken%20genome%3A%20current%20status%20and%20future%20opportunities&amp;author=D.W.%20Burt&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1692-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4141805" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4141805" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4141805"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Eriksson</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Larson</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Gunnarsson</div> <div class="given-names">U.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Identification of the Yellow skin gene reveals a hybrid origin of the domestic chicken</div>, <div class="source">PLoS Genet.</div> , <div class="volume">4</div>, <div class="fpage">e1000010</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20Yellow%20skin%20gene%20reveals%20a%20hybrid%20origin%20of%20the%20domestic%20chicken&amp;author=J.%20Eriksson&amp;author=G.%20Larson&amp;author=U.%20Gunnarsson&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000010" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18454198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Elferink</div> <div class="given-names">M.G.</div></div> <div class="name"><div class="surname">Megens</div> <div class="given-names">H.J.</div></div> <div class="name"><div class="surname">Vereijken</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Hu</div> <div class="given-names">X.X.</div></div> <div class="name"><div class="surname">Crooijmans</div> <div class="given-names">R.P.M.A.</div></div> <div class="name"><div class="surname">Groenen</div> <div class="given-names">M.A.M.</div></div></div> <div class="year">2012</div>, <div class="article-title">Signatures of selection in the genomes of commercial and non-commercial chicken breeds</div>, <div class="source">PLoS ONE.</div> , <div class="volume">7</div>, <div class="fpage">e32720</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signatures%20of%20selection%20in%20the%20genomes%20of%20commercial%20and%20non-commercial%20chicken%20breeds&amp;author=M.G.%20Elferink&amp;author=H.J.%20Megens&amp;author=A.%20Vereijken&amp;author=X.X.%20Hu&amp;author=R.P.M.A.%20Crooijmans&amp;author=M.A.M.%20Groenen&amp;publication_year=2012&amp;journal=PLoS%20ONE.&amp;volume=7&amp;pages=e32720" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0032720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0032720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0032720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Shimada</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Tritos</div> <div class="given-names">N.A.</div></div> <div class="name"><div class="surname">Lowell</div> <div class="given-names">B.B.</div></div> <div class="name"><div class="surname">Flier</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Maratos-Flier</div> <div class="given-names">E.</div></div></div> <div class="year">1998</div>, <div class="article-title">Mice lacking melanin-concentrating hormone are hypophagic and lean</div>, <div class="source">Nature</div> , <div class="volume">396</div>, <div class="fpage">670</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mice%20lacking%20melanin-concentrating%20hormone%20are%20hypophagic%20and%20lean&amp;author=M.%20Shimada&amp;author=N.A.%20Tritos&amp;author=B.B.%20Lowell&amp;author=J.S.%20Flier&amp;author=E.%20Maratos-Flier&amp;publication_year=1998&amp;journal=Nature&amp;volume=396&amp;pages=670-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/25341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F25341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F25341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Flink</div> <div class="given-names">L.G.</div></div> <div class="name"><div class="surname">Allen</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Barnett</div> <div class="given-names">R.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Establishing the validity of domestication genes using DNA from ancient chickens</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">6184</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishing%20the%20validity%20of%20domestication%20genes%20using%20DNA%20from%20ancient%20chickens&amp;author=L.G.%20Flink&amp;author=R.%20Allen&amp;author=R.%20Barnett&amp;publication_year=2014&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=6184-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1308939110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1308939110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1308939110"> </span></div> </div></div></div></div></div><div content-id="DSV005C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ono</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Hoshino</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Yasuo</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Involvement of thyrotropin in photoperiodic signal transduction in mice</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">105</div>, <div class="fpage">18238</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20thyrotropin%20in%20photoperiodic%20signal%20transduction%20in%20mice&amp;author=H.%20Ono&amp;author=Y.%20Hoshino&amp;author=S.%20Yasuo&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=18238-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0808952105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0808952105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0808952105"> </span></div> </div></div></div></div></div><div content-id="DSV005C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yoshimura</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Yasuo</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Watanabe</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">Light-induced hormone conversion of T-4 to T-3 regulates photoperiodic response of gonads in birds</div>, <div class="source">Nature</div> , <div class="volume">426</div>, <div class="fpage">178</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Light-induced%20hormone%20conversion%20of%20T-4%20to%20T-3%20regulates%20photoperiodic%20response%20of%20gonads%20in%20birds&amp;author=T.%20Yoshimura&amp;author=S.%20Yasuo&amp;author=M.%20Watanabe&amp;publication_year=2003&amp;journal=Nature&amp;volume=426&amp;pages=178-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14614506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div></div> <div class="year">2010</div>, <div class="article-title">Functional intronic polymorphisms: buried treasure awaiting discovery within our genes</div>, <div class="source">Hum. Genomics</div> , <div class="volume">4</div>, <div class="fpage">284</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20intronic%20polymorphisms%3A%20buried%20treasure%20awaiting%20discovery%20within%20our%20genes&amp;author=D.N.%20Cooper&amp;publication_year=2010&amp;journal=Hum.%20Genomics&amp;volume=4&amp;pages=284-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1479-7364-4-5-284" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1479-7364-4-5-284" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1479-7364-4-5-284"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20650817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Schreiweis</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Hester</div> <div class="given-names">P.Y.</div></div> <div class="name"><div class="surname">Moody</div> <div class="given-names">D.E.</div></div></div> <div class="year">2005</div>, <div class="article-title">Identification of quantitative trait loci associated with bone traits and body weight in an F2 resource population of chickens</div>, <div class="source">Genet. Sel. Evol.</div> , <div class="volume">37</div>, <div class="fpage">677</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20quantitative%20trait%20loci%20associated%20with%20bone%20traits%20and%20body%20weight%20in%20an%20F2%20resource%20population%20of%20chickens&amp;author=M.A.%20Schreiweis&amp;author=P.Y.%20Hester&amp;author=D.E.%20Moody&amp;publication_year=2005&amp;journal=Genet.%20Sel.%20Evol.&amp;volume=37&amp;pages=677-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1297-9686-37-7-677" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1297-9686-37-7-677" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1297-9686-37-7-677"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16277974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rubin</div> <div class="given-names">C.J.</div></div> <div class="name"><div class="surname">Brandstrom</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Wright</div> <div class="given-names">D.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Quantitative trait loci for BMD and bone strength in an intercross between domestic and wildtype chickens</div>, <div class="source">J. Bone Miner. Res.</div> , <div class="volume">22</div>, <div class="fpage">375</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20for%20BMD%20and%20bone%20strength%20in%20an%20intercross%20between%20domestic%20and%20wildtype%20chickens&amp;author=C.J.%20Rubin&amp;author=H.%20Brandstrom&amp;author=D.%20Wright&amp;publication_year=2007&amp;journal=J.%20Bone%20Miner.%20Res.&amp;volume=22&amp;pages=375-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1359/jbmr.061203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1359%2Fjbmr.061203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1359%2Fjbmr.061203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17181401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Christians</div> <div class="given-names">J.K.</div></div> <div class="name"><div class="surname">Bingham</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Oliver</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Heath</div> <div class="given-names">T.T.</div></div> <div class="name"><div class="surname">Keightley</div> <div class="given-names">P.D.</div></div></div> <div class="year">2003</div>, <div class="article-title">Characterization of a QTL affecting skeletal size in mice</div>, <div class="source">Mamm. Genome.</div> , <div class="volume">14</div>, <div class="fpage">175</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20QTL%20affecting%20skeletal%20size%20in%20mice&amp;author=J.K.%20Christians&amp;author=V.%20Bingham&amp;author=F.%20Oliver&amp;author=T.T.%20Heath&amp;author=P.D.%20Keightley&amp;publication_year=2003&amp;journal=Mamm.%20Genome.&amp;volume=14&amp;pages=175-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-002-3041-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-002-3041-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-002-3041-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12647240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Roberts</div> <div class="given-names">D</div></div></div>. <div class="article-title">Protection against malaria by abnormalities in red cell surface antigens and cytoskeletal proteins</div>, In: <div class="person-group"><div class="name"><div class="surname">Schrier</div> <div class="given-names">S.L.</div></div> <div class="name"><div class="surname">Daily</div> <div class="given-names">J.</div></div></div> (eds), <div class="comment">UpToDate</div> (<div class="comment"><a class="link link-uri" href="http://www.uptodate.com/home">http://www.uptodate.com/home</a></div>); <div class="comment">(11 August 2014, date last accessed)</div>.</div></div></div></div><div content-id="DSV005C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Neel</div> <div class="given-names">J.V.</div></div></div> <div class="year">1962</div>, <div class="article-title">Diabetes mellitus - a thrifty genotype rendered detrimental by progress</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">14</div>, <div class="fpage">353</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diabetes%20mellitus%20-%20a%20thrifty%20genotype%20rendered%20detrimental%20by%20progress&amp;author=J.V.%20Neel&amp;publication_year=1962&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=14&amp;pages=353-62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13937884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chun</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Fay</div> <div class="given-names">J.C.</div></div></div> <div class="year">2011</div>, <div class="article-title">Evidence for hitchhiking of deleterious mutations within the human genome</div>, <div class="source">PLoS Genet.</div> , <div class="volume">7</div>, <div class="fpage">e1002240</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20hitchhiking%20of%20deleterious%20mutations%20within%20the%20human%20genome&amp;author=S.%20Chun&amp;author=J.C.%20Fay&amp;publication_year=2011&amp;journal=PLoS%20Genet.&amp;volume=7&amp;pages=e1002240" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21901107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="article-title">GCNT3-GeneCard</div>. <div class="comment"><a class="link link-uri" href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=GCNT3">http://www.genecards.org/cgi-bin/carddisp.pl?gene=GCNT3</a> (27 August 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=GCNT3-GeneCard.+http://www.genecards.org/cgi-bin/carddisp.pl?gene=GCNT3+(27+August+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kufe</div> <div class="given-names">D.W.</div></div></div> <div class="year">2009</div>, <div class="article-title">Mucins in cancer: function, prognosis and therapy</div>, <div class="source">Nat. Rev. Cancer</div> , <div class="volume">9</div>, <div class="fpage">874</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mucins%20in%20cancer%3A%20function%2C%20prognosis%20and%20therapy&amp;author=D.W.%20Kufe&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Cancer&amp;volume=9&amp;pages=874-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrc2761" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrc2761" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrc2761"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19935676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="article-title">PLA2R1-GeneCard</div>. <div class="comment"><a class="link link-uri" href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=PLA2R1">http://www.genecards.org/cgi-bin/carddisp.pl?gene=PLA2R1</a> (27 August 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=PLA2R1-GeneCard.+http://www.genecards.org/cgi-bin/carddisp.pl?gene=PLA2R1+(27+August+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Vidarsson</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Westergren</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Heglind</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Blomqvist</div> <div class="given-names">S.R.</div></div> <div class="name"><div class="surname">Breton</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Enerback</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, <div class="article-title">The Forkhead Transcription Factor Foxi1 is a master regulator of vacuolar H+-ATPase proton pump subunits in the inner ear, kidney and epididymis</div>, <div class="source">PLoS ONE</div> , <div class="volume">4</div>, <div class="fpage">e4471</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Forkhead%20Transcription%20Factor%20Foxi1%20is%20a%20master%20regulator%20of%20vacuolar%20H%2B-ATPase%20proton%20pump%20subunits%20in%20the%20inner%20ear%2C%20kidney%20and%20epididymis&amp;author=H.%20Vidarsson&amp;author=R.%20Westergren&amp;author=M.%20Heglind&amp;author=S.R.%20Blomqvist&amp;author=S.%20Breton&amp;author=S.%20Enerback&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e4471" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0004471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0004471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0004471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19214237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Moreno-Estrada</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Aparicio-Prat</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Sikora</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">African signatures of recent positive selection in human FOXI1</div>, <div class="source">BMC Evol. Biol.</div> , <div class="volume">10</div>, <div class="fpage">267</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=African%20signatures%20of%20recent%20positive%20selection%20in%20human%20FOXI1&amp;author=A.%20Moreno-Estrada&amp;author=E.%20Aparicio-Prat&amp;author=M.%20Sikora&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol.&amp;volume=10&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20809947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="collab">ZNF507-GeneCard</div>. <div class="comment"><a class="link link-uri" href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNF507">http://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNF507</a> (27 August 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=ZNF507-GeneCard.+http://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNF507+(27+August+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Monsuur</div> <div class="given-names">A.J.</div></div> <div class="name"><div class="surname">de Bakker</div> <div class="given-names">P.I.W.</div></div> <div class="name"><div class="surname">Alizadeh</div> <div class="given-names">B.Z.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Myosin IXB variant increases the risk of celiac disease and points toward a primary intestinal barrier defect</div>, <div class="source">Nat. Genet.</div> , <div class="volume">37</div>, <div class="fpage">1341</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Myosin%20IXB%20variant%20increases%20the%20risk%20of%20celiac%20disease%20and%20points%20toward%20a%20primary%20intestinal%20barrier%20defect&amp;author=A.J.%20Monsuur&amp;author=P.I.W.%20de%20Bakker&amp;author=B.Z.%20Alizadeh&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=1341-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1680"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16282976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Van Bodegraven</div> <div class="given-names">A.A.</div></div> <div class="name"><div class="surname">Curley</div> <div class="given-names">C.R.</div></div> <div class="name"><div class="surname">Hunt</div> <div class="given-names">K.A.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">Genetic variation in myosin IXB is associated with ulcerative colitis</div>, <div class="source">Gastroenterology</div> , <div class="volume">131</div>, <div class="fpage">1768</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20myosin%20IXB%20is%20associated%20with%20ulcerative%20colitis&amp;author=A.A.%20Van%20Bodegraven&amp;author=C.R.%20Curley&amp;author=K.A.%20Hunt&amp;publication_year=2006&amp;journal=Gastroenterology&amp;volume=131&amp;pages=1768-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1053/j.gastro.2006.09.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1053%2Fj.gastro.2006.09.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1053%2Fj.gastro.2006.09.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17087940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brionne</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Nys</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Hennequet-Antier</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Gautron</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, <div class="article-title">Hen uterine gene expression profiling during eggshell formation reveals putative proteins involved in the supply of minerals or in the shell mineralization process</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">220</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hen%20uterine%20gene%20expression%20profiling%20during%20eggshell%20formation%20reveals%20putative%20proteins%20involved%20in%20the%20supply%20of%20minerals%20or%20in%20the%20shell%20mineralization%20process&amp;author=A.%20Brionne&amp;author=Y.%20Nys&amp;author=C.%20Hennequet-Antier&amp;author=J.%20Gautron&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24649854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tuiskula-Haavisto</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Honkatukia</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Preisinger</div> <div class="given-names">R.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Quantitative trait loci affecting eggshell traits in an F(2) population</div>, <div class="source">Anim. Genet.</div> , <div class="volume">42</div>, <div class="fpage">293</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20affecting%20eggshell%20traits%20in%20an%20F(2)%20population&amp;author=M.%20Tuiskula-Haavisto&amp;author=M.%20Honkatukia&amp;author=R.%20Preisinger&amp;publication_year=2011&amp;journal=Anim.%20Genet.&amp;volume=42&amp;pages=293-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2011.42.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2011.42.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2011.42.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21054450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="collab">ADAMTS-GeneFamily</div>. <div class="comment"><a class="link link-uri" href="http://ghr.nlm.nih.gov/geneFamily/adamts">http://ghr.nlm.nih.gov/geneFamily/adamts</a> (14 April 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=ADAMTS-GeneFamily.+http://ghr.nlm.nih.gov/geneFamily/adamts+(14+April+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Podisi</div> <div class="given-names">B.K.</div></div> <div class="name"><div class="surname">Knott</div> <div class="given-names">S.A.</div></div> <div class="name"><div class="surname">Dunn</div> <div class="given-names">I.C.</div></div> <div class="name"><div class="surname">Law</div> <div class="given-names">A.S.</div></div> <div class="name"><div class="surname">Burt</div> <div class="given-names">D.W.</div></div> <div class="name"><div class="surname">Hocking</div> <div class="given-names">P.M.</div></div></div> <div class="year">2011</div>, <div class="article-title">Overlap of quantitative trait loci for early growth rate, and for body weight and age at onset of sexual maturity in chickens</div>, <div class="source">Reproduction</div> , <div class="volume">141</div>, <div class="fpage">381</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overlap%20of%20quantitative%20trait%20loci%20for%20early%20growth%20rate%2C%20and%20for%20body%20weight%20and%20age%20at%20onset%20of%20sexual%20maturity%20in%20chickens&amp;author=B.K.%20Podisi&amp;author=S.A.%20Knott&amp;author=I.C.%20Dunn&amp;author=A.S.%20Law&amp;author=D.W.%20Burt&amp;author=P.M.%20Hocking&amp;publication_year=2011&amp;journal=Reproduction&amp;volume=141&amp;pages=381-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1530/REP-10-0276" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1530%2FREP-10-0276" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1530%2FREP-10-0276"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21177954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">O.</div></div> <div class="name"><div class="surname">Odawara</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2004</div>, <div class="article-title">Genetic mapping of quantitative trait loci affecting body weight, egg character and egg production in F2 intercross chickens</div>, <div class="source">Anim. Genet.</div> , <div class="volume">35</div>, <div class="fpage">188</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20quantitative%20trait%20loci%20affecting%20body%20weight%2C%20egg%20character%20and%20egg%20production%20in%20F2%20intercross%20chickens&amp;author=O.%20Sasaki&amp;author=S.%20Odawara&amp;author=H.%20Takahashi&amp;publication_year=2004&amp;journal=Anim.%20Genet.&amp;volume=35&amp;pages=188-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2004.35.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2004.35.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2004.35.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15147389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sparks</div> <div class="given-names">N.H.C.</div></div></div> <div class="year">2011</div>, <div class="article-title">Eggshell pigments-from formation to deposition</div>, <div class="source">Avian Biol. Res.</div> , <div class="volume">4</div>, <div class="fpage">162</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Eggshell%20pigments-from%20formation%20to%20deposition&amp;author=N.H.C.%20Sparks&amp;publication_year=2011&amp;journal=Avian%20Biol.%20Res.&amp;volume=4&amp;pages=162-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3184/175815511X13228269481875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3184%2F175815511X13228269481875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3184%2F175815511X13228269481875"> </span></div> </div></div></div></div></div><div content-id="DSV005C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zheng</div> <div class="given-names">C.W.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">Z.S.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Ning</div> <div class="given-names">Z.H.</div></div></div> <div class="year">2014</div>, <div class="article-title">Quantitative expression of candidate genes affecting eggshell color</div>, <div class="source">Anim. Sci. J.</div> , <div class="volume">85</div>, <div class="fpage">506</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20expression%20of%20candidate%20genes%20affecting%20eggshell%20color&amp;author=C.W.%20Zheng&amp;author=Z.S.%20Li&amp;author=N.%20Yang&amp;author=Z.H.%20Ning&amp;publication_year=2014&amp;journal=Anim.%20Sci.%20J.&amp;volume=85&amp;pages=506-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/asj.2014.85.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fasj.2014.85.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fasj.2014.85.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24612318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV005C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="collab">FRMPD2-GeneCard</div>. <div class="comment"><a class="link link-uri" href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=FRMPD2">http://www.genecards.org/cgi-bin/carddisp.pl?gene=FRMPD2</a> (27 August 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=FRMPD2-GeneCard.+http://www.genecards.org/cgi-bin/carddisp.pl?gene=FRMPD2+(27+August+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="collab">Wikipedia</div>. <div class="comment"><a class="link link-uri" href="http://en.wikipedia.org/wiki/Phosphatidylinositol">http://en.wikipedia.org/wiki/Phosphatidylinositol</a> (27 August 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Wikipedia.+http://en.wikipedia.org/wiki/Phosphatidylinositol+(27+August+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV005C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV005C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Gibson</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, <div class="article-title">Rare and common variants: twenty arguments</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">13</div>, <div class="fpage">135</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rare%20and%20common%20variants%3A%20twenty%20arguments&amp;author=G.%20Gibson&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=13&amp;pages=135-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22251874" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=591312&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp_fig6.png?Expires=2147483647&amp;Signature=jYhAv2X367sJBecDWoF9dQXT8iqD-CMtfGm2Lcls1~mYb1JlVrytu25weToOk4pfZ-ZrkC8-1MB0mrJxHpTstaMn45ATY6mFsyCCIDNYhGDIMZDr2kRp8eI80RX2tiFTD7LHP2NtQ0DRzJjZ6abj6g43v9osr6q2FwNb-wWiNhXY19TpEMU97xa2SrwNxbs5ugwHwxvMnel2~alcyTcVZslD10NWpjTZUpVYwM~jAuXt88-qFS0EMxJFt1l47enggVfRPBIH2qfQW4KjRRzkO7OkClPvKFlHtGvnO~SBT8lsRdbr8hplOAtV0XpJnBQAG5EP8v9jR6yHjmZb7jiitA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp_fig5.png?Expires=2147483647&amp;Signature=3CnWzrCdmO0-LUltnAMALsmvWy5jhI3mIzLHKT5xx0b0DCZLjQ~Pt-aagJ4Z9zGapk9f2vHYNnuPP7geEHkJNz9e6MVIY34-MpcMwUgAxy~rOmuAeaRRUR5Zl-iIX18jvlYo4hB853c9Q0s~D1H-ux22~2Zot4x89Cpf25yUGX14yDvazuldg5E4D6q0qzXj7k42t3EmMjvvUuKYLY-OFo0rrACB2AbixRiRjLGThqbHWQl5803EeJQLmkt5WC6OR3u6kYxPyoPirP~f2D~PyYUz68uDo8-LR6mNHEWBlVlRcos-KI0chDHW~-W3IrCFKL~pUB0s6a-2g-rtXXQN-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp_fig4.png?Expires=2147483647&amp;Signature=T7y7r9lS-GpRA0dwR-Y5qj6JywKFy~kB~tCh02p9h0LVKL7Kk0BLBlERJJbVyDQTFnuUo6bH85eGo9MoezdWKXGeUV0SfqGLHoh4-kydFaA7zpc2WyCqVJ0yIMe3cPQQHh93p0LxhyLpJhkrlrktiNcGm2wUAUzqQiTW--vAQOQxvOQxP6MOVbDA~iO78s4YTkePZbdv9cXcQqM6xT-fg6ntVTXFNnA2k04fUO9QnmjiRXUWAHXHfZl0nzKrmMKkuZY4VrM52Wpt8tb2rRti3MjB9rnbiVoVR7UwOb36ATlx4LCHM5W2-r59UFsllQ1l5lJn2ellmagtppg~u6IXtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp_fig3.png?Expires=2147483647&amp;Signature=r-CFbIG8gnFBFQCx4~QXfes3pRkZ3pVCtRdnOBlm5jczzdBMvgmpuyTVHqVFjHRvTNwdBFuppgPK9krfvq9PnnBb4ll1nT0CeV31kQDk3VSqLMubH2yaf0YHQM85JcVc2ucM39QtI1-2ata2gR4WoAm0bhkhrSAFumEEtptxs2amzTyxKwILWuF4vfkpxeGDCQERxn74cfHsXtdVHN13YrOjHg9XQENXhI0JlZB--QQCu5Xg0CXcaUZfpwcW0-mzoFoRrhZyR78nqU8RKO0phxLoN0NLysrfGbTPvMlYANkzWQwFBd1IUah5z47AtKfLDecJoQtEH5M3H5SorQDuRg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp_fig2.png?Expires=2147483647&amp;Signature=MonNQln~EIzbQSeH~UWXH2EJij8LCBEuZcQCnTzzB3OlGkcGZjlBEqqODGPh-btGbRXx8oAm7kTlyTidpJUq1CcE5PuTDIq4apmxd~7WW~G50h5qq6JeHZF1CJRV5RTjKIlSxwdn52xfOW23p8joeAr3F56yAccebSK4WMRGcx0vgYm2NnswQNMsGl1lcP1iQTiilPjOta~lrD-xg--xjJsDtd8858bFWQUiT5Xhkmp5zcGhelm-NdFlnnu24RgakoD~NshX0-V~~dh~wJyc~D46fjYlurENmckZkWzgzTtxtUcdI0-MAPkMOUNcDPEmhE2gd1-53O51jtyfUGIBZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp_fig1.png?Expires=2147483647&amp;Signature=M1D0SyhEcN9lWr~DsrDzOI~lT2T71sMVOxCofoiCTlI~Vll6TNvr6qAso1lmREYRxwmESyIxxMDO7UeE-j9PwzhInJrCP71tEn9N5ylcKoSC~WFox7Ntg8NEXzmnpeFJtQl7brnl6RMpkT8pR~gNoJ~9MHgRcMtrVGQFTMskNFgx0RdejXJcySaeu10mROQfw27x4Zpy8FcpbCdmK8icmYOxLqQED3ZwARhF~wmnmp-NhgkJiwG0t49jhbiSaBkNqL-L70mP7pzt1s~n1cm-s-TR~EFusGXal3z7X~rtVtmCpv8E9~WsBTYsnlapN02hkep-i1nmfvlv5TUfhu8hKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv005/2/dsv005-dsv005supp.doc?Expires=2147483647&amp;Signature=phXCuDZPfWByjrwW5KaXkGD6zAnLV6iaIvfh1kVcckVMr1fP7n~0nq~twsYJVu7Txofq3O5TgT9jV7WLBn8vRik1oQ0GTKbadBeCcbUWW~nT0Otepdv6oNSIXWVHYofH66hYNLJ5LoMAC0AKlO5qLDB7HneXhtchClJ7cTcRFegCAcBFJYv~WzMl7DhPXbQA3NoqLwXcFmc7gPrAtaDIfbDAw5Kvy2wlIpbZ4Rr8bfrnbJ~DNF3Qe392MXkhHWXtV7ElwfkVtwu-ouIrRRyR3au9waFRboekDa-Sa42hOKKcfC--M1jGpCEzYGzdSRBCUTIxnNMPDfrNfa07smdiOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">945</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="591312"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f3%2f205%2f591312">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000356658900003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F3%2F205%2F591312" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/591312'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Functional classification of 15 million SNPs detected from diverse chicken populations" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f205%2f591312">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

