

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/5/487/3778238","@type":"ScholarlyArticle","name":"A genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture","datePublished":"2017-04-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2017-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx019","keywords":["genome evolution","mtDNA","<italic>numt</italic>","polymorphism","<italic>Sus scrofa</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/5/487/3778238","author":[{"name":"Schiavo, Giuseppina","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Hoffmann, Orsolya Ivett","affiliation":"2 Ruminant Genome Biology Group, NARIC, Agricultural Biotechnology Center, H-2100 Gödöllő, Hungary","@type":"Person"},{"name":"Ribani, Anisa","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Utzeri, Valerio Joe","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Ghionda, Marco Ciro","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Bertolini, Francesca","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Geraci, Claudia","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Bovo, Samuele","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"},{"name":"Fontanesi, Luca","affiliation":"1 Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy","@type":"Person"}],"description":"Abstract.  Nuclear DNA sequences of mitochondrial origin (numts) are derived by insertion of mitochondrial DNA (mtDNA), into the nuclear genome. In this study, ","pageStart":"487","pageEnd":"498","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture" />
    <meta property="og:description" content="Abstract.  Nuclear DNA sequences of mitochondrial origin (numts) are derived by insertion of mitochondrial DNA (mtDNA), into the nuclear genome. In this study, " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/5/487/3778238" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-04-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Schiavo, Giuseppina" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Hoffmann, Orsolya Ivett" /><meta name="citation_author_institution" content="Ruminant Genome Biology Group, NARIC, Agricultural Biotechnology Center, H-2100 Gödöllő, Hungary" /><meta name="citation_author" content="Ribani, Anisa" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Utzeri, Valerio Joe" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Ghionda, Marco Ciro" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Bertolini, Francesca" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Geraci, Claudia" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Bovo, Samuele" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_author" content="Fontanesi, Luca" /><meta name="citation_author_institution" content="Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy" /><meta name="citation_title" content="A genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture" /><meta name="citation_firstpage" content="487" /><meta name="citation_lastpage" content="498" /><meta name="citation_doi" content="10.1093/dnares/dsx019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2017/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Mobile elements: drivers of genome evolution;  citation_journal_title=Science;  citation_year=2004;  citation_volume=303;  citation_pages=1626-32; " /><meta name="citation_reference" content="citation_title=Numt, a recent transfer and tandem amplification of mitochondrial DNA to the nuclear genome of the domestic cat;  citation_journal_title=J. Mol. Evol;  citation_year=1994;  citation_volume=39;  citation_pages=174-90; " /><meta name="citation_reference" content="citation_title=Sorting out tigers (Panthera tigris): mitochondrial sequences, nuclear inserts, systematics, and conservation genetics;  citation_journal_title=Anim. Conserv;  citation_year=1998;  citation_volume=1;  citation_pages=139-50; " /><meta name="citation_reference" content="citation_title=Molecular poltergeists: mitochondrial DNA copies (numts) in sequenced nuclear genomes;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1000834" /><meta name="citation_reference" content="citation_title=Distinguishing gorilla mitochondrial sequences from nuclear integrations and PCR recombinants: guidelines for their diagnosis in complex sequence databases;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2007;  citation_volume=43;  citation_pages=553-66; " /><meta name="citation_reference" content="citation_title=Ancient mtDNA sequences in the human nuclear genome: a potential source of errors in identifying pathogenic mutations;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1997;  citation_volume=94;  citation_pages=14900-5; " /><meta name="citation_reference" content="citation_title=Mitochondrial pseudogenes: evolution’s misplaced witnesses;  citation_journal_title=Trends Ecol. Evol;  citation_year=2001;  citation_volume=16;  citation_pages=314-21; " /><meta name="citation_reference" content="citation_title=Origin, evolution and genetic effects of nuclear insertions of organelle DNA;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=655-63; " /><meta name="citation_reference" content="citation_title=Mitochondrial DNA repairs double-strand breaks in yeast chromosomes;  citation_journal_title=Nature;  citation_year=1999;  citation_volume=402;  citation_pages=96-100; " /><meta name="citation_reference" content="citation_title=A comparative analysis of numt evolution in human and chimpanzee;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=13-8; " /><meta name="citation_reference" content="citation_title=Low number of mitochondrial pseudogenes in the chicken (Gallus gallus) nuclear genome: implications for molecular inference of population history and phylogenetics;  citation_journal_title=BMC Evol. Biol;  citation_year=2004;  citation_volume=4;  citation_pages=17" /><meta name="citation_reference" content="citation_title=Mitochondrial DNA insertions in the nuclear horse genome;  citation_journal_title=Anim. Genet;  citation_year=2010;  citation_volume=41(Suppl 2);  citation_pages=176-85; " /><meta name="citation_reference" content="citation_title=Analyses of pig genomes provide insight into porcine demography and evolution;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=393-8; " /><meta name="citation_reference" content="citation_title=Genome sequencing reveals fine scale diversification and reticulation history during speciation in Sus;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R107" /><meta name="citation_reference" content="citation_title=The origin of the domestic pig: independent domestication and subsequent introgression;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1785-91; " /><meta name="citation_reference" content="citation_title=Worldwide phylogeography of wild boar reveals multiple centers of pig domestication;  citation_journal_title=Science;  citation_year=2005;  citation_volume=307;  citation_pages=1618-21; " /><meta name="citation_reference" content="citation_title=Genomic analysis reveals selection for Asian genes in European pigs following human-mediated introgression;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=4392" /><meta name="citation_reference" content="citation_title=Adaptive seeds tame genomic sequence comparison;  citation_journal_title=Genome Res;  citation_year=2011;  citation_volume=21;  citation_pages=487-93; " /><meta name="citation_reference" content="citation_title=Parameters for accurate genome alignment;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=80" /><meta name="citation_reference" content="citation_title=Mammalian NUMT insertion is non-random;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=9073-88; " /><meta name="citation_reference" content="citation_title=The Evolution of Suidae;  citation_journal_title=Annu. Rev. Anim. Biosci;  citation_year=2016;  citation_volume=4;  citation_pages=61-85; " /><meta name="citation_reference" content="citation_title=MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=1870-4; " /><meta name="citation_reference" content="citation_title=The genomic landscape of polymorphic human nuclear mitochondrial insertions;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=12640-9; " /><meta name="citation_reference" content="citation_publisher=CABI, Cambridge;  citation_title=The Genetics of the Pig;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Mobile elements and mammalian genome evolution;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2003;  citation_volume=13;  citation_pages=651-8; " /><meta name="citation_reference" content="citation_title=Molecular evidence from retroposons that whales form a clade within even-toed ungulates;  citation_journal_title=Nature;  citation_year=1997;  citation_volume=388;  citation_pages=666-70; " /><meta name="citation_reference" content="citation_title=SINE insertions: powerful tools for molecular systematics;  citation_journal_title=Bioessays;  citation_year=2000;  citation_volume=22;  citation_pages=148-60; " /><meta name="citation_reference" content="citation_title=Revealing the history of sheep domestication using retrovirus integrations;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=532-6; " /><meta name="citation_reference" content="citation_title=Alu repeats and human disease;  citation_journal_title=Mol. Genet. Metab;  citation_year=1999;  citation_volume=67;  citation_pages=183-93; " /><meta name="citation_reference" content="citation_title=Retroviral elements and their hosts: insertional mutagenesis in the mouse germ line;  citation_journal_title=PLoS Genet;  citation_year=2006;  citation_volume=2;  citation_pages=e2" /><meta name="citation_reference" content="citation_title=An intronic insertion in KPL2 results in aberrant splicing and causes the immotile short-tail sperm defect in the pig;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2006;  citation_volume=103;  citation_pages=5006-11; " /><meta name="citation_reference" content="citation_title=Phylogenetic relationships of Asian and European pig breeds determined by mitochondrial DNA D-loop sequence polymorphism;  citation_journal_title=Anim. Genet;  citation_year=2002;  citation_volume=33;  citation_pages=19-25; " /><meta name="citation_reference" content="citation_title=Mitochondrial genomes of domestic animals need scrutiny;  citation_journal_title=Mol. Ecol;  citation_year=2014;  citation_volume=23;  citation_pages=5393-7; " /><meta name="citation_reference" content="citation_title=What fraction of the human genome is functional?;  citation_journal_title=Genome Res;  citation_year=2011;  citation_volume=21;  citation_pages=1769-76; " /><meta name="citation_reference" content="citation_title=Characterization of two novel splice site mutations in human factor VII gene causing severe plasma factor VII deficiency and bleeding diathesis;  citation_journal_title=Br. J. Haematol;  citation_year=2002;  citation_volume=117;  citation_pages=168-71; " /><meta name="citation_reference" content="citation_title=Human genetic disease caused by de novo mitochondrial-nuclear DNA transfer;  citation_journal_title=Hum. Genet;  citation_year=2003;  citation_volume=112;  citation_pages=303-9; " /><meta name="citation_reference" content="citation_title=Transfer of a mitochondrial DNA fragment to MCOLN1 causes an inherited case of mucolipidosis IV;  citation_journal_title=Hum. Mutat;  citation_year=2004;  citation_volume=24;  citation_pages=460-5; " /><meta name="citation_reference" content="citation_title=Meta-analysis of gross insertions causing human genetic disease: novel mutational mechanisms and the role of replication slippage;  citation_journal_title=Hum. Mutat;  citation_year=2005;  citation_volume=25;  citation_pages=207-21; " /><meta name="citation_reference" content="citation_title=Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing;  citation_journal_title=Nat. Genet;  citation_year=2015;  citation_volume=47;  citation_pages=217-25; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/5/487/22893027/dsx019.pdf" /><meta name="description" content="Abstract.  Nuclear DNA sequences of mitochondrial origin (numts) are derived by insertion of mitochondrial DNA (mtDNA), into the nuclear genome. In this study, " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/5/487/3778238" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/5/487/3778238" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/5/487/3778238");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx019';
    </script>
<title>
	genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3778238" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeBjPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hpYXZvLCBHaXVzZXBwaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyYWwgYW5kIEZvb2QgU2NpZW5jZXMsIERpdmlzaW9uIG9mIEFuaW1hbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBCb2xvZ25hLCA0MDEyNyBCb2xvZ25hLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhvZmZtYW5uLCBPcnNvbHlhIEl2ZXR0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUnVtaW5hbnQgR2Vub21lIEJpb2xvZ3kgR3JvdXAsIE5BUklDLCBBZ3JpY3VsdHVyYWwgQmlvdGVjaG5vbG9neSBDZW50ZXIsIEgtMjEwMOKAiUfDtmTDtmxsxZEsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSaWJhbmksIEFuaXNhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyYWwgYW5kIEZvb2QgU2NpZW5jZXMsIERpdmlzaW9uIG9mIEFuaW1hbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBCb2xvZ25hLCA0MDEyNyBCb2xvZ25hLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlV0emVyaSwgVmFsZXJpbyBKb2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCBhbmQgRm9vZCBTY2llbmNlcywgRGl2aXNpb24gb2YgQW5pbWFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIEJvbG9nbmEsIDQwMTI3IEJvbG9nbmEsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2hpb25kYSwgTWFyY28gQ2lybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsIGFuZCBGb29kIFNjaWVuY2VzLCBEaXZpc2lvbiBvZiBBbmltYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgQm9sb2duYSwgNDAxMjcgQm9sb2duYSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCZXJ0b2xpbmksIEZyYW5jZXNjYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsIGFuZCBGb29kIFNjaWVuY2VzLCBEaXZpc2lvbiBvZiBBbmltYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgQm9sb2duYSwgNDAxMjcgQm9sb2duYSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHZXJhY2ksIENsYXVkaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCBhbmQgRm9vZCBTY2llbmNlcywgRGl2aXNpb24gb2YgQW5pbWFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIEJvbG9nbmEsIDQwMTI3IEJvbG9nbmEsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm92bywgU2FtdWVsZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsIGFuZCBGb29kIFNjaWVuY2VzLCBEaXZpc2lvbiBvZiBBbmltYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgQm9sb2duYSwgNDAxMjcgQm9sb2duYSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGb250YW5lc2ksIEx1Y2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCBhbmQgRm9vZCBTY2llbmNlcywgRGl2aXNpb24gb2YgQW5pbWFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIEJvbG9nbmEsIDQwMTI3IEJvbG9nbmEsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBIGdlbm9taWMgbGFuZHNjYXBlIG9mIG1pdG9jaG9uZHJpYWwgRE5BIGluc2VydGlvbnMgaW4gdGhlIHBpZyBudWNsZWFyIGdlbm9tZSBwcm92aWRlcyBldm9sdXRpb25hcnkgc2lnbmF0dXJlcyBvZiBpbnRlcnNwZWNpZXMgYWRtaXh0dXJlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0OTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vYmlsZSBlbGVtZW50czogZHJpdmVycyBvZiBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMwMzsgIGNpdGF0aW9uX3BhZ2VzPTE2MjYtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51bXQsIGEgcmVjZW50IHRyYW5zZmVyIGFuZCB0YW5kZW0gYW1wbGlmaWNhdGlvbiBvZiBtaXRvY2hvbmRyaWFsIEROQSB0byB0aGUgbnVjbGVhciBnZW5vbWUgb2YgdGhlIGRvbWVzdGljIGNhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTc0LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb3J0aW5nIG91dCB0aWdlcnMgKFBhbnRoZXJhIHRpZ3Jpcyk6IG1pdG9jaG9uZHJpYWwgc2VxdWVuY2VzLCBudWNsZWFyIGluc2VydHMsIHN5c3RlbWF0aWNzLCBhbmQgY29uc2VydmF0aW9uIGdlbmV0aWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbmltLiBDb25zZXJ2OyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMzktNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBwb2x0ZXJnZWlzdHM6IG1pdG9jaG9uZHJpYWwgRE5BIGNvcGllcyAobnVtdHMpIGluIHNlcXVlbmNlZCBudWNsZWFyIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwODM0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluZ3Vpc2hpbmcgZ29yaWxsYSBtaXRvY2hvbmRyaWFsIHNlcXVlbmNlcyBmcm9tIG51Y2xlYXIgaW50ZWdyYXRpb25zIGFuZCBQQ1IgcmVjb21iaW5hbnRzOiBndWlkZWxpbmVzIGZvciB0aGVpciBkaWFnbm9zaXMgaW4gY29tcGxleCBzZXF1ZW5jZSBkYXRhYmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGh5bG9nZW5ldC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTU1My02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5jaWVudCBtdEROQSBzZXF1ZW5jZXMgaW4gdGhlIGh1bWFuIG51Y2xlYXIgZ2Vub21lOiBhIHBvdGVudGlhbCBzb3VyY2Ugb2YgZXJyb3JzIGluIGlkZW50aWZ5aW5nIHBhdGhvZ2VuaWMgbXV0YXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDkwMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIHBzZXVkb2dlbmVzOiBldm9sdXRpb27igJlzIG1pc3BsYWNlZCB3aXRuZXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBFY29sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MzE0LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4sIGV2b2x1dGlvbiBhbmQgZ2VuZXRpYyBlZmZlY3RzIG9mIG51Y2xlYXIgaW5zZXJ0aW9ucyBvZiBvcmdhbmVsbGUgRE5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02NTUtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgRE5BIHJlcGFpcnMgZG91YmxlLXN0cmFuZCBicmVha3MgaW4geWVhc3QgY2hyb21vc29tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00MDI7ICBjaXRhdGlvbl9wYWdlcz05Ni0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgbnVtdCBldm9sdXRpb24gaW4gaHVtYW4gYW5kIGNoaW1wYW56ZWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdyBudW1iZXIgb2YgbWl0b2Nob25kcmlhbCBwc2V1ZG9nZW5lcyBpbiB0aGUgY2hpY2tlbiAoR2FsbHVzIGdhbGx1cykgbnVjbGVhciBnZW5vbWU6IGltcGxpY2F0aW9ucyBmb3IgbW9sZWN1bGFyIGluZmVyZW5jZSBvZiBwb3B1bGF0aW9uIGhpc3RvcnkgYW5kIHBoeWxvZ2VuZXRpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgRE5BIGluc2VydGlvbnMgaW4gdGhlIG51Y2xlYXIgaG9yc2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbmltLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MShTdXBwbCAyKTsgIGNpdGF0aW9uX3BhZ2VzPTE3Ni04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzZXMgb2YgcGlnIGdlbm9tZXMgcHJvdmlkZSBpbnNpZ2h0IGludG8gcG9yY2luZSBkZW1vZ3JhcGh5IGFuZCBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTE7ICBjaXRhdGlvbl9wYWdlcz0zOTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgcmV2ZWFscyBmaW5lIHNjYWxlIGRpdmVyc2lmaWNhdGlvbiBhbmQgcmV0aWN1bGF0aW9uIGhpc3RvcnkgZHVyaW5nIHNwZWNpYXRpb24gaW4gU3VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVIxMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIG9mIHRoZSBkb21lc3RpYyBwaWc6IGluZGVwZW5kZW50IGRvbWVzdGljYXRpb24gYW5kIHN1YnNlcXVlbnQgaW50cm9ncmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MTc4NS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29ybGR3aWRlIHBoeWxvZ2VvZ3JhcGh5IG9mIHdpbGQgYm9hciByZXZlYWxzIG11bHRpcGxlIGNlbnRlcnMgb2YgcGlnIGRvbWVzdGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA3OyAgY2l0YXRpb25fcGFnZXM9MTYxOC0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyByZXZlYWxzIHNlbGVjdGlvbiBmb3IgQXNpYW4gZ2VuZXMgaW4gRXVyb3BlYW4gcGlncyBmb2xsb3dpbmcgaHVtYW4tbWVkaWF0ZWQgaW50cm9ncmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQzOTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGl2ZSBzZWVkcyB0YW1lIGdlbm9taWMgc2VxdWVuY2UgY29tcGFyaXNvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFyYW1ldGVycyBmb3IgYWNjdXJhdGUgZ2Vub21lIGFsaWdubWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYW1tYWxpYW4gTlVNVCBpbnNlcnRpb24gaXMgbm9uLXJhbmRvbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz05MDczLTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRXZvbHV0aW9uIG9mIFN1aWRhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBBbmltLiBCaW9zY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTYxLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNzogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB2ZXJzaW9uIDcuMCBmb3IgYmlnZ2VyIGRhdGFzZXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xODcwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWljIGxhbmRzY2FwZSBvZiBwb2x5bW9ycGhpYyBodW1hbiBudWNsZWFyIG1pdG9jaG9uZHJpYWwgaW5zZXJ0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMjY0MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q0FCSSwgQ2FtYnJpZGdlOyAgY2l0YXRpb25fdGl0bGU9VGhlIEdlbmV0aWNzIG9mIHRoZSBQaWc7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vYmlsZSBlbGVtZW50cyBhbmQgbWFtbWFsaWFuIGdlbm9tZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIEdlbmV0LiBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGV2aWRlbmNlIGZyb20gcmV0cm9wb3NvbnMgdGhhdCB3aGFsZXMgZm9ybSBhIGNsYWRlIHdpdGhpbiBldmVuLXRvZWQgdW5ndWxhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mzg4OyAgY2l0YXRpb25fcGFnZXM9NjY2LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TSU5FIGluc2VydGlvbnM6IHBvd2VyZnVsIHRvb2xzIGZvciBtb2xlY3VsYXIgc3lzdGVtYXRpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE0OC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV2ZWFsaW5nIHRoZSBoaXN0b3J5IG9mIHNoZWVwIGRvbWVzdGljYXRpb24gdXNpbmcgcmV0cm92aXJ1cyBpbnRlZ3JhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9NTMyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdSByZXBlYXRzIGFuZCBodW1hbiBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbmV0LiBNZXRhYjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTE4My05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV0cm92aXJhbCBlbGVtZW50cyBhbmQgdGhlaXIgaG9zdHM6IGluc2VydGlvbmFsIG11dGFnZW5lc2lzIGluIHRoZSBtb3VzZSBnZXJtIGxpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50cm9uaWMgaW5zZXJ0aW9uIGluIEtQTDIgcmVzdWx0cyBpbiBhYmVycmFudCBzcGxpY2luZyBhbmQgY2F1c2VzIHRoZSBpbW1vdGlsZSBzaG9ydC10YWlsIHNwZXJtIGRlZmVjdCBpbiB0aGUgcGlnOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NTAwNi0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHMgb2YgQXNpYW4gYW5kIEV1cm9wZWFuIHBpZyBicmVlZHMgZGV0ZXJtaW5lZCBieSBtaXRvY2hvbmRyaWFsIEROQSBELWxvb3Agc2VxdWVuY2UgcG9seW1vcnBoaXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbmltLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTE5LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIGdlbm9tZXMgb2YgZG9tZXN0aWMgYW5pbWFscyBuZWVkIHNjcnV0aW55OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz01MzkzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdoYXQgZnJhY3Rpb24gb2YgdGhlIGh1bWFuIGdlbm9tZSBpcyBmdW5jdGlvbmFsPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE3NjktNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdHdvIG5vdmVsIHNwbGljZSBzaXRlIG11dGF0aW9ucyBpbiBodW1hbiBmYWN0b3IgVklJIGdlbmUgY2F1c2luZyBzZXZlcmUgcGxhc21hIGZhY3RvciBWSUkgZGVmaWNpZW5jeSBhbmQgYmxlZWRpbmcgZGlhdGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Cci4gSi4gSGFlbWF0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9MTY4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBnZW5ldGljIGRpc2Vhc2UgY2F1c2VkIGJ5IGRlIG5vdm8gbWl0b2Nob25kcmlhbC1udWNsZWFyIEROQSB0cmFuc2ZlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SHVtLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0zMDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXIgb2YgYSBtaXRvY2hvbmRyaWFsIEROQSBmcmFnbWVudCB0byBNQ09MTjEgY2F1c2VzIGFuIGluaGVyaXRlZCBjYXNlIG9mIG11Y29saXBpZG9zaXMgSVY7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTXV0YXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz00NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YS1hbmFseXNpcyBvZiBncm9zcyBpbnNlcnRpb25zIGNhdXNpbmcgaHVtYW4gZ2VuZXRpYyBkaXNlYXNlOiBub3ZlbCBtdXRhdGlvbmFsIG1lY2hhbmlzbXMgYW5kIHRoZSByb2xlIG9mIHJlcGxpY2F0aW9uIHNsaXBwYWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIE11dGF0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGF0aW9uIGFuZCBwb3NzaWJsZSBhbmNpZW50IGludGVyc3BlY2llcyBpbnRyb2dyZXNzaW9uIGluIHBpZ3MgaWRlbnRpZmllZCBieSB3aG9sZS1nZW5vbWUgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTIxNy0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzUvNDg3LzIyODkzMDI3L2RzeDAxOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTnVjbGVhciBETkEgc2VxdWVuY2VzIG9mIG1pdG9jaG9uZHJpYWwgb3JpZ2luIChudW10cykgYXJlIGRlcml2ZWQgYnkgaW5zZXJ0aW9uIG9mIG1pdG9jaG9uZHJpYWwgRE5BIChtdEROQSksIGludG8gdGhlIG51Y2xlYXIgZ2Vub21lLiBJbiB0aGlzIHN0dWR5LCAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM3NzgyMzgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkeYK5Jlxx4XxM4jT65d8/cj7kgdU=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f487%2f3778238" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f487%2f3778238" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=99990" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&amp;Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/5">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106356178">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356178">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356180">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356180">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356186">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356186">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356201">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356201">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356225">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356225">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356232">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356232">Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356234">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356234">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356242">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356242">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/5/473/3769337">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/5/499/3835854">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Giuseppina Schiavo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Giuseppina+Schiavo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schiavo G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schiavo G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Giuseppina Schiavo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Orsolya Ivett Hoffmann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Ruminant Genome Biology Group, NARIC, Agricultural Biotechnology Center, H-2100 Gödöllő, Hungary</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Orsolya+Ivett+Hoffmann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hoffmann O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hoffmann O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Orsolya Ivett Hoffmann</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anisa Ribani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anisa+Ribani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ribani A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ribani A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anisa Ribani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Valerio Joe Utzeri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Valerio+Joe+Utzeri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Utzeri V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Utzeri V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Valerio Joe Utzeri</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marco Ciro Ghionda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marco+Ciro+Ghionda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ghionda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ghionda M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marco Ciro Ghionda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Francesca Bertolini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Francesca+Bertolini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bertolini F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bertolini F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Francesca Bertolini</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Claudia Geraci</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Claudia+Geraci">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Geraci C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Geraci C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Claudia Geraci</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Samuele Bovo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Samuele+Bovo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bovo S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bovo S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Samuele Bovo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Luca Fontanesi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural and Food Sciences, Division of Animal Sciences, University of Bologna, 40127 Bologna, Italy</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx019-cor1" data-open="dsx019-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. + 39 051 2096535. Fax. +39 051 2096355. E-mail: <a href="mailto:luca.fontanesi@unibo.it" target="_blank">luca.fontanesi@unibo.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Luca+Fontanesi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fontanesi L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fontanesi L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Luca Fontanesi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx019-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx019-FM2"><div class="fn"><div class="label -label">*</div><div class="footnote-content">Edited by Dr. Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 487–498, <a href='https://doi.org/10.1093/dnares/dsx019'>https://doi.org/10.1093/dnares/dsx019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 April 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 October 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 April 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3778238" href="/dnaresearch/article-pdf/24/5/487/22893027/dsx019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Giuseppina Schiavo, Orsolya Ivett Hoffmann, Anisa Ribani, Valerio Joe Utzeri, Marco Ciro Ghionda, Francesca Bertolini, Claudia Geraci, Samuele Bovo, Luca Fontanesi;  A genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture, <em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 487–498, <a href='https://doi.org/10.1093/dnares/dsx019'>https://doi.org/10.1093/dnares/dsx019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778238?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20genomic%20landscape%20of%20mitochondrial%20DNA%20insertions%20in%20the%20pig%20nuclear%20genome%20provides%20evolutionary%20signatures%20of%20interspecies%20admixture&amp;publicationDate=2017-04-27&amp;volumeNum=24&amp;issueNum=5&amp;author=Schiavo%2C%20Giuseppina%3B%20Hoffmann%2C%20Orsolya%20Ivett&amp;startPage=487&amp;endPage=498&amp;contentId=10.1093%2Fdnares%2Fdsx019&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f487%2f3778238" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106356178 id="106356178" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Nuclear DNA sequences of mitochondrial origin (<em>numts</em>) are derived by insertion of mitochondrial DNA (mtDNA), into the nuclear genome. In this study, we provide, for the first time, a genome picture of <em>numts</em> inserted in the pig nuclear genome. The <em>Sus scrofa</em> reference nuclear genome (Sscrofa10.2) was aligned with circularized and consensus mtDNA sequences using LAST software. A total of 430 <em>numt</em> sequences that may represent 246 different <em>numt</em> integration events (57 <em>numt</em> regions determined by at least two <em>numt</em> sequences and 189 singletons) were identified, covering about 0.0078% of the nuclear genome. <em>Numt</em> integration events were correlated (0.99) to the chromosome length. The longest <em>numt</em> sequence (about 11 kbp) was located on SSC2. Six <em>numts</em> were sequenced and PCR amplified in pigs of European commercial and local pig breeds, of the Chinese Meishan breed and in European wild boars. Three of them were polymorphic for the presence or absence of the insertion. Surprisingly, the estimated age of insertion of two of the three polymorphic <em>numts</em> was more ancient than that of the speciation time of the <em>Sus scrofa</em>, supporting that these polymorphic sites were originated from interspecies admixture that contributed to shape the pig genome.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome evolution&quot;">genome evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mtDNA">mtDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="numt"><em>numt</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="polymorphism">polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Sus scrofa&quot;"><em>Sus scrofa</em></a></div>                            <h2 scrollto-destination=106356180 id="106356180" class="section-title" >1. Introduction</h2>
<p>Eukaryotic genomes have been shaped by the insertion of new sequences of different origin into nuclear DNA, largely influencing their architecture and evolution.<a reveal-id="dsx019-B1" data-open="dsx019-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> A particular source of new sequences is provided by the horizontal transfer of mitochondrial DNA (mtDNA) fragments (from both coding and non-coding regions) into the nuclear genome, producing nuclear DNA sequences of mitochondrial origin (<em>numts</em>).<a reveal-id="dsx019-B2" data-open="dsx019-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><em>Numts</em>, which have been frequently discovered inadvertently in the search for <em>bona fide</em> mtDNA<a reveal-id="dsx019-B3" data-open="dsx019-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a>, represent sequence fossils present in the nuclear genome of many eukaryotes, constituting about 0.001–0.1% of its DNA.<a reveal-id="dsx019-B4" data-open="dsx019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> If <em>numts</em> are not recognized in actual mtDNA sequence-based studies, they can compromise result interpretation in detecting heteroplasmy<a reveal-id="dsx019-B5" data-open="dsx019-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> or mtDNA pathogenic variants<a reveal-id="dsx019-B6" data-open="dsx019-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> or leading to wrong phylogenetic reconstructions.<a reveal-id="dsx019-B7" data-open="dsx019-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p>The transfer of these genetic materials is however an ongoing, but rare, evolutionary process.<a reveal-id="dsx019-B8" data-open="dsx019-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The exact mechanisms by which mtDNA sequences are integrated into the nuclear genome are not completely understood. The most accepted hypothesis suggests that in presence of mutagenic agents or stress conditions, mtDNA fragments can escape from the mitochondrions and their integration into the nuclear genome could occur during the repair of DNA double-strand breaks.<a reveal-id="dsx019-B4" data-open="dsx019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx019-B9" data-open="dsx019-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><em>Numts</em>, similarly to other inserted elements, have been used as evolutionary markers by analysing orthologous regions in related species and considering their presence or absence as indications of species radiation times.<a reveal-id="dsx019-B5" data-open="dsx019-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx019-B10" data-open="dsx019-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>Several livestock genomes have been recently analysed to evaluate the presence and distribution of <em>numts</em>. Most of these studies have been based only on genome mining using BLAST search of mtDNA sequence in the available genome versions. Pereira and Baker<a reveal-id="dsx019-B11" data-open="dsx019-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> investigated a draft sequence of the chicken genome showing a relatively low number of <em>numts</em> in the avians compared with mammals. Liu and Zhao<a reveal-id="dsx019-B5" data-open="dsx019-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> reported a preliminary analysis of the cattle genome using a partial unassembled sequence version. Hazkani-Covo <em>et al.</em><a reveal-id="dsx019-B4" data-open="dsx019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> interrogated all genomes publicly available at that time (i.e. 2010), including cattle, chicken, rabbit and horse genomes. The horse genome was also interrogated by Nergadze <em>et al.</em><a reveal-id="dsx019-B12" data-open="dsx019-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> who reported polymorphic <em>numts</em> segregating within species.</p><p>The <em>Sus scrofa</em> genome has been recently published and the current released version (Sscrofa10.2), available since August 2011, from the Swine Genome Sequencing Consortium (SGSC), accounts for about 3 billions of base pairs.<a reveal-id="dsx019-B13" data-open="dsx019-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The genome of <em>S</em><em>.</em><em>scrofa</em> contains signature of inter-species gene flow that indicates reticulate inter taxa crossbreeding events in the constitution of <em>Sus</em> species, involving species of the Island Southeast Asia (ISEA) biodiversity hotspots.<a reveal-id="dsx019-B14" data-open="dsx019-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The domestication processes of <em>S</em><em>.</em><em>scrofa</em> occurred independently from European and Asian wild boars whose common ancestors diverged about 1 million of years before present (MYBP).<a reveal-id="dsx019-B15" data-open="dsx019-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx019-B16" data-open="dsx019-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Then, modern European pig breeds and lines were influenced by the introduction in the XVIII–XIX centuries of Asian blood into European populations.<a reveal-id="dsx019-B16" data-open="dsx019-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx019-B17" data-open="dsx019-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p><p>In this work, we identified and analysed <em>numts</em> in the pig reference genome which, to our best knowledge, is the first comprehensive study in this species that considered mtDNA insertions in the nuclear genome. Phylogenetic analyses provided estimations of <em>numt</em> insertional events during the evolutionary lineage of the Suinae subfamily. We also revealed that polymorphic <em>numts</em> (i.e. presence/absence of insertions) segregate in different pig breeds and can provide interesting signatures to capture reticulate evolutionary events between <em>Sus</em> species and information useful to understand the domestication processes of the pig and the constitution of pig breeds.</p>                            <h2 scrollto-destination=106356186 id="106356186" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=106356187 id="106356187" class="section-title" >2.1. Search for <em>numts</em> within the <em>S</em><em>.</em><em>scrofa</em> nuclear genome</h3>
                            <h4 scrollto-destination=106356188 id="106356188" class="section-title" >2.1.1. Data sources</h4>
<p>The unmasked reference pig genome was retrieved from Ensembl (Sscrofa10.2, GCA_000003025.4). It includes the porcine reference mtDNA sequence in linear form. Other full-length <em>S</em><em>.</em><em>scrofa</em> mtDNA sequences were retrieved from GenBank: these sequences were from different European originated pig breeds (Duroc: the reference mtDNA sequence; Hampshire, AY574046.1; Large White, KC250275.1; Mangalitsa, KJ746666.1) and European wild boar (FJ236998.1) as well as from different Asian originated pig breeds (Jeuma, KP223728.1; Longlin, KM433673.1; Rongchang, KM044239.1; Sandu-black, KM094194.1; Wuji, KM259826.1). A consensus sequence of all ten (European + Asian) mtDNA sequences was obtained by analysing multiple aligned sequences, generated with Multiple Sequence Alignment Tool MUSCLE (<a class="link link-uri" href="http://www.ebi.ac.uk/Tools/msa/muscle/" target="_blank">http://www.ebi.ac.uk/Tools/msa/muscle/</a>), with the CONS software at the EMBOSS-explorer web server (<a class="link link-uri" href="http://www.bioinformatics.nl/emboss-explorer/" target="_blank">http://www.bioinformatics.nl/emboss-explorer/</a>). Since mtDNA is circular, two linearized mtDNA sequences with the same consensus origin were concatenated to each other, so that subsequent comparison methods could not miss the possibility to identify matches at the beginning or at the end of a linearized single sequence due to boundary effects.</p>                            <h4 scrollto-destination=106356190 id="106356190" class="section-title" >2.1.2. LAST analysis</h4>
<p>This concatenated consensus sequence was used to query the reference <em>S</em><em>.</em><em>scrofa</em> nuclear genome using LAST software.<a reveal-id="dsx019-B18" data-open="dsx019-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> LAST software has been considered to better perform than BLAST in detecting <em>numts</em> as it can be tuned with parameters that can capture low identity expected when aligning old <em>numts</em> to modern mtDNA.<a reveal-id="dsx019-B18" data-open="dsx019-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> LAST alignment parameters were: +1 for matches, −1 for mismatches, 7 for gap-open penalty and 1 for gap-extension penalty, that are suitable for detecting distant homology.<a reveal-id="dsx019-B19" data-open="dsx019-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> LAST software gives a score based on the expected number of alignments of a random sequence with the same length as the query and a random sequence with the same length as the database. To identify a threshold score for LAST in this approach that minimizes the risk of identifying false positives matches, we followed what was proposed by Tsuji <em>et al.</em><a reveal-id="dsx019-B20" data-open="dsx019-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> using the reversed mitochondrial genome approach. The reversed mtDNA genome was obtained by simply reversing, without complementing, its sequence. This sequence was aligned to the reference nuclear pig genome. Any match between this newly generated sequence and the nuclear genome can be considered to be spurious as DNA sequences do not evolve by simple reversal.<a reveal-id="dsx019-B20" data-open="dsx019-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The number of matches that could be obtained by chance in this test is reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. A score of 37, which gives an expected probability equal to 0.000734 to identify spurious matches in our analysis, was set as score threshold to identify significant matches.</p>                            <h3 scrollto-destination=106356192 id="106356192" class="section-title" >2.2. Analyses of <em>numt</em> regions</h3>
<p>All LAST matches including overlapping sequences or consecutive sequences on the same chromosome were visually inspected using dot plot graphs produced with the ggplot2 R programme (y axis: the mtDNA consensus sequence; x axis: the nuclear genome sequence). Consecutive matches on both axes were considered to identify a <em>numt</em> region derived by a unique insertional event followed by subsequent insertions/deletions of other origin. Other events (i.e. duplications, rearrangements or non-unique insertions) were identified when more complex patterns in consecutive nuclear sequences were evident, compared with the mtDNA sequence (and considered as complex events).</p><p>The genomic context of the regions flanking the detected <em>numts</em> was analysed to identify: (i) repetitive elements (1,000 bp from both sides from the inserted <em>numts</em>) using RepeatMasker version 4.0.6 (<a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a>); (ii) GC content in the 1,000 bp flanking regions of the <em>numt</em> sequences; (iii) the position of insertion (intergenic, intronic, coding exons, 5′ and 3′ untranslated regions); (iv) the closest annotated gene in the Sscrofa10.2 genome version.</p>                            <h3 scrollto-destination=106356195 id="106356195" class="section-title" >2.3. Identification of polymorphic <em>numt</em> insertions</h3>
<p>Blood or muscle samples were used to extract DNA from a total of 66 pigs belonging to 3 commercial breeds (36 Italian Large White, 15 Italian Landrace, 15 Italian Duroc), 77 pigs from 5 Italian local pig breeds (16 Cinta Senese, 16 Mora Romagnola, 17 Casertana, 10 Apulo Calabrese, 18 Nero Siciliano), 13 European wild boars and 6 pigs from the Chinese Meishan breed. DNA was isolated using the Wizard Genomic DNA Purification Kit (Promega, Madison, WI, USA) following the manufacturer protocols and used for PCR analyses of six selected <em>numts</em>. <em>Numts</em> were chosen considering different criteria: (i) different levels of homology of the <em>numt</em> against the modern mtDNA sequence; (ii) different chromosomes; (iii) possibility to design unique primers (i.e. absense of repetead sequences in the flanking regions, as determid by RepeatMasker version 4.0.6; <a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a>); (iv) insertion size of about 200–250 bp to facilitate amplification reactions. Primer3Plus (<a class="link link-uri" href="http://primer3plus.com/web_3.0.0/primer3web_input.htm" target="_blank">http://primer3plus.com/web_3.0.0/primer3web_input.htm</a>) programme was used to design PCR primers within about 150 bp of the flanking regions of the six selected <em>numts</em>. PCR primers and PCR conditions are reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. A 2,720 thermal cycler (Life Technologies, Carlsbad, CA, USA) was used for amplification reactions (the following temperature profile was used: 5 min at 95 °C; 35 amplification cycles of 30 s at 95 °C, 30 s at the specific annealing temperature reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>, 30 s at 72 °C; 5 min at 72 °C) that were carried out with the Phusion High-Fidelity PCR Master Mix (New England Biolabs, Ipswich, MA, USA), according to the manufacturer’s instructions, on a 20 µl final reaction volume that contained about 50 ng of template porcine DNA. Amplified fragments were separated by electrophoresis in 1% agarose gels and visualized with 1× GelRed Nucleic Acid Gel Stain (Biotium Inc., Hayward, CA, USA). Amplicons obtained from all primer pairs for at least six different animals were used for Sanger sequencing. Amplified fragments from homozygous animals were purified from the PCR reactions using ExoSAP-IT (USB Corporation, Cleveland, OH, USA) or ethanol precipitation. Fragments from heterozygous animals were isolated from gels cutting the band of interest that was purified with the Gel Extraction Kit (Qiagen, Valencia, CA, USA). Purified fragments were sequenced using the BrightDye Terminatior Cycle Sequencing Kit (Nimagen, Nijmegen, The Netherlands), and loaded on ABI3100 Avant capillary sequencer (Applied Biosystems/Life Technologies, Carlsbad, CA, USA). All sequences were visually inspected and aligned with the help of CodonCode Aligner (version 5.1.5) software (<a class="link link-uri" href="http://www.codoncode.com/aligner/" target="_blank">http://www.codoncode.com/aligner/</a>). Obtained sequences were aligned to the Sscrofa10.2 reference genome or to the expected <em>numt</em> flanking regions without <em>numt</em> insertion using BLASTN.</p>                            <h3 scrollto-destination=106356197 id="106356197" class="section-title" >2.4. Phylogenetic analyses</h3>
<p>An ancestral mtDNA sequence was inferred using all publicly available and complete mtDNA sequences of Suinae species (subfamily of the Suidae family) that diverged about 10 MYBP.<a reveal-id="dsx019-B21" data-open="dsx019-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Sequences were from: <em>Phacochoerus africanus</em> (GenBank accession number: DQ409327), <em>Potamochoerus porcus</em> (JN632688), <em>S</em><em>.</em><em>scrofa</em> (mtDNA from Sscrofa10.2 reference genome), <em>Sus celebensis</em> (KM203891), <em>Sus cebifrons</em> (KF952600), <em>Sus verrucosus</em> (KF926379), <em>Sus barbatus</em> (KP789021). mtDNA sequences were aligned as described above and the consensus sequence was obtained with the CONS software (see above). The age of each <em>numt</em> inserted in the Sscrofa10.2 genome version was calculated according to Dayama <em>et al.</em><a reveal-id="dsx019-B23" data-open="dsx019-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> as follows: (i) each <em>numt</em> sequence was aligned to the previously aligned ancestral Suinae and modern mtDNA sequences; (ii) the total number of sites in the aligned regions where the ancestral and modern mitochondrial sequences differ were counted (ancestral_vs_modern); (iii) of these variable positions, the <em>numt</em> sequence that matched the modern mtDNA sequences was tabulated (numt_vs_modern); (iv) the allele matching ratio (amr) was calculated: amr = (numt_vs_modern)/(ancestral_vs_modern), and used to estimate when the insertion occurred along the Suinae lineage (insertion before present, ibp), considering 10 MYBP its divergence period, as follows: ibp = (1 – amr) × 10, expressed in MYBP. <em>Numts</em> that contained D-loop sequences were not used for age estimation due to their uncertaintly in predicting ancestral D-loop sequences. The ibp was also computed for <em>numt</em> regions derived by unique insertional events (defined earlier) by summing, for the different LAST matches, numt_vs_modern and ancestral_vs_modern to obtain a single allele matching ratio for each <em>numt</em> region. Phylogenetic trees were obtained for the validated <em>numt</em> (<a reveal-id="dsx019-T1" data-open="dsx019-T1" class="link link-reveal link-table xref-fig">Table 1</a>) and the largest <em>numt</em> insertions (&gt;500 bp). <em>Numt</em> sequences were aligned (using MUSCLE software) with the corresponding mtDNA sequence of all Suinae species reported earlier in addition to the mtDNA sequences of <em>Pecari tajacu</em> (Tajassuidae species, Suidae; accession number AP003427) and <em>Bos taurus</em> (AY526085), downloaded from GenBank. Trees were built with MEGA 7.0.18<a reveal-id="dsx019-B22" data-open="dsx019-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> using Maximum-Likelihood method (applying default options) with 1,000 bootstrap values.<a reveal-id="dsx019-B23" data-open="dsx019-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=QlUpHU02~8PPlbH9OJgCQBw5rV1FV1Qbh7t4ljzAbax2VMrB6qNTt9rTiqhwQa3xZdbop5zR-qxTyHA0Q08I31GsVNJa954gs-fzEckxhplXYH4DdqSFWqrDZe29Jxrn2PSRe~GimybF7YQcTJOl3tbbfFvZbL2FCf7n4WYVNmDIxxAmVcAiKKGuX5QOGUuaSyRRN5a7LJlqp6OQIy7jBN7GnL~oQmivn~4zk7JsSv7lAPQ0mU4U36FxqVNmMPCSBEarGmxuHL~p0kt7AfrOYa0WBIcB~ONR-61Ec0HwB70Z5xMUyKabY77oSbvqFuy7srISXrRDcHJmK4BoW6pfFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1 includes sequences used for the phylogenetic analyses. To evaluate the presence of ancient <em>numt</em> insertions in the genome of first closest species to the pig for which an annotated genome is available (the cattle genome, UMD3.1 genome version), LAST analysis (with the <em>B</em><em>.</em><em>taurus</em> mtDNA sequence indicated above and using parameters described for the mining of the pig genome) was carried out in the cattle genome. <em>Numt</em> insertions were declared orthologous by comparative mapping analysis using annotated regions in the two pig and cattle nuclear genomes and by veryfing the insertion of homologous mtDNA regions in two species using BLASTN. <div content-id="dsx019-T1" class="table-modal"><div class="table-wrap"><div id="dsx019-T1" data-id="dsx019-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Characteristics of investigated <em>numt</em> sequences</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Numt</em> ID<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Numt</em> length (bp) </th><th>Identity with the modern mtDNA (%) </th><th>No. of diagnostic positions </th><th>No. of positions matching modern mtDNA </th><th>Estimated age of insertion (MYBP)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Polymorphism<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>3_15 </td><td>207 </td><td>99 </td><td>3 </td><td>3 </td><td>&lt;0.5 </td><td>Yes </td></tr><tr><td>4_19 </td><td>218 </td><td>89 </td><td>3 </td><td>1 </td><td>6.7 </td><td>No </td></tr><tr><td>8_10 </td><td>214 </td><td>92 </td><td>4 </td><td>2 </td><td>5 </td><td>Yes </td></tr><tr><td>13_32 </td><td>202 </td><td>90 </td><td>6 </td><td>2 </td><td>6.7 </td><td>Yes </td></tr><tr><td>15_09 </td><td>239 </td><td>86 </td><td>6 </td><td>0 </td><td>10 </td><td>No </td></tr><tr><td>16_02 </td><td>228 </td><td>93 </td><td>0 </td><td>0 </td><td>nc <a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>No </td></tr></tbody></table></div><div content-id="dsx019-T1" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Numt</em> ID<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Numt</em> length (bp) </th><th>Identity with the modern mtDNA (%) </th><th>No. of diagnostic positions </th><th>No. of positions matching modern mtDNA </th><th>Estimated age of insertion (MYBP)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Polymorphism<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>3_15 </td><td>207 </td><td>99 </td><td>3 </td><td>3 </td><td>&lt;0.5 </td><td>Yes </td></tr><tr><td>4_19 </td><td>218 </td><td>89 </td><td>3 </td><td>1 </td><td>6.7 </td><td>No </td></tr><tr><td>8_10 </td><td>214 </td><td>92 </td><td>4 </td><td>2 </td><td>5 </td><td>Yes </td></tr><tr><td>13_32 </td><td>202 </td><td>90 </td><td>6 </td><td>2 </td><td>6.7 </td><td>Yes </td></tr><tr><td>15_09 </td><td>239 </td><td>86 </td><td>6 </td><td>0 </td><td>10 </td><td>No </td></tr><tr><td>16_02 </td><td>228 </td><td>93 </td><td>0 </td><td>0 </td><td>nc <a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>No </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Details of <em>numt</em> sequences are reported in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&amp;Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>. The first number indicates the porcine chromosome in which the corresponding <em>numt</em> is inserted. The second number is a progressive count of numts sequences in the indicated chromosome.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Calculated considering MYBP: millions of years before present.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>Presence or absence of the <em>numt</em> insertion in one or more pig breeds.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>d</sup>Not calculated. This alignment did not contain any informative site for this estimation.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx019-T1" data-id="dsx019-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Characteristics of investigated <em>numt</em> sequences</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Numt</em> ID<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Numt</em> length (bp) </th><th>Identity with the modern mtDNA (%) </th><th>No. of diagnostic positions </th><th>No. of positions matching modern mtDNA </th><th>Estimated age of insertion (MYBP)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Polymorphism<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>3_15 </td><td>207 </td><td>99 </td><td>3 </td><td>3 </td><td>&lt;0.5 </td><td>Yes </td></tr><tr><td>4_19 </td><td>218 </td><td>89 </td><td>3 </td><td>1 </td><td>6.7 </td><td>No </td></tr><tr><td>8_10 </td><td>214 </td><td>92 </td><td>4 </td><td>2 </td><td>5 </td><td>Yes </td></tr><tr><td>13_32 </td><td>202 </td><td>90 </td><td>6 </td><td>2 </td><td>6.7 </td><td>Yes </td></tr><tr><td>15_09 </td><td>239 </td><td>86 </td><td>6 </td><td>0 </td><td>10 </td><td>No </td></tr><tr><td>16_02 </td><td>228 </td><td>93 </td><td>0 </td><td>0 </td><td>nc <a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>No </td></tr></tbody></table></div><div content-id="dsx019-T1" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Numt</em> ID<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Numt</em> length (bp) </th><th>Identity with the modern mtDNA (%) </th><th>No. of diagnostic positions </th><th>No. of positions matching modern mtDNA </th><th>Estimated age of insertion (MYBP)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Polymorphism<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>3_15 </td><td>207 </td><td>99 </td><td>3 </td><td>3 </td><td>&lt;0.5 </td><td>Yes </td></tr><tr><td>4_19 </td><td>218 </td><td>89 </td><td>3 </td><td>1 </td><td>6.7 </td><td>No </td></tr><tr><td>8_10 </td><td>214 </td><td>92 </td><td>4 </td><td>2 </td><td>5 </td><td>Yes </td></tr><tr><td>13_32 </td><td>202 </td><td>90 </td><td>6 </td><td>2 </td><td>6.7 </td><td>Yes </td></tr><tr><td>15_09 </td><td>239 </td><td>86 </td><td>6 </td><td>0 </td><td>10 </td><td>No </td></tr><tr><td>16_02 </td><td>228 </td><td>93 </td><td>0 </td><td>0 </td><td>nc <a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td><td>No </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Details of <em>numt</em> sequences are reported in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&amp;Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>. The first number indicates the porcine chromosome in which the corresponding <em>numt</em> is inserted. The second number is a progressive count of numts sequences in the indicated chromosome.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Calculated considering MYBP: millions of years before present.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>Presence or absence of the <em>numt</em> insertion in one or more pig breeds.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>d</sup>Not calculated. This alignment did not contain any informative site for this estimation.</p></div></div></div></div></p>                            <h3 scrollto-destination=106356199 id="106356199" class="section-title" >2.5. Data release</h3>
<p>All new sequences have been submitted to EMBL/GenBank databases with accession numbers from LT707410 to LT707414. <em>Numt</em> annotation of the Sscrofa10.2 genome version is available as GFF file as <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s2.doc?Expires=2147483647&Signature=lqYxwTwZAu-bdU6ae-6FWFSVHbo0e5ex1CX9Idj6-AzW0E9Uk6iZdkSvGJ4trdEnmmfdIhFlLbzRKQwLX-~USJGSvnUgHohQjaw9gsqZ4O~lg4DKMZUGugn5VUhDRVDrhClK7uNVs3FXg8~KGUdtjZFZCzNj0HB1y~r2hthcR0kq04ERa73i-THID3pTwscQS1OfoK3r25y8ytmdHB57swj5npOu5z391Bfo3JWpYrYClG1TXnP0BlSKwfb4WkExBFb590TFa23KS-3JBe-zvw62bWMO2DJCkhTGsArZQQbXRwFfbaI4Iz2y6WWqeuazHrM-eROwZVeC61zMenGI9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S2</a></span>. GFF annotation is also available through the Dryad database (<a class="link link-uri" href="http://datadryad.org/" target="_blank">http://datadryad.org/</a>).</p>                            <h2 scrollto-destination=106356201 id="106356201" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=106356202 id="106356202" class="section-title" >3.1. Identification of <em>numt</em> sequences in the pig nuclear reference genome</h3>
<p>Mining the assembled and the unassembled (scaffolds) of the nuclear reference genome Sscrofa10.2 with LAST and a consensus modern mtDNA sequence obtained from European and Chinese pigs, we identified 401 and 29 significant matches (LAST score &gt;37), respectively, belonging to <em>numt</em> sequences (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&amp;Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>). Identity of these matched nuclear sequences with the modern mtDNA sequence ranged from 60 to 100% (mean of 78%). The lower value may indicate the detection limit of our approach. Distribution of <em>numt</em> sequences according to their size is presented in <a reveal-id="dsx019-T2" data-open="dsx019-T2" class="link link-reveal link-table xref-fig">Table 2</a>. <em>Numt</em> sequence lenght varied from 42 to 4,782 bp with an oulier of about 11 kbp integrated on porcine chromosome (SSC) 2, with 90% identity with the modern porcine mtDNA sequence. <div content-id="dsx019-T2" class="table-modal"><div class="table-wrap"><div id="dsx019-T2" data-id="dsx019-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Summary table of detected <em>numt</em> sequences according to their size range</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number </th><th>Size range (bp) </th><th>Similarity range (%) </th><th>MYBP<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> range </th><th>Average MYBP </th></tr></thead><tbody><tr><td>59 </td><td>40–99 </td><td>78–100 </td><td>0–10 </td><td>7.25 </td></tr><tr><td>130 </td><td>100–249 </td><td>65–100 </td><td>0–10 </td><td>7.02 </td></tr><tr><td>83 </td><td>250–499 </td><td>62–92 </td><td>0–10 </td><td>7.03 </td></tr><tr><td>52 </td><td>500–749 </td><td>60–94 </td><td>0–10 </td><td>6.76 </td></tr><tr><td>27 </td><td>750–999 </td><td>36–98 </td><td>0–10 </td><td>6.62 </td></tr><tr><td>39 </td><td>1000–1999 </td><td>63–95 </td><td>2.5–9.4 </td><td>7.19 </td></tr><tr><td>10 </td><td>2000–4999 </td><td>67–90 </td><td>5.8–8 </td><td>7.03 </td></tr><tr><td>1 </td><td>&gt;5000 </td><td>90 </td><td>7.6 </td><td>7.57 </td></tr></tbody></table></div><div content-id="dsx019-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Number </th><th>Size range (bp) </th><th>Similarity range (%) </th><th>MYBP<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> range </th><th>Average MYBP </th></tr></thead><tbody><tr><td>59 </td><td>40–99 </td><td>78–100 </td><td>0–10 </td><td>7.25 </td></tr><tr><td>130 </td><td>100–249 </td><td>65–100 </td><td>0–10 </td><td>7.02 </td></tr><tr><td>83 </td><td>250–499 </td><td>62–92 </td><td>0–10 </td><td>7.03 </td></tr><tr><td>52 </td><td>500–749 </td><td>60–94 </td><td>0–10 </td><td>6.76 </td></tr><tr><td>27 </td><td>750–999 </td><td>36–98 </td><td>0–10 </td><td>6.62 </td></tr><tr><td>39 </td><td>1000–1999 </td><td>63–95 </td><td>2.5–9.4 </td><td>7.19 </td></tr><tr><td>10 </td><td>2000–4999 </td><td>67–90 </td><td>5.8–8 </td><td>7.03 </td></tr><tr><td>1 </td><td>&gt;5000 </td><td>90 </td><td>7.6 </td><td>7.57 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>Millions of years before present as reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx019-T2" data-id="dsx019-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Summary table of detected <em>numt</em> sequences according to their size range</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number </th><th>Size range (bp) </th><th>Similarity range (%) </th><th>MYBP<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> range </th><th>Average MYBP </th></tr></thead><tbody><tr><td>59 </td><td>40–99 </td><td>78–100 </td><td>0–10 </td><td>7.25 </td></tr><tr><td>130 </td><td>100–249 </td><td>65–100 </td><td>0–10 </td><td>7.02 </td></tr><tr><td>83 </td><td>250–499 </td><td>62–92 </td><td>0–10 </td><td>7.03 </td></tr><tr><td>52 </td><td>500–749 </td><td>60–94 </td><td>0–10 </td><td>6.76 </td></tr><tr><td>27 </td><td>750–999 </td><td>36–98 </td><td>0–10 </td><td>6.62 </td></tr><tr><td>39 </td><td>1000–1999 </td><td>63–95 </td><td>2.5–9.4 </td><td>7.19 </td></tr><tr><td>10 </td><td>2000–4999 </td><td>67–90 </td><td>5.8–8 </td><td>7.03 </td></tr><tr><td>1 </td><td>&gt;5000 </td><td>90 </td><td>7.6 </td><td>7.57 </td></tr></tbody></table></div><div content-id="dsx019-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Number </th><th>Size range (bp) </th><th>Similarity range (%) </th><th>MYBP<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> range </th><th>Average MYBP </th></tr></thead><tbody><tr><td>59 </td><td>40–99 </td><td>78–100 </td><td>0–10 </td><td>7.25 </td></tr><tr><td>130 </td><td>100–249 </td><td>65–100 </td><td>0–10 </td><td>7.02 </td></tr><tr><td>83 </td><td>250–499 </td><td>62–92 </td><td>0–10 </td><td>7.03 </td></tr><tr><td>52 </td><td>500–749 </td><td>60–94 </td><td>0–10 </td><td>6.76 </td></tr><tr><td>27 </td><td>750–999 </td><td>36–98 </td><td>0–10 </td><td>6.62 </td></tr><tr><td>39 </td><td>1000–1999 </td><td>63–95 </td><td>2.5–9.4 </td><td>7.19 </td></tr><tr><td>10 </td><td>2000–4999 </td><td>67–90 </td><td>5.8–8 </td><td>7.03 </td></tr><tr><td>1 </td><td>&gt;5000 </td><td>90 </td><td>7.6 </td><td>7.57 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>Millions of years before present as reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p></div></div></div></div></p><p>All these <em>numts</em> accounted for a total of 0.0078% of the nuclear porcine genome (a total of 217,848 bp). <a reveal-id="dsx019-T3" data-open="dsx019-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <a reveal-id="dsx019-F1" data-open="dsx019-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> show the distribution of <em>numt</em> sequences in the different porcine chromosomes and their positions. Porcine chromosome (SSC) 14 showed the largest proportion of <em>numts</em> (∼0.02% of its size was covered by these sequences with also the largest number of detected <em>numt</em> sequences; <em>n</em> = 60) whereas SSC5 was covered by the lowest proportion of these integrated nuclear mtDNAs (∼0.001%). There is a modest linear relationship between the number of <em>numts</em> located on each chromosome and their relative length (Pearson’s <em>r</em> = 0.65, <em>P</em>-value = 0.003). The average distance between two close <em>numts</em> in the reference porcine genome is 6.5 Mbp (ranging from a few bp to 70 Mbp). <div content-id="dsx019-T3" class="table-modal"><div class="table-wrap"><div id="dsx019-T3" data-id="dsx019-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p><em>Numt</em> sequences in the pig nuclear genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SSC </th><th>No. of <em>numts</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Total <em>numt</em> sequences (bp) </th><th>Total <em>numt</em> sequences (% on the SSC) </th></tr></thead><tbody><tr><td>1 </td><td>37; 21 (28) </td><td>14,299 </td><td>0.004535 </td></tr><tr><td>2 </td><td>28; 15 (14) </td><td>27,824 </td><td>0.017115 </td></tr><tr><td>3 </td><td>28; 5 (13) </td><td>23,385 </td><td>0.016151 </td></tr><tr><td>4 </td><td>20; 17 (13) </td><td>5,169 </td><td>0.003603 </td></tr><tr><td>5 </td><td>9; 9 (10) </td><td>1,169 </td><td>0.001048 </td></tr><tr><td>6 </td><td>30; 17 (14) </td><td>12,196 </td><td>0.00773 </td></tr><tr><td>7 </td><td>10; 9 (12) </td><td>1,463 </td><td>0.001086 </td></tr><tr><td>8 </td><td>18; 13 (13) </td><td>12,897 </td><td>0.008665 </td></tr><tr><td>9 </td><td>25; 15 (14) </td><td>13,853 </td><td>0.009015 </td></tr><tr><td>10 </td><td>17; 6 (7) </td><td>9,670 </td><td>0.012225 </td></tr><tr><td>11 </td><td>11; 11 (8) </td><td>2,219 </td><td>0.00253 </td></tr><tr><td>12 </td><td>11; 6 (6) </td><td>3,676 </td><td>0.005781 </td></tr><tr><td>13 </td><td>35; 19 (19) </td><td>13,951 </td><td>0.006381 </td></tr><tr><td>14 </td><td>60; 20 (14) </td><td>31,592 </td><td>0.020534 </td></tr><tr><td>15 </td><td>23; 16 (14) </td><td>9,954 </td><td>0.006313 </td></tr><tr><td>16 </td><td>5; 5 (8) </td><td>983 </td><td>0.001131 </td></tr><tr><td>17 </td><td>11; 8 (6) </td><td>8,443 </td><td>0.012113 </td></tr><tr><td>18 </td><td>7; 6 (5) </td><td>1,512 </td><td>0.00247 </td></tr><tr><td>X </td><td>16; 12 (13) </td><td>15,987 </td><td>0.01108 </td></tr><tr><td>Un<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>29; 16 </td><td>7,606 </td><td>— </td></tr><tr><td><strong>Total</strong> </td><td><strong>430; 246</strong> </td><td><strong>217,848</strong> </td><td><strong>0.00776</strong> </td></tr></tbody></table></div><div content-id="dsx019-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>SSC </th><th>No. of <em>numts</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Total <em>numt</em> sequences (bp) </th><th>Total <em>numt</em> sequences (% on the SSC) </th></tr></thead><tbody><tr><td>1 </td><td>37; 21 (28) </td><td>14,299 </td><td>0.004535 </td></tr><tr><td>2 </td><td>28; 15 (14) </td><td>27,824 </td><td>0.017115 </td></tr><tr><td>3 </td><td>28; 5 (13) </td><td>23,385 </td><td>0.016151 </td></tr><tr><td>4 </td><td>20; 17 (13) </td><td>5,169 </td><td>0.003603 </td></tr><tr><td>5 </td><td>9; 9 (10) </td><td>1,169 </td><td>0.001048 </td></tr><tr><td>6 </td><td>30; 17 (14) </td><td>12,196 </td><td>0.00773 </td></tr><tr><td>7 </td><td>10; 9 (12) </td><td>1,463 </td><td>0.001086 </td></tr><tr><td>8 </td><td>18; 13 (13) </td><td>12,897 </td><td>0.008665 </td></tr><tr><td>9 </td><td>25; 15 (14) </td><td>13,853 </td><td>0.009015 </td></tr><tr><td>10 </td><td>17; 6 (7) </td><td>9,670 </td><td>0.012225 </td></tr><tr><td>11 </td><td>11; 11 (8) </td><td>2,219 </td><td>0.00253 </td></tr><tr><td>12 </td><td>11; 6 (6) </td><td>3,676 </td><td>0.005781 </td></tr><tr><td>13 </td><td>35; 19 (19) </td><td>13,951 </td><td>0.006381 </td></tr><tr><td>14 </td><td>60; 20 (14) </td><td>31,592 </td><td>0.020534 </td></tr><tr><td>15 </td><td>23; 16 (14) </td><td>9,954 </td><td>0.006313 </td></tr><tr><td>16 </td><td>5; 5 (8) </td><td>983 </td><td>0.001131 </td></tr><tr><td>17 </td><td>11; 8 (6) </td><td>8,443 </td><td>0.012113 </td></tr><tr><td>18 </td><td>7; 6 (5) </td><td>1,512 </td><td>0.00247 </td></tr><tr><td>X </td><td>16; 12 (13) </td><td>15,987 </td><td>0.01108 </td></tr><tr><td>Un<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>29; 16 </td><td>7,606 </td><td>— </td></tr><tr><td><strong>Total</strong> </td><td><strong>430; 246</strong> </td><td><strong>217,848</strong> </td><td><strong>0.00776</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p><sup>a</sup>Number of <em>numt</em> sequences determined by LAST; number of <em>numt</em> integration events as defined in the text; in parenthesis: expected number of <em>numt</em> integration events, considering equal distribution across the nuclear genome (rounded numbers without any decimals).</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>b</sup>Unassembled scaffolds.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx019-T3" data-id="dsx019-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p><em>Numt</em> sequences in the pig nuclear genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SSC </th><th>No. of <em>numts</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Total <em>numt</em> sequences (bp) </th><th>Total <em>numt</em> sequences (% on the SSC) </th></tr></thead><tbody><tr><td>1 </td><td>37; 21 (28) </td><td>14,299 </td><td>0.004535 </td></tr><tr><td>2 </td><td>28; 15 (14) </td><td>27,824 </td><td>0.017115 </td></tr><tr><td>3 </td><td>28; 5 (13) </td><td>23,385 </td><td>0.016151 </td></tr><tr><td>4 </td><td>20; 17 (13) </td><td>5,169 </td><td>0.003603 </td></tr><tr><td>5 </td><td>9; 9 (10) </td><td>1,169 </td><td>0.001048 </td></tr><tr><td>6 </td><td>30; 17 (14) </td><td>12,196 </td><td>0.00773 </td></tr><tr><td>7 </td><td>10; 9 (12) </td><td>1,463 </td><td>0.001086 </td></tr><tr><td>8 </td><td>18; 13 (13) </td><td>12,897 </td><td>0.008665 </td></tr><tr><td>9 </td><td>25; 15 (14) </td><td>13,853 </td><td>0.009015 </td></tr><tr><td>10 </td><td>17; 6 (7) </td><td>9,670 </td><td>0.012225 </td></tr><tr><td>11 </td><td>11; 11 (8) </td><td>2,219 </td><td>0.00253 </td></tr><tr><td>12 </td><td>11; 6 (6) </td><td>3,676 </td><td>0.005781 </td></tr><tr><td>13 </td><td>35; 19 (19) </td><td>13,951 </td><td>0.006381 </td></tr><tr><td>14 </td><td>60; 20 (14) </td><td>31,592 </td><td>0.020534 </td></tr><tr><td>15 </td><td>23; 16 (14) </td><td>9,954 </td><td>0.006313 </td></tr><tr><td>16 </td><td>5; 5 (8) </td><td>983 </td><td>0.001131 </td></tr><tr><td>17 </td><td>11; 8 (6) </td><td>8,443 </td><td>0.012113 </td></tr><tr><td>18 </td><td>7; 6 (5) </td><td>1,512 </td><td>0.00247 </td></tr><tr><td>X </td><td>16; 12 (13) </td><td>15,987 </td><td>0.01108 </td></tr><tr><td>Un<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>29; 16 </td><td>7,606 </td><td>— </td></tr><tr><td><strong>Total</strong> </td><td><strong>430; 246</strong> </td><td><strong>217,848</strong> </td><td><strong>0.00776</strong> </td></tr></tbody></table></div><div content-id="dsx019-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>SSC </th><th>No. of <em>numts</em><a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Total <em>numt</em> sequences (bp) </th><th>Total <em>numt</em> sequences (% on the SSC) </th></tr></thead><tbody><tr><td>1 </td><td>37; 21 (28) </td><td>14,299 </td><td>0.004535 </td></tr><tr><td>2 </td><td>28; 15 (14) </td><td>27,824 </td><td>0.017115 </td></tr><tr><td>3 </td><td>28; 5 (13) </td><td>23,385 </td><td>0.016151 </td></tr><tr><td>4 </td><td>20; 17 (13) </td><td>5,169 </td><td>0.003603 </td></tr><tr><td>5 </td><td>9; 9 (10) </td><td>1,169 </td><td>0.001048 </td></tr><tr><td>6 </td><td>30; 17 (14) </td><td>12,196 </td><td>0.00773 </td></tr><tr><td>7 </td><td>10; 9 (12) </td><td>1,463 </td><td>0.001086 </td></tr><tr><td>8 </td><td>18; 13 (13) </td><td>12,897 </td><td>0.008665 </td></tr><tr><td>9 </td><td>25; 15 (14) </td><td>13,853 </td><td>0.009015 </td></tr><tr><td>10 </td><td>17; 6 (7) </td><td>9,670 </td><td>0.012225 </td></tr><tr><td>11 </td><td>11; 11 (8) </td><td>2,219 </td><td>0.00253 </td></tr><tr><td>12 </td><td>11; 6 (6) </td><td>3,676 </td><td>0.005781 </td></tr><tr><td>13 </td><td>35; 19 (19) </td><td>13,951 </td><td>0.006381 </td></tr><tr><td>14 </td><td>60; 20 (14) </td><td>31,592 </td><td>0.020534 </td></tr><tr><td>15 </td><td>23; 16 (14) </td><td>9,954 </td><td>0.006313 </td></tr><tr><td>16 </td><td>5; 5 (8) </td><td>983 </td><td>0.001131 </td></tr><tr><td>17 </td><td>11; 8 (6) </td><td>8,443 </td><td>0.012113 </td></tr><tr><td>18 </td><td>7; 6 (5) </td><td>1,512 </td><td>0.00247 </td></tr><tr><td>X </td><td>16; 12 (13) </td><td>15,987 </td><td>0.01108 </td></tr><tr><td>Un<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>29; 16 </td><td>7,606 </td><td>— </td></tr><tr><td><strong>Total</strong> </td><td><strong>430; 246</strong> </td><td><strong>217,848</strong> </td><td><strong>0.00776</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p><sup>a</sup>Number of <em>numt</em> sequences determined by LAST; number of <em>numt</em> integration events as defined in the text; in parenthesis: expected number of <em>numt</em> integration events, considering equal distribution across the nuclear genome (rounded numbers without any decimals).</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>b</sup>Unassembled scaffolds.</p></div></div></div></div></p>                        <a id="106356205" scrollto-destination="106356205"></a>
<div id="dsx019-F1" data-id="dsx019-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f1.png?Expires=2147483647&Signature=GPHceAGyWcdzGjcBHZorQdmTpoId4y~t9Emxdbf8k6du~Gl65tnva7C9Bbgvbz61J-F9v4E22Hrm7JiXFSuZ4nH7Z4Jx7VHzl7NleVLZZbwqkE2-me1r~7DIzwDjufMs4k-O5fmXSvh8deboZe~q2GhN21vWJDMB5ugqIE5xVVCYQKtn59x-7jMbSIJU369hLhA46NOzNFeabf4T-dlPKmKb-T75Y8LM2YnWpbkJEeC0uXwmzrA6wWTBJghegKq7lAhqYKaxFodgCOpCEcSyePNFC8TSYN9gGlDTCvHggvh5ZPIHLTelleVTss4g7F4fGIpV4Fiqxb2WE3LZwNF0UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of numts in the pig nuclear genome. Numt positions are represented in the Sscrofa10.2 assembled chromosomes." path-from-xml="dsx019f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356205/dsx019f1.png" path-from-xml="dsx019f1.png" target="_blank">View large</a><a section="106356205" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f1.png?Expires=2147483647&Signature=rLq4BO9us3bk0gGe-PfFrYbOWul0AbnTtO9XX3LpvV0IQXpGBoG4btrz9ALVGHL029cVDEbloELNYoP5MTSuuYkotcocvBbRT01oucu5ZJKmLTsTpJuCONY8Kw74Sng5FFys1sHUTpyEHTuTrgrfWwuC8LjH8rwp39apUENzvvFub-bwBW1stNKk~ow1~ZyD0KeePcwVmCYr-Zk9jPIQwuAT-1~DEWyK9KqinCqo000QP8wY~-ksnuaSNR3kbmgnucZgadkFR~v-Fl20A85~Lxz~FdwSB4KIrOgVNeJ0gPzWsCrUMh1gysIZSArCDMIcVTfrNSfAXA9bqGS5gcdhKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356205&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of <em>numts</em> in the pig nuclear genome. <em>Numt</em> positions are represented in the Sscrofa10.2 assembled chromosomes.</p></div></div><div content-id="dsx019-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f1.png?Expires=2147483647&Signature=GPHceAGyWcdzGjcBHZorQdmTpoId4y~t9Emxdbf8k6du~Gl65tnva7C9Bbgvbz61J-F9v4E22Hrm7JiXFSuZ4nH7Z4Jx7VHzl7NleVLZZbwqkE2-me1r~7DIzwDjufMs4k-O5fmXSvh8deboZe~q2GhN21vWJDMB5ugqIE5xVVCYQKtn59x-7jMbSIJU369hLhA46NOzNFeabf4T-dlPKmKb-T75Y8LM2YnWpbkJEeC0uXwmzrA6wWTBJghegKq7lAhqYKaxFodgCOpCEcSyePNFC8TSYN9gGlDTCvHggvh5ZPIHLTelleVTss4g7F4fGIpV4Fiqxb2WE3LZwNF0UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of numts in the pig nuclear genome. Numt positions are represented in the Sscrofa10.2 assembled chromosomes." path-from-xml="dsx019f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356205/dsx019f1.png" path-from-xml="dsx019f1.png" target="_blank">View large</a><a section="106356205" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f1.png?Expires=2147483647&Signature=rLq4BO9us3bk0gGe-PfFrYbOWul0AbnTtO9XX3LpvV0IQXpGBoG4btrz9ALVGHL029cVDEbloELNYoP5MTSuuYkotcocvBbRT01oucu5ZJKmLTsTpJuCONY8Kw74Sng5FFys1sHUTpyEHTuTrgrfWwuC8LjH8rwp39apUENzvvFub-bwBW1stNKk~ow1~ZyD0KeePcwVmCYr-Zk9jPIQwuAT-1~DEWyK9KqinCqo000QP8wY~-ksnuaSNR3kbmgnucZgadkFR~v-Fl20A85~Lxz~FdwSB4KIrOgVNeJ0gPzWsCrUMh1gysIZSArCDMIcVTfrNSfAXA9bqGS5gcdhKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356205&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of <em>numts</em> in the pig nuclear genome. <em>Numt</em> positions are represented in the Sscrofa10.2 assembled chromosomes.</p></div></div><p>Close <em>numt</em> sequences could identify unique insertional events into the nuclear genome that underwent subsequent deletions or insertions (and/or other complex rearrangements) separating contiguous mtDNA regions that we however identified as separate <em>numt</em> sequences using LAST. To evaluate this issue, we dot-plotted all nuclear regions including <em>numt</em> sequences separated by up to 20 kb against the modern mtDNA sequence. We then called independent <em>numt</em> regions (derived by unique <em>numt</em> insertional events) according to the possibility to infer contiguity based on the integration of mtDNA sequences into the nuclear genome. Other <em>numt</em> regions included complex rearrangements or duplications that could not be possible to consider as derived by one or more than one insertional event. <a reveal-id="dsx019-F2" data-open="dsx019-F2" class="link link-reveal link-table xref-fig">Figure 2</a> shows a few examples of <em>numt</em> regions that could be described as derived by unique insertional events or that could be considered as complex <em>numt</em> regions, including duplications and rearrangements. Of the total 430 <em>numt</em> sequences that matched the modern mtDNA sequence with LAST, using the dot plot approach we could identify 57 <em>numt</em> regions including at least two <em>numt</em> sequences (54 <em>numt</em> regions in the assembled reference genome and 3 in the unassembled scaffolds) and 189 singleton <em>numt</em> sequences, accounting for a total of 246 separated <em>numt</em> integrations in the reference porcine nuclear genome. Of these 57 <em>numt</em> regions, 51 were considered as derived by independent insertional events and 6 were defined as complex <em>numt</em> regions. Detailed information on all <em>numt</em> regions and putative independent integrations are reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>. Linear relationship between the number of <em>numt</em> integrations located on each chromosome and their relative length increased substantially (Pearson’s <em>r</em> = 0.99, <em>P</em>-value &lt; 0.001). The average distance between two close <em>numt integrations</em> in the reference porcine genome is 11.3 Mbp. </p>                        <a id="106356207" scrollto-destination="106356207"></a>
<div id="dsx019-F2" data-id="dsx019-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f2.png?Expires=2147483647&amp;Signature=YW7BGI~Wj3fkn1t-2XOFHEXC5QVBFY0CD-mVM8JK7ysIou-qqfAYo7YdpR7rfqNevV5nuIZ2XTwbrDpP9PRFQ1ONeQx4Xe0~RYqghtT~QDFQJahYFKhF9DEHlzpFGcO53MDFh7~LxQBuCjQIsP4yp42VSUrQ6EjcNeVBJGzNAkSE9boYynZWjdSFvLeJUluQj1-vRFXMjX0-5ounGa5EMfx1lL2TZLW6mRemQ5oBmJ~nDr4fMiK3EShfl6ld9fOlaY-ac98rkS34tT2jaWWFsT1XXCHKvkvbJe0PR0OIl-2hNOtHBZdWWxGsPE31UjcIlOxPqGQ-uDsyXCfsXL3tNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot-plot analysis of a few numt regions. Numt region ID and numt sequences are reported at the top of each plot. The X axis reports the positions in the indicated chromosomes. The Y axis reports the mtDNA sequence coordinates. Details of these regions are reported in Supplementary Table S3. Different coloured lines in the plots indicate different numt sequences." path-from-xml="dsx019f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356207/dsx019f2.png" path-from-xml="dsx019f2.png" target="_blank">View large</a><a section="106356207" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f2.png?Expires=2147483647&amp;Signature=vQaRZlbrlf66K85RkVkymknCyBOfi5mnN8KpF9UGeXovm0achel41mgDRXp8XLxCNO2DFw-yRb9R5xFz1GCPIgsmCBDM8h4vYllFY23-QqwmM7HGLAZngteXBfec~RuqpSe4TH-3ysPceQmMhi41IAVZwtqf5LxbMsbr5yrtvsyISlo-dts7gH-~Lx7jS5FWdTLGlieLgTwhbsG-CrGPadsvOMXHRZuYVr4B9q0aHVq6qnrVptUVKM9IMWarEWW0-srD3ti~LltvqnE-cAVgmSGY4~uuKHqNEhxDhM039ygHmYJizpVK1m4iO7v1DcTK8YvbXQx18E9qF8CwCEcS8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106356207&amp;ar=3778238&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx019f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot-plot analysis of a few <em>numt</em> regions. <em>Numt</em> region ID and <em>numt</em> sequences are reported at the top of each plot. The X axis reports the positions in the indicated chromosomes. The Y axis reports the mtDNA sequence coordinates. Details of these regions are reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. Different coloured lines in the plots indicate different <em>numt</em> sequences.</p></div></div><div content-id="dsx019-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f2.png?Expires=2147483647&amp;Signature=YW7BGI~Wj3fkn1t-2XOFHEXC5QVBFY0CD-mVM8JK7ysIou-qqfAYo7YdpR7rfqNevV5nuIZ2XTwbrDpP9PRFQ1ONeQx4Xe0~RYqghtT~QDFQJahYFKhF9DEHlzpFGcO53MDFh7~LxQBuCjQIsP4yp42VSUrQ6EjcNeVBJGzNAkSE9boYynZWjdSFvLeJUluQj1-vRFXMjX0-5ounGa5EMfx1lL2TZLW6mRemQ5oBmJ~nDr4fMiK3EShfl6ld9fOlaY-ac98rkS34tT2jaWWFsT1XXCHKvkvbJe0PR0OIl-2hNOtHBZdWWxGsPE31UjcIlOxPqGQ-uDsyXCfsXL3tNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot-plot analysis of a few numt regions. Numt region ID and numt sequences are reported at the top of each plot. The X axis reports the positions in the indicated chromosomes. The Y axis reports the mtDNA sequence coordinates. Details of these regions are reported in Supplementary Table S3. Different coloured lines in the plots indicate different numt sequences." path-from-xml="dsx019f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356207/dsx019f2.png" path-from-xml="dsx019f2.png" target="_blank">View large</a><a section="106356207" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f2.png?Expires=2147483647&amp;Signature=vQaRZlbrlf66K85RkVkymknCyBOfi5mnN8KpF9UGeXovm0achel41mgDRXp8XLxCNO2DFw-yRb9R5xFz1GCPIgsmCBDM8h4vYllFY23-QqwmM7HGLAZngteXBfec~RuqpSe4TH-3ysPceQmMhi41IAVZwtqf5LxbMsbr5yrtvsyISlo-dts7gH-~Lx7jS5FWdTLGlieLgTwhbsG-CrGPadsvOMXHRZuYVr4B9q0aHVq6qnrVptUVKM9IMWarEWW0-srD3ti~LltvqnE-cAVgmSGY4~uuKHqNEhxDhM039ygHmYJizpVK1m4iO7v1DcTK8YvbXQx18E9qF8CwCEcS8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106356207&amp;ar=3778238&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx019f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot-plot analysis of a few <em>numt</em> regions. <em>Numt</em> region ID and <em>numt</em> sequences are reported at the top of each plot. The X axis reports the positions in the indicated chromosomes. The Y axis reports the mtDNA sequence coordinates. Details of these regions are reported in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. Different coloured lines in the plots indicate different <em>numt</em> sequences.</p></div></div><p>As the accuracy of the assembly of the available pig reference genome (Sscrofa10.2) is limited, it could be possible that a few <em>numts</em> derive by misassemblies. However, as almost all <em>numt</em> sequences had identity lower than 100% (similarity ranged from 60 to 100%, median similarity was 77%) with the modern porcine mtDNA (indication of evolutionary divergence), it seems plausible to exclude assembly errors that would have included true mtDNA fragments into the nuclear genome. The only two short numts with 100% identity with the modern mtDNA are compatible with a recent insertion into the nuclear genome (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>). Other sources of misassemblies could have been produced by duplications of <em>numt</em> sequences even if a close inspection of the results did not identify any obvious problem originated by duplicated regions of the porcine genome.</p><p><a reveal-id="dsx019-F3" data-open="dsx019-F3" class="link link-reveal link-table xref-fig">Figure 3</a> shows (i) the distribution of <em>numts</em> along the circularized mtDNA genome with indications of nuclear porcine chromosome in which the corresponding <em>numt</em> was integrated (<a reveal-id="dsx019-F3" data-open="dsx019-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>) and (ii) the distribution of <em>numt</em> hits in the linearized mtDNA sequence at the different nucleotides (<a reveal-id="dsx019-F3" data-open="dsx019-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). If we just exclude the variable repeated region of the control region, for which alignments were problematic due to highly variable number of repeats, all mtDNA regions were covered at least once by <em>numt</em> sequences (<a reveal-id="dsx019-F3" data-open="dsx019-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). The most covered mtDNA regions by <em>numts</em> included parts of the COX1, COX2 and 16S-rRNA genes. It was also interesting to note that the D-loop of the control mtDNA region, a non-coding region of the mitochondrial genome that controls the synthesis of DNA and RNA within the mitochondria and that usually exhibits a higher mutation rate than the other regions of the mtDNA, was covered by 34 <em>numts</em>. </p>                        <a id="106356210" scrollto-destination="106356210"></a>
<div id="dsx019-F3" data-id="dsx019-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f3.png?Expires=2147483647&Signature=uvQgOlJ1lNT0gpBQPWOKLVCehCqoukvnYbVgtbBWhD6aY1DX5uvKOVv-KHQlYfb0dScQDL0wTpAXwQ1Q9GsKjXf5-W4wSa8aZ627VBeFB5WK16-ks5Q7jyept4BzZ5lv-CEbs1XfvxnPbiBqXMQWJbALbCmX~bz~VfnqDBFSbXrli2Hig-0nnIWK3i4eGji9dE-z~~suun3nOImY-vhRUOT7d5lF27-W2C0V1xh5eRCpJroFGa33QujxPWkaMYiO3Lyi2kwJqCA0GBHQcsV6S1-EjgBZBrTdmEv7lZRbBZqL1D-~s6jTNqT0O3CctPzrvQugjJE01ht8snzacp8nPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) The porcine mtDNA with numts plotted along its circular structure. Numts are coloured according to the porcine chromosomes in which they are inserted. Mitochondrial genes are reported in the outer ring. (b) Coverage of the linearized porcine mtDNA with numts. The number of numt copies are plotted in the ‘Y’ axis. The mitochondrial genes are reported in the ‘X’ axis." path-from-xml="dsx019f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356210/dsx019f3.png" path-from-xml="dsx019f3.png" target="_blank">View large</a><a section="106356210" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f3.png?Expires=2147483647&Signature=GjaNjcvVk1b66OA9jDf-dha75cumrZ~nnztD8CIwqjyIXtgXb1O9JtRXDiHStYrZ6DtILMjLaVmdflBBOSaFwyGnQv-XYD5koXd81b9VHH1F~7z2Ya0rWCIm5rWfM3JB~hvpubvnsjHgsdWQ4VLBhvAGR0uOor2aqAS2Z5kIl4k54JB~8lf8W0OkCwe8WwU0klNM3jh6cRufoE9ekvbJtgLQ7Kb80Hha141W-lXtIZGusXlLyutgqiV6wMtWr6RRSzintPx5qDzYCeDVyMhDUKjjw-GuSF41drxxTHC~zVFPVqLGV2JYsuBfMPPpb0JEZIZY2v76Tfq8VXMIb-Aopw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356210&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The porcine mtDNA with <em>numts</em> plotted along its circular structure. <em>Numts</em> are coloured according to the porcine chromosomes in which they are inserted. Mitochondrial genes are reported in the outer ring. (b) Coverage of the linearized porcine mtDNA with <em>numts</em>. The number of <em>numt</em> copies are plotted in the ‘Y’ axis. The mitochondrial genes are reported in the ‘X’ axis.</p></div></div><div content-id="dsx019-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f3.png?Expires=2147483647&Signature=uvQgOlJ1lNT0gpBQPWOKLVCehCqoukvnYbVgtbBWhD6aY1DX5uvKOVv-KHQlYfb0dScQDL0wTpAXwQ1Q9GsKjXf5-W4wSa8aZ627VBeFB5WK16-ks5Q7jyept4BzZ5lv-CEbs1XfvxnPbiBqXMQWJbALbCmX~bz~VfnqDBFSbXrli2Hig-0nnIWK3i4eGji9dE-z~~suun3nOImY-vhRUOT7d5lF27-W2C0V1xh5eRCpJroFGa33QujxPWkaMYiO3Lyi2kwJqCA0GBHQcsV6S1-EjgBZBrTdmEv7lZRbBZqL1D-~s6jTNqT0O3CctPzrvQugjJE01ht8snzacp8nPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) The porcine mtDNA with numts plotted along its circular structure. Numts are coloured according to the porcine chromosomes in which they are inserted. Mitochondrial genes are reported in the outer ring. (b) Coverage of the linearized porcine mtDNA with numts. The number of numt copies are plotted in the ‘Y’ axis. The mitochondrial genes are reported in the ‘X’ axis." path-from-xml="dsx019f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356210/dsx019f3.png" path-from-xml="dsx019f3.png" target="_blank">View large</a><a section="106356210" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f3.png?Expires=2147483647&Signature=GjaNjcvVk1b66OA9jDf-dha75cumrZ~nnztD8CIwqjyIXtgXb1O9JtRXDiHStYrZ6DtILMjLaVmdflBBOSaFwyGnQv-XYD5koXd81b9VHH1F~7z2Ya0rWCIm5rWfM3JB~hvpubvnsjHgsdWQ4VLBhvAGR0uOor2aqAS2Z5kIl4k54JB~8lf8W0OkCwe8WwU0klNM3jh6cRufoE9ekvbJtgLQ7Kb80Hha141W-lXtIZGusXlLyutgqiV6wMtWr6RRSzintPx5qDzYCeDVyMhDUKjjw-GuSF41drxxTHC~zVFPVqLGV2JYsuBfMPPpb0JEZIZY2v76Tfq8VXMIb-Aopw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356210&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The porcine mtDNA with <em>numts</em> plotted along its circular structure. <em>Numts</em> are coloured according to the porcine chromosomes in which they are inserted. Mitochondrial genes are reported in the outer ring. (b) Coverage of the linearized porcine mtDNA with <em>numts</em>. The number of <em>numt</em> copies are plotted in the ‘Y’ axis. The mitochondrial genes are reported in the ‘X’ axis.</p></div></div>                            <h3 scrollto-destination=106356211 id="106356211" class="section-title" >3.2. Characteristics of <em>numt</em> insertion regions</h3>
<p><em>Numts</em> were found to be slightly preferentially inserted in regions with higher GC content (flanking regions have a GC% equal to 42%, compared with the overal GC content of the pig nuclear genome that is 39%). RepeatMasker analysis of the <em>numt</em> flanking sequences determined that only 8% (64 cases out of all 802 flanking regions analysed) includes non-repetitive sequences. Therefore, most <em>numt</em> flanking regions contain repetitive elements (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>). The most frequent reported repetitive elements were short interspersed nuclear elements or SINE (48% of <em>numts</em> are nearby SINE elements). The second most frequent repetitive elements were long interspersed nuclear elements or LINE (24%). The overall SINE and LINE content of the pig genome is 11 and 16%, respectively,<a reveal-id="dsx019-B13" data-open="dsx019-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx019-B24" data-open="dsx019-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> therefore it is clear that <em>numt</em> sequences are preferentially located near or within repetitive elements in the <em>S</em><em>.</em><em>scrofa</em> genome (<em>P</em> &lt; 0.0001, chi square test).</p><p>A total of 57 <em>numt</em> sequences out of 430 <em>numts</em> detected in the <em>S</em><em>.</em><em>scrofa</em> genome (from 51 different <em>numt</em> regions or singletons) were located within 49 different annotated genes. All <em>numts</em> were in intronic regions. All other <em>numt</em> sequences were located in intergenic regions. Details on the closest gene to all detected <em>numts</em> (as annotated in Sscrofa10.2) are reported in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>. Gene functional enrichment of the closest annotated gene to all identified <em>numts</em> did not identify any enriched function or features (data not shown).</p>                            <h3 scrollto-destination=106356214 id="106356214" class="section-title" >3.3. Validation of <em>numts</em> and identification of polymorphic <em>numts</em> (presence/absence of insertion)</h3>
<p>Six primer pairs were designed to validate the presence of <em>numt</em> insertions in pigs belonging to Chinese (Meishan) and Euro-American pig breeds (three commercial and five local breeds) and European wild boars. At the same time these primers could verify if polymorphic <em>numt</em> regions derived by the presence of absence of the inserted <em>numt</em> could be detected from this panel of animals. Features of the amplified <em>numts</em> are reported in <a reveal-id="dsx019-T1" data-open="dsx019-T1" class="link link-reveal link-table xref-fig">Table 1</a>. Three primer pairs designed for <em>numts</em> 4_19 (identity with the corresponding modern mtDNA: ∼87%), 15_09 (identity: ∼86%) and 16_02 (identity: ∼93%), respectively, produced the expected fragment size (<a reveal-id="dsx019-T1" data-open="dsx019-T1" class="link link-reveal link-table xref-fig">Table 1</a>, including the inserted <em>numt</em> sequence as confirmed by sequencing) in all analysed pigs across all breeds and European wild boars. Amplifications obtained for three other <em>numts</em> (3_15, identity: 99%; 8_10, identity: 92%; and 13_32, identity: 90%) showed fragments of lower size than that expected (<a reveal-id="dsx019-F4" data-open="dsx019-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Sequences of these fragments confirmed the absence of <em>numt</em> sequences in these amplicons (EMBL accession numbers: LT707411 for <em>numt</em> 3_15; LT707414, for <em>numt</em> 8_10; LT707412 for <em>numt</em> 13_32; see <a reveal-id="dsx019-F5" data-open="dsx019-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> for the sequence alignments between genomic regions with the presence and the absence of the <em>numts</em>). </p>                        <a id="106356216" scrollto-destination="106356216"></a>
<div id="dsx019-F4" data-id="dsx019-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f4.png?Expires=2147483647&Signature=4NRtwPg8nWRjQJVkfn-baBCZGC4xssU76Nx5vn73joVJlBwcpjrq6b53RN91Cl9af06XUQzIGq~86Dn4frw~FvrmSpkdeQ8Y0KWO~ADclgteVYYy7yhG19PUXTeWdj2-MmaiK-DZ5An0Oq~HeLZP6NZooJAti0jNOWSR4yVI4efNlQdCxoMqAJaaqDdLOpTLElCtuTDo4Ymg1N-T4FzQwmfuDi2THLj0UZBPH6qN~KUK~sMJPBcP3UKP22E3C-TRQq0WiDNmqXnck6PxUbnRO8fxRqX~bTb7Rj05m4px6sJEcRG9-RFOcNiOt19Z0ynU13pnO4qQruUwnPENyqX3hA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Electrophoretic patterns of amplified numt regions (3_15, 8_10 and 13_32) showing polymorphisms determined by the presence or absence of numt sequences. ‘+’ indicates the presence of insertion; ‘−’ indicates the absence of the insertion. The genotypes are reported at the top of each gel picture. The number of pigs with the three different genotypes for each numt loci is reported." path-from-xml="dsx019f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356216/dsx019f4.png" path-from-xml="dsx019f4.png" target="_blank">View large</a><a section="106356216" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f4.png?Expires=2147483647&Signature=yqJgMh9x6pqHX7-XSyRIMYGsyIo5m8EgNX1FK3wI7f0VuM~1JprFKUmn7wdBFwX2~dYtiXsQzZXuMbMklvaRCVcnEQItf-fnG79BxJ~nauMHzxP8pg0Kj8yst293fH04tyucDZTCqgPNu0yt8r5n6GPLgfI7WG29P8B-VsR~JAx~WWrdlHFZILcNHOItnfYfQue353aEHQ1tlED2-fBYpqMEm4qE4rOF8UN6dx4prfPx647ZEiOMrFZsF8vO5qscokfJc7v6m5QTeLKZ34VSc09HWYmKaFnfj7jLm438BED5GJsQg72uBvKJOKtBSxAneYv~VxLA8r7TGCmj0g8Z0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356216&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Electrophoretic patterns of amplified <em>numt</em> regions (3_15, 8_10 and 13_32) showing polymorphisms determined by the presence or absence of <em>numt</em> sequences. ‘+’ indicates the presence of insertion; ‘−’ indicates the absence of the insertion. The genotypes are reported at the top of each gel picture. The number of pigs with the three different genotypes for each <em>numt</em> loci is reported.</p></div></div><div content-id="dsx019-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f4.png?Expires=2147483647&Signature=4NRtwPg8nWRjQJVkfn-baBCZGC4xssU76Nx5vn73joVJlBwcpjrq6b53RN91Cl9af06XUQzIGq~86Dn4frw~FvrmSpkdeQ8Y0KWO~ADclgteVYYy7yhG19PUXTeWdj2-MmaiK-DZ5An0Oq~HeLZP6NZooJAti0jNOWSR4yVI4efNlQdCxoMqAJaaqDdLOpTLElCtuTDo4Ymg1N-T4FzQwmfuDi2THLj0UZBPH6qN~KUK~sMJPBcP3UKP22E3C-TRQq0WiDNmqXnck6PxUbnRO8fxRqX~bTb7Rj05m4px6sJEcRG9-RFOcNiOt19Z0ynU13pnO4qQruUwnPENyqX3hA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Electrophoretic patterns of amplified numt regions (3_15, 8_10 and 13_32) showing polymorphisms determined by the presence or absence of numt sequences. ‘+’ indicates the presence of insertion; ‘−’ indicates the absence of the insertion. The genotypes are reported at the top of each gel picture. The number of pigs with the three different genotypes for each numt loci is reported." path-from-xml="dsx019f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356216/dsx019f4.png" path-from-xml="dsx019f4.png" target="_blank">View large</a><a section="106356216" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f4.png?Expires=2147483647&Signature=yqJgMh9x6pqHX7-XSyRIMYGsyIo5m8EgNX1FK3wI7f0VuM~1JprFKUmn7wdBFwX2~dYtiXsQzZXuMbMklvaRCVcnEQItf-fnG79BxJ~nauMHzxP8pg0Kj8yst293fH04tyucDZTCqgPNu0yt8r5n6GPLgfI7WG29P8B-VsR~JAx~WWrdlHFZILcNHOItnfYfQue353aEHQ1tlED2-fBYpqMEm4qE4rOF8UN6dx4prfPx647ZEiOMrFZsF8vO5qscokfJc7v6m5QTeLKZ34VSc09HWYmKaFnfj7jLm438BED5GJsQg72uBvKJOKtBSxAneYv~VxLA8r7TGCmj0g8Z0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356216&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Electrophoretic patterns of amplified <em>numt</em> regions (3_15, 8_10 and 13_32) showing polymorphisms determined by the presence or absence of <em>numt</em> sequences. ‘+’ indicates the presence of insertion; ‘−’ indicates the absence of the insertion. The genotypes are reported at the top of each gel picture. The number of pigs with the three different genotypes for each <em>numt</em> loci is reported.</p></div></div>                        <a id="106356217" scrollto-destination="106356217"></a>
<div id="dsx019-F5" data-id="dsx019-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f5.png?Expires=2147483647&amp;Signature=QPOjlDuVqW7I-kmFbkFEbpvGXUeR~Ks2itBmf51wAlPYuYehT3nMjgRudeuv9efYrD2fYslVUW9pcwzxING3WKKH1HH7wKEt4c7MRjGnjmbYC5gzPoG0JUORor9gagTQ7LFQ1QoOqu3WsC2tEBLGMInmOIMwbERTObWzY-NQu~FS01M8CsVgfsULewjYk6ZlOjj8RF10Y5E1sVXvfjJYrWCxhE07hQdkbp-9tlAoHN8n82mPTgLW8GgT8ImLfSAsxT-NKR5Vszpio1Hskff7U6sGHxiITCojezF4O69nCIrvHrWbhNEwx41v0hdXKulxFrkep6OC5zjP2HoSFqE6Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignments of regions including numts versus the same regions without numts, when polymorphic. The sequence indicated as ‘with_numt’ is identical to the Sscrofa10.2 reference region, the sequence defined as ‘without_numt’ indicates the sequence obtained by Sanger sequencing of the fragments that do not include the numt (Fig. 4). These regions where polymorphic in the population and their sequence were submitted in EMBL database with the following accession numbers: Numt 3_15, LT707410 with numt, LT707411 without numt; Numt 8_10, LT707410 with numt, LT707411 without numt; Numt 13_32, LT707412 without numt, the sequence with numt is the same present in Sscrofa10.2. Primer regions used for the amplification are included in the reported sequences: see Supplementary Table S2 for the black and white version to identify the primer sequences; primer regions are highlighted in green in the online colour picture." path-from-xml="dsx019f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356217/dsx019f5.png" path-from-xml="dsx019f5.png" target="_blank">View large</a><a section="106356217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f5.png?Expires=2147483647&amp;Signature=QNDU9FmZoOB2fDOXf2MskzQ0fSwthpq-0hwmoBIyTTGdxwS~wQA~m0oQsG1TG70OWA1N73ctngxbZK8VIh0zyQ0JbqPosKlGdv5Px5Gb9OUmenaTlLA~~~Aa1UNezUeu2LXT-gdHfoHNKDP-I11Yl4l0DhVPuLzaHtYXbRCm5KO9tHfwj8sVcbREEOWxgch2dq0UhmZb6GZMsrI1fK3q6trWQ08gv5gca~pCXi3eM8-MQM-IRIWS8-naUbmeZXzwgF97k3d-KPlANM1pLqUa94mFLfZDQ5srvMC5KsS6EPKwy-bIr~zRDl73VV3TS7~gSIWyUDz5VCb9n5J6DEntKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106356217&amp;ar=3778238&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx019f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignments of regions including <em>numts</em> versus the same regions without <em>numts</em>, when polymorphic. The sequence indicated as ‘with_<em>numt</em>’ is identical to the Sscrofa10.2 reference region, the sequence defined as ‘without_<em>numt</em>’ indicates the sequence obtained by Sanger sequencing of the fragments that do not include the <em>numt</em> (<a reveal-id="dsx019-F4" data-open="dsx019-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). These regions where polymorphic in the population and their sequence were submitted in EMBL database with the following accession numbers: <em>Numt</em> 3_15, LT707410 with <em>numt</em>, LT707411 without <em>numt</em>; <em>Numt</em> 8_10, LT707410 with <em>numt</em>, LT707411 without <em>numt</em>; <em>Numt</em> 13_32, LT707412 without <em>numt</em>, the sequence with <em>numt</em> is the same present in Sscrofa10.2. Primer regions used for the amplification are included in the reported sequences: see <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> for the black and white version to identify the primer sequences; primer regions are highlighted in green in the online colour picture.</p></div></div><div content-id="dsx019-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f5.png?Expires=2147483647&amp;Signature=QPOjlDuVqW7I-kmFbkFEbpvGXUeR~Ks2itBmf51wAlPYuYehT3nMjgRudeuv9efYrD2fYslVUW9pcwzxING3WKKH1HH7wKEt4c7MRjGnjmbYC5gzPoG0JUORor9gagTQ7LFQ1QoOqu3WsC2tEBLGMInmOIMwbERTObWzY-NQu~FS01M8CsVgfsULewjYk6ZlOjj8RF10Y5E1sVXvfjJYrWCxhE07hQdkbp-9tlAoHN8n82mPTgLW8GgT8ImLfSAsxT-NKR5Vszpio1Hskff7U6sGHxiITCojezF4O69nCIrvHrWbhNEwx41v0hdXKulxFrkep6OC5zjP2HoSFqE6Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignments of regions including numts versus the same regions without numts, when polymorphic. The sequence indicated as ‘with_numt’ is identical to the Sscrofa10.2 reference region, the sequence defined as ‘without_numt’ indicates the sequence obtained by Sanger sequencing of the fragments that do not include the numt (Fig. 4). These regions where polymorphic in the population and their sequence were submitted in EMBL database with the following accession numbers: Numt 3_15, LT707410 with numt, LT707411 without numt; Numt 8_10, LT707410 with numt, LT707411 without numt; Numt 13_32, LT707412 without numt, the sequence with numt is the same present in Sscrofa10.2. Primer regions used for the amplification are included in the reported sequences: see Supplementary Table S2 for the black and white version to identify the primer sequences; primer regions are highlighted in green in the online colour picture." path-from-xml="dsx019f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356217/dsx019f5.png" path-from-xml="dsx019f5.png" target="_blank">View large</a><a section="106356217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f5.png?Expires=2147483647&amp;Signature=QNDU9FmZoOB2fDOXf2MskzQ0fSwthpq-0hwmoBIyTTGdxwS~wQA~m0oQsG1TG70OWA1N73ctngxbZK8VIh0zyQ0JbqPosKlGdv5Px5Gb9OUmenaTlLA~~~Aa1UNezUeu2LXT-gdHfoHNKDP-I11Yl4l0DhVPuLzaHtYXbRCm5KO9tHfwj8sVcbREEOWxgch2dq0UhmZb6GZMsrI1fK3q6trWQ08gv5gca~pCXi3eM8-MQM-IRIWS8-naUbmeZXzwgF97k3d-KPlANM1pLqUa94mFLfZDQ5srvMC5KsS6EPKwy-bIr~zRDl73VV3TS7~gSIWyUDz5VCb9n5J6DEntKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106356217&amp;ar=3778238&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx019f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignments of regions including <em>numts</em> versus the same regions without <em>numts</em>, when polymorphic. The sequence indicated as ‘with_<em>numt</em>’ is identical to the Sscrofa10.2 reference region, the sequence defined as ‘without_<em>numt</em>’ indicates the sequence obtained by Sanger sequencing of the fragments that do not include the <em>numt</em> (<a reveal-id="dsx019-F4" data-open="dsx019-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). These regions where polymorphic in the population and their sequence were submitted in EMBL database with the following accession numbers: <em>Numt</em> 3_15, LT707410 with <em>numt</em>, LT707411 without <em>numt</em>; <em>Numt</em> 8_10, LT707410 with <em>numt</em>, LT707411 without <em>numt</em>; <em>Numt</em> 13_32, LT707412 without <em>numt</em>, the sequence with <em>numt</em> is the same present in Sscrofa10.2. Primer regions used for the amplification are included in the reported sequences: see <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> for the black and white version to identify the primer sequences; primer regions are highlighted in green in the online colour picture.</p></div></div><p>Distribution of polymorphic <em>numts</em> is reported in <a reveal-id="dsx019-T4" data-open="dsx019-T4" class="link link-reveal link-table xref-fig">Table 4</a>. For <em>numt</em> 3_15, polymorphic insertions (i.e. presence of two alleles: one with the <em>numt</em> sequence and one without this insertion) were observed in two local pig breeds (Apulo Calabrese and Nero Siciliano, with two genotypes only) and in the Chinese Meishan, with the same frequency of all three genotypes in just six animals analysed. All other breeds were fixed for the insertion allele. For <em>numt</em> 8_10, presence/absense polymorphism was observed in Italian Large White, Italian Landrace, Italian Duroc and Nero Siciliano breeds in which the insertion allele was always the most frequent. All other breeds were fixed for inserted allele. Polymorphism derived by the presence/absence of the <em>numt</em> 13_32 was observed in Meishan and Italian Large White pigs in which the absence of insertion allele was the most frequent or was close to 50% and in three local breeds (Cinta Senese, Apulo Calabrese and Nero Siciliano) in which the insertion allele was far the most frequent. In all other breeds, the insertion allele for <em>numt</em> 13_32 was fixed. All European wild boars investigated were homozygous for the insertion allele at al three <em>numt</em> loci. <div content-id="dsx019-T4" class="table-modal"><div class="table-wrap"><div id="dsx019-T4" data-id="dsx019-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p><em>Numt</em> insertion polimorphisms for three <em>numt</em> loci (3_15, 8_10 and 13_32) in different pig breeds and populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Breed/population </th><th rowspan="2">No. of pigs </th><th colspan="3">3_15<hr /> </th><th colspan="3">8_10<hr /> </th><th colspan="3">13_32<hr /> </th></tr><tr><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th></tr></thead><tbody><tr><td>Italian Large White </td><td>36 </td><td>36 </td><td>— </td><td>— </td><td>24 </td><td>8 </td><td>4 </td><td>13 </td><td>12 </td><td>11 </td></tr><tr><td>Italian Landrace </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>14 </td><td>— </td><td>1 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Italian Duroc </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>8 </td><td>3 </td><td>4 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Cinta Senese </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>15 </td><td>— </td><td>1 </td></tr><tr><td>Mora Romagnola </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td></tr><tr><td>Casertana </td><td>17 </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td></tr><tr><td>Apulo Calabrese </td><td>10 </td><td>7 </td><td>— </td><td>3 </td><td>10 </td><td>— </td><td>— </td><td>9 </td><td>1 </td><td>— </td></tr><tr><td>Nero Siciliano </td><td>18 </td><td>17 </td><td>1 </td><td>— </td><td>13 </td><td>4 </td><td>1 </td><td>15 </td><td>1 </td><td>2 </td></tr><tr><td>European wild boar </td><td>13 </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td></tr><tr><td>Meishan </td><td>6 </td><td>2 </td><td>2 </td><td>2 </td><td>6 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div content-id="dsx019-T4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Breed/population </th><th rowspan="2">No. of pigs </th><th colspan="3">3_15<hr /> </th><th colspan="3">8_10<hr /> </th><th colspan="3">13_32<hr /> </th></tr><tr><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th></tr></thead><tbody><tr><td>Italian Large White </td><td>36 </td><td>36 </td><td>— </td><td>— </td><td>24 </td><td>8 </td><td>4 </td><td>13 </td><td>12 </td><td>11 </td></tr><tr><td>Italian Landrace </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>14 </td><td>— </td><td>1 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Italian Duroc </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>8 </td><td>3 </td><td>4 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Cinta Senese </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>15 </td><td>— </td><td>1 </td></tr><tr><td>Mora Romagnola </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td></tr><tr><td>Casertana </td><td>17 </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td></tr><tr><td>Apulo Calabrese </td><td>10 </td><td>7 </td><td>— </td><td>3 </td><td>10 </td><td>— </td><td>— </td><td>9 </td><td>1 </td><td>— </td></tr><tr><td>Nero Siciliano </td><td>18 </td><td>17 </td><td>1 </td><td>— </td><td>13 </td><td>4 </td><td>1 </td><td>15 </td><td>1 </td><td>2 </td></tr><tr><td>European wild boar </td><td>13 </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td></tr><tr><td>Meishan </td><td>6 </td><td>2 </td><td>2 </td><td>2 </td><td>6 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn8"><div class="fn"><p>‘+’ indicates the presence of insertion; ‘−’ indicates the absence of the insertion. The number of pigs with the three different genotypes for each <em>numt</em> locus is reported.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx019-T4" data-id="dsx019-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p><em>Numt</em> insertion polimorphisms for three <em>numt</em> loci (3_15, 8_10 and 13_32) in different pig breeds and populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Breed/population </th><th rowspan="2">No. of pigs </th><th colspan="3">3_15<hr /> </th><th colspan="3">8_10<hr /> </th><th colspan="3">13_32<hr /> </th></tr><tr><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th></tr></thead><tbody><tr><td>Italian Large White </td><td>36 </td><td>36 </td><td>— </td><td>— </td><td>24 </td><td>8 </td><td>4 </td><td>13 </td><td>12 </td><td>11 </td></tr><tr><td>Italian Landrace </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>14 </td><td>— </td><td>1 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Italian Duroc </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>8 </td><td>3 </td><td>4 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Cinta Senese </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>15 </td><td>— </td><td>1 </td></tr><tr><td>Mora Romagnola </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td></tr><tr><td>Casertana </td><td>17 </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td></tr><tr><td>Apulo Calabrese </td><td>10 </td><td>7 </td><td>— </td><td>3 </td><td>10 </td><td>— </td><td>— </td><td>9 </td><td>1 </td><td>— </td></tr><tr><td>Nero Siciliano </td><td>18 </td><td>17 </td><td>1 </td><td>— </td><td>13 </td><td>4 </td><td>1 </td><td>15 </td><td>1 </td><td>2 </td></tr><tr><td>European wild boar </td><td>13 </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td></tr><tr><td>Meishan </td><td>6 </td><td>2 </td><td>2 </td><td>2 </td><td>6 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div content-id="dsx019-T4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Breed/population </th><th rowspan="2">No. of pigs </th><th colspan="3">3_15<hr /> </th><th colspan="3">8_10<hr /> </th><th colspan="3">13_32<hr /> </th></tr><tr><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th><th>+/+ </th><th>−/+ </th><th>−/− </th></tr></thead><tbody><tr><td>Italian Large White </td><td>36 </td><td>36 </td><td>— </td><td>— </td><td>24 </td><td>8 </td><td>4 </td><td>13 </td><td>12 </td><td>11 </td></tr><tr><td>Italian Landrace </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>14 </td><td>— </td><td>1 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Italian Duroc </td><td>15 </td><td>15 </td><td>— </td><td>— </td><td>8 </td><td>3 </td><td>4 </td><td>15 </td><td>— </td><td>— </td></tr><tr><td>Cinta Senese </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>15 </td><td>— </td><td>1 </td></tr><tr><td>Mora Romagnola </td><td>16 </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td><td>16 </td><td>— </td><td>— </td></tr><tr><td>Casertana </td><td>17 </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td><td>17 </td><td>— </td><td>— </td></tr><tr><td>Apulo Calabrese </td><td>10 </td><td>7 </td><td>— </td><td>3 </td><td>10 </td><td>— </td><td>— </td><td>9 </td><td>1 </td><td>— </td></tr><tr><td>Nero Siciliano </td><td>18 </td><td>17 </td><td>1 </td><td>— </td><td>13 </td><td>4 </td><td>1 </td><td>15 </td><td>1 </td><td>2 </td></tr><tr><td>European wild boar </td><td>13 </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td><td>13 </td><td>— </td><td>— </td></tr><tr><td>Meishan </td><td>6 </td><td>2 </td><td>2 </td><td>2 </td><td>6 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn8"><div class="fn"><p>‘+’ indicates the presence of insertion; ‘−’ indicates the absence of the insertion. The number of pigs with the three different genotypes for each <em>numt</em> locus is reported.</p></div></div></div></div></p>                            <h3 scrollto-destination=106356219 id="106356219" class="section-title" >3.4. Phylogenetic analysis of <em>numts</em></h3>
<p>We estimated when the insertional events occurred in the Suinae lineage by comparing modern mtDNA sequences with a consensus Suinae mtDNA sequence and identifying diagnostic sites in the <em>numt</em> sequences. <a reveal-id="dsx019-T1" data-open="dsx019-T1" class="link link-reveal link-table xref-fig">Table 1</a> reports the estimated age of insertion of validated <em>numts</em>. Information for all <em>numts</em> (for which alignments and detection of diagnostic sites were possible) is reported in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>. Age of <em>numts</em> ranged from 10 MYBP (the back limit of our approach derived by the use of a Suinae consensus sequence used to identify informative nucleotides) to very recent integration times (&lt;0.5 MYBP). The largest <em>numt</em> spanning about 11 kb of the modern mtDNA, located on SSC2, was estimated to be inserted in the porcine nuclear genome about 7.7 MYBP (<em>numt</em> 2_25 in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span>). Phylogenetic trees placed more divergent <em>numt</em> sequences (identity &lt;80% with the modern mtDNAs) before the Tajassuidae lineage separation or even out from the <em>B</em><em>.</em><em>taurus</em> root. Confirmation of the presence of orthologous <em>numt</em> sequences in the bovine genome was obtained by comparing <em>numts</em> identified within the regions that are syntenic between <em>S</em><em>.</em><em>scrofa</em> and <em>B</em><em>.</em><em>taurus</em> genomes. Orthologous <em>numts</em> are reported in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span> with their respective bovine nuclear and mitochondrial coordinates. Five of them, namely 9_09, 10_10, 15_15, 18_05 and 18_06 were located within the orthologuos bovine genes (within the same intron), and in both species with identity from 63 to 71%, further indicating that the insertion was not recent and originated by a unique event that occurred in a common ancestral genome.</p><p>Results of phylogenetic analyses for the polymorphic <em>numts</em> indicated that the range of insertion age spanned a surprisingly large period of time if referred in the Suinae evolutionary line. In particular, all diagnostic sites of <em>numt</em> 3_15 were present in the modern mtDNA suggesting that its insertion was recent (below the resolution time of 1 MYBP) and probably after the separation of the common ancestral wild boar lineage from which the Asian and European pigs were domesticated. Phylogenetic tree reconstruction confirmed the recent insertion of this <em>numt</em> (<a reveal-id="dsx019-F6" data-open="dsx019-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). As most of European breeds are fixed for this insertion (<a reveal-id="dsx019-T4" data-open="dsx019-T4" class="link link-reveal link-table xref-fig">Table 4</a>), we could speculate that this <em>numt</em> was inserted during the European evolutionary differentiation within the <em>S</em><em>.</em><em>scrofa</em> species. Insertion of this <em>numt</em> might derive from a European wild boar ancestor of this lineage. Then this polymorphic site could have been introgressed into Asian domestic breeds as Meishan pigs show both alleles (presence and absence of the insertion). </p>                        <a id="106356222" scrollto-destination="106356222"></a>
<div id="dsx019-F6" data-id="dsx019-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f6.png?Expires=2147483647&Signature=1tPaq4GRMpILVKOy3fdMmRVkpVwMXJOKde8FvTxmhXqPj1p9g40hJcDBQ-LjEs2IgbFgTXs0s~RGH0tKG7u~Q2gY~MtH7DK8dQ7iWP0TQt8khBTSJ2XwS9JzUJtuGHvmD8cKm36WllIaV0Iqq5ukZg7wMEiIJKIQiK8lRT0gURzV9GtbyhN-X79q-27zFBCW7vLCW~d2OXtNUUxrPEQ8GtL6iroqBEphChq7xj68a6jefbeS5vF33Nw4LztOzQvesSCaVBReUUfBKtdd7D2~RUrWGrc0OHNA-YgGA96eBjn4lLzUUHmfvh1lTIPEcw-WtWeOZTMxtzy34j60usNvGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of four numt sequences reported as examples, three of which (numts 3_15, 8_10 and 13_32, indicated in red in the trees) were the polymorphic numts. The number at each node indicates the bootstrap frequency." path-from-xml="dsx019f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356222/dsx019f6.png" path-from-xml="dsx019f6.png" target="_blank">View large</a><a section="106356222" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f6.png?Expires=2147483647&Signature=mXSpwFYXmsoCfywWo84nR-9t3Qj5lnxsTXQU0NUKhjaX~Eatplktia53KePXmF8NTwOgFYRWY0qU3fQfFEZHDOD57GU9EqPMEuRoGGVo9bBtzwT7Wsq5YsqHhq0a50EwhyHCRcIQGVyU9LhYWH5C~LoPm-pelEHbIRFs8CA8qdRIgpOVnO6yb89nzbJTJZYhmTkyqUcfaGQlAK9KHAL~YaH~rEw1WSXcymLKgre6D6X68I5KHpAlszcu0V51-RvSHWtGnvhVsW858iiOb6dRNdzC2wI3qy8KOWeMyfRCBrthKYxMNNF9cR6fivHbgZyjJ0ZXKDld3rji0xSyQFmKCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356222&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees of four <em>numt</em> sequences reported as examples, three of which (<em>numts</em> 3_15, 8_10 and 13_32, indicated in red in the trees) were the polymorphic <em>numts</em>. The number at each node indicates the bootstrap frequency.</p></div></div><div content-id="dsx019-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/m_dsx019f6.png?Expires=2147483647&Signature=1tPaq4GRMpILVKOy3fdMmRVkpVwMXJOKde8FvTxmhXqPj1p9g40hJcDBQ-LjEs2IgbFgTXs0s~RGH0tKG7u~Q2gY~MtH7DK8dQ7iWP0TQt8khBTSJ2XwS9JzUJtuGHvmD8cKm36WllIaV0Iqq5ukZg7wMEiIJKIQiK8lRT0gURzV9GtbyhN-X79q-27zFBCW7vLCW~d2OXtNUUxrPEQ8GtL6iroqBEphChq7xj68a6jefbeS5vF33Nw4LztOzQvesSCaVBReUUfBKtdd7D2~RUrWGrc0OHNA-YgGA96eBjn4lLzUUHmfvh1lTIPEcw-WtWeOZTMxtzy34j60usNvGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of four numt sequences reported as examples, three of which (numts 3_15, 8_10 and 13_32, indicated in red in the trees) were the polymorphic numts. The number at each node indicates the bootstrap frequency." path-from-xml="dsx019f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356222/dsx019f6.png" path-from-xml="dsx019f6.png" target="_blank">View large</a><a section="106356222" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019f6.png?Expires=2147483647&Signature=mXSpwFYXmsoCfywWo84nR-9t3Qj5lnxsTXQU0NUKhjaX~Eatplktia53KePXmF8NTwOgFYRWY0qU3fQfFEZHDOD57GU9EqPMEuRoGGVo9bBtzwT7Wsq5YsqHhq0a50EwhyHCRcIQGVyU9LhYWH5C~LoPm-pelEHbIRFs8CA8qdRIgpOVnO6yb89nzbJTJZYhmTkyqUcfaGQlAK9KHAL~YaH~rEw1WSXcymLKgre6D6X68I5KHpAlszcu0V51-RvSHWtGnvhVsW858iiOb6dRNdzC2wI3qy8KOWeMyfRCBrthKYxMNNF9cR6fivHbgZyjJ0ZXKDld3rji0xSyQFmKCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356222&ar=3778238&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx019f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees of four <em>numt</em> sequences reported as examples, three of which (<em>numts</em> 3_15, 8_10 and 13_32, indicated in red in the trees) were the polymorphic <em>numts</em>. The number at each node indicates the bootstrap frequency.</p></div></div><p><em>Numt</em> 8_10 was estimated to be inserted about 5 MYBP (<a reveal-id="dsx019-T1" data-open="dsx019-T1" class="link link-reveal link-table xref-fig">Table 1</a>) as also confirmed by the phylogenetic tree analysis that placed it before the separation of all considered <em>Sus</em> species (<a reveal-id="dsx019-F6" data-open="dsx019-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). As Meishan breed was fixed for the inserted allele and all commercial European domestic pig breeds were polymorphic at <em>numt</em> 8_10, it is possible to speculate that the absence allele was introgressed into European populations from Asian pigs, that in turn might have acquired the old allele (absence of the <em>numt</em> sequence) from interspecific admixture with other <em>Sus</em> species living in the ISEA closely in contact with <em>Sus</em> species of the Mainland Southeast Asia (MSEA). This interspecific gene flow may explain the age of this <em>numt</em> and at the same time the fact that it is not fixed in the <em>S</em><em>.</em><em>scrofa</em> (as it might be considering the estimated age before the <em>Sus</em> lineage differentiation).</p><p>Estimation of the insertion age for <em>numt</em> 13_32 indicated about 6.7 MYBP. The topology of the phylogenetic tree including this <em>numt</em> was more difficult to be interpreted according to the evolutionary relationships between the <em>Sus</em> and the other Suinae taxa included in this study (<a reveal-id="dsx019-F6" data-open="dsx019-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). This might be due to differences in information content of different parts of the mtDNA. The position of <em>numt</em> 13_32 however may indicate an ancient integregation age that probably occurred before the separation of the <em>Sus</em> species (late Miocene/early Pliocene). In addition, the presence of polymorphisms in both modern European and Asian <em>S</em><em>.</em><em>scrofa</em> breeds and populations (<a reveal-id="dsx019-T1" data-open="dsx019-T1" class="link link-reveal link-table xref-fig">Table 1</a>) might indicate again a possible introgression of the ‘absence’ allele of putative Asian origin (not fixed in Asian breeds) into European domestic pigs, similarly to what might have been happened for <em>numt</em> 8_10.</p>                            <h2 scrollto-destination=106356225 id="106356225" class="section-title" >4. Discussion</h2>
<p>Evolution of eukaryotic genomes has been affected by the acquisition of new sequences of many different origins into the nuclear DNA. For example, in mammals mobile or transposable elements (the most important class of acquired elements) accounts for nearly half of their genomes<a reveal-id="dsx019-B1" data-open="dsx019-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>. The role and impact of the insertion of LINE and SINE and retroviruses into the nuclear genome have been investigated in many species, including farm animals. These insertions could be particularly relevant when genes were targeted, contributing to define novel functions and structures.<a reveal-id="dsx019-B25" data-open="dsx019-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Inserted elements may also provide phylogenetic information among related species, considering their ancient signature derived by their integration position (uniqueness) in an ancestral genome.<a reveal-id="dsx019-B26" data-open="dsx019-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx019-B27" data-open="dsx019-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> This ancestral signature has been used to reconstruct the history of domestication in animals.<a reveal-id="dsx019-B28" data-open="dsx019-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> In humans, model animals and livestock species, more recent mutagenic insertional events, polymorphic within species, have been reported to cause a number of diseases and novel phenotypes.<a reveal-id="dsx019-B29" data-open="dsx019-B29" class="link link-ref link-reveal xref-bibr"><sup>29–31</sup></a></p><p>Most of the eukaryotic genomes so far analysed have demonstrated the presence of mtDNA sequences into the nuclear genome.<a reveal-id="dsx019-B4" data-open="dsx019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> As expected, also for the <em>S</em><em>.</em><em>scrofa</em> nuclear genome we reported evidence for the transfer of mtDNA to the nuclear genome. The fraction of the ‘contaminated’ nuclear genome of the pig (0.0078%) is within the range already observed in several other mammals. For example, the percentage of the nuclear human and cattle genomes covered by <em>numts</em> was reported to be 0.0087 and 0.0023%, respectively.<a reveal-id="dsx019-B4" data-open="dsx019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> A few differences among studies carried out within the same species (e.g. humans) might be due to different mining approaches and applied thresholds to declare homology between nuclear DNA and mtDNA sequences.<a reveal-id="dsx019-B4" data-open="dsx019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx019-B20" data-open="dsx019-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Following what was suggested by Tsuji <em>et al.</em>,<a reveal-id="dsx019-B20" data-open="dsx019-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> who mined the genome of several mammals to find <em>numts</em>, we used the LAST programme to perform local sequence alignments instead of BLAST. Also in our hands, LAST was able to identify a larger number of <em>numt</em> sequences compared with BLAST (a total of 89 BLAST hits were identified in the Sscrofa10.2, <em>e</em>-value &lt; 0.001) that did not capture more divergent sequences derived by ancient integrations (that underwent several mutational differentiation processes) or complex rearrangements (data not shown).</p><p>Dot-plot analysis of <em>numts</em> indicated that a quite large number of porcine <em>numt</em> sequences could be derived by a single insertional event that subsequently underwent additional mutations as part of the evolutionary processes of the nuclear genome. A total of 247 separated insertional events (with 56 <em>numt</em> regions derived by more than one <em>numt</em> sequence) could be inferred after this close inspection of the inserted nuclear genomic regions. This analysis indicated that most of the complex <em>numt</em> regions were derived by nuclear mutations that split the integrated mtDNA in separated but co-linear regions with a few exceptions that included more complex rearrangements with duplications and invertions (<a reveal-id="dsx019-F2" data-open="dsx019-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Most of the complex <em>numt</em> regions were estimated to be caused by very ancient mtDNA integration events in the nuclear genome of this species (at least 10 MYBP that is the back limit defined by our consensus mtDNA sequence).</p><p>All mtDNA regions were present in porcine <em>numts</em>, including D-loop sequences. Tsuji <em>et al.</em><a reveal-id="dsx019-B20" data-open="dsx019-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> suggested that the D-loop region is under-represented in human <em>numts</em> as a specific effect of <em>numt</em> integration bias of the primate evolutionary lineage. This might not be true for the pig genome in which a quite large number of <em>numts</em> covered these highly polymorphic mtDNA regions. As this porcine mtDNA region has been used several times in population genetic studies and phylogenetic analyses<a reveal-id="dsx019-B16" data-open="dsx019-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx019-B32" data-open="dsx019-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> due to the presence of frequent informative polymorphisms, it will be important to carefully evaluate if unintentioned amplified <em>numts</em> could have been a source of unexpected variability in supposed mtDNA sequences. This problem could be relevant not only for the D-loop region but also for all other informative mtDNA regions used for different purposes. A recent analysis of livestock mtDNA sequences deposited in DNA databases<a reveal-id="dsx019-B33" data-open="dsx019-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> reported that of 127 near complete porcine mtDNA sequences available in GenBank, about 32% contained errors. For almost half of these problematic sequences the surplus of mutations observed in these entries was probably due to contamination from <em>numt</em> sequences. This problem might be also present in many other partial porcine mtDNA sequences deposited in public databases. As <em>numts</em> are not under strong selective constrains of the mitochondrial genome, they might usually contain pseudogene signatures (if they come from protein-coding regions) that can be used to distinguish them from true mtDNA sequencs (i.e. frameshift mutations, premature stop codons, inserions and deletions). However, recent transferred sequences that have not accululated these characteristics yet and <em>numts</em> from non-coding mtDNA regions might be more cryptic to be detected.<a reveal-id="dsx019-B7" data-open="dsx019-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> A closer inspection of this possible source of biases in mtDNA studies can be now possible using as reference the porcine <em>numt</em> sequences we identified.</p><p>Insertion position of porcine <em>numts</em> seems to co-localize with two classes of retrotransposon sequences (i.e. SINE and LINE). Similar co-localization on repeated regions has been reported for <em>numts</em> in a few mammalian genomes including the human genome,<a reveal-id="dsx019-B20" data-open="dsx019-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,<a reveal-id="dsx019-B23" data-open="dsx019-B23" class="link link-ref link-reveal xref-bibr">23</a></sup> further confirming a general feature that shapes the distribution of <em>numts</em> in mammals. The precise mechanism by which this occurs is not understood yet. This more frequent co-localization feature can hold even if the position of <em>numts</em> in the reference porcine genome (as well as in all other analysed mammalian genomes) may reflect a non-random sample of <em>numt</em> insertion events that were probably filtered out from deleterious events. This is true considering that only a fraction of the mammalian genome (10–15%) is functional<a reveal-id="dsx019-B34" data-open="dsx019-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and may accept the introduction of novel DNA during its evolutionary processes. All detected porcine <em>numts</em> were inserted in intergenic regions (<em>n</em> = 195) or in intronic regions (<em>n</em> = 51) and a potential functional role of these insertions was not obvious. Althought in most cases these insertions might be neutral, it could be possible that in rare cases <em>numts</em> are inserted into genes as already described in humans, producing genetic diseases or altered phenotypes.<a reveal-id="dsx019-B35" data-open="dsx019-B35" class="link link-ref link-reveal xref-bibr"><sup>35–38</sup></a> Additional <em>S</em><em>.</em><em>scrofa</em> genomes might be investigated to identify other <em>numts</em> not present in the reference genome version used in this study. Dayama <em>et al.</em><a reveal-id="dsx019-B23" data-open="dsx019-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> showed in humans that more recent <em>numt</em> sequences could be discovered when other genomes were analysed for the presence of this type of new sequence integration in the nuclear genome.</p><p>Polymorphic <em>numts</em> (i.e. presence or absence of the inserted mtDNA sequence) were identified to segregate in different pig populations. The presence of allelic <em>numt</em> loci within a species might usually indicate a very recent origin of the insertion events, i.e. after the speciation process. This is what was observed in humans<a reveal-id="dsx019-B23" data-open="dsx019-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and horse,<a reveal-id="dsx019-B12" data-open="dsx019-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> the only other two species in which this phenomenon has been described so far. Surprisingly, the estimated age of insertion of two of the three polymorphic <em>numts</em> was more ancient than that of the speciation time of the <em>S</em><em>.</em><em>scrofa</em>. The insertion ages of <em>numts</em> 8_10 and 13_32 were estimated to be about 5 MYBP (early Pliocene) and about 6.7 MYBP (late Miocene), respectively, i.e. before the differentiation of the <em>Sus</em> genera or just after the constitution of this lineage. Even if our estimation could not be very precise as it is based only on a few diagnostic mtDNA positions, it seems clear that these insertions were not recent (as also confirmed by the general identity of the two <em>numts</em> with the modern mtDNA: 92 and 90%, respectively). This estimation however contradicts the fact that these <em>numt</em> insertions were not fixed in <em>S</em><em>.</em><em>scrofa</em> species. Of the genus <em>Sus</em>, all extant species except <em>S</em><em>.</em><em>scrofa</em> (spread across all Euro-Asian regions) are restricted to ISEA region (<em>S</em><em>.</em><em>barbatus</em>: in Borneo, Malay Peninsula and Sumatra; <em>S</em><em>.</em><em>verrucosus</em>: in Jawa and Bawean; <em>S</em><em>.</em><em>cebifrons</em> and <em>Sus philippensis</em>: Philippines; <em>S</em><em>.</em><em>celebensis</em>: Sulawesi). Frantz <em>et al.</em><a reveal-id="dsx019-B14" data-open="dsx019-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> demonstrated that the evolutionary history of <em>Sus</em> species can be better explained by a reticulate history derived by many episodes of interspecies admixture. Ai <em>et al.</em><a reveal-id="dsx019-B39" data-open="dsx019-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> reported the presence of a peculiar divergent region, spanning several megabases on SSCX, differentiating southern domestic Chinese breeds (more similar to ISEA species) from northern domestic Chinese breeds (clustering with European sequences). This region constitutes a signature of interspecies admixture that contributed to the latitude adaptation of the two groups of Chinese domestic breeds.<a reveal-id="dsx019-B39" data-open="dsx019-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Ancient polymorphic <em>numts</em> we reported in our study further support the possibility that interspecies gene flow has contributed to shape the <em>S</em><em>.</em><em>scrofa</em> genome. Actually, an interesting advantage that <em>numts</em> can provide is derived by their sequence information that contains their approximate age of integration in the nuclear genome. Other <em>numt</em> regions should be tested to obtain a more complete picture of polymorphisms originated during the different events (interspecies admixture or recent insertions) that shaped the modern domestic pig genome and contributed to the constitution of European breeds and lines through different waves of gene flux and introgression between Asian and European domestic populations and breeds.</p>                            <h2 scrollto-destination=106356232 id="106356232" class="section-title" >Accession numbers</h2>
<p>LT707410, LT707411, LT707412, LT707413, LT707414</p>                            <h2 scrollto-destination=106356234 id="106356234" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the Italian Pig Breeders Association (ANAS) for providing samples used for DNA analyses.</p>                            <h3 scrollto-destination=106356236 id="106356236" class="section-title" >Conflict of interest statement</h3>
<p>None declared.</p>                            <h3 scrollto-destination=106356238 id="106356238" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s1.doc?Expires=2147483647&Signature=QlUpHU02~8PPlbH9OJgCQBw5rV1FV1Qbh7t4ljzAbax2VMrB6qNTt9rTiqhwQa3xZdbop5zR-qxTyHA0Q08I31GsVNJa954gs-fzEckxhplXYH4DdqSFWqrDZe29Jxrn2PSRe~GimybF7YQcTJOl3tbbfFvZbL2FCf7n4WYVNmDIxxAmVcAiKKGuX5QOGUuaSyRRN5a7LJlqp6OQIy7jBN7GnL~oQmivn~4zk7JsSv7lAPQ0mU4U36FxqVNmMPCSBEarGmxuHL~p0kt7AfrOYa0WBIcB~ONR-61Ec0HwB70Z5xMUyKabY77oSbvqFuy7srISXrRDcHJmK4BoW6pfFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=106356240 id="106356240" class="section-title" >Funding</h3>
<p>This work was supported by University of Bologna RFO funds and MiPAAF Innovagen project. O.I.H. was supported by OTKA 111964 and COST Action TD1101.</p>                            <h2 scrollto-destination=106356242 id="106356242" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx019-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kazazian</div>
               <div class="given-names">H.H.</div>
            </div>
          </div>
          <div class="year">2004</div>, 
<div class="article-title">Mobile elements: drivers of genome evolution</div>. <div class="source">Science</div> , <div class="volume">303</div>, <div class="fpage">1626</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20elements%3A%20drivers%20of%20genome%20evolution&amp;author=H.H.%20Kazazian&amp;publication_year=2004&amp;journal=Science&amp;volume=303&amp;pages=1626-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1089670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1089670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1089670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15016989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lopez</div>
               <div class="given-names">J. V</div>
            </div>, <div class="name string-name"><div class="surname">Yuhki</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Masuda</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Modi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">O’Brien</div> <div class="given-names">S.J.</div></div></div>
          <div class="year">1994</div>, 
<div class="article-title">Numt, a recent transfer and tandem amplification of mitochondrial DNA to the nuclear genome of the domestic cat</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">39</div>, <div class="fpage">174</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Numt%2C%20a%20recent%20transfer%20and%20tandem%20amplification%20of%20mitochondrial%20DNA%20to%20the%20nuclear%20genome%20of%20the%20domestic%20cat&amp;author=J.%20V%20Lopez&amp;author=N.%20Yuhki&amp;author=R.%20Masuda&amp;author=W.%20Modi&amp;author=S.J.%20O%E2%80%99Brien&amp;publication_year=1994&amp;journal=J.%20Mol.%20Evol&amp;volume=39&amp;pages=174-90" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7932781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cracraft</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Feinstein</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Vaughn</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Helm-Bychowski</div> <div class="given-names">K.</div></div></div>
          <div class="year">1998</div>, 
<div class="article-title">Sorting out tigers (Panthera tigris): mitochondrial sequences, nuclear inserts, systematics, and conservation genetics</div>. <div class="source">Anim. Conserv</div> ., <div class="volume">1</div>, <div class="fpage">139</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sorting%20out%20tigers%20(Panthera%20tigris)%3A%20mitochondrial%20sequences%2C%20nuclear%20inserts%2C%20systematics%2C%20and%20conservation%20genetics&amp;author=J.%20Cracraft&amp;author=J.%20Feinstein&amp;author=J.%20Vaughn&amp;author=K.%20Helm-Bychowski&amp;publication_year=1998&amp;journal=Anim.%20Conserv&amp;volume=1&amp;pages=139-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/acv.1998.1.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Facv.1998.1.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Facv.1998.1.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsx019-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hazkani-Covo</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Zeller</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">W.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Molecular poltergeists: mitochondrial DNA copies (numts) in sequenced nuclear genomes</div>. <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1000834</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20poltergeists%3A%20mitochondrial%20DNA%20copies%20(numts)%20in%20sequenced%20nuclear%20genomes&amp;author=E.%20Hazkani-Covo&amp;author=R.M.%20Zeller&amp;author=W.%20Martin&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1000834" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20168995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Anthony</div>
               <div class="given-names">N.M.</div>
            </div>, <div class="name string-name"><div class="surname">Clifford</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Bawe-Johnson</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Distinguishing gorilla mitochondrial sequences from nuclear integrations and PCR recombinants: guidelines for their diagnosis in complex sequence databases</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">43</div>, <div class="fpage">553</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinguishing%20gorilla%20mitochondrial%20sequences%20from%20nuclear%20integrations%20and%20PCR%20recombinants%3A%20guidelines%20for%20their%20diagnosis%20in%20complex%20sequence%20databases&amp;author=N.M.%20Anthony&amp;author=S.L.%20Clifford&amp;author=M.%20Bawe-Johnson&amp;publication_year=2007&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=43&amp;pages=553-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2006.09.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2006.09.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2006.09.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17084645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wallace</div>
               <div class="given-names">D.C.</div>
            </div>, <div class="name string-name"><div class="surname">Stugard</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Murdock</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Schurr</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">M.D.</div></div></div>
          <div class="year">1997</div>, 
<div class="article-title">Ancient mtDNA sequences in the human nuclear genome: a potential source of errors in identifying pathogenic mutations</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">94</div>, <div class="fpage">14900</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ancient%20mtDNA%20sequences%20in%20the%20human%20nuclear%20genome%3A%20a%20potential%20source%20of%20errors%20in%20identifying%20pathogenic%20mutations&amp;author=D.C.%20Wallace&amp;author=C.%20Stugard&amp;author=D.%20Murdock&amp;author=T.%20Schurr&amp;author=M.D.%20Brown&amp;publication_year=1997&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=94&amp;pages=14900-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.26.14900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.26.14900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.26.14900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9405711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bensasson</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2001</div>, 
<div class="article-title">Mitochondrial pseudogenes: evolution’s misplaced witnesses</div>. <div class="source">Trends Ecol. Evol</div> ., <div class="volume">16</div>, <div class="fpage">314</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20pseudogenes%3A%20evolution%E2%80%99s%20misplaced%20witnesses&amp;author=D.%20Bensasson&amp;publication_year=2001&amp;journal=Trends%20Ecol.%20Evol&amp;volume=16&amp;pages=314-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(01)02151-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(01)02151-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(01)02151-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11369110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Leister</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2005</div>, 
<div class="article-title">Origin, evolution and genetic effects of nuclear insertions of organelle DNA</div>. <div class="source">Trends Genet</div> ., <div class="volume">21</div>, <div class="fpage">655</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%2C%20evolution%20and%20genetic%20effects%20of%20nuclear%20insertions%20of%20organelle%20DNA&amp;author=D.%20Leister&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=655-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.09.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.09.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.09.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16216380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ricchetti</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Fairhead</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dujon</div> <div class="given-names">B.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">Mitochondrial DNA repairs double-strand breaks in yeast chromosomes</div>. <div class="source">Nature</div> , <div class="volume">402</div>, <div class="fpage">96</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20repairs%20double-strand%20breaks%20in%20yeast%20chromosomes&amp;author=M.%20Ricchetti&amp;author=C.%20Fairhead&amp;author=B.%20Dujon&amp;publication_year=1999&amp;journal=Nature&amp;volume=402&amp;pages=96-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/47076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F47076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F47076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10573425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hazkani-Covo</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Graur</div> <div class="given-names">D.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">A comparative analysis of numt evolution in human and chimpanzee</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">13</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20analysis%20of%20numt%20evolution%20in%20human%20and%20chimpanzee&amp;author=E.%20Hazkani-Covo&amp;author=D.%20Graur&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=13-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17056643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pereira</div>
               <div class="given-names">S.L.</div>
            </div>, <div class="name string-name"><div class="surname">Baker</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Margulis</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Low number of mitochondrial pseudogenes in the chicken (<em>Gallus gallus</em>) nuclear genome: implications for molecular inference of population history and phylogenetics</div>. <div class="source">BMC Evol. Biol</div> ., <div class="volume">4</div>, <div class="fpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low%20number%20of%20mitochondrial%20pseudogenes%20in%20the%20chicken%20(Gallus%20gallus)%20nuclear%20genome%3A%20implications%20for%20molecular%20inference%20of%20population%20history%20and%20phylogenetics&amp;author=S.L.%20Pereira&amp;author=A.J.%20Baker&amp;author=L.%20Margulis&amp;publication_year=2004&amp;journal=BMC%20Evol.%20Biol&amp;volume=4&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-4-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-4-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-4-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15219233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nergadze</div>
               <div class="given-names">S.G.</div>
            </div>, <div class="name string-name"><div class="surname">Lupotto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pellanda</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Santagostino</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Vitelli</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Giulotto</div> <div class="given-names">E.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Mitochondrial DNA insertions in the nuclear horse genome</div>. <div class="source">Anim. Genet</div> ., <div class="volume">41(Suppl 2)</div>, <div class="fpage">176</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20insertions%20in%20the%20nuclear%20horse%20genome&amp;author=S.G.%20Nergadze&amp;author=M.%20Lupotto&amp;author=P.%20Pellanda&amp;author=M.%20Santagostino&amp;author=V.%20Vitelli&amp;author=E.%20Giulotto&amp;publication_year=2010&amp;journal=Anim.%20Genet&amp;volume=41(Suppl 2)&amp;pages=176-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2010.41.issue-s2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2010.41.issue-s2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2010.41.issue-s2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21070293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Groenen</div>
               <div class="given-names">M.A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Archibald</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Uenishi</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Analyses of pig genomes provide insight into porcine demography and evolution</div>. <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">393</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyses%20of%20pig%20genomes%20provide%20insight%20into%20porcine%20demography%20and%20evolution&amp;author=M.A.M.%20Groenen&amp;author=A.L.%20Archibald&amp;author=H.%20Uenishi&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=393-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23151582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Frantz</div>
               <div class="given-names">L.A.F.</div>
            </div>, <div class="name string-name"><div class="surname">Schraiber</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">O.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genome sequencing reveals fine scale diversification and reticulation history during speciation in Sus</div>. <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20reveals%20fine%20scale%20diversification%20and%20reticulation%20history%20during%20speciation%20in%20Sus&amp;author=L.A.F.%20Frantz&amp;author=J.G.%20Schraiber&amp;author=O.%20Madsen&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-9-r107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-9-r107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-9-r107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24070215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Giuffra</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Kijas</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Amarger</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Carlborg</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Jeon</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Andersson</div> <div class="given-names">L.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">The origin of the domestic pig: independent domestication and subsequent introgression</div>. <div class="source">Genetics</div> , <div class="volume">154</div>, <div class="fpage">1785</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20the%20domestic%20pig%3A%20independent%20domestication%20and%20subsequent%20introgression&amp;author=E.%20Giuffra&amp;author=J.M.%20Kijas&amp;author=V.%20Amarger&amp;author=O.%20Carlborg&amp;author=J.T.%20Jeon&amp;author=L.%20Andersson&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1785-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larson</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Dobney</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Albarella</div> <div class="given-names">U.</div></div></div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Worldwide phylogeography of wild boar reveals multiple centers of pig domestication</div>. <div class="source">Science</div> , <div class="volume">307</div>, <div class="fpage">1618</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worldwide%20phylogeography%20of%20wild%20boar%20reveals%20multiple%20centers%20of%20pig%20domestication&amp;author=G.%20Larson&amp;author=K.%20Dobney&amp;author=U.%20Albarella&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1618-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1106927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1106927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1106927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15761152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bosse</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Megens</div> <div class="given-names">H.-J.</div></div>, <div class="name string-name"><div class="surname">Frantz</div> <div class="given-names">L.A.F.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genomic analysis reveals selection for Asian genes in European pigs following human-mediated introgression</div>. <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">4392</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20reveals%20selection%20for%20Asian%20genes%20in%20European%20pigs%20following%20human-mediated%20introgression&amp;author=M.%20Bosse&amp;author=H.-J.%20Megens&amp;author=L.A.F.%20Frantz&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=4392" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms5392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms5392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms5392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25025832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kiełbasa</div>
               <div class="given-names">S.M.</div>
            </div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Horton</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Adaptive seeds tame genomic sequence comparison</div>. <div class="source">Genome Res</div> ., <div class="volume">21</div>, <div class="fpage">487</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20seeds%20tame%20genomic%20sequence%20comparison&amp;author=S.M.%20Kie%C5%82basa&amp;author=R.%20Wan&amp;author=K.%20Sato&amp;author=P.%20Horton&amp;author=M.C.%20Frith&amp;publication_year=2011&amp;journal=Genome%20Res&amp;volume=21&amp;pages=487-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.113985.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.113985.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.113985.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21209072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Frith</div>
               <div class="given-names">M. C.</div>
            </div>, <div class="name string-name"><div class="surname">Hamada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Horton</div> <div class="given-names">P.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Parameters for accurate genome alignment</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parameters%20for%20accurate%20genome%20alignment&amp;author=M.%20C.%20Frith&amp;author=M.%20Hamada&amp;author=P.%20Horton&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-80"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20144198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsuji</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Tomii</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Horton</div> <div class="given-names">P.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Mammalian NUMT insertion is non-random</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">9073</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian%20NUMT%20insertion%20is%20non-random&amp;author=J.%20Tsuji&amp;author=M.%20C.%20Frith&amp;author=K.%20Tomii&amp;author=P.%20Horton&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=9073-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22761406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Frantz</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Meijaard</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Gongora</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Haile</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Groenen</div> <div class="given-names">M.A.M.</div></div>, <div class="name string-name"><div class="surname">Larson</div> <div class="given-names">G.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">The Evolution of Suidae</div>. <div class="source">Annu. Rev. Anim. Biosci</div> ., <div class="volume">4</div>, <div class="fpage">61</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Evolution%20of%20Suidae&amp;author=L.%20Frantz&amp;author=E.%20Meijaard&amp;author=J.%20Gongora&amp;author=J.%20Haile&amp;author=M.A.M.%20Groenen&amp;author=G.%20Larson&amp;publication_year=2016&amp;journal=Annu.%20Rev.%20Anim.%20Biosci&amp;volume=4&amp;pages=61-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-animal-021815-111155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-animal-021815-111155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-animal-021815-111155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26526544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">1870</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA7%3A%20molecular%20evolutionary%20genetics%20analysis%20version%207.0%20for%20bigger%20datasets&amp;author=S.%20Kumar&amp;author=G.%20Stecher&amp;author=K.%20Tamura&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=1870-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dayama</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Emery</div> <div class="given-names">S. B.</div></div>, <div class="name string-name"><div class="surname">Kidd</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">The genomic landscape of polymorphic human nuclear mitochondrial insertions</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">12640</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20landscape%20of%20polymorphic%20human%20nuclear%20mitochondrial%20insertions&amp;author=G.%20Dayama&amp;author=S.%20B.%20Emery&amp;author=J.M.%20Kidd&amp;author=R.E.%20Mills&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=12640-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25348406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rothschild</div>
               <div class="given-names">M.F.</div>
            </div>, <div class="name string-name"><div class="surname">Anatoly</div> <div class="given-names">R.</div></div></div> eds. <div class="year">2011</div>, <div class="source">The Genetics of the Pig</div> . 
<div class="publisher-loc">Cambridge</div>: 
<div class="publisher-name">CABI</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genetics%20of%20the%20Pig&amp;author=M.F.%20Rothschild&amp;author=R.%20Anatoly&amp;publication_year=2011&amp;book=The%20Genetics%20of%20the%20Pig" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1079/9781845937560.0000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1079%2F9781845937560.0000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1079%2F9781845937560.0000"> </span></div> </div></div></div></div></div><div content-id="dsx019-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Deininger</div>
               <div class="given-names">P.L.</div>
            </div>, <div class="name string-name"><div class="surname">Moran</div> <div class="given-names">J.V</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Mobile elements and mammalian genome evolution</div>. <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">13</div>, <div class="fpage">651</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20elements%20and%20mammalian%20genome%20evolution&amp;author=P.L.%20Deininger&amp;author=J.V%20Moran&amp;author=M.A.%20Batzer&amp;author=H.H.%20Kazazian&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=13&amp;pages=651-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2003.10.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2003.10.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2003.10.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14638329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Okada</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Shimamura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yasue</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">1997</div>, 
<div class="article-title">Molecular evidence from retroposons that whales form a clade within even-toed ungulates</div>. <div class="source">Nature</div> , <div class="volume">388</div>, <div class="fpage">666</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evidence%20from%20retroposons%20that%20whales%20form%20a%20clade%20within%20even-toed%20ungulates&amp;author=N.%20Okada&amp;author=M.%20Shimamura&amp;author=H.%20Yasue&amp;publication_year=1997&amp;journal=Nature&amp;volume=388&amp;pages=666-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/41759" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F41759" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F41759"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9262399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shedlock</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Okada</div> <div class="given-names">N.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">SINE insertions: powerful tools for molecular systematics</div>. <div class="source">Bioessays</div> , <div class="volume">22</div>, <div class="fpage">148</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SINE%20insertions%3A%20powerful%20tools%20for%20molecular%20systematics&amp;author=A.M.%20Shedlock&amp;author=N.%20Okada&amp;publication_year=2000&amp;journal=Bioessays&amp;volume=22&amp;pages=148-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(SICI)1521-1878(200002)22:2&lt;&gt;1.0.CO;2-O" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(SICI)1521-1878(200002)22:2&lt;&gt;1.0.CO;2-O" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(SICI)1521-1878(200002)22:2&lt;&gt;1.0.CO;2-O"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10655034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chessa</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Pereira</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Arnaud</div> <div class="given-names">F.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Revealing the history of sheep domestication using retrovirus integrations</div>. <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">532</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Revealing%20the%20history%20of%20sheep%20domestication%20using%20retrovirus%20integrations&amp;author=B.%20Chessa&amp;author=F.%20Pereira&amp;author=F.%20Arnaud&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=532-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170587" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170587" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170587"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Deiningera</div>
               <div class="given-names">P.L.</div>
            </div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">Alu repeats and human disease</div>. <div class="source">Mol. Genet. Metab</div> ., <div class="volume">67</div>, <div class="fpage">183</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alu%20repeats%20and%20human%20disease&amp;author=P.L.%20Deiningera&amp;author=M.A.%20Batzer&amp;publication_year=1999&amp;journal=Mol.%20Genet.%20Metab&amp;volume=67&amp;pages=183-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mgme.1999.2864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmgme.1999.2864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmgme.1999.2864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10381326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maksakova</div>
               <div class="given-names">I.A.</div>
            </div>, <div class="name string-name"><div class="surname">Romanish</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Gagnier</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Dunn</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">van de Lagemaat</div> <div class="given-names">L.N.</div></div>, <div class="name string-name"><div class="surname">Mager</div> <div class="given-names">D.L.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Retroviral elements and their hosts: insertional mutagenesis in the mouse germ line</div>. <div class="source">PLoS Genet</div> ., <div class="volume">2</div>, <div class="fpage">e2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retroviral%20elements%20and%20their%20hosts%3A%20insertional%20mutagenesis%20in%20the%20mouse%20germ%20line&amp;author=I.A.%20Maksakova&amp;author=M.T.%20Romanish&amp;author=L.%20Gagnier&amp;author=C.A.%20Dunn&amp;author=L.N.%20van%20de%20Lagemaat&amp;author=D.L.%20Mager&amp;publication_year=2006&amp;journal=PLoS%20Genet&amp;volume=2&amp;pages=e2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16440055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sironen</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Thomsen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Andersson</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ahola</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Vilkki</div> <div class="given-names">J.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">An intronic insertion in KPL2 results in aberrant splicing and causes the immotile short-tail sperm defect in the pig</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">103</div>, <div class="fpage">5006</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20intronic%20insertion%20in%20KPL2%20results%20in%20aberrant%20splicing%20and%20causes%20the%20immotile%20short-tail%20sperm%20defect%20in%20the%20pig&amp;author=A.%20Sironen&amp;author=B.%20Thomsen&amp;author=M.%20Andersson&amp;author=V.%20Ahola&amp;author=J.%20Vilkki&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=103&amp;pages=5006-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506318103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506318103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506318103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16549801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">K.I.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2002</div>, 
<div class="article-title">Phylogenetic relationships of Asian and European pig breeds determined by mitochondrial DNA D-loop sequence polymorphism</div>. <div class="source">Anim. Genet</div> ., <div class="volume">33</div>, <div class="fpage">19</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20of%20Asian%20and%20European%20pig%20breeds%20determined%20by%20mitochondrial%20DNA%20D-loop%20sequence%20polymorphism&amp;author=K.I.%20Kim&amp;author=J.H.%20Lee&amp;author=K.%20Li&amp;publication_year=2002&amp;journal=Anim.%20Genet&amp;volume=33&amp;pages=19-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2052.2002.00784.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2052.2002.00784.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2052.2002.00784.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11849133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shi</div>
               <div class="given-names">N.-N.</div>
            </div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">Y.-G.</div></div>, <div class="name string-name"><div class="surname">Peng</div> <div class="given-names">M.-S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.-P.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Mitochondrial genomes of domestic animals need scrutiny</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">23</div>, <div class="fpage">5393</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20genomes%20of%20domestic%20animals%20need%20scrutiny&amp;author=N.-N.%20Shi&amp;author=L.%20Fan&amp;author=Y.-G.%20Yao&amp;author=M.-S.%20Peng&amp;author=Y.-P.%20Zhang&amp;publication_year=2014&amp;journal=Mol.%20Ecol&amp;volume=23&amp;pages=5393-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25294192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ponting</div>
               <div class="given-names">C.P.</div>
            </div>, <div class="name string-name"><div class="surname">Hardison</div> <div class="given-names">R.C.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">What fraction of the human genome is functional?</div><div class="source">Genome Res</div> ., <div class="volume">21</div>, <div class="fpage">1769</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20fraction%20of%20the%20human%20genome%20is%20functional%3F&amp;author=C.P.%20Ponting&amp;author=R.C.%20Hardison&amp;publication_year=2011&amp;journal=Genome%20Res&amp;volume=21&amp;pages=1769-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.116814.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.116814.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.116814.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21875934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Borensztajn</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Chafa</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Alhenc-Gelas</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2002</div>, 
<div class="article-title">Characterization of two novel splice site mutations in human factor VII gene causing severe plasma factor VII deficiency and bleeding diathesis</div>. <div class="source">Br. J. Haematol</div> ., <div class="volume">117</div>, <div class="fpage">168</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20two%20novel%20splice%20site%20mutations%20in%20human%20factor%20VII%20gene%20causing%20severe%20plasma%20factor%20VII%20deficiency%20and%20bleeding%20diathesis&amp;author=K.%20Borensztajn&amp;author=O.%20Chafa&amp;author=M.%20Alhenc-Gelas&amp;publication_year=2002&amp;journal=Br.%20J.%20Haematol&amp;volume=117&amp;pages=168-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2141.2002.03397.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2141.2002.03397.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2141.2002.03397.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11918550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Killoran</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">N.S.T.</div></div></div>, et al.  <div class="year">2003</div>, 
<div class="article-title">Human genetic disease caused by de novo mitochondrial-nuclear DNA transfer</div>. <div class="source">Hum. Genet</div> ., <div class="volume">112</div>, <div class="fpage">303</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20genetic%20disease%20caused%20by%20de%20novo%20mitochondrial-nuclear%20DNA%20transfer&amp;author=C.%20Turner&amp;author=C.%20Killoran&amp;author=N.S.T.%20Thomas&amp;publication_year=2003&amp;journal=Hum.%20Genet&amp;volume=112&amp;pages=303-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12545275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goldin</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Stahl</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cooney</div> <div class="given-names">A.M.</div></div></div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Transfer of a mitochondrial DNA fragment to MCOLN1 causes an inherited case of mucolipidosis IV</div>. <div class="source">Hum. Mutat</div> ., <div class="volume">24</div>, <div class="fpage">460</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20of%20a%20mitochondrial%20DNA%20fragment%20to%20MCOLN1%20causes%20an%20inherited%20case%20of%20mucolipidosis%20IV&amp;author=E.%20Goldin&amp;author=S.%20Stahl&amp;author=A.M.%20Cooney&amp;publication_year=2004&amp;journal=Hum.%20Mutat&amp;volume=24&amp;pages=460-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15523648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">J.-M.</div>
            </div>, <div class="name string-name"><div class="surname">Chuzhanova</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Stenson</div> <div class="given-names">P.D.</div></div>, <div class="name string-name"><div class="surname">Férec</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Meta-analysis of gross insertions causing human genetic disease: novel mutational mechanisms and the role of replication slippage</div>. <div class="source">Hum. Mutat</div> ., <div class="volume">25</div>, <div class="fpage">207</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meta-analysis%20of%20gross%20insertions%20causing%20human%20genetic%20disease%3A%20novel%20mutational%20mechanisms%20and%20the%20role%20of%20replication%20slippage&amp;author=J.-M.%20Chen&amp;author=N.%20Chuzhanova&amp;author=P.D.%20Stenson&amp;author=C.%20F%C3%A9rec&amp;author=D.N.%20Cooper&amp;publication_year=2005&amp;journal=Hum.%20Mutat&amp;volume=25&amp;pages=207-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.20133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.20133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.20133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15643617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx019-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx019-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ai</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">B.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing</div>. <div class="source">Nat. Genet</div> ., <div class="volume">47</div>, <div class="fpage">217</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20and%20possible%20ancient%20interspecies%20introgression%20in%20pigs%20identified%20by%20whole-genome%20sequencing&amp;author=H.%20Ai&amp;author=X.%20Fang&amp;author=B.%20Yang&amp;publication_year=2015&amp;journal=Nat.%20Genet&amp;volume=47&amp;pages=217-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3778238&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=QlUpHU02~8PPlbH9OJgCQBw5rV1FV1Qbh7t4ljzAbax2VMrB6qNTt9rTiqhwQa3xZdbop5zR-qxTyHA0Q08I31GsVNJa954gs-fzEckxhplXYH4DdqSFWqrDZe29Jxrn2PSRe~GimybF7YQcTJOl3tbbfFvZbL2FCf7n4WYVNmDIxxAmVcAiKKGuX5QOGUuaSyRRN5a7LJlqp6OQIy7jBN7GnL~oQmivn~4zk7JsSv7lAPQ0mU4U36FxqVNmMPCSBEarGmxuHL~p0kt7AfrOYa0WBIcB~ONR-61Ec0HwB70Z5xMUyKabY77oSbvqFuy7srISXrRDcHJmK4BoW6pfFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s2.doc?Expires=2147483647&amp;Signature=lqYxwTwZAu-bdU6ae-6FWFSVHbo0e5ex1CX9Idj6-AzW0E9Uk6iZdkSvGJ4trdEnmmfdIhFlLbzRKQwLX-~USJGSvnUgHohQjaw9gsqZ4O~lg4DKMZUGugn5VUhDRVDrhClK7uNVs3FXg8~KGUdtjZFZCzNj0HB1y~r2hthcR0kq04ERa73i-THID3pTwscQS1OfoK3r25y8ytmdHB57swj5npOu5z391Bfo3JWpYrYClG1TXnP0BlSKwfb4WkExBFb590TFa23KS-3JBe-zvw62bWMO2DJCkhTGsArZQQbXRwFfbaI4Iz2y6WWqeuazHrM-eROwZVeC61zMenGI9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_file_s3.pdf?Expires=2147483647&amp;Signature=Cf0Jl2rOurbv3cXHw85k-4oQbk9OOPSYtNohcpHbSMoL90SefFrntxP1uvuvRo6g7-iDoqfKC7vk-CNx3d7STbjCcWU1dA454~sMRFvw3cLw27pLeOSq2JSPqbtgqryvhzwn5O3vzI7-LDTDLR5CDzbTcDrJ5l-FTEnovK74WnSvCgi3dMh4vJg6SDd-GpEdGYepaUoR933~t9~QlSrK4eHLNEeXybjbEI9lxSBH88OpfnG5GBFfUOoFNV7W3kteroRfhOoNPxpcyRIkrHdUtzGaysNTfr7vJYAqjZaorZaEaGo4CiBOLuhPAWRX22-aJqyoUlLOXAc7RIYThV5JsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx019/2/dsx019_supplementary_tables.doc?Expires=2147483647&amp;Signature=kTD8FQNB0xU8JsUNQge0kqntE8c4sczbZtNGvdClj~InF7SBBt7mYL7fkT4C-KatEA45u-XRTLZXyBxp6ICWmRmzX6lx4rGe6vaZ2zlxOoC5itdSbh7r91T9ATy6VriMI~G-TiG7D85gDNIr3WCoEIkEO-4YVPflFwq4jbsAn4h~t~uYLp0fCUSm2N8ZopUUEB2~-E5OHLyqL6iz71ZiRL7bMQGLF7s6Kw8ID0YU9Ajzbt5kb4-q1PCAR1arJeXdpdGsI79Ea3iISPKfH-NzosbyWWc3kFnyWyiXddCVS9GxXQGZQlL9kyi2RS7v1YCYhbHwxfpfu4UDnxqC-DKvMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">851</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3778238"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f5%2f487%2f3778238">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000413018400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F5%2F487%2F3778238" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3778238'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="genomic landscape of mitochondrial DNA insertions in the pig nuclear genome provides evolutionary signatures of interspecies admixture" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f487%2f3778238">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

