

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/6/499/478052","@type":"ScholarlyArticle","name":"Mutation Bias is the Driving Force of Codon Usage in the  Gallus gallus  genome","datePublished":"2011-10-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2011-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr035","keywords":["<italic>Gallus gallus</italic>","codon usage","mutation","selection"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/6/499/478052","author":[{"name":"Rao, Yousheng","affiliation":"1  Department of Biological Technology ,  Jiangxi Educational Institute ,  Nanchang, Jiangxi ,  China    2  Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding ,  South China Agricultural University ,  Guangzhou, Guangdong ,  China","@type":"Person"},{"name":"Wu, Guozuo","affiliation":"1  Department of Biological Technology ,  Jiangxi Educational Institute ,  Nanchang, Jiangxi ,  China","@type":"Person"},{"name":"Wang, Zhangfeng","affiliation":"1  Department of Biological Technology ,  Jiangxi Educational Institute ,  Nanchang, Jiangxi ,  China","@type":"Person"},{"name":"Chai, Xuewen","affiliation":"1  Department of Biological Technology ,  Jiangxi Educational Institute ,  Nanchang, Jiangxi ,  China","@type":"Person"},{"name":"Nie, Qinghua","affiliation":"2  Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding ,  South China Agricultural University ,  Guangzhou, Guangdong ,  China    3  College of Animal Science ,  South China Agricultural University ,  Guangzhou, Guangdong ,  China","@type":"Person"},{"name":"Zhang, Xiquan","affiliation":"2  Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding ,  South China Agricultural University ,  Guangzhou, Guangdong ,  China    3  College of Animal Science ,  South China Agricultural University ,  Guangzhou, Guangdong ,  China","@type":"Person"}],"description":"Abstract.  Synonymous codons are used with different frequencies both among species and among genes within the same genome and are controlled by neutral process","pageStart":"499","pageEnd":"512","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Mutation Bias is the Driving Force of Codon Usage in the  Gallus gallus  genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Mutation Bias is the Driving Force of Codon Usage in the  Gallus gallus  genome" />
    <meta property="og:description" content="Abstract.  Synonymous codons are used with different frequencies both among species and among genes within the same genome and are controlled by neutral process" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/6/499/478052" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-10-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Rao, Yousheng" /><meta name="citation_author_institution" content="Department of Biological Technology, Jiangxi Educational Institute, Nanchang, Jiangxi, China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding, South China Agricultural University, Guangzhou, Guangdong, China" /><meta name="citation_author" content="Wu, Guozuo" /><meta name="citation_author_institution" content="Department of Biological Technology, Jiangxi Educational Institute, Nanchang, Jiangxi, China" /><meta name="citation_author" content="Wang, Zhangfeng" /><meta name="citation_author_institution" content="Department of Biological Technology, Jiangxi Educational Institute, Nanchang, Jiangxi, China" /><meta name="citation_author" content="Chai, Xuewen" /><meta name="citation_author_institution" content="Department of Biological Technology, Jiangxi Educational Institute, Nanchang, Jiangxi, China" /><meta name="citation_author" content="Nie, Qinghua" /><meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding, South China Agricultural University, Guangzhou, Guangdong, China" /><meta name="citation_author_institution" content="College of Animal Science, South China Agricultural University, Guangzhou, Guangdong, China" /><meta name="citation_author" content="Zhang, Xiquan" /><meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding, South China Agricultural University, Guangzhou, Guangdong, China" /><meta name="citation_author_institution" content="College of Animal Science, South China Agricultural University, Guangzhou, Guangdong, China" /><meta name="citation_title" content="Mutation Bias is the Driving Force of Codon Usage in the Gallus gallus genome" /><meta name="citation_firstpage" content="499" /><meta name="citation_lastpage" content="512" /><meta name="citation_doi" content="10.1093/dnares/dsr035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2011/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Codon usage in bacteria: correlation with gene expressivity; Nucleic Acids Res.;  citation_year=1982;  citation_volume=10;  citation_pages=7055-74; " /><meta name="citation_reference" content="citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis; Proc. Natl. Acad. Sci.;  citation_year=1999;  citation_volume=96;  citation_pages=4482-7; " /><meta name="citation_reference" content="citation_title=Global analysis of protein expression in yeast; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=737-41; " /><meta name="citation_reference" content="citation_title=Correlation of codon bias measures with mRNA levels: analysis of transcriptome data from Escherichia coli; Biochem. Biophys. Res. Commun.;  citation_year=2005;  citation_volume=327;  citation_pages=4-7; " /><meta name="citation_reference" content="citation_title=Gene expression and protein length influence codon usage and rates of sequence evolution in Populus tremula; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=836-44; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multi-cellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon preferences in free-living microorganisms; Microbiol. Rev.;  citation_year=1990;  citation_volume=54;  citation_pages=198-210; " /><meta name="citation_reference" content="citation_title=Transfer RNA gene redundancy and translational selection in Saccharomyces cerevisiae; J. Mol. Biol.;  citation_year=1997;  citation_volume=268;  citation_pages=322-30; " /><meta name="citation_reference" content="citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis; Gene;  citation_year=1999;  citation_volume=238;  citation_pages=143-55; " /><meta name="citation_reference" content="citation_title=Levels of tRNAs in bacterial cells as affected by amino acid usage in proteins; Nucleic Acids Res.;  citation_year=1991;  citation_volume=19;  citation_pages=6119-22; " /><meta name="citation_reference" content="citation_title=tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes; Trends Genet.;  citation_year=2000;  citation_volume=16;  citation_pages=287-9; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis; J. Mol. Evol.;  citation_year=2001;  citation_volume=53;  citation_pages=290-8; " /><meta name="citation_reference" content="citation_title=Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2279-86; " /><meta name="citation_reference" content="citation_title=Codon usage determines translation rate in Escherichia coli.; J. Mol. Biol.;  citation_year=1989;  citation_volume=207;  citation_pages=365-77; " /><meta name="citation_reference" content="citation_title=Evolution of synonymous codon usage in metazoans; Curr. Opin. Genet. Dev.;  citation_year=2002;  citation_volume=12;  citation_pages=640-9; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=374-81; " /><meta name="citation_reference" content="citation_title=Selection on codon bias; Annu. Rev. Genet.;  citation_year=2008;  citation_volume=42;  citation_pages=287-99; " /><meta name="citation_reference" content="citation_title=Forces that influence the evolution of codon bias; Phil. Trans. R. Soc. B;  citation_year=2010;  citation_volume=1544;  citation_pages=1203-12; " /><meta name="citation_reference" content="citation_title=A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes; citation_text=RESEARCH0010; Genome Biol.;  citation_year=2001;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=Codon usage between genomes is constrained by genome-wide mutational processes; Proc. Natl. Acad. Sci.;  citation_year=2004;  citation_volume=101;  citation_pages=3480-5; " /><meta name="citation_reference" content="citation_title=General rules for optimal codon choice; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Evidence of selectively driven codon usage in rice: implications for G + C content evolution of Gramineae genes; FEBS Lett.;  citation_year=2007;  citation_volume=581;  citation_pages=1015-21; " /><meta name="citation_reference" content="citation_title=The selection-mutation-drift theory of synonymous codon usage; Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=897-907; " /><meta name="citation_reference" content="citation_title=Inferring weak selection from patterns of polymorphism and divergence at ‘silent’ sites in Drosophila DNA; Genetics;  citation_year=1995;  citation_volume=139;  citation_pages=1067-76; " /><meta name="citation_reference" content="citation_title=Mutation pressure, natural selection, and the evolution of base composition in Drosophila; Genetica;  citation_year=1998;  citation_volume=102–103;  citation_pages=49-60; " /><meta name="citation_reference" content="citation_title=Estimating translational selection in eukaryotic genomes; Mol. Biol. Evol.;  citation_year=2009;  citation_volume=26;  citation_pages=451-61; " /><meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms; J. Mol. Evol.;  citation_year=1986;  citation_volume=24;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=Reduced natural selection associated with low recombination in Drosophila melanogaster; Mol. Biol. Evol.;  citation_year=1993;  citation_volume=10;  citation_pages=1239-58; " /><meta name="citation_reference" content="citation_title=Natural selection on synonymous sites is correlated with gene length and recombination in Drosophila; Genetics;  citation_year=1999;  citation_volume=151;  citation_pages=239-49; " /><meta name="citation_reference" content="citation_title=Population, evolutionary and genomic consequences of interference selection; Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=389-410; " /><meta name="citation_reference" content="citation_title=Interactions between natural selection, recombination and gene density in the genes of Drosophila; Genetics;  citation_year=2002;  citation_volume=160;  citation_pages=595-608; " /><meta name="citation_reference" content="citation_title=Hill-Robertson interference is a minor determinant of variations in codon bias across Drosophila melanogaster and Caenorhabditis elegans genome; Mol. Biol. Evol.;  citation_year=2002;  citation_volume=19;  citation_pages=1399-406; " /><meta name="citation_reference" content="citation_title=Neutral effect of recombination on base composition in Drosophila; Genet. Res.;  citation_year=2003;  citation_volume=81;  citation_pages=79-87; " /><meta name="citation_reference" content="citation_title=Reduced efficacy of selection in regions of the Drosophila genome that lack crossing over; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R18" /><meta name="citation_reference" content="citation_title=Biased gene conversion affects patterns of codon usage and amino acid usage in the Saccharomyces sens stricto group of yeasts; Mol. Biol. Evol.;  citation_year=2010;  citation_volume=28;  citation_pages=117-29; " /><meta name="citation_reference" content="citation_title=Does recombination improve selection on codon usage? Lessons from nematode and fly complete genomes; Proc. Natl. Acad. Sci.;  citation_year=2001;  citation_volume=98;  citation_pages=5688-92; " /><meta name="citation_reference" content="citation_title=Gene length and codon usage bias in Drosophila melanogaster, Saccharomyces cerevisiae and Escherichia coli.; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=3188-93; " /><meta name="citation_reference" content="citation_title=Evolution of proteins and gene expression levels are coupled in Drosophila and are independently associated with mRNA abundance, protein length and number of protein-protein interactions; Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=1345-54; " /><meta name="citation_reference" content="citation_title=Adaptive basis of codon usage in the haploid moss Physcomitrella patens; Heredity;  citation_year=2005;  citation_volume=94;  citation_pages=87-93; " /><meta name="citation_reference" content="citation_title=A consensus linkage map of the chicken genome; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=137-47; " /><meta name="citation_reference" content="citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution; citation_author=International Chicken Genome Sequencing Consortium (ICGSC).; Nature;  citation_year=2004;  citation_volume=432;  citation_pages=695-716; " /><meta name="citation_reference" content="citation_title=Strong regional biases in nucleotide substitution in the chicken genome; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1203-216; " /><meta name="citation_reference" content="citation_title=Isochores and the evolutionary genomics of vertebrates; Gene;  citation_year=2000;  citation_volume=241;  citation_pages=3-17; " /><meta name="citation_reference" content="citation_title=Correlations between coding and contiguous non-coding sequences in isochore families from vertebrate genomes; Gene;  citation_year=2008;  citation_volume=410;  citation_pages=241-8; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; citation_author=International Human Genome Sequencing Consortium.; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Rapid evolution of female-biased, but not male-biased, genes expressed in the avian brain; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=2698-706; " /><meta name="citation_reference" content="citation_title=Selection for short introns in highly expressed genes; Nat. Genet.;  citation_year=2002;  citation_volume=31;  citation_pages=415-8; " /><meta name="citation_reference" content="citation_title=Selection for the compactness of highly expressed genes in Gallus gallus; Biology Direct;  citation_year=2010;  citation_volume=5;  citation_pages=35" /><meta name="citation_reference" content="citation_title=Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=650-9; " /><meta name="citation_reference" content="citation_title=Selection intensity on preferred codons correlates with overall codon usage bias in Caenorhabditis remanei; Curr. Biol.;  citation_year=2006;  citation_volume=16;  citation_pages=2053-7; " /><meta name="citation_reference" content="citation_title=A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=510-9; " /><meta name="citation_reference" content="citation_title=Microsatellite variation and recombination rate in the human genome; Genetics;  citation_year=2000;  citation_volume=156;  citation_pages=1285-98; " /><meta name="citation_reference" content="citation_title=A simple sequentially rejective Bonferroni test procedure; Scand. J. Stat.;  citation_year=1979;  citation_volume=6;  citation_pages=65-70; " /><meta name="citation_reference" content="citation_title=Co-variation of tRNA abundance and codon usage in Escherichia coli at different growth rates; J. Mol. Biol.;  citation_year=1996;  citation_volume=260;  citation_pages=649-63; " /><meta name="citation_reference" content="citation_title=Selective and mutational patterns associated with gene expression in humans: influences on synonymous composition and intron presence; Genetics;  citation_year=2004;  citation_volume=167;  citation_pages=1293-1304; " /><meta name="citation_reference" content="citation_title=Codon bias as a factor in regulating expression via translation rate in the human genome; Gene;  citation_year=2005;  citation_volume=345;  citation_pages=127-38; " /><meta name="citation_reference" content="citation_title=Effect of correlated tRNA abundances on translation errors and evolution of codon usage bias; PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1001128" /><meta name="citation_reference" content="citation_title=On the evolution and expression of Chlamydomonas reinhardtii nucleus-encoded transfer RNA genes; Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=113-23; " /><meta name="citation_reference" content="citation_title=Clustering of codons with rare cognate tRNAs in human genes suggests an extra level of expression regulation; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000548" /><meta name="citation_reference" content="citation_title=Gene expression and molecular evolution; Curr. Opin. Genet. Dev.;  citation_year=2001;  citation_volume=11;  citation_pages=660-6; " /><meta name="citation_reference" content="citation_title=Codon usage bias covaries with expression breadth and the rate of synonymous evolution in humans, but this is not evidence for selection; Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=1191-9; " /><meta name="citation_reference" content="citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis; Proc. Natl. Acad. Sci.;  citation_year=1999;  citation_volume=96;  citation_pages=4482-7; " /><meta name="citation_reference" content="citation_title=Background selection in single genes may explain patterns of codon bias; Genetics;  citation_year=2007;  citation_volume=175;  citation_pages=1381-93; " /><meta name="citation_reference" content="citation_title=Patterns of codon usage bias in Silene latifolia; Mol. Biol. Evol.;  citation_year=2010;  citation_volume=28;  citation_pages=771-80; " /><meta name="citation_reference" content="citation_title=Synonymous codon bias is related to gene length in Escherichia coli: selection for translational accuracy?; Mol. Biol. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=864-72; " /><meta name="citation_reference" content="citation_title=Intron size and exon ewolution in Drosophila; Genetics;  citation_year=2005;  citation_volume=170;  citation_pages=481-85; " /><meta name="citation_reference" content="citation_title=Codon usage in Caenorhabditis elegans: delineation of translational selection and mutational biases; Nucleic Acids Res.;  citation_year=1994;  citation_volume=13;  citation_pages=2437-46; " /><meta name="citation_reference" content="citation_title=Intron length and codon usage; J. Mol. Evol.;  citation_year=2001;  citation_volume=52;  citation_pages=2-5; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=374-81; " /><meta name="citation_reference" content="citation_title=The surprising negative correlation of gene length and optimal codon use—disentangling translational selection from GC-biased gene conversion in yeast; BMC Evol. Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=93" /><meta name="citation_reference" content="citation_title=Codon usage in twelve species of Drosophila; BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=226" /><meta name="citation_reference" content="citation_title=Assessment of compositional heterogeneity within and between eukaryotic genomes; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1986-95; " /><meta name="citation_reference" content="citation_title=A general model of codon bias due to GC mutational bias; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e13431" /><meta name="citation_reference" content="citation_title=Translational selection on codon usage in Xenopus laevis; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=1703-07; " /><meta name="citation_reference" content="citation_title=The influence of translational selection on codon usage in fishes from the family Cyprinidae; Gene;  citation_year=2003;  citation_volume=317;  citation_pages=141-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/6/499/1292542/dsr035.pdf" /><meta name="description" content="Abstract.  Synonymous codons are used with different frequencies both among species and among genes within the same genome and are controlled by neutral process" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/6/499/478052" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/6/499/478052" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/6/499/478052");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr035';
    </script>
<title>
	Mutation Bias is the Driving Force of Codon Usage in the Gallus gallus genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./478052" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbWeATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFvLCBZb3VzaGVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBUZWNobm9sb2d5LCBKaWFuZ3hpIEVkdWNhdGlvbmFsIEluc3RpdHV0ZSwgTmFuY2hhbmcsIEppYW5neGksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3Vhbmdkb25nIFByb3ZpbmNpYWwgS2V5IExhYm9yYXRvcnkgb2YgQWdyby1hbmltYWwgR2Vub21pY3MgYW5kIE1vbGVjdWxhciBCcmVlZGluZywgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSwgR3Vhbmdkb25nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ild1LCBHdW96dW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgVGVjaG5vbG9neSwgSmlhbmd4aSBFZHVjYXRpb25hbCBJbnN0aXR1dGUsIE5hbmNoYW5nLCBKaWFuZ3hpLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFpoYW5nZmVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBUZWNobm9sb2d5LCBKaWFuZ3hpIEVkdWNhdGlvbmFsIEluc3RpdHV0ZSwgTmFuY2hhbmcsIEppYW5neGksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hhaSwgWHVld2VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFRlY2hub2xvZ3ksIEppYW5neGkgRWR1Y2F0aW9uYWwgSW5zdGl0dXRlLCBOYW5jaGFuZywgSmlhbmd4aSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaWUsIFFpbmdodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFib3JhdG9yeSBvZiBBZ3JvLWFuaW1hbCBHZW5vbWljcyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91LCBHdWFuZ2RvbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSwgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSwgR3Vhbmdkb25nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBYaXF1YW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFib3JhdG9yeSBvZiBBZ3JvLWFuaW1hbCBHZW5vbWljcyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91LCBHdWFuZ2RvbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSwgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSwgR3Vhbmdkb25nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTXV0YXRpb24gQmlhcyBpcyB0aGUgRHJpdmluZyBGb3JjZSBvZiBDb2RvbiBVc2FnZSBpbiB0aGUgR2FsbHVzIGdhbGx1cyBnZW5vbWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0OTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjUxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW4gYmFjdGVyaWE6IGNvcnJlbGF0aW9uIHdpdGggZ2VuZSBleHByZXNzaXZpdHk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTcwNTUtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gcGF0dGVybiBhbmQsIHN1cnByaXNpbmdseSwgZ2VuZSBsZW5ndGggc2hhcGUgY29kb24gdXNhZ2UgaW4gQ2Flbm9yaGFiZGl0aXMsIERyb3NvcGhpbGEsIGFuZCBBcmFiaWRvcHNpczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTQ0ODItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGFuYWx5c2lzIG9mIHByb3RlaW4gZXhwcmVzc2lvbiBpbiB5ZWFzdDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQyNTsgIGNpdGF0aW9uX3BhZ2VzPTczNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb24gb2YgY29kb24gYmlhcyBtZWFzdXJlcyB3aXRoIG1STkEgbGV2ZWxzOiBhbmFseXNpcyBvZiB0cmFuc2NyaXB0b21lIGRhdGEgZnJvbSBFc2NoZXJpY2hpYSBjb2xpOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzI3OyAgY2l0YXRpb25fcGFnZXM9NC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gYW5kIHByb3RlaW4gbGVuZ3RoIGluZmx1ZW5jZSBjb2RvbiB1c2FnZSBhbmQgcmF0ZXMgb2Ygc2VxdWVuY2UgZXZvbHV0aW9uIGluIFBvcHVsdXMgdHJlbXVsYTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTgzNi00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRSTkEgY29udGVudCBpbiB1bmljZWxsdWxhciBhbmQgbXVsdGktY2VsbHVsYXIgb3JnYW5pc21zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gcHJlZmVyZW5jZXMgaW4gZnJlZS1saXZpbmcgbWljcm9vcmdhbmlzbXM7IE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTE5OC0yMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZmVyIFJOQSBnZW5lIHJlZHVuZGFuY3kgYW5kIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjg7ICBjaXRhdGlvbl9wYWdlcz0zMjItMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWRpZXMgb2YgY29kb24gdXNhZ2UgYW5kIHRSTkEgZ2VuZXMgb2YgMTggdW5pY2VsbHVsYXIgb3JnYW5pc21zIGFuZCBxdWFudGlmaWNhdGlvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpcyB0Uk5BczogZ2VuZSBleHByZXNzaW9uIGxldmVsIGFuZCBzcGVjaWVzLXNwZWNpZmljIGRpdmVyc2l0eSBvZiBjb2RvbiB1c2FnZSBiYXNlZCBvbiBtdWx0aXZhcmlhdGUgYW5hbHlzaXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjM4OyAgY2l0YXRpb25fcGFnZXM9MTQzLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZXZlbHMgb2YgdFJOQXMgaW4gYmFjdGVyaWFsIGNlbGxzIGFzIGFmZmVjdGVkIGJ5IGFtaW5vIGFjaWQgdXNhZ2UgaW4gcHJvdGVpbnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTYxMTktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkEgZ2VuZSBudW1iZXIgYW5kIGNvZG9uIHVzYWdlIGluIHRoZSBDLiBlbGVnYW5zIGdlbm9tZSBhcmUgY28tYWRhcHRlZCBmb3Igb3B0aW1hbCB0cmFuc2xhdGlvbiBvZiBoaWdobHkgZXhwcmVzc2VkIGdlbmVzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCB0Uk5BIGdlbmVzIGluIGV1a2FyeW90ZXM6IGNvcnJlbGF0aW9uIG9mIGNvZG9uIHVzYWdlIGRpdmVyc2l0eSB3aXRoIHRyYW5zbGF0aW9uIGVmZmljaWVuY3kgYW5kIHdpdGggQ0ctZGludWNsZW90aWRlIHVzYWdlIGFzIGFzc2Vzc2VkIGJ5IG11bHRpdmFyaWF0ZSBhbmFseXNpczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTI5MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBiaWFzIGZyb20gdFJOQSdzIHBvaW50IG9mIHZpZXc6IHJlZHVuZGFuY3ksIHNwZWNpYWxpemF0aW9uLCBhbmQgZWZmaWNpZW50IGRlY29kaW5nIGZvciB0cmFuc2xhdGlvbiBvcHRpbWl6YXRpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjI3OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgZGV0ZXJtaW5lcyB0cmFuc2xhdGlvbiByYXRlIGluIEVzY2hlcmljaGlhIGNvbGkuOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTIwNzsgIGNpdGF0aW9uX3BhZ2VzPTM2NS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gbWV0YXpvYW5zOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY0MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIEVzY2hlcmljaGlhIGNvbGk6IHNlbGVjdGlvbiBmb3IgdHJhbnNsYXRpb25hbCBhY2N1cmFjeTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM3NC04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIG9uIGNvZG9uIGJpYXM7IEFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3JjZXMgdGhhdCBpbmZsdWVuY2UgdGhlIGV2b2x1dGlvbiBvZiBjb2RvbiBiaWFzOyBQaGlsLiBUcmFucy4gUi4gU29jLiBCOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMjAzLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtb2RlbCBiYXNlZCBvbiBtdXRhdGlvbiBhbmQgc2VsZWN0aW9uIGV4cGxhaW5zIHRyZW5kcyBpbiBjb2RvbiBhbmQgYW1pbm8tYWNpZCB1c2FnZSBhbmQgR0MgY29tcG9zaXRpb24gd2l0aGluIGFuZCBhY3Jvc3MgZ2Vub21lczsgY2l0YXRpb25fdGV4dD1SRVNFQVJDSDAwMTA7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBiZXR3ZWVuIGdlbm9tZXMgaXMgY29uc3RyYWluZWQgYnkgZ2Vub21lLXdpZGUgbXV0YXRpb25hbCBwcm9jZXNzZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MzQ4MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmFsIHJ1bGVzIGZvciBvcHRpbWFsIGNvZG9uIGNob2ljZTsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIG9mIHNlbGVjdGl2ZWx5IGRyaXZlbiBjb2RvbiB1c2FnZSBpbiByaWNlOiBpbXBsaWNhdGlvbnMgZm9yIEcgKyBDIGNvbnRlbnQgZXZvbHV0aW9uIG9mIEdyYW1pbmVhZSBnZW5lczsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01ODE7ICBjaXRhdGlvbl9wYWdlcz0xMDE1LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VsZWN0aW9uLW11dGF0aW9uLWRyaWZ0IHRoZW9yeSBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz04OTctOTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcnJpbmcgd2VhayBzZWxlY3Rpb24gZnJvbSBwYXR0ZXJucyBvZiBwb2x5bW9ycGhpc20gYW5kIGRpdmVyZ2VuY2UgYXQg4oCYc2lsZW504oCZIHNpdGVzIGluIERyb3NvcGhpbGEgRE5BOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzk7ICBjaXRhdGlvbl9wYWdlcz0xMDY3LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbiBwcmVzc3VyZSwgbmF0dXJhbCBzZWxlY3Rpb24sIGFuZCB0aGUgZXZvbHV0aW9uIG9mIGJhc2UgY29tcG9zaXRpb24gaW4gRHJvc29waGlsYTsgR2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTAy4oCTMTAzOyAgY2l0YXRpb25fcGFnZXM9NDktNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NDUxLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcnkgcGVyc3BlY3RpdmUgb24gc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiB1bmljZWxsdWxhciBvcmdhbmlzbXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVkdWNlZCBuYXR1cmFsIHNlbGVjdGlvbiBhc3NvY2lhdGVkIHdpdGggbG93IHJlY29tYmluYXRpb24gaW4gRHJvc29waGlsYSBtZWxhbm9nYXN0ZXI7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMjM5LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIHNlbGVjdGlvbiBvbiBzeW5vbnltb3VzIHNpdGVzIGlzIGNvcnJlbGF0ZWQgd2l0aCBnZW5lIGxlbmd0aCBhbmQgcmVjb21iaW5hdGlvbiBpbiBEcm9zb3BoaWxhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0yMzktNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24sIGV2b2x1dGlvbmFyeSBhbmQgZ2Vub21pYyBjb25zZXF1ZW5jZXMgb2YgaW50ZXJmZXJlbmNlIHNlbGVjdGlvbjsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJhY3Rpb25zIGJldHdlZW4gbmF0dXJhbCBzZWxlY3Rpb24sIHJlY29tYmluYXRpb24gYW5kIGdlbmUgZGVuc2l0eSBpbiB0aGUgZ2VuZXMgb2YgRHJvc29waGlsYTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYwOyAgY2l0YXRpb25fcGFnZXM9NTk1LTYwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlsbC1Sb2JlcnRzb24gaW50ZXJmZXJlbmNlIGlzIGEgbWlub3IgZGV0ZXJtaW5hbnQgb2YgdmFyaWF0aW9ucyBpbiBjb2RvbiBiaWFzIGFjcm9zcyBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBhbmQgQ2Flbm9yaGFiZGl0aXMgZWxlZ2FucyBnZW5vbWU7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMzk5LTQwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV1dHJhbCBlZmZlY3Qgb2YgcmVjb21iaW5hdGlvbiBvbiBiYXNlIGNvbXBvc2l0aW9uIGluIERyb3NvcGhpbGE7IEdlbmV0LiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9NzktODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZHVjZWQgZWZmaWNhY3kgb2Ygc2VsZWN0aW9uIGluIHJlZ2lvbnMgb2YgdGhlIERyb3NvcGhpbGEgZ2Vub21lIHRoYXQgbGFjayBjcm9zc2luZyBvdmVyOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpYXNlZCBnZW5lIGNvbnZlcnNpb24gYWZmZWN0cyBwYXR0ZXJucyBvZiBjb2RvbiB1c2FnZSBhbmQgYW1pbm8gYWNpZCB1c2FnZSBpbiB0aGUgU2FjY2hhcm9teWNlcyBzZW5zIHN0cmljdG8gZ3JvdXAgb2YgeWVhc3RzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTE3LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb2VzIHJlY29tYmluYXRpb24gaW1wcm92ZSBzZWxlY3Rpb24gb24gY29kb24gdXNhZ2U/IExlc3NvbnMgZnJvbSBuZW1hdG9kZSBhbmQgZmx5IGNvbXBsZXRlIGdlbm9tZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz01Njg4LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGxlbmd0aCBhbmQgY29kb24gdXNhZ2UgYmlhcyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciwgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGFuZCBFc2NoZXJpY2hpYSBjb2xpLjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzE4OC05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHByb3RlaW5zIGFuZCBnZW5lIGV4cHJlc3Npb24gbGV2ZWxzIGFyZSBjb3VwbGVkIGluIERyb3NvcGhpbGEgYW5kIGFyZSBpbmRlcGVuZGVudGx5IGFzc29jaWF0ZWQgd2l0aCBtUk5BIGFidW5kYW5jZSwgcHJvdGVpbiBsZW5ndGggYW5kIG51bWJlciBvZiBwcm90ZWluLXByb3RlaW4gaW50ZXJhY3Rpb25zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTM0NS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRpdmUgYmFzaXMgb2YgY29kb24gdXNhZ2UgaW4gdGhlIGhhcGxvaWQgbW9zcyBQaHlzY29taXRyZWxsYSBwYXRlbnM7IEhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9ODctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29uc2Vuc3VzIGxpbmthZ2UgbWFwIG9mIHRoZSBjaGlja2VuIGdlbm9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFuZCBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgY2hpY2tlbiBnZW5vbWUgcHJvdmlkZSB1bmlxdWUgcGVyc3BlY3RpdmVzIG9uIHZlcnRlYnJhdGUgZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBDaGlja2VuIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW0gKElDR1NDKS47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzI7ICBjaXRhdGlvbl9wYWdlcz02OTUtNzE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJvbmcgcmVnaW9uYWwgYmlhc2VzIGluIG51Y2xlb3RpZGUgc3Vic3RpdHV0aW9uIGluIHRoZSBjaGlja2VuIGdlbm9tZTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEyMDMtMjE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29jaG9yZXMgYW5kIHRoZSBldm9sdXRpb25hcnkgZ2Vub21pY3Mgb2YgdmVydGVicmF0ZXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjQxOyAgY2l0YXRpb25fcGFnZXM9My0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb25zIGJldHdlZW4gY29kaW5nIGFuZCBjb250aWd1b3VzIG5vbi1jb2Rpbmcgc2VxdWVuY2VzIGluIGlzb2Nob3JlIGZhbWlsaWVzIGZyb20gdmVydGVicmF0ZSBnZW5vbWVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQxMDsgIGNpdGF0aW9uX3BhZ2VzPTI0MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIEh1bWFuIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW0uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9ODYwLTkyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgZXZvbHV0aW9uIG9mIGZlbWFsZS1iaWFzZWQsIGJ1dCBub3QgbWFsZS1iaWFzZWQsIGdlbmVzIGV4cHJlc3NlZCBpbiB0aGUgYXZpYW4gYnJhaW47IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yNjk4LTcwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIGZvciBzaG9ydCBpbnRyb25zIGluIGhpZ2hseSBleHByZXNzZWQgZ2VuZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NDE1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBmb3IgdGhlIGNvbXBhY3RuZXNzIG9mIGhpZ2hseSBleHByZXNzZWQgZ2VuZXMgaW4gR2FsbHVzIGdhbGx1czsgQmlvbG9neSBEaXJlY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgbWlkcmFuZ2UgdHJhbnNjcmlwdGlvbiBwcm9maWxlcyByZXZlYWwgZXhwcmVzc2lvbiBsZXZlbCByZWxhdGlvbnNoaXBzIGluIGh1bWFuIHRpc3N1ZSBzcGVjaWZpY2F0aW9uOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTY1MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gaW50ZW5zaXR5IG9uIHByZWZlcnJlZCBjb2RvbnMgY29ycmVsYXRlcyB3aXRoIG92ZXJhbGwgY29kb24gdXNhZ2UgYmlhcyBpbiBDYWVub3JoYWJkaXRpcyByZW1hbmVpOyBDdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIwNTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgU05QLWJhc2VkIGxpbmthZ2UgbWFwIG9mIHRoZSBjaGlja2VuIGdlbm9tZSByZXZlYWxzIHNlcXVlbmNlIGZlYXR1cmVzIGNvcnJlbGF0ZWQgd2l0aCByZWNvbWJpbmF0aW9uIHJhdGU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTEwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIHZhcmlhdGlvbiBhbmQgcmVjb21iaW5hdGlvbiByYXRlIGluIHRoZSBodW1hbiBnZW5vbWU7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTEyODUtOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIHNlcXVlbnRpYWxseSByZWplY3RpdmUgQm9uZmVycm9uaSB0ZXN0IHByb2NlZHVyZTsgU2NhbmQuIEouIFN0YXQuOyAgY2l0YXRpb25feWVhcj0xOTc5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz02NS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q28tdmFyaWF0aW9uIG9mIHRSTkEgYWJ1bmRhbmNlIGFuZCBjb2RvbiB1c2FnZSBpbiBFc2NoZXJpY2hpYSBjb2xpIGF0IGRpZmZlcmVudCBncm93dGggcmF0ZXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjYwOyAgY2l0YXRpb25fcGFnZXM9NjQ5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3RpdmUgYW5kIG11dGF0aW9uYWwgcGF0dGVybnMgYXNzb2NpYXRlZCB3aXRoIGdlbmUgZXhwcmVzc2lvbiBpbiBodW1hbnM6IGluZmx1ZW5jZXMgb24gc3lub255bW91cyBjb21wb3NpdGlvbiBhbmQgaW50cm9uIHByZXNlbmNlOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz0xMjkzLTEzMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGJpYXMgYXMgYSBmYWN0b3IgaW4gcmVndWxhdGluZyBleHByZXNzaW9uIHZpYSB0cmFuc2xhdGlvbiByYXRlIGluIHRoZSBodW1hbiBnZW5vbWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzQ1OyAgY2l0YXRpb25fcGFnZXM9MTI3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3Qgb2YgY29ycmVsYXRlZCB0Uk5BIGFidW5kYW5jZXMgb24gdHJhbnNsYXRpb24gZXJyb3JzIGFuZCBldm9sdXRpb24gb2YgY29kb24gdXNhZ2UgYmlhczsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMTI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gdGhlIGV2b2x1dGlvbiBhbmQgZXhwcmVzc2lvbiBvZiBDaGxhbXlkb21vbmFzIHJlaW5oYXJkdGlpIG51Y2xldXMtZW5jb2RlZCB0cmFuc2ZlciBSTkEgZ2VuZXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTExMy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2x1c3RlcmluZyBvZiBjb2RvbnMgd2l0aCByYXJlIGNvZ25hdGUgdFJOQXMgaW4gaHVtYW4gZ2VuZXMgc3VnZ2VzdHMgYW4gZXh0cmEgbGV2ZWwgb2YgZXhwcmVzc2lvbiByZWd1bGF0aW9uOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA1NDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gYW5kIG1vbGVjdWxhciBldm9sdXRpb247IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjYwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGJpYXMgY292YXJpZXMgd2l0aCBleHByZXNzaW9uIGJyZWFkdGggYW5kIHRoZSByYXRlIG9mIHN5bm9ueW1vdXMgZXZvbHV0aW9uIGluIGh1bWFucywgYnV0IHRoaXMgaXMgbm90IGV2aWRlbmNlIGZvciBzZWxlY3Rpb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTExOTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBwYXR0ZXJuIGFuZCwgc3VycHJpc2luZ2x5LCBnZW5lIGxlbmd0aCBzaGFwZSBjb2RvbiB1c2FnZSBpbiBDYWVub3JoYWJkaXRpcywgRHJvc29waGlsYSwgYW5kIEFyYWJpZG9wc2lzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9NDQ4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWNrZ3JvdW5kIHNlbGVjdGlvbiBpbiBzaW5nbGUgZ2VuZXMgbWF5IGV4cGxhaW4gcGF0dGVybnMgb2YgY29kb24gYmlhczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9MTM4MS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgb2YgY29kb24gdXNhZ2UgYmlhcyBpbiBTaWxlbmUgbGF0aWZvbGlhOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NzcxLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIGJpYXMgaXMgcmVsYXRlZCB0byBnZW5lIGxlbmd0aCBpbiBFc2NoZXJpY2hpYSBjb2xpOiBzZWxlY3Rpb24gZm9yIHRyYW5zbGF0aW9uYWwgYWNjdXJhY3k/OyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9ODY0LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb24gc2l6ZSBhbmQgZXhvbiBld29sdXRpb24gaW4gRHJvc29waGlsYTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTcwOyAgY2l0YXRpb25fcGFnZXM9NDgxLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiBDYWVub3JoYWJkaXRpcyBlbGVnYW5zOiBkZWxpbmVhdGlvbiBvZiB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBhbmQgbXV0YXRpb25hbCBiaWFzZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI0MzctNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbiBsZW5ndGggYW5kIGNvZG9uIHVzYWdlOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIEVzY2hlcmljaGlhIGNvbGk6IHNlbGVjdGlvbiBmb3IgdHJhbnNsYXRpb25hbCBhY2N1cmFjeTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM3NC04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN1cnByaXNpbmcgbmVnYXRpdmUgY29ycmVsYXRpb24gb2YgZ2VuZSBsZW5ndGggYW5kIG9wdGltYWwgY29kb24gdXNl4oCUZGlzZW50YW5nbGluZyB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBmcm9tIEdDLWJpYXNlZCBnZW5lIGNvbnZlcnNpb24gaW4geWVhc3Q7IEJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW4gdHdlbHZlIHNwZWNpZXMgb2YgRHJvc29waGlsYTsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IG9mIGNvbXBvc2l0aW9uYWwgaGV0ZXJvZ2VuZWl0eSB3aXRoaW4gYW5kIGJldHdlZW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE5ODYtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXJhbCBtb2RlbCBvZiBjb2RvbiBiaWFzIGR1ZSB0byBHQyBtdXRhdGlvbmFsIGJpYXM7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTM0MzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBvbiBjb2RvbiB1c2FnZSBpbiBYZW5vcHVzIGxhZXZpczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE3MDMtMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbmZsdWVuY2Ugb2YgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gb24gY29kb24gdXNhZ2UgaW4gZmlzaGVzIGZyb20gdGhlIGZhbWlseSBDeXByaW5pZGFlOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxNzsgIGNpdGF0aW9uX3BhZ2VzPTE0MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvNi80OTkvMTI5MjU0Mi9kc3IwMzUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFN5bm9ueW1vdXMgY29kb25zIGFyZSB1c2VkIHdpdGggZGlmZmVyZW50IGZyZXF1ZW5jaWVzIGJvdGggYW1vbmcgc3BlY2llcyBhbmQgYW1vbmcgZ2VuZXMgd2l0aGluIHRoZSBzYW1lIGdlbm9tZSBhbmQgYXJlIGNvbnRyb2xsZWQgYnkgbmV1dHJhbCBwcm9jZXNzIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NzgwNTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkMtFq3aD1pzbOntsiFHQSx2d+uD8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f499%2f478052" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f499%2f478052" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18719" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&amp;Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/6">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7225016">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225016">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225019">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225019">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225023">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225023">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225038">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225038">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225059">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225059">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225069">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225069">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225071">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225071">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225073">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225073">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225075">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225075">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7225077">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7225077">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/6/483/477646">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/6/513/478329">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Mutation Bias is the Driving Force of Codon Usage in the <em>Gallus gallus</em> genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yousheng Rao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Technology</div>, <div class="insititution">Jiangxi Educational Institute</div>, <div class="addr-line">Nanchang, Jiangxi</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding</div>, <div class="insititution">South China Agricultural University</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: 86 0791 88300376. Fax: 86 0791 83812190. E-mail: <a href="mailto:rys8323571@yahoo.com.cn" target="_blank">rys8323571@yahoo.com.cn</a> (Y.R.); <a href="mailto:xqzhang@scau.edu.cn" target="_blank">xqzhang@scau.edu.cn</a> (X.Z.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yousheng+Rao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rao Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rao Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yousheng Rao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guozuo Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Technology</div>, <div class="insititution">Jiangxi Educational Institute</div>, <div class="addr-line">Nanchang, Jiangxi</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guozuo+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guozuo Wu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhangfeng Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Technology</div>, <div class="insititution">Jiangxi Educational Institute</div>, <div class="addr-line">Nanchang, Jiangxi</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhangfeng+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhangfeng Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xuewen Chai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Technology</div>, <div class="insititution">Jiangxi Educational Institute</div>, <div class="addr-line">Nanchang, Jiangxi</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xuewen+Chai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chai X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chai X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xuewen Chai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qinghua Nie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding</div>, <div class="insititution">South China Agricultural University</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">College of Animal Science</div>, <div class="insititution">South China Agricultural University</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qinghua+Nie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nie Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nie Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qinghua Nie</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiquan Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Guangdong Provincial Key Laboratory of Agro-animal Genomics and Molecular Breeding</div>, <div class="insititution">South China Agricultural University</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">College of Animal Science</div>, <div class="insititution">South China Agricultural University</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: 86 0791 88300376. Fax: 86 0791 83812190. E-mail: <a href="mailto:rys8323571@yahoo.com.cn" target="_blank">rys8323571@yahoo.com.cn</a> (Y.R.); <a href="mailto:xqzhang@scau.edu.cn" target="_blank">xqzhang@scau.edu.cn</a> (X.Z.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiquan+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiquan Zhang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 6, 1 December 2011, Pages 499–512, <a href='https://doi.org/10.1093/dnares/dsr035'>https://doi.org/10.1093/dnares/dsr035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 October 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 March 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 September 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="478052" href="/dnaresearch/article-pdf/18/6/499/1292542/dsr035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yousheng Rao, Guozuo Wu, Zhangfeng Wang, Xuewen Chai, Qinghua Nie, Xiquan Zhang;  Mutation Bias is the Driving Force of Codon Usage in the <em>Gallus gallus</em> genome, <em>DNA Research</em>, Volume 18, Issue 6, 1 December 2011, Pages 499–512, <a href='https://doi.org/10.1093/dnares/dsr035'>https://doi.org/10.1093/dnares/dsr035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/478052?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/478052?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/478052?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/478052?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/478052?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/478052?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/478052?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Mutation%20Bias%20is%20the%20Driving%20Force%20of%20Codon%20Usage%20in%20the%20%26lt%3Bem%26gt%3BGallus%20gallus%26lt%3B%2Fem%26gt%3B%20genome&amp;publicationDate=2011-10-27&amp;volumeNum=18&amp;issueNum=6&amp;author=Rao%2C%20Yousheng%3B%20Wu%2C%20Guozuo&amp;startPage=499&amp;endPage=512&amp;contentId=10.1093%2Fdnares%2Fdsr035&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f499%2f478052" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7225016 id="7225016" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Synonymous codons are used with different frequencies both among species and among genes within the same genome and are controlled by neutral processes (such as mutation and drift) as well as by selection. Up to now, a systematic examination of the codon usage for the chicken genome has not been performed. Here, we carried out a whole genome analysis of the chicken genome by the use of the relative synonymous codon usage (RSCU) method and identified 11 putative optimal codons, all of them ending with uracil (U), which is significantly departing from the pattern observed in other eukaryotes. Optimal codons in the chicken genome are most likely the ones corresponding to highly expressed transfer RNA (tRNAs) or tRNA gene copy numbers in the cell. Codon bias, measured as the frequency of optimal codons (Fop), is negatively correlated with the G + C content, recombination rate, but positively correlated with gene expression, protein length, gene length and intron length. The positive correlation between codon bias and protein, gene and intron length is quite different from other multi-cellular organism, as this trend has been only found in unicellular organisms. Our data displayed that regional G + C content explains a large proportion of the variance of codon bias in chicken. Stepwise selection model analyses indicate that G + C content of coding sequence is the most important factor for codon bias. It appears that variation in the G + C content of CDSs accounts for over 60% of the variation of codon bias. This study suggests that both mutation bias and selection contribute to codon bias. However, mutation bias is the driving force of the codon usage in the <em>Gallus gallus</em> genome. Our data also provide evidence that the negative correlation between codon bias and recombination rates in <em>G. gallus</em> is determined mostly by recombination-dependent mutational patterns.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Gallus gallus&quot;"><em>Gallus gallus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage&quot;">codon usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mutation">mutation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="selection">selection</a></div>                            <h2 scrollto-destination=7225019 id="7225019" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Synonymous codons are used with different frequencies both among species and among genes within the same genome. Highly expressed genes (such as those encoding translation elongation factors and ribosomal proteins) tend to use optimal (preferred) codons and exhibit very high levels of codon bias.<sup><a reveal-id="DSR035C1  DSR035C5" data-open="DSR035C1  DSR035C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> The optimal codons also tend to correspond to highly expressed tRNAs and tRNA gene copy numbers.<sup><a reveal-id="DSR035C4  DSR035C13" data-open="DSR035C4  DSR035C13" class="link link-ref link-reveal xref-bibr">4–13</a></sup> These patterns have been interpreted as natural selection for more efficient and accurate translation.<sup><a reveal-id="DSR035C6" data-open="DSR035C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR035C14  DSR035C18" data-open="DSR035C14  DSR035C18" class="link link-ref link-reveal xref-bibr">14–18</a></sup> In contrast, some studies have demonstrated that the first factor shaping codon usage is nucleotide composition (G + C content) of genes and intergenic regions.<sup><a reveal-id="DSR035C12" data-open="DSR035C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR035C19  DSR035C22" data-open="DSR035C19  DSR035C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup> As G + C content is more likely determined by genome-wide processes rather than by selective forces acting specifically on coding sequence, these findings have been inferred to reflect the genome-wide patterns of codon usage by mutational biases. Based on the fact that both mutational pressures and selective forces are involved in the phenomenon of codon bias in a variety of organisms, an integrated model, known as the mutation–selection–drift balance model, has been proposed.<sup><a reveal-id="DSR035C15" data-open="DSR035C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSR035C23  DSR035C25" data-open="DSR035C23  DSR035C25" class="link link-ref link-reveal xref-bibr">23–25</a></sup> This model proposes that selection favours optimal codons over minor codons, while mutational pressure and genetic drift allow the minor codons to persist.<sup><a reveal-id="DSR035C17" data-open="DSR035C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Population genetics has shown that the selection of codon bias is generally weak<sup><a reveal-id="DSR035C17" data-open="DSR035C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR035C23" data-open="DSR035C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSR035C26" data-open="DSR035C26" class="link link-ref link-reveal xref-bibr">26</a></sup> (|<em>N</em><sub>e</sub><em>s</em>| ≈ 1), therefore, selection coefficients are expected to be more efficient in species with large effective population sizes (<em>N</em><sub>e</sub>) such as prokaryotes and unicellular eukaryotes.<sup><a reveal-id="DSR035C6" data-open="DSR035C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR035C27" data-open="DSR035C27" class="link link-ref link-reveal xref-bibr">27</a></sup> In species with low <em>N</em><sub>e</sub> values, genetic drift should be the main force shaping codon usage and overpowering translational selection of codon variants.</p><p>Codon bias has been determined to be positively correlated with recombination rates in <em>Drosophila</em>, as well as in many other species.<sup><a reveal-id="DSR035C28  DSR035C35" data-open="DSR035C28  DSR035C35" class="link link-ref link-reveal xref-bibr">28–35</a></sup> This observation has been explained by two hypotheses. The first proposed that the reduction of codon bias in the regions with limited recombination is consistent with Hill–Robertson interference.<sup><a reveal-id="DSR035C28" data-open="DSR035C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR035C30" data-open="DSR035C30" class="link link-ref link-reveal xref-bibr">30</a></sup> However, another hypothesis, called the GC-biased gene conversion model, suggested that the correlation between recombination and codon usage patterns is caused by recombination-related mutational bias rather than by Hill–Robertson interference, as the heteroduplex DNA appears to be biased toward the preferential fixation of AT → GC mutations.<sup><a reveal-id="DSR035C32" data-open="DSR035C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSR035C36" data-open="DSR035C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Except for nucleotide composition, gene expression and recombination rates, other additional parameters such as protein length, gene length and intron length also have been found to play an important role in shaping codon usage in a wide variety of organisms.<sup><a reveal-id="DSR035C2" data-open="DSR035C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR035C5" data-open="DSR035C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR035C37  DSR035C39" data-open="DSR035C37  DSR035C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup></p><p>The chicken (<em>Gallus gallus</em>) is an important model organism that bridges the evolutionary gap between mammals and other vertebrates. The chicken karyotype comprises 39 pairs of chromosomes, which are divided into 8 pairs of cytologically distinct macro-chromosomes, Z and W sex chromosomes, and 30 pairs of micro-chromosomes.<sup><a reveal-id="DSR035C40" data-open="DSR035C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Compared to other vertebrate genomes, the chicken genome has many distinctive characteristics such as a smaller genome size (less than half of humans and mouse), higher recombination rates and higher G + C content.<sup><a reveal-id="DSR035C41" data-open="DSR035C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSR035C42" data-open="DSR035C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Base composition is found to vary greatly between different genomic regions in many eukaryotes. In vertebrates, such as mammalian and birds, one of the most striking features of their genomes is the variation of G + C content that occurs over scales of hundreds of kilobases to megabases, the so-called ‘isochore structure’.<sup><a reveal-id="DSR035C43" data-open="DSR035C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSR035C44" data-open="DSR035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Although subsequent study indicated that the isochore model might need slight revision,<sup><a reveal-id="DSR035C45" data-open="DSR035C45" class="link link-ref link-reveal xref-bibr">45</a></sup> it is clear that the genomes of vertebrates are highly heterogeneous in G + C content and have acquired GC-rich regions.<sup><a reveal-id="DSR035C42" data-open="DSR035C42" class="link link-ref link-reveal xref-bibr">42</a></sup> This results in that a large proportion of variance in codon usage bias is explained by G + C content.<sup><a reveal-id="DSR035C20" data-open="DSR035C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSR035C21" data-open="DSR035C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Mank <em>et al</em>. investigated the chicken's properties of sex-biased genes (female biased genes 155, male biased genes 286) through a microarray data. They found that the codon usage of sex-biased genes showed some sex-biased effects, primarily for autosomal genes expressed in the gonad. Codon bias is greatest when GC<sub>3</sub> (the G + C content at third coding positions) is skewed away from equal usage of GC or AT.<sup><a reveal-id="DSR035C46" data-open="DSR035C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Up to now, a systematic examination of the codon usage for the <em>G. gallus</em> genome has not been performed. In the present study, we carried out a whole analysis of the chicken genome and showed that codon bias is negatively correlated with G + C content and recombination rates, but positively correlated with tRNA abundance, gene expression, protein length, gene length and intron length. Our data clearly displayed that regional G + C content explains a large proportion of the variance of codon bias in <em>G. gallus</em> genome. This study will benefit our understanding of how natural selection and mutation impacts codon usage in the <em>G. gallus</em> genome.</p>                            <h2 scrollto-destination=7225023 id="7225023" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7225024 id="7225024" class="section-title" data-legacy-id=s2a>2.1. Sequence data</h3>
<p>Only nuclear genes with complete information on protein-coding sequences (CDSs) with no evidence of multiple-splicing forms were included in this study. CDSs corresponding to all annotated genes in the chicken genome were downloaded from <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/gquery">http://www.ncbi.nlm.nih.gov/sites/gquery</a> and peptide information coded by the genes was derived from <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/protein">http://www.ncbi.nlm.nih.gov/protein</a>. Some CDSs lengths are obviously not consistent with the total length of corresponding exons and these genes were defined as annotation errors and were not included for analysis. CDSs that did not begin with an ATG start codon, did not have a length of &gt;300 bp, did not contain a multiple of three or that contained an internal stop codon were also ruled out. The final sequence collection contained 8631 CDSs with each corresponding to a unique gene in the <em>G. gallus</em> genome. For each gene, total gene length, protein length, first intron length and average intron length were determined.</p>                            <h3 scrollto-destination=7225026 id="7225026" class="section-title" data-legacy-id=s2b>2.2. Expression data</h3>
<div class="block-child-p">Expression databases were taken from the NCBI FTP website (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez">http://www.ncbi.nlm.nih.gov/sites/entrez</a>) and a total of 633 321 expressed sequence tag (EST) sequences were available. We used the number of EST sequences in this database that align unequivocally to a given gene, and compared the set of chicken mRNA/cDNA sequences with the ESTs using the program BLASTN. We accepted EST hits of &gt;400 nt and with &gt;96% identity to a mRNA/cDNA sequence as matches. If they showed &gt;98% identity, we accepted hits of 100–400 nt and discarded hits of &lt;100 nt.<sup><a reveal-id="DSR035C47" data-open="DSR035C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSR035C48" data-open="DSR035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> An EST matched to multiple genes was discarded. After excluding genes with multiple-splicing forms and genes with obvious annotation errors, the data on the 8631 genes from 18 tissues (including the blood, brain, bursa of fabricius, cecum, connective tissue, embryonic tissue, epiphyseal growth plate, gonad, head, heart, limb, liver, muscle, ovary, pancreas, spleen, testis and thymus) were taken into account. Tags per million were then calculated for each gene in each tissue. Total expression levels are defined as ESTs of a gene in the total number of tissues. Expression breadth is defined as the number of tissues in which the ESTs were found. The tissue specificity index (<em>τ</em>) is measured by both qualitative (i.e. presence/absence) and quantitative variations of expression levels among tissues, and is defined as:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr035ueq1.gif?Expires=2147483647&Signature=1y3jKzNCSvUMQSGiaM2m9g-HLSP-YAQN1pWsqcrE8arw2jKLCP4TkBbaTP8rdYKsfb7Y8V~WMfAhT7jGG1CUXNSP2u2XAW4z95Y-Z7w2SS7K8UpoSE3bfVeAX52IOxPxdx6dKXTOfJYgUTnON3PtawZ-o1vaFzlmghqVmevDti1PYb3IcfEa6eoUeWoVY5YkGbeGS6TSBeacwfp0npY5o~za7iBykTWvGeWf4SiaaXMFv45Ngwng1mTfBE3JlTiRtBEdov3WvmLEsxJ86zI-Rer0xMMkDTHVqFVS39oos1MZjEkyjd9NpWbgNIRXNQBFw8W4St2lndRX5Tws9KgHaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr035ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr035ueq1.gif?Expires=2147483647&Signature=0kB3bc6hQU1jP1pyt3lOzAo1WIonfDPmK-Ce3Zav8X1~LsA7Uhp4QNw4zJQgk5CsArMq65pB-wUCCjUcaIz-h4iBMGi3SAkU~N6~fVwAt6fNWO2uwXPQ4xHAo-~ByfNL-3~JQkVI4VPketLT8Q1UwxsoqXCFZvuyBiYWZTnsf6tyL1KRfGhNnAYHiKCwRm~ztviysbw72lLgKs6rPLhMuX3QEDG44vY1JM~3R1Hl2yD9~mxqANMdmvv4oB70C7oxJH7ipdt4BcRc27RPFPLYqS8Pm6RSL~RUmA4qxGuBZggkeZwbZ-jdcWZdtM69BmoVDHxiWYrGyUf5CMaMWCWmdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr035ueq1.gif" /></a></div></div></div></div><p>where <em>N</em> is the number of tissue samples examined, <em>x<sub>i</sub></em> is the expression level of the gene in sample <em>i</em> and <em>x</em><sub>max</sub> is the highest expression level of the gene across the <em>N</em> samples examined.<sup><a reveal-id="DSR035C49" data-open="DSR035C49" class="link link-ref link-reveal xref-bibr">49</a></sup></p>                            <h3 scrollto-destination=7225030 id="7225030" class="section-title" data-legacy-id=s2c>2.3. Identification of optimal codons and synonymous codon usage</h3>
<p>Optimal codons are defined as those that occur significantly more often in highly expressed genes relative to their frequency in lower expressed genes. We used 5% of the total genes with extremely high and low expression levels inferred from EST counts, as the high and low data set, respectively, and calculated the average RSCU (relative synonymous codon usage) of the two gene samples. RSCU was calculated by dividing the observed codon usage by that expected when all codons for the same amino acid are used equally. RSCU values close to 1.0 indicate a lack of bias. Putative optimal codons were inferred based on departures from equal codon usage by sets of loci with high and low gene expression.<sup><a reveal-id="DSR035C2" data-open="DSR035C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR035C50" data-open="DSR035C50" class="link link-ref link-reveal xref-bibr">50</a></sup> ΔRSCU for a given codon is the difference between the average RSCU of genes with high and low expression [significance tested using the one-way analysis of variance (ANOVA) by SAS]. If ΔRSCU is &gt;0.1 at <em>P</em>&lt; 0.05, this codon will be identified as an optimal codon. Then, we calculated Fop values using the codonW 1.4.2 program with customized optimal codon tables (J Peden, <a class="link link-uri" href="http://codonw.sourceforge.net">http://codonw.sourceforge.net</a>). Fop is the ratio of optimal codons to synonymous codons, ranging from 0 (where no optimal codons are used) to 1 (where only optimal codons are used). The nucleotide composition indices including GC<sub>3</sub> and G + C content of CDSs were also calculated using codonW 1.4.2.</p>                            <h3 scrollto-destination=7225032 id="7225032" class="section-title" data-legacy-id=s2d>2.4. tRNA gene copy number data</h3>
<p>The tRNA gene copy numbers for each codon in the <em>G. gallus</em> genome was taken from <a class="link link-uri" href="http://gtrnadb.ucsc.edu/Ggall/">http://gtrnadb.ucsc.edu/Ggall/</a>. In these data, pseudogenes have already been removed. We used tRNA gene copy numbers as an assumed estimate of cellular tRNA abundance. The relative gene frequency (RGF) of tRNAs is the observed frequency of an isoacceptor tRNA gene in the genome divided by the frequency expected if all isoacceptor tRNA genes for that amino acid occurred with equal frequencies.<sup><a reveal-id="DSR035C11" data-open="DSR035C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p>                            <h3 scrollto-destination=7225034 id="7225034" class="section-title" data-legacy-id=s2e>2.5. Recombination rate estimation</h3>
<p>The recombination rates for 1 Mbp windows were estimated. The versions of the genome assemblies (NCBI build 2.1, released November, 2006) and the latest chicken consensus linkage map (sex-averaged map) were used.<sup><a reveal-id="DSR035C51" data-open="DSR035C51" class="link link-ref link-reveal xref-bibr">51</a></sup> This high-resolution consensus map included 9268 markers, consisting of 34 linkage groups. It enabled us to estimate the local recombination rates using a narrower region. Locations of individual markers were determined based on alignments of the full sequence of the markers using BLAST. The linear function was fitted to the points representing genetic and physical map positions in the 1 Mbp windows. The slope of this line was interpreted as an estimate of recombination rates.<sup><a reveal-id="DSR035C52" data-open="DSR035C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Windows were removed that contained &gt;50% ‘<em>N</em>’ in the sequence assembly, as were windows at the beginning, end and centromere of chromosomes with no markers detected in them. Some windows with large discrepancies between the genetic map and the sequence assembly were also removed. A total of 745 windows were included, which covered ∼70% of the chicken genome.</p>                            <h3 scrollto-destination=7225036 id="7225036" class="section-title" data-legacy-id=s2f>2.6. Statistical analysis</h3>
<p>Correlation analysis between variables was performed by SAS Proprietary Software Release 8.1. In order to assess the actual strength of association, correlation coefficients reported in this study were obtained using all genes independently and avoided the approach of subdividing genes into groups to later investigate relationships among them. The significance tests were corrected for multiple testing by the Bonferroni step-down correction.<sup><a reveal-id="DSR035C53" data-open="DSR035C53" class="link link-ref link-reveal xref-bibr">53</a></sup> To determine the variables contributing to codon bias and how they may interact, we performed multiple linear regressions with the variables, excluding those not contributing significantly through the use of the <em>t</em>-statistical logarithm with backward stepwise regression.</p>                            <h2 scrollto-destination=7225038 id="7225038" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7225039 id="7225039" class="section-title" data-legacy-id=s3a>3.1. Putative optimal codons and tRNA abundance</h3>
<p>A total of 8631genes were included in this study. As shown in Table <a reveal-id="DSR035TB1" data-open="DSR035TB1" class="link link-reveal link-table xref-fig">1</a>, 11 codons have been identified as putatively optimal. Interestingly, all putative optimal codons in the chicken genome are ended by uracil (U). Previous studies suggested that the nucleotide composition (G + C content) plays an important role in the identities of optimal codons as selection for optimal codons for transcription and translation is not high enough to overcome compositional skews.<sup><a reveal-id="DSR035C12" data-open="DSR035C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR035C20" data-open="DSR035C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSR035C21" data-open="DSR035C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Base composition is found to vary greatly between different genomic regions in many eukaryotes. In vertebrates, such as mammalian and birds, one of the most striking features of their genomes is the variation of G + C content that occurs over scales of hundreds of kilobases to megabases.<sup><a reveal-id="DSR035C43" data-open="DSR035C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSR035C44" data-open="DSR035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> In order to further test whether the G + C content had a significant effect on the identities of optimal codons in the <em>G. gallus</em> genome, we produced a high G + C content sample (20% of the highest G + C content of the CDSs) and a low G + C content sample (20% of the lowest G + C content of the CDSs) and inferred the optimal codons by the use of the ΔRSCU method as described above. For the low G + C content sample, 11 codons were identified as optimal and 9 overlapped with the result of the whole data analysis, lacking the alanine (coded by CGU), the threonine (coded by ACU), plus the glutamine (coded by CAG) and the threonine (coded by ACA). For high G + C content sample, 13 codons were identified as optimal and among them, 10 codons overlapped with the result of the whole data analysis, lacking only the cysteine (coded by UGU), plus the phenylalanine (coded by UUU), the serine (coded by UCA) and the proline (coded by CCG, see additional files, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsr035/-/DC1" target="_blank">S2</a>). The analyses of samples of high and low G + C content revealed that G + C content of CDS has a significant effect on the identities of the optimal codons in the chicken genome. However, it should be noted that most putative optimal codons (9–10 codons) occurred coincidently in three samples. We believe that large samples can give more accurate estimates and, therefore, the codon bias (Fop, the ratio of optimal codons to synonymous codons) estimate next was based on the putatively optimal codons identified by the whole data set of 8631 genes. </p>                        <a id="7225041" scrollto-destination="7225041"></a>
<div content-id="DSR035TB1" class="table-modal"><div class="table-wrap"><div id="DSR035TB1" class="table-wrap-title" data-id="DSR035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The putative optimal codons and tRNA abundance</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03506.gif?Expires=2147483647&Signature=YRdL5ZYualTCkvcDPCBuKJW4FPqaVni8kGi57pGGmiOSh-NOzY1A9B44YGvtXwRdENZo~sDOmmyNSzk2ukt8vj8BjUEwVzzaeuCtTWguhezyrlNzGf6YQAqym4X-OUMNu0hkL~xkaEcsH3l29tlO5JYkHvUUiXAIOJH0NWD-~2sST3GG14h~bRKUxs5lbfnVVPaeTwXDV-7ztvuDDcwb67uVkw2xiFklu-uiwNl4bRQvbYX36NrVo86Ph3ejmEhDiI2MecjBAuHSq6N1cUz51pTeDJYLFjfYYGDmA74d0kBEQyNUAdxLguUmYGTVGVJDEra50dtM-RVsrOsNReMcLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr03506.gif" /> </td></tr></tbody></table></div><div content-id="DSR035TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03506.gif?Expires=2147483647&Signature=YRdL5ZYualTCkvcDPCBuKJW4FPqaVni8kGi57pGGmiOSh-NOzY1A9B44YGvtXwRdENZo~sDOmmyNSzk2ukt8vj8BjUEwVzzaeuCtTWguhezyrlNzGf6YQAqym4X-OUMNu0hkL~xkaEcsH3l29tlO5JYkHvUUiXAIOJH0NWD-~2sST3GG14h~bRKUxs5lbfnVVPaeTwXDV-7ztvuDDcwb67uVkw2xiFklu-uiwNl4bRQvbYX36NrVo86Ph3ejmEhDiI2MecjBAuHSq6N1cUz51pTeDJYLFjfYYGDmA74d0kBEQyNUAdxLguUmYGTVGVJDEra50dtM-RVsrOsNReMcLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr03506.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Putative optimal codons were inferred based on departures from equal codon usage by sets of loci with high (5% top) and low (5% down) gene expression (▵RSCU). ▵RSCU for a given codon is the difference between the average RSCU of genes with high and low expression (significance tested using the one-way ANOVA). If ▵RSCU is &gt;0.1 at <em>P</em>&lt; 0.05, this codon will be identified as optimal codon. Total optimal codons identified in this study are 11. The transfer RNA gene (tRNA) copy numbers for each codon was taken from <a class="link link-uri" href="http://gtrnadb.ucsc.edu/Ggall/">http://gtrnadb.ucsc.edu/Ggall/</a>. There is a good correspondence between tRNA abundance and optimal codons within codon classes. However, the above correlation reflects only partially the real co-adaptation of tRNA abundance and codon usage, as the same tRNA can decode several codons. Since we have no experimental data on base modifications in <em>Gallus gallus</em> tRNAs, we predicted the codons decoded by the different anticodons according to the ‘parsimony of wobbling’ criterion. The putative optimal codons with ▵RSCU is &gt;0.1 at <em>P</em>&lt; 0.05 and ▵RSCU &gt;0.2 at <em>P</em>&lt; 0.05 are denoted by ‘*’ and ‘**’, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7225041" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR035TB1" class="table-wrap-title" data-id="DSR035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The putative optimal codons and tRNA abundance</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03506.gif?Expires=2147483647&Signature=YRdL5ZYualTCkvcDPCBuKJW4FPqaVni8kGi57pGGmiOSh-NOzY1A9B44YGvtXwRdENZo~sDOmmyNSzk2ukt8vj8BjUEwVzzaeuCtTWguhezyrlNzGf6YQAqym4X-OUMNu0hkL~xkaEcsH3l29tlO5JYkHvUUiXAIOJH0NWD-~2sST3GG14h~bRKUxs5lbfnVVPaeTwXDV-7ztvuDDcwb67uVkw2xiFklu-uiwNl4bRQvbYX36NrVo86Ph3ejmEhDiI2MecjBAuHSq6N1cUz51pTeDJYLFjfYYGDmA74d0kBEQyNUAdxLguUmYGTVGVJDEra50dtM-RVsrOsNReMcLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr03506.gif" /> </td></tr></tbody></table></div><div content-id="DSR035TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03506.gif?Expires=2147483647&Signature=YRdL5ZYualTCkvcDPCBuKJW4FPqaVni8kGi57pGGmiOSh-NOzY1A9B44YGvtXwRdENZo~sDOmmyNSzk2ukt8vj8BjUEwVzzaeuCtTWguhezyrlNzGf6YQAqym4X-OUMNu0hkL~xkaEcsH3l29tlO5JYkHvUUiXAIOJH0NWD-~2sST3GG14h~bRKUxs5lbfnVVPaeTwXDV-7ztvuDDcwb67uVkw2xiFklu-uiwNl4bRQvbYX36NrVo86Ph3ejmEhDiI2MecjBAuHSq6N1cUz51pTeDJYLFjfYYGDmA74d0kBEQyNUAdxLguUmYGTVGVJDEra50dtM-RVsrOsNReMcLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr03506.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Putative optimal codons were inferred based on departures from equal codon usage by sets of loci with high (5% top) and low (5% down) gene expression (▵RSCU). ▵RSCU for a given codon is the difference between the average RSCU of genes with high and low expression (significance tested using the one-way ANOVA). If ▵RSCU is &gt;0.1 at <em>P</em>&lt; 0.05, this codon will be identified as optimal codon. Total optimal codons identified in this study are 11. The transfer RNA gene (tRNA) copy numbers for each codon was taken from <a class="link link-uri" href="http://gtrnadb.ucsc.edu/Ggall/">http://gtrnadb.ucsc.edu/Ggall/</a>. There is a good correspondence between tRNA abundance and optimal codons within codon classes. However, the above correlation reflects only partially the real co-adaptation of tRNA abundance and codon usage, as the same tRNA can decode several codons. Since we have no experimental data on base modifications in <em>Gallus gallus</em> tRNAs, we predicted the codons decoded by the different anticodons according to the ‘parsimony of wobbling’ criterion. The putative optimal codons with ▵RSCU is &gt;0.1 at <em>P</em>&lt; 0.05 and ▵RSCU &gt;0.2 at <em>P</em>&lt; 0.05 are denoted by ‘*’ and ‘**’, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7225041" target="_blank">View Large</a></div></div><p>For any given set of synonymous codons, the relevant isoacceptor tRNAs might not be equally abundant. Previous studies suggested that the most abundant tRNA for a given amino acid is predominantly recruited by the codons of highly expressed genes.<sup><a reveal-id="DSR035C54" data-open="DSR035C54" class="link link-ref link-reveal xref-bibr">54</a></sup> The optimal codons are most likely the ones corresponding to the most abundant and efficient cognate aa-tRNAs present in the cell.<sup><a reveal-id="DSR035C7" data-open="DSR035C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR035C13" data-open="DSR035C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This trend also existed in the human genome but with lower coefficient.<sup><a reveal-id="DSR035C55" data-open="DSR035C55" class="link link-ref link-reveal xref-bibr">55</a>–<a reveal-id="DSR035C57" data-open="DSR035C57" class="link link-ref link-reveal xref-bibr">57</a></sup> We conducted an analysis to test whether this trend also exists in the chicken genome. As tRNA gene copy numbers are generally correlated with cellular levels of tRNAs in both prokaryotes and eukaryotes,<sup><a reveal-id="DSR035C9" data-open="DSR035C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR035C58" data-open="DSR035C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSR035C59" data-open="DSR035C59" class="link link-ref link-reveal xref-bibr">59</a></sup> we used the abundance of tRNA genes as a substitute for the levels of tRNAs in the cell. We found that there is a good correspondence between tRNA abundance and optimal codons within codon classes (see Table <a reveal-id="DSR035TB1" data-open="DSR035TB1" class="link link-reveal link-table xref-fig">1</a>). We also computed the RGF of each isoacceptor tRNA and made a regression analysis between the RGF and RSCU in highly expressed genes and lesser expressed genes. As shown in Fig. <a reveal-id="DSR035F1" data-open="DSR035F1" class="link link-reveal link-table xref-fig">1</a>, there is a significant correlation between RGF and the RSCU of complementary codons in highly expressed genes (<em>r</em> = 0.4599, <em>P</em>= 0.0013). This positive trend also existed in lesser expressed genes, but with a lower correlation coefficient (<em>r</em> = 0.3548, <em>P</em>= 0.0121). </p>                        <a id="7225043" scrollto-destination="7225043"></a>
<div id="DSR035F1" data-id="DSR035F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03501.jpeg?Expires=2147483647&Signature=A~usqJytVTcyJs8aVRo-6l4W6YhvrhyKtZKG4PmSmz~Uq3WThX5f1lr-gGOI9L38KxkA530To0RPZHvw3I8Hgnn~MYiyIttBRi79eTVIvBghjmftkrNgE3JLv77X-j8BcLHKUZ9bJZ6hebvmW4nNzxEahlzxuX9kk-1toz6hlcOOuuy1GqJIlMEAaoQf4hGFTAQ9~uYPSeSYhX-~7Yl05VJfWFjPVdTZzwFE8fExFO71LXYSHNYvjX0oHCZ4Cm0YY7ALnvneaQ3ffwsH9TAHBFd1MVEFzZHCPIslXcCjpgg4RFqw9Dbf~IM2HZOiLC7nR5lcqohLe~sPJ6bimV2~KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of RGF versus RSCU. The RGF of tRNA genes is the observed frequency of an isoacceptor tRNA gene in Gallus gallus genome divided by the frequency expected if all isoacceptor tRNA genes for that amino acid were equally frequent in the genome. The RSCU is the observed frequency of a codon divided by the frequency expected if all synonyms for that amino acid were used equally. (a) RSCU was measured in the highly expressed genes and (b) RSCU was measured in the lowly expressed genes." path-from-xml="dsr03501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225043/dsr03501.jpeg" path-from-xml="dsr03501.jpeg" target="_blank">View large</a><a section="7225043" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03501.jpeg?Expires=2147483647&Signature=CSoHclpPdcYY697J~zScZJIwzShg1H11aChjJ1it1I~mf6BUMOj8u50Nuwh7WLTDJg3J-HV61gVGo4FfdorB7lPjgEdAvc5FfNxbsDhN6~dKRrFrrF8qFxNOHZvxxcSKQP~BxHBs~~XM-KxohAVoUeQOSuDnzNpLcYtRyuByIgadVq8KN9fuYNqqVzq1obVHedeufTbrrizyR5ToLC0y-aRMoU2Ut8XU1y4SvOJaFumkfTwE3sHvsvrvplfRyD0C9jc9gPosOwS-yY3Jk0DXoFio8oltSe-Ibj~haUlCOIHEkjfeCZOP2YFuUxc1x~0Vyx-AMVZeOgpVzgUqPmajWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225043&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of RGF versus RSCU. The RGF of tRNA genes is the observed frequency of an isoacceptor tRNA gene in <em>Gallus gallus</em> genome divided by the frequency expected if all isoacceptor tRNA genes for that amino acid were equally frequent in the genome. The RSCU is the observed frequency of a codon divided by the frequency expected if all synonyms for that amino acid were used equally. (a) RSCU was measured in the highly expressed genes and (b) RSCU was measured in the lowly expressed genes.</p></div></div><div content-id="DSR035F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03501.jpeg?Expires=2147483647&Signature=A~usqJytVTcyJs8aVRo-6l4W6YhvrhyKtZKG4PmSmz~Uq3WThX5f1lr-gGOI9L38KxkA530To0RPZHvw3I8Hgnn~MYiyIttBRi79eTVIvBghjmftkrNgE3JLv77X-j8BcLHKUZ9bJZ6hebvmW4nNzxEahlzxuX9kk-1toz6hlcOOuuy1GqJIlMEAaoQf4hGFTAQ9~uYPSeSYhX-~7Yl05VJfWFjPVdTZzwFE8fExFO71LXYSHNYvjX0oHCZ4Cm0YY7ALnvneaQ3ffwsH9TAHBFd1MVEFzZHCPIslXcCjpgg4RFqw9Dbf~IM2HZOiLC7nR5lcqohLe~sPJ6bimV2~KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of RGF versus RSCU. The RGF of tRNA genes is the observed frequency of an isoacceptor tRNA gene in Gallus gallus genome divided by the frequency expected if all isoacceptor tRNA genes for that amino acid were equally frequent in the genome. The RSCU is the observed frequency of a codon divided by the frequency expected if all synonyms for that amino acid were used equally. (a) RSCU was measured in the highly expressed genes and (b) RSCU was measured in the lowly expressed genes." path-from-xml="dsr03501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225043/dsr03501.jpeg" path-from-xml="dsr03501.jpeg" target="_blank">View large</a><a section="7225043" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03501.jpeg?Expires=2147483647&Signature=CSoHclpPdcYY697J~zScZJIwzShg1H11aChjJ1it1I~mf6BUMOj8u50Nuwh7WLTDJg3J-HV61gVGo4FfdorB7lPjgEdAvc5FfNxbsDhN6~dKRrFrrF8qFxNOHZvxxcSKQP~BxHBs~~XM-KxohAVoUeQOSuDnzNpLcYtRyuByIgadVq8KN9fuYNqqVzq1obVHedeufTbrrizyR5ToLC0y-aRMoU2Ut8XU1y4SvOJaFumkfTwE3sHvsvrvplfRyD0C9jc9gPosOwS-yY3Jk0DXoFio8oltSe-Ibj~haUlCOIHEkjfeCZOP2YFuUxc1x~0Vyx-AMVZeOgpVzgUqPmajWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225043&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of RGF versus RSCU. The RGF of tRNA genes is the observed frequency of an isoacceptor tRNA gene in <em>Gallus gallus</em> genome divided by the frequency expected if all isoacceptor tRNA genes for that amino acid were equally frequent in the genome. The RSCU is the observed frequency of a codon divided by the frequency expected if all synonyms for that amino acid were used equally. (a) RSCU was measured in the highly expressed genes and (b) RSCU was measured in the lowly expressed genes.</p></div></div>                            <h3 scrollto-destination=7225044 id="7225044" class="section-title" data-legacy-id=s3b>3.2. Relationships between codon bias and GC<sub>3,</sub> G + C content of CDSs and G + C content of intronic sequences</h3>
<p>Codon bias, measured as Fop, averaged 0.2560 ± 0.0042 (ranging from 0.0100 to 0.6183) across the 8631 genes in the <em>G. gallus</em> genome. The Fop values for genes residing on the macro-chromosomes, micro-chromosomes and Z chromosome are 0.2736 ± 0.0011, 0.2236 ± 0.0015 and 0.2992 ± 0.0032, respectively. There is a significant difference among them (<em>P</em>&lt; 0.0001, using the one-way ANOVA). This means that genes residing on the Z chromosome have the highest codon bias. The reasons for this significant difference is most likely owing to the different G + C content, CpG island motifs, gene density and recombination rates of the three types of chromosomes.</p><p>Regression analysis demonstrated that Fop is highly correlated with GC<sub>3</sub> and G + C content of CDSs, respectively (<em>r</em> = −0.8308, <em>P</em>&lt; 0.0001; <em>r</em> = −0.8264, <em>P</em>&lt; 0.0001, see Fig. <a reveal-id="DSR035F2" data-open="DSR035F2" class="link link-reveal link-table xref-fig">2</a>a and b). It appears that variation in GC mutational bias explains over 60% of the variation of the codon usage bias. This negative correlation is expected, as all putative optimal codons inferred in this study ended with U. We also retrieved all intronic sequences for each gene and provided the combined length of all introns for a particular gene exceeding 200 bp and calculated the G + C content of the intronic sequences. We found that Fop values also negatively correlated with the G + C content of the intronic sequences significantly (<em>r</em> = −0.5399, <em>P</em>&lt; 0.0001, see Fig. <a reveal-id="DSR035F2" data-open="DSR035F2" class="link link-reveal link-table xref-fig">2</a>c). </p>                        <a id="7225047" scrollto-destination="7225047"></a>
<div id="DSR035F2" data-id="DSR035F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03502.jpeg?Expires=2147483647&Signature=VhnuoLdvA3ukkmtFktp14HShf65PDp0OH9gW1ymbGqsaG0E80hbNjf2ac~-cK7vvNNEOXU8NvsycIrwE2DwRELe2tlaopTw8zNyLG~2eeGJlXZm5lfNvdMkWFB7ptmZFiqcWUvCd2Wia77a~1avRqSY1Z5Y-J1YKzmZagTLE6-grfpSkf7KMHZXmi71Wd3bKz5cFkM~wugsB7pV9H60HL3VnxEf6WG2t-JbC5CkDXt~VSMQDu4qJj0s6lJMHl3r-ikcrmJkx3WzhMwWFAu15uG6k0ifnDPRoj9p6ADHZMM6Cbva-5TkR4MUQL9ZzxqRfsXAVo6YwEkwos0-HbmRptA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of GC3, G + C content of coding sequences, G + C content of intronic sequences versus the frequency of optimal codons (Fop). Total gene included is 8631. Fop was estimated by codonW 1.4.2 with customized optimal codon table (see Table 1). (a) Fop versus GC3; (b) Fop versus G + C content of CDSs and (c) Fop versus G + C content of intronic sequences." path-from-xml="dsr03502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225047/dsr03502.jpeg" path-from-xml="dsr03502.jpeg" target="_blank">View large</a><a section="7225047" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03502.jpeg?Expires=2147483647&Signature=3cwnFLWy4TRBy9S0PdMYSQnmbUWNNv3ujOUhK2prEv2GXeQkubiHP09XXSQWSmwgotD08K9TQAIrqAdCOcU5HO0wXhJKW-XXq1kiLj56BgrEZqcFKXrV~1fjHrcxWiUCqkYUckawymJzWfXmBFBzKjlP4VBG-s8Aj56StU1vVRiFqdZ0O5h8Mv0d~t~XJlqZp9SZ8TffHbznk-WCr67MY-NVjmuBs3ZDAIgkDpEWkMe~WaXuCywzpY9GqvpVwkGC05FLR7ZDSAVcrto4RiX3KTDWfh8wsDRcqCwnByZhRmoBsWE0et3Xf3vLGEEEcW68eGOXvskqn0PZs2HH9fovpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225047&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of GC<sub>3</sub>, G + C content of coding sequences, G + C content of intronic sequences versus the frequency of optimal codons (Fop). Total gene included is 8631. Fop was estimated by codonW 1.4.2 with customized optimal codon table (see Table <a reveal-id="DSR035TB1" data-open="DSR035TB1" class="link link-reveal link-table xref-fig">1</a>). (a) Fop versus GC<sub>3</sub>; (b) Fop versus G + C content of CDSs and (c) Fop versus G + C content of intronic sequences.</p></div></div><div content-id="DSR035F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03502.jpeg?Expires=2147483647&Signature=VhnuoLdvA3ukkmtFktp14HShf65PDp0OH9gW1ymbGqsaG0E80hbNjf2ac~-cK7vvNNEOXU8NvsycIrwE2DwRELe2tlaopTw8zNyLG~2eeGJlXZm5lfNvdMkWFB7ptmZFiqcWUvCd2Wia77a~1avRqSY1Z5Y-J1YKzmZagTLE6-grfpSkf7KMHZXmi71Wd3bKz5cFkM~wugsB7pV9H60HL3VnxEf6WG2t-JbC5CkDXt~VSMQDu4qJj0s6lJMHl3r-ikcrmJkx3WzhMwWFAu15uG6k0ifnDPRoj9p6ADHZMM6Cbva-5TkR4MUQL9ZzxqRfsXAVo6YwEkwos0-HbmRptA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of GC3, G + C content of coding sequences, G + C content of intronic sequences versus the frequency of optimal codons (Fop). Total gene included is 8631. Fop was estimated by codonW 1.4.2 with customized optimal codon table (see Table 1). (a) Fop versus GC3; (b) Fop versus G + C content of CDSs and (c) Fop versus G + C content of intronic sequences." path-from-xml="dsr03502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225047/dsr03502.jpeg" path-from-xml="dsr03502.jpeg" target="_blank">View large</a><a section="7225047" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03502.jpeg?Expires=2147483647&Signature=3cwnFLWy4TRBy9S0PdMYSQnmbUWNNv3ujOUhK2prEv2GXeQkubiHP09XXSQWSmwgotD08K9TQAIrqAdCOcU5HO0wXhJKW-XXq1kiLj56BgrEZqcFKXrV~1fjHrcxWiUCqkYUckawymJzWfXmBFBzKjlP4VBG-s8Aj56StU1vVRiFqdZ0O5h8Mv0d~t~XJlqZp9SZ8TffHbznk-WCr67MY-NVjmuBs3ZDAIgkDpEWkMe~WaXuCywzpY9GqvpVwkGC05FLR7ZDSAVcrto4RiX3KTDWfh8wsDRcqCwnByZhRmoBsWE0et3Xf3vLGEEEcW68eGOXvskqn0PZs2HH9fovpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225047&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of GC<sub>3</sub>, G + C content of coding sequences, G + C content of intronic sequences versus the frequency of optimal codons (Fop). Total gene included is 8631. Fop was estimated by codonW 1.4.2 with customized optimal codon table (see Table <a reveal-id="DSR035TB1" data-open="DSR035TB1" class="link link-reveal link-table xref-fig">1</a>). (a) Fop versus GC<sub>3</sub>; (b) Fop versus G + C content of CDSs and (c) Fop versus G + C content of intronic sequences.</p></div></div>                            <h3 scrollto-destination=7225048 id="7225048" class="section-title" data-legacy-id=s3c>3.3. Relationships between codon bias and gene expression</h3>
<p>We assessed the effect of expression levels and expression breadth on codon usage bias in our samples. Our data demonstrated that codon bias is positively correlated with gene expression levels (<em>r</em> = 0.1735, <em>P</em>&lt; 0.0001). The correlation between codon bias and expression breadth was also positive and significant (<em>r</em> = 0.1982, <em>P</em>&lt; 0.0001, see Fig. <a reveal-id="DSR035F3" data-open="DSR035F3" class="link link-reveal link-table xref-fig">3</a>a and b). These results suggest that the genes with broader expression breadth and higher expression levels show a higher degree of codon usage bias. Total gene expression is known to be highly influenced by the number of different tissues where a gene is expressed (expression breadth) when expression data are calculated from pooled EST libraries.<sup><a reveal-id="DSR035C60" data-open="DSR035C60" class="link link-ref link-reveal xref-bibr">60</a>,<a reveal-id="DSR035C61" data-open="DSR035C61" class="link link-ref link-reveal xref-bibr">61</a></sup> If expression breadth is the predominant force affecting codon usage, a spurious correlation between codon bias and gene expression is more likely to be generated. In order to alleviate this problem, we divided genes into ubiquitously or narrowly expressed groups, if they were expressed in ≥15 tissues or ≤3 tissues. We regressed expression levels on codon bias for each group, and found that this significant trend also existed for the ubiquitous group (<em>r</em> = 0.1855, <em>P</em>= 0.0016) and for narrowly expressed groups (<em>r</em> = 0.0497, <em>P</em>= 0.0101). When the parameters were expanded to ≥16 or ≤2, we obtained similar results. </p>                        <a id="7225050" scrollto-destination="7225050"></a>
<div id="DSR035F3" data-id="DSR035F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03503.jpeg?Expires=2147483647&Signature=pphejGbAGnr639f67mxryssLaAG-8kK1KV7mVyvDa4N2OBHb2MSNWgS76~8m7NaqHjlhiUw5pI3AAKsoi~fXBOuXPfZNTWYTD4gh3tnoXsJmDRClA~J4qjAXJULz9oUFSAJ6tueCJbvQHgFXsiXTvRvLCFpbqccftKlqlX1dqPmps1Jz-fYP~jBdKP4KCYradCBxozeIJ6Hkgt0Ab~ZbhNwTZ5mil-YEfn-3CgxdG51htnbAAltaWKzoQiRwPmxP6fR4OU4Dz5TbPtYVjD3NJ-hFXK68L-6P8qVdUjCF1l9b9bqJuASGeOBwBc5z2hXhzfPaw3E7r1ew-o7-X1m67g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of Fop versus expression level, expression breadth and the tissue specificity index τ. Expression data on 8631 genes for 18 tissues, blood, brain, bursa of fabricius, cecum, connective tissue, embryonic tissue, epiphyseal growth plate, gonad, head, heart, limb, liver, muscle, ovary, pancreas, spleen, testis and thymus, were taken into account. Expression level is defined as the total expression level of a gene, which is the sum of the total 18 tissues' EST (transformed to logarithm with base 10). Expression breadth defined as the numbers of tissues in which EST was found. The calculation of τ can be seen from materials and methods. (a) Fop versus expression level; (b) Fop versus expression breadth and (c) Fop versus the tissue specificity index τ." path-from-xml="dsr03503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225050/dsr03503.jpeg" path-from-xml="dsr03503.jpeg" target="_blank">View large</a><a section="7225050" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03503.jpeg?Expires=2147483647&Signature=zOZhrQeO9BorwAIRnUBrx7kNf27YSILZGqAhmubNilTKQUX5D2au75qftr54aY940YWJTn8K0piw6KMBQyUcl6Zpt42Zrn6fvMi-RX9oYOHFWQU-DtQu-LZaGKcdrWU3pv7yb8LSZtrBXJ5liONLBiBCrHvTt9LmnnkEzJ1Le-cX3klkbpDnot8FPBfIRyfgnOZYVAAPVQFFkEKtUkTLJIgOAMddogTe6pgFhqmvKL5TDzy1rpWVZIH4X8KFL3SXkT8iVnCjjxfFD7soRGky5AGppL8-zeutdrPuMDk1KxuedbLiOJHmIo4TPeDa53nmXgzNI6YQcY~I63gZQrVdYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225050&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of Fop versus expression level, expression breadth and the tissue specificity index <em>τ</em>. Expression data on 8631 genes for 18 tissues, blood, brain, bursa of fabricius, cecum, connective tissue, embryonic tissue, epiphyseal growth plate, gonad, head, heart, limb, liver, muscle, ovary, pancreas, spleen, testis and thymus, were taken into account. Expression level is defined as the total expression level of a gene, which is the sum of the total 18 tissues' EST (transformed to logarithm with base 10). Expression breadth defined as the numbers of tissues in which EST was found. The calculation of <em>τ</em> can be seen from materials and methods. (a) Fop versus expression level; (b) Fop versus expression breadth and (c) Fop versus the tissue specificity index <em>τ</em>.</p></div></div><div content-id="DSR035F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03503.jpeg?Expires=2147483647&Signature=pphejGbAGnr639f67mxryssLaAG-8kK1KV7mVyvDa4N2OBHb2MSNWgS76~8m7NaqHjlhiUw5pI3AAKsoi~fXBOuXPfZNTWYTD4gh3tnoXsJmDRClA~J4qjAXJULz9oUFSAJ6tueCJbvQHgFXsiXTvRvLCFpbqccftKlqlX1dqPmps1Jz-fYP~jBdKP4KCYradCBxozeIJ6Hkgt0Ab~ZbhNwTZ5mil-YEfn-3CgxdG51htnbAAltaWKzoQiRwPmxP6fR4OU4Dz5TbPtYVjD3NJ-hFXK68L-6P8qVdUjCF1l9b9bqJuASGeOBwBc5z2hXhzfPaw3E7r1ew-o7-X1m67g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of Fop versus expression level, expression breadth and the tissue specificity index τ. Expression data on 8631 genes for 18 tissues, blood, brain, bursa of fabricius, cecum, connective tissue, embryonic tissue, epiphyseal growth plate, gonad, head, heart, limb, liver, muscle, ovary, pancreas, spleen, testis and thymus, were taken into account. Expression level is defined as the total expression level of a gene, which is the sum of the total 18 tissues' EST (transformed to logarithm with base 10). Expression breadth defined as the numbers of tissues in which EST was found. The calculation of τ can be seen from materials and methods. (a) Fop versus expression level; (b) Fop versus expression breadth and (c) Fop versus the tissue specificity index τ." path-from-xml="dsr03503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225050/dsr03503.jpeg" path-from-xml="dsr03503.jpeg" target="_blank">View large</a><a section="7225050" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03503.jpeg?Expires=2147483647&Signature=zOZhrQeO9BorwAIRnUBrx7kNf27YSILZGqAhmubNilTKQUX5D2au75qftr54aY940YWJTn8K0piw6KMBQyUcl6Zpt42Zrn6fvMi-RX9oYOHFWQU-DtQu-LZaGKcdrWU3pv7yb8LSZtrBXJ5liONLBiBCrHvTt9LmnnkEzJ1Le-cX3klkbpDnot8FPBfIRyfgnOZYVAAPVQFFkEKtUkTLJIgOAMddogTe6pgFhqmvKL5TDzy1rpWVZIH4X8KFL3SXkT8iVnCjjxfFD7soRGky5AGppL8-zeutdrPuMDk1KxuedbLiOJHmIo4TPeDa53nmXgzNI6YQcY~I63gZQrVdYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225050&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of Fop versus expression level, expression breadth and the tissue specificity index <em>τ</em>. Expression data on 8631 genes for 18 tissues, blood, brain, bursa of fabricius, cecum, connective tissue, embryonic tissue, epiphyseal growth plate, gonad, head, heart, limb, liver, muscle, ovary, pancreas, spleen, testis and thymus, were taken into account. Expression level is defined as the total expression level of a gene, which is the sum of the total 18 tissues' EST (transformed to logarithm with base 10). Expression breadth defined as the numbers of tissues in which EST was found. The calculation of <em>τ</em> can be seen from materials and methods. (a) Fop versus expression level; (b) Fop versus expression breadth and (c) Fop versus the tissue specificity index <em>τ</em>.</p></div></div><p>The tissue specificity index <em>τ</em> measures both qualitative and quantitative variations of expression levels amongst tissues.<sup><a reveal-id="DSR035C49" data-open="DSR035C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Obviously, <em>τ</em> is more representative than the expression breadth and expression levels alone for the expression pattern of a gene. We calculated the tissue specificity index <em>τ</em> for each gene, and made regression analyses between codon bias and the tissue specificity index <em>τ</em>. As shown in Fig. <a reveal-id="DSR035F3" data-open="DSR035F3" class="link link-reveal link-table xref-fig">3</a>c, codon bias is significantly correlated with the tissue specificity index <em>τ</em> (<em>r</em> = −0.1660, <em>P</em>&lt; 0.0001).</p>                            <h3 scrollto-destination=7225052 id="7225052" class="section-title" data-legacy-id=s3d>3.4. Relationships between codon bias and protein, gene and intron length</h3>
<p>Our data demonstrated that codon bias is significantly and positively correlated with protein length (<em>r</em> = 0.2358, <em>P</em>&lt; 0.0001) and gene length (<em>r</em> = 0.2256, <em>P</em>&lt; 0.0001, Fig. <a reveal-id="DSR035F4" data-open="DSR035F4" class="link link-reveal link-table xref-fig">4</a>a and b). Since protein length and expression levels displayed a strong correlation with codon bias,<sup><a reveal-id="DSR035C48" data-open="DSR035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> we tested whether the correlation between protein length and codon bias can be explained by gene expression levels. We fitted linear models of protein length and codon bias against expression levels. The results from these models correlated at the same levels as uncorrected protein lengths and codon bias. In the narrowly expressed gene samples as described above (breadth is three or less than three tissues), we conducted a regression analysis between codon bias and protein length at similar expression levels (EST counts ranging from 57 to 60) and found that the positive trend also existed (<em>r</em> = 0.2823, <em>P</em>&lt; 0.0001). The negative correlation between codon usage and protein length and gene length has been observed in many organisms, such as yeast, <em>Caenorhabditis elegans</em>, <em>Drosophila melanogaster</em>, <em>Arabidopsis thaliana</em>, <em>Populus tremula</em> and <em>Silene latifolia</em>.<sup><a reveal-id="DSR035C5" data-open="DSR035C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR035C29" data-open="DSR035C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSR035C36" data-open="DSR035C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSR035C62  DSR035C64" data-open="DSR035C62  DSR035C64" class="link link-ref link-reveal xref-bibr">62–64</a></sup> However, the positive correlation identified in this study has been found only in <em>Saccharomyces cerevisiae</em> and <em>Escherichia coli</em>, and was been explained through the model of selection on translational accuracy.<sup><a reveal-id="DSR035C37" data-open="DSR035C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSR035C65" data-open="DSR035C65" class="link link-ref link-reveal xref-bibr">65</a></sup></p>                        <a id="7225054" scrollto-destination="7225054"></a>
<div id="DSR035F4" data-id="DSR035F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03504.jpeg?Expires=2147483647&Signature=jVRnhaALeBOBiLvyI6SS4jx2N9m4EIBD53mhpOoc3i6V~Opmfr6m03XMs4tmoUiiIfSCdXCEAf3D~4SwxTQHP9TQwuJhsTPNJ5M8N6YLeB9k2yPReweGQVWph5voir5UjQd4mkcbSpT3U~QUuywImANgw49MShkcGRTCO~HOKSo-s-8epv2iPihKK9sbgRZEiS6mLM7N2cQsm8xxuQBk8vDZME1khr1xjPMiykrnAGEt6eCrZCya9F-n7GyA-hUKo~ptnTzFOlnXMTpIF0I0xvmEMtYMkWE7oGv1AbutBUMynvUzyv3Z~S4WgT2yKRpSOdmrglqbHexPzIkItve5CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of Fop versus protein length, gene length, average intron length and first intron length (logarithmically transformed). (a) Fop versus protein length; (b) Fop versus gene length; (c) Fop versus average intron length and (d) Fop versus first intron length." path-from-xml="dsr03504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225054/dsr03504.jpeg" path-from-xml="dsr03504.jpeg" target="_blank">View large</a><a section="7225054" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03504.jpeg?Expires=2147483647&Signature=VPK3rBXoi9YRAhMg~VEDA0TRqpiAIb9oeiVCatFNbKyTlcst1B8SSP~IIQZfGTx5aFa8so-lNFNFTon9B~dkE04ncfb8kdDMf-Gt9xMRKcYxAN6kvBHKPaQ~dVLgDmmHKfVbVrcGKLc2YMYDb6c-OF79Dy-k7OfmBvM3bCZHpDmUWMh8cEh4YpAf~OAXq7vQJQRarAq3HJfAOAbwe4tetfFGzO49w03dHYvvZ9EkpDQUVLMVaEF3k6S5JeOeLyk56PQ~TBI6kY26SwJ3GViuCwFGt~FhLcEZvoZcyp5~eV5i9n69uGQuChp5aFXsDZ5uNpHqwv9wfPI6kwAjFsPQAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225054&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of Fop versus protein length, gene length, average intron length and first intron length (logarithmically transformed). (a) Fop versus protein length; (b) Fop versus gene length; (c) Fop versus average intron length and (d) Fop versus first intron length.</p></div></div><div content-id="DSR035F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03504.jpeg?Expires=2147483647&Signature=jVRnhaALeBOBiLvyI6SS4jx2N9m4EIBD53mhpOoc3i6V~Opmfr6m03XMs4tmoUiiIfSCdXCEAf3D~4SwxTQHP9TQwuJhsTPNJ5M8N6YLeB9k2yPReweGQVWph5voir5UjQd4mkcbSpT3U~QUuywImANgw49MShkcGRTCO~HOKSo-s-8epv2iPihKK9sbgRZEiS6mLM7N2cQsm8xxuQBk8vDZME1khr1xjPMiykrnAGEt6eCrZCya9F-n7GyA-hUKo~ptnTzFOlnXMTpIF0I0xvmEMtYMkWE7oGv1AbutBUMynvUzyv3Z~S4WgT2yKRpSOdmrglqbHexPzIkItve5CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of Fop versus protein length, gene length, average intron length and first intron length (logarithmically transformed). (a) Fop versus protein length; (b) Fop versus gene length; (c) Fop versus average intron length and (d) Fop versus first intron length." path-from-xml="dsr03504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225054/dsr03504.jpeg" path-from-xml="dsr03504.jpeg" target="_blank">View large</a><a section="7225054" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03504.jpeg?Expires=2147483647&Signature=VPK3rBXoi9YRAhMg~VEDA0TRqpiAIb9oeiVCatFNbKyTlcst1B8SSP~IIQZfGTx5aFa8so-lNFNFTon9B~dkE04ncfb8kdDMf-Gt9xMRKcYxAN6kvBHKPaQ~dVLgDmmHKfVbVrcGKLc2YMYDb6c-OF79Dy-k7OfmBvM3bCZHpDmUWMh8cEh4YpAf~OAXq7vQJQRarAq3HJfAOAbwe4tetfFGzO49w03dHYvvZ9EkpDQUVLMVaEF3k6S5JeOeLyk56PQ~TBI6kY26SwJ3GViuCwFGt~FhLcEZvoZcyp5~eV5i9n69uGQuChp5aFXsDZ5uNpHqwv9wfPI6kwAjFsPQAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225054&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of Fop versus protein length, gene length, average intron length and first intron length (logarithmically transformed). (a) Fop versus protein length; (b) Fop versus gene length; (c) Fop versus average intron length and (d) Fop versus first intron length.</p></div></div><p>As shown in Fig. <a reveal-id="DSR035F4" data-open="DSR035F4" class="link link-reveal link-table xref-fig">4</a>c and d, codon bias also shows a positive correlation with average intron length and first intron length (<em>r</em> = 0.2337, <em>P</em>&lt; 0.0001; <em>r</em> = 0.1806, <em>P</em>&lt; 0.0001, respectively). The opposite trend has been found in <em>D. melanogaster</em>, <em>C. elegans</em> and <em>P. tremula</em>.<sup><a reveal-id="DSR035C5" data-open="DSR035C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR035C66" data-open="DSR035C66" class="link link-ref link-reveal xref-bibr">66</a>,<a reveal-id="DSR035C67" data-open="DSR035C67" class="link link-ref link-reveal xref-bibr">67</a></sup> This positive trend between codon bias and intron length has been only found in unicellular organisms;<sup><a reveal-id="DSR035C68" data-open="DSR035C68" class="link link-ref link-reveal xref-bibr">68</a></sup> however, the underlying mechanism for this is not clear. Stoletzki and Eyre-Walker<sup><a reveal-id="DSR035C69" data-open="DSR035C69" class="link link-ref link-reveal xref-bibr">69</a></sup> and Stoletzki<sup><a reveal-id="DSR035C70" data-open="DSR035C70" class="link link-ref link-reveal xref-bibr">70</a></sup> suggested that this trend (negative/positive) is related to whose optimal codons are biased towards codons that end with GC or AU. We found that the G + C content of introns is significantly and negatively correlated with average intron length and first intron length (<em>r</em> = −0.4024, <em>P</em>&lt; 0.0001; <em>r</em> = −0.2432, <em>P</em>&lt; 0.0001, respectively). Obviously, the positive trend between codon bias and intron length is related to the G + C content of intronic sequences. If mutation bias is the driving force of the codon usage in the <em>G. gallus</em> genome, the mutation bias hypotheses seems reasonable to explain this result.</p>                            <h3 scrollto-destination=7225056 id="7225056" class="section-title" data-legacy-id=s3e>3.5. Relationship between codon bias and recombination rate</h3>
<p>Fig. <a reveal-id="DSR035F5" data-open="DSR035F5" class="link link-reveal link-table xref-fig">5</a> shows that there is a negative correlation between the Fop values and the local recombination rates (<em>r</em> = −0.4546, <em>P</em>&lt; 0.0001). We also found that the relationship between codon usage bias and recombination rate is independent of the expression levels and protein length (data not shown). This significant trend seems to be expected, since the mutagenic effects of recombination result in a mutational bias toward G and C bases in regions of high recombination rates (GC-biased gene conversion). </p>                        <a id="7225058" scrollto-destination="7225058"></a>
<div id="DSR035F5" data-id="DSR035F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03505.jpeg?Expires=2147483647&Signature=atzatgxvLAnZ4cnB4BVcoesYcvoQSUJ9thOs-vC5lk9gsYkkxUHYsdHYQioyDRKjvm21LKVJp1VgVEcR4fQBrAjPhs-pAFnyrkDZ2GQk5kgfFbq2vIokqqW9Y~eKHZRsrjKmqjQlGlLWmbQTixp53WChtFuwO95mlc1M34Tfg33mS0AoCJf7g4-6MkzFFSEDmgjD~WKp9n1h5RrR5TMB52cGahk-srYzxPd9VaBkpIzFjuygwMMrSsMcNuJEH-R9j8tVCAQexkkCzGpafM9DjIDM52zlHSYiyzUBfT~Tkiz7nKl2Agy9EnVBGo1y3HnW~WrRkOLPVHR6qs9Cs2YHGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of Fop versus recombination rate. The recombination rates for 1 Mb windows were estimated. The versions of the genome assemblies (NCBI build 2.1, released November, 2006) and the latest chicken consensus linkage map were used. Total windows included is 745, covering ∼70% of the chicken genome. For each window, the average Fop for all genes residing in this window was calculated." path-from-xml="dsr03505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225058/dsr03505.jpeg" path-from-xml="dsr03505.jpeg" target="_blank">View large</a><a section="7225058" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03505.jpeg?Expires=2147483647&Signature=JbDlAYi1Bv3Kz8tlaWnaXvKfDdrg8pL52DtLeL8uTQlQS-0I4t0mS33cNde8PkeTx5RfuzsbX20egTvgGZisLYow8ZeW-2I8dXyQWfPZpn3svOSnO4HIyu9JQfubAxcAwQHbDi4HJZLQ35HvVhhhoazcN4BHAOt8DK7oKcUZ4rB9zyk-g8v-qlJ4A2pajCDpm~nTG8gm15wh18zncoaq4gqFI5t6q0lFb1up4GdH0ubQX~AoaG8poAWMUduLiWpkqb9VZU~fibhJ10l9Ljv385m8rk8QLawRxoHOueKwqX2sywTWohndBEVLutilMD2~R2RKIDTODR-KlAIDf4a~pA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225058&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of Fop versus recombination rate. The recombination rates for 1 Mb windows were estimated. The versions of the genome assemblies (NCBI build 2.1, released November, 2006) and the latest chicken consensus linkage map were used. Total windows included is 745, covering ∼70% of the chicken genome. For each window, the average Fop for all genes residing in this window was calculated.</p></div></div><div content-id="DSR035F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/m_dsr03505.jpeg?Expires=2147483647&Signature=atzatgxvLAnZ4cnB4BVcoesYcvoQSUJ9thOs-vC5lk9gsYkkxUHYsdHYQioyDRKjvm21LKVJp1VgVEcR4fQBrAjPhs-pAFnyrkDZ2GQk5kgfFbq2vIokqqW9Y~eKHZRsrjKmqjQlGlLWmbQTixp53WChtFuwO95mlc1M34Tfg33mS0AoCJf7g4-6MkzFFSEDmgjD~WKp9n1h5RrR5TMB52cGahk-srYzxPd9VaBkpIzFjuygwMMrSsMcNuJEH-R9j8tVCAQexkkCzGpafM9DjIDM52zlHSYiyzUBfT~Tkiz7nKl2Agy9EnVBGo1y3HnW~WrRkOLPVHR6qs9Cs2YHGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots of Fop versus recombination rate. The recombination rates for 1 Mb windows were estimated. The versions of the genome assemblies (NCBI build 2.1, released November, 2006) and the latest chicken consensus linkage map were used. Total windows included is 745, covering ∼70% of the chicken genome. For each window, the average Fop for all genes residing in this window was calculated." path-from-xml="dsr03505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7225058/dsr03505.jpeg" path-from-xml="dsr03505.jpeg" target="_blank">View large</a><a section="7225058" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr03505.jpeg?Expires=2147483647&Signature=JbDlAYi1Bv3Kz8tlaWnaXvKfDdrg8pL52DtLeL8uTQlQS-0I4t0mS33cNde8PkeTx5RfuzsbX20egTvgGZisLYow8ZeW-2I8dXyQWfPZpn3svOSnO4HIyu9JQfubAxcAwQHbDi4HJZLQ35HvVhhhoazcN4BHAOt8DK7oKcUZ4rB9zyk-g8v-qlJ4A2pajCDpm~nTG8gm15wh18zncoaq4gqFI5t6q0lFb1up4GdH0ubQX~AoaG8poAWMUduLiWpkqb9VZU~fibhJ10l9Ljv385m8rk8QLawRxoHOueKwqX2sywTWohndBEVLutilMD2~R2RKIDTODR-KlAIDf4a~pA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7225058&ar=478052&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots of Fop versus recombination rate. The recombination rates for 1 Mb windows were estimated. The versions of the genome assemblies (NCBI build 2.1, released November, 2006) and the latest chicken consensus linkage map were used. Total windows included is 745, covering ∼70% of the chicken genome. For each window, the average Fop for all genes residing in this window was calculated.</p></div></div>                            <h2 scrollto-destination=7225059 id="7225059" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=7225060 id="7225060" class="section-title" data-legacy-id=s4a>4.1. The identities of optimal codons in the <em>G. gallus</em> genome</h3>
<p>In the present study, we carried out a systematic examination of the codon usage in the chicken genome. By the use of the ΔRSCU method, we identified 11 codons as putatively optimal. All putative optimal codons in the chicken genome end with U. This is significantly departing from the pattern observed in other eukaryotes genomes, such as <em>Schizosaccharomyces pombe</em>, <em>D. melanogaster</em>, <em>C. elegans</em> and <em>Homo sapiens</em>. The rules governing the identities of optimal codons in different organisms remain obscure. Recently, Hershberg and Petrov<sup><a reveal-id="DSR035C21" data-open="DSR035C21" class="link link-ref link-reveal xref-bibr">21</a></sup> investigated the optimal codons in 675 bacteria, 52 archea and 10 fungi. They found that across all studied organisms, the identities of optimal codons mirrors the G + C content of the genomes. GC-rich organisms tend to have GC-rich optimal codons, while AT-rich organisms tend to have AT-rich optimal codons. However, in <em>Drosophila</em>, <em>C. elegance</em> and <em>Populus tremula</em>, most optimal codons end with G or C (majority are C ending), while their genomes contain 35, 36 and 45% G + C-rich content, respectively.<sup><a reveal-id="DSR035C5" data-open="DSR035C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR035C71" data-open="DSR035C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSR035C72" data-open="DSR035C72" class="link link-ref link-reveal xref-bibr">72</a></sup> In humans, optimal codons seem to be driven in two opposite directions, toward AT richness and GC richness. In other words, genes in the GC-rich regions of the genome preferentially use G and C ending codons, while those in the AT-rich regions use A and T ending codons.<sup><a reveal-id="DSR035C73" data-open="DSR035C73" class="link link-ref link-reveal xref-bibr">73</a></sup> The human genome comprises a mosaic of long stretches of GC-rich and AT-rich regions, the so-called isochore structure. Not only do they occur in silent sites of coding regions but also introns and flanking regions in the gene have a similar base composition.<sup><a reveal-id="DSR035C43" data-open="DSR035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> This isochore structure was also found in the avian genomes.<sup><a reveal-id="DSR035C41" data-open="DSR035C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Recently, some studies have suggested that the G + C content is becoming homogenized in humans.<sup><a reveal-id="DSR035C45" data-open="DSR035C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSR035C55" data-open="DSR035C55" class="link link-ref link-reveal xref-bibr">55</a></sup> However, Webster <em>et al</em>.<sup><a reveal-id="DSR035C42" data-open="DSR035C42" class="link link-ref link-reveal xref-bibr">42</a></sup> found that heterogeneity in the G + C content is being reinforced in the chicken genome. In order to test whether the G + C content influences the identities of optimal codons in <em>G. gallus</em>, we produced a high G + C content sample (20% of the highest G + C content of the CDSs) and a low G + C content sample (20% of the lowest G + C content of the CDSs), and inferred the optimal codons by the use of the ΔRSCU method. We found that most putative optimal codons (9–10 codons) occur coincidently in three samples. Using the abundance of tRNA genes as a substitute for the levels of tRNAs in the cell, we found that there is a good correspondence between tRNA abundance and optimal codons within codon classes. This implies that the optimal codons in the chicken genome are most likely the ones corresponding to the highly expressed tRNAs or tRNA gene copy numbers in the cell.</p>                            <h3 scrollto-destination=7225062 id="7225062" class="section-title" data-legacy-id=s4b>4.2. Mutation bias is the driving force of the codon usage in the <em>G. gallus</em> genome</h3>
<p>Codon bias, as measured by Fop, is significantly correlated with GC<sub>3</sub>, G + C content of CDS and G + C content of the intronic sequences. Our data clearly displayed that regional G + C content explains a large proportion of the variance of codon bias in chicken. Our data also provide strong evidence for the mutational bias hypothesis. However, we found that the G + C content of the intronic sequences is significantly lower than that of CDSs, which are not fully consistent with this hypothesis, as it predicts that G + C content is determined by genome-wide processes rather than by selective forces acting specifically on coding regions. It also should be noted that although the G + C content of intronic sequences show a negative trend with codon usage, the correlation is significantly lower than that between codon usage and G + C content of CDSs. These findings indicate that, except for mutation bias, other factors (such as selection) may have contribution to the codon usage.</p><p>Our data also show that codon bias is significantly and positively correlated with gene expression. The positive correlations between gene expression and codon bias have been shown in many organisms.<sup><a reveal-id="DSR035C1  DSR035C2" data-open="DSR035C1  DSR035C2" class="link link-ref link-reveal xref-bibr">1–2</a>,<a reveal-id="DSR035C4  DSR035C5" data-open="DSR035C4  DSR035C5" class="link link-ref link-reveal xref-bibr">4–5</a></sup> In vertebrates such as mammals, data also support a weak relationship between gene expression and codon usage.<sup><a reveal-id="DSR035C56" data-open="DSR035C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSR035C61" data-open="DSR035C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Both the match between tRNA abundance and optimal codons, and the high codon bias of the highly expressed genes, has been interpreted as natural selection for more efficient and accurate translation.<sup><a reveal-id="DSR035C14  DSR035C18" data-open="DSR035C14  DSR035C18" class="link link-ref link-reveal xref-bibr">14–18</a></sup> Our data provide evidence that natural selection also plays an important role in shaping the codon usage in the chicken genome.</p><p>Codon bias also shows a significant trend with protein length, intron length and recombination rate. To determine what all variables (G + C content of CDS, expression level, expression breadth, protein length, intron length and recombination rate) were contributing to the differences in codon bias and how they may interact, we performed multiple linear regressions with the above variables, excluding those not contributing significantly through the use of the <em>t</em>-statistical analysis and with backward stepwise regression. The best combinations of variables were G + C content of CDSs and expression breadth (<em>R</em><sup>2</sup> = 0.7829, <em>P</em>&lt; 0.0001). Stepwise selection model analyses indicated that the G + C content of CDSs are the most important factor responsible for codon bias (R<sup>2</sup> = 0.6831, <em>P</em>&lt; 0.0001). It appears that variation in the G + C content of CDSs explains over 60% of the variation of codon bias. Recently, a continuous-time Markov chain model to quantify the contribution of GC-biased synonymous substitution on codon usage was developed by Palidwor <em>et al</em>.<sup><a reveal-id="DSR035C73" data-open="DSR035C73" class="link link-ref link-reveal xref-bibr">73</a></sup> Although many other important factors such as selection, GC skew, did not included in their model, it also provided an informative clue to understand the mechanism of codon usage across a broad variety of organisms. This model indicated that GC bias is the dominant factor in determining codon bias for prokaryotes, plants and human. In the present study, our data suggested that both mutation bias and selection contributed to the codon bias. This seems to be consistent with the few studies in other vertebrates such as <em>Xenopus laevis</em> and fishes of Cyprinidae.<sup><a reveal-id="DSR035C74  DSR035C75" data-open="DSR035C74  DSR035C75" class="link link-ref link-reveal xref-bibr">74–75</a></sup> However, it should be noticed that mutation bias is the driving force of the codon usage in the <em>G. gallus</em> genome.</p>                            <h3 scrollto-destination=7225066 id="7225066" class="section-title" data-legacy-id=s4c>4.3. The negative association between codon bias and recombination in <em>G. gallus</em> is determined by recombination-dependent mutational patterns</h3>
<p>In contrast to <em>D. melanogaster</em> and <em>C. elegans</em>, a negative correlation between codon bias and local recombination rates was found in the chicken genome. The positive pattern in <em>D. melanogaster</em> and <em>C. elegans</em> has been interpreted by Hill–Robertson effects (hitchhiking and background selection)<sup><a reveal-id="DSR035C28" data-open="DSR035C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR035C29" data-open="DSR035C29" class="link link-ref link-reveal xref-bibr">29</a></sup> or by recombination-dependent mutational patterns (gene conversion).<sup><a reveal-id="DSR035C32" data-open="DSR035C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSR035C36" data-open="DSR035C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The <em>C. elegans</em> has 21 optimal codons, of which 16 end in G or C bases, and <em>D. melanogaster</em> has 22 optimal codons, of which 21 end in G or C bases. Marais <em>et al</em>. demonstrated that, in <em>C. elegans</em>, the frequency of GC-ending optimal codons (Fop-GC) increases with recombination rate, whereas the frequency of AU-ending optimal codons (Fop-AU) decreases with recombination rate. In <em>Drosophila</em>, the frequency of AU-ending non-optimal codons (Fnop-AU) decreases with recombination rate, whereas the frequency of GC ending non-optimal codons (Fnop-GC) increases with recombination rate.<sup><a reveal-id="DSR035C32" data-open="DSR035C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In yeasts, about 60% of the optimal codons end by GC, an overall positive correlation is also observed between recombination rate and Fop. However, there is a strong negative correlation between Fop-AU and recombination rate.<sup><a reveal-id="DSR035C35" data-open="DSR035C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Marais and Piganeau<sup><a reveal-id="DSR035C32" data-open="DSR035C32" class="link link-ref link-reveal xref-bibr">32</a></sup> suggested that the positive correlation between the frequency of optimal codons and recombination rates in <em>C. elegans</em> and <em>D. melanogaster</em> is not due to improved selection but to a mutational bias toward G and C bases in regions of high recombination rates (GC-biased gene conversion). If mutation bias variation patterns do occur, they should affect all base positions within the gene, including coding and non-coding sequences. We surely found this positive trend between recombination rate and G + C content of introns in the chickens (<em>r</em> = 0.2467, <em>P</em>&lt; 0.0001). The mutational bias explanation seems to be the case in <em>G. gallus</em>. As 11 putative optimal codons identified in this study all ended in U, a negative association between codon bias and recombination rate is expected.</p><p>Another question that should be addressed is whether the correlation between the codon usage bias and recombination rate in <em>G. gallus</em> is a direct consequence of the recombination process. Based on the chicken consensus linkage map,<sup><a reveal-id="DSR035C51" data-open="DSR035C51" class="link link-ref link-reveal xref-bibr">51</a></sup> we selected some chromosome centromere region (chromosome 1–13, chromosome 17, chromosome 23, chromosome 25, chromosome 28 and chromosome Z), and some non-centromere regions where the estimated recombination rate is null (chromosome 1: 96.4173–99.0780 Mbp and 102.5560–105.82 39 Mbp; chromosome 2: 24.8946–27.1233 Mbp; chromosome 3: 62.328–65.3834 and 91.8723–93.5081 Mbp; chromosome 4: 20.8076–22.4734 and 72.3143–75.3621 and, 76.1848–79.2 626 Mbp; chromosome 5: 31.8527–33.3023 Mbp; 33.3360–35.0246 Mbp; chromosome 6: 13.24 26–16.2627 Mbp; chromosome 7: 8.8062–11.1527 Mbp; chromosome 8: 11.7298–12.7 581 Mbp; chromosome 19: 7.8870–8.84 82 Mbp; chromosome 20: 7.1329–8.3848 Mbp), and compared their G + C content of CDSs, G + C content of introns with those of the top recombination regions (same number of intervals on the same chromosome were selected). Although the recombination rates are likely to vary in different populations, the above non-centromere regions have been identified in that no recombination occurred in a outbreed chicken population established from a crossbreeding between a Xinghua line and a White Recessive Rock line by a high-density SNP microarray (556 individuals, unpublished data). We found that the G + C content of CDSs at the high recombination regions (0.5164 ± 0.0056) is significantly higher than that of the regions incurring no recombination (0.4695 ± 0.0056; <em>P</em> &lt; 0.0001, <em>t</em>-test). The G + C content of introns of genes residing in the high recombination regions (0.4489 ± 0.0061) is also significantly higher than that of regions having no recombination(0.3912 ± 0.0044; <em>P</em>&lt; 0.0001, <em>t</em>-test). This implies that the correlation between codon usage bias and recombination rates in <em>G. gallus</em> is determined predominantly by recombination-dependent mutational patterns. However, this does not mean that selection did not act on the synonymous sites. It is more likely that their impact on codon usage has been masked by variations in mutation pressures associated with the high recombination rates in chickens.</p>                            <h2 scrollto-destination=7225069 id="7225069" class="section-title" data-legacy-id=s5>4. Conclusion</h2>
<p>In this whole genome analysis of the chicken, we identified 11 putative optimal codons, which all ended with U. There is a good correspondence between tRNA abundance and optimal codons within codon classes. Codon bias is negatively correlated with G + C content and recombination rates, but positively correlated with gene expression, protein length and intron length. The G + C content of coding sequences are the most important factors responsible for codon bias. It appears that variation in the G + C content of CDSs explains over 60% of the variation of codon bias. Our study suggests that both mutation bias and selection contribute to the codon bias. However, mutation bias is the driving force of the codon usage in the <em>G. gallus</em> genome. Our data also provide evidence that the negative correlation between codon bias and recombination rates in <em>G. gallus</em> is determined predominantly by recombination-dependent mutational patterns.</p>                            <h2 scrollto-destination=7225071 id="7225071" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7225073 id="7225073" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the Science and Technology Program of Jiangxi Education Department, Project No. GJJ8469, and the High Tech Program (863), China, Project No. 2006AA10A120.</p>                            <h2 scrollto-destination=7225075 id="7225075" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank three reviewers for their helpful comments on the manuscript.</p>                            <h2 scrollto-destination=7225077 id="7225077" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR035C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Codon usage in bacteria: correlation with gene expressivity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1982</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">7055</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20bacteria%3A%20correlation%20with%20gene%20expressivity&amp;author=M.%20Gouy&amp;author=C.%20Gautier&amp;publication_year=1982&amp;journal=Nucleic%20Acids%20Res.&amp;volume=10&amp;pages=7055-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/10.22.7055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F10.22.7055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F10.22.7055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6760125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em></div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">4482</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20pattern%20and%2C%20surprisingly%2C%20gene%20length%20shape%20codon%20usage%20in%20Caenorhabditis%2C%20Drosophila%2C%20and%20Arabidopsis&amp;author=L.%20Duret&amp;author=D.%20Mouchiroud&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=96&amp;pages=4482-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.8.4482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.8.4482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.8.4482"> </span></div> </div></div></div></div></div><div content-id="DSR035C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huh</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Bower</div> <div class="given-names">K.R.</div></div>, et al. </div><div class="article-title">Global analysis of protein expression in <em>yeast</em></div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">737</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20protein%20expression%20in%20yeast&amp;author=S.%20Ghaemmaghami&amp;author=W.K.%20Huh&amp;author=K.R.%20Bower&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=737-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goetz</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Fuglsang</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Correlation of codon bias measures with mRNA levels: analysis of transcriptome data from <em>Escherichia coli</em></div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2005</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">4</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20of%20codon%20bias%20measures%20with%20mRNA%20levels%3A%20analysis%20of%20transcriptome%20data%20from%20Escherichia%20coli&amp;author=R.M.%20Goetz&amp;author=A.%20Fuglsang&amp;publication_year=2005&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=327&amp;pages=4-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2004.11.134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2004.11.134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2004.11.134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ingvarsson</div> <div class="given-names">P.K.</div></div>. </div><div class="article-title">Gene expression and protein length influence codon usage and rates of sequence evolution in <em>Populus tremula</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">836</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20and%20protein%20length%20influence%20codon%20usage%20and%20rates%20of%20sequence%20evolution%20in%20Populus%20tremula&amp;author=P.K.%20Ingvarsson&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=836-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17204548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multi-cellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multi-cellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson</div> <div class="given-names">S.G.E.</div></div>,  <div class="name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div>. </div><div class="article-title">Codon preferences in free-living microorganisms</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1990</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">198</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20preferences%20in%20free-living%20microorganisms&amp;author=S.G.E.%20Andersson&amp;author=C.G.%20Kurland&amp;publication_year=1990&amp;journal=Microbiol.%20Rev.&amp;volume=54&amp;pages=198-210" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2194095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Percudani</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Pavesi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ottonello</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Transfer RNA gene redundancy and translational selection in <em>Saccharomyces cerevisiae</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">322</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20RNA%20gene%20redundancy%20and%20translational%20selection%20in%20Saccharomyces%20cerevisiae&amp;author=R.%20Percudani&amp;author=A.%20Pavesi&amp;author=S.%20Ottonello&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=268&amp;pages=322-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.0942" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.0942" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.0942"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9159473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kudo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of <em>Bacillus subtilis</em> tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis</div>, <div class="source">Gene</div> , <div class="year">1999</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">143</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20of%20codon%20usage%20and%20tRNA%20genes%20of%2018%20unicellular%20organisms%20and%20quantification%20of%20Bacillus%20subtilis%20tRNAs%3A%20gene%20expression%20level%20and%20species-specific%20diversity%20of%20codon%20usage%20based%20on%20multivariate%20analysis&amp;author=S.%20Kanaya&amp;author=Y.%20Yamada&amp;author=Y.%20Kudo&amp;author=T.%20Ikemura&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=143-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00225-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00225-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00225-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Andachi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Muto</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Osawa</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Levels of tRNAs in bacterial cells as affected by amino acid usage in proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1991</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">6119</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Levels%20of%20tRNAs%20in%20bacterial%20cells%20as%20affected%20by%20amino%20acid%20usage%20in%20proteins&amp;author=F.%20Yamao&amp;author=Y.%20Andachi&amp;author=A.%20Muto&amp;author=T.%20Ikemura&amp;author=S.%20Osawa&amp;publication_year=1991&amp;journal=Nucleic%20Acids%20Res.&amp;volume=19&amp;pages=6119-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.22.6119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.22.6119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.22.6119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1956771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes</div>, <div class="source">Trends Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">287</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20gene%20number%20and%20codon%20usage%20in%20the%20C.%20elegans%20genome%20are%20co-adapted%20for%20optimal%20translation%20of%20highly%20expressed%20genes&amp;author=L.%20Duret&amp;publication_year=2000&amp;journal=Trends%20Genet.&amp;volume=16&amp;pages=287-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02041-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02041-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02041-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10858656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kudo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">290</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20genes%20in%20eukaryotes%3A%20correlation%20of%20codon%20usage%20diversity%20with%20translation%20efficiency%20and%20with%20CG-dinucleotide%20usage%20as%20assessed%20by%20multivariate%20analysis&amp;author=S.%20Kanaya&amp;author=Y.%20Yamada&amp;author=M.%20Kinouchi&amp;author=Y.%20Kudo&amp;author=T.%20Ikemura&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol.&amp;volume=53&amp;pages=290-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002390010219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11675589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.</div></div>. </div><div class="article-title">Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2279</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20from%20tRNA's%20point%20of%20view%3A%20redundancy%2C%20specialization%2C%20and%20efficient%20decoding%20for%20translation%20optimization&amp;author=E.P.%20Rocha&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2896904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2896904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2896904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorensen</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div>,  <div class="name"><div class="surname">Pedersen</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Codon usage determines translation rate in <em>Escherichia coli</em>.</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1989</div>, vol. <div class="volume">207</div> (pg. <div class="fpage">365</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20determines%20translation%20rate%20in%20Escherichia%20coli.&amp;author=M.A.%20Sorensen&amp;author=C.G.%20Kurland&amp;author=S.%20Pedersen&amp;publication_year=1989&amp;journal=J.%20Mol.%20Biol.&amp;volume=207&amp;pages=365-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(89)90260-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(89)90260-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(89)90260-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2474074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Evolution of synonymous codon usage in metazoans</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">640</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20synonymous%20codon%20usage%20in%20metazoans&amp;author=L.%20Duret&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=12&amp;pages=640-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(02)00353-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(02)00353-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(02)00353-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12433576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoletzki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">374</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Escherichia%20coli%3A%20selection%20for%20translational%20accuracy&amp;author=N.%20Stoletzki&amp;author=A.%20Eyre-Walker&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=374-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17101719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Selection on codon bias</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">287</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20codon%20bias&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=42&amp;pages=287-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18983258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Fmery</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Forces that influence the evolution of codon bias</div>, <div class="source">Phil. Trans. R. Soc. B</div> , <div class="year">2010</div>, vol. <div class="volume">1544</div> (pg. <div class="fpage">1203</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forces%20that%20influence%20the%20evolution%20of%20codon%20bias&amp;author=P.M.%20Sharp&amp;author=L.%20Fmery&amp;author=K.%20Zeng&amp;publication_year=2010&amp;journal=Phil.%20Trans.%20R.%20Soc.%20B&amp;volume=1544&amp;pages=1203-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2009.0305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2009.0305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2009.0305"> </span></div> </div></div></div></div></div><div content-id="DSR035C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Knight</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Freeland</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Landweber</div> <div class="given-names">L.F.</div></div>. </div><div class="article-title">A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div>  <div class="comment">RESEARCH0010</div></div></div></div></div><div content-id="DSR035C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hottes</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">McAdams</div> <div class="given-names">H.H.</div></div>. </div><div class="article-title">Codon usage between genomes is constrained by genome-wide mutational processes</div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">3480</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20between%20genomes%20is%20constrained%20by%20genome-wide%20mutational%20processes&amp;author=S.L.%20Chen&amp;author=W.%20Lee&amp;author=A.K.%20Hottes&amp;author=L.%20Shapiro&amp;author=H.H.%20McAdams&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=101&amp;pages=3480-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307827100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307827100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307827100"> </span></div> </div></div></div></div></div><div content-id="DSR035C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">General rules for optimal codon choice</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20rules%20for%20optimal%20codon%20choice&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000556"> </span></div> </div></div></div></div></div><div content-id="DSR035C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Evidence of selectively driven codon usage in rice: implications for G + C content evolution of Gramineae genes</div>, <div class="source">FEBS Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">581</div> (pg. <div class="fpage">1015</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20selectively%20driven%20codon%20usage%20in%20rice%3A%20implications%20for%20G%20%2B%20C%20content%20evolution%20of%20Gramineae%20genes&amp;author=X.%20Guo&amp;author=J.%20Bao&amp;author=L.%20Fan&amp;publication_year=2007&amp;journal=FEBS%20Lett.&amp;volume=581&amp;pages=1015-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2007.01.088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2007.01.088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2007.01.088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17306258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bulmer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The selection-mutation-drift theory of synonymous codon usage</div>, <div class="source">Genetics</div> , <div class="year">1991</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">897</div>-<div class="lpage">907</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20selection-mutation-drift%20theory%20of%20synonymous%20codon%20usage&amp;author=M.%20Bulmer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=897-907" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1752426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Inferring weak selection from patterns of polymorphism and divergence at ‘silent’ sites in <em>Drosophila</em> DNA</div>, <div class="source">Genetics</div> , <div class="year">1995</div>, vol. <div class="volume">139</div> (pg. <div class="fpage">1067</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20weak%20selection%20from%20patterns%20of%20polymorphism%20and%20divergence%20at%20%E2%80%98silent%E2%80%99%20sites%20in%20Drosophila%20DNA&amp;author=H.%20Akashi&amp;publication_year=1995&amp;journal=Genetics&amp;volume=139&amp;pages=1067-76" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7713409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kliman</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Mutation pressure, natural selection, and the evolution of base composition in <em>Drosophila</em></div>, <div class="source">Genetica</div> , <div class="year">1998</div>, vol. <div class="volume">102–103</div> (pg. <div class="fpage">49</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20pressure%2C%20natural%20selection%2C%20and%20the%20evolution%20of%20base%20composition%20in%20Drosophila&amp;author=H.%20Akashi&amp;author=R.M.%20Kliman&amp;author=A.%20Eyre-Walker&amp;publication_year=1998&amp;journal=Genetica&amp;volume=102–103&amp;pages=49-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1017078607465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1017078607465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1017078607465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9720271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">dos Reis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Estimating translational selection in eukaryotic genomes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">451</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20translational%20selection%20in%20eukaryotic%20genomes&amp;author=M.%20dos%20Reis&amp;author=L.%20Wernisch&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=26&amp;pages=451-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1986</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">28</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20perspective%20on%20synonymous%20codon%20usage%20in%20unicellular%20organisms&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1986&amp;journal=J.%20Mol.%20Evol.&amp;volume=24&amp;pages=28-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02099948" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02099948" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02099948"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3104616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kliman</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Hey</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Reduced natural selection associated with low recombination in <em>Drosophila melanogaster</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1993</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1239</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20natural%20selection%20associated%20with%20low%20recombination%20in%20Drosophila%20melanogaster&amp;author=R.M.%20Kliman&amp;author=J.%20Hey&amp;publication_year=1993&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=10&amp;pages=1239-58" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8277853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Comeron</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Kreitman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aguade</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Natural selection on synonymous sites is correlated with gene length and recombination in <em>Drosophila</em></div>, <div class="source">Genetics</div> , <div class="year">1999</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">239</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20selection%20on%20synonymous%20sites%20is%20correlated%20with%20gene%20length%20and%20recombination%20in%20Drosophila&amp;author=J.M.%20Comeron&amp;author=M.%20Kreitman&amp;author=M.%20Aguade&amp;publication_year=1999&amp;journal=Genetics&amp;volume=151&amp;pages=239-49" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872963" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Comeron</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Kreitman</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Population, evolutionary and genomic consequences of interference selection</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">389</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%2C%20evolutionary%20and%20genomic%20consequences%20of%20interference%20selection&amp;author=J.M.%20Comeron&amp;author=M.%20Kreitman&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=389-410" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12019253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hey</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kliman</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Interactions between natural selection, recombination and gene density in the genes of <em>Drosophila</em></div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">160</div> (pg. <div class="fpage">595</div>-<div class="lpage">608</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactions%20between%20natural%20selection%2C%20recombination%20and%20gene%20density%20in%20the%20genes%20of%20Drosophila&amp;author=J.%20Hey&amp;author=R.M.%20Kliman&amp;publication_year=2002&amp;journal=Genetics&amp;volume=160&amp;pages=595-608" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11861564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marais</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Piganeau</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Hill-Robertson interference is a minor determinant of variations in codon bias across <em>Drosophila melanogaster</em> and <em>Caenorhabditis elegans</em> genome</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2002</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1399</div>-<div class="lpage">406</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hill-Robertson%20interference%20is%20a%20minor%20determinant%20of%20variations%20in%20codon%20bias%20across%20Drosophila%20melanogaster%20and%20Caenorhabditis%20elegans%20genome&amp;author=G.%20Marais&amp;author=G.%20Piganeau&amp;publication_year=2002&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=19&amp;pages=1399-406" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a004203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a004203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12200468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marais</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Neutral effect of recombination on base composition in <em>Drosophila</em></div>, <div class="source">Genet. Res.</div> , <div class="year">2003</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">79</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neutral%20effect%20of%20recombination%20on%20base%20composition%20in%20Drosophila&amp;author=G.%20Marais&amp;author=D.%20Mouchiroud&amp;author=L.%20Duret&amp;publication_year=2003&amp;journal=Genet.%20Res.&amp;volume=81&amp;pages=79-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672302006079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672302006079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672302006079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12872909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haddrill</div> <div class="given-names">P.R.</div></div>,  <div class="name"><div class="surname">Halligan</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Tomaras</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Reduced efficacy of selection in regions of the <em>Drosophila</em> genome that lack crossing over</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R18</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20efficacy%20of%20selection%20in%20regions%20of%20the%20Drosophila%20genome%20that%20lack%20crossing%20over&amp;author=P.R.%20Haddrill&amp;author=D.L.%20Halligan&amp;author=D.%20Tomaras&amp;author=B.%20Charlesworth&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-2-r18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-2-r18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-2-r18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17284312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Biased gene conversion affects patterns of codon usage and amino acid usage in the Saccharomyces <em>sens stricto</em> group of yeasts</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">117</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20gene%20conversion%20affects%20patterns%20of%20codon%20usage%20and%20amino%20acid%20usage%20in%20the%20Saccharomyces%20sens%20stricto%20group%20of%20yeasts&amp;author=R.%20Harrison&amp;author=B.%20Charlesworth&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=117-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20656793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marais</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Does recombination improve selection on codon usage? Lessons from nematode and fly complete genomes</div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">5688</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Does%20recombination%20improve%20selection%20on%20codon%20usage%3F%20Lessons%20from%20nematode%20and%20fly%20complete%20genomes&amp;author=G.%20Marais&amp;author=D.%20Mouchiroud&amp;author=L.%20Duret&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=98&amp;pages=5688-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.091427698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.091427698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.091427698"> </span></div> </div></div></div></div></div><div content-id="DSR035C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moriyama</div> <div class="given-names">E.N.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Gene length and codon usage bias in <em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae</em> and <em>Escherichia coli</em>.</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">3188</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20length%20and%20codon%20usage%20bias%20in%20Drosophila%20melanogaster%2C%20Saccharomyces%20cerevisiae%20and%20Escherichia%20coli.&amp;author=E.N.%20Moriyama&amp;author=J.R.%20Powell&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=3188-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.13.3188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.13.3188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.13.3188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9628917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lemos</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bettencourt</div> <div class="given-names">B.R.</div></div>,  <div class="name"><div class="surname">Meiklejohn</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Hartl</div> <div class="given-names">D.L.</div></div>. </div><div class="article-title">Evolution of proteins and gene expression levels are coupled in Drosophila and are independently associated with mRNA abundance, protein length and number of protein-protein interactions</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1345</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20proteins%20and%20gene%20expression%20levels%20are%20coupled%20in%20Drosophila%20and%20are%20independently%20associated%20with%20mRNA%20abundance%2C%20protein%20length%20and%20number%20of%20protein-protein%20interactions&amp;author=B.%20Lemos&amp;author=B.R.%20Bettencourt&amp;author=C.D.%20Meiklejohn&amp;author=D.L.%20Hartl&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=1345-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15746013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stenoien</div> <div class="given-names">H.K.</div></div>. </div><div class="article-title">Adaptive basis of codon usage in the haploid <em>moss Physcomitrella patens</em></div>, <div class="source">Heredity</div> , <div class="year">2005</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">87</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20basis%20of%20codon%20usage%20in%20the%20haploid%20moss%20Physcomitrella%20patens&amp;author=H.K.%20Stenoien&amp;publication_year=2005&amp;journal=Heredity&amp;volume=94&amp;pages=87-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800547" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800547" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800547"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15483656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groenen</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Bumstead</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A consensus linkage map of the chicken genome</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">137</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20linkage%20map%20of%20the%20chicken%20genome&amp;author=M.A.%20Groenen&amp;author=H.H.%20Cheng&amp;author=N.%20Bumstead&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=137-47" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10645958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="collab">International Chicken Genome Sequencing Consortium (ICGSC).</div><div class="article-title">Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">432</div> (pg. <div class="fpage">695</div>-<div class="lpage">716</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03154" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03154"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15592404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Webster</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Axelsson</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Strong regional biases in nucleotide substitution in the chicken genome</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1203</div>-<div class="lpage">216</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20regional%20biases%20in%20nucleotide%20substitution%20in%20the%20chicken%20genome&amp;author=M.T.%20Webster&amp;author=E.%20Axelsson&amp;author=H.%20Ellegren&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1203-216" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msk008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsk008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsk008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16551647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Isochores and the evolutionary genomics of vertebrates</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">241</div> (pg. <div class="fpage">3</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isochores%20and%20the%20evolutionary%20genomics%20of%20vertebrates&amp;author=G.%20Bernardi&amp;publication_year=2000&amp;journal=Gene&amp;volume=241&amp;pages=3-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00485-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00485-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00485-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10607893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Costantini</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Correlations between coding and contiguous non-coding sequences in isochore families from vertebrate genomes</div>, <div class="source">Gene</div> , <div class="year">2008</div>, vol. <div class="volume">410</div> (pg. <div class="fpage">241</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlations%20between%20coding%20and%20contiguous%20non-coding%20sequences%20in%20isochore%20families%20from%20vertebrate%20genomes&amp;author=M.%20Costantini&amp;author=G.%20Bernardi&amp;publication_year=2008&amp;journal=Gene&amp;volume=410&amp;pages=241-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.12.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.12.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.12.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18252269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="collab">International Human Genome Sequencing Consortium.</div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mank</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Hultin-Rosenberg</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Axelsson</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Rapid evolution of female-biased, but not male-biased, genes expressed in the avian brain</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2698</div>-<div class="lpage">706</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20evolution%20of%20female-biased%2C%20but%20not%20male-biased%2C%20genes%20expressed%20in%20the%20avian%20brain&amp;author=J.E.%20Mank&amp;author=L.%20Hultin-Rosenberg&amp;author=E.%20Axelsson&amp;author=H.%20Ellegren&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=2698-706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm208" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm208" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17893399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Castillo-Davis</div> <div class="given-names">C.I.</div></div>,  <div class="name"><div class="surname">Mekhedov</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Hartl</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Kondrashov</div> <div class="given-names">F.A.</div></div>. </div><div class="article-title">Selection for short introns in highly expressed genes</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">415</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20for%20short%20introns%20in%20highly%20expressed%20genes&amp;author=C.I.%20Castillo-Davis&amp;author=S.L.%20Mekhedov&amp;author=D.L.%20Hartl&amp;author=E.V.%20Koonin&amp;author=F.A.%20Kondrashov&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=31&amp;pages=415-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12134150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rao</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.F.</div></div>,  <div class="name"><div class="surname">Chai</div> <div class="given-names">X.W.</div></div>, et al. </div><div class="article-title">Selection for the compactness of highly expressed genes in <em>Gallus gallus</em></div>, <div class="source">Biology Direct</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">35</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20for%20the%20compactness%20of%20highly%20expressed%20genes%20in%20Gallus%20gallus&amp;author=Y.S.%20Rao&amp;author=Z.F.%20Wang&amp;author=X.W.%20Chai&amp;publication_year=2010&amp;journal=Biology%20Direct&amp;volume=5&amp;pages=35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-5-35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-5-35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-5-35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20465857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yanai</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Benjamin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shmoish</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">650</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20midrange%20transcription%20profiles%20reveal%20expression%20level%20relationships%20in%20human%20tissue%20specification&amp;author=I.%20Yanai&amp;author=H.%20Benjamin&amp;author=M.%20Shmoish&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=650-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15388519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cutter</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Selection intensity on preferred codons correlates with overall codon usage bias in Caenorhabditis remanei</div>, <div class="source">Curr. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">2053</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20intensity%20on%20preferred%20codons%20correlates%20with%20overall%20codon%20usage%20bias%20in%20Caenorhabditis%20remanei&amp;author=A.D.%20Cutter&amp;author=B.%20Charlesworth&amp;publication_year=2006&amp;journal=Curr.%20Biol.&amp;volume=16&amp;pages=2053-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2006.08.067" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2006.08.067" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2006.08.067"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17055986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groenen</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Wahlberg</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Foglio</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">510</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20SNP-based%20linkage%20map%20of%20the%20chicken%20genome%20reveals%20sequence%20features%20correlated%20with%20recombination%20rate&amp;author=M.A.%20Groenen&amp;author=P.%20Wahlberg&amp;author=M.%20Foglio&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=510-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.086538.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.086538.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.086538.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19088305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Payseur</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Nachman</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">Microsatellite variation and recombination rate in the human genome</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1285</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20variation%20and%20recombination%20rate%20in%20the%20human%20genome&amp;author=B.A.%20Payseur&amp;author=M.W.%20Nachman&amp;publication_year=2000&amp;journal=Genetics&amp;volume=156&amp;pages=1285-98" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11063702" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holm</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A simple sequentially rejective Bonferroni test procedure</div>, <div class="source">Scand. J. Stat.</div> , <div class="year">1979</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">65</div>-<div class="lpage">70</div>)</div></div></div></div><div content-id="DSR035C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div>. </div><div class="article-title">Co-variation of tRNA abundance and codon usage in <em>Escherichia coli</em> at different growth rates</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">260</div> (pg. <div class="fpage">649</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-variation%20of%20tRNA%20abundance%20and%20codon%20usage%20in%20Escherichia%20coli%20at%20different%20growth%20rates&amp;author=H.%20Dong&amp;author=L.%20Nilsson&amp;author=C.G.%20Kurland&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=260&amp;pages=649-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8709146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Comeron</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Selective and mutational patterns associated with gene expression in humans: influences on synonymous composition and intron presence</div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">1293</div>-<div class="lpage">1304</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20and%20mutational%20patterns%20associated%20with%20gene%20expression%20in%20humans%3A%20influences%20on%20synonymous%20composition%20and%20intron%20presence&amp;author=J.M.%20Comeron&amp;publication_year=2004&amp;journal=Genetics&amp;volume=167&amp;pages=1293-1304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.026351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.026351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.026351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15280243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lavner</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kotlar</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Codon bias as a factor in regulating expression via translation rate in the human genome</div>, <div class="source">Gene</div> , <div class="year">2005</div>, vol. <div class="volume">345</div> (pg. <div class="fpage">127</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20as%20a%20factor%20in%20regulating%20expression%20via%20translation%20rate%20in%20the%20human%20genome&amp;author=Y.%20Lavner&amp;author=D.%20Kotlar&amp;publication_year=2005&amp;journal=Gene&amp;volume=345&amp;pages=127-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.11.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.11.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.11.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shah</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gilchrist</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Effect of correlated tRNA abundances on translation errors and evolution of codon usage bias</div>, <div class="source">PLoS Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1001128</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20correlated%20tRNA%20abundances%20on%20translation%20errors%20and%20evolution%20of%20codon%20usage%20bias&amp;author=P.%20Shah&amp;author=M.A.%20Gilchrist&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1001128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20862306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cognat</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Deragon</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Vinogradova</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Salinas</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Remacle</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Maréchal-Drouard</div> <div class="given-names">L.</div></div>. </div><div class="article-title">On the evolution and expression of Chlamydomonas reinhardtii nucleus-encoded transfer RNA genes</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">113</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20evolution%20and%20expression%20of%20Chlamydomonas%20reinhardtii%20nucleus-encoded%20transfer%20RNA%20genes&amp;author=V.%20Cognat&amp;author=J.M.%20Deragon&amp;author=E.%20Vinogradova&amp;author=T.%20Salinas&amp;author=C.%20Remacle&amp;author=L.%20Mar%C3%A9chal-Drouard&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=113-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.085688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.085688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.085688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18493044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parmley</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Huynen</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Clustering of codons with rare cognate tRNAs in human genes suggests an extra level of expression regulation</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000548</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustering%20of%20codons%20with%20rare%20cognate%20tRNAs%20in%20human%20genes%20suggests%20an%20extra%20level%20of%20expression%20regulation&amp;author=J.L.%20Parmley&amp;author=M.A.%20Huynen&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19578405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Gene expression and molecular evolution</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">660</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20and%20molecular%20evolution&amp;author=H.%20Akashi&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=11&amp;pages=660-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(00)00250-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(00)00250-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(00)00250-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11682310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Urrutia</div> <div class="given-names">A.O.</div></div>,  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>. </div><div class="article-title">Codon usage bias covaries with expression breadth and the rate of synonymous evolution in humans, but this is not evidence for selection</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">1191</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20covaries%20with%20expression%20breadth%20and%20the%20rate%20of%20synonymous%20evolution%20in%20humans%2C%20but%20this%20is%20not%20evidence%20for%20selection&amp;author=A.O.%20Urrutia&amp;author=L.D.%20Hurst&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=1191-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11729162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em></div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">4482</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20pattern%20and%2C%20surprisingly%2C%20gene%20length%20shape%20codon%20usage%20in%20Caenorhabditis%2C%20Drosophila%2C%20and%20Arabidopsis&amp;author=D.%20Mouchiroud&amp;author=L.%20Duret&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=96&amp;pages=4482-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.8.4482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.8.4482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.8.4482"> </span></div> </div></div></div></div></div><div content-id="DSR035C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Loewe</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Background selection in single genes may explain patterns of codon bias</div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">1381</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Background%20selection%20in%20single%20genes%20may%20explain%20patterns%20of%20codon%20bias&amp;author=B.%20Charlesworth&amp;author=L.%20Loewe&amp;publication_year=2007&amp;journal=Genetics&amp;volume=175&amp;pages=1381-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17194784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qiu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bergero</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Patterns of codon usage bias in <em>Silene latifolia</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">771</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20codon%20usage%20bias%20in%20Silene%20latifolia&amp;author=S.%20Qiu&amp;author=R.%20Bergero&amp;author=K.%20Zeng&amp;author=D.%20Charlesworth&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=771-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20855431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walker</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">Synonymous codon bias is related to gene length in <em>Escherichia coli</em>: selection for translational accuracy?</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">864</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20bias%20is%20related%20to%20gene%20length%20in%20Escherichia%20coli%3A%20selection%20for%20translational%20accuracy%3F&amp;author=A.E.%20Walker&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=13&amp;pages=864-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8754221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marais</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nouvellet</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Keightley</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Intron size and exon ewolution in <em>Drosophila</em></div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">170</div> (pg. <div class="fpage">481</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20size%20and%20exon%20ewolution%20in%20Drosophila&amp;author=G.%20Marais&amp;author=P.%20Nouvellet&amp;author=P.D.%20Keightley&amp;author=B.%20Charlesworth&amp;publication_year=2005&amp;journal=Genetics&amp;volume=170&amp;pages=481-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.037333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.037333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.037333"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15781704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stenico</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">Codon usage in <em>Caenorhabditis elegans</em>: delineation of translational selection and mutational biases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2437</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20Caenorhabditis%20elegans%3A%20delineation%20of%20translational%20selection%20and%20mutational%20biases&amp;author=M.%20Stenico&amp;author=A.T.%20Lloyd&amp;author=P.M.%20Sharp&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=13&amp;pages=2437-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.13.2437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.13.2437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.13.2437"> </span></div> </div></div></div></div></div><div content-id="DSR035C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vinogradov</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">Intron length and codon usage</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">2</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20length%20and%20codon%20usage&amp;author=A.E.%20Vinogradov&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol.&amp;volume=52&amp;pages=2-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002390010128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11139289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoletzki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">374</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Escherichia%20coli%3A%20selection%20for%20translational%20accuracy&amp;author=N.%20Stoletzki&amp;author=A.%20Eyre-Walker&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=374-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17101719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoletzki</div> <div class="given-names">N.</div></div>. </div><div class="article-title">The surprising negative correlation of gene length and optimal codon use—disentangling translational selection from GC-biased gene conversion in yeast</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">93</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20surprising%20negative%20correlation%20of%20gene%20length%20and%20optimal%20codon%20use%E2%80%94disentangling%20translational%20selection%20from%20GC-biased%20gene%20conversion%20in%20yeast&amp;author=N.%20Stoletzki&amp;publication_year=2011&amp;journal=BMC%20Evol.%20Biol.&amp;volume=11&amp;pages=93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-11-93" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-11-93" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-11-93"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21481245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vicario</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Moriyama</div> <div class="given-names">E.N.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Codon usage in twelve species of <em>Drosophila</em></div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> pg. <div class="fpage">226</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20twelve%20species%20of%20Drosophila&amp;author=S.%20Vicario&amp;author=E.N.%20Moriyama&amp;author=J.R.%20Powell&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=226" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18005411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nekrutenko</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Assessment of compositional heterogeneity within and between eukaryotic genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1986</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20compositional%20heterogeneity%20within%20and%20between%20eukaryotic%20genomes&amp;author=A.%20Nekrutenko&amp;author=W.H.%20Li&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1986-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.12.1986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.12.1986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.12.1986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11116093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palidwor</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Perkins</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">X.</div></div>. </div><div class="article-title">A general model of codon bias due to GC mutational bias</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e13431</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20model%20of%20codon%20bias%20due%20to%20GC%20mutational%20bias&amp;author=G.A.%20Palidwor&amp;author=T.J.%20Perkins&amp;author=X.%20Xia&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e13431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cruveiller</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">D'Onofrio</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Translational selection on codon usage in Xenopus laevis</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1703</div>-<div class="lpage">07</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20selection%20on%20codon%20usage%20in%20Xenopus%20laevis&amp;author=H.%20Musto&amp;author=S.%20Cruveiller&amp;author=G.%20D'Onofrio&amp;author=H.%20Romero&amp;author=G.%20Bernardi&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=1703-07" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11504850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR035C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR035C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zavala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The influence of translational selection on codon usage in fishes from the family Cyprinidae</div>, <div class="source">Gene</div> , <div class="year">2003</div>, vol. <div class="volume">317</div> (pg. <div class="fpage">141</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20influence%20of%20translational%20selection%20on%20codon%20usage%20in%20fishes%20from%20the%20family%20Cyprinidae&amp;author=H.%20Romero&amp;author=A.%20Zavala&amp;author=H.%20Musto&amp;author=G.%20Bernardi&amp;publication_year=2003&amp;journal=Gene&amp;volume=317&amp;pages=141-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(03)00701-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(03)00701-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(03)00701-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14604802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=478052&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr035/2/dsr035-dsr035supp.doc?Expires=2147483647&amp;Signature=w0a9QQ5kH2sQSHYGARjmhB0AS6T66EySm5T3LbWPuI66ViTSjCaDmuThhFm~3kmS~Q2P6rMHKvPlfi-f5ussw82ZwkZzoj7f6JslefJLCc4tZmklrD4YVml5I2zwfLMwkHtzuMHbYjEWzWVzN7ohGaoSfw3dRRhgMFK5eilT5bUG1S3~MAI0R7uP-Qw~Qm1yYGVWIjMLr2XoXABcIm8PH-6KwW4D7f8UBEBvu6ZEny41KZMQ9d-WFDQCqlQNMkjmNhbszbumVYsdwfemCS5xsegfQLjeqNLNSV7nGNwsb2-nmZy4wipmKSbQM-z9T2rdpl24YXTsB5dbFD3IE-vJGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">363</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000297990700008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">40</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="478052"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000297990700008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f6%2f499%2f478052">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000297990700008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (40)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F6%2F499%2F478052" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/478052'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Mutation Bias is the Driving Force of Codon Usage in the Gallus gallus genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f499%2f478052">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

