

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/3/153/444270","@type":"ScholarlyArticle","name":"Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea ( Cajanus cajan  L.)","datePublished":"2011-05-12","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2011-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr007","keywords":["<italic>Cajanus cajan</italic> L.","next generation sequencing","transcriptome assembly","molecular markers and gene discovery"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/3/153/444270","author":[{"name":"Dubey, Anuja","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India    2  Barkatullah University ,  Bhopal 462 026 ,  India","@type":"Person"},{"name":"Farmer, Andrew","affiliation":"3  National Centre for Genome Resources (NCGR) ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Schlueter, Jessica","affiliation":"4  University of North Carolina at Charlotte ,  Charlotte, NC 28223 ,  USA","@type":"Person"},{"name":"Cannon, Steven B.","affiliation":"5  United States Department of Agriculture-Agricultural Research Service (USDA-ARS) ,  Corn Insects and Crop Genetics Research Unit ,  Ames ,  IA 50011 ,  USA","@type":"Person"},{"name":"Abernathy, Brian","affiliation":"6  Purdue University ,  West Lafayette, IN 47907-2054 ,  USA","@type":"Person"},{"name":"Tuteja, Reetu","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India","@type":"Person"},{"name":"Woodward, Jimmy","affiliation":"3  National Centre for Genome Resources (NCGR) ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Shah, Trushar","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India","@type":"Person"},{"name":"Mulasmanovic, Benjamin","affiliation":"5  United States Department of Agriculture-Agricultural Research Service (USDA-ARS) ,  Corn Insects and Crop Genetics Research Unit ,  Ames ,  IA 50011 ,  USA","@type":"Person"},{"name":"Kudapa, Himabindu","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India","@type":"Person"},{"name":"Raju, Nikku L.","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India","@type":"Person"},{"name":"Gothalwal, Ragini","affiliation":"2  Barkatullah University ,  Bhopal 462 026 ,  India","@type":"Person"},{"name":"Pande, Suresh","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India","@type":"Person"},{"name":"Xiao, Yongli","affiliation":"7  J. Craig Venter Institute (JCVI) ,  Rockville, MD 20850 ,  USA","@type":"Person"},{"name":"Town, Chris D.","affiliation":"7  J. Craig Venter Institute (JCVI) ,  Rockville, MD 20850 ,  USA","@type":"Person"},{"name":"Singh, Nagendra K.","affiliation":"8  National Research Centre on Plant Biotechnology ,  New Delhi 110012 ,  India","@type":"Person"},{"name":"May, Gregory D.","affiliation":"3  National Centre for Genome Resources (NCGR) ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Jackson, Scott","affiliation":"6  Purdue University ,  West Lafayette, IN 47907-2054 ,  USA","@type":"Person"},{"name":"Varshney, Rajeev K.","affiliation":"1  Centre of Excellence in Genomics (CEG), Building #300 ,  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502 324, Greater Hyderabad ,  India    9  CGIAR Generation Challenge Programme (GCP), c/o CIMMYT ,  06600 Mexico DF ,  Mexico","@type":"Person"}],"description":"Abstract.  This study reports generation of large-scale genomic resources for pigeonpea, a so-called ‘orphan crop species’ of the semi-arid tropic regions. FLX/","pageStart":"153","pageEnd":"164","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea ( Cajanus cajan  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea ( Cajanus cajan  L.)" />
    <meta property="og:description" content="Abstract.  This study reports generation of large-scale genomic resources for pigeonpea, a so-called ‘orphan crop species’ of the semi-arid tropic regions. FLX/" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/3/153/444270" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-05-12" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Dubey, Anuja" /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author_institution" content="Barkatullah University, Bhopal 462 026, India" /><meta name="citation_author" content="Farmer, Andrew" /><meta name="citation_author_institution" content="National Centre for Genome Resources (NCGR), Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Schlueter, Jessica" /><meta name="citation_author_institution" content="University of North Carolina at Charlotte, Charlotte, NC 28223, USA" /><meta name="citation_author" content="Cannon, Steven B." /><meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Corn Insects and Crop Genetics Research Unit, Ames, IA 50011, USA" /><meta name="citation_author" content="Abernathy, Brian" /><meta name="citation_author_institution" content="Purdue University, West Lafayette, IN 47907-2054, USA" /><meta name="citation_author" content="Tuteja, Reetu" /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author" content="Woodward, Jimmy" /><meta name="citation_author_institution" content="National Centre for Genome Resources (NCGR), Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Shah, Trushar" /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author" content="Mulasmanovic, Benjamin" /><meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Corn Insects and Crop Genetics Research Unit, Ames, IA 50011, USA" /><meta name="citation_author" content="Kudapa, Himabindu" /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author" content="Raju, Nikku L." /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author" content="Gothalwal, Ragini" /><meta name="citation_author_institution" content="Barkatullah University, Bhopal 462 026, India" /><meta name="citation_author" content="Pande, Suresh" /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author" content="Xiao, Yongli" /><meta name="citation_author_institution" content="J. Craig Venter Institute (JCVI), Rockville, MD 20850, USA" /><meta name="citation_author" content="Town, Chris D." /><meta name="citation_author_institution" content="J. Craig Venter Institute (JCVI), Rockville, MD 20850, USA" /><meta name="citation_author" content="Singh, Nagendra K." /><meta name="citation_author_institution" content="National Research Centre on Plant Biotechnology, New Delhi 110012, India" /><meta name="citation_author" content="May, Gregory D." /><meta name="citation_author_institution" content="National Centre for Genome Resources (NCGR), Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Jackson, Scott" /><meta name="citation_author_institution" content="Purdue University, West Lafayette, IN 47907-2054, USA" /><meta name="citation_author" content="Varshney, Rajeev K." /><meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), Building #300, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502 324, Greater Hyderabad, India" /><meta name="citation_author_institution" content="CGIAR Generation Challenge Programme (GCP), c/o CIMMYT, 06600 Mexico DF, Mexico" /><meta name="citation_title" content="Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea (Cajanus cajan L.)" /><meta name="citation_firstpage" content="153" /><meta name="citation_lastpage" content="164" /><meta name="citation_doi" content="10.1093/dnares/dsr007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2011/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome size variation in Cajanus cajan (Fabaceae): a reconsideration; Plant Syst. Evol.;  citation_year=1998;  citation_volume=212;  citation_pages=135-41; " /><meta name="citation_reference" content="citation_title=Genomics-assisted breeding for crop improvement; Trends Plant Sci.;  citation_year=2005;  citation_volume=10;  citation_pages=621-30; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and assay development in common bean; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=475" /><meta name="citation_reference" content="citation_title=Development of simple sequence repeat (SSR) markers for the assessment of gene flow and genetic diversity in pigeonpea (Cajanus cajan); Mol. Ecol. Notes;  citation_year=2001;  citation_volume=1;  citation_pages=283-5; " /><meta name="citation_reference" content="citation_title=Development, characterization and utilization of microsatellite markers in pigeonpea; Plant Breed.;  citation_year=2007;  citation_volume=126;  citation_pages=130-6; " /><meta name="citation_reference" content="citation_title=New microsatellite markers for pigeonpea (Cajanus cajan (L.) Millsp.); BMC Res. Notes;  citation_year=2009;  citation_volume=2;  citation_pages=35" /><meta name="citation_reference" content="citation_title=Novel SSR markers for polymorphism detection in pigeonpea (Cajanus spp.); Plant Breed.;  citation_year=2010;  citation_volume=129;  citation_pages=142-8; " /><meta name="citation_reference" content="citation_title=Analysis of BAC-end sequences (BESs) and development of BES-SSR markers for genetic mapping and hybrid purity assessment in pigeonpea; (Cajanus spp.), BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=56" /><meta name="citation_reference" content="citation_title=Gene expression profiling in susceptible interaction of grapevine with its fungal pathogen Eutypa lata: extending MapMan ontology for grapevine; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=104" /><meta name="citation_reference" content="citation_title=A Populus EST resource for plant functional genomics; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=13951-6; " /><meta name="citation_reference" content="citation_title=Extensive duplication and reshuffling in the Arabidopsis genome; Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=1093-102; " /><meta name="citation_reference" content="citation_title=Next generation sequencing technologies and their implications for crop genetics and breeding; Trends Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=522-30; " /><meta name="citation_reference" content="citation_title=A rapid and simple method for preparation of RNA from Saccharomyces cerevisiae; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=3091-2; " /><meta name="citation_reference" content="citation_title=Reverse transcriptase template switching: a SMART approach for full length cDNA library construction; Biotechniques;  citation_year=2001;  citation_volume=4;  citation_pages=892-7; " /><meta name="citation_reference" content="citation_title=Sequencing Medicago truncatula expressed sequenced tags using 454 life sciences technology; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=272" /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=Fast index based algorithms and software for matching position specific scoring matrices; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=389" /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=A codon-based model of nucleotide substitution for protein coding DNA sequences; Mol. Biol. Evol.;  citation_year=1994;  citation_volume=11;  citation_pages=725-36; " /><meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood; Comput. Appl. Biosci;  citation_year=1997;  citation_volume=15;  citation_pages=555-6; " /><meta name="citation_reference" content="citation_title=Exploiting EST databases for the development and characterization of gene derived SSR-markers in barley (Hordeum vulgare L.); Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=411-22; " /><meta name="citation_reference" content=" citation_url=http://fokker.wi.mit.edu/primer3/; citation_publisher=Humana Press, Totowa, NJ; Bioinformatics Methods and Protocols;  citation_year=2000;  citation_pages=365-368; " /><meta name="citation_reference" content="citation_title=Management of high-throughput DNA sequencing projects: Alpheus; J. Comput. Sci. Syst. Biol.;  citation_year=2008;  citation_volume=1;  citation_pages=132-48; " /><meta name="citation_reference" content="citation_title=High-throughput gene and SNP discovery in Eucalyptus grandis, an uncharacterized genome; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=312" /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=QTL mapping of ten agronomic traits on the soybean (Glycine max L. Merr.) genetic map and their association with EST markers; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1131-9; " /><meta name="citation_reference" content="citation_title=A soybean gene encoding a proline-rich protein is regulated by salicylic acid, an endogenous circadian rhythm and by various stresses; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=1125-31; " /><meta name="citation_reference" content="citation_title=The zinc-finger protein Zat12 plays a central role in reactive oxygen and abiotic stress signaling in Arabidopsis; Plant Physiol.;  citation_year=2005;  citation_volume=139;  citation_pages=847-56; " /><meta name="citation_reference" content="citation_title=The zinc finger protein Zat12 is required for cytosolic ascorbate peroxidase 1 expression during oxidative stress in Arabidopsis; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=11736-43; " /><meta name="citation_reference" content="citation_title=The phenylpropanoid pathway and plant defense—a genomics perspective; Mol. Plant Pathol.;  citation_year=2002;  citation_volume=3;  citation_pages=371-90; " /><meta name="citation_reference" content="citation_title=Sequential expression of two 1-aminocyclopropane-1-carboxylate synthase genes in response to biotic and abiotic stresses in potato (Solanum tuberosum L.) leaves; Plant Mol. Biol.;  citation_year=1997;  citation_volume=35;  citation_pages=683-88; " /><meta name="citation_reference" content="citation_title=Comparative gene expression analysis of susceptible and resistant near-isogenic lines in common wheat infected by; Puccinia triticina, DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=211-222; " /><meta name="citation_reference" content="citation_title=Generation and analysis of expressed sequence tags (ESTs) for marker development in yam; (Dioscorea alata L.), BMC Genomics.;  citation_year=2011;  citation_volume=12;  citation_pages=100" /><meta name="citation_reference" content="citation_title=The first set of EST resource for gene discovery and marker development in pigeonpea (Cajanus cajan L.); BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=45" /><meta name="citation_reference" content="citation_publisher=Springer, The Netherlands; Molecular Techniques in Crop Improvement;  citation_year=2010;  citation_volume=vol. 2;  citation_pages=119-42; " /><meta name="citation_reference" content="citation_title=Isolation of EST-derived microsatellite markers for genotyping the A and B genomes of wheat; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=399-407; " /><meta name="citation_reference" content="citation_title=Development of EST-SSR and genomic-SSR markers to assess genetic diversity in Jatropha curcas L.; BMC Res. Notes;  citation_year=2010;  citation_volume=3;  citation_pages=42" /><meta name="citation_reference" content="citation_title=The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics; PLoS Biol.;  citation_year=2003;  citation_volume=1;  citation_pages=e45" /><meta name="citation_reference" content="citation_title=EST derived single nucleotide polymorphism markers for assembling genetic and physical maps of the barley genome; Func. Integrat. Genom.;  citation_year=2008;  citation_volume=8;  citation_pages=223-33; " /><meta name="citation_reference" content="citation_title=Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (Cicer arietinum L.); citation_text=doi:10.1007/s00122-011-1556-1; Theor. Appl. Genet.;  citation_year=2011; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/3/153/1216137/dsr007.pdf" /><meta name="description" content="Abstract.  This study reports generation of large-scale genomic resources for pigeonpea, a so-called ‘orphan crop species’ of the semi-arid tropic regions. FLX/" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/3/153/444270" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/3/153/444270" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/3/153/444270");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr007';
    </script>
<title>
	Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea (Cajanus cajan L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./444270" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bed3PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEdWJleSwgQW51amEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgb2YgRXhjZWxsZW5jZSBpbiBHZW5vbWljcyAoQ0VHKSwgQnVpbGRpbmcgIzMwMCwgSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyIDMyNCwgR3JlYXRlciBIeWRlcmFiYWQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmFya2F0dWxsYWggVW5pdmVyc2l0eSwgQmhvcGFsIDQ2MiAwMjYsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmFybWVyLCBBbmRyZXciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBDZW50cmUgZm9yIEdlbm9tZSBSZXNvdXJjZXMgKE5DR1IpLCBTYW50YSBGZSwgTk0gODc1MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjaGx1ZXRlciwgSmVzc2ljYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgTm9ydGggQ2Fyb2xpbmEgYXQgQ2hhcmxvdHRlLCBDaGFybG90dGUsIE5DIDI4MjIzLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDYW5ub24sIFN0ZXZlbiBCLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXRlZCBTdGF0ZXMgRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyZS1BZ3JpY3VsdHVyYWwgUmVzZWFyY2ggU2VydmljZSAoVVNEQS1BUlMpLCBDb3JuIEluc2VjdHMgYW5kIENyb3AgR2VuZXRpY3MgUmVzZWFyY2ggVW5pdCwgQW1lcywgSUEgNTAwMTEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFiZXJuYXRoeSwgQnJpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQdXJkdWUgVW5pdmVyc2l0eSwgV2VzdCBMYWZheWV0dGUsIElOIDQ3OTA3LTIwNTQsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlR1dGVqYSwgUmVldHUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgb2YgRXhjZWxsZW5jZSBpbiBHZW5vbWljcyAoQ0VHKSwgQnVpbGRpbmcgIzMwMCwgSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyIDMyNCwgR3JlYXRlciBIeWRlcmFiYWQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV29vZHdhcmQsIEppbW15IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQ2VudHJlIGZvciBHZW5vbWUgUmVzb3VyY2VzIChOQ0dSKSwgU2FudGEgRmUsIE5NIDg3NTA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGFoLCBUcnVzaGFyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIG9mIEV4Y2VsbGVuY2UgaW4gR2Vub21pY3MgKENFRyksIEJ1aWxkaW5nICMzMDAsIEludGVybmF0aW9uYWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciB0aGUgU2VtaS1BcmlkIFRyb3BpY3MgKElDUklTQVQpLCBQYXRhbmNoZXJ1IDUwMiAzMjQsIEdyZWF0ZXIgSHlkZXJhYmFkLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik11bGFzbWFub3ZpYywgQmVuamFtaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml0ZWQgU3RhdGVzIERlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmUtQWdyaWN1bHR1cmFsIFJlc2VhcmNoIFNlcnZpY2UgKFVTREEtQVJTKSwgQ29ybiBJbnNlY3RzIGFuZCBDcm9wIEdlbmV0aWNzIFJlc2VhcmNoIFVuaXQsIEFtZXMsIElBIDUwMDExLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdWRhcGEsIEhpbWFiaW5kdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBvZiBFeGNlbGxlbmNlIGluIEdlbm9taWNzIChDRUcpLCBCdWlsZGluZyAjMzAwLCBJbnRlcm5hdGlvbmFsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgdGhlIFNlbWktQXJpZCBUcm9waWNzIChJQ1JJU0FUKSwgUGF0YW5jaGVydSA1MDIgMzI0LCBHcmVhdGVyIEh5ZGVyYWJhZCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYWp1LCBOaWtrdSBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBvZiBFeGNlbGxlbmNlIGluIEdlbm9taWNzIChDRUcpLCBCdWlsZGluZyAjMzAwLCBJbnRlcm5hdGlvbmFsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgdGhlIFNlbWktQXJpZCBUcm9waWNzIChJQ1JJU0FUKSwgUGF0YW5jaGVydSA1MDIgMzI0LCBHcmVhdGVyIEh5ZGVyYWJhZCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb3RoYWx3YWwsIFJhZ2luaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJhcmthdHVsbGFoIFVuaXZlcnNpdHksIEJob3BhbCA0NjIgMDI2LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhbmRlLCBTdXJlc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgb2YgRXhjZWxsZW5jZSBpbiBHZW5vbWljcyAoQ0VHKSwgQnVpbGRpbmcgIzMwMCwgSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyIDMyNCwgR3JlYXRlciBIeWRlcmFiYWQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWGlhbywgWW9uZ2xpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSi4gQ3JhaWcgVmVudGVyIEluc3RpdHV0ZSAoSkNWSSksIFJvY2t2aWxsZSwgTUQgMjA4NTAsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvd24sIENocmlzIEQuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSi4gQ3JhaWcgVmVudGVyIEluc3RpdHV0ZSAoSkNWSSksIFJvY2t2aWxsZSwgTUQgMjA4NTAsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNpbmdoLCBOYWdlbmRyYSBLLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIFJlc2VhcmNoIENlbnRyZSBvbiBQbGFudCBCaW90ZWNobm9sb2d5LCBOZXcgRGVsaGkgMTEwMDEyLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1heSwgR3JlZ29yeSBELiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIENlbnRyZSBmb3IgR2Vub21lIFJlc291cmNlcyAoTkNHUiksIFNhbnRhIEZlLCBOTSA4NzUwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmFja3NvbiwgU2NvdHQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQdXJkdWUgVW5pdmVyc2l0eSwgV2VzdCBMYWZheWV0dGUsIElOIDQ3OTA3LTIwNTQsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhcnNobmV5LCBSYWplZXYgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgb2YgRXhjZWxsZW5jZSBpbiBHZW5vbWljcyAoQ0VHKSwgQnVpbGRpbmcgIzMwMCwgSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyIDMyNCwgR3JlYXRlciBIeWRlcmFiYWQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ0dJQVIgR2VuZXJhdGlvbiBDaGFsbGVuZ2UgUHJvZ3JhbW1lIChHQ1ApLCBjL28gQ0lNTVlULCAwNjYwMCBNZXhpY28gREYsIE1leGljbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGVmaW5pbmcgdGhlIFRyYW5zY3JpcHRvbWUgQXNzZW1ibHkgYW5kIEl0cyBVc2UgZm9yIEdlbm9tZSBEeW5hbWljcyBhbmQgVHJhbnNjcmlwdG9tZSBQcm9maWxpbmcgU3R1ZGllcyBpbiBQaWdlb25wZWEgKENhamFudXMgY2FqYW4gTC4pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzaXplIHZhcmlhdGlvbiBpbiBDYWphbnVzIGNhamFuIChGYWJhY2VhZSk6IGEgcmVjb25zaWRlcmF0aW9uOyBQbGFudCBTeXN0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMTI7ICBjaXRhdGlvbl9wYWdlcz0xMzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzLWFzc2lzdGVkIGJyZWVkaW5nIGZvciBjcm9wIGltcHJvdmVtZW50OyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTYyMS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGFzc2F5IGRldmVsb3BtZW50IGluIGNvbW1vbiBiZWFuOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IChTU1IpIG1hcmtlcnMgZm9yIHRoZSBhc3Nlc3NtZW50IG9mIGdlbmUgZmxvdyBhbmQgZ2VuZXRpYyBkaXZlcnNpdHkgaW4gcGlnZW9ucGVhIChDYWphbnVzIGNhamFuKTsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MjgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50LCBjaGFyYWN0ZXJpemF0aW9uIGFuZCB1dGlsaXphdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBpZ2VvbnBlYTsgUGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEyNjsgIGNpdGF0aW9uX3BhZ2VzPTEzMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmb3IgcGlnZW9ucGVhIChDYWphbnVzIGNhamFuIChMLikgTWlsbHNwLik7IEJNQyBSZXMuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIFNTUiBtYXJrZXJzIGZvciBwb2x5bW9ycGhpc20gZGV0ZWN0aW9uIGluIHBpZ2VvbnBlYSAoQ2FqYW51cyBzcHAuKTsgUGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyOTsgIGNpdGF0aW9uX3BhZ2VzPTE0Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBCQUMtZW5kIHNlcXVlbmNlcyAoQkVTcykgYW5kIGRldmVsb3BtZW50IG9mIEJFUy1TU1IgbWFya2VycyBmb3IgZ2VuZXRpYyBtYXBwaW5nIGFuZCBoeWJyaWQgcHVyaXR5IGFzc2Vzc21lbnQgaW4gcGlnZW9ucGVhOyAoQ2FqYW51cyBzcHAuKSwgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gcHJvZmlsaW5nIGluIHN1c2NlcHRpYmxlIGludGVyYWN0aW9uIG9mIGdyYXBldmluZSB3aXRoIGl0cyBmdW5nYWwgcGF0aG9nZW4gRXV0eXBhIGxhdGE6IGV4dGVuZGluZyBNYXBNYW4gb250b2xvZ3kgZm9yIGdyYXBldmluZTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIFBvcHVsdXMgRVNUIHJlc291cmNlIGZvciBwbGFudCBmdW5jdGlvbmFsIGdlbm9taWNzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTEzOTUxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBkdXBsaWNhdGlvbiBhbmQgcmVzaHVmZmxpbmcgaW4gdGhlIEFyYWJpZG9wc2lzIGdlbm9tZTsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEwOTMtMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2dpZXMgYW5kIHRoZWlyIGltcGxpY2F0aW9ucyBmb3IgY3JvcCBnZW5ldGljcyBhbmQgYnJlZWRpbmc7IFRyZW5kcyBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTUyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByYXBpZCBhbmQgc2ltcGxlIG1ldGhvZCBmb3IgcHJlcGFyYXRpb24gb2YgUk5BIGZyb20gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0zMDkxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldmVyc2UgdHJhbnNjcmlwdGFzZSB0ZW1wbGF0ZSBzd2l0Y2hpbmc6IGEgU01BUlQgYXBwcm9hY2ggZm9yIGZ1bGwgbGVuZ3RoIGNETkEgbGlicmFyeSBjb25zdHJ1Y3Rpb247IEJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTg5Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIE1lZGljYWdvIHRydW5jYXR1bGEgZXhwcmVzc2VkIHNlcXVlbmNlZCB0YWdzIHVzaW5nIDQ1NCBsaWZlIHNjaWVuY2VzIHRlY2hub2xvZ3k7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogYSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBpbmRleCBiYXNlZCBhbGdvcml0aG1zIGFuZCBzb2Z0d2FyZSBmb3IgbWF0Y2hpbmcgcG9zaXRpb24gc3BlY2lmaWMgc2NvcmluZyBtYXRyaWNlczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zODkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMIFc6IGltcHJvdmluZyB0aGUgc2Vuc2l0aXZpdHkgb2YgcHJvZ3Jlc3NpdmUgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHRocm91Z2ggc2VxdWVuY2Ugd2VpZ2h0aW5nLCBwb3NpdGlvbi1zcGVjaWZpYyBnYXAgcGVuYWx0aWVzIGFuZCB3ZWlnaHQgbWF0cml4IGNob2ljZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NDY3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb2Rvbi1iYXNlZCBtb2RlbCBvZiBudWNsZW90aWRlIHN1YnN0aXR1dGlvbiBmb3IgcHJvdGVpbiBjb2RpbmcgRE5BIHNlcXVlbmNlczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTcyNS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFNTDogYSBwcm9ncmFtIHBhY2thZ2UgZm9yIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBieSBtYXhpbXVtIGxpa2VsaWhvb2Q7IENvbXB1dC4gQXBwbC4gQmlvc2NpOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTU1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvaXRpbmcgRVNUIGRhdGFiYXNlcyBmb3IgdGhlIGRldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGdlbmUgZGVyaXZlZCBTU1ItbWFya2VycyBpbiBiYXJsZXkgKEhvcmRldW0gdnVsZ2FyZSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9NDExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fdXJsPWh0dHA6Ly9mb2trZXIud2kubWl0LmVkdS9wcmltZXIzLzsgY2l0YXRpb25fcHVibGlzaGVyPUh1bWFuYSBQcmVzcywgVG90b3dhLCBOSjsgQmlvaW5mb3JtYXRpY3MgTWV0aG9kcyBhbmQgUHJvdG9jb2xzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fcGFnZXM9MzY1LTM2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFuYWdlbWVudCBvZiBoaWdoLXRocm91Z2hwdXQgRE5BIHNlcXVlbmNpbmcgcHJvamVjdHM6IEFscGhldXM7IEouIENvbXB1dC4gU2NpLiBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTMyLTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgZ2VuZSBhbmQgU05QIGRpc2NvdmVyeSBpbiBFdWNhbHlwdHVzIGdyYW5kaXMsIGFuIHVuY2hhcmFjdGVyaXplZCBnZW5vbWU7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBwYWxhZW9wb2x5cGxvaWQgc295YmVhbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2MzsgIGNpdGF0aW9uX3BhZ2VzPTE3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R01BUDogYSBnZW5vbWljIG1hcHBpbmcgYW5kIGFsaWdubWVudCBwcm9ncmFtIGZvciBtUk5BIGFuZCBFU1Qgc2VxdWVuY2VzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE4NTktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBtYXBwaW5nIG9mIHRlbiBhZ3Jvbm9taWMgdHJhaXRzIG9uIHRoZSBzb3liZWFuIChHbHljaW5lIG1heCBMLiBNZXJyLikgZ2VuZXRpYyBtYXAgYW5kIHRoZWlyIGFzc29jaWF0aW9uIHdpdGggRVNUIG1hcmtlcnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTEzMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNveWJlYW4gZ2VuZSBlbmNvZGluZyBhIHByb2xpbmUtcmljaCBwcm90ZWluIGlzIHJlZ3VsYXRlZCBieSBzYWxpY3lsaWMgYWNpZCwgYW4gZW5kb2dlbm91cyBjaXJjYWRpYW4gcmh5dGhtIGFuZCBieSB2YXJpb3VzIHN0cmVzc2VzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTExMjUtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB6aW5jLWZpbmdlciBwcm90ZWluIFphdDEyIHBsYXlzIGEgY2VudHJhbCByb2xlIGluIHJlYWN0aXZlIG94eWdlbiBhbmQgYWJpb3RpYyBzdHJlc3Mgc2lnbmFsaW5nIGluIEFyYWJpZG9wc2lzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzk7ICBjaXRhdGlvbl9wYWdlcz04NDctNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB6aW5jIGZpbmdlciBwcm90ZWluIFphdDEyIGlzIHJlcXVpcmVkIGZvciBjeXRvc29saWMgYXNjb3JiYXRlIHBlcm94aWRhc2UgMSBleHByZXNzaW9uIGR1cmluZyBveGlkYXRpdmUgc3RyZXNzIGluIEFyYWJpZG9wc2lzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzk7ICBjaXRhdGlvbl9wYWdlcz0xMTczNi00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBoZW55bHByb3Bhbm9pZCBwYXRod2F5IGFuZCBwbGFudCBkZWZlbnNl4oCUYSBnZW5vbWljcyBwZXJzcGVjdGl2ZTsgTW9sLiBQbGFudCBQYXRob2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zNzEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbnRpYWwgZXhwcmVzc2lvbiBvZiB0d28gMS1hbWlub2N5Y2xvcHJvcGFuZS0xLWNhcmJveHlsYXRlIHN5bnRoYXNlIGdlbmVzIGluIHJlc3BvbnNlIHRvIGJpb3RpYyBhbmQgYWJpb3RpYyBzdHJlc3NlcyBpbiBwb3RhdG8gKFNvbGFudW0gdHViZXJvc3VtIEwuKSBsZWF2ZXM7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz02ODMtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbmUgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiBzdXNjZXB0aWJsZSBhbmQgcmVzaXN0YW50IG5lYXItaXNvZ2VuaWMgbGluZXMgaW4gY29tbW9uIHdoZWF0IGluZmVjdGVkIGJ5OyBQdWNjaW5pYSB0cml0aWNpbmEsIEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjExLTIyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgKEVTVHMpIGZvciBtYXJrZXIgZGV2ZWxvcG1lbnQgaW4geWFtOyAoRGlvc2NvcmVhIGFsYXRhIEwuKSwgQk1DIEdlbm9taWNzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmaXJzdCBzZXQgb2YgRVNUIHJlc291cmNlIGZvciBnZW5lIGRpc2NvdmVyeSBhbmQgbWFya2VyIGRldmVsb3BtZW50IGluIHBpZ2VvbnBlYSAoQ2FqYW51cyBjYWphbiBMLik7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBUaGUgTmV0aGVybGFuZHM7IE1vbGVjdWxhciBUZWNobmlxdWVzIGluIENyb3AgSW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9dm9sLiAyOyAgY2l0YXRpb25fcGFnZXM9MTE5LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gb2YgRVNULWRlcml2ZWQgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmb3IgZ2Vub3R5cGluZyB0aGUgQSBhbmQgQiBnZW5vbWVzIG9mIHdoZWF0OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTM5OS00MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIEVTVC1TU1IgYW5kIGdlbm9taWMtU1NSIG1hcmtlcnMgdG8gYXNzZXNzIGdlbmV0aWMgZGl2ZXJzaXR5IGluIEphdHJvcGhhIGN1cmNhcyBMLjsgQk1DIFJlcy4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBDYWVub3JoYWJkaXRpcyBicmlnZ3NhZTogYSBwbGF0Zm9ybSBmb3IgY29tcGFyYXRpdmUgZ2Vub21pY3M7IFBMb1MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPWU0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVCBkZXJpdmVkIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzIGZvciBhc3NlbWJsaW5nIGdlbmV0aWMgYW5kIHBoeXNpY2FsIG1hcHMgb2YgdGhlIGJhcmxleSBnZW5vbWU7IEZ1bmMuIEludGVncmF0LiBHZW5vbS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTIyMy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIHVzZSBvZiBnZW5pYyBtb2xlY3VsYXIgbWFya2VycyAoR01NcykgZm9yIGNvbnN0cnVjdGlvbiBvZiBhIHRyYW5zY3JpcHQgbWFwIG9mIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDA3L3MwMDEyMi0wMTEtMTU1Ni0xOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvMy8xNTMvMTIxNjEzNy9kc3IwMDcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoaXMgc3R1ZHkgcmVwb3J0cyBnZW5lcmF0aW9uIG9mIGxhcmdlLXNjYWxlIGdlbm9taWMgcmVzb3VyY2VzIGZvciBwaWdlb25wZWEsIGEgc28tY2FsbGVkIOKAmG9ycGhhbiBjcm9wIHNwZWNpZXPigJkgb2YgdGhlIHNlbWktYXJpZCB0cm9waWMgcmVnaW9ucy4gRkxYLyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQ0MjcwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKD+l71xXWfZnirhtcn2+HnqbbFm" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f153%2f444270" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f153%2f444270" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17382" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&amp;Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/3">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6640772">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640772">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640774">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640780">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640780">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640800">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640800">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640841">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640841">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640843">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640843">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640845">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640845">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6640847">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6640847">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/3/137/444115">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/3/165/444512">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea (<em>Cajanus cajan</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anuja Dubey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Barkatullah University</div>, <div class="addr-line">Bhopal 462 026</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anuja+Dubey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dubey A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dubey A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Anuja Dubey</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew Farmer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Centre for Genome Resources (NCGR)</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+Farmer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Farmer A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Farmer A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Andrew Farmer</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jessica Schlueter</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">University of North Carolina at Charlotte</div>, <div class="addr-line">Charlotte, NC 28223</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jessica+Schlueter">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schlueter J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schlueter J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jessica Schlueter</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Steven B. Cannon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">United States Department of Agriculture-Agricultural Research Service (USDA-ARS)</div>, <div class="insititution">Corn Insects and Crop Genetics Research Unit</div>, <div class="addr-line">Ames</div>, <div class="addr-line">IA 50011</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Steven+B.+Cannon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cannon S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cannon S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Steven B. Cannon</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Brian Abernathy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Purdue University</div>, <div class="addr-line">West Lafayette, IN 47907-2054</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Brian+Abernathy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abernathy B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abernathy B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Brian Abernathy</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reetu Tuteja</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reetu+Tuteja">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tuteja R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tuteja R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Reetu Tuteja</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jimmy Woodward</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Centre for Genome Resources (NCGR)</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jimmy+Woodward">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Woodward J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Woodward J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jimmy Woodward</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Trushar Shah</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Trushar+Shah">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shah T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shah T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Trushar Shah</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Benjamin Mulasmanovic</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">United States Department of Agriculture-Agricultural Research Service (USDA-ARS)</div>, <div class="insititution">Corn Insects and Crop Genetics Research Unit</div>, <div class="addr-line">Ames</div>, <div class="addr-line">IA 50011</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Benjamin+Mulasmanovic">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mulasmanovic B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mulasmanovic B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Benjamin Mulasmanovic</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Himabindu Kudapa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Himabindu+Kudapa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kudapa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kudapa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Himabindu Kudapa</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nikku L. Raju</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nikku+L.+Raju">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Raju N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Raju N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nikku L. Raju</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ragini Gothalwal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Barkatullah University</div>, <div class="addr-line">Bhopal 462 026</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ragini+Gothalwal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gothalwal R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gothalwal R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ragini Gothalwal</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suresh Pande</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suresh+Pande">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pande S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pande S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Suresh Pande</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongli Xiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">J. Craig Venter Institute (JCVI)</div>, <div class="addr-line">Rockville, MD 20850</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongli+Xiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xiao Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xiao Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yongli Xiao</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chris D. Town</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">J. Craig Venter Institute (JCVI)</div>, <div class="addr-line">Rockville, MD 20850</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chris+D.+Town">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Town C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Town C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chris D. Town</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nagendra K. Singh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">National Research Centre on Plant Biotechnology</div>, <div class="addr-line">New Delhi 110012</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nagendra+K.+Singh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Singh N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Singh N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nagendra K. Singh</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gregory D. May</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Centre for Genome Resources (NCGR)</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gregory+D.+May">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=May G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;May G&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Gregory D. May</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Scott Jackson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Purdue University</div>, <div class="addr-line">West Lafayette, IN 47907-2054</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Scott+Jackson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jackson S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jackson S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Scott Jackson</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rajeev K. Varshney</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre of Excellence in Genomics (CEG), Building #300</div>, <div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502 324, Greater Hyderabad</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">9</div><div class="insititution">CGIAR Generation Challenge Programme (GCP), c/o CIMMYT</div>, <div class="addr-line">06600 Mexico DF</div>, <div class="country">Mexico</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 40-30713305. Fax. +91 40-3071-3074/3075. E-mail: <a href="mailto:r.k.varshney@cgiar.org" target="_blank">r.k.varshney@cgiar.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rajeev+K.+Varshney">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Varshney R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Varshney R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Rajeev K. Varshney</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Mikio Nishimura</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 153–164, <a href='https://doi.org/10.1093/dnares/dsr007'>https://doi.org/10.1093/dnares/dsr007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">12 May 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 February 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 April 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="444270" href="/dnaresearch/article-pdf/18/3/153/1216137/dsr007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Anuja Dubey, Andrew Farmer, Jessica Schlueter, Steven B. Cannon, Brian Abernathy, Reetu Tuteja, Jimmy Woodward, Trushar Shah, Benjamin Mulasmanovic, Himabindu Kudapa, Nikku L. Raju, Ragini Gothalwal, Suresh Pande, Yongli Xiao, Chris D. Town, Nagendra K. Singh, Gregory D. May, Scott Jackson, Rajeev K. Varshney;  Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea (<em>Cajanus cajan</em> L.), <em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 153–164, <a href='https://doi.org/10.1093/dnares/dsr007'>https://doi.org/10.1093/dnares/dsr007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/444270?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/444270?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/444270?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/444270?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/444270?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/444270?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/444270?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Defining%20the%20Transcriptome%20Assembly%20and%20Its%20Use%20for%20Genome%20Dynamics%20and%20Transcriptome%20Profiling%20Studies%20in%20Pigeonpea%20(%26lt%3Bem%26gt%3BCajanus%20cajan%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2011-05-12&amp;volumeNum=18&amp;issueNum=3&amp;author=Dubey%2C%20Anuja%3B%20Farmer%2C%20Andrew&amp;startPage=153&amp;endPage=164&amp;contentId=10.1093%2Fdnares%2Fdsr007&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f153%2f444270" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6640772 id="6640772" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>This study reports generation of large-scale genomic resources for pigeonpea, a so-called ‘orphan crop species’ of the semi-arid tropic regions. FLX/454 sequencing carried out on a normalized cDNA pool prepared from 31 tissues produced 494 353 short transcript reads (STRs). Cluster analysis of these STRs, together with 10 817 Sanger ESTs, resulted in a pigeonpea trancriptome assembly (CcTA) comprising of 127 754 tentative unique sequences (TUSs). Functional analysis of these TUSs highlights several active pathways and processes in the sampled tissues. Comparison of the CcTA with the soybean genome showed similarity to 10 857 and 16 367 soybean gene models (depending on alignment methods). Additionally, Illumina 1G sequencing was performed on <em>Fusarium</em> wilt (FW)- and sterility mosaic disease (SMD)-challenged root tissues of 10 resistant and susceptible genotypes. More than 160 million sequence tags were used to identify FW- and SMD-responsive genes. Sequence analysis of CcTA and the Illumina tags identified a large new set of markers for use in genetics and breeding, including 8137 simple sequence repeats, 12 141 single-nucleotide polymorphisms and 5845 intron-spanning regions. Genomic resources developed in this study should be useful for basic and applied research, not only for pigeonpea improvement but also for other related, agronomically important legumes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Cajanus cajan L.&quot;"><em>Cajanus cajan</em> L.</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next generation sequencing&quot;">next generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptome assembly&quot;">transcriptome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular markers and gene discovery&quot;">molecular markers and gene discovery</a></div>                            <h2 scrollto-destination=6640774 id="6640774" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Pigeonpea (<em>Cajanus cajan</em> L.) is one of the major pulse crops of the tropics and subtropics. The crop is grown on over 4.8 Mha across the world, with an annual production of 4.1 Mt (<a class="link link-uri" href="http://faostat.fao.org">http://faostat.fao.org</a>). It is a major food legume crop in South Asia and East Africa, with India as the largest producer (3.07 Mha) followed by Myanmar (0.54 Mha) and Kenya (0.20 Mha). It is the only cultivated food crop of the <em>Cajaninae</em> subtribe and has a diploid genome with 11 pairs of chromosomes (2<em>n</em> = 2<em>x</em> = 22) and an estimated genome size of 858 Mb.<sup><a reveal-id="DSR007C1" data-open="DSR007C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Pigeonpea is a rich source of protein and vitamin B, and as a leguminous plant, it contributes as much as 40 kg nitrogen per hectare to the soil.</p><p>Despite its economic and ecological importance, this crop has gained less attention in terms of improvement in production. As a result, production has reached a plateau. As demonstrated in other crops, modern genomics approaches can facilitate breeding, leading to enhanced crop productivity. Integration of genomic approaches in breeding programmes has been referred to as ‘genomics-assisted breeding’.<sup><a reveal-id="DSR007C2" data-open="DSR007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Availability of genomic resources like molecular markers, genetic maps, transcriptomic or genome sequence data, and metabolome analyses are the pre-requisites for undertaking genomics-assisted breeding. While these platforms are available in many cereal and major legume crops like soybean, cowpea and common bean,<sup><a reveal-id="DSR007C3" data-open="DSR007C3" class="link link-ref link-reveal xref-bibr">3</a></sup> pigeonpea has not had the development and application of this genomic revolution. In this crop, only a few hundred (172) simple sequence repeat (SSR) markers<sup><a reveal-id="DSR007C4  DSR007C7" data-open="DSR007C4  DSR007C7" class="link link-ref link-reveal xref-bibr">4–7</a></sup> and a recent genetic map<sup><a reveal-id="DSR007C8" data-open="DSR007C8" class="link link-ref link-reveal xref-bibr">8</a></sup> have become available.</p><p>Without a genome sequence, transcriptome sequencing is an effective approach for gene discovery and identifying transcripts involved in specific biological processes. Expressed sequence tag (EST) studies have provided insight into genomic architecture and helped to elucidate genes involved in biological processes [grapevine, Populus, Arabidopsis].<sup><a reveal-id="DSR007C9  DSR007C11" data-open="DSR007C9  DSR007C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> As of October 2010, ∼10 817 pigeonpea EST sequences were available at the NCBI GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/gquery?term=pigeonpea">www.ncbi.nlm.nih.gov/sites/gquery?term=pigeonpea</a>). This small set of ESTs highlights the need for a larger collection of sequence information before employing effective functional genomics studies in pigeonpea research.</p><p>In recent years, the advent of next generation sequencing (NGS) technologies has made it possible to inexpensively and quickly generate large-scale transcript sequence data.<sup><a reveal-id="DSR007C12" data-open="DSR007C12" class="link link-ref link-reveal xref-bibr">12</a></sup> With an objective of characterizing functionally important genes in pigeonpea, transcriptome analysis was undertaken by sampling a large number of reads from normalized cDNA libraries prepared from different tissues, developmental conditions and physiological stages. The initial part of the present study was aimed at discovering and characterizing genes from the pigeonpea variety Pusa Ageti by developing and analysing the transcriptome assembly of pigeonpea based on FLX/454 sequencing. Secondly, the Illumina 1G, RNA-seq approach was employed on mRNA of 10 parental genotypes of five mapping populations that segregate for <em>Fusarium</em> wilt (FW) and/or sterility mosaic disease (SMD). Alignment of Illumina sequence data of these parental lines with the transcriptome assembly provided a framework for quantitative measurement of gene expression as well as single-nucleotide polymorphism (SNP) detection in these parental lines.</p><p>This may be the first report in pigeonpea of large-scale transcriptome data generation and the corresponding comprehensive analysis to understand transcriptome and evolutionary genome dynamics. Based on digital expression analyses, we have also identified candidate FW- and SMD-responsive genes. Finally, this study provides a resource of both SSR and SNP markers identified from these sequences.</p>                            <h2 scrollto-destination=6640780 id="6640780" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6640781 id="6640781" class="section-title" data-legacy-id=s2a>2.1. Plant material and library construction</h3>
<p>Pusa Ageti (ICP 28), an early maturing pigeonpea variety, was selected for library construction and transcriptome analysis. Seeds were sown in pots (three seeds per pot), and maintained in a glass-house. In order to maximize the diversity of expressed genes in pigeonpea, tissues from different developmental stages were targeted for collection and construction of cDNA libraries. These tissue samples included embryo, cotyledon, root and shoot primordia, apical meristem, leaves, senescing leaves, flowers, stamen and roots, harvested from several individual glass-house grown pigeonpea plants at different time intervals. This was done to analyse gene expression associated with the developmental process. Tissues were washed briefly with 0.1% DEPC water and then were frozen in liquid nitrogen. Total RNA was extracted from all the harvested tissues using modified hot-acid phenol method.<sup><a reveal-id="DSR007C13" data-open="DSR007C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The integrity and purity of all the samples were assessed both on 1.2% formaldehyde agarose gel and UV Spectrophotometer at <em>A</em><sub>260</sub>:<em>A</em><sub>280</sub>. An equal amount of each appropriate RNA sample was pooled to form a composite collection of total RNA sample for each tissue. Ten cDNA libraries were constructed to characterize specific stages of gene expression.</p><p>In order to minimize differences among the abundance of different transcripts (i.e. genes expressed at different levels), amplified cDNA was normalized employing the Smart cloning methodology<sup><a reveal-id="DSR007C14" data-open="DSR007C14" class="link link-ref link-reveal xref-bibr">14</a></sup> using the services of Evrogen (<a class="link link-uri" href="http://www.evrogen.com">www.evrogen.com</a>) and Sfi IA/B primers/adapters that permit directional cloning. The detailed methodology of library construction and normalization has been described in Cheung <em>et al.</em><sup><a reveal-id="DSR007C15" data-open="DSR007C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p>                            <h3 scrollto-destination=6640784 id="6640784" class="section-title" data-legacy-id=s2b>2.2. Sequence data assembly and clustering</h3>
<p>Sequence analyses and assembly was conducted using publicly available software and custom Perl scripts. Quality trimming of the sequences involved trimming adapter sequences, removing short sequences (&lt;50 nucleotides) for the assembly process as this will lead to false joining of reads, and chimeras that were sequenced hence reducing the quality of unique sequences. The vector-trimmed high-quality sequences were selected for further clustering and alignment to form transcript assemblies (TAs) using the CAP3 program.<sup><a reveal-id="DSR007C16" data-open="DSR007C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The following parameters were used for all CAP3 assemblies: –p 95–o 50–g 3–y 50–t 1000. These parameters were chosen to satisfy three primary goals: (i) to maximize contig length, (ii) to minimize production of contigs with highly variable read coverage, as these tend to be spurious assemblies and (iii) increasing the value of the ‘–t’ parameter improves the quality of the assembly at the cost of using additional memory on the assembly server; the value of 1000 was chosen as it was higher than the default but remained within the memory constraints of the assembly server. The assembly included the publicly available 10 817 ESTs of pigeonpea along with the FLX/454 reads.</p>                            <h3 scrollto-destination=6640786 id="6640786" class="section-title" data-legacy-id=s2c>2.3. Identification of paralogoues</h3>
<p>Identification of paralogous genes was conducted using both the contig consensus sequences and the singletons following assembly. The longest open reading frame was identified using EMBOSS: getorf (<a class="link link-uri" href="http://emboss.open-bio.org/wiki/Appdocs">http://emboss.open-bio.org/wiki/Appdocs</a>) to identify all open reading frames and a custom Perl script to retain only the longest. Clustering of these sequences followed using a virtual suffix tree generation with six frame translation using Vmatch.<sup><a reveal-id="DSR007C17" data-open="DSR007C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Gene families of size 2–6 were clustered with the following parameters, i.e. subject per cent match of 85, query per cent match of 70, a minimum length of 20 amino acids and an exdrop of 30. Pair-wise alignments were obtained using ClustalW<sup><a reveal-id="DSR007C18" data-open="DSR007C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and synonymous distances (Ks values) calculated using the method of Goldman and Yang<sup><a reveal-id="DSR007C19" data-open="DSR007C19" class="link link-ref link-reveal xref-bibr">19</a></sup> as implemented in PAML.<sup><a reveal-id="DSR007C20" data-open="DSR007C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                            <h3 scrollto-destination=6640788 id="6640788" class="section-title" data-legacy-id=s2d>2.4. Alignment of CcTA to the soybean genome</h3>
<p>Alignments of CcTA with the soybean genome were made with GMAP, requiring 80% identity and 80% coverage, maximum intron length of 10 000 bp, and maximum of 10 introns per gene fragment. The highest scoring alignment satisfying the stringency criteria was taken as the best hit. Alignments within 1% identity and 1% coverage of each other were retained as multiple equally good matches.</p>                            <h3 scrollto-destination=6640790 id="6640790" class="section-title" data-legacy-id=s2e>2.5. Functional annotation and similarity search</h3>
<p>Functional annotations of 127 754 TUSs were made using BLASTX comparisons against the UniRef non-redundant protein database. Sequence similarity was considered at a bit-score &gt;50 and a significant <em>e</em>-value ≤ 1E−08. Each TUS was assigned a putative cellular function based on the significant database hit with the lowest <em>E</em>-value. Subsequently, TUSs that showed a significant BLASTX hit were used for functional annotation based on Gene Ontology (GO) categories from the UniProt database (UniProt-GO). TUSs were thus assigned to primary and sub-GO functional categories.</p>                            <h3 scrollto-destination=6640792 id="6640792" class="section-title" data-legacy-id=s2f>2.6. Gene expression analysis</h3>
<p>cDNA pools from disease-stressed tissues of 10 pigeonpea genotypes were sequenced using Illumina 1G sequencing. These 10 genotypes are responsive to SMD and FW and represent parental genotypes of five mapping populations and segregate for the given stresses. FW stress was induced in four genotypes: ICPL 87119 and ICPL 99050 (resistant), ICPL 87091 and ICPB 2049 (susceptible). SMD stress was induced in six genotypes: ICPL 20096, ICPL 7035 and BSMR 736 (resistant), ICPL 332, TTB 7 and TAT 10 (susceptible). Stress was imposed on 15th day after sowing, using two methods: (i) root dipping method for FW infection and (ii) leaf stapling method for sterility mosaic virus infection. The tissues were harvested after 10 days of infection. Total RNA was extracted from all the harvested tissues using modified hot-acid phenol method.<sup><a reveal-id="DSR007C13" data-open="DSR007C13" class="link link-ref link-reveal xref-bibr">13</a></sup> cDNA libraries were constructed and subjected to Illumina 1G sequencing. Illumina tags were aligned to the CcTA and counts were assigned to each TUS for all 10 genotypes. These expression values were used for estimation of expression patterns of these TUSs in the parental combination of each cross. Differentially expressed genes/TUSs between pairs of SMD- and FW<em>-</em>responsive genotypes were identified. The expression values of TUSs were transformed to log 2 scale. Expression values of each gene were used to compare respective libraries of susceptible and resistant genotypes in each cross. Expression differences between TUSs from susceptible and resistant lines were considered with a minimum of a two-fold difference in log 2 expression. The set of TUSs with high differential expression ≥5 were considered for functional annotation using BLASTX analysis as described above.</p>                            <h3 scrollto-destination=6640794 id="6640794" class="section-title" data-legacy-id=s2g>2.7. Identification of microsatellite/SSRs</h3>
<p>SSR mining of 127 754 TUSs was carried out using the MIcroSAtellite (<em>MISA</em>)<sup><a reveal-id="DSR007C21" data-open="DSR007C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (<a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>) program, with the following parameters: at least 10 repeats for mono-, 6 repeats for di- and 5 repeats for tri-, tetra-, penta- and hexa-nucleotide for simple SSRs. Both perfect (i.e. SSRs containing a single repeat motif such as ‘AGG’) and compound (i.e. composed of two or more SSRs separated by ≤100 bp) SSRs were identified. The Primer3 program<sup><a reveal-id="DSR007C22" data-open="DSR007C22" class="link link-ref link-reveal xref-bibr">22</a></sup> was used for designing the primer pairs for identified SSRs based on the following criteria: (i) annealing temperature (Tm) between 50–65°C with 60°C as optimum; (ii) product size ranging from 100 to 350 bp; (iii) primer length ranging from 18 to 24 bp with an optimum of 20 bp; and (iv) GC% content in the range of 40–60%.</p>                            <h3 scrollto-destination=6640796 id="6640796" class="section-title" data-legacy-id=s2h>2.8. Identification of SNPs</h3>
<p>Identification of SNPs from Illumina data was carried out using the Alpheus software system.<sup><a reveal-id="DSR007C23" data-open="DSR007C23" class="link link-ref link-reveal xref-bibr">23</a></sup> SNPs were identified on the basis of alignment of Illumina reads generated from each of the genotypes against a reference—in this case, the CcTA and respective counter genotype, allowing not more than two mismatches. Based on alignment results, variants at a particular nucleotide position were identified. Significant variants were selected based on two criteria, allele frequency between two genotypes &gt;0.8, and number of tags aligned to the reference &gt;5.</p>                            <h3 scrollto-destination=6640798 id="6640798" class="section-title" data-legacy-id=s2i>2.9. Identification of intron-spanning region (ISR) markers</h3>
<p>Using alignments of paralogous genes (Section 2.4), primers were selected to span introns in predicted genes. The Primer3 program was used to design primers with default parameters, except for the requirement of spanning one predicted intron.</p>                            <h2 scrollto-destination=6640800 id="6640800" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
<p>This is the most comprehensive study of pigeonpea transcriptomic data to date. Pusa Ageti (ICP 28), a leading pigeonpea variety in India, was chosen for developing these genomic/transcriptomic resources, based on its phenology and the utility of this genotype in breeding programmes. The sequence data generated have been analysed to understand the transcriptome architecture and genome organization with respect to potential duplication, identification of candidate genes for FW and SMD based on digital gene expression profiling and the development of genetic markers.</p>                            <h3 scrollto-destination=6640802 id="6640802" class="section-title" data-legacy-id=s3a>3.1. Clustering and assembly of transcript reads</h3>
<p>Until recently, only 10 817 ESTs were available, of which &gt;90% were developed during last 2 years. With the objective of generating a comprehensive transcriptomic resource, deep sequencing was undertaken on cDNAs pools of 31 different developmental stages from early vegetative growth through the reproductive organs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Following cDNA synthesis, these libraries were pooled and normalized. FLX/454 sequencing of this normalized cDNA pool generated 494 353 short transcript reads (STRs), with an average length of 171 bp. At the time of data analysis, 10 817 Sanger ESTs, with average read length 527 bp, were available in the public domain via NCBI. These two data sets were analysed separately and in combination. Clustering of 354 131 STRs alone yielded 52 827 contigs, with an average length of 262 bp including 4308 high confidence singletons (that contain only two reads with zero read coverage) and 140 222 singletons. Clustering of Sanger ESTs yielded 746 contigs, with an average length 637 bp, and 5553 singletons. In order to develop a transcriptome reference in pigeonpea, 505 170 FLX/454 STRs and Sanger ESTs were assembled in combination to yield a pigeonpea transcriptome assembly (CcTA) comprising of a total 127 754 tentative unique sequences (TUSs). The sequence data from this study have been submitted to the Legume Information System (LIS) (<a class="link link-uri" href="http://comparative-legumes.org">http://comparative-legumes.org</a>). The sequence data can be accessed at <a class="link link-uri" href="http://cajca.comparative-legumes.org/data/2011/1e0b2bbdb4a3dca874759a9c7d23d46b/transcript_contigs.fa.gz">http://cajca.comparative-legumes.org/data/2011/1e0b2bbdb4a3dca874759a9c7d23d46b/transcript_contigs.fa.gz</a>.</p><p>The CcTA includes 48 726 (38.1%) contigs (average length 273 bp, maximum length 2067 bp) and 79 028 (61.9%) singletons (average length 198 bp, maximum length 1720 bp). A total of 3021 contigs (6.1%) were longer than 500 bp. Details about length distribution and read depth of contigs are given in Table <a reveal-id="DSR007TB1" data-open="DSR007TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSR007F1" data-open="DSR007F1" class="link link-reveal link-table xref-fig">1</a>, respectively. The overall redundancy of the library was calculated at 25.2%, suggesting that the normalization process was effective and that the libraries generated have the potential to uncover many more unique transcripts. These results support that FLX/454-based gene discovery represents a viable and perhaps favourable alternative to Sanger-based sequencing of EST libraries, when a diverse sampling of genes is more important than obtaining full transcript-length contigs.<sup><a reveal-id="DSR007C15" data-open="DSR007C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSR007C24" data-open="DSR007C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                        <a id="6640805" scrollto-destination="6640805"></a>
<div content-id="DSR007TB1" class="table-modal"><div class="table-wrap"><div id="DSR007TB1" class="table-wrap-title" data-id="DSR007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sequence length distribution before and after assembly of FLX/454 STRs and Sanger ESTs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Range of nucleotide length (bp) </th><th>Raw 454 STRs </th><th>Raw Sanger ESTs </th><th>Assembled 454 STRs </th><th>Assembled Sanger ESTs </th><th>Assembled 454 STRs + Sanger ESTs </th></tr></thead><tbody><tr><td>50 </td><td>31 876 (6.4%) </td><td>44 (0.4%) </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>51–100 </td><td>61 172 (12.3%) </td><td>180 (1.6%) </td><td>2282 (4.7%) </td><td>5 (0.6%) </td><td>2253 (4.6%) </td></tr><tr><td>101–150 </td><td>84 878 (17.1%) </td><td>420 (3.8%) </td><td>4854 (10.0%) </td><td>4 (0.5%) </td><td>4829 (9.9%) </td></tr><tr><td>151–200 </td><td>88 806 (17.9%) </td><td>449 (4.1%) </td><td>5934 (12.2%) </td><td>17 (2.2%) </td><td>5874 (12.0%) </td></tr><tr><td>201–250 </td><td>185 863 (37.5%) </td><td>658 (6.0%) </td><td>12 780 (26.3%) </td><td>24 (3.2%) </td><td>12 561 (25.7%) </td></tr><tr><td>251–300 </td><td>41 758 (8.4%) </td><td>630 (5.8%) </td><td>9224 (19.0%) </td><td>20 (2.6%) </td><td>9015 (18.5%) </td></tr><tr><td>301–350 </td><td> </td><td>401 (3.7%) </td><td>4960(10.2%) </td><td>21 (2.8%) </td><td>4821 (9.8%) </td></tr><tr><td>351–400 </td><td> </td><td>603 (5.5%) </td><td>3415 (7.0%) </td><td>42 (5.6%) </td><td>3349 (6.8%) </td></tr><tr><td>401–450 </td><td> </td><td>666 (6.1%) </td><td>1901 (3.9%) </td><td>37 (4.9%) </td><td>1879 (3.8%) </td></tr><tr><td>451–500 </td><td> </td><td>573 (5.2%) </td><td>3169 (6.5%) </td><td>57 (7.6%) </td><td>1124 (2.3%) </td></tr><tr><td>501–550 </td><td> </td><td>740 (6.8%) </td><td> </td><td>58 (7.7%) </td><td>3021 (6.1%) </td></tr><tr><td>551–600 </td><td> </td><td>575 (5.3%) </td><td> </td><td>65 (8.7%) </td><td> </td></tr><tr><td>601–650 </td><td> </td><td>621 (5.7%) </td><td> </td><td>51 (6.8%) </td><td> </td></tr><tr><td>651–700 </td><td> </td><td>887 (8.2%) </td><td> </td><td>45 (6.0%) </td><td> </td></tr><tr><td>701–750 </td><td> </td><td>1590 (14.6%) </td><td> </td><td>79 (10.5%) </td><td> </td></tr><tr><td>751–800 </td><td> </td><td>682 (6.3%) </td><td> </td><td>42 (5.6%) </td><td> </td></tr><tr><td>801–850 </td><td> </td><td>1098 (10.1%) </td><td> </td><td>74 (9.9%) </td><td> </td></tr><tr><td>851–900 </td><td> </td><td> </td><td> </td><td>24 (3.2%) </td><td> </td></tr><tr><td>901–950 </td><td> </td><td> </td><td> </td><td>18 (2.4%) </td><td> </td></tr><tr><td>951–1000 </td><td> </td><td> </td><td> </td><td>12 (1.6%) </td><td> </td></tr><tr><td>1001–1050 </td><td> </td><td> </td><td> </td><td>11 (1.4%) </td><td> </td></tr><tr><td>1051–1100 </td><td> </td><td> </td><td> </td><td>40 (5.3%) </td><td> </td></tr><tr><td>Total reads </td><td>494 353 </td><td>10 817 </td><td>48 519 </td><td>746 </td><td>48 726 </td></tr></tbody></table></div><div content-id="DSR007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Range of nucleotide length (bp) </th><th>Raw 454 STRs </th><th>Raw Sanger ESTs </th><th>Assembled 454 STRs </th><th>Assembled Sanger ESTs </th><th>Assembled 454 STRs + Sanger ESTs </th></tr></thead><tbody><tr><td>50 </td><td>31 876 (6.4%) </td><td>44 (0.4%) </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>51–100 </td><td>61 172 (12.3%) </td><td>180 (1.6%) </td><td>2282 (4.7%) </td><td>5 (0.6%) </td><td>2253 (4.6%) </td></tr><tr><td>101–150 </td><td>84 878 (17.1%) </td><td>420 (3.8%) </td><td>4854 (10.0%) </td><td>4 (0.5%) </td><td>4829 (9.9%) </td></tr><tr><td>151–200 </td><td>88 806 (17.9%) </td><td>449 (4.1%) </td><td>5934 (12.2%) </td><td>17 (2.2%) </td><td>5874 (12.0%) </td></tr><tr><td>201–250 </td><td>185 863 (37.5%) </td><td>658 (6.0%) </td><td>12 780 (26.3%) </td><td>24 (3.2%) </td><td>12 561 (25.7%) </td></tr><tr><td>251–300 </td><td>41 758 (8.4%) </td><td>630 (5.8%) </td><td>9224 (19.0%) </td><td>20 (2.6%) </td><td>9015 (18.5%) </td></tr><tr><td>301–350 </td><td> </td><td>401 (3.7%) </td><td>4960(10.2%) </td><td>21 (2.8%) </td><td>4821 (9.8%) </td></tr><tr><td>351–400 </td><td> </td><td>603 (5.5%) </td><td>3415 (7.0%) </td><td>42 (5.6%) </td><td>3349 (6.8%) </td></tr><tr><td>401–450 </td><td> </td><td>666 (6.1%) </td><td>1901 (3.9%) </td><td>37 (4.9%) </td><td>1879 (3.8%) </td></tr><tr><td>451–500 </td><td> </td><td>573 (5.2%) </td><td>3169 (6.5%) </td><td>57 (7.6%) </td><td>1124 (2.3%) </td></tr><tr><td>501–550 </td><td> </td><td>740 (6.8%) </td><td> </td><td>58 (7.7%) </td><td>3021 (6.1%) </td></tr><tr><td>551–600 </td><td> </td><td>575 (5.3%) </td><td> </td><td>65 (8.7%) </td><td> </td></tr><tr><td>601–650 </td><td> </td><td>621 (5.7%) </td><td> </td><td>51 (6.8%) </td><td> </td></tr><tr><td>651–700 </td><td> </td><td>887 (8.2%) </td><td> </td><td>45 (6.0%) </td><td> </td></tr><tr><td>701–750 </td><td> </td><td>1590 (14.6%) </td><td> </td><td>79 (10.5%) </td><td> </td></tr><tr><td>751–800 </td><td> </td><td>682 (6.3%) </td><td> </td><td>42 (5.6%) </td><td> </td></tr><tr><td>801–850 </td><td> </td><td>1098 (10.1%) </td><td> </td><td>74 (9.9%) </td><td> </td></tr><tr><td>851–900 </td><td> </td><td> </td><td> </td><td>24 (3.2%) </td><td> </td></tr><tr><td>901–950 </td><td> </td><td> </td><td> </td><td>18 (2.4%) </td><td> </td></tr><tr><td>951–1000 </td><td> </td><td> </td><td> </td><td>12 (1.6%) </td><td> </td></tr><tr><td>1001–1050 </td><td> </td><td> </td><td> </td><td>11 (1.4%) </td><td> </td></tr><tr><td>1051–1100 </td><td> </td><td> </td><td> </td><td>40 (5.3%) </td><td> </td></tr><tr><td>Total reads </td><td>494 353 </td><td>10 817 </td><td>48 519 </td><td>746 </td><td>48 726 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6640805" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR007TB1" class="table-wrap-title" data-id="DSR007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sequence length distribution before and after assembly of FLX/454 STRs and Sanger ESTs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Range of nucleotide length (bp) </th><th>Raw 454 STRs </th><th>Raw Sanger ESTs </th><th>Assembled 454 STRs </th><th>Assembled Sanger ESTs </th><th>Assembled 454 STRs + Sanger ESTs </th></tr></thead><tbody><tr><td>50 </td><td>31 876 (6.4%) </td><td>44 (0.4%) </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>51–100 </td><td>61 172 (12.3%) </td><td>180 (1.6%) </td><td>2282 (4.7%) </td><td>5 (0.6%) </td><td>2253 (4.6%) </td></tr><tr><td>101–150 </td><td>84 878 (17.1%) </td><td>420 (3.8%) </td><td>4854 (10.0%) </td><td>4 (0.5%) </td><td>4829 (9.9%) </td></tr><tr><td>151–200 </td><td>88 806 (17.9%) </td><td>449 (4.1%) </td><td>5934 (12.2%) </td><td>17 (2.2%) </td><td>5874 (12.0%) </td></tr><tr><td>201–250 </td><td>185 863 (37.5%) </td><td>658 (6.0%) </td><td>12 780 (26.3%) </td><td>24 (3.2%) </td><td>12 561 (25.7%) </td></tr><tr><td>251–300 </td><td>41 758 (8.4%) </td><td>630 (5.8%) </td><td>9224 (19.0%) </td><td>20 (2.6%) </td><td>9015 (18.5%) </td></tr><tr><td>301–350 </td><td> </td><td>401 (3.7%) </td><td>4960(10.2%) </td><td>21 (2.8%) </td><td>4821 (9.8%) </td></tr><tr><td>351–400 </td><td> </td><td>603 (5.5%) </td><td>3415 (7.0%) </td><td>42 (5.6%) </td><td>3349 (6.8%) </td></tr><tr><td>401–450 </td><td> </td><td>666 (6.1%) </td><td>1901 (3.9%) </td><td>37 (4.9%) </td><td>1879 (3.8%) </td></tr><tr><td>451–500 </td><td> </td><td>573 (5.2%) </td><td>3169 (6.5%) </td><td>57 (7.6%) </td><td>1124 (2.3%) </td></tr><tr><td>501–550 </td><td> </td><td>740 (6.8%) </td><td> </td><td>58 (7.7%) </td><td>3021 (6.1%) </td></tr><tr><td>551–600 </td><td> </td><td>575 (5.3%) </td><td> </td><td>65 (8.7%) </td><td> </td></tr><tr><td>601–650 </td><td> </td><td>621 (5.7%) </td><td> </td><td>51 (6.8%) </td><td> </td></tr><tr><td>651–700 </td><td> </td><td>887 (8.2%) </td><td> </td><td>45 (6.0%) </td><td> </td></tr><tr><td>701–750 </td><td> </td><td>1590 (14.6%) </td><td> </td><td>79 (10.5%) </td><td> </td></tr><tr><td>751–800 </td><td> </td><td>682 (6.3%) </td><td> </td><td>42 (5.6%) </td><td> </td></tr><tr><td>801–850 </td><td> </td><td>1098 (10.1%) </td><td> </td><td>74 (9.9%) </td><td> </td></tr><tr><td>851–900 </td><td> </td><td> </td><td> </td><td>24 (3.2%) </td><td> </td></tr><tr><td>901–950 </td><td> </td><td> </td><td> </td><td>18 (2.4%) </td><td> </td></tr><tr><td>951–1000 </td><td> </td><td> </td><td> </td><td>12 (1.6%) </td><td> </td></tr><tr><td>1001–1050 </td><td> </td><td> </td><td> </td><td>11 (1.4%) </td><td> </td></tr><tr><td>1051–1100 </td><td> </td><td> </td><td> </td><td>40 (5.3%) </td><td> </td></tr><tr><td>Total reads </td><td>494 353 </td><td>10 817 </td><td>48 519 </td><td>746 </td><td>48 726 </td></tr></tbody></table></div><div content-id="DSR007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Range of nucleotide length (bp) </th><th>Raw 454 STRs </th><th>Raw Sanger ESTs </th><th>Assembled 454 STRs </th><th>Assembled Sanger ESTs </th><th>Assembled 454 STRs + Sanger ESTs </th></tr></thead><tbody><tr><td>50 </td><td>31 876 (6.4%) </td><td>44 (0.4%) </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>51–100 </td><td>61 172 (12.3%) </td><td>180 (1.6%) </td><td>2282 (4.7%) </td><td>5 (0.6%) </td><td>2253 (4.6%) </td></tr><tr><td>101–150 </td><td>84 878 (17.1%) </td><td>420 (3.8%) </td><td>4854 (10.0%) </td><td>4 (0.5%) </td><td>4829 (9.9%) </td></tr><tr><td>151–200 </td><td>88 806 (17.9%) </td><td>449 (4.1%) </td><td>5934 (12.2%) </td><td>17 (2.2%) </td><td>5874 (12.0%) </td></tr><tr><td>201–250 </td><td>185 863 (37.5%) </td><td>658 (6.0%) </td><td>12 780 (26.3%) </td><td>24 (3.2%) </td><td>12 561 (25.7%) </td></tr><tr><td>251–300 </td><td>41 758 (8.4%) </td><td>630 (5.8%) </td><td>9224 (19.0%) </td><td>20 (2.6%) </td><td>9015 (18.5%) </td></tr><tr><td>301–350 </td><td> </td><td>401 (3.7%) </td><td>4960(10.2%) </td><td>21 (2.8%) </td><td>4821 (9.8%) </td></tr><tr><td>351–400 </td><td> </td><td>603 (5.5%) </td><td>3415 (7.0%) </td><td>42 (5.6%) </td><td>3349 (6.8%) </td></tr><tr><td>401–450 </td><td> </td><td>666 (6.1%) </td><td>1901 (3.9%) </td><td>37 (4.9%) </td><td>1879 (3.8%) </td></tr><tr><td>451–500 </td><td> </td><td>573 (5.2%) </td><td>3169 (6.5%) </td><td>57 (7.6%) </td><td>1124 (2.3%) </td></tr><tr><td>501–550 </td><td> </td><td>740 (6.8%) </td><td> </td><td>58 (7.7%) </td><td>3021 (6.1%) </td></tr><tr><td>551–600 </td><td> </td><td>575 (5.3%) </td><td> </td><td>65 (8.7%) </td><td> </td></tr><tr><td>601–650 </td><td> </td><td>621 (5.7%) </td><td> </td><td>51 (6.8%) </td><td> </td></tr><tr><td>651–700 </td><td> </td><td>887 (8.2%) </td><td> </td><td>45 (6.0%) </td><td> </td></tr><tr><td>701–750 </td><td> </td><td>1590 (14.6%) </td><td> </td><td>79 (10.5%) </td><td> </td></tr><tr><td>751–800 </td><td> </td><td>682 (6.3%) </td><td> </td><td>42 (5.6%) </td><td> </td></tr><tr><td>801–850 </td><td> </td><td>1098 (10.1%) </td><td> </td><td>74 (9.9%) </td><td> </td></tr><tr><td>851–900 </td><td> </td><td> </td><td> </td><td>24 (3.2%) </td><td> </td></tr><tr><td>901–950 </td><td> </td><td> </td><td> </td><td>18 (2.4%) </td><td> </td></tr><tr><td>951–1000 </td><td> </td><td> </td><td> </td><td>12 (1.6%) </td><td> </td></tr><tr><td>1001–1050 </td><td> </td><td> </td><td> </td><td>11 (1.4%) </td><td> </td></tr><tr><td>1051–1100 </td><td> </td><td> </td><td> </td><td>40 (5.3%) </td><td> </td></tr><tr><td>Total reads </td><td>494 353 </td><td>10 817 </td><td>48 519 </td><td>746 </td><td>48 726 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6640805" target="_blank">View Large</a></div></div>                        <a id="6640806" scrollto-destination="6640806"></a>
<div id="DSR007F1" data-id="DSR007F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00701.jpeg?Expires=2147483647&Signature=SSCEWpZMUlwS10EFGaNi5ZUbTiADSPWnh6KDA2uNlgRwwauVac675Q0UnSxdocq1VKp7Nyo4O-lyZWMVJF1Ro9UKvs0CjWCWTT4PICPqPbPjgYuMul1pr3ia307j~GoEbrHSaHq3~T8FYxV2wZnbIASHNauovKNd6-bn-aYq2Gf5d3hIHBg4RFnzad6m7r-DHecTsJS8~5DCi-xnIWE6an4VriTvhXhrt5d2LLWcZHu7opBkI0h6AxKbwf2nQQUJGk0O5lAto7gMMayyHshnU-EuyH81FVTD~l5sx4BgaJBWaC0Y8EawbEqsGY4nB7sB1D5x0U1BDEnocY8HnWC6Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution and read depth of sequence tags in the contigs. Number of 454 STRs aligning to form a contig ranged from 2 to &gt;1000. A total of 36 152 contigs showed a read depth ranging from 2 to 5 tags, followed by 6755 contigs with read depth 6–10 tags, 5517 contigs with read depth 11–100 tags and 290 contigs with read depth ranging from 100 to 1000 tags. A maximum of 12 contigs showed a read depth of &gt;1000 tags." path-from-xml="dsr00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640806/dsr00701.jpeg" path-from-xml="dsr00701.jpeg" target="_blank">View large</a><a section="6640806" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00701.jpeg?Expires=2147483647&Signature=QRSaqvbmtsA2lCHlrbI-v5VQz5EgslyQ6B23TcpEVs~6R6gcPwgJ8CKn0sJdkKzrfyew-Z394-A2Jvt7tRYe1XqbpHdkxCSS~gYTnGcergWNqsspa7gCmVGZK1ItWrSgQ0xYCrK6EqDKIPNMQEFKuA41cBNl~uzwqO5Fst11Rw5LJjzDVgM9xrujxaq2WaPYnsgq~O3pA~fggxQOBX1VKn5GA82nrpxRQqoTgO0KypoS8JloRIWnM2PZx-BbwZV6OZufbPLaH7cY-IThe1ep8~iBO9RlzAtLVrFeu~0PnhoHqeOYe2y1SinFwKUUzEg1sqka-Dcp9yuLzCw3MxACOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640806&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution and read depth of sequence tags in the contigs. Number of 454 STRs aligning to form a contig ranged from 2 to &gt;1000. A total of 36 152 contigs showed a read depth ranging from 2 to 5 tags, followed by 6755 contigs with read depth 6–10 tags, 5517 contigs with read depth 11–100 tags and 290 contigs with read depth ranging from 100 to 1000 tags. A maximum of 12 contigs showed a read depth of &gt;1000 tags.</p></div></div><div content-id="DSR007F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00701.jpeg?Expires=2147483647&Signature=SSCEWpZMUlwS10EFGaNi5ZUbTiADSPWnh6KDA2uNlgRwwauVac675Q0UnSxdocq1VKp7Nyo4O-lyZWMVJF1Ro9UKvs0CjWCWTT4PICPqPbPjgYuMul1pr3ia307j~GoEbrHSaHq3~T8FYxV2wZnbIASHNauovKNd6-bn-aYq2Gf5d3hIHBg4RFnzad6m7r-DHecTsJS8~5DCi-xnIWE6an4VriTvhXhrt5d2LLWcZHu7opBkI0h6AxKbwf2nQQUJGk0O5lAto7gMMayyHshnU-EuyH81FVTD~l5sx4BgaJBWaC0Y8EawbEqsGY4nB7sB1D5x0U1BDEnocY8HnWC6Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution and read depth of sequence tags in the contigs. Number of 454 STRs aligning to form a contig ranged from 2 to &gt;1000. A total of 36 152 contigs showed a read depth ranging from 2 to 5 tags, followed by 6755 contigs with read depth 6–10 tags, 5517 contigs with read depth 11–100 tags and 290 contigs with read depth ranging from 100 to 1000 tags. A maximum of 12 contigs showed a read depth of &gt;1000 tags." path-from-xml="dsr00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640806/dsr00701.jpeg" path-from-xml="dsr00701.jpeg" target="_blank">View large</a><a section="6640806" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00701.jpeg?Expires=2147483647&Signature=QRSaqvbmtsA2lCHlrbI-v5VQz5EgslyQ6B23TcpEVs~6R6gcPwgJ8CKn0sJdkKzrfyew-Z394-A2Jvt7tRYe1XqbpHdkxCSS~gYTnGcergWNqsspa7gCmVGZK1ItWrSgQ0xYCrK6EqDKIPNMQEFKuA41cBNl~uzwqO5Fst11Rw5LJjzDVgM9xrujxaq2WaPYnsgq~O3pA~fggxQOBX1VKn5GA82nrpxRQqoTgO0KypoS8JloRIWnM2PZx-BbwZV6OZufbPLaH7cY-IThe1ep8~iBO9RlzAtLVrFeu~0PnhoHqeOYe2y1SinFwKUUzEg1sqka-Dcp9yuLzCw3MxACOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640806&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution and read depth of sequence tags in the contigs. Number of 454 STRs aligning to form a contig ranged from 2 to &gt;1000. A total of 36 152 contigs showed a read depth ranging from 2 to 5 tags, followed by 6755 contigs with read depth 6–10 tags, 5517 contigs with read depth 11–100 tags and 290 contigs with read depth ranging from 100 to 1000 tags. A maximum of 12 contigs showed a read depth of &gt;1000 tags.</p></div></div>                            <h3 scrollto-destination=6640807 id="6640807" class="section-title" data-legacy-id=s3b>3.2. Evaluation of paralogous genes</h3>
<p>To evaluate characteristics of the CcTA sequences and to identify potential signatures of genome duplication in pigeonpea, the transcriptome assembly was analysed by comparing all TUSs against one another. Of the 127 754 TUSs sequences, 9.8% (12 515) could be clustered into families of similar genes (requiring subject per cent match of 85, query per cent match of 70, and a minimum alignment length of 60 nt). Of these, 3098 occurred in clusters of size 2, 537 of size 3, 181 of size 4, 89 of size 5 and 68 of size 6. The modal per cent identity among paralogues was 98.5, with the proportion of paralogues dropping to less than half the modal value at ∼96.5 per cent identity. Stated differently, the paralogue pairs are dominated by nearly identical or highly similar sequences. These alignments can also be used to find pair-wise synonymous distance measures, and tallied for Ks ranges for comparison with published Ks values for soybean (Fig. <a reveal-id="DSR007F2" data-open="DSR007F2" class="link link-reveal link-table xref-fig">2</a>). There is a sharp Ks peak for <em>Cajanus</em> at roughly 0.04—contrasting with the soybean peak at ∼0.12.<sup><a reveal-id="DSR007C25" data-open="DSR007C25" class="link link-ref link-reveal xref-bibr">25</a></sup> While this might be indicative of a recent whole-genome (or other large-scale segmental) duplication in <em>Cajanus</em>, a likely cause is incompletely collapsed contigs. The chromosome number of pigeonpea (2<em>n</em> = 22) is the same as in the other phaseoloids such as common bean (<em>Phaseolus vulgaris</em>) and cowpea (<em>Vigna unguiculata</em>), which are not known to have experienced recent polyploidy. Examination of 40 random near-identical paralogous alignments (data not shown) shows that the majority (&gt;60%) of the differences are indels in the vicinity of homopolymer runs—which very likely derive from the large proportion of 454 STRs used in the TA construction. </p>                        <a id="6640809" scrollto-destination="6640809"></a>
<div id="DSR007F2" data-id="DSR007F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00702.jpeg?Expires=2147483647&Signature=ypwukF4PGzb0LGNlzKHctWxm1ejEdhmkFGYU30mcragHWzI5Z-TXdKE7jH3gZdBbA3oL5P8WKOeDHanC-9C9F5-g0tUFJOkGWgmbfl6jiV5wujbqFq5glMOKfTLgwoDUjeWwwtBvh6yA8PjfL2~BDn3ALW63Ua5lzvnZUViCpPhpTQF2qsNBhePGRXEEunH627WtBNp8-ordmsbG4GJ6L7KHWWv7onJhybVnCDqgSGHvGoM97wMdpGiSDGDMSR3wuvZ8RTT16hxRFNSSXzr2udI5-5Ni0vBKrh6nEreqhDkJIcrvjZIqVU2b~iVAbSvbQIki1trGoBECZn8t0K0vkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram plot of pair-wise synonymous distances of pigeonpea duplicated genes when compared with soybean. Histogram plot of percentage pair-wise distance to the synonymous distance value (Ks) showing a peak in pigeonpea at 0.06 which gives a divergence estimate of ∼4.9 Mya." path-from-xml="dsr00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640809/dsr00702.jpeg" path-from-xml="dsr00702.jpeg" target="_blank">View large</a><a section="6640809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00702.jpeg?Expires=2147483647&Signature=3OVBlLvXi8TcuY5QpRQM2BEzlqBcZJOEyi11dtGC7nL2g7sozLfFMG~DBrVIOykG3Bn8dWT4BWn7ikCoAetdc-hDcBG2XTpKICZnsu8Rh47BdZqDmqmHZUZ1SujZInz1wDl1n~diafMLiBNNAFzopZk739zL518XJB8hsDcaS6MrGdNcezGbDq4Jlyv5o8qu5tW1KyFA5XKn7zNvhgl5TcSzrmIRWhKL8ArIiSHVYvnof1ggBf4mOtDz8jNLDAAPnYfq14a4DexzxFtw6RY2ivtPavJGXAtL0a42N7cG5MlJzmi1zJXWOtSJr1watgpM9Y62fFnLrAQyeLJofjZlYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640809&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram plot of pair-wise synonymous distances of pigeonpea duplicated genes when compared with soybean. Histogram plot of percentage pair-wise distance to the synonymous distance value (Ks) showing a peak in pigeonpea at 0.06 which gives a divergence estimate of ∼4.9 Mya.</p></div></div><div content-id="DSR007F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00702.jpeg?Expires=2147483647&Signature=ypwukF4PGzb0LGNlzKHctWxm1ejEdhmkFGYU30mcragHWzI5Z-TXdKE7jH3gZdBbA3oL5P8WKOeDHanC-9C9F5-g0tUFJOkGWgmbfl6jiV5wujbqFq5glMOKfTLgwoDUjeWwwtBvh6yA8PjfL2~BDn3ALW63Ua5lzvnZUViCpPhpTQF2qsNBhePGRXEEunH627WtBNp8-ordmsbG4GJ6L7KHWWv7onJhybVnCDqgSGHvGoM97wMdpGiSDGDMSR3wuvZ8RTT16hxRFNSSXzr2udI5-5Ni0vBKrh6nEreqhDkJIcrvjZIqVU2b~iVAbSvbQIki1trGoBECZn8t0K0vkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram plot of pair-wise synonymous distances of pigeonpea duplicated genes when compared with soybean. Histogram plot of percentage pair-wise distance to the synonymous distance value (Ks) showing a peak in pigeonpea at 0.06 which gives a divergence estimate of ∼4.9 Mya." path-from-xml="dsr00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640809/dsr00702.jpeg" path-from-xml="dsr00702.jpeg" target="_blank">View large</a><a section="6640809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00702.jpeg?Expires=2147483647&Signature=3OVBlLvXi8TcuY5QpRQM2BEzlqBcZJOEyi11dtGC7nL2g7sozLfFMG~DBrVIOykG3Bn8dWT4BWn7ikCoAetdc-hDcBG2XTpKICZnsu8Rh47BdZqDmqmHZUZ1SujZInz1wDl1n~diafMLiBNNAFzopZk739zL518XJB8hsDcaS6MrGdNcezGbDq4Jlyv5o8qu5tW1KyFA5XKn7zNvhgl5TcSzrmIRWhKL8ArIiSHVYvnof1ggBf4mOtDz8jNLDAAPnYfq14a4DexzxFtw6RY2ivtPavJGXAtL0a42N7cG5MlJzmi1zJXWOtSJr1watgpM9Y62fFnLrAQyeLJofjZlYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640809&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram plot of pair-wise synonymous distances of pigeonpea duplicated genes when compared with soybean. Histogram plot of percentage pair-wise distance to the synonymous distance value (Ks) showing a peak in pigeonpea at 0.06 which gives a divergence estimate of ∼4.9 Mya.</p></div></div>                            <h3 scrollto-destination=6640810 id="6640810" class="section-title" data-legacy-id=s3c>3.3. Characterization of the pigeonpea transcriptome</h3>
                            <h4 scrollto-destination=6640811 id="6640811" class="section-title" data-legacy-id=s3c1>3.3.1. Comparison with the soybean genome</h4>
<p>As an effort to validate gene structures in the newly developed assembly, the 127 754 TUSs were aligned to soybean using GMAP, with identity and coverage thresholds of 90% and 80%, respectively.<sup><a reveal-id="DSR007C26" data-open="DSR007C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Of these TUSs, 33 874 had matches at these thresholds, corresponding to 10 857 soybean genes. The TUSs were distributed similarly across the 20 chromosomes of soybean, with an average of ∼1693 loci on each chromosome, with the exception of chromosome 13, which had 4162 matches. The excess on chromosome 13 is primarily due to many matches across a long ribosomal array on this chromosome. The alignment results are available as a track in the SoyBase genome browser (<a class="link link-uri" href="http://soybase.org/gbrowse/cgi-bin/gbrowse/gmax1.01/">http://soybase.org/gbrowse/cgi-bin/gbrowse/gmax1.01/</a>).</p>                            <h4 scrollto-destination=6640813 id="6640813" class="section-title" data-legacy-id=s3c2>3.3.2. Functional annotation and GO categorization</h4>
<p>Putative assignments of 127 754 TUSs into functional categories resulted in the assignment of 32 719 TUSs (25.6%) with similarity to the UniRef non-redundant protein database (BLASTX bit-score &gt;50 and a significant <em>e</em>-value ≤ 1E−08), while 8949 sequences (7.0%) had low similarity and 86 086 (67.3%) sequences had no significant matches (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The TUSs could be placed into GO categories: biological process (5455), cellular component (3958) and molecular function (6491). Enzyme IDs retrieved from the UniProt database were distributed in one of the six major enzyme classes: transferases 31% (474), followed by hydrolases 28% (443), oxido-reductases 25% (389) ligases 6% (98), lyases 5% (79) and isomerases 5% (79). Further details about the GO categories and enzyme IDs of the TUSs are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. A noteworthy aspect of this analysis is that the majority of the transcripts were involved in metabolic and cellular process—as expected since most libraries were derived from developing tissues.<sup><a reveal-id="DSR007C27" data-open="DSR007C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h4 scrollto-destination=6640815 id="6640815" class="section-title" data-legacy-id=s3c3>3.3.3. Identification of disease-responsive genes</h4>
<p>SMD and FW are two serious diseases that adversely affect pigeonpea production. With an objective of identifying candidate genes for these diseases, Illumina 1G sequencing was used on the transcriptomes of FW-challenged root tissues and SMD-challenged leaf tissues of five each resistant (ICPL 87119, ICPB 2049, ICPL 20096, BSMR 736 and ICPL 7035) and susceptible genotypes (ICPL 87091, ICPL 99050, TAT 10, ICPL 332 and TTB 7). The number of Illumina tags (36 bp long) ranged from 18 644 113 (ICPL 87119) to 14 514 194 (TAT 10) for the 10 genotypes. The sequence data of these Illumina tags have been submitted to the National Center for Biotechnology Information (NCBI). The data can be accessed at (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) and accession numbers are: SRA030523.1 to SRP005971.1. These tags were aligned to the CcTA (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). As a result, ∼35 million Illumina tags could be aligned to 54 426 TUSs. Numerical comparison of these tags between a pair of resistant and susceptible genotypes for a disease (usually the parents of a mapping population) was used to identify differentially expressed genes for a disease.</p><p>Since the numbers of Illumina tags mapped to the transcriptome assembly varied among genotypes, the data were normalized per million reads. For the SMD study, a numerical comparison of SMD-responsive reads generated from three resistant (ICPL 20096, BSMR 736 and ICPL 7035) and three susceptible (ICPL 332, TAT 10 and TTB 7) genotypes representing three mapping populations was conducted. The Log 2 threshold for this analysis was taken as −2 to +2. The number of TUSs showing expression differences at these cutoffs ranged from 7505 (BSMR 736 × TAT 10) to 10 497 (ICPL 20096 × ICPL 332). In the case of the TTB 7 × ICPL 7035 combination, the number of differentially expressed genes was 9402. Similarly, in the FW study, a comparison was made between the specific parental combinations used to develop two different mapping populations (with the same thresholds) to find TUSs with differential expression. The number of TUSs with significant differentially expressed genes ranged from 6673 (ICPB 2049 × ICPL 99050) to 11 518 (ICPL 87119 × ICPL 87091) (Fig. <a reveal-id="DSR007F3" data-open="DSR007F3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="6640818" scrollto-destination="6640818"></a>
<div id="DSR007F3" data-id="DSR007F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00703.jpeg?Expires=2147483647&Signature=NjF8qUQQfLoSY04IT7DElnoZOScLNMJxZmxUXHrnVZAEMaquDP2xfRfr-pYHQs5dRJRx7hEYfY9Wsgkqehm9-BrMl~p3~jbah8pyajeISMhHLAfaJAYTPCeqrb2tQNZtD7GMjFfnNiBKdiTPYZ3pBOA1XRetQ5cyytOQv2Aw4hrlHFCiQymvJXuSGOQOAXqa1NSnKueHmlFuPru97LWiM9vNK1DidLjg5KVmAo9LUrsWZGv3JUBlLy7RlvkBXz0k0JHkCBbZm0teoVy9hw4VhzsJYZPHWsCore4BUJnRgE7UqWlJdpk1Sf-3AXNZ892rur3YcWKPUnLNve3Q9DX2eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of differentially expressed genes in SMD- and FW-responsive genotypes. Differential expression was calculated based on Log 2 value, with a threshold of less than −2 to greater than +2 number of differentially expressed gene was calculated for three SMD- and two FW-parental combinations. Comparison of expression values from susceptible parent to the resistant gave an estimate of up- and down-regulated genes in each cross." path-from-xml="dsr00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640818/dsr00703.jpeg" path-from-xml="dsr00703.jpeg" target="_blank">View large</a><a section="6640818" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00703.jpeg?Expires=2147483647&Signature=HuIP5DUKMBxGnWcxG6jCKlvvesbOOoWDXp2PwL9LPTGQV1j99wdN9BgMi5jyQ5OSIog0fkACwEb3WLHU3xsZtjr2QUjQhWgZbXu5mnad9BDgL6UP-FPPOAkYyg4wbfYODIwv30W7SruQWLTRKWOf8UGqUnpmUbbXBkag1i4U-Q04Hvs3NMB4-Rt4HgUGp0mYgVyONGCnA7LuJts-7zB5KJjaG5GZRWepYDO8pO4VfCmN9z3LzIYy2aNIEFUeUj03Yx1OTQa9ZV7t5NfJv3R1GwTYdUnCjRLeAi1PVXkfqz0On8b90SJWIUEZYG9nG5HU7A3xkbGww5u5KvBKSlsLWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640818&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of differentially expressed genes in SMD- and FW-responsive genotypes. Differential expression was calculated based on Log 2 value, with a threshold of less than −2 to greater than +2 number of differentially expressed gene was calculated for three SMD- and two FW-parental combinations. Comparison of expression values from susceptible parent to the resistant gave an estimate of up- and down-regulated genes in each cross.</p></div></div><div content-id="DSR007F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00703.jpeg?Expires=2147483647&Signature=NjF8qUQQfLoSY04IT7DElnoZOScLNMJxZmxUXHrnVZAEMaquDP2xfRfr-pYHQs5dRJRx7hEYfY9Wsgkqehm9-BrMl~p3~jbah8pyajeISMhHLAfaJAYTPCeqrb2tQNZtD7GMjFfnNiBKdiTPYZ3pBOA1XRetQ5cyytOQv2Aw4hrlHFCiQymvJXuSGOQOAXqa1NSnKueHmlFuPru97LWiM9vNK1DidLjg5KVmAo9LUrsWZGv3JUBlLy7RlvkBXz0k0JHkCBbZm0teoVy9hw4VhzsJYZPHWsCore4BUJnRgE7UqWlJdpk1Sf-3AXNZ892rur3YcWKPUnLNve3Q9DX2eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of differentially expressed genes in SMD- and FW-responsive genotypes. Differential expression was calculated based on Log 2 value, with a threshold of less than −2 to greater than +2 number of differentially expressed gene was calculated for three SMD- and two FW-parental combinations. Comparison of expression values from susceptible parent to the resistant gave an estimate of up- and down-regulated genes in each cross." path-from-xml="dsr00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640818/dsr00703.jpeg" path-from-xml="dsr00703.jpeg" target="_blank">View large</a><a section="6640818" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00703.jpeg?Expires=2147483647&Signature=HuIP5DUKMBxGnWcxG6jCKlvvesbOOoWDXp2PwL9LPTGQV1j99wdN9BgMi5jyQ5OSIog0fkACwEb3WLHU3xsZtjr2QUjQhWgZbXu5mnad9BDgL6UP-FPPOAkYyg4wbfYODIwv30W7SruQWLTRKWOf8UGqUnpmUbbXBkag1i4U-Q04Hvs3NMB4-Rt4HgUGp0mYgVyONGCnA7LuJts-7zB5KJjaG5GZRWepYDO8pO4VfCmN9z3LzIYy2aNIEFUeUj03Yx1OTQa9ZV7t5NfJv3R1GwTYdUnCjRLeAi1PVXkfqz0On8b90SJWIUEZYG9nG5HU7A3xkbGww5u5KvBKSlsLWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640818&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of differentially expressed genes in SMD- and FW-responsive genotypes. Differential expression was calculated based on Log 2 value, with a threshold of less than −2 to greater than +2 number of differentially expressed gene was calculated for three SMD- and two FW-parental combinations. Comparison of expression values from susceptible parent to the resistant gave an estimate of up- and down-regulated genes in each cross.</p></div></div><p>Based on the expression values for differentially expressed genes in SMD- and FW-responsive genotypes, hierarchical clustering was done for SMD- and FW-responsive genes separately to compare the pattern of gene expression. These clusters show the pattern of co-regulated genes for the SMD-responsive genotypes (Fig. <a reveal-id="DSR007F4" data-open="DSR007F4" class="link link-reveal link-table xref-fig">4</a>A) and for the FW-responsive genotypes (Fig. <a reveal-id="DSR007F4" data-open="DSR007F4" class="link link-reveal link-table xref-fig">4</a>B). </p>                        <a id="6640820" scrollto-destination="6640820"></a>
<div id="DSR007F4" data-id="DSR007F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00704.jpeg?Expires=2147483647&Signature=LzlOylBRxQB8YdQSjfsJtelqseyVoZrz0FjRdcPpV3bLLBtMp7H3DAWWNZpO5ndyz~sww854efdSdjQ0f9t2pblyH2-FBYcAAnNrsysl2yXIJiqF1VjvV48dPnTEYiOJZYC6-XHpFqfdjpiUXMelOeHXN5Q2aIhj0at17bM~rGIr5HcWvb2eKdtk2xIxSj9740vkGAoyxW5T07nSfAi8ocRmoG0OsvnkYeOsRzVqdw1tRzmWGMWr-bCCFLOxzYYMeNLlTCSqzi-~oyHR5timeo17z3xY~4xxv5hBaDNXknKpxv7~pYZpnsJKjectSS1lRtBeuVFVSs~u-xR1eHUJPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical clustering of differentially expressed TAs within SMD- and FW-responsive genotypes. Hierarchial clustering of the gene involved in SMD- and FW-stress responses was done using HCE version 2.0 beta web tool. These two dendrograms illustrate the co-regulation of genes in resistant and susceptible genotypes under stress, (A) clustering of 5106 genes showing expression variation across six SMD-responsive genotypes (three resistant, ICPL 20096, ICPL 7035, BSMR 736, and three susceptible, ICPL 332, TTB 7, TAT 10). (B) Clustering of 3384 genes showing expression variation across four FW-responsive genotypes (two resistant, ICPL 87119, ICPL 99050, and two susceptible, ICPB 2049, ICPL 87091). Colour scale (from green to red) represents the range of expression level." path-from-xml="dsr00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640820/dsr00704.jpeg" path-from-xml="dsr00704.jpeg" target="_blank">View large</a><a section="6640820" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00704.jpeg?Expires=2147483647&Signature=os3DIiAkBLW6N4-PKwzl0kmVqa63TYUMi-V~~mvwCO~ldR-2AH2Mv~xQ9Pj~iMW7nMxFO-mfLnsCZwcdB2gR6ITAfSG-5B7RD7qoWRa9IsU~YPseuYKc9b2Qob91iXR6povQSlr6eu986z1EY9IYOMESrIHQPlWqzyNUghWLAXn4j4mJAKMiX34f78g-EKMx5bjCxFdLm3aejI4Z-l4eCfpvhPxEUtrJQTSRgaify87LcI2E48U75dtuBfKKjzLpyudMxeMaCSke8k1gQxmkQkPGDeeUf8Fd4Gv6PccHYXBCJt44U0kPr46NDUEi89TikX5uY5K60dPL0ohdq9kkwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640820&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical clustering of differentially expressed TAs within SMD- and FW-responsive genotypes. Hierarchial clustering of the gene involved in SMD- and FW-stress responses was done using HCE version 2.0 beta web tool. These two dendrograms illustrate the co-regulation of genes in resistant and susceptible genotypes under stress, (A) clustering of 5106 genes showing expression variation across six SMD-responsive genotypes (three resistant, ICPL 20096, ICPL 7035, BSMR 736, and three susceptible, ICPL 332, TTB 7, TAT 10). (B) Clustering of 3384 genes showing expression variation across four FW-responsive genotypes (two resistant, ICPL 87119, ICPL 99050, and two susceptible, ICPB 2049, ICPL 87091). Colour scale (from green to red) represents the range of expression level.</p></div></div><div content-id="DSR007F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/m_dsr00704.jpeg?Expires=2147483647&Signature=LzlOylBRxQB8YdQSjfsJtelqseyVoZrz0FjRdcPpV3bLLBtMp7H3DAWWNZpO5ndyz~sww854efdSdjQ0f9t2pblyH2-FBYcAAnNrsysl2yXIJiqF1VjvV48dPnTEYiOJZYC6-XHpFqfdjpiUXMelOeHXN5Q2aIhj0at17bM~rGIr5HcWvb2eKdtk2xIxSj9740vkGAoyxW5T07nSfAi8ocRmoG0OsvnkYeOsRzVqdw1tRzmWGMWr-bCCFLOxzYYMeNLlTCSqzi-~oyHR5timeo17z3xY~4xxv5hBaDNXknKpxv7~pYZpnsJKjectSS1lRtBeuVFVSs~u-xR1eHUJPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical clustering of differentially expressed TAs within SMD- and FW-responsive genotypes. Hierarchial clustering of the gene involved in SMD- and FW-stress responses was done using HCE version 2.0 beta web tool. These two dendrograms illustrate the co-regulation of genes in resistant and susceptible genotypes under stress, (A) clustering of 5106 genes showing expression variation across six SMD-responsive genotypes (three resistant, ICPL 20096, ICPL 7035, BSMR 736, and three susceptible, ICPL 332, TTB 7, TAT 10). (B) Clustering of 3384 genes showing expression variation across four FW-responsive genotypes (two resistant, ICPL 87119, ICPL 99050, and two susceptible, ICPB 2049, ICPL 87091). Colour scale (from green to red) represents the range of expression level." path-from-xml="dsr00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6640820/dsr00704.jpeg" path-from-xml="dsr00704.jpeg" target="_blank">View large</a><a section="6640820" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr00704.jpeg?Expires=2147483647&Signature=os3DIiAkBLW6N4-PKwzl0kmVqa63TYUMi-V~~mvwCO~ldR-2AH2Mv~xQ9Pj~iMW7nMxFO-mfLnsCZwcdB2gR6ITAfSG-5B7RD7qoWRa9IsU~YPseuYKc9b2Qob91iXR6povQSlr6eu986z1EY9IYOMESrIHQPlWqzyNUghWLAXn4j4mJAKMiX34f78g-EKMx5bjCxFdLm3aejI4Z-l4eCfpvhPxEUtrJQTSRgaify87LcI2E48U75dtuBfKKjzLpyudMxeMaCSke8k1gQxmkQkPGDeeUf8Fd4Gv6PccHYXBCJt44U0kPr46NDUEi89TikX5uY5K60dPL0ohdq9kkwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6640820&ar=444270&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical clustering of differentially expressed TAs within SMD- and FW-responsive genotypes. Hierarchial clustering of the gene involved in SMD- and FW-stress responses was done using HCE version 2.0 beta web tool. These two dendrograms illustrate the co-regulation of genes in resistant and susceptible genotypes under stress, (A) clustering of 5106 genes showing expression variation across six SMD-responsive genotypes (three resistant, ICPL 20096, ICPL 7035, BSMR 736, and three susceptible, ICPL 332, TTB 7, TAT 10). (B) Clustering of 3384 genes showing expression variation across four FW-responsive genotypes (two resistant, ICPL 87119, ICPL 99050, and two susceptible, ICPB 2049, ICPL 87091). Colour scale (from green to red) represents the range of expression level.</p></div></div><p>In order to study the gene expression pattern between two parental genotypes of a mapping population, the numbers of up-regulated and down-regulated TUSs were calculated with respect to the resistant parent. The numbers of up-regulated TUSs remained high in all the crosses studied, with an exception of ICPL 87119 × ICPL 87091, which had more down-regulated TUSs (11 364) when compared with up-regulated (154). Log 2 fold differences between the parental combinations are shown in Table <a reveal-id="DSR007TB2" data-open="DSR007TB2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="6640822" scrollto-destination="6640822"></a>
<div content-id="DSR007TB2" class="table-modal"><div class="table-wrap"><div id="DSR007TB2" class="table-wrap-title" data-id="DSR007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of differentially expressed TUSs across five parental combinations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Fold difference </th><th>ICPL 20096 × ICPL 332 </th><th>BSMR 736 × TAT 10 </th><th>TTB 7 × ICPL 7035 </th><th>ICPL 87119 × ICPL 87091 </th><th>ICPB 2049 × ICPL 99050 </th></tr></thead><tbody><tr><td>Exclusively expressed in resistant </td><td>9149 </td><td>8867 </td><td>9455 </td><td>243 </td><td>10 675 </td></tr><tr><td>Exclusively expressed in susceptible </td><td>1964 </td><td>269 </td><td>200 </td><td>9784 </td><td>2098 </td></tr><tr><td>2 to &lt;3 fold </td><td>3385 </td><td>2663 </td><td>3024 </td><td>3715 </td><td>2638 </td></tr><tr><td>3 to &lt;4 fold </td><td>2782 </td><td>2111 </td><td>3082 </td><td>3859 </td><td>1779 </td></tr><tr><td>4 to &lt;5 fold </td><td>2108 </td><td>1499 </td><td>2180 </td><td>2618 </td><td>1167 </td></tr><tr><td>5 to &lt;6 fold </td><td>1544 </td><td>807 </td><td>856 </td><td>1053 </td><td>674 </td></tr><tr><td>≥6 fold </td><td>998 </td><td>385 </td><td>260 </td><td>273 </td><td>415 </td></tr></tbody></table></div><div content-id="DSR007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Fold difference </th><th>ICPL 20096 × ICPL 332 </th><th>BSMR 736 × TAT 10 </th><th>TTB 7 × ICPL 7035 </th><th>ICPL 87119 × ICPL 87091 </th><th>ICPB 2049 × ICPL 99050 </th></tr></thead><tbody><tr><td>Exclusively expressed in resistant </td><td>9149 </td><td>8867 </td><td>9455 </td><td>243 </td><td>10 675 </td></tr><tr><td>Exclusively expressed in susceptible </td><td>1964 </td><td>269 </td><td>200 </td><td>9784 </td><td>2098 </td></tr><tr><td>2 to &lt;3 fold </td><td>3385 </td><td>2663 </td><td>3024 </td><td>3715 </td><td>2638 </td></tr><tr><td>3 to &lt;4 fold </td><td>2782 </td><td>2111 </td><td>3082 </td><td>3859 </td><td>1779 </td></tr><tr><td>4 to &lt;5 fold </td><td>2108 </td><td>1499 </td><td>2180 </td><td>2618 </td><td>1167 </td></tr><tr><td>5 to &lt;6 fold </td><td>1544 </td><td>807 </td><td>856 </td><td>1053 </td><td>674 </td></tr><tr><td>≥6 fold </td><td>998 </td><td>385 </td><td>260 </td><td>273 </td><td>415 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6640822" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR007TB2" class="table-wrap-title" data-id="DSR007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of differentially expressed TUSs across five parental combinations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Fold difference </th><th>ICPL 20096 × ICPL 332 </th><th>BSMR 736 × TAT 10 </th><th>TTB 7 × ICPL 7035 </th><th>ICPL 87119 × ICPL 87091 </th><th>ICPB 2049 × ICPL 99050 </th></tr></thead><tbody><tr><td>Exclusively expressed in resistant </td><td>9149 </td><td>8867 </td><td>9455 </td><td>243 </td><td>10 675 </td></tr><tr><td>Exclusively expressed in susceptible </td><td>1964 </td><td>269 </td><td>200 </td><td>9784 </td><td>2098 </td></tr><tr><td>2 to &lt;3 fold </td><td>3385 </td><td>2663 </td><td>3024 </td><td>3715 </td><td>2638 </td></tr><tr><td>3 to &lt;4 fold </td><td>2782 </td><td>2111 </td><td>3082 </td><td>3859 </td><td>1779 </td></tr><tr><td>4 to &lt;5 fold </td><td>2108 </td><td>1499 </td><td>2180 </td><td>2618 </td><td>1167 </td></tr><tr><td>5 to &lt;6 fold </td><td>1544 </td><td>807 </td><td>856 </td><td>1053 </td><td>674 </td></tr><tr><td>≥6 fold </td><td>998 </td><td>385 </td><td>260 </td><td>273 </td><td>415 </td></tr></tbody></table></div><div content-id="DSR007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Fold difference </th><th>ICPL 20096 × ICPL 332 </th><th>BSMR 736 × TAT 10 </th><th>TTB 7 × ICPL 7035 </th><th>ICPL 87119 × ICPL 87091 </th><th>ICPB 2049 × ICPL 99050 </th></tr></thead><tbody><tr><td>Exclusively expressed in resistant </td><td>9149 </td><td>8867 </td><td>9455 </td><td>243 </td><td>10 675 </td></tr><tr><td>Exclusively expressed in susceptible </td><td>1964 </td><td>269 </td><td>200 </td><td>9784 </td><td>2098 </td></tr><tr><td>2 to &lt;3 fold </td><td>3385 </td><td>2663 </td><td>3024 </td><td>3715 </td><td>2638 </td></tr><tr><td>3 to &lt;4 fold </td><td>2782 </td><td>2111 </td><td>3082 </td><td>3859 </td><td>1779 </td></tr><tr><td>4 to &lt;5 fold </td><td>2108 </td><td>1499 </td><td>2180 </td><td>2618 </td><td>1167 </td></tr><tr><td>5 to &lt;6 fold </td><td>1544 </td><td>807 </td><td>856 </td><td>1053 </td><td>674 </td></tr><tr><td>≥6 fold </td><td>998 </td><td>385 </td><td>260 </td><td>273 </td><td>415 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6640822" target="_blank">View Large</a></div></div><p>Functional annotations of differentially expressed TUSs are described next, with the additional requirement of ≥5 fold differences across all the parental combinations. The annotation analysis was conducted in three ways: (i) TUSs differentially expressed across all the 10 genotypes, (ii) TUSs differentially expressed in 6 SMD-responsive genotypes separately and in 4 FW-responsive genotypes separately and (iii) the common set of TUSs that is differentially expressed in both FW- and SMD-responsive genotypes. Based on these analyses, in the first category, 6107 TUSs (with fold difference ≥5) were selected for functional classification. Considering an <em>E</em>-value cutoff of ≤1E−08 and a bit-score value of ≥50, functional annotation for 3698 TUSs showed significant similarity with the UniRef non-redundant protein database. No significant matches were found for 2409 TUSs. For 3698 TUSs with functional classes, we found that in addition to basic housekeeping genes, these TUSs also showed homology to genes involved in stress response, such as proline-rich protein, Syringolide-induced protein, desiccation protective protein of soybean, ABA-responsive protein, and leucine zipper protein (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>Among these 3698 TUSs, 2106 could be assigned into three major categories: (i) molecular function, (ii) biological process and (iii) cellular component. These categories were further subcategorized, i.e. under molecular function category, the subcategory ‘binding’ accounted for highest percentage of TUSs (594), followed by ‘catalytic activity’ (513), ‘transporter’ (58), ‘structural molecule’ (52) and rest of the subcategories accounting for 75 TUSs. Similarly, under ‘biological process’ category, the highest number of TUSs were assigned to the subcategory ‘metabolic process’ (571) followed by ‘cellular process’ (542), response to ‘stimuli’ (152), ‘biological regulation’ (118), ‘establishment of localization’ (108) and 363 TUSs accounted for rest of the subcategories. Under ‘cellular component’ category, the highest percentage of TUSs was assigned to the subcategory ‘cell part’ (562) followed by ‘organelle’ (381), ‘organelle part’ (202), ‘macro molecule complex’ (158) and 75 TUSs were assigned to rest of the subcategories.</p><p>Differentially expressed genes included those encoding proline-rich proteins, zinc finger proteins, leucoanthocyanidin dioxygenase and RAS-related protein. There were seven TUSs that correspond to proline-rich protein. This protein forms a component of glutamate pathway and has multiple developmental and stress-related functions. High expression of this protein in leaves has been reported to play major role in the early stage of virus infection in soybean.<sup><a reveal-id="DSR007C28" data-open="DSR007C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The glutamate pathway assimilates nitrogen and produces glutamate, which then acts as a starting point for amino acid synthesis. A 5-fold up-regulation of this gene in resistant genotype (ICPL 2049) probably implicates its role in response to this stress. A gene for zinc finger protein showed an average of 5-fold differential expression in both SMD- and FW-responsive genotypes. This protein is a component of mitogen-activated protein kinase (MAPK) pathways which are demonstrated to play an important role in regulating the gene expression in response to various biotic as well as abiotic stress in species such as Arabidopsis.<sup><a reveal-id="DSR007C29" data-open="DSR007C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSR007C30" data-open="DSR007C30" class="link link-ref link-reveal xref-bibr">30</a></sup> MAPK pathways transduce a large variety of external signals, leading to a wide range of cellular responses, including growth, differentiation, inflammation and apoptosis. A total of six TUSs had homologies to leucoanthocyanidin dioxygenase, and showed an average of 5-fold differential expression. The up-regulation of this gene (in the flavonoid pathway) is known to play an important role in defence against both biotic and abiotic stress by acting as a passive or inducible barrier against pathogens.<sup><a reveal-id="DSR007C31" data-open="DSR007C31" class="link link-ref link-reveal xref-bibr">31</a></sup> A total of 21 TUSs showed homology to gene for RAS-related protein ARA-3. These TUSs were up-regulated 6-fold. This gene is involved in the ethylene-mediated signalling pathway, suggesting an important role in stress response.</p><p>With an objective of identifying candidate genes for FW and SMD, as mentioned above, the common set of TUSs with ≥5 fold expression difference was identified in SMD- and FW-responsive genotypes, separately. From this, 99 common TUSs were found for FW-responsive genotypes and 13 for SMD-responsive genotypes. Functional characterization of these genes showed function for 51 FW-responsive TUSs and 3 SMD-responsive TUSs (oxygen-evolving enhancer protein, NADH-ubiquinone oxidoreductase and sedoheptulose-1,7-biphosphate). FW responsive TUSs include genes such as mannose-1-phosphate guanyltransferase, prolinedehydrogenase, cellulose synthase, pectinesterase inhibitor, superoxide dismutase [Fe] and vacuolar protein sorting-associated protein.</p><p>Among FW-responsive genes were TUSs showing cellulose synthase homology. These genes are essential for secondary cell wall synthesis. Among the SMD-responsive TUSs, one showed homology to oxygen evolving enhancer protein, with an average 5-fold expression difference. These proteins are components of glycine-rich protein 3/wall-associated kinase. One TUS showed homology to a gene coding for NADH-ubiquinone oxidoreductase and was up-regulated in resistant with respect to susceptible genotypes for SMD. This is a common component for energy evolving pathways in the cell.</p><p>Considering biotic stress responsive genes in common for FW and SMD, no TUS was found common at the threshold of ≥5 fold difference. When this threshold was decreased to ≥2 fold, the number of common TUSs across FW- and SMD-responsive genotypes was found to be 192. Of this set, 99 TUSs were functionally annotated and 93 were uncharacterized proteins. These annotated TUSs showed sequence similarity to several stress responsive genes such as zinc protein, aminocyclopropane carboxylate oxidase, cysteine protease and hexokinase. For example, two TAs showed homology to a gene corresponding to zinc finger protein and expressed with an average of 3.4-fold difference. As mentioned, this gene is a component of MAPK. TUSs with sequence similarity to gene for 1-aminocyclopropane-1-carboxylate oxidase were expressed with an average 3.3 folds. This gene is a component of ethylene-biosynthesis pathway which plays an important role in ethylene biosynthesis at stress conditions.<sup><a reveal-id="DSR007C32" data-open="DSR007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> TUSs with sequence similarity to gene for synthesis of germination-specific cysteine protease also showed an average of 3-fold difference in expression value, this gene is responsible for cell death hence regulating response to stress. Sequence similarity for another gene encoding for hexokinase-2 was also discovered for one TUS which showed an average-fold difference of 2.9. This gene is known to play a major role in metabolic pathways, e.g. fructose and mannose metabolism, galactose metabolism and glycolysis.</p><p>Genes responding in the FW- and SMD-resistant lines will provide a rich basis for further explorations of the mechanisms of disease resistance for these important viral and fungal diseases, and may also be useful in identifying regulatory networks and targets for breeding efforts, As no controls (Illumina tags generated from non-challenged tissues) were used for identification for FW- and SMD-responsive genes, like recent studies in wheat (<em>Triticum aestivum</em>)<sup><a reveal-id="DSR007C33" data-open="DSR007C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and yam (<em>Dioscorea alata</em> L.)<sup><a reveal-id="DSR007C34" data-open="DSR007C34" class="link link-ref link-reveal xref-bibr">34</a></sup>, it is, therefore suggested to use other techniques like qRT–PCR to validate and establish magnitudes of expression levels of identified genes before they are used for further studies.</p>                            <h3 scrollto-destination=6640830 id="6640830" class="section-title" data-legacy-id=s3d>3.4. Development of gene-based molecular markers</h3>
<p>Genetic markers are important tools for understanding variation, and for identification of gene/QTLs for traits of interest in molecular breeding activities. Until recently, a very limited number of genetic markers have been available for pigeonpea.<sup><a reveal-id="DSR007C35" data-open="DSR007C35" class="link link-ref link-reveal xref-bibr">35</a></sup> One of the main reasons for the lack of mapping resources in pigeonpea is the low level of polymorphism. An approach to develop genetic markers is the mining of ESTs or transcript sequences for the presence of SSRs and SNPs.<sup><a reveal-id="DSR007C36" data-open="DSR007C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Although markers developed from ESTs/transcripts are less polymorphic, they have been found useful for assaying the functional diversity in the germplasm collection<sup><a reveal-id="DSR007C37" data-open="DSR007C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSR007C38" data-open="DSR007C38" class="link link-ref link-reveal xref-bibr">38</a></sup>, trait mapping<sup><a reveal-id="DSR007C27" data-open="DSR007C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and comparative genomics studies.<sup><a reveal-id="DSR007C39" data-open="DSR007C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h4 scrollto-destination=6640832 id="6640832" class="section-title" data-legacy-id=s3d1>3.4.1. SSR discovery</h4>
<p>As microsatellite or SSR markers are the markers of choice for many plant breeding applications, the TUSs were analysed for identification and development of SSR markers. Analysis of 127 754 TUSs with the <em>MISA</em> search program<sup><a reveal-id="DSR007C21" data-open="DSR007C21" class="link link-ref link-reveal xref-bibr">21</a></sup> identified 50 566 SSRs in 41 899 TUSs (32.7%), with a frequency of one SSR per 570 bp. In terms of abundance, mono-nucleotide repeats were most abundant (33 262, 65.7%) followed by di- (13 204, 26.1%) and tri-nucleotide repeats (3063, 6%). Other type of repeat units occurred at &lt;1% each (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). SSRs were divided into perfect (i.e. SSRs containing a single repeat motif such as ‘AAG’) or compound (i.e. composed of two or more SSRs separated by ≤100 bp) SSRs. A total of 6350 (15.1%) compound SSRs were identified. The frequency of SSR repeat motifs was calculated after excluding the mono-nucleotide repeats. Among di-nucleotide repeats motifs, AG/CT was the most abundant with 46.8%; tri-nucleotide repeats motifs were rich in AAG/CTT (32.5%); and among tetra-, penta- and hexa- nucleotide repeat motifs, the most abundant repeats were AAAT/ATTT (24.1%), AAGGT/ATTCC (20.8%) and AAAAAG/CTTTTT (8.4%), respectively.</p><p>With an objective of converting the identified SSRs into genetic markers, 9157 primer pairs were designed for all SSRs except mono-nucleotides. Recently, 3312 SSR markers have been designed for pigeonpea<sup><a reveal-id="DSR007C4  DSR007C8" data-open="DSR007C4  DSR007C8" class="link link-ref link-reveal xref-bibr">4–8</a>,<a reveal-id="DSR007C35" data-open="DSR007C35" class="link link-ref link-reveal xref-bibr">35</a></sup>. An analysis to determine overlap between the 9157 newly designed primer pairs and 3312 published SSR markers (using BLAST) identified 8137 primer pairs as novel SSR markers for pigeonpea (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Validation of these SSR primer pairs, however, is required to determine their potential to amplify and to detect polymorphisms. In several crop species, SNP markers are becoming more popular because of their low cost and potential for automation.<sup><a reveal-id="DSR007C40" data-open="DSR007C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p>                            <h4 scrollto-destination=6640835 id="6640835" class="section-title" data-legacy-id=s3d2>3.4.2. SNP discovery</h4>
<p>In total, 150.8 million Illumina sequence tags were generated from 10 genotypes. For identification of SNPs, tags for two genotypes of a given mapping population were aligned with 127 754 TUSs (the pigeonpea transcriptome assembly), and variants were identified using the Alpheus program of NCGR.<sup><a reveal-id="DSR007C23" data-open="DSR007C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The number of SNPs in an individual cross ranged from 704 (BSMR 736 × TAT 10) to 6263 (ICPL 87119 × ICPL 87091) (Table <a reveal-id="DSR007TB3" data-open="DSR007TB3" class="link link-reveal link-table xref-fig">3</a>). In total, 12 141 SNPs were identified; however, only six SNPs were found in common across three populations (ICPL 20096 × ICPL 332, ICPL 7035 × TTB7 and BSMR 736 × TAT 10). The number of common SNPs across any two mapping populations ranged from 8 (ICPL 99050 × ICPL 2049 and BSMR736 × TAT 10) and 39 (ICPL 87119 × ICPL 87091 and ICLP 20096 × ICPL 332). Although a large number of SNP genotyping platforms are available,<sup><a reveal-id="DSR007C36" data-open="DSR007C36" class="link link-ref link-reveal xref-bibr">36</a></sup> depending on the need and requirements, most suitable platform can be selected for using the SNPs in pigeonpea genetics and breeding applications. For instance, GoldenGate assays of Illumina (<a class="link link-uri" href="http://www.illumina.com/">www.illumina.com/</a>) will offer high-throughput SNP genotyping while KASPar (<a class="link link-uri" href="http://www.kbioscience.co.uk">www.kbioscience.co.uk</a>) or cleaved amplified polymorphic sequence (CAPS) assays<sup><a reveal-id="DSR007C41" data-open="DSR007C41" class="link link-ref link-reveal xref-bibr">41</a></sup> will allow low-cost SNP genotyping. </p>                        <a id="6640837" scrollto-destination="6640837"></a>
<div content-id="DSR007TB3" class="table-modal"><div class="table-wrap"><div id="DSR007TB3" class="table-wrap-title" data-id="DSR007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Illumina sequencing based SNP discovery in five parental combinations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>ICPL 87119 </th><th>ICPL 87091 </th><th>BSMR 736 </th><th>TAT 10 </th><th>TTB 7 </th><th>ICPL 7035 </th><th>ICPL 20096 </th><th>ICPL 332 </th><th>ICPB 2049 </th><th>ICPL 99050 </th></tr></thead><tbody><tr><td>Number of reads (in millions) </td><td>18.4 </td><td>16.8 </td><td>16.9 </td><td>14.5 </td><td>15.7 </td><td>17.2 </td><td>15.5 </td><td>14.8 </td><td>15.7 </td><td>17.3 </td></tr><tr><td>Number of SNPs in parental combination </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Substitution </td><td colspan="2">5965 </td><td colspan="2">573 </td><td colspan="2">955 </td><td colspan="2">1580 </td><td colspan="2">1878 </td></tr><tr><td> Insertion </td><td colspan="2">176 </td><td colspan="2">87 </td><td colspan="2">28 </td><td colspan="2">31 </td><td colspan="2">61 </td></tr><tr><td> Deletion </td><td colspan="2">122 </td><td colspan="2">44 </td><td colspan="2">44 </td><td colspan="2">43 </td><td colspan="2">38 </td></tr><tr><td> Total SNPs </td><td colspan="2">6263 </td><td colspan="2">704 </td><td colspan="2">1027 </td><td colspan="2">1654 </td><td colspan="2">1977 </td></tr></tbody></table></div><div content-id="DSR007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>ICPL 87119 </th><th>ICPL 87091 </th><th>BSMR 736 </th><th>TAT 10 </th><th>TTB 7 </th><th>ICPL 7035 </th><th>ICPL 20096 </th><th>ICPL 332 </th><th>ICPB 2049 </th><th>ICPL 99050 </th></tr></thead><tbody><tr><td>Number of reads (in millions) </td><td>18.4 </td><td>16.8 </td><td>16.9 </td><td>14.5 </td><td>15.7 </td><td>17.2 </td><td>15.5 </td><td>14.8 </td><td>15.7 </td><td>17.3 </td></tr><tr><td>Number of SNPs in parental combination </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Substitution </td><td colspan="2">5965 </td><td colspan="2">573 </td><td colspan="2">955 </td><td colspan="2">1580 </td><td colspan="2">1878 </td></tr><tr><td> Insertion </td><td colspan="2">176 </td><td colspan="2">87 </td><td colspan="2">28 </td><td colspan="2">31 </td><td colspan="2">61 </td></tr><tr><td> Deletion </td><td colspan="2">122 </td><td colspan="2">44 </td><td colspan="2">44 </td><td colspan="2">43 </td><td colspan="2">38 </td></tr><tr><td> Total SNPs </td><td colspan="2">6263 </td><td colspan="2">704 </td><td colspan="2">1027 </td><td colspan="2">1654 </td><td colspan="2">1977 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6640837" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR007TB3" class="table-wrap-title" data-id="DSR007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Illumina sequencing based SNP discovery in five parental combinations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>ICPL 87119 </th><th>ICPL 87091 </th><th>BSMR 736 </th><th>TAT 10 </th><th>TTB 7 </th><th>ICPL 7035 </th><th>ICPL 20096 </th><th>ICPL 332 </th><th>ICPB 2049 </th><th>ICPL 99050 </th></tr></thead><tbody><tr><td>Number of reads (in millions) </td><td>18.4 </td><td>16.8 </td><td>16.9 </td><td>14.5 </td><td>15.7 </td><td>17.2 </td><td>15.5 </td><td>14.8 </td><td>15.7 </td><td>17.3 </td></tr><tr><td>Number of SNPs in parental combination </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Substitution </td><td colspan="2">5965 </td><td colspan="2">573 </td><td colspan="2">955 </td><td colspan="2">1580 </td><td colspan="2">1878 </td></tr><tr><td> Insertion </td><td colspan="2">176 </td><td colspan="2">87 </td><td colspan="2">28 </td><td colspan="2">31 </td><td colspan="2">61 </td></tr><tr><td> Deletion </td><td colspan="2">122 </td><td colspan="2">44 </td><td colspan="2">44 </td><td colspan="2">43 </td><td colspan="2">38 </td></tr><tr><td> Total SNPs </td><td colspan="2">6263 </td><td colspan="2">704 </td><td colspan="2">1027 </td><td colspan="2">1654 </td><td colspan="2">1977 </td></tr></tbody></table></div><div content-id="DSR007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>ICPL 87119 </th><th>ICPL 87091 </th><th>BSMR 736 </th><th>TAT 10 </th><th>TTB 7 </th><th>ICPL 7035 </th><th>ICPL 20096 </th><th>ICPL 332 </th><th>ICPB 2049 </th><th>ICPL 99050 </th></tr></thead><tbody><tr><td>Number of reads (in millions) </td><td>18.4 </td><td>16.8 </td><td>16.9 </td><td>14.5 </td><td>15.7 </td><td>17.2 </td><td>15.5 </td><td>14.8 </td><td>15.7 </td><td>17.3 </td></tr><tr><td>Number of SNPs in parental combination </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Substitution </td><td colspan="2">5965 </td><td colspan="2">573 </td><td colspan="2">955 </td><td colspan="2">1580 </td><td colspan="2">1878 </td></tr><tr><td> Insertion </td><td colspan="2">176 </td><td colspan="2">87 </td><td colspan="2">28 </td><td colspan="2">31 </td><td colspan="2">61 </td></tr><tr><td> Deletion </td><td colspan="2">122 </td><td colspan="2">44 </td><td colspan="2">44 </td><td colspan="2">43 </td><td colspan="2">38 </td></tr><tr><td> Total SNPs </td><td colspan="2">6263 </td><td colspan="2">704 </td><td colspan="2">1027 </td><td colspan="2">1654 </td><td colspan="2">1977 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6640837" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=6640838 id="6640838" class="section-title" data-legacy-id=s3d3>3.4.3. Identification of intron-spanning region markers</h4>
<p>Identification of potential splice site was undertaken as an extension to the usability of data generated from 127 754 TUSs aligned against soybean genome sequence for development of genetic markers in pigeonpea. Thresholds were set to a minimum alignment length of 70 nucleotides, minimum query coverage of 90% and minimum per cent identity of 80%. A total of 8532 (6.0%) TUSs showed valid alignment with 8491 unique soybean loci, and having not more than 1000 bp gaps between aligned components. Alignment of these 8532 TUSs with one or more splice sites yielded 13 862 putative intron-spanning splice junctions. Alignment results were used for development of intron-spanning region (ISR) markers <em>in silico</em> for pigeonpea. These markers were derived for portions of genes and were designed from low copy sequences (sequences showing single match to the reference). In summary, a total of 5845 ISR primer pairs were designed across the pigeonpea genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). These potential ISR markers are also available as a GBrowse track at <a class="link link-uri" href="http://soybase.org/gbrowse/cgi-bin/gbrowse/gmax1.01">http://soybase.org/gbrowse/cgi-bin/gbrowse/gmax1.01</a>. These ISR markers can be assayed on mutation detection enhancement (MDE) gel for detection of polymorphism.</p><p>In summary, large-scale transcriptomic resource has been developed in an under-resourced crop species by deploying two prominent NGS technologies namely FLX/454 and Illumina IG sequencing. These transcript data have been used for both basic and applied aspects in pigeonpea genetics and breeding. Similarly, genes responsive to FW and SMD have been identified, and are ready for further study and validation. Additionally, a large number of molecular markers have been developed that will accelerate molecular mapping and molecular breeding activities for pigeonpea improvement.</p>                            <h2 scrollto-destination=6640841 id="6640841" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6640843 id="6640843" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This study was supported financially by SP2 Leader Discretionary Grant of CGIAR Generation Challenge Programme, Mexico; and Pigeonpea Genomics Initiative (PGI) of Indian Council of Agricultural Research (ICAR) of Government of India.</p>                            <h2 scrollto-destination=6640845 id="6640845" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are thankful to Dr Pawan Kulwal, Dr Punjabrao Deshmukh Krishi Vishwavidalaya, Akola (India), for providing the seeds of some genotypes used in this study.</p>                            <h2 scrollto-destination=6640847 id="6640847" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR007C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greilhuber</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Obermayer</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Genome size variation in <em>Cajanus cajan</em> (Fabaceae): a reconsideration</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">1998</div>, vol. <div class="volume">212</div> (pg. <div class="fpage">135</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20size%20variation%20in%20Cajanus%20cajan%20(Fabaceae)%3A%20a%20reconsideration&amp;author=J.%20Greilhuber&amp;author=R.%20Obermayer&amp;publication_year=1998&amp;journal=Plant%20Syst.%20Evol.&amp;volume=212&amp;pages=135-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00985225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00985225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00985225"> </span></div> </div></div></div></div></div><div content-id="DSR007C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genomics-assisted breeding for crop improvement</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">621</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics-assisted%20breeding%20for%20crop%20improvement&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Plant%20Sci.&amp;volume=10&amp;pages=621-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2005.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2005.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2005.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16290213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Fickus</div> <div class="given-names">E.W.</div></div>, et al. </div><div class="article-title">High-throughput SNP discovery and assay development in common bean</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">475</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20assay%20development%20in%20common%20bean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=E.W.%20Fickus&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=475" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20712881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burns</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Newbury</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Ford-Lloyd</div> <div class="given-names">B.V.</div></div>,  <div class="name"><div class="surname">Baggott</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">Development of simple sequence repeat (SSR) markers for the assessment of gene flow and genetic diversity in pigeonpea (<em>Cajanus cajan</em>)</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2001</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">283</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20simple%20sequence%20repeat%20(SSR)%20markers%20for%20the%20assessment%20of%20gene%20flow%20and%20genetic%20diversity%20in%20pigeonpea%20(Cajanus%20cajan)&amp;author=M.J.%20Burns&amp;author=K.J.%20Edwards&amp;author=H.J.%20Newbury&amp;author=B.V.%20Ford-Lloyd&amp;author=C.D.%20Baggott&amp;publication_year=2001&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=1&amp;pages=283-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8278.2001.00109.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8278.2001.00109.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8278.2001.00109.x"> </span></div> </div></div></div></div></div><div content-id="DSR007C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Odeny</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Jayashree</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ferguson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gebhardt</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Development, characterization and utilization of microsatellite markers in pigeonpea</div>, <div class="source">Plant Breed.</div> , <div class="year">2007</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">130</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%2C%20characterization%20and%20utilization%20of%20microsatellite%20markers%20in%20pigeonpea&amp;author=D.A.%20Odeny&amp;author=B.%20Jayashree&amp;author=M.%20Ferguson&amp;author=D.%20Hoisington&amp;author=J.%20Crouch&amp;author=C.%20Gebhardt&amp;publication_year=2007&amp;journal=Plant%20Breed.&amp;volume=126&amp;pages=130-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2007.126.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2007.126.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2007.126.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSR007C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Odeny</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Jayashree</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gebhardt</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">J.</div></div>. </div><div class="article-title">New microsatellite markers for pigeonpea (<em>Cajanus cajan</em> (L.) Millsp.)</div>, <div class="source">BMC Res. Notes</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> pg. <div class="fpage">35</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20microsatellite%20markers%20for%20pigeonpea%20(Cajanus%20cajan%20(L.)%20Millsp.)&amp;author=D.A.%20Odeny&amp;author=B.%20Jayashree&amp;author=C.%20Gebhardt&amp;author=J.%20Crouch&amp;publication_year=2009&amp;journal=BMC%20Res.%20Notes&amp;volume=2&amp;pages=35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-2-35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-2-35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-2-35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19284532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Prathima</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">K.B.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Novel SSR markers for polymorphism detection in pigeonpea (<em>Cajanus</em> spp.)</div>, <div class="source">Plant Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">142</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20SSR%20markers%20for%20polymorphism%20detection%20in%20pigeonpea%20(Cajanus%20spp.)&amp;author=R.K.%20Saxena&amp;author=C.%20Prathima&amp;author=K.B.%20Saxena&amp;author=D.A.%20Hoisington&amp;author=N.K.%20Singh&amp;author=R.K.%20Varshney&amp;publication_year=2010&amp;journal=Plant%20Breed.&amp;volume=129&amp;pages=142-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2010.129.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2010.129.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2010.129.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSR007C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohra</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dubey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">R. K.</div></div>, et al. </div><div class="article-title">Analysis of BAC-end sequences (BESs) and development of BES-SSR markers for genetic mapping and hybrid purity assessment in pigeonpea</div>, <div class="source"><em>(</em>Cajanus spp.<em>)</em>, BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">56</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20BAC-end%20sequences%20(BESs)%20and%20development%20of%20BES-SSR%20markers%20for%20genetic%20mapping%20and%20hybrid%20purity%20assessment%20in%20pigeonpea&amp;author=A.%20Bohra&amp;author=A.%20Dubey&amp;author=R.%20K.%20Saxena&amp;publication_year=2011&amp;journal=(Cajanus%20spp.)%2C%20BMC%20Plant%20Biol.&amp;volume=11&amp;pages=56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-56" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-56" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-56"> </span></div> </div></div></div></div></div><div content-id="DSR007C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rotter</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Camps</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Gene expression profiling in susceptible interaction of grapevine with its fungal pathogen <em>Eutypa lata</em>: extending MapMan ontology for grapevine</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">104</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profiling%20in%20susceptible%20interaction%20of%20grapevine%20with%20its%20fungal%20pathogen%20Eutypa%20lata%3A%20extending%20MapMan%20ontology%20for%20grapevine&amp;author=A.%20Rotter&amp;author=C.%20Camps&amp;author=M.%20Lohse&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19656401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sterky</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bhalerao</div> <div class="given-names">R.R.</div></div>,  <div class="name"><div class="surname">Unneberg</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A <em>Populus</em> EST resource for plant functional genomics</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">13951</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Populus%20EST%20resource%20for%20plant%20functional%20genomics&amp;author=F.%20Sterky&amp;author=R.R.%20Bhalerao&amp;author=P.%20Unneberg&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=13951-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0401641101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0401641101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0401641101"> </span></div> </div></div></div></div></div><div content-id="DSR007C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blanc</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Barakat</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Guyot</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cooke</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Delseny</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Extensive duplication and reshuffling in the <em>Arabidopsis</em> genome</div>, <div class="source">Plant Cell</div> , <div class="year">2000</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1093</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20duplication%20and%20reshuffling%20in%20the%20Arabidopsis%20genome&amp;author=G.%20Blanc&amp;author=A.%20Barakat&amp;author=R.%20Guyot&amp;author=R.%20Cooke&amp;author=M.%20Delseny&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=1093-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.7.1093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.7.1093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.7.1093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Nayak</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Next generation sequencing technologies and their implications for crop genetics and breeding</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">522</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20technologies%20and%20their%20implications%20for%20crop%20genetics%20and%20breeding&amp;author=R.K.%20Varshney&amp;author=S.N.%20Nayak&amp;author=G.D.%20May&amp;author=S.A.%20Jackson&amp;publication_year=2009&amp;journal=Trends%20Biotechnol.&amp;volume=27&amp;pages=522-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2009.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2009.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2009.05.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19679362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmitt</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">T.A.</div></div>,  <div class="name"><div class="surname">Trumpower</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">A rapid and simple method for preparation of RNA from <em>Saccharomyces cerevisiae</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">3091</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20and%20simple%20method%20for%20preparation%20of%20RNA%20from%20Saccharomyces%20cerevisiae&amp;author=M.E.%20Schmitt&amp;author=T.A.%20Brown&amp;author=B.L.%20Trumpower&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=3091-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.10.3091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.10.3091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.10.3091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2190191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Machleder</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Chenchik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Siebert</div> <div class="given-names">P.D.</div></div>. </div><div class="article-title">Reverse transcriptase template switching: a SMART approach for full length cDNA library construction</div>, <div class="source">Biotechniques</div> , <div class="year">2001</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">892</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DSR007C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Goldberg</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Town</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">Sequencing <em>Medicago truncatula</em> expressed sequenced tags using 454 life sciences technology</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">272</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20Medicago%20truncatula%20expressed%20sequenced%20tags%20using%20454%20life%20sciences%20technology&amp;author=F.%20Cheung&amp;author=B.J.%20Haas&amp;author=M.D.%20Goldberg&amp;author=G.D.%20May&amp;author=Y.%20Xiao&amp;author=C.D.%20Town&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=272" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17062153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beckstette</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Homann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Giegerich</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Fast index based algorithms and software for matching position specific scoring matrices</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">389</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20index%20based%20algorithms%20and%20software%20for%20matching%20position%20specific%20scoring%20matrices&amp;author=M.%20Beckstette&amp;author=R.%20Homann&amp;author=R.%20Giegerich&amp;author=S.%20Kurtz&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16930469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldman</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">A codon-based model of nucleotide substitution for protein coding DNA sequences</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1994</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">725</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20codon-based%20model%20of%20nucleotide%20substitution%20for%20protein%20coding%20DNA%20sequences&amp;author=N.%20Goldman&amp;author=Z.%20Yang&amp;publication_year=1994&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=11&amp;pages=725-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7968486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">PAML: a program package for phylogenetic analysis by maximum likelihood</div>, <div class="source">Comput. Appl. Biosci</div> , <div class="year">1997</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">555</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DSR007C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thiel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Michalek</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Exploiting EST databases for the development and characterization of gene derived SSR-markers in barley (<em>Hordeum vulgare</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">411</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20EST%20databases%20for%20the%20development%20and%20characterization%20of%20gene%20derived%20SSR-markers%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=T.%20Thiel&amp;author=W.%20Michalek&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=411-22" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12589540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Krawetz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Misener</div> <div class="given-names">S.</div></div>. </div>, <div class="source">Bioinformatics Methods and Protocols</div> , <div class="year">2000</div><div class="publisher-loc">Totowa, NJ</div><div class="publisher-name">Humana Press</div>(pg. <div class="fpage">365</div>-<div class="lpage">368</div>) <div class="comment"><a class="link link-uri" href="http://fokker.wi.mit.edu/primer3/">http://fokker.wi.mit.edu/primer3/</a></div></div></div></div></div><div content-id="DSR007C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Kingsmore</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Management of high-throughput DNA sequencing projects: Alpheus</div>, <div class="source">J. Comput. Sci. Syst. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">132</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Management%20of%20high-throughput%20DNA%20sequencing%20projects%3A%20Alpheus&amp;author=N.A.%20Miller&amp;author=S.F.%20Kingsmore&amp;author=A.%20Farmer&amp;publication_year=2008&amp;journal=J.%20Comput.%20Sci.%20Syst.%20Biol.&amp;volume=1&amp;pages=132-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20151039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Novaes</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Drost</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Farmerie</div> <div class="given-names">W.G.</div></div>, et al. </div><div class="article-title">High-throughput gene and SNP discovery in <em>Eucalyptus grandis</em>, an uncharacterized genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">312</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20gene%20and%20SNP%20discovery%20in%20Eucalyptus%20grandis%2C%20an%20uncharacterized%20genome&amp;author=G.J.%20Novaes&amp;author=D.R.%20Drost&amp;author=W.G.%20Farmerie&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-312"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18590545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">178</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">C.K.</div></div>. </div><div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1859</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.J.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">G.Z.</div></div>, et al. </div><div class="article-title">QTL mapping of ten agronomic traits on the soybean (<em>Glycine max</em> L. Merr.) genetic map and their association with EST markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1131</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20of%20ten%20agronomic%20traits%20on%20the%20soybean%20(Glycine%20max%20L.%20Merr.)%20genetic%20map%20and%20their%20association%20with%20EST%20markers&amp;author=W.K.%20Zhang&amp;author=Y.J.%20Wang&amp;author=G.Z.%20Luo&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1131-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1527-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1527-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1527-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">He</div> <div class="given-names">C-Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J-S.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">S-Y.</div></div>. </div><div class="article-title">A soybean gene encoding a proline-rich protein is regulated by salicylic acid, an endogenous circadian rhythm and by various stresses</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">1125</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20soybean%20gene%20encoding%20a%20proline-rich%20protein%20is%20regulated%20by%20salicylic%20acid%2C%20an%20endogenous%20circadian%20rhythm%20and%20by%20various%20stresses&amp;author=C-Y.%20He&amp;author=J-S.%20Zhang&amp;author=S-Y.%20Chen&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=1125-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-001-0853-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-001-0853-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-001-0853-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davletova</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schlauch</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Coutu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mittler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The zinc-finger protein Zat12 plays a central role in reactive oxygen and abiotic stress signaling in <em>Arabidopsis</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">139</div> (pg. <div class="fpage">847</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zinc-finger%20protein%20Zat12%20plays%20a%20central%20role%20in%20reactive%20oxygen%20and%20abiotic%20stress%20signaling%20in%20Arabidopsis&amp;author=S.%20Davletova&amp;author=K.%20Schlauch&amp;author=J.%20Coutu&amp;author=R.%20Mittler&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=139&amp;pages=847-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.068254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.068254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.068254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16183833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rizhsky</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Davletova</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Mittler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The zinc finger protein Zat12 is required for cytosolic ascorbate peroxidase 1 expression during oxidative stress in <em>Arabidopsis</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">11736</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zinc%20finger%20protein%20Zat12%20is%20required%20for%20cytosolic%20ascorbate%20peroxidase%201%20expression%20during%20oxidative%20stress%20in%20Arabidopsis&amp;author=L.%20Rizhsky&amp;author=S.%20Davletova&amp;author=H.%20Liang&amp;author=R.%20Mittler&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=11736-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M313350200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M313350200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M313350200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14722088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dixon</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Achnine</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kota</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">C-J.</div></div>,  <div class="name"><div class="surname">Reddy</div> <div class="given-names">M.S.S.</div></div>. </div><div class="article-title">The phenylpropanoid pathway and plant defense—a genomics perspective</div>, <div class="source">Mol. Plant Pathol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">371</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20phenylpropanoid%20pathway%20and%20plant%20defense%E2%80%94a%20genomics%20perspective&amp;author=R.A.%20Dixon&amp;author=L.%20Achnine&amp;author=P.%20Kota&amp;author=C-J.%20Liu&amp;author=M.S.S.%20Reddy&amp;publication_year=2002&amp;journal=Mol.%20Plant%20Pathol.&amp;volume=3&amp;pages=371-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1364-3703.2002.00131.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1364-3703.2002.00131.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1364-3703.2002.00131.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20569344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlagnhaufer</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Arteca</div> <div class="given-names">R.N.</div></div>,  <div class="name"><div class="surname">Pell</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">Sequential expression of two 1-aminocyclopropane-1-carboxylate synthase genes in response to biotic and abiotic stresses in potato (<em>Solanum tuberosum</em> L.) leaves</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">683</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequential%20expression%20of%20two%201-aminocyclopropane-1-carboxylate%20synthase%20genes%20in%20response%20to%20biotic%20and%20abiotic%20stresses%20in%20potato%20(Solanum%20tuberosum%20L.)%20leaves&amp;author=C.D.%20Schlagnhaufer&amp;author=R.N.%20Arteca&amp;author=E.J.%20Pell&amp;publication_year=1997&amp;journal=Plant%20Mol.%20Biol.&amp;volume=35&amp;pages=683-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1005857717196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1005857717196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1005857717196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9426590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manickavelu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kawaura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Oishi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Comparative gene expression analysis of susceptible and resistant near-isogenic lines in common wheat infected by</div>, <div class="source">Puccinia triticina, DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">211</div>-<div class="lpage">222</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20gene%20expression%20analysis%20of%20susceptible%20and%20resistant%20near-isogenic%20lines%20in%20common%20wheat%20infected%20by&amp;author=A.%20Manickavelu&amp;author=K.%20Kawaura&amp;author=K.%20Oishi&amp;publication_year=2010&amp;journal=Puccinia%20triticina%2C%20DNA%20Res.&amp;volume=17&amp;pages=211-222" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq009"> </span></div> </div></div></div></div></div><div content-id="DSR007C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Narina</div> <div class="given-names">S. S.</div></div>,  <div class="name"><div class="surname">Buyyarapu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kottapalli</div> <div class="given-names">K. R.</div></div>, et al. </div><div class="article-title">Generation and analysis of expressed sequence tags (ESTs) for marker development in yam</div>, <div class="source"><em>(</em>Dioscorea alata L.<em>)</em>, BMC Genomics.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">100</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20and%20analysis%20of%20expressed%20sequence%20tags%20(ESTs)%20for%20marker%20development%20in%20yam&amp;author=S.%20S.%20Narina&amp;author=R.%20Buyyarapu&amp;author=K.%20R.%20Kottapalli&amp;publication_year=2011&amp;journal=(Dioscorea%20alata%20L.)%2C%20BMC%20Genomics.&amp;volume=12&amp;pages=100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-100"> </span></div> </div></div></div></div></div><div content-id="DSR007C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raju</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Gnanesh</div> <div class="given-names">B.N.</div></div>,  <div class="name"><div class="surname">Lekha</div> <div class="given-names">P.T.</div></div>, et al. </div><div class="article-title">The first set of EST resource for gene discovery and marker development in pigeonpea (<em>Cajanus cajan</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20set%20of%20EST%20resource%20for%20gene%20discovery%20and%20marker%20development%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=N.L.%20Raju&amp;author=B.N.%20Gnanesh&amp;author=P.T.%20Lekha&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20222972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Brar</div> <div class="given-names">D.S.</div></div>. </div>, <div class="source">Molecular Techniques in Crop Improvement</div> , <div class="year">2010</div>, vol. <div class="volume">vol. 2</div> <div class="publisher-loc">The Netherlands</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">119</div>-<div class="lpage">42</div>)</div></div></div></div><div content-id="DSR007C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eujayl</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Baum</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wolters</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Isolation of EST-derived microsatellite markers for genotyping the A and B genomes of wheat</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">399</div>-<div class="lpage">407</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20EST-derived%20microsatellite%20markers%20for%20genotyping%20the%20A%20and%20B%20genomes%20of%20wheat&amp;author=I.%20Eujayl&amp;author=M.E.%20Sorrells&amp;author=M.%20Baum&amp;author=P.%20Wolters&amp;author=W.%20Powell&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=399-407" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100738" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100738" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100738"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Development of EST-SSR and genomic-SSR markers to assess genetic diversity in <em>Jatropha curcas</em> L.</div>, <div class="source">BMC Res. Notes</div> , <div class="year">2010</div>, vol. <div class="volume">3</div> pg. <div class="fpage">42</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20EST-SSR%20and%20genomic-SSR%20markers%20to%20assess%20genetic%20diversity%20in%20Jatropha%20curcas%20L.&amp;author=M.%20Wen&amp;author=H.%20Wang&amp;author=Z.%20Xia&amp;author=M.%20Zou&amp;author=C.%20Lu&amp;author=W.%20Wang&amp;publication_year=2010&amp;journal=BMC%20Res.%20Notes&amp;volume=3&amp;pages=42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-3-42" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-3-42" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-3-42"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20181259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stein</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Blasiar</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">The genome sequence of <em>Caenorhabditis briggsae</em>: a platform for comparative genomics</div>, <div class="source">PLoS Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">1</div> pg. <div class="fpage">e45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Caenorhabditis%20briggsae%3A%20a%20platform%20for%20comparative%20genomics&amp;author=L.D.%20Stein&amp;author=Z.%20Bao&amp;author=D.%20Blasiar&amp;publication_year=2003&amp;journal=PLoS%20Biol.&amp;volume=1&amp;pages=e45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0000045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0000045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14624247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR007C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kota</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stein</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EST derived single nucleotide polymorphism markers for assembling genetic and physical maps of the barley genome</div>, <div class="source">Func. Integrat. Genom.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">223</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%20derived%20single%20nucleotide%20polymorphism%20markers%20for%20assembling%20genetic%20and%20physical%20maps%20of%20the%20barley%20genome&amp;author=R.%20Kota&amp;author=R.K.%20Varshney&amp;author=M.%20Prasad&amp;author=H.%20Zhang&amp;author=N.%20Stein&amp;author=A.%20Graner&amp;publication_year=2008&amp;journal=Func.%20Integrat.%20Genom.&amp;volume=8&amp;pages=223-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-007-0060-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-007-0060-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-007-0060-9"> </span></div> </div></div></div></div></div><div content-id="DSR007C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR007C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gujaria</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dauthal</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div> <div class="comment">doi:10.1007/s00122-011-1556-1</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=444270&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table7.xls?Expires=2147483647&amp;Signature=rYgc7TpIbtv38O7rtNlqdye4xCJsXPjkw59TiI71pfm6ogdBhA-LXogds9rfLoBFAnuUxwcYGEUc-~2njQwAvqDiUXioeG6oEre2Zul7YIrLZ9mp9-mj4NWUj4ezPcaRBhU6VpDmEiUTTpdep1wFqN0tValNP8ah0ut3VjBEnJZWeOGt20t7-D-Xf05qR8MuTI4~zDJILFuAbv1Z7vDDl4ad~bQKzePc4K0RfnTHqTiZo42oN-hS8kV918Bu~Jql19~jnmGpGRNC1ZMMfNn7H9uT5gKbisSpj0ZHWrISd2BDIYR8V9XobnwSRq-pjxg-c9xLHXhj6ljuV8RW7fTXww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table6.xls?Expires=2147483647&amp;Signature=pRp93l74TWv5qrWcw5Hoxcn54Azj-ksWE-0BxBAL7B-lBfUDr-D80Zph2UqK0PuzM5T4aY0cgR2K7dzGpuop2cuBIWUTGm0pkkzNwyvgsm28sN0mFp6osfESpJFKTlzc37a8BCwB2xkFR6IHRu4sxdNHmYf3bTHeS8VqNBtJYltot8LJz47tlqT0Q7yWeZML82tbqW6lt5hig4WnBwcfWL1o8WX8hACcwKqG8zkxoPtpXdNINuwcYlGJ-dYvVAW1DtrfBd2Z7bDFCj1AX59qMJa9dqS73PNaKKzdVxZVWBuHiqX2C-EDFhkitSw~YED6RkDgLHzlCZ9g3N6z6GqOuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table5.xls?Expires=2147483647&amp;Signature=GofPGC4QGRLzPhkyS4F-J2JTtxIYgxGMBwcuJWBh~RCn1RwVSvvvTSKyEVZ3xlW5Qo-kopEm8PRpJ0z2AT1WTbZk6yerbluLQQnque9im~hHRvFzz8LElP7WJffmGVQGfaF-KhvxOsJu~CA4xpxgGPjPe4DRM29kbONoJLoLLgSJreF2EZQ9OnFgAHLRcbnC4o34N4eEU3lUGgDGySiT-5-TXknxKkhukwZx98e86WudwQhlE7Yr4QSO4gLpON2O70--vijZ2BZ0VFXfolb5ReF950wf1QM7xaP7N0e-U~VGjjAtKAsquIKb4e~eIvzrA2z6VUVzEC47e~R1BCRjwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table4.xls?Expires=2147483647&amp;Signature=q--VlENb6x6INv7yR5g6h9NBBIxacmxpPwdBAosb~lKOR9kNHEuuDsU~h8bPNtD7qzWF5jF6t~A6-7SfUk24QRJItFl1d6U5V-79xCa7YQf4-ZB4cAUbYgKzucYzXGt94DqQ~IChkcvx5ao0fcj1fdYVNYXHyZdwYWnIKBF5gSHx4JvleFV8E6Rp9p1fFnTPHT9eKEyyBu~6I0BAIKxGZlkDA9RwGNyVrhWxUWovcNRUcPxLaRJQPgh5lqvMIFwL7LLvfU-Fi-qZHWcWbBxNDVy7RwMDnAkZr0oKQZiusw1GgdKFLhdZ~Oc95rhdmIfy6uvHN6eI3bH9P2UwEbkloQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table3.xls?Expires=2147483647&amp;Signature=bcQ8uVZ9rvc9N8fkvW3etgwjVu0IvRSV8BYXnxZmmLBhYDaruHNZj9N22XSAkBXHvA8aX9XzXtz~55geVvx75UOgmS3L9V3gxjZdx5xrrmPak0itUCl8r84y696O5pE69zsVToNfi~Ofi99ecDPVPR4wr8sjIkAvOQiFi0QTDG8n4t7ryu59lhTHEadpD0UcxAgIBTXHL5h3CSTuUVKL5Aq~ULmBbcYjfpqQXhFRCX0vVtnTmCPTOteuMt3Y7gdOJsrDYOOKxc-RiP6P9eb6Td4og-9NQspUXjpDahp6vUq89nOUvxUsTBQRVBn~aK8DdXZVBsO8TbmTlFwmPGAWXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table2.xls?Expires=2147483647&amp;Signature=t5AsaxB41pCe3zDv3dCSLNO5T4ExWWi7ba4pHV85-NtDIEAvkZe8UURxfsFRQEI5i07iy~YFxPOipkcOaUcqVWLEZgSD0BJ-wTGEL8TeWWj89CSz4XiBHGG1tIoKJw1-yN1mq1SRTkAt9YZaTGpGoknNDO-FRcn4guoMrZe2kFimEV~8U7NmTkeGhDvJDhXO2Iu4PIoOV1rKJxtgBuTfZ86Ez6LuQkKSBoKBziaBiQdxSN84Us59gdCUZZDsNTb0dgUNR-~dY3NLtQbh-q8crDjeaGMm6Hi9LmZJ4rOS7GExW4jC8PQAOhLEXI3XEhg7Y6upfeB~b-LVFMiuQzISww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_table1.xls?Expires=2147483647&amp;Signature=Zcnm6qBJRXSkMewSXwWmMtWppLEzurRRf3WL9SXK~VxZrITHiY8F0Hq0oe2fMD5W3t3YHgyXMNVzt0HebcNWFC6ZDqsmQvd4ITrA4EdXZfi6Q5Hiw~vYZcsyl78zrgLgxSEA45wsOzgint51nB2yWIGbXKgAajUd9nDoP3BtHN0TtpWQvWOGlFpcZRRyiTODM07-idf-Hz5Swqs9gBQkUBcZHe7O3iJA~ChdxFfCNNFA9vIuviCK56YORmvn~MaNwqXTj4zSQc7BXqIWPQzrxEU7XWD4G~tmXyMBogct8q5HGecDMwXgZFmjN1-b8HGH7T1EMPhmsnURdozSlKgQjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr007/2/dsr007-dsr007_fig1.png?Expires=2147483647&amp;Signature=Z7FxX6lWIXG9siS1pJtRMhd3-SAY~aXUl0DzcereNDmWpoHCk3r4htEDDle3KgH8sD-dw6YuZtlG4SETRKTRMqjYVNGpto9NrHO2aBMnjTN-7Ku0QyG3rTSmZWPGxFLEA5miQM3Ts-wSWRPjMzFfZimu0U30Km~-Pgp~2UdHgfMI4dhsL9QTvEv~yHZ-pyfnZ0VwA4kLR2k6aP1mzUnq~vrLfyoXgHYfVfR2GCPQgtqrMAXorC3A1uIqDkv78K8-lPHY6e0TwdBKUDKQ-OaUZTIUopLRNXN2WbeVevMWwmne9jXofm~rCfZ~y5o-XJpcoh2AMaZrHsxTN6kJ0tP6Fw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">330</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">56</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="444270"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f3%2f153%2f444270">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000292070400002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (56)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F3%2F153%2F444270" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/444270'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Defining the Transcriptome Assembly and Its Use for Genome Dynamics and Transcriptome Profiling Studies in Pigeonpea (Cajanus cajan L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f153%2f444270">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

