

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/5/441/2236160","@type":"ScholarlyArticle","name":"Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria","datePublished":"2016-07-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2016-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw027","keywords":["codon usage bias","optimal codon","genome composition","high expression genes","bacteria"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/5/441/2236160","author":[{"name":"Satapathy, Siddhartha Sankar","affiliation":"Department of Computer Science and Engineering, Tezpur University, Napaam, Tezpur 784028, Assam, India","@type":"Person"},{"name":"Powdel, Bhesh Raj","affiliation":"Department of Statistics, Darrang College, Tezpur 784001, Assam, India","@type":"Person"},{"name":"Buragohain, Alak Kumar","affiliation":"Department of Molecular Biology and Biotechnology, Tezpur University, Napaam, Tezpur 784028, Assam, India  Office of the Vice-Chancellor, Dibrugarh University, Dibrugarh 786004, Assam, India","@type":"Person"},{"name":"Ray, Suvendra Kumar","affiliation":"Department of Molecular Biology and Biotechnology, Tezpur University, Napaam, Tezpur 784028, Assam, India","@type":"Person"}],"description":"Abstract.  The different triplets encoding the same amino acid, termed as synonymous codons, are not equally abundant in a genome. Factors such as G + C% and tR","pageStart":"441","pageEnd":"449","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria" />
    <meta property="og:description" content="Abstract.  The different triplets encoding the same amino acid, termed as synonymous codons, are not equally abundant in a genome. Factors such as G + C% and tR" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/5/441/2236160" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-07-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Satapathy, Siddhartha Sankar" /><meta name="citation_author_institution" content="Department of Computer Science and Engineering, Tezpur University, Napaam, Tezpur 784028, Assam, India" /><meta name="citation_author" content="Powdel, Bhesh Raj" /><meta name="citation_author_institution" content="Department of Statistics, Darrang College, Tezpur 784001, Assam, India" /><meta name="citation_author" content="Buragohain, Alak Kumar" /><meta name="citation_author_institution" content="Department of Molecular Biology and Biotechnology, Tezpur University, Napaam, Tezpur 784028, Assam, India" /><meta name="citation_author_institution" content="Office of the Vice-Chancellor, Dibrugarh University, Dibrugarh 786004, Assam, India" /><meta name="citation_author" content="Ray, Suvendra Kumar" /><meta name="citation_author_institution" content="Department of Molecular Biology and Biotechnology, Tezpur University, Napaam, Tezpur 784028, Assam, India" /><meta name="citation_title" content="Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria" /><meta name="citation_firstpage" content="441" /><meta name="citation_lastpage" content="449" /><meta name="citation_doi" content="10.1093/dnares/dsw027" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2016/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The codon Adaptation Index – a measure of directional synonymous codon usage bias, and its potential applications,; citation_author=Sharp  P.M.; citation_author=Li  W.H.;  citation_journal_title=Nucleic Acids Res;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=Forces that influence the evolution of codon bias,; citation_author=Sharp  P.M.; citation_author=Emery  L.R.; citation_author=Zeng  K.;  citation_journal_title=Philos. Trans. R. Soc. Lond. B. Biol. Sci;  citation_year=2010;  citation_volume=365;  citation_pages=1203-12; " /><meta name="citation_reference" content="citation_title=The selection-mutation-drift theory of synonymous codon usage,; citation_author=Bulmer  M.;  citation_journal_title=Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=897-907; " /><meta name="citation_reference" content="citation_title=The influence of anticodon-codon interactions and modified bases on codon usage bias in bacteria,; citation_author=Ran  W.; citation_author=Higgs  P.G.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2010;  citation_volume=27;  citation_pages=2129-40; " /><meta name="citation_reference" content="citation_title=Codon bias as a means to fine-tune gene expression,; citation_author=Quax  T.E.; citation_author=Claassens  N.J.; citation_author=Söll  D.; citation_author=van der Oost  J.;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=149-61; " /><meta name="citation_reference" content="citation_title=Codon usage inﬂuences the local rate of translation elongation to regulate co-translational protein folding,; citation_author=Yu  C.H.; citation_author=Dang  Y.; citation_author=Zhou  Z.;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Synonymous codons are not same with respect to the speed of translation elongation; citation_author=Ray  S.K.; citation_author=Goswami  I.;  citation_journal_title=Curr. Sci.;  citation_year=2016;  citation_volume=110;  citation_pages=1612-4; " /><meta name="citation_reference" content="citation_title=General rules for optimal codon choice,; citation_author=Hershberg  R.; citation_author=Petrov  D.A.;  citation_journal_title=PLoS Genet;  citation_year=2009;  citation_volume=5;  citation_pages=e1000556" /><meta name="citation_reference" content="citation_title=Preferential codon usage in prokaryotic genes: the optimal codon-anticodon interaction energy and the selective codon usage in efficiently expressed genes,; citation_author=Grosjean  H.; citation_author=Fiers  W.;  citation_journal_title=Gene;  citation_year=1982;  citation_volume=18;  citation_pages=199-209; " /><meta name="citation_reference" content="citation_title=Deciphering synonymous codons in the three domains of life: co-evolution with specific tRNA modification enzymes,; citation_author=Grosjean  H.; citation_author=de Crécy-Lagard  V.; citation_author=Marck  C.;  citation_journal_title=FEBS Lett;  citation_year=2010;  citation_volume=584;  citation_pages=252-64; " /><meta name="citation_reference" content="citation_title=A role for tRNA modifications in genome structure and codon usage,; citation_author=Novoa  E.M.; citation_author=Pavon-Eternod  M.; citation_author=Pan  T.; citation_author=de Pouplana  L.;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=149;  citation_pages=202-13; " /><meta name="citation_reference" content="citation_title=Optimization of codon translation rates via tRNA modifications maintains proteome integrity,; citation_author=Nedialkova  D.D.; citation_author=Leidel  S.A.;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=161;  citation_pages=1606-18; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system,; citation_author=Ikemura  T.;  citation_journal_title=J. Mol. Biol;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms,; citation_author=Ikemura  T.;  citation_journal_title=Mol. Biol. Evol;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon usage bias from tRNA’s point of view, redundancy, specialization, and efficient decoding for translation optimization,; citation_author=Rocha  E.P.;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=2279-86; " /><meta name="citation_reference" content="citation_title=Coevolution of codon usage and tRNA genes leads to alternative stable states of biased codon usage,; citation_author=Higgs  P.G.; citation_author=Ran  W.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=2279-91; " /><meta name="citation_reference" content="citation_title=Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift,; citation_author=Shah  P.; citation_author=Gilchrist  M.A.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2011;  citation_volume=108;  citation_pages=10231-6; " /><meta name="citation_reference" content="citation_title=Wide spread selection for local RNA secondary structure in coding regions of bacterial genes,; citation_author=Katz  L.; citation_author=Burge  C.B.;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2042-51; " /><meta name="citation_reference" content="citation_title=A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes,; citation_author=Gu  W.; citation_author=Zhou  T.; citation_author=Wilke  C.O.;  citation_journal_title=PLoS Comput. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=e1000664" /><meta name="citation_reference" content="citation_title=Strong association between mRNA folding strength and protein abundance in S. cerevisiae,; citation_author=Zur  H.; citation_author=Tuller  T.;  citation_journal_title=EMBO Rep;  citation_year=2012;  citation_volume=13;  citation_pages=272-7; " /><meta name="citation_reference" content="citation_title=Differential requirements for mRNA folding partially explain why highly expressed proteins evolve slowly,; citation_author=Park  C.; citation_author=Chen  X.; citation_author=Yang  J.R.; citation_author=Zhang  J.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2013;  citation_volume=110;  citation_pages=E678-86; " /><meta name="citation_reference" content="citation_title=Codon optimality is a major determinant of mRNA stability,; citation_author=Presnyak  V.; citation_author=Alhusaini  N.; citation_author=Chen  Y.H.;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=160;  citation_pages=1111-24; " /><meta name="citation_reference" content="citation_title=Nonoptimal codon usage inﬂuences protein structure in intrinsically disordered regions,; citation_author=Zhou  M.; citation_author=Wang  T.; citation_author=Fu  J.; citation_author=Xiao  G.; citation_author=Liu  Y.;  citation_journal_title=Mol. Microbiol;  citation_year=2015;  citation_volume=97;  citation_pages=974-87; " /><meta name="citation_reference" content="citation_title=Co-translational protein folding is revealing the selective use of synonymous codons along the coding sequence of a low expression gene,; citation_author=Ray  S.K.; citation_author=Baruah  V.J.; citation_author=Satapathy  S.S.; citation_author=Banerjee  R.;  citation_journal_title=J. Genet;  citation_year=2014;  citation_volume=93;  citation_pages=613-7; " /><meta name="citation_reference" content="citation_title=Accurate prediction of cellular co-translational folding indicates proteins can switch from post- to co-translational folding; citation_author=Nissley  D.A.; citation_author=Sharma  A.K.; citation_author=Ahmed  N.;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=7;  citation_pages=10341" /><meta name="citation_reference" content="citation_title=Synonymous codons direct cotranslational folding toward different protein conformations,; citation_author=Buhr  F.; citation_author=Jha  S.; citation_author=Thommen  M.;  citation_journal_title=Mol. Cell;  citation_year=2016;  citation_volume=61;  citation_pages=341-51; " /><meta name="citation_reference" content="citation_title=Variation in the strength of selected codon usage bias among bacteria,; citation_author=Sharp  P.M.; citation_author=Bailes  E.; citation_author=Grocock  R.J.; citation_author=Peden  J.F.; citation_author=Sockett  R.E.;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=1141-53; " /><meta name="citation_reference" content="citation_title=Transfer RNA gene numbers may not be completely responsible for the codon usage bias in asparagine, isoleucine, phenylalanine and tyrosine in the high expression genes in bacteria,; citation_author=Satapathy  S.S.; citation_author=Dutta  M.; citation_author=Buragohain  A.K.; citation_author=Ray  S.K.;  citation_journal_title=J. Mol. Evol;  citation_year=2012;  citation_volume=75;  citation_pages=34-42; " /><meta name="citation_reference" content="citation_title=Codon usage bias in prokaryotic pyrimidine-ending codons is associated with the degeneracy of the encoded amino acids,; citation_author=Wald  N.; citation_author=Alroy  M.; citation_author=Botzman  M.; citation_author=Margalit  H.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=7074-83; " /><meta name="citation_reference" content="citation_title=Selection on GGU and CGU codons in the high expression genes in bacteria,; citation_author=Satapathy  S.S.; citation_author=Powdel  B.R.; citation_author=Dutta  M.; citation_author=Buragohain  A.K.; citation_author=Ray  S.K.;  citation_journal_title=J. Mol. Evol;  citation_year=2014;  citation_volume=78;  citation_pages=13-23; " /><meta name="citation_reference" content="citation_title=Optimal codon identities in bacteria: implications from the conflicting results of two different methods,; citation_author=Wang  B.; citation_author=Shao  Z.-Q.; citation_author=Xu  Y.;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e22714" /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence,; citation_author=Lowe  T.M.; citation_author=Eddy  S.R.;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms,; citation_author=Sharp  P.M.; citation_author=Li  W.-H.;  citation_journal_title=J. Mol. Evol;  citation_year=1986;  citation_volume=24;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=Evolution of optimal codon choices in the family Enterobacteriaceae,; citation_author=Iriarte  A.; citation_author=Baraibar  J.D.; citation_author=Romero  H.; citation_author=Castro-Sowinski  S.; citation_author=Musto  H.;  citation_journal_title=Microbiology;  citation_year=2013;  citation_volume=159;  citation_pages=555-64; " /><meta name="citation_reference" content="citation_title=Translational selection and yeast proteome evolution,; citation_author=Akashi  H.;  citation_journal_title=Genetics;  citation_year=2002;  citation_volume=164;  citation_pages=1291-303; " /><meta name="citation_reference" content="citation_title=Effect of correlated tRAN abundances on translation errors and evolution of codon usage bias,; citation_author=Shah  P.; citation_author=Gilchrist  M.A.;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001128" /><meta name="citation_reference" content="citation_title=Measuring and detecting molecular adaptation in codon usage against non-sense errors during protein translation,; citation_author=Gilchrist  M.A.; citation_author=Shah  P.; citation_author=Zaretzki  R.;  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=183;  citation_pages=1493-505; " /><meta name="citation_reference" content="citation_title=Codon usage determines the translation rate in Escherichia coli,; citation_author=Sørensen  M.A.; citation_author=Kurland  C.G.; citation_author=Pedersen  S.;  citation_journal_title=J. Mol. Biol;  citation_year=1989;  citation_volume=207;  citation_pages=365-77; " /><meta name="citation_reference" content="citation_title=Absolute in vivo translation rates of individual codons in Escherichia coli the two glutamic acid codons, GAA and GAG are translated with threefold differences in rate,; citation_author=Sørensen  M.A.; citation_author=Pedersen  S.;  citation_journal_title=J. Mol. Biol;  citation_year=1991;  citation_volume=222;  citation_pages=265-80; " /><meta name="citation_reference" content="citation_title=Evidences indicating the involvement of selection mechanisms for the occurrence of C34 anticodon in bacteria; citation_author=Prajapati  V.K.; citation_author=Satapathy  S.S.; citation_author=Satish Kumar  M.V.; citation_author=Buragohain  A.K.; citation_author=Ray  S.K.;  citation_journal_title=J. Cell Sci. Mol. Biol;  citation_year=2015;  citation_volume=2;  citation_pages=112" /><meta name="citation_reference" content="citation_title=The modification of the wobble base of tRNAGlu modulates the translation rate of glutamic acid codon in vivo,; citation_author=Krüger  M.K.; citation_author=Pedersen  S.; citation_author=Hagervall  T.G.; citation_author=Sørensen  M.A.;  citation_journal_title=J. Mol. Biol;  citation_year=1998;  citation_volume=284;  citation_pages=621-31; " /><meta name="citation_reference" content="citation_title=tRNA’s modifications bring order to gene expression,; citation_author=Gustilo  E.M.; citation_author=Vendeix  F.A.P.; citation_author=Agris  P.F.;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2008;  citation_volume=11;  citation_pages=134-40; " /><meta name="citation_reference" content="citation_title=Convergent evolution of AUA decoding in bacteria and archaea,; citation_author=Suzuki  T.; citation_author=Numata  T.;  citation_journal_title=RNA Biol;  citation_year=2014;  citation_volume=11;  citation_pages=1586-96; " /><meta name="citation_reference" content="citation_title=Constraint on dinucleotides by codon usage bias in bacterial genomes,; citation_author=Satapathy  S.S.; citation_author=Powdel  B.R.; citation_author=Dutta  M.; citation_author=Buragohain  A.K.; citation_author=Ray  S.K.;  citation_journal_title=Gene;  citation_year=2014;  citation_volume=536;  citation_pages=18-28; " /><meta name="citation_reference" content="citation_title=tRNA imbalance promotes -1 frameshifting via near-cognate decoding; citation_author=O’Çonnor  M.;  citation_journal_title=J. Mol. Biol;  citation_year=1998;  citation_volume=279;  citation_pages=727-36; " /><meta name="citation_reference" content="citation_title=Imbalance of tRNAPro isoacceptors induces +1 frameshifting at near-cognate codons; citation_author=O’Çonnor  M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=759-65; " /><meta name="citation_reference" content="citation_title=Synonymous codons, ribosome speed, and eukaryotic gene expression regulation,; citation_author=Tarrant  D.; citation_author=von der Haar  T.;  citation_journal_title=Cell Mol. Life Sci;  citation_year=2014;  citation_volume=71;  citation_pages=4195-206; " /><meta name="citation_reference" content="citation_title=The replication-related organization of bacterial genomes,; citation_author=Rocha  E.P.;  citation_journal_title=Microbiology;  citation_year=2004;  citation_volume=150;  citation_pages=1609-27; " /><meta name="citation_reference" content="citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution,; citation_author=Muto  A.; citation_author=Osawa  S.;  citation_journal_title=Proc. Natl Acad. Sci. U. S. A;  citation_year=1987;  citation_volume=84;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=Codon usage between genomes is constrained by genome wide mutational processes,; citation_author=Chen  S.L.; citation_author=Lee  W.; citation_author=Hotts  A.K.; citation_author=Shapiro  L.; citation_author=McAdams  H.H.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2004;  citation_volume=101;  citation_pages=3480-5; " /><meta name="citation_reference" content="citation_title=A general model of codon bias due to GC mutational bias,; citation_author=Palidwor  G.A.; citation_author=Perkins  T.J.; citation_author=Xia  X.;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e13431" /><meta name="citation_reference" content="citation_title=Translational misreading: a tRNA modification counteracts a + 2 ribosomal frameshift,; citation_author=Brégeon  D.; citation_author=Colot  V.; citation_author=Radman  M.; citation_author=Taddei  F.;  citation_journal_title=Genes Dev;  citation_year=2001;  citation_volume=15;  citation_pages=2295-306; " /><meta name="citation_reference" content="citation_title=Bringing order to translation: the contributions of transfer RNA anticodon-domain modifications,; citation_author=Agris  P.F.;  citation_journal_title=EMBO Rep;  citation_year=2008;  citation_volume=9;  citation_pages=629-35; " /><meta name="citation_reference" content="citation_title=Modification of the wobble uridine in bacterial and mitochondrial tRNAs reading NNA/NNG triplets of 2-codon boxes,; citation_author=Armengod  M.E.; citation_author=Meseguer  S.; citation_author=Villarroya  M.;  citation_journal_title=RNA Biol;  citation_year=2014;  citation_volume=11;  citation_pages=1495-507; " /><meta name="citation_reference" content="citation_title=Post-transcriptional modifications to tRNA – a response to the genetic code degeneracy,; citation_author=Hou  Y.-M.; citation_author=Gamper  H.; citation_author=Yang  W.;  citation_journal_title=RNA;  citation_year=2015;  citation_volume=21;  citation_pages=642-44; " /><meta name="citation_reference" content="citation_title=Comparative DNA analysis across diverse genomes,; citation_author=Karlin  S.; citation_author=Campbell  A.M.; citation_author=Mrázek  J.;  citation_journal_title=Annu. Rev. Genet;  citation_year=1998;  citation_volume=32;  citation_pages=185-225; " /><meta name="citation_reference" content="citation_title=Translational selection is ubiquitous in prokaryotes,; citation_author=Supek  F.; citation_author=Škunca  N.; citation_author=Repar  J.; citation_author=Vlahoviček  K.; citation_author=Šmuc  T.;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001004" /><meta name="citation_reference" content="citation_title=tRNA genes rapidly change in evolution to meet novel translational demands,; citation_author=Yona  A.H.; citation_author=Bloom-Ackermann  Z.; citation_author=Frumkin  I.;  citation_journal_title=eLife;  citation_year=2013;  citation_volume=2;  citation_pages=e01339" /><meta name="citation_reference" content="citation_title=Codon-biased translation can be regulated by wobble-base tRNA modification systems during cellular stress responses,; citation_author=Endres  L.; citation_author=Dedon  P.C.; citation_author=Begley  T.J.;  citation_journal_title=RNA Biol;  citation_year=2015;  citation_volume=12;  citation_pages=603-14; " /><meta name="citation_reference" content="citation_title=Cotranslational protein folding on the ribosome monitored in real time,; citation_author=Holtkamp  W.; citation_author=Kokic  G.; citation_author=Jäger  M.; citation_author=Mittelstät  J.; citation_author=Komar  A.; citation_author=Rodnina  M.V.;  citation_journal_title=Science;  citation_year=2015;  citation_volume=350;  citation_pages=1104-7; " /><meta name="citation_reference" content="citation_title=Heterologous protein expression is enhanced by harmonizing the codon usage frequencies of the target gene with those of the expression hosts,; citation_author=Angov  E.; citation_author=Hiller  C.J.; citation_author=Kincaid  R.L.; citation_author=Lyon  J.A.;  citation_journal_title=PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e2189" /><meta name="citation_reference" content="citation_title=Codon usage: nature’s roadmap to expression and folding of proteins,; citation_author=Angov  E.;  citation_journal_title=Biotechnol. J;  citation_year=2011;  citation_volume=6;  citation_pages=650-9; " /><meta name="citation_reference" content="citation_title=Comparative analysis of codon usage bias in Crenarchaea and Euryarchaea genome reveals differential preference of synonymous codons to encode highly expressed ribosomal and RNA polymerase proteins; citation_author=Baruah  V.J.; citation_author=Satapathy  S.S.; citation_author=Powdel  B.R.; citation_author=Konwarh  R.; citation_author=Buragohain  A.K.; citation_author=Ray  S.K.;  citation_journal_title=J. Genet;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=Codon usage bias is not significantly different between the high and the low expression genes in human; citation_author=Satapathy  S.S.; citation_author=Ray  S.K.; citation_author=Sahoo  A.K.; citation_author=Begum  T.; citation_author=Ghosh  T.C.;  citation_journal_title=Int. J. Mol. Genet. Gene Ther;  citation_year=2015;  citation_volume=1;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Synonymous codon bias and functional constraint on GC3-related DNA backbone dynamics in the prokaryotic nucleoid,; citation_author=Babbitt  G.A.; citation_author=Alawad  M.A.; citation_author=Schulze  K.V.; citation_author=Hudson  A.O.;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=10915-26; " /><meta name="citation_reference" content="citation_title=Codon choice and gene expression: synonymous codons differ in their ability to direct aminoacylated-transfer RNA binding to ribosomes in vitro,; citation_author=Thomas  L.K.; citation_author=Dix  D.B.; citation_author=Thompson  R.C.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1988;  citation_volume=85;  citation_pages=4242-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="description" content="Abstract.  The different triplets encoding the same amino acid, termed as synonymous codons, are not equally abundant in a genome. Factors such as G + C% and tR" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/5/441/2236160" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/5/441/2236160" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/5/441/2236160");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw027';
    </script>
<title>
	Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2236160" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZXCATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0YXBhdGh5LCBTaWRkaGFydGhhIFNhbmthciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0ZXIgU2NpZW5jZSBhbmQgRW5naW5lZXJpbmcsIFRlenB1ciBVbml2ZXJzaXR5LCBOYXBhYW0sIFRlenB1ciA3ODQwMjgsIEFzc2FtLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBvd2RlbCwgQmhlc2ggUmFqIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBTdGF0aXN0aWNzLCBEYXJyYW5nIENvbGxlZ2UsIFRlenB1ciA3ODQwMDEsIEFzc2FtLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJ1cmFnb2hhaW4sIEFsYWsgS3VtYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBCaW9sb2d5IGFuZCBCaW90ZWNobm9sb2d5LCBUZXpwdXIgVW5pdmVyc2l0eSwgTmFwYWFtLCBUZXpwdXIgNzg0MDI4LCBBc3NhbSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPZmZpY2Ugb2YgdGhlIFZpY2UtQ2hhbmNlbGxvciwgRGlicnVnYXJoIFVuaXZlcnNpdHksIERpYnJ1Z2FyaCA3ODYwMDQsIEFzc2FtLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJheSwgU3V2ZW5kcmEgS3VtYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBCaW9sb2d5IGFuZCBCaW90ZWNobm9sb2d5LCBUZXpwdXIgVW5pdmVyc2l0eSwgTmFwYWFtLCBUZXpwdXIgNzg0MDI4LCBBc3NhbSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRpc2NyZXBhbmN5IGFtb25nIHRoZSBzeW5vbnltb3VzIGNvZG9ucyB3aXRoIHJlc3BlY3QgdG8gdGhlaXIgc2VsZWN0aW9uIGFzIG9wdGltYWwgY29kb24gaW4gYmFjdGVyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvZG9uIEFkYXB0YXRpb24gSW5kZXgg4oCTIGEgbWVhc3VyZSBvZiBkaXJlY3Rpb25hbCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMsIGFuZCBpdHMgcG90ZW50aWFsIGFwcGxpY2F0aW9ucyw7IGNpdGF0aW9uX2F1dGhvcj1TaGFycCAgUC5NLjsgY2l0YXRpb25fYXV0aG9yPUxpICBXLkguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEyODEtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvcmNlcyB0aGF0IGluZmx1ZW5jZSB0aGUgZXZvbHV0aW9uIG9mIGNvZG9uIGJpYXMsOyBjaXRhdGlvbl9hdXRob3I9U2hhcnAgIFAuTS47IGNpdGF0aW9uX2F1dGhvcj1FbWVyeSAgTC5SLjsgY2l0YXRpb25fYXV0aG9yPVplbmcgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaGlsb3MuIFRyYW5zLiBSLiBTb2MuIExvbmQuIEIuIEJpb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjU7ICBjaXRhdGlvbl9wYWdlcz0xMjAzLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VsZWN0aW9uLW11dGF0aW9uLWRyaWZ0IHRoZW9yeSBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlLDsgY2l0YXRpb25fYXV0aG9yPUJ1bG1lciAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTEyOTsgIGNpdGF0aW9uX3BhZ2VzPTg5Ny05MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbmZsdWVuY2Ugb2YgYW50aWNvZG9uLWNvZG9uIGludGVyYWN0aW9ucyBhbmQgbW9kaWZpZWQgYmFzZXMgb24gY29kb24gdXNhZ2UgYmlhcyBpbiBiYWN0ZXJpYSw7IGNpdGF0aW9uX2F1dGhvcj1SYW4gIFcuOyBjaXRhdGlvbl9hdXRob3I9SGlnZ3MgIFAuRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIxMjktNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGJpYXMgYXMgYSBtZWFucyB0byBmaW5lLXR1bmUgZ2VuZSBleHByZXNzaW9uLDsgY2l0YXRpb25fYXV0aG9yPVF1YXggIFQuRS47IGNpdGF0aW9uX2F1dGhvcj1DbGFhc3NlbnMgIE4uSi47IGNpdGF0aW9uX2F1dGhvcj1Tw7ZsbCAgRC47IGNpdGF0aW9uX2F1dGhvcj12YW4gZGVyIE9vc3QgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xNDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGlu76yCdWVuY2VzIHRoZSBsb2NhbCByYXRlIG9mIHRyYW5zbGF0aW9uIGVsb25nYXRpb24gdG8gcmVndWxhdGUgY28tdHJhbnNsYXRpb25hbCBwcm90ZWluIGZvbGRpbmcsOyBjaXRhdGlvbl9hdXRob3I9WXUgIEMuSC47IGNpdGF0aW9uX2F1dGhvcj1EYW5nICBZLjsgY2l0YXRpb25fYXV0aG9yPVpob3UgIFouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9ucyBhcmUgbm90IHNhbWUgd2l0aCByZXNwZWN0IHRvIHRoZSBzcGVlZCBvZiB0cmFuc2xhdGlvbiBlbG9uZ2F0aW9uOyBjaXRhdGlvbl9hdXRob3I9UmF5ICBTLksuOyBjaXRhdGlvbl9hdXRob3I9R29zd2FtaSAgSS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MTYxMi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmFsIHJ1bGVzIGZvciBvcHRpbWFsIGNvZG9uIGNob2ljZSw7IGNpdGF0aW9uX2F1dGhvcj1IZXJzaGJlcmcgIFIuOyBjaXRhdGlvbl9hdXRob3I9UGV0cm92ICBELkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDU1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWZlcmVudGlhbCBjb2RvbiB1c2FnZSBpbiBwcm9rYXJ5b3RpYyBnZW5lczogdGhlIG9wdGltYWwgY29kb24tYW50aWNvZG9uIGludGVyYWN0aW9uIGVuZXJneSBhbmQgdGhlIHNlbGVjdGl2ZSBjb2RvbiB1c2FnZSBpbiBlZmZpY2llbnRseSBleHByZXNzZWQgZ2VuZXMsOyBjaXRhdGlvbl9hdXRob3I9R3Jvc2plYW4gIEguOyBjaXRhdGlvbl9hdXRob3I9RmllcnMgIFcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTk5LTIwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVjaXBoZXJpbmcgc3lub255bW91cyBjb2RvbnMgaW4gdGhlIHRocmVlIGRvbWFpbnMgb2YgbGlmZTogY28tZXZvbHV0aW9uIHdpdGggc3BlY2lmaWMgdFJOQSBtb2RpZmljYXRpb24gZW56eW1lcyw7IGNpdGF0aW9uX2F1dGhvcj1Hcm9zamVhbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1kZSBDcsOpY3ktTGFnYXJkICBWLjsgY2l0YXRpb25fYXV0aG9yPU1hcmNrICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVCUyBMZXR0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU4NDsgIGNpdGF0aW9uX3BhZ2VzPTI1Mi02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2xlIGZvciB0Uk5BIG1vZGlmaWNhdGlvbnMgaW4gZ2Vub21lIHN0cnVjdHVyZSBhbmQgY29kb24gdXNhZ2UsOyBjaXRhdGlvbl9hdXRob3I9Tm92b2EgIEUuTS47IGNpdGF0aW9uX2F1dGhvcj1QYXZvbi1FdGVybm9kICBNLjsgY2l0YXRpb25fYXV0aG9yPVBhbiAgVC47IGNpdGF0aW9uX2F1dGhvcj1kZSBQb3VwbGFuYSAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTQ5OyAgY2l0YXRpb25fcGFnZXM9MjAyLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6YXRpb24gb2YgY29kb24gdHJhbnNsYXRpb24gcmF0ZXMgdmlhIHRSTkEgbW9kaWZpY2F0aW9ucyBtYWludGFpbnMgcHJvdGVvbWUgaW50ZWdyaXR5LDsgY2l0YXRpb25fYXV0aG9yPU5lZGlhbGtvdmEgIEQuRC47IGNpdGF0aW9uX2F1dGhvcj1MZWlkZWwgIFMuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MTYwNi0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb24gYmV0d2VlbiB0aGUgYWJ1bmRhbmNlIG9mIEVzY2hlcmljaGlhIGNvbGkgdHJhbnNmZXIgUk5BcyBhbmQgdGhlIG9jY3VycmVuY2Ugb2YgdGhlIHJlc3BlY3RpdmUgY29kb25zIGluIGl0cyBwcm90ZWluIGdlbmVzOiBhIHByb3Bvc2FsIGZvciBhIHN5bm9ueW1vdXMgY29kb24gY2hvaWNlIHRoYXQgaXMgb3B0aW1hbCBmb3IgdGhlIEUuIGNvbGkgdHJhbnNsYXRpb25hbCBzeXN0ZW0sOyBjaXRhdGlvbl9hdXRob3I9SWtlbXVyYSAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0zODktNDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBhbmQgdFJOQSBjb250ZW50IGluIHVuaWNlbGx1bGFyIGFuZCBtdWx0aWNlbGx1bGFyIG9yZ2FuaXNtcyw7IGNpdGF0aW9uX2F1dGhvcj1Ja2VtdXJhICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYmlhcyBmcm9tIHRSTkHigJlzIHBvaW50IG9mIHZpZXcsIHJlZHVuZGFuY3ksIHNwZWNpYWxpemF0aW9uLCBhbmQgZWZmaWNpZW50IGRlY29kaW5nIGZvciB0cmFuc2xhdGlvbiBvcHRpbWl6YXRpb24sOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yMjc5LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2V2b2x1dGlvbiBvZiBjb2RvbiB1c2FnZSBhbmQgdFJOQSBnZW5lcyBsZWFkcyB0byBhbHRlcm5hdGl2ZSBzdGFibGUgc3RhdGVzIG9mIGJpYXNlZCBjb2RvbiB1c2FnZSw7IGNpdGF0aW9uX2F1dGhvcj1IaWdncyAgUC5HLjsgY2l0YXRpb25fYXV0aG9yPVJhbiAgVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIyNzktOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxhaW5pbmcgY29tcGxleCBjb2RvbiB1c2FnZSBwYXR0ZXJucyB3aXRoIHNlbGVjdGlvbiBmb3IgdHJhbnNsYXRpb25hbCBlZmZpY2llbmN5LCBtdXRhdGlvbiBiaWFzLCBhbmQgZ2VuZXRpYyBkcmlmdCw7IGNpdGF0aW9uX2F1dGhvcj1TaGFoICBQLjsgY2l0YXRpb25fYXV0aG9yPUdpbGNocmlzdCAgTS5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTEwMjMxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGUgc3ByZWFkIHNlbGVjdGlvbiBmb3IgbG9jYWwgUk5BIHNlY29uZGFyeSBzdHJ1Y3R1cmUgaW4gY29kaW5nIHJlZ2lvbnMgb2YgYmFjdGVyaWFsIGdlbmVzLDsgY2l0YXRpb25fYXV0aG9yPUthdHogIEwuOyBjaXRhdGlvbl9hdXRob3I9QnVyZ2UgIEMuQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMDQyLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHVuaXZlcnNhbCB0cmVuZCBvZiByZWR1Y2VkIG1STkEgc3RhYmlsaXR5IG5lYXIgdGhlIHRyYW5zbGF0aW9uLWluaXRpYXRpb24gc2l0ZSBpbiBwcm9rYXJ5b3RlcyBhbmQgZXVrYXJ5b3Rlcyw7IGNpdGF0aW9uX2F1dGhvcj1HdSAgVy47IGNpdGF0aW9uX2F1dGhvcj1aaG91ICBULjsgY2l0YXRpb25fYXV0aG9yPVdpbGtlICBDLk8uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA2NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJvbmcgYXNzb2NpYXRpb24gYmV0d2VlbiBtUk5BIGZvbGRpbmcgc3RyZW5ndGggYW5kIHByb3RlaW4gYWJ1bmRhbmNlIGluIFMuIGNlcmV2aXNpYWUsOyBjaXRhdGlvbl9hdXRob3I9WnVyICBILjsgY2l0YXRpb25fYXV0aG9yPVR1bGxlciAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gUmVwOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjcyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCByZXF1aXJlbWVudHMgZm9yIG1STkEgZm9sZGluZyBwYXJ0aWFsbHkgZXhwbGFpbiB3aHkgaGlnaGx5IGV4cHJlc3NlZCBwcm90ZWlucyBldm9sdmUgc2xvd2x5LDsgY2l0YXRpb25fYXV0aG9yPVBhcmsgIEMuOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgWC47IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBKLlIuOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9RTY3OC04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gb3B0aW1hbGl0eSBpcyBhIG1ham9yIGRldGVybWluYW50IG9mIG1STkEgc3RhYmlsaXR5LDsgY2l0YXRpb25fYXV0aG9yPVByZXNueWFrICBWLjsgY2l0YXRpb25fYXV0aG9yPUFsaHVzYWluaSAgTi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBZLkguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2MDsgIGNpdGF0aW9uX3BhZ2VzPTExMTEtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbm9wdGltYWwgY29kb24gdXNhZ2UgaW7vrIJ1ZW5jZXMgcHJvdGVpbiBzdHJ1Y3R1cmUgaW4gaW50cmluc2ljYWxseSBkaXNvcmRlcmVkIHJlZ2lvbnMsOyBjaXRhdGlvbl9hdXRob3I9WmhvdSAgTS47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBULjsgY2l0YXRpb25fYXV0aG9yPUZ1ICBKLjsgY2l0YXRpb25fYXV0aG9yPVhpYW8gIEcuOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz05NzQtODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvLXRyYW5zbGF0aW9uYWwgcHJvdGVpbiBmb2xkaW5nIGlzIHJldmVhbGluZyB0aGUgc2VsZWN0aXZlIHVzZSBvZiBzeW5vbnltb3VzIGNvZG9ucyBhbG9uZyB0aGUgY29kaW5nIHNlcXVlbmNlIG9mIGEgbG93IGV4cHJlc3Npb24gZ2VuZSw7IGNpdGF0aW9uX2F1dGhvcj1SYXkgIFMuSy47IGNpdGF0aW9uX2F1dGhvcj1CYXJ1YWggIFYuSi47IGNpdGF0aW9uX2F1dGhvcj1TYXRhcGF0aHkgIFMuUy47IGNpdGF0aW9uX2F1dGhvcj1CYW5lcmplZSAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9NjEzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VyYXRlIHByZWRpY3Rpb24gb2YgY2VsbHVsYXIgY28tdHJhbnNsYXRpb25hbCBmb2xkaW5nIGluZGljYXRlcyBwcm90ZWlucyBjYW4gc3dpdGNoIGZyb20gcG9zdC0gdG8gY28tdHJhbnNsYXRpb25hbCBmb2xkaW5nOyBjaXRhdGlvbl9hdXRob3I9Tmlzc2xleSAgRC5BLjsgY2l0YXRpb25fYXV0aG9yPVNoYXJtYSAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPUFobWVkICBOLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEwMzQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbnMgZGlyZWN0IGNvdHJhbnNsYXRpb25hbCBmb2xkaW5nIHRvd2FyZCBkaWZmZXJlbnQgcHJvdGVpbiBjb25mb3JtYXRpb25zLDsgY2l0YXRpb25fYXV0aG9yPUJ1aHIgIEYuOyBjaXRhdGlvbl9hdXRob3I9SmhhICBTLjsgY2l0YXRpb25fYXV0aG9yPVRob21tZW4gIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz0zNDEtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiB0aGUgc3RyZW5ndGggb2Ygc2VsZWN0ZWQgY29kb24gdXNhZ2UgYmlhcyBhbW9uZyBiYWN0ZXJpYSw7IGNpdGF0aW9uX2F1dGhvcj1TaGFycCAgUC5NLjsgY2l0YXRpb25fYXV0aG9yPUJhaWxlcyAgRS47IGNpdGF0aW9uX2F1dGhvcj1Hcm9jb2NrICBSLkouOyBjaXRhdGlvbl9hdXRob3I9UGVkZW4gIEouRi47IGNpdGF0aW9uX2F1dGhvcj1Tb2NrZXR0ICBSLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTExNDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZmVyIFJOQSBnZW5lIG51bWJlcnMgbWF5IG5vdCBiZSBjb21wbGV0ZWx5IHJlc3BvbnNpYmxlIGZvciB0aGUgY29kb24gdXNhZ2UgYmlhcyBpbiBhc3BhcmFnaW5lLCBpc29sZXVjaW5lLCBwaGVueWxhbGFuaW5lIGFuZCB0eXJvc2luZSBpbiB0aGUgaGlnaCBleHByZXNzaW9uIGdlbmVzIGluIGJhY3RlcmlhLDsgY2l0YXRpb25fYXV0aG9yPVNhdGFwYXRoeSAgUy5TLjsgY2l0YXRpb25fYXV0aG9yPUR1dHRhICBNLjsgY2l0YXRpb25fYXV0aG9yPUJ1cmFnb2hhaW4gIEEuSy47IGNpdGF0aW9uX2F1dGhvcj1SYXkgIFMuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTM0LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBiaWFzIGluIHByb2thcnlvdGljIHB5cmltaWRpbmUtZW5kaW5nIGNvZG9ucyBpcyBhc3NvY2lhdGVkIHdpdGggdGhlIGRlZ2VuZXJhY3kgb2YgdGhlIGVuY29kZWQgYW1pbm8gYWNpZHMsOyBjaXRhdGlvbl9hdXRob3I9V2FsZCAgTi47IGNpdGF0aW9uX2F1dGhvcj1BbHJveSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Cb3R6bWFuICBNLjsgY2l0YXRpb25fYXV0aG9yPU1hcmdhbGl0ICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz03MDc0LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gb24gR0dVIGFuZCBDR1UgY29kb25zIGluIHRoZSBoaWdoIGV4cHJlc3Npb24gZ2VuZXMgaW4gYmFjdGVyaWEsOyBjaXRhdGlvbl9hdXRob3I9U2F0YXBhdGh5ICBTLlMuOyBjaXRhdGlvbl9hdXRob3I9UG93ZGVsICBCLlIuOyBjaXRhdGlvbl9hdXRob3I9RHV0dGEgIE0uOyBjaXRhdGlvbl9hdXRob3I9QnVyYWdvaGFpbiAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgUy5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9MTMtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltYWwgY29kb24gaWRlbnRpdGllcyBpbiBiYWN0ZXJpYTogaW1wbGljYXRpb25zIGZyb20gdGhlIGNvbmZsaWN0aW5nIHJlc3VsdHMgb2YgdHdvIGRpZmZlcmVudCBtZXRob2RzLDsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEIuOyBjaXRhdGlvbl9hdXRob3I9U2hhbyAgWi4tUS47IGNpdGF0aW9uX2F1dGhvcj1YdSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjI3MTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2UsOyBjaXRhdGlvbl9hdXRob3I9TG93ZSAgVC5NLjsgY2l0YXRpb25fYXV0aG9yPUVkZHkgIFMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcnkgcGVyc3BlY3RpdmUgb24gc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiB1bmljZWxsdWxhciBvcmdhbmlzbXMsOyBjaXRhdGlvbl9hdXRob3I9U2hhcnAgIFAuTS47IGNpdGF0aW9uX2F1dGhvcj1MaSAgVy4tSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2Ygb3B0aW1hbCBjb2RvbiBjaG9pY2VzIGluIHRoZSBmYW1pbHkgRW50ZXJvYmFjdGVyaWFjZWFlLDsgY2l0YXRpb25fYXV0aG9yPUlyaWFydGUgIEEuOyBjaXRhdGlvbl9hdXRob3I9QmFyYWliYXIgIEouRC47IGNpdGF0aW9uX2F1dGhvcj1Sb21lcm8gIEguOyBjaXRhdGlvbl9hdXRob3I9Q2FzdHJvLVNvd2luc2tpICBTLjsgY2l0YXRpb25fYXV0aG9yPU11c3RvICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTU1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gYW5kIHllYXN0IHByb3Rlb21lIGV2b2x1dGlvbiw7IGNpdGF0aW9uX2F1dGhvcj1Ba2FzaGkgIEguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz0xMjkxLTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0IG9mIGNvcnJlbGF0ZWQgdFJBTiBhYnVuZGFuY2VzIG9uIHRyYW5zbGF0aW9uIGVycm9ycyBhbmQgZXZvbHV0aW9uIG9mIGNvZG9uIHVzYWdlIGJpYXMsOyBjaXRhdGlvbl9hdXRob3I9U2hhaCAgUC47IGNpdGF0aW9uX2F1dGhvcj1HaWxjaHJpc3QgIE0uQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMTI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVhc3VyaW5nIGFuZCBkZXRlY3RpbmcgbW9sZWN1bGFyIGFkYXB0YXRpb24gaW4gY29kb24gdXNhZ2UgYWdhaW5zdCBub24tc2Vuc2UgZXJyb3JzIGR1cmluZyBwcm90ZWluIHRyYW5zbGF0aW9uLDsgY2l0YXRpb25fYXV0aG9yPUdpbGNocmlzdCAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPVNoYWggIFAuOyBjaXRhdGlvbl9hdXRob3I9WmFyZXR6a2kgIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz0xNDkzLTUwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgZGV0ZXJtaW5lcyB0aGUgdHJhbnNsYXRpb24gcmF0ZSBpbiBFc2NoZXJpY2hpYSBjb2xpLDsgY2l0YXRpb25fYXV0aG9yPVPDuHJlbnNlbiAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPUt1cmxhbmQgIEMuRy47IGNpdGF0aW9uX2F1dGhvcj1QZWRlcnNlbiAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDc7ICBjaXRhdGlvbl9wYWdlcz0zNjUtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFic29sdXRlIGluIHZpdm8gdHJhbnNsYXRpb24gcmF0ZXMgb2YgaW5kaXZpZHVhbCBjb2RvbnMgaW4gRXNjaGVyaWNoaWEgY29saSB0aGUgdHdvIGdsdXRhbWljIGFjaWQgY29kb25zLCBHQUEgYW5kIEdBRyBhcmUgdHJhbnNsYXRlZCB3aXRoIHRocmVlZm9sZCBkaWZmZXJlbmNlcyBpbiByYXRlLDsgY2l0YXRpb25fYXV0aG9yPVPDuHJlbnNlbiAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPVBlZGVyc2VuICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTIyMjsgIGNpdGF0aW9uX3BhZ2VzPTI2NS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2VzIGluZGljYXRpbmcgdGhlIGludm9sdmVtZW50IG9mIHNlbGVjdGlvbiBtZWNoYW5pc21zIGZvciB0aGUgb2NjdXJyZW5jZSBvZiBDMzQgYW50aWNvZG9uIGluIGJhY3RlcmlhOyBjaXRhdGlvbl9hdXRob3I9UHJhamFwYXRpICBWLksuOyBjaXRhdGlvbl9hdXRob3I9U2F0YXBhdGh5ICBTLlMuOyBjaXRhdGlvbl9hdXRob3I9U2F0aXNoIEt1bWFyICBNLlYuOyBjaXRhdGlvbl9hdXRob3I9QnVyYWdvaGFpbiAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgUy5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2VsbCBTY2kuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vZGlmaWNhdGlvbiBvZiB0aGUgd29iYmxlIGJhc2Ugb2YgdFJOQUdsdSBtb2R1bGF0ZXMgdGhlIHRyYW5zbGF0aW9uIHJhdGUgb2YgZ2x1dGFtaWMgYWNpZCBjb2RvbiBpbiB2aXZvLDsgY2l0YXRpb25fYXV0aG9yPUtyw7xnZXIgIE0uSy47IGNpdGF0aW9uX2F1dGhvcj1QZWRlcnNlbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1IYWdlcnZhbGwgIFQuRy47IGNpdGF0aW9uX2F1dGhvcj1Tw7hyZW5zZW4gIE0uQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yODQ7ICBjaXRhdGlvbl9wYWdlcz02MjEtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkHigJlzIG1vZGlmaWNhdGlvbnMgYnJpbmcgb3JkZXIgdG8gZ2VuZSBleHByZXNzaW9uLDsgY2l0YXRpb25fYXV0aG9yPUd1c3RpbG8gIEUuTS47IGNpdGF0aW9uX2F1dGhvcj1WZW5kZWl4ICBGLkEuUC47IGNpdGF0aW9uX2F1dGhvcj1BZ3JpcyAgUC5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTM0LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db252ZXJnZW50IGV2b2x1dGlvbiBvZiBBVUEgZGVjb2RpbmcgaW4gYmFjdGVyaWEgYW5kIGFyY2hhZWEsOyBjaXRhdGlvbl9hdXRob3I9U3V6dWtpICBULjsgY2l0YXRpb25fYXV0aG9yPU51bWF0YSAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTU4Ni05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RyYWludCBvbiBkaW51Y2xlb3RpZGVzIGJ5IGNvZG9uIHVzYWdlIGJpYXMgaW4gYmFjdGVyaWFsIGdlbm9tZXMsOyBjaXRhdGlvbl9hdXRob3I9U2F0YXBhdGh5ICBTLlMuOyBjaXRhdGlvbl9hdXRob3I9UG93ZGVsICBCLlIuOyBjaXRhdGlvbl9hdXRob3I9RHV0dGEgIE0uOyBjaXRhdGlvbl9hdXRob3I9QnVyYWdvaGFpbiAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgUy5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MzY7ICBjaXRhdGlvbl9wYWdlcz0xOC0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQSBpbWJhbGFuY2UgcHJvbW90ZXMgLTEgZnJhbWVzaGlmdGluZyB2aWEgbmVhci1jb2duYXRlIGRlY29kaW5nOyBjaXRhdGlvbl9hdXRob3I9T+KAmcOHb25ub3IgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9NzI3LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbWJhbGFuY2Ugb2YgdFJOQVBybyBpc29hY2NlcHRvcnMgaW5kdWNlcyArMSBmcmFtZXNoaWZ0aW5nIGF0IG5lYXItY29nbmF0ZSBjb2RvbnM7IGNpdGF0aW9uX2F1dGhvcj1P4oCZw4dvbm5vciAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NzU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9ucywgcmlib3NvbWUgc3BlZWQsIGFuZCBldWthcnlvdGljIGdlbmUgZXhwcmVzc2lvbiByZWd1bGF0aW9uLDsgY2l0YXRpb25fYXV0aG9yPVRhcnJhbnQgIEQuOyBjaXRhdGlvbl9hdXRob3I9dm9uIGRlciBIYWFyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBNb2wuIExpZmUgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9NDE5NS0yMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZXBsaWNhdGlvbi1yZWxhdGVkIG9yZ2FuaXphdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lcyw7IGNpdGF0aW9uX2F1dGhvcj1Sb2NoYSAgRS5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTE2MDktMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBndWFuaW5lIGFuZCBjeXRvc2luZSBjb250ZW50IG9mIGdlbm9taWMgRE5BIGFuZCBiYWN0ZXJpYWwgZXZvbHV0aW9uLDsgY2l0YXRpb25fYXV0aG9yPU11dG8gIEEuOyBjaXRhdGlvbl9hdXRob3I9T3Nhd2EgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBiZXR3ZWVuIGdlbm9tZXMgaXMgY29uc3RyYWluZWQgYnkgZ2Vub21lIHdpZGUgbXV0YXRpb25hbCBwcm9jZXNzZXMsOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPUxlZSAgVy47IGNpdGF0aW9uX2F1dGhvcj1Ib3R0cyAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPVNoYXBpcm8gIEwuOyBjaXRhdGlvbl9hdXRob3I9TWNBZGFtcyAgSC5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTM0ODAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lcmFsIG1vZGVsIG9mIGNvZG9uIGJpYXMgZHVlIHRvIEdDIG11dGF0aW9uYWwgYmlhcyw7IGNpdGF0aW9uX2F1dGhvcj1QYWxpZHdvciAgRy5BLjsgY2l0YXRpb25fYXV0aG9yPVBlcmtpbnMgIFQuSi47IGNpdGF0aW9uX2F1dGhvcj1YaWEgIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEzNDMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBtaXNyZWFkaW5nOiBhIHRSTkEgbW9kaWZpY2F0aW9uIGNvdW50ZXJhY3RzIGEgKyAyIHJpYm9zb21hbCBmcmFtZXNoaWZ0LDsgY2l0YXRpb25fYXV0aG9yPUJyw6lnZW9uICBELjsgY2l0YXRpb25fYXV0aG9yPUNvbG90ICBWLjsgY2l0YXRpb25fYXV0aG9yPVJhZG1hbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1UYWRkZWkgIEYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lcyBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMjk1LTMwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJpbmdpbmcgb3JkZXIgdG8gdHJhbnNsYXRpb246IHRoZSBjb250cmlidXRpb25zIG9mIHRyYW5zZmVyIFJOQSBhbnRpY29kb24tZG9tYWluIG1vZGlmaWNhdGlvbnMsOyBjaXRhdGlvbl9hdXRob3I9QWdyaXMgIFAuRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gUmVwOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02MjktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGlmaWNhdGlvbiBvZiB0aGUgd29iYmxlIHVyaWRpbmUgaW4gYmFjdGVyaWFsIGFuZCBtaXRvY2hvbmRyaWFsIHRSTkFzIHJlYWRpbmcgTk5BL05ORyB0cmlwbGV0cyBvZiAyLWNvZG9uIGJveGVzLDsgY2l0YXRpb25fYXV0aG9yPUFybWVuZ29kICBNLkUuOyBjaXRhdGlvbl9hdXRob3I9TWVzZWd1ZXIgIFMuOyBjaXRhdGlvbl9hdXRob3I9VmlsbGFycm95YSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTQ5NS01MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvc3QtdHJhbnNjcmlwdGlvbmFsIG1vZGlmaWNhdGlvbnMgdG8gdFJOQSDigJMgYSByZXNwb25zZSB0byB0aGUgZ2VuZXRpYyBjb2RlIGRlZ2VuZXJhY3ksOyBjaXRhdGlvbl9hdXRob3I9SG91ICBZLi1NLjsgY2l0YXRpb25fYXV0aG9yPUdhbXBlciAgSC47IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBXLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjQyLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBETkEgYW5hbHlzaXMgYWNyb3NzIGRpdmVyc2UgZ2Vub21lcyw7IGNpdGF0aW9uX2F1dGhvcj1LYXJsaW4gIFMuOyBjaXRhdGlvbl9hdXRob3I9Q2FtcGJlbGwgIEEuTS47IGNpdGF0aW9uX2F1dGhvcj1NcsOhemVrICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTE4NS0yMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uIGlzIHViaXF1aXRvdXMgaW4gcHJva2FyeW90ZXMsOyBjaXRhdGlvbl9hdXRob3I9U3VwZWsgIEYuOyBjaXRhdGlvbl9hdXRob3I9xaBrdW5jYSAgTi47IGNpdGF0aW9uX2F1dGhvcj1SZXBhciAgSi47IGNpdGF0aW9uX2F1dGhvcj1WbGFob3ZpxI1layAgSy47IGNpdGF0aW9uX2F1dGhvcj3FoG11YyAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMDA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQSBnZW5lcyByYXBpZGx5IGNoYW5nZSBpbiBldm9sdXRpb24gdG8gbWVldCBub3ZlbCB0cmFuc2xhdGlvbmFsIGRlbWFuZHMsOyBjaXRhdGlvbl9hdXRob3I9WW9uYSAgQS5ILjsgY2l0YXRpb25fYXV0aG9yPUJsb29tLUFja2VybWFubiAgWi47IGNpdGF0aW9uX2F1dGhvcj1GcnVta2luICBJLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9ZUxpZmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUwMTMzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uLWJpYXNlZCB0cmFuc2xhdGlvbiBjYW4gYmUgcmVndWxhdGVkIGJ5IHdvYmJsZS1iYXNlIHRSTkEgbW9kaWZpY2F0aW9uIHN5c3RlbXMgZHVyaW5nIGNlbGx1bGFyIHN0cmVzcyByZXNwb25zZXMsOyBjaXRhdGlvbl9hdXRob3I9RW5kcmVzICBMLjsgY2l0YXRpb25fYXV0aG9yPURlZG9uICBQLkMuOyBjaXRhdGlvbl9hdXRob3I9QmVnbGV5ICBULkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYwMy0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q290cmFuc2xhdGlvbmFsIHByb3RlaW4gZm9sZGluZyBvbiB0aGUgcmlib3NvbWUgbW9uaXRvcmVkIGluIHJlYWwgdGltZSw7IGNpdGF0aW9uX2F1dGhvcj1Ib2x0a2FtcCAgVy47IGNpdGF0aW9uX2F1dGhvcj1Lb2tpYyAgRy47IGNpdGF0aW9uX2F1dGhvcj1Kw6RnZXIgIE0uOyBjaXRhdGlvbl9hdXRob3I9TWl0dGVsc3TDpHQgIEouOyBjaXRhdGlvbl9hdXRob3I9S29tYXIgIEEuOyBjaXRhdGlvbl9hdXRob3I9Um9kbmluYSAgTS5WLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTA7ICBjaXRhdGlvbl9wYWdlcz0xMTA0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhldGVyb2xvZ291cyBwcm90ZWluIGV4cHJlc3Npb24gaXMgZW5oYW5jZWQgYnkgaGFybW9uaXppbmcgdGhlIGNvZG9uIHVzYWdlIGZyZXF1ZW5jaWVzIG9mIHRoZSB0YXJnZXQgZ2VuZSB3aXRoIHRob3NlIG9mIHRoZSBleHByZXNzaW9uIGhvc3RzLDsgY2l0YXRpb25fYXV0aG9yPUFuZ292ICBFLjsgY2l0YXRpb25fYXV0aG9yPUhpbGxlciAgQy5KLjsgY2l0YXRpb25fYXV0aG9yPUtpbmNhaWQgIFIuTC47IGNpdGF0aW9uX2F1dGhvcj1MeW9uICBKLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTIxODkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZTogbmF0dXJl4oCZcyByb2FkbWFwIHRvIGV4cHJlc3Npb24gYW5kIGZvbGRpbmcgb2YgcHJvdGVpbnMsOyBjaXRhdGlvbl9hdXRob3I9QW5nb3YgIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz02NTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgYmlhcyBpbiBDcmVuYXJjaGFlYSBhbmQgRXVyeWFyY2hhZWEgZ2Vub21lIHJldmVhbHMgZGlmZmVyZW50aWFsIHByZWZlcmVuY2Ugb2Ygc3lub255bW91cyBjb2RvbnMgdG8gZW5jb2RlIGhpZ2hseSBleHByZXNzZWQgcmlib3NvbWFsIGFuZCBSTkEgcG9seW1lcmFzZSBwcm90ZWluczsgY2l0YXRpb25fYXV0aG9yPUJhcnVhaCAgVi5KLjsgY2l0YXRpb25fYXV0aG9yPVNhdGFwYXRoeSAgUy5TLjsgY2l0YXRpb25fYXV0aG9yPVBvd2RlbCAgQi5SLjsgY2l0YXRpb25fYXV0aG9yPUtvbndhcmggIFIuOyBjaXRhdGlvbl9hdXRob3I9QnVyYWdvaGFpbiAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgUy5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGJpYXMgaXMgbm90IHNpZ25pZmljYW50bHkgZGlmZmVyZW50IGJldHdlZW4gdGhlIGhpZ2ggYW5kIHRoZSBsb3cgZXhwcmVzc2lvbiBnZW5lcyBpbiBodW1hbjsgY2l0YXRpb25fYXV0aG9yPVNhdGFwYXRoeSAgUy5TLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgUy5LLjsgY2l0YXRpb25fYXV0aG9yPVNhaG9vICBBLksuOyBjaXRhdGlvbl9hdXRob3I9QmVndW0gIFQuOyBjaXRhdGlvbl9hdXRob3I9R2hvc2ggIFQuQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gTW9sLiBHZW5ldC4gR2VuZSBUaGVyOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbiBiaWFzIGFuZCBmdW5jdGlvbmFsIGNvbnN0cmFpbnQgb24gR0MzLXJlbGF0ZWQgRE5BIGJhY2tib25lIGR5bmFtaWNzIGluIHRoZSBwcm9rYXJ5b3RpYyBudWNsZW9pZCw7IGNpdGF0aW9uX2F1dGhvcj1CYWJiaXR0ICBHLkEuOyBjaXRhdGlvbl9hdXRob3I9QWxhd2FkICBNLkEuOyBjaXRhdGlvbl9hdXRob3I9U2NodWx6ZSAgSy5WLjsgY2l0YXRpb25fYXV0aG9yPUh1ZHNvbiAgQS5PLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMDkxNS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gY2hvaWNlIGFuZCBnZW5lIGV4cHJlc3Npb246IHN5bm9ueW1vdXMgY29kb25zIGRpZmZlciBpbiB0aGVpciBhYmlsaXR5IHRvIGRpcmVjdCBhbWlub2FjeWxhdGVkLXRyYW5zZmVyIFJOQSBiaW5kaW5nIHRvIHJpYm9zb21lcyBpbiB2aXRybyw7IGNpdGF0aW9uX2F1dGhvcj1UaG9tYXMgIEwuSy47IGNpdGF0aW9uX2F1dGhvcj1EaXggIEQuQi47IGNpdGF0aW9uX2F1dGhvcj1UaG9tcHNvbiAgUi5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9NDI0Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgZGlmZmVyZW50IHRyaXBsZXRzIGVuY29kaW5nIHRoZSBzYW1lIGFtaW5vIGFjaWQsIHRlcm1lZCBhcyBzeW5vbnltb3VzIGNvZG9ucywgYXJlIG5vdCBlcXVhbGx5IGFidW5kYW50IGluIGEgZ2Vub21lLiBGYWN0b3JzIHN1Y2ggYXMgRyArIEMlIGFuZCB0UiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjIzNjE2MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRrXKR7JxE9T6FZNxGt3dNxPykQWA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f441%2f2236160" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f441%2f2236160" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=79980" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&amp;Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/5">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199233">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199233">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199235">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199235">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199240">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199240">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199244">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199244">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199276">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199276">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199283">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199283">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199289">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199289">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/5/427/2236148">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/5/451/2236165">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Siddhartha Sankar Satapathy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Computer Science and Engineering, Tezpur University, Napaam, Tezpur 784028, Assam, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Siddhartha+Sankar+Satapathy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satapathy S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Satapathy S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Siddhartha Sankar Satapathy</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bhesh Raj Powdel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div>Department of Statistics, Darrang College, Tezpur 784001, Assam, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bhesh+Raj+Powdel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Powdel B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Powdel B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bhesh Raj Powdel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alak Kumar Buragohain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div>Department of Molecular Biology and Biotechnology, Tezpur University, Napaam, Tezpur 784028, Assam, India</div><div class="aff"><div class="label -label"><sup>4</sup></div>Office of the Vice-Chancellor, Dibrugarh University, Dibrugarh 786004, Assam, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alak+Kumar+Buragohain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Buragohain A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Buragohain A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alak Kumar Buragohain</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suvendra Kumar Ray</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div>Department of Molecular Biology and Biotechnology, Tezpur University, Napaam, Tezpur 784028, Assam, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw027-cor1" data-open="dsw027-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Tel. (+91) 3712 275406. Fax. (+91) 3712 - 267005/267006. Email: <a href="mailto:suven@tezu.ernet.in" target="_blank">suven@tezu.ernet.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suvendra+Kumar+Ray">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ray S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ray S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Suvendra Kumar Ray</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw027-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 441–449, <a href='https://doi.org/10.1093/dnares/dsw027'>https://doi.org/10.1093/dnares/dsw027</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 July 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 January 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 May 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                            
                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Siddhartha Sankar Satapathy, Bhesh Raj Powdel, Alak Kumar Buragohain, Suvendra Kumar Ray;  Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria, <em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 441–449, <a href='https://doi.org/10.1093/dnares/dsw027'>https://doi.org/10.1093/dnares/dsw027</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236160?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Discrepancy%20among%20the%20synonymous%20codons%20with%20respect%20to%20their%20selection%20as%20optimal%20codon%20in%20bacteria&amp;publicationDate=2016-07-17&amp;volumeNum=23&amp;issueNum=5&amp;author=Satapathy%2C%20Siddhartha%20Sankar%3B%20Powdel%2C%20Bhesh%20Raj&amp;startPage=441&amp;endPage=449&amp;contentId=10.1093%2Fdnares%2Fdsw027&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f441%2f2236160" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199233 id="87199233" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The different triplets encoding the same amino acid, termed as synonymous codons, are not equally abundant in a genome. Factors such as G + C% and tRNA are known to influence their abundance in a genome. However, the order of the nucleotide in each codon <em>per se</em> might also be another factor impacting on its abundance values. Of the synonymous codons for specific amino acids, some are preferentially used in the high expression genes that are referred to as the ‘optimal codons’ (OCs). In this study, we compared OCs of the 18 amino acids in 221 species of bacteria. It is observed that there is amino acid specific influence for the selection of OCs. There is also influence of phylogeny in the choice of OCs for some amino acids such as Glu, Gln, Lys and Leu. The phenomenon of codon bias is also supported by the comparative studies of the abundance values of the synonymous codons with same G + C. It is likely that the order of the nucleotides in the triplet codon is also perhaps involved in the phenomenon of codon usage bias in organisms.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage bias&quot;">codon usage bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;optimal codon&quot;">optimal codon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome composition&quot;">genome composition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high expression genes&quot;">high expression genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="bacteria">bacteria</a></div>                            <h2 scrollto-destination=87199235 id="87199235" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>In the genetic codes, although synonymous codons encode the same amino acid, these codons are not used with equal proportions. Certain synonymous codons, known as the optimal codons (OCs), are preferred to the other codons, as witnessed more prominently in the high expression genes (HEGs) in genomes.<a reveal-id="dsw027-B1" data-open="dsw027-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw027-B2" data-open="dsw027-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The selection on OCs in the HEGs is reportedly a function of cellular adaptation for faster and/or more accurate translation.<a reveal-id="dsw027-B3 dsw027-B4 dsw027-B5 dsw027-B6 dsw027-B7" data-open="dsw027-B3 dsw027-B4 dsw027-B5 dsw027-B6 dsw027-B7" class="link link-ref link-reveal xref-bibr"><sup>3–7</sup></a> Different factors such as genome G + C%,<a reveal-id="dsw027-B8" data-open="dsw027-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> anticodon modification and codon–anticodon interaction<a reveal-id="dsw027-B4" data-open="dsw027-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw027-B9 dsw027-B10 dsw027-B11 dsw027-B12" data-open="dsw027-B9 dsw027-B10 dsw027-B11 dsw027-B12" class="link link-ref link-reveal xref-bibr"><sup>9–12</sup></a> and tRNA gene numbers<a reveal-id="dsw027-B13 dsw027-B14 dsw027-B15 dsw027-B16 dsw027-B17" data-open="dsw027-B13 dsw027-B14 dsw027-B15 dsw027-B16 dsw027-B17" class="link link-ref link-reveal xref-bibr"><sup>13–17</sup></a> are believed to influence OC selection in the HEGs in bacteria. In addition to the above, selection on mRNA secondary structure and stability,<a reveal-id="dsw027-B18 dsw027-B19 dsw027-B20 dsw027-B21 dsw027-B22" data-open="dsw027-B18 dsw027-B19 dsw027-B20 dsw027-B21 dsw027-B22" class="link link-ref link-reveal xref-bibr"><sup>18–22</sup></a> protein structure,<a reveal-id="dsw027-B23" data-open="dsw027-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> protein folding kinetics,<a reveal-id="dsw027-B6" data-open="dsw027-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw027-B24 dsw027-B25 dsw027-B26" data-open="dsw027-B24 dsw027-B25 dsw027-B26" class="link link-ref link-reveal xref-bibr"><sup>24–26</sup></a> are known to influence codon usage bias in genes.</p><p>Whether the selection of OCs is dependent upon these extrinsic factors mentioned above or upon the triplet sequence itself, has not been explored in detail. One of the approaches to find out the influence of the nucleotide sequence itself on their selection as OCs for different amino acids is to study and compare the OCs for different amino acids across different bacteria. A simple assumption is that the extrinsic factors influencing codon usage bias is likely to vary across bacteria but the intrinsic feature of a codon remains invariable. In this regard, there are several reports of analysis of the OCs in bacteria. Sharp et al. (2005)<a reveal-id="dsw027-B27" data-open="dsw027-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> observed that the C-ending codons of Phe, Tyr, Asn and Ile are always used as the OCs in preference to the U-ending codons in different species of bacteria. The findings of Sharp et al. (2005)<a reveal-id="dsw027-B27" data-open="dsw027-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> were in support of the earlier observation that growth rate influences the strength of translational selection in bacteria.<a reveal-id="dsw027-B15" data-open="dsw027-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> So, it is the C-ending codons in these amino acids that are selected as the OCs across bacteria.<a reveal-id="dsw027-B27 dsw027-B28 dsw027-B29" data-open="dsw027-B27 dsw027-B28 dsw027-B29" class="link link-ref link-reveal xref-bibr"><sup>27–29</sup></a> Sharp et al. (2005)<a reveal-id="dsw027-B27" data-open="dsw027-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> analysed only the WWY codons of the four amino acids that are decoded by one anticodon each. Whether the observation is also valid for the other three amino acids such as His, Asp and Cys that are encoded only by the NNY codons was not explored in their study.</p><p>In case of amino acids encoded by six or four synonymous codons, there are isoacceptor tRNAs in an organism for decoding these codons. The variation of the copy number of the isoacceptor tRNAs, multiple codon–anticodon interactions increase the variability of the extrinsic mechanisms in the selection of the OC in a bacterium. Recently, we reported that GGU and CGU with respect to Gly and Arg, respectively are selected as the OCs in bacteria.<a reveal-id="dsw027-B30" data-open="dsw027-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> This finding is important because both Gly and Arg are encoded by the codons belonging to the 4- and 6-fold degenerate families, respectively. This study<a reveal-id="dsw027-B30" data-open="dsw027-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> indicates that the sequence of nucleotides in a triplet is important for OC selection.</p><p>Published articles on the roles of the extrinsic factors such as genome composition and tRNA gene number on the selection of OCs have not provided any conclusion in this regard.<a reveal-id="dsw027-B8" data-open="dsw027-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw027-B31" data-open="dsw027-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> To address the above issue, we analysed the codon usage bias in all the 18 amino acids (Met and Trp encoded by one codon each were not considered) in 221 different species of bacteria belonging to 15 different phylogenetic groups. Comparison of the codon usage bias between the HEGs and the whole genome <em>vis-à-vis</em> the genome composition suggested that there is amino acid specific influence with regard to OC selection across bacteria. What is interesting is that only some of synonymous codons are selected as the OCs across bacteria. It was also observed that OCs for some amino acids can be different among bacteria belonging to different phylogenetic groups. Observations in this study indicate that the intrinsic differences among the synonymous codons may contribute towards their abundance values in a genome and in the choice of specific synonymous codon as the OC with respect to particular amino acid.</p>                            <h2 scrollto-destination=87199240 id="87199240" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
<p>Gene sequences of the bacterial genomes considered in the present study were downloaded from the DDBJ site (<a class="link link-uri" href="http://www.gib.genes.nig.ac.jp" target="_blank">www.gib.genes.nig.ac.jp</a>) (the gene sequences were taken from the site in 2011). Several HEG sequences (ribosomal protein, translation factor, <em>rpoB</em> and <em>rpoC</em> gene sequences)<a reveal-id="dsw027-B27" data-open="dsw027-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> were also taken from the NCBI site (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov" target="_blank">www.ncbi.nlm.nih.gov</a>). The total tRNA gene numbers were collected from the Genomic tRNA Database (<a class="link link-uri" href="http://gtrnadb.ucsc.edu" target="_blank">http://gtrnadb.ucsc.edu</a>), which uses tRNAscan-SE to classify tRNA into different groups on the basis of their anticodon sequences.<a reveal-id="dsw027-B32" data-open="dsw027-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> In this study, we considered 221 unique species of bacteria whose tRNA gene numbers and gene annotations were available in the online databases. These bacteria were with genomic G + C% between 72.83 and 22.44 and belong to 15 different phylogenetic groups of bacteria (<a reveal-id="dsw027-T2" data-open="dsw027-T2" class="link link-reveal link-table xref-fig">Table 1b</a>). </p>                        <a id="87199242" scrollto-destination="87199242"></a>
<div content-id="dsw027-T2" class="table-modal"><div class="table-wrap"><div id="dsw027-T2" class="table-wrap-title" data-id="dsw027-T2"><div class="label -label">Table 1b.</div><div class="caption"><p> Bacteria belonging to different phylogenetic groups and their genomic G + C% considered in the present study</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>S. No. </th><th>Group </th><th>No. of strains </th><th>Maximum G + C% </th><th>Minimum G + C% </th><th>Average G + C% </th></tr></thead><tbody><tr><td>1 </td><td>Actinobacteria </td><td char=".">23 </td><td char=".">72.83 </td><td char=".">46.31 </td><td char=".">64.45 </td></tr><tr><td>2 </td><td>α-Proteobacteria </td><td char=".">35 </td><td char=".">68.79 </td><td char=".">27.51 </td><td char=".">48.11 </td></tr><tr><td>3 </td><td>Aquificae </td><td char=".">1 </td><td char=".">43.3 </td><td char=".">43.3 </td><td char=".">43.3 </td></tr><tr><td>4 </td><td>Bacteroidetes </td><td char=".">8 </td><td char=".">66.13 </td><td char=".">22.44 </td><td char=".">40.26 </td></tr><tr><td>5 </td><td>β-Proteobacteria </td><td char=".">17 </td><td char=".">68.49 </td><td char=".">50.72 </td><td char=".">63.47 </td></tr><tr><td>6 </td><td>Chlamydiae </td><td char=".">6 </td><td char=".">41.31 </td><td char=".">39.19 </td><td char=".">40.10 </td></tr><tr><td>7 </td><td>Chlorobi </td><td char=".">1 </td><td char=".">56.53 </td><td char=".">56.53 </td><td char=".">56.53 </td></tr><tr><td>8 </td><td>Chloroflexi </td><td char=".">1 </td><td char=".">47.03 </td><td char=".">47.03 </td><td char=".">47.03 </td></tr><tr><td>9 </td><td>Cyanobacteria </td><td char=".">8 </td><td char=".">62.00 </td><td char=".">31.32 </td><td char=".">47.09 </td></tr><tr><td>10 </td><td>Δ-Proteobacteria </td><td char=".">6 </td><td char=".">71.38 </td><td char=".">50.65 </td><td char=".">62.27 </td></tr><tr><td>11 </td><td>ϵ-Proteobacteria </td><td char=".">11 </td><td char=".">44.54 </td><td char=".">27.05 </td><td char=".">36.56 </td></tr><tr><td>12 </td><td>Firmicutes </td><td char=".">38 </td><td char=".">56.98 </td><td char=".">28.21 </td><td char=".">38.44 </td></tr><tr><td>13 </td><td>γ-Proteobacteria </td><td char=".">47 </td><td char=".">68.67 </td><td char=".">22.48 </td><td char=".">48.19 </td></tr><tr><td>14 </td><td>Spirochaetes </td><td char=".">5 </td><td char=".">40.24 </td><td char=".">27.77 </td><td char=".">33.81 </td></tr><tr><td>15 </td><td>Tenericutes </td><td char=".">14 </td><td char=".">40.01 </td><td char=".">23.77 </td><td char=".">28.58 </td></tr></tbody></table></div><div content-id="dsw027-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>S. No. </th><th>Group </th><th>No. of strains </th><th>Maximum G + C% </th><th>Minimum G + C% </th><th>Average G + C% </th></tr></thead><tbody><tr><td>1 </td><td>Actinobacteria </td><td char=".">23 </td><td char=".">72.83 </td><td char=".">46.31 </td><td char=".">64.45 </td></tr><tr><td>2 </td><td>α-Proteobacteria </td><td char=".">35 </td><td char=".">68.79 </td><td char=".">27.51 </td><td char=".">48.11 </td></tr><tr><td>3 </td><td>Aquificae </td><td char=".">1 </td><td char=".">43.3 </td><td char=".">43.3 </td><td char=".">43.3 </td></tr><tr><td>4 </td><td>Bacteroidetes </td><td char=".">8 </td><td char=".">66.13 </td><td char=".">22.44 </td><td char=".">40.26 </td></tr><tr><td>5 </td><td>β-Proteobacteria </td><td char=".">17 </td><td char=".">68.49 </td><td char=".">50.72 </td><td char=".">63.47 </td></tr><tr><td>6 </td><td>Chlamydiae </td><td char=".">6 </td><td char=".">41.31 </td><td char=".">39.19 </td><td char=".">40.10 </td></tr><tr><td>7 </td><td>Chlorobi </td><td char=".">1 </td><td char=".">56.53 </td><td char=".">56.53 </td><td char=".">56.53 </td></tr><tr><td>8 </td><td>Chloroflexi </td><td char=".">1 </td><td char=".">47.03 </td><td char=".">47.03 </td><td char=".">47.03 </td></tr><tr><td>9 </td><td>Cyanobacteria </td><td char=".">8 </td><td char=".">62.00 </td><td char=".">31.32 </td><td char=".">47.09 </td></tr><tr><td>10 </td><td>Δ-Proteobacteria </td><td char=".">6 </td><td char=".">71.38 </td><td char=".">50.65 </td><td char=".">62.27 </td></tr><tr><td>11 </td><td>ϵ-Proteobacteria </td><td char=".">11 </td><td char=".">44.54 </td><td char=".">27.05 </td><td char=".">36.56 </td></tr><tr><td>12 </td><td>Firmicutes </td><td char=".">38 </td><td char=".">56.98 </td><td char=".">28.21 </td><td char=".">38.44 </td></tr><tr><td>13 </td><td>γ-Proteobacteria </td><td char=".">47 </td><td char=".">68.67 </td><td char=".">22.48 </td><td char=".">48.19 </td></tr><tr><td>14 </td><td>Spirochaetes </td><td char=".">5 </td><td char=".">40.24 </td><td char=".">27.77 </td><td char=".">33.81 </td></tr><tr><td>15 </td><td>Tenericutes </td><td char=".">14 </td><td char=".">40.01 </td><td char=".">23.77 </td><td char=".">28.58 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF2"><div class="fn"><p>Note: Aquificae, Chlorobi and Chloroflexi groups consisting of only one bacterium each are not considered in the result analysis shown in <a reveal-id="dsw027-F2" data-open="dsw027-F2" class="link link-reveal link-table xref-fig">Fig. 1b</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199242" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw027-T2" class="table-wrap-title" data-id="dsw027-T2"><div class="label -label">Table 1b.</div><div class="caption"><p> Bacteria belonging to different phylogenetic groups and their genomic G + C% considered in the present study</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>S. No. </th><th>Group </th><th>No. of strains </th><th>Maximum G + C% </th><th>Minimum G + C% </th><th>Average G + C% </th></tr></thead><tbody><tr><td>1 </td><td>Actinobacteria </td><td char=".">23 </td><td char=".">72.83 </td><td char=".">46.31 </td><td char=".">64.45 </td></tr><tr><td>2 </td><td>α-Proteobacteria </td><td char=".">35 </td><td char=".">68.79 </td><td char=".">27.51 </td><td char=".">48.11 </td></tr><tr><td>3 </td><td>Aquificae </td><td char=".">1 </td><td char=".">43.3 </td><td char=".">43.3 </td><td char=".">43.3 </td></tr><tr><td>4 </td><td>Bacteroidetes </td><td char=".">8 </td><td char=".">66.13 </td><td char=".">22.44 </td><td char=".">40.26 </td></tr><tr><td>5 </td><td>β-Proteobacteria </td><td char=".">17 </td><td char=".">68.49 </td><td char=".">50.72 </td><td char=".">63.47 </td></tr><tr><td>6 </td><td>Chlamydiae </td><td char=".">6 </td><td char=".">41.31 </td><td char=".">39.19 </td><td char=".">40.10 </td></tr><tr><td>7 </td><td>Chlorobi </td><td char=".">1 </td><td char=".">56.53 </td><td char=".">56.53 </td><td char=".">56.53 </td></tr><tr><td>8 </td><td>Chloroflexi </td><td char=".">1 </td><td char=".">47.03 </td><td char=".">47.03 </td><td char=".">47.03 </td></tr><tr><td>9 </td><td>Cyanobacteria </td><td char=".">8 </td><td char=".">62.00 </td><td char=".">31.32 </td><td char=".">47.09 </td></tr><tr><td>10 </td><td>Δ-Proteobacteria </td><td char=".">6 </td><td char=".">71.38 </td><td char=".">50.65 </td><td char=".">62.27 </td></tr><tr><td>11 </td><td>ϵ-Proteobacteria </td><td char=".">11 </td><td char=".">44.54 </td><td char=".">27.05 </td><td char=".">36.56 </td></tr><tr><td>12 </td><td>Firmicutes </td><td char=".">38 </td><td char=".">56.98 </td><td char=".">28.21 </td><td char=".">38.44 </td></tr><tr><td>13 </td><td>γ-Proteobacteria </td><td char=".">47 </td><td char=".">68.67 </td><td char=".">22.48 </td><td char=".">48.19 </td></tr><tr><td>14 </td><td>Spirochaetes </td><td char=".">5 </td><td char=".">40.24 </td><td char=".">27.77 </td><td char=".">33.81 </td></tr><tr><td>15 </td><td>Tenericutes </td><td char=".">14 </td><td char=".">40.01 </td><td char=".">23.77 </td><td char=".">28.58 </td></tr></tbody></table></div><div content-id="dsw027-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>S. No. </th><th>Group </th><th>No. of strains </th><th>Maximum G + C% </th><th>Minimum G + C% </th><th>Average G + C% </th></tr></thead><tbody><tr><td>1 </td><td>Actinobacteria </td><td char=".">23 </td><td char=".">72.83 </td><td char=".">46.31 </td><td char=".">64.45 </td></tr><tr><td>2 </td><td>α-Proteobacteria </td><td char=".">35 </td><td char=".">68.79 </td><td char=".">27.51 </td><td char=".">48.11 </td></tr><tr><td>3 </td><td>Aquificae </td><td char=".">1 </td><td char=".">43.3 </td><td char=".">43.3 </td><td char=".">43.3 </td></tr><tr><td>4 </td><td>Bacteroidetes </td><td char=".">8 </td><td char=".">66.13 </td><td char=".">22.44 </td><td char=".">40.26 </td></tr><tr><td>5 </td><td>β-Proteobacteria </td><td char=".">17 </td><td char=".">68.49 </td><td char=".">50.72 </td><td char=".">63.47 </td></tr><tr><td>6 </td><td>Chlamydiae </td><td char=".">6 </td><td char=".">41.31 </td><td char=".">39.19 </td><td char=".">40.10 </td></tr><tr><td>7 </td><td>Chlorobi </td><td char=".">1 </td><td char=".">56.53 </td><td char=".">56.53 </td><td char=".">56.53 </td></tr><tr><td>8 </td><td>Chloroflexi </td><td char=".">1 </td><td char=".">47.03 </td><td char=".">47.03 </td><td char=".">47.03 </td></tr><tr><td>9 </td><td>Cyanobacteria </td><td char=".">8 </td><td char=".">62.00 </td><td char=".">31.32 </td><td char=".">47.09 </td></tr><tr><td>10 </td><td>Δ-Proteobacteria </td><td char=".">6 </td><td char=".">71.38 </td><td char=".">50.65 </td><td char=".">62.27 </td></tr><tr><td>11 </td><td>ϵ-Proteobacteria </td><td char=".">11 </td><td char=".">44.54 </td><td char=".">27.05 </td><td char=".">36.56 </td></tr><tr><td>12 </td><td>Firmicutes </td><td char=".">38 </td><td char=".">56.98 </td><td char=".">28.21 </td><td char=".">38.44 </td></tr><tr><td>13 </td><td>γ-Proteobacteria </td><td char=".">47 </td><td char=".">68.67 </td><td char=".">22.48 </td><td char=".">48.19 </td></tr><tr><td>14 </td><td>Spirochaetes </td><td char=".">5 </td><td char=".">40.24 </td><td char=".">27.77 </td><td char=".">33.81 </td></tr><tr><td>15 </td><td>Tenericutes </td><td char=".">14 </td><td char=".">40.01 </td><td char=".">23.77 </td><td char=".">28.58 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF2"><div class="fn"><p>Note: Aquificae, Chlorobi and Chloroflexi groups consisting of only one bacterium each are not considered in the result analysis shown in <a reveal-id="dsw027-F2" data-open="dsw027-F2" class="link link-reveal link-table xref-fig">Fig. 1b</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199242" target="_blank">View Large</a></div></div><p>In general, the codons present in greater abundance and therefore used more frequently in the HEGs in comparison to all the genes in the genome are considered as the OCs.<a reveal-id="dsw027-B4" data-open="dsw027-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw027-B30" data-open="dsw027-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> In our study here, to avoid the borderline cases, we considered those codons as the OCs, which are with abundance values of 5% or higher in the HEGs than the same with respect to all the genes of the genome. Codon abundance values, synonymous codon frequency ranks and the percentage of amino acid usage were calculated using a computer programme written in the C language. Figures were plotted using the Microsoft Excel. The relative synonymous codon usage values (RSCU) of codons were calculated according to the formula given by Sharp and Li (1986).<a reveal-id="dsw027-B33" data-open="dsw027-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> RSCU<sub>diff</sub> is defined as the subtraction value between the RSCU value of a codon in the HEGs and the RSCU value of a codon in the genome.</p>                            <h2 scrollto-destination=87199244 id="87199244" class="section-title" data-legacy-id=SEC3>3. Results</h2>
                            <h3 scrollto-destination=87199245 id="87199245" class="section-title" data-legacy-id=SEC3.1>3.1. Optimal codons for different amino acids do not follow a common pattern with regard to genome composition</h3>
<p>The study was based on the codon usage bias in 221 bacteria (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 1</a></span>). The bacteria considered in this study included single strain from different species and also contained bacteria from 15 different phylogenetic groups. The 221 bacteria were divided into two groups depending on the total number of tRNA genes present in their genomes: (i) the high tRNA number (HTN) group having a number of tRNA genes ≥ 50 and (ii) the low tRNA number (LTN) group having the number of tRNA genes &lt; 50. Bacteria included in the HTN were presumed to have a strong translational selection in codon usage bias and those included in the LTN were presumed to have a weak translational selection as had been defined earlier.<a reveal-id="dsw027-B15" data-open="dsw027-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw027-B31" data-open="dsw027-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Both HTN and LTN bacteria were further arranged in different groups according to their genome G + C%. As the maximum G + C% known in bacterial genomes is 75%, five different groups were made ranging from the very high (VH) G + C% to the very low (VL) G + C% genomes as follows: VH (G + C% ≥ 65), high (H; 55 ≤ G + C% &lt; 65), moderate (M; 45 ≤ G + C% &lt; 55), low (L; 35 ≤ G + C% &lt; 45) and VL (G + C% &lt; 35). The HTN included total 119 bacteria whereas the LTN included 102 bacteria (<a reveal-id="dsw027-T1" data-open="dsw027-T1" class="link link-reveal link-table xref-fig">Table 1a</a>). </p>                        <a id="87199247" scrollto-destination="87199247"></a>
<div content-id="dsw027-T1" class="table-modal"><div class="table-wrap"><div id="dsw027-T1" class="table-wrap-title" data-id="dsw027-T1"><div class="label -label">Table 1a.</div><div class="caption"><p>Number of bacteria studied in the HTN and the LTN groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>G + C% </th><th>HTN </th><th>LTN </th></tr></thead><tbody><tr><td>VH </td><td char=".">21 </td><td char=".">13 </td></tr><tr><td>H </td><td char=".">30 </td><td char=".">8 </td></tr><tr><td>M </td><td char=".">24 </td><td char=".">10 </td></tr><tr><td>L </td><td char=".">32 </td><td char=".">35 </td></tr><tr><td>VL </td><td char=".">12 </td><td char=".">36 </td></tr><tr><td>Total </td><td char=".">119 </td><td char=".">102 </td></tr></tbody></table></div><div content-id="dsw027-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>G + C% </th><th>HTN </th><th>LTN </th></tr></thead><tbody><tr><td>VH </td><td char=".">21 </td><td char=".">13 </td></tr><tr><td>H </td><td char=".">30 </td><td char=".">8 </td></tr><tr><td>M </td><td char=".">24 </td><td char=".">10 </td></tr><tr><td>L </td><td char=".">32 </td><td char=".">35 </td></tr><tr><td>VL </td><td char=".">12 </td><td char=".">36 </td></tr><tr><td>Total </td><td char=".">119 </td><td char=".">102 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF1"><div class="fn"><p>VH (very high; G + C% ≥ 65), H (high; 55 ≤ G + C% &lt; 65), M (moderate; 45 ≤ G + C% &lt; 55), L (low; 35 ≤ G + C% &lt; 45) and VL (very low; G + C%&lt;35).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199247" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw027-T1" class="table-wrap-title" data-id="dsw027-T1"><div class="label -label">Table 1a.</div><div class="caption"><p>Number of bacteria studied in the HTN and the LTN groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>G + C% </th><th>HTN </th><th>LTN </th></tr></thead><tbody><tr><td>VH </td><td char=".">21 </td><td char=".">13 </td></tr><tr><td>H </td><td char=".">30 </td><td char=".">8 </td></tr><tr><td>M </td><td char=".">24 </td><td char=".">10 </td></tr><tr><td>L </td><td char=".">32 </td><td char=".">35 </td></tr><tr><td>VL </td><td char=".">12 </td><td char=".">36 </td></tr><tr><td>Total </td><td char=".">119 </td><td char=".">102 </td></tr></tbody></table></div><div content-id="dsw027-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>G + C% </th><th>HTN </th><th>LTN </th></tr></thead><tbody><tr><td>VH </td><td char=".">21 </td><td char=".">13 </td></tr><tr><td>H </td><td char=".">30 </td><td char=".">8 </td></tr><tr><td>M </td><td char=".">24 </td><td char=".">10 </td></tr><tr><td>L </td><td char=".">32 </td><td char=".">35 </td></tr><tr><td>VL </td><td char=".">12 </td><td char=".">36 </td></tr><tr><td>Total </td><td char=".">119 </td><td char=".">102 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF1"><div class="fn"><p>VH (very high; G + C% ≥ 65), H (high; 55 ≤ G + C% &lt; 65), M (moderate; 45 ≤ G + C% &lt; 55), L (low; 35 ≤ G + C% &lt; 45) and VL (very low; G + C%&lt;35).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199247" target="_blank">View Large</a></div></div><p>In several instances, bacteria belonging to different phylogenetic groups are also found with restricted genome G + C composition. For example, Actinobacteria and β-Proteobacteria are with high genome G + C composition, whereas Firmicutes and Tenericutes are with low genome G + C composition. In this study, the number of bacteria from the different phylogenetic groups considered for analysis of OC is variable. To find out if there is specific OC choice in bacteria according to their phylogenetic group, that might have been overlooked during analysis by dividing the bacteria according to genome G + C composition as described above, analysis of OCs was also performed in these bacteria by dividing them into different groups according to their phylogeny (<a reveal-id="dsw027-T2" data-open="dsw027-T2" class="link link-reveal link-table xref-fig">Table 1b</a>). Therefore, the OCs for the different amino acids were found out in the above bacteria on the basis of genome composition as well as phylogenetic affiliations (<a reveal-id="dsw027-T1" data-open="dsw027-T1" class="link link-reveal link-table xref-fig">Table 1a</a> and <a reveal-id="dsw027-T2" data-open="dsw027-T2" class="link link-reveal link-table xref-fig">b;</a><a reveal-id="dsw027-F1" data-open="dsw027-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a> and <a reveal-id="dsw027-F2" data-open="dsw027-F2" class="link link-reveal link-table xref-fig">b</a>). The OC compositions in different amino acids grouped under different degenerate codons were as follows. </p>                        <a id="87199249" scrollto-destination="87199249"></a>
<div id="dsw027-F1" data-id="dsw027-F1" class="fig fig-section"><div class="label fig-label">Figure 1a</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/m_dsw027f1p.png?Expires=2147483647&amp;Signature=t1Z5KaM2jdddfDV7iPDWTzPd0jdMxaE1iK5GJm6aI2LKHD6~nkij2g2F5PcPwpJhOUI6BUqSSnXnOG94MeUq781VlwRvQt8750wtbtW6HK0oFn6~EBU9xFVaNLG6i00a6pFD6V1MlJlA5hHxLkliYBmm9AusyUFQfrlofmAi8alpWgUsPMTS2EfqSN7fBAcYSaeABCYUBycx2t7YULU~j~T-YES10YPVvMznMnTNENJiqMR-TgR9KLNT3OPAzvoJxEu1n5egQUAfVt4sZrIezVBDcY5bUH5jBIw3XXHCtzAuc6N0My-oaVe0z-SwjUAsQIBTGP2Zq-GY4QvYMpeTbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimal codons of 18 amino acids in the HTN and in the LTN groups of bacteria within different genomic G + C% groups. A codon is considered as the OCs if it is used more frequently in the set of HEGs in comparison to all the genes in a genome. In our study here, to avoid any borderline cases, we considered those codons as the OCs, which are with abundance values of 5% or higher in the HEGs than the same with respect to all the genes of the genome. OCs were found out in 221 bacteria (Supplementary Table 1). The bacteria were considered in two groups: (A) HTN and (B) LTN. Further, in each of the two groups, bacteria were grouped into five sub-groups according to their genome G + C% (i) VH, 65.0 ≤ G + C%; (ii) H, 55.0 ≤ G + C% &lt; 65.0; (iii) M, 45.0 ≤ G + C% &lt; 55.0; (iv) L, 35.0 ≤ G + C% &lt; 45.0 and (v) VL, G + C% &lt; 35.0. Numbers of bacteria considered in each sub-group were given in Table 1. The colour code is used to represent % of bacteria in the G + C% group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to Supplementary Table 2." path-from-xml="dsw027f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199249/dsw027f1p.tiff" path-from-xml="dsw027f1p.tiff" target="_blank">View large</a><a section="87199249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027f1p.png?Expires=2147483647&amp;Signature=jrpe~wkauMgofvkaUY65W7ey~lDtu1jTJWFHKlUaSn3L-6iADMxoFPU5tXdTsm0YBxKjviIAEaimi0hX1D9nSCPRb9kF7KZGqLtbJX9gCH2adiKKv5CeupOyT5PHny2tnnnD9NcboE8q1KWFUkUQH-3IVu8xWlBo-UmcR4tcz2kpzCg4oizNiJED~n9sXy3FaZV9bmC2GRDBuELYo2OxzbIlGH00AjaQy4XTsrvMYb-VWXFIz~8ontLckl6lKW1H3nJyXFxid5BriC4iGsi~QI6U8pGoL5J67ArStrnXIjWh-is~AsTg2FN~3GENpaWw2eLqIbGjjEIeABTRxJCxNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199249&amp;ar=2236160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw027f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimal codons of 18 amino acids in the HTN and in the LTN groups of bacteria within different genomic G + C% groups. A codon is considered as the OCs if it is used more frequently in the set of HEGs in comparison to all the genes in a genome. In our study here, to avoid any borderline cases, we considered those codons as the OCs, which are with abundance values of 5% or higher in the HEGs than the same with respect to all the genes of the genome. OCs were found out in 221 bacteria (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 1</a></span>). The bacteria were considered in two groups: (A) HTN and (B) LTN. Further, in each of the two groups, bacteria were grouped into five sub-groups according to their genome G + C% (i) VH, 65.0 ≤ G + C%; (ii) H, 55.0 ≤ G + C% &lt; 65.0; (iii) M, 45.0 ≤ G + C% &lt; 55.0; (iv) L, 35.0 ≤ G + C% &lt; 45.0 and (v) VL, G + C% &lt; 35.0. Numbers of bacteria considered in each sub-group were given in <a reveal-id="dsw027-T1" data-open="dsw027-T1" class="link link-reveal link-table xref-fig">Table 1</a>. The colour code is used to represent % of bacteria in the G + C% group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 2</a></span>.</p></div></div><div content-id="dsw027-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1a</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/m_dsw027f1p.png?Expires=2147483647&amp;Signature=t1Z5KaM2jdddfDV7iPDWTzPd0jdMxaE1iK5GJm6aI2LKHD6~nkij2g2F5PcPwpJhOUI6BUqSSnXnOG94MeUq781VlwRvQt8750wtbtW6HK0oFn6~EBU9xFVaNLG6i00a6pFD6V1MlJlA5hHxLkliYBmm9AusyUFQfrlofmAi8alpWgUsPMTS2EfqSN7fBAcYSaeABCYUBycx2t7YULU~j~T-YES10YPVvMznMnTNENJiqMR-TgR9KLNT3OPAzvoJxEu1n5egQUAfVt4sZrIezVBDcY5bUH5jBIw3XXHCtzAuc6N0My-oaVe0z-SwjUAsQIBTGP2Zq-GY4QvYMpeTbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimal codons of 18 amino acids in the HTN and in the LTN groups of bacteria within different genomic G + C% groups. A codon is considered as the OCs if it is used more frequently in the set of HEGs in comparison to all the genes in a genome. In our study here, to avoid any borderline cases, we considered those codons as the OCs, which are with abundance values of 5% or higher in the HEGs than the same with respect to all the genes of the genome. OCs were found out in 221 bacteria (Supplementary Table 1). The bacteria were considered in two groups: (A) HTN and (B) LTN. Further, in each of the two groups, bacteria were grouped into five sub-groups according to their genome G + C% (i) VH, 65.0 ≤ G + C%; (ii) H, 55.0 ≤ G + C% &lt; 65.0; (iii) M, 45.0 ≤ G + C% &lt; 55.0; (iv) L, 35.0 ≤ G + C% &lt; 45.0 and (v) VL, G + C% &lt; 35.0. Numbers of bacteria considered in each sub-group were given in Table 1. The colour code is used to represent % of bacteria in the G + C% group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to Supplementary Table 2." path-from-xml="dsw027f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199249/dsw027f1p.tiff" path-from-xml="dsw027f1p.tiff" target="_blank">View large</a><a section="87199249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027f1p.png?Expires=2147483647&amp;Signature=jrpe~wkauMgofvkaUY65W7ey~lDtu1jTJWFHKlUaSn3L-6iADMxoFPU5tXdTsm0YBxKjviIAEaimi0hX1D9nSCPRb9kF7KZGqLtbJX9gCH2adiKKv5CeupOyT5PHny2tnnnD9NcboE8q1KWFUkUQH-3IVu8xWlBo-UmcR4tcz2kpzCg4oizNiJED~n9sXy3FaZV9bmC2GRDBuELYo2OxzbIlGH00AjaQy4XTsrvMYb-VWXFIz~8ontLckl6lKW1H3nJyXFxid5BriC4iGsi~QI6U8pGoL5J67ArStrnXIjWh-is~AsTg2FN~3GENpaWw2eLqIbGjjEIeABTRxJCxNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199249&amp;ar=2236160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw027f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimal codons of 18 amino acids in the HTN and in the LTN groups of bacteria within different genomic G + C% groups. A codon is considered as the OCs if it is used more frequently in the set of HEGs in comparison to all the genes in a genome. In our study here, to avoid any borderline cases, we considered those codons as the OCs, which are with abundance values of 5% or higher in the HEGs than the same with respect to all the genes of the genome. OCs were found out in 221 bacteria (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 1</a></span>). The bacteria were considered in two groups: (A) HTN and (B) LTN. Further, in each of the two groups, bacteria were grouped into five sub-groups according to their genome G + C% (i) VH, 65.0 ≤ G + C%; (ii) H, 55.0 ≤ G + C% &lt; 65.0; (iii) M, 45.0 ≤ G + C% &lt; 55.0; (iv) L, 35.0 ≤ G + C% &lt; 45.0 and (v) VL, G + C% &lt; 35.0. Numbers of bacteria considered in each sub-group were given in <a reveal-id="dsw027-T1" data-open="dsw027-T1" class="link link-reveal link-table xref-fig">Table 1</a>. The colour code is used to represent % of bacteria in the G + C% group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 2</a></span>.</p></div></div>                        <a id="87199250" scrollto-destination="87199250"></a>
<div id="dsw027-F2" data-id="dsw027-F2" class="fig fig-section"><div class="label fig-label">Figure 1b</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/m_dsw027f2p.png?Expires=2147483647&amp;Signature=Q~0iedl-iQOl8T2-j4HWkLUa18GfGF1qVtG4lyfxgxvwdLgMop4Clb-Mwx~m3LYqBWqm4FTwV3Jl57w5nRstlr1Uh51I6O8xzWxnoZLUZRkYMv8nehoMXEo~Zt3BTVxxQt-6a4niJKZA8dj1oxQ2RR-~fiXcWwmPOWsL-vuYBdGNeNAP9jMcnThQAAGiA6aScg6lNs8N1jakJkUa1PQyVCWcz1AK7dg3V3rdfyyrS~FEAEWt60vWKslAu8D3s3XyGpV5gfYG0Z7MG3p0IjE4buaSFRW4myQA~eXotw2PympdWNY7~GdQ4HUSiHxC-VEKY7cGCvWnAp8pJ5Xg~r3SDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimal codons of 18 amino acids in bacteria within different phylogenetic groups. As described earlier for Fig.1a, the OCs found out in 221 bacteria and were considered in 15 groups (Table 1b) according to the phylogeny. Out of the 15 groups, three groups having only one bacterium each were not considered for further analysis. The colour code is used to represent % of bacteria in the phylogenetic group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to Supplementary Supplementary Table 3." path-from-xml="dsw027f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199250/dsw027f2p.tiff" path-from-xml="dsw027f2p.tiff" target="_blank">View large</a><a section="87199250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027f2p.png?Expires=2147483647&amp;Signature=ulI5k98tqbOX1nHsofwGYOYyGJ7SJwrUsb8cQ9n3gwJui7J9tdiQfW-iq14BcCywa224fDhchjzAM~5HeKBEQhKBjiMgetD5CE8i~BE1uFQ3W6tUu2UucctbRAxCcflCURRq8QuwS8M2sZfZcVyW17XC~8md65qhEASuASKWb2GjhxpJA4eQ3RCzonND3ESxYESr-U7RoQ~OHAy-5YO0muDyUXgWtqJofVqgQkeAz8dazp~wIIKJjyBgOgjaT5wQ96JpzcWxc~KDD5lY0ykvw5A6ar-pNtQYUG3vcMbONu8kAhAIsM3Vz1XlJLX~A5MgxCBJy03VADGvj5o4X67VRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199250&amp;ar=2236160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw027f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimal codons of 18 amino acids in bacteria within different phylogenetic groups. As described earlier for Fig.1a, the OCs found out in 221 bacteria and were considered in 15 groups (<a reveal-id="dsw027-T1" data-open="dsw027-T1" class="link link-reveal link-table xref-fig">Table 1b</a>) according to the phylogeny. Out of the 15 groups, three groups having only one bacterium each were not considered for further analysis. The colour code is used to represent % of bacteria in the phylogenetic group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to Supplementary <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 3</a></span>.</p></div></div><div content-id="dsw027-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1b</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/m_dsw027f2p.png?Expires=2147483647&amp;Signature=Q~0iedl-iQOl8T2-j4HWkLUa18GfGF1qVtG4lyfxgxvwdLgMop4Clb-Mwx~m3LYqBWqm4FTwV3Jl57w5nRstlr1Uh51I6O8xzWxnoZLUZRkYMv8nehoMXEo~Zt3BTVxxQt-6a4niJKZA8dj1oxQ2RR-~fiXcWwmPOWsL-vuYBdGNeNAP9jMcnThQAAGiA6aScg6lNs8N1jakJkUa1PQyVCWcz1AK7dg3V3rdfyyrS~FEAEWt60vWKslAu8D3s3XyGpV5gfYG0Z7MG3p0IjE4buaSFRW4myQA~eXotw2PympdWNY7~GdQ4HUSiHxC-VEKY7cGCvWnAp8pJ5Xg~r3SDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimal codons of 18 amino acids in bacteria within different phylogenetic groups. As described earlier for Fig.1a, the OCs found out in 221 bacteria and were considered in 15 groups (Table 1b) according to the phylogeny. Out of the 15 groups, three groups having only one bacterium each were not considered for further analysis. The colour code is used to represent % of bacteria in the phylogenetic group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to Supplementary Supplementary Table 3." path-from-xml="dsw027f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199250/dsw027f2p.tiff" path-from-xml="dsw027f2p.tiff" target="_blank">View large</a><a section="87199250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027f2p.png?Expires=2147483647&amp;Signature=ulI5k98tqbOX1nHsofwGYOYyGJ7SJwrUsb8cQ9n3gwJui7J9tdiQfW-iq14BcCywa224fDhchjzAM~5HeKBEQhKBjiMgetD5CE8i~BE1uFQ3W6tUu2UucctbRAxCcflCURRq8QuwS8M2sZfZcVyW17XC~8md65qhEASuASKWb2GjhxpJA4eQ3RCzonND3ESxYESr-U7RoQ~OHAy-5YO0muDyUXgWtqJofVqgQkeAz8dazp~wIIKJjyBgOgjaT5wQ96JpzcWxc~KDD5lY0ykvw5A6ar-pNtQYUG3vcMbONu8kAhAIsM3Vz1XlJLX~A5MgxCBJy03VADGvj5o4X67VRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199250&amp;ar=2236160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw027f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimal codons of 18 amino acids in bacteria within different phylogenetic groups. As described earlier for Fig.1a, the OCs found out in 221 bacteria and were considered in 15 groups (<a reveal-id="dsw027-T1" data-open="dsw027-T1" class="link link-reveal link-table xref-fig">Table 1b</a>) according to the phylogeny. Out of the 15 groups, three groups having only one bacterium each were not considered for further analysis. The colour code is used to represent % of bacteria in the phylogenetic group where the codon was observed more frequent in the HEGs than the whole genome. For the raw data, please refer to Supplementary <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 3</a></span>.</p></div></div>                            <h3 scrollto-destination=87199251 id="87199251" class="section-title" data-legacy-id=SEC3.2>3.2. Optimal codons for the amino acids encoded only by the Y-ending codons</h3>
<p>Only the C-ending codons were found as the OCs for the five amino acids, <em>viz</em>, Asn, Asp, His, Phe and Tyr (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>) in all the bacteria irrespective of the genome G + C compositions. The observation was also consistent across the different phylogenetic groups (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>). Selection of the C-ending codons in these amino acids can be explained on the basis of codon–anticodon interaction.<a reveal-id="dsw027-B28" data-open="dsw027-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> The only anticodon G<sub>34</sub> (G at the first anticodon position) is used to decode both the synonymous codons of an amino acid. The C<sub>3</sub>:G<sub>34</sub> (C<sub>3</sub>: C at the third codon position; G<sub>34</sub>: G at the first anticodon position) pairing at the wobble position is preferred to the U<sub>3</sub>:G<sub>34</sub> pairing at the same position. But the same explanation is not tenable in case of the Cys codons. In the case of Cys, though the C-ending codon was found to be the OC in the high G + C% genomes as we observed for the other amino acids discussed above, the U-ending codon was the OC in the low G + C% genomes. Our observation here for the Cys codons is consistent with the observations made earlier by Iriarte et al. (2013)<a reveal-id="dsw027-B34" data-open="dsw027-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> in the family Enterobacteriaceae. Usually, the abundance of Cys is not high in the proteome of an organism. Whether this is the reason for the exceptional behaviour of the Cys codons in the low G + C% genomes is not known. In case of <em>Saccharomyces cerevisiae</em>, UGU was observed as the selected codon in the HEGs.<a reveal-id="dsw027-B35" data-open="dsw027-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw027-B36" data-open="dsw027-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Wald et al. (2012)<a reveal-id="dsw027-B29" data-open="dsw027-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> gave an additional explanation of translational accuracy to explain the selection of C<sub>3</sub> to U<sub>3</sub> in amino acids encoded only by the Y-ending codons in the split codon boxes (the four codons in a box are not synonymous; His and Gln are encoded by split codon box). In a split codon box, usually the U<sub>34</sub> anticodon decodes the NNR codons (the Gln codons, for example) while the G<sub>34</sub> anticodon decodes NNY codons (the His codons, for example). While the G<sub>34</sub> accurately decodes the NNY codons, the U<sub>34</sub> apart from accurately decoding the NNR codons has a possibility of wrongly coding the U<sub>3</sub> codon of the split codon box. So to avoid the translational error in the HEGs due to mispairing of the U<sub>34</sub> anticodon with the U<sub>3</sub> codon of a split codon box, the C<sub>3</sub> codons are selected more than the U<sub>3</sub> codons in these genes. As the UGG codon for Trp (Trp and Cys are part of the same spilt codon box) is decoded by the C<sub>34</sub> anticodon, the greater abundance of the U<sub>3</sub> codon Cys is not likely to be a problem for the cell. If it is true, then the observations in the case of Cys should also have been extended to the Tyr (UAY) and Ile (AUY) codons as there is no U<sub>34</sub> anticodon in these two split codon boxes. While in the case of Tyr, the selection of U-ending codon was not observed in general, in case of Ile, the selection of U-ending codon was observed in some bacterial groups. It is pertinent to note that there is a strong G:C base pairing at the second codon position in case of the Cys codons, and there is absence of any G:C base pairing at the first and the second codon positions in case of Ile and Tyr codons, which makes the C-ending codon more important for Ile and Tyr than in the case of Cys. The other possibility that selection of the U-ending codons in the HEGs might be an adaptation against the non-sense error,<a reveal-id="dsw027-B37" data-open="dsw027-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> i.e. the release factor has an erroneous preference to UGC than to UGU to cause premature translation termination. But this argument contradicts the selection of UGC in the high G + C genomes and also the selection of the C-ending codon in case of Tyr, which is encoded by the spilt codon box having stop codons. The other possibility of the selection of OCs in the direction of genome composition may be attributed to more stable mRNA secondary structure of HEGs. </p>                        <a id="87199253" scrollto-destination="87199253"></a>
<div content-id="dsw027-T3" class="table-modal"><div class="table-wrap"><div id="dsw027-T3" class="table-wrap-title" data-id="dsw027-T3"><div class="label -label">Table 2.</div><div class="caption"><p>A general pattern indicating amino acid specific determination of OCs in different bacterial groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Degeneracy </th><th>Amino acids </th><th>Selected OCs in the high expression genes<a reveal-id="dsw027-TF3" data-open="dsw027-TF3" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td rowspan="2">Two (NNY) </td><td>Phe, Tyr, Asn,His and Asp </td><td>C-ending codons in all genomes </td></tr><tr><td>Cys </td><td>C-ending codon in G + C high genomes (e.g. Actinobacteria, β-Proteobacteria) and U-ending codon in G + C low genomes (e.g. Firmicutes) </td></tr><tr><td rowspan="2">Two (NNR) </td><td>Gln and Lys </td><td>The G-ending codons in the G + C high (e.g. Actinobacteria, β-Proteobacteria) and the A-ending codons in the G + C low genomes (Firmicutes). Exception is Spirochaetes (G + C low) where the G-ending codon </td></tr><tr><td>Glu </td><td>The G-ending codons in G + C high/low genomes (e.g. Actinobacteria and Spirochaetes) and the A-ending in G + C high/low genomes (e.g. β-Proteobacteria and Firmicutes) </td></tr><tr><td>Three </td><td>Ile </td><td>C-ending codons in G + C high and U/C-ending in G + C low genomes </td></tr><tr><td rowspan="3">Four </td><td>Val, Thr and Ala </td><td>Y-ending codons (C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td>Pro </td><td>R-ending codons (G-ending codons in G + C high and A-ending in G + C low genomes) </td></tr><tr><td>Gly </td><td>Y-ending codons (U/C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td rowspan="2">Six </td><td>Leu and Ser </td><td>Family box codons (G/C-ending codons in G + C high and A/U-ending in G + C low genomes) </td></tr><tr><td>Arg </td><td>Y-ending codon (U/C-ending codons in G + C high and U-ending codon in G + C low genomes) </td></tr></tbody></table></div><div content-id="dsw027-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>Degeneracy </th><th>Amino acids </th><th>Selected OCs in the high expression genes<a reveal-id="dsw027-TF3" data-open="dsw027-TF3" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td rowspan="2">Two (NNY) </td><td>Phe, Tyr, Asn,His and Asp </td><td>C-ending codons in all genomes </td></tr><tr><td>Cys </td><td>C-ending codon in G + C high genomes (e.g. Actinobacteria, β-Proteobacteria) and U-ending codon in G + C low genomes (e.g. Firmicutes) </td></tr><tr><td rowspan="2">Two (NNR) </td><td>Gln and Lys </td><td>The G-ending codons in the G + C high (e.g. Actinobacteria, β-Proteobacteria) and the A-ending codons in the G + C low genomes (Firmicutes). Exception is Spirochaetes (G + C low) where the G-ending codon </td></tr><tr><td>Glu </td><td>The G-ending codons in G + C high/low genomes (e.g. Actinobacteria and Spirochaetes) and the A-ending in G + C high/low genomes (e.g. β-Proteobacteria and Firmicutes) </td></tr><tr><td>Three </td><td>Ile </td><td>C-ending codons in G + C high and U/C-ending in G + C low genomes </td></tr><tr><td rowspan="3">Four </td><td>Val, Thr and Ala </td><td>Y-ending codons (C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td>Pro </td><td>R-ending codons (G-ending codons in G + C high and A-ending in G + C low genomes) </td></tr><tr><td>Gly </td><td>Y-ending codons (U/C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td rowspan="2">Six </td><td>Leu and Ser </td><td>Family box codons (G/C-ending codons in G + C high and A/U-ending in G + C low genomes) </td></tr><tr><td>Arg </td><td>Y-ending codon (U/C-ending codons in G + C high and U-ending codon in G + C low genomes) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF3"><div class="fn"><p>*All are statistically significant (<em>chi</em>-square significance test <em>P</em> value &lt; 0.01).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199253" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw027-T3" class="table-wrap-title" data-id="dsw027-T3"><div class="label -label">Table 2.</div><div class="caption"><p>A general pattern indicating amino acid specific determination of OCs in different bacterial groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Degeneracy </th><th>Amino acids </th><th>Selected OCs in the high expression genes<a reveal-id="dsw027-TF3" data-open="dsw027-TF3" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td rowspan="2">Two (NNY) </td><td>Phe, Tyr, Asn,His and Asp </td><td>C-ending codons in all genomes </td></tr><tr><td>Cys </td><td>C-ending codon in G + C high genomes (e.g. Actinobacteria, β-Proteobacteria) and U-ending codon in G + C low genomes (e.g. Firmicutes) </td></tr><tr><td rowspan="2">Two (NNR) </td><td>Gln and Lys </td><td>The G-ending codons in the G + C high (e.g. Actinobacteria, β-Proteobacteria) and the A-ending codons in the G + C low genomes (Firmicutes). Exception is Spirochaetes (G + C low) where the G-ending codon </td></tr><tr><td>Glu </td><td>The G-ending codons in G + C high/low genomes (e.g. Actinobacteria and Spirochaetes) and the A-ending in G + C high/low genomes (e.g. β-Proteobacteria and Firmicutes) </td></tr><tr><td>Three </td><td>Ile </td><td>C-ending codons in G + C high and U/C-ending in G + C low genomes </td></tr><tr><td rowspan="3">Four </td><td>Val, Thr and Ala </td><td>Y-ending codons (C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td>Pro </td><td>R-ending codons (G-ending codons in G + C high and A-ending in G + C low genomes) </td></tr><tr><td>Gly </td><td>Y-ending codons (U/C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td rowspan="2">Six </td><td>Leu and Ser </td><td>Family box codons (G/C-ending codons in G + C high and A/U-ending in G + C low genomes) </td></tr><tr><td>Arg </td><td>Y-ending codon (U/C-ending codons in G + C high and U-ending codon in G + C low genomes) </td></tr></tbody></table></div><div content-id="dsw027-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>Degeneracy </th><th>Amino acids </th><th>Selected OCs in the high expression genes<a reveal-id="dsw027-TF3" data-open="dsw027-TF3" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td rowspan="2">Two (NNY) </td><td>Phe, Tyr, Asn,His and Asp </td><td>C-ending codons in all genomes </td></tr><tr><td>Cys </td><td>C-ending codon in G + C high genomes (e.g. Actinobacteria, β-Proteobacteria) and U-ending codon in G + C low genomes (e.g. Firmicutes) </td></tr><tr><td rowspan="2">Two (NNR) </td><td>Gln and Lys </td><td>The G-ending codons in the G + C high (e.g. Actinobacteria, β-Proteobacteria) and the A-ending codons in the G + C low genomes (Firmicutes). Exception is Spirochaetes (G + C low) where the G-ending codon </td></tr><tr><td>Glu </td><td>The G-ending codons in G + C high/low genomes (e.g. Actinobacteria and Spirochaetes) and the A-ending in G + C high/low genomes (e.g. β-Proteobacteria and Firmicutes) </td></tr><tr><td>Three </td><td>Ile </td><td>C-ending codons in G + C high and U/C-ending in G + C low genomes </td></tr><tr><td rowspan="3">Four </td><td>Val, Thr and Ala </td><td>Y-ending codons (C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td>Pro </td><td>R-ending codons (G-ending codons in G + C high and A-ending in G + C low genomes) </td></tr><tr><td>Gly </td><td>Y-ending codons (U/C-ending codons in G + C high and U-ending in G + C low genomes) </td></tr><tr><td rowspan="2">Six </td><td>Leu and Ser </td><td>Family box codons (G/C-ending codons in G + C high and A/U-ending in G + C low genomes) </td></tr><tr><td>Arg </td><td>Y-ending codon (U/C-ending codons in G + C high and U-ending codon in G + C low genomes) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF3"><div class="fn"><p>*All are statistically significant (<em>chi</em>-square significance test <em>P</em> value &lt; 0.01).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199253" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87199254 id="87199254" class="section-title" data-legacy-id=SEC3.3>3.3. Optimal codons for the amino acids encoded only by the R-ending codons</h3>
<p>In case of Glu, the A-ending codon was found as the OC across all G + C compositions. In case of Gln and Lys, the G-ending codons were found as the OC in G + C% high genomes while the A-ending codons were found as the OC in low G + C% genomes. Observations in the HTN and LTN were similar. The analysis in bacteria revealed that the selection of OCs was phylogeny specific in case of Glu. For example, in Actinobacteria (high G + C) and Spirochaetes (low G + C) the GAG was the OC whereas in β-Proteobacteria (high G + C) and Firmicutes (low G + C) the GAA was the OC. In case of y-Proteobacteria, GAA is selected as the OC, irrespective of genome composition. Glu is an example where the OC is selected according to the phylogenetic affiliation of the bacteria but not by genome G + C%. Earlier, it was demonstrated that GAA was a favoured codon over GAG in the HEGs in <em>Escherichia coli</em> and the speed of translation was higher in case of GAA than that of GAG.<a reveal-id="dsw027-B38" data-open="dsw027-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsw027-B39" data-open="dsw027-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Unlike the amino acids encoded by the Y-ending codons described before, for the amino acids encoded by the NNR codons, bacteria uses either U<sub>34</sub> or both U<sub>34</sub> and C<sub>34</sub> anticodons: the U<sub>34</sub> can decode both the A<sub>3</sub> and the G<sub>3</sub> codons, while the C<sub>34</sub>, decodes only the G<sub>3</sub> codons. While the occurrence of the U<sub>34</sub> is there in all bacteria, the C<sub>34</sub> is generally found in bacteria with high G + C genomes.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> The availability of two anticodons increases the effectiveness/probability of codon–anticodon pairing for an amino acid. Though the C<sub>34</sub> anticodon usually occurs in the high G + C% genomes, in case of β-Proteobacteria, in spite of being high G + C genomes, C<sub>34-</sub>Glu anticodon was not observed.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> This might be due to the fact that GAA is selected as the OC in this group of bacteria. In the case of Actinobacteria, which is also high G + C genome, where the GAG codon was selected as the OC, the C<sub>34-</sub>Glu was present in all the bacteria in this group.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> However, in case of Spirochaetes, C<sub>34-</sub>Glu was not found in all the bacteria, though GAG was selected as the OC. This indicates that the occurrence of C<sub>34</sub> is not essential for the GAG selection as the OC, rather some phylogeny specific feature may contribute for its selection.</p><p>The U<sub>34</sub> modification is same in case of tRNA of Gln (tQ), Glu (tE) and Lys(tK).<a reveal-id="dsw027-B10" data-open="dsw027-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> However, the choice of OC in case Glu was not found to be the same as for Lys and Gln in all the bacteria under consideration. A recent study with anticodon modifying enzyme in eukaryotes have revealed that the hypomodification of U<sub>34</sub> affects translation of CAA and AAA codons of Gln and Lys, but not GAA,<a reveal-id="dsw027-B12" data-open="dsw027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> the codon of Glu. This indicates that choice of OC in case of these three amino acids may not be always dependent upon the anticodon modification. In the Tenericutes, Chlamydiae and ϵ-Proteobacteria with low G + C genomes, no OC was evident as far as these three amino acid codons are concerned, though the selection of OC for the other amino acids was observed in these bacteria. The reason for the lack of specific choice of any synonymous codons as an OC in these groups needs further investigation. In case of Lys and Gln, the G-ending codon selection in the case of low G + C Spirochaetes was found as an exception also.</p><p>Krüger et al. (1998)<a reveal-id="dsw027-B41" data-open="dsw027-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> reported that the different modifications of the U<sub>34</sub> nucleotide in the anticodon of tE had different effects on the decoding rates of the GAA and GAG codons. Anticodon modifying enzymes has an impact on the selection of OC in bacteria.<a reveal-id="dsw027-B4" data-open="dsw027-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> It has been demonstrated now that this modification is important for the rate of decoding of the cognate codons in tune with the kinetics of translation and co-translational protein folding.<a reveal-id="dsw027-B12" data-open="dsw027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Therefore, a detailed study of the effect of U<sub>34</sub> modification enzymes on the anticodon of tRNA in decoding the codons of split boxes will give insight into the role of these enzymes and the choice of the OC in bacteria belonging to different phylogenetic groups.</p>                            <h3 scrollto-destination=87199258 id="87199258" class="section-title" data-legacy-id=SEC3.4>3.4. Optimal codon for Ile</h3>
<p>The C-ending codon was the OC in most of the bacteria while the U-ending codon was the OC in some low G + C genomes such as Tenericutes and Chlamydiae. Observations with respect to the HTN and the LTN were similar (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>). As has been discussed above, the selection of AUU as the OC in some low G + C bacteria in case of Ile is likely to have similarity with Cys due to the absence of the U<sub>34</sub> anticodon for the split codon box. Except in some bacteria belonging to Firmicutes, AUA was generally not observed as the OC in any other group of bacteria. AUA is a rare codon in bacteria, which is decoded by the anticodon CAU. The C<sub>34</sub> is modified to k2C<sub>34</sub>, to make correct pairing with AUA and to avoid incorrect pairing with the AUG codon.<a reveal-id="dsw027-B42" data-open="dsw027-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> There are also instances where AUA is decoded by U<sub>34</sub> in some specific groups of bacteria.<a reveal-id="dsw027-B43" data-open="dsw027-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The intrinsic feature such as the UA dinucleotide, which is generally avoided in the HEGs in bacteria,<a reveal-id="dsw027-B44" data-open="dsw027-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> might be a reason for AUA not being the OC in bacteria.</p>                            <h3 scrollto-destination=87199260 id="87199260" class="section-title" data-legacy-id=SEC3.5>3.5. Optimal codon for the amino acids encoded only by the 4-fold degenerate family box codons</h3>
<p>Based upon the choice of the OCs in bacteria, in general, the five amino acids, <em>viz</em>., Ala, Gly, Pro, Thr and Val, can be divided into three groups—first, in case of Ala, Thr and Val, the C-ending codons were found as the OC in the high G + C% genomes while the U-ending codons were found as the OC in the low G + C% genomes; second, in the case of Gly, both the U-ending and the C-ending codons were found as the OC in the high G + C% genomes, while only the U-ending codon was found as the OC in the low G + C% genomes; third, in case of Pro, the G-ending codon was found as the OC in the G + C% high genomes while the A-ending codon was found as the OC in the G + C% low genomes (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>). Therefore, in these five amino acids either the Y-ending codons or the R-ending codons were observed as the OCs in bacteria. The HTN and the LTN groups were similar for all the amino acids with respect to the OC.</p><p>The observation of more than one OCs for these amino acids in a genome is possible due to higher degeneracy. We analysed the OCs for these five amino acids in different bacteria according to their phylogenetic affiliations (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>). Bacteria use different isoacceptor tRNAs to decode the four different synonymous codons in a family box, which is described under various sparing strategies.<a reveal-id="dsw027-B10" data-open="dsw027-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Choice of OCs in family box codons is influenced by anticodon modification and tRNA gene copy number in bacteria.<a reveal-id="dsw027-B4" data-open="dsw027-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> The modification of U<sub>34</sub> in tRNAs across different family box codons is similar and this modification is different from that of the U<sub>34</sub> in tRNAs of split codon boxes.<a reveal-id="dsw027-B10" data-open="dsw027-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In case of Gly, the Y-ending codons were selected as the OC across different groups of bacteria. It has been reported earlier that decoding of the GGG codon is prone to −1 frameshift during translation.<a reveal-id="dsw027-B45" data-open="dsw027-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> This might be a reason for the less preference of this codon in the HEGs. In case of Pro, the CCC codon was not selected as the OC in bacteria which might be due to the fact that this codon is prone to +1 frameshift during translation.<a reveal-id="dsw027-B46" data-open="dsw027-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Apart from this, translation speed is hindered by the Pro codons both at the ‘P’ and the ‘A’ sites of a ribosome due to the difference in the chemical structure of this amino acid from the other amino acids.<a reveal-id="dsw027-B47" data-open="dsw027-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> In a case similar to Gly, the G-ending codons were not selected as the OC in case of Val, Ala and Thr, across different groups of bacteria. We believe that the intrinsic nature of these codons of different amino acids is likely to contribute towards their choice of OCs in bacteria.</p><p>The choice for the OCs is likely to be influenced by the tRNA gene copy number in bacteria. The gene copy number of G<sub>34</sub> anticodon, which is used to decode the Y-ending codons, is higher for Gly and is lower for Pro than the G<sub>34</sub> anticodons for the other amino acids.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> This is in concordance with the choice of GGU/GGC as the OC(s) and avoidance of CCC as an OC in bacteria. Transfer RNA with either U<sub>34</sub> or G<sub>34</sub> at the wobble position can pair with either the R-ending or the Y-ending codons, respectively. Therefore, if the GGG triplet is not favoured as the OC, there will be no selection for the higher copy number of the tRNA with U<sub>34</sub> anticodon. Low U<sub>34</sub> tRNA will automatically not prefer the GGA triplet as the OC. Therefore, in general, either the Y-ending or the R-ending codons are being selected as the OC for these amino acids in bacteria. As the G-ending codon in Val and Ala are not preferred as the OC, the C<sub>34</sub> is also a less preferred anticodon for these amino acids.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h3 scrollto-destination=87199264 id="87199264" class="section-title" data-legacy-id=SEC3.6>3.6. Optimal codon for the amino acids encoded by 6-fold degenerate codons</h3>
<p>Depending upon the choice of the OCs across bacteria, the three amino acids Arg, Leu and Ser can be divided into two groups—first, both U-ending and C-ending codons were found as the OC in the high G + C% genomes, while only the U-ending codon was found as the OC in the low G + C% genomes, in the case of Arg; second, in the case of Ser and Leu, the C-ending and the G-ending codons were found as the OC in the high G + C% genomes, while the U-ending and the A-ending codons were found as the OC in the low G + C% genomes (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>). In the three amino acids, the OCs were observed mostly from the family box codons and were less preferred from the split codon boxes. The observation with respect to the HTN and the LTN were similar for all the amino acids.</p><p>We studied the OCs of these three amino acids in bacteria that were divided according to their phylogenetic groupings (<a reveal-id="dsw027-T3" data-open="dsw027-T3" class="link link-reveal link-table xref-fig">Table 2</a>). In case of Arg, the CGU and the CGC codons were largely selected as the OC across different bacteria. The other four synonymous codons were generally not observed as the OC; however, in some bacterial groups such as Spirochaetes, Firmicutes and ϵ-Proteobacteria with low genome G + C%, AGA was also observed as the OC to some extent. It is pertinent to note that anticodons and its modifying enzymes used in bacteria to decode the Arg family box codons are different from the family box codons of the other amino acids.<a reveal-id="dsw027-B10" data-open="dsw027-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> I<sub>34</sub> in tR is used to decode the U/C/A-ending codons whereas C<sub>34</sub> is used to decode the G-ending codon. It is important to note that though there is similarity between Arg and Gly with respect to the choice of the OC (<a reveal-id="dsw027-F2" data-open="dsw027-F2" class="link link-reveal link-table xref-fig">Fig. 1b</a>), there is a difference regarding modification in anticodons of tG and tR.</p><p>In case of Leu, the choice of the OC in bacteria was found to be influenced by their phylogenetic affiliations. In case of γ-Proteobacteria and β-Proteobacteria which are more closely related phylogenetically in comparison to the Proteobacteria,<a reveal-id="dsw027-B48" data-open="dsw027-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> CUG was selected as the OC, which is different from δ-Proteobacteria and ϵ-Proteobacteria. In case of Firmicutes, UUA, UUG and CUU were selected as the OCs. CUA was avoided as the OC in all the bacterial group. In case of γ-Proteobacteria and β-Proteobacteria, the tRNA gene copy number with C<sub>34</sub> anticodon, which decodes the CUG codon, was higher than that in the other group of bacteria.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p><p>In case of Ser, the choice of OCs was found only from the four family box codons. OCs were found to be selected according to genome composition. For example, UCA and UCU were selected as OC in Firmicutes, Tenericutes (low G + C% genomes) whereas UCC and UCG were selected as OC in Actinobacteria, β-Proteobacteria (high G + C% genomes). In case of ϵ-Proteobacteria and Chlamydiae, low selection of OC was observed.</p><p>Study of codon usage bias in different amino acids exhibited that there is amino acid specific influence for the selection of OCs in bacteria. For example, out of the six codons of Ser, the AGY codons were never selected as the OCs in any group of bacteria. Though the OCs were selected within the family box codons, the principle for the Leu and Ser that are encoded by six synonymous codons is different from the amino acids encoded by the four synonymous codons. This relationship between degeneracy of the codons and the principle of selection for the OCs in amino acids is surprising and interesting considering the fact that the set of anticodons used to decode the different family box codons as well as the anticodon modifying enzymes for family box tRNA is similar in a bacterium. Therefore, it is less likely that the above observation was a consequence of the anticodons.</p>                            <h3 scrollto-destination=87199270 id="87199270" class="section-title" data-legacy-id=SEC3.7>3.7. The abundance of most of the OCs is higher than the synonymous codons with same G + C composition in a genome</h3>
<p>It is known that genome G + C% strongly influences the abundance value of a synonymous codon in genomes.<a reveal-id="dsw027-B49 dsw027-B50 dsw027-B51" data-open="dsw027-B49 dsw027-B50 dsw027-B51" class="link link-ref link-reveal xref-bibr"><sup>49–51</sup></a> Two synonymous codons with the same G + C composition were available in case of amino acids encoded by 4- or 6-fold degeneracy codons. In the above section, we observed that out of the two synonymous codons with same G + C%, one of the codons was selected as the OC in many bacteria. This prompted us to compare the abundance values between two synonymous codons having the same G + C% and their selection as the OCs in genomes. Their abundance comparison in amino acids encoded by 4-fold degeneracy and 6-fold degeneracy codons was presented in <a reveal-id="dsw027-T4" data-open="dsw027-T4" class="link link-reveal link-table xref-fig">Table 3</a>. The results of the comparison between the C-ending and the G-ending codons in VH, H and M groups in the eight family boxes were presented. </p>                        <a id="87199272" scrollto-destination="87199272"></a>
<div content-id="dsw027-T4" class="table-modal"><div class="table-wrap"><div id="dsw027-T4" class="table-wrap-title" data-id="dsw027-T4"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of the abundance values of the synonymous codons in bacteria with same G + C composition within family boxes</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th colspan="2">AA </th><th>VH </th><th>H </th><th>M </th></tr></thead><tbody><tr><td rowspan="2">Val </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td></tr><tr><td rowspan="2">Pro </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Thr </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ala </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub>∼G<sub>3</sub> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Gly </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Leu </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ser </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Arg </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr></tbody></table></div><div content-id="dsw027-T4" class="table-modal"><table><thead name="thead" align="left"><tr><th colspan="2">AA </th><th>VH </th><th>H </th><th>M </th></tr></thead><tbody><tr><td rowspan="2">Val </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td></tr><tr><td rowspan="2">Pro </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Thr </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ala </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub>∼G<sub>3</sub> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Gly </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Leu </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ser </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Arg </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF4"><div class="fn"><p>*<em>P</em> value &lt;0.05.</p></div></div><div content-id="dsw027-TF5"><div class="fn"><p>**<em>P</em> value &lt;0.01 (<em>chi-</em>square significant test; H<sub>0</sub>: G and C equal preference; H<sub>A</sub>: unequal preference between G and C).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199272" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw027-T4" class="table-wrap-title" data-id="dsw027-T4"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of the abundance values of the synonymous codons in bacteria with same G + C composition within family boxes</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th colspan="2">AA </th><th>VH </th><th>H </th><th>M </th></tr></thead><tbody><tr><td rowspan="2">Val </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td></tr><tr><td rowspan="2">Pro </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Thr </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ala </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub>∼G<sub>3</sub> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Gly </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Leu </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ser </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Arg </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr></tbody></table></div><div content-id="dsw027-T4" class="table-modal"><table><thead name="thead" align="left"><tr><th colspan="2">AA </th><th>VH </th><th>H </th><th>M </th></tr></thead><tbody><tr><td rowspan="2">Val </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">**</a> </td></tr><tr><td rowspan="2">Pro </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Thr </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ala </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub>∼G<sub>3</sub> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Gly </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Leu </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td rowspan="2">Ser </td><td>HTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td>LTN </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub> &gt; C<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>G<sub>3</sub>∼C<sub>3</sub> </td></tr><tr><td rowspan="2">Arg </td><td>HTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td>LTN </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF4" data-open="dsw027-TF4" class="link link-ref link-reveal xref-fn">*</a> </td><td>C<sub>3</sub> &gt; G<sub>3</sub><a reveal-id="dsw027-TF5" data-open="dsw027-TF5" class="link link-ref link-reveal xref-fn">*</a> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw027-TF4"><div class="fn"><p>*<em>P</em> value &lt;0.05.</p></div></div><div content-id="dsw027-TF5"><div class="fn"><p>**<em>P</em> value &lt;0.01 (<em>chi-</em>square significant test; H<sub>0</sub>: G and C equal preference; H<sub>A</sub>: unequal preference between G and C).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199272" target="_blank">View Large</a></div></div><p>The C-ending codons were more abundant than the G-ending codons in case of Ala, Gly and Thr. In fact, the C-ending codons were the OCs in these amino acids. In case of Pro, the G-ending codon was more abundant than the C-ending codon, which was in concordance with the selection as the OC. However, in case of Val, the G-ending codon was more abundant than the C-ending codon, though the latter was selected as the OC. In the HTN and the LTN, the abundance patterns of the different amino acid codons were similar.</p><p>In case of Arg, the abundance value of CGC was more than that of CGG. In fact, CGC was selected as the OC. In case of Leu, CUG abundance was higher than CUC. CUG was selected as the OC more often than CUC. In case of Ser, abundance values of the synonymous codons with the same G + C% were similar.</p><p>It is often observed that the abundance values of the OCs are higher than the abundance values of the synonymous codons with the same G + C%. In some cases, variations were observed. The differential abundance values between two synonymous codons with the same G + C composition for an amino acid were following a uniform pattern across different bacteria. Why in some amino acids such as Val and Ser, abundance values of OCs is not higher than the other synonymous codons with same G + C composition is an interesting question.</p>                            <h2 scrollto-destination=87199276 id="87199276" class="section-title" data-legacy-id=SEC4>4. Discussion</h2>
<p>The earlier notion was that the extrinsic factors such as genome G + C%, tRNA gene numbers, etc., cause the difference in abundance values among the synonymous codons. The interesting point emerged in this study is the amino acid specific choice of OC in bacteria depending upon their phylogenetic affiliations, which is independent of their genome composition and tRNA gene number. For example, the observation of GAA as OC in β-Proteobacteria and GAG as OC in the Spirochaetes in this study are the vindication of the influence of some bacterial phylogeny specific mechanisms influencing OC choice for Glu in these bacteria. Similarly, the observation of CUG as OCs in β-Proteobacteria and γ-Proteobacteria, that are phylogenetically closed than any other Proteobacteria also indicates a role of phylogeny specific mechanisms for OC choice in case of Leu in bacteria. The OCs for the three amino acids Gln, Glu and Lys, encoded only by the R-ending codons in split codon boxes were observed to be influenced maximally by the phylogeny of bacteria. The U<sub>34</sub> of these tRNA undergoes different modifications to avoid frameshift mutation,<a reveal-id="dsw027-B52" data-open="dsw027-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> correct decoding,<a reveal-id="dsw027-B53" data-open="dsw027-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a><sup>,</sup><a reveal-id="dsw027-B54" data-open="dsw027-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> to avoid miscoding of the non-synonymous codons of the amino acids of the split codon boxes,<a reveal-id="dsw027-B54" data-open="dsw027-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsw027-B55" data-open="dsw027-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> effective speed of decoding,<a reveal-id="dsw027-B12" data-open="dsw027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> during the translation. Our future research is aimed at doing a comparative study of the anticodon modifying enzymes across different phylogenetic groups of bacteria.</p><p>The observation of the OCs limiting to only some synonymous codons for different amino acids across bacteria is interesting in this study. Why is this discrepancy among the codons? There are some known mechanisms such as frameshift mutation in case of GGG and CCC codon for which these codons are avoided as the OC in bacteria. But the reason is not known for some other non-OCs observed in this study such as CUA, CGR, G-ending codons of Ala, Val and Thr, and AGY. The findings in this study clearly indicate that the potential of all synonymous codons with regard to their selection as OCs in bacteria is not same even though they are encoding the same amino acid.</p><p>There may be an influence of the preference and avoidance of dinucleotides during OC selection. The UA dinucleotide is not preferred in genomes,<a reveal-id="dsw027-B44" data-open="dsw027-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsw027-B56" data-open="dsw027-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> which might influence UUA, CUA, AUA and GUA for not being selected as the OC in any bacteria. Similarly for selection of CGU, GGU and UGU as the OCs, it may be argued that GU is the selected dinucleotide at the second codon position. Interestingly, the reverse complement of GU, AC are also observed to be preferred at the second position, as UAC, CAC are selected as the OCs in bacteria. Our finding on OCs is in concordance with the dinucleotide constraint on codon usage bias already reported in bacteria.<a reveal-id="dsw027-B44" data-open="dsw027-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p><p>According to our present understanding, codon usage bias in organisms is a cumulative effect of the extrinsic and intrinsic factors. In the selection of the OCs in bacteria whether it is the extrinsic or the intrinsic factor that plays dominant role is not clear. We believe that the amino acid specific role for the selection of the OC in bacteria is decided first by the codon–anticodon interaction. The selection of the C-ending codons in case of amino acids encoded by the NNY codons is a suitable example. In these amino acids, only one anticodon is available to decode both the codons. So out of the two codon–anticodon interactions, the more stable one was selected in all the bacteria irrespective of genome composition. It is also not dependent upon the tRNA gene numbers.<a reveal-id="dsw027-B28" data-open="dsw027-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> In case of the amino acids encoded by higher degeneracy codons, more than one codon can function as the OCs due to multiple codon–anticodon interactions. In this context, the increase in the copy number of the cognate tRNA complements the efficient translation of the OCs, which is required for bacterial adaptation to growth rate.<a reveal-id="dsw027-B15" data-open="dsw027-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw027-B16" data-open="dsw027-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> It is not the tRNA gene copy number <em>per se</em> which may be essential for selection of the OC for an amino acid.<a reveal-id="dsw027-B28" data-open="dsw027-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsw027-B57" data-open="dsw027-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Recently, it has been demonstrated that tRNA genes rapidly change in evolution to meet novel translational demands.<a reveal-id="dsw027-B58" data-open="dsw027-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> It has also been observed that the copy number of only specific isoacceptor tRNA genes increased across bacteria.<a reveal-id="dsw027-B40" data-open="dsw027-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> The role of anticodon modification enzymes and its effect on the selection of codons in different major taxa of life<a reveal-id="dsw027-B11" data-open="dsw027-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw027-B59" data-open="dsw027-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> also supports our assumption that codon–anticodon interactions are important determinant factors for the selection of the OCs in bacteria. More research on codon–anticodon pairing during translation is likely to provide empirical evidence in favour of the role of codon–anticodon interaction on the selection of OCs. It is usually believed that mutation pressure maintains the abundance values of two synonymous codons with same G + C% similar in a genome. The consistently higher abundance value of one synonymous codon over the other synonymous codons with same G + C composition across bacteria further supports the contribution from the inherent nucleotide sequence of the triplet codon to its abundance in genomes.</p><p>What is the evolutionary advantage or the selective force for having the amino acid specific principle for OC choice across bacteria? It is likely that the principle has enabled microorganisms to receive and express genes from the other microorganisms by lateral gene transfer. For example, both the U- and the C-ending codons of Gly are efficiently translated by the same anticodon.<a reveal-id="dsw027-B10" data-open="dsw027-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The translation rate of the gene inside the receptive host is less likely to be affected by the difference in codon usage bias. Now the well-known phenomenon of co-translational protein folding<a reveal-id="dsw027-B60" data-open="dsw027-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and the kinetics of translation is important for the structure and function of a protein.<a reveal-id="dsw027-B26" data-open="dsw027-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Thus, lateral gene transfer may also get inhibited due to the differential translational kinetics of a gene across different bacteria.<a reveal-id="dsw027-B61" data-open="dsw027-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a><sup>,</sup><a reveal-id="dsw027-B62" data-open="dsw027-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> A recent study has revealed that OC choice between the two archaea groups, Crenarchaea and Euryarchaea, is different for many amino acids,<a reveal-id="dsw027-B63" data-open="dsw027-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> unlike what we observed for bacteria in this study. As bacteria are evolutionarily different from archaea, the selection force on codon usage bias between the two groups of archaea may not be the same. Similarly in human, which is phylogenetically very different from bacteria and archaea, there is no difference between the high and the low expression genes with respect to codon usage bias.<a reveal-id="dsw027-B64" data-open="dsw027-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a></p><p>A recent publication by Babbitt et al. (2014)<a reveal-id="dsw027-B65" data-open="dsw027-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> discusses intrinsic DNA flexibility and codon usage bias across different bacteria, which is also in support of the analysis discussed in this paper. In conclusion, our study indicates a vital role of the nucleotide sequence of the triplet in selection as OCs in bacteria that was proposed three decades ago by Thomas et al. (1988).<a reveal-id="dsw027-B66" data-open="dsw027-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a></p>                            <h2 scrollto-destination=87199283 id="87199283" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are very much thankful to the two anonymous reviewers for their comments which helped us to improve the manuscript in different ways and to analyse the data differents. We also thank Ms Ruksana Aziz, Ms Ishani Goswami, Department of Molecular Biology and Biotechnology, Tezpur University for comments on the manuscript. S.K.R. and S.S.S. are thankful to DBT, Govt. of India for the twinning project under Bioinformatics.</p>                            <h3 scrollto-destination=87199285 id="87199285" class="section-title" >Conflict of Interest</h3>
<p>Authors do not have any conflict of interest.</p>                            <h3 scrollto-destination=87199287 id="87199287" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=87199289 id="87199289" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw027-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">W.H.</div>
            </div>
          </div>
          <div class="year">1987</div>, 
<div class="article-title">The codon Adaptation Index – a measure of directional synonymous codon usage bias, and its potential applications,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">15</div>, <div class="fpage">1281</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20Adaptation%20Index%20%E2%80%93%20a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications%2C&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
             <div class="name">
              <div class="surname">Emery</div>
               <div class="given-names">L.R.</div>
            </div>
             <div class="name">
              <div class="surname">Zeng</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">Forces that influence the evolution of codon bias,</div><div class="source">Philos. Trans. R. Soc. Lond. B. Biol. Sci</div> ., <div class="volume">365</div>, <div class="fpage">1203</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forces%20that%20influence%20the%20evolution%20of%20codon%20bias%2C&amp;author=P.M.%20Sharp&amp;author=L.R.%20Emery&amp;author=K.%20Zeng&amp;publication_year=2010&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B.%20Biol.%20Sci&amp;volume=365&amp;pages=1203-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2009.0305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2009.0305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2009.0305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20308095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bulmer</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1991</div>, 
<div class="article-title">The selection-mutation-drift theory of synonymous codon usage,</div><div class="source">Genetics</div> , <div class="volume">129</div>, <div class="fpage">897</div>–<div class="lpage">907</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20selection-mutation-drift%20theory%20of%20synonymous%20codon%20usage%2C&amp;author=M.%20Bulmer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=897-907" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1752426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ran</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Higgs</div>
               <div class="given-names">P.G.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">The influence of anticodon-codon interactions and modified bases on codon usage bias in bacteria,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">2129</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20influence%20of%20anticodon-codon%20interactions%20and%20modified%20bases%20on%20codon%20usage%20bias%20in%20bacteria%2C&amp;author=W.%20Ran&amp;author=P.G.%20Higgs&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol&amp;volume=27&amp;pages=2129-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Quax</div>
               <div class="given-names">T.E.</div>
            </div>
             <div class="name">
              <div class="surname">Claassens</div>
               <div class="given-names">N.J.</div>
            </div>
             <div class="name">
              <div class="surname">Söll</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">van der Oost</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Codon bias as a means to fine-tune gene expression,</div><div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">149</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20as%20a%20means%20to%20fine-tune%20gene%20expression%2C&amp;author=T.E.%20Quax&amp;author=N.J.%20Claassens&amp;author=D.%20S%C3%B6ll&amp;author=J.%20van%20der%20Oost&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=149-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.05.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.05.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.05.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26186290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">C.H.</div>
            </div>
             <div class="name">
              <div class="surname">Dang</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">Z.</div>
            </div>
          </div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Codon usage inﬂuences the local rate of translation elongation to regulate co-translational protein folding,</div><div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">1</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%EF%AC%82uences%20the%20local%20rate%20of%20translation%20elongation%20to%20regulate%20co-translational%20protein%20folding%2C&amp;author=C.H.%20Yu&amp;author=Y.%20Dang&amp;author=Z.%20Zhou&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.07.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.07.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.07.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26140365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
             <div class="name">
              <div class="surname">Goswami</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="year">2016</div>, 
<div class="article-title">Synonymous codons are not same with respect to the speed of translation elongation</div>, <div class="source">Curr. Sci.</div> , <div class="volume">110</div>, <div class="fpage">1612</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codons%20are%20not%20same%20with%20respect%20to%20the%20speed%20of%20translation%20elongation&amp;author=S.K.%20Ray&amp;author=I.%20Goswami&amp;publication_year=2016&amp;journal=Curr.%20Sci.&amp;volume=110&amp;pages=1612-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.18520/cs/v110/i4/682-686" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.18520%2Fcs%2Fv110%2Fi4%2F682-686" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.18520%2Fcs%2Fv110%2Fi4%2F682-686"> </span></div> </div></div></div></div></div><div content-id="dsw027-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hershberg</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Petrov</div>
               <div class="given-names">D.A.</div>
            </div>
          </div>
          <div class="year">2009</div>, 
<div class="article-title">General rules for optimal codon choice,</div><div class="source">PLoS Genet</div> ., <div class="volume">5</div>, <div class="fpage">e1000556</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20rules%20for%20optimal%20codon%20choice%2C&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2009&amp;journal=PLoS%20Genet&amp;volume=5&amp;pages=e1000556" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19593368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Grosjean</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Fiers</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="year">1982</div>, 
<div class="article-title">Preferential codon usage in prokaryotic genes: the optimal codon-anticodon interaction energy and the selective codon usage in efficiently expressed genes,</div><div class="source">Gene</div> , <div class="volume">18</div>, <div class="fpage">199</div>–<div class="lpage">209</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preferential%20codon%20usage%20in%20prokaryotic%20genes%3A%20the%20optimal%20codon-anticodon%20interaction%20energy%20and%20the%20selective%20codon%20usage%20in%20efficiently%20expressed%20genes%2C&amp;author=H.%20Grosjean&amp;author=W.%20Fiers&amp;publication_year=1982&amp;journal=Gene&amp;volume=18&amp;pages=199-209" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(82)90157-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(82)90157-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(82)90157-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6751939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Grosjean</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">de Crécy-Lagard</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Marck</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">Deciphering synonymous codons in the three domains of life: co-evolution with specific tRNA modification enzymes,</div><div class="source">FEBS Lett</div> ., <div class="volume">584</div>, <div class="fpage">252</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20synonymous%20codons%20in%20the%20three%20domains%20of%20life%3A%20co-evolution%20with%20specific%20tRNA%20modification%20enzymes%2C&amp;author=H.%20Grosjean&amp;author=V.%20de%20Cr%C3%A9cy-Lagard&amp;author=C.%20Marck&amp;publication_year=2010&amp;journal=FEBS%20Lett&amp;volume=584&amp;pages=252-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2009.11.052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2009.11.052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2009.11.052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19931533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Novoa</div>
               <div class="given-names">E.M.</div>
            </div>
             <div class="name">
              <div class="surname">Pavon-Eternod</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Pan</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">de Pouplana</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">A role for tRNA modifications in genome structure and codon usage,</div><div class="source">Cell</div> , <div class="volume">149</div>, <div class="fpage">202</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20tRNA%20modifications%20in%20genome%20structure%20and%20codon%20usage%2C&amp;author=E.M.%20Novoa&amp;author=M.%20Pavon-Eternod&amp;author=T.%20Pan&amp;author=L.%20de%20Pouplana&amp;publication_year=2012&amp;journal=Cell&amp;volume=149&amp;pages=202-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.01.050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.01.050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.01.050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22464330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nedialkova</div>
               <div class="given-names">D.D.</div>
            </div>
             <div class="name">
              <div class="surname">Leidel</div>
               <div class="given-names">S.A.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Optimization of codon translation rates via tRNA modifications maintains proteome integrity,</div><div class="source">Cell</div> , <div class="volume">161</div>, <div class="fpage">1606</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20of%20codon%20translation%20rates%20via%20tRNA%20modifications%20maintains%20proteome%20integrity%2C&amp;author=D.D.%20Nedialkova&amp;author=S.A.%20Leidel&amp;publication_year=2015&amp;journal=Cell&amp;volume=161&amp;pages=1606-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.05.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.05.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.05.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26052047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1981</div>, 
<div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">151</div>, <div class="fpage">389</div>–<div class="lpage">409</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system%2C&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1985</div>, 
<div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">2</div>, <div class="fpage">13</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms%2C&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.</div>
            </div>
          </div>
          <div class="year">2004</div>, 
<div class="article-title">Codon usage bias from tRNA’s point of view, redundancy, specialization, and efficient decoding for translation optimization,</div><div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">2279</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20from%20tRNA%E2%80%99s%20point%20of%20view%2C%20redundancy%2C%20specialization%2C%20and%20efficient%20decoding%20for%20translation%20optimization%2C&amp;author=E.P.%20Rocha&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=2279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2896904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2896904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2896904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Higgs</div>
               <div class="given-names">P.G.</div>
            </div>
             <div class="name">
              <div class="surname">Ran</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">Coevolution of codon usage and tRNA genes leads to alternative stable states of biased codon usage,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">2279</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coevolution%20of%20codon%20usage%20and%20tRNA%20genes%20leads%20to%20alternative%20stable%20states%20of%20biased%20codon%20usage%2C&amp;author=P.G.%20Higgs&amp;author=W.%20Ran&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=2279-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18687657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Shah</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Gilchrist</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift,</div><div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">108</div>, <div class="fpage">10231</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Explaining%20complex%20codon%20usage%20patterns%20with%20selection%20for%20translational%20efficiency%2C%20mutation%20bias%2C%20and%20genetic%20drift%2C&amp;author=P.%20Shah&amp;author=M.A.%20Gilchrist&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=108&amp;pages=10231-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1016719108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1016719108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1016719108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21646514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Katz</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Burge</div>
               <div class="given-names">C.B.</div>
            </div>
          </div>
          <div class="year">2003</div>, 
<div class="article-title">Wide spread selection for local RNA secondary structure in coding regions of bacterial genes,</div><div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2042</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wide%20spread%20selection%20for%20local%20RNA%20secondary%20structure%20in%20coding%20regions%20of%20bacterial%20genes%2C&amp;author=L.%20Katz&amp;author=C.B.%20Burge&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2042-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1257503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1257503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1257503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Wilke</div>
               <div class="given-names">C.O.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes,</div><div class="source">PLoS Comput. Biol</div> ., <div class="volume">6</div>, <div class="fpage">e1000664</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20universal%20trend%20of%20reduced%20mRNA%20stability%20near%20the%20translation-initiation%20site%20in%20prokaryotes%20and%20eukaryotes%2C&amp;author=W.%20Gu&amp;author=T.%20Zhou&amp;author=C.O.%20Wilke&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol&amp;volume=6&amp;pages=e1000664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zur</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Strong association between mRNA folding strength and protein abundance in <em>S. cerevisiae</em>,</div><div class="source">EMBO Rep</div> ., <div class="volume">13</div>, <div class="fpage">272</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20association%20between%20mRNA%20folding%20strength%20and%20protein%20abundance%20in%20S.%20cerevisiae%2C&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2012&amp;journal=EMBO%20Rep&amp;volume=13&amp;pages=272-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/embor.2011.262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fembor.2011.262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fembor.2011.262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22249164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Park</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">J.R.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2013</div>, 
<div class="article-title">Differential requirements for mRNA folding partially explain why highly expressed proteins evolve slowly,</div><div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">110</div>, <div class="fpage">E678</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20requirements%20for%20mRNA%20folding%20partially%20explain%20why%20highly%20expressed%20proteins%20evolve%20slowly%2C&amp;author=C.%20Park&amp;author=X.%20Chen&amp;author=J.R.%20Yang&amp;author=J.%20Zhang&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=110&amp;pages=E678-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1218066110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1218066110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1218066110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Presnyak</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Alhusaini</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">Y.H.</div>
            </div>
          </div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Codon optimality is a major determinant of mRNA stability,</div><div class="source">Cell</div> , <div class="volume">160</div>, <div class="fpage">1111</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20optimality%20is%20a%20major%20determinant%20of%20mRNA%20stability%2C&amp;author=V.%20Presnyak&amp;author=N.%20Alhusaini&amp;author=Y.H.%20Chen&amp;publication_year=2015&amp;journal=Cell&amp;volume=160&amp;pages=1111-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.02.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.02.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.02.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25768907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Fu</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Xiao</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Nonoptimal codon usage inﬂuences protein structure in intrinsically disordered regions,</div><div class="source">Mol. Microbiol</div> ., <div class="volume">97</div>, <div class="fpage">974</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonoptimal%20codon%20usage%20in%EF%AC%82uences%20protein%20structure%20in%20intrinsically%20disordered%20regions%2C&amp;author=M.%20Zhou&amp;author=T.%20Wang&amp;author=J.%20Fu&amp;author=G.%20Xiao&amp;author=Y.%20Liu&amp;publication_year=2015&amp;journal=Mol.%20Microbiol&amp;volume=97&amp;pages=974-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2015.97.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2015.97.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2015.97.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26032251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
             <div class="name">
              <div class="surname">Baruah</div>
               <div class="given-names">V.J.</div>
            </div>
             <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Banerjee</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Co-translational protein folding is revealing the selective use of synonymous codons along the coding sequence of a low expression gene,</div><div class="source">J. Genet</div> ., <div class="volume">93</div>, <div class="fpage">613</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-translational%20protein%20folding%20is%20revealing%20the%20selective%20use%20of%20synonymous%20codons%20along%20the%20coding%20sequence%20of%20a%20low%20expression%20gene%2C&amp;author=S.K.%20Ray&amp;author=V.J.%20Baruah&amp;author=S.S.%20Satapathy&amp;author=R.%20Banerjee&amp;publication_year=2014&amp;journal=J.%20Genet&amp;volume=93&amp;pages=613-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12041-014-0429-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12041-014-0429-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12041-014-0429-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25572219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nissley</div>
               <div class="given-names">D.A.</div>
            </div>
             <div class="name">
              <div class="surname">Sharma</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ahmed</div>
               <div class="given-names">N.</div>
            </div>
          </div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Accurate prediction of cellular co-translational folding indicates proteins can switch from post- to co-translational folding</div>, <div class="source">Nat. Commun</div> ., <div class="volume">7</div>, <div class="fpage">10341</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20prediction%20of%20cellular%20co-translational%20folding%20indicates%20proteins%20can%20switch%20from%20post-%20to%20co-translational%20folding&amp;author=D.A.%20Nissley&amp;author=A.K.%20Sharma&amp;author=N.%20Ahmed&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=10341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms10341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms10341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms10341"> </span></div> </div></div></div></div></div><div content-id="dsw027-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Buhr</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Jha</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Thommen</div>
               <div class="given-names">M.</div>
            </div>
          </div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Synonymous codons direct cotranslational folding toward different protein conformations,</div><div class="source">Mol. Cell</div> , <div class="volume">61</div>, <div class="fpage">341</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codons%20direct%20cotranslational%20folding%20toward%20different%20protein%20conformations%2C&amp;author=F.%20Buhr&amp;author=S.%20Jha&amp;author=M.%20Thommen&amp;publication_year=2016&amp;journal=Mol.%20Cell&amp;volume=61&amp;pages=341-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2016.01.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2016.01.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2016.01.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26849192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
             <div class="name">
              <div class="surname">Bailes</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Grocock</div>
               <div class="given-names">R.J.</div>
            </div>
             <div class="name">
              <div class="surname">Peden</div>
               <div class="given-names">J.F.</div>
            </div>
             <div class="name">
              <div class="surname">Sockett</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">2005</div>, 
<div class="article-title">Variation in the strength of selected codon usage bias among bacteria,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">1141</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20the%20strength%20of%20selected%20codon%20usage%20bias%20among%20bacteria%2C&amp;author=P.M.%20Sharp&amp;author=E.%20Bailes&amp;author=R.J.%20Grocock&amp;author=J.F.%20Peden&amp;author=R.E.%20Sockett&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=1141-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Dutta</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Buragohain</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Transfer RNA gene numbers may not be completely responsible for the codon usage bias in asparagine, isoleucine, phenylalanine and tyrosine in the high expression genes in bacteria,</div><div class="source">J. Mol. Evol</div> ., <div class="volume">75</div>, <div class="fpage">34</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20RNA%20gene%20numbers%20may%20not%20be%20completely%20responsible%20for%20the%20codon%20usage%20bias%20in%20asparagine%2C%20isoleucine%2C%20phenylalanine%20and%20tyrosine%20in%20the%20high%20expression%20genes%20in%20bacteria%2C&amp;author=S.S.%20Satapathy&amp;author=M.%20Dutta&amp;author=A.K.%20Buragohain&amp;author=S.K.%20Ray&amp;publication_year=2012&amp;journal=J.%20Mol.%20Evol&amp;volume=75&amp;pages=34-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-012-9524-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-012-9524-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-012-9524-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23053196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wald</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Alroy</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Botzman</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Margalit</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Codon usage bias in prokaryotic pyrimidine-ending codons is associated with the degeneracy of the encoded amino acids,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">7074</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20in%20prokaryotic%20pyrimidine-ending%20codons%20is%20associated%20with%20the%20degeneracy%20of%20the%20encoded%20amino%20acids%2C&amp;author=N.%20Wald&amp;author=M.%20Alroy&amp;author=M.%20Botzman&amp;author=H.%20Margalit&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=7074-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22581775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Powdel</div>
               <div class="given-names">B.R.</div>
            </div>
             <div class="name">
              <div class="surname">Dutta</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Buragohain</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Selection on GGU and CGU codons in the high expression genes in bacteria,</div><div class="source">J. Mol. Evol</div> ., <div class="volume">78</div>, <div class="fpage">13</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20GGU%20and%20CGU%20codons%20in%20the%20high%20expression%20genes%20in%20bacteria%2C&amp;author=S.S.%20Satapathy&amp;author=B.R.%20Powdel&amp;author=M.%20Dutta&amp;author=A.K.%20Buragohain&amp;author=S.K.%20Ray&amp;publication_year=2014&amp;journal=J.%20Mol.%20Evol&amp;volume=78&amp;pages=13-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-013-9596-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-013-9596-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-013-9596-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24271854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Shao</div>
               <div class="given-names">Z.-Q.</div>
            </div>
             <div class="name">
              <div class="surname">Xu</div>
               <div class="given-names">Y.</div>
            </div>
          </div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Optimal codon identities in bacteria: implications from the conflicting results of two different methods,</div><div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e22714</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimal%20codon%20identities%20in%20bacteria%3A%20implications%20from%20the%20conflicting%20results%20of%20two%20different%20methods%2C&amp;author=B.%20Wang&amp;author=Z.-Q.%20Shao&amp;author=Y.%20Xu&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e22714" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0022714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0022714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0022714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21829489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lowe</div>
               <div class="given-names">T.M.</div>
            </div>
             <div class="name">
              <div class="surname">Eddy</div>
               <div class="given-names">S.R.</div>
            </div>
          </div>
          <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence%2C&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">W.-H.</div>
            </div>
          </div>
          <div class="year">1986</div>, 
<div class="article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms,</div><div class="source">J. Mol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">28</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20perspective%20on%20synonymous%20codon%20usage%20in%20unicellular%20organisms%2C&amp;author=P.M.%20Sharp&amp;author=W.-H.%20Li&amp;publication_year=1986&amp;journal=J.%20Mol.%20Evol&amp;volume=24&amp;pages=28-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02099948" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02099948" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02099948"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3104616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Iriarte</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Baraibar</div>
               <div class="given-names">J.D.</div>
            </div>
             <div class="name">
              <div class="surname">Romero</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Castro-Sowinski</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Musto</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2013</div>, 
<div class="article-title">Evolution of optimal codon choices in the family Enterobacteriaceae,</div><div class="source">Microbiology</div> , <div class="volume">159</div>, <div class="fpage">555</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20optimal%20codon%20choices%20in%20the%20family%20Enterobacteriaceae%2C&amp;author=A.%20Iriarte&amp;author=J.D.%20Baraibar&amp;author=H.%20Romero&amp;author=S.%20Castro-Sowinski&amp;author=H.%20Musto&amp;publication_year=2013&amp;journal=Microbiology&amp;volume=159&amp;pages=555-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.061952-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.061952-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.061952-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23288542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Akashi</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2002</div>, 
<div class="article-title">Translational selection and yeast proteome evolution,</div><div class="source">Genetics</div> , <div class="volume">164</div>, <div class="fpage">1291</div>–<div class="lpage">303</div>.</div></div></div></div><div content-id="dsw027-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Shah</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Gilchrist</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">Effect of correlated tRAN abundances on translation errors and evolution of codon usage bias,</div><div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1001128</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20correlated%20tRAN%20abundances%20on%20translation%20errors%20and%20evolution%20of%20codon%20usage%20bias%2C&amp;author=P.%20Shah&amp;author=M.A.%20Gilchrist&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20862306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gilchrist</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Shah</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Zaretzki</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2009</div>, 
<div class="article-title">Measuring and detecting molecular adaptation in codon usage against non-sense errors during protein translation,</div><div class="source">Genetics</div> , <div class="volume">183</div>, <div class="fpage">1493</div>–<div class="lpage">505</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measuring%20and%20detecting%20molecular%20adaptation%20in%20codon%20usage%20against%20non-sense%20errors%20during%20protein%20translation%2C&amp;author=M.A.%20Gilchrist&amp;author=P.%20Shah&amp;author=R.%20Zaretzki&amp;publication_year=2009&amp;journal=Genetics&amp;volume=183&amp;pages=1493-505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.108209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.108209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.108209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19822731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sørensen</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Kurland</div>
               <div class="given-names">C.G.</div>
            </div>
             <div class="name">
              <div class="surname">Pedersen</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1989</div>, 
<div class="article-title">Codon usage determines the translation rate in <em>Escherichia coli</em>,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">207</div>, <div class="fpage">365</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20determines%20the%20translation%20rate%20in%20Escherichia%20coli%2C&amp;author=M.A.%20S%C3%B8rensen&amp;author=C.G.%20Kurland&amp;author=S.%20Pedersen&amp;publication_year=1989&amp;journal=J.%20Mol.%20Biol&amp;volume=207&amp;pages=365-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(89)90260-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(89)90260-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(89)90260-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2474074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sørensen</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Pedersen</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1991</div>, 
<div class="article-title">Absolute <em>in vivo</em> translation rates of individual codons in <em>Escherichia coli</em> the two glutamic acid codons, GAA and GAG are translated with threefold differences in rate,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">222</div>, <div class="fpage">265</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absolute%20in%20vivo%20translation%20rates%20of%20individual%20codons%20in%20Escherichia%20coli%20the%20two%20glutamic%20acid%20codons%2C%20GAA%20and%20GAG%20are%20translated%20with%20threefold%20differences%20in%20rate%2C&amp;author=M.A.%20S%C3%B8rensen&amp;author=S.%20Pedersen&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol&amp;volume=222&amp;pages=265-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90211-N" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90211-N" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90211-N"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1960727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Prajapati</div>
               <div class="given-names">V.K.</div>
            </div>
             <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Satish Kumar</div>
               <div class="given-names">M.V.</div>
            </div>
             <div class="name">
              <div class="surname">Buragohain</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Evidences indicating the involvement of selection mechanisms for the occurrence of C34 anticodon in bacteria</div>, <div class="source">J. Cell Sci. Mol. Biol</div> ., <div class="volume">2</div>, <div class="fpage">112</div>.</div></div></div></div><div content-id="dsw027-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Krüger</div>
               <div class="given-names">M.K.</div>
            </div>
             <div class="name">
              <div class="surname">Pedersen</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Hagervall</div>
               <div class="given-names">T.G.</div>
            </div>
             <div class="name">
              <div class="surname">Sørensen</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="year">1998</div>, 
<div class="article-title">The modification of the wobble base of tRNA<sup>Glu</sup> modulates the translation rate of glutamic acid codon <em>in vivo</em>,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">284</div>, <div class="fpage">621</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20modification%20of%20the%20wobble%20base%20of%20tRNAGlu%20modulates%20the%20translation%20rate%20of%20glutamic%20acid%20codon%20in%20vivo%2C&amp;author=M.K.%20Kr%C3%BCger&amp;author=S.%20Pedersen&amp;author=T.G.%20Hagervall&amp;author=M.A.%20S%C3%B8rensen&amp;publication_year=1998&amp;journal=J.%20Mol.%20Biol&amp;volume=284&amp;pages=621-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1998.2196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1998.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1998.2196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9826503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gustilo</div>
               <div class="given-names">E.M.</div>
            </div>
             <div class="name">
              <div class="surname">Vendeix</div>
               <div class="given-names">F.A.P.</div>
            </div>
             <div class="name">
              <div class="surname">Agris</div>
               <div class="given-names">P.F.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">tRNA’s modifications bring order to gene expression,</div><div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">11</div>, <div class="fpage">134</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%E2%80%99s%20modifications%20bring%20order%20to%20gene%20expression%2C&amp;author=E.M.%20Gustilo&amp;author=F.A.P.%20Vendeix&amp;author=P.F.%20Agris&amp;publication_year=2008&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=11&amp;pages=134-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2008.02.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2008.02.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2008.02.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18378185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Suzuki</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Numata</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Convergent evolution of AUA decoding in bacteria and archaea,</div><div class="source">RNA Biol</div> ., <div class="volume">11</div>, <div class="fpage">1586</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Convergent%20evolution%20of%20AUA%20decoding%20in%20bacteria%20and%20archaea%2C&amp;author=T.%20Suzuki&amp;author=T.%20Numata&amp;publication_year=2014&amp;journal=RNA%20Biol&amp;volume=11&amp;pages=1586-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/15476286.2014.992281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2F15476286.2014.992281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2F15476286.2014.992281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25629511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Powdel</div>
               <div class="given-names">B.R.</div>
            </div>
             <div class="name">
              <div class="surname">Dutta</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Buragohain</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Constraint on dinucleotides by codon usage bias in bacterial genomes,</div><div class="source">Gene</div> , <div class="volume">536</div>, <div class="fpage">18</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constraint%20on%20dinucleotides%20by%20codon%20usage%20bias%20in%20bacterial%20genomes%2C&amp;author=S.S.%20Satapathy&amp;author=B.R.%20Powdel&amp;author=M.%20Dutta&amp;author=A.K.%20Buragohain&amp;author=S.K.%20Ray&amp;publication_year=2014&amp;journal=Gene&amp;volume=536&amp;pages=18-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2013.11.098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2013.11.098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2013.11.098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24333347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">O’Çonnor</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1998</div>, 
<div class="article-title">tRNA imbalance promotes -1 frameshifting via near-cognate decoding</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">279</div>, <div class="fpage">727</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20imbalance%20promotes%20-1%20frameshifting%20via%20near-cognate%20decoding&amp;author=M.%20O%E2%80%99%C3%87onnor&amp;publication_year=1998&amp;journal=J.%20Mol.%20Biol&amp;volume=279&amp;pages=727-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1998.1832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1998.1832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1998.1832"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9642056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">O’Çonnor</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2002</div>, 
<div class="article-title">Imbalance of tRNA<sup>Pro</sup> isoacceptors induces +1 frameshifting at near-cognate codons</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">759</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Imbalance%20of%20tRNAPro%20isoacceptors%20induces%20%2B1%20frameshifting%20at%20near-cognate%20codons&amp;author=M.%20O%E2%80%99%C3%87onnor&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=759-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.3.759" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.3.759" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.3.759"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11809889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tarrant</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">von der Haar</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Synonymous codons, ribosome speed, and eukaryotic gene expression regulation,</div><div class="source">Cell Mol. Life Sci</div> ., <div class="volume">71</div>, <div class="fpage">4195</div>–<div class="lpage">206</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codons%2C%20ribosome%20speed%2C%20and%20eukaryotic%20gene%20expression%20regulation%2C&amp;author=D.%20Tarrant&amp;author=T.%20von%20der%20Haar&amp;publication_year=2014&amp;journal=Cell%20Mol.%20Life%20Sci&amp;volume=71&amp;pages=4195-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-014-1684-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-014-1684-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-014-1684-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25038778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.</div>
            </div>
          </div>
          <div class="year">2004</div>, 
<div class="article-title">The replication-related organization of bacterial genomes,</div><div class="source">Microbiology</div> , <div class="volume">150</div>, <div class="fpage">1609</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20replication-related%20organization%20of%20bacterial%20genomes%2C&amp;author=E.P.%20Rocha&amp;publication_year=2004&amp;journal=Microbiology&amp;volume=150&amp;pages=1609-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.26974-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.26974-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.26974-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15184548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Muto</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Osawa</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1987</div>, 
<div class="article-title">The guanine and cytosine content of genomic DNA and bacterial evolution,</div><div class="source">Proc. Natl Acad. Sci. U. S. A</div> ., <div class="volume">84</div>, <div class="fpage">166</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20guanine%20and%20cytosine%20content%20of%20genomic%20DNA%20and%20bacterial%20evolution%2C&amp;author=A.%20Muto&amp;author=S.%20Osawa&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=84&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.1.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.1.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.1.166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3467347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">S.L.</div>
            </div>
             <div class="name">
              <div class="surname">Lee</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Hotts</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Shapiro</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">McAdams</div>
               <div class="given-names">H.H.</div>
            </div>
          </div>
          <div class="year">2004</div>, 
<div class="article-title">Codon usage between genomes is constrained by genome wide mutational processes,</div><div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">101</div>, <div class="fpage">3480</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20between%20genomes%20is%20constrained%20by%20genome%20wide%20mutational%20processes%2C&amp;author=S.L.%20Chen&amp;author=W.%20Lee&amp;author=A.K.%20Hotts&amp;author=L.%20Shapiro&amp;author=H.H.%20McAdams&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=101&amp;pages=3480-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307827100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307827100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307827100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14990797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Palidwor</div>
               <div class="given-names">G.A.</div>
            </div>
             <div class="name">
              <div class="surname">Perkins</div>
               <div class="given-names">T.J.</div>
            </div>
             <div class="name">
              <div class="surname">Xia</div>
               <div class="given-names">X.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">A general model of codon bias due to GC mutational bias,</div><div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e13431</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20model%20of%20codon%20bias%20due%20to%20GC%20mutational%20bias%2C&amp;author=G.A.%20Palidwor&amp;author=T.J.%20Perkins&amp;author=X.%20Xia&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e13431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Brégeon</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Colot</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Radman</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Taddei</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2001</div>. 
<div class="article-title">Translational misreading: a tRNA modification counteracts a + 2 ribosomal frameshift,</div><div class="source">Genes Dev</div> ., <div class="volume">15</div>, <div class="fpage">2295</div>–<div class="lpage">306</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20misreading%3A%20a%20tRNA%20modification%20counteracts%20a%20%2B%202%20ribosomal%20frameshift%2C&amp;author=D.%20Br%C3%A9geon&amp;author=V.%20Colot&amp;author=M.%20Radman&amp;author=F.%20Taddei&amp;publication_year=2001&amp;journal=Genes%20Dev&amp;volume=15&amp;pages=2295-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.207701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.207701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.207701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11544186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Agris</div>
               <div class="given-names">P.F.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">Bringing order to translation: the contributions of transfer RNA anticodon-domain modifications,</div><div class="source">EMBO Rep</div> ., <div class="volume">9</div>, <div class="fpage">629</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bringing%20order%20to%20translation%3A%20the%20contributions%20of%20transfer%20RNA%20anticodon-domain%20modifications%2C&amp;author=P.F.%20Agris&amp;publication_year=2008&amp;journal=EMBO%20Rep&amp;volume=9&amp;pages=629-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/embor.2008.104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fembor.2008.104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fembor.2008.104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18552770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Armengod</div>
               <div class="given-names">M.E.</div>
            </div>
             <div class="name">
              <div class="surname">Meseguer</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Villarroya</div>
               <div class="given-names">M.</div>
            </div>
          </div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Modification of the wobble uridine in bacterial and mitochondrial tRNAs reading NNA/NNG triplets of 2-codon boxes,</div><div class="source">RNA Biol</div> ., <div class="volume">11</div>, <div class="fpage">1495</div>–<div class="lpage">507</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modification%20of%20the%20wobble%20uridine%20in%20bacterial%20and%20mitochondrial%20tRNAs%20reading%20NNA%2FNNG%20triplets%20of%202-codon%20boxes%2C&amp;author=M.E.%20Armengod&amp;author=S.%20Meseguer&amp;author=M.%20Villarroya&amp;publication_year=2014&amp;journal=RNA%20Biol&amp;volume=11&amp;pages=1495-507" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/15476286.2014.992269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2F15476286.2014.992269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2F15476286.2014.992269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25607529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hou</div>
               <div class="given-names">Y.-M.</div>
            </div>
             <div class="name">
              <div class="surname">Gamper</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Post-transcriptional modifications to tRNA – a response to the genetic code degeneracy,</div><div class="source">RNA</div> , <div class="volume">21</div>, <div class="fpage">642</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Post-transcriptional%20modifications%20to%20tRNA%20%E2%80%93%20a%20response%20to%20the%20genetic%20code%20degeneracy%2C&amp;author=Y.-M.%20Hou&amp;author=H.%20Gamper&amp;author=W.%20Yang&amp;publication_year=2015&amp;journal=RNA&amp;volume=21&amp;pages=642-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.049825.115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.049825.115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.049825.115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25780173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Campbell</div>
               <div class="given-names">A.M.</div>
            </div>
             <div class="name">
              <div class="surname">Mrázek</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">1998</div>, 
<div class="article-title">Comparative DNA analysis across diverse genomes,</div><div class="source">Annu. Rev. Genet</div> ., <div class="volume">32</div>, <div class="fpage">185</div>–<div class="lpage">225</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20DNA%20analysis%20across%20diverse%20genomes%2C&amp;author=S.%20Karlin&amp;author=A.M.%20Campbell&amp;author=J.%20Mr%C3%A1zek&amp;publication_year=1998&amp;journal=Annu.%20Rev.%20Genet&amp;volume=32&amp;pages=185-225" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.32.1.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.32.1.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.32.1.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9928479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Supek</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Škunca</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Repar</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Vlahoviček</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Šmuc</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">Translational selection is ubiquitous in prokaryotes,</div><div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1001004</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20selection%20is%20ubiquitous%20in%20prokaryotes%2C&amp;author=F.%20Supek&amp;author=N.%20%C5%A0kunca&amp;author=J.%20Repar&amp;author=K.%20Vlahovi%C4%8Dek&amp;author=T.%20%C5%A0muc&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001004" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20585573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yona</div>
               <div class="given-names">A.H.</div>
            </div>
             <div class="name">
              <div class="surname">Bloom-Ackermann</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Frumkin</div>
               <div class="given-names">I.</div>
            </div>
          </div>, et al. <div class="year">2013</div>, 
<div class="article-title">tRNA genes rapidly change in evolution to meet novel translational demands,</div><div class="source">eLife</div> , <div class="volume">2</div>, <div class="fpage">e01339</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20genes%20rapidly%20change%20in%20evolution%20to%20meet%20novel%20translational%20demands%2C&amp;author=A.H.%20Yona&amp;author=Z.%20Bloom-Ackermann&amp;author=I.%20Frumkin&amp;publication_year=2013&amp;journal=eLife&amp;volume=2&amp;pages=e01339" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.01339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.01339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.01339"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24363105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Endres</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Dedon</div>
               <div class="given-names">P.C.</div>
            </div>
             <div class="name">
              <div class="surname">Begley</div>
               <div class="given-names">T.J.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Codon-biased translation can be regulated by wobble-base tRNA modification systems during cellular stress responses,</div><div class="source">RNA Biol</div> ., <div class="volume">12</div>, <div class="fpage">603</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon-biased%20translation%20can%20be%20regulated%20by%20wobble-base%20tRNA%20modification%20systems%20during%20cellular%20stress%20responses%2C&amp;author=L.%20Endres&amp;author=P.C.%20Dedon&amp;author=T.J.%20Begley&amp;publication_year=2015&amp;journal=RNA%20Biol&amp;volume=12&amp;pages=603-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15476286.2015.1031947" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15476286.2015.1031947" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15476286.2015.1031947"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25892531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Holtkamp</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Kokic</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Jäger</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Mittelstät</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Komar</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Rodnina</div>
               <div class="given-names">M.V.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Cotranslational protein folding on the ribosome monitored in real time,</div><div class="source">Science</div> , <div class="volume">350</div>, <div class="fpage">1104</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cotranslational%20protein%20folding%20on%20the%20ribosome%20monitored%20in%20real%20time%2C&amp;author=W.%20Holtkamp&amp;author=G.%20Kokic&amp;author=M.%20J%C3%A4ger&amp;author=J.%20Mittelst%C3%A4t&amp;author=A.%20Komar&amp;author=M.V.%20Rodnina&amp;publication_year=2015&amp;journal=Science&amp;volume=350&amp;pages=1104-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aad0344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad0344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aad0344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26612953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Angov</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Hiller</div>
               <div class="given-names">C.J.</div>
            </div>
             <div class="name">
              <div class="surname">Kincaid</div>
               <div class="given-names">R.L.</div>
            </div>
             <div class="name">
              <div class="surname">Lyon</div>
               <div class="given-names">J.A.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">Heterologous protein expression is enhanced by harmonizing the codon usage frequencies of the target gene with those of the expression hosts,</div><div class="source">PLoS One</div> , <div class="volume">3</div>, <div class="fpage">e2189</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterologous%20protein%20expression%20is%20enhanced%20by%20harmonizing%20the%20codon%20usage%20frequencies%20of%20the%20target%20gene%20with%20those%20of%20the%20expression%20hosts%2C&amp;author=E.%20Angov&amp;author=C.J.%20Hiller&amp;author=R.L.%20Kincaid&amp;author=J.A.%20Lyon&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e2189" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0002189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0002189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0002189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18478103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Angov</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">Codon usage: nature’s roadmap to expression and folding of proteins,</div><div class="source">Biotechnol. J</div> ., <div class="volume">6</div>, <div class="fpage">650</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%3A%20nature%E2%80%99s%20roadmap%20to%20expression%20and%20folding%20of%20proteins%2C&amp;author=E.%20Angov&amp;publication_year=2011&amp;journal=Biotechnol.%20J&amp;volume=6&amp;pages=650-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/biot.201000332" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbiot.201000332" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbiot.201000332"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21567958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Baruah</div>
               <div class="given-names">V.J.</div>
            </div>
             <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Powdel</div>
               <div class="given-names">B.R.</div>
            </div>
             <div class="name">
              <div class="surname">Konwarh</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Buragohain</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Comparative analysis of codon usage bias in Crenarchaea and Euryarchaea genome reveals differential preference of synonymous codons to encode highly expressed ribosomal and RNA polymerase proteins</div>, <div class="source">J. Genet</div> . (In press).</div></div></div></div><div content-id="dsw027-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Satapathy</div>
               <div class="given-names">S.S.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">S.K.</div>
            </div>
             <div class="name">
              <div class="surname">Sahoo</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Begum</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Ghosh</div>
               <div class="given-names">T.C.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Codon usage bias is not significantly different between the high and the low expression genes in human</div>, <div class="source">Int. J. Mol. Genet. Gene Ther</div> ., <div class="volume">1</div>, <div class="fpage">1</div>.</div></div></div></div><div content-id="dsw027-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Babbitt</div>
               <div class="given-names">G.A.</div>
            </div>
             <div class="name">
              <div class="surname">Alawad</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Schulze</div>
               <div class="given-names">K.V.</div>
            </div>
             <div class="name">
              <div class="surname">Hudson</div>
               <div class="given-names">A.O.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Synonymous codon bias and functional constraint on GC3-related DNA backbone dynamics in the prokaryotic nucleoid,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">10915</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20bias%20and%20functional%20constraint%20on%20GC3-related%20DNA%20backbone%20dynamics%20in%20the%20prokaryotic%20nucleoid%2C&amp;author=G.A.%20Babbitt&amp;author=M.A.%20Alawad&amp;author=K.V.%20Schulze&amp;author=A.O.%20Hudson&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=10915-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku811" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25200075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw027-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw027-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Thomas</div>
               <div class="given-names">L.K.</div>
            </div>
             <div class="name">
              <div class="surname">Dix</div>
               <div class="given-names">D.B.</div>
            </div>
             <div class="name">
              <div class="surname">Thompson</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="year">1988</div>, 
<div class="article-title">Codon choice and gene expression: synonymous codons differ in their ability to direct aminoacylated-transfer RNA binding to ribosomes <em>in vitro</em>,</div><div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">85</div>, <div class="fpage">4242</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20choice%20and%20gene%20expression%3A%20synonymous%20codons%20differ%20in%20their%20ability%20to%20direct%20aminoacylated-transfer%20RNA%20binding%20to%20ribosomes%20in%20vitro%2C&amp;author=L.K.%20Thomas&amp;author=D.B.%20Dix&amp;author=R.C.%20Thompson&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=85&amp;pages=4242-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.12.4242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.12.4242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.12.4242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3288988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2236160&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw027/1/dsw027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4oQ3rkI3AIMlcUk1mK6dO3bx8ATBW3a5-AKLzvRprtI3X8M9c6QFDboUmFpBJBfEwMh43DhtVTDqgI-LYEMOMnrSjH7HQwcMvv96NX-H3V5D4u2ICgCtguEjH46DLa~YbvesignIO4xe4AyX0WUlPFJqCzxVZFOyOVrPY5kHpXi9ikuV~56nwRcmJfpEk-Jo1hMZ~~IMZLUbNF-qUzwAK5R8gVIsJktpu52xMnpnB2cf5dn40lz~2kA7YV03pmDjQc07O4c5~IeiUNays14LwT1-jT-AlkcSDiR0UEX0Jjh-JZ4a0ZF7BVyJrpNZUMoS3FqUIUkdOxF7EebqtyFyFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">402</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000386964800003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw027"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2236160"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000386964800003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f5%2f441%2f2236160">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000386964800003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F5%2F441%2F2236160" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2236160'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Discrepancy among the synonymous codons with respect to their selection as optimal codon in bacteria" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw027");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f441%2f2236160">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

