

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/5/277/359806","@type":"ScholarlyArticle","name":"Superpositioning of Deletions Promotes Growth of  Escherichia coli  with a Reduced Genome","datePublished":"2008-08-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2008-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn019","inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/5/277/359806","author":[{"name":"Mizoguchi, Hiroshi","affiliation":"1  Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533 ,  Japan","@type":"Person"},{"name":"Sawano, Yoshie","affiliation":"1  Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533 ,  Japan","@type":"Person"},{"name":"Kato, Jun-ichi","affiliation":"2  Department of Biological Sciences, Graduate Schools of Science and Engineering ,  Tokyo Metropolitan University ,  Minamiohsawa, Hachioji, Tokyo 192-0397 ,  Japan","@type":"Person"},{"name":"Mori, Hideo","affiliation":"1  Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533 ,  Japan","@type":"Person"}],"description":"Abstract.  Escherichia coli has dispensable genome regions and eliminating them may improve cell use by reducing unnecessary metabolic pathways and complex regu","pageStart":"277","pageEnd":"284","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Superpositioning of Deletions Promotes Growth of  Escherichia coli  with a Reduced Genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Superpositioning of Deletions Promotes Growth of  Escherichia coli  with a Reduced Genome" />
    <meta property="og:description" content="Abstract.  Escherichia coli has dispensable genome regions and eliminating them may improve cell use by reducing unnecessary metabolic pathways and complex regu" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/5/277/359806" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-08-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Mizoguchi, Hiroshi" /><meta name="citation_author_institution" content="Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533, Japan" /><meta name="citation_author" content="Sawano, Yoshie" /><meta name="citation_author_institution" content="Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533, Japan" /><meta name="citation_author" content="Kato, Jun-ichi" /><meta name="citation_author_institution" content="Department of Biological Sciences, Graduate Schools of Science and Engineering, Tokyo Metropolitan University, Minamiohsawa, Hachioji, Tokyo 192-0397, Japan" /><meta name="citation_author" content="Mori, Hideo" /><meta name="citation_author_institution" content="Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533, Japan" /><meta name="citation_title" content="Superpositioning of Deletions Promotes Growth of Escherichia coli with a Reduced Genome" /><meta name="citation_firstpage" content="277" /><meta name="citation_lastpage" content="284" /><meta name="citation_doi" content="10.1093/dnares/dsn019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2008/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Direct expression in Escherichia coli of a DNA sequence coding for human growth hormone; Nature;  citation_year=1979;  citation_volume=281;  citation_pages=544-548; " /><meta name="citation_reference" content="citation_title=Isolation of the arginine repressor in Escherichia coli; Nature;  citation_year=1970;  citation_volume=228;  citation_pages=336-338; " /><meta name="citation_reference" content="citation_title=Amino acid production processes; Adv. Biochem. Eng. Biotechnol.;  citation_year=2003;  citation_volume=79;  citation_pages=1-35; " /><meta name="citation_reference" content="citation_title=Enzymatic production of trans-4-hydroxy-l-proline by region- and sterospecific hydroxylation of L-proline; Biosci. Biotechnol. Biochem.;  citation_year=2000;  citation_volume=64;  citation_pages=746-750; " /><meta name="citation_reference" content="citation_title=Improving 1,3-propanediol production from glycerol in a metabolically engineered Escherichia coli by reducing accumulation of sn-glycerol-3-phosphate; Biotechnol. Prog.;  citation_year=2002;  citation_volume=18;  citation_pages=694-699; " /><meta name="citation_reference" content="citation_title=Production of l-threonine by auxotrophs and analogue-resistant mutants of Escherichia coli.; Kor. J. Appl. Microbiol. Biotechnol.;  citation_year=1991;  citation_volume=19;  citation_pages=583-587; " /><meta name="citation_reference" content="citation_title=Escherichia coli K-12: a cooperatively developed annotation snapshot—2005; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Minimization of the Escherichia coli genome using a Tn5-targeted Cre/loxP excision system; Nat. Biotechnol.;  citation_year=2002;  citation_volume=20;  citation_pages=1018-1023; " /><meta name="citation_reference" content="citation_title=Chromosomal deletion formation system based on Tn5 double transposition: use for making minimal genomes and essential gene analysis; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=644-653; " /><meta name="citation_reference" content="citation_title=Cell size and nucleoid organization of engineered Escherichia coli cells with a reduced genome; Mol. Microbiol.;  citation_year=2005;  citation_volume=55;  citation_pages=137-149; " /><meta name="citation_reference" content="citation_title=Emergent properties of reduced-genome Escherichia coli; Science;  citation_year=2006;  citation_volume=312;  citation_pages=1044-1046; " /><meta name="citation_reference" content="citation_title=Extensive genomic diversity in pathogenic Escherichia coli and Shigella strains revealed by comparative genomic hybridization microarray; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=3911-3921; " /><meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection; citation_text=2006.0008.; Mol. Syst. Biol.;  citation_year=2006;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=Use of bacteriophage lambda recombination functions to promote gene replacement in Escherichia coli; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=2063-2071; " /><meta name="citation_reference" content="citation_title=Global analyses of transcriptomes and proteomes of a parent strain and an l-threonine-overproducing mutant strain; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=5442-5451; " /><meta name="citation_reference" content=" citation_edition=3rd Ed.; citation_publisher=Cold Spring Harbor Laboratory Press, New York; Molecular Cloning: a Laboratory Manual;  citation_year=1989; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12; Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-1474; " /><meta name="citation_reference" content="citation_title=Genome sequence of the endocellular bacterial symbiont of aphid Buchnera sp. APS; Nature;  citation_year=2000;  citation_volume=407;  citation_pages=81-86; " /><meta name="citation_reference" content="citation_title=Construction of consecutive deletions of the Escherichia coli chromosome; Mol. Syst. Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=132" /><meta name="citation_reference" content="citation_title=The ERGO genome analysis and discovery system; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=164-171; " /><meta name="citation_reference" content="citation_title=A simple method for multiple modification of the Escherichia coli K-12 chromosome; Biosci. Biotechnol. Biochem.;  citation_year=2007;  citation_volume=71;  citation_pages=2905-2911; " /><meta name="citation_reference" content="citation_title=Chromosome partitioning in Escherichia coli: novel mutants producing anucleate cells; J. Bacteriol.;  citation_year=1989;  citation_volume=171;  citation_pages=1496-1505; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of deoxyadenosine methyltransferase-mediated control of gene expression in Escherichia coli; Mol. Microbiol.;  citation_year=2002;  citation_volume=45;  citation_pages=673-695; " /><meta name="citation_reference" content="citation_title=Comparison of growth, acetate production, and acetate inhibition of Escherichia coli strains in batch and fed-batch fermentations; Appl. Environ. Microbiol.;  citation_year=1990;  citation_volume=56;  citation_pages=1004-1011; " /><meta name="citation_reference" content="citation_title=Investigation of the TCA cycle and the glyoxylate shunt in Escherichia coli BL21 and JM109 using (13)C-NMR/MS; Biotechnol. Bioeng.;  citation_year=2000;  citation_volume=68;  citation_pages=316-327; " /><meta name="citation_reference" content="citation_title=Tricarboxylic acid cycle and glyoxylate bypass; citation_publisher=American Society for Microbiology, Washington D. C.; Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd Ed.;  citation_year=1996;  citation_pages=206-216; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/5/277/1030025/dsn019.pdf" /><meta name="description" content="Abstract.  Escherichia coli has dispensable genome regions and eliminating them may improve cell use by reducing unnecessary metabolic pathways and complex regu" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/5/277/359806" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/5/277/359806" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/5/277/359806");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn019';
    </script>
<title>
	Superpositioning of Deletions Promotes Growth of Escherichia coli with a Reduced Genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359806" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfQ/PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXpvZ3VjaGksIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9mcm9udGllciBMYWJvcmF0b3JpZXMsIEt5b3dhIEhha2tvIEtvZ3lvIENvLiBMdGQsIDMtNi02IEFzYWhpbWFjaGksIE1hY2hpZGFzaGksIFRva3lvIDE5NC04NTMzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhd2FubywgWW9zaGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZnJvbnRpZXIgTGFib3JhdG9yaWVzLCBLeW93YSBIYWtrbyBLb2d5byBDby4gTHRkLCAzLTYtNiBBc2FoaW1hY2hpLCBNYWNoaWRhc2hpLCBUb2t5byAxOTQtODUzMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRvLCBKdW4taWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29scyBvZiBTY2llbmNlIGFuZCBFbmdpbmVlcmluZywgVG9reW8gTWV0cm9wb2xpdGFuIFVuaXZlcnNpdHksIE1pbmFtaW9oc2F3YSwgSGFjaGlvamksIFRva3lvIDE5Mi0wMzk3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmksIEhpZGVvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZnJvbnRpZXIgTGFib3JhdG9yaWVzLCBLeW93YSBIYWtrbyBLb2d5byBDby4gTHRkLCAzLTYtNiBBc2FoaW1hY2hpLCBNYWNoaWRhc2hpLCBUb2t5byAxOTQtODUzMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlN1cGVycG9zaXRpb25pbmcgb2YgRGVsZXRpb25zIFByb21vdGVzIEdyb3d0aCBvZiBFc2NoZXJpY2hpYSBjb2xpIHdpdGggYSBSZWR1Y2VkIEdlbm9tZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjg0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3QgZXhwcmVzc2lvbiBpbiBFc2NoZXJpY2hpYSBjb2xpIG9mIGEgRE5BIHNlcXVlbmNlIGNvZGluZyBmb3IgaHVtYW4gZ3Jvd3RoIGhvcm1vbmU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk3OTsgIGNpdGF0aW9uX3ZvbHVtZT0yODE7ICBjaXRhdGlvbl9wYWdlcz01NDQtNTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gb2YgdGhlIGFyZ2luaW5lIHJlcHJlc3NvciBpbiBFc2NoZXJpY2hpYSBjb2xpOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5NzA7ICBjaXRhdGlvbl92b2x1bWU9MjI4OyAgY2l0YXRpb25fcGFnZXM9MzM2LTMzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm8gYWNpZCBwcm9kdWN0aW9uIHByb2Nlc3NlczsgQWR2LiBCaW9jaGVtLiBFbmcuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9MS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW56eW1hdGljIHByb2R1Y3Rpb24gb2YgdHJhbnMtNC1oeWRyb3h5LWwtcHJvbGluZSBieSByZWdpb24tIGFuZCBzdGVyb3NwZWNpZmljIGh5ZHJveHlsYXRpb24gb2YgTC1wcm9saW5lOyBCaW9zY2kuIEJpb3RlY2hub2wuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9NzQ2LTc1MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92aW5nIDEsMy1wcm9wYW5lZGlvbCBwcm9kdWN0aW9uIGZyb20gZ2x5Y2Vyb2wgaW4gYSBtZXRhYm9saWNhbGx5IGVuZ2luZWVyZWQgRXNjaGVyaWNoaWEgY29saSBieSByZWR1Y2luZyBhY2N1bXVsYXRpb24gb2Ygc24tZ2x5Y2Vyb2wtMy1waG9zcGhhdGU7IEJpb3RlY2hub2wuIFByb2cuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9Njk0LTY5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZHVjdGlvbiBvZiBsLXRocmVvbmluZSBieSBhdXhvdHJvcGhzIGFuZCBhbmFsb2d1ZS1yZXNpc3RhbnQgbXV0YW50cyBvZiBFc2NoZXJpY2hpYSBjb2xpLjsgS29yLiBKLiBBcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTgzLTU4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjaGVyaWNoaWEgY29saSBLLTEyOiBhIGNvb3BlcmF0aXZlbHkgZGV2ZWxvcGVkIGFubm90YXRpb24gc25hcHNob3TigJQyMDA1OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmltaXphdGlvbiBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBnZW5vbWUgdXNpbmcgYSBUbjUtdGFyZ2V0ZWQgQ3JlL2xveFAgZXhjaXNpb24gc3lzdGVtOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTAxOC0xMDIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBkZWxldGlvbiBmb3JtYXRpb24gc3lzdGVtIGJhc2VkIG9uIFRuNSBkb3VibGUgdHJhbnNwb3NpdGlvbjogdXNlIGZvciBtYWtpbmcgbWluaW1hbCBnZW5vbWVzIGFuZCBlc3NlbnRpYWwgZ2VuZSBhbmFseXNpczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NDQtNjUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZWxsIHNpemUgYW5kIG51Y2xlb2lkIG9yZ2FuaXphdGlvbiBvZiBlbmdpbmVlcmVkIEVzY2hlcmljaGlhIGNvbGkgY2VsbHMgd2l0aCBhIHJlZHVjZWQgZ2Vub21lOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0xMzctMTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbWVyZ2VudCBwcm9wZXJ0aWVzIG9mIHJlZHVjZWQtZ2Vub21lIEVzY2hlcmljaGlhIGNvbGk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzEyOyAgY2l0YXRpb25fcGFnZXM9MTA0NC0xMDQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgZ2Vub21pYyBkaXZlcnNpdHkgaW4gcGF0aG9nZW5pYyBFc2NoZXJpY2hpYSBjb2xpIGFuZCBTaGlnZWxsYSBzdHJhaW5zIHJldmVhbGVkIGJ5IGNvbXBhcmF0aXZlIGdlbm9taWMgaHlicmlkaXphdGlvbiBtaWNyb2FycmF5OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTM5MTEtMzkyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIEVzY2hlcmljaGlhIGNvbGkgSy0xMiBpbi1mcmFtZSwgc2luZ2xlLWdlbmUga25vY2tvdXQgbXV0YW50czogdGhlIEtlaW8gY29sbGVjdGlvbjsgY2l0YXRpb25fdGV4dD0yMDA2LjAwMDguOyBNb2wuIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzZSBvZiBiYWN0ZXJpb3BoYWdlIGxhbWJkYSByZWNvbWJpbmF0aW9uIGZ1bmN0aW9ucyB0byBwcm9tb3RlIGdlbmUgcmVwbGFjZW1lbnQgaW4gRXNjaGVyaWNoaWEgY29saTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODA7ICBjaXRhdGlvbl9wYWdlcz0yMDYzLTIwNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBhbmFseXNlcyBvZiB0cmFuc2NyaXB0b21lcyBhbmQgcHJvdGVvbWVzIG9mIGEgcGFyZW50IHN0cmFpbiBhbmQgYW4gbC10aHJlb25pbmUtb3ZlcnByb2R1Y2luZyBtdXRhbnQgc3RyYWluOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPTU0NDItNTQ1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249M3JkIEVkLjsgY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciBMYWJvcmF0b3J5IFByZXNzLCBOZXcgWW9yazsgTW9sZWN1bGFyIENsb25pbmc6IGEgTGFib3JhdG9yeSBNYW51YWw7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSBLLTEyOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTE0NTMtMTQ3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBlbmRvY2VsbHVsYXIgYmFjdGVyaWFsIHN5bWJpb250IG9mIGFwaGlkIEJ1Y2huZXJhIHNwLiBBUFM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDc7ICBjaXRhdGlvbl9wYWdlcz04MS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGNvbnNlY3V0aXZlIGRlbGV0aW9ucyBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBjaHJvbW9zb21lOyBNb2wuIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRVJHTyBnZW5vbWUgYW5hbHlzaXMgYW5kIGRpc2NvdmVyeSBzeXN0ZW07IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTE2NC0xNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIG1ldGhvZCBmb3IgbXVsdGlwbGUgbW9kaWZpY2F0aW9uIG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIEstMTIgY2hyb21vc29tZTsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTI5MDUtMjkxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZSBwYXJ0aXRpb25pbmcgaW4gRXNjaGVyaWNoaWEgY29saTogbm92ZWwgbXV0YW50cyBwcm9kdWNpbmcgYW51Y2xlYXRlIGNlbGxzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTE0OTYtMTUwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgZGVveHlhZGVub3NpbmUgbWV0aHlsdHJhbnNmZXJhc2UtbWVkaWF0ZWQgY29udHJvbCBvZiBnZW5lIGV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NjczLTY5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBncm93dGgsIGFjZXRhdGUgcHJvZHVjdGlvbiwgYW5kIGFjZXRhdGUgaW5oaWJpdGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIHN0cmFpbnMgaW4gYmF0Y2ggYW5kIGZlZC1iYXRjaCBmZXJtZW50YXRpb25zOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9MTAwNC0xMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnZlc3RpZ2F0aW9uIG9mIHRoZSBUQ0EgY3ljbGUgYW5kIHRoZSBnbHlveHlsYXRlIHNodW50IGluIEVzY2hlcmljaGlhIGNvbGkgQkwyMSBhbmQgSk0xMDkgdXNpbmcgKDEzKUMtTk1SL01TOyBCaW90ZWNobm9sLiBCaW9lbmcuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MzE2LTMyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpY2FyYm94eWxpYyBhY2lkIGN5Y2xlIGFuZCBnbHlveHlsYXRlIGJ5cGFzczsgY2l0YXRpb25fcHVibGlzaGVyPUFtZXJpY2FuIFNvY2lldHkgZm9yIE1pY3JvYmlvbG9neSwgV2FzaGluZ3RvbiBELiBDLjsgRXNjaGVyaWNoaWEgY29saSBhbmQgU2FsbW9uZWxsYSB0eXBoaW11cml1bTogQ2VsbHVsYXIgYW5kIE1vbGVjdWxhciBCaW9sb2d5LCAybmQgRWQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fcGFnZXM9MjA2LTIxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE1LzUvMjc3LzEwMzAwMjUvZHNuMDE5LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBFc2NoZXJpY2hpYSBjb2xpIGhhcyBkaXNwZW5zYWJsZSBnZW5vbWUgcmVnaW9ucyBhbmQgZWxpbWluYXRpbmcgdGhlbSBtYXkgaW1wcm92ZSBjZWxsIHVzZSBieSByZWR1Y2luZyB1bm5lY2Vzc2FyeSBtZXRhYm9saWMgcGF0aHdheXMgYW5kIGNvbXBsZXggcmVndSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzU5ODA2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZATF/r2tB8P4Zh6x/QqLFea9137t" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f277%2f359806" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f277%2f359806" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13976" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&amp;Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/5">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5189558">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189558">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189559">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189562">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189562">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189572">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189572">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189602">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189602">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189613">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189613">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189615">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189615">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189617">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5189619">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5189619">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/5/267/359884">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/5/285/359655">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Superpositioning of Deletions Promotes Growth of <em>Escherichia coli</em> with a Reduced Genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Mizoguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 42-725-2555. Fax. <div class="fax">+81 42-726-8330</div>. E-mail: <a href="mailto:mizoguchi.hiroshi@kyowa.co.jp" target="_blank">mizoguchi.hiroshi@kyowa.co.jp</a> (<a href="mailto:hiroshi.mizoguchi@kyowa-kirin.co.jp" target="_blank">hiroshi.mizoguchi@kyowa-kirin.co.jp</a>), <a href="mailto:hmori@kyowa.co.jp" target="_blank">hmori@kyowa.co.jp</a> (<a href="mailto:hideo.mori@kyowa-kirin.co.jp" target="_blank">hideo.mori@kyowa-kirin.co.jp</a>)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Mizoguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mizoguchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mizoguchi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Mizoguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Sawano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Sawano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sawano Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sawano Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshie Sawano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun-ichi Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Sciences, Graduate Schools of Science and Engineering</div>, <div class="insititution">Tokyo Metropolitan University</div>, <div class="addr-line">Minamiohsawa, Hachioji, Tokyo 192-0397</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun-ichi+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun-ichi Kato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideo Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biofrontier Laboratories, Kyowa Hakko Kogyo Co. Ltd, 3-6-6 Asahimachi, Machidashi, Tokyo 194-8533</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 42-725-2555. Fax. <div class="fax">+81 42-726-8330</div>. E-mail: <a href="mailto:mizoguchi.hiroshi@kyowa.co.jp" target="_blank">mizoguchi.hiroshi@kyowa.co.jp</a> (<a href="mailto:hiroshi.mizoguchi@kyowa-kirin.co.jp" target="_blank">hiroshi.mizoguchi@kyowa-kirin.co.jp</a>), <a href="mailto:hmori@kyowa.co.jp" target="_blank">hmori@kyowa.co.jp</a> (<a href="mailto:hideo.mori@kyowa-kirin.co.jp" target="_blank">hideo.mori@kyowa-kirin.co.jp</a>)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideo+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideo Mori</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 277–284, <a href='https://doi.org/10.1093/dnares/dsn019'>https://doi.org/10.1093/dnares/dsn019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 August 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 December 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 July 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359806" href="/dnaresearch/article-pdf/15/5/277/1030025/dsn019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hiroshi Mizoguchi, Yoshie Sawano, Jun-ichi Kato, Hideo Mori;  Superpositioning of Deletions Promotes Growth of <em>Escherichia coli</em> with a Reduced Genome, <em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 277–284, <a href='https://doi.org/10.1093/dnares/dsn019'>https://doi.org/10.1093/dnares/dsn019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359806?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359806?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359806?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359806?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359806?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359806?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359806?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Superpositioning%20of%20Deletions%20Promotes%20Growth%20of%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B%20with%20a%20Reduced%20Genome&amp;publicationDate=2008-08-27&amp;volumeNum=15&amp;issueNum=5&amp;author=Mizoguchi%2C%20Hiroshi%3B%20Sawano%2C%20Yoshie&amp;startPage=277&amp;endPage=284&amp;contentId=10.1093%2Fdnares%2Fdsn019&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f277%2f359806" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5189558 id="5189558" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Escherichia coli</em> has dispensable genome regions and eliminating them may improve cell use by reducing unnecessary metabolic pathways and complex regulatory networks. Although several strains with reduced genomes have already been constructed, there have been no reports of strains constructed with deletions assayed for influence on growth. To retain robust growth and fundamental metabolic pathways, the growth of each deletion strain and combination effects of deletions were checked using M9 minimal medium. Then a new strain, MGF-01, with a 1 Mb reduced genome was constructed by integrating deletions that did not affect growth. MGF-01 grew as well as the wild type in the exponential phase and continued growing after the wild type had entered the stationary phase. The final cell density of MGF-01 was 1.5 times higher than that of the wild-type strain. Using MGF-01 as a production host, a 2.4-fold increase in <span class="small-caps">l</span>-threonine production was achieved.</p></section>                        <div class="article-metadata-panel clearfix"></div>
                            <h2 scrollto-destination=5189559 id="5189559" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Escherichia coli</em> is one of the most widely used organisms for the industrial production of recombinant proteins, amino acids and other chemical products.<sup><a reveal-id="DSN019C1  DSN019C5" data-open="DSN019C1  DSN019C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> But it is necessary to remove some regulatory mechanisms such as depression by the product to improve its productivity.<sup><a reveal-id="DSN019C6" data-open="DSN019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Furthermore, there are &gt;400 function unknown genes.<sup><a reveal-id="DSN019C7" data-open="DSN019C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Reducing dispensable genome regions may decrease such complicated regulatory mechanisms and function unknown genes. Functional analysis and modeling, therefore, may be easier. Several groups have already established <em>E</em>. <em>coli</em> cells with fewer genes.<sup><a reveal-id="DSN019C8  DSN019C11" data-open="DSN019C8  DSN019C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> Also, reduced genomes have been designed mainly <em>in silico</em> using gene annotations and information from comparative or functional genomics to identify minimal gene sets.<sup><a reveal-id="DSN019C9" data-open="DSN019C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSN019C10" data-open="DSN019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Hashimoto et al.<sup><a reveal-id="DSN019C10" data-open="DSN019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> eliminated regions between essential genes as much as possible. Sixteen regions with a total length of 1.38 Mb were deleted from the <em>E</em>. <em>coli</em> K-12 MG1655 chromosome, the resulting strain being designated as Δ16. Although the genome size of Δ16 is the smallest, it exhibits an aberrant cell morphology and an increased doubling time relative to the number of deletions. Alternatively, Pósfai et al.<sup><a reveal-id="DSN019C11" data-open="DSN019C11" class="link link-ref link-reveal xref-bibr">11</a></sup> selected strain-specific genomic regions as deletion targets by comparing six sequenced <em>E</em>. <em>coli</em> genomes. Forty-three segments were sequentially deleted from the MG1655 chromosome, resulting in 0.71 Mb genome reduction. The resulting strain, MDS43, had lost all mobile elements such as insertion sequences (ISs) and prophages. A related MDS41 strain with 41 deletions showed 20–25% lower mutation than the wild type because of its IS-free genotype. This strain as well as MG1655 exhibited good growth in minimal medium. Another strain, MDS42, with 42 deletions exhibited twice the electroporation efficiency of MG1655. The <em>in silico</em> design for MDS strains worked well to produce reduced genome strains with unique properties; however, to reduce the genome more than that of MDS43, it is necessary to delete common genomic regions of several <em>E</em>. <em>coli</em> strains.<sup><a reveal-id="DSN019C12" data-open="DSN019C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The opportunity of managing growth is higher in common regions, which include multiple metabolic pathway and regulatory genes. Single-gene knockout mutants have been constructed but little is known about how much each of those affects the growth of <em>E</em>. <em>coli</em> in minimal medium.<sup><a reveal-id="DSN019C13" data-open="DSN019C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Furthermore, little is known about the cumulative effects of deletions.</p><p>Here, we report construction of a reduced genome <em>E</em>. <em>coli</em> cell by assessing the growth on M9 minimal medium of all intermediate deletion mutants. In order to maintain robust cell growth, only deletions that did not affect cell growth were used to construct a reduced genome strain. The total length of the eliminated regions was 1 Mb, and the resulting strain was designated as MGF-01. MGF-01 showed unexpected but beneficial growth on M9 minimal medium. We believe this strain can be used as a tool for functional analysis of <em>E</em>. <em>coli</em> and as a host for industrial production.</p>                            <h2 scrollto-destination=5189562 id="5189562" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5189563 id="5189563" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and growth conditions</h3>
<p><em>Escherichia</em><em>coli</em> strain KM22 with <em>recBCD</em> genes replaced by <em>λ</em> Red recombination functions (Δ<em>recBCD</em>::P<em>lac</em>-<em>bet exo kan</em>) was obtained from Dr Kenan C. Murphy.<sup><a reveal-id="DSN019C14" data-open="DSN019C14" class="link link-ref link-reveal xref-bibr">14</a></sup> A chromosomal region including <em>λ</em> Red genes was introduced into wild-type strain W3110, using P1 transduction, the resulting strain being designated as W3110red (W3110 Δ<em>recBCD</em>::P<em>lac</em>-<em>bet exo kan</em>). A <span class="small-caps">l</span>-threonine production strain was constructed by replacing the homoserine <em>O</em>-succinyltransferase gene, <em>metA</em>, with the desensitized (canceled feedback regulation of <span class="small-caps">l</span>-threonine) aspartokinase/homoserine dehydrogenase gene, <em>thrA345</em>; homoserine kinase gene, <em>thrB</em>; threonine synthase gene, <em>thrC</em>; and chloramphenicol acetyltransferase gene, <em>cat</em> (Δ<em>metA</em>::<em>thrA345BC cat</em>).<sup><a reveal-id="DSN019C15" data-open="DSN019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Luria–Bertani (LB) medium and M9 minimal medium are described elsewhere.<sup><a reveal-id="DSN019C16" data-open="DSN019C16" class="link link-ref link-reveal xref-bibr">16</a></sup> For the growth test, 10 mg/L FeSO<sub>4</sub> 20 g/L CaCO<sub>3</sub> and 10 g/L glucose were added to the M9 minimal medium. A seed culture was grown at 37°C for 14 h in a test tube containing 5 mL LB medium. Two hundred microliters of seed culture was inoculated into a 300 mL flask with baffles (IWAKI, Tokyo, Japan) containing 20 mL M9 minimal medium. The cultivation conditions were 37°C and 250 rpm. During the cultivation, 5 g/L glucose was added at 20 h to avoid glucose depletion. Two hundred microliters of culture was diluted with 1.8 mL 0.1 N HCl to evaluate growth. For <span class="small-caps">l</span>-threonine production, strains W3110_thr and MGF-01_thr were each grown in a 300 mL flask with baffles containing 20 mL fermentation medium. The fermentation medium was composed of 70 g of glucose, 20 g of (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 1 g of KH<sub>2</sub>PO<sub>4</sub>, 0.5 g of MgSO<sub>4</sub>·7H<sub>2</sub>O, 5 mg of FeSO<sub>4</sub>·7H<sub>2</sub>O, 5 mg of MnSO<sub>4</sub>·4H<sub>2</sub>O, 2 g of yeast extract, 120 mg of methionine and 30 g/L CaCO<sub>3</sub> per liter of water at pH 7.0.<sup><a reveal-id="DSN019C6" data-open="DSN019C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p>                            <h3 scrollto-destination=5189565 id="5189565" class="section-title" data-legacy-id=s2b>2.2. Selection of deletion targets</h3>
<p>The genome sequences of <em>E</em>. <em>coli</em> K-12 MG1655 (accession no. U00096)<sup><a reveal-id="DSN019C17" data-open="DSN019C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and <em>Buchnera</em> sp. APS (accession no. NC002528)<sup><a reveal-id="DSN019C18" data-open="DSN019C18" class="link link-ref link-reveal xref-bibr">18</a></sup> were used for comparative genomics, and <em>E</em>. <em>coli</em> unique genes were selected for elimination. Essential genes reported in the PEC database (<a class="link link-uri" href="http://www.shigen.nig.ac.jp/ecoli/pec/index.jsp">http://www.shigen.nig.ac.jp/ecoli/pec/index.jsp</a>) were eliminated as deletion candidates.<sup><a reveal-id="DSN019C19" data-open="DSN019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The annotations of the remaining candidates were surveyed in the ERGO databases (<a class="link link-uri" href="http://www.integratedgenomics.com/">http://www.integratedgenomics.com/</a>) to judge their necessity for good growth in M9 minimal medium.<sup><a reveal-id="DSN019C20" data-open="DSN019C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Regions with more than 10 continuous unnecessary genes were chosen for deletion. These candidate regions for deletion are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=5189567 id="5189567" class="section-title" data-legacy-id=s2c>2.3. Construction of reduced genome strains</h3>
<p>The details of the used markerless deletion method are given in Supplemental Materials. Single-deletion strains as to each candidate region were prepared using a λ Red recombination system and negative selection marker <em>sacB</em> (markerless deletion method) previously reported (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1, Supplementary Table S1</a>).<sup><a reveal-id="DSN019C21" data-open="DSN019C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Deletions distributed in 50 kb were integrated to construct deletion-unit strains by continual use of the markerless deletion method (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). All single-deletion strains and deletion-unit strains were carefully assessed for growth in M9 minimal medium. Deletions that did not affect growth were integrated to construct multiple-deletion strains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Multiple-deletion strains were also assayed for growth in M9 minimal medium.</p>                            <h3 scrollto-destination=5189569 id="5189569" class="section-title" data-legacy-id=s2d>2.4. Analytical methods</h3>
<p>Comparative genomic hybridization was performed as described previously.<sup><a reveal-id="DSN019C12" data-open="DSN019C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Cells were stained using 4,6-diamidino-2-phenylinodole (DAPI) after fixation for fluorescent microscopic observation of chromosomes.<sup><a reveal-id="DSN019C22" data-open="DSN019C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Supernatants of cultures were used for <span class="small-caps">l</span>-threonine, acetate and glucose quantification. A high-pressure liquid chromatography system with post-column amino acid labeling and detection ‘Prominence’ (SHIMADZU, Kyoto, Japan) was used to quantify <span class="small-caps">l</span>-threonine. A DX 500 Ion Chromatograph System (DIONEX, Tokyo, Japan) was used to quantify acetate and a 7070 Automatic analyzer (HITACHI, Tokyo, Japan) was used to quantify glucose. The flanking regions of deletion sites were amplified by PCR, and sequenced with the Big Dye terminator method and an ABI3700 sequencer.</p><p>Dual-label microarray experiments with TaKaRa <em>E</em>. <em>coli</em> W3110 custom DNA tip (TAKARA SHUZO, Shiga, Japan) were performed as previously described.<sup><a reveal-id="DSN019C23" data-open="DSN019C23" class="link link-ref link-reveal xref-bibr">23</a></sup> After 21 h cultivation in M9 minimal medium, RNA was prepared using RNAprotect Bacteria Reagent and RNeasy Mini Kit (QIAGEN). Signal intensity of each spot in the microarray was quantified using GenePix™ Pro 4.0 (Axon Instruments) software. Further data analyses were conducted by using computer software programs, Microsoft<sup>®</sup> Excel and GeneSpring<sup>®</sup> 5.0.2 (Silicon Genetics, Redwood, CA, USA). Only genes existed in MGF-01 were used to compare gene expression between MGF-01 and W3110red.</p>                            <h2 scrollto-destination=5189572 id="5189572" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5189573 id="5189573" class="section-title" data-legacy-id=s3a>3.1. Candidate regions for deletion</h3>
<p>Dispensable regions of <em>E</em>. <em>coli</em> were selected using comparative genomics between <em>E</em>. <em>coli</em> and <em>Buchnera</em> sp. The species <em>Buchnera</em> sp. was thought to share a common ancestor with <em>E</em>. <em>coli</em>, and this symbiont attained a 0.64 Mb small genome through reduction in its size during evolution.<sup><a reveal-id="DSN019C18" data-open="DSN019C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Therefore, it was suitable for <em>E</em>. <em>coli</em> to compare genome construction with that of <em>Buchnera</em> sp. to extract candidate regions for deletions. Regions that did not exist in <em>Buchnera</em> sp. were selected. Meanwhile, this symbiont was missing the biosynthesis genes for some amino acids, lipopolysaccharides and phospholipids which were required for growth on M9 minimal medium. Consequently, after eliminating <em>E</em>. <em>coli</em> essential genes from the list of candidates, the list was again checked in order to exclude genes necessary for normal growth on M9 minimal medium. Regions of more than 10 continuous unnecessary genes were chosen as candidate regions for deletion. Using these criteria, 83 regions were selected (Δ001–Δ089 including missing numbers). Furthermore, transporter genes, ISs and toxin–antitoxin pairs were selected, and 20 more regions were designed for deletion (Δ090–Δ109). One hundred and three regions were selected and the total length of the candidates was 1830 kb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p>                            <h3 scrollto-destination=5189575 id="5189575" class="section-title" data-legacy-id=s3b>3.2. Construction of an E. coli cell with a reduced genome</h3>
<p>We tried to delete all the target regions by the markerless deletion method. The growth of each deletion mutant was examined by measuring the time-dependent change in cell density and 84 of the 103 deletion mutants showed comparable growth to the wild type. Deletions of normal-growth strains were used for constructing a reduced genome strain. Deletions were accumulated in one strain as described in Fig. <a reveal-id="DSN019F1" data-open="DSN019F1" class="link link-reveal link-table xref-fig">1</a>. Finally, 53 deletions were combined in one strain via 28 cycles of deletion-transfer (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). This strain was designated as MGF-01. All 53 deletions of MGF-01 and a substitution at the <em>recBCD</em> site, replaced by λ Red recombinase genes, were confirmed by comparative genomic hybridization (Fig. <a reveal-id="DSN019F2" data-open="DSN019F2" class="link link-reveal link-table xref-fig">2</a>). To confirm the construction of each junction, both 1 kb flanking regions of all 53 deletions were sequenced. Comparison of the deletion maps of Δ16, MDS43 and MGF-01 is shown in Fig. <a reveal-id="DSN019F3" data-open="DSN019F3" class="link link-reveal link-table xref-fig">3</a>, and a Venn diagram of Δ16, MDS43 and MGF-01 is shown in Fig. <a reveal-id="DSN019F4" data-open="DSN019F4" class="link link-reveal link-table xref-fig">4</a>. One thousand and eighty-one out of 4396 CDSs were deleted from W3110 to construct MGF-01, and function unknown genes were reduced from 471 to 346 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Although there are 301 common CDSs deleted in all three strains, unique CDSs were deleted in each strain because deletion targets were selected with different criteria. The total length of deletions in MGF-01 was 1.03 Mb and the GC content of the reduced-size genome was increased by 0.27% and reached to 51.8%.</p>                        <a id="5189577" scrollto-destination="5189577"></a>
<div id="DSN019F1" data-id="DSN019F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01901.jpeg?Expires=2147483647&Signature=S6-TL5FPgt8YYv1cdQdIZc7F6UU0rzQJyzz-vXLIrwWcC7ZDKtxJgFeCFmSbHBxMAWGKnIbgXRxYZVWY~HgjLvcP7osV4RTRm2z69KO7xs3S5Ww7HK4ihvpVLECQu9ltgXrhNwiGBPQYNiqBVYVasUSkg02IWSSyMH1z5KzAzKqYO6oEZ8~WitGp2SF2r6IzxE4i62QC~EybP9KswHpfGYe8yLMPD~rTmq4p4pzGnT-xYYehY7MjlPY~qqwWk9ZXQBiiv3-sAryV9oXEENFabP2Rl2nnC3vTCfi5noi4~TByXiYGhdqsw-OjfnZxapyDGWM2O4rgCvDaG5oP3t0Ghw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Method used to integrate deletion-units. (A) Multiple deletion strain at step N was used as the recipient for P1 transduction. A deletion-unit strain was used as the donor of deletions. (B) Integrated new deletion units were selected as to the chloramphenicol resistance of the marker cassette. Transduction of the most distal deletion from the marker cassette was confirmed by PCR. (C) The marker cassette including B. subtilis sacB was replaced by a DNA fragment without the marker cassette using a λ red recombination system or P1 transduction using a P1 phage prepared from a markerless-unit strain.19 A strain with additional deletions (strain at N + 1 step) was selected as to its sucrose resistance. Each circle represents an E. coli genome and wedges denote deletions. The white arrow represents the sacB_cat marker cassette." path-from-xml="dsn01901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189577/dsn01901.jpeg" path-from-xml="dsn01901.jpeg" target="_blank">View large</a><a section="5189577" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01901.jpeg?Expires=2147483647&Signature=YWbObL6mHp7ly0Jvc6sYQA2QVKJbK~8IpTaMql1jjVB2YTSrWpq166p-eumAoZKxswAB2hiqV0BzCn-KoN~~b6XV0xup2om0IJb6a8TPeAhs8hCue720NwZYyseLDKh3AY9XbOKCvN-inUbvWEpYV4aXNAmVrG32LnxawMKX-8gLCrSfdU-wJvlr9hmGsrqrHSHNZO-FbtMW92sJfyI7Qj0f7C-pJhkAzySjw29vJI9~Z2TvfKEoCNweStkuCMfR2AS3L2oP21uJxQjrfeVn6mM6McS-jrKEtCs~w7DkUz5~sGKCpz1LjfB52R5m5iCzdWB5TVK2UYiy4HUqIrm9uQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189577&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Method used to integrate deletion-units. (<strong>A</strong>) Multiple deletion strain at step <em>N</em> was used as the recipient for P1 transduction. A deletion-unit strain was used as the donor of deletions. (<strong>B</strong>) Integrated new deletion units were selected as to the chloramphenicol resistance of the marker cassette. Transduction of the most distal deletion from the marker cassette was confirmed by PCR. (<strong>C</strong>) The marker cassette including <em>B</em>. <em>subtilis sacB</em> was replaced by a DNA fragment without the marker cassette using a <em>λ</em> red recombination system or P1 transduction using a P1 phage prepared from a markerless-unit strain.<sup><a reveal-id="DSN019C19" data-open="DSN019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> A strain with additional deletions (strain at <em>N</em> + 1 step) was selected as to its sucrose resistance. Each circle represents an <em>E</em>. <em>coli</em> genome and wedges denote deletions. The white arrow represents the <em>sacB</em>_<em>cat</em> marker cassette.</p></div></div><div content-id="DSN019F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01901.jpeg?Expires=2147483647&Signature=S6-TL5FPgt8YYv1cdQdIZc7F6UU0rzQJyzz-vXLIrwWcC7ZDKtxJgFeCFmSbHBxMAWGKnIbgXRxYZVWY~HgjLvcP7osV4RTRm2z69KO7xs3S5Ww7HK4ihvpVLECQu9ltgXrhNwiGBPQYNiqBVYVasUSkg02IWSSyMH1z5KzAzKqYO6oEZ8~WitGp2SF2r6IzxE4i62QC~EybP9KswHpfGYe8yLMPD~rTmq4p4pzGnT-xYYehY7MjlPY~qqwWk9ZXQBiiv3-sAryV9oXEENFabP2Rl2nnC3vTCfi5noi4~TByXiYGhdqsw-OjfnZxapyDGWM2O4rgCvDaG5oP3t0Ghw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Method used to integrate deletion-units. (A) Multiple deletion strain at step N was used as the recipient for P1 transduction. A deletion-unit strain was used as the donor of deletions. (B) Integrated new deletion units were selected as to the chloramphenicol resistance of the marker cassette. Transduction of the most distal deletion from the marker cassette was confirmed by PCR. (C) The marker cassette including B. subtilis sacB was replaced by a DNA fragment without the marker cassette using a λ red recombination system or P1 transduction using a P1 phage prepared from a markerless-unit strain.19 A strain with additional deletions (strain at N + 1 step) was selected as to its sucrose resistance. Each circle represents an E. coli genome and wedges denote deletions. The white arrow represents the sacB_cat marker cassette." path-from-xml="dsn01901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189577/dsn01901.jpeg" path-from-xml="dsn01901.jpeg" target="_blank">View large</a><a section="5189577" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01901.jpeg?Expires=2147483647&Signature=YWbObL6mHp7ly0Jvc6sYQA2QVKJbK~8IpTaMql1jjVB2YTSrWpq166p-eumAoZKxswAB2hiqV0BzCn-KoN~~b6XV0xup2om0IJb6a8TPeAhs8hCue720NwZYyseLDKh3AY9XbOKCvN-inUbvWEpYV4aXNAmVrG32LnxawMKX-8gLCrSfdU-wJvlr9hmGsrqrHSHNZO-FbtMW92sJfyI7Qj0f7C-pJhkAzySjw29vJI9~Z2TvfKEoCNweStkuCMfR2AS3L2oP21uJxQjrfeVn6mM6McS-jrKEtCs~w7DkUz5~sGKCpz1LjfB52R5m5iCzdWB5TVK2UYiy4HUqIrm9uQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189577&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Method used to integrate deletion-units. (<strong>A</strong>) Multiple deletion strain at step <em>N</em> was used as the recipient for P1 transduction. A deletion-unit strain was used as the donor of deletions. (<strong>B</strong>) Integrated new deletion units were selected as to the chloramphenicol resistance of the marker cassette. Transduction of the most distal deletion from the marker cassette was confirmed by PCR. (<strong>C</strong>) The marker cassette including <em>B</em>. <em>subtilis sacB</em> was replaced by a DNA fragment without the marker cassette using a <em>λ</em> red recombination system or P1 transduction using a P1 phage prepared from a markerless-unit strain.<sup><a reveal-id="DSN019C19" data-open="DSN019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> A strain with additional deletions (strain at <em>N</em> + 1 step) was selected as to its sucrose resistance. Each circle represents an <em>E</em>. <em>coli</em> genome and wedges denote deletions. The white arrow represents the <em>sacB</em>_<em>cat</em> marker cassette.</p></div></div>                        <a id="5189578" scrollto-destination="5189578"></a>
<div id="DSN019F2" data-id="DSN019F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01902.jpeg?Expires=2147483647&Signature=XsHbgFm48j-fOrwo~SqhjNRVkgvchwnSsuS024BN3Aub0o46bIsrDQ-y04wU-tbgw8InRqg~jhU~6qJbDGp-SX-pceCBX3AFMbk1DKxjyVYl9HgR5acaQcR2lSIc~0thaK4QKAwTcrkXlNKz1kTqCde0C3W6eIO-QjfhCvbMyQxeiGWZPcs4SNoMjVw2jBxere6hRwlvs7oUztugKu6Ekfb49U13~5rJMEJfDHWqqyF3WlW3yS2AjENUqLgr21BJwt23T~pbeh20TaWlQMLj3qOno7fdyAk7cFkLdR0DYZXGWP3JIYE8H2t9PEyCwFT262CzJsfweHLaxzT9Jp3i~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Map of deletions detected in MGF-01 on microarray analysis. Each bar represents the log ratio of normalized signal intensities (MGF-01 ORF signal/W3110 ORF signal). Deleted regions were determined when the signal ratios were below –1.8. The signal ratios were plotted against ORF locations of JW0001 to JW4366. The corresponding deletion identification numbers are indicated at the bottom of the figure." path-from-xml="dsn01902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189578/dsn01902.jpeg" path-from-xml="dsn01902.jpeg" target="_blank">View large</a><a section="5189578" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01902.jpeg?Expires=2147483647&Signature=K4CZ2NZ8cMR9sLYVt29TcFR2FOxjv3CA-Fet056wLiMmb3tpCFrJgaEkj40OaPoT0gKBZZWOMKE3feZMZTIfmCMBHEuwrdiGdJunzJsxNUqbiHr2RjUJ5tT-MSBaX-hQmARRmvISaAma1BP1cNC5IWBT9rwQXfJfbPepxonPzdtwEpKr8Z8qfbE2BUEYMnnNSeiy6O8OjQiWgB1naEoIlngepZReL~j4qOMnn3gzZjVXzJAtwqqv6lFiIkfpzzp0zJjyv1X~ndOfmAXQBqAEbVE2XVMcayquOuK-1J~zh2P4OIApkkNKB1MBtZZ8SnxcYZa1CEKl6xWR29kNCBUQIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189578&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Map of deletions detected in MGF-01 on microarray analysis. Each bar represents the log ratio of normalized signal intensities (MGF-01 ORF signal/W3110 ORF signal). Deleted regions were determined when the signal ratios were below –1.8. The signal ratios were plotted against ORF locations of JW0001 to JW4366. The corresponding deletion identification numbers are indicated at the bottom of the figure.</p></div></div><div content-id="DSN019F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01902.jpeg?Expires=2147483647&Signature=XsHbgFm48j-fOrwo~SqhjNRVkgvchwnSsuS024BN3Aub0o46bIsrDQ-y04wU-tbgw8InRqg~jhU~6qJbDGp-SX-pceCBX3AFMbk1DKxjyVYl9HgR5acaQcR2lSIc~0thaK4QKAwTcrkXlNKz1kTqCde0C3W6eIO-QjfhCvbMyQxeiGWZPcs4SNoMjVw2jBxere6hRwlvs7oUztugKu6Ekfb49U13~5rJMEJfDHWqqyF3WlW3yS2AjENUqLgr21BJwt23T~pbeh20TaWlQMLj3qOno7fdyAk7cFkLdR0DYZXGWP3JIYE8H2t9PEyCwFT262CzJsfweHLaxzT9Jp3i~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Map of deletions detected in MGF-01 on microarray analysis. Each bar represents the log ratio of normalized signal intensities (MGF-01 ORF signal/W3110 ORF signal). Deleted regions were determined when the signal ratios were below –1.8. The signal ratios were plotted against ORF locations of JW0001 to JW4366. The corresponding deletion identification numbers are indicated at the bottom of the figure." path-from-xml="dsn01902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189578/dsn01902.jpeg" path-from-xml="dsn01902.jpeg" target="_blank">View large</a><a section="5189578" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01902.jpeg?Expires=2147483647&Signature=K4CZ2NZ8cMR9sLYVt29TcFR2FOxjv3CA-Fet056wLiMmb3tpCFrJgaEkj40OaPoT0gKBZZWOMKE3feZMZTIfmCMBHEuwrdiGdJunzJsxNUqbiHr2RjUJ5tT-MSBaX-hQmARRmvISaAma1BP1cNC5IWBT9rwQXfJfbPepxonPzdtwEpKr8Z8qfbE2BUEYMnnNSeiy6O8OjQiWgB1naEoIlngepZReL~j4qOMnn3gzZjVXzJAtwqqv6lFiIkfpzzp0zJjyv1X~ndOfmAXQBqAEbVE2XVMcayquOuK-1J~zh2P4OIApkkNKB1MBtZZ8SnxcYZa1CEKl6xWR29kNCBUQIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189578&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Map of deletions detected in MGF-01 on microarray analysis. Each bar represents the log ratio of normalized signal intensities (MGF-01 ORF signal/W3110 ORF signal). Deleted regions were determined when the signal ratios were below –1.8. The signal ratios were plotted against ORF locations of JW0001 to JW4366. The corresponding deletion identification numbers are indicated at the bottom of the figure.</p></div></div>                        <a id="5189579" scrollto-destination="5189579"></a>
<div id="DSN019F3" data-id="DSN019F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01903.jpeg?Expires=2147483647&Signature=qiQsgjeU0EYt0WG5cv~3IIc76xM-iOF6j3GXDhWAwtr2w9WxBLNXknGl-jWE0URJNJNsKmvVGbx7GGg-VF4E-bs0ShVruSle9u4aZVNkazDVFf4XErw30jXzugKgOc59WOzkLH~CB7SckLupSxZD7Lpp1SH-8uRSNIP5R-V9b7VmM2M9k6V1IkXJSOUmlCaSerrNlBtBwFuAKu9COpeJrg3z36SSvHz073Y5WWmXb482x89~cSOIxJuPK2u72639P8zX7~b0o1YjoXSkXuirQT~3vSR1fRYE9KJAq76fBtyuYmFTHxcj9bI-oESVtBXva1KSJkgorZabpplnWd6SLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic illustration of the reduced genomes of E. coli. Rings represent the E. coli W3110 genome. From the center, the deleted regions of Δ0169, MDS4310 and MGF-01 are indicated by black boxes. Strains Δ16 and MDS43 were derived from MG1655. The parental strain of MGF-01 was W3110 possessing a large inversion from rrnD to rrnE. In the figure, the order of the genes of Δ16 and MDS43 between rrnD and rrnE is inverted according to that of W3110 in order to compare deletions in this region with those in MGF-01. Ori and Ter indicate the origin and terminus domains for replication, respectively." path-from-xml="dsn01903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189579/dsn01903.jpeg" path-from-xml="dsn01903.jpeg" target="_blank">View large</a><a section="5189579" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01903.jpeg?Expires=2147483647&Signature=lP0GZcPTk~erUBYV-3ZQf1-piKnx2J4LErDKLMlz6JEq34ZvYKNggm5i3KFQfiXtPvBKBogj6IrMYrIkBdulSYL7DOLH2tc465u2rw7J-bs4uaZn-ZlySlwCAqGwwIyE1rzksUyiBBWvWjjvzTtJ7u1w3BbxfpLgHRTKC1azEuCQBVNody7p2cfpAHn22tGI9vU7Yxpql5G6M3ttmnE2F-q32GZYjttlm87d7NSJBSH0i0KXqxS2janfH185oVQqaQvhXOjcXU4tmZV7oXArE-cO16aI-0eFyc690Bjlp0urUt8KN5ptdAFnHyQSBDOhR6Vb6KjkrlMSzFxQaZEAlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189579&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic illustration of the reduced genomes of <em>E</em>. <em>coli</em>. Rings represent the <em>E</em>. <em>coli</em> W3110 genome. From the center, the deleted regions of Δ016<sup><a reveal-id="DSN019C9" data-open="DSN019C9" class="link link-ref link-reveal xref-bibr">9</a></sup>, MDS43<sup><a reveal-id="DSN019C10" data-open="DSN019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and MGF-01 are indicated by black boxes. Strains Δ16 and MDS43 were derived from MG1655. The parental strain of MGF-01 was W3110 possessing a large inversion from <em>rrnD</em> to <em>rrnE</em>. In the figure, the order of the genes of Δ16 and MDS43 between <em>rrnD</em> and <em>rrnE</em> is inverted according to that of W3110 in order to compare deletions in this region with those in MGF-01. Ori and Ter indicate the origin and terminus domains for replication, respectively.</p></div></div><div content-id="DSN019F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01903.jpeg?Expires=2147483647&Signature=qiQsgjeU0EYt0WG5cv~3IIc76xM-iOF6j3GXDhWAwtr2w9WxBLNXknGl-jWE0URJNJNsKmvVGbx7GGg-VF4E-bs0ShVruSle9u4aZVNkazDVFf4XErw30jXzugKgOc59WOzkLH~CB7SckLupSxZD7Lpp1SH-8uRSNIP5R-V9b7VmM2M9k6V1IkXJSOUmlCaSerrNlBtBwFuAKu9COpeJrg3z36SSvHz073Y5WWmXb482x89~cSOIxJuPK2u72639P8zX7~b0o1YjoXSkXuirQT~3vSR1fRYE9KJAq76fBtyuYmFTHxcj9bI-oESVtBXva1KSJkgorZabpplnWd6SLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic illustration of the reduced genomes of E. coli. Rings represent the E. coli W3110 genome. From the center, the deleted regions of Δ0169, MDS4310 and MGF-01 are indicated by black boxes. Strains Δ16 and MDS43 were derived from MG1655. The parental strain of MGF-01 was W3110 possessing a large inversion from rrnD to rrnE. In the figure, the order of the genes of Δ16 and MDS43 between rrnD and rrnE is inverted according to that of W3110 in order to compare deletions in this region with those in MGF-01. Ori and Ter indicate the origin and terminus domains for replication, respectively." path-from-xml="dsn01903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189579/dsn01903.jpeg" path-from-xml="dsn01903.jpeg" target="_blank">View large</a><a section="5189579" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01903.jpeg?Expires=2147483647&Signature=lP0GZcPTk~erUBYV-3ZQf1-piKnx2J4LErDKLMlz6JEq34ZvYKNggm5i3KFQfiXtPvBKBogj6IrMYrIkBdulSYL7DOLH2tc465u2rw7J-bs4uaZn-ZlySlwCAqGwwIyE1rzksUyiBBWvWjjvzTtJ7u1w3BbxfpLgHRTKC1azEuCQBVNody7p2cfpAHn22tGI9vU7Yxpql5G6M3ttmnE2F-q32GZYjttlm87d7NSJBSH0i0KXqxS2janfH185oVQqaQvhXOjcXU4tmZV7oXArE-cO16aI-0eFyc690Bjlp0urUt8KN5ptdAFnHyQSBDOhR6Vb6KjkrlMSzFxQaZEAlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189579&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic illustration of the reduced genomes of <em>E</em>. <em>coli</em>. Rings represent the <em>E</em>. <em>coli</em> W3110 genome. From the center, the deleted regions of Δ016<sup><a reveal-id="DSN019C9" data-open="DSN019C9" class="link link-ref link-reveal xref-bibr">9</a></sup>, MDS43<sup><a reveal-id="DSN019C10" data-open="DSN019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and MGF-01 are indicated by black boxes. Strains Δ16 and MDS43 were derived from MG1655. The parental strain of MGF-01 was W3110 possessing a large inversion from <em>rrnD</em> to <em>rrnE</em>. In the figure, the order of the genes of Δ16 and MDS43 between <em>rrnD</em> and <em>rrnE</em> is inverted according to that of W3110 in order to compare deletions in this region with those in MGF-01. Ori and Ter indicate the origin and terminus domains for replication, respectively.</p></div></div>                        <a id="5189580" scrollto-destination="5189580"></a>
<div id="DSN019F4" data-id="DSN019F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01904.jpeg?Expires=2147483647&Signature=GLFyXLq5t1I-XInFiCCkB36CyQeuB62fvcazmzhUozUc~r4B2J8Vl-DV0JwIy~0JA~Pup4NKy79rQQ2FGkqWa8ECXgXdS2LRvgYujWSIEjg7610htjZmLe4zVpxW8AzUxvQaVw9W1-xZgG8U93FLt5PblySJlgHPW8LoS8X3gFMoqCy4s4GH5C8vuQ~R0yac2gPx~MN24mfvbOZtptRdtmJESNDP3Xsm4yKLtBm2JpsVeVc-2fwemqyvgC766XndHWgmKdnLEQL4w53K4FanNgZn4yJj27UsLKnGvSpDET40GCrqNyxwRb6yfv2cOp4hP3WCs~Em8AWA-r6UDpt5yQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A Venn diagram of data sets of deleted CDSs from strains Δ16, MDS43 and MGF-01. Strains Δ16 and MDS43 were derived from MG1655, and MGF-01 is derived from W3110. The number of deleted CDSs in each strain was determined." path-from-xml="dsn01904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189580/dsn01904.jpeg" path-from-xml="dsn01904.jpeg" target="_blank">View large</a><a section="5189580" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01904.jpeg?Expires=2147483647&Signature=4KKFWz0ZyiQcWI0~hJJx5NqvxuF-aZbEYVPex2eo0yvi5mSRw-5mp29VxFxJTDZG0isWCCWLRhuYvWrWjHO6ejW0rpQhPqxsW6F1s7M4KyccLwb-jEl68pYAxs0zBpNSWXI39RcCOeCfHnjbr3nIcBH6Yk6J7BpFO7PBPkeOIsh2jRwMLnWAeItljczVZpjeWo7AL8rh6toPerqVX8vqncM94Abae3QsWoXFf7EjSNzOPSU2OrqxuNlUOF-Ic63A-xeJ~HclD-K6Ay~L8YS8KiMTncQ6A~t~Ys04lJKNNZGJyZo9laYiiUJgmYFufu8qicyPtc20clkC1xph8G99EA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189580&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A Venn diagram of data sets of deleted CDSs from strains Δ16, MDS43 and MGF-01. Strains Δ16 and MDS43 were derived from MG1655, and MGF-01 is derived from W3110. The number of deleted CDSs in each strain was determined.</p></div></div><div content-id="DSN019F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01904.jpeg?Expires=2147483647&Signature=GLFyXLq5t1I-XInFiCCkB36CyQeuB62fvcazmzhUozUc~r4B2J8Vl-DV0JwIy~0JA~Pup4NKy79rQQ2FGkqWa8ECXgXdS2LRvgYujWSIEjg7610htjZmLe4zVpxW8AzUxvQaVw9W1-xZgG8U93FLt5PblySJlgHPW8LoS8X3gFMoqCy4s4GH5C8vuQ~R0yac2gPx~MN24mfvbOZtptRdtmJESNDP3Xsm4yKLtBm2JpsVeVc-2fwemqyvgC766XndHWgmKdnLEQL4w53K4FanNgZn4yJj27UsLKnGvSpDET40GCrqNyxwRb6yfv2cOp4hP3WCs~Em8AWA-r6UDpt5yQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A Venn diagram of data sets of deleted CDSs from strains Δ16, MDS43 and MGF-01. Strains Δ16 and MDS43 were derived from MG1655, and MGF-01 is derived from W3110. The number of deleted CDSs in each strain was determined." path-from-xml="dsn01904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189580/dsn01904.jpeg" path-from-xml="dsn01904.jpeg" target="_blank">View large</a><a section="5189580" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01904.jpeg?Expires=2147483647&Signature=4KKFWz0ZyiQcWI0~hJJx5NqvxuF-aZbEYVPex2eo0yvi5mSRw-5mp29VxFxJTDZG0isWCCWLRhuYvWrWjHO6ejW0rpQhPqxsW6F1s7M4KyccLwb-jEl68pYAxs0zBpNSWXI39RcCOeCfHnjbr3nIcBH6Yk6J7BpFO7PBPkeOIsh2jRwMLnWAeItljczVZpjeWo7AL8rh6toPerqVX8vqncM94Abae3QsWoXFf7EjSNzOPSU2OrqxuNlUOF-Ic63A-xeJ~HclD-K6Ay~L8YS8KiMTncQ6A~t~Ys04lJKNNZGJyZo9laYiiUJgmYFufu8qicyPtc20clkC1xph8G99EA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189580&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A Venn diagram of data sets of deleted CDSs from strains Δ16, MDS43 and MGF-01. Strains Δ16 and MDS43 were derived from MG1655, and MGF-01 is derived from W3110. The number of deleted CDSs in each strain was determined.</p></div></div>                            <h3 scrollto-destination=5189581 id="5189581" class="section-title" data-legacy-id=s3c>3.3. Phenotypes of MGF-01</h3>
<p>Time courses of growth of MGF-01 and W3110red in M9 minimal medium are shown in Fig. <a reveal-id="DSN019F5" data-open="DSN019F5" class="link link-reveal link-table xref-fig">5</a>A. MGF-01 grew as the wild-type strain in the exponential phase and continued growing after the wild-type strain had entered the stationary phase. The final cell density of MGF-01 was 1.5 times higher than that of the wild-type strain. Colony forming units (CFU/mL) of MGF-01 and W3110red at 30 h incubation were 9.6 × 10<sup>9</sup> and 6.3 × 10<sup>9</sup>, respectively. The glucose consumption speed after 20 h was lower for MGF-01 compared with the wild-type strain despite its higher cell density (Fig. <a reveal-id="DSN019F5" data-open="DSN019F5" class="link link-reveal link-table xref-fig">5</a>B). Under aerobic batch culture conditions, the glucose catabolism of <em>E</em>. <em>coli</em> often overflows, leading to the accumulation of acetate that they inhibit <em>E</em>. <em>coli</em> growth.<sup><a reveal-id="DSN019C24" data-open="DSN019C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The levels of acetate accumulation in MGF-01 and the wild type were 0.50 and 1.37 g/L, respectively. The correlation among the high cell density, low glucose consumption and low acetate accumulation indicates that MGF-01 may utilize glucose more efficiently than the wild type.</p>                        <a id="5189583" scrollto-destination="5189583"></a>
<div id="DSN019F5" data-id="DSN019F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01905.jpeg?Expires=2147483647&Signature=NpLPKkrMxpupblae5MwdKAnl1aV-rBQTojR8IpCJwi9BbB5CWjaaSTbkLdKyk8x-~x--JrhORyk0oZEqmhdwM5mwVI506H~S40JOtU9QdJHosuMDgelFLNVRuBAa7xqx0w1Ita2WlIsL-~RmLr1vinLKC3jnKRXJTzjMALi~64P6PnuD9R1HP~UoUJvRcAlXtdYQlppQoNW-T4RoOdyjq9epWN72bzqNz2kvc7Xt8lnATO2LLqgxIl~PggkkH7DXfNrIwSOBSG5qCILTzA3Dw1QwCiwmRcl7QEZphKgUYy-zAn69iEBQHXMC7BgLwojsdFbDryCN3T9MCmX43DUzsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cell density and glucose consumption of MGF-01 in M9 minimal medium. (A) Growth in M9 minimal medium. (B) Glucose consumption in M9 minimal medium. The results of duplicate experiments are shown in the figure. W3110red, filled circle and filled diamond; MGF-01, open circle and open diamond." path-from-xml="dsn01905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189583/dsn01905.jpeg" path-from-xml="dsn01905.jpeg" target="_blank">View large</a><a section="5189583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01905.jpeg?Expires=2147483647&Signature=QBZpnTWAwljoIAOYg67eBqkWXuATOWYpwL8CaHzduf1UC3UaYachme4Wy-9YZIqD8biz~~Pxmr3iX8gG~5E9WT6VZOWPl~JnBqF6Hk7WxIfOkz-B4MzkxLsmz9naGOG8S63vruN4Ut-AtJKXNUzy2hEFxU44hU83-CZbx71mSSnpa78-Kgyk~3xnMxtvsqHhepSEVcZJADr6cMGFRJMGF9ZWh0oyuHWkohhSr5k1kI70JqCqXeW1FDh~0ovON3CyXJ~gWx~70E5XfCSrkjS8UAbh7hLuS9w5KM48zpb4GnwCPUgYIqprHb-umv~DP5Q4mkry3riJ84eouCUQJhtrSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189583&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cell density and glucose consumption of MGF-01 in M9 minimal medium. (<strong>A</strong>) Growth in M9 minimal medium. (<strong>B</strong>) Glucose consumption in M9 minimal medium. The results of duplicate experiments are shown in the figure. W3110red, filled circle and filled diamond; MGF-01, open circle and open diamond.</p></div></div><div content-id="DSN019F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01905.jpeg?Expires=2147483647&Signature=NpLPKkrMxpupblae5MwdKAnl1aV-rBQTojR8IpCJwi9BbB5CWjaaSTbkLdKyk8x-~x--JrhORyk0oZEqmhdwM5mwVI506H~S40JOtU9QdJHosuMDgelFLNVRuBAa7xqx0w1Ita2WlIsL-~RmLr1vinLKC3jnKRXJTzjMALi~64P6PnuD9R1HP~UoUJvRcAlXtdYQlppQoNW-T4RoOdyjq9epWN72bzqNz2kvc7Xt8lnATO2LLqgxIl~PggkkH7DXfNrIwSOBSG5qCILTzA3Dw1QwCiwmRcl7QEZphKgUYy-zAn69iEBQHXMC7BgLwojsdFbDryCN3T9MCmX43DUzsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cell density and glucose consumption of MGF-01 in M9 minimal medium. (A) Growth in M9 minimal medium. (B) Glucose consumption in M9 minimal medium. The results of duplicate experiments are shown in the figure. W3110red, filled circle and filled diamond; MGF-01, open circle and open diamond." path-from-xml="dsn01905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189583/dsn01905.jpeg" path-from-xml="dsn01905.jpeg" target="_blank">View large</a><a section="5189583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01905.jpeg?Expires=2147483647&Signature=QBZpnTWAwljoIAOYg67eBqkWXuATOWYpwL8CaHzduf1UC3UaYachme4Wy-9YZIqD8biz~~Pxmr3iX8gG~5E9WT6VZOWPl~JnBqF6Hk7WxIfOkz-B4MzkxLsmz9naGOG8S63vruN4Ut-AtJKXNUzy2hEFxU44hU83-CZbx71mSSnpa78-Kgyk~3xnMxtvsqHhepSEVcZJADr6cMGFRJMGF9ZWh0oyuHWkohhSr5k1kI70JqCqXeW1FDh~0ovON3CyXJ~gWx~70E5XfCSrkjS8UAbh7hLuS9w5KM48zpb4GnwCPUgYIqprHb-umv~DP5Q4mkry3riJ84eouCUQJhtrSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189583&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cell density and glucose consumption of MGF-01 in M9 minimal medium. (<strong>A</strong>) Growth in M9 minimal medium. (<strong>B</strong>) Glucose consumption in M9 minimal medium. The results of duplicate experiments are shown in the figure. W3110red, filled circle and filled diamond; MGF-01, open circle and open diamond.</p></div></div><p>The final cell densities of the multiple-deletion strains and corresponding deletion-unit strains or single-deletion strains at each deletion step are shown in Fig. <a reveal-id="DSN019F6" data-open="DSN019F6" class="link link-reveal link-table xref-fig">6</a>. Although no individual single-deletion strain or deletion-unit strain showed a significant increase in final cell density, multiple-deletion strains with stepwise genome reduction showed increased final cell densities. Consequently, the phenotype of increasing cell density may be brought about by combination(s) of deletions.</p>                        <a id="5189585" scrollto-destination="5189585"></a>
<div id="DSN019F6" data-id="DSN019F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01906.jpeg?Expires=2147483647&Signature=OxCLshnEkdDsSnS39ppz~hmz2ieSg7NfZzNGDsP3baWaFgbUUSuYeSjZeZz~aBUviQN7bAO0PvoksjDx~TJaxkOnmIUrUDZ2G8WKnb~yBFkXkVVzxv~jaBr04brOnjG7L5lVhlmGb7bWBt8TMxpivyWoTnlnNJfoCxtGFtyyAONL53w74Yr-Hfk1LRGtQ2660DRYaoV3K5Ts3JWsLssmknsNdb2jFfXvE2ccN~s0pBVUm0DatOD18ICwfqTe5t~4VVVRp9hT~WFVMa53OsjkJz0uySz~k9p10LcKSAN~IFfupFmErIoSuwGJaJgpEN-wuT5VFgiS3WzOYMqZZaUHLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Final cell densities of multiple deletion strains in M9 minimal medium. Direct lineage multiple deletion strains of MGF-01, deletion unit strains and single deletion strains used to construct MGF-01, were horizontally ordered according to the integration lineage of deletions. The final cell densities in M9 minimal medium are indicated as follows. Bars: final cell densities of deletion-unit strains or single-deletion strains. Diamonds: final cell densities of direct-lineage strains of MGF-01." path-from-xml="dsn01906.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189585/dsn01906.jpeg" path-from-xml="dsn01906.jpeg" target="_blank">View large</a><a section="5189585" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01906.jpeg?Expires=2147483647&Signature=43Xzp2gmTn5GZOU5-CXJi4fj4WuUyCvrN17sS8tkyY62zDFW3~~Ykh1O00SIcN264Mb3ZQ9uqjeL7kbi~5zmoNw~SsH1RDCmYNUtGGcnKB8PuKwfFRJuRoahOI1Iuc91dpE15ACOTGEXhXzzPjxHSYSzTYYQS7j-XaVgw3e2oumqaqdA-JRfuJwnjchCtG2shPIPTHi2ASOriEp7JQy7ISCnJkODpfESMjNoujrLMJUCamLyLUzNMcIUMiXHV8qkk7rCHXaxXRtm05tiNqXG97ZZjefwregDvCMS4ecZgTysYbPwCi7I92hw4ga1XcrpSBfT7e32Pm3TvpfsMwOY1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189585&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01906.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Final cell densities of multiple deletion strains in M9 minimal medium. Direct lineage multiple deletion strains of MGF-01, deletion unit strains and single deletion strains used to construct MGF-01, were horizontally ordered according to the integration lineage of deletions. The final cell densities in M9 minimal medium are indicated as follows. Bars: final cell densities of deletion-unit strains or single-deletion strains. Diamonds: final cell densities of direct-lineage strains of MGF-01.</p></div></div><div content-id="DSN019F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01906.jpeg?Expires=2147483647&Signature=OxCLshnEkdDsSnS39ppz~hmz2ieSg7NfZzNGDsP3baWaFgbUUSuYeSjZeZz~aBUviQN7bAO0PvoksjDx~TJaxkOnmIUrUDZ2G8WKnb~yBFkXkVVzxv~jaBr04brOnjG7L5lVhlmGb7bWBt8TMxpivyWoTnlnNJfoCxtGFtyyAONL53w74Yr-Hfk1LRGtQ2660DRYaoV3K5Ts3JWsLssmknsNdb2jFfXvE2ccN~s0pBVUm0DatOD18ICwfqTe5t~4VVVRp9hT~WFVMa53OsjkJz0uySz~k9p10LcKSAN~IFfupFmErIoSuwGJaJgpEN-wuT5VFgiS3WzOYMqZZaUHLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Final cell densities of multiple deletion strains in M9 minimal medium. Direct lineage multiple deletion strains of MGF-01, deletion unit strains and single deletion strains used to construct MGF-01, were horizontally ordered according to the integration lineage of deletions. The final cell densities in M9 minimal medium are indicated as follows. Bars: final cell densities of deletion-unit strains or single-deletion strains. Diamonds: final cell densities of direct-lineage strains of MGF-01." path-from-xml="dsn01906.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189585/dsn01906.jpeg" path-from-xml="dsn01906.jpeg" target="_blank">View large</a><a section="5189585" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01906.jpeg?Expires=2147483647&Signature=43Xzp2gmTn5GZOU5-CXJi4fj4WuUyCvrN17sS8tkyY62zDFW3~~Ykh1O00SIcN264Mb3ZQ9uqjeL7kbi~5zmoNw~SsH1RDCmYNUtGGcnKB8PuKwfFRJuRoahOI1Iuc91dpE15ACOTGEXhXzzPjxHSYSzTYYQS7j-XaVgw3e2oumqaqdA-JRfuJwnjchCtG2shPIPTHi2ASOriEp7JQy7ISCnJkODpfESMjNoujrLMJUCamLyLUzNMcIUMiXHV8qkk7rCHXaxXRtm05tiNqXG97ZZjefwregDvCMS4ecZgTysYbPwCi7I92hw4ga1XcrpSBfT7e32Pm3TvpfsMwOY1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189585&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01906.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Final cell densities of multiple deletion strains in M9 minimal medium. Direct lineage multiple deletion strains of MGF-01, deletion unit strains and single deletion strains used to construct MGF-01, were horizontally ordered according to the integration lineage of deletions. The final cell densities in M9 minimal medium are indicated as follows. Bars: final cell densities of deletion-unit strains or single-deletion strains. Diamonds: final cell densities of direct-lineage strains of MGF-01.</p></div></div><p>Chromosomal DNA was stained with DAPI to visualize the intracellular distribution of the chromosome (Fig. <a reveal-id="DSN019F7" data-open="DSN019F7" class="link link-reveal link-table xref-fig">7</a>). The shape of MGF-01 was slightly rounder but there were little prolonged or anucleate cells. The normal cell shape of MGF-01 and the CFU result indicate that the optical density represented the cell density of MGF-01, which was 1.5-fold higher than that of the wild type. The intracellular distribution of the chromosome was apparently the same as in the wild type. Therefore, the growth of MGF-01 was not affected by the multiple deletions from the viewpoints of division and intracellular distribution of the chromosome.</p>                        <a id="5189590" scrollto-destination="5189590"></a>
<div id="DSN019F7" data-id="DSN019F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01907.jpeg?Expires=2147483647&Signature=1HGgODlx7KtHJDBjE6464qObNvQzRtcxFbnZUf3i-nXQz3fhq~TczfkQy2C7MJHbnnYgaQ0QQrpqPI97fyPQzheIYdgK5KD~WCuv2N2soUBcage9QPHeXuH4TDyJlD177Lzb7PQ1EqukMQaaCbTs6isaLDcprPQznb0r543VlehxWG4g2PhOKLY-OBkUAXGuHAVk~6OeAqr8dgm3LYUhdcIk8Uzkw2hZYjl~vu4fG1qjxaaMjm-jowx~GSYEVYy~n7YGQS7Diy6KVBrGAdviQqTE7gRqFt9lUP1qPNfBni5EFcvElPMES~Pswi-IbNWU2p1yUzQv0vb6ct7CSdJfmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Microscopic images of DAPI-stained cells. Cells cultivated in M9 minimal medium were harvested at 11, 15 and 32 h, and stained using DAPI." path-from-xml="dsn01907.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189590/dsn01907.jpeg" path-from-xml="dsn01907.jpeg" target="_blank">View large</a><a section="5189590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01907.jpeg?Expires=2147483647&Signature=eV88c5xH8I8PuW~oqpTl43f8qvVNev8XxXEBfCWWgRZ28tm18A7prinwKmjEa3qWPgc7CF~oZU9TCOXTmf-93cWVxQNu0IorK6jZ8Ui5qv49U4e2H-4985JeQsf-aThuOo4zsCaYK~PlNNTPOo3MwkklerwbTYEK9DU4RqKfXvQQFmA-ieJmG7m4LamwKBGxSp4JAQxTcWy38p6FxprO4jw2BxCTVQm4QHGkMq9VOCudfzOcfrEXzCGW4YOrdZM0FoXfLDn4rNalXgPI2Jt-S15A0qOCOo5CpWiyGOt4-ekQsyVpMpoXcPAX-u9S~7U~349qA7yLZ77ih0nupR~ktA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189590&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01907.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Microscopic images of DAPI-stained cells. Cells cultivated in M9 minimal medium were harvested at 11, 15 and 32 h, and stained using DAPI.</p></div></div><div content-id="DSN019F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/m_dsn01907.jpeg?Expires=2147483647&Signature=1HGgODlx7KtHJDBjE6464qObNvQzRtcxFbnZUf3i-nXQz3fhq~TczfkQy2C7MJHbnnYgaQ0QQrpqPI97fyPQzheIYdgK5KD~WCuv2N2soUBcage9QPHeXuH4TDyJlD177Lzb7PQ1EqukMQaaCbTs6isaLDcprPQznb0r543VlehxWG4g2PhOKLY-OBkUAXGuHAVk~6OeAqr8dgm3LYUhdcIk8Uzkw2hZYjl~vu4fG1qjxaaMjm-jowx~GSYEVYy~n7YGQS7Diy6KVBrGAdviQqTE7gRqFt9lUP1qPNfBni5EFcvElPMES~Pswi-IbNWU2p1yUzQv0vb6ct7CSdJfmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Microscopic images of DAPI-stained cells. Cells cultivated in M9 minimal medium were harvested at 11, 15 and 32 h, and stained using DAPI." path-from-xml="dsn01907.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5189590/dsn01907.jpeg" path-from-xml="dsn01907.jpeg" target="_blank">View large</a><a section="5189590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn01907.jpeg?Expires=2147483647&Signature=eV88c5xH8I8PuW~oqpTl43f8qvVNev8XxXEBfCWWgRZ28tm18A7prinwKmjEa3qWPgc7CF~oZU9TCOXTmf-93cWVxQNu0IorK6jZ8Ui5qv49U4e2H-4985JeQsf-aThuOo4zsCaYK~PlNNTPOo3MwkklerwbTYEK9DU4RqKfXvQQFmA-ieJmG7m4LamwKBGxSp4JAQxTcWy38p6FxprO4jw2BxCTVQm4QHGkMq9VOCudfzOcfrEXzCGW4YOrdZM0FoXfLDn4rNalXgPI2Jt-S15A0qOCOo5CpWiyGOt4-ekQsyVpMpoXcPAX-u9S~7U~349qA7yLZ77ih0nupR~ktA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5189590&ar=359806&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01907.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Microscopic images of DAPI-stained cells. Cells cultivated in M9 minimal medium were harvested at 11, 15 and 32 h, and stained using DAPI.</p></div></div>                            <h3 scrollto-destination=5189596 id="5189596" class="section-title" data-legacy-id=s3d>3.4. Threonine production of MGF-01</h3>
<p>MGF-01 showed good growth in M9 minimal medium. To evaluate the production capability of MGF-01, an <span class="small-caps">l</span>-threonine production unit was introduced. <span class="small-caps">l</span>-Threonine is one of the major amino acids produced in fermentation processes. In the <span class="small-caps">l</span>-threonine biosynthesis pathway, the key enzyme, aspartokinase/homoserine dehydrogenase, encoded by <em>thrA</em> is feedback-regulated and depressed by the product, <span class="small-caps">l</span>-threonine, and a mutation to remove this feedback inhibition has been reported (<em>thrA345</em>).<sup><a reveal-id="DSN019C15" data-open="DSN019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The methionine biosynthesis pathway is a branch pathway of <span class="small-caps">l</span>-threonine biosynthesis. To block this route, we substituted <em>metA</em>, which encodes the first enzyme of this branch pathway, by an <span class="small-caps">l</span>-threonine production unit containing the desensitized <em>thrA</em> gene (Δ<em>metA</em>::<em>thrA345BC</em>_<em>cat</em>). This <span class="small-caps">l</span>-threonine production unit was transferred to W3110red and MGF-01 by P1 transduction, and the resulting strains were designated as W3110red_thr and MGF-01_thr, respectively. The production medium was different from M9 minimal medium, and contained yeast extract (Materials and methods). In this medium, both strains showed higher cell densities than in M9 minimal medium; however, the cell density of MGF-01 was only 1.2 times higher than that of wild type (1.5 times higher in M9 minimal medium). The <span class="small-caps">l</span>-threonine production after 48 h cultivation is summarized in Table <a reveal-id="DSN019TB1" data-open="DSN019TB1" class="link link-reveal link-table xref-fig">1</a>. Interestingly, the <span class="small-caps">l</span>-threonine production and the yield of MGF-01_thr were 2.44 and 1.69 times higher than those of W3110_thr. A high glucose consumption rate (1.44 times) and a low byproduct (acetate) synthesis rate (0.09 time) contributed to this high productivity and high yield. MGF-01 also showed beneficial properties for <span class="small-caps">l</span>-threonine production.</p>                        <a id="5189600" scrollto-destination="5189600"></a>
<div content-id="DSN019TB1" class="table-modal"><div class="table-wrap"><div id="DSN019TB1" class="table-wrap-title" data-id="DSN019TB1"><div class="label -label">Table 1</div><div class="caption"><p>Results of <span class="small-caps">l</span>-threonine production</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>W3110_thr </th><th>MGF-01_thr </th><th>MGF-01/W3110 </th></tr></thead><tbody><tr><td>Threonine (g/L) </td><td>4.4 </td><td>10.6 </td><td>2.44 </td></tr><tr><td>Cell density (OD<sub>660</sub>) </td><td>19.3 </td><td>23.3 </td><td>1.21 </td></tr><tr><td>Glucose consumption (g/dL) </td><td>4.8 </td><td>6.9 </td><td>1.44 </td></tr><tr><td>Yield (% of glucose) </td><td>9.1 </td><td>15.4 </td><td>1.69 </td></tr><tr><td>Acetate (g/L) </td><td>9.4 </td><td>0.85 </td><td>0.09 </td></tr></tbody></table></div><div content-id="DSN019TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>W3110_thr </th><th>MGF-01_thr </th><th>MGF-01/W3110 </th></tr></thead><tbody><tr><td>Threonine (g/L) </td><td>4.4 </td><td>10.6 </td><td>2.44 </td></tr><tr><td>Cell density (OD<sub>660</sub>) </td><td>19.3 </td><td>23.3 </td><td>1.21 </td></tr><tr><td>Glucose consumption (g/dL) </td><td>4.8 </td><td>6.9 </td><td>1.44 </td></tr><tr><td>Yield (% of glucose) </td><td>9.1 </td><td>15.4 </td><td>1.69 </td></tr><tr><td>Acetate (g/L) </td><td>9.4 </td><td>0.85 </td><td>0.09 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5189600" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN019TB1" class="table-wrap-title" data-id="DSN019TB1"><div class="label -label">Table 1</div><div class="caption"><p>Results of <span class="small-caps">l</span>-threonine production</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>W3110_thr </th><th>MGF-01_thr </th><th>MGF-01/W3110 </th></tr></thead><tbody><tr><td>Threonine (g/L) </td><td>4.4 </td><td>10.6 </td><td>2.44 </td></tr><tr><td>Cell density (OD<sub>660</sub>) </td><td>19.3 </td><td>23.3 </td><td>1.21 </td></tr><tr><td>Glucose consumption (g/dL) </td><td>4.8 </td><td>6.9 </td><td>1.44 </td></tr><tr><td>Yield (% of glucose) </td><td>9.1 </td><td>15.4 </td><td>1.69 </td></tr><tr><td>Acetate (g/L) </td><td>9.4 </td><td>0.85 </td><td>0.09 </td></tr></tbody></table></div><div content-id="DSN019TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>W3110_thr </th><th>MGF-01_thr </th><th>MGF-01/W3110 </th></tr></thead><tbody><tr><td>Threonine (g/L) </td><td>4.4 </td><td>10.6 </td><td>2.44 </td></tr><tr><td>Cell density (OD<sub>660</sub>) </td><td>19.3 </td><td>23.3 </td><td>1.21 </td></tr><tr><td>Glucose consumption (g/dL) </td><td>4.8 </td><td>6.9 </td><td>1.44 </td></tr><tr><td>Yield (% of glucose) </td><td>9.1 </td><td>15.4 </td><td>1.69 </td></tr><tr><td>Acetate (g/L) </td><td>9.4 </td><td>0.85 </td><td>0.09 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5189600" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=5189602 id="5189602" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In order to use reduced genome strains for industrial processes, it is necessary to maintain robust growth and the fundamental metabolic mechanism of <em>E</em>. <em>coli</em>. Therefore, we assessed the growth of deletion mutants in M9 minimal medium. Some reduced genome <em>E</em>. <em>coli</em> strains were previously reported; however, the smallest genome <em>E</em>. <em>coli</em>, Δ16, cannot growth in M9 minimal medium.<sup><a reveal-id="DSN019C10" data-open="DSN019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> MDS41 with a 0.66 Mb reduced genome can grow in M9 minimal medium as well as the wild type.<sup><a reveal-id="DSN019C11" data-open="DSN019C11" class="link link-ref link-reveal xref-bibr">11</a></sup> MGF-01 with a 1.03 Mb reduce genome in this study can grow in M9 minimal medium. Moreover, the final cell density in M9 minimal medium of MGF-01 was 1.5 times higher than that of W3110red (Fig. <a reveal-id="DSN019F5" data-open="DSN019F5" class="link link-reveal link-table xref-fig">5</a>). This unexpected but beneficial phenotype was not observed for other reduced genome strains.</p><p>Another distinct phenotype in M9 minimal medium was that the glucose consumption speed of MGF-01 was significantly decreased compared with that of its parental strain, W3110red, after 20 h cultivation instead of its higher cell density (Fig. <a reveal-id="DSN019F5" data-open="DSN019F5" class="link link-reveal link-table xref-fig">5</a>B). MGF-01 accumulated less than half the acetate that W3110red did and this lower acetate accumulation of MGF-01 may contribute to the more efficient glucose assimilation. The low acetate accumulation by MGF-01 may be brought about by the high glyoxylate shunt activity which accelerates acetyl-CoA consumption leading not to acetate but to the TCA cycle.<sup><a reveal-id="DSN019C25" data-open="DSN019C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The gene expression of glyoxylate shunt-related genes, isocitrate lyase (<em>aceA</em>) and malate synthase (<em>aceB</em>), was 6.9 and 4.7 times higher in MGF-01 than those of W3110red in the DNA microarray analysis. Since repressors of the glyoxylate shunt, aerobic respiration control protein (<em>arcA</em>) and acetate operon repressor (<em>iclR</em>), were not eliminated, other regulatory systems may be deleted in MGF-01.<sup><a reveal-id="DSN019C26" data-open="DSN019C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>In <span class="small-caps">l</span>-threonine production medium, MGF-01_thr also showed reduced acetate accumulation from 9.4 g/L (W3110_thr) to 0.85 g/L, and produced twice as much <span class="small-caps">l</span>-threonine as W3110_thr (Table <a reveal-id="DSN019TB1" data-open="DSN019TB1" class="link link-reveal link-table xref-fig">1</a>). It is known that in an <span class="small-caps">l</span>-threonine production strain, the glyoxylate shunt is activated to provide high amounts of carbon to the <span class="small-caps">l</span>-threonine pathway. Low acetate accumulation and high <span class="small-caps">l</span>-threonine production indicate that the glyoxylate shunt in MGF-01_thr may be activated as in the case of M9 minimal medium.<sup><a reveal-id="DSN019C15" data-open="DSN019C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>In M9 minimal medium, glucose consumption was lower than that of the wild type; however, it was higher than that of wild type in <span class="small-caps">l</span>-threonine production medium. This higher glucose consumption rate may be the result of higher <span class="small-caps">l</span>-threonine production by MGF-01_thr. MGF-01 may have higher ability for taking up glucose when there is a sufficient demand such as for <span class="small-caps">l</span>-threonine production. We believe that cellular metabolism may be activated in MGF-01 and glucose is used efficiently for <span class="small-caps">l</span>-threonine production.</p><p>Our next aims are to determine what combinations of deletions affect growth and <span class="small-caps">l</span>-threonine production of MGF-01, and to construct a strain with a further reduced genome.</p>                            <h2 scrollto-destination=5189613 id="5189613" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary material is available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5189615 id="5189615" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The New Energy and Industrial Technology Development Organization (NEDO).</p>                            <h2 scrollto-destination=5189617 id="5189617" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank Dr T. Fujio and Dr M. Hashimoto for the helpful discussions, and Dr K. C. Murphy for providing <em>E</em>. <em>coli</em> strain KM22. This work was carried out as part of The Project for Development of a Technological Infrastructure for Industrial Bioprocesses on R&amp;D of New Industrial Science and Technology Frontiers of the Ministry of Economy, Trade &amp; Industry (METI).</p>                            <h2 scrollto-destination=5189619 id="5189619" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN019C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goeddel</div> <div class="given-names">D. V.</div></div>,  <div class="name"><div class="surname">Heyneker</div> <div class="given-names">H. L.</div></div>,  <div class="name"><div class="surname">Hozumi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Direct expression in <em>Escherichia coli</em> of a DNA sequence coding for human growth hormone</div>, <div class="source">Nature</div> , <div class="year">1979</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">544</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20expression%20in%20Escherichia%20coli%20of%20a%20DNA%20sequence%20coding%20for%20human%20growth%20hormone&amp;author=D.%20V.%20Goeddel&amp;author=H.%20L.%20Heyneker&amp;author=T.%20Hozumi&amp;publication_year=1979&amp;journal=Nature&amp;volume=281&amp;pages=544-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/281544a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F281544a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F281544a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/386136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Udaka</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Isolation of the arginine repressor in <em>Escherichia coli</em></div>, <div class="source">Nature</div> , <div class="year">1970</div>, vol. <div class="volume">228</div> (pg. <div class="fpage">336</div>-<div class="lpage">338</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20the%20arginine%20repressor%20in%20Escherichia%20coli&amp;author=S.%20Udaka&amp;publication_year=1970&amp;journal=Nature&amp;volume=228&amp;pages=336-338" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/228336a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F228336a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F228336a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4919771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikeda</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Amino acid production processes</div>, <div class="source">Adv. Biochem. Eng. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">1</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino%20acid%20production%20processes&amp;author=M.%20Ikeda&amp;publication_year=2003&amp;journal=Adv.%20Biochem.%20Eng.%20Biotechnol.&amp;volume=79&amp;pages=1-35" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12523387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shibasaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ozaki</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Enzymatic production of <em>trans</em>-4-hydroxy-<span class="small-caps">l</span>-proline by region- and sterospecific hydroxylation of L-proline</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">746</div>-<div class="lpage">750</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enzymatic%20production%20of%20trans-4-hydroxy-l-proline%20by%20region-%20and%20sterospecific%20hydroxylation%20of%20L-proline&amp;author=T.%20Shibasaki&amp;author=H.%20Mori&amp;author=A.%20Ozaki&amp;publication_year=2000&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=64&amp;pages=746-750" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.64.746" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.64.746" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.64.746"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10830487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">M. M.</div></div>,  <div class="name"><div class="surname">Lawman</div> <div class="given-names">P. D.</div></div>,  <div class="name"><div class="surname">Cameron</div> <div class="given-names">D. C.</div></div>. </div><div class="article-title">Improving 1,3-propanediol production from glycerol in a metabolically engineered <em>Escherichia coli</em> by reducing accumulation of <em>sn</em>-glycerol-3-phosphate</div>, <div class="source">Biotechnol. Prog.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">694</div>-<div class="lpage">699</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%201%2C3-propanediol%20production%20from%20glycerol%20in%20a%20metabolically%20engineered%20Escherichia%20coli%20by%20reducing%20accumulation%20of%20sn-glycerol-3-phosphate&amp;author=M.%20M.%20Zhu&amp;author=P.%20D.%20Lawman&amp;author=D.%20C.%20Cameron&amp;publication_year=2002&amp;journal=Biotechnol.%20Prog.&amp;volume=18&amp;pages=694-699" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bp020281+" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbp020281+" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbp020281+"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12153300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Oh</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">H. H.</div></div>,  <div class="name"><div class="surname">Hyun</div> <div class="given-names">H. H.</div></div>. </div><div class="article-title">Production of <span class="small-caps">l</span>-threonine by auxotrophs and analogue-resistant mutants of <em>Escherichia coli.</em></div>, <div class="source">Kor. J. Appl. Microbiol. Biotechnol.</div> , <div class="year">1991</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">583</div>-<div class="lpage">587</div>)</div></div></div></div><div content-id="DSN019C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riley</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Arnaud</div> <div class="given-names">M. B.</div></div>, et al. </div><div class="article-title"><em>Escherichia coli</em> K-12: a cooperatively developed annotation snapshot—2005</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20K-12%3A%20a%20cooperatively%20developed%20annotation%20snapshot%E2%80%942005&amp;author=M.%20Riley&amp;author=T.%20Abe&amp;author=M.%20B.%20Arnaud&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16397293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">B. J.</div></div>,  <div class="name"><div class="surname">Sung</div> <div class="given-names">B. H.</div></div>,  <div class="name"><div class="surname">Koob</div> <div class="given-names">M. D.</div></div>, et al. </div><div class="article-title">Minimization of the <em>Escherichia coli</em> genome using a Tn<em>5</em>-targeted Cre/<em>loxP</em> excision system</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2002</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1018</div>-<div class="lpage">1023</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimization%20of%20the%20Escherichia%20coli%20genome%20using%20a%20Tn5-targeted%20Cre%2FloxP%20excision%20system&amp;author=B.%20J.%20Yu&amp;author=B.%20H.%20Sung&amp;author=M.%20D.%20Koob&amp;publication_year=2002&amp;journal=Nat.%20Biotechnol.&amp;volume=20&amp;pages=1018-1023" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12244329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goryshin</div> <div class="given-names">I. Y.</div></div>,  <div class="name"><div class="surname">Naumann</div> <div class="given-names">T. A.</div></div>,  <div class="name"><div class="surname">Apodaca</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Reznikoff</div> <div class="given-names">W. S.</div></div>. </div><div class="article-title">Chromosomal deletion formation system based on Tn<em>5</em> double transposition: use for making minimal genomes and essential gene analysis</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">644</div>-<div class="lpage">653</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20deletion%20formation%20system%20based%20on%20Tn5%20double%20transposition%3A%20use%20for%20making%20minimal%20genomes%20and%20essential%20gene%20analysis&amp;author=I.%20Y.%20Goryshin&amp;author=T.%20A.%20Naumann&amp;author=J.%20Apodaca&amp;author=W.%20S.%20Reznikoff&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=644-653" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.611403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.611403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.611403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hashimoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ichimura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Cell size and nucleoid organization of engineered <em>Escherichia coli</em> cells with a reduced genome</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">137</div>-<div class="lpage">149</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20size%20and%20nucleoid%20organization%20of%20engineered%20Escherichia%20coli%20cells%20with%20a%20reduced%20genome&amp;author=M.%20Hashimoto&amp;author=T.%20Ichimura&amp;author=H.%20Mizoguchi&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=55&amp;pages=137-149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04386.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04386.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04386.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15612923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pósfai</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>III</div>,  <div class="name"><div class="surname">Fehér</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Emergent properties of reduced-genome <em>Escherichia coli</em></div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">1044</div>-<div class="lpage">1046</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emergent%20properties%20of%20reduced-genome%20Escherichia%20coli&amp;author=G.%20P%C3%B3sfai&amp;author=G.%20Plunkett&amp;author=T.%20Feh%C3%A9r&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1044-1046" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1126439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1126439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1126439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16645050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukiya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tobe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Extensive genomic diversity in pathogenic <em>Escherichia coli</em> and <em>Shigella</em> strains revealed by comparative genomic hybridization microarray</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">3911</div>-<div class="lpage">3921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20genomic%20diversity%20in%20pathogenic%20Escherichia%20coli%20and%20Shigella%20strains%20revealed%20by%20comparative%20genomic%20hybridization%20microarray&amp;author=S.%20Fukiya&amp;author=H.%20Mizoguchi&amp;author=T.%20Tobe&amp;author=H.%20Mori&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=3911-3921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.12.3911-3921.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.12.3911-3921.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.12.3911-3921.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15175305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Takai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Okumura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Baba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Datsenko</div> <div class="given-names">K. A.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wanner</div> <div class="given-names">B. L.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Construction of <em>Escherichia coli</em> K-12 in-frame, single-gene knockout mutants: the Keio collection</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div>  <div class="comment">2006.0008.</div></div></div></div></div><div content-id="DSN019C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murphy</div> <div class="given-names">K. C.</div></div>. </div><div class="article-title">Use of bacteriophage lambda recombination functions to promote gene replacement in <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">2063</div>-<div class="lpage">2071</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20bacteriophage%20lambda%20recombination%20functions%20to%20promote%20gene%20replacement%20in%20Escherichia%20coli&amp;author=K.%20C.%20Murphy&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=2063-2071" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9555887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">D. E.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">B. U.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">H. S.</div></div>. </div><div class="article-title">Global analyses of transcriptomes and proteomes of a parent strain and an <span class="small-caps">l</span>-threonine-overproducing mutant strain</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">5442</div>-<div class="lpage">5451</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analyses%20of%20transcriptomes%20and%20proteomes%20of%20a%20parent%20strain%20and%20an%20l-threonine-overproducing%20mutant%20strain&amp;author=J.%20H.%20Lee&amp;author=D.%20E.%20Lee&amp;author=B.%20U.%20Lee&amp;author=H.%20S.%20Kim&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=5442-5451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.18.5442-5451.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.18.5442-5451.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.18.5442-5451.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">D. W.</div></div>. </div>, <div class="source">Molecular Cloning: a Laboratory Manual</div> , <div class="year">1989</div><div class="edition">3rd Ed.</div><div class="publisher-loc">New York</div><div class="publisher-name">Cold Spring Harbor Laboratory Press</div></div></div></div></div><div content-id="DSN019C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blattner</div> <div class="given-names">F. R.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>III</div>,  <div class="name"><div class="surname">Bloch</div> <div class="given-names">C. A.</div></div>, et al. </div><div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</div>, <div class="source">Science</div> , <div class="year">1997</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">1453</div>-<div class="lpage">1474</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;author=F.%20R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.%20A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-1474" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shigenobu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome sequence of the endocellular bacterial symbiont of aphid <em>Buchnera</em> sp. APS</div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">407</div> (pg. <div class="fpage">81</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20endocellular%20bacterial%20symbiont%20of%20aphid%20Buchnera%20sp.%20APS&amp;author=S.%20Shigenobu&amp;author=H.%20Watanabe&amp;author=M.%20Hattori&amp;publication_year=2000&amp;journal=Nature&amp;volume=407&amp;pages=81-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35024074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35024074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35024074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10993077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hashimoto</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Construction of consecutive deletions of the <em>Escherichia coli</em> chromosome</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> pg. <div class="fpage">132</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20consecutive%20deletions%20of%20the%20Escherichia%20coli%20chromosome&amp;author=J.%20Kato&amp;author=M.%20Hashimoto&amp;publication_year=2007&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=3&amp;pages=132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17700540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Overbeek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Larsen</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Walunas</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">The ERGO genome analysis and discovery system</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">164</div>-<div class="lpage">171</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ERGO%20genome%20analysis%20and%20discovery%20system&amp;author=R.%20Overbeek&amp;author=N.%20Larsen&amp;author=T.%20Walunas&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=164-171" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tanaka-Masuda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A simple method for multiple modification of the <em>Escherichia coli</em> K-12 chromosome</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">2905</div>-<div class="lpage">2911</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20multiple%20modification%20of%20the%20Escherichia%20coli%20K-12%20chromosome&amp;author=H.%20Mizoguchi&amp;author=K.%20Tanaka-Masuda&amp;author=H.%20Mori&amp;publication_year=2007&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=71&amp;pages=2905-2911" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.70274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.70274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.70274"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18071259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiraga</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Niki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogura</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Chromosome partitioning in <em>Escherichia coli</em>: novel mutants producing anucleate cells</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1989</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1496</div>-<div class="lpage">1505</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20partitioning%20in%20Escherichia%20coli%3A%20novel%20mutants%20producing%20anucleate%20cells&amp;author=S.%20Hiraga&amp;author=H.%20Niki&amp;author=T.%20Ogura&amp;publication_year=1989&amp;journal=J.%20Bacteriol.&amp;volume=171&amp;pages=1496-1505" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2646284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wada</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kawagoe</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maeda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Masuda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hiraga</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genome-wide analysis of deoxyadenosine methyltransferase-mediated control of gene expression in Escherichia coli</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">673</div>-<div class="lpage">695</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20deoxyadenosine%20methyltransferase-mediated%20control%20of%20gene%20expression%20in%20Escherichia%20coli&amp;author=T.%20Oshima&amp;author=C.%20Wada&amp;author=Y.%20Kawagoe&amp;author=T.%20Ara&amp;author=M.%20Maeda&amp;author=Y.%20Masuda&amp;author=S.%20Hiraga&amp;author=H.%20Mori&amp;publication_year=2002&amp;journal=Mol.%20Microbiol.&amp;volume=45&amp;pages=673-695" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03037.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2002.03037.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2002.03037.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12139615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luli</div> <div class="given-names">G. W.</div></div>,  <div class="name"><div class="surname">Strohl</div> <div class="given-names">W. R.</div></div>. </div><div class="article-title">Comparison of growth, acetate production, and acetate inhibition of <em>Escherichia coli</em> strains in batch and fed-batch fermentations</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1990</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">1004</div>-<div class="lpage">1011</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20growth%2C%20acetate%20production%2C%20and%20acetate%20inhibition%20of%20Escherichia%20coli%20strains%20in%20batch%20and%20fed-batch%20fermentations&amp;author=G.%20W.%20Luli&amp;author=W.%20R.%20Strohl&amp;publication_year=1990&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=56&amp;pages=1004-1011" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2187400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noronha</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Yeh</div> <div class="given-names">H. J.</div></div>,  <div class="name"><div class="surname">Spande</div> <div class="given-names">T. F.</div></div>,  <div class="name"><div class="surname">Shiloach</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Investigation of the TCA cycle and the glyoxylate shunt in <em>Escherichia coli</em> BL21 and JM109 using (13)C-NMR/MS</div>, <div class="source">Biotechnol. Bioeng.</div> , <div class="year">2000</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">316</div>-<div class="lpage">327</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigation%20of%20the%20TCA%20cycle%20and%20the%20glyoxylate%20shunt%20in%20Escherichia%20coli%20BL21%20and%20JM109%20using%20(13)C-NMR%2FMS&amp;author=S.%20B.%20Noronha&amp;author=H.%20J.%20Yeh&amp;author=T.%20F.%20Spande&amp;author=J.%20Shiloach&amp;publication_year=2000&amp;journal=Biotechnol.%20Bioeng.&amp;volume=68&amp;pages=316-327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10745200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN019C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN019C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cronan</div> <div class="given-names">J. E.</div>Jr</div>,  <div class="name"><div class="surname">LaPorte</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Tricarboxylic acid cycle and glyoxylate bypass</div>, <div class="source">Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd Ed.</div> , <div class="year">1996</div><div class="publisher-loc">Washington D. C.</div><div class="publisher-name">American Society for Microbiology</div>(pg. <div class="fpage">206</div>-<div class="lpage">216</div>)</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359806&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn019/2/dsn019-dsn019supp.pdf?Expires=2147483647&amp;Signature=HMVWJOrwbcVRPdLoYkqjoO7MJtR3U~WXYhkC8LFgqJFar-pTLFqnWLFGL1zy1yxlVYhEAy6GL7A4dySMGjYoXqAeP~0Zu0bO0VCcLQp5T4pjhF3p7E~21vL5gXIOqvhE0FAA4lwK7UdqR~-hprJgeSh5pLiyudhmPPQkSqhNbdezFHokrAdfmhLSGTiX5TmBwGv9tOjpAqoTFjplJ6SjuX2C2aJYz1X~KkBb-eeeXl1pyQEjK7GP3drKlO5tj6s8VDEzRhSmFwkc-wtpf0eCGv0zmiduXN3OkqoOi5DtQyhOzN5zq2~b~J2rk4DuXkozKZ1Xbq4BMfpUYx5FIKMoHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">322</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">33</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359806"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f5%2f277%2f359806">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260843400002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (33)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F5%2F277%2F359806" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359806'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Superpositioning of Deletions Promotes Growth of Escherichia coli with a Reduced Genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f277%2f359806">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

