

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/3/265/4793385","@type":"ScholarlyArticle","name":"Cell-type-resolved alternative splicing patterns in mouse liver","datePublished":"2018-01-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2018-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx055","keywords":["alternative splicing","cell specificity","hepatic cell types","splicing factor","isoform function"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/3/265/4793385","author":[{"name":"Wu, Peng","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"Zhou, Donghu","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"Lin, Weiran","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"Li, Yanyan","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"Wei, Handong","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"Qian, Xiaohong","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"Jiang, Ying","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"},{"name":"He, Fuchu","affiliation":"State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) is an important post-transcriptional regulatory mechanism to generate transcription diversity. However, the functional role","pageStart":"265","pageEnd":"275","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Cell-type-resolved alternative splicing patterns in mouse liver","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Cell-type-resolved alternative splicing patterns in mouse liver" />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) is an important post-transcriptional regulatory mechanism to generate transcription diversity. However, the functional role" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/3/265/4793385" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-01-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wu, Peng" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="Zhou, Donghu" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="Lin, Weiran" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="Li, Yanyan" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="Wei, Handong" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="Qian, Xiaohong" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="Jiang, Ying" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_author" content="He, Fuchu" /><meta name="citation_author_institution" content="State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China" /><meta name="citation_title" content="Cell-type-resolved alternative splicing patterns in mouse liver" /><meta name="citation_firstpage" content="265" /><meta name="citation_lastpage" content="275" /><meta name="citation_doi" content="10.1093/dnares/dsx055" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2018/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Expansion of the eukaryotic proteome by alternative splicing;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=457-63; " /><meta name="citation_reference" content="citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing;  citation_journal_title=Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=1413-5; " /><meta name="citation_reference" content="citation_title=Alternative isoform regulation in human tissue transcriptomes;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=470-6; " /><meta name="citation_reference" content="citation_title=Transcriptional diversity during lineage commitment of human blood progenitors;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=1251033" /><meta name="citation_reference" content="citation_title=Human tRNA synthetase catalytic nulls with diverse functions;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=328-32; " /><meta name="citation_reference" content="citation_title=Tissue-specific splicing of disordered segments that embed binding motifs rewires protein interaction networks;  citation_journal_title=Mol. Cell;  citation_year=2012;  citation_volume=46;  citation_pages=871-83; " /><meta name="citation_reference" content="citation_title=Tissue-specific alternative splicing remodels protein-protein interaction networks;  citation_journal_title=Mol. Cell;  citation_year=2012;  citation_volume=46;  citation_pages=884-92; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of gene and isoform regulation in mammalian tissues;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1593-9; " /><meta name="citation_reference" content="citation_title=The evolutionary landscape of alternative splicing in vertebrate species;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1587-93; " /><meta name="citation_reference" content="citation_title=Meta-analysis of RNA-seq expression data across species, tissues and studies;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=287" /><meta name="citation_reference" content="citation_title=Differential splicing across immune system lineages;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=14324-9; " /><meta name="citation_reference" content="citation_title=Cooperation of liver cells in health and disease;  citation_journal_title=Adv. Anat. Embryol. Cell Biol;  citation_year=2001;  citation_volume=161;  citation_pages=1-151; " /><meta name="citation_reference" content="citation_title=Long-term coculture strategies for primary hepatocytes and liver sinusoidal endothelial cells;  citation_journal_title=Tissue Eng. Part C Methods;  citation_year=2015;  citation_volume=21;  citation_pages=413-22; " /><meta name="citation_reference" content="citation_title=Liver resident macrophages (Kupffer Cells) share several functional antigens in common with endothelial cells;  citation_journal_title=Scand. J. Immunol;  citation_year=2016;  citation_volume=83;  citation_pages=139-50; " /><meta name="citation_reference" content="citation_title=Dioscin alleviates alcoholic liver fibrosis by attenuating hepatic stellate cell activation via the TLR4/MyD88/NF-kappaB signaling pathway;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=5;  citation_pages=18038" /><meta name="citation_reference" content="citation_title=Cell-type-resolved quantitative proteomics of murine liver;  citation_journal_title=Cell Metab;  citation_year=2014;  citation_volume=20;  citation_pages=1076-87; " /><meta name="citation_reference" content="citation_title=A Cell-type-resolved liver proteome;  citation_journal_title=Mol. Cell Proteomics;  citation_year=2016;  citation_volume=15;  citation_pages=3190-202; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=MATS: a Bayesian framework for flexible detection of differential alternative splicing from RNA-Seq data;  citation_journal_title=Nucl. Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e61" /><meta name="citation_reference" content="citation_title=The Pfam protein families database;  citation_journal_title=Nucl. Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=D211-22; " /><meta name="citation_reference" content="citation_title=IUPred: web server for the prediction of intrinsically unstructured regions of proteins based on estimated energy content;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3433-4; " /><meta name="citation_reference" content="citation_title=Prediction of protein binding regions in disordered proteins;  citation_journal_title=PLoS Comput. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=e1000376" /><meta name="citation_reference" content="citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists;  citation_journal_title=Nucl. Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Enrichment map: a network-based method for gene-set enrichment visualization and interpretation;  citation_journal_title=PloS One;  citation_year=2010;  citation_volume=5;  citation_pages=e13984" /><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2498-504; " /><meta name="citation_reference" content="citation_title=RBPmap: a web server for mapping binding sites of RNA-binding proteins;  citation_journal_title=Nucl. Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=W361-7; " /><meta name="citation_reference" content="citation_title=Ion channels and oxidative stress as a potential link for the diagnosis or treatment of liver diseases;  citation_journal_title=Oxid. Med. Cell. Longev;  citation_year=2016;  citation_volume=2016;  citation_pages=3928714" /><meta name="citation_reference" content="citation_title=Alternative splicing and gene structure of the transforming growth factor beta-activated kinase 1;  citation_journal_title=Biochim Biophys Acta;  citation_year=2000;  citation_volume=1517;  citation_pages=46-52; " /><meta name="citation_reference" content="citation_title=Liver Sinusoidal Endothelial Cells;  citation_journal_title=Comp. Physiol;  citation_year=2015;  citation_volume=5;  citation_pages=1751-74; " /><meta name="citation_reference" content="citation_title=Sinusoidal communication in liver fibrosis and regeneration;  citation_journal_title=J. Hepatol;  citation_year=2016;  citation_volume=65;  citation_pages=608-17; " /><meta name="citation_reference" content="citation_title=Isolation and co-culture of rat parenchymal and non-parenchymal liver cells to evaluate cellular interactions and response;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=25329" /><meta name="citation_reference" content="citation_title=Alternative splicing in the mammalian nervous system: recent insights into mechanisms and functional roles;  citation_journal_title=Neuron;  citation_year=2015;  citation_volume=87;  citation_pages=14-27; " /><meta name="citation_reference" content="citation_title=TNNT1, TNNT2, and TNNT3: isoform genes, regulation, and structure-function relationships;  citation_journal_title=Gene;  citation_year=2016;  citation_volume=582;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis reveals SR protein cooperation and competition in regulated splicing;  citation_journal_title=Mol. Cell;  citation_year=2013;  citation_volume=50;  citation_pages=223-35; " /><meta name="citation_reference" content="citation_title=MIsoMine: a genome-scale high-resolution data portal of expression, function and networks at the splice isoform level in the mouse;  citation_journal_title=Database (Oxford);  citation_year=2015;  citation_volume=2015; " /><meta name="citation_reference" content="citation_title=Mitogen-activated protein kinases in innate immunity;  citation_journal_title=Nat. Rev. Immunol;  citation_year=2013;  citation_volume=13;  citation_pages=679-92; " /><meta name="citation_reference" content="citation_title=An alternative splicing event amplifies evolutionary differences between vertebrates;  citation_journal_title=Science;  citation_year=2015;  citation_volume=349;  citation_pages=868-73; " /><meta name="citation_reference" content="citation_title=Alternative splicing: decoding an expansive regulatory layer;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2012;  citation_volume=24;  citation_pages=323-32; " /><meta name="citation_reference" content="citation_title=Building robust transcriptomes with master splicing factors;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=159;  citation_pages=487-98; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/3/265/25072219/dsx055.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) is an important post-transcriptional regulatory mechanism to generate transcription diversity. However, the functional role" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/3/265/4793385" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/3/265/4793385" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/3/265/4793385");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx055';
    </script>
<title>
	Cell-type-resolved alternative splicing patterns in mouse liver | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4793385" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb9mPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXdSwgUGVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFByb3Rlb21pY3MsIE5hdGlvbmFsIENlbnRlciBmb3IgUHJvdGVpbiBTY2llbmNlcyBCZWlqaW5nLCBCZWlqaW5nIFByb3Rlb21lIFJlc2VhcmNoIENlbnRlciwgQmVpamluZyBJbnN0aXR1dGUgb2YgTGlmZW9taWNzLCBCZWlqaW5nIDEwMjIwNiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaG91LCBEb25naHUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQcm90ZW9taWNzLCBOYXRpb25hbCBDZW50ZXIgZm9yIFByb3RlaW4gU2NpZW5jZXMgQmVpamluZywgQmVpamluZyBQcm90ZW9tZSBSZXNlYXJjaCBDZW50ZXIsIEJlaWppbmcgSW5zdGl0dXRlIG9mIExpZmVvbWljcywgQmVpamluZyAxMDIyMDYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGluLCBXZWlyYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQcm90ZW9taWNzLCBOYXRpb25hbCBDZW50ZXIgZm9yIFByb3RlaW4gU2NpZW5jZXMgQmVpamluZywgQmVpamluZyBQcm90ZW9tZSBSZXNlYXJjaCBDZW50ZXIsIEJlaWppbmcgSW5zdGl0dXRlIG9mIExpZmVvbWljcywgQmVpamluZyAxMDIyMDYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFlhbnlhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFByb3Rlb21pY3MsIE5hdGlvbmFsIENlbnRlciBmb3IgUHJvdGVpbiBTY2llbmNlcyBCZWlqaW5nLCBCZWlqaW5nIFByb3Rlb21lIFJlc2VhcmNoIENlbnRlciwgQmVpamluZyBJbnN0aXR1dGUgb2YgTGlmZW9taWNzLCBCZWlqaW5nIDEwMjIwNiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWksIEhhbmRvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQcm90ZW9taWNzLCBOYXRpb25hbCBDZW50ZXIgZm9yIFByb3RlaW4gU2NpZW5jZXMgQmVpamluZywgQmVpamluZyBQcm90ZW9tZSBSZXNlYXJjaCBDZW50ZXIsIEJlaWppbmcgSW5zdGl0dXRlIG9mIExpZmVvbWljcywgQmVpamluZyAxMDIyMDYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUWlhbiwgWGlhb2hvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQcm90ZW9taWNzLCBOYXRpb25hbCBDZW50ZXIgZm9yIFByb3RlaW4gU2NpZW5jZXMgQmVpamluZywgQmVpamluZyBQcm90ZW9tZSBSZXNlYXJjaCBDZW50ZXIsIEJlaWppbmcgSW5zdGl0dXRlIG9mIExpZmVvbWljcywgQmVpamluZyAxMDIyMDYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmlhbmcsIFlpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQcm90ZW9taWNzLCBOYXRpb25hbCBDZW50ZXIgZm9yIFByb3RlaW4gU2NpZW5jZXMgQmVpamluZywgQmVpamluZyBQcm90ZW9tZSBSZXNlYXJjaCBDZW50ZXIsIEJlaWppbmcgSW5zdGl0dXRlIG9mIExpZmVvbWljcywgQmVpamluZyAxMDIyMDYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGUsIEZ1Y2h1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgUHJvdGVvbWljcywgTmF0aW9uYWwgQ2VudGVyIGZvciBQcm90ZWluIFNjaWVuY2VzIEJlaWppbmcsIEJlaWppbmcgUHJvdGVvbWUgUmVzZWFyY2ggQ2VudGVyLCBCZWlqaW5nIEluc3RpdHV0ZSBvZiBMaWZlb21pY3MsIEJlaWppbmcgMTAyMjA2LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ2VsbC10eXBlLXJlc29sdmVkIGFsdGVybmF0aXZlIHNwbGljaW5nIHBhdHRlcm5zIGluIG1vdXNlIGxpdmVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGFuc2lvbiBvZiB0aGUgZXVrYXJ5b3RpYyBwcm90ZW9tZSBieSBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2MzsgIGNpdGF0aW9uX3BhZ2VzPTQ1Ny02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVlcCBzdXJ2ZXlpbmcgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgY29tcGxleGl0eSBpbiB0aGUgaHVtYW4gdHJhbnNjcmlwdG9tZSBieSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0xNDEzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIGlzb2Zvcm0gcmVndWxhdGlvbiBpbiBodW1hbiB0aXNzdWUgdHJhbnNjcmlwdG9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTY7ICBjaXRhdGlvbl9wYWdlcz00NzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIGRpdmVyc2l0eSBkdXJpbmcgbGluZWFnZSBjb21taXRtZW50IG9mIGh1bWFuIGJsb29kIHByb2dlbml0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0NTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTEwMzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiB0Uk5BIHN5bnRoZXRhc2UgY2F0YWx5dGljIG51bGxzIHdpdGggZGl2ZXJzZSBmdW5jdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ1OyAgY2l0YXRpb25fcGFnZXM9MzI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaXNzdWUtc3BlY2lmaWMgc3BsaWNpbmcgb2YgZGlzb3JkZXJlZCBzZWdtZW50cyB0aGF0IGVtYmVkIGJpbmRpbmcgbW90aWZzIHJld2lyZXMgcHJvdGVpbiBpbnRlcmFjdGlvbiBuZXR3b3JrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9ODcxLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaXNzdWUtc3BlY2lmaWMgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgcmVtb2RlbHMgcHJvdGVpbi1wcm90ZWluIGludGVyYWN0aW9uIG5ldHdvcmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz04ODQtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBkeW5hbWljcyBvZiBnZW5lIGFuZCBpc29mb3JtIHJlZ3VsYXRpb24gaW4gbWFtbWFsaWFuIHRpc3N1ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM4OyAgY2l0YXRpb25fcGFnZXM9MTU5My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uYXJ5IGxhbmRzY2FwZSBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiB2ZXJ0ZWJyYXRlIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM4OyAgY2l0YXRpb25fcGFnZXM9MTU4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YS1hbmFseXNpcyBvZiBSTkEtc2VxIGV4cHJlc3Npb24gZGF0YSBhY3Jvc3Mgc3BlY2llcywgdGlzc3VlcyBhbmQgc3R1ZGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgc3BsaWNpbmcgYWNyb3NzIGltbXVuZSBzeXN0ZW0gbGluZWFnZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTE0MzI0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvb3BlcmF0aW9uIG9mIGxpdmVyIGNlbGxzIGluIGhlYWx0aCBhbmQgZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBBbmF0LiBFbWJyeW9sLiBDZWxsIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MS0xNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmctdGVybSBjb2N1bHR1cmUgc3RyYXRlZ2llcyBmb3IgcHJpbWFyeSBoZXBhdG9jeXRlcyBhbmQgbGl2ZXIgc2ludXNvaWRhbCBlbmRvdGhlbGlhbCBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGlzc3VlIEVuZy4gUGFydCBDIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz00MTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpdmVyIHJlc2lkZW50IG1hY3JvcGhhZ2VzIChLdXBmZmVyIENlbGxzKSBzaGFyZSBzZXZlcmFsIGZ1bmN0aW9uYWwgYW50aWdlbnMgaW4gY29tbW9uIHdpdGggZW5kb3RoZWxpYWwgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjYW5kLiBKLiBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9MTM5LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaW9zY2luIGFsbGV2aWF0ZXMgYWxjb2hvbGljIGxpdmVyIGZpYnJvc2lzIGJ5IGF0dGVudWF0aW5nIGhlcGF0aWMgc3RlbGxhdGUgY2VsbCBhY3RpdmF0aW9uIHZpYSB0aGUgVExSNC9NeUQ4OC9ORi1rYXBwYUIgc2lnbmFsaW5nIHBhdGh3YXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xODAzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbGwtdHlwZS1yZXNvbHZlZCBxdWFudGl0YXRpdmUgcHJvdGVvbWljcyBvZiBtdXJpbmUgbGl2ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgTWV0YWI7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMDc2LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIENlbGwtdHlwZS1yZXNvbHZlZCBsaXZlciBwcm90ZW9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsIFByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zMTkwLTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGFzc2VtYmx5IGFuZCBxdWFudGlmaWNhdGlvbiBieSBSTkEtU2VxIHJldmVhbHMgdW5hbm5vdGF0ZWQgdHJhbnNjcmlwdHMgYW5kIGlzb2Zvcm0gc3dpdGNoaW5nIGR1cmluZyBjZWxsIGRpZmZlcmVudGlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NTExLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BVFM6IGEgQmF5ZXNpYW4gZnJhbWV3b3JrIGZvciBmbGV4aWJsZSBkZXRlY3Rpb24gb2YgZGlmZmVyZW50aWFsIGFsdGVybmF0aXZlIHNwbGljaW5nIGZyb20gUk5BLVNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1lNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1EMjExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JVVByZWQ6IHdlYiBzZXJ2ZXIgZm9yIHRoZSBwcmVkaWN0aW9uIG9mIGludHJpbnNpY2FsbHkgdW5zdHJ1Y3R1cmVkIHJlZ2lvbnMgb2YgcHJvdGVpbnMgYmFzZWQgb24gZXN0aW1hdGVkIGVuZXJneSBjb250ZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM0MzMtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiBwcm90ZWluIGJpbmRpbmcgcmVnaW9ucyBpbiBkaXNvcmRlcmVkIHByb3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAzNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9pbmZvcm1hdGljcyBlbnJpY2htZW50IHRvb2xzOiBwYXRocyB0b3dhcmQgdGhlIGNvbXByZWhlbnNpdmUgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiBsYXJnZSBnZW5lIGxpc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnJpY2htZW50IG1hcDogYSBuZXR3b3JrLWJhc2VkIG1ldGhvZCBmb3IgZ2VuZS1zZXQgZW5yaWNobWVudCB2aXN1YWxpemF0aW9uIGFuZCBpbnRlcnByZXRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMzk4NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9zY2FwZTogYSBzb2Z0d2FyZSBlbnZpcm9ubWVudCBmb3IgaW50ZWdyYXRlZCBtb2RlbHMgb2YgYmlvbW9sZWN1bGFyIGludGVyYWN0aW9uIG5ldHdvcmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjQ5OC01MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJCUG1hcDogYSB3ZWIgc2VydmVyIGZvciBtYXBwaW5nIGJpbmRpbmcgc2l0ZXMgb2YgUk5BLWJpbmRpbmcgcHJvdGVpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPVczNjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW9uIGNoYW5uZWxzIGFuZCBveGlkYXRpdmUgc3RyZXNzIGFzIGEgcG90ZW50aWFsIGxpbmsgZm9yIHRoZSBkaWFnbm9zaXMgb3IgdHJlYXRtZW50IG9mIGxpdmVyIGRpc2Vhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PeGlkLiBNZWQuIENlbGwuIExvbmdldjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDE2OyAgY2l0YXRpb25fcGFnZXM9MzkyODcxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGFuZCBnZW5lIHN0cnVjdHVyZSBvZiB0aGUgdHJhbnNmb3JtaW5nIGdyb3d0aCBmYWN0b3IgYmV0YS1hY3RpdmF0ZWQga2luYXNlIDE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW0gQmlvcGh5cyBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1MTc7ICBjaXRhdGlvbl9wYWdlcz00Ni01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGl2ZXIgU2ludXNvaWRhbCBFbmRvdGhlbGlhbCBDZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcC4gUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTc1MS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ludXNvaWRhbCBjb21tdW5pY2F0aW9uIGluIGxpdmVyIGZpYnJvc2lzIGFuZCByZWdlbmVyYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEhlcGF0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz02MDgtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY28tY3VsdHVyZSBvZiByYXQgcGFyZW5jaHltYWwgYW5kIG5vbi1wYXJlbmNoeW1hbCBsaXZlciBjZWxscyB0byBldmFsdWF0ZSBjZWxsdWxhciBpbnRlcmFjdGlvbnMgYW5kIHJlc3BvbnNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjUzMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiB0aGUgbWFtbWFsaWFuIG5lcnZvdXMgc3lzdGVtOiByZWNlbnQgaW5zaWdodHMgaW50byBtZWNoYW5pc21zIGFuZCBmdW5jdGlvbmFsIHJvbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXVyb247ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9ODc7ICBjaXRhdGlvbl9wYWdlcz0xNC0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VE5OVDEsIFROTlQyLCBhbmQgVE5OVDM6IGlzb2Zvcm0gZ2VuZXMsIHJlZ3VsYXRpb24sIGFuZCBzdHJ1Y3R1cmUtZnVuY3Rpb24gcmVsYXRpb25zaGlwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01ODI7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyByZXZlYWxzIFNSIHByb3RlaW4gY29vcGVyYXRpb24gYW5kIGNvbXBldGl0aW9uIGluIHJlZ3VsYXRlZCBzcGxpY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MjIzLTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NSXNvTWluZTogYSBnZW5vbWUtc2NhbGUgaGlnaC1yZXNvbHV0aW9uIGRhdGEgcG9ydGFsIG9mIGV4cHJlc3Npb24sIGZ1bmN0aW9uIGFuZCBuZXR3b3JrcyBhdCB0aGUgc3BsaWNlIGlzb2Zvcm0gbGV2ZWwgaW4gdGhlIG1vdXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EYXRhYmFzZSAoT3hmb3JkKTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvZ2VuLWFjdGl2YXRlZCBwcm90ZWluIGtpbmFzZXMgaW4gaW5uYXRlIGltbXVuaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gSW1tdW5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTY3OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnQgYW1wbGlmaWVzIGV2b2x1dGlvbmFyeSBkaWZmZXJlbmNlcyBiZXR3ZWVuIHZlcnRlYnJhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM0OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmc6IGRlY29kaW5nIGFuIGV4cGFuc2l2ZSByZWd1bGF0b3J5IGxheWVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBDZWxsIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0zMjMtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJ1aWxkaW5nIHJvYnVzdCB0cmFuc2NyaXB0b21lcyB3aXRoIG1hc3RlciBzcGxpY2luZyBmYWN0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzMvMjY1LzI1MDcyMjE5L2RzeDA1NS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWx0ZXJuYXRpdmUgc3BsaWNpbmcgKEFTKSBpcyBhbiBpbXBvcnRhbnQgcG9zdC10cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBtZWNoYW5pc20gdG8gZ2VuZXJhdGUgdHJhbnNjcmlwdGlvbiBkaXZlcnNpdHkuIEhvd2V2ZXIsIHRoZSBmdW5jdGlvbmFsIHJvbGUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ3OTMzODUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkqbOaVhRZR8t8XIoCPTxQzAeb0o0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f265%2f4793385" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f265%2f4793385" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109141" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&amp;Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/3">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="118057690">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057690">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057693">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057693">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057698">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057698">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057757">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057757">3. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057762">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057762">4. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057792">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057792">5. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057799">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057799">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057805">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057805">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/3/257/4791395">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/3/277/4816134">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Cell-type-resolved alternative splicing patterns in mouse liver
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peng Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peng+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Peng Wu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Donghu Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Donghu+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Donghu Zhou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Weiran Lin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Weiran+Lin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lin W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lin W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Weiran Lin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanyan Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanyan+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yanyan Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Handong Wei</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Handong+Wei">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wei H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wei H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Handong Wei</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaohong Qian</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaohong+Qian">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qian X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qian X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaohong Qian</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ying Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx055-cor1" data-open="dsx055-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel and Fax. +8610 80705299. Email: <a href="mailto:jiangying304@hotmail.com" target="_blank">jiangying304@hotmail.com</a> (Y.J.); Tel and Fax. +8610 68171208. Email: <a href="mailto:hefc@nic.bmi.ac.cn" target="_blank">hefc@nic.bmi.ac.cn</a> (F.H.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ying+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ying Jiang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fuchu He</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Proteomics, National Center for Protein Sciences Beijing, Beijing Proteome Research Center, Beijing Institute of Lifeomics, Beijing 102206, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx055-cor1" data-open="dsx055-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel and Fax. +8610 80705299. Email: <a href="mailto:jiangying304@hotmail.com" target="_blank">jiangying304@hotmail.com</a> (Y.J.); Tel and Fax. +8610 68171208. Email: <a href="mailto:hefc@nic.bmi.ac.cn" target="_blank">hefc@nic.bmi.ac.cn</a> (F.H.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fuchu+He">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=He F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;He F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fuchu He</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx055-FM1"><div class="fn"><div class="footnote-content">Peng Wu and Donghu Zhou authors are Co-first authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx055-FM2"><div class="fn"><div class="footnote-content">Edited by Dr Minoru Yoshida</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 265–275, <a href='https://doi.org/10.1093/dnares/dsx055'>https://doi.org/10.1093/dnares/dsx055</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 January 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 September 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 December 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4793385" href="/dnaresearch/article-pdf/25/3/265/25072219/dsx055.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Peng Wu, Donghu Zhou, Weiran Lin, Yanyan Li, Handong Wei, Xiaohong Qian, Ying Jiang, Fuchu He;  Cell-type-resolved alternative splicing patterns in mouse liver, <em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 265–275, <a href='https://doi.org/10.1093/dnares/dsx055'>https://doi.org/10.1093/dnares/dsx055</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4793385?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Cell-type-resolved%20alternative%20splicing%20patterns%20in%20mouse%20liver&amp;publicationDate=2018-01-08&amp;volumeNum=25&amp;issueNum=3&amp;author=Wu%2C%20Peng%3B%20Zhou%2C%20Donghu&amp;startPage=265&amp;endPage=275&amp;contentId=10.1093%2Fdnares%2Fdsx055&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f265%2f4793385" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=118057690 id="118057690" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) is an important post-transcriptional regulatory mechanism to generate transcription diversity. However, the functional roles of AS in multiple cell types from one organ have not been reported. Here, we provide the most comprehensive profile for cell-type-resolved AS patterns in mouse liver. A total of 13,637 AS events are detected, representing 81.5% of all known AS events in the database. About 46.2% of multi-exon genes undergo AS from the four cell types of mouse liver: hepatocyte, liver sinusoidal endothelial cell, Kupffer cell and hepatic stellate cell, which regulates cell-specific functions and maintains cell characteristics. We also present a cell-type-specific splicing factors network in these four cell types of mouse liver, allowing data mining and generating knowledge to elucidate the roles of splicing factors in sustaining the cell-type-specialized AS profiles and functions. The splicing switching of <em>Tak1</em> gene between different cell types is firstly discovered and the specific <em>Tak1</em> isoform regulates hepatic cell-type-specific functions is verified. Thus, our work constructs a hepatic cell-specific splicing landscape and reveals the considerable contribution of AS to the cell type constitution and organ features.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cell specificity&quot;">cell specificity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;hepatic cell types&quot;">hepatic cell types</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;splicing factor&quot;">splicing factor</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;isoform function&quot;">isoform function</a></div>                            <h2 scrollto-destination=118057693 id="118057693" class="section-title" >1. Introduction</h2>
<p>Alternative splicing (AS) plays a major role in increasing transcriptome variability and proteome diversity.<a reveal-id="dsx055-B1" data-open="dsx055-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx055-B2" data-open="dsx055-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> It is estimated that 95% of multi-exon genes in human undergo AS,<a reveal-id="dsx055-B3" data-open="dsx055-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and the resulting isoforms from one gene could present identical, similar or even opposite protein functions.<a reveal-id="dsx055-B4" data-open="dsx055-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx055-B5" data-open="dsx055-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> AS variants are variably expressed between different tissues and cell types, and most of the recent studies focused on the varying effects of AS in different species or tissue types.<a reveal-id="dsx055-B4" data-open="dsx055-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx055-B6" data-open="dsx055-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx055-B7" data-open="dsx055-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> AS is highly prevalent across distinct species, and the divergences in AS patterns significantly contribute to shaping species-specific differences.<a reveal-id="dsx055-B8 dsx055-B9 dsx055-B10" data-open="dsx055-B8 dsx055-B9 dsx055-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> Furthermore, it was suggested that proteins with tissue-specific isoforms were likely to occupy key hubs in protein interaction networks.<a reveal-id="dsx055-B4" data-open="dsx055-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx055-B6" data-open="dsx055-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx055-B7" data-open="dsx055-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx055-B11" data-open="dsx055-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Although the roles of AS in tissue specificity have been well-studied, splicing differences at the cellular level and in multiple cell types from one single organ were almost neglected to date.</p><p>Thus, we expected to establish splicing profiling of four cell types from the mouse liver as a paradigm of exploring cell-type-specific splicing patterns. The liver is the largest metabolic organ of the mammals and participates in multiple biological functions, the majority of which are performed by parenchymal cells, also named hepatocytes (HCs), occupying approximately 80% of the liver volume.<a reveal-id="dsx055-B12" data-open="dsx055-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The remainders are nonparenchymal cells (NPCs), including the liver sinusoidal endothelial cells (LSECs), the Kupffer cells (KCs), the hepatic stellate cells (HSCs) and several rare cell types.<a reveal-id="dsx055-B12" data-open="dsx055-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Despite the small fraction in total liver volume, NPCs also perform significant functions.<a reveal-id="dsx055-B13 dsx055-B14 dsx055-B15" data-open="dsx055-B13 dsx055-B14 dsx055-B15" class="link link-ref link-reveal xref-bibr"><sup>13–15</sup></a></p><p>Recently, although some cell-type-specific proteomic analyses had performed in mouse liver<a reveal-id="dsx055-B16" data-open="dsx055-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx055-B17" data-open="dsx055-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> that globally investigated the proteome profiles of individual cell types and possible mechanisms of intercellular crosstalk, we still want to know how this proteomic complexity was generated and how AS contributed in such processes. Given that tissue-specific exons play crucial roles in determining tissue identity, we wondered whether this feature also expands to the cellular level in the liver and how AS regulates the biological functions of isoforms in different cell types. Thus, in our study, we obtained highly purified liver cells and applied RNA-Seq to build the first dataset of cell-type-specific transcriptomic profiling. Next, we comprehensively compared the splicing patterns between the four hepatic cell types and analyzed their biological characteristics. Furthermore, we identified the cell-type-specific splicing factors that dominated the splicing patterns and functional characteristics in respective cell types. Last, we took HCs-specific Tak1 isoform as an example to demonstrate the effect of splicing switching on cell-type specificities. Overall, our results revealed a hepatic cell-type-specific splicing profile that played a considerable role in determining cellular-specific functions.</p>                            <h2 scrollto-destination=118057698 id="118057698" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=118057700 id="118057700" class="section-title" >2.1. Isolation of four hepatic cell types</h3>
<p>C57BL/6 male mice (8–10 weeks old) were killed, and their livers were excised by operation according to the ethics committee guidelines of the Beijing Institute of Radiation Medicine. To reduce the biological variation among individuals, the liver samples from three mice were pooled for the sequencing of each cell type. All procedures were reported before.<a reveal-id="dsx055-B17" data-open="dsx055-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p>                            <h3 scrollto-destination=118057705 id="118057705" class="section-title" >2.2. RNA-seq data generation and processing</h3>
<p>Total RNA of four hepatic cell types was isolated with Trizol Reagent (Invitrogen) according to the manufacturer’s protocol. RNA quality was assessed using Agilent 2100 Bioanalyzer. RNA-seq library was prepared using Illumina’s reagent and protocol. Paried-end 90 bp sequencing was performed on an Illumina HiSeq 2000 genome analyzer. Clean reads were mapped to the mouse GENCODE M4 (GRCm38) using Tophat2 (version 2.0.10)<a reveal-id="dsx055-B18" data-open="dsx055-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> with a mate inner size of 20. FPKM were then calculated as relative abundance value for each gene and transcript by Cufflinks (version 2.2.1)<a reveal-id="dsx055-B19" data-open="dsx055-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> supplied with a protein-coding gene model annotation file in GTF format (GENCODE M4).</p>                            <h3 scrollto-destination=118057709 id="118057709" class="section-title" >2.3. Differential analysis of AS between four cell types</h3>
<p>MATS<a reveal-id="dsx055-B20" data-open="dsx055-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> was applied to identify the differential alternative splicing between four cell types with options ‘-t paired –len 90 –a 8 –c 0.0001 –expressionChange 20’. Significant different events were filtered under the threshold of FDR &lt; 0.05 for five common splicing classes. Percent spliced-in (PSI) values for each alternative exon were extracted from MATS outputs.</p>                            <h3 scrollto-destination=118057714 id="118057714" class="section-title" >2.4. Feature analysis of CEEs</h3>
<p>Cell-type-enriched exons (CEEs) required statistically significant differences (ΔPSI &gt; 0.25 and FDR &lt; 0.05) between at least two cell types. All exons were mapped to annotated proteins (Ensembl release 78) using the method of Ensembl Perl API. Protein domains of Pfam<a reveal-id="dsx055-B21" data-open="dsx055-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> were extracted from Ensembl annotations for mapped exons. IUPred<a reveal-id="dsx055-B22" data-open="dsx055-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was applied to predict protein disorder information. The fraction of segments classified as disordered by the IUPred cutoff of 0.4. ANCHOR<a reveal-id="dsx055-B23" data-open="dsx055-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> was used for predicting protein binding regions. PTM sites were collected from the UniProt database after mapping the Ensembl protein ID to UniProt accessions. The statistically significant differences were assessed by χ<sup>2</sup> test.</p>                            <h3 scrollto-destination=118057719 id="118057719" class="section-title" >2.5. Splicing validation by RT-PCR</h3>
<p>Total RNA was reverse transcribed using the PrimeScript<sup>TM</sup> RT Master Mix (TaKaRa Bio Technology), and PCR was performed using Premix Taq<sup>TM</sup> (TaKaRa Taq<sup>TM</sup> Version 2.0 plus dye, TaKaRa BioTechnology). The PCR products were separated by electrophoresis in 2% agarose gels and detected using the Gel Doc EZ System (Bio-Rad, USA). Sequencing of the PCR products was performed by BGI.tech (Beijing, China). All the primers were listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>.</p>                            <h3 scrollto-destination=118057724 id="118057724" class="section-title" >2.6. GO and pathway enrichment analysis</h3>
<p>All enrichment analysis was performed by the DAVID online tool (<a class="link link-uri" href="http://david.abcc.ncifcrf.gov/" target="_blank">http://david.abcc.ncifcrf.gov/</a>).<a reveal-id="dsx055-B24" data-open="dsx055-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Enriched functional terms mainly included three ontology terms (BP_FAT, CC_FAT and MF_FAT) and the KEGG pathway. Significantly enriched terms were determined under FDR &lt; 0.05. The GO and pathway networks for splicing factor-regulated target genes were built using Enrichment Map<a reveal-id="dsx055-B25" data-open="dsx055-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> in Cytoscape 3.2.0<a reveal-id="dsx055-B26" data-open="dsx055-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> with the parameters of <em>P</em> &lt; 0.001 and FDR &lt; 0.1.</p>                            <h3 scrollto-destination=118057729 id="118057729" class="section-title" >2.7. Oligonucleotide synthesis</h3>
<p>The splice switching oligonucleotides (SSOs) were synthesised by Sangon Biotech (Shanghai) Co., Ltd. The sequences were listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>.</p>                            <h3 scrollto-destination=118057737 id="118057737" class="section-title" >2.8. Cell culture</h3>
<p>The AML12 (alpha mouse liver 12) cell line was purchased from ATCC (CRL-2254), which was established from HCs from a mouse (CD1 strain, line MT42) transgenic for human TGFα. These cells exhibit typical HC features, being a suitable alternative option for the mouse primary HCs. The cells were cultured in DMEM (GIBCO, USA) with 10% of fetal bovine serum (GIBCO, USA) and supplied with 1% of insulin-transferrin-selenium (GIBCO, USA) and 40 ng/ml dexamethasone (Sigma, USA).</p>                            <h3 scrollto-destination=118057742 id="118057742" class="section-title" >2.9. The knockdown of the splicing factors</h3>
<p>The siRNAs used for the knockdown of the indicated splicing factors were designed and synthesised by Sangon Biotech (Shanghai) Co., Ltd. After 72-hs transfection, the total RNA of AML12 cells was extracted and the AS of <em>Tak1</em> gene was analyzed. The siRNA sequences were listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>.</p>                            <h3 scrollto-destination=118057747 id="118057747" class="section-title" >2.10. Enrichment analysis of RNA-binding sites</h3>
<p>RNA-binding sites of splicing factors were mapped to pre-mRNA using the RBPmap online tool (<a class="link link-uri" href="http://rbpmap.technion.ac.il/" target="_blank">http://rbpmap.technion.ac.il/</a>)<a reveal-id="dsx055-B27" data-open="dsx055-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with the parameters of a high stringency level and conservation filter. The input sequences included alternative exon regions and their neighboring intron regions (300 bp). Enrichment of RNA-binding sites was performed using the hypergeometric test, and FDR &lt; 0.05 was used to assess significance.</p>                            <h3 scrollto-destination=118057753 id="118057753" class="section-title" >2.11. Data presentation</h3>
<p>Unsupervised hierarchical clustering was performed by Perseus software with default options (distance = ‘Euclidean’). Cumulative distribution curves were visualized using the ggplot2 package in R. Other statistical tests were also performed by R software.</p>                            <h2 scrollto-destination=118057757 id="118057757" class="section-title" >3. Data availability</h2>
<p>The RNA-Seq data from this study have been deposited into the NCBI Short Read Archive database under study accession number SRP033468.</p>                            <h2 scrollto-destination=118057762 id="118057762" class="section-title" >4. Results</h2>
                            <h3 scrollto-destination=118057764 id="118057764" class="section-title" >4.1. A global view of gene expression and splicing patterns</h3>
<p>In total, we identified 12,947 genes with FPKM ≥ 1 (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), each cell type had a series of cell-specific genes, while the majority (65.6%) were detected in all. The detected gene number of HCs, LSECs, KCs and HSCs was 9,967, 10,573, 10,964 and 11,767, respectively. The parenchymal cells, which participate in the liver’s main biological functions, had the least number of total genes but more high-abundance (FPKM ≥ 100) genes than NPCs (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). RT-qPCR of 20 randomly selected genes was performed in additional biological samples, the expression patterns of which were similar to the patterns obtained from the RNA-Seq, indicating the high correlation and good reproducibility between multiple samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>). </p>                        <a id="118057766" scrollto-destination="118057766"></a>
<div id="dsx055-F1" data-id="dsx055-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f1.png?Expires=2147483647&Signature=fETRmphsT03~EVfTdHRCzCKUXb9M6kdN9kHINa79wrtYBk3bncGd1UtpQ0tbS~GOlOs04RmowOCaTlofiouLwYFb8wlDevzVjTBaZIwIVQv2g5Vta~~ToIHgDJJv28Wyr3uemImf7C-s4I5DRV0KpKIuKXxz-jPIrCsRxWN-ZFFlD7J7rEVBtaxmLOoQVDfE4iWJoPyNtT7gaDGdxfEMjZxF34bpGlOzqjh-vwvL8oHuIIXgOo-FtnG32jUyWIQE-BCi5NHGuiy8sa28Cowr2BTm4DhbkU5bb2iMJ8WkmEK9Tu6hq2ro5YcwYJbbzNMiPcF29w1MPkXVaXNju2YhwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cell-type-resolved transcriptome expression and AS profiling in the mouse liver. (A) The overlap of identified genes and the number of detected genes at different FPKM levels. (B) The overlap of AS events and the number of five splicing types. (C) Left: Percentage of alternatively spliced genes in all identified genes. Right: Percentage of alternatively spliced genes with a different number of splicing variants. (D) The correlation analysis of PSI values calculated from RNA-Seq and RT-PCR." path-from-xml="dsx055f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057766/dsx055f1.tif" path-from-xml="dsx055f1.tif" target="_blank">View large</a><a section="118057766" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f1.png?Expires=2147483647&Signature=UifeCS24fUiJaiJaP14NHvgzrQ25lpIgbtasQVOzpDpM5DyLd25M-qNFR9k1M3N~8L-OAuB0b~6vgm8n~Pg68TibCY74vVHwMSulXp~XBh6U6PfUiuUzqykD3R8ibJuTjEIfnrDr3QCHEXsFvR~craOEpwN5xvn-VyrHe0LK-ZzrchDlkYQFa5nPS0qVaVg5xIL0DAm6xNI1KoIGCBE6Gb05UszZutdYyAYc97VJN5HSU-ISyKeVRQz9jSyjBGKRMhfmY6pzJNt5YLuWP1aGZrC6nF3cvN-MVKuaWOXpbWtjt72ZjAMrSW6pjudf7ZoUk9pnjDMDCiPSrb10gZvhHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057766&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cell-type-resolved transcriptome expression and AS profiling in the mouse liver. (A) The overlap of identified genes and the number of detected genes at different FPKM levels. (B) The overlap of AS events and the number of five splicing types. (C) Left: Percentage of alternatively spliced genes in all identified genes. Right: Percentage of alternatively spliced genes with a different number of splicing variants. (D) The correlation analysis of PSI values calculated from RNA-Seq and RT-PCR.</p></div></div><div content-id="dsx055-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f1.png?Expires=2147483647&Signature=fETRmphsT03~EVfTdHRCzCKUXb9M6kdN9kHINa79wrtYBk3bncGd1UtpQ0tbS~GOlOs04RmowOCaTlofiouLwYFb8wlDevzVjTBaZIwIVQv2g5Vta~~ToIHgDJJv28Wyr3uemImf7C-s4I5DRV0KpKIuKXxz-jPIrCsRxWN-ZFFlD7J7rEVBtaxmLOoQVDfE4iWJoPyNtT7gaDGdxfEMjZxF34bpGlOzqjh-vwvL8oHuIIXgOo-FtnG32jUyWIQE-BCi5NHGuiy8sa28Cowr2BTm4DhbkU5bb2iMJ8WkmEK9Tu6hq2ro5YcwYJbbzNMiPcF29w1MPkXVaXNju2YhwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cell-type-resolved transcriptome expression and AS profiling in the mouse liver. (A) The overlap of identified genes and the number of detected genes at different FPKM levels. (B) The overlap of AS events and the number of five splicing types. (C) Left: Percentage of alternatively spliced genes in all identified genes. Right: Percentage of alternatively spliced genes with a different number of splicing variants. (D) The correlation analysis of PSI values calculated from RNA-Seq and RT-PCR." path-from-xml="dsx055f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057766/dsx055f1.tif" path-from-xml="dsx055f1.tif" target="_blank">View large</a><a section="118057766" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f1.png?Expires=2147483647&Signature=UifeCS24fUiJaiJaP14NHvgzrQ25lpIgbtasQVOzpDpM5DyLd25M-qNFR9k1M3N~8L-OAuB0b~6vgm8n~Pg68TibCY74vVHwMSulXp~XBh6U6PfUiuUzqykD3R8ibJuTjEIfnrDr3QCHEXsFvR~craOEpwN5xvn-VyrHe0LK-ZzrchDlkYQFa5nPS0qVaVg5xIL0DAm6xNI1KoIGCBE6Gb05UszZutdYyAYc97VJN5HSU-ISyKeVRQz9jSyjBGKRMhfmY6pzJNt5YLuWP1aGZrC6nF3cvN-MVKuaWOXpbWtjt72ZjAMrSW6pjudf7ZoUk9pnjDMDCiPSrb10gZvhHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057766&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cell-type-resolved transcriptome expression and AS profiling in the mouse liver. (A) The overlap of identified genes and the number of detected genes at different FPKM levels. (B) The overlap of AS events and the number of five splicing types. (C) Left: Percentage of alternatively spliced genes in all identified genes. Right: Percentage of alternatively spliced genes with a different number of splicing variants. (D) The correlation analysis of PSI values calculated from RNA-Seq and RT-PCR.</p></div></div><p>The genome-wide extent of AS was analyzed by searching against known splicing junctions. We focused on five common types of AS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span>). The AS events identified in our data achieved an average coverage of 81.5% of all known events in the Ensembl database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>), consistent with the fact of that the liver was frequently regulated by AS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1D</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S2</a></span>). Most of the AS events existed in all four cell types (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>), and in our data, 46.2% of multi-exon genes underwent AS, half of which had three or more AS variants, indicating the distinguishing contribution of AS to the proteome diversity in mouse liver cells (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). Consistent with previous studies in mammals,<a reveal-id="dsx055-B3" data-open="dsx055-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> the skipped exon (SE) was the main AS type in mouse liver, followed by A3SS (Alternative 3′ splice site) and A5SS (Alternative 5′ splice site), and a low frequency was observed for retained intron and mutually exclusive exons (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>). All types tended to occur equally in different cell types, with a rank from 69.8% to 72.5% for SE (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). Although LSECs didn’t have the largest gene number, it had more AS events than the other three cell types, which indicated a precise regulation on the splicing level beyond the transcriptional regulation in LSECs.</p><p>To confirm the identification of AS from RNA-Seq data, randomly selected AS events were validated by RT-PCR (<a reveal-id="dsx055-F1" data-open="dsx055-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1E</a></span>), and the results displayed a high Pearson correlation (<em>R</em><sup>2</sup> = 0.808) of PSI values from RT-PCR and RNA-Seq.</p>                            <h3 scrollto-destination=118057769 id="118057769" class="section-title" >4.2. Association of gene functions with AS</h3>
<p>We classified the AS-regulated genes detected in all cell types into several function categories using MetaCore software. In general, the genes that most frequently regulated by AS were those encoding ‘regulators (GDI, GAP and GEF)’ (62.2%), ‘transcription factor’ (56.7%) and ‘generic kinase’ (56.2%). While for ‘receptor ligand’, ‘GPCR’ and ‘ligand-gated ion channel’, only 33.2, 29.7 and 24.1% of genes were regulated by AS (<a reveal-id="dsx055-F2" data-open="dsx055-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), indicating a putative mechanism that the distinct regulation of AS on genes of different function categories. Besides, AS did not happen equally across different cell types even for the genes in the same function categories. We found that the genes of ligand-gated ion channels did not undergo AS equally across cell types, the genes were more often regulated by AS in LSECs (45.8%) than in HCs (16.7%) (<a reveal-id="dsx055-F2" data-open="dsx055-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The statistical significance of spliced genes proportion for each function category between different cell types was shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>. In the liver, several diseases were caused by ion channel dysfunction.<a reveal-id="dsx055-B28" data-open="dsx055-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Different ion channels might have specialized biological significances in one particular cell type, yet no relevant investigation has been reported so far. </p>                        <a id="118057771" scrollto-destination="118057771"></a>
<div id="dsx055-F2" data-id="dsx055-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f2.png?Expires=2147483647&Signature=zoikTAqy5~ib5~UsLR3lj~IEdhaUJeZxkm~nhTMNyDDrWmE3cgEO21JMmXEpCQ6sT1ofrKTIu6DfO9kBwKNi2zIJEC6spyeQyHbKEJutYjhh9v93BUSDtT0~gwvjnnlMqI9gQhLuUVG35dlQzmMB2uNuf8VzcAtGeLww7wjORq60Jc~o810oh-JV78jAVlEHNVK4vV~ufvsnHcrl4And6E3VgldzZB3szA8yyU2zwxwepEo23K63MaDrDGuQXSdIe8WUoVz-WagL3P4R8yxn7XaHiC72ZWBGFiNYuZd1D58gPoPy1fzJQj32LfEvIIz8bDyUSVIDEEKcpOjniVhNhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional categories of spliced genes across cell types. (A) Altered percentages of alternatively spliced genes for different functional categories. ‘General’ column represents the combination of spliced genes in all four hepatic cell types. (B) Differentially enriched function terms revealed by enrichment analysis in four hepatic cell types. (C) Unsupervised hierarchical clustering for PSI values of AS events. Enrichment analysis shows the corresponding functional annotation for spliced gene clusters." path-from-xml="dsx055f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057771/dsx055f2.tif" path-from-xml="dsx055f2.tif" target="_blank">View large</a><a section="118057771" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f2.png?Expires=2147483647&Signature=CJRGLscJh6eJXwvKY16DWmReCdwQOI9ArF-GjqVKk1N9E7U-0~XbSsMSTbZuFBhM2mQVQZ3BwFUkIBaburo~AWBE2N9FFI3YAOHRcuW3mHQKWsuWiJiCJszf5qE9DaWcUVWHs6wvlky~UdSCE~DIBnXSd6Wi-933jh-~bucS4M-fTRNTkSyABqt5bkcLqpvyytGMmDdLpTOIkVmKATlwrCNLNYeF0-mb35Ny5SBCkqg1Ck8WtDzINTVbKF2PzT5sEN0MtJ6j3jmQcIQk0cjdvmY5DwB1NJPLszbPKevfnN5VYP4wpQUaKhXEza7Yw7M6Lqo94RsV51z~7BOQu0YAsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057771&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional categories of spliced genes across cell types. (A) Altered percentages of alternatively spliced genes for different functional categories. ‘General’ column represents the combination of spliced genes in all four hepatic cell types. (B) Differentially enriched function terms revealed by enrichment analysis in four hepatic cell types. (C) Unsupervised hierarchical clustering for PSI values of AS events. Enrichment analysis shows the corresponding functional annotation for spliced gene clusters.</p></div></div><div content-id="dsx055-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f2.png?Expires=2147483647&Signature=zoikTAqy5~ib5~UsLR3lj~IEdhaUJeZxkm~nhTMNyDDrWmE3cgEO21JMmXEpCQ6sT1ofrKTIu6DfO9kBwKNi2zIJEC6spyeQyHbKEJutYjhh9v93BUSDtT0~gwvjnnlMqI9gQhLuUVG35dlQzmMB2uNuf8VzcAtGeLww7wjORq60Jc~o810oh-JV78jAVlEHNVK4vV~ufvsnHcrl4And6E3VgldzZB3szA8yyU2zwxwepEo23K63MaDrDGuQXSdIe8WUoVz-WagL3P4R8yxn7XaHiC72ZWBGFiNYuZd1D58gPoPy1fzJQj32LfEvIIz8bDyUSVIDEEKcpOjniVhNhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional categories of spliced genes across cell types. (A) Altered percentages of alternatively spliced genes for different functional categories. ‘General’ column represents the combination of spliced genes in all four hepatic cell types. (B) Differentially enriched function terms revealed by enrichment analysis in four hepatic cell types. (C) Unsupervised hierarchical clustering for PSI values of AS events. Enrichment analysis shows the corresponding functional annotation for spliced gene clusters." path-from-xml="dsx055f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057771/dsx055f2.tif" path-from-xml="dsx055f2.tif" target="_blank">View large</a><a section="118057771" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f2.png?Expires=2147483647&Signature=CJRGLscJh6eJXwvKY16DWmReCdwQOI9ArF-GjqVKk1N9E7U-0~XbSsMSTbZuFBhM2mQVQZ3BwFUkIBaburo~AWBE2N9FFI3YAOHRcuW3mHQKWsuWiJiCJszf5qE9DaWcUVWHs6wvlky~UdSCE~DIBnXSd6Wi-933jh-~bucS4M-fTRNTkSyABqt5bkcLqpvyytGMmDdLpTOIkVmKATlwrCNLNYeF0-mb35Ny5SBCkqg1Ck8WtDzINTVbKF2PzT5sEN0MtJ6j3jmQcIQk0cjdvmY5DwB1NJPLszbPKevfnN5VYP4wpQUaKhXEza7Yw7M6Lqo94RsV51z~7BOQu0YAsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057771&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional categories of spliced genes across cell types. (A) Altered percentages of alternatively spliced genes for different functional categories. ‘General’ column represents the combination of spliced genes in all four hepatic cell types. (B) Differentially enriched function terms revealed by enrichment analysis in four hepatic cell types. (C) Unsupervised hierarchical clustering for PSI values of AS events. Enrichment analysis shows the corresponding functional annotation for spliced gene clusters.</p></div></div><p>We next analyzed the enriched functional terms of AS-regulated genes in distinct cell contexts. The AS-regulated genes in HCs were highly enriched in multiple metabolic processes, such as the organic substance metabolic process and the primary metabolic process. Similarly, in NPCs, the AS-regulated genes also tended to play roles in cell-type-specific functions, just as cytoskeleton organization and immune system process (<a reveal-id="dsx055-F2" data-open="dsx055-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>). To our surprise, plenty of AS-regulated genes functioning in regulating biological processes, such as ‘positive regulation of metabolic process’, were significantly enriched in NPCs instead of in HCs (<a reveal-id="dsx055-F2" data-open="dsx055-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). To eliminate the potential effects of low abundance genes on enrichment analysis, we separately chose the 1000 and 2000 most abundant AS genes in each cell type to reanalyze the enriched biological processes and obtained the consistent trend (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2C</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">D</a></span>). Besides, all the four cell types showed an almost uniform cumulative distribution and the bulk of transcription was dominated by a few high abundance genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>), demonstrating they would be identically affected by detection sensitivity for low abundance genes. These findings confirmed the important roles involving regulation of metabolic process in NPCs and provided new evidence of the cooperation between HCs and NPCs on the isoform level.</p><p>Furthermore, we explored the association of AS with gene functions using the quantitative data of splicing in different cell types. We firstly defined the CEEs as described in the Materials and Methods section. The unsupervised hierarchical clustering revealed that these exons exhibited obvious splicing switches between different cell types (<a reveal-id="dsx055-F2" data-open="dsx055-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Functionally, the genes with HC-enriched exons were linked to ‘regulation of kinase cascade’ and ‘phosphoprotein’, largely reflecting the identities of the HCs and differing them from NPC-enriched exons. In NPCs, each cluster also revealed significant enrichment in specific GO terms, including ‘cell surface receptor linked signal transduction’ (LSEC-enriched), ‘regulation of immune effector process’ (KC-enriched) and ‘membrane organization’ (HSC-enriched). These CEE-contained genes reflected the core functions of respective cell types, as the visualized global diversity of all genes with FPKM ≥ 1 and the functional enrichment analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B and C</a></span>).</p>                            <h3 scrollto-destination=118057774 id="118057774" class="section-title" >4.3. Functional properties of CEEs</h3>
<p>Tissue-enriched exons had exhibited multiple properties that distinguished them from constitutive exons.<a reveal-id="dsx055-B6" data-open="dsx055-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx055-B7" data-open="dsx055-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Thus, we wondered if the CEEs in mouse liver also had similar properties and how AS affecting protein functions. In our data, although some of the CEEs destroyed the functional protein domain (e.g. the immunoglobulin domain 1 of <em>Fgfr1</em>) when excluded from the pre-mRNA, the CEEs had a significantly smaller fraction of overlapping to protein domains compared with other alternative exons and constitutive exons (<a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3A;</a><em>P</em>&lt;2.07E-15). In contrast, an increased fraction overlapping to predicted disordered regions was observed (<a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3B;</a><em>P</em>&lt;4.56E-11). The disordered regions had important regulatory effects on protein–protein interaction (PPI) networks,<a reveal-id="dsx055-B6" data-open="dsx055-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx055-B7" data-open="dsx055-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and consistently, the CEEs were significantly enriched in regions that involved in PPI regulation (<a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3C;</a><em>P</em>&lt;7.16E-04). In addition, these CEEs contained more post-translational modification (PTM) sites than others (<a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, adjusted to the length of the exons; <em>P</em>&lt;0.022). Together, in mouse liver, our results suggested that the CEEs might regulate isoform functions through affecting PPIs and modifying PTM process, consistent with the previous reports on the tissue-specific exons.<a reveal-id="dsx055-B6" data-open="dsx055-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx055-B7" data-open="dsx055-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p>                        <a id="118057776" scrollto-destination="118057776"></a>
<div id="dsx055-F3" data-id="dsx055-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f3.png?Expires=2147483647&Signature=JKzLwOrMWQa-EtK3VBlRpPCIbVkauZHtuZYWir6iFxgvP7BTKpgfIm4ZBWjOX2MJ7ewn0I~ggYAoHuNgfTKXjZUrUsG6CMqJnsHb4vCmJv7ayswHMh4WK2gnVDq5egtiCLHT7L7y6nJGYjgkUTjprKmNGCB3ZU7MBgp~pOpmQfeyRmmP04PtW7JXP25mS5RizHW6P4H-6iJJwbNSKZFgY7TGbE3OAsgS4H2qLQM60IlKMxQ1Wt5I1vEnxyoXVYrHC3PUrYZVlsGVU8kS-Zak4B9kdvwq9psO28NGt7Du6T7GhGlWkrDk~WYlI9BZ7DVBWUEkQIOy6HoUwXsWuoQCtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of CEEs. (A–D) Percentages of three groups of exons encoding (A) protein domains, (B) disordered regions, (C) binding regions and (D) PTM sites. Significance was calculated using the χ2 test. (E) Different PSI values of alternative exons between HCs and NPCs. Exons in circles are highlighted as examples. (F) Variants-switching validation between HCs and NPCs for Tak1, Sidt2, Lrch3 and Mbnl1." path-from-xml="dsx055f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057776/dsx055f3.tif" path-from-xml="dsx055f3.tif" target="_blank">View large</a><a section="118057776" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f3.png?Expires=2147483647&Signature=XInAc9e1Gkxqut7rkCRo6EM6opUkt5cVP0weECjOUo4PBXhav5clfYrKN89AMJCs7XibiHUB56wq~fU-r2cPMBl6SGFhFUFIPb8XtfonpQ~6-tD-HC4-1d5FqshZouAivthzmzb1xnqktJ3mjpBwm6McX~NqFdY-dyHQKWxV43R6Ncy9XoJeRf~Z~fdgqCUNQlJPUqRoigsP5vHTjn1-drOs17XE90x0v4z2KNG8Y6eeqTXoDUYlw2VEYdSzRDkm32399~xo-za~B4UobNREuCMiLr3vJ1s77GbfmyWjMfXwND19dCS595jpjmaowkKzSs6c~fYuIkp~7E5nqV8gsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057776&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characterization of CEEs. (A–D) Percentages of three groups of exons encoding (A) protein domains, (B) disordered regions, (C) binding regions and (D) PTM sites. Significance was calculated using the χ<sup>2</sup> test. (E) Different PSI values of alternative exons between HCs and NPCs. Exons in circles are highlighted as examples. (F) Variants-switching validation between HCs and NPCs for <em>Tak1</em>, <em>Sidt2</em>, <em>Lrch3</em> and <em>Mbnl1</em>.</p></div></div><div content-id="dsx055-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f3.png?Expires=2147483647&Signature=JKzLwOrMWQa-EtK3VBlRpPCIbVkauZHtuZYWir6iFxgvP7BTKpgfIm4ZBWjOX2MJ7ewn0I~ggYAoHuNgfTKXjZUrUsG6CMqJnsHb4vCmJv7ayswHMh4WK2gnVDq5egtiCLHT7L7y6nJGYjgkUTjprKmNGCB3ZU7MBgp~pOpmQfeyRmmP04PtW7JXP25mS5RizHW6P4H-6iJJwbNSKZFgY7TGbE3OAsgS4H2qLQM60IlKMxQ1Wt5I1vEnxyoXVYrHC3PUrYZVlsGVU8kS-Zak4B9kdvwq9psO28NGt7Du6T7GhGlWkrDk~WYlI9BZ7DVBWUEkQIOy6HoUwXsWuoQCtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of CEEs. (A–D) Percentages of three groups of exons encoding (A) protein domains, (B) disordered regions, (C) binding regions and (D) PTM sites. Significance was calculated using the χ2 test. (E) Different PSI values of alternative exons between HCs and NPCs. Exons in circles are highlighted as examples. (F) Variants-switching validation between HCs and NPCs for Tak1, Sidt2, Lrch3 and Mbnl1." path-from-xml="dsx055f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057776/dsx055f3.tif" path-from-xml="dsx055f3.tif" target="_blank">View large</a><a section="118057776" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f3.png?Expires=2147483647&Signature=XInAc9e1Gkxqut7rkCRo6EM6opUkt5cVP0weECjOUo4PBXhav5clfYrKN89AMJCs7XibiHUB56wq~fU-r2cPMBl6SGFhFUFIPb8XtfonpQ~6-tD-HC4-1d5FqshZouAivthzmzb1xnqktJ3mjpBwm6McX~NqFdY-dyHQKWxV43R6Ncy9XoJeRf~Z~fdgqCUNQlJPUqRoigsP5vHTjn1-drOs17XE90x0v4z2KNG8Y6eeqTXoDUYlw2VEYdSzRDkm32399~xo-za~B4UobNREuCMiLr3vJ1s77GbfmyWjMfXwND19dCS595jpjmaowkKzSs6c~fYuIkp~7E5nqV8gsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057776&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characterization of CEEs. (A–D) Percentages of three groups of exons encoding (A) protein domains, (B) disordered regions, (C) binding regions and (D) PTM sites. Significance was calculated using the χ<sup>2</sup> test. (E) Different PSI values of alternative exons between HCs and NPCs. Exons in circles are highlighted as examples. (F) Variants-switching validation between HCs and NPCs for <em>Tak1</em>, <em>Sidt2</em>, <em>Lrch3</em> and <em>Mbnl1</em>.</p></div></div><p>A scatter diagram illustrated all the alternative exons that exhibited markedly different PSIs between HCs and NPCs (<a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3E</a>). The circles shown in <a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3E</a> highlighted the flexibility of the splicing regulatory machinery, with a large number of exons predominantly included in one cell type but excluded in others (just like exon 12 of <em>Tak1</em>, with a PSI of 0.886 in HCs, but 0.051, 0.059 and 0.117 in LSECs, KCs and HSCs, respectively; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>). We used RT-PCR assays to validate individual AS events detected by RNA-Seq (<a reveal-id="dsx055-F3" data-open="dsx055-F3" class="link link-reveal link-table xref-fig">Fig. 3F</a>). The validated examples included (but not limited to) the genes that functioning in regulating the MAPK pathway (<em>Tak1</em>), splicing regulation (<em>Mbnl1</em>), glucose homeostasis (<em>Sidt2</em>) and a gene with putative immune functions (<em>Lrch3</em>). Most of the exons correlated well with our RNA-Seq data, exhibiting a strong cell-type-regulated AS pattern.</p>                            <h3 scrollto-destination=118057778 id="118057778" class="section-title" >4.4. Regulation of as by splicing factors</h3>
<p>We were interested in how these CEEs generated and which particular splicing factor (SF) might dominate the splicing patterns. First, we collected 94 splicing factors from RBPmap,<a reveal-id="dsx055-B27" data-open="dsx055-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> 78 of which (with 114 experimentally confirmed binding motifs) were expressed in our data with FPKM ≥ 1 and displayed variable cell-type specificities (<a reveal-id="dsx055-F4" data-open="dsx055-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). The most significant diversity was observed between HCs and NPCs, and each cell type had a set of highly expressed splicing factors, as listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S4</a></span>. </p>                        <a id="118057780" scrollto-destination="118057780"></a>
<div id="dsx055-F4" data-id="dsx055-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f4.png?Expires=2147483647&Signature=necGzjtjhy~2t8x4ZHjy2qDilVUiE9RHO5iMdSBDQ74xTRY09dffUhJH9CFsEwYj3PJX0vZJAkUY7zPy3YgzMws8lnDX7B9TpXbO9KGIlur3kwADjkkh3m605UDv5q5jUkuqx4Wulhs4F6-BRAeFazURdSipyUxK2bxYmL725W-m3daJGIJ0mqFd~pQAhR4tcoHlBCoy7ihzjrS-U8s7iG1tXCq~A1D8c9ZBjRAheLoZZzJSv-Ld9A6ON~nkA8lEXj~ZASIsMGcG1livhKGNwbcqfyo57GL6JfsGe5K2CLVVOK9uj2mVx5F4gR1JF8797~STkEV65H94wrhT1n-Wtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of cell-type-specific splicing factors. (A) Differential expression patterns of splicing factors revealed by unsupervised hierarchical clustering. (B) Differential expression abundances of four cell-type-specific splicing factors in both mRNA and protein level." path-from-xml="dsx055f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057780/dsx055f4.tif" path-from-xml="dsx055f4.tif" target="_blank">View large</a><a section="118057780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f4.png?Expires=2147483647&Signature=cuQYtMjo-xxPioT8JjGfIVox8pGbGslzKyPFv3PJ2uUyZgH39H7bm-Nt~skDNUVywSV9kdp2f1u6FlOWEPt0QePGRG3LwbxQYfmR1ohtRIv1wZylB1OSpyvr1rKEeKHnC3dMaJn5isnKXz6ivlGwHJq2OnTcpc4LTFhtuJ-EHI5zct06hLA4YZoc9UYJmy55Si0PN3X2TEpS0-NMuKN3iSKuOwFVjNWvmtNXRmeuBLfHsOOYtSXNqFJ1x53W0KMwNE-ZiDpRRjYOtRDYks7zt1ITr2O0tbqWB4N6PVJY8nlu~EkmxhA8-H1onSGQMAo3XfQ0Khrbk8Y~CNLG8TvlIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057780&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of cell-type-specific splicing factors. (A) Differential expression patterns of splicing factors revealed by unsupervised hierarchical clustering. (B) Differential expression abundances of four cell-type-specific splicing factors in both mRNA and protein level.</p></div></div><div content-id="dsx055-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f4.png?Expires=2147483647&Signature=necGzjtjhy~2t8x4ZHjy2qDilVUiE9RHO5iMdSBDQ74xTRY09dffUhJH9CFsEwYj3PJX0vZJAkUY7zPy3YgzMws8lnDX7B9TpXbO9KGIlur3kwADjkkh3m605UDv5q5jUkuqx4Wulhs4F6-BRAeFazURdSipyUxK2bxYmL725W-m3daJGIJ0mqFd~pQAhR4tcoHlBCoy7ihzjrS-U8s7iG1tXCq~A1D8c9ZBjRAheLoZZzJSv-Ld9A6ON~nkA8lEXj~ZASIsMGcG1livhKGNwbcqfyo57GL6JfsGe5K2CLVVOK9uj2mVx5F4gR1JF8797~STkEV65H94wrhT1n-Wtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of cell-type-specific splicing factors. (A) Differential expression patterns of splicing factors revealed by unsupervised hierarchical clustering. (B) Differential expression abundances of four cell-type-specific splicing factors in both mRNA and protein level." path-from-xml="dsx055f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057780/dsx055f4.tif" path-from-xml="dsx055f4.tif" target="_blank">View large</a><a section="118057780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f4.png?Expires=2147483647&Signature=cuQYtMjo-xxPioT8JjGfIVox8pGbGslzKyPFv3PJ2uUyZgH39H7bm-Nt~skDNUVywSV9kdp2f1u6FlOWEPt0QePGRG3LwbxQYfmR1ohtRIv1wZylB1OSpyvr1rKEeKHnC3dMaJn5isnKXz6ivlGwHJq2OnTcpc4LTFhtuJ-EHI5zct06hLA4YZoc9UYJmy55Si0PN3X2TEpS0-NMuKN3iSKuOwFVjNWvmtNXRmeuBLfHsOOYtSXNqFJ1x53W0KMwNE-ZiDpRRjYOtRDYks7zt1ITr2O0tbqWB4N6PVJY8nlu~EkmxhA8-H1onSGQMAo3XfQ0Khrbk8Y~CNLG8TvlIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057780&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of cell-type-specific splicing factors. (A) Differential expression patterns of splicing factors revealed by unsupervised hierarchical clustering. (B) Differential expression abundances of four cell-type-specific splicing factors in both mRNA and protein level.</p></div></div><p>In both mRNA and protein levels, we identified the high-expressed splicing factors in each cell type, such as Esrp2 in HCs, Rbfox2 in LSECs, Srsf9 in KCs and Rbms3 in HSCs (<a reveal-id="dsx055-F4" data-open="dsx055-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, proteomics data came from the literature 17). We used RBPmap to search the target genes of these splicing factors, followed by the functional enrichment analysis. These splicing factors controlled a large proportion of the cell-type-specific AS events, and the target genes were highly enriched in cell specialized functions. For example, <em>Esrp2</em> was highly expressed in HCs but not in three other NPCs, and the targets of <em>Esrp2</em> were significantly enriched in terms of liver specialized functions, such as lipoprotein and coenzyme metabolism (<a reveal-id="dsx055-F5" data-open="dsx055-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). The target genes of the LSEC-specific splicing factor <em>Rbfox2</em> were enriched in cytoskeleton organization and cell adhesion (<a reveal-id="dsx055-F5" data-open="dsx055-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). Rbfox2 was reported to splice the <em>Tak1</em> pre-mRNA to the short variant <em>Tak1-A</em> instead of the full-length <em>Tak1-B</em>, consistent with the distribution of <em>Tak1</em> variants in HCs and LSECs.<a reveal-id="dsx055-B29" data-open="dsx055-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Similarly, in Kupffer Cells, we found that the target genes of <em>Srsf9</em> were highly enriched in multiple immunity-related biological processes and pathways (<a reveal-id="dsx055-F5" data-open="dsx055-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span>). Furthermore, although little was reported about the functions of <em>Rbms3</em> in the liver, our data showed that <em>Rbms3</em> were exclusively high-expressed in HSCs, regulating a range of biological processes, especially those HSC-specific functions like cell-matrix adhesion (<a reveal-id="dsx055-F5" data-open="dsx055-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). </p>                        <a id="118057782" scrollto-destination="118057782"></a>
<div id="dsx055-F5" data-id="dsx055-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f5.png?Expires=2147483647&Signature=EBgqmEVLFl5d2tDpZrPzmrtDLpqJU5xHQPbKqS1hNJac9Cu4wvlB1njpGEz5W0fb49l0vtiil7H51wn4eHvCKCCa43cst85fFgnUNJsHH3-PmV4BcKmM~MqdmuLHZjcJy6CQU20JtfsUWk5HBx1NKSEr-4AduKQ7vMVceGRaV-qX6H~1vXrrDAzpynbsJaejULnZ4l-w8Yhi5a43nu7mrXEBNwcSQg~5KhyKZne-I5vrrcmQoiCNZw2f96zVQifbgWH~BJvOcQ3dImmscZ0L4aLo1lRKncpfszuA3PMQrQwW2OjZLswgOa3~dtULvkEeAboJevlVnyaeGryMFQnvAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional regulation by splicing factors. Enriched biological processes of spliced genes regulated by (A) Esrp2 in HCs, (B) Rbfox2 in LSECs, (C) Srsf9 in KCs and (D) Rbms3 in HSCs." path-from-xml="dsx055f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057782/dsx055f5.tif" path-from-xml="dsx055f5.tif" target="_blank">View large</a><a section="118057782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f5.png?Expires=2147483647&Signature=sNoAE0Gh1TN~jxpMEj4dd~gh~fsbAgYoInVPfCTO6ozbH5bgcbMO8WOSn~v1WfOL8IxpHnrkW8sbDV6Pmy5PAn7P4T9m9P4tNvQFWqCUQVmMkl5pTtBmYHoex56DpdNpVrKx46NJrsfGd1CLfY5Y27P8U9eBImRFl7uxGjsLkyj2TlAgMYD14fByW9egKDeKearpK6usCyfas1jX6hK-sf0GixQO9xcfKy92qfROcckzuROX61NRw2dY2sZDQiSjUanf-dBhcF52O6fsHKvoTIm86OmKI6LxQyAv959Qxj2xhoYYp0ZXQFU6ZxHhNuG59bCy~1qvzLFOm9esmTJHAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057782&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional regulation by splicing factors. Enriched biological processes of spliced genes regulated by (A) <em>Esrp2</em> in HCs, (B) <em>Rbfox2</em> in LSECs, (C) <em>Srsf9</em> in KCs and (D) <em>Rbms3</em> in HSCs.</p></div></div><div content-id="dsx055-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f5.png?Expires=2147483647&Signature=EBgqmEVLFl5d2tDpZrPzmrtDLpqJU5xHQPbKqS1hNJac9Cu4wvlB1njpGEz5W0fb49l0vtiil7H51wn4eHvCKCCa43cst85fFgnUNJsHH3-PmV4BcKmM~MqdmuLHZjcJy6CQU20JtfsUWk5HBx1NKSEr-4AduKQ7vMVceGRaV-qX6H~1vXrrDAzpynbsJaejULnZ4l-w8Yhi5a43nu7mrXEBNwcSQg~5KhyKZne-I5vrrcmQoiCNZw2f96zVQifbgWH~BJvOcQ3dImmscZ0L4aLo1lRKncpfszuA3PMQrQwW2OjZLswgOa3~dtULvkEeAboJevlVnyaeGryMFQnvAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional regulation by splicing factors. Enriched biological processes of spliced genes regulated by (A) Esrp2 in HCs, (B) Rbfox2 in LSECs, (C) Srsf9 in KCs and (D) Rbms3 in HSCs." path-from-xml="dsx055f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057782/dsx055f5.tif" path-from-xml="dsx055f5.tif" target="_blank">View large</a><a section="118057782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f5.png?Expires=2147483647&Signature=sNoAE0Gh1TN~jxpMEj4dd~gh~fsbAgYoInVPfCTO6ozbH5bgcbMO8WOSn~v1WfOL8IxpHnrkW8sbDV6Pmy5PAn7P4T9m9P4tNvQFWqCUQVmMkl5pTtBmYHoex56DpdNpVrKx46NJrsfGd1CLfY5Y27P8U9eBImRFl7uxGjsLkyj2TlAgMYD14fByW9egKDeKearpK6usCyfas1jX6hK-sf0GixQO9xcfKy92qfROcckzuROX61NRw2dY2sZDQiSjUanf-dBhcF52O6fsHKvoTIm86OmKI6LxQyAv959Qxj2xhoYYp0ZXQFU6ZxHhNuG59bCy~1qvzLFOm9esmTJHAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057782&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional regulation by splicing factors. Enriched biological processes of spliced genes regulated by (A) <em>Esrp2</em> in HCs, (B) <em>Rbfox2</em> in LSECs, (C) <em>Srsf9</em> in KCs and (D) <em>Rbms3</em> in HSCs.</p></div></div><p>Besides analyzing the differential expressed splicing factors, we combined the <em>trans</em>-acting splicing factors and <em>cis</em>-element (binding motifs) surrounding CEEs and built a splicing regulatory map, in order to further identify the important splicing factors in each cell type (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4B</a></span>). Specifically, LSECs exhibited the most significant enrichment of RNA-binding motifs of splicing factors across all cell types, consistent with the largest AS event number in LSECs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4B</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">1B</a></span>), suggesting the functional diversity of LSECs and potential interplay with other cell types in various aspects.<a reveal-id="dsx055-B30 dsx055-B31 dsx055-B32" data-open="dsx055-B30 dsx055-B31 dsx055-B32" class="link link-ref link-reveal xref-bibr"><sup>30–32</sup></a></p>                            <h3 scrollto-destination=118057784 id="118057784" class="section-title" >4.5. Specific Tak1 isoform regulates hepatic cell-type-specific functions</h3>
<p>Growing evidence suggested that protein isoforms generated from AS can perform distinct or even opposite functions.<a reveal-id="dsx055-B33" data-open="dsx055-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx055-B34" data-open="dsx055-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> We attempted to reveal the functions of cell-specific isoforms of Tak1 in distinct cell types. <em>Tak1</em> has four main variants generated by AS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>), depending on the spliced-in or -out of exon 12 and exon 16.<a reveal-id="dsx055-B29" data-open="dsx055-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> We provided a detailed splicing map of exon 12 of <em>Tak1</em> (<em>TGF-activated kinase 1</em>) gene. Rbfox2, which highly expressed in LSECs, was reported to splice the <em>Tak1</em> pre-mRNA to the short variant <em>Tak1-A</em>, consistent with the distribution of <em>Tak1</em> variants in HCs and LSECs. Besides, we used RBPmap to predict the splicing factors that considered to be involved in the formation of <em>Tak1’s</em> splicing pattern, and several candidates with distinct expression patterns were selected, as shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span>. We considered that splicing factors with different expression patterns were more likely to contribute in the formation of the cell-type-specific splicing patterns, while in the exonic regions, the predicted splicing factors did not exhibit significant expression differences between HCs and NPCs, thus, the intronic area might be more important in the AS regulation of <em>Tak1</em>. Several splicing factors were proved to be involved in this regulation, but single splicing factor did not play a decisive role (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5C</a></span>), which indicated that the specific splicing pattern of <em>Tak1</em> resulted from the possible cooperation and competition of multiple splicing regulators in mouse liver cells.<a reveal-id="dsx055-B35" data-open="dsx055-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p>We collected 561 public RNA-Seq datasets from the Sequence Read Archive database with RPBmap tools to obtain a comprehensive expression profile of <em>Tak1</em> variants.<a reveal-id="dsx055-B36" data-open="dsx055-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><em>Tak1-B</em> was highly expressed in tissues with strong proliferation and metabolism properties, whereas <em>Tak1-A</em> was usually existed in tissues with immune functions, such as the spleen and several immune cells (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5D</a></span>). Coordinately, a multiple instance learning based analysis<a reveal-id="dsx055-B36" data-open="dsx055-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> revealed that <em>Tak1-B</em> was more concentrated in some basic metabolic processes, while <em>Tak1-A</em> was primarily responsible for immune functions (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>), consistent with the distribution of <em>Tak1</em> variants in HCs and NPCs. </p>                        <a id="118057787" scrollto-destination="118057787"></a>
<div id="dsx055-F6" data-id="dsx055-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f6.png?Expires=2147483647&Signature=45zdKawhe396i2N5AEZ3~orqX8lP9F5XKRkocZhc4BJjl45wVD-RiCRYG1unRzqGjMw1qzO1r89dYBpxb86oWU99tV-DOTrJq1l~3yp6xlavUrWwef0r10-7on-OLX0WJskJwsi-HbqrqrMEfZLok905ErH4Jvfv9VADO5U28NWfKNj~grGDTVZZLrRQ~FBuQBmV7d-jOd3~OtXoP2BoNHr34s0PTB409uZBWLHoJoQ43~cPYRjWbZ~NAclqDlzRdBl0FUVNoQS7b51VqhEOMXtsHKnvQWFIAIan3OqyM7izaxzRaoFb7UC5ixnkmzUB~vROsZzd84n9qUKMclWa6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The functional differences between Tak1 isoforms. (A) Two variants of Tak1 were responsible for diverse biological processes. (B) The differences in protein structure between Tak1-A and Tak1-B isoforms. (C) SSOs converted Tak1-B to Tak1-A in AML12 cells. (D) Tak1-A bound to Tab2 (not Tab1) more easily than Tak1-B did. The ratio of Tak1/Tab1 or Tak1/Tab2 was calculated from image intensity. (E) The MAPK and NF-κB pathways were activated when Tak1-B converted to Tak1-A. (F) Several lipids and cholesterol synthesis related genes were up-regulated after the conversion. Significance was calculated using the t-test. *P&lt;0.05." path-from-xml="dsx055f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057787/dsx055f6.tif" path-from-xml="dsx055f6.tif" target="_blank">View large</a><a section="118057787" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f6.png?Expires=2147483647&Signature=pXTk7OAyuYBy~sVqd2Ur~o1-rJ4dI7-juoEGKdtLNmuOxKk6Yz1e433-PH82cUEF4LJ2Twy454MCzbtPpYlFcLky2nDlsiiRZ2b6KxNHUw0aAlKnaA10q7wG46vvS-FigJnBgnhKAmMyx87Yj1j2l8tGmHFC7Xq20HuOcUtbEq3LFwdwbTFUZvzB5f6hjQMLF3Ha9xEKpF~ecvc18ss7NnECEv0LwaYsKi6H4R~RlObR8Za00zXazpyJNTvybAjv3Y5jHxUgJdKQVo1XRcJlS2W15mvAQIILxulMImMY5HsMg53WViPEN9kvR47NEBlPcD9pqmF8FjP2GD-BjEFyvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057787&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The functional differences between Tak1 isoforms. (A) Two variants of <em>Tak1</em> were responsible for diverse biological processes. (B) The differences in protein structure between Tak1-A and Tak1-B isoforms. (C) SSOs converted <em>Tak1-B</em> to <em>Tak1-A</em> in AML12 cells. (D) Tak1-A bound to Tab2 (not Tab1) more easily than Tak1-B did. The ratio of Tak1/Tab1 or Tak1/Tab2 was calculated from image intensity. (E) The MAPK and NF-κB pathways were activated when Tak1-B converted to Tak1-A. (F) Several lipids and cholesterol synthesis related genes were up-regulated after the conversion. Significance was calculated using the <em>t</em>-test. *<em>P</em>&lt;0.05.</p></div></div><div content-id="dsx055-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/m_dsx055f6.png?Expires=2147483647&Signature=45zdKawhe396i2N5AEZ3~orqX8lP9F5XKRkocZhc4BJjl45wVD-RiCRYG1unRzqGjMw1qzO1r89dYBpxb86oWU99tV-DOTrJq1l~3yp6xlavUrWwef0r10-7on-OLX0WJskJwsi-HbqrqrMEfZLok905ErH4Jvfv9VADO5U28NWfKNj~grGDTVZZLrRQ~FBuQBmV7d-jOd3~OtXoP2BoNHr34s0PTB409uZBWLHoJoQ43~cPYRjWbZ~NAclqDlzRdBl0FUVNoQS7b51VqhEOMXtsHKnvQWFIAIan3OqyM7izaxzRaoFb7UC5ixnkmzUB~vROsZzd84n9qUKMclWa6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The functional differences between Tak1 isoforms. (A) Two variants of Tak1 were responsible for diverse biological processes. (B) The differences in protein structure between Tak1-A and Tak1-B isoforms. (C) SSOs converted Tak1-B to Tak1-A in AML12 cells. (D) Tak1-A bound to Tab2 (not Tab1) more easily than Tak1-B did. The ratio of Tak1/Tab1 or Tak1/Tab2 was calculated from image intensity. (E) The MAPK and NF-κB pathways were activated when Tak1-B converted to Tak1-A. (F) Several lipids and cholesterol synthesis related genes were up-regulated after the conversion. Significance was calculated using the t-test. *P&lt;0.05." path-from-xml="dsx055f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057787/dsx055f6.tif" path-from-xml="dsx055f6.tif" target="_blank">View large</a><a section="118057787" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055f6.png?Expires=2147483647&Signature=pXTk7OAyuYBy~sVqd2Ur~o1-rJ4dI7-juoEGKdtLNmuOxKk6Yz1e433-PH82cUEF4LJ2Twy454MCzbtPpYlFcLky2nDlsiiRZ2b6KxNHUw0aAlKnaA10q7wG46vvS-FigJnBgnhKAmMyx87Yj1j2l8tGmHFC7Xq20HuOcUtbEq3LFwdwbTFUZvzB5f6hjQMLF3Ha9xEKpF~ecvc18ss7NnECEv0LwaYsKi6H4R~RlObR8Za00zXazpyJNTvybAjv3Y5jHxUgJdKQVo1XRcJlS2W15mvAQIILxulMImMY5HsMg53WViPEN9kvR47NEBlPcD9pqmF8FjP2GD-BjEFyvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057787&ar=4793385&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx055f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The functional differences between Tak1 isoforms. (A) Two variants of <em>Tak1</em> were responsible for diverse biological processes. (B) The differences in protein structure between Tak1-A and Tak1-B isoforms. (C) SSOs converted <em>Tak1-B</em> to <em>Tak1-A</em> in AML12 cells. (D) Tak1-A bound to Tab2 (not Tab1) more easily than Tak1-B did. The ratio of Tak1/Tab1 or Tak1/Tab2 was calculated from image intensity. (E) The MAPK and NF-κB pathways were activated when Tak1-B converted to Tak1-A. (F) Several lipids and cholesterol synthesis related genes were up-regulated after the conversion. Significance was calculated using the <em>t</em>-test. *<em>P</em>&lt;0.05.</p></div></div><p>We downloaded the PDB files of the two isoforms to predict the protein structure differences between Tak1-A and Tak1-B by searching Protein Data Bank (PDB) archive. Although the serine-threonine/tyrosine protein kinase catalytic domain of the two isoforms were unaffected by AS (from Pfam database), they had differences in spatial structures around the peptides that translated from exon 11 to exon 13 (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). The peptide segment altered by the spliced exon 12 fell in the protein-binding region (marked in <a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>), suggesting that gain or loss of Tak1 isoform function might be caused by the affected interaction protein partner.</p><p>We then used SSOs to explore the exact functional differences between Tak1 isoforms. We converted <em>Tak1-B</em> to <em>Tak1-A</em> in AML12 cells, in which <em>Tak1-B</em> was the main variant, just like in the HCs (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>). As we analyzed previously, the CEEs could change the PPIs by affecting the protein binding domains, here, Co-IP showed that Tak1-A bound to Tab2 (but not to Tab1) more easily than Tak1-B did (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). It was reported that the binding to Tab1 and Tab2 was necessary for the activation of Tak1 and the downstream pathways. Our results showed that as the conversion of Tak1-B to Tak1-A, the downstream JNK, p38 and NF-κB pathways were significantly activated (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6E</a>). Activated JNK and p38 pathways had important roles in cellular inflammatory response,<a reveal-id="dsx055-B37" data-open="dsx055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> which consistent with the distribution and the functionality of Tak1-A in NPCs.</p><p>We also found that the conversion of Tak1-B to Tak1-A triggered a significant up-regulation of genes related with lipids and cholesterol synthesis in AML12 cells, whereas lipolysis-related genes (<em>Lipc</em>) were down-regulated (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6F</a>). These results were also consistent with the potential metabolic regulation function of Tak1-B in HCs (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>). In addition, <em>Lkb1</em>, the gene associated with maintenance of cell polarity in HCs, was significantly down-regulated (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6F</a>), suggesting that Tak1-B also played important roles in maintaining cell polarity (<a reveal-id="dsx055-F6" data-open="dsx055-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>).</p><p>Thus, for the <em>Tak1</em> gene, isoforms distributed in a cell-type-specific manner adequately fulfilled its functional plasticity. The full-length Tak1-B was the dominant isoform in HCs, regulating the cell-type specific functions such as lipid metabolism. While in NPCs, Rbfox2, as well as some other splicing factors, made Tak1-A the main isoform. Tak1-A bound to Tab2 more easily than Tak1-B did, activated the downstream JNK and p38 pathways. Thus, Tak1-A exhibited the immune process regulation properties in NPCs. These observations indicated that the regulation of CEEs by AS was closely associated with the cell-type-specific functions and had important roles in the formation of liver cell-type specificities.</p>                            <h2 scrollto-destination=118057792 id="118057792" class="section-title" >5. Discussion</h2>
<p>Although the cell-type-specific proteomic research in the mouse liver provided a better understanding of the identities of the individual cell type and how these cell types worked together,<a reveal-id="dsx055-B16" data-open="dsx055-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> we still wanted to know how this complexity of proteomics was generated and how AS variants distributed and functioned across different cell types. In this article, we identified 13,637 AS events from all four cell types, with an average coverage of 81.5% of all known events. We revealed the first profile of cell-type-specific AS events defined in mouse liver, and this profile was associated with a broad range of functional processes of the liver.</p><p>AS variants had differential distribution patterns across various cell types. In HCs, the AS-regulated genes were highly enriched in functions that include ‘regulation of kinase cascade’, ‘phosphoprotein’ and ‘metabolism’, whereas the genes involved in ’regulation of immune effector process’ in KCs were most frequently regulated by AS. This finding indicated that the genes with distinct cell-type-specific splicing manners were relevant to the core biological functions of respective cell types.</p><p>Increasing studies reported that some AS events had crucial impacts on multiple biological processes and developmental stages, especially in the nervous system.<a reveal-id="dsx055-B33" data-open="dsx055-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx055-B38" data-open="dsx055-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> In our research, the genes coding the kinase and transcription factors were more likely to be regulated by AS, which might be the important mechanism that AS caused wide impacts on multiple biological processes. More widely, the CEEs were significantly enriched in regions that involved in PPI regulation, which might also be the approach that AS affecting isoform functions.</p><p>AS is regulated by complicated interplays of <em>cis</em>-elements and <em>trans</em>-acting factors (splicing factors). The splicing factors recognize the ‘binding motifs’ to promote or suppress the splicing of the exon.<a reveal-id="dsx055-B39" data-open="dsx055-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> According to Professor Phillip A. Sharp, some splicing factors are required for dominating the specialized splicing patterns and have critical roles in maintaining cellular homeostasis, namely the ‘master splicing factors’.<a reveal-id="dsx055-B40" data-open="dsx055-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Here, we found several relatively high-expressed splicing factors in each cell type, like <em>Esrp2</em> in HCs and <em>Rbfox2</em> in LSECs. These splicing factors dominated the formation of the cell-type-specific splicing patterns (like the splicing pattern of <em>Tak1</em> in HCs and NPCs), demonstrating that the terminal of an alternative exon was depended on the splicing factors, cell types and the motif locations, revealing a precise regulation on the pre-mRNA level in specific cellular contexts. Besides, the functional analysis also showed that they regulated the specialized biological functions in respective cell types, which made them strong candidates for the ‘master splicing factors’ in mouse liver cells.</p><p>We were most interested in how a single AS event functioned diversely in distinct cell types. The conversion of Tak1-B to Tak1-A activated the downstream JNK and p38 pathways because of the enhanced Tak1-Tab2 interaction. The activated signal pathways had important roles in cellular inflammatory response, consistent with the distribution and the functionality of <em>Tak1-A</em> variant in NPCs. Besides, the conversion triggered a significant up-regulation of genes related to lipids and cholesterol synthesis, whereas lipolysis-related genes were down-regulated, coordinating with the functional predictions of <em>Tak1-B</em>. This example showed how AS regulated the isoform functions to shape the liver cell-type specificities.</p><p>In summary, our research provided the first profile of cell-type-specific transcriptome and splicing patterns in mouse liver and confirmed that the cell identities were largely defined by cell-type-specific AS events. These events altered isoform functions through remodeling PPI networks and modifying PTM sites and were controlled by a series of splicing factors. Analysis of cell-type-specific splicing enabled us to better demonstrating the functional landscape of hepatic cell types. Spliced genes played diverse roles in the corresponding cell types by differential exon usage even if they exhibited similar expression profiling at the gene level. This information improved our knowledge of additional regulation and appropriate composition of hepatic cell types.</p>                            <h2 scrollto-destination=118057799 id="118057799" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was partially supported by National Key R&amp;D Program of China (nos. 2016YFC0902400 and 2017YFC0906603), Chinese State Key Projects for Basic Research (‘973 Program’) (nos. 2014CBA02001 and 2013CB910502), National Natural Science Foundation of China (nos. 81770581, 81570526 and 81123001), Innovation project (16CXZ027), the Program of International S&amp;T Cooperation (2014DFB30020 and 2014DFB30010), Natural Science Foundation of Beijing (7152036), Beijing Science and Technology Project (Z161100002616036), Open Project Program of the State Key Laboratory of Proteomics (Academy of Military Medical Sciences, SKLP-O201509) and China Postdoctoral Science Foundation (2015M582850).</p>                            <h3 scrollto-destination=118057801 id="118057801" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=118057803 id="118057803" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=118057805 id="118057805" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx055-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nilsen</div> <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Graveley</div> <div class="given-names">B.R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Expansion of the eukaryotic proteome by alternative splicing</div>, <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">457</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expansion%20of%20the%20eukaryotic%20proteome%20by%20alternative%20splicing&amp;author=T.W.%20Nilsen&amp;author=B.R.%20Graveley&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=457-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08909" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08909"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Shai</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">L.J.</div></div>, <div class="name string-name"><div class="surname">Frey</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</div>, <div class="source">Nat. Genet.</div>  , <div class="volume">40</div>, <div class="fpage">1413</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=L.J.%20Lee&amp;author=B.J.%20Frey&amp;author=B.J.%20Blencowe&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=1413-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">E.T.</div></div>, <div class="name string-name"><div class="surname">Sandberg</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Alternative isoform regulation in human tissue transcriptomes</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">470</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;author=E.T.%20Wang&amp;author=R.%20Sandberg&amp;author=S.%20Luo&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=470-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kostadima</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Martens</div> <div class="given-names">J.H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transcriptional diversity during lineage commitment of human blood progenitors</div>, <div class="source">Science</div> , <div class="volume">345</div>, <div class="fpage">1251033</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20diversity%20during%20lineage%20commitment%20of%20human%20blood%20progenitors&amp;author=L.%20Chen&amp;author=M.%20Kostadima&amp;author=J.H.%20Martens&amp;publication_year=2014&amp;journal=Science&amp;volume=345&amp;pages=1251033" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1251033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1251033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1251033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25258084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lo</div> <div class="given-names">W.S.</div></div>, <div class="name string-name"><div class="surname">Gardiner</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Human tRNA synthetase catalytic nulls with diverse functions</div>, <div class="source">Science</div>  , <div class="volume">345</div>, <div class="fpage">328</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20tRNA%20synthetase%20catalytic%20nulls%20with%20diverse%20functions&amp;author=W.S.%20Lo&amp;author=E.%20Gardiner&amp;author=Z.%20Xu&amp;publication_year=2014&amp;journal=Science&amp;volume=345&amp;pages=328-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1252943" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1252943" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1252943"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25035493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buljan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chalancon</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Eustermann</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Tissue-specific splicing of disordered segments that embed binding motifs rewires protein interaction networks</div>, <div class="source">Mol. Cell</div> , <div class="volume">46</div>, <div class="fpage">871</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue-specific%20splicing%20of%20disordered%20segments%20that%20embed%20binding%20motifs%20rewires%20protein%20interaction%20networks&amp;author=M.%20Buljan&amp;author=G.%20Chalancon&amp;author=S.%20Eustermann&amp;publication_year=2012&amp;journal=Mol.%20Cell&amp;volume=46&amp;pages=871-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.05.039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.05.039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.05.039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22749400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ellis</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Barrios-Rodiles</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Colak</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Tissue-specific alternative splicing remodels protein-protein interaction networks</div>, <div class="source">Mol. Cell</div> , <div class="volume">46</div>, <div class="fpage">884</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue-specific%20alternative%20splicing%20remodels%20protein-protein%20interaction%20networks&amp;author=J.D.%20Ellis&amp;author=M.%20Barrios-Rodiles&amp;author=R.%20Colak&amp;publication_year=2012&amp;journal=Mol.%20Cell&amp;volume=46&amp;pages=884-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.05.037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.05.037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.05.037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22749401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Merkin</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Burge</div> <div class="given-names">C. B.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evolutionary dynamics of gene and isoform regulation in mammalian tissues</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">1593</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20gene%20and%20isoform%20regulation%20in%20mammalian%20tissues&amp;author=J.%20Merkin&amp;author=C.%20Russell&amp;author=P.%20Chen&amp;author=C.%20B.%20Burge&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1593-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1228186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1228186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1228186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barbosa-Morais</div> <div class="given-names">N.L.</div></div>, <div class="name string-name"><div class="surname">Irimia</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The evolutionary landscape of alternative splicing in vertebrate species</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">1587</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20landscape%20of%20alternative%20splicing%20in%20vertebrate%20species&amp;author=N.L.%20Barbosa-Morais&amp;author=M.%20Irimia&amp;author=Q.%20Pan&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1587-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1230612" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1230612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1230612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sudmant</div> <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Alexis</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Meta-analysis of RNA-seq expression data across species, tissues and studies</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">287</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meta-analysis%20of%20RNA-seq%20expression%20data%20across%20species%2C%20tissues%20and%20studies&amp;author=P.H.%20Sudmant&amp;author=M.S.%20Alexis&amp;author=C.B.%20Burge&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=287" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0853-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0853-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0853-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26694591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ergun</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Doran</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Costello</div> <div class="given-names">J.C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Differential splicing across immune system lineages</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">14324</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20splicing%20across%20immune%20system%20lineages&amp;author=A.%20Ergun&amp;author=G.%20Doran&amp;author=J.C.%20Costello&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=14324-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1311839110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1311839110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1311839110"> </span></div> </div></div></div></div></div><div content-id="dsx055-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kmiec</div> <div class="given-names">Z.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Cooperation of liver cells in health and disease</div>, <div class="source">Adv. Anat. Embryol. Cell Biol</div> ., <div class="volume">161</div>:III–XIII, <div class="fpage">1</div>–<div class="lpage">151</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cooperation%20of%20liver%20cells%20in%20health%20and%20disease&amp;author=Z.%20Kmiec&amp;publication_year=2001&amp;journal=Adv.%20Anat.%20Embryol.%20Cell%20Biol&amp;volume=161&amp;pages=1-151" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-642-56553-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-642-56553-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-642-56553-3"> </span></div> </div></div></div></div></div><div content-id="dsx055-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bale</div> <div class="given-names">S.S.</div></div>, <div class="name string-name"><div class="surname">Golberg</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Jindal</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Long-term coculture strategies for primary hepatocytes and liver sinusoidal endothelial cells</div>, <div class="source">Tissue Eng. Part C Methods</div> , <div class="volume">21</div>, <div class="fpage">413</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-term%20coculture%20strategies%20for%20primary%20hepatocytes%20and%20liver%20sinusoidal%20endothelial%20cells&amp;author=S.S.%20Bale&amp;author=I.%20Golberg&amp;author=R.%20Jindal&amp;publication_year=2015&amp;journal=Tissue%20Eng.%20Part%20C%20Methods&amp;volume=21&amp;pages=413-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/ten.tec.2014.0152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Ften.tec.2014.0152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Ften.tec.2014.0152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25233394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Okada</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kanki</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Liver resident macrophages (Kupffer Cells) share several functional antigens in common with endothelial cells</div>, <div class="source">Scand. J. Immunol</div> ., <div class="volume">83</div>, <div class="fpage">139</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Liver%20resident%20macrophages%20(Kupffer%20Cells)%20share%20several%20functional%20antigens%20in%20common%20with%20endothelial%20cells&amp;author=T.%20Okada&amp;author=A.%20Kimura&amp;author=K.%20Kanki&amp;publication_year=2016&amp;journal=Scand.%20J.%20Immunol&amp;volume=83&amp;pages=139-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/sji.12402" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fsji.12402" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fsji.12402"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26678711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Dioscin alleviates alcoholic liver fibrosis by attenuating hepatic stellate cell activation via the TLR4/MyD88/NF-kappaB signaling pathway</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">18038</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dioscin%20alleviates%20alcoholic%20liver%20fibrosis%20by%20attenuating%20hepatic%20stellate%20cell%20activation%20via%20the%20TLR4%2FMyD88%2FNF-kappaB%20signaling%20pathway&amp;author=M.%20Liu&amp;author=Y.%20Xu&amp;author=X.%20Han&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=18038" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep18038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep18038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep18038"> </span></div> </div></div></div></div></div><div content-id="dsx055-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Azimifar</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Nagaraj</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Mann</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Cell-type-resolved quantitative proteomics of murine liver</div>, <div class="source">Cell Metab</div> ., <div class="volume">20</div>, <div class="fpage">1076</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell-type-resolved%20quantitative%20proteomics%20of%20murine%20liver&amp;author=S.B.%20Azimifar&amp;author=N.%20Nagaraj&amp;author=J.%20Cox&amp;author=M.%20Mann&amp;publication_year=2014&amp;journal=Cell%20Metab&amp;volume=20&amp;pages=1076-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cmet.2014.11.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cmet.2014.11.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cmet.2014.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25470552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ding</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A Cell-type-resolved liver proteome</div>, <div class="source">Mol. Cell Proteomics</div> , <div class="volume">15</div>, <div class="fpage">3190</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Cell-type-resolved%20liver%20proteome&amp;author=C.%20Ding&amp;author=Y.%20Li&amp;author=F.%20Guo&amp;publication_year=2016&amp;journal=Mol.%20Cell%20Proteomics&amp;volume=15&amp;pages=3190-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.M116.060145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.M116.060145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.M116.060145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27562671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div>  , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.%20L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">MATS: a Bayesian framework for flexible detection of differential alternative splicing from RNA-Seq data</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MATS%3A%20a%20Bayesian%20framework%20for%20flexible%20detection%20of%20differential%20alternative%20splicing%20from%20RNA-Seq%20data&amp;author=S.%20Shen&amp;author=J.W.%20Park&amp;author=J.%20Huang&amp;publication_year=2012&amp;journal=Nucl.%20Acids%20Res&amp;volume=40&amp;pages=e61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1291"> </span></div> </div></div></div></div></div><div content-id="dsx055-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tate</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The Pfam protein families database</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">38</div>, <div class="fpage">D211</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Mistry&amp;author=J.%20Tate&amp;publication_year=2010&amp;journal=Nucl.%20Acids%20Res&amp;volume=38&amp;pages=D211-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp985"> </span></div> </div></div></div></div></div><div content-id="dsx055-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dosztanyi</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Csizmok</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Tompa</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Simon</div> <div class="given-names">I.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">IUPred: web server for the prediction of intrinsically unstructured regions of proteins based on estimated energy content</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">3433</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IUPred%3A%20web%20server%20for%20the%20prediction%20of%20intrinsically%20unstructured%20regions%20of%20proteins%20based%20on%20estimated%20energy%20content&amp;author=Z.%20Dosztanyi&amp;author=V.%20Csizmok&amp;author=P.%20Tompa&amp;author=I.%20Simon&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3433-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15955779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meszaros</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Simon</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Dosztanyi</div> <div class="given-names">Z.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Prediction of protein binding regions in disordered proteins</div>, <div class="source">PLoS Comput. Biol</div> ., <div class="volume">5</div>, <div class="fpage">e1000376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20protein%20binding%20regions%20in%20disordered%20proteins&amp;author=B.%20Meszaros&amp;author=I.%20Simon&amp;author=Z.%20Dosztanyi&amp;publication_year=2009&amp;journal=PLoS%20Comput.%20Biol&amp;volume=5&amp;pages=e1000376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19412530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang da</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>, <div class="name string-name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20enrichment%20tools%3A%20paths%20toward%20the%20comprehensive%20functional%20analysis%20of%20large%20gene%20lists&amp;author=W.%20Huang%20da&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2009&amp;journal=Nucl.%20Acids%20Res&amp;volume=37&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn923"> </span></div> </div></div></div></div></div><div content-id="dsx055-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Merico</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Isserlin</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Stueker</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Emili</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bader</div> <div class="given-names">G.D.</div></div>, <div class="name string-name"><div class="surname">Ravasi</div> <div class="given-names">T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Enrichment map: a network-based method for gene-set enrichment visualization and interpretation</div>, <div class="source">PloS One</div> , <div class="volume">5</div>, <div class="fpage">e13984</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20map%3A%20a%20network-based%20method%20for%20gene-set%20enrichment%20visualization%20and%20interpretation&amp;author=D.%20Merico&amp;author=R.%20Isserlin&amp;author=O.%20Stueker&amp;author=A.%20Emili&amp;author=G.D.%20Bader&amp;author=T.%20Ravasi&amp;publication_year=2010&amp;journal=PloS%20One&amp;volume=5&amp;pages=e13984" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013984" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013984" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013984"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21085593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shannon</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Markiel</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ozier</div> <div class="given-names">O.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2498</div>–<div class="lpage">504</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoscape%3A%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P.%20Shannon&amp;author=A.%20Markiel&amp;author=O.%20Ozier&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2498-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1239303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1239303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1239303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paz</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Kosti</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Ares</div> <div class="given-names">M.</div>Jr.</div>, <div class="name string-name"><div class="surname">Cline</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mandel-Gutfreund</div> <div class="given-names">Y.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">RBPmap: a web server for mapping binding sites of RNA-binding proteins</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">W361</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RBPmap%3A%20a%20web%20server%20for%20mapping%20binding%20sites%20of%20RNA-binding%20proteins&amp;author=I.%20Paz&amp;author=I.%20Kosti&amp;author=M.%20Ares&amp;author=M.%20Cline&amp;author=Y.%20Mandel-Gutfreund&amp;publication_year=2014&amp;journal=Nucl.%20Acids%20Res&amp;volume=42&amp;pages=W361-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku406" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku406" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku406"> </span></div> </div></div></div></div></div><div content-id="dsx055-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramirez</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vazquez-Sanchez</div> <div class="given-names">A.Y.</div></div>, <div class="name string-name"><div class="surname">Carrion-Robalino</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Camacho</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Ion channels and oxidative stress as a potential link for the diagnosis or treatment of liver diseases</div>, <div class="source">Oxid. Med. Cell. Longev</div> ., <div class="volume">2016</div>, <div class="fpage">3928714</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ion%20channels%20and%20oxidative%20stress%20as%20a%20potential%20link%20for%20the%20diagnosis%20or%20treatment%20of%20liver%20diseases&amp;author=A.%20Ramirez&amp;author=A.Y.%20Vazquez-Sanchez&amp;author=N.%20Carrion-Robalino&amp;author=J.%20Camacho&amp;publication_year=2016&amp;journal=Oxid.%20Med.%20Cell.%20Longev&amp;volume=2016&amp;pages=3928714" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2016/3928714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2016%2F3928714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2016%2F3928714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26881024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dempsey</div> <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Sakurai</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sugita</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Guesdon</div> <div class="given-names">F.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Alternative splicing and gene structure of the transforming growth factor beta-activated kinase 1</div>, <div class="source">Biochim Biophys Acta</div> , <div class="volume">1517</div>, <div class="fpage">46</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20gene%20structure%20of%20the%20transforming%20growth%20factor%20beta-activated%20kinase%201&amp;author=C.E.%20Dempsey&amp;author=H.%20Sakurai&amp;author=T.%20Sugita&amp;author=F.%20Guesdon&amp;publication_year=2000&amp;journal=Biochim%20Biophys%20Acta&amp;volume=1517&amp;pages=46-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-4781(00)00258-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-4781(00)00258-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-4781(00)00258-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11118615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sorensen</div> <div class="given-names">K.K.</div></div>, <div class="name string-name"><div class="surname">Simon-Santamaria</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">McCuskey</div> <div class="given-names">R.S.</div></div>, <div class="name string-name"><div class="surname">Smedsrod</div> <div class="given-names">B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Liver Sinusoidal Endothelial Cells</div>, <div class="source">Comp. Physiol</div> ., <div class="volume">5</div>, <div class="fpage">1751</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Liver%20Sinusoidal%20Endothelial%20Cells&amp;author=K.K.%20Sorensen&amp;author=J.%20Simon-Santamaria&amp;author=R.S.%20McCuskey&amp;author=B.%20Smedsrod&amp;publication_year=2015&amp;journal=Comp.%20Physiol&amp;volume=5&amp;pages=1751-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/cphy.c140078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fcphy.c140078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fcphy.c140078"> </span></div> </div></div></div></div></div><div content-id="dsx055-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marrone</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Shah</div> <div class="given-names">V.H.</div></div>, <div class="name string-name"><div class="surname">Gracia-Sancho</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Sinusoidal communication in liver fibrosis and regeneration</div>, <div class="source">J. Hepatol</div> , <div class="volume">65</div>, <div class="fpage">608</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sinusoidal%20communication%20in%20liver%20fibrosis%20and%20regeneration&amp;author=G.%20Marrone&amp;author=V.H.%20Shah&amp;author=J.%20Gracia-Sancho&amp;publication_year=2016&amp;journal=J.%20Hepatol&amp;volume=65&amp;pages=608-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jhep.2016.04.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jhep.2016.04.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jhep.2016.04.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27151183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bale</div> <div class="given-names">S.S.</div></div>, <div class="name string-name"><div class="surname">Geerts</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jindal</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yarmush</div> <div class="given-names">M.L.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Isolation and co-culture of rat parenchymal and non-parenchymal liver cells to evaluate cellular interactions and response</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">25329</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20co-culture%20of%20rat%20parenchymal%20and%20non-parenchymal%20liver%20cells%20to%20evaluate%20cellular%20interactions%20and%20response&amp;author=S.S.%20Bale&amp;author=S.%20Geerts&amp;author=R.%20Jindal&amp;author=M.L.%20Yarmush&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=25329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep25329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep25329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep25329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27142224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Raj</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Alternative splicing in the mammalian nervous system: recent insights into mechanisms and functional roles</div>, <div class="source">Neuron</div> , <div class="volume">87</div>, <div class="fpage">14</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20in%20the%20mammalian%20nervous%20system%3A%20recent%20insights%20into%20mechanisms%20and%20functional%20roles&amp;author=B.%20Raj&amp;author=B.J.%20Blencowe&amp;publication_year=2015&amp;journal=Neuron&amp;volume=87&amp;pages=14-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.neuron.2015.05.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.neuron.2015.05.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.neuron.2015.05.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26139367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wei</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">J.P.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">TNNT1, TNNT2, and TNNT3: isoform genes, regulation, and structure-function relationships</div>, <div class="source">Gene</div> , <div class="volume">582</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TNNT1%2C%20TNNT2%2C%20and%20TNNT3%3A%20isoform%20genes%2C%20regulation%2C%20and%20structure-function%20relationships&amp;author=B.%20Wei&amp;author=J.P.%20Jin&amp;publication_year=2016&amp;journal=Gene&amp;volume=582&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2016.01.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2016.01.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2016.01.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26774798" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pandit</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shiue</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Genome-wide analysis reveals SR protein cooperation and competition in regulated splicing</div>, <div class="source">Mol. Cell</div> ., <div class="volume">50</div>, <div class="fpage">223</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20reveals%20SR%20protein%20cooperation%20and%20competition%20in%20regulated%20splicing&amp;author=S.%20Pandit&amp;author=Y.%20Zhou&amp;author=L.%20Shiue&amp;publication_year=2013&amp;journal=Mol.%20Cell&amp;volume=50&amp;pages=223-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2013.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2013.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2013.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23562324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Omenn</div> <div class="given-names">G.S.</div></div>, <div class="name string-name"><div class="surname">Guan</div> <div class="given-names">Y.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">MIsoMine: a genome-scale high-resolution data portal of expression, function and networks at the splice isoform level in the mouse</div>. <div class="source">Database (Oxford)</div> , <div class="volume">2015</div>, bav045.</div></div></div></div><div content-id="dsx055-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arthur</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Ley</div> <div class="given-names">S.C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Mitogen-activated protein kinases in innate immunity</div>, <div class="source">Nat. Rev. Immunol</div> ., <div class="volume">13</div>, <div class="fpage">679</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitogen-activated%20protein%20kinases%20in%20innate%20immunity&amp;author=J.S.%20Arthur&amp;author=S.C.%20Ley&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Immunol&amp;volume=13&amp;pages=679-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nri3495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnri3495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnri3495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23954936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gueroussov</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gonatopoulos-Pournatzis</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Irimia</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">An alternative splicing event amplifies evolutionary differences between vertebrates</div>, <div class="source">Science</div> , <div class="volume">349</div>, <div class="fpage">868</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20alternative%20splicing%20event%20amplifies%20evolutionary%20differences%20between%20vertebrates&amp;author=S.%20Gueroussov&amp;author=T.%20Gonatopoulos-Pournatzis&amp;author=M.%20Irimia&amp;publication_year=2015&amp;journal=Science&amp;volume=349&amp;pages=868-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aaa8381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aaa8381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aaa8381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26293963" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Irimia</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Alternative splicing: decoding an expansive regulatory layer</div>, <div class="source">Curr. Opin. Cell Biol</div> ., <div class="volume">24</div>, <div class="fpage">323</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%3A%20decoding%20an%20expansive%20regulatory%20layer&amp;author=M.%20Irimia&amp;author=B.J.%20Blencowe&amp;publication_year=2012&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=24&amp;pages=323-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2012.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2012.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2012.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22465326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx055-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx055-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jangi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P.A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Building robust transcriptomes with master splicing factors</div>, <div class="source">Cell</div> , <div class="volume">159</div>, <div class="fpage">487</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Building%20robust%20transcriptomes%20with%20master%20splicing%20factors&amp;author=M.%20Jangi&amp;author=P.A.%20Sharp&amp;publication_year=2014&amp;journal=Cell&amp;volume=159&amp;pages=487-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.09.054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.09.054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.09.054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25417102" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4793385&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsx055/1/dsx055_supp.zip?Expires=2147483647&amp;Signature=Sp8gZNPrbWPOrHHk4V0IsWi8yL7ZxORMNW1toyMlUGCeA7odYkJ-pWByiTYkz1K~ITqhyb02xWyjcWbkh31ePAgxC2E-f4KgfatTSb7uKvffqJy4LKPX99I-fm4ninWGcJXfKhHOVONEOLKhgh5jIyS6K7lDPdgcCeA7iQiVf2ZmTbSEwLg4wdnmvW62XUPVGxnTNwCzoUOZpzejoJ~izph6Na85PCcXbNtDpIy6O9YGGlhmJcsgo4zhIDtRCR~DVa7Ds2Z3wyhXnNnrmEHAY~kzt2kCdzy52SBaOmjbSYSoVOcB3b0yDajJ-rVHfl4T2jOKoH4UOuCyqNp~rj9suw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,446</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx055"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4793385"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f3%2f265%2f4793385">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F3%2F265%2F4793385" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4793385'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Cell-type-resolved alternative splicing patterns in mouse liver" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx055");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f265%2f4793385">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

