

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/5/281/353619","@type":"ScholarlyArticle","name":"Genome Comparison and Phylogenetic Analysis of  Orientia tsutsugamushi  Strains","datePublished":"2010-08-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2010-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq018","keywords":["<italic>Orientia tsutsugamushi</italic>","genome sequence comparison","genome rearrangement","phylogenetic analysis","multilocus sequence analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/5/281/353619","author":[{"name":"Nakayama, Keisuke","affiliation":"1  Division of Microbiology, Department of Infectious Diseases ,  Faculty of Medicine, University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"2  Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 Nagatsuta, Yokohama, Kanagawa 226-0026 ,  Japan","@type":"Person"},{"name":"Fukuhara, Masahiro","affiliation":"3  Laboratory of Microbiology, Faculty of Pharmacy ,  Niigata University of Pharmacy and Applied Life Sciences ,  265-1 Higashijima, Akiha-ku, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Urakami, Hiroshi","affiliation":"4  Laboratory of Food Microbiology and Food Safety, Faculty of Applied Life Sciences ,  Niigata University of Pharmacy and Applied Life Sciences ,  265-1 Higashijima, Akiha-ku, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Yamamoto, Seigo","affiliation":"5  Miyazaki Prefectural Institute for Public Health and Environment, 2-3-2 Gakuen Kibanadai Nishi, Miyazaki 889-2155 ,  Japan","@type":"Person"},{"name":"Yamazaki, Kazuko","affiliation":"6  Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"6  Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Ooka, Tadasuke","affiliation":"1  Division of Microbiology, Department of Infectious Diseases ,  Faculty of Medicine, University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"1  Division of Microbiology, Department of Infectious Diseases ,  Faculty of Medicine, University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan    6  Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 899-1692 ,  Japan","@type":"Person"}],"description":"Abstract.  Orientia tsutsugamushi (OT) is an obligate intracellular bacterium belonging to the family Rickettsiaceae and is the causative agent of scrub typhus,","pageStart":"281","pageEnd":"291","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome Comparison and Phylogenetic Analysis of  Orientia tsutsugamushi  Strains","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome Comparison and Phylogenetic Analysis of  Orientia tsutsugamushi  Strains" />
    <meta property="og:description" content="Abstract.  Orientia tsutsugamushi (OT) is an obligate intracellular bacterium belonging to the family Rickettsiaceae and is the causative agent of scrub typhus," />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/5/281/353619" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-08-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nakayama, Keisuke" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 Nagatsuta, Yokohama, Kanagawa 226-0026, Japan" /><meta name="citation_author" content="Fukuhara, Masahiro" /><meta name="citation_author_institution" content="Laboratory of Microbiology, Faculty of Pharmacy, Niigata University of Pharmacy and Applied Life Sciences, 265-1 Higashijima, Akiha-ku, Niigata 956-8603, Japan" /><meta name="citation_author" content="Urakami, Hiroshi" /><meta name="citation_author_institution" content="Laboratory of Food Microbiology and Food Safety, Faculty of Applied Life Sciences, Niigata University of Pharmacy and Applied Life Sciences, 265-1 Higashijima, Akiha-ku, Niigata 956-8603, Japan" /><meta name="citation_author" content="Yamamoto, Seigo" /><meta name="citation_author_institution" content="Miyazaki Prefectural Institute for Public Health and Environment, 2-3-2 Gakuen Kibanadai Nishi, Miyazaki 889-2155, Japan" /><meta name="citation_author" content="Yamazaki, Kazuko" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Ooka, Tadasuke" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 899-1692, Japan" /><meta name="citation_title" content="Genome Comparison and Phylogenetic Analysis of Orientia tsutsugamushi Strains" /><meta name="citation_firstpage" content="281" /><meta name="citation_lastpage" content="291" /><meta name="citation_doi" content="10.1093/dnares/dsq018" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2010/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Vertical transmission in chigger borne rickettsiosis; Indian J. Med. Res.;  citation_year=1978;  citation_volume=68;  citation_pages=31-8; " /><meta name="citation_reference" content="citation_title=Transovarial transmission of Rickettsia tsutsugamushi in Leptotrombidium (Leptotrombidium) arenicola Traub (Acarina: Trombiculidae); J. Med. Entomol.;  citation_year=1972;  citation_volume=9;  citation_pages=71-2; " /><meta name="citation_reference" content="citation_title=Transovarial development of scrub typhus rickettsiae in a colony of vector mites; Trans. R. Soc. Trop. Med. Hyg.;  citation_year=1969;  citation_volume=63;  citation_pages=251-8; " /><meta name="citation_reference" content="citation_title=Transovarial transmission of Rickettsia tsutsugamushi in Leptotrombidium pallidum successively reared in the laboratory; Jpn J. Exp. Med.;  citation_year=1988;  citation_volume=58;  citation_pages=213-8; " /><meta name="citation_reference" content="citation_title=An ultrastructural study of vertical transmission of Rickettsia tsutsugamushi during oogenesis and spermatogenesis in Leptotrombidium pallidum; Am. J. Trop. Med. Hyg.;  citation_year=1994;  citation_volume=50;  citation_pages=219-28; " /><meta name="citation_reference" content="citation_title=Failed vertical transmission of Rickettsia tsutsugamushi (Rickettsiales: Rickettsiaceae) acquired from rickettsemic mice by Leptotrombidium pallidum (Acari: Trombiculidae); J. Med. Entomol.;  citation_year=1994;  citation_volume=31;  citation_pages=212-6; " /><meta name="citation_reference" content="citation_title=Attempts to infect and demonstrate transovarial transmission of R. tsutsugamushi in three species of Leptotrombidium mites; Ann. N Y Acad. Sci.;  citation_year=1975;  citation_volume=266;  citation_pages=80-90; " /><meta name="citation_reference" content="citation_title=The acquisition of Rickettsia tsutsugamushi by chiggers (trombiculid mites) during the feeding process; Ann. N Y Acad. Sci.;  citation_year=1975;  citation_volume=266;  citation_pages=91-114; " /><meta name="citation_reference" content="citation_title=Scrub typhus: the geographic distribution of phenotypic and genotypic variants of Orientia tsutsugamushi; Clin. Infect. Dis.;  citation_year=2009;  citation_volume=48;  citation_pages=S203-30; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of Orientia tsutsugamushi strains based on the sequence homologies of 56-kDa type-specific antigen genes; FEMS Microbiol. Lett.;  citation_year=1999;  citation_volume=15;  citation_pages=163-9; " /><meta name="citation_reference" content="citation_title=Demonstration of antigenic and genotypic variation in Orientia tsutsugamushi which were isolated in Japan, and their classification into type and subtype; Microbiol. Immunol.;  citation_year=1996;  citation_volume=40;  citation_pages=627-38; " /><meta name="citation_reference" content="citation_title=The Orientia tsutsugamushi genome reveals massive proliferation of conjugative type IV secretion system and host-cell interaction genes; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=8;  citation_pages=7981-6; " /><meta name="citation_reference" content="citation_title=The whole-genome sequencing of the obligate intracellular bacterium Orientia tsutsugamushi revealed massive gene amplification during reductive genome evolution; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=185-99; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Improved tools for biological sequence comparison; Proc. Natl Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2444-8; " /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison; BMC Bioinformatics;  citation_year=2008;  citation_volume=16;  citation_pages=376" /><meta name="citation_reference" content="citation_title=Genomic diversity of enterohemorrhagic Escherichia coli O157 revealed by whole genome PCR scanning; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=17043-8; " /><meta name="citation_reference" content="citation_title=MEGA: a biologist-centric software for evolutionary analysis of DNA and protein sequences; Brief Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=299-306; " /><meta name="citation_reference" content="citation_title=Ori-Finder: a web-based system for finding oriCs in unannotated bacterial genomes; BMC Bioinformatics;  citation_year=2008;  citation_volume=1;  citation_pages=79" /><meta name="citation_reference" content="citation_title=In situ detection of novel bacterial endosymbionts of Acanthamoeba spp. phylogenetically related to members of the order Rickettsiales; Appl. Environ. Microbiol.;  citation_year=1999;  citation_volume=65;  citation_pages=206-12; " /><meta name="citation_reference" content="citation_title=Genome sequence of Rickettsia bellii illuminates the role of amoebae in gene exchanges between intracellular pathogens; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=733-44; " /><meta name="citation_reference" content="citation_title=The genome of the amoeba symbiont ‘Candidatus Amoebophilus asiaticus’ reveals common mechanisms for host cell interaction among amoeba-associated bacteria; J. Bacteriol.;  citation_year=2010;  citation_volume=194;  citation_pages=1045-57; " /><meta name="citation_reference" content="citation_title=Diversity of immunodominant 56-kDa type-specific antigen (TSA) of Rickettsia tsutsugamushi. Sequence and comparative analyses of the genes encoding TSA homologues from four antigenic variants; J. Biol. Chem.;  citation_year=1992;  citation_volume=267;  citation_pages=12728-35; " /><meta name="citation_reference" content="citation_title=Epidemiology of Tsutsugamushi disease in relation to the serotypes of Rickettsia tsutsugamushi isolated from patients, field mice, and unfed chiggers on the eastern slope of Mount Fuji, Shizuoka Prefecture, Japan; J. Clin. Microbiol.;  citation_year=1992;  citation_volume=30;  citation_pages=2842-6; " /><meta name="citation_reference" content="citation_title=Epidemiological survey by means of complement fixation test on scrub typhus in Japan; Acta Med. Biol.;  citation_year=1967;  citation_volume=15;  citation_issue=Suppl;  citation_pages=69-85; " /><meta name="citation_reference" content="citation_title=Strain variation of Rickettsia orientalis in the complement fixation test; Jpn J. Med. Sci. Biol.;  citation_year=1964;  citation_volume=17;  citation_pages=59-72; " /><meta name="citation_reference" content="citation_title=Epidemiological survey of Orientia tsutsugamushi distribution in field rodents in Saitama Prefecture, Japan, and discovery of a new type; Microbiol. Immunol.;  citation_year=2001;  citation_volume=45;  citation_pages=439-46; " /><meta name="citation_reference" content="citation_title=Isolation of Rickettsia tsutsugamushi antigenically different from Kato, Karp, and Gilliam strains from patients; Microbiol. Immunol.;  citation_year=1984;  citation_volume=28;  citation_pages=873-882; " /><meta name="citation_reference" content="citation_title=Serological classification by monoclonal antibodies of Rickettsia tsutsugamushi isolated in Korea; J. Clin. Microbiol;  citation_year=1990;  citation_volume=28;  citation_pages=685-688; " /><meta name="citation_reference" content="citation_title=Apparent serological heterogeneity among strains of tsutsugamushi disease (scrub typhus); Public Health Rep.;  citation_year=1945;  citation_volume=60;  citation_pages=1483-1488; " /><meta name="citation_reference" content="citation_title=The nature of immunity against scrub typhus in mice. I. The resistance of mice, surviving subcutaneous infection of scrub typhus rickettsia, to intraperitoneal reinfection of the same agent; Jpn. J. Med. Sci. Biol.;  citation_year=1958;  citation_volume=11;  citation_pages=383-399; " /><meta name="citation_reference" content="citation_title=Isolation of the Karp strain of rickettsia tsutsugamushi; Lancet;  citation_year=1949;  citation_volume=2;  citation_pages=150-151; " /><meta name="citation_reference" content="citation_title=Characterization of a new antigenic type, Kuroki, of Rickettsia tsutsugamushi isolated from a patient in Japan; J. Clin. Microbiol.;  citation_year=1990;  citation_volume=28;  citation_pages=2111-2113; " /><meta name="citation_reference" content="citation_title=Immunological properties of Rickettsia tsutsugamushi, Kawasaki strain, isolated from a patient in Kyushu; Microbiol. Immunol.;  citation_year=1986;  citation_volume=30;  citation_pages=611-620; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/5/281/1011320/dsq018.pdf" /><meta name="description" content="Abstract.  Orientia tsutsugamushi (OT) is an obligate intracellular bacterium belonging to the family Rickettsiaceae and is the causative agent of scrub typhus," />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/5/281/353619" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/5/281/353619" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/5/281/353619");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq018';
    </script>
<title>
	Genome Comparison and Phylogenetic Analysis of Orientia tsutsugamushi Strains | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353619" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BftgPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgS2Vpc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdXJva2F3YSwgS2VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCA0MjU5IE5hZ2F0c3V0YSwgWW9rb2hhbWEsIEthbmFnYXdhIDIyNi0wMDI2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3VoYXJhLCBNYXNhaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTWljcm9iaW9sb2d5LCBGYWN1bHR5IG9mIFBoYXJtYWN5LCBOaWlnYXRhIFVuaXZlcnNpdHkgb2YgUGhhcm1hY3kgYW5kIEFwcGxpZWQgTGlmZSBTY2llbmNlcywgMjY1LTEgSGlnYXNoaWppbWEsIEFraWhhLWt1LCBOaWlnYXRhIDk1Ni04NjAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVyYWthbWksIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEZvb2QgTWljcm9iaW9sb2d5IGFuZCBGb29kIFNhZmV0eSwgRmFjdWx0eSBvZiBBcHBsaWVkIExpZmUgU2NpZW5jZXMsIE5paWdhdGEgVW5pdmVyc2l0eSBvZiBQaGFybWFjeSBhbmQgQXBwbGllZCBMaWZlIFNjaWVuY2VzLCAyNjUtMSBIaWdhc2hpamltYSwgQWtpaGEta3UsIE5paWdhdGEgOTU2LTg2MDMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYW1vdG8sIFNlaWdvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWl5YXpha2kgUHJlZmVjdHVyYWwgSW5zdGl0dXRlIGZvciBQdWJsaWMgSGVhbHRoIGFuZCBFbnZpcm9ubWVudCwgMi0zLTIgR2FrdWVuIEtpYmFuYWRhaSBOaXNoaSwgTWl5YXpha2kgODg5LTIxNTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYXpha2ksIEthenVrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ3VyYSwgWW9zaGl0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPb2thLCBUYWRhc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXlhc2hpLCBUZXRzdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDUyMDAgS2l5b3Rha2UsIE1peWF6YWtpIDg5OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4OTktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZSBDb21wYXJpc29uIGFuZCBQaHlsb2dlbmV0aWMgQW5hbHlzaXMgb2YgT3JpZW50aWEgdHN1dHN1Z2FtdXNoaSBTdHJhaW5zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3EwMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlcnRpY2FsIHRyYW5zbWlzc2lvbiBpbiBjaGlnZ2VyIGJvcm5lIHJpY2tldHRzaW9zaXM7IEluZGlhbiBKLiBNZWQuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0zMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc292YXJpYWwgdHJhbnNtaXNzaW9uIG9mIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaSBpbiBMZXB0b3Ryb21iaWRpdW0gKExlcHRvdHJvbWJpZGl1bSkgYXJlbmljb2xhIFRyYXViIChBY2FyaW5hOiBUcm9tYmljdWxpZGFlKTsgSi4gTWVkLiBFbnRvbW9sLjsgIGNpdGF0aW9uX3llYXI9MTk3MjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NzEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNvdmFyaWFsIGRldmVsb3BtZW50IG9mIHNjcnViIHR5cGh1cyByaWNrZXR0c2lhZSBpbiBhIGNvbG9ueSBvZiB2ZWN0b3IgbWl0ZXM7IFRyYW5zLiBSLiBTb2MuIFRyb3AuIE1lZC4gSHlnLjsgIGNpdGF0aW9uX3llYXI9MTk2OTsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTI1MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc292YXJpYWwgdHJhbnNtaXNzaW9uIG9mIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaSBpbiBMZXB0b3Ryb21iaWRpdW0gcGFsbGlkdW0gc3VjY2Vzc2l2ZWx5IHJlYXJlZCBpbiB0aGUgbGFib3JhdG9yeTsgSnBuIEouIEV4cC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTIxMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiB1bHRyYXN0cnVjdHVyYWwgc3R1ZHkgb2YgdmVydGljYWwgdHJhbnNtaXNzaW9uIG9mIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaSBkdXJpbmcgb29nZW5lc2lzIGFuZCBzcGVybWF0b2dlbmVzaXMgaW4gTGVwdG90cm9tYmlkaXVtIHBhbGxpZHVtOyBBbS4gSi4gVHJvcC4gTWVkLiBIeWcuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MjE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYWlsZWQgdmVydGljYWwgdHJhbnNtaXNzaW9uIG9mIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaSAoUmlja2V0dHNpYWxlczogUmlja2V0dHNpYWNlYWUpIGFjcXVpcmVkIGZyb20gcmlja2V0dHNlbWljIG1pY2UgYnkgTGVwdG90cm9tYmlkaXVtIHBhbGxpZHVtIChBY2FyaTogVHJvbWJpY3VsaWRhZSk7IEouIE1lZC4gRW50b21vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yMTItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXR0ZW1wdHMgdG8gaW5mZWN0IGFuZCBkZW1vbnN0cmF0ZSB0cmFuc292YXJpYWwgdHJhbnNtaXNzaW9uIG9mIFIuIHRzdXRzdWdhbXVzaGkgaW4gdGhyZWUgc3BlY2llcyBvZiBMZXB0b3Ryb21iaWRpdW0gbWl0ZXM7IEFubi4gTiBZIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9ODAtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhY3F1aXNpdGlvbiBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgYnkgY2hpZ2dlcnMgKHRyb21iaWN1bGlkIG1pdGVzKSBkdXJpbmcgdGhlIGZlZWRpbmcgcHJvY2VzczsgQW5uLiBOIFkgQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjY7ICBjaXRhdGlvbl9wYWdlcz05MS0xMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjcnViIHR5cGh1czogdGhlIGdlb2dyYXBoaWMgZGlzdHJpYnV0aW9uIG9mIHBoZW5vdHlwaWMgYW5kIGdlbm90eXBpYyB2YXJpYW50cyBvZiBPcmllbnRpYSB0c3V0c3VnYW11c2hpOyBDbGluLiBJbmZlY3QuIERpcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz1TMjAzLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgYW5hbHlzaXMgb2YgT3JpZW50aWEgdHN1dHN1Z2FtdXNoaSBzdHJhaW5zIGJhc2VkIG9uIHRoZSBzZXF1ZW5jZSBob21vbG9naWVzIG9mIDU2LWtEYSB0eXBlLXNwZWNpZmljIGFudGlnZW4gZ2VuZXM7IEZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE2My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZW1vbnN0cmF0aW9uIG9mIGFudGlnZW5pYyBhbmQgZ2Vub3R5cGljIHZhcmlhdGlvbiBpbiBPcmllbnRpYSB0c3V0c3VnYW11c2hpIHdoaWNoIHdlcmUgaXNvbGF0ZWQgaW4gSmFwYW4sIGFuZCB0aGVpciBjbGFzc2lmaWNhdGlvbiBpbnRvIHR5cGUgYW5kIHN1YnR5cGU7IE1pY3JvYmlvbC4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz02MjctMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBPcmllbnRpYSB0c3V0c3VnYW11c2hpIGdlbm9tZSByZXZlYWxzIG1hc3NpdmUgcHJvbGlmZXJhdGlvbiBvZiBjb25qdWdhdGl2ZSB0eXBlIElWIHNlY3JldGlvbiBzeXN0ZW0gYW5kIGhvc3QtY2VsbCBpbnRlcmFjdGlvbiBnZW5lczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Nzk4MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgd2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2YgdGhlIG9ibGlnYXRlIGludHJhY2VsbHVsYXIgYmFjdGVyaXVtIE9yaWVudGlhIHRzdXRzdWdhbXVzaGkgcmV2ZWFsZWQgbWFzc2l2ZSBnZW5lIGFtcGxpZmljYXRpb24gZHVyaW5nIHJlZHVjdGl2ZSBnZW5vbWUgZXZvbHV0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE4NS05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCB0b29scyBmb3IgYmlvbG9naWNhbCBzZXF1ZW5jZSBjb21wYXJpc29uOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9MjQ0NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWVNYXRjaGVyOiBhIGdyYXBoaWNhbCB1c2VyIGludGVyZmFjZSBmb3IgRE5BIHNlcXVlbmNlIGNvbXBhcmlzb247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgZGl2ZXJzaXR5IG9mIGVudGVyb2hlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgTzE1NyByZXZlYWxlZCBieSB3aG9sZSBnZW5vbWUgUENSIHNjYW5uaW5nOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTcwNDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTogYSBiaW9sb2dpc3QtY2VudHJpYyBzb2Z0d2FyZSBmb3IgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIEROQSBhbmQgcHJvdGVpbiBzZXF1ZW5jZXM7IEJyaWVmIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaS1GaW5kZXI6IGEgd2ViLWJhc2VkIHN5c3RlbSBmb3IgZmluZGluZyBvcmlDcyBpbiB1bmFubm90YXRlZCBiYWN0ZXJpYWwgZ2Vub21lczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz03OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHNpdHUgZGV0ZWN0aW9uIG9mIG5vdmVsIGJhY3RlcmlhbCBlbmRvc3ltYmlvbnRzIG9mIEFjYW50aGFtb2ViYSBzcHAuIHBoeWxvZ2VuZXRpY2FsbHkgcmVsYXRlZCB0byBtZW1iZXJzIG9mIHRoZSBvcmRlciBSaWNrZXR0c2lhbGVzOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9MjA2LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgUmlja2V0dHNpYSBiZWxsaWkgaWxsdW1pbmF0ZXMgdGhlIHJvbGUgb2YgYW1vZWJhZSBpbiBnZW5lIGV4Y2hhbmdlcyBiZXR3ZWVuIGludHJhY2VsbHVsYXIgcGF0aG9nZW5zOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NzMzLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBhbW9lYmEgc3ltYmlvbnQg4oCYQ2FuZGlkYXR1cyBBbW9lYm9waGlsdXMgYXNpYXRpY3Vz4oCZIHJldmVhbHMgY29tbW9uIG1lY2hhbmlzbXMgZm9yIGhvc3QgY2VsbCBpbnRlcmFjdGlvbiBhbW9uZyBhbW9lYmEtYXNzb2NpYXRlZCBiYWN0ZXJpYTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDQ1LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgb2YgaW1tdW5vZG9taW5hbnQgNTYta0RhIHR5cGUtc3BlY2lmaWMgYW50aWdlbiAoVFNBKSBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkuIFNlcXVlbmNlIGFuZCBjb21wYXJhdGl2ZSBhbmFseXNlcyBvZiB0aGUgZ2VuZXMgZW5jb2RpbmcgVFNBIGhvbW9sb2d1ZXMgZnJvbSBmb3VyIGFudGlnZW5pYyB2YXJpYW50czsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjY3OyAgY2l0YXRpb25fcGFnZXM9MTI3MjgtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWRlbWlvbG9neSBvZiBUc3V0c3VnYW11c2hpIGRpc2Vhc2UgaW4gcmVsYXRpb24gdG8gdGhlIHNlcm90eXBlcyBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgaXNvbGF0ZWQgZnJvbSBwYXRpZW50cywgZmllbGQgbWljZSwgYW5kIHVuZmVkIGNoaWdnZXJzIG9uIHRoZSBlYXN0ZXJuIHNsb3BlIG9mIE1vdW50IEZ1amksIFNoaXp1b2thIFByZWZlY3R1cmUsIEphcGFuOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9Mjg0Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlkZW1pb2xvZ2ljYWwgc3VydmV5IGJ5IG1lYW5zIG9mIGNvbXBsZW1lbnQgZml4YXRpb24gdGVzdCBvbiBzY3J1YiB0eXBodXMgaW4gSmFwYW47IEFjdGEgTWVkLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk2NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsOyAgY2l0YXRpb25fcGFnZXM9NjktODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmFpbiB2YXJpYXRpb24gb2YgUmlja2V0dHNpYSBvcmllbnRhbGlzIGluIHRoZSBjb21wbGVtZW50IGZpeGF0aW9uIHRlc3Q7IEpwbiBKLiBNZWQuIFNjaS4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5NjQ7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz01OS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZGVtaW9sb2dpY2FsIHN1cnZleSBvZiBPcmllbnRpYSB0c3V0c3VnYW11c2hpIGRpc3RyaWJ1dGlvbiBpbiBmaWVsZCByb2RlbnRzIGluIFNhaXRhbWEgUHJlZmVjdHVyZSwgSmFwYW4sIGFuZCBkaXNjb3Zlcnkgb2YgYSBuZXcgdHlwZTsgTWljcm9iaW9sLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTQzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIFJpY2tldHRzaWEgdHN1dHN1Z2FtdXNoaSBhbnRpZ2VuaWNhbGx5IGRpZmZlcmVudCBmcm9tIEthdG8sIEthcnAsIGFuZCBHaWxsaWFtIHN0cmFpbnMgZnJvbSBwYXRpZW50czsgTWljcm9iaW9sLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTg3My04ODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcm9sb2dpY2FsIGNsYXNzaWZpY2F0aW9uIGJ5IG1vbm9jbG9uYWwgYW50aWJvZGllcyBvZiBSaWNrZXR0c2lhIHRzdXRzdWdhbXVzaGkgaXNvbGF0ZWQgaW4gS29yZWE7IEouIENsaW4uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTY4NS02ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGFyZW50IHNlcm9sb2dpY2FsIGhldGVyb2dlbmVpdHkgYW1vbmcgc3RyYWlucyBvZiB0c3V0c3VnYW11c2hpIGRpc2Vhc2UgKHNjcnViIHR5cGh1cyk7IFB1YmxpYyBIZWFsdGggUmVwLjsgIGNpdGF0aW9uX3llYXI9MTk0NTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTE0ODMtMTQ4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG5hdHVyZSBvZiBpbW11bml0eSBhZ2FpbnN0IHNjcnViIHR5cGh1cyBpbiBtaWNlLiBJLiBUaGUgcmVzaXN0YW5jZSBvZiBtaWNlLCBzdXJ2aXZpbmcgc3ViY3V0YW5lb3VzIGluZmVjdGlvbiBvZiBzY3J1YiB0eXBodXMgcmlja2V0dHNpYSwgdG8gaW50cmFwZXJpdG9uZWFsIHJlaW5mZWN0aW9uIG9mIHRoZSBzYW1lIGFnZW50OyBKcG4uIEouIE1lZC4gU2NpLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk1ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM4My0zOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiB0aGUgS2FycCBzdHJhaW4gb2Ygcmlja2V0dHNpYSB0c3V0c3VnYW11c2hpOyBMYW5jZXQ7ICBjaXRhdGlvbl95ZWFyPTE5NDk7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE1MC0xNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBuZXcgYW50aWdlbmljIHR5cGUsIEt1cm9raSwgb2YgUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpIGlzb2xhdGVkIGZyb20gYSBwYXRpZW50IGluIEphcGFuOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjExMS0yMTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbW11bm9sb2dpY2FsIHByb3BlcnRpZXMgb2YgUmlja2V0dHNpYSB0c3V0c3VnYW11c2hpLCBLYXdhc2FraSBzdHJhaW4sIGlzb2xhdGVkIGZyb20gYSBwYXRpZW50IGluIEt5dXNodTsgTWljcm9iaW9sLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTYxMS02MjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy81LzI4MS8xMDExMzIwL2RzcTAxOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgT3JpZW50aWEgdHN1dHN1Z2FtdXNoaSAoT1QpIGlzIGFuIG9ibGlnYXRlIGludHJhY2VsbHVsYXIgYmFjdGVyaXVtIGJlbG9uZ2luZyB0byB0aGUgZmFtaWx5IFJpY2tldHRzaWFjZWFlIGFuZCBpcyB0aGUgY2F1c2F0aXZlIGFnZW50IG9mIHNjcnViIHR5cGh1cywiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1MzYxOSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQcGDbO4eaVmUEegVmtHnFmdz4dqQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f281%2f353619" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f281%2f353619" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13694" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&amp;Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/5">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5034118">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034118">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034120">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034120">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034125">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034125">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034136">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034136">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034206">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034206">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034208">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034208">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034210">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034210">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034212">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034212">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/5/271/353434">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/5/293/353856">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome Comparison and Phylogenetic Analysis of <em>Orientia tsutsugamushi</em> Strains
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keisuke Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases</div>, <div class="insititution">Faculty of Medicine, University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keisuke+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Keisuke Nakayama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 Nagatsuta, Yokohama, Kanagawa 226-0026</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahiro Fukuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Laboratory of Microbiology, Faculty of Pharmacy</div>, <div class="insititution">Niigata University of Pharmacy and Applied Life Sciences</div>, <div class="addr-line">265-1 Higashijima, Akiha-ku, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahiro+Fukuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuhara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuhara M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masahiro Fukuhara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Urakami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Food Microbiology and Food Safety, Faculty of Applied Life Sciences</div>, <div class="insititution">Niigata University of Pharmacy and Applied Life Sciences</div>, <div class="addr-line">265-1 Higashijima, Akiha-ku, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Urakami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Urakami H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Urakami H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Urakami</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seigo Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Miyazaki Prefectural Institute for Public Health and Environment, 2-3-2 Gakuen Kibanadai Nishi, Miyazaki 889-2155</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seigo+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seigo Yamamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuko Yamazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuko+Yamazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamazaki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamazaki K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuko Yamazaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadasuke Ooka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases</div>, <div class="insititution">Faculty of Medicine, University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadasuke+Ooka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ooka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ooka T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tadasuke Ooka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases</div>, <div class="insititution">Faculty of Medicine, University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 899-1692</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 985-85-0871. Fax. <div class="fax">+81 985-85-6475</div>. E-mail: <a href="mailto:thayash@med.miyazaki-u.ac.jp" target="_blank">thayash@med.miyazaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 5, 1 October 2010, Pages 281–291, <a href='https://doi.org/10.1093/dnares/dsq018'>https://doi.org/10.1093/dnares/dsq018</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 August 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 April 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 June 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353619" href="/dnaresearch/article-pdf/17/5/281/1011320/dsq018.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Keisuke Nakayama, Ken Kurokawa, Masahiro Fukuhara, Hiroshi Urakami, Seigo Yamamoto, Kazuko Yamazaki, Yoshitoshi Ogura, Tadasuke Ooka, Tetsuya Hayashi;  Genome Comparison and Phylogenetic Analysis of <em>Orientia tsutsugamushi</em> Strains, <em>DNA Research</em>, Volume 17, Issue 5, 1 October 2010, Pages 281–291, <a href='https://doi.org/10.1093/dnares/dsq018'>https://doi.org/10.1093/dnares/dsq018</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353619?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353619?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353619?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353619?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353619?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353619?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353619?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20Comparison%20and%20Phylogenetic%20Analysis%20of%20%26lt%3Bem%26gt%3BOrientia%20tsutsugamushi%26lt%3B%2Fem%26gt%3B%20Strains&amp;publicationDate=2010-08-03&amp;volumeNum=17&amp;issueNum=5&amp;author=Nakayama%2C%20Keisuke%3B%20Kurokawa%2C%20Ken&amp;startPage=281&amp;endPage=291&amp;contentId=10.1093%2Fdnares%2Fdsq018&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f281%2f353619" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5034118 id="5034118" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Orientia tsutsugamushi</em> (OT) is an obligate intracellular bacterium belonging to the family Rickettsiaceae and is the causative agent of scrub typhus, or Tsutsugamushi disease. The complete genome sequences of two OT strains (Boryong and Ikeda) have recently been determined. In the present study, we performed a fine genome sequence comparison of these strains. Our results indicate that although the core gene set of the family Rickettsiaceae is highly conserved between the two strains, a common set of repetitive sequences have been explosively amplified in both genomes. These amplified repetitive sequences have induced extensive genome shuffling and duplications and deletions of many genes. On the basis of the results of the genome sequence comparison, we selected 11 housekeeping genes and carried out multilocus sequence analysis of OT strains using the nucleotide sequences of these genes. This analysis revealed for the first time the phylogenetic relationships of representative OT strains. Furthermore, the results suggest the presence of an OT lineage with higher potential for virulence, which may explain the clinical and epidemiological differences between ‘classic’ and ‘new’ types of Tsutsugamushi disease in Japan.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Orientia tsutsugamushi&quot;"><em>Orientia tsutsugamushi</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence comparison&quot;">genome sequence comparison</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome rearrangement&quot;">genome rearrangement</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic analysis&quot;">phylogenetic analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;multilocus sequence analysis&quot;">multilocus sequence analysis</a></div>                            <h2 scrollto-destination=5034120 id="5034120" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Orientia tsutsugamushi</em> (OT), an obligate intracellular bacterium belonging to the family Rickettsiaceae of the subdivision alpha-Proteobacteria, is the causative agent of scrub typhus, or Tsutsugamushi disease. The vector and the reservoir of OT are trombiculid mites. The bacteria reside in the cytosol of mite cells in various organs and are efficiently inherited by their offspring through transovarial transmission.<sup><a reveal-id="DSQ018C1  DSQ018C5" data-open="DSQ018C1  DSQ018C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> Reverse transfer from infected animals to mites occurs infrequently, and the bacteria transmitted in this way are not usually passed on to offspring. Thus, limited lines of mites retain the bacterium,<sup><a reveal-id="DSQ018C6  DSQ018C8" data-open="DSQ018C6  DSQ018C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> and a correlation is observed between the species of host mite and the serotype of the colonizing OT strain.<sup><a reveal-id="DSQ018C9" data-open="DSQ018C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>Strain typing of OT was first made on the basis of antigenic variation (serotyping), but a universal scheme for serotyping has not yet been established. Later, on the basis of nucleotide (nt) sequence variation in the gene encoding a major outer membrane protein called the 56-kDa <em>Orientia</em> type-specific antigen (TSA), Tamura and his collaborators divided OT strains into the following subtypes: Gilliam, JG (Japanese Gilliam), Karp, JP-1 (Japanese Karp type-1), JP-2 (Japanese Karp type-2), Kato, Kawasaki, Kuroki, Shimokoshi and others.<sup><a reveal-id="DSQ018C10" data-open="DSQ018C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> This typing system is widely used for epidemiological studies in Japan. Importantly, Ohashi <em>et al</em>.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> determined the 50% mouse lethal doses (MLD<sub>50</sub>) for representative strains of each subtype and found that strains from subtypes Kato, JG, Gilliam and Karp exhibit high-level virulence to mice, that from JP-2 intermediate-level virulence and those from JP-1, Kawasaki, Kuroki and Shimokoshi low-level virulence. The factors responsible for the strain-to-strain (or serotype-to-serotype) difference of mouse MLD<sub>50</sub> have not yet been identified.</p><p>The complete genome sequences of two OT strains have recently been determined. Cho <em>et al</em>. sequenced strain Boryong, which was isolated from a patient in Korea,<sup><a reveal-id="DSQ018C12" data-open="DSQ018C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and our group sequenced strain Ikeda, which was isolated in 1979 from a patient in Niigata prefecture, Japan.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Ikeda is a representative strain of subtype JG and thus highly virulent in mice.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Genomic analysis of the two OT strains revealed that extensive reductive genome evolution as well as explosive and comprehensive amplification of repetitive sequences have occurred in OT. In both strains, repetitive sequences occupy nearly half the genome. Through intensive analysis of the repetitive sequences identified in Ikeda, we categorized them into three types<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup>: (i) an integrative and conjugative element (ICE) named ‘OT amplified genetic element’ (OtAGE); (ii) transposable elements (TEs) and (iii) short repetitive sequences of unknown origins (short repeats). TEs included five types of insertion sequence (IS), four types of miniature inverted-repeat TEs (MITEs) and a Group II (GII) intron.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The results of our preliminary analysis suggested that extensive genome rearrangements mediated by the repetitive sequences have taken place between the two strains. However, because of the highly complex and repeat-rich feature of the OT genomes, details of the genomic differences between the two strains remain to be clarified.</p><p>In the present study, we performed fine comparison of the Ikeda and Boryong genomes to identify the common and variable genomic features among OT strains. In addition, using 11 genes that are conserved in OT and closely related <em>Rickettsia</em> species, we executed multilocus sequencing (MLS) analysis of 10 OT strains representing each TSA subtype to reveal the precise phylogenetic relationship of the strains and examined the distribution of strain-specific sequences identified in Ikeda or Boryong among the OT strains.</p>                            <h2 scrollto-destination=5034125 id="5034125" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5034126 id="5034126" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and DNA preparation</h3>
<p>Strains used in this study are listed in Table <a reveal-id="DSQ018TB1" data-open="DSQ018TB1" class="link link-reveal link-table xref-fig">1</a>. Nine strains other than Boryong represent each of the nine TSA subtypes, and their virulence in mice was analyzed in a previous study.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Bacterial cells were inoculated onto confluent monolayers of L929 cells grown in maintenance medium (MEM medium supplemented with 1% FBS, 0.075% NaHCO<sub>3</sub>, 0.03% glutamine, 100 U/ml penicillin, 100 μg/ml streptomycin and 0.25 μg/ml amphotericin B) and incubated at 37°C for 7 days in 5% CO<sub>2</sub>. Strain Matsuzawa was the exception in that it was incubated for 14 days because of its slow growth rate. After 7 or 14 days of cultivation, infected L929 cells were scraped from culture dishes, and the supernatant and L929 cells were separated by centrifugation at 200 g for 5 min. The cell pellet was resuspended in the maintenance medium and homogenized gently with a glass Dounce homogenizer (GPE Scientific Ltd). The cell homogenate was re-mixed with the supernatant and centrifuged at 200 g for 10 min to remove the host cell debris. Bacterial cells were collected by centrifugation at 20 000 g for 10 min and suspended in PBS for genomic DNA extraction with the DNeasy Blood and Tissue kit (Qiagen). To reconfirm the subtype of each strain, the TSA gene was amplified by polymerase chain reaction (PCR) and sequenced using primers listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. JP-1 and JP-2 strains were distinguished by restriction fragment length polymorphism (RFLP) of the PCR-amplified product of the TSA gene,<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and the TSA gene of strain 423H (subtype JP-2) had not been determined previously. Therefore, based on the RFLP pattern deduced from the 423H sequence, we confirmed that our strain belongs to the JP-2 subtype. The TSA gene sequence of strain Matsuzawa showed a significant number of one-base mismatches to that in the database (AF173043.1). Therefore, we resequenced the entire TSA gene of strain Matsuzawa and deposited it in the DDBJ/EMBL/GenBank databases (accession no. AB534164). We used this new sequence for our analysis. </p>                        <a id="5034128" scrollto-destination="5034128"></a>
<div content-id="DSQ018TB1" class="table-modal"><div class="table-wrap"><div id="DSQ018TB1" class="table-wrap-title" data-id="DSQ018TB1"><div class="label -label">Table 1</div><div class="caption"><p>OT strains used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain name </th><th>Subtype<sup>a</sup> </th><th>Geographic origin </th><th>Year of isolation </th><th>Ref. </th></tr></thead><tbody><tr><td>Ikeda </td><td>JG </td><td>Japan/Niigata pref. </td><td>1979 </td><td>28 </td></tr><tr><td>Boryong </td><td>not defined </td><td>Korea/Boryong pref. </td><td>1995 </td><td>29 </td></tr><tr><td>Gilliam </td><td>Gilliam </td><td>Myanmar </td><td>1943 </td><td>30 </td></tr><tr><td>Kato </td><td>Kato </td><td>Japan/Niigata pref. </td><td>1955 </td><td>31 </td></tr><tr><td>Karp </td><td>Karp </td><td>New Guinea </td><td>1943 </td><td>32 </td></tr><tr><td>Matsuzawa </td><td>JP-1 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>10 </td></tr><tr><td>423H </td><td>JP-2 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>11 </td></tr><tr><td>Kuroki </td><td>Kuroki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>33 </td></tr><tr><td>Kawasaki </td><td>Kawasaki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>34 </td></tr><tr><td>Shimokoshi </td><td>Shimokoshi </td><td>Japan/Niigata pref. </td><td>1980 </td><td>28 </td></tr></tbody></table></div><div content-id="DSQ018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain name </th><th>Subtype<sup>a</sup> </th><th>Geographic origin </th><th>Year of isolation </th><th>Ref. </th></tr></thead><tbody><tr><td>Ikeda </td><td>JG </td><td>Japan/Niigata pref. </td><td>1979 </td><td>28 </td></tr><tr><td>Boryong </td><td>not defined </td><td>Korea/Boryong pref. </td><td>1995 </td><td>29 </td></tr><tr><td>Gilliam </td><td>Gilliam </td><td>Myanmar </td><td>1943 </td><td>30 </td></tr><tr><td>Kato </td><td>Kato </td><td>Japan/Niigata pref. </td><td>1955 </td><td>31 </td></tr><tr><td>Karp </td><td>Karp </td><td>New Guinea </td><td>1943 </td><td>32 </td></tr><tr><td>Matsuzawa </td><td>JP-1 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>10 </td></tr><tr><td>423H </td><td>JP-2 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>11 </td></tr><tr><td>Kuroki </td><td>Kuroki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>33 </td></tr><tr><td>Kawasaki </td><td>Kawasaki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>34 </td></tr><tr><td>Shimokoshi </td><td>Shimokoshi </td><td>Japan/Niigata pref. </td><td>1980 </td><td>28 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Subtypes of each strain are derived from Ohashi <em>et al</em>.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034128" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ018TB1" class="table-wrap-title" data-id="DSQ018TB1"><div class="label -label">Table 1</div><div class="caption"><p>OT strains used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain name </th><th>Subtype<sup>a</sup> </th><th>Geographic origin </th><th>Year of isolation </th><th>Ref. </th></tr></thead><tbody><tr><td>Ikeda </td><td>JG </td><td>Japan/Niigata pref. </td><td>1979 </td><td>28 </td></tr><tr><td>Boryong </td><td>not defined </td><td>Korea/Boryong pref. </td><td>1995 </td><td>29 </td></tr><tr><td>Gilliam </td><td>Gilliam </td><td>Myanmar </td><td>1943 </td><td>30 </td></tr><tr><td>Kato </td><td>Kato </td><td>Japan/Niigata pref. </td><td>1955 </td><td>31 </td></tr><tr><td>Karp </td><td>Karp </td><td>New Guinea </td><td>1943 </td><td>32 </td></tr><tr><td>Matsuzawa </td><td>JP-1 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>10 </td></tr><tr><td>423H </td><td>JP-2 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>11 </td></tr><tr><td>Kuroki </td><td>Kuroki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>33 </td></tr><tr><td>Kawasaki </td><td>Kawasaki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>34 </td></tr><tr><td>Shimokoshi </td><td>Shimokoshi </td><td>Japan/Niigata pref. </td><td>1980 </td><td>28 </td></tr></tbody></table></div><div content-id="DSQ018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain name </th><th>Subtype<sup>a</sup> </th><th>Geographic origin </th><th>Year of isolation </th><th>Ref. </th></tr></thead><tbody><tr><td>Ikeda </td><td>JG </td><td>Japan/Niigata pref. </td><td>1979 </td><td>28 </td></tr><tr><td>Boryong </td><td>not defined </td><td>Korea/Boryong pref. </td><td>1995 </td><td>29 </td></tr><tr><td>Gilliam </td><td>Gilliam </td><td>Myanmar </td><td>1943 </td><td>30 </td></tr><tr><td>Kato </td><td>Kato </td><td>Japan/Niigata pref. </td><td>1955 </td><td>31 </td></tr><tr><td>Karp </td><td>Karp </td><td>New Guinea </td><td>1943 </td><td>32 </td></tr><tr><td>Matsuzawa </td><td>JP-1 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>10 </td></tr><tr><td>423H </td><td>JP-2 </td><td>Japan/Niigata pref. </td><td>1984 </td><td>11 </td></tr><tr><td>Kuroki </td><td>Kuroki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>33 </td></tr><tr><td>Kawasaki </td><td>Kawasaki </td><td>Japan/Miyazaki pref. </td><td>1981 </td><td>34 </td></tr><tr><td>Shimokoshi </td><td>Shimokoshi </td><td>Japan/Niigata pref. </td><td>1980 </td><td>28 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Subtypes of each strain are derived from Ohashi <em>et al</em>.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034128" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5034129 id="5034129" class="section-title" data-legacy-id=s2b>2.2. Genome comparison of strains Ikeda and Boryong</h3>
<p>An all-to-all BLASTN search<sup><a reveal-id="DSQ018C14" data-open="DSQ018C14" class="link link-ref link-reveal xref-bibr">14</a></sup> of the protein-coding sequences (CDSs) identified in the two strains was performed to examine the conservation of protein-coding genes in each strain. CDSs annotated as pseudogenes in each genome were excluded from this analysis. CDSs that were not detected, or those with different sizes, were further examined by CLUSTALW.<sup><a reveal-id="DSQ018C15" data-open="DSQ018C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>To identify repetitive sequences in the two strains with the same criteria, we constructed consensus sequences for each type of repetitive sequence that we previously identified in Ikeda, searched the two genomes for sequences homologous to each consensus using BLASTN and manually inspected each sequence. Thus, the positions of some repetitive sequences in Ikeda differ slightly from those described in the previous report.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The repetitive sequence-free genome sequences (RSFGSs) of each strain were obtained by removing the repetitive sequences from the entire genome sequence. Regions where the linearity of the genome sequence is conserved between the two strains were identified by Harplott analysis of the RSFGSs using the GenomeMatcher ver. 1.11 software.<sup><a reveal-id="DSQ018C16" data-open="DSQ018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> These regions (&gt;1 kb) were defined as linearity-conserved regions (LCRs) and were used to analyze genome rearrangement. Strain-specific sequences were identified by reciprocal BLASTN analysis of the RSFGSs. Sequences specific to each strain (&gt;1 kb) were defined as Ikeda- or Boryong-specific sequences (Ike_spe1 to Ike_spe19 and Bor_spe1 to Bor_spe19, respectively). To search additional Boryong-specific repetitive sequences, a self-to-self BLASTN search was performed for all Boryong-specific sequences (&gt;100 bp).</p>                            <h3 scrollto-destination=5034132 id="5034132" class="section-title" data-legacy-id=s2c>2.3. PCR scanning</h3>
<p>Distribution of the Ikeda- and Boryong-specific sequences among OT strains was examined by PCR scanning analysis<sup><a reveal-id="DSQ018C17" data-open="DSQ018C17" class="link link-ref link-reveal xref-bibr">17</a></sup> using the LA Long PCR kit (Takara) and 20 ng genomic DNA as template. Each strain-specific sequence (&gt;1 kb) was examined by two pairs of scanning primers (F1/R1 and F2/R2) except for Ike_spe6, which was examined using six primer pairs. When only one PCR product was obtained, the second PCR using different primer combinations (F1/R2 and F2/R1) was performed. The sequences of scanning primers are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=5034134 id="5034134" class="section-title" data-legacy-id=s2d>2.4. MLS analysis</h3>
<p>Internal segments of the following 11 housekeeping genes were amplified from each of the eight strains (Gilliam, Karp, Matsuzawa, 423H, Kato, Kawasaki, Kuroki and Shimokoshi) by PCR using the LA Long PCR kit and sequenced using the ABI3730 sequencer (Applied Biosystems): <em>atpD</em> (coding for ATP synthase beta-chain), <em>clpX</em> (ATP-dependent Clp protease ATP-binding subunit), <em>dnaJ</em> (DnaJ protein), <em>dnaK</em> (DnaK protein), <em>fabD</em> (malonyl CoA-acyl carrier protein transacylase), <em>gyrB</em> (DNA gyrase subunit B), <em>icd</em> (isocitrate dehydrogenase), <em>mdh</em> (malate dehydrogenase), <em>nrdA</em> (ribonucleoside-diphosphate reductase alpha-chain), <em>sucD</em> (succinyl-CoA synthetase alpha-chain) and <em>ubiD</em> (3-octaprenyl-4-hydroxybenzoate carboxy-lyase). Amplification and sequencing primers are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The nt sequences obtained from each strain were concatenated (5247 bp) and used for MLS analysis. Phylogenetic trees were constructed by the neighborhood-joining method under the Tamura–Nei model using MEGA4 in the Molecular Evolutionary Genetics Analysis software.<sup><a reveal-id="DSQ018C18" data-open="DSQ018C18" class="link link-ref link-reveal xref-bibr">18</a></sup> All sequences used for MLS analysis have been deposited in the DDBJ/EMBL/GenBank database with the accession numbers AB537240–AB537327.</p>                            <h2 scrollto-destination=5034136 id="5034136" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5034137 id="5034137" class="section-title" data-legacy-id=s3a>3.1. Comparison of general genomic features between strains Ikeda and Boryong</h3>
<p>Both strains have a single circular chromosome and possess no plasmid. The chromosomes are very similar in size (2 008 987 bp in Ikeda and 2 127 051 bp in Boryong) with almost identical average G + C contents (30.5% in both strains). The numbers of rRNA and tRNA genes are identical. The numbers of protein-coding genes and pseudogenes, the coding content (%) and the repeat content (%) were unable to be directly compared as the annotation criteria were different between the two genomes.</p><p>In conventional GC skew analysis, no clear skew in leading and lagging strands was observed in either genome. Therefore, we reassigned the origin of replication (<em>oriC</em>) using Ori-Finder.<sup><a reveal-id="DSQ018C19" data-open="DSQ018C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The <em>oriC</em> region was found to lie between the <em>rho</em> and <em>hemE</em> genes, which are located from nt 539552 to nt 541039 in Ikeda and nt 996058 to nt 997543 in Boryong.</p>                            <h3 scrollto-destination=5034141 id="5034141" class="section-title" data-legacy-id=s3b>3.2. Comparison of repetitive sequences between strains Ikeda and Boryong</h3>
<p>Because the large number of repetitive sequences in both strains makes comparison of their genome sequences difficult, we first identified all repetitive sequences for the two strains under the same criteria, established on the basis of the results of our previous analysis.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> We found that Boryong contains all types of repetitive sequence identified in Ikeda (Table <a reveal-id="DSQ018TB2" data-open="DSQ018TB2" class="link link-reveal link-table xref-fig">2</a>). No additional repetitive sequence was found in Boryong. Total lengths of repetitive sequences are 851.6 kb (42.4% of the total genome) in Ikeda and 911.9 kb (42.9%) in Boryong. In both strains, the repetitive sequences are scattered throughout the genome (Fig. <a reveal-id="DSQ018F1" data-open="DSQ018F1" class="link link-reveal link-table xref-fig">1</a>A). OtAGEs and TEs, the main components of the repetitive sequences, often form small clusters in which one element is inserted into another. Because of such complexity, we did not determine the exact copy number for each type of repetitive sequence in Boryong. </p>                        <a id="5034144" scrollto-destination="5034144"></a>
<div content-id="DSQ018TB2" class="table-modal"><div class="table-wrap"><div id="DSQ018TB2" class="table-wrap-title" data-id="DSQ018TB2"><div class="label -label">Table 2</div><div class="caption"><p>Repeated sequences of OT strains IKEDA and Boryong.<sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Strain Ikeda </th><th>Strain Boryong </th></tr></thead><tbody><tr><td>OtAGEs </td><td>578 383 (28.79) </td><td>597 432 (28.09) </td></tr><tr><td colspan="3">Transposable elements<sup>b</sup> </td></tr><tr><td> IS<em>Ots1</em> </td><td>47 090 (2.34) </td><td>91 682 (4.31) </td></tr><tr><td> mIS<em>Ot1</em> </td><td>8415 (0.42) </td><td>15 468 (0.73) </td></tr><tr><td> IS<em>Ots2</em> </td><td>9621 (0.48) </td><td>9848 (0.46) </td></tr><tr><td> mIS<em>Ot2a</em> </td><td>4455 (0.22) </td><td>5921 (0.28) </td></tr><tr><td> mIS<em>Ot2b</em> </td><td>6043 (0.3) </td><td>4364 (0.21) </td></tr><tr><td> IS<em>Ots3</em> </td><td>25 509 (1.27) </td><td>3944 (0.19) </td></tr><tr><td> IS<em>Ots4</em> </td><td>43 077 (2.14) </td><td>46 343 (2.18) </td></tr><tr><td> mIS<em>Ot4</em> </td><td>11 442 (0.57) </td><td>17 842 (0.84) </td></tr><tr><td> IS<em>Ots5</em> </td><td>45 824 (2.28) </td><td>20 328 (0.96) </td></tr><tr><td> GII<em>Ot1</em> </td><td>55 506 (2.76) </td><td>79 494 (3.74) </td></tr><tr><td> Total </td><td>256 982 (12.79) </td><td>295 234 (13.88) </td></tr><tr><td colspan="3">Short Repeats </td></tr><tr><td> SR<em>Ot1</em> </td><td>10 613 (0.53) </td><td>13 890 (0.65) </td></tr><tr><td> SR<em>Ot2</em> </td><td>1006 (0.05) </td><td>1046 (0.05) </td></tr><tr><td> SR<em>Ot3</em> </td><td>1432 (0.07) </td><td>1429 (0.07) </td></tr><tr><td> SR<em>Ot4</em> </td><td>1246 (0.06) </td><td>1080 (0.05) </td></tr><tr><td> SR<em>Ot5</em> </td><td>693 (0.03) </td><td>495 (0.02) </td></tr><tr><td> SR<em>Ot6</em> </td><td>875 (0.04) </td><td>837 (0.04) </td></tr><tr><td> SR<em>Ot7</em> </td><td>366 (0.02) </td><td>409 (0.02) </td></tr><tr><td> Total </td><td>16 231 (0.81) </td><td>19 186 (0.9) </td></tr><tr><td>Total </td><td>851 596 (42.39) </td><td>911 852 (42.87) </td></tr></tbody></table></div><div content-id="DSQ018TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Strain Ikeda </th><th>Strain Boryong </th></tr></thead><tbody><tr><td>OtAGEs </td><td>578 383 (28.79) </td><td>597 432 (28.09) </td></tr><tr><td colspan="3">Transposable elements<sup>b</sup> </td></tr><tr><td> IS<em>Ots1</em> </td><td>47 090 (2.34) </td><td>91 682 (4.31) </td></tr><tr><td> mIS<em>Ot1</em> </td><td>8415 (0.42) </td><td>15 468 (0.73) </td></tr><tr><td> IS<em>Ots2</em> </td><td>9621 (0.48) </td><td>9848 (0.46) </td></tr><tr><td> mIS<em>Ot2a</em> </td><td>4455 (0.22) </td><td>5921 (0.28) </td></tr><tr><td> mIS<em>Ot2b</em> </td><td>6043 (0.3) </td><td>4364 (0.21) </td></tr><tr><td> IS<em>Ots3</em> </td><td>25 509 (1.27) </td><td>3944 (0.19) </td></tr><tr><td> IS<em>Ots4</em> </td><td>43 077 (2.14) </td><td>46 343 (2.18) </td></tr><tr><td> mIS<em>Ot4</em> </td><td>11 442 (0.57) </td><td>17 842 (0.84) </td></tr><tr><td> IS<em>Ots5</em> </td><td>45 824 (2.28) </td><td>20 328 (0.96) </td></tr><tr><td> GII<em>Ot1</em> </td><td>55 506 (2.76) </td><td>79 494 (3.74) </td></tr><tr><td> Total </td><td>256 982 (12.79) </td><td>295 234 (13.88) </td></tr><tr><td colspan="3">Short Repeats </td></tr><tr><td> SR<em>Ot1</em> </td><td>10 613 (0.53) </td><td>13 890 (0.65) </td></tr><tr><td> SR<em>Ot2</em> </td><td>1006 (0.05) </td><td>1046 (0.05) </td></tr><tr><td> SR<em>Ot3</em> </td><td>1432 (0.07) </td><td>1429 (0.07) </td></tr><tr><td> SR<em>Ot4</em> </td><td>1246 (0.06) </td><td>1080 (0.05) </td></tr><tr><td> SR<em>Ot5</em> </td><td>693 (0.03) </td><td>495 (0.02) </td></tr><tr><td> SR<em>Ot6</em> </td><td>875 (0.04) </td><td>837 (0.04) </td></tr><tr><td> SR<em>Ot7</em> </td><td>366 (0.02) </td><td>409 (0.02) </td></tr><tr><td> Total </td><td>16 231 (0.81) </td><td>19 186 (0.9) </td></tr><tr><td>Total </td><td>851 596 (42.39) </td><td>911 852 (42.87) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Values given in parentheses are the ratio to the total genome length (%).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Names of the IS elements have been changed from ISOt1 ∼ 5 to ISOts1 ∼ 5 according to the suggestions from LMGE (the organizer of ISFinder: <a class="link link-uri" href="http://www-is.biotoul.fr/">http://www-is.biotoul.fr/</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034144" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ018TB2" class="table-wrap-title" data-id="DSQ018TB2"><div class="label -label">Table 2</div><div class="caption"><p>Repeated sequences of OT strains IKEDA and Boryong.<sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Strain Ikeda </th><th>Strain Boryong </th></tr></thead><tbody><tr><td>OtAGEs </td><td>578 383 (28.79) </td><td>597 432 (28.09) </td></tr><tr><td colspan="3">Transposable elements<sup>b</sup> </td></tr><tr><td> IS<em>Ots1</em> </td><td>47 090 (2.34) </td><td>91 682 (4.31) </td></tr><tr><td> mIS<em>Ot1</em> </td><td>8415 (0.42) </td><td>15 468 (0.73) </td></tr><tr><td> IS<em>Ots2</em> </td><td>9621 (0.48) </td><td>9848 (0.46) </td></tr><tr><td> mIS<em>Ot2a</em> </td><td>4455 (0.22) </td><td>5921 (0.28) </td></tr><tr><td> mIS<em>Ot2b</em> </td><td>6043 (0.3) </td><td>4364 (0.21) </td></tr><tr><td> IS<em>Ots3</em> </td><td>25 509 (1.27) </td><td>3944 (0.19) </td></tr><tr><td> IS<em>Ots4</em> </td><td>43 077 (2.14) </td><td>46 343 (2.18) </td></tr><tr><td> mIS<em>Ot4</em> </td><td>11 442 (0.57) </td><td>17 842 (0.84) </td></tr><tr><td> IS<em>Ots5</em> </td><td>45 824 (2.28) </td><td>20 328 (0.96) </td></tr><tr><td> GII<em>Ot1</em> </td><td>55 506 (2.76) </td><td>79 494 (3.74) </td></tr><tr><td> Total </td><td>256 982 (12.79) </td><td>295 234 (13.88) </td></tr><tr><td colspan="3">Short Repeats </td></tr><tr><td> SR<em>Ot1</em> </td><td>10 613 (0.53) </td><td>13 890 (0.65) </td></tr><tr><td> SR<em>Ot2</em> </td><td>1006 (0.05) </td><td>1046 (0.05) </td></tr><tr><td> SR<em>Ot3</em> </td><td>1432 (0.07) </td><td>1429 (0.07) </td></tr><tr><td> SR<em>Ot4</em> </td><td>1246 (0.06) </td><td>1080 (0.05) </td></tr><tr><td> SR<em>Ot5</em> </td><td>693 (0.03) </td><td>495 (0.02) </td></tr><tr><td> SR<em>Ot6</em> </td><td>875 (0.04) </td><td>837 (0.04) </td></tr><tr><td> SR<em>Ot7</em> </td><td>366 (0.02) </td><td>409 (0.02) </td></tr><tr><td> Total </td><td>16 231 (0.81) </td><td>19 186 (0.9) </td></tr><tr><td>Total </td><td>851 596 (42.39) </td><td>911 852 (42.87) </td></tr></tbody></table></div><div content-id="DSQ018TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Strain Ikeda </th><th>Strain Boryong </th></tr></thead><tbody><tr><td>OtAGEs </td><td>578 383 (28.79) </td><td>597 432 (28.09) </td></tr><tr><td colspan="3">Transposable elements<sup>b</sup> </td></tr><tr><td> IS<em>Ots1</em> </td><td>47 090 (2.34) </td><td>91 682 (4.31) </td></tr><tr><td> mIS<em>Ot1</em> </td><td>8415 (0.42) </td><td>15 468 (0.73) </td></tr><tr><td> IS<em>Ots2</em> </td><td>9621 (0.48) </td><td>9848 (0.46) </td></tr><tr><td> mIS<em>Ot2a</em> </td><td>4455 (0.22) </td><td>5921 (0.28) </td></tr><tr><td> mIS<em>Ot2b</em> </td><td>6043 (0.3) </td><td>4364 (0.21) </td></tr><tr><td> IS<em>Ots3</em> </td><td>25 509 (1.27) </td><td>3944 (0.19) </td></tr><tr><td> IS<em>Ots4</em> </td><td>43 077 (2.14) </td><td>46 343 (2.18) </td></tr><tr><td> mIS<em>Ot4</em> </td><td>11 442 (0.57) </td><td>17 842 (0.84) </td></tr><tr><td> IS<em>Ots5</em> </td><td>45 824 (2.28) </td><td>20 328 (0.96) </td></tr><tr><td> GII<em>Ot1</em> </td><td>55 506 (2.76) </td><td>79 494 (3.74) </td></tr><tr><td> Total </td><td>256 982 (12.79) </td><td>295 234 (13.88) </td></tr><tr><td colspan="3">Short Repeats </td></tr><tr><td> SR<em>Ot1</em> </td><td>10 613 (0.53) </td><td>13 890 (0.65) </td></tr><tr><td> SR<em>Ot2</em> </td><td>1006 (0.05) </td><td>1046 (0.05) </td></tr><tr><td> SR<em>Ot3</em> </td><td>1432 (0.07) </td><td>1429 (0.07) </td></tr><tr><td> SR<em>Ot4</em> </td><td>1246 (0.06) </td><td>1080 (0.05) </td></tr><tr><td> SR<em>Ot5</em> </td><td>693 (0.03) </td><td>495 (0.02) </td></tr><tr><td> SR<em>Ot6</em> </td><td>875 (0.04) </td><td>837 (0.04) </td></tr><tr><td> SR<em>Ot7</em> </td><td>366 (0.02) </td><td>409 (0.02) </td></tr><tr><td> Total </td><td>16 231 (0.81) </td><td>19 186 (0.9) </td></tr><tr><td>Total </td><td>851 596 (42.39) </td><td>911 852 (42.87) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Values given in parentheses are the ratio to the total genome length (%).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Names of the IS elements have been changed from ISOt1 ∼ 5 to ISOts1 ∼ 5 according to the suggestions from LMGE (the organizer of ISFinder: <a class="link link-uri" href="http://www-is.biotoul.fr/">http://www-is.biotoul.fr/</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034144" target="_blank">View Large</a></div></div>                        <a id="5034146" scrollto-destination="5034146"></a>
<div id="DSQ018F1" data-id="DSQ018F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01801.jpeg?Expires=2147483647&Signature=BaQRK-WTu9WQPUDi-KBJebWv4dI9PIaqm-h1DsjpFgZzLorPZ4dxjS6nCJA3HTJxFNigdPIBLXn~mpwRba8DUXTHX5c-xcaSUPlnbCwzGC8xW4wZUBiMep~nNQNF0gdW9OD-XqfEaKWjsaVGiA~U9afJ2vZ5EAc58f6gLQfoVVFr~Izbc66Ccg3ARp4xnVyWjKmZJ8K79ve-cPGVK2HlHSqbnmuVVEmbCQDOq132I7VDTTW1NGH3CK1sZ3rky4yElGCTrsggSMrbvdZihP7XstDKkpWGf~pCtjq6CY-tJgSHQNC431JXQrJdmpMGfOulMLDShx2LRSMEoIRu-i9lCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Extensive shuffling of OT genomes induced by explosively amplified repetitive sequences. (a) Dot plot representation of DNA sequence homology between Ikeda and Boryong chromosomes. Explosively amplified repetitive sequences are scattered throughout the genomes in both strains, and it is therefore difficult to identify regions with colinearity between the two genomes. (b) Dot plot representation of DNA sequence homology for the repetitive sequence-free genomic sequences (RSFGSs) of the two strains. The RSFGSs were constructed by removing all repetitive sequences from each genome. By comparing the RSFGSs, extensive rearrangements of the core genome of OT can be clearly visualized. Each dot represents a high-scoring segment pair of E-value &lt;1E−200, which was identified by BLASTN. The GenomeMatcher ver. 1.11 software16 was used for BLASTN analysis and visualization of the results." path-from-xml="dsq01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034146/dsq01801.jpeg" path-from-xml="dsq01801.jpeg" target="_blank">View large</a><a section="5034146" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01801.jpeg?Expires=2147483647&Signature=1Kgl29R8PfuzJleV9OvA3Q2UswOYlILl2FxsrzMWRMGY0g2CKz6O563yBswyoX75myT2TMM8enHlm1FvoxsHStr~J0weORPBGPYi36JJ~kUFjLG3OCuwV~3AZeFPF9pFbBWuTzWfnQYIL6ng2gtR82v~JcMtN0we-DNeH2gNSGbcDIPkdKC0aBLtA1xYbNAuylUc76zowWdcBgvX2eMD2XdPFsT9LoCVtLJLKnnLgiRWWN3kXUFPiFQ6sGC0x33KPdjMezpMSc0x3wBtDNrxfqZdb0f08La0v-p9mSXIEyz1j0yo~r4fHpkkeO49WNp90KMJg53Nt~YEDxbo22jGSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034146&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Extensive shuffling of OT genomes induced by explosively amplified repetitive sequences. (a) Dot plot representation of DNA sequence homology between Ikeda and Boryong chromosomes. Explosively amplified repetitive sequences are scattered throughout the genomes in both strains, and it is therefore difficult to identify regions with colinearity between the two genomes. (b) Dot plot representation of DNA sequence homology for the repetitive sequence-free genomic sequences (RSFGSs) of the two strains. The RSFGSs were constructed by removing all repetitive sequences from each genome. By comparing the RSFGSs, extensive rearrangements of the core genome of OT can be clearly visualized. Each dot represents a high-scoring segment pair of <em>E</em>-value &lt;1E−200, which was identified by BLASTN. The GenomeMatcher ver. 1.11 software<sup><a reveal-id="DSQ018C16" data-open="DSQ018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> was used for BLASTN analysis and visualization of the results.</p></div></div><div content-id="DSQ018F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01801.jpeg?Expires=2147483647&Signature=BaQRK-WTu9WQPUDi-KBJebWv4dI9PIaqm-h1DsjpFgZzLorPZ4dxjS6nCJA3HTJxFNigdPIBLXn~mpwRba8DUXTHX5c-xcaSUPlnbCwzGC8xW4wZUBiMep~nNQNF0gdW9OD-XqfEaKWjsaVGiA~U9afJ2vZ5EAc58f6gLQfoVVFr~Izbc66Ccg3ARp4xnVyWjKmZJ8K79ve-cPGVK2HlHSqbnmuVVEmbCQDOq132I7VDTTW1NGH3CK1sZ3rky4yElGCTrsggSMrbvdZihP7XstDKkpWGf~pCtjq6CY-tJgSHQNC431JXQrJdmpMGfOulMLDShx2LRSMEoIRu-i9lCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Extensive shuffling of OT genomes induced by explosively amplified repetitive sequences. (a) Dot plot representation of DNA sequence homology between Ikeda and Boryong chromosomes. Explosively amplified repetitive sequences are scattered throughout the genomes in both strains, and it is therefore difficult to identify regions with colinearity between the two genomes. (b) Dot plot representation of DNA sequence homology for the repetitive sequence-free genomic sequences (RSFGSs) of the two strains. The RSFGSs were constructed by removing all repetitive sequences from each genome. By comparing the RSFGSs, extensive rearrangements of the core genome of OT can be clearly visualized. Each dot represents a high-scoring segment pair of E-value &lt;1E−200, which was identified by BLASTN. The GenomeMatcher ver. 1.11 software16 was used for BLASTN analysis and visualization of the results." path-from-xml="dsq01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034146/dsq01801.jpeg" path-from-xml="dsq01801.jpeg" target="_blank">View large</a><a section="5034146" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01801.jpeg?Expires=2147483647&Signature=1Kgl29R8PfuzJleV9OvA3Q2UswOYlILl2FxsrzMWRMGY0g2CKz6O563yBswyoX75myT2TMM8enHlm1FvoxsHStr~J0weORPBGPYi36JJ~kUFjLG3OCuwV~3AZeFPF9pFbBWuTzWfnQYIL6ng2gtR82v~JcMtN0we-DNeH2gNSGbcDIPkdKC0aBLtA1xYbNAuylUc76zowWdcBgvX2eMD2XdPFsT9LoCVtLJLKnnLgiRWWN3kXUFPiFQ6sGC0x33KPdjMezpMSc0x3wBtDNrxfqZdb0f08La0v-p9mSXIEyz1j0yo~r4fHpkkeO49WNp90KMJg53Nt~YEDxbo22jGSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034146&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Extensive shuffling of OT genomes induced by explosively amplified repetitive sequences. (a) Dot plot representation of DNA sequence homology between Ikeda and Boryong chromosomes. Explosively amplified repetitive sequences are scattered throughout the genomes in both strains, and it is therefore difficult to identify regions with colinearity between the two genomes. (b) Dot plot representation of DNA sequence homology for the repetitive sequence-free genomic sequences (RSFGSs) of the two strains. The RSFGSs were constructed by removing all repetitive sequences from each genome. By comparing the RSFGSs, extensive rearrangements of the core genome of OT can be clearly visualized. Each dot represents a high-scoring segment pair of <em>E</em>-value &lt;1E−200, which was identified by BLASTN. The GenomeMatcher ver. 1.11 software<sup><a reveal-id="DSQ018C16" data-open="DSQ018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> was used for BLASTN analysis and visualization of the results.</p></div></div><p>As in Ikeda, OtAGEs have been extensively amplified in Boryong (Table <a reveal-id="DSQ018TB2" data-open="DSQ018TB2" class="link link-reveal link-table xref-fig">2</a>). The total length of OtAGE is 578.4 kb in Ikeda (28.8% of the total genome) and 597.4 kb in Boryong (28.1%). In Ikeda, various degrees of decay were observed in all OtAGEs, and a large number of fragmented OtAGEs have been scattered throughout the genome. We could, however, reconstitute a possible genetic organization of original OtAGE by comparing fragmented elements.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Although extensive decay has also occurred in the OtAGEs of Boryong, we found one copy of OtAGE very similar to the reconstituted OtAGE of strain Ikeda (Fig. <a reveal-id="DSQ018F2" data-open="DSQ018F2" class="link link-reveal link-table xref-fig">2</a>).<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, duplication of the attachment sequence, which must have been generated upon integration, was not detected on the element, suggesting that some deletion has occurred at either or both ends of the element. Several IS insertions and genomic deletions were also observed in the element. Thus, this element of Boryong appears to be no more functional as an ICE. </p>                        <a id="5034151" scrollto-destination="5034151"></a>
<div id="DSQ018F2" data-id="DSQ018F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01802.jpeg?Expires=2147483647&Signature=IOwAfDKyJbQmtbKJViwN1XrVnU8JE7D4qYCZ~FTAM52g3hRw38CBNU6IEhgos2tjkC2wWTUdnlPf7tf99y3qyyjs6jEcEZGy1XHzBkuBA-dRznlGl9QErYAJUx7OedIVojCGoVL2drA-hNAlx1VcRr7lHbvZQLFUZrTgSvLCbJM2~k68lJ2dfZhxRfF8-KpNL3Py9s89~sCpjivVQuGI~kdCudo-4-yYQD0TSO~WM09HaLlBB3~Jmx4YdyrrHBykXhZIjgS2kSJ-67YRgCKBskG-fx13AMjU3xufAh-tC8nwY2ESMO0L0qjs3Vfjubb5BiRvHkJWm9T8YXaXaz4v6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic organizations of the reconstituted OtAGE of strain Ikeda and the largest OtAGE found in strain Boryong. Both strains contain a large number of OtAGEs, but extensive decay has occurred in all OtAGE copies. A possible original structure of Ikeda OtAGE was reconstituted by comparing larger fragments in our previous report.13 The Ikeda OtAGE shown in this figure was constructed by combining OtAGE_178 and OtAGE_40. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white and OT-specific hypothetical genes by gray. All genes in the OtAGEs are drawn to the scale. Numbers above the reconstituted Ikeda OtAGE indicate family numbers of OtRG families. Numbers below the Boryong OtAGE indicate CDS numbers in Boryong. Homologous regions indicated by light green shading were identified by the BLASTN program (E-value threshold: ≤1E−10). Note that the Boryong OtAGE contains several deletions and IS insertions, suggesting that this largest OtAGE copy is also defective." path-from-xml="dsq01802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034151/dsq01802.jpeg" path-from-xml="dsq01802.jpeg" target="_blank">View large</a><a section="5034151" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01802.jpeg?Expires=2147483647&Signature=pCDg09ZKmg0Fxiv44mCXmmBxQjRhWSRnl6TdlyB8CJRbClassUAlT-fdzG6FvcOEn3PIiAZJuWrcw~kexCAM8Go9SKGRByZSwlH9HNDfiAEvyE1b7nPSycMhZFLwmeCIwyC8GvfGis49iZoFTGrslCZbTCvzzSfZVa9xjJjFF6Mu7WUE3M1n1s2UBmAOTeCAS~25iFob8Nq916rU0WkDdgKQ7J9fa5CWGQ-PWgCbwxLu7UsODSqc4JzqNRSjrb2T8e8v6enR6RVLA5ZXhf0YF9hxSX6ChF8jX1GJsX3i-XTZOYLDpvRtKqWnAa-ufqMMsiDKwrp42lWSv4~QYK7s5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034151&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic organizations of the reconstituted OtAGE of strain Ikeda and the largest OtAGE found in strain Boryong. Both strains contain a large number of OtAGEs, but extensive decay has occurred in all OtAGE copies. A possible original structure of Ikeda OtAGE was reconstituted by comparing larger fragments in our previous report.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The Ikeda OtAGE shown in this figure was constructed by combining OtAGE_178 and OtAGE_40. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white and OT-specific hypothetical genes by gray. All genes in the OtAGEs are drawn to the scale. Numbers above the reconstituted Ikeda OtAGE indicate family numbers of OtRG families. Numbers below the Boryong OtAGE indicate CDS numbers in Boryong. Homologous regions indicated by light green shading were identified by the BLASTN program (<em>E</em>-value threshold: ≤1E−10). Note that the Boryong OtAGE contains several deletions and IS insertions, suggesting that this largest OtAGE copy is also defective.</p></div></div><div content-id="DSQ018F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01802.jpeg?Expires=2147483647&Signature=IOwAfDKyJbQmtbKJViwN1XrVnU8JE7D4qYCZ~FTAM52g3hRw38CBNU6IEhgos2tjkC2wWTUdnlPf7tf99y3qyyjs6jEcEZGy1XHzBkuBA-dRznlGl9QErYAJUx7OedIVojCGoVL2drA-hNAlx1VcRr7lHbvZQLFUZrTgSvLCbJM2~k68lJ2dfZhxRfF8-KpNL3Py9s89~sCpjivVQuGI~kdCudo-4-yYQD0TSO~WM09HaLlBB3~Jmx4YdyrrHBykXhZIjgS2kSJ-67YRgCKBskG-fx13AMjU3xufAh-tC8nwY2ESMO0L0qjs3Vfjubb5BiRvHkJWm9T8YXaXaz4v6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic organizations of the reconstituted OtAGE of strain Ikeda and the largest OtAGE found in strain Boryong. Both strains contain a large number of OtAGEs, but extensive decay has occurred in all OtAGE copies. A possible original structure of Ikeda OtAGE was reconstituted by comparing larger fragments in our previous report.13 The Ikeda OtAGE shown in this figure was constructed by combining OtAGE_178 and OtAGE_40. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white and OT-specific hypothetical genes by gray. All genes in the OtAGEs are drawn to the scale. Numbers above the reconstituted Ikeda OtAGE indicate family numbers of OtRG families. Numbers below the Boryong OtAGE indicate CDS numbers in Boryong. Homologous regions indicated by light green shading were identified by the BLASTN program (E-value threshold: ≤1E−10). Note that the Boryong OtAGE contains several deletions and IS insertions, suggesting that this largest OtAGE copy is also defective." path-from-xml="dsq01802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034151/dsq01802.jpeg" path-from-xml="dsq01802.jpeg" target="_blank">View large</a><a section="5034151" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01802.jpeg?Expires=2147483647&Signature=pCDg09ZKmg0Fxiv44mCXmmBxQjRhWSRnl6TdlyB8CJRbClassUAlT-fdzG6FvcOEn3PIiAZJuWrcw~kexCAM8Go9SKGRByZSwlH9HNDfiAEvyE1b7nPSycMhZFLwmeCIwyC8GvfGis49iZoFTGrslCZbTCvzzSfZVa9xjJjFF6Mu7WUE3M1n1s2UBmAOTeCAS~25iFob8Nq916rU0WkDdgKQ7J9fa5CWGQ-PWgCbwxLu7UsODSqc4JzqNRSjrb2T8e8v6enR6RVLA5ZXhf0YF9hxSX6ChF8jX1GJsX3i-XTZOYLDpvRtKqWnAa-ufqMMsiDKwrp42lWSv4~QYK7s5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034151&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic organizations of the reconstituted OtAGE of strain Ikeda and the largest OtAGE found in strain Boryong. Both strains contain a large number of OtAGEs, but extensive decay has occurred in all OtAGE copies. A possible original structure of Ikeda OtAGE was reconstituted by comparing larger fragments in our previous report.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The Ikeda OtAGE shown in this figure was constructed by combining OtAGE_178 and OtAGE_40. Genes for conjugal transfer are depicted by green boxes, those for integrase by red, those for transposase by yellow, those for ATPase domain-containing proteins by blue, conserved hypothetical genes by white and OT-specific hypothetical genes by gray. All genes in the OtAGEs are drawn to the scale. Numbers above the reconstituted Ikeda OtAGE indicate family numbers of OtRG families. Numbers below the Boryong OtAGE indicate CDS numbers in Boryong. Homologous regions indicated by light green shading were identified by the BLASTN program (<em>E</em>-value threshold: ≤1E−10). Note that the Boryong OtAGE contains several deletions and IS insertions, suggesting that this largest OtAGE copy is also defective.</p></div></div><p>Although Ikeda and Boryong share the same sets of TEs and the total lengths of the TEs are nearly equivalent (257.0 kb and 12.8% of the total genome in Ikeda and 295.2 kb and 13.9% in Boryong), amounts of each type of TE differ significantly between the strains (Table <a reveal-id="DSQ018TB2" data-open="DSQ018TB2" class="link link-reveal link-table xref-fig">2</a>). This result suggests that intensive amplification and decay of these elements have occurred in each strain after their divergence.</p><p>All types of short repeats identified in Ikeda are also present in Boryong. However, in contrast to the TEs, total lengths of the short repeats and amounts of each type of short repeat are very similar in the two strains (Table <a reveal-id="DSQ018TB2" data-open="DSQ018TB2" class="link link-reveal link-table xref-fig">2</a>).</p>                            <h3 scrollto-destination=5034159 id="5034159" class="section-title" data-legacy-id=s3c>3.3 Extensive repetitive sequence-mediated genome shuffling</h3>
<p>By removing all repetitive sequences from each genome, we constructed RSFGSs of the two strains. The lengths of the RSFGSs of Ikeda and Boryong are 1156.8 and 1214.8 kb, respectively. The result of Harplott analysis of the RSFGSs clearly indicated that an extremely high degree of genome shuffling has occurred between the OT strains (Fig. <a reveal-id="DSQ018F1" data-open="DSQ018F1" class="link link-reveal link-table xref-fig">1</a>). This genome shuffling was obviously induced by extensively amplified repetitive sequences. By comparing the RSFGSs of the two strains, we identified 66 LCRs of &gt;1 kb in size, where the linearity of the genome sequence is conserved between the two strains (positions of all LCRs in each genome are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The average length of these LCRs is 16.0 kb, with the longest being ∼46 kb.</p>                            <h3 scrollto-destination=5034163 id="5034163" class="section-title" data-legacy-id=s3d>3.4 Strain-specific sequences</h3>
<p>In the RSFGSs, we identified 19 Ikeda-specific (Ike_spe1 to Ike_spe19) and 19 Boryong-specific (Bor_spe1 to Bor_spe19) sequences of &gt;1 kb in length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> and Fig. <a reveal-id="DSQ018F3" data-open="DSQ018F3" class="link link-reveal link-table xref-fig">3</a>). These sequences were not found in other bacteria of the family Rickettsiaceae. Several sequences have been duplicated in each genome. Their duplications appear to have been induced by repetitive sequences that flank these sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Total lengths of the Ikeda- and Boryong-specific sequences are 52.4 (average 2.8) and 40.8 (average 2.1) kb, respectively. Among these sequences, Ike_spe6 is particularly interesting. At 19.2 kb, this is the longest of the strain-specific sequences, and it shows a higher G + C content compared with other parts of the Ikeda genome and is flanked by IS<em>Ots4</em> and mIS<em>Ots4</em>, forming a composite transposon-like structure with 95-bp-long terminal inverted repeats (Fig. <a reveal-id="DSQ018F4" data-open="DSQ018F4" class="link link-reveal link-table xref-fig">4</a>). A direct repeat of 5 bp, which was most likely created upon insertion (target sequence duplication), was also identified. Many of the 26 genes encoded in Ike_spe6 have the closest homologs in <em>Legionella</em> spp. and <em>Parachlamydia</em> spp. A few genes have homologs in <em>Rickettsia</em> species, but their sequence similarity to the Ike_spe6 genes was significantly lower than to homologs in <em>Legionella</em> and <em>Parachlamydia</em>, suggesting that Ikeda may have acquired this sequence by horizontal gene transfer from related bacterial species. Intriguingly, both <em>Legionella</em> and <em>Parachlamydia</em> are intracellular microorganisms, and their main niches in natural environments are thought to be amoebas. There are some articles suggesting the relationship between the order Rickettsiales and amoebae.<sup><a reveal-id="DSQ018C20" data-open="DSQ018C20" class="link link-ref link-reveal xref-bibr">20</a>–<a reveal-id="DSQ018C22" data-open="DSQ018C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The presence of Ike_spe6 may provide the new evidence showing that an amoeba may be involved in the life cycle of OT as well. </p>                        <a id="5034167" scrollto-destination="5034167"></a>
<div id="DSQ018F3" data-id="DSQ018F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01803.jpeg?Expires=2147483647&Signature=Oz62PobzswecVV3iQXllQ5yJWKsAt3GNQbiZdtJK1RSFTR7kU6WQWrLSv37iRxN4eAlThKt~lG39SG51bmurcTw-z1LcTUTT6RIclz7076YipdM~Os74nQIVN9D49IW0Et3lYv-n2zArQqK9hrJZZgYPjW4dgOZb0RziRcu-VpXNzpgdyCaP8WZ2I2YVrr8JhYCwknVHGI0nZT-oou4Mt0Zga~~~5HN4dqyzttbZEn2Bv8TTRVwV08yhF2-xj5fGiVuhgeeR8-RnMkhr0lQHjhkt8u~edBn9WGMCayllUJt2aMQATVCvGNF1CE9n~XgCa1Lx7rpLMGu4WSDXinrjXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of Ikeda- and Boryong-specific sequences among other OT strains. The presence of each strain-specific region (&gt;1 kb) was examined by PCR scanning using two pairs of PCR primers, except for Ike_spe6, which was examined using six pairs of primers. Regions present in each strain are indicated by black cells with white plus signs. Regions where only one primer pair yielded a PCR product are indicated by gray cells with black plus signs. Regions yielding PCR products with different sizes compared with those of the reference strain (Ikeda or Boryong) are also indicated by gray cells. In Ike_spe6 of strain Kato, five primer pairs yielded PCR products. Phylogenetic relationships of the OT strains examined are also indicated (Fig. 5)." path-from-xml="dsq01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034167/dsq01803.jpeg" path-from-xml="dsq01803.jpeg" target="_blank">View large</a><a section="5034167" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01803.jpeg?Expires=2147483647&Signature=N-uPn07lNdghTcB4QVrDoA1O35nnb5Oe7zQkhdok7qkPP0ySH7KYPk3E8cWHsNEkEtcM0lcTsMgv5XH-HPDP5ejOuRz73Q7Awa9C1oWwebNxw~DeIPbf1aq-ORvMjnF0zGnFc9Fns320cvxuuvQFUFNYwthoXBDEytxDiN3ZYBf9r7LtOuEegUu~84tOwZrSHNMGvw~8k3vKZBTSXv8tG4D0k4B7QUY9zgLE0mv5gMfD0IxbvKf~5qcPtVkRcJO4t7fqPrT0Lx~dVb4EMB~4iHFVFz-Ab-Wo~uriF-fcIryDLsWcMy9~9KmoJXBhKNOHswSjaZeBzBYxPq70LCgZjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034167&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of Ikeda- and Boryong-specific sequences among other OT strains. The presence of each strain-specific region (&gt;1 kb) was examined by PCR scanning using two pairs of PCR primers, except for Ike_spe6, which was examined using six pairs of primers. Regions present in each strain are indicated by black cells with white plus signs. Regions where only one primer pair yielded a PCR product are indicated by gray cells with black plus signs. Regions yielding PCR products with different sizes compared with those of the reference strain (Ikeda or Boryong) are also indicated by gray cells. In Ike_spe6 of strain Kato, five primer pairs yielded PCR products. Phylogenetic relationships of the OT strains examined are also indicated (Fig. <a reveal-id="DSQ018F5" data-open="DSQ018F5" class="link link-reveal link-table xref-fig">5</a>).</p></div></div><div content-id="DSQ018F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01803.jpeg?Expires=2147483647&Signature=Oz62PobzswecVV3iQXllQ5yJWKsAt3GNQbiZdtJK1RSFTR7kU6WQWrLSv37iRxN4eAlThKt~lG39SG51bmurcTw-z1LcTUTT6RIclz7076YipdM~Os74nQIVN9D49IW0Et3lYv-n2zArQqK9hrJZZgYPjW4dgOZb0RziRcu-VpXNzpgdyCaP8WZ2I2YVrr8JhYCwknVHGI0nZT-oou4Mt0Zga~~~5HN4dqyzttbZEn2Bv8TTRVwV08yhF2-xj5fGiVuhgeeR8-RnMkhr0lQHjhkt8u~edBn9WGMCayllUJt2aMQATVCvGNF1CE9n~XgCa1Lx7rpLMGu4WSDXinrjXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of Ikeda- and Boryong-specific sequences among other OT strains. The presence of each strain-specific region (&gt;1 kb) was examined by PCR scanning using two pairs of PCR primers, except for Ike_spe6, which was examined using six pairs of primers. Regions present in each strain are indicated by black cells with white plus signs. Regions where only one primer pair yielded a PCR product are indicated by gray cells with black plus signs. Regions yielding PCR products with different sizes compared with those of the reference strain (Ikeda or Boryong) are also indicated by gray cells. In Ike_spe6 of strain Kato, five primer pairs yielded PCR products. Phylogenetic relationships of the OT strains examined are also indicated (Fig. 5)." path-from-xml="dsq01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034167/dsq01803.jpeg" path-from-xml="dsq01803.jpeg" target="_blank">View large</a><a section="5034167" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01803.jpeg?Expires=2147483647&Signature=N-uPn07lNdghTcB4QVrDoA1O35nnb5Oe7zQkhdok7qkPP0ySH7KYPk3E8cWHsNEkEtcM0lcTsMgv5XH-HPDP5ejOuRz73Q7Awa9C1oWwebNxw~DeIPbf1aq-ORvMjnF0zGnFc9Fns320cvxuuvQFUFNYwthoXBDEytxDiN3ZYBf9r7LtOuEegUu~84tOwZrSHNMGvw~8k3vKZBTSXv8tG4D0k4B7QUY9zgLE0mv5gMfD0IxbvKf~5qcPtVkRcJO4t7fqPrT0Lx~dVb4EMB~4iHFVFz-Ab-Wo~uriF-fcIryDLsWcMy9~9KmoJXBhKNOHswSjaZeBzBYxPq70LCgZjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034167&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of Ikeda- and Boryong-specific sequences among other OT strains. The presence of each strain-specific region (&gt;1 kb) was examined by PCR scanning using two pairs of PCR primers, except for Ike_spe6, which was examined using six pairs of primers. Regions present in each strain are indicated by black cells with white plus signs. Regions where only one primer pair yielded a PCR product are indicated by gray cells with black plus signs. Regions yielding PCR products with different sizes compared with those of the reference strain (Ikeda or Boryong) are also indicated by gray cells. In Ike_spe6 of strain Kato, five primer pairs yielded PCR products. Phylogenetic relationships of the OT strains examined are also indicated (Fig. <a reveal-id="DSQ018F5" data-open="DSQ018F5" class="link link-reveal link-table xref-fig">5</a>).</p></div></div>                        <a id="5034170" scrollto-destination="5034170"></a>
<div id="DSQ018F4" data-id="DSQ018F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01804.jpeg?Expires=2147483647&Signature=IaYnxKiOTp86-aW0~pH6UnteB8P9IzSyPPdFbZ0nTCxXSUjiuzOC4f1PZnnTC1Ipy~jyje98Rt~dCpBB1rmTHiCbzBLFIb2Z7N9sV5VjMWnui3X1RTQzfb8payHqfpdf8s5BjI2~0wMPk28E4ip21NOeiyIsyyYdidalw7EDuevekEgUpcbTee1SV-TYp-cuXmsT6k3KMPicEiNYpVesYqJF9g9x3B1yAMfSJrec1gf0LzEg64qfN5bm8voi0r5PxQ~DW2M3JafOg4Is8ic2nmrLVCpGkrrEW84jbNnxhNqx6YEPjVJCmvH4xNGTYYOafnmvBElUrjFoINjLoHNq6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genetic organization of Ikeda_spe6 and distribution of genes homologous to CDSs on Ike_spe6. Ike_spe6 is the longest strain-specific sequence found in the Ikeda genome. In (a), the G + C content of this region (top part; the average G + C content of the Ikeda genome is indicated by a broken line), distribution of homologs for Ike_spe6-encoded genes (middle part) and the genetic organizations of Ike_spe6 (bottom part) are shown. White, gray and black boxes indicate singleton genes, repeated genes and IS elements (or MITEs), respectively. Species and strain designations are as follows: Rp, Rickettsia prowazekii strain Madrid E; Rf, Rickettsia felis strain URRWXCal2; Rb, Rickettsia bellii strain RML369-C; Wol, Wolbachia pipientis strain wMel; Wbm, Wolbachia sp. strain TRS; Pcu, Candidatus Protochlamydia amoebophila strain UWE25; Lpn, Legionella pneumophila strain Philadelphia-1; Lpp, Legionella pneumophila strain Paris; Lpf, Legionella pneumophila strain Lens. The presence of pseudogenes is indicated by ‘i*’. In (b), the genetic organization of the insertion site for Ike_spe6 is shown. Ike_spe6 is flanked by two mISOts4 elements (ISOts4-related MITE), thus displaying a composite transposon (Tn)-like structure with terminal inverted repeats of 95 bp. This Tn-like element is integrated into OtRG34, creating a target site duplication of 5 bp (black arrowheads)." path-from-xml="dsq01804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034170/dsq01804.jpeg" path-from-xml="dsq01804.jpeg" target="_blank">View large</a><a section="5034170" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01804.jpeg?Expires=2147483647&Signature=a~vj9GXp2PFIsK9jc38u1Gdu7EDUv0h-cccuL0mf7t2JGS62hMyPW-2r4OGWfZBzwdd1NsUoWPrETGzNKOfU5xxTYUcN~0fk~R4guo~xyEm4s2kA4RjPcrdbbyFMUsSDrtKtx1LBU47Ri0u6~zlqhcg3uc6lAimx2sDefS3-9yNWU93x-ZQi459fmKUvPeyIpgrphT4Fxdtn5O0HbBoDzkx1Yh-6FWLjXinwkECDSoO57Puog0wAJKBvwX6RuUxDE8eG~XJpWkDBhob3TYxxIQq8WFHxotqHMBaLzGxKgS3ny3nwTy3JpNZg2jD-0htsIBjqClboZznI3pfEGmOEyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034170&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genetic organization of Ikeda_spe6 and distribution of genes homologous to CDSs on Ike_spe6. Ike_spe6 is the longest strain-specific sequence found in the Ikeda genome. In (a), the G + C content of this region (top part; the average G + C content of the Ikeda genome is indicated by a broken line), distribution of homologs for Ike_spe6-encoded genes (middle part) and the genetic organizations of Ike_spe6 (bottom part) are shown. White, gray and black boxes indicate singleton genes, repeated genes and IS elements (or MITEs), respectively. Species and strain designations are as follows: <em>Rp</em>, <em>Rickettsia prowazekii</em> strain Madrid E; <em>Rf</em>, <em>Rickettsia felis</em> strain URRWXCal2; <em>Rb, Rickettsia bellii</em> strain RML369-C; Wol, <em>Wolbachia pipientis</em> strain wMel; Wbm, <em>Wolbachia sp</em>. strain TRS; Pcu, <em>Candidatus Protochlamydia amoebophila</em> strain UWE25; Lpn, <em>Legionella pneumophila</em> strain Philadelphia-1; Lpp, <em>Legionella pneumophila</em> strain Paris; Lpf, <em>Legionella pneumophila</em> strain Lens. The presence of pseudogenes is indicated by ‘i*’. In (b), the genetic organization of the insertion site for Ike_spe6 is shown. Ike_spe6 is flanked by two mIS<em>Ots4</em> elements (ISOts4-related MITE), thus displaying a composite transposon (Tn)-like structure with terminal inverted repeats of 95 bp. This Tn-like element is integrated into OtRG34, creating a target site duplication of 5 bp (black arrowheads).</p></div></div><div content-id="DSQ018F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01804.jpeg?Expires=2147483647&Signature=IaYnxKiOTp86-aW0~pH6UnteB8P9IzSyPPdFbZ0nTCxXSUjiuzOC4f1PZnnTC1Ipy~jyje98Rt~dCpBB1rmTHiCbzBLFIb2Z7N9sV5VjMWnui3X1RTQzfb8payHqfpdf8s5BjI2~0wMPk28E4ip21NOeiyIsyyYdidalw7EDuevekEgUpcbTee1SV-TYp-cuXmsT6k3KMPicEiNYpVesYqJF9g9x3B1yAMfSJrec1gf0LzEg64qfN5bm8voi0r5PxQ~DW2M3JafOg4Is8ic2nmrLVCpGkrrEW84jbNnxhNqx6YEPjVJCmvH4xNGTYYOafnmvBElUrjFoINjLoHNq6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genetic organization of Ikeda_spe6 and distribution of genes homologous to CDSs on Ike_spe6. Ike_spe6 is the longest strain-specific sequence found in the Ikeda genome. In (a), the G + C content of this region (top part; the average G + C content of the Ikeda genome is indicated by a broken line), distribution of homologs for Ike_spe6-encoded genes (middle part) and the genetic organizations of Ike_spe6 (bottom part) are shown. White, gray and black boxes indicate singleton genes, repeated genes and IS elements (or MITEs), respectively. Species and strain designations are as follows: Rp, Rickettsia prowazekii strain Madrid E; Rf, Rickettsia felis strain URRWXCal2; Rb, Rickettsia bellii strain RML369-C; Wol, Wolbachia pipientis strain wMel; Wbm, Wolbachia sp. strain TRS; Pcu, Candidatus Protochlamydia amoebophila strain UWE25; Lpn, Legionella pneumophila strain Philadelphia-1; Lpp, Legionella pneumophila strain Paris; Lpf, Legionella pneumophila strain Lens. The presence of pseudogenes is indicated by ‘i*’. In (b), the genetic organization of the insertion site for Ike_spe6 is shown. Ike_spe6 is flanked by two mISOts4 elements (ISOts4-related MITE), thus displaying a composite transposon (Tn)-like structure with terminal inverted repeats of 95 bp. This Tn-like element is integrated into OtRG34, creating a target site duplication of 5 bp (black arrowheads)." path-from-xml="dsq01804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034170/dsq01804.jpeg" path-from-xml="dsq01804.jpeg" target="_blank">View large</a><a section="5034170" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01804.jpeg?Expires=2147483647&Signature=a~vj9GXp2PFIsK9jc38u1Gdu7EDUv0h-cccuL0mf7t2JGS62hMyPW-2r4OGWfZBzwdd1NsUoWPrETGzNKOfU5xxTYUcN~0fk~R4guo~xyEm4s2kA4RjPcrdbbyFMUsSDrtKtx1LBU47Ri0u6~zlqhcg3uc6lAimx2sDefS3-9yNWU93x-ZQi459fmKUvPeyIpgrphT4Fxdtn5O0HbBoDzkx1Yh-6FWLjXinwkECDSoO57Puog0wAJKBvwX6RuUxDE8eG~XJpWkDBhob3TYxxIQq8WFHxotqHMBaLzGxKgS3ny3nwTy3JpNZg2jD-0htsIBjqClboZznI3pfEGmOEyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034170&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genetic organization of Ikeda_spe6 and distribution of genes homologous to CDSs on Ike_spe6. Ike_spe6 is the longest strain-specific sequence found in the Ikeda genome. In (a), the G + C content of this region (top part; the average G + C content of the Ikeda genome is indicated by a broken line), distribution of homologs for Ike_spe6-encoded genes (middle part) and the genetic organizations of Ike_spe6 (bottom part) are shown. White, gray and black boxes indicate singleton genes, repeated genes and IS elements (or MITEs), respectively. Species and strain designations are as follows: <em>Rp</em>, <em>Rickettsia prowazekii</em> strain Madrid E; <em>Rf</em>, <em>Rickettsia felis</em> strain URRWXCal2; <em>Rb, Rickettsia bellii</em> strain RML369-C; Wol, <em>Wolbachia pipientis</em> strain wMel; Wbm, <em>Wolbachia sp</em>. strain TRS; Pcu, <em>Candidatus Protochlamydia amoebophila</em> strain UWE25; Lpn, <em>Legionella pneumophila</em> strain Philadelphia-1; Lpp, <em>Legionella pneumophila</em> strain Paris; Lpf, <em>Legionella pneumophila</em> strain Lens. The presence of pseudogenes is indicated by ‘i*’. In (b), the genetic organization of the insertion site for Ike_spe6 is shown. Ike_spe6 is flanked by two mIS<em>Ots4</em> elements (ISOts4-related MITE), thus displaying a composite transposon (Tn)-like structure with terminal inverted repeats of 95 bp. This Tn-like element is integrated into OtRG34, creating a target site duplication of 5 bp (black arrowheads).</p></div></div>                            <h3 scrollto-destination=5034173 id="5034173" class="section-title" data-legacy-id=s3e>3.5 Comparison of gene content between strains Ikeda and Boryong</h3>
                            <h4 scrollto-destination=5034176 id="5034176" class="section-title" data-legacy-id=s3e1>3.5.1 Repeated genes</h4>
<p>In our previous analysis of the Ikeda genome,<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> we defined repeated gene families as the genes whose products exhibited at least 90% amino acid sequence identity over 60% of the alignment length. According to this definition, we found 1196 repeated genes that were classified into 85 OT repeated gene (OtRG) families. OtRG1–OtRG9 are genes in TEs, and OtRG10–OtRG54 are those in OtAGEs. Thus, these families are also conserved in Boryong. Other repeated genes (OtRG55–OtRG85) are not components of repetitive elements, but their frequent association with repetitive sequences suggested that they have been duplicated or multiplexed in conjunction with amplification of repetitive sequences.<sup><a reveal-id="DSQ018C13" data-open="DSQ018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In support of this hypothesis, copy numbers for many of these OtRGs differ significantly between the two strains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). At least six genes have duplicated in Ikeda but not in Boryong. In addition, several repeated gene families are present in only one strain (three families specific to Ikeda and two families specific to Boryong). Extensive gene decay has also taken place in many Boryong-repeated genes as in those of Ikeda.</p>                            <h4 scrollto-destination=5034180 id="5034180" class="section-title" data-legacy-id=s3e2>3.5.2 Singleton genes</h4>
<p>We previously found that, among the 1967 CDSs of Ikeda, 771 are singleton genes and 543 of the 771 genes are shared with <em>R. prowazekii</em>. Because most OT/<em>R. prowazekii</em> shared genes (520 genes) are conserved in all five sequenced <em>Rickettsia</em> species, they appear to represent the core gene set of the family Rickettsiaceae. As expected, 541 of the 543 shared genes are conserved in Boryong (Table <a reveal-id="DSQ018TB3" data-open="DSQ018TB3" class="link link-reveal link-table xref-fig">3</a>). No other Boryong singleton gene was conserved in <em>R. prowazekii</em>. Genomic locations of these 541 genes have been extensively shuffled between the two OT genomes, but the average nt sequence identity of these genes is 97.5%. </p>                        <a id="5034184" scrollto-destination="5034184"></a>
<div content-id="DSQ018TB3" class="table-modal"><div class="table-wrap"><div id="DSQ018TB3" class="table-wrap-title" data-id="DSQ018TB3"><div class="label -label">Table 3</div><div class="caption"><p>Conservation of Ikeda singleton genes in strain Boryong</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Ikeda singleton gene </th><th colspan="2">Strain Boryong<hr /> </th></tr><tr><th>Present </th><th>Absent </th></tr></thead><tbody><tr><td>Ikeda (+)/<em>R. prowazekii</em> (+) </td><td>543<sup>a</sup> (84)<sup>b</sup> </td><td>541 (83) </td><td>2 (1) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> sp.<sup>c</sup> </td><td>520 (80) </td><td>519 (80) </td><td>1 (0) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>23 (4) </td><td>22 (3) </td><td>1 (1) </td></tr><tr><td>Ikeda (+)/<em>R. prowazekii</em> (−) </td><td>195 (149) </td><td>124 (91) </td><td>71 (58) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> sp. </td><td>161 (131) </td><td>96 (77) </td><td>65 (54) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>34 (18) </td><td>28 (14) </td><td>6 (4) </td></tr></tbody></table></div><div content-id="DSQ018TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Ikeda singleton gene </th><th colspan="2">Strain Boryong<hr /> </th></tr><tr><th>Present </th><th>Absent </th></tr></thead><tbody><tr><td>Ikeda (+)/<em>R. prowazekii</em> (+) </td><td>543<sup>a</sup> (84)<sup>b</sup> </td><td>541 (83) </td><td>2 (1) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> sp.<sup>c</sup> </td><td>520 (80) </td><td>519 (80) </td><td>1 (0) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>23 (4) </td><td>22 (3) </td><td>1 (1) </td></tr><tr><td>Ikeda (+)/<em>R. prowazekii</em> (−) </td><td>195 (149) </td><td>124 (91) </td><td>71 (58) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> sp. </td><td>161 (131) </td><td>96 (77) </td><td>65 (54) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>34 (18) </td><td>28 (14) </td><td>6 (4) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of shared genes has been changed from 542 to 543, because OTT_0717 was added to this gene set after re-inspection.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Numbers of pseudogenes are indicated in parentheses.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The five <em>Rickettsia</em> species examined here include <em>R. prowazekii</em>, <em>R. conorii</em>, <em>R. felis</em>, <em>R. typhi</em> and <em>R. bellii</em>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034184" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ018TB3" class="table-wrap-title" data-id="DSQ018TB3"><div class="label -label">Table 3</div><div class="caption"><p>Conservation of Ikeda singleton genes in strain Boryong</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Ikeda singleton gene </th><th colspan="2">Strain Boryong<hr /> </th></tr><tr><th>Present </th><th>Absent </th></tr></thead><tbody><tr><td>Ikeda (+)/<em>R. prowazekii</em> (+) </td><td>543<sup>a</sup> (84)<sup>b</sup> </td><td>541 (83) </td><td>2 (1) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> sp.<sup>c</sup> </td><td>520 (80) </td><td>519 (80) </td><td>1 (0) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>23 (4) </td><td>22 (3) </td><td>1 (1) </td></tr><tr><td>Ikeda (+)/<em>R. prowazekii</em> (−) </td><td>195 (149) </td><td>124 (91) </td><td>71 (58) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> sp. </td><td>161 (131) </td><td>96 (77) </td><td>65 (54) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>34 (18) </td><td>28 (14) </td><td>6 (4) </td></tr></tbody></table></div><div content-id="DSQ018TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Ikeda singleton gene </th><th colspan="2">Strain Boryong<hr /> </th></tr><tr><th>Present </th><th>Absent </th></tr></thead><tbody><tr><td>Ikeda (+)/<em>R. prowazekii</em> (+) </td><td>543<sup>a</sup> (84)<sup>b</sup> </td><td>541 (83) </td><td>2 (1) </td></tr><tr><td> Present in all the five <em>Rickettsia</em> sp.<sup>c</sup> </td><td>520 (80) </td><td>519 (80) </td><td>1 (0) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>23 (4) </td><td>22 (3) </td><td>1 (1) </td></tr><tr><td>Ikeda (+)/<em>R. prowazekii</em> (−) </td><td>195 (149) </td><td>124 (91) </td><td>71 (58) </td></tr><tr><td> Absent in all the five <em>Rickettsia</em> sp. </td><td>161 (131) </td><td>96 (77) </td><td>65 (54) </td></tr><tr><td> Variably present in the five <em>Rickettsia</em> sp. </td><td>34 (18) </td><td>28 (14) </td><td>6 (4) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of shared genes has been changed from 542 to 543, because OTT_0717 was added to this gene set after re-inspection.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Numbers of pseudogenes are indicated in parentheses.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The five <em>Rickettsia</em> species examined here include <em>R. prowazekii</em>, <em>R. conorii</em>, <em>R. felis</em>, <em>R. typhi</em> and <em>R. bellii</em>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034184" target="_blank">View Large</a></div></div><p>Two of the 543 OT/<em>R. prowazekii</em> shared genes are not conserved in Boryong: the <em>atpF</em> gene encoding the beta-subunit of ATP synthase F0 sector (OTT_0385 in Ikeda) and a gene encoding a hypothetical protein containing ankyrin repeats (OTT_1381). The <em>atpF</em> gene of Boryong contains a frameshift mutation, and thus the N-terminal 47 amino acids are missing. This mutation may be due to a sequence error. OTT_1481 is located in an Ikeda strain-specific sequence (Ike_spe13) that is flanked by OtAGEs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Boryong may have lost this gene by the OtAGE-mediated deletion of Ike_spe13. Conversely, one of the 543 OT/<em>R. prowazekii</em> shared genes (the tryptophanyl-tRNA synthetase gene) has been duplicated in Boryong (OTBS_0762 and OTBS_1824). IS<em>Ots1</em> and GIIOt1 flanking these genes suggest that this gene duplication has also been induced by TEs.</p><p>In addition to the 543 shared genes, Ikeda possesses 195 singleton genes. Among these, 124 are present in Boryong but 71 are absent (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Of the former 124 genes, OTT_0440 (encoding a repeat-containing protein) and OTT_0622 (encoding a hypothetical protein) are duplicated in Boryong. OtAGEs and/or TEs are also located at both the ends of these CDSs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>), suggesting that their duplications have also been induced by repetitive sequences. Although many Ikeda-specific singleton genes (58 out of 71) are of unknown function, these genes may have introduced phenotypic differences between the two strains.</p><p>Boryong also possesses 74 singleton genes that are not present in Ikeda. Sequences corresponding to 49 of these genes are also present in Ikeda, but they were not annotated as CDSs under our annotation criteria. The remaining 25 Boryong-specific genes are of unknown function.</p><p>Another interesting finding is that, among 27 disrupted Ikeda singleton genes, 23 have also been disrupted and 4 were not identified in the Boryong genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). This finding may suggest that inactivation of these genes took place before the divergence of the two strains.</p>                            <h3 scrollto-destination=5034193 id="5034193" class="section-title" data-legacy-id=s3f>3.6 Lineage analysis of OT strains by MLS analysis</h3>
<p>Partial nt sequences of the TSA gene were used for lineage analysis of OT strains.<sup><a reveal-id="DSQ018C10" data-open="DSQ018C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The TSA gene encodes a major surface protein. It exhibits remarkable strain-to-strain sequence variation and contains four variable domains. Phylogenetic trees constructed using each domain sequence are not consistent between the domains.<sup><a reveal-id="DSQ018C23" data-open="DSQ018C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Thus, the sequence of this gene is not suitable for phylogenetic analysis of OT strains. Therefore, we selected 11 genes among the core gene set of the family Rickettsiaceae and performed MLS analysis of various OT strains using concatenated sequences of these 11 genes. Strains examined in this analysis included nine OT strains (Ikeda, Gilliam, Kato, Karp, Matsuzawa, 423H, Kuroki, Kawasaki and Shimokoshi) representing each of the known nine TSA subtypes. Strain Boryong was also included in this analysis. The phylogenetic tree obtained by this MLS analysis differed considerably from that generated using TSA gene sequences and probably represents the true phylogenetic relationship among OT strains (Fig. <a reveal-id="DSQ018F5" data-open="DSQ018F5" class="link link-reveal link-table xref-fig">5</a>). Strain Boryong, which can be classified as the Kuroki subtype with TSA-based subtyping, has the same MLS type as strain Kuroki. </p>                        <a id="5034195" scrollto-destination="5034195"></a>
<div id="DSQ018F5" data-id="DSQ018F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01805.jpeg?Expires=2147483647&Signature=gjYhOynHkPNumhUWMrVhWXrgZYfYuAwqJmk0aXd4tpEy2zTYvbPkJoK8BxXywt5uiDAECdkEVkCon1nheLHKVZGl9GF10Z8A0wfTweGFNwzooyNhuqhUqCMmMWLk-QNx-JjhFv1sqq~nvdcfLwyF3y7wXSd3SdYEeLfU7sfWOATW-MRs5NcLrxfWjLCHTBXUnchobg7DaM5QEH22eRbajuzbGKBoamsd0Hlec8j7PYjLjxm0j0sr3RAc5jSF7-GjRTHeQMp9WF-tlQ4E3odRMagNVSlNNILppvtDvOjWv3~rcRxoy-8Ro-2Aq8UPWi3RdSWaVaB7gzoVq0orSJc7Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships of OT strains representing each known TSA subtype. (a) Phylogenetic tree constructed using the concatenated nt sequences of 11 housekeeping genes. The 11 genes used were selected from the core genes of five Rickettsia species: R. prowazekii, R. conorii, R. felis, R. typhi and R. bellii. The reliability of the internal branches was assessed by bootstrapping with 250 pseudoreplicates. Strains with higher virulence to mice clustered together in this phylogram. Virulence of each strain to mice was categorized according to their 50% mice lethal doses (MLD50), which were reported by Ohashi et al.11: +++, MLD50 (log10) &gt; 2.0; ++, 2.0 &gt; MLD50 &gt; 1.0; +, 1.0 &gt; MLD50 &gt; 0; -, all alive. The strain Matsuzawa does not show higher virulence to mice (indicated by asterisks), but this may be due to the slow growth of this strain. The virulence of strain Boryong to mice has not been determined. (b) Phylogenetic tree constructed using the sequences of TSA genes, which encode an OT-specific surface protein of 56 kDa. The scale bar represents the number of substitutions per site." path-from-xml="dsq01805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034195/dsq01805.jpeg" path-from-xml="dsq01805.jpeg" target="_blank">View large</a><a section="5034195" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01805.jpeg?Expires=2147483647&Signature=uHNxZCBhEjRc36Bg~Zbrs~AHEGJJMFK4qG74Ld7cAxtDUAZDpeCeU1WHAtNskV5KHfvCoGjfppltRrFyrm3hA0Ngd2gDBpuOz0X6cX14HHvXFj1Dcu3E9kqH9JZ2J5Ispl0~ls62kiAfIJjtu5oB~ZwrT-2~SxhJPy4Fwlu1jNked-VeXvI1SBK1qiOgCKNIwY8fMbvwV4vjMgY2StgHDawqh9qenFzZI76BY5CtQR2HguHZ75nYskal0q81xUuu~X4RY574rx0~9PYKmCA2VKM1yQtcFmJpjt7voLqcBjAFj90KVCsnw-nGKDnMIkuNuDsFzNg1pt7lGA907mcq3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034195&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships of OT strains representing each known TSA subtype. (a) Phylogenetic tree constructed using the concatenated nt sequences of 11 housekeeping genes. The 11 genes used were selected from the core genes of five <em>Rickettsia</em> species: <em>R. prowazekii</em>, <em>R. conorii</em>, <em>R. felis</em>, <em>R. typhi</em> and <em>R. bellii</em>. The reliability of the internal branches was assessed by bootstrapping with 250 pseudoreplicates. Strains with higher virulence to mice clustered together in this phylogram. Virulence of each strain to mice was categorized according to their 50% mice lethal doses (MLD<sub>50</sub>), which were reported by Ohashi <em>et al</em>.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup>: +++, MLD<sub>50</sub> (log<sub>10</sub>) &gt; 2.0; ++, 2.0 &gt; MLD<sub>50</sub> &gt; 1.0; +, 1.0 &gt; MLD<sub>50</sub> &gt; 0; -, all alive. The strain Matsuzawa does not show higher virulence to mice (indicated by asterisks), but this may be due to the slow growth of this strain. The virulence of strain Boryong to mice has not been determined. (b) Phylogenetic tree constructed using the sequences of TSA genes, which encode an OT-specific surface protein of 56 kDa. The scale bar represents the number of substitutions per site.</p></div></div><div content-id="DSQ018F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/m_dsq01805.jpeg?Expires=2147483647&Signature=gjYhOynHkPNumhUWMrVhWXrgZYfYuAwqJmk0aXd4tpEy2zTYvbPkJoK8BxXywt5uiDAECdkEVkCon1nheLHKVZGl9GF10Z8A0wfTweGFNwzooyNhuqhUqCMmMWLk-QNx-JjhFv1sqq~nvdcfLwyF3y7wXSd3SdYEeLfU7sfWOATW-MRs5NcLrxfWjLCHTBXUnchobg7DaM5QEH22eRbajuzbGKBoamsd0Hlec8j7PYjLjxm0j0sr3RAc5jSF7-GjRTHeQMp9WF-tlQ4E3odRMagNVSlNNILppvtDvOjWv3~rcRxoy-8Ro-2Aq8UPWi3RdSWaVaB7gzoVq0orSJc7Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships of OT strains representing each known TSA subtype. (a) Phylogenetic tree constructed using the concatenated nt sequences of 11 housekeeping genes. The 11 genes used were selected from the core genes of five Rickettsia species: R. prowazekii, R. conorii, R. felis, R. typhi and R. bellii. The reliability of the internal branches was assessed by bootstrapping with 250 pseudoreplicates. Strains with higher virulence to mice clustered together in this phylogram. Virulence of each strain to mice was categorized according to their 50% mice lethal doses (MLD50), which were reported by Ohashi et al.11: +++, MLD50 (log10) &gt; 2.0; ++, 2.0 &gt; MLD50 &gt; 1.0; +, 1.0 &gt; MLD50 &gt; 0; -, all alive. The strain Matsuzawa does not show higher virulence to mice (indicated by asterisks), but this may be due to the slow growth of this strain. The virulence of strain Boryong to mice has not been determined. (b) Phylogenetic tree constructed using the sequences of TSA genes, which encode an OT-specific surface protein of 56 kDa. The scale bar represents the number of substitutions per site." path-from-xml="dsq01805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034195/dsq01805.jpeg" path-from-xml="dsq01805.jpeg" target="_blank">View large</a><a section="5034195" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq01805.jpeg?Expires=2147483647&Signature=uHNxZCBhEjRc36Bg~Zbrs~AHEGJJMFK4qG74Ld7cAxtDUAZDpeCeU1WHAtNskV5KHfvCoGjfppltRrFyrm3hA0Ngd2gDBpuOz0X6cX14HHvXFj1Dcu3E9kqH9JZ2J5Ispl0~ls62kiAfIJjtu5oB~ZwrT-2~SxhJPy4Fwlu1jNked-VeXvI1SBK1qiOgCKNIwY8fMbvwV4vjMgY2StgHDawqh9qenFzZI76BY5CtQR2HguHZ75nYskal0q81xUuu~X4RY574rx0~9PYKmCA2VKM1yQtcFmJpjt7voLqcBjAFj90KVCsnw-nGKDnMIkuNuDsFzNg1pt7lGA907mcq3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034195&ar=353619&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships of OT strains representing each known TSA subtype. (a) Phylogenetic tree constructed using the concatenated nt sequences of 11 housekeeping genes. The 11 genes used were selected from the core genes of five <em>Rickettsia</em> species: <em>R. prowazekii</em>, <em>R. conorii</em>, <em>R. felis</em>, <em>R. typhi</em> and <em>R. bellii</em>. The reliability of the internal branches was assessed by bootstrapping with 250 pseudoreplicates. Strains with higher virulence to mice clustered together in this phylogram. Virulence of each strain to mice was categorized according to their 50% mice lethal doses (MLD<sub>50</sub>), which were reported by Ohashi <em>et al</em>.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup>: +++, MLD<sub>50</sub> (log<sub>10</sub>) &gt; 2.0; ++, 2.0 &gt; MLD<sub>50</sub> &gt; 1.0; +, 1.0 &gt; MLD<sub>50</sub> &gt; 0; -, all alive. The strain Matsuzawa does not show higher virulence to mice (indicated by asterisks), but this may be due to the slow growth of this strain. The virulence of strain Boryong to mice has not been determined. (b) Phylogenetic tree constructed using the sequences of TSA genes, which encode an OT-specific surface protein of 56 kDa. The scale bar represents the number of substitutions per site.</p></div></div><p>All OT strains used in this MLS analysis (except for Boryong) were previously examined for virulence in mice by Ohashi <em>et al.</em><sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In the new phylogram, strains that showed higher virulence to mice (strains Ikeda, Kato, Karp, 423H and Gilliam) clustered together. The sole exception is strain Matsuzawa belonging to the JP-1 type, which may be due to its slow growth in mammalian cells. This result suggests the presence of an OT lineage with higher virulence to mice.</p><p>Although the natural hosts of each OT strains are not fully understood, hosts for the OT strains of higher virulence are mainly <em>Leptotrombidium akamushi</em> and <em>L. pallidum</em>, whereas the hosts for the other strains are <em>L. scutellare</em> and some other <em>Leptotrombidium</em> spp.<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSQ018C24  DSQ018C27" data-open="DSQ018C24  DSQ018C27" class="link link-ref link-reveal xref-bibr">24–27</a></sup> This host specificity may be linked to the generation of such sublineages of OT. In Japan, many severe scrub typhus cases previously occurred only in the several local areas where <em>L. akamushi</em> is predominantly distributed, and the causative agents of these diseases (so-called classic-type Tsutsugamushi diseases) were the strains of serotype Kato. Recently, many scrub typhus cases (several hundred cases per year) are occurring in other areas of Japan, where <em>L. scutellare</em> and <em>L. pallidum</em> are densely distributed. Most of these new-type Tsutsugamushi diseases are caused by strains of serotypes Kawasaki (the natural host is <em>L. scutellare</em>) and Kuroki (unknown), and clinical symptoms of these diseases are generally milder than that of the classic type.<sup><a reveal-id="DSQ018C27" data-open="DSQ018C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Our findings may at least partly explain these clinical and epidemiological differences between the classic- and new-type Tsutsugamushi diseases.</p><p>Because strain Boryong has the same MLS type as that of strain Kuroki which belongs to the lower virulence group,<sup><a reveal-id="DSQ018C11" data-open="DSQ018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> it is possible that some of the Ikeda-specific sequences (or genes on the sequences) could be responsible for the virulence difference between the two groups. Therefore, we examined the distribution of 19 Ikeda- and 19 Boryong-specific sequences among the 10 OT strains by PCR scanning (Fig. <a reveal-id="DSQ018F3" data-open="DSQ018F3" class="link link-reveal link-table xref-fig">3</a>). The result of this analysis indicates that the distribution patterns of these sequences largely followed the phylogeny of each strain (Fig. <a reveal-id="DSQ018F3" data-open="DSQ018F3" class="link link-reveal link-table xref-fig">3</a>). For example, an intact copy of the above-mentioned Ike_spe6 was found only in strain Kato, which is most closely related to Ikeda. The insertion site of Ike_spe6 in strain Kato is the same as that in Ikeda (data not shown). However, none of the Ikeda-specific sequences showed a higher virulence group-specific distribution (shared by all strains of the higher virulence group but absent in the lower virulence group). Thus, the factors responsible for the virulence difference between the two groups still remain to be elucidated.</p>                            <h3 scrollto-destination=5034204 id="5034204" class="section-title" data-legacy-id=s3g>3.7 Conclusion</h3>
<p>Fine genome sequence comparison of two recently sequenced OT strains revealed that they contain the same set of repetitive sequences and that these repetitive sequences have been amplified explosively in both strains and have induced extensive genome shuffling. In addition, the core gene set of family Rickettsiaceae is highly conserved between the two strains. On the basis of the results of MLS analysis, we present a phylogram that represents a true phylogenetic relationship among OT strains. This analysis further revealed the presence of an OT sublineage with higher potential virulence, which might explain the clinical and epidemiological differences between the classic and new types of Tsutsugamushi disease in Japan.</p>                            <h2 scrollto-destination=5034206 id="5034206" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5034208 id="5034208" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by grants from the Ministry of Education, Culture, Sports, Science and Technology of Japan and the Ministry of Health, Labor and Welfare of Japan, and partly by a grant from Miyazaki Prefecture.</p>                            <h2 scrollto-destination=5034210 id="5034210" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Yoshiro Terawaki and Akira Tamura for their encouragement and Akemi Yoshida, Yumiko Takeshita, Noriko Kanemaru, Miki Shinbara, and Nobuko Fujii for their technical assistance.</p>                            <h2 scrollto-destination=5034212 id="5034212" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ018C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bandopadhyay</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Vertical transmission in chigger borne rickettsiosis</div>, <div class="source">Indian J. Med. Res.</div> , <div class="year">1978</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">31</div>-<div class="lpage">8</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/581279" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vertical%20transmission%20in%20chigger%20borne%20rickettsiosis&amp;author=J.%20Rai&amp;author=D.%20Bandopadhyay&amp;publication_year=1978&amp;journal=Indian%20J.%20Med.%20Res.&amp;volume=68&amp;pages=31-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/581279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rapmund</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Dohany</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Manikumaran</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Transovarial transmission of <em>Rickettsia tsutsugamushi</em> in <em>Leptotrombidium</em> (<em>Leptotrombidium</em>) <em>arenicola</em> Traub (Acarina: Trombiculidae)</div>, <div class="source">J. Med. Entomol.</div> , <div class="year">1972</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">71</div>-<div class="lpage">2</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/4623054" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transovarial%20transmission%20of%20Rickettsia%20tsutsugamushi%20in%20Leptotrombidium%20(Leptotrombidium)%20arenicola%20Traub%20(Acarina%3A%20Trombiculidae)&amp;author=G.%20Rapmund&amp;author=A.L.%20Dohany&amp;author=C.%20Manikumaran&amp;author=T.C.%20Chan&amp;publication_year=1972&amp;journal=J.%20Med.%20Entomol.&amp;volume=9&amp;pages=71-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jmedent/9.1.71" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjmedent%2F9.1.71" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjmedent%2F9.1.71"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4623054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rapmund</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Upham</div> <div class="given-names">R.W.</div>Jr</div>,  <div class="name"><div class="surname">Kundin</div> <div class="given-names">W.D.</div></div>,  <div class="name"><div class="surname">Manikumaran</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Transovarial development of scrub typhus rickettsiae in a colony of vector mites</div>, <div class="source">Trans. R. Soc. Trop. Med. Hyg.</div> , <div class="year">1969</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">251</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0035-9203(69)90155-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0035-9203(69)90155-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transovarial%20development%20of%20scrub%20typhus%20rickettsiae%20in%20a%20colony%20of%20vector%20mites&amp;author=G.%20Rapmund&amp;author=R.W.%20Upham&amp;author=W.D.%20Kundin&amp;author=C.%20Manikumaran&amp;author=T.C.%20Chan&amp;publication_year=1969&amp;journal=Trans.%20R.%20Soc.%20Trop.%20Med.%20Hyg.&amp;volume=63&amp;pages=251-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0035-9203(69)90155-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0035-9203(69)90155-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0035-9203(69)90155-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4978468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nogami</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hori</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kawamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Transovarial transmission of <em>Rickettsia tsutsugamushi</em> in <em>Leptotrombidium pallidum</em> successively reared in the laboratory</div>, <div class="source">Jpn J. Exp. Med.</div> , <div class="year">1988</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">213</div>-<div class="lpage">8</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/3149693" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transovarial%20transmission%20of%20Rickettsia%20tsutsugamushi%20in%20Leptotrombidium%20pallidum%20successively%20reared%20in%20the%20laboratory&amp;author=M.%20Takahashi&amp;author=M.%20Murata&amp;author=S.%20Nogami&amp;author=E.%20Hori&amp;author=A.%20Kawamura&amp;author=H.%20Tanaka&amp;publication_year=1988&amp;journal=Jpn%20J.%20Exp.%20Med.&amp;volume=58&amp;pages=213-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3149693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takahasi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hori</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">An ultrastructural study of vertical transmission of <em>Rickettsia tsutsugamushi</em> during oogenesis and spermatogenesis in <em>Leptotrombidium pallidum</em></div>, <div class="source">Am. J. Trop. Med. Hyg.</div> , <div class="year">1994</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">219</div>-<div class="lpage">28</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8116816" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ultrastructural%20study%20of%20vertical%20transmission%20of%20Rickettsia%20tsutsugamushi%20during%20oogenesis%20and%20spermatogenesis%20in%20Leptotrombidium%20pallidum&amp;author=H.%20Urakami&amp;author=M.%20Takahasi&amp;author=E.%20Hori&amp;author=A.%20Tamura&amp;publication_year=1994&amp;journal=Am.%20J.%20Trop.%20Med.%20Hyg.&amp;volume=50&amp;pages=219-28" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8116816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Misumi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hori</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kawamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Failed vertical transmission of <em>Rickettsia tsutsugamushi</em> (Rickettsiales: Rickettsiaceae) acquired from rickettsemic mice by <em>Leptotrombidium pallidum</em> (Acari: Trombiculidae)</div>, <div class="source">J. Med. Entomol.</div> , <div class="year">1994</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">212</div>-<div class="lpage">6</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8189412" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Failed%20vertical%20transmission%20of%20Rickettsia%20tsutsugamushi%20(Rickettsiales%3A%20Rickettsiaceae)%20acquired%20from%20rickettsemic%20mice%20by%20Leptotrombidium%20pallidum%20(Acari%3A%20Trombiculidae)&amp;author=M.%20Takahashi&amp;author=M.%20Murata&amp;author=H.%20Misumi&amp;author=E.%20Hori&amp;author=A.%20Kawamura&amp;author=H.%20Tanaka&amp;publication_year=1994&amp;journal=J.%20Med.%20Entomol.&amp;volume=31&amp;pages=212-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jmedent/31.2.212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjmedent%2F31.2.212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjmedent%2F31.2.212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8189412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walker</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Manikumaran</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Elisberg</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">Attempts to infect and demonstrate transovarial transmission of <em>R. tsutsugamushi</em> in three species of <em>Leptotrombidium mites</em></div>, <div class="source">Ann. N Y Acad. Sci.</div> , <div class="year">1975</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">80</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1749-6632.1975.tb35090.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1749-6632.1975.tb35090.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Attempts%20to%20infect%20and%20demonstrate%20transovarial%20transmission%20of%20R.%20tsutsugamushi%20in%20three%20species%20of%20Leptotrombidium%20mites&amp;author=J.S.%20Walker&amp;author=C.T.%20Chan&amp;author=C.%20Manikumaran&amp;author=B.L.%20Elisberg&amp;publication_year=1975&amp;journal=Ann.%20N%20Y%20Acad.%20Sci.&amp;volume=266&amp;pages=80-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nyas.1975.266.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnyas.1975.266.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnyas.1975.266.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/829478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Traub</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wisseman</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">O'Keefe</div> <div class="given-names">J.J.</div></div>. </div><div class="article-title">The acquisition of <em>Rickettsia tsutsugamushi</em> by chiggers (trombiculid mites) during the feeding process</div>, <div class="source">Ann. N Y Acad. Sci.</div> , <div class="year">1975</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">91</div>-<div class="lpage">114</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1749-6632.1975.tb35091.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1749-6632.1975.tb35091.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20acquisition%20of%20Rickettsia%20tsutsugamushi%20by%20chiggers%20(trombiculid%20mites)%20during%20the%20feeding%20process&amp;author=R.%20Traub&amp;author=C.L.%20Wisseman&amp;author=M.R.%20Jones&amp;author=J.J.%20O'Keefe&amp;publication_year=1975&amp;journal=Ann.%20N%20Y%20Acad.%20Sci.&amp;volume=266&amp;pages=91-114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nyas.1975.266.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnyas.1975.266.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnyas.1975.266.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/829479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kelly</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Fuerst</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Ching</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Richards</div> <div class="given-names">A.L.</div></div>. </div><div class="article-title">Scrub typhus: the geographic distribution of phenotypic and genotypic variants of <em>Orientia tsutsugamushi</em></div>, <div class="source">Clin. Infect. Dis.</div> , <div class="year">2009</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">S203</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1086/596576" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1086/596576</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scrub%20typhus%3A%20the%20geographic%20distribution%20of%20phenotypic%20and%20genotypic%20variants%20of%20Orientia%20tsutsugamushi&amp;author=D.J.%20Kelly&amp;author=P.A.%20Fuerst&amp;author=W.M.%20Ching&amp;author=A.L.%20Richards&amp;publication_year=2009&amp;journal=Clin.%20Infect.%20Dis.&amp;volume=48&amp;pages=S203-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/598225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F598225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F598225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19220144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Enatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Phylogenetic analysis of <em>Orientia tsutsugamushi</em> strains based on the sequence homologies of 56-kDa type-specific antigen genes</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">163</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20Orientia%20tsutsugamushi%20strains%20based%20on%20the%20sequence%20homologies%20of%2056-kDa%20type-specific%20antigen%20genes&amp;author=T.%20Enatsu&amp;author=H.%20Urakami&amp;author=A.%20Tamura&amp;publication_year=1999&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=15&amp;pages=163-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.1999.180.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.1999.180.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.1999.180.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSQ018C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Koyama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Urakami</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Demonstration of antigenic and genotypic variation in <em>Orientia tsutsugamushi</em> which were isolated in Japan, and their classification into type and subtype</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1996</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">627</div>-<div class="lpage">38</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8908607" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Demonstration%20of%20antigenic%20and%20genotypic%20variation%20in%20Orientia%20tsutsugamushi%20which%20were%20isolated%20in%20Japan%2C%20and%20their%20classification%20into%20type%20and%20subtype&amp;author=N.%20Ohashi&amp;author=Y.%20Koyama&amp;author=H.%20Urakami&amp;publication_year=1996&amp;journal=Microbiol.%20Immunol.&amp;volume=40&amp;pages=627-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1996.40.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1996.40.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1996.40.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8908607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J.H.</div></div>, et al. </div><div class="article-title">The <em>Orientia tsutsugamushi</em> genome reveals massive proliferation of conjugative type IV secretion system and host-cell interaction genes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">7981</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Orientia%20tsutsugamushi%20genome%20reveals%20massive%20proliferation%20of%20conjugative%20type%20IV%20secretion%20system%20and%20host-cell%20interaction%20genes&amp;author=N.H.%20Cho&amp;author=H.R.%20Kim&amp;author=J.H.%20Lee&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=8&amp;pages=7981-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611553104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611553104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611553104"> </span></div> </div></div></div></div></div><div content-id="DSQ018C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamashita</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">The whole-genome sequencing of the obligate intracellular bacterium <em>Orientia tsutsugamushi</em> revealed massive gene amplification during reductive genome evolution</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">185</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn011" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn011</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20whole-genome%20sequencing%20of%20the%20obligate%20intracellular%20bacterium%20Orientia%20tsutsugamushi%20revealed%20massive%20gene%20amplification%20during%20reductive%20genome%20evolution&amp;author=K.%20Nakayama&amp;author=A.%20Yamashita&amp;author=K.%20Kurokawa&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=185-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18508905" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.17.3389" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.17.3389</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Improved tools for biological sequence comparison</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2444</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.85.8.2444" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.85.8.2444</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20tools%20for%20biological%20sequence%20comparison&amp;author=W.R.%20Pearson&amp;author=D.J.%20Lipman&amp;publication_year=1988&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=85&amp;pages=2444-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2444"> </span></div> </div></div></div></div></div><div content-id="DSQ018C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">16</div> pg. <div class="fpage">376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=16&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div> </div></div></div></div></div><div content-id="DSQ018C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Terajima</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genomic diversity of enterohemorrhagic <em>Escherichia coli</em> O157 revealed by whole genome PCR scanning</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">17043</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.262441699" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.262441699</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20diversity%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%20revealed%20by%20whole%20genome%20PCR%20scanning&amp;author=M.%20Ohnishi&amp;author=J.%20Terajima&amp;author=K.%20Kurokawa&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=17043-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.262441699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.262441699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.262441699"> </span></div> </div></div></div></div></div><div content-id="DSQ018C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>. </div><div class="article-title">MEGA: a biologist-centric software for evolutionary analysis of DNA and protein sequences</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">299</div>-<div class="lpage">306</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bib/bbn017" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bib/bbn017</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA%3A%20a%20biologist-centric%20software%20for%20evolutionary%20analysis%20of%20DNA%20and%20protein%20sequences&amp;author=S.%20Kumar&amp;author=M.%20Nei&amp;author=J.%20Dudley&amp;author=K.%20Tamura&amp;publication_year=2008&amp;journal=Brief%20Bioinform.&amp;volume=9&amp;pages=299-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18417537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Ori-Finder: a web-based system for finding oriCs in unannotated bacterial genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> pg. <div class="fpage">79</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ori-Finder%3A%20a%20web-based%20system%20for%20finding%20oriCs%20in%20unannotated%20bacterial%20genomes&amp;author=F.%20Gao&amp;author=C.T.%20Zhang&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=1&amp;pages=79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-79" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-79" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-79"> </span></div> </div></div></div></div></div><div content-id="DSQ018C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fritsche</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Horn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Seyedirashti</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">In situ detection of novel bacterial endosymbionts of <em>Acanthamoeba</em> spp. phylogenetically related to members of the order <em>Rickettsiales</em></div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">206</div>-<div class="lpage">12</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9872781" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20situ%20detection%20of%20novel%20bacterial%20endosymbionts%20of%20Acanthamoeba%20spp.%20phylogenetically%20related%20to%20members%20of%20the%20order%20Rickettsiales&amp;author=T.R.%20Fritsche&amp;author=M.%20Horn&amp;author=S.%20Seyedirashti&amp;publication_year=1999&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=65&amp;pages=206-12" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">La Scola</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome sequence of <em>Rickettsia bellii</em> illuminates the role of amoebae in gene exchanges between intracellular pathogens</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">733</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20Rickettsia%20bellii%20illuminates%20the%20role%20of%20amoebae%20in%20gene%20exchanges%20between%20intracellular%20pathogens&amp;author=H.%20Ogata&amp;author=B.%20La%20Scola&amp;author=S.%20Audic&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=733-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020076"> </span></div> </div></div></div></div></div><div content-id="DSQ018C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmitz-Esser</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tischler</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Arnold</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The genome of the amoeba symbiont ‘<em>Candidatus Amoebophilus asiaticus</em>’ reveals common mechanisms for host cell interaction among amoeba-associated bacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2010</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">1045</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20amoeba%20symbiont%20%E2%80%98Candidatus%20Amoebophilus%20asiaticus%E2%80%99%20reveals%20common%20mechanisms%20for%20host%20cell%20interaction%20among%20amoeba-associated%20bacteria&amp;author=S.%20Schmitz-Esser&amp;author=P.%20Tischler&amp;author=R.%20Arnold&amp;publication_year=2010&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=1045-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01379-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01379-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01379-09"> </span></div> </div></div></div></div></div><div content-id="DSQ018C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nashimoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Diversity of immunodominant 56-kDa type-specific antigen (TSA) of <em>Rickettsia tsutsugamushi</em>. Sequence and comparative analyses of the genes encoding TSA homologues from four antigenic variants</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1992</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">12728</div>-<div class="lpage">35</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1618776" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20immunodominant%2056-kDa%20type-specific%20antigen%20(TSA)%20of%20Rickettsia%20tsutsugamushi.%20Sequence%20and%20comparative%20analyses%20of%20the%20genes%20encoding%20TSA%20homologues%20from%20four%20antigenic%20variants&amp;author=N.%20Ohashi&amp;author=H.%20Nashimoto&amp;author=H.%20Ikeda&amp;author=A.%20Tamura&amp;publication_year=1992&amp;journal=J.%20Biol.%20Chem.&amp;volume=267&amp;pages=12728-35" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1618776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawamori</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Akiyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sugieda</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Epidemiology of Tsutsugamushi disease in relation to the serotypes of <em>Rickettsia tsutsugamushi</em> isolated from patients, field mice, and unfed chiggers on the eastern slope of Mount Fuji, Shizuoka Prefecture, Japan</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="year">1992</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">2842</div>-<div class="lpage">6</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1452653" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiology%20of%20Tsutsugamushi%20disease%20in%20relation%20to%20the%20serotypes%20of%20Rickettsia%20tsutsugamushi%20isolated%20from%20patients%2C%20field%20mice%2C%20and%20unfed%20chiggers%20on%20the%20eastern%20slope%20of%20Mount%20Fuji%2C%20Shizuoka%20Prefecture%2C%20Japan&amp;author=F.%20Kawamori&amp;author=M.%20Akiyama&amp;author=M.%20Sugieda&amp;publication_year=1992&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=30&amp;pages=2842-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1452653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kitaoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okubo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asanuma</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Epidemiological survey by means of complement fixation test on scrub typhus in Japan</div>, <div class="source">Acta Med. Biol.</div> , <div class="year">1967</div>, vol. <div class="volume">15</div> <div class="issue">Suppl</div>(pg. <div class="fpage">69</div>-<div class="lpage">85</div>)</div></div></div></div><div content-id="DSQ018C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shishido</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Strain variation of <em>Rickettsia orientalis</em> in the complement fixation test</div>, <div class="source">Jpn J. Med. Sci. Biol.</div> , <div class="year">1964</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">59</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strain%20variation%20of%20Rickettsia%20orientalis%20in%20the%20complement%20fixation%20test&amp;author=A.%20Shishido&amp;publication_year=1964&amp;journal=Jpn%20J.%20Med.%20Sci.%20Biol.&amp;volume=17&amp;pages=59-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7883/yoken1952.17.59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7883%2Fyoken1952.17.59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7883%2Fyoken1952.17.59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14206509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Koyama</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Epidemiological survey of <em>Orientia tsutsugamushi</em> distribution in field rodents in Saitama Prefecture, Japan, and discovery of a new type</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">2001</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">439</div>-<div class="lpage">46</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11497219" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiological%20survey%20of%20Orientia%20tsutsugamushi%20distribution%20in%20field%20rodents%20in%20Saitama%20Prefecture%2C%20Japan%2C%20and%20discovery%20of%20a%20new%20type&amp;author=A.%20Tamura&amp;author=N.%20Yamamoto&amp;author=S.%20Koyama&amp;publication_year=2001&amp;journal=Microbiol.%20Immunol.&amp;volume=45&amp;pages=439-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.2001.45.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.2001.45.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.2001.45.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11497219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsuruhara</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Isolation of <em>Rickettsia tsutsugamushi</em> antigenically different from Kato, Karp, and Gilliam strains from patients</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1984</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">873</div>-<div class="lpage">882</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20Rickettsia%20tsutsugamushi%20antigenically%20different%20from%20Kato%2C%20Karp%2C%20and%20Gilliam%20strains%20from%20patients&amp;author=A.%20Tamura&amp;author=K.%20Takahashi&amp;author=T.%20Tsuruhara&amp;publication_year=1984&amp;journal=Microbiol.%20Immunol.&amp;volume=28&amp;pages=873-882" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1984.28.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1984.28.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1984.28.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6438448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W.K.</div></div>, et al. </div><div class="article-title">Serological classification by monoclonal antibodies of <em>Rickettsia tsutsugamushi</em> isolated in Korea</div>, <div class="source">J. Clin. Microbiol</div> , <div class="year">1990</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">685</div>-<div class="lpage">688</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Serological%20classification%20by%20monoclonal%20antibodies%20of%20Rickettsia%20tsutsugamushi%20isolated%20in%20Korea&amp;author=W.H.%20Chang&amp;author=J.S.%20Kang&amp;author=W.K.%20Lee&amp;publication_year=1990&amp;journal=J.%20Clin.%20Microbiol&amp;volume=28&amp;pages=685-688" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2110179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bengston</div> <div class="given-names">I.A.</div></div>. </div><div class="article-title">Apparent serological heterogeneity among strains of tsutsugamushi disease (scrub typhus)</div>, <div class="source">Public Health Rep.</div> , <div class="year">1945</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1483</div>-<div class="lpage">1488</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Apparent%20serological%20heterogeneity%20among%20strains%20of%20tsutsugamushi%20disease%20(scrub%20typhus)&amp;author=I.A.%20Bengston&amp;publication_year=1945&amp;journal=Public%20Health%20Rep.&amp;volume=60&amp;pages=1483-1488" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/4585496" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F4585496" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F4585496"> </span></div> </div></div></div></div></div><div content-id="DSQ018C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shishido</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ohtawara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tateno</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The nature of immunity against scrub typhus in mice. I. The resistance of mice, surviving subcutaneous infection of scrub typhus rickettsia, to intraperitoneal reinfection of the same agent</div>, <div class="source">Jpn. J. Med. Sci. Biol.</div> , <div class="year">1958</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">383</div>-<div class="lpage">399</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nature%20of%20immunity%20against%20scrub%20typhus%20in%20mice.%20I.%20The%20resistance%20of%20mice%2C%20surviving%20subcutaneous%20infection%20of%20scrub%20typhus%20rickettsia%2C%20to%20intraperitoneal%20reinfection%20of%20the%20same%20agent&amp;author=A.%20Shishido&amp;author=M.%20Ohtawara&amp;author=S.%20Tateno&amp;publication_year=1958&amp;journal=Jpn.%20J.%20Med.%20Sci.%20Biol.&amp;volume=11&amp;pages=383-399" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7883/yoken1952.11.383" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7883%2Fyoken1952.11.383" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7883%2Fyoken1952.11.383"> </span></div> </div></div></div></div></div><div content-id="DSQ018C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Derrick</div> <div class="given-names">E.H.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">H.E.</div></div>. </div><div class="article-title">Isolation of the Karp strain of <em>rickettsia tsutsugamushi</em></div>, <div class="source">Lancet</div> , <div class="year">1949</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">150</div>-<div class="lpage">151</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20the%20Karp%20strain%20of%20rickettsia%20tsutsugamushi&amp;author=E.H.%20Derrick&amp;author=H.E.%20Brown&amp;publication_year=1949&amp;journal=Lancet&amp;volume=2&amp;pages=150-151" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0140-6736(49)90996-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(49)90996-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0140-6736(49)90996-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18146951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Characterization of a new antigenic type, Kuroki, of <em>Rickettsia tsutsugamushi</em> isolated from a patient in Japan</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="year">1990</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2111</div>-<div class="lpage">2113</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20new%20antigenic%20type%2C%20Kuroki%2C%20of%20Rickettsia%20tsutsugamushi%20isolated%20from%20a%20patient%20in%20Japan&amp;author=N.%20Ohashi&amp;author=A.%20Tamura&amp;author=H.%20Sakurai&amp;author=S.%20Yamamoto&amp;publication_year=1990&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=28&amp;pages=2111-2113" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2121793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ018C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ018C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawabata</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Immunological properties of <em>Rickettsia tsutsugamushi</em>, Kawasaki strain, isolated from a patient in Kyushu</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1986</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">611</div>-<div class="lpage">620</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunological%20properties%20of%20Rickettsia%20tsutsugamushi%2C%20Kawasaki%20strain%2C%20isolated%20from%20a%20patient%20in%20Kyushu&amp;author=S.%20Yamamoto&amp;author=N.%20Kawabata&amp;author=A.%20Tamura&amp;publication_year=1986&amp;journal=Microbiol.%20Immunol.&amp;volume=30&amp;pages=611-620" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1986.30.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1986.30.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1986.30.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3095612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353619&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq018-dsq018supp_Tables.xls?Expires=2147483647&amp;Signature=DtgAzBlWEHSWqfRQelHyBIxrb~mhsMr5ZT2f4kfZ3LHVBg4D3F5QP5uAVS-zzqF9MG4R-Vv3LKISgCtjOQnXp01Nip4OQUPVNwVxe3R3Wlp-AiI4yZSt6ZY7EG7o2EddHkBUcytOv8ks-n1DzHIVDfeD7VGASAm0EJCEkLvu~RQAT5r9n088WehY0~kHp6fxJI~1AfafNK8Ws3WCc8ZLgCNbb~cFUMSPmdz-G27hlCdSN-BZlBURHLishMxoArAZpDsQfnt7EDFDSXA-9ZcZsb9pQeh6ermW6uBFlmhDi6I7Qt2MpSDydsegsRjuFrARGBWFxPZCYyC7YVh~ADMCGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq018/2/dsq018-dsq018supp.pdf?Expires=2147483647&amp;Signature=hywxU87EZs3ARROtoJ0yGL5FGlkVOjhSBN173hzQDhijHSKu8IV2JDGXjX5BAkWKkqlIDvJq9ApafBZmpPzpPbh1uA3SFCQGir5gLBowKMc2n77af5Qfk2JHQKv9eBIyX03mCWKtEEIWkftaEG9i-QW6rnBHXfIfuK9yPCsf3FeMn6Z46~UN0BrbeAqXxi0smfvqu8raU1CUQ6bkFFkArJmWtOA8Drxvd9k6~7BfF2NMZDp87AZgfkPH60o3~MDTQmRcuqas07gxfWayS2hM5J5uXO5-e1xtkBgT3DdFggNOqprq6yhclAduJS9BpfOY9vkI272jNs48XGXJwlv2RA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">180</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000283631300003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">23</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq018"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353619"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000283631300003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f5%2f281%2f353619">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000283631300003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (23)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F5%2F281%2F353619" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353619'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome Comparison and Phylogenetic Analysis of Orientia tsutsugamushi Strains" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq018");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f281%2f353619">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

